

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/23/4/339/2469970","@type":"ScholarlyArticle","name":" Complete telomere-to-telomere  de novo  assembly of the  Plasmodium falciparum  genome through long-read (&gt;11 kb), single molecule, real-time sequencing ","datePublished":"2016-06-26","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/23/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2016-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw022","keywords":["<italic>Plasmodium falciparum</italic>","AT-biased","long-read sequencing","<italic>de novo</italic> assembly ","structural variation"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/23/4/339/2469970","author":[{"name":"Vembar, Shruthi Sridhar","affiliation":"Unité Biologie des Interactions Hôte-Parasite, Département de Parasites et Insectes Vecteurs, Institut Pasteur, Paris 75015, France    CNRS, ERL 9195, Paris 75015, France    INSERM, Unit U1201, Paris 75015, France","@type":"Person"},{"name":"Seetin, Matthew","affiliation":"Pacific Biosciences, Menlo Park, CA, USA","@type":"Person"},{"name":"Lambert, Christine","affiliation":"Pacific Biosciences, Menlo Park, CA, USA","@type":"Person"},{"name":"Nattestad, Maria","affiliation":"Cold Spring Harbor Laboratory, Cold Spring Harbor, NY, USA","@type":"Person"},{"name":"Schatz, Michael C.","affiliation":"Cold Spring Harbor Laboratory, Cold Spring Harbor, NY, USA    Department of Computer Science, Johns Hopkins University, Baltimore, MD, USA","@type":"Person"},{"name":"Baybayan, Primo","affiliation":"Pacific Biosciences, Menlo Park, CA, USA","@type":"Person"},{"name":"Scherf, Artur","affiliation":"Unité Biologie des Interactions Hôte-Parasite, Département de Parasites et Insectes Vecteurs, Institut Pasteur, Paris 75015, France    CNRS, ERL 9195, Paris 75015, France    INSERM, Unit U1201, Paris 75015, France","@type":"Person"},{"name":"Smith, Melissa Laird","affiliation":"Pacific Biosciences, Menlo Park, CA, USA","@type":"Person"}],"description":"Abstract.   The application of next-generation sequencing to estimate genetic diversity of Plasmodium falciparum , the most lethal malaria parasite, has proved ","pageStart":"339","pageEnd":"351","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/4/0/m_cover.gif?Expires=2147483647&Signature=XWSbFJdI5lGDaRny1g7BRzfZg~ebzc~OT20sUUxiNUi3-W~7uvtu5IDIgO3L5~QjmCQzUopLMGz-3gEHtIKUOFbRCHM2eH4h7l2Wu3nqAQC9YYHygJi-TLC9x3FpQRq3lTBcdWmpj185LJbB-1H-PKlUoEU9gxYyCNvjJUg~GUaHNRtVxbV0a7bDIfdyclZJ~w1BhGonEiYqh-IZSS0VekFWZ7mnLia~WbOEtL8AzSLsoIVak5-aADczuqIYGRkdfxcEh8zTdO-NlGhKxLoi5zlJy0Y~XuIxHZNxQxDFoVA3s~0J2fujGZsWX~ElpEU3I2Vma3M6MgFM80P2xPWpTw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":" Complete telomere-to-telomere  de novo  assembly of the  Plasmodium falciparum  genome through long-read (&gt;11 kb), single molecule, real-time sequencing ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/4/0/m_cover.gif?Expires=2147483647&Signature=XWSbFJdI5lGDaRny1g7BRzfZg~ebzc~OT20sUUxiNUi3-W~7uvtu5IDIgO3L5~QjmCQzUopLMGz-3gEHtIKUOFbRCHM2eH4h7l2Wu3nqAQC9YYHygJi-TLC9x3FpQRq3lTBcdWmpj185LJbB-1H-PKlUoEU9gxYyCNvjJUg~GUaHNRtVxbV0a7bDIfdyclZJ~w1BhGonEiYqh-IZSS0VekFWZ7mnLia~WbOEtL8AzSLsoIVak5-aADczuqIYGRkdfxcEh8zTdO-NlGhKxLoi5zlJy0Y~XuIxHZNxQxDFoVA3s~0J2fujGZsWX~ElpEU3I2Vma3M6MgFM80P2xPWpTw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content=" Complete telomere-to-telomere  de novo  assembly of the  Plasmodium falciparum  genome through long-read (&amp;gt;11 kb), single molecule, real-time sequencing " />
    <meta property="og:description" content="Abstract.   The application of next-generation sequencing to estimate genetic diversity of Plasmodium falciparum , the most lethal malaria parasite, has proved " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/23/4/339/2469970" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-06-26" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Vembar, Shruthi Sridhar" /><meta name="citation_author_institution" content=" Unité Biologie des Interactions Hôte-Parasite, Département de Parasites et Insectes Vecteurs, Institut Pasteur, Paris 75015, France " /><meta name="citation_author_institution" content=" CNRS, ERL 9195, Paris 75015, France " /><meta name="citation_author_institution" content=" INSERM, Unit U1201, Paris 75015, France " /><meta name="citation_author" content="Seetin, Matthew" /><meta name="citation_author_institution" content=" Pacific Biosciences, Menlo Park, CA, USA " /><meta name="citation_author" content="Lambert, Christine" /><meta name="citation_author_institution" content=" Pacific Biosciences, Menlo Park, CA, USA " /><meta name="citation_author" content="Nattestad, Maria" /><meta name="citation_author_institution" content=" Cold Spring Harbor Laboratory, Cold Spring Harbor, NY, USA " /><meta name="citation_author" content="Schatz, Michael C." /><meta name="citation_author_institution" content=" Cold Spring Harbor Laboratory, Cold Spring Harbor, NY, USA " /><meta name="citation_author_institution" content=" Department of Computer Science, Johns Hopkins University, Baltimore, MD, USA " /><meta name="citation_author" content="Baybayan, Primo" /><meta name="citation_author_institution" content=" Pacific Biosciences, Menlo Park, CA, USA " /><meta name="citation_author" content="Scherf, Artur" /><meta name="citation_author_institution" content=" Unité Biologie des Interactions Hôte-Parasite, Département de Parasites et Insectes Vecteurs, Institut Pasteur, Paris 75015, France " /><meta name="citation_author_institution" content=" CNRS, ERL 9195, Paris 75015, France " /><meta name="citation_author_institution" content=" INSERM, Unit U1201, Paris 75015, France " /><meta name="citation_author" content="Smith, Melissa Laird" /><meta name="citation_author_institution" content=" Pacific Biosciences, Menlo Park, CA, USA " /><meta name="citation_title" content=" Complete telomere-to-telomere de novo assembly of the Plasmodium falciparum genome through long-read (&gt;11 kb), single molecule, real-time sequencing " /><meta name="citation_firstpage" content="339" /><meta name="citation_lastpage" content="351" /><meta name="citation_doi" content="10.1093/dnares/dsw022" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="23" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2016/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Malaria biology and disease pathogenesis: insights for new treatments,; citation_author=Miller  L.H.; citation_author=Ackerman  H.C.; citation_author=Su  X.Z.; citation_author=Wellems  T.E.;  citation_journal_title=Nat. Med;  citation_year=2013;  citation_volume=19;  citation_pages=156-67; " /><meta name="citation_reference" content="citation_title=Genetic linkage and association analyses for trait mapping in Plasmodium falciparum ,; citation_author=Su  X.; citation_author=Hayton  K.; citation_author=Wellems  T.E.;  citation_journal_title=Nat. Rev. Genet;  citation_year=2007;  citation_volume=8;  citation_pages=497-506; " /><meta name="citation_reference" content="citation_title=Mitotic evolution of Plasmodium falciparum shows a stable core genome but recombination in antigen families,; citation_author=Bopp  S.E.; citation_author=Manary  M.J.; citation_author=Bright  A.T.;  citation_journal_title=PLoS Genet;  citation_year=2013;  citation_volume=9;  citation_pages=e1003293" /><meta name="citation_reference" content="citation_title=Generation of antigenic diversity in Plasmodium falciparum by structured rearrangement of Var genes during mitosis,; citation_author=Claessens  A.; citation_author=Hamilton  W.L.; citation_author=Kekre  M.;  citation_journal_title=PLoS Genet;  citation_year=2014;  citation_volume=10;  citation_pages=e1004812" /><meta name="citation_reference" content="citation_title=Frequent ectopic recombination of virulence factor genes in telomeric chromosome clusters of P. falciparum ,; citation_author=Freitas-Junior  L.H.; citation_author=Bottius  E.; citation_author=Pirrit  L.A.;  citation_journal_title=Nature;  citation_year=2000;  citation_volume=407;  citation_pages=1018-22; " /><meta name="citation_reference" content="citation_title=A molecular marker of artemisinin-resistant Plasmodium falciparum malaria,; citation_author=Ariey  F.; citation_author=Witkowski  B.; citation_author=Amaratunga  C.;  citation_journal_title=Nature;  citation_year=2014;  citation_volume=505;  citation_pages=50-5; " /><meta name="citation_reference" content="citation_title=Spiroindolones, a potent compound class for the treatment of malaria,; citation_author=Rottmann  M.; citation_author=McNamara  C.; citation_author=Yeung  B.K.;  citation_journal_title=Science;  citation_year=2010;  citation_volume=329;  citation_pages=1175-80; " /><meta name="citation_reference" content="citation_title=Genome sequence of the human malaria parasite Plasmodium falciparum ,; citation_author=Gardner  M.J.; citation_author=Hall  N.; citation_author=Fung  E.;  citation_journal_title=Nature;  citation_year=2002;  citation_volume=419;  citation_pages=498-511; " /><meta name="citation_reference" content="citation_title=The large diverse gene family var encodes proteins involved in cytoadherence and antigenic variation of Plasmodium falciparum -infected erythrocytes,; citation_author=Su  X.Z.; citation_author=Heatwole  V.M.; citation_author=Wertheimer  S.P.;  citation_journal_title=Cell;  citation_year=1995;  citation_volume=82;  citation_pages=89-100; " /><meta name="citation_reference" content="citation_title=Optimizing illumina next-generation sequencing library preparation for extremely AT-biased genomes,; citation_author=Oyola  S.O.; citation_author=Otto  T.D.; citation_author=Gu  Y.;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=1" /><meta name="citation_reference" content="citation_title=Optimized whole-genome amplification strategy for extremely AT-biased template,; citation_author=Oyola  S.O.; citation_author=Manske  M.; citation_author=Campino  S.;  citation_journal_title=DNA Res;  citation_year=2014;  citation_volume=21;  citation_pages=661-71; " /><meta name="citation_reference" content="citation_title=Optimal enzymes for amplifying sequencing libraries,; citation_author=Quail  M.A.; citation_author=Otto  T.D.; citation_author=Gu  Y.;  citation_journal_title=Nat. Methods;  citation_year=2012;  citation_volume=9;  citation_pages=10-11; " /><meta name="citation_reference" content="citation_title=A tale of three next generation sequencing platforms: comparison of Ion Torrent, Pacific Biosciences and Illumina MiSeq sequencers,; citation_author=Quail  M.A.; citation_author=Smith  M.; citation_author=Coupland  P.;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=341" /><meta name="citation_reference" content="citation_title=Characterizing and measuring bias in sequence data,; citation_author=Ross  M.G.; citation_author=Russ  C.; citation_author=Costello  M.;  citation_journal_title=Genome Biol;  citation_year=2013;  citation_volume=14;  citation_pages=R51" /><meta name="citation_reference" content="citation_author=Carlton  J.M.; citation_author=Sullivan  S.A.; citation_author=Le Roch  K.G.; citation_author=Carlton  J.M.; citation_author=Perkins  S.L.; citation_author=Deitsch  K.W.; citation_author=Carlton  J.M.; citation_author=Perkins  S.L.; citation_author=Deitsch  K.W.; citation_publisher=Caister Academic Press, Norfolk, UK;  citation_title=Malaria parasites: comparative genomics, evolution, and molecular biology;  citation_year=2013;  citation_pages=35-58; " /><meta name="citation_reference" content="citation_title=Malaria genomics: tracking a diverse and evolving parasite population,; citation_author=Kwiatkowski  D.;  citation_journal_title=Int. Health;  citation_year=2015;  citation_volume=7;  citation_pages=82-4; " /><meta name="citation_reference" content="citation_title=K13-propeller polymorphisms in Plasmodium falciparum parasites from sub-Saharan Africa; citation_author=Kamau  E.; citation_author=Campino  S.; citation_author=Amenga-Etego  L.;  citation_journal_title=J. Infect. Dis;  citation_year=2015;  citation_volume=211;  citation_pages=1352-5; " /><meta name="citation_reference" content="citation_title=Analysis of Plasmodium falciparum diversity in natural infections by deep sequencing,; citation_author=Manske  M.; citation_author=Miotto  O.; citation_author=Campino  S.;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=487;  citation_pages=375-9; " /><meta name="citation_reference" content="citation_title=Multiple populations of artemisinin-resistant Plasmodium falciparum in Cambodia,; citation_author=Miotto  O.; citation_author=Almagro-Garcia  J.; citation_author=Manske  M.;  citation_journal_title=Nat. Genet;  citation_year=2013;  citation_volume=45;  citation_pages=648-55; " /><meta name="citation_reference" content="citation_title=Genetic architecture of artemisinin-resistant Plasmodium falciparum ,; citation_author=Miotto  O.; citation_author=Amato  R.; citation_author=Ashley  E.A.;  citation_journal_title=Nat. Genet;  citation_year=2015;  citation_volume=47;  citation_pages=226-34; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of selection on the malaria parasite Plasmodium falciparum in West African populations of differing infection endemicity,; citation_author=Mobegi  V.A.; citation_author=Duffy  C.W.; citation_author=Amambua-Ngwa  A.;  citation_journal_title=Mol. Biol. Evol;  citation_year=2014;  citation_volume=31;  citation_pages=1490-9; " /><meta name="citation_reference" content="citation_title=Single-cell genomics for dissection of complex malaria infections,; citation_author=Nair  S.; citation_author=Nkhoma  S.C.; citation_author=Serre  D.;  citation_journal_title=Genome Res;  citation_year=2014;  citation_volume=24;  citation_pages=1028-38; " /><meta name="citation_reference" content="citation_title=Challenges and opportunities in estimating viral genetic diversity from next-generation sequencing data,; citation_author=Beerenwinkel  N.; citation_author=Günthard  H.F.; citation_author=Roth  V.; citation_author=Metzner  K.J.;  citation_journal_title=Front. Microbiol;  citation_year=2012;  citation_volume=3;  citation_pages=329" /><meta name="citation_reference" content="citation_title=Real-time DNA sequencing from single polymerase molecules,; citation_author=Eid  J.; citation_author=Fehr  A.; citation_author=Gray  J.;  citation_journal_title=Science;  citation_year=2009;  citation_volume=323;  citation_pages=133-8; " /><meta name="citation_reference" content="citation_title=Genetic variation and the de novo assembly of human genomes,; citation_author=Chaisson  M.J.; citation_author=Wilson  R.K.; citation_author=Eichler  E.E.;  citation_journal_title=Nat. Rev. Genet;  citation_year=2015;  citation_volume=16;  citation_pages=627-40; " /><meta name="citation_reference" content="citation_title=Multiplexed highly-accurate DNA sequencing of closely-related HIV-1 variants using continuous long reads from single molecule, real-time sequencing; citation_author=Dilernia  D.A.; citation_author=Chien  J.T.; citation_author=Monaco  D.C.;  citation_journal_title=Nucleic Acids Res;  citation_year=2015;  citation_volume=43;  citation_pages=e129" /><meta name="citation_reference" content="citation_title=Assessing structural variation in a personal genome-towards a human reference diploid genome,; citation_author=English  A.C.; citation_author=Salerno  W.J.; citation_author=Hampton  O.A.;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=286" /><meta name="citation_reference" content="citation_title=Resolving the complexity of the human genome using single-molecule sequencing,; citation_author=Chaisson  M.J.; citation_author=Huddleston  J.; citation_author=Dennis  M.Y.;  citation_journal_title=Nature;  citation_year=2015;  citation_volume=517;  citation_pages=608-11; " /><meta name="citation_reference" content="citation_title=Nonhybrid, finished microbial genome assemblies from long-read SMRT sequencing data,; citation_author=Chin  C.S.; citation_author=Alexander  D.H.; citation_author=Marks  P.;  citation_journal_title=Nat. Methods;  citation_year=2013;  citation_volume=10;  citation_pages=563-9; " /><meta name="citation_reference" content="citation_title=Hybrid error correction and de novo assembly of single-molecule sequencing reads; citation_author=Koren  S.; citation_author=Schatz  M.C.; citation_author=Walenz  B.P.;  citation_journal_title=Nat. Biotechnol;  citation_year=2012;  citation_volume=30;  citation_pages=693-700; " /><meta name="citation_reference" content="citation_title=Pacific biosciences sequencing technology for genotyping and variation discovery in human data,; citation_author=Carneiro  M.O.; citation_author=Russ  C.; citation_author=Ross  M.G.; citation_author=Gabriel  S.B.; citation_author=Nusbaum  C.; citation_author=DePristo  M.A.;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=375" /><meta name="citation_reference" content="citation_title=Human malaria parasites in continuous culture,; citation_author=Trager  W.; citation_author=Jensen  J.B.;  citation_journal_title=Science;  citation_year=1976;  citation_volume=193;  citation_pages=673-5; " /><meta name="citation_reference" content="citation_title=An alternative method for Plasmodium culture synchronization,; citation_author=Lelièvre  J.; citation_author=Berry  A.; citation_author=Benoit-Vical  F.;  citation_journal_title=Exp. Parasitol;  citation_year=2005;  citation_volume=109;  citation_pages=195-7; " /><meta name="citation_reference" content="citation_title=Synchronization of Plasmodium falciparum erythrocytic stages in culture,; citation_author=Lambros  C.; citation_author=Vanderberg  J.P.;  citation_journal_title=J. Parasitol;  citation_year=1979;  citation_volume=65;  citation_pages=418-20; " /><meta name="citation_reference" content="Moll, K., Kaneko, A., Scherf, A., Wahlgren, M. 2013 Methods in Malaria Research, 6th edition. " /><meta name="citation_reference" content="citation_title=A whole-genome assembly of Drosophila,; citation_author=Myers  E.W.; citation_author=Sutton  G.G.; citation_author=Delcher  A.L.;  citation_journal_title=Science;  citation_year=2000;  citation_volume=287;  citation_pages=2196-204; " /><meta name="citation_reference" content="citation_title=Mapping single molecule sequencing reads using basic local alignment with successive refinement (BLASR): application and theory,; citation_author=Chaisson  M.J.; citation_author=Tesler  G.;  citation_journal_title=BMC Bioinformatics;  citation_year=2012;  citation_volume=13;  citation_pages=238" /><meta name="citation_reference" content="citation_title=Gepard: a rapid and sensitive tool for creating dotplots on genome scale,; citation_author=Krumsiek  J.; citation_author=Arnold  R.; citation_author=Rattei  T.;  citation_journal_title=Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=1026-8; " /><meta name="citation_reference" content="citation_title=Assemblytics: a web analytics tool for the detection of assembly-based variants,; citation_author=Nattestad  M.; citation_author=Schatz  M.C.;  citation_journal_title=bioRxiv;  citation_year=2016; " /><meta name="citation_reference" content="citation_title=Versatile and open software for comparing large genomes,; citation_author=Kurtz  S.; citation_author=Phillippy  A.; citation_author=Delcher  A.L.;  citation_journal_title=Genome Biol;  citation_year=2004;  citation_volume=5;  citation_pages=R12" /><meta name="citation_reference" content="citation_title=BEDTools: a flexible suite of utilities for comparing genomic features,; citation_author=Quinlan  A.R.; citation_author=Hall  I.M.;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=841-2; " /><meta name="citation_reference" content="citation_title=Identification of the heme compound copurified with deoxyribonucleic acid (DNA) from bloodstains, a major inhibitor of polymerase chain reaction (PCR) amplification,; citation_author=Akane  A.; citation_author=Matsubara  K.; citation_author=Nakamura  H.; citation_author=Takahashi  S.; citation_author=Kimura  K.;  citation_journal_title=J. Forensic Sci;  citation_year=1994;  citation_volume=39;  citation_pages=362-72; " /><meta name="citation_reference" content="citation_title=Mutants of Taq DNA polymerase resistant to PCR inhibitors allow DNA amplification from whole blood and crude soil samples,; citation_author=Kermekchiev  M.B.; citation_author=Kirilova  L.I.; citation_author=Vail  E.E.; citation_author=Barnes  W.M.;  citation_journal_title=Nucleic Acids Res;  citation_year=2009;  citation_volume=37;  citation_pages=e40" /><meta name="citation_reference" content="citation_title=Direct DNA amplification from crude clinical samples using a PCR enhancer cocktail and novel mutants of Taq,; citation_author=Zhang  Z.; citation_author=Kermekchiev  M.B.; citation_author=Barnes  W.M.;  citation_journal_title=J. Mol. Diagn;  citation_year=2010;  citation_volume=12;  citation_pages=152-61; " /><meta name="citation_reference" content="citation_title=High-throughput 454 resequencing for allele discovery and recombination mapping in Plasmodium falciparum ,; citation_author=Samarakoon  U.; citation_author=Regier  A.; citation_author=Tan  A.;  citation_journal_title=BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=116" /><meta name="citation_reference" content="citation_title=A genome-wide map of diversity in Plasmodium falciparum ,; citation_author=Volkman  S.K.; citation_author=Sabeti  P.C.; citation_author=DeCaprio  D.;  citation_journal_title=Nat. Genet;  citation_year=2007;  citation_volume=39;  citation_pages=113-9; " /><meta name="citation_reference" content="citation_title=Amplification-free Illumina sequencing-library preparation facilitates improved mapping and assembly of (G+C)-biased genomes,; citation_author=Kozarewa  I.; citation_author=Ning  Z.; citation_author=Quail  M.A.; citation_author=Sanders  M.J.; citation_author=Berriman  M.; citation_author=Turner  D.J.;  citation_journal_title=Nat. Methods;  citation_year=2009;  citation_volume=6;  citation_pages=291-5; " /><meta name="citation_reference" content="citation_title=Plasmodium falciparum centromeres display a unique epigenetic makeup and cluster prior to and during schizogony,; citation_author=Hoeijmakers  W.A.; citation_author=Flueck  C.; citation_author=Francoijs  K.J.;  citation_journal_title=Cell. Microbiol;  citation_year=2012;  citation_volume=14;  citation_pages=1391-401; " /><meta name="citation_reference" content="citation_title=Evidence on the chromosomal location of centromeric DNA in Plasmodium falciparum from etoposide-mediated topoisomerase-II cleavage,; citation_author=Kelly  J.M.; citation_author=McRobert  L.; citation_author=Baker  D.A.;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2006;  citation_volume=103;  citation_pages=6706-11; " /><meta name="citation_reference" content="citation_title=A central role for Plasmodium falciparum subtelomeric regions in spatial positioning and telomere length regulation,; citation_author=Figueiredo  L.M.; citation_author=Freitas-Junior  L.H.; citation_author=Bottius  E.; citation_author=Olivo-Marin  J.C.; citation_author=Scherf  A.;  citation_journal_title=EMBO J;  citation_year=2002;  citation_volume=21;  citation_pages=815-24; " /><meta name="citation_reference" content="citation_title=RIFINs are adhesins implicated in severe Plasmodium falciparum malaria,; citation_author=Goel  S.; citation_author=Palmkvist  M.; citation_author=Moll  K.;  citation_journal_title=Nat. Med;  citation_year=2015;  citation_volume=21;  citation_pages=314-7; " /><meta name="citation_reference" content="citation_title=Rifins: a second family of clonally variant proteins expressed on the surface of red cells infected with Plasmodium falciparum ,; citation_author=Kyes  S.A.; citation_author=Rowe  J.A.; citation_author=Kriek  N.; citation_author=Newbold  C.I.;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=1999;  citation_volume=96;  citation_pages=9333-8; " /><meta name="citation_reference" content="citation_title=Monitoring parasite diversity for malaria elimination in sub-Saharan Africa,; citation_author=Ghansah  A.; citation_author=Amenga-Etego  L.; citation_author=Amambua-Ngwa  A.;  citation_journal_title=Science;  citation_year=2014;  citation_volume=345;  citation_pages=1297-8; " /><meta name="citation_reference" content="citation_title=Efficient depletion of host DNA contamination in malaria clinical sequencing,; citation_author=Oyola  S.O.; citation_author=Gu  Y.; citation_author=Manske  M.;  citation_journal_title=J. Clin. Microbiol;  citation_year=2013;  citation_volume=51;  citation_pages=745-51; " /><meta name="citation_reference" content="citation_title=An adjustable gas-mixing device to increase feasibility of in vitro culture of Plasmodium falciparum parasites in the field,; citation_author=Bei  A.K.; citation_author=Patel  S.D.; citation_author=Volkman  S.K.;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e90928" /><meta name="citation_reference" content="citation_title=Improved in vitro culture of Plasmodium falciparum permits establishment of clinical isolates with preserved multiplication, invasion and rosetting phenotypes,; citation_author=Ribacke  U.; citation_author=Moll  K.; citation_author=Albrecht  L.;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e69781" /><meta name="citation_reference" content="citation_title=Going beyond five bases in DNA sequencing,; citation_author=Korlach  J.; citation_author=Turner  S.W.;  citation_journal_title=Curr. Opin. Struct. Biol;  citation_year=2012;  citation_volume=22;  citation_pages=251-61; " /><meta name="citation_reference" content="citation_title=Entering the era of bacterial epigenomics with single molecule real time DNA sequencing,; citation_author=Davis  B.M.; citation_author=Chao  M.C.; citation_author=Waldor  M.K.;  citation_journal_title=Curr. Opin. Microbiol;  citation_year=2013;  citation_volume=16;  citation_pages=192-8; " /><meta name="citation_reference" content="citation_title=DNA methylation on N6-Adenine in C. elegans ,; citation_author=Greer  E.L.; citation_author=Blanco  M.A.; citation_author=Gu  L.;  citation_journal_title=Cell;  citation_year=2015;  citation_volume=161;  citation_pages=868-78; " /><meta name="citation_reference" content="citation_title=DNA methylation on N(6)-adenine in mammalian embryonic stem cells; citation_author=Wu  T.P.; citation_author=Wang  T.; citation_author=Seetin  M.G.;  citation_journal_title=Nature;  citation_year=2016;  citation_volume=532;  citation_pages=329-333; " /><meta name="citation_reference" content="citation_title=Genome-wide mapping of DNA methylation in the human malaria parasite Plasmodium falciparum ,; citation_author=Ponts  N.; citation_author=Fu  L.; citation_author=Harris  E.Y.;  citation_journal_title=Cell Host Microbe;  citation_year=2013;  citation_volume=14;  citation_pages=696-706; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/23/4/339/17463165/dsw022.pdf" /><meta name="description" content="Abstract.   The application of next-generation sequencing to estimate genetic diversity of Plasmodium falciparum , the most lethal malaria parasite, has proved " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/23/4/339/2469970" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/23/4/339/2469970" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/23/4/339/2469970");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw022';
    </script>
<title>
	 Complete telomere-to-telomere de novo assembly of the Plasmodium falciparum genome through long-read (&gt;11 kb), single molecule, real-time sequencing  | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2469970" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Baq8ATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVmVtYmFyLCBTaHJ1dGhpIFNyaWRoYXIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIgVW5pdMOpIEJpb2xvZ2llIGRlcyBJbnRlcmFjdGlvbnMgSMO0dGUtUGFyYXNpdGUsIETDqXBhcnRlbWVudCBkZSBQYXJhc2l0ZXMgZXQgSW5zZWN0ZXMgVmVjdGV1cnMsIEluc3RpdHV0IFBhc3RldXIsIFBhcmlzIDc1MDE1LCBGcmFuY2UgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iIENOUlMsIEVSTCA5MTk1LCBQYXJpcyA3NTAxNSwgRnJhbmNlICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IiBJTlNFUk0sIFVuaXQgVTEyMDEsIFBhcmlzIDc1MDE1LCBGcmFuY2UgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2VldGluLCBNYXR0aGV3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iIFBhY2lmaWMgQmlvc2NpZW5jZXMsIE1lbmxvIFBhcmssIENBLCBVU0EgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGFtYmVydCwgQ2hyaXN0aW5lIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iIFBhY2lmaWMgQmlvc2NpZW5jZXMsIE1lbmxvIFBhcmssIENBLCBVU0EgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmF0dGVzdGFkLCBNYXJpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IiBDb2xkIFNwcmluZyBIYXJib3IgTGFib3JhdG9yeSwgQ29sZCBTcHJpbmcgSGFyYm9yLCBOWSwgVVNBICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNjaGF0eiwgTWljaGFlbCBDLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IiBDb2xkIFNwcmluZyBIYXJib3IgTGFib3JhdG9yeSwgQ29sZCBTcHJpbmcgSGFyYm9yLCBOWSwgVVNBICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IiBEZXBhcnRtZW50IG9mIENvbXB1dGVyIFNjaWVuY2UsIEpvaG5zIEhvcGtpbnMgVW5pdmVyc2l0eSwgQmFsdGltb3JlLCBNRCwgVVNBICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkJheWJheWFuLCBQcmltbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IiBQYWNpZmljIEJpb3NjaWVuY2VzLCBNZW5sbyBQYXJrLCBDQSwgVVNBICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNjaGVyZiwgQXJ0dXIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIgVW5pdMOpIEJpb2xvZ2llIGRlcyBJbnRlcmFjdGlvbnMgSMO0dGUtUGFyYXNpdGUsIETDqXBhcnRlbWVudCBkZSBQYXJhc2l0ZXMgZXQgSW5zZWN0ZXMgVmVjdGV1cnMsIEluc3RpdHV0IFBhc3RldXIsIFBhcmlzIDc1MDE1LCBGcmFuY2UgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iIENOUlMsIEVSTCA5MTk1LCBQYXJpcyA3NTAxNSwgRnJhbmNlICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IiBJTlNFUk0sIFVuaXQgVTEyMDEsIFBhcmlzIDc1MDE1LCBGcmFuY2UgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU21pdGgsIE1lbGlzc2EgTGFpcmQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIgUGFjaWZpYyBCaW9zY2llbmNlcywgTWVubG8gUGFyaywgQ0EsIFVTQSAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IiBDb21wbGV0ZSB0ZWxvbWVyZS10by10ZWxvbWVyZSBkZSBub3ZvIGFzc2VtYmx5IG9mIHRoZSBQbGFzbW9kaXVtIGZhbGNpcGFydW0gZ2Vub21lIHRocm91Z2ggbG9uZy1yZWFkICgmZ3Q7MTHigIlrYiksIHNpbmdsZSBtb2xlY3VsZSwgcmVhbC10aW1lIHNlcXVlbmNpbmcgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMzM5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIzNTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3cwMjIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNi8wOC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hbGFyaWEgYmlvbG9neSBhbmQgZGlzZWFzZSBwYXRob2dlbmVzaXM6IGluc2lnaHRzIGZvciBuZXcgdHJlYXRtZW50cyw7IGNpdGF0aW9uX2F1dGhvcj1NaWxsZXIgIEwuSC47IGNpdGF0aW9uX2F1dGhvcj1BY2tlcm1hbiAgSC5DLjsgY2l0YXRpb25fYXV0aG9yPVN1ICBYLlouOyBjaXRhdGlvbl9hdXRob3I9V2VsbGVtcyAgVC5FLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBNZWQ7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0xNTYtNjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgbGlua2FnZSBhbmQgYXNzb2NpYXRpb24gYW5hbHlzZXMgZm9yIHRyYWl0IG1hcHBpbmcgaW4gUGxhc21vZGl1bSBmYWxjaXBhcnVtICw7IGNpdGF0aW9uX2F1dGhvcj1TdSAgWC47IGNpdGF0aW9uX2F1dGhvcj1IYXl0b24gIEsuOyBjaXRhdGlvbl9hdXRob3I9V2VsbGVtcyAgVC5FLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz00OTctNTA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaXRvdGljIGV2b2x1dGlvbiBvZiBQbGFzbW9kaXVtIGZhbGNpcGFydW0gc2hvd3MgYSBzdGFibGUgY29yZSBnZW5vbWUgYnV0IHJlY29tYmluYXRpb24gaW4gYW50aWdlbiBmYW1pbGllcyw7IGNpdGF0aW9uX2F1dGhvcj1Cb3BwICBTLkUuOyBjaXRhdGlvbl9hdXRob3I9TWFuYXJ5ICBNLkouOyBjaXRhdGlvbl9hdXRob3I9QnJpZ2h0ICBBLlQuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMzI5MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmVyYXRpb24gb2YgYW50aWdlbmljIGRpdmVyc2l0eSBpbiBQbGFzbW9kaXVtIGZhbGNpcGFydW0gYnkgc3RydWN0dXJlZCByZWFycmFuZ2VtZW50IG9mIFZhciBnZW5lcyBkdXJpbmcgbWl0b3Npcyw7IGNpdGF0aW9uX2F1dGhvcj1DbGFlc3NlbnMgIEEuOyBjaXRhdGlvbl9hdXRob3I9SGFtaWx0b24gIFcuTC47IGNpdGF0aW9uX2F1dGhvcj1LZWtyZSAgTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz1lMTAwNDgxMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZyZXF1ZW50IGVjdG9waWMgcmVjb21iaW5hdGlvbiBvZiB2aXJ1bGVuY2UgZmFjdG9yIGdlbmVzIGluIHRlbG9tZXJpYyBjaHJvbW9zb21lIGNsdXN0ZXJzIG9mIFAuIGZhbGNpcGFydW0gLDsgY2l0YXRpb25fYXV0aG9yPUZyZWl0YXMtSnVuaW9yICBMLkguOyBjaXRhdGlvbl9hdXRob3I9Qm90dGl1cyAgRS47IGNpdGF0aW9uX2F1dGhvcj1QaXJyaXQgIEwuQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT00MDc7ICBjaXRhdGlvbl9wYWdlcz0xMDE4LTIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG1vbGVjdWxhciBtYXJrZXIgb2YgYXJ0ZW1pc2luaW4tcmVzaXN0YW50IFBsYXNtb2RpdW0gZmFsY2lwYXJ1bSBtYWxhcmlhLDsgY2l0YXRpb25fYXV0aG9yPUFyaWV5ICBGLjsgY2l0YXRpb25fYXV0aG9yPVdpdGtvd3NraSAgQi47IGNpdGF0aW9uX2F1dGhvcj1BbWFyYXR1bmdhICBDLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTUwNTsgIGNpdGF0aW9uX3BhZ2VzPTUwLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNwaXJvaW5kb2xvbmVzLCBhIHBvdGVudCBjb21wb3VuZCBjbGFzcyBmb3IgdGhlIHRyZWF0bWVudCBvZiBtYWxhcmlhLDsgY2l0YXRpb25fYXV0aG9yPVJvdHRtYW5uICBNLjsgY2l0YXRpb25fYXV0aG9yPU1jTmFtYXJhICBDLjsgY2l0YXRpb25fYXV0aG9yPVlldW5nICBCLksuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTMyOTsgIGNpdGF0aW9uX3BhZ2VzPTExNzUtODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgaHVtYW4gbWFsYXJpYSBwYXJhc2l0ZSBQbGFzbW9kaXVtIGZhbGNpcGFydW0gLDsgY2l0YXRpb25fYXV0aG9yPUdhcmRuZXIgIE0uSi47IGNpdGF0aW9uX2F1dGhvcj1IYWxsICBOLjsgY2l0YXRpb25fYXV0aG9yPUZ1bmcgIEUuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9NDE5OyAgY2l0YXRpb25fcGFnZXM9NDk4LTUxMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGxhcmdlIGRpdmVyc2UgZ2VuZSBmYW1pbHkgdmFyIGVuY29kZXMgcHJvdGVpbnMgaW52b2x2ZWQgaW4gY3l0b2FkaGVyZW5jZSBhbmQgYW50aWdlbmljIHZhcmlhdGlvbiBvZiBQbGFzbW9kaXVtIGZhbGNpcGFydW0gLWluZmVjdGVkIGVyeXRocm9jeXRlcyw7IGNpdGF0aW9uX2F1dGhvcj1TdSAgWC5aLjsgY2l0YXRpb25fYXV0aG9yPUhlYXR3b2xlICBWLk0uOyBjaXRhdGlvbl9hdXRob3I9V2VydGhlaW1lciAgUy5QLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT04MjsgIGNpdGF0aW9uX3BhZ2VzPTg5LTEwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3B0aW1pemluZyBpbGx1bWluYSBuZXh0LWdlbmVyYXRpb24gc2VxdWVuY2luZyBsaWJyYXJ5IHByZXBhcmF0aW9uIGZvciBleHRyZW1lbHkgQVQtYmlhc2VkIGdlbm9tZXMsOyBjaXRhdGlvbl9hdXRob3I9T3lvbGEgIFMuTy47IGNpdGF0aW9uX2F1dGhvcj1PdHRvICBULkQuOyBjaXRhdGlvbl9hdXRob3I9R3UgIFkuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0xIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3B0aW1pemVkIHdob2xlLWdlbm9tZSBhbXBsaWZpY2F0aW9uIHN0cmF0ZWd5IGZvciBleHRyZW1lbHkgQVQtYmlhc2VkIHRlbXBsYXRlLDsgY2l0YXRpb25fYXV0aG9yPU95b2xhICBTLk8uOyBjaXRhdGlvbl9hdXRob3I9TWFuc2tlICBNLjsgY2l0YXRpb25fYXV0aG9yPUNhbXBpbm8gIFMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9NjYxLTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcHRpbWFsIGVuenltZXMgZm9yIGFtcGxpZnlpbmcgc2VxdWVuY2luZyBsaWJyYXJpZXMsOyBjaXRhdGlvbl9hdXRob3I9UXVhaWwgIE0uQS47IGNpdGF0aW9uX2F1dGhvcj1PdHRvICBULkQuOyBjaXRhdGlvbl9hdXRob3I9R3UgIFkuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTEwLTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHRhbGUgb2YgdGhyZWUgbmV4dCBnZW5lcmF0aW9uIHNlcXVlbmNpbmcgcGxhdGZvcm1zOiBjb21wYXJpc29uIG9mIElvbiBUb3JyZW50LCBQYWNpZmljIEJpb3NjaWVuY2VzIGFuZCBJbGx1bWluYSBNaVNlcSBzZXF1ZW5jZXJzLDsgY2l0YXRpb25fYXV0aG9yPVF1YWlsICBNLkEuOyBjaXRhdGlvbl9hdXRob3I9U21pdGggIE0uOyBjaXRhdGlvbl9hdXRob3I9Q291cGxhbmQgIFAuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0zNDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpemluZyBhbmQgbWVhc3VyaW5nIGJpYXMgaW4gc2VxdWVuY2UgZGF0YSw7IGNpdGF0aW9uX2F1dGhvcj1Sb3NzICBNLkcuOyBjaXRhdGlvbl9hdXRob3I9UnVzcyAgQy47IGNpdGF0aW9uX2F1dGhvcj1Db3N0ZWxsbyAgTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9UjUxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fYXV0aG9yPUNhcmx0b24gIEouTS47IGNpdGF0aW9uX2F1dGhvcj1TdWxsaXZhbiAgUy5BLjsgY2l0YXRpb25fYXV0aG9yPUxlIFJvY2ggIEsuRy47IGNpdGF0aW9uX2F1dGhvcj1DYXJsdG9uICBKLk0uOyBjaXRhdGlvbl9hdXRob3I9UGVya2lucyAgUy5MLjsgY2l0YXRpb25fYXV0aG9yPURlaXRzY2ggIEsuVy47IGNpdGF0aW9uX2F1dGhvcj1DYXJsdG9uICBKLk0uOyBjaXRhdGlvbl9hdXRob3I9UGVya2lucyAgUy5MLjsgY2l0YXRpb25fYXV0aG9yPURlaXRzY2ggIEsuVy47IGNpdGF0aW9uX3B1Ymxpc2hlcj1DYWlzdGVyIEFjYWRlbWljIFByZXNzLCBOb3Jmb2xrLCBVSzsgIGNpdGF0aW9uX3RpdGxlPU1hbGFyaWEgcGFyYXNpdGVzOiBjb21wYXJhdGl2ZSBnZW5vbWljcywgZXZvbHV0aW9uLCBhbmQgbW9sZWN1bGFyIGJpb2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl9wYWdlcz0zNS01ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFsYXJpYSBnZW5vbWljczogdHJhY2tpbmcgYSBkaXZlcnNlIGFuZCBldm9sdmluZyBwYXJhc2l0ZSBwb3B1bGF0aW9uLDsgY2l0YXRpb25fYXV0aG9yPUt3aWF0a293c2tpICBELjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SW50LiBIZWFsdGg7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTgyLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUsxMy1wcm9wZWxsZXIgcG9seW1vcnBoaXNtcyBpbiBQbGFzbW9kaXVtIGZhbGNpcGFydW0gcGFyYXNpdGVzIGZyb20gc3ViLVNhaGFyYW4gQWZyaWNhOyBjaXRhdGlvbl9hdXRob3I9S2FtYXUgIEUuOyBjaXRhdGlvbl9hdXRob3I9Q2FtcGlubyAgUy47IGNpdGF0aW9uX2F1dGhvcj1BbWVuZ2EtRXRlZ28gIEwuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBJbmZlY3QuIERpczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTE7ICBjaXRhdGlvbl9wYWdlcz0xMzUyLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIFBsYXNtb2RpdW0gZmFsY2lwYXJ1bSBkaXZlcnNpdHkgaW4gbmF0dXJhbCBpbmZlY3Rpb25zIGJ5IGRlZXAgc2VxdWVuY2luZyw7IGNpdGF0aW9uX2F1dGhvcj1NYW5za2UgIE0uOyBjaXRhdGlvbl9hdXRob3I9TWlvdHRvICBPLjsgY2l0YXRpb25fYXV0aG9yPUNhbXBpbm8gIFMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDg3OyAgY2l0YXRpb25fcGFnZXM9Mzc1LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11bHRpcGxlIHBvcHVsYXRpb25zIG9mIGFydGVtaXNpbmluLXJlc2lzdGFudCBQbGFzbW9kaXVtIGZhbGNpcGFydW0gaW4gQ2FtYm9kaWEsOyBjaXRhdGlvbl9hdXRob3I9TWlvdHRvICBPLjsgY2l0YXRpb25fYXV0aG9yPUFsbWFncm8tR2FyY2lhICBKLjsgY2l0YXRpb25fYXV0aG9yPU1hbnNrZSAgTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDU7ICBjaXRhdGlvbl9wYWdlcz02NDgtNTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgYXJjaGl0ZWN0dXJlIG9mIGFydGVtaXNpbmluLXJlc2lzdGFudCBQbGFzbW9kaXVtIGZhbGNpcGFydW0gLDsgY2l0YXRpb25fYXV0aG9yPU1pb3R0byAgTy47IGNpdGF0aW9uX2F1dGhvcj1BbWF0byAgUi47IGNpdGF0aW9uX2F1dGhvcj1Bc2hsZXkgIEUuQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NDc7ICBjaXRhdGlvbl9wYWdlcz0yMjYtMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFuYWx5c2lzIG9mIHNlbGVjdGlvbiBvbiB0aGUgbWFsYXJpYSBwYXJhc2l0ZSBQbGFzbW9kaXVtIGZhbGNpcGFydW0gaW4gV2VzdCBBZnJpY2FuIHBvcHVsYXRpb25zIG9mIGRpZmZlcmluZyBpbmZlY3Rpb24gZW5kZW1pY2l0eSw7IGNpdGF0aW9uX2F1dGhvcj1Nb2JlZ2kgIFYuQS47IGNpdGF0aW9uX2F1dGhvcj1EdWZmeSAgQy5XLjsgY2l0YXRpb25fYXV0aG9yPUFtYW1idWEtTmd3YSAgQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTE0OTAtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2luZ2xlLWNlbGwgZ2Vub21pY3MgZm9yIGRpc3NlY3Rpb24gb2YgY29tcGxleCBtYWxhcmlhIGluZmVjdGlvbnMsOyBjaXRhdGlvbl9hdXRob3I9TmFpciAgUy47IGNpdGF0aW9uX2F1dGhvcj1Oa2hvbWEgIFMuQy47IGNpdGF0aW9uX2F1dGhvcj1TZXJyZSAgRC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xMDI4LTM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFsbGVuZ2VzIGFuZCBvcHBvcnR1bml0aWVzIGluIGVzdGltYXRpbmcgdmlyYWwgZ2VuZXRpYyBkaXZlcnNpdHkgZnJvbSBuZXh0LWdlbmVyYXRpb24gc2VxdWVuY2luZyBkYXRhLDsgY2l0YXRpb25fYXV0aG9yPUJlZXJlbndpbmtlbCAgTi47IGNpdGF0aW9uX2F1dGhvcj1Hw7xudGhhcmQgIEguRi47IGNpdGF0aW9uX2F1dGhvcj1Sb3RoICBWLjsgY2l0YXRpb25fYXV0aG9yPU1ldHpuZXIgIEsuSi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZyb250LiBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTMyOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlYWwtdGltZSBETkEgc2VxdWVuY2luZyBmcm9tIHNpbmdsZSBwb2x5bWVyYXNlIG1vbGVjdWxlcyw7IGNpdGF0aW9uX2F1dGhvcj1FaWQgIEouOyBjaXRhdGlvbl9hdXRob3I9RmVociAgQS47IGNpdGF0aW9uX2F1dGhvcj1HcmF5ICBKLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zMjM7ICBjaXRhdGlvbl9wYWdlcz0xMzMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyB2YXJpYXRpb24gYW5kIHRoZSBkZSBub3ZvIGFzc2VtYmx5IG9mIGh1bWFuIGdlbm9tZXMsOyBjaXRhdGlvbl9hdXRob3I9Q2hhaXNzb24gIE0uSi47IGNpdGF0aW9uX2F1dGhvcj1XaWxzb24gIFIuSy47IGNpdGF0aW9uX2F1dGhvcj1FaWNobGVyICBFLkUuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz02MjctNDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11bHRpcGxleGVkIGhpZ2hseS1hY2N1cmF0ZSBETkEgc2VxdWVuY2luZyBvZiBjbG9zZWx5LXJlbGF0ZWQgSElWLTEgdmFyaWFudHMgdXNpbmcgY29udGludW91cyBsb25nIHJlYWRzIGZyb20gc2luZ2xlIG1vbGVjdWxlLCByZWFsLXRpbWUgc2VxdWVuY2luZzsgY2l0YXRpb25fYXV0aG9yPURpbGVybmlhICBELkEuOyBjaXRhdGlvbl9hdXRob3I9Q2hpZW4gIEouVC47IGNpdGF0aW9uX2F1dGhvcj1Nb25hY28gIEQuQy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9ZTEyOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzc2Vzc2luZyBzdHJ1Y3R1cmFsIHZhcmlhdGlvbiBpbiBhIHBlcnNvbmFsIGdlbm9tZS10b3dhcmRzIGEgaHVtYW4gcmVmZXJlbmNlIGRpcGxvaWQgZ2Vub21lLDsgY2l0YXRpb25fYXV0aG9yPUVuZ2xpc2ggIEEuQy47IGNpdGF0aW9uX2F1dGhvcj1TYWxlcm5vICBXLkouOyBjaXRhdGlvbl9hdXRob3I9SGFtcHRvbiAgTy5BLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9Mjg2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVzb2x2aW5nIHRoZSBjb21wbGV4aXR5IG9mIHRoZSBodW1hbiBnZW5vbWUgdXNpbmcgc2luZ2xlLW1vbGVjdWxlIHNlcXVlbmNpbmcsOyBjaXRhdGlvbl9hdXRob3I9Q2hhaXNzb24gIE0uSi47IGNpdGF0aW9uX2F1dGhvcj1IdWRkbGVzdG9uICBKLjsgY2l0YXRpb25fYXV0aG9yPURlbm5pcyAgTS5ZLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTUxNzsgIGNpdGF0aW9uX3BhZ2VzPTYwOC0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm9uaHlicmlkLCBmaW5pc2hlZCBtaWNyb2JpYWwgZ2Vub21lIGFzc2VtYmxpZXMgZnJvbSBsb25nLXJlYWQgU01SVCBzZXF1ZW5jaW5nIGRhdGEsOyBjaXRhdGlvbl9hdXRob3I9Q2hpbiAgQy5TLjsgY2l0YXRpb25fYXV0aG9yPUFsZXhhbmRlciAgRC5ILjsgY2l0YXRpb25fYXV0aG9yPU1hcmtzICBQLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9NTYzLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUh5YnJpZCBlcnJvciBjb3JyZWN0aW9uIGFuZCBkZSBub3ZvIGFzc2VtYmx5IG9mIHNpbmdsZS1tb2xlY3VsZSBzZXF1ZW5jaW5nIHJlYWRzOyBjaXRhdGlvbl9hdXRob3I9S29yZW4gIFMuOyBjaXRhdGlvbl9hdXRob3I9U2NoYXR6ICBNLkMuOyBjaXRhdGlvbl9hdXRob3I9V2FsZW56ICBCLlAuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz02OTMtNzAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QYWNpZmljIGJpb3NjaWVuY2VzIHNlcXVlbmNpbmcgdGVjaG5vbG9neSBmb3IgZ2Vub3R5cGluZyBhbmQgdmFyaWF0aW9uIGRpc2NvdmVyeSBpbiBodW1hbiBkYXRhLDsgY2l0YXRpb25fYXV0aG9yPUNhcm5laXJvICBNLk8uOyBjaXRhdGlvbl9hdXRob3I9UnVzcyAgQy47IGNpdGF0aW9uX2F1dGhvcj1Sb3NzICBNLkcuOyBjaXRhdGlvbl9hdXRob3I9R2FicmllbCAgUy5CLjsgY2l0YXRpb25fYXV0aG9yPU51c2JhdW0gIEMuOyBjaXRhdGlvbl9hdXRob3I9RGVQcmlzdG8gIE0uQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTM3NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUh1bWFuIG1hbGFyaWEgcGFyYXNpdGVzIGluIGNvbnRpbnVvdXMgY3VsdHVyZSw7IGNpdGF0aW9uX2F1dGhvcj1UcmFnZXIgIFcuOyBjaXRhdGlvbl9hdXRob3I9SmVuc2VuICBKLkIuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0xOTc2OyAgY2l0YXRpb25fdm9sdW1lPTE5MzsgIGNpdGF0aW9uX3BhZ2VzPTY3My01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBhbHRlcm5hdGl2ZSBtZXRob2QgZm9yIFBsYXNtb2RpdW0gY3VsdHVyZSBzeW5jaHJvbml6YXRpb24sOyBjaXRhdGlvbl9hdXRob3I9TGVsacOodnJlICBKLjsgY2l0YXRpb25fYXV0aG9yPUJlcnJ5ICBBLjsgY2l0YXRpb25fYXV0aG9yPUJlbm9pdC1WaWNhbCAgRi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUV4cC4gUGFyYXNpdG9sOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTEwOTsgIGNpdGF0aW9uX3BhZ2VzPTE5NS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeW5jaHJvbml6YXRpb24gb2YgUGxhc21vZGl1bSBmYWxjaXBhcnVtIGVyeXRocm9jeXRpYyBzdGFnZXMgaW4gY3VsdHVyZSw7IGNpdGF0aW9uX2F1dGhvcj1MYW1icm9zICBDLjsgY2l0YXRpb25fYXV0aG9yPVZhbmRlcmJlcmcgIEouUC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIFBhcmFzaXRvbDsgIGNpdGF0aW9uX3llYXI9MTk3OTsgIGNpdGF0aW9uX3ZvbHVtZT02NTsgIGNpdGF0aW9uX3BhZ2VzPTQxOC0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iTW9sbCwgSy4sIEthbmVrbywgQS4sIFNjaGVyZiwgQS4sIFdhaGxncmVuLCBNLiAyMDEzIE1ldGhvZHMgaW4gTWFsYXJpYSBSZXNlYXJjaCwgNnRoIGVkaXRpb24uICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgd2hvbGUtZ2Vub21lIGFzc2VtYmx5IG9mIERyb3NvcGhpbGEsOyBjaXRhdGlvbl9hdXRob3I9TXllcnMgIEUuVy47IGNpdGF0aW9uX2F1dGhvcj1TdXR0b24gIEcuRy47IGNpdGF0aW9uX2F1dGhvcj1EZWxjaGVyICBBLkwuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTI4NzsgIGNpdGF0aW9uX3BhZ2VzPTIxOTYtMjA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXBwaW5nIHNpbmdsZSBtb2xlY3VsZSBzZXF1ZW5jaW5nIHJlYWRzIHVzaW5nIGJhc2ljIGxvY2FsIGFsaWdubWVudCB3aXRoIHN1Y2Nlc3NpdmUgcmVmaW5lbWVudCAoQkxBU1IpOiBhcHBsaWNhdGlvbiBhbmQgdGhlb3J5LDsgY2l0YXRpb25fYXV0aG9yPUNoYWlzc29uICBNLkouOyBjaXRhdGlvbl9hdXRob3I9VGVzbGVyICBHLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MjM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VwYXJkOiBhIHJhcGlkIGFuZCBzZW5zaXRpdmUgdG9vbCBmb3IgY3JlYXRpbmcgZG90cGxvdHMgb24gZ2Vub21lIHNjYWxlLDsgY2l0YXRpb25fYXV0aG9yPUtydW1zaWVrICBKLjsgY2l0YXRpb25fYXV0aG9yPUFybm9sZCAgUi47IGNpdGF0aW9uX2F1dGhvcj1SYXR0ZWkgIFQuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTEwMjYtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNzZW1ibHl0aWNzOiBhIHdlYiBhbmFseXRpY3MgdG9vbCBmb3IgdGhlIGRldGVjdGlvbiBvZiBhc3NlbWJseS1iYXNlZCB2YXJpYW50cyw7IGNpdGF0aW9uX2F1dGhvcj1OYXR0ZXN0YWQgIE0uOyBjaXRhdGlvbl9hdXRob3I9U2NoYXR6ICBNLkMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1iaW9SeGl2OyAgY2l0YXRpb25feWVhcj0yMDE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1WZXJzYXRpbGUgYW5kIG9wZW4gc29mdHdhcmUgZm9yIGNvbXBhcmluZyBsYXJnZSBnZW5vbWVzLDsgY2l0YXRpb25fYXV0aG9yPUt1cnR6ICBTLjsgY2l0YXRpb25fYXV0aG9yPVBoaWxsaXBweSAgQS47IGNpdGF0aW9uX2F1dGhvcj1EZWxjaGVyICBBLkwuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9UjEyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QkVEVG9vbHM6IGEgZmxleGlibGUgc3VpdGUgb2YgdXRpbGl0aWVzIGZvciBjb21wYXJpbmcgZ2Vub21pYyBmZWF0dXJlcyw7IGNpdGF0aW9uX2F1dGhvcj1RdWlubGFuICBBLlIuOyBjaXRhdGlvbl9hdXRob3I9SGFsbCAgSS5NLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz04NDEtMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgdGhlIGhlbWUgY29tcG91bmQgY29wdXJpZmllZCB3aXRoIGRlb3h5cmlib251Y2xlaWMgYWNpZCAoRE5BKSBmcm9tIGJsb29kc3RhaW5zLCBhIG1ham9yIGluaGliaXRvciBvZiBwb2x5bWVyYXNlIGNoYWluIHJlYWN0aW9uIChQQ1IpIGFtcGxpZmljYXRpb24sOyBjaXRhdGlvbl9hdXRob3I9QWthbmUgIEEuOyBjaXRhdGlvbl9hdXRob3I9TWF0c3ViYXJhICBLLjsgY2l0YXRpb25fYXV0aG9yPU5ha2FtdXJhICBILjsgY2l0YXRpb25fYXV0aG9yPVRha2FoYXNoaSAgUy47IGNpdGF0aW9uX2F1dGhvcj1LaW11cmEgIEsuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBGb3JlbnNpYyBTY2k7ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz0zNjItNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11dGFudHMgb2YgVGFxIEROQSBwb2x5bWVyYXNlIHJlc2lzdGFudCB0byBQQ1IgaW5oaWJpdG9ycyBhbGxvdyBETkEgYW1wbGlmaWNhdGlvbiBmcm9tIHdob2xlIGJsb29kIGFuZCBjcnVkZSBzb2lsIHNhbXBsZXMsOyBjaXRhdGlvbl9hdXRob3I9S2VybWVrY2hpZXYgIE0uQi47IGNpdGF0aW9uX2F1dGhvcj1LaXJpbG92YSAgTC5JLjsgY2l0YXRpb25fYXV0aG9yPVZhaWwgIEUuRS47IGNpdGF0aW9uX2F1dGhvcj1CYXJuZXMgIFcuTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9ZTQwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlyZWN0IEROQSBhbXBsaWZpY2F0aW9uIGZyb20gY3J1ZGUgY2xpbmljYWwgc2FtcGxlcyB1c2luZyBhIFBDUiBlbmhhbmNlciBjb2NrdGFpbCBhbmQgbm92ZWwgbXV0YW50cyBvZiBUYXEsOyBjaXRhdGlvbl9hdXRob3I9WmhhbmcgIFouOyBjaXRhdGlvbl9hdXRob3I9S2VybWVrY2hpZXYgIE0uQi47IGNpdGF0aW9uX2F1dGhvcj1CYXJuZXMgIFcuTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIE1vbC4gRGlhZ247ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xNTItNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2gtdGhyb3VnaHB1dCA0NTQgcmVzZXF1ZW5jaW5nIGZvciBhbGxlbGUgZGlzY292ZXJ5IGFuZCByZWNvbWJpbmF0aW9uIG1hcHBpbmcgaW4gUGxhc21vZGl1bSBmYWxjaXBhcnVtICw7IGNpdGF0aW9uX2F1dGhvcj1TYW1hcmFrb29uICBVLjsgY2l0YXRpb25fYXV0aG9yPVJlZ2llciAgQS47IGNpdGF0aW9uX2F1dGhvcj1UYW4gIEEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xMTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGdlbm9tZS13aWRlIG1hcCBvZiBkaXZlcnNpdHkgaW4gUGxhc21vZGl1bSBmYWxjaXBhcnVtICw7IGNpdGF0aW9uX2F1dGhvcj1Wb2xrbWFuICBTLksuOyBjaXRhdGlvbl9hdXRob3I9U2FiZXRpICBQLkMuOyBjaXRhdGlvbl9hdXRob3I9RGVDYXByaW8gIEQuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9MTEzLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFtcGxpZmljYXRpb24tZnJlZSBJbGx1bWluYSBzZXF1ZW5jaW5nLWxpYnJhcnkgcHJlcGFyYXRpb24gZmFjaWxpdGF0ZXMgaW1wcm92ZWQgbWFwcGluZyBhbmQgYXNzZW1ibHkgb2YgKEcrQyktYmlhc2VkIGdlbm9tZXMsOyBjaXRhdGlvbl9hdXRob3I9S296YXJld2EgIEkuOyBjaXRhdGlvbl9hdXRob3I9TmluZyAgWi47IGNpdGF0aW9uX2F1dGhvcj1RdWFpbCAgTS5BLjsgY2l0YXRpb25fYXV0aG9yPVNhbmRlcnMgIE0uSi47IGNpdGF0aW9uX2F1dGhvcj1CZXJyaW1hbiAgTS47IGNpdGF0aW9uX2F1dGhvcj1UdXJuZXIgIEQuSi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MjkxLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBsYXNtb2RpdW0gZmFsY2lwYXJ1bSBjZW50cm9tZXJlcyBkaXNwbGF5IGEgdW5pcXVlIGVwaWdlbmV0aWMgbWFrZXVwIGFuZCBjbHVzdGVyIHByaW9yIHRvIGFuZCBkdXJpbmcgc2NoaXpvZ29ueSw7IGNpdGF0aW9uX2F1dGhvcj1Ib2Vpam1ha2VycyAgVy5BLjsgY2l0YXRpb25fYXV0aG9yPUZsdWVjayAgQy47IGNpdGF0aW9uX2F1dGhvcj1GcmFuY29panMgIEsuSi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGwuIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTEzOTEtNDAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdmlkZW5jZSBvbiB0aGUgY2hyb21vc29tYWwgbG9jYXRpb24gb2YgY2VudHJvbWVyaWMgRE5BIGluIFBsYXNtb2RpdW0gZmFsY2lwYXJ1bSBmcm9tIGV0b3Bvc2lkZS1tZWRpYXRlZCB0b3BvaXNvbWVyYXNlLUlJIGNsZWF2YWdlLDsgY2l0YXRpb25fYXV0aG9yPUtlbGx5ICBKLk0uOyBjaXRhdGlvbl9hdXRob3I9TWNSb2JlcnQgIEwuOyBjaXRhdGlvbl9hdXRob3I9QmFrZXIgIEQuQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVS4gUy4gQTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDM7ICBjaXRhdGlvbl9wYWdlcz02NzA2LTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNlbnRyYWwgcm9sZSBmb3IgUGxhc21vZGl1bSBmYWxjaXBhcnVtIHN1YnRlbG9tZXJpYyByZWdpb25zIGluIHNwYXRpYWwgcG9zaXRpb25pbmcgYW5kIHRlbG9tZXJlIGxlbmd0aCByZWd1bGF0aW9uLDsgY2l0YXRpb25fYXV0aG9yPUZpZ3VlaXJlZG8gIEwuTS47IGNpdGF0aW9uX2F1dGhvcj1GcmVpdGFzLUp1bmlvciAgTC5ILjsgY2l0YXRpb25fYXV0aG9yPUJvdHRpdXMgIEUuOyBjaXRhdGlvbl9hdXRob3I9T2xpdm8tTWFyaW4gIEouQy47IGNpdGF0aW9uX2F1dGhvcj1TY2hlcmYgIEEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FTUJPIEo7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz04MTUtMjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJJRklOcyBhcmUgYWRoZXNpbnMgaW1wbGljYXRlZCBpbiBzZXZlcmUgUGxhc21vZGl1bSBmYWxjaXBhcnVtIG1hbGFyaWEsOyBjaXRhdGlvbl9hdXRob3I9R29lbCAgUy47IGNpdGF0aW9uX2F1dGhvcj1QYWxta3Zpc3QgIE0uOyBjaXRhdGlvbl9hdXRob3I9TW9sbCAgSy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gTWVkOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MzE0LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJpZmluczogYSBzZWNvbmQgZmFtaWx5IG9mIGNsb25hbGx5IHZhcmlhbnQgcHJvdGVpbnMgZXhwcmVzc2VkIG9uIHRoZSBzdXJmYWNlIG9mIHJlZCBjZWxscyBpbmZlY3RlZCB3aXRoIFBsYXNtb2RpdW0gZmFsY2lwYXJ1bSAsOyBjaXRhdGlvbl9hdXRob3I9S3llcyAgUy5BLjsgY2l0YXRpb25fYXV0aG9yPVJvd2UgIEouQS47IGNpdGF0aW9uX2F1dGhvcj1LcmllayAgTi47IGNpdGF0aW9uX2F1dGhvcj1OZXdib2xkICBDLkkuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFUuIFMuIEE7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9OTY7ICBjaXRhdGlvbl9wYWdlcz05MzMzLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbml0b3JpbmcgcGFyYXNpdGUgZGl2ZXJzaXR5IGZvciBtYWxhcmlhIGVsaW1pbmF0aW9uIGluIHN1Yi1TYWhhcmFuIEFmcmljYSw7IGNpdGF0aW9uX2F1dGhvcj1HaGFuc2FoICBBLjsgY2l0YXRpb25fYXV0aG9yPUFtZW5nYS1FdGVnbyAgTC47IGNpdGF0aW9uX2F1dGhvcj1BbWFtYnVhLU5nd2EgIEEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTM0NTsgIGNpdGF0aW9uX3BhZ2VzPTEyOTctODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RWZmaWNpZW50IGRlcGxldGlvbiBvZiBob3N0IEROQSBjb250YW1pbmF0aW9uIGluIG1hbGFyaWEgY2xpbmljYWwgc2VxdWVuY2luZyw7IGNpdGF0aW9uX2F1dGhvcj1PeW9sYSAgUy5PLjsgY2l0YXRpb25fYXV0aG9yPUd1ICBZLjsgY2l0YXRpb25fYXV0aG9yPU1hbnNrZSAgTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIENsaW4uIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT01MTsgIGNpdGF0aW9uX3BhZ2VzPTc0NS01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gYWRqdXN0YWJsZSBnYXMtbWl4aW5nIGRldmljZSB0byBpbmNyZWFzZSBmZWFzaWJpbGl0eSBvZiBpbiB2aXRybyBjdWx0dXJlIG9mIFBsYXNtb2RpdW0gZmFsY2lwYXJ1bSBwYXJhc2l0ZXMgaW4gdGhlIGZpZWxkLDsgY2l0YXRpb25fYXV0aG9yPUJlaSAgQS5LLjsgY2l0YXRpb25fYXV0aG9yPVBhdGVsICBTLkQuOyBjaXRhdGlvbl9hdXRob3I9Vm9sa21hbiAgUy5LLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPWU5MDkyOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUltcHJvdmVkIGluIHZpdHJvIGN1bHR1cmUgb2YgUGxhc21vZGl1bSBmYWxjaXBhcnVtIHBlcm1pdHMgZXN0YWJsaXNobWVudCBvZiBjbGluaWNhbCBpc29sYXRlcyB3aXRoIHByZXNlcnZlZCBtdWx0aXBsaWNhdGlvbiwgaW52YXNpb24gYW5kIHJvc2V0dGluZyBwaGVub3R5cGVzLDsgY2l0YXRpb25fYXV0aG9yPVJpYmFja2UgIFUuOyBjaXRhdGlvbl9hdXRob3I9TW9sbCAgSy47IGNpdGF0aW9uX2F1dGhvcj1BbGJyZWNodCAgTC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lNjk3ODEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Hb2luZyBiZXlvbmQgZml2ZSBiYXNlcyBpbiBETkEgc2VxdWVuY2luZyw7IGNpdGF0aW9uX2F1dGhvcj1Lb3JsYWNoICBKLjsgY2l0YXRpb25fYXV0aG9yPVR1cm5lciAgUy5XLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gT3Bpbi4gU3RydWN0LiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MjUxLTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbnRlcmluZyB0aGUgZXJhIG9mIGJhY3RlcmlhbCBlcGlnZW5vbWljcyB3aXRoIHNpbmdsZSBtb2xlY3VsZSByZWFsIHRpbWUgRE5BIHNlcXVlbmNpbmcsOyBjaXRhdGlvbl9hdXRob3I9RGF2aXMgIEIuTS47IGNpdGF0aW9uX2F1dGhvcj1DaGFvICBNLkMuOyBjaXRhdGlvbl9hdXRob3I9V2FsZG9yICBNLksuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBPcGluLiBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0xOTItODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIG1ldGh5bGF0aW9uIG9uIE42LUFkZW5pbmUgaW4gQy4gZWxlZ2FucyAsOyBjaXRhdGlvbl9hdXRob3I9R3JlZXIgIEUuTC47IGNpdGF0aW9uX2F1dGhvcj1CbGFuY28gIE0uQS47IGNpdGF0aW9uX2F1dGhvcj1HdSAgTC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTYxOyAgY2l0YXRpb25fcGFnZXM9ODY4LTc4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgbWV0aHlsYXRpb24gb24gTig2KS1hZGVuaW5lIGluIG1hbW1hbGlhbiBlbWJyeW9uaWMgc3RlbSBjZWxsczsgY2l0YXRpb25fYXV0aG9yPVd1ICBULlAuOyBjaXRhdGlvbl9hdXRob3I9V2FuZyAgVC47IGNpdGF0aW9uX2F1dGhvcj1TZWV0aW4gIE0uRy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT01MzI7ICBjaXRhdGlvbl9wYWdlcz0zMjktMzMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBtYXBwaW5nIG9mIEROQSBtZXRoeWxhdGlvbiBpbiB0aGUgaHVtYW4gbWFsYXJpYSBwYXJhc2l0ZSBQbGFzbW9kaXVtIGZhbGNpcGFydW0gLDsgY2l0YXRpb25fYXV0aG9yPVBvbnRzICBOLjsgY2l0YXRpb25fYXV0aG9yPUZ1ICBMLjsgY2l0YXRpb25fYXV0aG9yPUhhcnJpcyAgRS5ZLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbCBIb3N0IE1pY3JvYmU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz02OTYtNzA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjMvNC8zMzkvMTc0NjMxNjUvZHN3MDIyLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICAgVGhlIGFwcGxpY2F0aW9uIG9mIG5leHQtZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nIHRvIGVzdGltYXRlIGdlbmV0aWMgZGl2ZXJzaXR5IG9mIFBsYXNtb2RpdW0gZmFsY2lwYXJ1bSAsIHRoZSBtb3N0IGxldGhhbCBtYWxhcmlhIHBhcmFzaXRlLCBoYXMgcHJvdmVkICIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MjQ2OTk3MCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGSK7T74TFECFo3Wx/k30Uag899uFQ==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f4%2f339%2f2469970" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f4%2f339%2f2469970" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=84049" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/23/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/4/0/m_cover.gif?Expires=2147483647&amp;Signature=XWSbFJdI5lGDaRny1g7BRzfZg~ebzc~OT20sUUxiNUi3-W~7uvtu5IDIgO3L5~QjmCQzUopLMGz-3gEHtIKUOFbRCHM2eH4h7l2Wu3nqAQC9YYHygJi-TLC9x3FpQRq3lTBcdWmpj185LJbB-1H-PKlUoEU9gxYyCNvjJUg~GUaHNRtVxbV0a7bDIfdyclZJ~w1BhGonEiYqh-IZSS0VekFWZ7mnLia~WbOEtL8AzSLsoIVak5-aADczuqIYGRkdfxcEh8zTdO-NlGhKxLoi5zlJy0Y~XuIxHZNxQxDFoVA3s~0J2fujGZsWX~ElpEU3I2Vma3M6MgFM80P2xPWpTw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/23/4">
                        <div class="volume trailing-comma">Volume 23</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87199685">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199685">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199687">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199687">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199692">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199692">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199706">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199706">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199730">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199730">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199735">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199735">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199737">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199737">Conflict of interest</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199739">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199739">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199741">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199741">Availability of Supporting Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199743">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199743">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/23/4/325/2469971">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/23/4/353/2469968">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main"> Complete telomere-to-telomere <em>de novo</em> assembly of the <em>Plasmodium falciparum</em> genome through long-read (&gt;11 kb), single molecule, real-time sequencing 
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shruthi Sridhar Vembar</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div> Unité Biologie des Interactions Hôte-Parasite, Département de Parasites et Insectes Vecteurs, Institut Pasteur, Paris 75015, France </div><div class="aff"><div class="label -label"><sup>2</sup></div> CNRS, ERL 9195, Paris 75015, France </div><div class="aff"><div class="label -label"><sup>3</sup></div> INSERM, Unit U1201, Paris 75015, France </div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw022-COR1" data-open="dsw022-COR1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
             *To whom correspondence should be addressed. Tel. +33145688622. Email: <a href="mailto:shruthi-sridhar.vembar@pasteur.fr" target="_blank">shruthi-sridhar.vembar@pasteur.fr</a> (S.S.V.); Tel. + 1-650-521-8240 . Email: <a href="mailto:mlaird@pacificbiosciences.com" target="_blank">mlaird@pacificbiosciences.com</a> (M.L.S.) 
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shruthi+Sridhar+Vembar">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Vembar S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Vembar S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shruthi Sridhar Vembar</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Matthew Seetin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>4</sup></div> Pacific Biosciences, Menlo Park, CA, USA </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Matthew+Seetin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Seetin M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Seetin M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Matthew Seetin</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Christine Lambert</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>4</sup></div> Pacific Biosciences, Menlo Park, CA, USA </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Christine+Lambert">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lambert C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lambert C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Christine Lambert</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Maria Nattestad</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>5</sup></div> Cold Spring Harbor Laboratory, Cold Spring Harbor, NY, USA </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Maria+Nattestad">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nattestad M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nattestad M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Maria Nattestad</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Michael C. Schatz</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>5</sup></div> Cold Spring Harbor Laboratory, Cold Spring Harbor, NY, USA </div><div class="aff"><div class="label -label"><sup>6</sup></div> Department of Computer Science, Johns Hopkins University, Baltimore, MD, USA </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Michael+C.+Schatz">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Schatz M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Schatz M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Michael C. Schatz</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Primo Baybayan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>4</sup></div> Pacific Biosciences, Menlo Park, CA, USA </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Primo+Baybayan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Baybayan P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Baybayan P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Primo Baybayan</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Artur Scherf</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div> Unité Biologie des Interactions Hôte-Parasite, Département de Parasites et Insectes Vecteurs, Institut Pasteur, Paris 75015, France </div><div class="aff"><div class="label -label"><sup>2</sup></div> CNRS, ERL 9195, Paris 75015, France </div><div class="aff"><div class="label -label"><sup>3</sup></div> INSERM, Unit U1201, Paris 75015, France </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Artur+Scherf">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Scherf A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Scherf A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Artur Scherf</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Melissa Laird Smith</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>4</sup></div> Pacific Biosciences, Menlo Park, CA, USA </div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw022-COR1" data-open="dsw022-COR1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
             *To whom correspondence should be addressed. Tel. +33145688622. Email: <a href="mailto:shruthi-sridhar.vembar@pasteur.fr" target="_blank">shruthi-sridhar.vembar@pasteur.fr</a> (S.S.V.); Tel. + 1-650-521-8240 . Email: <a href="mailto:mlaird@pacificbiosciences.com" target="_blank">mlaird@pacificbiosciences.com</a> (M.L.S.) 
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Melissa+Laird+Smith">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Smith M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Smith M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Melissa Laird Smith</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw022-FM1"><div class="fn"><div class="footnote-content">Edited by Dr Yuji Kohara</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 23, Issue 4, 1 August 2016, Pages 339–351, <a href='https://doi.org/10.1093/dnares/dsw022'>https://doi.org/10.1093/dnares/dsw022</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">26 June 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">24 November 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">10 May 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2469970" href="/dnaresearch/article-pdf/23/4/339/17463165/dsw022.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Shruthi Sridhar Vembar, Matthew Seetin, Christine Lambert, Maria Nattestad, Michael C. Schatz, Primo Baybayan, Artur Scherf, Melissa Laird Smith;   Complete telomere-to-telomere <em>de novo</em> assembly of the <em>Plasmodium falciparum</em> genome through long-read (&gt;11 kb), single molecule, real-time sequencing , <em>DNA Research</em>, Volume 23, Issue 4, 1 August 2016, Pages 339–351, <a href='https://doi.org/10.1093/dnares/dsw022'>https://doi.org/10.1093/dnares/dsw022</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2469970?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469970?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469970?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469970?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469970?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469970?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469970?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=%20Complete%20telomere-to-telomere%20%26lt%3Bem%26gt%3Bde%20novo%26lt%3B%2Fem%26gt%3B%20assembly%20of%20the%20%26lt%3Bem%26gt%3BPlasmodium%20falciparum%26lt%3B%2Fem%26gt%3B%20genome%20through%20long-read%20(%26amp%3Bgt%3B11%E2%80%89kb)%2C%20single%20molecule%2C%20real-time%20sequencing%20&amp;publicationDate=2016-06-26&amp;volumeNum=23&amp;issueNum=4&amp;author=Vembar%2C%20Shruthi%20Sridhar%3B%20Seetin%2C%20Matthew&amp;startPage=339&amp;endPage=351&amp;contentId=10.1093%2Fdnares%2Fdsw022&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f4%2f339%2f2469970" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87199685 id="87199685" class="abstract-title" >Abstract</h2>
<section class="abstract"><p> The application of next-generation sequencing to estimate genetic diversity of <em>Plasmodium falciparum</em> , the most lethal malaria parasite, has proved challenging due to the skewed AT-richness [∼80.6% (A + T)] of its genome and the lack of technology to assemble highly polymorphic subtelomeric regions that contain clonally variant, multigene virulence families (Ex: <em>var</em> and <em>rifin</em> ). To address this, we performed amplification-free, single molecule, real-time sequencing of <em>P. falciparum</em> genomic DNA and generated reads of average length 12 kb, with 50% of the reads between 15.5 and 50 kb in length. Next, using the Hierarchical Genome Assembly Process, we assembled the <em>P. falciparum</em> genome <em>de novo</em> and successfully compiled all 14 nuclear chromosomes telomere-to-telomere. We also accurately resolved centromeres [∼90–99% (A + T)] and subtelomeric regions and identified large insertions and duplications that add extra <em>var</em> and <em>rifin</em> genes to the genome, along with smaller structural variants such as homopolymer tract expansions. Overall, we show that amplification-free, long-read sequencing combined with <em>de novo</em> assembly overcomes major challenges inherent to studying the <em>P. falciparum</em> genome. Indeed, this technology may not only identify the polymorphic and repetitive subtelomeric sequences of parasite populations from endemic areas but may also evaluate structural variation linked to virulence, drug resistance and disease transmission. </p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Plasmodium falciparum&quot;"><em>Plasmodium falciparum</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="AT-biased">AT-biased</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;long-read sequencing&quot;">long-read sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;de novo assembly &quot;"><em>de novo</em> assembly </a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;structural variation&quot;">structural variation</a></div>                            <h2 scrollto-destination=87199687 id="87199687" class="section-title" >1. Introduction</h2>
<p><em>Plasmodium falciparum</em> , the causative agent of the most lethal form of malaria, uses a complex developmental programme to propagate within the human host and mosquito vector. <a reveal-id="dsw022-B1" data-open="dsw022-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> Disease pathogenesis in humans correlates with asexual development when the parasite resides within mature erythrocytes and mitotically replicates its haploid genome. In addition to meiotic recombination during diploid stages in mosquitoes, <a reveal-id="dsw022-B2" data-open="dsw022-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> it is during mitosis that the parasite expands its genetic diversity via homologous recombination, leading to the acquisition of new variants of virulence-associated surface adhesion molecules such as erythrocyte membrane protein 1 [ <em>P. falciparum</em> Erythrocyte Membrane Protein 1(PfEMP1); encoded by <em>var</em> genes]. <a reveal-id="dsw022-B3 dsw022-B4 dsw022-B5" data-open="dsw022-B3 dsw022-B4 dsw022-B5" class="link link-ref link-reveal xref-bibr"><sup>3–5</sup></a> Importantly, this is the stage at which the parasite evolves drug resistance. <a reveal-id="dsw022-B6" data-open="dsw022-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsw022-B7" data-open="dsw022-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> In fact, it has been estimated that for each cycle of intraerythrocytic replication, laboratory-adapted parasites can tolerate a single nucleotide polymorphism (SNP) mutation rate of approximately 0.5–1 × 10 <sup>−9</sup> per base, <a reveal-id="dsw022-B3" data-open="dsw022-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsw022-B4" data-open="dsw022-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> and that nearly 0.2% of daughter parasites carry a new chimeric PfEMP1 molecule. <a reveal-id="dsw022-B4" data-open="dsw022-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> Furthermore, the heterogeneity within the human host can be amplified by multiple mosquito bites, which may harbour genetically diverse parasite populations. There is, therefore, great interest in evaluating the genetic complexity of the infectious pool of parasites in malaria-endemic regions, with the specific aims of improving surveillance and intervention strategies. </p><p> The ∼23 Mb <em>P. falciparum</em> genome is organized into 14 chromosomes that range in size from 0.65 to 3.4 Mb. The draft sequence of the <em>P. falciparum</em> genome, which was first reported in 2002 using shotgun-sequencing methods for the laboratory-adapted strain 3D7, <a reveal-id="dsw022-B8" data-open="dsw022-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> revealed that the genome has an overall (A + T) composition of 80.6%, making it one of the most AT-rich genomes identified to date. The complexity of the genome is further underscored by the presence of extended tracts of As, Ts and TAs, in introns, intergenic and centromeric regions [up to 99% (A + T) content]; subtelomeric, hypervariable multigene virulence families, including the ∼60-member <em>var</em> gene family <a reveal-id="dsw022-B9" data-open="dsw022-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> ; and large segments of repetitive sequences, especially in subtelomeric regions. Given these unique features, not only has accurate sequencing of <em>P. falciparum</em> DNA presented a technical challenge for most next-generation sequencing (NGS) technologies, <a reveal-id="dsw022-B10 dsw022-B11 dsw022-B12 dsw022-B13 dsw022-B14" data-open="dsw022-B10 dsw022-B11 dsw022-B12 dsw022-B13 dsw022-B14" class="link link-ref link-reveal xref-bibr"><sup>10–14</sup></a> it has been suggested that the use of the current 3D7 genome sequence as a reference for clinical isolates results in incomplete estimates of genetic diversity. <a reveal-id="dsw022-B15" data-open="dsw022-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsw022-B16" data-open="dsw022-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a></p><p> To date, researchers have primarily used PCR-based whole genome amplification (WGA) methods to prepare short read sequencing libraries of <em>P. falciparum</em> laboratory-adapted and clinical strains, <a reveal-id="dsw022-B3" data-open="dsw022-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsw022-B4" data-open="dsw022-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsw022-B6" data-open="dsw022-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsw022-B17 dsw022-B18 dsw022-B19 dsw022-B20 dsw022-B21" data-open="dsw022-B17 dsw022-B18 dsw022-B19 dsw022-B20 dsw022-B21" class="link link-ref link-reveal xref-bibr"><sup>17–21</sup></a> with Nair et al. applying this to single cell sequencing. <a reveal-id="dsw022-B22" data-open="dsw022-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> More recently, Oyola et al. used φ29 DNA polymerase-based multiple displacement alignment (MDA), in the presence of the detergent tetramethylammonium chloride, to analyse the <em>P</em> . <em>falciparum</em> genome and showed that very low quantities of genomic DNA (∼10 pg) were sufficient to generate multiplexed Illumina libraries. <a reveal-id="dsw022-B11" data-open="dsw022-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> However, the introduction of errors and bias during PCR-based WGA, <a reveal-id="dsw022-B23" data-open="dsw022-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> and the subsequent alignment-based mapping of short reads to the reference 3D7 genome <a reveal-id="dsw022-B8" data-open="dsw022-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> ( <a class="link link-uri" href="http://genedb.org" target="_blank">http://genedb.org</a> ; 23.3 Mb assembly) may have led to an overestimation of SNPs in the sequencing data. Indeed, Oyola et al. observed that MDA introduces several per cent (2–6%) of <em>de novo</em> SNP calls as compared to a non-amplified library. <a reveal-id="dsw022-B11" data-open="dsw022-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> Furthermore, none of these studies analysed larger structural variants, except for Bopp et al. <a reveal-id="dsw022-B3" data-open="dsw022-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> and Claessens et al. <a reveal-id="dsw022-B4" data-open="dsw022-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> Therefore, we have a fragmented view of <em>P. falciparum</em> genome plasticity, in which SNPs are evaluated at high frequency but polymorphisms such as insertions and deletions, copy number variants, chromosomal rearrangements and structural variants in hypervariable and highly repetitive regions, are often underestimated or largely ignored. <a reveal-id="dsw022-B16" data-open="dsw022-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a></p><p> One solution that may overcome all of these caveats is the utilization of amplification-free long-read NGS technologies to sequence the <em>P. falciparum</em> genome. Single molecule real-time (SMRT) sequencing, which was the first such technology described, <a reveal-id="dsw022-B24" data-open="dsw022-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> generates long-reads with little to no sequence context bias, <a reveal-id="dsw022-B13" data-open="dsw022-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsw022-B14" data-open="dsw022-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsw022-B25" data-open="dsw022-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> with the most recent version of the DNA polymerase (P6) combined with C4 sequencing chemistry producing reads of average length 10–15 kb. <a reveal-id="dsw022-B25" data-open="dsw022-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> Numerous studies have shown that by oversampling a genome, structural variants can be detected with confidence, <a reveal-id="dsw022-B26" data-open="dsw022-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a><sup>,</sup><a reveal-id="dsw022-B27" data-open="dsw022-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> and <em>de novo</em> assembly can be performed with high accuracy. <a reveal-id="dsw022-B28 dsw022-B29 dsw022-B30" data-open="dsw022-B28 dsw022-B29 dsw022-B30" class="link link-ref link-reveal xref-bibr"><sup>28–30</sup></a> Attempts to analyse <em>P. falciparum</em> genomic DNA with early SMRT sequencing chemistry (P1-C1) generated ∼700–1,500 base long-reads, which did not allow for complete <em>de novo</em> assembly or evaluation of structural variations. <a reveal-id="dsw022-B13" data-open="dsw022-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsw022-B14" data-open="dsw022-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsw022-B31" data-open="dsw022-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> Therefore, to develop a robust long-read sequencing and <em>de novo</em> assembly protocol to analyse the <em>P. falciparum</em> genome, we utilized the Pacific Biosciences RS II System with P6-C4 chemistry. Accordingly, we sequenced the genome of the strain 3D7 ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022_Supplementary_Data.zip?Expires=2147483647&Signature=NEv6aZ5j6R-RjghVh1O9KGbIrr5vHbMXEXLhW6GuhgTSY4cMvH39S0gtZ0dxo5iKqWTgwu5kdJrP01cEURAnENEi5jSpSaSpf9YezvPgu7SBrFZUw5zm7i~jatBRJAMwH8o1xJ2PccCD0u7nkhAmF7q3g6BgPUwbf7a-frfruUxZdEbBtauplmhJxldCZ~Iwkux~b5-zLTmszX2rdn3V9sEkO7FQFjgdvgJjNVtLGN9fTCeZ~0m2uhLOBcJbJQrDCHteRjzBmTYe3Gc7lIXrzoA-CCzl46~Rg9pq~UEYk4v1kvl-XKpU~1JUfbBoDERiqc2DazIKU4BZKJohwjlzfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span> ), and generated sequencing reads that had an average read length of 11–13 kb (maximum 45–50 kb), comprising over 5.26 Gb of data. The resulting sequences were assembled <em>de novo</em> into a highly accurate <em>P. falciparum</em> genome using the Hierarchical Genome Assembly Process (HGAP), <a reveal-id="dsw022-B29" data-open="dsw022-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> with all 14 chromosomes resolved into single contigs. Even extremely AT-rich regions, including the centromeres, were resolved with uniform coverage and for the first time, subtelomeric regions of all chromosomes were successfully assembled in a single run. We present an initial analysis of the <em>de novo-</em> assembled <em>P. falciparum</em> genome and discuss the advances that can now be made with regards to estimating <em>P. falciparum</em> genetic diversity using long-read sequencing technologies. </p>                            <h2 scrollto-destination=87199692 id="87199692" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=87199693 id="87199693" class="section-title" >2.1. Growth of <em>P. falciparum</em></h3>
<p> Blood stages of the <em>P. falciparum</em> laboratory strain 3D7 were grown according to Trager and Jensen <a reveal-id="dsw022-B32" data-open="dsw022-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> with a few changes. Briefly, a mixed stage culture of <em>P. falciparum</em> was grown in white blood cell (WBC)-free O+ human erythrocytes (prepared from whole blood by treatment with leucocyte-specific filters) at a haematocrit of 4% in Roswell Park Memorial Institute 1640 medium containing L-glutamine (Invitrogen) supplemented with 10% v/v Albumax II (Invitrogen) and 200 μm hypoxanthine (C.C.Pro). The cultures were grown in a gas environment of 5% CO <sub>2</sub> , 1% O <sub>2</sub> and 94% N <sub>2</sub> to a parasitaemia of 3–8% before harvesting for downstream analysis. For synchronization, knob-positive parasites were selected by gelatin flotation using Plasmion (Fresenius Kabi) <a reveal-id="dsw022-B33" data-open="dsw022-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> and after re-invasion, treated twice with 5% sorbitol (Sigma) <a reveal-id="dsw022-B34" data-open="dsw022-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> to obtain parasites that were synchronized within a window of approximately 6 h, as evaluated by Giemsa staining. </p>                            <h3 scrollto-destination=87199695 id="87199695" class="section-title" >2.2. Genomic DNA isolation</h3>
<p> Infected human erythrocytes at different parasitaemia were harvested and 1 ml or 5 ml aliquots were frozen at − 20 °C. Subsequently, genomic DNA was prepared using the DNeasy Blood and Tissue kit (Qiagen) or the Genomic Tip kit (Qiagen) according to manufacturer’s instructions. For the DNeasy kit, free parasites that were obtained from the infected human erythrocyte pellet by saponin lysis <a reveal-id="dsw022-B35" data-open="dsw022-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> were resuspended in phosphate-buffered saline (PBS) and treated as per manufacturer’s instructions. For the Genomic Tip kit, nuclei were prepared directly from the infected human erythrocyte pellet as per manufacturer’s instructions. </p>                            <h3 scrollto-destination=87199697 id="87199697" class="section-title" >2.3. DNA purification prior to library preparation</h3>
<p> To remove heme and its derivatives from <em>P. falciparum</em> genomic DNA, the DNA was purified using one of the three independent methods: (i) magnetic bead-based cleanup; (ii) electrophoretic DNA extraction using the Aurora platform (Boreal Genomics) or (iii) phenol-chloroform extraction. The starting amount of DNA used for each method is indicated in <a reveal-id="dsw022-F1" data-open="dsw022-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a> . For (i), AMPure PB magnetic beads (Pacific Biosciences) were mixed with <em>P. falciparum</em> genomic DNA at a 1:1 (vol:vol) ratio and incubated for 20 min at room temperature (RT) with gentle end-over-end rotation. Following this, the beads were washed twice with 1.5 ml 70% ethanol and allowed to dry briefly at RT before elution with 100 μl of Pacific Biosciences Elution Buffer (EB). For (ii), <em>P. falciparum</em> DNA was electrophoretically purified using the Aurora System (Boreal Genomics), following manufacturer’s instructions in the ‘Experienced User Protocol’. For (iii), <em>P. falciparum</em> genomic DNA was initially mixed with 500 μl buffer containing 1 M NaCl and 2 mM ethylenediaminetetraacetic acid (EDTA). Next, an equal volume of phenol:chloroform:isoamyl alcohol (24:23:1) was added to the DNA mixture, inverted to mix and spun in a microcentrifuge at 10,000 g for 10 min at RT. The upper aqueous phase was transferred to a new microcentrifuge tube, mixed with an equal volume of chloroform:isoamyl alcohol (24:1) and spun at 10,000 g for 10 min at RT. Then, to remove excess polysaccharide, 0.3 volumes of 99.99% ethanol was added to the upper aqueous phase, the mixture was inverted and spun at 10,000 g for 15 min at RT. Finally, DNA present in the upper aqueous phase was precipitated by adding 1.7 volumes of 99.99% ethanol and spinning at 10,000 g for 15 min at RT, followed by two 70% ethanol washes. The DNA pellet was allowed to air-dry at RT for up to 5 min and resuspended in 100 µl of Pacific Biosciences EB. </p>                            <h3 scrollto-destination=87199699 id="87199699" class="section-title" >2.4. SMRTbell library preparation and sequencing</h3>
<p> Three SMRTbell libraries were constructed for genomic DNA purified with each of the methods described above ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022_Supplementary_Data.zip?Expires=2147483647&Signature=NEv6aZ5j6R-RjghVh1O9KGbIrr5vHbMXEXLhW6GuhgTSY4cMvH39S0gtZ0dxo5iKqWTgwu5kdJrP01cEURAnENEi5jSpSaSpf9YezvPgu7SBrFZUw5zm7i~jatBRJAMwH8o1xJ2PccCD0u7nkhAmF7q3g6BgPUwbf7a-frfruUxZdEbBtauplmhJxldCZ~Iwkux~b5-zLTmszX2rdn3V9sEkO7FQFjgdvgJjNVtLGN9fTCeZ~0m2uhLOBcJbJQrDCHteRjzBmTYe3Gc7lIXrzoA-CCzl46~Rg9pq~UEYk4v1kvl-XKpU~1JUfbBoDERiqc2DazIKU4BZKJohwjlzfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span> ). Each library was constructed using ∼4 μg of purified DNA and the SMRTbell Template Prep Kit 1.0, according to the protocol described in ‘Procedure &amp; Checklist—20 kb Template Preparation Using BluePippin™ Size-Selection System’ (Pacific Biosciences). Briefly, <em>P. falciparum</em> DNA was sheared for 5 min at 3000g or 5,500 rpm using a g-TUBE (Covaris), concentrated with AMPure PB beads and subjected to DNA damage repair and ligation of SMRTbell adapters. Following ligation, extraneous DNA was digested with exonucleases and the SMRTbell library was cleaned and concentrated with AMPure PB beads. The libraries were then subjected to a 20 kb DNA size-selection step using the BluePippin System (SageScience) to remove shorter DNA inserts with a size cut-off of 15 kb. Library quality and quantity were assessed using the Agilent 12000 DNA Kit and 2100 Bioanalyzer System (Agilent Technologies), as well as the Qubit dsDNA Broad Range Assay kit and Qubit Fluorometer (Thermo Fisher). Sequencing primer and P6 polymerase were annealed and bound, respectively, to the SMRTbell libraries as recommended by the manufacturer (Pacific Biosciences). To identify the library concentration that would achieve optimal Poisson loading on the SMRT Cell (i.e. ∼40% of zero mode waveguides loaded with a single DNA molecule), loading titrations were performed for each library. Based on this analysis, polymerase-bound SMRTbell libraries were loaded at a concentration of 200 pM for libraries cleaned up using magnetic beads and electrophoretic extraction, and at 100 pM for the phenol-chloroform extracted library to achieve comparable sequencing efficiencies. SMRT sequencing was performed on the Pacific Biosciences RS II System using the C4 sequencing kit (Pacific Biosciences), with magnetic bead loading and 240 min movies. Three SMRT cells were run per library type, providing a total of nine SMRT cells worth of data to be used for downstream analysis. Prior to data analysis, raw reads with a predicted polymerase read quality less than 0.80 were filtered out. </p>                            <h3 scrollto-destination=87199701 id="87199701" class="section-title" >2.5. Data analysis</h3>
<p><em>De novo</em> assembly of the <em>P. falciparum</em> genome was carried out using the RS_HGAP_Assembly.3 protocol within Pacific Biosciences’ SMRT Analysis Portal 2.3.0.p2 as previously described <a reveal-id="dsw022-B29" data-open="dsw022-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022_Supplementary_Data.zip?Expires=2147483647&Signature=NEv6aZ5j6R-RjghVh1O9KGbIrr5vHbMXEXLhW6GuhgTSY4cMvH39S0gtZ0dxo5iKqWTgwu5kdJrP01cEURAnENEi5jSpSaSpf9YezvPgu7SBrFZUw5zm7i~jatBRJAMwH8o1xJ2PccCD0u7nkhAmF7q3g6BgPUwbf7a-frfruUxZdEbBtauplmhJxldCZ~Iwkux~b5-zLTmszX2rdn3V9sEkO7FQFjgdvgJjNVtLGN9fTCeZ~0m2uhLOBcJbJQrDCHteRjzBmTYe3Gc7lIXrzoA-CCzl46~Rg9pq~UEYk4v1kvl-XKpU~1JUfbBoDERiqc2DazIKU4BZKJohwjlzfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span> ). All parameters were set at their default values with the following exceptions: (i) the minimum subread length was set to 13,000 based on the size distribution of the reads ( <a reveal-id="dsw022-F2" data-open="dsw022-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a> ) and for computational expediency, so as to not exceed 100-fold coverage going into the analysis; (ii) the minimum seed read length was increased to 20,000 from a default of 6,000—this was relative to the value of the minimum subread length; (iii) the genome size was set to 24,000,000 and (iv) the target coverage parameter was increased to 30 from a default of 25 to enhance coverage in the preassembly process (described below), in turn improving the quality of the finished assembly. The first step in RS_HGAP_Assembly.3, i.e. preassembly, utilizes a directed acyclic graph-based consensus procedure to align shorter reads to the longest reads in the sequencing data, thus generating corrected, continuous preassembled reads. <a reveal-id="dsw022-B29" data-open="dsw022-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> Next, the preassembled reads are assembled into larger contigs with the AssembleUnitig algorithm (Pacific Biosciences; <a class="link link-uri" href="https://github.com/PacificBiosciences/SMRT-Analysis/wiki/SMRT-Pipe-Reference-Guide-v2.3.0" target="_blank">https://github.com/PacificBiosciences/SMRT-Analysis/wiki/SMRT-Pipe-Reference-Guide</a> ), which incorporates elements of Celera Assembler <a reveal-id="dsw022-B36" data-open="dsw022-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> for overlapping and layout. Finally, the filtered reads are mapped back to the contigs using Blasr, <a reveal-id="dsw022-B37" data-open="dsw022-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> error-corrected with Quiver <a reveal-id="dsw022-B29" data-open="dsw022-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> to generate a high-quality consensus (referred to here as the SMRT assembly), and then visualized using SMRT View 2.3.0 (Pacific Biosciences). </p><p> To assemble the ∼32 kb <em>P. falciparum</em> apicoplast genome, reads that did not align to contigs in the SMRT assembly (which comprised only the nuclear genome) were reanalysed in SMRT Analysis Portal 2.3.0.p2 resulting in an apicoplast contig. For the ∼6 kb <em>P. falciparum</em> mitochondrial genome, Blasr was used to align the raw sequencing data to the M76611 mitochondrial sequence ( <a class="link link-uri" href="http://plasmodb.org/plasmo/showRecord.do?name=SequenceRecordClasses.SequenceRecordClass&amp;project_id=PlasmoDB&amp;primary_key=M76611" target="_blank">http://plasmodb.org/plasmo/showRecord.do?name=SequenceRecordClasses.SequenceRecordClass&amp;project_id=PlasmoDB&amp;primary_key=M76611</a> ) and assembly performed with reads that presented partial or full homology to M76611 mitochondrial DNA. </p><p> Dot plots comparing the SMRT assembly to the <em>P. falciparum</em> reference genome, Pf3D7_v3.0 ( <a class="link link-uri" href="http://genedb.org" target="_blank">http://genedb.org</a> ), were rendered with Gepard <a reveal-id="dsw022-B38" data-open="dsw022-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> using default parameters and a word length of 30. To identify structural variations in the genome, we used our new algorithm Assemblytics. <a reveal-id="dsw022-B39" data-open="dsw022-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> Briefly, Assemblytics analyses the whole genome alignment computed by MUMmer <a reveal-id="dsw022-B40" data-open="dsw022-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> and applies a unique length filtering approach to robustly identify structural variations in six classes of variants—insertions, deletions, tandem expansions, tandem contractions, repeat expansions and repeat contractions. Based on the size of the <em>P. falciparum</em> genome, the ‘minimum size of variant’ was set to 2 bp and the ‘minimum unique sequence length to anchor an alignment’ was set to 10 kb, which also becomes the maximum variant size to avoid calling variants above the size of the uniquely mapped sequence. Next, the presence of poly-A, poly-T and poly-AT in the smaller insertions (less than 10 bp) was determined by the following rules: (i) insertion must only contain the repeat sequence (for instance ATA for poly-AT) and (ii) either the 10 bp on the left or the 10 bp on the right must contain at least 6 bp of the repeat (for example, ATATAT in the case of the poly-AT repeat, AAAAAA in the case of poly-A). Finally, to determine whether a variant overlapped with a genomic feature, we performed a left outer join intersect using BEDTools <a reveal-id="dsw022-B41" data-open="dsw022-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> against Pf3D7_v3.0, annotation release 13.0 from plasmodb.org. </p><p> Direct alignment of the raw sequencing reads to Pf3D7_v3.0 was carried out using Blasr <a reveal-id="dsw022-B37" data-open="dsw022-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> with default settings. The resulting coverage obtained was 166-fold. </p>                            <h2 scrollto-destination=87199706 id="87199706" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=87199707 id="87199707" class="section-title" >3.1. High molecular weight genomic DNA was prepared from different <em>P. falciparum</em> intraerythrocytic stages</h3>
<p> Critical to obtaining long-reads with Pacific Biosciences’ SMRT sequencing technology is the extraction of high quality, high molecular weight genomic DNA, with a recommended size distribution of 50–150 kb ( <a class="link link-uri" href="http://www.pacb.com/wp-content/uploads/2015/09/User-Bulletin-Guidelines-for-Preparing-20-kb-SMRTbell-Templates.pdf" target="_blank">http://www.pacb.com/wp-content/uploads/2015/09/User-Bulletin-Guidelines-for-Preparing-20-kb-SMRTbell-Templates.pdf</a> ). To achieve this, we prepared genomic DNA from the <em>P. falciparum</em> laboratory strain 3D7 cultured in human blood using two different Qiagen kits: the DNeasy Blood and Tissue kit, which is routinely used by malaria researchers, <a reveal-id="dsw022-B35" data-open="dsw022-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> and the Genomic Tip kit. As shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022_Supplementary_Data.zip?Expires=2147483647&Signature=NEv6aZ5j6R-RjghVh1O9KGbIrr5vHbMXEXLhW6GuhgTSY4cMvH39S0gtZ0dxo5iKqWTgwu5kdJrP01cEURAnENEi5jSpSaSpf9YezvPgu7SBrFZUw5zm7i~jatBRJAMwH8o1xJ2PccCD0u7nkhAmF7q3g6BgPUwbf7a-frfruUxZdEbBtauplmhJxldCZ~Iwkux~b5-zLTmszX2rdn3V9sEkO7FQFjgdvgJjNVtLGN9fTCeZ~0m2uhLOBcJbJQrDCHteRjzBmTYe3Gc7lIXrzoA-CCzl46~Rg9pq~UEYk4v1kvl-XKpU~1JUfbBoDERiqc2DazIKU4BZKJohwjlzfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2A</a></span> , the size distribution of genomic DNA prepared using the DNeasy kit was between 33.5 and 48.5 kb, in contrast to a size distribution of  &gt;50 kb for genomic DNA prepared with the Genomic Tip kit. We also determined the output of the Genomic Tip kit for 3D7 parasites synchronized at ring (6–18 h post-invasion; non-replicating), trophozoite (28–35 h post-invasion, when DNA replication is initiated) and schizont stages (38–45 h post-invasion, when DNA replication is complete) and found that 3.1 × 10 <sup>8</sup> ring stage parasites yielded ∼2.5 μg of genomic DNA, while &gt;15 μg of genomic DNA could be extracted from 0.8 × 10 <sup>8</sup> schizonts ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022_Supplementary_Data.zip?Expires=2147483647&Signature=NEv6aZ5j6R-RjghVh1O9KGbIrr5vHbMXEXLhW6GuhgTSY4cMvH39S0gtZ0dxo5iKqWTgwu5kdJrP01cEURAnENEi5jSpSaSpf9YezvPgu7SBrFZUw5zm7i~jatBRJAMwH8o1xJ2PccCD0u7nkhAmF7q3g6BgPUwbf7a-frfruUxZdEbBtauplmhJxldCZ~Iwkux~b5-zLTmszX2rdn3V9sEkO7FQFjgdvgJjNVtLGN9fTCeZ~0m2uhLOBcJbJQrDCHteRjzBmTYe3Gc7lIXrzoA-CCzl46~Rg9pq~UEYk4v1kvl-XKpU~1JUfbBoDERiqc2DazIKU4BZKJohwjlzfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2B</a></span> ). Extrapolating from these results, we conclude that &gt;10 μg of high molecular weight genomic DNA can be extracted from a <em>P. falciparum</em> schizont culture growing in 500 μl of blood at a parasitaemia of ∼2% with Qiagen’s Genomic Tip kit, which is sufficient for downstream library preparation, size-selection and sequencing. </p>                            <h3 scrollto-destination=87199709 id="87199709" class="section-title" >3.2. SMRT sequencing of <em>P. falciparum</em> genomic DNA yielded reads of average length ∼12 kb</h3>
<p> Genomic DNA prepared from blood stage <em>P. falciparum</em> parasites is routinely contaminated with heme and its derivatives. Because these contaminants adversely affect downstream analyses such as PCR and sequencing, <a reveal-id="dsw022-B42 dsw022-B43 dsw022-B44" data-open="dsw022-B42 dsw022-B43 dsw022-B44" class="link link-ref link-reveal xref-bibr"><sup>42–44</sup></a> we examined the efficiency of three different purification methods to clean up <em>P. falciparum</em> genomic DNA isolated from an asynchronous 3D7 culture, prior to SMRTbell library preparation ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022_Supplementary_Data.zip?Expires=2147483647&Signature=NEv6aZ5j6R-RjghVh1O9KGbIrr5vHbMXEXLhW6GuhgTSY4cMvH39S0gtZ0dxo5iKqWTgwu5kdJrP01cEURAnENEi5jSpSaSpf9YezvPgu7SBrFZUw5zm7i~jatBRJAMwH8o1xJ2PccCD0u7nkhAmF7q3g6BgPUwbf7a-frfruUxZdEbBtauplmhJxldCZ~Iwkux~b5-zLTmszX2rdn3V9sEkO7FQFjgdvgJjNVtLGN9fTCeZ~0m2uhLOBcJbJQrDCHteRjzBmTYe3Gc7lIXrzoA-CCzl46~Rg9pq~UEYk4v1kvl-XKpU~1JUfbBoDERiqc2DazIKU4BZKJohwjlzfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span> ). These included: (i) magnetic bead-based cleanup, using a 1:1 ratio of AMPure PB beads, (ii) electrophoretic DNA extraction using the Aurora System from Boreal Genomics and (iii) phenol-chloroform extraction. We found that all three purification methods efficiently removed heme and other contaminants and did not affect the size distribution of the genomic DNA ( <a reveal-id="dsw022-F1" data-open="dsw022-F1" class="link link-reveal link-table xref-fig">Fig. 1A;</a> ‘no shear’). Notably, the magnetic bead-based purification method demonstrated the highest per cent DNA recovery values (54% versus 33 and 36% for phenol-chloroform extraction and Aurora System clean up, respectively) ( <a reveal-id="dsw022-F1" data-open="dsw022-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a> ). Subsequently, we prepared sequencing libraries and observed that DNA purification method did not impact shearing ( <a reveal-id="dsw022-F1" data-open="dsw022-F1" class="link link-reveal link-table xref-fig">Fig. 1A;</a> ‘shear’), SMRTbell library yield ( <a reveal-id="dsw022-F1" data-open="dsw022-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a> ) and BluePippin 20 kb size-selection ( <a reveal-id="dsw022-F1" data-open="dsw022-F1" class="link link-reveal link-table xref-fig">Figs 1B and C</a> ), and efficiently generated size-selected SMRTbell libraries with average insert lengths of ∼21 kb ( <a reveal-id="dsw022-F1" data-open="dsw022-F1" class="link link-reveal link-table xref-fig">Figs 1B and C</a> ). Of note, the SMRTbell libraries prepared from phenol-chloroform extracted DNA showed the highest recovery (44%) after size-selection ( <a reveal-id="dsw022-F1" data-open="dsw022-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a> ). </p>                        <a id="87199711" scrollto-destination="87199711"></a>
<div id="dsw022-F1" data-id="dsw022-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/m_dsw022f1p.png?Expires=2147483647&Signature=Y~NdKGU7-J-t~rL6c4caPUQXuu2qcsErqMCkyB~-NbmYwARtJgCnAYwEB2xP3oby~fW3JU5lhS59XEsvHGT3AHRAs1aGg9bR44yMo9WXOgkb4p894oER67DVjWyvSqOginFRsl3FReqkiLbFO~beyZiNbMOzOLjcX~ehYdzHVReeDX-E-dhbWw9ZsRwmKOlfCnVm0BGO2G~BwqtmlybLqPVwyGJaA0thetecoRfN6yrpwmAFygfobWi-nxnF-yYx2wsaPp8nzYxR0uvKK8F2FvFaT9Hf23eS650OIawdMsfzeHcnYmbLd3rLhy5tXiOhN1JBscfjKqLPuShZXQz38Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Comparison of SMRTbell library preparation efficiency from P. falciparum genomic DNA purified using three different methods. (A) High molecular weight P. falciparum genomic DNA prepared from an asynchronous culture using the Genomic tip kit was purified by three different methods: (i) AMPure PB magnetic bead-based clean up (Lanes 1 &amp; 2), (ii) electrophoretic DNA extraction using the Aurora System (Lanes 3 &amp; 4) or (iii) phenol-chloroform extraction (Lanes 5 &amp; 6), and sheared as described in Materials and Methods. Quality and size distribution of the sheared DNA (140 ng) was assessed using field-inversion gel electrophoresis (Pippin Pulse System, Sage Science). Size markers included CHEF 8-48 kb DNA Size Standard (Bio-Rad) and 2.5 kb Molecular Ruler (Bio-Rad). (B) SMRTbell libraries prepared using the indicated amount of purified genomic DNA was subjected to size selection on the BluePippin System using a 15 kb cut-off. The DNA yield and % recovery of various steps, library preparation efficiency and size-selection distribution (based on Fig. 2C) of the three DNA purification methods were compared. (C) Size, quantity and quality of SMRTbell libraries before and after size-selection were assessed using the Agilent DNA 12000 kit on the Agilent 2100 Bioanalyzer System. " path-from-xml="dsw022f1p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199711/dsw022f1p.png" path-from-xml="dsw022f1p.png" target="_blank">View large</a><a section="87199711" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022f1p.png?Expires=2147483647&Signature=NfIgCwJ9ioQeMHo9F3x6AyvzESxHgHQPG5a0rdb3LZ10tJLhCLe3uS0mII~RqI1jfSJRTweTaf86uX-5gldS1c~EU-~M1FYWRBfJG-xcSYDQAnH6mxP-7ls5OP2amsx1K8WxduSQN6SLJSupfOzWJvBjFSFptHQZWSCxK~DZVw2Kmn7D~OxLnX86LTx6KUC692qSVXp5p--6a8hYvgfwegzgsCfSd3rbHmLn5z6OPQyVEzCywNn2TAtvHxt5BppMr~dixo7gZvjxt~FW~ef6wDFg-xs7hOGVZn2oHBQtsJtQmLr320ftaWaohMM89hEbtkmZgvVGbHcuMGmxNJA2bg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199711&ar=2469970&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw022f1p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Comparison of SMRTbell library preparation efficiency from <em>P. falciparum</em> genomic DNA purified using three different methods. (A) High molecular weight <em>P. falciparum</em> genomic DNA prepared from an asynchronous culture using the Genomic tip kit was purified by three different methods: (i) AMPure PB magnetic bead-based clean up (Lanes 1 &amp; 2), (ii) electrophoretic DNA extraction using the Aurora System (Lanes 3 &amp; 4) or (iii) phenol-chloroform extraction (Lanes 5 &amp; 6), and sheared as described in Materials and Methods. Quality and size distribution of the sheared DNA (140 ng) was assessed using field-inversion gel electrophoresis (Pippin Pulse System, Sage Science). Size markers included CHEF 8-48 kb DNA Size Standard (Bio-Rad) and 2.5 kb Molecular Ruler (Bio-Rad). (B) SMRTbell libraries prepared using the indicated amount of purified genomic DNA was subjected to size selection on the BluePippin System using a 15 kb cut-off. The DNA yield and % recovery of various steps, library preparation efficiency and size-selection distribution (based on Fig. 2C) of the three DNA purification methods were compared. (C) Size, quantity and quality of SMRTbell libraries before and after size-selection were assessed using the Agilent DNA 12000 kit on the Agilent 2100 Bioanalyzer System. </p></div></div><div content-id="dsw022-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/m_dsw022f1p.png?Expires=2147483647&Signature=Y~NdKGU7-J-t~rL6c4caPUQXuu2qcsErqMCkyB~-NbmYwARtJgCnAYwEB2xP3oby~fW3JU5lhS59XEsvHGT3AHRAs1aGg9bR44yMo9WXOgkb4p894oER67DVjWyvSqOginFRsl3FReqkiLbFO~beyZiNbMOzOLjcX~ehYdzHVReeDX-E-dhbWw9ZsRwmKOlfCnVm0BGO2G~BwqtmlybLqPVwyGJaA0thetecoRfN6yrpwmAFygfobWi-nxnF-yYx2wsaPp8nzYxR0uvKK8F2FvFaT9Hf23eS650OIawdMsfzeHcnYmbLd3rLhy5tXiOhN1JBscfjKqLPuShZXQz38Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Comparison of SMRTbell library preparation efficiency from P. falciparum genomic DNA purified using three different methods. (A) High molecular weight P. falciparum genomic DNA prepared from an asynchronous culture using the Genomic tip kit was purified by three different methods: (i) AMPure PB magnetic bead-based clean up (Lanes 1 &amp; 2), (ii) electrophoretic DNA extraction using the Aurora System (Lanes 3 &amp; 4) or (iii) phenol-chloroform extraction (Lanes 5 &amp; 6), and sheared as described in Materials and Methods. Quality and size distribution of the sheared DNA (140 ng) was assessed using field-inversion gel electrophoresis (Pippin Pulse System, Sage Science). Size markers included CHEF 8-48 kb DNA Size Standard (Bio-Rad) and 2.5 kb Molecular Ruler (Bio-Rad). (B) SMRTbell libraries prepared using the indicated amount of purified genomic DNA was subjected to size selection on the BluePippin System using a 15 kb cut-off. The DNA yield and % recovery of various steps, library preparation efficiency and size-selection distribution (based on Fig. 2C) of the three DNA purification methods were compared. (C) Size, quantity and quality of SMRTbell libraries before and after size-selection were assessed using the Agilent DNA 12000 kit on the Agilent 2100 Bioanalyzer System. " path-from-xml="dsw022f1p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199711/dsw022f1p.png" path-from-xml="dsw022f1p.png" target="_blank">View large</a><a section="87199711" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022f1p.png?Expires=2147483647&Signature=NfIgCwJ9ioQeMHo9F3x6AyvzESxHgHQPG5a0rdb3LZ10tJLhCLe3uS0mII~RqI1jfSJRTweTaf86uX-5gldS1c~EU-~M1FYWRBfJG-xcSYDQAnH6mxP-7ls5OP2amsx1K8WxduSQN6SLJSupfOzWJvBjFSFptHQZWSCxK~DZVw2Kmn7D~OxLnX86LTx6KUC692qSVXp5p--6a8hYvgfwegzgsCfSd3rbHmLn5z6OPQyVEzCywNn2TAtvHxt5BppMr~dixo7gZvjxt~FW~ef6wDFg-xs7hOGVZn2oHBQtsJtQmLr320ftaWaohMM89hEbtkmZgvVGbHcuMGmxNJA2bg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199711&ar=2469970&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw022f1p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Comparison of SMRTbell library preparation efficiency from <em>P. falciparum</em> genomic DNA purified using three different methods. (A) High molecular weight <em>P. falciparum</em> genomic DNA prepared from an asynchronous culture using the Genomic tip kit was purified by three different methods: (i) AMPure PB magnetic bead-based clean up (Lanes 1 &amp; 2), (ii) electrophoretic DNA extraction using the Aurora System (Lanes 3 &amp; 4) or (iii) phenol-chloroform extraction (Lanes 5 &amp; 6), and sheared as described in Materials and Methods. Quality and size distribution of the sheared DNA (140 ng) was assessed using field-inversion gel electrophoresis (Pippin Pulse System, Sage Science). Size markers included CHEF 8-48 kb DNA Size Standard (Bio-Rad) and 2.5 kb Molecular Ruler (Bio-Rad). (B) SMRTbell libraries prepared using the indicated amount of purified genomic DNA was subjected to size selection on the BluePippin System using a 15 kb cut-off. The DNA yield and % recovery of various steps, library preparation efficiency and size-selection distribution (based on Fig. 2C) of the three DNA purification methods were compared. (C) Size, quantity and quality of SMRTbell libraries before and after size-selection were assessed using the Agilent DNA 12000 kit on the Agilent 2100 Bioanalyzer System. </p></div></div><p> Thereafter, we used the Pacific Biosciences P6 polymerase, C4 sequencing chemistry and the RS II Sequencing System to analyse the size-selected <em>P. falciparum</em> SMRTbell libraries ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022_Supplementary_Data.zip?Expires=2147483647&Signature=NEv6aZ5j6R-RjghVh1O9KGbIrr5vHbMXEXLhW6GuhgTSY4cMvH39S0gtZ0dxo5iKqWTgwu5kdJrP01cEURAnENEi5jSpSaSpf9YezvPgu7SBrFZUw5zm7i~jatBRJAMwH8o1xJ2PccCD0u7nkhAmF7q3g6BgPUwbf7a-frfruUxZdEbBtauplmhJxldCZ~Iwkux~b5-zLTmszX2rdn3V9sEkO7FQFjgdvgJjNVtLGN9fTCeZ~0m2uhLOBcJbJQrDCHteRjzBmTYe3Gc7lIXrzoA-CCzl46~Rg9pq~UEYk4v1kvl-XKpU~1JUfbBoDERiqc2DazIKU4BZKJohwjlzfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span> ). For libraries derived from each purification method, we ran three SMRT cells for 4 h, resulting in 525,996 raw sequencing reads that totalled 5.26 Gb from nine pooled SMRT cells ( <a reveal-id="dsw022-F2" data-open="dsw022-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a> ). SMRTbell libraries derived from phenol-chloroform extracted DNA provided greater sequencing yields at lower loading requirements, although average read lengths were slightly shorter (not statistically significant) than libraries generated from other purification methods. On average, 58,444 reads were generated per SMRT cell, with a read length of 12.1 kb ( <a reveal-id="dsw022-F2" data-open="dsw022-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a> ), and a tail in the read length distribution reaching close to 50 kb ( <a reveal-id="dsw022-F2" data-open="dsw022-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a> ). Moreover, 50% of the sequenced bases originated from reads that were ≥15.5 kb long ( <a reveal-id="dsw022-F2" data-open="dsw022-F2" class="link link-reveal link-table xref-fig">Fig. 2A;</a> read N50), suggesting that the Pacific Biosciences P6 polymerase may be capable of sequencing through long stretches of highly AT-rich genomic content, confirming prior reports of little to no sequence-context bias of SMRT sequencing. <a reveal-id="dsw022-B13" data-open="dsw022-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsw022-B14" data-open="dsw022-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsw022-B25" data-open="dsw022-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a></p>                        <a id="87199713" scrollto-destination="87199713"></a>
<div id="dsw022-F2" data-id="dsw022-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/m_dsw022f2p.png?Expires=2147483647&Signature=3q1~8rFr2z9hWp6VQZYAuLjTu95hRV7g2wr9jiyey1HAq2CJ3XHd6Z5zWnrD1mslvPoWCTJ-VfPE2lDQ8EiB4GVuKfp81kh5S1cbs3OhULahKXSyCwNrwy3VszvnVHX~-~0BbzKgXxlntto9eJ~K4eBQLQBt1wBmrYAZkS8KhWcTm7ozMUSw0oQBEokK9dCDMSGPfIojbrV0xCBofIjwiY2BcI9mgs66mjvvJTa~tXW2l4dBcqL~gu-SfGJHQvsGWTACs3pb8GsO6pblypMHIOd3jVx9qHz-uVspcxunAG9cVeQLZisI3LHi-AxMQruK94tv2kCq7iieoPsioY0VOg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" SMRT sequencing of P. falciparum genomic DNA yields &gt;500,000 reads of average length 12.31 kb, with a read length N50 of 15.46 kb. (A) SMRTbell libraries were analysed on nine SMRT cells (three cells per DNA purification method) in a Pacific Biosciences RS II Sequencing System using 4 h-long movies. Sequencing metrics are shown for each SMRT cell. Sequencing data from nine SMRT cells were pooled and analysed for read length distribution (B) pre- and (C) post-filtering. The x -axis represents read length in bases, while the y -axis represents number of reads (gray columns) and megabases (Mb) greater than read length (black curve). " path-from-xml="dsw022f2p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199713/dsw022f2p.png" path-from-xml="dsw022f2p.png" target="_blank">View large</a><a section="87199713" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022f2p.png?Expires=2147483647&Signature=IrIyK41juiLIFFoILr3thQqtT7l9e8bZjQL2~uAi23NU~qGvO8ZPWeDmOe6MBBPSLLhQgEuEdJhaSPzqDkfx2jsFTLPhTTiqO8Ul-TBqUwatDHBfgCDOgWYoTw1R2154AVx~41z-LN-29Ynu6t8e4VUeU9i55CRb7T~iUrTgUL8JnzjPSrdpt9oBFn56g4l96dcWkohGWF3ENh~9OfC6ApvV8jNu4vOR5ZUgi-RGZZY1H9xgnGq2PKH~5Ph7ZIs68kBn4HIu7QOsUEQkF9FXfSBuq0ZVs~lmdAM1ghOlN1XYV1MPQMPjn~HSx7lYOuSUAhLyx9I1tQNlWqbd2PCoFg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199713&ar=2469970&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw022f2p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> SMRT sequencing of <em>P. falciparum</em> genomic DNA yields &gt;500,000 reads of average length 12.31 kb, with a read length N50 of 15.46 kb. (A) SMRTbell libraries were analysed on nine SMRT cells (three cells per DNA purification method) in a Pacific Biosciences RS II Sequencing System using 4 h-long movies. Sequencing metrics are shown for each SMRT cell. Sequencing data from nine SMRT cells were pooled and analysed for read length distribution (B) pre- and (C) post-filtering. The <em>x</em> -axis represents read length in bases, while the <em>y</em> -axis represents number of reads (gray columns) and megabases (Mb) greater than read length (black curve). </p></div></div><div content-id="dsw022-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/m_dsw022f2p.png?Expires=2147483647&Signature=3q1~8rFr2z9hWp6VQZYAuLjTu95hRV7g2wr9jiyey1HAq2CJ3XHd6Z5zWnrD1mslvPoWCTJ-VfPE2lDQ8EiB4GVuKfp81kh5S1cbs3OhULahKXSyCwNrwy3VszvnVHX~-~0BbzKgXxlntto9eJ~K4eBQLQBt1wBmrYAZkS8KhWcTm7ozMUSw0oQBEokK9dCDMSGPfIojbrV0xCBofIjwiY2BcI9mgs66mjvvJTa~tXW2l4dBcqL~gu-SfGJHQvsGWTACs3pb8GsO6pblypMHIOd3jVx9qHz-uVspcxunAG9cVeQLZisI3LHi-AxMQruK94tv2kCq7iieoPsioY0VOg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" SMRT sequencing of P. falciparum genomic DNA yields &gt;500,000 reads of average length 12.31 kb, with a read length N50 of 15.46 kb. (A) SMRTbell libraries were analysed on nine SMRT cells (three cells per DNA purification method) in a Pacific Biosciences RS II Sequencing System using 4 h-long movies. Sequencing metrics are shown for each SMRT cell. Sequencing data from nine SMRT cells were pooled and analysed for read length distribution (B) pre- and (C) post-filtering. The x -axis represents read length in bases, while the y -axis represents number of reads (gray columns) and megabases (Mb) greater than read length (black curve). " path-from-xml="dsw022f2p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199713/dsw022f2p.png" path-from-xml="dsw022f2p.png" target="_blank">View large</a><a section="87199713" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022f2p.png?Expires=2147483647&Signature=IrIyK41juiLIFFoILr3thQqtT7l9e8bZjQL2~uAi23NU~qGvO8ZPWeDmOe6MBBPSLLhQgEuEdJhaSPzqDkfx2jsFTLPhTTiqO8Ul-TBqUwatDHBfgCDOgWYoTw1R2154AVx~41z-LN-29Ynu6t8e4VUeU9i55CRb7T~iUrTgUL8JnzjPSrdpt9oBFn56g4l96dcWkohGWF3ENh~9OfC6ApvV8jNu4vOR5ZUgi-RGZZY1H9xgnGq2PKH~5Ph7ZIs68kBn4HIu7QOsUEQkF9FXfSBuq0ZVs~lmdAM1ghOlN1XYV1MPQMPjn~HSx7lYOuSUAhLyx9I1tQNlWqbd2PCoFg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199713&ar=2469970&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw022f2p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> SMRT sequencing of <em>P. falciparum</em> genomic DNA yields &gt;500,000 reads of average length 12.31 kb, with a read length N50 of 15.46 kb. (A) SMRTbell libraries were analysed on nine SMRT cells (three cells per DNA purification method) in a Pacific Biosciences RS II Sequencing System using 4 h-long movies. Sequencing metrics are shown for each SMRT cell. Sequencing data from nine SMRT cells were pooled and analysed for read length distribution (B) pre- and (C) post-filtering. The <em>x</em> -axis represents read length in bases, while the <em>y</em> -axis represents number of reads (gray columns) and megabases (Mb) greater than read length (black curve). </p></div></div>                            <h3 scrollto-destination=87199714 id="87199714" class="section-title" >3.3. End-to-end <em>de novo</em> assembly of all 14 <em>P. falciparum</em> chromosomes</h3>
<p> We next performed <em>de novo</em> assembly of filtered reads (4.26 Gb data comprising 325,565 reads of N50 18.168 kb; <a reveal-id="dsw022-F2" data-open="dsw022-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a> ) using the HGAP protocol in SMRT Portal 2.3.0.p2, with changes made to the default settings as described in Materials and Methods. The resulting assembly (hereafter referred to as the SMRT assembly) had a total genome size of 23.6 Mb ( <a reveal-id="dsw022-F3" data-open="dsw022-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a> ) and produced a total of 21 contigs ( <a reveal-id="dsw022-F3" data-open="dsw022-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022_Supplementary_Data.zip?Expires=2147483647&Signature=NEv6aZ5j6R-RjghVh1O9KGbIrr5vHbMXEXLhW6GuhgTSY4cMvH39S0gtZ0dxo5iKqWTgwu5kdJrP01cEURAnENEi5jSpSaSpf9YezvPgu7SBrFZUw5zm7i~jatBRJAMwH8o1xJ2PccCD0u7nkhAmF7q3g6BgPUwbf7a-frfruUxZdEbBtauplmhJxldCZ~Iwkux~b5-zLTmszX2rdn3V9sEkO7FQFjgdvgJjNVtLGN9fTCeZ~0m2uhLOBcJbJQrDCHteRjzBmTYe3Gc7lIXrzoA-CCzl46~Rg9pq~UEYk4v1kvl-XKpU~1JUfbBoDERiqc2DazIKU4BZKJohwjlzfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3A–U</a></span> ; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022_Supplementary_Data.zip?Expires=2147483647&Signature=NEv6aZ5j6R-RjghVh1O9KGbIrr5vHbMXEXLhW6GuhgTSY4cMvH39S0gtZ0dxo5iKqWTgwu5kdJrP01cEURAnENEi5jSpSaSpf9YezvPgu7SBrFZUw5zm7i~jatBRJAMwH8o1xJ2PccCD0u7nkhAmF7q3g6BgPUwbf7a-frfruUxZdEbBtauplmhJxldCZ~Iwkux~b5-zLTmszX2rdn3V9sEkO7FQFjgdvgJjNVtLGN9fTCeZ~0m2uhLOBcJbJQrDCHteRjzBmTYe3Gc7lIXrzoA-CCzl46~Rg9pq~UEYk4v1kvl-XKpU~1JUfbBoDERiqc2DazIKU4BZKJohwjlzfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span> ). Thirteen of the 21 contigs were complete, individual <em>P. falciparum</em> chromosomes (chr. 2–14) ranging in length from 0.943 to 3.286 Mb ( <a reveal-id="dsw022-F3" data-open="dsw022-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a> ), while the remaining eight contigs represented a 620 kb portion of chromosome 1 (contig 20) and 20–60 kb repeat regions that aligned to chromosome ends. One of these repeats corresponded to the left arm of chromosome 1 (contig 64; discussed in detail below), but the remaining six contigs could not be correctly assigned to a single chromosome end ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022_Supplementary_Data.zip?Expires=2147483647&Signature=NEv6aZ5j6R-RjghVh1O9KGbIrr5vHbMXEXLhW6GuhgTSY4cMvH39S0gtZ0dxo5iKqWTgwu5kdJrP01cEURAnENEi5jSpSaSpf9YezvPgu7SBrFZUw5zm7i~jatBRJAMwH8o1xJ2PccCD0u7nkhAmF7q3g6BgPUwbf7a-frfruUxZdEbBtauplmhJxldCZ~Iwkux~b5-zLTmszX2rdn3V9sEkO7FQFjgdvgJjNVtLGN9fTCeZ~0m2uhLOBcJbJQrDCHteRjzBmTYe3Gc7lIXrzoA-CCzl46~Rg9pq~UEYk4v1kvl-XKpU~1JUfbBoDERiqc2DazIKU4BZKJohwjlzfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span> ; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022_Supplementary_Data.zip?Expires=2147483647&Signature=NEv6aZ5j6R-RjghVh1O9KGbIrr5vHbMXEXLhW6GuhgTSY4cMvH39S0gtZ0dxo5iKqWTgwu5kdJrP01cEURAnENEi5jSpSaSpf9YezvPgu7SBrFZUw5zm7i~jatBRJAMwH8o1xJ2PccCD0u7nkhAmF7q3g6BgPUwbf7a-frfruUxZdEbBtauplmhJxldCZ~Iwkux~b5-zLTmszX2rdn3V9sEkO7FQFjgdvgJjNVtLGN9fTCeZ~0m2uhLOBcJbJQrDCHteRjzBmTYe3Gc7lIXrzoA-CCzl46~Rg9pq~UEYk4v1kvl-XKpU~1JUfbBoDERiqc2DazIKU4BZKJohwjlzfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs. S3P–U</a></span> ). We consider these to be spurious contigs that may have arisen from high coverage of repetitive regions and do not have any discernible protein-coding or non-coding RNA (ncRNA) features. </p>                        <a id="87199716" scrollto-destination="87199716"></a>
<div id="dsw022-F3" data-id="dsw022-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/m_dsw022f3p.png?Expires=2147483647&Signature=BMNnqAih2S-jzK17vuXCL58EqetKVgsKPr0mCRbPp5u~4gIzF-OXtlBRT9vWUkP8dLBQdfYgOTD1TL6EeamUSLeOOedxDJ3JPeVN7lH1cVRl9TDHNl-qMqCQuz0sItCM1vtbDtGBFnRbnlTlijaVzOSHLJ0HiZVChkMHyn-bvczcHG2ZU86b70rOXzLbzd8zYBtabyrWN89qJWaBcFjurpghXBepp2yVBjGW-aCQzVLJzVvZltfO1wnvdhk0LDL5o8qxl83DfvrKEaWYMpn2zqkUueLC2S-K~zeeE1JrNgC5utJGKYFuIgtxBVko66vksgjU8CBLdTJkjCDg~Jrdiw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="De novo assembly using HGAP resolves all 14 P. falciparum chromosomes. (A) Summary of the SMRT assembly metrics for the P. falciparum genome. Contig N50 is a weighted median statistic such that 50% of the entire assembly is contained in contigs or scaffolds equal to or larger than this value, and QV50 indicates an estimated accuracy of 99.999% for the assembly. (B) All 14 P. falciparum chromosomes were assembled end-to-end using HGAP, without any gaps. The only exception was chr. 1, whose left arm was assembled as separate contig of 0.35 kb. The contig name in the SMRT assembly, its size, the corresponding chromosome and its coverage are indicated. The scale of coverage is between ×0 (blue arrowhead) and 150 (red arrowhead) except for contig 64, where the scale is from 0 to 100. The centromere position in each contig is indicated with the green line. (C) Contig 12 of the SMRT assembly, which aligns to P. falciparum chr. 7, was completely assembled using HGAP and showed a depth of coverage between ×70 and ×140. " path-from-xml="dsw022f3p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199716/dsw022f3p.png" path-from-xml="dsw022f3p.png" target="_blank">View large</a><a section="87199716" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022f3p.png?Expires=2147483647&Signature=Y47L9nFsGp3yOJKwZnTsBivqdVePk0KYcysBeve7~0axAsq4S07XelJlT-Hy4PxOYTKZ~9K2eKcT70q3pDPtEf04OYDaNmfaa3pG7SMAfvvw2Fu40T88jZp-xSMWQLjMqerWyqYGJpRGnSUMpjMoYAaBKUFA35NQvCd-lGqjJxDjmKZfxUgVq640Ig1Dt8DtTPFD7lp-e-HOgjucUV-MO5UGVdjIhCcbzfbYf6rypf1815w6Sji4OAmsPwXrMDICQlcHTJ-3YItqqrxbI4auJLWVFnbnabZgoknQaukwlSQNkJ2a4qc33qnYE0LU7Ofe7Z4~qmq6AFNTcws2qhMFZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199716&ar=2469970&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw022f3p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>De novo</em> assembly using HGAP resolves all 14 <em>P. falciparum</em> chromosomes. (A) Summary of the SMRT assembly metrics for the <em>P. falciparum</em> genome. Contig N50 is a weighted median statistic such that 50% of the entire assembly is contained in contigs or scaffolds equal to or larger than this value, and QV50 indicates an estimated accuracy of 99.999% for the assembly. (B) All 14 <em>P. falciparum</em> chromosomes were assembled end-to-end using HGAP, without any gaps. The only exception was chr. 1, whose left arm was assembled as separate contig of 0.35 kb. The contig name in the SMRT assembly, its size, the corresponding chromosome and its coverage are indicated. The scale of coverage is between ×0 (blue arrowhead) and 150 (red arrowhead) except for contig 64, where the scale is from 0 to 100. The centromere position in each contig is indicated with the green line. (C) Contig 12 of the SMRT assembly, which aligns to <em>P. falciparum</em> chr. 7, was completely assembled using HGAP and showed a depth of coverage between ×70 and ×140. </p></div></div><div content-id="dsw022-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/m_dsw022f3p.png?Expires=2147483647&Signature=BMNnqAih2S-jzK17vuXCL58EqetKVgsKPr0mCRbPp5u~4gIzF-OXtlBRT9vWUkP8dLBQdfYgOTD1TL6EeamUSLeOOedxDJ3JPeVN7lH1cVRl9TDHNl-qMqCQuz0sItCM1vtbDtGBFnRbnlTlijaVzOSHLJ0HiZVChkMHyn-bvczcHG2ZU86b70rOXzLbzd8zYBtabyrWN89qJWaBcFjurpghXBepp2yVBjGW-aCQzVLJzVvZltfO1wnvdhk0LDL5o8qxl83DfvrKEaWYMpn2zqkUueLC2S-K~zeeE1JrNgC5utJGKYFuIgtxBVko66vksgjU8CBLdTJkjCDg~Jrdiw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="De novo assembly using HGAP resolves all 14 P. falciparum chromosomes. (A) Summary of the SMRT assembly metrics for the P. falciparum genome. Contig N50 is a weighted median statistic such that 50% of the entire assembly is contained in contigs or scaffolds equal to or larger than this value, and QV50 indicates an estimated accuracy of 99.999% for the assembly. (B) All 14 P. falciparum chromosomes were assembled end-to-end using HGAP, without any gaps. The only exception was chr. 1, whose left arm was assembled as separate contig of 0.35 kb. The contig name in the SMRT assembly, its size, the corresponding chromosome and its coverage are indicated. The scale of coverage is between ×0 (blue arrowhead) and 150 (red arrowhead) except for contig 64, where the scale is from 0 to 100. The centromere position in each contig is indicated with the green line. (C) Contig 12 of the SMRT assembly, which aligns to P. falciparum chr. 7, was completely assembled using HGAP and showed a depth of coverage between ×70 and ×140. " path-from-xml="dsw022f3p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199716/dsw022f3p.png" path-from-xml="dsw022f3p.png" target="_blank">View large</a><a section="87199716" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022f3p.png?Expires=2147483647&Signature=Y47L9nFsGp3yOJKwZnTsBivqdVePk0KYcysBeve7~0axAsq4S07XelJlT-Hy4PxOYTKZ~9K2eKcT70q3pDPtEf04OYDaNmfaa3pG7SMAfvvw2Fu40T88jZp-xSMWQLjMqerWyqYGJpRGnSUMpjMoYAaBKUFA35NQvCd-lGqjJxDjmKZfxUgVq640Ig1Dt8DtTPFD7lp-e-HOgjucUV-MO5UGVdjIhCcbzfbYf6rypf1815w6Sji4OAmsPwXrMDICQlcHTJ-3YItqqrxbI4auJLWVFnbnabZgoknQaukwlSQNkJ2a4qc33qnYE0LU7Ofe7Z4~qmq6AFNTcws2qhMFZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199716&ar=2469970&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw022f3p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>De novo</em> assembly using HGAP resolves all 14 <em>P. falciparum</em> chromosomes. (A) Summary of the SMRT assembly metrics for the <em>P. falciparum</em> genome. Contig N50 is a weighted median statistic such that 50% of the entire assembly is contained in contigs or scaffolds equal to or larger than this value, and QV50 indicates an estimated accuracy of 99.999% for the assembly. (B) All 14 <em>P. falciparum</em> chromosomes were assembled end-to-end using HGAP, without any gaps. The only exception was chr. 1, whose left arm was assembled as separate contig of 0.35 kb. The contig name in the SMRT assembly, its size, the corresponding chromosome and its coverage are indicated. The scale of coverage is between ×0 (blue arrowhead) and 150 (red arrowhead) except for contig 64, where the scale is from 0 to 100. The centromere position in each contig is indicated with the green line. (C) Contig 12 of the SMRT assembly, which aligns to <em>P. falciparum</em> chr. 7, was completely assembled using HGAP and showed a depth of coverage between ×70 and ×140. </p></div></div><p> Each of the 13 fully resolved chromosomes featured distinct edges at either end, where coverage abruptly stopped and beyond which reads did not extend ( <a reveal-id="dsw022-F3" data-open="dsw022-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022_Supplementary_Data.zip?Expires=2147483647&Signature=NEv6aZ5j6R-RjghVh1O9KGbIrr5vHbMXEXLhW6GuhgTSY4cMvH39S0gtZ0dxo5iKqWTgwu5kdJrP01cEURAnENEi5jSpSaSpf9YezvPgu7SBrFZUw5zm7i~jatBRJAMwH8o1xJ2PccCD0u7nkhAmF7q3g6BgPUwbf7a-frfruUxZdEbBtauplmhJxldCZ~Iwkux~b5-zLTmszX2rdn3V9sEkO7FQFjgdvgJjNVtLGN9fTCeZ~0m2uhLOBcJbJQrDCHteRjzBmTYe3Gc7lIXrzoA-CCzl46~Rg9pq~UEYk4v1kvl-XKpU~1JUfbBoDERiqc2DazIKU4BZKJohwjlzfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3A–M</a></span> ). These ends are highly homologous to each other, both within the same chromosome and between different chromosomes, and contain several copies of the telomeric repetitive element CCCTNAA, indicating complete telomere-to-telomere assembly of chr. 2–14. A representative coverage plot for contig 12, i.e. chr. 7, is shown ( <a reveal-id="dsw022-F3" data-open="dsw022-F3" class="link link-reveal link-table xref-fig">Fig. 3C</a> ). In the case of chr. 1, we observed that the 620 kb contig 20 began with a gradual increase in coverage (left end) and ended with a distinct edge containing telomeric repeats (right end) ( <a reveal-id="dsw022-F3" data-open="dsw022-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022_Supplementary_Data.zip?Expires=2147483647&Signature=NEv6aZ5j6R-RjghVh1O9KGbIrr5vHbMXEXLhW6GuhgTSY4cMvH39S0gtZ0dxo5iKqWTgwu5kdJrP01cEURAnENEi5jSpSaSpf9YezvPgu7SBrFZUw5zm7i~jatBRJAMwH8o1xJ2PccCD0u7nkhAmF7q3g6BgPUwbf7a-frfruUxZdEbBtauplmhJxldCZ~Iwkux~b5-zLTmszX2rdn3V9sEkO7FQFjgdvgJjNVtLGN9fTCeZ~0m2uhLOBcJbJQrDCHteRjzBmTYe3Gc7lIXrzoA-CCzl46~Rg9pq~UEYk4v1kvl-XKpU~1JUfbBoDERiqc2DazIKU4BZKJohwjlzfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3O</a></span> ). We, therefore, searched for a contig that could be contiguous with contig 20 and identified the 37 kb contig 64 ( <a reveal-id="dsw022-F3" data-open="dsw022-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022_Supplementary_Data.zip?Expires=2147483647&Signature=NEv6aZ5j6R-RjghVh1O9KGbIrr5vHbMXEXLhW6GuhgTSY4cMvH39S0gtZ0dxo5iKqWTgwu5kdJrP01cEURAnENEi5jSpSaSpf9YezvPgu7SBrFZUw5zm7i~jatBRJAMwH8o1xJ2PccCD0u7nkhAmF7q3g6BgPUwbf7a-frfruUxZdEbBtauplmhJxldCZ~Iwkux~b5-zLTmszX2rdn3V9sEkO7FQFjgdvgJjNVtLGN9fTCeZ~0m2uhLOBcJbJQrDCHteRjzBmTYe3Gc7lIXrzoA-CCzl46~Rg9pq~UEYk4v1kvl-XKpU~1JUfbBoDERiqc2DazIKU4BZKJohwjlzfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3N</a></span> ), which had a distinct telomeric repeat-containing edge at its left end and a gradual drop off in coverage at its right end. The manual joining of these two contigs yielded full-length chr. 1. Finally, because the genomes of the mitochondria [∼6 kb; 68.4% (A + T) content] and apicoplast [∼32 kb; 85.8% (A + T) content] were absent from the SMRT assembly, we selected reads that did not align to the nuclear chromosomes from the raw sequencing data and reanalysed them with independent HGAP calculations (detailed in Materials and Methods). In doing so, we completely assembled the genomes of these key organelles ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022_Supplementary_Data.zip?Expires=2147483647&Signature=NEv6aZ5j6R-RjghVh1O9KGbIrr5vHbMXEXLhW6GuhgTSY4cMvH39S0gtZ0dxo5iKqWTgwu5kdJrP01cEURAnENEi5jSpSaSpf9YezvPgu7SBrFZUw5zm7i~jatBRJAMwH8o1xJ2PccCD0u7nkhAmF7q3g6BgPUwbf7a-frfruUxZdEbBtauplmhJxldCZ~Iwkux~b5-zLTmszX2rdn3V9sEkO7FQFjgdvgJjNVtLGN9fTCeZ~0m2uhLOBcJbJQrDCHteRjzBmTYe3Gc7lIXrzoA-CCzl46~Rg9pq~UEYk4v1kvl-XKpU~1JUfbBoDERiqc2DazIKU4BZKJohwjlzfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S3W and S3X</a></span> ). Of note, the mitochondrial genome was assembled as a ∼38 kb contig ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022_Supplementary_Data.zip?Expires=2147483647&Signature=NEv6aZ5j6R-RjghVh1O9KGbIrr5vHbMXEXLhW6GuhgTSY4cMvH39S0gtZ0dxo5iKqWTgwu5kdJrP01cEURAnENEi5jSpSaSpf9YezvPgu7SBrFZUw5zm7i~jatBRJAMwH8o1xJ2PccCD0u7nkhAmF7q3g6BgPUwbf7a-frfruUxZdEbBtauplmhJxldCZ~Iwkux~b5-zLTmszX2rdn3V9sEkO7FQFjgdvgJjNVtLGN9fTCeZ~0m2uhLOBcJbJQrDCHteRjzBmTYe3Gc7lIXrzoA-CCzl46~Rg9pq~UEYk4v1kvl-XKpU~1JUfbBoDERiqc2DazIKU4BZKJohwjlzfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3W</a></span> ), indicating that it exists as circular tandem duplications of itself: in this case, as six copies of the 6 kb genome. </p><p> We compared our SMRT assembly to other <em>de novo</em> assemblies of the <em>P. falciparum</em> genome that were generated using data from Roche 454 pyrosequencing, <a reveal-id="dsw022-B45" data-open="dsw022-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> Sanger shotgun sequencing <a reveal-id="dsw022-B46" data-open="dsw022-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> or Illumina-based sequencing by synthesis <a reveal-id="dsw022-B47" data-open="dsw022-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> of different <em>P. falciparum</em> strains ( <a reveal-id="dsw022-T1" data-open="dsw022-T1" class="link link-reveal link-table xref-fig">Table 1</a> ). We found that our assembly was the most complete: it comprised 23.6 Mb and contained the least number of contigs, with an average contig N50 of 1.71 Mb. Moreover, in comparison to previous studies that performed SMRT sequencing of <em>P. falciparum</em> genomic DNA, <a reveal-id="dsw022-B13" data-open="dsw022-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsw022-B14" data-open="dsw022-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsw022-B31" data-open="dsw022-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> we generated over 10-fold longer reads with an average genome coverage of × 94; indeed, we believe that our HGAP-derived SMRT assembly is the first of its kind for <em>P. falciparum</em> . </p>                        <a id="87199719" scrollto-destination="87199719"></a>
<div content-id="dsw022-T1" class="table-modal"><div class="table-wrap"><div id="dsw022-T1" class="table-wrap-title" data-id="dsw022-T1"><div class="label -label">Table 1.</div><div class="caption"><p> A comparison of <em>de novo P. falciparum</em> genome assemblies compiled using different NGS technologies </p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="2"> Sanger sequencing <a reveal-id="dsw022-TF1" data-open="dsw022-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><hr /> </th><th colspan="2"> Illumina sequencing <a reveal-id="dsw022-TF2" data-open="dsw022-TF2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a><hr /> </th><th colspan="2"> 454 pyrosequencing <a reveal-id="dsw022-TF3" data-open="dsw022-TF3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a><hr /> </th><th>SMRT sequencing (this study) </th></tr><tr><th>Parasite strain </th><th>Dd2 </th><th>HB3 </th><th>NP-3D7-S </th><th>NP-3D7-L </th><th>7C126 </th><th>SC05 </th><th>3D7 </th></tr></thead><tbody><tr><td>Read length (bases) </td><td>600–700 </td><td> </td><td>36 </td><td>76 </td><td colspan="2">3,000(paired-end) </td><td>12,130 </td></tr><tr><td>Number of contigs </td><td>4,511 </td><td>2,971 </td><td>26,920 </td><td>22,839 </td><td>9,452 </td><td>9,597 </td><td>21 </td></tr><tr><td>N50 contig size (kb) </td><td>11.6 </td><td>20.6 </td><td>1.5 </td><td>1.6 </td><td>3.3 </td><td>3.3 </td><td>1,710 </td></tr><tr><td>Largest contig (kb) </td><td>79.2 </td><td>111.9 </td><td>29.1 </td><td>24.0 </td><td>36.7 </td><td>34.4 </td><td>3,290 </td></tr><tr><td>Number of assembled bases (Mb) </td><td>19.5 </td><td>23.4 </td><td>19.0 </td><td>21.1 </td><td>20.8 </td><td>21.1 </td><td>23.6 </td></tr><tr><td>Average coverage </td><td>×7.8 </td><td>×7.1 </td><td>×43 </td><td>×64 </td><td>×33 </td><td>×36 </td><td>×94 </td></tr></tbody></table></div><div content-id="dsw022-T1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="2"> Sanger sequencing <a reveal-id="dsw022-TF1" data-open="dsw022-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><hr /> </th><th colspan="2"> Illumina sequencing <a reveal-id="dsw022-TF2" data-open="dsw022-TF2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a><hr /> </th><th colspan="2"> 454 pyrosequencing <a reveal-id="dsw022-TF3" data-open="dsw022-TF3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a><hr /> </th><th>SMRT sequencing (this study) </th></tr><tr><th>Parasite strain </th><th>Dd2 </th><th>HB3 </th><th>NP-3D7-S </th><th>NP-3D7-L </th><th>7C126 </th><th>SC05 </th><th>3D7 </th></tr></thead><tbody><tr><td>Read length (bases) </td><td>600–700 </td><td> </td><td>36 </td><td>76 </td><td colspan="2">3,000(paired-end) </td><td>12,130 </td></tr><tr><td>Number of contigs </td><td>4,511 </td><td>2,971 </td><td>26,920 </td><td>22,839 </td><td>9,452 </td><td>9,597 </td><td>21 </td></tr><tr><td>N50 contig size (kb) </td><td>11.6 </td><td>20.6 </td><td>1.5 </td><td>1.6 </td><td>3.3 </td><td>3.3 </td><td>1,710 </td></tr><tr><td>Largest contig (kb) </td><td>79.2 </td><td>111.9 </td><td>29.1 </td><td>24.0 </td><td>36.7 </td><td>34.4 </td><td>3,290 </td></tr><tr><td>Number of assembled bases (Mb) </td><td>19.5 </td><td>23.4 </td><td>19.0 </td><td>21.1 </td><td>20.8 </td><td>21.1 </td><td>23.6 </td></tr><tr><td>Average coverage </td><td>×7.8 </td><td>×7.1 </td><td>×43 </td><td>×64 </td><td>×33 </td><td>×36 </td><td>×94 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="dsw022-TF1"><div class="fn"><p><sup>a</sup> Volkman et al., 2006. </p></div></div><div content-id="dsw022-TF2"><div class="fn"><p><sup>b</sup> Kozarewa et al., 2009; NP: No PCR; S and L indicate short and long sequencing runs performed on the same library. </p></div></div><div content-id="dsw022-TF3"><div class="fn"><p><sup>c</sup> Samarakoon et al., 2011. </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87199719" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="dsw022-T1" class="table-wrap-title" data-id="dsw022-T1"><div class="label -label">Table 1.</div><div class="caption"><p> A comparison of <em>de novo P. falciparum</em> genome assemblies compiled using different NGS technologies </p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="2"> Sanger sequencing <a reveal-id="dsw022-TF1" data-open="dsw022-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><hr /> </th><th colspan="2"> Illumina sequencing <a reveal-id="dsw022-TF2" data-open="dsw022-TF2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a><hr /> </th><th colspan="2"> 454 pyrosequencing <a reveal-id="dsw022-TF3" data-open="dsw022-TF3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a><hr /> </th><th>SMRT sequencing (this study) </th></tr><tr><th>Parasite strain </th><th>Dd2 </th><th>HB3 </th><th>NP-3D7-S </th><th>NP-3D7-L </th><th>7C126 </th><th>SC05 </th><th>3D7 </th></tr></thead><tbody><tr><td>Read length (bases) </td><td>600–700 </td><td> </td><td>36 </td><td>76 </td><td colspan="2">3,000(paired-end) </td><td>12,130 </td></tr><tr><td>Number of contigs </td><td>4,511 </td><td>2,971 </td><td>26,920 </td><td>22,839 </td><td>9,452 </td><td>9,597 </td><td>21 </td></tr><tr><td>N50 contig size (kb) </td><td>11.6 </td><td>20.6 </td><td>1.5 </td><td>1.6 </td><td>3.3 </td><td>3.3 </td><td>1,710 </td></tr><tr><td>Largest contig (kb) </td><td>79.2 </td><td>111.9 </td><td>29.1 </td><td>24.0 </td><td>36.7 </td><td>34.4 </td><td>3,290 </td></tr><tr><td>Number of assembled bases (Mb) </td><td>19.5 </td><td>23.4 </td><td>19.0 </td><td>21.1 </td><td>20.8 </td><td>21.1 </td><td>23.6 </td></tr><tr><td>Average coverage </td><td>×7.8 </td><td>×7.1 </td><td>×43 </td><td>×64 </td><td>×33 </td><td>×36 </td><td>×94 </td></tr></tbody></table></div><div content-id="dsw022-T1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="2"> Sanger sequencing <a reveal-id="dsw022-TF1" data-open="dsw022-TF1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><hr /> </th><th colspan="2"> Illumina sequencing <a reveal-id="dsw022-TF2" data-open="dsw022-TF2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a><hr /> </th><th colspan="2"> 454 pyrosequencing <a reveal-id="dsw022-TF3" data-open="dsw022-TF3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a><hr /> </th><th>SMRT sequencing (this study) </th></tr><tr><th>Parasite strain </th><th>Dd2 </th><th>HB3 </th><th>NP-3D7-S </th><th>NP-3D7-L </th><th>7C126 </th><th>SC05 </th><th>3D7 </th></tr></thead><tbody><tr><td>Read length (bases) </td><td>600–700 </td><td> </td><td>36 </td><td>76 </td><td colspan="2">3,000(paired-end) </td><td>12,130 </td></tr><tr><td>Number of contigs </td><td>4,511 </td><td>2,971 </td><td>26,920 </td><td>22,839 </td><td>9,452 </td><td>9,597 </td><td>21 </td></tr><tr><td>N50 contig size (kb) </td><td>11.6 </td><td>20.6 </td><td>1.5 </td><td>1.6 </td><td>3.3 </td><td>3.3 </td><td>1,710 </td></tr><tr><td>Largest contig (kb) </td><td>79.2 </td><td>111.9 </td><td>29.1 </td><td>24.0 </td><td>36.7 </td><td>34.4 </td><td>3,290 </td></tr><tr><td>Number of assembled bases (Mb) </td><td>19.5 </td><td>23.4 </td><td>19.0 </td><td>21.1 </td><td>20.8 </td><td>21.1 </td><td>23.6 </td></tr><tr><td>Average coverage </td><td>×7.8 </td><td>×7.1 </td><td>×43 </td><td>×64 </td><td>×33 </td><td>×36 </td><td>×94 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="dsw022-TF1"><div class="fn"><p><sup>a</sup> Volkman et al., 2006. </p></div></div><div content-id="dsw022-TF2"><div class="fn"><p><sup>b</sup> Kozarewa et al., 2009; NP: No PCR; S and L indicate short and long sequencing runs performed on the same library. </p></div></div><div content-id="dsw022-TF3"><div class="fn"><p><sup>c</sup> Samarakoon et al., 2011. </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87199719" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=87199720 id="87199720" class="section-title" >3.4. Analysis of centromeres and subtelomeric regions</h3>
<p> Given the telomere-to-telomere nature of the SMRT assembly, we were specifically interested in the coverage of two challenging <em>P. falciparum</em> genomic regions: (i) centromeres, which have elevated AT-richness [between 90 and 99% (A + T) content] <a reveal-id="dsw022-B8" data-open="dsw022-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsw022-B48" data-open="dsw022-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a><sup>,</sup><a reveal-id="dsw022-B49" data-open="dsw022-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> and (ii) subtelomeric regions, which contain six varieties of telomere-associated repeats, multigene families such as <em>var, rifin</em> and <em>stevor</em> , <a reveal-id="dsw022-B8" data-open="dsw022-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> and where much of the chromosomal length variation occurs. <a reveal-id="dsw022-B50" data-open="dsw022-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> We observed that in the case of all centromeres, depth of coverage was ×80–140 ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022_Supplementary_Data.zip?Expires=2147483647&Signature=NEv6aZ5j6R-RjghVh1O9KGbIrr5vHbMXEXLhW6GuhgTSY4cMvH39S0gtZ0dxo5iKqWTgwu5kdJrP01cEURAnENEi5jSpSaSpf9YezvPgu7SBrFZUw5zm7i~jatBRJAMwH8o1xJ2PccCD0u7nkhAmF7q3g6BgPUwbf7a-frfruUxZdEbBtauplmhJxldCZ~Iwkux~b5-zLTmszX2rdn3V9sEkO7FQFjgdvgJjNVtLGN9fTCeZ~0m2uhLOBcJbJQrDCHteRjzBmTYe3Gc7lIXrzoA-CCzl46~Rg9pq~UEYk4v1kvl-XKpU~1JUfbBoDERiqc2DazIKU4BZKJohwjlzfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022_Supplementary_Data.zip?Expires=2147483647&Signature=NEv6aZ5j6R-RjghVh1O9KGbIrr5vHbMXEXLhW6GuhgTSY4cMvH39S0gtZ0dxo5iKqWTgwu5kdJrP01cEURAnENEi5jSpSaSpf9YezvPgu7SBrFZUw5zm7i~jatBRJAMwH8o1xJ2PccCD0u7nkhAmF7q3g6BgPUwbf7a-frfruUxZdEbBtauplmhJxldCZ~Iwkux~b5-zLTmszX2rdn3V9sEkO7FQFjgdvgJjNVtLGN9fTCeZ~0m2uhLOBcJbJQrDCHteRjzBmTYe3Gc7lIXrzoA-CCzl46~Rg9pq~UEYk4v1kvl-XKpU~1JUfbBoDERiqc2DazIKU4BZKJohwjlzfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span> —see the green line that indicates centromere position): for example, the region of chr. 14 that is marked by the centromeric histone PfCenH3, PF3D7_14:1070851-1075311, <a reveal-id="dsw022-B48" data-open="dsw022-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> was sequenced with a coverage of ∼×140, as was the core ∼2.5 kb centromere ( <a reveal-id="dsw022-F4" data-open="dsw022-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a> ). This indicated that the highly AT-rich nature of centromeres, as well as the presence of atypical repeats such as AATTAA, <a reveal-id="dsw022-B48" data-open="dsw022-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> did not impede the processivity of the polymerase during sequencing. Similarly, we observed even coverage of most telomeric and subtelomeric ends (×80–140; <a reveal-id="dsw022-F4" data-open="dsw022-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022_Supplementary_Data.zip?Expires=2147483647&Signature=NEv6aZ5j6R-RjghVh1O9KGbIrr5vHbMXEXLhW6GuhgTSY4cMvH39S0gtZ0dxo5iKqWTgwu5kdJrP01cEURAnENEi5jSpSaSpf9YezvPgu7SBrFZUw5zm7i~jatBRJAMwH8o1xJ2PccCD0u7nkhAmF7q3g6BgPUwbf7a-frfruUxZdEbBtauplmhJxldCZ~Iwkux~b5-zLTmszX2rdn3V9sEkO7FQFjgdvgJjNVtLGN9fTCeZ~0m2uhLOBcJbJQrDCHteRjzBmTYe3Gc7lIXrzoA-CCzl46~Rg9pq~UEYk4v1kvl-XKpU~1JUfbBoDERiqc2DazIKU4BZKJohwjlzfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span> ), suggesting that the &gt;11 kb length of the SMRT reads is sufficient to differentiate between regions of very high homology. </p>                        <a id="87199722" scrollto-destination="87199722"></a>
<div id="dsw022-F4" data-id="dsw022-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/m_dsw022f4p.png?Expires=2147483647&Signature=EQslFE0JkdACtDhYa~QtJ2Dkq23uC08dhAX2mq8rpeywSFCSKXXOTBShFtgWkNjQtDJbTG6JFBI1U1PWYF-okSCZeEmRxiu08B46KKoqAkCGjUErUg6c6-c7PQh50hfVH1I5hPN0Bra3D5nmwFPxY9kzIg7yw706phyKOihx7XiG0rp0gX-DMa5kMMPCot8w6sov1kRjUUFNegm3z8N7qDAMPyJMLv7I76-84uMlsNHo3rIZCA7-zg9Z9sywALMavcQoMqRx~~DP7ciTV1x~jfIcdtxc14ruQAKuzGOPswm9zqjCXYd7ZIi84MzyxMb0R3-YbvgWRODuZYse9N03Aw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The depth of coverage of the SMRT assembly and length of SMRT reads are sufficient to resolve centromeres and subtelomeric regions. (A) The depth of coverage of the ∼4.5 kb PfCenH3-occupied region of chromosome 14, which averages 92% (A + T), is shown. Zooming in, the raw sequences obtained for a 176 bp fragment of the core centromere are shown. (B) The depth of coverage of the ∼34.6 kb telomeric/subtelomeric region of chromosome 5 is shown, where each grey line represents a single read. Note that the horizontal black lines represent reads that do not map uniquely to the assembly and have a mapping QV of zero." path-from-xml="dsw022f4p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199722/dsw022f4p.png" path-from-xml="dsw022f4p.png" target="_blank">View large</a><a section="87199722" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022f4p.png?Expires=2147483647&Signature=pI267lBCYQf2VxpBuGOIshgL2PSEwsDQPSm-vMyxDnxI~6SmE83iagYvuvedPjD72zhyOTSVs5jqxqhrPcdOn4bjj-KHt3o27xZ6-Sqiu70Up9Hk2zvqec618tL33m0~yEZjAXrgnU3golLL6GoT6~KNvXlT81ISyj~Yx71LVEU3xcheMENXKX4OsdAplZRNHkIYBuZug6nptG4pztUQMQNP~5fl8Ec8O-uWNp-m6QTxtLENKAGy7W3CfcZIl8M8ZXQEMrzU4cL0MugtWaEUClmh5dvcl979J-2AKsn5FBp-lknxcqTr9Qh~xpHQsCAAKpsI5e-uAdZzscNEcz7lYw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199722&ar=2469970&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw022f4p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The depth of coverage of the SMRT assembly and length of SMRT reads are sufficient to resolve centromeres and subtelomeric regions. (A) The depth of coverage of the ∼4.5 kb PfCenH3-occupied region of chromosome 14, which averages 92% (A + T), is shown. Zooming in, the raw sequences obtained for a 176 bp fragment of the core centromere are shown. (B) The depth of coverage of the ∼34.6 kb telomeric/subtelomeric region of chromosome 5 is shown, where each grey line represents a single read. Note that the horizontal black lines represent reads that do not map uniquely to the assembly and have a mapping QV of zero.</p></div></div><div content-id="dsw022-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/m_dsw022f4p.png?Expires=2147483647&Signature=EQslFE0JkdACtDhYa~QtJ2Dkq23uC08dhAX2mq8rpeywSFCSKXXOTBShFtgWkNjQtDJbTG6JFBI1U1PWYF-okSCZeEmRxiu08B46KKoqAkCGjUErUg6c6-c7PQh50hfVH1I5hPN0Bra3D5nmwFPxY9kzIg7yw706phyKOihx7XiG0rp0gX-DMa5kMMPCot8w6sov1kRjUUFNegm3z8N7qDAMPyJMLv7I76-84uMlsNHo3rIZCA7-zg9Z9sywALMavcQoMqRx~~DP7ciTV1x~jfIcdtxc14ruQAKuzGOPswm9zqjCXYd7ZIi84MzyxMb0R3-YbvgWRODuZYse9N03Aw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The depth of coverage of the SMRT assembly and length of SMRT reads are sufficient to resolve centromeres and subtelomeric regions. (A) The depth of coverage of the ∼4.5 kb PfCenH3-occupied region of chromosome 14, which averages 92% (A + T), is shown. Zooming in, the raw sequences obtained for a 176 bp fragment of the core centromere are shown. (B) The depth of coverage of the ∼34.6 kb telomeric/subtelomeric region of chromosome 5 is shown, where each grey line represents a single read. Note that the horizontal black lines represent reads that do not map uniquely to the assembly and have a mapping QV of zero." path-from-xml="dsw022f4p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199722/dsw022f4p.png" path-from-xml="dsw022f4p.png" target="_blank">View large</a><a section="87199722" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022f4p.png?Expires=2147483647&Signature=pI267lBCYQf2VxpBuGOIshgL2PSEwsDQPSm-vMyxDnxI~6SmE83iagYvuvedPjD72zhyOTSVs5jqxqhrPcdOn4bjj-KHt3o27xZ6-Sqiu70Up9Hk2zvqec618tL33m0~yEZjAXrgnU3golLL6GoT6~KNvXlT81ISyj~Yx71LVEU3xcheMENXKX4OsdAplZRNHkIYBuZug6nptG4pztUQMQNP~5fl8Ec8O-uWNp-m6QTxtLENKAGy7W3CfcZIl8M8ZXQEMrzU4cL0MugtWaEUClmh5dvcl979J-2AKsn5FBp-lknxcqTr9Qh~xpHQsCAAKpsI5e-uAdZzscNEcz7lYw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199722&ar=2469970&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw022f4p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The depth of coverage of the SMRT assembly and length of SMRT reads are sufficient to resolve centromeres and subtelomeric regions. (A) The depth of coverage of the ∼4.5 kb PfCenH3-occupied region of chromosome 14, which averages 92% (A + T), is shown. Zooming in, the raw sequences obtained for a 176 bp fragment of the core centromere are shown. (B) The depth of coverage of the ∼34.6 kb telomeric/subtelomeric region of chromosome 5 is shown, where each grey line represents a single read. Note that the horizontal black lines represent reads that do not map uniquely to the assembly and have a mapping QV of zero.</p></div></div>                            <h3 scrollto-destination=87199723 id="87199723" class="section-title" >3.5. Novel genomic features and structural variants were resolved in the SMRT assembly</h3>
<p> To determine if this new SMRT assembly could enhance our understanding of parasite genome organization, we compared our data to the reference 3D7 genome <a reveal-id="dsw022-B8" data-open="dsw022-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> and noted that the total size of the chromosomal contigs in the SMRT assembly, i.e. 23,294,534 bp, was comparable to the 23,292,622 bp resolved in the reference genome ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022_Supplementary_Data.zip?Expires=2147483647&Signature=NEv6aZ5j6R-RjghVh1O9KGbIrr5vHbMXEXLhW6GuhgTSY4cMvH39S0gtZ0dxo5iKqWTgwu5kdJrP01cEURAnENEi5jSpSaSpf9YezvPgu7SBrFZUw5zm7i~jatBRJAMwH8o1xJ2PccCD0u7nkhAmF7q3g6BgPUwbf7a-frfruUxZdEbBtauplmhJxldCZ~Iwkux~b5-zLTmszX2rdn3V9sEkO7FQFjgdvgJjNVtLGN9fTCeZ~0m2uhLOBcJbJQrDCHteRjzBmTYe3Gc7lIXrzoA-CCzl46~Rg9pq~UEYk4v1kvl-XKpU~1JUfbBoDERiqc2DazIKU4BZKJohwjlzfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span> ). Moreover, the sizes of most chromosomes were similar between the two assemblies with the maximum increase in size presented by the stitched SMRT chr. 1 at 3% and the maximum decrease in size presented by chr. 8 at 1.2% ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022_Supplementary_Data.zip?Expires=2147483647&Signature=NEv6aZ5j6R-RjghVh1O9KGbIrr5vHbMXEXLhW6GuhgTSY4cMvH39S0gtZ0dxo5iKqWTgwu5kdJrP01cEURAnENEi5jSpSaSpf9YezvPgu7SBrFZUw5zm7i~jatBRJAMwH8o1xJ2PccCD0u7nkhAmF7q3g6BgPUwbf7a-frfruUxZdEbBtauplmhJxldCZ~Iwkux~b5-zLTmszX2rdn3V9sEkO7FQFjgdvgJjNVtLGN9fTCeZ~0m2uhLOBcJbJQrDCHteRjzBmTYe3Gc7lIXrzoA-CCzl46~Rg9pq~UEYk4v1kvl-XKpU~1JUfbBoDERiqc2DazIKU4BZKJohwjlzfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span> ). When we visualized the comparison using dot plots, as shown in the representative dot plot for chr. 7 ( <a reveal-id="dsw022-F5" data-open="dsw022-F5" class="link link-reveal link-table xref-fig">Fig. 5A</a> ) and other chromosomes ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022_Supplementary_Data.zip?Expires=2147483647&Signature=NEv6aZ5j6R-RjghVh1O9KGbIrr5vHbMXEXLhW6GuhgTSY4cMvH39S0gtZ0dxo5iKqWTgwu5kdJrP01cEURAnENEi5jSpSaSpf9YezvPgu7SBrFZUw5zm7i~jatBRJAMwH8o1xJ2PccCD0u7nkhAmF7q3g6BgPUwbf7a-frfruUxZdEbBtauplmhJxldCZ~Iwkux~b5-zLTmszX2rdn3V9sEkO7FQFjgdvgJjNVtLGN9fTCeZ~0m2uhLOBcJbJQrDCHteRjzBmTYe3Gc7lIXrzoA-CCzl46~Rg9pq~UEYk4v1kvl-XKpU~1JUfbBoDERiqc2DazIKU4BZKJohwjlzfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span> ), we found that while the majority of assembled sequences aligned well with the reference genome, the increase in chr. 1 size in the SMRT assembly was due to the lengthening of its left arm ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022_Supplementary_Data.zip?Expires=2147483647&Signature=NEv6aZ5j6R-RjghVh1O9KGbIrr5vHbMXEXLhW6GuhgTSY4cMvH39S0gtZ0dxo5iKqWTgwu5kdJrP01cEURAnENEi5jSpSaSpf9YezvPgu7SBrFZUw5zm7i~jatBRJAMwH8o1xJ2PccCD0u7nkhAmF7q3g6BgPUwbf7a-frfruUxZdEbBtauplmhJxldCZ~Iwkux~b5-zLTmszX2rdn3V9sEkO7FQFjgdvgJjNVtLGN9fTCeZ~0m2uhLOBcJbJQrDCHteRjzBmTYe3Gc7lIXrzoA-CCzl46~Rg9pq~UEYk4v1kvl-XKpU~1JUfbBoDERiqc2DazIKU4BZKJohwjlzfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4A</a></span> ), and the decrease in chr. 8 size was due to a shortening of its right arm ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022_Supplementary_Data.zip?Expires=2147483647&Signature=NEv6aZ5j6R-RjghVh1O9KGbIrr5vHbMXEXLhW6GuhgTSY4cMvH39S0gtZ0dxo5iKqWTgwu5kdJrP01cEURAnENEi5jSpSaSpf9YezvPgu7SBrFZUw5zm7i~jatBRJAMwH8o1xJ2PccCD0u7nkhAmF7q3g6BgPUwbf7a-frfruUxZdEbBtauplmhJxldCZ~Iwkux~b5-zLTmszX2rdn3V9sEkO7FQFjgdvgJjNVtLGN9fTCeZ~0m2uhLOBcJbJQrDCHteRjzBmTYe3Gc7lIXrzoA-CCzl46~Rg9pq~UEYk4v1kvl-XKpU~1JUfbBoDERiqc2DazIKU4BZKJohwjlzfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4H</a></span> ). Upon closer analysis, we concluded that both of these size discrepancies originated from changes in the lengths of subtelomeric repeat sequences. </p>                        <a id="87199725" scrollto-destination="87199725"></a>
<div id="dsw022-F5" data-id="dsw022-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/m_dsw022f5p.png?Expires=2147483647&Signature=gGO5qUBdx~NrxwPBunzN7QDfu6Vl7h6A4NU1EeA7mrRYIsHN08zKStJ907sSlnzLwWn7~6ux7Rho1w6a9wRHmvehW~aJ4ZEMxVUGFRkKQc4U3peW~gPjOJS-YGKDSoLAj675opzOegt4PDv5rRf-CFeD3NrlE~~kqqr15d8918xArfo-ewcqbnbMuelmntaFVluzqiS4RovwbGeCSt6kGMabGp4dJpvGuep8Jv7pHEtdaleIGtR3tsiFAaDav2JU5YyfxVXk5BqDK7xChZ6UvPTRxE8Vyo2XMoNtB5DYDF-yn5FTjaj7DQsFjgfzUOyNBOSwzk-av7v0iHZvrzl~1w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Dot plots comparing the SMRT assembly to the reference 3D7 genome identify large genomic variants. Dot plots were generated from Gepard nucleotide alignments of chromosomal contigs in the SMRT assembly ( x -axis) and chromosomes assembled in the reference 3D7 genome ( y -axis; Sanger reference 8 ). Each dot is a grey-scale representation of nucleotide identity within a 30-nucleotide window centred on that position. The main diagonal line shows the alignment between the SMRT assembly and the reference genome. Off-diagonal lines parallel to the main diagonal indicate parallel duplications in the chromosome while off-diagonal perpendicular lines indicate inversions. (A) Chromosome 7. (B) Chromosome 10. Zooming into position 1,612,060, a 14 kb tandem duplication was resolved in the SMRT assembly relative to the reference. (C) Chromosome 4. Zooming into position 939,044, a 29 kb insertion was resolved in the SMRT assembly relative to the reference. Further zooming into a position immediately downstream of this insertion, a 1.5 kb stretch was detected in the SMRT assembly that showed very low homology to the corresponding 2.5 kb stretch present in the reference genome, and hence labelled ‘novel’. (D) Chromosome 12. Zooming into position 1,707,528, a 16 kb tandem duplication was resolved in the SMRT assembly relative to the reference. Plots not drawn to scale. " path-from-xml="dsw022f5p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199725/dsw022f5p.png" path-from-xml="dsw022f5p.png" target="_blank">View large</a><a section="87199725" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022f5p.png?Expires=2147483647&Signature=ZUdcNUpuKKqdaPRmxbtjEiv-pKUvoGgYI1AyUQfoIKq0vdlUbQ6NiMtzGcHMO8Z0ft~ZOdF5g-9EKTVUkPqWopDCWNScmuSfpPAziqnUFVG5aWVvfYbfoWafx9j6d1YCodO3VB7YE~-RxtJSPhpzGkgjgyH1d7pUWkKFsNf4U8m9dUN1iHkGl2EzCC2-hka4Cjj95CbpFNzVfuRHUCJqaepaTwMQH0SB-R48e1UuM7v1DE3GKP09HFA-3TD4BW8RmYrevFXntDDfbHp-q6hAxSN1r49kgLt7gmaJm5OPfYSINlGSXSCMmY02eZocfVB0klX8N8IsIxYKwGV6gk~80A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199725&ar=2469970&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw022f5p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Dot plots comparing the SMRT assembly to the reference 3D7 genome identify large genomic variants. Dot plots were generated from Gepard nucleotide alignments of chromosomal contigs in the SMRT assembly ( <em>x</em> -axis) and chromosomes assembled in the reference 3D7 genome ( <em>y</em> -axis; Sanger reference <a reveal-id="dsw022-B8" data-open="dsw022-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> ). Each dot is a grey-scale representation of nucleotide identity within a 30-nucleotide window centred on that position. The main diagonal line shows the alignment between the SMRT assembly and the reference genome. Off-diagonal lines parallel to the main diagonal indicate parallel duplications in the chromosome while off-diagonal perpendicular lines indicate inversions. (A) Chromosome 7. (B) Chromosome 10. Zooming into position 1,612,060, a 14 kb tandem duplication was resolved in the SMRT assembly relative to the reference. (C) Chromosome 4. Zooming into position 939,044, a 29 kb insertion was resolved in the SMRT assembly relative to the reference. Further zooming into a position immediately downstream of this insertion, a 1.5 kb stretch was detected in the SMRT assembly that showed very low homology to the corresponding 2.5 kb stretch present in the reference genome, and hence labelled ‘novel’. (D) Chromosome 12. Zooming into position 1,707,528, a 16 kb tandem duplication was resolved in the SMRT assembly relative to the reference. Plots not drawn to scale. </p></div></div><div content-id="dsw022-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/m_dsw022f5p.png?Expires=2147483647&Signature=gGO5qUBdx~NrxwPBunzN7QDfu6Vl7h6A4NU1EeA7mrRYIsHN08zKStJ907sSlnzLwWn7~6ux7Rho1w6a9wRHmvehW~aJ4ZEMxVUGFRkKQc4U3peW~gPjOJS-YGKDSoLAj675opzOegt4PDv5rRf-CFeD3NrlE~~kqqr15d8918xArfo-ewcqbnbMuelmntaFVluzqiS4RovwbGeCSt6kGMabGp4dJpvGuep8Jv7pHEtdaleIGtR3tsiFAaDav2JU5YyfxVXk5BqDK7xChZ6UvPTRxE8Vyo2XMoNtB5DYDF-yn5FTjaj7DQsFjgfzUOyNBOSwzk-av7v0iHZvrzl~1w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Dot plots comparing the SMRT assembly to the reference 3D7 genome identify large genomic variants. Dot plots were generated from Gepard nucleotide alignments of chromosomal contigs in the SMRT assembly ( x -axis) and chromosomes assembled in the reference 3D7 genome ( y -axis; Sanger reference 8 ). Each dot is a grey-scale representation of nucleotide identity within a 30-nucleotide window centred on that position. The main diagonal line shows the alignment between the SMRT assembly and the reference genome. Off-diagonal lines parallel to the main diagonal indicate parallel duplications in the chromosome while off-diagonal perpendicular lines indicate inversions. (A) Chromosome 7. (B) Chromosome 10. Zooming into position 1,612,060, a 14 kb tandem duplication was resolved in the SMRT assembly relative to the reference. (C) Chromosome 4. Zooming into position 939,044, a 29 kb insertion was resolved in the SMRT assembly relative to the reference. Further zooming into a position immediately downstream of this insertion, a 1.5 kb stretch was detected in the SMRT assembly that showed very low homology to the corresponding 2.5 kb stretch present in the reference genome, and hence labelled ‘novel’. (D) Chromosome 12. Zooming into position 1,707,528, a 16 kb tandem duplication was resolved in the SMRT assembly relative to the reference. Plots not drawn to scale. " path-from-xml="dsw022f5p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199725/dsw022f5p.png" path-from-xml="dsw022f5p.png" target="_blank">View large</a><a section="87199725" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022f5p.png?Expires=2147483647&Signature=ZUdcNUpuKKqdaPRmxbtjEiv-pKUvoGgYI1AyUQfoIKq0vdlUbQ6NiMtzGcHMO8Z0ft~ZOdF5g-9EKTVUkPqWopDCWNScmuSfpPAziqnUFVG5aWVvfYbfoWafx9j6d1YCodO3VB7YE~-RxtJSPhpzGkgjgyH1d7pUWkKFsNf4U8m9dUN1iHkGl2EzCC2-hka4Cjj95CbpFNzVfuRHUCJqaepaTwMQH0SB-R48e1UuM7v1DE3GKP09HFA-3TD4BW8RmYrevFXntDDfbHp-q6hAxSN1r49kgLt7gmaJm5OPfYSINlGSXSCMmY02eZocfVB0klX8N8IsIxYKwGV6gk~80A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199725&ar=2469970&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw022f5p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Dot plots comparing the SMRT assembly to the reference 3D7 genome identify large genomic variants. Dot plots were generated from Gepard nucleotide alignments of chromosomal contigs in the SMRT assembly ( <em>x</em> -axis) and chromosomes assembled in the reference 3D7 genome ( <em>y</em> -axis; Sanger reference <a reveal-id="dsw022-B8" data-open="dsw022-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> ). Each dot is a grey-scale representation of nucleotide identity within a 30-nucleotide window centred on that position. The main diagonal line shows the alignment between the SMRT assembly and the reference genome. Off-diagonal lines parallel to the main diagonal indicate parallel duplications in the chromosome while off-diagonal perpendicular lines indicate inversions. (A) Chromosome 7. (B) Chromosome 10. Zooming into position 1,612,060, a 14 kb tandem duplication was resolved in the SMRT assembly relative to the reference. (C) Chromosome 4. Zooming into position 939,044, a 29 kb insertion was resolved in the SMRT assembly relative to the reference. Further zooming into a position immediately downstream of this insertion, a 1.5 kb stretch was detected in the SMRT assembly that showed very low homology to the corresponding 2.5 kb stretch present in the reference genome, and hence labelled ‘novel’. (D) Chromosome 12. Zooming into position 1,707,528, a 16 kb tandem duplication was resolved in the SMRT assembly relative to the reference. Plots not drawn to scale. </p></div></div><p> Furthermore, in select cases, the longer reads generated by SMRT sequencing allowed for the resolution of genomic features that were not previously described for the reference. These included large duplications and insertions ( <a reveal-id="dsw022-F5" data-open="dsw022-F5" class="link link-reveal link-table xref-fig">Figs 5B–D</a> ). For example, at position 1,612,060 of chromosome 10, a 14 kb duplication was apparent in the SMRT assembly relative to the reference ( <a reveal-id="dsw022-F5" data-open="dsw022-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a> ), which results in the addition of three more <em>rifin</em> genes to the <em>P. falciparum</em> genome; PfRifins are virulence molecules that are expressed on the surface of infected erythrocytes, mediate the binding of infected erythrocytes to the vasculature <a reveal-id="dsw022-B51" data-open="dsw022-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> and undergo antigenic variation. <a reveal-id="dsw022-B52" data-open="dsw022-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> Another example is position 939,044 of chromosome 4, where a 29 kb insertion relative to the reference results in the duplication of the following features to the <em>P. falciparum</em> genome: three <em>var</em> genes, one ncRNA-encoding open reading frame (ORF), one <em>rif</em> and one gene encoding a conserved protein of unknown function ( <a reveal-id="dsw022-F5" data-open="dsw022-F5" class="link link-reveal link-table xref-fig">Fig. 5C</a> ). However, this region shows a substantial pileup of extra coverage, indicative of highly repetitive sequences, and may contain additional features that were not fully resolved in the SMRT assembly. One way to address this would be to analyse 40 kb size-selected SMRTbell libraries of <em>P. falciparum</em> DNA. Furthermore, adjacent to this insertion, we observed a 1.5 kb stretch of DNA that was novel compared to the corresponding 2.5 kb stretch in the reference ( <a reveal-id="dsw022-F5" data-open="dsw022-F5" class="link link-reveal link-table xref-fig">Fig. 5C;</a> ‘zoom—novel’) indicating that this subtelomeric region of chromosome 4 may be more complex than previously annotated. As a third example, we detected a 16 kb duplication at position 1,707,528 of chromosome 12 of the SMRT assembly ( <a reveal-id="dsw022-F5" data-open="dsw022-F5" class="link link-reveal link-table xref-fig">Fig. 5D</a> ), which results in the addition of 2 <em>var</em> genes and one ncRNA-encoding ORF to the <em>P. falciparum</em> annotation. </p><p> In addition to the features described above that span several kilobases (&gt;10 kb), we utilized our assembly comparison tool called Assemblytics <a reveal-id="dsw022-B39" data-open="dsw022-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> to identify additional structural variants in our SMRT assembly relative to the reference genome. As summarized in <a reveal-id="dsw022-T2" data-open="dsw022-T2" class="link link-reveal link-table xref-fig">Table 2</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022_Supplementary_Data.zip?Expires=2147483647&Signature=NEv6aZ5j6R-RjghVh1O9KGbIrr5vHbMXEXLhW6GuhgTSY4cMvH39S0gtZ0dxo5iKqWTgwu5kdJrP01cEURAnENEi5jSpSaSpf9YezvPgu7SBrFZUw5zm7i~jatBRJAMwH8o1xJ2PccCD0u7nkhAmF7q3g6BgPUwbf7a-frfruUxZdEbBtauplmhJxldCZ~Iwkux~b5-zLTmszX2rdn3V9sEkO7FQFjgdvgJjNVtLGN9fTCeZ~0m2uhLOBcJbJQrDCHteRjzBmTYe3Gc7lIXrzoA-CCzl46~Rg9pq~UEYk4v1kvl-XKpU~1JUfbBoDERiqc2DazIKU4BZKJohwjlzfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span> , we identified 10,248 insertions, 426 deletions, 22 tandem expansions, 8 tandem contractions, 8 repeat expansions and 3 repeat contractions of size 2 bp to 10 kb in the SMRT assembly; the size distribution of these structural variations is depicted in <a reveal-id="dsw022-F6" data-open="dsw022-F6" class="link link-reveal link-table xref-fig">Figs 6A and B</a> . Interestingly, homopolymer tract expansions, i.e. poly-A, poly-T and poly-AT expansions, of  &lt; 10 bp in size account for ∼61% of all insertions identified ( <a reveal-id="dsw022-T2" data-open="dsw022-T2" class="link link-reveal link-table xref-fig">Table 2</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022_Supplementary_Data.zip?Expires=2147483647&Signature=NEv6aZ5j6R-RjghVh1O9KGbIrr5vHbMXEXLhW6GuhgTSY4cMvH39S0gtZ0dxo5iKqWTgwu5kdJrP01cEURAnENEi5jSpSaSpf9YezvPgu7SBrFZUw5zm7i~jatBRJAMwH8o1xJ2PccCD0u7nkhAmF7q3g6BgPUwbf7a-frfruUxZdEbBtauplmhJxldCZ~Iwkux~b5-zLTmszX2rdn3V9sEkO7FQFjgdvgJjNVtLGN9fTCeZ~0m2uhLOBcJbJQrDCHteRjzBmTYe3Gc7lIXrzoA-CCzl46~Rg9pq~UEYk4v1kvl-XKpU~1JUfbBoDERiqc2DazIKU4BZKJohwjlzfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span> ). Some examples of variants include a 1,019 bp insertion at position 2,328,302 of chr. 13; 280 and 117 bp deletions at positions 1,452,348 of chr. 9 and 754,796 of chr. 13, respectively; a 4,131 bp tandem expansion at position 1,543,910 of chr. 10; a 378 bp tandem contraction at position 17,963 of chr. 4; a 5,100 bp repeat expansion at position 973,156 of chr. 12; and a 1,018 bp repeat contraction at position 965,633 of chr. 4 ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022_Supplementary_Data.zip?Expires=2147483647&Signature=NEv6aZ5j6R-RjghVh1O9KGbIrr5vHbMXEXLhW6GuhgTSY4cMvH39S0gtZ0dxo5iKqWTgwu5kdJrP01cEURAnENEi5jSpSaSpf9YezvPgu7SBrFZUw5zm7i~jatBRJAMwH8o1xJ2PccCD0u7nkhAmF7q3g6BgPUwbf7a-frfruUxZdEbBtauplmhJxldCZ~Iwkux~b5-zLTmszX2rdn3V9sEkO7FQFjgdvgJjNVtLGN9fTCeZ~0m2uhLOBcJbJQrDCHteRjzBmTYe3Gc7lIXrzoA-CCzl46~Rg9pq~UEYk4v1kvl-XKpU~1JUfbBoDERiqc2DazIKU4BZKJohwjlzfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span> ). These changes affect genic, intergenic and subtelomeric regions of chromosomes: for example, the 117 bp deletion at position 754,796 of chromosome 13 is within the intronless gene <em>PF3D7_1318300</em> , which encodes a conserved <em>Plasmodium</em> protein of unknown function, while the 280 bp deletion in chr. 9 affects an intergenic region. Similarly, the 4,131 bp tandem expansion at position 1,543,910 of chr. 10 is within the intronless gene <em>PF3D7_1038400</em> , which encodes the gametocyte-specific protein Pf11-1, while the 378 bp tandem contraction in chr. 4 is in a subtelomeric region. Therefore, these variants will need to be rigorously curated to understand their impact on parasite biology. Given that the SMRT assembly and the reference match well in contiguity and have nearly the same total sequence length ( <a reveal-id="dsw022-F6" data-open="dsw022-F6" class="link link-reveal link-table xref-fig">Fig. 6C</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022_Supplementary_Data.zip?Expires=2147483647&Signature=NEv6aZ5j6R-RjghVh1O9KGbIrr5vHbMXEXLhW6GuhgTSY4cMvH39S0gtZ0dxo5iKqWTgwu5kdJrP01cEURAnENEi5jSpSaSpf9YezvPgu7SBrFZUw5zm7i~jatBRJAMwH8o1xJ2PccCD0u7nkhAmF7q3g6BgPUwbf7a-frfruUxZdEbBtauplmhJxldCZ~Iwkux~b5-zLTmszX2rdn3V9sEkO7FQFjgdvgJjNVtLGN9fTCeZ~0m2uhLOBcJbJQrDCHteRjzBmTYe3Gc7lIXrzoA-CCzl46~Rg9pq~UEYk4v1kvl-XKpU~1JUfbBoDERiqc2DazIKU4BZKJohwjlzfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span> ), and given its high-quality score ( <a reveal-id="dsw022-F3" data-open="dsw022-F3" class="link link-reveal link-table xref-fig">Fig. 3A;</a> QV50), we can conclude that our SMRT assembly and the reference genome are of equal quality and that the structural variants identified here are significant to understanding the complexity of the parasite genome. </p>                        <a id="87199728" scrollto-destination="87199728"></a>
<div id="dsw022-F6" data-id="dsw022-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/m_dsw022f6p.png?Expires=2147483647&Signature=dsZkZIlRMJ5CEkdddUJ6aasdJu8Xc3MPUwPmsoVJhlMNg8F3bia8EGe3ktxS7jzR4NQfgBdDzrIvX2G4IS95ok8XfdZZ9q54Z3-1sublbK4BSYM-xY5ExYm2v4GQuJF8xB2Fl83IS2OiwiE0tDs3PsM2bKeApIEJLWtrzH-LcH0eeFtkSwDP2jYJ-Pi6FY6qsT1hWQwtZmf8L6nGfLiX~vmzYWms-Nm4BwZY4YgvhgCw4DMtHQLxO3b2nCQpRWaLmx1WQtBCcEiv35FXhxXMze~hfvxHQ0npshjeq-cmq4b6-d5kc~dqJ758BTS5XTXNHPgV2oMuNpHd4MmTmxb1Mg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" The majority of variations in the SMRT assembly relative to the reference are small insertions. Variants ranging from 2 to 10 kb in size were called using Assemblytics. 39 (A) Size distribution analysis of variants from 2 to 500 bp in size showed that the majority of insertions are &lt;50 bp in size while the majority of deletions are between 10 and 500 bp in size. The x -axis represents variant size in bp and the y -axis represents a variant number. (B) Large structural variants from 500 bp to 10 kb in size are depicted with the x -axis representing variant size in bp and the y -axis representing variant number. (C) Cumulative sequence length plot showing the nearly identical contiguity and the total size of the SMRT assembly (query; in green) versus the reference  (in blue). The length of each individual sequence is indicated on the y -axis with the cumulative sum of sorted sequence lengths on the x -axis. The N50 of the reference (50% on the x -axis) is 1.688 Mb, while the N50 of the SMRT assembly is 1.712 Mb. " path-from-xml="dsw022f6p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199728/dsw022f6p.png" path-from-xml="dsw022f6p.png" target="_blank">View large</a><a section="87199728" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022f6p.png?Expires=2147483647&Signature=bGnWmo48W6Rw~pm-KY8JCbUruIWgiPPQ3hWzptMuXiCQ6UQR9SYS72pcciDxLgCo6RqVpea1wCvKoTYHkrcyGiri9v7ekZZOrQe0tKHD7py6q-3L6yVfjEKleWzzDd8AHEFSyrudkEoKi9GWGVZusfuRRFBDY1U3GT248f8WpvoH55ye2hRLXe6Mvoe8ETmOpKBeGvnDZV3QpdBbKhLG5ClG7~YvK38FSwm~KKHS6j9houl0bTh~iRxSZMEn3IeNvsjm77Pt9yGlOVmJQ2a4pKOsxxFaOvsUMPatLeMHV12Xnj96n-IO6AFae-2EoGmkkp~wHzxIcvRvsaAWitoXSg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199728&ar=2469970&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw022f6p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> The majority of variations in the SMRT assembly relative to the reference are small insertions. Variants ranging from 2 to 10 kb in size were called using Assemblytics. <a reveal-id="dsw022-B39" data-open="dsw022-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> (A) Size distribution analysis of variants from 2 to 500 bp in size showed that the majority of insertions are &lt;50 bp in size while the majority of deletions are between 10 and 500 bp in size. The <em>x</em> -axis represents variant size in bp and the <em>y</em> -axis represents a variant number. (B) Large structural variants from 500 bp to 10 kb in size are depicted with the <em>x</em> -axis representing variant size in bp and the <em>y</em> -axis representing variant number. (C) Cumulative sequence length plot showing the nearly identical contiguity and the total size of the SMRT assembly (query; in green) versus the reference  (in blue). The length of each individual sequence is indicated on the <em>y</em> -axis with the cumulative sum of sorted sequence lengths on the <em>x</em> -axis. The N50 of the reference (50% on the <em>x</em> -axis) is 1.688 Mb, while the N50 of the SMRT assembly is 1.712 Mb. </p></div></div><div content-id="dsw022-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/m_dsw022f6p.png?Expires=2147483647&Signature=dsZkZIlRMJ5CEkdddUJ6aasdJu8Xc3MPUwPmsoVJhlMNg8F3bia8EGe3ktxS7jzR4NQfgBdDzrIvX2G4IS95ok8XfdZZ9q54Z3-1sublbK4BSYM-xY5ExYm2v4GQuJF8xB2Fl83IS2OiwiE0tDs3PsM2bKeApIEJLWtrzH-LcH0eeFtkSwDP2jYJ-Pi6FY6qsT1hWQwtZmf8L6nGfLiX~vmzYWms-Nm4BwZY4YgvhgCw4DMtHQLxO3b2nCQpRWaLmx1WQtBCcEiv35FXhxXMze~hfvxHQ0npshjeq-cmq4b6-d5kc~dqJ758BTS5XTXNHPgV2oMuNpHd4MmTmxb1Mg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" The majority of variations in the SMRT assembly relative to the reference are small insertions. Variants ranging from 2 to 10 kb in size were called using Assemblytics. 39 (A) Size distribution analysis of variants from 2 to 500 bp in size showed that the majority of insertions are &lt;50 bp in size while the majority of deletions are between 10 and 500 bp in size. The x -axis represents variant size in bp and the y -axis represents a variant number. (B) Large structural variants from 500 bp to 10 kb in size are depicted with the x -axis representing variant size in bp and the y -axis representing variant number. (C) Cumulative sequence length plot showing the nearly identical contiguity and the total size of the SMRT assembly (query; in green) versus the reference  (in blue). The length of each individual sequence is indicated on the y -axis with the cumulative sum of sorted sequence lengths on the x -axis. The N50 of the reference (50% on the x -axis) is 1.688 Mb, while the N50 of the SMRT assembly is 1.712 Mb. " path-from-xml="dsw022f6p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199728/dsw022f6p.png" path-from-xml="dsw022f6p.png" target="_blank">View large</a><a section="87199728" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022f6p.png?Expires=2147483647&Signature=bGnWmo48W6Rw~pm-KY8JCbUruIWgiPPQ3hWzptMuXiCQ6UQR9SYS72pcciDxLgCo6RqVpea1wCvKoTYHkrcyGiri9v7ekZZOrQe0tKHD7py6q-3L6yVfjEKleWzzDd8AHEFSyrudkEoKi9GWGVZusfuRRFBDY1U3GT248f8WpvoH55ye2hRLXe6Mvoe8ETmOpKBeGvnDZV3QpdBbKhLG5ClG7~YvK38FSwm~KKHS6j9houl0bTh~iRxSZMEn3IeNvsjm77Pt9yGlOVmJQ2a4pKOsxxFaOvsUMPatLeMHV12Xnj96n-IO6AFae-2EoGmkkp~wHzxIcvRvsaAWitoXSg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199728&ar=2469970&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw022f6p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> The majority of variations in the SMRT assembly relative to the reference are small insertions. Variants ranging from 2 to 10 kb in size were called using Assemblytics. <a reveal-id="dsw022-B39" data-open="dsw022-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> (A) Size distribution analysis of variants from 2 to 500 bp in size showed that the majority of insertions are &lt;50 bp in size while the majority of deletions are between 10 and 500 bp in size. The <em>x</em> -axis represents variant size in bp and the <em>y</em> -axis represents a variant number. (B) Large structural variants from 500 bp to 10 kb in size are depicted with the <em>x</em> -axis representing variant size in bp and the <em>y</em> -axis representing variant number. (C) Cumulative sequence length plot showing the nearly identical contiguity and the total size of the SMRT assembly (query; in green) versus the reference  (in blue). The length of each individual sequence is indicated on the <em>y</em> -axis with the cumulative sum of sorted sequence lengths on the <em>x</em> -axis. The N50 of the reference (50% on the <em>x</em> -axis) is 1.688 Mb, while the N50 of the SMRT assembly is 1.712 Mb. </p></div></div>                        <a id="87199729" scrollto-destination="87199729"></a>
<div content-id="dsw022-T2" class="table-modal"><div class="table-wrap"><div id="dsw022-T2" class="table-wrap-title" data-id="dsw022-T2"><div class="label -label">Table 2.</div><div class="caption"><p>Size distribution of structural variants in the SMRT assembly relative to the reference genome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="3">Size range (bp) </th><th colspan="14"> Variant type <hr /> </th></tr><tr><th colspan="2"> Insertion <hr /> </th><th colspan="2"> Deletion <hr /> </th><th colspan="2"> Tandem expansion <hr /> </th><th colspan="2"> Tandem contraction <hr /> </th><th colspan="2"> Repeat expansion <hr /> </th><th colspan="2"> Repeat contraction <hr /> </th><th colspan="2"> Homo-polymer tract expansion <a reveal-id="dsw022-TF4" data-open="dsw022-TF4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><hr /> </th></tr><tr><th>Count </th><th>Total (bp) </th><th>Count </th><th>Total (bp) </th><th>Count </th><th>Total (bp) </th><th>Count </th><th>Total (bp) </th><th>Count </th><th>Total (bp) </th><th>Count </th><th>Total (bp) </th><th>Type </th><th>Count </th></tr></thead><tbody><tr><td>2–10 </td><td char=".">10,112 </td><td char=".">26,499 </td><td char=".">335 </td><td char=".">1,148 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">8 </td><td char=".">0 </td><td char=".">0 </td><td>poly-A </td><td char=".">2,535 </td></tr><tr><td>10–50 </td><td char=".">130 </td><td char=".">2,410 </td><td char=".">82 </td><td char=".">1,786 </td><td char=".">2 </td><td char=".">90 </td><td char=".">3 </td><td char=".">114 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td>poly-T </td><td char=".">2,517 </td></tr><tr><td>50–100 </td><td char=".">1 </td><td char=".">66 </td><td char=".">6 </td><td char=".">401 </td><td char=".">2 </td><td char=".">142 </td><td char=".">2 </td><td char=".">123 </td><td char=".">1 </td><td char=".">77 </td><td char=".">1 </td><td char=".">97 </td><td>poly-TA </td><td char=".">1,108 </td></tr><tr><td>100–1,000 </td><td char=".">4 </td><td char=".">647 </td><td char=".">3 </td><td char=".">507 </td><td char=".">15 </td><td char=".">5,446 </td><td char=".">3 </td><td char=".">865 </td><td char=".">4 </td><td char=".">2,123 </td><td char=".">1 </td><td char=".">808 </td><td>poly-TG </td><td char=".">3 </td></tr><tr><td>1,000–10,000 </td><td char=".">1 </td><td char=".">1,019 </td><td char=".">0 </td><td char=".">0 </td><td char=".">3 </td><td char=".">8,294 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">7,526 </td><td char=".">1 </td><td char=".">1,018 </td><td>poly-AG and poly-TC </td><td char=".">1 each </td></tr><tr><td>Total </td><td char=".">10,248 </td><td char=".">30,641 </td><td char=".">426 </td><td char=".">3,842 </td><td char=".">22 </td><td char=".">13,972 </td><td char=".">8 </td><td char=".">1,102 </td><td char=".">8 </td><td char=".">9,734 </td><td char=".">3 </td><td char=".">1,923 </td><td>Total </td><td char=".">6,164 </td></tr></tbody></table></div><div content-id="dsw022-T2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="3">Size range (bp) </th><th colspan="14"> Variant type <hr /> </th></tr><tr><th colspan="2"> Insertion <hr /> </th><th colspan="2"> Deletion <hr /> </th><th colspan="2"> Tandem expansion <hr /> </th><th colspan="2"> Tandem contraction <hr /> </th><th colspan="2"> Repeat expansion <hr /> </th><th colspan="2"> Repeat contraction <hr /> </th><th colspan="2"> Homo-polymer tract expansion <a reveal-id="dsw022-TF4" data-open="dsw022-TF4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><hr /> </th></tr><tr><th>Count </th><th>Total (bp) </th><th>Count </th><th>Total (bp) </th><th>Count </th><th>Total (bp) </th><th>Count </th><th>Total (bp) </th><th>Count </th><th>Total (bp) </th><th>Count </th><th>Total (bp) </th><th>Type </th><th>Count </th></tr></thead><tbody><tr><td>2–10 </td><td char=".">10,112 </td><td char=".">26,499 </td><td char=".">335 </td><td char=".">1,148 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">8 </td><td char=".">0 </td><td char=".">0 </td><td>poly-A </td><td char=".">2,535 </td></tr><tr><td>10–50 </td><td char=".">130 </td><td char=".">2,410 </td><td char=".">82 </td><td char=".">1,786 </td><td char=".">2 </td><td char=".">90 </td><td char=".">3 </td><td char=".">114 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td>poly-T </td><td char=".">2,517 </td></tr><tr><td>50–100 </td><td char=".">1 </td><td char=".">66 </td><td char=".">6 </td><td char=".">401 </td><td char=".">2 </td><td char=".">142 </td><td char=".">2 </td><td char=".">123 </td><td char=".">1 </td><td char=".">77 </td><td char=".">1 </td><td char=".">97 </td><td>poly-TA </td><td char=".">1,108 </td></tr><tr><td>100–1,000 </td><td char=".">4 </td><td char=".">647 </td><td char=".">3 </td><td char=".">507 </td><td char=".">15 </td><td char=".">5,446 </td><td char=".">3 </td><td char=".">865 </td><td char=".">4 </td><td char=".">2,123 </td><td char=".">1 </td><td char=".">808 </td><td>poly-TG </td><td char=".">3 </td></tr><tr><td>1,000–10,000 </td><td char=".">1 </td><td char=".">1,019 </td><td char=".">0 </td><td char=".">0 </td><td char=".">3 </td><td char=".">8,294 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">7,526 </td><td char=".">1 </td><td char=".">1,018 </td><td>poly-AG and poly-TC </td><td char=".">1 each </td></tr><tr><td>Total </td><td char=".">10,248 </td><td char=".">30,641 </td><td char=".">426 </td><td char=".">3,842 </td><td char=".">22 </td><td char=".">13,972 </td><td char=".">8 </td><td char=".">1,102 </td><td char=".">8 </td><td char=".">9,734 </td><td char=".">3 </td><td char=".">1,923 </td><td>Total </td><td char=".">6,164 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="dsw022-TF4"><div class="fn"><p><sup>a</sup> Length of expansion considered is between 2 and 10 bp. </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87199729" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="dsw022-T2" class="table-wrap-title" data-id="dsw022-T2"><div class="label -label">Table 2.</div><div class="caption"><p>Size distribution of structural variants in the SMRT assembly relative to the reference genome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="3">Size range (bp) </th><th colspan="14"> Variant type <hr /> </th></tr><tr><th colspan="2"> Insertion <hr /> </th><th colspan="2"> Deletion <hr /> </th><th colspan="2"> Tandem expansion <hr /> </th><th colspan="2"> Tandem contraction <hr /> </th><th colspan="2"> Repeat expansion <hr /> </th><th colspan="2"> Repeat contraction <hr /> </th><th colspan="2"> Homo-polymer tract expansion <a reveal-id="dsw022-TF4" data-open="dsw022-TF4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><hr /> </th></tr><tr><th>Count </th><th>Total (bp) </th><th>Count </th><th>Total (bp) </th><th>Count </th><th>Total (bp) </th><th>Count </th><th>Total (bp) </th><th>Count </th><th>Total (bp) </th><th>Count </th><th>Total (bp) </th><th>Type </th><th>Count </th></tr></thead><tbody><tr><td>2–10 </td><td char=".">10,112 </td><td char=".">26,499 </td><td char=".">335 </td><td char=".">1,148 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">8 </td><td char=".">0 </td><td char=".">0 </td><td>poly-A </td><td char=".">2,535 </td></tr><tr><td>10–50 </td><td char=".">130 </td><td char=".">2,410 </td><td char=".">82 </td><td char=".">1,786 </td><td char=".">2 </td><td char=".">90 </td><td char=".">3 </td><td char=".">114 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td>poly-T </td><td char=".">2,517 </td></tr><tr><td>50–100 </td><td char=".">1 </td><td char=".">66 </td><td char=".">6 </td><td char=".">401 </td><td char=".">2 </td><td char=".">142 </td><td char=".">2 </td><td char=".">123 </td><td char=".">1 </td><td char=".">77 </td><td char=".">1 </td><td char=".">97 </td><td>poly-TA </td><td char=".">1,108 </td></tr><tr><td>100–1,000 </td><td char=".">4 </td><td char=".">647 </td><td char=".">3 </td><td char=".">507 </td><td char=".">15 </td><td char=".">5,446 </td><td char=".">3 </td><td char=".">865 </td><td char=".">4 </td><td char=".">2,123 </td><td char=".">1 </td><td char=".">808 </td><td>poly-TG </td><td char=".">3 </td></tr><tr><td>1,000–10,000 </td><td char=".">1 </td><td char=".">1,019 </td><td char=".">0 </td><td char=".">0 </td><td char=".">3 </td><td char=".">8,294 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">7,526 </td><td char=".">1 </td><td char=".">1,018 </td><td>poly-AG and poly-TC </td><td char=".">1 each </td></tr><tr><td>Total </td><td char=".">10,248 </td><td char=".">30,641 </td><td char=".">426 </td><td char=".">3,842 </td><td char=".">22 </td><td char=".">13,972 </td><td char=".">8 </td><td char=".">1,102 </td><td char=".">8 </td><td char=".">9,734 </td><td char=".">3 </td><td char=".">1,923 </td><td>Total </td><td char=".">6,164 </td></tr></tbody></table></div><div content-id="dsw022-T2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="3">Size range (bp) </th><th colspan="14"> Variant type <hr /> </th></tr><tr><th colspan="2"> Insertion <hr /> </th><th colspan="2"> Deletion <hr /> </th><th colspan="2"> Tandem expansion <hr /> </th><th colspan="2"> Tandem contraction <hr /> </th><th colspan="2"> Repeat expansion <hr /> </th><th colspan="2"> Repeat contraction <hr /> </th><th colspan="2"> Homo-polymer tract expansion <a reveal-id="dsw022-TF4" data-open="dsw022-TF4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><hr /> </th></tr><tr><th>Count </th><th>Total (bp) </th><th>Count </th><th>Total (bp) </th><th>Count </th><th>Total (bp) </th><th>Count </th><th>Total (bp) </th><th>Count </th><th>Total (bp) </th><th>Count </th><th>Total (bp) </th><th>Type </th><th>Count </th></tr></thead><tbody><tr><td>2–10 </td><td char=".">10,112 </td><td char=".">26,499 </td><td char=".">335 </td><td char=".">1,148 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">8 </td><td char=".">0 </td><td char=".">0 </td><td>poly-A </td><td char=".">2,535 </td></tr><tr><td>10–50 </td><td char=".">130 </td><td char=".">2,410 </td><td char=".">82 </td><td char=".">1,786 </td><td char=".">2 </td><td char=".">90 </td><td char=".">3 </td><td char=".">114 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td>poly-T </td><td char=".">2,517 </td></tr><tr><td>50–100 </td><td char=".">1 </td><td char=".">66 </td><td char=".">6 </td><td char=".">401 </td><td char=".">2 </td><td char=".">142 </td><td char=".">2 </td><td char=".">123 </td><td char=".">1 </td><td char=".">77 </td><td char=".">1 </td><td char=".">97 </td><td>poly-TA </td><td char=".">1,108 </td></tr><tr><td>100–1,000 </td><td char=".">4 </td><td char=".">647 </td><td char=".">3 </td><td char=".">507 </td><td char=".">15 </td><td char=".">5,446 </td><td char=".">3 </td><td char=".">865 </td><td char=".">4 </td><td char=".">2,123 </td><td char=".">1 </td><td char=".">808 </td><td>poly-TG </td><td char=".">3 </td></tr><tr><td>1,000–10,000 </td><td char=".">1 </td><td char=".">1,019 </td><td char=".">0 </td><td char=".">0 </td><td char=".">3 </td><td char=".">8,294 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">7,526 </td><td char=".">1 </td><td char=".">1,018 </td><td>poly-AG and poly-TC </td><td char=".">1 each </td></tr><tr><td>Total </td><td char=".">10,248 </td><td char=".">30,641 </td><td char=".">426 </td><td char=".">3,842 </td><td char=".">22 </td><td char=".">13,972 </td><td char=".">8 </td><td char=".">1,102 </td><td char=".">8 </td><td char=".">9,734 </td><td char=".">3 </td><td char=".">1,923 </td><td>Total </td><td char=".">6,164 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="dsw022-TF4"><div class="fn"><p><sup>a</sup> Length of expansion considered is between 2 and 10 bp. </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87199729" target="_blank">View Large</a></div></div>                            <h2 scrollto-destination=87199730 id="87199730" class="section-title" >4. Discussion</h2>
<p> In malaria-endemic areas, <em>P. falciparum</em> parasite genomes are constantly in flux. <a reveal-id="dsw022-B16" data-open="dsw022-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsw022-B53" data-open="dsw022-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> However, our knowledge of <em>P. falciparum</em> genomic variability, which is built on PCR-based genotyping, microarray analyses and short-read NGS, is largely restricted to SNPs. <a reveal-id="dsw022-B16" data-open="dsw022-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> To characterize other genomic variants, particularly those that occur in multigene virulence families and regulatory regions characterized by low sequence complexity such as subtelomeric repeats, there is an urgent need to apply new methods to analyse the <em>P. falciparum</em> genome. Therefore, we adapted the amplification-free, single molecule, long-read sequencing technology developed by Pacific Biosciences to analyse <em>P. falciparum</em> genomic DNA and compiled a new HGAP-derived 23.6 Mb genomic assembly that comprised 14 contiguous end-to-end chromosome sequences. In particular, we accurately resolved repetitive and polymorphic chromosome ends and identified large duplication and insertion events in subtelomeric regions and intra-chromosomal regions (adjacent to coding sequences) with high confidence, suggesting a greater complexity of the <em>P. falciparum</em> genome. In addition, we identified smaller insertions, deletions and tandem expansions in coding and non-coding regions of the genome, all of which can be used to correct and complete the <em>P. falciparum</em> reference genome <a reveal-id="dsw022-B8" data-open="dsw022-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> . It is noteworthy that these variants would have been missed if we had directly aligned the SMRT reads to the reference genome, thereby strongly supporting a hypothesis-free <em>de novo</em> assembly approach, not bound by reference bias. </p><p> In the past five yrs, the adaptation of NGS to study <em>P. falciparum</em> clinical isolates in Africa and South-East Asia has provided genome-wide estimates of allele frequency distribution, population structure and linkage disequilibrium in malaria-endemic regions. <a reveal-id="dsw022-B18" data-open="dsw022-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a><sup>,</sup><a reveal-id="dsw022-B20" data-open="dsw022-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> In addition, NGS analysis of the genetic architecture of drug-resistant subpopulations of <em>P. falciparum</em> in Cambodia has identified SNPs that show high levels of differentiation, and other genomic correlates of the spread of resistance. <a reveal-id="dsw022-B19" data-open="dsw022-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> Complementarily, longitudinal studies that measured the genomic plasticity of laboratory-adapted <em>P. falciparum</em> parasites have determined the rates at which <em>var</em> gene polymorphisms are generated during intraerythrocytic development. <a reveal-id="dsw022-B3" data-open="dsw022-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsw022-B4" data-open="dsw022-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> However, all of these studies used PCR-based WGA combined with short-read NGS and relied on the alignment-based mapping of sequencing reads to the reference 3D7 genome <a reveal-id="dsw022-B8" data-open="dsw022-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> to assess genetic diversity. While this method will prove invaluable to analysing SNPs in clinical settings where researchers mostly dispose of low sample volumes, we believe that the read length limitations of these approaches will make the identification of other critical structural variants extremely difficult. Given the lack of sequence bias in SMRT sequencing, <a reveal-id="dsw022-B24" data-open="dsw022-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> length of SMRT reads (&gt;11 kb; compared to the ∼8 kb size of a <em>var</em> ORF) and our complete resolution of chromosomal ends and copy number variants of virulence genes such as <em>var</em> and <em>rifin</em> , we argue that long-read sequencing will provide complete reference-grade genomes for <em>P. falciparum</em> field isolates. Such analyses, in combination with other approaches to document SNP variation, will comprehensively determine the geographical diversity of <em>P. falciparum</em> populations and monitor longitudinal population changes of both <em>P. falciparum</em> and other human malaria species. </p><p> We recognize that one limitation of our whole genome sequencing approach may be the amount of starting material (∼10 μg) that is necessary to prepare high quality, size-selected SMRT sequencing libraries for <em>de novo</em> assembly. For example, if a malaria patient presents with a parasitaemia of &gt;3% rings, around 4 ml of blood will be required to yield ∼10 μg of <em>P. falciparum</em> genomic DNA, after eliminating human genomic DNA contamination. <a reveal-id="dsw022-B54" data-open="dsw022-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> In contrast, for cases with lower parasitaemia, ∼200–500 μl of patient blood will have to be frozen and recultured with fresh WBC-free blood for one or two generations, or until the cultures reach a parasitaemia of 2% schizonts, before being processed for DNA extraction. Nonetheless, because several studies have shown that clinical isolates can be cultured <em>in vitro</em> from frozen stocks for short periods of time without losing their multiplication and erythrocyte invasion properties, <a reveal-id="dsw022-B55" data-open="dsw022-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a><sup>,</sup><a reveal-id="dsw022-B56" data-open="dsw022-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> we believe that the latter option is feasible to compile SMRT sequencing-based whole genomes of <em>P. falciparum</em> clinical isolates. Furthermore, as sequencing technologies progress and develop, we anticipate that system requirements will change rapidly, consequently leading to a reduction in initial sample input, cost and time to project completion. </p><p> In conclusion, our study emphasizes the value of long-read sequencing technologies and <em>de novo</em> genome assembly to fully resolve pathogen genome architecture and complexity, paving the way for comprehensively assessing genetic variation at all size scales. In the specific case of <em>Plasmodium</em> , not only will it provide information about within-host <em>P. falciparum</em> diversity, but it will fill existing gaps in the genome sequences of laboratory-adapted strains as well as of field isolates. Furthermore, the resolution and understanding of <em>P. falciparum</em> structural variants, in particular of multigene families involved in adhesion and immune evasion (Ex: PfEMP1 and PfRifin), and genes involved in erythrocyte invasion (vaccine candidates such as PfAMA1 and PfRh proteins) and sexual stage biology (vaccine candidates such as P48/45, P25 and P28), will provide genetic correlates of parasite virulence and transmissibility. Given the recent assembly of the haploid human genome using SMRT sequencing, <a reveal-id="dsw022-B28" data-open="dsw022-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> we propose that long-read sequencing technologies will be crucial to combining <em>Plasmodium</em> genomic epidemiology with human population genetics to obtain a comprehensive view of parasite behaviour, genetic predisposition of humans to malaria and the co-evolution of parasite and host at a molecular level. Finally, SMRT technology can be used to identify epigenomic variations (Ex: genome-wide DNA methylation patterns) based on the kinetics of the DNA synthesis reaction performed by the polymerase, concurrent with sequencing. <a reveal-id="dsw022-B57" data-open="dsw022-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> Because this method is being utilized to examine both bacterial and eukaryotic epigenomes <a reveal-id="dsw022-B58 dsw022-B59 dsw022-B60" data-open="dsw022-B58 dsw022-B59 dsw022-B60" class="link link-ref link-reveal xref-bibr"><sup>58–60</sup></a> and given the tractable size of the <em>P. falciparum</em> genome, future work could investigate if dynamic changes in DNA methylation <a reveal-id="dsw022-B61" data-open="dsw022-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> of <em>P. falciparum</em> clinical isolates are linked to varying degrees of severe malaria. </p>                            <h2 scrollto-destination=87199735 id="87199735" class="backacknowledgements-title" >Acknowledgements</h2>
<p> We are grateful to Cameron Macpherson and Sebastian Baumgarten for helpful scientific discussions and Jonas Korlach for critical reading of the manuscript. We also acknowledge PlasmoDB, a member of pathogen-databases that are housed under the NIAID-funded EuPathDB Bioinformatics Resource Center (BRC) umbrella, for regular updates and ready access to <em>P. falciparum</em> genome information. </p>                            <h2 scrollto-destination=87199737 id="87199737" class="backsection-title" >Conflict of interest</h2>
<p>C.L., M.S., P.B. and M.L.S. are full-time employees at Pacific Biosciences, a company that commercializes single molecule, real-time sequencing technologies.</p>                            <h2 scrollto-destination=87199739 id="87199739" class="backsection-title" >Funding</h2>
<p>This work was supported by a European Research Council Advanced Grant (ERC AdG PlasmoEscape 250320) and the French Parasitology consortium ParaFrap (ANR-11-LABX0024) to A. S. S.S.V. was supported by the European Molecular Biology Organization Long-Term Fellowship, the Marie Sklodowska-Curie International Incoming Fellowship (FP7-MC-IIF-302451) and the Institut Pasteur Bourse Roux post-doctoral fellowship. Funding for open access charge is provided by the grant ERC 2014 AdG PlasmoSilencing awarded to A.S.</p>                            <h2 scrollto-destination=87199741 id="87199741" class="backsection-title" >Availability of Supporting Data</h2>
<p> The raw data generated in this study have been deposited in the National Centre for Biotechnology Information (NCBI) Sequence Read Archive under the accession number SRA360189 ( <a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/sra/?term=SRA360189" target="_blank">http://www.ncbi.nlm.nih.gov/sra/?term=SRA360189</a> ; BioProject accession number = PRJNA313199 and BioSample accession number = SRS1315102 last accessed on June 8, 2016). The SMRT assembly has been deposited in the European Nucleotide Archive under the study accession number PRJEB11803 ( <a class="link link-uri" href="http://www.ebi.ac.uk/ena/data/view/PRJEB11803" target="_blank">http://www.ebi.ac.uk/ena/data/view/PRJEB11803 last accessed on June 8, 2016</a> ). Note that in the fastq file for the final assembly, the term Unitig is used instead of Contig. </p>                            <h2 scrollto-destination=87199743 id="87199743" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw022-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Miller</div>
               <div class="given-names">L.H.</div>
            </div>
             <div class="name">
              <div class="surname">Ackerman</div>
               <div class="given-names">H.C.</div>
            </div>
             <div class="name">
              <div class="surname">Su</div>
               <div class="given-names">X.Z.</div>
            </div>
             <div class="name">
              <div class="surname">Wellems</div>
               <div class="given-names">T.E.</div>
            </div>
          </div>
          <div class="year">2013</div>
            , 
            <div class="article-title">Malaria biology and disease pathogenesis: insights for new treatments,</div><div class="source">Nat. Med</div> 
            ., 
            <div class="volume">19</div>
            , 
            <div class="fpage">156</div>
            –
            <div class="lpage">67</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Malaria%20biology%20and%20disease%20pathogenesis%3A%20insights%20for%20new%20treatments%2C&amp;author=L.H.%20Miller&amp;author=H.C.%20Ackerman&amp;author=X.Z.%20Su&amp;author=T.E.%20Wellems&amp;publication_year=2013&amp;journal=Nat.%20Med&amp;volume=19&amp;pages=156-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nm.3073" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnm.3073" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnm.3073"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23389616" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Su</div>
               <div class="given-names">X.</div>
            </div>
             <div class="name">
              <div class="surname">Hayton</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Wellems</div>
               <div class="given-names">T.E.</div>
            </div>
          </div>
          <div class="year">2007</div>
            , 
            <div class="article-title">
              Genetic linkage and association analyses for trait mapping in 
              <em>Plasmodium falciparum</em>
              ,
            </div><div class="source">Nat. Rev. Genet</div> 
            ., 
            <div class="volume">8</div>
            , 
            <div class="fpage">497</div>
            –
            <div class="lpage">506</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20linkage%20and%20association%20analyses%20for%20trait%20mapping%20in%20Plasmodium%20falciparum%20%2C&amp;author=X.%20Su&amp;author=K.%20Hayton&amp;author=T.E.%20Wellems&amp;publication_year=2007&amp;journal=Nat.%20Rev.%20Genet&amp;volume=8&amp;pages=497-506" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2126" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2126" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2126"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17572690" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Bopp</div>
               <div class="given-names">S.E.</div>
            </div>
             <div class="name">
              <div class="surname">Manary</div>
               <div class="given-names">M.J.</div>
            </div>
             <div class="name">
              <div class="surname">Bright</div>
               <div class="given-names">A.T.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2013</div>
            , 
            <div class="article-title">
              Mitotic evolution of 
              <em>Plasmodium falciparum</em>
               shows a stable core genome but recombination in antigen families,
            </div><div class="source">PLoS Genet</div> 
            ., 
            <div class="volume">9</div>
            , 
            <div class="fpage">e1003293</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mitotic%20evolution%20of%20Plasmodium%20falciparum%20shows%20a%20stable%20core%20genome%20but%20recombination%20in%20antigen%20families%2C&amp;author=S.E.%20Bopp&amp;author=M.J.%20Manary&amp;author=A.T.%20Bright&amp;publication_year=2013&amp;journal=PLoS%20Genet&amp;volume=9&amp;pages=e1003293" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1003293" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1003293" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1003293"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23408914" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Claessens</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Hamilton</div>
               <div class="given-names">W.L.</div>
            </div>
             <div class="name">
              <div class="surname">Kekre</div>
               <div class="given-names">M.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2014</div>
            , 
            <div class="article-title">
              Generation of antigenic diversity in 
              <em>Plasmodium falciparum</em>
               by structured rearrangement of Var genes during mitosis,
            </div><div class="source">PLoS Genet</div> 
            ., 
            <div class="volume">10</div>
            , 
            <div class="fpage">e1004812</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Generation%20of%20antigenic%20diversity%20in%20Plasmodium%20falciparum%20by%20structured%20rearrangement%20of%20Var%20genes%20during%20mitosis%2C&amp;author=A.%20Claessens&amp;author=W.L.%20Hamilton&amp;author=M.%20Kekre&amp;publication_year=2014&amp;journal=PLoS%20Genet&amp;volume=10&amp;pages=e1004812" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1004812" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1004812" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1004812"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25521112" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Freitas-Junior</div>
               <div class="given-names">L.H.</div>
            </div>
             <div class="name">
              <div class="surname">Bottius</div>
               <div class="given-names">E.</div>
            </div>
             <div class="name">
              <div class="surname">Pirrit</div>
               <div class="given-names">L.A.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2000</div>
            , 
            <div class="article-title">
              Frequent ectopic recombination of virulence factor genes in telomeric chromosome clusters of 
              <em>P. falciparum</em>
              ,
            </div><div class="source">Nature</div> 
            , 
            <div class="volume">407</div>
            , 
            <div class="fpage">1018</div>
            –
            <div class="lpage">22</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Frequent%20ectopic%20recombination%20of%20virulence%20factor%20genes%20in%20telomeric%20chromosome%20clusters%20of%20P.%20falciparum%20%2C&amp;author=L.H.%20Freitas-Junior&amp;author=E.%20Bottius&amp;author=L.A.%20Pirrit&amp;publication_year=2000&amp;journal=Nature&amp;volume=407&amp;pages=1018-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35039531" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35039531" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35039531"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11069183" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Ariey</div>
               <div class="given-names">F.</div>
            </div>
             <div class="name">
              <div class="surname">Witkowski</div>
               <div class="given-names">B.</div>
            </div>
             <div class="name">
              <div class="surname">Amaratunga</div>
               <div class="given-names">C.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2014</div>
            , 
            <div class="article-title">
              A molecular marker of artemisinin-resistant 
              <em>Plasmodium falciparum</em>
               malaria,
            </div><div class="source">Nature</div> 
            , 
            <div class="volume">505</div>
            , 
            <div class="fpage">50</div>
            –
            <div class="lpage">5</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20molecular%20marker%20of%20artemisinin-resistant%20Plasmodium%20falciparum%20malaria%2C&amp;author=F.%20Ariey&amp;author=B.%20Witkowski&amp;author=C.%20Amaratunga&amp;publication_year=2014&amp;journal=Nature&amp;volume=505&amp;pages=50-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature12876" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature12876" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature12876"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24352242" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Rottmann</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">McNamara</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Yeung</div>
               <div class="given-names">B.K.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2010</div>
            , 
            <div class="article-title">Spiroindolones, a potent compound class for the treatment of malaria,</div><div class="source">Science</div> 
            , 
            <div class="volume">329</div>
            , 
            <div class="fpage">1175</div>
            –
            <div class="lpage">80</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Spiroindolones%2C%20a%20potent%20compound%20class%20for%20the%20treatment%20of%20malaria%2C&amp;author=M.%20Rottmann&amp;author=C.%20McNamara&amp;author=B.K.%20Yeung&amp;publication_year=2010&amp;journal=Science&amp;volume=329&amp;pages=1175-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1193225" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1193225" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1193225"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20813948" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Gardner</div>
               <div class="given-names">M.J.</div>
            </div>
             <div class="name">
              <div class="surname">Hall</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Fung</div>
               <div class="given-names">E.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2002</div>
            , 
            <div class="article-title">
              Genome sequence of the human malaria parasite 
              <em>Plasmodium falciparum</em>
              ,
            </div><div class="source">Nature</div> 
            , 
            <div class="volume">419</div>
            , 
            <div class="fpage">498</div>
            –
            <div class="lpage">511</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20human%20malaria%20parasite%20Plasmodium%20falciparum%20%2C&amp;author=M.J.%20Gardner&amp;author=N.%20Hall&amp;author=E.%20Fung&amp;publication_year=2002&amp;journal=Nature&amp;volume=419&amp;pages=498-511" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature01097" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature01097" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature01097"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12368864" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Su</div>
               <div class="given-names">X.Z.</div>
            </div>
             <div class="name">
              <div class="surname">Heatwole</div>
               <div class="given-names">V.M.</div>
            </div>
             <div class="name">
              <div class="surname">Wertheimer</div>
               <div class="given-names">S.P.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">1995</div>
            , 
            <div class="article-title">
              The large diverse gene family var encodes proteins involved in cytoadherence and antigenic variation of 
              <em>Plasmodium falciparum</em>
              -infected erythrocytes,
            </div><div class="source">Cell</div> 
            , 
            <div class="volume">82</div>
            , 
            <div class="fpage">89</div>
            –
            <div class="lpage">100</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20large%20diverse%20gene%20family%20var%20encodes%20proteins%20involved%20in%20cytoadherence%20and%20antigenic%20variation%20of%20Plasmodium%20falciparum%20-infected%20erythrocytes%2C&amp;author=X.Z.%20Su&amp;author=V.M.%20Heatwole&amp;author=S.P.%20Wertheimer&amp;publication_year=1995&amp;journal=Cell&amp;volume=82&amp;pages=89-100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0092-8674(95)90055-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0092-8674(95)90055-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0092-8674(95)90055-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7606788" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Oyola</div>
               <div class="given-names">S.O.</div>
            </div>
             <div class="name">
              <div class="surname">Otto</div>
               <div class="given-names">T.D.</div>
            </div>
             <div class="name">
              <div class="surname">Gu</div>
               <div class="given-names">Y.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2012</div>
            , 
            <div class="article-title">Optimizing illumina next-generation sequencing library preparation for extremely AT-biased genomes,</div><div class="source">BMC Genomics</div> 
            , 
            <div class="volume">13</div>
            , 
            <div class="fpage">1</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Optimizing%20illumina%20next-generation%20sequencing%20library%20preparation%20for%20extremely%20AT-biased%20genomes%2C&amp;author=S.O.%20Oyola&amp;author=T.D.%20Otto&amp;author=Y.%20Gu&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22214261" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Oyola</div>
               <div class="given-names">S.O.</div>
            </div>
             <div class="name">
              <div class="surname">Manske</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Campino</div>
               <div class="given-names">S.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2014</div>
            , 
            <div class="article-title">Optimized whole-genome amplification strategy for extremely AT-biased template,</div><div class="source">DNA Res</div> 
            ., 
            <div class="volume">21</div>
            , 
            <div class="fpage">661</div>
            –
            <div class="lpage">71</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Optimized%20whole-genome%20amplification%20strategy%20for%20extremely%20AT-biased%20template%2C&amp;author=S.O.%20Oyola&amp;author=M.%20Manske&amp;author=S.%20Campino&amp;publication_year=2014&amp;journal=DNA%20Res&amp;volume=21&amp;pages=661-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsu028" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsu028" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsu028"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25240466" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Quail</div>
               <div class="given-names">M.A.</div>
            </div>
             <div class="name">
              <div class="surname">Otto</div>
               <div class="given-names">T.D.</div>
            </div>
             <div class="name">
              <div class="surname">Gu</div>
               <div class="given-names">Y.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2012</div>
            , 
            <div class="article-title">Optimal enzymes for amplifying sequencing libraries,</div><div class="source">Nat. Methods</div> 
            , 
            <div class="volume">9</div>
            , 
            <div class="fpage">10</div>
            –
            <div class="lpage">11</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Optimal%20enzymes%20for%20amplifying%20sequencing%20libraries%2C&amp;author=M.A.%20Quail&amp;author=T.D.%20Otto&amp;author=Y.%20Gu&amp;publication_year=2012&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=10-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1814" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1814" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1814"> </span></div> </div></div></div></div></div><div content-id="dsw022-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Quail</div>
               <div class="given-names">M.A.</div>
            </div>
             <div class="name">
              <div class="surname">Smith</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Coupland</div>
               <div class="given-names">P.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2012</div>
            , 
            <div class="article-title">A tale of three next generation sequencing platforms: comparison of Ion Torrent, Pacific Biosciences and Illumina MiSeq sequencers,</div><div class="source">BMC Genomics</div> 
            , 
            <div class="volume">13</div>
            , 
            <div class="fpage">341</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20tale%20of%20three%20next%20generation%20sequencing%20platforms%3A%20comparison%20of%20Ion%20Torrent%2C%20Pacific%20Biosciences%20and%20Illumina%20MiSeq%20sequencers%2C&amp;author=M.A.%20Quail&amp;author=M.%20Smith&amp;author=P.%20Coupland&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=341" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-341" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-341" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-341"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22827831" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Ross</div>
               <div class="given-names">M.G.</div>
            </div>
             <div class="name">
              <div class="surname">Russ</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Costello</div>
               <div class="given-names">M.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2013</div>
            , 
            <div class="article-title">Characterizing and measuring bias in sequence data,</div><div class="source">Genome Biol</div> 
            ., 
            <div class="volume">14</div>
            , 
            <div class="fpage">R51</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterizing%20and%20measuring%20bias%20in%20sequence%20data%2C&amp;author=M.G.%20Ross&amp;author=C.%20Russ&amp;author=M.%20Costello&amp;publication_year=2013&amp;journal=Genome%20Biol&amp;volume=14&amp;pages=R51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2013-14-5-r51" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2013-14-5-r51" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2013-14-5-r51"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23718773" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Carlton</div>
               <div class="given-names">J.M.</div>
            </div>
             <div class="name">
              <div class="surname">Sullivan</div>
               <div class="given-names">S.A.</div>
            </div>
             <div class="name">
              <div class="surname">Le Roch</div>
               <div class="given-names">K.G.</div>
            </div>
             <div class="name">
              <div class="surname">Carlton</div>
               <div class="given-names">J.M.</div>
            </div>
             <div class="name">
              <div class="surname">Perkins</div>
               <div class="given-names">S.L.</div>
            </div>
             <div class="name">
              <div class="surname">Deitsch</div>
               <div class="given-names">K.W.</div>
            </div>
          </div>
          <div class="year">2013</div>
            , 
            Plasmodium genomics and the art of sequencing malaria parasite genomes
            . In: 
            <div class="person-group"><div class="name"><div class="surname">Carlton</div> <div class="given-names">J.M.</div></div> <div class="name"><div class="surname">Perkins</div> <div class="given-names">S.L.</div></div> <div class="name"><div class="surname">Deitsch</div> <div class="given-names">K.W.</div></div></div>
             (eds.), 
            <div class="source">Malaria parasites: comparative genomics, evolution, and molecular biology</div> 
            , 
            <div class="publisher-loc">Norfolk, UK</div>
            : 
            <div class="publisher-name">Caister Academic Press</div>
            , pp. 
            <div class="fpage">35</div>
            –
            <div class="lpage">58</div>
            .
          </div></div></div></div><div content-id="dsw022-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kwiatkowski</div>
               <div class="given-names">D.</div>
            </div>
          </div>
          <div class="year">2015</div>
            , 
            <div class="article-title">Malaria genomics: tracking a diverse and evolving parasite population,</div><div class="source">Int. Health</div> 
            , 
            <div class="volume">7</div>
            , 
            <div class="fpage">82</div>
            –
            <div class="lpage">4</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Malaria%20genomics%3A%20tracking%20a%20diverse%20and%20evolving%20parasite%20population%2C&amp;author=D.%20Kwiatkowski&amp;publication_year=2015&amp;journal=Int.%20Health&amp;volume=7&amp;pages=82-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/inthealth/ihv007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Finthealth%2Fihv007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Finthealth%2Fihv007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25733556" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kamau</div>
               <div class="given-names">E.</div>
            </div>
             <div class="name">
              <div class="surname">Campino</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Amenga-Etego</div>
               <div class="given-names">L.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2015</div>
            , 
            <div class="article-title">
              K13-propeller polymorphisms in 
              <em>Plasmodium falciparum</em>
               parasites from sub-Saharan Africa
            </div>
            , 
            <div class="source">J. Infect. Dis</div> 
            ., 
            <div class="volume">211</div>
            , 
            <div class="fpage">1352</div>
            –
            <div class="lpage">5</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=K13-propeller%20polymorphisms%20in%20Plasmodium%20falciparum%20parasites%20from%20sub-Saharan%20Africa&amp;author=E.%20Kamau&amp;author=S.%20Campino&amp;author=L.%20Amenga-Etego&amp;publication_year=2015&amp;journal=J.%20Infect.%20Dis&amp;volume=211&amp;pages=1352-5" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25367300" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Manske</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Miotto</div>
               <div class="given-names">O.</div>
            </div>
             <div class="name">
              <div class="surname">Campino</div>
               <div class="given-names">S.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2012</div>
            , 
            <div class="article-title">
              Analysis of 
              <em>Plasmodium falciparum</em>
               diversity in natural infections by deep sequencing,
            </div><div class="source">Nature</div> 
            , 
            <div class="volume">487</div>
            , 
            <div class="fpage">375</div>
            –
            <div class="lpage">9</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20Plasmodium%20falciparum%20diversity%20in%20natural%20infections%20by%20deep%20sequencing%2C&amp;author=M.%20Manske&amp;author=O.%20Miotto&amp;author=S.%20Campino&amp;publication_year=2012&amp;journal=Nature&amp;volume=487&amp;pages=375-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11174" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11174" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11174"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22722859" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Miotto</div>
               <div class="given-names">O.</div>
            </div>
             <div class="name">
              <div class="surname">Almagro-Garcia</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Manske</div>
               <div class="given-names">M.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2013</div>
            , 
            <div class="article-title">
              Multiple populations of artemisinin-resistant 
              <em>Plasmodium falciparum</em>
               in Cambodia,
            </div><div class="source">Nat. Genet</div> 
            ., 
            <div class="volume">45</div>
            , 
            <div class="fpage">648</div>
            –
            <div class="lpage">55</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20populations%20of%20artemisinin-resistant%20Plasmodium%20falciparum%20in%20Cambodia%2C&amp;author=O.%20Miotto&amp;author=J.%20Almagro-Garcia&amp;author=M.%20Manske&amp;publication_year=2013&amp;journal=Nat.%20Genet&amp;volume=45&amp;pages=648-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2624" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2624" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2624"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23624527" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Miotto</div>
               <div class="given-names">O.</div>
            </div>
             <div class="name">
              <div class="surname">Amato</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Ashley</div>
               <div class="given-names">E.A.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2015</div>
            , 
            <div class="article-title">
              Genetic architecture of artemisinin-resistant 
              <em>Plasmodium falciparum</em>
              ,
            </div><div class="source">Nat. Genet</div> 
            ., 
            <div class="volume">47</div>
            , 
            <div class="fpage">226</div>
            –
            <div class="lpage">34</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20architecture%20of%20artemisinin-resistant%20Plasmodium%20falciparum%20%2C&amp;author=O.%20Miotto&amp;author=R.%20Amato&amp;author=E.A.%20Ashley&amp;publication_year=2015&amp;journal=Nat.%20Genet&amp;volume=47&amp;pages=226-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.3189" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.3189" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.3189"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25599401" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Mobegi</div>
               <div class="given-names">V.A.</div>
            </div>
             <div class="name">
              <div class="surname">Duffy</div>
               <div class="given-names">C.W.</div>
            </div>
             <div class="name">
              <div class="surname">Amambua-Ngwa</div>
               <div class="given-names">A.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2014</div>
            , 
            <div class="article-title">
              Genome-wide analysis of selection on the malaria parasite 
              <em>Plasmodium falciparum</em>
               in West African populations of differing infection endemicity,
            </div><div class="source">Mol. Biol. Evol</div> 
            ., 
            <div class="volume">31</div>
            , 
            <div class="fpage">1490</div>
            –
            <div class="lpage">9</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20selection%20on%20the%20malaria%20parasite%20Plasmodium%20falciparum%20in%20West%20African%20populations%20of%20differing%20infection%20endemicity%2C&amp;author=V.A.%20Mobegi&amp;author=C.W.%20Duffy&amp;author=A.%20Amambua-Ngwa&amp;publication_year=2014&amp;journal=Mol.%20Biol.%20Evol&amp;volume=31&amp;pages=1490-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msu106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsu106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsu106"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24644299" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Nair</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Nkhoma</div>
               <div class="given-names">S.C.</div>
            </div>
             <div class="name">
              <div class="surname">Serre</div>
               <div class="given-names">D.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2014</div>
            , 
            <div class="article-title">Single-cell genomics for dissection of complex malaria infections,</div><div class="source">Genome Res</div> 
            ., 
            <div class="volume">24</div>
            , 
            <div class="fpage">1028</div>
            –
            <div class="lpage">38</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single-cell%20genomics%20for%20dissection%20of%20complex%20malaria%20infections%2C&amp;author=S.%20Nair&amp;author=S.C.%20Nkhoma&amp;author=D.%20Serre&amp;publication_year=2014&amp;journal=Genome%20Res&amp;volume=24&amp;pages=1028-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.168286.113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.168286.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.168286.113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24812326" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Beerenwinkel</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Günthard</div>
               <div class="given-names">H.F.</div>
            </div>
             <div class="name">
              <div class="surname">Roth</div>
               <div class="given-names">V.</div>
            </div>
             <div class="name">
              <div class="surname">Metzner</div>
               <div class="given-names">K.J.</div>
            </div>
          </div>
          <div class="year">2012</div>
            , 
            <div class="article-title">Challenges and opportunities in estimating viral genetic diversity from next-generation sequencing data,</div><div class="source">Front. Microbiol</div> 
            ., 
            <div class="volume">3</div>
            , 
            <div class="fpage">329</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Challenges%20and%20opportunities%20in%20estimating%20viral%20genetic%20diversity%20from%20next-generation%20sequencing%20data%2C&amp;author=N.%20Beerenwinkel&amp;author=H.F.%20G%C3%BCnthard&amp;author=V.%20Roth&amp;author=K.J.%20Metzner&amp;publication_year=2012&amp;journal=Front.%20Microbiol&amp;volume=3&amp;pages=329" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fmicb.2012.00329" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffmicb.2012.00329" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffmicb.2012.00329"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22973268" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Eid</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Fehr</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Gray</div>
               <div class="given-names">J.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2009</div>
            , 
            <div class="article-title">Real-time DNA sequencing from single polymerase molecules,</div><div class="source">Science</div> 
            , 
            <div class="volume">323</div>
            , 
            <div class="fpage">133</div>
            –
            <div class="lpage">8</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Real-time%20DNA%20sequencing%20from%20single%20polymerase%20molecules%2C&amp;author=J.%20Eid&amp;author=A.%20Fehr&amp;author=J.%20Gray&amp;publication_year=2009&amp;journal=Science&amp;volume=323&amp;pages=133-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1162986" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1162986" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1162986"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19023044" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Chaisson</div>
               <div class="given-names">M.J.</div>
            </div>
             <div class="name">
              <div class="surname">Wilson</div>
               <div class="given-names">R.K.</div>
            </div>
             <div class="name">
              <div class="surname">Eichler</div>
               <div class="given-names">E.E.</div>
            </div>
          </div>
          <div class="year">2015</div>
            , 
            <div class="article-title">
              Genetic variation and the 
              <em>de novo</em>
               assembly of human genomes,
            </div><div class="source">Nat. Rev. Genet</div> 
            ., 
            <div class="volume">16</div>
            , 
            <div class="fpage">627</div>
            –
            <div class="lpage">40</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20variation%20and%20the%20de%20novo%20assembly%20of%20human%20genomes%2C&amp;author=M.J.%20Chaisson&amp;author=R.K.%20Wilson&amp;author=E.E.%20Eichler&amp;publication_year=2015&amp;journal=Nat.%20Rev.%20Genet&amp;volume=16&amp;pages=627-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3933" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3933" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3933"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26442640" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Dilernia</div>
               <div class="given-names">D.A.</div>
            </div>
             <div class="name">
              <div class="surname">Chien</div>
               <div class="given-names">J.T.</div>
            </div>
             <div class="name">
              <div class="surname">Monaco</div>
               <div class="given-names">D.C.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2015</div>
            , 
            <div class="article-title">Multiplexed highly-accurate DNA sequencing of closely-related HIV-1 variants using continuous long reads from single molecule, real-time sequencing</div>
            , 
            <div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">43</div>
            , 
            <div class="fpage">e129</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiplexed%20highly-accurate%20DNA%20sequencing%20of%20closely-related%20HIV-1%20variants%20using%20continuous%20long%20reads%20from%20single%20molecule%2C%20real-time%20sequencing&amp;author=D.A.%20Dilernia&amp;author=J.T.%20Chien&amp;author=D.C.%20Monaco&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res&amp;volume=43&amp;pages=e129" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkv630" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkv630" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkv630"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26101252" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">English</div>
               <div class="given-names">A.C.</div>
            </div>
             <div class="name">
              <div class="surname">Salerno</div>
               <div class="given-names">W.J.</div>
            </div>
             <div class="name">
              <div class="surname">Hampton</div>
               <div class="given-names">O.A.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2015</div>
            , 
            <div class="article-title">Assessing structural variation in a personal genome-towards a human reference diploid genome,</div><div class="source">BMC Genomics</div> 
            , 
            <div class="volume">16</div>
            , 
            <div class="fpage">286</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessing%20structural%20variation%20in%20a%20personal%20genome-towards%20a%20human%20reference%20diploid%20genome%2C&amp;author=A.C.%20English&amp;author=W.J.%20Salerno&amp;author=O.A.%20Hampton&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=286" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-1479-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-1479-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-1479-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25886820" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Chaisson</div>
               <div class="given-names">M.J.</div>
            </div>
             <div class="name">
              <div class="surname">Huddleston</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Dennis</div>
               <div class="given-names">M.Y.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2015</div>
            , 
            <div class="article-title">Resolving the complexity of the human genome using single-molecule sequencing,</div><div class="source">Nature</div> 
            , 
            <div class="volume">517</div>
            , 
            <div class="fpage">608</div>
            –
            <div class="lpage">11</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Resolving%20the%20complexity%20of%20the%20human%20genome%20using%20single-molecule%20sequencing%2C&amp;author=M.J.%20Chaisson&amp;author=J.%20Huddleston&amp;author=M.Y.%20Dennis&amp;publication_year=2015&amp;journal=Nature&amp;volume=517&amp;pages=608-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature13907" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature13907" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature13907"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25383537" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Chin</div>
               <div class="given-names">C.S.</div>
            </div>
             <div class="name">
              <div class="surname">Alexander</div>
               <div class="given-names">D.H.</div>
            </div>
             <div class="name">
              <div class="surname">Marks</div>
               <div class="given-names">P.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2013</div>
            , 
            <div class="article-title">Nonhybrid, finished microbial genome assemblies from long-read SMRT sequencing data,</div><div class="source">Nat. Methods</div> 
            , 
            <div class="volume">10</div>
            , 
            <div class="fpage">563</div>
            –
            <div class="lpage">9</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nonhybrid%2C%20finished%20microbial%20genome%20assemblies%20from%20long-read%20SMRT%20sequencing%20data%2C&amp;author=C.S.%20Chin&amp;author=D.H.%20Alexander&amp;author=P.%20Marks&amp;publication_year=2013&amp;journal=Nat.%20Methods&amp;volume=10&amp;pages=563-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.2474" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.2474" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.2474"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23644548" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Koren</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Schatz</div>
               <div class="given-names">M.C.</div>
            </div>
             <div class="name">
              <div class="surname">Walenz</div>
               <div class="given-names">B.P.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2012</div>
            , 
            <div class="article-title">
              Hybrid error correction and 
              <em>de novo</em>
               assembly of single-molecule sequencing reads
            </div>
            , 
            <div class="source">Nat. Biotechnol</div> 
            ., 
            <div class="volume">30</div>
            , 
            <div class="fpage">693</div>
            –
            <div class="lpage">700</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hybrid%20error%20correction%20and%20de%20novo%20assembly%20of%20single-molecule%20sequencing%20reads&amp;author=S.%20Koren&amp;author=M.C.%20Schatz&amp;author=B.P.%20Walenz&amp;publication_year=2012&amp;journal=Nat.%20Biotechnol&amp;volume=30&amp;pages=693-700" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2280" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2280" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2280"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22750884" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Carneiro</div>
               <div class="given-names">M.O.</div>
            </div>
             <div class="name">
              <div class="surname">Russ</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Ross</div>
               <div class="given-names">M.G.</div>
            </div>
             <div class="name">
              <div class="surname">Gabriel</div>
               <div class="given-names">S.B.</div>
            </div>
             <div class="name">
              <div class="surname">Nusbaum</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">DePristo</div>
               <div class="given-names">M.A.</div>
            </div>
          </div>
          <div class="year">2012</div>
            , 
            <div class="article-title">Pacific biosciences sequencing technology for genotyping and variation discovery in human data,</div><div class="source">BMC Genomics</div> 
            , 
            <div class="volume">13</div>
            , 
            <div class="fpage">375</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pacific%20biosciences%20sequencing%20technology%20for%20genotyping%20and%20variation%20discovery%20in%20human%20data%2C&amp;author=M.O.%20Carneiro&amp;author=C.%20Russ&amp;author=M.G.%20Ross&amp;author=S.B.%20Gabriel&amp;author=C.%20Nusbaum&amp;author=M.A.%20DePristo&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=375" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-375" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-375" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-375"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22863213" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Trager</div>
               <div class="given-names">W.</div>
            </div>
             <div class="name">
              <div class="surname">Jensen</div>
               <div class="given-names">J.B.</div>
            </div>
          </div>
          <div class="year">1976</div>
            , 
            <div class="article-title">Human malaria parasites in continuous culture,</div><div class="source">Science</div> 
            , 
            <div class="volume">193</div>
            , 
            <div class="fpage">673</div>
            –
            <div class="lpage">5</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Human%20malaria%20parasites%20in%20continuous%20culture%2C&amp;author=W.%20Trager&amp;author=J.B.%20Jensen&amp;publication_year=1976&amp;journal=Science&amp;volume=193&amp;pages=673-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.781840" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.781840" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.781840"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/781840" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Lelièvre</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Berry</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Benoit-Vical</div>
               <div class="given-names">F.</div>
            </div>
          </div>
          <div class="year">2005</div>
            , 
            <div class="article-title">An alternative method for Plasmodium culture synchronization,</div><div class="source">Exp. Parasitol</div> 
            ., 
            <div class="volume">109</div>
            , 
            <div class="fpage">195</div>
            –
            <div class="lpage">7</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20alternative%20method%20for%20Plasmodium%20culture%20synchronization%2C&amp;author=J.%20Leli%C3%A8vre&amp;author=A.%20Berry&amp;author=F.%20Benoit-Vical&amp;publication_year=2005&amp;journal=Exp.%20Parasitol&amp;volume=109&amp;pages=195-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.exppara.2004.11.012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.exppara.2004.11.012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.exppara.2004.11.012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15713452" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Lambros</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Vanderberg</div>
               <div class="given-names">J.P.</div>
            </div>
          </div>
          <div class="year">1979</div>
            , 
            <div class="article-title">
              Synchronization of 
              <em>Plasmodium falciparum</em>
               erythrocytic stages in culture,
            </div><div class="source">J. Parasitol</div> 
            ., 
            <div class="volume">65</div>
            , 
            <div class="fpage">418</div>
            –
            <div class="lpage">20</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synchronization%20of%20Plasmodium%20falciparum%20erythrocytic%20stages%20in%20culture%2C&amp;author=C.%20Lambros&amp;author=J.P.%20Vanderberg&amp;publication_year=1979&amp;journal=J.%20Parasitol&amp;volume=65&amp;pages=418-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/3280287" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F3280287" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F3280287"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/383936" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">Moll, K., Kaneko, A., Scherf, A., Wahlgren, M. 2013 Methods in Malaria Research, 6th edition.</div></div></div></div><div content-id="dsw022-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Myers</div>
               <div class="given-names">E.W.</div>
            </div>
             <div class="name">
              <div class="surname">Sutton</div>
               <div class="given-names">G.G.</div>
            </div>
             <div class="name">
              <div class="surname">Delcher</div>
               <div class="given-names">A.L.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2000</div>
            , 
            <div class="article-title">A whole-genome assembly of Drosophila,</div><div class="source">Science</div> 
            , 
            <div class="volume">287</div>
            , 
            <div class="fpage">2196</div>
            –
            <div class="lpage">204</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20whole-genome%20assembly%20of%20Drosophila%2C&amp;author=E.W.%20Myers&amp;author=G.G.%20Sutton&amp;author=A.L.%20Delcher&amp;publication_year=2000&amp;journal=Science&amp;volume=287&amp;pages=2196-204" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.287.5461.2196" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.287.5461.2196" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.287.5461.2196"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10731133" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Chaisson</div>
               <div class="given-names">M.J.</div>
            </div>
             <div class="name">
              <div class="surname">Tesler</div>
               <div class="given-names">G.</div>
            </div>
          </div>
          <div class="year">2012</div>
            , 
            <div class="article-title">Mapping single molecule sequencing reads using basic local alignment with successive refinement (BLASR): application and theory,</div><div class="source">BMC Bioinformatics</div> 
            , 
            <div class="volume">13</div>
            , 
            <div class="fpage">238</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20single%20molecule%20sequencing%20reads%20using%20basic%20local%20alignment%20with%20successive%20refinement%20(BLASR)%3A%20application%20and%20theory%2C&amp;author=M.J.%20Chaisson&amp;author=G.%20Tesler&amp;publication_year=2012&amp;journal=BMC%20Bioinformatics&amp;volume=13&amp;pages=238" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-13-238" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-13-238" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-13-238"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22988817" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Krumsiek</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Arnold</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Rattei</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="year">2007</div>
            , 
            <div class="article-title">Gepard: a rapid and sensitive tool for creating dotplots on genome scale,</div><div class="source">Bioinformatics</div> 
            , 
            <div class="volume">23</div>
            , 
            <div class="fpage">1026</div>
            –
            <div class="lpage">8</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gepard%3A%20a%20rapid%20and%20sensitive%20tool%20for%20creating%20dotplots%20on%20genome%20scale%2C&amp;author=J.%20Krumsiek&amp;author=R.%20Arnold&amp;author=T.%20Rattei&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=1026-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm039" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm039" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm039"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17309896" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Nattestad</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Schatz</div>
               <div class="given-names">M.C.</div>
            </div>
          </div>
          <div class="year">2016</div>
            , 
            <div class="article-title">Assemblytics: a web analytics tool for the detection of assembly-based variants,</div><div class="source">bioRxiv</div> 
            . 
            <div class="comment">doi: http://dx.doi.org/10.1101/044925.</div></div></div></div></div><div content-id="dsw022-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kurtz</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Phillippy</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Delcher</div>
               <div class="given-names">A.L.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2004</div>
            , 
            <div class="article-title">Versatile and open software for comparing large genomes,</div><div class="source">Genome Biol</div> 
            ., 
            <div class="volume">5</div>
            , 
            <div class="fpage">R12</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Versatile%20and%20open%20software%20for%20comparing%20large%20genomes%2C&amp;author=S.%20Kurtz&amp;author=A.%20Phillippy&amp;author=A.L.%20Delcher&amp;publication_year=2004&amp;journal=Genome%20Biol&amp;volume=5&amp;pages=R12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2004-5-2-r12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2004-5-2-r12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2004-5-2-r12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14759262" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Quinlan</div>
               <div class="given-names">A.R.</div>
            </div>
             <div class="name">
              <div class="surname">Hall</div>
               <div class="given-names">I.M.</div>
            </div>
          </div>
          <div class="year">2010</div>
            , 
            <div class="article-title">BEDTools: a flexible suite of utilities for comparing genomic features,</div><div class="source">Bioinformatics</div> 
            , 
            <div class="volume">26</div>
            , 
            <div class="fpage">841</div>
            –
            <div class="lpage">2</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BEDTools%3A%20a%20flexible%20suite%20of%20utilities%20for%20comparing%20genomic%20features%2C&amp;author=A.R.%20Quinlan&amp;author=I.M.%20Hall&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=841-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtq033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20110278" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Akane</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Matsubara</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Nakamura</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Takahashi</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Kimura</div>
               <div class="given-names">K.</div>
            </div>
          </div>
          <div class="year">1994</div>
            , 
            <div class="article-title">Identification of the heme compound copurified with deoxyribonucleic acid (DNA) from bloodstains, a major inhibitor of polymerase chain reaction (PCR) amplification,</div><div class="source">J. Forensic Sci</div> 
            ., 
            <div class="volume">39</div>
            , 
            <div class="fpage">362</div>
            –
            <div class="lpage">72</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20the%20heme%20compound%20copurified%20with%20deoxyribonucleic%20acid%20(DNA)%20from%20bloodstains%2C%20a%20major%20inhibitor%20of%20polymerase%20chain%20reaction%20(PCR)%20amplification%2C&amp;author=A.%20Akane&amp;author=K.%20Matsubara&amp;author=H.%20Nakamura&amp;author=S.%20Takahashi&amp;author=K.%20Kimura&amp;publication_year=1994&amp;journal=J.%20Forensic%20Sci&amp;volume=39&amp;pages=362-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1520/JFS13607J" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1520%2FJFS13607J" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1520%2FJFS13607J"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8195750" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kermekchiev</div>
               <div class="given-names">M.B.</div>
            </div>
             <div class="name">
              <div class="surname">Kirilova</div>
               <div class="given-names">L.I.</div>
            </div>
             <div class="name">
              <div class="surname">Vail</div>
               <div class="given-names">E.E.</div>
            </div>
             <div class="name">
              <div class="surname">Barnes</div>
               <div class="given-names">W.M.</div>
            </div>
          </div>
          <div class="year">2009</div>
            , 
            <div class="article-title">Mutants of Taq DNA polymerase resistant to PCR inhibitors allow DNA amplification from whole blood and crude soil samples,</div><div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">37</div>
            , 
            <div class="fpage">e40</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutants%20of%20Taq%20DNA%20polymerase%20resistant%20to%20PCR%20inhibitors%20allow%20DNA%20amplification%20from%20whole%20blood%20and%20crude%20soil%20samples%2C&amp;author=M.B.%20Kermekchiev&amp;author=L.I.%20Kirilova&amp;author=E.E.%20Vail&amp;author=W.M.%20Barnes&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res&amp;volume=37&amp;pages=e40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn1055" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn1055" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn1055"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19208643" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Zhang</div>
               <div class="given-names">Z.</div>
            </div>
             <div class="name">
              <div class="surname">Kermekchiev</div>
               <div class="given-names">M.B.</div>
            </div>
             <div class="name">
              <div class="surname">Barnes</div>
               <div class="given-names">W.M.</div>
            </div>
          </div>
          <div class="year">2010</div>
            , 
            <div class="article-title">Direct DNA amplification from crude clinical samples using a PCR enhancer cocktail and novel mutants of Taq,</div><div class="source">J. Mol. Diagn</div> 
            ., 
            <div class="volume">12</div>
            , 
            <div class="fpage">152</div>
            –
            <div class="lpage">61</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Direct%20DNA%20amplification%20from%20crude%20clinical%20samples%20using%20a%20PCR%20enhancer%20cocktail%20and%20novel%20mutants%20of%20Taq%2C&amp;author=Z.%20Zhang&amp;author=M.B.%20Kermekchiev&amp;author=W.M.%20Barnes&amp;publication_year=2010&amp;journal=J.%20Mol.%20Diagn&amp;volume=12&amp;pages=152-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2353/jmoldx.2010.090070" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2353%2Fjmoldx.2010.090070" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2353%2Fjmoldx.2010.090070"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20075207" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Samarakoon</div>
               <div class="given-names">U.</div>
            </div>
             <div class="name">
              <div class="surname">Regier</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Tan</div>
               <div class="given-names">A.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2011</div>
            , 
            <div class="article-title">
              High-throughput 454 resequencing for allele discovery and recombination mapping in 
              <em>Plasmodium falciparum</em>
              ,
            </div><div class="source">BMC Genomics</div> 
            , 
            <div class="volume">12</div>
            , 
            <div class="fpage">116</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20454%20resequencing%20for%20allele%20discovery%20and%20recombination%20mapping%20in%20Plasmodium%20falciparum%20%2C&amp;author=U.%20Samarakoon&amp;author=A.%20Regier&amp;author=A.%20Tan&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=116" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-116" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-116" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-116"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21324207" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Volkman</div>
               <div class="given-names">S.K.</div>
            </div>
             <div class="name">
              <div class="surname">Sabeti</div>
               <div class="given-names">P.C.</div>
            </div>
             <div class="name">
              <div class="surname">DeCaprio</div>
               <div class="given-names">D.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2007</div>
            , 
            <div class="article-title">
              A genome-wide map of diversity in 
              <em>Plasmodium falciparum</em>
              ,
            </div><div class="source">Nat. Genet</div> 
            ., 
            <div class="volume">39</div>
            , 
            <div class="fpage">113</div>
            –
            <div class="lpage">9</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genome-wide%20map%20of%20diversity%20in%20Plasmodium%20falciparum%20%2C&amp;author=S.K.%20Volkman&amp;author=P.C.%20Sabeti&amp;author=D.%20DeCaprio&amp;publication_year=2007&amp;journal=Nat.%20Genet&amp;volume=39&amp;pages=113-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1930" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1930" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1930"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17159979" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kozarewa</div>
               <div class="given-names">I.</div>
            </div>
             <div class="name">
              <div class="surname">Ning</div>
               <div class="given-names">Z.</div>
            </div>
             <div class="name">
              <div class="surname">Quail</div>
               <div class="given-names">M.A.</div>
            </div>
             <div class="name">
              <div class="surname">Sanders</div>
               <div class="given-names">M.J.</div>
            </div>
             <div class="name">
              <div class="surname">Berriman</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Turner</div>
               <div class="given-names">D.J.</div>
            </div>
          </div>
          <div class="year">2009</div>
            , 
            <div class="article-title">Amplification-free Illumina sequencing-library preparation facilitates improved mapping and assembly of (G+C)-biased genomes,</div><div class="source">Nat. Methods</div> 
            , 
            <div class="volume">6</div>
            , 
            <div class="fpage">291</div>
            –
            <div class="lpage">5</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Amplification-free%20Illumina%20sequencing-library%20preparation%20facilitates%20improved%20mapping%20and%20assembly%20of%20(G%2BC)-biased%20genomes%2C&amp;author=I.%20Kozarewa&amp;author=Z.%20Ning&amp;author=M.A.%20Quail&amp;author=M.J.%20Sanders&amp;author=M.%20Berriman&amp;author=D.J.%20Turner&amp;publication_year=2009&amp;journal=Nat.%20Methods&amp;volume=6&amp;pages=291-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1311" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1311" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1311"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19287394" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Hoeijmakers</div>
               <div class="given-names">W.A.</div>
            </div>
             <div class="name">
              <div class="surname">Flueck</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Francoijs</div>
               <div class="given-names">K.J.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2012</div>
            , 
            <div class="article-title"><em>Plasmodium falciparum</em>
               centromeres display a unique epigenetic makeup and cluster prior to and during schizogony,
            </div><div class="source">Cell. Microbiol</div> 
            ., 
            <div class="volume">14</div>
            , 
            <div class="fpage">1391</div>
            –
            <div class="lpage">401</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plasmodium%20falciparum%20centromeres%20display%20a%20unique%20epigenetic%20makeup%20and%20cluster%20prior%20to%20and%20during%20schizogony%2C&amp;author=W.A.%20Hoeijmakers&amp;author=C.%20Flueck&amp;author=K.J.%20Francoijs&amp;publication_year=2012&amp;journal=Cell.%20Microbiol&amp;volume=14&amp;pages=1391-401" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1462-5822.2012.01803.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1462-5822.2012.01803.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1462-5822.2012.01803.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22507744" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kelly</div>
               <div class="given-names">J.M.</div>
            </div>
             <div class="name">
              <div class="surname">McRobert</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Baker</div>
               <div class="given-names">D.A.</div>
            </div>
          </div>
          <div class="year">2006</div>
            , 
            <div class="article-title">
              Evidence on the chromosomal location of centromeric DNA in 
              <em>Plasmodium falciparum</em>
               from etoposide-mediated topoisomerase-II cleavage,
            </div><div class="source">Proc. Natl. Acad. Sci. U. S. A</div> 
            ., 
            <div class="volume">103</div>
            , 
            <div class="fpage">6706</div>
            –
            <div class="lpage">11</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20on%20the%20chromosomal%20location%20of%20centromeric%20DNA%20in%20Plasmodium%20falciparum%20from%20etoposide-mediated%20topoisomerase-II%20cleavage%2C&amp;author=J.M.%20Kelly&amp;author=L.%20McRobert&amp;author=D.A.%20Baker&amp;publication_year=2006&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=103&amp;pages=6706-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0510363103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0510363103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0510363103"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16617116" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Figueiredo</div>
               <div class="given-names">L.M.</div>
            </div>
             <div class="name">
              <div class="surname">Freitas-Junior</div>
               <div class="given-names">L.H.</div>
            </div>
             <div class="name">
              <div class="surname">Bottius</div>
               <div class="given-names">E.</div>
            </div>
             <div class="name">
              <div class="surname">Olivo-Marin</div>
               <div class="given-names">J.C.</div>
            </div>
             <div class="name">
              <div class="surname">Scherf</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="year">2002</div>
            , 
            <div class="article-title">
              A central role for 
              <em>Plasmodium falciparum</em>
               subtelomeric regions in spatial positioning and telomere length regulation,
            </div><div class="source">EMBO J</div> 
            ., 
            <div class="volume">21</div>
            , 
            <div class="fpage">815</div>
            –
            <div class="lpage">24</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20central%20role%20for%20Plasmodium%20falciparum%20subtelomeric%20regions%20in%20spatial%20positioning%20and%20telomere%20length%20regulation%2C&amp;author=L.M.%20Figueiredo&amp;author=L.H.%20Freitas-Junior&amp;author=E.%20Bottius&amp;author=J.C.%20Olivo-Marin&amp;author=A.%20Scherf&amp;publication_year=2002&amp;journal=EMBO%20J&amp;volume=21&amp;pages=815-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/emboj/21.4.815" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Femboj%2F21.4.815" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Femboj%2F21.4.815"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11847128" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Goel</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Palmkvist</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Moll</div>
               <div class="given-names">K.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2015</div>
            , 
            <div class="article-title">
              RIFINs are adhesins implicated in severe 
              <em>Plasmodium falciparum</em>
               malaria,
            </div><div class="source">Nat. Med</div> 
            ., 
            <div class="volume">21</div>
            , 
            <div class="fpage">314</div>
            –
            <div class="lpage">7</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RIFINs%20are%20adhesins%20implicated%20in%20severe%20Plasmodium%20falciparum%20malaria%2C&amp;author=S.%20Goel&amp;author=M.%20Palmkvist&amp;author=K.%20Moll&amp;publication_year=2015&amp;journal=Nat.%20Med&amp;volume=21&amp;pages=314-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nm.3812" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnm.3812" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnm.3812"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25751816" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kyes</div>
               <div class="given-names">S.A.</div>
            </div>
             <div class="name">
              <div class="surname">Rowe</div>
               <div class="given-names">J.A.</div>
            </div>
             <div class="name">
              <div class="surname">Kriek</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Newbold</div>
               <div class="given-names">C.I.</div>
            </div>
          </div>
          <div class="year">1999</div>
            , 
            <div class="article-title">
              Rifins: a second family of clonally variant proteins expressed on the surface of red cells infected with 
              <em>Plasmodium falciparum</em>
              ,
            </div><div class="source">Proc. Natl. Acad. Sci. U. S. A</div> 
            ., 
            <div class="volume">96</div>
            , 
            <div class="fpage">9333</div>
            –
            <div class="lpage">8</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rifins%3A%20a%20second%20family%20of%20clonally%20variant%20proteins%20expressed%20on%20the%20surface%20of%20red%20cells%20infected%20with%20Plasmodium%20falciparum%20%2C&amp;author=S.A.%20Kyes&amp;author=J.A.%20Rowe&amp;author=N.%20Kriek&amp;author=C.I.%20Newbold&amp;publication_year=1999&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=96&amp;pages=9333-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.96.16.9333" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.96.16.9333" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.96.16.9333"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10430943" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Ghansah</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Amenga-Etego</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Amambua-Ngwa</div>
               <div class="given-names">A.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2014</div>
            , 
            <div class="article-title">Monitoring parasite diversity for malaria elimination in sub-Saharan Africa,</div><div class="source">Science</div> 
            , 
            <div class="volume">345</div>
            , 
            <div class="fpage">1297</div>
            –
            <div class="lpage">8</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Monitoring%20parasite%20diversity%20for%20malaria%20elimination%20in%20sub-Saharan%20Africa%2C&amp;author=A.%20Ghansah&amp;author=L.%20Amenga-Etego&amp;author=A.%20Amambua-Ngwa&amp;publication_year=2014&amp;journal=Science&amp;volume=345&amp;pages=1297-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1259423" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1259423" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1259423"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25214619" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Oyola</div>
               <div class="given-names">S.O.</div>
            </div>
             <div class="name">
              <div class="surname">Gu</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Manske</div>
               <div class="given-names">M.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2013</div>
            , 
            <div class="article-title">Efficient depletion of host DNA contamination in malaria clinical sequencing,</div><div class="source">J. Clin. Microbiol</div> 
            ., 
            <div class="volume">51</div>
            , 
            <div class="fpage">745</div>
            –
            <div class="lpage">51</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efficient%20depletion%20of%20host%20DNA%20contamination%20in%20malaria%20clinical%20sequencing%2C&amp;author=S.O.%20Oyola&amp;author=Y.%20Gu&amp;author=M.%20Manske&amp;publication_year=2013&amp;journal=J.%20Clin.%20Microbiol&amp;volume=51&amp;pages=745-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JCM.02507-12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJCM.02507-12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJCM.02507-12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23224084" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Bei</div>
               <div class="given-names">A.K.</div>
            </div>
             <div class="name">
              <div class="surname">Patel</div>
               <div class="given-names">S.D.</div>
            </div>
             <div class="name">
              <div class="surname">Volkman</div>
               <div class="given-names">S.K.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2014</div>
            , 
            <div class="article-title">
              An adjustable gas-mixing device to increase feasibility of 
              <em>in vitro</em>
               culture of 
              <em>Plasmodium falciparum</em>
               parasites in the field,
            </div><div class="source">PLoS One</div> 
            , 
            <div class="volume">9</div>
            , 
            <div class="fpage">e90928</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20adjustable%20gas-mixing%20device%20to%20increase%20feasibility%20of%20in%20vitro%20culture%20of%20Plasmodium%20falciparum%20parasites%20in%20the%20field%2C&amp;author=A.K.%20Bei&amp;author=S.D.%20Patel&amp;author=S.K.%20Volkman&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e90928" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0090928" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0090928" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0090928"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24603696" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Ribacke</div>
               <div class="given-names">U.</div>
            </div>
             <div class="name">
              <div class="surname">Moll</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Albrecht</div>
               <div class="given-names">L.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2013</div>
            , 
            <div class="article-title">
              Improved 
              <em>in vitro</em>
               culture of 
              <em>Plasmodium falciparum</em>
               permits establishment of clinical isolates with preserved multiplication, invasion and rosetting phenotypes,
            </div><div class="source">PLoS One</div> 
            , 
            <div class="volume">8</div>
            , 
            <div class="fpage">e69781</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improved%20in%20vitro%20culture%20of%20Plasmodium%20falciparum%20permits%20establishment%20of%20clinical%20isolates%20with%20preserved%20multiplication%2C%20invasion%20and%20rosetting%20phenotypes%2C&amp;author=U.%20Ribacke&amp;author=K.%20Moll&amp;author=L.%20Albrecht&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e69781" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0069781" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0069781" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0069781"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23894537" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Korlach</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Turner</div>
               <div class="given-names">S.W.</div>
            </div>
          </div>
          <div class="year">2012</div>
            , 
            <div class="article-title">Going beyond five bases in DNA sequencing,</div><div class="source">Curr. Opin. Struct. Biol</div> 
            ., 
            <div class="volume">22</div>
            , 
            <div class="fpage">251</div>
            –
            <div class="lpage">61</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Going%20beyond%20five%20bases%20in%20DNA%20sequencing%2C&amp;author=J.%20Korlach&amp;author=S.W.%20Turner&amp;publication_year=2012&amp;journal=Curr.%20Opin.%20Struct.%20Biol&amp;volume=22&amp;pages=251-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.sbi.2012.04.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.sbi.2012.04.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.sbi.2012.04.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22575758" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Davis</div>
               <div class="given-names">B.M.</div>
            </div>
             <div class="name">
              <div class="surname">Chao</div>
               <div class="given-names">M.C.</div>
            </div>
             <div class="name">
              <div class="surname">Waldor</div>
               <div class="given-names">M.K.</div>
            </div>
          </div>
          <div class="year">2013</div>
            , 
            <div class="article-title">Entering the era of bacterial epigenomics with single molecule real time DNA sequencing,</div><div class="source">Curr. Opin. Microbiol</div> 
            ., 
            <div class="volume">16</div>
            , 
            <div class="fpage">192</div>
            –
            <div class="lpage">8</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Entering%20the%20era%20of%20bacterial%20epigenomics%20with%20single%20molecule%20real%20time%20DNA%20sequencing%2C&amp;author=B.M.%20Davis&amp;author=M.C.%20Chao&amp;author=M.K.%20Waldor&amp;publication_year=2013&amp;journal=Curr.%20Opin.%20Microbiol&amp;volume=16&amp;pages=192-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mib.2013.01.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mib.2013.01.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mib.2013.01.011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23434113" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Greer</div>
               <div class="given-names">E.L.</div>
            </div>
             <div class="name">
              <div class="surname">Blanco</div>
               <div class="given-names">M.A.</div>
            </div>
             <div class="name">
              <div class="surname">Gu</div>
               <div class="given-names">L.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2015</div>
            , 
            <div class="article-title">
              DNA methylation on N6-Adenine in 
              <em>C. elegans</em>
              ,
            </div><div class="source">Cell</div> 
            , 
            <div class="volume">161</div>
            , 
            <div class="fpage">868</div>
            –
            <div class="lpage">78</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20methylation%20on%20N6-Adenine%20in%20C.%20elegans%20%2C&amp;author=E.L.%20Greer&amp;author=M.A.%20Blanco&amp;author=L.%20Gu&amp;publication_year=2015&amp;journal=Cell&amp;volume=161&amp;pages=868-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2015.04.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2015.04.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2015.04.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25936839" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Wu</div>
               <div class="given-names">T.P.</div>
            </div>
             <div class="name">
              <div class="surname">Wang</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Seetin</div>
               <div class="given-names">M.G.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2016</div>
            , 
            <div class="article-title">DNA methylation on N(6)-adenine in mammalian embryonic stem cells</div>
            , 
            <div class="source">Nature</div> 
            , 
            <div class="volume">532</div>
            , 
            <div class="fpage">329</div>
            –
            <div class="lpage">333</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20methylation%20on%20N(6)-adenine%20in%20mammalian%20embryonic%20stem%20cells&amp;author=T.P.%20Wu&amp;author=T.%20Wang&amp;author=M.G.%20Seetin&amp;publication_year=2016&amp;journal=Nature&amp;volume=532&amp;pages=329-333" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature17640" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature17640" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature17640"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27027282" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw022-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw022-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Ponts</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Fu</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Harris</div>
               <div class="given-names">E.Y.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2013</div>
            , 
            <div class="article-title">
              Genome-wide mapping of DNA methylation in the human malaria parasite 
              <em>Plasmodium falciparum</em>
              ,
            </div><div class="source">Cell Host Microbe</div> 
            , 
            <div class="volume">14</div>
            , 
            <div class="fpage">696</div>
            –
            <div class="lpage">706</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20mapping%20of%20DNA%20methylation%20in%20the%20human%20malaria%20parasite%20Plasmodium%20falciparum%20%2C&amp;author=N.%20Ponts&amp;author=L.%20Fu&amp;author=E.Y.%20Harris&amp;publication_year=2013&amp;journal=Cell%20Host%20Microbe&amp;volume=14&amp;pages=696-706" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.chom.2013.11.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.chom.2013.11.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.chom.2013.11.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24331467" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p"> This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License ( <a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a> ), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com </div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2469970&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw022/2/dsw022_Supplementary_Data.zip?Expires=2147483647&amp;Signature=NEv6aZ5j6R-RjghVh1O9KGbIrr5vHbMXEXLhW6GuhgTSY4cMvH39S0gtZ0dxo5iKqWTgwu5kdJrP01cEURAnENEi5jSpSaSpf9YezvPgu7SBrFZUw5zm7i~jatBRJAMwH8o1xJ2PccCD0u7nkhAmF7q3g6BgPUwbf7a-frfruUxZdEbBtauplmhJxldCZ~Iwkux~b5-zLTmszX2rdn3V9sEkO7FQFjgdvgJjNVtLGN9fTCeZ~0m2uhLOBcJbJQrDCHteRjzBmTYe3Gc7lIXrzoA-CCzl46~Rg9pq~UEYk4v1kvl-XKpU~1JUfbBoDERiqc2DazIKU4BZKJohwjlzfA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,277</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000384768700004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">7</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw022"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2469970"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000384768700004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f23%2f4%2f339%2f2469970">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000384768700004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (7)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F23%2F4%2F339%2F2469970" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2469970'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw022");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f23%2f4%2f339%2f2469970">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

