

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/19/3/219/474767","@type":"ScholarlyArticle","name":"Characterization of Species-Specific Repeats in 613 Prokaryotic Species","datePublished":"2012-02-24","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/19/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2012-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dss006","keywords":["species-specific","repeat","identification","prokaryote","pathogen"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/19/3/219/474767","author":[{"name":"Koressaar, Triinu","affiliation":"Department of Bioinformatics ,  Institute of Molecular and Cell Biology, University of Tartu ,  Tartu ,  Estonia","@type":"Person"},{"name":"Remm, Maido","affiliation":"Department of Bioinformatics ,  Institute of Molecular and Cell Biology, University of Tartu ,  Tartu ,  Estonia","@type":"Person"}],"description":"Abstract.  Prokaryotes are in general believed to possess small, compactly organized genomes, with repetitive sequences forming only a small part of them. Nonet","pageStart":"219","pageEnd":"230","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/3/1/m_cover.gif?Expires=2147483647&Signature=Fdyy0J9TWIOvyTsobWsaSe0-4dpdJpPal81dGnPphMlVPQ-~doKkHPlJp1UwhE6iBMO5EiCDjakz8lsMXANxWPYyveBKgVvoq77bYMGovqWIYBOCCELrdx0j6HVYZwX9Y74Ddg90XP77LfXNnNk59wdvjztMTMkN-3mdBevmXsNmZKVV5PmBqkKimC9vUcC6~0jrzJOVoAsfn74p43Pf6SY8Ru3oEUvpmOEuP9brDQjSX9QkiRbtOgqGZ2ApS-2m29LaARntNx2YmeI-Qi7~alsILkg5AbRepwVxxLytg4b29~~Z7IOxt-m0JR0e1LCuqAZKEhE4gn5lRwgzeExSjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Characterization of Species-Specific Repeats in 613 Prokaryotic Species","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/3/1/m_cover.gif?Expires=2147483647&Signature=Fdyy0J9TWIOvyTsobWsaSe0-4dpdJpPal81dGnPphMlVPQ-~doKkHPlJp1UwhE6iBMO5EiCDjakz8lsMXANxWPYyveBKgVvoq77bYMGovqWIYBOCCELrdx0j6HVYZwX9Y74Ddg90XP77LfXNnNk59wdvjztMTMkN-3mdBevmXsNmZKVV5PmBqkKimC9vUcC6~0jrzJOVoAsfn74p43Pf6SY8Ru3oEUvpmOEuP9brDQjSX9QkiRbtOgqGZ2ApS-2m29LaARntNx2YmeI-Qi7~alsILkg5AbRepwVxxLytg4b29~~Z7IOxt-m0JR0e1LCuqAZKEhE4gn5lRwgzeExSjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Characterization of Species-Specific Repeats in 613 Prokaryotic Species" />
    <meta property="og:description" content="Abstract.  Prokaryotes are in general believed to possess small, compactly organized genomes, with repetitive sequences forming only a small part of them. Nonet" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/19/3/219/474767" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2012-02-24" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Koressaar, Triinu" /><meta name="citation_author_institution" content="Department of Bioinformatics, Institute of Molecular and Cell Biology, University of Tartu, Tartu, Estonia" /><meta name="citation_author" content="Remm, Maido" /><meta name="citation_author_institution" content="Department of Bioinformatics, Institute of Molecular and Cell Biology, University of Tartu, Tartu, Estonia" /><meta name="citation_title" content="Characterization of Species-Specific Repeats in 613 Prokaryotic Species" /><meta name="citation_firstpage" content="219" /><meta name="citation_lastpage" content="230" /><meta name="citation_doi" content="10.1093/dnares/dss006" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="19" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2012/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Genomics of bacteria and archaea: the emerging dynamic view of the prokaryotic world; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=6688-719; " /><meta name="citation_reference" content="citation_title=The 160-kilobase genome of the bacterial endosymbiont Carsonella; Science;  citation_year=2006;  citation_volume=314;  citation_pages=267" /><meta name="citation_reference" content="citation_title=Complete genome sequence of the myxobacterium Sorangium cellulosum; Nat. Biotechnol.;  citation_year=2007;  citation_volume=25;  citation_pages=1281-9; " /><meta name="citation_reference" content="citation_title=Small mobile sequences in bacteria display diverse structure/function motifs; Mol. Microbiol.;  citation_year=2008;  citation_volume=67;  citation_pages=475-81; " /><meta name="citation_reference" content="citation_title=Origin and fate of repeats in bacteria; Nucleic Acids Res.;  citation_year=2002;  citation_volume=30;  citation_pages=2987-94; " /><meta name="citation_reference" content="citation_title=Phylogenomics of the reproductive parasite Wolbachia pipientis wMel: a streamlined genome overrun by mobile genetic elements; PLoS Biol.;  citation_year=2004;  citation_volume=2;  citation_pages=E69" /><meta name="citation_reference" content="citation_title=Functional and evolutionary roles of long repeats in prokaryotes; Res. Microbiol.;  citation_year=1999;  citation_volume=150;  citation_pages=725-33; " /><meta name="citation_reference" content="citation_title=Analysis of long repeats in bacterial genomes reveals alternative evolutionary mechanisms in Bacillus subtilis and other competent prokaryotes; Mol. Biol. Evol.;  citation_year=1999;  citation_volume=16;  citation_pages=1219-30; " /><meta name="citation_reference" content="citation_title=Mobility of plasmids; Microbiol. Mol. Biol. Rev.;  citation_year=2010;  citation_volume=74;  citation_pages=434-52; " /><meta name="citation_reference" content="citation_title=Mobile genetic elements in the genome of the beneficial rhizobacterium Pseudomonas fluorescens Pf-5; BMC Microbiol.;  citation_year=2009;  citation_volume=9;  citation_pages=8" /><meta name="citation_reference" content="citation_title=ISCR elements: novel gene-capturing systems of the 21st century?; Microbiol. Mol. Biol. Rev.;  citation_year=2006;  citation_volume=70;  citation_pages=296-316; " /><meta name="citation_reference" content="citation_title=DDE transposases: Structural similarity and diversity; Adv. Drug Deliv. Rev.;  citation_year=2010;  citation_volume=62;  citation_pages=1187-1195; " /><meta name="citation_reference" content="citation_title=Atypical association of DDE transposition with conjugation specifies a new family of mobile elements; Mol. Microbiol.;  citation_year=2009;  citation_volume=71;  citation_pages=948-59; " /><meta name="citation_reference" content="citation_title=Integrative and conjugative elements: mosaic mobile genetic elements enabling dynamic lateral gene flow; Nat. Rev. Microbiol.;  citation_year=2010;  citation_volume=8;  citation_pages=552-63; " /><meta name="citation_reference" content="citation_title=Bacterial repetitive extragenic palindromic sequences are DNA targets for insertion sequence elements; BMC Genomics;  citation_year=2006;  citation_volume=7;  citation_pages=62" /><meta name="citation_reference" content="citation_title=Positive selection on transposase genes of insertion sequences in the Crocosphaera watsonii genome; J. Bacteriol.;  citation_year=2006;  citation_volume=188;  citation_pages=7176-85; " /><meta name="citation_reference" content="citation_title=Identification and characterization of repetitive extragenic palindromes (REP)-associated tyrosine transposases: implications for REP evolution and dynamics in bacterial genomes; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=44" /><meta name="citation_reference" content="citation_title=A hidden reservoir of integrative elements is the major source of recently acquired foreign genes and ORFans in archaeal and bacterial genomes; Genome Biol.;  citation_year=2009;  citation_volume=10;  citation_pages=R65" /><meta name="citation_reference" content="citation_title=Horizontal gene transfer, genome innovation and evolution; Nat. Rev. Microbiol.;  citation_year=2005;  citation_volume=3;  citation_pages=679-87; " /><meta name="citation_reference" content="citation_title=Phage as agents of lateral gene transfer; Curr. Opin. Microbiol.;  citation_year=2003;  citation_volume=6;  citation_pages=417-24; " /><meta name="citation_reference" content="citation_title=Mobile genetic elements: the agents of open source evolution; Nat. Rev. Microbiol.;  citation_year=2005;  citation_volume=3;  citation_pages=722-32; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of the anaerobic, protein-degrading hyperthermophilic crenarchaeon Desulfurococcus kamchatkensis; J. Bacteriol.;  citation_year=2009;  citation_volume=191;  citation_pages=2371-9; " /><meta name="citation_reference" content="citation_title=Genome sequence and rapid evolution of the rice pathogen Xanthomonas oryzae pv. oryzae PXO99A; BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=204" /><meta name="citation_reference" content="citation_title=Automatic identification of species-specific repetitive DNA sequences and their utilization for detecting microbial organisms; Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1349-55; " /><meta name="citation_reference" content="citation_title=Phage_Finder: automated identification and classification of prophage regions in complete bacterial genome sequences; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=5839-51; " /><meta name="citation_reference" content="citation_title=ISsaga is an ensemble of web-based methods for high throughput identification and semi-automatic annotation of insertion sequences in prokaryotic genomes; Genome Biol.;  citation_year=2011;  citation_volume=12;  citation_pages=R30" /><meta name="citation_reference" content="citation_title=CRISPRFinder: a web tool to identify clustered regularly interspaced short palindromic repeats; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=W52-7; " /><meta name="citation_reference" content="citation_title=The Pfam protein families database; Nucleic Acids Res.;  citation_year=2010;  citation_volume=38;  citation_pages=D211-22; " /><meta name="citation_reference" content="citation_title=Transfer of antibiotic multiresistant plasmid RP4 from Escherichia coli to activated sludge bacteria; J. Biosci. Bioeng.;  citation_year=2008;  citation_volume=106;  citation_pages=292-6; " /><meta name="citation_reference" content="citation_title=Sequencing and genetic variation of multidrug resistance plasmids in Klebsiella pneumoniae; PLoS One;  citation_year=2010;  citation_volume=5;  citation_pages=e10141" /><meta name="citation_reference" content="citation_title=Bacterial interspersed mosaic elements (BIMEs) are a major source of sequence polymorphism in Escherichia coli intergenic regions including specific associations with a new insertion sequence; Genetics;  citation_year=1997;  citation_volume=145;  citation_pages=551-62; " /><meta name="citation_reference" content="citation_title=A novel repeated DNA sequence located in the intergenic regions of bacterial chromosomes; Nucleic Acids Res.;  citation_year=1990;  citation_volume=18;  citation_pages=6503-8; " /><meta name="citation_reference" content="citation_title=Identification of long intergenic repeat sequences associated with DNA methylation sites in Caulobacter crescentus and other alpha-proteobacteria; J. Bacteriol.;  citation_year=2003;  citation_volume=185;  citation_pages=4997-5002; " /><meta name="citation_reference" content="citation_title=Selfish genes, the phenotype paradigm and genome evolution; Nature;  citation_year=1980;  citation_volume=284;  citation_pages=601-3; " /><meta name="citation_reference" content="citation_title=Selfish DNA: the ultimate parasite; Nature;  citation_year=1980;  citation_volume=284;  citation_pages=604-7; " /><meta name="citation_reference" content="citation_title=The probability of preservation of a newly arisen gene duplicate; Genetics;  citation_year=2001;  citation_volume=159;  citation_pages=1789-804; " /><meta name="citation_reference" content="citation_title=The archaeal Sec-dependent protein translocation pathway; Philos. Trans. R. Soc. Lond. B Biol. Sci.;  citation_year=2004;  citation_volume=359;  citation_pages=919-27; " /><meta name="citation_reference" content="citation_title=Adventures with ABC-proteins: highly conserved ATP-dependent transporters; Acta Microbiol. Immunol. Hung.;  citation_year=2005;  citation_volume=52;  citation_pages=309-22; " /><meta name="citation_reference" content="citation_title=The interaction between RTX toxins and target cells; Trends Microbiol.;  citation_year=1999;  citation_volume=7;  citation_pages=356-61; " /><meta name="citation_reference" content="citation_title=Microbial diversity and the genetic nature of microbial species; Nat. Rev. Microbiol.;  citation_year=2008;  citation_volume=6;  citation_pages=431-40; " /><meta name="citation_reference" content="citation_title=Rapid detection of Shiga toxin-producing bacteria in feces by multiplex PCR with molecular beacons on the smart cycler; J. Clin. Microbiol.;  citation_year=2002;  citation_volume=40;  citation_pages=1436-40; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/19/3/219/1283594/dss006.pdf" /><meta name="description" content="Abstract.  Prokaryotes are in general believed to possess small, compactly organized genomes, with repetitive sequences forming only a small part of them. Nonet" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/19/3/219/474767" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/19/3/219/474767" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/19/3/219/474767");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dss006';
    </script>
<title>
	Characterization of Species-Specific Repeats in 613 Prokaryotic Species | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./474767" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BbRWPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLb3Jlc3NhYXIsIFRyaWludSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvaW5mb3JtYXRpY3MsIEluc3RpdHV0ZSBvZiBNb2xlY3VsYXIgYW5kIENlbGwgQmlvbG9neSwgVW5pdmVyc2l0eSBvZiBUYXJ0dSwgVGFydHUsIEVzdG9uaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJSZW1tLCBNYWlkbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvaW5mb3JtYXRpY3MsIEluc3RpdHV0ZSBvZiBNb2xlY3VsYXIgYW5kIENlbGwgQmlvbG9neSwgVW5pdmVyc2l0eSBvZiBUYXJ0dSwgVGFydHUsIEVzdG9uaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkNoYXJhY3Rlcml6YXRpb24gb2YgU3BlY2llcy1TcGVjaWZpYyBSZXBlYXRzIGluIDYxMyBQcm9rYXJ5b3RpYyBTcGVjaWVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMjE5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIyMzAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3MwMDYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxMi8wNi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWNzIG9mIGJhY3RlcmlhIGFuZCBhcmNoYWVhOiB0aGUgZW1lcmdpbmcgZHluYW1pYyB2aWV3IG9mIHRoZSBwcm9rYXJ5b3RpYyB3b3JsZDsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9NjY4OC03MTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSAxNjAta2lsb2Jhc2UgZ2Vub21lIG9mIHRoZSBiYWN0ZXJpYWwgZW5kb3N5bWJpb250IENhcnNvbmVsbGE7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzE0OyAgY2l0YXRpb25fcGFnZXM9MjY3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGxldGUgZ2Vub21lIHNlcXVlbmNlIG9mIHRoZSBteXhvYmFjdGVyaXVtIFNvcmFuZ2l1bSBjZWxsdWxvc3VtOyBOYXQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MTI4MS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TbWFsbCBtb2JpbGUgc2VxdWVuY2VzIGluIGJhY3RlcmlhIGRpc3BsYXkgZGl2ZXJzZSBzdHJ1Y3R1cmUvZnVuY3Rpb24gbW90aWZzOyBNb2wuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9Njc7ICBjaXRhdGlvbl9wYWdlcz00NzUtODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9yaWdpbiBhbmQgZmF0ZSBvZiByZXBlYXRzIGluIGJhY3RlcmlhOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0yOTg3LTk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaHlsb2dlbm9taWNzIG9mIHRoZSByZXByb2R1Y3RpdmUgcGFyYXNpdGUgV29sYmFjaGlhIHBpcGllbnRpcyB3TWVsOiBhIHN0cmVhbWxpbmVkIGdlbm9tZSBvdmVycnVuIGJ5IG1vYmlsZSBnZW5ldGljIGVsZW1lbnRzOyBQTG9TIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz1FNjkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdW5jdGlvbmFsIGFuZCBldm9sdXRpb25hcnkgcm9sZXMgb2YgbG9uZyByZXBlYXRzIGluIHByb2thcnlvdGVzOyBSZXMuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTUwOyAgY2l0YXRpb25fcGFnZXM9NzI1LTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiBsb25nIHJlcGVhdHMgaW4gYmFjdGVyaWFsIGdlbm9tZXMgcmV2ZWFscyBhbHRlcm5hdGl2ZSBldm9sdXRpb25hcnkgbWVjaGFuaXNtcyBpbiBCYWNpbGx1cyBzdWJ0aWxpcyBhbmQgb3RoZXIgY29tcGV0ZW50IHByb2thcnlvdGVzOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MTIxOS0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9iaWxpdHkgb2YgcGxhc21pZHM7IE1pY3JvYmlvbC4gTW9sLiBCaW9sLiBSZXYuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTc0OyAgY2l0YXRpb25fcGFnZXM9NDM0LTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2JpbGUgZ2VuZXRpYyBlbGVtZW50cyBpbiB0aGUgZ2Vub21lIG9mIHRoZSBiZW5lZmljaWFsIHJoaXpvYmFjdGVyaXVtIFBzZXVkb21vbmFzIGZsdW9yZXNjZW5zIFBmLTU7IEJNQyBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz04IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SVNDUiBlbGVtZW50czogbm92ZWwgZ2VuZS1jYXB0dXJpbmcgc3lzdGVtcyBvZiB0aGUgMjFzdCBjZW50dXJ5PzsgTWljcm9iaW9sLiBNb2wuIEJpb2wuIFJldi47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NzA7ICBjaXRhdGlvbl9wYWdlcz0yOTYtMzE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EREUgdHJhbnNwb3Nhc2VzOiBTdHJ1Y3R1cmFsIHNpbWlsYXJpdHkgYW5kIGRpdmVyc2l0eTsgQWR2LiBEcnVnIERlbGl2LiBSZXYuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTYyOyAgY2l0YXRpb25fcGFnZXM9MTE4Ny0xMTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BdHlwaWNhbCBhc3NvY2lhdGlvbiBvZiBEREUgdHJhbnNwb3NpdGlvbiB3aXRoIGNvbmp1Z2F0aW9uIHNwZWNpZmllcyBhIG5ldyBmYW1pbHkgb2YgbW9iaWxlIGVsZW1lbnRzOyBNb2wuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NzE7ICBjaXRhdGlvbl9wYWdlcz05NDgtNTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVncmF0aXZlIGFuZCBjb25qdWdhdGl2ZSBlbGVtZW50czogbW9zYWljIG1vYmlsZSBnZW5ldGljIGVsZW1lbnRzIGVuYWJsaW5nIGR5bmFtaWMgbGF0ZXJhbCBnZW5lIGZsb3c7IE5hdC4gUmV2LiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz01NTItNjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhY3RlcmlhbCByZXBldGl0aXZlIGV4dHJhZ2VuaWMgcGFsaW5kcm9taWMgc2VxdWVuY2VzIGFyZSBETkEgdGFyZ2V0cyBmb3IgaW5zZXJ0aW9uIHNlcXVlbmNlIGVsZW1lbnRzOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTYyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG9zaXRpdmUgc2VsZWN0aW9uIG9uIHRyYW5zcG9zYXNlIGdlbmVzIG9mIGluc2VydGlvbiBzZXF1ZW5jZXMgaW4gdGhlIENyb2Nvc3BoYWVyYSB3YXRzb25paSBnZW5vbWU7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTg4OyAgY2l0YXRpb25fcGFnZXM9NzE3Ni04NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgcmVwZXRpdGl2ZSBleHRyYWdlbmljIHBhbGluZHJvbWVzIChSRVApLWFzc29jaWF0ZWQgdHlyb3NpbmUgdHJhbnNwb3Nhc2VzOiBpbXBsaWNhdGlvbnMgZm9yIFJFUCBldm9sdXRpb24gYW5kIGR5bmFtaWNzIGluIGJhY3RlcmlhbCBnZW5vbWVzOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz00NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgaGlkZGVuIHJlc2Vydm9pciBvZiBpbnRlZ3JhdGl2ZSBlbGVtZW50cyBpcyB0aGUgbWFqb3Igc291cmNlIG9mIHJlY2VudGx5IGFjcXVpcmVkIGZvcmVpZ24gZ2VuZXMgYW5kIE9SRmFucyBpbiBhcmNoYWVhbCBhbmQgYmFjdGVyaWFsIGdlbm9tZXM7IEdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPVI2NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhvcml6b250YWwgZ2VuZSB0cmFuc2ZlciwgZ2Vub21lIGlubm92YXRpb24gYW5kIGV2b2x1dGlvbjsgTmF0LiBSZXYuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTY3OS04NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGhhZ2UgYXMgYWdlbnRzIG9mIGxhdGVyYWwgZ2VuZSB0cmFuc2ZlcjsgQ3Vyci4gT3Bpbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9NDE3LTI0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2JpbGUgZ2VuZXRpYyBlbGVtZW50czogdGhlIGFnZW50cyBvZiBvcGVuIHNvdXJjZSBldm9sdXRpb247IE5hdC4gUmV2LiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz03MjItMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBsZXRlIGdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgYW5hZXJvYmljLCBwcm90ZWluLWRlZ3JhZGluZyBoeXBlcnRoZXJtb3BoaWxpYyBjcmVuYXJjaGFlb24gRGVzdWxmdXJvY29jY3VzIGthbWNoYXRrZW5zaXM7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTkxOyAgY2l0YXRpb25fcGFnZXM9MjM3MS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgc2VxdWVuY2UgYW5kIHJhcGlkIGV2b2x1dGlvbiBvZiB0aGUgcmljZSBwYXRob2dlbiBYYW50aG9tb25hcyBvcnl6YWUgcHYuIG9yeXphZSBQWE85OUE7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MjA0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXV0b21hdGljIGlkZW50aWZpY2F0aW9uIG9mIHNwZWNpZXMtc3BlY2lmaWMgcmVwZXRpdGl2ZSBETkEgc2VxdWVuY2VzIGFuZCB0aGVpciB1dGlsaXphdGlvbiBmb3IgZGV0ZWN0aW5nIG1pY3JvYmlhbCBvcmdhbmlzbXM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MTM0OS01NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGhhZ2VfRmluZGVyOiBhdXRvbWF0ZWQgaWRlbnRpZmljYXRpb24gYW5kIGNsYXNzaWZpY2F0aW9uIG9mIHByb3BoYWdlIHJlZ2lvbnMgaW4gY29tcGxldGUgYmFjdGVyaWFsIGdlbm9tZSBzZXF1ZW5jZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIGNpdGF0aW9uX3BhZ2VzPTU4MzktNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlTc2FnYSBpcyBhbiBlbnNlbWJsZSBvZiB3ZWItYmFzZWQgbWV0aG9kcyBmb3IgaGlnaCB0aHJvdWdocHV0IGlkZW50aWZpY2F0aW9uIGFuZCBzZW1pLWF1dG9tYXRpYyBhbm5vdGF0aW9uIG9mIGluc2VydGlvbiBzZXF1ZW5jZXMgaW4gcHJva2FyeW90aWMgZ2Vub21lczsgR2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9UjMwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q1JJU1BSRmluZGVyOiBhIHdlYiB0b29sIHRvIGlkZW50aWZ5IGNsdXN0ZXJlZCByZWd1bGFybHkgaW50ZXJzcGFjZWQgc2hvcnQgcGFsaW5kcm9taWMgcmVwZWF0czsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9VzUyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBQZmFtIHByb3RlaW4gZmFtaWxpZXMgZGF0YWJhc2U7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0zODsgIGNpdGF0aW9uX3BhZ2VzPUQyMTEtMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zZmVyIG9mIGFudGliaW90aWMgbXVsdGlyZXNpc3RhbnQgcGxhc21pZCBSUDQgZnJvbSBFc2NoZXJpY2hpYSBjb2xpIHRvIGFjdGl2YXRlZCBzbHVkZ2UgYmFjdGVyaWE7IEouIEJpb3NjaS4gQmlvZW5nLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xMDY7ICBjaXRhdGlvbl9wYWdlcz0yOTItNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VxdWVuY2luZyBhbmQgZ2VuZXRpYyB2YXJpYXRpb24gb2YgbXVsdGlkcnVnIHJlc2lzdGFuY2UgcGxhc21pZHMgaW4gS2xlYnNpZWxsYSBwbmV1bW9uaWFlOyBQTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9ZTEwMTQxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFjdGVyaWFsIGludGVyc3BlcnNlZCBtb3NhaWMgZWxlbWVudHMgKEJJTUVzKSBhcmUgYSBtYWpvciBzb3VyY2Ugb2Ygc2VxdWVuY2UgcG9seW1vcnBoaXNtIGluIEVzY2hlcmljaGlhIGNvbGkgaW50ZXJnZW5pYyByZWdpb25zIGluY2x1ZGluZyBzcGVjaWZpYyBhc3NvY2lhdGlvbnMgd2l0aCBhIG5ldyBpbnNlcnRpb24gc2VxdWVuY2U7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTE0NTsgIGNpdGF0aW9uX3BhZ2VzPTU1MS02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBub3ZlbCByZXBlYXRlZCBETkEgc2VxdWVuY2UgbG9jYXRlZCBpbiB0aGUgaW50ZXJnZW5pYyByZWdpb25zIG9mIGJhY3RlcmlhbCBjaHJvbW9zb21lczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9NjUwMy04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBsb25nIGludGVyZ2VuaWMgcmVwZWF0IHNlcXVlbmNlcyBhc3NvY2lhdGVkIHdpdGggRE5BIG1ldGh5bGF0aW9uIHNpdGVzIGluIENhdWxvYmFjdGVyIGNyZXNjZW50dXMgYW5kIG90aGVyIGFscGhhLXByb3Rlb2JhY3RlcmlhOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE4NTsgIGNpdGF0aW9uX3BhZ2VzPTQ5OTctNTAwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VsZmlzaCBnZW5lcywgdGhlIHBoZW5vdHlwZSBwYXJhZGlnbSBhbmQgZ2Vub21lIGV2b2x1dGlvbjsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0xOTgwOyAgY2l0YXRpb25fdm9sdW1lPTI4NDsgIGNpdGF0aW9uX3BhZ2VzPTYwMS0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZWxmaXNoIEROQTogdGhlIHVsdGltYXRlIHBhcmFzaXRlOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTE5ODA7ICBjaXRhdGlvbl92b2x1bWU9Mjg0OyAgY2l0YXRpb25fcGFnZXM9NjA0LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBwcm9iYWJpbGl0eSBvZiBwcmVzZXJ2YXRpb24gb2YgYSBuZXdseSBhcmlzZW4gZ2VuZSBkdXBsaWNhdGU7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTE1OTsgIGNpdGF0aW9uX3BhZ2VzPTE3ODktODA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgYXJjaGFlYWwgU2VjLWRlcGVuZGVudCBwcm90ZWluIHRyYW5zbG9jYXRpb24gcGF0aHdheTsgUGhpbG9zLiBUcmFucy4gUi4gU29jLiBMb25kLiBCIEJpb2wuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzU5OyAgY2l0YXRpb25fcGFnZXM9OTE5LTI3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZHZlbnR1cmVzIHdpdGggQUJDLXByb3RlaW5zOiBoaWdobHkgY29uc2VydmVkIEFUUC1kZXBlbmRlbnQgdHJhbnNwb3J0ZXJzOyBBY3RhIE1pY3JvYmlvbC4gSW1tdW5vbC4gSHVuZy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NTI7ICBjaXRhdGlvbl9wYWdlcz0zMDktMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBpbnRlcmFjdGlvbiBiZXR3ZWVuIFJUWCB0b3hpbnMgYW5kIHRhcmdldCBjZWxsczsgVHJlbmRzIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTM1Ni02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9iaWFsIGRpdmVyc2l0eSBhbmQgdGhlIGdlbmV0aWMgbmF0dXJlIG9mIG1pY3JvYmlhbCBzcGVjaWVzOyBOYXQuIFJldi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9NDMxLTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SYXBpZCBkZXRlY3Rpb24gb2YgU2hpZ2EgdG94aW4tcHJvZHVjaW5nIGJhY3RlcmlhIGluIGZlY2VzIGJ5IG11bHRpcGxleCBQQ1Igd2l0aCBtb2xlY3VsYXIgYmVhY29ucyBvbiB0aGUgc21hcnQgY3ljbGVyOyBKLiBDbGluLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9MTQzNi00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE5LzMvMjE5LzEyODM1OTQvZHNzMDA2LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBQcm9rYXJ5b3RlcyBhcmUgaW4gZ2VuZXJhbCBiZWxpZXZlZCB0byBwb3NzZXNzIHNtYWxsLCBjb21wYWN0bHkgb3JnYW5pemVkIGdlbm9tZXMsIHdpdGggcmVwZXRpdGl2ZSBzZXF1ZW5jZXMgZm9ybWluZyBvbmx5IGEgc21hbGwgcGFydCBvZiB0aGVtLiBOb25ldCIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDc0NzY3JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZBlLZK2f67oqcd1gsBn7nsUCZYyR" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f3%2f219%2f474767" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f3%2f219%2f474767" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=18620" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/19/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/3/1/m_cover.gif?Expires=2147483647&amp;Signature=Fdyy0J9TWIOvyTsobWsaSe0-4dpdJpPal81dGnPphMlVPQ-~doKkHPlJp1UwhE6iBMO5EiCDjakz8lsMXANxWPYyveBKgVvoq77bYMGovqWIYBOCCELrdx0j6HVYZwX9Y74Ddg90XP77LfXNnNk59wdvjztMTMkN-3mdBevmXsNmZKVV5PmBqkKimC9vUcC6~0jrzJOVoAsfn74p43Pf6SY8Ru3oEUvpmOEuP9brDQjSX9QkiRbtOgqGZ2ApS-2m29LaARntNx2YmeI-Qi7~alsILkg5AbRepwVxxLytg4b29~~Z7IOxt-m0JR0e1LCuqAZKEhE4gn5lRwgzeExSjA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/19/3">
                        <div class="volume trailing-comma">Volume 19</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2012        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="7157471">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7157471">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7157473">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7157473">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7157482">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7157482">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7157496">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7157496">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7157550">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7157550">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7157555">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7157555">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7157557">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7157557">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7157559">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7157559">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/19/3/209/474733">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/19/3/231/474863">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
<span class="article-flag"><a href="https://academic.oup.com/dnaresearch/search-results?f_OUPSeries=Editor's+Choice">Editor's Choice</a></span>            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Characterization of Species-Specific Repeats in 613 Prokaryotic Species
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Triinu Koressaar</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Department of Bioinformatics</div>, <div class="insititution">Institute of Molecular and Cell Biology, University of Tartu</div>, <div class="addr-line">Tartu</div>, <div class="country">Estonia</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +372 7375001. Fax. +372 7420286. E-mail: <a href="mailto:triinu.koressaar@ut.ee" target="_blank">triinu.koressaar@ut.ee</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Triinu+Koressaar">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Koressaar T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Koressaar T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Triinu Koressaar</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Maido Remm</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Department of Bioinformatics</div>, <div class="insititution">Institute of Molecular and Cell Biology, University of Tartu</div>, <div class="addr-line">Tartu</div>, <div class="country">Estonia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Maido+Remm">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Remm M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Remm M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Maido Remm</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 19, Issue 3, 1 June 2012, Pages 219–230, <a href='https://doi.org/10.1093/dnares/dss006'>https://doi.org/10.1093/dnares/dss006</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">24 February 2012</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">04 November 2011</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">26 January 2012</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="474767" href="/dnaresearch/article-pdf/19/3/219/1283594/dss006.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Triinu Koressaar, Maido Remm;  Characterization of Species-Specific Repeats in 613 Prokaryotic Species, <em>DNA Research</em>, Volume 19, Issue 3, 1 June 2012, Pages 219–230, <a href='https://doi.org/10.1093/dnares/dss006'>https://doi.org/10.1093/dnares/dss006</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/474767?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/474767?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/474767?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/474767?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/474767?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/474767?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/474767?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Characterization%20of%20Species-Specific%20Repeats%20in%20613%20Prokaryotic%20Species&amp;publicationDate=2012-02-24&amp;volumeNum=19&amp;issueNum=3&amp;author=Koressaar%2C%20Triinu%3B%20Remm%2C%20Maido&amp;startPage=219&amp;endPage=230&amp;contentId=10.1093%2Fdnares%2Fdss006&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f3%2f219%2f474767" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=7157471 id="7157471" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Prokaryotes are in general believed to possess small, compactly organized genomes, with repetitive sequences forming only a small part of them. Nonetheless, many prokaryotic genomes in fact contain species-specific repeats (&gt;85 bp long genomic sequences with less than 60% identity to other species) as we have previously demonstrated. However, it is not known at present how frequent such species-specific repeats are and what their functional roles in bacterial genomes may be. Therefore, we have conducted a comprehensive survey of prokaryotic species-specific repeats and characterized them to examine as to whether there are functional classes among different repeats or not and how they are mutually related to each other. Of the 613 distinct prokaryotic species analyzed, 97% were found to contain at least one species-specific repeats. It seems interesting to note that the species-specific repeats thus identified appear to be functionally variable in different genomes: in some genomes, they are mostly associated with duplicated protein-coding genes, whereas in some other genomes with rRNA and tRNA genes. Contrary to what may be expected, only one-fourth of the species-specific repeats were found to be associated with mobile genetic elements.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="species-specific">species-specific</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="repeat">repeat</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="identification">identification</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="prokaryote">prokaryote</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="pathogen">pathogen</a></div>                            <h2 scrollto-destination=7157473 id="7157473" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>To date, more than 1000 prokaryotic species have been completely sequenced. These genome data provide an opportunity to conduct large-scale genome-wide studies in comparative genomics. Many analyses are carried out to explore eukaryotic genomes side by side, but not many comparative analyses are performed with prokaryotes. Even though prokaryotic organisms demonstrate recognizable, common architectural principles, they have a great variability in ecology and in metabolic and genomic complexity.<sup><a reveal-id="DSS006C1" data-open="DSS006C1" class="link link-ref link-reveal xref-bibr">1</a></sup> For example, the lengths of sequenced bacterial genomes range from 180 kb (<em>Carsonella ruddii</em>)<sup><a reveal-id="DSS006C2" data-open="DSS006C2" class="link link-ref link-reveal xref-bibr">2</a></sup> to 13 Mb (<em>Sorangium cellulosum</em>).<sup><a reveal-id="DSS006C3" data-open="DSS006C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Although a lot of this variability can be attributed to varying metabolic complexity, it also suggests that some species have more compact genomes than others. Large differences exist in the fraction of intergenic regions.<sup><a reveal-id="DSS006C1" data-open="DSS006C1" class="link link-ref link-reveal xref-bibr">1</a></sup> The percentage of intergenic regions in the genomic sequence varies from ∼5% (<em>Thermotoga neapolitana</em>, NC_011978) to 50% (<em>Sodalis glossinidius</em>, NC_007712). The median value of the fraction of intergenic regions per genome size is 12%.</p><p>Though prokaryotic genomes are known to be compactly organized, there is still room for different repetitive sequences in them.<sup><a reveal-id="DSS006C4" data-open="DSS006C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSS006C5" data-open="DSS006C5" class="link link-ref link-reveal xref-bibr">5</a></sup> There are very few comprehensive studies of repetitive sequences; most of the studies focus on specific types of repeats or on a limited number of bacterial species.<sup><a reveal-id="DSS006C4  DSS006C8" data-open="DSS006C4  DSS006C8" class="link link-ref link-reveal xref-bibr">4–8</a></sup></p><p>The processes generating repeats include duplication, horizontal gene transfer, transposition, and replicon fusion.<sup><a reveal-id="DSS006C1" data-open="DSS006C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSS006C5" data-open="DSS006C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSS006C9  DSS006C17" data-open="DSS006C9  DSS006C17" class="link link-ref link-reveal xref-bibr">9–17</a></sup> These processes allow bacteria to adapt to environmental changes or to evolve into pathogens (owing to the distribution of genes encoding toxins, effector proteins, cell wall modification enzymes, fitness factors, and antibiotic and heavy metal resistance determinants).<sup><a reveal-id="DSS006C10" data-open="DSS006C10" class="link link-ref link-reveal xref-bibr">10</a></sup> A large fraction of known types of repetitive sequences belong to integrative and conjugative elements, also known as mobilome.<sup><a reveal-id="DSS006C18" data-open="DSS006C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSS006C9" data-open="DSS006C9" class="link link-ref link-reveal xref-bibr">9</a></sup> The mobilome consists of bacteriophages, plasmids, and transposable elements. It mediates the movement of DNA within and between genomes. Integrative elements play a key role in the emergence of infectious diseases, antibiotic resistance and other problems.<sup><a reveal-id="DSS006C1" data-open="DSS006C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSS006C19  DSS006C21" data-open="DSS006C19  DSS006C21" class="link link-ref link-reveal xref-bibr">19–21</a></sup></p><p>Thus, different mechanisms give rise to repeats and also different roles are suggested for repeats. Though many papers about completely sequenced prokaryotic genomes also try to analyze repetitive sequences, the characterization is frequently limited to some types of repeats (e.g. CRISPR, MITE, IS) or repeats as whole without classification.<sup><a reveal-id="DSS006C22" data-open="DSS006C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DSS006C23" data-open="DSS006C23" class="link link-ref link-reveal xref-bibr">23</a></sup> No exhaustive research about the characterization of prokaryotic species-specific repeats is published. However, as further shown, different types of species-specific repeats with different functions are present in prokaryotic genomes. At least some of them are the characteristic of a particular species (e.g. specific protein- biosynthesis-coding genes or specific bacterial toxins).</p><p>We have previously reported the existence of species-specific repeats in several bacterial species of medical importance and we have developed a computational method for <em>de novo</em> detection of such repeats previously.<sup><a reveal-id="DSS006C24" data-open="DSS006C24" class="link link-ref link-reveal xref-bibr">24</a></sup> However, two questions remain unanswered: do all species have species-specific repeats and can we associate these repeats with sequences other than mobile genetic elements?</p><p>The first topic we cover in this paper is the general description of prokaryotic repeats. It is important for species characterization and identification to know what fraction of a prokaryotic genome contains repetitive DNA regions, how many of them are species-specific, and what is the common size range of the repetitive sequences. Our main focus is on species-specific repeats, with a few comparisons with universal repeats (definitions of these repeat types are given below, at the beginning of the ‘Results’ section).</p><p>Thereafter, we focus on the functional characterization of species-specific repeats. The main interest here is whether species-specific repeats are reliable for use as target sequences in identifying a species or whether they are really only selfish elements or sequences with unknown function according to the common view. For this, different functional classes of repeats are distinguished. In this paper we use the following classes to describe repeats: (I) mobile genetic elements, (II) repeats associated with RNA genes, (III) repeats associated with protein-coding genes, and (IV) non-coding repeats.</p><p>Finally, we discuss the different practical aspects of detecting prokaryotic repetitive sequences. The <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary material</a> contains a list of all of the detected sequence-specific repeats.</p>                            <h2 scrollto-destination=7157482 id="7157482" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=7157483 id="7157483" class="section-title" data-legacy-id=s2a>2.1. Genomic sequences used</h3>
<p>To identify prokaryotic repeats, DNA sequences for all completely sequenced prokaryotic chromosomes and plasmids were retrieved from the FTP site of the NCBI RefSeq (<a class="link link-uri" href="http://ftp://ftp.ncbi.nih.gov/genomes/Bacteria">ftp://ftp.ncbi.nih.gov/genomes/Bacteria</a>). The human genome (version NCBI36) was downloaded from Ensembl (<a class="link link-uri" href="http://ftp://ensembl.org/pub/">ftp://ensembl.org/pub/</a>). A full list of analyzed genomes and their basic features (genome length, superkingdom, number of chromosomes, number of plasmids, etc.) is provided in the <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>. In total, 876 chromosomal genomes (63 archaeal and 819 eubacterial) and 613 prokaryotic species (containing 54 archaeal and 561 eubacterial genomes) were analyzed. Of the 613 species, 95 are represented with more than one strain of chromosomal genome sequence. For 232 prokaryotic species at least one plasmid genome was available, 733 prokaryotic plasmid genomes were downloaded in total. The RefSeq accession numbers of the virus genomes used in the current work are provided in the additional files (Additional File 1: <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). To find strain-specific repeats online, the NCBI BLAST database for microbes was used on genomic sequences with sequencing status ‘Complete, in Progress or Assembly’.</p>                            <h3 scrollto-destination=7157485 id="7157485" class="section-title" data-legacy-id=s2b>2.2. Annotation data</h3>
<p>To explore the positions of prokaryotic repeats in relation to genes, gene coordinates were retrieved from the FTP site of the NCBI RefSeq (<a class="link link-uri" href="http://ftp://ftp.ncbi.nih.gov/genomes/Bacteria">ftp://ftp.ncbi.nih.gov/genomes/Bacteria</a>) for all analyzed prokaryotic chromosomal genomes.</p>                            <h3 scrollto-destination=7157487 id="7157487" class="section-title" data-legacy-id=s2c>2.3. Method for identifying microbial repetitive sequences</h3>
<p>The method for finding microbial repeats has been described earlier.<sup><a reveal-id="DSS006C24" data-open="DSS006C24" class="link link-ref link-reveal xref-bibr">24</a></sup> The availability of complete genome sequences for the species of interest is required in order to find repetitive regions. The species of interest is called the ‘target genome’ in this article. In the context of species-specific repetitive sequences, the ‘non-target genome’ is defined as genomic DNA sequence(s) of all other organisms not in the target genome group. In the case of bacterial species-specific repeats, all sequenced chromosomal genomes of prokaryotic species and their plasmid genomes were used as non-target genomes. In this study, the human genome was always used by default as one of the non-target genomes.</p><p>To identify repeats, the genomic sequence of the target genome was segmented into consecutive windows. The length of a window was 100 bp and the length of overlap between consecutive windows was 50 bp. We also tested different length for the overlap (75 bp) to find species-specific repeats (50 randomly chosen species was used). However, no additional species-specific repeats could be found. Thus, to achieve smaller search time, we decided to use 50 bp for overlap. Matches of sequence windows inside the target genome were identified using the similarity search software BLAST. A sequence window was classified as a candidate repeat if the following two criteria were met: the length of the BLAST match in target genome was between 85 and 115 bp and the BLAST identity between the matching region and the query window was &gt;80%. Thereafter, in the case of finding species-specific repeats, the specificity of candidate repeats was checked. A candidate repeat was defined as a non-species-specific repeat if the length of any BLAST match in any non-target genome had a length of at least 50 bp and the identity between the match of background sequence and the query window was &gt;60%. Finally, overlapping candidate repeats were joined to form complete (species-specific) repeats. All scripts were written in the Perl programming language.</p>                            <h3 scrollto-destination=7157490 id="7157490" class="section-title" data-legacy-id=s2d>2.4. Definition of length and number of copies of the repeat</h3>
<p>The length of different copies of the same repeat is, to some extent, variable between species or between strains of the same species. In this analysis, the median length of all copies of a given repeat is shown as the length of the repeat. To define the copy number of a given species-specific repeat, the median value of all copy numbers of that repeat over the analyzed strains of that species was used. Similarly, the median copy number of particular repeat over different species where this repeat was present was used as the copy number of a universal repeat.</p>                            <h3 scrollto-destination=7157492 id="7157492" class="section-title" data-legacy-id=s2e>2.5. Repetitive sequence functionality analysis</h3>
<p>First, we identified prophage regions using the Phage_finder program.<sup><a reveal-id="DSS006C25" data-open="DSS006C25" class="link link-ref link-reveal xref-bibr">25</a></sup> After that, IS elements and transposons were identified from the ISfinder database<sup><a reveal-id="DSS006C26" data-open="DSS006C26" class="link link-ref link-reveal xref-bibr">26</a></sup> and from RefSeq annotations. Next, a BLAST search against plasmid sequences (2473 completely sequenced plasmid genomes downloaded from the NCBI FTP site <a class="link link-uri" href="http://ftp://ftp.ncbi.nih.gov/genomes/Plasmids">ftp://ftp.ncbi.nih.gov/genomes/Plasmids</a>) was conducted to find repeats of possible plasmidic origin. Clustered regularly interspaced short palindromic repeat (CRISPR) sequences were identified within repetitive sequences with CRISPRFinder.<sup><a reveal-id="DSS006C27" data-open="DSS006C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Intergenic repeat units (IRUs; ERIC/IRU sequences) were retrieved from the collection of short repeated palindromes (<a class="link link-uri" href="http://www.pasteur.fr/recherche/unites/pmtg/repet/intro.IRU.html">http://www.pasteur.fr/recherche/unites/pmtg/repet/intro.IRU.html</a>).</p><p>We used RefSeq gene annotation data for calculating how many species-specific and universal repetitive sequences cover genes (RNA genes and protein-coding genes) in more than 50% of the repeat length. Even if only one copy of a repeat contained a gene annotation then the repeat was considered a repeat associated with a gene (we assumed annotation faults are responsible for the missing annotation of other copies). The same logic was used to identify other features of the repeats except intergenic regions. None of the copies of intergenic repeats were allowed to contain any of the aforementioned features by more than 5% of the length of the repeated region.</p><p>Finally, Pfam database was used to predict the functions of NCBI RefSeq hypothetical proteins.<sup><a reveal-id="DSS006C28" data-open="DSS006C28" class="link link-ref link-reveal xref-bibr">28</a></sup></p>                            <h2 scrollto-destination=7157496 id="7157496" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=7157497 id="7157497" class="section-title" data-legacy-id=s3a>3.1. Definition of repeats</h3>
<p>Throughout this paper, ‘repeats’ are defined as DNA sequences at least 85 bp long that are present in at least two copies in the genome of each strain of the species. Repeat copies within the same species are required to have at least 80% identity with each other. ‘Species-specific repeat’ is defined as a repeat that is present in only one species and there are no similar sequences (more than 60% identity over 50 bp or more) in chromosomes of other fully sequenced species. If more than one strain of a species has been sequenced, the species-specific repeat must be present in all strains, with at least two copies. The minimum length and homology cutoffs for repeat definition in this paper were chosen for practical reasons—to allow the design of species-specific polymerase chain reaction (PCR) primers for each identified repeat.<sup><a reveal-id="DSS006C24" data-open="DSS006C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Our methodology is also able to identify ‘strain-specific repeats’, but we deliberately chose not to analyze these in order to maintain the focus of this article on species-specific repeats.</p><p>Some repeats are detected in more than one species. These can be further divided into ‘universal repeats’ and ‘intermediate repeats’. Universal repeat is a repeat that is present in at least two species (in both as a repeat) which must belong to different genera. Thus, universal repeats are repeats that are shared between some species of separate genera. For example, widely known universal repeats are ribosomal RNA genes and elongation factor Tu. All the rest of the repeats are defined as intermediate repeats. For example, intermediate repeat can be identified in one species as a repeat and can be present in one strain of another species with only one copy. Also intermediate repeats are repeats present in different species of one genus. In our analysis, we compare species-specific repeats to universal repeats; intermediate repeats are not analyzed further. Generally, intermediate repeats share characteristics of both universal and species-specific repeats.</p><p>An additional difficulty in repeat definition comes from the existence of bacterial plasmids. Typically, people search for repeats in bacterial chromosome(s), but some repeat copies can also be located in plasmids. In our data set, 232 of 613 species contained plasmids. Some of these plasmids also contained species-specific repeats. However, the analysis of species-specific plasmidic repeats is unequivocal because plasmid transfer between different bacterial species is a common phenomenon.<sup><a reveal-id="DSS006C29" data-open="DSS006C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DSS006C30" data-open="DSS006C30" class="link link-ref link-reveal xref-bibr">30</a></sup> It has been shown that at least 51% of sequenced proteobacterial plasmids are transmissible.<sup><a reveal-id="DSS006C9" data-open="DSS006C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Thus, the plasmid-based repeats are not reliable targets for characterization of bacterial species. For this reason, we have omitted plasmid sequences from our analysis and identified only the repeats that have multiple copies on chromosomes (no repetitive sequences were searched from species plasmid sequences).</p><p>As a first step, we analyzed the frequencies and lengths of all types of repeats in prokaryotic genomes. In the following analysis, we show the characteristic features of the species-specific repeats.</p>                            <h3 scrollto-destination=7157502 id="7157502" class="section-title" data-legacy-id=s3b>3.2. Characterization of repeats</h3>
                            <h4 scrollto-destination=7157503 id="7157503" class="section-title" data-legacy-id=s3b1>3.2.1. Almost all prokaryotes contain repetitive sequences</h4>
<p>In total, 613 microbial species comprising 876 different strains were analyzed. Repetitive DNA was detected in almost all species, with three exceptions: <em>Buchnera aphidicola</em> (six sequenced strains; NC_002528, NC_004061, NC_004545, NC_008513, NC_011833, and NC_011834), <em>Candidatus Carsonella ruddii</em> (NC_008512), and <em>Candidatus Hodgkinia cicadicola</em> (NC_012960). All these bacteria are symbionts with extremely small genomes.</p><p>The number of different repeats per genome was highly variable (Fig. <a reveal-id="DSS006F1" data-open="DSS006F1" class="link link-reveal link-table xref-fig">1</a>), ranging from 1 to 690 with the median 32 repeats per species. 95% of species contained fewer than 149 repeats. The highest number of repeats was found in the genome of the myxobacterium <em>S. cellulosum</em>' So ce 56′, which also has the largest size (13 Mb) of all the analyzed genomes. </p>                        <a id="7157506" scrollto-destination="7157506"></a>
<div id="DSS006F1" data-id="DSS006F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss006/2/m_dss00601.jpeg?Expires=2147483647&Signature=LLgFfVn0qBBX2IOqd47LWI-PH5tR3NLUqEnr0c2qjBN8ntjLNLDk2WHL8yVl~K0~13Rgei1~-7pIsfO6ClaMMaWerncrrAp-nLU23OlajZTxyAcS2OymrqFBGKXIvUcnaZUSyVfOIrIfrllfLT9s22I4d0Z8vvsHXqaJTL8kUVKMO66-bGTciek1fgaJg4rWAM5FpEXyOvCvEbqKRQ3SJmjAmG9tr5ww9mnK8U76i91I5ZqQMBo2LkVi4w8mY-F3~gV44XNvWx6qEJhMnrtsvulpZnWj9bA6Z~bVk1-a37SxeJeD9-D1PhgydvLQjQaXSIOFeMe6eZh1whzs2Er8rg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The number of repetitive sequences per species. Most of the species contain fewer than 50 repeats." path-from-xml="dss00601.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7157506/dss00601.jpeg" path-from-xml="dss00601.jpeg" target="_blank">View large</a><a section="7157506" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss006/2/dss00601.jpeg?Expires=2147483647&Signature=eIk0pguVYfCmffN1NhV76r~G8x-N~biGR2c90e0lIOGBnHspgBrgpqAYoGWDI~ne2oE5dLiYqcgZnwv4pMASKG-M5ka3MZKUSxzruDXDxpMI03QDHdEQTcX632SWl~bidZjnwCDfJJbFWQbCAABSaHkznCzwmBSHxanBbck5rbgVxRvlVCDM4aOsWbkWBfYOZ72f1Lfg-BtmBaNAnVLbL7YPCANhhRRCHuHx3emKGiKgjRfI0gVS4coq0VaIvG9xArn134BcoObrNa3eHn3Cjop6bRbyZnacTyDaeT2UvR24FGn-S6X5CenBKaa0dMt6V0V9J9-8gBa~KcC1Bir7RQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7157506&ar=474767&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00601.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The number of repetitive sequences per species. Most of the species contain fewer than 50 repeats.</p></div></div><div content-id="DSS006F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss006/2/m_dss00601.jpeg?Expires=2147483647&Signature=LLgFfVn0qBBX2IOqd47LWI-PH5tR3NLUqEnr0c2qjBN8ntjLNLDk2WHL8yVl~K0~13Rgei1~-7pIsfO6ClaMMaWerncrrAp-nLU23OlajZTxyAcS2OymrqFBGKXIvUcnaZUSyVfOIrIfrllfLT9s22I4d0Z8vvsHXqaJTL8kUVKMO66-bGTciek1fgaJg4rWAM5FpEXyOvCvEbqKRQ3SJmjAmG9tr5ww9mnK8U76i91I5ZqQMBo2LkVi4w8mY-F3~gV44XNvWx6qEJhMnrtsvulpZnWj9bA6Z~bVk1-a37SxeJeD9-D1PhgydvLQjQaXSIOFeMe6eZh1whzs2Er8rg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The number of repetitive sequences per species. Most of the species contain fewer than 50 repeats." path-from-xml="dss00601.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7157506/dss00601.jpeg" path-from-xml="dss00601.jpeg" target="_blank">View large</a><a section="7157506" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss006/2/dss00601.jpeg?Expires=2147483647&Signature=eIk0pguVYfCmffN1NhV76r~G8x-N~biGR2c90e0lIOGBnHspgBrgpqAYoGWDI~ne2oE5dLiYqcgZnwv4pMASKG-M5ka3MZKUSxzruDXDxpMI03QDHdEQTcX632SWl~bidZjnwCDfJJbFWQbCAABSaHkznCzwmBSHxanBbck5rbgVxRvlVCDM4aOsWbkWBfYOZ72f1Lfg-BtmBaNAnVLbL7YPCANhhRRCHuHx3emKGiKgjRfI0gVS4coq0VaIvG9xArn134BcoObrNa3eHn3Cjop6bRbyZnacTyDaeT2UvR24FGn-S6X5CenBKaa0dMt6V0V9J9-8gBa~KcC1Bir7RQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7157506&ar=474767&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00601.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The number of repetitive sequences per species. Most of the species contain fewer than 50 repeats.</p></div></div>                            <h4 scrollto-destination=7157507 id="7157507" class="section-title" data-legacy-id=s3b2>3.2.2. The fraction of genome sequence covered by repeats</h4>
<p>The median value of the repeat coverage was 1.8% of the bacterial genome. In only 12 bacterial species, repetitive sequences made up 7% of the total size of their genome (Fig. <a reveal-id="DSS006F2" data-open="DSS006F2" class="link link-reveal link-table xref-fig">2</a>). Most of these bacteria are pathogens (insect pathogen <em>Wolbachia</em> sp. wRi NC_012416, plant pathogens <em>Phytoplasma mali</em> NC_011047 and <em>Hamiltonella defensa</em> NC_012751, fish pathogen <em>Aliivibrio salmonicida</em> NC_011312 and NC_011313, and mammalian pathogens <em>Mycoplasma mycoides</em> NC_005364 and NC_015431, <em>Bartonella tribocorum</em> NC_010161, <em>Anaplasma phagocytophilum</em> NC_007797, <em>Bartonella grahamii</em> NC_012846, and <em>Bartonella henselae</em> NC_005956). In general, larger genomes tend to contain larger number of repeats and also a higher fraction of repetitive sequences from the genome sequence. </p>                        <a id="7157509" scrollto-destination="7157509"></a>
<div id="DSS006F2" data-id="DSS006F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss006/2/m_dss00602.jpeg?Expires=2147483647&Signature=iyeOXJ42v~0v4EQwgnxlLLAlN6QZaWl84B6tQnNjkbEl0WbzZoSz2gmspxl5j5teroD4eRs5a-QPjNLlTh45-5QuFMmOf0cLmTVh27UcL4JGtPBgAanswOuED7KfZ8fO2F7UJaMJEJeWurWCASdmmDzvtoT4UwJA8wk2QPv261zYYV0m8-p1aTMmM5-LXrUeIl2evqBna9j~Rt0qcznkbxy4NUBDQHBc8Ef3dMR-Jszt~fyRO~p2byTublni24mgw5jTU8XXzhe995kokpCBHhnrHjN8n-AWkD-KN4sQQZiyvAemeVbSmi7D~5LGqojTOVxUYywBrKgNW7Z0bwo7nA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The repeat coverage in bacterial genomes. In 95% of the species, the fraction of all repeats from the length of the genome is lower than 6%." path-from-xml="dss00602.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7157509/dss00602.jpeg" path-from-xml="dss00602.jpeg" target="_blank">View large</a><a section="7157509" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss006/2/dss00602.jpeg?Expires=2147483647&Signature=VVZnF9J9g1rMGFxdyotPNieeJgWLh8Fcss6N5CgUB62DFZpiMJXdxf4ERJA-Hi-m54NXwZeUT5Islf7~WdFI5CkfwCCQyD5A3zFuXdAo6PdrBB7bnE1iujlQSIUyk2LaoAAdMusi3GovrKq-rA8DNEFnnjHlgqxMYEMi3Zsa55U8rPpwAiBqWZMBb4guXBxADhh8mTIJ7cJerBWEU9bFgPaO9hOUdZKNmU-L1tm17DyHIf5nFyCLkVApwR4j3O8Fxl6wX~u7BeUCS4lKcG4L7eqde-XzvaEKILjyKCpGj7HRq39e3A-BpfsTOkCMj14qWeaL78nAXu~YOHcyzEql5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7157509&ar=474767&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00602.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The repeat coverage in bacterial genomes. In 95% of the species, the fraction of all repeats from the length of the genome is lower than 6%.</p></div></div><div content-id="DSS006F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss006/2/m_dss00602.jpeg?Expires=2147483647&Signature=iyeOXJ42v~0v4EQwgnxlLLAlN6QZaWl84B6tQnNjkbEl0WbzZoSz2gmspxl5j5teroD4eRs5a-QPjNLlTh45-5QuFMmOf0cLmTVh27UcL4JGtPBgAanswOuED7KfZ8fO2F7UJaMJEJeWurWCASdmmDzvtoT4UwJA8wk2QPv261zYYV0m8-p1aTMmM5-LXrUeIl2evqBna9j~Rt0qcznkbxy4NUBDQHBc8Ef3dMR-Jszt~fyRO~p2byTublni24mgw5jTU8XXzhe995kokpCBHhnrHjN8n-AWkD-KN4sQQZiyvAemeVbSmi7D~5LGqojTOVxUYywBrKgNW7Z0bwo7nA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The repeat coverage in bacterial genomes. In 95% of the species, the fraction of all repeats from the length of the genome is lower than 6%." path-from-xml="dss00602.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7157509/dss00602.jpeg" path-from-xml="dss00602.jpeg" target="_blank">View large</a><a section="7157509" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss006/2/dss00602.jpeg?Expires=2147483647&Signature=VVZnF9J9g1rMGFxdyotPNieeJgWLh8Fcss6N5CgUB62DFZpiMJXdxf4ERJA-Hi-m54NXwZeUT5Islf7~WdFI5CkfwCCQyD5A3zFuXdAo6PdrBB7bnE1iujlQSIUyk2LaoAAdMusi3GovrKq-rA8DNEFnnjHlgqxMYEMi3Zsa55U8rPpwAiBqWZMBb4guXBxADhh8mTIJ7cJerBWEU9bFgPaO9hOUdZKNmU-L1tm17DyHIf5nFyCLkVApwR4j3O8Fxl6wX~u7BeUCS4lKcG4L7eqde-XzvaEKILjyKCpGj7HRq39e3A-BpfsTOkCMj14qWeaL78nAXu~YOHcyzEql5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7157509&ar=474767&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00602.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The repeat coverage in bacterial genomes. In 95% of the species, the fraction of all repeats from the length of the genome is lower than 6%.</p></div></div><p>The fraction of pathogens among species with elevated percentages of repetitive sequences [9 of 12 species (75%) are pathogenic] is somewhat higher than one would expect. The fraction of pathogenic species among all analyzed genomes (all that are known to be pathogenic or non-pathogenic) is 37.7%. We suggest that pathogenic bacteria have a pressure to adapt to the environment more quickly and repetitive sequences may aid this or, alternatively, be the result of this. In many pathogenic bacteria, a fraction of repetitive sequences are associated with pathogenicity genes. For example, repetitive sequences in <em>A. phagocytophilum</em> (∼9% of the genome comprises repeats) contain genes functionally involved with the type IV secretion system. Also, repetitive sequences in <em>B. tribocorum</em> and <em>B. henselae</em> (∼11 and ∼7% of the genomes are repeats, respectively) contain several genes associated with pathogenicity—the Trw-conjugation system, filamentous hemagglutinin, specific adhesions, and components of the type IV secretion system. As many pathogenic processes remain uncharacterized, the fraction of genes involved in pathogenesis may be larger than we could interpret. An example of a non-pathogenic genome containing a high fraction of repeats is <em>Dehalococcoides ethenogenes</em>. It contains large duplications that include genes involved in the ability of dechlorinate groundwater pollutants. <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a> gives an overview of the numbers of identified repetitive sequences from all analyzed prokaryotic species.</p>                            <h4 scrollto-destination=7157511 id="7157511" class="section-title" data-legacy-id=s3b3>3.2.3. Most of the repeats are species-specific</h4>
<p>From a total of 33 921 different repeats found in this work, 29 771 (88%) were species-specific (Fig. <a reveal-id="DSS006F3" data-open="DSS006F3" class="link link-reveal link-table xref-fig">3</a>). The total number of universal repeats (those that are present in two or more different genera) found in this work was 554; approximately half of them were present in only two species. After analysis of the universal repeats, we estimated that only 0.25% (84 repeats) of all repeats were truly universal, present in more than six species. Measured in nucleotides, species-specific repeats constituted more than half of the total length of all repetitive sequences. Perhaps more informative is the number of nucleotides included in either universal or species-specific repeats. In Fig. <a reveal-id="DSS006F3" data-open="DSS006F3" class="link link-reveal link-table xref-fig">3</a> (second set of three columns), one can see that species-specific repeats constitute ca. 58% of the total nucleotides included in all repeats. In all analyzed genomes, there are three times more nucleotides in species-specific repeats than in universal repeats (from Fig. <a reveal-id="DSS006F3" data-open="DSS006F3" class="link link-reveal link-table xref-fig">3</a>, 58% divided to 19%). </p>                        <a id="7157513" scrollto-destination="7157513"></a>
<div id="DSS006F3" data-id="DSS006F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss006/2/m_dss00603.jpeg?Expires=2147483647&Signature=hibHqyRbggKUda2TAd9~npI962b4fGdQXAXM6UQbJip8lbM8nlRnrRIa8trK32a0fVwDThFFtop97DjNyubsYFX2inYDRg2xB~gX6UkeoP-9so1-FKiOUX7khoIOLqyoVcqkygAHNZrYeblGDJ2kVgee7z3mp~aC0x8~o9BeTkOicfpH6lbGGHJ~o3lldnvyIaSSNogtmRiNvHyQPiFjoerptUN7jvCdpZd3tl7D4tklwJ7~AlX7VX0UHLck6tlfGD2h0QNXfQNTZOvb9Dp-~5qex6fW-06XP0glLIHhNV11RwjdUv5mDAy2IdV1HihLidCqQXdX4vQMDmQN59QJnQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The frequencies of repeats considered in this study. The first three columns represent the percentage of the number of particular repeats from the total number of repeats found in this work. The second three columns show the percentage of repeats in nucleotides from the total length of repeats found in this work. This figure can be viewed in colour online." path-from-xml="dss00603.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7157513/dss00603.jpeg" path-from-xml="dss00603.jpeg" target="_blank">View large</a><a section="7157513" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss006/2/dss00603.jpeg?Expires=2147483647&Signature=1W2zIfnXN3P7OipyDYD8g71FO7EowXqSztJJP92A9XMJNMrPOc6QNFJlmdpuOh2O9izDZ3hE0AL9oQTk-3hPGN0kfLqfqe3cSMY1myfc-iCTXXaLTpVIMjY~2Z4AamzCKM0~xshqZrGbuYqW671IYoQ8FbUsS30JUVtPvroLrxwwRUhESm9gOuJGw5QWWua5FQ2EcLklzVxrreaD8JdCYx2eV1mOviYnPU9nNucr7cCCbR7yS1GR3XZD8xPZrP35hpPDOQx0Kbk8LkGp7p1dNLH7tW-LXagopXGOB6Ns6no8Pf5-VnSgFlASs4lUFWvxuDrUDTSTkwJ6Tk-5mb5lYw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7157513&ar=474767&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00603.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The frequencies of repeats considered in this study. The first three columns represent the percentage of the number of particular repeats from the total number of repeats found in this work. The second three columns show the percentage of repeats in nucleotides from the total length of repeats found in this work. This figure can be viewed in colour online.</p></div></div><div content-id="DSS006F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss006/2/m_dss00603.jpeg?Expires=2147483647&Signature=hibHqyRbggKUda2TAd9~npI962b4fGdQXAXM6UQbJip8lbM8nlRnrRIa8trK32a0fVwDThFFtop97DjNyubsYFX2inYDRg2xB~gX6UkeoP-9so1-FKiOUX7khoIOLqyoVcqkygAHNZrYeblGDJ2kVgee7z3mp~aC0x8~o9BeTkOicfpH6lbGGHJ~o3lldnvyIaSSNogtmRiNvHyQPiFjoerptUN7jvCdpZd3tl7D4tklwJ7~AlX7VX0UHLck6tlfGD2h0QNXfQNTZOvb9Dp-~5qex6fW-06XP0glLIHhNV11RwjdUv5mDAy2IdV1HihLidCqQXdX4vQMDmQN59QJnQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The frequencies of repeats considered in this study. The first three columns represent the percentage of the number of particular repeats from the total number of repeats found in this work. The second three columns show the percentage of repeats in nucleotides from the total length of repeats found in this work. This figure can be viewed in colour online." path-from-xml="dss00603.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7157513/dss00603.jpeg" path-from-xml="dss00603.jpeg" target="_blank">View large</a><a section="7157513" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss006/2/dss00603.jpeg?Expires=2147483647&Signature=1W2zIfnXN3P7OipyDYD8g71FO7EowXqSztJJP92A9XMJNMrPOc6QNFJlmdpuOh2O9izDZ3hE0AL9oQTk-3hPGN0kfLqfqe3cSMY1myfc-iCTXXaLTpVIMjY~2Z4AamzCKM0~xshqZrGbuYqW671IYoQ8FbUsS30JUVtPvroLrxwwRUhESm9gOuJGw5QWWua5FQ2EcLklzVxrreaD8JdCYx2eV1mOviYnPU9nNucr7cCCbR7yS1GR3XZD8xPZrP35hpPDOQx0Kbk8LkGp7p1dNLH7tW-LXagopXGOB6Ns6no8Pf5-VnSgFlASs4lUFWvxuDrUDTSTkwJ6Tk-5mb5lYw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7157513&ar=474767&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00603.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The frequencies of repeats considered in this study. The first three columns represent the percentage of the number of particular repeats from the total number of repeats found in this work. The second three columns show the percentage of repeats in nucleotides from the total length of repeats found in this work. This figure can be viewed in colour online.</p></div></div><p>Of the analyzed species, 97% contained at least one species-specific repeat. Species-specific repetitive DNA was not detected in 20 of 613 species (listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). There are several possible explanations as to why we were not able to detect species-specific repeats in these 20 species. For example, <em>B. aphidicola</em> (six sequenced strains in our data set) is very widely defined and contains many variable genomic sequences, not sharing sufficient similarity over their repeats. Two species (<em>Candidatus Carsonella ruddii</em> and <em>Candidatus Hodgkinia cicadicola</em>) contain no repeats at all, probably because of their small genomes. In the other species (e.g. <em>Brucella ovis</em>, <em>Brucella canis</em>, and <em>Brucella abortus</em>), the existence of closely related fully sequenced species makes it impossible to find truly species-specific repeats.</p>                            <h4 scrollto-destination=7157515 id="7157515" class="section-title" data-legacy-id=s3b4>3.2.4. The majority of species-specific repeats are shorter than 200 bp</h4>
<p>In Fig. <a reveal-id="DSS006F4" data-open="DSS006F4" class="link link-reveal link-table xref-fig">4</a>, the length distribution for species-specific repeats is shown. We found that 95% of species-specific repeats were shorter than 1000 bp. Universal repeats had a similar repeat length distribution. </p>                        <a id="7157517" scrollto-destination="7157517"></a>
<div id="DSS006F4" data-id="DSS006F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss006/2/m_dss00604.jpeg?Expires=2147483647&Signature=JXiU6IFYh~0IZg0a8HvzVsBuiRcV0rycwidFYHr1ODN2zrDUlts4Z9~uXAzybaeaZg2MS0xnWAmR-AVH0IguZnNCq4o7VQFvMB~ky6VexgRYAzCx2yYXRl4xZUoftuQ0NUehmPOpsc30IOkgSpsF8WDQNsPut58mEL-rsB05EZXKrk0uoo32z0oJEXeUIfJhpvQT0zQzNq9k71Pyr6nr0aZmZl0xtuTLUtp~94Q-tKVq8~NEEFanVAncEzjHLygTNtagOblv62xkHwt6l7EgRYbjlxE0kb6bKWpDnB1A-kEwi6L~MJRaFyhWYTLdV7VBtgDH~BvsY9ircRxzpRb7Vw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The length distribution of species-specific and universal repeats. Only repeats shorter than 2000 nucleotides are shown (which constitute ∼98% of all species-specific and ∼98% of all universal repeats). Light blue triangles represent species-specific repeats and dark blue squares represent universal repeats. y-axis represents the fraction of repeats of specific length to the total number of the repeats." path-from-xml="dss00604.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7157517/dss00604.jpeg" path-from-xml="dss00604.jpeg" target="_blank">View large</a><a section="7157517" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss006/2/dss00604.jpeg?Expires=2147483647&Signature=r~eOardGS~yIMLMNndr-6RGjn5FfoEz5uIfkDfZapSVag-QN9Xbfk4awhOzEDvToo42yWz5We-4a0rrbHCdj8kfl3zKcyqvj1SN-hfscB0Q1MM3KhSP-hpE6IoCy8KFkoILipf8KGFQBp-qS5GczgJrIWvC1yH3s7IfkYJo6t6Rc05TECIEop4vh4lz~TK2pgd2NsZdfn8XsPY-y9paSAZrsH2cw6Vx1Ou-r7fCps-29~KCK0ANPqIUh0L9xL3JGo-OxOTln6acT3ugQJVlM8z8HF7OHCQNPCnwyg13DRj-4Qb0pMZAJ66RfVA7DKy-QaTRivObzfS0h0G-9rcgZtw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7157517&ar=474767&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00604.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The length distribution of species-specific and universal repeats. Only repeats shorter than 2000 nucleotides are shown (which constitute ∼98% of all species-specific and ∼98% of all universal repeats). Light blue triangles represent species-specific repeats and dark blue squares represent universal repeats. <em>y</em>-axis represents the fraction of repeats of specific length to the total number of the repeats.</p></div></div><div content-id="DSS006F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss006/2/m_dss00604.jpeg?Expires=2147483647&Signature=JXiU6IFYh~0IZg0a8HvzVsBuiRcV0rycwidFYHr1ODN2zrDUlts4Z9~uXAzybaeaZg2MS0xnWAmR-AVH0IguZnNCq4o7VQFvMB~ky6VexgRYAzCx2yYXRl4xZUoftuQ0NUehmPOpsc30IOkgSpsF8WDQNsPut58mEL-rsB05EZXKrk0uoo32z0oJEXeUIfJhpvQT0zQzNq9k71Pyr6nr0aZmZl0xtuTLUtp~94Q-tKVq8~NEEFanVAncEzjHLygTNtagOblv62xkHwt6l7EgRYbjlxE0kb6bKWpDnB1A-kEwi6L~MJRaFyhWYTLdV7VBtgDH~BvsY9ircRxzpRb7Vw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The length distribution of species-specific and universal repeats. Only repeats shorter than 2000 nucleotides are shown (which constitute ∼98% of all species-specific and ∼98% of all universal repeats). Light blue triangles represent species-specific repeats and dark blue squares represent universal repeats. y-axis represents the fraction of repeats of specific length to the total number of the repeats." path-from-xml="dss00604.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7157517/dss00604.jpeg" path-from-xml="dss00604.jpeg" target="_blank">View large</a><a section="7157517" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss006/2/dss00604.jpeg?Expires=2147483647&Signature=r~eOardGS~yIMLMNndr-6RGjn5FfoEz5uIfkDfZapSVag-QN9Xbfk4awhOzEDvToo42yWz5We-4a0rrbHCdj8kfl3zKcyqvj1SN-hfscB0Q1MM3KhSP-hpE6IoCy8KFkoILipf8KGFQBp-qS5GczgJrIWvC1yH3s7IfkYJo6t6Rc05TECIEop4vh4lz~TK2pgd2NsZdfn8XsPY-y9paSAZrsH2cw6Vx1Ou-r7fCps-29~KCK0ANPqIUh0L9xL3JGo-OxOTln6acT3ugQJVlM8z8HF7OHCQNPCnwyg13DRj-4Qb0pMZAJ66RfVA7DKy-QaTRivObzfS0h0G-9rcgZtw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7157517&ar=474767&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00604.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The length distribution of species-specific and universal repeats. Only repeats shorter than 2000 nucleotides are shown (which constitute ∼98% of all species-specific and ∼98% of all universal repeats). Light blue triangles represent species-specific repeats and dark blue squares represent universal repeats. <em>y</em>-axis represents the fraction of repeats of specific length to the total number of the repeats.</p></div></div><p>Although most repeats are very short, many long species-specific repeats also exist in prokaryotic genomes. The longest species-specific repeat (36 kb) belongs to <em>Alkaliphilus metalliredigens</em> (NC_009633), which has a genome size of 4.9 Mb. The repeat has three copies overlapping 178 gene sequences with different RefSeq gene functions, mostly annotated as hypothetical proteins. Part of this repeat (23 kb of 36 kb) contains a prophage sequence identified by the computer software Phage_finder.<sup><a reveal-id="DSS006C25" data-open="DSS006C25" class="link link-ref link-reveal xref-bibr">25</a></sup> The remaining sequences of this repeat may also have a phage origin as the genes present in this region are associated with regulation and initiation of transcription, cell wall component breakdown, and regeneration.</p>                            <h4 scrollto-destination=7157519 id="7157519" class="section-title" data-legacy-id=s3b5>3.2.5. Most of the species-specific repeats have two copies</h4>
<p>As the copy number of a species-specific repeat might differ between different strains of the same species, the median value of copies of particular repeat in the analyzed strains of particular species was used as the copy number of a given repeat. Similarly, the median copy number of particular repeat in different species was used as a copy number of a universal repeat.</p><p>Approximately 77% of species-specific repeats had two copies per repeat (Fig. <a reveal-id="DSS006F5" data-open="DSS006F5" class="link link-reveal link-table xref-fig">5</a>). Only a small fraction had a large number of copies per repeat. For example, only 0.4% of species-specific repeats had more than 20 copies per repeat. The maximum number of copies of a species-specific repeat was identified in the marine bacterium <em>Hahella chejuensis</em> KCTC 2396 (NC_007645, genome length 7.2 Mb) with 111 copies. The median length of the copies of this repeat was 141 bp. The copies are dispersed over the genome and the origin of this sequence is unclear. The fact that several copies overlap by a few nucleotides at the 5′ ends of different genes indicates that the repeat may play a role in the regulation of transcription. An overview of the copy numbers for all identified repeats is shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>. </p>                        <a id="7157522" scrollto-destination="7157522"></a>
<div id="DSS006F5" data-id="DSS006F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss006/2/m_dss00605.jpeg?Expires=2147483647&Signature=Wtu0QrN1g6UXm~ILShvoKIZT3y82kcyBj6RF4DqAiNOX3J2fFGsZldsdlaRWgY-YAgmlDXFqcWzPDBnOQ9XLj1bvSzczO-IFE9Nr5K2Pb6k9TYPtN7QxFr-PDEkZ8hi9LgwYKFtNFfoB3bebHUh7xiWN9EJ0VJ-Wqmi8ldQJ4pjKX4vIuYhkA72rSDhOPuTHx-r6NiAZsdxMXqISoTkY97bGgHmutU1uS2Pn2xJ4MHFAmFDw3WVIDuydfDY6dMlRsp03dTvkvqKpx7b-RZ-sXWm~JHHplPEFt262IaULMHYt2WrYqYSdX00xHa3CmRG287g-FgmWSmBSklJfgxyTrA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The distribution of the number of copies per species-specific repeat. Although the number of copies per species-specific repeat in different species varies, the median value (rounded down to the nearest integer) over the number of copies per particular repeat is considered. The number above the column shows the percentage of prevalence of particular number of copies per repeat from the total number of repeats. Only the values over 1% are shown. y-axis is on a logarithmic scale." path-from-xml="dss00605.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7157522/dss00605.jpeg" path-from-xml="dss00605.jpeg" target="_blank">View large</a><a section="7157522" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss006/2/dss00605.jpeg?Expires=2147483647&Signature=ko3Nt7T9wk7bbIpJjngBl1J3ZYTsu3dmTlVbG5EHWtDuQqdBzhetKb9vufAuZfScYReS2Mh8AjAmnqsEXS9ejBIko9DlPsZCL7OH9tPSPyDZF5oXV52OMJJArF5yM08oKLpvadW0MMWlbtRKJJqvoTtG9EfuGfk13EsM2C7YvWU3Jsyj16xlonlEaIJgIbtJAggb51-HEltm0xM6eH5bEAZ4VAY81K2u71k7UvRfXf~BT4CaPBfaumXKr5S05zu12HBEEbqkfllsKuqZvwuKiF1uv4-TQXxtUWIeewFLDSlmO~WMv8IOj-l9DXKO7CTezNXqzQaZlHDXAmBaJxiUHA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7157522&ar=474767&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00605.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The distribution of the number of copies per species-specific repeat. Although the number of copies per species-specific repeat in different species varies, the median value (rounded down to the nearest integer) over the number of copies per particular repeat is considered. The number above the column shows the percentage of prevalence of particular number of copies per repeat from the total number of repeats. Only the values over 1% are shown. <em>y</em>-axis is on a logarithmic scale.</p></div></div><div content-id="DSS006F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss006/2/m_dss00605.jpeg?Expires=2147483647&Signature=Wtu0QrN1g6UXm~ILShvoKIZT3y82kcyBj6RF4DqAiNOX3J2fFGsZldsdlaRWgY-YAgmlDXFqcWzPDBnOQ9XLj1bvSzczO-IFE9Nr5K2Pb6k9TYPtN7QxFr-PDEkZ8hi9LgwYKFtNFfoB3bebHUh7xiWN9EJ0VJ-Wqmi8ldQJ4pjKX4vIuYhkA72rSDhOPuTHx-r6NiAZsdxMXqISoTkY97bGgHmutU1uS2Pn2xJ4MHFAmFDw3WVIDuydfDY6dMlRsp03dTvkvqKpx7b-RZ-sXWm~JHHplPEFt262IaULMHYt2WrYqYSdX00xHa3CmRG287g-FgmWSmBSklJfgxyTrA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The distribution of the number of copies per species-specific repeat. Although the number of copies per species-specific repeat in different species varies, the median value (rounded down to the nearest integer) over the number of copies per particular repeat is considered. The number above the column shows the percentage of prevalence of particular number of copies per repeat from the total number of repeats. Only the values over 1% are shown. y-axis is on a logarithmic scale." path-from-xml="dss00605.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7157522/dss00605.jpeg" path-from-xml="dss00605.jpeg" target="_blank">View large</a><a section="7157522" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss006/2/dss00605.jpeg?Expires=2147483647&Signature=ko3Nt7T9wk7bbIpJjngBl1J3ZYTsu3dmTlVbG5EHWtDuQqdBzhetKb9vufAuZfScYReS2Mh8AjAmnqsEXS9ejBIko9DlPsZCL7OH9tPSPyDZF5oXV52OMJJArF5yM08oKLpvadW0MMWlbtRKJJqvoTtG9EfuGfk13EsM2C7YvWU3Jsyj16xlonlEaIJgIbtJAggb51-HEltm0xM6eH5bEAZ4VAY81K2u71k7UvRfXf~BT4CaPBfaumXKr5S05zu12HBEEbqkfllsKuqZvwuKiF1uv4-TQXxtUWIeewFLDSlmO~WMv8IOj-l9DXKO7CTezNXqzQaZlHDXAmBaJxiUHA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7157522&ar=474767&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00605.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The distribution of the number of copies per species-specific repeat. Although the number of copies per species-specific repeat in different species varies, the median value (rounded down to the nearest integer) over the number of copies per particular repeat is considered. The number above the column shows the percentage of prevalence of particular number of copies per repeat from the total number of repeats. Only the values over 1% are shown. <em>y</em>-axis is on a logarithmic scale.</p></div></div><p>In general, shorter repeats tend to have more copies (Fig. <a reveal-id="DSS006F6" data-open="DSS006F6" class="link link-reveal link-table xref-fig">6</a>). However, Fig. <a reveal-id="DSS006F6" data-open="DSS006F6" class="link link-reveal link-table xref-fig">6</a> also shows somewhat higher copy numbers for species-specific repeats with lengths between 1000 and 2000 bp. This is due to species-specific transposons that contribute to the higher copy numbers in the length range between 900 and 1900 nucleotides. </p>                        <a id="7157524" scrollto-destination="7157524"></a>
<div id="DSS006F6" data-id="DSS006F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss006/2/m_dss00606.jpeg?Expires=2147483647&Signature=le~ogDg5Vtv3-m7~sZfRcix7JES6iXQf6R~KUnxe7NFR-2twLg0GFQU~2MjF2bvLAQIb24ubEuwazg~EWGp6hIfFws3VsnifbuPodCrtWgPrxg9Pk0qzq4p3G7lyRc1palf~Dm4YukMVXgb72sJvzPYOU2nC90kd9rEeZLADtOrppqRgkOOtKcGnr0jbofhvDyC8-s13z6sDUoy6050VEia~kT~pg9QyMFMGipH4GRAZyPO4cDeIgiSMPNHKV6SPaq7jr2RQIaoNzQhtoIkSf3jLb-Wq5RgvGAymqBPaQ4wS1DpO2owzex5gSmTM0QgsZ64m1CtOmhMWXdUSm5TsZA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between the length of species-specific repeat and the number of copies per species-specific repeat. Only repeats up to 3000 nucleotides are shown. y-axis is on a logarithmic scale." path-from-xml="dss00606.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7157524/dss00606.jpeg" path-from-xml="dss00606.jpeg" target="_blank">View large</a><a section="7157524" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss006/2/dss00606.jpeg?Expires=2147483647&Signature=W~nvbAaxE8IvQonxT4LbSSloQevrHciT0CojDN4uOzXCYoX~OX1nq~Gkf5zPCw0vACZ12XSKNuwIIXKTjgcqIE0eL04LQ7Fj2HjOgClQHg7W3ZWEdknRY25RHy-Yot8kc-eL2FNwL-se43rP6oLxA1hGDHXckLFuJ4CaFnHYZ-bJoCQpon-9mLO8FKhs3voYw8zME82uqT2Njlg4Ux2zcYifQCt7fGOAX0EU8JeIsVuFw2QGY1NM5bh5BMNUZTZw1iFNOfQnuSSjRt~9I9~8ZPFzunKqQd4w4PWPtnQX3iH~3mgeyO6~uZSpoHfWujUKbqULLZBa5ohP0yOoF~BW3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7157524&ar=474767&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00606.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between the length of species-specific repeat and the number of copies per species-specific repeat. Only repeats up to 3000 nucleotides are shown. <em>y</em>-axis is on a logarithmic scale.</p></div></div><div content-id="DSS006F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss006/2/m_dss00606.jpeg?Expires=2147483647&Signature=le~ogDg5Vtv3-m7~sZfRcix7JES6iXQf6R~KUnxe7NFR-2twLg0GFQU~2MjF2bvLAQIb24ubEuwazg~EWGp6hIfFws3VsnifbuPodCrtWgPrxg9Pk0qzq4p3G7lyRc1palf~Dm4YukMVXgb72sJvzPYOU2nC90kd9rEeZLADtOrppqRgkOOtKcGnr0jbofhvDyC8-s13z6sDUoy6050VEia~kT~pg9QyMFMGipH4GRAZyPO4cDeIgiSMPNHKV6SPaq7jr2RQIaoNzQhtoIkSf3jLb-Wq5RgvGAymqBPaQ4wS1DpO2owzex5gSmTM0QgsZ64m1CtOmhMWXdUSm5TsZA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between the length of species-specific repeat and the number of copies per species-specific repeat. Only repeats up to 3000 nucleotides are shown. y-axis is on a logarithmic scale." path-from-xml="dss00606.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7157524/dss00606.jpeg" path-from-xml="dss00606.jpeg" target="_blank">View large</a><a section="7157524" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss006/2/dss00606.jpeg?Expires=2147483647&Signature=W~nvbAaxE8IvQonxT4LbSSloQevrHciT0CojDN4uOzXCYoX~OX1nq~Gkf5zPCw0vACZ12XSKNuwIIXKTjgcqIE0eL04LQ7Fj2HjOgClQHg7W3ZWEdknRY25RHy-Yot8kc-eL2FNwL-se43rP6oLxA1hGDHXckLFuJ4CaFnHYZ-bJoCQpon-9mLO8FKhs3voYw8zME82uqT2Njlg4Ux2zcYifQCt7fGOAX0EU8JeIsVuFw2QGY1NM5bh5BMNUZTZw1iFNOfQnuSSjRt~9I9~8ZPFzunKqQd4w4PWPtnQX3iH~3mgeyO6~uZSpoHfWujUKbqULLZBa5ohP0yOoF~BW3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7157524&ar=474767&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00606.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between the length of species-specific repeat and the number of copies per species-specific repeat. Only repeats up to 3000 nucleotides are shown. <em>y</em>-axis is on a logarithmic scale.</p></div></div><p>From the repeats analyzed in this study, ∼11% of species-specific repeats were located in the genome in tandem. Interestingly, species-specific repeats were much more frequently located in tandem than universal repeats, ∼2% of which were located in tandem.</p>                            <h3 scrollto-destination=7157526 id="7157526" class="section-title" data-legacy-id=s3c>3.3. Functional analysis of species-specific repeats</h3>
                            <h4 scrollto-destination=7157527 id="7157527" class="section-title" data-legacy-id=s3c1>3.3.1. A large fraction of the species-specific repeats are associated with protein-coding genes</h4>
<p>We analyzed 29 771 species-specific repeats in order to understand the possible functions of repetitive sequences. We were interested in the general functional categories encoded within species-specific and universal repeats.</p><p>Here, we have used the following classification of bacterial repetitive elements: (I) mobile genetic elements such as insertion sequences (ISs), transposons, phage sequences, and plasmid sequences; (II) RNA genes; (III) protein-coding genes; (IV) non-coding short interspersed repeats (ERIC/IRU, CRISPR, intergenic regions). All these repeat classes were represented among the species-specific repeats (Fig. <a reveal-id="DSS006F7" data-open="DSS006F7" class="link link-reveal link-table xref-fig">7</a>). The protein-coding genes (class III) were the most abundant class (64% of all repeats). Mobile genetic elements (class I) and non-coding short interspersed repeats (class IV) both constituted 13–14% of all species-specific repeats. The smallest class (class II) of repeats included tRNA and rRNA genes. Approximately 6% of species-specific repeats could not be classified with our methodology. </p>                        <a id="7157530" scrollto-destination="7157530"></a>
<div id="DSS006F7" data-id="DSS006F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss006/2/m_dss00607.jpeg?Expires=2147483647&Signature=pcTyhYo9C1CI07tBLwVHL9pPHlrk5qpzsaFajKrku5RG6yH1GatdazWefXMsnaRPUoKd542xZ~-479UYni86RAkryVgScRBY~qut60bsUin4Cwre8sPLsmciJ-PUc8GcJbifdEerZByMwuw--deU2XfSCo3vUPP3ZaBoD9Vg-VUhMi6kIyCI1ArUkKcZ68LV9uI~aZuLcrkoZtn-6Mf~VjmY-cwY9CRpVteaRnGdAVD7SbAF6vGZ0K~G-0QEz5yGSSFZP8cIFlgT70D9FPh8FxnAp-RtUHQ2QKvtQaUXTyh9jEZ1Ub3Ndcn2Xgn6kqa7cieZt~DVLmERrV0FiyM-nw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Different classes of species-specific repeats. (A) Repeat frequencies of species-specific repeats. (B) Repeat coverage of species-specific repeats. As ERIC/IRU sequences constitute under 0.03% of all species-specific repeats, they are not shown." path-from-xml="dss00607.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7157530/dss00607.jpeg" path-from-xml="dss00607.jpeg" target="_blank">View large</a><a section="7157530" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss006/2/dss00607.jpeg?Expires=2147483647&Signature=QXjQ8EFjdbV7Wic20fILwT3QypxZzpQNMFt-30DC67zaMT~-29L27-QKXGl~iU-KX4aILNwoFVW5dvB7xl3ArHYc7THYOxbbDgevfZ6e4hRiqGGHLPnBHhqc346fQCGSHXyFp7nf1Ofn7vB-Xs3snFIuTvT0AX9mBdqyR5ffl7cyb2vZpbNlrBEKroCara1Lh4667wUVmKO6Ck9wKMQe242mrFA10l0RDD0g0zPEPsEQvpDrXEimB0mCmFHDpIm6c8H2O5UlfIb1zsdX1m5foyKhVE2ld2T1reDXWHRVkbFPjrnR4p0d6-WapfFvDKXj-DZs4xT~8MY-4i4B1qjz3w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7157530&ar=474767&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00607.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Different classes of species-specific repeats. (A) Repeat frequencies of species-specific repeats. (B) Repeat coverage of species-specific repeats. As ERIC/IRU sequences constitute under 0.03% of all species-specific repeats, they are not shown.</p></div></div><div content-id="DSS006F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss006/2/m_dss00607.jpeg?Expires=2147483647&Signature=pcTyhYo9C1CI07tBLwVHL9pPHlrk5qpzsaFajKrku5RG6yH1GatdazWefXMsnaRPUoKd542xZ~-479UYni86RAkryVgScRBY~qut60bsUin4Cwre8sPLsmciJ-PUc8GcJbifdEerZByMwuw--deU2XfSCo3vUPP3ZaBoD9Vg-VUhMi6kIyCI1ArUkKcZ68LV9uI~aZuLcrkoZtn-6Mf~VjmY-cwY9CRpVteaRnGdAVD7SbAF6vGZ0K~G-0QEz5yGSSFZP8cIFlgT70D9FPh8FxnAp-RtUHQ2QKvtQaUXTyh9jEZ1Ub3Ndcn2Xgn6kqa7cieZt~DVLmERrV0FiyM-nw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Different classes of species-specific repeats. (A) Repeat frequencies of species-specific repeats. (B) Repeat coverage of species-specific repeats. As ERIC/IRU sequences constitute under 0.03% of all species-specific repeats, they are not shown." path-from-xml="dss00607.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7157530/dss00607.jpeg" path-from-xml="dss00607.jpeg" target="_blank">View large</a><a section="7157530" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss006/2/dss00607.jpeg?Expires=2147483647&Signature=QXjQ8EFjdbV7Wic20fILwT3QypxZzpQNMFt-30DC67zaMT~-29L27-QKXGl~iU-KX4aILNwoFVW5dvB7xl3ArHYc7THYOxbbDgevfZ6e4hRiqGGHLPnBHhqc346fQCGSHXyFp7nf1Ofn7vB-Xs3snFIuTvT0AX9mBdqyR5ffl7cyb2vZpbNlrBEKroCara1Lh4667wUVmKO6Ck9wKMQe242mrFA10l0RDD0g0zPEPsEQvpDrXEimB0mCmFHDpIm6c8H2O5UlfIb1zsdX1m5foyKhVE2ld2T1reDXWHRVkbFPjrnR4p0d6-WapfFvDKXj-DZs4xT~8MY-4i4B1qjz3w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7157530&ar=474767&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00607.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Different classes of species-specific repeats. (A) Repeat frequencies of species-specific repeats. (B) Repeat coverage of species-specific repeats. As ERIC/IRU sequences constitute under 0.03% of all species-specific repeats, they are not shown.</p></div></div><p>For comparison, the universal repeats (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>) contained mainly mobilome elements (class I), some long regions of universally repeated rRNA genes (class II), and ∼40% were protein-coding genes (class III).</p><p>We identified that 19% of species-specific repeats belong to the class of non-coding short intergenic repeats and repeats with unknown origin. A fraction of these repeats may comprise a certain conserved regulatory elements. For example, IRUs and specific intergenic repeat motifs (CIR motifs, &gt;100 bp) are supposedly associated with gene conversion or with regulatory or structural requirements of the bacterial genomes.<sup><a reveal-id="DSS006C31  DSS006C33" data-open="DSS006C31  DSS006C33" class="link link-ref link-reveal xref-bibr">31–33</a></sup> In our data set, we found that more than half of the species (365 species) contain at least one species-specific repeat which is located prior to, following to, or between RNA gene sequences (e.g. between 23S rRNA and 5S rRNA gene sequences, between different tRNA sequences, etc). The overlap between the gene sequence and the repeat constitutes a small extent from repeat/gene sequence. Thus, in many cases, sequences between RNA genes may characterize a species uniquely. Still, many intergenic repeats have no associated function and are considered to be predominantly genomic parasites.<sup><a reveal-id="DSS006C34" data-open="DSS006C34" class="link link-ref link-reveal xref-bibr">34</a>,<a reveal-id="DSS006C35" data-open="DSS006C35" class="link link-ref link-reveal xref-bibr">35</a></sup></p><p>An overview of the most frequent functions of species-specific repeats associated with genes (class III) is shown in Table <a reveal-id="DSS006TB1" data-open="DSS006TB1" class="link link-reveal link-table xref-fig">1</a>. A notable group of species-specific repeated genes contains membrane proteins, transport proteins, and repeat-containing proteins. High sequence variation of these genes between species may be related to the need to adapt to specific environments or the need of pathogenic microbes to vary their immunogenic properties. The largest group, 31% of all repeat-associated genes, is annotated as ‘hypothetical protein’ by NCBI RefSeq. We analyzed the regions of hypothetical proteins containing repeats against Pfam database and found Pfam entries for 28% (2372 repeats) of them. Over the half of the repetitive sequences searched for are found from automatically generated Pfam-B families (866 families) or from the curated Pfam-A collection (471 families) predicted as proteins of unknown function. The rest of the matches represent variable functions of proteins. For example, the highest number of matches (for 48 repeats) was gained from the tetratricopeptide repeat superfamily, which are found from numerous and diverse proteins involved in such functions as cell cycle regulation, transcriptional control, mitochondrial and peroxisomal protein transport, neurogenesis, and protein folding. </p>                        <a id="7157537" scrollto-destination="7157537"></a>
<div content-id="DSS006TB1" class="table-modal"><div class="table-wrap"><div id="DSS006TB1" class="table-wrap-title" data-id="DSS006TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Classification of genes containing species-specific repeats</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene class </th><th>Number of repeats contained in the gene </th><th>Gene class </th><th>Number of repeats contained in the gene </th></tr></thead><tbody><tr><td>Hypothetical protein </td><td>8264 </td><td>Kinase </td><td>497 </td></tr><tr><td>Transport protein </td><td>1711 </td><td>ATP-dependent protein </td><td>453 </td></tr><tr><td>Pseudogene </td><td>1212 </td><td>Transcriptional regulator </td><td>431 </td></tr><tr><td>Membrane protein </td><td>656 </td><td>Synthetase </td><td>337 </td></tr><tr><td>Hydrogenase </td><td>603 </td><td>Bacterial chemotaxis </td><td>272 </td></tr><tr><td>Repeat containing protein </td><td>548 </td><td>Hydrolase </td><td>254 </td></tr><tr><td>Synthase </td><td>504 </td><td> </td><td> </td></tr></tbody></table></div><div content-id="DSS006TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene class </th><th>Number of repeats contained in the gene </th><th>Gene class </th><th>Number of repeats contained in the gene </th></tr></thead><tbody><tr><td>Hypothetical protein </td><td>8264 </td><td>Kinase </td><td>497 </td></tr><tr><td>Transport protein </td><td>1711 </td><td>ATP-dependent protein </td><td>453 </td></tr><tr><td>Pseudogene </td><td>1212 </td><td>Transcriptional regulator </td><td>431 </td></tr><tr><td>Membrane protein </td><td>656 </td><td>Synthetase </td><td>337 </td></tr><tr><td>Hydrogenase </td><td>603 </td><td>Bacterial chemotaxis </td><td>272 </td></tr><tr><td>Repeat containing protein </td><td>548 </td><td>Hydrolase </td><td>254 </td></tr><tr><td>Synthase </td><td>504 </td><td> </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The number shows how many repeats are overlapping the gene by at least 50% of the length of a given repeat. The total number of repeats overlapping gene sequences is 18 961, of which 15 742 repeats are shown in this table.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7157537" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS006TB1" class="table-wrap-title" data-id="DSS006TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Classification of genes containing species-specific repeats</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene class </th><th>Number of repeats contained in the gene </th><th>Gene class </th><th>Number of repeats contained in the gene </th></tr></thead><tbody><tr><td>Hypothetical protein </td><td>8264 </td><td>Kinase </td><td>497 </td></tr><tr><td>Transport protein </td><td>1711 </td><td>ATP-dependent protein </td><td>453 </td></tr><tr><td>Pseudogene </td><td>1212 </td><td>Transcriptional regulator </td><td>431 </td></tr><tr><td>Membrane protein </td><td>656 </td><td>Synthetase </td><td>337 </td></tr><tr><td>Hydrogenase </td><td>603 </td><td>Bacterial chemotaxis </td><td>272 </td></tr><tr><td>Repeat containing protein </td><td>548 </td><td>Hydrolase </td><td>254 </td></tr><tr><td>Synthase </td><td>504 </td><td> </td><td> </td></tr></tbody></table></div><div content-id="DSS006TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene class </th><th>Number of repeats contained in the gene </th><th>Gene class </th><th>Number of repeats contained in the gene </th></tr></thead><tbody><tr><td>Hypothetical protein </td><td>8264 </td><td>Kinase </td><td>497 </td></tr><tr><td>Transport protein </td><td>1711 </td><td>ATP-dependent protein </td><td>453 </td></tr><tr><td>Pseudogene </td><td>1212 </td><td>Transcriptional regulator </td><td>431 </td></tr><tr><td>Membrane protein </td><td>656 </td><td>Synthetase </td><td>337 </td></tr><tr><td>Hydrogenase </td><td>603 </td><td>Bacterial chemotaxis </td><td>272 </td></tr><tr><td>Repeat containing protein </td><td>548 </td><td>Hydrolase </td><td>254 </td></tr><tr><td>Synthase </td><td>504 </td><td> </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The number shows how many repeats are overlapping the gene by at least 50% of the length of a given repeat. The total number of repeats overlapping gene sequences is 18 961, of which 15 742 repeats are shown in this table.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7157537" target="_blank">View Large</a></div></div>                            <h4 scrollto-destination=7157538 id="7157538" class="section-title" data-legacy-id=s3c2>3.3.2. The function of repetitive full-length species-specific genes remains unknown</h4>
<p>We analyzed repeats that contain full-length protein-coding gene sequences that are represented in at least two copies of a particular repeat. We found that 1034 repetitive sequences contain full-length species-specific protein-coding genes in at least two copies of that repeat, resulting in 4001 different genes in total. Most of these genes (56.7 versus the 30.6% expected on the basis of RefSeq annotations of prokaryotic protein-coding genes) are hypothetical proteins (statistically over-represented <em>P</em> &lt; 0.0001).</p><p>However, several interesting details emerged. For example, we found that pseudogenes are statistically significantly over-represented (<em>P</em> &lt; 0.0001). The RefSeq annotations of prokaryotic protein-coding genes imply that on average ∼1.68% of randomly chosen protein-coding genes are pseudogenes. In contrast, we found that 6.0% of the analyzed genes are pseudogenes. We suggest that this can be explained by gene duplication. It is known that gene duplication is a significant contributor to the evolution of genomes. In some cases, one duplicated copy may become non-functionalized by randomly accumulating degenerative mutations in the absence of selective advantage.<sup><a reveal-id="DSS006C36" data-open="DSS006C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Another notable group of genes, encoding transport proteins, is under-represented (<em>P</em> &lt; 0.0001) among the analyzed genes (12.6 and 5.8% are the percentages of expected and observed transport proteins, respectively). It is shown that at least some transport proteins are conserved among a wide range of species.<sup><a reveal-id="DSS006C37" data-open="DSS006C37" class="link link-ref link-reveal xref-bibr">37</a>,<a reveal-id="DSS006C38" data-open="DSS006C38" class="link link-ref link-reveal xref-bibr">38</a></sup> This is also supported by our analysis of universal repeats where transport proteins are over-represented.</p>                            <h4 scrollto-destination=7157541 id="7157541" class="section-title" data-legacy-id=s3c3>3.3.3. Example of species-specific repeats of <em>H. defensa</em></h4>
<p><em>Hamiltonella defensa</em> (NC_012751) is an endosymbiont found in sap-feeding insects, including aphids, psyllids, and whiteflies. <em>Hamiltonella defensa</em> is an example of species containing relatively various repeat classes, whereas many species contain only one or two different classes of repeats.</p><p>The complete list of species-specific repeats in <em>H. defensa</em> contains 95 repeats (plus three universal repeats that contain rRNA and/or tRNA sequences and four repeats that were present in other species but not as repeats). In total, 55 species-specific repetitive sequences are classified as mobile genetic elements; 4 are classified as ISs; 34 are classified as phage-related sequences; and 17 are classified as plasmidic repeats.</p><p>The remaining 40 species-specific repeats are associated with protein-coding genes. Sixteen of them are related to proteins of the RTX family, which contains putative virulence factors, RTX toxins (repeats in toxins), which are exported proteins.<sup><a reveal-id="DSS006C39" data-open="DSS006C39" class="link link-ref link-reveal xref-bibr">39</a></sup> This could explain why there are 10 species-specific repeats associated with different inner and outer membrane proteins, some of which are suggested to be auto-transporters. Also, one repeat contains genes of which the functions involve different type IV pilus biosynthesis proteins (type IV pilus proteins are also essential for virulence).</p><p>Furthermore, only two repeats contain hypothetical protein-coding genes and only four species-specific repeats contain pseudogenes.</p><p>The final six repetitive sequences are associated with genes with various functions (methionine sulfoxide reductase, phosphomannomutase, heat shock protein, amidase and lipoprotein, serine endoprotease, transcriptional regulator, and GTP cyclohydrolase). Some of these genes could also be involved with pathogenicity but no direct reference can be indicated.</p>                            <h3 scrollto-destination=7157547 id="7157547" class="section-title" data-legacy-id=s3d>3.4. Firmness of species-specific repeats</h3>
<p>One of the major questions that arose during this work was related to the definition of species and the genetic variation within species. Does a species-specific repeat exist as a repeat in every newly sequenced strain? Or, if a new, closely related species is sequenced, does the repeat remain species-specific? In reality, of all the prokaryotic genomes existing on the Earth, only a small part of each has been sequenced; this part may represent distantly related genomes. Because of these questions, we have validated species-specific repeats against a prokaryotic database that consists of ∼1.42 times more species than the database used for finding those repeats (data not shown). Approximately 88% of detected species-specific repeats remained species-specific in this larger data set. Interestingly, closest species where the species-specific repeats could still be found are 99% identical in terms of 16S rRNA (e.g. <em>Shewanella pealeana</em> NC_009901 and <em>Shewanella halifaxensis</em> NC_010334). Nevertheless, some repeats turned out to be non-species-specific in the larger data set, because they existed in another newly sequenced species or the repeat was not present in the genome of a newly sequenced strain of the same species. This raises the question of whether the species-specific repeats we have identified are real species-specific repeats or just data analysis artifacts, dependent on the number of available genomes and the choice of species for analysis.</p><p>The possibility of finding, or not finding, a species-specific repeat is partly determined by the controversial definition of bacterial species. Currently, the definition comprises 70% DNA–DNA re-association as a standard.<sup><a reveal-id="DSS006C40" data-open="DSS006C40" class="link link-ref link-reveal xref-bibr">40</a></sup> As this definition is somewhat arbitrary, the definition of species-specific repeats is also arbitrary. However, the problem of the species concept is not the topic of this article. We use the definition of species as it comes from the NCBI genome database, together with the genomic DNA sequence. In most cases, the repeats we define as species-specific repeats will remain specific to a group of phylogenetically related strains, even if the concept of the species will be changed in future. Yet, we have identified strain-specific repetitive sequences (data not shown) in 50 of 91 species for which 2–26 completely sequenced strains per species were available. Strain-specific repetitive sequences add some confidence to the existence of repeats specific to a small phylogenetically related group.</p>                            <h2 scrollto-destination=7157550 id="7157550" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>Although repetitive sequences are less common in prokaryotes than in eukaryotes, almost all prokaryotic species still have some identifiable repeats. In this work, we have characterized mainly the species-specific repetitive sequences—repeats common to all strains of a species, but not to other species.</p><p>In accordance with widely accepted understanding that prokaryotic genomes are compactly organized, we have demonstrated that the median value of the repeat coverage of bacterial genome is 1.8% (species-specific repeats constitute ∼1% from this). Despite of a small fraction of repetitive sequences in prokaryotic genomes, many novel findings have arisen. We have found that not all repetitive sequences in prokaryotic genomes belong to mobile genetic elements. Our results indicate that only ∼14% of the species-specific repetitive sequences and ∼53% of universal repeats are associated with mobile genetic elements. Further, the functions of many intergenic repeats and repeats from the class of unknown origin (in total ∼17% from the species-specific repeats) are not understood. However, a fraction of intergenic repeats may comprise a certain conserved regulatory elements. For example, specific intergenic repeats (CIR motifs, &gt;100 bp) in <em>Caulobacter crescentus</em> may be associated with gene conversion and with gene regulation.<sup><a reveal-id="DSS006C33" data-open="DSS006C33" class="link link-ref link-reveal xref-bibr">33</a></sup> A large number of various functions of protein-coding genes, reflecting the diversity of sequenced prokaryotic genomes, have emerged from the functional analysis. In different species, genes associated with repeats have variable roles and no common phenomenon can be identified for all species or for groups of particular species. As shown, the function of many protein-coding genes associated with species-specific repeats remains unknown. We rather think that this interesting question cannot be answered by automatic analysis but must be left to biologists who investigate a particular genome in more detail. The functional analysis also revealed that different repeat classes appear in different species. In some species, phage-related sequences are in prevalence; in the other, only rDNA-related repeats appear; or in the third, one large duplication containing different protein-coding genes is found. Every species seems to be somewhat unique regarding to its repeats.</p><p>One important application of species-specific repeats is their usage in the identification of prokaryotic species. The repetitive sequences increase the sensitivity of the PCR test simply because of the higher concentration (double or more) of the initial sequence.<sup><a reveal-id="DSS006C24" data-open="DSS006C24" class="link link-ref link-reveal xref-bibr">24</a></sup> In this work, we have shown that many of the species-specific repetitive sequences are associated with protein-coding genes, so at least these repeats can be reliably used as target sequences in PCR tests. Species-specific or a group of phylogenetically related strains-specific repeats may sometimes not be the preferred target in medical diagnostics. This is in cases where pathogenic genes are known and the objective of the diagnostic test is to detect such a gene rather than to detect the species.<sup><a reveal-id="DSS006C41" data-open="DSS006C41" class="link link-ref link-reveal xref-bibr">41</a></sup> However, we believe that this does not diminish the value of species-specific repeats as a PCR target region for the detection of species. Pathogenic gene-specific PCR primers can be designed with similar methodology and used in addition to species-specific PCR primers in medical tests.</p><p>In conclusion, the current work enhances knowledge about prokaryotic species-specific repeats in many novel aspects. Most prokaryotic species contain several short species-specific repeats, which are represented in the genome with a low copy number. In a typical prokaryotic species, the species-specific repeats cover ∼1% of the genome. Over half of these repeats contain protein-coding genes, approximately one-eighth are associated with mobile genetic elements and one-eighth are associated with short non-coding interspersed elements. Thus, most of the species-specific repeats are associated with protein-coding genes and probably related to the biochemical processes required for adaptation and survival of any given prokaryote. Unfortunately, the functions of many species-specific repeats related to protein-coding genes currently remain unexplained.</p>                            <h2 scrollto-destination=7157555 id="7157555" class="section-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available online at www.dnaresearch.oxfordjournal.org</a>.</p>                            <h2 scrollto-destination=7157557 id="7157557" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was funded by the EU European Regional Development Fund, through the Estonian Centre of Excellence in Genomics and grant SF0180026s09 from the Estonian Ministry of Education and Research.</p>                            <h2 scrollto-destination=7157559 id="7157559" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSS006C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Koonin</div> <div class="given-names">E. V.</div></div>,  <div class="name"><div class="surname">Wolf</div> <div class="given-names">Y. I.</div></div>. </div><div class="article-title">Genomics of bacteria and archaea: the emerging dynamic view of the prokaryotic world</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">6688</div>-<div class="lpage">719</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomics%20of%20bacteria%20and%20archaea%3A%20the%20emerging%20dynamic%20view%20of%20the%20prokaryotic%20world&amp;author=E.%20V.%20Koonin&amp;author=Y.%20I.%20Wolf&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=6688-719" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn668" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn668" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn668"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18948295" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakabachi</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Yamashita</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Toh</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">The 160-kilobase genome of the bacterial endosymbiont <em>Carsonella</em></div>, <div class="source">Science</div> , <div class="year">2006</div>, vol. <div class="volume">314</div> pg. <div class="fpage">267</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20160-kilobase%20genome%20of%20the%20bacterial%20endosymbiont%20Carsonella&amp;author=A.%20Nakabachi&amp;author=A.%20Yamashita&amp;author=H.%20Toh&amp;publication_year=2006&amp;journal=Science&amp;volume=314&amp;pages=267" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1134196" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1134196" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1134196"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17038615" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schneiker</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Perlova</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Kaiser</div> <div class="given-names">O.</div></div>, et al. </div><div class="article-title">Complete genome sequence of the myxobacterium <em>Sorangium cellulosum</em></div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2007</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">1281</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20the%20myxobacterium%20Sorangium%20cellulosum&amp;author=S.%20Schneiker&amp;author=O.%20Perlova&amp;author=O.%20Kaiser&amp;publication_year=2007&amp;journal=Nat.%20Biotechnol.&amp;volume=25&amp;pages=1281-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1354" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1354" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1354"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17965706" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Delihas</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Small mobile sequences in bacteria display diverse structure/function motifs</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2008</div>, vol. <div class="volume">67</div> (pg. <div class="fpage">475</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Small%20mobile%20sequences%20in%20bacteria%20display%20diverse%20structure%2Ffunction%20motifs&amp;author=N.%20Delihas&amp;publication_year=2008&amp;journal=Mol.%20Microbiol.&amp;volume=67&amp;pages=475-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.2008.67.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.2008.67.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.2008.67.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18086200" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Achaz</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Rocha</div> <div class="given-names">E. P.</div></div>,  <div class="name"><div class="surname">Netter</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Coissac</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Origin and fate of repeats in bacteria</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2002</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">2987</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Origin%20and%20fate%20of%20repeats%20in%20bacteria&amp;author=G.%20Achaz&amp;author=E.%20P.%20Rocha&amp;author=P.%20Netter&amp;author=E.%20Coissac&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res.&amp;volume=30&amp;pages=2987-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkf391" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkf391" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkf391"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12087185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wu</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Sun</div> <div class="given-names">L. V.</div></div>,  <div class="name"><div class="surname">Vamathevan</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Phylogenomics of the reproductive parasite <em>Wolbachia pipientis</em> wMel: a streamlined genome overrun by mobile genetic elements</div>, <div class="source">PLoS Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">2</div> pg. <div class="fpage">E69</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenomics%20of%20the%20reproductive%20parasite%20Wolbachia%20pipientis%20wMel%3A%20a%20streamlined%20genome%20overrun%20by%20mobile%20genetic%20elements&amp;author=M.%20Wu&amp;author=L.%20V.%20Sun&amp;author=J.%20Vamathevan&amp;publication_year=2004&amp;journal=PLoS%20Biol.&amp;volume=2&amp;pages=E69" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0020069" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0020069" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0020069"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15024419" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rocha</div> <div class="given-names">E. P. C.</div></div>,  <div class="name"><div class="surname">Danchin</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Viari</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Functional and evolutionary roles of long repeats in prokaryotes</div>, <div class="source">Res. Microbiol.</div> , <div class="year">1999</div>, vol. <div class="volume">150</div> (pg. <div class="fpage">725</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20and%20evolutionary%20roles%20of%20long%20repeats%20in%20prokaryotes&amp;author=E.%20P.%20C.%20Rocha&amp;author=A.%20Danchin&amp;author=A.%20Viari&amp;publication_year=1999&amp;journal=Res.%20Microbiol.&amp;volume=150&amp;pages=725-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0923-2508(99)00120-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0923-2508(99)00120-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0923-2508(99)00120-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10673010" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rocha</div> <div class="given-names">E. P.</div></div>,  <div class="name"><div class="surname">Danchin</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Viari</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Analysis of long repeats in bacterial genomes reveals alternative evolutionary mechanisms in <em>Bacillus subtilis</em> and other competent prokaryotes</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1999</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">1219</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20long%20repeats%20in%20bacterial%20genomes%20reveals%20alternative%20evolutionary%20mechanisms%20in%20Bacillus%20subtilis%20and%20other%20competent%20prokaryotes&amp;author=E.%20P.%20Rocha&amp;author=A.%20Danchin&amp;author=A.%20Viari&amp;publication_year=1999&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=16&amp;pages=1219-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026212" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a026212" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a026212"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10486977" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Smillie</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Garcillan-Barcia</div> <div class="given-names">M. P.</div></div>,  <div class="name"><div class="surname">Francia</div> <div class="given-names">M. V.</div></div>,  <div class="name"><div class="surname">Rocha</div> <div class="given-names">E. P.</div></div>,  <div class="name"><div class="surname">de la Cruz</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Mobility of plasmids</div>, <div class="source">Microbiol. Mol. Biol. Rev.</div> , <div class="year">2010</div>, vol. <div class="volume">74</div> (pg. <div class="fpage">434</div>-<div class="lpage">52</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mobility%20of%20plasmids&amp;author=C.%20Smillie&amp;author=M.%20P.%20Garcillan-Barcia&amp;author=M.%20V.%20Francia&amp;author=E.%20P.%20Rocha&amp;author=F.%20de%20la%20Cruz&amp;publication_year=2010&amp;journal=Microbiol.%20Mol.%20Biol.%20Rev.&amp;volume=74&amp;pages=434-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MMBR.00020-10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMMBR.00020-10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMMBR.00020-10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20805406" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mavrodi</div> <div class="given-names">D. V.</div></div>,  <div class="name"><div class="surname">Loper</div> <div class="given-names">J. E.</div></div>,  <div class="name"><div class="surname">Paulsen</div> <div class="given-names">I. T.</div></div>,  <div class="name"><div class="surname">Thomashow</div> <div class="given-names">L. S.</div></div>. </div><div class="article-title">Mobile genetic elements in the genome of the beneficial rhizobacterium <em>Pseudomonas fluorescens</em> Pf-5</div>, <div class="source">BMC Microbiol.</div> , <div class="year">2009</div>, vol. <div class="volume">9</div> pg. <div class="fpage">8</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mobile%20genetic%20elements%20in%20the%20genome%20of%20the%20beneficial%20rhizobacterium%20Pseudomonas%20fluorescens%20Pf-5&amp;author=D.%20V.%20Mavrodi&amp;author=J.%20E.%20Loper&amp;author=I.%20T.%20Paulsen&amp;author=L.%20S.%20Thomashow&amp;publication_year=2009&amp;journal=BMC%20Microbiol.&amp;volume=9&amp;pages=8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2180-9-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2180-9-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2180-9-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19144133" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Toleman</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Bennett</div> <div class="given-names">P. M.</div></div>,  <div class="name"><div class="surname">Walsh</div> <div class="given-names">T. R.</div></div>. </div><div class="article-title">ISCR elements: novel gene-capturing systems of the 21st century?</div>, <div class="source">Microbiol. Mol. Biol. Rev.</div> , <div class="year">2006</div>, vol. <div class="volume">70</div> (pg. <div class="fpage">296</div>-<div class="lpage">316</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ISCR%20elements%3A%20novel%20gene-capturing%20systems%20of%20the%2021st%20century%3F&amp;author=M.%20A.%20Toleman&amp;author=P.%20M.%20Bennett&amp;author=T.%20R.%20Walsh&amp;publication_year=2006&amp;journal=Microbiol.%20Mol.%20Biol.%20Rev.&amp;volume=70&amp;pages=296-316" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MMBR.00048-05" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMMBR.00048-05" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMMBR.00048-05"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16760305" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nesmelova</div> <div class="given-names">I. V.</div></div>,  <div class="name"><div class="surname">Hackett</div> <div class="given-names">P. B.</div></div>. </div><div class="article-title">DDE transposases: Structural similarity and diversity</div>, <div class="source">Adv. Drug Deliv. Rev.</div> , <div class="year">2010</div>, vol. <div class="volume">62</div> (pg. <div class="fpage">1187</div>-<div class="lpage">1195</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DDE%20transposases%3A%20Structural%20similarity%20and%20diversity&amp;author=I.%20V.%20Nesmelova&amp;author=P.%20B.%20Hackett&amp;publication_year=2010&amp;journal=Adv.%20Drug%20Deliv.%20Rev.&amp;volume=62&amp;pages=1187-1195" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.addr.2010.06.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.addr.2010.06.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.addr.2010.06.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20615441" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brochet</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Da Cunha</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Couve</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Rusniok</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Trieu-Cuot</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Glaser</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Atypical association of DDE transposition with conjugation specifies a new family of mobile elements</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2009</div>, vol. <div class="volume">71</div> (pg. <div class="fpage">948</div>-<div class="lpage">59</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Atypical%20association%20of%20DDE%20transposition%20with%20conjugation%20specifies%20a%20new%20family%20of%20mobile%20elements&amp;author=M.%20Brochet&amp;author=V.%20Da%20Cunha&amp;author=E.%20Couve&amp;author=C.%20Rusniok&amp;author=P.%20Trieu-Cuot&amp;author=P.%20Glaser&amp;publication_year=2009&amp;journal=Mol.%20Microbiol.&amp;volume=71&amp;pages=948-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.2009.71.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.2009.71.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.2009.71.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19183283" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wozniak</div> <div class="given-names">R. A.</div></div>,  <div class="name"><div class="surname">Waldor</div> <div class="given-names">M. K.</div></div>. </div><div class="article-title">Integrative and conjugative elements: mosaic mobile genetic elements enabling dynamic lateral gene flow</div>, <div class="source">Nat. Rev. Microbiol.</div> , <div class="year">2010</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">552</div>-<div class="lpage">63</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrative%20and%20conjugative%20elements%3A%20mosaic%20mobile%20genetic%20elements%20enabling%20dynamic%20lateral%20gene%20flow&amp;author=R.%20A.%20Wozniak&amp;author=M.%20K.%20Waldor&amp;publication_year=2010&amp;journal=Nat.%20Rev.%20Microbiol.&amp;volume=8&amp;pages=552-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrmicro2382" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrmicro2382" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrmicro2382"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20601965" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tobes</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Pareja</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Bacterial repetitive extragenic palindromic sequences are DNA targets for insertion sequence elements</div>, <div class="source">BMC Genomics</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">62</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacterial%20repetitive%20extragenic%20palindromic%20sequences%20are%20DNA%20targets%20for%20insertion%20sequence%20elements&amp;author=R.%20Tobes&amp;author=E.%20Pareja&amp;publication_year=2006&amp;journal=BMC%20Genomics&amp;volume=7&amp;pages=62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-7-62" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-7-62" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-7-62"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16563168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mes</div> <div class="given-names">T. H.</div></div>,  <div class="name"><div class="surname">Doeleman</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Positive selection on transposase genes of insertion sequences in the <em>Crocosphaera watsonii</em> genome</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2006</div>, vol. <div class="volume">188</div> (pg. <div class="fpage">7176</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Positive%20selection%20on%20transposase%20genes%20of%20insertion%20sequences%20in%20the%20Crocosphaera%20watsonii%20genome&amp;author=T.%20H.%20Mes&amp;author=M.%20Doeleman&amp;publication_year=2006&amp;journal=J.%20Bacteriol.&amp;volume=188&amp;pages=7176-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.01021-06" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.01021-06" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.01021-06"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17015656" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nunvar</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Huckova</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Licha</div> <div class="given-names">I.</div></div>. </div><div class="article-title">Identification and characterization of repetitive extragenic palindromes (REP)-associated tyrosine transposases: implications for REP evolution and dynamics in bacterial genomes</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">44</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20characterization%20of%20repetitive%20extragenic%20palindromes%20(REP)-associated%20tyrosine%20transposases%3A%20implications%20for%20REP%20evolution%20and%20dynamics%20in%20bacterial%20genomes&amp;author=J.%20Nunvar&amp;author=T.%20Huckova&amp;author=I.%20Licha&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-44" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-44" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-44"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20085626" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cortez</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Forterre</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Gribaldo</div> <div class="given-names">S.</div></div>. </div><div class="article-title">A hidden reservoir of integrative elements is the major source of recently acquired foreign genes and ORFans in archaeal and bacterial genomes</div>, <div class="source">Genome Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">R65</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20hidden%20reservoir%20of%20integrative%20elements%20is%20the%20major%20source%20of%20recently%20acquired%20foreign%20genes%20and%20ORFans%20in%20archaeal%20and%20bacterial%20genomes&amp;author=D.%20Cortez&amp;author=P.%20Forterre&amp;author=S.%20Gribaldo&amp;publication_year=2009&amp;journal=Genome%20Biol.&amp;volume=10&amp;pages=R65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2009-10-6-r65" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2009-10-6-r65" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2009-10-6-r65"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19531232" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gogarten</div> <div class="given-names">J. P.</div></div>,  <div class="name"><div class="surname">Townsend</div> <div class="given-names">J. P.</div></div>. </div><div class="article-title">Horizontal gene transfer, genome innovation and evolution</div>, <div class="source">Nat. Rev. Microbiol.</div> , <div class="year">2005</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">679</div>-<div class="lpage">87</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Horizontal%20gene%20transfer%2C%20genome%20innovation%20and%20evolution&amp;author=J.%20P.%20Gogarten&amp;author=J.%20P.%20Townsend&amp;publication_year=2005&amp;journal=Nat.%20Rev.%20Microbiol.&amp;volume=3&amp;pages=679-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrmicro1204" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrmicro1204" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrmicro1204"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16138096" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Canchaya</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Fournous</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Chibani-Chennoufi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Dillmann</div> <div class="given-names">M. L.</div></div>,  <div class="name"><div class="surname">Brussow</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Phage as agents of lateral gene transfer</div>, <div class="source">Curr. Opin. Microbiol.</div> , <div class="year">2003</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">417</div>-<div class="lpage">24</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phage%20as%20agents%20of%20lateral%20gene%20transfer&amp;author=C.%20Canchaya&amp;author=G.%20Fournous&amp;author=S.%20Chibani-Chennoufi&amp;author=M.%20L.%20Dillmann&amp;author=H.%20Brussow&amp;publication_year=2003&amp;journal=Curr.%20Opin.%20Microbiol.&amp;volume=6&amp;pages=417-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1369-5274(03)00086-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1369-5274(03)00086-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1369-5274(03)00086-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12941415" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Frost</div> <div class="given-names">L. S.</div></div>,  <div class="name"><div class="surname">Leplae</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Summers</div> <div class="given-names">A. O.</div></div>,  <div class="name"><div class="surname">Toussaint</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Mobile genetic elements: the agents of open source evolution</div>, <div class="source">Nat. Rev. Microbiol.</div> , <div class="year">2005</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">722</div>-<div class="lpage">32</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mobile%20genetic%20elements%3A%20the%20agents%20of%20open%20source%20evolution&amp;author=L.%20S.%20Frost&amp;author=R.%20Leplae&amp;author=A.%20O.%20Summers&amp;author=A.%20Toussaint&amp;publication_year=2005&amp;journal=Nat.%20Rev.%20Microbiol.&amp;volume=3&amp;pages=722-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrmicro1235" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrmicro1235" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrmicro1235"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16138100" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ravin</div> <div class="given-names">N. V.</div></div>,  <div class="name"><div class="surname">Mardanov</div> <div class="given-names">A. V.</div></div>,  <div class="name"><div class="surname">Beletsky</div> <div class="given-names">A. V.</div></div>, et al. </div><div class="article-title">Complete genome sequence of the anaerobic, protein-degrading hyperthermophilic crenarchaeon <em>Desulfurococcus kamchatkensis</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2009</div>, vol. <div class="volume">191</div> (pg. <div class="fpage">2371</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20the%20anaerobic%2C%20protein-degrading%20hyperthermophilic%20crenarchaeon%20Desulfurococcus%20kamchatkensis&amp;author=N.%20V.%20Ravin&amp;author=A.%20V.%20Mardanov&amp;author=A.%20V.%20Beletsky&amp;publication_year=2009&amp;journal=J.%20Bacteriol.&amp;volume=191&amp;pages=2371-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.01525-08" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.01525-08" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.01525-08"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19114480" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div>,  <div class="name"><div class="surname">Sommer</div> <div class="given-names">D. D.</div></div>,  <div class="name"><div class="surname">Schatz</div> <div class="given-names">M. C.</div></div>, et al. </div><div class="article-title">Genome sequence and rapid evolution of the rice pathogen <em>Xanthomonas oryzae</em> pv. <em>oryzae</em> PXO99A</div>, <div class="source">BMC Genomics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">204</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20and%20rapid%20evolution%20of%20the%20rice%20pathogen%20Xanthomonas%20oryzae%20pv.%20oryzae%20PXO99A&amp;author=S.%20L.%20Salzberg&amp;author=D.%20D.%20Sommer&amp;author=M.%20C.%20Schatz&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=204" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-204" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-204" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-204"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18452608" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Koressaar</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Joers</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Remm</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Automatic identification of species-specific repetitive DNA sequences and their utilization for detecting microbial organisms</div>, <div class="source">Bioinformatics</div> , <div class="year">2009</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">1349</div>-<div class="lpage">55</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automatic%20identification%20of%20species-specific%20repetitive%20DNA%20sequences%20and%20their%20utilization%20for%20detecting%20microbial%20organisms&amp;author=T.%20Koressaar&amp;author=K.%20Joers&amp;author=M.%20Remm&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1349-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp241" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp241" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp241"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19357101" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fouts</div> <div class="given-names">D. E.</div></div>. </div><div class="article-title">Phage_Finder: automated identification and classification of prophage regions in complete bacterial genome sequences</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">5839</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phage_Finder%3A%20automated%20identification%20and%20classification%20of%20prophage%20regions%20in%20complete%20bacterial%20genome%20sequences&amp;author=D.%20E.%20Fouts&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=5839-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl732" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl732" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl732"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17062630" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Varani</div> <div class="given-names">A. M.</div></div>,  <div class="name"><div class="surname">Siguier</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Gourbeyre</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Charneau</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Chandler</div> <div class="given-names">M.</div></div>. </div><div class="article-title">ISsaga is an ensemble of web-based methods for high throughput identification and semi-automatic annotation of insertion sequences in prokaryotic genomes</div>, <div class="source">Genome Biol.</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">R30</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ISsaga%20is%20an%20ensemble%20of%20web-based%20methods%20for%20high%20throughput%20identification%20and%20semi-automatic%20annotation%20of%20insertion%20sequences%20in%20prokaryotic%20genomes&amp;author=A.%20M.%20Varani&amp;author=P.%20Siguier&amp;author=E.%20Gourbeyre&amp;author=V.%20Charneau&amp;author=M.%20Chandler&amp;publication_year=2011&amp;journal=Genome%20Biol.&amp;volume=12&amp;pages=R30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2011-12-3-r30" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2011-12-3-r30" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2011-12-3-r30"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21443786" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grissa</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Vergnaud</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Pourcel</div> <div class="given-names">C.</div></div>. </div><div class="article-title">CRISPRFinder: a web tool to identify clustered regularly interspaced short palindromic repeats</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">W52</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CRISPRFinder%3A%20a%20web%20tool%20to%20identify%20clustered%20regularly%20interspaced%20short%20palindromic%20repeats&amp;author=I.%20Grissa&amp;author=G.%20Vergnaud&amp;author=C.%20Pourcel&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=W52-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm360" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm360" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm360"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17537822" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Finn</div> <div class="given-names">R. D.</div></div>,  <div class="name"><div class="surname">Mistry</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Tate</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">The Pfam protein families database</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2010</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">D211</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Pfam%20protein%20families%20database&amp;author=R.%20D.%20Finn&amp;author=J.%20Mistry&amp;author=J.%20Tate&amp;publication_year=2010&amp;journal=Nucleic%20Acids%20Res.&amp;volume=38&amp;pages=D211-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkp985" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkp985" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkp985"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19920124" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Soda</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Otsuki</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Inoue</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Tsutsui</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Sei</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ike</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Transfer of antibiotic multiresistant plasmid RP4 from <em>Escherichia coli</em> to activated sludge bacteria</div>, <div class="source">J. Biosci. Bioeng.</div> , <div class="year">2008</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">292</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transfer%20of%20antibiotic%20multiresistant%20plasmid%20RP4%20from%20Escherichia%20coli%20to%20activated%20sludge%20bacteria&amp;author=S.%20Soda&amp;author=H.%20Otsuki&amp;author=D.%20Inoue&amp;author=H.%20Tsutsui&amp;author=K.%20Sei&amp;author=M.%20Ike&amp;publication_year=2008&amp;journal=J.%20Biosci.%20Bioeng.&amp;volume=106&amp;pages=292-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1263/jbb.106.292" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1263%2Fjbb.106.292" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1263%2Fjbb.106.292"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18930008" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhao</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Bai</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Sequencing and genetic variation of multidrug resistance plasmids in <em>Klebsiella pneumoniae</em></div>, <div class="source">PLoS One</div> , <div class="year">2010</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e10141</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20and%20genetic%20variation%20of%20multidrug%20resistance%20plasmids%20in%20Klebsiella%20pneumoniae&amp;author=F.%20Zhao&amp;author=J.%20Bai&amp;author=J.%20Wu&amp;publication_year=2010&amp;journal=PLoS%20One&amp;volume=5&amp;pages=e10141" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0010141" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0010141" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0010141"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20405037" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bachellier</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Clement</div> <div class="given-names">J. M.</div></div>,  <div class="name"><div class="surname">Hofnung</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Gilson</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Bacterial interspersed mosaic elements (BIMEs) are a major source of sequence polymorphism in <em>Escherichia coli</em> intergenic regions including specific associations with a new insertion sequence</div>, <div class="source">Genetics</div> , <div class="year">1997</div>, vol. <div class="volume">145</div> (pg. <div class="fpage">551</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacterial%20interspersed%20mosaic%20elements%20(BIMEs)%20are%20a%20major%20source%20of%20sequence%20polymorphism%20in%20Escherichia%20coli%20intergenic%20regions%20including%20specific%20associations%20with%20a%20new%20insertion%20sequence&amp;author=S.%20Bachellier&amp;author=J.%20M.%20Clement&amp;author=M.%20Hofnung&amp;author=E.%20Gilson&amp;publication_year=1997&amp;journal=Genetics&amp;volume=145&amp;pages=551-62" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9055066" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharples</div> <div class="given-names">G. J.</div></div>,  <div class="name"><div class="surname">Lloyd</div> <div class="given-names">R. G.</div></div>. </div><div class="article-title">A novel repeated DNA sequence located in the intergenic regions of bacterial chromosomes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1990</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">6503</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20repeated%20DNA%20sequence%20located%20in%20the%20intergenic%20regions%20of%20bacterial%20chromosomes&amp;author=G.%20J.%20Sharples&amp;author=R.%20G.%20Lloyd&amp;publication_year=1990&amp;journal=Nucleic%20Acids%20Res.&amp;volume=18&amp;pages=6503-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/18.22.6503" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F18.22.6503" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F18.22.6503"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2251112" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">S. L.</div></div>,  <div class="name"><div class="surname">Shapiro</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Identification of long intergenic repeat sequences associated with DNA methylation sites in <em>Caulobacter crescentus</em> and other alpha-proteobacteria</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2003</div>, vol. <div class="volume">185</div> (pg. <div class="fpage">4997</div>-<div class="lpage">5002</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20long%20intergenic%20repeat%20sequences%20associated%20with%20DNA%20methylation%20sites%20in%20Caulobacter%20crescentus%20and%20other%20alpha-proteobacteria&amp;author=S.%20L.%20Chen&amp;author=L.%20Shapiro&amp;publication_year=2003&amp;journal=J.%20Bacteriol.&amp;volume=185&amp;pages=4997-5002" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.185.16.4997-5002.2003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.185.16.4997-5002.2003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.185.16.4997-5002.2003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12897020" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Doolittle</div> <div class="given-names">W. F.</div></div>,  <div class="name"><div class="surname">Sapienza</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Selfish genes, the phenotype paradigm and genome evolution</div>, <div class="source">Nature</div> , <div class="year">1980</div>, vol. <div class="volume">284</div> (pg. <div class="fpage">601</div>-<div class="lpage">3</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selfish%20genes%2C%20the%20phenotype%20paradigm%20and%20genome%20evolution&amp;author=W.%20F.%20Doolittle&amp;author=C.%20Sapienza&amp;publication_year=1980&amp;journal=Nature&amp;volume=284&amp;pages=601-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/284601a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F284601a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F284601a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6245369" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Orgel</div> <div class="given-names">L. E.</div></div>,  <div class="name"><div class="surname">Crick</div> <div class="given-names">F. H.</div></div>. </div><div class="article-title">Selfish DNA: the ultimate parasite</div>, <div class="source">Nature</div> , <div class="year">1980</div>, vol. <div class="volume">284</div> (pg. <div class="fpage">604</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selfish%20DNA%3A%20the%20ultimate%20parasite&amp;author=L.%20E.%20Orgel&amp;author=F.%20H.%20Crick&amp;publication_year=1980&amp;journal=Nature&amp;volume=284&amp;pages=604-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/284604a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F284604a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F284604a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7366731" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lynch</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">O'Hely</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Walsh</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Force</div> <div class="given-names">A.</div></div>. </div><div class="article-title">The probability of preservation of a newly arisen gene duplicate</div>, <div class="source">Genetics</div> , <div class="year">2001</div>, vol. <div class="volume">159</div> (pg. <div class="fpage">1789</div>-<div class="lpage">804</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20probability%20of%20preservation%20of%20a%20newly%20arisen%20gene%20duplicate&amp;author=M.%20Lynch&amp;author=M.%20O'Hely&amp;author=B.%20Walsh&amp;author=A.%20Force&amp;publication_year=2001&amp;journal=Genetics&amp;volume=159&amp;pages=1789-804" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11779815" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bolhuis</div> <div class="given-names">A.</div></div>. </div><div class="article-title">The archaeal Sec-dependent protein translocation pathway</div>, <div class="source">Philos. Trans. R. Soc. Lond. B Biol. Sci.</div> , <div class="year">2004</div>, vol. <div class="volume">359</div> (pg. <div class="fpage">919</div>-<div class="lpage">27</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20archaeal%20Sec-dependent%20protein%20translocation%20pathway&amp;author=A.%20Bolhuis&amp;publication_year=2004&amp;journal=Philos.%20Trans.%20R.%20Soc.%20Lond.%20B%20Biol.%20Sci.&amp;volume=359&amp;pages=919-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1098/rstb.2003.1461" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frstb.2003.1461" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1098%2Frstb.2003.1461"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15306407" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Holland</div> <div class="given-names">K. A.</div></div>,  <div class="name"><div class="surname">Holland</div> <div class="given-names">I. B.</div></div>. </div><div class="article-title">Adventures with ABC-proteins: highly conserved ATP-dependent transporters</div>, <div class="source">Acta Microbiol. Immunol. Hung.</div> , <div class="year">2005</div>, vol. <div class="volume">52</div> (pg. <div class="fpage">309</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Adventures%20with%20ABC-proteins%3A%20highly%20conserved%20ATP-dependent%20transporters&amp;author=K.%20A.%20Holland&amp;author=I.%20B.%20Holland&amp;publication_year=2005&amp;journal=Acta%20Microbiol.%20Immunol.%20Hung.&amp;volume=52&amp;pages=309-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1556/AMicr.52.2005.3-4.4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1556%2FAMicr.52.2005.3-4.4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1556%2FAMicr.52.2005.3-4.4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16400872" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lally</div> <div class="given-names">E. T.</div></div>,  <div class="name"><div class="surname">Hill</div> <div class="given-names">R. B.</div></div>,  <div class="name"><div class="surname">Kieba</div> <div class="given-names">I. R.</div></div>,  <div class="name"><div class="surname">Korostoff</div> <div class="given-names">J.</div></div>. </div><div class="article-title">The interaction between RTX toxins and target cells</div>, <div class="source">Trends Microbiol.</div> , <div class="year">1999</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">356</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20interaction%20between%20RTX%20toxins%20and%20target%20cells&amp;author=E.%20T.%20Lally&amp;author=R.%20B.%20Hill&amp;author=I.%20R.%20Kieba&amp;author=J.%20Korostoff&amp;publication_year=1999&amp;journal=Trends%20Microbiol.&amp;volume=7&amp;pages=356-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0966-842X(99)01530-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0966-842X(99)01530-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0966-842X(99)01530-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10470043" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Achtman</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Wagner</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Microbial diversity and the genetic nature of microbial species</div>, <div class="source">Nat. Rev. Microbiol.</div> , <div class="year">2008</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">431</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microbial%20diversity%20and%20the%20genetic%20nature%20of%20microbial%20species&amp;author=M.%20Achtman&amp;author=M.%20Wagner&amp;publication_year=2008&amp;journal=Nat.%20Rev.%20Microbiol.&amp;volume=6&amp;pages=431-40" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18461076" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS006C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS006C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Belanger</div> <div class="given-names">S. D.</div></div>,  <div class="name"><div class="surname">Boissinot</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Menard</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Picard</div> <div class="given-names">F. J.</div></div>,  <div class="name"><div class="surname">Bergeron</div> <div class="given-names">M. G.</div></div>. </div><div class="article-title">Rapid detection of Shiga toxin-producing bacteria in feces by multiplex PCR with molecular beacons on the smart cycler</div>, <div class="source">J. Clin. Microbiol.</div> , <div class="year">2002</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">1436</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20detection%20of%20Shiga%20toxin-producing%20bacteria%20in%20feces%20by%20multiplex%20PCR%20with%20molecular%20beacons%20on%20the%20smart%20cycler&amp;author=S.%20D.%20Belanger&amp;author=M.%20Boissinot&amp;author=C.%20Menard&amp;author=F.%20J.%20Picard&amp;author=M.%20G.%20Bergeron&amp;publication_year=2002&amp;journal=J.%20Clin.%20Microbiol.&amp;volume=40&amp;pages=1436-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JCM.40.4.1436-1440.2002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJCM.40.4.1436-1440.2002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJCM.40.4.1436-1440.2002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11923369" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Katsumi Isono</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2012. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=474767&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss006/2/dss006-dss006supp_table.xlsx?Expires=2147483647&amp;Signature=1NA~XZcSc277tfFCOl~XknuMHUQCGDR3RByhokaCNSpf3jZiQIsXSJ4RNsEUqMVvHy1TjvtWI2CWtL1uhwmZh0sATlndqWTYHHdzpdwB-fGmy0zrrL4wEYt586iw2UJEWRheceyukT8RAl0UjWva47ZD6UYJ601Js-fC0cDG9QBxauPGs7rhrkKR6twvqRee~dobin6td0l1ge6fFh8bDhwAn92eDfPaoXUrIyJ4mvIZ7IZJmssYYWTLn-FnOOq~4~U005J5kNxyWOYKwM3ZKi4R712p1nsbYwCYbnI9HVOaZeRvw~SoMTiuGdx~oP9Y6s8Z~HqK5h1HTiJWsTT2Sw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss006/2/dss006-dss006supp.doc?Expires=2147483647&amp;Signature=FwTI9qBcV6V9o5Fzvy-lIM4cVzOGN~M4bT0G0tyvUg-BfsijfG7u8HpkVW2DVcfDghVTY0lYiQAPMRCScX4Wz9MOc4kTzfAhlg2YG5Vdny0P2E6VUhBa5wmG7iw8epaIbzbB6gnDAXM2BXPv79AMVMvWLavpD6Ldm3JZkfqnMr5I8QE8dmeSVtffChuZ1ITLMpk0JbFvIkWWECCJKSB8RRS~UZNh3ccUeuCw86lFQRJusWsbpREUeLawCBGcu3wjSHdiC9gVSnv7rDaS1oignDFQptsDqWN6WXmEacyvEzwO2LjozHvrbcRby9~DC93Nbgy60CsOyIsDzySOooO7Zg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">301</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000305723200002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">4</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dss006"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="474767"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000305723200002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f19%2f3%2f219%2f474767">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000305723200002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (4)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F19%2F3%2F219%2F474767" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/474767'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Characterization of Species-Specific Repeats in 613 Prokaryotic Species" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dss006");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f19%2f3%2f219%2f474767">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

