

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/1/11/2670108","@type":"ScholarlyArticle","name":"A genomic view of food-related and probiotic Enterococcus strains","datePublished":"2016-10-23","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2017-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw043","keywords":["<italic>Enterococcus</italic>","food","genomics","niche","probiotics"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/1/11/2670108","author":[{"name":"Bonacina, Julieta","affiliation":"1 Laboratorio de Genética y Biología Molecular, CERELA-CONICET, Centro de Referencia para Lactobacilos, San Miguel de Tucumán (T4000ILC), Tucumán, Argentina","@type":"Person"},{"name":"Suárez, Nadia","affiliation":"1 Laboratorio de Genética y Biología Molecular, CERELA-CONICET, Centro de Referencia para Lactobacilos, San Miguel de Tucumán (T4000ILC), Tucumán, Argentina","@type":"Person"},{"name":"Hormigo, Ricardo","affiliation":"1 Laboratorio de Genética y Biología Molecular, CERELA-CONICET, Centro de Referencia para Lactobacilos, San Miguel de Tucumán (T4000ILC), Tucumán, Argentina","@type":"Person"},{"name":"Fadda, Silvina","affiliation":"1 Laboratorio de Genética y Biología Molecular, CERELA-CONICET, Centro de Referencia para Lactobacilos, San Miguel de Tucumán (T4000ILC), Tucumán, Argentina","@type":"Person"},{"name":"Lechner, Marcus","affiliation":"2 Department of Pharmaceutical Chemistry, Philipps-University Marburg, 35037 Marburg, Germany","@type":"Person"},{"name":"Saavedra, Lucila","affiliation":"1 Laboratorio de Genética y Biología Molecular, CERELA-CONICET, Centro de Referencia para Lactobacilos, San Miguel de Tucumán (T4000ILC), Tucumán, Argentina","@type":"Person"}],"description":"Abstract.  The study of enterococcal genomes has grown considerably in recent years. While special attention is paid to comparative genomic analysis among clini","pageStart":"11","pageEnd":"24","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/1/1/m_dnares_24_1cover.png?Expires=2147483647&Signature=EXqRr~0bCuMnRgDilKfchgK8HfHjGx8NkkSlq7yX2eX9zgSD-cVPXfhVwj7kuo0oVIPzSixZgq5R9TFiP~OOgz8ybfHpezxNM2sixr9SShZ0K1LhBnO6322A9GuHybRZq3DsAqdkcL3vL0duBS2dzcsmON4yge6VoGCPo6kzuwcmoM-0E9z7h2t0RSYSTFF896ydP484zApd5VOOafFGZ8RyK5rTzaKEJMAeldpSZjhoijN~lS-XpQOIA9~KMdeI0frmy3d5rUnXtXBDnLYMvKmhHEY2pEtZAo9h7w-9UPONxolZtZkyNv5UUIS3z9gJT3HGO3QFQvDoa8hGk7vH6Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"A genomic view of food-related and probiotic Enterococcus strains","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/1/1/m_dnares_24_1cover.png?Expires=2147483647&Signature=EXqRr~0bCuMnRgDilKfchgK8HfHjGx8NkkSlq7yX2eX9zgSD-cVPXfhVwj7kuo0oVIPzSixZgq5R9TFiP~OOgz8ybfHpezxNM2sixr9SShZ0K1LhBnO6322A9GuHybRZq3DsAqdkcL3vL0duBS2dzcsmON4yge6VoGCPo6kzuwcmoM-0E9z7h2t0RSYSTFF896ydP484zApd5VOOafFGZ8RyK5rTzaKEJMAeldpSZjhoijN~lS-XpQOIA9~KMdeI0frmy3d5rUnXtXBDnLYMvKmhHEY2pEtZAo9h7w-9UPONxolZtZkyNv5UUIS3z9gJT3HGO3QFQvDoa8hGk7vH6Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="genomic view of food-related and probiotic Enterococcus strains" />
    <meta property="og:description" content="Abstract.  The study of enterococcal genomes has grown considerably in recent years. While special attention is paid to comparative genomic analysis among clini" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/1/11/2670108" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-10-23" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Bonacina, Julieta" /><meta name="citation_author_institution" content="Laboratorio de Genética y Biología Molecular, CERELA-CONICET, Centro de Referencia para Lactobacilos, San Miguel de Tucumán (T4000ILC), Tucumán, Argentina" /><meta name="citation_author" content="Suárez, Nadia" /><meta name="citation_author_institution" content="Laboratorio de Genética y Biología Molecular, CERELA-CONICET, Centro de Referencia para Lactobacilos, San Miguel de Tucumán (T4000ILC), Tucumán, Argentina" /><meta name="citation_author" content="Hormigo, Ricardo" /><meta name="citation_author_institution" content="Laboratorio de Genética y Biología Molecular, CERELA-CONICET, Centro de Referencia para Lactobacilos, San Miguel de Tucumán (T4000ILC), Tucumán, Argentina" /><meta name="citation_author" content="Fadda, Silvina" /><meta name="citation_author_institution" content="Laboratorio de Genética y Biología Molecular, CERELA-CONICET, Centro de Referencia para Lactobacilos, San Miguel de Tucumán (T4000ILC), Tucumán, Argentina" /><meta name="citation_author" content="Lechner, Marcus" /><meta name="citation_author_institution" content="Department of Pharmaceutical Chemistry, Philipps-University Marburg, 35037 Marburg, Germany" /><meta name="citation_author" content="Saavedra, Lucila" /><meta name="citation_author_institution" content="Laboratorio de Genética y Biología Molecular, CERELA-CONICET, Centro de Referencia para Lactobacilos, San Miguel de Tucumán (T4000ILC), Tucumán, Argentina" /><meta name="citation_title" content="A genomic view of food-related and probiotic Enterococcus strains" /><meta name="citation_firstpage" content="11" /><meta name="citation_lastpage" content="24" /><meta name="citation_doi" content="10.1093/dnares/dsw043" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2017/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=The role and application of enterococci in food and health;  citation_journal_title=Int. J. Food Microbiol;  citation_year=2006;  citation_volume=106;  citation_pages=1-24; " /><meta name="citation_reference" content="citation_title=Friend turned foe: evolution of enterococcal virulence and antibiotic resistance;  citation_journal_title=Annu. Rev. Microbiol;  citation_year=2014;  citation_volume=68;  citation_pages=337-56; " /><meta name="citation_reference" content="citation_title=Functional and safety aspects of enterococci in dairy foods;  citation_journal_title=Indian J. Microbiol;  citation_year=2008;  citation_volume=48;  citation_pages=317-325; " /><meta name="citation_reference" content="citation_title=Draft genome sequence of probiotic Enterococcus faecium strain L-3;  citation_journal_title=Genome Announ;  citation_year=2016;  citation_volume=4; " /><meta name="citation_reference" content="citation_title=Influence of different probiotic lactic Acid bacteria on microbiota and metabolism of rats with dysbiosis;  citation_journal_title=Biosci Microbiota Food Health;  citation_year=2013;  citation_volume=32;  citation_pages=41-9; " /><meta name="citation_reference" content="citation_publisher=Boston;  citation_title=Enterococci: From Commensals to Leading Causes of Drug Resistant Infection;  citation_year=2014; " /><meta name="citation_reference" content="citation_title=EDGAR: a software framework for the comparative analysis of prokaryotic genomes;  citation_journal_title=BMC Bioinformatics;  citation_year=2009;  citation_volume=10;  citation_pages=154" /><meta name="citation_reference" content="citation_title=Genome analysis of multiple pathogenic isolates of Streptococcus agalactiae: implications for the microbial “pan-genome”;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2005;  citation_volume=102;  citation_pages=13950-55; " /><meta name="citation_reference" content="citation_title=Unique core genomes of the bacterial family Vibrionaceae: insights into niche adaptation and speciation;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=179" /><meta name="citation_reference" content="citation_title=Comparative genomics and functional analysis of niche-specific adaptation in Pseudomonas putida;  citation_journal_title=FEMS Microbiol. Rev;  citation_year=2011;  citation_volume=35;  citation_pages=299-323; " /><meta name="citation_reference" content="citation_title=Functional genomics of lactic acid bacteria: from food to health;  citation_journal_title=Microb. Cell Factories;  citation_year=2014;  citation_volume=13;  citation_pages=S8" /><meta name="citation_reference" content="citation_title=Nonclinical and clinical Enterococcus faecium strains, but not Enterococcus faecalis strains, have distinct structural and functional genomic features;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2014;  citation_volume=80;  citation_pages=154-65; " /><meta name="citation_reference" content="citation_title=Selection of potential probiotic Enterococcus faecium isolated from Portuguese fermented food;  citation_journal_title=Int. J. Food Microbiol;  citation_year=2014;  citation_volume=191;  citation_pages=144-48; " /><meta name="citation_reference" content="citation_title=Probiotic assessment of Lactobacillus plantarum 15HN and Enterococcus mundtii 50H isolated from traditional dairies microbiota;  citation_journal_title=Adv. Pharm. Bull;  citation_year=2016;  citation_volume=6;  citation_pages=37-47; " /><meta name="citation_reference" content="citation_title=The RAST Server: rapid annotations using subsystems technology;  citation_journal_title=BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=75" /><meta name="citation_reference" content="citation_title=Mauve: multiple alignment of conserved genomic sequence with rearrangements;  citation_journal_title=Genome Res;  citation_year=2004;  citation_volume=14;  citation_pages=1394-403; " /><meta name="citation_reference" content="citation_title=RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=1312-13; " /><meta name="citation_reference" content="citation_title=An improved general amino acid replacement matrix;  citation_journal_title=Mol. Biol. Evol;  citation_year=2008;  citation_volume=25;  citation_pages=1307-20; " /><meta name="citation_reference" content="citation_title=Proteinortho: detection of (co-)orthologs in large-scale analysis;  citation_journal_title=BMC Bioinformat;  citation_year=2011;  citation_volume=12;  citation_pages=124" /><meta name="citation_reference" content="citation_title=Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega;  citation_journal_title=Mol. Syst. Biol;  citation_year=2011;  citation_volume=7;  citation_pages=539" /><meta name="citation_reference" content="citation_title=BAGEL3: Automated identification of genes encoding bacteriocins and (non-)bactericidal posttranslationally modified peptides;  citation_journal_title=Nucleic Acids Res;  citation_year=2013;  citation_volume=41;  citation_pages=W448-453; " /><meta name="citation_reference" content="citation_title=Basic local alignment search tool;  citation_journal_title=J. Mol. Biol;  citation_year=1990;  citation_volume=215;  citation_pages=403-10; " /><meta name="citation_reference" content="citation_title=Biochemical and genetic characterization of enterocin A from Enterococcus faecium, a new antilisterial bacteriocin in the pediocin family of bacteriocins;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=1996;  citation_volume=62;  citation_pages=1676-82; " /><meta name="citation_reference" content="citation_title=ARG-ANNOT, a new bioinformatic tool to discover antibiotic resistance genes in bacterial genomes;  citation_journal_title=Antimicrobial Agents Chemother;  citation_year=2014;  citation_volume=58;  citation_pages=212-20; " /><meta name="citation_reference" content="Hall T.  2013 , BioEdit : biological sequence alignment editor for Win95/98/NT/2K/XP. pp. p 95–98 " /><meta name="citation_reference" content="citation_title=Amino acid substitution matrices from protein blocks;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=1992;  citation_volume=89;  citation_pages=10915-19; " /><meta name="citation_reference" content="citation_title=PHAST: a fast phage search tool;  citation_journal_title=Nucleic Acids Res;  citation_year=2011;  citation_volume=39;  citation_pages=W347-52; " /><meta name="citation_reference" content="citation_title=Microbial gene identification using interpolated Markov models;  citation_journal_title=Nucleic Acids Res;  citation_year=1998;  citation_volume=26;  citation_pages=544-48; " /><meta name="citation_reference" content="citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence;  citation_journal_title=Nucleic Acids Res;  citation_year=1997;  citation_volume=25;  citation_pages=955-64; " /><meta name="citation_reference" content="citation_title=ARAGORN, a program to detect tRNA genes and tmRNA genes in nucleotide sequences;  citation_journal_title=Nucleic Acids Res;  citation_year=2004;  citation_volume=32;  citation_pages=11-6; " /><meta name="citation_reference" content="citation_title=CRISPRFinder: a web tool to identify clustered regularly interspaced short palindromic repeats;  citation_journal_title=Nucleic Acids Res;  citation_year=2007;  citation_volume=35;  citation_pages=W52-57; " /><meta name="citation_reference" content="citation_title=IslandViewer 3: more flexible, interactive genomic island discovery, visualization and analysis;  citation_journal_title=Nucleic Acids Res;  citation_year=2015;  citation_volume=43;  citation_pages=W104-8; " /><meta name="citation_reference" content="citation_title=IslandPath: aiding detection of genomic islands in prokaryotes;  citation_journal_title=Bioinformatics;  citation_year=2003;  citation_volume=19;  citation_pages=418-20; " /><meta name="citation_reference" content="citation_title=Score-based prediction of genomic islands in prokaryotic genomes using hidden Markov models;  citation_journal_title=BMC Bioinform;  citation_year=2006;  citation_volume=7;  citation_pages=142" /><meta name="citation_reference" content="citation_title=Evaluation of genomic island predictors using a comparative genomics approach;  citation_journal_title=BMC Bioinform;  citation_year=2008;  citation_volume=9;  citation_pages=329" /><meta name="citation_reference" content="citation_title=IslandViewer update: Improved genomic island discovery and visualization;  citation_journal_title=Nucleic Acids Res;  citation_year=2013;  citation_volume=41;  citation_pages=W129-32; " /><meta name="citation_reference" content="citation_title=IslandViewer: an integrated interface for computational identification and visualization of genomic islands;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=664-65; " /><meta name="citation_reference" content="citation_title=Comprehensive molecular, genomic and phenotypic analysis of a major clone of Enterococcus faecalis MLST ST40;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=175" /><meta name="citation_reference" content="citation_publisher=Springer, Heidelberg;  citation_title=Bergey's Manual of Systematic Bacteriology;  citation_year=2009;  citation_pages=600" /><meta name="citation_reference" content="citation_title=Phylogeny and identification of Enterococci by atpA gene sequence analysis;  citation_journal_title=J. Clin. Microbiol;  citation_year=2005;  citation_volume=43;  citation_pages=2224-30; " /><meta name="citation_reference" content="citation_title=Comparative genomics of enterococci: variation in Enterococcus faecalis, clade structure in E. faecium, and defining characteristics of E. gallinarum and E. casseliflavus;  citation_journal_title=mBio;  citation_year=2012;  citation_volume=3;  citation_pages=e00318-00311; " /><meta name="citation_reference" content="Ramsey M. Hartke A. Huycke M.  2014 , The Physiology and Metabolism of Enterococci. " /><meta name="citation_reference" content="citation_title=Genome sequence and comparative genome analysis of Lactobacillus casei: insights into their niche-associated evolution;  citation_journal_title=Genome Biol. Evol;  citation_year=2009;  citation_volume=1;  citation_pages=239-57; " /><meta name="citation_reference" content="citation_title=Multiple posttranscriptional regulatory mechanisms partner to control ethanolamine utilization in Enterococcus faecalis;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2009;  citation_volume=106;  citation_pages=4435-40; " /><meta name="citation_reference" content="citation_title=A biphasic epigenetic switch controls immunoevasion, virulence and niche adaptation in non-typeable Haemophilus influenzae;  citation_journal_title=Nat. Commun;  citation_year=2015;  citation_volume=6;  citation_pages=7828" /><meta name="citation_reference" content="citation_publisher=Marcel Dekker Inc, NY;  citation_title=Food Additives;  citation_year=2002;  citation_pages=988" /><meta name="citation_reference" content="citation_title=Occurrence of antilisterial structural bacteriocins genes in meat borne lactic acid bacteria;  citation_journal_title=Food Control;  citation_year=2015;  citation_volume=47;  citation_pages=53-9; " /><meta name="citation_reference" content="citation_title=The dual role of bacteriocins as anti- and probiotics;  citation_journal_title=Appl. Microbiol. Biotechnol;  citation_year=2008;  citation_volume=81;  citation_pages=591-606; " /><meta name="citation_reference" content="citation_title=Genome mining and transcriptional analysis of bacteriocin genes in Enterococcus faecium CRL1879,;  citation_journal_title=J. Data Mining Genomics Proteomics;  citation_year=2015;  citation_volume=6;  citation_pages=1-8; " /><meta name="citation_reference" content="citation_title=Phenotypic and genotypic characterization of bacteriocins in enterococcal isolates of different sources;  citation_journal_title=Indian J. Microbiol;  citation_year=2011;  citation_volume=51;  citation_pages=182-187; " /><meta name="citation_reference" content="citation_title=Draft genome sequence of the nonstarter bacteriocin-producing strain Enterococcus mundtii CRL35;  citation_journal_title=Genome Announ;  citation_year=2014;  citation_volume=2; " /><meta name="citation_reference" content="citation_title=Draft genome sequence of Enterococcus mundtii CRL1656;  citation_journal_title=J. Bacteriol;  citation_year=2012;  citation_volume=194;  citation_pages=550" /><meta name="citation_reference" content="citation_title=Modulation of virulence within a pathogenicity island in vancomycin-resistant Enterococcus faecalis;  citation_journal_title=Nature;  citation_year=2002;  citation_volume=417;  citation_pages=746-50; " /><meta name="citation_reference" content="citation_title=Virulence genes, antibiotic resistance and plasmid profiles of Enterococcus faecalis and Enterococcus faecium from naturally fermented Turkish foods,;  citation_journal_title=J. Appl. Microbiol;  citation_year=2010;  citation_volume=109;  citation_pages=1084-92; " /><meta name="citation_reference" content="citation_title=Investigating the mobilome in clinically important lineages of Enterococcus faecium and Enterococcus faecalis;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=282" /><meta name="citation_reference" content="citation_title=Enterococci from foods;  citation_journal_title=FEMS Microbiol. Rev;  citation_year=2002;  citation_volume=26;  citation_pages=163-71; " /><meta name="citation_reference" content="citation_title=Antibiotic susceptibility of enterococci isolated from traditional fermented meat products;  citation_journal_title=Food Microbiol;  citation_year=2009;  citation_volume=26;  citation_pages=527-32; " /><meta name="citation_reference" content="citation_title=Macrolide resistance genes in Enterococcus spp;  citation_journal_title=Antimicrob. Agents Chemother;  citation_year=2000;  citation_volume=44;  citation_pages=967-71; " /><meta name="citation_reference" content="citation_title=Aminoglycoside modifying enzymes;  citation_journal_title=Drug Resistance Updates;  citation_year=2010;  citation_volume=13;  citation_pages=151-71; " /><meta name="citation_reference" content="citation_title=Differences in the Enterococcus faecalis lsa locus that influence susceptibility to quinupristin-dalfopristin and clindamycin;  citation_journal_title=Antimicrob. Agents Chemother;  citation_year=2005;  citation_volume=49;  citation_pages=32-9; " /><meta name="citation_reference" content="citation_title=Update on macrolide-lincosamide-streptogramin, ketolide, and oxazolidinone resistance genes;  citation_journal_title=FEMS Microbiol. Lett;  citation_year=2008;  citation_volume=282;  citation_pages=147-59; " /><meta name="citation_reference" content="Duerkop B.A. Palmer K.L. Horsburgh M.J.  2014 , Enterococcal bacteriophages and genome defense. " /><meta name="citation_reference" content="citation_title=Genome and proteome of Listeria monocytogenes phage PSA: an unusual case for programmed + 1 translational frameshifting in structural protein synthesis;  citation_journal_title=Mol. Microbiol;  citation_year=2003;  citation_volume=50;  citation_pages=303-17; " /><meta name="citation_reference" content="citation_title=Comparative genomic and morphological analyses of Listeria phages isolated from farm environments;  citation_journal_title=Appl. Environm. Microbiol;  citation_year=2014;  citation_volume=80;  citation_pages=4616-25; " /><meta name="citation_reference" content="Palmer K. van Schaik W. Willems R. Gilmore M.  2014. Enterococcal Genomics. In: M. Gilmore, D. Clewell, Y. Ike and N. Shankar, ed., Enterococci From Commensals to Leading Causes of Drug Resistant Infection, 1st ed. [online] Boston, Massachusetts: The Harvard Medical School, pp. 189-216. " /><meta name="citation_reference" content="citation_title=Unification of Cas protein families and a simple scenario for the origin and evolution of CRISPR-Cas systems;  citation_journal_title=Biol. Direct;  citation_year=2011;  citation_volume=6;  citation_pages=38" /><meta name="citation_reference" content="citation_title=An updated evolutionary classification of CRISPR-Cas systems;  citation_journal_title=Nat. Rev. Microbiol;  citation_year=2015;  citation_volume=13;  citation_pages=722-36; " /><meta name="citation_reference" content="citation_title=Genomic islands: tools of bacterial horizontal gene transfer and evolution;  citation_journal_title=FEMS Microbiol. Rev;  citation_year=2009;  citation_volume=33;  citation_pages=376-93; " /><meta name="citation_reference" content="citation_title=Gene islands integrated into tRNA(Gly) genes confer genome diversity on a Pseudomonas aeruginosa clone;  citation_journal_title=J. Bacteriol;  citation_year=2002;  citation_volume=184;  citation_pages=6665-80; " /><meta name="citation_reference" content="citation_title=Comparative sequence analysis of the symbiosis island of Mesorhizobium loti strain R7A;  citation_journal_title=J. Bacteriol;  citation_year=2002;  citation_volume=184;  citation_pages=3086-95; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/1/11/10731780/dsw043.pdf" /><meta name="description" content="Abstract.  The study of enterococcal genomes has grown considerably in recent years. While special attention is paid to comparative genomic analysis among clini" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/1/11/2670108" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/1/11/2670108" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/1/11/2670108");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw043';
    </script>
<title>
	genomic view of food-related and probiotic Enterococcus strains | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2670108" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BbudATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQm9uYWNpbmEsIEp1bGlldGEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJMYWJvcmF0b3JpbyBkZSBHZW7DqXRpY2EgeSBCaW9sb2fDrWEgTW9sZWN1bGFyLCBDRVJFTEEtQ09OSUNFVCwgQ2VudHJvIGRlIFJlZmVyZW5jaWEgcGFyYSBMYWN0b2JhY2lsb3MsIFNhbiBNaWd1ZWwgZGUgVHVjdW3DoW4gKFQ0MDAwSUxDKSwgVHVjdW3DoW4sIEFyZ2VudGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlN1w6FyZXosIE5hZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTGFib3JhdG9yaW8gZGUgR2Vuw6l0aWNhIHkgQmlvbG9nw61hIE1vbGVjdWxhciwgQ0VSRUxBLUNPTklDRVQsIENlbnRybyBkZSBSZWZlcmVuY2lhIHBhcmEgTGFjdG9iYWNpbG9zLCBTYW4gTWlndWVsIGRlIFR1Y3Vtw6FuIChUNDAwMElMQyksIFR1Y3Vtw6FuLCBBcmdlbnRpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIb3JtaWdvLCBSaWNhcmRvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTGFib3JhdG9yaW8gZGUgR2Vuw6l0aWNhIHkgQmlvbG9nw61hIE1vbGVjdWxhciwgQ0VSRUxBLUNPTklDRVQsIENlbnRybyBkZSBSZWZlcmVuY2lhIHBhcmEgTGFjdG9iYWNpbG9zLCBTYW4gTWlndWVsIGRlIFR1Y3Vtw6FuIChUNDAwMElMQyksIFR1Y3Vtw6FuLCBBcmdlbnRpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGYWRkYSwgU2lsdmluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcmlvIGRlIEdlbsOpdGljYSB5IEJpb2xvZ8OtYSBNb2xlY3VsYXIsIENFUkVMQS1DT05JQ0VULCBDZW50cm8gZGUgUmVmZXJlbmNpYSBwYXJhIExhY3RvYmFjaWxvcywgU2FuIE1pZ3VlbCBkZSBUdWN1bcOhbiAoVDQwMDBJTEMpLCBUdWN1bcOhbiwgQXJnZW50aW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGVjaG5lciwgTWFyY3VzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBQaGFybWFjZXV0aWNhbCBDaGVtaXN0cnksIFBoaWxpcHBzLVVuaXZlcnNpdHkgTWFyYnVyZywgMzUwMzcgTWFyYnVyZywgR2VybWFueSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNhYXZlZHJhLCBMdWNpbGEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJMYWJvcmF0b3JpbyBkZSBHZW7DqXRpY2EgeSBCaW9sb2fDrWEgTW9sZWN1bGFyLCBDRVJFTEEtQ09OSUNFVCwgQ2VudHJvIGRlIFJlZmVyZW5jaWEgcGFyYSBMYWN0b2JhY2lsb3MsIFNhbiBNaWd1ZWwgZGUgVHVjdW3DoW4gKFQ0MDAwSUxDKSwgVHVjdW3DoW4sIEFyZ2VudGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iQSBnZW5vbWljIHZpZXcgb2YgZm9vZC1yZWxhdGVkIGFuZCBwcm9iaW90aWMgRW50ZXJvY29jY3VzIHN0cmFpbnMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIxMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3cwNDMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNy8wMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSByb2xlIGFuZCBhcHBsaWNhdGlvbiBvZiBlbnRlcm9jb2NjaSBpbiBmb29kIGFuZCBoZWFsdGg7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUludC4gSi4gRm9vZCBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTA2OyAgY2l0YXRpb25fcGFnZXM9MS0yNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnJpZW5kIHR1cm5lZCBmb2U6IGV2b2x1dGlvbiBvZiBlbnRlcm9jb2NjYWwgdmlydWxlbmNlIGFuZCBhbnRpYmlvdGljIHJlc2lzdGFuY2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFubnUuIFJldi4gTWljcm9iaW9sOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTY4OyAgY2l0YXRpb25fcGFnZXM9MzM3LTU2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdW5jdGlvbmFsIGFuZCBzYWZldHkgYXNwZWN0cyBvZiBlbnRlcm9jb2NjaSBpbiBkYWlyeSBmb29kczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SW5kaWFuIEouIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00ODsgIGNpdGF0aW9uX3BhZ2VzPTMxNy0zMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURyYWZ0IGdlbm9tZSBzZXF1ZW5jZSBvZiBwcm9iaW90aWMgRW50ZXJvY29jY3VzIGZhZWNpdW0gc3RyYWluIEwtMzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEFubm91bjsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmZsdWVuY2Ugb2YgZGlmZmVyZW50IHByb2Jpb3RpYyBsYWN0aWMgQWNpZCBiYWN0ZXJpYSBvbiBtaWNyb2Jpb3RhIGFuZCBtZXRhYm9saXNtIG9mIHJhdHMgd2l0aCBkeXNiaW9zaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb3NjaSBNaWNyb2Jpb3RhIEZvb2QgSGVhbHRoOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9NDEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUJvc3RvbjsgIGNpdGF0aW9uX3RpdGxlPUVudGVyb2NvY2NpOiBGcm9tIENvbW1lbnNhbHMgdG8gTGVhZGluZyBDYXVzZXMgb2YgRHJ1ZyBSZXNpc3RhbnQgSW5mZWN0aW9uOyAgY2l0YXRpb25feWVhcj0yMDE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FREdBUjogYSBzb2Z0d2FyZSBmcmFtZXdvcmsgZm9yIHRoZSBjb21wYXJhdGl2ZSBhbmFseXNpcyBvZiBwcm9rYXJ5b3RpYyBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0xNTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgYW5hbHlzaXMgb2YgbXVsdGlwbGUgcGF0aG9nZW5pYyBpc29sYXRlcyBvZiBTdHJlcHRvY29jY3VzIGFnYWxhY3RpYWU6IGltcGxpY2F0aW9ucyBmb3IgdGhlIG1pY3JvYmlhbCDigJxwYW4tZ2Vub21l4oCdOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDI7ICBjaXRhdGlvbl9wYWdlcz0xMzk1MC01NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VW5pcXVlIGNvcmUgZ2Vub21lcyBvZiB0aGUgYmFjdGVyaWFsIGZhbWlseSBWaWJyaW9uYWNlYWU6IGluc2lnaHRzIGludG8gbmljaGUgYWRhcHRhdGlvbiBhbmQgc3BlY2lhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MTc5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgZ2Vub21pY3MgYW5kIGZ1bmN0aW9uYWwgYW5hbHlzaXMgb2YgbmljaGUtc3BlY2lmaWMgYWRhcHRhdGlvbiBpbiBQc2V1ZG9tb25hcyBwdXRpZGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZFTVMgTWljcm9iaW9sLiBSZXY7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz0yOTktMzIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdW5jdGlvbmFsIGdlbm9taWNzIG9mIGxhY3RpYyBhY2lkIGJhY3RlcmlhOiBmcm9tIGZvb2QgdG8gaGVhbHRoOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1NaWNyb2IuIENlbGwgRmFjdG9yaWVzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9UzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ob25jbGluaWNhbCBhbmQgY2xpbmljYWwgRW50ZXJvY29jY3VzIGZhZWNpdW0gc3RyYWlucywgYnV0IG5vdCBFbnRlcm9jb2NjdXMgZmFlY2FsaXMgc3RyYWlucywgaGF2ZSBkaXN0aW5jdCBzdHJ1Y3R1cmFsIGFuZCBmdW5jdGlvbmFsIGdlbm9taWMgZmVhdHVyZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFwcGwuIEVudmlyb24uIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT04MDsgIGNpdGF0aW9uX3BhZ2VzPTE1NC02NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VsZWN0aW9uIG9mIHBvdGVudGlhbCBwcm9iaW90aWMgRW50ZXJvY29jY3VzIGZhZWNpdW0gaXNvbGF0ZWQgZnJvbSBQb3J0dWd1ZXNlIGZlcm1lbnRlZCBmb29kOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JbnQuIEouIEZvb2QgTWljcm9iaW9sOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE5MTsgIGNpdGF0aW9uX3BhZ2VzPTE0NC00ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvYmlvdGljIGFzc2Vzc21lbnQgb2YgTGFjdG9iYWNpbGx1cyBwbGFudGFydW0gMTVITiBhbmQgRW50ZXJvY29jY3VzIG11bmR0aWkgNTBIIGlzb2xhdGVkIGZyb20gdHJhZGl0aW9uYWwgZGFpcmllcyBtaWNyb2Jpb3RhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BZHYuIFBoYXJtLiBCdWxsOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0zNy00NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFJBU1QgU2VydmVyOiByYXBpZCBhbm5vdGF0aW9ucyB1c2luZyBzdWJzeXN0ZW1zIHRlY2hub2xvZ3k7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9NzUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXV2ZTogbXVsdGlwbGUgYWxpZ25tZW50IG9mIGNvbnNlcnZlZCBnZW5vbWljIHNlcXVlbmNlIHdpdGggcmVhcnJhbmdlbWVudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0xMzk0LTQwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UkF4TUwgdmVyc2lvbiA4OiBhIHRvb2wgZm9yIHBoeWxvZ2VuZXRpYyBhbmFseXNpcyBhbmQgcG9zdC1hbmFseXNpcyBvZiBsYXJnZSBwaHlsb2dlbmllczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0xMzEyLTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBpbXByb3ZlZCBnZW5lcmFsIGFtaW5vIGFjaWQgcmVwbGFjZW1lbnQgbWF0cml4OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xMzA3LTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm90ZWlub3J0aG86IGRldGVjdGlvbiBvZiAoY28tKW9ydGhvbG9ncyBpbiBsYXJnZS1zY2FsZSBhbmFseXNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEJpb2luZm9ybWF0OyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmFzdCwgc2NhbGFibGUgZ2VuZXJhdGlvbiBvZiBoaWdoLXF1YWxpdHkgcHJvdGVpbiBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnRzIHVzaW5nIENsdXN0YWwgT21lZ2E7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gU3lzdC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9NTM5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QkFHRUwzOiBBdXRvbWF0ZWQgaWRlbnRpZmljYXRpb24gb2YgZ2VuZXMgZW5jb2RpbmcgYmFjdGVyaW9jaW5zIGFuZCAobm9uLSliYWN0ZXJpY2lkYWwgcG9zdHRyYW5zbGF0aW9uYWxseSBtb2RpZmllZCBwZXB0aWRlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDE7ICBjaXRhdGlvbl9wYWdlcz1XNDQ4LTQ1MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFzaWMgbG9jYWwgYWxpZ25tZW50IHNlYXJjaCB0b29sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9MjE1OyAgY2l0YXRpb25fcGFnZXM9NDAzLTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CaW9jaGVtaWNhbCBhbmQgZ2VuZXRpYyBjaGFyYWN0ZXJpemF0aW9uIG9mIGVudGVyb2NpbiBBIGZyb20gRW50ZXJvY29jY3VzIGZhZWNpdW0sIGEgbmV3IGFudGlsaXN0ZXJpYWwgYmFjdGVyaW9jaW4gaW4gdGhlIHBlZGlvY2luIGZhbWlseSBvZiBiYWN0ZXJpb2NpbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFwcGwuIEVudmlyb24uIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT02MjsgIGNpdGF0aW9uX3BhZ2VzPTE2NzYtODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFSRy1BTk5PVCwgYSBuZXcgYmlvaW5mb3JtYXRpYyB0b29sIHRvIGRpc2NvdmVyIGFudGliaW90aWMgcmVzaXN0YW5jZSBnZW5lcyBpbiBiYWN0ZXJpYWwgZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW50aW1pY3JvYmlhbCBBZ2VudHMgQ2hlbW90aGVyOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTU4OyAgY2l0YXRpb25fcGFnZXM9MjEyLTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJIYWxsIFQuICAyMDEzICwgQmlvRWRpdCA6IGJpb2xvZ2ljYWwgc2VxdWVuY2UgYWxpZ25tZW50IGVkaXRvciBmb3IgV2luOTUvOTgvTlQvMksvWFAuIHBwLiBwIDk14oCTOTggIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW1pbm8gYWNpZCBzdWJzdGl0dXRpb24gbWF0cmljZXMgZnJvbSBwcm90ZWluIGJsb2NrczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9ODk7ICBjaXRhdGlvbl9wYWdlcz0xMDkxNS0xOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UEhBU1Q6IGEgZmFzdCBwaGFnZSBzZWFyY2ggdG9vbDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz1XMzQ3LTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb2JpYWwgZ2VuZSBpZGVudGlmaWNhdGlvbiB1c2luZyBpbnRlcnBvbGF0ZWQgTWFya292IG1vZGVsczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz01NDQtNDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPXRSTkFzY2FuLVNFOiBhIHByb2dyYW0gZm9yIGltcHJvdmVkIGRldGVjdGlvbiBvZiB0cmFuc2ZlciBSTkEgZ2VuZXMgaW4gZ2Vub21pYyBzZXF1ZW5jZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz05NTUtNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFSQUdPUk4sIGEgcHJvZ3JhbSB0byBkZXRlY3QgdFJOQSBnZW5lcyBhbmQgdG1STkEgZ2VuZXMgaW4gbnVjbGVvdGlkZSBzZXF1ZW5jZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9MTEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q1JJU1BSRmluZGVyOiBhIHdlYiB0b29sIHRvIGlkZW50aWZ5IGNsdXN0ZXJlZCByZWd1bGFybHkgaW50ZXJzcGFjZWQgc2hvcnQgcGFsaW5kcm9taWMgcmVwZWF0czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz1XNTItNTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlzbGFuZFZpZXdlciAzOiBtb3JlIGZsZXhpYmxlLCBpbnRlcmFjdGl2ZSBnZW5vbWljIGlzbGFuZCBkaXNjb3ZlcnksIHZpc3VhbGl6YXRpb24gYW5kIGFuYWx5c2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPVcxMDQtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNsYW5kUGF0aDogYWlkaW5nIGRldGVjdGlvbiBvZiBnZW5vbWljIGlzbGFuZHMgaW4gcHJva2FyeW90ZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9NDE4LTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TY29yZS1iYXNlZCBwcmVkaWN0aW9uIG9mIGdlbm9taWMgaXNsYW5kcyBpbiBwcm9rYXJ5b3RpYyBnZW5vbWVzIHVzaW5nIGhpZGRlbiBNYXJrb3YgbW9kZWxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgQmlvaW5mb3JtOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0xNDIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdmFsdWF0aW9uIG9mIGdlbm9taWMgaXNsYW5kIHByZWRpY3RvcnMgdXNpbmcgYSBjb21wYXJhdGl2ZSBnZW5vbWljcyBhcHByb2FjaDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEJpb2luZm9ybTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MzI5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNsYW5kVmlld2VyIHVwZGF0ZTogSW1wcm92ZWQgZ2Vub21pYyBpc2xhbmQgZGlzY292ZXJ5IGFuZCB2aXN1YWxpemF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00MTsgIGNpdGF0aW9uX3BhZ2VzPVcxMjktMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlzbGFuZFZpZXdlcjogYW4gaW50ZWdyYXRlZCBpbnRlcmZhY2UgZm9yIGNvbXB1dGF0aW9uYWwgaWRlbnRpZmljYXRpb24gYW5kIHZpc3VhbGl6YXRpb24gb2YgZ2Vub21pYyBpc2xhbmRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTY2NC02NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcHJlaGVuc2l2ZSBtb2xlY3VsYXIsIGdlbm9taWMgYW5kIHBoZW5vdHlwaWMgYW5hbHlzaXMgb2YgYSBtYWpvciBjbG9uZSBvZiBFbnRlcm9jb2NjdXMgZmFlY2FsaXMgTUxTVCBTVDQwOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0xNzUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9U3ByaW5nZXIsIEhlaWRlbGJlcmc7ICBjaXRhdGlvbl90aXRsZT1CZXJnZXkncyBNYW51YWwgb2YgU3lzdGVtYXRpYyBCYWN0ZXJpb2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl9wYWdlcz02MDAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaHlsb2dlbnkgYW5kIGlkZW50aWZpY2F0aW9uIG9mIEVudGVyb2NvY2NpIGJ5IGF0cEEgZ2VuZSBzZXF1ZW5jZSBhbmFseXNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQ2xpbi4gTWljcm9iaW9sOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9MjIyNC0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgZ2Vub21pY3Mgb2YgZW50ZXJvY29jY2k6IHZhcmlhdGlvbiBpbiBFbnRlcm9jb2NjdXMgZmFlY2FsaXMsIGNsYWRlIHN0cnVjdHVyZSBpbiBFLiBmYWVjaXVtLCBhbmQgZGVmaW5pbmcgY2hhcmFjdGVyaXN0aWNzIG9mIEUuIGdhbGxpbmFydW0gYW5kIEUuIGNhc3NlbGlmbGF2dXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPW1CaW87ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPWUwMDMxOC0wMDMxMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iUmFtc2V5IE0uIEhhcnRrZSBBLiBIdXlja2UgTS4gIDIwMTQgLCBUaGUgUGh5c2lvbG9neSBhbmQgTWV0YWJvbGlzbSBvZiBFbnRlcm9jb2NjaS4gIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHNlcXVlbmNlIGFuZCBjb21wYXJhdGl2ZSBnZW5vbWUgYW5hbHlzaXMgb2YgTGFjdG9iYWNpbGx1cyBjYXNlaTogaW5zaWdodHMgaW50byB0aGVpciBuaWNoZS1hc3NvY2lhdGVkIGV2b2x1dGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTIzOS01NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXVsdGlwbGUgcG9zdHRyYW5zY3JpcHRpb25hbCByZWd1bGF0b3J5IG1lY2hhbmlzbXMgcGFydG5lciB0byBjb250cm9sIGV0aGFub2xhbWluZSB1dGlsaXphdGlvbiBpbiBFbnRlcm9jb2NjdXMgZmFlY2FsaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwNjsgIGNpdGF0aW9uX3BhZ2VzPTQ0MzUtNDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgYmlwaGFzaWMgZXBpZ2VuZXRpYyBzd2l0Y2ggY29udHJvbHMgaW1tdW5vZXZhc2lvbiwgdmlydWxlbmNlIGFuZCBuaWNoZSBhZGFwdGF0aW9uIGluIG5vbi10eXBlYWJsZSBIYWVtb3BoaWx1cyBpbmZsdWVuemFlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIENvbW11bjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9NzgyOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1NYXJjZWwgRGVra2VyIEluYywgTlk7ICBjaXRhdGlvbl90aXRsZT1Gb29kIEFkZGl0aXZlczsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3BhZ2VzPTk4OCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9jY3VycmVuY2Ugb2YgYW50aWxpc3RlcmlhbCBzdHJ1Y3R1cmFsIGJhY3RlcmlvY2lucyBnZW5lcyBpbiBtZWF0IGJvcm5lIGxhY3RpYyBhY2lkIGJhY3RlcmlhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Gb29kIENvbnRyb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NDc7ICBjaXRhdGlvbl9wYWdlcz01My05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZHVhbCByb2xlIG9mIGJhY3RlcmlvY2lucyBhcyBhbnRpLSBhbmQgcHJvYmlvdGljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QXBwbC4gTWljcm9iaW9sLiBCaW90ZWNobm9sOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTgxOyAgY2l0YXRpb25fcGFnZXM9NTkxLTYwNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIG1pbmluZyBhbmQgdHJhbnNjcmlwdGlvbmFsIGFuYWx5c2lzIG9mIGJhY3RlcmlvY2luIGdlbmVzIGluIEVudGVyb2NvY2N1cyBmYWVjaXVtIENSTDE4NzksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBEYXRhIE1pbmluZyBHZW5vbWljcyBQcm90ZW9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0xLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoZW5vdHlwaWMgYW5kIGdlbm90eXBpYyBjaGFyYWN0ZXJpemF0aW9uIG9mIGJhY3RlcmlvY2lucyBpbiBlbnRlcm9jb2NjYWwgaXNvbGF0ZXMgb2YgZGlmZmVyZW50IHNvdXJjZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUluZGlhbiBKLiBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NTE7ICBjaXRhdGlvbl9wYWdlcz0xODItMTg3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EcmFmdCBnZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIG5vbnN0YXJ0ZXIgYmFjdGVyaW9jaW4tcHJvZHVjaW5nIHN0cmFpbiBFbnRlcm9jb2NjdXMgbXVuZHRpaSBDUkwzNTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEFubm91bjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EcmFmdCBnZW5vbWUgc2VxdWVuY2Ugb2YgRW50ZXJvY29jY3VzIG11bmR0aWkgQ1JMMTY1NjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmFjdGVyaW9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE5NDsgIGNpdGF0aW9uX3BhZ2VzPTU1MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vZHVsYXRpb24gb2YgdmlydWxlbmNlIHdpdGhpbiBhIHBhdGhvZ2VuaWNpdHkgaXNsYW5kIGluIHZhbmNvbXljaW4tcmVzaXN0YW50IEVudGVyb2NvY2N1cyBmYWVjYWxpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTQxNzsgIGNpdGF0aW9uX3BhZ2VzPTc0Ni01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VmlydWxlbmNlIGdlbmVzLCBhbnRpYmlvdGljIHJlc2lzdGFuY2UgYW5kIHBsYXNtaWQgcHJvZmlsZXMgb2YgRW50ZXJvY29jY3VzIGZhZWNhbGlzIGFuZCBFbnRlcm9jb2NjdXMgZmFlY2l1bSBmcm9tIG5hdHVyYWxseSBmZXJtZW50ZWQgVHVya2lzaCBmb29kcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEFwcGwuIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDk7ICBjaXRhdGlvbl9wYWdlcz0xMDg0LTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnZlc3RpZ2F0aW5nIHRoZSBtb2JpbG9tZSBpbiBjbGluaWNhbGx5IGltcG9ydGFudCBsaW5lYWdlcyBvZiBFbnRlcm9jb2NjdXMgZmFlY2l1bSBhbmQgRW50ZXJvY29jY3VzIGZhZWNhbGlzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0yODIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbnRlcm9jb2NjaSBmcm9tIGZvb2RzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GRU1TIE1pY3JvYmlvbC4gUmV2OyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9MTYzLTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbnRpYmlvdGljIHN1c2NlcHRpYmlsaXR5IG9mIGVudGVyb2NvY2NpIGlzb2xhdGVkIGZyb20gdHJhZGl0aW9uYWwgZmVybWVudGVkIG1lYXQgcHJvZHVjdHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZvb2QgTWljcm9iaW9sOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9NTI3LTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYWNyb2xpZGUgcmVzaXN0YW5jZSBnZW5lcyBpbiBFbnRlcm9jb2NjdXMgc3BwOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BbnRpbWljcm9iLiBBZ2VudHMgQ2hlbW90aGVyOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTQ0OyAgY2l0YXRpb25fcGFnZXM9OTY3LTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbWlub2dseWNvc2lkZSBtb2RpZnlpbmcgZW56eW1lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RHJ1ZyBSZXNpc3RhbmNlIFVwZGF0ZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0xNTEtNzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpZmZlcmVuY2VzIGluIHRoZSBFbnRlcm9jb2NjdXMgZmFlY2FsaXMgbHNhIGxvY3VzIHRoYXQgaW5mbHVlbmNlIHN1c2NlcHRpYmlsaXR5IHRvIHF1aW51cHJpc3Rpbi1kYWxmb3ByaXN0aW4gYW5kIGNsaW5kYW15Y2luOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BbnRpbWljcm9iLiBBZ2VudHMgQ2hlbW90aGVyOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTQ5OyAgY2l0YXRpb25fcGFnZXM9MzItOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VXBkYXRlIG9uIG1hY3JvbGlkZS1saW5jb3NhbWlkZS1zdHJlcHRvZ3JhbWluLCBrZXRvbGlkZSwgYW5kIG94YXpvbGlkaW5vbmUgcmVzaXN0YW5jZSBnZW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RkVNUyBNaWNyb2Jpb2wuIExldHQ7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjgyOyAgY2l0YXRpb25fcGFnZXM9MTQ3LTU5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJEdWVya29wIEIuQS4gUGFsbWVyIEsuTC4gSG9yc2J1cmdoIE0uSi4gIDIwMTQgLCBFbnRlcm9jb2NjYWwgYmFjdGVyaW9waGFnZXMgYW5kIGdlbm9tZSBkZWZlbnNlLiAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgYW5kIHByb3Rlb21lIG9mIExpc3RlcmlhIG1vbm9jeXRvZ2VuZXMgcGhhZ2UgUFNBOiBhbiB1bnVzdWFsIGNhc2UgZm9yIHByb2dyYW1tZWQgKyAxIHRyYW5zbGF0aW9uYWwgZnJhbWVzaGlmdGluZyBpbiBzdHJ1Y3R1cmFsIHByb3RlaW4gc3ludGhlc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT01MDsgIGNpdGF0aW9uX3BhZ2VzPTMwMy0xNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgZ2Vub21pYyBhbmQgbW9ycGhvbG9naWNhbCBhbmFseXNlcyBvZiBMaXN0ZXJpYSBwaGFnZXMgaXNvbGF0ZWQgZnJvbSBmYXJtIGVudmlyb25tZW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QXBwbC4gRW52aXJvbm0uIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT04MDsgIGNpdGF0aW9uX3BhZ2VzPTQ2MTYtMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IlBhbG1lciBLLiB2YW4gU2NoYWlrIFcuIFdpbGxlbXMgUi4gR2lsbW9yZSBNLiAgMjAxNC4gRW50ZXJvY29jY2FsIEdlbm9taWNzLiBJbjogTS4gR2lsbW9yZSwgRC4gQ2xld2VsbCwgWS4gSWtlIGFuZCBOLiBTaGFua2FyLCBlZC4sIEVudGVyb2NvY2NpIEZyb20gQ29tbWVuc2FscyB0byBMZWFkaW5nIENhdXNlcyBvZiBEcnVnIFJlc2lzdGFudCBJbmZlY3Rpb24sIDFzdCBlZC4gW29ubGluZV0gQm9zdG9uLCBNYXNzYWNodXNldHRzOiBUaGUgSGFydmFyZCBNZWRpY2FsIFNjaG9vbCwgcHAuIDE4OS0yMTYuICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVuaWZpY2F0aW9uIG9mIENhcyBwcm90ZWluIGZhbWlsaWVzIGFuZCBhIHNpbXBsZSBzY2VuYXJpbyBmb3IgdGhlIG9yaWdpbiBhbmQgZXZvbHV0aW9uIG9mIENSSVNQUi1DYXMgc3lzdGVtczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvbC4gRGlyZWN0OyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0zOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIHVwZGF0ZWQgZXZvbHV0aW9uYXJ5IGNsYXNzaWZpY2F0aW9uIG9mIENSSVNQUi1DYXMgc3lzdGVtczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTcyMi0zNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBpc2xhbmRzOiB0b29scyBvZiBiYWN0ZXJpYWwgaG9yaXpvbnRhbCBnZW5lIHRyYW5zZmVyIGFuZCBldm9sdXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZFTVMgTWljcm9iaW9sLiBSZXY7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz0zNzYtOTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgaXNsYW5kcyBpbnRlZ3JhdGVkIGludG8gdFJOQShHbHkpIGdlbmVzIGNvbmZlciBnZW5vbWUgZGl2ZXJzaXR5IG9uIGEgUHNldWRvbW9uYXMgYWVydWdpbm9zYSBjbG9uZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmFjdGVyaW9sOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTE4NDsgIGNpdGF0aW9uX3BhZ2VzPTY2NjUtODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIHNlcXVlbmNlIGFuYWx5c2lzIG9mIHRoZSBzeW1iaW9zaXMgaXNsYW5kIG9mIE1lc29yaGl6b2JpdW0gbG90aSBzdHJhaW4gUjdBOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCYWN0ZXJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTg0OyAgY2l0YXRpb25fcGFnZXM9MzA4Ni05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzI0LzEvMTEvMTA3MzE3ODAvZHN3MDQzLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBUaGUgc3R1ZHkgb2YgZW50ZXJvY29jY2FsIGdlbm9tZXMgaGFzIGdyb3duIGNvbnNpZGVyYWJseSBpbiByZWNlbnQgeWVhcnMuIFdoaWxlIHNwZWNpYWwgYXR0ZW50aW9uIGlzIHBhaWQgdG8gY29tcGFyYXRpdmUgZ2Vub21pYyBhbmFseXNpcyBhbW9uZyBjbGluaSIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MjY3MDEwOCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGSwFi8OBGnwFmYoI/PfXHINLhNOLA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f1%2f11%2f2670108" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f1%2f11%2f2670108" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=96271" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/1/1/m_dnares_24_1cover.png?Expires=2147483647&amp;Signature=EXqRr~0bCuMnRgDilKfchgK8HfHjGx8NkkSlq7yX2eX9zgSD-cVPXfhVwj7kuo0oVIPzSixZgq5R9TFiP~OOgz8ybfHpezxNM2sixr9SShZ0K1LhBnO6322A9GuHybRZq3DsAqdkcL3vL0duBS2dzcsmON4yge6VoGCPo6kzuwcmoM-0E9z7h2t0RSYSTFF896ydP484zApd5VOOafFGZ8RyK5rTzaKEJMAeldpSZjhoijN~lS-XpQOIA9~KMdeI0frmy3d5rUnXtXBDnLYMvKmhHEY2pEtZAo9h7w-9UPONxolZtZkyNv5UUIS3z9gJT3HGO3QFQvDoa8hGk7vH6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/1">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="61462822">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61462822">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61462824">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61462824">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61462829">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61462829">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61462838">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61462838">3. Niche-specific proteins and core proteome</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61462841">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61462841">4. Genomic features</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61462850">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61462850">5. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61462869">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61462869">6. Genomic features among food-related and probiotic enterococcal strains</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61462889">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61462889">7. Conclusion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61462896">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61462896">Acknowledgement</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="61462902">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#61462902">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/1/1/2669389">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/1/25/2449750">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">A genomic view of food-related and probiotic Enterococcus strains
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Julieta Bonacina</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Laboratorio de Genética y Biología Molecular, CERELA-CONICET, Centro de Referencia para Lactobacilos, San Miguel de Tucumán (T4000ILC), Tucumán, Argentina</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Julieta+Bonacina">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bonacina J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Bonacina J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Julieta Bonacina</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nadia Su&#225;rez</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Laboratorio de Genética y Biología Molecular, CERELA-CONICET, Centro de Referencia para Lactobacilos, San Miguel de Tucumán (T4000ILC), Tucumán, Argentina</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nadia+Su%c3%a1rez">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Su&#225;rez N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Su&#225;rez N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Nadia Suárez</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ricardo Hormigo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Laboratorio de Genética y Biología Molecular, CERELA-CONICET, Centro de Referencia para Lactobacilos, San Miguel de Tucumán (T4000ILC), Tucumán, Argentina</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ricardo+Hormigo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hormigo R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hormigo R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ricardo Hormigo</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Silvina Fadda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Laboratorio de Genética y Biología Molecular, CERELA-CONICET, Centro de Referencia para Lactobacilos, San Miguel de Tucumán (T4000ILC), Tucumán, Argentina</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Silvina+Fadda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fadda S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fadda S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Silvina Fadda</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Marcus Lechner</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Department of Pharmaceutical Chemistry, Philipps-University Marburg, 35037 Marburg, Germany</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw043-cor1" data-open="dsw043-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +54 381 4310465 ext. 169. Fax: +54 381 4310465. ext 124. Email: <a href="mailto:lucila@cerela.org.ar" target="_blank">lucila@cerela.org.ar</a> (L.S.);Tel. +49 6421 2825925. Fax: +49 6421 2825854. Email: <a href="mailto:lechner@staff.uni-marburg.de" target="_blank">lechner@staff.uni-marburg.de</a> (M.L.)  Edited by Dr. Naotake Ogasawara
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Marcus+Lechner">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lechner M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lechner M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Marcus Lechner</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Lucila Saavedra</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Laboratorio de Genética y Biología Molecular, CERELA-CONICET, Centro de Referencia para Lactobacilos, San Miguel de Tucumán (T4000ILC), Tucumán, Argentina</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw043-cor1" data-open="dsw043-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +54 381 4310465 ext. 169. Fax: +54 381 4310465. ext 124. Email: <a href="mailto:lucila@cerela.org.ar" target="_blank">lucila@cerela.org.ar</a> (L.S.);Tel. +49 6421 2825925. Fax: +49 6421 2825854. Email: <a href="mailto:lechner@staff.uni-marburg.de" target="_blank">lechner@staff.uni-marburg.de</a> (M.L.)  Edited by Dr. Naotake Ogasawara
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Lucila+Saavedra">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Saavedra L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Saavedra L&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Lucila Saavedra</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 1, 1 February 2017, Pages 11–24, <a href='https://doi.org/10.1093/dnares/dsw043'>https://doi.org/10.1093/dnares/dsw043</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">23 October 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">27 January 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">18 August 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2670108" href="/dnaresearch/article-pdf/24/1/11/10731780/dsw043.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Julieta Bonacina, Nadia Suárez, Ricardo Hormigo, Silvina Fadda, Marcus Lechner, Lucila Saavedra;  A genomic view of food-related and probiotic Enterococcus strains, <em>DNA Research</em>, Volume 24, Issue 1, 1 February 2017, Pages 11–24, <a href='https://doi.org/10.1093/dnares/dsw043'>https://doi.org/10.1093/dnares/dsw043</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2670108?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2670108?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2670108?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2670108?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2670108?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2670108?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2670108?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=A%20genomic%20view%20of%20food-related%20and%20probiotic%20Enterococcus%20strains&amp;publicationDate=2016-10-23&amp;volumeNum=24&amp;issueNum=1&amp;author=Bonacina%2C%20Julieta%3B%20Su%C3%A1rez%2C%20Nadia&amp;startPage=11&amp;endPage=24&amp;contentId=10.1093%2Fdnares%2Fdsw043&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f1%2f11%2f2670108" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=61462822 id="61462822" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The study of enterococcal genomes has grown considerably in recent years. While special attention is paid to comparative genomic analysis among clinical relevant isolates, in this study we performed an exhaustive comparative analysis of enterococcal genomes of food origin and/or with potential to be used as probiotics. Beyond common genetic features, we especially aimed to identify those that are specific to enterococcal strains isolated from a certain food-related source as well as features present in a species-specific manner. Thus, the genome sequences of 25 <em>Enterococcus</em> strains, from 7 different species, were examined and compared. Their phylogenetic relationship was reconstructed based on orthologous proteins and whole genomes. Likewise, markers associated with a successful colonization (bacteriocin genes and genomic islands) and genome plasticity (phages and clustered regularly interspaced short palindromic repeats) were investigated for lifestyle specific genetic features. At the same time, a search for antibiotic resistance genes was carried out, since they are of big concern in the food industry. Finally, it was possible to locate 1617 FIGfam families as a core proteome universally present among the genera and to determine that most of the accessory genes code for hypothetical proteins, providing reasonable hints to support their functional characterization.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="Enterococcus"><em>Enterococcus</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="food">food</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="genomics">genomics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="niche">niche</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="probiotics">probiotics</a></div>                            <h2 scrollto-destination=61462824 id="61462824" class="section-title" data-legacy-id=SEC1>1. Introduction</h2>
<p>The genus <em>Enterococcus</em> (<em>E.</em>) belongs to the lactic acid bacteria (LAB) group and includes nearly 50 species characterized for being facultative anaerobic, non-spore forming and catalase-negative.<a reveal-id="dsw043-B1" data-open="dsw043-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsw043-B2" data-open="dsw043-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> They can be associated with humans, animals, plants, water and fermented foods or used as probiotics. Their application in the production of artisanal fermented products such as cheeses, meats and olives, inter alia is founded on their contribution in the typical taste and flavor of the final products as well as on their hygienic quality.<a reveal-id="dsw043-B1" data-open="dsw043-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsw043-B3" data-open="dsw043-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> The genus has also the potential to promote health by stimulating the immune system, assisting digestion and maintaining normal intestinal microflora and is thus frequently used in probiotic preparations.<a reveal-id="dsw043-B4" data-open="dsw043-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsw043-B5" data-open="dsw043-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> Despite the aforementioned, they are also known for their potential to cause disease being the antimicrobial resistance transferability and virulence phenomena of paramount importance.<a reveal-id="dsw043-B6" data-open="dsw043-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a></p><p>The advent of next-generation technologies has allowed the number of sequenced bacterial genomes to multiply and consequently, has given the possibility of carrying out numerous comparative studies seeking to establish patterns of behavior and evolutionary relationships.<a reveal-id="dsw043-B7" data-open="dsw043-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> Tettelin <em>et al</em><em>.</em><a reveal-id="dsw043-B8" data-open="dsw043-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> introduced the concept of <em>pan-genome</em>. This term refers to the global gene repertoire of a group of organisms consisting of a core genome (genes present in all members of the group) and a dispensable genome (unique and accessory genes present in one or more than one but not all organisms, respectively).<a reveal-id="dsw043-B8" data-open="dsw043-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> Numerous studies evidenced that the gene content of a genome and it structure can be affected during organism’s adaptation to a specific niche<a reveal-id="dsw043-B9" data-open="dsw043-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> and are the consequence, between others, of processes like horizontal gene transfer (HGT), genome rearrangement and gene loss. For the bacterial family <em>Vibrionaceae</em>, Kahlke <em>et al</em><em>.</em> showed that unique core genes appear more often in groups of isolates with a common ancestor (monophyletic) and, in the particular case of <em>Vibrio cholerae</em>, play a fundamental role in its adaptation to the ecological niche. Likewise, they indicated that in genophyletic groups of isolates (with no closest common ancestor) core genes are mainly the result of HGT.<a reveal-id="dsw043-B9" data-open="dsw043-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> In <em>Pseudomonas putida</em>, meanwhile, the presence of genomic islands carrying genes related to it specific lifestyle pointed out HGT as the driving force of it adaptation to a certain niche.<a reveal-id="dsw043-B10" data-open="dsw043-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> In relation to LAB, Douillard and de Vos described that the enormous diversity in the members of this group, as evidenced by comparative genomics studies, is the result of genome-environment interactions. The availability of nutrients in a specific habitat have a direct impact on metabolic properties of LAB. Many species present an enrichment in genes encoding for transporters (ABC or PTS systems) to compensate the loss of genes of biosynthetic pathways.<a reveal-id="dsw043-B11" data-open="dsw043-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a></p><p>The study of enterococcal genomes has grown considerably in recent years with special attention in comparative genomic analysis among clinical relevant isolates. In this sense, several studies have focused on the comparison of the safety aspects of clinical and non-clinical <em>E. faecium</em> and <em>E. faecalis</em> strains<a reveal-id="dsw043-B12" data-open="dsw043-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> but, to our knowledge, <em>Enterococcus</em> species of food origin have not been investigated in detail so far, what is surprising given the abundance of these microorganisms in certain fermented food.</p><p>Through an exhaustive comparative genomic analysis the goal of our manuscript was to identify common genetic features, features that are specific to enterococcal strains isolated from a certain food related source as well as those present in a species-specific manner. Furthermore, since several studies have shown that foodborne enterococci have a great potential as probiotics,<a reveal-id="dsw043-B13" data-open="dsw043-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsw043-B14" data-open="dsw043-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><em>Enterococcus</em> strains with probiotic properties were also incorporated to the analysis in spite of having or not food as natural niche. In this way, we sought to evaluate, at the genomic level, which were the shared traits between probiotics and food related strains. For this, the genome sequences of 25 <em>Enterococcus</em> strains, from 7 different species, who were the only one with full information about the isolation source by the time of data collection, were selected. Their phylogenetic relationship was determined at nucleotide and amino acid level comparing whole genome alignments and orthologous proteins, respectively. In addition, a search for markers associated with a successful colonization (bacteriocin genes and genomic islands) and genome plasticity (phages and clustered regularly interspaced short palindromic repeats) was performed. Since antibiotic resistance is of big concern in the food industry, the presence of antibiotic resistance genes was also investigated.</p>                            <h2 scrollto-destination=61462829 id="61462829" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=61462830 id="61462830" class="section-title" >2.1. Genome sequences and annotation</h3>
<p>According to their isolation source and probiotic potential 25 enterococcal genomic sequences, belonging to seven different species, were retrieved from the NCBI database from December 2014 to July 2015; these included: <em>E. faecalis</em>, <em>E. faecium</em>, <em>E. mundtii</em>, <em>E. hirae</em>, <em>E. malodoratus</em>, <em>E. raffinosus</em> and <em>E. durans</em>. In a next step, the strains were assigned to one of three major groups: dairy isolates, meat isolates and probiotics; and a fourth group was created: probiotics with dairy origin, for those strains with both properties. The dataset thus comprises 3 complete and 22 draft genomes.</p><p>In order rely on a directly comparable annotation of genes, we performed a functional annotation combined with a metabolic reconstruction using the RAST (Rapid Annotation Subsystem Technology) webservice.<a reveal-id="dsw043-B15" data-open="dsw043-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> For running the analyses the following settings were made: RAST for gene calling, FIGfam Release 70 as the collection of protein families, backfilling of gaps and automatic fixing of errors. Besides rRNAs and tRNAs, RAST identifies protein coding genes and allowed us to achieve a high quality assessment of the gene functions and their genomic context.</p>                            <h3 scrollto-destination=61462833 id="61462833" class="section-title" >2.2. Phylogenetic analysis</h3>
                            <h4 scrollto-destination=61462834 id="61462834" class="section-title" >2.1.1. Whole genome-based phylogenetic relationships</h4>
<p>Whole genomes of all enterococcal species and strains including <em>Lactococcus lactis</em> subsp. <em>cremoris</em> MG1363, <em>Lactococcus garvieae</em> Lg2, <em>Listeria monocytogenes</em> HCC23 and <em>Lactobacillus johnsonii</em> NCC 533 as outgroups were aligned using progressive Mauve.<a reveal-id="dsw043-B16" data-open="dsw043-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> All reported alignment blocks present in at least two genomes were concatenated. As using all species and strains at once partially lead to weak bootstrap scores (&lt;10%, probably due to much missing data for incomplete genomes, data not shown), the tree was constructed iteratively in two steps. At species level, we substituted species with multiple strains by a single representative strain that had a fully sequenced genome, namely <em>E. faecalis</em> str. Symbioflor 1 for <em>E. faecalis</em>, <em>E. faecium</em> NRRLB-2354 for <em>E. faecium</em> and <em>E. mundtii</em> ATCC882 for <em>E. mundtii</em>. The resulting 8,602,893 nt long multiple alignment was used in a rapid bootstrap analysis with RAxML v8.1.20<a reveal-id="dsw043-B17" data-open="dsw043-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> with 1000 replicates to search for best-scoring maximum likelihood tree with respect to the GTR substitution model and the Gamma model of rate heterogeneity.<a reveal-id="dsw043-B18" data-open="dsw043-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> We performed the same analysis at strain level separately for each species resulting in multiple alignments of lengths 3,336,408 nt for <em>E. faecalis</em>, 4,304,502 nt for <em>E. faecium</em> and 5,683,060 nt for <em>E. mundtii</em>. In the latter alignment, <em>E. malodoratus</em> had to be added as phylogenetic trees cannot be reconstructed from three instances (strains) only. It was removed from the resulting tree after reconstruction.</p>                            <h4 scrollto-destination=61462836 id="61462836" class="section-title" >2.1.2. Orthologous proteins-based phylogenetic relationships</h4>
<p>We used Proteinortho v5.1.12<a reveal-id="dsw043-B19" data-open="dsw043-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> to determine universally conserved 1:1-orthologs in all species and strains (including the four outgroup species as above) with respect to the proteins annotated through RAST (<em>E</em>-value: 1e−10, algebraic connectivity:0). Groups containing any paralogs were removed. Hence, the resulting 455 orthologous groups contained highly similar proteins present once in all species of interest. Amino acid sequences of orthologous proteins were aligned using ClustalOmega v1.2.1.<a reveal-id="dsw043-B20" data-open="dsw043-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> The resulting alignment was cropped at both ends till leading and trailing gaps were removed. All alignments were concatenated to single sequences representing one species/strain each resulting in a multiple sequence alignment of 134,771 aa. Based on these, the phylogenetic tree was constructed using RAxML v8.1.20<a reveal-id="dsw043-B17" data-open="dsw043-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a>. A rapid bootstrap analysis with 1000 replicates to search for best-scoring maximum likelihood tree with respect to the LG matrix and the Gamma model of rate heterogeneity<a reveal-id="dsw043-B18" data-open="dsw043-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> was performed. To make sure we did not observe effects due to over-representation of some species (<em>E. faecium</em> is present with 13 strains in the dataset comprising 29 genomes) the phylogenetic reconstruction was repeated using only one representative strain for species with multiple strains as described above. Their tree topologies were equivalent (data not shown) indicating no such effect.</p>                            <h2 scrollto-destination=61462838 id="61462838" class="section-title" >3. Niche-specific proteins and core proteome</h2>
<p>We directly used the FIGfam annotation classes provided by RAST to group predicted proteins according to their function. We want to note that these classes usually encapsulate multiple instances of different proteins (e.g. paralogs) and thus provide a rough approximation of function rather than reflecting fine-grained homology relations. For each enterococci group (dairy, meat, probiotics) and combination of groups (e.g. dairy and meat and so on), we set a filter such that only those classes remain that are present in at least a given percentage of the respective group but in no species/strain of any other group. Thereby, the two strains that are probiotics but were extracted from a dairy origin, <em>E. faecium</em> L-3 and <em>E. faecalis</em> MB5259, were dynamically assigned to either group (e.g. to dairy when the dairy group was checked). In order to avoid species specific results, we removed all genes solely found in strains of a single species.</p><p>Given the amount of incomplete genomes (missing data) and diversity of seven different species it was not surprising that no niche specific genes could be located when applying a presence threshold of 100–60%. At 50% a number of specific gene classes were located but not for dairy and the meat/probiotic combination. We decreased it further to 40% in order to recover specificities of all groups under this study. As statistical evaluation we repeated the analysis 1000 times with random group labels (fixed group sizes) and counted how often a gene family would be reported specific for any group. The results were used as expectation values (<em>E</em>-values), e.g. FIG00632402 was reported 26 times in random 1000 runs and therefore is represented with an <em>E</em>-value of 0.026.</p>                            <h2 scrollto-destination=61462841 id="61462841" class="section-title" >4. Genomic features</h2>
                            <h3 scrollto-destination=61462842 id="61462842" class="section-title" >4.1. Bacteriocin and antibiotic resistance genes</h3>
<p>Bacteriocin-coding genes were searched using BAGEL 3<a reveal-id="dsw043-B21" data-open="dsw043-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a>. Mining for modified and non-modified bacteriocins was carried out following two different approaches, one based on the analysis of the gene context, and the other on the identification of the gene itself. Since the gene coding for Enterocin A, <em>entA</em>, could not be located by this program, it was manually assessed by BLAST<a reveal-id="dsw043-B22" data-open="dsw043-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> using the sequence firstly described by Aymerich <em>et al</em><em>.</em><a reveal-id="dsw043-B23" data-open="dsw043-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> (GI: 1296521) as a query and the enterococcal genomes as the database (<em>E</em>-value ≤ 8e−100).</p><p>Furthermore, a search for antibiotic resistance genes was performed through the ARG-ANNOT tool.<a reveal-id="dsw043-B24" data-open="dsw043-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> For this a BLAST<a reveal-id="dsw043-B22" data-open="dsw043-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> in Bioedit<a reveal-id="dsw043-B25" data-open="dsw043-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> against the ARG-ANNOT database (April 2015, <a class="link link-uri" href="http://en.mediterranee-infection.com/article.php?laref=23%26titre=arg-annot" target="_blank">http://en.mediterranee-infection.com/article.php?laref=23%26titre=arg-annot</a>) was carried out. We used moderately stringent conditions (<em>E</em>-Value  ≤  1e−5), the Blosum 62 substitution matrix<a reveal-id="dsw043-B26" data-open="dsw043-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> and allowed for gapped blast hits. To reduce redundancy, only those results with ≥ 70% of query coverage and ≥ 90% sequence identity were taken into account. The antibiotics classes included in the database are: aminoglycosides, beta-lactamases, fosfomycin, fluoroquinolones, glycopeptides, macrolide-lincosamid-estreptogramin, phenicols, rifampicin, sulfonamides, tetracyclines and trimethoprim.</p>                            <h3 scrollto-destination=61462845 id="61462845" class="section-title" >4.2. Prophage sequences and CRISPRs—Cas systems</h3>
<p>Intact and incomplete prophage regions were identified through the integrated search and annotation tool PHAST.<a reveal-id="dsw043-B27" data-open="dsw043-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> This involved an ORF prediction and translation with GLIMMER 3.02<a reveal-id="dsw043-B28" data-open="dsw043-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a>, a BLAST-based annotation (using a non-redundant bacterial protein library), a tRNA and tmRNA sites localization for attachment sites recognition (tRNAscan-SE<a reveal-id="dsw043-B29" data-open="dsw043-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> and ARAGORN<a reveal-id="dsw043-B30" data-open="dsw043-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a>), a phage sequence identification (BLAST against a custom phage/prophage sequence database available at <a class="link link-uri" href="http://phast.wishartlab.com/Download.html" target="_blank">http://phast.wishartlab.com/Download.html</a>) and a density calculation of gene clusters via DBSCAN.<a reveal-id="dsw043-B27" data-open="dsw043-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> Only intact regions were analyzed in depth.</p><p>Clustered regularly interspaced short palindromic repeats (CRISPRs) were predicted with the CRISPRFinder web service.<a reveal-id="dsw043-B31" data-open="dsw043-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> In this way, the highly conserved regions known as direct repeats (DR) (23–55 bp long) and spacers (0.6–2.5 the repeated length) were localized. Putative CRISPR cassettes having at least three motifs (DR + spacer) and a minimum of two identical DRs were considered as confirmed CRISPRs, while the remaining candidates were named as questionable. CRISPRFinder databases are available at <a class="link link-uri" href="http://crispr.u-psud.fr/crispr/" target="_blank">http://crispr.u-psud.fr/crispr/</a>(Database status: Last update 2014-08-05).</p>                            <h3 scrollto-destination=61462848 id="61462848" class="section-title" >4.3. Genomic islands</h3>
<p>IslandViewer 3<a reveal-id="dsw043-B32" data-open="dsw043-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> was used for the identification of genes of probable horizontal origin. For localizing genomic islands (GEIs) this tool integrates three different methods: IslandPath-DIMOB,<a reveal-id="dsw043-B33" data-open="dsw043-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> SIGI-HMM<a reveal-id="dsw043-B34" data-open="dsw043-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> and IslandPick.<a reveal-id="dsw043-B35" data-open="dsw043-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> The first two predict GEIs considering the sequence composition, while the latter uses a comparative genomics approach.<a reveal-id="dsw043-B36" data-open="dsw043-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a><sup>,</sup><a reveal-id="dsw043-B37" data-open="dsw043-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> For further analysis, the islands predicted for at least one method that were completely included in a contig (in the case of draft genomes), were taken into account. Next, in order to find which GEIs are the same in different organisms we run Proteinortho (all vs all blastp of protein sequences encoded on each GEI) with default options but omitting the final clustering step (algebraic connectivity: 0, <em>E</em>-value ≤ 1e−10). Based on the resulting graph, we extracted the pairwise amount of shared proteins for the respective GEIs and plotted them in a heatmap using the gplots R-package.<a reveal-id="dsw043-B31" data-open="dsw043-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> Thereby predicted paralogs could be assigned to a single gene in another GEI (e.g. if one GEI has two paralogs that are orthologous to a gene in another GEI, then both have a partner in this GEI resulting in a similarity of 2/2 = 100% in one direction and 1/1 = 100% vice versa). All islands that did not share at least 50% of their proteins with any other island were removed as were pairs of GEIs with less than four proteins. For sake of simplicity, the names of species were cropped to the end of the strain name followed by the number of the strain-specific genomic island. Thus 19116-i1 means <em>E. faecalis</em> 19116 genomic island 1. All groups of homologous GEIs present in at least four strains were investigated in more detail.</p>                            <h2 scrollto-destination=61462850 id="61462850" class="section-title" >5. Results and discussion</h2>
                            <h3 scrollto-destination=61462851 id="61462851" class="section-title" >5.1. Overview of the enterococcal genomes</h3>
<p>We investigated 25 <em>Enterococcus</em> strains, from 7 different species, that had a food related origin and/or probiotic properties. Among them, a total of 13 <em>E. faecium</em>, 5 <em>E. faecalis</em>, 3 <em>E. mundtii</em> and 1 <em>E. durans, E. hirae</em>, <em>E. malodoratus</em> as well as <em>E. raffinosus</em> strains were included. Based on available data they were next incorporated into four major groups: dairy isolates (11 strains isolated from cheese or milk environment), meat isolates (6 strains of meat origin), probiotics (7 strains known for their ability or potential to be used as probiotics) and dairy isolates and probiotics (2 strains which were probiotics and had a dairy origin as well). In order to avoid erroneous conclusions, the members of the latter group were dynamically assigned to either group dairy isolates or probiotics when comparing.</p><p>General genomic features of all the strains analyzed in this study are provided in <a reveal-id="dsw043-T1" data-open="dsw043-T1" class="link link-reveal link-table xref-fig">Table 1</a>. No significant differences (<em>P</em> ≤ 0.05, Kruskal–Wallis statistical test) were observed between the groups in terms of the average number of genes and proteins (RAST annotation as stated above), genome size and GC content. Genome sizes ranged from approximately 2.5–4.6 Mb and GC contents from 36.8 to 39.9%. In agreement with our results, genomic similarities were previously described among <em>E. faecalis</em> strains irrespective of geographical, host temporal or clinical/non-clinical origin of the isolates.<a reveal-id="dsw043-B38" data-open="dsw043-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> Singularly, the number of protein encoding genes and ncRNAs detected in <em>E. malodoratus</em> ATCC 43197 (dairy origin) and <em>E. raffinosus</em> CFTRI 2200 (probiotic) draft genomes were the highest, and this was in accordance with their genome sizes that were about 1.6 Mb bigger than the average size of the genomes of the rest of the organisms under study. Interestingly, the strains with the highest GC% in each group were those with a higher number of coding DNA sequence (CDS) in the respective group (<em>E. faecium</em> E1613 for meat isolates). <div content-id="dsw043-T1" class="table-modal"><div class="table-wrap"><div id="dsw043-T1" data-id="dsw043-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Relevant genomic features and isolation source of the enterococcal species under study</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Origin/feature </th><th>Organism </th><th>Genbank<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> accession Nº </th><th>Status </th><th>Size<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> (Mbp) </th><th>GC% </th><th>Genes (NCBI) </th><th>Proteins (NCBI) </th><th>CDS (RAST) </th><th>Source </th></tr></thead><tbody><tr><td rowspan="12">Dairy isolates </td><td><em>Enterococcus durans</em> IPLA 655 </td><td>AOSM00000000.1 </td><td>Contig </td><td char=".">3.1 </td><td char=".">37.7 </td><td char=".">2941 </td><td char=".">2789 </td><td char=".">2978 </td><td>Cheese </td></tr><tr><td><em>E. hirae</em> INF E1 </td><td>JMIG00000000.1 </td><td>Scaffold </td><td char=".">2.8 </td><td char=".">36.8 </td><td char=".">2501 </td><td char=".">2404 </td><td char=".">2537 </td><td>Cultured milk </td></tr><tr><td><em>E. faecium</em> CRL1879 </td><td>AOUK00000000.1 </td><td>Contig </td><td char=".">2.9 </td><td char=".">37.7 </td><td char=".">2931 </td><td char=".">2616 </td><td char=".">3118 </td><td>Artisanal cheese </td></tr><tr><td><em>E. faecium</em> E1604 </td><td>AHXD00000000.1 </td><td>Scaffold </td><td char=".">2.9 </td><td char=".">38.3 </td><td char=".">2823 </td><td char=".">2749 </td><td char=".">2826 </td><td>Cheese </td></tr><tr><td rowspan="2"><em>E. faecium</em> NRRL B-2354 </td><td>CP004063.1 (chromosome) </td><td rowspan="2" char=".">Complete </td><td rowspan="2" char=".">2.8 </td><td rowspan="2" char=".">37.85 </td><td rowspan="2" char=".">2771 </td><td rowspan="2" char=".">2669 </td><td rowspan="2" char=".">2837 </td><td rowspan="2">Milk and dairy utensils </td></tr><tr><td>CP004064.1 (plasmid) </td></tr><tr><td><em>E. faecium</em> UC7256 </td><td>AWWM00000000.1 </td><td>Contig </td><td char=".">2.8 </td><td char=".">38.1 </td><td char=".">2699 </td><td char=".">2507 </td><td char=".">2878 </td><td>Pannerone cheese </td></tr><tr><td><em>E. faecium</em> UC7267 </td><td>ASAM00000000.1 </td><td>Contig </td><td char=".">2.7 </td><td char=".">37.9 </td><td char=".">2674 </td><td char=".">2549 </td><td char=".">2702 </td><td>Fontina cheese </td></tr><tr><td><em>E. faecium</em> UC8668 </td><td>AWWN00000000.1 </td><td>Contig </td><td char=".">2.9 </td><td char=".">38.1 </td><td char=".">2814 </td><td char=".">2667 </td><td char=".">2899 </td><td>Cheese </td></tr><tr><td><em>E. malodoratus</em> ATCC 43197 </td><td>ASWA00000000.1 </td><td>Scaffold </td><td char=".">4.6 </td><td char=".">39.9 </td><td char=".">4581 </td><td char=".">4499 </td><td char=".">4524 </td><td>Gouda cheese </td></tr><tr><td><em>E. mundtii</em> ATCC 882 </td><td>ASWC00000000.1 </td><td>Scaffold </td><td char=".">3.1 </td><td char=".">38.4 </td><td char=".">2986 </td><td char=".">2902 </td><td char=".">2915 </td><td>Dairy products </td></tr><tr><td><em>E. mundtii</em> CRL1656 </td><td>AFWZ00000000.1 </td><td>Contig </td><td char=".">3.1 </td><td char=".">38.4 </td><td char=".">2785 </td><td char=".">2651 </td><td char=".">2765 </td><td>Stripping milk of an argentinean cow </td></tr><tr><td>  </td><td><em>E. mundtii</em> CRL35 </td><td>JDFT00000000.1 </td><td>Contig </td><td char=".">2.9 </td><td char=".">38.3 </td><td char=".">2851 </td><td char=".">2757 </td><td char=".">2759 </td><td>Artisanal cheese </td></tr><tr><td rowspan="4">Meat isolates </td><td><em>E. faecalis</em> 2924 </td><td>AIIV00000000.1 </td><td>Scaffold </td><td char=".">3.0 </td><td char=".">37.5 </td><td char=".">2835 </td><td char=".">2788 </td><td char=".">2819 </td><td>Turkey meat </td></tr><tr><td><em>E. faecalis</em> 19116 </td><td>AIIT00000000.1 </td><td>Scaffold </td><td char=".">3.0 </td><td char=".">37.3 </td><td char=".">2852 </td><td char=".">2797 </td><td char=".">2816 </td><td>Pork meat </td></tr><tr><td><em>E. faecium</em> E1613 </td><td>AHXE00000000.1 </td><td>Scaffold </td><td char=".">2.9 </td><td char=".">38.2 </td><td char=".">2881 </td><td char=".">2801 </td><td char=".">2849 </td><td>Fish burguer </td></tr><tr><td><em>E. faecium</em> UC7251 </td><td>ASAL00000000.1 </td><td>Contig </td><td char=".">2.7 </td><td char=".">37.9 </td><td char=".">2598 </td><td char=".">2480 </td><td char=".">2680 </td><td>Fermented sausage </td></tr><tr><td rowspan="2"> </td><td><em>E. faecium</em> UC8733 </td><td>AWWO00000000.1 </td><td>Contig </td><td char=".">2.7 </td><td char=".">37.9 </td><td char=".">2628 </td><td char=".">2491 </td><td char=".">2748 </td><td>Fermented sausage </td></tr><tr><td><em>E. faecium</em> UC10237 </td><td>AWWP00000000.1 </td><td>Contig </td><td char=".">2.6 </td><td char=".">37.9 </td><td char=".">2477 </td><td char=".">2390 </td><td char=".">2525 </td><td>Llama sausage </td></tr><tr><td rowspan="6">Probiotics </td><td><em>E. faecium</em> L-X </td><td>JRGY00000000.1 </td><td>Contig </td><td char=".">2.7 </td><td char=".">38.3 </td><td char=".">2641 </td><td char=".">2442 </td><td char=".">2728 </td><td>Probiotic preparation </td></tr><tr><td><em>E. faecalis</em> Symbioflor 1 </td><td>HF558530.1 </td><td>Complete </td><td char=".">2.8 </td><td char=".">37.7 </td><td char=".">2810 </td><td char=".">2705 </td><td char=".">2724 </td><td>Stool specimen of a healthy human adult </td></tr><tr><td><em>E. faecalis</em> PC1.1 </td><td>ADKN00000000.1 </td><td>Contig </td><td char=".">2.8 </td><td char=".">37.6 </td><td char=".">2661 </td><td char=".">2560 </td><td char=".">2612 </td><td>Pooled fecal sample collected from healthy human </td></tr><tr><td rowspan="2"><em>E. faecium</em> T-110 </td><td>CP006030.1 (chromosome) </td><td rowspan="2" char=".">Complete </td><td rowspan="2" char=".">2.7 </td><td rowspan="2" char=".">38.46 </td><td rowspan="2" char=".">2639 </td><td rowspan="2" char=".">2512 </td><td rowspan="2" char=".">2653 </td><td rowspan="2">Not specified </td></tr><tr><td>CP006031.1 (plasmid) </td></tr><tr><td><em>E. raffinosus</em> CFTRI 2200 </td><td>ATKJ00000000.2 </td><td>Contig </td><td char=".">4.2 </td><td char=".">39.4 </td><td char=".">4129 </td><td char=".">4016 </td><td char=".">4180 </td><td>Infant fecal material </td></tr><tr><td rowspan="2" char=".">Dairy isolates and probiotics </td><td><em>E. faecium</em> L-3 </td><td>JRGX00000000.1 </td><td>Scaffold </td><td char=".">2.6 </td><td char=".">38.0 </td><td char=".">2559 </td><td char=".">2367 </td><td char=".">2646 </td><td>Probiotic preparation </td></tr><tr><td><em>E. faecalis</em> MB5259 </td><td>JMEC00000000.1 </td><td>Contig </td><td char=".">3.1 </td><td char=".">37.5 </td><td char=".">3051 </td><td char=".">2734 </td><td char=".">3180 </td><td>Dairy origin </td></tr></tbody></table></div><div content-id="dsw043-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Origin/feature </th><th>Organism </th><th>Genbank<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> accession Nº </th><th>Status </th><th>Size<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> (Mbp) </th><th>GC% </th><th>Genes (NCBI) </th><th>Proteins (NCBI) </th><th>CDS (RAST) </th><th>Source </th></tr></thead><tbody><tr><td rowspan="12">Dairy isolates </td><td><em>Enterococcus durans</em> IPLA 655 </td><td>AOSM00000000.1 </td><td>Contig </td><td char=".">3.1 </td><td char=".">37.7 </td><td char=".">2941 </td><td char=".">2789 </td><td char=".">2978 </td><td>Cheese </td></tr><tr><td><em>E. hirae</em> INF E1 </td><td>JMIG00000000.1 </td><td>Scaffold </td><td char=".">2.8 </td><td char=".">36.8 </td><td char=".">2501 </td><td char=".">2404 </td><td char=".">2537 </td><td>Cultured milk </td></tr><tr><td><em>E. faecium</em> CRL1879 </td><td>AOUK00000000.1 </td><td>Contig </td><td char=".">2.9 </td><td char=".">37.7 </td><td char=".">2931 </td><td char=".">2616 </td><td char=".">3118 </td><td>Artisanal cheese </td></tr><tr><td><em>E. faecium</em> E1604 </td><td>AHXD00000000.1 </td><td>Scaffold </td><td char=".">2.9 </td><td char=".">38.3 </td><td char=".">2823 </td><td char=".">2749 </td><td char=".">2826 </td><td>Cheese </td></tr><tr><td rowspan="2"><em>E. faecium</em> NRRL B-2354 </td><td>CP004063.1 (chromosome) </td><td rowspan="2" char=".">Complete </td><td rowspan="2" char=".">2.8 </td><td rowspan="2" char=".">37.85 </td><td rowspan="2" char=".">2771 </td><td rowspan="2" char=".">2669 </td><td rowspan="2" char=".">2837 </td><td rowspan="2">Milk and dairy utensils </td></tr><tr><td>CP004064.1 (plasmid) </td></tr><tr><td><em>E. faecium</em> UC7256 </td><td>AWWM00000000.1 </td><td>Contig </td><td char=".">2.8 </td><td char=".">38.1 </td><td char=".">2699 </td><td char=".">2507 </td><td char=".">2878 </td><td>Pannerone cheese </td></tr><tr><td><em>E. faecium</em> UC7267 </td><td>ASAM00000000.1 </td><td>Contig </td><td char=".">2.7 </td><td char=".">37.9 </td><td char=".">2674 </td><td char=".">2549 </td><td char=".">2702 </td><td>Fontina cheese </td></tr><tr><td><em>E. faecium</em> UC8668 </td><td>AWWN00000000.1 </td><td>Contig </td><td char=".">2.9 </td><td char=".">38.1 </td><td char=".">2814 </td><td char=".">2667 </td><td char=".">2899 </td><td>Cheese </td></tr><tr><td><em>E. malodoratus</em> ATCC 43197 </td><td>ASWA00000000.1 </td><td>Scaffold </td><td char=".">4.6 </td><td char=".">39.9 </td><td char=".">4581 </td><td char=".">4499 </td><td char=".">4524 </td><td>Gouda cheese </td></tr><tr><td><em>E. mundtii</em> ATCC 882 </td><td>ASWC00000000.1 </td><td>Scaffold </td><td char=".">3.1 </td><td char=".">38.4 </td><td char=".">2986 </td><td char=".">2902 </td><td char=".">2915 </td><td>Dairy products </td></tr><tr><td><em>E. mundtii</em> CRL1656 </td><td>AFWZ00000000.1 </td><td>Contig </td><td char=".">3.1 </td><td char=".">38.4 </td><td char=".">2785 </td><td char=".">2651 </td><td char=".">2765 </td><td>Stripping milk of an argentinean cow </td></tr><tr><td>  </td><td><em>E. mundtii</em> CRL35 </td><td>JDFT00000000.1 </td><td>Contig </td><td char=".">2.9 </td><td char=".">38.3 </td><td char=".">2851 </td><td char=".">2757 </td><td char=".">2759 </td><td>Artisanal cheese </td></tr><tr><td rowspan="4">Meat isolates </td><td><em>E. faecalis</em> 2924 </td><td>AIIV00000000.1 </td><td>Scaffold </td><td char=".">3.0 </td><td char=".">37.5 </td><td char=".">2835 </td><td char=".">2788 </td><td char=".">2819 </td><td>Turkey meat </td></tr><tr><td><em>E. faecalis</em> 19116 </td><td>AIIT00000000.1 </td><td>Scaffold </td><td char=".">3.0 </td><td char=".">37.3 </td><td char=".">2852 </td><td char=".">2797 </td><td char=".">2816 </td><td>Pork meat </td></tr><tr><td><em>E. faecium</em> E1613 </td><td>AHXE00000000.1 </td><td>Scaffold </td><td char=".">2.9 </td><td char=".">38.2 </td><td char=".">2881 </td><td char=".">2801 </td><td char=".">2849 </td><td>Fish burguer </td></tr><tr><td><em>E. faecium</em> UC7251 </td><td>ASAL00000000.1 </td><td>Contig </td><td char=".">2.7 </td><td char=".">37.9 </td><td char=".">2598 </td><td char=".">2480 </td><td char=".">2680 </td><td>Fermented sausage </td></tr><tr><td rowspan="2"> </td><td><em>E. faecium</em> UC8733 </td><td>AWWO00000000.1 </td><td>Contig </td><td char=".">2.7 </td><td char=".">37.9 </td><td char=".">2628 </td><td char=".">2491 </td><td char=".">2748 </td><td>Fermented sausage </td></tr><tr><td><em>E. faecium</em> UC10237 </td><td>AWWP00000000.1 </td><td>Contig </td><td char=".">2.6 </td><td char=".">37.9 </td><td char=".">2477 </td><td char=".">2390 </td><td char=".">2525 </td><td>Llama sausage </td></tr><tr><td rowspan="6">Probiotics </td><td><em>E. faecium</em> L-X </td><td>JRGY00000000.1 </td><td>Contig </td><td char=".">2.7 </td><td char=".">38.3 </td><td char=".">2641 </td><td char=".">2442 </td><td char=".">2728 </td><td>Probiotic preparation </td></tr><tr><td><em>E. faecalis</em> Symbioflor 1 </td><td>HF558530.1 </td><td>Complete </td><td char=".">2.8 </td><td char=".">37.7 </td><td char=".">2810 </td><td char=".">2705 </td><td char=".">2724 </td><td>Stool specimen of a healthy human adult </td></tr><tr><td><em>E. faecalis</em> PC1.1 </td><td>ADKN00000000.1 </td><td>Contig </td><td char=".">2.8 </td><td char=".">37.6 </td><td char=".">2661 </td><td char=".">2560 </td><td char=".">2612 </td><td>Pooled fecal sample collected from healthy human </td></tr><tr><td rowspan="2"><em>E. faecium</em> T-110 </td><td>CP006030.1 (chromosome) </td><td rowspan="2" char=".">Complete </td><td rowspan="2" char=".">2.7 </td><td rowspan="2" char=".">38.46 </td><td rowspan="2" char=".">2639 </td><td rowspan="2" char=".">2512 </td><td rowspan="2" char=".">2653 </td><td rowspan="2">Not specified </td></tr><tr><td>CP006031.1 (plasmid) </td></tr><tr><td><em>E. raffinosus</em> CFTRI 2200 </td><td>ATKJ00000000.2 </td><td>Contig </td><td char=".">4.2 </td><td char=".">39.4 </td><td char=".">4129 </td><td char=".">4016 </td><td char=".">4180 </td><td>Infant fecal material </td></tr><tr><td rowspan="2" char=".">Dairy isolates and probiotics </td><td><em>E. faecium</em> L-3 </td><td>JRGX00000000.1 </td><td>Scaffold </td><td char=".">2.6 </td><td char=".">38.0 </td><td char=".">2559 </td><td char=".">2367 </td><td char=".">2646 </td><td>Probiotic preparation </td></tr><tr><td><em>E. faecalis</em> MB5259 </td><td>JMEC00000000.1 </td><td>Contig </td><td char=".">3.1 </td><td char=".">37.5 </td><td char=".">3051 </td><td char=".">2734 </td><td char=".">3180 </td><td>Dairy origin </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><div class="label -label">a</div><p>All Accession numbers are from genbank.</p></div></div><div content-id="tblfn2"><div class="fn"><div class="label -label">b</div><p>Genomes sizes were calculated as the total length of contigs for each genome.</p></div></div></div></div></div><div class="table-wrap"><div id="dsw043-T1" data-id="dsw043-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Relevant genomic features and isolation source of the enterococcal species under study</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Origin/feature </th><th>Organism </th><th>Genbank<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> accession Nº </th><th>Status </th><th>Size<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> (Mbp) </th><th>GC% </th><th>Genes (NCBI) </th><th>Proteins (NCBI) </th><th>CDS (RAST) </th><th>Source </th></tr></thead><tbody><tr><td rowspan="12">Dairy isolates </td><td><em>Enterococcus durans</em> IPLA 655 </td><td>AOSM00000000.1 </td><td>Contig </td><td char=".">3.1 </td><td char=".">37.7 </td><td char=".">2941 </td><td char=".">2789 </td><td char=".">2978 </td><td>Cheese </td></tr><tr><td><em>E. hirae</em> INF E1 </td><td>JMIG00000000.1 </td><td>Scaffold </td><td char=".">2.8 </td><td char=".">36.8 </td><td char=".">2501 </td><td char=".">2404 </td><td char=".">2537 </td><td>Cultured milk </td></tr><tr><td><em>E. faecium</em> CRL1879 </td><td>AOUK00000000.1 </td><td>Contig </td><td char=".">2.9 </td><td char=".">37.7 </td><td char=".">2931 </td><td char=".">2616 </td><td char=".">3118 </td><td>Artisanal cheese </td></tr><tr><td><em>E. faecium</em> E1604 </td><td>AHXD00000000.1 </td><td>Scaffold </td><td char=".">2.9 </td><td char=".">38.3 </td><td char=".">2823 </td><td char=".">2749 </td><td char=".">2826 </td><td>Cheese </td></tr><tr><td rowspan="2"><em>E. faecium</em> NRRL B-2354 </td><td>CP004063.1 (chromosome) </td><td rowspan="2" char=".">Complete </td><td rowspan="2" char=".">2.8 </td><td rowspan="2" char=".">37.85 </td><td rowspan="2" char=".">2771 </td><td rowspan="2" char=".">2669 </td><td rowspan="2" char=".">2837 </td><td rowspan="2">Milk and dairy utensils </td></tr><tr><td>CP004064.1 (plasmid) </td></tr><tr><td><em>E. faecium</em> UC7256 </td><td>AWWM00000000.1 </td><td>Contig </td><td char=".">2.8 </td><td char=".">38.1 </td><td char=".">2699 </td><td char=".">2507 </td><td char=".">2878 </td><td>Pannerone cheese </td></tr><tr><td><em>E. faecium</em> UC7267 </td><td>ASAM00000000.1 </td><td>Contig </td><td char=".">2.7 </td><td char=".">37.9 </td><td char=".">2674 </td><td char=".">2549 </td><td char=".">2702 </td><td>Fontina cheese </td></tr><tr><td><em>E. faecium</em> UC8668 </td><td>AWWN00000000.1 </td><td>Contig </td><td char=".">2.9 </td><td char=".">38.1 </td><td char=".">2814 </td><td char=".">2667 </td><td char=".">2899 </td><td>Cheese </td></tr><tr><td><em>E. malodoratus</em> ATCC 43197 </td><td>ASWA00000000.1 </td><td>Scaffold </td><td char=".">4.6 </td><td char=".">39.9 </td><td char=".">4581 </td><td char=".">4499 </td><td char=".">4524 </td><td>Gouda cheese </td></tr><tr><td><em>E. mundtii</em> ATCC 882 </td><td>ASWC00000000.1 </td><td>Scaffold </td><td char=".">3.1 </td><td char=".">38.4 </td><td char=".">2986 </td><td char=".">2902 </td><td char=".">2915 </td><td>Dairy products </td></tr><tr><td><em>E. mundtii</em> CRL1656 </td><td>AFWZ00000000.1 </td><td>Contig </td><td char=".">3.1 </td><td char=".">38.4 </td><td char=".">2785 </td><td char=".">2651 </td><td char=".">2765 </td><td>Stripping milk of an argentinean cow </td></tr><tr><td>  </td><td><em>E. mundtii</em> CRL35 </td><td>JDFT00000000.1 </td><td>Contig </td><td char=".">2.9 </td><td char=".">38.3 </td><td char=".">2851 </td><td char=".">2757 </td><td char=".">2759 </td><td>Artisanal cheese </td></tr><tr><td rowspan="4">Meat isolates </td><td><em>E. faecalis</em> 2924 </td><td>AIIV00000000.1 </td><td>Scaffold </td><td char=".">3.0 </td><td char=".">37.5 </td><td char=".">2835 </td><td char=".">2788 </td><td char=".">2819 </td><td>Turkey meat </td></tr><tr><td><em>E. faecalis</em> 19116 </td><td>AIIT00000000.1 </td><td>Scaffold </td><td char=".">3.0 </td><td char=".">37.3 </td><td char=".">2852 </td><td char=".">2797 </td><td char=".">2816 </td><td>Pork meat </td></tr><tr><td><em>E. faecium</em> E1613 </td><td>AHXE00000000.1 </td><td>Scaffold </td><td char=".">2.9 </td><td char=".">38.2 </td><td char=".">2881 </td><td char=".">2801 </td><td char=".">2849 </td><td>Fish burguer </td></tr><tr><td><em>E. faecium</em> UC7251 </td><td>ASAL00000000.1 </td><td>Contig </td><td char=".">2.7 </td><td char=".">37.9 </td><td char=".">2598 </td><td char=".">2480 </td><td char=".">2680 </td><td>Fermented sausage </td></tr><tr><td rowspan="2"> </td><td><em>E. faecium</em> UC8733 </td><td>AWWO00000000.1 </td><td>Contig </td><td char=".">2.7 </td><td char=".">37.9 </td><td char=".">2628 </td><td char=".">2491 </td><td char=".">2748 </td><td>Fermented sausage </td></tr><tr><td><em>E. faecium</em> UC10237 </td><td>AWWP00000000.1 </td><td>Contig </td><td char=".">2.6 </td><td char=".">37.9 </td><td char=".">2477 </td><td char=".">2390 </td><td char=".">2525 </td><td>Llama sausage </td></tr><tr><td rowspan="6">Probiotics </td><td><em>E. faecium</em> L-X </td><td>JRGY00000000.1 </td><td>Contig </td><td char=".">2.7 </td><td char=".">38.3 </td><td char=".">2641 </td><td char=".">2442 </td><td char=".">2728 </td><td>Probiotic preparation </td></tr><tr><td><em>E. faecalis</em> Symbioflor 1 </td><td>HF558530.1 </td><td>Complete </td><td char=".">2.8 </td><td char=".">37.7 </td><td char=".">2810 </td><td char=".">2705 </td><td char=".">2724 </td><td>Stool specimen of a healthy human adult </td></tr><tr><td><em>E. faecalis</em> PC1.1 </td><td>ADKN00000000.1 </td><td>Contig </td><td char=".">2.8 </td><td char=".">37.6 </td><td char=".">2661 </td><td char=".">2560 </td><td char=".">2612 </td><td>Pooled fecal sample collected from healthy human </td></tr><tr><td rowspan="2"><em>E. faecium</em> T-110 </td><td>CP006030.1 (chromosome) </td><td rowspan="2" char=".">Complete </td><td rowspan="2" char=".">2.7 </td><td rowspan="2" char=".">38.46 </td><td rowspan="2" char=".">2639 </td><td rowspan="2" char=".">2512 </td><td rowspan="2" char=".">2653 </td><td rowspan="2">Not specified </td></tr><tr><td>CP006031.1 (plasmid) </td></tr><tr><td><em>E. raffinosus</em> CFTRI 2200 </td><td>ATKJ00000000.2 </td><td>Contig </td><td char=".">4.2 </td><td char=".">39.4 </td><td char=".">4129 </td><td char=".">4016 </td><td char=".">4180 </td><td>Infant fecal material </td></tr><tr><td rowspan="2" char=".">Dairy isolates and probiotics </td><td><em>E. faecium</em> L-3 </td><td>JRGX00000000.1 </td><td>Scaffold </td><td char=".">2.6 </td><td char=".">38.0 </td><td char=".">2559 </td><td char=".">2367 </td><td char=".">2646 </td><td>Probiotic preparation </td></tr><tr><td><em>E. faecalis</em> MB5259 </td><td>JMEC00000000.1 </td><td>Contig </td><td char=".">3.1 </td><td char=".">37.5 </td><td char=".">3051 </td><td char=".">2734 </td><td char=".">3180 </td><td>Dairy origin </td></tr></tbody></table></div><div content-id="dsw043-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Origin/feature </th><th>Organism </th><th>Genbank<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> accession Nº </th><th>Status </th><th>Size<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> (Mbp) </th><th>GC% </th><th>Genes (NCBI) </th><th>Proteins (NCBI) </th><th>CDS (RAST) </th><th>Source </th></tr></thead><tbody><tr><td rowspan="12">Dairy isolates </td><td><em>Enterococcus durans</em> IPLA 655 </td><td>AOSM00000000.1 </td><td>Contig </td><td char=".">3.1 </td><td char=".">37.7 </td><td char=".">2941 </td><td char=".">2789 </td><td char=".">2978 </td><td>Cheese </td></tr><tr><td><em>E. hirae</em> INF E1 </td><td>JMIG00000000.1 </td><td>Scaffold </td><td char=".">2.8 </td><td char=".">36.8 </td><td char=".">2501 </td><td char=".">2404 </td><td char=".">2537 </td><td>Cultured milk </td></tr><tr><td><em>E. faecium</em> CRL1879 </td><td>AOUK00000000.1 </td><td>Contig </td><td char=".">2.9 </td><td char=".">37.7 </td><td char=".">2931 </td><td char=".">2616 </td><td char=".">3118 </td><td>Artisanal cheese </td></tr><tr><td><em>E. faecium</em> E1604 </td><td>AHXD00000000.1 </td><td>Scaffold </td><td char=".">2.9 </td><td char=".">38.3 </td><td char=".">2823 </td><td char=".">2749 </td><td char=".">2826 </td><td>Cheese </td></tr><tr><td rowspan="2"><em>E. faecium</em> NRRL B-2354 </td><td>CP004063.1 (chromosome) </td><td rowspan="2" char=".">Complete </td><td rowspan="2" char=".">2.8 </td><td rowspan="2" char=".">37.85 </td><td rowspan="2" char=".">2771 </td><td rowspan="2" char=".">2669 </td><td rowspan="2" char=".">2837 </td><td rowspan="2">Milk and dairy utensils </td></tr><tr><td>CP004064.1 (plasmid) </td></tr><tr><td><em>E. faecium</em> UC7256 </td><td>AWWM00000000.1 </td><td>Contig </td><td char=".">2.8 </td><td char=".">38.1 </td><td char=".">2699 </td><td char=".">2507 </td><td char=".">2878 </td><td>Pannerone cheese </td></tr><tr><td><em>E. faecium</em> UC7267 </td><td>ASAM00000000.1 </td><td>Contig </td><td char=".">2.7 </td><td char=".">37.9 </td><td char=".">2674 </td><td char=".">2549 </td><td char=".">2702 </td><td>Fontina cheese </td></tr><tr><td><em>E. faecium</em> UC8668 </td><td>AWWN00000000.1 </td><td>Contig </td><td char=".">2.9 </td><td char=".">38.1 </td><td char=".">2814 </td><td char=".">2667 </td><td char=".">2899 </td><td>Cheese </td></tr><tr><td><em>E. malodoratus</em> ATCC 43197 </td><td>ASWA00000000.1 </td><td>Scaffold </td><td char=".">4.6 </td><td char=".">39.9 </td><td char=".">4581 </td><td char=".">4499 </td><td char=".">4524 </td><td>Gouda cheese </td></tr><tr><td><em>E. mundtii</em> ATCC 882 </td><td>ASWC00000000.1 </td><td>Scaffold </td><td char=".">3.1 </td><td char=".">38.4 </td><td char=".">2986 </td><td char=".">2902 </td><td char=".">2915 </td><td>Dairy products </td></tr><tr><td><em>E. mundtii</em> CRL1656 </td><td>AFWZ00000000.1 </td><td>Contig </td><td char=".">3.1 </td><td char=".">38.4 </td><td char=".">2785 </td><td char=".">2651 </td><td char=".">2765 </td><td>Stripping milk of an argentinean cow </td></tr><tr><td>  </td><td><em>E. mundtii</em> CRL35 </td><td>JDFT00000000.1 </td><td>Contig </td><td char=".">2.9 </td><td char=".">38.3 </td><td char=".">2851 </td><td char=".">2757 </td><td char=".">2759 </td><td>Artisanal cheese </td></tr><tr><td rowspan="4">Meat isolates </td><td><em>E. faecalis</em> 2924 </td><td>AIIV00000000.1 </td><td>Scaffold </td><td char=".">3.0 </td><td char=".">37.5 </td><td char=".">2835 </td><td char=".">2788 </td><td char=".">2819 </td><td>Turkey meat </td></tr><tr><td><em>E. faecalis</em> 19116 </td><td>AIIT00000000.1 </td><td>Scaffold </td><td char=".">3.0 </td><td char=".">37.3 </td><td char=".">2852 </td><td char=".">2797 </td><td char=".">2816 </td><td>Pork meat </td></tr><tr><td><em>E. faecium</em> E1613 </td><td>AHXE00000000.1 </td><td>Scaffold </td><td char=".">2.9 </td><td char=".">38.2 </td><td char=".">2881 </td><td char=".">2801 </td><td char=".">2849 </td><td>Fish burguer </td></tr><tr><td><em>E. faecium</em> UC7251 </td><td>ASAL00000000.1 </td><td>Contig </td><td char=".">2.7 </td><td char=".">37.9 </td><td char=".">2598 </td><td char=".">2480 </td><td char=".">2680 </td><td>Fermented sausage </td></tr><tr><td rowspan="2"> </td><td><em>E. faecium</em> UC8733 </td><td>AWWO00000000.1 </td><td>Contig </td><td char=".">2.7 </td><td char=".">37.9 </td><td char=".">2628 </td><td char=".">2491 </td><td char=".">2748 </td><td>Fermented sausage </td></tr><tr><td><em>E. faecium</em> UC10237 </td><td>AWWP00000000.1 </td><td>Contig </td><td char=".">2.6 </td><td char=".">37.9 </td><td char=".">2477 </td><td char=".">2390 </td><td char=".">2525 </td><td>Llama sausage </td></tr><tr><td rowspan="6">Probiotics </td><td><em>E. faecium</em> L-X </td><td>JRGY00000000.1 </td><td>Contig </td><td char=".">2.7 </td><td char=".">38.3 </td><td char=".">2641 </td><td char=".">2442 </td><td char=".">2728 </td><td>Probiotic preparation </td></tr><tr><td><em>E. faecalis</em> Symbioflor 1 </td><td>HF558530.1 </td><td>Complete </td><td char=".">2.8 </td><td char=".">37.7 </td><td char=".">2810 </td><td char=".">2705 </td><td char=".">2724 </td><td>Stool specimen of a healthy human adult </td></tr><tr><td><em>E. faecalis</em> PC1.1 </td><td>ADKN00000000.1 </td><td>Contig </td><td char=".">2.8 </td><td char=".">37.6 </td><td char=".">2661 </td><td char=".">2560 </td><td char=".">2612 </td><td>Pooled fecal sample collected from healthy human </td></tr><tr><td rowspan="2"><em>E. faecium</em> T-110 </td><td>CP006030.1 (chromosome) </td><td rowspan="2" char=".">Complete </td><td rowspan="2" char=".">2.7 </td><td rowspan="2" char=".">38.46 </td><td rowspan="2" char=".">2639 </td><td rowspan="2" char=".">2512 </td><td rowspan="2" char=".">2653 </td><td rowspan="2">Not specified </td></tr><tr><td>CP006031.1 (plasmid) </td></tr><tr><td><em>E. raffinosus</em> CFTRI 2200 </td><td>ATKJ00000000.2 </td><td>Contig </td><td char=".">4.2 </td><td char=".">39.4 </td><td char=".">4129 </td><td char=".">4016 </td><td char=".">4180 </td><td>Infant fecal material </td></tr><tr><td rowspan="2" char=".">Dairy isolates and probiotics </td><td><em>E. faecium</em> L-3 </td><td>JRGX00000000.1 </td><td>Scaffold </td><td char=".">2.6 </td><td char=".">38.0 </td><td char=".">2559 </td><td char=".">2367 </td><td char=".">2646 </td><td>Probiotic preparation </td></tr><tr><td><em>E. faecalis</em> MB5259 </td><td>JMEC00000000.1 </td><td>Contig </td><td char=".">3.1 </td><td char=".">37.5 </td><td char=".">3051 </td><td char=".">2734 </td><td char=".">3180 </td><td>Dairy origin </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><div class="label -label">a</div><p>All Accession numbers are from genbank.</p></div></div><div content-id="tblfn2"><div class="fn"><div class="label -label">b</div><p>Genomes sizes were calculated as the total length of contigs for each genome.</p></div></div></div></div></p>                            <h3 scrollto-destination=61462854 id="61462854" class="section-title" >5.2. Phylogeny</h3>
<div class="block-child-p">The phylogenetic relationships of all strains were determined using two independent approaches. First, a genome based phylogenetic tree was built obtaining two main groups at species level (<a reveal-id="dsw043-F1" data-open="dsw043-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>). One of them, the most ancient, included the <em>E. malodorous</em> and <em>E. raffinosus</em> species represented by the strains <em>E. malodoratus</em> ATCC 43197 and <em>E. raffinosus</em> CFTRI 2200, respectively; and the other was comprised by the rest of the species under study: <em>E. faecalis</em>, <em>E. mundtii</em>, <em>E. durans</em>, <em>E. hirae</em> and <em>E. faecium</em>. In the latter group, <em>E. faecalis</em> formed a separate branch at the very beginning, and later in evolution, the same was observed first with <em>E. mundtti</em> and then with <em>E. faecium</em>. For their part, <em>E. durans</em> IPLA 655 and <em>E. hirae</em> INF E1 clustered together having with <em>E. faecium</em> the last common ancestor. This topology is in line with that of the 16S rRNA tree presented in Bergey’s manual.<a reveal-id="dsw043-B39" data-open="dsw043-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> In the same way, the relationships observed when comparing a 1,102-bp fragment of the gene encoding the alpha subunit of ATP synthase <em>atpA</em>, from 91 <em>Enterococcus</em> strains, were in agreement with the phylogeny inferred from whole genomes in our study.<a reveal-id="dsw043-B40" data-open="dsw043-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> According to Naser <em>et al</em><em>.</em><a reveal-id="dsw043-B40" data-open="dsw043-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> this gene has an optimal discriminatory power that enables enterococcal species differentiation. <div data-id="dsw043-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/m_dsw043f1.png?Expires=2147483647&Signature=e9RSuo58alYkgMdegBBNhE85qZst-huq4CMCjgNGCFjMJf~X3FrDfX7ky-WkuYsMyr-MhiYXMKbcUAS0lje6SIOaLI2K~iDSGszaw9dH7C1wtsqjQRy6l0RIO6M~yHAk9SVDtX8L8c~NgyM-i9r3Gcp48GW0X3UWkveZlgS8Ty7OrZbtyXxrtCT6YcRU5oVu8WkktYT4xNqxCpTR4xjog24WLSotJr5tzMEKREGI1U0OcJnurb3Aj4pF5JJIfmyTWLCX5KtQRPVKrwGBzqQUDitRhgCz18nNR5ognXbYV7w3M~LEUW-JZcxn3Qqn4J9Lj9uNqWWWkW4RQzEuQwB7UQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic relationships among food isolated and probiotic enterococci. Strains belonging to different groups are indicated on the bottom left. As outgroups Lactococcus lactis subsp. cremoris MG1363, Lactococcus garvieae Lg2, Listeria monocytogenes HCC23 and Lactobacillus johnsonii NCC 533, were selected. The scale bar indicates the relative distance of sequences. Bootstrap support values are indicated at the nodes. A) Genome-based phylogenetic tree. Genomes were aligned using pMauve and phylogeny reconstructed using a rapid bootstrap analysis with 1000 replicates with RAxML in two steps. First representative strains for each species were used to retrieve the species topology then the procedure was repeated at strain level to maximize resolution here. B) Protein-based phylogenetic tree. The tree is based on all 1:1-orthologs identified with Proteinortho. As in A, a rapid bootstrap analysis with 1000 replicates to search for best-scoring maximum likelihood tree was performed using RAxML. For details, see Materials and Methods." path-from-xml="dsw043f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61462855/dsw043f1.tif" path-from-xml="dsw043f1.tif" target="_blank">View large</a><a section="61462855" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/dsw043f1.png?Expires=2147483647&Signature=WEC3X-CUEwB-~RuPhIc~sIFAtyBG29HSTqMWEDUevjkH7RkfgC3yt03hgK6Fre9n-Ap6n9IabkbI~TqF46wt1a1BnZRv3r4DsV7fK3urTzyQwE~CPYBc3AsR3HFB6mw-HKBJlfUzCGaS1ySG1GJeZui~IUAIydu5p8eZOwUGa3G3z1Cp6iaF9QSUWUrU~fom4e0DQpXV3pEEOeVkvsMLgbvrt6aAvvebAZUpIZUv3CxxC~4TSOwMPukhBcTNXbgf9yOTBm6f2PjolQ9JrgT2X4MuaXnz4e5RZYWlP0TX06GCB1rs2agHl3iilBGOQNoKEPMlBoeFKPqFuk-zeze0oQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=61462855&ar=2670108&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw043f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic relationships among food isolated and probiotic enterococci. Strains belonging to different groups are indicated on the bottom left. As outgroups <em>Lactococcus lactis</em> subsp. <em>cremoris</em> MG1363, <em>Lactococcus garvieae</em> Lg2, <em>Listeria monocytogenes</em> HCC23 and <em>Lactobacillus johnsonii</em> NCC 533, were selected. The scale bar indicates the relative distance of sequences. Bootstrap support values are indicated at the nodes. A) Genome-based phylogenetic tree. Genomes were aligned using pMauve and phylogeny reconstructed using a rapid bootstrap analysis with 1000 replicates with RAxML in two steps. First representative strains for each species were used to retrieve the species topology then the procedure was repeated at strain level to maximize resolution here. B) Protein-based phylogenetic tree. The tree is based on all 1:1-orthologs identified with Proteinortho. As in A, a rapid bootstrap analysis with 1000 replicates to search for best-scoring maximum likelihood tree was performed using RAxML. For details, see Materials and Methods.</p></div></div><div content-id="dsw043-F1" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/m_dsw043f1.png?Expires=2147483647&Signature=e9RSuo58alYkgMdegBBNhE85qZst-huq4CMCjgNGCFjMJf~X3FrDfX7ky-WkuYsMyr-MhiYXMKbcUAS0lje6SIOaLI2K~iDSGszaw9dH7C1wtsqjQRy6l0RIO6M~yHAk9SVDtX8L8c~NgyM-i9r3Gcp48GW0X3UWkveZlgS8Ty7OrZbtyXxrtCT6YcRU5oVu8WkktYT4xNqxCpTR4xjog24WLSotJr5tzMEKREGI1U0OcJnurb3Aj4pF5JJIfmyTWLCX5KtQRPVKrwGBzqQUDitRhgCz18nNR5ognXbYV7w3M~LEUW-JZcxn3Qqn4J9Lj9uNqWWWkW4RQzEuQwB7UQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic relationships among food isolated and probiotic enterococci. Strains belonging to different groups are indicated on the bottom left. As outgroups Lactococcus lactis subsp. cremoris MG1363, Lactococcus garvieae Lg2, Listeria monocytogenes HCC23 and Lactobacillus johnsonii NCC 533, were selected. The scale bar indicates the relative distance of sequences. Bootstrap support values are indicated at the nodes. A) Genome-based phylogenetic tree. Genomes were aligned using pMauve and phylogeny reconstructed using a rapid bootstrap analysis with 1000 replicates with RAxML in two steps. First representative strains for each species were used to retrieve the species topology then the procedure was repeated at strain level to maximize resolution here. B) Protein-based phylogenetic tree. The tree is based on all 1:1-orthologs identified with Proteinortho. As in A, a rapid bootstrap analysis with 1000 replicates to search for best-scoring maximum likelihood tree was performed using RAxML. For details, see Materials and Methods." path-from-xml="dsw043f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61462855/dsw043f1.tif" path-from-xml="dsw043f1.tif" target="_blank">View large</a><a section="61462855" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/dsw043f1.png?Expires=2147483647&Signature=WEC3X-CUEwB-~RuPhIc~sIFAtyBG29HSTqMWEDUevjkH7RkfgC3yt03hgK6Fre9n-Ap6n9IabkbI~TqF46wt1a1BnZRv3r4DsV7fK3urTzyQwE~CPYBc3AsR3HFB6mw-HKBJlfUzCGaS1ySG1GJeZui~IUAIydu5p8eZOwUGa3G3z1Cp6iaF9QSUWUrU~fom4e0DQpXV3pEEOeVkvsMLgbvrt6aAvvebAZUpIZUv3CxxC~4TSOwMPukhBcTNXbgf9yOTBm6f2PjolQ9JrgT2X4MuaXnz4e5RZYWlP0TX06GCB1rs2agHl3iilBGOQNoKEPMlBoeFKPqFuk-zeze0oQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=61462855&ar=2670108&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw043f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic relationships among food isolated and probiotic enterococci. Strains belonging to different groups are indicated on the bottom left. As outgroups <em>Lactococcus lactis</em> subsp. <em>cremoris</em> MG1363, <em>Lactococcus garvieae</em> Lg2, <em>Listeria monocytogenes</em> HCC23 and <em>Lactobacillus johnsonii</em> NCC 533, were selected. The scale bar indicates the relative distance of sequences. Bootstrap support values are indicated at the nodes. A) Genome-based phylogenetic tree. Genomes were aligned using pMauve and phylogeny reconstructed using a rapid bootstrap analysis with 1000 replicates with RAxML in two steps. First representative strains for each species were used to retrieve the species topology then the procedure was repeated at strain level to maximize resolution here. B) Protein-based phylogenetic tree. The tree is based on all 1:1-orthologs identified with Proteinortho. As in A, a rapid bootstrap analysis with 1000 replicates to search for best-scoring maximum likelihood tree was performed using RAxML. For details, see Materials and Methods.</p></div></div></div><p>To provide an in-depth phylogenetic reconstruction we iterated the procedure at strain level. The <em>E. faecium</em> strains were separated into two major subgroups of which one also exhibited a more diverged subgroup comprising the two probiotic and very similar strains L-X and T-110. These results are in agreement with those reported by Palmer <em>et al</em><em>.</em><a reveal-id="dsw043-B41" data-open="dsw043-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> which describe the <em>E. faecium</em> strains separation into two clades using orthologous groups but without including the two probiotic strains used here. In addition, non-clinical <em>E. faecium</em> strains clustered into two clades according to an analysis carried out by Kim <em>et al</em><em>.</em><a reveal-id="dsw043-B12" data-open="dsw043-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a></p><p>In a second approach, we used protein sequences commonly shared in the species and strains under this study (<a reveal-id="dsw043-F1" data-open="dsw043-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>). Both analyses are in good agreement. However, in contrast to the genome based analysis, <em>E. faecalis</em> is reported as the most ancient branch of enterococci followed by the <em>E. raffinosus</em> and <em>E. maloduratus</em> cluster. As both trees branch with fairly high bootstrap values (0.96 and 0.86) a different evolutional pattern at nucleic and amino acid level can be assumed, leading to this result. Similarly, we find minor differences in the topology of the <em>E. faecium</em> group. We want to note that these findings were highly reproducible in all genome- and protein-based bootstrap runs. As for the genome-based tree, <em>E. mundtti</em> branched separately before <em>E. faecium</em> who shared it last ancestor with <em>E. durans</em> IPLA 655 and <em>E. hirae</em> INF E1.</p><p>Both approaches led to the conclusion that non correlations between isolation source/probiotic properties and phylogenetic signal exist neither at species or strain levels. This indicates that niche specific adaptations do not affect the genome and proteome as a whole. In this direction, and analyzing strains of the same species individually, it was only seen that <em>E. faecalis</em> 2924 and 19116 from meat on the one hand, and the probiotic strains <em>E. faecium</em> L-X and <em>E. faecium</em> T-110, but not L-3, on the other hand, clustered together. We thus suppose adaptations to be located in the part of the proteome and genome that is not commonly shared among all strains of a certain species.</p>                            <h3 scrollto-destination=61462859 id="61462859" class="section-title" >5.3. Functional annotation</h3>
<div class="block-child-p">As a result of the functional annotation conducted with RAST it was possible to determine the features, assigned to subsystems that were (i) present in all organisms, (ii) related to the isolation source or strain property and (iii) species-specific (<a reveal-id="dsw043-F2" data-open="dsw043-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>, Dataset 1). Among the strains of diary origin the average number of annotated protein encoding genes (PEGs) was 2,968; 2,739 for the meat isolates and 2,960 for the probiotics. In addition, it was seen that, for the former group, an average of 46.64% of the identified CDS were assigned to subsystems, 48.50% in case of the strains isolated from meat products and 49.29% for probiotics (<a reveal-id="dsw043-F2" data-open="dsw043-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). Of those CDS assigned to subsystems, carbohydrates metabolism was the most enriched metabolic category. This is in accordance with the enterococcís ability to grow in different environments motivated by their capability of metabolizing at least 13 different kinds of sugar.<a reveal-id="dsw043-B42" data-open="dsw043-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> Over 30 more can be utilized by at least two members of the genus. These carbohydrates sources are not limited to monomers, they are also usable as polymers.<a reveal-id="dsw043-B42" data-open="dsw043-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> The second highest percentage of PEGs was in the protein metabolism category for <em>E. faecium, E. hirae</em> and <em>E. mundtii</em>, strains some of which share the dairy environment as source of isolation; meanwhile for <em>E. malodoratus</em>, <em>E. raffinosus</em> and <em>E. faecalis</em> species it was in the amino acids and derivatives category (<a reveal-id="dsw043-F2" data-open="dsw043-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>). In this regard, the <em>Lactobacillus</em> adaptation to the dairy niche has been associated with an increase in genes for peptide transport and hydrolysis.<a reveal-id="dsw043-B43" data-open="dsw043-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a><div data-id="dsw043-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/m_dsw043f2.png?Expires=2147483647&Signature=pxgjh6qMmQVN75I9hUWbQbvPdbHySQExpTRO5PQBkWODQf4DiYqG4BJFTSbtDakv1ovhJsQrxWUap~rlWbzAPKf47kQSn5SSM6MEnfBAQxU~DGfvJx~qPsA2upcy7DUI46D8Neangh7iBdiWfk72Luf2JPum~PpREq4GhDFDtZWcJdNYSNkFVEBYpmZa9WjSJjLSGjdfLh9Z3WVKDcZ~QfU~40bz7-VePmVzH7OHkU-QJEj4Gt8ay-kHONJJl49aTW1AAAutCLWiJ83x4-t0A0lcWaMnk9cf2xjAgQeIp0YdZYOcBPL4qWLLKto2U7BqF~5UwUp4Hedt6Xu7eltMyw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Coding sequences dispensed in subsystems and their distribution in categories of metabolic function. A) Percentage of predicted CDS assigned and not assigned to subsystems. B) Distribution of metabolic functions in categories based on the subsystems features counts." path-from-xml="dsw043f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61462860/dsw043f2.tif" path-from-xml="dsw043f2.tif" target="_blank">View large</a><a section="61462860" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/dsw043f2.png?Expires=2147483647&Signature=QkztstB425qezFCv2~NnMxKkQ6PNP-PbEdss0DQoh3dl-syFpJ0tW93oMMI2ENxRPEu644W6gwBl1dI42zfsXhVh6b6od9BPhuxmT4TeBvt5oHO9zxO3CR38UmmZrlfngXgLlW0Im~2bVeEgTxkK-D4WT8Nuz8eliuMKjtQ5Ppaa3pTOwSkNTy55SIYGXHNUnPOKiKupukQxHkxxzTi0H31GhRnwNDISFYgsaaPhOX0LPo4ms5QJxJLWtILBNEzt8S8ZAHaWbXJsTUfJx-DypzXaeRZDwZNhcURUxXAS9SLWUeir5e9j0uVVfYv~Gu15rxC-dFUwPo-Ggehu4WUPyQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=61462860&ar=2670108&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw043f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Coding sequences dispensed in subsystems and their distribution in categories of metabolic function. A) Percentage of predicted CDS assigned and not assigned to subsystems. B) Distribution of metabolic functions in categories based on the subsystems features counts.</p></div></div><div content-id="dsw043-F2" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/m_dsw043f2.png?Expires=2147483647&Signature=pxgjh6qMmQVN75I9hUWbQbvPdbHySQExpTRO5PQBkWODQf4DiYqG4BJFTSbtDakv1ovhJsQrxWUap~rlWbzAPKf47kQSn5SSM6MEnfBAQxU~DGfvJx~qPsA2upcy7DUI46D8Neangh7iBdiWfk72Luf2JPum~PpREq4GhDFDtZWcJdNYSNkFVEBYpmZa9WjSJjLSGjdfLh9Z3WVKDcZ~QfU~40bz7-VePmVzH7OHkU-QJEj4Gt8ay-kHONJJl49aTW1AAAutCLWiJ83x4-t0A0lcWaMnk9cf2xjAgQeIp0YdZYOcBPL4qWLLKto2U7BqF~5UwUp4Hedt6Xu7eltMyw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Coding sequences dispensed in subsystems and their distribution in categories of metabolic function. A) Percentage of predicted CDS assigned and not assigned to subsystems. B) Distribution of metabolic functions in categories based on the subsystems features counts." path-from-xml="dsw043f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61462860/dsw043f2.tif" path-from-xml="dsw043f2.tif" target="_blank">View large</a><a section="61462860" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/dsw043f2.png?Expires=2147483647&Signature=QkztstB425qezFCv2~NnMxKkQ6PNP-PbEdss0DQoh3dl-syFpJ0tW93oMMI2ENxRPEu644W6gwBl1dI42zfsXhVh6b6od9BPhuxmT4TeBvt5oHO9zxO3CR38UmmZrlfngXgLlW0Im~2bVeEgTxkK-D4WT8Nuz8eliuMKjtQ5Ppaa3pTOwSkNTy55SIYGXHNUnPOKiKupukQxHkxxzTi0H31GhRnwNDISFYgsaaPhOX0LPo4ms5QJxJLWtILBNEzt8S8ZAHaWbXJsTUfJx-DypzXaeRZDwZNhcURUxXAS9SLWUeir5e9j0uVVfYv~Gu15rxC-dFUwPo-Ggehu4WUPyQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=61462860&ar=2670108&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw043f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Coding sequences dispensed in subsystems and their distribution in categories of metabolic function. A) Percentage of predicted CDS assigned and not assigned to subsystems. B) Distribution of metabolic functions in categories based on the subsystems features counts.</p></div></div></div><p>None of the features of a certain functional role were exclusive of the members of one of the three major groups (dairy isolates, meat isolates, probiotics) under consideration (Dataset 1). This observation was valid when taking into account the 100, 60 and 40% of the members of each group for the respective comparison. However, it should be emphasized that 115 of a total of 254 subsystems analyzed were present in all the organisms and can thus be regarded as basic equipment of this genus. The number of CDS for the beta-glucoside metabolism was the most abundant in all groups (Dataset 1). Two beta-glucosides operons were present in all strains, the Bgl (beta-glucoside) and Cel (cellobiose) operons. The genes <em>alsS</em> and <em>alsD</em> of the alpha-acetolactate operon, encoding for the acetolactate synthase and alpha-acetolactate decarboxylase, respectively, were also found in all the <em>Enterococcus</em> strains; both genes are involved in acetoin production from piruvate.<a reveal-id="dsw043-B32" data-open="dsw043-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> Seventy-four out of the 115 subsystems were highly conserved as there was a difference of ≤ 1 in the average number of features assigned to the strains of the different groups (Dataset 1). Interestingly, a higher number of genes of the beta-glucoside metabolism, lactose and galactose uptake and utilization, and fructose utilization subsystems were found, in most cases, in the dairy isolates and in probiotics; meanwhile, meat isolates had a greater abundance of genes involved in the oxidative stress response. Dairy bacteria had, at the same time, on average more CDS related to chitin and <em>N</em>-acetylglucosamine utilization than the member of the other groups and probiotics, for their part, were in general more enriched in genes coding for the ABC transporter alkylphosphonate and the energy-coupling factor transporters.</p><p>In parallel, an analysis aimed at evaluating the existence of species-specific genomic fingerprints sheds more conclusive results. Specifically, for those species that included more than one strain, it was seen that <em>E. faecalis</em>, for instance, had several features nucleated in subsystems of five different categories, some of them part of the lactate and ethanolamine utilization machinery. Ethanolamine utilization is considered a survival advantage for bacteria that inhabit the gastrointestinal tract and was also related to pathogens virulence.<a reveal-id="dsw043-B44" data-open="dsw043-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a><em>E. faecium</em> strains had, on the other hand, genes for the late competence event, fundamental for exogenous DNA acquisition, not localized in any other species; and <em>E. mundtii</em> strains had CDS of the carotenoids subsystem, responsible of their particular color, and one gene encoding a NG, NG-dimethylarginine dimethylaminohydrolase 1.</p>                            <h3 scrollto-destination=61462863 id="61462863" class="section-title" >5.4. Niche-specific proteins and core proteome</h3>
<div class="block-child-p">We used the RAST annotation to determine genes of certain classes that could be located in a niche specific pattern and that were present in all species, irrespective of their origin (<a reveal-id="dsw043-F3" data-open="dsw043-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/dsw043_Supp.zip?Expires=2147483647&amp;Signature=qfrsEYitdaJFRtVb4lVTNX16upeZv8S1FjY1XCqewM9sqQh0e4qpEC679hfVK0HFqPdKNcQnFO1Gwa-dzW4JwoTXPeTDnBceFgGx7NZ3YWjVXWKJOh4x54hNDn57A9V46VBYbxEnR6dnIUmjr4FdsbLmOTACDeDJedoAFYuVLWryR6FHza5a23P88etL6BkNLUglUF6UYULgr0cd2tRX-QIgzhsKu21VyNcv5B7O7LHKc6mhhmOTzKWiJxE8BTHIijLs6MDBlnmsBkB9czJxa~FAznSfC-NsCoetlHpJ9z~bQwtssQ09AAjbTcHT~9c-3Q3xe8MiIXzYFKcJybo1CA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1, Data set 2</a></span>). The least specific group in this regard was represented by the strains isolated from meat (all belonging to <em>E. faecium</em> or <em>E. faecalis</em>). This group exclusively exhibited the protein IbrB, a co-activator of gene expression which is most likely derived from prophage, and a methyltransferase that modifies adenine bases of the 18S rRNA (in eukaryotes). Both genes must thus be results of horizontal gene transfer related to the environment. On the other hand, most genes specific to dairy and probiotic strains were hypothetical proteins. In the dairy group, these were most pronounced as homologs were located in strains of four different species. Among others, this included an <em>E. faecium</em> strain, a species that is otherwise present in all three groups (dairy, meat, probiotics), indicating these hypothetical proteins to be highly niche specific adaptations, however, of unknown function. Interestingly, one of the niche-specific genes codes for a membrane transport protein that passively transfer molecules or ions through the cell membrane and could thus be related to the peculiarities of the dairy environment. For probiotic strains we have additionally located a putative DNA methyltransferase. Homologs are mainly known from the <em>Haemophilus</em> genus.<a reveal-id="dsw043-B45" data-open="dsw043-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a><div data-id="dsw043-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/m_dsw043f3.png?Expires=2147483647&amp;Signature=tQbDB231J~UgF7jGZ~8lW3CJSAN-lfK8BbC8ABGN6uaB1Cq9EKsztrAI807iDUId7zARQ05~-pP6s7y9BDyKabDrGQ2xCG6TmamXa4a1g105BKsJbIqN-27ISVh~ZQO-HejI03F3fn-48vkH6nX1evW8QKrymXGyA1o77w6Js4lttmFlfjH-tgjnUa5vVp05I21SRVzfQLauIUfafRmnIyFqrWaZt~fYZTPEV7VOfywY3gR8ZcZnXfCZt4u714doarVYvdThsy616Xv0p7cJW~suBHPmtQBT8zbWzQEmQDr-YqGSSt2KRjtz4ZbkYcpKHCdKfQHcbBzbk5mrGGSW2A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Venn diagram showing i) the number of unique FIGfam protein classes that were present in at least 40% of the strains and in at least two different species of the respective group but in no species/strain of any other group (unique), ii) the number of classes that were similarly present in two or more groups (overlapped), iii) the number of protein classes that were similarly present in all groups (in the center of the diagram), and iv) the subset of classes that were found to be present in all groups and in at least four different Enterococci species rather than two (indicated in parentheses)." path-from-xml="dsw043f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61462864/dsw043f3.tif" path-from-xml="dsw043f3.tif" target="_blank">View large</a><a section="61462864" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/dsw043f3.png?Expires=2147483647&amp;Signature=xdESCq506~M0O37yUBvsor8ibGD2bms2clsb8qcGU5qyWOUUSQ~Arzy0z7o32PoBvQT46nSUd-BHPjP3mBP1cS3v0Ytm7wDeTtlulJAayXLojxovbuMK8oX4YUlhvT0SCN4cUeW9NcphLchAjq1754uut4vr-Zc-FpRj44vGXlolUbSs6kYBqtQB89JdtQklDSEGvQ5KIJjmW8t5B4K9IrkSjPhRH4WTgm84QeW-CND9Fu39EFQp7YVRjkKy6ate99jXRmqN~JhoW4TdE-YbVWTn~h~6DZIRCjV3QhgywLEd2gZHpORGzrmTBsnKD0nmd~jyYDOirK8-swdPNMe5Ew__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=61462864&amp;ar=2670108&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw043f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Venn diagram showing i) the number of unique FIGfam protein classes that were present in at least 40% of the strains and in at least two different species of the respective group but in no species/strain of any other group (unique), ii) the number of classes that were similarly present in two or more groups (overlapped), iii) the number of protein classes that were similarly present in all groups (in the center of the diagram), and iv) the subset of classes that were found to be present in all groups and in at least four different Enterococci species rather than two (indicated in parentheses).</p></div></div><div content-id="dsw043-F3" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/m_dsw043f3.png?Expires=2147483647&amp;Signature=tQbDB231J~UgF7jGZ~8lW3CJSAN-lfK8BbC8ABGN6uaB1Cq9EKsztrAI807iDUId7zARQ05~-pP6s7y9BDyKabDrGQ2xCG6TmamXa4a1g105BKsJbIqN-27ISVh~ZQO-HejI03F3fn-48vkH6nX1evW8QKrymXGyA1o77w6Js4lttmFlfjH-tgjnUa5vVp05I21SRVzfQLauIUfafRmnIyFqrWaZt~fYZTPEV7VOfywY3gR8ZcZnXfCZt4u714doarVYvdThsy616Xv0p7cJW~suBHPmtQBT8zbWzQEmQDr-YqGSSt2KRjtz4ZbkYcpKHCdKfQHcbBzbk5mrGGSW2A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Venn diagram showing i) the number of unique FIGfam protein classes that were present in at least 40% of the strains and in at least two different species of the respective group but in no species/strain of any other group (unique), ii) the number of classes that were similarly present in two or more groups (overlapped), iii) the number of protein classes that were similarly present in all groups (in the center of the diagram), and iv) the subset of classes that were found to be present in all groups and in at least four different Enterococci species rather than two (indicated in parentheses)." path-from-xml="dsw043f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61462864/dsw043f3.tif" path-from-xml="dsw043f3.tif" target="_blank">View large</a><a section="61462864" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/dsw043f3.png?Expires=2147483647&amp;Signature=xdESCq506~M0O37yUBvsor8ibGD2bms2clsb8qcGU5qyWOUUSQ~Arzy0z7o32PoBvQT46nSUd-BHPjP3mBP1cS3v0Ytm7wDeTtlulJAayXLojxovbuMK8oX4YUlhvT0SCN4cUeW9NcphLchAjq1754uut4vr-Zc-FpRj44vGXlolUbSs6kYBqtQB89JdtQklDSEGvQ5KIJjmW8t5B4K9IrkSjPhRH4WTgm84QeW-CND9Fu39EFQp7YVRjkKy6ate99jXRmqN~JhoW4TdE-YbVWTn~h~6DZIRCjV3QhgywLEd2gZHpORGzrmTBsnKD0nmd~jyYDOirK8-swdPNMe5Ew__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=61462864&amp;ar=2670108&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw043f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Venn diagram showing i) the number of unique FIGfam protein classes that were present in at least 40% of the strains and in at least two different species of the respective group but in no species/strain of any other group (unique), ii) the number of classes that were similarly present in two or more groups (overlapped), iii) the number of protein classes that were similarly present in all groups (in the center of the diagram), and iv) the subset of classes that were found to be present in all groups and in at least four different Enterococci species rather than two (indicated in parentheses).</p></div></div></div><p>Looking at the overlap of shared genes, the meat and probiotic group had the least in common. Above of a phage lysin found in both groups, the fumarate reductase probably represented a true functional finding. Fumaric acid is a food additive used for antimicrobial preservation of meat to reduce the spread of pathogens.<a reveal-id="dsw043-B46" data-open="dsw043-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> With respect to this background, it is reasonable to conclude that the respective strains from the group are able to degrade this acid in their habitat (meat isolates). Probiotic strains, on the other hand, might be in contact with fumaric acid in the gastrointestinal habitat, while it does not play a role for strains from the dairy group.</p><p>The overlap of the dairy and the probiotic groups can also be seen as set of genes that were not present in the meat group. It contained three phosphonate ABC transporter proteins, all found linked in the same strains, indicating that phosphonates play an important role here but not in meat related environments. In addition two proteins related to galactoside metabolism were located. Both were reported with an <em>E</em>-value slightly above the 0.05 significance threshold (0.076). Again, this energy-source is obviously not important for meat related strains. Alongside with galactoside related proteins, a not further specifiable membrane transporter was identified to be present. Similarly the alpha-glycosyltransferase was found solely in the dairy and probiotic groups.</p><p>With 15 shared proteins, the overlap of the meat and the dairy group seems rather large. Most of them, however, were related to phages (six) or were hypothetical proteins (five) often exhibiting weaker <em>E</em>-values closer to or slightly above 0.05. The remaining specific proteins were the ATP-binding protein p271, an ATPase component of an uncharacterized ABC-type transport system and, most interestingly a probable extracellular solute-binding protein that was present in the same strains as an arabinofuranosidase. Both might thus be related, indicating that arabinose might not be as relevant for probiotic strains.</p><p>Finally, we determined the core proteome of all enterococci used in this study by asking which genes were present in 40% of all strains in all groups (see Materials and Methods). As a result, 1617 FIGfam families were located to be universally present. Out of them 1457 were found to be present in at least four different enterococci species. A detailed list can be found in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/dsw043_Supp.zip?Expires=2147483647&Signature=qfrsEYitdaJFRtVb4lVTNX16upeZv8S1FjY1XCqewM9sqQh0e4qpEC679hfVK0HFqPdKNcQnFO1Gwa-dzW4JwoTXPeTDnBceFgGx7NZ3YWjVXWKJOh4x54hNDn57A9V46VBYbxEnR6dnIUmjr4FdsbLmOTACDeDJedoAFYuVLWryR6FHza5a23P88etL6BkNLUglUF6UYULgr0cd2tRX-QIgzhsKu21VyNcv5B7O7LHKc6mhhmOTzKWiJxE8BTHIijLs6MDBlnmsBkB9czJxa~FAznSfC-NsCoetlHpJ9z~bQwtssQ09AAjbTcHT~9c-3Q3xe8MiIXzYFKcJybo1CA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>.</p>                            <h2 scrollto-destination=61462869 id="61462869" class="section-title" >6. Genomic features among food-related and probiotic enterococcal strains</h2>
                            <h3 scrollto-destination=61462870 id="61462870" class="section-title" >6.1. Bacteriocins</h3>
<p>Genes encoding bacteriocins are widely disseminated among enterococci from different origins.<a reveal-id="dsw043-B47" data-open="dsw043-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> For probiotic bacteria, the ability to successfully outcompete undesired species is often due to, or enhanced by, the production of bacteriocins.<a reveal-id="dsw043-B48" data-open="dsw043-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> With this background, a search for genes related to bacteriocins was carried out through BAGEL 3 and BLAST (<a reveal-id="dsw043-T2" data-open="dsw043-T2" class="link link-reveal link-table xref-fig">Table 2</a>). <div content-id="dsw043-T2" class="table-modal"><div class="table-wrap"><div id="dsw043-T2" data-id="dsw043-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Number of antibiotic resistance and bacteriocin genes in the enterococcal genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th colspan="2">  </th><th colspan="6">ARG for each antibiotic class<hr /> </th><th colspan="4">Bacteriocin class<hr /> </th></tr><tr><th>Group </th><th>Strain </th><th>AGly </th><th>Gly </th><th>MLS </th><th>Tet </th><th>Tmt </th><th>∑ </th><th>I </th><th>II </th><th>III </th><th>∑ </th></tr></thead><tbody><tr><td rowspan="12">Dairy isolates </td><td>IPLA 655 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td></tr><tr><td>INF E1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td></tr><tr><td>CRL 1879 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">6 </td><td char=".">1 </td><td char=".">7 </td></tr><tr><td>E1604 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td><td char=".">2 </td></tr><tr><td>NRRL B-2354 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">6 </td><td char=".">1 </td><td char=".">7 </td></tr><tr><td>UC7256 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td></tr><tr><td>UC 7267 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td></tr><tr><td>UC8668 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td></tr><tr><td>ATCC 43197 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td></tr><tr><td>ATCC 882 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td><td char=".">2 </td></tr><tr><td>CRL1656 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td></tr><tr><td>CRL35 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td></tr><tr><td rowspan="6">Meat isolates </td><td>2924 </td><td char=".">0 </td><td char=".">7 </td><td char=".">4 </td><td char=".">2 </td><td char=".">1 </td><td char=".">14 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td></tr><tr><td>19116 </td><td char=".">5 </td><td char=".">0 </td><td char=".">4 </td><td char=".">1 </td><td char=".">0 </td><td char=".">10 </td><td char=".">2 </td><td char=".">1 </td><td char=".">1 </td><td char=".">4 </td></tr><tr><td>E1613 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">1 </td><td char=".">4 </td><td char=".">1 </td><td char=".">6 </td></tr><tr><td>UC7251 </td><td char=".">3 </td><td char=".">0 </td><td char=".">3 </td><td char=".">2 </td><td char=".">0 </td><td char=".">8 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td></tr><tr><td>UC8733 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">8 </td><td char=".">2 </td><td char=".">10 </td></tr><tr><td>UC10237 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td></tr><tr><td rowspan="5">Probiotics </td><td>PC1.1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td></tr><tr><td>Symbioflor 1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td></tr><tr><td>L-X </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">3 </td><td char=".">1 </td><td char=".">4 </td></tr><tr><td>T-110 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">2 </td><td char=".">1 </td><td char=".">3 </td></tr><tr><td>CFTRI 2200 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">2 </td><td char=".">0 </td><td char=".">4 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td></tr><tr><td rowspan="2">Dairy isolates and probiotics </td><td>L-3 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">3 </td><td char=".">1 </td><td char=".">4 </td></tr><tr><td>MB5259 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">2 </td><td char=".">1 </td><td char=".">0 </td><td char=".">3 </td></tr></tbody></table></div><div content-id="dsw043-T2" class="table-modal"><table><thead name="thead" align=""><tr><th colspan="2">  </th><th colspan="6">ARG for each antibiotic class<hr /> </th><th colspan="4">Bacteriocin class<hr /> </th></tr><tr><th>Group </th><th>Strain </th><th>AGly </th><th>Gly </th><th>MLS </th><th>Tet </th><th>Tmt </th><th>∑ </th><th>I </th><th>II </th><th>III </th><th>∑ </th></tr></thead><tbody><tr><td rowspan="12">Dairy isolates </td><td>IPLA 655 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td></tr><tr><td>INF E1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td></tr><tr><td>CRL 1879 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">6 </td><td char=".">1 </td><td char=".">7 </td></tr><tr><td>E1604 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td><td char=".">2 </td></tr><tr><td>NRRL B-2354 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">6 </td><td char=".">1 </td><td char=".">7 </td></tr><tr><td>UC7256 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td></tr><tr><td>UC 7267 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td></tr><tr><td>UC8668 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td></tr><tr><td>ATCC 43197 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td></tr><tr><td>ATCC 882 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td><td char=".">2 </td></tr><tr><td>CRL1656 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td></tr><tr><td>CRL35 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td></tr><tr><td rowspan="6">Meat isolates </td><td>2924 </td><td char=".">0 </td><td char=".">7 </td><td char=".">4 </td><td char=".">2 </td><td char=".">1 </td><td char=".">14 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td></tr><tr><td>19116 </td><td char=".">5 </td><td char=".">0 </td><td char=".">4 </td><td char=".">1 </td><td char=".">0 </td><td char=".">10 </td><td char=".">2 </td><td char=".">1 </td><td char=".">1 </td><td char=".">4 </td></tr><tr><td>E1613 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">1 </td><td char=".">4 </td><td char=".">1 </td><td char=".">6 </td></tr><tr><td>UC7251 </td><td char=".">3 </td><td char=".">0 </td><td char=".">3 </td><td char=".">2 </td><td char=".">0 </td><td char=".">8 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td></tr><tr><td>UC8733 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">8 </td><td char=".">2 </td><td char=".">10 </td></tr><tr><td>UC10237 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td></tr><tr><td rowspan="5">Probiotics </td><td>PC1.1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td></tr><tr><td>Symbioflor 1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td></tr><tr><td>L-X </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">3 </td><td char=".">1 </td><td char=".">4 </td></tr><tr><td>T-110 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">2 </td><td char=".">1 </td><td char=".">3 </td></tr><tr><td>CFTRI 2200 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">2 </td><td char=".">0 </td><td char=".">4 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td></tr><tr><td rowspan="2">Dairy isolates and probiotics </td><td>L-3 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">3 </td><td char=".">1 </td><td char=".">4 </td></tr><tr><td>MB5259 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">2 </td><td char=".">1 </td><td char=".">0 </td><td char=".">3 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn3"><div class="fn"><p>ARG: antibiotic resistance gene; AGly: aminoglycosides; Gly: glycopeptides; MLS: macrolide-lincosamidestreptogramin; Tet: tetracyclines; Tmt: trimethoprim.</p></div></div></div></div></div><div class="table-wrap"><div id="dsw043-T2" data-id="dsw043-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Number of antibiotic resistance and bacteriocin genes in the enterococcal genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th colspan="2">  </th><th colspan="6">ARG for each antibiotic class<hr /> </th><th colspan="4">Bacteriocin class<hr /> </th></tr><tr><th>Group </th><th>Strain </th><th>AGly </th><th>Gly </th><th>MLS </th><th>Tet </th><th>Tmt </th><th>∑ </th><th>I </th><th>II </th><th>III </th><th>∑ </th></tr></thead><tbody><tr><td rowspan="12">Dairy isolates </td><td>IPLA 655 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td></tr><tr><td>INF E1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td></tr><tr><td>CRL 1879 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">6 </td><td char=".">1 </td><td char=".">7 </td></tr><tr><td>E1604 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td><td char=".">2 </td></tr><tr><td>NRRL B-2354 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">6 </td><td char=".">1 </td><td char=".">7 </td></tr><tr><td>UC7256 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td></tr><tr><td>UC 7267 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td></tr><tr><td>UC8668 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td></tr><tr><td>ATCC 43197 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td></tr><tr><td>ATCC 882 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td><td char=".">2 </td></tr><tr><td>CRL1656 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td></tr><tr><td>CRL35 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td></tr><tr><td rowspan="6">Meat isolates </td><td>2924 </td><td char=".">0 </td><td char=".">7 </td><td char=".">4 </td><td char=".">2 </td><td char=".">1 </td><td char=".">14 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td></tr><tr><td>19116 </td><td char=".">5 </td><td char=".">0 </td><td char=".">4 </td><td char=".">1 </td><td char=".">0 </td><td char=".">10 </td><td char=".">2 </td><td char=".">1 </td><td char=".">1 </td><td char=".">4 </td></tr><tr><td>E1613 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">1 </td><td char=".">4 </td><td char=".">1 </td><td char=".">6 </td></tr><tr><td>UC7251 </td><td char=".">3 </td><td char=".">0 </td><td char=".">3 </td><td char=".">2 </td><td char=".">0 </td><td char=".">8 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td></tr><tr><td>UC8733 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">8 </td><td char=".">2 </td><td char=".">10 </td></tr><tr><td>UC10237 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td></tr><tr><td rowspan="5">Probiotics </td><td>PC1.1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td></tr><tr><td>Symbioflor 1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td></tr><tr><td>L-X </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">3 </td><td char=".">1 </td><td char=".">4 </td></tr><tr><td>T-110 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">2 </td><td char=".">1 </td><td char=".">3 </td></tr><tr><td>CFTRI 2200 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">2 </td><td char=".">0 </td><td char=".">4 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td></tr><tr><td rowspan="2">Dairy isolates and probiotics </td><td>L-3 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">3 </td><td char=".">1 </td><td char=".">4 </td></tr><tr><td>MB5259 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">2 </td><td char=".">1 </td><td char=".">0 </td><td char=".">3 </td></tr></tbody></table></div><div content-id="dsw043-T2" class="table-modal"><table><thead name="thead" align=""><tr><th colspan="2">  </th><th colspan="6">ARG for each antibiotic class<hr /> </th><th colspan="4">Bacteriocin class<hr /> </th></tr><tr><th>Group </th><th>Strain </th><th>AGly </th><th>Gly </th><th>MLS </th><th>Tet </th><th>Tmt </th><th>∑ </th><th>I </th><th>II </th><th>III </th><th>∑ </th></tr></thead><tbody><tr><td rowspan="12">Dairy isolates </td><td>IPLA 655 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td></tr><tr><td>INF E1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td></tr><tr><td>CRL 1879 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">6 </td><td char=".">1 </td><td char=".">7 </td></tr><tr><td>E1604 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td><td char=".">2 </td></tr><tr><td>NRRL B-2354 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">6 </td><td char=".">1 </td><td char=".">7 </td></tr><tr><td>UC7256 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td></tr><tr><td>UC 7267 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td></tr><tr><td>UC8668 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td></tr><tr><td>ATCC 43197 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td></tr><tr><td>ATCC 882 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td><td char=".">2 </td></tr><tr><td>CRL1656 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td></tr><tr><td>CRL35 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td></tr><tr><td rowspan="6">Meat isolates </td><td>2924 </td><td char=".">0 </td><td char=".">7 </td><td char=".">4 </td><td char=".">2 </td><td char=".">1 </td><td char=".">14 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td></tr><tr><td>19116 </td><td char=".">5 </td><td char=".">0 </td><td char=".">4 </td><td char=".">1 </td><td char=".">0 </td><td char=".">10 </td><td char=".">2 </td><td char=".">1 </td><td char=".">1 </td><td char=".">4 </td></tr><tr><td>E1613 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">1 </td><td char=".">4 </td><td char=".">1 </td><td char=".">6 </td></tr><tr><td>UC7251 </td><td char=".">3 </td><td char=".">0 </td><td char=".">3 </td><td char=".">2 </td><td char=".">0 </td><td char=".">8 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td></tr><tr><td>UC8733 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">8 </td><td char=".">2 </td><td char=".">10 </td></tr><tr><td>UC10237 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td></tr><tr><td rowspan="5">Probiotics </td><td>PC1.1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">1 </td><td char=".">1 </td></tr><tr><td>Symbioflor 1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td><td char=".">0 </td></tr><tr><td>L-X </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">3 </td><td char=".">1 </td><td char=".">4 </td></tr><tr><td>T-110 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">2 </td><td char=".">1 </td><td char=".">3 </td></tr><tr><td>CFTRI 2200 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">2 </td><td char=".">0 </td><td char=".">4 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td></tr><tr><td rowspan="2">Dairy isolates and probiotics </td><td>L-3 </td><td char=".">1 </td><td char=".">0 </td><td char=".">1 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">3 </td><td char=".">1 </td><td char=".">4 </td></tr><tr><td>MB5259 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">0 </td><td char=".">0 </td><td char=".">2 </td><td char=".">2 </td><td char=".">1 </td><td char=".">0 </td><td char=".">3 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn3"><div class="fn"><p>ARG: antibiotic resistance gene; AGly: aminoglycosides; Gly: glycopeptides; MLS: macrolide-lincosamidestreptogramin; Tet: tetracyclines; Tmt: trimethoprim.</p></div></div></div></div></p><p>Of all species under examination, <em>E. faecium</em> was the one that harbored the larger number of bacteriocin genes in general, being the meat isolate <em>E. faecium</em> UC8733 the strain with the highest bacteriocinogenic potential with eight genes encoding for class II and two for class III bacteriocins. Reinforcing the idea, in previous studies we could identify six class II bacteriocin gene clusters in <em>E. faecium</em> CRL 1879<a reveal-id="dsw043-B49" data-open="dsw043-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> coincident with those found in NRRL B-2354, also of dairy origin. Moreover, and in agreement with our previous findings related to non-clinical and clinical <em>E. faecium</em><a reveal-id="dsw043-B47" data-open="dsw043-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a>, no correlations between the presence of enterocin structural genes and strains origin were observed in the present analysis. Özdemir <em>et al</em><em>.</em><a reveal-id="dsw043-B50" data-open="dsw043-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> also described the same phenomenon among enterococcal species isolated from different origins (including river, treatment plant, spring and garbage water, soil, animal and vegetables).</p><p>Enterocins A and B were always considered as <em>hallmark</em> of this genus. However, as we have shown previously, not all strains carrying enterocin A genes (<em>entA</em>) also harbor a enterocin B gene (<em>entB</em>).<a reveal-id="dsw043-B49" data-open="dsw043-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> The first one (<em>entA)</em> was detected in CRL 1879, NRRL B-2354, UC8733, L-3, L-X and UC10237, all belonging to the <em>E. faecium</em> species. Surprisingly, the class III bacteriocin enterolysin A, a bacteriolysin, first characterized in <em>E. faecalis</em>, was not found in all strains belonging to this species. However, this gene was present in all <em>E. faecium</em> belonging to the three different groups (dairy isolates, meat isolates and probiotics) except in <em>E. faecium</em> UC10237 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/dsw043_Supp.zip?Expires=2147483647&Signature=qfrsEYitdaJFRtVb4lVTNX16upeZv8S1FjY1XCqewM9sqQh0e4qpEC679hfVK0HFqPdKNcQnFO1Gwa-dzW4JwoTXPeTDnBceFgGx7NZ3YWjVXWKJOh4x54hNDn57A9V46VBYbxEnR6dnIUmjr4FdsbLmOTACDeDJedoAFYuVLWryR6FHza5a23P88etL6BkNLUglUF6UYULgr0cd2tRX-QIgzhsKu21VyNcv5B7O7LHKc6mhhmOTzKWiJxE8BTHIijLs6MDBlnmsBkB9czJxa~FAznSfC-NsCoetlHpJ9z~bQwtssQ09AAjbTcHT~9c-3Q3xe8MiIXzYFKcJybo1CA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Meanwhile, <em>E. faecium</em> E1613 (meat isolate) was the only with a CDS for Subtilosin A. As previously described<a reveal-id="dsw043-B51" data-open="dsw043-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a><sup>,</sup><a reveal-id="dsw043-B52" data-open="dsw043-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a><sup>,</sup><em>E. mundtii</em> CRL35 and <em>E. mundtii</em> CRL1656 (diary isolates) have the CDS for Enterocin CRL35 and Mundticin CRL1656, respectively. According to BAGEL, these were the only bacteriocin encoding genes that they carried within their genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/dsw043_Supp.zip?Expires=2147483647&Signature=qfrsEYitdaJFRtVb4lVTNX16upeZv8S1FjY1XCqewM9sqQh0e4qpEC679hfVK0HFqPdKNcQnFO1Gwa-dzW4JwoTXPeTDnBceFgGx7NZ3YWjVXWKJOh4x54hNDn57A9V46VBYbxEnR6dnIUmjr4FdsbLmOTACDeDJedoAFYuVLWryR6FHza5a23P88etL6BkNLUglUF6UYULgr0cd2tRX-QIgzhsKu21VyNcv5B7O7LHKc6mhhmOTzKWiJxE8BTHIijLs6MDBlnmsBkB9czJxa~FAznSfC-NsCoetlHpJ9z~bQwtssQ09AAjbTcHT~9c-3Q3xe8MiIXzYFKcJybo1CA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Regarding <em>E. faecalis</em>, the strains 19,116 and MB5259 (meat and dairy/probiotics, respectively) were the only harboring genes encoding for Cytolysin structural subunits CylLS and CylLL (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/dsw043_Supp.zip?Expires=2147483647&Signature=qfrsEYitdaJFRtVb4lVTNX16upeZv8S1FjY1XCqewM9sqQh0e4qpEC679hfVK0HFqPdKNcQnFO1Gwa-dzW4JwoTXPeTDnBceFgGx7NZ3YWjVXWKJOh4x54hNDn57A9V46VBYbxEnR6dnIUmjr4FdsbLmOTACDeDJedoAFYuVLWryR6FHza5a23P88etL6BkNLUglUF6UYULgr0cd2tRX-QIgzhsKu21VyNcv5B7O7LHKc6mhhmOTzKWiJxE8BTHIijLs6MDBlnmsBkB9czJxa~FAznSfC-NsCoetlHpJ9z~bQwtssQ09AAjbTcHT~9c-3Q3xe8MiIXzYFKcJybo1CA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). In animal models of enterococcal infections it was observed that Cytolysin contributes to virulence,<a reveal-id="dsw043-B53" data-open="dsw043-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> rendering this strain potentially pathogenic. Finally, no CDS related to bacteriocin production were found in the genomes of <em>E. malodoratus</em> ATCC 43197 (incomplete genome) and <em>E. faecalis</em> Symbioflor 1 (complete genome).</p>                            <h3 scrollto-destination=61462874 id="61462874" class="section-title" >6.2. Antibiotic resistance genes</h3>
<p>Enterococci have been defined as increasingly resistant to multiple antibiotics in recent years.<a reveal-id="dsw043-B54" data-open="dsw043-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> They have an intrinsic resistance to many antibiotics and can also acquire resistance to many others, including those of clinical use. <em>E. faecium</em> and <em>E. faecalis</em> are the most successful strains evolving as multi-resistant pathogens because of their ability to acquire and share adaptive traits, including antimicrobial resistance genes, encoded by mobile genetic elements.<a reveal-id="dsw043-B55" data-open="dsw043-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> In this study, genes that conferred resistances to glycopeptides, macrolide-lincosamide-streptogramin, tetracyclines and trimethoprim were detected. The <em>in silico</em> prediction of antibiotic resistance genes revealed that <em>E. faecalis</em> 2924, isolated from turkey meat in 2005, was the only strain carrying genes involved in vancomycin resistance, which is of high clinical importance. At the same time, this was the strain with the highest number of antibiotic resistance genes (14 protein-coding genes, see <a reveal-id="dsw043-T2" data-open="dsw043-T2" class="link link-reveal link-table xref-fig">Table 2</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/dsw043_Supp.zip?Expires=2147483647&Signature=qfrsEYitdaJFRtVb4lVTNX16upeZv8S1FjY1XCqewM9sqQh0e4qpEC679hfVK0HFqPdKNcQnFO1Gwa-dzW4JwoTXPeTDnBceFgGx7NZ3YWjVXWKJOh4x54hNDn57A9V46VBYbxEnR6dnIUmjr4FdsbLmOTACDeDJedoAFYuVLWryR6FHza5a23P88etL6BkNLUglUF6UYULgr0cd2tRX-QIgzhsKu21VyNcv5B7O7LHKc6mhhmOTzKWiJxE8BTHIijLs6MDBlnmsBkB9czJxa~FAznSfC-NsCoetlHpJ9z~bQwtssQ09AAjbTcHT~9c-3Q3xe8MiIXzYFKcJybo1CA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). This finding is in line with the idea that the increased use of antibiotics in healthcare and animal husbandry is linked with the emergence, spreading and persistence of resistant strains in animal products.<a reveal-id="dsw043-B56" data-open="dsw043-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> For example, some enterococci isolated from Portuguese traditional fermented meat products were resistant to erythromycin, nitrofurantoin, rifampicin and tetracycline, but in this case none was to vancomycin.<a reveal-id="dsw043-B57" data-open="dsw043-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> With ten resistance genes <em>E. faecalis</em> 19116 was the second in abundance (<a reveal-id="dsw043-T2" data-open="dsw043-T2" class="link link-reveal link-table xref-fig">Table 2</a>). In contrast, strains belonging to <em>E. mundtii</em>, <em>E. durans</em> and <em>E. maloduratus</em> species did not contain antibiotic resistance related CDS at all (<a reveal-id="dsw043-T3" data-open="dsw043-T3" class="link link-reveal link-table xref-fig">Table 3</a>). Remarkably, all of them were isolated from dairy origin indicating that at least in some instance, antibiotic resistances are of minor relevance in dairy environment compared to the other groups where no strain investigated completely lacks antibiotic resistance genes. <div content-id="dsw043-T3" class="table-modal"><div class="table-wrap"><div id="dsw043-T3" data-id="dsw043-T3" class="table-wrap-title"><div class="label -label">Table 3.</div><div class="caption"><p>Distribution of the intact prophage regions among the enterococcal strains</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Group </th><th>Strain </th><th>Region<a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>Length<a reveal-id="tblfn5" data-open="tblfn5" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> (Kb) </th><th>Nº CDS </th><th>GC% </th><th>Most common phage (hit genes count)<a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th></tr></thead><tbody><tr><td rowspan="18">Dairy isolates </td><td rowspan="3">IPLA 655 </td><td char=".">1 </td><td char=".">40.7 </td><td char=".">55 </td><td char=".">35.4 </td><td>Lister 2389 (16) </td></tr><tr><td char=".">2 </td><td char=".">44.8 </td><td char=".">64 </td><td char=".">36.8 </td><td>Entero phiEf11 (13) </td></tr><tr><td char=".">3 </td><td char=".">16.5 </td><td char=".">27 </td><td char=".">35.1 </td><td>Bacill G (2) </td></tr><tr><td>INF E1 </td><td char=".">1 </td><td char=".">38.8 </td><td char=".">54 </td><td char=".">33.4 </td><td>Entero phiFL1A (10) </td></tr><tr><td>CRL 1879 </td><td char=".">1 </td><td char=".">52.5 </td><td char=".">61 </td><td char=".">37.5 </td><td>Entero EfaCPT1 (11) </td></tr><tr><td>E1604 </td><td char=".">1 </td><td char=".">37.6 </td><td char=".">51 </td><td char=".">36.8 </td><td>Entero IME EF4 (11) </td></tr><tr><td rowspan="2">NRRL B-2354 </td><td char=".">1 </td><td char=".">56.8 </td><td char=".">59 </td><td char=".">36.64 </td><td>Lactob phig1e (12) </td></tr><tr><td char=".">2 </td><td char=".">46 </td><td char=".">58 </td><td char=".">34.96 </td><td>Lactob phig1e (13) </td></tr><tr><td>UC7256 </td><td char=".">1 </td><td char=".">48.7 </td><td char=".">54 </td><td char=".">36.8 </td><td>Lactob phig1e (12) </td></tr><tr><td>UC 7267 </td><td char=".">1 </td><td char=".">31 </td><td char=".">36 </td><td char=".">38.5 </td><td>Bacill BCJA1c (10) </td></tr><tr><td rowspan="2">UC8668 </td><td char=".">1 </td><td char=".">25.8 </td><td char=".">31 </td><td char=".">36.8 </td><td>Lactob phig1e (8) </td></tr><tr><td char=".">2 </td><td char=".">47 </td><td char=".">59 </td><td char=".">36.1 </td><td>Lister 2389 (15) </td></tr><tr><td>ATCC 43197 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td>— </td></tr><tr><td>ATCC 882 </td><td char=".">1 </td><td char=".">46.5 </td><td char=".">66 </td><td char=".">36.7 </td><td>Bacill BCJA1c (10) </td></tr><tr><td>CRL1656 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td>— </td></tr><tr><td rowspan="3">CRL35 </td><td char=".">1 </td><td char=".">38.5 </td><td char=".">50 </td><td char=".">36.3 </td><td>Lactob PL 1 (13) </td></tr><tr><td char=".">2 </td><td char=".">44.4 </td><td char=".">68 </td><td char=".">37.3 </td><td>Lactoc Tuc2009 (13). Entero phiEf11 (13) </td></tr><tr><td char=".">3 </td><td char=".">44.1 </td><td char=".">53 </td><td char=".">37.6 </td><td>Bacill BCJA1c (9) </td></tr><tr><td rowspan="6">Meat isolates </td><td>2924 </td><td char=".">1 </td><td char=".">36.8 </td><td char=".">46 </td><td char=".">34.5 </td><td>Strept phi3396 (8). Lister LP 101 (8) </td></tr><tr><td>19116 </td><td char=".">1 </td><td char=".">68.8 </td><td char=".">72 </td><td char=".">36.2 </td><td>Entero EFC 1 (38) </td></tr><tr><td>E1613 </td><td char=".">1 </td><td char=".">37.1 </td><td char=".">50 </td><td char=".">35.4 </td><td>Lister 2389 (14) </td></tr><tr><td>UC7251 </td><td char=".">1 </td><td char=".">70.1 </td><td char=".">64 </td><td char=".">35.9 </td><td>Lister 2389 (16) </td></tr><tr><td>UC8733 </td><td char=".">1 </td><td char=".">31.9 </td><td char=".">35 </td><td char=".">37.9 </td><td>Entero IME EF4 (12) </td></tr><tr><td>UC10237 </td><td char=".">1 </td><td char=".">21.8 </td><td char=".">23 </td><td char=".">37.5 </td><td>Lister B025 (5) </td></tr><tr><td rowspan="8">Probiotics </td><td>PC1.1 </td><td char=".">1 </td><td char=".">21.6 </td><td char=".">27 </td><td char=".">36.8 </td><td>Lister LP 101 (9) </td></tr><tr><td rowspan="2">Symbioflor 1 </td><td char=".">1 </td><td char=".">41.6 </td><td char=".">54 </td><td char=".">35.83 </td><td>Entero phiEf11 (12). Temper phiNIH1 1 (12) </td></tr><tr><td char=".">2 </td><td char=".">45.2 </td><td char=".">66 </td><td char=".">34.63 </td><td>Entero phiEf11 (63) </td></tr><tr><td rowspan="2">L-X </td><td char=".">1 </td><td char=".">25.5 </td><td char=".">30 </td><td char=".">34.4 </td><td>Bacill BCJA1c (7) </td></tr><tr><td char=".">2 </td><td char=".">41.9 </td><td char=".">55 </td><td char=".">36.1 </td><td>Lactoc Tuc2009 (13). Entero phiEf11 (13). Lactoc TP901 1 (13) </td></tr><tr><td>T-110 </td><td char=".">1 </td><td char=".">41.9 </td><td char=".">56 </td><td char=".">36.16 </td><td>Lactoc Tuc2009 (13). Entero phiEf11 (13) </td></tr><tr><td rowspan="2">CFTRI 2200 </td><td char=".">1 </td><td char=".">50.1 </td><td char=".">71 </td><td char=".">37.1 </td><td>Entero phiFL3A (18) </td></tr><tr><td char=".">2 </td><td char=".">18.8 </td><td char=".">32 </td><td char=".">40.5 </td><td>Entero EFC 1 (3) </td></tr><tr><td rowspan="6">Dairy isolates and probiotics </td><td>L-3 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td>— </td></tr><tr><td rowspan="5">MB5259 </td><td char=".">1 </td><td char=".">30.8 </td><td char=".">22 </td><td char=".">33 </td><td>Lactob phiAT3 (2) </td></tr><tr><td char=".">2 </td><td char=".">43.9 </td><td char=".">68 </td><td char=".">34.4 </td><td>Entero phiEf11 (63) </td></tr><tr><td char=".">3 </td><td char=".">42.5 </td><td char=".">60 </td><td char=".">34.5 </td><td>Entero phiFL1A (15) </td></tr><tr><td char=".">4 </td><td char=".">35.3 </td><td char=".">55 </td><td char=".">35.2 </td><td>Entero phiFL3A (13) </td></tr><tr><td char=".">5 </td><td char=".">41.7 </td><td char=".">31 </td><td char=".">37 </td><td>Staphy StauST398 4 (2) </td></tr></tbody></table></div><div content-id="dsw043-T3" class="table-modal"><table><thead name="thead" align=""><tr><th>Group </th><th>Strain </th><th>Region<a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>Length<a reveal-id="tblfn5" data-open="tblfn5" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> (Kb) </th><th>Nº CDS </th><th>GC% </th><th>Most common phage (hit genes count)<a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th></tr></thead><tbody><tr><td rowspan="18">Dairy isolates </td><td rowspan="3">IPLA 655 </td><td char=".">1 </td><td char=".">40.7 </td><td char=".">55 </td><td char=".">35.4 </td><td>Lister 2389 (16) </td></tr><tr><td char=".">2 </td><td char=".">44.8 </td><td char=".">64 </td><td char=".">36.8 </td><td>Entero phiEf11 (13) </td></tr><tr><td char=".">3 </td><td char=".">16.5 </td><td char=".">27 </td><td char=".">35.1 </td><td>Bacill G (2) </td></tr><tr><td>INF E1 </td><td char=".">1 </td><td char=".">38.8 </td><td char=".">54 </td><td char=".">33.4 </td><td>Entero phiFL1A (10) </td></tr><tr><td>CRL 1879 </td><td char=".">1 </td><td char=".">52.5 </td><td char=".">61 </td><td char=".">37.5 </td><td>Entero EfaCPT1 (11) </td></tr><tr><td>E1604 </td><td char=".">1 </td><td char=".">37.6 </td><td char=".">51 </td><td char=".">36.8 </td><td>Entero IME EF4 (11) </td></tr><tr><td rowspan="2">NRRL B-2354 </td><td char=".">1 </td><td char=".">56.8 </td><td char=".">59 </td><td char=".">36.64 </td><td>Lactob phig1e (12) </td></tr><tr><td char=".">2 </td><td char=".">46 </td><td char=".">58 </td><td char=".">34.96 </td><td>Lactob phig1e (13) </td></tr><tr><td>UC7256 </td><td char=".">1 </td><td char=".">48.7 </td><td char=".">54 </td><td char=".">36.8 </td><td>Lactob phig1e (12) </td></tr><tr><td>UC 7267 </td><td char=".">1 </td><td char=".">31 </td><td char=".">36 </td><td char=".">38.5 </td><td>Bacill BCJA1c (10) </td></tr><tr><td rowspan="2">UC8668 </td><td char=".">1 </td><td char=".">25.8 </td><td char=".">31 </td><td char=".">36.8 </td><td>Lactob phig1e (8) </td></tr><tr><td char=".">2 </td><td char=".">47 </td><td char=".">59 </td><td char=".">36.1 </td><td>Lister 2389 (15) </td></tr><tr><td>ATCC 43197 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td>— </td></tr><tr><td>ATCC 882 </td><td char=".">1 </td><td char=".">46.5 </td><td char=".">66 </td><td char=".">36.7 </td><td>Bacill BCJA1c (10) </td></tr><tr><td>CRL1656 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td>— </td></tr><tr><td rowspan="3">CRL35 </td><td char=".">1 </td><td char=".">38.5 </td><td char=".">50 </td><td char=".">36.3 </td><td>Lactob PL 1 (13) </td></tr><tr><td char=".">2 </td><td char=".">44.4 </td><td char=".">68 </td><td char=".">37.3 </td><td>Lactoc Tuc2009 (13). Entero phiEf11 (13) </td></tr><tr><td char=".">3 </td><td char=".">44.1 </td><td char=".">53 </td><td char=".">37.6 </td><td>Bacill BCJA1c (9) </td></tr><tr><td rowspan="6">Meat isolates </td><td>2924 </td><td char=".">1 </td><td char=".">36.8 </td><td char=".">46 </td><td char=".">34.5 </td><td>Strept phi3396 (8). Lister LP 101 (8) </td></tr><tr><td>19116 </td><td char=".">1 </td><td char=".">68.8 </td><td char=".">72 </td><td char=".">36.2 </td><td>Entero EFC 1 (38) </td></tr><tr><td>E1613 </td><td char=".">1 </td><td char=".">37.1 </td><td char=".">50 </td><td char=".">35.4 </td><td>Lister 2389 (14) </td></tr><tr><td>UC7251 </td><td char=".">1 </td><td char=".">70.1 </td><td char=".">64 </td><td char=".">35.9 </td><td>Lister 2389 (16) </td></tr><tr><td>UC8733 </td><td char=".">1 </td><td char=".">31.9 </td><td char=".">35 </td><td char=".">37.9 </td><td>Entero IME EF4 (12) </td></tr><tr><td>UC10237 </td><td char=".">1 </td><td char=".">21.8 </td><td char=".">23 </td><td char=".">37.5 </td><td>Lister B025 (5) </td></tr><tr><td rowspan="8">Probiotics </td><td>PC1.1 </td><td char=".">1 </td><td char=".">21.6 </td><td char=".">27 </td><td char=".">36.8 </td><td>Lister LP 101 (9) </td></tr><tr><td rowspan="2">Symbioflor 1 </td><td char=".">1 </td><td char=".">41.6 </td><td char=".">54 </td><td char=".">35.83 </td><td>Entero phiEf11 (12). Temper phiNIH1 1 (12) </td></tr><tr><td char=".">2 </td><td char=".">45.2 </td><td char=".">66 </td><td char=".">34.63 </td><td>Entero phiEf11 (63) </td></tr><tr><td rowspan="2">L-X </td><td char=".">1 </td><td char=".">25.5 </td><td char=".">30 </td><td char=".">34.4 </td><td>Bacill BCJA1c (7) </td></tr><tr><td char=".">2 </td><td char=".">41.9 </td><td char=".">55 </td><td char=".">36.1 </td><td>Lactoc Tuc2009 (13). Entero phiEf11 (13). Lactoc TP901 1 (13) </td></tr><tr><td>T-110 </td><td char=".">1 </td><td char=".">41.9 </td><td char=".">56 </td><td char=".">36.16 </td><td>Lactoc Tuc2009 (13). Entero phiEf11 (13) </td></tr><tr><td rowspan="2">CFTRI 2200 </td><td char=".">1 </td><td char=".">50.1 </td><td char=".">71 </td><td char=".">37.1 </td><td>Entero phiFL3A (18) </td></tr><tr><td char=".">2 </td><td char=".">18.8 </td><td char=".">32 </td><td char=".">40.5 </td><td>Entero EFC 1 (3) </td></tr><tr><td rowspan="6">Dairy isolates and probiotics </td><td>L-3 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td>— </td></tr><tr><td rowspan="5">MB5259 </td><td char=".">1 </td><td char=".">30.8 </td><td char=".">22 </td><td char=".">33 </td><td>Lactob phiAT3 (2) </td></tr><tr><td char=".">2 </td><td char=".">43.9 </td><td char=".">68 </td><td char=".">34.4 </td><td>Entero phiEf11 (63) </td></tr><tr><td char=".">3 </td><td char=".">42.5 </td><td char=".">60 </td><td char=".">34.5 </td><td>Entero phiFL1A (15) </td></tr><tr><td char=".">4 </td><td char=".">35.3 </td><td char=".">55 </td><td char=".">35.2 </td><td>Entero phiFL3A (13) </td></tr><tr><td char=".">5 </td><td char=".">41.7 </td><td char=".">31 </td><td char=".">37 </td><td>Staphy StauST398 4 (2) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn4"><div class="fn"><div class="label -label">a</div><p>Intact prophage region.</p></div></div><div content-id="tblfn5"><div class="fn"><div class="label -label">b</div><p>Intact prophage region length.</p></div></div><div content-id="tblfn6"><div class="fn"><div class="label -label">c</div><p>Phage with the highest number of CDS in the region and the number og gene counts in brackets.</p></div></div></div></div></div><div class="table-wrap"><div id="dsw043-T3" data-id="dsw043-T3" class="table-wrap-title"><div class="label -label">Table 3.</div><div class="caption"><p>Distribution of the intact prophage regions among the enterococcal strains</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Group </th><th>Strain </th><th>Region<a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>Length<a reveal-id="tblfn5" data-open="tblfn5" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> (Kb) </th><th>Nº CDS </th><th>GC% </th><th>Most common phage (hit genes count)<a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th></tr></thead><tbody><tr><td rowspan="18">Dairy isolates </td><td rowspan="3">IPLA 655 </td><td char=".">1 </td><td char=".">40.7 </td><td char=".">55 </td><td char=".">35.4 </td><td>Lister 2389 (16) </td></tr><tr><td char=".">2 </td><td char=".">44.8 </td><td char=".">64 </td><td char=".">36.8 </td><td>Entero phiEf11 (13) </td></tr><tr><td char=".">3 </td><td char=".">16.5 </td><td char=".">27 </td><td char=".">35.1 </td><td>Bacill G (2) </td></tr><tr><td>INF E1 </td><td char=".">1 </td><td char=".">38.8 </td><td char=".">54 </td><td char=".">33.4 </td><td>Entero phiFL1A (10) </td></tr><tr><td>CRL 1879 </td><td char=".">1 </td><td char=".">52.5 </td><td char=".">61 </td><td char=".">37.5 </td><td>Entero EfaCPT1 (11) </td></tr><tr><td>E1604 </td><td char=".">1 </td><td char=".">37.6 </td><td char=".">51 </td><td char=".">36.8 </td><td>Entero IME EF4 (11) </td></tr><tr><td rowspan="2">NRRL B-2354 </td><td char=".">1 </td><td char=".">56.8 </td><td char=".">59 </td><td char=".">36.64 </td><td>Lactob phig1e (12) </td></tr><tr><td char=".">2 </td><td char=".">46 </td><td char=".">58 </td><td char=".">34.96 </td><td>Lactob phig1e (13) </td></tr><tr><td>UC7256 </td><td char=".">1 </td><td char=".">48.7 </td><td char=".">54 </td><td char=".">36.8 </td><td>Lactob phig1e (12) </td></tr><tr><td>UC 7267 </td><td char=".">1 </td><td char=".">31 </td><td char=".">36 </td><td char=".">38.5 </td><td>Bacill BCJA1c (10) </td></tr><tr><td rowspan="2">UC8668 </td><td char=".">1 </td><td char=".">25.8 </td><td char=".">31 </td><td char=".">36.8 </td><td>Lactob phig1e (8) </td></tr><tr><td char=".">2 </td><td char=".">47 </td><td char=".">59 </td><td char=".">36.1 </td><td>Lister 2389 (15) </td></tr><tr><td>ATCC 43197 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td>— </td></tr><tr><td>ATCC 882 </td><td char=".">1 </td><td char=".">46.5 </td><td char=".">66 </td><td char=".">36.7 </td><td>Bacill BCJA1c (10) </td></tr><tr><td>CRL1656 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td>— </td></tr><tr><td rowspan="3">CRL35 </td><td char=".">1 </td><td char=".">38.5 </td><td char=".">50 </td><td char=".">36.3 </td><td>Lactob PL 1 (13) </td></tr><tr><td char=".">2 </td><td char=".">44.4 </td><td char=".">68 </td><td char=".">37.3 </td><td>Lactoc Tuc2009 (13). Entero phiEf11 (13) </td></tr><tr><td char=".">3 </td><td char=".">44.1 </td><td char=".">53 </td><td char=".">37.6 </td><td>Bacill BCJA1c (9) </td></tr><tr><td rowspan="6">Meat isolates </td><td>2924 </td><td char=".">1 </td><td char=".">36.8 </td><td char=".">46 </td><td char=".">34.5 </td><td>Strept phi3396 (8). Lister LP 101 (8) </td></tr><tr><td>19116 </td><td char=".">1 </td><td char=".">68.8 </td><td char=".">72 </td><td char=".">36.2 </td><td>Entero EFC 1 (38) </td></tr><tr><td>E1613 </td><td char=".">1 </td><td char=".">37.1 </td><td char=".">50 </td><td char=".">35.4 </td><td>Lister 2389 (14) </td></tr><tr><td>UC7251 </td><td char=".">1 </td><td char=".">70.1 </td><td char=".">64 </td><td char=".">35.9 </td><td>Lister 2389 (16) </td></tr><tr><td>UC8733 </td><td char=".">1 </td><td char=".">31.9 </td><td char=".">35 </td><td char=".">37.9 </td><td>Entero IME EF4 (12) </td></tr><tr><td>UC10237 </td><td char=".">1 </td><td char=".">21.8 </td><td char=".">23 </td><td char=".">37.5 </td><td>Lister B025 (5) </td></tr><tr><td rowspan="8">Probiotics </td><td>PC1.1 </td><td char=".">1 </td><td char=".">21.6 </td><td char=".">27 </td><td char=".">36.8 </td><td>Lister LP 101 (9) </td></tr><tr><td rowspan="2">Symbioflor 1 </td><td char=".">1 </td><td char=".">41.6 </td><td char=".">54 </td><td char=".">35.83 </td><td>Entero phiEf11 (12). Temper phiNIH1 1 (12) </td></tr><tr><td char=".">2 </td><td char=".">45.2 </td><td char=".">66 </td><td char=".">34.63 </td><td>Entero phiEf11 (63) </td></tr><tr><td rowspan="2">L-X </td><td char=".">1 </td><td char=".">25.5 </td><td char=".">30 </td><td char=".">34.4 </td><td>Bacill BCJA1c (7) </td></tr><tr><td char=".">2 </td><td char=".">41.9 </td><td char=".">55 </td><td char=".">36.1 </td><td>Lactoc Tuc2009 (13). Entero phiEf11 (13). Lactoc TP901 1 (13) </td></tr><tr><td>T-110 </td><td char=".">1 </td><td char=".">41.9 </td><td char=".">56 </td><td char=".">36.16 </td><td>Lactoc Tuc2009 (13). Entero phiEf11 (13) </td></tr><tr><td rowspan="2">CFTRI 2200 </td><td char=".">1 </td><td char=".">50.1 </td><td char=".">71 </td><td char=".">37.1 </td><td>Entero phiFL3A (18) </td></tr><tr><td char=".">2 </td><td char=".">18.8 </td><td char=".">32 </td><td char=".">40.5 </td><td>Entero EFC 1 (3) </td></tr><tr><td rowspan="6">Dairy isolates and probiotics </td><td>L-3 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td>— </td></tr><tr><td rowspan="5">MB5259 </td><td char=".">1 </td><td char=".">30.8 </td><td char=".">22 </td><td char=".">33 </td><td>Lactob phiAT3 (2) </td></tr><tr><td char=".">2 </td><td char=".">43.9 </td><td char=".">68 </td><td char=".">34.4 </td><td>Entero phiEf11 (63) </td></tr><tr><td char=".">3 </td><td char=".">42.5 </td><td char=".">60 </td><td char=".">34.5 </td><td>Entero phiFL1A (15) </td></tr><tr><td char=".">4 </td><td char=".">35.3 </td><td char=".">55 </td><td char=".">35.2 </td><td>Entero phiFL3A (13) </td></tr><tr><td char=".">5 </td><td char=".">41.7 </td><td char=".">31 </td><td char=".">37 </td><td>Staphy StauST398 4 (2) </td></tr></tbody></table></div><div content-id="dsw043-T3" class="table-modal"><table><thead name="thead" align=""><tr><th>Group </th><th>Strain </th><th>Region<a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>Length<a reveal-id="tblfn5" data-open="tblfn5" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> (Kb) </th><th>Nº CDS </th><th>GC% </th><th>Most common phage (hit genes count)<a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th></tr></thead><tbody><tr><td rowspan="18">Dairy isolates </td><td rowspan="3">IPLA 655 </td><td char=".">1 </td><td char=".">40.7 </td><td char=".">55 </td><td char=".">35.4 </td><td>Lister 2389 (16) </td></tr><tr><td char=".">2 </td><td char=".">44.8 </td><td char=".">64 </td><td char=".">36.8 </td><td>Entero phiEf11 (13) </td></tr><tr><td char=".">3 </td><td char=".">16.5 </td><td char=".">27 </td><td char=".">35.1 </td><td>Bacill G (2) </td></tr><tr><td>INF E1 </td><td char=".">1 </td><td char=".">38.8 </td><td char=".">54 </td><td char=".">33.4 </td><td>Entero phiFL1A (10) </td></tr><tr><td>CRL 1879 </td><td char=".">1 </td><td char=".">52.5 </td><td char=".">61 </td><td char=".">37.5 </td><td>Entero EfaCPT1 (11) </td></tr><tr><td>E1604 </td><td char=".">1 </td><td char=".">37.6 </td><td char=".">51 </td><td char=".">36.8 </td><td>Entero IME EF4 (11) </td></tr><tr><td rowspan="2">NRRL B-2354 </td><td char=".">1 </td><td char=".">56.8 </td><td char=".">59 </td><td char=".">36.64 </td><td>Lactob phig1e (12) </td></tr><tr><td char=".">2 </td><td char=".">46 </td><td char=".">58 </td><td char=".">34.96 </td><td>Lactob phig1e (13) </td></tr><tr><td>UC7256 </td><td char=".">1 </td><td char=".">48.7 </td><td char=".">54 </td><td char=".">36.8 </td><td>Lactob phig1e (12) </td></tr><tr><td>UC 7267 </td><td char=".">1 </td><td char=".">31 </td><td char=".">36 </td><td char=".">38.5 </td><td>Bacill BCJA1c (10) </td></tr><tr><td rowspan="2">UC8668 </td><td char=".">1 </td><td char=".">25.8 </td><td char=".">31 </td><td char=".">36.8 </td><td>Lactob phig1e (8) </td></tr><tr><td char=".">2 </td><td char=".">47 </td><td char=".">59 </td><td char=".">36.1 </td><td>Lister 2389 (15) </td></tr><tr><td>ATCC 43197 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td>— </td></tr><tr><td>ATCC 882 </td><td char=".">1 </td><td char=".">46.5 </td><td char=".">66 </td><td char=".">36.7 </td><td>Bacill BCJA1c (10) </td></tr><tr><td>CRL1656 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td>— </td></tr><tr><td rowspan="3">CRL35 </td><td char=".">1 </td><td char=".">38.5 </td><td char=".">50 </td><td char=".">36.3 </td><td>Lactob PL 1 (13) </td></tr><tr><td char=".">2 </td><td char=".">44.4 </td><td char=".">68 </td><td char=".">37.3 </td><td>Lactoc Tuc2009 (13). Entero phiEf11 (13) </td></tr><tr><td char=".">3 </td><td char=".">44.1 </td><td char=".">53 </td><td char=".">37.6 </td><td>Bacill BCJA1c (9) </td></tr><tr><td rowspan="6">Meat isolates </td><td>2924 </td><td char=".">1 </td><td char=".">36.8 </td><td char=".">46 </td><td char=".">34.5 </td><td>Strept phi3396 (8). Lister LP 101 (8) </td></tr><tr><td>19116 </td><td char=".">1 </td><td char=".">68.8 </td><td char=".">72 </td><td char=".">36.2 </td><td>Entero EFC 1 (38) </td></tr><tr><td>E1613 </td><td char=".">1 </td><td char=".">37.1 </td><td char=".">50 </td><td char=".">35.4 </td><td>Lister 2389 (14) </td></tr><tr><td>UC7251 </td><td char=".">1 </td><td char=".">70.1 </td><td char=".">64 </td><td char=".">35.9 </td><td>Lister 2389 (16) </td></tr><tr><td>UC8733 </td><td char=".">1 </td><td char=".">31.9 </td><td char=".">35 </td><td char=".">37.9 </td><td>Entero IME EF4 (12) </td></tr><tr><td>UC10237 </td><td char=".">1 </td><td char=".">21.8 </td><td char=".">23 </td><td char=".">37.5 </td><td>Lister B025 (5) </td></tr><tr><td rowspan="8">Probiotics </td><td>PC1.1 </td><td char=".">1 </td><td char=".">21.6 </td><td char=".">27 </td><td char=".">36.8 </td><td>Lister LP 101 (9) </td></tr><tr><td rowspan="2">Symbioflor 1 </td><td char=".">1 </td><td char=".">41.6 </td><td char=".">54 </td><td char=".">35.83 </td><td>Entero phiEf11 (12). Temper phiNIH1 1 (12) </td></tr><tr><td char=".">2 </td><td char=".">45.2 </td><td char=".">66 </td><td char=".">34.63 </td><td>Entero phiEf11 (63) </td></tr><tr><td rowspan="2">L-X </td><td char=".">1 </td><td char=".">25.5 </td><td char=".">30 </td><td char=".">34.4 </td><td>Bacill BCJA1c (7) </td></tr><tr><td char=".">2 </td><td char=".">41.9 </td><td char=".">55 </td><td char=".">36.1 </td><td>Lactoc Tuc2009 (13). Entero phiEf11 (13). Lactoc TP901 1 (13) </td></tr><tr><td>T-110 </td><td char=".">1 </td><td char=".">41.9 </td><td char=".">56 </td><td char=".">36.16 </td><td>Lactoc Tuc2009 (13). Entero phiEf11 (13) </td></tr><tr><td rowspan="2">CFTRI 2200 </td><td char=".">1 </td><td char=".">50.1 </td><td char=".">71 </td><td char=".">37.1 </td><td>Entero phiFL3A (18) </td></tr><tr><td char=".">2 </td><td char=".">18.8 </td><td char=".">32 </td><td char=".">40.5 </td><td>Entero EFC 1 (3) </td></tr><tr><td rowspan="6">Dairy isolates and probiotics </td><td>L-3 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td>— </td></tr><tr><td rowspan="5">MB5259 </td><td char=".">1 </td><td char=".">30.8 </td><td char=".">22 </td><td char=".">33 </td><td>Lactob phiAT3 (2) </td></tr><tr><td char=".">2 </td><td char=".">43.9 </td><td char=".">68 </td><td char=".">34.4 </td><td>Entero phiEf11 (63) </td></tr><tr><td char=".">3 </td><td char=".">42.5 </td><td char=".">60 </td><td char=".">34.5 </td><td>Entero phiFL1A (15) </td></tr><tr><td char=".">4 </td><td char=".">35.3 </td><td char=".">55 </td><td char=".">35.2 </td><td>Entero phiFL3A (13) </td></tr><tr><td char=".">5 </td><td char=".">41.7 </td><td char=".">31 </td><td char=".">37 </td><td>Staphy StauST398 4 (2) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn4"><div class="fn"><div class="label -label">a</div><p>Intact prophage region.</p></div></div><div content-id="tblfn5"><div class="fn"><div class="label -label">b</div><p>Intact prophage region length.</p></div></div><div content-id="tblfn6"><div class="fn"><div class="label -label">c</div><p>Phage with the highest number of CDS in the region and the number og gene counts in brackets.</p></div></div></div></div></p><p>Genes related to resistance to macrolide-lincosamide-streptogramin (<em>msrC</em>) and aminoglycosides (<em>aac (6')-Ii)</em> were found in all <em>E. faecium</em> strains but not in <em>E. faecalis</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/dsw043_Supp.zip?Expires=2147483647&Signature=qfrsEYitdaJFRtVb4lVTNX16upeZv8S1FjY1XCqewM9sqQh0e4qpEC679hfVK0HFqPdKNcQnFO1Gwa-dzW4JwoTXPeTDnBceFgGx7NZ3YWjVXWKJOh4x54hNDn57A9V46VBYbxEnR6dnIUmjr4FdsbLmOTACDeDJedoAFYuVLWryR6FHza5a23P88etL6BkNLUglUF6UYULgr0cd2tRX-QIgzhsKu21VyNcv5B7O7LHKc6mhhmOTzKWiJxE8BTHIijLs6MDBlnmsBkB9czJxa~FAznSfC-NsCoetlHpJ9z~bQwtssQ09AAjbTcHT~9c-3Q3xe8MiIXzYFKcJybo1CA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). The <em>aac</em>(<em>6</em>′)-<em>Ii</em> gene, is species-specific and encodes for a chromosomal aminoglycoside acetyltransferase specific for <em>E. faecium</em> strains.<a reveal-id="dsw043-B58" data-open="dsw043-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> The 6′-N-aminoglycoside acetyltransferases [AAC (6′)s] are of particular interest because they can modify a number of clinically important aminoglycosides and some of them are often present in integrons, transposons, plasmids, genomic islands and other genetic structures. AAC (6′)-I-type enzymes effectively acetylate amikacin but not gentamicin. To date 45 such genes have been characterized.<a reveal-id="dsw043-B59" data-open="dsw043-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> On the other hand, <em>msrC encodes a putative efflux pump of the ABC transporter family and</em> it seems that the presence of this gene confers an advantage for the members of this species.<a reveal-id="dsw043-B58" data-open="dsw043-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> Similarly, the genes <em>lsaA</em> and <em>mphD</em> were found exclusively in the <em>E. faecalis</em> strains (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/dsw043_Supp.zip?Expires=2147483647&Signature=qfrsEYitdaJFRtVb4lVTNX16upeZv8S1FjY1XCqewM9sqQh0e4qpEC679hfVK0HFqPdKNcQnFO1Gwa-dzW4JwoTXPeTDnBceFgGx7NZ3YWjVXWKJOh4x54hNDn57A9V46VBYbxEnR6dnIUmjr4FdsbLmOTACDeDJedoAFYuVLWryR6FHza5a23P88etL6BkNLUglUF6UYULgr0cd2tRX-QIgzhsKu21VyNcv5B7O7LHKc6mhhmOTzKWiJxE8BTHIijLs6MDBlnmsBkB9czJxa~FAznSfC-NsCoetlHpJ9z~bQwtssQ09AAjbTcHT~9c-3Q3xe8MiIXzYFKcJybo1CA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). <em>Lsa</em> gene, encodes a putative ABC protein that confers resistance to quinupristin-dalfopristin and clindamycin<a reveal-id="dsw043-B60" data-open="dsw043-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> and is located in the chromosome. The gen <em>mphD</em> codes for a phosphorylase and belongs to the group of inactivating enzymes<a reveal-id="dsw043-B61" data-open="dsw043-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/dsw043_Supp.zip?Expires=2147483647&Signature=qfrsEYitdaJFRtVb4lVTNX16upeZv8S1FjY1XCqewM9sqQh0e4qpEC679hfVK0HFqPdKNcQnFO1Gwa-dzW4JwoTXPeTDnBceFgGx7NZ3YWjVXWKJOh4x54hNDn57A9V46VBYbxEnR6dnIUmjr4FdsbLmOTACDeDJedoAFYuVLWryR6FHza5a23P88etL6BkNLUglUF6UYULgr0cd2tRX-QIgzhsKu21VyNcv5B7O7LHKc6mhhmOTzKWiJxE8BTHIijLs6MDBlnmsBkB9czJxa~FAznSfC-NsCoetlHpJ9z~bQwtssQ09AAjbTcHT~9c-3Q3xe8MiIXzYFKcJybo1CA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). The meat isolate <em>E. faecium</em> UC7251, unlike the other strains of this species, additionally had genes for resistance to antibiotics of the tetracyclines class, apart from the MLS resistance genes <em>ermB</em> and <em>lnuB</em>, and aminoglycosides resistance genes <em>aph3-III</em> and <em>sat4A</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/dsw043_Supp.zip?Expires=2147483647&Signature=qfrsEYitdaJFRtVb4lVTNX16upeZv8S1FjY1XCqewM9sqQh0e4qpEC679hfVK0HFqPdKNcQnFO1Gwa-dzW4JwoTXPeTDnBceFgGx7NZ3YWjVXWKJOh4x54hNDn57A9V46VBYbxEnR6dnIUmjr4FdsbLmOTACDeDJedoAFYuVLWryR6FHza5a23P88etL6BkNLUglUF6UYULgr0cd2tRX-QIgzhsKu21VyNcv5B7O7LHKc6mhhmOTzKWiJxE8BTHIijLs6MDBlnmsBkB9czJxa~FAznSfC-NsCoetlHpJ9z~bQwtssQ09AAjbTcHT~9c-3Q3xe8MiIXzYFKcJybo1CA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>).</p>                            <h3 scrollto-destination=61462877 id="61462877" class="section-title" >6.3. Prophage sequences and CRISPRs—Cas systems</h3>
<p>Lysogenic phages are an important source of information related to bacterial pathogenesis.<a reveal-id="dsw043-B62" data-open="dsw043-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> Comparative genomic analysis evidenced that polylysogeny is a common phenomenon in the <em>Enterococcus</em> genus.<a reveal-id="dsw043-B62" data-open="dsw043-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> Particularly, in <em>E. faecalis</em> V583 temperate phages constitute the main source of horizontally acquired DNA representing nearly 10% the size of the genome.<a reveal-id="dsw043-B62" data-open="dsw043-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a></p><p>Within the scope of this study we search for prophage-like sequences in the genomes. In this sense, as a result of the analysis performed with PHAST, we evidenced in all strains but <em>E. mundtii</em> CRL1656, <em>E. malodoratus</em> ATCC 43197 and <em>E. faecium</em> L-3, several intact regions related to different prophages associated with the <em>Lactobacillus</em>, <em>Lactococcus</em>, <em>Listeria</em>, <em>Bacillus</em>, among other genera (<a reveal-id="dsw043-T3" data-open="dsw043-T3" class="link link-reveal link-table xref-fig">Table 3</a>). The temperate bacteriophages EFC-1 and PhiEf11, originally described in <em>E. faecalis</em>, had the highest number of CDS. PhiEf11 contained 63 from a total of 68 phage CDS in the probiotic strain MB5259, and EFC-1 38 CDS from a total of 72 in the meat isolate 19116. At the same time, the φEf11 phage was present in organisms belonging to four different species: <em>E. durans</em> IPLA 655, <em>E. faecalis</em> MB5259, <em>E. mundtii</em> CRL35, <em>E. faecalis</em> Symbioflor 1, <em>E. faecium</em> L-X and <em>E. faecium</em> T-110. Additionally, <em>E. faecalis</em> MB5259, a strain with probiotic properties and a dairy origin had a total of five intact phage related regions with CDS from different phages (<a reveal-id="dsw043-T3" data-open="dsw043-T3" class="link link-reveal link-table xref-fig">Table 3</a>).</p><p>It is interesting to note that all strains from meat origin harbored only one intact prophage region and that <em>E. faecium</em> UC7251 contained the largest intact phage-related region (70.1 kb) with the bacteriophage Lister 2389, isolated from <em>Listeria monocytogenes</em> strain Scott A,<a reveal-id="dsw043-B63" data-open="dsw043-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a> having 16 from a total of 64 CDS located in that region. In <em>E. faecalis</em> 2924 and PC1.1 another <em>Listeria phage</em>, LP-101, was located. Interestingly, this phage was first obtained from silage samples collected on a dairy farm.<a reveal-id="dsw043-B64" data-open="dsw043-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a> Then <em>Listeria</em> phage B025 related coding sequences were localized on the intact region of <em>E. faecium</em> UC10237 (<a reveal-id="dsw043-T3" data-open="dsw043-T3" class="link link-reveal link-table xref-fig">Table 3</a>). <em>Bacillus</em> phage BCJA1c had the highest protein counts in UC7267, ATCC-882, CRL35 and L-X, but none in those meat related strains and <em>Lactococcus</em> phages related proteins were the most abundant in <em>E. mundtii</em> CRL35 and <em>E. faecium</em> L-X intact phage regions (<a reveal-id="dsw043-T3" data-open="dsw043-T3" class="link link-reveal link-table xref-fig">Table 3</a>).</p><p>Related to the acquisition of phages and other mobile elements, enterococci that possess CRISPR-Cas systems, are less likely to acquire phages and other mobile elements.<a reveal-id="dsw043-B65" data-open="dsw043-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a> We found that <em>E. raffinosus</em> CFTRI 2200, with four individual gene cassettes, was the strain with the highest number of these prokaryotic immune systems among all under evaluation (<a reveal-id="dsw043-T4" data-open="dsw043-T4" class="link link-reveal link-table xref-fig">Table 4</a>). At the same time, one of these four CRISPRs, with a size of 1215 bp, was the largest identified. <em>E. faecium</em> UC7256 and <em>E. hirae</em> INF E1, both dairy isolates, had two confirmed CRISPRS systems each. Regarding meat isolates, <em>E. faecalis</em> 2924, <em>E. faecium</em> E1613 and <em>E. faecium</em> UC7251, had confirmed clusters with five, three and three spacers, respectively. All members of the probiotics group but <em>E. faecium</em> T-110 had confirmed CRISPR systems (<a reveal-id="dsw043-T4" data-open="dsw043-T4" class="link link-reveal link-table xref-fig">Table 4</a>). Among the confirmed CRISPR systems of all organisms DR sizes ranged from 23 to 37 bp and the number of spacers from 3 to 18. <div content-id="dsw043-T4" class="table-modal"><div class="table-wrap"><div id="dsw043-T4" data-id="dsw043-T4" class="table-wrap-title"><div class="label -label">Table 4.</div><div class="caption"><p>Confirmed CRISPR structures in the enterococcal genomes predicted with CRISPRFinder</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Group </th><th>Strain </th><th>Confirmed CRISPR </th><th>Length<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>DR length<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th>Nº of spacers </th></tr></thead><tbody><tr><td rowspan="14">Dairy isolates </td><td>IPLA 655 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td rowspan="2">INF E1 </td><td char=".">1 </td><td char=".">300 </td><td char=".">37 </td><td char=".">4 </td></tr><tr><td char=".">2 </td><td char=".">683 </td><td char=".">37 </td><td char=".">9 </td></tr><tr><td>CRL 1879 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td>E1604 </td><td char=".">1 </td><td char=".">199 </td><td char=".">24 </td><td char=".">3 </td></tr><tr><td>NRRL B-2354 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td rowspan="2">UC7256 </td><td char=".">1 </td><td char=".">199 </td><td char=".">24 </td><td char=".">3 </td></tr><tr><td char=".">2 </td><td char=".">201 </td><td char=".">28 </td><td char=".">3 </td></tr><tr><td>UC7267 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td>UC8668 </td><td char=".">1 </td><td char=".">201 </td><td char=".">28 </td><td char=".">3 </td></tr><tr><td>ATCC 43197 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td>ATCC 882 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td>CRL1656 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td>CRL35 </td><td char=".">1 </td><td char=".">227 </td><td char=".">28 </td><td char=".">3 </td></tr><tr><td rowspan="6">Meat isolates </td><td>2924 </td><td char=".">1 </td><td char=".">365 </td><td char=".">36 </td><td char=".">5 </td></tr><tr><td>19116 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td>E1613 </td><td char=".">1 </td><td char=".">199 </td><td char=".">24 </td><td char=".">3 </td></tr><tr><td>UC7251 </td><td char=".">1 </td><td char=".">199 </td><td char=".">24 </td><td char=".">3 </td></tr><tr><td>UC8733 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td>UC10237 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td rowspan="8">Probiotics </td><td>PC1.1 </td><td char=".">1 </td><td char=".">365 </td><td char=".">36 </td><td char=".">5 </td></tr><tr><td>Symbioflor 1 </td><td char=".">1 </td><td char=".">629 </td><td char=".">36 </td><td char=".">9 </td></tr><tr><td>L-X </td><td char=".">1 </td><td char=".">199 </td><td char=".">24 </td><td char=".">3 </td></tr><tr><td>T-110 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td rowspan="4">CFTRI 2200 </td><td char=".">1 </td><td char=".">484 </td><td char=".">23 </td><td char=".">6 </td></tr><tr><td char=".">2 </td><td char=".">221 </td><td char=".">27 </td><td char=".">3 </td></tr><tr><td char=".">3 </td><td char=".">1215 </td><td char=".">30 </td><td char=".">18 </td></tr><tr><td char=".">4 </td><td char=".">537 </td><td char=".">24 </td><td char=".">8 </td></tr><tr><td rowspan="2">Dairy isolates and probiotics </td><td>L-3 </td><td char=".">1 </td><td char=".">199 </td><td char=".">24 </td><td char=".">3 </td></tr><tr><td>MB5259 </td><td char=".">1 </td><td char=".">431 </td><td char=".">36 </td><td char=".">6 </td></tr></tbody></table></div><div content-id="dsw043-T4" class="table-modal"><table><thead name="thead" align=""><tr><th>Group </th><th>Strain </th><th>Confirmed CRISPR </th><th>Length<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>DR length<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th>Nº of spacers </th></tr></thead><tbody><tr><td rowspan="14">Dairy isolates </td><td>IPLA 655 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td rowspan="2">INF E1 </td><td char=".">1 </td><td char=".">300 </td><td char=".">37 </td><td char=".">4 </td></tr><tr><td char=".">2 </td><td char=".">683 </td><td char=".">37 </td><td char=".">9 </td></tr><tr><td>CRL 1879 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td>E1604 </td><td char=".">1 </td><td char=".">199 </td><td char=".">24 </td><td char=".">3 </td></tr><tr><td>NRRL B-2354 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td rowspan="2">UC7256 </td><td char=".">1 </td><td char=".">199 </td><td char=".">24 </td><td char=".">3 </td></tr><tr><td char=".">2 </td><td char=".">201 </td><td char=".">28 </td><td char=".">3 </td></tr><tr><td>UC7267 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td>UC8668 </td><td char=".">1 </td><td char=".">201 </td><td char=".">28 </td><td char=".">3 </td></tr><tr><td>ATCC 43197 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td>ATCC 882 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td>CRL1656 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td>CRL35 </td><td char=".">1 </td><td char=".">227 </td><td char=".">28 </td><td char=".">3 </td></tr><tr><td rowspan="6">Meat isolates </td><td>2924 </td><td char=".">1 </td><td char=".">365 </td><td char=".">36 </td><td char=".">5 </td></tr><tr><td>19116 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td>E1613 </td><td char=".">1 </td><td char=".">199 </td><td char=".">24 </td><td char=".">3 </td></tr><tr><td>UC7251 </td><td char=".">1 </td><td char=".">199 </td><td char=".">24 </td><td char=".">3 </td></tr><tr><td>UC8733 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td>UC10237 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td rowspan="8">Probiotics </td><td>PC1.1 </td><td char=".">1 </td><td char=".">365 </td><td char=".">36 </td><td char=".">5 </td></tr><tr><td>Symbioflor 1 </td><td char=".">1 </td><td char=".">629 </td><td char=".">36 </td><td char=".">9 </td></tr><tr><td>L-X </td><td char=".">1 </td><td char=".">199 </td><td char=".">24 </td><td char=".">3 </td></tr><tr><td>T-110 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td rowspan="4">CFTRI 2200 </td><td char=".">1 </td><td char=".">484 </td><td char=".">23 </td><td char=".">6 </td></tr><tr><td char=".">2 </td><td char=".">221 </td><td char=".">27 </td><td char=".">3 </td></tr><tr><td char=".">3 </td><td char=".">1215 </td><td char=".">30 </td><td char=".">18 </td></tr><tr><td char=".">4 </td><td char=".">537 </td><td char=".">24 </td><td char=".">8 </td></tr><tr><td rowspan="2">Dairy isolates and probiotics </td><td>L-3 </td><td char=".">1 </td><td char=".">199 </td><td char=".">24 </td><td char=".">3 </td></tr><tr><td>MB5259 </td><td char=".">1 </td><td char=".">431 </td><td char=".">36 </td><td char=".">6 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn7"><div class="fn"><div class="label -label">a</div><p>Total length of the CRISPR cassette.</p></div></div><div content-id="tblfn8"><div class="fn"><div class="label -label">b</div><p>Length of the repeated sequences.</p></div></div></div></div></div><div class="table-wrap"><div id="dsw043-T4" data-id="dsw043-T4" class="table-wrap-title"><div class="label -label">Table 4.</div><div class="caption"><p>Confirmed CRISPR structures in the enterococcal genomes predicted with CRISPRFinder</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Group </th><th>Strain </th><th>Confirmed CRISPR </th><th>Length<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>DR length<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th>Nº of spacers </th></tr></thead><tbody><tr><td rowspan="14">Dairy isolates </td><td>IPLA 655 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td rowspan="2">INF E1 </td><td char=".">1 </td><td char=".">300 </td><td char=".">37 </td><td char=".">4 </td></tr><tr><td char=".">2 </td><td char=".">683 </td><td char=".">37 </td><td char=".">9 </td></tr><tr><td>CRL 1879 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td>E1604 </td><td char=".">1 </td><td char=".">199 </td><td char=".">24 </td><td char=".">3 </td></tr><tr><td>NRRL B-2354 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td rowspan="2">UC7256 </td><td char=".">1 </td><td char=".">199 </td><td char=".">24 </td><td char=".">3 </td></tr><tr><td char=".">2 </td><td char=".">201 </td><td char=".">28 </td><td char=".">3 </td></tr><tr><td>UC7267 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td>UC8668 </td><td char=".">1 </td><td char=".">201 </td><td char=".">28 </td><td char=".">3 </td></tr><tr><td>ATCC 43197 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td>ATCC 882 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td>CRL1656 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td>CRL35 </td><td char=".">1 </td><td char=".">227 </td><td char=".">28 </td><td char=".">3 </td></tr><tr><td rowspan="6">Meat isolates </td><td>2924 </td><td char=".">1 </td><td char=".">365 </td><td char=".">36 </td><td char=".">5 </td></tr><tr><td>19116 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td>E1613 </td><td char=".">1 </td><td char=".">199 </td><td char=".">24 </td><td char=".">3 </td></tr><tr><td>UC7251 </td><td char=".">1 </td><td char=".">199 </td><td char=".">24 </td><td char=".">3 </td></tr><tr><td>UC8733 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td>UC10237 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td rowspan="8">Probiotics </td><td>PC1.1 </td><td char=".">1 </td><td char=".">365 </td><td char=".">36 </td><td char=".">5 </td></tr><tr><td>Symbioflor 1 </td><td char=".">1 </td><td char=".">629 </td><td char=".">36 </td><td char=".">9 </td></tr><tr><td>L-X </td><td char=".">1 </td><td char=".">199 </td><td char=".">24 </td><td char=".">3 </td></tr><tr><td>T-110 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td rowspan="4">CFTRI 2200 </td><td char=".">1 </td><td char=".">484 </td><td char=".">23 </td><td char=".">6 </td></tr><tr><td char=".">2 </td><td char=".">221 </td><td char=".">27 </td><td char=".">3 </td></tr><tr><td char=".">3 </td><td char=".">1215 </td><td char=".">30 </td><td char=".">18 </td></tr><tr><td char=".">4 </td><td char=".">537 </td><td char=".">24 </td><td char=".">8 </td></tr><tr><td rowspan="2">Dairy isolates and probiotics </td><td>L-3 </td><td char=".">1 </td><td char=".">199 </td><td char=".">24 </td><td char=".">3 </td></tr><tr><td>MB5259 </td><td char=".">1 </td><td char=".">431 </td><td char=".">36 </td><td char=".">6 </td></tr></tbody></table></div><div content-id="dsw043-T4" class="table-modal"><table><thead name="thead" align=""><tr><th>Group </th><th>Strain </th><th>Confirmed CRISPR </th><th>Length<a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>DR length<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th>Nº of spacers </th></tr></thead><tbody><tr><td rowspan="14">Dairy isolates </td><td>IPLA 655 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td rowspan="2">INF E1 </td><td char=".">1 </td><td char=".">300 </td><td char=".">37 </td><td char=".">4 </td></tr><tr><td char=".">2 </td><td char=".">683 </td><td char=".">37 </td><td char=".">9 </td></tr><tr><td>CRL 1879 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td>E1604 </td><td char=".">1 </td><td char=".">199 </td><td char=".">24 </td><td char=".">3 </td></tr><tr><td>NRRL B-2354 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td rowspan="2">UC7256 </td><td char=".">1 </td><td char=".">199 </td><td char=".">24 </td><td char=".">3 </td></tr><tr><td char=".">2 </td><td char=".">201 </td><td char=".">28 </td><td char=".">3 </td></tr><tr><td>UC7267 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td>UC8668 </td><td char=".">1 </td><td char=".">201 </td><td char=".">28 </td><td char=".">3 </td></tr><tr><td>ATCC 43197 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td>ATCC 882 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td>CRL1656 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td>CRL35 </td><td char=".">1 </td><td char=".">227 </td><td char=".">28 </td><td char=".">3 </td></tr><tr><td rowspan="6">Meat isolates </td><td>2924 </td><td char=".">1 </td><td char=".">365 </td><td char=".">36 </td><td char=".">5 </td></tr><tr><td>19116 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td>E1613 </td><td char=".">1 </td><td char=".">199 </td><td char=".">24 </td><td char=".">3 </td></tr><tr><td>UC7251 </td><td char=".">1 </td><td char=".">199 </td><td char=".">24 </td><td char=".">3 </td></tr><tr><td>UC8733 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td>UC10237 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td rowspan="8">Probiotics </td><td>PC1.1 </td><td char=".">1 </td><td char=".">365 </td><td char=".">36 </td><td char=".">5 </td></tr><tr><td>Symbioflor 1 </td><td char=".">1 </td><td char=".">629 </td><td char=".">36 </td><td char=".">9 </td></tr><tr><td>L-X </td><td char=".">1 </td><td char=".">199 </td><td char=".">24 </td><td char=".">3 </td></tr><tr><td>T-110 </td><td char=".">— </td><td char=".">— </td><td char=".">— </td><td char=".">— </td></tr><tr><td rowspan="4">CFTRI 2200 </td><td char=".">1 </td><td char=".">484 </td><td char=".">23 </td><td char=".">6 </td></tr><tr><td char=".">2 </td><td char=".">221 </td><td char=".">27 </td><td char=".">3 </td></tr><tr><td char=".">3 </td><td char=".">1215 </td><td char=".">30 </td><td char=".">18 </td></tr><tr><td char=".">4 </td><td char=".">537 </td><td char=".">24 </td><td char=".">8 </td></tr><tr><td rowspan="2">Dairy isolates and probiotics </td><td>L-3 </td><td char=".">1 </td><td char=".">199 </td><td char=".">24 </td><td char=".">3 </td></tr><tr><td>MB5259 </td><td char=".">1 </td><td char=".">431 </td><td char=".">36 </td><td char=".">6 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn7"><div class="fn"><div class="label -label">a</div><p>Total length of the CRISPR cassette.</p></div></div><div content-id="tblfn8"><div class="fn"><div class="label -label">b</div><p>Length of the repeated sequences.</p></div></div></div></div></p><p>Certain enterococcal lineages have recently emerged as one of the main causes of hospital infection outbreaks around the world. These are characterized by abundant mobile DNA, including phages, plasmids and transposons that carries multiple antibiotic resistances. Palmer <em>et al.</em> found that antibiotic resistance and possession of complete CRISPR loci are inversely related and that members of recently emerged high-risk enterococcal lineages lack complete CRISPR loci suggesting that antibiotic therapy inadvertently selects for enterococci with compromised genome defense.<a reveal-id="dsw043-B65" data-open="dsw043-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a> However, we cannot confirm these findings with our results regarding food-related strains. Especially for dairy isolates we frequently found no confirmed CRISPR system but also no antibiotic resistance genes (IPLA 655, ATCC 43197, ATCC 882, CRL1656). <em>Vice versa</em> the probiotic isolate CFTRI 2200 was heavily armed both with four confirmed CRISPR systems and at least four antibiotic resistance genes (see <a reveal-id="dsw043-T2" data-open="dsw043-T2" class="link link-reveal link-table xref-fig">Tables 2</a> and <a reveal-id="dsw043-T4" data-open="dsw043-T4" class="link link-reveal link-table xref-fig">4</a>), observations that are opposite to the findings described for the strains of clinical relevance.</p><p>Interestingly, the analysis of the gene context of all confirmed CRISPR systems identified by CRISPRFinder evidenced the existence of CRISPR associated genes (<em>cas</em>) upstream the CRISPR cassette only in INF E1 (confirmed CRISPR #2) and CFTRI 2200 (confirmed CRISPR #3). The former has genes exclusive of a Type II CRISPR-Cas system subtype II-A, while the latter related to a Type I CRISPR-Cas system subtype I-B.<a reveal-id="dsw043-B66" data-open="dsw043-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a><sup>,</sup><a reveal-id="dsw043-B67" data-open="dsw043-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> These results suggest that the systems detected in the other strains might not be functional or their specific <em>cas</em> related genes are located further than 2000bp upstream or downstream the CRISPR cassette.</p><p>On the other hand, No confirmed CRISPR system was detected in CRL1879, NRRL B-2354, UC7267, 19116, UC8733 and UC10237 strains.</p>                            <h3 scrollto-destination=61462885 id="61462885" class="section-title" >6.4. Genomic islands</h3>
<div class="block-child-p">Horizontal gene transfer (HGT) is one of the phenomena responsible of the diversification and adaptation of microorganisms, and has a direct impact on the genome plasticity and size; it is believed that the majority of the HGT has been facilitated by genomic islands (GEIs).<a reveal-id="dsw043-B68" data-open="dsw043-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a> In this respect, a detailed analysis of the enterococcal genomes allowed us to identify many GEIs and to correlate them according to the similarity of the encoded CDS (<a reveal-id="dsw043-F4" data-open="dsw043-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). From the GEI detected, all groups found in at least five different strains were further evaluated. With the exception of the GEI number 5, all four remaining islands were <em>E. faecium</em>-related to the major extend. A correlation with respect to the isolation origin cannot be observed nor can an exclusive occurrence in strains of a certain species (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/dsw043_Supp.zip?Expires=2147483647&amp;Signature=qfrsEYitdaJFRtVb4lVTNX16upeZv8S1FjY1XCqewM9sqQh0e4qpEC679hfVK0HFqPdKNcQnFO1Gwa-dzW4JwoTXPeTDnBceFgGx7NZ3YWjVXWKJOh4x54hNDn57A9V46VBYbxEnR6dnIUmjr4FdsbLmOTACDeDJedoAFYuVLWryR6FHza5a23P88etL6BkNLUglUF6UYULgr0cd2tRX-QIgzhsKu21VyNcv5B7O7LHKc6mhhmOTzKWiJxE8BTHIijLs6MDBlnmsBkB9czJxa~FAznSfC-NsCoetlHpJ9z~bQwtssQ09AAjbTcHT~9c-3Q3xe8MiIXzYFKcJybo1CA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). <div data-id="dsw043-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/m_dsw043f4.png?Expires=2147483647&amp;Signature=02uBWTT3XyaZucNRkQOAMFPS8d-SZWLzWQYaYtSq0eeAeNjxlEl2NZw9nBt4mPJ03FxgPaATyhNVvPX0qG3mPS51EilBEM~9mV-LgLg8IxJ-HScVUT0OmJwUHm~DWpIm6aC5J4WzBqcwolBiCSpsKybPpi-qRWiPbDNGNSQcBd-fXvZY6jgGzX9aOYbTLidjkAWP9Mej2Fvg8SF-8HH2zU9ACCSqQ2js5UXUCvECbVjK2QgMtxGzwiHITPagB35nGhZjrvCkdMpOBITFInNLhGKKuUsI9i0Onl7wadbJgUm8hZo61Cjdm8uWbEvOPZrAA3Lkl8xirZYmMlZwoNn6kg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Heat map showing the percentage of shared proteins between the genomic islands identified with IslandViewer3. All islands with at least 50% shared proteins to any other island were considered. Dark colour indicates nearly perfect overlaps while white means no detectable relation at all. " path-from-xml="dsw043f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61462886/dsw043f4.tif" path-from-xml="dsw043f4.tif" target="_blank">View large</a><a section="61462886" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/dsw043f4.png?Expires=2147483647&amp;Signature=W9lWpjOuzqCeHjwtBhaFXFm1bufFs3ZqcEoKT-eY5X71db5uR6K-XuXtW~0DJwMwejArsbh6UVODPtKg0EAV42b4KaZVDIF7f3h90AOnwzazRaQZbHzseVGi2FECly7OnsMCbTNnUQxdbEixicFhupnScsF1gDhgSkKY7rTQDvQJZSLRmhxYInmPWRVezL-e9uvRpdZr5gFBSgCK1icUkMs706qVccswr4P~KQfUuk8D0hBiBe2Wz4XbrVJIuknkScKlvNWoZIc4Upz46tViWWLyYSSh7YdXC-1BlX8iqCJYr6fGRxk1bDOodZep3pnd4iRzihm5xw3LHsrNRysS~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=61462886&amp;ar=2670108&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw043f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Heat map showing the percentage of shared proteins between the genomic islands identified with IslandViewer3. All islands with at least 50% shared proteins to any other island were considered. Dark colour indicates nearly perfect overlaps while white means no detectable relation at all. </p></div></div><div content-id="dsw043-F4" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/m_dsw043f4.png?Expires=2147483647&amp;Signature=02uBWTT3XyaZucNRkQOAMFPS8d-SZWLzWQYaYtSq0eeAeNjxlEl2NZw9nBt4mPJ03FxgPaATyhNVvPX0qG3mPS51EilBEM~9mV-LgLg8IxJ-HScVUT0OmJwUHm~DWpIm6aC5J4WzBqcwolBiCSpsKybPpi-qRWiPbDNGNSQcBd-fXvZY6jgGzX9aOYbTLidjkAWP9Mej2Fvg8SF-8HH2zU9ACCSqQ2js5UXUCvECbVjK2QgMtxGzwiHITPagB35nGhZjrvCkdMpOBITFInNLhGKKuUsI9i0Onl7wadbJgUm8hZo61Cjdm8uWbEvOPZrAA3Lkl8xirZYmMlZwoNn6kg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Heat map showing the percentage of shared proteins between the genomic islands identified with IslandViewer3. All islands with at least 50% shared proteins to any other island were considered. Dark colour indicates nearly perfect overlaps while white means no detectable relation at all. " path-from-xml="dsw043f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/61462886/dsw043f4.tif" path-from-xml="dsw043f4.tif" target="_blank">View large</a><a section="61462886" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/dsw043f4.png?Expires=2147483647&amp;Signature=W9lWpjOuzqCeHjwtBhaFXFm1bufFs3ZqcEoKT-eY5X71db5uR6K-XuXtW~0DJwMwejArsbh6UVODPtKg0EAV42b4KaZVDIF7f3h90AOnwzazRaQZbHzseVGi2FECly7OnsMCbTNnUQxdbEixicFhupnScsF1gDhgSkKY7rTQDvQJZSLRmhxYInmPWRVezL-e9uvRpdZr5gFBSgCK1icUkMs706qVccswr4P~KQfUuk8D0hBiBe2Wz4XbrVJIuknkScKlvNWoZIc4Upz46tViWWLyYSSh7YdXC-1BlX8iqCJYr6fGRxk1bDOodZep3pnd4iRzihm5xw3LHsrNRysS~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=61462886&amp;ar=2670108&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw043f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Heat map showing the percentage of shared proteins between the genomic islands identified with IslandViewer3. All islands with at least 50% shared proteins to any other island were considered. Dark colour indicates nearly perfect overlaps while white means no detectable relation at all. </p></div></div></div><p>GEIs play a crucial role in dissemination of variable genes, such us antibiotic resistance and virulence genes, and can contribute to the generation of the currently well-known named hospital ‘superbugs’. However, none of the five GEIs here evaluated contains CDS related to known pathogenic markers of clinical relevant enterococcal species. With 33 to 36 CDS GEI 1 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/dsw043_Supp.zip?Expires=2147483647&Signature=qfrsEYitdaJFRtVb4lVTNX16upeZv8S1FjY1XCqewM9sqQh0e4qpEC679hfVK0HFqPdKNcQnFO1Gwa-dzW4JwoTXPeTDnBceFgGx7NZ3YWjVXWKJOh4x54hNDn57A9V46VBYbxEnR6dnIUmjr4FdsbLmOTACDeDJedoAFYuVLWryR6FHza5a23P88etL6BkNLUglUF6UYULgr0cd2tRX-QIgzhsKu21VyNcv5B7O7LHKc6mhhmOTzKWiJxE8BTHIijLs6MDBlnmsBkB9czJxa~FAznSfC-NsCoetlHpJ9z~bQwtssQ09AAjbTcHT~9c-3Q3xe8MiIXzYFKcJybo1CA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4a</a></span>) was the biggest island containing mainly ribosomal proteins and translation factors along with accessory proteins for transcription and translation and protein secretion (e.g. RNA polymerase, Preprotein translocase secY).</p><p>In agreement with one of the ideas that considers GEIs an overarching family of elements, as is the case of mobile DNA elements constituted by integrative and conjugative elements (ICEs), conjugative transposons and some prophages, we found for instance integrases in GEI 2 (XerD, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/dsw043_Supp.zip?Expires=2147483647&Signature=qfrsEYitdaJFRtVb4lVTNX16upeZv8S1FjY1XCqewM9sqQh0e4qpEC679hfVK0HFqPdKNcQnFO1Gwa-dzW4JwoTXPeTDnBceFgGx7NZ3YWjVXWKJOh4x54hNDn57A9V46VBYbxEnR6dnIUmjr4FdsbLmOTACDeDJedoAFYuVLWryR6FHza5a23P88etL6BkNLUglUF6UYULgr0cd2tRX-QIgzhsKu21VyNcv5B7O7LHKc6mhhmOTzKWiJxE8BTHIijLs6MDBlnmsBkB9czJxa~FAznSfC-NsCoetlHpJ9z~bQwtssQ09AAjbTcHT~9c-3Q3xe8MiIXzYFKcJybo1CA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4b</a></span>) and GEI 5 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/dsw043_Supp.zip?Expires=2147483647&Signature=qfrsEYitdaJFRtVb4lVTNX16upeZv8S1FjY1XCqewM9sqQh0e4qpEC679hfVK0HFqPdKNcQnFO1Gwa-dzW4JwoTXPeTDnBceFgGx7NZ3YWjVXWKJOh4x54hNDn57A9V46VBYbxEnR6dnIUmjr4FdsbLmOTACDeDJedoAFYuVLWryR6FHza5a23P88etL6BkNLUglUF6UYULgr0cd2tRX-QIgzhsKu21VyNcv5B7O7LHKc6mhhmOTzKWiJxE8BTHIijLs6MDBlnmsBkB9czJxa~FAznSfC-NsCoetlHpJ9z~bQwtssQ09AAjbTcHT~9c-3Q3xe8MiIXzYFKcJybo1CA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4e</a></span>) and phage-related proteins in GEI 4 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/dsw043_Supp.zip?Expires=2147483647&Signature=qfrsEYitdaJFRtVb4lVTNX16upeZv8S1FjY1XCqewM9sqQh0e4qpEC679hfVK0HFqPdKNcQnFO1Gwa-dzW4JwoTXPeTDnBceFgGx7NZ3YWjVXWKJOh4x54hNDn57A9V46VBYbxEnR6dnIUmjr4FdsbLmOTACDeDJedoAFYuVLWryR6FHza5a23P88etL6BkNLUglUF6UYULgr0cd2tRX-QIgzhsKu21VyNcv5B7O7LHKc6mhhmOTzKWiJxE8BTHIijLs6MDBlnmsBkB9czJxa~FAznSfC-NsCoetlHpJ9z~bQwtssQ09AAjbTcHT~9c-3Q3xe8MiIXzYFKcJybo1CA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4d</a></span>). It has been previously described that the coding capacity of GEIs is not limited to pathogenicity functions and can be very diverse including such traits as symbiosis, sucrose and aromatic compound metabolism, mercury resistance and siderophore synthesis.<a reveal-id="dsw043-B69" data-open="dsw043-B69" class="link link-ref link-reveal xref-bibr"><sup>69</sup></a><sup>,</sup><a reveal-id="dsw043-B70" data-open="dsw043-B70" class="link link-ref link-reveal xref-bibr"><sup>70</sup></a> Thereof especially GEIs 2 and 3 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/dsw043_Supp.zip?Expires=2147483647&Signature=qfrsEYitdaJFRtVb4lVTNX16upeZv8S1FjY1XCqewM9sqQh0e4qpEC679hfVK0HFqPdKNcQnFO1Gwa-dzW4JwoTXPeTDnBceFgGx7NZ3YWjVXWKJOh4x54hNDn57A9V46VBYbxEnR6dnIUmjr4FdsbLmOTACDeDJedoAFYuVLWryR6FHza5a23P88etL6BkNLUglUF6UYULgr0cd2tRX-QIgzhsKu21VyNcv5B7O7LHKc6mhhmOTzKWiJxE8BTHIijLs6MDBlnmsBkB9czJxa~FAznSfC-NsCoetlHpJ9z~bQwtssQ09AAjbTcHT~9c-3Q3xe8MiIXzYFKcJybo1CA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4b and c</a></span>) encoded some interesting CDS. Among others, some examples were the ferric uptake regulation protein (oxidative stress), an enolase (pyruvate metabolism) and a triosephosphate isomerase (sugar metabolism). Finally, the third group of genes was that of hypothetical proteins with unknown function which was highly abundant in GEI 4 and 5 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/dsw043_Supp.zip?Expires=2147483647&Signature=qfrsEYitdaJFRtVb4lVTNX16upeZv8S1FjY1XCqewM9sqQh0e4qpEC679hfVK0HFqPdKNcQnFO1Gwa-dzW4JwoTXPeTDnBceFgGx7NZ3YWjVXWKJOh4x54hNDn57A9V46VBYbxEnR6dnIUmjr4FdsbLmOTACDeDJedoAFYuVLWryR6FHza5a23P88etL6BkNLUglUF6UYULgr0cd2tRX-QIgzhsKu21VyNcv5B7O7LHKc6mhhmOTzKWiJxE8BTHIijLs6MDBlnmsBkB9czJxa~FAznSfC-NsCoetlHpJ9z~bQwtssQ09AAjbTcHT~9c-3Q3xe8MiIXzYFKcJybo1CA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4d and e</a></span>) rendering any interpretation futile.</p>                            <h2 scrollto-destination=61462889 id="61462889" class="section-title" >7. Conclusion</h2>
<p>In this work we performed a detailed comparative analysis of food-related and probiotic enterococcal genomes. As previously demonstrated by our group, bacteriocin genes were widely distributed among <em>E. faecium</em> strains. Interestingly, antibiotic resistance genes were predominant in those organisms of meat origin coincidentally with the notion of the unrestricted use of antibiotics in husbandry. A search for regions enriched in phage related coding sequences evidenced that they were extensively propagated throughout the genus. In dairy isolates, phages derived from bacteria commonly associated to the milk environment, such us <em>Listeria, Lactobacillus</em> and <em>Lactococcus.</em> Interestingly, only two strains carried a complete CRISPR-Cas system (CFTRI 2200 and INF E1) but potential orphan cassettes were localized in the rest of the species. Importantly, five GEIs were found to be the same in different organisms and none of them contained CDS related with known pathogenic markers of clinical relevant enterococcus species.</p><p>Our in-depth phylogenetic reconstruction yielded that strains form clusters in accordance to their species but not to their isolation origin or probiotic properties. While we determined a set of 1617 FIGfam families as core proteome universally present among the genera, we were also able to identify several proteins that seem to occur in a strain-overarching but niche-specific pattern. Besides some phage-related regions/genes that seemingly occur in a habitat-dependent fashion, there was a specific membrane transport protein found exclusively in dairy isolates. Moreover, our data indicated that only dairy isolates could be found without any antibiotic resistance genes. All strains from the other groups code for at least one such gene. On the other hand, all dairy isolates were missing a gene coding for fumarate reductase generally present in meat isolates and probiotics. Enterococci isolated from meat origins lacked three phosphonate ABC transporters and two proteins related to galactoside metabolism that were otherwise found in most dairy isolates and probiotics. Moreover, arabinose-related proteins were missing from probiotics. Finally, we located a number of additional niche-specific but hypothetical proteins, a finding that might nevertheless aid in their future functional characterization. In principal, any of these specific genes could be used as indicator for the original habitat of a species making it possible to trace e.g. the origin of a strain that developed pathogenicity.</p><p>Based on the above described we can suggest that dairy isolates have greater potential than meat isolates for use as probiotics, at least at the genomic level, and their formal application in the future will require deep analysis including biochemical tests and <em>in vitro</em> as well as <em>in vivo</em> experiments.</p><p>Other than that, we also observed niche-specific features at species level. For instance, the antibiotic resistance genes <em>lsaA</em> and <em>mphD</em> were found exclusively in the <em>E. faecalis</em> strains that belonged to the probiotics and meat isolates groups but not in those isolated from dairy. Also, only <em>Enterococcus faecalis</em> strains from meat origin and probiotics had a set of genes for ethanolamine utilization.</p><p>Regarding the data, parallel evolution via horizontal gene transfer seems a likely cause for many specific genes reported here. We assume that most were acquired after speciation due to similar environments. This is certainly true for phage-related genes, likely being a result of the environment rather than a prerequisite. Also a number of specific genes were only present in less than half of all representatives of a niche and most were located mainly in <em>E. faecalis</em> and <em>E. faecium</em> strains specific to a certain niche (see <a reveal-id="dsw043-T2" data-open="dsw043-T2" class="link link-reveal link-table xref-fig">Table 2</a>). With respect to the phylogenetic tree (<a reveal-id="dsw043-F1" data-open="dsw043-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>) one would expect them to occur in some species in between as well if they were already present in a common ancestor. There are, however, notable exceptions. Specific genes present in five or even six of the seven enterococci species under this study that occur in a niche-specific manner were likely present in the common ancestor and probably got lost in strains that did not gain advantage of them anymore due to specialization, e.g. to a different niche or habitat. Here we especially refer to the phosphonate ABC transporters and the genes related to galactoside metabolism.</p><p>Altogether, our data demonstrated that while a few niche-specific genomic features of enterococci can be identified, relevant genomic idiosyncrasies mainly depend on the species in the first place rather than a specific niche or habitat.</p>                            <h2 scrollto-destination=61462896 id="61462896" class="backacknowledgements-title" >Acknowledgement</h2>
<p>This study was carried out with the financial support from CONICET (PIP0406/12) and MinCyT (PICT2011 N°0175) from Argentina.</p>                            <h3 scrollto-destination=61462898 id="61462898" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/dsw043_Supp.zip?Expires=2147483647&Signature=qfrsEYitdaJFRtVb4lVTNX16upeZv8S1FjY1XCqewM9sqQh0e4qpEC679hfVK0HFqPdKNcQnFO1Gwa-dzW4JwoTXPeTDnBceFgGx7NZ3YWjVXWKJOh4x54hNDn57A9V46VBYbxEnR6dnIUmjr4FdsbLmOTACDeDJedoAFYuVLWryR6FHza5a23P88etL6BkNLUglUF6UYULgr0cd2tRX-QIgzhsKu21VyNcv5B7O7LHKc6mhhmOTzKWiJxE8BTHIijLs6MDBlnmsBkB9czJxa~FAznSfC-NsCoetlHpJ9z~bQwtssQ09AAjbTcHT~9c-3Q3xe8MiIXzYFKcJybo1CA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h3 scrollto-destination=61462900 id="61462900" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h2 scrollto-destination=61462902 id="61462902" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw043-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Foulquie Moreno</div> <div class="given-names">M.R.</div></div><div class="name string-name"><div class="surname">Sarantinopoulos</div> <div class="given-names">P.</div></div><div class="name string-name"><div class="surname">Tsakalidou</div> <div class="given-names">E.</div></div><div class="name string-name"><div class="surname">De Vuyst</div> <div class="given-names">L.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">The role and application of enterococci in food and health</div>, <div class="source">Int. J. Food Microbiol</div> ., <div class="volume">106</div>, <div class="fpage">1</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20role%20and%20application%20of%20enterococci%20in%20food%20and%20health&amp;author=M.R.%20Foulquie%20Moreno&amp;author=P.%20Sarantinopoulos&amp;author=E.%20Tsakalidou&amp;author=L.%20De%20Vuyst&amp;publication_year=2006&amp;journal=Int.%20J.%20Food%20Microbiol&amp;volume=106&amp;pages=1-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ijfoodmicro.2005.06.026" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ijfoodmicro.2005.06.026" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ijfoodmicro.2005.06.026"> </span></div> </div></div></div></div></div><div content-id="dsw043-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Van Tyne</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Gilmore</div> <div class="given-names">M.S.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Friend turned foe: evolution of enterococcal virulence and antibiotic resistance</div>, <div class="source">Annu. Rev. Microbiol</div> ., <div class="volume">68</div>, <div class="fpage">337</div>–<div class="lpage">56</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Friend%20turned%20foe%3A%20evolution%20of%20enterococcal%20virulence%20and%20antibiotic%20resistance&amp;author=D.%20Van%20Tyne&amp;author=M.S.%20Gilmore&amp;publication_year=2014&amp;journal=Annu.%20Rev.%20Microbiol&amp;volume=68&amp;pages=337-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-micro-091213-113003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-micro-091213-113003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-micro-091213-113003"> </span></div> </div></div></div></div></div><div content-id="dsw043-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bhardwaj</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Malik</div> <div class="given-names">R.K.</div></div><div class="name string-name"><div class="surname">Chauhan</div> <div class="given-names">P.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Functional and safety aspects of enterococci in dairy foods</div>, <div class="source">Indian J. Microbiol</div> ., <div class="volume">48</div>, <div class="fpage">317</div>–<div class="lpage">325</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20and%20safety%20aspects%20of%20enterococci%20in%20dairy%20foods&amp;author=A.%20Bhardwaj&amp;author=R.K.%20Malik&amp;author=P.%20Chauhan&amp;publication_year=2008&amp;journal=Indian%20J.%20Microbiol&amp;volume=48&amp;pages=317-325" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s12088-008-0041-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs12088-008-0041-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs12088-008-0041-2"> </span></div> </div></div></div></div></div><div content-id="dsw043-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Karaseva</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Tsapieva</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Pachebat</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Suvorov</div> <div class="given-names">A.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Draft genome sequence of probiotic <em>Enterococcus faecium</em> strain L-3</div>, <div class="source">Genome Announ</div> ., <div class="volume">4</div>.</div></div></div></div><div content-id="dsw043-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ermolenko</div> <div class="given-names">E.</div></div><div class="name string-name"><div class="surname">Gromova</div> <div class="given-names">L.</div></div><div class="name string-name"><div class="surname">Borschev</div> <div class="given-names">Y.</div></div></div>, et al.  . <div class="year">2013</div>, 
<div class="article-title">Influence of different probiotic lactic Acid bacteria on microbiota and metabolism of rats with dysbiosis</div>, <div class="source">Biosci Microbiota Food Health</div> , <div class="volume">32</div>, <div class="fpage">41</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Influence%20of%20different%20probiotic%20lactic%20Acid%20bacteria%20on%20microbiota%20and%20metabolism%20of%20rats%20with%20dysbiosis&amp;author=E.%20Ermolenko&amp;author=L.%20Gromova&amp;author=Y.%20Borschev&amp;publication_year=2013&amp;journal=Biosci%20Microbiota%20Food%20Health&amp;volume=32&amp;pages=41-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.12938/bmfh.32.41" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.12938%2Fbmfh.32.41" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.12938%2Fbmfh.32.41"> </span></div> </div></div></div></div></div><div content-id="dsw043-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kristich</div> <div class="given-names">C.J.</div></div><div class="name string-name"><div class="surname">Rice</div> <div class="given-names">L.B.</div></div><div class="name string-name"><div class="surname">Arias</div> <div class="given-names">C.A.</div></div></div> <div class="year">2014</div>, Enterococcal Infection-Treatment and Antibiotic Resistance. In: <div class="person-group"><div class="name string-name"><div class="surname">M.S.</div> <div class="given-names">Gilmore</div></div><div class="name string-name"><div class="surname">D.B.</div> <div class="given-names">Clewell</div></div><div class="name string-name"><div class="surname">Y.</div> <div class="given-names">Ike</div></div><div class="name string-name"><div class="surname">N.</div> <div class="given-names">Shankar</div></div></div> (eds), <div class="source">Enterococci: From Commensals to Leading Causes of Drug Resistant Infection</div> , 
<div class="publisher-loc">Boston</div>.</div></div></div></div><div content-id="dsw043-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Blom</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Albaum</div> <div class="given-names">S.P.</div></div><div class="name string-name"><div class="surname">Doppmeier</div> <div class="given-names">D.</div></div></div>, et al.  . <div class="year">2009</div>, 
<div class="article-title">EDGAR: a software framework for the comparative analysis of prokaryotic genomes</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">10</div>, <div class="fpage">154</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EDGAR%3A%20a%20software%20framework%20for%20the%20comparative%20analysis%20of%20prokaryotic%20genomes&amp;author=J.%20Blom&amp;author=S.P.%20Albaum&amp;author=D.%20Doppmeier&amp;publication_year=2009&amp;journal=BMC%20Bioinformatics&amp;volume=10&amp;pages=154" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-10-154" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-10-154" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-10-154"> </span></div> </div></div></div></div></div><div content-id="dsw043-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tettelin</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Masignani</div> <div class="given-names">V.</div></div><div class="name string-name"><div class="surname">Cieslewicz</div> <div class="given-names">M.J.</div></div></div>, et al.  . <div class="year">2005</div>, 
<div class="article-title">Genome analysis of multiple pathogenic isolates of <em>Streptococcus agalactiae</em>: implications for the microbial “pan-genome”</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">102</div>, <div class="fpage">13950</div>–<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20analysis%20of%20multiple%20pathogenic%20isolates%20of%20Streptococcus%20agalactiae%3A%20implications%20for%20the%20microbial%20%E2%80%9Cpan-genome%E2%80%9D&amp;author=H.%20Tettelin&amp;author=V.%20Masignani&amp;author=M.J.%20Cieslewicz&amp;publication_year=2005&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=102&amp;pages=13950-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0506758102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0506758102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0506758102"> </span></div> </div></div></div></div></div><div content-id="dsw043-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kahlke</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Goesmann</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Hjerde</div> <div class="given-names">E.</div></div><div class="name string-name"><div class="surname">Willassen</div> <div class="given-names">N.P.</div></div><div class="name string-name"><div class="surname">Haugen</div> <div class="given-names">P.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Unique core genomes of the bacterial family Vibrionaceae: insights into niche adaptation and speciation</div>, <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">179</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unique%20core%20genomes%20of%20the%20bacterial%20family%20Vibrionaceae%3A%20insights%20into%20niche%20adaptation%20and%20speciation&amp;author=T.%20Kahlke&amp;author=A.%20Goesmann&amp;author=E.%20Hjerde&amp;author=N.P.%20Willassen&amp;author=P.%20Haugen&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=179" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-179" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-179" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-179"> </span></div> </div></div></div></div></div><div content-id="dsw043-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wu</div> <div class="given-names">X.</div></div><div class="name string-name"><div class="surname">Monchy</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Taghavi</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">W.</div></div><div class="name string-name"><div class="surname">Ramos</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">van der Lelie</div> <div class="given-names">D.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Comparative genomics and functional analysis of niche-specific adaptation in <em>Pseudomonas putida</em></div>, <div class="source">FEMS Microbiol. Rev</div> ., <div class="volume">35</div>, <div class="fpage">299</div>–<div class="lpage">323</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genomics%20and%20functional%20analysis%20of%20niche-specific%20adaptation%20in%20Pseudomonas%20putida&amp;author=X.%20Wu&amp;author=S.%20Monchy&amp;author=S.%20Taghavi&amp;author=W.%20Zhu&amp;author=J.%20Ramos&amp;author=D.%20van%20der%20Lelie&amp;publication_year=2011&amp;journal=FEMS%20Microbiol.%20Rev&amp;volume=35&amp;pages=299-323" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1574-6976.2010.00249.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1574-6976.2010.00249.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1574-6976.2010.00249.x"> </span></div> </div></div></div></div></div><div content-id="dsw043-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Douillard</div> <div class="given-names">F.P.</div></div><div class="name string-name"><div class="surname">de Vos</div> <div class="given-names">W.M.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Functional genomics of lactic acid bacteria: from food to health</div>, <div class="source">Microb. Cell Factories</div> , <div class="volume">13</div>, <div class="fpage">S8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20genomics%20of%20lactic%20acid%20bacteria%3A%20from%20food%20to%20health&amp;author=F.P.%20Douillard&amp;author=W.M.%20de%20Vos&amp;publication_year=2014&amp;journal=Microb.%20Cell%20Factories&amp;volume=13&amp;pages=S8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1475-2859-13-S1-S8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1475-2859-13-S1-S8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1475-2859-13-S1-S8"> </span></div> </div></div></div></div></div><div content-id="dsw043-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kim</div> <div class="given-names">E. B.</div></div><div class="name string-name"><div class="surname">Marco</div> <div class="given-names">M. L.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Nonclinical and clinical <em>Enterococcus faecium</em> strains, but not Enterococcus faecalis strains, have distinct structural and functional genomic features</div>, <div class="source">Appl. Environ. Microbiol</div> ., <div class="volume">80</div>, <div class="fpage">154</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nonclinical%20and%20clinical%20Enterococcus%20faecium%20strains%2C%20but%20not%20Enterococcus%20faecalis%20strains%2C%20have%20distinct%20structural%20and%20functional%20genomic%20features&amp;author=E.%20B.%20Kim&amp;author=M.%20L.%20Marco&amp;publication_year=2014&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=80&amp;pages=154-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.03108-13" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.03108-13" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.03108-13"> </span></div> </div></div></div></div></div><div content-id="dsw043-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Barbosa</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Borges</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Teixeira</div> <div class="given-names">P.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Selection of potential probiotic <em>Enterococcus faecium</em> isolated from Portuguese fermented food</div>, <div class="source">Int. J. Food Microbiol</div> ., <div class="volume">191</div>, <div class="fpage">144</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selection%20of%20potential%20probiotic%20Enterococcus%20faecium%20isolated%20from%20Portuguese%20fermented%20food&amp;author=J.%20Barbosa&amp;author=S.%20Borges&amp;author=P.%20Teixeira&amp;publication_year=2014&amp;journal=Int.%20J.%20Food%20Microbiol&amp;volume=191&amp;pages=144-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ijfoodmicro.2014.09.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ijfoodmicro.2014.09.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ijfoodmicro.2014.09.009"> </span></div> </div></div></div></div></div><div content-id="dsw043-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Haghshenas</div> <div class="given-names">B.</div></div><div class="name string-name"><div class="surname">Haghshenas</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Nami</div> <div class="given-names">Y.</div></div></div>, et al.  . <div class="year">2016</div>, 
<div class="article-title">Probiotic assessment of <em>Lactobacillus plantarum</em> 15HN and <em>Enterococcus mundtii</em> 50H isolated from traditional dairies microbiota</div>, <div class="source">Adv. Pharm. Bull</div> ., <div class="volume">6</div>, <div class="fpage">37</div>–<div class="lpage">47</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Probiotic%20assessment%20of%20Lactobacillus%20plantarum%2015HN%20and%20Enterococcus%20mundtii%2050H%20isolated%20from%20traditional%20dairies%20microbiota&amp;author=B.%20Haghshenas&amp;author=M.%20Haghshenas&amp;author=Y.%20Nami&amp;publication_year=2016&amp;journal=Adv.%20Pharm.%20Bull&amp;volume=6&amp;pages=37-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.15171/apb.2016.07" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.15171%2Fapb.2016.07" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.15171%2Fapb.2016.07"> </span></div> </div></div></div></div></div><div content-id="dsw043-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Aziz</div> <div class="given-names">R.K.</div></div><div class="name string-name"><div class="surname">Bartels</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Best</div> <div class="given-names">A.A.</div></div></div>, et al.  . <div class="year">2008</div>, 
<div class="article-title">The RAST Server: rapid annotations using subsystems technology</div>, <div class="source">BMC Genomics</div> , <div class="volume">9</div>, <div class="fpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20RAST%20Server%3A%20rapid%20annotations%20using%20subsystems%20technology&amp;author=R.K.%20Aziz&amp;author=D.%20Bartels&amp;author=A.A.%20Best&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-75" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-75" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-75"> </span></div> </div></div></div></div></div><div content-id="dsw043-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Darling</div> <div class="given-names">A.C.</div></div><div class="name string-name"><div class="surname">Mau</div> <div class="given-names">B.</div></div><div class="name string-name"><div class="surname">Blattner</div> <div class="given-names">F.R.</div></div><div class="name string-name"><div class="surname">Perna</div> <div class="given-names">N.T.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Mauve: multiple alignment of conserved genomic sequence with rearrangements</div>, <div class="source">Genome Res</div> ., <div class="volume">14</div>, <div class="fpage">1394</div>–<div class="lpage">403</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mauve%3A%20multiple%20alignment%20of%20conserved%20genomic%20sequence%20with%20rearrangements&amp;author=A.C.%20Darling&amp;author=B.%20Mau&amp;author=F.R.%20Blattner&amp;author=N.T.%20Perna&amp;publication_year=2004&amp;journal=Genome%20Res&amp;volume=14&amp;pages=1394-403" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.2289704" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.2289704" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.2289704"> </span></div> </div></div></div></div></div><div content-id="dsw043-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stamatakis</div> <div class="given-names">A.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies</div>, <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">1312</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RAxML%20version%208%3A%20a%20tool%20for%20phylogenetic%20analysis%20and%20post-analysis%20of%20large%20phylogenies&amp;author=A.%20Stamatakis&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=1312-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu033"> </span></div> </div></div></div></div></div><div content-id="dsw043-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Le</div> <div class="given-names">S. Q.</div></div><div class="name string-name"><div class="surname">Gascuel</div> <div class="given-names">O.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">An improved general amino acid replacement matrix</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">25</div>, <div class="fpage">1307</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20improved%20general%20amino%20acid%20replacement%20matrix&amp;author=S.%20Q.%20Le&amp;author=O.%20Gascuel&amp;publication_year=2008&amp;journal=Mol.%20Biol.%20Evol&amp;volume=25&amp;pages=1307-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msn067" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsn067" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsn067"> </span></div> </div></div></div></div></div><div content-id="dsw043-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lechner</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Findeiss</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Steiner</div> <div class="given-names">L.</div></div><div class="name string-name"><div class="surname">Marz</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Stadler</div> <div class="given-names">P.F.</div></div><div class="name string-name"><div class="surname">Prohaska</div> <div class="given-names">S.J.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Proteinortho: detection of (co-)orthologs in large-scale analysis</div>, <div class="source">BMC Bioinformat</div> ., <div class="volume">12</div>, <div class="fpage">124</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Proteinortho%3A%20detection%20of%20(co-)orthologs%20in%20large-scale%20analysis&amp;author=M.%20Lechner&amp;author=S.%20Findeiss&amp;author=L.%20Steiner&amp;author=M.%20Marz&amp;author=P.F.%20Stadler&amp;author=S.J.%20Prohaska&amp;publication_year=2011&amp;journal=BMC%20Bioinformat&amp;volume=12&amp;pages=124" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-12-124" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-12-124" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-12-124"> </span></div> </div></div></div></div></div><div content-id="dsw043-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sievers</div> <div class="given-names">F.</div></div><div class="name string-name"><div class="surname">Wilm</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Dineen</div> <div class="given-names">D.</div></div></div>, et al.  . <div class="year">2011</div>, 
<div class="article-title">Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega</div>, <div class="source">Mol. Syst. Biol</div> ., <div class="volume">7</div>, <div class="fpage">539</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%2C%20scalable%20generation%20of%20high-quality%20protein%20multiple%20sequence%20alignments%20using%20Clustal%20Omega&amp;author=F.%20Sievers&amp;author=A.%20Wilm&amp;author=D.%20Dineen&amp;publication_year=2011&amp;journal=Mol.%20Syst.%20Biol&amp;volume=7&amp;pages=539" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/msb.2011.75" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fmsb.2011.75" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fmsb.2011.75"> </span></div> </div></div></div></div></div><div content-id="dsw043-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">van Heel</div> <div class="given-names">A. J.</div></div><div class="name string-name"><div class="surname">de Jong</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Montalban-Lopez</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Kok</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Kuipers</div> <div class="given-names">O.P.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">BAGEL3: Automated identification of genes encoding bacteriocins and (non-)bactericidal posttranslationally modified peptides</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">41</div>, <div class="fpage">W448</div>–<div class="lpage">453</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BAGEL3%3A%20Automated%20identification%20of%20genes%20encoding%20bacteriocins%20and%20(non-)bactericidal%20posttranslationally%20modified%20peptides&amp;author=A.%20J.%20van%20Heel&amp;author=A.%20de%20Jong&amp;author=M.%20Montalban-Lopez&amp;author=J.%20Kok&amp;author=O.P.%20Kuipers&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;pages=W448-453" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt391" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt391" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt391"> </span></div> </div></div></div></div></div><div content-id="dsw043-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div><div class="name string-name"><div class="surname">Gish</div> <div class="given-names">W.</div></div><div class="name string-name"><div class="surname">Miller</div> <div class="given-names">W.</div></div><div class="name string-name"><div class="surname">Myers</div> <div class="given-names">E.W.</div></div><div class="name string-name"><div class="surname">Lipman</div> <div class="given-names">D. J.</div></div></div> <div class="year">1990</div>, 
<div class="article-title">Basic local alignment search tool</div>, <div class="source">J. Mol. Biol</div> ., <div class="volume">215</div>, <div class="fpage">403</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=S.F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.W.%20Myers&amp;author=D.%20J.%20Lipman&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol&amp;volume=215&amp;pages=403-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div> </div></div></div></div></div><div content-id="dsw043-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Aymerich</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Holo</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Havarstein</div> <div class="given-names">L.S.</div></div><div class="name string-name"><div class="surname">Hugas</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Garriga</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Nes</div> <div class="given-names">I. F.</div></div></div> <div class="year">1996</div>, 
<div class="article-title">Biochemical and genetic characterization of enterocin A from <em>Enterococcus faecium</em>, a new antilisterial bacteriocin in the pediocin family of bacteriocins</div>, <div class="source">Appl. Environ. Microbiol</div> ., <div class="volume">62</div>, <div class="fpage">1676</div>–<div class="lpage">82</div>.</div></div></div></div><div content-id="dsw043-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gupta</div> <div class="given-names">S. K.</div></div><div class="name string-name"><div class="surname">Padmanabhan</div> <div class="given-names">B. R.</div></div><div class="name string-name"><div class="surname">Diene</div> <div class="given-names">S. M.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">ARG-ANNOT, a new bioinformatic tool to discover antibiotic resistance genes in bacterial genomes</div>, <div class="source">Antimicrobial Agents Chemother</div> ., <div class="volume">58</div>, <div class="fpage">212</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ARG-ANNOT%2C%20a%20new%20bioinformatic%20tool%20to%20discover%20antibiotic%20resistance%20genes%20in%20bacterial%20genomes&amp;author=S.%20K.%20Gupta&amp;author=B.%20R.%20Padmanabhan&amp;author=S.%20M.%20Diene&amp;publication_year=2014&amp;journal=Antimicrobial%20Agents%20Chemother&amp;volume=58&amp;pages=212-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AAC.01310-13" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAAC.01310-13" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAAC.01310-13"> </span></div> </div></div></div></div></div><div content-id="dsw043-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hall</div> <div class="given-names">T.</div></div></div> <div class="year">2013</div>, <div class="collab">BioEdit</div>: biological sequence alignment editor for Win95/98/NT/2K/XP. pp. p 95–98</div></div></div></div><div content-id="dsw043-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Henikoff</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Henikoff</div> <div class="given-names">J.G.</div></div></div> <div class="year">1992</div>, 
<div class="article-title">Amino acid substitution matrices from protein blocks</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">89</div>, <div class="fpage">10915</div>–<div class="lpage">19</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Amino%20acid%20substitution%20matrices%20from%20protein%20blocks&amp;author=S.%20Henikoff&amp;author=J.G.%20Henikoff&amp;publication_year=1992&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=89&amp;pages=10915-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.89.22.10915" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.89.22.10915" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.89.22.10915"> </span></div> </div></div></div></div></div><div content-id="dsw043-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Liang</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Lynch</div> <div class="given-names">K.H.</div></div><div class="name string-name"><div class="surname">Dennis</div> <div class="given-names">J.J.</div></div><div class="name string-name"><div class="surname">Wishart</div> <div class="given-names">D.S.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">PHAST: a fast phage search tool</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">39</div>, <div class="fpage">W347</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PHAST%3A%20a%20fast%20phage%20search%20tool&amp;author=Y.%20Zhou&amp;author=Y.%20Liang&amp;author=K.H.%20Lynch&amp;author=J.J.%20Dennis&amp;author=D.S.%20Wishart&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res&amp;volume=39&amp;pages=W347-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr485" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr485" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr485"> </span></div> </div></div></div></div></div><div content-id="dsw043-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div><div class="name string-name"><div class="surname">Delcher</div> <div class="given-names">A.L.</div></div><div class="name string-name"><div class="surname">Kasif</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">White</div> <div class="given-names">O.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">Microbial gene identification using interpolated Markov models</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">26</div>, <div class="fpage">544</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microbial%20gene%20identification%20using%20interpolated%20Markov%20models&amp;author=S.L.%20Salzberg&amp;author=A.L.%20Delcher&amp;author=S.%20Kasif&amp;author=O.%20White&amp;publication_year=1998&amp;journal=Nucleic%20Acids%20Res&amp;volume=26&amp;pages=544-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/26.2.544" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F26.2.544" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F26.2.544"> </span></div> </div></div></div></div></div><div content-id="dsw043-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lowe</div> <div class="given-names">T. M.</div></div><div class="name string-name"><div class="surname">Eddy</div> <div class="given-names">S. R.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">25</div>, <div class="fpage">955</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNAscan-SE%3A%20a%20program%20for%20improved%20detection%20of%20transfer%20RNA%20genes%20in%20genomic%20sequence&amp;author=T.%20M.%20Lowe&amp;author=S.%20R.%20Eddy&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res&amp;volume=25&amp;pages=955-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.5.0955" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.5.0955" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.5.0955"> </span></div> </div></div></div></div></div><div content-id="dsw043-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Laslett</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Canback</div> <div class="given-names">B.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">ARAGORN, a program to detect tRNA genes and tmRNA genes in nucleotide sequences</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">32</div>, <div class="fpage">11</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ARAGORN%2C%20a%20program%20to%20detect%20tRNA%20genes%20and%20tmRNA%20genes%20in%20nucleotide%20sequences&amp;author=D.%20Laslett&amp;author=B.%20Canback&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res&amp;volume=32&amp;pages=11-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh152" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh152" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh152"> </span></div> </div></div></div></div></div><div content-id="dsw043-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Grissa</div> <div class="given-names">I.</div></div><div class="name string-name"><div class="surname">Vergnaud</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Pourcel</div> <div class="given-names">C.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">CRISPRFinder: a web tool to identify clustered regularly interspaced short palindromic repeats</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">35</div>, <div class="fpage">W52</div>–<div class="lpage">57</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CRISPRFinder%3A%20a%20web%20tool%20to%20identify%20clustered%20regularly%20interspaced%20short%20palindromic%20repeats&amp;author=I.%20Grissa&amp;author=G.%20Vergnaud&amp;author=C.%20Pourcel&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res&amp;volume=35&amp;pages=W52-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm360" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm360" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm360"> </span></div> </div></div></div></div></div><div content-id="dsw043-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dhillon</div> <div class="given-names">B.K.</div></div><div class="name string-name"><div class="surname">Laird</div> <div class="given-names">M.R.</div></div><div class="name string-name"><div class="surname">Shay</div> <div class="given-names">J.A.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">IslandViewer 3: more flexible, interactive genomic island discovery, visualization and analysis</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">43</div>, <div class="fpage">W104</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=IslandViewer%203%3A%20more%20flexible%2C%20interactive%20genomic%20island%20discovery%2C%20visualization%20and%20analysis&amp;author=B.K.%20Dhillon&amp;author=M.R.%20Laird&amp;author=J.A.%20Shay&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res&amp;volume=43&amp;pages=W104-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkv401" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkv401" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkv401"> </span></div> </div></div></div></div></div><div content-id="dsw043-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hsiao</div> <div class="given-names">W.</div></div><div class="name string-name"><div class="surname">Wan</div> <div class="given-names">I.</div></div><div class="name string-name"><div class="surname">Jones</div> <div class="given-names">S.J.</div></div><div class="name string-name"><div class="surname">Brinkman</div> <div class="given-names">F.S.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">IslandPath: aiding detection of genomic islands in prokaryotes</div>, <div class="source">Bioinformatics</div> , <div class="volume">19</div>, <div class="fpage">418</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=IslandPath%3A%20aiding%20detection%20of%20genomic%20islands%20in%20prokaryotes&amp;author=W.%20Hsiao&amp;author=I.%20Wan&amp;author=S.J.%20Jones&amp;author=F.S.%20Brinkman&amp;publication_year=2003&amp;journal=Bioinformatics&amp;volume=19&amp;pages=418-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btg004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtg004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtg004"> </span></div> </div></div></div></div></div><div content-id="dsw043-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Waack</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Keller</div> <div class="given-names">O.</div></div><div class="name string-name"><div class="surname">Asper</div> <div class="given-names">R.</div></div></div>, et al.  . <div class="year">2006</div>, 
<div class="article-title">Score-based prediction of genomic islands in prokaryotic genomes using hidden Markov models</div>, <div class="source">BMC Bioinform</div> ., <div class="volume">7</div>, <div class="fpage">142</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Score-based%20prediction%20of%20genomic%20islands%20in%20prokaryotic%20genomes%20using%20hidden%20Markov%20models&amp;author=S.%20Waack&amp;author=O.%20Keller&amp;author=R.%20Asper&amp;publication_year=2006&amp;journal=BMC%20Bioinform&amp;volume=7&amp;pages=142" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-7-142" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-7-142" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-7-142"> </span></div> </div></div></div></div></div><div content-id="dsw043-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Langille</div> <div class="given-names">M.G.</div></div><div class="name string-name"><div class="surname">Hsiao</div> <div class="given-names">W.W.</div></div><div class="name string-name"><div class="surname">Brinkman</div> <div class="given-names">F.S.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Evaluation of genomic island predictors using a comparative genomics approach</div>, <div class="source">BMC Bioinform</div> ., <div class="volume">9</div>, <div class="fpage">329</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evaluation%20of%20genomic%20island%20predictors%20using%20a%20comparative%20genomics%20approach&amp;author=M.G.%20Langille&amp;author=W.W.%20Hsiao&amp;author=F.S.%20Brinkman&amp;publication_year=2008&amp;journal=BMC%20Bioinform&amp;volume=9&amp;pages=329" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-9-329" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-9-329" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-9-329"> </span></div> </div></div></div></div></div><div content-id="dsw043-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dhillon</div> <div class="given-names">B.K.</div></div><div class="name string-name"><div class="surname">Chiu</div> <div class="given-names">T.A.</div></div><div class="name string-name"><div class="surname">Laird</div> <div class="given-names">M.R.</div></div><div class="name string-name"><div class="surname">Langille</div> <div class="given-names">M.G.</div></div><div class="name string-name"><div class="surname">Brinkman</div> <div class="given-names">F.S.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">IslandViewer update: Improved genomic island discovery and visualization</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">41</div>, <div class="fpage">W129</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=IslandViewer%20update%3A%20Improved%20genomic%20island%20discovery%20and%20visualization&amp;author=B.K.%20Dhillon&amp;author=T.A.%20Chiu&amp;author=M.R.%20Laird&amp;author=M.G.%20Langille&amp;author=F.S.%20Brinkman&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;pages=W129-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt394" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt394" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt394"> </span></div> </div></div></div></div></div><div content-id="dsw043-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Langille</div> <div class="given-names">M.G.</div></div><div class="name string-name"><div class="surname">Brinkman</div> <div class="given-names">F.S.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">IslandViewer: an integrated interface for computational identification and visualization of genomic islands</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">664</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=IslandViewer%3A%20an%20integrated%20interface%20for%20computational%20identification%20and%20visualization%20of%20genomic%20islands&amp;author=M.G.%20Langille&amp;author=F.S.%20Brinkman&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=664-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp030" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp030" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp030"> </span></div> </div></div></div></div></div><div content-id="dsw043-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zischka</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Kunne</div> <div class="given-names">C. T.</div></div><div class="name string-name"><div class="surname">Blom</div> <div class="given-names">J.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">Comprehensive molecular, genomic and phenotypic analysis of a major clone of <em>Enterococcus faecalis</em> MLST ST40</div>, <div class="source">BMC Genomics</div> , <div class="volume">16</div>, <div class="fpage">175</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20molecular%2C%20genomic%20and%20phenotypic%20analysis%20of%20a%20major%20clone%20of%20Enterococcus%20faecalis%20MLST%20ST40&amp;author=M.%20Zischka&amp;author=C.%20T.%20Kunne&amp;author=J.%20Blom&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=175" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-1367-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-1367-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-1367-x"> </span></div> </div></div></div></div></div><div content-id="dsw043-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ludwing</div> <div class="given-names">W.</div></div><div class="name string-name"><div class="surname">S</div> <div class="given-names">K.-H.</div></div><div class="name string-name"><div class="surname">Whitman</div> <div class="given-names">WB</div></div></div> <div class="year">2009</div>, Family IV. Enterococcaceae. <div class="source">Bergey's Manual of Systematic Bacteriology</div>  <div class="edition">2nd Ed.,</div> 
<div class="publisher-loc">Heidelberg</div>: 
<div class="publisher-name">Springer</div>, <div class="fpage">600</div>.</div></div></div></div><div content-id="dsw043-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Naser</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Thompson</div> <div class="given-names">F.L.</div></div><div class="name string-name"><div class="surname">Hoste</div> <div class="given-names">B.</div></div></div>, et al.  . <div class="year">2005</div>, 
<div class="article-title">Phylogeny and identification of Enterococci by <em>atp</em>A gene sequence analysis</div>. <div class="source">J. Clin. Microbiol</div> ., <div class="volume">43</div>, <div class="fpage">2224</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogeny%20and%20identification%20of%20Enterococci%20by%20atpA%20gene%20sequence%20analysis&amp;author=S.%20Naser&amp;author=F.L.%20Thompson&amp;author=B.%20Hoste&amp;publication_year=2005&amp;journal=J.%20Clin.%20Microbiol&amp;volume=43&amp;pages=2224-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JCM.43.5.2224-2230.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJCM.43.5.2224-2230.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJCM.43.5.2224-2230.2005"> </span></div> </div></div></div></div></div><div content-id="dsw043-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Palmer</div> <div class="given-names">K.L.</div></div><div class="name string-name"><div class="surname">Godfrey</div> <div class="given-names">P.</div></div><div class="name string-name"><div class="surname">Griggs</div> <div class="given-names">A.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Comparative genomics of enterococci: variation in <em>Enterococcus faecalis</em>, clade structure in <em>E. faecium</em>, and defining characteristics of <em>E. gallinarum</em> and <em>E. casseliflavus</em></div>. <div class="source">mBio</div> , <div class="volume">3</div>, <div class="fpage">e00318</div>–<div class="lpage">00311</div>.</div></div></div></div><div content-id="dsw043-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ramsey</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Hartke</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Huycke</div> <div class="given-names">M.</div></div></div> <div class="year">2014</div>, The Physiology and Metabolism of Enterococci.</div></div></div></div><div content-id="dsw043-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cai</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Thompson</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Budinich</div> <div class="given-names">M.F.</div></div><div class="name string-name"><div class="surname">Broadbent</div> <div class="given-names">J.R.</div></div><div class="name string-name"><div class="surname">Steele</div> <div class="given-names">J.L.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Genome sequence and comparative genome analysis of <em>Lactobacillus casei</em>: insights into their niche-associated evolution</div>, <div class="source">Genome Biol. Evol</div> ., <div class="volume">1</div>, <div class="fpage">239</div>–<div class="lpage">57</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20and%20comparative%20genome%20analysis%20of%20Lactobacillus%20casei%3A%20insights%20into%20their%20niche-associated%20evolution&amp;author=H.%20Cai&amp;author=R.%20Thompson&amp;author=M.F.%20Budinich&amp;author=J.R.%20Broadbent&amp;author=J.L.%20Steele&amp;publication_year=2009&amp;journal=Genome%20Biol.%20Evol&amp;volume=1&amp;pages=239-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evp019" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevp019" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevp019"> </span></div> </div></div></div></div></div><div content-id="dsw043-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fox</div> <div class="given-names">K.A.</div></div><div class="name string-name"><div class="surname">Ramesh</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Stearns</div> <div class="given-names">J.E.</div></div></div>, et al.  . <div class="year">2009</div>, 
<div class="article-title">Multiple posttranscriptional regulatory mechanisms partner to control ethanolamine utilization in <em>Enterococcus faecalis</em></div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">106</div>, <div class="fpage">4435</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20posttranscriptional%20regulatory%20mechanisms%20partner%20to%20control%20ethanolamine%20utilization%20in%20Enterococcus%20faecalis&amp;author=K.A.%20Fox&amp;author=A.%20Ramesh&amp;author=J.E.%20Stearns&amp;publication_year=2009&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=106&amp;pages=4435-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0812194106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0812194106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0812194106"> </span></div> </div></div></div></div></div><div content-id="dsw043-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Atack</div> <div class="given-names">J.M.</div></div><div class="name string-name"><div class="surname">Srikhanta</div> <div class="given-names">Y.N.</div></div><div class="name string-name"><div class="surname">Fox</div> <div class="given-names">K.L.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">A biphasic epigenetic switch controls immunoevasion, virulence and niche adaptation in non-typeable <em>Haemophilus influenzae</em></div>. <div class="source">Nat. Commun</div> ., <div class="volume">6</div>, <div class="fpage">7828</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20biphasic%20epigenetic%20switch%20controls%20immunoevasion%2C%20virulence%20and%20niche%20adaptation%20in%20non-typeable%20Haemophilus%20influenzae&amp;author=J.M.%20Atack&amp;author=Y.N.%20Srikhanta&amp;author=K.L.%20Fox&amp;publication_year=2015&amp;journal=Nat.%20Commun&amp;volume=6&amp;pages=7828" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms8828" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms8828" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms8828"> </span></div> </div></div></div></div></div><div content-id="dsw043-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Doores</div> <div class="given-names">S.</div></div></div> <div class="year">2002</div>, pH control agents and acidulants. In <div class="person-group"><div class="name string-name"><div class="surname">A. L.</div> <div class="given-names">Branen</div></div><div class="name string-name"><div class="surname">P. M.</div> <div class="given-names">Davidson</div></div><div class="name string-name"><div class="surname">S.</div> <div class="given-names">Salminen</div></div><div class="name string-name"><div class="surname">J. H.</div> <div class="given-names">Thorngate</div>III</div></div> (eds), <div class="source">Food Additives</div> , 
<div class="publisher-name">Marcel Dekker Inc</div>., 
<div class="publisher-loc">NY</div>, <div class="fpage">988</div>.</div></div></div></div><div content-id="dsw043-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fontana</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Cocconcelli</div> <div class="given-names">P.S.</div></div><div class="name string-name"><div class="surname">Vignolo</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Saavedra</div> <div class="given-names">L.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Occurrence of antilisterial structural bacteriocins genes in meat borne lactic acid bacteria</div>, <div class="source">Food Control</div> , <div class="volume">47</div>, <div class="fpage">53</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Occurrence%20of%20antilisterial%20structural%20bacteriocins%20genes%20in%20meat%20borne%20lactic%20acid%20bacteria&amp;author=C.%20Fontana&amp;author=P.S.%20Cocconcelli&amp;author=G.%20Vignolo&amp;author=L.%20Saavedra&amp;publication_year=2015&amp;journal=Food%20Control&amp;volume=47&amp;pages=53-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.foodcont.2014.06.021" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.foodcont.2014.06.021" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.foodcont.2014.06.021"> </span></div> </div></div></div></div></div><div content-id="dsw043-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gillor</div> <div class="given-names">O.</div></div><div class="name string-name"><div class="surname">Etzion</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Riley</div> <div class="given-names">M.A.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">The dual role of bacteriocins as anti- and probiotics</div>. <div class="source">Appl. Microbiol. Biotechnol</div> ., <div class="volume">81</div>, <div class="fpage">591</div>–<div class="lpage">606</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20dual%20role%20of%20bacteriocins%20as%20anti-%20and%20probiotics&amp;author=O.%20Gillor&amp;author=A.%20Etzion&amp;author=M.A.%20Riley&amp;publication_year=2008&amp;journal=Appl.%20Microbiol.%20Biotechnol&amp;volume=81&amp;pages=591-606" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00253-008-1726-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00253-008-1726-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00253-008-1726-5"> </span></div> </div></div></div></div></div><div content-id="dsw043-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Suárez</div> <div class="given-names">N.</div></div><div class="name string-name"><div class="surname">Hebert</div> <div class="given-names">E.M.</div></div><div class="name string-name"><div class="surname">Saavedra</div> <div class="given-names">L</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Genome mining and transcriptional analysis of bacteriocin genes in <em>Enterococcus faecium</em> CRL1879,</div> <div class="source">J. Data Mining Genomics Proteomics</div> , <div class="volume">6</div>, <div class="fpage">1</div>–<div class="lpage">8</div>.</div></div></div></div><div content-id="dsw043-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ozdemir</div> <div class="given-names">G.B.</div></div><div class="name string-name"><div class="surname">Oryasin</div> <div class="given-names">E.</div></div><div class="name string-name"><div class="surname">Biyik</div> <div class="given-names">H.H.</div></div><div class="name string-name"><div class="surname">Ozteber</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Bozdogan</div> <div class="given-names">B.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Phenotypic and genotypic characterization of bacteriocins in enterococcal isolates of different sources</div>, <div class="source">Indian J. Microbiol</div> ., <div class="volume">51</div>, <div class="fpage">182</div>–<div class="lpage">187</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phenotypic%20and%20genotypic%20characterization%20of%20bacteriocins%20in%20enterococcal%20isolates%20of%20different%20sources&amp;author=G.B.%20Ozdemir&amp;author=E.%20Oryasin&amp;author=H.H.%20Biyik&amp;author=M.%20Ozteber&amp;author=B.%20Bozdogan&amp;publication_year=2011&amp;journal=Indian%20J.%20Microbiol&amp;volume=51&amp;pages=182-187" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s12088-011-0143-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs12088-011-0143-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs12088-011-0143-0"> </span></div> </div></div></div></div></div><div content-id="dsw043-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bonacina</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Saavedra</div> <div class="given-names">L.</div></div><div class="name string-name"><div class="surname">Suarez</div> <div class="given-names">N.E.</div></div><div class="name string-name"><div class="surname">Sesma</div> <div class="given-names">F.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Draft genome sequence of the nonstarter bacteriocin-producing strain <em>Enterococcus mundtii</em> CRL35</div>, <div class="source">Genome Announ</div> ., <div class="volume">2</div>.</div></div></div></div><div content-id="dsw043-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Magni</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Espeche</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Repizo</div> <div class="given-names">G.D.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Draft genome sequence of <em>Enterococcus mundtii</em> CRL1656</div>, <div class="source">J. Bacteriol</div> ., <div class="volume">194</div>, <div class="fpage">550</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20sequence%20of%20Enterococcus%20mundtii%20CRL1656&amp;author=C.%20Magni&amp;author=C.%20Espeche&amp;author=G.D.%20Repizo&amp;publication_year=2012&amp;journal=J.%20Bacteriol&amp;volume=194&amp;pages=550" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.06415-11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.06415-11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.06415-11"> </span></div> </div></div></div></div></div><div content-id="dsw043-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shankar</div> <div class="given-names">N.</div></div><div class="name string-name"><div class="surname">Baghdayan</div> <div class="given-names">A. S.</div></div><div class="name string-name"><div class="surname">Gilmore</div> <div class="given-names">M.S.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">Modulation of virulence within a pathogenicity island in vancomycin-resistant <em>Enterococcus faecalis</em></div>, <div class="source">Nature</div> , <div class="volume">417</div>, <div class="fpage">746</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Modulation%20of%20virulence%20within%20a%20pathogenicity%20island%20in%20vancomycin-resistant%20Enterococcus%20faecalis&amp;author=N.%20Shankar&amp;author=A.%20S.%20Baghdayan&amp;author=M.S.%20Gilmore&amp;publication_year=2002&amp;journal=Nature&amp;volume=417&amp;pages=746-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature00802" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature00802" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature00802"> </span></div> </div></div></div></div></div><div content-id="dsw043-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Togay</div> <div class="given-names">S.O.</div></div><div class="name string-name"><div class="surname">Keskin</div> <div class="given-names">A.C.</div></div><div class="name string-name"><div class="surname">Acik</div> <div class="given-names">L.</div></div><div class="name string-name"><div class="surname">Temiz</div> <div class="given-names">A.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Virulence genes, antibiotic resistance and plasmid profiles of <em>Enterococcus faecalis</em> and <em>Enterococcus faecium</em> from naturally fermented Turkish foods,</div> <div class="source">J. Appl. Microbiol</div> ., <div class="volume">109</div>, <div class="fpage">1084</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Virulence%20genes%2C%20antibiotic%20resistance%20and%20plasmid%20profiles%20of%20Enterococcus%20faecalis%20and%20Enterococcus%20faecium%20from%20naturally%20fermented%20Turkish%20foods%2C&amp;author=S.O.%20Togay&amp;author=A.C.%20Keskin&amp;author=L.%20Acik&amp;author=A.%20Temiz&amp;publication_year=2010&amp;journal=J.%20Appl.%20Microbiol&amp;volume=109&amp;pages=1084-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/jam.2010.109.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjam.2010.109.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fjam.2010.109.issue-3"> </span></div> </div></div></div></div></div><div content-id="dsw043-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mikalsen</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Pedersen</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Willems</div> <div class="given-names">R.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">Investigating the mobilome in clinically important lineages of <em>Enterococcus faecium</em> and <em>Enterococcus faecalis</em></div>, <div class="source">BMC Genomics</div> , <div class="volume">16</div>, <div class="fpage">282</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Investigating%20the%20mobilome%20in%20clinically%20important%20lineages%20of%20Enterococcus%20faecium%20and%20Enterococcus%20faecalis&amp;author=T.%20Mikalsen&amp;author=T.%20Pedersen&amp;author=R.%20Willems&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=282" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-1407-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-1407-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-1407-6"> </span></div> </div></div></div></div></div><div content-id="dsw043-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Giraffa</div> <div class="given-names">G.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">Enterococci from foods</div>, <div class="source">FEMS Microbiol. Rev</div> ., <div class="volume">26</div>, <div class="fpage">163</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Enterococci%20from%20foods&amp;author=G.%20Giraffa&amp;publication_year=2002&amp;journal=FEMS%20Microbiol.%20Rev&amp;volume=26&amp;pages=163-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1574-6976.2002.tb00608.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1574-6976.2002.tb00608.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1574-6976.2002.tb00608.x"> </span></div> </div></div></div></div></div><div content-id="dsw043-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Barbosa</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Ferreira</div> <div class="given-names">V.</div></div><div class="name string-name"><div class="surname">Teixeira</div> <div class="given-names">P.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Antibiotic susceptibility of enterococci isolated from traditional fermented meat products</div>, <div class="source">Food Microbiol</div> ., <div class="volume">26</div>, <div class="fpage">527</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Antibiotic%20susceptibility%20of%20enterococci%20isolated%20from%20traditional%20fermented%20meat%20products&amp;author=J.%20Barbosa&amp;author=V.%20Ferreira&amp;author=P.%20Teixeira&amp;publication_year=2009&amp;journal=Food%20Microbiol&amp;volume=26&amp;pages=527-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fm.2009.03.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fm.2009.03.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fm.2009.03.005"> </span></div> </div></div></div></div></div><div content-id="dsw043-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Portillo</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Ruiz-Larrea</div> <div class="given-names">F.</div></div><div class="name string-name"><div class="surname">Zarazaga</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Alonso</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Martinez</div> <div class="given-names">J. L.</div></div><div class="name string-name"><div class="surname">Torres</div> <div class="given-names">C.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Macrolide resistance genes in <em>Enterococcus</em> spp</div>, <div class="source">Antimicrob. Agents Chemother</div> ., <div class="volume">44</div>, <div class="fpage">967</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Macrolide%20resistance%20genes%20in%20Enterococcus%20spp&amp;author=A.%20Portillo&amp;author=F.%20Ruiz-Larrea&amp;author=M.%20Zarazaga&amp;author=A.%20Alonso&amp;author=J.%20L.%20Martinez&amp;author=C.%20Torres&amp;publication_year=2000&amp;journal=Antimicrob.%20Agents%20Chemother&amp;volume=44&amp;pages=967-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AAC.44.4.967-971.2000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAAC.44.4.967-971.2000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAAC.44.4.967-971.2000"> </span></div> </div></div></div></div></div><div content-id="dsw043-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ramirez</div> <div class="given-names">M.S.</div></div><div class="name string-name"><div class="surname">Tolmasky</div> <div class="given-names">M.E.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Aminoglycoside modifying enzymes</div>, <div class="source">Drug Resistance Updates</div> , <div class="volume">13</div>, <div class="fpage">151</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Aminoglycoside%20modifying%20enzymes&amp;author=M.S.%20Ramirez&amp;author=M.E.%20Tolmasky&amp;publication_year=2010&amp;journal=Drug%20Resistance%20Updates&amp;volume=13&amp;pages=151-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.drup.2010.08.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.drup.2010.08.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.drup.2010.08.003"> </span></div> </div></div></div></div></div><div content-id="dsw043-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Singh</div> <div class="given-names">K.V.</div></div><div class="name string-name"><div class="surname">Murray</div> <div class="given-names">B.E.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Differences in the <em>Enterococcus faecalis lsa</em> locus that influence susceptibility to quinupristin-dalfopristin and clindamycin</div>, <div class="source">Antimicrob. Agents Chemother</div> ., <div class="volume">49</div>, <div class="fpage">32</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differences%20in%20the%20Enterococcus%20faecalis%20lsa%20locus%20that%20influence%20susceptibility%20to%20quinupristin-dalfopristin%20and%20clindamycin&amp;author=K.V.%20Singh&amp;author=B.E.%20Murray&amp;publication_year=2005&amp;journal=Antimicrob.%20Agents%20Chemother&amp;volume=49&amp;pages=32-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AAC.49.1.32-39.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAAC.49.1.32-39.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAAC.49.1.32-39.2005"> </span></div> </div></div></div></div></div><div content-id="dsw043-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Roberts</div> <div class="given-names">M.C.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Update on macrolide-lincosamide-streptogramin, ketolide, and oxazolidinone resistance genes</div>, <div class="source">FEMS Microbiol. Lett</div> ., <div class="volume">282</div>, <div class="fpage">147</div>–<div class="lpage">59</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Update%20on%20macrolide-lincosamide-streptogramin%2C%20ketolide%2C%20and%20oxazolidinone%20resistance%20genes&amp;author=M.C.%20Roberts&amp;publication_year=2008&amp;journal=FEMS%20Microbiol.%20Lett&amp;volume=282&amp;pages=147-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fml.2008.282.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffml.2008.282.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffml.2008.282.issue-2"> </span></div> </div></div></div></div></div><div content-id="dsw043-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Duerkop</div> <div class="given-names">B.A.</div></div><div class="name string-name"><div class="surname">Palmer</div> <div class="given-names">K.L.</div></div><div class="name string-name"><div class="surname">Horsburgh</div> <div class="given-names">M.J.</div></div></div> <div class="year">2014</div>, <div class="collab">Enterococcal bacteriophages and genome defense</div>.</div></div></div></div><div content-id="dsw043-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zimmer</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Sattelberger</div> <div class="given-names">E.</div></div><div class="name string-name"><div class="surname">Inman</div> <div class="given-names">R.B.</div></div><div class="name string-name"><div class="surname">Calendar</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Loessner</div> <div class="given-names">M. J.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Genome and proteome of <em>Listeria monocytogenes</em> phage PSA: an unusual case for programmed + 1 translational frameshifting in structural protein synthesis</div>, <div class="source">Mol. Microbiol</div> ., <div class="volume">50</div>, <div class="fpage">303</div>–<div class="lpage">17</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20and%20proteome%20of%20Listeria%20monocytogenes%20phage%20PSA%3A%20an%20unusual%20case%20for%20programmed%20%2B%201%20translational%20frameshifting%20in%20structural%20protein%20synthesis&amp;author=M.%20Zimmer&amp;author=E.%20Sattelberger&amp;author=R.B.%20Inman&amp;author=R.%20Calendar&amp;author=M.%20J.%20Loessner&amp;publication_year=2003&amp;journal=Mol.%20Microbiol&amp;volume=50&amp;pages=303-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.2003.03684.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.2003.03684.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.2003.03684.x"> </span></div> </div></div></div></div></div><div content-id="dsw043-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Denes</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Vongkamjan</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Ackermann</div> <div class="given-names">H.W.</div></div><div class="name string-name"><div class="surname">Moreno Switt</div> <div class="given-names">A.I.</div></div><div class="name string-name"><div class="surname">Wiedmann</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">den Bakker</div> <div class="given-names">H.C.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Comparative genomic and morphological analyses of <em>Listeria</em> phages isolated from farm environments</div>, <div class="source">Appl. Environm. Microbiol</div> ., <div class="volume">80</div>, <div class="fpage">4616</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genomic%20and%20morphological%20analyses%20of%20Listeria%20phages%20isolated%20from%20farm%20environments&amp;author=T.%20Denes&amp;author=K.%20Vongkamjan&amp;author=H.W.%20Ackermann&amp;author=A.I.%20Moreno%20Switt&amp;author=M.%20Wiedmann&amp;author=H.C.%20den%20Bakker&amp;publication_year=2014&amp;journal=Appl.%20Environm.%20Microbiol&amp;volume=80&amp;pages=4616-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.00720-14" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.00720-14" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.00720-14"> </span></div> </div></div></div></div></div><div content-id="dsw043-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Palmer</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">van Schaik</div> <div class="given-names">W.</div></div><div class="name string-name"><div class="surname">Willems</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Gilmore</div> <div class="given-names">M.</div></div></div> <div class="year">2014</div>. Enterococcal Genomics. In: M. Gilmore, D. Clewell, Y. Ike and N. Shankar, ed., Enterococci From Commensals to Leading Causes of Drug Resistant Infection, 1st ed. [online] Boston, Massachusetts: The Harvard Medical School, pp. 189-216.</div></div></div></div><div content-id="dsw043-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Makarova</div> <div class="given-names">K.S.</div></div><div class="name string-name"><div class="surname">Aravind</div> <div class="given-names">L.</div></div><div class="name string-name"><div class="surname">Wolf</div> <div class="given-names">Y.I.</div></div><div class="name string-name"><div class="surname">Koonin</div> <div class="given-names">E. V.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Unification of Cas protein families and a simple scenario for the origin and evolution of CRISPR-Cas systems</div>, <div class="source">Biol. Direct</div> , <div class="volume">6</div>, <div class="fpage">38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unification%20of%20Cas%20protein%20families%20and%20a%20simple%20scenario%20for%20the%20origin%20and%20evolution%20of%20CRISPR-Cas%20systems&amp;author=K.S.%20Makarova&amp;author=L.%20Aravind&amp;author=Y.I.%20Wolf&amp;author=E.%20V.%20Koonin&amp;publication_year=2011&amp;journal=Biol.%20Direct&amp;volume=6&amp;pages=38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1745-6150-6-38" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1745-6150-6-38" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1745-6150-6-38"> </span></div> </div></div></div></div></div><div content-id="dsw043-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Makarova</div> <div class="given-names">K.S.</div></div><div class="name string-name"><div class="surname">Wolf</div> <div class="given-names">Y.I.</div></div><div class="name string-name"><div class="surname">Alkhnbashi</div> <div class="given-names">O.S.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">An updated evolutionary classification of CRISPR-Cas systems</div>, <div class="source">Nat. Rev. Microbiol</div> ., <div class="volume">13</div>, <div class="fpage">722</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20updated%20evolutionary%20classification%20of%20CRISPR-Cas%20systems&amp;author=K.S.%20Makarova&amp;author=Y.I.%20Wolf&amp;author=O.S.%20Alkhnbashi&amp;publication_year=2015&amp;journal=Nat.%20Rev.%20Microbiol&amp;volume=13&amp;pages=722-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrmicro3569" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrmicro3569" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrmicro3569"> </span></div> </div></div></div></div></div><div content-id="dsw043-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Juhas</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">van der Meer</div> <div class="given-names">J.R.</div></div><div class="name string-name"><div class="surname">Gaillard</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Harding</div> <div class="given-names">R.M.</div></div><div class="name string-name"><div class="surname">Hood</div> <div class="given-names">D.W.</div></div><div class="name string-name"><div class="surname">Crook</div> <div class="given-names">D.W.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Genomic islands: tools of bacterial horizontal gene transfer and evolution</div>, <div class="source">FEMS Microbiol. Rev</div> ., <div class="volume">33</div>, <div class="fpage">376</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20islands%3A%20tools%20of%20bacterial%20horizontal%20gene%20transfer%20and%20evolution&amp;author=M.%20Juhas&amp;author=J.R.%20van%20der%20Meer&amp;author=M.%20Gaillard&amp;author=R.M.%20Harding&amp;author=D.W.%20Hood&amp;author=D.W.%20Crook&amp;publication_year=2009&amp;journal=FEMS%20Microbiol.%20Rev&amp;volume=33&amp;pages=376-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1574-6976.2008.00136.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1574-6976.2008.00136.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1574-6976.2008.00136.x"> </span></div> </div></div></div></div></div><div content-id="dsw043-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Larbig</div> <div class="given-names">K.D.</div></div><div class="name string-name"><div class="surname">Christmann</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Johann</div> <div class="given-names">A.</div></div></div>, et al.  . <div class="year">2002</div>, 
<div class="article-title">Gene islands integrated into tRNA(Gly) genes confer genome diversity on a <em>Pseudomonas aeruginosa</em> clone</div>, <div class="source">J. Bacteriol</div> ., <div class="volume">184</div>, <div class="fpage">6665</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20islands%20integrated%20into%20tRNA(Gly)%20genes%20confer%20genome%20diversity%20on%20a%20Pseudomonas%20aeruginosa%20clone&amp;author=K.D.%20Larbig&amp;author=A.%20Christmann&amp;author=A.%20Johann&amp;publication_year=2002&amp;journal=J.%20Bacteriol&amp;volume=184&amp;pages=6665-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.184.23.6665-6680.2002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.184.23.6665-6680.2002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.184.23.6665-6680.2002"> </span></div> </div></div></div></div></div><div content-id="dsw043-B70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw043-B70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sullivan</div> <div class="given-names">J.T.</div></div><div class="name string-name"><div class="surname">Trzebiatowski</div> <div class="given-names">J.R.</div></div><div class="name string-name"><div class="surname">Cruickshank</div> <div class="given-names">R.W.</div></div></div>, et al.  . <div class="year">2002</div>, 
<div class="article-title">Comparative sequence analysis of the symbiosis island of <em>Mesorhizobium loti</em> strain R7A</div>, <div class="source">J. Bacteriol</div> ., <div class="volume">184</div>, <div class="fpage">3086</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20sequence%20analysis%20of%20the%20symbiosis%20island%20of%20Mesorhizobium%20loti%20strain%20R7A&amp;author=J.T.%20Sullivan&amp;author=J.R.%20Trzebiatowski&amp;author=R.W.%20Cruickshank&amp;publication_year=2002&amp;journal=J.%20Bacteriol&amp;volume=184&amp;pages=3086-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.184.11.3086-3095.2002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.184.11.3086-3095.2002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.184.11.3086-3095.2002"> </span></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p"> This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2670108&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/1/10.1093_dnares_dsw043/1/dsw043_Supp.zip?Expires=2147483647&amp;Signature=qfrsEYitdaJFRtVb4lVTNX16upeZv8S1FjY1XCqewM9sqQh0e4qpEC679hfVK0HFqPdKNcQnFO1Gwa-dzW4JwoTXPeTDnBceFgGx7NZ3YWjVXWKJOh4x54hNDn57A9V46VBYbxEnR6dnIUmjr4FdsbLmOTACDeDJedoAFYuVLWryR6FHza5a23P88etL6BkNLUglUF6UYULgr0cd2tRX-QIgzhsKu21VyNcv5B7O7LHKc6mhhmOTzKWiJxE8BTHIijLs6MDBlnmsBkB9czJxa~FAznSfC-NsCoetlHpJ9z~bQwtssQ09AAjbTcHT~9c-3Q3xe8MiIXzYFKcJybo1CA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">568</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000397051500002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">5</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw043"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2670108"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000397051500002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f1%2f11%2f2670108">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000397051500002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (5)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F1%2F11%2F2670108" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2670108'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="genomic view of food-related and probiotic Enterococcus strains" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw043");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f1%2f11%2f2670108">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

