

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":0,"applicationTime":1937,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/14/3/117/578022","@type":"ScholarlyArticle","name":" Transcriptome Profiling of  Lotus japonicus  Roots During Arbuscular Mycorrhiza Development and Comparison with that of Nodulation ","datePublished":"2007-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/14/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2007-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsm014","keywords":["cysteine proteinase","defense response","phenylalanine ammonia-lyase","symbiosis"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/14/3/117/578022","author":[{"name":"Deguchi, Yuichi","affiliation":"1  Graduate School of Biostudies  ,  Kyoto University  ,  Sakyo-ku, Kyoto 606-8502  ,  Japan","@type":"Person"},{"name":"Banba, Mari","affiliation":"1  Graduate School of Biostudies  ,  Kyoto University  ,  Sakyo-ku, Kyoto 606-8502  ,  Japan","@type":"Person"},{"name":"Shimoda, Yoshikazu","affiliation":"2  Graduate School of Science and Technology  ,  Kagoshima University  ,  Kagoshima 890-0065  ,  Japan","@type":"Person"},{"name":"Chechetka, Svetlana A.","affiliation":"1  Graduate School of Biostudies  ,  Kyoto University  ,  Sakyo-ku, Kyoto 606-8502  ,  Japan","@type":"Person"},{"name":"Suzuri, Ryota","affiliation":"1  Graduate School of Biostudies  ,  Kyoto University  ,  Sakyo-ku, Kyoto 606-8502  ,  Japan","@type":"Person"},{"name":"Okusako, Yasuhiro","affiliation":"1  Graduate School of Biostudies  ,  Kyoto University  ,  Sakyo-ku, Kyoto 606-8502  ,  Japan","@type":"Person"},{"name":"Ooki, Yasuhiro","affiliation":"1  Graduate School of Biostudies  ,  Kyoto University  ,  Sakyo-ku, Kyoto 606-8502  ,  Japan","@type":"Person"},{"name":"Toyokura, Koichi","affiliation":"1  Graduate School of Biostudies  ,  Kyoto University  ,  Sakyo-ku, Kyoto 606-8502  ,  Japan","@type":"Person"},{"name":"Suzuki, Akihiro","affiliation":"3  Department of Chemistry and BioScience  ,  Kagoshima University  ,  Kagoshima 890-0065  ,  Japan","@type":"Person"},{"name":"Uchiumi, Toshiki","affiliation":"3  Department of Chemistry and BioScience  ,  Kagoshima University  ,  Kagoshima 890-0065  ,  Japan","@type":"Person"},{"name":"Higashi, Shiro","affiliation":"3  Department of Chemistry and BioScience  ,  Kagoshima University  ,  Kagoshima 890-0065  ,  Japan","@type":"Person"},{"name":"Abe, Mikiko","affiliation":"3  Department of Chemistry and BioScience  ,  Kagoshima University  ,  Kagoshima 890-0065  ,  Japan","@type":"Person"},{"name":"Kouchi, Hiroshi","affiliation":"4  National Institute of Agrobiological Sciences  ,  Tsukuba, Ibaraki 305-8602  ,  Japan","@type":"Person"},{"name":"Izui, Katsura","affiliation":"1  Graduate School of Biostudies  ,  Kyoto University  ,  Sakyo-ku, Kyoto 606-8502  ,  Japan","@type":"Person"},{"name":"Hata, Shingo","affiliation":"1  Graduate School of Biostudies  ,  Kyoto University  ,  Sakyo-ku, Kyoto 606-8502  ,  Japan    5  Core Research for Evolutional Science and Technology, Japan Science and Technology Agency  ,  Saitama 332-0012  ,  Japan","@type":"Person"}],"description":"Abstract.   To better understand the molecular responses of plants to arbuscular mycorrhizal (AM) fungi, we analyzed the differential gene expression patterns o","pageStart":"117","pageEnd":"133","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/3/0/m_dnaresearch14_3.cover.gif?Expires=2147483647&Signature=mMLx1qgQSoxMydWe0ZBNyatOi81iPNGZN59-N2j3ZGCXiGW-PJJB~9AMR~YJK1cFo0ovCThllgKwjHX5Pc7ByoVIVbi1~Iv8mmATs9QVv8pgbsrsKYz5hhgoFSvRFGlC9xxuUx6e9Y0Tu2NeIdWd0xEz5vCYEYm8xUeqKPc-Q~pmQayp6hW3k2w49NPmJ42Y3TDTDgg5VwhqQ3sy8I7I2puWhM9rZLgZqfKecCudhTesr6JCWg~PZNDKXI7m6emAJSOOih0oSvh-42LKbNQTeC3MXoOJjyLTVDQhjAQFJqCmheHjPu7UzqjeQD8L~eRkuwc8WKaIAVAMF4eV05GR0w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":" Transcriptome Profiling of  Lotus japonicus  Roots During Arbuscular Mycorrhiza Development and Comparison with that of Nodulation ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/3/0/m_dnaresearch14_3.cover.gif?Expires=2147483647&Signature=mMLx1qgQSoxMydWe0ZBNyatOi81iPNGZN59-N2j3ZGCXiGW-PJJB~9AMR~YJK1cFo0ovCThllgKwjHX5Pc7ByoVIVbi1~Iv8mmATs9QVv8pgbsrsKYz5hhgoFSvRFGlC9xxuUx6e9Y0Tu2NeIdWd0xEz5vCYEYm8xUeqKPc-Q~pmQayp6hW3k2w49NPmJ42Y3TDTDgg5VwhqQ3sy8I7I2puWhM9rZLgZqfKecCudhTesr6JCWg~PZNDKXI7m6emAJSOOih0oSvh-42LKbNQTeC3MXoOJjyLTVDQhjAQFJqCmheHjPu7UzqjeQD8L~eRkuwc8WKaIAVAMF4eV05GR0w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content=" Transcriptome Profiling of  Lotus japonicus  Roots During Arbuscular Mycorrhiza Development and Comparison with that of Nodulation " />
    <meta property="og:description" content="Abstract.   To better understand the molecular responses of plants to arbuscular mycorrhizal (AM) fungi, we analyzed the differential gene expression patterns o" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/14/3/117/578022" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2007-06-01" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Deguchi, Yuichi" /><meta name="citation_author_institution" content="Graduate School of Biostudies , Kyoto University , Sakyo-ku, Kyoto 606-8502 , Japan" /><meta name="citation_author" content="Banba, Mari" /><meta name="citation_author_institution" content="Graduate School of Biostudies , Kyoto University , Sakyo-ku, Kyoto 606-8502 , Japan" /><meta name="citation_author" content="Shimoda, Yoshikazu" /><meta name="citation_author_institution" content="Graduate School of Science and Technology , Kagoshima University , Kagoshima 890-0065 , Japan" /><meta name="citation_author" content="Chechetka, Svetlana A." /><meta name="citation_author_institution" content="Graduate School of Biostudies , Kyoto University , Sakyo-ku, Kyoto 606-8502 , Japan" /><meta name="citation_author" content="Suzuri, Ryota" /><meta name="citation_author_institution" content="Graduate School of Biostudies , Kyoto University , Sakyo-ku, Kyoto 606-8502 , Japan" /><meta name="citation_author" content="Okusako, Yasuhiro" /><meta name="citation_author_institution" content="Graduate School of Biostudies , Kyoto University , Sakyo-ku, Kyoto 606-8502 , Japan" /><meta name="citation_author" content="Ooki, Yasuhiro" /><meta name="citation_author_institution" content="Graduate School of Biostudies , Kyoto University , Sakyo-ku, Kyoto 606-8502 , Japan" /><meta name="citation_author" content="Toyokura, Koichi" /><meta name="citation_author_institution" content="Graduate School of Biostudies , Kyoto University , Sakyo-ku, Kyoto 606-8502 , Japan" /><meta name="citation_author" content="Suzuki, Akihiro" /><meta name="citation_author_institution" content="Department of Chemistry and BioScience , Kagoshima University , Kagoshima 890-0065 , Japan" /><meta name="citation_author" content="Uchiumi, Toshiki" /><meta name="citation_author_institution" content="Department of Chemistry and BioScience , Kagoshima University , Kagoshima 890-0065 , Japan" /><meta name="citation_author" content="Higashi, Shiro" /><meta name="citation_author_institution" content="Department of Chemistry and BioScience , Kagoshima University , Kagoshima 890-0065 , Japan" /><meta name="citation_author" content="Abe, Mikiko" /><meta name="citation_author_institution" content="Department of Chemistry and BioScience , Kagoshima University , Kagoshima 890-0065 , Japan" /><meta name="citation_author" content="Kouchi, Hiroshi" /><meta name="citation_author_institution" content="National Institute of Agrobiological Sciences , Tsukuba, Ibaraki 305-8602 , Japan" /><meta name="citation_author" content="Izui, Katsura" /><meta name="citation_author_institution" content="Graduate School of Biostudies , Kyoto University , Sakyo-ku, Kyoto 606-8502 , Japan" /><meta name="citation_author" content="Hata, Shingo" /><meta name="citation_author_institution" content="Graduate School of Biostudies , Kyoto University , Sakyo-ku, Kyoto 606-8502 , Japan" /><meta name="citation_author_institution" content="Core Research for Evolutional Science and Technology, Japan Science and Technology Agency , Saitama 332-0012 , Japan" /><meta name="citation_title" content=" Transcriptome Profiling of Lotus japonicus Roots During Arbuscular Mycorrhiza Development and Comparison with that of Nodulation " /><meta name="citation_firstpage" content="117" /><meta name="citation_lastpage" content="133" /><meta name="citation_doi" content="10.1093/dnares/dsm014" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="14" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2007/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=A new fungal phylum, the Glomeromycota : phylogeny and evolution; Mycol. Res.;  citation_year=2001;  citation_volume=105;  citation_pages=1413-1421; " /><meta name="citation_reference" content=" citation_edition=2nd Ed.; citation_publisher=Academic Press, San Diego; Mycorrhizal Symbiosis;  citation_year=1997; " /><meta name="citation_reference" content="citation_title=Signaling in the arbuscular mycorrhizal symbiosis; Annu. Rev. Microbiol.;  citation_year=2005;  citation_volume=59;  citation_pages=19-42; " /><meta name="citation_reference" content="citation_title=Mycorrhizas in natural ecosystem; Adv. Ecol. Res.;  citation_year=1991;  citation_volume=21;  citation_pages=171-313; " /><meta name="citation_reference" content=" citation_edition=2nd Ed.; citation_publisher=Academic Press, San Diego; Mineral Nutrition of Higher Plants;  citation_year=1995;  citation_pages=566-595; " /><meta name="citation_reference" content="citation_title=Plant cell responses to arbuscular mycorrhizal fungi: getting to the roots of the symbiosis; Plant Cell;  citation_year=1996;  citation_volume=8;  citation_pages=1871-1883; " /><meta name="citation_reference" content="citation_title=The origin of land plants: a matter of mycotrophism; Biosystems;  citation_year=1975;  citation_volume=6;  citation_pages=153-164; " /><meta name="citation_reference" content="citation_title=Four hundred-million-year-old vesicular arbuscular mycorrhizae; Proc. Natl. Acad. Sci. USA;  citation_year=1994;  citation_volume=91;  citation_pages=11841-11843; " /><meta name="citation_reference" content="citation_title=Glomalean fungi from the Ordovician; Science;  citation_year=2000;  citation_volume=289;  citation_pages=1920-1921; " /><meta name="citation_reference" content="citation_title=Seven Lotus japonicus genes required for transcriptional reprogramming of the root during fungal and bacterial symbiosis; Plant Cell;  citation_year=2005;  citation_volume=17;  citation_pages=2217-2229; " /><meta name="citation_reference" content="citation_title=Evolution of signal transduction in intracellular symbiosis; Trends Plant Sci.;  citation_year=2002;  citation_volume=7;  citation_pages=511-518; " /><meta name="citation_reference" content="citation_title=Exploring root symbiotic programs in the model legume Medicago truncatula using EST analysis; Nucleic Acids Res.;  citation_year=2002;  citation_volume=30;  citation_pages=5579-5592; " /><meta name="citation_reference" content="citation_title=Transcriptional changes in response to arbuscular mycorrhiza development in the model plant Medicago truncatula; Mol. Plant Microbe Interact.;  citation_year=2003;  citation_volume=16;  citation_pages=306-314; " /><meta name="citation_reference" content="citation_title=Expression profiling of up-regulated plant and fungal genes in early and late stages of Medicago truncatula-Glomus mosseae interactions; Mycorrhiza;  citation_year=2004;  citation_volume=14;  citation_pages=253-262; " /><meta name="citation_reference" content="citation_title=Combined transcriptome profiling reveals a novel family of arbuscular mycorrhizal-specific Medicago truncatula lectin genes; Mol. Plant Microbe Interact.;  citation_year=2005;  citation_volume=18;  citation_pages=771-782; " /><meta name="citation_reference" content="citation_title=Transcript profiling coupled with spatial expression analyses reveals genes involved in distinct developmental stages of an arbuscular mycorrhizal symbiosis; Plant Cell;  citation_year=2003;  citation_volume=15;  citation_pages=2106-2123; " /><meta name="citation_reference" content="citation_title=Transcriptome profiling in root nodules and arbuscular mycorrhiza identifies a collection of novel genes induced during Medicago truncatula root endosymbioses; Mol. Plant Microbe Interact.;  citation_year=2004;  citation_volume=17;  citation_pages=1063-1077; " /><meta name="citation_reference" content="citation_title=Overlaps in the transcriptional profiles of Medicago truncatula roots inoculated with two different Glomus fungi provide insights into the genetic program activated during arbuscular mycorrhiza; Plant Physiol.;  citation_year=2005;  citation_volume=137;  citation_pages=1283-1301; " /><meta name="citation_reference" content="citation_title=Medicago truncatula gene responses specific to arbuscular mycorrhiza interactions with different species and genera of Glomeromycota; Mycorrhiza;  citation_year=2007;  citation_volume=17;  citation_pages=223-234; " /><meta name="citation_reference" content="citation_title=Arbuscular mycorrhizal symbiosis is accompanied by local and systemic alterations in gene expression and an increase in disease resistance in the shoots; Plant J.;  citation_year=2007;  citation_volume=50;  citation_pages=529-544; " /><meta name="citation_reference" content="citation_title=Medicago truncatula , a model plant for studying the molecular genetics of the Rhizobium-legume symbiosis; Plant Mol. Biol. Rep.;  citation_year=1990;  citation_volume=8;  citation_pages=40-49; " /><meta name="citation_reference" content="citation_title=Medicago truncatula â€”a model in the making!; Curr. Opin. Plant Biol.;  citation_year=1999;  citation_volume=2;  citation_pages=301-304; " /><meta name="citation_reference" content="citation_title=Differential expression of eight chitinase genes in Medicago truncatula roots during mycorrhiza formation, nodulation, and pathogen infection; Mol. Plant Microbe Interact.;  citation_year=2000;  citation_volume=13;  citation_pages=763-777; " /><meta name="citation_reference" content="citation_title=Lotus japonicus , an autogamous, diploid legume species for classical and molecular genetics; Plant J.;  citation_year=1992;  citation_volume=2;  citation_pages=487-496; " /><meta name="citation_reference" content="citation_title=Plant sesquiterpenes induce hyphal branching in arbuscular mycorrhizal fungi; Nature;  citation_year=2005;  citation_volume=435;  citation_pages=824-827; " /><meta name="citation_reference" content="citation_title=Large-scale analysis of gene expression profiles during early stages of root nodule formation in a model legume, Lotus japonicus; DNA Res.;  citation_year=2004;  citation_volume=11;  citation_pages=263-274; " /><meta name="citation_reference" content="citation_title=cDNA macroarray analysis of gene expression in ineffective nodules induced on the Lotus japonicus sen1 mutant; Mol. Plant Microbe Interact.;  citation_year=2004;  citation_volume=17;  citation_pages=1223-1233; " /><meta name="citation_reference" content="citation_title=An evaluation of techniques for measuring vesicular-arbuscular mycorrhizal infection in roots; New Phytol.;  citation_year=1980;  citation_volume=84;  citation_pages=489-500; " /><meta name="citation_reference" content="citation_title=Improved procedure for clearing roots and staining parasitic and vesicular-arbuscular mycorrhizal fungi for rapid assessment of infection; Trans. Br. Mycol. Soc.;  citation_year=1970;  citation_volume=55;  citation_pages=158-161; " /><meta name="citation_reference" content="citation_title=Production of near-synchronous fungal colonization in tomato for developmental and molecular analyses of mycorrhiza; Mycol. Res.;  citation_year=1997;  citation_volume=101;  citation_pages=966-970; " /><meta name="citation_reference" content="citation_title=Lotus japonicus forms early senescent root nodules with Rhizobium etli; Mol. Plant Microbe Interact.;  citation_year=2001;  citation_volume=14;  citation_pages=173-180; " /><meta name="citation_reference" content="citation_title=Characterization of the Lotus japonicus symbiotic mutant lot1 that shows a reduced nodule number and distorted trichomes; Plant Physiol.;  citation_year=2005;  citation_volume=137;  citation_pages=1261-1271; " /><meta name="citation_reference" content="citation_title=Knockdown of an arbuscular mycorrhiza-inducible phosphate transporter gene of Lotus japonicus suppresses mutualistic symbiosis; Plant Cell Physiol.;  citation_year=2006;  citation_volume=47;  citation_pages=807-817; " /><meta name="citation_reference" content="citation_title=Isolation and characterization of novel nodulin cDNAs representing genes expressed at early stages of soybean nodule development; Mol. Gen. Genet.;  citation_year=1993;  citation_volume=238;  citation_pages=106-119; " /><meta name="citation_reference" content="citation_title=Gene silencing by expression of hairpin RNA in Lotus japonicus roots and root nodules; Mol. Plant Microbe Interact.;  citation_year=2003;  citation_volume=16;  citation_pages=663-668; " /><meta name="citation_reference" content="citation_title=GUS fusions: beta-glucuronidase as a sensitive and versatile gene fusion marker in higher plants; EMBO J.;  citation_year=1987;  citation_volume=6;  citation_pages=3901-3907; " /><meta name="citation_reference" content="citation_title=Methods to estimate the proportion of plant and fungal RNA in an arbuscular mycorrhiza; Mycorrhiza;  citation_year=2002;  citation_volume=12;  citation_pages=67-74; " /><meta name="citation_reference" content="citation_title=Significance analysis of microarrays applied to the ionizing radiation response; Proc. Natl. Acad. Sci. USA;  citation_year=2001;  citation_volume=98;  citation_pages=5116-5121; " /><meta name="citation_reference" content="citation_title=Stress-induced phenylpropanoid metabolism; Plant Cell;  citation_year=1995;  citation_volume=7;  citation_pages=1085-1097; " /><meta name="citation_reference" content="citation_title=Non-self recognition, transcriptional reprogramming, and secondary metabolite accumulation during plant/pathogen interactions; Proc. Natl. Acad. Sci. USA;  citation_year=2003;  citation_volume=100;  citation_pages=14569-14576; " /><meta name="citation_reference" content="citation_title=Comparative transcriptomics of rice reveals an ancient pattern of response to microbial colonization; Proc. Natl. Acad. Sci. USA;  citation_year=2005;  citation_volume=102;  citation_pages=8066-8070; " /><meta name="citation_reference" content="citation_title=Immunolocalization of a cysteine protease in vacuoles, vesicles, and symbiosomes of pea nodule cells; Plant Physiol.;  citation_year=2000;  citation_volume=123;  citation_pages=521-530; " /><meta name="citation_reference" content="citation_title=Senescence-associated vacuoles with intense proteolytic activity develop in leaves of Arabidopsis and soybean; Plant J.;  citation_year=2005;  citation_volume=41;  citation_pages=831-844; " /><meta name="citation_reference" content="citation_title=Cloning and characterization of two phosphate transporters from Medicago truncatula roots: regulation in response to phosphate and to colonization by arbuscular mycorrhizal (AM) fungi; Mol. Plant Microbe Interact.;  citation_year=1998;  citation_volume=11;  citation_pages=14-22; " /><meta name="citation_reference" content="citation_title=A phosphate transporter expressed in arbuscule-containing cells in potato; Nature;  citation_year=2001;  citation_volume=414;  citation_pages=462-470; " /><meta name="citation_reference" content="citation_title=Rice phosphate transporters include an evolutionarily divergent gene specifically activated in arbuscular mycorrhizal symbiosis; Proc. Natl. Acad. Sci. USA;  citation_year=2002;  citation_volume=99;  citation_pages=13324-13329; " /><meta name="citation_reference" content="citation_title=Defense-related transcript accumulation in Phaseolus vulgaris L. colonized by the arbuscular mycorrhizal fungus Glomus intraradices Schenck &amp; Smith; Plant Physiol.;  citation_year=1996;  citation_volume=110;  citation_pages=675-688; " /><meta name="citation_reference" content="citation_title=Expression patterns of defense-related genes in different types of arbuscular mycorrhizal development in wild-type and mycorrhiza-defective mutant tomato; Mol. Plant Microbe Interact.;  citation_year=2004;  citation_volume=17;  citation_pages=1103-1113; " /><meta name="citation_reference" content="citation_title=Isoflavonoid accumulation and expression of defense gene transcripts during the establishment of vesicular-arbuscular mycorrhizal associations in roots of Medicago truncatula; Mol. Plant Microbe Interact.;  citation_year=1993;  citation_volume=6;  citation_pages=643-654; " /><meta name="citation_reference" content="citation_title=Spatial patterns of expression of flavonoid/isoflavonoid pathway genes during interactions between roots of Medicago truncatula and the mycorrhizal fungus Glomus versiforme; Plant J.;  citation_year=1994;  citation_volume=6;  citation_pages=9-20; " /><meta name="citation_reference" content="citation_title=A vesicular arbuscular mycorrhizal fungus ( Glomus intraradix ) induces a defense response in alfalfa roots; Plant Physiol.;  citation_year=1994;  citation_volume=104;  citation_pages=683-689; " /><meta name="citation_reference" content="citation_title=Suppression of an isoflavonoid phytoalexin defense response in micorrhizal alfalfa roots; Plant Physiol.;  citation_year=1995;  citation_volume=108;  citation_pages=1449-1454; " /><meta name="citation_reference" content="citation_title=A gene encoding a receptor-like protein kinase in the roots of common bean is differentially regulated in response to pathogens, symbionts and nodulation factors; Plant Sci.;  citation_year=1999;  citation_volume=142;  citation_pages=133-145; " /><meta name="citation_reference" content="citation_title=Novel receptor-like protein kinases induced by Erwinia carotovora and short oligogalacturonides in potato; Mol. Plant Pathol.;  citation_year=2001;  citation_volume=2;  citation_pages=339-346; " /><meta name="citation_reference" content="citation_title=Modes of expression and common structural features of the complete phenylalanine ammonia-lyase gene family in parsley; Proc. Natl. Acad. Sci. USA;  citation_year=1995;  citation_volume=92;  citation_pages=5905-5909; " /><meta name="citation_reference" content="citation_title=WRKY transcription factors: from DNA binding towards biological function; Curr. Opin. Plant Biol.;  citation_year=2004;  citation_volume=7;  citation_pages=491-498; " /><meta name="citation_reference" content="citation_title=The plant hormone abscisic acid mediates the drought-induced expression but not the seed-specific expression of rd22, a gene responsive to dehydration stress in Arabidopsis thaliana; Mol. Gen. Genet.;  citation_year=1993;  citation_volume=238;  citation_pages=17-25; " /><meta name="citation_reference" content="citation_title=Differential expression of phenylalanine ammonia-lyase and chalcone synthase during soybean nodule development; Plant Cell;  citation_year=1991;  citation_volume=3;  citation_pages=299-308; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/14/3/117/6935590/dsm014.pdf" /><meta name="description" content="Abstract.   To better understand the molecular responses of plants to arbuscular mycorrhizal (AM) fungi, we analyzed the differential gene expression patterns o" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/14/3/117/578022" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/14/3/117/578022" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/14/3/117/578022");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsm014';
    </script>
<title>
	 Transcriptome Profiling of Lotus japonicus Roots During Arbuscular Mycorrhiza Development and Comparison with that of Nodulation  | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./578022" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZ+UATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRGVndWNoaSwgWXVpY2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIEJpb3N0dWRpZXMgLCBLeW90byBVbml2ZXJzaXR5ICwgU2FreW8ta3UsIEt5b3RvIDYwNi04NTAyICwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJCYW5iYSwgTWFyaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBCaW9zdHVkaWVzICwgS3lvdG8gVW5pdmVyc2l0eSAsIFNha3lvLWt1LCBLeW90byA2MDYtODUwMiAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2hpbW9kYSwgWW9zaGlrYXp1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3kgLCBLYWdvc2hpbWEgVW5pdmVyc2l0eSAsIEthZ29zaGltYSA4OTAtMDA2NSAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2hlY2hldGthLCBTdmV0bGFuYSBBLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBCaW9zdHVkaWVzICwgS3lvdG8gVW5pdmVyc2l0eSAsIFNha3lvLWt1LCBLeW90byA2MDYtODUwMiAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU3V6dXJpLCBSeW90YSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBCaW9zdHVkaWVzICwgS3lvdG8gVW5pdmVyc2l0eSAsIFNha3lvLWt1LCBLeW90byA2MDYtODUwMiAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iT2t1c2FrbywgWWFzdWhpcm8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgQmlvc3R1ZGllcyAsIEt5b3RvIFVuaXZlcnNpdHkgLCBTYWt5by1rdSwgS3lvdG8gNjA2LTg1MDIgLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik9va2ksIFlhc3VoaXJvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIEJpb3N0dWRpZXMgLCBLeW90byBVbml2ZXJzaXR5ICwgU2FreW8ta3UsIEt5b3RvIDYwNi04NTAyICwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUb3lva3VyYSwgS29pY2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIEJpb3N0dWRpZXMgLCBLeW90byBVbml2ZXJzaXR5ICwgU2FreW8ta3UsIEt5b3RvIDYwNi04NTAyICwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTdXp1a2ksIEFraWhpcm8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIENoZW1pc3RyeSBhbmQgQmlvU2NpZW5jZSAsIEthZ29zaGltYSBVbml2ZXJzaXR5ICwgS2Fnb3NoaW1hIDg5MC0wMDY1ICwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJVY2hpdW1pLCBUb3NoaWtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBDaGVtaXN0cnkgYW5kIEJpb1NjaWVuY2UgLCBLYWdvc2hpbWEgVW5pdmVyc2l0eSAsIEthZ29zaGltYSA4OTAtMDA2NSAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGlnYXNoaSwgU2hpcm8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIENoZW1pc3RyeSBhbmQgQmlvU2NpZW5jZSAsIEthZ29zaGltYSBVbml2ZXJzaXR5ICwgS2Fnb3NoaW1hIDg5MC0wMDY1ICwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBYmUsIE1pa2lrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQ2hlbWlzdHJ5IGFuZCBCaW9TY2llbmNlICwgS2Fnb3NoaW1hIFVuaXZlcnNpdHkgLCBLYWdvc2hpbWEgODkwLTAwNjUgLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IktvdWNoaSwgSGlyb3NoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBBZ3JvYmlvbG9naWNhbCBTY2llbmNlcyAsIFRzdWt1YmEsIEliYXJha2kgMzA1LTg2MDIgLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikl6dWksIEthdHN1cmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgQmlvc3R1ZGllcyAsIEt5b3RvIFVuaXZlcnNpdHkgLCBTYWt5by1rdSwgS3lvdG8gNjA2LTg1MDIgLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkhhdGEsIFNoaW5nbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBCaW9zdHVkaWVzICwgS3lvdG8gVW5pdmVyc2l0eSAsIFNha3lvLWt1LCBLeW90byA2MDYtODUwMiAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ29yZSBSZXNlYXJjaCBmb3IgRXZvbHV0aW9uYWwgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgSmFwYW4gU2NpZW5jZSBhbmQgVGVjaG5vbG9neSBBZ2VuY3kgLCBTYWl0YW1hIDMzMi0wMDEyICwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IiBUcmFuc2NyaXB0b21lIFByb2ZpbGluZyBvZiBMb3R1cyBqYXBvbmljdXMgUm9vdHMgRHVyaW5nIEFyYnVzY3VsYXIgTXljb3JyaGl6YSBEZXZlbG9wbWVudCBhbmQgQ29tcGFyaXNvbiB3aXRoIHRoYXQgb2YgTm9kdWxhdGlvbiAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIxMTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjEzMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzbTAxNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDA3LzAxLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBuZXcgZnVuZ2FsIHBoeWx1bSwgdGhlIEdsb21lcm9teWNvdGEgOiBwaHlsb2dlbnkgYW5kIGV2b2x1dGlvbjsgTXljb2wuIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTA1OyAgY2l0YXRpb25fcGFnZXM9MTQxMy0xNDIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSIgY2l0YXRpb25fZWRpdGlvbj0ybmQgRWQuOyBjaXRhdGlvbl9wdWJsaXNoZXI9QWNhZGVtaWMgUHJlc3MsIFNhbiBEaWVnbzsgTXljb3JyaGl6YWwgU3ltYmlvc2lzOyAgY2l0YXRpb25feWVhcj0xOTk3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaWduYWxpbmcgaW4gdGhlIGFyYnVzY3VsYXIgbXljb3JyaGl6YWwgc3ltYmlvc2lzOyBBbm51LiBSZXYuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NTk7ICBjaXRhdGlvbl9wYWdlcz0xOS00MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXljb3JyaGl6YXMgaW4gbmF0dXJhbCBlY29zeXN0ZW07IEFkdi4gRWNvbC4gUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5MTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTE3MS0zMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IiBjaXRhdGlvbl9lZGl0aW9uPTJuZCBFZC47IGNpdGF0aW9uX3B1Ymxpc2hlcj1BY2FkZW1pYyBQcmVzcywgU2FuIERpZWdvOyBNaW5lcmFsIE51dHJpdGlvbiBvZiBIaWdoZXIgUGxhbnRzOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fcGFnZXM9NTY2LTU5NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGxhbnQgY2VsbCByZXNwb25zZXMgdG8gYXJidXNjdWxhciBteWNvcnJoaXphbCBmdW5naTogZ2V0dGluZyB0byB0aGUgcm9vdHMgb2YgdGhlIHN5bWJpb3NpczsgUGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MTg3MS0xODgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgb3JpZ2luIG9mIGxhbmQgcGxhbnRzOiBhIG1hdHRlciBvZiBteWNvdHJvcGhpc207IEJpb3N5c3RlbXM7ICBjaXRhdGlvbl95ZWFyPTE5NzU7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTE1My0xNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZvdXIgaHVuZHJlZC1taWxsaW9uLXllYXItb2xkIHZlc2ljdWxhciBhcmJ1c2N1bGFyIG15Y29ycmhpemFlOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT05MTsgIGNpdGF0aW9uX3BhZ2VzPTExODQxLTExODQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HbG9tYWxlYW4gZnVuZ2kgZnJvbSB0aGUgT3Jkb3ZpY2lhbjsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yODk7ICBjaXRhdGlvbl9wYWdlcz0xOTIwLTE5MjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNldmVuIExvdHVzIGphcG9uaWN1cyBnZW5lcyByZXF1aXJlZCBmb3IgdHJhbnNjcmlwdGlvbmFsIHJlcHJvZ3JhbW1pbmcgb2YgdGhlIHJvb3QgZHVyaW5nIGZ1bmdhbCBhbmQgYmFjdGVyaWFsIHN5bWJpb3NpczsgUGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTIyMTctMjIyOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uIG9mIHNpZ25hbCB0cmFuc2R1Y3Rpb24gaW4gaW50cmFjZWxsdWxhciBzeW1iaW9zaXM7IFRyZW5kcyBQbGFudCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz01MTEtNTE4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHBsb3Jpbmcgcm9vdCBzeW1iaW90aWMgcHJvZ3JhbXMgaW4gdGhlIG1vZGVsIGxlZ3VtZSBNZWRpY2FnbyB0cnVuY2F0dWxhIHVzaW5nIEVTVCBhbmFseXNpczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9NTU3OS01NTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0aW9uYWwgY2hhbmdlcyBpbiByZXNwb25zZSB0byBhcmJ1c2N1bGFyIG15Y29ycmhpemEgZGV2ZWxvcG1lbnQgaW4gdGhlIG1vZGVsIHBsYW50IE1lZGljYWdvIHRydW5jYXR1bGE7IE1vbC4gUGxhbnQgTWljcm9iZSBJbnRlcmFjdC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0zMDYtMzE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHByZXNzaW9uIHByb2ZpbGluZyBvZiB1cC1yZWd1bGF0ZWQgcGxhbnQgYW5kIGZ1bmdhbCBnZW5lcyBpbiBlYXJseSBhbmQgbGF0ZSBzdGFnZXMgb2YgTWVkaWNhZ28gdHJ1bmNhdHVsYS1HbG9tdXMgbW9zc2VhZSBpbnRlcmFjdGlvbnM7IE15Y29ycmhpemE7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0yNTMtMjYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21iaW5lZCB0cmFuc2NyaXB0b21lIHByb2ZpbGluZyByZXZlYWxzIGEgbm92ZWwgZmFtaWx5IG9mIGFyYnVzY3VsYXIgbXljb3JyaGl6YWwtc3BlY2lmaWMgTWVkaWNhZ28gdHJ1bmNhdHVsYSBsZWN0aW4gZ2VuZXM7IE1vbC4gUGxhbnQgTWljcm9iZSBJbnRlcmFjdC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz03NzEtNzgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0IHByb2ZpbGluZyBjb3VwbGVkIHdpdGggc3BhdGlhbCBleHByZXNzaW9uIGFuYWx5c2VzIHJldmVhbHMgZ2VuZXMgaW52b2x2ZWQgaW4gZGlzdGluY3QgZGV2ZWxvcG1lbnRhbCBzdGFnZXMgb2YgYW4gYXJidXNjdWxhciBteWNvcnJoaXphbCBzeW1iaW9zaXM7IFBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0yMTA2LTIxMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRvbWUgcHJvZmlsaW5nIGluIHJvb3Qgbm9kdWxlcyBhbmQgYXJidXNjdWxhciBteWNvcnJoaXphIGlkZW50aWZpZXMgYSBjb2xsZWN0aW9uIG9mIG5vdmVsIGdlbmVzIGluZHVjZWQgZHVyaW5nIE1lZGljYWdvIHRydW5jYXR1bGEgcm9vdCBlbmRvc3ltYmlvc2VzOyBNb2wuIFBsYW50IE1pY3JvYmUgSW50ZXJhY3QuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MTA2My0xMDc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PdmVybGFwcyBpbiB0aGUgdHJhbnNjcmlwdGlvbmFsIHByb2ZpbGVzIG9mIE1lZGljYWdvIHRydW5jYXR1bGEgcm9vdHMgaW5vY3VsYXRlZCB3aXRoIHR3byBkaWZmZXJlbnQgR2xvbXVzIGZ1bmdpIHByb3ZpZGUgaW5zaWdodHMgaW50byB0aGUgZ2VuZXRpYyBwcm9ncmFtIGFjdGl2YXRlZCBkdXJpbmcgYXJidXNjdWxhciBteWNvcnJoaXphOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMzc7ICBjaXRhdGlvbl9wYWdlcz0xMjgzLTEzMDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1lZGljYWdvIHRydW5jYXR1bGEgZ2VuZSByZXNwb25zZXMgc3BlY2lmaWMgdG8gYXJidXNjdWxhciBteWNvcnJoaXphIGludGVyYWN0aW9ucyB3aXRoIGRpZmZlcmVudCBzcGVjaWVzIGFuZCBnZW5lcmEgb2YgR2xvbWVyb215Y290YTsgTXljb3JyaGl6YTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTIyMy0yMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFyYnVzY3VsYXIgbXljb3JyaGl6YWwgc3ltYmlvc2lzIGlzIGFjY29tcGFuaWVkIGJ5IGxvY2FsIGFuZCBzeXN0ZW1pYyBhbHRlcmF0aW9ucyBpbiBnZW5lIGV4cHJlc3Npb24gYW5kIGFuIGluY3JlYXNlIGluIGRpc2Vhc2UgcmVzaXN0YW5jZSBpbiB0aGUgc2hvb3RzOyBQbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT01MDsgIGNpdGF0aW9uX3BhZ2VzPTUyOS01NDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1lZGljYWdvIHRydW5jYXR1bGEgLCBhIG1vZGVsIHBsYW50IGZvciBzdHVkeWluZyB0aGUgbW9sZWN1bGFyIGdlbmV0aWNzIG9mIHRoZSBSaGl6b2JpdW0tbGVndW1lIHN5bWJpb3NpczsgUGxhbnQgTW9sLiBCaW9sLiBSZXAuOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz00MC00OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWVkaWNhZ28gdHJ1bmNhdHVsYSDigJRhIG1vZGVsIGluIHRoZSBtYWtpbmchOyBDdXJyLiBPcGluLiBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9MzAxLTMwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlmZmVyZW50aWFsIGV4cHJlc3Npb24gb2YgZWlnaHQgY2hpdGluYXNlIGdlbmVzIGluIE1lZGljYWdvIHRydW5jYXR1bGEgcm9vdHMgZHVyaW5nIG15Y29ycmhpemEgZm9ybWF0aW9uLCBub2R1bGF0aW9uLCBhbmQgcGF0aG9nZW4gaW5mZWN0aW9uOyBNb2wuIFBsYW50IE1pY3JvYmUgSW50ZXJhY3QuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9NzYzLTc3NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TG90dXMgamFwb25pY3VzICwgYW4gYXV0b2dhbW91cywgZGlwbG9pZCBsZWd1bWUgc3BlY2llcyBmb3IgY2xhc3NpY2FsIGFuZCBtb2xlY3VsYXIgZ2VuZXRpY3M7IFBsYW50IEouOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz00ODctNDk2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QbGFudCBzZXNxdWl0ZXJwZW5lcyBpbmR1Y2UgaHlwaGFsIGJyYW5jaGluZyBpbiBhcmJ1c2N1bGFyIG15Y29ycmhpemFsIGZ1bmdpOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NDM1OyAgY2l0YXRpb25fcGFnZXM9ODI0LTgyNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGFyZ2Utc2NhbGUgYW5hbHlzaXMgb2YgZ2VuZSBleHByZXNzaW9uIHByb2ZpbGVzIGR1cmluZyBlYXJseSBzdGFnZXMgb2Ygcm9vdCBub2R1bGUgZm9ybWF0aW9uIGluIGEgbW9kZWwgbGVndW1lLCBMb3R1cyBqYXBvbmljdXM7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MjYzLTI3NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Y0ROQSBtYWNyb2FycmF5IGFuYWx5c2lzIG9mIGdlbmUgZXhwcmVzc2lvbiBpbiBpbmVmZmVjdGl2ZSBub2R1bGVzIGluZHVjZWQgb24gdGhlIExvdHVzIGphcG9uaWN1cyBzZW4xIG11dGFudDsgTW9sLiBQbGFudCBNaWNyb2JlIEludGVyYWN0LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTEyMjMtMTIzMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gZXZhbHVhdGlvbiBvZiB0ZWNobmlxdWVzIGZvciBtZWFzdXJpbmcgdmVzaWN1bGFyLWFyYnVzY3VsYXIgbXljb3JyaGl6YWwgaW5mZWN0aW9uIGluIHJvb3RzOyBOZXcgUGh5dG9sLjsgIGNpdGF0aW9uX3llYXI9MTk4MDsgIGNpdGF0aW9uX3ZvbHVtZT04NDsgIGNpdGF0aW9uX3BhZ2VzPTQ4OS01MDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUltcHJvdmVkIHByb2NlZHVyZSBmb3IgY2xlYXJpbmcgcm9vdHMgYW5kIHN0YWluaW5nIHBhcmFzaXRpYyBhbmQgdmVzaWN1bGFyLWFyYnVzY3VsYXIgbXljb3JyaGl6YWwgZnVuZ2kgZm9yIHJhcGlkIGFzc2Vzc21lbnQgb2YgaW5mZWN0aW9uOyBUcmFucy4gQnIuIE15Y29sLiBTb2MuOyAgY2l0YXRpb25feWVhcj0xOTcwOyAgY2l0YXRpb25fdm9sdW1lPTU1OyAgY2l0YXRpb25fcGFnZXM9MTU4LTE2MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvZHVjdGlvbiBvZiBuZWFyLXN5bmNocm9ub3VzIGZ1bmdhbCBjb2xvbml6YXRpb24gaW4gdG9tYXRvIGZvciBkZXZlbG9wbWVudGFsIGFuZCBtb2xlY3VsYXIgYW5hbHlzZXMgb2YgbXljb3JyaGl6YTsgTXljb2wuIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MTAxOyAgY2l0YXRpb25fcGFnZXM9OTY2LTk3MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TG90dXMgamFwb25pY3VzIGZvcm1zIGVhcmx5IHNlbmVzY2VudCByb290IG5vZHVsZXMgd2l0aCBSaGl6b2JpdW0gZXRsaTsgTW9sLiBQbGFudCBNaWNyb2JlIEludGVyYWN0LjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTE3My0xODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgdGhlIExvdHVzIGphcG9uaWN1cyBzeW1iaW90aWMgbXV0YW50IGxvdDEgdGhhdCBzaG93cyBhIHJlZHVjZWQgbm9kdWxlIG51bWJlciBhbmQgZGlzdG9ydGVkIHRyaWNob21lczsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTM3OyAgY2l0YXRpb25fcGFnZXM9MTI2MS0xMjcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Lbm9ja2Rvd24gb2YgYW4gYXJidXNjdWxhciBteWNvcnJoaXphLWluZHVjaWJsZSBwaG9zcGhhdGUgdHJhbnNwb3J0ZXIgZ2VuZSBvZiBMb3R1cyBqYXBvbmljdXMgc3VwcHJlc3NlcyBtdXR1YWxpc3RpYyBzeW1iaW9zaXM7IFBsYW50IENlbGwgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NDc7ICBjaXRhdGlvbl9wYWdlcz04MDctODE3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jc29sYXRpb24gYW5kIGNoYXJhY3Rlcml6YXRpb24gb2Ygbm92ZWwgbm9kdWxpbiBjRE5BcyByZXByZXNlbnRpbmcgZ2VuZXMgZXhwcmVzc2VkIGF0IGVhcmx5IHN0YWdlcyBvZiBzb3liZWFuIG5vZHVsZSBkZXZlbG9wbWVudDsgTW9sLiBHZW4uIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzg7ICBjaXRhdGlvbl9wYWdlcz0xMDYtMTE5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIHNpbGVuY2luZyBieSBleHByZXNzaW9uIG9mIGhhaXJwaW4gUk5BIGluIExvdHVzIGphcG9uaWN1cyByb290cyBhbmQgcm9vdCBub2R1bGVzOyBNb2wuIFBsYW50IE1pY3JvYmUgSW50ZXJhY3QuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9NjYzLTY2ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R1VTIGZ1c2lvbnM6IGJldGEtZ2x1Y3Vyb25pZGFzZSBhcyBhIHNlbnNpdGl2ZSBhbmQgdmVyc2F0aWxlIGdlbmUgZnVzaW9uIG1hcmtlciBpbiBoaWdoZXIgcGxhbnRzOyBFTUJPIEouOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0zOTAxLTM5MDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1ldGhvZHMgdG8gZXN0aW1hdGUgdGhlIHByb3BvcnRpb24gb2YgcGxhbnQgYW5kIGZ1bmdhbCBSTkEgaW4gYW4gYXJidXNjdWxhciBteWNvcnJoaXphOyBNeWNvcnJoaXphOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9NjctNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpZ25pZmljYW5jZSBhbmFseXNpcyBvZiBtaWNyb2FycmF5cyBhcHBsaWVkIHRvIHRoZSBpb25pemluZyByYWRpYXRpb24gcmVzcG9uc2U7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTk4OyAgY2l0YXRpb25fcGFnZXM9NTExNi01MTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJlc3MtaW5kdWNlZCBwaGVueWxwcm9wYW5vaWQgbWV0YWJvbGlzbTsgUGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MTA4NS0xMDk3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ob24tc2VsZiByZWNvZ25pdGlvbiwgdHJhbnNjcmlwdGlvbmFsIHJlcHJvZ3JhbW1pbmcsIGFuZCBzZWNvbmRhcnkgbWV0YWJvbGl0ZSBhY2N1bXVsYXRpb24gZHVyaW5nIHBsYW50L3BhdGhvZ2VuIGludGVyYWN0aW9uczsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTAwOyAgY2l0YXRpb25fcGFnZXM9MTQ1NjktMTQ1NzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIHRyYW5zY3JpcHRvbWljcyBvZiByaWNlIHJldmVhbHMgYW4gYW5jaWVudCBwYXR0ZXJuIG9mIHJlc3BvbnNlIHRvIG1pY3JvYmlhbCBjb2xvbml6YXRpb247IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTEwMjsgIGNpdGF0aW9uX3BhZ2VzPTgwNjYtODA3MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW1tdW5vbG9jYWxpemF0aW9uIG9mIGEgY3lzdGVpbmUgcHJvdGVhc2UgaW4gdmFjdW9sZXMsIHZlc2ljbGVzLCBhbmQgc3ltYmlvc29tZXMgb2YgcGVhIG5vZHVsZSBjZWxsczsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTIzOyAgY2l0YXRpb25fcGFnZXM9NTIxLTUzMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VuZXNjZW5jZS1hc3NvY2lhdGVkIHZhY3VvbGVzIHdpdGggaW50ZW5zZSBwcm90ZW9seXRpYyBhY3Rpdml0eSBkZXZlbG9wIGluIGxlYXZlcyBvZiBBcmFiaWRvcHNpcyBhbmQgc295YmVhbjsgUGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NDE7ICBjaXRhdGlvbl9wYWdlcz04MzEtODQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbG9uaW5nIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIHR3byBwaG9zcGhhdGUgdHJhbnNwb3J0ZXJzIGZyb20gTWVkaWNhZ28gdHJ1bmNhdHVsYSByb290czogcmVndWxhdGlvbiBpbiByZXNwb25zZSB0byBwaG9zcGhhdGUgYW5kIHRvIGNvbG9uaXphdGlvbiBieSBhcmJ1c2N1bGFyIG15Y29ycmhpemFsIChBTSkgZnVuZ2k7IE1vbC4gUGxhbnQgTWljcm9iZSBJbnRlcmFjdC47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xNC0yMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBwaG9zcGhhdGUgdHJhbnNwb3J0ZXIgZXhwcmVzc2VkIGluIGFyYnVzY3VsZS1jb250YWluaW5nIGNlbGxzIGluIHBvdGF0bzsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTQxNDsgIGNpdGF0aW9uX3BhZ2VzPTQ2Mi00NzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJpY2UgcGhvc3BoYXRlIHRyYW5zcG9ydGVycyBpbmNsdWRlIGFuIGV2b2x1dGlvbmFyaWx5IGRpdmVyZ2VudCBnZW5lIHNwZWNpZmljYWxseSBhY3RpdmF0ZWQgaW4gYXJidXNjdWxhciBteWNvcnJoaXphbCBzeW1iaW9zaXM7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTk5OyAgY2l0YXRpb25fcGFnZXM9MTMzMjQtMTMzMjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlZmVuc2UtcmVsYXRlZCB0cmFuc2NyaXB0IGFjY3VtdWxhdGlvbiBpbiBQaGFzZW9sdXMgdnVsZ2FyaXMgTC4gY29sb25pemVkIGJ5IHRoZSBhcmJ1c2N1bGFyIG15Y29ycmhpemFsIGZ1bmd1cyBHbG9tdXMgaW50cmFyYWRpY2VzIFNjaGVuY2sgJmFtcDsgU21pdGg7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTExMDsgIGNpdGF0aW9uX3BhZ2VzPTY3NS02ODg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cHJlc3Npb24gcGF0dGVybnMgb2YgZGVmZW5zZS1yZWxhdGVkIGdlbmVzIGluIGRpZmZlcmVudCB0eXBlcyBvZiBhcmJ1c2N1bGFyIG15Y29ycmhpemFsIGRldmVsb3BtZW50IGluIHdpbGQtdHlwZSBhbmQgbXljb3JyaGl6YS1kZWZlY3RpdmUgbXV0YW50IHRvbWF0bzsgTW9sLiBQbGFudCBNaWNyb2JlIEludGVyYWN0LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTExMDMtMTExMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNvZmxhdm9ub2lkIGFjY3VtdWxhdGlvbiBhbmQgZXhwcmVzc2lvbiBvZiBkZWZlbnNlIGdlbmUgdHJhbnNjcmlwdHMgZHVyaW5nIHRoZSBlc3RhYmxpc2htZW50IG9mIHZlc2ljdWxhci1hcmJ1c2N1bGFyIG15Y29ycmhpemFsIGFzc29jaWF0aW9ucyBpbiByb290cyBvZiBNZWRpY2FnbyB0cnVuY2F0dWxhOyBNb2wuIFBsYW50IE1pY3JvYmUgSW50ZXJhY3QuOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz02NDMtNjU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TcGF0aWFsIHBhdHRlcm5zIG9mIGV4cHJlc3Npb24gb2YgZmxhdm9ub2lkL2lzb2ZsYXZvbm9pZCBwYXRod2F5IGdlbmVzIGR1cmluZyBpbnRlcmFjdGlvbnMgYmV0d2VlbiByb290cyBvZiBNZWRpY2FnbyB0cnVuY2F0dWxhIGFuZCB0aGUgbXljb3JyaGl6YWwgZnVuZ3VzIEdsb211cyB2ZXJzaWZvcm1lOyBQbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9OS0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSB2ZXNpY3VsYXIgYXJidXNjdWxhciBteWNvcnJoaXphbCBmdW5ndXMgKCBHbG9tdXMgaW50cmFyYWRpeCApIGluZHVjZXMgYSBkZWZlbnNlIHJlc3BvbnNlIGluIGFsZmFsZmEgcm9vdHM7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTEwNDsgIGNpdGF0aW9uX3BhZ2VzPTY4My02ODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN1cHByZXNzaW9uIG9mIGFuIGlzb2ZsYXZvbm9pZCBwaHl0b2FsZXhpbiBkZWZlbnNlIHJlc3BvbnNlIGluIG1pY29ycmhpemFsIGFsZmFsZmEgcm9vdHM7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTEwODsgIGNpdGF0aW9uX3BhZ2VzPTE0NDktMTQ1NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBnZW5lIGVuY29kaW5nIGEgcmVjZXB0b3ItbGlrZSBwcm90ZWluIGtpbmFzZSBpbiB0aGUgcm9vdHMgb2YgY29tbW9uIGJlYW4gaXMgZGlmZmVyZW50aWFsbHkgcmVndWxhdGVkIGluIHJlc3BvbnNlIHRvIHBhdGhvZ2Vucywgc3ltYmlvbnRzIGFuZCBub2R1bGF0aW9uIGZhY3RvcnM7IFBsYW50IFNjaS47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTQyOyAgY2l0YXRpb25fcGFnZXM9MTMzLTE0NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm92ZWwgcmVjZXB0b3ItbGlrZSBwcm90ZWluIGtpbmFzZXMgaW5kdWNlZCBieSBFcndpbmlhIGNhcm90b3ZvcmEgYW5kIHNob3J0IG9saWdvZ2FsYWN0dXJvbmlkZXMgaW4gcG90YXRvOyBNb2wuIFBsYW50IFBhdGhvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTMzOS0zNDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vZGVzIG9mIGV4cHJlc3Npb24gYW5kIGNvbW1vbiBzdHJ1Y3R1cmFsIGZlYXR1cmVzIG9mIHRoZSBjb21wbGV0ZSBwaGVueWxhbGFuaW5lIGFtbW9uaWEtbHlhc2UgZ2VuZSBmYW1pbHkgaW4gcGFyc2xleTsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9OTI7ICBjaXRhdGlvbl9wYWdlcz01OTA1LTU5MDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdSS1kgdHJhbnNjcmlwdGlvbiBmYWN0b3JzOiBmcm9tIEROQSBiaW5kaW5nIHRvd2FyZHMgYmlvbG9naWNhbCBmdW5jdGlvbjsgQ3Vyci4gT3Bpbi4gUGxhbnQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTQ5MS00OTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBwbGFudCBob3Jtb25lIGFic2Npc2ljIGFjaWQgbWVkaWF0ZXMgdGhlIGRyb3VnaHQtaW5kdWNlZCBleHByZXNzaW9uIGJ1dCBub3QgdGhlIHNlZWQtc3BlY2lmaWMgZXhwcmVzc2lvbiBvZiByZDIyLCBhIGdlbmUgcmVzcG9uc2l2ZSB0byBkZWh5ZHJhdGlvbiBzdHJlc3MgaW4gQXJhYmlkb3BzaXMgdGhhbGlhbmE7IE1vbC4gR2VuLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9MjM4OyAgY2l0YXRpb25fcGFnZXM9MTctMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpZmZlcmVudGlhbCBleHByZXNzaW9uIG9mIHBoZW55bGFsYW5pbmUgYW1tb25pYS1seWFzZSBhbmQgY2hhbGNvbmUgc3ludGhhc2UgZHVyaW5nIHNveWJlYW4gbm9kdWxlIGRldmVsb3BtZW50OyBQbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0yOTktMzA4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTQvMy8xMTcvNjkzNTU5MC9kc20wMTQucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gICBUbyBiZXR0ZXIgdW5kZXJzdGFuZCB0aGUgbW9sZWN1bGFyIHJlc3BvbnNlcyBvZiBwbGFudHMgdG8gYXJidXNjdWxhciBteWNvcnJoaXphbCAoQU0pIGZ1bmdpLCB3ZSBhbmFseXplZCB0aGUgZGlmZmVyZW50aWFsIGdlbmUgZXhwcmVzc2lvbiBwYXR0ZXJucyBvIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj01NzgwMjImeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkQz4pUXe7wFWFMfzsNhIK0uRUckc=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f3%2f117%2f578022" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f3%2f117%2f578022" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=23256" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/14/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/3/0/m_dnaresearch14_3.cover.gif?Expires=2147483647&amp;Signature=mMLx1qgQSoxMydWe0ZBNyatOi81iPNGZN59-N2j3ZGCXiGW-PJJB~9AMR~YJK1cFo0ovCThllgKwjHX5Pc7ByoVIVbi1~Iv8mmATs9QVv8pgbsrsKYz5hhgoFSvRFGlC9xxuUx6e9Y0Tu2NeIdWd0xEz5vCYEYm8xUeqKPc-Q~pmQayp6hW3k2w49NPmJ42Y3TDTDgg5VwhqQ3sy8I7I2puWhM9rZLgZqfKecCudhTesr6JCWg~PZNDKXI7m6emAJSOOih0oSvh-42LKbNQTeC3MXoOJjyLTVDQhjAQFJqCmheHjPu7UzqjeQD8L~eRkuwc8WKaIAVAMF4eV05GR0w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/14/3">
                        <div class="volume trailing-comma">Volume 14</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
2007        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="37283977">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#37283977">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="37283979">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#37283979">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="37283984">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#37283984">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="37284004">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#37284004">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="37284031">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#37284031">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="37284033">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#37284033">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="37284035">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#37284035">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/14/3/103/577859">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/14/3/135/577619">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main"> Transcriptome Profiling of <em>Lotus japonicus</em> Roots During Arbuscular Mycorrhiza Development and Comparison with that of Nodulation 
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuichi Deguchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Biostudies</div> , <div class="insititution">Kyoto University</div> , <div class="addr-line">Sakyo-ku, Kyoto 606-8502</div> , <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuichi+Deguchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Deguchi Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Deguchi Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yuichi Deguchi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mari Banba</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Biostudies</div> , <div class="insititution">Kyoto University</div> , <div class="addr-line">Sakyo-ku, Kyoto 606-8502</div> , <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mari+Banba">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Banba M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Banba M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Mari Banba</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshikazu Shimoda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Graduate School of Science and Technology</div> , <div class="insititution">Kagoshima University</div> , <div class="addr-line">Kagoshima 890-0065</div> , <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshikazu+Shimoda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shimoda Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shimoda Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yoshikazu Shimoda</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Svetlana A. Chechetka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Biostudies</div> , <div class="insititution">Kyoto University</div> , <div class="addr-line">Sakyo-ku, Kyoto 606-8502</div> , <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Svetlana+A.+Chechetka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chechetka S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chechetka S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Svetlana A. Chechetka</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ryota Suzuri</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Biostudies</div> , <div class="insititution">Kyoto University</div> , <div class="addr-line">Sakyo-ku, Kyoto 606-8502</div> , <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ryota+Suzuri">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Suzuri R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Suzuri R&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Ryota Suzuri</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yasuhiro Okusako</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Biostudies</div> , <div class="insititution">Kyoto University</div> , <div class="addr-line">Sakyo-ku, Kyoto 606-8502</div> , <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yasuhiro+Okusako">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Okusako Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Okusako Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yasuhiro Okusako</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yasuhiro Ooki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Biostudies</div> , <div class="insititution">Kyoto University</div> , <div class="addr-line">Sakyo-ku, Kyoto 606-8502</div> , <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yasuhiro+Ooki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ooki Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ooki Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yasuhiro Ooki</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Koichi Toyokura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Biostudies</div> , <div class="insititution">Kyoto University</div> , <div class="addr-line">Sakyo-ku, Kyoto 606-8502</div> , <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Koichi+Toyokura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Toyokura K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Toyokura K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Koichi Toyokura</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akihiro Suzuki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Chemistry and BioScience</div> , <div class="insititution">Kagoshima University</div> , <div class="addr-line">Kagoshima 890-0065</div> , <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akihiro+Suzuki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Suzuki A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Suzuki A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Akihiro Suzuki</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Toshiki Uchiumi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Chemistry and BioScience</div> , <div class="insititution">Kagoshima University</div> , <div class="addr-line">Kagoshima 890-0065</div> , <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Toshiki+Uchiumi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Uchiumi T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Uchiumi T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Toshiki Uchiumi</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shiro Higashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Chemistry and BioScience</div> , <div class="insititution">Kagoshima University</div> , <div class="addr-line">Kagoshima 890-0065</div> , <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shiro+Higashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Higashi S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Higashi S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Shiro Higashi</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mikiko Abe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Chemistry and BioScience</div> , <div class="insititution">Kagoshima University</div> , <div class="addr-line">Kagoshima 890-0065</div> , <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mikiko+Abe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Abe M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Abe M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Mikiko Abe</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroshi Kouchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">National Institute of Agrobiological Sciences</div> , <div class="addr-line">Tsukuba, Ibaraki 305-8602</div> , <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroshi+Kouchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kouchi H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kouchi H&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Hiroshi Kouchi</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Katsura Izui</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Biostudies</div> , <div class="insititution">Kyoto University</div> , <div class="addr-line">Sakyo-ku, Kyoto 606-8502</div> , <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Katsura+Izui">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Izui K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Izui K&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Katsura Izui</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shingo Hata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Biostudies</div> , <div class="insititution">Kyoto University</div> , <div class="addr-line">Sakyo-ku, Kyoto 606-8502</div> , <div class="country">Japan</div></div><div class="aff"><div class="label -label">5</div><div class="insititution">Core Research for Evolutional Science and Technology, Japan Science and Technology Agency</div> , <div class="addr-line">Saitama 332-0012</div> , <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div> To whom correspondence should be addressed. <div class="addr-line">Graduate School of Biostudies</div> , <div class="insititution">Kyoto University</div> , <div class="addr-line">Sakyo-ku, Kyoto 606-8502</div> , <div class="country">Japan</div> . Tel. +81 75-753-6141 . Fax. <div class="fax">+81 75-753-6470</div> . E-mail: <a href="mailto:shing@kais.kyoto-u.ac.jp" target="_blank">shing@kais.kyoto-u.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shingo+Hata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hata S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hata S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Shingo Hata</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Satoshi Tabata</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <label class="label label--footnote">â€ </label><div content-id="FN1"><div class="fn"><div class="footnote-content">Current address: Friedrich Miescher Institute, Maulbeerstrasse 66, CH-4058 Basel, Switzerland.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <label class="label label--footnote">â€¡</label><div content-id="FN2"><div class="fn"><div class="footnote-content">Current address: National Institute of Agrobiological Sciences, Tsukuba, Ibaraki 305-8602, Japan.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <label class="label label--footnote">Â¶</label><div content-id="FN3"><div class="fn"><div class="footnote-content">Current address: Kazusa DNA Research Institute, Chiba 292-0812, Japan.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <label class="label label--footnote">Â§</label><div content-id="FN4"><div class="fn"><div class="footnote-content">Current address: Faculty of Agriculture, Saga University, Saga 840-8502, Japan.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <label class="label label--footnote">â€–</label><div content-id="FN5"><div class="fn"><div class="footnote-content">Current address: Department of Biotechnological Science, Kinki University, Wakayama 649-6493, Japan.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 14, Issue 3, 1 January 2007, Pages 117â€“133, <a href='https://doi.org/10.1093/dnares/dsm014'>https://doi.org/10.1093/dnares/dsm014</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">01 June 2007</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">17 January 2007</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">22 June 2007</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="578022" href="/dnaresearch/article-pdf/14/3/117/6935590/dsm014.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Yuichi Deguchi, Mari Banba, Yoshikazu Shimoda, Svetlana A. Chechetka, Ryota Suzuri, Yasuhiro Okusako, Yasuhiro Ooki, Koichi Toyokura, Akihiro Suzuki, Toshiki Uchiumi, Shiro Higashi, Mikiko Abe, Hiroshi Kouchi, Katsura Izui, Shingo Hata;   Transcriptome Profiling of <em>Lotus japonicus</em> Roots During Arbuscular Mycorrhiza Development and Comparison with that of Nodulation , <em>DNA Research</em>, Volume 14, Issue 3, 1 January 2007, Pages 117â€“133, <a href='https://doi.org/10.1093/dnares/dsm014'>https://doi.org/10.1093/dnares/dsm014</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/578022?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/578022?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/578022?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/578022?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/578022?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/578022?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/578022?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=%20Transcriptome%20Profiling%20of%20%26lt%3Bem%26gt%3BLotus%20japonicus%26lt%3B%2Fem%26gt%3B%20Roots%20During%20Arbuscular%20Mycorrhiza%20Development%20and%20Comparison%20with%20that%20of%20Nodulation%20&amp;publicationDate=2007-06-01&amp;volumeNum=14&amp;issueNum=3&amp;author=Deguchi%2C%20Yuichi%3B%20Banba%2C%20Mari&amp;startPage=117&amp;endPage=133&amp;contentId=10.1093%2Fdnares%2Fdsm014&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f3%2f117%2f578022" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=37283977 id="37283977" class="abstract-title" >Abstract</h2>
<section class="abstract"><p> To better understand the molecular responses of plants to arbuscular mycorrhizal (AM) fungi, we analyzed the differential gene expression patterns of <em>Lotus japonicus</em> , a model legume, with the aid of a large-scale cDNA macroarray. Experiments were carried out considering the effects of contaminating microorganisms in the soil inoculants. When the colonization by AM fungi, i.e. <em>Glomus mosseae</em> and <em>Gigaspora margarita</em> , was well established, four cysteine protease genes were induced. <em>In situ</em> hybridization revealed that these cysteine protease genes were specifically expressed in arbuscule-containing inner cortical cells of AM roots. On the other hand, phenylpropanoid biosynthesis-related genes for phenylalanine ammonia-lyase (PAL), chalcone synthase, etc. were repressed in the later stage, although they were moderately up-regulated on the initial association with the AM fungus. Real-time RTâ€“PCR experiments supported the array experiments. To further confirm the characteristic expression, a <em>PAL</em> promoter was fused with a reporter gene and introduced into <em>L. japonicus</em> , and then the transformants were grown with a commercial inoculum of <em>G. mosseae</em> . The reporter activity was augmented throughout the roots due to the presence of contaminating microorganisms in the inoculum. Interestingly, <em>G. mosseae</em> only colonized where the reporter activity was low. Comparison of the transcriptome profiles of AM roots and nitrogen-fixing root nodules formed with <em>Mesorhizobium loti</em> indicated that the PAL genes and other phenylpropanoid biosynthesis-related genes were similarly repressed in the two organs. </p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;cysteine proteinase&quot;">cysteine proteinase</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;defense response&quot;">defense response</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;phenylalanine ammonia-lyase&quot;">phenylalanine ammonia-lyase</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="symbiosis">symbiosis</a></div>                            <h2 scrollto-destination=37283979 id="37283979" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p> Arbuscular mycorrhizal (AM) fungi of the phylum <em>Glomeromycota</em><sup><a reveal-id="DSM014C1" data-open="DSM014C1" class="link link-ref link-reveal xref-bibr">1</a></sup> establish ecologically important symbiotic associations with the majority of land-plant species, allowing improved uptake of phosphate and other nutrients from the soil in exchange for plant-assimilated carbohydrates. <sup><a reveal-id="DSM014C2" data-open="DSM014C2" class="link link-ref link-reveal xref-bibr">2</a> , <a reveal-id="DSM014C3" data-open="DSM014C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Additionally, AM fungi endow plants with tolerance to pathogens and abiotic stress. <sup><a reveal-id="DSM014C4" data-open="DSM014C4" class="link link-ref link-reveal xref-bibr">4</a> , <a reveal-id="DSM014C5" data-open="DSM014C5" class="link link-ref link-reveal xref-bibr">5</a></sup> In the process of colonization by AM fungi, the hyphae of extraradical mycelia branch near the host roots and form appressoria on the root surface, from which hyphae penetrate the epidermis and grow inter- and intracellularly in the root cortex. In the case of <em>Arum</em> -type AM (as formed in <em>Lotus japonicus</em> by <em>Glomus mosseae</em> or <em>Gigaspora margarita</em> ), the hyphae of intraradical mycelia form arbuscules, which are highly branched structures thought to be the main site of nutrient exchange between the two symbiotic partners. <sup><a reveal-id="DSM014C2" data-open="DSM014C2" class="link link-ref link-reveal xref-bibr">2</a> , <a reveal-id="DSM014C3" data-open="DSM014C3" class="link link-ref link-reveal xref-bibr">3</a> , <a reveal-id="DSM014C6" data-open="DSM014C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Early land-plant fossils contain structures that appear similar to arbuscules, suggesting the important role of AM fungi in the colonization of land by plants. <sup><a reveal-id="DSM014C7  DSM014C9" data-open="DSM014C7  DSM014C9" class="link link-ref link-reveal xref-bibr">7â€“9</a></sup></p><p> In addition to AM symbiosis, leguminous plants establish a better-characterized symbiotic association with rhizobia, forming nitrogen-fixing root nodules. Recent molecular and genetic data suggest that the mechanism governing nodule formation evolved from that of AM symbiosis over time. <sup><a reveal-id="DSM014C10" data-open="DSM014C10" class="link link-ref link-reveal xref-bibr">10</a> , <a reveal-id="DSM014C11" data-open="DSM014C11" class="link link-ref link-reveal xref-bibr">11</a></sup></p><p> The development of AM symbiosis is generally thought to accompany complex signal perception and transduction, but the understanding of the latter at the molecular level is very limited, mainly because AM fungi are obligate symbionts and the leading model plant <em>Arabidopsis thaliana</em> does not form AM roots. For a better understanding, in silico data mining, <sup><a reveal-id="DSM014C12" data-open="DSM014C12" class="link link-ref link-reveal xref-bibr">12</a></sup> the subtractive hybridization approach, <sup><a reveal-id="DSM014C13  DSM014C15" data-open="DSM014C13  DSM014C15" class="link link-ref link-reveal xref-bibr">13â€“15</a></sup> and cDNA and oligonucleotide array analyses <sup><a reveal-id="DSM014C16  DSM014C20" data-open="DSM014C16  DSM014C20" class="link link-ref link-reveal xref-bibr">16â€“20</a></sup> have been performed for <em>Medicago truncatula</em> , a model legume. <sup><a reveal-id="DSM014C21" data-open="DSM014C21" class="link link-ref link-reveal xref-bibr">21</a> , <a reveal-id="DSM014C22" data-open="DSM014C22" class="link link-ref link-reveal xref-bibr">22</a></sup><em>Medicago truncatula</em> was also used to investigate the differential expression of chitinase genes in AM colonization, nodulation, and plantâ€“pathogen interactions. <sup><a reveal-id="DSM014C20" data-open="DSM014C20" class="link link-ref link-reveal xref-bibr">20</a> , <a reveal-id="DSM014C23" data-open="DSM014C23" class="link link-ref link-reveal xref-bibr">23</a></sup><em>Lotus japonicus</em> is another valuable model legume. <sup><a reveal-id="DSM014C24" data-open="DSM014C24" class="link link-ref link-reveal xref-bibr">24</a></sup> For example, <em>L. japonicus</em> has been used for elucidation of the molecular mechanisms of plantâ€“AM fungi interactions. <sup><a reveal-id="DSM014C25" data-open="DSM014C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Gene expression profiling with the aid of cDNA-amplified fragment length polymorphism has also been carried out. <sup><a reveal-id="DSM014C10" data-open="DSM014C10" class="link link-ref link-reveal xref-bibr">10</a></sup></p><p> For transcriptome analyses of host responses to AM fungi, we here made use of a large-scale cDNA array of <em>L. japonicus</em> , <sup><a reveal-id="DSM014C26" data-open="DSM014C26" class="link link-ref link-reveal xref-bibr">26</a> , <a reveal-id="DSM014C27" data-open="DSM014C27" class="link link-ref link-reveal xref-bibr">27</a></sup> carefully eliminating the effects of contaminating microorganisms in the soil inoculants. We compared the results with a gene expression profile of root-nodule formation with <em>Mesorhizobium loti</em> , finding a number of genes commonly regulated during AM symbiosis and nodule formation. </p>                            <h2 scrollto-destination=37283984 id="37283984" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=37283985 id="37283985" class="section-title" data-legacy-id=s2a>2.1. Plant material and microorganisms</h3>
<p><em>Lotus japonicus</em> Gifu B-129 seeds were scarified, surface-sterilized with 1% NaClO, rinsed eight times with sterile water, and then spread on 0.7% water agar plates for germination. The plates were placed for a week in a controlled-environment growth chamber (Sanyo, Tokyo, Japan) with a 16-h-day and 8-h-night cycle at 25Â°C, and a light intensity of 260 ÂµEs <sup>âˆ’1</sup> m <sup>âˆ’2</sup> with 60% humidity. </p><p> For AM colonization, glass tubes (30 mm diameter Ã— 120 mm length) containing 55 mL of vermiculite supplemented with 45 mL of modified Hornum nutrient solution were autoclaved before transferring the seedlings. The concentration of phosphate was reduced from 640 ÂµM <sup>24</sup> to 250 ÂµM to facilitate the colonization. The soil inoculant of <em>G. mosseae</em> (2 g/tube; a gift from K. Nagashima, Idemitsu Kosan, Tokyo, Japan) was suspended in sterilized water and then added to the tubes. For the control plants, the <em>G. mosseae</em> inoculant suspension was filtered through a 38 Âµm stainless mesh and the filtrate was added to the tubes. The resulting sieved carrier was free of <em>G. mosseae</em> spores but contaminated by microorganisms equivalent to those in the whole inoculum suspension. The seedlings were grown for up to 8 weeks in a growth chamber with occasional irrigation with the modified Hornum solution. For inoculation of <em>G. margarita</em> (Central Glass Co., Tokyo, Japan), large spores were picked up with forceps under a stereomicroscope, surface-treated with 0.1% NaClO for 7 min, and then rinsed five times with sterilized water. The seedlings were inoculated with the spores and grown as above. The control plants were mock-inoculated with the final rinse and then allowed to grow further. Assessment of AM colonization was carried out by the gridline intersect method <sup><a reveal-id="DSM014C28" data-open="DSM014C28" class="link link-ref link-reveal xref-bibr">28</a></sup> after staining with trypan blue. <sup><a reveal-id="DSM014C29" data-open="DSM014C29" class="link link-ref link-reveal xref-bibr">29</a></sup></p><p> When the initial stage of AM symbiosis was examined, we modified the â€˜nurse potâ€™ method, <sup><a reveal-id="DSM014C30" data-open="DSM014C30" class="link link-ref link-reveal xref-bibr">30</a></sup> as follows. Giant spores of <em>G. margarita</em> were picked up with forceps from a commercial inoculum (Central Glass Co., Tokyo, Japan), surface-treated with 0.1% NaClO, and then rinsed with sterile water. Three sterile <em>L. japonicus</em> seedlings (1-week-old) were inoculated with 500 spores in an autoclaved plastic container (11 cm diameter Ã— 16 cm height; Takeya Chemical Co., Osaka, Japan) with a lid and then allowed to grow further. As a non-inoculated control, the final rinse of the sterilized spores was applied to sterile seedlings in another container, followed by further growth. After 2 months, freshly prepared sterile seedlings (2-week-old) were transplanted into the containers and then allowed to grow for a week. Then, roots of the younger plants were harvested from the container inoculated with <em>G. margarita</em> or the mock-inoculated container. </p><p> For root-nodule formation, <em>L. japonicus</em> seedlings were inoculated with <em>M. loti</em> Tono and then grown for 2 weeks on vermiculite supplied with nitrogen-free Broughton and Dilworth medium as described previously. <sup><a reveal-id="DSM014C31" data-open="DSM014C31" class="link link-ref link-reveal xref-bibr">31</a> , <a reveal-id="DSM014C32" data-open="DSM014C32" class="link link-ref link-reveal xref-bibr">32</a></sup> The resulting young nodules were harvested. </p>                            <h3 scrollto-destination=37283990 id="37283990" class="section-title" data-legacy-id=s2b>2.2. cDNA array analysis</h3>
<p> Total RNA was extracted from AM roots, root nodules, or control roots using an RNeasy Plant Mini-Kit (Qiagen, Hilden, Germany). Labeling of target cDNA, hybridization of a large-scale nylon filter array with the target, washing of membranes under high-stringency conditions, detection of radioactive images, and data mining were all carried out as described previously. <sup><a reveal-id="DSM014C26" data-open="DSM014C26" class="link link-ref link-reveal xref-bibr">26</a></sup></p>                            <h3 scrollto-destination=37283992 id="37283992" class="section-title" data-legacy-id=s2c>2.3. Real-time RTâ€“PCR analysis</h3>
<p> After treating the total RNA preparation with DNase, reverse transcription was performed with oligo(dT) and Superscript II (Invitrogen, Carlsbad, CA). Real-time PCR with a real-time RTâ€“PCR Core Kit (Takara Bio, Otsu, Japan) and a Smart Cycler system (Cepheid, Sunnyvale, CA) was carried out as described previously. <sup><a reveal-id="DSM014C32" data-open="DSM014C32" class="link link-ref link-reveal xref-bibr">32</a> , <a reveal-id="DSM014C33" data-open="DSM014C33" class="link link-ref link-reveal xref-bibr">33</a></sup> The forward and reverse primer sets and annealing temperatures (in parentheses) were as follows: 5â€²-CAGTGACAAAAGGTTTGGACCTAC-3â€² and 5â€²-ATGCAGAGAGATGTTGCTGCTG-3â€² (68Â°C) for <em>LjCyp2</em> ; 5â€²-AACTTTATTAGTAACTTTTAG-3â€² and 5â€²-CTTTCACATCCGAGGAAATTG-3â€² (55Â°C) for <em>LjPAL1</em> ; 5â€²-GCTCAGGTGGCTGCCATCGCC-3â€² and 5â€²-GGCAGTGTGTGGTTTGTCTCG-3â€² (55Â°C) for <em>LjPAL2</em> ; 5â€²-AACTTTACTAG TTTCTTCAGG-3â€² and 5â€²-TAATTCCATATTCCGCAAATT-3â€² (55Â°C) for <em>LjPAL3</em> ; 5â€²-GAATGCAGATCTTACCCGCTA-3â€² and 5â€²-TTTGCTTAAATACAAAGAATG-3â€² (50Â°C) for <em>LjPAL4</em> ; 5â€²-GAATGCAGATCTTACCCGCTG-3â€² and 5â€²-ATTGCATTTGCATAAATACAG-3â€² (50Â°C) for <em>LjPAL5</em> ; 5â€²-AACTTAACCATTTATTTTTTT-3â€² and 5â€²-TTGTAATGTAATGTGAGATGG-3â€² (55Â°C) for <em>LjPAL6</em> ; 5â€²-TTGGCTAGCATCGATTCAGGA-3â€² and 5â€²-GTCCAGGGTGGTGCTTAAGCC-3â€² (50Â°C) for <em>LjPAL7</em> ; 5â€²-GCTCAGGTGGCTGCCATCGCA-3â€² and 5â€²-GGCAGGGTGTGAGTTGATTCA-3â€² (55Â°C) for <em>LjPAL8</em> ; 5â€²-AACTTGCCTGCCAGTTATGTT-3â€² and 5â€²-CTCTTGTGTTTTTCTGTAGTG-3â€² (55Â°C) for <em>LjPAL9</em> ; and 5â€²-AGAACAGTTTGTTTGTTTGAG-3â€² and 5â€²-CATAAAGGAGAACTTAAAGGA-3â€² (55Â°C) for <em>LjPAL10</em> . Amplification of the Î²-actin gene was carried out as described previously. <sup><a reveal-id="DSM014C33" data-open="DSM014C33" class="link link-ref link-reveal xref-bibr">33</a></sup> A single amplicon of expected size, 100â€“300 bp, with each primer set was observed on agarose gel electrophoresis, irrespective of whether the reverse-transcribed template was from AM roots or control roots. In order to calculate the transcript level ratios, it was assumed that each PCR cycle results in exact doubling of the amounts of amplicons. </p>                            <h3 scrollto-destination=37283994 id="37283994" class="section-title" data-legacy-id=s2d>2.4. In situ hybridization</h3>
<p><em>In situ</em> hybridization of paraffin-embedded sections was carried out as described previously. <sup><a reveal-id="DSM014C31" data-open="DSM014C31" class="link link-ref link-reveal xref-bibr">31</a> , <a reveal-id="DSM014C33" data-open="DSM014C33" class="link link-ref link-reveal xref-bibr">33</a> , <a reveal-id="DSM014C34" data-open="DSM014C34" class="link link-ref link-reveal xref-bibr">34</a></sup></p>                            <h3 scrollto-destination=37283996 id="37283996" class="section-title" data-legacy-id=s2e>2.5. Promoter-Î²-glucuronidase construction, hairy root transformation and histochemical analysis of L. japonicus</h3>
<p> The 2 kb 5â€² flanking region of <em>LjPAL1</em> contains a <em>Bam</em> HI site. Therefore, to amplify the region derived from genomic DNA of <em>L. japonicus</em> , forward primer 5â€²-ATGCGGCCGCTGACCGACAATGGTTTATGAAC TAGCC-3â€² and reverse primer 5â€²-ATTGATCACTTAGTATATATGATCTCTCACTTACA-3â€², containing <em>Not</em> I and <em>BclI</em> sites, respectively, were used for PCR. The <em>BclI</em> end of the promoter was ligated to the <em>Bam</em> HI site 24 bp upstream of the coding sequence of the <em>uidA</em> gene for the Î²-glucuronidase (GUS) reporter with a nopaline synthase terminator. Then, making use of the <em>SalI</em> sites at the ends of the intermediate construct, the promoter-GUS unit was ligated into the <em>Sal</em> I site of pHKN29, <sup><a reveal-id="DSM014C35" data-open="DSM014C35" class="link link-ref link-reveal xref-bibr">35</a></sup> which is a derivative of pCAMBIA 1300 (CAMBIA, Canberra, Australia). </p><p> Hairy root transformation with <em>Agrobacterium rhizogenes</em> LBA 1334 was performed following the protocol of Diaz and Schlaman, Leiden University, as described previously. <sup><a reveal-id="DSM014C33" data-open="DSM014C33" class="link link-ref link-reveal xref-bibr">33</a> , <a reveal-id="DSM014C35" data-open="DSM014C35" class="link link-ref link-reveal xref-bibr">35</a></sup> Transformants with green fluorescent protein (GFP)-positive hairy roots were transferred to vermiculite containing the modified Hornum solution, inoculated with the entire <em>G. mosseae</em> inoculum or sieved carrier, and then grown as described above. When nodule formation was examined, the transformants were transferred to nitrogen-free Broughton and Dilworth medium and then inoculated with <em>M. loti</em> Tono. </p><p> Detached roots were stained with 5-bromo-4-chloro-3-indolyl-Î²- <span class="small-caps">d</span> -glucuronide, and then the reaction was stopped with 75% ethanol as described previously. <sup><a reveal-id="DSM014C35" data-open="DSM014C35" class="link link-ref link-reveal xref-bibr">35</a></sup> When AM fungi were re-stained, the roots were immersed in 0.02% safranin and then observed under a stereomicroscope. Quantitative assaying of GUS activity in hairy roots was performed as described previously, <sup><a reveal-id="DSM014C35" data-open="DSM014C35" class="link link-ref link-reveal xref-bibr">35</a></sup> based on the method of Jefferson <em>et al.</em><sup><a reveal-id="DSM014C36" data-open="DSM014C36" class="link link-ref link-reveal xref-bibr">36</a></sup> GUS-stained roots were also fixed in 4% paraformaldehyde and 0.25% glutaraldehyde in 50 mM Na-phosphate buffer (pH 7.2), washed with Na-phosphate buffer, dehydrated in an ascending ethanol series (10, 30, 50, 60, 70, 90, and 100%), immersed in 50% Technovit 7100 (Heraeus Kulzer, Wehrheim, Germany) in ethanol, and then left to stand overnight. Then, they were embedded in Technovit 7100 at room temperature by adding the polymerization agent provided in the kit. Six-micrometer sections were prepared and re-stained with 0.02% safranin when necessary. </p>                            <h3 scrollto-destination=37284000 id="37284000" class="section-title" data-legacy-id=s2f>2.6. Accession numbers</h3>
<p> The entire nucleotide sequences of cDNAs for cysteine proteinases and PALs were determined. The accession numbers for the sequences mentioned in this paper are as follows: AB300459 ( <em>LjCyp1</em> ), AB300460 ( <em>LjCyp2</em> ), AB300461 ( <em>LjCyp3</em> ), AB300462 ( <em>LjCyp4</em> ), AB283031 ( <em>LjPAL1</em> ), AB283032 ( <em>LjPAL2</em> ), AB283033 ( <em>LjPAL3</em> ), AB283034 ( <em>LjPAL4</em> ), AB283035 ( <em>LjPAL5</em> ), AB283036 ( <em>LjPAL6</em> ), AB283037 ( <em>LjPAL7</em> ), AB283038 ( <em>LjPAL8</em> ), AB283039 ( <em>LjPAL9</em> ), and AB283040 ( <em>LjPAL10</em> ). </p>                        <a id="37284002" scrollto-destination="37284002"></a>
<div content-id="DSM014TB1" class="table-modal"><div class="table-wrap"><div id="DSM014TB1" class="table-wrap-title" data-id="DSM014TB1"><div class="label -label"><strong>TableÂ 1</strong></div><div class="caption"><p> Up-regulated genes in <em>L. japonicus</em> roots after establishment of symbiosis with <em>G. mosseae</em> and <em>G. margarita</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Current annotationÂ </th><th>Gm3/SC3Â </th><th>Gm6/SC3Â </th><th>Gi8/gni3Â </th><th>Gene IDÂ </th><th>Max REÂ </th><th>e-valueÂ </th></tr></thead><tbody><tr><td><em>Amino acid and nitrogen metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Asparagine synthetaseÂ </td><td>1.3Â </td><td>4.0Â </td><td>1.7Â </td><td>MWL032c11_rÂ </td><td>499Â </td><td>2Eâˆ’39Â </td></tr><tr><td>Asparagine synthetaseÂ </td><td>1.2Â </td><td>4.0Â </td><td>1.7Â </td><td>GNf053e06Â </td><td>1005Â </td><td>3Eâˆ’49Â </td></tr><tr><td>Asparagine synthetaseÂ </td><td>0.9Â </td><td>3.3Â </td><td>1.8Â </td><td>MWM233f05_rÂ </td><td>460Â </td><td>3Eâˆ’36Â </td></tr><tr><td>Asparagine synthetaseÂ </td><td>1.1Â </td><td>2.4Â </td><td>1.4Â </td><td>GNf021f11Â </td><td>569Â </td><td>4Eâˆ’81Â </td></tr><tr><td><em>Carbon metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Alpha-mannosidaseÂ </td><td>1.4Â </td><td>3.6Â </td><td>2.7Â </td><td>MPDL053f07_fÂ </td><td>217Â </td><td>Â </td></tr><tr><td>Alpha-mannosidaseÂ </td><td>1.3Â </td><td>3.2Â </td><td>1.8Â </td><td>MPDL018e02_fÂ </td><td>232Â </td><td>Â </td></tr><tr><td>Putative alpha-mannosidaseÂ </td><td>1.6Â </td><td>3.1Â </td><td>1.9Â </td><td>MPDL061d01_fÂ </td><td>313Â </td><td>1Eâˆ’09Â </td></tr><tr><td><em>Secondary metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Chalcone reductaseÂ </td><td>1.4Â </td><td>4.1Â </td><td>3.0Â </td><td>GNf040a09Â </td><td>231Â </td><td>6Eâˆ’06Â </td></tr><tr><td>Chalcone reductaseÂ </td><td>0.7Â </td><td>2.1Â </td><td>2.3Â </td><td>GNf086d03Â </td><td>160Â </td><td>5Eâˆ’53Â </td></tr><tr><td> Flavonoid 3- <em>O</em> -galactosyl transferase Â </td><td>1.1Â </td><td>2.0Â </td><td>3.4Â </td><td>MPD011f01_fÂ </td><td>158Â </td><td>2Eâˆ’29Â </td></tr><tr><td><em>Transport/membrane</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Plasmamembrane intrinsic proteinÂ </td><td>2.3Â </td><td>2.8Â </td><td>0.8Â </td><td>MWL070f05_rÂ </td><td>1440Â </td><td>9Eâˆ’56Â </td></tr><tr><td>Putative nitrate transporterÂ </td><td>0.9Â </td><td>2.2Â </td><td>2.6Â </td><td>MWM134h03_rÂ </td><td>88Â </td><td>2Eâˆ’07Â </td></tr><tr><td>Aquaporin1Â </td><td>1.1Â </td><td>2.1Â </td><td>1.1Â </td><td>MWM132a07_rÂ </td><td>208Â </td><td>5Eâˆ’66Â </td></tr><tr><td>Plasma-membrane intrinsic proteinÂ </td><td>2.0Â </td><td>2.1Â </td><td>0.7Â </td><td>MWM091f06_rÂ </td><td>1490Â </td><td>5Eâˆ’13Â </td></tr><tr><td>Tonoplast intrinsic proteinÂ </td><td>2.3Â </td><td>2.1Â </td><td>1.7Â </td><td>MWM074b06_rÂ </td><td>444Â </td><td>2Eâˆ’42Â </td></tr><tr><td><em>Nodulin</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Nodulin 26-like proteinÂ </td><td>1.1Â </td><td>2.7Â </td><td>2.3Â </td><td>MWM104a10_rÂ </td><td>174Â </td><td>5Eâˆ’09Â </td></tr><tr><td>Nodule-enhanced sucrose synthaseÂ </td><td>1.3Â </td><td>2.3Â </td><td>1.2Â </td><td>MWL080e04_rÂ </td><td>797Â </td><td>2Eâˆ’78Â </td></tr><tr><td><em>Signal transduction</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>AnnexinÂ </td><td>1.2Â </td><td>3.2Â </td><td>2.8Â </td><td>MPD097d02_fÂ </td><td>135Â </td><td>3Eâˆ’36Â </td></tr><tr><td>AnnexinÂ </td><td>1.0Â </td><td>2.4Â </td><td>2.5Â </td><td>MPD065b05_fÂ </td><td>215Â </td><td>7Eâˆ’36Â </td></tr><tr><td>AnnexinÂ </td><td>1.0Â </td><td>2.2Â </td><td>2.6Â </td><td>MPD042e01_fÂ </td><td>185Â </td><td>2Eâˆ’62Â </td></tr><tr><td>ANTI-H(O) lectin (LTA)Â </td><td>1.8Â </td><td>2.0Â </td><td>7.0Â </td><td>MWM231h03_rÂ </td><td>24488Â </td><td>4Eâˆ’39Â </td></tr><tr><td><em>Protein fate</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Cysteine proteinase (LjCyp4)Â </td><td>3.8Â </td><td>19.7Â </td><td>8.0Â </td><td>GNf089d01Â </td><td>664Â </td><td>6Eâˆ’40Â </td></tr><tr><td>Cysteine proteinase (LjCyp1)Â </td><td>1.9Â </td><td>10.6Â </td><td>3.3Â </td><td>GNf032f12Â </td><td>251Â </td><td>Â </td></tr><tr><td>Cysteine proteinase (LjCyp2)Â </td><td>1.7Â </td><td>5.0Â </td><td>2.3Â </td><td>GNf037h07Â </td><td>964Â </td><td>6Eâˆ’23Â </td></tr><tr><td>Cysteine proteinase (LjCyp3)Â </td><td>1.3Â </td><td>4.0Â </td><td>2.3Â </td><td>GNf071h01Â </td><td>226Â </td><td>6Eâˆ’12Â </td></tr><tr><td><em>Cell wall</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Yieldin precursorÂ </td><td>1.0Â </td><td>4.3Â </td><td>1.9Â </td><td>MWM140d02_rÂ </td><td>75Â </td><td>3Eâˆ’16Â </td></tr><tr><td><em>Pathogen-related</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Putative disease resistant proteinÂ </td><td>1.6Â </td><td>2.0Â </td><td>1.2Â </td><td>MPDL019h09_fÂ </td><td>980Â </td><td>5Eâˆ’15Â </td></tr><tr><td><em>Phytohormone-related</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Jasmonic acid 2Â </td><td>1.4Â </td><td>6.0Â </td><td>4.6Â </td><td>MWL076b07_rÂ </td><td>149Â </td><td>7Eâˆ’73Â </td></tr><tr><td><em>Other enzyme</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Nicotianamine synthaseÂ </td><td>4.7Â </td><td>5.3Â </td><td>2.1Â </td><td>GNf070f09Â </td><td>316Â </td><td>5Eâˆ’21Â </td></tr><tr><td><em>Other category</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Dehydrin 3Â </td><td>2.2Â </td><td>8.0Â </td><td>6.9Â </td><td>MR001a01_fÂ </td><td>228Â </td><td>0.0002Â </td></tr></tbody></table></div><div content-id="DSM014TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Current annotationÂ </th><th>Gm3/SC3Â </th><th>Gm6/SC3Â </th><th>Gi8/gni3Â </th><th>Gene IDÂ </th><th>Max REÂ </th><th>e-valueÂ </th></tr></thead><tbody><tr><td><em>Amino acid and nitrogen metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Asparagine synthetaseÂ </td><td>1.3Â </td><td>4.0Â </td><td>1.7Â </td><td>MWL032c11_rÂ </td><td>499Â </td><td>2Eâˆ’39Â </td></tr><tr><td>Asparagine synthetaseÂ </td><td>1.2Â </td><td>4.0Â </td><td>1.7Â </td><td>GNf053e06Â </td><td>1005Â </td><td>3Eâˆ’49Â </td></tr><tr><td>Asparagine synthetaseÂ </td><td>0.9Â </td><td>3.3Â </td><td>1.8Â </td><td>MWM233f05_rÂ </td><td>460Â </td><td>3Eâˆ’36Â </td></tr><tr><td>Asparagine synthetaseÂ </td><td>1.1Â </td><td>2.4Â </td><td>1.4Â </td><td>GNf021f11Â </td><td>569Â </td><td>4Eâˆ’81Â </td></tr><tr><td><em>Carbon metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Alpha-mannosidaseÂ </td><td>1.4Â </td><td>3.6Â </td><td>2.7Â </td><td>MPDL053f07_fÂ </td><td>217Â </td><td>Â </td></tr><tr><td>Alpha-mannosidaseÂ </td><td>1.3Â </td><td>3.2Â </td><td>1.8Â </td><td>MPDL018e02_fÂ </td><td>232Â </td><td>Â </td></tr><tr><td>Putative alpha-mannosidaseÂ </td><td>1.6Â </td><td>3.1Â </td><td>1.9Â </td><td>MPDL061d01_fÂ </td><td>313Â </td><td>1Eâˆ’09Â </td></tr><tr><td><em>Secondary metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Chalcone reductaseÂ </td><td>1.4Â </td><td>4.1Â </td><td>3.0Â </td><td>GNf040a09Â </td><td>231Â </td><td>6Eâˆ’06Â </td></tr><tr><td>Chalcone reductaseÂ </td><td>0.7Â </td><td>2.1Â </td><td>2.3Â </td><td>GNf086d03Â </td><td>160Â </td><td>5Eâˆ’53Â </td></tr><tr><td> Flavonoid 3- <em>O</em> -galactosyl transferase Â </td><td>1.1Â </td><td>2.0Â </td><td>3.4Â </td><td>MPD011f01_fÂ </td><td>158Â </td><td>2Eâˆ’29Â </td></tr><tr><td><em>Transport/membrane</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Plasmamembrane intrinsic proteinÂ </td><td>2.3Â </td><td>2.8Â </td><td>0.8Â </td><td>MWL070f05_rÂ </td><td>1440Â </td><td>9Eâˆ’56Â </td></tr><tr><td>Putative nitrate transporterÂ </td><td>0.9Â </td><td>2.2Â </td><td>2.6Â </td><td>MWM134h03_rÂ </td><td>88Â </td><td>2Eâˆ’07Â </td></tr><tr><td>Aquaporin1Â </td><td>1.1Â </td><td>2.1Â </td><td>1.1Â </td><td>MWM132a07_rÂ </td><td>208Â </td><td>5Eâˆ’66Â </td></tr><tr><td>Plasma-membrane intrinsic proteinÂ </td><td>2.0Â </td><td>2.1Â </td><td>0.7Â </td><td>MWM091f06_rÂ </td><td>1490Â </td><td>5Eâˆ’13Â </td></tr><tr><td>Tonoplast intrinsic proteinÂ </td><td>2.3Â </td><td>2.1Â </td><td>1.7Â </td><td>MWM074b06_rÂ </td><td>444Â </td><td>2Eâˆ’42Â </td></tr><tr><td><em>Nodulin</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Nodulin 26-like proteinÂ </td><td>1.1Â </td><td>2.7Â </td><td>2.3Â </td><td>MWM104a10_rÂ </td><td>174Â </td><td>5Eâˆ’09Â </td></tr><tr><td>Nodule-enhanced sucrose synthaseÂ </td><td>1.3Â </td><td>2.3Â </td><td>1.2Â </td><td>MWL080e04_rÂ </td><td>797Â </td><td>2Eâˆ’78Â </td></tr><tr><td><em>Signal transduction</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>AnnexinÂ </td><td>1.2Â </td><td>3.2Â </td><td>2.8Â </td><td>MPD097d02_fÂ </td><td>135Â </td><td>3Eâˆ’36Â </td></tr><tr><td>AnnexinÂ </td><td>1.0Â </td><td>2.4Â </td><td>2.5Â </td><td>MPD065b05_fÂ </td><td>215Â </td><td>7Eâˆ’36Â </td></tr><tr><td>AnnexinÂ </td><td>1.0Â </td><td>2.2Â </td><td>2.6Â </td><td>MPD042e01_fÂ </td><td>185Â </td><td>2Eâˆ’62Â </td></tr><tr><td>ANTI-H(O) lectin (LTA)Â </td><td>1.8Â </td><td>2.0Â </td><td>7.0Â </td><td>MWM231h03_rÂ </td><td>24488Â </td><td>4Eâˆ’39Â </td></tr><tr><td><em>Protein fate</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Cysteine proteinase (LjCyp4)Â </td><td>3.8Â </td><td>19.7Â </td><td>8.0Â </td><td>GNf089d01Â </td><td>664Â </td><td>6Eâˆ’40Â </td></tr><tr><td>Cysteine proteinase (LjCyp1)Â </td><td>1.9Â </td><td>10.6Â </td><td>3.3Â </td><td>GNf032f12Â </td><td>251Â </td><td>Â </td></tr><tr><td>Cysteine proteinase (LjCyp2)Â </td><td>1.7Â </td><td>5.0Â </td><td>2.3Â </td><td>GNf037h07Â </td><td>964Â </td><td>6Eâˆ’23Â </td></tr><tr><td>Cysteine proteinase (LjCyp3)Â </td><td>1.3Â </td><td>4.0Â </td><td>2.3Â </td><td>GNf071h01Â </td><td>226Â </td><td>6Eâˆ’12Â </td></tr><tr><td><em>Cell wall</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Yieldin precursorÂ </td><td>1.0Â </td><td>4.3Â </td><td>1.9Â </td><td>MWM140d02_rÂ </td><td>75Â </td><td>3Eâˆ’16Â </td></tr><tr><td><em>Pathogen-related</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Putative disease resistant proteinÂ </td><td>1.6Â </td><td>2.0Â </td><td>1.2Â </td><td>MPDL019h09_fÂ </td><td>980Â </td><td>5Eâˆ’15Â </td></tr><tr><td><em>Phytohormone-related</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Jasmonic acid 2Â </td><td>1.4Â </td><td>6.0Â </td><td>4.6Â </td><td>MWL076b07_rÂ </td><td>149Â </td><td>7Eâˆ’73Â </td></tr><tr><td><em>Other enzyme</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Nicotianamine synthaseÂ </td><td>4.7Â </td><td>5.3Â </td><td>2.1Â </td><td>GNf070f09Â </td><td>316Â </td><td>5Eâˆ’21Â </td></tr><tr><td><em>Other category</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Dehydrin 3Â </td><td>2.2Â </td><td>8.0Â </td><td>6.9Â </td><td>MR001a01_fÂ </td><td>228Â </td><td>0.0002Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p> The data in the Gm3/SC3 and Gm6/SC3 columns are the gene expression levels in roots 3 and 6 weeks, respectively, after inoculation of the whole <em>G. mosseae</em> inoculum relative to those in control roots 3 weeks after inoculation of the sieved carrier. Since prolonged cultivation with low concentrations of phosphate may cause stress, <sup>18</sup> we do not think that there is any problem with the use of younger control roots. The data in the Gi8/gni3 column are the gene expression levels in roots 8 weeks after inoculation of NaClO-treated <em>G. margarita</em> relative to those in roots 3 weeks after mock-inoculation. MaxRE is the highest normalized expression level in the experiments. Genes that match hypothetical proteins of unknown function and ones that encode proteins exhibiting no homology to thus far known ones have been omitted from this table. </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/37284002" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM014TB1" class="table-wrap-title" data-id="DSM014TB1"><div class="label -label"><strong>TableÂ 1</strong></div><div class="caption"><p> Up-regulated genes in <em>L. japonicus</em> roots after establishment of symbiosis with <em>G. mosseae</em> and <em>G. margarita</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Current annotationÂ </th><th>Gm3/SC3Â </th><th>Gm6/SC3Â </th><th>Gi8/gni3Â </th><th>Gene IDÂ </th><th>Max REÂ </th><th>e-valueÂ </th></tr></thead><tbody><tr><td><em>Amino acid and nitrogen metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Asparagine synthetaseÂ </td><td>1.3Â </td><td>4.0Â </td><td>1.7Â </td><td>MWL032c11_rÂ </td><td>499Â </td><td>2Eâˆ’39Â </td></tr><tr><td>Asparagine synthetaseÂ </td><td>1.2Â </td><td>4.0Â </td><td>1.7Â </td><td>GNf053e06Â </td><td>1005Â </td><td>3Eâˆ’49Â </td></tr><tr><td>Asparagine synthetaseÂ </td><td>0.9Â </td><td>3.3Â </td><td>1.8Â </td><td>MWM233f05_rÂ </td><td>460Â </td><td>3Eâˆ’36Â </td></tr><tr><td>Asparagine synthetaseÂ </td><td>1.1Â </td><td>2.4Â </td><td>1.4Â </td><td>GNf021f11Â </td><td>569Â </td><td>4Eâˆ’81Â </td></tr><tr><td><em>Carbon metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Alpha-mannosidaseÂ </td><td>1.4Â </td><td>3.6Â </td><td>2.7Â </td><td>MPDL053f07_fÂ </td><td>217Â </td><td>Â </td></tr><tr><td>Alpha-mannosidaseÂ </td><td>1.3Â </td><td>3.2Â </td><td>1.8Â </td><td>MPDL018e02_fÂ </td><td>232Â </td><td>Â </td></tr><tr><td>Putative alpha-mannosidaseÂ </td><td>1.6Â </td><td>3.1Â </td><td>1.9Â </td><td>MPDL061d01_fÂ </td><td>313Â </td><td>1Eâˆ’09Â </td></tr><tr><td><em>Secondary metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Chalcone reductaseÂ </td><td>1.4Â </td><td>4.1Â </td><td>3.0Â </td><td>GNf040a09Â </td><td>231Â </td><td>6Eâˆ’06Â </td></tr><tr><td>Chalcone reductaseÂ </td><td>0.7Â </td><td>2.1Â </td><td>2.3Â </td><td>GNf086d03Â </td><td>160Â </td><td>5Eâˆ’53Â </td></tr><tr><td> Flavonoid 3- <em>O</em> -galactosyl transferase Â </td><td>1.1Â </td><td>2.0Â </td><td>3.4Â </td><td>MPD011f01_fÂ </td><td>158Â </td><td>2Eâˆ’29Â </td></tr><tr><td><em>Transport/membrane</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Plasmamembrane intrinsic proteinÂ </td><td>2.3Â </td><td>2.8Â </td><td>0.8Â </td><td>MWL070f05_rÂ </td><td>1440Â </td><td>9Eâˆ’56Â </td></tr><tr><td>Putative nitrate transporterÂ </td><td>0.9Â </td><td>2.2Â </td><td>2.6Â </td><td>MWM134h03_rÂ </td><td>88Â </td><td>2Eâˆ’07Â </td></tr><tr><td>Aquaporin1Â </td><td>1.1Â </td><td>2.1Â </td><td>1.1Â </td><td>MWM132a07_rÂ </td><td>208Â </td><td>5Eâˆ’66Â </td></tr><tr><td>Plasma-membrane intrinsic proteinÂ </td><td>2.0Â </td><td>2.1Â </td><td>0.7Â </td><td>MWM091f06_rÂ </td><td>1490Â </td><td>5Eâˆ’13Â </td></tr><tr><td>Tonoplast intrinsic proteinÂ </td><td>2.3Â </td><td>2.1Â </td><td>1.7Â </td><td>MWM074b06_rÂ </td><td>444Â </td><td>2Eâˆ’42Â </td></tr><tr><td><em>Nodulin</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Nodulin 26-like proteinÂ </td><td>1.1Â </td><td>2.7Â </td><td>2.3Â </td><td>MWM104a10_rÂ </td><td>174Â </td><td>5Eâˆ’09Â </td></tr><tr><td>Nodule-enhanced sucrose synthaseÂ </td><td>1.3Â </td><td>2.3Â </td><td>1.2Â </td><td>MWL080e04_rÂ </td><td>797Â </td><td>2Eâˆ’78Â </td></tr><tr><td><em>Signal transduction</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>AnnexinÂ </td><td>1.2Â </td><td>3.2Â </td><td>2.8Â </td><td>MPD097d02_fÂ </td><td>135Â </td><td>3Eâˆ’36Â </td></tr><tr><td>AnnexinÂ </td><td>1.0Â </td><td>2.4Â </td><td>2.5Â </td><td>MPD065b05_fÂ </td><td>215Â </td><td>7Eâˆ’36Â </td></tr><tr><td>AnnexinÂ </td><td>1.0Â </td><td>2.2Â </td><td>2.6Â </td><td>MPD042e01_fÂ </td><td>185Â </td><td>2Eâˆ’62Â </td></tr><tr><td>ANTI-H(O) lectin (LTA)Â </td><td>1.8Â </td><td>2.0Â </td><td>7.0Â </td><td>MWM231h03_rÂ </td><td>24488Â </td><td>4Eâˆ’39Â </td></tr><tr><td><em>Protein fate</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Cysteine proteinase (LjCyp4)Â </td><td>3.8Â </td><td>19.7Â </td><td>8.0Â </td><td>GNf089d01Â </td><td>664Â </td><td>6Eâˆ’40Â </td></tr><tr><td>Cysteine proteinase (LjCyp1)Â </td><td>1.9Â </td><td>10.6Â </td><td>3.3Â </td><td>GNf032f12Â </td><td>251Â </td><td>Â </td></tr><tr><td>Cysteine proteinase (LjCyp2)Â </td><td>1.7Â </td><td>5.0Â </td><td>2.3Â </td><td>GNf037h07Â </td><td>964Â </td><td>6Eâˆ’23Â </td></tr><tr><td>Cysteine proteinase (LjCyp3)Â </td><td>1.3Â </td><td>4.0Â </td><td>2.3Â </td><td>GNf071h01Â </td><td>226Â </td><td>6Eâˆ’12Â </td></tr><tr><td><em>Cell wall</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Yieldin precursorÂ </td><td>1.0Â </td><td>4.3Â </td><td>1.9Â </td><td>MWM140d02_rÂ </td><td>75Â </td><td>3Eâˆ’16Â </td></tr><tr><td><em>Pathogen-related</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Putative disease resistant proteinÂ </td><td>1.6Â </td><td>2.0Â </td><td>1.2Â </td><td>MPDL019h09_fÂ </td><td>980Â </td><td>5Eâˆ’15Â </td></tr><tr><td><em>Phytohormone-related</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Jasmonic acid 2Â </td><td>1.4Â </td><td>6.0Â </td><td>4.6Â </td><td>MWL076b07_rÂ </td><td>149Â </td><td>7Eâˆ’73Â </td></tr><tr><td><em>Other enzyme</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Nicotianamine synthaseÂ </td><td>4.7Â </td><td>5.3Â </td><td>2.1Â </td><td>GNf070f09Â </td><td>316Â </td><td>5Eâˆ’21Â </td></tr><tr><td><em>Other category</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Dehydrin 3Â </td><td>2.2Â </td><td>8.0Â </td><td>6.9Â </td><td>MR001a01_fÂ </td><td>228Â </td><td>0.0002Â </td></tr></tbody></table></div><div content-id="DSM014TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Current annotationÂ </th><th>Gm3/SC3Â </th><th>Gm6/SC3Â </th><th>Gi8/gni3Â </th><th>Gene IDÂ </th><th>Max REÂ </th><th>e-valueÂ </th></tr></thead><tbody><tr><td><em>Amino acid and nitrogen metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Asparagine synthetaseÂ </td><td>1.3Â </td><td>4.0Â </td><td>1.7Â </td><td>MWL032c11_rÂ </td><td>499Â </td><td>2Eâˆ’39Â </td></tr><tr><td>Asparagine synthetaseÂ </td><td>1.2Â </td><td>4.0Â </td><td>1.7Â </td><td>GNf053e06Â </td><td>1005Â </td><td>3Eâˆ’49Â </td></tr><tr><td>Asparagine synthetaseÂ </td><td>0.9Â </td><td>3.3Â </td><td>1.8Â </td><td>MWM233f05_rÂ </td><td>460Â </td><td>3Eâˆ’36Â </td></tr><tr><td>Asparagine synthetaseÂ </td><td>1.1Â </td><td>2.4Â </td><td>1.4Â </td><td>GNf021f11Â </td><td>569Â </td><td>4Eâˆ’81Â </td></tr><tr><td><em>Carbon metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Alpha-mannosidaseÂ </td><td>1.4Â </td><td>3.6Â </td><td>2.7Â </td><td>MPDL053f07_fÂ </td><td>217Â </td><td>Â </td></tr><tr><td>Alpha-mannosidaseÂ </td><td>1.3Â </td><td>3.2Â </td><td>1.8Â </td><td>MPDL018e02_fÂ </td><td>232Â </td><td>Â </td></tr><tr><td>Putative alpha-mannosidaseÂ </td><td>1.6Â </td><td>3.1Â </td><td>1.9Â </td><td>MPDL061d01_fÂ </td><td>313Â </td><td>1Eâˆ’09Â </td></tr><tr><td><em>Secondary metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Chalcone reductaseÂ </td><td>1.4Â </td><td>4.1Â </td><td>3.0Â </td><td>GNf040a09Â </td><td>231Â </td><td>6Eâˆ’06Â </td></tr><tr><td>Chalcone reductaseÂ </td><td>0.7Â </td><td>2.1Â </td><td>2.3Â </td><td>GNf086d03Â </td><td>160Â </td><td>5Eâˆ’53Â </td></tr><tr><td> Flavonoid 3- <em>O</em> -galactosyl transferase Â </td><td>1.1Â </td><td>2.0Â </td><td>3.4Â </td><td>MPD011f01_fÂ </td><td>158Â </td><td>2Eâˆ’29Â </td></tr><tr><td><em>Transport/membrane</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Plasmamembrane intrinsic proteinÂ </td><td>2.3Â </td><td>2.8Â </td><td>0.8Â </td><td>MWL070f05_rÂ </td><td>1440Â </td><td>9Eâˆ’56Â </td></tr><tr><td>Putative nitrate transporterÂ </td><td>0.9Â </td><td>2.2Â </td><td>2.6Â </td><td>MWM134h03_rÂ </td><td>88Â </td><td>2Eâˆ’07Â </td></tr><tr><td>Aquaporin1Â </td><td>1.1Â </td><td>2.1Â </td><td>1.1Â </td><td>MWM132a07_rÂ </td><td>208Â </td><td>5Eâˆ’66Â </td></tr><tr><td>Plasma-membrane intrinsic proteinÂ </td><td>2.0Â </td><td>2.1Â </td><td>0.7Â </td><td>MWM091f06_rÂ </td><td>1490Â </td><td>5Eâˆ’13Â </td></tr><tr><td>Tonoplast intrinsic proteinÂ </td><td>2.3Â </td><td>2.1Â </td><td>1.7Â </td><td>MWM074b06_rÂ </td><td>444Â </td><td>2Eâˆ’42Â </td></tr><tr><td><em>Nodulin</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Nodulin 26-like proteinÂ </td><td>1.1Â </td><td>2.7Â </td><td>2.3Â </td><td>MWM104a10_rÂ </td><td>174Â </td><td>5Eâˆ’09Â </td></tr><tr><td>Nodule-enhanced sucrose synthaseÂ </td><td>1.3Â </td><td>2.3Â </td><td>1.2Â </td><td>MWL080e04_rÂ </td><td>797Â </td><td>2Eâˆ’78Â </td></tr><tr><td><em>Signal transduction</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>AnnexinÂ </td><td>1.2Â </td><td>3.2Â </td><td>2.8Â </td><td>MPD097d02_fÂ </td><td>135Â </td><td>3Eâˆ’36Â </td></tr><tr><td>AnnexinÂ </td><td>1.0Â </td><td>2.4Â </td><td>2.5Â </td><td>MPD065b05_fÂ </td><td>215Â </td><td>7Eâˆ’36Â </td></tr><tr><td>AnnexinÂ </td><td>1.0Â </td><td>2.2Â </td><td>2.6Â </td><td>MPD042e01_fÂ </td><td>185Â </td><td>2Eâˆ’62Â </td></tr><tr><td>ANTI-H(O) lectin (LTA)Â </td><td>1.8Â </td><td>2.0Â </td><td>7.0Â </td><td>MWM231h03_rÂ </td><td>24488Â </td><td>4Eâˆ’39Â </td></tr><tr><td><em>Protein fate</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Cysteine proteinase (LjCyp4)Â </td><td>3.8Â </td><td>19.7Â </td><td>8.0Â </td><td>GNf089d01Â </td><td>664Â </td><td>6Eâˆ’40Â </td></tr><tr><td>Cysteine proteinase (LjCyp1)Â </td><td>1.9Â </td><td>10.6Â </td><td>3.3Â </td><td>GNf032f12Â </td><td>251Â </td><td>Â </td></tr><tr><td>Cysteine proteinase (LjCyp2)Â </td><td>1.7Â </td><td>5.0Â </td><td>2.3Â </td><td>GNf037h07Â </td><td>964Â </td><td>6Eâˆ’23Â </td></tr><tr><td>Cysteine proteinase (LjCyp3)Â </td><td>1.3Â </td><td>4.0Â </td><td>2.3Â </td><td>GNf071h01Â </td><td>226Â </td><td>6Eâˆ’12Â </td></tr><tr><td><em>Cell wall</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Yieldin precursorÂ </td><td>1.0Â </td><td>4.3Â </td><td>1.9Â </td><td>MWM140d02_rÂ </td><td>75Â </td><td>3Eâˆ’16Â </td></tr><tr><td><em>Pathogen-related</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Putative disease resistant proteinÂ </td><td>1.6Â </td><td>2.0Â </td><td>1.2Â </td><td>MPDL019h09_fÂ </td><td>980Â </td><td>5Eâˆ’15Â </td></tr><tr><td><em>Phytohormone-related</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Jasmonic acid 2Â </td><td>1.4Â </td><td>6.0Â </td><td>4.6Â </td><td>MWL076b07_rÂ </td><td>149Â </td><td>7Eâˆ’73Â </td></tr><tr><td><em>Other enzyme</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Nicotianamine synthaseÂ </td><td>4.7Â </td><td>5.3Â </td><td>2.1Â </td><td>GNf070f09Â </td><td>316Â </td><td>5Eâˆ’21Â </td></tr><tr><td><em>Other category</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Dehydrin 3Â </td><td>2.2Â </td><td>8.0Â </td><td>6.9Â </td><td>MR001a01_fÂ </td><td>228Â </td><td>0.0002Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p> The data in the Gm3/SC3 and Gm6/SC3 columns are the gene expression levels in roots 3 and 6 weeks, respectively, after inoculation of the whole <em>G. mosseae</em> inoculum relative to those in control roots 3 weeks after inoculation of the sieved carrier. Since prolonged cultivation with low concentrations of phosphate may cause stress, <sup>18</sup> we do not think that there is any problem with the use of younger control roots. The data in the Gi8/gni3 column are the gene expression levels in roots 8 weeks after inoculation of NaClO-treated <em>G. margarita</em> relative to those in roots 3 weeks after mock-inoculation. MaxRE is the highest normalized expression level in the experiments. Genes that match hypothetical proteins of unknown function and ones that encode proteins exhibiting no homology to thus far known ones have been omitted from this table. </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/37284002" target="_blank">View Large</a></div></div>                        <a id="37284003" scrollto-destination="37284003"></a>
<div content-id="DSM014TB2" class="table-modal"><div class="table-wrap"><div id="DSM014TB2" class="table-wrap-title" data-id="DSM014TB2"><div class="label -label"><strong>TableÂ 2</strong></div><div class="caption"><p> Down-regulated genes in <em>L. japonicus</em> roots after establishment of symbiosis with <em>G. mosseae</em> and <em>G. margarita</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Current annotationÂ </th><th>Gm3/SC3Â </th><th>Gm6/SC3Â </th><th>Gi8/gni3Â </th><th>Gene IDÂ </th><th>Max REÂ </th><th>e-valueÂ </th></tr></thead><tbody><tr><td><em>Amino acid and nitrogen metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Serine decarboxylaseÂ </td><td>0.24Â </td><td>0.17Â </td><td>0.06Â </td><td>GENf054a02Â </td><td>956Â </td><td>1Eâˆ’52Â </td></tr><tr><td>Serine decarboxylaseÂ </td><td>0.30Â </td><td>0.24Â </td><td>0.11Â </td><td>MWM231b10_rÂ </td><td>609Â </td><td>0.004Â </td></tr><tr><td>Prephenate dehydrataseÂ </td><td>0.80Â </td><td>0.46Â </td><td>0.60Â </td><td>MR013b06_fÂ </td><td>153Â </td><td>4Eâˆ’15Â </td></tr><tr><td><em>Carbon metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Phosphoenolpyruvate carboxylase (LjPEPC2)Â </td><td>0.49Â </td><td>0.31Â </td><td>0.49Â </td><td>MWM088d03_rÂ </td><td>297Â </td><td>5Eâˆ’68Â </td></tr><tr><td>UDP-glucose:protein transglucosylaseÂ </td><td>0.49Â </td><td>0.32Â </td><td>0.54Â </td><td>MWM177b05_rÂ </td><td>429Â </td><td>3Eâˆ’32Â </td></tr><tr><td>Glucose-6-phosphoate 1- dehydrogenaseÂ </td><td>0.40Â </td><td>0.34Â </td><td>0.45Â </td><td>GENf019d07Â </td><td>397Â </td><td>4Eâˆ’23Â </td></tr><tr><td>Xyloglucan endotransglycosylaseÂ </td><td>1.00Â </td><td>0.37Â </td><td>0.11Â </td><td>MR065e10_fÂ </td><td>843Â </td><td>1Eâˆ’17Â </td></tr><tr><td><em>Secondary metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Phenylalanine ammonia-lyase (LjPAL3)Â </td><td>0.21Â </td><td>0.15Â </td><td>0.17Â </td><td>MR060a09_fÂ </td><td>1262Â </td><td>1Eâˆ’17Â </td></tr><tr><td>Phenylalanine ammonia-lyase (LjPAL4)Â </td><td>0.21Â </td><td>0.16Â </td><td>0.17Â </td><td>GENLf025c04Â </td><td>514Â </td><td>2Eâˆ’38Â </td></tr><tr><td>Phenylalanine ammonia-lyase (LjPAL1)Â </td><td>0.25Â </td><td>0.24Â </td><td>0.17Â </td><td>MRL007g11_fÂ </td><td>485Â </td><td>9Eâˆ’38Â </td></tr><tr><td>Phenylalanine ammonia-lyase (LjPAL8)Â </td><td>0.31Â </td><td>0.25Â </td><td>0.26Â </td><td>MWL032c01_rÂ </td><td>430Â </td><td>2Eâˆ’40Â </td></tr><tr><td>Phenylalanine ammonia-lyase (LjPAL2)Â </td><td>0.27Â </td><td>0.26Â </td><td>0.22Â </td><td>GENLf058e04Â </td><td>411Â </td><td>0.006Â </td></tr><tr><td>Chalcone synthaseÂ </td><td>0.44Â </td><td>0.27Â </td><td>0.39Â </td><td>MWM170f10_rÂ </td><td>536Â </td><td>8Eâˆ’48Â </td></tr><tr><td>Chalcone synthaseÂ </td><td>0.28Â </td><td>0.36Â </td><td>0.63Â </td><td>MWM193h03_rÂ </td><td>714Â </td><td>3Eâˆ’80Â </td></tr><tr><td>Chalcone synthaseÂ </td><td>0.50Â </td><td>0.39Â </td><td>0.65Â </td><td>MWL020g05_rÂ </td><td>450Â </td><td>1Eâˆ’14Â </td></tr><tr><td>Deoxychalcone synthaseÂ </td><td>0.40Â </td><td>0.21Â </td><td>0.36Â </td><td>MWM174f04_rÂ </td><td>485Â </td><td>1Eâˆ’18Â </td></tr><tr><td>Chalcone reductaseÂ </td><td>0.28Â </td><td>0.19Â </td><td>0.17Â </td><td>MWM002d07_rÂ </td><td>709Â </td><td>1Eâˆ’60Â </td></tr><tr><td>Chalcone reductaseÂ </td><td>0.35Â </td><td>0.22Â </td><td>0.17Â </td><td>GNf090d05Â </td><td>728Â </td><td>6Eâˆ’64Â </td></tr><tr><td> Caffeoyl-CoA <em>O</em> -methyltransferase Â </td><td>0.98Â </td><td>0.43Â </td><td>0.38Â </td><td>MPD011e05_fÂ </td><td>486Â </td><td>7Eâˆ’66Â </td></tr><tr><td> Caffeoyl-CoA <em>O</em> -methyltransferase Â </td><td>0.93Â </td><td>0.43Â </td><td>0.46Â </td><td>MWM071f11_rÂ </td><td>240Â </td><td>0.00008Â </td></tr><tr><td>Isoprene synthaseÂ </td><td>1.82Â </td><td>0.26Â </td><td>0.39Â </td><td>MWL054c12_rÂ </td><td>686Â </td><td>3Eâˆ’37Â </td></tr><tr><td>Lupeol synthaseÂ </td><td>0.46Â </td><td>0.19Â </td><td>0.56Â </td><td>GNf046g09Â </td><td>309Â </td><td>8Eâˆ’50Â </td></tr><tr><td><em>Transport/membrane</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Phosphate transporter (LjPT1)Â </td><td>0.36Â </td><td>0.18Â </td><td>0.68Â </td><td>MWM077d10_rÂ </td><td>334Â </td><td>4Eâˆ’59Â </td></tr><tr><td>Mitochondrial dicarboxylate carrier proteinÂ </td><td>1.01Â </td><td>0.35Â </td><td>0.54Â </td><td>MPD024c12_fÂ </td><td>266Â </td><td>7Eâˆ’14Â </td></tr><tr><td> Plasma membrane Ca <sup>2+</sup> -ATPase Â </td><td>1.05Â </td><td>0.38Â </td><td>0.26Â </td><td>GENLf026c07Â </td><td>473Â </td><td>5Eâˆ’28Â </td></tr><tr><td>Sucrose transport proteinÂ </td><td>0.40Â </td><td>0.39Â </td><td>0.53Â </td><td>MWM221d11_rÂ </td><td>342Â </td><td>2Eâˆ’47Â </td></tr><tr><td><em>Signal transduction</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Putative acid phosphataseÂ </td><td>0.85Â </td><td>0.39Â </td><td>0.23Â </td><td>MWM048e06_rÂ </td><td>1670Â </td><td>8Eâˆ’24Â </td></tr><tr><td><em>Transcription/translation</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Transcription factor WRKY4Â </td><td>0.35Â </td><td>0.19Â </td><td>0.18Â </td><td>MWM168c07_rÂ </td><td>690Â </td><td>4Eâˆ’45Â </td></tr><tr><td>WRKY-type DNA binding proteinÂ </td><td>0.50Â </td><td>0.35Â </td><td>0.24Â </td><td>MWM240a07_rÂ </td><td>624Â </td><td>5Eâˆ’07Â </td></tr><tr><td>WRKY DNA-binding proteinÂ </td><td>0.36Â </td><td>0.35Â </td><td>0.31Â </td><td>MR083f05_fÂ </td><td>263Â </td><td>0.0001Â </td></tr><tr><td><em>Cell wall</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Extensin-like proteinÂ </td><td>1.01Â </td><td>0.35Â </td><td>0.48Â </td><td>MWM170b07_rÂ </td><td>911Â </td><td>0.0004Â </td></tr><tr><td><em>Pathogen-related</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>PeroxidaseÂ </td><td>0.46Â </td><td>0.22Â </td><td>0.44Â </td><td>GENf076g12Â </td><td>384Â </td><td>2Eâˆ’17Â </td></tr><tr><td>PeroxidaseÂ </td><td>0.47Â </td><td>0.37Â </td><td>0.76Â </td><td>GNf069g02Â </td><td>136Â </td><td>6Eâˆ’14Â </td></tr><tr><td>Syringolide-induced proteinÂ </td><td>0.32Â </td><td>0.23Â </td><td>0.20Â </td><td>MWM033e05_rÂ </td><td>745Â </td><td>8Eâˆ’14Â </td></tr><tr><td>Syringolide-induced proteinÂ </td><td>0.35Â </td><td>0.29Â </td><td>0.39Â </td><td>GNf002b04Â </td><td>258Â </td><td>2Eâˆ’23Â </td></tr><tr><td>Syringolide-induced proteinÂ </td><td>0.99Â </td><td>0.39Â </td><td>0.15Â </td><td>GNf095h04Â </td><td>258Â </td><td>6Eâˆ’31Â </td></tr><tr><td>Similar to the BURP domainÂ </td><td>0.56Â </td><td>0.22Â </td><td>0.13Â </td><td>MPDL062c05_fÂ </td><td>4760Â </td><td>1Eâˆ’48Â </td></tr><tr><td>Seed coat BURP domain proteinÂ </td><td>0.42Â </td><td>0.25Â </td><td>0.15Â </td><td>MPDL082d06_fÂ </td><td>3753Â </td><td>9Eâˆ’59Â </td></tr><tr><td>Seed coat BURP domain proteinÂ </td><td>0.52Â </td><td>0.35Â </td><td>0.10Â </td><td>MPD013h01_fÂ </td><td>4069Â </td><td>2Eâˆ’18Â </td></tr><tr><td>PR10-1 proteinÂ </td><td>0.62Â </td><td>0.35Â </td><td>0.82Â </td><td>GNf017d12Â </td><td>1268Â </td><td>8Eâˆ’16Â </td></tr><tr><td><em>Other enzymes</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Cytochrome P450-1Â </td><td>0.18Â </td><td>0.17Â </td><td>0.16Â </td><td>MR095g09_fÂ </td><td>678Â </td><td>1Eâˆ’19Â </td></tr><tr><td>Mannan endo-1,4-beta-mannosidaseÂ </td><td>0.55Â </td><td>0.22Â </td><td>0.59Â </td><td>MWM099c01_rÂ </td><td>304Â </td><td>3Eâˆ’57Â </td></tr><tr><td>Soluble inorganic pyrophosphataseÂ </td><td>0.75Â </td><td>0.35Â </td><td>0.28Â </td><td>MPD001e10_fÂ </td><td>603Â </td><td>8Eâˆ’07Â </td></tr><tr><td>Cytochrome P450, putativeÂ </td><td>0.38Â </td><td>0.30Â </td><td>0.28Â </td><td>MR076b02_fÂ </td><td>233Â </td><td>4Eâˆ’10Â </td></tr><tr><td> Glutathione <em>S</em> -transferase Â </td><td>0.30Â </td><td>0.35Â </td><td>0.85Â </td><td>MWM067e12_rÂ </td><td>1452Â </td><td>3Eâˆ’33Â </td></tr><tr><td>Cytochrome P450 82C1Â </td><td>1.28Â </td><td>0.36Â </td><td>0.51Â </td><td>MR061f02_fÂ </td><td>219Â </td><td>3Eâˆ’35Â </td></tr><tr><td>ATP synthase 9Â </td><td>0.24Â </td><td>0.20Â </td><td>0.21Â </td><td>MWM223c10_rÂ </td><td>456Â </td><td>3Eâˆ’22Â </td></tr><tr><td><em>Other category</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Putative acyl-CoA oxidaseÂ </td><td>0.26Â </td><td>0.18Â </td><td>0.19Â </td><td>MWM217b02_rÂ </td><td>628Â </td><td>1Eâˆ’42Â </td></tr></tbody></table></div><div content-id="DSM014TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Current annotationÂ </th><th>Gm3/SC3Â </th><th>Gm6/SC3Â </th><th>Gi8/gni3Â </th><th>Gene IDÂ </th><th>Max REÂ </th><th>e-valueÂ </th></tr></thead><tbody><tr><td><em>Amino acid and nitrogen metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Serine decarboxylaseÂ </td><td>0.24Â </td><td>0.17Â </td><td>0.06Â </td><td>GENf054a02Â </td><td>956Â </td><td>1Eâˆ’52Â </td></tr><tr><td>Serine decarboxylaseÂ </td><td>0.30Â </td><td>0.24Â </td><td>0.11Â </td><td>MWM231b10_rÂ </td><td>609Â </td><td>0.004Â </td></tr><tr><td>Prephenate dehydrataseÂ </td><td>0.80Â </td><td>0.46Â </td><td>0.60Â </td><td>MR013b06_fÂ </td><td>153Â </td><td>4Eâˆ’15Â </td></tr><tr><td><em>Carbon metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Phosphoenolpyruvate carboxylase (LjPEPC2)Â </td><td>0.49Â </td><td>0.31Â </td><td>0.49Â </td><td>MWM088d03_rÂ </td><td>297Â </td><td>5Eâˆ’68Â </td></tr><tr><td>UDP-glucose:protein transglucosylaseÂ </td><td>0.49Â </td><td>0.32Â </td><td>0.54Â </td><td>MWM177b05_rÂ </td><td>429Â </td><td>3Eâˆ’32Â </td></tr><tr><td>Glucose-6-phosphoate 1- dehydrogenaseÂ </td><td>0.40Â </td><td>0.34Â </td><td>0.45Â </td><td>GENf019d07Â </td><td>397Â </td><td>4Eâˆ’23Â </td></tr><tr><td>Xyloglucan endotransglycosylaseÂ </td><td>1.00Â </td><td>0.37Â </td><td>0.11Â </td><td>MR065e10_fÂ </td><td>843Â </td><td>1Eâˆ’17Â </td></tr><tr><td><em>Secondary metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Phenylalanine ammonia-lyase (LjPAL3)Â </td><td>0.21Â </td><td>0.15Â </td><td>0.17Â </td><td>MR060a09_fÂ </td><td>1262Â </td><td>1Eâˆ’17Â </td></tr><tr><td>Phenylalanine ammonia-lyase (LjPAL4)Â </td><td>0.21Â </td><td>0.16Â </td><td>0.17Â </td><td>GENLf025c04Â </td><td>514Â </td><td>2Eâˆ’38Â </td></tr><tr><td>Phenylalanine ammonia-lyase (LjPAL1)Â </td><td>0.25Â </td><td>0.24Â </td><td>0.17Â </td><td>MRL007g11_fÂ </td><td>485Â </td><td>9Eâˆ’38Â </td></tr><tr><td>Phenylalanine ammonia-lyase (LjPAL8)Â </td><td>0.31Â </td><td>0.25Â </td><td>0.26Â </td><td>MWL032c01_rÂ </td><td>430Â </td><td>2Eâˆ’40Â </td></tr><tr><td>Phenylalanine ammonia-lyase (LjPAL2)Â </td><td>0.27Â </td><td>0.26Â </td><td>0.22Â </td><td>GENLf058e04Â </td><td>411Â </td><td>0.006Â </td></tr><tr><td>Chalcone synthaseÂ </td><td>0.44Â </td><td>0.27Â </td><td>0.39Â </td><td>MWM170f10_rÂ </td><td>536Â </td><td>8Eâˆ’48Â </td></tr><tr><td>Chalcone synthaseÂ </td><td>0.28Â </td><td>0.36Â </td><td>0.63Â </td><td>MWM193h03_rÂ </td><td>714Â </td><td>3Eâˆ’80Â </td></tr><tr><td>Chalcone synthaseÂ </td><td>0.50Â </td><td>0.39Â </td><td>0.65Â </td><td>MWL020g05_rÂ </td><td>450Â </td><td>1Eâˆ’14Â </td></tr><tr><td>Deoxychalcone synthaseÂ </td><td>0.40Â </td><td>0.21Â </td><td>0.36Â </td><td>MWM174f04_rÂ </td><td>485Â </td><td>1Eâˆ’18Â </td></tr><tr><td>Chalcone reductaseÂ </td><td>0.28Â </td><td>0.19Â </td><td>0.17Â </td><td>MWM002d07_rÂ </td><td>709Â </td><td>1Eâˆ’60Â </td></tr><tr><td>Chalcone reductaseÂ </td><td>0.35Â </td><td>0.22Â </td><td>0.17Â </td><td>GNf090d05Â </td><td>728Â </td><td>6Eâˆ’64Â </td></tr><tr><td> Caffeoyl-CoA <em>O</em> -methyltransferase Â </td><td>0.98Â </td><td>0.43Â </td><td>0.38Â </td><td>MPD011e05_fÂ </td><td>486Â </td><td>7Eâˆ’66Â </td></tr><tr><td> Caffeoyl-CoA <em>O</em> -methyltransferase Â </td><td>0.93Â </td><td>0.43Â </td><td>0.46Â </td><td>MWM071f11_rÂ </td><td>240Â </td><td>0.00008Â </td></tr><tr><td>Isoprene synthaseÂ </td><td>1.82Â </td><td>0.26Â </td><td>0.39Â </td><td>MWL054c12_rÂ </td><td>686Â </td><td>3Eâˆ’37Â </td></tr><tr><td>Lupeol synthaseÂ </td><td>0.46Â </td><td>0.19Â </td><td>0.56Â </td><td>GNf046g09Â </td><td>309Â </td><td>8Eâˆ’50Â </td></tr><tr><td><em>Transport/membrane</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Phosphate transporter (LjPT1)Â </td><td>0.36Â </td><td>0.18Â </td><td>0.68Â </td><td>MWM077d10_rÂ </td><td>334Â </td><td>4Eâˆ’59Â </td></tr><tr><td>Mitochondrial dicarboxylate carrier proteinÂ </td><td>1.01Â </td><td>0.35Â </td><td>0.54Â </td><td>MPD024c12_fÂ </td><td>266Â </td><td>7Eâˆ’14Â </td></tr><tr><td> Plasma membrane Ca <sup>2+</sup> -ATPase Â </td><td>1.05Â </td><td>0.38Â </td><td>0.26Â </td><td>GENLf026c07Â </td><td>473Â </td><td>5Eâˆ’28Â </td></tr><tr><td>Sucrose transport proteinÂ </td><td>0.40Â </td><td>0.39Â </td><td>0.53Â </td><td>MWM221d11_rÂ </td><td>342Â </td><td>2Eâˆ’47Â </td></tr><tr><td><em>Signal transduction</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Putative acid phosphataseÂ </td><td>0.85Â </td><td>0.39Â </td><td>0.23Â </td><td>MWM048e06_rÂ </td><td>1670Â </td><td>8Eâˆ’24Â </td></tr><tr><td><em>Transcription/translation</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Transcription factor WRKY4Â </td><td>0.35Â </td><td>0.19Â </td><td>0.18Â </td><td>MWM168c07_rÂ </td><td>690Â </td><td>4Eâˆ’45Â </td></tr><tr><td>WRKY-type DNA binding proteinÂ </td><td>0.50Â </td><td>0.35Â </td><td>0.24Â </td><td>MWM240a07_rÂ </td><td>624Â </td><td>5Eâˆ’07Â </td></tr><tr><td>WRKY DNA-binding proteinÂ </td><td>0.36Â </td><td>0.35Â </td><td>0.31Â </td><td>MR083f05_fÂ </td><td>263Â </td><td>0.0001Â </td></tr><tr><td><em>Cell wall</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Extensin-like proteinÂ </td><td>1.01Â </td><td>0.35Â </td><td>0.48Â </td><td>MWM170b07_rÂ </td><td>911Â </td><td>0.0004Â </td></tr><tr><td><em>Pathogen-related</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>PeroxidaseÂ </td><td>0.46Â </td><td>0.22Â </td><td>0.44Â </td><td>GENf076g12Â </td><td>384Â </td><td>2Eâˆ’17Â </td></tr><tr><td>PeroxidaseÂ </td><td>0.47Â </td><td>0.37Â </td><td>0.76Â </td><td>GNf069g02Â </td><td>136Â </td><td>6Eâˆ’14Â </td></tr><tr><td>Syringolide-induced proteinÂ </td><td>0.32Â </td><td>0.23Â </td><td>0.20Â </td><td>MWM033e05_rÂ </td><td>745Â </td><td>8Eâˆ’14Â </td></tr><tr><td>Syringolide-induced proteinÂ </td><td>0.35Â </td><td>0.29Â </td><td>0.39Â </td><td>GNf002b04Â </td><td>258Â </td><td>2Eâˆ’23Â </td></tr><tr><td>Syringolide-induced proteinÂ </td><td>0.99Â </td><td>0.39Â </td><td>0.15Â </td><td>GNf095h04Â </td><td>258Â </td><td>6Eâˆ’31Â </td></tr><tr><td>Similar to the BURP domainÂ </td><td>0.56Â </td><td>0.22Â </td><td>0.13Â </td><td>MPDL062c05_fÂ </td><td>4760Â </td><td>1Eâˆ’48Â </td></tr><tr><td>Seed coat BURP domain proteinÂ </td><td>0.42Â </td><td>0.25Â </td><td>0.15Â </td><td>MPDL082d06_fÂ </td><td>3753Â </td><td>9Eâˆ’59Â </td></tr><tr><td>Seed coat BURP domain proteinÂ </td><td>0.52Â </td><td>0.35Â </td><td>0.10Â </td><td>MPD013h01_fÂ </td><td>4069Â </td><td>2Eâˆ’18Â </td></tr><tr><td>PR10-1 proteinÂ </td><td>0.62Â </td><td>0.35Â </td><td>0.82Â </td><td>GNf017d12Â </td><td>1268Â </td><td>8Eâˆ’16Â </td></tr><tr><td><em>Other enzymes</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Cytochrome P450-1Â </td><td>0.18Â </td><td>0.17Â </td><td>0.16Â </td><td>MR095g09_fÂ </td><td>678Â </td><td>1Eâˆ’19Â </td></tr><tr><td>Mannan endo-1,4-beta-mannosidaseÂ </td><td>0.55Â </td><td>0.22Â </td><td>0.59Â </td><td>MWM099c01_rÂ </td><td>304Â </td><td>3Eâˆ’57Â </td></tr><tr><td>Soluble inorganic pyrophosphataseÂ </td><td>0.75Â </td><td>0.35Â </td><td>0.28Â </td><td>MPD001e10_fÂ </td><td>603Â </td><td>8Eâˆ’07Â </td></tr><tr><td>Cytochrome P450, putativeÂ </td><td>0.38Â </td><td>0.30Â </td><td>0.28Â </td><td>MR076b02_fÂ </td><td>233Â </td><td>4Eâˆ’10Â </td></tr><tr><td> Glutathione <em>S</em> -transferase Â </td><td>0.30Â </td><td>0.35Â </td><td>0.85Â </td><td>MWM067e12_rÂ </td><td>1452Â </td><td>3Eâˆ’33Â </td></tr><tr><td>Cytochrome P450 82C1Â </td><td>1.28Â </td><td>0.36Â </td><td>0.51Â </td><td>MR061f02_fÂ </td><td>219Â </td><td>3Eâˆ’35Â </td></tr><tr><td>ATP synthase 9Â </td><td>0.24Â </td><td>0.20Â </td><td>0.21Â </td><td>MWM223c10_rÂ </td><td>456Â </td><td>3Eâˆ’22Â </td></tr><tr><td><em>Other category</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Putative acyl-CoA oxidaseÂ </td><td>0.26Â </td><td>0.18Â </td><td>0.19Â </td><td>MWM217b02_rÂ </td><td>628Â </td><td>1Eâˆ’42Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p> The data in the Gm3/SC3 and Gm6/SC3 columns are the gene expression levels in roots 3 and 6 weeks, respectively, after inoculation of the whole <em>G. mosseae</em> inoculum relative to those in control roots 3 weeks after inoculation of the sieved carrier. Since prolonged cultivation with low concentrations of phosphate may cause stress, <sup><a reveal-id="DSM014C18" data-open="DSM014C18" class="link link-ref link-reveal xref-bibr">18</a></sup> we do not think that there is any problem with the use of younger control roots. The data in the Gi8/gni3 column are the gene expression levels in roots 8 weeks after inoculation of NaClO-treated <em>G. margarita</em> relative to those in roots 3 weeks after mock-inoculation. MaxRE is the highest normalized expression level in the experiments. Genes that match hypothetical proteins of unknown function and ones that encode proteins exhibiting no homology to thus far known ones have been omitted from this table. </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/37284003" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM014TB2" class="table-wrap-title" data-id="DSM014TB2"><div class="label -label"><strong>TableÂ 2</strong></div><div class="caption"><p> Down-regulated genes in <em>L. japonicus</em> roots after establishment of symbiosis with <em>G. mosseae</em> and <em>G. margarita</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Current annotationÂ </th><th>Gm3/SC3Â </th><th>Gm6/SC3Â </th><th>Gi8/gni3Â </th><th>Gene IDÂ </th><th>Max REÂ </th><th>e-valueÂ </th></tr></thead><tbody><tr><td><em>Amino acid and nitrogen metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Serine decarboxylaseÂ </td><td>0.24Â </td><td>0.17Â </td><td>0.06Â </td><td>GENf054a02Â </td><td>956Â </td><td>1Eâˆ’52Â </td></tr><tr><td>Serine decarboxylaseÂ </td><td>0.30Â </td><td>0.24Â </td><td>0.11Â </td><td>MWM231b10_rÂ </td><td>609Â </td><td>0.004Â </td></tr><tr><td>Prephenate dehydrataseÂ </td><td>0.80Â </td><td>0.46Â </td><td>0.60Â </td><td>MR013b06_fÂ </td><td>153Â </td><td>4Eâˆ’15Â </td></tr><tr><td><em>Carbon metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Phosphoenolpyruvate carboxylase (LjPEPC2)Â </td><td>0.49Â </td><td>0.31Â </td><td>0.49Â </td><td>MWM088d03_rÂ </td><td>297Â </td><td>5Eâˆ’68Â </td></tr><tr><td>UDP-glucose:protein transglucosylaseÂ </td><td>0.49Â </td><td>0.32Â </td><td>0.54Â </td><td>MWM177b05_rÂ </td><td>429Â </td><td>3Eâˆ’32Â </td></tr><tr><td>Glucose-6-phosphoate 1- dehydrogenaseÂ </td><td>0.40Â </td><td>0.34Â </td><td>0.45Â </td><td>GENf019d07Â </td><td>397Â </td><td>4Eâˆ’23Â </td></tr><tr><td>Xyloglucan endotransglycosylaseÂ </td><td>1.00Â </td><td>0.37Â </td><td>0.11Â </td><td>MR065e10_fÂ </td><td>843Â </td><td>1Eâˆ’17Â </td></tr><tr><td><em>Secondary metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Phenylalanine ammonia-lyase (LjPAL3)Â </td><td>0.21Â </td><td>0.15Â </td><td>0.17Â </td><td>MR060a09_fÂ </td><td>1262Â </td><td>1Eâˆ’17Â </td></tr><tr><td>Phenylalanine ammonia-lyase (LjPAL4)Â </td><td>0.21Â </td><td>0.16Â </td><td>0.17Â </td><td>GENLf025c04Â </td><td>514Â </td><td>2Eâˆ’38Â </td></tr><tr><td>Phenylalanine ammonia-lyase (LjPAL1)Â </td><td>0.25Â </td><td>0.24Â </td><td>0.17Â </td><td>MRL007g11_fÂ </td><td>485Â </td><td>9Eâˆ’38Â </td></tr><tr><td>Phenylalanine ammonia-lyase (LjPAL8)Â </td><td>0.31Â </td><td>0.25Â </td><td>0.26Â </td><td>MWL032c01_rÂ </td><td>430Â </td><td>2Eâˆ’40Â </td></tr><tr><td>Phenylalanine ammonia-lyase (LjPAL2)Â </td><td>0.27Â </td><td>0.26Â </td><td>0.22Â </td><td>GENLf058e04Â </td><td>411Â </td><td>0.006Â </td></tr><tr><td>Chalcone synthaseÂ </td><td>0.44Â </td><td>0.27Â </td><td>0.39Â </td><td>MWM170f10_rÂ </td><td>536Â </td><td>8Eâˆ’48Â </td></tr><tr><td>Chalcone synthaseÂ </td><td>0.28Â </td><td>0.36Â </td><td>0.63Â </td><td>MWM193h03_rÂ </td><td>714Â </td><td>3Eâˆ’80Â </td></tr><tr><td>Chalcone synthaseÂ </td><td>0.50Â </td><td>0.39Â </td><td>0.65Â </td><td>MWL020g05_rÂ </td><td>450Â </td><td>1Eâˆ’14Â </td></tr><tr><td>Deoxychalcone synthaseÂ </td><td>0.40Â </td><td>0.21Â </td><td>0.36Â </td><td>MWM174f04_rÂ </td><td>485Â </td><td>1Eâˆ’18Â </td></tr><tr><td>Chalcone reductaseÂ </td><td>0.28Â </td><td>0.19Â </td><td>0.17Â </td><td>MWM002d07_rÂ </td><td>709Â </td><td>1Eâˆ’60Â </td></tr><tr><td>Chalcone reductaseÂ </td><td>0.35Â </td><td>0.22Â </td><td>0.17Â </td><td>GNf090d05Â </td><td>728Â </td><td>6Eâˆ’64Â </td></tr><tr><td> Caffeoyl-CoA <em>O</em> -methyltransferase Â </td><td>0.98Â </td><td>0.43Â </td><td>0.38Â </td><td>MPD011e05_fÂ </td><td>486Â </td><td>7Eâˆ’66Â </td></tr><tr><td> Caffeoyl-CoA <em>O</em> -methyltransferase Â </td><td>0.93Â </td><td>0.43Â </td><td>0.46Â </td><td>MWM071f11_rÂ </td><td>240Â </td><td>0.00008Â </td></tr><tr><td>Isoprene synthaseÂ </td><td>1.82Â </td><td>0.26Â </td><td>0.39Â </td><td>MWL054c12_rÂ </td><td>686Â </td><td>3Eâˆ’37Â </td></tr><tr><td>Lupeol synthaseÂ </td><td>0.46Â </td><td>0.19Â </td><td>0.56Â </td><td>GNf046g09Â </td><td>309Â </td><td>8Eâˆ’50Â </td></tr><tr><td><em>Transport/membrane</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Phosphate transporter (LjPT1)Â </td><td>0.36Â </td><td>0.18Â </td><td>0.68Â </td><td>MWM077d10_rÂ </td><td>334Â </td><td>4Eâˆ’59Â </td></tr><tr><td>Mitochondrial dicarboxylate carrier proteinÂ </td><td>1.01Â </td><td>0.35Â </td><td>0.54Â </td><td>MPD024c12_fÂ </td><td>266Â </td><td>7Eâˆ’14Â </td></tr><tr><td> Plasma membrane Ca <sup>2+</sup> -ATPase Â </td><td>1.05Â </td><td>0.38Â </td><td>0.26Â </td><td>GENLf026c07Â </td><td>473Â </td><td>5Eâˆ’28Â </td></tr><tr><td>Sucrose transport proteinÂ </td><td>0.40Â </td><td>0.39Â </td><td>0.53Â </td><td>MWM221d11_rÂ </td><td>342Â </td><td>2Eâˆ’47Â </td></tr><tr><td><em>Signal transduction</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Putative acid phosphataseÂ </td><td>0.85Â </td><td>0.39Â </td><td>0.23Â </td><td>MWM048e06_rÂ </td><td>1670Â </td><td>8Eâˆ’24Â </td></tr><tr><td><em>Transcription/translation</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Transcription factor WRKY4Â </td><td>0.35Â </td><td>0.19Â </td><td>0.18Â </td><td>MWM168c07_rÂ </td><td>690Â </td><td>4Eâˆ’45Â </td></tr><tr><td>WRKY-type DNA binding proteinÂ </td><td>0.50Â </td><td>0.35Â </td><td>0.24Â </td><td>MWM240a07_rÂ </td><td>624Â </td><td>5Eâˆ’07Â </td></tr><tr><td>WRKY DNA-binding proteinÂ </td><td>0.36Â </td><td>0.35Â </td><td>0.31Â </td><td>MR083f05_fÂ </td><td>263Â </td><td>0.0001Â </td></tr><tr><td><em>Cell wall</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Extensin-like proteinÂ </td><td>1.01Â </td><td>0.35Â </td><td>0.48Â </td><td>MWM170b07_rÂ </td><td>911Â </td><td>0.0004Â </td></tr><tr><td><em>Pathogen-related</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>PeroxidaseÂ </td><td>0.46Â </td><td>0.22Â </td><td>0.44Â </td><td>GENf076g12Â </td><td>384Â </td><td>2Eâˆ’17Â </td></tr><tr><td>PeroxidaseÂ </td><td>0.47Â </td><td>0.37Â </td><td>0.76Â </td><td>GNf069g02Â </td><td>136Â </td><td>6Eâˆ’14Â </td></tr><tr><td>Syringolide-induced proteinÂ </td><td>0.32Â </td><td>0.23Â </td><td>0.20Â </td><td>MWM033e05_rÂ </td><td>745Â </td><td>8Eâˆ’14Â </td></tr><tr><td>Syringolide-induced proteinÂ </td><td>0.35Â </td><td>0.29Â </td><td>0.39Â </td><td>GNf002b04Â </td><td>258Â </td><td>2Eâˆ’23Â </td></tr><tr><td>Syringolide-induced proteinÂ </td><td>0.99Â </td><td>0.39Â </td><td>0.15Â </td><td>GNf095h04Â </td><td>258Â </td><td>6Eâˆ’31Â </td></tr><tr><td>Similar to the BURP domainÂ </td><td>0.56Â </td><td>0.22Â </td><td>0.13Â </td><td>MPDL062c05_fÂ </td><td>4760Â </td><td>1Eâˆ’48Â </td></tr><tr><td>Seed coat BURP domain proteinÂ </td><td>0.42Â </td><td>0.25Â </td><td>0.15Â </td><td>MPDL082d06_fÂ </td><td>3753Â </td><td>9Eâˆ’59Â </td></tr><tr><td>Seed coat BURP domain proteinÂ </td><td>0.52Â </td><td>0.35Â </td><td>0.10Â </td><td>MPD013h01_fÂ </td><td>4069Â </td><td>2Eâˆ’18Â </td></tr><tr><td>PR10-1 proteinÂ </td><td>0.62Â </td><td>0.35Â </td><td>0.82Â </td><td>GNf017d12Â </td><td>1268Â </td><td>8Eâˆ’16Â </td></tr><tr><td><em>Other enzymes</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Cytochrome P450-1Â </td><td>0.18Â </td><td>0.17Â </td><td>0.16Â </td><td>MR095g09_fÂ </td><td>678Â </td><td>1Eâˆ’19Â </td></tr><tr><td>Mannan endo-1,4-beta-mannosidaseÂ </td><td>0.55Â </td><td>0.22Â </td><td>0.59Â </td><td>MWM099c01_rÂ </td><td>304Â </td><td>3Eâˆ’57Â </td></tr><tr><td>Soluble inorganic pyrophosphataseÂ </td><td>0.75Â </td><td>0.35Â </td><td>0.28Â </td><td>MPD001e10_fÂ </td><td>603Â </td><td>8Eâˆ’07Â </td></tr><tr><td>Cytochrome P450, putativeÂ </td><td>0.38Â </td><td>0.30Â </td><td>0.28Â </td><td>MR076b02_fÂ </td><td>233Â </td><td>4Eâˆ’10Â </td></tr><tr><td> Glutathione <em>S</em> -transferase Â </td><td>0.30Â </td><td>0.35Â </td><td>0.85Â </td><td>MWM067e12_rÂ </td><td>1452Â </td><td>3Eâˆ’33Â </td></tr><tr><td>Cytochrome P450 82C1Â </td><td>1.28Â </td><td>0.36Â </td><td>0.51Â </td><td>MR061f02_fÂ </td><td>219Â </td><td>3Eâˆ’35Â </td></tr><tr><td>ATP synthase 9Â </td><td>0.24Â </td><td>0.20Â </td><td>0.21Â </td><td>MWM223c10_rÂ </td><td>456Â </td><td>3Eâˆ’22Â </td></tr><tr><td><em>Other category</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Putative acyl-CoA oxidaseÂ </td><td>0.26Â </td><td>0.18Â </td><td>0.19Â </td><td>MWM217b02_rÂ </td><td>628Â </td><td>1Eâˆ’42Â </td></tr></tbody></table></div><div content-id="DSM014TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Current annotationÂ </th><th>Gm3/SC3Â </th><th>Gm6/SC3Â </th><th>Gi8/gni3Â </th><th>Gene IDÂ </th><th>Max REÂ </th><th>e-valueÂ </th></tr></thead><tbody><tr><td><em>Amino acid and nitrogen metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Serine decarboxylaseÂ </td><td>0.24Â </td><td>0.17Â </td><td>0.06Â </td><td>GENf054a02Â </td><td>956Â </td><td>1Eâˆ’52Â </td></tr><tr><td>Serine decarboxylaseÂ </td><td>0.30Â </td><td>0.24Â </td><td>0.11Â </td><td>MWM231b10_rÂ </td><td>609Â </td><td>0.004Â </td></tr><tr><td>Prephenate dehydrataseÂ </td><td>0.80Â </td><td>0.46Â </td><td>0.60Â </td><td>MR013b06_fÂ </td><td>153Â </td><td>4Eâˆ’15Â </td></tr><tr><td><em>Carbon metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Phosphoenolpyruvate carboxylase (LjPEPC2)Â </td><td>0.49Â </td><td>0.31Â </td><td>0.49Â </td><td>MWM088d03_rÂ </td><td>297Â </td><td>5Eâˆ’68Â </td></tr><tr><td>UDP-glucose:protein transglucosylaseÂ </td><td>0.49Â </td><td>0.32Â </td><td>0.54Â </td><td>MWM177b05_rÂ </td><td>429Â </td><td>3Eâˆ’32Â </td></tr><tr><td>Glucose-6-phosphoate 1- dehydrogenaseÂ </td><td>0.40Â </td><td>0.34Â </td><td>0.45Â </td><td>GENf019d07Â </td><td>397Â </td><td>4Eâˆ’23Â </td></tr><tr><td>Xyloglucan endotransglycosylaseÂ </td><td>1.00Â </td><td>0.37Â </td><td>0.11Â </td><td>MR065e10_fÂ </td><td>843Â </td><td>1Eâˆ’17Â </td></tr><tr><td><em>Secondary metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Phenylalanine ammonia-lyase (LjPAL3)Â </td><td>0.21Â </td><td>0.15Â </td><td>0.17Â </td><td>MR060a09_fÂ </td><td>1262Â </td><td>1Eâˆ’17Â </td></tr><tr><td>Phenylalanine ammonia-lyase (LjPAL4)Â </td><td>0.21Â </td><td>0.16Â </td><td>0.17Â </td><td>GENLf025c04Â </td><td>514Â </td><td>2Eâˆ’38Â </td></tr><tr><td>Phenylalanine ammonia-lyase (LjPAL1)Â </td><td>0.25Â </td><td>0.24Â </td><td>0.17Â </td><td>MRL007g11_fÂ </td><td>485Â </td><td>9Eâˆ’38Â </td></tr><tr><td>Phenylalanine ammonia-lyase (LjPAL8)Â </td><td>0.31Â </td><td>0.25Â </td><td>0.26Â </td><td>MWL032c01_rÂ </td><td>430Â </td><td>2Eâˆ’40Â </td></tr><tr><td>Phenylalanine ammonia-lyase (LjPAL2)Â </td><td>0.27Â </td><td>0.26Â </td><td>0.22Â </td><td>GENLf058e04Â </td><td>411Â </td><td>0.006Â </td></tr><tr><td>Chalcone synthaseÂ </td><td>0.44Â </td><td>0.27Â </td><td>0.39Â </td><td>MWM170f10_rÂ </td><td>536Â </td><td>8Eâˆ’48Â </td></tr><tr><td>Chalcone synthaseÂ </td><td>0.28Â </td><td>0.36Â </td><td>0.63Â </td><td>MWM193h03_rÂ </td><td>714Â </td><td>3Eâˆ’80Â </td></tr><tr><td>Chalcone synthaseÂ </td><td>0.50Â </td><td>0.39Â </td><td>0.65Â </td><td>MWL020g05_rÂ </td><td>450Â </td><td>1Eâˆ’14Â </td></tr><tr><td>Deoxychalcone synthaseÂ </td><td>0.40Â </td><td>0.21Â </td><td>0.36Â </td><td>MWM174f04_rÂ </td><td>485Â </td><td>1Eâˆ’18Â </td></tr><tr><td>Chalcone reductaseÂ </td><td>0.28Â </td><td>0.19Â </td><td>0.17Â </td><td>MWM002d07_rÂ </td><td>709Â </td><td>1Eâˆ’60Â </td></tr><tr><td>Chalcone reductaseÂ </td><td>0.35Â </td><td>0.22Â </td><td>0.17Â </td><td>GNf090d05Â </td><td>728Â </td><td>6Eâˆ’64Â </td></tr><tr><td> Caffeoyl-CoA <em>O</em> -methyltransferase Â </td><td>0.98Â </td><td>0.43Â </td><td>0.38Â </td><td>MPD011e05_fÂ </td><td>486Â </td><td>7Eâˆ’66Â </td></tr><tr><td> Caffeoyl-CoA <em>O</em> -methyltransferase Â </td><td>0.93Â </td><td>0.43Â </td><td>0.46Â </td><td>MWM071f11_rÂ </td><td>240Â </td><td>0.00008Â </td></tr><tr><td>Isoprene synthaseÂ </td><td>1.82Â </td><td>0.26Â </td><td>0.39Â </td><td>MWL054c12_rÂ </td><td>686Â </td><td>3Eâˆ’37Â </td></tr><tr><td>Lupeol synthaseÂ </td><td>0.46Â </td><td>0.19Â </td><td>0.56Â </td><td>GNf046g09Â </td><td>309Â </td><td>8Eâˆ’50Â </td></tr><tr><td><em>Transport/membrane</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Phosphate transporter (LjPT1)Â </td><td>0.36Â </td><td>0.18Â </td><td>0.68Â </td><td>MWM077d10_rÂ </td><td>334Â </td><td>4Eâˆ’59Â </td></tr><tr><td>Mitochondrial dicarboxylate carrier proteinÂ </td><td>1.01Â </td><td>0.35Â </td><td>0.54Â </td><td>MPD024c12_fÂ </td><td>266Â </td><td>7Eâˆ’14Â </td></tr><tr><td> Plasma membrane Ca <sup>2+</sup> -ATPase Â </td><td>1.05Â </td><td>0.38Â </td><td>0.26Â </td><td>GENLf026c07Â </td><td>473Â </td><td>5Eâˆ’28Â </td></tr><tr><td>Sucrose transport proteinÂ </td><td>0.40Â </td><td>0.39Â </td><td>0.53Â </td><td>MWM221d11_rÂ </td><td>342Â </td><td>2Eâˆ’47Â </td></tr><tr><td><em>Signal transduction</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Putative acid phosphataseÂ </td><td>0.85Â </td><td>0.39Â </td><td>0.23Â </td><td>MWM048e06_rÂ </td><td>1670Â </td><td>8Eâˆ’24Â </td></tr><tr><td><em>Transcription/translation</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Transcription factor WRKY4Â </td><td>0.35Â </td><td>0.19Â </td><td>0.18Â </td><td>MWM168c07_rÂ </td><td>690Â </td><td>4Eâˆ’45Â </td></tr><tr><td>WRKY-type DNA binding proteinÂ </td><td>0.50Â </td><td>0.35Â </td><td>0.24Â </td><td>MWM240a07_rÂ </td><td>624Â </td><td>5Eâˆ’07Â </td></tr><tr><td>WRKY DNA-binding proteinÂ </td><td>0.36Â </td><td>0.35Â </td><td>0.31Â </td><td>MR083f05_fÂ </td><td>263Â </td><td>0.0001Â </td></tr><tr><td><em>Cell wall</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Extensin-like proteinÂ </td><td>1.01Â </td><td>0.35Â </td><td>0.48Â </td><td>MWM170b07_rÂ </td><td>911Â </td><td>0.0004Â </td></tr><tr><td><em>Pathogen-related</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>PeroxidaseÂ </td><td>0.46Â </td><td>0.22Â </td><td>0.44Â </td><td>GENf076g12Â </td><td>384Â </td><td>2Eâˆ’17Â </td></tr><tr><td>PeroxidaseÂ </td><td>0.47Â </td><td>0.37Â </td><td>0.76Â </td><td>GNf069g02Â </td><td>136Â </td><td>6Eâˆ’14Â </td></tr><tr><td>Syringolide-induced proteinÂ </td><td>0.32Â </td><td>0.23Â </td><td>0.20Â </td><td>MWM033e05_rÂ </td><td>745Â </td><td>8Eâˆ’14Â </td></tr><tr><td>Syringolide-induced proteinÂ </td><td>0.35Â </td><td>0.29Â </td><td>0.39Â </td><td>GNf002b04Â </td><td>258Â </td><td>2Eâˆ’23Â </td></tr><tr><td>Syringolide-induced proteinÂ </td><td>0.99Â </td><td>0.39Â </td><td>0.15Â </td><td>GNf095h04Â </td><td>258Â </td><td>6Eâˆ’31Â </td></tr><tr><td>Similar to the BURP domainÂ </td><td>0.56Â </td><td>0.22Â </td><td>0.13Â </td><td>MPDL062c05_fÂ </td><td>4760Â </td><td>1Eâˆ’48Â </td></tr><tr><td>Seed coat BURP domain proteinÂ </td><td>0.42Â </td><td>0.25Â </td><td>0.15Â </td><td>MPDL082d06_fÂ </td><td>3753Â </td><td>9Eâˆ’59Â </td></tr><tr><td>Seed coat BURP domain proteinÂ </td><td>0.52Â </td><td>0.35Â </td><td>0.10Â </td><td>MPD013h01_fÂ </td><td>4069Â </td><td>2Eâˆ’18Â </td></tr><tr><td>PR10-1 proteinÂ </td><td>0.62Â </td><td>0.35Â </td><td>0.82Â </td><td>GNf017d12Â </td><td>1268Â </td><td>8Eâˆ’16Â </td></tr><tr><td><em>Other enzymes</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Cytochrome P450-1Â </td><td>0.18Â </td><td>0.17Â </td><td>0.16Â </td><td>MR095g09_fÂ </td><td>678Â </td><td>1Eâˆ’19Â </td></tr><tr><td>Mannan endo-1,4-beta-mannosidaseÂ </td><td>0.55Â </td><td>0.22Â </td><td>0.59Â </td><td>MWM099c01_rÂ </td><td>304Â </td><td>3Eâˆ’57Â </td></tr><tr><td>Soluble inorganic pyrophosphataseÂ </td><td>0.75Â </td><td>0.35Â </td><td>0.28Â </td><td>MPD001e10_fÂ </td><td>603Â </td><td>8Eâˆ’07Â </td></tr><tr><td>Cytochrome P450, putativeÂ </td><td>0.38Â </td><td>0.30Â </td><td>0.28Â </td><td>MR076b02_fÂ </td><td>233Â </td><td>4Eâˆ’10Â </td></tr><tr><td> Glutathione <em>S</em> -transferase Â </td><td>0.30Â </td><td>0.35Â </td><td>0.85Â </td><td>MWM067e12_rÂ </td><td>1452Â </td><td>3Eâˆ’33Â </td></tr><tr><td>Cytochrome P450 82C1Â </td><td>1.28Â </td><td>0.36Â </td><td>0.51Â </td><td>MR061f02_fÂ </td><td>219Â </td><td>3Eâˆ’35Â </td></tr><tr><td>ATP synthase 9Â </td><td>0.24Â </td><td>0.20Â </td><td>0.21Â </td><td>MWM223c10_rÂ </td><td>456Â </td><td>3Eâˆ’22Â </td></tr><tr><td><em>Other category</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Putative acyl-CoA oxidaseÂ </td><td>0.26Â </td><td>0.18Â </td><td>0.19Â </td><td>MWM217b02_rÂ </td><td>628Â </td><td>1Eâˆ’42Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p> The data in the Gm3/SC3 and Gm6/SC3 columns are the gene expression levels in roots 3 and 6 weeks, respectively, after inoculation of the whole <em>G. mosseae</em> inoculum relative to those in control roots 3 weeks after inoculation of the sieved carrier. Since prolonged cultivation with low concentrations of phosphate may cause stress, <sup><a reveal-id="DSM014C18" data-open="DSM014C18" class="link link-ref link-reveal xref-bibr">18</a></sup> we do not think that there is any problem with the use of younger control roots. The data in the Gi8/gni3 column are the gene expression levels in roots 8 weeks after inoculation of NaClO-treated <em>G. margarita</em> relative to those in roots 3 weeks after mock-inoculation. MaxRE is the highest normalized expression level in the experiments. Genes that match hypothetical proteins of unknown function and ones that encode proteins exhibiting no homology to thus far known ones have been omitted from this table. </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/37284003" target="_blank">View Large</a></div></div>                            <h2 scrollto-destination=37284004 id="37284004" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=37284005 id="37284005" class="section-title" data-legacy-id=s3a>3.1. Setting up cDNA array experiments with AM roots</h3>
<p> In this work, we made use of a large-scale array of cDNAs from entire seedlings, pods, roots, and root nodules of <em>L. japonicus</em> . <sup><a reveal-id="DSM014C26" data-open="DSM014C26" class="link link-ref link-reveal xref-bibr">26</a> , <a reveal-id="DSM014C27" data-open="DSM014C27" class="link link-ref link-reveal xref-bibr">27</a></sup> We grew <em>L. japonicus</em> plants with or without AM fungi in the presence of 250 ÂµM phosphate. At that phosphate concentration, AM fungi colonized well and the effects of phosphate depletion did not need to be taken into account. Under our growth conditions, the root length colonization by <em>G. mosseae</em> was 20 and 60% at 3 and 6 weeks after inoculation, respectively. The colonization by <em>G. margarita</em> was 10 and 30â€“40% at 5 and 8 weeks after inoculation, respectively. It has been pointed out that fungal transcripts account for up to 12% of the entire transcripts of AM roots, <sup><a reveal-id="DSM014C37" data-open="DSM014C37" class="link link-ref link-reveal xref-bibr">37</a></sup> which sometimes complicates analysis. <sup><a reveal-id="DSM014C10" data-open="DSM014C10" class="link link-ref link-reveal xref-bibr">10</a> , <a reveal-id="DSM014C12  DSM014C15" data-open="DSM014C12  DSM014C15" class="link link-ref link-reveal xref-bibr">12â€“15</a></sup> On the other hand, since the plant materials used for our array were grown avoiding microorganisms other than <em>M. loti</em> , a nitrogen-fixing symbiont of <em>L. japonicus</em> , our array did not contain fungal genes, making the analysis of global plant gene-expression easier. </p><p> Extraction of RNA, preparation of radioactive targets, and hybridization were principally performed for two biological replicates, although in the experiments involving <em>G. margarita</em> , the procedures were carried out in duplicate for a single biological replicate. The signal intensities of array filters in each experiment were normalized as described previously. <sup><a reveal-id="DSM014C26" data-open="DSM014C26" class="link link-ref link-reveal xref-bibr">26</a></sup> When the normalized signal intensities were compared after the two independent series of experiments, the variation was found to be basically within the twofold expression ratio (Supplementary Fig. S1A), indicating the sufficient reproducibility of our experiments. </p><p> In the initial experiments, we compared the gene-expression patterns of AM roots formed with a commercial inoculant and sterile non-infected roots, as in most previous studies. <sup><a reveal-id="DSM014C12  DSM014C15" data-open="DSM014C12  DSM014C15" class="link link-ref link-reveal xref-bibr">12â€“15</a> , <a reveal-id="DSM014C17  DSM014C19" data-open="DSM014C17  DSM014C19" class="link link-ref link-reveal xref-bibr">17â€“19</a></sup> Then, we picked up statistically significantly different genes expressed in roots 6 weeks after inoculation of the whole <em>G. mosseae</em> inoculum compared with those in control roots 3 weeks after inoculation of the sieved carrier, by means of the Significance Analysis of Microarrays Program. <sup><a reveal-id="DSM014C38" data-open="DSM014C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Supplementary Table S1 shows a list of the apparently up-regulated genes in AM roots. Closely related genes annotated as caffeic acid <em>O</em> -methyltransferase were most differently expressed. Lectin genes were also differentially expressed, as previously reported. <sup><a reveal-id="DSM014C13" data-open="DSM014C13" class="link link-ref link-reveal xref-bibr">13</a> , <a reveal-id="DSM014C15" data-open="DSM014C15" class="link link-ref link-reveal xref-bibr">15</a> , <a reveal-id="DSM014C19" data-open="DSM014C19" class="link link-ref link-reveal xref-bibr">19</a></sup> The up-regulation of a gene for subtilisin-like serine protease was similar to the finding of Liu <em>et al.</em> , <sup><a reveal-id="DSM014C16" data-open="DSM014C16" class="link link-ref link-reveal xref-bibr">16</a></sup> although serine carboxypeptidase genes were not listed in our experiment. The differential expression of chitinase genes was in accordance with a previous study. <sup><a reveal-id="DSM014C23" data-open="DSM014C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Glutathione <em>S</em> -transferase genes were reported to be up-regulated in AM roots. <sup><a reveal-id="DSM014C12  DSM014C14" data-open="DSM014C12  DSM014C14" class="link link-ref link-reveal xref-bibr">12â€“14</a> , <a reveal-id="DSM014C20" data-open="DSM014C20" class="link link-ref link-reveal xref-bibr">20</a></sup> In our experiment, a gene for glutathione <em>S</em> -transferase (GNf044a01) was also up-regulated by 1.61- and 2.51-fold 3 and 6 weeks, respectively, after inoculantion (not included in the supplementary table). The expression levels of blue copper protein genes <sup><a reveal-id="DSM014C12" data-open="DSM014C12" class="link link-ref link-reveal xref-bibr">12</a> , <a reveal-id="DSM014C13" data-open="DSM014C13" class="link link-ref link-reveal xref-bibr">13</a> , <a reveal-id="DSM014C18" data-open="DSM014C18" class="link link-ref link-reveal xref-bibr">18</a> , <a reveal-id="DSM014C20" data-open="DSM014C20" class="link link-ref link-reveal xref-bibr">20</a></sup> varied from experiment to experiment under our conditions (data not shown). Overall, the data in Supplementary Table S1 are consistent with those in previous papers. <sup><a reveal-id="DSM014C12  DSM014C20" data-open="DSM014C12  DSM014C20" class="link link-ref link-reveal xref-bibr">12â€“20</a></sup></p><p> Notably, when the <em>G. mosseae</em> inoculum suspension and sieved carrier were diluted and streaked on yeast extract/peptone/glucose plates, many colonies of contaminating microorganisms appeared, their numbers and appearances being similar to each other (data not shown). Thus, the above cDNA array analysis was performed in the constant presence of background microorganisms in the AM root material. We next filtered the <em>G. mosseae</em> inoculant suspension through a 38 Âµm stainless mesh, <em>L. japonicus</em> seedlings were grown in the presence of the filtrate, and then the gene expression in the resulting roots was compared with that in non-infected ones. Supplementary Table S2 shows the effects of contaminating microorganisms. Genes encoding PAL, chalcone synthase and chalcone reductase, which are involved in important steps of flavonoid phytoalexin synthesis, <sup><a reveal-id="DSM014C39" data-open="DSM014C39" class="link link-ref link-reveal xref-bibr">39</a></sup> and WRKY transcription factors, which are mainly involved in tolerance to pathogen-related stress, <sup><a reveal-id="DSM014C40" data-open="DSM014C40" class="link link-ref link-reveal xref-bibr">40</a></sup> were remarkably induced. The genes annotated as caffeic acid <em>O</em> -methyltransferase, and those for chitinase and glutathione <em>S</em> -transferase were also induced. Therefore, the results in Supplementary Table S1 represent super-positioning of the effects of the AM fungus and contaminating microorganisms in the inoculant. It is noteworthy that commercial AM fungus inoculants have been used easily in a number of investigations on plant gene expression in AM roots. <sup><a reveal-id="DSM014C12  DSM014C15" data-open="DSM014C12  DSM014C15" class="link link-ref link-reveal xref-bibr">12â€“15</a> , <a reveal-id="DSM014C17  DSM014C19" data-open="DSM014C17  DSM014C19" class="link link-ref link-reveal xref-bibr">17â€“19</a></sup> Care must be taken regarding contamination in nurse plants used for inoculation of the AM fungus. <sup><a reveal-id="DSM014C10" data-open="DSM014C10" class="link link-ref link-reveal xref-bibr">10</a></sup> On the other hand, in the works of Liu <em>et al.</em><sup><a reveal-id="DSM014C16" data-open="DSM014C16" class="link link-ref link-reveal xref-bibr">16</a> , <a reveal-id="DSM014C20" data-open="DSM014C20" class="link link-ref link-reveal xref-bibr">20</a></sup> and Salzer et al. <sup><a reveal-id="DSM014C23" data-open="DSM014C23" class="link link-ref link-reveal xref-bibr">23</a></sup> on <em>M. truncatula</em> , Guimil <em>et al.</em><sup><a reveal-id="DSM014C41" data-open="DSM014C41" class="link link-ref link-reveal xref-bibr">41</a></sup> on rice, and ours on <em>L. japonicus</em> and <em>G. margarita</em> (see below), aseptic spores of AM fungi were inoculated into plants, making the populations of contaminating microorganisms, if any, similar between AM roots and control roots. </p>                            <h3 scrollto-destination=37284010 id="37284010" class="section-title" data-legacy-id=s3b>3.2. Expression profiling of up- and down-regulated plant genes after colonization by AM fungi</h3>
<p> In order to subtract the above-described effects of contaminating microorganisms, we compared the gene expression patterns of AM roots inoculated with the whole <em>G. mosseae</em> inoculum and control roots inoculated with the sieved carrier only. When the average intensities on duplicate determination of gene expression were compared, the patterns indicated a significant difference in gene expression (Supplementary Fig. S1B). AM-enhanced genes were first identified after colonization by <em>G. mosseae</em> and <em>G. margarita</em> (TableÂ  <a reveal-id="DSM014TB1" data-open="DSM014TB1" class="link link-reveal link-table xref-fig">1</a> ) because they have attracted more interest than repressed ones. <sup><a reveal-id="DSM014C12  DSM014C15" data-open="DSM014C12  DSM014C15" class="link link-ref link-reveal xref-bibr">12â€“15</a> , <a reveal-id="DSM014C17" data-open="DSM014C17" class="link link-ref link-reveal xref-bibr">17</a> , <a reveal-id="DSM014C19" data-open="DSM014C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Genes for aquaporins, also annotated as plasma-membrane intrinsic protein, tonoplast intrinsic protein, and nodulin 26-like protein, were up-regulated in AM roots, confirming the results in several reports. <sup><a reveal-id="DSM014C14" data-open="DSM014C14" class="link link-ref link-reveal xref-bibr">14</a> , <a reveal-id="DSM014C17" data-open="DSM014C17" class="link link-ref link-reveal xref-bibr">17</a> , <a reveal-id="DSM014C19" data-open="DSM014C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Annexin genes were also induced in AM roots, in accordance with Manthey <em>et al.</em><sup><a reveal-id="DSM014C17" data-open="DSM014C17" class="link link-ref link-reveal xref-bibr">17</a></sup></p><p> Four cysteine proteinase genes, designated as <em>LjCyp1-4</em> , were most obviously up-regulated among the AM-enhanced genes (TableÂ  <a reveal-id="DSM014TB1" data-open="DSM014TB1" class="link link-reveal link-table xref-fig">1</a> ), confirming previous reports. <sup><a reveal-id="DSM014C10" data-open="DSM014C10" class="link link-ref link-reveal xref-bibr">10</a> , <a reveal-id="DSM014C16" data-open="DSM014C16" class="link link-ref link-reveal xref-bibr">16</a> â€“ <a reveal-id="DSM014C18" data-open="DSM014C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Although there were around 20 cysteine proteinase genes on our array membrane, the expression of other genes did not change or was rather repressed in AM roots. Real-time RTâ€“PCR showed that <em>LjCyp2</em> , a representative of the four genes, was induced only at the late stage of <em>G. mosseae</em> colonization (Fig.Â  <a reveal-id="DSM014F1" data-open="DSM014F1" class="link link-reveal link-table xref-fig">1</a> A). The expression of <em>LjCyp2</em> was also high in <em>G. margarita</em> -colonized roots at the late stage (data not shown). Our <em>in situ</em> localization revealed that the induced <em>LjCyp2</em> gene was specifically expressed in arbuscule-containing inner cortical cells of <em>G. mosseae</em> -colonized roots (Fig.Â  <a reveal-id="DSM014F1" data-open="DSM014F1" class="link link-reveal link-table xref-fig">1</a> C). The <em>LjCyp1</em> transcript showed a very similar localization (not shown) to that of <em>LjCyp2</em> . The spatial expression patterns of AM-induced genes fall into two groups. The glutathione <em>S</em> -transferase, <sup><a reveal-id="DSM014C13" data-open="DSM014C13" class="link link-ref link-reveal xref-bibr">13</a></sup> serine carboxypeptidase, <sup><a reveal-id="DSM014C16" data-open="DSM014C16" class="link link-ref link-reveal xref-bibr">16</a></sup> annexin, <sup><a reveal-id="DSM014C17" data-open="DSM014C17" class="link link-ref link-reveal xref-bibr">17</a></sup> and calcium-binding protein <sup><a reveal-id="DSM014C10" data-open="DSM014C10" class="link link-ref link-reveal xref-bibr">10</a></sup> genes were reported to be expressed not only in arbuscule-containing cells but also in the cells around them. In contrast, the endoglucanase (MtCel1), <sup><a reveal-id="DSM014C16" data-open="DSM014C16" class="link link-ref link-reveal xref-bibr">16</a></sup> cysteine-rich antifungal protein, <sup><a reveal-id="DSM014C19" data-open="DSM014C19" class="link link-ref link-reveal xref-bibr">19</a></sup> and AM-induced phosphate transporter <sup><a reveal-id="DSM014C33" data-open="DSM014C33" class="link link-ref link-reveal xref-bibr">33</a></sup> genes were specifically expressed in cells that contained fungal arbuscules. The present study revealed that the <em>Lotus</em> cysteine proteinase genes are members of the latter group. It is noteworthy that the cysteine proteinase genes are expressed early in cells containing arbuscules just after maturation, whereas their levels are quite low in cells with very young arbuscules (Fig.Â  <a reveal-id="DSM014F1" data-open="DSM014F1" class="link link-reveal link-table xref-fig">1</a> E). The induced cysteine proteinases may be involved in the degradation of arbuscules, short-lived fungal organs, <sup><a reveal-id="DSM014C2" data-open="DSM014C2" class="link link-ref link-reveal xref-bibr">2</a></sup> since the PSORT program ( <a class="link link-uri" href="http://psort.nibb.ac.jp/">http://psort.nibb.ac.jp/</a> ) predicted that they are secreted proteins. Alternatively, these proteases may stay within the cells, e.g. in vacuoles <sup><a reveal-id="DSM014C42" data-open="DSM014C42" class="link link-ref link-reveal xref-bibr">42</a> , <a reveal-id="DSM014C43" data-open="DSM014C43" class="link link-ref link-reveal xref-bibr">43</a></sup> and play important roles in remodeling of intracellular structures, cell cycle progression, protein turnover etc. It is also interesting that the four cysteine proteinase genes are exactly the same genes as those that are highly induced in early-senescent root nodules of ineffective nitrogen fixation. <sup><a reveal-id="DSM014C27" data-open="DSM014C27" class="link link-ref link-reveal xref-bibr">27</a></sup></p>                        <a id="37284013" scrollto-destination="37284013"></a>
<div id="DSM014F1" data-id="DSM014F1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm014/3/m_dsm01401.jpeg?Expires=2147483647&Signature=A1rYn48SrOC3Syso38FdDohNv0VHfVXKmZvPFzJ-ptlOeIwwANBzvM13vNhtd3YOuw1QLUn-BU4Y~DQoPp614olI6UwBCxr87chkdsSd1vxXItXmBqgytpKlelGRPicNJA89~ViN-t5XBvtVmytz3jNErqDjhhEHnalDV4N9K4eIQcxHu1TqFw5TLT-Gw199Ytf~MdHVT~zTzFjLkn-mwgqsoLF2wPQ9NnzhL2~u1QV-WpwYaadfaKV0vCdniWK-y9XUQlWrY5J~Z81v728RWsAHffxzGBejhn4yhtdFuvriqZpeIWH3-bYLLaEj0-xLOrsld2Aw6kgFvLQbpxrbIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Expression levels and in situ localization of the LjCyp2 gene. The expression levels of the LjCyp2 ( A ) and ÃŸ-actin ( B ) genes relative to those in control roots 3 weeks after inoculation of the sieved carrier (SC3, level = 1) were determined by real-time RTâ€“PCR. The means and variation of two independent experiments are shown. ni3, sterile non-infected roots grown for 3 weeks; Gm6, roots 6 weeks after inoculation of the whole G. mosseae inoculum; gni1, roots 1 week after mock-inoculation; and Gi1, roots 1 week after inoculation of G. margarita (see Materials and Methods and the legend to TableÂ  3 for details). Longitudinal AM root sections were probed with digoxygenine-labeled antisense RNA prepared from the entire LjCyp2 cDNA ( C and E ). Hybridization signals are visible as a dark blue color. When sense RNA was used as a negative control probe, much lower hybridization signals were detected except in central cylinders ( D ). Bars, 50 Âµm. " path-from-xml="dsm01401.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/37284013/dsm01401.jpeg" path-from-xml="dsm01401.jpeg" target="_blank">View large</a><a section="37284013" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm014/3/dsm01401.jpeg?Expires=2147483647&Signature=diza5FQ2srHAUWDGMwcSy3nGGlYEFlEHW~oUYg6DXliMR7AFdlgLO3z~m4kVorYHZeCajfKPtCsPu8EQxjj13HnnCENLpzgxRh1FVL71d8FzLUHjkEKUQyBeWjRqGptLpVEMYFR2GKyWhYhNhl3IMqYHlxkLysq1AyWqrbEo7VSUBtDVQnATQpaPvbCBr7jeHR0rKnBCb4aAgl0mONPef4UHFBwdKRmrnwUZ94E4X45UfTqIcXPvHyhUwaJH~~UwToT6BkS039gmPkoxCQmD2BrlRpHN9Exfeq3D-SexcHjG8mWt~niUqUb4t~HDm79WLr~iTHzgC9Aqpp52HiVoHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=37284013&ar=578022&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01401.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Expression levels and <em>in situ</em> localization of the <em>LjCyp2</em> gene. The expression levels of the <em>LjCyp2</em> ( <strong>A</strong> ) and <em>ÃŸ-actin</em> ( <strong>B</strong> ) genes relative to those in control roots 3 weeks after inoculation of the sieved carrier (SC3, level = 1) were determined by real-time RTâ€“PCR. The means and variation of two independent experiments are shown. ni3, sterile non-infected roots grown for 3 weeks; Gm6, roots 6 weeks after inoculation of the whole <em>G. mosseae</em> inoculum; gni1, roots 1 week after mock-inoculation; and Gi1, roots 1 week after inoculation of <em>G. margarita</em> (see Materials and Methods and the legend to TableÂ  <a reveal-id="DSM014TB3" data-open="DSM014TB3" class="link link-reveal link-table xref-fig">3</a> for details). Longitudinal AM root sections were probed with digoxygenine-labeled antisense RNA prepared from the entire <em>LjCyp2</em> cDNA ( <strong>C</strong> and <strong>E</strong> ). Hybridization signals are visible as a dark blue color. When sense RNA was used as a negative control probe, much lower hybridization signals were detected except in central cylinders ( <strong>D</strong> ). Bars, 50 Âµm. </p></div></div><div content-id="DSM014F1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm014/3/m_dsm01401.jpeg?Expires=2147483647&Signature=A1rYn48SrOC3Syso38FdDohNv0VHfVXKmZvPFzJ-ptlOeIwwANBzvM13vNhtd3YOuw1QLUn-BU4Y~DQoPp614olI6UwBCxr87chkdsSd1vxXItXmBqgytpKlelGRPicNJA89~ViN-t5XBvtVmytz3jNErqDjhhEHnalDV4N9K4eIQcxHu1TqFw5TLT-Gw199Ytf~MdHVT~zTzFjLkn-mwgqsoLF2wPQ9NnzhL2~u1QV-WpwYaadfaKV0vCdniWK-y9XUQlWrY5J~Z81v728RWsAHffxzGBejhn4yhtdFuvriqZpeIWH3-bYLLaEj0-xLOrsld2Aw6kgFvLQbpxrbIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Expression levels and in situ localization of the LjCyp2 gene. The expression levels of the LjCyp2 ( A ) and ÃŸ-actin ( B ) genes relative to those in control roots 3 weeks after inoculation of the sieved carrier (SC3, level = 1) were determined by real-time RTâ€“PCR. The means and variation of two independent experiments are shown. ni3, sterile non-infected roots grown for 3 weeks; Gm6, roots 6 weeks after inoculation of the whole G. mosseae inoculum; gni1, roots 1 week after mock-inoculation; and Gi1, roots 1 week after inoculation of G. margarita (see Materials and Methods and the legend to TableÂ  3 for details). Longitudinal AM root sections were probed with digoxygenine-labeled antisense RNA prepared from the entire LjCyp2 cDNA ( C and E ). Hybridization signals are visible as a dark blue color. When sense RNA was used as a negative control probe, much lower hybridization signals were detected except in central cylinders ( D ). Bars, 50 Âµm. " path-from-xml="dsm01401.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/37284013/dsm01401.jpeg" path-from-xml="dsm01401.jpeg" target="_blank">View large</a><a section="37284013" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm014/3/dsm01401.jpeg?Expires=2147483647&Signature=diza5FQ2srHAUWDGMwcSy3nGGlYEFlEHW~oUYg6DXliMR7AFdlgLO3z~m4kVorYHZeCajfKPtCsPu8EQxjj13HnnCENLpzgxRh1FVL71d8FzLUHjkEKUQyBeWjRqGptLpVEMYFR2GKyWhYhNhl3IMqYHlxkLysq1AyWqrbEo7VSUBtDVQnATQpaPvbCBr7jeHR0rKnBCb4aAgl0mONPef4UHFBwdKRmrnwUZ94E4X45UfTqIcXPvHyhUwaJH~~UwToT6BkS039gmPkoxCQmD2BrlRpHN9Exfeq3D-SexcHjG8mWt~niUqUb4t~HDm79WLr~iTHzgC9Aqpp52HiVoHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=37284013&ar=578022&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01401.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Expression levels and <em>in situ</em> localization of the <em>LjCyp2</em> gene. The expression levels of the <em>LjCyp2</em> ( <strong>A</strong> ) and <em>ÃŸ-actin</em> ( <strong>B</strong> ) genes relative to those in control roots 3 weeks after inoculation of the sieved carrier (SC3, level = 1) were determined by real-time RTâ€“PCR. The means and variation of two independent experiments are shown. ni3, sterile non-infected roots grown for 3 weeks; Gm6, roots 6 weeks after inoculation of the whole <em>G. mosseae</em> inoculum; gni1, roots 1 week after mock-inoculation; and Gi1, roots 1 week after inoculation of <em>G. margarita</em> (see Materials and Methods and the legend to TableÂ  <a reveal-id="DSM014TB3" data-open="DSM014TB3" class="link link-reveal link-table xref-fig">3</a> for details). Longitudinal AM root sections were probed with digoxygenine-labeled antisense RNA prepared from the entire <em>LjCyp2</em> cDNA ( <strong>C</strong> and <strong>E</strong> ). Hybridization signals are visible as a dark blue color. When sense RNA was used as a negative control probe, much lower hybridization signals were detected except in central cylinders ( <strong>D</strong> ). Bars, 50 Âµm. </p></div></div><p> A promoter region of a calcium-binding protein gene of <em>L. japonicus</em> was reported to be activated during AM development. <sup><a reveal-id="DSM014C10" data-open="DSM014C10" class="link link-ref link-reveal xref-bibr">10</a></sup> We found that a cDNA for the calcium-binding protein (MWM036h04_r) is present on our array membrane. Unexpectedly, however, the mRNA level did not show significant variation under our experimental conditions. The mRNA level of the gene did not vary on root-nodule formation, either ( <a class="link link-uri" href="http://est.kazusa.or.jp/en/plant/lotus/EST/cDNA.html">http://est.kazusa.or.jp/en/plant/lotus/EST/cDNA.html</a> ). The promoter activity of the gene may not coincide with its transcript level. </p><p> In the present study, AM-repressed genes were also identified after colonization by <em>G. mosseae</em> or <em>G. margarita</em> (TableÂ  <a reveal-id="DSM014TB2" data-open="DSM014TB2" class="link link-reveal link-table xref-fig">2</a> ). Five <em>PAL</em> genes were repressed most drastically after colonization by <em>G. mosseae</em> . In addition, four and two genes for chalcone synthase and chalcone reductase, respectively, were found to be repressed. These three enzymes catalyze key reactions in the biosynthesis of phenylpropanoid compounds. Another series of duplicate experiments involving <em>G. margarita</em> supported this finding. Thus, the reproducibility of the repression of phenylpropanoid biosynthesis-related genes was confirmed unequivocally. Liu <em>et al.</em> presented a small list of AM-repressed genes. <sup><a reveal-id="DSM014C16" data-open="DSM014C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Our finding that particular forms of phosphoenolpyruvate carboxylase and glutathione <em>S</em> -transferase are repressed is in accordance with their results. Hohnjec <em>et al.</em> , <sup><a reveal-id="DSM014C18" data-open="DSM014C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Kistner <em>et al.</em> , <sup><a reveal-id="DSM014C10" data-open="DSM014C10" class="link link-ref link-reveal xref-bibr">10</a></sup> and Guimil <em>et al.</em><sup><a reveal-id="DSM014C41" data-open="DSM014C41" class="link link-ref link-reveal xref-bibr">41</a></sup> presented larger lists of AM-repressed genes of <em>M. truncatula</em> , <em>L. japonicus</em> , and rice, respectively, but neither <em>PAL</em> genes nor chalcone synthase ones were included in the lists. In the work of Hohnjec <em>et al.</em> , <sup><a reveal-id="DSM014C18" data-open="DSM014C18" class="link link-ref link-reveal xref-bibr">18</a></sup> for example, many stress-related genes were listed as 	AM-repressed genes, because they were highly up-regulated in the phosphate-starved control roots. Very recently, Liu <em>et al.</em> presented the largest list of AM-repressed genes in <em>M. truncatula</em> roots as well as those in other portions. <sup><a reveal-id="DSM014C20" data-open="DSM014C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Again, however, PAL genes were not included in their list of repressed genes. We will confirm our current results by promoter analysis and discuss the discrepancy (see below). Besides phenylpropanoid biosynthesis-related genes, a phosphate transporter gene ( <em>LjPT1</em> ) was also repressed (TableÂ  <a reveal-id="DSM014TB2" data-open="DSM014TB2" class="link link-reveal link-table xref-fig">2</a> ). This finding is in accord with the general tendency that the expression of common phosphate transporters is suppressed in AM roots. <sup><a reveal-id="DSM014C44" data-open="DSM014C44" class="link link-ref link-reveal xref-bibr">44</a> â€“ <a reveal-id="DSM014C46" data-open="DSM014C46" class="link link-ref link-reveal xref-bibr">46</a></sup> A recently found AM root-enhanced phosphate transporter gene of <em>L. japonicus</em><sup><a reveal-id="DSM014C33" data-open="DSM014C33" class="link link-ref link-reveal xref-bibr">33</a></sup> was not found on the present nylon filter. </p>                            <h3 scrollto-destination=37284016 id="37284016" class="section-title" data-legacy-id=s3c>3.3. Differential expression of plant genes caused by G. margarita infection in the initial stage of symbiosis</h3>
<p> In contrast to the later stage of symbiosis (TableÂ  <a reveal-id="DSM014TB1" data-open="DSM014TB1" class="link link-reveal link-table xref-fig">1</a> ), a number of genes were found to be up-regulated or down-regulated on the initial association with the AM fungus (TableÂ  <a reveal-id="DSM014TB3" data-open="DSM014TB3" class="link link-reveal link-table xref-fig">3</a> ). In accordance with previous reports, <sup><a reveal-id="DSM014C16" data-open="DSM014C16" class="link link-ref link-reveal xref-bibr">16</a> , <a reveal-id="DSM014C47" data-open="DSM014C47" class="link link-ref link-reveal xref-bibr">47</a> â€“ <a reveal-id="DSM014C52" data-open="DSM014C52" class="link link-ref link-reveal xref-bibr">52</a></sup> the genes for enzymes involved in defense-related secondary metabolism and the pathogen response, such as PALs, chalcone synthases, and peroxidases, were moderately up-regulated at this stage. A number of genes for transcription or translation were also induced, suggesting that a dynamic cellular change in plant roots occurs at the initial stage of the AM association. In addition, several genes involved in signal transduction were up-regulated (TableÂ  <a reveal-id="DSM014TB3" data-open="DSM014TB3" class="link link-reveal link-table xref-fig">3</a> ). For example, the gene for a pathogen-induced receptor protein kinase with a characteristic extracellular domain was induced. <sup><a reveal-id="DSM014C53" data-open="DSM014C53" class="link link-ref link-reveal xref-bibr">53</a> , <a reveal-id="DSM014C54" data-open="DSM014C54" class="link link-ref link-reveal xref-bibr">54</a></sup> Transcripts for a heterotrimeric G protein-coupled receptor, small GTP-binding proteins, protein serine/threonine kinases, and a mitogen-activated protein kinase were also accumulated. These gene products may represent signal transduction pathways for AM colonization. </p>                        <a id="37284018" scrollto-destination="37284018"></a>
<div content-id="DSM014TB3" class="table-modal"><div class="table-wrap"><div id="DSM014TB3" class="table-wrap-title" data-id="DSM014TB3"><div class="label -label"><strong>TableÂ 3</strong></div><div class="caption"><p> Transcriptional changes caused by <em>G. margarita</em> infection in the initial stage of symbiosis </p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Current annotationÂ </th><th>Fold (Gi1/gni1)Â </th><th>Gene IDÂ </th><th>gni1Â </th><th>Gi1Â </th><th>e-valueÂ </th></tr></thead><tbody><tr><td><em>Amino acid and nitrogen metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Selenocysteine methyltransferaseÂ </td><td>2.24Â </td><td>MWM066h10_rÂ </td><td>62Â </td><td>139Â </td><td>6Eâˆ’25Â </td></tr><tr><td>Diaminopimelate decarboxylaseÂ </td><td>2.05Â </td><td>MWM149b05_rÂ </td><td>128Â </td><td>262Â </td><td>1Eâˆ’66Â </td></tr><tr><td><em>S</em> -adenosyl methionine synthetase Â </td><td>2.02Â </td><td>MWM180f07_rÂ </td><td>145Â </td><td>292Â </td><td>4Eâˆ’67Â </td></tr><tr><td>VuP5CRÂ </td><td>0.28Â </td><td>GENLf018g02Â </td><td>78Â </td><td>21Â </td><td>4Eâˆ’24Â </td></tr><tr><td>Arginine decarboxylaseÂ </td><td>0.29Â </td><td>MWM198e12_rÂ </td><td>102Â </td><td>29Â </td><td>2Eâˆ’49Â </td></tr><tr><td>Aminotransferase 2Â </td><td>0.33Â </td><td>MWM222b09_rÂ </td><td>101Â </td><td>33Â </td><td>5Eâˆ’66Â </td></tr><tr><td>Proline dehydrogenaseÂ </td><td>0.36Â </td><td>MWM135h10_rÂ </td><td>99Â </td><td>35Â </td><td>8Eâˆ’29Â </td></tr><tr><td>Delta-1-pyrroline-5-carboxylate synthaseÂ </td><td>0.36Â </td><td>GENLf045e06Â </td><td>121Â </td><td>43Â </td><td>4Eâˆ’09Â </td></tr><tr><td><em>Carbon metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>InvertaseÂ </td><td>2.32Â </td><td>MWM224d02_rÂ </td><td>150Â </td><td>347Â </td><td>6Eâˆ’08Â </td></tr><tr><td>Sucrose synthaseÂ </td><td>2.27Â </td><td>MWL068h11_rÂ </td><td>271Â </td><td>614Â </td><td>1Eâˆ’58Â </td></tr><tr><td>Glucose-1-phosphate adenylyltransferaseÂ </td><td>2.13Â </td><td>MWM086h02_rÂ </td><td>128Â </td><td>271Â </td><td>7Eâˆ’65Â </td></tr><tr><td>Alpha-mannosidaseÂ </td><td>2.11Â </td><td>GENLf064h06Â </td><td>138Â </td><td>291Â </td><td>4Eâˆ’24Â </td></tr><tr><td>Glucose-6-phosphate dehydrogenaseÂ </td><td>2.07Â </td><td>MR098a03_fÂ </td><td>47Â </td><td>93Â </td><td>7Eâˆ’06Â </td></tr><tr><td>Beta-amylaseÂ </td><td>2.06Â </td><td>GENf097b02Â </td><td>111Â </td><td>229Â </td><td>1Eâˆ’132Â </td></tr><tr><td>Triosephosphate isomeraseÂ </td><td>2.05Â </td><td>MWM193g10_rÂ </td><td>272Â </td><td>559Â </td><td>1Eâˆ’77Â </td></tr><tr><td>Glucosyltransferase-like proteinÂ </td><td>2.01Â </td><td>MWL049f07_rÂ </td><td>126Â </td><td>257Â </td><td>9Eâˆ’06Â </td></tr><tr><td> Beta- <span class="small-caps">d</span> -xylosidase Â </td><td>0.26Â </td><td>MWM219c11_rÂ </td><td>119Â </td><td>31Â </td><td>9Eâˆ’55Â </td></tr><tr><td>Fructose-bisphosphate aldolaseÂ </td><td>0.29Â </td><td>MWM024h09_rÂ </td><td>95Â </td><td>28Â </td><td>2Eâˆ’39Â </td></tr><tr><td>Malonyl-CoA: acyl carrier protein transacylaseÂ </td><td>0.30Â </td><td>MWL014e01_rÂ </td><td>115Â </td><td>34Â </td><td>4Eâˆ’33Â </td></tr><tr><td>Putative 2-isopropylmalate synthaseÂ </td><td>0.31Â </td><td>GENf086f07Â </td><td>76Â </td><td>23Â </td><td>4Eâˆ’34Â </td></tr><tr><td>Mannosyltransferase-like proteinÂ </td><td>0.32Â </td><td>MWM235d07_rÂ </td><td>92Â </td><td>28Â </td><td>3Eâˆ’04Â </td></tr><tr><td>Citrate synthaseÂ </td><td>0.37Â </td><td>MWM239b12_rÂ </td><td>147Â </td><td>54Â </td><td>4Eâˆ’38Â </td></tr><tr><td><em>Secondary metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Chalcone reductaseÂ </td><td>3.03Â </td><td>MWM002d07_rÂ </td><td>98Â </td><td>306Â </td><td>1Eâˆ’60Â </td></tr><tr><td>Phenylalanine ammonia-lyase (LjPAL2)Â </td><td>2.16Â </td><td>GENLf058e04Â </td><td>95Â </td><td>202Â </td><td>0.006Â </td></tr><tr><td>Phenylalanine ammonia-lyase (LjPAL1)Â </td><td>2.09Â </td><td>MRL007g11_fÂ </td><td>136Â </td><td>283Â </td><td>9Eâˆ’38Â </td></tr><tr><td>4-coumarate:CoA ligaseÂ </td><td>2.10Â </td><td>MWL020d04_rÂ </td><td>110Â </td><td>235Â </td><td>3Eâˆ’34Â </td></tr><tr><td>Chalcone synthaseÂ </td><td>2.05Â </td><td>MWL020g05_rÂ </td><td>144Â </td><td>295Â </td><td>1Eâˆ’14Â </td></tr><tr><td>LaccaseÂ </td><td>0.30Â </td><td>MWM219c08_rÂ </td><td>124Â </td><td>36Â </td><td>3Eâˆ’44Â </td></tr><tr><td>Putative diphenol oxidaseÂ </td><td>0.34Â </td><td>GENf055e07Â </td><td>87Â </td><td>29Â </td><td>1Eâˆ’13Â </td></tr><tr><td><em>Transport/membrane</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td> Plasma membrane Ca2 <sup>+</sup> -ATPase Â </td><td>2.34Â </td><td>MWM178b05_rÂ </td><td>169Â </td><td>396Â </td><td>3Eâˆ’74Â </td></tr><tr><td>Aquaporin protein PIP1Â </td><td>2.34Â </td><td>MWL033d08_rÂ </td><td>279Â </td><td>652Â </td><td>8Eâˆ’51Â </td></tr><tr><td>Putative amino acid transporterÂ </td><td>2.14Â </td><td>MWL064b03_rÂ </td><td>71Â </td><td>150Â </td><td>8Eâˆ’51Â </td></tr><tr><td>Putative ABC transporter proteinÂ </td><td>2.13Â </td><td>MWL077b08_rÂ </td><td>158Â </td><td>335Â </td><td>1Eâˆ’14Â </td></tr><tr><td>Putative nuclear transport factorÂ </td><td>2.09Â </td><td>MWM105a05_rÂ </td><td>118Â </td><td>248Â </td><td>1Eâˆ’43Â </td></tr><tr><td>Vacuolar ATPaseÂ </td><td>2.06Â </td><td>MWM238d10_rÂ </td><td>84Â </td><td>173Â </td><td>4Eâˆ’56Â </td></tr><tr><td>Nuclear transport factorÂ </td><td>2.02Â </td><td>GNf048c09Â </td><td>84Â </td><td>165Â </td><td>1Eâˆ’38Â </td></tr><tr><td>Sorbitol transporterÂ </td><td>2.01Â </td><td>MWM244a03_rÂ </td><td>41Â </td><td>79Â </td><td>4Eâˆ’57Â </td></tr><tr><td>MATE efflux family proteinÂ </td><td>0.23Â </td><td>GENLf013e12Â </td><td>111Â </td><td>25Â </td><td>2Eâˆ’06Â </td></tr><tr><td>Cation-transporting ATPaseÂ </td><td>0.31Â </td><td>MWM087d06_rÂ </td><td>154Â </td><td>46Â </td><td>3Eâˆ’20Â </td></tr><tr><td>Plastidic phosphate translocator-like proteinÂ </td><td>0.32Â </td><td>GENf029g12Â </td><td>108Â </td><td>34Â </td><td>6Eâˆ’96Â </td></tr><tr><td><em>Nodulin</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Early nodulin ENOD18Â </td><td>2.65Â </td><td>GENf079d10Â </td><td>34Â </td><td>90Â </td><td>2Eâˆ’14Â </td></tr><tr><td><em>Signal transduction</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Protein phosphatase 2CÂ </td><td>2.40Â </td><td>MWM035c07_rÂ </td><td>248Â </td><td>596Â </td><td>1Eâˆ’52Â </td></tr><tr><td>Protein serine/threonine kinaseÂ </td><td>2.35Â </td><td>MWM206a09_rÂ </td><td>74Â </td><td>174Â </td><td>4Eâˆ’29Â </td></tr><tr><td>G protein-coupled receptorÂ </td><td>2.33Â </td><td>GENLf064g09Â </td><td>70Â </td><td>163Â </td><td>Â </td></tr><tr><td>Receptor protein kinaseÂ </td><td>2.31Â </td><td>MPDL044b10_fÂ </td><td>56Â </td><td>130Â </td><td>7Eâˆ’18Â </td></tr><tr><td>Small GTP-binding proteinÂ </td><td>2.18Â </td><td>MWM122f01_rÂ </td><td>174Â </td><td>380Â </td><td>5Eâˆ’44Â </td></tr><tr><td>GUN4 regulatorÂ </td><td>2.12Â </td><td>MPDL091h07_fÂ </td><td>149Â </td><td>316Â </td><td>1Eâˆ’63Â </td></tr><tr><td>Putative acid phosphataseÂ </td><td>2.09Â </td><td>MWM048e06_rÂ </td><td>415Â </td><td>866Â </td><td>8Eâˆ’24Â </td></tr><tr><td>MAP kinase 3Â </td><td>2.09Â </td><td>MR062e03_fÂ </td><td>582Â </td><td>1215Â </td><td>3Eâˆ’84Â </td></tr><tr><td>Putative GTP-binding proteinÂ </td><td>2.05Â </td><td>MWM119e05_rÂ </td><td>161Â </td><td>329Â </td><td>4Eâˆ’34Â </td></tr><tr><td>Protein phosphatase-2CÂ </td><td>2.05Â </td><td>MWM050f11_rÂ </td><td>89Â </td><td>182Â </td><td>0.0003Â </td></tr><tr><td>Serine/threonine protein phosphataseÂ </td><td>2.05Â </td><td>MR028a01_fÂ </td><td>85Â </td><td>173Â </td><td>2Eâˆ’27Â </td></tr><tr><td>Calcium-dependent protein kinaseÂ </td><td>2.01Â </td><td>MWM060b03_rÂ </td><td>56Â </td><td>113Â </td><td>3Eâˆ’05Â </td></tr><tr><td>Protein kinaseÂ </td><td>0.25Â </td><td>MWL036f06_rÂ </td><td>128Â </td><td>32Â </td><td>3Eâˆ’10Â </td></tr><tr><td>PAP-specific phosphataseÂ </td><td>0.26Â </td><td>MWM204g03_rÂ </td><td>104Â </td><td>28Â </td><td>7Eâˆ’25Â </td></tr><tr><td>PP2A regulatory subunitÂ </td><td>0.32Â </td><td>MWM123a10_rÂ </td><td>124Â </td><td>39Â </td><td>4Eâˆ’22Â </td></tr><tr><td>Hydrolase/ inositol or phosphatidylinositol phosphataseÂ </td><td>0.37Â </td><td>MWM231g02_rÂ </td><td>75Â </td><td>27Â </td><td>5Eâˆ’27Â </td></tr><tr><td><em>Transcription/translation</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Putative bZIP transcription factorÂ </td><td>3.10Â </td><td>MWM014e09_rÂ </td><td>65Â </td><td>202Â </td><td>1Eâˆ’30Â </td></tr><tr><td>Eukaryotic initiation factorÂ </td><td>2.71Â </td><td>MWM099b12_rÂ </td><td>380Â </td><td>1030Â </td><td>2Eâˆ’04Â </td></tr><tr><td>Poly(A)-binding proteinÂ </td><td>2.40Â </td><td>MWM214d01_rÂ </td><td>148Â </td><td>352Â </td><td>1Eâˆ’44Â </td></tr><tr><td>Putative aspartate-tRNA ligaseÂ </td><td>2.34Â </td><td>MWM096c11_rÂ </td><td>187Â </td><td>438Â </td><td>3Eâˆ’23Â </td></tr><tr><td>Homeobox domain proteinÂ </td><td>2.07Â </td><td>MPD034c07_fÂ </td><td>51Â </td><td>104Â </td><td>8Eâˆ’79Â </td></tr><tr><td>Heat shock transcription factorÂ </td><td>2.03Â </td><td>MR008f01_fÂ </td><td>58Â </td><td>117Â </td><td>8Eâˆ’39Â </td></tr><tr><td>Glycine-rich RNA-binding proteinÂ </td><td>2.01Â </td><td>GENLf028b01Â </td><td>189Â </td><td>379Â </td><td>3Eâˆ’36Â </td></tr><tr><td>Putative squamosa promoter-binding proteinÂ </td><td>0.25Â </td><td>GENLf063g01Â </td><td>132Â </td><td>33Â </td><td>7Eâˆ’07Â </td></tr><tr><td>SDL-1 plastid proteinÂ </td><td>0.29Â </td><td>GENLf045h01Â </td><td>112Â </td><td>32Â </td><td>9Eâˆ’92Â </td></tr><tr><td>Transcription factor MYB4 homologÂ </td><td>0.31Â </td><td>MPD092a11_fÂ </td><td>67Â </td><td>20Â </td><td>1Eâˆ’22Â </td></tr><tr><td>Transcription regulatory proteinÂ </td><td>0.32Â </td><td>MWM065b02_rÂ </td><td>116Â </td><td>37Â </td><td>Â </td></tr><tr><td>Putative DOF zinc finger proteinÂ </td><td>0.33Â </td><td>MWM178e06_rÂ </td><td>118Â </td><td>38Â </td><td>1Eâˆ’21Â </td></tr><tr><td>Putative translation initiation proteinÂ </td><td>0.33Â </td><td>GENLf057g11Â </td><td>106Â </td><td>32Â </td><td>Â </td></tr><tr><td><em>Cell wall</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Pectin acetylesteraseÂ </td><td>2.44Â </td><td>MWM096a11_rÂ </td><td>174Â </td><td>424Â </td><td>9Eâˆ’54Â </td></tr><tr><td>Putative pectinesteraseÂ </td><td>2.05Â </td><td>MWM097c10_rÂ </td><td>278Â </td><td>570Â </td><td>3Eâˆ’08Â </td></tr><tr><td>PectinesteraseÂ </td><td>2.02Â </td><td>MWM132g12_rÂ </td><td>85Â </td><td>172Â </td><td>1Eâˆ’55Â </td></tr><tr><td>Callose synthaseÂ </td><td>0.36Â </td><td>GENLf063h05Â </td><td>92Â </td><td>32Â </td><td>2Eâˆ’28Â </td></tr><tr><td><em>Protein fate</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Protein secretion pathway proteinÂ </td><td>2.08Â </td><td>GENLf046b01Â </td><td>124Â </td><td>257Â </td><td>Â </td></tr><tr><td>Dipeptidyl peptidase IV-like proteinÂ </td><td>0.27Â </td><td>MPDL020f10_fÂ </td><td>106Â </td><td>28Â </td><td>7Eâˆ’35Â </td></tr><tr><td>Oligopeptidase AÂ </td><td>0.29Â </td><td>MWM031e10_rÂ </td><td>131Â </td><td>37Â </td><td>3Eâˆ’50Â </td></tr><tr><td>26S proteasome ATPase subunitÂ </td><td>0.31Â </td><td>MWM223f06_rÂ </td><td>80Â </td><td>24Â </td><td>3Eâˆ’30Â </td></tr><tr><td>Putative ubiquitin carboxyl terminal hydrolaseÂ </td><td>0.33Â </td><td>MPDL041a06_fÂ </td><td>108Â </td><td>35Â </td><td>2Eâˆ’18Â </td></tr><tr><td>Pro-X carboxypeptidase-like proteinÂ </td><td>0.36Â </td><td>MPD016e03_fÂ </td><td>86Â </td><td>31Â </td><td>Â </td></tr><tr><td>Serine protease inhibitor phloem serpin-1Â </td><td>0.37Â </td><td>GENf065b05Â </td><td>77Â </td><td>30Â </td><td>3Eâˆ’26Â </td></tr><tr><td><em>Pathogen-related</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Respiratory burst oxidase protein DÂ </td><td>2.57Â </td><td>GENLf020h11Â </td><td>54Â </td><td>138Â </td><td>3Eâˆ’41Â </td></tr><tr><td>Syringolide-induced proteinÂ </td><td>2.43Â </td><td>MWM033e05_rÂ </td><td>123Â </td><td>296Â </td><td>8Eâˆ’14Â </td></tr><tr><td>Peroxidase 3 precursorÂ </td><td>2.12Â </td><td>MWM241c09_rÂ </td><td>42Â </td><td>88Â </td><td>1Eâˆ’13Â </td></tr><tr><td>Syringolide-induced protein 14-1-1Â </td><td>2.06Â </td><td>MWM031c04_rÂ </td><td>95Â </td><td>195Â </td><td>8Eâˆ’27Â </td></tr><tr><td>Disease resistance-related proteinÂ </td><td>2.04Â </td><td>MWM067e07_rÂ </td><td>74Â </td><td>153Â </td><td>6Eâˆ’23Â </td></tr><tr><td>Class III peroxidase PSYP1Â </td><td>2.03Â </td><td>MWL018a05_rÂ </td><td>82Â </td><td>166Â </td><td>2Eâˆ’37Â </td></tr><tr><td>Endo-1,4-beta-glucanaseÂ </td><td>0.14Â </td><td>MWL011b05_rÂ </td><td>244Â </td><td>34Â </td><td>9Eâˆ’30Â </td></tr><tr><td>Syringolide-induced proteinÂ </td><td>0.35Â </td><td>MWM037b07_rÂ </td><td>139Â </td><td>48Â </td><td>2Eâˆ’07Â </td></tr><tr><td>Syringolide-induced proteinÂ </td><td>0.41Â </td><td>MWM014d11_rÂ </td><td>102Â </td><td>41Â </td><td>8Eâˆ’64Â </td></tr><tr><td><em>Phytohormone-related</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Auxin-repressed proteinÂ </td><td>2.13Â </td><td>MPDL064h08_fÂ </td><td>270Â </td><td>574Â </td><td>1Eâˆ’14Â </td></tr><tr><td>Cytokinin oxidaseÂ </td><td>0.24Â </td><td>MWM042d03_rÂ </td><td>129Â </td><td>28Â </td><td>3Eâˆ’35Â </td></tr><tr><td><em>Other enzymes</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Cytochrome P450Â </td><td>2.52Â </td><td>MWM049d04_rÂ </td><td>234Â </td><td>589Â </td><td>8Eâˆ’55Â </td></tr><tr><td>Cytochrome P450Â </td><td>2.51Â </td><td>MR061f02_fÂ </td><td>167Â </td><td>418Â </td><td>7Eâˆ’35Â </td></tr><tr><td>Phosphogluconate dehydrogenaseÂ </td><td>2.47Â </td><td>MWM228b11_rÂ </td><td>193Â </td><td>475Â </td><td>1Eâˆ’49Â </td></tr><tr><td>Fatty acid hydroxylase cytochrome P450Â </td><td>2.28Â </td><td>MWM051a05_rÂ </td><td>88Â </td><td>201Â </td><td>3Eâˆ’26Â </td></tr><tr><td>Cytochrome P450Â </td><td>2.21Â </td><td>MR043g06_fÂ </td><td>465Â </td><td>1027Â </td><td>2Eâˆ’09Â </td></tr><tr><td>Cytochrome P450Â </td><td>2.20Â </td><td>MWM152a11_rÂ </td><td>43Â </td><td>93Â </td><td>4Eâˆ’29Â </td></tr><tr><td>Epoxide hydrolaseÂ </td><td>2.11Â </td><td>MWM079e11_rÂ </td><td>56Â </td><td>119Â </td><td>4Eâˆ’54Â </td></tr><tr><td>Thiazole biosynthetic enzymeÂ </td><td>2.08Â </td><td>MWM107g04_rÂ </td><td>150Â </td><td>311Â </td><td>7Eâˆ’33Â </td></tr><tr><td>Putative helicaseÂ </td><td>2.03Â </td><td>MWL079f07_rÂ </td><td>157Â </td><td>317Â </td><td>7Eâˆ’11Â </td></tr><tr><td>Phosphatidylserine decarboxylaseÂ </td><td>2.02Â </td><td>MWM214c03_rÂ </td><td>65Â </td><td>131Â </td><td>1Eâˆ’57Â </td></tr><tr><td>Isopentenyl-diphosphate isomerase IIÂ </td><td>0.21Â </td><td>MWM082f11_rÂ </td><td>123Â </td><td>25Â </td><td>2Eâˆ’26Â </td></tr><tr><td>Retroelement pol polyprotein-likeÂ </td><td>0.24Â </td><td>MWL062c10_rÂ </td><td>129Â </td><td>30Â </td><td>2Eâˆ’26Â </td></tr><tr><td>Histone acetyltransferase HAT BÂ </td><td>0.24Â </td><td>MWM193c03_rÂ </td><td>89Â </td><td>20Â </td><td>4Eâˆ’13Â </td></tr><tr><td>Thiamine biosynthetic enzymeÂ </td><td>0.25Â </td><td>GENf012a12Â </td><td>102Â </td><td>25Â </td><td>3Eâˆ’57Â </td></tr><tr><td>Cytochrome P450Â </td><td>0.32Â </td><td>MWM170d07_rÂ </td><td>158Â </td><td>50Â </td><td>2Eâˆ’63Â </td></tr><tr><td>Obtusifoliol 14-alpha demethylaseÂ </td><td>0.33Â </td><td>GENf014g11Â </td><td>137Â </td><td>45Â </td><td>5Eâˆ’54Â </td></tr><tr><td>Magnesium chelataseÂ </td><td>0.34Â </td><td>MWL046f07_rÂ </td><td>81Â </td><td>27Â </td><td>5Eâˆ’56Â </td></tr><tr><td>UMP synthaseÂ </td><td>0.35Â </td><td>MWM187d07_rÂ </td><td>120Â </td><td>41Â </td><td>3Eâˆ’53Â </td></tr><tr><td>Putative cytochrome P450Â </td><td>0.35Â </td><td>MWM139c03_rÂ </td><td>95Â </td><td>33Â </td><td>3Eâˆ’59Â </td></tr><tr><td>Dihydroneopterin aldolaseÂ </td><td>0.36Â </td><td>GENLf038a07Â </td><td>74Â </td><td>26Â </td><td>2Eâˆ’24Â </td></tr><tr><td><em>Other categories</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Polyubiquitin 4Â </td><td>2.60Â </td><td>MWM214g11_rÂ </td><td>119Â </td><td>308Â </td><td>3Eâˆ’09Â </td></tr><tr><td>Metallothionein-like protein class IIÂ </td><td>2.54Â </td><td>MWM200f03_rÂ </td><td>1109Â </td><td>2821Â </td><td>3Eâˆ’19Â </td></tr><tr><td>CPRD49Â </td><td>2.42Â </td><td>MWM128g09_rÂ </td><td>148Â </td><td>357Â </td><td>1Eâˆ’17Â </td></tr><tr><td>Ubiquitin precursorÂ </td><td>2.28Â </td><td>MWM011f03_rÂ </td><td>544Â </td><td>1242Â </td><td>2Eâˆ’10Â </td></tr><tr><td>DnaJ-like proteinÂ </td><td>2.04Â </td><td>MWM184b12_rÂ </td><td>141Â </td><td>288Â </td><td>8Eâˆ’71Â </td></tr><tr><td>Bax inhibitor-1 likeÂ </td><td>2.04Â </td><td>MWM016c06_rÂ </td><td>143Â </td><td>293Â </td><td>4Eâˆ’50Â </td></tr><tr><td>Heat shock protein 70 cognateÂ </td><td>2.02Â </td><td>MWM159a01_rÂ </td><td>174Â </td><td>351Â </td><td>2Eâˆ’62Â </td></tr><tr><td>Putative 2Fe-2S ironâ€“sulfur cluster proteinÂ </td><td>2.01Â </td><td>MPD065e04_fÂ </td><td>51Â </td><td>102Â </td><td>3Eâˆ’07Â </td></tr><tr><td>Ankyrin-repeat proteinÂ </td><td>2.00Â </td><td>MWM067b10_rÂ </td><td>428Â </td><td>856Â </td><td>8Eâˆ’17Â </td></tr><tr><td>Vacuolar sorting receptor protein BP-80Â </td><td>2.00Â </td><td>MWL009b09_rÂ </td><td>173Â </td><td>345Â </td><td>9Eâˆ’20Â </td></tr><tr><td>Calcineurin B-like proteinÂ </td><td>0.25Â </td><td>MWM143g03_rÂ </td><td>82Â </td><td>20Â </td><td>0.007Â </td></tr><tr><td>Peroxiredoxin QÂ </td><td>0.31Â </td><td>MWM126d02_rÂ </td><td>97Â </td><td>29Â </td><td>1Eâˆ’16Â </td></tr><tr><td>Senescence-associated putative proteinÂ </td><td>0.33Â </td><td>MWL051e09_rÂ </td><td>110Â </td><td>35Â </td><td>1Eâˆ’16Â </td></tr><tr><td>PSII low MW proteinÂ </td><td>0.34Â </td><td>MWL078e10_rÂ </td><td>578Â </td><td>198Â </td><td>4Eâˆ’19Â </td></tr><tr><td>Histone H2AÂ </td><td>0.35Â </td><td>MWM209e08_rÂ </td><td>133Â </td><td>46Â </td><td>9Eâˆ’51Â </td></tr><tr><td>ActinÂ </td><td>0.36Â </td><td>GENf007a10Â </td><td>82Â </td><td>29Â </td><td>1Eâˆ’48Â </td></tr><tr><td>Phosphatidylinositol transfer-like protein IVÂ </td><td>0.36Â </td><td>GENf020a06Â </td><td>125Â </td><td>44Â </td><td>6Eâˆ’20Â </td></tr><tr><td>Chlorophyll a/b-binding protein type IIÂ </td><td>0.36Â </td><td>MPD059g06_fÂ </td><td>95Â </td><td>34Â </td><td>7Eâˆ’46Â </td></tr><tr><td>Early light-inducible proteinÂ </td><td>0.37Â </td><td>MWL040f08_rÂ </td><td>85Â </td><td>31Â </td><td>2Eâˆ’43Â </td></tr><tr><td>Cytochrome b/fÂ </td><td>0.37Â </td><td>MWM225h10_rÂ </td><td>325Â </td><td>120Â </td><td>4Eâˆ’71Â </td></tr></tbody></table></div><div content-id="DSM014TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Current annotationÂ </th><th>Fold (Gi1/gni1)Â </th><th>Gene IDÂ </th><th>gni1Â </th><th>Gi1Â </th><th>e-valueÂ </th></tr></thead><tbody><tr><td><em>Amino acid and nitrogen metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Selenocysteine methyltransferaseÂ </td><td>2.24Â </td><td>MWM066h10_rÂ </td><td>62Â </td><td>139Â </td><td>6Eâˆ’25Â </td></tr><tr><td>Diaminopimelate decarboxylaseÂ </td><td>2.05Â </td><td>MWM149b05_rÂ </td><td>128Â </td><td>262Â </td><td>1Eâˆ’66Â </td></tr><tr><td><em>S</em> -adenosyl methionine synthetase Â </td><td>2.02Â </td><td>MWM180f07_rÂ </td><td>145Â </td><td>292Â </td><td>4Eâˆ’67Â </td></tr><tr><td>VuP5CRÂ </td><td>0.28Â </td><td>GENLf018g02Â </td><td>78Â </td><td>21Â </td><td>4Eâˆ’24Â </td></tr><tr><td>Arginine decarboxylaseÂ </td><td>0.29Â </td><td>MWM198e12_rÂ </td><td>102Â </td><td>29Â </td><td>2Eâˆ’49Â </td></tr><tr><td>Aminotransferase 2Â </td><td>0.33Â </td><td>MWM222b09_rÂ </td><td>101Â </td><td>33Â </td><td>5Eâˆ’66Â </td></tr><tr><td>Proline dehydrogenaseÂ </td><td>0.36Â </td><td>MWM135h10_rÂ </td><td>99Â </td><td>35Â </td><td>8Eâˆ’29Â </td></tr><tr><td>Delta-1-pyrroline-5-carboxylate synthaseÂ </td><td>0.36Â </td><td>GENLf045e06Â </td><td>121Â </td><td>43Â </td><td>4Eâˆ’09Â </td></tr><tr><td><em>Carbon metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>InvertaseÂ </td><td>2.32Â </td><td>MWM224d02_rÂ </td><td>150Â </td><td>347Â </td><td>6Eâˆ’08Â </td></tr><tr><td>Sucrose synthaseÂ </td><td>2.27Â </td><td>MWL068h11_rÂ </td><td>271Â </td><td>614Â </td><td>1Eâˆ’58Â </td></tr><tr><td>Glucose-1-phosphate adenylyltransferaseÂ </td><td>2.13Â </td><td>MWM086h02_rÂ </td><td>128Â </td><td>271Â </td><td>7Eâˆ’65Â </td></tr><tr><td>Alpha-mannosidaseÂ </td><td>2.11Â </td><td>GENLf064h06Â </td><td>138Â </td><td>291Â </td><td>4Eâˆ’24Â </td></tr><tr><td>Glucose-6-phosphate dehydrogenaseÂ </td><td>2.07Â </td><td>MR098a03_fÂ </td><td>47Â </td><td>93Â </td><td>7Eâˆ’06Â </td></tr><tr><td>Beta-amylaseÂ </td><td>2.06Â </td><td>GENf097b02Â </td><td>111Â </td><td>229Â </td><td>1Eâˆ’132Â </td></tr><tr><td>Triosephosphate isomeraseÂ </td><td>2.05Â </td><td>MWM193g10_rÂ </td><td>272Â </td><td>559Â </td><td>1Eâˆ’77Â </td></tr><tr><td>Glucosyltransferase-like proteinÂ </td><td>2.01Â </td><td>MWL049f07_rÂ </td><td>126Â </td><td>257Â </td><td>9Eâˆ’06Â </td></tr><tr><td> Beta- <span class="small-caps">d</span> -xylosidase Â </td><td>0.26Â </td><td>MWM219c11_rÂ </td><td>119Â </td><td>31Â </td><td>9Eâˆ’55Â </td></tr><tr><td>Fructose-bisphosphate aldolaseÂ </td><td>0.29Â </td><td>MWM024h09_rÂ </td><td>95Â </td><td>28Â </td><td>2Eâˆ’39Â </td></tr><tr><td>Malonyl-CoA: acyl carrier protein transacylaseÂ </td><td>0.30Â </td><td>MWL014e01_rÂ </td><td>115Â </td><td>34Â </td><td>4Eâˆ’33Â </td></tr><tr><td>Putative 2-isopropylmalate synthaseÂ </td><td>0.31Â </td><td>GENf086f07Â </td><td>76Â </td><td>23Â </td><td>4Eâˆ’34Â </td></tr><tr><td>Mannosyltransferase-like proteinÂ </td><td>0.32Â </td><td>MWM235d07_rÂ </td><td>92Â </td><td>28Â </td><td>3Eâˆ’04Â </td></tr><tr><td>Citrate synthaseÂ </td><td>0.37Â </td><td>MWM239b12_rÂ </td><td>147Â </td><td>54Â </td><td>4Eâˆ’38Â </td></tr><tr><td><em>Secondary metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Chalcone reductaseÂ </td><td>3.03Â </td><td>MWM002d07_rÂ </td><td>98Â </td><td>306Â </td><td>1Eâˆ’60Â </td></tr><tr><td>Phenylalanine ammonia-lyase (LjPAL2)Â </td><td>2.16Â </td><td>GENLf058e04Â </td><td>95Â </td><td>202Â </td><td>0.006Â </td></tr><tr><td>Phenylalanine ammonia-lyase (LjPAL1)Â </td><td>2.09Â </td><td>MRL007g11_fÂ </td><td>136Â </td><td>283Â </td><td>9Eâˆ’38Â </td></tr><tr><td>4-coumarate:CoA ligaseÂ </td><td>2.10Â </td><td>MWL020d04_rÂ </td><td>110Â </td><td>235Â </td><td>3Eâˆ’34Â </td></tr><tr><td>Chalcone synthaseÂ </td><td>2.05Â </td><td>MWL020g05_rÂ </td><td>144Â </td><td>295Â </td><td>1Eâˆ’14Â </td></tr><tr><td>LaccaseÂ </td><td>0.30Â </td><td>MWM219c08_rÂ </td><td>124Â </td><td>36Â </td><td>3Eâˆ’44Â </td></tr><tr><td>Putative diphenol oxidaseÂ </td><td>0.34Â </td><td>GENf055e07Â </td><td>87Â </td><td>29Â </td><td>1Eâˆ’13Â </td></tr><tr><td><em>Transport/membrane</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td> Plasma membrane Ca2 <sup>+</sup> -ATPase Â </td><td>2.34Â </td><td>MWM178b05_rÂ </td><td>169Â </td><td>396Â </td><td>3Eâˆ’74Â </td></tr><tr><td>Aquaporin protein PIP1Â </td><td>2.34Â </td><td>MWL033d08_rÂ </td><td>279Â </td><td>652Â </td><td>8Eâˆ’51Â </td></tr><tr><td>Putative amino acid transporterÂ </td><td>2.14Â </td><td>MWL064b03_rÂ </td><td>71Â </td><td>150Â </td><td>8Eâˆ’51Â </td></tr><tr><td>Putative ABC transporter proteinÂ </td><td>2.13Â </td><td>MWL077b08_rÂ </td><td>158Â </td><td>335Â </td><td>1Eâˆ’14Â </td></tr><tr><td>Putative nuclear transport factorÂ </td><td>2.09Â </td><td>MWM105a05_rÂ </td><td>118Â </td><td>248Â </td><td>1Eâˆ’43Â </td></tr><tr><td>Vacuolar ATPaseÂ </td><td>2.06Â </td><td>MWM238d10_rÂ </td><td>84Â </td><td>173Â </td><td>4Eâˆ’56Â </td></tr><tr><td>Nuclear transport factorÂ </td><td>2.02Â </td><td>GNf048c09Â </td><td>84Â </td><td>165Â </td><td>1Eâˆ’38Â </td></tr><tr><td>Sorbitol transporterÂ </td><td>2.01Â </td><td>MWM244a03_rÂ </td><td>41Â </td><td>79Â </td><td>4Eâˆ’57Â </td></tr><tr><td>MATE efflux family proteinÂ </td><td>0.23Â </td><td>GENLf013e12Â </td><td>111Â </td><td>25Â </td><td>2Eâˆ’06Â </td></tr><tr><td>Cation-transporting ATPaseÂ </td><td>0.31Â </td><td>MWM087d06_rÂ </td><td>154Â </td><td>46Â </td><td>3Eâˆ’20Â </td></tr><tr><td>Plastidic phosphate translocator-like proteinÂ </td><td>0.32Â </td><td>GENf029g12Â </td><td>108Â </td><td>34Â </td><td>6Eâˆ’96Â </td></tr><tr><td><em>Nodulin</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Early nodulin ENOD18Â </td><td>2.65Â </td><td>GENf079d10Â </td><td>34Â </td><td>90Â </td><td>2Eâˆ’14Â </td></tr><tr><td><em>Signal transduction</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Protein phosphatase 2CÂ </td><td>2.40Â </td><td>MWM035c07_rÂ </td><td>248Â </td><td>596Â </td><td>1Eâˆ’52Â </td></tr><tr><td>Protein serine/threonine kinaseÂ </td><td>2.35Â </td><td>MWM206a09_rÂ </td><td>74Â </td><td>174Â </td><td>4Eâˆ’29Â </td></tr><tr><td>G protein-coupled receptorÂ </td><td>2.33Â </td><td>GENLf064g09Â </td><td>70Â </td><td>163Â </td><td>Â </td></tr><tr><td>Receptor protein kinaseÂ </td><td>2.31Â </td><td>MPDL044b10_fÂ </td><td>56Â </td><td>130Â </td><td>7Eâˆ’18Â </td></tr><tr><td>Small GTP-binding proteinÂ </td><td>2.18Â </td><td>MWM122f01_rÂ </td><td>174Â </td><td>380Â </td><td>5Eâˆ’44Â </td></tr><tr><td>GUN4 regulatorÂ </td><td>2.12Â </td><td>MPDL091h07_fÂ </td><td>149Â </td><td>316Â </td><td>1Eâˆ’63Â </td></tr><tr><td>Putative acid phosphataseÂ </td><td>2.09Â </td><td>MWM048e06_rÂ </td><td>415Â </td><td>866Â </td><td>8Eâˆ’24Â </td></tr><tr><td>MAP kinase 3Â </td><td>2.09Â </td><td>MR062e03_fÂ </td><td>582Â </td><td>1215Â </td><td>3Eâˆ’84Â </td></tr><tr><td>Putative GTP-binding proteinÂ </td><td>2.05Â </td><td>MWM119e05_rÂ </td><td>161Â </td><td>329Â </td><td>4Eâˆ’34Â </td></tr><tr><td>Protein phosphatase-2CÂ </td><td>2.05Â </td><td>MWM050f11_rÂ </td><td>89Â </td><td>182Â </td><td>0.0003Â </td></tr><tr><td>Serine/threonine protein phosphataseÂ </td><td>2.05Â </td><td>MR028a01_fÂ </td><td>85Â </td><td>173Â </td><td>2Eâˆ’27Â </td></tr><tr><td>Calcium-dependent protein kinaseÂ </td><td>2.01Â </td><td>MWM060b03_rÂ </td><td>56Â </td><td>113Â </td><td>3Eâˆ’05Â </td></tr><tr><td>Protein kinaseÂ </td><td>0.25Â </td><td>MWL036f06_rÂ </td><td>128Â </td><td>32Â </td><td>3Eâˆ’10Â </td></tr><tr><td>PAP-specific phosphataseÂ </td><td>0.26Â </td><td>MWM204g03_rÂ </td><td>104Â </td><td>28Â </td><td>7Eâˆ’25Â </td></tr><tr><td>PP2A regulatory subunitÂ </td><td>0.32Â </td><td>MWM123a10_rÂ </td><td>124Â </td><td>39Â </td><td>4Eâˆ’22Â </td></tr><tr><td>Hydrolase/ inositol or phosphatidylinositol phosphataseÂ </td><td>0.37Â </td><td>MWM231g02_rÂ </td><td>75Â </td><td>27Â </td><td>5Eâˆ’27Â </td></tr><tr><td><em>Transcription/translation</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Putative bZIP transcription factorÂ </td><td>3.10Â </td><td>MWM014e09_rÂ </td><td>65Â </td><td>202Â </td><td>1Eâˆ’30Â </td></tr><tr><td>Eukaryotic initiation factorÂ </td><td>2.71Â </td><td>MWM099b12_rÂ </td><td>380Â </td><td>1030Â </td><td>2Eâˆ’04Â </td></tr><tr><td>Poly(A)-binding proteinÂ </td><td>2.40Â </td><td>MWM214d01_rÂ </td><td>148Â </td><td>352Â </td><td>1Eâˆ’44Â </td></tr><tr><td>Putative aspartate-tRNA ligaseÂ </td><td>2.34Â </td><td>MWM096c11_rÂ </td><td>187Â </td><td>438Â </td><td>3Eâˆ’23Â </td></tr><tr><td>Homeobox domain proteinÂ </td><td>2.07Â </td><td>MPD034c07_fÂ </td><td>51Â </td><td>104Â </td><td>8Eâˆ’79Â </td></tr><tr><td>Heat shock transcription factorÂ </td><td>2.03Â </td><td>MR008f01_fÂ </td><td>58Â </td><td>117Â </td><td>8Eâˆ’39Â </td></tr><tr><td>Glycine-rich RNA-binding proteinÂ </td><td>2.01Â </td><td>GENLf028b01Â </td><td>189Â </td><td>379Â </td><td>3Eâˆ’36Â </td></tr><tr><td>Putative squamosa promoter-binding proteinÂ </td><td>0.25Â </td><td>GENLf063g01Â </td><td>132Â </td><td>33Â </td><td>7Eâˆ’07Â </td></tr><tr><td>SDL-1 plastid proteinÂ </td><td>0.29Â </td><td>GENLf045h01Â </td><td>112Â </td><td>32Â </td><td>9Eâˆ’92Â </td></tr><tr><td>Transcription factor MYB4 homologÂ </td><td>0.31Â </td><td>MPD092a11_fÂ </td><td>67Â </td><td>20Â </td><td>1Eâˆ’22Â </td></tr><tr><td>Transcription regulatory proteinÂ </td><td>0.32Â </td><td>MWM065b02_rÂ </td><td>116Â </td><td>37Â </td><td>Â </td></tr><tr><td>Putative DOF zinc finger proteinÂ </td><td>0.33Â </td><td>MWM178e06_rÂ </td><td>118Â </td><td>38Â </td><td>1Eâˆ’21Â </td></tr><tr><td>Putative translation initiation proteinÂ </td><td>0.33Â </td><td>GENLf057g11Â </td><td>106Â </td><td>32Â </td><td>Â </td></tr><tr><td><em>Cell wall</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Pectin acetylesteraseÂ </td><td>2.44Â </td><td>MWM096a11_rÂ </td><td>174Â </td><td>424Â </td><td>9Eâˆ’54Â </td></tr><tr><td>Putative pectinesteraseÂ </td><td>2.05Â </td><td>MWM097c10_rÂ </td><td>278Â </td><td>570Â </td><td>3Eâˆ’08Â </td></tr><tr><td>PectinesteraseÂ </td><td>2.02Â </td><td>MWM132g12_rÂ </td><td>85Â </td><td>172Â </td><td>1Eâˆ’55Â </td></tr><tr><td>Callose synthaseÂ </td><td>0.36Â </td><td>GENLf063h05Â </td><td>92Â </td><td>32Â </td><td>2Eâˆ’28Â </td></tr><tr><td><em>Protein fate</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Protein secretion pathway proteinÂ </td><td>2.08Â </td><td>GENLf046b01Â </td><td>124Â </td><td>257Â </td><td>Â </td></tr><tr><td>Dipeptidyl peptidase IV-like proteinÂ </td><td>0.27Â </td><td>MPDL020f10_fÂ </td><td>106Â </td><td>28Â </td><td>7Eâˆ’35Â </td></tr><tr><td>Oligopeptidase AÂ </td><td>0.29Â </td><td>MWM031e10_rÂ </td><td>131Â </td><td>37Â </td><td>3Eâˆ’50Â </td></tr><tr><td>26S proteasome ATPase subunitÂ </td><td>0.31Â </td><td>MWM223f06_rÂ </td><td>80Â </td><td>24Â </td><td>3Eâˆ’30Â </td></tr><tr><td>Putative ubiquitin carboxyl terminal hydrolaseÂ </td><td>0.33Â </td><td>MPDL041a06_fÂ </td><td>108Â </td><td>35Â </td><td>2Eâˆ’18Â </td></tr><tr><td>Pro-X carboxypeptidase-like proteinÂ </td><td>0.36Â </td><td>MPD016e03_fÂ </td><td>86Â </td><td>31Â </td><td>Â </td></tr><tr><td>Serine protease inhibitor phloem serpin-1Â </td><td>0.37Â </td><td>GENf065b05Â </td><td>77Â </td><td>30Â </td><td>3Eâˆ’26Â </td></tr><tr><td><em>Pathogen-related</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Respiratory burst oxidase protein DÂ </td><td>2.57Â </td><td>GENLf020h11Â </td><td>54Â </td><td>138Â </td><td>3Eâˆ’41Â </td></tr><tr><td>Syringolide-induced proteinÂ </td><td>2.43Â </td><td>MWM033e05_rÂ </td><td>123Â </td><td>296Â </td><td>8Eâˆ’14Â </td></tr><tr><td>Peroxidase 3 precursorÂ </td><td>2.12Â </td><td>MWM241c09_rÂ </td><td>42Â </td><td>88Â </td><td>1Eâˆ’13Â </td></tr><tr><td>Syringolide-induced protein 14-1-1Â </td><td>2.06Â </td><td>MWM031c04_rÂ </td><td>95Â </td><td>195Â </td><td>8Eâˆ’27Â </td></tr><tr><td>Disease resistance-related proteinÂ </td><td>2.04Â </td><td>MWM067e07_rÂ </td><td>74Â </td><td>153Â </td><td>6Eâˆ’23Â </td></tr><tr><td>Class III peroxidase PSYP1Â </td><td>2.03Â </td><td>MWL018a05_rÂ </td><td>82Â </td><td>166Â </td><td>2Eâˆ’37Â </td></tr><tr><td>Endo-1,4-beta-glucanaseÂ </td><td>0.14Â </td><td>MWL011b05_rÂ </td><td>244Â </td><td>34Â </td><td>9Eâˆ’30Â </td></tr><tr><td>Syringolide-induced proteinÂ </td><td>0.35Â </td><td>MWM037b07_rÂ </td><td>139Â </td><td>48Â </td><td>2Eâˆ’07Â </td></tr><tr><td>Syringolide-induced proteinÂ </td><td>0.41Â </td><td>MWM014d11_rÂ </td><td>102Â </td><td>41Â </td><td>8Eâˆ’64Â </td></tr><tr><td><em>Phytohormone-related</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Auxin-repressed proteinÂ </td><td>2.13Â </td><td>MPDL064h08_fÂ </td><td>270Â </td><td>574Â </td><td>1Eâˆ’14Â </td></tr><tr><td>Cytokinin oxidaseÂ </td><td>0.24Â </td><td>MWM042d03_rÂ </td><td>129Â </td><td>28Â </td><td>3Eâˆ’35Â </td></tr><tr><td><em>Other enzymes</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Cytochrome P450Â </td><td>2.52Â </td><td>MWM049d04_rÂ </td><td>234Â </td><td>589Â </td><td>8Eâˆ’55Â </td></tr><tr><td>Cytochrome P450Â </td><td>2.51Â </td><td>MR061f02_fÂ </td><td>167Â </td><td>418Â </td><td>7Eâˆ’35Â </td></tr><tr><td>Phosphogluconate dehydrogenaseÂ </td><td>2.47Â </td><td>MWM228b11_rÂ </td><td>193Â </td><td>475Â </td><td>1Eâˆ’49Â </td></tr><tr><td>Fatty acid hydroxylase cytochrome P450Â </td><td>2.28Â </td><td>MWM051a05_rÂ </td><td>88Â </td><td>201Â </td><td>3Eâˆ’26Â </td></tr><tr><td>Cytochrome P450Â </td><td>2.21Â </td><td>MR043g06_fÂ </td><td>465Â </td><td>1027Â </td><td>2Eâˆ’09Â </td></tr><tr><td>Cytochrome P450Â </td><td>2.20Â </td><td>MWM152a11_rÂ </td><td>43Â </td><td>93Â </td><td>4Eâˆ’29Â </td></tr><tr><td>Epoxide hydrolaseÂ </td><td>2.11Â </td><td>MWM079e11_rÂ </td><td>56Â </td><td>119Â </td><td>4Eâˆ’54Â </td></tr><tr><td>Thiazole biosynthetic enzymeÂ </td><td>2.08Â </td><td>MWM107g04_rÂ </td><td>150Â </td><td>311Â </td><td>7Eâˆ’33Â </td></tr><tr><td>Putative helicaseÂ </td><td>2.03Â </td><td>MWL079f07_rÂ </td><td>157Â </td><td>317Â </td><td>7Eâˆ’11Â </td></tr><tr><td>Phosphatidylserine decarboxylaseÂ </td><td>2.02Â </td><td>MWM214c03_rÂ </td><td>65Â </td><td>131Â </td><td>1Eâˆ’57Â </td></tr><tr><td>Isopentenyl-diphosphate isomerase IIÂ </td><td>0.21Â </td><td>MWM082f11_rÂ </td><td>123Â </td><td>25Â </td><td>2Eâˆ’26Â </td></tr><tr><td>Retroelement pol polyprotein-likeÂ </td><td>0.24Â </td><td>MWL062c10_rÂ </td><td>129Â </td><td>30Â </td><td>2Eâˆ’26Â </td></tr><tr><td>Histone acetyltransferase HAT BÂ </td><td>0.24Â </td><td>MWM193c03_rÂ </td><td>89Â </td><td>20Â </td><td>4Eâˆ’13Â </td></tr><tr><td>Thiamine biosynthetic enzymeÂ </td><td>0.25Â </td><td>GENf012a12Â </td><td>102Â </td><td>25Â </td><td>3Eâˆ’57Â </td></tr><tr><td>Cytochrome P450Â </td><td>0.32Â </td><td>MWM170d07_rÂ </td><td>158Â </td><td>50Â </td><td>2Eâˆ’63Â </td></tr><tr><td>Obtusifoliol 14-alpha demethylaseÂ </td><td>0.33Â </td><td>GENf014g11Â </td><td>137Â </td><td>45Â </td><td>5Eâˆ’54Â </td></tr><tr><td>Magnesium chelataseÂ </td><td>0.34Â </td><td>MWL046f07_rÂ </td><td>81Â </td><td>27Â </td><td>5Eâˆ’56Â </td></tr><tr><td>UMP synthaseÂ </td><td>0.35Â </td><td>MWM187d07_rÂ </td><td>120Â </td><td>41Â </td><td>3Eâˆ’53Â </td></tr><tr><td>Putative cytochrome P450Â </td><td>0.35Â </td><td>MWM139c03_rÂ </td><td>95Â </td><td>33Â </td><td>3Eâˆ’59Â </td></tr><tr><td>Dihydroneopterin aldolaseÂ </td><td>0.36Â </td><td>GENLf038a07Â </td><td>74Â </td><td>26Â </td><td>2Eâˆ’24Â </td></tr><tr><td><em>Other categories</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Polyubiquitin 4Â </td><td>2.60Â </td><td>MWM214g11_rÂ </td><td>119Â </td><td>308Â </td><td>3Eâˆ’09Â </td></tr><tr><td>Metallothionein-like protein class IIÂ </td><td>2.54Â </td><td>MWM200f03_rÂ </td><td>1109Â </td><td>2821Â </td><td>3Eâˆ’19Â </td></tr><tr><td>CPRD49Â </td><td>2.42Â </td><td>MWM128g09_rÂ </td><td>148Â </td><td>357Â </td><td>1Eâˆ’17Â </td></tr><tr><td>Ubiquitin precursorÂ </td><td>2.28Â </td><td>MWM011f03_rÂ </td><td>544Â </td><td>1242Â </td><td>2Eâˆ’10Â </td></tr><tr><td>DnaJ-like proteinÂ </td><td>2.04Â </td><td>MWM184b12_rÂ </td><td>141Â </td><td>288Â </td><td>8Eâˆ’71Â </td></tr><tr><td>Bax inhibitor-1 likeÂ </td><td>2.04Â </td><td>MWM016c06_rÂ </td><td>143Â </td><td>293Â </td><td>4Eâˆ’50Â </td></tr><tr><td>Heat shock protein 70 cognateÂ </td><td>2.02Â </td><td>MWM159a01_rÂ </td><td>174Â </td><td>351Â </td><td>2Eâˆ’62Â </td></tr><tr><td>Putative 2Fe-2S ironâ€“sulfur cluster proteinÂ </td><td>2.01Â </td><td>MPD065e04_fÂ </td><td>51Â </td><td>102Â </td><td>3Eâˆ’07Â </td></tr><tr><td>Ankyrin-repeat proteinÂ </td><td>2.00Â </td><td>MWM067b10_rÂ </td><td>428Â </td><td>856Â </td><td>8Eâˆ’17Â </td></tr><tr><td>Vacuolar sorting receptor protein BP-80Â </td><td>2.00Â </td><td>MWL009b09_rÂ </td><td>173Â </td><td>345Â </td><td>9Eâˆ’20Â </td></tr><tr><td>Calcineurin B-like proteinÂ </td><td>0.25Â </td><td>MWM143g03_rÂ </td><td>82Â </td><td>20Â </td><td>0.007Â </td></tr><tr><td>Peroxiredoxin QÂ </td><td>0.31Â </td><td>MWM126d02_rÂ </td><td>97Â </td><td>29Â </td><td>1Eâˆ’16Â </td></tr><tr><td>Senescence-associated putative proteinÂ </td><td>0.33Â </td><td>MWL051e09_rÂ </td><td>110Â </td><td>35Â </td><td>1Eâˆ’16Â </td></tr><tr><td>PSII low MW proteinÂ </td><td>0.34Â </td><td>MWL078e10_rÂ </td><td>578Â </td><td>198Â </td><td>4Eâˆ’19Â </td></tr><tr><td>Histone H2AÂ </td><td>0.35Â </td><td>MWM209e08_rÂ </td><td>133Â </td><td>46Â </td><td>9Eâˆ’51Â </td></tr><tr><td>ActinÂ </td><td>0.36Â </td><td>GENf007a10Â </td><td>82Â </td><td>29Â </td><td>1Eâˆ’48Â </td></tr><tr><td>Phosphatidylinositol transfer-like protein IVÂ </td><td>0.36Â </td><td>GENf020a06Â </td><td>125Â </td><td>44Â </td><td>6Eâˆ’20Â </td></tr><tr><td>Chlorophyll a/b-binding protein type IIÂ </td><td>0.36Â </td><td>MPD059g06_fÂ </td><td>95Â </td><td>34Â </td><td>7Eâˆ’46Â </td></tr><tr><td>Early light-inducible proteinÂ </td><td>0.37Â </td><td>MWL040f08_rÂ </td><td>85Â </td><td>31Â </td><td>2Eâˆ’43Â </td></tr><tr><td>Cytochrome b/fÂ </td><td>0.37Â </td><td>MWM225h10_rÂ </td><td>325Â </td><td>120Â </td><td>4Eâˆ’71Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p> Because there is a varying lag time between sporulation and the arrival of AM hyphae on the host roots, the initial response to mycorrhizae is not necessarily synchronous. Therefore, we modified the â€˜nurse potâ€™ method <sup><a reveal-id="DSM014C30" data-open="DSM014C30" class="link link-ref link-reveal xref-bibr">30</a></sup> as described under Materials and Methods. Freshly prepared sterile seedlings were transplanted into containers containing <em>L. japonicus</em> plants well-colonized by <em>G. margarita</em> or mock-inoculated plants and then allowed to grow for a week. Then, roots of the younger seedlings were harvested from the container inoculated with <em>G. margarita</em> (Gi1) or the mock-inoculated container (gni1). Radio-labeled target cDNAs were synthesized from total RNAs in the roots and then hybridized to a nylon filter cDNA array. <sup><a reveal-id="DSM014C26" data-open="DSM014C26" class="link link-ref link-reveal xref-bibr">26</a></sup> The normalized expression levels are shown in the gni1 and Gi1 columns. The expression levels relative to the mock-infected controls are given in the fold column. Genes that match hypothetical proteins of unknown function and ones that encode proteins exhibiting no homology to thus far known ones have been omitted from this table. </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/37284018" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM014TB3" class="table-wrap-title" data-id="DSM014TB3"><div class="label -label"><strong>TableÂ 3</strong></div><div class="caption"><p> Transcriptional changes caused by <em>G. margarita</em> infection in the initial stage of symbiosis </p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Current annotationÂ </th><th>Fold (Gi1/gni1)Â </th><th>Gene IDÂ </th><th>gni1Â </th><th>Gi1Â </th><th>e-valueÂ </th></tr></thead><tbody><tr><td><em>Amino acid and nitrogen metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Selenocysteine methyltransferaseÂ </td><td>2.24Â </td><td>MWM066h10_rÂ </td><td>62Â </td><td>139Â </td><td>6Eâˆ’25Â </td></tr><tr><td>Diaminopimelate decarboxylaseÂ </td><td>2.05Â </td><td>MWM149b05_rÂ </td><td>128Â </td><td>262Â </td><td>1Eâˆ’66Â </td></tr><tr><td><em>S</em> -adenosyl methionine synthetase Â </td><td>2.02Â </td><td>MWM180f07_rÂ </td><td>145Â </td><td>292Â </td><td>4Eâˆ’67Â </td></tr><tr><td>VuP5CRÂ </td><td>0.28Â </td><td>GENLf018g02Â </td><td>78Â </td><td>21Â </td><td>4Eâˆ’24Â </td></tr><tr><td>Arginine decarboxylaseÂ </td><td>0.29Â </td><td>MWM198e12_rÂ </td><td>102Â </td><td>29Â </td><td>2Eâˆ’49Â </td></tr><tr><td>Aminotransferase 2Â </td><td>0.33Â </td><td>MWM222b09_rÂ </td><td>101Â </td><td>33Â </td><td>5Eâˆ’66Â </td></tr><tr><td>Proline dehydrogenaseÂ </td><td>0.36Â </td><td>MWM135h10_rÂ </td><td>99Â </td><td>35Â </td><td>8Eâˆ’29Â </td></tr><tr><td>Delta-1-pyrroline-5-carboxylate synthaseÂ </td><td>0.36Â </td><td>GENLf045e06Â </td><td>121Â </td><td>43Â </td><td>4Eâˆ’09Â </td></tr><tr><td><em>Carbon metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>InvertaseÂ </td><td>2.32Â </td><td>MWM224d02_rÂ </td><td>150Â </td><td>347Â </td><td>6Eâˆ’08Â </td></tr><tr><td>Sucrose synthaseÂ </td><td>2.27Â </td><td>MWL068h11_rÂ </td><td>271Â </td><td>614Â </td><td>1Eâˆ’58Â </td></tr><tr><td>Glucose-1-phosphate adenylyltransferaseÂ </td><td>2.13Â </td><td>MWM086h02_rÂ </td><td>128Â </td><td>271Â </td><td>7Eâˆ’65Â </td></tr><tr><td>Alpha-mannosidaseÂ </td><td>2.11Â </td><td>GENLf064h06Â </td><td>138Â </td><td>291Â </td><td>4Eâˆ’24Â </td></tr><tr><td>Glucose-6-phosphate dehydrogenaseÂ </td><td>2.07Â </td><td>MR098a03_fÂ </td><td>47Â </td><td>93Â </td><td>7Eâˆ’06Â </td></tr><tr><td>Beta-amylaseÂ </td><td>2.06Â </td><td>GENf097b02Â </td><td>111Â </td><td>229Â </td><td>1Eâˆ’132Â </td></tr><tr><td>Triosephosphate isomeraseÂ </td><td>2.05Â </td><td>MWM193g10_rÂ </td><td>272Â </td><td>559Â </td><td>1Eâˆ’77Â </td></tr><tr><td>Glucosyltransferase-like proteinÂ </td><td>2.01Â </td><td>MWL049f07_rÂ </td><td>126Â </td><td>257Â </td><td>9Eâˆ’06Â </td></tr><tr><td> Beta- <span class="small-caps">d</span> -xylosidase Â </td><td>0.26Â </td><td>MWM219c11_rÂ </td><td>119Â </td><td>31Â </td><td>9Eâˆ’55Â </td></tr><tr><td>Fructose-bisphosphate aldolaseÂ </td><td>0.29Â </td><td>MWM024h09_rÂ </td><td>95Â </td><td>28Â </td><td>2Eâˆ’39Â </td></tr><tr><td>Malonyl-CoA: acyl carrier protein transacylaseÂ </td><td>0.30Â </td><td>MWL014e01_rÂ </td><td>115Â </td><td>34Â </td><td>4Eâˆ’33Â </td></tr><tr><td>Putative 2-isopropylmalate synthaseÂ </td><td>0.31Â </td><td>GENf086f07Â </td><td>76Â </td><td>23Â </td><td>4Eâˆ’34Â </td></tr><tr><td>Mannosyltransferase-like proteinÂ </td><td>0.32Â </td><td>MWM235d07_rÂ </td><td>92Â </td><td>28Â </td><td>3Eâˆ’04Â </td></tr><tr><td>Citrate synthaseÂ </td><td>0.37Â </td><td>MWM239b12_rÂ </td><td>147Â </td><td>54Â </td><td>4Eâˆ’38Â </td></tr><tr><td><em>Secondary metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Chalcone reductaseÂ </td><td>3.03Â </td><td>MWM002d07_rÂ </td><td>98Â </td><td>306Â </td><td>1Eâˆ’60Â </td></tr><tr><td>Phenylalanine ammonia-lyase (LjPAL2)Â </td><td>2.16Â </td><td>GENLf058e04Â </td><td>95Â </td><td>202Â </td><td>0.006Â </td></tr><tr><td>Phenylalanine ammonia-lyase (LjPAL1)Â </td><td>2.09Â </td><td>MRL007g11_fÂ </td><td>136Â </td><td>283Â </td><td>9Eâˆ’38Â </td></tr><tr><td>4-coumarate:CoA ligaseÂ </td><td>2.10Â </td><td>MWL020d04_rÂ </td><td>110Â </td><td>235Â </td><td>3Eâˆ’34Â </td></tr><tr><td>Chalcone synthaseÂ </td><td>2.05Â </td><td>MWL020g05_rÂ </td><td>144Â </td><td>295Â </td><td>1Eâˆ’14Â </td></tr><tr><td>LaccaseÂ </td><td>0.30Â </td><td>MWM219c08_rÂ </td><td>124Â </td><td>36Â </td><td>3Eâˆ’44Â </td></tr><tr><td>Putative diphenol oxidaseÂ </td><td>0.34Â </td><td>GENf055e07Â </td><td>87Â </td><td>29Â </td><td>1Eâˆ’13Â </td></tr><tr><td><em>Transport/membrane</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td> Plasma membrane Ca2 <sup>+</sup> -ATPase Â </td><td>2.34Â </td><td>MWM178b05_rÂ </td><td>169Â </td><td>396Â </td><td>3Eâˆ’74Â </td></tr><tr><td>Aquaporin protein PIP1Â </td><td>2.34Â </td><td>MWL033d08_rÂ </td><td>279Â </td><td>652Â </td><td>8Eâˆ’51Â </td></tr><tr><td>Putative amino acid transporterÂ </td><td>2.14Â </td><td>MWL064b03_rÂ </td><td>71Â </td><td>150Â </td><td>8Eâˆ’51Â </td></tr><tr><td>Putative ABC transporter proteinÂ </td><td>2.13Â </td><td>MWL077b08_rÂ </td><td>158Â </td><td>335Â </td><td>1Eâˆ’14Â </td></tr><tr><td>Putative nuclear transport factorÂ </td><td>2.09Â </td><td>MWM105a05_rÂ </td><td>118Â </td><td>248Â </td><td>1Eâˆ’43Â </td></tr><tr><td>Vacuolar ATPaseÂ </td><td>2.06Â </td><td>MWM238d10_rÂ </td><td>84Â </td><td>173Â </td><td>4Eâˆ’56Â </td></tr><tr><td>Nuclear transport factorÂ </td><td>2.02Â </td><td>GNf048c09Â </td><td>84Â </td><td>165Â </td><td>1Eâˆ’38Â </td></tr><tr><td>Sorbitol transporterÂ </td><td>2.01Â </td><td>MWM244a03_rÂ </td><td>41Â </td><td>79Â </td><td>4Eâˆ’57Â </td></tr><tr><td>MATE efflux family proteinÂ </td><td>0.23Â </td><td>GENLf013e12Â </td><td>111Â </td><td>25Â </td><td>2Eâˆ’06Â </td></tr><tr><td>Cation-transporting ATPaseÂ </td><td>0.31Â </td><td>MWM087d06_rÂ </td><td>154Â </td><td>46Â </td><td>3Eâˆ’20Â </td></tr><tr><td>Plastidic phosphate translocator-like proteinÂ </td><td>0.32Â </td><td>GENf029g12Â </td><td>108Â </td><td>34Â </td><td>6Eâˆ’96Â </td></tr><tr><td><em>Nodulin</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Early nodulin ENOD18Â </td><td>2.65Â </td><td>GENf079d10Â </td><td>34Â </td><td>90Â </td><td>2Eâˆ’14Â </td></tr><tr><td><em>Signal transduction</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Protein phosphatase 2CÂ </td><td>2.40Â </td><td>MWM035c07_rÂ </td><td>248Â </td><td>596Â </td><td>1Eâˆ’52Â </td></tr><tr><td>Protein serine/threonine kinaseÂ </td><td>2.35Â </td><td>MWM206a09_rÂ </td><td>74Â </td><td>174Â </td><td>4Eâˆ’29Â </td></tr><tr><td>G protein-coupled receptorÂ </td><td>2.33Â </td><td>GENLf064g09Â </td><td>70Â </td><td>163Â </td><td>Â </td></tr><tr><td>Receptor protein kinaseÂ </td><td>2.31Â </td><td>MPDL044b10_fÂ </td><td>56Â </td><td>130Â </td><td>7Eâˆ’18Â </td></tr><tr><td>Small GTP-binding proteinÂ </td><td>2.18Â </td><td>MWM122f01_rÂ </td><td>174Â </td><td>380Â </td><td>5Eâˆ’44Â </td></tr><tr><td>GUN4 regulatorÂ </td><td>2.12Â </td><td>MPDL091h07_fÂ </td><td>149Â </td><td>316Â </td><td>1Eâˆ’63Â </td></tr><tr><td>Putative acid phosphataseÂ </td><td>2.09Â </td><td>MWM048e06_rÂ </td><td>415Â </td><td>866Â </td><td>8Eâˆ’24Â </td></tr><tr><td>MAP kinase 3Â </td><td>2.09Â </td><td>MR062e03_fÂ </td><td>582Â </td><td>1215Â </td><td>3Eâˆ’84Â </td></tr><tr><td>Putative GTP-binding proteinÂ </td><td>2.05Â </td><td>MWM119e05_rÂ </td><td>161Â </td><td>329Â </td><td>4Eâˆ’34Â </td></tr><tr><td>Protein phosphatase-2CÂ </td><td>2.05Â </td><td>MWM050f11_rÂ </td><td>89Â </td><td>182Â </td><td>0.0003Â </td></tr><tr><td>Serine/threonine protein phosphataseÂ </td><td>2.05Â </td><td>MR028a01_fÂ </td><td>85Â </td><td>173Â </td><td>2Eâˆ’27Â </td></tr><tr><td>Calcium-dependent protein kinaseÂ </td><td>2.01Â </td><td>MWM060b03_rÂ </td><td>56Â </td><td>113Â </td><td>3Eâˆ’05Â </td></tr><tr><td>Protein kinaseÂ </td><td>0.25Â </td><td>MWL036f06_rÂ </td><td>128Â </td><td>32Â </td><td>3Eâˆ’10Â </td></tr><tr><td>PAP-specific phosphataseÂ </td><td>0.26Â </td><td>MWM204g03_rÂ </td><td>104Â </td><td>28Â </td><td>7Eâˆ’25Â </td></tr><tr><td>PP2A regulatory subunitÂ </td><td>0.32Â </td><td>MWM123a10_rÂ </td><td>124Â </td><td>39Â </td><td>4Eâˆ’22Â </td></tr><tr><td>Hydrolase/ inositol or phosphatidylinositol phosphataseÂ </td><td>0.37Â </td><td>MWM231g02_rÂ </td><td>75Â </td><td>27Â </td><td>5Eâˆ’27Â </td></tr><tr><td><em>Transcription/translation</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Putative bZIP transcription factorÂ </td><td>3.10Â </td><td>MWM014e09_rÂ </td><td>65Â </td><td>202Â </td><td>1Eâˆ’30Â </td></tr><tr><td>Eukaryotic initiation factorÂ </td><td>2.71Â </td><td>MWM099b12_rÂ </td><td>380Â </td><td>1030Â </td><td>2Eâˆ’04Â </td></tr><tr><td>Poly(A)-binding proteinÂ </td><td>2.40Â </td><td>MWM214d01_rÂ </td><td>148Â </td><td>352Â </td><td>1Eâˆ’44Â </td></tr><tr><td>Putative aspartate-tRNA ligaseÂ </td><td>2.34Â </td><td>MWM096c11_rÂ </td><td>187Â </td><td>438Â </td><td>3Eâˆ’23Â </td></tr><tr><td>Homeobox domain proteinÂ </td><td>2.07Â </td><td>MPD034c07_fÂ </td><td>51Â </td><td>104Â </td><td>8Eâˆ’79Â </td></tr><tr><td>Heat shock transcription factorÂ </td><td>2.03Â </td><td>MR008f01_fÂ </td><td>58Â </td><td>117Â </td><td>8Eâˆ’39Â </td></tr><tr><td>Glycine-rich RNA-binding proteinÂ </td><td>2.01Â </td><td>GENLf028b01Â </td><td>189Â </td><td>379Â </td><td>3Eâˆ’36Â </td></tr><tr><td>Putative squamosa promoter-binding proteinÂ </td><td>0.25Â </td><td>GENLf063g01Â </td><td>132Â </td><td>33Â </td><td>7Eâˆ’07Â </td></tr><tr><td>SDL-1 plastid proteinÂ </td><td>0.29Â </td><td>GENLf045h01Â </td><td>112Â </td><td>32Â </td><td>9Eâˆ’92Â </td></tr><tr><td>Transcription factor MYB4 homologÂ </td><td>0.31Â </td><td>MPD092a11_fÂ </td><td>67Â </td><td>20Â </td><td>1Eâˆ’22Â </td></tr><tr><td>Transcription regulatory proteinÂ </td><td>0.32Â </td><td>MWM065b02_rÂ </td><td>116Â </td><td>37Â </td><td>Â </td></tr><tr><td>Putative DOF zinc finger proteinÂ </td><td>0.33Â </td><td>MWM178e06_rÂ </td><td>118Â </td><td>38Â </td><td>1Eâˆ’21Â </td></tr><tr><td>Putative translation initiation proteinÂ </td><td>0.33Â </td><td>GENLf057g11Â </td><td>106Â </td><td>32Â </td><td>Â </td></tr><tr><td><em>Cell wall</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Pectin acetylesteraseÂ </td><td>2.44Â </td><td>MWM096a11_rÂ </td><td>174Â </td><td>424Â </td><td>9Eâˆ’54Â </td></tr><tr><td>Putative pectinesteraseÂ </td><td>2.05Â </td><td>MWM097c10_rÂ </td><td>278Â </td><td>570Â </td><td>3Eâˆ’08Â </td></tr><tr><td>PectinesteraseÂ </td><td>2.02Â </td><td>MWM132g12_rÂ </td><td>85Â </td><td>172Â </td><td>1Eâˆ’55Â </td></tr><tr><td>Callose synthaseÂ </td><td>0.36Â </td><td>GENLf063h05Â </td><td>92Â </td><td>32Â </td><td>2Eâˆ’28Â </td></tr><tr><td><em>Protein fate</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Protein secretion pathway proteinÂ </td><td>2.08Â </td><td>GENLf046b01Â </td><td>124Â </td><td>257Â </td><td>Â </td></tr><tr><td>Dipeptidyl peptidase IV-like proteinÂ </td><td>0.27Â </td><td>MPDL020f10_fÂ </td><td>106Â </td><td>28Â </td><td>7Eâˆ’35Â </td></tr><tr><td>Oligopeptidase AÂ </td><td>0.29Â </td><td>MWM031e10_rÂ </td><td>131Â </td><td>37Â </td><td>3Eâˆ’50Â </td></tr><tr><td>26S proteasome ATPase subunitÂ </td><td>0.31Â </td><td>MWM223f06_rÂ </td><td>80Â </td><td>24Â </td><td>3Eâˆ’30Â </td></tr><tr><td>Putative ubiquitin carboxyl terminal hydrolaseÂ </td><td>0.33Â </td><td>MPDL041a06_fÂ </td><td>108Â </td><td>35Â </td><td>2Eâˆ’18Â </td></tr><tr><td>Pro-X carboxypeptidase-like proteinÂ </td><td>0.36Â </td><td>MPD016e03_fÂ </td><td>86Â </td><td>31Â </td><td>Â </td></tr><tr><td>Serine protease inhibitor phloem serpin-1Â </td><td>0.37Â </td><td>GENf065b05Â </td><td>77Â </td><td>30Â </td><td>3Eâˆ’26Â </td></tr><tr><td><em>Pathogen-related</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Respiratory burst oxidase protein DÂ </td><td>2.57Â </td><td>GENLf020h11Â </td><td>54Â </td><td>138Â </td><td>3Eâˆ’41Â </td></tr><tr><td>Syringolide-induced proteinÂ </td><td>2.43Â </td><td>MWM033e05_rÂ </td><td>123Â </td><td>296Â </td><td>8Eâˆ’14Â </td></tr><tr><td>Peroxidase 3 precursorÂ </td><td>2.12Â </td><td>MWM241c09_rÂ </td><td>42Â </td><td>88Â </td><td>1Eâˆ’13Â </td></tr><tr><td>Syringolide-induced protein 14-1-1Â </td><td>2.06Â </td><td>MWM031c04_rÂ </td><td>95Â </td><td>195Â </td><td>8Eâˆ’27Â </td></tr><tr><td>Disease resistance-related proteinÂ </td><td>2.04Â </td><td>MWM067e07_rÂ </td><td>74Â </td><td>153Â </td><td>6Eâˆ’23Â </td></tr><tr><td>Class III peroxidase PSYP1Â </td><td>2.03Â </td><td>MWL018a05_rÂ </td><td>82Â </td><td>166Â </td><td>2Eâˆ’37Â </td></tr><tr><td>Endo-1,4-beta-glucanaseÂ </td><td>0.14Â </td><td>MWL011b05_rÂ </td><td>244Â </td><td>34Â </td><td>9Eâˆ’30Â </td></tr><tr><td>Syringolide-induced proteinÂ </td><td>0.35Â </td><td>MWM037b07_rÂ </td><td>139Â </td><td>48Â </td><td>2Eâˆ’07Â </td></tr><tr><td>Syringolide-induced proteinÂ </td><td>0.41Â </td><td>MWM014d11_rÂ </td><td>102Â </td><td>41Â </td><td>8Eâˆ’64Â </td></tr><tr><td><em>Phytohormone-related</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Auxin-repressed proteinÂ </td><td>2.13Â </td><td>MPDL064h08_fÂ </td><td>270Â </td><td>574Â </td><td>1Eâˆ’14Â </td></tr><tr><td>Cytokinin oxidaseÂ </td><td>0.24Â </td><td>MWM042d03_rÂ </td><td>129Â </td><td>28Â </td><td>3Eâˆ’35Â </td></tr><tr><td><em>Other enzymes</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Cytochrome P450Â </td><td>2.52Â </td><td>MWM049d04_rÂ </td><td>234Â </td><td>589Â </td><td>8Eâˆ’55Â </td></tr><tr><td>Cytochrome P450Â </td><td>2.51Â </td><td>MR061f02_fÂ </td><td>167Â </td><td>418Â </td><td>7Eâˆ’35Â </td></tr><tr><td>Phosphogluconate dehydrogenaseÂ </td><td>2.47Â </td><td>MWM228b11_rÂ </td><td>193Â </td><td>475Â </td><td>1Eâˆ’49Â </td></tr><tr><td>Fatty acid hydroxylase cytochrome P450Â </td><td>2.28Â </td><td>MWM051a05_rÂ </td><td>88Â </td><td>201Â </td><td>3Eâˆ’26Â </td></tr><tr><td>Cytochrome P450Â </td><td>2.21Â </td><td>MR043g06_fÂ </td><td>465Â </td><td>1027Â </td><td>2Eâˆ’09Â </td></tr><tr><td>Cytochrome P450Â </td><td>2.20Â </td><td>MWM152a11_rÂ </td><td>43Â </td><td>93Â </td><td>4Eâˆ’29Â </td></tr><tr><td>Epoxide hydrolaseÂ </td><td>2.11Â </td><td>MWM079e11_rÂ </td><td>56Â </td><td>119Â </td><td>4Eâˆ’54Â </td></tr><tr><td>Thiazole biosynthetic enzymeÂ </td><td>2.08Â </td><td>MWM107g04_rÂ </td><td>150Â </td><td>311Â </td><td>7Eâˆ’33Â </td></tr><tr><td>Putative helicaseÂ </td><td>2.03Â </td><td>MWL079f07_rÂ </td><td>157Â </td><td>317Â </td><td>7Eâˆ’11Â </td></tr><tr><td>Phosphatidylserine decarboxylaseÂ </td><td>2.02Â </td><td>MWM214c03_rÂ </td><td>65Â </td><td>131Â </td><td>1Eâˆ’57Â </td></tr><tr><td>Isopentenyl-diphosphate isomerase IIÂ </td><td>0.21Â </td><td>MWM082f11_rÂ </td><td>123Â </td><td>25Â </td><td>2Eâˆ’26Â </td></tr><tr><td>Retroelement pol polyprotein-likeÂ </td><td>0.24Â </td><td>MWL062c10_rÂ </td><td>129Â </td><td>30Â </td><td>2Eâˆ’26Â </td></tr><tr><td>Histone acetyltransferase HAT BÂ </td><td>0.24Â </td><td>MWM193c03_rÂ </td><td>89Â </td><td>20Â </td><td>4Eâˆ’13Â </td></tr><tr><td>Thiamine biosynthetic enzymeÂ </td><td>0.25Â </td><td>GENf012a12Â </td><td>102Â </td><td>25Â </td><td>3Eâˆ’57Â </td></tr><tr><td>Cytochrome P450Â </td><td>0.32Â </td><td>MWM170d07_rÂ </td><td>158Â </td><td>50Â </td><td>2Eâˆ’63Â </td></tr><tr><td>Obtusifoliol 14-alpha demethylaseÂ </td><td>0.33Â </td><td>GENf014g11Â </td><td>137Â </td><td>45Â </td><td>5Eâˆ’54Â </td></tr><tr><td>Magnesium chelataseÂ </td><td>0.34Â </td><td>MWL046f07_rÂ </td><td>81Â </td><td>27Â </td><td>5Eâˆ’56Â </td></tr><tr><td>UMP synthaseÂ </td><td>0.35Â </td><td>MWM187d07_rÂ </td><td>120Â </td><td>41Â </td><td>3Eâˆ’53Â </td></tr><tr><td>Putative cytochrome P450Â </td><td>0.35Â </td><td>MWM139c03_rÂ </td><td>95Â </td><td>33Â </td><td>3Eâˆ’59Â </td></tr><tr><td>Dihydroneopterin aldolaseÂ </td><td>0.36Â </td><td>GENLf038a07Â </td><td>74Â </td><td>26Â </td><td>2Eâˆ’24Â </td></tr><tr><td><em>Other categories</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Polyubiquitin 4Â </td><td>2.60Â </td><td>MWM214g11_rÂ </td><td>119Â </td><td>308Â </td><td>3Eâˆ’09Â </td></tr><tr><td>Metallothionein-like protein class IIÂ </td><td>2.54Â </td><td>MWM200f03_rÂ </td><td>1109Â </td><td>2821Â </td><td>3Eâˆ’19Â </td></tr><tr><td>CPRD49Â </td><td>2.42Â </td><td>MWM128g09_rÂ </td><td>148Â </td><td>357Â </td><td>1Eâˆ’17Â </td></tr><tr><td>Ubiquitin precursorÂ </td><td>2.28Â </td><td>MWM011f03_rÂ </td><td>544Â </td><td>1242Â </td><td>2Eâˆ’10Â </td></tr><tr><td>DnaJ-like proteinÂ </td><td>2.04Â </td><td>MWM184b12_rÂ </td><td>141Â </td><td>288Â </td><td>8Eâˆ’71Â </td></tr><tr><td>Bax inhibitor-1 likeÂ </td><td>2.04Â </td><td>MWM016c06_rÂ </td><td>143Â </td><td>293Â </td><td>4Eâˆ’50Â </td></tr><tr><td>Heat shock protein 70 cognateÂ </td><td>2.02Â </td><td>MWM159a01_rÂ </td><td>174Â </td><td>351Â </td><td>2Eâˆ’62Â </td></tr><tr><td>Putative 2Fe-2S ironâ€“sulfur cluster proteinÂ </td><td>2.01Â </td><td>MPD065e04_fÂ </td><td>51Â </td><td>102Â </td><td>3Eâˆ’07Â </td></tr><tr><td>Ankyrin-repeat proteinÂ </td><td>2.00Â </td><td>MWM067b10_rÂ </td><td>428Â </td><td>856Â </td><td>8Eâˆ’17Â </td></tr><tr><td>Vacuolar sorting receptor protein BP-80Â </td><td>2.00Â </td><td>MWL009b09_rÂ </td><td>173Â </td><td>345Â </td><td>9Eâˆ’20Â </td></tr><tr><td>Calcineurin B-like proteinÂ </td><td>0.25Â </td><td>MWM143g03_rÂ </td><td>82Â </td><td>20Â </td><td>0.007Â </td></tr><tr><td>Peroxiredoxin QÂ </td><td>0.31Â </td><td>MWM126d02_rÂ </td><td>97Â </td><td>29Â </td><td>1Eâˆ’16Â </td></tr><tr><td>Senescence-associated putative proteinÂ </td><td>0.33Â </td><td>MWL051e09_rÂ </td><td>110Â </td><td>35Â </td><td>1Eâˆ’16Â </td></tr><tr><td>PSII low MW proteinÂ </td><td>0.34Â </td><td>MWL078e10_rÂ </td><td>578Â </td><td>198Â </td><td>4Eâˆ’19Â </td></tr><tr><td>Histone H2AÂ </td><td>0.35Â </td><td>MWM209e08_rÂ </td><td>133Â </td><td>46Â </td><td>9Eâˆ’51Â </td></tr><tr><td>ActinÂ </td><td>0.36Â </td><td>GENf007a10Â </td><td>82Â </td><td>29Â </td><td>1Eâˆ’48Â </td></tr><tr><td>Phosphatidylinositol transfer-like protein IVÂ </td><td>0.36Â </td><td>GENf020a06Â </td><td>125Â </td><td>44Â </td><td>6Eâˆ’20Â </td></tr><tr><td>Chlorophyll a/b-binding protein type IIÂ </td><td>0.36Â </td><td>MPD059g06_fÂ </td><td>95Â </td><td>34Â </td><td>7Eâˆ’46Â </td></tr><tr><td>Early light-inducible proteinÂ </td><td>0.37Â </td><td>MWL040f08_rÂ </td><td>85Â </td><td>31Â </td><td>2Eâˆ’43Â </td></tr><tr><td>Cytochrome b/fÂ </td><td>0.37Â </td><td>MWM225h10_rÂ </td><td>325Â </td><td>120Â </td><td>4Eâˆ’71Â </td></tr></tbody></table></div><div content-id="DSM014TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Current annotationÂ </th><th>Fold (Gi1/gni1)Â </th><th>Gene IDÂ </th><th>gni1Â </th><th>Gi1Â </th><th>e-valueÂ </th></tr></thead><tbody><tr><td><em>Amino acid and nitrogen metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Selenocysteine methyltransferaseÂ </td><td>2.24Â </td><td>MWM066h10_rÂ </td><td>62Â </td><td>139Â </td><td>6Eâˆ’25Â </td></tr><tr><td>Diaminopimelate decarboxylaseÂ </td><td>2.05Â </td><td>MWM149b05_rÂ </td><td>128Â </td><td>262Â </td><td>1Eâˆ’66Â </td></tr><tr><td><em>S</em> -adenosyl methionine synthetase Â </td><td>2.02Â </td><td>MWM180f07_rÂ </td><td>145Â </td><td>292Â </td><td>4Eâˆ’67Â </td></tr><tr><td>VuP5CRÂ </td><td>0.28Â </td><td>GENLf018g02Â </td><td>78Â </td><td>21Â </td><td>4Eâˆ’24Â </td></tr><tr><td>Arginine decarboxylaseÂ </td><td>0.29Â </td><td>MWM198e12_rÂ </td><td>102Â </td><td>29Â </td><td>2Eâˆ’49Â </td></tr><tr><td>Aminotransferase 2Â </td><td>0.33Â </td><td>MWM222b09_rÂ </td><td>101Â </td><td>33Â </td><td>5Eâˆ’66Â </td></tr><tr><td>Proline dehydrogenaseÂ </td><td>0.36Â </td><td>MWM135h10_rÂ </td><td>99Â </td><td>35Â </td><td>8Eâˆ’29Â </td></tr><tr><td>Delta-1-pyrroline-5-carboxylate synthaseÂ </td><td>0.36Â </td><td>GENLf045e06Â </td><td>121Â </td><td>43Â </td><td>4Eâˆ’09Â </td></tr><tr><td><em>Carbon metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>InvertaseÂ </td><td>2.32Â </td><td>MWM224d02_rÂ </td><td>150Â </td><td>347Â </td><td>6Eâˆ’08Â </td></tr><tr><td>Sucrose synthaseÂ </td><td>2.27Â </td><td>MWL068h11_rÂ </td><td>271Â </td><td>614Â </td><td>1Eâˆ’58Â </td></tr><tr><td>Glucose-1-phosphate adenylyltransferaseÂ </td><td>2.13Â </td><td>MWM086h02_rÂ </td><td>128Â </td><td>271Â </td><td>7Eâˆ’65Â </td></tr><tr><td>Alpha-mannosidaseÂ </td><td>2.11Â </td><td>GENLf064h06Â </td><td>138Â </td><td>291Â </td><td>4Eâˆ’24Â </td></tr><tr><td>Glucose-6-phosphate dehydrogenaseÂ </td><td>2.07Â </td><td>MR098a03_fÂ </td><td>47Â </td><td>93Â </td><td>7Eâˆ’06Â </td></tr><tr><td>Beta-amylaseÂ </td><td>2.06Â </td><td>GENf097b02Â </td><td>111Â </td><td>229Â </td><td>1Eâˆ’132Â </td></tr><tr><td>Triosephosphate isomeraseÂ </td><td>2.05Â </td><td>MWM193g10_rÂ </td><td>272Â </td><td>559Â </td><td>1Eâˆ’77Â </td></tr><tr><td>Glucosyltransferase-like proteinÂ </td><td>2.01Â </td><td>MWL049f07_rÂ </td><td>126Â </td><td>257Â </td><td>9Eâˆ’06Â </td></tr><tr><td> Beta- <span class="small-caps">d</span> -xylosidase Â </td><td>0.26Â </td><td>MWM219c11_rÂ </td><td>119Â </td><td>31Â </td><td>9Eâˆ’55Â </td></tr><tr><td>Fructose-bisphosphate aldolaseÂ </td><td>0.29Â </td><td>MWM024h09_rÂ </td><td>95Â </td><td>28Â </td><td>2Eâˆ’39Â </td></tr><tr><td>Malonyl-CoA: acyl carrier protein transacylaseÂ </td><td>0.30Â </td><td>MWL014e01_rÂ </td><td>115Â </td><td>34Â </td><td>4Eâˆ’33Â </td></tr><tr><td>Putative 2-isopropylmalate synthaseÂ </td><td>0.31Â </td><td>GENf086f07Â </td><td>76Â </td><td>23Â </td><td>4Eâˆ’34Â </td></tr><tr><td>Mannosyltransferase-like proteinÂ </td><td>0.32Â </td><td>MWM235d07_rÂ </td><td>92Â </td><td>28Â </td><td>3Eâˆ’04Â </td></tr><tr><td>Citrate synthaseÂ </td><td>0.37Â </td><td>MWM239b12_rÂ </td><td>147Â </td><td>54Â </td><td>4Eâˆ’38Â </td></tr><tr><td><em>Secondary metabolism</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Chalcone reductaseÂ </td><td>3.03Â </td><td>MWM002d07_rÂ </td><td>98Â </td><td>306Â </td><td>1Eâˆ’60Â </td></tr><tr><td>Phenylalanine ammonia-lyase (LjPAL2)Â </td><td>2.16Â </td><td>GENLf058e04Â </td><td>95Â </td><td>202Â </td><td>0.006Â </td></tr><tr><td>Phenylalanine ammonia-lyase (LjPAL1)Â </td><td>2.09Â </td><td>MRL007g11_fÂ </td><td>136Â </td><td>283Â </td><td>9Eâˆ’38Â </td></tr><tr><td>4-coumarate:CoA ligaseÂ </td><td>2.10Â </td><td>MWL020d04_rÂ </td><td>110Â </td><td>235Â </td><td>3Eâˆ’34Â </td></tr><tr><td>Chalcone synthaseÂ </td><td>2.05Â </td><td>MWL020g05_rÂ </td><td>144Â </td><td>295Â </td><td>1Eâˆ’14Â </td></tr><tr><td>LaccaseÂ </td><td>0.30Â </td><td>MWM219c08_rÂ </td><td>124Â </td><td>36Â </td><td>3Eâˆ’44Â </td></tr><tr><td>Putative diphenol oxidaseÂ </td><td>0.34Â </td><td>GENf055e07Â </td><td>87Â </td><td>29Â </td><td>1Eâˆ’13Â </td></tr><tr><td><em>Transport/membrane</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td> Plasma membrane Ca2 <sup>+</sup> -ATPase Â </td><td>2.34Â </td><td>MWM178b05_rÂ </td><td>169Â </td><td>396Â </td><td>3Eâˆ’74Â </td></tr><tr><td>Aquaporin protein PIP1Â </td><td>2.34Â </td><td>MWL033d08_rÂ </td><td>279Â </td><td>652Â </td><td>8Eâˆ’51Â </td></tr><tr><td>Putative amino acid transporterÂ </td><td>2.14Â </td><td>MWL064b03_rÂ </td><td>71Â </td><td>150Â </td><td>8Eâˆ’51Â </td></tr><tr><td>Putative ABC transporter proteinÂ </td><td>2.13Â </td><td>MWL077b08_rÂ </td><td>158Â </td><td>335Â </td><td>1Eâˆ’14Â </td></tr><tr><td>Putative nuclear transport factorÂ </td><td>2.09Â </td><td>MWM105a05_rÂ </td><td>118Â </td><td>248Â </td><td>1Eâˆ’43Â </td></tr><tr><td>Vacuolar ATPaseÂ </td><td>2.06Â </td><td>MWM238d10_rÂ </td><td>84Â </td><td>173Â </td><td>4Eâˆ’56Â </td></tr><tr><td>Nuclear transport factorÂ </td><td>2.02Â </td><td>GNf048c09Â </td><td>84Â </td><td>165Â </td><td>1Eâˆ’38Â </td></tr><tr><td>Sorbitol transporterÂ </td><td>2.01Â </td><td>MWM244a03_rÂ </td><td>41Â </td><td>79Â </td><td>4Eâˆ’57Â </td></tr><tr><td>MATE efflux family proteinÂ </td><td>0.23Â </td><td>GENLf013e12Â </td><td>111Â </td><td>25Â </td><td>2Eâˆ’06Â </td></tr><tr><td>Cation-transporting ATPaseÂ </td><td>0.31Â </td><td>MWM087d06_rÂ </td><td>154Â </td><td>46Â </td><td>3Eâˆ’20Â </td></tr><tr><td>Plastidic phosphate translocator-like proteinÂ </td><td>0.32Â </td><td>GENf029g12Â </td><td>108Â </td><td>34Â </td><td>6Eâˆ’96Â </td></tr><tr><td><em>Nodulin</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Early nodulin ENOD18Â </td><td>2.65Â </td><td>GENf079d10Â </td><td>34Â </td><td>90Â </td><td>2Eâˆ’14Â </td></tr><tr><td><em>Signal transduction</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Protein phosphatase 2CÂ </td><td>2.40Â </td><td>MWM035c07_rÂ </td><td>248Â </td><td>596Â </td><td>1Eâˆ’52Â </td></tr><tr><td>Protein serine/threonine kinaseÂ </td><td>2.35Â </td><td>MWM206a09_rÂ </td><td>74Â </td><td>174Â </td><td>4Eâˆ’29Â </td></tr><tr><td>G protein-coupled receptorÂ </td><td>2.33Â </td><td>GENLf064g09Â </td><td>70Â </td><td>163Â </td><td>Â </td></tr><tr><td>Receptor protein kinaseÂ </td><td>2.31Â </td><td>MPDL044b10_fÂ </td><td>56Â </td><td>130Â </td><td>7Eâˆ’18Â </td></tr><tr><td>Small GTP-binding proteinÂ </td><td>2.18Â </td><td>MWM122f01_rÂ </td><td>174Â </td><td>380Â </td><td>5Eâˆ’44Â </td></tr><tr><td>GUN4 regulatorÂ </td><td>2.12Â </td><td>MPDL091h07_fÂ </td><td>149Â </td><td>316Â </td><td>1Eâˆ’63Â </td></tr><tr><td>Putative acid phosphataseÂ </td><td>2.09Â </td><td>MWM048e06_rÂ </td><td>415Â </td><td>866Â </td><td>8Eâˆ’24Â </td></tr><tr><td>MAP kinase 3Â </td><td>2.09Â </td><td>MR062e03_fÂ </td><td>582Â </td><td>1215Â </td><td>3Eâˆ’84Â </td></tr><tr><td>Putative GTP-binding proteinÂ </td><td>2.05Â </td><td>MWM119e05_rÂ </td><td>161Â </td><td>329Â </td><td>4Eâˆ’34Â </td></tr><tr><td>Protein phosphatase-2CÂ </td><td>2.05Â </td><td>MWM050f11_rÂ </td><td>89Â </td><td>182Â </td><td>0.0003Â </td></tr><tr><td>Serine/threonine protein phosphataseÂ </td><td>2.05Â </td><td>MR028a01_fÂ </td><td>85Â </td><td>173Â </td><td>2Eâˆ’27Â </td></tr><tr><td>Calcium-dependent protein kinaseÂ </td><td>2.01Â </td><td>MWM060b03_rÂ </td><td>56Â </td><td>113Â </td><td>3Eâˆ’05Â </td></tr><tr><td>Protein kinaseÂ </td><td>0.25Â </td><td>MWL036f06_rÂ </td><td>128Â </td><td>32Â </td><td>3Eâˆ’10Â </td></tr><tr><td>PAP-specific phosphataseÂ </td><td>0.26Â </td><td>MWM204g03_rÂ </td><td>104Â </td><td>28Â </td><td>7Eâˆ’25Â </td></tr><tr><td>PP2A regulatory subunitÂ </td><td>0.32Â </td><td>MWM123a10_rÂ </td><td>124Â </td><td>39Â </td><td>4Eâˆ’22Â </td></tr><tr><td>Hydrolase/ inositol or phosphatidylinositol phosphataseÂ </td><td>0.37Â </td><td>MWM231g02_rÂ </td><td>75Â </td><td>27Â </td><td>5Eâˆ’27Â </td></tr><tr><td><em>Transcription/translation</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Putative bZIP transcription factorÂ </td><td>3.10Â </td><td>MWM014e09_rÂ </td><td>65Â </td><td>202Â </td><td>1Eâˆ’30Â </td></tr><tr><td>Eukaryotic initiation factorÂ </td><td>2.71Â </td><td>MWM099b12_rÂ </td><td>380Â </td><td>1030Â </td><td>2Eâˆ’04Â </td></tr><tr><td>Poly(A)-binding proteinÂ </td><td>2.40Â </td><td>MWM214d01_rÂ </td><td>148Â </td><td>352Â </td><td>1Eâˆ’44Â </td></tr><tr><td>Putative aspartate-tRNA ligaseÂ </td><td>2.34Â </td><td>MWM096c11_rÂ </td><td>187Â </td><td>438Â </td><td>3Eâˆ’23Â </td></tr><tr><td>Homeobox domain proteinÂ </td><td>2.07Â </td><td>MPD034c07_fÂ </td><td>51Â </td><td>104Â </td><td>8Eâˆ’79Â </td></tr><tr><td>Heat shock transcription factorÂ </td><td>2.03Â </td><td>MR008f01_fÂ </td><td>58Â </td><td>117Â </td><td>8Eâˆ’39Â </td></tr><tr><td>Glycine-rich RNA-binding proteinÂ </td><td>2.01Â </td><td>GENLf028b01Â </td><td>189Â </td><td>379Â </td><td>3Eâˆ’36Â </td></tr><tr><td>Putative squamosa promoter-binding proteinÂ </td><td>0.25Â </td><td>GENLf063g01Â </td><td>132Â </td><td>33Â </td><td>7Eâˆ’07Â </td></tr><tr><td>SDL-1 plastid proteinÂ </td><td>0.29Â </td><td>GENLf045h01Â </td><td>112Â </td><td>32Â </td><td>9Eâˆ’92Â </td></tr><tr><td>Transcription factor MYB4 homologÂ </td><td>0.31Â </td><td>MPD092a11_fÂ </td><td>67Â </td><td>20Â </td><td>1Eâˆ’22Â </td></tr><tr><td>Transcription regulatory proteinÂ </td><td>0.32Â </td><td>MWM065b02_rÂ </td><td>116Â </td><td>37Â </td><td>Â </td></tr><tr><td>Putative DOF zinc finger proteinÂ </td><td>0.33Â </td><td>MWM178e06_rÂ </td><td>118Â </td><td>38Â </td><td>1Eâˆ’21Â </td></tr><tr><td>Putative translation initiation proteinÂ </td><td>0.33Â </td><td>GENLf057g11Â </td><td>106Â </td><td>32Â </td><td>Â </td></tr><tr><td><em>Cell wall</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Pectin acetylesteraseÂ </td><td>2.44Â </td><td>MWM096a11_rÂ </td><td>174Â </td><td>424Â </td><td>9Eâˆ’54Â </td></tr><tr><td>Putative pectinesteraseÂ </td><td>2.05Â </td><td>MWM097c10_rÂ </td><td>278Â </td><td>570Â </td><td>3Eâˆ’08Â </td></tr><tr><td>PectinesteraseÂ </td><td>2.02Â </td><td>MWM132g12_rÂ </td><td>85Â </td><td>172Â </td><td>1Eâˆ’55Â </td></tr><tr><td>Callose synthaseÂ </td><td>0.36Â </td><td>GENLf063h05Â </td><td>92Â </td><td>32Â </td><td>2Eâˆ’28Â </td></tr><tr><td><em>Protein fate</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Protein secretion pathway proteinÂ </td><td>2.08Â </td><td>GENLf046b01Â </td><td>124Â </td><td>257Â </td><td>Â </td></tr><tr><td>Dipeptidyl peptidase IV-like proteinÂ </td><td>0.27Â </td><td>MPDL020f10_fÂ </td><td>106Â </td><td>28Â </td><td>7Eâˆ’35Â </td></tr><tr><td>Oligopeptidase AÂ </td><td>0.29Â </td><td>MWM031e10_rÂ </td><td>131Â </td><td>37Â </td><td>3Eâˆ’50Â </td></tr><tr><td>26S proteasome ATPase subunitÂ </td><td>0.31Â </td><td>MWM223f06_rÂ </td><td>80Â </td><td>24Â </td><td>3Eâˆ’30Â </td></tr><tr><td>Putative ubiquitin carboxyl terminal hydrolaseÂ </td><td>0.33Â </td><td>MPDL041a06_fÂ </td><td>108Â </td><td>35Â </td><td>2Eâˆ’18Â </td></tr><tr><td>Pro-X carboxypeptidase-like proteinÂ </td><td>0.36Â </td><td>MPD016e03_fÂ </td><td>86Â </td><td>31Â </td><td>Â </td></tr><tr><td>Serine protease inhibitor phloem serpin-1Â </td><td>0.37Â </td><td>GENf065b05Â </td><td>77Â </td><td>30Â </td><td>3Eâˆ’26Â </td></tr><tr><td><em>Pathogen-related</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Respiratory burst oxidase protein DÂ </td><td>2.57Â </td><td>GENLf020h11Â </td><td>54Â </td><td>138Â </td><td>3Eâˆ’41Â </td></tr><tr><td>Syringolide-induced proteinÂ </td><td>2.43Â </td><td>MWM033e05_rÂ </td><td>123Â </td><td>296Â </td><td>8Eâˆ’14Â </td></tr><tr><td>Peroxidase 3 precursorÂ </td><td>2.12Â </td><td>MWM241c09_rÂ </td><td>42Â </td><td>88Â </td><td>1Eâˆ’13Â </td></tr><tr><td>Syringolide-induced protein 14-1-1Â </td><td>2.06Â </td><td>MWM031c04_rÂ </td><td>95Â </td><td>195Â </td><td>8Eâˆ’27Â </td></tr><tr><td>Disease resistance-related proteinÂ </td><td>2.04Â </td><td>MWM067e07_rÂ </td><td>74Â </td><td>153Â </td><td>6Eâˆ’23Â </td></tr><tr><td>Class III peroxidase PSYP1Â </td><td>2.03Â </td><td>MWL018a05_rÂ </td><td>82Â </td><td>166Â </td><td>2Eâˆ’37Â </td></tr><tr><td>Endo-1,4-beta-glucanaseÂ </td><td>0.14Â </td><td>MWL011b05_rÂ </td><td>244Â </td><td>34Â </td><td>9Eâˆ’30Â </td></tr><tr><td>Syringolide-induced proteinÂ </td><td>0.35Â </td><td>MWM037b07_rÂ </td><td>139Â </td><td>48Â </td><td>2Eâˆ’07Â </td></tr><tr><td>Syringolide-induced proteinÂ </td><td>0.41Â </td><td>MWM014d11_rÂ </td><td>102Â </td><td>41Â </td><td>8Eâˆ’64Â </td></tr><tr><td><em>Phytohormone-related</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Auxin-repressed proteinÂ </td><td>2.13Â </td><td>MPDL064h08_fÂ </td><td>270Â </td><td>574Â </td><td>1Eâˆ’14Â </td></tr><tr><td>Cytokinin oxidaseÂ </td><td>0.24Â </td><td>MWM042d03_rÂ </td><td>129Â </td><td>28Â </td><td>3Eâˆ’35Â </td></tr><tr><td><em>Other enzymes</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Cytochrome P450Â </td><td>2.52Â </td><td>MWM049d04_rÂ </td><td>234Â </td><td>589Â </td><td>8Eâˆ’55Â </td></tr><tr><td>Cytochrome P450Â </td><td>2.51Â </td><td>MR061f02_fÂ </td><td>167Â </td><td>418Â </td><td>7Eâˆ’35Â </td></tr><tr><td>Phosphogluconate dehydrogenaseÂ </td><td>2.47Â </td><td>MWM228b11_rÂ </td><td>193Â </td><td>475Â </td><td>1Eâˆ’49Â </td></tr><tr><td>Fatty acid hydroxylase cytochrome P450Â </td><td>2.28Â </td><td>MWM051a05_rÂ </td><td>88Â </td><td>201Â </td><td>3Eâˆ’26Â </td></tr><tr><td>Cytochrome P450Â </td><td>2.21Â </td><td>MR043g06_fÂ </td><td>465Â </td><td>1027Â </td><td>2Eâˆ’09Â </td></tr><tr><td>Cytochrome P450Â </td><td>2.20Â </td><td>MWM152a11_rÂ </td><td>43Â </td><td>93Â </td><td>4Eâˆ’29Â </td></tr><tr><td>Epoxide hydrolaseÂ </td><td>2.11Â </td><td>MWM079e11_rÂ </td><td>56Â </td><td>119Â </td><td>4Eâˆ’54Â </td></tr><tr><td>Thiazole biosynthetic enzymeÂ </td><td>2.08Â </td><td>MWM107g04_rÂ </td><td>150Â </td><td>311Â </td><td>7Eâˆ’33Â </td></tr><tr><td>Putative helicaseÂ </td><td>2.03Â </td><td>MWL079f07_rÂ </td><td>157Â </td><td>317Â </td><td>7Eâˆ’11Â </td></tr><tr><td>Phosphatidylserine decarboxylaseÂ </td><td>2.02Â </td><td>MWM214c03_rÂ </td><td>65Â </td><td>131Â </td><td>1Eâˆ’57Â </td></tr><tr><td>Isopentenyl-diphosphate isomerase IIÂ </td><td>0.21Â </td><td>MWM082f11_rÂ </td><td>123Â </td><td>25Â </td><td>2Eâˆ’26Â </td></tr><tr><td>Retroelement pol polyprotein-likeÂ </td><td>0.24Â </td><td>MWL062c10_rÂ </td><td>129Â </td><td>30Â </td><td>2Eâˆ’26Â </td></tr><tr><td>Histone acetyltransferase HAT BÂ </td><td>0.24Â </td><td>MWM193c03_rÂ </td><td>89Â </td><td>20Â </td><td>4Eâˆ’13Â </td></tr><tr><td>Thiamine biosynthetic enzymeÂ </td><td>0.25Â </td><td>GENf012a12Â </td><td>102Â </td><td>25Â </td><td>3Eâˆ’57Â </td></tr><tr><td>Cytochrome P450Â </td><td>0.32Â </td><td>MWM170d07_rÂ </td><td>158Â </td><td>50Â </td><td>2Eâˆ’63Â </td></tr><tr><td>Obtusifoliol 14-alpha demethylaseÂ </td><td>0.33Â </td><td>GENf014g11Â </td><td>137Â </td><td>45Â </td><td>5Eâˆ’54Â </td></tr><tr><td>Magnesium chelataseÂ </td><td>0.34Â </td><td>MWL046f07_rÂ </td><td>81Â </td><td>27Â </td><td>5Eâˆ’56Â </td></tr><tr><td>UMP synthaseÂ </td><td>0.35Â </td><td>MWM187d07_rÂ </td><td>120Â </td><td>41Â </td><td>3Eâˆ’53Â </td></tr><tr><td>Putative cytochrome P450Â </td><td>0.35Â </td><td>MWM139c03_rÂ </td><td>95Â </td><td>33Â </td><td>3Eâˆ’59Â </td></tr><tr><td>Dihydroneopterin aldolaseÂ </td><td>0.36Â </td><td>GENLf038a07Â </td><td>74Â </td><td>26Â </td><td>2Eâˆ’24Â </td></tr><tr><td><em>Other categories</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td><td>Â </td></tr><tr><td>Polyubiquitin 4Â </td><td>2.60Â </td><td>MWM214g11_rÂ </td><td>119Â </td><td>308Â </td><td>3Eâˆ’09Â </td></tr><tr><td>Metallothionein-like protein class IIÂ </td><td>2.54Â </td><td>MWM200f03_rÂ </td><td>1109Â </td><td>2821Â </td><td>3Eâˆ’19Â </td></tr><tr><td>CPRD49Â </td><td>2.42Â </td><td>MWM128g09_rÂ </td><td>148Â </td><td>357Â </td><td>1Eâˆ’17Â </td></tr><tr><td>Ubiquitin precursorÂ </td><td>2.28Â </td><td>MWM011f03_rÂ </td><td>544Â </td><td>1242Â </td><td>2Eâˆ’10Â </td></tr><tr><td>DnaJ-like proteinÂ </td><td>2.04Â </td><td>MWM184b12_rÂ </td><td>141Â </td><td>288Â </td><td>8Eâˆ’71Â </td></tr><tr><td>Bax inhibitor-1 likeÂ </td><td>2.04Â </td><td>MWM016c06_rÂ </td><td>143Â </td><td>293Â </td><td>4Eâˆ’50Â </td></tr><tr><td>Heat shock protein 70 cognateÂ </td><td>2.02Â </td><td>MWM159a01_rÂ </td><td>174Â </td><td>351Â </td><td>2Eâˆ’62Â </td></tr><tr><td>Putative 2Fe-2S ironâ€“sulfur cluster proteinÂ </td><td>2.01Â </td><td>MPD065e04_fÂ </td><td>51Â </td><td>102Â </td><td>3Eâˆ’07Â </td></tr><tr><td>Ankyrin-repeat proteinÂ </td><td>2.00Â </td><td>MWM067b10_rÂ </td><td>428Â </td><td>856Â </td><td>8Eâˆ’17Â </td></tr><tr><td>Vacuolar sorting receptor protein BP-80Â </td><td>2.00Â </td><td>MWL009b09_rÂ </td><td>173Â </td><td>345Â </td><td>9Eâˆ’20Â </td></tr><tr><td>Calcineurin B-like proteinÂ </td><td>0.25Â </td><td>MWM143g03_rÂ </td><td>82Â </td><td>20Â </td><td>0.007Â </td></tr><tr><td>Peroxiredoxin QÂ </td><td>0.31Â </td><td>MWM126d02_rÂ </td><td>97Â </td><td>29Â </td><td>1Eâˆ’16Â </td></tr><tr><td>Senescence-associated putative proteinÂ </td><td>0.33Â </td><td>MWL051e09_rÂ </td><td>110Â </td><td>35Â </td><td>1Eâˆ’16Â </td></tr><tr><td>PSII low MW proteinÂ </td><td>0.34Â </td><td>MWL078e10_rÂ </td><td>578Â </td><td>198Â </td><td>4Eâˆ’19Â </td></tr><tr><td>Histone H2AÂ </td><td>0.35Â </td><td>MWM209e08_rÂ </td><td>133Â </td><td>46Â </td><td>9Eâˆ’51Â </td></tr><tr><td>ActinÂ </td><td>0.36Â </td><td>GENf007a10Â </td><td>82Â </td><td>29Â </td><td>1Eâˆ’48Â </td></tr><tr><td>Phosphatidylinositol transfer-like protein IVÂ </td><td>0.36Â </td><td>GENf020a06Â </td><td>125Â </td><td>44Â </td><td>6Eâˆ’20Â </td></tr><tr><td>Chlorophyll a/b-binding protein type IIÂ </td><td>0.36Â </td><td>MPD059g06_fÂ </td><td>95Â </td><td>34Â </td><td>7Eâˆ’46Â </td></tr><tr><td>Early light-inducible proteinÂ </td><td>0.37Â </td><td>MWL040f08_rÂ </td><td>85Â </td><td>31Â </td><td>2Eâˆ’43Â </td></tr><tr><td>Cytochrome b/fÂ </td><td>0.37Â </td><td>MWM225h10_rÂ </td><td>325Â </td><td>120Â </td><td>4Eâˆ’71Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p> Because there is a varying lag time between sporulation and the arrival of AM hyphae on the host roots, the initial response to mycorrhizae is not necessarily synchronous. Therefore, we modified the â€˜nurse potâ€™ method <sup><a reveal-id="DSM014C30" data-open="DSM014C30" class="link link-ref link-reveal xref-bibr">30</a></sup> as described under Materials and Methods. Freshly prepared sterile seedlings were transplanted into containers containing <em>L. japonicus</em> plants well-colonized by <em>G. margarita</em> or mock-inoculated plants and then allowed to grow for a week. Then, roots of the younger seedlings were harvested from the container inoculated with <em>G. margarita</em> (Gi1) or the mock-inoculated container (gni1). Radio-labeled target cDNAs were synthesized from total RNAs in the roots and then hybridized to a nylon filter cDNA array. <sup><a reveal-id="DSM014C26" data-open="DSM014C26" class="link link-ref link-reveal xref-bibr">26</a></sup> The normalized expression levels are shown in the gni1 and Gi1 columns. The expression levels relative to the mock-infected controls are given in the fold column. Genes that match hypothetical proteins of unknown function and ones that encode proteins exhibiting no homology to thus far known ones have been omitted from this table. </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/37284018" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=37284019 id="37284019" class="section-title" data-legacy-id=s3d>3.4. Expression patterns of PAL genes in L. japonicus</h3>
<p> PALs connect primary and secondary metabolism in plants, catalyzing common rate-limiting steps of flavonoid phytoalexin synthesis, lignin synthesis, salicylic acid synthesis, etc. The expression patterns of PAL genes in our experiments are very characteristic compared with those in previous studies. <sup><a reveal-id="DSM014C10" data-open="DSM014C10" class="link link-ref link-reveal xref-bibr">10</a> , <a reveal-id="DSM014C12" data-open="DSM014C12" class="link link-ref link-reveal xref-bibr">12</a> â€“ <a reveal-id="DSM014C20" data-open="DSM014C20" class="link link-ref link-reveal xref-bibr">20</a> , <a reveal-id="DSM014C41" data-open="DSM014C41" class="link link-ref link-reveal xref-bibr">41</a></sup> Since PAL genes are known to form a family in a number of plant species, <sup><a reveal-id="DSM014C55" data-open="DSM014C55" class="link link-ref link-reveal xref-bibr">55</a></sup> we first checked how many PAL genes were present on the array membrane and found nine non-redundant ones. In addition, we found a TAC clone (Accession no. AP004502) containing a unique PAL gene, <em>LjPAL5</em> , in the databases. As shown in TableÂ  <a reveal-id="DSM014TB4" data-open="DSM014TB4" class="link link-reveal link-table xref-fig">4</a> , most PAL genes were induced in the initial stage of AM infection and then repressed in the later stage. However, <em>LjPAL10</em> did not seem to be expressed differentially. In addition, other genes, <em>LjPAL7 and LjPAL9</em> , might be of the intermediate type. Thus, as pointed out previously, <sup><a reveal-id="DSM014C55" data-open="DSM014C55" class="link link-ref link-reveal xref-bibr">55</a></sup> care must be taken that PAL genes do not show similar expression patterns. Although the array membrane was washed under high-stringency conditions after hybridization, cross hybridization among the gene family members could not be excluded since the members are more than 80% identical to each other at the nucleotide level in their coding regions. Therefore, we performed real-time RTâ€“PCR experiments with gene-specific primer sets to validate the differential expression of the PAL genes. The results of RTâ€“PCR for all PAL genes were more or less the same as those of array analysis (TableÂ  <a reveal-id="DSM014TB4" data-open="DSM014TB4" class="link link-reveal link-table xref-fig">4</a> ). In addition, we found that the <em>LjPAL5</em> gene, which was not found on the array membrane, was severely down-regulated in AM roots (TableÂ  <a reveal-id="DSM014TB4" data-open="DSM014TB4" class="link link-reveal link-table xref-fig">4</a> ). </p>                        <a id="37284021" scrollto-destination="37284021"></a>
<div content-id="DSM014TB4" class="table-modal"><div class="table-wrap"><div id="DSM014TB4" class="table-wrap-title" data-id="DSM014TB4"><div class="label -label"><strong>TableÂ 4</strong></div><div class="caption"><p><em>L. japonicus</em> genes for PALs and their expression patterns </p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene nameÂ </th><th>Gene IDÂ </th><th>Gi1/gni1Â </th><th>Gm6/SC3Â </th><th>Gm6/SC3 (RTâ€“PCR)Â </th><th>Gi8/gni3Â </th></tr></thead><tbody><tr><td><em>LjPAL1</em>Â </td><td>MRL007g11_fÂ </td><td>2.08 (283/136)Â </td><td>0.24 (107/445)Â </td><td><strong>0.35 Â± 0.17</strong>Â </td><td>0.17 (80/466)Â </td></tr><tr><td><em>LjPAL2</em>Â </td><td>GENLf058e04Â </td><td>2.20 (202/92)Â </td><td>0.26 (72/274)Â </td><td><strong>0.29 Â± 0.11</strong>Â </td><td>0.22 (84/384)Â </td></tr><tr><td><em>LjPAL3</em>Â </td><td>MR060a09_fÂ </td><td>1.70 (426/250)Â </td><td>0.15 (148/999)Â </td><td><strong>0.23 Â± 0.16</strong>Â </td><td>0.17 (186/1093)Â </td></tr><tr><td><em>LjPAL4</em>Â </td><td>GENLf025c04Â </td><td>1.49 (202/136)Â </td><td>0.16 (69/431)Â </td><td><strong>0.09 Â± 0.09</strong>Â </td><td>0.17 (80/458)Â </td></tr><tr><td><em>LjPAL5</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td><strong>0.16 Â± 0.08</strong>Â </td><td>Â </td></tr><tr><td><em>LjPAL6</em>Â </td><td>MWL047f06_rÂ </td><td>1.53 (197/129)Â </td><td>0.56 (160/286)Â </td><td><strong>0.10 Â± 0.03</strong>Â </td><td>0.33 (114/344)Â </td></tr><tr><td><em>LjPAL7</em>Â </td><td>MWL052f09_rÂ </td><td>1.54 (330/214)Â </td><td>0.77 (312/407)Â </td><td><strong>0.40 Â± 0.27</strong>Â </td><td>0.22 (190/873)Â </td></tr><tr><td><em>LjPAL8</em>Â </td><td>MWL032c01_rÂ </td><td>0.93 (165/178)Â </td><td>0.25 (67/273)Â </td><td><strong>0.09 Â± 0.04</strong>Â </td><td>0.26 (96/361)Â </td></tr><tr><td><em>LjPAL9</em>Â </td><td>MWM088g05_rÂ </td><td>1.33 (326/245)Â </td><td>0.95 (621/652)Â </td><td><strong>0.26 Â± 0.08</strong>Â </td><td>0.34 (207/612)Â </td></tr><tr><td><em>LjPAL10</em>Â </td><td>MR078c05_fÂ </td><td>1.44 (340/236)Â </td><td>1.15 (161/140)Â </td><td><strong>1.34 Â± 0.56</strong>Â </td><td>0.67 (166/248)Â </td></tr><tr><td><em>ÃŸ-actin</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td><strong>1.35 Â± 1.38</strong>Â </td><td>Â </td></tr></tbody></table></div><div content-id="DSM014TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene nameÂ </th><th>Gene IDÂ </th><th>Gi1/gni1Â </th><th>Gm6/SC3Â </th><th>Gm6/SC3 (RTâ€“PCR)Â </th><th>Gi8/gni3Â </th></tr></thead><tbody><tr><td><em>LjPAL1</em>Â </td><td>MRL007g11_fÂ </td><td>2.08 (283/136)Â </td><td>0.24 (107/445)Â </td><td><strong>0.35 Â± 0.17</strong>Â </td><td>0.17 (80/466)Â </td></tr><tr><td><em>LjPAL2</em>Â </td><td>GENLf058e04Â </td><td>2.20 (202/92)Â </td><td>0.26 (72/274)Â </td><td><strong>0.29 Â± 0.11</strong>Â </td><td>0.22 (84/384)Â </td></tr><tr><td><em>LjPAL3</em>Â </td><td>MR060a09_fÂ </td><td>1.70 (426/250)Â </td><td>0.15 (148/999)Â </td><td><strong>0.23 Â± 0.16</strong>Â </td><td>0.17 (186/1093)Â </td></tr><tr><td><em>LjPAL4</em>Â </td><td>GENLf025c04Â </td><td>1.49 (202/136)Â </td><td>0.16 (69/431)Â </td><td><strong>0.09 Â± 0.09</strong>Â </td><td>0.17 (80/458)Â </td></tr><tr><td><em>LjPAL5</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td><strong>0.16 Â± 0.08</strong>Â </td><td>Â </td></tr><tr><td><em>LjPAL6</em>Â </td><td>MWL047f06_rÂ </td><td>1.53 (197/129)Â </td><td>0.56 (160/286)Â </td><td><strong>0.10 Â± 0.03</strong>Â </td><td>0.33 (114/344)Â </td></tr><tr><td><em>LjPAL7</em>Â </td><td>MWL052f09_rÂ </td><td>1.54 (330/214)Â </td><td>0.77 (312/407)Â </td><td><strong>0.40 Â± 0.27</strong>Â </td><td>0.22 (190/873)Â </td></tr><tr><td><em>LjPAL8</em>Â </td><td>MWL032c01_rÂ </td><td>0.93 (165/178)Â </td><td>0.25 (67/273)Â </td><td><strong>0.09 Â± 0.04</strong>Â </td><td>0.26 (96/361)Â </td></tr><tr><td><em>LjPAL9</em>Â </td><td>MWM088g05_rÂ </td><td>1.33 (326/245)Â </td><td>0.95 (621/652)Â </td><td><strong>0.26 Â± 0.08</strong>Â </td><td>0.34 (207/612)Â </td></tr><tr><td><em>LjPAL10</em>Â </td><td>MR078c05_fÂ </td><td>1.44 (340/236)Â </td><td>1.15 (161/140)Â </td><td><strong>1.34 Â± 0.56</strong>Â </td><td>0.67 (166/248)Â </td></tr><tr><td><em>ÃŸ-actin</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td><strong>1.35 Â± 1.38</strong>Â </td><td>Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p> The lightface data in the Gi1/gni1, Gi8/gni3 and Gm6/SC3 columns are the fold values for gene expression in roots at 1 week and 8 weeks after inoculation of <em>G. margarita</em> , and 6 weeks after inoculation of <em>G. mosseae</em> , respectively, compared with those for control roots. The normalized expression levels observed in the array analyses are also given in parentheses. The boldface Gm6/SC3 column shows the results of real time RTâ€“PCR (mean Â± SD for four replicates) for gene expression in roots at 6 weeks after inoculation of <em>G. mosseae</em> , compared with those for control roots. </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/37284021" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM014TB4" class="table-wrap-title" data-id="DSM014TB4"><div class="label -label"><strong>TableÂ 4</strong></div><div class="caption"><p><em>L. japonicus</em> genes for PALs and their expression patterns </p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene nameÂ </th><th>Gene IDÂ </th><th>Gi1/gni1Â </th><th>Gm6/SC3Â </th><th>Gm6/SC3 (RTâ€“PCR)Â </th><th>Gi8/gni3Â </th></tr></thead><tbody><tr><td><em>LjPAL1</em>Â </td><td>MRL007g11_fÂ </td><td>2.08 (283/136)Â </td><td>0.24 (107/445)Â </td><td><strong>0.35 Â± 0.17</strong>Â </td><td>0.17 (80/466)Â </td></tr><tr><td><em>LjPAL2</em>Â </td><td>GENLf058e04Â </td><td>2.20 (202/92)Â </td><td>0.26 (72/274)Â </td><td><strong>0.29 Â± 0.11</strong>Â </td><td>0.22 (84/384)Â </td></tr><tr><td><em>LjPAL3</em>Â </td><td>MR060a09_fÂ </td><td>1.70 (426/250)Â </td><td>0.15 (148/999)Â </td><td><strong>0.23 Â± 0.16</strong>Â </td><td>0.17 (186/1093)Â </td></tr><tr><td><em>LjPAL4</em>Â </td><td>GENLf025c04Â </td><td>1.49 (202/136)Â </td><td>0.16 (69/431)Â </td><td><strong>0.09 Â± 0.09</strong>Â </td><td>0.17 (80/458)Â </td></tr><tr><td><em>LjPAL5</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td><strong>0.16 Â± 0.08</strong>Â </td><td>Â </td></tr><tr><td><em>LjPAL6</em>Â </td><td>MWL047f06_rÂ </td><td>1.53 (197/129)Â </td><td>0.56 (160/286)Â </td><td><strong>0.10 Â± 0.03</strong>Â </td><td>0.33 (114/344)Â </td></tr><tr><td><em>LjPAL7</em>Â </td><td>MWL052f09_rÂ </td><td>1.54 (330/214)Â </td><td>0.77 (312/407)Â </td><td><strong>0.40 Â± 0.27</strong>Â </td><td>0.22 (190/873)Â </td></tr><tr><td><em>LjPAL8</em>Â </td><td>MWL032c01_rÂ </td><td>0.93 (165/178)Â </td><td>0.25 (67/273)Â </td><td><strong>0.09 Â± 0.04</strong>Â </td><td>0.26 (96/361)Â </td></tr><tr><td><em>LjPAL9</em>Â </td><td>MWM088g05_rÂ </td><td>1.33 (326/245)Â </td><td>0.95 (621/652)Â </td><td><strong>0.26 Â± 0.08</strong>Â </td><td>0.34 (207/612)Â </td></tr><tr><td><em>LjPAL10</em>Â </td><td>MR078c05_fÂ </td><td>1.44 (340/236)Â </td><td>1.15 (161/140)Â </td><td><strong>1.34 Â± 0.56</strong>Â </td><td>0.67 (166/248)Â </td></tr><tr><td><em>ÃŸ-actin</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td><strong>1.35 Â± 1.38</strong>Â </td><td>Â </td></tr></tbody></table></div><div content-id="DSM014TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene nameÂ </th><th>Gene IDÂ </th><th>Gi1/gni1Â </th><th>Gm6/SC3Â </th><th>Gm6/SC3 (RTâ€“PCR)Â </th><th>Gi8/gni3Â </th></tr></thead><tbody><tr><td><em>LjPAL1</em>Â </td><td>MRL007g11_fÂ </td><td>2.08 (283/136)Â </td><td>0.24 (107/445)Â </td><td><strong>0.35 Â± 0.17</strong>Â </td><td>0.17 (80/466)Â </td></tr><tr><td><em>LjPAL2</em>Â </td><td>GENLf058e04Â </td><td>2.20 (202/92)Â </td><td>0.26 (72/274)Â </td><td><strong>0.29 Â± 0.11</strong>Â </td><td>0.22 (84/384)Â </td></tr><tr><td><em>LjPAL3</em>Â </td><td>MR060a09_fÂ </td><td>1.70 (426/250)Â </td><td>0.15 (148/999)Â </td><td><strong>0.23 Â± 0.16</strong>Â </td><td>0.17 (186/1093)Â </td></tr><tr><td><em>LjPAL4</em>Â </td><td>GENLf025c04Â </td><td>1.49 (202/136)Â </td><td>0.16 (69/431)Â </td><td><strong>0.09 Â± 0.09</strong>Â </td><td>0.17 (80/458)Â </td></tr><tr><td><em>LjPAL5</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td><strong>0.16 Â± 0.08</strong>Â </td><td>Â </td></tr><tr><td><em>LjPAL6</em>Â </td><td>MWL047f06_rÂ </td><td>1.53 (197/129)Â </td><td>0.56 (160/286)Â </td><td><strong>0.10 Â± 0.03</strong>Â </td><td>0.33 (114/344)Â </td></tr><tr><td><em>LjPAL7</em>Â </td><td>MWL052f09_rÂ </td><td>1.54 (330/214)Â </td><td>0.77 (312/407)Â </td><td><strong>0.40 Â± 0.27</strong>Â </td><td>0.22 (190/873)Â </td></tr><tr><td><em>LjPAL8</em>Â </td><td>MWL032c01_rÂ </td><td>0.93 (165/178)Â </td><td>0.25 (67/273)Â </td><td><strong>0.09 Â± 0.04</strong>Â </td><td>0.26 (96/361)Â </td></tr><tr><td><em>LjPAL9</em>Â </td><td>MWM088g05_rÂ </td><td>1.33 (326/245)Â </td><td>0.95 (621/652)Â </td><td><strong>0.26 Â± 0.08</strong>Â </td><td>0.34 (207/612)Â </td></tr><tr><td><em>LjPAL10</em>Â </td><td>MR078c05_fÂ </td><td>1.44 (340/236)Â </td><td>1.15 (161/140)Â </td><td><strong>1.34 Â± 0.56</strong>Â </td><td>0.67 (166/248)Â </td></tr><tr><td><em>ÃŸ-actin</em>Â </td><td>Â </td><td>Â </td><td>Â </td><td><strong>1.35 Â± 1.38</strong>Â </td><td>Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p> The lightface data in the Gi1/gni1, Gi8/gni3 and Gm6/SC3 columns are the fold values for gene expression in roots at 1 week and 8 weeks after inoculation of <em>G. margarita</em> , and 6 weeks after inoculation of <em>G. mosseae</em> , respectively, compared with those for control roots. The normalized expression levels observed in the array analyses are also given in parentheses. The boldface Gm6/SC3 column shows the results of real time RTâ€“PCR (mean Â± SD for four replicates) for gene expression in roots at 6 weeks after inoculation of <em>G. mosseae</em> , compared with those for control roots. </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/37284021" target="_blank">View Large</a></div></div><p> To further confirm the repression of some PAL genes after AM colonization, we searched for genomic sequences of the PAL genes in databases, finding that <em>LjPAL1</em> , <em>LjPAL4</em> , and <em>LjPAL5</em> lie in tandem on a single TAC clone, AP004502. We chose the <em>LjPAL1</em> promoter, which shows typical differential expression, for further analysis. This promoter, 2 kb in size, was amplified by PCR, fused with the <em>uidA</em> reporter for GUS, and then introduced into <em>L. japonicus</em> by the hairy root method with <em>A. rhizogenes</em> . The transformants showed basal activity, especially in central cylinders, in the absence of any microorganisms (Fig.Â  <a reveal-id="DSM014F2" data-open="DSM014F2" class="link link-reveal link-table xref-fig">2</a> A). The GUS activity was augmented throughout the roots in the presence of contaminating microorganisms in the sieved carrier (Fig.Â  <a reveal-id="DSM014F2" data-open="DSM014F2" class="link link-reveal link-table xref-fig">2</a> B). When the transformants were inoculated with the whole <em>G. mosseae</em> inoculum, the area of expression decreased (Fig.Â  <a reveal-id="DSM014F2" data-open="DSM014F2" class="link link-reveal link-table xref-fig">2</a> C). The specific GUS activity levels in the entire hairy roots of the above transformants were 1.2 Â± 0.2, 7.1 Â± 2.9, and 4.1 Â± 1.5 pmol/min/Âµg protein, respectively. Unexpectedly, when GUS-stained AM roots were re-stained with safranin, a red dye that stains fungal cells better than plant cells, it turned out that <em>G. mosseae</em> only colonized where GUS activity was low (Fig.Â  <a reveal-id="DSM014F2" data-open="DSM014F2" class="link link-reveal link-table xref-fig">2</a> C). To confirm this observation, sections of GUS-stained AM roots were prepared and then re-stained with safranin. As shown in Fig.Â  <a reveal-id="DSM014F2" data-open="DSM014F2" class="link link-reveal link-table xref-fig">2</a> D and E, the root portions exhibiting high <em>LjPAL1</em> promoter activity did not contain <em>G. mosseae</em> . In contrast, the AM fungus colonized well where the GUS level was low (Fig.Â  <a reveal-id="DSM014F2" data-open="DSM014F2" class="link link-reveal link-table xref-fig">2</a> F and G). In some cases, <em>G. mosseae</em> was observed where GUS activity was also significant, but the level of GUS was not very high either (Fig.Â  <a reveal-id="DSM014F2" data-open="DSM014F2" class="link link-reveal link-table xref-fig">2</a> H and I). As described above, the whole <em>G. mosseae</em> inoculum and the sieved carrier contained equivalent amounts of contaminating microorganisms. Therefore, host plants repress PAL gene expression where AM fungi colonize, preventing infection by pathogenic microorganisms. This repression pattern is similar to that of isoflavone reductase of <em>M. truncatula</em> previously reported, <sup><a reveal-id="DSM014C50" data-open="DSM014C50" class="link link-ref link-reveal xref-bibr">50</a></sup> but different from those of PAL and chalcone synthase observed in that study. Comprehensive expression analysis of every family member for the latter enzymes of <em>M. truncatula</em> would be necessary to resolve this discrepancy. </p>                        <a id="37284023" scrollto-destination="37284023"></a>
<div id="DSM014F2" data-id="DSM014F2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm014/3/m_dsm01402.jpeg?Expires=2147483647&Signature=Py1MSbVIE1GF6TWfLspWycl7NxL8CTzQyXvgQwZnIjWHEOdmUmnm6OLGt5kb7G9W8tHbpLua7mjemhwuz6rcxmGlyQOnpMDXq12z9y6geRR2AJBTGnGg7HQLCqsuZxrMRmEj~UpLuE0L3IBzIxw0DcEQLNckFiRG4x5Ab-dNc25zbGa041FcRp-RSIGoxNPr87q60O8u3nAPVolBibtw7ktdhhfRmx4lgp-c9alEF78E7zEUhjqvr6mNOMMzahU60CU3HWpj~jM62BOJqJRqfypAbKXYKBsGOJRXL8ZJ5ShvwkFygZSOw~1OydrzO01jjp3oc2Qww297oa7uDsbvEg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Spatial patterns of LjPAL1 promoter activity in L. japonicus hairy roots. GUS activity is visible as a blue color in intact roots &#x9;( A â€“ C ) or longitudinal sections of AM roots ( D â€“ I ). A, a non-infected root. ( B ), 4 weeks after inoculation of the sieved carrier containing contaminating microorganisms. ( C ) An AM root, 4 weeks after addition of the whole G. mosseae inoculum. The arrow indicates the region heavily colonized by the AM fungus. After regular GUS staining, the root was re-stained with safranin. Sections of GUS-stained roots were prepared ( D , F , and H ) and then stained on slide glasses with safranin ( E , G , and I ). The small arrows in G and I indicate arbuscules of the AM fungus. ( J and K ) LjPAL1 promoter activity in nodules on L. japonicus hairy roots. Two weeks after inoculation of M. loti , GUS activity in the nodulated roots was examined under a stereomicroscope. ( J ) A nodule primordium. ( K ) A more mature nodule. Bars, 100 Âµm. " path-from-xml="dsm01402.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/37284023/dsm01402.jpeg" path-from-xml="dsm01402.jpeg" target="_blank">View large</a><a section="37284023" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm014/3/dsm01402.jpeg?Expires=2147483647&Signature=dsNyp1sXKEtULCCt-JY3XNoUhm16-4gfV4C2s9gK1hG2TMmUo11pbpLdowXdXjHGBFI2~S7IgVY8Y4hOk69y89j-oGRjfBQvolG9cR4-CTNSJyDLdzvtEq0PXBtfjV0tzouJ06UyisGDNe5CXR1iU4GZqsn7bkQ~Pggus7uUHdPNEmHjSzXPxJUPS5s~3m9GhdivPGAMSMPvdN6PQrmRdsegCGmsZaYdvkL8eiuzkKwOeBVG~OEOt3yjJHstXrvDlUDciW6IKTuvEXooqzIknDvrd1yACBkB~2q7~s~gQsb4XMozdeuxl6bquIukJ9qbKy6Erx6OvbHkfCGyB9CPkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=37284023&ar=578022&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01402.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Spatial patterns of <em>LjPAL1</em> promoter activity in <em>L. japonicus</em> hairy roots. GUS activity is visible as a blue color in intact roots 	( <strong>A</strong> â€“ <strong>C</strong> ) or longitudinal sections of AM roots ( <strong>D</strong> â€“ <strong>I</strong> ). A, a non-infected root. ( <strong>B</strong> ), 4 weeks after inoculation of the sieved carrier containing contaminating microorganisms. ( <strong>C</strong> ) An AM root, 4 weeks after addition of the whole <em>G. mosseae</em> inoculum. The arrow indicates the region heavily colonized by the AM fungus. After regular GUS staining, the root was re-stained with safranin. Sections of GUS-stained roots were prepared ( <strong>D</strong> , <strong>F</strong> , and <strong>H</strong> ) and then stained on slide glasses with safranin ( <strong>E</strong> , <strong>G</strong> , and <strong>I</strong> ). The small arrows in G and I indicate arbuscules of the AM fungus. ( <strong>J</strong> and <strong>K</strong> ) <em>LjPAL1</em> promoter activity in nodules on <em>L. japonicus</em> hairy roots. Two weeks after inoculation of <em>M. loti</em> , GUS activity in the nodulated roots was examined under a stereomicroscope. ( <strong>J</strong> ) A nodule primordium. ( <strong>K</strong> ) A more mature nodule. Bars, 100 Âµm. </p></div></div><div content-id="DSM014F2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm014/3/m_dsm01402.jpeg?Expires=2147483647&Signature=Py1MSbVIE1GF6TWfLspWycl7NxL8CTzQyXvgQwZnIjWHEOdmUmnm6OLGt5kb7G9W8tHbpLua7mjemhwuz6rcxmGlyQOnpMDXq12z9y6geRR2AJBTGnGg7HQLCqsuZxrMRmEj~UpLuE0L3IBzIxw0DcEQLNckFiRG4x5Ab-dNc25zbGa041FcRp-RSIGoxNPr87q60O8u3nAPVolBibtw7ktdhhfRmx4lgp-c9alEF78E7zEUhjqvr6mNOMMzahU60CU3HWpj~jM62BOJqJRqfypAbKXYKBsGOJRXL8ZJ5ShvwkFygZSOw~1OydrzO01jjp3oc2Qww297oa7uDsbvEg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Spatial patterns of LjPAL1 promoter activity in L. japonicus hairy roots. GUS activity is visible as a blue color in intact roots &#x9;( A â€“ C ) or longitudinal sections of AM roots ( D â€“ I ). A, a non-infected root. ( B ), 4 weeks after inoculation of the sieved carrier containing contaminating microorganisms. ( C ) An AM root, 4 weeks after addition of the whole G. mosseae inoculum. The arrow indicates the region heavily colonized by the AM fungus. After regular GUS staining, the root was re-stained with safranin. Sections of GUS-stained roots were prepared ( D , F , and H ) and then stained on slide glasses with safranin ( E , G , and I ). The small arrows in G and I indicate arbuscules of the AM fungus. ( J and K ) LjPAL1 promoter activity in nodules on L. japonicus hairy roots. Two weeks after inoculation of M. loti , GUS activity in the nodulated roots was examined under a stereomicroscope. ( J ) A nodule primordium. ( K ) A more mature nodule. Bars, 100 Âµm. " path-from-xml="dsm01402.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/37284023/dsm01402.jpeg" path-from-xml="dsm01402.jpeg" target="_blank">View large</a><a section="37284023" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/3/10.1093_dnares_dsm014/3/dsm01402.jpeg?Expires=2147483647&Signature=dsNyp1sXKEtULCCt-JY3XNoUhm16-4gfV4C2s9gK1hG2TMmUo11pbpLdowXdXjHGBFI2~S7IgVY8Y4hOk69y89j-oGRjfBQvolG9cR4-CTNSJyDLdzvtEq0PXBtfjV0tzouJ06UyisGDNe5CXR1iU4GZqsn7bkQ~Pggus7uUHdPNEmHjSzXPxJUPS5s~3m9GhdivPGAMSMPvdN6PQrmRdsegCGmsZaYdvkL8eiuzkKwOeBVG~OEOt3yjJHstXrvDlUDciW6IKTuvEXooqzIknDvrd1yACBkB~2q7~s~gQsb4XMozdeuxl6bquIukJ9qbKy6Erx6OvbHkfCGyB9CPkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=37284023&ar=578022&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01402.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Spatial patterns of <em>LjPAL1</em> promoter activity in <em>L. japonicus</em> hairy roots. GUS activity is visible as a blue color in intact roots 	( <strong>A</strong> â€“ <strong>C</strong> ) or longitudinal sections of AM roots ( <strong>D</strong> â€“ <strong>I</strong> ). A, a non-infected root. ( <strong>B</strong> ), 4 weeks after inoculation of the sieved carrier containing contaminating microorganisms. ( <strong>C</strong> ) An AM root, 4 weeks after addition of the whole <em>G. mosseae</em> inoculum. The arrow indicates the region heavily colonized by the AM fungus. After regular GUS staining, the root was re-stained with safranin. Sections of GUS-stained roots were prepared ( <strong>D</strong> , <strong>F</strong> , and <strong>H</strong> ) and then stained on slide glasses with safranin ( <strong>E</strong> , <strong>G</strong> , and <strong>I</strong> ). The small arrows in G and I indicate arbuscules of the AM fungus. ( <strong>J</strong> and <strong>K</strong> ) <em>LjPAL1</em> promoter activity in nodules on <em>L. japonicus</em> hairy roots. Two weeks after inoculation of <em>M. loti</em> , GUS activity in the nodulated roots was examined under a stereomicroscope. ( <strong>J</strong> ) A nodule primordium. ( <strong>K</strong> ) A more mature nodule. Bars, 100 Âµm. </p></div></div>                            <h3 scrollto-destination=37284024 id="37284024" class="section-title" data-legacy-id=s3e>3.5. Commonly repressed genes of L. japonicus in AM roots and nitrogen-fixing nodules</h3>
<p> When the results of cDNA array experiments on AM roots with <em>G. mosseae</em> , and ones on <em>G. margarita</em> and mature root nodules with <em>M. loti</em> were compared with each other, the overlapping of induced genes or repressed genes was found to be limited (Supplementary Fig. S2), in accord with previous reports. <sup><a reveal-id="DSM014C17" data-open="DSM014C17" class="link link-ref link-reveal xref-bibr">17</a> , <a reveal-id="DSM014C18" data-open="DSM014C18" class="link link-ref link-reveal xref-bibr">18</a></sup> However, when the commonly regulated genes in <em>G. mosseae</em> -colonized roots and mature root nodules were listed up, it was obvious that many defense-related and stress-induced genes were included in the commonly repressed list (TableÂ  <a reveal-id="DSM014TB3" data-open="DSM014TB3" class="link link-reveal link-table xref-fig">3</a> ). They include genes for WRKY transcription factors, which are up-regulated in response to biotic or abiotic stress, <sup><a reveal-id="DSM014C40" data-open="DSM014C40" class="link link-ref link-reveal xref-bibr">40</a> , <a reveal-id="DSM014C56" data-open="DSM014C56" class="link link-ref link-reveal xref-bibr">56</a></sup> and those for BURP domain proteins, one of which is a stress-induced transcription factor, <sup><a reveal-id="DSM014C57" data-open="DSM014C57" class="link link-ref link-reveal xref-bibr">57</a></sup> besides PAL genes. These results suggest that host plants accept AM fungi and compatible rhizobia in similar manners, their defense mechanisms being suppressed. </p><p> Because <em>LjPAL1</em> is one of the commonly repressed genes in AM roots and nodules (TableÂ  <a reveal-id="DSM014TB5" data-open="DSM014TB5" class="link link-reveal link-table xref-fig">5</a> ), we inoculated <em>M. loti</em> into hairy roots transformed with the <em>LjPAL1</em> promoter-GUS construct. As shown in Fig.Â  <a reveal-id="DSM014F2" data-open="DSM014F2" class="link link-reveal link-table xref-fig">2</a> J and K, strong GUS activity was detected at the top of a nodule primordium, but it had soon disappeared in a slightly more mature nodule, in accordance with the results of the array experiments (TableÂ  <a reveal-id="DSM014TB5" data-open="DSM014TB5" class="link link-reveal link-table xref-fig">5</a> ). </p>                        <a id="37284027" scrollto-destination="37284027"></a>
<div content-id="DSM014TB5" class="table-modal"><div class="table-wrap"><div id="DSM014TB5" class="table-wrap-title" data-id="DSM014TB5"><div class="label -label"><strong>TableÂ 5</strong></div><div class="caption"><p> Co-regulated genes of <em>L. japonicus</em> in AM roots and nitrogen-fixing nodules </p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Current annotationÂ </th><th>Gene IDÂ </th><th>Gm6/SC3Â </th><th>NodÂ </th></tr></thead><tbody><tr><td>Beta-amylase-like proteinÂ </td><td>MWL048f05_rÂ </td><td>4.6Â </td><td>2.9Â </td></tr><tr><td><strong>Chitinase</strong>Â </td><td>MWM140d02_rÂ </td><td>4.3Â </td><td>6.6Â </td></tr><tr><td><strong>Chalcone reductase</strong>Â </td><td>GNf040a09Â </td><td>4.1Â </td><td>5.6Â </td></tr><tr><td>Asparagine synthetaseÂ </td><td>MWL032c11_rÂ </td><td>4.0Â </td><td>67.2Â </td></tr><tr><td>Asparagine synthetaseÂ </td><td>GNf053e06Â </td><td>4.0Â </td><td>34.2Â </td></tr><tr><td><strong>Chitinase</strong>Â </td><td>MWM034g12_rÂ </td><td>3.9Â </td><td>13.4Â </td></tr><tr><td>Asparagine synthetaseÂ </td><td>MWM233f05_rÂ </td><td>3.3Â </td><td>25.1Â </td></tr><tr><td>Putative PGPD14 proteinÂ </td><td>MWL059c01_rÂ </td><td>2.7Â </td><td>3.9Â </td></tr><tr><td>Sterigmatocystin biosynthesis proteinÂ </td><td>GNf018c04Â </td><td>2.6Â </td><td>3.4Â </td></tr><tr><td>Branched chain alpha-keto acid dehydrogenaseÂ </td><td>MWM092a07_rÂ </td><td>2.6Â </td><td>5.0Â </td></tr><tr><td>Seed imbibition protein, putativeÂ </td><td>MWL069f08_rÂ </td><td>2.5Â </td><td>3.4Â </td></tr><tr><td>AnnexinÂ </td><td>MPD065b05_fÂ </td><td>2.4Â </td><td>4.4Â </td></tr><tr><td>Asparagine synthetaseÂ </td><td>GNf021f11Â </td><td>2.4Â </td><td>30.2Â </td></tr><tr><td>Nodule-enhanced sucrose synthaseÂ </td><td>MWL080e04_rÂ </td><td>2.3Â </td><td>4.8Â </td></tr><tr><td>AnnexinÂ </td><td>MPD042e01_fÂ </td><td>2.2Â </td><td>6.1Â </td></tr><tr><td><strong>Phenylalanine ammonia-lyase (LjPAL5)</strong>Â </td><td>MWM056d02_rÂ </td><td>0.13Â </td><td>0.12Â </td></tr><tr><td><strong>Phenylalanine ammonia lyase (LjPAL3)</strong>Â </td><td>MR060a09_fÂ </td><td>0.15Â </td><td>0.14Â </td></tr><tr><td><strong>Phenylalanine ammonia-lyase (LjPAL4)</strong>Â </td><td>GENLf025c04Â </td><td>0.16Â </td><td>0.11Â </td></tr><tr><td>Histidine decarboxylaseÂ </td><td>GENf054a02Â </td><td>0.17Â </td><td>0.05Â </td></tr><tr><td>Naphthalene dioxygenase iron sulfur proteinÂ </td><td>MPDL068f03_fÂ </td><td>0.17Â </td><td>0.16Â </td></tr><tr><td><strong>Cytochrome P450-1</strong>Â </td><td>MR095g09_fÂ </td><td>0.17Â </td><td>0.06Â </td></tr><tr><td>Phosphate transporterÂ </td><td>MWM077d10_rÂ </td><td>0.18Â </td><td>0.25Â </td></tr><tr><td><strong>Transcription factor WRKY4</strong>Â </td><td>MWM168c07_rÂ </td><td>0.19Â </td><td>0.15Â </td></tr><tr><td>Lupeol synthaseÂ </td><td>GNf046g09Â </td><td>0.19Â </td><td>0.30Â </td></tr><tr><td>ATP synthase 9Â </td><td>MWM223c10_rÂ </td><td>0.20Â </td><td>0.14Â </td></tr><tr><td>NAD(P)H dependent 6â€²-deoxychalcone synthaseÂ </td><td>MWM174f04_rÂ </td><td>0.21Â </td><td>0.31Â </td></tr><tr><td><strong>Similar to the BURP domain</strong>Â </td><td>MPDL062c05_fÂ </td><td>0.22Â </td><td>0.13Â </td></tr><tr><td><strong>HSP100/ClpB</strong>Â </td><td>MRL022b06_fÂ </td><td>0.23Â </td><td>0.15Â </td></tr><tr><td><strong>Syringolide-induced protein B13-1-9</strong>Â </td><td>MWM033e05_rÂ </td><td>0.23Â </td><td>0.14Â </td></tr><tr><td>Histidine decarboxylase, putativeÂ </td><td>MWM231b10_rÂ </td><td>0.24Â </td><td>0.04Â </td></tr><tr><td><strong>Phenylalanine ammonia-lyase (LjPAL1)</strong>Â </td><td>MRL007g11_fÂ </td><td>0.24Â </td><td>0.17Â </td></tr><tr><td>Glycogen synthase kinase-3 homolog MsK-3Â </td><td>MWL017b06_rÂ </td><td>0.24Â </td><td>0.31Â </td></tr><tr><td><strong>Phenylalanine ammonia-lyase (LjPAL8)</strong>Â </td><td>MWL032c01_rÂ </td><td>0.25Â </td><td>0.19Â </td></tr><tr><td><strong>Seed coat BURP domain protein</strong>Â </td><td>MPDL082d06_fÂ </td><td>0.25Â </td><td>0.18Â </td></tr><tr><td>Ribonuclease non-SÂ </td><td>MWM082g02_rÂ </td><td>0.26Â </td><td>0.31Â </td></tr><tr><td>Putative zinc finger POZ proteinÂ </td><td>MWM026d08_rÂ </td><td>0.26Â </td><td>0.09Â </td></tr><tr><td><strong>WRKY transcription factor</strong>Â </td><td>GENLf072f04Â </td><td>0.26Â </td><td>0.30Â </td></tr><tr><td><strong>Phenylalanine ammonia-lyase (LjPAL2)</strong>Â </td><td>GENLf058e04Â </td><td>0.26Â </td><td>0.17Â </td></tr><tr><td>Phosphate transporterÂ </td><td>MR054e04_fÂ </td><td>0.27Â </td><td>0.27Â </td></tr><tr><td><strong>Chalcone synthase</strong>Â </td><td>MWM170f10_rÂ </td><td>0.27Â </td><td>0.30Â </td></tr><tr><td><strong>WRKY transcription factor</strong>Â </td><td>MWM029g02_rÂ </td><td>0.28Â </td><td>0.17Â </td></tr><tr><td><strong>Cytochrome P450</strong>Â </td><td>MWL061f11_rÂ </td><td>0.28Â </td><td>0.05Â </td></tr><tr><td><strong>Syringolide-induced protein 14-1-1</strong>Â </td><td>GNf002b04Â </td><td>0.29Â </td><td>0.20Â </td></tr><tr><td><strong>Cytochrome P450, putative</strong>Â </td><td>MR076b02_fÂ </td><td>0.30Â </td><td>0.06Â </td></tr><tr><td>Putative anthocyanidine rhamnosyl-transferaseÂ </td><td>GNf060a01Â </td><td>0.30Â </td><td>0.30Â </td></tr></tbody></table></div><div content-id="DSM014TB5" class="table-modal"><table><thead name="thead" align=""><tr><th>Current annotationÂ </th><th>Gene IDÂ </th><th>Gm6/SC3Â </th><th>NodÂ </th></tr></thead><tbody><tr><td>Beta-amylase-like proteinÂ </td><td>MWL048f05_rÂ </td><td>4.6Â </td><td>2.9Â </td></tr><tr><td><strong>Chitinase</strong>Â </td><td>MWM140d02_rÂ </td><td>4.3Â </td><td>6.6Â </td></tr><tr><td><strong>Chalcone reductase</strong>Â </td><td>GNf040a09Â </td><td>4.1Â </td><td>5.6Â </td></tr><tr><td>Asparagine synthetaseÂ </td><td>MWL032c11_rÂ </td><td>4.0Â </td><td>67.2Â </td></tr><tr><td>Asparagine synthetaseÂ </td><td>GNf053e06Â </td><td>4.0Â </td><td>34.2Â </td></tr><tr><td><strong>Chitinase</strong>Â </td><td>MWM034g12_rÂ </td><td>3.9Â </td><td>13.4Â </td></tr><tr><td>Asparagine synthetaseÂ </td><td>MWM233f05_rÂ </td><td>3.3Â </td><td>25.1Â </td></tr><tr><td>Putative PGPD14 proteinÂ </td><td>MWL059c01_rÂ </td><td>2.7Â </td><td>3.9Â </td></tr><tr><td>Sterigmatocystin biosynthesis proteinÂ </td><td>GNf018c04Â </td><td>2.6Â </td><td>3.4Â </td></tr><tr><td>Branched chain alpha-keto acid dehydrogenaseÂ </td><td>MWM092a07_rÂ </td><td>2.6Â </td><td>5.0Â </td></tr><tr><td>Seed imbibition protein, putativeÂ </td><td>MWL069f08_rÂ </td><td>2.5Â </td><td>3.4Â </td></tr><tr><td>AnnexinÂ </td><td>MPD065b05_fÂ </td><td>2.4Â </td><td>4.4Â </td></tr><tr><td>Asparagine synthetaseÂ </td><td>GNf021f11Â </td><td>2.4Â </td><td>30.2Â </td></tr><tr><td>Nodule-enhanced sucrose synthaseÂ </td><td>MWL080e04_rÂ </td><td>2.3Â </td><td>4.8Â </td></tr><tr><td>AnnexinÂ </td><td>MPD042e01_fÂ </td><td>2.2Â </td><td>6.1Â </td></tr><tr><td><strong>Phenylalanine ammonia-lyase (LjPAL5)</strong>Â </td><td>MWM056d02_rÂ </td><td>0.13Â </td><td>0.12Â </td></tr><tr><td><strong>Phenylalanine ammonia lyase (LjPAL3)</strong>Â </td><td>MR060a09_fÂ </td><td>0.15Â </td><td>0.14Â </td></tr><tr><td><strong>Phenylalanine ammonia-lyase (LjPAL4)</strong>Â </td><td>GENLf025c04Â </td><td>0.16Â </td><td>0.11Â </td></tr><tr><td>Histidine decarboxylaseÂ </td><td>GENf054a02Â </td><td>0.17Â </td><td>0.05Â </td></tr><tr><td>Naphthalene dioxygenase iron sulfur proteinÂ </td><td>MPDL068f03_fÂ </td><td>0.17Â </td><td>0.16Â </td></tr><tr><td><strong>Cytochrome P450-1</strong>Â </td><td>MR095g09_fÂ </td><td>0.17Â </td><td>0.06Â </td></tr><tr><td>Phosphate transporterÂ </td><td>MWM077d10_rÂ </td><td>0.18Â </td><td>0.25Â </td></tr><tr><td><strong>Transcription factor WRKY4</strong>Â </td><td>MWM168c07_rÂ </td><td>0.19Â </td><td>0.15Â </td></tr><tr><td>Lupeol synthaseÂ </td><td>GNf046g09Â </td><td>0.19Â </td><td>0.30Â </td></tr><tr><td>ATP synthase 9Â </td><td>MWM223c10_rÂ </td><td>0.20Â </td><td>0.14Â </td></tr><tr><td>NAD(P)H dependent 6â€²-deoxychalcone synthaseÂ </td><td>MWM174f04_rÂ </td><td>0.21Â </td><td>0.31Â </td></tr><tr><td><strong>Similar to the BURP domain</strong>Â </td><td>MPDL062c05_fÂ </td><td>0.22Â </td><td>0.13Â </td></tr><tr><td><strong>HSP100/ClpB</strong>Â </td><td>MRL022b06_fÂ </td><td>0.23Â </td><td>0.15Â </td></tr><tr><td><strong>Syringolide-induced protein B13-1-9</strong>Â </td><td>MWM033e05_rÂ </td><td>0.23Â </td><td>0.14Â </td></tr><tr><td>Histidine decarboxylase, putativeÂ </td><td>MWM231b10_rÂ </td><td>0.24Â </td><td>0.04Â </td></tr><tr><td><strong>Phenylalanine ammonia-lyase (LjPAL1)</strong>Â </td><td>MRL007g11_fÂ </td><td>0.24Â </td><td>0.17Â </td></tr><tr><td>Glycogen synthase kinase-3 homolog MsK-3Â </td><td>MWL017b06_rÂ </td><td>0.24Â </td><td>0.31Â </td></tr><tr><td><strong>Phenylalanine ammonia-lyase (LjPAL8)</strong>Â </td><td>MWL032c01_rÂ </td><td>0.25Â </td><td>0.19Â </td></tr><tr><td><strong>Seed coat BURP domain protein</strong>Â </td><td>MPDL082d06_fÂ </td><td>0.25Â </td><td>0.18Â </td></tr><tr><td>Ribonuclease non-SÂ </td><td>MWM082g02_rÂ </td><td>0.26Â </td><td>0.31Â </td></tr><tr><td>Putative zinc finger POZ proteinÂ </td><td>MWM026d08_rÂ </td><td>0.26Â </td><td>0.09Â </td></tr><tr><td><strong>WRKY transcription factor</strong>Â </td><td>GENLf072f04Â </td><td>0.26Â </td><td>0.30Â </td></tr><tr><td><strong>Phenylalanine ammonia-lyase (LjPAL2)</strong>Â </td><td>GENLf058e04Â </td><td>0.26Â </td><td>0.17Â </td></tr><tr><td>Phosphate transporterÂ </td><td>MR054e04_fÂ </td><td>0.27Â </td><td>0.27Â </td></tr><tr><td><strong>Chalcone synthase</strong>Â </td><td>MWM170f10_rÂ </td><td>0.27Â </td><td>0.30Â </td></tr><tr><td><strong>WRKY transcription factor</strong>Â </td><td>MWM029g02_rÂ </td><td>0.28Â </td><td>0.17Â </td></tr><tr><td><strong>Cytochrome P450</strong>Â </td><td>MWL061f11_rÂ </td><td>0.28Â </td><td>0.05Â </td></tr><tr><td><strong>Syringolide-induced protein 14-1-1</strong>Â </td><td>GNf002b04Â </td><td>0.29Â </td><td>0.20Â </td></tr><tr><td><strong>Cytochrome P450, putative</strong>Â </td><td>MR076b02_fÂ </td><td>0.30Â </td><td>0.06Â </td></tr><tr><td>Putative anthocyanidine rhamnosyl-transferaseÂ </td><td>GNf060a01Â </td><td>0.30Â </td><td>0.30Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p> The gene expression levels in roots 6 weeks after inoculation with the whole inoculum of <em>G. mosseae</em> relative to those with the sieved carrier (Gm6/SC3) and those in mature nodules (4 weeks after inoculation of <em>M. loti</em> ) relative to in non-infected roots (Nod) were compared. Genes of which the expression levels relative to controls were &gt; 2.2 or &lt; 0.30 are listed up. Defense-related or stress-induced genes are highlighted in bold. Genes that match hypothetical proteins of unknown function and ones that encode proteins with no homology to thus far known ones have been omitted from this table. </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/37284027" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM014TB5" class="table-wrap-title" data-id="DSM014TB5"><div class="label -label"><strong>TableÂ 5</strong></div><div class="caption"><p> Co-regulated genes of <em>L. japonicus</em> in AM roots and nitrogen-fixing nodules </p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Current annotationÂ </th><th>Gene IDÂ </th><th>Gm6/SC3Â </th><th>NodÂ </th></tr></thead><tbody><tr><td>Beta-amylase-like proteinÂ </td><td>MWL048f05_rÂ </td><td>4.6Â </td><td>2.9Â </td></tr><tr><td><strong>Chitinase</strong>Â </td><td>MWM140d02_rÂ </td><td>4.3Â </td><td>6.6Â </td></tr><tr><td><strong>Chalcone reductase</strong>Â </td><td>GNf040a09Â </td><td>4.1Â </td><td>5.6Â </td></tr><tr><td>Asparagine synthetaseÂ </td><td>MWL032c11_rÂ </td><td>4.0Â </td><td>67.2Â </td></tr><tr><td>Asparagine synthetaseÂ </td><td>GNf053e06Â </td><td>4.0Â </td><td>34.2Â </td></tr><tr><td><strong>Chitinase</strong>Â </td><td>MWM034g12_rÂ </td><td>3.9Â </td><td>13.4Â </td></tr><tr><td>Asparagine synthetaseÂ </td><td>MWM233f05_rÂ </td><td>3.3Â </td><td>25.1Â </td></tr><tr><td>Putative PGPD14 proteinÂ </td><td>MWL059c01_rÂ </td><td>2.7Â </td><td>3.9Â </td></tr><tr><td>Sterigmatocystin biosynthesis proteinÂ </td><td>GNf018c04Â </td><td>2.6Â </td><td>3.4Â </td></tr><tr><td>Branched chain alpha-keto acid dehydrogenaseÂ </td><td>MWM092a07_rÂ </td><td>2.6Â </td><td>5.0Â </td></tr><tr><td>Seed imbibition protein, putativeÂ </td><td>MWL069f08_rÂ </td><td>2.5Â </td><td>3.4Â </td></tr><tr><td>AnnexinÂ </td><td>MPD065b05_fÂ </td><td>2.4Â </td><td>4.4Â </td></tr><tr><td>Asparagine synthetaseÂ </td><td>GNf021f11Â </td><td>2.4Â </td><td>30.2Â </td></tr><tr><td>Nodule-enhanced sucrose synthaseÂ </td><td>MWL080e04_rÂ </td><td>2.3Â </td><td>4.8Â </td></tr><tr><td>AnnexinÂ </td><td>MPD042e01_fÂ </td><td>2.2Â </td><td>6.1Â </td></tr><tr><td><strong>Phenylalanine ammonia-lyase (LjPAL5)</strong>Â </td><td>MWM056d02_rÂ </td><td>0.13Â </td><td>0.12Â </td></tr><tr><td><strong>Phenylalanine ammonia lyase (LjPAL3)</strong>Â </td><td>MR060a09_fÂ </td><td>0.15Â </td><td>0.14Â </td></tr><tr><td><strong>Phenylalanine ammonia-lyase (LjPAL4)</strong>Â </td><td>GENLf025c04Â </td><td>0.16Â </td><td>0.11Â </td></tr><tr><td>Histidine decarboxylaseÂ </td><td>GENf054a02Â </td><td>0.17Â </td><td>0.05Â </td></tr><tr><td>Naphthalene dioxygenase iron sulfur proteinÂ </td><td>MPDL068f03_fÂ </td><td>0.17Â </td><td>0.16Â </td></tr><tr><td><strong>Cytochrome P450-1</strong>Â </td><td>MR095g09_fÂ </td><td>0.17Â </td><td>0.06Â </td></tr><tr><td>Phosphate transporterÂ </td><td>MWM077d10_rÂ </td><td>0.18Â </td><td>0.25Â </td></tr><tr><td><strong>Transcription factor WRKY4</strong>Â </td><td>MWM168c07_rÂ </td><td>0.19Â </td><td>0.15Â </td></tr><tr><td>Lupeol synthaseÂ </td><td>GNf046g09Â </td><td>0.19Â </td><td>0.30Â </td></tr><tr><td>ATP synthase 9Â </td><td>MWM223c10_rÂ </td><td>0.20Â </td><td>0.14Â </td></tr><tr><td>NAD(P)H dependent 6â€²-deoxychalcone synthaseÂ </td><td>MWM174f04_rÂ </td><td>0.21Â </td><td>0.31Â </td></tr><tr><td><strong>Similar to the BURP domain</strong>Â </td><td>MPDL062c05_fÂ </td><td>0.22Â </td><td>0.13Â </td></tr><tr><td><strong>HSP100/ClpB</strong>Â </td><td>MRL022b06_fÂ </td><td>0.23Â </td><td>0.15Â </td></tr><tr><td><strong>Syringolide-induced protein B13-1-9</strong>Â </td><td>MWM033e05_rÂ </td><td>0.23Â </td><td>0.14Â </td></tr><tr><td>Histidine decarboxylase, putativeÂ </td><td>MWM231b10_rÂ </td><td>0.24Â </td><td>0.04Â </td></tr><tr><td><strong>Phenylalanine ammonia-lyase (LjPAL1)</strong>Â </td><td>MRL007g11_fÂ </td><td>0.24Â </td><td>0.17Â </td></tr><tr><td>Glycogen synthase kinase-3 homolog MsK-3Â </td><td>MWL017b06_rÂ </td><td>0.24Â </td><td>0.31Â </td></tr><tr><td><strong>Phenylalanine ammonia-lyase (LjPAL8)</strong>Â </td><td>MWL032c01_rÂ </td><td>0.25Â </td><td>0.19Â </td></tr><tr><td><strong>Seed coat BURP domain protein</strong>Â </td><td>MPDL082d06_fÂ </td><td>0.25Â </td><td>0.18Â </td></tr><tr><td>Ribonuclease non-SÂ </td><td>MWM082g02_rÂ </td><td>0.26Â </td><td>0.31Â </td></tr><tr><td>Putative zinc finger POZ proteinÂ </td><td>MWM026d08_rÂ </td><td>0.26Â </td><td>0.09Â </td></tr><tr><td><strong>WRKY transcription factor</strong>Â </td><td>GENLf072f04Â </td><td>0.26Â </td><td>0.30Â </td></tr><tr><td><strong>Phenylalanine ammonia-lyase (LjPAL2)</strong>Â </td><td>GENLf058e04Â </td><td>0.26Â </td><td>0.17Â </td></tr><tr><td>Phosphate transporterÂ </td><td>MR054e04_fÂ </td><td>0.27Â </td><td>0.27Â </td></tr><tr><td><strong>Chalcone synthase</strong>Â </td><td>MWM170f10_rÂ </td><td>0.27Â </td><td>0.30Â </td></tr><tr><td><strong>WRKY transcription factor</strong>Â </td><td>MWM029g02_rÂ </td><td>0.28Â </td><td>0.17Â </td></tr><tr><td><strong>Cytochrome P450</strong>Â </td><td>MWL061f11_rÂ </td><td>0.28Â </td><td>0.05Â </td></tr><tr><td><strong>Syringolide-induced protein 14-1-1</strong>Â </td><td>GNf002b04Â </td><td>0.29Â </td><td>0.20Â </td></tr><tr><td><strong>Cytochrome P450, putative</strong>Â </td><td>MR076b02_fÂ </td><td>0.30Â </td><td>0.06Â </td></tr><tr><td>Putative anthocyanidine rhamnosyl-transferaseÂ </td><td>GNf060a01Â </td><td>0.30Â </td><td>0.30Â </td></tr></tbody></table></div><div content-id="DSM014TB5" class="table-modal"><table><thead name="thead" align=""><tr><th>Current annotationÂ </th><th>Gene IDÂ </th><th>Gm6/SC3Â </th><th>NodÂ </th></tr></thead><tbody><tr><td>Beta-amylase-like proteinÂ </td><td>MWL048f05_rÂ </td><td>4.6Â </td><td>2.9Â </td></tr><tr><td><strong>Chitinase</strong>Â </td><td>MWM140d02_rÂ </td><td>4.3Â </td><td>6.6Â </td></tr><tr><td><strong>Chalcone reductase</strong>Â </td><td>GNf040a09Â </td><td>4.1Â </td><td>5.6Â </td></tr><tr><td>Asparagine synthetaseÂ </td><td>MWL032c11_rÂ </td><td>4.0Â </td><td>67.2Â </td></tr><tr><td>Asparagine synthetaseÂ </td><td>GNf053e06Â </td><td>4.0Â </td><td>34.2Â </td></tr><tr><td><strong>Chitinase</strong>Â </td><td>MWM034g12_rÂ </td><td>3.9Â </td><td>13.4Â </td></tr><tr><td>Asparagine synthetaseÂ </td><td>MWM233f05_rÂ </td><td>3.3Â </td><td>25.1Â </td></tr><tr><td>Putative PGPD14 proteinÂ </td><td>MWL059c01_rÂ </td><td>2.7Â </td><td>3.9Â </td></tr><tr><td>Sterigmatocystin biosynthesis proteinÂ </td><td>GNf018c04Â </td><td>2.6Â </td><td>3.4Â </td></tr><tr><td>Branched chain alpha-keto acid dehydrogenaseÂ </td><td>MWM092a07_rÂ </td><td>2.6Â </td><td>5.0Â </td></tr><tr><td>Seed imbibition protein, putativeÂ </td><td>MWL069f08_rÂ </td><td>2.5Â </td><td>3.4Â </td></tr><tr><td>AnnexinÂ </td><td>MPD065b05_fÂ </td><td>2.4Â </td><td>4.4Â </td></tr><tr><td>Asparagine synthetaseÂ </td><td>GNf021f11Â </td><td>2.4Â </td><td>30.2Â </td></tr><tr><td>Nodule-enhanced sucrose synthaseÂ </td><td>MWL080e04_rÂ </td><td>2.3Â </td><td>4.8Â </td></tr><tr><td>AnnexinÂ </td><td>MPD042e01_fÂ </td><td>2.2Â </td><td>6.1Â </td></tr><tr><td><strong>Phenylalanine ammonia-lyase (LjPAL5)</strong>Â </td><td>MWM056d02_rÂ </td><td>0.13Â </td><td>0.12Â </td></tr><tr><td><strong>Phenylalanine ammonia lyase (LjPAL3)</strong>Â </td><td>MR060a09_fÂ </td><td>0.15Â </td><td>0.14Â </td></tr><tr><td><strong>Phenylalanine ammonia-lyase (LjPAL4)</strong>Â </td><td>GENLf025c04Â </td><td>0.16Â </td><td>0.11Â </td></tr><tr><td>Histidine decarboxylaseÂ </td><td>GENf054a02Â </td><td>0.17Â </td><td>0.05Â </td></tr><tr><td>Naphthalene dioxygenase iron sulfur proteinÂ </td><td>MPDL068f03_fÂ </td><td>0.17Â </td><td>0.16Â </td></tr><tr><td><strong>Cytochrome P450-1</strong>Â </td><td>MR095g09_fÂ </td><td>0.17Â </td><td>0.06Â </td></tr><tr><td>Phosphate transporterÂ </td><td>MWM077d10_rÂ </td><td>0.18Â </td><td>0.25Â </td></tr><tr><td><strong>Transcription factor WRKY4</strong>Â </td><td>MWM168c07_rÂ </td><td>0.19Â </td><td>0.15Â </td></tr><tr><td>Lupeol synthaseÂ </td><td>GNf046g09Â </td><td>0.19Â </td><td>0.30Â </td></tr><tr><td>ATP synthase 9Â </td><td>MWM223c10_rÂ </td><td>0.20Â </td><td>0.14Â </td></tr><tr><td>NAD(P)H dependent 6â€²-deoxychalcone synthaseÂ </td><td>MWM174f04_rÂ </td><td>0.21Â </td><td>0.31Â </td></tr><tr><td><strong>Similar to the BURP domain</strong>Â </td><td>MPDL062c05_fÂ </td><td>0.22Â </td><td>0.13Â </td></tr><tr><td><strong>HSP100/ClpB</strong>Â </td><td>MRL022b06_fÂ </td><td>0.23Â </td><td>0.15Â </td></tr><tr><td><strong>Syringolide-induced protein B13-1-9</strong>Â </td><td>MWM033e05_rÂ </td><td>0.23Â </td><td>0.14Â </td></tr><tr><td>Histidine decarboxylase, putativeÂ </td><td>MWM231b10_rÂ </td><td>0.24Â </td><td>0.04Â </td></tr><tr><td><strong>Phenylalanine ammonia-lyase (LjPAL1)</strong>Â </td><td>MRL007g11_fÂ </td><td>0.24Â </td><td>0.17Â </td></tr><tr><td>Glycogen synthase kinase-3 homolog MsK-3Â </td><td>MWL017b06_rÂ </td><td>0.24Â </td><td>0.31Â </td></tr><tr><td><strong>Phenylalanine ammonia-lyase (LjPAL8)</strong>Â </td><td>MWL032c01_rÂ </td><td>0.25Â </td><td>0.19Â </td></tr><tr><td><strong>Seed coat BURP domain protein</strong>Â </td><td>MPDL082d06_fÂ </td><td>0.25Â </td><td>0.18Â </td></tr><tr><td>Ribonuclease non-SÂ </td><td>MWM082g02_rÂ </td><td>0.26Â </td><td>0.31Â </td></tr><tr><td>Putative zinc finger POZ proteinÂ </td><td>MWM026d08_rÂ </td><td>0.26Â </td><td>0.09Â </td></tr><tr><td><strong>WRKY transcription factor</strong>Â </td><td>GENLf072f04Â </td><td>0.26Â </td><td>0.30Â </td></tr><tr><td><strong>Phenylalanine ammonia-lyase (LjPAL2)</strong>Â </td><td>GENLf058e04Â </td><td>0.26Â </td><td>0.17Â </td></tr><tr><td>Phosphate transporterÂ </td><td>MR054e04_fÂ </td><td>0.27Â </td><td>0.27Â </td></tr><tr><td><strong>Chalcone synthase</strong>Â </td><td>MWM170f10_rÂ </td><td>0.27Â </td><td>0.30Â </td></tr><tr><td><strong>WRKY transcription factor</strong>Â </td><td>MWM029g02_rÂ </td><td>0.28Â </td><td>0.17Â </td></tr><tr><td><strong>Cytochrome P450</strong>Â </td><td>MWL061f11_rÂ </td><td>0.28Â </td><td>0.05Â </td></tr><tr><td><strong>Syringolide-induced protein 14-1-1</strong>Â </td><td>GNf002b04Â </td><td>0.29Â </td><td>0.20Â </td></tr><tr><td><strong>Cytochrome P450, putative</strong>Â </td><td>MR076b02_fÂ </td><td>0.30Â </td><td>0.06Â </td></tr><tr><td>Putative anthocyanidine rhamnosyl-transferaseÂ </td><td>GNf060a01Â </td><td>0.30Â </td><td>0.30Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p> The gene expression levels in roots 6 weeks after inoculation with the whole inoculum of <em>G. mosseae</em> relative to those with the sieved carrier (Gm6/SC3) and those in mature nodules (4 weeks after inoculation of <em>M. loti</em> ) relative to in non-infected roots (Nod) were compared. Genes of which the expression levels relative to controls were &gt; 2.2 or &lt; 0.30 are listed up. Defense-related or stress-induced genes are highlighted in bold. Genes that match hypothetical proteins of unknown function and ones that encode proteins with no homology to thus far known ones have been omitted from this table. </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/37284027" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=37284028 id="37284028" class="section-title" data-legacy-id=s3f>3.6. Concluding remarks</h3>
<p> We performed comprehensive transcriptome analysis and spatial examination of gene expression in AM roots and root nodules of <em>L. japonicus</em> , taking into account the effects of contaminating microorganisms. We found that several cysteine protease genes were specifically induced in arbuscule-containing cells of AM roots. Moreover, we also found that PAL and other phenylpropanoid biosynthesis-related genes were moderately induced on the initial infection of the symbionts and then repressed concomitant with the establishment of the two symbioses. Characteristic expression patterns were observed both in the absence of contaminating microorganisms (TableÂ  <a reveal-id="DSM014TB4" data-open="DSM014TB4" class="link link-reveal link-table xref-fig">4</a> , experiments with <em>G. margarita</em> ; Fig.Â  <a reveal-id="DSM014F2" data-open="DSM014F2" class="link link-reveal link-table xref-fig">2</a> J and K) and more drastically in their presence (TableÂ  <a reveal-id="DSM014TB4" data-open="DSM014TB4" class="link link-reveal link-table xref-fig">4</a> , experiments with <em>G. mosseae</em> ; Fig.Â  <a reveal-id="DSM014F2" data-open="DSM014F2" class="link link-reveal link-table xref-fig">2</a> Aâ€“I). So far, it has been suggested that defense genes for AM fungi or rhizobia are initially up-regulated and then down-regulated. <sup><a reveal-id="DSM014C16" data-open="DSM014C16" class="link link-ref link-reveal xref-bibr">16</a> , <a reveal-id="DSM014C26" data-open="DSM014C26" class="link link-ref link-reveal xref-bibr">26</a> , <a reveal-id="DSM014C47" data-open="DSM014C47" class="link link-ref link-reveal xref-bibr">47</a> â€“ <a reveal-id="DSM014C52" data-open="DSM014C52" class="link link-ref link-reveal xref-bibr">52</a> , <a reveal-id="DSM014C58" data-open="DSM014C58" class="link link-ref link-reveal xref-bibr">58</a></sup> Nevertheless, the current study is unexpectedly the first demonstration that this prediction is correct especially for AM root formation with <em>G. mosseae</em> and <em>G. margarita</em> using a large scale cDNA array. Then, why did previous works on AM roots not reveal the unique expression patterns of PAL and other phenylpropanoid biosynthesis-related genes? When the expression levels of these genes in roots with commercial inoculants of AM fungi applied were examined, <sup><a reveal-id="DSM014C12" data-open="DSM014C12" class="link link-ref link-reveal xref-bibr">12</a> â€“ <a reveal-id="DSM014C15" data-open="DSM014C15" class="link link-ref link-reveal xref-bibr">15</a> , <a reveal-id="DSM014C17" data-open="DSM014C17" class="link link-ref link-reveal xref-bibr">17</a> â€“ <a reveal-id="DSM014C19" data-open="DSM014C19" class="link link-ref link-reveal xref-bibr">19</a></sup> it is possible that their induction by contaminating microorganisms and their repression by AM fungus colonization were super-imposed, resulting in comparable levels to those in sterile non-infected roots. Actually, when we did a similar experiment, <sup><a reveal-id="DSM014C12" data-open="DSM014C12" class="link link-ref link-reveal xref-bibr">12</a> â€“ <a reveal-id="DSM014C15" data-open="DSM014C15" class="link link-ref link-reveal xref-bibr">15</a> , <a reveal-id="DSM014C17" data-open="DSM014C17" class="link link-ref link-reveal xref-bibr">17</a> â€“ <a reveal-id="DSM014C19" data-open="DSM014C19" class="link link-ref link-reveal xref-bibr">19</a></sup> we did not detect the differential expression of most PAL genes except <em>LjPAL10</em> , which was moderately up-regulated (Supplementary Table S1). Other previous works in which aseptic spores of AM fungi were inoculated did not show significant down-regulation of these phenylpropanoid biosynthesis-related genes, either. <sup><a reveal-id="DSM014C16" data-open="DSM014C16" class="link link-ref link-reveal xref-bibr">16</a> , <a reveal-id="DSM014C20" data-open="DSM014C20" class="link link-ref link-reveal xref-bibr">20</a> , <a reveal-id="DSM014C41" data-open="DSM014C41" class="link link-ref link-reveal xref-bibr">41</a></sup> On the other hand, our experiments involving NaClO-treated <em>G. margarita</em> spores revealed repression of the genes. It is difficult at present to fully explain this discrepancy. As revealed in this work, however, the varying microbial population around AM roots significantly affects gene expression and hence the reproducibility of the experiments. If our surface-sterilization of the spores was not complete, for example, the differential expression of plant genes on <em>G. margarita</em> colonization might be similar to that on application of a commercial <em>G. mosseae</em> inoculant. </p><p> The presence of contaminating microorganisms is, in a sense, closer to natural field conditions than the inoculation of aceptical spores of AM fungi into sterile plants. The spatial investigation in this study revealed that a PAL gene, <em>LjPAL1</em> , is repressed where AM fungi colonized. Although PALs are multi-functional enzymes, we consider that the defense response including <em>de novo</em> synthesis of flavonoid phytoalexins against other microorganisms than AM fungi is suppressed. In nature, host plants may accept microsymbionts by suppressing their defense reactions to a minimum level at which they may still prevent infection by pathogens. </p>                            <h2 scrollto-destination=37284031 id="37284031" class="backacknowledgements-title" >Acknowledgements</h2>
<p> We wish to thank Drs. K. Akiyama, Osaka Prefecture University, and Y. Tamada, Kyoto University, for the valuable discussions. We also thank Drs. C. L. Diaz-Argueta (Leiden University), K. Nagashima (Idemitsu Kosan Co.), and K. Nakamori (Kyoto University) for providing the protocol for hairy root transformation with <em>A. rhizogenes</em> LBA1334, the kind gift of the <em>G. mosseae</em> inoculum, and the instructions for embedding AM roots in plastic resin, respectively. This work was supported in part by an AIST Research Grant and Special Coordination Funds for Promoting Science and Technology from the Ministry of Education, Culture, Sports, Science and Technology of the Japanese Government. The array experiments were carried out at the Radioisotope Research Center, Kyoto University. </p>                            <h2 scrollto-destination=37284033 id="37284033" class="backsection-title" data-legacy-id=s4>Supplementary Data</h2>
<p> Supplementary data are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org">www.dnaresearch.oxfordjournals.org</a></p>                            <h2 scrollto-destination=37284035 id="37284035" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSM014C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schussler</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Schwarzott</div> <div class="given-names">D.</div></div>,Â  <div class="name"><div class="surname">Walker</div> <div class="given-names">C</div></div>.Â </div><div class="article-title"> A new fungal phylum, the <em>Glomeromycota</em> : phylogeny and evolution </div>,Â <div class="source">Mycol. Res.</div>Â ,Â <div class="year">2001</div>,Â vol.Â <div class="volume">105</div>Â (pg.Â <div class="fpage">1413</div>-<div class="lpage">1421</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20fungal%20phylum%2C%20the%20Glomeromycota%20%3A%20phylogeny%20and%20evolution&amp;author=A.%20Schussler&amp;author=D.%20Schwarzott&amp;author=C%20Walker&amp;publication_year=2001&amp;journal=Mycol.%20Res.&amp;volume=105&amp;pages=1413-1421" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1017/S0953756201005196" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1017%2FS0953756201005196" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1017%2FS0953756201005196"> </span></div>Â </div></div></div></div></div><div content-id="DSM014C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Smith</div> <div class="given-names">S. E.</div></div>,Â  <div class="name"><div class="surname">Read</div> <div class="given-names">D. J</div></div>.Â </div>,Â <div class="source">Mycorrhizal Symbiosis</div>Â ,Â <div class="year">1997</div><div class="edition">2nd Ed.</div><div class="publisher-loc">San Diego</div><div class="publisher-name">Academic Press</div></div></div></div></div><div content-id="DSM014C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Harrison</div> <div class="given-names">M. J</div></div>.Â </div><div class="article-title">Signaling in the arbuscular mycorrhizal symbiosis</div>,Â <div class="source">Annu. Rev. Microbiol.</div>Â ,Â <div class="year">2005</div>,Â vol.Â <div class="volume">59</div>Â (pg.Â <div class="fpage">19</div>-<div class="lpage">42</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Signaling%20in%20the%20arbuscular%20mycorrhizal%20symbiosis&amp;author=M.%20J%20Harrison&amp;publication_year=2005&amp;journal=Annu.%20Rev.%20Microbiol.&amp;volume=59&amp;pages=19-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.micro.58.030603.123749" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.micro.58.030603.123749" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.micro.58.030603.123749"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16153162" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brundrett</div> <div class="given-names">M</div></div>.Â </div><div class="article-title">Mycorrhizas in natural ecosystem</div>,Â <div class="source">Adv. Ecol. Res.</div>Â ,Â <div class="year">1991</div>,Â vol.Â <div class="volume">21</div>Â (pg.Â <div class="fpage">171</div>-<div class="lpage">313</div>)</div></div></div></div><div content-id="DSM014C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Marschner</div> <div class="given-names">H</div></div>.Â </div>,Â <div class="source">Mineral Nutrition of Higher Plants</div>Â ,Â <div class="year">1995</div><div class="edition">2nd Ed.</div><div class="publisher-loc">San Diego</div><div class="publisher-name">Academic Press</div>(pg.Â <div class="fpage">566</div>-<div class="lpage">595</div>)</div></div></div></div><div content-id="DSM014C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Glaninazzi-Pearson</div> <div class="given-names">V</div></div>.Â </div><div class="article-title">Plant cell responses to arbuscular mycorrhizal fungi: getting to the roots of the symbiosis</div>,Â <div class="source">Plant Cell</div>Â ,Â <div class="year">1996</div>,Â vol.Â <div class="volume">8</div>Â (pg.Â <div class="fpage">1871</div>-<div class="lpage">1883</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20cell%20responses%20to%20arbuscular%20mycorrhizal%20fungi%3A%20getting%20to%20the%20roots%20of%20the%20symbiosis&amp;author=V%20Glaninazzi-Pearson&amp;publication_year=1996&amp;journal=Plant%20Cell&amp;volume=8&amp;pages=1871-1883" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.8.10.1871" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.8.10.1871" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.8.10.1871"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12239368" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pirozynski</div> <div class="given-names">K. A.</div></div>,Â  <div class="name"><div class="surname">Malloch</div> <div class="given-names">D. W</div></div>.Â </div><div class="article-title">The origin of land plants: a matter of mycotrophism</div>,Â <div class="source">Biosystems</div>Â ,Â <div class="year">1975</div>,Â vol.Â <div class="volume">6</div>Â (pg.Â <div class="fpage">153</div>-<div class="lpage">164</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20origin%20of%20land%20plants%3A%20a%20matter%20of%20mycotrophism&amp;author=K.%20A.%20Pirozynski&amp;author=D.%20W%20Malloch&amp;publication_year=1975&amp;journal=Biosystems&amp;volume=6&amp;pages=153-164" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0303-2647(75)90023-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0303-2647(75)90023-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0303-2647(75)90023-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1120179" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Remy</div> <div class="given-names">W.</div></div>,Â  <div class="name"><div class="surname">Taylor</div> <div class="given-names">T. N.</div></div>,Â  <div class="name"><div class="surname">Hass</div> <div class="given-names">H.</div></div>,Â  <div class="name"><div class="surname">Kerp</div> <div class="given-names">H</div></div>.Â </div><div class="article-title">Four hundred-million-year-old vesicular arbuscular mycorrhizae</div>,Â <div class="source">Proc. Natl. Acad. Sci. USA</div>Â ,Â <div class="year">1994</div>,Â vol.Â <div class="volume">91</div>Â (pg.Â <div class="fpage">11841</div>-<div class="lpage">11843</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Four%20hundred-million-year-old%20vesicular%20arbuscular%20mycorrhizae&amp;author=W.%20Remy&amp;author=T.%20N.%20Taylor&amp;author=H.%20Hass&amp;author=H%20Kerp&amp;publication_year=1994&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=91&amp;pages=11841-11843" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.91.25.11841" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.91.25.11841" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.91.25.11841"> </span></div>Â </div></div></div></div></div><div content-id="DSM014C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Redeker</div> <div class="given-names">D.</div></div>,Â  <div class="name"><div class="surname">Kodner</div> <div class="given-names">R.</div></div>,Â  <div class="name"><div class="surname">Graham</div> <div class="given-names">L</div></div>.Â </div><div class="article-title">Glomalean fungi from the Ordovician</div>,Â <div class="source">Science</div>Â ,Â <div class="year">2000</div>,Â vol.Â <div class="volume">289</div>Â (pg.Â <div class="fpage">1920</div>-<div class="lpage">1921</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Glomalean%20fungi%20from%20the%20Ordovician&amp;author=D.%20Redeker&amp;author=R.%20Kodner&amp;author=L%20Graham&amp;publication_year=2000&amp;journal=Science&amp;volume=289&amp;pages=1920-1921" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.289.5486.1920" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.289.5486.1920" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.289.5486.1920"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10988069" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kistner</div> <div class="given-names">C.</div></div>,Â  <div class="name"><div class="surname">Winzer</div> <div class="given-names">T.</div></div>,Â  <div class="name"><div class="surname">Pitzschke</div> <div class="given-names">A.</div></div>,Â et al.Â </div><div class="article-title"> Seven <em>Lotus japonicus</em> genes required for transcriptional reprogramming of the root during fungal and bacterial symbiosis </div>,Â <div class="source">Plant Cell</div>Â ,Â <div class="year">2005</div>,Â vol.Â <div class="volume">17</div>Â (pg.Â <div class="fpage">2217</div>-<div class="lpage">2229</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Seven%20Lotus%20japonicus%20genes%20required%20for%20transcriptional%20reprogramming%20of%20the%20root%20during%20fungal%20and%20bacterial%20symbiosis&amp;author=C.%20Kistner&amp;author=T.%20Winzer&amp;author=A.%20Pitzschke&amp;publication_year=2005&amp;journal=Plant%20Cell&amp;volume=17&amp;pages=2217-2229" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.105.032714" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.105.032714" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.105.032714"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15980262" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kistner</div> <div class="given-names">C.</div></div>,Â  <div class="name"><div class="surname">Parniske</div> <div class="given-names">M</div></div>.Â </div><div class="article-title">Evolution of signal transduction in intracellular symbiosis</div>,Â <div class="source">Trends Plant Sci.</div>Â ,Â <div class="year">2002</div>,Â vol.Â <div class="volume">7</div>Â (pg.Â <div class="fpage">511</div>-<div class="lpage">518</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20signal%20transduction%20in%20intracellular%20symbiosis&amp;author=C.%20Kistner&amp;author=M%20Parniske&amp;publication_year=2002&amp;journal=Trends%20Plant%20Sci.&amp;volume=7&amp;pages=511-518" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1360-1385(02)02356-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1360-1385(02)02356-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1360-1385(02)02356-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12417152" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Journet</div> <div class="given-names">E. P.</div></div>,Â  <div class="name"><div class="surname">van Tuinen</div> <div class="given-names">D.</div></div>,Â  <div class="name"><div class="surname">Gouzy</div> <div class="given-names">J.</div></div>,Â et al.Â </div><div class="article-title"> Exploring root symbiotic programs in the model legume <em>Medicago truncatula</em> using EST analysis </div>,Â <div class="source">Nucleic Acids Res.</div>Â ,Â <div class="year">2002</div>,Â vol.Â <div class="volume">30</div>Â (pg.Â <div class="fpage">5579</div>-<div class="lpage">5592</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exploring%20root%20symbiotic%20programs%20in%20the%20model%20legume%20Medicago%20truncatula%20using%20EST%20analysis&amp;author=E.%20P.%20Journet&amp;author=D.%20van%20Tuinen&amp;author=J.%20Gouzy&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res.&amp;volume=30&amp;pages=5579-5592" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkf685" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkf685" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkf685"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12490726" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wulf</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Manthey</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">Doll</div> <div class="given-names">J.</div></div>,Â et al.Â </div><div class="article-title"> Transcriptional changes in response to arbuscular mycorrhiza development in the model plant <em>Medicago truncatula</em></div>,Â <div class="source">Mol. Plant Microbe Interact.</div>Â ,Â <div class="year">2003</div>,Â vol.Â <div class="volume">16</div>Â (pg.Â <div class="fpage">306</div>-<div class="lpage">314</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20changes%20in%20response%20to%20arbuscular%20mycorrhiza%20development%20in%20the%20model%20plant%20Medicago%20truncatula&amp;author=A.%20Wulf&amp;author=K.%20Manthey&amp;author=J.%20Doll&amp;publication_year=2003&amp;journal=Mol.%20Plant%20Microbe%20Interact.&amp;volume=16&amp;pages=306-314" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI.2003.16.4.306" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI.2003.16.4.306" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI.2003.16.4.306"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12744459" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brechenmacher</div> <div class="given-names">L.</div></div>,Â  <div class="name"><div class="surname">Weidmann</div> <div class="given-names">S.</div></div>,Â  <div class="name"><div class="surname">van Tuinen</div> <div class="given-names">D.</div></div>,Â et al.Â </div><div class="article-title"> Expression profiling of up-regulated plant and fungal genes in early and late stages of <em>Medicago truncatula-Glomus mosseae</em> interactions </div>,Â <div class="source">Mycorrhiza</div>Â ,Â <div class="year">2004</div>,Â vol.Â <div class="volume">14</div>Â (pg.Â <div class="fpage">253</div>-<div class="lpage">262</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20profiling%20of%20up-regulated%20plant%20and%20fungal%20genes%20in%20early%20and%20late%20stages%20of%20Medicago%20truncatula-Glomus%20mosseae%20interactions&amp;author=L.%20Brechenmacher&amp;author=S.%20Weidmann&amp;author=D.%20van%20Tuinen&amp;publication_year=2004&amp;journal=Mycorrhiza&amp;volume=14&amp;pages=253-262" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00572-003-0263-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00572-003-0263-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00572-003-0263-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/13680319" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Frenzel</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Manthey</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">Perlick</div> <div class="given-names">A. M.</div></div>,Â et al.Â </div><div class="article-title"> Combined transcriptome profiling reveals a novel family of arbuscular mycorrhizal-specific <em>Medicago truncatula</em> lectin genes </div>,Â <div class="source">Mol. Plant Microbe Interact.</div>Â ,Â <div class="year">2005</div>,Â vol.Â <div class="volume">18</div>Â (pg.Â <div class="fpage">771</div>-<div class="lpage">782</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Combined%20transcriptome%20profiling%20reveals%20a%20novel%20family%20of%20arbuscular%20mycorrhizal-specific%20Medicago%20truncatula%20lectin%20genes&amp;author=A.%20Frenzel&amp;author=K.%20Manthey&amp;author=A.%20M.%20Perlick&amp;publication_year=2005&amp;journal=Mol.%20Plant%20Microbe%20Interact.&amp;volume=18&amp;pages=771-782" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI-18-0771" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI-18-0771" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI-18-0771"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16134889" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">Blaylock</div> <div class="given-names">L. A.</div></div>,Â  <div class="name"><div class="surname">Endre</div> <div class="given-names">G.</div></div>,Â et al.Â </div><div class="article-title">Transcript profiling coupled with spatial expression analyses reveals genes involved in distinct developmental stages of an arbuscular mycorrhizal symbiosis</div>,Â <div class="source">Plant Cell</div>Â ,Â <div class="year">2003</div>,Â vol.Â <div class="volume">15</div>Â (pg.Â <div class="fpage">2106</div>-<div class="lpage">2123</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcript%20profiling%20coupled%20with%20spatial%20expression%20analyses%20reveals%20genes%20involved%20in%20distinct%20developmental%20stages%20of%20an%20arbuscular%20mycorrhizal%20symbiosis&amp;author=J.%20Liu&amp;author=L.%20A.%20Blaylock&amp;author=G.%20Endre&amp;publication_year=2003&amp;journal=Plant%20Cell&amp;volume=15&amp;pages=2106-2123" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.014183" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.014183" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.014183"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12953114" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Manthey</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Krajinski</div> <div class="given-names">F.</div></div>,Â  <div class="name"><div class="surname">Hohnjec</div> <div class="given-names">N.</div></div>,Â et al.Â </div><div class="article-title"> Transcriptome profiling in root nodules and arbuscular mycorrhiza identifies a collection of novel genes induced during <em>Medicago truncatula</em> root endosymbioses </div>,Â <div class="source">Mol. Plant Microbe Interact.</div>Â ,Â <div class="year">2004</div>,Â vol.Â <div class="volume">17</div>Â (pg.Â <div class="fpage">1063</div>-<div class="lpage">1077</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20profiling%20in%20root%20nodules%20and%20arbuscular%20mycorrhiza%20identifies%20a%20collection%20of%20novel%20genes%20induced%20during%20Medicago%20truncatula%20root%20endosymbioses&amp;author=M.%20Manthey&amp;author=F.%20Krajinski&amp;author=N.%20Hohnjec&amp;publication_year=2004&amp;journal=Mol.%20Plant%20Microbe%20Interact.&amp;volume=17&amp;pages=1063-1077" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI.2004.17.10.1063" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI.2004.17.10.1063" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI.2004.17.10.1063"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15497399" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hohnjec</div> <div class="given-names">N.</div></div>,Â  <div class="name"><div class="surname">Vieweg</div> <div class="given-names">M. F.</div></div>,Â  <div class="name"><div class="surname">Puhler</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Becker</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Kuster</div> <div class="given-names">H</div></div>.Â </div><div class="article-title"> Overlaps in the transcriptional profiles of <em>Medicago truncatula</em> roots inoculated with two different <em>Glomus</em> fungi provide insights into the genetic program activated during arbuscular mycorrhiza </div>,Â <div class="source">Plant Physiol.</div>Â ,Â <div class="year">2005</div>,Â vol.Â <div class="volume">137</div>Â (pg.Â <div class="fpage">1283</div>-<div class="lpage">1301</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Overlaps%20in%20the%20transcriptional%20profiles%20of%20Medicago%20truncatula%20roots%20inoculated%20with%20two%20different%20Glomus%20fungi%20provide%20insights%20into%20the%20genetic%20program%20activated%20during%20arbuscular%20mycorrhiza&amp;author=N.%20Hohnjec&amp;author=M.%20F.%20Vieweg&amp;author=A.%20Puhler&amp;author=A.%20Becker&amp;author=H%20Kuster&amp;publication_year=2005&amp;journal=Plant%20Physiol.&amp;volume=137&amp;pages=1283-1301" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.104.056572" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.104.056572" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.104.056572"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15778460" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Massoumou</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">van Tuinen</div> <div class="given-names">D.</div></div>,Â  <div class="name"><div class="surname">Chatagnier</div> <div class="given-names">O.</div></div>,Â et al.Â </div><div class="article-title"><em>Medicago truncatula</em> gene responses specific to arbuscular mycorrhiza interactions with different species and genera of Glomeromycota </div>,Â <div class="source">Mycorrhiza</div>Â ,Â <div class="year">2007</div>,Â vol.Â <div class="volume">17</div>Â (pg.Â <div class="fpage">223</div>-<div class="lpage">234</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Medicago%20truncatula%20gene%20responses%20specific%20to%20arbuscular%20mycorrhiza%20interactions%20with%20different%20species%20and%20genera%20of%20Glomeromycota&amp;author=M.%20Massoumou&amp;author=D.%20van%20Tuinen&amp;author=O.%20Chatagnier&amp;publication_year=2007&amp;journal=Mycorrhiza&amp;volume=17&amp;pages=223-234" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00572-006-0099-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00572-006-0099-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00572-006-0099-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17245570" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">Maldonado-Mendoza</div> <div class="given-names">I.</div></div>,Â  <div class="name"><div class="surname">Lopez-Meyer</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Cheung</div> <div class="given-names">F.</div></div>,Â  <div class="name"><div class="surname">Town</div> <div class="given-names">C. D.</div></div>,Â  <div class="name"><div class="surname">Harrison</div> <div class="given-names">M. J</div></div>.Â </div><div class="article-title">Arbuscular mycorrhizal symbiosis is accompanied by local and systemic alterations in gene expression and an increase in disease resistance in the shoots</div>,Â <div class="source">Plant J.</div>Â ,Â <div class="year">2007</div>,Â vol.Â <div class="volume">50</div>Â (pg.Â <div class="fpage">529</div>-<div class="lpage">544</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Arbuscular%20mycorrhizal%20symbiosis%20is%20accompanied%20by%20local%20and%20systemic%20alterations%20in%20gene%20expression%20and%20an%20increase%20in%20disease%20resistance%20in%20the%20shoots&amp;author=J.%20Liu&amp;author=I.%20Maldonado-Mendoza&amp;author=M.%20Lopez-Meyer&amp;author=F.%20Cheung&amp;author=C.%20D.%20Town&amp;author=M.%20J%20Harrison&amp;publication_year=2007&amp;journal=Plant%20J.&amp;volume=50&amp;pages=529-544" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-313X.2007.03069.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-313X.2007.03069.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-313X.2007.03069.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17419842" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barker</div> <div class="given-names">D. G.</div></div>,Â  <div class="name"><div class="surname">Bianchi</div> <div class="given-names">S.</div></div>,Â  <div class="name"><div class="surname">Blondon</div> <div class="given-names">F.</div></div>,Â et al.Â </div><div class="article-title"><em>Medicago truncatula</em> , a model plant for studying the molecular genetics of the Rhizobium-legume symbiosis </div>,Â <div class="source">Plant Mol. Biol. Rep.</div>Â ,Â <div class="year">1990</div>,Â vol.Â <div class="volume">8</div>Â (pg.Â <div class="fpage">40</div>-<div class="lpage">49</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Medicago%20truncatula%20%2C%20a%20model%20plant%20for%20studying%20the%20molecular%20genetics%20of%20the%20Rhizobium-legume%20symbiosis&amp;author=D.%20G.%20Barker&amp;author=S.%20Bianchi&amp;author=F.%20Blondon&amp;publication_year=1990&amp;journal=Plant%20Mol.%20Biol.%20Rep.&amp;volume=8&amp;pages=40-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02668879" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02668879" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02668879"> </span></div>Â </div></div></div></div></div><div content-id="DSM014C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cook</div> <div class="given-names">D. R</div></div>.Â </div><div class="article-title"><em>Medicago truncatula</em> â€”a model in the making! </div>,Â <div class="source">Curr. Opin. Plant Biol.</div>Â ,Â <div class="year">1999</div>,Â vol.Â <div class="volume">2</div>Â (pg.Â <div class="fpage">301</div>-<div class="lpage">304</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Medicago%20truncatula%20%E2%80%94a%20model%20in%20the%20making!&amp;author=D.%20R%20Cook&amp;publication_year=1999&amp;journal=Curr.%20Opin.%20Plant%20Biol.&amp;volume=2&amp;pages=301-304" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1369-5266(99)80053-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1369-5266(99)80053-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1369-5266(99)80053-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10459004" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Salzer</div> <div class="given-names">P.</div></div>,Â  <div class="name"><div class="surname">Bonanomi</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Beyer</div> <div class="given-names">K.</div></div>,Â et al.Â </div><div class="article-title"> Differential expression of eight chitinase genes in <em>Medicago truncatula</em> roots during mycorrhiza formation, nodulation, and pathogen infection </div>,Â <div class="source">Mol. Plant Microbe Interact.</div>Â ,Â <div class="year">2000</div>,Â vol.Â <div class="volume">13</div>Â (pg.Â <div class="fpage">763</div>-<div class="lpage">777</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20expression%20of%20eight%20chitinase%20genes%20in%20Medicago%20truncatula%20roots%20during%20mycorrhiza%20formation%2C%20nodulation%2C%20and%20pathogen%20infection&amp;author=P.%20Salzer&amp;author=A.%20Bonanomi&amp;author=K.%20Beyer&amp;publication_year=2000&amp;journal=Mol.%20Plant%20Microbe%20Interact.&amp;volume=13&amp;pages=763-777" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI.2000.13.7.763" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI.2000.13.7.763" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI.2000.13.7.763"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10875337" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Handberg</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">Stougaard</div> <div class="given-names">J</div></div>.Â </div><div class="article-title"><em>Lotus japonicus</em> , an autogamous, diploid legume species for classical and molecular genetics </div>,Â <div class="source">Plant J.</div>Â ,Â <div class="year">1992</div>,Â vol.Â <div class="volume">2</div>Â (pg.Â <div class="fpage">487</div>-<div class="lpage">496</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lotus%20japonicus%20%2C%20an%20autogamous%2C%20diploid%20legume%20species%20for%20classical%20and%20molecular%20genetics&amp;author=K.%20Handberg&amp;author=J%20Stougaard&amp;publication_year=1992&amp;journal=Plant%20J.&amp;volume=2&amp;pages=487-496" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.1992.2.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.1992.2.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.1992.2.issue-4"> </span></div>Â </div></div></div></div></div><div content-id="DSM014C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Akiyama</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">Matsuzaki</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">Hayashi</div> <div class="given-names">H</div></div>.Â </div><div class="article-title">Plant sesquiterpenes induce hyphal branching in arbuscular mycorrhizal fungi</div>,Â <div class="source">Nature</div>Â ,Â <div class="year">2005</div>,Â vol.Â <div class="volume">435</div>Â (pg.Â <div class="fpage">824</div>-<div class="lpage">827</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20sesquiterpenes%20induce%20hyphal%20branching%20in%20arbuscular%20mycorrhizal%20fungi&amp;author=K.%20Akiyama&amp;author=K.%20Matsuzaki&amp;author=H%20Hayashi&amp;publication_year=2005&amp;journal=Nature&amp;volume=435&amp;pages=824-827" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature03608" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature03608" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature03608"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15944706" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kouchi</div> <div class="given-names">H.</div></div>,Â  <div class="name"><div class="surname">Shimomura</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">Hata</div> <div class="given-names">S.</div></div>,Â et al.Â </div><div class="article-title"> Large-scale analysis of gene expression profiles during early stages of root nodule formation in a model legume, <em>Lotus japonicus</em></div>,Â <div class="source">DNA Res.</div>Â ,Â <div class="year">2004</div>,Â vol.Â <div class="volume">11</div>Â (pg.Â <div class="fpage">263</div>-<div class="lpage">274</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20analysis%20of%20gene%20expression%20profiles%20during%20early%20stages%20of%20root%20nodule%20formation%20in%20a%20model%20legume%2C%20Lotus%20japonicus&amp;author=H.%20Kouchi&amp;author=K.%20Shimomura&amp;author=S.%20Hata&amp;publication_year=2004&amp;journal=DNA%20Res.&amp;volume=11&amp;pages=263-274" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/11.4.263" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F11.4.263" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F11.4.263"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15500251" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suganuma</div> <div class="given-names">N.</div></div>,Â  <div class="name"><div class="surname">Yamamoto</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Itou</div> <div class="given-names">A.</div></div>,Â et al.Â </div><div class="article-title"> cDNA macroarray analysis of gene expression in ineffective nodules induced on the <em>Lotus japonicus sen1</em> mutant </div>,Â <div class="source">Mol. Plant Microbe Interact.</div>Â ,Â <div class="year">2004</div>,Â vol.Â <div class="volume">17</div>Â (pg.Â <div class="fpage">1223</div>-<div class="lpage">1233</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=cDNA%20macroarray%20analysis%20of%20gene%20expression%20in%20ineffective%20nodules%20induced%20on%20the%20Lotus%20japonicus%20sen1%20mutant&amp;author=N.%20Suganuma&amp;author=A.%20Yamamoto&amp;author=A.%20Itou&amp;publication_year=2004&amp;journal=Mol.%20Plant%20Microbe%20Interact.&amp;volume=17&amp;pages=1223-1233" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI.2004.17.11.1223" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI.2004.17.11.1223" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI.2004.17.11.1223"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15553247" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Giovanetti</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Mosse</div> <div class="given-names">B</div></div>.Â </div><div class="article-title">An evaluation of techniques for measuring vesicular-arbuscular mycorrhizal infection in roots</div>,Â <div class="source">New Phytol.</div>Â ,Â <div class="year">1980</div>,Â vol.Â <div class="volume">84</div>Â (pg.Â <div class="fpage">489</div>-<div class="lpage">500</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20evaluation%20of%20techniques%20for%20measuring%20vesicular-arbuscular%20mycorrhizal%20infection%20in%20roots&amp;author=M.%20Giovanetti&amp;author=B%20Mosse&amp;publication_year=1980&amp;journal=New%20Phytol.&amp;volume=84&amp;pages=489-500" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-8137.1980.tb04556.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-8137.1980.tb04556.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-8137.1980.tb04556.x"> </span></div>Â </div></div></div></div></div><div content-id="DSM014C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Phillips</div> <div class="given-names">J. M.</div></div>,Â  <div class="name"><div class="surname">Hayman</div> <div class="given-names">D. S</div></div>.Â </div><div class="article-title">Improved procedure for clearing roots and staining parasitic and vesicular-arbuscular mycorrhizal fungi for rapid assessment of infection</div>,Â <div class="source">Trans. Br. Mycol. Soc.</div>Â ,Â <div class="year">1970</div>,Â vol.Â <div class="volume">55</div>Â (pg.Â <div class="fpage">158</div>-<div class="lpage">161</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improved%20procedure%20for%20clearing%20roots%20and%20staining%20parasitic%20and%20vesicular-arbuscular%20mycorrhizal%20fungi%20for%20rapid%20assessment%20of%20infection&amp;author=J.%20M.%20Phillips&amp;author=D.%20S%20Hayman&amp;publication_year=1970&amp;journal=Trans.%20Br.%20Mycol.%20Soc.&amp;volume=55&amp;pages=158-161" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0007-1536(70)80110-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0007-1536(70)80110-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0007-1536(70)80110-3"> </span></div>Â </div></div></div></div></div><div content-id="DSM014C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rosewarne</div> <div class="given-names">G. M.</div></div>,Â  <div class="name"><div class="surname">Barker</div> <div class="given-names">S. J.</div></div>,Â  <div class="name"><div class="surname">Smith</div> <div class="given-names">S. E</div></div>.Â </div><div class="article-title">Production of near-synchronous fungal colonization in tomato for developmental and molecular analyses of mycorrhiza</div>,Â <div class="source">Mycol. Res.</div>Â ,Â <div class="year">1997</div>,Â vol.Â <div class="volume">101</div>Â (pg.Â <div class="fpage">966</div>-<div class="lpage">970</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Production%20of%20near-synchronous%20fungal%20colonization%20in%20tomato%20for%20developmental%20and%20molecular%20analyses%20of%20mycorrhiza&amp;author=G.%20M.%20Rosewarne&amp;author=S.%20J.%20Barker&amp;author=S.%20E%20Smith&amp;publication_year=1997&amp;journal=Mycol.%20Res.&amp;volume=101&amp;pages=966-970" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1017/S0953756297003626" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1017%2FS0953756297003626" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1017%2FS0953756297003626"> </span></div>Â </div></div></div></div></div><div content-id="DSM014C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Banba</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Siddique</div> <div class="given-names">A. B. M.</div></div>,Â  <div class="name"><div class="surname">Kouchi</div> <div class="given-names">H.</div></div>,Â  <div class="name"><div class="surname">Izui</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">Hata</div> <div class="given-names">S</div></div>.Â </div><div class="article-title"><em>Lotus japonicus</em> forms early senescent root nodules with <em>Rhizobium etli</em></div>,Â <div class="source">Mol. Plant Microbe Interact.</div>Â ,Â <div class="year">2001</div>,Â vol.Â <div class="volume">14</div>Â (pg.Â <div class="fpage">173</div>-<div class="lpage">180</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lotus%20japonicus%20forms%20early%20senescent%20root%20nodules%20with%20Rhizobium%20etli&amp;author=M.%20Banba&amp;author=A.%20B.%20M.%20Siddique&amp;author=H.%20Kouchi&amp;author=K.%20Izui&amp;author=S%20Hata&amp;publication_year=2001&amp;journal=Mol.%20Plant%20Microbe%20Interact.&amp;volume=14&amp;pages=173-180" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI.2001.14.2.173" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI.2001.14.2.173" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI.2001.14.2.173"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11204780" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ooki</div> <div class="given-names">Y.</div></div>,Â  <div class="name"><div class="surname">Banba</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Yano</div> <div class="given-names">K.</div></div>,Â et al.Â </div><div class="article-title"> Characterization of the <em>Lotus japonicus</em> symbiotic mutant <em>lot1</em> that shows a reduced nodule number and distorted trichomes </div>,Â <div class="source">Plant Physiol.</div>Â ,Â <div class="year">2005</div>,Â vol.Â <div class="volume">137</div>Â (pg.Â <div class="fpage">1261</div>-<div class="lpage">1271</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20the%20Lotus%20japonicus%20symbiotic%20mutant%20lot1%20that%20shows%20a%20reduced%20nodule%20number%20and%20distorted%20trichomes&amp;author=Y.%20Ooki&amp;author=M.%20Banba&amp;author=K.%20Yano&amp;publication_year=2005&amp;journal=Plant%20Physiol.&amp;volume=137&amp;pages=1261-1271" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.104.056630" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.104.056630" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.104.056630"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15793069" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Maeda</div> <div class="given-names">D.</div></div>,Â  <div class="name"><div class="surname">Ashida</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">Iguchi</div> <div class="given-names">K.</div></div>,Â et al.Â </div><div class="article-title"> Knockdown of an arbuscular mycorrhiza-inducible phosphate transporter gene of <em>Lotus japonicus</em> suppresses mutualistic symbiosis </div>,Â <div class="source">Plant Cell Physiol.</div>Â ,Â <div class="year">2006</div>,Â vol.Â <div class="volume">47</div>Â (pg.Â <div class="fpage">807</div>-<div class="lpage">817</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Knockdown%20of%20an%20arbuscular%20mycorrhiza-inducible%20phosphate%20transporter%20gene%20of%20Lotus%20japonicus%20suppresses%20mutualistic%20symbiosis&amp;author=D.%20Maeda&amp;author=K.%20Ashida&amp;author=K.%20Iguchi&amp;publication_year=2006&amp;journal=Plant%20Cell%20Physiol.&amp;volume=47&amp;pages=807-817" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcj069" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcj069" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcj069"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16774930" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kouchi</div> <div class="given-names">H.</div></div>,Â  <div class="name"><div class="surname">Hata</div> <div class="given-names">S</div></div>.Â </div><div class="article-title">Isolation and characterization of novel nodulin cDNAs representing genes expressed at early stages of soybean nodule development</div>,Â <div class="source">Mol. Gen. Genet.</div>Â ,Â <div class="year">1993</div>,Â vol.Â <div class="volume">238</div>Â (pg.Â <div class="fpage">106</div>-<div class="lpage">119</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20characterization%20of%20novel%20nodulin%20cDNAs%20representing%20genes%20expressed%20at%20early%20stages%20of%20soybean%20nodule%20development&amp;author=H.%20Kouchi&amp;author=S%20Hata&amp;publication_year=1993&amp;journal=Mol.%20Gen.%20Genet.&amp;volume=238&amp;pages=106-119" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7683079" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kumagai</div> <div class="given-names">H.</div></div>,Â  <div class="name"><div class="surname">Kouchi</div> <div class="given-names">H</div></div>.Â </div><div class="article-title"> Gene silencing by expression of hairpin RNA in <em>Lotus japonicus</em> roots and root nodules </div>,Â <div class="source">Mol. Plant Microbe Interact.</div>Â ,Â <div class="year">2003</div>,Â vol.Â <div class="volume">16</div>Â (pg.Â <div class="fpage">663</div>-<div class="lpage">668</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20silencing%20by%20expression%20of%20hairpin%20RNA%20in%20Lotus%20japonicus%20roots%20and%20root%20nodules&amp;author=H.%20Kumagai&amp;author=H%20Kouchi&amp;publication_year=2003&amp;journal=Mol.%20Plant%20Microbe%20Interact.&amp;volume=16&amp;pages=663-668" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI.2003.16.8.663" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI.2003.16.8.663" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI.2003.16.8.663"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12906110" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jefferson</div> <div class="given-names">R. A.</div></div>,Â  <div class="name"><div class="surname">Kavanagh</div> <div class="given-names">T. A.</div></div>,Â  <div class="name"><div class="surname">Bevan</div> <div class="given-names">M. W</div></div>.Â </div><div class="article-title">GUS fusions: beta-glucuronidase as a sensitive and versatile gene fusion marker in higher plants</div>,Â <div class="source">EMBO J.</div>Â ,Â <div class="year">1987</div>,Â vol.Â <div class="volume">6</div>Â (pg.Â <div class="fpage">3901</div>-<div class="lpage">3907</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GUS%20fusions%3A%20beta-glucuronidase%20as%20a%20sensitive%20and%20versatile%20gene%20fusion%20marker%20in%20higher%20plants&amp;author=R.%20A.%20Jefferson&amp;author=T.%20A.%20Kavanagh&amp;author=M.%20W%20Bevan&amp;publication_year=1987&amp;journal=EMBO%20J.&amp;volume=6&amp;pages=3901-3907" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3327686" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Maldonado-Mendoza</div> <div class="given-names">I. E.</div></div>,Â  <div class="name"><div class="surname">Dewbre</div> <div class="given-names">G. R.</div></div>,Â  <div class="name"><div class="surname">van Buuren</div> <div class="given-names"> M. L.</div></div>,Â  <div class="name"><div class="surname">Versaw</div> <div class="given-names">W. K.</div></div>,Â  <div class="name"><div class="surname">Harrison</div> <div class="given-names">M. J</div></div>.Â </div><div class="article-title">Methods to estimate the proportion of plant and fungal RNA in an arbuscular mycorrhiza</div>,Â <div class="source">Mycorrhiza</div>Â ,Â <div class="year">2002</div>,Â vol.Â <div class="volume">12</div>Â (pg.Â <div class="fpage">67</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Methods%20to%20estimate%20the%20proportion%20of%20plant%20and%20fungal%20RNA%20in%20an%20arbuscular%20mycorrhiza&amp;author=I.%20E.%20Maldonado-Mendoza&amp;author=G.%20R.%20Dewbre&amp;author=M.%20L.%20van%20Buuren&amp;author=W.%20K.%20Versaw&amp;author=M.%20J%20Harrison&amp;publication_year=2002&amp;journal=Mycorrhiza&amp;volume=12&amp;pages=67-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00572-001-0149-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00572-001-0149-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00572-001-0149-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12035729" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tusher</div> <div class="given-names">V. G.</div></div>,Â  <div class="name"><div class="surname">Tibshirani</div> <div class="given-names">R.</div></div>,Â  <div class="name"><div class="surname">Chu</div> <div class="given-names">G</div></div>.Â </div><div class="article-title">Significance analysis of microarrays applied to the ionizing radiation response</div>,Â <div class="source">Proc. Natl. Acad. Sci. USA</div>Â ,Â <div class="year">2001</div>,Â vol.Â <div class="volume">98</div>Â (pg.Â <div class="fpage">5116</div>-<div class="lpage">5121</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Significance%20analysis%20of%20microarrays%20applied%20to%20the%20ionizing%20radiation%20response&amp;author=V.%20G.%20Tusher&amp;author=R.%20Tibshirani&amp;author=G%20Chu&amp;publication_year=2001&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=98&amp;pages=5116-5121" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.091062498" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.091062498" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.091062498"> </span></div>Â </div></div></div></div></div><div content-id="DSM014C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dixon</div> <div class="given-names">R. A.</div></div>,Â  <div class="name"><div class="surname">Paiva</div> <div class="given-names">N. L</div></div>.Â </div><div class="article-title">Stress-induced phenylpropanoid metabolism</div>,Â <div class="source">Plant Cell</div>Â ,Â <div class="year">1995</div>,Â vol.Â <div class="volume">7</div>Â (pg.Â <div class="fpage">1085</div>-<div class="lpage">1097</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stress-induced%20phenylpropanoid%20metabolism&amp;author=R.%20A.%20Dixon&amp;author=N.%20L%20Paiva&amp;publication_year=1995&amp;journal=Plant%20Cell&amp;volume=7&amp;pages=1085-1097" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.7.7.1085" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.7.7.1085" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.7.7.1085"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12242399" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hahlbrock</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">Bednarek</div> <div class="given-names">P.</div></div>,Â  <div class="name"><div class="surname">Ciolkowski</div> <div class="given-names">I.</div></div>,Â et al.Â </div><div class="article-title">Non-self recognition, transcriptional reprogramming, and secondary metabolite accumulation during plant/pathogen interactions</div>,Â <div class="source">Proc. Natl. Acad. Sci. USA</div>Â ,Â <div class="year">2003</div>,Â vol.Â <div class="volume">100</div>Â (pg.Â <div class="fpage">14569</div>-<div class="lpage">14576</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Non-self%20recognition%2C%20transcriptional%20reprogramming%2C%20and%20secondary%20metabolite%20accumulation%20during%20plant%2Fpathogen%20interactions&amp;author=K.%20Hahlbrock&amp;author=P.%20Bednarek&amp;author=I.%20Ciolkowski&amp;publication_year=2003&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=100&amp;pages=14569-14576" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0831246100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0831246100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0831246100"> </span></div>Â </div></div></div></div></div><div content-id="DSM014C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guimil</div> <div class="given-names">S.</div></div>,Â  <div class="name"><div class="surname">Chang</div> <div class="given-names">H. S.</div></div>,Â  <div class="name"><div class="surname">Zhu</div> <div class="given-names">T.</div></div>,Â et al.Â </div><div class="article-title">Comparative transcriptomics of rice reveals an ancient pattern of response to microbial colonization</div>,Â <div class="source">Proc. Natl. Acad. Sci. USA</div>Â ,Â <div class="year">2005</div>,Â vol.Â <div class="volume">102</div>Â (pg.Â <div class="fpage">8066</div>-<div class="lpage">8070</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20transcriptomics%20of%20rice%20reveals%20an%20ancient%20pattern%20of%20response%20to%20microbial%20colonization&amp;author=S.%20Guimil&amp;author=H.%20S.%20Chang&amp;author=T.%20Zhu&amp;publication_year=2005&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=102&amp;pages=8066-8070" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0502999102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0502999102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0502999102"> </span></div>Â </div></div></div></div></div><div content-id="DSM014C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vincent</div> <div class="given-names">J. L.</div></div>,Â  <div class="name"><div class="surname">Brewin</div> <div class="given-names">N. J</div></div>.Â </div><div class="article-title">Immunolocalization of a cysteine protease in vacuoles, vesicles, and symbiosomes of pea nodule cells</div>,Â <div class="source">Plant Physiol.</div>Â ,Â <div class="year">2000</div>,Â vol.Â <div class="volume">123</div>Â (pg.Â <div class="fpage">521</div>-<div class="lpage">530</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Immunolocalization%20of%20a%20cysteine%20protease%20in%20vacuoles%2C%20vesicles%2C%20and%20symbiosomes%20of%20pea%20nodule%20cells&amp;author=J.%20L.%20Vincent&amp;author=N.%20J%20Brewin&amp;publication_year=2000&amp;journal=Plant%20Physiol.&amp;volume=123&amp;pages=521-530" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.123.2.521" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.123.2.521" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.123.2.521"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10859182" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Otegui</div> <div class="given-names">M. S.</div></div>,Â  <div class="name"><div class="surname">Noh</div> <div class="given-names">Y. S.</div></div>,Â  <div class="name"><div class="surname">Martinez</div> <div class="given-names">D. E.</div></div>,Â et al.Â </div><div class="article-title">Senescence-associated vacuoles with intense proteolytic activity develop in leaves of Arabidopsis and soybean</div>,Â <div class="source">Plant J.</div>Â ,Â <div class="year">2005</div>,Â vol.Â <div class="volume">41</div>Â (pg.Â <div class="fpage">831</div>-<div class="lpage">844</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Senescence-associated%20vacuoles%20with%20intense%20proteolytic%20activity%20develop%20in%20leaves%20of%20Arabidopsis%20and%20soybean&amp;author=M.%20S.%20Otegui&amp;author=Y.%20S.%20Noh&amp;author=D.%20E.%20Martinez&amp;publication_year=2005&amp;journal=Plant%20J.&amp;volume=41&amp;pages=831-844" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2005.41.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2005.41.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2005.41.issue-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15743448" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">H.</div></div>,Â  <div class="name"><div class="surname">Trieu</div> <div class="given-names">A. T.</div></div>,Â  <div class="name"><div class="surname">Blaylock</div> <div class="given-names">L. A.</div></div>,Â  <div class="name"><div class="surname">Harrison</div> <div class="given-names">M. J</div></div>.Â </div><div class="article-title"> Cloning and characterization of two phosphate transporters from <em>Medicago truncatula</em> roots: regulation in response to phosphate and to colonization by arbuscular mycorrhizal (AM) fungi </div>,Â <div class="source">Mol. Plant Microbe Interact.</div>Â ,Â <div class="year">1998</div>,Â vol.Â <div class="volume">11</div>Â (pg.Â <div class="fpage">14</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cloning%20and%20characterization%20of%20two%20phosphate%20transporters%20from%20Medicago%20truncatula%20roots%3A%20regulation%20in%20response%20to%20phosphate%20and%20to%20colonization%20by%20arbuscular%20mycorrhizal%20(AM)%20fungi&amp;author=H.%20Liu&amp;author=A.%20T.%20Trieu&amp;author=L.%20A.%20Blaylock&amp;author=M.%20J%20Harrison&amp;publication_year=1998&amp;journal=Mol.%20Plant%20Microbe%20Interact.&amp;volume=11&amp;pages=14-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI.1998.11.1.14" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI.1998.11.1.14" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI.1998.11.1.14"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9425684" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rausch</div> <div class="given-names">C.</div></div>,Â  <div class="name"><div class="surname">Daram</div> <div class="given-names">P.</div></div>,Â  <div class="name"><div class="surname">Brunner</div> <div class="given-names">S.</div></div>,Â et al.Â </div><div class="article-title">A phosphate transporter expressed in arbuscule-containing cells in potato</div>,Â <div class="source">Nature</div>Â ,Â <div class="year">2001</div>,Â vol.Â <div class="volume">414</div>Â (pg.Â <div class="fpage">462</div>-<div class="lpage">470</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20phosphate%20transporter%20expressed%20in%20arbuscule-containing%20cells%20in%20potato&amp;author=C.%20Rausch&amp;author=P.%20Daram&amp;author=S.%20Brunner&amp;publication_year=2001&amp;journal=Nature&amp;volume=414&amp;pages=462-470" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35106601" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35106601" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35106601"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11719809" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Paszkowski</div> <div class="given-names">U.</div></div>,Â  <div class="name"><div class="surname">Kroken</div> <div class="given-names">S.</div></div>,Â  <div class="name"><div class="surname">Roux</div> <div class="given-names">C.</div></div>,Â  <div class="name"><div class="surname">Briggs</div> <div class="given-names">S. P</div></div>.Â </div><div class="article-title">Rice phosphate transporters include an evolutionarily divergent gene specifically activated in arbuscular mycorrhizal symbiosis</div>,Â <div class="source">Proc. Natl. Acad. Sci. USA</div>Â ,Â <div class="year">2002</div>,Â vol.Â <div class="volume">99</div>Â (pg.Â <div class="fpage">13324</div>-<div class="lpage">13329</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rice%20phosphate%20transporters%20include%20an%20evolutionarily%20divergent%20gene%20specifically%20activated%20in%20arbuscular%20mycorrhizal%20symbiosis&amp;author=U.%20Paszkowski&amp;author=S.%20Kroken&amp;author=C.%20Roux&amp;author=S.%20P%20Briggs&amp;publication_year=2002&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=99&amp;pages=13324-13329" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.202474599" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.202474599" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.202474599"> </span></div>Â </div></div></div></div></div><div content-id="DSM014C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Blee</div> <div class="given-names">K. A.</div></div>,Â  <div class="name"><div class="surname">Anderson</div> <div class="given-names">A. J</div></div>.Â </div><div class="article-title"> Defense-related transcript accumulation in <em>Phaseolus vulgaris</em> L. colonized by the arbuscular mycorrhizal fungus <em>Glomus intraradices</em> Schenck &amp; Smith </div>,Â <div class="source">Plant Physiol.</div>Â ,Â <div class="year">1996</div>,Â vol.Â <div class="volume">110</div>Â (pg.Â <div class="fpage">675</div>-<div class="lpage">688</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Defense-related%20transcript%20accumulation%20in%20Phaseolus%20vulgaris%20L.%20colonized%20by%20the%20arbuscular%20mycorrhizal%20fungus%20Glomus%20intraradices%20Schenck%20%26%20Smith&amp;author=K.%20A.%20Blee&amp;author=A.%20J%20Anderson&amp;publication_year=1996&amp;journal=Plant%20Physiol.&amp;volume=110&amp;pages=675-688" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.110.2.675" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.110.2.675" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.110.2.675"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12226210" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gao</div> <div class="given-names">L. L.</div></div>,Â  <div class="name"><div class="surname">Knogge</div> <div class="given-names">W.</div></div>,Â  <div class="name"><div class="surname">Delp</div> <div class="given-names">G.</div></div>,Â  <div class="name"><div class="surname">Smith</div> <div class="given-names">F. A.</div></div>,Â  <div class="name"><div class="surname">Smith</div> <div class="given-names">S. E</div></div>.Â </div><div class="article-title">Expression patterns of defense-related genes in different types of arbuscular mycorrhizal development in wild-type and mycorrhiza-defective mutant tomato</div>,Â <div class="source">Mol. Plant Microbe Interact.</div>Â ,Â <div class="year">2004</div>,Â vol.Â <div class="volume">17</div>Â (pg.Â <div class="fpage">1103</div>-<div class="lpage">1113</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20patterns%20of%20defense-related%20genes%20in%20different%20types%20of%20arbuscular%20mycorrhizal%20development%20in%20wild-type%20and%20mycorrhiza-defective%20mutant%20tomato&amp;author=L.%20L.%20Gao&amp;author=W.%20Knogge&amp;author=G.%20Delp&amp;author=F.%20A.%20Smith&amp;author=S.%20E%20Smith&amp;publication_year=2004&amp;journal=Mol.%20Plant%20Microbe%20Interact.&amp;volume=17&amp;pages=1103-1113" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI.2004.17.10.1103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI.2004.17.10.1103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI.2004.17.10.1103"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15497403" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Harrison</div> <div class="given-names">M. J.</div></div>,Â  <div class="name"><div class="surname">Dixon</div> <div class="given-names">R. A</div></div>.Â </div><div class="article-title"> Isoflavonoid accumulation and expression of defense gene transcripts during the establishment of vesicular-arbuscular mycorrhizal associations in roots of <em>Medicago truncatula</em></div>,Â <div class="source">Mol. Plant Microbe Interact.</div>Â ,Â <div class="year">1993</div>,Â vol.Â <div class="volume">6</div>Â (pg.Â <div class="fpage">643</div>-<div class="lpage">654</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isoflavonoid%20accumulation%20and%20expression%20of%20defense%20gene%20transcripts%20during%20the%20establishment%20of%20vesicular-arbuscular%20mycorrhizal%20associations%20in%20roots%20of%20Medicago%20truncatula&amp;author=M.%20J.%20Harrison&amp;author=R.%20A%20Dixon&amp;publication_year=1993&amp;journal=Mol.%20Plant%20Microbe%20Interact.&amp;volume=6&amp;pages=643-654" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI-6-643" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI-6-643" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI-6-643"> </span></div>Â </div></div></div></div></div><div content-id="DSM014C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Harrison</div> <div class="given-names">M. J.</div></div>,Â  <div class="name"><div class="surname">Dixon</div> <div class="given-names">R. A</div></div>.Â </div><div class="article-title"> Spatial patterns of expression of flavonoid/isoflavonoid pathway genes during interactions between roots of <em>Medicago truncatula</em> and the mycorrhizal fungus <em>Glomus versiforme</em></div>,Â <div class="source">Plant J.</div>Â ,Â <div class="year">1994</div>,Â vol.Â <div class="volume">6</div>Â (pg.Â <div class="fpage">9</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Spatial%20patterns%20of%20expression%20of%20flavonoid%2Fisoflavonoid%20pathway%20genes%20during%20interactions%20between%20roots%20of%20Medicago%20truncatula%20and%20the%20mycorrhizal%20fungus%20Glomus%20versiforme&amp;author=M.%20J.%20Harrison&amp;author=R.%20A%20Dixon&amp;publication_year=1994&amp;journal=Plant%20J.&amp;volume=6&amp;pages=9-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313X.1994.6010009.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313X.1994.6010009.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313X.1994.6010009.x"> </span></div>Â </div></div></div></div></div><div content-id="DSM014C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Volpin</div> <div class="given-names">H.</div></div>,Â  <div class="name"><div class="surname">Elkind</div> <div class="given-names">Y.</div></div>,Â  <div class="name"><div class="surname">Okon</div> <div class="given-names">Y.</div></div>,Â  <div class="name"><div class="surname">Kapulnik</div> <div class="given-names">Y</div></div>.Â </div><div class="article-title"> A vesicular arbuscular mycorrhizal fungus ( <em>Glomus intraradix</em> ) induces a defense response in alfalfa roots </div>,Â <div class="source">Plant Physiol.</div>Â ,Â <div class="year">1994</div>,Â vol.Â <div class="volume">104</div>Â (pg.Â <div class="fpage">683</div>-<div class="lpage">689</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20vesicular%20arbuscular%20mycorrhizal%20fungus%20(%20Glomus%20intraradix%20)%20induces%20a%20defense%20response%20in%20alfalfa%20roots&amp;author=H.%20Volpin&amp;author=Y.%20Elkind&amp;author=Y.%20Okon&amp;author=Y%20Kapulnik&amp;publication_year=1994&amp;journal=Plant%20Physiol.&amp;volume=104&amp;pages=683-689" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.104.2.683" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.104.2.683" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.104.2.683"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12232119" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Volpin</div> <div class="given-names">H.</div></div>,Â  <div class="name"><div class="surname">Phillips</div> <div class="given-names">D. A.</div></div>,Â  <div class="name"><div class="surname">Okon</div> <div class="given-names">Y.</div></div>,Â  <div class="name"><div class="surname">Kapulnik</div> <div class="given-names">Y</div></div>.Â </div><div class="article-title">Suppression of an isoflavonoid phytoalexin defense response in micorrhizal alfalfa roots</div>,Â <div class="source">Plant Physiol.</div>Â ,Â <div class="year">1995</div>,Â vol.Â <div class="volume">108</div>Â (pg.Â <div class="fpage">1449</div>-<div class="lpage">1454</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Suppression%20of%20an%20isoflavonoid%20phytoalexin%20defense%20response%20in%20micorrhizal%20alfalfa%20roots&amp;author=H.%20Volpin&amp;author=D.%20A.%20Phillips&amp;author=Y.%20Okon&amp;author=Y%20Kapulnik&amp;publication_year=1995&amp;journal=Plant%20Physiol.&amp;volume=108&amp;pages=1449-1454" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.108.4.1449" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.108.4.1449" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.108.4.1449"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12228554" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lange</div> <div class="given-names">J.</div></div>,Â  <div class="name"><div class="surname">Xie</div> <div class="given-names">Z. P.</div></div>,Â  <div class="name"><div class="surname">Broughton</div> <div class="given-names">W. J.</div></div>,Â  <div class="name"><div class="surname">Vogeli-Lange</div> <div class="given-names">R.</div></div>,Â  <div class="name"><div class="surname">Boller</div> <div class="given-names">T</div></div>.Â </div><div class="article-title">A gene encoding a receptor-like protein kinase in the roots of common bean is differentially regulated in response to pathogens, symbionts and nodulation factors</div>,Â <div class="source">Plant Sci.</div>Â ,Â <div class="year">1999</div>,Â vol.Â <div class="volume">142</div>Â (pg.Â <div class="fpage">133</div>-<div class="lpage">145</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20gene%20encoding%20a%20receptor-like%20protein%20kinase%20in%20the%20roots%20of%20common%20bean%20is%20differentially%20regulated%20in%20response%20to%20pathogens%2C%20symbionts%20and%20nodulation%20factors&amp;author=J.%20Lange&amp;author=Z.%20P.%20Xie&amp;author=W.%20J.%20Broughton&amp;author=R.%20Vogeli-Lange&amp;author=T%20Boller&amp;publication_year=1999&amp;journal=Plant%20Sci.&amp;volume=142&amp;pages=133-145" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9452(99)00006-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9452(99)00006-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9452(99)00006-0"> </span></div>Â </div></div></div></div></div><div content-id="DSM014C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Montesano</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Koiv</div> <div class="given-names">V.</div></div>,Â  <div class="name"><div class="surname">Mae</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Palva</div> <div class="given-names">E. T</div></div>.Â </div><div class="article-title"> Novel receptor-like protein kinases induced by <em>Erwinia carotovora</em> and short oligogalacturonides in potato </div>,Â <div class="source">Mol. Plant Pathol.</div>Â ,Â <div class="year">2001</div>,Â vol.Â <div class="volume">2</div>Â (pg.Â <div class="fpage">339</div>-<div class="lpage">346</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20receptor-like%20protein%20kinases%20induced%20by%20Erwinia%20carotovora%20and%20short%20oligogalacturonides%20in%20potato&amp;author=M.%20Montesano&amp;author=V.%20Koiv&amp;author=A.%20Mae&amp;author=E.%20T%20Palva&amp;publication_year=2001&amp;journal=Mol.%20Plant%20Pathol.&amp;volume=2&amp;pages=339-346" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1464-6722.2001.00083.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1464-6722.2001.00083.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1464-6722.2001.00083.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20573023" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Logemann</div> <div class="given-names">E.</div></div>,Â  <div class="name"><div class="surname">Parniske</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Hahlbrock</div> <div class="given-names">K</div></div>.Â </div><div class="article-title">Modes of expression and common structural features of the complete phenylalanine ammonia-lyase gene family in parsley</div>,Â <div class="source">Proc. Natl. Acad. Sci. USA</div>Â ,Â <div class="year">1995</div>,Â vol.Â <div class="volume">92</div>Â (pg.Â <div class="fpage">5905</div>-<div class="lpage">5909</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Modes%20of%20expression%20and%20common%20structural%20features%20of%20the%20complete%20phenylalanine%20ammonia-lyase%20gene%20family%20in%20parsley&amp;author=E.%20Logemann&amp;author=M.%20Parniske&amp;author=K%20Hahlbrock&amp;publication_year=1995&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=92&amp;pages=5905-5909" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.92.13.5905" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.92.13.5905" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.92.13.5905"> </span></div>Â </div></div></div></div></div><div content-id="DSM014C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ulker</div> <div class="given-names">B.</div></div>,Â  <div class="name"><div class="surname">Somssich</div> <div class="given-names">I. E</div></div>.Â </div><div class="article-title">WRKY transcription factors: from DNA binding towards biological function</div>,Â <div class="source">Curr. Opin. Plant Biol.</div>Â ,Â <div class="year">2004</div>,Â vol.Â <div class="volume">7</div>Â (pg.Â <div class="fpage">491</div>-<div class="lpage">498</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=WRKY%20transcription%20factors%3A%20from%20DNA%20binding%20towards%20biological%20function&amp;author=B.%20Ulker&amp;author=I.%20E%20Somssich&amp;publication_year=2004&amp;journal=Curr.%20Opin.%20Plant%20Biol.&amp;volume=7&amp;pages=491-498" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pbi.2004.07.012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2004.07.012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pbi.2004.07.012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15337090" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C57" class="ref-content "><div class="label -label">57</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamaguchi-Shinozaki</div> <div class="given-names">K.</div></div>,Â  <div class="name"><div class="surname">Shinozaki</div> <div class="given-names">K</div></div>.Â </div><div class="article-title"> The plant hormone abscisic acid mediates the drought-induced expression but not the seed-specific expression of rd22, a gene responsive to dehydration stress in <em>Arabidopsis thaliana</em></div>,Â <div class="source">Mol. Gen. Genet.</div>Â ,Â <div class="year">1993</div>,Â vol.Â <div class="volume">238</div>Â (pg.Â <div class="fpage">17</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20plant%20hormone%20abscisic%20acid%20mediates%20the%20drought-induced%20expression%20but%20not%20the%20seed-specific%20expression%20of%20rd22%2C%20a%20gene%20responsive%20to%20dehydration%20stress%20in%20Arabidopsis%20thaliana&amp;author=K.%20Yamaguchi-Shinozaki&amp;author=K%20Shinozaki&amp;publication_year=1993&amp;journal=Mol.%20Gen.%20Genet.&amp;volume=238&amp;pages=17-25" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8479424" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSM014C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM014C58" class="ref-content "><div class="label -label">58</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Estabrook</div> <div class="given-names">E. M.</div></div>,Â  <div class="name"><div class="surname">Sengupta-Gopalan</div> <div class="given-names">C</div></div>.Â </div><div class="article-title">Differential expression of phenylalanine ammonia-lyase and chalcone synthase during soybean nodule development</div>,Â <div class="source">Plant Cell</div>Â ,Â <div class="year">1991</div>,Â vol.Â <div class="volume">3</div>Â (pg.Â <div class="fpage">299</div>-<div class="lpage">308</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20expression%20of%20phenylalanine%20ammonia-lyase%20and%20chalcone%20synthase%20during%20soybean%20nodule%20development&amp;author=E.%20M.%20Estabrook&amp;author=C%20Sengupta-Gopalan&amp;publication_year=1991&amp;journal=Plant%20Cell&amp;volume=3&amp;pages=299-308" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.3.3.299" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.3.3.299" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.3.3.299"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1840912" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">Â© The Author 2007. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=578022&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">256</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000248634100003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">41</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsm014"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="578022"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000248634100003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f14%2f3%2f117%2f578022">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000248634100003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (41)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F14%2F3%2F117%2F578022" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/578022'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transpositionâ€“duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value=" Transcriptome Profiling of Lotus japonicus Roots During Arbuscular Mycorrhiza Development and Comparison with that of Nodulation " />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsm014");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f14%2f3%2f117%2f578022">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

