

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/14/2/59/333979","@type":"ScholarlyArticle","name":"The First Genetic and Comparative Map of White Lupin ( Lupinus albus  L.): Identification of QTLs for Anthracnose Resistance and Flowering Time, and a Locus for Alkaloid Content","datePublished":"2007-05-26","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/14/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2007-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsm009","keywords":["<italic>Lupinus albus</italic>","<italic>Medicago truncatula</italic>","synteny","anthracnose"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/14/2/59/333979","author":[{"name":"Phan, Huyen T. T.","affiliation":"1  Australian Centre for Necrotrophic Fungal Pathogens, SABC, Department of Health Sciences ,  Murdoch University ,  Murdoch 6150 ,  Australia","@type":"Person"},{"name":"Ellwood, Simon R.","affiliation":"1  Australian Centre for Necrotrophic Fungal Pathogens, SABC, Department of Health Sciences ,  Murdoch University ,  Murdoch 6150 ,  Australia","@type":"Person"},{"name":"Adhikari, Kedar","affiliation":"2  Department of Agriculture and Food Western Australia, 3 Baron-Hay Court, South Perth 6151 ,  Australia    3  Centre for Legumes in Mediterranean Agriculture, Faculty of Natural and Agricultural Sciences ,  University of Western Australia ,  Crawley 6009 ,  Australia","@type":"Person"},{"name":"Nelson, Matthew N.","affiliation":"4  School of Plant Biology, Faculty of Natural and Agricultural Sciences ,  The University of Western Australia ,  Crawley 6009 ,  Australia","@type":"Person"},{"name":"Oliver, Richard P.","affiliation":"1  Australian Centre for Necrotrophic Fungal Pathogens, SABC, Department of Health Sciences ,  Murdoch University ,  Murdoch 6150 ,  Australia","@type":"Person"}],"description":"Abstract.  We report the first genetic linkage map of white lupin (Lupinus albus L.). An F8 recombinant inbred line population developed from Kiev mutant × P271","pageStart":"59","pageEnd":"70","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/2/0/m_dnaresearch14_2.cover.gif?Expires=2147483647&Signature=5BTa-~goe1WzTtgupy8W5naLIOKYXKJijMDT9~iND6E57C9GS8DnKmgf4xLGiAjAgFQ33xxQGY5XEA2IyH5sxNAchcYGydjD62Blbb3HoCSwzq-ZA0-QrYedFeiN-nl~F4fWgD1eJCN2K69ZEHoUWQqIDlDDD1JhHQnJNcB5lrrNLK6wlcMVRgK0aqJ1Xfk9avS0yFPUV2wTYbI3OOMUsYWIppy7d-BlzWSWPoBfkZar9pX9D9gb52oz134NZ~DyHw-3qm~BemwxBV-O7Uaglh2bFd3cqoLv7fidd1f5Wb74aH7JVK94AzFTLiZ-L8A3WqOO7o6HKGSWlUHJrR5JiA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"The First Genetic and Comparative Map of White Lupin ( Lupinus albus  L.): Identification of QTLs for Anthracnose Resistance and Flowering Time, and a Locus for Alkaloid Content","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/2/0/m_dnaresearch14_2.cover.gif?Expires=2147483647&Signature=5BTa-~goe1WzTtgupy8W5naLIOKYXKJijMDT9~iND6E57C9GS8DnKmgf4xLGiAjAgFQ33xxQGY5XEA2IyH5sxNAchcYGydjD62Blbb3HoCSwzq-ZA0-QrYedFeiN-nl~F4fWgD1eJCN2K69ZEHoUWQqIDlDDD1JhHQnJNcB5lrrNLK6wlcMVRgK0aqJ1Xfk9avS0yFPUV2wTYbI3OOMUsYWIppy7d-BlzWSWPoBfkZar9pX9D9gb52oz134NZ~DyHw-3qm~BemwxBV-O7Uaglh2bFd3cqoLv7fidd1f5Wb74aH7JVK94AzFTLiZ-L8A3WqOO7o6HKGSWlUHJrR5JiA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="First Genetic and Comparative Map of White Lupin ( Lupinus albus  L.): Identification of QTLs for Anthracnose Resistance and Flowering Time, and a Locus for Alkaloid Content" />
    <meta property="og:description" content="Abstract.  We report the first genetic linkage map of white lupin (Lupinus albus L.). An F8 recombinant inbred line population developed from Kiev mutant &#215; P271" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/14/2/59/333979" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2007-05-26" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Phan, Huyen T. T." /><meta name="citation_author_institution" content="Australian Centre for Necrotrophic Fungal Pathogens, SABC, Department of Health Sciences, Murdoch University, Murdoch 6150, Australia" /><meta name="citation_author" content="Ellwood, Simon R." /><meta name="citation_author_institution" content="Australian Centre for Necrotrophic Fungal Pathogens, SABC, Department of Health Sciences, Murdoch University, Murdoch 6150, Australia" /><meta name="citation_author" content="Adhikari, Kedar" /><meta name="citation_author_institution" content="Department of Agriculture and Food Western Australia, 3 Baron-Hay Court, South Perth 6151, Australia" /><meta name="citation_author_institution" content="Centre for Legumes in Mediterranean Agriculture, Faculty of Natural and Agricultural Sciences, University of Western Australia, Crawley 6009, Australia" /><meta name="citation_author" content="Nelson, Matthew N." /><meta name="citation_author_institution" content="School of Plant Biology, Faculty of Natural and Agricultural Sciences, The University of Western Australia, Crawley 6009, Australia" /><meta name="citation_author" content="Oliver, Richard P." /><meta name="citation_author_institution" content="Australian Centre for Necrotrophic Fungal Pathogens, SABC, Department of Health Sciences, Murdoch University, Murdoch 6150, Australia" /><meta name="citation_title" content="The First Genetic and Comparative Map of White Lupin (Lupinus albus L.): Identification of QTLs for Anthracnose Resistance and Flowering Time, and a Locus for Alkaloid Content" /><meta name="citation_firstpage" content="59" /><meta name="citation_lastpage" content="70" /><meta name="citation_doi" content="10.1093/dnares/dsm009" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="14" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2007/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Honeybee pollination of lupins (Lupinus albus cv.Hamburg); Aust. J. Expt. Agric.;  citation_year=1985;  citation_volume=25;  citation_pages=220-223; " /><meta name="citation_reference" content="citation_title=Agronomic potential of narrow-leafed and white lupins in the inland Pacific northwest; Agron J;  citation_year=2004;  citation_volume=96;  citation_pages=1501-1508; " /><meta name="citation_reference" content="citation_title=Studies in populations of the Old World Lupinus species. I. Chromosomes of the East-Mediterranean lupines; Israel J. Bot.;  citation_year=1977;  citation_volume=26;  citation_pages=115-127; " /><meta name="citation_reference" content="citation_title=Nuclear DNA content variation and species relationships in the genus Lupinus (Fabaceae); Ann. Bot.;  citation_year=2003;  citation_volume=92;  citation_pages=349-355; " /><meta name="citation_reference" content="citation_title=Survey for anthracnose caused by Colletotrichum gloeosporioides in crop lupins (Lupinus angustifolius, L. albus) and ornamental lupins (L. polyphyllus) in New South Wales; Australasian Plant Pathology;  citation_year=2001;  citation_volume=27;  citation_pages=259-262; " /><meta name="citation_reference" content="citation_title=Evaluation of Lupinus albus L. Germplasm for the Southeastern USA; Crop Sci;  citation_year=2005;  citation_volume=45;  citation_pages=1941-1950; " /><meta name="citation_reference" content="citation_publisher=Karger, Basel, ; World Review of Nutrition and Dietetics;  citation_year=1995;  citation_volume=77;  citation_pages=75-88; " /><meta name="citation_reference" content="citation_title=Lupin kernel fibre-enriched foods beneficially modify serum lipids in men; Eur. J. Clin. Nutr.;  citation_year=2004;  citation_volume=59;  citation_pages=325-333; " /><meta name="citation_reference" content="citation_publisher=CAB International, Wallingford; Lupins as Crop Plants: Biology, Production and Utilization;  citation_year=1998;  citation_pages=385-410; " /><meta name="citation_reference" content="citation_title=Phytosterols, triterpene alcohols, and phospholipids in seed oil from white lupin; J. Am. Oil Chem. Soc.;  citation_year=2004;  citation_volume=81;  citation_pages=1039-1044; " /><meta name="citation_reference" content="citation_title=Citric acid excretion and precipitation of calcium citrate in the rhizosphere of white lupin (Lupinus albus L.); Plant Cell Environ.;  citation_year=1989;  citation_volume=12;  citation_pages=285-292; " /><meta name="citation_reference" content="citation_title=The acquisition of phosphorus by Lupinus albus L. I. Some characteristics of the soil/root interface; Plant Soil;  citation_year=1982;  citation_volume=68;  citation_pages=19-32; " /><meta name="citation_reference" content="citation_title=Phosphorus deficiency in Lupinus albus (altered lateral root development and enhanced expression of phosphoenolpyruvate carboxylase); Plant Physiol.;  citation_year=1996;  citation_volume=112;  citation_pages=31-41; " /><meta name="citation_reference" content="citation_title=Metabolic changes associated with cluster root development in white lupin (Lupinus albus L.): relationship between organic acid excretion, sucrose metabolism and energy status; Planta;  citation_year=2001;  citation_volume=213;  citation_pages=534-542; " /><meta name="citation_reference" content="citation_title=Cluster roots—an underground adaptation for survival in extreme environments; Trends Plant Sci.;  citation_year=2002;  citation_volume=7;  citation_pages=162-167; " /><meta name="citation_reference" content="citation_title=Genome relationships: the grass model in current research; Plant Cell;  citation_year=2000;  citation_volume=12;  citation_pages=637-646; " /><meta name="citation_reference" content="citation_title=Comparative genome mapping in Brassica; Genetics;  citation_year=1996;  citation_volume=144;  citation_pages=1903-1910; " /><meta name="citation_reference" content="citation_title=Cereal genome evolution—grasses, line up and form a circle; Curr. Biol.;  citation_year=1995;  citation_volume=5;  citation_pages=737-739; " /><meta name="citation_reference" content="citation_title=A direct comparison between the genetic maps of sorghum and rice; Theor. Appl. Genet.;  citation_year=2001;  citation_volume=102;  citation_pages=379-386; " /><meta name="citation_reference" content="citation_title=Estimating genome conservation between crop and model legume species; PNAS;  citation_year=2004;  citation_volume=101;  citation_pages=15289-15294; " /><meta name="citation_reference" content="citation_title=Comparative mapping between Medicago sativa and Pisum sativum; Mol. Genet. Genomics;  citation_year=2004;  citation_volume=272;  citation_pages=235-246; " /><meta name="citation_reference" content="citation_title=Bridging model and crop legumes through comparative genomics; Plant Physiol.;  citation_year=2005;  citation_volume=137;  citation_pages=1189-1196; " /><meta name="citation_reference" content="citation_title=The first gene-based map of Lupinus angustifolius L.-location of domestication genes and conserved synteny with Medicago truncatula; Theor. Appl. Genet.;  citation_year=2006;  citation_volume=113;  citation_pages=225-238; " /><meta name="citation_reference" content="citation_title=Differences in syntenic complexity between Medicago truncatula with Lens culinaris and Lupinus albus; Funct. Plant Biol.;  citation_year=2006;  citation_volume=33;  citation_pages=775-782; " /><meta name="citation_reference" content="citation_title=Comparison of a Brassica oleracea genetic map with the genome of Arabidopsis thaliana; Genetics;  citation_year=2003;  citation_volume=164;  citation_pages=359-372; " /><meta name="citation_reference" content="citation_title=Extensive conservation of linkage relationships between pea and lentil genetic maps; J. Hered.;  citation_year=1992;  citation_volume=83;  citation_pages=123-129; " /><meta name="citation_reference" content="citation_title=Extensive macrosynteny between Medicago truncatula and Lens culinaris ssp. culinaris; Theor. Appl. Genet.;  citation_year=2007;  citation_volume=114;  citation_pages=549-558; " /><meta name="citation_reference" content="citation_title=Functional mapping in pea, as an aid to the candidate gene selection and for investigating synteny with the model legume Medicago truncatula; Theor. Appl. Genet.;  citation_year=2006;  citation_volume=112;  citation_pages=1024-1041; " /><meta name="citation_reference" content="citation_title=Development of nuclear gene-derived molecular markers linked to legume genetic maps; Mol. Genet. Genomics;  citation_year=2006;  citation_volume=276;  citation_pages=56-70; " /><meta name="citation_reference" content="citation_title=A sequence-based genetic map of Medicago truncatula and comparison of marker colinearity with M. sativa; Genetics;  citation_year=2004;  citation_volume=166;  citation_pages=1463-1502; " /><meta name="citation_reference" content="citation_title=Medicago truncatula EST-SSRs reveal cross-species genetic markers for Medicago spp; Theor. Appl. Genet.;  citation_year=2004;  citation_volume=108;  citation_pages=414-422; " /><meta name="citation_reference" content="citation_title=Cross-species amplification of Medicago truncatula microsatellites across three major pulse crops; Theor. Appl. Genet.;  citation_year=2005;  citation_volume=110;  citation_pages=1210-1217; " /><meta name="citation_reference" content="citation_title=AFLP: a new technique for DNA fingerprinting; Nucl. Acids Res.;  citation_year=1995;  citation_volume=23;  citation_pages=4407-4414; " /><meta name="citation_reference" content="citation_title=SSR analysis of the Medicago truncatula SARDI core collection reveals substantial diversity and unusual genotype dispersal throughout the Mediterranean basin; Theor. Appl. Genet.;  citation_year=2006;  citation_volume=112;  citation_pages=977-983; " /><meta name="citation_reference" content="citation_title=Map Manager QTX, cross-platform software for genetic mapping; Mamm. Genome;  citation_year=2001;  citation_volume=12;  citation_pages=930-932; " /><meta name="citation_reference" content="citation_title=Through a genome, darkly: comparative analysis of plant chromosomal DNA; Plant Mol. Biol.;  citation_year=2002;  citation_volume=48;  citation_pages=5-20; " /><meta name="citation_reference" content="citation_title=Construction of a genetic linkage map using MFLP and identification of molecular markers linked to domestication genes in narrow-leafed lupin (Lupinus angustifolius L.); Cell. Mol. Biol. Lett.;  citation_year=2005;  citation_volume=10;  citation_pages=331-344; " /><meta name="citation_reference" content="citation_title=An intraspecific linkage map of the chickpea (Cicer arietinum L.) genome based on sequence tagged microsatellite site and resistance gene analog markers; Theor. Appl. Genet.;  citation_year=2003;  citation_volume=106;  citation_pages=1447-1456; " /><meta name="citation_reference" content="citation_title=Application of biotechnology in breeding lentil for resistance to biotic and abiotic stress; Euphytica;  citation_year=2006;  citation_volume=147;  citation_pages=149-165; " /><meta name="citation_reference" content="citation_title=Recombinant inbred lines for genetic mapping in tomato; Theor. Appl. Genet.;  citation_year=1995;  citation_volume=90;  citation_pages=542-548; " /><meta name="citation_reference" content="citation_title=Mapping genetic factors controlling pollen viability in an interspecific cross in Helianthus sect. Helianthus; Theor. Appl. Genet.;  citation_year=1995;  citation_volume=91;  citation_pages=1195-1202; " /><meta name="citation_reference" content="citation_title=Genetic mapping of an ancient translocation in the genus Lens; Theor. Appl. Genet.;  citation_year=1987;  citation_volume=73;  citation_pages=883-892; " /><meta name="citation_reference" content="citation_title=Chromosomal regions associated with segregation distortion of molecular markers in F2, backcross, doubled haploid, and recombinant inbred populations in rice (Oryza sativa L.); Mol. Genet. Genomics;  citation_year=1997;  citation_volume=253;  citation_pages=535-545; " /><meta name="citation_reference" content="citation_title=Phylogeny and genomic organization of the TIR and non-TIR NBS-LRR resistance gene family in Medicago truncatula; Mol. Plant Microbe Interact.;  citation_year=2002;  citation_volume=15;  citation_pages=529-359; " /><meta name="citation_reference" content="citation_title=Genetic basis of adaptation: flowering time in Arabidopsis thaliana; Theor. Appl. Genet.;  citation_year=1997;  citation_volume=95;  citation_pages=573-583; " /><meta name="citation_reference" content="citation_title=Mapping quantitative trait loci in chickpea associated with time to flowering and resistance to Didymella rabiei the causal agent of Ascochyta blight; Theor. Appl. Genet.;  citation_year=2006;  citation_volume=113;  citation_pages=1357-1369; " /><meta name="citation_reference" content="citation_title=An informative linkage map of soybean reveals QTLs for flowering time, leaflet morphology and regions of segregation distortion; DNA Res;  citation_year=2001;  citation_volume=8;  citation_pages=61-72; " /><meta name="citation_reference" content="citation_publisher=CRC Press, Boca Raton; Statistical Genomics: Linkage, Mapping and QTL Analysis;  citation_year=1985; " /><meta name="citation_reference" content="citation_title=Mapping polygenes; Annual Review of Genetics;  citation_year=1993;  citation_volume=27;  citation_pages=205-233; " /><meta name="citation_reference" content="citation_title=Genetical control of alkaloids in Lupinus albus; Euphytica;  citation_year=1982;  citation_volume=31;  citation_pages=357-364; " /><meta name="citation_reference" content="citation_title=Comparative physical mapping reveals features of microsynteny between Glycine max, Medicago truncatula, and Arabidopsis thaliana; Genome;  citation_year=2004;  citation_volume=47;  citation_pages=141-155; " /><meta name="citation_reference" content="citation_title=Comparative genomics enabled the isolation of the R3a late blight resistance gene in potato; Plant J.;  citation_year=2005;  citation_volume=42;  citation_pages=251-261; " /><meta name="citation_reference" content="citation_title=Identification of perennial ryegrass (Lolium perenne (L.)) and meadow fescue (Festuca pratensis (Huds.)) candidate orthologous sequences to the rice Hd1(Se1) and barley HvCO1 CONSTANS-like genes through comparative mapping and microsynteny; New Phytol.;  citation_year=2005;  citation_volume=167;  citation_pages=239-247; " /><meta name="citation_reference" content="citation_title=Comparative genetic mapping of loci effecting plant height and development in cereals; Euphytica;  citation_year=1998;  citation_volume=100;  citation_pages=245-248; " /><meta name="citation_reference" content="citation_title=High-resolution mapping of the barley leaf rust resistance gene Rph5 using barley expressed sequence tags (ESTs) and synteny with rice; Theor. Appl. Genet.;  citation_year=2005;  citation_volume=111;  citation_pages=1651-1660; " /><meta name="citation_reference" content="citation_title=Comparative mapping and marker-assisted selection in Rosaceae fruit crops; PNAS;  citation_year=2004;  citation_volume=101;  citation_pages=9891-9896; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/14/2/59/17462681/dsm009.pdf" /><meta name="description" content="Abstract.  We report the first genetic linkage map of white lupin (Lupinus albus L.). An F8 recombinant inbred line population developed from Kiev mutant × P271" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/14/2/59/333979" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/14/2/59/333979" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/14/2/59/333979");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsm009';
    </script>
<title>
	First Genetic and Comparative Map of White Lupin (Lupinus albus L.): Identification of QTLs for Anthracnose Resistance and Flowering Time, and a Locus for Alkaloid Content | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./333979" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BfB8PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJQaGFuLCBIdXllbiBULiBULiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkF1c3RyYWxpYW4gQ2VudHJlIGZvciBOZWNyb3Ryb3BoaWMgRnVuZ2FsIFBhdGhvZ2VucywgU0FCQywgRGVwYXJ0bWVudCBvZiBIZWFsdGggU2NpZW5jZXMsIE11cmRvY2ggVW5pdmVyc2l0eSwgTXVyZG9jaCA2MTUwLCBBdXN0cmFsaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJFbGx3b29kLCBTaW1vbiBSLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkF1c3RyYWxpYW4gQ2VudHJlIGZvciBOZWNyb3Ryb3BoaWMgRnVuZ2FsIFBhdGhvZ2VucywgU0FCQywgRGVwYXJ0bWVudCBvZiBIZWFsdGggU2NpZW5jZXMsIE11cmRvY2ggVW5pdmVyc2l0eSwgTXVyZG9jaCA2MTUwLCBBdXN0cmFsaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBZGhpa2FyaSwgS2VkYXIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEFncmljdWx0dXJlIGFuZCBGb29kIFdlc3Rlcm4gQXVzdHJhbGlhLCAzIEJhcm9uLUhheSBDb3VydCwgU291dGggUGVydGggNjE1MSwgQXVzdHJhbGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2VudHJlIGZvciBMZWd1bWVzIGluIE1lZGl0ZXJyYW5lYW4gQWdyaWN1bHR1cmUsIEZhY3VsdHkgb2YgTmF0dXJhbCBhbmQgQWdyaWN1bHR1cmFsIFNjaWVuY2VzLCBVbml2ZXJzaXR5IG9mIFdlc3Rlcm4gQXVzdHJhbGlhLCBDcmF3bGV5IDYwMDksIEF1c3RyYWxpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik5lbHNvbiwgTWF0dGhldyBOLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlNjaG9vbCBvZiBQbGFudCBCaW9sb2d5LCBGYWN1bHR5IG9mIE5hdHVyYWwgYW5kIEFncmljdWx0dXJhbCBTY2llbmNlcywgVGhlIFVuaXZlcnNpdHkgb2YgV2VzdGVybiBBdXN0cmFsaWEsIENyYXdsZXkgNjAwOSwgQXVzdHJhbGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iT2xpdmVyLCBSaWNoYXJkIFAuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQXVzdHJhbGlhbiBDZW50cmUgZm9yIE5lY3JvdHJvcGhpYyBGdW5nYWwgUGF0aG9nZW5zLCBTQUJDLCBEZXBhcnRtZW50IG9mIEhlYWx0aCBTY2llbmNlcywgTXVyZG9jaCBVbml2ZXJzaXR5LCBNdXJkb2NoIDYxNTAsIEF1c3RyYWxpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iVGhlIEZpcnN0IEdlbmV0aWMgYW5kIENvbXBhcmF0aXZlIE1hcCBvZiBXaGl0ZSBMdXBpbiAoTHVwaW51cyBhbGJ1cyBMLik6IElkZW50aWZpY2F0aW9uIG9mIFFUTHMgZm9yIEFudGhyYWNub3NlIFJlc2lzdGFuY2UgYW5kIEZsb3dlcmluZyBUaW1lLCBhbmQgYSBMb2N1cyBmb3IgQWxrYWxvaWQgQ29udGVudCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjU5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI3MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzbTAwOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDA3LzAxLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SG9uZXliZWUgcG9sbGluYXRpb24gb2YgbHVwaW5zIChMdXBpbnVzIGFsYnVzIGN2LkhhbWJ1cmcpOyBBdXN0LiBKLiBFeHB0LiBBZ3JpYy47ICBjaXRhdGlvbl95ZWFyPTE5ODU7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0yMjAtMjIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZ3Jvbm9taWMgcG90ZW50aWFsIG9mIG5hcnJvdy1sZWFmZWQgYW5kIHdoaXRlIGx1cGlucyBpbiB0aGUgaW5sYW5kIFBhY2lmaWMgbm9ydGh3ZXN0OyBBZ3JvbiBKOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTk2OyAgY2l0YXRpb25fcGFnZXM9MTUwMS0xNTA4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHVkaWVzIGluIHBvcHVsYXRpb25zIG9mIHRoZSBPbGQgV29ybGQgTHVwaW51cyBzcGVjaWVzLiBJLiBDaHJvbW9zb21lcyBvZiB0aGUgRWFzdC1NZWRpdGVycmFuZWFuIGx1cGluZXM7IElzcmFlbCBKLiBCb3QuOyAgY2l0YXRpb25feWVhcj0xOTc3OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9MTE1LTEyNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TnVjbGVhciBETkEgY29udGVudCB2YXJpYXRpb24gYW5kIHNwZWNpZXMgcmVsYXRpb25zaGlwcyBpbiB0aGUgZ2VudXMgTHVwaW51cyAoRmFiYWNlYWUpOyBBbm4uIEJvdC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9OTI7ICBjaXRhdGlvbl9wYWdlcz0zNDktMzU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdXJ2ZXkgZm9yIGFudGhyYWNub3NlIGNhdXNlZCBieSBDb2xsZXRvdHJpY2h1bSBnbG9lb3Nwb3Jpb2lkZXMgaW4gY3JvcCBsdXBpbnMgKEx1cGludXMgYW5ndXN0aWZvbGl1cywgTC4gYWxidXMpIGFuZCBvcm5hbWVudGFsIGx1cGlucyAoTC4gcG9seXBoeWxsdXMpIGluIE5ldyBTb3V0aCBXYWxlczsgQXVzdHJhbGFzaWFuIFBsYW50IFBhdGhvbG9neTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTI1OS0yNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2YWx1YXRpb24gb2YgTHVwaW51cyBhbGJ1cyBMLiBHZXJtcGxhc20gZm9yIHRoZSBTb3V0aGVhc3Rlcm4gVVNBOyBDcm9wIFNjaTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00NTsgIGNpdGF0aW9uX3BhZ2VzPTE5NDEtMTk1MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUthcmdlciwgQmFzZWwsIDsgV29ybGQgUmV2aWV3IG9mIE51dHJpdGlvbiBhbmQgRGlldGV0aWNzOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTc3OyAgY2l0YXRpb25fcGFnZXM9NzUtODg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUx1cGluIGtlcm5lbCBmaWJyZS1lbnJpY2hlZCBmb29kcyBiZW5lZmljaWFsbHkgbW9kaWZ5IHNlcnVtIGxpcGlkcyBpbiBtZW47IEV1ci4gSi4gQ2xpbi4gTnV0ci47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTk7ICBjaXRhdGlvbl9wYWdlcz0zMjUtMzMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9Q0FCIEludGVybmF0aW9uYWwsIFdhbGxpbmdmb3JkOyBMdXBpbnMgYXMgQ3JvcCBQbGFudHM6IEJpb2xvZ3ksIFByb2R1Y3Rpb24gYW5kIFV0aWxpemF0aW9uOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fcGFnZXM9Mzg1LTQxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5dG9zdGVyb2xzLCB0cml0ZXJwZW5lIGFsY29ob2xzLCBhbmQgcGhvc3Bob2xpcGlkcyBpbiBzZWVkIG9pbCBmcm9tIHdoaXRlIGx1cGluOyBKLiBBbS4gT2lsIENoZW0uIFNvYy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9ODE7ICBjaXRhdGlvbl9wYWdlcz0xMDM5LTEwNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNpdHJpYyBhY2lkIGV4Y3JldGlvbiBhbmQgcHJlY2lwaXRhdGlvbiBvZiBjYWxjaXVtIGNpdHJhdGUgaW4gdGhlIHJoaXpvc3BoZXJlIG9mIHdoaXRlIGx1cGluIChMdXBpbnVzIGFsYnVzIEwuKTsgUGxhbnQgQ2VsbCBFbnZpcm9uLjsgIGNpdGF0aW9uX3llYXI9MTk4OTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTI4NS0yOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBhY3F1aXNpdGlvbiBvZiBwaG9zcGhvcnVzIGJ5IEx1cGludXMgYWxidXMgTC4gSS4gU29tZSBjaGFyYWN0ZXJpc3RpY3Mgb2YgdGhlIHNvaWwvcm9vdCBpbnRlcmZhY2U7IFBsYW50IFNvaWw7ICBjaXRhdGlvbl95ZWFyPTE5ODI7ICBjaXRhdGlvbl92b2x1bWU9Njg7ICBjaXRhdGlvbl9wYWdlcz0xOS0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGhvc3Bob3J1cyBkZWZpY2llbmN5IGluIEx1cGludXMgYWxidXMgKGFsdGVyZWQgbGF0ZXJhbCByb290IGRldmVsb3BtZW50IGFuZCBlbmhhbmNlZCBleHByZXNzaW9uIG9mIHBob3NwaG9lbm9scHlydXZhdGUgY2FyYm94eWxhc2UpOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTI7ICBjaXRhdGlvbl9wYWdlcz0zMS00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWV0YWJvbGljIGNoYW5nZXMgYXNzb2NpYXRlZCB3aXRoIGNsdXN0ZXIgcm9vdCBkZXZlbG9wbWVudCBpbiB3aGl0ZSBsdXBpbiAoTHVwaW51cyBhbGJ1cyBMLik6IHJlbGF0aW9uc2hpcCBiZXR3ZWVuIG9yZ2FuaWMgYWNpZCBleGNyZXRpb24sIHN1Y3Jvc2UgbWV0YWJvbGlzbSBhbmQgZW5lcmd5IHN0YXR1czsgUGxhbnRhOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTIxMzsgIGNpdGF0aW9uX3BhZ2VzPTUzNC01NDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNsdXN0ZXIgcm9vdHPigJRhbiB1bmRlcmdyb3VuZCBhZGFwdGF0aW9uIGZvciBzdXJ2aXZhbCBpbiBleHRyZW1lIGVudmlyb25tZW50czsgVHJlbmRzIFBsYW50IFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTE2Mi0xNjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSByZWxhdGlvbnNoaXBzOiB0aGUgZ3Jhc3MgbW9kZWwgaW4gY3VycmVudCByZXNlYXJjaDsgUGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTYzNy02NDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIGdlbm9tZSBtYXBwaW5nIGluIEJyYXNzaWNhOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0xNDQ7ICBjaXRhdGlvbl9wYWdlcz0xOTAzLTE5MTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNlcmVhbCBnZW5vbWUgZXZvbHV0aW9u4oCUZ3Jhc3NlcywgbGluZSB1cCBhbmQgZm9ybSBhIGNpcmNsZTsgQ3Vyci4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTczNy03Mzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZGlyZWN0IGNvbXBhcmlzb24gYmV0d2VlbiB0aGUgZ2VuZXRpYyBtYXBzIG9mIHNvcmdodW0gYW5kIHJpY2U7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTAyOyAgY2l0YXRpb25fcGFnZXM9Mzc5LTM4NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXN0aW1hdGluZyBnZW5vbWUgY29uc2VydmF0aW9uIGJldHdlZW4gY3JvcCBhbmQgbW9kZWwgbGVndW1lIHNwZWNpZXM7IFBOQVM7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTAxOyAgY2l0YXRpb25fcGFnZXM9MTUyODktMTUyOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIG1hcHBpbmcgYmV0d2VlbiBNZWRpY2FnbyBzYXRpdmEgYW5kIFBpc3VtIHNhdGl2dW07IE1vbC4gR2VuZXQuIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTI3MjsgIGNpdGF0aW9uX3BhZ2VzPTIzNS0yNDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJyaWRnaW5nIG1vZGVsIGFuZCBjcm9wIGxlZ3VtZXMgdGhyb3VnaCBjb21wYXJhdGl2ZSBnZW5vbWljczsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTM3OyAgY2l0YXRpb25fcGFnZXM9MTE4OS0xMTk2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZmlyc3QgZ2VuZS1iYXNlZCBtYXAgb2YgTHVwaW51cyBhbmd1c3RpZm9saXVzIEwuLWxvY2F0aW9uIG9mIGRvbWVzdGljYXRpb24gZ2VuZXMgYW5kIGNvbnNlcnZlZCBzeW50ZW55IHdpdGggTWVkaWNhZ28gdHJ1bmNhdHVsYTsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTM7ICBjaXRhdGlvbl9wYWdlcz0yMjUtMjM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaWZmZXJlbmNlcyBpbiBzeW50ZW5pYyBjb21wbGV4aXR5IGJldHdlZW4gTWVkaWNhZ28gdHJ1bmNhdHVsYSB3aXRoIExlbnMgY3VsaW5hcmlzIGFuZCBMdXBpbnVzIGFsYnVzOyBGdW5jdC4gUGxhbnQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz03NzUtNzgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJpc29uIG9mIGEgQnJhc3NpY2Egb2xlcmFjZWEgZ2VuZXRpYyBtYXAgd2l0aCB0aGUgZ2Vub21lIG9mIEFyYWJpZG9wc2lzIHRoYWxpYW5hOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNjQ7ICBjaXRhdGlvbl9wYWdlcz0zNTktMzcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHRlbnNpdmUgY29uc2VydmF0aW9uIG9mIGxpbmthZ2UgcmVsYXRpb25zaGlwcyBiZXR3ZWVuIHBlYSBhbmQgbGVudGlsIGdlbmV0aWMgbWFwczsgSi4gSGVyZWQuOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTgzOyAgY2l0YXRpb25fcGFnZXM9MTIzLTEyOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXh0ZW5zaXZlIG1hY3Jvc3ludGVueSBiZXR3ZWVuIE1lZGljYWdvIHRydW5jYXR1bGEgYW5kIExlbnMgY3VsaW5hcmlzIHNzcC4gY3VsaW5hcmlzOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTExNDsgIGNpdGF0aW9uX3BhZ2VzPTU0OS01NTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZ1bmN0aW9uYWwgbWFwcGluZyBpbiBwZWEsIGFzIGFuIGFpZCB0byB0aGUgY2FuZGlkYXRlIGdlbmUgc2VsZWN0aW9uIGFuZCBmb3IgaW52ZXN0aWdhdGluZyBzeW50ZW55IHdpdGggdGhlIG1vZGVsIGxlZ3VtZSBNZWRpY2FnbyB0cnVuY2F0dWxhOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTExMjsgIGNpdGF0aW9uX3BhZ2VzPTEwMjQtMTA0MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgb2YgbnVjbGVhciBnZW5lLWRlcml2ZWQgbW9sZWN1bGFyIG1hcmtlcnMgbGlua2VkIHRvIGxlZ3VtZSBnZW5ldGljIG1hcHM7IE1vbC4gR2VuZXQuIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTI3NjsgIGNpdGF0aW9uX3BhZ2VzPTU2LTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHNlcXVlbmNlLWJhc2VkIGdlbmV0aWMgbWFwIG9mIE1lZGljYWdvIHRydW5jYXR1bGEgYW5kIGNvbXBhcmlzb24gb2YgbWFya2VyIGNvbGluZWFyaXR5IHdpdGggTS4gc2F0aXZhOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjY7ICBjaXRhdGlvbl9wYWdlcz0xNDYzLTE1MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1lZGljYWdvIHRydW5jYXR1bGEgRVNULVNTUnMgcmV2ZWFsIGNyb3NzLXNwZWNpZXMgZ2VuZXRpYyBtYXJrZXJzIGZvciBNZWRpY2FnbyBzcHA7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTA4OyAgY2l0YXRpb25fcGFnZXM9NDE0LTQyMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q3Jvc3Mtc3BlY2llcyBhbXBsaWZpY2F0aW9uIG9mIE1lZGljYWdvIHRydW5jYXR1bGEgbWljcm9zYXRlbGxpdGVzIGFjcm9zcyB0aHJlZSBtYWpvciBwdWxzZSBjcm9wczsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTA7ICBjaXRhdGlvbl9wYWdlcz0xMjEwLTEyMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFGTFA6IGEgbmV3IHRlY2huaXF1ZSBmb3IgRE5BIGZpbmdlcnByaW50aW5nOyBOdWNsLiBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9NDQwNy00NDE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TU1IgYW5hbHlzaXMgb2YgdGhlIE1lZGljYWdvIHRydW5jYXR1bGEgU0FSREkgY29yZSBjb2xsZWN0aW9uIHJldmVhbHMgc3Vic3RhbnRpYWwgZGl2ZXJzaXR5IGFuZCB1bnVzdWFsIGdlbm90eXBlIGRpc3BlcnNhbCB0aHJvdWdob3V0IHRoZSBNZWRpdGVycmFuZWFuIGJhc2luOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTExMjsgIGNpdGF0aW9uX3BhZ2VzPTk3Ny05ODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcCBNYW5hZ2VyIFFUWCwgY3Jvc3MtcGxhdGZvcm0gc29mdHdhcmUgZm9yIGdlbmV0aWMgbWFwcGluZzsgTWFtbS4gR2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9OTMwLTkzMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhyb3VnaCBhIGdlbm9tZSwgZGFya2x5OiBjb21wYXJhdGl2ZSBhbmFseXNpcyBvZiBwbGFudCBjaHJvbW9zb21hbCBETkE7IFBsYW50IE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9NDg7ICBjaXRhdGlvbl9wYWdlcz01LTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgYSBnZW5ldGljIGxpbmthZ2UgbWFwIHVzaW5nIE1GTFAgYW5kIGlkZW50aWZpY2F0aW9uIG9mIG1vbGVjdWxhciBtYXJrZXJzIGxpbmtlZCB0byBkb21lc3RpY2F0aW9uIGdlbmVzIGluIG5hcnJvdy1sZWFmZWQgbHVwaW4gKEx1cGludXMgYW5ndXN0aWZvbGl1cyBMLik7IENlbGwuIE1vbC4gQmlvbC4gTGV0dC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0zMzEtMzQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBpbnRyYXNwZWNpZmljIGxpbmthZ2UgbWFwIG9mIHRoZSBjaGlja3BlYSAoQ2ljZXIgYXJpZXRpbnVtIEwuKSBnZW5vbWUgYmFzZWQgb24gc2VxdWVuY2UgdGFnZ2VkIG1pY3Jvc2F0ZWxsaXRlIHNpdGUgYW5kIHJlc2lzdGFuY2UgZ2VuZSBhbmFsb2cgbWFya2VyczsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDY7ICBjaXRhdGlvbl9wYWdlcz0xNDQ3LTE0NTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFwcGxpY2F0aW9uIG9mIGJpb3RlY2hub2xvZ3kgaW4gYnJlZWRpbmcgbGVudGlsIGZvciByZXNpc3RhbmNlIHRvIGJpb3RpYyBhbmQgYWJpb3RpYyBzdHJlc3M7IEV1cGh5dGljYTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNDc7ICBjaXRhdGlvbl9wYWdlcz0xNDktMTY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWNvbWJpbmFudCBpbmJyZWQgbGluZXMgZm9yIGdlbmV0aWMgbWFwcGluZyBpbiB0b21hdG87IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9OTA7ICBjaXRhdGlvbl9wYWdlcz01NDItNTQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXBwaW5nIGdlbmV0aWMgZmFjdG9ycyBjb250cm9sbGluZyBwb2xsZW4gdmlhYmlsaXR5IGluIGFuIGludGVyc3BlY2lmaWMgY3Jvc3MgaW4gSGVsaWFudGh1cyBzZWN0LiBIZWxpYW50aHVzOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTkxOyAgY2l0YXRpb25fcGFnZXM9MTE5NS0xMjAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIG1hcHBpbmcgb2YgYW4gYW5jaWVudCB0cmFuc2xvY2F0aW9uIGluIHRoZSBnZW51cyBMZW5zOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTczOyAgY2l0YXRpb25fcGFnZXM9ODgzLTg5MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hyb21vc29tYWwgcmVnaW9ucyBhc3NvY2lhdGVkIHdpdGggc2VncmVnYXRpb24gZGlzdG9ydGlvbiBvZiBtb2xlY3VsYXIgbWFya2VycyBpbiBGMiwgYmFja2Nyb3NzLCBkb3VibGVkIGhhcGxvaWQsIGFuZCByZWNvbWJpbmFudCBpbmJyZWQgcG9wdWxhdGlvbnMgaW4gcmljZSAoT3J5emEgc2F0aXZhIEwuKTsgTW9sLiBHZW5ldC4gR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjUzOyAgY2l0YXRpb25fcGFnZXM9NTM1LTU0NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5bG9nZW55IGFuZCBnZW5vbWljIG9yZ2FuaXphdGlvbiBvZiB0aGUgVElSIGFuZCBub24tVElSIE5CUy1MUlIgcmVzaXN0YW5jZSBnZW5lIGZhbWlseSBpbiBNZWRpY2FnbyB0cnVuY2F0dWxhOyBNb2wuIFBsYW50IE1pY3JvYmUgSW50ZXJhY3QuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9NTI5LTM1OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBiYXNpcyBvZiBhZGFwdGF0aW9uOiBmbG93ZXJpbmcgdGltZSBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT05NTsgIGNpdGF0aW9uX3BhZ2VzPTU3My01ODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcHBpbmcgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgaW4gY2hpY2twZWEgYXNzb2NpYXRlZCB3aXRoIHRpbWUgdG8gZmxvd2VyaW5nIGFuZCByZXNpc3RhbmNlIHRvIERpZHltZWxsYSByYWJpZWkgdGhlIGNhdXNhbCBhZ2VudCBvZiBBc2NvY2h5dGEgYmxpZ2h0OyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTExMzsgIGNpdGF0aW9uX3BhZ2VzPTEzNTctMTM2OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gaW5mb3JtYXRpdmUgbGlua2FnZSBtYXAgb2Ygc295YmVhbiByZXZlYWxzIFFUTHMgZm9yIGZsb3dlcmluZyB0aW1lLCBsZWFmbGV0IG1vcnBob2xvZ3kgYW5kIHJlZ2lvbnMgb2Ygc2VncmVnYXRpb24gZGlzdG9ydGlvbjsgRE5BIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9NjEtNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1DUkMgUHJlc3MsIEJvY2EgUmF0b247IFN0YXRpc3RpY2FsIEdlbm9taWNzOiBMaW5rYWdlLCBNYXBwaW5nIGFuZCBRVEwgQW5hbHlzaXM7ICBjaXRhdGlvbl95ZWFyPTE5ODU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcHBpbmcgcG9seWdlbmVzOyBBbm51YWwgUmV2aWV3IG9mIEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MjA1LTIzMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpY2FsIGNvbnRyb2wgb2YgYWxrYWxvaWRzIGluIEx1cGludXMgYWxidXM7IEV1cGh5dGljYTsgIGNpdGF0aW9uX3llYXI9MTk4MjsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTM1Ny0zNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIHBoeXNpY2FsIG1hcHBpbmcgcmV2ZWFscyBmZWF0dXJlcyBvZiBtaWNyb3N5bnRlbnkgYmV0d2VlbiBHbHljaW5lIG1heCwgTWVkaWNhZ28gdHJ1bmNhdHVsYSwgYW5kIEFyYWJpZG9wc2lzIHRoYWxpYW5hOyBHZW5vbWU7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NDc7ICBjaXRhdGlvbl9wYWdlcz0xNDEtMTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBnZW5vbWljcyBlbmFibGVkIHRoZSBpc29sYXRpb24gb2YgdGhlIFIzYSBsYXRlIGJsaWdodCByZXNpc3RhbmNlIGdlbmUgaW4gcG90YXRvOyBQbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPTI1MS0yNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIHBlcmVubmlhbCByeWVncmFzcyAoTG9saXVtIHBlcmVubmUgKEwuKSkgYW5kIG1lYWRvdyBmZXNjdWUgKEZlc3R1Y2EgcHJhdGVuc2lzIChIdWRzLikpIGNhbmRpZGF0ZSBvcnRob2xvZ291cyBzZXF1ZW5jZXMgdG8gdGhlIHJpY2UgSGQxKFNlMSkgYW5kIGJhcmxleSBIdkNPMSBDT05TVEFOUy1saWtlIGdlbmVzIHRocm91Z2ggY29tcGFyYXRpdmUgbWFwcGluZyBhbmQgbWljcm9zeW50ZW55OyBOZXcgUGh5dG9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjc7ICBjaXRhdGlvbl9wYWdlcz0yMzktMjQ3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBnZW5ldGljIG1hcHBpbmcgb2YgbG9jaSBlZmZlY3RpbmcgcGxhbnQgaGVpZ2h0IGFuZCBkZXZlbG9wbWVudCBpbiBjZXJlYWxzOyBFdXBoeXRpY2E7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTAwOyAgY2l0YXRpb25fcGFnZXM9MjQ1LTI0ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC1yZXNvbHV0aW9uIG1hcHBpbmcgb2YgdGhlIGJhcmxleSBsZWFmIHJ1c3QgcmVzaXN0YW5jZSBnZW5lIFJwaDUgdXNpbmcgYmFybGV5IGV4cHJlc3NlZCBzZXF1ZW5jZSB0YWdzIChFU1RzKSBhbmQgc3ludGVueSB3aXRoIHJpY2U7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTExOyAgY2l0YXRpb25fcGFnZXM9MTY1MS0xNjYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBtYXBwaW5nIGFuZCBtYXJrZXItYXNzaXN0ZWQgc2VsZWN0aW9uIGluIFJvc2FjZWFlIGZydWl0IGNyb3BzOyBQTkFTOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwMTsgIGNpdGF0aW9uX3BhZ2VzPTk4OTEtOTg5NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE0LzIvNTkvMTc0NjI2ODEvZHNtMDA5LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBXZSByZXBvcnQgdGhlIGZpcnN0IGdlbmV0aWMgbGlua2FnZSBtYXAgb2Ygd2hpdGUgbHVwaW4gKEx1cGludXMgYWxidXMgTC4pLiBBbiBGOCByZWNvbWJpbmFudCBpbmJyZWQgbGluZSBwb3B1bGF0aW9uIGRldmVsb3BlZCBmcm9tIEtpZXYgbXV0YW50IMOXIFAyNzEiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTMzMzk3OSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGTJmunHMYKs8ut+vkqLlGNg9raHNQ==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f2%2f59%2f333979" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f2%2f59%2f333979" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=12722" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/14/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/2/0/m_dnaresearch14_2.cover.gif?Expires=2147483647&amp;Signature=5BTa-~goe1WzTtgupy8W5naLIOKYXKJijMDT9~iND6E57C9GS8DnKmgf4xLGiAjAgFQ33xxQGY5XEA2IyH5sxNAchcYGydjD62Blbb3HoCSwzq-ZA0-QrYedFeiN-nl~F4fWgD1eJCN2K69ZEHoUWQqIDlDDD1JhHQnJNcB5lrrNLK6wlcMVRgK0aqJ1Xfk9avS0yFPUV2wTYbI3OOMUsYWIppy7d-BlzWSWPoBfkZar9pX9D9gb52oz134NZ~DyHw-3qm~BemwxBV-O7Uaglh2bFd3cqoLv7fidd1f5Wb74aH7JVK94AzFTLiZ-L8A3WqOO7o6HKGSWlUHJrR5JiA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/14/2">
                        <div class="volume trailing-comma">Volume 14</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
2007        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87195707">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195707">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195709">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195709">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195716">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195716">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195736">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195736">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195774">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195774">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195776">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195776">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/14/2/47/333882">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/14/2/71/333534">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">The First Genetic and Comparative Map of White Lupin (<em>Lupinus albus</em> L.): Identification of QTLs for Anthracnose Resistance and Flowering Time, and a Locus for Alkaloid Content
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Huyen T. T. Phan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Australian Centre for Necrotrophic Fungal Pathogens, SABC, Department of Health Sciences</div>, <div class="insititution">Murdoch University</div>, <div class="addr-line">Murdoch 6150</div>, <div class="country">Australia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Huyen+T.+T.+Phan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Phan H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Phan H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Huyen T. T. Phan</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Simon R. Ellwood</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Australian Centre for Necrotrophic Fungal Pathogens, SABC, Department of Health Sciences</div>, <div class="insititution">Murdoch University</div>, <div class="addr-line">Murdoch 6150</div>, <div class="country">Australia</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +61 08-9360-6323. Fax. <div class="fax">+61 08-9360-6303</div>. E-mail: <a href="mailto:s.ellwood@murdoch.edu.au" target="_blank">s.ellwood@murdoch.edu.au</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Simon+R.+Ellwood">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ellwood S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ellwood S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Simon R. Ellwood</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kedar Adhikari</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Agriculture and Food Western Australia, 3 Baron-Hay Court, South Perth 6151</div>, <div class="country">Australia</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Centre for Legumes in Mediterranean Agriculture, Faculty of Natural and Agricultural Sciences</div>, <div class="insititution">University of Western Australia</div>, <div class="addr-line">Crawley 6009</div>, <div class="country">Australia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kedar+Adhikari">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Adhikari K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Adhikari K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kedar Adhikari</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Matthew N. Nelson</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">School of Plant Biology, Faculty of Natural and Agricultural Sciences</div>, <div class="insititution">The University of Western Australia</div>, <div class="addr-line">Crawley 6009</div>, <div class="country">Australia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Matthew+N.+Nelson">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nelson M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nelson M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Matthew N. Nelson</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Richard P. Oliver</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Australian Centre for Necrotrophic Fungal Pathogens, SABC, Department of Health Sciences</div>, <div class="insititution">Murdoch University</div>, <div class="addr-line">Murdoch 6150</div>, <div class="country">Australia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Richard+P.+Oliver">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Oliver R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Oliver R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Richard P. Oliver</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Communicated by Satoshi Tabata</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 14, Issue 2, 1 January 2007, Pages 59–70, <a href='https://doi.org/10.1093/dnares/dsm009'>https://doi.org/10.1093/dnares/dsm009</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">26 May 2007</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">07 March 2007</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Received:</div>
                                <div class="wi-date">13 April 2007</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="333979" href="/dnaresearch/article-pdf/14/2/59/17462681/dsm009.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Huyen T. T. Phan, Simon R. Ellwood, Kedar Adhikari, Matthew N. Nelson, Richard P. Oliver;  The First Genetic and Comparative Map of White Lupin (<em>Lupinus albus</em> L.): Identification of QTLs for Anthracnose Resistance and Flowering Time, and a Locus for Alkaloid Content, <em>DNA Research</em>, Volume 14, Issue 2, 1 January 2007, Pages 59–70, <a href='https://doi.org/10.1093/dnares/dsm009'>https://doi.org/10.1093/dnares/dsm009</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/333979?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/333979?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/333979?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/333979?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/333979?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/333979?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/333979?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=The%20First%20Genetic%20and%20Comparative%20Map%20of%20White%20Lupin%20(%26lt%3Bem%26gt%3BLupinus%20albus%26lt%3B%2Fem%26gt%3B%20L.)%3A%20Identification%20of%20QTLs%20for%20Anthracnose%20Resistance%20and%20Flowering%20Time%2C%20and%20a%20Locus%20for%20Alkaloid%20Content&amp;publicationDate=2007-05-26&amp;volumeNum=14&amp;issueNum=2&amp;author=Phan%2C%20Huyen%20T.%20T.%3B%20Ellwood%2C%20Simon%20R.&amp;startPage=59&amp;endPage=70&amp;contentId=10.1093%2Fdnares%2Fdsm009&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f2%2f59%2f333979" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87195707 id="87195707" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>We report the first genetic linkage map of white lupin (<em>Lupinus albus</em> L.). An F8 recombinant inbred line population developed from Kiev mutant × P27174 was mapped with 220 amplified fragment length polymorphism and 105 gene-based markers. The genetic map consists of 28 main linkage groups (LGs) that varied in length from 22.7 cM to 246.5 cM and spanned a total length of 2951 cM. There were seven additional pairs and 15 unlinked markers, and 12.8% of markers showed segregation distortion at <em>P</em> &lt; 0.05. Syntenic relationships between <em>Medicago truncatula</em> and <em>L. albus</em> were complex. Forty-five orthologous markers that mapped between <em>M. truncatula</em> and <em>L. albus</em> identified 17 small syntenic blocks, and each <em>M. truncatula</em> chromosome aligned to between one and six syntenic blocks in <em>L. albus</em>. Genetic mapping of three important traits: anthracnose resistance, flowering time, and alkaloid content allowed loci governing these traits to be defined. Two quantitative trait loci (QTLs) with significant effects were identified for anthracnose resistance on LG4 and LG17, and two QTLs were detected for flowering time on the top of LG1 and LG3. Alkaloid content was mapped as a Mendelian trait to LG11.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Lupinus albus&quot;"><em>Lupinus albus</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Medicago truncatula&quot;"><em>Medicago truncatula</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="synteny">synteny</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="anthracnose">anthracnose</a></div>                            <h2 scrollto-destination=87195709 id="87195709" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p><em>Lupinus</em> is a genus of self- or cross-pollinating, mostly indeterminate plant species native to diverse geographic regions.<sup><a reveal-id="DSM009C1" data-open="DSM009C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSM009C2" data-open="DSM009C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Lupins are considered to be polyploid in origin and are found in both the New and Old Worlds. Old World species exhibit different basic chromosome numbers,<sup><a reveal-id="DSM009C3" data-open="DSM009C3" class="link link-ref link-reveal xref-bibr">3</a></sup> a large variation in the number of their chromosomes, and a 2.5-fold variation in their estimated 2C DNA values.<sup><a reveal-id="DSM009C4" data-open="DSM009C4" class="link link-ref link-reveal xref-bibr">4</a></sup> They are subdivided into the Scabrispermae (rough-seeded) and the Malacospermae (smooth-seeded) to which white lupin (<em>Lupinus albus</em> L.) belongs in conjunction with two other cultivated species: <em>L. angustifolius</em> and <em>L. luteus</em>. White lupin has a diploid chromosome number of 50 and a 2C DNA content of 1.16 ± 0.044 pg.<sup><a reveal-id="DSM009C4" data-open="DSM009C4" class="link link-ref link-reveal xref-bibr">4</a></sup></p><p>Lupin species have a relatively short domestication history compared with most crops. Focused breeding efforts began in Germany during World War I due to a need for high-protein pulse crops adapted to temperate conditions. Subsequent breeding has concentrated on the introduction of key traits such as early flowering, reduced pod-shattering, soft seed, and anthracnose disease resistance.<sup><a reveal-id="DSM009C5" data-open="DSM009C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSM009C6" data-open="DSM009C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Lupin grain is high in protein (30–40%) and like soybean is high in dietary fibre (30%), low in oil (6%), and contains minimal starch. Lupin has the lowest Glycaemic Index of any commonly consumed grain (<span class="underline"><a class="link link-uri" href="http://www.glycemicindex.com">www.glycemicindex.com</a></span>), which has significant implications for Western societies with an increasing incidence of obesity and associated risk of diabetes and cardiovascular disease.<sup><a reveal-id="DSM009C7" data-open="DSM009C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Lupin fibre acts as a soluble fibre and drops total cholesterol levels without affecting beneficial HDL cholesterol.<sup><a reveal-id="DSM009C8" data-open="DSM009C8" class="link link-ref link-reveal xref-bibr">8</a></sup></p><p>White lupin itself is used as feed for livestock and has established a growing market for human consumption due to the development of low alkaloid varieties with a lack of protease inhibitors.<sup><a reveal-id="DSM009C9" data-open="DSM009C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSM009C10" data-open="DSM009C10" class="link link-ref link-reveal xref-bibr">10</a></sup> In addition, white lupin has become an illuminating model for the study of plant adaptation to extreme phosphorus (P) deficiency.<sup><a reveal-id="DSM009C11" data-open="DSM009C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSM009C12" data-open="DSM009C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Adaptation to low P is linked to modifications of root development and biochemistry resulting in proteoid (or cluster) roots.<sup><a reveal-id="DSM009C13  DSM009C15" data-open="DSM009C13  DSM009C15" class="link link-ref link-reveal xref-bibr">13–15</a></sup></p><p>Comparative genetic mapping has been studied in many families of crops such as cereals and crucifers,<sup><a reveal-id="DSM009C16  DSM009C19" data-open="DSM009C16  DSM009C19" class="link link-ref link-reveal xref-bibr">16–19</a></sup> and also in legumes.<sup><a reveal-id="DSM009C20  DSM009C22" data-open="DSM009C20  DSM009C22" class="link link-ref link-reveal xref-bibr">20–22</a></sup> More recently, the first comparative map in <em>Lupinus</em> was developed between <em>L. angustifolius</em> and the model legume <em>Medicago truncatula</em>,<sup><a reveal-id="DSM009C23" data-open="DSM009C23" class="link link-ref link-reveal xref-bibr">23</a></sup> and subsequently a preliminary comparison was reported between <em>L. albus</em> and <em>M. truncatula.</em><sup><a reveal-id="DSM009C24" data-open="DSM009C24" class="link link-ref link-reveal xref-bibr">24</a></sup></p><p>Initial comparative mapping studies predominantly used common enzymes, morphological markers, and RFLP probes.<sup><a reveal-id="DSM009C25" data-open="DSM009C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DSM009C26" data-open="DSM009C26" class="link link-ref link-reveal xref-bibr">26</a></sup> These have been superseded by PCR-based codominant marker systems that have markedly increased the efficiency and reduced the cost of transferring genetic information across species. In this approach, oligonucleotide primers are designed from sequences of conserved regions such as gene exons that span polymorphic introns or microsatellites to produce ITAPs (intron targeted amplified polymorphic sequence). Examples include the comparison of <em>Medicago truncatula</em> with alfalfa, chickpea, faba bean, narrow-leafed lupin, and pea.<sup><a reveal-id="DSM009C23" data-open="DSM009C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSM009C27  DSM009C32" data-open="DSM009C27  DSM009C32" class="link link-ref link-reveal xref-bibr">27–32</a></sup></p><p>To date only a preliminary genetic map showing three linkage groups (LGs) has been published for white lupin.<sup><a reveal-id="DSM009C24" data-open="DSM009C24" class="link link-ref link-reveal xref-bibr">24</a></sup> The aims of this study were (1) to construct a complete genetic map of <em>L. albus</em> using orthologous gene-based codominant markers and amplified fragment length polymorphism (AFLP)<sup><a reveal-id="DSM009C33" data-open="DSM009C33" class="link link-ref link-reveal xref-bibr">33</a></sup> markers, as a resource for comparing lupin genomes and in legume evolution studies; (2) to map key agronomic traits including anthracnose resistance, low alkaloid content, and flowering time; and (3) to assess the extent of conserved synteny between <em>L. albus</em> and the model legume <em>M. truncatula</em>.</p>                            <h2 scrollto-destination=87195716 id="87195716" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87195717 id="87195717" class="section-title" data-legacy-id=s2a>2.1. Genetic mapping population</h3>
<p>One hundred and ninety-five F8-derived recombinant inbred lines (RILs) were used to phenotype traits. This population was developed from a cross between cv. Kiev Mutant and an Ethiopian landrace P27174. Kiev Mutant was bred by Dr V.I. Golovchenko of the Ukrainian Agricultural Research Institute, Kiev and was first released in the USSR in 1969. Kiev Mutant was first introduced into Australia by CSIRO in 1972 and subsequently released by the Western Australian Department of Agriculture and Food in 1982. This variety has low alkaloid content and is early flowering but is highly susceptible to anthracnose. P27174 has high alkaloid levels (&lt;1%), is late flowering, and is resistant to anthracnose disease.</p><p>To produce the RIL population, Kiev Mutant was used as the female parent and P27174 as the male. The F8 RIL population was primarily made to identify molecular markers linked to anthracnose resistance genes, but variation also exists for other traits such as flowering time, alkaloid content, plant height, pod shape, order of podding, branching, leaf colour and size, brown markings on stems and yield. Only the first three characters were phenotyped for this study. From the 195 F8 RILs, a subset of 94 was randomly chosen for comparative mapping. Total genomic DNA was extracted as previously described by Ellwood et al.<sup><a reveal-id="DSM009C34" data-open="DSM009C34" class="link link-ref link-reveal xref-bibr">34</a></sup></p>                            <h3 scrollto-destination=87195720 id="87195720" class="section-title" data-legacy-id=s2b>2.2. Anthracnose resistance phenotyping</h3>
<p>The F8 lines were tested against anthracnose from 2004 to 2005 in a disease nursery. The nursery was established in a high rainfall site at Medina, Western Australia, and supplemented with irrigation as required. Spreader rows consisting of a mixture of the susceptible cvs. Kiev Mutant, Wodjil, and Myallie were planted a week before the test lines. The whole population consisting of 195 RILs together with parental controls were planted in 2 m long rows in three replicates between the spreader rows. The test rows were perpendicular to the spreader rows to enable flow of the disease inoculum from both sides. After 3–4 weeks of planting, glasshouse infected Kiev Mutant seedlings were transplanted into the spreader rows at approximately 1 m spacing to ensure a high level of early inoculum. The disease was assessed twice during the season employing a score of 1–5, where 1 is highly resistant and 5 is extremely susceptible. The second, usually higher, scores were used to determine the resistance level of a line.</p>                            <h3 scrollto-destination=87195722 id="87195722" class="section-title" data-legacy-id=s2c>2.3. Flowering time</h3>
<p>Flowering time of F8 RILs was recorded when 50% plants in a test row had one open flower with an erect standard petal on the main stem inflorescence. Due to a large variation in flowering times among lines, the flowering time for each line was recorded in weeks, rather than days, relative to the earliest flowering parent Kiev Mutant. For example, lines that flowered as early as Kiev Mutant were categorized into week 1.</p>                            <h3 scrollto-destination=87195724 id="87195724" class="section-title" data-legacy-id=s2d>2.4. Alkaloid content</h3>
<p>The F8 lines were scored for alkaloid content using two techniques, the Dragendorff paper test and by ultra violet (UV) fluorescence. Pink discolouration of Dragendorff paper indicated high alkaloid content and a white or bleached discolouration indicated no or low alkaloids. Tests were conducted using sap from the petiole of a young leaflet from seedlings at the 8–9 leaf-stage. Similarly, after harvesting seeds were viewed under an UV lamp (Vilber Lourmat, France) at 365 nm. Seeds with a high alkaloid content are fluorescent pink under UV light, whereas those with low alkaloids levels are dull whitish.</p>                            <h3 scrollto-destination=87195726 id="87195726" class="section-title" data-legacy-id=s2e>2.5. Gene-based PCR markers</h3>
<p>Six hundred and twenty-six genic markers belonging to three sets of ITAP primers were utilized in this study.<sup><a reveal-id="DSM009C27" data-open="DSM009C27" class="link link-ref link-reveal xref-bibr">27</a></sup> These were composed of three sets of ITAPs; the ‘ML’ primers, that were developed from alignment between <em>M. truncatula</em> and <em>Lupinus</em> spp. EST sequences.<sup><a reveal-id="DSM009C23" data-open="DSM009C23" class="link link-ref link-reveal xref-bibr">23</a></sup> A small proportion of these markers were designed to span microsatellite motifs rather than introns; the ‘MLG’ primers were designed from alignment of <em>M. truncatula</em>, <em>L. albus</em>, and <em>Glycine max</em> EST sequences;<sup><a reveal-id="DSM009C27" data-open="DSM009C27" class="link link-ref link-reveal xref-bibr">27</a></sup> and the cross-species ‘MP’ markers developed at the Department of Plant Pathology, University of California, Davis, USA.<sup><a reveal-id="DSM009C20" data-open="DSM009C20" class="link link-ref link-reveal xref-bibr">20</a></sup> The majority of the ITAP primers were from genes in characterized chromosomal regions, and therefore the <em>L. albus</em> mapped markers could be physically mapped in <em>M. truncatula</em> using <em>M. truncatula</em> BACs.</p>                            <h3 scrollto-destination=87195728 id="87195728" class="section-title" data-legacy-id=s2f>2.6. Polymorphism detection</h3>
<p>Each primer pair was screened on <em>L. albus</em> parental DNA.<sup><a reveal-id="DSM009C23" data-open="DSM009C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Single PCR products of the same size were purified and directly sequenced. DNA polymorphisms were identified by manual inspection of alignments and chromatograms using Vector NTI (Invitrogen, Carlsbad, California). Different procedures were used to genotype the F8 population depending on types of polymorphism identified.<sup><a reveal-id="DSM009C27" data-open="DSM009C27" class="link link-ref link-reveal xref-bibr">27</a></sup></p>                            <h3 scrollto-destination=87195730 id="87195730" class="section-title" data-legacy-id=s2g>2.7. AFLP markers</h3>
<p>AFLP markers were generated as described by Nelson et al.<sup><a reveal-id="DSM009C23" data-open="DSM009C23" class="link link-ref link-reveal xref-bibr">23</a></sup> and Vos et al.,<sup><a reveal-id="DSM009C33" data-open="DSM009C33" class="link link-ref link-reveal xref-bibr">33</a></sup> using one selective nucleotide for pre-amplification (<em>Eco</em>RI-C and <em>Mse</em>I-A) and two selective nucleotides for selective amplification. The AFLP products were resolved with an AB3730 DNA Sequencer using a GeneScan™ -500 LIZ<sup>®</sup> Size Standard [Applied Biosystems (AB), Foster City, California, USA]. Data were analysed by the automated allele calling parameters in GeneMapper<sup>®</sup> (AB). Marker nomenclature included the name of selective marker combination; either A or B depending on whether inherited from Kiev mutant or P27274, respectively; and the marker size (in bp).</p>                            <h3 scrollto-destination=87195732 id="87195732" class="section-title" data-legacy-id=s2h>2.8. Genetic, comparative, and QTL mapping</h3>
<p>Ninety-four F8 RILs were scored for polymorphic gene-based and AFLP molecular markers together with three agronomic traits (anthracnose resistance, alkaloid, and flowering time). Genetic linkage mapping was conducted with MultiPoint v. 1.2 software (MultiQTL Ltd, Institute of Evolution, Haifa University, Israel). The Kosambi mapping function was used to convert the recombination frequencies into genetic distance (cM). Chi-square analysis (<em>P</em> &lt; 0.05) was applied to test the segregation of the mapped markers against the expected Mendelian segregation ratio for codominant inheritance in an F8 RIL population. Groups of linked markers that were similarly distorted were accepted for linkage mapping and QTL analysis. Independent markers showing significant segregation distortion and markers with missing data (&gt;10%) were rejected for linkage and QTL analysis to avoid bias and false linkages.</p><p>Markers mapped in <em>L. albus</em> were located on the <em>M. truncatula</em> map by aligning marker EST template sequences using BLASTN to phases 1, 2 and 3 sequenced BACs (Build 17.06.06, <a class="link link-uri" href="http://www.medicago.org/">http://www.medicago.org/</a>). Alignments with a BLAST E-values &lt; 1 × 10<sup>−20</sup>, hsp identity ≥ 60%, and hsp length &gt; 50 nt were retained.</p><p>Association between each LG and the putative QTL regions related to flowering time and anthracnose resistance were determined by interval mapping using MultiQTL software, version 2.5 (MultiQTL Ltd, Institute of Evolution, Haifa University, Israel). Significance levels for each QTL detected were determined with 1000 permutations and the standard deviation (SD) for each QTL determined by 1000 Bootstraps. Analysis of variance (ANOVA) using JMP-IN 5.1 (SAS Institute, Cary, NC) was also used to determine correlation between markers flanking each QTL and the corresponding trait. All QTLs identified were further confirmed with MapManager QTX v. b20.<sup><a reveal-id="DSM009C35" data-open="DSM009C35" class="link link-ref link-reveal xref-bibr">35</a></sup></p>                            <h2 scrollto-destination=87195736 id="87195736" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=87195737 id="87195737" class="section-title" data-legacy-id=s3a>3.1. General features of the first L. albus genetic linkage map</h3>
<p>A total of 325 markers consisting of 105 genic markers and 220 AFLP markers were used to generate the first comprehensive genetic linkage map for <em>L. albus</em>. Amplicon sequences have been submitted to Genbank under accession codes EI394666, EI394667, and EI392791–EI392938.</p><p>The multilocus AFLPs were employed to generate sufficient markers to coalesce the LGs described by gene-based markers. This was necessary due to the large number of chromosomes in <em>L. albus</em>, (<em>n</em> = 25),<sup><a reveal-id="DSM009C4" data-open="DSM009C4" class="link link-ref link-reveal xref-bibr">4</a></sup> and the tendency of conserved genes as genic markers to cluster in isolated blocks.<sup><a reveal-id="DSM009C36" data-open="DSM009C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Genic markers were chosen to enable syntenic relationships between <em>L. albus</em> and the model legume species, <em>M. truncatula</em>, to be established.</p><p>Table <a reveal-id="DSM009TB1" data-open="DSM009TB1" class="link link-reveal link-table xref-fig">1</a> and Fig. <a reveal-id="DSM009F1" data-open="DSM009F1" class="link link-reveal link-table xref-fig">1</a> summarize general properties of the map. The map comprised 28 LGs, that varied in length from 246.5 to 22.7 cM and the total map length was 2951 cM. The average spacing between markers was 12.6 cM, determined at a recombination fraction (rf) of 0.27. In addition, there were seven pairs of linked markers. Unique placements within LGs could not be determined for 25 markers and 15 markers remained unlinked. Ambiguous markers positioning in Multipoint v. 1.2 may occur due to missing data, miss-genotyping, or rare recombination events that push out the length of the chromosome. The first two occurred in this study and are the result of residual heterozygosity in F8 lines. Thus, before mapping, ‘H's are removed from the data for co-dominant markers, while the dominant nature of AFLPs disguises otherwise co-dominant loci. Genic markers were evenly distributed throughout the genome with the exceptions of LGs 9, 14, and 28 which were composed only of AFLP markers. Only two examples of markers mapped to multiple loci occurred in this study: LSSR16 and LSSR33. Unlike RFLPs, ITAPs are limited in their ability to detect duplications. Exceptions occur when two or more amplicons for a given marker are of different lengths but share similar amplification intensities and are individually polymorphic between the parents.</p>                        <a id="87195741" scrollto-destination="87195741"></a>
<div id="DSM009F1" data-id="DSM009F1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/2/10.1093_dnares_dsm009/1/m_dsm00901a.jpeg?Expires=2147483647&amp;Signature=aXff6wA-YFVSfuPr2AByn5dSWdd5Hp0uCArAPMX5l~HW6jXD-WLwC8x~K030rl5A4hgi7B~wxnCmCiDfvJssLVXuFbl099D6oF02m-FTRxKBc5INutMevpjM0H5T97cTVxVyogyOckEUY-VZTnn~bh7acaaLTnIFcLEvQ6rw6KqvLdIsaIrcz5W76CxuRHZ5KNU5DGpAfig8SpP46~4seY5sLDRpNX3wGHtS3R4BYBYb3y-0sVz5DGzBM7c-KGr-mNZ3OyaN0ewtRdo8aRCCEeWEcKV4SKfbxDw1cAWsOP03iH-tmecWtE3NuAWsCljChMdwc5kOg8GrKVElBL1fHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The first genetic linkage map of L. albus. Marker distances in cM are provided on the vertical scale bars. Markers showing distorted segregation are indicated with an asterisk (*). Markers with ambiguous order described by Multipoint v. 1.2 are shown in a second column. Genomic locations of QTLs for flowering time (LG1 and 3) and anthracnose resistance (LG4 and 17) are highlighted to the left of LGs with criss-crossed and diagonally patterned boxes, respectively, and SDs depicted by lines to the sides. Refer to Table 3 for QTL parameters. The locus for alkaloid content on LG11 is indicated in bold." path-from-xml="dsm00901a.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195741/dsm00901a.jpeg" path-from-xml="dsm00901a.jpeg" target="_blank">View large</a><a section="87195741" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/2/10.1093_dnares_dsm009/1/dsm00901a.jpeg?Expires=2147483647&amp;Signature=ItVYmERTpIo5z~58bB81UJeSk7u8vwqt0uP7Fx6yNBrdMImc0NNL8dHbDkr~CwpQw65AHpcQNHH6I2ILCujWh6QuNc0G6vgGdAJ2EaKO-QXw8ULiXGpP6~s8yE4YZXLAeKKW0GEe5p~iKDkoq1zXhG1YC-PVl9EzwFZnkAmUIuZ5yWkA~8akCi5dM5mw8xzexIu04q3cCWm3~GVGqNUcBPIFq38GsPbz5h250BMwHufzilaXekXIXGn71ZFg5peRFKV9rkQb8BThAKSyqG9G4CvvJUSzg79VyY-I5IvE6S7uBvU4-E~pqo6r1tZlefMuNpXWtLmFm1KTan~npieggw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87195741&amp;ar=333979&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsm00901a.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/2/10.1093_dnares_dsm009/1/m_dsm00901b.jpeg?Expires=2147483647&amp;Signature=K1gklDFdZ807i8yGp6LHQmlJyvyZttKTRnmM7sErbevCs~lpVBio23jg-uRAnAlw0KGZcHFRQ~qApA8uHtVdZVNrGUR-20rmb-GwxKzO17mgoG8sOrEcKfGXtjhCFkKeTw7kPwMbH1gHM7nYNZLPeSM1tIh6vVquMoVRzRg8A4pXnKv3a1AvCVNA36P1MwglhTcQ0AMHNd1gE887r94XstPLJH8ocf09cKPIxNAh6Ujz5kobpglG7Az10G9oF2Uar-2YPb2iPK~m6HE2~HJGmaa6WXXsD5tys4MxGPOk~RmfcjQNc87MwBwORDWUZfZTSyrtZgXZEaWpMowWZfNT7Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The first genetic linkage map of L. albus. Marker distances in cM are provided on the vertical scale bars. Markers showing distorted segregation are indicated with an asterisk (*). Markers with ambiguous order described by Multipoint v. 1.2 are shown in a second column. Genomic locations of QTLs for flowering time (LG1 and 3) and anthracnose resistance (LG4 and 17) are highlighted to the left of LGs with criss-crossed and diagonally patterned boxes, respectively, and SDs depicted by lines to the sides. Refer to Table 3 for QTL parameters. The locus for alkaloid content on LG11 is indicated in bold." path-from-xml="dsm00901b.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195741/dsm00901b.jpeg" path-from-xml="dsm00901b.jpeg" target="_blank">View large</a><a section="87195741" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/2/10.1093_dnares_dsm009/1/dsm00901b.jpeg?Expires=2147483647&amp;Signature=MRrnbfsWJnGMpy-gPwSBtlk03f2hnxDSYmHk7hTAbRlEzhqrOaTPtgl4~jZYqPSCToDILm~p5mFyM7zUBHA6Vpn9mKm7k6bSk9PxApCY-lCiiXRugkxdkagplrsCl27x49wqHXxFi4EJ0wEDrlO~BELokdzB8f6A9L5-rDZ--LOna~4wL1ifyd5R4zAWmDFUHuIA8uQ0EyoeBzbuQbqIEaPYuqDAs-h2G5vQZ0uYM6hftkyMgnPe2eA~UtNiSQBEOfTb0m-9ANaai7SZ7HJ5FPF6akKyZ8UA2DYTL5qKoo6I~ZieL877dPje3uHcAEvrQg0vSw4Tared63j1MrMDxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87195741&amp;ar=333979&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsm00901b.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The first genetic linkage map of <em>L. albus</em>. Marker distances in cM are provided on the vertical scale bars. Markers showing distorted segregation are indicated with an asterisk (*). Markers with ambiguous order described by Multipoint v. 1.2 are shown in a second column. Genomic locations of QTLs for flowering time (LG1 and 3) and anthracnose resistance (LG4 and 17) are highlighted to the left of LGs with criss-crossed and diagonally patterned boxes, respectively, and SDs depicted by lines to the sides. Refer to Table <a reveal-id="DSM009TB3" data-open="DSM009TB3" class="link link-reveal link-table xref-fig">3</a> for QTL parameters. The locus for alkaloid content on LG11 is indicated in bold.</p></div></div><div content-id="DSM009F1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/2/10.1093_dnares_dsm009/1/m_dsm00901a.jpeg?Expires=2147483647&amp;Signature=aXff6wA-YFVSfuPr2AByn5dSWdd5Hp0uCArAPMX5l~HW6jXD-WLwC8x~K030rl5A4hgi7B~wxnCmCiDfvJssLVXuFbl099D6oF02m-FTRxKBc5INutMevpjM0H5T97cTVxVyogyOckEUY-VZTnn~bh7acaaLTnIFcLEvQ6rw6KqvLdIsaIrcz5W76CxuRHZ5KNU5DGpAfig8SpP46~4seY5sLDRpNX3wGHtS3R4BYBYb3y-0sVz5DGzBM7c-KGr-mNZ3OyaN0ewtRdo8aRCCEeWEcKV4SKfbxDw1cAWsOP03iH-tmecWtE3NuAWsCljChMdwc5kOg8GrKVElBL1fHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The first genetic linkage map of L. albus. Marker distances in cM are provided on the vertical scale bars. Markers showing distorted segregation are indicated with an asterisk (*). Markers with ambiguous order described by Multipoint v. 1.2 are shown in a second column. Genomic locations of QTLs for flowering time (LG1 and 3) and anthracnose resistance (LG4 and 17) are highlighted to the left of LGs with criss-crossed and diagonally patterned boxes, respectively, and SDs depicted by lines to the sides. Refer to Table 3 for QTL parameters. The locus for alkaloid content on LG11 is indicated in bold." path-from-xml="dsm00901a.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195741/dsm00901a.jpeg" path-from-xml="dsm00901a.jpeg" target="_blank">View large</a><a section="87195741" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/2/10.1093_dnares_dsm009/1/dsm00901a.jpeg?Expires=2147483647&amp;Signature=ItVYmERTpIo5z~58bB81UJeSk7u8vwqt0uP7Fx6yNBrdMImc0NNL8dHbDkr~CwpQw65AHpcQNHH6I2ILCujWh6QuNc0G6vgGdAJ2EaKO-QXw8ULiXGpP6~s8yE4YZXLAeKKW0GEe5p~iKDkoq1zXhG1YC-PVl9EzwFZnkAmUIuZ5yWkA~8akCi5dM5mw8xzexIu04q3cCWm3~GVGqNUcBPIFq38GsPbz5h250BMwHufzilaXekXIXGn71ZFg5peRFKV9rkQb8BThAKSyqG9G4CvvJUSzg79VyY-I5IvE6S7uBvU4-E~pqo6r1tZlefMuNpXWtLmFm1KTan~npieggw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87195741&amp;ar=333979&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsm00901a.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/2/10.1093_dnares_dsm009/1/m_dsm00901b.jpeg?Expires=2147483647&amp;Signature=K1gklDFdZ807i8yGp6LHQmlJyvyZttKTRnmM7sErbevCs~lpVBio23jg-uRAnAlw0KGZcHFRQ~qApA8uHtVdZVNrGUR-20rmb-GwxKzO17mgoG8sOrEcKfGXtjhCFkKeTw7kPwMbH1gHM7nYNZLPeSM1tIh6vVquMoVRzRg8A4pXnKv3a1AvCVNA36P1MwglhTcQ0AMHNd1gE887r94XstPLJH8ocf09cKPIxNAh6Ujz5kobpglG7Az10G9oF2Uar-2YPb2iPK~m6HE2~HJGmaa6WXXsD5tys4MxGPOk~RmfcjQNc87MwBwORDWUZfZTSyrtZgXZEaWpMowWZfNT7Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The first genetic linkage map of L. albus. Marker distances in cM are provided on the vertical scale bars. Markers showing distorted segregation are indicated with an asterisk (*). Markers with ambiguous order described by Multipoint v. 1.2 are shown in a second column. Genomic locations of QTLs for flowering time (LG1 and 3) and anthracnose resistance (LG4 and 17) are highlighted to the left of LGs with criss-crossed and diagonally patterned boxes, respectively, and SDs depicted by lines to the sides. Refer to Table 3 for QTL parameters. The locus for alkaloid content on LG11 is indicated in bold." path-from-xml="dsm00901b.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195741/dsm00901b.jpeg" path-from-xml="dsm00901b.jpeg" target="_blank">View large</a><a section="87195741" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/2/10.1093_dnares_dsm009/1/dsm00901b.jpeg?Expires=2147483647&amp;Signature=MRrnbfsWJnGMpy-gPwSBtlk03f2hnxDSYmHk7hTAbRlEzhqrOaTPtgl4~jZYqPSCToDILm~p5mFyM7zUBHA6Vpn9mKm7k6bSk9PxApCY-lCiiXRugkxdkagplrsCl27x49wqHXxFi4EJ0wEDrlO~BELokdzB8f6A9L5-rDZ--LOna~4wL1ifyd5R4zAWmDFUHuIA8uQ0EyoeBzbuQbqIEaPYuqDAs-h2G5vQZ0uYM6hftkyMgnPe2eA~UtNiSQBEOfTb0m-9ANaai7SZ7HJ5FPF6akKyZ8UA2DYTL5qKoo6I~ZieL877dPje3uHcAEvrQg0vSw4Tared63j1MrMDxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87195741&amp;ar=333979&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsm00901b.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The first genetic linkage map of <em>L. albus</em>. Marker distances in cM are provided on the vertical scale bars. Markers showing distorted segregation are indicated with an asterisk (*). Markers with ambiguous order described by Multipoint v. 1.2 are shown in a second column. Genomic locations of QTLs for flowering time (LG1 and 3) and anthracnose resistance (LG4 and 17) are highlighted to the left of LGs with criss-crossed and diagonally patterned boxes, respectively, and SDs depicted by lines to the sides. Refer to Table <a reveal-id="DSM009TB3" data-open="DSM009TB3" class="link link-reveal link-table xref-fig">3</a> for QTL parameters. The locus for alkaloid content on LG11 is indicated in bold.</p></div></div>                        <a id="87195742" scrollto-destination="87195742"></a>
<div content-id="DSM009TB1" class="table-modal"><div class="table-wrap"><div id="DSM009TB1" class="table-wrap-title" data-id="DSM009TB1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Properties of the first genetic map of <em>L. albus</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Linkage group </th><th>Length (cM) </th><th>Number of polymorphic markers </th><th>Number of defined loci<sup>a</sup> </th><th>Average marker spacing (cM) </th><th>Associated trait </th></tr></thead><tbody><tr><td>LG1 </td><td>246.5 </td><td>26 </td><td>19 </td><td>9.9 </td><td>Flowering time </td></tr><tr><td>LG2 </td><td>255.1 </td><td>21 </td><td>15 </td><td>13.4 </td><td>Nil </td></tr><tr><td>LG3 </td><td>116.1 </td><td>20 </td><td>15 </td><td>7.7 </td><td>Flowering time </td></tr><tr><td>LG4 </td><td>144.2 </td><td>18 </td><td>16 </td><td>8.5 </td><td>Anthracnose resistance </td></tr><tr><td>LG5 </td><td>161.6 </td><td>17 </td><td>12 </td><td>14.7 </td><td>Nil </td></tr><tr><td>LG6 </td><td>125.6 </td><td>16 </td><td>13 </td><td>10.5 </td><td>Nil </td></tr><tr><td>LG7 </td><td>102.4 </td><td>14 </td><td>13 </td><td>8.5 </td><td>Nil </td></tr><tr><td>LG8 </td><td>194.3 </td><td>14 </td><td>13 </td><td>16.2 </td><td>Nil </td></tr><tr><td>LG9 </td><td>139.0 </td><td>13 </td><td>10 </td><td>13.9 </td><td>Nil </td></tr><tr><td>LG10 </td><td>131 </td><td>12 </td><td>12 </td><td>11.9 </td><td>Nil </td></tr><tr><td>LG11 </td><td>157.1 </td><td>12 </td><td>11 </td><td>15.7 </td><td>Alkaloid content </td></tr><tr><td>LG12 </td><td>95.1 </td><td>10 </td><td>9 </td><td>11.9 </td><td>Nil </td></tr><tr><td>LG13 </td><td>116.7 </td><td>10 </td><td>9 </td><td>14.6 </td><td>Nil </td></tr><tr><td>LG14 </td><td>116.3 </td><td>9 </td><td>8 </td><td>16.6 </td><td>Nil </td></tr><tr><td>LG15 </td><td>95.1 </td><td>8 </td><td>8 </td><td>13.6 </td><td>Nil </td></tr><tr><td>LG16 </td><td>46.9 </td><td>8 </td><td>8 </td><td>6.7 </td><td>Nil </td></tr><tr><td>LG17 </td><td>58.7 </td><td>8 </td><td>8 </td><td>8.4 </td><td>Anthracnose resistance </td></tr><tr><td>LG18 </td><td>85.1 </td><td>8 </td><td>8 </td><td>12.2 </td><td>Nil </td></tr><tr><td>LG19 </td><td>74.5 </td><td>7 </td><td>7 </td><td>12.4 </td><td>Nil </td></tr><tr><td>LG20 </td><td>91.6 </td><td>7 </td><td>6 </td><td>18.3 </td><td>Nil </td></tr><tr><td>LG21 </td><td>53.1 </td><td>7 </td><td>7 </td><td>8.9 </td><td>Nil </td></tr><tr><td>LG22 </td><td>53.1 </td><td>6 </td><td>6 </td><td>10.6 </td><td>Nil </td></tr><tr><td>LG23 </td><td>64.3 </td><td>6 </td><td>6 </td><td>12.9 </td><td>Nil </td></tr><tr><td>LG24 </td><td>45.5 </td><td>5 </td><td>4 </td><td>15.2 </td><td>Nil </td></tr><tr><td>LG25 </td><td>92.7 </td><td>5 </td><td>5 </td><td>23.2 </td><td>Nil </td></tr><tr><td>LG26 </td><td>40.8 </td><td>4 </td><td>4 </td><td>13.6 </td><td>Nil </td></tr><tr><td>LG27 </td><td>26 </td><td>3 </td><td>3 </td><td>13 </td><td>Nil </td></tr><tr><td>LG28 </td><td>22.7 </td><td>3 </td><td>3 </td><td>11.4 </td><td>Nil </td></tr><tr><td>Total </td><td>2951.1 </td><td>296 </td><td>274 </td><td> </td><td> </td></tr></tbody></table></div><div content-id="DSM009TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Linkage group </th><th>Length (cM) </th><th>Number of polymorphic markers </th><th>Number of defined loci<sup>a</sup> </th><th>Average marker spacing (cM) </th><th>Associated trait </th></tr></thead><tbody><tr><td>LG1 </td><td>246.5 </td><td>26 </td><td>19 </td><td>9.9 </td><td>Flowering time </td></tr><tr><td>LG2 </td><td>255.1 </td><td>21 </td><td>15 </td><td>13.4 </td><td>Nil </td></tr><tr><td>LG3 </td><td>116.1 </td><td>20 </td><td>15 </td><td>7.7 </td><td>Flowering time </td></tr><tr><td>LG4 </td><td>144.2 </td><td>18 </td><td>16 </td><td>8.5 </td><td>Anthracnose resistance </td></tr><tr><td>LG5 </td><td>161.6 </td><td>17 </td><td>12 </td><td>14.7 </td><td>Nil </td></tr><tr><td>LG6 </td><td>125.6 </td><td>16 </td><td>13 </td><td>10.5 </td><td>Nil </td></tr><tr><td>LG7 </td><td>102.4 </td><td>14 </td><td>13 </td><td>8.5 </td><td>Nil </td></tr><tr><td>LG8 </td><td>194.3 </td><td>14 </td><td>13 </td><td>16.2 </td><td>Nil </td></tr><tr><td>LG9 </td><td>139.0 </td><td>13 </td><td>10 </td><td>13.9 </td><td>Nil </td></tr><tr><td>LG10 </td><td>131 </td><td>12 </td><td>12 </td><td>11.9 </td><td>Nil </td></tr><tr><td>LG11 </td><td>157.1 </td><td>12 </td><td>11 </td><td>15.7 </td><td>Alkaloid content </td></tr><tr><td>LG12 </td><td>95.1 </td><td>10 </td><td>9 </td><td>11.9 </td><td>Nil </td></tr><tr><td>LG13 </td><td>116.7 </td><td>10 </td><td>9 </td><td>14.6 </td><td>Nil </td></tr><tr><td>LG14 </td><td>116.3 </td><td>9 </td><td>8 </td><td>16.6 </td><td>Nil </td></tr><tr><td>LG15 </td><td>95.1 </td><td>8 </td><td>8 </td><td>13.6 </td><td>Nil </td></tr><tr><td>LG16 </td><td>46.9 </td><td>8 </td><td>8 </td><td>6.7 </td><td>Nil </td></tr><tr><td>LG17 </td><td>58.7 </td><td>8 </td><td>8 </td><td>8.4 </td><td>Anthracnose resistance </td></tr><tr><td>LG18 </td><td>85.1 </td><td>8 </td><td>8 </td><td>12.2 </td><td>Nil </td></tr><tr><td>LG19 </td><td>74.5 </td><td>7 </td><td>7 </td><td>12.4 </td><td>Nil </td></tr><tr><td>LG20 </td><td>91.6 </td><td>7 </td><td>6 </td><td>18.3 </td><td>Nil </td></tr><tr><td>LG21 </td><td>53.1 </td><td>7 </td><td>7 </td><td>8.9 </td><td>Nil </td></tr><tr><td>LG22 </td><td>53.1 </td><td>6 </td><td>6 </td><td>10.6 </td><td>Nil </td></tr><tr><td>LG23 </td><td>64.3 </td><td>6 </td><td>6 </td><td>12.9 </td><td>Nil </td></tr><tr><td>LG24 </td><td>45.5 </td><td>5 </td><td>4 </td><td>15.2 </td><td>Nil </td></tr><tr><td>LG25 </td><td>92.7 </td><td>5 </td><td>5 </td><td>23.2 </td><td>Nil </td></tr><tr><td>LG26 </td><td>40.8 </td><td>4 </td><td>4 </td><td>13.6 </td><td>Nil </td></tr><tr><td>LG27 </td><td>26 </td><td>3 </td><td>3 </td><td>13 </td><td>Nil </td></tr><tr><td>LG28 </td><td>22.7 </td><td>3 </td><td>3 </td><td>11.4 </td><td>Nil </td></tr><tr><td>Total </td><td>2951.1 </td><td>296 </td><td>274 </td><td> </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Excluding co-segregating markers and markers ambiguously positioned in Multipoint v.1.2.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195742" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM009TB1" class="table-wrap-title" data-id="DSM009TB1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Properties of the first genetic map of <em>L. albus</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Linkage group </th><th>Length (cM) </th><th>Number of polymorphic markers </th><th>Number of defined loci<sup>a</sup> </th><th>Average marker spacing (cM) </th><th>Associated trait </th></tr></thead><tbody><tr><td>LG1 </td><td>246.5 </td><td>26 </td><td>19 </td><td>9.9 </td><td>Flowering time </td></tr><tr><td>LG2 </td><td>255.1 </td><td>21 </td><td>15 </td><td>13.4 </td><td>Nil </td></tr><tr><td>LG3 </td><td>116.1 </td><td>20 </td><td>15 </td><td>7.7 </td><td>Flowering time </td></tr><tr><td>LG4 </td><td>144.2 </td><td>18 </td><td>16 </td><td>8.5 </td><td>Anthracnose resistance </td></tr><tr><td>LG5 </td><td>161.6 </td><td>17 </td><td>12 </td><td>14.7 </td><td>Nil </td></tr><tr><td>LG6 </td><td>125.6 </td><td>16 </td><td>13 </td><td>10.5 </td><td>Nil </td></tr><tr><td>LG7 </td><td>102.4 </td><td>14 </td><td>13 </td><td>8.5 </td><td>Nil </td></tr><tr><td>LG8 </td><td>194.3 </td><td>14 </td><td>13 </td><td>16.2 </td><td>Nil </td></tr><tr><td>LG9 </td><td>139.0 </td><td>13 </td><td>10 </td><td>13.9 </td><td>Nil </td></tr><tr><td>LG10 </td><td>131 </td><td>12 </td><td>12 </td><td>11.9 </td><td>Nil </td></tr><tr><td>LG11 </td><td>157.1 </td><td>12 </td><td>11 </td><td>15.7 </td><td>Alkaloid content </td></tr><tr><td>LG12 </td><td>95.1 </td><td>10 </td><td>9 </td><td>11.9 </td><td>Nil </td></tr><tr><td>LG13 </td><td>116.7 </td><td>10 </td><td>9 </td><td>14.6 </td><td>Nil </td></tr><tr><td>LG14 </td><td>116.3 </td><td>9 </td><td>8 </td><td>16.6 </td><td>Nil </td></tr><tr><td>LG15 </td><td>95.1 </td><td>8 </td><td>8 </td><td>13.6 </td><td>Nil </td></tr><tr><td>LG16 </td><td>46.9 </td><td>8 </td><td>8 </td><td>6.7 </td><td>Nil </td></tr><tr><td>LG17 </td><td>58.7 </td><td>8 </td><td>8 </td><td>8.4 </td><td>Anthracnose resistance </td></tr><tr><td>LG18 </td><td>85.1 </td><td>8 </td><td>8 </td><td>12.2 </td><td>Nil </td></tr><tr><td>LG19 </td><td>74.5 </td><td>7 </td><td>7 </td><td>12.4 </td><td>Nil </td></tr><tr><td>LG20 </td><td>91.6 </td><td>7 </td><td>6 </td><td>18.3 </td><td>Nil </td></tr><tr><td>LG21 </td><td>53.1 </td><td>7 </td><td>7 </td><td>8.9 </td><td>Nil </td></tr><tr><td>LG22 </td><td>53.1 </td><td>6 </td><td>6 </td><td>10.6 </td><td>Nil </td></tr><tr><td>LG23 </td><td>64.3 </td><td>6 </td><td>6 </td><td>12.9 </td><td>Nil </td></tr><tr><td>LG24 </td><td>45.5 </td><td>5 </td><td>4 </td><td>15.2 </td><td>Nil </td></tr><tr><td>LG25 </td><td>92.7 </td><td>5 </td><td>5 </td><td>23.2 </td><td>Nil </td></tr><tr><td>LG26 </td><td>40.8 </td><td>4 </td><td>4 </td><td>13.6 </td><td>Nil </td></tr><tr><td>LG27 </td><td>26 </td><td>3 </td><td>3 </td><td>13 </td><td>Nil </td></tr><tr><td>LG28 </td><td>22.7 </td><td>3 </td><td>3 </td><td>11.4 </td><td>Nil </td></tr><tr><td>Total </td><td>2951.1 </td><td>296 </td><td>274 </td><td> </td><td> </td></tr></tbody></table></div><div content-id="DSM009TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Linkage group </th><th>Length (cM) </th><th>Number of polymorphic markers </th><th>Number of defined loci<sup>a</sup> </th><th>Average marker spacing (cM) </th><th>Associated trait </th></tr></thead><tbody><tr><td>LG1 </td><td>246.5 </td><td>26 </td><td>19 </td><td>9.9 </td><td>Flowering time </td></tr><tr><td>LG2 </td><td>255.1 </td><td>21 </td><td>15 </td><td>13.4 </td><td>Nil </td></tr><tr><td>LG3 </td><td>116.1 </td><td>20 </td><td>15 </td><td>7.7 </td><td>Flowering time </td></tr><tr><td>LG4 </td><td>144.2 </td><td>18 </td><td>16 </td><td>8.5 </td><td>Anthracnose resistance </td></tr><tr><td>LG5 </td><td>161.6 </td><td>17 </td><td>12 </td><td>14.7 </td><td>Nil </td></tr><tr><td>LG6 </td><td>125.6 </td><td>16 </td><td>13 </td><td>10.5 </td><td>Nil </td></tr><tr><td>LG7 </td><td>102.4 </td><td>14 </td><td>13 </td><td>8.5 </td><td>Nil </td></tr><tr><td>LG8 </td><td>194.3 </td><td>14 </td><td>13 </td><td>16.2 </td><td>Nil </td></tr><tr><td>LG9 </td><td>139.0 </td><td>13 </td><td>10 </td><td>13.9 </td><td>Nil </td></tr><tr><td>LG10 </td><td>131 </td><td>12 </td><td>12 </td><td>11.9 </td><td>Nil </td></tr><tr><td>LG11 </td><td>157.1 </td><td>12 </td><td>11 </td><td>15.7 </td><td>Alkaloid content </td></tr><tr><td>LG12 </td><td>95.1 </td><td>10 </td><td>9 </td><td>11.9 </td><td>Nil </td></tr><tr><td>LG13 </td><td>116.7 </td><td>10 </td><td>9 </td><td>14.6 </td><td>Nil </td></tr><tr><td>LG14 </td><td>116.3 </td><td>9 </td><td>8 </td><td>16.6 </td><td>Nil </td></tr><tr><td>LG15 </td><td>95.1 </td><td>8 </td><td>8 </td><td>13.6 </td><td>Nil </td></tr><tr><td>LG16 </td><td>46.9 </td><td>8 </td><td>8 </td><td>6.7 </td><td>Nil </td></tr><tr><td>LG17 </td><td>58.7 </td><td>8 </td><td>8 </td><td>8.4 </td><td>Anthracnose resistance </td></tr><tr><td>LG18 </td><td>85.1 </td><td>8 </td><td>8 </td><td>12.2 </td><td>Nil </td></tr><tr><td>LG19 </td><td>74.5 </td><td>7 </td><td>7 </td><td>12.4 </td><td>Nil </td></tr><tr><td>LG20 </td><td>91.6 </td><td>7 </td><td>6 </td><td>18.3 </td><td>Nil </td></tr><tr><td>LG21 </td><td>53.1 </td><td>7 </td><td>7 </td><td>8.9 </td><td>Nil </td></tr><tr><td>LG22 </td><td>53.1 </td><td>6 </td><td>6 </td><td>10.6 </td><td>Nil </td></tr><tr><td>LG23 </td><td>64.3 </td><td>6 </td><td>6 </td><td>12.9 </td><td>Nil </td></tr><tr><td>LG24 </td><td>45.5 </td><td>5 </td><td>4 </td><td>15.2 </td><td>Nil </td></tr><tr><td>LG25 </td><td>92.7 </td><td>5 </td><td>5 </td><td>23.2 </td><td>Nil </td></tr><tr><td>LG26 </td><td>40.8 </td><td>4 </td><td>4 </td><td>13.6 </td><td>Nil </td></tr><tr><td>LG27 </td><td>26 </td><td>3 </td><td>3 </td><td>13 </td><td>Nil </td></tr><tr><td>LG28 </td><td>22.7 </td><td>3 </td><td>3 </td><td>11.4 </td><td>Nil </td></tr><tr><td>Total </td><td>2951.1 </td><td>296 </td><td>274 </td><td> </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Excluding co-segregating markers and markers ambiguously positioned in Multipoint v.1.2.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195742" target="_blank">View Large</a></div></div><p>Overall, 12.8% of markers segregated in a distorted fashion at <em>P</em> &lt; 0.05 and only 4% at <em>P</em> &lt; 0.01. Segregation distortion has been commonly encountered in various plant species such as narrow-leafed lupin (9%),<sup><a reveal-id="DSM009C37" data-open="DSM009C37" class="link link-ref link-reveal xref-bibr">37</a></sup> chickpea (20.4%),<sup><a reveal-id="DSM009C38" data-open="DSM009C38" class="link link-ref link-reveal xref-bibr">38</a></sup> and is a common phenomenon in lentil.<sup><a reveal-id="DSM009C39" data-open="DSM009C39" class="link link-ref link-reveal xref-bibr">39</a></sup> Many factors may be responsible for marker distortion such as structural rearrangement or differences in DNA content, non-random abortion of male and female gametes, and selection for or against a particular allele of one of the parents during propagation of RILs.<sup><a reveal-id="DSM009C40  DSM009C43" data-open="DSM009C40  DSM009C43" class="link link-ref link-reveal xref-bibr">40–43</a></sup></p><p>Few of the markers in this study were shared with <em>L. angustifolius</em>, which is partly due to the Kiev Mutant × P27174 cross being relatively narrow. This is evidenced in the number of polymorphic ITAP markers detected, 112/626 for <em>L. albus</em> and 108/424 for <em>L. angustifolius</em>.<sup><a reveal-id="DSM009C23" data-open="DSM009C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Of these only 24 were shared in common, and to describe synteny, two or more syntenic markers are required. Only two instances were found with two markers: LG63 and Lup3 (LG1 in <em>L. albus</em> and LG8 in <em>L. angustifolius</em>), and CPCB2 and Lup56 (LG24 in <em>L. albus</em> and LG2 in <em>L. angustifolius</em>). The three extra LGs in <em>L. albus</em>, compared to an expected chromosome number of 25,<sup><a reveal-id="DSM009C4" data-open="DSM009C4" class="link link-ref link-reveal xref-bibr">4</a></sup> seven additional pairs, and 15 unlinked markers might be expected to coalesce and join to the ends of other LGs as more markers are mapped.</p>                            <h3 scrollto-destination=87195745 id="87195745" class="section-title" data-legacy-id=s3b>3.2. Characteristics of genetic markers</h3>
                            <h4 scrollto-destination=87195746 id="87195746" class="section-title" data-legacy-id=s3b1>3.2.1. Intron targeted amplified polymorphic sequence</h4>
<p>Of the 626 ITAP markers that were screened, 537 amplified in the <em>L. albus</em> genome. Three hundred and seventy-eight of these produced clear single band amplicons (Table <a reveal-id="DSM009TB2" data-open="DSM009TB2" class="link link-reveal link-table xref-fig">2</a>). From these, 112 polymorphic markers were identified between Kiev Mutant and P27174 (Table <a reveal-id="DSM009TB2" data-open="DSM009TB2" class="link link-reveal link-table xref-fig">2</a>), of which 105 were used to genotype the 94 individuals of the F8 RIL population (Supplementary Table S1). Chi-squared tests revealed 10 genic markers (about 10%) with distorted segregation (<em>P</em> &lt; 0.05). Of the three genic marker groups, the ‘ML’ markers gave the highest amplification and polymorphism rates (95 and 35%, respectively; Table <a reveal-id="DSM009TB2" data-open="DSM009TB2" class="link link-reveal link-table xref-fig">2</a>). The ‘MLG’ group amplified with a similar frequency, but fewer amplicons were polymorphic. The ‘MP’ primers amplified at a noticeably lower frequency (60%) but were slightly more polymorphic than the MLG group (Table <a reveal-id="DSM009TB2" data-open="DSM009TB2" class="link link-reveal link-table xref-fig">2</a>). This data compliments previous studies in <em>Lupinus angustifolius</em> and <em>Lens culinaris</em>,<sup><a reveal-id="DSM009C23" data-open="DSM009C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSM009C27" data-open="DSM009C27" class="link link-ref link-reveal xref-bibr">27</a></sup> and implies an association between the success rates of amplification in the target species based on the phlyogenetic context of primer design, and that primers based on conserved sequences between three species (i.e. MLG primers) may have more conserved introns.</p>                        <a id="87195748" scrollto-destination="87195748"></a>
<div content-id="DSM009TB2" class="table-modal"><div class="table-wrap"><div id="DSM009TB2" class="table-wrap-title" data-id="DSM009TB2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Efficiency of genic markers used to construct the comparative genetic linkage map of <em>L. albus</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Marker type<sup>a</sup> </th><th>Screened </th><th>Amplification<sup>b</sup> (%) </th><th>Sequenced </th><th>Polymorphism<sup>c</sup> (%) </th><th>Mapped </th></tr></thead><tbody><tr><td>MP </td><td>126 </td><td>75 (60) </td><td>69 </td><td>14 (20) </td><td>14 </td></tr><tr><td>ML </td><td>350 </td><td>332 (95) </td><td>246 </td><td>87 (35) </td><td>81 </td></tr><tr><td>MLG </td><td>150 </td><td>130 (87) </td><td>63 </td><td>11 (17) </td><td>10 </td></tr><tr><td>Total </td><td>626 </td><td>537 </td><td>378 </td><td>112 </td><td>105 </td></tr></tbody></table></div><div content-id="DSM009TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Marker type<sup>a</sup> </th><th>Screened </th><th>Amplification<sup>b</sup> (%) </th><th>Sequenced </th><th>Polymorphism<sup>c</sup> (%) </th><th>Mapped </th></tr></thead><tbody><tr><td>MP </td><td>126 </td><td>75 (60) </td><td>69 </td><td>14 (20) </td><td>14 </td></tr><tr><td>ML </td><td>350 </td><td>332 (95) </td><td>246 </td><td>87 (35) </td><td>81 </td></tr><tr><td>MLG </td><td>150 </td><td>130 (87) </td><td>63 </td><td>11 (17) </td><td>10 </td></tr><tr><td>Total </td><td>626 </td><td>537 </td><td>378 </td><td>112 </td><td>105 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Predominant modes of design: MP, markers primarily between <em>M. truncatula</em> and <em>G. Max</em>; ML, between <em>M. truncatula</em> and <em>L. albus</em>; MLG, between <em>M. truncatula</em>, <em>G. Max</em> and <em>L. albus.</em></p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Figures in parentheses are percentages of amplified markers of the total markers screened.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Figures in parentheses are percentages of polymorphic markers of the total markers sequenced.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195748" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM009TB2" class="table-wrap-title" data-id="DSM009TB2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Efficiency of genic markers used to construct the comparative genetic linkage map of <em>L. albus</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Marker type<sup>a</sup> </th><th>Screened </th><th>Amplification<sup>b</sup> (%) </th><th>Sequenced </th><th>Polymorphism<sup>c</sup> (%) </th><th>Mapped </th></tr></thead><tbody><tr><td>MP </td><td>126 </td><td>75 (60) </td><td>69 </td><td>14 (20) </td><td>14 </td></tr><tr><td>ML </td><td>350 </td><td>332 (95) </td><td>246 </td><td>87 (35) </td><td>81 </td></tr><tr><td>MLG </td><td>150 </td><td>130 (87) </td><td>63 </td><td>11 (17) </td><td>10 </td></tr><tr><td>Total </td><td>626 </td><td>537 </td><td>378 </td><td>112 </td><td>105 </td></tr></tbody></table></div><div content-id="DSM009TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Marker type<sup>a</sup> </th><th>Screened </th><th>Amplification<sup>b</sup> (%) </th><th>Sequenced </th><th>Polymorphism<sup>c</sup> (%) </th><th>Mapped </th></tr></thead><tbody><tr><td>MP </td><td>126 </td><td>75 (60) </td><td>69 </td><td>14 (20) </td><td>14 </td></tr><tr><td>ML </td><td>350 </td><td>332 (95) </td><td>246 </td><td>87 (35) </td><td>81 </td></tr><tr><td>MLG </td><td>150 </td><td>130 (87) </td><td>63 </td><td>11 (17) </td><td>10 </td></tr><tr><td>Total </td><td>626 </td><td>537 </td><td>378 </td><td>112 </td><td>105 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Predominant modes of design: MP, markers primarily between <em>M. truncatula</em> and <em>G. Max</em>; ML, between <em>M. truncatula</em> and <em>L. albus</em>; MLG, between <em>M. truncatula</em>, <em>G. Max</em> and <em>L. albus.</em></p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Figures in parentheses are percentages of amplified markers of the total markers screened.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Figures in parentheses are percentages of polymorphic markers of the total markers sequenced.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195748" target="_blank">View Large</a></div></div>                            <h4 scrollto-destination=87195749 id="87195749" class="section-title" data-legacy-id=s3b2>3.2.2. AFLP markers</h4>
<p>One of the most popular molecular marker techniques used for genetic mapping is AFLP.<sup><a reveal-id="DSM009C33" data-open="DSM009C33" class="link link-ref link-reveal xref-bibr">33</a></sup> The advantage of AFLPs is their efficiency in simultaneously generating large numbers of molecular markers in a single assay. This technique provides a useful complement to gene-based markers by joining together blocks of genic markers used to identify synteny, as shown between <em>L. angustifolius</em> and <em>M. truncatula.</em><sup><a reveal-id="DSM009C23" data-open="DSM009C23" class="link link-ref link-reveal xref-bibr">23</a></sup></p><p>Two hundred and twenty-three discrete markers were generated from 64 selective primer combinations, of which 24 were codominant (i.e. A or B) and 199 dominant (i.e. present or absent). Of these 223 markers, 32 did not segregate in the expected Mendelian ratio of 1:1 (<em>P</em> &lt; 0.05). However, only three of these distorted markers with <em>P</em> &lt; 0.01 were excluded from the map construction process since the remaining markers did not interfere with LG construction.</p>                            <h3 scrollto-destination=87195752 id="87195752" class="section-title" data-legacy-id=s3c>3.3. L. albus comparative mapping with M. truncatula</h3>
<p>To assess the level of synteny between <em>L. albus</em> and <em>M. truncatula</em>, the <em>L. albus</em> genetic map was compared to the most up-to-date version of the <em>M. truncatula</em> physical map. Forty-five of 97 (46%) orthologous markers that have been mapped in <em>M. truncatula</em> revealed evidence of conserved gene order between <em>M. truncatula</em> and <em>L. albus</em>. As shown in Fig. <a reveal-id="DSM009F2" data-open="DSM009F2" class="link link-reveal link-table xref-fig">2</a>, there are numerous breaks in synteny between the two genomes that would be expected given the wide evolutionary distance and difference in chromosome number between the two species (<em>M. truncatula</em>, <em>n</em> = 8; <em>L. albus</em>, <em>n</em> = 25). The orthologous markers identified 17 small syntenic blocks of two or more markers on 17 separate LGs. The remaining 52 markers showed no evidence of macrosynteny, and 15 were unlinked.</p>                        <a id="87195754" scrollto-destination="87195754"></a>
<div id="DSM009F2" data-id="DSM009F2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/2/10.1093_dnares_dsm009/1/m_dsm00902.jpeg?Expires=2147483647&Signature=WDymXVYtshILcne33e7qClSATMps2uBcEQJmleD73DxHeQiBlYzF9cJDbj-4ykVxfKmZdVzOgKEHquxorzpH2UbRyjWFhktQQxvUXAvcc8wcHCHN-VLVivh3VeDzmiHi10TmWF~0-41H9JFbdxjgKI~RrxR9jYMeWcxnG0bd5NHANpVoj~dTsNLOT9KnFYVP7iZrBtjvtpgIsQvqmML0VZW2cVXRRyqp7yqz86MDyeqxG~0MiGVm9MNjqJZR2AHtuiSn~vyYcQH5XxUZDA2phPf8hl6eO-abUYyLDhxxqlpQPcDnZBauLxevmslvTadsI2CXo2n1Zdqq2TIAqUT3PA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evidence of macrosynteny between L. albus and M. truncatula genomes. The L. albus LGs are shown in black and the medicago LGs are shown in grey. Orthologous markers joining L. albus and M. truncatula LGs are indicated by broken lines. Distances for M. truncatula markers are provided in cM." path-from-xml="dsm00902.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195754/dsm00902.jpeg" path-from-xml="dsm00902.jpeg" target="_blank">View large</a><a section="87195754" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/2/10.1093_dnares_dsm009/1/dsm00902.jpeg?Expires=2147483647&Signature=CZGRF0tdSLOIplaXwDCU4F3uKwxbQxYFvjGj~ZiTafwv9jK3oXVkwVINBdMBL0yISh7wM0W1notLWjAAGmGMdLHK07GovyCp2386jck9SqSWq7FdlGMx0-rwsDj5J8RUPzHAdWZpmq8zNYvWoqHSFQh5dRaqfLtakIfoapUD40kmB4T0UAXMk3s0RuDtipSUjlM7wwHrCSBkrLkFseP4C9kziChGVnZtOpUM31sYzPjfYo9VHgDlslabdykTyM9pomi0xKPqHr9wytCOhlhAS0uPTNAjFPuIr3hQnCdOsBCR-XzXjTAIm6CVe2v0rxbjUL8IbeZY-QcYuh4YSlQlyA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195754&ar=333979&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm00902.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Evidence of macrosynteny between <em>L. albus</em> and <em>M. truncatula</em> genomes. The <em>L. albus</em> LGs are shown in black and the medicago LGs are shown in grey. Orthologous markers joining <em>L. albus</em> and <em>M. truncatula</em> LGs are indicated by broken lines. Distances for <em>M. truncatula</em> markers are provided in cM.</p></div></div><div content-id="DSM009F2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/2/10.1093_dnares_dsm009/1/m_dsm00902.jpeg?Expires=2147483647&Signature=WDymXVYtshILcne33e7qClSATMps2uBcEQJmleD73DxHeQiBlYzF9cJDbj-4ykVxfKmZdVzOgKEHquxorzpH2UbRyjWFhktQQxvUXAvcc8wcHCHN-VLVivh3VeDzmiHi10TmWF~0-41H9JFbdxjgKI~RrxR9jYMeWcxnG0bd5NHANpVoj~dTsNLOT9KnFYVP7iZrBtjvtpgIsQvqmML0VZW2cVXRRyqp7yqz86MDyeqxG~0MiGVm9MNjqJZR2AHtuiSn~vyYcQH5XxUZDA2phPf8hl6eO-abUYyLDhxxqlpQPcDnZBauLxevmslvTadsI2CXo2n1Zdqq2TIAqUT3PA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evidence of macrosynteny between L. albus and M. truncatula genomes. The L. albus LGs are shown in black and the medicago LGs are shown in grey. Orthologous markers joining L. albus and M. truncatula LGs are indicated by broken lines. Distances for M. truncatula markers are provided in cM." path-from-xml="dsm00902.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195754/dsm00902.jpeg" path-from-xml="dsm00902.jpeg" target="_blank">View large</a><a section="87195754" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/2/10.1093_dnares_dsm009/1/dsm00902.jpeg?Expires=2147483647&Signature=CZGRF0tdSLOIplaXwDCU4F3uKwxbQxYFvjGj~ZiTafwv9jK3oXVkwVINBdMBL0yISh7wM0W1notLWjAAGmGMdLHK07GovyCp2386jck9SqSWq7FdlGMx0-rwsDj5J8RUPzHAdWZpmq8zNYvWoqHSFQh5dRaqfLtakIfoapUD40kmB4T0UAXMk3s0RuDtipSUjlM7wwHrCSBkrLkFseP4C9kziChGVnZtOpUM31sYzPjfYo9VHgDlslabdykTyM9pomi0xKPqHr9wytCOhlhAS0uPTNAjFPuIr3hQnCdOsBCR-XzXjTAIm6CVe2v0rxbjUL8IbeZY-QcYuh4YSlQlyA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195754&ar=333979&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm00902.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Evidence of macrosynteny between <em>L. albus</em> and <em>M. truncatula</em> genomes. The <em>L. albus</em> LGs are shown in black and the medicago LGs are shown in grey. Orthologous markers joining <em>L. albus</em> and <em>M. truncatula</em> LGs are indicated by broken lines. Distances for <em>M. truncatula</em> markers are provided in cM.</p></div></div><p>These results imply genome fragmentation, or duplication followed by divergence and selective loss of chromosomes. This would break down lengthy stretches of syntenic chromosome segments observed between closely related legumes, such as those detected between <em>M. truncatula</em> and <em>L. culinaris</em>,<sup><a reveal-id="DSM009C27" data-open="DSM009C27" class="link link-ref link-reveal xref-bibr">27</a></sup><em>M. truncatula</em> and <em>M. sativa</em>,<sup><a reveal-id="DSM009C30" data-open="DSM009C30" class="link link-ref link-reveal xref-bibr">30</a></sup> and <em>M. sativa</em> and <em>Pisum sativum</em>.<sup><a reveal-id="DSM009C21" data-open="DSM009C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Frequently, two or more loci originating from different <em>L. albus</em> LGs were syntenic with a common <em>M. truncatula</em> chromosome. This supports a hypothesis of polyploidization followed by rearrangements, and gene divergence or selective loss of genes and chromosomal fragments. Macrosynteny restricted only to small genomic intervals was also observed between <em>L. angustifolius</em> and <em>M. truncatula</em>,<sup><a reveal-id="DSM009C23" data-open="DSM009C23" class="link link-ref link-reveal xref-bibr">23</a></sup> where only 95 of 181 common markers between the two genomes were mapped into 11 small conserved regions, and has been reported in other comparative studies between distantly related legumes such as <em>M. truncatula</em> and soybean.<sup><a reveal-id="DSM009C22" data-open="DSM009C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p><p>Each of the eight <em>M. truncatula</em> chromosomes showed corresponding syntenic regions in <em>L. albus</em>, with the exception of chromosome 6. This augments previous results between <em>L. angustifolius</em> and <em>M. truncatula</em>,<sup><a reveal-id="DSM009C23" data-open="DSM009C23" class="link link-ref link-reveal xref-bibr">23</a></sup> and between <em>M. truncatula</em> and more closely related legume species.<sup><a reveal-id="DSM009C20" data-open="DSM009C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Chromosome 6 is the smallest <em>M. truncatula</em> chromosome, contains few conserved genes,<sup><a reveal-id="DSM009C20" data-open="DSM009C20" class="link link-ref link-reveal xref-bibr">20</a></sup> a large proportion of resistance genes,<sup><a reveal-id="DSM009C44" data-open="DSM009C44" class="link link-ref link-reveal xref-bibr">44</a></sup> and is responsible for the different chromosome numbers between <em>M. truncatula</em> and field pea (<em>P. sativum</em>),<sup><a reveal-id="DSM009C21" data-open="DSM009C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSM009C30" data-open="DSM009C30" class="link link-ref link-reveal xref-bibr">30</a></sup> and between <em>M. truncatula</em> and chickpea (<em>Cicer arietinum</em>).<sup><a reveal-id="DSM009C22" data-open="DSM009C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p>                            <h3 scrollto-destination=87195757 id="87195757" class="section-title" data-legacy-id=s3d>3.4. Segregation of agronomic traits</h3>
<p>One hundred and ninety-one F8 RILs were phenotyped for anthracnose resistance, alkaloid content, and flowering time. The phenotypic distribution of the anthracnose resistance and flowering time were continuous suggesting that these traits were quantitatively inherited (<em>W</em> &gt; 0.05). ANOVA indicated genetic variation between the recombination inbred lines (<em>r</em><sup><a reveal-id="DSM009C2" data-open="DSM009C2" class="link link-ref link-reveal xref-bibr">2</a></sup> = 0.89, <em>P</em> &lt; 0.001 for anthracnose resistance and <em>r</em><sup><a reveal-id="DSM009C2" data-open="DSM009C2" class="link link-ref link-reveal xref-bibr">2</a></sup> = 0.76 with <em>P</em> &lt; 0.0001 for flowering time). The means of disease scores and flowering time for parental lines and the F8 population are shown in Fig. <a reveal-id="DSM009F3" data-open="DSM009F3" class="link link-reveal link-table xref-fig">3</a>.</p>                        <a id="87195759" scrollto-destination="87195759"></a>
<div id="DSM009F3" data-id="DSM009F3" class="fig fig-section"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/2/10.1093_dnares_dsm009/1/m_dsm00903.jpeg?Expires=2147483647&Signature=qcDdYmqNps44H2~5VCOA0zJXag-4Hu~P~n~A4CJIekCtFooH9GqMKYOSW4pPXlqDcaJ9L3CLCDr3XMSdMkWwJmLIJwAAavhoKXQqaO7IEA5nyt1myt7g3gh~wmfo2jycKnI8089-CSR0emrcqYRBfTdhkUjxRjQxq~VN6ONbpoMTdBgeTMvI6IetBnhc8syRMEVu6T1qWklJqZ3TcMSiJrz3J9N2J~ixatDqYDHczdt22N9qP-XMlEos0DtYWgFO2GIPL41-sk70sMlwsfLZjp-SA~XJZcwrA9v3XqUfbuWNshGmSWk6bG4Z0RIBwCfzLoAgfOCNYaY0rofvKaF3Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Frequency distribution of disease scores for anthracnose resistance (A) and flowering time (B) in 195 L. albus F8 RILs developed from Kiev Mutant and P27174. Three replicates were conducted. The mean disease scores of the anthracnose resistant and susceptible parents are indicated by the letters S and R, respectively. The mean scores of flowering time of the parents are indicated with black arrows. The mean scores of disease and flowering time for all F8 RILs are indicated with white arrows." path-from-xml="dsm00903.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195759/dsm00903.jpeg" path-from-xml="dsm00903.jpeg" target="_blank">View large</a><a section="87195759" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/2/10.1093_dnares_dsm009/1/dsm00903.jpeg?Expires=2147483647&Signature=oWmxVRKZCSy7LsFHVxkx2rZ3AybVaR6-LH1-U8IRyHyx4cvw9t8uEWU-oBsz4Tc8s0f4uXtld7ySHVAuyoxqiPFw-Tpe35BrrVglap4~pAnTccpaE4xomSPGksQmWKZvDOyaCET~0hCqktrTFngEC4~eJ4SR2ZQSOihYcXr4DOg-Aj2PQ-MfGPK9id-6-hL6scFTueH4Yy7AWQkJyAf0KnC86NIsMkM4EHIljE7DS8XYz1BqOQ1dQb7SNqpfZZKh9JltrFLL5qrn6vnLFbvsvK84xQv8L1qnjAT1fWxkT7EpUiohMPfxfONJNY1MrQtHQ5SvIs-gFdOCscdHX0vHyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195759&ar=333979&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm00903.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Frequency distribution of disease scores for anthracnose resistance (<strong>A</strong>) and flowering time (<strong>B</strong>) in 195 <em>L. albus</em> F8 RILs developed from Kiev Mutant and P27174. Three replicates were conducted. The mean disease scores of the anthracnose resistant and susceptible parents are indicated by the letters S and R, respectively. The mean scores of flowering time of the parents are indicated with black arrows. The mean scores of disease and flowering time for all F8 RILs are indicated with white arrows.</p></div></div><div content-id="DSM009F3" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/2/10.1093_dnares_dsm009/1/m_dsm00903.jpeg?Expires=2147483647&Signature=qcDdYmqNps44H2~5VCOA0zJXag-4Hu~P~n~A4CJIekCtFooH9GqMKYOSW4pPXlqDcaJ9L3CLCDr3XMSdMkWwJmLIJwAAavhoKXQqaO7IEA5nyt1myt7g3gh~wmfo2jycKnI8089-CSR0emrcqYRBfTdhkUjxRjQxq~VN6ONbpoMTdBgeTMvI6IetBnhc8syRMEVu6T1qWklJqZ3TcMSiJrz3J9N2J~ixatDqYDHczdt22N9qP-XMlEos0DtYWgFO2GIPL41-sk70sMlwsfLZjp-SA~XJZcwrA9v3XqUfbuWNshGmSWk6bG4Z0RIBwCfzLoAgfOCNYaY0rofvKaF3Lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Frequency distribution of disease scores for anthracnose resistance (A) and flowering time (B) in 195 L. albus F8 RILs developed from Kiev Mutant and P27174. Three replicates were conducted. The mean disease scores of the anthracnose resistant and susceptible parents are indicated by the letters S and R, respectively. The mean scores of flowering time of the parents are indicated with black arrows. The mean scores of disease and flowering time for all F8 RILs are indicated with white arrows." path-from-xml="dsm00903.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195759/dsm00903.jpeg" path-from-xml="dsm00903.jpeg" target="_blank">View large</a><a section="87195759" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/2/10.1093_dnares_dsm009/1/dsm00903.jpeg?Expires=2147483647&Signature=oWmxVRKZCSy7LsFHVxkx2rZ3AybVaR6-LH1-U8IRyHyx4cvw9t8uEWU-oBsz4Tc8s0f4uXtld7ySHVAuyoxqiPFw-Tpe35BrrVglap4~pAnTccpaE4xomSPGksQmWKZvDOyaCET~0hCqktrTFngEC4~eJ4SR2ZQSOihYcXr4DOg-Aj2PQ-MfGPK9id-6-hL6scFTueH4Yy7AWQkJyAf0KnC86NIsMkM4EHIljE7DS8XYz1BqOQ1dQb7SNqpfZZKh9JltrFLL5qrn6vnLFbvsvK84xQv8L1qnjAT1fWxkT7EpUiohMPfxfONJNY1MrQtHQ5SvIs-gFdOCscdHX0vHyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195759&ar=333979&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm00903.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Frequency distribution of disease scores for anthracnose resistance (<strong>A</strong>) and flowering time (<strong>B</strong>) in 195 <em>L. albus</em> F8 RILs developed from Kiev Mutant and P27174. Three replicates were conducted. The mean disease scores of the anthracnose resistant and susceptible parents are indicated by the letters S and R, respectively. The mean scores of flowering time of the parents are indicated with black arrows. The mean scores of disease and flowering time for all F8 RILs are indicated with white arrows.</p></div></div><p>Two methods (Dragendorff and UV) were used to determine alkaloid content. The segregation ratio among the RILs was similar in both the methods (89:94 and 88:91, low alkaloid:high alkaloid content, using the Dragendorff and U/V tests, respectively. 1:1, <em>P</em> &lt; 0.05) This suggested a single gene controls alkaloid content. The scores from the two tests were highly correlated (<em>r</em> = 0.85, <em>P</em> = 0.00001, for <em>n</em> = 192).</p>                            <h3 scrollto-destination=87195761 id="87195761" class="section-title" data-legacy-id=s3e>3.5. QTL mapping of anthracnose resistance</h3>
<p>The frequency distribution of anthracnose disease resistance scores in <em>L. albus</em> was continuous implying polygenic control. Interval mapping conducted with MultiQTL software detected two regions significantly associated with anthracnose resistance on LG4 and LG17 at an LOD &gt; 3 (Fig. <a reveal-id="DSM009F1" data-open="DSM009F1" class="link link-reveal link-table xref-fig">1</a>). These QTLs explain over 31 and 26% of the phenotypic variance, respectively, and were inherited from the resistant parent P27174. ANOVA also revealed significant correlation with resistance of two flanking markers (Lup241 and M47E35A209) on LG4 (<em>P</em> &lt; 0.0018 and <em>P</em> &lt; 0.0004, respectively) and two others (Lup313 and M65E35C315 both with <em>P</em> &lt; 0.0001) on LG17. No epistatic interactions were identified between QTLs and unlinked markers using the interactions function in Map Manager QTX. The major QTL on LG17 and a second on LG4 conferring resistance contrasts with <em>L. angustifolius</em>, where a single gene was found to be responsible.<sup><a reveal-id="DSM009C37" data-open="DSM009C37" class="link link-ref link-reveal xref-bibr">37</a></sup></p>                            <h3 scrollto-destination=87195763 id="87195763" class="section-title" data-legacy-id=s3f>3.6. QTL mapping of flowering time</h3>
<p>Flowering is an important stage in plant development that initiates grain production and is sensitive to stress. The frequency distribution of flowering time was normal in this study, consistent with the polygenic control of this trait observed in other crops. In <em>Arabidopsis</em>, seven QTLs were detected controlling flowering time,<sup><a reveal-id="DSM009C45" data-open="DSM009C45" class="link link-ref link-reveal xref-bibr">45</a></sup> while in chickpea and soybean, two and four QTLs were identified, respectively.<sup><a reveal-id="DSM009C46" data-open="DSM009C46" class="link link-ref link-reveal xref-bibr">46</a>,<a reveal-id="DSM009C47" data-open="DSM009C47" class="link link-ref link-reveal xref-bibr">47</a></sup> This result in <em>L. albus</em> contrasts with previous mapping studies in <em>L. angustifolius</em> where flowering time was controlled by a major gene, <em>Ku</em><sup><a reveal-id="DSM009C23" data-open="DSM009C23" class="link link-ref link-reveal xref-bibr">23</a></sup>.</p><p>Two significant QTLs at the top of LG1 and LG3 at a LOD score of &gt; 3 were detected using interval mapping (Fig. <a reveal-id="DSM009F1" data-open="DSM009F1" class="link link-reveal link-table xref-fig">1</a>; Table <a reveal-id="DSM009TB3" data-open="DSM009TB3" class="link link-reveal link-table xref-fig">3</a>), which together explain more than 52% of the total phenotypic variation. The strongest QTL, explaining 31% of the phenotypic variance at the top of LG3, covered a relatively wide distance of ∼10 cM. Due to population size and/or the number of recombinants in a particular region, interval mapping usually underestimates or is unable to identify the number of genes conditioning quantitative traits. Therefore two or more QTLs may be detected as a single QTL in small populations (&lt;500 individuals) if closer together than ∼20 cM.<sup><a reveal-id="DSM009C48" data-open="DSM009C48" class="link link-ref link-reveal xref-bibr">48</a>,<a reveal-id="DSM009C49" data-open="DSM009C49" class="link link-ref link-reveal xref-bibr">49</a></sup></p>                        <a id="87195766" scrollto-destination="87195766"></a>
<div content-id="DSM009TB3" class="table-modal"><div class="table-wrap"><div id="DSM009TB3" class="table-wrap-title" data-id="DSM009TB3"><div class="label -label"><strong>Table 3</strong></div><div class="caption"><p>Estimates of QTLs determined using MultiQTL with significant effects on anthracnose resistance and flowering time in <em>L. albus</em> Kiev Mutant x P<sub>27174</sub> F8 RILs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>QTL estimates<sup>a</sup> </th><th colspan="2">Anthracnose resistance </th><th colspan="2">Flowering time </th></tr></thead><tbody><tr><td>LG [interval]<sup>b</sup> </td><td>4 [9] </td><td>17 [6] </td><td>1 [1] </td><td>3 [1 and 2] </td></tr><tr><td><em>P</em><sup>c</sup> </td><td>0.012 </td><td>0.000*** </td><td>0.002** </td><td>0.000*** </td></tr><tr><td>LOD<sup>d</sup> </td><td>5.58 (1.01) </td><td>5.68 (1.7) </td><td>4.76 (1.06) </td><td>7.35 (1.98) </td></tr><tr><td>Position<sup>e</sup> </td><td>67.3 (38.4) </td><td>51.9 (6.02) </td><td>1.35 (9.4) </td><td>10.7 (8.73) </td></tr><tr><td>PEV<sup>f</sup> </td><td>0.31 (11.7) </td><td>0.26 (0.07) </td><td>0.21 (0.04) </td><td>0.31 (0.07) </td></tr><tr><td>Response mean<sup>g</sup> </td><td>3.26 (0.08) </td><td>3.18 (0.08) </td><td>3.5 (0.07) </td><td>3.5 (0.05) </td></tr><tr><td>Effect<sup>h</sup> </td><td>−0.95 (0.18) </td><td>−0.86 (0.13) </td><td>0.69 (0.09) </td><td>0.81 (0.17) </td></tr><tr><td>Linked/flanking markers </td><td>Lup241M47E35A209 </td><td>Lup313M65E35C315 </td><td>M61E32B396 </td><td>M47E32A266M75E38A83M75E38B73 </td></tr></tbody></table></div><div content-id="DSM009TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>QTL estimates<sup>a</sup> </th><th colspan="2">Anthracnose resistance </th><th colspan="2">Flowering time </th></tr></thead><tbody><tr><td>LG [interval]<sup>b</sup> </td><td>4 [9] </td><td>17 [6] </td><td>1 [1] </td><td>3 [1 and 2] </td></tr><tr><td><em>P</em><sup>c</sup> </td><td>0.012 </td><td>0.000*** </td><td>0.002** </td><td>0.000*** </td></tr><tr><td>LOD<sup>d</sup> </td><td>5.58 (1.01) </td><td>5.68 (1.7) </td><td>4.76 (1.06) </td><td>7.35 (1.98) </td></tr><tr><td>Position<sup>e</sup> </td><td>67.3 (38.4) </td><td>51.9 (6.02) </td><td>1.35 (9.4) </td><td>10.7 (8.73) </td></tr><tr><td>PEV<sup>f</sup> </td><td>0.31 (11.7) </td><td>0.26 (0.07) </td><td>0.21 (0.04) </td><td>0.31 (0.07) </td></tr><tr><td>Response mean<sup>g</sup> </td><td>3.26 (0.08) </td><td>3.18 (0.08) </td><td>3.5 (0.07) </td><td>3.5 (0.05) </td></tr><tr><td>Effect<sup>h</sup> </td><td>−0.95 (0.18) </td><td>−0.86 (0.13) </td><td>0.69 (0.09) </td><td>0.81 (0.17) </td></tr><tr><td>Linked/flanking markers </td><td>Lup241M47E35A209 </td><td>Lup313M65E35C315 </td><td>M61E32B396 </td><td>M47E32A266M75E38A83M75E38B73 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>SDs are shown in parentheses.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>LG and interval within LG associated with the quantitative trait.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Probability values from 1000 permutation tests. Asterisks indicate genome-wide significance for false discovery rate: 10 (*), 5 (**), and 1% (***).</p></div></div><div content-id=""><div class="fn"><p><sup>d–h</sup>Estimated by 1000 bootstrap tests.</p></div></div><div content-id=""><div class="fn"><p><sup>d</sup>Maximum LOD score for a given interval.</p></div></div><div content-id=""><div class="fn"><p><sup>e</sup>Position (cM) of maximum LOD value measured from the first marker in corresponding LG (0 cM).</p></div></div><div content-id=""><div class="fn"><p><sup>f</sup>Proportion of explained variability by the putative QTL.</p></div></div><div content-id=""><div class="fn"><p><sup>g</sup>Means of anthracnose resistance and flowering time.</p></div></div><div content-id=""><div class="fn"><p><sup>h</sup>Effects of the QTLs on the response of anthracnose resistance and flowering time.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195766" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM009TB3" class="table-wrap-title" data-id="DSM009TB3"><div class="label -label"><strong>Table 3</strong></div><div class="caption"><p>Estimates of QTLs determined using MultiQTL with significant effects on anthracnose resistance and flowering time in <em>L. albus</em> Kiev Mutant x P<sub>27174</sub> F8 RILs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>QTL estimates<sup>a</sup> </th><th colspan="2">Anthracnose resistance </th><th colspan="2">Flowering time </th></tr></thead><tbody><tr><td>LG [interval]<sup>b</sup> </td><td>4 [9] </td><td>17 [6] </td><td>1 [1] </td><td>3 [1 and 2] </td></tr><tr><td><em>P</em><sup>c</sup> </td><td>0.012 </td><td>0.000*** </td><td>0.002** </td><td>0.000*** </td></tr><tr><td>LOD<sup>d</sup> </td><td>5.58 (1.01) </td><td>5.68 (1.7) </td><td>4.76 (1.06) </td><td>7.35 (1.98) </td></tr><tr><td>Position<sup>e</sup> </td><td>67.3 (38.4) </td><td>51.9 (6.02) </td><td>1.35 (9.4) </td><td>10.7 (8.73) </td></tr><tr><td>PEV<sup>f</sup> </td><td>0.31 (11.7) </td><td>0.26 (0.07) </td><td>0.21 (0.04) </td><td>0.31 (0.07) </td></tr><tr><td>Response mean<sup>g</sup> </td><td>3.26 (0.08) </td><td>3.18 (0.08) </td><td>3.5 (0.07) </td><td>3.5 (0.05) </td></tr><tr><td>Effect<sup>h</sup> </td><td>−0.95 (0.18) </td><td>−0.86 (0.13) </td><td>0.69 (0.09) </td><td>0.81 (0.17) </td></tr><tr><td>Linked/flanking markers </td><td>Lup241M47E35A209 </td><td>Lup313M65E35C315 </td><td>M61E32B396 </td><td>M47E32A266M75E38A83M75E38B73 </td></tr></tbody></table></div><div content-id="DSM009TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>QTL estimates<sup>a</sup> </th><th colspan="2">Anthracnose resistance </th><th colspan="2">Flowering time </th></tr></thead><tbody><tr><td>LG [interval]<sup>b</sup> </td><td>4 [9] </td><td>17 [6] </td><td>1 [1] </td><td>3 [1 and 2] </td></tr><tr><td><em>P</em><sup>c</sup> </td><td>0.012 </td><td>0.000*** </td><td>0.002** </td><td>0.000*** </td></tr><tr><td>LOD<sup>d</sup> </td><td>5.58 (1.01) </td><td>5.68 (1.7) </td><td>4.76 (1.06) </td><td>7.35 (1.98) </td></tr><tr><td>Position<sup>e</sup> </td><td>67.3 (38.4) </td><td>51.9 (6.02) </td><td>1.35 (9.4) </td><td>10.7 (8.73) </td></tr><tr><td>PEV<sup>f</sup> </td><td>0.31 (11.7) </td><td>0.26 (0.07) </td><td>0.21 (0.04) </td><td>0.31 (0.07) </td></tr><tr><td>Response mean<sup>g</sup> </td><td>3.26 (0.08) </td><td>3.18 (0.08) </td><td>3.5 (0.07) </td><td>3.5 (0.05) </td></tr><tr><td>Effect<sup>h</sup> </td><td>−0.95 (0.18) </td><td>−0.86 (0.13) </td><td>0.69 (0.09) </td><td>0.81 (0.17) </td></tr><tr><td>Linked/flanking markers </td><td>Lup241M47E35A209 </td><td>Lup313M65E35C315 </td><td>M61E32B396 </td><td>M47E32A266M75E38A83M75E38B73 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>SDs are shown in parentheses.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>LG and interval within LG associated with the quantitative trait.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Probability values from 1000 permutation tests. Asterisks indicate genome-wide significance for false discovery rate: 10 (*), 5 (**), and 1% (***).</p></div></div><div content-id=""><div class="fn"><p><sup>d–h</sup>Estimated by 1000 bootstrap tests.</p></div></div><div content-id=""><div class="fn"><p><sup>d</sup>Maximum LOD score for a given interval.</p></div></div><div content-id=""><div class="fn"><p><sup>e</sup>Position (cM) of maximum LOD value measured from the first marker in corresponding LG (0 cM).</p></div></div><div content-id=""><div class="fn"><p><sup>f</sup>Proportion of explained variability by the putative QTL.</p></div></div><div content-id=""><div class="fn"><p><sup>g</sup>Means of anthracnose resistance and flowering time.</p></div></div><div content-id=""><div class="fn"><p><sup>h</sup>Effects of the QTLs on the response of anthracnose resistance and flowering time.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195766" target="_blank">View Large</a></div></div><p>The second QTL for flowering time at the top of LG1 explained 21% of the phenotypic variation observed. In the present map, three AFLP markers (M61E32B396, M75E38A83, M75E38B73) cosegregate with the QTL at the end of the LG, suggesting that more markers will be needed to extend this LG to more precisely define the locus.</p><p>A third QTL was identified on LG12 that accounted for 11% of the trait at an LOD = 2.4. However, Map Manager QTX did not support this QTL (data not shown). Single-point analysis showed four linked and/or flanking markers (M61E32B396 on LG1 and M47E32A266, M75E38A83, M75E38B73 on LG3) significantly associated with flowering time (<em>P</em> &lt; 0.001). Markers M61E32B396 and M75E38B73 were inherited from Kiev Mutant, whereas M47E32A266 and M75E38A83 were from P27174. No epistatic interactions were detected between these QTLs or unlinked markers using the interactions function in Map Manager QTX.</p>                            <h3 scrollto-destination=87195769 id="87195769" class="section-title" data-legacy-id=s3g>3.7. Genetic mapping of alkaloid content</h3>
<p>A single locus determining alkaloid content mapped to LG11, flanked by markers M61E35A142 and Lup123 (Fig. <a reveal-id="DSM009F1" data-open="DSM009F1" class="link link-reveal link-table xref-fig">1</a>). These results are consistent with the earlier results that Kiev Mutant has a single gene (<em>pauper</em>) for low alkaloid content.<sup><a reveal-id="DSM009C50" data-open="DSM009C50" class="link link-ref link-reveal xref-bibr">50</a></sup> In <em>L. angustifolius</em>, alkaloid content was also found to be controlled by a single locus.<sup><a reveal-id="DSM009C23" data-open="DSM009C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Interestingly, Lup268 which flanks the QTL in <em>L. albus</em>, mapped to a separate chromosome to alkaloid content in <em>L. angustifolius</em>. This suggests either these two loci are independent or there has been rearrangement between the lupin genomes.</p>                            <h3 scrollto-destination=87195771 id="87195771" class="section-title" data-legacy-id=s3h>3.8. Future prospects for the application of synteny in L. albus</h3>
<p>While synteny between <em>L. albus</em> and <em>M. truncatula</em> in this study was limited to small conserved blocks of genes, these may help in identifying more closely linked markers or candidate genes if desirable traits map to such regions. Where structural differences exist between more distantly related species such as <em>M. truncatula</em> and soybean, it is notable that significant microsynteny, or conservation at the DNA sequence level, exists.<sup><a reveal-id="DSM009C51" data-open="DSM009C51" class="link link-ref link-reveal xref-bibr">51</a></sup> Thus one would expect progress in the application of both macro- and microsynteny to identify closely linked markers and to isolate important genes; precedents in other families include the Solanaceae,<sup><a reveal-id="DSM009C52" data-open="DSM009C52" class="link link-ref link-reveal xref-bibr">52</a></sup> Poaceae,<sup><a reveal-id="DSM009C53  DSM009C55" data-open="DSM009C53  DSM009C55" class="link link-ref link-reveal xref-bibr">53–55</a></sup> and Rosaceae.<sup><a reveal-id="DSM009C56" data-open="DSM009C56" class="link link-ref link-reveal xref-bibr">56</a></sup></p><p>Although the <em>L. albus</em> genetic and comparative map comprised more than 300 markers, there were not enough shared markers to make a detailed comparison of the <em>L. albus</em> and <em>L. angustifolius</em> genomes, as this is the most obvious way to investigate Genistoid evolution. More markers are also needed to consolidate the number of chromosomes and increase marker density. The present study provides markers associated with anthracnose and flowering QTLs that were significantly associated with these traits. More tightly linked molecular markers are required, however, for efficient marker-assisted selection in breeding programs involving these parental lines. Nevertheless, this research throws light on the processes of chromosomal evolution in the Genistoid clade, is the first report on the genetic mapping of economically important traits in <em>L. albus</em>, and forms the basis of a resource for the <em>L. albus</em> and legume research community.</p>                            <h2 scrollto-destination=87195774 id="87195774" class="backacknowledgements-title" >Acknowledgements</h2>
<p>This research was supported by ARC Linkage project LP0454871 and NSW Department of Primary Industries. Special thanks are due to Dr Huaan Yang at DAFWA for provision of RIL seeds, and to James Hane and Angela Williams at Murdoch University for assistance.</p>                            <h2 scrollto-destination=87195776 id="87195776" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSM009C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Langridge</div> <div class="given-names">D. F.</div></div>,  <div class="name"><div class="surname">Goodman</div> <div class="given-names">R. D.</div></div>. </div><div class="article-title">Honeybee pollination of lupins (<em>Lupinus albus</em> cv.Hamburg)</div>, <div class="source">Aust. J. Expt. Agric.</div> , <div class="year">1985</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">220</div>-<div class="lpage">223</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Honeybee%20pollination%20of%20lupins%20(Lupinus%20albus%20cv.Hamburg)&amp;author=D.%20F.%20Langridge&amp;author=R.%20D.%20Goodman&amp;publication_year=1985&amp;journal=Aust.%20J.%20Expt.%20Agric.&amp;volume=25&amp;pages=220-223" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1071/EA9850220" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1071%2FEA9850220" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1071%2FEA9850220"> </span></div> </div></div></div></div></div><div content-id="DSM009C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Payne</div> <div class="given-names">W. A.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Ball</div> <div class="given-names">D. A.</div></div>. </div><div class="article-title">Agronomic potential of narrow-leafed and white lupins in the inland Pacific northwest</div>, <div class="source">Agron J</div> , <div class="year">2004</div>, vol. <div class="volume">96</div> (pg. <div class="fpage">1501</div>-<div class="lpage">1508</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Agronomic%20potential%20of%20narrow-leafed%20and%20white%20lupins%20in%20the%20inland%20Pacific%20northwest&amp;author=W.%20A.%20Payne&amp;author=C.%20Chen&amp;author=D.%20A.%20Ball&amp;publication_year=2004&amp;journal=Agron%20J&amp;volume=96&amp;pages=1501-1508" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2134/agronj2004.1501" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2134%2Fagronj2004.1501" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2134%2Fagronj2004.1501"> </span></div> </div></div></div></div></div><div content-id="DSM009C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pazy</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Heyn</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Herrnstadt</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Plitmann</div> <div class="given-names">U.</div></div>. </div><div class="article-title">Studies in populations of the Old World <em>Lupinus</em> species. I. Chromosomes of the East-Mediterranean lupines</div>, <div class="source">Israel J. Bot.</div> , <div class="year">1977</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">115</div>-<div class="lpage">127</div>)</div></div></div></div><div content-id="DSM009C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Naganowska</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Wolko</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Sliwinska</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Kaczmarek</div> <div class="given-names">Z.</div></div>. </div><div class="article-title">Nuclear DNA content variation and species relationships in the genus <em>Lupinus</em> (Fabaceae)</div>, <div class="source">Ann. Bot.</div> , <div class="year">2003</div>, vol. <div class="volume">92</div> (pg. <div class="fpage">349</div>-<div class="lpage">355</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nuclear%20DNA%20content%20variation%20and%20species%20relationships%20in%20the%20genus%20Lupinus%20(Fabaceae)&amp;author=B.%20Naganowska&amp;author=B.%20Wolko&amp;author=E.%20Sliwinska&amp;author=Z.%20Kaczmarek&amp;publication_year=2003&amp;journal=Ann.%20Bot.&amp;volume=92&amp;pages=349-355" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/aob/mcg145" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Faob%2Fmcg145" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Faob%2Fmcg145"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12853281" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lindbeck</div> <div class="given-names">K. D.</div></div>,  <div class="name"><div class="surname">Murray</div> <div class="given-names">G. M.</div></div>,  <div class="name"><div class="surname">Priest</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Dominiak</div> <div class="given-names">B. C.</div></div>,  <div class="name"><div class="surname">Nikandrow</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Survey for anthracnose caused by <em>Colletotrichum gloeosporioides</em> in crop lupins (<em>Lupinus angustifolius</em>, <em>L. albus</em>) and ornamental lupins (<em>L. polyphyllus</em>) in New South Wales</div>, <div class="source">Australasian Plant Pathology</div> , <div class="year">2001</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">259</div>-<div class="lpage">262</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Survey%20for%20anthracnose%20caused%20by%20Colletotrichum%20gloeosporioides%20in%20crop%20lupins%20(Lupinus%20angustifolius%2C%20L.%20albus)%20and%20ornamental%20lupins%20(L.%20polyphyllus)%20in%20New%20South%20Wales&amp;author=K.%20D.%20Lindbeck&amp;author=G.%20M.%20Murray&amp;author=M.%20Priest&amp;author=B.%20C.%20Dominiak&amp;author=A.%20Nikandrow&amp;publication_year=2001&amp;journal=Australasian%20Plant%20Pathology&amp;volume=27&amp;pages=259-262" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1071/AP98029" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1071%2FAP98029" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1071%2FAP98029"> </span></div> </div></div></div></div></div><div content-id="DSM009C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Noffsinger</div> <div class="given-names">S.L.</div></div>,  <div class="name"><div class="surname">van Santen</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Evaluation of <em>Lupinus albus</em> L. Germplasm for the Southeastern USA</div>, <div class="source">Crop Sci</div> , <div class="year">2005</div>, vol. <div class="volume">45</div> (pg. <div class="fpage">1941</div>-<div class="lpage">1950</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evaluation%20of%20Lupinus%20albus%20L.%20Germplasm%20for%20the%20Southeastern%20USA&amp;author=S.L.%20Noffsinger&amp;author=E.%20van%20Santen&amp;publication_year=2005&amp;journal=Crop%20Sci&amp;volume=45&amp;pages=1941-1950" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci2004.0575" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2004.0575" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci2004.0575"> </span></div> </div></div></div></div></div><div content-id="DSM009C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Uauy</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Gattas</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Yanez</div> <div class="given-names">E.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Simopoulos</div> <div class="given-names">A. M.</div></div>. </div>, <div class="source">World Review of Nutrition and Dietetics</div> , <div class="year">1995</div>, vol. <div class="volume">77</div> <div class="publisher-name">Karger, Basel</div>(pg. <div class="fpage">75</div>-<div class="lpage">88</div>)<div class="citation-links"><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7732701" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hall</div> <div class="given-names">R. S.</div></div>,  <div class="name"><div class="surname">Johnson</div> <div class="given-names">S. K.</div></div>,  <div class="name"><div class="surname">Baxter</div> <div class="given-names">A. L.</div></div>,  <div class="name"><div class="surname">Ball</div> <div class="given-names">M. J.</div></div>. </div><div class="article-title">Lupin kernel fibre-enriched foods beneficially modify serum lipids in men</div>, <div class="source">Eur. J. Clin. Nutr.</div> , <div class="year">2004</div>, vol. <div class="volume">59</div> (pg. <div class="fpage">325</div>-<div class="lpage">333</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lupin%20kernel%20fibre-enriched%20foods%20beneficially%20modify%20serum%20lipids%20in%20men&amp;author=R.%20S.%20Hall&amp;author=S.%20K.%20Johnson&amp;author=A.%20L.%20Baxter&amp;author=M.%20J.%20Ball&amp;publication_year=2004&amp;journal=Eur.%20J.%20Clin.%20Nutr.&amp;volume=59&amp;pages=325-333" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/sj.ejcn.1602077" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.ejcn.1602077" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsj.ejcn.1602077"> </span></div> </div></div></div></div></div><div content-id="DSM009C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Edwards</div> <div class="given-names">A. C.</div></div>,  <div class="name"><div class="surname">van Barneveld</div> <div class="given-names">R. J.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Gladstones</div> <div class="given-names">J. S.</div></div>,  <div class="name"><div class="surname">Atkins</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Hamblin</div> <div class="given-names">J.</div></div>. </div>, <div class="source">Lupins as Crop Plants: Biology, Production and Utilization</div> , <div class="year">1998</div><div class="publisher-loc">Wallingford</div><div class="publisher-name">CAB International</div>(pg. <div class="fpage">385</div>-<div class="lpage">410</div>)</div></div></div></div><div content-id="DSM009C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hamama</div> <div class="given-names">A. A.</div></div>,  <div class="name"><div class="surname">Bhardwaj</div> <div class="given-names">H. L.</div></div>. </div><div class="article-title">Phytosterols, triterpene alcohols, and phospholipids in seed oil from white lupin</div>, <div class="source">J. Am. Oil Chem. Soc.</div> , <div class="year">2004</div>, vol. <div class="volume">81</div> (pg. <div class="fpage">1039</div>-<div class="lpage">1044</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phytosterols%2C%20triterpene%20alcohols%2C%20and%20phospholipids%20in%20seed%20oil%20from%20white%20lupin&amp;author=A.%20A.%20Hamama&amp;author=H.%20L.%20Bhardwaj&amp;publication_year=2004&amp;journal=J.%20Am.%20Oil%20Chem.%20Soc.&amp;volume=81&amp;pages=1039-1044" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11746-004-1019-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11746-004-1019-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11746-004-1019-z"> </span></div> </div></div></div></div></div><div content-id="DSM009C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dinkelaker</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Romheld</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Marschner</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Citric acid excretion and precipitation of calcium citrate in the rhizosphere of white lupin (<em>Lupinus albus</em> L.)</div>, <div class="source">Plant Cell Environ.</div> , <div class="year">1989</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">285</div>-<div class="lpage">292</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Citric%20acid%20excretion%20and%20precipitation%20of%20calcium%20citrate%20in%20the%20rhizosphere%20of%20white%20lupin%20(Lupinus%20albus%20L.)&amp;author=B.%20Dinkelaker&amp;author=V.%20Romheld&amp;author=H.%20Marschner&amp;publication_year=1989&amp;journal=Plant%20Cell%20Environ.&amp;volume=12&amp;pages=285-292" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pce.1989.12.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpce.1989.12.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpce.1989.12.issue-3"> </span></div> </div></div></div></div></div><div content-id="DSM009C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gardner</div> <div class="given-names">W. K.</div></div>,  <div class="name"><div class="surname">Parbery</div> <div class="given-names">D. G.</div></div>,  <div class="name"><div class="surname">Barber</div> <div class="given-names">D. A.</div></div>. </div><div class="article-title">The acquisition of phosphorus by <em>Lupinus albus</em> L. I. Some characteristics of the soil/root interface</div>, <div class="source">Plant Soil</div> , <div class="year">1982</div>, vol. <div class="volume">68</div> (pg. <div class="fpage">19</div>-<div class="lpage">32</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20acquisition%20of%20phosphorus%20by%20Lupinus%20albus%20L.%20I.%20Some%20characteristics%20of%20the%20soil%2Froot%20interface&amp;author=W.%20K.%20Gardner&amp;author=D.%20G.%20Parbery&amp;author=D.%20A.%20Barber&amp;publication_year=1982&amp;journal=Plant%20Soil&amp;volume=68&amp;pages=19-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02374724" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02374724" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02374724"> </span></div> </div></div></div></div></div><div content-id="DSM009C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Johnson</div> <div class="given-names">J. F.</div></div>,  <div class="name"><div class="surname">Vance</div> <div class="given-names">C. P.</div></div>,  <div class="name"><div class="surname">Allan</div> <div class="given-names">D. L.</div></div>. </div><div class="article-title">Phosphorus deficiency in <em>Lupinus albus</em> (altered lateral root development and enhanced expression of phosphoenolpyruvate carboxylase)</div>, <div class="source">Plant Physiol.</div> , <div class="year">1996</div>, vol. <div class="volume">112</div> (pg. <div class="fpage">31</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phosphorus%20deficiency%20in%20Lupinus%20albus%20(altered%20lateral%20root%20development%20and%20enhanced%20expression%20of%20phosphoenolpyruvate%20carboxylase)&amp;author=J.%20F.%20Johnson&amp;author=C.%20P.%20Vance&amp;author=D.%20L.%20Allan&amp;publication_year=1996&amp;journal=Plant%20Physiol.&amp;volume=112&amp;pages=31-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.112.1.31" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.112.1.31" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.112.1.31"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8819319" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Massonneau</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Langlade</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Léon</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Metabolic changes associated with cluster root development in white lupin (<em>Lupinus albus</em> L.): relationship between organic acid excretion, sucrose metabolism and energy status</div>, <div class="source">Planta</div> , <div class="year">2001</div>, vol. <div class="volume">213</div> (pg. <div class="fpage">534</div>-<div class="lpage">542</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metabolic%20changes%20associated%20with%20cluster%20root%20development%20in%20white%20lupin%20(Lupinus%20albus%20L.)%3A%20relationship%20between%20organic%20acid%20excretion%2C%20sucrose%20metabolism%20and%20energy%20status&amp;author=A.%20Massonneau&amp;author=N.%20Langlade&amp;author=S.%20L%C3%A9on&amp;publication_year=2001&amp;journal=Planta&amp;volume=213&amp;pages=534-542" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s004250100529" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs004250100529" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs004250100529"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11556785" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Neumann</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Martinoia</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Cluster roots—an underground adaptation for survival in extreme environments</div>, <div class="source">Trends Plant Sci.</div> , <div class="year">2002</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">162</div>-<div class="lpage">167</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cluster%20roots%E2%80%94an%20underground%20adaptation%20for%20survival%20in%20extreme%20environments&amp;author=G.%20Neumann&amp;author=E.%20Martinoia&amp;publication_year=2002&amp;journal=Trends%20Plant%20Sci.&amp;volume=7&amp;pages=162-167" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1360-1385(02)02241-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1360-1385(02)02241-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1360-1385(02)02241-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11950612" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Devos</div> <div class="given-names">K. M.</div></div>,  <div class="name"><div class="surname">Gale</div> <div class="given-names">M. D.</div></div>. </div><div class="article-title">Genome relationships: the grass model in current research</div>, <div class="source">Plant Cell</div> , <div class="year">2000</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">637</div>-<div class="lpage">646</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20relationships%3A%20the%20grass%20model%20in%20current%20research&amp;author=K.%20M.%20Devos&amp;author=M.%20D.%20Gale&amp;publication_year=2000&amp;journal=Plant%20Cell&amp;volume=12&amp;pages=637-646" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.12.5.637" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.12.5.637" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.12.5.637"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10810140" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lagercrantz</div> <div class="given-names">U.</div></div>,  <div class="name"><div class="surname">Lydiate</div> <div class="given-names">D. J.</div></div>. </div><div class="article-title">Comparative genome mapping in Brassica</div>, <div class="source">Genetics</div> , <div class="year">1996</div>, vol. <div class="volume">144</div> (pg. <div class="fpage">1903</div>-<div class="lpage">1910</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genome%20mapping%20in%20Brassica&amp;author=U.%20Lagercrantz&amp;author=D.%20J.%20Lydiate&amp;publication_year=1996&amp;journal=Genetics&amp;volume=144&amp;pages=1903-1910" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8978073" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moore</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Devos</div> <div class="given-names">K. M.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Gale</div> <div class="given-names">M. D.</div></div>. </div><div class="article-title">Cereal genome evolution—grasses, line up and form a circle</div>, <div class="source">Curr. Biol.</div> , <div class="year">1995</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">737</div>-<div class="lpage">739</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cereal%20genome%20evolution%E2%80%94grasses%2C%20line%20up%20and%20form%20a%20circle&amp;author=G.%20Moore&amp;author=K.%20M.%20Devos&amp;author=Z.%20Wang&amp;author=M.%20D.%20Gale&amp;publication_year=1995&amp;journal=Curr.%20Biol.&amp;volume=5&amp;pages=737-739" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0960-9822(95)00148-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0960-9822(95)00148-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0960-9822(95)00148-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7583118" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ventelon</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Deu</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Garsmeur</div> <div class="given-names">O.</div></div>, et al. </div><div class="article-title">A direct comparison between the genetic maps of sorghum and rice</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2001</div>, vol. <div class="volume">102</div> (pg. <div class="fpage">379</div>-<div class="lpage">386</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20direct%20comparison%20between%20the%20genetic%20maps%20of%20sorghum%20and%20rice&amp;author=M.%20Ventelon&amp;author=M.%20Deu&amp;author=O.%20Garsmeur&amp;publication_year=2001&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=102&amp;pages=379-386" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220051657" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220051657" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220051657"> </span></div> </div></div></div></div></div><div content-id="DSM009C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Choi</div> <div class="given-names">H.-K.</div></div>,  <div class="name"><div class="surname">Mun</div> <div class="given-names">J.-H.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">D.-J.</div></div>, et al. </div><div class="article-title">Estimating genome conservation between crop and model legume species</div>, <div class="source">PNAS</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">15289</div>-<div class="lpage">15294</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimating%20genome%20conservation%20between%20crop%20and%20model%20legume%20species&amp;author=H.-K.%20Choi&amp;author=J.-H.%20Mun&amp;author=D.-J.%20Kim&amp;publication_year=2004&amp;journal=PNAS&amp;volume=101&amp;pages=15289-15294" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0402251101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0402251101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0402251101"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15489274" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kalo</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Seres</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Taylor</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Comparative mapping between <em>Medicago sativa</em> and <em>Pisum sativum</em></div>, <div class="source">Mol. Genet. Genomics</div> , <div class="year">2004</div>, vol. <div class="volume">272</div> (pg. <div class="fpage">235</div>-<div class="lpage">246</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20mapping%20between%20Medicago%20sativa%20and%20Pisum%20sativum&amp;author=P.%20Kalo&amp;author=A.%20Seres&amp;author=S.%20Taylor&amp;publication_year=2004&amp;journal=Mol.%20Genet.%20Genomics&amp;volume=272&amp;pages=235-246" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00438-004-1055-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00438-004-1055-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00438-004-1055-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15340836" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhu</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Choi</div> <div class="given-names">H.-K.</div></div>,  <div class="name"><div class="surname">Cook</div> <div class="given-names">D. R.</div></div>,  <div class="name"><div class="surname">Shoemaker</div> <div class="given-names">R. C.</div></div>. </div><div class="article-title">Bridging model and crop legumes through comparative genomics</div>, <div class="source">Plant Physiol.</div> , <div class="year">2005</div>, vol. <div class="volume">137</div> (pg. <div class="fpage">1189</div>-<div class="lpage">1196</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bridging%20model%20and%20crop%20legumes%20through%20comparative%20genomics&amp;author=H.%20Zhu&amp;author=H.-K.%20Choi&amp;author=D.%20R.%20Cook&amp;author=R.%20C.%20Shoemaker&amp;publication_year=2005&amp;journal=Plant%20Physiol.&amp;volume=137&amp;pages=1189-1196" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.104.058891" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.104.058891" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.104.058891"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15824281" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nelson</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Phan</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ellwood</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">The first gene-based map of <em>Lupinus angustifolius</em> L.-location of domestication genes and conserved synteny with <em>Medicago truncatula</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">113</div> (pg. <div class="fpage">225</div>-<div class="lpage">238</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20first%20gene-based%20map%20of%20Lupinus%20angustifolius%20L.-location%20of%20domestication%20genes%20and%20conserved%20synteny%20with%20Medicago%20truncatula&amp;author=M.%20Nelson&amp;author=H.%20Phan&amp;author=S.%20Ellwood&amp;publication_year=2006&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=113&amp;pages=225-238" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-006-0288-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-006-0288-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-006-0288-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16791689" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Phan</div> <div class="given-names">H. T. T.</div></div>,  <div class="name"><div class="surname">Ellwood</div> <div class="given-names">S. R.</div></div>,  <div class="name"><div class="surname">Ford</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Thomas</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Oliver</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Differences in syntenic complexity between <em>Medicago truncatula</em> with <em>Lens culinaris</em> and <em>Lupinus albus</em></div>, <div class="source">Funct. Plant Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">775</div>-<div class="lpage">782</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differences%20in%20syntenic%20complexity%20between%20Medicago%20truncatula%20with%20Lens%20culinaris%20and%20Lupinus%20albus&amp;author=H.%20T.%20T.%20Phan&amp;author=S.%20R.%20Ellwood&amp;author=R.%20Ford&amp;author=S.%20Thomas&amp;author=R.%20Oliver&amp;publication_year=2006&amp;journal=Funct.%20Plant%20Biol.&amp;volume=33&amp;pages=775-782" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1071/FP06102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1071%2FFP06102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1071%2FFP06102"> </span></div> </div></div></div></div></div><div content-id="DSM009C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lukens</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Zou</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Lydiate</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Parkin</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Osborn</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Comparison of a <em>Brassica oleracea</em> genetic map with the genome of <em>Arabidopsis thaliana</em></div>, <div class="source">Genetics</div> , <div class="year">2003</div>, vol. <div class="volume">164</div> (pg. <div class="fpage">359</div>-<div class="lpage">372</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20a%20Brassica%20oleracea%20genetic%20map%20with%20the%20genome%20of%20Arabidopsis%20thaliana&amp;author=L.%20Lukens&amp;author=F.%20Zou&amp;author=D.%20Lydiate&amp;author=I.%20Parkin&amp;author=T.%20Osborn&amp;publication_year=2003&amp;journal=Genetics&amp;volume=164&amp;pages=359-372" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12750346" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Weeden</div> <div class="given-names">N. F.</div></div>,  <div class="name"><div class="surname">Muehlbauer</div> <div class="given-names">F. J.</div></div>,  <div class="name"><div class="surname">Ladizinsky</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Extensive conservation of linkage relationships between pea and lentil genetic maps</div>, <div class="source">J. Hered.</div> , <div class="year">1992</div>, vol. <div class="volume">83</div> (pg. <div class="fpage">123</div>-<div class="lpage">129</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extensive%20conservation%20of%20linkage%20relationships%20between%20pea%20and%20lentil%20genetic%20maps&amp;author=N.%20F.%20Weeden&amp;author=F.%20J.%20Muehlbauer&amp;author=G.%20Ladizinsky&amp;publication_year=1992&amp;journal=J.%20Hered.&amp;volume=83&amp;pages=123-129" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.jhered.a111171" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.jhered.a111171" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.jhered.a111171"> </span></div> </div></div></div></div></div><div content-id="DSM009C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Phan</div> <div class="given-names">H. T. T.</div></div>,  <div class="name"><div class="surname">Ellwood</div> <div class="given-names">S. R.</div></div>,  <div class="name"><div class="surname">Hane</div> <div class="given-names">J. K.</div></div>,  <div class="name"><div class="surname">Ford</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Materne</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Oliver</div> <div class="given-names">R. P.</div></div>. </div><div class="article-title">Extensive macrosynteny between <em>Medicago truncatula</em> and <em>Lens culinaris</em> ssp. <em>culinaris</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2007</div>, vol. <div class="volume">114</div> (pg. <div class="fpage">549</div>-<div class="lpage">558</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extensive%20macrosynteny%20between%20Medicago%20truncatula%20and%20Lens%20culinaris%20ssp.%20culinaris&amp;author=H.%20T.%20T.%20Phan&amp;author=S.%20R.%20Ellwood&amp;author=J.%20K.%20Hane&amp;author=R.%20Ford&amp;author=M.%20Materne&amp;author=R.%20P.%20Oliver&amp;publication_year=2007&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=114&amp;pages=549-558" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-006-0455-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-006-0455-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-006-0455-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17119911" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Aubert</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Morin</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Jacquin</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">Functional mapping in pea, as an aid to the candidate gene selection and for investigating synteny with the model legume <em>Medicago truncatula</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">112</div> (pg. <div class="fpage">1024</div>-<div class="lpage">1041</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20mapping%20in%20pea%2C%20as%20an%20aid%20to%20the%20candidate%20gene%20selection%20and%20for%20investigating%20synteny%20with%20the%20model%20legume%20Medicago%20truncatula&amp;author=G.%20Aubert&amp;author=J.%20Morin&amp;author=F.%20Jacquin&amp;publication_year=2006&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=112&amp;pages=1024-1041" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-005-0205-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-005-0205-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-005-0205-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16416153" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Choi</div> <div class="given-names">H. K.</div></div>,  <div class="name"><div class="surname">Luckow</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Doyle</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Cook</div> <div class="given-names">D. R.</div></div>. </div><div class="article-title">Development of nuclear gene-derived molecular markers linked to legume genetic maps</div>, <div class="source">Mol. Genet. Genomics</div> , <div class="year">2006</div>, vol. <div class="volume">276</div> (pg. <div class="fpage">56</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20nuclear%20gene-derived%20molecular%20markers%20linked%20to%20legume%20genetic%20maps&amp;author=H.%20K.%20Choi&amp;author=M.%20A.%20Luckow&amp;author=J.%20Doyle&amp;author=D.%20R.%20Cook&amp;publication_year=2006&amp;journal=Mol.%20Genet.%20Genomics&amp;volume=276&amp;pages=56-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00438-006-0118-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00438-006-0118-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00438-006-0118-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16642337" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Choi</div> <div class="given-names">H.-K.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Uhm</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">A sequence-based genetic map of <em>Medicago truncatula</em> and comparison of marker colinearity with <em>M. sativa</em></div>, <div class="source">Genetics</div> , <div class="year">2004</div>, vol. <div class="volume">166</div> (pg. <div class="fpage">1463</div>-<div class="lpage">1502</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20sequence-based%20genetic%20map%20of%20Medicago%20truncatula%20and%20comparison%20of%20marker%20colinearity%20with%20M.%20sativa&amp;author=H.-K.%20Choi&amp;author=D.%20Kim&amp;author=T.%20Uhm&amp;publication_year=2004&amp;journal=Genetics&amp;volume=166&amp;pages=1463-1502" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.166.3.1463" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.166.3.1463" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.166.3.1463"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15082563" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eujayl</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Sledge</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title"><em>Medicago truncatula</em> EST-SSRs reveal cross-species genetic markers for <em>Medicago</em> spp</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">108</div> (pg. <div class="fpage">414</div>-<div class="lpage">422</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Medicago%20truncatula%20EST-SSRs%20reveal%20cross-species%20genetic%20markers%20for%20Medicago%20spp&amp;author=I.%20Eujayl&amp;author=M.%20Sledge&amp;author=L.%20Wang&amp;publication_year=2004&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=108&amp;pages=414-422" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-003-1450-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-003-1450-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-003-1450-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/13679975" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gutierrez</div> <div class="given-names">M. V.</div></div>,  <div class="name"><div class="surname">Vaz Patto</div> <div class="given-names">M. C.</div></div>,  <div class="name"><div class="surname">Huguet</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Cubero</div> <div class="given-names">J. I.</div></div>,  <div class="name"><div class="surname">Moreno</div> <div class="given-names">M. T.</div></div>,  <div class="name"><div class="surname">Torres</div> <div class="given-names">A. M.</div></div>. </div><div class="article-title">Cross-species amplification of <em>Medicago truncatula</em> microsatellites across three major pulse crops</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">110</div> (pg. <div class="fpage">1210</div>-<div class="lpage">1217</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cross-species%20amplification%20of%20Medicago%20truncatula%20microsatellites%20across%20three%20major%20pulse%20crops&amp;author=M.%20V.%20Gutierrez&amp;author=M.%20C.%20Vaz%20Patto&amp;author=T.%20Huguet&amp;author=J.%20I.%20Cubero&amp;author=M.%20T.%20Moreno&amp;author=A.%20M.%20Torres&amp;publication_year=2005&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=110&amp;pages=1210-1217" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-005-1951-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-005-1951-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-005-1951-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15806349" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vos</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Hogers</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Bleeker</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">AFLP: a new technique for DNA fingerprinting</div>, <div class="source">Nucl. Acids Res.</div> , <div class="year">1995</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">4407</div>-<div class="lpage">4414</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AFLP%3A%20a%20new%20technique%20for%20DNA%20fingerprinting&amp;author=P.%20Vos&amp;author=R.%20Hogers&amp;author=M.%20Bleeker&amp;publication_year=1995&amp;journal=Nucl.%20Acids%20Res.&amp;volume=23&amp;pages=4407-4414" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/23.21.4407" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F23.21.4407" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F23.21.4407"> </span></div> </div></div></div></div></div><div content-id="DSM009C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ellwood</div> <div class="given-names">S. R.</div></div>,  <div class="name"><div class="surname">D'Souza</div> <div class="given-names">N. K.</div></div>,  <div class="name"><div class="surname">Kamphuis</div> <div class="given-names">L. G.</div></div>,  <div class="name"><div class="surname">Burgess</div> <div class="given-names">T. I.</div></div>,  <div class="name"><div class="surname">Nair</div> <div class="given-names">R. M.</div></div>,  <div class="name"><div class="surname">Oliver</div> <div class="given-names">R. P.</div></div>. </div><div class="article-title">SSR analysis of the <em>Medicago truncatula</em> SARDI core collection reveals substantial diversity and unusual genotype dispersal throughout the Mediterranean basin</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">112</div> (pg. <div class="fpage">977</div>-<div class="lpage">983</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SSR%20analysis%20of%20the%20Medicago%20truncatula%20SARDI%20core%20collection%20reveals%20substantial%20diversity%20and%20unusual%20genotype%20dispersal%20throughout%20the%20Mediterranean%20basin&amp;author=S.%20R.%20Ellwood&amp;author=N.%20K.%20D'Souza&amp;author=L.%20G.%20Kamphuis&amp;author=T.%20I.%20Burgess&amp;author=R.%20M.%20Nair&amp;author=R.%20P.%20Oliver&amp;publication_year=2006&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=112&amp;pages=977-983" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-005-0202-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-005-0202-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-005-0202-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16402186" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Manly</div> <div class="given-names">K. F.</div></div>,  <div class="name"><div class="surname">Cudmore</div> <div class="given-names">R. H.</div></div>,  <div class="name"><div class="surname">Meer</div> <div class="given-names">J. M.</div></div>. </div><div class="article-title">Map Manager QTX, cross-platform software for genetic mapping</div>, <div class="source">Mamm. Genome</div> , <div class="year">2001</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">930</div>-<div class="lpage">932</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Map%20Manager%20QTX%2C%20cross-platform%20software%20for%20genetic%20mapping&amp;author=K.%20F.%20Manly&amp;author=R.%20H.%20Cudmore&amp;author=J.%20M.%20Meer&amp;publication_year=2001&amp;journal=Mamm.%20Genome&amp;volume=12&amp;pages=930-932" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00335-001-1016-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00335-001-1016-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00335-001-1016-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11707780" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">King</div> <div class="given-names">G. J.</div></div>. </div><div class="article-title">Through a genome, darkly: comparative analysis of plant chromosomal DNA</div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">2002</div>, vol. <div class="volume">48</div> (pg. <div class="fpage">5</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Through%20a%20genome%2C%20darkly%3A%20comparative%20analysis%20of%20plant%20chromosomal%20DNA&amp;author=G.%20J.%20King&amp;publication_year=2002&amp;journal=Plant%20Mol.%20Biol.&amp;volume=48&amp;pages=5-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1013705704925" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1013705704925" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1013705704925"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11860213" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Boersma</div> <div class="given-names">J. G.</div></div>,  <div class="name"><div class="surname">Pallotta</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Buirchell</div> <div class="given-names">B. J.</div></div>,  <div class="name"><div class="surname">Sivasithamparam</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Construction of a genetic linkage map using MFLP and identification of molecular markers linked to domestication genes in narrow-leafed lupin (<em>Lupinus angustifolius</em> L.)</div>, <div class="source">Cell. Mol. Biol. Lett.</div> , <div class="year">2005</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">331</div>-<div class="lpage">344</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20genetic%20linkage%20map%20using%20MFLP%20and%20identification%20of%20molecular%20markers%20linked%20to%20domestication%20genes%20in%20narrow-leafed%20lupin%20(Lupinus%20angustifolius%20L.)&amp;author=J.%20G.%20Boersma&amp;author=M.%20Pallotta&amp;author=C.%20Li&amp;author=B.%20J.%20Buirchell&amp;author=K.%20Sivasithamparam&amp;author=H.%20Yang&amp;publication_year=2005&amp;journal=Cell.%20Mol.%20Biol.%20Lett.&amp;volume=10&amp;pages=331-344" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16010297" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Flandez-Galvez</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ford</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Pang</div> <div class="given-names">E. C. K.</div></div>,  <div class="name"><div class="surname">Taylor</div> <div class="given-names">P. W. J.</div></div>. </div><div class="article-title">An intraspecific linkage map of the chickpea (<em>Cicer arietinum</em> L.) genome based on sequence tagged microsatellite site and resistance gene analog markers</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2003</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">1447</div>-<div class="lpage">1456</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20intraspecific%20linkage%20map%20of%20the%20chickpea%20(Cicer%20arietinum%20L.)%20genome%20based%20on%20sequence%20tagged%20microsatellite%20site%20and%20resistance%20gene%20analog%20markers&amp;author=H.%20Flandez-Galvez&amp;author=R.%20Ford&amp;author=E.%20C.%20K.%20Pang&amp;author=P.%20W.%20J.%20Taylor&amp;publication_year=2003&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=106&amp;pages=1447-1456" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-003-1199-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-003-1199-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-003-1199-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12750788" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Muehlbauer</div> <div class="given-names">F. J.</div></div>,  <div class="name"><div class="surname">Cho</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sarker</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Application of biotechnology in breeding lentil for resistance to biotic and abiotic stress</div>, <div class="source">Euphytica</div> , <div class="year">2006</div>, vol. <div class="volume">147</div> (pg. <div class="fpage">149</div>-<div class="lpage">165</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Application%20of%20biotechnology%20in%20breeding%20lentil%20for%20resistance%20to%20biotic%20and%20abiotic%20stress&amp;author=F.%20J.%20Muehlbauer&amp;author=S.%20Cho&amp;author=A.%20Sarker&amp;publication_year=2006&amp;journal=Euphytica&amp;volume=147&amp;pages=149-165" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10681-006-7108-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10681-006-7108-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10681-006-7108-0"> </span></div> </div></div></div></div></div><div content-id="DSM009C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Paran</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Goldman</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Tanksley</div> <div class="given-names">S. D.</div></div>,  <div class="name"><div class="surname">Zamir</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Recombinant inbred lines for genetic mapping in tomato</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">1995</div>, vol. <div class="volume">90</div> (pg. <div class="fpage">542</div>-<div class="lpage">548</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recombinant%20inbred%20lines%20for%20genetic%20mapping%20in%20tomato&amp;author=I.%20Paran&amp;author=I.%20Goldman&amp;author=S.%20D.%20Tanksley&amp;author=D.%20Zamir&amp;publication_year=1995&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=90&amp;pages=542-548" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00222001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00222001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00222001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24173949" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Quillet</div> <div class="given-names">M. C.</div></div>,  <div class="name"><div class="surname">Madjidian</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Griveau</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Mapping genetic factors controlling pollen viability in an interspecific cross in <em>Helianthus</em> sect. <em>Helianthus</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">1995</div>, vol. <div class="volume">91</div> (pg. <div class="fpage">1195</div>-<div class="lpage">1202</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20genetic%20factors%20controlling%20pollen%20viability%20in%20an%20interspecific%20cross%20in%20Helianthus%20sect.%20Helianthus&amp;author=M.%20C.%20Quillet&amp;author=N.%20Madjidian&amp;author=Y.%20Griveau&amp;publication_year=1995&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=91&amp;pages=1195-1202" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00220929" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00220929" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00220929"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24170046" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tadmor</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Zamir</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Ladizinsky</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Genetic mapping of an ancient translocation in the genus <em>Lens</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">1987</div>, vol. <div class="volume">73</div> (pg. <div class="fpage">883</div>-<div class="lpage">892</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20mapping%20of%20an%20ancient%20translocation%20in%20the%20genus%20Lens&amp;author=Y.%20Tadmor&amp;author=D.%20Zamir&amp;author=G.%20Ladizinsky&amp;publication_year=1987&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=73&amp;pages=883-892" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00289394" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00289394" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00289394"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24241299" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xu</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Xiao</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Huang</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">McCouch</div> <div class="given-names">S. R.</div></div>. </div><div class="article-title">Chromosomal regions associated with segregation distortion of molecular markers in F<sub>2</sub>, backcross, doubled haploid, and recombinant inbred populations in rice (<em>Oryza sativa</em> L.)</div>, <div class="source">Mol. Genet. Genomics</div> , <div class="year">1997</div>, vol. <div class="volume">253</div> (pg. <div class="fpage">535</div>-<div class="lpage">545</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosomal%20regions%20associated%20with%20segregation%20distortion%20of%20molecular%20markers%20in%20F2%2C%20backcross%2C%20doubled%20haploid%2C%20and%20recombinant%20inbred%20populations%20in%20rice%20(Oryza%20sativa%20L.)&amp;author=Y.%20Xu&amp;author=L.%20Zhu&amp;author=J.%20Xiao&amp;author=N.%20Huang&amp;author=S.%20R.%20McCouch&amp;publication_year=1997&amp;journal=Mol.%20Genet.%20Genomics&amp;volume=253&amp;pages=535-545" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s004380050355" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs004380050355" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs004380050355"> </span></div> </div></div></div></div></div><div content-id="DSM009C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhu</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Cannon</div> <div class="given-names">S. B.</div></div>,  <div class="name"><div class="surname">Young</div> <div class="given-names">N. D.</div></div>,  <div class="name"><div class="surname">Cook</div> <div class="given-names">D. R.</div></div>. </div><div class="article-title">Phylogeny and genomic organization of the TIR and non-TIR NBS-LRR resistance gene family in <em>Medicago truncatula</em></div>, <div class="source">Mol. Plant Microbe Interact.</div> , <div class="year">2002</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">529</div>-<div class="lpage">359</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogeny%20and%20genomic%20organization%20of%20the%20TIR%20and%20non-TIR%20NBS-LRR%20resistance%20gene%20family%20in%20Medicago%20truncatula&amp;author=H.%20Zhu&amp;author=S.%20B.%20Cannon&amp;author=N.%20D.%20Young&amp;author=D.%20R.%20Cook&amp;publication_year=2002&amp;journal=Mol.%20Plant%20Microbe%20Interact.&amp;volume=15&amp;pages=529-359" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI.2002.15.6.529" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI.2002.15.6.529" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI.2002.15.6.529"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12059101" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kuittinen</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Sillanpaa</div> <div class="given-names">M. J.</div></div>,  <div class="name"><div class="surname">Savolainen</div> <div class="given-names">O.</div></div>. </div><div class="article-title">Genetic basis of adaptation: flowering time in <em>Arabidopsis thaliana</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">1997</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">573</div>-<div class="lpage">583</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20basis%20of%20adaptation%3A%20flowering%20time%20in%20Arabidopsis%20thaliana&amp;author=H.%20Kuittinen&amp;author=M.%20J.%20Sillanpaa&amp;author=O.%20Savolainen&amp;publication_year=1997&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=95&amp;pages=573-583" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220050598" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220050598" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220050598"> </span></div> </div></div></div></div></div><div content-id="DSM009C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lichtenzveig</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bonfil</div> <div class="given-names">D. J.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">H.-B.</div></div>,  <div class="name"><div class="surname">Shtienberg</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Abbo</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Mapping quantitative trait loci in chickpea associated with time to flowering and resistance to <em>Didymella rabiei</em> the causal agent of <em>Ascochyta</em> blight</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">113</div> (pg. <div class="fpage">1357</div>-<div class="lpage">1369</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20quantitative%20trait%20loci%20in%20chickpea%20associated%20with%20time%20to%20flowering%20and%20resistance%20to%20Didymella%20rabiei%20the%20causal%20agent%20of%20Ascochyta%20blight&amp;author=J.%20Lichtenzveig&amp;author=D.%20J.%20Bonfil&amp;author=H.-B.%20Zhang&amp;author=D.%20Shtienberg&amp;author=S.%20Abbo&amp;publication_year=2006&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=113&amp;pages=1357-1369" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-006-0390-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-006-0390-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-006-0390-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17016689" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamanaka</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Ninomiya</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Hoshi</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">An informative linkage map of soybean reveals QTLs for flowering time, leaflet morphology and regions of segregation distortion</div>, <div class="source">DNA Res</div> , <div class="year">2001</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">61</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20informative%20linkage%20map%20of%20soybean%20reveals%20QTLs%20for%20flowering%20time%2C%20leaflet%20morphology%20and%20regions%20of%20segregation%20distortion&amp;author=N.%20Yamanaka&amp;author=S.%20Ninomiya&amp;author=M.%20Hoshi&amp;publication_year=2001&amp;journal=DNA%20Res&amp;volume=8&amp;pages=61-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/8.2.61" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F8.2.61" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F8.2.61"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11347903" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">B. H.</div></div>. </div>, <div class="source">Statistical Genomics: Linkage, Mapping and QTL Analysis</div> , <div class="year">1985</div><div class="publisher-loc">Boca Raton</div><div class="publisher-name">CRC Press</div></div></div></div></div><div content-id="DSM009C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tanksley</div> <div class="given-names">S. D.</div></div>. </div><div class="article-title">Mapping polygenes</div>, <div class="source">Annual Review of Genetics</div> , <div class="year">1993</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">205</div>-<div class="lpage">233</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20polygenes&amp;author=S.%20D.%20Tanksley&amp;publication_year=1993&amp;journal=Annual%20Review%20of%20Genetics&amp;volume=27&amp;pages=205-233" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.ge.27.120193.001225" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.ge.27.120193.001225" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.ge.27.120193.001225"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8122902" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Harrison</div> <div class="given-names">J. E. M.</div></div>,  <div class="name"><div class="surname">Williams</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Genetical control of alkaloids in Lupinus albus</div>, <div class="source">Euphytica</div> , <div class="year">1982</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">357</div>-<div class="lpage">364</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetical%20control%20of%20alkaloids%20in%20Lupinus%20albus&amp;author=J.%20E.%20M.%20Harrison&amp;author=W.%20Williams&amp;publication_year=1982&amp;journal=Euphytica&amp;volume=31&amp;pages=357-364" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00021651" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00021651" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00021651"> </span></div> </div></div></div></div></div><div content-id="DSM009C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yan</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Mudge</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Shoemaker</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Cook</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Young</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Comparative physical mapping reveals features of microsynteny between <em>Glycine max</em>, <em>Medicago truncatula</em>, and <em>Arabidopsis thaliana</em></div>, <div class="source">Genome</div> , <div class="year">2004</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">141</div>-<div class="lpage">155</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20physical%20mapping%20reveals%20features%20of%20microsynteny%20between%20Glycine%20max%2C%20Medicago%20truncatula%2C%20and%20Arabidopsis%20thaliana&amp;author=H.%20Yan&amp;author=J.%20Mudge&amp;author=D.%20Kim&amp;author=R.%20Shoemaker&amp;author=D.%20Cook&amp;author=N.%20Young&amp;publication_year=2004&amp;journal=Genome&amp;volume=47&amp;pages=141-155" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g03-106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg03-106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg03-106"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15060611" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">van der Vossen</div> <div class="given-names">E. A.</div></div>,  <div class="name"><div class="surname">Kuang</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Comparative genomics enabled the isolation of the <em>R3a</em> late blight resistance gene in potato</div>, <div class="source">Plant J.</div> , <div class="year">2005</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">251</div>-<div class="lpage">261</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genomics%20enabled%20the%20isolation%20of%20the%20R3a%20late%20blight%20resistance%20gene%20in%20potato&amp;author=S.%20Huang&amp;author=E.%20A.%20van%20der%20Vossen&amp;author=H.%20Kuang&amp;publication_year=2005&amp;journal=Plant%20J.&amp;volume=42&amp;pages=251-261" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2005.42.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2005.42.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2005.42.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15807786" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Armstead</div> <div class="given-names">I. P.</div></div>,  <div class="name"><div class="surname">Skot</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Turner</div> <div class="given-names">L. B.</div></div>, et al. </div><div class="article-title">Identification of perennial ryegrass (<em>Lolium perenne</em> (L.)) and meadow fescue (<em>Festuca pratensis</em> (Huds.)) candidate orthologous sequences to the rice Hd1(Se1) and barley HvCO1 CONSTANS-like genes through comparative mapping and microsynteny</div>, <div class="source">New Phytol.</div> , <div class="year">2005</div>, vol. <div class="volume">167</div> (pg. <div class="fpage">239</div>-<div class="lpage">247</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20perennial%20ryegrass%20(Lolium%20perenne%20(L.))%20and%20meadow%20fescue%20(Festuca%20pratensis%20(Huds.))%20candidate%20orthologous%20sequences%20to%20the%20rice%20Hd1(Se1)%20and%20barley%20HvCO1%20CONSTANS-like%20genes%20through%20comparative%20mapping%20and%20microsynteny&amp;author=I.%20P.%20Armstead&amp;author=L.%20Skot&amp;author=L.%20B.%20Turner&amp;publication_year=2005&amp;journal=New%20Phytol.&amp;volume=167&amp;pages=239-247" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-8137.2005.01392.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-8137.2005.01392.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-8137.2005.01392.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15948846" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Börner</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Korzun</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Worland</div> <div class="given-names">A. J.</div></div>. </div><div class="article-title">Comparative genetic mapping of loci effecting plant height and development in cereals</div>, <div class="source">Euphytica</div> , <div class="year">1998</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">245</div>-<div class="lpage">248</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genetic%20mapping%20of%20loci%20effecting%20plant%20height%20and%20development%20in%20cereals&amp;author=A.%20B%C3%B6rner&amp;author=V.%20Korzun&amp;author=A.%20J.%20Worland&amp;publication_year=1998&amp;journal=Euphytica&amp;volume=100&amp;pages=245-248" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1018364425150" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1018364425150" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1018364425150"> </span></div> </div></div></div></div></div><div content-id="DSM009C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mammadov</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Steffenson</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Maroof</div> <div class="given-names">M.</div></div>. </div><div class="article-title">High-resolution mapping of the barley leaf rust resistance gene <em>Rph5</em> using barley expressed sequence tags (ESTs) and synteny with rice</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">111</div> (pg. <div class="fpage">1651</div>-<div class="lpage">1660</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-resolution%20mapping%20of%20the%20barley%20leaf%20rust%20resistance%20gene%20Rph5%20using%20barley%20expressed%20sequence%20tags%20(ESTs)%20and%20synteny%20with%20rice&amp;author=J.%20Mammadov&amp;author=B.%20Steffenson&amp;author=M.%20Maroof&amp;publication_year=2005&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=111&amp;pages=1651-1660" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-005-0100-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-005-0100-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-005-0100-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16195886" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM009C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM009C56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dirlewanger</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Graziano</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Joobeur</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Comparative mapping and marker-assisted selection in Rosaceae fruit crops</div>, <div class="source">PNAS</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">9891</div>-<div class="lpage">9896</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20mapping%20and%20marker-assisted%20selection%20in%20Rosaceae%20fruit%20crops&amp;author=E.%20Dirlewanger&amp;author=E.%20Graziano&amp;author=T.%20Joobeur&amp;publication_year=2004&amp;journal=PNAS&amp;volume=101&amp;pages=9891-9896" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0307937101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0307937101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0307937101"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15159547" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2007. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=333979&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/2/10.1093_dnares_dsm009/1/dsm009_Supplementary_Data.zip?Expires=2147483647&amp;Signature=NRYWDqdRWRYGdttwpPO6STibE2VRwVYhxUbVk1DYvgcbqDZaFnGK6sO6plve9pUXYcfYPL~aw~0yVRAqo3IYBG~qXw~lYla5VuKBCLsjqXqLDbSkjJtHH3CvRWlUBtygyvR9vAlJ0YeWgkLIlrk-q4D34kjAZ2qSREVIm4qJ300isZnkTs4CDu2koC1qAkSXY~gZHmnaPI2Yek6skmV1iSJ69tTZI9fbWp~50BhnrigUH~agkUFAbGxh1ZfCnF5RpB1Jt7waO6tegTZEtzfsy-VnQyX4PCDO~Ukncy4GypaK1DIuqo8YsGZBnqlbcolYSLjlwdpCHvcMcnUbaxzAZw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">284</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000248042600002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">40</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsm009"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="333979"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000248042600002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f14%2f2%2f59%2f333979">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000248042600002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (40)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F14%2F2%2F59%2F333979" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/333979'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="First Genetic and Comparative Map of White Lupin (Lupinus albus L.): Identification of QTLs for Anthracnose Resistance and Flowering Time, and a Locus for Alkaloid Content" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsm009");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f14%2f2%2f59%2f333979">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

