

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/12/6/417/343854","@type":"ScholarlyArticle","name":" Genome-wide Investigation of Intron Length Polymorphisms and Their Potential as Molecular Markers in Rice (  Oryza sativa  L.) ","datePublished":"2005-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/12/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2005-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsi019","keywords":[" rice ( <italic>Oryza sativa</italic> L.) ","intron length polymorphism (ILP)","molecular marker","genome"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/12/6/417/343854","author":[{"name":"Wang, Xusheng","affiliation":"Institute of Bioinformatics, Huajiachi Campus, Zhejiang University  Hangzhou, 310029, P. R. China","@type":"Person"},{"name":"Zhao, Xiangqian","affiliation":"Institute of Bioinformatics, Huajiachi Campus, Zhejiang University  Hangzhou, 310029, P. R. China","@type":"Person"},{"name":"Zhu, Jun","affiliation":"Institute of Bioinformatics, Huajiachi Campus, Zhejiang University  Hangzhou, 310029, P. R. China","@type":"Person"},{"name":"Wu, Weiren","affiliation":"Institute of Bioinformatics, Huajiachi Campus, Zhejiang University  Hangzhou, 310029, P. R. China","@type":"Person"}],"description":"Abstract.   Intron length polymorphisms (ILPs) have been used as genetic markers in some studies. However, a systematic investigation and large-scale exploitati","pageStart":"417","pageEnd":"427","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/12/6/0/m_dnaresearch12_6.cover.gif?Expires=2147483647&Signature=vmBj-dEpEdSISGGZZVQQ800UdbaQ61VgN5SO6nMO30mtIcmmHegLeBqq~JmeB3Cl71F-HZSxEtC05awn7JLvbmS~R8ehKSKlG67TzUmDNfl2uJ9iwd8X3XVTg535TrTFouPO8FrSAc3H4exNAvHuAshR8~cghIB3WqxBjwF3q4V-gWs6yVOhRl523VyJV6-aueYhJA8pI1jFRHUmPhSY-RKzDUrwgfwrVrH153kk1IiVH3chlu~YmBF~XczCNeM~pNFI-0DwLreboeiSvqsHnGPMQGgfWos-lNOnEyjyXTWeiJBGXgUTUflQE1fArs2l0eo950LpKbLah37z6bPoSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":" Genome-wide Investigation of Intron Length Polymorphisms and Their Potential as Molecular Markers in Rice (  Oryza sativa  L.) ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/12/6/0/m_dnaresearch12_6.cover.gif?Expires=2147483647&Signature=vmBj-dEpEdSISGGZZVQQ800UdbaQ61VgN5SO6nMO30mtIcmmHegLeBqq~JmeB3Cl71F-HZSxEtC05awn7JLvbmS~R8ehKSKlG67TzUmDNfl2uJ9iwd8X3XVTg535TrTFouPO8FrSAc3H4exNAvHuAshR8~cghIB3WqxBjwF3q4V-gWs6yVOhRl523VyJV6-aueYhJA8pI1jFRHUmPhSY-RKzDUrwgfwrVrH153kk1IiVH3chlu~YmBF~XczCNeM~pNFI-0DwLreboeiSvqsHnGPMQGgfWos-lNOnEyjyXTWeiJBGXgUTUflQE1fArs2l0eo950LpKbLah37z6bPoSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content=" Genome-wide Investigation of Intron Length Polymorphisms and Their Potential as Molecular Markers in Rice (  Oryza sativa  L.) " />
    <meta property="og:description" content="Abstract.   Intron length polymorphisms (ILPs) have been used as genetic markers in some studies. However, a systematic investigation and large-scale exploitati" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/12/6/417/343854" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2005-01-01" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Wang, Xusheng" /><meta name="citation_author_institution" content="Institute of Bioinformatics, Huajiachi Campus, Zhejiang UniversityHangzhou, 310029, P. R. China" /><meta name="citation_author" content="Zhao, Xiangqian" /><meta name="citation_author_institution" content="Institute of Bioinformatics, Huajiachi Campus, Zhejiang UniversityHangzhou, 310029, P. R. China" /><meta name="citation_author" content="Zhu, Jun" /><meta name="citation_author_institution" content="Institute of Bioinformatics, Huajiachi Campus, Zhejiang UniversityHangzhou, 310029, P. R. China" /><meta name="citation_author" content="Wu, Weiren" /><meta name="citation_author_institution" content="Institute of Bioinformatics, Huajiachi Campus, Zhejiang UniversityHangzhou, 310029, P. R. China" /><meta name="citation_title" content=" Genome-wide Investigation of Intron Length Polymorphisms and Their Potential as Molecular Markers in Rice ( Oryza sativa L.) " /><meta name="citation_firstpage" content="417" /><meta name="citation_lastpage" content="427" /><meta name="citation_doi" content="10.1093/dnares/dsi019" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="12" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2005/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Spliced segments at the 5′ terminus of adenovirus 2 late mRNA; Proc. Natl Acad. Sci. USA;  citation_year=1977;  citation_volume=74;  citation_pages=3171-3175; " /><meta name="citation_reference" content="citation_title=An amazing sequence arrangement at the 5′ ends of adenovirus 2 messenger RNA; Cell;  citation_year=1977;  citation_volume=12;  citation_pages=1-8; " /><meta name="citation_reference" content="citation_title=A survey on intron and exon lengths; Nucleic Acids Res.;  citation_year=1988;  citation_volume=16;  citation_pages=9893-9908; " /><meta name="citation_reference" content="citation_title=Intron–exon structures of eukaryotic model organisms; Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=3219-3228; " /><meta name="citation_reference" content="citation_title=The genome sequence of Drosophila melanogaster; Science;  citation_year=2000;  citation_volume=287;  citation_pages=2185-2195; " /><meta name="citation_reference" content="citation_title=The sequence of the human genome; Science;  citation_year=2001;  citation_volume=291;  citation_pages=1304-1351; " /><meta name="citation_reference" content="citation_title=Large-scale identification, mapping, and genotyping of single-nucleotide polymorphisms in the human genome; Science;  citation_year=1998;  citation_volume=280;  citation_pages=1077-1082; " /><meta name="citation_reference" content="citation_title=Construction of a genetic linkage map in man using restriction fragment length polymorphisms; Am. J. Hum. Genet.;  citation_year=1980;  citation_volume=32;  citation_pages=314-331; " /><meta name="citation_reference" content="citation_title=DNA polymorphism amplified by arbitrary primers are useful as genetic markers; Nucleic Acids Res.;  citation_year=1990;  citation_volume=18;  citation_pages=6531-6535; " /><meta name="citation_reference" content="citation_title=AFLP: a new technique for DNA fingerprinting; Nucleic Acid Res.;  citation_year=1995;  citation_volume=23;  citation_pages=4407-4414; " /><meta name="citation_reference" content="citation_title=Barley microsatellites: allele variation and mapping; Plant Mol. Biol.;  citation_year=1995;  citation_volume=27;  citation_pages=835-845; " /><meta name="citation_reference" content="citation_title=Combined mapping of AFLP and RFLP markers in barley; Mol. Gen. Genet.;  citation_year=1995;  citation_volume=249;  citation_pages=65-73; " /><meta name="citation_reference" content="citation_title=Sequence-related amplified polymorphism (SRAP) a new marker system based on a simple PCR reaction: its application to mapping and gene tagging in Brassica; Theor. Appl. Genet.;  citation_year=2001;  citation_volume=103;  citation_pages=455-461; " /><meta name="citation_reference" content="citation_title=Variation is the spice of life; Nat. Genet.;  citation_year=2001;  citation_volume=27;  citation_pages=234-236; " /><meta name="citation_reference" content="citation_title=Rapid survey of DNA sequence variation in natural populations; Mol. Biol. Evol.;  citation_year=1992;  citation_volume=9;  citation_pages=323-330; " /><meta name="citation_reference" content="citation_title=Intron-targeted PCR: a new approach to survey neutral DNA polymorphism in bivalve populations; Mar. Biol.;  citation_year=1994;  citation_volume=120;  citation_pages=407-413; " /><meta name="citation_reference" content="citation_title=Genetic characterisation of Mytilus galloprovincialis Lmk. in North West Africa using nuclear DNA markers; J Exp. Mar. Biol. Ecol.;  citation_year=1999;  citation_volume=235;  citation_pages=55-65; " /><meta name="citation_reference" content="citation_title=Use of an intron length polymorphism to localize the tropoelastin gene to chromosome 5 in a region of linkage conservation with human chromosome 7; Genomics;  citation_year=1994;  citation_volume=23;  citation_pages=125-131; " /><meta name="citation_reference" content="citation_title=Nucleic acids II: the polymerase chain reaction;  citation_edition=2nd edn; citation_publisher=Sinauer Associates Inc., Sunderland, MA; Molecular Systematics;  citation_year=1995;  citation_pages=205-247; " /><meta name="citation_reference" content="citation_title=Screening for intron-length polymorphisms in penaeid shrimps using exon-primed intron-crossing (EPIC)-PCR; Mol. Ecol.;  citation_year=2000;  citation_volume=9;  citation_pages=233-235; " /><meta name="citation_reference" content="citation_title=A draft sequence of the rice genome ( Oryza sativa L. ssp. indica ); Science;  citation_year=2002;  citation_volume=296;  citation_pages=79-92; " /><meta name="citation_reference" content="citation_title=A draft sequence of the rice genomes ( Oryza sativa L. ssp. japonica ); Science;  citation_year=2002;  citation_volume=296;  citation_pages=92-100; " /><meta name="citation_reference" content="citation_title=Collection, mapping, and annotation of over 28 000 cDNA clones from japonica rice; Science;  citation_year=2003;  citation_volume=301;  citation_pages=376-379; " /><meta name="citation_reference" content="citation_title=The genome sequence and structure of rice chromosome 1; Nature;  citation_year=2002;  citation_volume=420;  citation_pages=312-316; " /><meta name="citation_reference" content="citation_title=Sequence and analysis of rice chromosome 4; Nature;  citation_year=2002;  citation_volume=420;  citation_pages=316-320; " /><meta name="citation_reference" content="citation_title=In-depth view of structure, activity, and evolution of rice chromosome 10; citation_author=The Rice Chromosome 10 Sequencing Consortium; Science;  citation_year=2003;  citation_volume=300;  citation_pages=1566-1569; " /><meta name="citation_reference" content="citation_title=The map-based sequence of the rice genome; citation_author=International Rice Genome Sequencing Project; Nature;  citation_year=2005;  citation_volume=436;  citation_pages=793-800; " /><meta name="citation_reference" content="citation_title=Development of genome-wide DNA polymorphism database for map-based cloning of rice genes; Plant physiol.;  citation_year=2004;  citation_volume=135;  citation_pages=1198-1205; " /><meta name="citation_reference" content="citation_title=An SNP resource for rice genetics and breeding based on subspecies indica and japonica genome alignments; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=1812-1819; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: A new generation of protein database search programs; Nucleic Acids. Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-3402; " /><meta name="citation_reference" content="citation_title=A computer program for aligning a cDNA sequence with a genomic DNA sequence; Genome Res.;  citation_year=1998;  citation_volume=8;  citation_pages=967-974; " /><meta name="citation_reference" content="citation_title=EMBOSS: The European Molecular Biology Open Software Suite; Trends Genet.;  citation_year=2000;  citation_volume=16;  citation_pages=276-277; " /><meta name="citation_reference" content="citation_title=Sequence mapping by electronic PCR; Genome Res.;  citation_year=1997;  citation_volume=7;  citation_pages=541-550; " /><meta name="citation_reference" content="citation_title=A high-density rice genetic linkage map with 2275 markers using a single F 2 population; Genetics;  citation_year=1998;  citation_volume=148;  citation_pages=479-494; " /><meta name="citation_reference" content="citation_title=Rapid isolation of high-molecular- weight plant DNA; Nucleic Acids. Res.;  citation_year=1980;  citation_volume=8;  citation_pages=4321-4325; " /><meta name="citation_reference" content="citation_title=‘Touchdown’ PCR to circumvent spurious priming during gene amplification; Nucleic Acids. Res.;  citation_year=1991;  citation_volume=19;  citation_pages=4008" /><meta name="citation_reference" content="citation_title=A simple and rapid method used for silver staining and gel preservation; Hereditas (Beijing);  citation_year=2002;  citation_volume=24;  citation_pages=335-336; " /><meta name="citation_reference" content="citation_title=Optimizing parental selection for genetic linkage maps; Genome;  citation_year=1993;  citation_volume=36;  citation_pages=181-186; " /><meta name="citation_reference" content="citation_title=Genome-wide intraspecific DNA-sequence variations in rice; Curr. Opin. Plant Biol.;  citation_year=2003;  citation_volume=6;  citation_pages=134-138; " /><meta name="citation_reference" content="citation_title=Semidwarf ( sd-1 ), “green revolution” rice, contains a defective gibberellin 20-oxidase gene; Proc. Natl Acad. Sci. USA;  citation_year=2002;  citation_volume=99;  citation_pages=9043-9048; " /><meta name="citation_reference" content="citation_title=Isozymes and classification of Asian rice varieties; Theor. Appl. Genet.;  citation_year=1987;  citation_volume=74;  citation_pages=21-30; " /><meta name="citation_reference" content="citation_title=Classifying japonica rice cultivars with RAPD markers; Crop Sci.;  citation_year=1995;  citation_volume=35;  citation_pages=889-894; " /><meta name="citation_reference" content="citation_title=Genetic differentiation of wild relatives of rice as assessed by RFLP analysis; Theor. Appl. Genet.;  citation_year=2002;  citation_volume=106;  citation_pages=101-106; " /><meta name="citation_reference" content="citation_title=The ClustalX windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools; Nucleic Acids Res.;  citation_year=1997;  citation_volume=24;  citation_pages=4876-4882; " /><meta name="citation_reference" content="Nicholas K. B. Nicholas H. B. GeneDoc: a tool for editing and annotating multiple sequence alignments 1997 Distributed by the authors ( ) " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/12/6/417/6718550/dsi019.pdf" /><meta name="description" content="Abstract.   Intron length polymorphisms (ILPs) have been used as genetic markers in some studies. However, a systematic investigation and large-scale exploitati" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/12/6/417/343854" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/12/6/417/343854" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/12/6/417/343854");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsi019';
    </script>
<title>
	 Genome-wide Investigation of Intron Length Polymorphisms and Their Potential as Molecular Markers in Rice ( Oryza sativa L.)  | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./343854" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BYNcPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXYW5nLCBYdXNoZW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIEJpb2luZm9ybWF0aWNzLCBIdWFqaWFjaGkgQ2FtcHVzLCBaaGVqaWFuZyBVbml2ZXJzaXR5SGFuZ3pob3UsIDMxMDAyOSwgUC4gUi4gQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJaaGFvLCBYaWFuZ3FpYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgb2YgQmlvaW5mb3JtYXRpY3MsIEh1YWppYWNoaSBDYW1wdXMsIFpoZWppYW5nIFVuaXZlcnNpdHlIYW5nemhvdSwgMzEwMDI5LCBQLiBSLiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlpodSwgSnVuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIEJpb2luZm9ybWF0aWNzLCBIdWFqaWFjaGkgQ2FtcHVzLCBaaGVqaWFuZyBVbml2ZXJzaXR5SGFuZ3pob3UsIDMxMDAyOSwgUC4gUi4gQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXdSwgV2VpcmVuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIEJpb2luZm9ybWF0aWNzLCBIdWFqaWFjaGkgQ2FtcHVzLCBaaGVqaWFuZyBVbml2ZXJzaXR5SGFuZ3pob3UsIDMxMDAyOSwgUC4gUi4gQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IiBHZW5vbWUtd2lkZSBJbnZlc3RpZ2F0aW9uIG9mIEludHJvbiBMZW5ndGggUG9seW1vcnBoaXNtcyBhbmQgVGhlaXIgUG90ZW50aWFsIGFzIE1vbGVjdWxhciBNYXJrZXJzIGluIFJpY2UgKCBPcnl6YSBzYXRpdmEgTC4pICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjQxNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNDI3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNpMDE5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjEyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMDUvMDEvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TcGxpY2VkIHNlZ21lbnRzIGF0IHRoZSA14oCyIHRlcm1pbnVzIG9mIGFkZW5vdmlydXMgMiBsYXRlIG1STkE7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5Nzc7ICBjaXRhdGlvbl92b2x1bWU9NzQ7ICBjaXRhdGlvbl9wYWdlcz0zMTcxLTMxNzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGFtYXppbmcgc2VxdWVuY2UgYXJyYW5nZW1lbnQgYXQgdGhlIDXigLIgZW5kcyBvZiBhZGVub3ZpcnVzIDIgbWVzc2VuZ2VyIFJOQTsgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MTk3NzsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzdXJ2ZXkgb24gaW50cm9uIGFuZCBleG9uIGxlbmd0aHM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk4ODsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTk4OTMtOTkwODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50cm9u4oCTZXhvbiBzdHJ1Y3R1cmVzIG9mIGV1a2FyeW90aWMgbW9kZWwgb3JnYW5pc21zOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0zMjE5LTMyMjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWUgc2VxdWVuY2Ugb2YgRHJvc29waGlsYSBtZWxhbm9nYXN0ZXI7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9Mjg3OyAgY2l0YXRpb25fcGFnZXM9MjE4NS0yMTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgc2VxdWVuY2Ugb2YgdGhlIGh1bWFuIGdlbm9tZTsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTE7ICBjaXRhdGlvbl9wYWdlcz0xMzA0LTEzNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxhcmdlLXNjYWxlIGlkZW50aWZpY2F0aW9uLCBtYXBwaW5nLCBhbmQgZ2Vub3R5cGluZyBvZiBzaW5nbGUtbnVjbGVvdGlkZSBwb2x5bW9ycGhpc21zIGluIHRoZSBodW1hbiBnZW5vbWU7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MjgwOyAgY2l0YXRpb25fcGFnZXM9MTA3Ny0xMDgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgYSBnZW5ldGljIGxpbmthZ2UgbWFwIGluIG1hbiB1c2luZyByZXN0cmljdGlvbiBmcmFnbWVudCBsZW5ndGggcG9seW1vcnBoaXNtczsgQW0uIEouIEh1bS4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTgwOyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9MzE0LTMzMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIHBvbHltb3JwaGlzbSBhbXBsaWZpZWQgYnkgYXJiaXRyYXJ5IHByaW1lcnMgYXJlIHVzZWZ1bCBhcyBnZW5ldGljIG1hcmtlcnM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTY1MzEtNjUzNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QUZMUDogYSBuZXcgdGVjaG5pcXVlIGZvciBETkEgZmluZ2VycHJpbnRpbmc7IE51Y2xlaWMgQWNpZCBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9NDQwNy00NDE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYXJsZXkgbWljcm9zYXRlbGxpdGVzOiBhbGxlbGUgdmFyaWF0aW9uIGFuZCBtYXBwaW5nOyBQbGFudCBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9ODM1LTg0NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tYmluZWQgbWFwcGluZyBvZiBBRkxQIGFuZCBSRkxQIG1hcmtlcnMgaW4gYmFybGV5OyBNb2wuIEdlbi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTI0OTsgIGNpdGF0aW9uX3BhZ2VzPTY1LTczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jZS1yZWxhdGVkIGFtcGxpZmllZCBwb2x5bW9ycGhpc20gKFNSQVApIGEgbmV3IG1hcmtlciBzeXN0ZW0gYmFzZWQgb24gYSBzaW1wbGUgUENSIHJlYWN0aW9uOiBpdHMgYXBwbGljYXRpb24gdG8gbWFwcGluZyBhbmQgZ2VuZSB0YWdnaW5nIGluIEJyYXNzaWNhOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTEwMzsgIGNpdGF0aW9uX3BhZ2VzPTQ1NS00NjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZhcmlhdGlvbiBpcyB0aGUgc3BpY2Ugb2YgbGlmZTsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0yMzQtMjM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SYXBpZCBzdXJ2ZXkgb2YgRE5BIHNlcXVlbmNlIHZhcmlhdGlvbiBpbiBuYXR1cmFsIHBvcHVsYXRpb25zOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0zMjMtMzMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRyb24tdGFyZ2V0ZWQgUENSOiBhIG5ldyBhcHByb2FjaCB0byBzdXJ2ZXkgbmV1dHJhbCBETkEgcG9seW1vcnBoaXNtIGluIGJpdmFsdmUgcG9wdWxhdGlvbnM7IE1hci4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MTIwOyAgY2l0YXRpb25fcGFnZXM9NDA3LTQxMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBjaGFyYWN0ZXJpc2F0aW9uIG9mIE15dGlsdXMgZ2FsbG9wcm92aW5jaWFsaXMgTG1rLiBpbiBOb3J0aCBXZXN0IEFmcmljYSB1c2luZyBudWNsZWFyIEROQSBtYXJrZXJzOyBKIEV4cC4gTWFyLiBCaW9sLiBFY29sLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yMzU7ICBjaXRhdGlvbl9wYWdlcz01NS02NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VXNlIG9mIGFuIGludHJvbiBsZW5ndGggcG9seW1vcnBoaXNtIHRvIGxvY2FsaXplIHRoZSB0cm9wb2VsYXN0aW4gZ2VuZSB0byBjaHJvbW9zb21lIDUgaW4gYSByZWdpb24gb2YgbGlua2FnZSBjb25zZXJ2YXRpb24gd2l0aCBodW1hbiBjaHJvbW9zb21lIDc7IEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MTI1LTEzMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TnVjbGVpYyBhY2lkcyBJSTogdGhlIHBvbHltZXJhc2UgY2hhaW4gcmVhY3Rpb247ICBjaXRhdGlvbl9lZGl0aW9uPTJuZCBlZG47IGNpdGF0aW9uX3B1Ymxpc2hlcj1TaW5hdWVyIEFzc29jaWF0ZXMgSW5jLiwgU3VuZGVybGFuZCwgTUE7IE1vbGVjdWxhciBTeXN0ZW1hdGljczsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3BhZ2VzPTIwNS0yNDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNjcmVlbmluZyBmb3IgaW50cm9uLWxlbmd0aCBwb2x5bW9ycGhpc21zIGluIHBlbmFlaWQgc2hyaW1wcyB1c2luZyBleG9uLXByaW1lZCBpbnRyb24tY3Jvc3NpbmcgKEVQSUMpLVBDUjsgTW9sLiBFY29sLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MjMzLTIzNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBkcmFmdCBzZXF1ZW5jZSBvZiB0aGUgcmljZSBnZW5vbWUgKCBPcnl6YSBzYXRpdmEgTC4gc3NwLiBpbmRpY2EgKTsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0yOTY7ICBjaXRhdGlvbl9wYWdlcz03OS05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBkcmFmdCBzZXF1ZW5jZSBvZiB0aGUgcmljZSBnZW5vbWVzICggT3J5emEgc2F0aXZhIEwuIHNzcC4gamFwb25pY2EgKTsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0yOTY7ICBjaXRhdGlvbl9wYWdlcz05Mi0xMDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbGxlY3Rpb24sIG1hcHBpbmcsIGFuZCBhbm5vdGF0aW9uIG9mIG92ZXIgMjjigIkwMDAgY0ROQSBjbG9uZXMgZnJvbSBqYXBvbmljYSByaWNlOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMwMTsgIGNpdGF0aW9uX3BhZ2VzPTM3Ni0zNzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWUgc2VxdWVuY2UgYW5kIHN0cnVjdHVyZSBvZiByaWNlIGNocm9tb3NvbWUgMTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTQyMDsgIGNpdGF0aW9uX3BhZ2VzPTMxMi0zMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNlIGFuZCBhbmFseXNpcyBvZiByaWNlIGNocm9tb3NvbWUgNDsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTQyMDsgIGNpdGF0aW9uX3BhZ2VzPTMxNi0zMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluLWRlcHRoIHZpZXcgb2Ygc3RydWN0dXJlLCBhY3Rpdml0eSwgYW5kIGV2b2x1dGlvbiBvZiByaWNlIGNocm9tb3NvbWUgMTA7IGNpdGF0aW9uX2F1dGhvcj1UaGUgUmljZSBDaHJvbW9zb21lIDEwIFNlcXVlbmNpbmcgQ29uc29ydGl1bTsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMDA7ICBjaXRhdGlvbl9wYWdlcz0xNTY2LTE1Njk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBtYXAtYmFzZWQgc2VxdWVuY2Ugb2YgdGhlIHJpY2UgZ2Vub21lOyBjaXRhdGlvbl9hdXRob3I9SW50ZXJuYXRpb25hbCBSaWNlIEdlbm9tZSBTZXF1ZW5jaW5nIFByb2plY3Q7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00MzY7ICBjaXRhdGlvbl9wYWdlcz03OTMtODAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBvZiBnZW5vbWUtd2lkZSBETkEgcG9seW1vcnBoaXNtIGRhdGFiYXNlIGZvciBtYXAtYmFzZWQgY2xvbmluZyBvZiByaWNlIGdlbmVzOyBQbGFudCBwaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMzU7ICBjaXRhdGlvbl9wYWdlcz0xMTk4LTEyMDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIFNOUCByZXNvdXJjZSBmb3IgcmljZSBnZW5ldGljcyBhbmQgYnJlZWRpbmcgYmFzZWQgb24gc3Vic3BlY2llcyBpbmRpY2EgYW5kIGphcG9uaWNhIGdlbm9tZSBhbGlnbm1lbnRzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTE4MTItMTgxOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2FwcGVkIEJMQVNUIGFuZCBQU0ktQkxBU1Q6IEEgbmV3IGdlbmVyYXRpb24gb2YgcHJvdGVpbiBkYXRhYmFzZSBzZWFyY2ggcHJvZ3JhbXM7IE51Y2xlaWMgQWNpZHMuIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0zMzg5LTM0MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY29tcHV0ZXIgcHJvZ3JhbSBmb3IgYWxpZ25pbmcgYSBjRE5BIHNlcXVlbmNlIHdpdGggYSBnZW5vbWljIEROQSBzZXF1ZW5jZTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTk2Ny05NzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVNQk9TUzogVGhlIEV1cm9wZWFuIE1vbGVjdWxhciBCaW9sb2d5IE9wZW4gU29mdHdhcmUgU3VpdGU7IFRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0yNzYtMjc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jZSBtYXBwaW5nIGJ5IGVsZWN0cm9uaWMgUENSOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9NTQxLTU1MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBoaWdoLWRlbnNpdHkgcmljZSBnZW5ldGljIGxpbmthZ2UgbWFwIHdpdGggMjI3NSBtYXJrZXJzIHVzaW5nIGEgc2luZ2xlIEYgMiBwb3B1bGF0aW9uOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xNDg7ICBjaXRhdGlvbl9wYWdlcz00NzktNDk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SYXBpZCBpc29sYXRpb24gb2YgaGlnaC1tb2xlY3VsYXItIHdlaWdodCBwbGFudCBETkE7IE51Y2xlaWMgQWNpZHMuIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5ODA7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTQzMjEtNDMyNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU94oCYVG91Y2hkb3du4oCZIFBDUiB0byBjaXJjdW12ZW50IHNwdXJpb3VzIHByaW1pbmcgZHVyaW5nIGdlbmUgYW1wbGlmaWNhdGlvbjsgTnVjbGVpYyBBY2lkcy4gUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5MTsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTQwMDgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHNpbXBsZSBhbmQgcmFwaWQgbWV0aG9kIHVzZWQgZm9yIHNpbHZlciBzdGFpbmluZyBhbmQgZ2VsIHByZXNlcnZhdGlvbjsgSGVyZWRpdGFzIChCZWlqaW5nKTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTMzNS0zMzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9wdGltaXppbmcgcGFyZW50YWwgc2VsZWN0aW9uIGZvciBnZW5ldGljIGxpbmthZ2UgbWFwczsgR2Vub21lOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9MTgxLTE4NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgaW50cmFzcGVjaWZpYyBETkEtc2VxdWVuY2UgdmFyaWF0aW9ucyBpbiByaWNlOyBDdXJyLiBPcGluLiBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MTM0LTEzODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VtaWR3YXJmICggc2QtMSApLCDigJxncmVlbiByZXZvbHV0aW9u4oCdIHJpY2UsIGNvbnRhaW5zIGEgZGVmZWN0aXZlIGdpYmJlcmVsbGluIDIwLW94aWRhc2UgZ2VuZTsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT05OTsgIGNpdGF0aW9uX3BhZ2VzPTkwNDMtOTA0ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNvenltZXMgYW5kIGNsYXNzaWZpY2F0aW9uIG9mIEFzaWFuIHJpY2UgdmFyaWV0aWVzOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTc0OyAgY2l0YXRpb25fcGFnZXM9MjEtMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNsYXNzaWZ5aW5nIGphcG9uaWNhIHJpY2UgY3VsdGl2YXJzIHdpdGggUkFQRCBtYXJrZXJzOyBDcm9wIFNjaS47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz04ODktODk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGRpZmZlcmVudGlhdGlvbiBvZiB3aWxkIHJlbGF0aXZlcyBvZiByaWNlIGFzIGFzc2Vzc2VkIGJ5IFJGTFAgYW5hbHlzaXM7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTA2OyAgY2l0YXRpb25fcGFnZXM9MTAxLTEwNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIENsdXN0YWxYIHdpbmRvd3MgaW50ZXJmYWNlOiBmbGV4aWJsZSBzdHJhdGVnaWVzIGZvciBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnQgYWlkZWQgYnkgcXVhbGl0eSBhbmFseXNpcyB0b29sczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9NDg3Ni00ODgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJOaWNob2xhcyBLLiBCLiBOaWNob2xhcyBILiBCLiBHZW5lRG9jOiBhIHRvb2wgZm9yIGVkaXRpbmcgYW5kIGFubm90YXRpbmcgbXVsdGlwbGUgc2VxdWVuY2UgYWxpZ25tZW50cyAxOTk3IERpc3RyaWJ1dGVkIGJ5IHRoZSBhdXRob3JzICggKSAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTIvNi80MTcvNjcxODU1MC9kc2kwMTkucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gICBJbnRyb24gbGVuZ3RoIHBvbHltb3JwaGlzbXMgKElMUHMpIGhhdmUgYmVlbiB1c2VkIGFzIGdlbmV0aWMgbWFya2VycyBpbiBzb21lIHN0dWRpZXMuIEhvd2V2ZXIsIGEgc3lzdGVtYXRpYyBpbnZlc3RpZ2F0aW9uIGFuZCBsYXJnZS1zY2FsZSBleHBsb2l0YXRpIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zNDM4NTQmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkVulOzNSTTo+lMLa3aSA7lJOcOtQ=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f12%2f6%2f417%2f343854" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f12%2f6%2f417%2f343854" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13225" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/12/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/12/6/0/m_dnaresearch12_6.cover.gif?Expires=2147483647&amp;Signature=vmBj-dEpEdSISGGZZVQQ800UdbaQ61VgN5SO6nMO30mtIcmmHegLeBqq~JmeB3Cl71F-HZSxEtC05awn7JLvbmS~R8ehKSKlG67TzUmDNfl2uJ9iwd8X3XVTg535TrTFouPO8FrSAc3H4exNAvHuAshR8~cghIB3WqxBjwF3q4V-gWs6yVOhRl523VyJV6-aueYhJA8pI1jFRHUmPhSY-RKzDUrwgfwrVrH153kk1IiVH3chlu~YmBF~XczCNeM~pNFI-0DwLreboeiSvqsHnGPMQGgfWos-lNOnEyjyXTWeiJBGXgUTUflQE1fArs2l0eo950LpKbLah37z6bPoSQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/12/6">
                        <div class="volume trailing-comma">Volume 12</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
2005        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="35868886">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#35868886">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="35868888">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#35868888">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="35868894">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#35868894">2. Materials and Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="35868907">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#35868907">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="35868947">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#35868947">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/12/6/403/344476">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/12/6/429/344258">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main"> Genome-wide Investigation of Intron Length Polymorphisms and Their Potential as Molecular Markers in Rice ( <em>Oryza sativa</em> L.) 
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xusheng Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Institute of Bioinformatics, Huajiachi Campus, Zhejiang University</div><div class="addr-line">Hangzhou, 310029, P. R. China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xusheng+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wang X&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Xusheng Wang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xiangqian Zhao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Institute of Bioinformatics, Huajiachi Campus, Zhejiang University</div><div class="addr-line">Hangzhou, 310029, P. R. China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xiangqian+Zhao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhao X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhao X&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Xiangqian Zhao</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jun Zhu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Institute of Bioinformatics, Huajiachi Campus, Zhejiang University</div><div class="addr-line">Hangzhou, 310029, P. R. China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jun+Zhu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhu J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhu J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jun Zhu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Weiren Wu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Institute of Bioinformatics, Huajiachi Campus, Zhejiang University</div><div class="addr-line">Hangzhou, 310029, P. R. China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <sup>*</sup> To whom correspondence should be addressed. Tel/Fax. +86-571-86971910, E-mail: <a href="mailto:wuwr@zju.edu.cn" target="_blank">wuwr@zju.edu.cn</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Weiren+Wu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wu W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wu W&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Weiren Wu</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="fn1"><div class="fn"><div class="footnote-content"><sup>†</sup> These authors contributed equally to this work. </div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Communicated by Satoshi Tabata</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 12, Issue 6, 1 January 2005, Pages 417–427, <a href='https://doi.org/10.1093/dnares/dsi019'>https://doi.org/10.1093/dnares/dsi019</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">01 January 2005</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">06 October 2005</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="343854" href="/dnaresearch/article-pdf/12/6/417/6718550/dsi019.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Xusheng Wang, Xiangqian Zhao, Jun Zhu, Weiren Wu;   Genome-wide Investigation of Intron Length Polymorphisms and Their Potential as Molecular Markers in Rice ( <em>Oryza sativa</em> L.) , <em>DNA Research</em>, Volume 12, Issue 6, 1 January 2005, Pages 417–427, <a href='https://doi.org/10.1093/dnares/dsi019'>https://doi.org/10.1093/dnares/dsi019</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/343854?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/343854?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/343854?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/343854?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/343854?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/343854?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/343854?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=%20Genome-wide%20Investigation%20of%20Intron%20Length%20Polymorphisms%20and%20Their%20Potential%20as%20Molecular%20Markers%20in%20Rice%20(%20%26lt%3Bem%26gt%3BOryza%20sativa%26lt%3B%2Fem%26gt%3B%20L.)%20&amp;publicationDate=2005-01-01&amp;volumeNum=12&amp;issueNum=6&amp;author=Wang%2C%20Xusheng%3B%20Zhao%2C%20Xiangqian&amp;startPage=417&amp;endPage=427&amp;contentId=10.1093%2Fdnares%2Fdsi019&amp;oa=&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f12%2f6%2f417%2f343854" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=35868886 id="35868886" class="abstract-title" >Abstract</h2>
<section class="abstract"><p> Intron length polymorphisms (ILPs) have been used as genetic markers in some studies. However, a systematic investigation and large-scale exploitation of ILP markers has not been reported. In this study, we performed a genome-wide search of ILPs between two subspecies ( <em>indica</em> and <em>japonica</em> ) in rice using the draft genomic sequences of cultivars 93-11 ( <em>indica</em> ) and Nipponbare ( <em>japonica</em> ) and 32 127 full-length cDNA sequences of Nipponbare obtained from public databases. We identified 13 308 putative ILPs. Based on these putative ILPs, we developed 5811 candidate ILP markers via electronic-PCR with primers designed in flanking exons. We further conducted experiment to verify the candidate ILP markers. Out of 215 candidate ILP markers tested on 93-11, Nipponbare and their hybrid, we successfully exploited 173 codominant ILP markers. Further analyses on 10 rice accessions showed that these ILP markers were widely applicable and most (71.1%) exhibited subspecies specificity. This feature suggests that ILPs would be useful for the studies of genome evolution and inter-subspecies heterosis and for cross-subspecies marker-assisted selection in rice. In addition, by testing 51 pairs of the ILP primers on five <em>Gramineae</em> plants and three dicot plants, we found another desirable characteristic of rice ILP markers that they have high transferability to other plants. </p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot; rice ( Oryza sativa L.) &quot;"> rice ( <em>Oryza sativa</em> L.) </a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;intron length polymorphism (ILP)&quot;">intron length polymorphism (ILP)</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;molecular marker&quot;">molecular marker</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="genome">genome</a></div>                            <h2 scrollto-destination=35868888 id="35868888" class="section-title" >1. Introduction</h2>
<p> Introns are non-coding sequences in a gene that are transcribed but spliced out of the precursor mRNA. <a reveal-id="b1" data-open="b1" class="link link-ref link-reveal xref-bibr">1</a> , <a reveal-id="b2" data-open="b2" class="link link-ref link-reveal xref-bibr">2</a> Introns are widespread and abundant in eukaryotic genomes. <a reveal-id="b3" data-open="b3" class="link link-ref link-reveal xref-bibr">3</a> , <a reveal-id="b4" data-open="b4" class="link link-ref link-reveal xref-bibr">4</a> For example, introns constitute ∼11 and 24% of the fruit fly <a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a> and human <a reveal-id="b6" data-open="b6" class="link link-ref link-reveal xref-bibr">6</a> genomes, respectively. Generally speaking, introns have little functional significance, although some introns may influence the level of gene expression. <a reveal-id="b7" data-open="b7" class="link link-ref link-reveal xref-bibr">7</a> Therefore, introns are more variable than coding sequences. </p><p> Variations or polymorphisms in DNA sequences can be exploited as genetic markers (usually called molecular markers), which are very useful tools for genetic research (e.g. construction of genetic maps, mapping of genes or quantitative trait loci) and breeding (e.g. marker-assisted selection). Botstein et al. <a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a> first utilized restriction fragment length polymorphisms (RFLPs) as genetic markers to construct human genetic map. Since then, many new molecular markers have been developed, such as random amplified polymorphic DNA (RAPD), <a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a> amplified fragment length polymorphism (AFLP), <a reveal-id="b10" data-open="b10" class="link link-ref link-reveal xref-bibr">10</a> microsatellite or simple sequence repeat (SSR), <a reveal-id="b11" data-open="b11" class="link link-ref link-reveal xref-bibr">11</a> , <a reveal-id="b12" data-open="b12" class="link link-ref link-reveal xref-bibr">12</a> sequence-related amplified polymorphism (SRAP) <a reveal-id="b13" data-open="b13" class="link link-ref link-reveal xref-bibr">13</a> and single-nucleotide polymorphism (SNP). <a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a></p><p> Intron polymorphisms can also be exploited as genetic markers. They have been successfully utilized in population genetics surveys <a reveal-id="b15  b17" data-open="b15  b17" class="link link-ref link-reveal xref-bibr">15–17</a> and gene mapping. <a reveal-id="b18" data-open="b18" class="link link-ref link-reveal xref-bibr">18</a> There could be various polymorphisms in introns, but intron length polymorphism (ILP) is the most easily recognizable type. It can be conveniently detected by the PCR. To amplify introns by PCR, primers can be designed in flanking exons. This approach is called exon-primed intron-crossing PCR (EPIC-PCR). <a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a> The advantage of EPIC-PCR is that exon sequences are relatively more conservative and therefore the primers designed in exons may have more extensive applications than those designed in non-coding sequences. Using this approach, Bierne et al. <a reveal-id="b20" data-open="b20" class="link link-ref link-reveal xref-bibr">20</a> developed several ILP markers in penaeid shrimps. To date, however, studies of exploiting intron polymorphism markers have been restricted to a few genes. No efforts on genome-wide exploitation of intron polymorphism markers have been reported. </p><p> In rice ( <em>Oryza sativa</em> ), draft genome sequences of two cultivars, 93-11 <a reveal-id="b21" data-open="b21" class="link link-ref link-reveal xref-bibr">21</a> and Nipponbare, <a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a> representing <em>indica</em> and <em>japonica</em> subspecies, respectively, and a set of over 28 000 full-length cDNA sequences from Nipponbare <a reveal-id="b23" data-open="b23" class="link link-ref link-reveal xref-bibr">23</a> have been released. Moreover, complete sequences of chromosomes 1, 4 and 10 and, more recently, the whole genome of Nipponbare have been published. <a reveal-id="b24  b27" data-open="b24  b27" class="link link-ref link-reveal xref-bibr">24–27</a> In addition, a tentative assembly of all chromosomes of rice has been released (The Institute of Genomic Research, TIGR; <a class="link link-uri" href="http://www.tigr.org" target="_blank"></a> ). These data provide us with an opportunity to systematically search for DNA polymorphisms between the two subspecies and to exploit DNA markers on a large scale in rice. Recently, Shen et al. <a reveal-id="b28" data-open="b28" class="link link-ref link-reveal xref-bibr">28</a> and Feltus et al. <a reveal-id="b29" data-open="b29" class="link link-ref link-reveal xref-bibr">29</a> independently conducted genome-wide investigations of SNPs and InDels or single-base InDels using the same set of released genome drafts of 93-11 and Nipponbare. However, it is astonishing that while Shen et al. <a reveal-id="b28" data-open="b28" class="link link-ref link-reveal xref-bibr">28</a> identified 1 703 176 SNPs and 479 406 InDels, among which there were 277 858 single-base InDels according to our count from their online database, Feltus et al. <a reveal-id="b29" data-open="b29" class="link link-ref link-reveal xref-bibr">29</a> only identified 384 341 SNPs and 24 557 single-base InDels—the numbers of SNPs and single-base InDels identified by Shen et al. <a reveal-id="b28" data-open="b28" class="link link-ref link-reveal xref-bibr">28</a> are 4.4 and 11.3 times greater than those identified by Feltus et al. <a reveal-id="b29" data-open="b29" class="link link-ref link-reveal xref-bibr">29</a> respectively. The great inconsistency between the two studies suggests that there might be many mistakes in the reported SNPs and InDels. </p><p>ILPs are caused by InDels, but many of them cannot be simply considered equivalent to the generally defined InDels because an ILP may contain several (instead of only one) InDels. In addition, even if we take all ILPs as InDels, they are at least a special subset of InDels exhibiting polymorphisms in the non-coding regions of genes. This may make ILPs possess special characteristics and usefulness. In the work described here, we performed a genome-wide search for ILPs and a large-scale exploitation of candidate ILP markers via electronic EPIC-PCR based on the released genomic and cDNA sequence data in rice. We also developed a set of ILP markers selected from the candidates and investigated their characteristics by experiment. Moreover, we established a web-accessible database for rice ILP markers.</p>                            <h2 scrollto-destination=35868894 id="35868894" class="section-title" >2. Materials and Methods</h2>
                            <h3 scrollto-destination=35868895 id="35868895" class="section-title" >2.1. Data sources of rice genomic and cDNA sequences</h3>
<p> We downloaded genomic sequence data of two rice cultivars, Nipponbare (ssp. <em>japonica</em> ) and 93-11 (ssp. <em>indica</em> ), released by the International Rice Genome Sequencing Project (IRGSP) and Beijing Genomics Institute (BGI), from web sites <a class="link link-uri" href="http://www.tigr.org/" target="_blank"></a> and <a class="link link-uri" href="http://rise.genomics.org.cn/rice/link/download.jsp" target="_blank"></a> , respectively. In addition, we downloaded 32 127 full-length cDNA sequences of Nipponbare from web site <a class="link link-uri" href="http://cdna01.dna.affrc.go.jp/cDNA/" target="_blank"></a> . </p>                            <h3 scrollto-destination=35868897 id="35868897" class="section-title" >2.2. Search of ILPs</h3>
<p> We developed a pipeline using Perl script to search ILPs between Nipponbare and 93-11 and exploit candidate ILP markers. The initial step was to identify the most likely positions of the available cDNA sequences in rice genome by aligning the cDNA sequences of Nipponbare with the genomic sequences of Nipponbare and 93-11 using BLASTN. <a reveal-id="b30" data-open="b30" class="link link-ref link-reveal xref-bibr">30</a> We used a high <em>E</em> -value (= 10 <sup>−20</sup> ) for the BLASTN to remove paralogues. Then we used the program SIM4 <a reveal-id="b31" data-open="b31" class="link link-ref link-reveal xref-bibr">31</a> to align each cDNA with its corresponding BAC clone from Nipponbare and Scaffold from 93-11 to examine the gene structure (number of introns and positions of splice sites) and putative ILPs between the two cultivars in the gene. Although there could be several possible types of ILPs, we restricted our ILP search to those genes that showed the same structure (i.e. same number of introns with same positions of splice sites) in the two cultivars because ILPs in those genes could potentially be exploited as PCR-based codominant markers. </p>                            <h3 scrollto-destination=35868899 id="35868899" class="section-title" >2.3. Exploitation of candidate ILP markers by electronic EPIC-PCR</h3>
<p> To exploit ILP markers from the putative ILPs identified by BLASTN and SIM4, we designed PCR primers based on the cDNA sequences (of Nipponbare) corresponding to the flanking exons using ePrime3 ( <a class="link link-uri" href="http://www.hgmp.mrc.ac.uk/" target="_blank"></a> ). <a reveal-id="b32" data-open="b32" class="link link-ref link-reveal xref-bibr">32</a> For convenience, we used a 200 bp cDNA sequence with 100 bp on each side of the target intron for the primer design for each ILP. A simulation study conducted beforehand had shown that the cDNA length could guarantee 93% success in primer design. Then we tested the designed primers by electronic PCR (e-PCR) <a reveal-id="b33" data-open="b33" class="link link-ref link-reveal xref-bibr">33</a> on the genomic sequences of Nipponbare and 93-11, respectively. To increase the quality and usability of the <em>in silico</em> exploited ILP markers, we required exact matches between primers and templates and set a 1100 bp margin on the product size for the e-PCR. We took a putative ILP locus as a candidate ILP marker when it was successfully and uniquely detected by the e-PCR and named it with the abbreviation RI (for Rice ILP) followed by a unique number (e.g. RI03281). </p>                            <h3 scrollto-destination=35868901 id="35868901" class="section-title" >2.4. Verification and evaluation of ILP markers by experiment</h3>
<p> We selected 215 candidate ILP markers for the experimental evaluation. According to the RGP's rice genetic map, <a reveal-id="b34" data-open="b34" class="link link-ref link-reveal xref-bibr">34</a> the selected candidate ILP markers were approximately evenly distributed in rice genome with an average of ∼8 cM between adjacent ILP loci. While we directly adopted the PCR primers designed in flanking exons for most of the selected candidate ILP markers, we also redesigned primers in introns for some with smaller intron length difference (ILD) but larger intron size in order that polymorphisms could be detected via electrophoresis. All primers were synthesized by either Shanghai Sangon Biological Engineering &amp; Technology Company or Shanghai BioAsia Biotechnology Company. </p><p> We tested the synthesized primers at first using Nipponbare, 93-11 and their F <sub>1</sub> . We extracted genomic DNAs from young leaves using CTAB method <a reveal-id="b35" data-open="b35" class="link link-ref link-reveal xref-bibr">35</a> with modification. We conducted PCR in a 15 µl reaction mixture containing 50 ng template DNA, 0.5 µM of each primer, 200 µM of each dNTP, 1.5 mM MgCl <sub>2</sub> , 0.1% Triton X-100 and 1 U <em>Taq</em> polymerase and 1.5 µl of 10× PCR buffer. We first tested all primer pairs with a touchdown PCR (Td-PCR) <a reveal-id="b36" data-open="b36" class="link link-ref link-reveal xref-bibr">36</a> program: 5 min at 94°C; 10 cycles of 30 s at 94°C, 30 s at 59°C minus 0.3°C/cycle, 1 min at 72°C; 20 cycles of 30 s at 94°C, 30 s at 56°C, 1 min at 72°C; and 5 min at 72°C for final extension. For primer pairs that did not generate good amplification results, we adjusted the initial annealing temperature (59°C) to 60 or 57°C. The purpose of using Td-PCR was to increase the specificity of amplification, but some primer pairs only required a routine PCR program: 5 min at 94°C; 35 cycles of 30 s at 94°C, 30 s at 54°C, and 1 min at 72°C; and 5 min at 72°C for final extension. For most primers, we used 6% non-denaturing PAGE (250 V, 2 h) for separating PCR products and silver staining for visualizing DNA bands following Xu et al. <a reveal-id="b37" data-open="b37" class="link link-ref link-reveal xref-bibr">37</a> with modification. We also used 2% agarose gel for some primers. </p><p> For primers generating correct PCR products as expected, we further tested them using 10 rice varieties including Nipponbare and 93-11 ( <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a> ), which were kindly provided by the China National Rice Research Institute (CNRRI). Based on the PCR data, we evaluated the allelic diversity of each ILP marker using the polymorphism information content (PIC) value defined as <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mrow><mtext>PIC</mtext></mrow><mi>i</mi></msub><mo>=</mo><mn>1</mn><mo>−</mo><msubsup><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mi>n</mi></msubsup><mrow><msubsup><mi>p</mi><mrow><mi>i</mi><mi>j</mi></mrow><mn>2</mn></msubsup></mrow></mrow></math></span> , where <em> p <sub>ij</sub></em> is the frequency of the <em>j</em> th pattern for the <em>i</em> th marker. <a reveal-id="b38" data-open="b38" class="link link-ref link-reveal xref-bibr">38</a></p>                        <a id="35868905" scrollto-destination="35868905"></a>
<div content-id="tbl1" class="table-modal"><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Rice accessions used for testing ILP markers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Accession </th><th>Type </th><th>Origin </th></tr></thead><tbody><tr><td>93-11 </td><td><em>Indica</em> </td><td>China </td></tr><tr><td>Guangluai-4 </td><td><em>indica</em> </td><td>China </td></tr><tr><td>Xieqingzao </td><td><em>indica</em> </td><td>China </td></tr><tr><td>IR64 </td><td><em>Indica</em><a reveal-id="tf1-1" data-open="tf1-1" class="link link-ref link-reveal xref-fn">a</a> </td><td>Philippines </td></tr><tr><td>Kyeema </td><td><em>Indica</em><a reveal-id="tf1-1" data-open="tf1-1" class="link link-ref link-reveal xref-fn">a</a> </td><td>Australia </td></tr><tr><td>Nipponbare </td><td><em>japonica</em> </td><td>Japan </td></tr><tr><td>Xiushui-11 </td><td><em>japonica</em> </td><td>China </td></tr><tr><td>Koshihikari </td><td><em>japonica</em> </td><td>Japan </td></tr><tr><td>Katy </td><td><em>Japonica</em><a reveal-id="tf1-1" data-open="tf1-1" class="link link-ref link-reveal xref-fn">a</a> </td><td>USA </td></tr><tr><td>Merim </td><td><em>javanica</em> </td><td>Indonesia </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th>Accession </th><th>Type </th><th>Origin </th></tr></thead><tbody><tr><td>93-11 </td><td><em>Indica</em> </td><td>China </td></tr><tr><td>Guangluai-4 </td><td><em>indica</em> </td><td>China </td></tr><tr><td>Xieqingzao </td><td><em>indica</em> </td><td>China </td></tr><tr><td>IR64 </td><td><em>Indica</em><a reveal-id="tf1-1" data-open="tf1-1" class="link link-ref link-reveal xref-fn">a</a> </td><td>Philippines </td></tr><tr><td>Kyeema </td><td><em>Indica</em><a reveal-id="tf1-1" data-open="tf1-1" class="link link-ref link-reveal xref-fn">a</a> </td><td>Australia </td></tr><tr><td>Nipponbare </td><td><em>japonica</em> </td><td>Japan </td></tr><tr><td>Xiushui-11 </td><td><em>japonica</em> </td><td>China </td></tr><tr><td>Koshihikari </td><td><em>japonica</em> </td><td>Japan </td></tr><tr><td>Katy </td><td><em>Japonica</em><a reveal-id="tf1-1" data-open="tf1-1" class="link link-ref link-reveal xref-fn">a</a> </td><td>USA </td></tr><tr><td>Merim </td><td><em>javanica</em> </td><td>Indonesia </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tf1-1"><div class="fn"><p><sup>a</sup> Might not be typical <em>indica</em> or <em>japonica</em> according to its pedigree. </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/35868905" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Rice accessions used for testing ILP markers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Accession </th><th>Type </th><th>Origin </th></tr></thead><tbody><tr><td>93-11 </td><td><em>Indica</em> </td><td>China </td></tr><tr><td>Guangluai-4 </td><td><em>indica</em> </td><td>China </td></tr><tr><td>Xieqingzao </td><td><em>indica</em> </td><td>China </td></tr><tr><td>IR64 </td><td><em>Indica</em><a reveal-id="tf1-1" data-open="tf1-1" class="link link-ref link-reveal xref-fn">a</a> </td><td>Philippines </td></tr><tr><td>Kyeema </td><td><em>Indica</em><a reveal-id="tf1-1" data-open="tf1-1" class="link link-ref link-reveal xref-fn">a</a> </td><td>Australia </td></tr><tr><td>Nipponbare </td><td><em>japonica</em> </td><td>Japan </td></tr><tr><td>Xiushui-11 </td><td><em>japonica</em> </td><td>China </td></tr><tr><td>Koshihikari </td><td><em>japonica</em> </td><td>Japan </td></tr><tr><td>Katy </td><td><em>Japonica</em><a reveal-id="tf1-1" data-open="tf1-1" class="link link-ref link-reveal xref-fn">a</a> </td><td>USA </td></tr><tr><td>Merim </td><td><em>javanica</em> </td><td>Indonesia </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th>Accession </th><th>Type </th><th>Origin </th></tr></thead><tbody><tr><td>93-11 </td><td><em>Indica</em> </td><td>China </td></tr><tr><td>Guangluai-4 </td><td><em>indica</em> </td><td>China </td></tr><tr><td>Xieqingzao </td><td><em>indica</em> </td><td>China </td></tr><tr><td>IR64 </td><td><em>Indica</em><a reveal-id="tf1-1" data-open="tf1-1" class="link link-ref link-reveal xref-fn">a</a> </td><td>Philippines </td></tr><tr><td>Kyeema </td><td><em>Indica</em><a reveal-id="tf1-1" data-open="tf1-1" class="link link-ref link-reveal xref-fn">a</a> </td><td>Australia </td></tr><tr><td>Nipponbare </td><td><em>japonica</em> </td><td>Japan </td></tr><tr><td>Xiushui-11 </td><td><em>japonica</em> </td><td>China </td></tr><tr><td>Koshihikari </td><td><em>japonica</em> </td><td>Japan </td></tr><tr><td>Katy </td><td><em>Japonica</em><a reveal-id="tf1-1" data-open="tf1-1" class="link link-ref link-reveal xref-fn">a</a> </td><td>USA </td></tr><tr><td>Merim </td><td><em>javanica</em> </td><td>Indonesia </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tf1-1"><div class="fn"><p><sup>a</sup> Might not be typical <em>indica</em> or <em>japonica</em> according to its pedigree. </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/35868905" target="_blank">View Large</a></div></div><p> We also employed some of the ILP primers to perform PCR in other plants including five <em>Gramineae</em> plants (wheat, barley, maize, sorghum and bamboo) and three dicot plants (rape, cotton and tobacco), using either a Td-PCR program (with an initial annealing temperature of 55°C) or a routine PCR program (with an annealing temperature of 52°C). </p>                            <h2 scrollto-destination=35868907 id="35868907" class="section-title" >3. Results and discussion</h2>
                            <h3 scrollto-destination=35868908 id="35868908" class="section-title" >3.1. Number, distribution and density of ILPs in rice</h3>
<p> By aligning 32 127 full-length cDNA sequences from Nipponbare with the genomic sequences of Nipponbare and 93-11 using BLASTN and SIM4, we found 120 489 and 108 312 introns in the two cultivars, respectively, and identified 13 308 putative ILPs between the two cultivars. All the cDNAs were localized to the BAC clones of Nipponbare as expected, but 1279 (3.98%) cDNAs did not align to the scaffolds of 93-11 with an <em>E</em> -value below the BLAST criterion of 10 <sup>−20</sup> . That could be the reason that fewer introns were found in 93-11. By referring to the TIGR psuedomolecule assembly of rice, we have plotted the density distribution curve of ILPs in rice genome ( <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a> ). Due to discrepancies between GenBank and TIGR in the assembly of some BAC clones, 32 ILPs could not be assigned to chromosomes and 302 ILPs could not be located, although their chromosomes were known. Therefore, we did not count these ILPs when plotting the density distribution curves, but still took those ILPs into account when calculating the total number and overall density of ILPs on each chromosome ( <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a> ). Most (&gt;100) of the non-located ILPs were in chromosome 10. </p>                        <a id="35868910" scrollto-destination="35868910"></a>
<div id="fig1" data-id="fig1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi019/3/m_dsi019f1.jpeg?Expires=2147483647&Signature=pTWix~FH8GMs1yA9d9fuZ~VO07SmRB7JGybMuMm-YdFcyxS1rVJomD5mNtAiYBmhIOE0pwmg2CmEdDscbKj~k8BzKkF6xtDf9iPYtgQbK6E5GKo3sEuvNfgf5mm9kwjMqBRaI-mZ~YACKkPdCY9gtCfT4MQAOlDuJLqFBBgojgbDkULEtUSqZ-OuNI-tj7AqfD6dKaYF9A9LOl5JjGmoo5Zibwen4agJ22pYBwseSENwwsXss5FE58k0zF-S3vVPsodNk~gT3mOAynHMWX-3ZLEGe8-U4X2gLh318ZO~bPwe~4r2jj~YA8SRNNybAtmrXRThzjRm3OIGK-YqQ3Zrfg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Density distribution of ILPs across rice genome. A gray rectangle on each x -axis indicates the position of centromere. The total number and overall density (number per Mb) of ILPs on each chromosome are also presented. " path-from-xml="dsi019f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35868910/dsi019f1.jpeg" path-from-xml="dsi019f1.jpeg" target="_blank">View large</a><a section="35868910" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi019/3/dsi019f1.jpeg?Expires=2147483647&Signature=GJFjR~2ZHF5ABnN4lpecX8xuorXfJzX6Q4pGtHB8o7b371pg~1VkHnIQdUCOaR6aybKhyHOm06OwSRe1qIiDbk4QBdWC3IpNgEuzuApjBQnBEn9kRa~RsLVp7UdA-BNZ--5tBM03Ox8MN4Vb-4USJHa~h0I1D6u-k8S4FIBUpJK6NIlffhSZfrU3QEpWSdrejKauBV7bTDiiwFWtqbef9VwW1wEjDWdPe56v0vUL28B3l9arhVT9nF9CmzI6~0RQAdKreA6Ev4CgEpc3vASDwrY-4PqK-4oCK7jF24ZdwIMOsq2-31tDM~aYVGZX~G6Fxm7o2~9icsxXP-b5w2sr6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35868910&ar=343854&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi019f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Density distribution of ILPs across rice genome. A gray rectangle on each <em>x</em> -axis indicates the position of centromere. The total number and overall density (number per Mb) of ILPs on each chromosome are also presented. </p></div></div><div content-id="fig1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi019/3/m_dsi019f1.jpeg?Expires=2147483647&Signature=pTWix~FH8GMs1yA9d9fuZ~VO07SmRB7JGybMuMm-YdFcyxS1rVJomD5mNtAiYBmhIOE0pwmg2CmEdDscbKj~k8BzKkF6xtDf9iPYtgQbK6E5GKo3sEuvNfgf5mm9kwjMqBRaI-mZ~YACKkPdCY9gtCfT4MQAOlDuJLqFBBgojgbDkULEtUSqZ-OuNI-tj7AqfD6dKaYF9A9LOl5JjGmoo5Zibwen4agJ22pYBwseSENwwsXss5FE58k0zF-S3vVPsodNk~gT3mOAynHMWX-3ZLEGe8-U4X2gLh318ZO~bPwe~4r2jj~YA8SRNNybAtmrXRThzjRm3OIGK-YqQ3Zrfg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Density distribution of ILPs across rice genome. A gray rectangle on each x -axis indicates the position of centromere. The total number and overall density (number per Mb) of ILPs on each chromosome are also presented. " path-from-xml="dsi019f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35868910/dsi019f1.jpeg" path-from-xml="dsi019f1.jpeg" target="_blank">View large</a><a section="35868910" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi019/3/dsi019f1.jpeg?Expires=2147483647&Signature=GJFjR~2ZHF5ABnN4lpecX8xuorXfJzX6Q4pGtHB8o7b371pg~1VkHnIQdUCOaR6aybKhyHOm06OwSRe1qIiDbk4QBdWC3IpNgEuzuApjBQnBEn9kRa~RsLVp7UdA-BNZ--5tBM03Ox8MN4Vb-4USJHa~h0I1D6u-k8S4FIBUpJK6NIlffhSZfrU3QEpWSdrejKauBV7bTDiiwFWtqbef9VwW1wEjDWdPe56v0vUL28B3l9arhVT9nF9CmzI6~0RQAdKreA6Ev4CgEpc3vASDwrY-4PqK-4oCK7jF24ZdwIMOsq2-31tDM~aYVGZX~G6Fxm7o2~9icsxXP-b5w2sr6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35868910&ar=343854&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi019f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Density distribution of ILPs across rice genome. A gray rectangle on each <em>x</em> -axis indicates the position of centromere. The total number and overall density (number per Mb) of ILPs on each chromosome are also presented. </p></div></div><p> It is obvious from <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a> that the ILP density fluctuates dramatically along the genome and varies among chromosomes, ranging from 23.04 per Mb (chromosome 12) to 45.31 per Mb (chromosome 2) with an average of 32.52 per Mb; or from 6.25 per cM (chromosome 9) to 11.45 per cM (chromosome 2) with an average of 8.48 per cM. ILPs are clearly not randomly distributed in rice genome. In addition, the number of ILPs on each chromosome also varies greatly, ranging from 584 (chromosome 9) to 1848 (chromosome 3) with an average of 1106 ( <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a> ). The ILPs on chromosomes 1, 2 and 3 together constitute ∼40% of the total number. </p><p> Rice genome is estimated to contain 46 022–55 615 genes. <a reveal-id="b21" data-open="b21" class="link link-ref link-reveal xref-bibr">21</a> In this study, we have found 13 308 ILPs between Nipponbare and 93-11 based on 32 127 full-length cDNA sequences, suggesting that there are 0.414 ILPs per cDNA on average. If we approximately consider a cDNA as a gene, then we can deduce that the total number of ILPs between the two cultivars would be 19 064 to 23 037 according to the estimate obtained in this study. It should be emphasized that we restricted the ILP search to those genes having the same structure (number and positions of introns) in both Nipponbare and 93-11. Therefore, there were some genes not taken into account. In fact, we have mentioned above that 1279 (3.98%) cDNAs did not hit the scaffolds of 93-11 in the BLAST with an <em>E</em> -value below 10 <sup>−20</sup> . These genes might either be specific to <em>japonica</em> or have large variation between <em>indica</em> and <em>japonica</em> during evolution. <a reveal-id="b39" data-open="b39" class="link link-ref link-reveal xref-bibr">39</a> For the former case, all introns in the genes could be taken as ILPs with null alleles in <em>indica</em> and therefore could be potentially exploited as dominant ILP markers. For the latter case, ILPs could also exist. We thus see that ILPs are very rich in the rice genome and should be a huge resource of molecular markers. </p>                            <h3 scrollto-destination=35868913 id="35868913" class="section-title" >3.2. Candidate ILP markers</h3>
<p> Using primers designed in flanking exons, we successfully obtained e-PCR products from 10 572 (79.4%) and 7742 (58.2%) putative ILP loci in Nipponbare and 93-11, respectively. Although we designed the primers based on the cDNAs from Nipponbare, we failed to acquire e-PCR products from ∼1/5 putative ILP loci in Nipponbare, probably due to the several constraint conditions set for the primer design and e-PCR (see Materials and Methods). There were a higher proportion of putative ILPs in 93-11 not detected by e-PCR because mismatches might occur between some primers and the genomic sequence of 93-11. Although perfect match between primer and template was required in the e-PCR, there still were 1009 primer pairs detecting multiple BAC clones located on different chromosomes and appearing to have multiple copies in Nipponbare. Similarly, 880 primer pairs detected multiple scaffolds and showed multiple copies in 93-11. A typical example was the primer pair designed in cDNA AK064639, which detected a total of 106 occurrences on <em>japonica</em> genome, indicating that the primer pair might be designed in the conserved sequences of a big gene family. As multiple-copy is not desirable for molecular markers, we discarded these primer pairs. It is noted that the 10 572 ILP loci detected by e-PCR in Nipponbare were located only on 2405 (∼61%) BAC clones, leaving 1526 clones without ILP hits. The result also reflects the nonrandom distribution of ILPs in rice genome. </p><p> By combining the e-PCR results in Nipponbare and 93-11, we obtained 5811 candidate single-copy ILP markers. The number of candidate ILP markers on each chromosome ranged from 130 (chromosome 9) to 990 (chromosome 2) with an average of 484; and the density on each chromosome ranged from 4.91 per Mb (chromosome 4) to 24.81 per Mb (chromosome 2) with an average of 13.06 per Mb, or from 1.39 per cM (chromosome 9) to 6.27 per cM (chromosome 2) with an average of 3.42 per cM ( <a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2</a> ). Based on the TIGR psuedomolecule assembly of rice, we have constructed a physical map of the 5811 candidate ILP markers and a comparative map of 2275 RFLP and 2740 SSR markers ( <a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2</a> ). The map shows that the candidate ILP markers are not evenly distributed. This seems to be consistent with the distribution of ILPs ( <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a> ). However, it is surprising that some genomic regions (particularly in the long arms of chromosomes 1, 4, 9 and 12) are nearly devoid of candidate ILP markers although the putative ILPs in these regions are not rare ( <a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2</a> ). By examining the distribution of e-PCR hits in the whole genome, we found that these regions appeared to have high frequencies of No-EPCR-Hit-in-93-11 (NEH9) ( <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3</a> ). This could explain why there were so few candidate ILP markers obtained in these regions. The major reason of NEH9 for a putative ILP locus could be that mismatches occurred between the primers designed based on the cDNA of Nipponbare and the genomic sequence of 93-11. The high frequencies of mismatches in these regions imply that these regions might have a higher level of genetic variation between the two subspecies. To develop ILP markers in these regions, we need to identify conserved sequences of each gene for designing PCR primers. </p>                        <a id="35868916" scrollto-destination="35868916"></a>
<div id="fig2" data-id="fig2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi019/3/m_dsi019f2.jpeg?Expires=2147483647&Signature=lu1fO7GQIkQoyRxlUJ8lRfV0oftUidm9RUw0u7p-5iFjoB7LzzSWuokSU0q0ptp74SVcZVawfRVdaIIidzUdFdEKZiZ3ET6FDnUD6ElsRbLaHxCz1w8phMeucUglyyLqA7IUHqdzLjBldIZijZ~8-JFVscQ40zr9i-gNnV9OxMGzAPSVSuGzxXXHE~pKVWjfhN11DM7RT9~FheG4e90ZfuY~yNDKQLCk1ElET1jMIOcZ-ZvRoR8-ciHOAJD2JxJVXtEAaOP29rSs1jzIGlR26GUwYvHiP6qnjS3YSBlEhQ2ieQTIJ6WbrTvs8CyAyPolU5bKokQHLeSWrQ~Cqc8AVg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Physical map of 5811 candidate ILP markers and comparative map of 2275 RFLP and 2740 SSR markers. Each vertical short bar indicates the position of a candidate ILP marker. The three numbers in the brackets on the right of the map of each chromosome are the total number, number per Mb and number per cM of candidate ILP markers." path-from-xml="dsi019f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35868916/dsi019f2.jpeg" path-from-xml="dsi019f2.jpeg" target="_blank">View large</a><a section="35868916" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi019/3/dsi019f2.jpeg?Expires=2147483647&Signature=HlLzMkdB0hvgPEE64LTur66Dmu5a2yGOmbKFNgSUqqsjk8dVA261LP9aqYcfINiyhUlvbicQn78j7pmi~t58A0-a1aigNfmTS1bFTePB7iE9na7LuVMSBm0TexA87FqMfSMhoUrZWyWckHRMZVEhIPfP7S2DZFYzMzkSFtva7PDzdVzHx5knwkLK9VjDvXWURzxKCsCP~Xn4glZRMd2iExjcosIs1K9SCQHmo1PEH3X8eo-6Wvevn7CoFOlCGQK-B8erZN4d2SRfS7dNGTra20FD8EYNifywyQkZRG1-iU3enjsBIP3TJgcKAKJdE9YqJTlh9sDpIbnA9Difaz3bsA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35868916&ar=343854&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi019f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Physical map of 5811 candidate ILP markers and comparative map of 2275 RFLP and 2740 SSR markers. Each vertical short bar indicates the position of a candidate ILP marker. The three numbers in the brackets on the right of the map of each chromosome are the total number, number per Mb and number per cM of candidate ILP markers.</p></div></div><div content-id="fig2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi019/3/m_dsi019f2.jpeg?Expires=2147483647&Signature=lu1fO7GQIkQoyRxlUJ8lRfV0oftUidm9RUw0u7p-5iFjoB7LzzSWuokSU0q0ptp74SVcZVawfRVdaIIidzUdFdEKZiZ3ET6FDnUD6ElsRbLaHxCz1w8phMeucUglyyLqA7IUHqdzLjBldIZijZ~8-JFVscQ40zr9i-gNnV9OxMGzAPSVSuGzxXXHE~pKVWjfhN11DM7RT9~FheG4e90ZfuY~yNDKQLCk1ElET1jMIOcZ-ZvRoR8-ciHOAJD2JxJVXtEAaOP29rSs1jzIGlR26GUwYvHiP6qnjS3YSBlEhQ2ieQTIJ6WbrTvs8CyAyPolU5bKokQHLeSWrQ~Cqc8AVg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Physical map of 5811 candidate ILP markers and comparative map of 2275 RFLP and 2740 SSR markers. Each vertical short bar indicates the position of a candidate ILP marker. The three numbers in the brackets on the right of the map of each chromosome are the total number, number per Mb and number per cM of candidate ILP markers." path-from-xml="dsi019f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35868916/dsi019f2.jpeg" path-from-xml="dsi019f2.jpeg" target="_blank">View large</a><a section="35868916" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi019/3/dsi019f2.jpeg?Expires=2147483647&Signature=HlLzMkdB0hvgPEE64LTur66Dmu5a2yGOmbKFNgSUqqsjk8dVA261LP9aqYcfINiyhUlvbicQn78j7pmi~t58A0-a1aigNfmTS1bFTePB7iE9na7LuVMSBm0TexA87FqMfSMhoUrZWyWckHRMZVEhIPfP7S2DZFYzMzkSFtva7PDzdVzHx5knwkLK9VjDvXWURzxKCsCP~Xn4glZRMd2iExjcosIs1K9SCQHmo1PEH3X8eo-6Wvevn7CoFOlCGQK-B8erZN4d2SRfS7dNGTra20FD8EYNifywyQkZRG1-iU3enjsBIP3TJgcKAKJdE9YqJTlh9sDpIbnA9Difaz3bsA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35868916&ar=343854&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi019f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Physical map of 5811 candidate ILP markers and comparative map of 2275 RFLP and 2740 SSR markers. Each vertical short bar indicates the position of a candidate ILP marker. The three numbers in the brackets on the right of the map of each chromosome are the total number, number per Mb and number per cM of candidate ILP markers.</p></div></div>                        <a id="35868917" scrollto-destination="35868917"></a>
<div id="fig3" data-id="fig3" class="fig fig-section"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi019/3/m_dsi019f3.jpeg?Expires=2147483647&Signature=o0CBSdUxPv8i-Z2iO7rzbqXH5hTGFAvvpImwx8aTtzy3hliAba9kq8S8Aelx1naWtTLqnDA81B-uBLFOcOy~zqiqnqGBmq29h6iN0virCH6NtHsiSyQQVO1PFvU7F0BuhM6g-efY~phBeiGab5nFoi0aO7~gKMl-V3hLbcZWBjJRHM1ubzlKLLkUAe8uChM5wMp9DMCgCy44AicOdHqvDczSiuG6y4FOgxgd1yFg8nvYt6ip3xvALjFo639dpCoTNek1jRz9HZ7mS4wmYbH6TXdNdr2NvE4km1UrUhik431OXNuEs8Yn4K2ntJUYxH0Ar6INo9hzB6A-5o8U64L99g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Density distribution of NEH9s on chromosomes 1, 4, 9 and 12. The horizontal axis shows the pseudomolecule position (Mb); the vertical axis shows the number of NEH9s per Mb." path-from-xml="dsi019f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35868917/dsi019f3.jpeg" path-from-xml="dsi019f3.jpeg" target="_blank">View large</a><a section="35868917" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi019/3/dsi019f3.jpeg?Expires=2147483647&Signature=ipz9agIX1eYQOfnKT9hZmTbl3ScQUAmMiJXwy-3TCilAC13D5UmkTUOv1exJ3xey8oqBOCSw9NyKPZgWpQz79vxCRO2EKnSUdD4ktlf4~PeZE-eWoToUqdVeMViX0uZJHbsNhJpH92ztB7lkX7Th6voWgyhQLvPp0mBwCV015smzCKRGYILop0HRwzXKHIrmUQ9962WuJ2Meaqe5STcUCOOXwrCTKUihJLf49KkP4-H2RrgXkMjxG3BhfANfZkOIaeBMJqhWc3OrwOm7OemzTnWs7PY4Fq6RzTpmWoCgT8AvUmIPxtujb8mX8eIO0lr8rGzby0rEDPwmVXtP6qr-eQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35868917&ar=343854&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi019f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Density distribution of NEH9s on chromosomes 1, 4, 9 and 12. The horizontal axis shows the pseudomolecule position (Mb); the vertical axis shows the number of NEH9s per Mb.</p></div></div><div content-id="fig3" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi019/3/m_dsi019f3.jpeg?Expires=2147483647&Signature=o0CBSdUxPv8i-Z2iO7rzbqXH5hTGFAvvpImwx8aTtzy3hliAba9kq8S8Aelx1naWtTLqnDA81B-uBLFOcOy~zqiqnqGBmq29h6iN0virCH6NtHsiSyQQVO1PFvU7F0BuhM6g-efY~phBeiGab5nFoi0aO7~gKMl-V3hLbcZWBjJRHM1ubzlKLLkUAe8uChM5wMp9DMCgCy44AicOdHqvDczSiuG6y4FOgxgd1yFg8nvYt6ip3xvALjFo639dpCoTNek1jRz9HZ7mS4wmYbH6TXdNdr2NvE4km1UrUhik431OXNuEs8Yn4K2ntJUYxH0Ar6INo9hzB6A-5o8U64L99g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Density distribution of NEH9s on chromosomes 1, 4, 9 and 12. The horizontal axis shows the pseudomolecule position (Mb); the vertical axis shows the number of NEH9s per Mb." path-from-xml="dsi019f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35868917/dsi019f3.jpeg" path-from-xml="dsi019f3.jpeg" target="_blank">View large</a><a section="35868917" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi019/3/dsi019f3.jpeg?Expires=2147483647&Signature=ipz9agIX1eYQOfnKT9hZmTbl3ScQUAmMiJXwy-3TCilAC13D5UmkTUOv1exJ3xey8oqBOCSw9NyKPZgWpQz79vxCRO2EKnSUdD4ktlf4~PeZE-eWoToUqdVeMViX0uZJHbsNhJpH92ztB7lkX7Th6voWgyhQLvPp0mBwCV015smzCKRGYILop0HRwzXKHIrmUQ9962WuJ2Meaqe5STcUCOOXwrCTKUihJLf49KkP4-H2RrgXkMjxG3BhfANfZkOIaeBMJqhWc3OrwOm7OemzTnWs7PY4Fq6RzTpmWoCgT8AvUmIPxtujb8mX8eIO0lr8rGzby0rEDPwmVXtP6qr-eQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35868917&ar=343854&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi019f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Density distribution of NEH9s on chromosomes 1, 4, 9 and 12. The horizontal axis shows the pseudomolecule position (Mb); the vertical axis shows the number of NEH9s per Mb.</p></div></div><p> The length difference between allelic introns (referred to as intron length difference, ILD) in the candidate ILP marker loci appeared to follow an exponential distribution with a mean value of 11.42 bp ( <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4</a> ). Most (72.6%) of the ILDs were &lt;5 bp; 23.5% fell between 5 and 50 bp; and very few (3.9%) were &gt;50 bp. Generally speaking, the larger the ILD is, the easier the detection will be. Therefore, the candidate ILP markers with ILDs ≥ 5 bp should be preferentially considered in practical studies. However, the candidate ILP markers with small ILDs (&lt;5 bp) should not be ignored too, because they are in the majority. </p>                        <a id="35868919" scrollto-destination="35868919"></a>
<div id="fig4" data-id="fig4" class="fig fig-section"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi019/3/m_dsi019f4.jpeg?Expires=2147483647&Signature=lmMCtHL8axmxd8rboz8khNsS3~IzVHgiAS60bDToW1OmoAhT-fgA7EFUSZBZAHdsVMXUCCrfsO~Fo1FFeT-KMg1maGcsi2mprwjgr7WEb~YoiuSQeSnf7MppdzE-cfCbmRJ2OL02PMbvDmmjzkgQoBxPCeogD8wr2IWIp4botDMJOk2p-955kUFNmrsUyRzoohe-Awt7QNmCnAZNHiKpU58zj6awbnrnugBmOjOXPHuGyNgbHJcxEJQbymgwOVkMlsvERDgQDVO1fJIQnORuHTXo3qUvAb1kvgWNnObfXn9O1gM~hYRXRIQqR~ofg~0bgyx8zaElv92XLtjtf8Wy9A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The number distribution of intron length difference in 5811 candidate ILP markers." path-from-xml="dsi019f4.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35868919/dsi019f4.jpeg" path-from-xml="dsi019f4.jpeg" target="_blank">View large</a><a section="35868919" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi019/3/dsi019f4.jpeg?Expires=2147483647&Signature=WCiPxGpKuEXqxWTCbXl-ZRqdtbwMi11kL-ZgNPFsjwmOJtPmZSc~Hg8Oz~R16S0SkIZg0zQOLSfJdTN0suo5obk1bZPCIZXOzxktDDp-I8YsSnR7R2yjZexP0Y09po10P5EUYgCOyzcTTinZQD8ChU~IuJfBZ5OM8xvxWijYncJYJOCXG0Wj-18aFmT0hb8xwAOo3lUj~N~nopA8Zb7AycnQVMNpv9awD-VXJNdx3cmPvPqMAuUzXhFNudbnn4aUyXma4FWkmQNtvsaRb5hwuzx7Pj25ItsOGh4tr-~txpMSjsiR0t~P7QstduVnd~qx6NRxwZOlfpszi53f3tBWCg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35868919&ar=343854&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi019f4.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The number distribution of intron length difference in 5811 candidate ILP markers.</p></div></div><div content-id="fig4" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi019/3/m_dsi019f4.jpeg?Expires=2147483647&Signature=lmMCtHL8axmxd8rboz8khNsS3~IzVHgiAS60bDToW1OmoAhT-fgA7EFUSZBZAHdsVMXUCCrfsO~Fo1FFeT-KMg1maGcsi2mprwjgr7WEb~YoiuSQeSnf7MppdzE-cfCbmRJ2OL02PMbvDmmjzkgQoBxPCeogD8wr2IWIp4botDMJOk2p-955kUFNmrsUyRzoohe-Awt7QNmCnAZNHiKpU58zj6awbnrnugBmOjOXPHuGyNgbHJcxEJQbymgwOVkMlsvERDgQDVO1fJIQnORuHTXo3qUvAb1kvgWNnObfXn9O1gM~hYRXRIQqR~ofg~0bgyx8zaElv92XLtjtf8Wy9A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The number distribution of intron length difference in 5811 candidate ILP markers." path-from-xml="dsi019f4.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35868919/dsi019f4.jpeg" path-from-xml="dsi019f4.jpeg" target="_blank">View large</a><a section="35868919" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi019/3/dsi019f4.jpeg?Expires=2147483647&Signature=WCiPxGpKuEXqxWTCbXl-ZRqdtbwMi11kL-ZgNPFsjwmOJtPmZSc~Hg8Oz~R16S0SkIZg0zQOLSfJdTN0suo5obk1bZPCIZXOzxktDDp-I8YsSnR7R2yjZexP0Y09po10P5EUYgCOyzcTTinZQD8ChU~IuJfBZ5OM8xvxWijYncJYJOCXG0Wj-18aFmT0hb8xwAOo3lUj~N~nopA8Zb7AycnQVMNpv9awD-VXJNdx3cmPvPqMAuUzXhFNudbnn4aUyXma4FWkmQNtvsaRb5hwuzx7Pj25ItsOGh4tr-~txpMSjsiR0t~P7QstduVnd~qx6NRxwZOlfpszi53f3tBWCg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35868919&ar=343854&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi019f4.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The number distribution of intron length difference in 5811 candidate ILP markers.</p></div></div><p> Since ILPs are detected with specific PCR primers, they usually can be used as STS (sequence-tagging site) markers like SSRs. By examining the intron sequences of the 5811 candidate ILP markers, we found that only 208 (3.58%) of the ILPs were due to SSR variation, of which TA was the most frequent motif (19.2%), followed by GA (14.4%). This means that there is only a very small overlap between ILPs and SSRs in rice. In addition, we have seen that ILPs exist in many gaps in the physical map of RFLP and SSR markers ( <a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2</a> ). Therefore, ILPs are a new source of STS markers different from SSRs and can complement SSR and RFLP markers. In principle, every ILP is potentially a genetic marker as long as a suitable detection method is available. </p>                            <h3 scrollto-destination=35868921 id="35868921" class="section-title" >3.3. ILP markers exploited by experiment</h3>
<p> In order to detect ILPs by non-denaturing PAGE or agarose gel electrophoresis, we only chose candidate ILP markers with ILD ≥ 3 bp for the experiment. Of the 215 candidate ILP markers tested, 173 (80.47%) yielded stable and clear PCR products as expected in both Nipponbare and 93-11, and appeared to be codominant in the F <sub>1</sub> . Besides, six (2.79%) candidate ILP markers yielded the expected PCR products in either of the parents and appeared to be dominant in the F <sub>1</sub> . The number of ILP markers on each chromosome is shown in <a reveal-id="tbl2" data-open="tbl2" class="link link-reveal link-table xref-fig">Table 2</a> . </p>                        <a id="35868923" scrollto-destination="35868923"></a>
<div content-id="tbl2" class="table-modal"><div class="table-wrap"><div id="tbl2" class="table-wrap-title" data-id="tbl2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Number of ILP markers obtained by experiment</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>chromosome </th><th colspan="3">Based on EPIC-PCR </th><th colspan="3">Based on WIN-PCR </th></tr><tr><th> </th><th colspan="3"><hr /> </th><th colspan="3"><hr /> </th></tr><tr><th> </th><th>Analyzed </th><th>Detected </th><th>% </th><th>Analyzed </th><th>Detected </th><th>% </th></tr></thead><tbody><tr><td>1 </td><td>14 </td><td>10 </td><td char=".">71.43 </td><td>6 </td><td>5 </td><td char=".">83.33 </td></tr><tr><td>2 </td><td>20 </td><td>15 </td><td char=".">75.00 </td><td>6 </td><td>6 </td><td char=".">100.00 </td></tr><tr><td>3 </td><td>16 </td><td>14 </td><td char=".">87.50 </td><td>9 </td><td>7 </td><td char=".">77.78 </td></tr><tr><td>4 </td><td>5 </td><td>5 </td><td char=".">100.00 </td><td>4 </td><td>4 </td><td char=".">100.00 </td></tr><tr><td>5 </td><td>14 </td><td>13 </td><td char=".">92.86 </td><td>5 </td><td>4 </td><td char=".">80.00 </td></tr><tr><td>6 </td><td>12 </td><td>10 </td><td char=".">83.33 </td><td>10 </td><td>6 </td><td char=".">60.00 </td></tr><tr><td>7 </td><td>13 </td><td>11 </td><td char=".">84.62 </td><td>5 </td><td>4 </td><td char=".">80.00 </td></tr><tr><td>8 </td><td>15 </td><td>12 </td><td char=".">80.00 </td><td>8 </td><td>6 </td><td char=".">75.00 </td></tr><tr><td>9 </td><td>5 </td><td>5 </td><td char=".">100.00 </td><td>3 </td><td>3 </td><td char=".">100.00 </td></tr><tr><td>10 </td><td>10 </td><td>8 </td><td char=".">80.00 </td><td>6 </td><td>5 </td><td char=".">83.33 </td></tr><tr><td>11 </td><td>12 </td><td>11 </td><td char=".">91.67 </td><td>3 </td><td>3 </td><td char=".">100.00 </td></tr><tr><td>12 </td><td>10 </td><td>8 </td><td char=".">80.00 </td><td>4 </td><td>4 </td><td char=".">100.00 </td></tr><tr><td>Total </td><td>146 </td><td>122 </td><td char=".">83.56 </td><td>69 </td><td>57 </td><td char=".">82.61 </td></tr></tbody></table></div><div content-id="tbl2" class="table-modal"><table><thead name="thead" align=""><tr><th>chromosome </th><th colspan="3">Based on EPIC-PCR </th><th colspan="3">Based on WIN-PCR </th></tr><tr><th> </th><th colspan="3"><hr /> </th><th colspan="3"><hr /> </th></tr><tr><th> </th><th>Analyzed </th><th>Detected </th><th>% </th><th>Analyzed </th><th>Detected </th><th>% </th></tr></thead><tbody><tr><td>1 </td><td>14 </td><td>10 </td><td char=".">71.43 </td><td>6 </td><td>5 </td><td char=".">83.33 </td></tr><tr><td>2 </td><td>20 </td><td>15 </td><td char=".">75.00 </td><td>6 </td><td>6 </td><td char=".">100.00 </td></tr><tr><td>3 </td><td>16 </td><td>14 </td><td char=".">87.50 </td><td>9 </td><td>7 </td><td char=".">77.78 </td></tr><tr><td>4 </td><td>5 </td><td>5 </td><td char=".">100.00 </td><td>4 </td><td>4 </td><td char=".">100.00 </td></tr><tr><td>5 </td><td>14 </td><td>13 </td><td char=".">92.86 </td><td>5 </td><td>4 </td><td char=".">80.00 </td></tr><tr><td>6 </td><td>12 </td><td>10 </td><td char=".">83.33 </td><td>10 </td><td>6 </td><td char=".">60.00 </td></tr><tr><td>7 </td><td>13 </td><td>11 </td><td char=".">84.62 </td><td>5 </td><td>4 </td><td char=".">80.00 </td></tr><tr><td>8 </td><td>15 </td><td>12 </td><td char=".">80.00 </td><td>8 </td><td>6 </td><td char=".">75.00 </td></tr><tr><td>9 </td><td>5 </td><td>5 </td><td char=".">100.00 </td><td>3 </td><td>3 </td><td char=".">100.00 </td></tr><tr><td>10 </td><td>10 </td><td>8 </td><td char=".">80.00 </td><td>6 </td><td>5 </td><td char=".">83.33 </td></tr><tr><td>11 </td><td>12 </td><td>11 </td><td char=".">91.67 </td><td>3 </td><td>3 </td><td char=".">100.00 </td></tr><tr><td>12 </td><td>10 </td><td>8 </td><td char=".">80.00 </td><td>4 </td><td>4 </td><td char=".">100.00 </td></tr><tr><td>Total </td><td>146 </td><td>122 </td><td char=".">83.56 </td><td>69 </td><td>57 </td><td char=".">82.61 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/35868923" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl2" class="table-wrap-title" data-id="tbl2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Number of ILP markers obtained by experiment</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>chromosome </th><th colspan="3">Based on EPIC-PCR </th><th colspan="3">Based on WIN-PCR </th></tr><tr><th> </th><th colspan="3"><hr /> </th><th colspan="3"><hr /> </th></tr><tr><th> </th><th>Analyzed </th><th>Detected </th><th>% </th><th>Analyzed </th><th>Detected </th><th>% </th></tr></thead><tbody><tr><td>1 </td><td>14 </td><td>10 </td><td char=".">71.43 </td><td>6 </td><td>5 </td><td char=".">83.33 </td></tr><tr><td>2 </td><td>20 </td><td>15 </td><td char=".">75.00 </td><td>6 </td><td>6 </td><td char=".">100.00 </td></tr><tr><td>3 </td><td>16 </td><td>14 </td><td char=".">87.50 </td><td>9 </td><td>7 </td><td char=".">77.78 </td></tr><tr><td>4 </td><td>5 </td><td>5 </td><td char=".">100.00 </td><td>4 </td><td>4 </td><td char=".">100.00 </td></tr><tr><td>5 </td><td>14 </td><td>13 </td><td char=".">92.86 </td><td>5 </td><td>4 </td><td char=".">80.00 </td></tr><tr><td>6 </td><td>12 </td><td>10 </td><td char=".">83.33 </td><td>10 </td><td>6 </td><td char=".">60.00 </td></tr><tr><td>7 </td><td>13 </td><td>11 </td><td char=".">84.62 </td><td>5 </td><td>4 </td><td char=".">80.00 </td></tr><tr><td>8 </td><td>15 </td><td>12 </td><td char=".">80.00 </td><td>8 </td><td>6 </td><td char=".">75.00 </td></tr><tr><td>9 </td><td>5 </td><td>5 </td><td char=".">100.00 </td><td>3 </td><td>3 </td><td char=".">100.00 </td></tr><tr><td>10 </td><td>10 </td><td>8 </td><td char=".">80.00 </td><td>6 </td><td>5 </td><td char=".">83.33 </td></tr><tr><td>11 </td><td>12 </td><td>11 </td><td char=".">91.67 </td><td>3 </td><td>3 </td><td char=".">100.00 </td></tr><tr><td>12 </td><td>10 </td><td>8 </td><td char=".">80.00 </td><td>4 </td><td>4 </td><td char=".">100.00 </td></tr><tr><td>Total </td><td>146 </td><td>122 </td><td char=".">83.56 </td><td>69 </td><td>57 </td><td char=".">82.61 </td></tr></tbody></table></div><div content-id="tbl2" class="table-modal"><table><thead name="thead" align=""><tr><th>chromosome </th><th colspan="3">Based on EPIC-PCR </th><th colspan="3">Based on WIN-PCR </th></tr><tr><th> </th><th colspan="3"><hr /> </th><th colspan="3"><hr /> </th></tr><tr><th> </th><th>Analyzed </th><th>Detected </th><th>% </th><th>Analyzed </th><th>Detected </th><th>% </th></tr></thead><tbody><tr><td>1 </td><td>14 </td><td>10 </td><td char=".">71.43 </td><td>6 </td><td>5 </td><td char=".">83.33 </td></tr><tr><td>2 </td><td>20 </td><td>15 </td><td char=".">75.00 </td><td>6 </td><td>6 </td><td char=".">100.00 </td></tr><tr><td>3 </td><td>16 </td><td>14 </td><td char=".">87.50 </td><td>9 </td><td>7 </td><td char=".">77.78 </td></tr><tr><td>4 </td><td>5 </td><td>5 </td><td char=".">100.00 </td><td>4 </td><td>4 </td><td char=".">100.00 </td></tr><tr><td>5 </td><td>14 </td><td>13 </td><td char=".">92.86 </td><td>5 </td><td>4 </td><td char=".">80.00 </td></tr><tr><td>6 </td><td>12 </td><td>10 </td><td char=".">83.33 </td><td>10 </td><td>6 </td><td char=".">60.00 </td></tr><tr><td>7 </td><td>13 </td><td>11 </td><td char=".">84.62 </td><td>5 </td><td>4 </td><td char=".">80.00 </td></tr><tr><td>8 </td><td>15 </td><td>12 </td><td char=".">80.00 </td><td>8 </td><td>6 </td><td char=".">75.00 </td></tr><tr><td>9 </td><td>5 </td><td>5 </td><td char=".">100.00 </td><td>3 </td><td>3 </td><td char=".">100.00 </td></tr><tr><td>10 </td><td>10 </td><td>8 </td><td char=".">80.00 </td><td>6 </td><td>5 </td><td char=".">83.33 </td></tr><tr><td>11 </td><td>12 </td><td>11 </td><td char=".">91.67 </td><td>3 </td><td>3 </td><td char=".">100.00 </td></tr><tr><td>12 </td><td>10 </td><td>8 </td><td char=".">80.00 </td><td>4 </td><td>4 </td><td char=".">100.00 </td></tr><tr><td>Total </td><td>146 </td><td>122 </td><td char=".">83.56 </td><td>69 </td><td>57 </td><td char=".">82.61 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/35868923" target="_blank">View Large</a></div></div><p>To increase PCR specificity in the amplification of ILP loci, we adopted a Td-PCR program. Most (138/173) of the ILP markers obtained could be amplified well by the Td-PCR with the default initial annealing temperature (59°C). Some other (25 or 8) markers required a lower (57°C) or higher (60°C) initial annealing temperature. Eight markers could be well amplified at a constant annealing temperature (54°C).</p><p> We further tested the 173 codominant markers on 10 rice varieties. All the markers were successfully detected in those varieties ( <a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5</a> ), suggesting that the markers exploited are widely applicable. Based on the resolution capacity of the non-denaturing PAGE or agarose gel used, it appeared that most of the markers only possessed 2 (i.e. Nipponbare's and 93-11's) alleles among the 10 varieties. Only nine markers appeared to have multiple alleles, of which five were attributed to SSR variation. The PIC values of the markers varied from 0.18 to 0.66 with an average of 0.451. The results indicate that the polymorphism level of ILP marker is not high in general. However, a higher estimate of the polymorphism level of ILP markers could probably be obtained if methods of DNA fragment analysis with higher resolution capacity (e.g. denaturing PAGE, usually used for SSR analysis or DNA sequencing) were adopted. </p>                        <a id="35868926" scrollto-destination="35868926"></a>
<div id="fig5" data-id="fig5" class="fig fig-section"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi019/3/m_dsi019f5.jpeg?Expires=2147483647&Signature=FalppjHVSkQS9Pp4vTgcolf1ucahGwniYwyQR8RsCT36jJkqV7fPHFD4nVd~XEIMckKfvpe8dEKx7yAIgTOssm3~3d0Im2AS~FNcjmMjsGr08~WZ1lUmdCJhxWZaJMzqEyVVuitium-o50WJboQa6L9aM19SCA1VnWIO3KGSyrCiJ5MmD1QzKrKDPYva23PyofiTZMF2oV2gbSxWyyLma~CY3OjxSReyxWfYwTkarC2orNxgb-fJed-p8c9E9f3Jrlnz-7AWVz~oZmkcYhm7Xuae9cJ6qiq7wf1Y7A38GXKyMgIb1m1owZmCp7qzaio5rlGd~D3UUKFTYU0vun6~Qg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" PCR products of ILP marker RI01015 in rice accessions separated by electrophoresis on 2% agarose gel. Lanes from left to right: M = DNA molecular weight marker; 1 = 93-11; 2 = Nipponbare; 3 = 9311/Nipponbare F 1 ; 4 = Guangluai-4; 5 = Xieqingzao; 6 = IR64; 7 = Koshihikari; 8 = Xiushui-11; 9 = Merim; 10 = Katy; 11 = Kyeema. " path-from-xml="dsi019f5.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35868926/dsi019f5.jpeg" path-from-xml="dsi019f5.jpeg" target="_blank">View large</a><a section="35868926" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi019/3/dsi019f5.jpeg?Expires=2147483647&Signature=zmRdfv22JyBsg85pHxwnAzBld1ZvF4rmhT9YdgU22429ZXST5AFbGObxxKcsm68EwtVOzkPHVjPWM0wnD00XvzzeWFB5dGwJv~Tzx86kSjpU7F6rvJQTVT0826j0XNSKOkG0axZ8hhfI~GeXRmidUlJseFMOnMkco-sHJerWhpRv0Jlb9Z4UAJ1xmcuHEUmnKv0BIIQpWl9CHZES12BKgGbOnxSGfDkRV4on-KCuXhCRUGFedUujycyuMjqAsReX1ZCN54If0g6hJSo4OzbgFziFIU~Xt2-Om7jHcayParpKbsdUhyMglZDkXgoCwEJVFGP0dRVH7jBBNFeYCRmvOg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35868926&ar=343854&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi019f5.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> PCR products of ILP marker RI01015 in rice accessions separated by electrophoresis on 2% agarose gel. Lanes from left to right: M = DNA molecular weight marker; 1 = 93-11; 2 = Nipponbare; 3 = 9311/Nipponbare F <sub>1</sub> ; 4 = Guangluai-4; 5 = Xieqingzao; 6 = IR64; 7 = Koshihikari; 8 = Xiushui-11; 9 = Merim; 10 = Katy; 11 = Kyeema. </p></div></div><div content-id="fig5" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi019/3/m_dsi019f5.jpeg?Expires=2147483647&Signature=FalppjHVSkQS9Pp4vTgcolf1ucahGwniYwyQR8RsCT36jJkqV7fPHFD4nVd~XEIMckKfvpe8dEKx7yAIgTOssm3~3d0Im2AS~FNcjmMjsGr08~WZ1lUmdCJhxWZaJMzqEyVVuitium-o50WJboQa6L9aM19SCA1VnWIO3KGSyrCiJ5MmD1QzKrKDPYva23PyofiTZMF2oV2gbSxWyyLma~CY3OjxSReyxWfYwTkarC2orNxgb-fJed-p8c9E9f3Jrlnz-7AWVz~oZmkcYhm7Xuae9cJ6qiq7wf1Y7A38GXKyMgIb1m1owZmCp7qzaio5rlGd~D3UUKFTYU0vun6~Qg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" PCR products of ILP marker RI01015 in rice accessions separated by electrophoresis on 2% agarose gel. Lanes from left to right: M = DNA molecular weight marker; 1 = 93-11; 2 = Nipponbare; 3 = 9311/Nipponbare F 1 ; 4 = Guangluai-4; 5 = Xieqingzao; 6 = IR64; 7 = Koshihikari; 8 = Xiushui-11; 9 = Merim; 10 = Katy; 11 = Kyeema. " path-from-xml="dsi019f5.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35868926/dsi019f5.jpeg" path-from-xml="dsi019f5.jpeg" target="_blank">View large</a><a section="35868926" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi019/3/dsi019f5.jpeg?Expires=2147483647&Signature=zmRdfv22JyBsg85pHxwnAzBld1ZvF4rmhT9YdgU22429ZXST5AFbGObxxKcsm68EwtVOzkPHVjPWM0wnD00XvzzeWFB5dGwJv~Tzx86kSjpU7F6rvJQTVT0826j0XNSKOkG0axZ8hhfI~GeXRmidUlJseFMOnMkco-sHJerWhpRv0Jlb9Z4UAJ1xmcuHEUmnKv0BIIQpWl9CHZES12BKgGbOnxSGfDkRV4on-KCuXhCRUGFedUujycyuMjqAsReX1ZCN54If0g6hJSo4OzbgFziFIU~Xt2-Om7jHcayParpKbsdUhyMglZDkXgoCwEJVFGP0dRVH7jBBNFeYCRmvOg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35868926&ar=343854&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi019f5.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> PCR products of ILP marker RI01015 in rice accessions separated by electrophoresis on 2% agarose gel. Lanes from left to right: M = DNA molecular weight marker; 1 = 93-11; 2 = Nipponbare; 3 = 9311/Nipponbare F <sub>1</sub> ; 4 = Guangluai-4; 5 = Xieqingzao; 6 = IR64; 7 = Koshihikari; 8 = Xiushui-11; 9 = Merim; 10 = Katy; 11 = Kyeema. </p></div></div>                            <h3 scrollto-destination=35868927 id="35868927" class="section-title" >3.4. WIN-PCR for ILP detection</h3>
<p> In the present study, we adopted electronic EPIC-PCR to screen for candidate ILP markers. However, because the ILDs of &gt;70% candidate ILP markers were &lt;5 bp ( <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3</a> ) and the average size of the EPIC-PCR products was ∼580 bp, most of the candidate ILP markers would be difficult to detect by real EPIC-PCR. To solve this problem, a possible way is to design PCR primers within introns so as to obtain smaller PCR products. For this purpose, we examined nucleotide substitutions (SNPs) in introns between Nipponbare and 93-11 by sequence comparison <em>in silico</em> . We identified 17 374 putative nucleotide substitutions in the introns of the 5811 candidate ILP markers, approximately equivalent to 6 SNPs/kb ( <a reveal-id="tbl3" data-open="tbl3" class="link link-reveal link-table xref-fig">Table 3</a> ). Although the estimate of SNP frequency in intron sequences between the two subspecies of rice is much larger than that of whole genome average (1.06 SNPs/kb), <a reveal-id="b29" data-open="b29" class="link link-ref link-reveal xref-bibr">29</a> it is still not very high. Hence, it should be suitable to design PCR primers within introns to detect cross-subspecies ILPs in rice. To distinguish this technique, we call this approach Within INtron PCR (WIN-PCR). In this study, we have obtained 57 ILP markers from 69 candidate ILP markers by WIN-PCR. The success rate (57/69 = 82.61%) is very similar to that of EPIC-PCR (122/146 = 83.56%) ( <a reveal-id="tbl2" data-open="tbl2" class="link link-reveal link-table xref-fig">Table 2</a> ). The results indicate that WIN-PCR could be as efficient as EPIC-PCR. In principle, WIN-PCR would permit the detection of single-nucleotide ILPs. </p>                        <a id="35868929" scrollto-destination="35868929"></a>
<div content-id="tbl3" class="table-modal"><div class="table-wrap"><div id="tbl3" class="table-wrap-title" data-id="tbl3"><div class="label -label"><strong>Table 3</strong></div><div class="caption"><p>Single-nucleotide substitutions between 93-11 and Nipponbare in the intron sequences of 5811 candidate ILP markers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Type of substitution </th><th>Number </th><th>Proportion (%) </th></tr></thead><tbody><tr><td>G/C </td><td>1757 </td><td>10.11 </td></tr><tr><td>A/T </td><td>3032 </td><td>17.45 </td></tr><tr><td>A/C </td><td>2180 </td><td>12.55 </td></tr><tr><td>G/T </td><td>2145 </td><td>12.35 </td></tr><tr><td>A/G </td><td>4117 </td><td>23.70 </td></tr><tr><td>T/C </td><td>4143 </td><td>23.84 </td></tr><tr><td>Total </td><td>17374 </td><td>100 </td></tr></tbody></table></div><div content-id="tbl3" class="table-modal"><table><thead name="thead" align=""><tr><th>Type of substitution </th><th>Number </th><th>Proportion (%) </th></tr></thead><tbody><tr><td>G/C </td><td>1757 </td><td>10.11 </td></tr><tr><td>A/T </td><td>3032 </td><td>17.45 </td></tr><tr><td>A/C </td><td>2180 </td><td>12.55 </td></tr><tr><td>G/T </td><td>2145 </td><td>12.35 </td></tr><tr><td>A/G </td><td>4117 </td><td>23.70 </td></tr><tr><td>T/C </td><td>4143 </td><td>23.84 </td></tr><tr><td>Total </td><td>17374 </td><td>100 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/35868929" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl3" class="table-wrap-title" data-id="tbl3"><div class="label -label"><strong>Table 3</strong></div><div class="caption"><p>Single-nucleotide substitutions between 93-11 and Nipponbare in the intron sequences of 5811 candidate ILP markers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Type of substitution </th><th>Number </th><th>Proportion (%) </th></tr></thead><tbody><tr><td>G/C </td><td>1757 </td><td>10.11 </td></tr><tr><td>A/T </td><td>3032 </td><td>17.45 </td></tr><tr><td>A/C </td><td>2180 </td><td>12.55 </td></tr><tr><td>G/T </td><td>2145 </td><td>12.35 </td></tr><tr><td>A/G </td><td>4117 </td><td>23.70 </td></tr><tr><td>T/C </td><td>4143 </td><td>23.84 </td></tr><tr><td>Total </td><td>17374 </td><td>100 </td></tr></tbody></table></div><div content-id="tbl3" class="table-modal"><table><thead name="thead" align=""><tr><th>Type of substitution </th><th>Number </th><th>Proportion (%) </th></tr></thead><tbody><tr><td>G/C </td><td>1757 </td><td>10.11 </td></tr><tr><td>A/T </td><td>3032 </td><td>17.45 </td></tr><tr><td>A/C </td><td>2180 </td><td>12.55 </td></tr><tr><td>G/T </td><td>2145 </td><td>12.35 </td></tr><tr><td>A/G </td><td>4117 </td><td>23.70 </td></tr><tr><td>T/C </td><td>4143 </td><td>23.84 </td></tr><tr><td>Total </td><td>17374 </td><td>100 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/35868929" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=35868930 id="35868930" class="section-title" >3.5. Subspecies specificity and intra-subspecies diversity of ILPs</h3>
<p> Among the 10 accessions used for ILP analysis in the present study ( <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a> ), 93-11, Guangluai-4 and Xieqingzao could be taken as typical <em>indica</em> cultivars and Nipponbare, Xiushui-11 and Koshihikari as typical <em>japonica</em> cultivars according to their origins. Based on these typical accessions, 123 (71.1%) out of the 173 ILP markers tested showed subspecies-specific genotypes (i.e. different between but the same within subspecies). The result suggests that ILPs in rice have apparent subspecies specificity. This feature could be useful for analyzing the genetic compositions of rice cultivars. A typical example comes from the accession Kyeema. The accession was categorized as an <em>indica</em> cultivar based on morphological characters. <a reveal-id="b40" data-open="b40" class="link link-ref link-reveal xref-bibr">40</a> However, we have found that it is more likely to be a <em>japonica</em> cultivar because out of the 123 subspecies-specific ILP markers assayed on it, only 21 (17.1%) exhibited <em>indica</em> genotype, while 100 (81.3%) showed <em>japonica</em> genotype. This is consistent with its pedigree. In fact, Kyeema was derived from a triple cross involving one <em>indica</em> (Della) and two <em>japonica</em> (Pelde and Kulu) parents. We can expect that the offspring of the triple cross (Pelde//Della/Kulu) would contain 25% <em>indica</em> and 75% <em>japonica</em> genetic components on average. We see that the proportions in Kyeema's genome estimated by ILP markers are close to the expected values. Another example worthy of note is the accession Katy, a suggested <em>japonica</em> cultivar derived from a complicated cross Bonnet73/CI9722//Starbonnet/Tetep///Lebonnet, where Tetep is a typical <em>indica</em> parent. Our study has found that out of the 123 subspecies-specific ILP markers, 22 (17.9%) showed <em>indica</em> genotype in Katy. Therefore, Katy cannot be a typical <em>japonica</em> cultivar. In addition, 9 ILP markers showed heterozygous genotypes in Katy, suggesting that the accession might not be a pure line. A more interesting result we obtained concerns the <em>javanica</em> variety Merim. In this accession, 94 (or 76.4%) out of the 123 subspecies-specific ILP markers showed <em>japonica</em> genotype. This confirms that <em>javanica</em> belongs to <em>japonica</em> . <a reveal-id="b41" data-open="b41" class="link link-ref link-reveal xref-bibr">41</a></p><p> Although ILPs in rice have strong subspecies specificity, they still exist within subspecies. Based on the three typical <em>indica</em> and three typical <em>japonica</em> cultivars mentioned above, we found that of the 173 ILP markers tested, 44 (25.4%) showed polymorphisms among <em>indica</em> cultivars and 10 (5.8%) showed polymorphisms among <em>japonica</em> cultivars. The result indicates that <em>indica</em> rice has much higher genetic diversity than <em>japonica</em> rice. This is consistent with previous studies based on RAPD <a reveal-id="b42" data-open="b42" class="link link-ref link-reveal xref-bibr">42</a> and RFLP <a reveal-id="b43" data-open="b43" class="link link-ref link-reveal xref-bibr">43</a> markers. The finding implies that <em>indica</em> rice might have evolved earlier than <em>japonica</em> rice. </p>                            <h3 scrollto-destination=35868933 id="35868933" class="section-title" >3.6. Transferability of ILP markers to other plants</h3>
<p> We randomly selected 51 pairs of rice ILP primers to perform EPIC-PCR in other 8 plants (see Materials and Methods) and found that 24 (47.1%) pairs yielded desirable results with 1–5 clear and stable bands in all the plants; 31 (60.8%) pairs worked well in all the 5 <em>Gramineae</em> plants; only 6 (11.8%) pairs did not generate any PCR products. In cotton alone, 36 (70.6%) pairs of the primers yielded clear and stable PCR products and 11 (30.6%) of them revealed polymorphisms between two cultivated cotton species, <em>Gossypium barbadense</em> L. and <em>Gossypium hirsutum</em> L. The results suggest that a high proportion of rice ILP markers are transferable to other plants. </p><p> To examine whether the PCR products in other plants were really specifically amplified or homologous to the target genes in rice, we randomly isolated electrophoretic bands produced by primer pair RI02862 from wheat, maize and cotton, respectively ( <a reveal-id="fig6" data-open="fig6" class="link link-reveal link-table xref-fig">Fig. 6</a> ) and sequenced them (by Shanghai Sangon Biological Engineering &amp; Technology Company). Multiple alignment of the sequences of wheat, maize and cotton together with those of rice using computer programs ClustalX <a reveal-id="b44" data-open="b44" class="link link-ref link-reveal xref-bibr">44</a> and GeneDoc <a reveal-id="b45" data-open="b45" class="link link-ref link-reveal xref-bibr">45</a> showed that they were really from homologous genes as expected: the exon regions (two sides) were well conserved and the intron region (middle) were highly varied among the plants ( <a reveal-id="fig7" data-open="fig7" class="link link-reveal link-table xref-fig">Fig. 7</a> ). This suggests that most, if not all, of the clear and stable PCR products obtained in other plants must be resulted from specific amplification. </p>                        <a id="35868936" scrollto-destination="35868936"></a>
<div id="fig6" data-id="fig6" class="fig fig-section"><div class="label fig-label"><strong>Figure 6</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi019/3/m_dsi019f6.jpeg?Expires=2147483647&Signature=WoVFc58SuEHqq0tQlkGxZ6VdleBxfRHOEGZQkMJXCrYJIzNIBp1FhExLxG3fJYaIamB5sBcGUHT8ED0TeD6frR~hyNq8eXYpOPWQJKL1dVBaBuWmFnW8OaUaB3tSA8DjzaqSaKE09A-E8-LqkCbamrevXWQJJQ4UGJB5sMPwtUBh7M0WxAK5kSaM7dz8jTvJrAfLleJ4P3UcbLxssnGwH3-NCyfJwjs~mWOOevdihWTYfvmANi3Y5BI1zrLpc8olQ3SQw1Cfim4HfLI-KEydA20i-uzltIIiuz~D8qrrSUAeFt52IyPMCdMezztqq9bZ0hwfRJiHPm6HVZhkOqbNfQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" PCR products of ILP marker RI02862 in rice and other plants separated by electrophoresis on 6% non-denaturing PAGE. Lanes from left to right: 1 = japonica rice (Nipponbare); 2 = indica rice (93-11); 3 = barley; 4 = wheat; 5 = maize; 6 = sorghum; 7 = bamboo; 8 = rape; 9 = tobacco; 10 = cotton ( Gossypium hirsutum L.); 11 = cotton ( Gossypium barbadense L.). " path-from-xml="dsi019f6.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35868936/dsi019f6.jpeg" path-from-xml="dsi019f6.jpeg" target="_blank">View large</a><a section="35868936" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi019/3/dsi019f6.jpeg?Expires=2147483647&Signature=y2G4KX3omXV~0OoxJjI6bQ3Y9cGTTLdAizhYoF8rt4~ZBihBADdCN7kx5A2OSdZLBU8wQQ9gvd6L8p1zfRQiPuEVajozU61YCIYFl9eZjBCYXV2NQ~tKeiFm-DeCJbx4rhUYF1mMHHCOBUomZm03FA~~nZ4eTiKBTcd8EWkhXHgvsIJR69o2UJgLL5s5a5m-zTWDU8eIkjllJ64ddCF3r9hGYH5j6BjtyS5JEaLHtklaqI0N1kI1jLMpu531W4hvsQWfNH0UTl1zQneVqPjbx7a4pAq3G8yU~sSmzsHRyDnULK7SAOZ1naWkqNF2YJInkznpUba6A-9tOEJA6n8xFA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35868936&ar=343854&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi019f6.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> PCR products of ILP marker RI02862 in rice and other plants separated by electrophoresis on 6% non-denaturing PAGE. Lanes from left to right: 1 = <em>japonica</em> rice (Nipponbare); 2 = <em>indica</em> rice (93-11); 3 = barley; 4 = wheat; 5 = maize; 6 = sorghum; 7 = bamboo; 8 = rape; 9 = tobacco; 10 = cotton ( <em>Gossypium hirsutum</em> L.); 11 = cotton ( <em>Gossypium barbadense</em> L.). </p></div></div><div content-id="fig6" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 6</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi019/3/m_dsi019f6.jpeg?Expires=2147483647&Signature=WoVFc58SuEHqq0tQlkGxZ6VdleBxfRHOEGZQkMJXCrYJIzNIBp1FhExLxG3fJYaIamB5sBcGUHT8ED0TeD6frR~hyNq8eXYpOPWQJKL1dVBaBuWmFnW8OaUaB3tSA8DjzaqSaKE09A-E8-LqkCbamrevXWQJJQ4UGJB5sMPwtUBh7M0WxAK5kSaM7dz8jTvJrAfLleJ4P3UcbLxssnGwH3-NCyfJwjs~mWOOevdihWTYfvmANi3Y5BI1zrLpc8olQ3SQw1Cfim4HfLI-KEydA20i-uzltIIiuz~D8qrrSUAeFt52IyPMCdMezztqq9bZ0hwfRJiHPm6HVZhkOqbNfQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" PCR products of ILP marker RI02862 in rice and other plants separated by electrophoresis on 6% non-denaturing PAGE. Lanes from left to right: 1 = japonica rice (Nipponbare); 2 = indica rice (93-11); 3 = barley; 4 = wheat; 5 = maize; 6 = sorghum; 7 = bamboo; 8 = rape; 9 = tobacco; 10 = cotton ( Gossypium hirsutum L.); 11 = cotton ( Gossypium barbadense L.). " path-from-xml="dsi019f6.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35868936/dsi019f6.jpeg" path-from-xml="dsi019f6.jpeg" target="_blank">View large</a><a section="35868936" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi019/3/dsi019f6.jpeg?Expires=2147483647&Signature=y2G4KX3omXV~0OoxJjI6bQ3Y9cGTTLdAizhYoF8rt4~ZBihBADdCN7kx5A2OSdZLBU8wQQ9gvd6L8p1zfRQiPuEVajozU61YCIYFl9eZjBCYXV2NQ~tKeiFm-DeCJbx4rhUYF1mMHHCOBUomZm03FA~~nZ4eTiKBTcd8EWkhXHgvsIJR69o2UJgLL5s5a5m-zTWDU8eIkjllJ64ddCF3r9hGYH5j6BjtyS5JEaLHtklaqI0N1kI1jLMpu531W4hvsQWfNH0UTl1zQneVqPjbx7a4pAq3G8yU~sSmzsHRyDnULK7SAOZ1naWkqNF2YJInkznpUba6A-9tOEJA6n8xFA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35868936&ar=343854&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi019f6.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> PCR products of ILP marker RI02862 in rice and other plants separated by electrophoresis on 6% non-denaturing PAGE. Lanes from left to right: 1 = <em>japonica</em> rice (Nipponbare); 2 = <em>indica</em> rice (93-11); 3 = barley; 4 = wheat; 5 = maize; 6 = sorghum; 7 = bamboo; 8 = rape; 9 = tobacco; 10 = cotton ( <em>Gossypium hirsutum</em> L.); 11 = cotton ( <em>Gossypium barbadense</em> L.). </p></div></div>                        <a id="35868937" scrollto-destination="35868937"></a>
<div id="fig7" data-id="fig7" class="fig fig-section"><div class="label fig-label"><strong>Figure 7</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi019/3/m_dsi019f7.jpeg?Expires=2147483647&Signature=0AxZCYZvCzVOrUmh0yr3HcasEBhiStCT2a9FM~2Mpy-WEKFv~D2~kNAPA7seOwuZBFDeHFDTsnz4ASlo45AM3jpOFEvfYrfsOE9V3rcDJgRBLMf~xz6NUFKugWLjs~C1IBcr74PH3l-Rrwg50kNyRWlwgjrf35tJz7Mrwc1FklokmzSmDXsK6UEEfSrLyVX9PnwvqSHCtz0zVcISV57D7jN~dNYjt9WzO7tYLbmbMdvMXX5gkEhal4cbFycl6Yag5mq5JVpZVOpHAfTP~wAEJvwuyFuJipM4wbZ8eSIYxW8JNHaV1Z8NPj7rmi--YX9CA1wZtlTkojvTYNb13CdzzA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Multiple alignment of sequences amplified from maize, wheat and cotton ( Gossypium hirsutum L.) by primer pair RI02862 and available target sequences from japonica rice (Nipponbare) and indica rice (93-11). " path-from-xml="dsi019f7.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35868937/dsi019f7.jpeg" path-from-xml="dsi019f7.jpeg" target="_blank">View large</a><a section="35868937" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi019/3/dsi019f7.jpeg?Expires=2147483647&Signature=pVDVRricdKRXskOdHV2pYUjsEFzjO02SV7fYSGbLOvKac6jTH5sPi7Q8SZ-3lmg5DF2N0P~UOiuqxRS8UXDm9axaHdnTmR~QCR0vIbN7s7IJT~Sbx1f0V7bjE~WulmkfP4axpEGqm0cKGud~EhA-XMYZiAhW1QCXPAcqbjOhpp50Nnwgvi5Xq1lYl3oMNuYTbHcJCvs9wm7wo6aLbF-6ZWAjxafyCrytKpjFA~DNAMyr6n~LajprZdW6DXkhFX0Y-CRmQ~ZOYeWj9Xm95JvtDSACuy5N8L7dlmQf7W090YC6Pe~J-3yytb-FOK3SxIWiLaS-B45Wt03n3whTOlbPcQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35868937&ar=343854&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi019f7.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Multiple alignment of sequences amplified from maize, wheat and cotton ( <em>Gossypium hirsutum</em> L.) by primer pair RI02862 and available target sequences from <em>japonica</em> rice (Nipponbare) and <em>indica</em> rice (93-11). </p></div></div><div content-id="fig7" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 7</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi019/3/m_dsi019f7.jpeg?Expires=2147483647&Signature=0AxZCYZvCzVOrUmh0yr3HcasEBhiStCT2a9FM~2Mpy-WEKFv~D2~kNAPA7seOwuZBFDeHFDTsnz4ASlo45AM3jpOFEvfYrfsOE9V3rcDJgRBLMf~xz6NUFKugWLjs~C1IBcr74PH3l-Rrwg50kNyRWlwgjrf35tJz7Mrwc1FklokmzSmDXsK6UEEfSrLyVX9PnwvqSHCtz0zVcISV57D7jN~dNYjt9WzO7tYLbmbMdvMXX5gkEhal4cbFycl6Yag5mq5JVpZVOpHAfTP~wAEJvwuyFuJipM4wbZ8eSIYxW8JNHaV1Z8NPj7rmi--YX9CA1wZtlTkojvTYNb13CdzzA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Multiple alignment of sequences amplified from maize, wheat and cotton ( Gossypium hirsutum L.) by primer pair RI02862 and available target sequences from japonica rice (Nipponbare) and indica rice (93-11). " path-from-xml="dsi019f7.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35868937/dsi019f7.jpeg" path-from-xml="dsi019f7.jpeg" target="_blank">View large</a><a section="35868937" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi019/3/dsi019f7.jpeg?Expires=2147483647&Signature=pVDVRricdKRXskOdHV2pYUjsEFzjO02SV7fYSGbLOvKac6jTH5sPi7Q8SZ-3lmg5DF2N0P~UOiuqxRS8UXDm9axaHdnTmR~QCR0vIbN7s7IJT~Sbx1f0V7bjE~WulmkfP4axpEGqm0cKGud~EhA-XMYZiAhW1QCXPAcqbjOhpp50Nnwgvi5Xq1lYl3oMNuYTbHcJCvs9wm7wo6aLbF-6ZWAjxafyCrytKpjFA~DNAMyr6n~LajprZdW6DXkhFX0Y-CRmQ~ZOYeWj9Xm95JvtDSACuy5N8L7dlmQf7W090YC6Pe~J-3yytb-FOK3SxIWiLaS-B45Wt03n3whTOlbPcQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35868937&ar=343854&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi019f7.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Multiple alignment of sequences amplified from maize, wheat and cotton ( <em>Gossypium hirsutum</em> L.) by primer pair RI02862 and available target sequences from <em>japonica</em> rice (Nipponbare) and <em>indica</em> rice (93-11). </p></div></div><p> For comparison, we also applied 32 pairs of rice SSR primers to the 9 plants. We found that although all the SSR primers could generate PCR products in all the plants, the electrophoretic bands produced by each pair of primers were generally quite many (at least 10) and unstable, suggesting that most, if not all, of the bands are produced by unspecific amplification as seen in the randomly primed markers such as RAPD. <a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a> Therefore, it appears that most of the rice SSR markers are not transferable to other plants, but can only serve as unspecific PCR markers in other plants. </p>                            <h3 scrollto-destination=35868939 id="35868939" class="section-title" >3.7. Advantages of ILP markers</h3>
<p>ILP is a new type of molecular marker, which has not been reported extensively. We have seen that ILPs are abundant between the two cultivated subspecies in rice. ILP has many similar advantages to SSR including specific (being a STS marker), codominant (providing complete information of genotypes), neutral (no phenotypic effect), convenient (detectable by PCR) and reliable (result stable). In addition, ILP has a special advantage, namely, it directly reflects variation within genes. Therefore, the genetic maps constructed with ILP markers would be more valuable for genetic studies because they are similar to conventional maps consisting of morphological markers. Moreover, ILP marker would be more useful for marker-assisted breeding because it allows us to trace a gene directly as long as an ILP can be found in the gene.</p><p>We have seen that ILPs have significant subspecies specificity in rice. This characteristic could be useful for genetic study and breeding in rice. Apart from the use for analyzing genetic compositions of rice cultivars discussed above, it might be useful for the studies of genome evolution and inter-subspecies heterosis and for cross-subspecies marker-assisted breeding.</p><p>In addition, we have seen the high transferability of rice ILP markers to other plants. This characteristic would make ILP markers very useful for (i) construction of molecular marker maps in other plants that have weaker genetic research basis; (ii) genome comparison among plants; (iii) gene mapping with the help of synteny or collinearity between model plants and other plants; (iv) research of phylogenetic relationships among different species, genera, families or even higher taxonomic ranks in plants and (v) marker-assisted breeding in other plants.</p>                            <h3 scrollto-destination=35868943 id="35868943" class="section-title" >3.8. ILP database</h3>
<p> We have established a database ( <a class="link link-uri" href="http://ibi.zju.edu.cn/ILPs/index.htm" target="_blank"></a> ) for depositing the information of the 5811 candidate ILP markers, including ILP name, cDNA name, <em>japonica</em> BAC clone accession number, <em>japonica</em> BAC clone name, marker start position in <em>japonica</em> (bp), maker end position in <em>japonica</em> (bp), marker length in <em>japonica</em> (bp), indica scaffold name, marker start position in <em>indica</em> (bp), marker end position in <em>indica</em> (bp), marker length in <em>indica</em> (bp), length difference between <em>japonica</em> and <em>indica</em> (bp), position in RFLP map (cM), forward primer and reverse primer. For convenience, the ILP information is searched by key words, such as ILP name, BAC clone and scaffold. In addition, the 173 codominant ILP markers obtained have been submitted to GenBank (accession nos.: <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/BV209990" target="_blank">BV209990</a> – <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/BV210161" target="_blank">BV210161</a> , <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/BV210393" target="_blank">BV210393</a> ). </p><p>This work was funded by the National High-Tech Research and Development Program of China (project: 2003AA207160 &amp; 2002AA234031) and by IBM Shared University Research (SUR) program. The authors thank Dr Adrian Cutler from Plant Biotechnology Institute, National Research Council of Canada for helpful suggestions on the manuscript.</p>                            <h2 scrollto-destination=35868947 id="35868947" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="b1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Berget</div> <div class="given-names">S. M.</div></div>,  <div class="name"><div class="surname">Moore</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Sharp</div> <div class="given-names">P. A.</div></div>. </div><div class="article-title">Spliced segments at the 5′ terminus of adenovirus 2 late mRNA</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1977</div>, vol. <div class="volume">74</div> (pg. <div class="fpage">3171</div>-<div class="lpage">3175</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Spliced%20segments%20at%20the%205%E2%80%B2%20terminus%20of%20adenovirus%202%20late%20mRNA&amp;author=S.%20M.%20Berget&amp;author=C.%20Moore&amp;author=P.%20A.%20Sharp&amp;publication_year=1977&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=74&amp;pages=3171-3175" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.74.8.3171" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.74.8.3171" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.74.8.3171"> </span></div> </div></div></div></div></div><div content-id="b2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chow</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Gilinas</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Broker</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Roberts</div> <div class="given-names">R.</div></div>. </div><div class="article-title">An amazing sequence arrangement at the 5′ ends of adenovirus 2 messenger RNA</div>, <div class="source">Cell</div> , <div class="year">1977</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">1</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20amazing%20sequence%20arrangement%20at%20the%205%E2%80%B2%20ends%20of%20adenovirus%202%20messenger%20RNA&amp;author=L.%20Chow&amp;author=R.%20Gilinas&amp;author=T.%20Broker&amp;author=R.%20Roberts&amp;publication_year=1977&amp;journal=Cell&amp;volume=12&amp;pages=1-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0092-8674(77)90180-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0092-8674(77)90180-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0092-8674(77)90180-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/902310" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hawkins</div> <div class="given-names">J. D.</div></div>. </div><div class="article-title">A survey on intron and exon lengths</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1988</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">9893</div>-<div class="lpage">9908</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20survey%20on%20intron%20and%20exon%20lengths&amp;author=J.%20D.%20Hawkins&amp;publication_year=1988&amp;journal=Nucleic%20Acids%20Res.&amp;volume=16&amp;pages=9893-9908" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/16.21.9893" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F16.21.9893" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F16.21.9893"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3057449" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Deutsch</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Long</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Intron–exon structures of eukaryotic model organisms</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1999</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">3219</div>-<div class="lpage">3228</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intron%E2%80%93exon%20structures%20of%20eukaryotic%20model%20organisms&amp;author=M.%20Deutsch&amp;author=M.%20Long&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=3219-3228" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.15.3219" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.15.3219" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.15.3219"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10454621" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Adams</div> <div class="given-names">M. D.</div></div>,  <div class="name"><div class="surname">Celniker</div> <div class="given-names">S. E.</div></div>,  <div class="name"><div class="surname">Holt</div> <div class="given-names">R. A.</div></div>, et al. </div><div class="article-title"> The genome sequence of <em>Drosophila melanogaster</em></div>, <div class="source">Science</div> , <div class="year">2000</div>, vol. <div class="volume">287</div> (pg. <div class="fpage">2185</div>-<div class="lpage">2195</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20sequence%20of%20Drosophila%20melanogaster&amp;author=M.%20D.%20Adams&amp;author=S.%20E.%20Celniker&amp;author=R.%20A.%20Holt&amp;publication_year=2000&amp;journal=Science&amp;volume=287&amp;pages=2185-2195" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.287.5461.2185" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.287.5461.2185" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.287.5461.2185"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10731132" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Venter</div> <div class="given-names">J. C.</div></div>,  <div class="name"><div class="surname">Adams</div> <div class="given-names">M. C.</div></div>,  <div class="name"><div class="surname">Myers</div> <div class="given-names">E. W.</div></div>, et al. </div><div class="article-title">The sequence of the human genome</div>, <div class="source">Science</div> , <div class="year">2001</div>, vol. <div class="volume">291</div> (pg. <div class="fpage">1304</div>-<div class="lpage">1351</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20sequence%20of%20the%20human%20genome&amp;author=J.%20C.%20Venter&amp;author=M.%20C.%20Adams&amp;author=E.%20W.%20Myers&amp;publication_year=2001&amp;journal=Science&amp;volume=291&amp;pages=1304-1351" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1058040" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1058040" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1058040"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11181995" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">D. G.</div></div>,  <div class="name"><div class="surname">Fan</div> <div class="given-names">J. B.</div></div>,  <div class="name"><div class="surname">Siao</div> <div class="given-names">C. J.</div></div>, et al. </div><div class="article-title">Large-scale identification, mapping, and genotyping of single-nucleotide polymorphisms in the human genome</div>, <div class="source">Science</div> , <div class="year">1998</div>, vol. <div class="volume">280</div> (pg. <div class="fpage">1077</div>-<div class="lpage">1082</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20identification%2C%20mapping%2C%20and%20genotyping%20of%20single-nucleotide%20polymorphisms%20in%20the%20human%20genome&amp;author=D.%20G.%20Wang&amp;author=J.%20B.%20Fan&amp;author=C.%20J.%20Siao&amp;publication_year=1998&amp;journal=Science&amp;volume=280&amp;pages=1077-1082" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.280.5366.1077" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.280.5366.1077" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.280.5366.1077"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9582121" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Botstein</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">White</div> <div class="given-names">R. L.</div></div>,  <div class="name"><div class="surname">Skolnick</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Construction of a genetic linkage map in man using restriction fragment length polymorphisms</div>, <div class="source">Am. J. Hum. Genet.</div> , <div class="year">1980</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">314</div>-<div class="lpage">331</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20genetic%20linkage%20map%20in%20man%20using%20restriction%20fragment%20length%20polymorphisms&amp;author=D.%20Botstein&amp;author=R.%20L.%20White&amp;author=M.%20Skolnick&amp;publication_year=1980&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=32&amp;pages=314-331" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6247908" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Williams</div> <div class="given-names">J. G. K.</div></div>,  <div class="name"><div class="surname">Kubelik</div> <div class="given-names">A. R.</div></div>,  <div class="name"><div class="surname">Livak</div> <div class="given-names">K. J.</div></div>, et al. </div><div class="article-title">DNA polymorphism amplified by arbitrary primers are useful as genetic markers</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1990</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">6531</div>-<div class="lpage">6535</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20polymorphism%20amplified%20by%20arbitrary%20primers%20are%20useful%20as%20genetic%20markers&amp;author=J.%20G.%20K.%20Williams&amp;author=A.%20R.%20Kubelik&amp;author=K.%20J.%20Livak&amp;publication_year=1990&amp;journal=Nucleic%20Acids%20Res.&amp;volume=18&amp;pages=6531-6535" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/18.22.6531" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F18.22.6531" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F18.22.6531"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1979162" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vos</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Hogers</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Bleeker</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">AFLP: a new technique for DNA fingerprinting</div>, <div class="source">Nucleic Acid Res.</div> , <div class="year">1995</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">4407</div>-<div class="lpage">4414</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AFLP%3A%20a%20new%20technique%20for%20DNA%20fingerprinting&amp;author=P.%20Vos&amp;author=R.%20Hogers&amp;author=M.%20Bleeker&amp;publication_year=1995&amp;journal=Nucleic%20Acid%20Res.&amp;volume=23&amp;pages=4407-4414" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/23.21.4407" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F23.21.4407" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F23.21.4407"> </span></div> </div></div></div></div></div><div content-id="b11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Becker</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Heun</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Barley microsatellites: allele variation and mapping</div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">1995</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">835</div>-<div class="lpage">845</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Barley%20microsatellites%3A%20allele%20variation%20and%20mapping&amp;author=J.%20Becker&amp;author=M.%20Heun&amp;publication_year=1995&amp;journal=Plant%20Mol.%20Biol.&amp;volume=27&amp;pages=835-845" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00020238" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00020238" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00020238"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7727762" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Becker</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Vos</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Kuiper</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Combined mapping of AFLP and RFLP markers in barley</div>, <div class="source">Mol. Gen. Genet.</div> , <div class="year">1995</div>, vol. <div class="volume">249</div> (pg. <div class="fpage">65</div>-<div class="lpage">73</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Combined%20mapping%20of%20AFLP%20and%20RFLP%20markers%20in%20barley&amp;author=J.%20Becker&amp;author=P.%20Vos&amp;author=M.%20Kuiper&amp;publication_year=1995&amp;journal=Mol.%20Gen.%20Genet.&amp;volume=249&amp;pages=65-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00290237" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00290237" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00290237"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8552035" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Quiros</div> <div class="given-names">C. F.</div></div>. </div><div class="article-title">Sequence-related amplified polymorphism (SRAP) a new marker system based on a simple PCR reaction: its application to mapping and gene tagging in Brassica</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2001</div>, vol. <div class="volume">103</div> (pg. <div class="fpage">455</div>-<div class="lpage">461</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence-related%20amplified%20polymorphism%20(SRAP)%20a%20new%20marker%20system%20based%20on%20a%20simple%20PCR%20reaction%3A%20its%20application%20to%20mapping%20and%20gene%20tagging%20in%20Brassica&amp;author=G.%20Li&amp;author=C.%20F.%20Quiros&amp;publication_year=2001&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=103&amp;pages=455-461" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220100570" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220100570" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220100570"> </span></div> </div></div></div></div></div><div content-id="b14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kruglyak</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Nickerson</div> <div class="given-names">D. A.</div></div>. </div><div class="article-title">Variation is the spice of life</div>, <div class="source">Nat. Genet.</div> , <div class="year">2001</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">234</div>-<div class="lpage">236</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Variation%20is%20the%20spice%20of%20life&amp;author=L.%20Kruglyak&amp;author=D.%20A.%20Nickerson&amp;publication_year=2001&amp;journal=Nat.%20Genet.&amp;volume=27&amp;pages=234-236" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/85776" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F85776" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F85776"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11242096" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lessa</div> <div class="given-names">E. P.</div></div>. </div><div class="article-title">Rapid survey of DNA sequence variation in natural populations</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1992</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">323</div>-<div class="lpage">330</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20survey%20of%20DNA%20sequence%20variation%20in%20natural%20populations&amp;author=E.%20P.%20Lessa&amp;publication_year=1992&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=9&amp;pages=323-330" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1560767" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Côrte-Real</div> <div class="given-names">H. B. S. M.</div></div>,  <div class="name"><div class="surname">Dixon</div> <div class="given-names">D. R.</div></div>,  <div class="name"><div class="surname">Holland</div> <div class="given-names">P. W. H.</div></div>. </div><div class="article-title">Intron-targeted PCR: a new approach to survey neutral DNA polymorphism in bivalve populations</div>, <div class="source">Mar. Biol.</div> , <div class="year">1994</div>, vol. <div class="volume">120</div> (pg. <div class="fpage">407</div>-<div class="lpage">413</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intron-targeted%20PCR%3A%20a%20new%20approach%20to%20survey%20neutral%20DNA%20polymorphism%20in%20bivalve%20populations&amp;author=H.%20B.%20S.%20M.%20C%C3%B4rte-Real&amp;author=D.%20R.%20Dixon&amp;author=P.%20W.%20H.%20Holland&amp;publication_year=1994&amp;journal=Mar.%20Biol.&amp;volume=120&amp;pages=407-413" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00680214" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00680214" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00680214"> </span></div> </div></div></div></div></div><div content-id="b17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Daguin</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Borsa</div> <div class="given-names">P.</div></div>. </div><div class="article-title"> Genetic characterisation of <em>Mytilus galloprovincialis</em> Lmk. in North West Africa using nuclear DNA markers </div>, <div class="source">J Exp. Mar. Biol. Ecol.</div> , <div class="year">1999</div>, vol. <div class="volume">235</div> (pg. <div class="fpage">55</div>-<div class="lpage">65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20characterisation%20of%20Mytilus%20galloprovincialis%20Lmk.%20in%20North%20West%20Africa%20using%20nuclear%20DNA%20markers&amp;author=C.%20Daguin&amp;author=P.%20Borsa&amp;publication_year=1999&amp;journal=J%20Exp.%20Mar.%20Biol.%20Ecol.&amp;volume=235&amp;pages=55-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-0981(98)00163-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-0981(98)00163-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-0981(98)00163-4"> </span></div> </div></div></div></div></div><div content-id="b18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wydner</div> <div class="given-names">K. S.</div></div>,  <div class="name"><div class="surname">Sechler</div> <div class="given-names">J. L.</div></div>,  <div class="name"><div class="surname">Boyd</div> <div class="given-names">C. D.</div></div>, et al. </div><div class="article-title">Use of an intron length polymorphism to localize the tropoelastin gene to chromosome 5 in a region of linkage conservation with human chromosome 7</div>, <div class="source">Genomics</div> , <div class="year">1994</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">125</div>-<div class="lpage">131</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Use%20of%20an%20intron%20length%20polymorphism%20to%20localize%20the%20tropoelastin%20gene%20to%20chromosome%205%20in%20a%20region%20of%20linkage%20conservation%20with%20human%20chromosome%207&amp;author=K.%20S.%20Wydner&amp;author=J.%20L.%20Sechler&amp;author=C.%20D.%20Boyd&amp;publication_year=1994&amp;journal=Genomics&amp;volume=23&amp;pages=125-131" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/geno.1994.1467" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fgeno.1994.1467" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fgeno.1994.1467"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7829060" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Palumbi</div> <div class="given-names">S. R.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Hillis</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Moritz</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Nucleic acids II: the polymerase chain reaction</div>, <div class="source">Molecular Systematics</div> , <div class="year">1995</div><div class="edition">2nd edn</div><div class="publisher-loc">Sunderland, MA</div><div class="publisher-name">Sinauer Associates Inc.</div>(pg. <div class="fpage">205</div>-<div class="lpage">247</div>)</div></div></div></div><div content-id="b20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bierne</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Lehnert</div> <div class="given-names">S. A.</div></div>,  <div class="name"><div class="surname">Bédier</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Screening for intron-length polymorphisms in penaeid shrimps using exon-primed intron-crossing (EPIC)-PCR</div>, <div class="source">Mol. Ecol.</div> , <div class="year">2000</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">233</div>-<div class="lpage">235</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Screening%20for%20intron-length%20polymorphisms%20in%20penaeid%20shrimps%20using%20exon-primed%20intron-crossing%20(EPIC)-PCR&amp;author=N.%20Bierne&amp;author=S.%20A.%20Lehnert&amp;author=E.%20B%C3%A9dier&amp;publication_year=2000&amp;journal=Mol.%20Ecol.&amp;volume=9&amp;pages=233-235" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-294x.2000.00842.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-294x.2000.00842.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-294x.2000.00842.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10672168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Hu</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title"> A draft sequence of the rice genome ( <em>Oryza sativa</em> L. ssp. <em>indica</em> ) </div>, <div class="source">Science</div> , <div class="year">2002</div>, vol. <div class="volume">296</div> (pg. <div class="fpage">79</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20draft%20sequence%20of%20the%20rice%20genome%20(%20Oryza%20sativa%20L.%20ssp.%20indica%20)&amp;author=J.%20Yu&amp;author=S.%20Hu&amp;author=J.%20Wang&amp;publication_year=2002&amp;journal=Science&amp;volume=296&amp;pages=79-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1068037" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1068037" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1068037"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11935017" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Goff</div> <div class="given-names">S. A.</div></div>,  <div class="name"><div class="surname">Ricke</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Lan</div> <div class="given-names">T. H.</div></div>, et al. </div><div class="article-title"> A draft sequence of the rice genomes ( <em>Oryza sativa</em> L. ssp. <em>japonica</em> ) </div>, <div class="source">Science</div> , <div class="year">2002</div>, vol. <div class="volume">296</div> (pg. <div class="fpage">92</div>-<div class="lpage">100</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20draft%20sequence%20of%20the%20rice%20genomes%20(%20Oryza%20sativa%20L.%20ssp.%20japonica%20)&amp;author=S.%20A.%20Goff&amp;author=D.%20Ricke&amp;author=T.%20H.%20Lan&amp;publication_year=2002&amp;journal=Science&amp;volume=296&amp;pages=92-100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1068275" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1068275" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1068275"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11935018" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kikuchi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Satoh</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nagata</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Collection, mapping, and annotation of over 28 000 cDNA clones from japonica rice</div>, <div class="source">Science</div> , <div class="year">2003</div>, vol. <div class="volume">301</div> (pg. <div class="fpage">376</div>-<div class="lpage">379</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Collection%2C%20mapping%2C%20and%20annotation%20of%20over%2028%E2%80%89000%20cDNA%20clones%20from%20japonica%20rice&amp;author=S.%20Kikuchi&amp;author=K.%20Satoh&amp;author=T.%20Nagata&amp;publication_year=2003&amp;journal=Science&amp;volume=301&amp;pages=376-379" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1081288" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1081288" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1081288"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12869764" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sasaki</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Matsumoto</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Yamamoto</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">The genome sequence and structure of rice chromosome 1</div>, <div class="source">Nature</div> , <div class="year">2002</div>, vol. <div class="volume">420</div> (pg. <div class="fpage">312</div>-<div class="lpage">316</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20sequence%20and%20structure%20of%20rice%20chromosome%201&amp;author=T.%20Sasaki&amp;author=T.%20Matsumoto&amp;author=K.%20Yamamoto&amp;publication_year=2002&amp;journal=Nature&amp;volume=420&amp;pages=312-316" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature01184" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature01184" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature01184"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12447438" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Feng</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Hao</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Sequence and analysis of rice chromosome 4</div>, <div class="source">Nature</div> , <div class="year">2002</div>, vol. <div class="volume">420</div> (pg. <div class="fpage">316</div>-<div class="lpage">320</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20and%20analysis%20of%20rice%20chromosome%204&amp;author=Q.%20Feng&amp;author=Y.%20Zhang&amp;author=P.%20Hao&amp;publication_year=2002&amp;journal=Nature&amp;volume=420&amp;pages=316-320" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature01183" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature01183" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature01183"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12447439" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="collab">The Rice Chromosome 10 Sequencing Consortium</div><div class="article-title">In-depth view of structure, activity, and evolution of rice chromosome 10</div>, <div class="source">Science</div> , <div class="year">2003</div>, vol. <div class="volume">300</div> (pg. <div class="fpage">1566</div>-<div class="lpage">1569</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1083523" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1083523" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1083523"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12791992" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="collab">International Rice Genome Sequencing Project</div><div class="article-title">The map-based sequence of the rice genome</div>, <div class="source">Nature</div> , <div class="year">2005</div>, vol. <div class="volume">436</div> (pg. <div class="fpage">793</div>-<div class="lpage">800</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature03895" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature03895" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature03895"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16100779" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shen</div> <div class="given-names">Y. J.</div></div>,  <div class="name"><div class="surname">Jiang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Jin</div> <div class="given-names">J. P.</div></div>, et al. </div><div class="article-title">Development of genome-wide DNA polymorphism database for map-based cloning of rice genes</div>, <div class="source">Plant physiol.</div> , <div class="year">2004</div>, vol. <div class="volume">135</div> (pg. <div class="fpage">1198</div>-<div class="lpage">1205</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20genome-wide%20DNA%20polymorphism%20database%20for%20map-based%20cloning%20of%20rice%20genes&amp;author=Y.%20J.%20Shen&amp;author=H.%20Jiang&amp;author=J.%20P.%20Jin&amp;publication_year=2004&amp;journal=Plant%20physiol.&amp;volume=135&amp;pages=1198-1205" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.103.038463" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.103.038463" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.103.038463"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15266053" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Feltus</div> <div class="given-names">F. A.</div></div>,  <div class="name"><div class="surname">Wan</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Schulze</div> <div class="given-names">S. R.</div></div>, et al. </div><div class="article-title"> An SNP resource for rice genetics and breeding based on subspecies <em>indica</em> and <em>japonica</em> genome alignments </div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">1812</div>-<div class="lpage">1819</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20SNP%20resource%20for%20rice%20genetics%20and%20breeding%20based%20on%20subspecies%20indica%20and%20japonica%20genome%20alignments&amp;author=F.%20A.%20Feltus&amp;author=J.%20Wan&amp;author=S.%20R.%20Schulze&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=1812-1819" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.2479404" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.2479404" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.2479404"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15342564" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Madden</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Schaffer</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Gapped BLAST and PSI-BLAST: A new generation of protein database search programs</div>, <div class="source">Nucleic Acids. Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3389</div>-<div class="lpage">3402</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20A%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.%20Altschul&amp;author=T.%20Madden&amp;author=A.%20Schaffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids.%20Res.&amp;volume=25&amp;pages=3389-3402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Florea</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Hartzell</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>, et al. </div><div class="article-title">A computer program for aligning a cDNA sequence with a genomic DNA sequence</div>, <div class="source">Genome Res.</div> , <div class="year">1998</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">967</div>-<div class="lpage">974</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20computer%20program%20for%20aligning%20a%20cDNA%20sequence%20with%20a%20genomic%20DNA%20sequence&amp;author=L.%20Florea&amp;author=G.%20Hartzell&amp;author=Z.%20Zhang&amp;publication_year=1998&amp;journal=Genome%20Res.&amp;volume=8&amp;pages=967-974" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9750195" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rice</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Longden</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Bleasby</div> <div class="given-names">A.</div></div>. </div><div class="article-title">EMBOSS: The European Molecular Biology Open Software Suite</div>, <div class="source">Trends Genet.</div> , <div class="year">2000</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">276</div>-<div class="lpage">277</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EMBOSS%3A%20The%20European%20Molecular%20Biology%20Open%20Software%20Suite&amp;author=P.%20Rice&amp;author=I.%20Longden&amp;author=A.%20Bleasby&amp;publication_year=2000&amp;journal=Trends%20Genet.&amp;volume=16&amp;pages=276-277" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(00)02024-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(00)02024-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(00)02024-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10827456" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schuler</div> <div class="given-names">G. D.</div></div>. </div><div class="article-title">Sequence mapping by electronic PCR</div>, <div class="source">Genome Res.</div> , <div class="year">1997</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">541</div>-<div class="lpage">550</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20mapping%20by%20electronic%20PCR&amp;author=G.%20D.%20Schuler&amp;publication_year=1997&amp;journal=Genome%20Res.&amp;volume=7&amp;pages=541-550" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9149949" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Harushima</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Yano</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Shomura</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title"> A high-density rice genetic linkage map with 2275 markers using a single F <sub>2</sub> population </div>, <div class="source">Genetics</div> , <div class="year">1998</div>, vol. <div class="volume">148</div> (pg. <div class="fpage">479</div>-<div class="lpage">494</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high-density%20rice%20genetic%20linkage%20map%20with%202275%20markers%20using%20a%20single%20F%202%20population&amp;author=Y.%20Harushima&amp;author=M.%20Yano&amp;author=A.%20Shomura&amp;publication_year=1998&amp;journal=Genetics&amp;volume=148&amp;pages=479-494" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9475757" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Murray</div> <div class="given-names">M. G.</div></div>,  <div class="name"><div class="surname">Thompson</div> <div class="given-names">W. F.</div></div>. </div><div class="article-title">Rapid isolation of high-molecular- weight plant DNA</div>, <div class="source">Nucleic Acids. Res.</div> , <div class="year">1980</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">4321</div>-<div class="lpage">4325</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20isolation%20of%20high-molecular-%20weight%20plant%20DNA&amp;author=M.%20G.%20Murray&amp;author=W.%20F.%20Thompson&amp;publication_year=1980&amp;journal=Nucleic%20Acids.%20Res.&amp;volume=8&amp;pages=4321-4325" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/8.19.4321" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F8.19.4321" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F8.19.4321"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7433111" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Don</div> <div class="given-names">R. H.</div></div>,  <div class="name"><div class="surname">Cox</div> <div class="given-names">P. T.</div></div>,  <div class="name"><div class="surname">Wainwright</div> <div class="given-names">B. J.</div></div>, et al. </div><div class="article-title">‘Touchdown’ PCR to circumvent spurious priming during gene amplification</div>, <div class="source">Nucleic Acids. Res.</div> , <div class="year">1991</div>, vol. <div class="volume">19</div> pg. <div class="fpage">4008</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=%E2%80%98Touchdown%E2%80%99%20PCR%20to%20circumvent%20spurious%20priming%20during%20gene%20amplification&amp;author=R.%20H.%20Don&amp;author=P.%20T.%20Cox&amp;author=B.%20J.%20Wainwright&amp;publication_year=1991&amp;journal=Nucleic%20Acids.%20Res.&amp;volume=19&amp;pages=4008" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/19.14.4008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F19.14.4008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F19.14.4008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1861999" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xu</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Tao</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">Z.</div></div>, et al. </div><div class="article-title">A simple and rapid method used for silver staining and gel preservation</div>, <div class="source">Hereditas (Beijing)</div> , <div class="year">2002</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">335</div>-<div class="lpage">336</div>)</div></div></div></div><div content-id="b38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Anderson</div> <div class="given-names">J. A.</div></div>,  <div class="name"><div class="surname">Churchill</div> <div class="given-names">G. A.</div></div>,  <div class="name"><div class="surname">Autrique</div> <div class="given-names">J. E.</div></div>, et al. </div><div class="article-title">Optimizing parental selection for genetic linkage maps</div>, <div class="source">Genome</div> , <div class="year">1993</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">181</div>-<div class="lpage">186</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Optimizing%20parental%20selection%20for%20genetic%20linkage%20maps&amp;author=J.%20A.%20Anderson&amp;author=G.%20A.%20Churchill&amp;author=J.%20E.%20Autrique&amp;publication_year=1993&amp;journal=Genome&amp;volume=36&amp;pages=181-186" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g93-024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg93-024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg93-024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18469981" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Han</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Xue</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Genome-wide intraspecific DNA-sequence variations in rice</div>, <div class="source">Curr. Opin. Plant Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">134</div>-<div class="lpage">138</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20intraspecific%20DNA-sequence%20variations%20in%20rice&amp;author=B.%20Han&amp;author=Y.%20Xue&amp;publication_year=2003&amp;journal=Curr.%20Opin.%20Plant%20Biol.&amp;volume=6&amp;pages=134-138" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1369-5266(03)00004-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1369-5266(03)00004-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1369-5266(03)00004-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12667869" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wolfgang</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Marc</div> <div class="given-names">H. E.</div></div>,  <div class="name"><div class="surname">Peter</div> <div class="given-names">M. C.</div></div>. </div><div class="article-title"> Semidwarf ( <em>sd-1</em> ), “green revolution” rice, contains a defective gibberellin 20-oxidase gene </div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2002</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">9043</div>-<div class="lpage">9048</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Semidwarf%20(%20sd-1%20)%2C%20%E2%80%9Cgreen%20revolution%E2%80%9D%20rice%2C%20contains%20a%20defective%20gibberellin%2020-oxidase%20gene&amp;author=S.%20Wolfgang&amp;author=H.%20E.%20Marc&amp;author=M.%20C.%20Peter&amp;publication_year=2002&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=99&amp;pages=9043-9048" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.132266399" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.132266399" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.132266399"> </span></div> </div></div></div></div></div><div content-id="b41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Glaszmann</div> <div class="given-names">J. C.</div></div>. </div><div class="article-title">Isozymes and classification of Asian rice varieties</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">1987</div>, vol. <div class="volume">74</div> (pg. <div class="fpage">21</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isozymes%20and%20classification%20of%20Asian%20rice%20varieties&amp;author=J.%20C.%20Glaszmann&amp;publication_year=1987&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=74&amp;pages=21-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00290078" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00290078" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00290078"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24241451" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Machill</div> <div class="given-names">D. J.</div></div>. </div><div class="article-title"> Classifying <em>japonica</em> rice cultivars with RAPD markers </div>, <div class="source">Crop Sci.</div> , <div class="year">1995</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">889</div>-<div class="lpage">894</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Classifying%20japonica%20rice%20cultivars%20with%20RAPD%20markers&amp;author=D.%20J.%20Machill&amp;publication_year=1995&amp;journal=Crop%20Sci.&amp;volume=35&amp;pages=889-894" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci1995.0011183X003500030043x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci1995.0011183X003500030043x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci1995.0011183X003500030043x"> </span></div> </div></div></div></div></div><div content-id="b43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lu</div> <div class="given-names">B. R.</div></div>,  <div class="name"><div class="surname">Zheng</div> <div class="given-names">K. L.</div></div>,  <div class="name"><div class="surname">Qian</div> <div class="given-names">H. R.</div></div>, et al. </div><div class="article-title">Genetic differentiation of wild relatives of rice as assessed by RFLP analysis</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">101</div>-<div class="lpage">106</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20differentiation%20of%20wild%20relatives%20of%20rice%20as%20assessed%20by%20RFLP%20analysis&amp;author=B.%20R.%20Lu&amp;author=K.%20L.%20Zheng&amp;author=H.%20R.%20Qian&amp;publication_year=2002&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=106&amp;pages=101-106" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12582876" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thompson</div> <div class="given-names">J. D.</div></div>,  <div class="name"><div class="surname">Gibson</div> <div class="given-names">T. J.</div></div>,  <div class="name"><div class="surname">Plewniak</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">The ClustalX windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">4876</div>-<div class="lpage">4882</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20ClustalX%20windows%20interface%3A%20flexible%20strategies%20for%20multiple%20sequence%20alignment%20aided%20by%20quality%20analysis%20tools&amp;author=J.%20D.%20Thompson&amp;author=T.%20J.%20Gibson&amp;author=F.%20Plewniak&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=24&amp;pages=4876-4882" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.24.4876" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.24.4876" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.24.4876"> </span></div> </div></div></div></div></div><div content-id="b45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nicholas</div> <div class="given-names">K. B.</div></div>,  <div class="name"><div class="surname">Nicholas</div> <div class="given-names">H. B.</div></div>. </div><div class="article-title">GeneDoc: a tool for editing and annotating multiple sequence alignments</div>, <div class="year">1997</div> <div class="comment"> Distributed by the authors ( <a class="link link-uri" href="http://www.psc.edu/biomed/genedoc" target="_blank"></a> ) </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2006. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=343854&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">444</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000242119900003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">60</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsi019"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="343854"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000242119900003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f12%2f6%2f417%2f343854">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000242119900003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (60)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F12%2F6%2F417%2F343854" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/343854'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value=" Genome-wide Investigation of Intron Length Polymorphisms and Their Potential as Molecular Markers in Rice ( Oryza sativa L.) " />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsi019");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f12%2f6%2f417%2f343854">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

