

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":1,"applicationTime":1648,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/20/2/135/333376","@type":"ScholarlyArticle","name":"Evaluation of Codon Biology in  Citrus  and  Poncirus trifoliata  Based on Genomic Features and Frame Corrected Expressed Sequence Tags","datePublished":"2013-01-11","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/20/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2013-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dss039","keywords":["<italic>Citrus</italic>","coevolution","mutational bias","<italic>Poncirus trifoliata</italic>","stress","GC<sub>3</sub> biology"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/20/2/135/333376","author":[{"name":"Ahmad, Touqeer","affiliation":"1  Key Laboratory of Horticultural Plant Biology (MOE) ,  Huazhong Agricultural University ,  Wuhan 430070 ,  China","@type":"Person"},{"name":"Sablok, Gaurav","affiliation":"2  Research and Innovation Center, Fondazione Edmund Mach (FEM), Istituto Agrario San Michele (IASMA) ,  Via Mach 1., San Michele all'Adige, Trentino 38010 ,  Italy","@type":"Person"},{"name":"Tatarinova, Tatiana V.","affiliation":"3  Glamorgan Computational Biology Group, University of Glamorgan, Pontypridd CF37 1DL ,  UK","@type":"Person"},{"name":"Xu, Qiang","affiliation":"1  Key Laboratory of Horticultural Plant Biology (MOE) ,  Huazhong Agricultural University ,  Wuhan 430070 ,  China","@type":"Person"},{"name":"Deng, Xiu-Xin","affiliation":"1  Key Laboratory of Horticultural Plant Biology (MOE) ,  Huazhong Agricultural University ,  Wuhan 430070 ,  China","@type":"Person"},{"name":"Guo, Wen-Wu","affiliation":"1  Key Laboratory of Horticultural Plant Biology (MOE) ,  Huazhong Agricultural University ,  Wuhan 430070 ,  China","@type":"Person"}],"description":"Abstract.  Citrus, as one of the globally important fruit trees, has been an object of interest for understanding genetics and evolutionary process in fruit cro","pageStart":"135","pageEnd":"150","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/2/1/m_cover.gif?Expires=2147483647&Signature=Yq7dbhq6hb-Iyo5ewEGt-Y1T99brR4Xh8FvVnCI2Bg1SIlPGztlGCk-Bm0A9A72SaqDKcetVlEQehNtxyAGT~T55ZFFD19A3hKk~A1XbnQ-JfXYKj2-gYHz-VirOlvLe-8XkC6qvT4h2PniinfIJH6d85JsIf3rhRRtN086VVGvi2qNVR0THPJOeMpQvElVQEEN-4bpo3wEYmq80uwX58y5ScJ0pcJt7sV40xw1Fd6mp92v-su9vXuGT~IIVjyHRDyuiGt~ZiUzcXtor6WX4MT1zcCCf5ySC3Wm7DfaZruIvVsbtJxHUYCwePrgmcTq1vhdeOWHam2svM-jBZyoDoA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Evaluation of Codon Biology in  Citrus  and  Poncirus trifoliata  Based on Genomic Features and Frame Corrected Expressed Sequence Tags","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/2/1/m_cover.gif?Expires=2147483647&Signature=Yq7dbhq6hb-Iyo5ewEGt-Y1T99brR4Xh8FvVnCI2Bg1SIlPGztlGCk-Bm0A9A72SaqDKcetVlEQehNtxyAGT~T55ZFFD19A3hKk~A1XbnQ-JfXYKj2-gYHz-VirOlvLe-8XkC6qvT4h2PniinfIJH6d85JsIf3rhRRtN086VVGvi2qNVR0THPJOeMpQvElVQEEN-4bpo3wEYmq80uwX58y5ScJ0pcJt7sV40xw1Fd6mp92v-su9vXuGT~IIVjyHRDyuiGt~ZiUzcXtor6WX4MT1zcCCf5ySC3Wm7DfaZruIvVsbtJxHUYCwePrgmcTq1vhdeOWHam2svM-jBZyoDoA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Evaluation of Codon Biology in  Citrus  and  Poncirus trifoliata  Based on Genomic Features and Frame Corrected Expressed Sequence Tags" />
    <meta property="og:description" content="Abstract.  Citrus, as one of the globally important fruit trees, has been an object of interest for understanding genetics and evolutionary process in fruit cro" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/20/2/135/333376" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2013-01-11" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Ahmad, Touqeer" /><meta name="citation_author_institution" content="Key Laboratory of Horticultural Plant Biology (MOE), Huazhong Agricultural University, Wuhan 430070, China" /><meta name="citation_author" content="Sablok, Gaurav" /><meta name="citation_author_institution" content="Research and Innovation Center, Fondazione Edmund Mach (FEM), Istituto Agrario San Michele (IASMA), Via Mach 1., San Michele all'Adige, Trentino 38010, Italy" /><meta name="citation_author" content="Tatarinova, Tatiana V." /><meta name="citation_author_institution" content="Glamorgan Computational Biology Group, University of Glamorgan, Pontypridd CF37 1DL, UK" /><meta name="citation_author" content="Xu, Qiang" /><meta name="citation_author_institution" content="Key Laboratory of Horticultural Plant Biology (MOE), Huazhong Agricultural University, Wuhan 430070, China" /><meta name="citation_author" content="Deng, Xiu-Xin" /><meta name="citation_author_institution" content="Key Laboratory of Horticultural Plant Biology (MOE), Huazhong Agricultural University, Wuhan 430070, China" /><meta name="citation_author" content="Guo, Wen-Wu" /><meta name="citation_author_institution" content="Key Laboratory of Horticultural Plant Biology (MOE), Huazhong Agricultural University, Wuhan 430070, China" /><meta name="citation_title" content="Evaluation of Codon Biology in Citrus and Poncirus trifoliata Based on Genomic Features and Frame Corrected Expressed Sequence Tags" /><meta name="citation_firstpage" content="135" /><meta name="citation_lastpage" content="150" /><meta name="citation_doi" content="10.1093/dnares/dss039" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="20" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2013/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Patterns and evolution of nucleotide landscapes in seed plants; Plant Cell;  citation_year=2012;  citation_volume=24;  citation_pages=1379-97; " /><meta name="citation_reference" content="citation_title=General rules for optimal codon choice; PLoS Genet.;  citation_year=2009;  citation_volume=5;  citation_pages=e1000556" /><meta name="citation_reference" content="citation_title=Codon catalog usage is a genome strategy modulated for gene expressivity; Nucleic Acids Res.;  citation_year=1981;  citation_volume=9;  citation_pages=43-74; " /><meta name="citation_reference" content="citation_title=Gene expression and molecular evolution; Curr. Opin. Genet. Dev.;  citation_year=2001;  citation_volume=11;  citation_pages=660-6; " /><meta name="citation_reference" content="citation_title=Fine-tuning translation kinetics selection as the driving force of codon usage bias in the hepatitis A virus capsid; PLoS Pathol.;  citation_year=2010;  citation_volume=6;  citation_pages=e1000797" /><meta name="citation_reference" content="citation_title=Evidence for horizontal gene transfer in Escherichia coli speciation; J. Mol. Biol.;  citation_year=1991;  citation_volume=222;  citation_pages=851-6; " /><meta name="citation_reference" content="citation_title=Universal biases in protein composition of model prokaryotes; Proteins;  citation_year=2005;  citation_volume=60;  citation_pages=27-35; " /><meta name="citation_reference" content="citation_title=Evolution of synonymous codon usage in metazoans; Curr. Opin. Genet. Dev.;  citation_year=2002;  citation_volume=12;  citation_pages=640-9; " /><meta name="citation_reference" content="citation_title=The selection-mutation drift theory of synonymous codon usage; Genetics;  citation_year=1991;  citation_volume=129;  citation_pages=897-907; " /><meta name="citation_reference" content="citation_title=Codon usage and genome evolution; Curr. Opin. Genet. Dev.;  citation_year=1994;  citation_volume=4;  citation_pages=851-60; " /><meta name="citation_reference" content="citation_title=Translational selection and molecular evolution; Curr. Opin. Genet. Dev.;  citation_year=1998;  citation_volume=8;  citation_pages=688-93; " /><meta name="citation_reference" content="citation_title=Gene expressivity is the main factor in dictating the codon usage variation among the genes in Pseudomonas aeruginosa; Gene;  citation_year=2001;  citation_volume=273;  citation_pages=63-70; " /><meta name="citation_reference" content="citation_title=Selection on codon bias; Annu. Rev. Genet.;  citation_year=2008;  citation_volume=42;  citation_pages=287-99; " /><meta name="citation_reference" content="citation_title=Biased gene conversion affects patterns of codon usage and amino acid usage in the Saccharomyces sensu stricto group of yeasts; Mol. Biol. Evol.;  citation_year=2011;  citation_volume=28;  citation_pages=117-29; " /><meta name="citation_reference" content="citation_title=The effect of linkage on limits to artificial selection; Genet. Res.;  citation_year=1966;  citation_volume=8;  citation_pages=269-94; " /><meta name="citation_reference" content="citation_title=Replicational and transcriptional selection on codon usage in Borrelia burgdorferi; Proc. Natl. Acad. Sci. USA;  citation_year=1998;  citation_volume=95;  citation_pages=10698-703; " /><meta name="citation_reference" content="citation_title=Codon usage between genomes is constrained by genome-wide mutational processes; Proc. Natl. Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=3480-85; " /><meta name="citation_reference" content="citation_title=Intragenic spatial patterns of codon usage bias in prokaryotic and eukaryotic genomes; Genetics;  citation_year=2004;  citation_volume=168;  citation_pages=2245-60; " /><meta name="citation_reference" content="citation_title=Synonymous codon usage in Escherichia coli: selection for translational accuracy; Mol. Biol. Evol.;  citation_year=2007;  citation_volume=24;  citation_pages=374-81; " /><meta name="citation_reference" content="citation_title=Codon-usage bias versus gene conversion in the evolution of yeast duplicate genes; Proc. Natl. Acad. Sci. USA;  citation_year=2006;  citation_volume=103;  citation_pages=14412-16; " /><meta name="citation_reference" content="citation_title=Synonymous codon usage in environmental Chlamydia UWE25 reflects an evolutional divergence from pathogenic Chlamydiae; Gene;  citation_year=2006;  citation_volume=368;  citation_pages=117-25; " /><meta name="citation_reference" content="citation_title=Forces that influence the evolution of codon bias; Philos. Trans. R. Soc. Lond. B Biol. Sci.;  citation_year=2010;  citation_volume=365;  citation_pages=1203-12; " /><meta name="citation_reference" content="citation_title=The ‘effective number of codons’ used in a gene; Gene;  citation_year=1990;  citation_volume=87;  citation_pages=23-29; " /><meta name="citation_reference" content="citation_title=Quantitative relationship between synonymous bias and GC composition across unicellular genomes; BMC Evol. Biol.;  citation_year=2004;  citation_volume=4;  citation_pages=19" /><meta name="citation_reference" content="citation_title=The draft genome of sweet orange (Citrus sinensis); Nat. Genet.;  citation_year=2012; " /><meta name="citation_reference" content="citation_title=CAP3: a DNA sequence assembly program; Genome Res.;  citation_year=1999;  citation_volume=9;  citation_pages=868-77; " /><meta name="citation_reference" content="citation_title=FrameDP: sensitive peptide detection on noisy matured sequences; Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=670-71; " /><meta name="citation_reference" content="citation_title=FrameD: a flexible program for quality check and gene prediction in prokaryotic genomes and noisy matured eukaryotic sequences; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=373-81; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-402; " /><meta name="citation_reference" content="citation_publisher=Academic Press, London; Theory and Application of Correspondence Analysis;  citation_year=1984;  citation_pages=223" /><meta name="citation_reference" content="citation_title=An evolutionary perspective on synonymous codon usage in unicellular organisms; J. Mol. Biol.;  citation_year=1986;  citation_volume=24;  citation_pages=28-38; " /><meta name="citation_reference" content="citation_title=A simple method for displaying the hydropathic character of a protein; J. Mol. Biol.;  citation_year=1982;  citation_volume=157;  citation_pages=105-32; " /><meta name="citation_reference" content="citation_title=Codon usage and tRNA content in unicellular and multicellular organisms; Mol. Biol. Evol.;  citation_year=1985;  citation_volume=2;  citation_pages=13-34; " /><meta name="citation_reference" content="citation_title=Synonymous codon usage, GC3 and evolutionary patterns across plastomes of three pooid model species – emerging grass genome models for monocots; Mol. Biotechnol.;  citation_year=2011;  citation_volume=49;  citation_pages=116-28; " /><meta name="citation_reference" content="citation_title=Proteinortho: detection of (co-) orthologs in large-scale analysis; BMC Bioinformatics;  citation_year=2011;  citation_volume=12;  citation_pages=124" /><meta name="citation_reference" content="citation_title=Nucleotide composition bias affects amino acid content in proteins coded by animal mitochondria; J. Mol. Evol.;  citation_year=1997;  citation_volume=44;  citation_pages=282-88; " /><meta name="citation_reference" content="citation_title=Nucleotide bias causes a genomewide bias in the amino acid composition of proteins; Mol. Biol. Evol.;  citation_year=2000;  citation_volume=17;  citation_pages=1581-88; " /><meta name="citation_reference" content="citation_title=Analysis of codon usage in Citrus; J. Fruit Sci.;  citation_year=2006;  citation_volume=23;  citation_pages=479-85; " /><meta name="citation_reference" content="citation_title=On the evolution of informational macromolecules; citation_publisher=Academic Press, New York; Evolving Genes and Proteins;  citation_year=1964;  citation_pages=479-96; " /><meta name="citation_reference" content="citation_title=DNA G+ C content of the third codon position and codon usage biases of human genes; Gene;  citation_year=2000;  citation_volume=261;  citation_pages=53-62; " /><meta name="citation_reference" content="citation_title=Directional mutation pressure and neutral molecular evolution; Proc. Natl. Acad. Sci. USA;  citation_year=1988;  citation_volume=85;  citation_pages=2653-57; " /><meta name="citation_reference" content="citation_title=Rapid divergence of codon usage patterns within the rice genome; BMC Evol. Biol.;  citation_year=2007;  citation_volume=7;  citation_issue=Suppl. 1;  citation_pages=S6" /><meta name="citation_reference" content="citation_title=Synonymous codon usage in Lactococcus lactis: mutational bias versus translational selection; J. Biomol. Struct. Dyn.;  citation_year=2004;  citation_volume=21;  citation_pages=527-36; " /><meta name="citation_reference" content="citation_title=Codon usage in the G+C-rich Streptomyces genome; Gene;  citation_year=1992;  citation_volume=113;  citation_pages=55-65; " /><meta name="citation_reference" content="citation_title=GC3 biology in corn, rice, sorghum and other grasses; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=308" /><meta name="citation_reference" content="citation_title=Codon usage in Escherichia coli, Bacillus subtilis, Saccharomyces cerevisiae, Schizosaccharomyces pombe, Drosophila melanogaster and Homo sapiens; a review of the considerable within-species diversity; Nucleic Acids Res.;  citation_year=1988;  citation_volume=16;  citation_pages=8207-11; " /><meta name="citation_reference" content="citation_title=Codon preferences in free-living microorganisms; Microbiol. Rev.;  citation_year=1990;  citation_volume=54;  citation_pages=198-210; " /><meta name="citation_reference" content="citation_title=Balanced codon usage optimizes eukaryotic translational efficiency; PLoS Genet.;  citation_year=2012;  citation_volume=8;  citation_pages=e1002603" /><meta name="citation_reference" content="citation_title=Absolute in vivo translation rates of individual codons in Escherichia coli: the two glutamic acid codons GAA and GAG are translated with a threefold difference in rate; J. Mol. Biol.;  citation_year=1991;  citation_volume=222;  citation_pages=265-80; " /><meta name="citation_reference" content="citation_title=Coding-sequence determinants of gene expression in Escherichia coli; Science;  citation_year=2009;  citation_volume=324;  citation_pages=255-58; " /><meta name="citation_reference" content="citation_title=The significance of digital gene expression profiles; Genome Res.;  citation_year=1997;  citation_volume=7;  citation_pages=986-95; " /><meta name="citation_reference" content="citation_title=Microarray and EST database estimates of mRNA expression levels differ: the protein length versus expression curve for C. elegans; BMC Genomics;  citation_year=2004;  citation_volume=5;  citation_pages=30" /><meta name="citation_reference" content="citation_title=Patterns of molecular evolution in Caenorhabditis preclude ancient origins of selfing; Genetics;  citation_year=2008;  citation_volume=178;  citation_pages=2093-104; " /><meta name="citation_reference" content="citation_title=Molecular evolution of synonymous codon usage in Populus; BMC Evol. Biol.;  citation_year=2008;  citation_volume=8;  citation_pages=307" /><meta name="citation_reference" content="citation_title=Evolution of synonymous codon usage in Neurospora tetrasperma and Neurospora discrete; Genome Biol. Evol.;  citation_year=2011;  citation_volume=3;  citation_pages=332-43; " /><meta name="citation_reference" content="citation_title=A simple model based on mutation and selection explains trends in codon and amino acid usage and GC composition within and across genomes; Genome Biol.;  citation_year=2001;  citation_volume=2;  citation_pages=research0010" /><meta name="citation_reference" content="citation_title=Codon usage patterns in Nematoda: analysis based on over 25 million codons in thirty-two species; Genome Biol.;  citation_year=2006;  citation_volume=7;  citation_pages=R75" /><meta name="citation_reference" content="citation_title=A general model of codon bias due to GC mutational bias; PLoS ONE;  citation_year=2010;  citation_volume=5;  citation_pages=e13431" /><meta name="citation_reference" content="citation_title=High-level accumulation of recombinant miraculin protein in transgenic tomatoes expressing a synthetic miraculin gene with optimized codon usage terminated by the native miraculin terminator; Plant Cell Rep.;  citation_year=2011;  citation_volume=30;  citation_pages=113-24; " /><meta name="citation_reference" content="citation_title=Mutation bias favors protein folding stability in the evolution of small populations; PLoS Comput. Biol.;  citation_year=2010;  citation_volume=6;  citation_pages=e1000767" /><meta name="citation_reference" content="citation_title=Pack-mutator-like transposable elements (Pack-MULEs) induce directional modification of genes through biased insertion and DNA acquisition; Proc. Natl. Acad. Sci. USA;  citation_year=2011;  citation_volume=108;  citation_pages=1537-42; " /><meta name="citation_reference" content="citation_title=Developmental programming of CpG island methylation profiles in the human genome; Nat. Struct. Mol. Biol.;  citation_year=2009;  citation_volume=16;  citation_pages=564-71; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of major intrinsic proteins in the tree plant Populus trichocarpa: characterization of XIP subfamily of aquaporins from evolutionary perspective; BMC Plant Biol.;  citation_year=2009;  citation_volume=9;  citation_pages=134" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/20/2/135/953305/dss039.pdf" /><meta name="description" content="Abstract.  Citrus, as one of the globally important fruit trees, has been an object of interest for understanding genetics and evolutionary process in fruit cro" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/20/2/135/333376" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/20/2/135/333376" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/20/2/135/333376");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dss039';
    </script>
<title>
	Evaluation of Codon Biology in Citrus and Poncirus trifoliata Based on Genomic Features and Frame Corrected Expressed Sequence Tags | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./333376" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BceBATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQWhtYWQsIFRvdXFlZXIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBvZiBIb3J0aWN1bHR1cmFsIFBsYW50IEJpb2xvZ3kgKE1PRSksIEh1YXpob25nIEFncmljdWx0dXJhbCBVbml2ZXJzaXR5LCBXdWhhbiA0MzAwNzAsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2FibG9rLCBHYXVyYXYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJSZXNlYXJjaCBhbmQgSW5ub3ZhdGlvbiBDZW50ZXIsIEZvbmRhemlvbmUgRWRtdW5kIE1hY2ggKEZFTSksIElzdGl0dXRvIEFncmFyaW8gU2FuIE1pY2hlbGUgKElBU01BKSwgVmlhIE1hY2ggMS4sIFNhbiBNaWNoZWxlIGFsbCdBZGlnZSwgVHJlbnRpbm8gMzgwMTAsIEl0YWx5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGF0YXJpbm92YSwgVGF0aWFuYSBWLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdsYW1vcmdhbiBDb21wdXRhdGlvbmFsIEJpb2xvZ3kgR3JvdXAsIFVuaXZlcnNpdHkgb2YgR2xhbW9yZ2FuLCBQb250eXByaWRkIENGMzcgMURMLCBVSyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ilh1LCBRaWFuZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIEhvcnRpY3VsdHVyYWwgUGxhbnQgQmlvbG9neSAoTU9FKSwgSHVhemhvbmcgQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIFd1aGFuIDQzMDA3MCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJEZW5nLCBYaXUtWGluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgSG9ydGljdWx0dXJhbCBQbGFudCBCaW9sb2d5IChNT0UpLCBIdWF6aG9uZyBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgV3VoYW4gNDMwMDcwLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikd1bywgV2VuLVd1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgSG9ydGljdWx0dXJhbCBQbGFudCBCaW9sb2d5IChNT0UpLCBIdWF6aG9uZyBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgV3VoYW4gNDMwMDcwLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iRXZhbHVhdGlvbiBvZiBDb2RvbiBCaW9sb2d5IGluIENpdHJ1cyBhbmQgUG9uY2lydXMgdHJpZm9saWF0YSBCYXNlZCBvbiBHZW5vbWljIEZlYXR1cmVzIGFuZCBGcmFtZSBDb3JyZWN0ZWQgRXhwcmVzc2VkIFNlcXVlbmNlIFRhZ3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIxMzUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjE1MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzczAzOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDEzLzA0LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGF0dGVybnMgYW5kIGV2b2x1dGlvbiBvZiBudWNsZW90aWRlIGxhbmRzY2FwZXMgaW4gc2VlZCBwbGFudHM7IFBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xMzc5LTk3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lcmFsIHJ1bGVzIGZvciBvcHRpbWFsIGNvZG9uIGNob2ljZTsgUExvUyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAwNTU2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gY2F0YWxvZyB1c2FnZSBpcyBhIGdlbm9tZSBzdHJhdGVneSBtb2R1bGF0ZWQgZm9yIGdlbmUgZXhwcmVzc2l2aXR5OyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5ODE7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTQzLTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIGV4cHJlc3Npb24gYW5kIG1vbGVjdWxhciBldm9sdXRpb247IEN1cnIuIE9waW4uIEdlbmV0LiBEZXYuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NjYwLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZpbmUtdHVuaW5nIHRyYW5zbGF0aW9uIGtpbmV0aWNzIHNlbGVjdGlvbiBhcyB0aGUgZHJpdmluZyBmb3JjZSBvZiBjb2RvbiB1c2FnZSBiaWFzIGluIHRoZSBoZXBhdGl0aXMgQSB2aXJ1cyBjYXBzaWQ7IFBMb1MgUGF0aG9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDA3OTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdmlkZW5jZSBmb3IgaG9yaXpvbnRhbCBnZW5lIHRyYW5zZmVyIGluIEVzY2hlcmljaGlhIGNvbGkgc3BlY2lhdGlvbjsgSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5MTsgIGNpdGF0aW9uX3ZvbHVtZT0yMjI7ICBjaXRhdGlvbl9wYWdlcz04NTEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VW5pdmVyc2FsIGJpYXNlcyBpbiBwcm90ZWluIGNvbXBvc2l0aW9uIG9mIG1vZGVsIHByb2thcnlvdGVzOyBQcm90ZWluczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT02MDsgIGNpdGF0aW9uX3BhZ2VzPTI3LTM1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb24gb2Ygc3lub255bW91cyBjb2RvbiB1c2FnZSBpbiBtZXRhem9hbnM7IEN1cnIuIE9waW4uIEdlbmV0LiBEZXYuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9NjQwLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBzZWxlY3Rpb24tbXV0YXRpb24gZHJpZnQgdGhlb3J5IG9mIHN5bm9ueW1vdXMgY29kb24gdXNhZ2U7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTEyOTsgIGNpdGF0aW9uX3BhZ2VzPTg5Ny05MDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9uIHVzYWdlIGFuZCBnZW5vbWUgZXZvbHV0aW9uOyBDdXJyLiBPcGluLiBHZW5ldC4gRGV2LjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9ODUxLTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2xhdGlvbmFsIHNlbGVjdGlvbiBhbmQgbW9sZWN1bGFyIGV2b2x1dGlvbjsgQ3Vyci4gT3Bpbi4gR2VuZXQuIERldi47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTY4OC05MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBleHByZXNzaXZpdHkgaXMgdGhlIG1haW4gZmFjdG9yIGluIGRpY3RhdGluZyB0aGUgY29kb24gdXNhZ2UgdmFyaWF0aW9uIGFtb25nIHRoZSBnZW5lcyBpbiBQc2V1ZG9tb25hcyBhZXJ1Z2lub3NhOyBHZW5lOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI3MzsgIGNpdGF0aW9uX3BhZ2VzPTYzLTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZWxlY3Rpb24gb24gY29kb24gYmlhczsgQW5udS4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz0yODctOTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJpYXNlZCBnZW5lIGNvbnZlcnNpb24gYWZmZWN0cyBwYXR0ZXJucyBvZiBjb2RvbiB1c2FnZSBhbmQgYW1pbm8gYWNpZCB1c2FnZSBpbiB0aGUgU2FjY2hhcm9teWNlcyBzZW5zdSBzdHJpY3RvIGdyb3VwIG9mIHllYXN0czsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTExNy0yOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGVmZmVjdCBvZiBsaW5rYWdlIG9uIGxpbWl0cyB0byBhcnRpZmljaWFsIHNlbGVjdGlvbjsgR2VuZXQuIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5NjY7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTI2OS05NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVwbGljYXRpb25hbCBhbmQgdHJhbnNjcmlwdGlvbmFsIHNlbGVjdGlvbiBvbiBjb2RvbiB1c2FnZSBpbiBCb3JyZWxpYSBidXJnZG9yZmVyaTsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9OTU7ICBjaXRhdGlvbl9wYWdlcz0xMDY5OC03MDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9uIHVzYWdlIGJldHdlZW4gZ2Vub21lcyBpcyBjb25zdHJhaW5lZCBieSBnZW5vbWUtd2lkZSBtdXRhdGlvbmFsIHByb2Nlc3NlczsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTAxOyAgY2l0YXRpb25fcGFnZXM9MzQ4MC04NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50cmFnZW5pYyBzcGF0aWFsIHBhdHRlcm5zIG9mIGNvZG9uIHVzYWdlIGJpYXMgaW4gcHJva2FyeW90aWMgYW5kIGV1a2FyeW90aWMgZ2Vub21lczsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTY4OyAgY2l0YXRpb25fcGFnZXM9MjI0NS02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3lub255bW91cyBjb2RvbiB1c2FnZSBpbiBFc2NoZXJpY2hpYSBjb2xpOiBzZWxlY3Rpb24gZm9yIHRyYW5zbGF0aW9uYWwgYWNjdXJhY3k7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0zNzQtODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9uLXVzYWdlIGJpYXMgdmVyc3VzIGdlbmUgY29udmVyc2lvbiBpbiB0aGUgZXZvbHV0aW9uIG9mIHllYXN0IGR1cGxpY2F0ZSBnZW5lczsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTAzOyAgY2l0YXRpb25fcGFnZXM9MTQ0MTItMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN5bm9ueW1vdXMgY29kb24gdXNhZ2UgaW4gZW52aXJvbm1lbnRhbCBDaGxhbXlkaWEgVVdFMjUgcmVmbGVjdHMgYW4gZXZvbHV0aW9uYWwgZGl2ZXJnZW5jZSBmcm9tIHBhdGhvZ2VuaWMgQ2hsYW15ZGlhZTsgR2VuZTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNjg7ICBjaXRhdGlvbl9wYWdlcz0xMTctMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZvcmNlcyB0aGF0IGluZmx1ZW5jZSB0aGUgZXZvbHV0aW9uIG9mIGNvZG9uIGJpYXM7IFBoaWxvcy4gVHJhbnMuIFIuIFNvYy4gTG9uZC4gQiBCaW9sLiBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTM2NTsgIGNpdGF0aW9uX3BhZ2VzPTEyMDMtMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSDigJhlZmZlY3RpdmUgbnVtYmVyIG9mIGNvZG9uc+KAmSB1c2VkIGluIGEgZ2VuZTsgR2VuZTsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT04NzsgIGNpdGF0aW9uX3BhZ2VzPTIzLTI5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1RdWFudGl0YXRpdmUgcmVsYXRpb25zaGlwIGJldHdlZW4gc3lub255bW91cyBiaWFzIGFuZCBHQyBjb21wb3NpdGlvbiBhY3Jvc3MgdW5pY2VsbHVsYXIgZ2Vub21lczsgQk1DIEV2b2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz0xOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBkcmFmdCBnZW5vbWUgb2Ygc3dlZXQgb3JhbmdlIChDaXRydXMgc2luZW5zaXMpOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q0FQMzogYSBETkEgc2VxdWVuY2UgYXNzZW1ibHkgcHJvZ3JhbTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTg2OC03NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnJhbWVEUDogc2Vuc2l0aXZlIHBlcHRpZGUgZGV0ZWN0aW9uIG9uIG5vaXN5IG1hdHVyZWQgc2VxdWVuY2VzOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTY3MC03MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnJhbWVEOiBhIGZsZXhpYmxlIHByb2dyYW0gZm9yIHF1YWxpdHkgY2hlY2sgYW5kIGdlbmUgcHJlZGljdGlvbiBpbiBwcm9rYXJ5b3RpYyBnZW5vbWVzIGFuZCBub2lzeSBtYXR1cmVkIGV1a2FyeW90aWMgc2VxdWVuY2VzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0zNzMtODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdhcHBlZCBCTEFTVCBhbmQgUFNJLUJMQVNUOiBhIG5ldyBnZW5lcmF0aW9uIG9mIHByb3RlaW4gZGF0YWJhc2Ugc2VhcmNoIHByb2dyYW1zOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0zMzg5LTQwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUFjYWRlbWljIFByZXNzLCBMb25kb247IFRoZW9yeSBhbmQgQXBwbGljYXRpb24gb2YgQ29ycmVzcG9uZGVuY2UgQW5hbHlzaXM7ICBjaXRhdGlvbl95ZWFyPTE5ODQ7ICBjaXRhdGlvbl9wYWdlcz0yMjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBldm9sdXRpb25hcnkgcGVyc3BlY3RpdmUgb24gc3lub255bW91cyBjb2RvbiB1c2FnZSBpbiB1bmljZWxsdWxhciBvcmdhbmlzbXM7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODY7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0yOC0zODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzaW1wbGUgbWV0aG9kIGZvciBkaXNwbGF5aW5nIHRoZSBoeWRyb3BhdGhpYyBjaGFyYWN0ZXIgb2YgYSBwcm90ZWluOyBKLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTgyOyAgY2l0YXRpb25fdm9sdW1lPTE1NzsgIGNpdGF0aW9uX3BhZ2VzPTEwNS0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gdXNhZ2UgYW5kIHRSTkEgY29udGVudCBpbiB1bmljZWxsdWxhciBhbmQgbXVsdGljZWxsdWxhciBvcmdhbmlzbXM7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODU7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTEzLTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeW5vbnltb3VzIGNvZG9uIHVzYWdlLCBHQzMgYW5kIGV2b2x1dGlvbmFyeSBwYXR0ZXJucyBhY3Jvc3MgcGxhc3RvbWVzIG9mIHRocmVlIHBvb2lkIG1vZGVsIHNwZWNpZXMg4oCTIGVtZXJnaW5nIGdyYXNzIGdlbm9tZSBtb2RlbHMgZm9yIG1vbm9jb3RzOyBNb2wuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTQ5OyAgY2l0YXRpb25fcGFnZXM9MTE2LTI4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm90ZWlub3J0aG86IGRldGVjdGlvbiBvZiAoY28tKSBvcnRob2xvZ3MgaW4gbGFyZ2Utc2NhbGUgYW5hbHlzaXM7IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTEyNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU51Y2xlb3RpZGUgY29tcG9zaXRpb24gYmlhcyBhZmZlY3RzIGFtaW5vIGFjaWQgY29udGVudCBpbiBwcm90ZWlucyBjb2RlZCBieSBhbmltYWwgbWl0b2Nob25kcmlhOyBKLiBNb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTQ0OyAgY2l0YXRpb25fcGFnZXM9MjgyLTg4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OdWNsZW90aWRlIGJpYXMgY2F1c2VzIGEgZ2Vub21ld2lkZSBiaWFzIGluIHRoZSBhbWlubyBhY2lkIGNvbXBvc2l0aW9uIG9mIHByb3RlaW5zOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MTU4MS04ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgY29kb24gdXNhZ2UgaW4gQ2l0cnVzOyBKLiBGcnVpdCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9NDc5LTg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PbiB0aGUgZXZvbHV0aW9uIG9mIGluZm9ybWF0aW9uYWwgbWFjcm9tb2xlY3VsZXM7IGNpdGF0aW9uX3B1Ymxpc2hlcj1BY2FkZW1pYyBQcmVzcywgTmV3IFlvcms7IEV2b2x2aW5nIEdlbmVzIGFuZCBQcm90ZWluczsgIGNpdGF0aW9uX3llYXI9MTk2NDsgIGNpdGF0aW9uX3BhZ2VzPTQ3OS05NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIEcrIEMgY29udGVudCBvZiB0aGUgdGhpcmQgY29kb24gcG9zaXRpb24gYW5kIGNvZG9uIHVzYWdlIGJpYXNlcyBvZiBodW1hbiBnZW5lczsgR2VuZTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNjE7ICBjaXRhdGlvbl9wYWdlcz01My02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlyZWN0aW9uYWwgbXV0YXRpb24gcHJlc3N1cmUgYW5kIG5ldXRyYWwgbW9sZWN1bGFyIGV2b2x1dGlvbjsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5ODg7ICBjaXRhdGlvbl92b2x1bWU9ODU7ICBjaXRhdGlvbl9wYWdlcz0yNjUzLTU3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SYXBpZCBkaXZlcmdlbmNlIG9mIGNvZG9uIHVzYWdlIHBhdHRlcm5zIHdpdGhpbiB0aGUgcmljZSBnZW5vbWU7IEJNQyBFdm9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25faXNzdWU9U3VwcGwuIDE7ICBjaXRhdGlvbl9wYWdlcz1TNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN5bm9ueW1vdXMgY29kb24gdXNhZ2UgaW4gTGFjdG9jb2NjdXMgbGFjdGlzOiBtdXRhdGlvbmFsIGJpYXMgdmVyc3VzIHRyYW5zbGF0aW9uYWwgc2VsZWN0aW9uOyBKLiBCaW9tb2wuIFN0cnVjdC4gRHluLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTUyNy0zNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gdXNhZ2UgaW4gdGhlIEcrQy1yaWNoIFN0cmVwdG9teWNlcyBnZW5vbWU7IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9MTEzOyAgY2l0YXRpb25fcGFnZXM9NTUtNjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdDMyBiaW9sb2d5IGluIGNvcm4sIHJpY2UsIHNvcmdodW0gYW5kIG90aGVyIGdyYXNzZXM7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTMwOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9uIHVzYWdlIGluIEVzY2hlcmljaGlhIGNvbGksIEJhY2lsbHVzIHN1YnRpbGlzLCBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWUsIFNjaGl6b3NhY2NoYXJvbXljZXMgcG9tYmUsIERyb3NvcGhpbGEgbWVsYW5vZ2FzdGVyIGFuZCBIb21vIHNhcGllbnM7IGEgcmV2aWV3IG9mIHRoZSBjb25zaWRlcmFibGUgd2l0aGluLXNwZWNpZXMgZGl2ZXJzaXR5OyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5ODg7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz04MjA3LTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiBwcmVmZXJlbmNlcyBpbiBmcmVlLWxpdmluZyBtaWNyb29yZ2FuaXNtczsgTWljcm9iaW9sLiBSZXYuOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTU0OyAgY2l0YXRpb25fcGFnZXM9MTk4LTIxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFsYW5jZWQgY29kb24gdXNhZ2Ugb3B0aW1pemVzIGV1a2FyeW90aWMgdHJhbnNsYXRpb25hbCBlZmZpY2llbmN5OyBQTG9TIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDI2MDMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BYnNvbHV0ZSBpbiB2aXZvIHRyYW5zbGF0aW9uIHJhdGVzIG9mIGluZGl2aWR1YWwgY29kb25zIGluIEVzY2hlcmljaGlhIGNvbGk6IHRoZSB0d28gZ2x1dGFtaWMgYWNpZCBjb2RvbnMgR0FBIGFuZCBHQUcgYXJlIHRyYW5zbGF0ZWQgd2l0aCBhIHRocmVlZm9sZCBkaWZmZXJlbmNlIGluIHJhdGU7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9MjIyOyAgY2l0YXRpb25fcGFnZXM9MjY1LTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2Rpbmctc2VxdWVuY2UgZGV0ZXJtaW5hbnRzIG9mIGdlbmUgZXhwcmVzc2lvbiBpbiBFc2NoZXJpY2hpYSBjb2xpOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTMyNDsgIGNpdGF0aW9uX3BhZ2VzPTI1NS01ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHNpZ25pZmljYW5jZSBvZiBkaWdpdGFsIGdlbmUgZXhwcmVzc2lvbiBwcm9maWxlczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTk4Ni05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9hcnJheSBhbmQgRVNUIGRhdGFiYXNlIGVzdGltYXRlcyBvZiBtUk5BIGV4cHJlc3Npb24gbGV2ZWxzIGRpZmZlcjogdGhlIHByb3RlaW4gbGVuZ3RoIHZlcnN1cyBleHByZXNzaW9uIGN1cnZlIGZvciBDLiBlbGVnYW5zOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTMwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGF0dGVybnMgb2YgbW9sZWN1bGFyIGV2b2x1dGlvbiBpbiBDYWVub3JoYWJkaXRpcyBwcmVjbHVkZSBhbmNpZW50IG9yaWdpbnMgb2Ygc2VsZmluZzsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTc4OyAgY2l0YXRpb25fcGFnZXM9MjA5My0xMDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBldm9sdXRpb24gb2Ygc3lub255bW91cyBjb2RvbiB1c2FnZSBpbiBQb3B1bHVzOyBCTUMgRXZvbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTMwNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbiBvZiBzeW5vbnltb3VzIGNvZG9uIHVzYWdlIGluIE5ldXJvc3BvcmEgdGV0cmFzcGVybWEgYW5kIE5ldXJvc3BvcmEgZGlzY3JldGU7IEdlbm9tZSBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MzMyLTQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHNpbXBsZSBtb2RlbCBiYXNlZCBvbiBtdXRhdGlvbiBhbmQgc2VsZWN0aW9uIGV4cGxhaW5zIHRyZW5kcyBpbiBjb2RvbiBhbmQgYW1pbm8gYWNpZCB1c2FnZSBhbmQgR0MgY29tcG9zaXRpb24gd2l0aGluIGFuZCBhY3Jvc3MgZ2Vub21lczsgR2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz1yZXNlYXJjaDAwMTAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiB1c2FnZSBwYXR0ZXJucyBpbiBOZW1hdG9kYTogYW5hbHlzaXMgYmFzZWQgb24gb3ZlciAyNSBtaWxsaW9uIGNvZG9ucyBpbiB0aGlydHktdHdvIHNwZWNpZXM7IEdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9Ujc1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBnZW5lcmFsIG1vZGVsIG9mIGNvZG9uIGJpYXMgZHVlIHRvIEdDIG11dGF0aW9uYWwgYmlhczsgUExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPWUxMzQzMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2gtbGV2ZWwgYWNjdW11bGF0aW9uIG9mIHJlY29tYmluYW50IG1pcmFjdWxpbiBwcm90ZWluIGluIHRyYW5zZ2VuaWMgdG9tYXRvZXMgZXhwcmVzc2luZyBhIHN5bnRoZXRpYyBtaXJhY3VsaW4gZ2VuZSB3aXRoIG9wdGltaXplZCBjb2RvbiB1c2FnZSB0ZXJtaW5hdGVkIGJ5IHRoZSBuYXRpdmUgbWlyYWN1bGluIHRlcm1pbmF0b3I7IFBsYW50IENlbGwgUmVwLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTExMy0yNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXV0YXRpb24gYmlhcyBmYXZvcnMgcHJvdGVpbiBmb2xkaW5nIHN0YWJpbGl0eSBpbiB0aGUgZXZvbHV0aW9uIG9mIHNtYWxsIHBvcHVsYXRpb25zOyBQTG9TIENvbXB1dC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAwNzY3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGFjay1tdXRhdG9yLWxpa2UgdHJhbnNwb3NhYmxlIGVsZW1lbnRzIChQYWNrLU1VTEVzKSBpbmR1Y2UgZGlyZWN0aW9uYWwgbW9kaWZpY2F0aW9uIG9mIGdlbmVzIHRocm91Z2ggYmlhc2VkIGluc2VydGlvbiBhbmQgRE5BIGFjcXVpc2l0aW9uOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDg7ICBjaXRhdGlvbl9wYWdlcz0xNTM3LTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudGFsIHByb2dyYW1taW5nIG9mIENwRyBpc2xhbmQgbWV0aHlsYXRpb24gcHJvZmlsZXMgaW4gdGhlIGh1bWFuIGdlbm9tZTsgTmF0LiBTdHJ1Y3QuIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz01NjQtNzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFuYWx5c2lzIG9mIG1ham9yIGludHJpbnNpYyBwcm90ZWlucyBpbiB0aGUgdHJlZSBwbGFudCBQb3B1bHVzIHRyaWNob2NhcnBhOiBjaGFyYWN0ZXJpemF0aW9uIG9mIFhJUCBzdWJmYW1pbHkgb2YgYXF1YXBvcmlucyBmcm9tIGV2b2x1dGlvbmFyeSBwZXJzcGVjdGl2ZTsgQk1DIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0xMzQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjAvMi8xMzUvOTUzMzA1L2RzczAzOS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgQ2l0cnVzLCBhcyBvbmUgb2YgdGhlIGdsb2JhbGx5IGltcG9ydGFudCBmcnVpdCB0cmVlcywgaGFzIGJlZW4gYW4gb2JqZWN0IG9mIGludGVyZXN0IGZvciB1bmRlcnN0YW5kaW5nIGdlbmV0aWNzIGFuZCBldm9sdXRpb25hcnkgcHJvY2VzcyBpbiBmcnVpdCBjcm8iIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTMzMzM3NiZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGRe7aeRGu9p5NH5ZFImXdcKQn+82A==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f2%2f135%2f333376" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f2%2f135%2f333376" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=12698" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/20/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/2/1/m_cover.gif?Expires=2147483647&amp;Signature=Yq7dbhq6hb-Iyo5ewEGt-Y1T99brR4Xh8FvVnCI2Bg1SIlPGztlGCk-Bm0A9A72SaqDKcetVlEQehNtxyAGT~T55ZFFD19A3hKk~A1XbnQ-JfXYKj2-gYHz-VirOlvLe-8XkC6qvT4h2PniinfIJH6d85JsIf3rhRRtN086VVGvi2qNVR0THPJOeMpQvElVQEEN-4bpo3wEYmq80uwX58y5ScJ0pcJt7sV40xw1Fd6mp92v-su9vXuGT~IIVjyHRDyuiGt~ZiUzcXtor6WX4MT1zcCCf5ySC3Wm7DfaZruIvVsbtJxHUYCwePrgmcTq1vhdeOWHam2svM-jBZyoDoA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/20/2">
                        <div class="volume trailing-comma">Volume 20</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2013        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="4607970">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4607970">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4607972">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4607972">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4607978">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4607978">2. Materials and methodology</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4607995">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4607995">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4608030">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4608030">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4608032">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4608032">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4608034">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4608034">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4608036">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4608036">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/20/2/127/333239">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/20/2/151/333539">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Evaluation of Codon Biology in <em>Citrus</em> and <em>Poncirus trifoliata</em> Based on Genomic Features and Frame Corrected Expressed Sequence Tags
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Touqeer Ahmad</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Key Laboratory of Horticultural Plant Biology (MOE)</div>, <div class="insititution">Huazhong Agricultural University</div>, <div class="addr-line">Wuhan 430070</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Touqeer+Ahmad">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ahmad T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ahmad T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Touqeer Ahmad</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Gaurav Sablok</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Research and Innovation Center, Fondazione Edmund Mach (FEM), Istituto Agrario San Michele (IASMA)</div>, <div class="addr-line">Via Mach 1., San Michele all'Adige, Trentino 38010</div>, <div class="country">Italy</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Gaurav+Sablok">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sablok G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sablok G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Gaurav Sablok</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tatiana V. Tatarinova</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Glamorgan Computational Biology Group, University of Glamorgan, Pontypridd CF37 1DL</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tatiana+V.+Tatarinova">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tatarinova T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tatarinova T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tatiana V. Tatarinova</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Qiang Xu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Key Laboratory of Horticultural Plant Biology (MOE)</div>, <div class="insititution">Huazhong Agricultural University</div>, <div class="addr-line">Wuhan 430070</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Qiang+Xu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Xu Q">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Xu Q&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Qiang Xu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xiu-Xin Deng</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Key Laboratory of Horticultural Plant Biology (MOE)</div>, <div class="insititution">Huazhong Agricultural University</div>, <div class="addr-line">Wuhan 430070</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xiu-Xin+Deng">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Deng X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Deng X&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Xiu-Xin Deng</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wen-Wu Guo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Key Laboratory of Horticultural Plant Biology (MOE)</div>, <div class="insititution">Huazhong Agricultural University</div>, <div class="addr-line">Wuhan 430070</div>, <div class="country">China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86 27-87281543. Fax. +86 27-87280016. Email: <a href="mailto:guoww@mail.hzau.edu.cn" target="_blank">guoww@mail.hzau.edu.cn</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wen-Wu+Guo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Guo W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Guo W&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Wen-Wu Guo</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 20, Issue 2, 1 April 2013, Pages 135–150, <a href='https://doi.org/10.1093/dnares/dss039'>https://doi.org/10.1093/dnares/dss039</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">11 January 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">09 August 2012</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">27 November 2012</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="333376" href="/dnaresearch/article-pdf/20/2/135/953305/dss039.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Touqeer Ahmad, Gaurav Sablok, Tatiana V. Tatarinova, Qiang Xu, Xiu-Xin Deng, Wen-Wu Guo;  Evaluation of Codon Biology in <em>Citrus</em> and <em>Poncirus trifoliata</em> Based on Genomic Features and Frame Corrected Expressed Sequence Tags, <em>DNA Research</em>, Volume 20, Issue 2, 1 April 2013, Pages 135–150, <a href='https://doi.org/10.1093/dnares/dss039'>https://doi.org/10.1093/dnares/dss039</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/333376?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/333376?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/333376?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/333376?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/333376?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/333376?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/333376?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Evaluation%20of%20Codon%20Biology%20in%20%26lt%3Bem%26gt%3BCitrus%26lt%3B%2Fem%26gt%3B%20and%20%26lt%3Bem%26gt%3BPoncirus%20trifoliata%26lt%3B%2Fem%26gt%3B%20Based%20on%20Genomic%20Features%20and%20Frame%20Corrected%20Expressed%20Sequence%20Tags&amp;publicationDate=2013-01-11&amp;volumeNum=20&amp;issueNum=2&amp;author=Ahmad%2C%20Touqeer%3B%20Sablok%2C%20Gaurav&amp;startPage=135&amp;endPage=150&amp;contentId=10.1093%2Fdnares%2Fdss039&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f2%2f135%2f333376" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=4607970 id="4607970" class="abstract-title" >Abstract</h2>
<section class="abstract"><p><em>Citrus</em>, as one of the globally important fruit trees, has been an object of interest for understanding genetics and evolutionary process in fruit crops. Meta-analyses of 19 <em>Citrus</em> species, including 4 globally and economically important <em>Citrus sinensis</em>, <em>Citrus clementina</em>, <em>Citrus reticulata</em>, and 1 <em>Citrus</em> relative <em>Poncirus trifoliata</em>, were performed. We observed that codons ending with A- or T- at the wobble position were preferred in contrast to C- or G- ending codons, indicating a close association with AT richness of <em>Citrus</em> species and <em>P. trifoliata</em>. The present study postulates a large repertoire of a set of optimal codons for the <em>Citrus</em> genus and <em>P. trifoliata</em> and demonstrates that GCT and GGT are evolutionary conserved optimal codons. Our observation suggested that mutational bias is the dominating force in shaping the codon usage bias (CUB) in <em>Citrus</em> and <em>P. trifoliata</em>. Correspondence analysis (COA) revealed that the principal axis [axis 1; COA/relative synonymous codon usage (RSCU)] contributes only a minor portion (∼10.96%) of the recorded variance. In all analysed species, except <em>P. trifoliata</em>, Gravy and aromaticity played minor roles in resolving CUB. Compositional constraints were found to be strongly associated with the amino acid signatures in <em>Citrus</em> species and <em>P. trifoliata</em>. Our present analysis postulates compositional constraints in <em>Citrus</em> species and <em>P. trifoliata</em> and plausible role of the stress with GC<sub>3</sub> and coevolution pattern of amino acid.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="Citrus"><em>Citrus</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="coevolution">coevolution</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;mutational bias&quot;">mutational bias</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Poncirus trifoliata&quot;"><em>Poncirus trifoliata</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="stress">stress</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;GC3 biology&quot;">GC<sub>3</sub> biology</a></div>                            <h2 scrollto-destination=4607972 id="4607972" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Genome composition (such as GC- and AT-content) and subsequent balance of codon usage and eukaryotic translation machinery play an important role in the evolution of the nucleotides at the wobble position.<sup><a reveal-id="DSS039C1" data-open="DSS039C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Degeneracy in biased codon usage explains the concept behind the usage of same amino acids with multiple synonymous codons except methionine (Met) and tryptophan (Trp).<sup><a reveal-id="DSS039C2" data-open="DSS039C2" class="link link-ref link-reveal xref-bibr">2</a></sup> This genome composition bias leads to the usage of some codons at a higher frequency as compared to the synonymous codons for encoding the particular amino acid. These differences in the usage of the synonymous codons have also been one of the factors for the evolution of proteome diversity and can help us to understand the evolution of those proteins that have structural differences in spite of being conserved at the sequence level.<sup><a reveal-id="DSS039C3  DSS039C7" data-open="DSS039C3  DSS039C7" class="link link-ref link-reveal xref-bibr">3–7</a></sup></p><p>Two major paradigms of codon usage were proposed as plausible answers to clarify the non-randomness of codon usage at intra- and inter-species levels: (i) natural selection that is expected to yield a correlation with codon bias in highly expressed genes,<sup><a reveal-id="DSS039C8" data-open="DSS039C8" class="link link-ref link-reveal xref-bibr">8</a></sup> rapidly regulated and variably expressed genes, and (ii) neutral processes, such as mutational biases (MBs), where some mutations occur more often than others across the genome of an organism because of local variations in the base composition. Several lines of evidence support the argument that the usage of synonymous codons with unequal frequencies in both prokaryotic and eukaryotic genes is the result of a complex balance between MB and/or natural selection and genetic drift.<sup><a reveal-id="DSS039C9  DSS039C12" data-open="DSS039C9  DSS039C12" class="link link-ref link-reveal xref-bibr">9–12</a></sup> It has been suggested that the codon bias could be positively selected because of a more efficient and accurate translation, and favoured codons may correspond to the most highly expressed genes.<sup><a reveal-id="DSS039C2" data-open="DSS039C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSS039C13" data-open="DSS039C13" class="link link-ref link-reveal xref-bibr">13</a></sup> In addition to neutral and selection processes, GC-biased gene conversion, which depends on the local recombination rate, is an important factor in shaping codon and amino acid usage.<sup><a reveal-id="DSS039C14" data-open="DSS039C14" class="link link-ref link-reveal xref-bibr">14</a></sup></p><p>To date, wide variations have been observed in codon usage patterns in many organisms and have provided clues to understand the evolution of genes and gene families. The factors that potentially affect biased usage of codons are MB that correlates the codon usage bias (CUB) with the genomic GC content, Hill−Robertson effect explaining the interference of selection of one locus with another locus, translational selection, and a cumulative effect of replication and translational selection in shaping the codon usage across the genes of several bacterial species.<sup><a reveal-id="DSS039C15  DSS039C19" data-open="DSS039C15  DSS039C19" class="link link-ref link-reveal xref-bibr">15–19</a></sup> It has been shown that CUB and protein functional conservation play a major role for the decelerated evolution of whole genome duplication in <em>Saccharomyces cerevisiae</em>.<sup><a reveal-id="DSS039C20" data-open="DSS039C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Several other factors that might affect the codon usage are amino acid conservation and hydrophobicity,<sup><a reveal-id="DSS039C21" data-open="DSS039C21" class="link link-ref link-reveal xref-bibr">21</a></sup> gene expression,<sup><a reveal-id="DSS039C22" data-open="DSS039C22" class="link link-ref link-reveal xref-bibr">22</a></sup> mRNA folding stability, codon−anticodon interaction, and gene length.<sup><a reveal-id="DSS039C22" data-open="DSS039C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p><p><em>Citrus</em> is a diploid genus of the Rutaceae family, whose cultivated forms are important for human diet with more than 122 MT of annual world fruit production. We have systematically analysed codon usage patterns and genomic heterogeneity using a meta-analyses approach that involves multivariate tools and codon usage indices such as relative synonymous codon usage (RSCU) and effective number of codons (Nc) in the studied gene sets.<sup><a reveal-id="DSS039C23" data-open="DSS039C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSS039C24" data-open="DSS039C24" class="link link-ref link-reveal xref-bibr">24</a></sup> In present study, we inferred global pattern of CUB, mutational pressures, and association of GC<sub>3</sub> with potential stress events. We have collectively analysed horticulturally important <em>Citrus</em> species that includes recently sequenced double haploid <em>Citrus sinensis</em> genome<sup><a reveal-id="DSS039C25" data-open="DSS039C25" class="link link-ref link-reveal xref-bibr">25</a></sup> and 18 additional <em>Citrus</em> species with expressed sequence tags (ESTs) counts of more than 1000 per species. To make this analysis comparative, we have also included <em>Poncirus trifoliata</em> that is considered to be a distant relative of <em>Citrus</em> genus<em>.</em> These species show wide variation in traits such as cold and drought tolerance and are of commercial importance. We have restricted our analyses to <em>Citrus</em> genera to develop resource information for <em>Citrus</em> species.</p><p>Our study demonstrated that CUB in <em>Citrus</em> species and <em>P. trifoliata</em> is biased towards AT richness, and a relatively higher occurrence of A- and T-ending codons was observed. We found several interesting and diverse patterns of optimal codons, and it was observed that two optimal codons coding for Alanine and Glycine were evolutionary conserved between the <em>Citrus</em> species and <em>P. trifoliata</em>. To the best of our knowledge, our analysis presents the first complete report on the identification of optimal codons across the entire <em>Citrus</em> genus and <em>P. trifoliata</em>, which could serve as the potential source for developing transgenic <em>Citrus</em> cultivars using codon optimization. GC<sub>3</sub> and evolution were correlated using Hamming distance parameter to identify suggestive evolutionary pairs of genes. Co-orthologous genes matrix identified using the alignment ratio suggested close association of <em>Citrus clementina</em>, <em>Citrus sinensis</em>, <em>and Citrus reticulata</em>, as they belong to the same phylogenetic clade. We further demonstrated that nucleotide bias has a genome-wide influence on amino acid composition of <em>Citrus</em> species and <em>P. trifoliata</em>.</p>                            <h2 scrollto-destination=4607978 id="4607978" class="section-title" data-legacy-id=s2>2. Materials and methodology</h2>
                            <h3 scrollto-destination=4607979 id="4607979" class="section-title" data-legacy-id=s2a>2.1. Sequence information and processing</h3>
<p>Our dataset consists of genome-predicted coding regions and ESTs. All the genome-predicted coding sequences of <em>C. sinensis</em> were retrieved from recently sequenced <em>Citrus</em> genome.<sup><a reveal-id="DSS039C25" data-open="DSS039C25" class="link link-ref link-reveal xref-bibr">25</a></sup> In case of other <em>Citrus</em> species, ESTs were downloaded from the National Center for Biotechnology Information (NCBI) EST repository (NCBI; <a class="link link-uri" href="http://www.ncbi.nlm.nih.gov">http://www.ncbi.nlm.nih.gov</a>). In addition, we downloaded putative unique transcripts for all studied species from the PlantGDB database (<a class="link link-uri" href="http://www.plantgdb.org">http://www.plantgdb.org</a>). A detailed description of the data used is shown in Table <a reveal-id="DSS039TB1" data-open="DSS039TB1" class="link link-reveal link-table xref-fig">1</a>. In case of ESTs, the ESTs were first clustered into contigs and singletons using CAP3 with default parameters.<sup><a reveal-id="DSS039C26" data-open="DSS039C26" class="link link-ref link-reveal xref-bibr">26</a></sup> A minimum match percentage cutoff of 95% for 40 overlapping bases was used to assign 2 sequences to a cluster. </p>                        <a id="4607981" scrollto-destination="4607981"></a>
<div content-id="DSS039TB1" class="table-modal"><div class="table-wrap"><div id="DSS039TB1" class="table-wrap-title" data-id="DSS039TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Genomic composition of coding region of <em>Citrus</em> species and <em>P. trifoliata</em> at GC, GC<sub>1</sub>, GC<sub>2</sub>, GC<sub>3</sub>, and GC<sub>3<em>s</em></sub></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>No. </th><th><em>Citrus</em> species </th><th>Genes/EST<sup>a</sup> </th><th>Unigenes count </th><th>Genes<sup>b</sup> </th><th>GC </th><th>GC<sub>1</sub> </th><th>GC<sub>2</sub> </th><th>GC<sub>3</sub> </th><th>GC<sub>3<em>s</em></sub> </th></tr></thead><tbody><tr><td>1 </td><td><em>C. sinensis</em> </td><td>44 275 </td><td>– </td><td>41362 </td><td>43.92 </td><td>50.4 </td><td>40.33 </td><td>41.03 </td><td>38.8 </td></tr><tr><td>2 </td><td><em>C. aurantifolia</em> </td><td>8219 </td><td>7550 </td><td>4715 </td><td>47.78 </td><td>52.96 </td><td>43.87 </td><td>46.52 </td><td>44.73 </td></tr><tr><td>3 </td><td><em>C. aurantium</em> </td><td>14 584 </td><td>11 952 </td><td>6787 </td><td>46.93 </td><td>52.47 </td><td>42.59 </td><td>45.72 </td><td>43.94 </td></tr><tr><td>4 </td><td><em>C. clementina</em> </td><td>118 365 </td><td>51 591 </td><td>33 765 </td><td>47.04 </td><td>51.84 </td><td>42.51 </td><td>46.76 </td><td>44.99 </td></tr><tr><td>5 </td><td><em>C. clementina</em> × <em>C. tangerina</em> </td><td>1843 </td><td>1283 </td><td>677 </td><td>44.66 </td><td>51.67 </td><td>39.77 </td><td>42.55 </td><td>40.47 </td></tr><tr><td>6 </td><td><em>C. jambhiri</em> </td><td>1017 </td><td>858 </td><td>701 </td><td>45.86 </td><td>51.79 </td><td>41.13 </td><td>44.66 </td><td>42.67 </td></tr><tr><td>7 </td><td><em>C. japonica</em> var<em>. margarita</em> </td><td>2924 </td><td>1628 </td><td>588 </td><td>46.11 </td><td>52.67 </td><td>41.42 </td><td>44.25 </td><td>42.35 </td></tr><tr><td>8 </td><td><em>C. limettioides</em> </td><td>8188 </td><td>7933 </td><td>3964 </td><td>46.47 </td><td>50.76 </td><td>42.75 </td><td>45.9 </td><td>44.25 </td></tr><tr><td>9 </td><td><em>C. limonia</em> </td><td>11 045 </td><td>9761 </td><td>4256 </td><td>45.82 </td><td>51.13 </td><td>41.37 </td><td>44.96 </td><td>43.17 </td></tr><tr><td>10 </td><td><em>C. medica</em> </td><td>1115 </td><td>896 </td><td>711 </td><td>45.65 </td><td>51.91 </td><td>40.9 </td><td>44.13 </td><td>42.15 </td></tr><tr><td>11 </td><td><em>C. reshni</em> </td><td>5768 </td><td>3735 </td><td>2644 </td><td>45.2 </td><td>51.51 </td><td>40.57 </td><td>43.52 </td><td>41.5 </td></tr><tr><td>12 </td><td><em>C. reticulata</em> </td><td>55 980 </td><td>46 170 </td><td>30 816 </td><td>46.85 </td><td>52.3 </td><td>42.77 </td><td>45.48 </td><td>43.62 </td></tr><tr><td>13 </td><td><em>C. reticulata</em> × <em>C. temple</em> </td><td>5823 </td><td>3685 </td><td>2253 </td><td>46.48 </td><td>52.41 </td><td>41.64 </td><td>45.41 </td><td>43.41 </td></tr><tr><td>14 </td><td><em>C. sinensis</em> × <em>P. trifoliata</em> </td><td>1837 </td><td>1522 </td><td>992 </td><td>46.07 </td><td>51.67 </td><td>41.41 </td><td>45.15 </td><td>43.23 </td></tr><tr><td>15 </td><td><em>C. sunki</em> </td><td>5216 </td><td>4688 </td><td>1746 </td><td>47.58 </td><td>51.51 </td><td>43.58 </td><td>47.64 </td><td>46.17 </td></tr><tr><td>16 </td><td><em>P. trifoliata</em> </td><td>62 695 </td><td>35 740 </td><td>19 445 </td><td>46.78 </td><td>52.33 </td><td>42.67 </td><td>45.35 </td><td>43.52 </td></tr><tr><td>17 </td><td><em>C. unshiu</em> </td><td>19 072 </td><td>9289 </td><td>4592 </td><td>45.67 </td><td>51.72 </td><td>41.14 </td><td>44.15 </td><td>42.21 </td></tr><tr><td>18 </td><td><em>C. paradisi</em> </td><td>8039 </td><td>4621 </td><td>2517 </td><td>45.32 </td><td>52.19 </td><td>40.51 </td><td>43.27 </td><td>41.3 </td></tr><tr><td>19 </td><td><em>C. paradisi</em> × <em>P. trifoliata</em> </td><td>7954 </td><td>3335 </td><td>2596 </td><td>45.23 </td><td>51.96 </td><td>40.77 </td><td>42.98 </td><td>40.96 </td></tr></tbody></table></div><div content-id="DSS039TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>No. </th><th><em>Citrus</em> species </th><th>Genes/EST<sup>a</sup> </th><th>Unigenes count </th><th>Genes<sup>b</sup> </th><th>GC </th><th>GC<sub>1</sub> </th><th>GC<sub>2</sub> </th><th>GC<sub>3</sub> </th><th>GC<sub>3<em>s</em></sub> </th></tr></thead><tbody><tr><td>1 </td><td><em>C. sinensis</em> </td><td>44 275 </td><td>– </td><td>41362 </td><td>43.92 </td><td>50.4 </td><td>40.33 </td><td>41.03 </td><td>38.8 </td></tr><tr><td>2 </td><td><em>C. aurantifolia</em> </td><td>8219 </td><td>7550 </td><td>4715 </td><td>47.78 </td><td>52.96 </td><td>43.87 </td><td>46.52 </td><td>44.73 </td></tr><tr><td>3 </td><td><em>C. aurantium</em> </td><td>14 584 </td><td>11 952 </td><td>6787 </td><td>46.93 </td><td>52.47 </td><td>42.59 </td><td>45.72 </td><td>43.94 </td></tr><tr><td>4 </td><td><em>C. clementina</em> </td><td>118 365 </td><td>51 591 </td><td>33 765 </td><td>47.04 </td><td>51.84 </td><td>42.51 </td><td>46.76 </td><td>44.99 </td></tr><tr><td>5 </td><td><em>C. clementina</em> × <em>C. tangerina</em> </td><td>1843 </td><td>1283 </td><td>677 </td><td>44.66 </td><td>51.67 </td><td>39.77 </td><td>42.55 </td><td>40.47 </td></tr><tr><td>6 </td><td><em>C. jambhiri</em> </td><td>1017 </td><td>858 </td><td>701 </td><td>45.86 </td><td>51.79 </td><td>41.13 </td><td>44.66 </td><td>42.67 </td></tr><tr><td>7 </td><td><em>C. japonica</em> var<em>. margarita</em> </td><td>2924 </td><td>1628 </td><td>588 </td><td>46.11 </td><td>52.67 </td><td>41.42 </td><td>44.25 </td><td>42.35 </td></tr><tr><td>8 </td><td><em>C. limettioides</em> </td><td>8188 </td><td>7933 </td><td>3964 </td><td>46.47 </td><td>50.76 </td><td>42.75 </td><td>45.9 </td><td>44.25 </td></tr><tr><td>9 </td><td><em>C. limonia</em> </td><td>11 045 </td><td>9761 </td><td>4256 </td><td>45.82 </td><td>51.13 </td><td>41.37 </td><td>44.96 </td><td>43.17 </td></tr><tr><td>10 </td><td><em>C. medica</em> </td><td>1115 </td><td>896 </td><td>711 </td><td>45.65 </td><td>51.91 </td><td>40.9 </td><td>44.13 </td><td>42.15 </td></tr><tr><td>11 </td><td><em>C. reshni</em> </td><td>5768 </td><td>3735 </td><td>2644 </td><td>45.2 </td><td>51.51 </td><td>40.57 </td><td>43.52 </td><td>41.5 </td></tr><tr><td>12 </td><td><em>C. reticulata</em> </td><td>55 980 </td><td>46 170 </td><td>30 816 </td><td>46.85 </td><td>52.3 </td><td>42.77 </td><td>45.48 </td><td>43.62 </td></tr><tr><td>13 </td><td><em>C. reticulata</em> × <em>C. temple</em> </td><td>5823 </td><td>3685 </td><td>2253 </td><td>46.48 </td><td>52.41 </td><td>41.64 </td><td>45.41 </td><td>43.41 </td></tr><tr><td>14 </td><td><em>C. sinensis</em> × <em>P. trifoliata</em> </td><td>1837 </td><td>1522 </td><td>992 </td><td>46.07 </td><td>51.67 </td><td>41.41 </td><td>45.15 </td><td>43.23 </td></tr><tr><td>15 </td><td><em>C. sunki</em> </td><td>5216 </td><td>4688 </td><td>1746 </td><td>47.58 </td><td>51.51 </td><td>43.58 </td><td>47.64 </td><td>46.17 </td></tr><tr><td>16 </td><td><em>P. trifoliata</em> </td><td>62 695 </td><td>35 740 </td><td>19 445 </td><td>46.78 </td><td>52.33 </td><td>42.67 </td><td>45.35 </td><td>43.52 </td></tr><tr><td>17 </td><td><em>C. unshiu</em> </td><td>19 072 </td><td>9289 </td><td>4592 </td><td>45.67 </td><td>51.72 </td><td>41.14 </td><td>44.15 </td><td>42.21 </td></tr><tr><td>18 </td><td><em>C. paradisi</em> </td><td>8039 </td><td>4621 </td><td>2517 </td><td>45.32 </td><td>52.19 </td><td>40.51 </td><td>43.27 </td><td>41.3 </td></tr><tr><td>19 </td><td><em>C. paradisi</em> × <em>P. trifoliata</em> </td><td>7954 </td><td>3335 </td><td>2596 </td><td>45.23 </td><td>51.96 </td><td>40.77 </td><td>42.98 </td><td>40.96 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Means of GC% were calculated at the first, second, and third positions. GC<sub>3<em>s</em></sub> represents the GC at the third synonymous position.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>In <em>C. sinensis</em>, genome-predicted coding regions are used, whereas for the rest of the species, the number represents the EST count in the column (genes/EST).</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Selected genes above 300 bp threshold.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4607981" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS039TB1" class="table-wrap-title" data-id="DSS039TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Genomic composition of coding region of <em>Citrus</em> species and <em>P. trifoliata</em> at GC, GC<sub>1</sub>, GC<sub>2</sub>, GC<sub>3</sub>, and GC<sub>3<em>s</em></sub></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>No. </th><th><em>Citrus</em> species </th><th>Genes/EST<sup>a</sup> </th><th>Unigenes count </th><th>Genes<sup>b</sup> </th><th>GC </th><th>GC<sub>1</sub> </th><th>GC<sub>2</sub> </th><th>GC<sub>3</sub> </th><th>GC<sub>3<em>s</em></sub> </th></tr></thead><tbody><tr><td>1 </td><td><em>C. sinensis</em> </td><td>44 275 </td><td>– </td><td>41362 </td><td>43.92 </td><td>50.4 </td><td>40.33 </td><td>41.03 </td><td>38.8 </td></tr><tr><td>2 </td><td><em>C. aurantifolia</em> </td><td>8219 </td><td>7550 </td><td>4715 </td><td>47.78 </td><td>52.96 </td><td>43.87 </td><td>46.52 </td><td>44.73 </td></tr><tr><td>3 </td><td><em>C. aurantium</em> </td><td>14 584 </td><td>11 952 </td><td>6787 </td><td>46.93 </td><td>52.47 </td><td>42.59 </td><td>45.72 </td><td>43.94 </td></tr><tr><td>4 </td><td><em>C. clementina</em> </td><td>118 365 </td><td>51 591 </td><td>33 765 </td><td>47.04 </td><td>51.84 </td><td>42.51 </td><td>46.76 </td><td>44.99 </td></tr><tr><td>5 </td><td><em>C. clementina</em> × <em>C. tangerina</em> </td><td>1843 </td><td>1283 </td><td>677 </td><td>44.66 </td><td>51.67 </td><td>39.77 </td><td>42.55 </td><td>40.47 </td></tr><tr><td>6 </td><td><em>C. jambhiri</em> </td><td>1017 </td><td>858 </td><td>701 </td><td>45.86 </td><td>51.79 </td><td>41.13 </td><td>44.66 </td><td>42.67 </td></tr><tr><td>7 </td><td><em>C. japonica</em> var<em>. margarita</em> </td><td>2924 </td><td>1628 </td><td>588 </td><td>46.11 </td><td>52.67 </td><td>41.42 </td><td>44.25 </td><td>42.35 </td></tr><tr><td>8 </td><td><em>C. limettioides</em> </td><td>8188 </td><td>7933 </td><td>3964 </td><td>46.47 </td><td>50.76 </td><td>42.75 </td><td>45.9 </td><td>44.25 </td></tr><tr><td>9 </td><td><em>C. limonia</em> </td><td>11 045 </td><td>9761 </td><td>4256 </td><td>45.82 </td><td>51.13 </td><td>41.37 </td><td>44.96 </td><td>43.17 </td></tr><tr><td>10 </td><td><em>C. medica</em> </td><td>1115 </td><td>896 </td><td>711 </td><td>45.65 </td><td>51.91 </td><td>40.9 </td><td>44.13 </td><td>42.15 </td></tr><tr><td>11 </td><td><em>C. reshni</em> </td><td>5768 </td><td>3735 </td><td>2644 </td><td>45.2 </td><td>51.51 </td><td>40.57 </td><td>43.52 </td><td>41.5 </td></tr><tr><td>12 </td><td><em>C. reticulata</em> </td><td>55 980 </td><td>46 170 </td><td>30 816 </td><td>46.85 </td><td>52.3 </td><td>42.77 </td><td>45.48 </td><td>43.62 </td></tr><tr><td>13 </td><td><em>C. reticulata</em> × <em>C. temple</em> </td><td>5823 </td><td>3685 </td><td>2253 </td><td>46.48 </td><td>52.41 </td><td>41.64 </td><td>45.41 </td><td>43.41 </td></tr><tr><td>14 </td><td><em>C. sinensis</em> × <em>P. trifoliata</em> </td><td>1837 </td><td>1522 </td><td>992 </td><td>46.07 </td><td>51.67 </td><td>41.41 </td><td>45.15 </td><td>43.23 </td></tr><tr><td>15 </td><td><em>C. sunki</em> </td><td>5216 </td><td>4688 </td><td>1746 </td><td>47.58 </td><td>51.51 </td><td>43.58 </td><td>47.64 </td><td>46.17 </td></tr><tr><td>16 </td><td><em>P. trifoliata</em> </td><td>62 695 </td><td>35 740 </td><td>19 445 </td><td>46.78 </td><td>52.33 </td><td>42.67 </td><td>45.35 </td><td>43.52 </td></tr><tr><td>17 </td><td><em>C. unshiu</em> </td><td>19 072 </td><td>9289 </td><td>4592 </td><td>45.67 </td><td>51.72 </td><td>41.14 </td><td>44.15 </td><td>42.21 </td></tr><tr><td>18 </td><td><em>C. paradisi</em> </td><td>8039 </td><td>4621 </td><td>2517 </td><td>45.32 </td><td>52.19 </td><td>40.51 </td><td>43.27 </td><td>41.3 </td></tr><tr><td>19 </td><td><em>C. paradisi</em> × <em>P. trifoliata</em> </td><td>7954 </td><td>3335 </td><td>2596 </td><td>45.23 </td><td>51.96 </td><td>40.77 </td><td>42.98 </td><td>40.96 </td></tr></tbody></table></div><div content-id="DSS039TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>No. </th><th><em>Citrus</em> species </th><th>Genes/EST<sup>a</sup> </th><th>Unigenes count </th><th>Genes<sup>b</sup> </th><th>GC </th><th>GC<sub>1</sub> </th><th>GC<sub>2</sub> </th><th>GC<sub>3</sub> </th><th>GC<sub>3<em>s</em></sub> </th></tr></thead><tbody><tr><td>1 </td><td><em>C. sinensis</em> </td><td>44 275 </td><td>– </td><td>41362 </td><td>43.92 </td><td>50.4 </td><td>40.33 </td><td>41.03 </td><td>38.8 </td></tr><tr><td>2 </td><td><em>C. aurantifolia</em> </td><td>8219 </td><td>7550 </td><td>4715 </td><td>47.78 </td><td>52.96 </td><td>43.87 </td><td>46.52 </td><td>44.73 </td></tr><tr><td>3 </td><td><em>C. aurantium</em> </td><td>14 584 </td><td>11 952 </td><td>6787 </td><td>46.93 </td><td>52.47 </td><td>42.59 </td><td>45.72 </td><td>43.94 </td></tr><tr><td>4 </td><td><em>C. clementina</em> </td><td>118 365 </td><td>51 591 </td><td>33 765 </td><td>47.04 </td><td>51.84 </td><td>42.51 </td><td>46.76 </td><td>44.99 </td></tr><tr><td>5 </td><td><em>C. clementina</em> × <em>C. tangerina</em> </td><td>1843 </td><td>1283 </td><td>677 </td><td>44.66 </td><td>51.67 </td><td>39.77 </td><td>42.55 </td><td>40.47 </td></tr><tr><td>6 </td><td><em>C. jambhiri</em> </td><td>1017 </td><td>858 </td><td>701 </td><td>45.86 </td><td>51.79 </td><td>41.13 </td><td>44.66 </td><td>42.67 </td></tr><tr><td>7 </td><td><em>C. japonica</em> var<em>. margarita</em> </td><td>2924 </td><td>1628 </td><td>588 </td><td>46.11 </td><td>52.67 </td><td>41.42 </td><td>44.25 </td><td>42.35 </td></tr><tr><td>8 </td><td><em>C. limettioides</em> </td><td>8188 </td><td>7933 </td><td>3964 </td><td>46.47 </td><td>50.76 </td><td>42.75 </td><td>45.9 </td><td>44.25 </td></tr><tr><td>9 </td><td><em>C. limonia</em> </td><td>11 045 </td><td>9761 </td><td>4256 </td><td>45.82 </td><td>51.13 </td><td>41.37 </td><td>44.96 </td><td>43.17 </td></tr><tr><td>10 </td><td><em>C. medica</em> </td><td>1115 </td><td>896 </td><td>711 </td><td>45.65 </td><td>51.91 </td><td>40.9 </td><td>44.13 </td><td>42.15 </td></tr><tr><td>11 </td><td><em>C. reshni</em> </td><td>5768 </td><td>3735 </td><td>2644 </td><td>45.2 </td><td>51.51 </td><td>40.57 </td><td>43.52 </td><td>41.5 </td></tr><tr><td>12 </td><td><em>C. reticulata</em> </td><td>55 980 </td><td>46 170 </td><td>30 816 </td><td>46.85 </td><td>52.3 </td><td>42.77 </td><td>45.48 </td><td>43.62 </td></tr><tr><td>13 </td><td><em>C. reticulata</em> × <em>C. temple</em> </td><td>5823 </td><td>3685 </td><td>2253 </td><td>46.48 </td><td>52.41 </td><td>41.64 </td><td>45.41 </td><td>43.41 </td></tr><tr><td>14 </td><td><em>C. sinensis</em> × <em>P. trifoliata</em> </td><td>1837 </td><td>1522 </td><td>992 </td><td>46.07 </td><td>51.67 </td><td>41.41 </td><td>45.15 </td><td>43.23 </td></tr><tr><td>15 </td><td><em>C. sunki</em> </td><td>5216 </td><td>4688 </td><td>1746 </td><td>47.58 </td><td>51.51 </td><td>43.58 </td><td>47.64 </td><td>46.17 </td></tr><tr><td>16 </td><td><em>P. trifoliata</em> </td><td>62 695 </td><td>35 740 </td><td>19 445 </td><td>46.78 </td><td>52.33 </td><td>42.67 </td><td>45.35 </td><td>43.52 </td></tr><tr><td>17 </td><td><em>C. unshiu</em> </td><td>19 072 </td><td>9289 </td><td>4592 </td><td>45.67 </td><td>51.72 </td><td>41.14 </td><td>44.15 </td><td>42.21 </td></tr><tr><td>18 </td><td><em>C. paradisi</em> </td><td>8039 </td><td>4621 </td><td>2517 </td><td>45.32 </td><td>52.19 </td><td>40.51 </td><td>43.27 </td><td>41.3 </td></tr><tr><td>19 </td><td><em>C. paradisi</em> × <em>P. trifoliata</em> </td><td>7954 </td><td>3335 </td><td>2596 </td><td>45.23 </td><td>51.96 </td><td>40.77 </td><td>42.98 </td><td>40.96 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Means of GC% were calculated at the first, second, and third positions. GC<sub>3<em>s</em></sub> represents the GC at the third synonymous position.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>In <em>C. sinensis</em>, genome-predicted coding regions are used, whereas for the rest of the species, the number represents the EST count in the column (genes/EST).</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Selected genes above 300 bp threshold.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4607981" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=4607982 id="4607982" class="section-title" data-legacy-id=s2b>2.2. Frame correction of ESTs</h3>
<p>All the UniGenes (contigs + singletons) were analysed for frame correction and prediction of protein-coding region using FrameDP.<sup><a reveal-id="DSS039C27" data-open="DSS039C27" class="link link-ref link-reveal xref-bibr">27</a>,<a reveal-id="DSS039C28" data-open="DSS039C28" class="link link-ref link-reveal xref-bibr">28</a></sup> Briefly, the following pipeline was implemented using FrameDP to identify open reading frames (ORFs): firstly, each EST was compared against the TAIR database (<em>Arabidopsis</em> Information Resource; <a class="link link-uri" href="http://www.arabidopsis.org/">http://www.arabidopsis.org/</a>) using BLASTX<sup><a reveal-id="DSS039C29" data-open="DSS039C29" class="link link-ref link-reveal xref-bibr">29</a></sup> with E-value = 10<sup>−4</sup>, identity percent (%) = 40% over 100 amino acids.<sup><a reveal-id="DSS039C27" data-open="DSS039C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Secondly, the training dataset was generated from the BLASTX results, and, subsequently, the training matrix was calculated, which represents the coding style of the species. Thirdly, a collection of putative protein-coding sequences (CDSs) was generated for each <em>Citrus</em> species and <em>P. trifoliata</em> based on its homology with known protein dataset and on coding style recognition matrix.</p>                            <h3 scrollto-destination=4607984 id="4607984" class="section-title" data-legacy-id=s2c>2.3. Sequence filtering and GC variation</h3>
<p>From the set of corrected sequences, we discarded proteins shorter than 100 amino acids to create a reliable dataset for all the studied <em>Citrus</em> species and <em>P. trifoliata</em> for further analysis.<sup><a reveal-id="DSS039C23" data-open="DSS039C23" class="link link-ref link-reveal xref-bibr">23</a></sup> The final sequence dataset was subsequently analysed by tabulating the frequency of GC at the first, second, and third codon positions (GC<sub>1</sub>, GC<sub>2,</sub> GC<sub>3</sub>, and GC<sub>3<em>s</em></sub>, respectively) using in-house written Perl and C++ scripts. GC<sub>3</sub> is defined as the fraction of cytosines (C) and guanines (G) in the third position of the codon: GC<sub>3</sub> = 3(C<sub>3</sub> + G<sub>3</sub>)/L for the ORF of length L, whereas GC<sub>3S</sub> is defined as G + C base composition at the third synonymously degenerate position of codons. To define GC<sub>3</sub>-rich and GC<sub>3</sub>-poor groups, we have selected 5% of the genes with the highest and the lowest GC<sub>3</sub> values. For the genes in the GC<sub>3</sub>-rich and-poor groups, we have computed two measures: (i) positional gradients of GC<sub>3</sub> and (ii) CG<sub>3</sub> skew that are defined as: CG<sub>3</sub> skew is the difference in fraction of cytosines (C) and guanines (G) in the third position of the codon divided by the sum of C and G in the third position: CG<sub>3</sub>-skew = (C<sub>3</sub>−G<sub>3</sub>)/(C<sub>3</sub> + G<sub>3</sub>) and positional gradient GC<sub>3</sub> as a [G<sub>3</sub> (<em>x</em>) + C<sub>3</sub> (<em>x</em>)]/Nseq, where <em>x</em> is the distance measured as number of codons from the first ATG, and Nseq is the number of sequences.</p>                            <h3 scrollto-destination=4607986 id="4607986" class="section-title" data-legacy-id=s2d>2.4. Indices of codon usage and correspondence analysis</h3>
<div class="block-child-p">The effective number of codons (Nc) provides an independent measure of CUB, regardless of the gene length.<sup><a reveal-id="DSS039C23" data-open="DSS039C23" class="link link-ref link-reveal xref-bibr">23</a></sup> The expected Nc values were computed according to the equation proposed by Wright, which assumes equal use of G and C (A and T) in degenerate codon groups.<sup><a reveal-id="DSS039C23" data-open="DSS039C23" class="link link-ref link-reveal xref-bibr">23</a></sup><a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss039/2/dss039ueq1.gif?Expires=2147483647&Signature=MM26ewDroZP9eh1SuG2FwOl68NZywhkB1PtU0x0HKfmeDLsERzxZ1erCvPoq9UWl~byyWDgd5c1cJSjchzRjh4jnapwWwhIb6F6Xyi6QL-LzIVEmZyyX1dX4akoK7STODSvHyl7m-XjwSRAzcmL~IbV4hpen60i7QppbHQL1emhWXrd~WHjKBbRmr54fb0J3Gqm9~RbA1MY4DtJzr0p9NcHPpsYgvEyitVTba6LwpMtKJlzIlf7FurT0dom7tNyyfgft4fqCq5bTtprCr0mlD6SyQX4ilzVScxv0y8P74bc0nNrbud2g6Gg6tGuscGgew86nk6WTOr6pxSBQBWqwnw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dss039ueq1.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss039/2/m_dss039ueq1.gif?Expires=2147483647&Signature=4onauyMzLNqsV9I3-CcRxmgslOG0FEmP0mViFQRbRNFNJoVOruwgJbSFt5Q1C-wifyBZV5lRdZ13RskL59JSrUnj83L01sQS5uUcWGlDGEujSncFph978yWXITXMA~cYsct-KJqsHI3kA6n9HPdpWt2LTeHunky~0PabKvMIArvoHLg6mN3dg6oSjU11C0ozmjd3v8pSHfQK9qZQTIs50W3Qowb2L49~O10pBwcZTN6jyZgXEw6fdavRmIrkVIpm~ptO70UJfgYVDUbOkR5J9kMFoRj84~MbMVzs16460V5DmDo44w8dwX9jVnuo8Rd82mrFErHacXODLFSd3ik6pw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dss039ueq1.gif" /></a></div></div></div></div><p>We have further analysed RSCU using multivariate analysis for all the 59 informative codons (excluding Met, Trp, and the three stop codons).<sup><a reveal-id="DSS039C30" data-open="DSS039C30" class="link link-ref link-reveal xref-bibr">30</a>,<a reveal-id="DSS039C31" data-open="DSS039C31" class="link link-ref link-reveal xref-bibr">31</a></sup> As proposed earlier, if RSCU values are close to 1.0, it indicates that all the synonymous codons are used equally without any bias towards the usage of a particular codon in a gene of length <em>L</em>. In our study, axis 1 (COA/RSCU) and axis 2 (COA/RSCU) represent the first and second major axes of correspondence analysis. The Kyte–Doolittle scale was used to calculate the hydropathy score that is the arithmetic mean of the sum of hydropathic indices of each amino acid. This scale also provides information on transmembrane or surface regions.<sup><a reveal-id="DSS039C32" data-open="DSS039C32" class="link link-ref link-reveal xref-bibr">32</a></sup></p>                            <h3 scrollto-destination=4607989 id="4607989" class="section-title" data-legacy-id=s2e>2.5. Identification of optimal codons</h3>
<p>Optimal codons are believed to achieve faster translation rates and high accuracy, therefore the effect is more pronounced in highly expressed genes.<sup><a reveal-id="DSS039C33" data-open="DSS039C33" class="link link-ref link-reveal xref-bibr">33</a></sup> For the identification of optimal codons, we used 10% of total genes from extreme ends of the principal axis of correspondence analysis (axis 1; COA/RSCU). Codon usage was then compared using χ<sup>2</sup> contingency (χ<sup>2</sup>) of the two groups, and codons whose frequency of usage was significantly higher at three different levels of statistical precision (<em>P</em>-value &lt; 0.5; <em>P</em>-value &lt; 0.01; <em>P</em>-value &lt; 0.001) in highly expressed genes when compared with lowly expressed genes were defined as the optimal codons. We also identified the genes related to the ribosomal proteins and observed the association of the ribosomal proteins with axis 1 (COA/RSCU) and axis 2 (COA/RSCU) of the correspondence analysis to optimize the identification of the optimal codons.</p>                            <h3 scrollto-destination=4607991 id="4607991" class="section-title" data-legacy-id=s2f>2.6. GO and stress-associated annotation, Hamming parameter, and coevolution of amino acid composition</h3>
<p>Stress-related genes were identified using reciprocal best bidirectional hits (RBH) using NCBI BLASTP program with E-value 10<sup>−30</sup>, and gene ontology (GO) annotation of the model dicot plant <em>Arabidopsis thaliana</em> was then used by ‘guilt-by-association’ approach. To optimize our annotation pipeline, we selected GO annotations ‘involved in’ the category containing word ‘stress’ and/or ‘response’ and grouped together as ‘stress related’. The normalized Hamming distance was then evaluated according to the method described in Sablok et al.<sup><a reveal-id="DSS039C34" data-open="DSS039C34" class="link link-ref link-reveal xref-bibr">34</a></sup> All the frame-corrected coding regions and the genome-predicted coding regions were parsed. Subsequent proteins were used to identify co-orthologous genes, and a suggestive phylogeny was drawn using the co-orthologous matrix as described in Lechner et al.<sup><a reveal-id="DSS039C35" data-open="DSS039C35" class="link link-ref link-reveal xref-bibr">35</a></sup> To identify the effect of nucleotide bias on amino acid composition, we partitioned the codons according to GC-rich (the so-called GARP amino acids: Glycine, Alanine, Arginine, and Proline) and AT-rich, FYMINK amino acids (Phenylalanine, Tyrosine, Methionine, Isoleucine, Asparagine, and Lysine) amino acids.<sup><a reveal-id="DSS039C36" data-open="DSS039C36" class="link link-ref link-reveal xref-bibr">36</a>,<a reveal-id="DSS039C37" data-open="DSS039C37" class="link link-ref link-reveal xref-bibr">37</a></sup> We have maintained the exclusion of Leucine and Arginine from the dataset as per Singer and Hickey.<sup><a reveal-id="DSS039C37" data-open="DSS039C37" class="link link-ref link-reveal xref-bibr">37</a></sup></p>                            <h3 scrollto-destination=4607993 id="4607993" class="section-title" data-legacy-id=s2g>2.7. Statistical analysis</h3>
<p>All the indices of codon usage were calculated using CodonW (<a class="link link-uri" href="http://codonw.sourceforge.net">http://codonw.sourceforge.net</a>), and several in-house developed Perl, R, and C++ scripts were written to streamline the downstream analyses. Statistical analyses were performed using R in R studio (<a class="link link-uri" href="http://rstudio.org/">http://rstudio.org/</a>). All the results were interpreted based on the non-parametric Spearman's rank correlation (<em>ρ</em>).</p>                            <h2 scrollto-destination=4607995 id="4607995" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=4607996 id="4607996" class="section-title" data-legacy-id=s3a>3.1. Patterns of genomic content and codon usage variation across <em>Citrus species</em> and <em>P. trifoliata</em></h3>
<p>The recently sequenced nuclear genome of <em>C. sinensis</em><sup><a reveal-id="DSS039C25" data-open="DSS039C25" class="link link-ref link-reveal xref-bibr">25</a></sup> presents an opportunity to analyse the nucleotide compositional pressure and heterogenetic variation, which could possibly help us to understand the molecular adaptation of these horticulturally important fruit species. We have used meta-analyses approach using the predicted genomic coding regions of recently sequenced <em>C. sinensis</em> and frame-corrected ESTs of related species and <em>P. trifoliata</em>. Nucleotide composition analysis indicated that <em>Citrus</em> species are AT rich (∼46% GC, typical for many sequenced genomes of dicot species), and this bias towards the AT richness is dominant and is observed across all the studied species (Table <a reveal-id="DSS039TB1" data-open="DSS039TB1" class="link link-reveal link-table xref-fig">1</a>).</p><p>Based on the above observation, we could hypothesize that these species are biased towards the A- and/or T-ending codons in the coding region across these two genera. Generally, most of the analysed dicot species prefer to use A- or T-ending codons at the third position; this observation is in accordance with a previous study in <em>Citrus</em> using a small dataset of 177 CDS regions.<sup><a reveal-id="DSS039C38" data-open="DSS039C38" class="link link-ref link-reveal xref-bibr">38</a></sup> We found that average GC<sub>3</sub> is significantly lower than GC<sub>1</sub> and the observed GC (<em>P</em>-value &lt; 0.05) which potentially explains that the genome composition is biased towards the dominating AT usage encoding genes at the third, ‘wobble’ position.</p><p>In eukaryotes, the intra-genomic heterogeneity is high, and interspecific variation of the average GC content is low.<sup><a reveal-id="DSS039C39" data-open="DSS039C39" class="link link-ref link-reveal xref-bibr">39</a>,<a reveal-id="DSS039C40" data-open="DSS039C40" class="link link-ref link-reveal xref-bibr">40</a></sup> In <em>Citrus</em> species, GC usage varies by position, but with much greater variance, with higher usage in position 1 (∼51.8% GC) and lower GC usage in positions 2 and 3 (∼41.6 versus ∼44.7% and also at the third synonymous positions GC<sub>3s</sub> ∼ 42.8%). The observed results are consistent with the previous reports demonstrating preference for G is in position 1 and T/A in positions 2 and 3<sup><a reveal-id="DSS039C41" data-open="DSS039C41" class="link link-ref link-reveal xref-bibr">41</a></sup> and demonstrate the role of mutational pressure in the evolution of CUB across <em>Citrus</em> species and <em>P. trifoliata</em>. Furthermore, there is a significant wide variation in GC usage at the third synonymous position (GC<sub>3s</sub> ∼42.8%) (Table <a reveal-id="DSS039TB1" data-open="DSS039TB1" class="link link-reveal link-table xref-fig">1</a>; Fig. <a reveal-id="DSS039F1" data-open="DSS039F1" class="link link-reveal link-table xref-fig">1</a>). In our analysis, we estimated codon usage patterns of frame-corrected coding regions obtained from ESTs versus gene predictions derived from a fully assembled and annotated genome. </p>                        <a id="4608000" scrollto-destination="4608000"></a>
<div id="DSS039F1" data-id="DSS039F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss039/2/m_dss03901.jpeg?Expires=2147483647&Signature=qGmZymLbOYyhlY9Ba8ERcvfICoPqCsWBi3F3bbpC5bAMvE8~Kxz11s580t4Oa1Uq2vFtPxogMFNzqiKl4wKHIK3qKsxV1LWmNV4LSwSFNtmLgesqH6y6grs9oJNmF1ps3B6BvuvLBnnwcdkyfG8xG-ZvxZaV~v--uckyWkubqZxuIN~~ztqGjCN28B30EFpzT3BomV-7HT0YPbCgfiTpUH1oNY5XpihMvbZjj9tEWxYrN0qCkZHky1-eVR-rUnX~eWIGKq9ywVD~aPZBcR843tae-KxPD7MbyuDbJcMFYAafAXqLjlGYin-Mx6IZgGN3ldPRHzOZ~7abYHjZoAe2eQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The distribution of GC3 content in Citrus species, P. trifoliata and A. thaliana genes. The GC3 content showed unimodal distribution." path-from-xml="dss03901.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4608000/dss03901.jpeg" path-from-xml="dss03901.jpeg" target="_blank">View large</a><a section="4608000" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss039/2/dss03901.jpeg?Expires=2147483647&Signature=rcm6-gP9H-LwOm8DHuFVDbHNIK32-6-cNBwKauzJSkf0Z05H9mLBr6sDSwfq7tYz-O8~gfyr54g76FKPO1cb6KRPFv05Fogejoed1ohbimNdtHvsRRCvKrfFs~DjSTn2sgI2Gou6lXXjHowJu3khrY3U9Lbl60vy~NcJtP2CDnWVlDUCg4R~4fcAwSdZuQs1PEZJY99MYV4N5jY2SV7E630ev5dA4WXeTX9GPiqfyGgNzFbZdfajaMLSosWliyaiycpAXcalDwRsXRQqj5pdxb7Or~TCiSTCyRS9YbH6CtqM57DaI2vyMNL1oNi1Z-KxrXcj-IqITx3QSI0VN5s4Wg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4608000&ar=333376&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03901.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The distribution of GC<sub>3</sub> content in <em>Citrus</em> species, <em>P. trifoliata</em> and <em>A. thaliana</em> genes. The GC<sub>3</sub> content showed unimodal distribution.</p></div></div><div content-id="DSS039F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss039/2/m_dss03901.jpeg?Expires=2147483647&Signature=qGmZymLbOYyhlY9Ba8ERcvfICoPqCsWBi3F3bbpC5bAMvE8~Kxz11s580t4Oa1Uq2vFtPxogMFNzqiKl4wKHIK3qKsxV1LWmNV4LSwSFNtmLgesqH6y6grs9oJNmF1ps3B6BvuvLBnnwcdkyfG8xG-ZvxZaV~v--uckyWkubqZxuIN~~ztqGjCN28B30EFpzT3BomV-7HT0YPbCgfiTpUH1oNY5XpihMvbZjj9tEWxYrN0qCkZHky1-eVR-rUnX~eWIGKq9ywVD~aPZBcR843tae-KxPD7MbyuDbJcMFYAafAXqLjlGYin-Mx6IZgGN3ldPRHzOZ~7abYHjZoAe2eQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The distribution of GC3 content in Citrus species, P. trifoliata and A. thaliana genes. The GC3 content showed unimodal distribution." path-from-xml="dss03901.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4608000/dss03901.jpeg" path-from-xml="dss03901.jpeg" target="_blank">View large</a><a section="4608000" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss039/2/dss03901.jpeg?Expires=2147483647&Signature=rcm6-gP9H-LwOm8DHuFVDbHNIK32-6-cNBwKauzJSkf0Z05H9mLBr6sDSwfq7tYz-O8~gfyr54g76FKPO1cb6KRPFv05Fogejoed1ohbimNdtHvsRRCvKrfFs~DjSTn2sgI2Gou6lXXjHowJu3khrY3U9Lbl60vy~NcJtP2CDnWVlDUCg4R~4fcAwSdZuQs1PEZJY99MYV4N5jY2SV7E630ev5dA4WXeTX9GPiqfyGgNzFbZdfajaMLSosWliyaiycpAXcalDwRsXRQqj5pdxb7Or~TCiSTCyRS9YbH6CtqM57DaI2vyMNL1oNi1Z-KxrXcj-IqITx3QSI0VN5s4Wg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4608000&ar=333376&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03901.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The distribution of GC<sub>3</sub> content in <em>Citrus</em> species, <em>P. trifoliata</em> and <em>A. thaliana</em> genes. The GC<sub>3</sub> content showed unimodal distribution.</p></div></div><p>To look for biased association between the nucleotide content and codon usage, we plotted Nc against GC<sub>3s</sub>, also described as the Nc plot (Fig. <a reveal-id="DSS039F2" data-open="DSS039F2" class="link link-reveal link-table xref-fig">2</a>) that has been widely demonstrated as an important parameter to evaluate codon usage variation among genes, as this codon usage index has a definite relationship with the GC<sub>3</sub> (more compositionally biased DNA is expected to encode a smaller subset of codons).<sup><a reveal-id="DSS039C23" data-open="DSS039C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSS039C42" data-open="DSS039C42" class="link link-ref link-reveal xref-bibr">42</a></sup> Wright<sup><a reveal-id="DSS039C23" data-open="DSS039C23" class="link link-ref link-reveal xref-bibr">23</a></sup> argued that the comparison of actual distribution of genes, with expected distribution under no selection could be indicative, if CUB of genes has some other influences other than compositional constraints. A significant correlation was found between Nc and GC<sub>3s</sub> (0.466** <em>C. sinensis</em>; −0.025** <em>Citrus aurantifolia</em>; 0.070<em>** Citrus aurantium</em>; 0.125<em>** C. clementina</em>; 0.183<em>** C. clementina</em> × <em>Citrus tangerina</em>; 0.184<em>** Citrus jambhiri</em>; 0.179<em>** Citrus japonica</em> var. <em>margarita</em>; −0.048<em>** Citrus limettioides</em>; −0.031<em>** Citrus limonia</em>; 0.093<em>** Citrus medica</em>; 0.216<em>** Citrus reshni</em>; 0.061<em>** C. reticulata</em>; 0.101<em>** C. reticulata</em> × <em>Citrus temple</em>; 0.119<em>** C. sinensis</em> × <em>P. trifoliata</em>; −0.213<em>** Citrus sunki</em>; 0.061<em>** P. trifoliata</em>; 0.139<em>** Citrus unshiu</em>; 0.160<em>** Citrus paradisi</em>; 0.212<em>** C. paradisi</em> × <em>P. trifoliata</em>; **<em>P</em>-value &lt; 0.01). </p>                        <a id="4608002" scrollto-destination="4608002"></a>
<div id="DSS039F2" data-id="DSS039F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss039/2/m_dss03902.jpeg?Expires=2147483647&Signature=GsLDLlzWPzinLzOtDo5Mk2-md5djSRrWDfBqsmlPhUElLfqTXfe8BH1bKSilndh20mudN6SwxaiEAsV2Vp1qk-P9UsMymElqy4QzDcSz57logccTs8bKQ6GRsoCXsavx-NwggX9v4sxMP9Fww~Q5nmAaCQk4KEV65Y4UsyEqJn4SfLgR10F2yO8YaTwQlDV9~AdwDAm2w0V~wXjjU0nkPIEdFY-YvemrfboOhUChpew5CKY1HODiF6zVDn4fy9AoXPvHqdq6PNVZxq4buq54AwZIV56drQjLfuQYgsH~bqXNasz3eyvghRxBaPszlKO3~GglQ~jFsmOgYE~cnT30DA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Nc versus GC3s plot of Citrus species and P. trifoliata genes. The solid black line indicates the expected Nc value, if the codon bias is only due to GC3s. Species order: A = C. sinensis; B = C. aurantifolia; C = C. aurantium; D = C. clementina; E = C. clementina × C. tangerina; F = C. jambhiri; G = C. japonica var. margarita; H = C. limettioides; I = C. limonia; J = C. medica; K = C. reshni; L = C. reticulata; M = C. reticulata × C. temple; N = C. sinensis × P. trifoliata; O = C. sunki; P = P. trifoliata; Q = C. unshiu; R = C. paradisi; and S = C. paradisi × P. trifoliata." path-from-xml="dss03902.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4608002/dss03902.jpeg" path-from-xml="dss03902.jpeg" target="_blank">View large</a><a section="4608002" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss039/2/dss03902.jpeg?Expires=2147483647&Signature=X4hV8aOYWD4BonQ46kew6fTpgwG2f5cCOWQ5i~ogdDfpc7EJUAMB1pTlHtHW414NjRaVdWgI24P3BpVa3pokgOLjuFWrgfbbksNnjjxtSXq6tpGdT~CGtHyI96AvP2ZK7~cvAfqEddQVqWN7Fm~xDTa9lFznsAx9ufS8ZLCGFsPbw03uZpHREx21-surzzlgEcPvVSBBuq-YeW3zzTICYl-QCoc4A5KY--K7Ts65zylk4HxIItiCq1VTqu4qBm4Z2LLKFCBx2-Z-YjXz4nxDV0Um31ZJ3Iry-eZMw4zNJCUIat2XXPzKbpxp-3gkjIIxKbLcAu32y3ppTbMgW2LOYA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4608002&ar=333376&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03902.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Nc versus GC<sub>3<em>s</em></sub> plot of <em>Citrus</em> species and <em>P. trifoliata</em> genes. The solid black line indicates the expected Nc value, if the codon bias is only due to GC<sub>3s.</sub> Species order: A = <em>C. sinensis</em>; B = <em>C. aurantifolia</em>; C = <em>C. aurantium</em>; D = <em>C. clementina</em>; E = <em>C. clementina</em> × <em>C. tangerina</em>; F = <em>C. jambhiri</em>; G = <em>C. japonica</em> var<em>. margarita</em>; H = <em>C. limettioides</em>; I = <em>C. limonia</em>; J = <em>C. medica</em>; K = <em>C. reshni</em>; L = <em>C. reticulata</em>; M = <em>C. reticulata</em> × <em>C. temple</em>; N = <em>C. sinensis</em> × <em>P. trifoliata</em>; O = <em>C. sunki</em>; P = <em>P. trifoliata</em>; Q = <em>C. unshiu</em>; R = <em>C. paradisi</em>; and S = <em>C. paradisi</em> × <em>P. trifoliata</em>.</p></div></div><div content-id="DSS039F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss039/2/m_dss03902.jpeg?Expires=2147483647&Signature=GsLDLlzWPzinLzOtDo5Mk2-md5djSRrWDfBqsmlPhUElLfqTXfe8BH1bKSilndh20mudN6SwxaiEAsV2Vp1qk-P9UsMymElqy4QzDcSz57logccTs8bKQ6GRsoCXsavx-NwggX9v4sxMP9Fww~Q5nmAaCQk4KEV65Y4UsyEqJn4SfLgR10F2yO8YaTwQlDV9~AdwDAm2w0V~wXjjU0nkPIEdFY-YvemrfboOhUChpew5CKY1HODiF6zVDn4fy9AoXPvHqdq6PNVZxq4buq54AwZIV56drQjLfuQYgsH~bqXNasz3eyvghRxBaPszlKO3~GglQ~jFsmOgYE~cnT30DA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Nc versus GC3s plot of Citrus species and P. trifoliata genes. The solid black line indicates the expected Nc value, if the codon bias is only due to GC3s. Species order: A = C. sinensis; B = C. aurantifolia; C = C. aurantium; D = C. clementina; E = C. clementina × C. tangerina; F = C. jambhiri; G = C. japonica var. margarita; H = C. limettioides; I = C. limonia; J = C. medica; K = C. reshni; L = C. reticulata; M = C. reticulata × C. temple; N = C. sinensis × P. trifoliata; O = C. sunki; P = P. trifoliata; Q = C. unshiu; R = C. paradisi; and S = C. paradisi × P. trifoliata." path-from-xml="dss03902.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4608002/dss03902.jpeg" path-from-xml="dss03902.jpeg" target="_blank">View large</a><a section="4608002" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss039/2/dss03902.jpeg?Expires=2147483647&Signature=X4hV8aOYWD4BonQ46kew6fTpgwG2f5cCOWQ5i~ogdDfpc7EJUAMB1pTlHtHW414NjRaVdWgI24P3BpVa3pokgOLjuFWrgfbbksNnjjxtSXq6tpGdT~CGtHyI96AvP2ZK7~cvAfqEddQVqWN7Fm~xDTa9lFznsAx9ufS8ZLCGFsPbw03uZpHREx21-surzzlgEcPvVSBBuq-YeW3zzTICYl-QCoc4A5KY--K7Ts65zylk4HxIItiCq1VTqu4qBm4Z2LLKFCBx2-Z-YjXz4nxDV0Um31ZJ3Iry-eZMw4zNJCUIat2XXPzKbpxp-3gkjIIxKbLcAu32y3ppTbMgW2LOYA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4608002&ar=333376&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03902.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Nc versus GC<sub>3<em>s</em></sub> plot of <em>Citrus</em> species and <em>P. trifoliata</em> genes. The solid black line indicates the expected Nc value, if the codon bias is only due to GC<sub>3s.</sub> Species order: A = <em>C. sinensis</em>; B = <em>C. aurantifolia</em>; C = <em>C. aurantium</em>; D = <em>C. clementina</em>; E = <em>C. clementina</em> × <em>C. tangerina</em>; F = <em>C. jambhiri</em>; G = <em>C. japonica</em> var<em>. margarita</em>; H = <em>C. limettioides</em>; I = <em>C. limonia</em>; J = <em>C. medica</em>; K = <em>C. reshni</em>; L = <em>C. reticulata</em>; M = <em>C. reticulata</em> × <em>C. temple</em>; N = <em>C. sinensis</em> × <em>P. trifoliata</em>; O = <em>C. sunki</em>; P = <em>P. trifoliata</em>; Q = <em>C. unshiu</em>; R = <em>C. paradisi</em>; and S = <em>C. paradisi</em> × <em>P. trifoliata</em>.</p></div></div><p>We noted that most genes tend to lie below the standard trajectory path and are poised towards GC<sub>3s</sub>, which clearly demonstrates that MB is acting as a major factor for the wide variation in codon usage across the <em>Citrus</em> species and <em>P. trifoliata</em> (Fig. <a reveal-id="DSS039F2" data-open="DSS039F2" class="link link-reveal link-table xref-fig">2</a>). However, there might be additional factors that might influence codon usage across these species. This dependence of codon usage on genome base composition (AT- or GC-richness) has been previously reported in several unicellular genomes.<sup><a reveal-id="DSS039C43" data-open="DSS039C43" class="link link-ref link-reveal xref-bibr">43</a>,<a reveal-id="DSS039C44" data-open="DSS039C44" class="link link-ref link-reveal xref-bibr">44</a></sup> In a genome-wide analysis of eubacterial and archeal genomes, it has been suggested that genome-wide variance in codon usage is primarily due to MB as the GC content shows wide variation along the isochors.<sup><a reveal-id="DSS039C17" data-open="DSS039C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Simultaneously, alternative views associate GC variability with various factors such as transcriptional optimization, methylation, recombination, and horizontal gene transfer.<sup><a reveal-id="DSS039C45" data-open="DSS039C45" class="link link-ref link-reveal xref-bibr">45</a></sup> It can be inferred that the cloning of orthologs and homologues conserved across the <em>Citrus</em> species and <em>P. trifoliata</em> that are AT rich and have low Nc values will require few degenerate primers. On the contrary, genes that are GC rich and having high GC values will require more degenerate primers for enhancement of cloning efficiency.</p>                            <h3 scrollto-destination=4608004 id="4608004" class="section-title" data-legacy-id=s3b>3.2. Correspondence analysis</h3>
<p>It has been reported that there is a significant heterogeneity within the genes and genomes.<sup><a reveal-id="DSS039C33" data-open="DSS039C33" class="link link-ref link-reveal xref-bibr">33</a>,<a reveal-id="DSS039C46" data-open="DSS039C46" class="link link-ref link-reveal xref-bibr">46</a></sup> A heat map was constructed using the observed RSCU values for all the 59 informative codons (excluding the Met, Trp, and the stop codons) using the average linkage clustering method (Fig. <a reveal-id="DSS039F3" data-open="DSS039F3" class="link link-reveal link-table xref-fig">3</a>). Heat map and the supporting values (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table 1</a>) clearly define that the global codon usage is biased towards AT richness. We have partitioned the genes based on the high and low GC<sub>3s</sub> content to see the global pattern of deviation across the two principal axes displaying the major variation in accordance with the synonymous GC<sub>3s</sub> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. 1</a>). </p>                        <a id="4608006" scrollto-destination="4608006"></a>
<div id="DSS039F3" data-id="DSS039F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss039/2/m_dss03903.jpeg?Expires=2147483647&Signature=dWKYufe5uUqSjzWdMGv-V2PR5rzWEDse8DsSur5~JAs6fTtmvFQoBFGJsRCM9zha36-Nl5SzJNquBYzckntYPGGOWAVTgWFBvoERbHo4Sbc8k4E4WuoVIbPRd6uK8f-5x-o-grHKfncrsqgCQIfi725OC-2JLsrSDSFd4g6c2nqXM0CUhGjD5itG8nF7yRSLBKROhX8Tk2VdqWED60-vgw9tjpS~ZYLQQfCWEtapgkQgVlk~1wxqFGcbRiCnSbxDX~yMCY8DYP4XamLp0y7FQCHjPL5fbjAyebM0BGBc4EMEbK39Nc~i2rRTQI6e3RTBLgowieaTLHYkaNhC3BByVw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Heat map of the average RSCU of the 59 degenerate codons in the Citrus species and P. trifoliata using Euclidean distance and average linkage clustering module." path-from-xml="dss03903.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4608006/dss03903.jpeg" path-from-xml="dss03903.jpeg" target="_blank">View large</a><a section="4608006" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss039/2/dss03903.jpeg?Expires=2147483647&Signature=O-dv7Qn1zpI4IE2HIJE55vVpm8ecp2sBHN0dj3jNAqLPRyDqi8Q45jz7tJU~1v7ypkkQl~B0xE-P4dJSr7LuajLNmXfgo8hDOODgcz7gnbHlFYajk2OiVrChWEHVL2c0ryeK~dFYpg~4Z~ddEB2FnbGAKQ3DTeldP9w9huSVJVo5nH2Z84WTd-kq7BAlJ0PN3RrTI-Ad1Hr~ndUym8zs55r7YnzNMKfWFhdHrnKUEnpZ2jJdnIli0DxEJsg3aaIfD2ebb3rIhScvSXfJlPkqNh3UxsO-UwWV1gHbbNKnHRwl54CG3Vdip72-NCsY-h5bdHS55sowytWeum94cAJJnA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4608006&ar=333376&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03903.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Heat map of the average RSCU of the 59 degenerate codons in the <em>Citrus</em> species and <em>P. trifoliata</em> using Euclidean distance and average linkage clustering module.</p></div></div><div content-id="DSS039F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss039/2/m_dss03903.jpeg?Expires=2147483647&Signature=dWKYufe5uUqSjzWdMGv-V2PR5rzWEDse8DsSur5~JAs6fTtmvFQoBFGJsRCM9zha36-Nl5SzJNquBYzckntYPGGOWAVTgWFBvoERbHo4Sbc8k4E4WuoVIbPRd6uK8f-5x-o-grHKfncrsqgCQIfi725OC-2JLsrSDSFd4g6c2nqXM0CUhGjD5itG8nF7yRSLBKROhX8Tk2VdqWED60-vgw9tjpS~ZYLQQfCWEtapgkQgVlk~1wxqFGcbRiCnSbxDX~yMCY8DYP4XamLp0y7FQCHjPL5fbjAyebM0BGBc4EMEbK39Nc~i2rRTQI6e3RTBLgowieaTLHYkaNhC3BByVw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Heat map of the average RSCU of the 59 degenerate codons in the Citrus species and P. trifoliata using Euclidean distance and average linkage clustering module." path-from-xml="dss03903.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4608006/dss03903.jpeg" path-from-xml="dss03903.jpeg" target="_blank">View large</a><a section="4608006" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss039/2/dss03903.jpeg?Expires=2147483647&Signature=O-dv7Qn1zpI4IE2HIJE55vVpm8ecp2sBHN0dj3jNAqLPRyDqi8Q45jz7tJU~1v7ypkkQl~B0xE-P4dJSr7LuajLNmXfgo8hDOODgcz7gnbHlFYajk2OiVrChWEHVL2c0ryeK~dFYpg~4Z~ddEB2FnbGAKQ3DTeldP9w9huSVJVo5nH2Z84WTd-kq7BAlJ0PN3RrTI-Ad1Hr~ndUym8zs55r7YnzNMKfWFhdHrnKUEnpZ2jJdnIli0DxEJsg3aaIfD2ebb3rIhScvSXfJlPkqNh3UxsO-UwWV1gHbbNKnHRwl54CG3Vdip72-NCsY-h5bdHS55sowytWeum94cAJJnA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4608006&ar=333376&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03903.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Heat map of the average RSCU of the 59 degenerate codons in the <em>Citrus</em> species and <em>P. trifoliata</em> using Euclidean distance and average linkage clustering module.</p></div></div><p>Correspondence analysis showed that relative inertia tends to decrease along the axes, and in all the studies species, it was observed that axis 1 (COA/RSCU) contributed to the major portion of relative inertia indicating that the major trend of codon usage variation is associated with axis 1 (10.75 <em>C. sinensis</em>; 12.15 <em>C. aurantifolia</em>; 12.27 <em>C. aurantium</em>; 14.54 <em>C. clementina</em>; 6.65 <em>C. clementina</em> × <em>C. tangrina</em>; 11.62 <em>C. jambhiri</em>; 7.01 <em>C. japonica</em> var. <em>margarita</em>; 10.41 <em>C. limettioides</em>; 14.66 <em>C. limonia</em>; 11.24 <em>C. medica</em>; 10.67 <em>C. reshni</em>; 12.54 <em>C. reticulata</em>; 8.66 <em>C. reticulata</em> × <em>C. temple</em>; 10.85 <em>C. sinensis</em> × <em>P. trifoliata</em>; 13.61 <em>C. sunki</em>; 12.06 <em>P. trifoliata</em>; 9.11 <em>C. unshiu</em>; 8.72 <em>C. paradisi</em>; 10.75 <em>C. paradisi</em> × <em>P. trifoliata</em>). High significant correlation (+/−) was also observed between axis 1 and GC<sub>3<em>s</em></sub> (0.860**, <em>C. sinensis</em>; 0.888<em>**</em>, <em>C. aurantifolia</em>; 0.887<em>**</em>, <em>C. aurantium</em>; 0.903<em>**</em>, −0.708<em>** C. clementina</em>; −0.836<em>**</em>, <em>C. clementina</em> × <em>C. tangrina</em>; −0.772<em>**</em>, <em>C. jambhiri</em>; −0.831<em>**</em>, <em>C. japonica</em> var. <em>margarita</em>; −0.803<em>**</em>, <em>C. limettioides</em>; 0.849<em>**</em>, <em>C. limonia</em>; 0.894<em>**</em>, <em>C. medica</em>; −0.855<em>**</em>, <em>C. reshni</em>; 0.894<em>**</em>, <em>C. reticulata</em>; 0.846<em>**</em>, <em>C. reticulata</em> × <em>C. temple</em>; −0.884<em>**</em>, <em>C. sinensis</em> × <em>P. trifoliata</em>; 0.806<em>**</em>, <em>C. sunki</em>; 0.879<em>**</em>, <em>P. trifoliata</em>; 0.834<em>**</em>, <em>C. unshiu</em>; −0.848<em>**</em>, <em>C. paradisi</em>; −0.879<em>**</em>, <em>C. paradisi</em> × <em>P. trifoliata</em>; <em>**P</em>-value &lt; 0.01). The observed results indicate dominance of MB in the <em>Citrus</em> species and <em>P. trifoliata</em> and suggest that the variation in the usage of synonymous codons among the genes in <em>Citrus</em> species and <em>P. trifoliata</em> is largely a biased representation of nucleotide content of the genes.</p>                            <h3 scrollto-destination=4608008 id="4608008" class="section-title" data-legacy-id=s3c>3.3. Identification of optimal codons in <em>Citrus species</em> and <em>P. trifoliata</em></h3>
<p>Earlier and recent reports suggest that the usage of optimal codons and balanced codon usage enhances the efficiency of translation by increasing the translation rate of the preferred codons over the other synonymous codon choices.<sup><a reveal-id="DSS039C47" data-open="DSS039C47" class="link link-ref link-reveal xref-bibr">47</a>,<a reveal-id="DSS039C48" data-open="DSS039C48" class="link link-ref link-reveal xref-bibr">48</a></sup> Genes using optimal codons have higher translation rate as compared to genes using non-optimal codons, which in turn increases the ribosome usage efficiency and potentially reduces the ribosome drop off.<sup><a reveal-id="DSS039C22" data-open="DSS039C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DSS039C49" data-open="DSS039C49" class="link link-ref link-reveal xref-bibr">49</a>,<a reveal-id="DSS039C50" data-open="DSS039C50" class="link link-ref link-reveal xref-bibr">50</a></sup> ESTs constitute partial transcriptome representation correlated with gene abundance and expression.<sup><a reveal-id="DSS039C51" data-open="DSS039C51" class="link link-ref link-reveal xref-bibr">51</a>,<a reveal-id="DSS039C52" data-open="DSS039C52" class="link link-ref link-reveal xref-bibr">52</a></sup> In 14 <em>Citrus</em> species (<em>C. aurantifolia</em>; <em>C. aurantium</em>; <em>C. clementina</em>; <em>C. jambhiri</em>; <em>C. limettioides</em>; <em>C. limonia</em>; <em>C. medica</em>; <em>C. reshni</em>; <em>C. reticulata</em>; <em>C. reticulata</em> × <em>C. temple</em>; <em>C. sinensis</em> × <em>P. trifoliata</em>; <em>C. unshiu</em>; <em>C. paradisi</em>; <em>C. paradisi</em> × <em>P. trifoliata</em>) and <em>P. trifoliata</em>, we extracted the expression of ESTs count using the CAP3 assembly (.ace files) and compared the usage of codons in accordance with earlier reports.<sup><a reveal-id="DSS039C53  DSS039C55" data-open="DSS039C53  DSS039C55" class="link link-ref link-reveal xref-bibr">53–55</a></sup> We also selected the ribosomal protein-encoding genes and quantified the association of the ribosomal encoding genes with axis 1 (COA/RSCU) and axis 2 (COA/RSCU) of correspondence analysis.</p><p>A star map showing optimal codons was constructed taking 10% of the genes from the extreme tails of multivariate analysis (Table <a reveal-id="DSS039TB2" data-open="DSS039TB2" class="link link-reveal link-table xref-fig">2</a>), and several distinct trends of optimal codons were detected. In an earlier study,<sup><a reveal-id="DSS039C38" data-open="DSS039C38" class="link link-ref link-reveal xref-bibr">38</a></sup> optimal codons in <em>Citrus</em> were estimated based on the correspondence analysis of codon usage, and the relative frequency of synonymous codon using 177 CDS and A- or T-ending optimal codons (TAA, GCT, GAT, CTT, AGG, AGA, and GTT) was observed. Overall, we observed that the trend of optimal codons usage was not conserved across all analysed species. However, in our analyses, we have observed that GCT (∼1.57; RSCU) and GGT (∼1.09; RSCU) representing Alanine and Glycine were found to be evolutionary conserved across all the species. Because both these optimal codons are T-ending codons, which is an indication of the dominant role of the MB in the conservation of the A- or T-ending codons, it potentially represents the biased genome composition. However, for the other observed optimal codons, pattern genomic composition was not able to explain the deviation. We further observed that for most amino acids with 2- to 6-fold degeneracy level, there has been a general preference for the usage of two or more codons as optimal codons. For example, in Glycine, two highly distributed optimal codons GGA and GGT were identified and they could be classified as the primary and secondary optimal codons preferentially based on the RSCU (GGA, ∼1.16 and GGT, ∼1.09). A curious trend observed among the <em>Citrus</em> species and <em>P. trifoliata</em> is that Leucine is frequently encoded optimally by a G-ending codon (TTG; ∼RSCU: 1.49) in four species (<em>C. sinensis</em>, <em>C. clementina</em>, <em>C. reticulata</em>, and <em>P. trifoliata</em>) rather than synonymous A- or T-ending codon. However, in other species, CTT (∼RSCU: 1.49) and CTC (∼RSCU: 0.89) (TTA, RSCU: 1.16; CTT RSCU: 1.51) were the optimal codons. </p>                        <a id="4608011" scrollto-destination="4608011"></a>
<div content-id="DSS039TB2" class="table-modal"><div class="table-wrap"><div id="DSS039TB2" class="table-wrap-title" data-id="DSS039TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Star map showing optimal codon distribution in <em>Citrus</em> species and <em>P. trifoliata</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Codon/AA </th><th>A </th><th>B </th><th>C </th><th>D </th><th>E </th><th>F </th><th>G </th><th>H </th><th>I </th><th>J </th><th>K </th><th>L </th><th>M </th><th>N </th><th>O </th><th>P </th></tr></thead><tbody><tr><td>TTT/Phe </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>TTC/Phe </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>TTA/Leu </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>TTG/Leu </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>CTT/Leu </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td></tr><tr><td>CTC/Leu </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>CTA/Leu </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>CTG/Leu </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>ATT/Ile </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>ATC/Ile </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>ATA/Ile </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>GTT/Val </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td>* </td></tr><tr><td>GTC/Val </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>GTA/Val </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>GTG/Val </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>TCT/Ser </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td></tr><tr><td>TCC/Ser </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>TCA/Ser </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td></tr><tr><td>TCG/Ser </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>AGT/Ser </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>AGC/Ser </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>CCT/Pro </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td></tr><tr><td>CCC/Pro </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td></tr><tr><td>CCA/Pro </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td></tr><tr><td>CCG/Pro </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>ACT/Thr </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td></tr><tr><td>ACC/Thr </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>ACA/Thr </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>ACG/Thr </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>GCT/Ala </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td></tr><tr><td>GCC/Ala </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>GCA/Ala </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>GCG/Ala </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>TAT/Tyr </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>TAC/Tyr </td><td> </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>CAT/His </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>CAC/His </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>CAA/Gln </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>CAG/Gln </td><td> </td><td>* </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td>* </td></tr><tr><td>AAT/Asn </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td> </td></tr><tr><td>AAA/Lys </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>AAG/Lys </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>GAT/Asp </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td></tr><tr><td>GAA/Glu </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>GAG/Glu </td><td> </td><td>* </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td>* </td></tr><tr><td>TGT/Cys </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>TGC/Cys </td><td> </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td></tr><tr><td>CGT/Arg </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>CGC/Arg </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>CGA/Arg </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>CGG/Arg </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>AGA/Arg </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>AGG/Arg </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td></tr><tr><td>GGT/Gly </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td></tr><tr><td>GGC/Gly </td><td> </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>GGA/Gly </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td>* </td><td> </td></tr></tbody></table></div><div content-id="DSS039TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Codon/AA </th><th>A </th><th>B </th><th>C </th><th>D </th><th>E </th><th>F </th><th>G </th><th>H </th><th>I </th><th>J </th><th>K </th><th>L </th><th>M </th><th>N </th><th>O </th><th>P </th></tr></thead><tbody><tr><td>TTT/Phe </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>TTC/Phe </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>TTA/Leu </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>TTG/Leu </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>CTT/Leu </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td></tr><tr><td>CTC/Leu </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>CTA/Leu </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>CTG/Leu </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>ATT/Ile </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>ATC/Ile </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>ATA/Ile </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>GTT/Val </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td>* </td></tr><tr><td>GTC/Val </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>GTA/Val </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>GTG/Val </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>TCT/Ser </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td></tr><tr><td>TCC/Ser </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>TCA/Ser </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td></tr><tr><td>TCG/Ser </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>AGT/Ser </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>AGC/Ser </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>CCT/Pro </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td></tr><tr><td>CCC/Pro </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td></tr><tr><td>CCA/Pro </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td></tr><tr><td>CCG/Pro </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>ACT/Thr </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td></tr><tr><td>ACC/Thr </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>ACA/Thr </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>ACG/Thr </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>GCT/Ala </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td></tr><tr><td>GCC/Ala </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>GCA/Ala </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>GCG/Ala </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>TAT/Tyr </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>TAC/Tyr </td><td> </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>CAT/His </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>CAC/His </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>CAA/Gln </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>CAG/Gln </td><td> </td><td>* </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td>* </td></tr><tr><td>AAT/Asn </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td> </td></tr><tr><td>AAA/Lys </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>AAG/Lys </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>GAT/Asp </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td></tr><tr><td>GAA/Glu </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>GAG/Glu </td><td> </td><td>* </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td>* </td></tr><tr><td>TGT/Cys </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>TGC/Cys </td><td> </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td></tr><tr><td>CGT/Arg </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>CGC/Arg </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>CGA/Arg </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>CGG/Arg </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>AGA/Arg </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>AGG/Arg </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td></tr><tr><td>GGT/Gly </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td></tr><tr><td>GGC/Gly </td><td> </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>GGA/Gly </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td>* </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Species are in the following order: A = <em>C. sinensis</em>; B = <em>C. aurantifolia</em>; C = <em>C. aurantium</em>; D = <em>C. clementina</em>; E = <em>C. jambhiri</em>; F = <em>C. limettioides</em>; G = <em>C. limonia</em>; H = <em>C. medica</em>; I = <em>C. reshni</em>; J = <em>C. reticulata</em>; K = <em>C. reticulata</em> × <em>C. temple</em>; L = <em>C. sinensis</em> × <em>P. trifoliata</em>; M = <em>P. trifoliata</em>; N = <em>C. unshiu</em>; O = <em>C. paradisi</em>; and P = <em>C. paradisi</em> × <em>P. trifoliata</em>. AA represents amino acid.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4608011" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS039TB2" class="table-wrap-title" data-id="DSS039TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Star map showing optimal codon distribution in <em>Citrus</em> species and <em>P. trifoliata</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Codon/AA </th><th>A </th><th>B </th><th>C </th><th>D </th><th>E </th><th>F </th><th>G </th><th>H </th><th>I </th><th>J </th><th>K </th><th>L </th><th>M </th><th>N </th><th>O </th><th>P </th></tr></thead><tbody><tr><td>TTT/Phe </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>TTC/Phe </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>TTA/Leu </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>TTG/Leu </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>CTT/Leu </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td></tr><tr><td>CTC/Leu </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>CTA/Leu </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>CTG/Leu </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>ATT/Ile </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>ATC/Ile </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>ATA/Ile </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>GTT/Val </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td>* </td></tr><tr><td>GTC/Val </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>GTA/Val </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>GTG/Val </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>TCT/Ser </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td></tr><tr><td>TCC/Ser </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>TCA/Ser </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td></tr><tr><td>TCG/Ser </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>AGT/Ser </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>AGC/Ser </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>CCT/Pro </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td></tr><tr><td>CCC/Pro </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td></tr><tr><td>CCA/Pro </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td></tr><tr><td>CCG/Pro </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>ACT/Thr </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td></tr><tr><td>ACC/Thr </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>ACA/Thr </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>ACG/Thr </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>GCT/Ala </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td></tr><tr><td>GCC/Ala </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>GCA/Ala </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>GCG/Ala </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>TAT/Tyr </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>TAC/Tyr </td><td> </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>CAT/His </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>CAC/His </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>CAA/Gln </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>CAG/Gln </td><td> </td><td>* </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td>* </td></tr><tr><td>AAT/Asn </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td> </td></tr><tr><td>AAA/Lys </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>AAG/Lys </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>GAT/Asp </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td></tr><tr><td>GAA/Glu </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>GAG/Glu </td><td> </td><td>* </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td>* </td></tr><tr><td>TGT/Cys </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>TGC/Cys </td><td> </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td></tr><tr><td>CGT/Arg </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>CGC/Arg </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>CGA/Arg </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>CGG/Arg </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>AGA/Arg </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>AGG/Arg </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td></tr><tr><td>GGT/Gly </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td></tr><tr><td>GGC/Gly </td><td> </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>GGA/Gly </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td>* </td><td> </td></tr></tbody></table></div><div content-id="DSS039TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Codon/AA </th><th>A </th><th>B </th><th>C </th><th>D </th><th>E </th><th>F </th><th>G </th><th>H </th><th>I </th><th>J </th><th>K </th><th>L </th><th>M </th><th>N </th><th>O </th><th>P </th></tr></thead><tbody><tr><td>TTT/Phe </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>TTC/Phe </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>TTA/Leu </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>TTG/Leu </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>CTT/Leu </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td></tr><tr><td>CTC/Leu </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>CTA/Leu </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>CTG/Leu </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>ATT/Ile </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>ATC/Ile </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>ATA/Ile </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>GTT/Val </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td>* </td></tr><tr><td>GTC/Val </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>GTA/Val </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>GTG/Val </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>TCT/Ser </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td></tr><tr><td>TCC/Ser </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>TCA/Ser </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td></tr><tr><td>TCG/Ser </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>AGT/Ser </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>AGC/Ser </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>CCT/Pro </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td></tr><tr><td>CCC/Pro </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td></tr><tr><td>CCA/Pro </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td></tr><tr><td>CCG/Pro </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>ACT/Thr </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td></tr><tr><td>ACC/Thr </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>ACA/Thr </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>ACG/Thr </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>GCT/Ala </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td></tr><tr><td>GCC/Ala </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>GCA/Ala </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>GCG/Ala </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>TAT/Tyr </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>TAC/Tyr </td><td> </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>CAT/His </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>CAC/His </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>CAA/Gln </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>CAG/Gln </td><td> </td><td>* </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td>* </td></tr><tr><td>AAT/Asn </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td> </td></tr><tr><td>AAA/Lys </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>AAG/Lys </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>GAT/Asp </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td></tr><tr><td>GAA/Glu </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>GAG/Glu </td><td> </td><td>* </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td>* </td></tr><tr><td>TGT/Cys </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>TGC/Cys </td><td> </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td></tr><tr><td>CGT/Arg </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>CGC/Arg </td><td> </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td>* </td></tr><tr><td>CGA/Arg </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td>* </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>CGG/Arg </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>AGA/Arg </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td>* </td><td> </td><td> </td><td> </td></tr><tr><td>AGG/Arg </td><td> </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>* </td></tr><tr><td>GGT/Gly </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td><td>* </td></tr><tr><td>GGC/Gly </td><td> </td><td>* </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>GGA/Gly </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td>* </td><td> </td><td> </td><td>* </td><td>* </td><td> </td><td>* </td><td> </td><td>* </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Species are in the following order: A = <em>C. sinensis</em>; B = <em>C. aurantifolia</em>; C = <em>C. aurantium</em>; D = <em>C. clementina</em>; E = <em>C. jambhiri</em>; F = <em>C. limettioides</em>; G = <em>C. limonia</em>; H = <em>C. medica</em>; I = <em>C. reshni</em>; J = <em>C. reticulata</em>; K = <em>C. reticulata</em> × <em>C. temple</em>; L = <em>C. sinensis</em> × <em>P. trifoliata</em>; M = <em>P. trifoliata</em>; N = <em>C. unshiu</em>; O = <em>C. paradisi</em>; and P = <em>C. paradisi</em> × <em>P. trifoliata</em>. AA represents amino acid.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4608011" target="_blank">View Large</a></div></div><p>We found deviations in the usage of optimal codons encoding Lysine. For example, Lysine is frequently encoded by AAA (∼RSCU: 0.98) as optimal codon in three species (<em>C. sinensis</em>, <em>C. reticulata</em>, and <em>P. trifoliata</em>), whereas AAG (∼RSCU: 1.01) was found to be the optimal codon for Lysine in rest of the <em>Citrus</em> species. In Arginine, we observed that in two species (<em>C. aurantifolia</em> and <em>C. paradisi</em> × <em>P. trifoliata</em>), AGG represents the potential optimal codon instead of the synonymous AGA. But based on the RSCU values, AGA (∼RSCU: 1.66) was assumed to be more dominant over the AGG codon (∼RSCU: 1.52) at the respective levels of significance <em>P</em>-value &lt; 0.01 and <em>P</em>-value &lt; 0.05. Using a mutation and selection model, Knight et al.<sup><a reveal-id="DSS039C56" data-open="DSS039C56" class="link link-ref link-reveal xref-bibr">56</a></sup> demonstrated that ‘pairs of species with convergent GC content might also evolve convergent protein sequences, especially at functionally unconstrained positions’. For instance, the frequencies of both Lysine and Arginine are highly anti-correlated with GC content, and Lysine and Arginine can easily be substituted for one another in proteins. This observation explains the inclusion of Lysine and Arginine in our study and is well supported by an earlier study in nematodes.<sup><a reveal-id="DSS039C57" data-open="DSS039C57" class="link link-ref link-reveal xref-bibr">57</a></sup> It has been proposed that Arginine and Leucine have a tendency to show different codon usage patterns because of the prevalence of the synonymous GC substitutions in the first and the third codon position.<sup><a reveal-id="DSS039C58" data-open="DSS039C58" class="link link-ref link-reveal xref-bibr">58</a></sup> Recently, it has been postulated that codon optimization significantly enhanced <em>MIR</em> gene expression in <em>Solanum lycopersicum</em> cv. Microtom,<sup><a reveal-id="DSS039C59" data-open="DSS039C59" class="link link-ref link-reveal xref-bibr">59</a></sup> which potentially depicts the importance and the usage of the optimal codons in gene expression and transgenics. To our knowledge, this is the first time large-scale identification of optimal codons in <em>Citrus</em> species and <em>P. trifoliata</em>, which could serve as a model repertoire to enhance the transformation efficiency.</p>                            <h3 scrollto-destination=4608013 id="4608013" class="section-title" data-legacy-id=s3d>3.4. Role of other selective constraints on codon bias in <em>Citrus species</em> and <em>P. trifoliata</em></h3>
<p>A recent study has revealed that MB deeply influences the folding stability of proteins, making proteins on the average less hydrophobic and, therefore, less stable with respect to unfolding and also less susceptible to misfolding and aggregation.<sup><a reveal-id="DSS039C60" data-open="DSS039C60" class="link link-ref link-reveal xref-bibr">60</a></sup> To identify the potential effects of Gravy and aromaticity, we computed non-parametric correlation coefficients between axis 1 (COA/RSCU) and Gravy or aromaticity scores for all studied species. We observed that Gravy and aromaticity played a minor role in shaping the variation of codon usage across <em>Citrus</em> species and <em>P. trifoliata</em>. In the case of <em>P. trifoliata</em>, no significant correlation was observed for aromaticity, suggesting that aromaticity has no significant role in shaping the codon usage variation in this genus.</p>                            <h3 scrollto-destination=4608015 id="4608015" class="section-title" data-legacy-id=s3e>3.5. Variations in GC<sub>3</sub> across <em>Citrus</em> species and <em>P. trifoliata</em></h3>
<p>Recent reports suggest that GC<sub>3</sub> composition and GC gradient are acting as major factors along the orientation of transcription in monocots. It has been also suggested that GC composition is vital for understanding chromatin remodelling, gene expression, and recombination.<sup><a reveal-id="DSS039C45" data-open="DSS039C45" class="link link-ref link-reveal xref-bibr">45</a>,<a reveal-id="DSS039C61" data-open="DSS039C61" class="link link-ref link-reveal xref-bibr">61</a></sup> Some studies failed to depict GC gradient along the genes of dicot plants using <em>A. thaliana</em> as a model.<sup><a reveal-id="DSS039C61" data-open="DSS039C61" class="link link-ref link-reveal xref-bibr">61</a></sup> The reason for this failure is that high and low GC<sub>3</sub> genes have opposite gradients along the genes, and when high and low GC<sub>3</sub> genes are clumped together, the effect disappears. In <em>Citrus</em> species and <em>P. trifoliata</em>, unimodal bell-shaped distribution of GC<sub>3</sub>, centred at 0.39, is considered to be a typical mode of GC<sub>3</sub> distribution for dicot species. We selected the top and bottom 5% of genes across each studied <em>Citrus</em> species and <em>P. trifoliata</em>, and strikingly distinct gradients of GC<sub>3</sub> and CG<sub>3</sub> skew for high and low GC<sub>3</sub> genes were revealed. Genes with high GC<sub>3</sub> showed higher level of GC<sub>3</sub> codons in their middle coding regions than in terminal coding regions. In addition, high GC<sub>3</sub> genes have a preference for C over G in the middle coding regions, and this preference was not found in the 3′-coding end ∼100 bp in size (Fig. <a reveal-id="DSS039F4" data-open="DSS039F4" class="link link-reveal link-table xref-fig">4</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. 2</a>). </p>                        <a id="4608017" scrollto-destination="4608017"></a>
<div id="DSS039F4" data-id="DSS039F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss039/2/m_dss03904.jpeg?Expires=2147483647&Signature=4SC0Hu6j260xlgWD4Z2fZK8CHRsp0gvf4ZNIy2vZmyAtAQcqVvbrlqaxxHzGQDlYNHLwr--TEgo8qBHsxRyymMcq6IPeDZ5aw7IzEHxIZCWmzsl3iwoGOq1eg6S-K2PjK5VLBJ8Tmf93L~51YYYw5-ToEydZTfmCRzoE-CsBF0pSYMizSsPV3VJeLVTVsU6Dn1gGbaeYKKSDmhv2mT65AZ4m6yCntHChf7-U6qgw1T9DuOSGSs5kAibclGJDSKKO7sFQ3VSGoj5hXU9f34mc2mP-QATntFEi5yYa1BtyKG-xFthzIqalh1~2XJTR~WLjxqhR6Vhb0qSSkZuXaw8nPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="GC3 gradient across Citrus species and P. trifoliata. Species order: A = C. sinensis; B = C. aurantifolia; C = C. aurantium; D = C. clementina; E = C. clementina × C. tangerina; F = C. jambhiri; G = C. japonica var. margarita; H = C. limettioides; I = C. limonia; J = C. medica; K = C. reshni; L = C. reticulata; M = C. reticulata × C. temple; N = C. sinensis × P. trifoliata; O = C. sunki; P = P. trifoliata; Q = C. unshiu; R = C. paradisi; and S = C. paradisi × P. trifoliata. These plots show gradients of GC3 for 5% of GC3-rich and GC3-poor genes for all analysed genomes. Gradients from 5′ and 3′ ends of the CDS are shown in the same plot, separated by a vertical line. For all genomes, GC3-rich genes become more GC3 rich towards the middle of the CDS." path-from-xml="dss03904.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4608017/dss03904.jpeg" path-from-xml="dss03904.jpeg" target="_blank">View large</a><a section="4608017" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss039/2/dss03904.jpeg?Expires=2147483647&Signature=WTClZEp3SrqL4Wch3EImW2sAPhXzZ1NRbVYkLHbjqi30mj1VV847RSaUtAkSm~HhqKLayoulaabmTEFbjJPZ8D0s9VHF7Ze6thWbuNffj7SvmnjXziftGuA-aupuQpbrd5bcZpWN38Fi0XPqjDpK2bP25mrESLgXxKwrVifmatpJeWZofcTeDbTKQdJfnzYQjoUoGyM~CzuEgclfbKl8ZUEJw36lrGE7gzX0-5XbUkBk~~ceKdqHW3m2PpuFqb97qKUydrn-zcuNrq00hOLyeU6pRC~~Ufz39lvbgVB5zh20eTXJq3sV3isstgycMboTJ4MIVjwLbfX2IrYmPXXyxQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4608017&ar=333376&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03904.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>GC<sub>3</sub> gradient across <em>Citrus</em> species and <em>P. trifoliata</em>. Species order: <em>A</em> = <em>C. sinensis</em>; <em>B</em> = <em>C. aurantifolia</em>; <em>C</em> = <em>C. aurantium</em>; <em>D</em> = <em>C. clementina</em>; <em>E</em> = <em>C. clementina</em> × <em>C. tangerina</em>; <em>F</em> = <em>C. jambhiri</em>; <em>G</em> = <em>C. japonica</em> var. <em>margarita</em>; <em>H</em> = <em>C. limettioides</em>; <em>I</em> = <em>C. limonia</em>; <em>J</em> = <em>C. medica</em>; <em>K</em> = <em>C. reshni</em>; <em>L</em> = <em>C. reticulata</em>; <em>M</em> = <em>C. reticulata</em> × <em>C. temple</em>; <em>N</em> = <em>C. sinensis</em> × <em>P. trifoliata</em>; <em>O</em> = <em>C. sunki</em>; <em>P</em> = <em>P. trifoliata</em>; <em>Q</em> = <em>C. unshiu</em>; <em>R</em> = <em>C. paradisi</em>; and <em>S</em> = <em>C. paradisi</em> × <em>P. trifoliata.</em> These plots show gradients of GC<sub>3</sub> for 5% of GC<sub>3</sub>-rich and GC<sub>3</sub>-poor genes for all analysed genomes. Gradients from 5′ and 3′ ends of the CDS are shown in the same plot, separated by a vertical line. For all genomes, GC<sub>3</sub>-rich genes become more GC<sub>3</sub> rich towards the middle of the CDS.</p></div></div><div content-id="DSS039F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss039/2/m_dss03904.jpeg?Expires=2147483647&Signature=4SC0Hu6j260xlgWD4Z2fZK8CHRsp0gvf4ZNIy2vZmyAtAQcqVvbrlqaxxHzGQDlYNHLwr--TEgo8qBHsxRyymMcq6IPeDZ5aw7IzEHxIZCWmzsl3iwoGOq1eg6S-K2PjK5VLBJ8Tmf93L~51YYYw5-ToEydZTfmCRzoE-CsBF0pSYMizSsPV3VJeLVTVsU6Dn1gGbaeYKKSDmhv2mT65AZ4m6yCntHChf7-U6qgw1T9DuOSGSs5kAibclGJDSKKO7sFQ3VSGoj5hXU9f34mc2mP-QATntFEi5yYa1BtyKG-xFthzIqalh1~2XJTR~WLjxqhR6Vhb0qSSkZuXaw8nPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="GC3 gradient across Citrus species and P. trifoliata. Species order: A = C. sinensis; B = C. aurantifolia; C = C. aurantium; D = C. clementina; E = C. clementina × C. tangerina; F = C. jambhiri; G = C. japonica var. margarita; H = C. limettioides; I = C. limonia; J = C. medica; K = C. reshni; L = C. reticulata; M = C. reticulata × C. temple; N = C. sinensis × P. trifoliata; O = C. sunki; P = P. trifoliata; Q = C. unshiu; R = C. paradisi; and S = C. paradisi × P. trifoliata. These plots show gradients of GC3 for 5% of GC3-rich and GC3-poor genes for all analysed genomes. Gradients from 5′ and 3′ ends of the CDS are shown in the same plot, separated by a vertical line. For all genomes, GC3-rich genes become more GC3 rich towards the middle of the CDS." path-from-xml="dss03904.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4608017/dss03904.jpeg" path-from-xml="dss03904.jpeg" target="_blank">View large</a><a section="4608017" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss039/2/dss03904.jpeg?Expires=2147483647&Signature=WTClZEp3SrqL4Wch3EImW2sAPhXzZ1NRbVYkLHbjqi30mj1VV847RSaUtAkSm~HhqKLayoulaabmTEFbjJPZ8D0s9VHF7Ze6thWbuNffj7SvmnjXziftGuA-aupuQpbrd5bcZpWN38Fi0XPqjDpK2bP25mrESLgXxKwrVifmatpJeWZofcTeDbTKQdJfnzYQjoUoGyM~CzuEgclfbKl8ZUEJw36lrGE7gzX0-5XbUkBk~~ceKdqHW3m2PpuFqb97qKUydrn-zcuNrq00hOLyeU6pRC~~Ufz39lvbgVB5zh20eTXJq3sV3isstgycMboTJ4MIVjwLbfX2IrYmPXXyxQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4608017&ar=333376&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03904.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>GC<sub>3</sub> gradient across <em>Citrus</em> species and <em>P. trifoliata</em>. Species order: <em>A</em> = <em>C. sinensis</em>; <em>B</em> = <em>C. aurantifolia</em>; <em>C</em> = <em>C. aurantium</em>; <em>D</em> = <em>C. clementina</em>; <em>E</em> = <em>C. clementina</em> × <em>C. tangerina</em>; <em>F</em> = <em>C. jambhiri</em>; <em>G</em> = <em>C. japonica</em> var. <em>margarita</em>; <em>H</em> = <em>C. limettioides</em>; <em>I</em> = <em>C. limonia</em>; <em>J</em> = <em>C. medica</em>; <em>K</em> = <em>C. reshni</em>; <em>L</em> = <em>C. reticulata</em>; <em>M</em> = <em>C. reticulata</em> × <em>C. temple</em>; <em>N</em> = <em>C. sinensis</em> × <em>P. trifoliata</em>; <em>O</em> = <em>C. sunki</em>; <em>P</em> = <em>P. trifoliata</em>; <em>Q</em> = <em>C. unshiu</em>; <em>R</em> = <em>C. paradisi</em>; and <em>S</em> = <em>C. paradisi</em> × <em>P. trifoliata.</em> These plots show gradients of GC<sub>3</sub> for 5% of GC<sub>3</sub>-rich and GC<sub>3</sub>-poor genes for all analysed genomes. Gradients from 5′ and 3′ ends of the CDS are shown in the same plot, separated by a vertical line. For all genomes, GC<sub>3</sub>-rich genes become more GC<sub>3</sub> rich towards the middle of the CDS.</p></div></div><p>It was previously reported that stress-related genes in grasses are GC3-rich.<sup><a reveal-id="DSS039C45" data-open="DSS039C45" class="link link-ref link-reveal xref-bibr">45</a></sup> We stratified all <em>Citrus</em> species and <em>P. trifoliata</em> genes into three groups by GC<sub>3</sub>: rich (top 5%), poor (bottom 5%), and medium (middle 90%). We observed that among the four major economically important <em>Citrus</em> species, in <em>C. reticulata</em> (1718), <em>C. clementina</em> (1522), <em>and P. trifoliata</em> (951), most of the GC<sub>3</sub>-rich genes were related to stress in comparison to the total number of observed stress-related genes, but in the case of <em>C. sinensis</em>, it was observed that a high number of medium GC<sub>3</sub> genes (1857) were also abundant in stress-related genes. The relative abundance of the medium GC<sub>3</sub> genes in <em>C. sinensis</em> may be due to the time-course adaptability of this species during the period of evolution, suggesting an adaptive evolution towards the stress.</p><p>A high abundance of stress-related genes in <em>C. reticulata</em> and <em>P. trifoliata</em> may be closely related to their high stress-tolerance trait and especially to <em>P. trifoliata</em> that has the highest cold-resistant character among <em>Citrus</em> species and its wild relatives. The occurrence of the high GC<sub>3</sub> genes in all these species suggests an association between DNA methylation and GC<sub>3</sub> as it has been previously suggested that high GC<sub>3</sub> composition has been influenced by the GC mismatch-repair mechanism that is dominant in stress-associated genes as the absence of this positive bias may lead to the loss of the recombination repair mechanism and could be detrimental to the plant adaptation in the evolving stress conditions. Genomic regions under higher selective pressure are more frequently recombining, and as a result relative increase in GC<sub>3</sub> content can be observed.<sup><a reveal-id="DSS039C45" data-open="DSS039C45" class="link link-ref link-reveal xref-bibr">45</a></sup> As shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table 2</a> and Fig. <a reveal-id="DSS039F5" data-open="DSS039F5" class="link link-reveal link-table xref-fig">5</a>, in all <em>Citrus</em> species and <em>P. trifoliata</em>, the ratio of stress to non-stress-related genes in GC<sub>3</sub>-rich group was elevated in comparison to the GC<sub>3</sub>-medium group and depleted in the GC<sub>3</sub>-poor group. It is worth noting that GC<sub>3</sub>-poor group has fewer genes functionally described as stress-related when compared with the GC<sub>3</sub>-rich group. </p>                        <a id="4608020" scrollto-destination="4608020"></a>
<div id="DSS039F5" data-id="DSS039F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss039/2/m_dss03905.jpeg?Expires=2147483647&Signature=xKQ-vtKciF8A-Huc2m3I5iy7LGyFLRrgtfWBmcYpZ8klMJAuVU-GbjAFPKfw55RlFvMCb2KZwnyS6U~8Xk5OSMwISzt9erkdnwmi7ONWekg9MukLvtfdtuLiGKs5qnvCddfQyU7p7igVNLLpbM9GarghPr6hUqArYw6GZysIKsVOJ9dyjB3ulzxg1G-wDS~OZSAEFZMU2FfLhZFVu7hsRACmCBjrRohdRsNOV4045rK0Yecx--bicLqX-R5z0b-x-3-6YTAtciMr0MPdAWo-jNQbaAKleteB~HRfypU2d~vpvKfBJ1upKcneL5itH1tzOABpURYlUdJglhL0anRXWg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of stress-related genes in Citrus species and P. trifoliata according to the GO of Arabidopsis." path-from-xml="dss03905.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4608020/dss03905.jpeg" path-from-xml="dss03905.jpeg" target="_blank">View large</a><a section="4608020" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss039/2/dss03905.jpeg?Expires=2147483647&Signature=vd1b5ge0zzyPBS1SXtvsZ421A1V-R2TOdjB1vFs3ay-GlCHIxMCeyoN74v36tGNH0UEEAhJxnF5kBSgpE1yMUM0BKLWTggl9vEmQH0XuJfh678e2oLo0d1vxhN00Pm1r2K86DNkoLNLOFiNm4XDNr-pll6JCcKylJFVgCAsZYyZfBfvDd9GNiHtjFb-~EwA9eq7QN2ON9ByuXFR46Y~7g5EmN9ZrjRAfXeDQINt8hkG1dewQFYvCxwuxK~XfvNNPL4uIqbC-4iTmFlX6x6u79SXDq5Me6kqavE5eZM0veci1Eqx-TCs1ivU7X~bZEuqPaaUuBh1gPqgs-6vkq05NCg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4608020&ar=333376&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03905.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of stress-related genes in <em>Citrus</em> species and <em>P. trifoliata</em> according to the GO of <em>Arabidopsis</em>.</p></div></div><div content-id="DSS039F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss039/2/m_dss03905.jpeg?Expires=2147483647&Signature=xKQ-vtKciF8A-Huc2m3I5iy7LGyFLRrgtfWBmcYpZ8klMJAuVU-GbjAFPKfw55RlFvMCb2KZwnyS6U~8Xk5OSMwISzt9erkdnwmi7ONWekg9MukLvtfdtuLiGKs5qnvCddfQyU7p7igVNLLpbM9GarghPr6hUqArYw6GZysIKsVOJ9dyjB3ulzxg1G-wDS~OZSAEFZMU2FfLhZFVu7hsRACmCBjrRohdRsNOV4045rK0Yecx--bicLqX-R5z0b-x-3-6YTAtciMr0MPdAWo-jNQbaAKleteB~HRfypU2d~vpvKfBJ1upKcneL5itH1tzOABpURYlUdJglhL0anRXWg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of stress-related genes in Citrus species and P. trifoliata according to the GO of Arabidopsis." path-from-xml="dss03905.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4608020/dss03905.jpeg" path-from-xml="dss03905.jpeg" target="_blank">View large</a><a section="4608020" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss039/2/dss03905.jpeg?Expires=2147483647&Signature=vd1b5ge0zzyPBS1SXtvsZ421A1V-R2TOdjB1vFs3ay-GlCHIxMCeyoN74v36tGNH0UEEAhJxnF5kBSgpE1yMUM0BKLWTggl9vEmQH0XuJfh678e2oLo0d1vxhN00Pm1r2K86DNkoLNLOFiNm4XDNr-pll6JCcKylJFVgCAsZYyZfBfvDd9GNiHtjFb-~EwA9eq7QN2ON9ByuXFR46Y~7g5EmN9ZrjRAfXeDQINt8hkG1dewQFYvCxwuxK~XfvNNPL4uIqbC-4iTmFlX6x6u79SXDq5Me6kqavE5eZM0veci1Eqx-TCs1ivU7X~bZEuqPaaUuBh1gPqgs-6vkq05NCg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4608020&ar=333376&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03905.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of stress-related genes in <em>Citrus</em> species and <em>P. trifoliata</em> according to the GO of <em>Arabidopsis</em>.</p></div></div><p>In all the studied species, GC<sub>3</sub>-rich and -poor genes have different trends from 5′ to 3′ end of the genes. GC<sub>3</sub>-rich genes become even more GC<sub>3</sub> rich, and GC<sub>3</sub>-poor genes become more GC<sub>3</sub> poor. Firstly, the GC<sub>3</sub>-rich group has more stress related genes than the GC<sub>3</sub>-poor group. Secondly, the GC<sub>3</sub>-rich group has a positive gradient of GC<sub>3</sub> from 5′ and 3′ flanks to the middle portion of the CDS. The low-GC<sub>3</sub> group has a negative gradient of GC<sub>3</sub> from 5′ and 3′ flanks to the middle portion of the CDS. Thirdly, rich- and poor-GC<sub>3</sub> groups showed different CG<sub>3</sub> skew trends along the CDS: GC<sub>3</sub>-rich genes favour C<sub>3</sub> over G<sub>3</sub>, and this preference is most pronounced in first 150 codons, whereas GC<sub>3</sub>-poor genes favour G<sub>3</sub> over C<sub>3</sub> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. 2</a>). It was suggested that transcriptional and translational optimization of stress-related and tissue-specific genes is the major force responsible for maintaining high GC<sub>3</sub> content.<sup><a reveal-id="DSS039C46" data-open="DSS039C46" class="link link-ref link-reveal xref-bibr">46</a></sup> The replacement of the AT pair with the GC pair at the third codon position enhances the transcriptional activity that in turn enhances the array of ribosomal-binding proteins. Stayssman et al.<sup><a reveal-id="DSS039C62" data-open="DSS039C62" class="link link-ref link-reveal xref-bibr">62</a></sup> provide a strong positive correlation between the methylation of internal unmethylated regions and expression of the host gene and postulated that the genes with high GC<sub>3</sub> provide more targets for methylation. Genes involved in response to various stresses need to produce a protein with a faster rate as a response to external stimulus. This results in shortened length of transcript and preference for G and C (and especially, C) in the third position of the codon (to avoid abortive transcription and ribosome congestion). In addition, high GC<sub>3</sub> genes have more methylation targets that allow for fine-tuning of transcriptional regulation.<sup><a reveal-id="DSS039C58" data-open="DSS039C58" class="link link-ref link-reveal xref-bibr">58</a></sup></p>                            <h3 scrollto-destination=4608022 id="4608022" class="section-title" data-legacy-id=s3f>3.6. Evolution and coevolution of nucleotide and amino acid composition</h3>
<p>We analysed the relationship between amino acid sequence divergence and variation in GC<sub>3</sub>, to test the effect of evolutionary divergence on codon usage. We observed that there is an overall positive correlation (0.43) between relative change in GC<sub>3</sub> and normalized Hamming distance (Fig. <a reveal-id="DSS039F6" data-open="DSS039F6" class="link link-reveal link-table xref-fig">6</a>). This was expected because more diverse amino acid sequences are likely to have more diverse nucleotide sequences. The trend is not uniform: for the genes with GC<sub>3</sub> in the range between 0.5 and 0.6, the correlation is 0.596 and it drops to 0.32 for GC<sub>3</sub> above 0.8 or below 0.4. There are some pairs of organisms that have negative correlation between GC<sub>3</sub> and Hamming distance (<em>C. jambhiri</em>: <em>C. limettioides</em>, <em>C. limonia</em>: <em>C. sinensis</em> × <em>P. trifoliata</em>, <em>C. aurantiifolia</em>: <em>C. jambhiri</em>, <em>C. jambhiri</em>: <em>C. paradisi</em>, <em>C. aurantiifolia</em>: <em>C. paradisi</em>, <em>C. jambhiri</em>: <em>P. trifoliata</em>, <em>C. medica</em><span class="underline">:</span><em>C. sinensis</em> × <em>P. trifoliata</em>, <em>C. jambhiri</em>: <em>C. unshiu</em>, <em>C. jambhiri</em>: <em>C. reshni</em>, <em>C. japonica</em> var. <em>margarita</em>: <em>C. paradisi</em>, <em>C. aurantiifolia</em>: <em>C. clementina</em> × <em>C. tangerina</em>, <em>C. aurantium</em>: <em>C. jambhiri</em>, <em>C. clementina</em> × <em>C. tangerina</em>: <em>C. jambhiri</em>). Note that in the list of negative correlations, <em>C. jambhiri</em> occurs eight times. </p>                        <a id="4608024" scrollto-destination="4608024"></a>
<div id="DSS039F6" data-id="DSS039F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss039/2/m_dss03906.jpeg?Expires=2147483647&Signature=uGY94ev4yWd1-U-d8ykST1-IabJ3nxx1PKuA4~ipOUDJEkMKVkyNoqBHW9IjPwZCb7LkFK7I9Yf9lyG0Hl-dH-etLOlcv4iO-PxZXINBkyTTcZk88NwnYQITvfFvYdIdDnigMlt9FpIKFoWqtrVvZ2WiF1zUE2AqjmdEP5gfokzJx61g6Vv-kXth~ytu8eUu--V5~tREdM7WAWWvHw1CqINzKf~MijGiP3FrzXYyB5nbClBhbbV~89eKDGzXiXUpwtVWbmhHtFOBwbvG8q6IMM0X-HQ0FTmlrXLJ5dQ0XMaH4EjmE5hCecDG2cM2OPEY~09pa62i3YCv6Ef33WqDyw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Hamming distance versus change in GC3 visualization across Citrus species and P. trifoliata. Relative difference GC3 composition between genes A and B, defined as GC3 (A)-GC3 (B)/[GC3 (A) + GC3 (B)] is positively correlated with the Hamming distance between corresponding protein sequences, calculated as 1–[I (AB) + I (BA)]/[I (AA) + I (BB)], where I (AB) is the number of identities in alignment of protein A to protein B. The resulting Hamming distance was rounded to 1 DP, and an average difference in GC3 was computed for each category." path-from-xml="dss03906.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4608024/dss03906.jpeg" path-from-xml="dss03906.jpeg" target="_blank">View large</a><a section="4608024" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss039/2/dss03906.jpeg?Expires=2147483647&Signature=nuToS3RTlcG4p2Kx-tdnZqGJa~ns-KBR3EXWVBIHqYu9YNjcCCo7fALwsFrFYzrmHcxTKtZI4iZJN~5yZ72UtSLxcPojnqeQzaZQ7EeNSMZ7C1LPWbFdO4zbwgqIyq352Zu5mChLAmxmv5ZkaTomU9Qm98doL0WtPHe-wDT1upWMTu8sSJphR-cJBZvTl2LAR3~lAN6TDSGGdOhzqmfkXEiLr6-bfAPzaGvHoHruGH5QzCx8AeS0eYX5Cs~m7fGile6g2iWo2vECHZwv7qy4tA3aVYC09CL16cm0-rt8lyOr21Miu9~JrUOMiug3jloHmOvFjhStlNN7ubADvrEC-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4608024&ar=333376&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03906.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Hamming distance versus change in GC<sub>3</sub> visualization across <em>Citrus</em> species and <em>P. trifoliata</em>. Relative difference GC<sub>3</sub> composition between genes A and B, defined as GC<sub>3</sub> (A)-GC<sub>3</sub> (B)/[GC<sub>3</sub> (A) + GC<sub>3</sub> (B)] is positively correlated with the Hamming distance between corresponding protein sequences, calculated as 1–[I (AB) + I (BA)]/[I (AA) + I (BB)], where I (AB) is the number of identities in alignment of protein A to protein B. The resulting Hamming distance was rounded to 1 DP, and an average difference in GC<sub>3</sub> was computed for each category.</p></div></div><div content-id="DSS039F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss039/2/m_dss03906.jpeg?Expires=2147483647&Signature=uGY94ev4yWd1-U-d8ykST1-IabJ3nxx1PKuA4~ipOUDJEkMKVkyNoqBHW9IjPwZCb7LkFK7I9Yf9lyG0Hl-dH-etLOlcv4iO-PxZXINBkyTTcZk88NwnYQITvfFvYdIdDnigMlt9FpIKFoWqtrVvZ2WiF1zUE2AqjmdEP5gfokzJx61g6Vv-kXth~ytu8eUu--V5~tREdM7WAWWvHw1CqINzKf~MijGiP3FrzXYyB5nbClBhbbV~89eKDGzXiXUpwtVWbmhHtFOBwbvG8q6IMM0X-HQ0FTmlrXLJ5dQ0XMaH4EjmE5hCecDG2cM2OPEY~09pa62i3YCv6Ef33WqDyw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Hamming distance versus change in GC3 visualization across Citrus species and P. trifoliata. Relative difference GC3 composition between genes A and B, defined as GC3 (A)-GC3 (B)/[GC3 (A) + GC3 (B)] is positively correlated with the Hamming distance between corresponding protein sequences, calculated as 1–[I (AB) + I (BA)]/[I (AA) + I (BB)], where I (AB) is the number of identities in alignment of protein A to protein B. The resulting Hamming distance was rounded to 1 DP, and an average difference in GC3 was computed for each category." path-from-xml="dss03906.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4608024/dss03906.jpeg" path-from-xml="dss03906.jpeg" target="_blank">View large</a><a section="4608024" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss039/2/dss03906.jpeg?Expires=2147483647&Signature=nuToS3RTlcG4p2Kx-tdnZqGJa~ns-KBR3EXWVBIHqYu9YNjcCCo7fALwsFrFYzrmHcxTKtZI4iZJN~5yZ72UtSLxcPojnqeQzaZQ7EeNSMZ7C1LPWbFdO4zbwgqIyq352Zu5mChLAmxmv5ZkaTomU9Qm98doL0WtPHe-wDT1upWMTu8sSJphR-cJBZvTl2LAR3~lAN6TDSGGdOhzqmfkXEiLr6-bfAPzaGvHoHruGH5QzCx8AeS0eYX5Cs~m7fGile6g2iWo2vECHZwv7qy4tA3aVYC09CL16cm0-rt8lyOr21Miu9~JrUOMiug3jloHmOvFjhStlNN7ubADvrEC-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4608024&ar=333376&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss03906.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Hamming distance versus change in GC<sub>3</sub> visualization across <em>Citrus</em> species and <em>P. trifoliata</em>. Relative difference GC<sub>3</sub> composition between genes A and B, defined as GC<sub>3</sub> (A)-GC<sub>3</sub> (B)/[GC<sub>3</sub> (A) + GC<sub>3</sub> (B)] is positively correlated with the Hamming distance between corresponding protein sequences, calculated as 1–[I (AB) + I (BA)]/[I (AA) + I (BB)], where I (AB) is the number of identities in alignment of protein A to protein B. The resulting Hamming distance was rounded to 1 DP, and an average difference in GC<sub>3</sub> was computed for each category.</p></div></div><p>Genes that have low GC<sub>3</sub> content in <em>C. jambhiri</em> (&lt;0.4) tend to have negative correlation between difference in GC<sub>3</sub> and Hamming distance, and genes with high GC<sub>3</sub> content (&gt;0.6) are slightly positively correlated. This atypical behaviour can be explained by larger evolutionary distance between <em>C. jambhiri</em> and other species analysed in the present investigation. The highest correlation was observed between <em>C. japonica</em> var. <em>margarita</em> and <em>C. limonia</em> (<em>ρ</em> = 0.86, number of orthologous pairs <em>N</em> = 12, <em>t</em>-statistics <em>t</em> = 5.22, <em>P</em>-value = 0.0002). <em>C. japonica</em> var. <em>margarita</em> and <em>C. limonia</em> are evolutionary distant, and the number of orthologous pairs is only <em>N</em> = 12. Hence, a high value of correlation coefficient may be a result of an evolutionary pressure to conserve codon usage for a selected subset of conserved proteins. We observed that pairs of species having positive correlation between GC<sub>3</sub> and Hamming distance are enriched in translation, transport, proteolysis, photorespiration, and genes involved in photosynthesis; pairs of species with negative correlation are enriched in stress-response genes.</p><p>Similar patterns of hierarchal clustering were depicted; when we clustered the species based on co-orthologous genes using genome-predicted and frame-corrected reconstructed proteins of the <em>Citrus</em> species and <em>P. trifoliata</em>. The phylogenetic clades were rerooted using <em>P. trifoliata</em> as an outgroup, and it was observed that <em>C. sinensis</em>, <em>C. clementina</em>, and <em>C. reticulata</em> all belong to the same clade, which are in accordance with the previous reports using major intrinsic proteins (XIP subfamily of aquaporins) and further support the conserved homology between these two species.<sup><a reveal-id="DSS039C63" data-open="DSS039C63" class="link link-ref link-reveal xref-bibr">63</a></sup> These supportive views suggest that the identified frame-corrected coding regions are trustworthy and accurate to be used for the predictions in the species, with no genome information available till now (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. 3</a>A and B).</p><p>Because a nucleotide bias can lead to an overall bias in amino acid composition of proteins, it is possible that a genome with nucleotide bias may have introduced atypical amino acid substitutions in its proteome. Hence, AT-rich coding sequences would encode proteins rich in FYMINK amino acids (Phenylalanine, Tyrosine, Methionine, Isoleucine, Asparagine, and Lysine), whereas GC-rich coding sequences would produce proteins containing high levels of GARP amino acids (Glycine, Alanine, Arginine, and Proline). Because <em>Citrus</em> species and <em>P. trifoliata</em> are AT-rich, we found higher proportion of FYMINK [(FYMINK and CDS<sub>GC</sub> (<em>y</em> = 4.1431<em>x</em> + 22.341, <em>R</em><sup>2</sup> = 0.9099); GARP and CDS<sub>GC</sub> (<em>y</em> = 3.0687<em>x</em> + 45.414, <em>R</em><sup>2</sup> = 0.7803); <em>P</em>-value &lt; 10<sup>−9</sup>)].</p><p>Because the third synonymous position does not influence the protein sequence, we computed the correlation coefficient between the FYMINK and GARP parameters and the GC<sub>3</sub>. This correlation is an important factor in describing nucleotide bias at synonymous and non-synonymous sites. We observed a high significant correlation between nucleotide composition and amino acid composition at the third synonymous position [(FYMINK and CDS<sub>GC3</sub> (<em>y</em> = 10.862<em>x</em> + 142.4, <em>R</em><sup>2</sup> = 0.9296); GARP and CDS<sub>GC3</sub> (<em>y</em> = 9.842<em>x</em> + 212.15, <em>R</em><sup>2</sup> = 0.7495)], suggesting that nucleotide bias has an influence on amino acid composition in <em>Citrus</em> species and <em>P. trifoliata</em>.</p><p>In summary, we found prevalence of A- or T-ending codons with an exception for Lysine and Arginine in <em>P. trifoliata</em>. We suggest that although the patterns of optimal codons were not conserved, two codons (GCT and GGT) were found to be conserved across all the <em>Citrus</em> species and <em>P. trifoliata</em>. We analysed GC<sub>3</sub>-rich and -poor genes and their association with stress, and our results provided novel insights into stress evolution of stress adaptation in <em>Citrus</em> species and <em>P. trifoliata</em> in accordance with the GC<sub>3</sub> biology. This research is critical for all <em>Citrus</em> species, as it might facilitate understanding of genome dynamics and evolution in <em>Citrus</em> and <em>P. trifoliata</em>, transformation of interested target genes, designing multitargeting gene systems, and ultimately genetically improving these important fruit crops.</p>                            <h2 scrollto-destination=4608030 id="4608030" class="section-title" data-legacy-id=s4>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at</a><a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=4608032 id="4608032" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This research was financially supported by the Ministry of Science and Technology of China (nos. 2011AA100205, 2011CB100606), the National NSF of China, and the Ministry of Agriculture of China (no. 200903044).</p>                            <h2 scrollto-destination=4608034 id="4608034" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The authors thank Dr Kenta Nakai (the handling editor) and the two anonymous reviewers for helpful and constructive comments and Dr Yuepeng Han, Wuhan Botanic Garden (CAS) for critical reading of the manuscript. G.S. thanks the computational support provided from Research and Innovation Center, CRI-FEM, IASMA, Italy. T.V.T. thanks the Research Investment Scheme, University of Glamorgan, UK.</p>                            <h2 scrollto-destination=4608036 id="4608036" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSS039C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Serres-Giardi</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Belkhir</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">David</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Glémin</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Patterns and evolution of nucleotide landscapes in seed plants</div>, <div class="source">Plant Cell</div> , <div class="year">2012</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">1379</div>-<div class="lpage">97</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Patterns%20and%20evolution%20of%20nucleotide%20landscapes%20in%20seed%20plants&amp;author=L.%20Serres-Giardi&amp;author=K.%20Belkhir&amp;author=J.%20David&amp;author=S.%20Gl%C3%A9min&amp;publication_year=2012&amp;journal=Plant%20Cell&amp;volume=24&amp;pages=1379-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.111.093674" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.111.093674" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.111.093674"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22492812" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hershberg</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Petrov</div> <div class="given-names">D.A.</div></div>. </div><div class="article-title">General rules for optimal codon choice</div>, <div class="source">PLoS Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e1000556</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=General%20rules%20for%20optimal%20codon%20choice&amp;author=R.%20Hershberg&amp;author=D.A.%20Petrov&amp;publication_year=2009&amp;journal=PLoS%20Genet.&amp;volume=5&amp;pages=e1000556" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000556" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000556" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000556"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19593368" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grantham</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Gautier</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Gouy</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Jacobzone</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Mercier</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Codon catalog usage is a genome strategy modulated for gene expressivity</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1981</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">43</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20catalog%20usage%20is%20a%20genome%20strategy%20modulated%20for%20gene%20expressivity&amp;author=R.%20Grantham&amp;author=C.%20Gautier&amp;author=M.%20Gouy&amp;author=M.%20Jacobzone&amp;author=R.%20Mercier&amp;publication_year=1981&amp;journal=Nucleic%20Acids%20Res.&amp;volume=9&amp;pages=43-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/9.1.213-b" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F9.1.213-b" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F9.1.213-b"> </span></div> </div></div></div></div></div><div content-id="DSS039C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Akashi</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Gene expression and molecular evolution</div>, <div class="source">Curr. Opin. Genet. Dev.</div> , <div class="year">2001</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">660</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20expression%20and%20molecular%20evolution&amp;author=H.%20Akashi&amp;publication_year=2001&amp;journal=Curr.%20Opin.%20Genet.%20Dev.&amp;volume=11&amp;pages=660-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0959-437X(00)00250-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0959-437X(00)00250-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0959-437X(00)00250-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11682310" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Aragonès</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Guix</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ribes</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Bosch</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Pintó</div> <div class="given-names">R.M.</div></div>. </div><div class="article-title">Fine-tuning translation kinetics selection as the driving force of codon usage bias in the hepatitis A virus capsid</div>, <div class="source">PLoS Pathol.</div> , <div class="year">2010</div>, vol. <div class="volume">6</div> pg. <div class="fpage">e1000797</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fine-tuning%20translation%20kinetics%20selection%20as%20the%20driving%20force%20of%20codon%20usage%20bias%20in%20the%20hepatitis%20A%20virus%20capsid&amp;author=L.%20Aragon%C3%A8s&amp;author=S.%20Guix&amp;author=E.%20Ribes&amp;author=A.%20Bosch&amp;author=R.M.%20Pint%C3%B3&amp;publication_year=2010&amp;journal=PLoS%20Pathol.&amp;volume=6&amp;pages=e1000797" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.ppat.1000797" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.ppat.1000797" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.ppat.1000797"> </span></div> </div></div></div></div></div><div content-id="DSS039C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Medigue</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Rouxel</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Vigier</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Henaut</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Danchin</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Evidence for horizontal gene transfer in <em>Escherichia coli</em> speciation</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1991</div>, vol. <div class="volume">222</div> (pg. <div class="fpage">851</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20horizontal%20gene%20transfer%20in%20Escherichia%20coli%20speciation&amp;author=C.%20Medigue&amp;author=T.%20Rouxel&amp;author=P.%20Vigier&amp;author=A.%20Henaut&amp;author=A.%20Danchin&amp;publication_year=1991&amp;journal=J.%20Mol.%20Biol.&amp;volume=222&amp;pages=851-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(91)90575-Q" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(91)90575-Q" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(91)90575-Q"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1762151" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pascal</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">M'edigue</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Danchin</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Universal biases in protein composition of model prokaryotes</div>, <div class="source">Proteins</div> , <div class="year">2005</div>, vol. <div class="volume">60</div> (pg. <div class="fpage">27</div>-<div class="lpage">35</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Universal%20biases%20in%20protein%20composition%20of%20model%20prokaryotes&amp;author=G.%20Pascal&amp;author=C.%20M'edigue&amp;author=A.%20Danchin&amp;publication_year=2005&amp;journal=Proteins&amp;volume=60&amp;pages=27-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/prot.20475" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fprot.20475" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fprot.20475"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15849754" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Duret</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Evolution of synonymous codon usage in metazoans</div>, <div class="source">Curr. Opin. Genet. Dev.</div> , <div class="year">2002</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">640</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20synonymous%20codon%20usage%20in%20metazoans&amp;author=L.%20Duret&amp;publication_year=2002&amp;journal=Curr.%20Opin.%20Genet.%20Dev.&amp;volume=12&amp;pages=640-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0959-437X(02)00353-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0959-437X(02)00353-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0959-437X(02)00353-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12433576" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bulmer</div> <div class="given-names">M.</div></div>. </div><div class="article-title">The selection-mutation drift theory of synonymous codon usage</div>, <div class="source">Genetics</div> , <div class="year">1991</div>, vol. <div class="volume">129</div> (pg. <div class="fpage">897</div>-<div class="lpage">907</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20selection-mutation%20drift%20theory%20of%20synonymous%20codon%20usage&amp;author=M.%20Bulmer&amp;publication_year=1991&amp;journal=Genetics&amp;volume=129&amp;pages=897-907" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1752426" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharp</div> <div class="given-names">P.M.</div></div>,  <div class="name"><div class="surname">Matassi</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Codon usage and genome evolution</div>, <div class="source">Curr. Opin. Genet. Dev.</div> , <div class="year">1994</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">851</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20and%20genome%20evolution&amp;author=P.M.%20Sharp&amp;author=G.%20Matassi&amp;publication_year=1994&amp;journal=Curr.%20Opin.%20Genet.%20Dev.&amp;volume=4&amp;pages=851-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0959-437X(94)90070-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0959-437X(94)90070-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0959-437X(94)90070-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7888755" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Akashi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Eyre-Walker</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Translational selection and molecular evolution</div>, <div class="source">Curr. Opin. Genet. Dev.</div> , <div class="year">1998</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">688</div>-<div class="lpage">93</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translational%20selection%20and%20molecular%20evolution&amp;author=H.%20Akashi&amp;author=A.%20Eyre-Walker&amp;publication_year=1998&amp;journal=Curr.%20Opin.%20Genet.%20Dev.&amp;volume=8&amp;pages=688-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0959-437X(98)80038-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0959-437X(98)80038-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0959-437X(98)80038-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9914211" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gupta</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Ghosh</div> <div class="given-names">T.C.</div></div>. </div><div class="article-title">Gene expressivity is the main factor in dictating the codon usage variation among the genes in <em>Pseudomonas aeruginosa</em></div>, <div class="source">Gene</div> , <div class="year">2001</div>, vol. <div class="volume">273</div> (pg. <div class="fpage">63</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20expressivity%20is%20the%20main%20factor%20in%20dictating%20the%20codon%20usage%20variation%20among%20the%20genes%20in%20Pseudomonas%20aeruginosa&amp;author=S.K.%20Gupta&amp;author=T.C.%20Ghosh&amp;publication_year=2001&amp;journal=Gene&amp;volume=273&amp;pages=63-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(01)00576-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(01)00576-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(01)00576-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11483361" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hershberg</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Petrov</div> <div class="given-names">D.A.</div></div>. </div><div class="article-title">Selection on codon bias</div>, <div class="source">Annu. Rev. Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">287</div>-<div class="lpage">99</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selection%20on%20codon%20bias&amp;author=R.%20Hershberg&amp;author=D.A.%20Petrov&amp;publication_year=2008&amp;journal=Annu.%20Rev.%20Genet.&amp;volume=42&amp;pages=287-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.genet.42.110807.091442" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genet.42.110807.091442" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.genet.42.110807.091442"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18983258" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Harrison</div> <div class="given-names">R.J.</div></div>,  <div class="name"><div class="surname">Charlesworth</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Biased gene conversion affects patterns of codon usage and amino acid usage in the <em>Saccharomyces</em> sensu stricto group of yeasts</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2011</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">117</div>-<div class="lpage">29</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biased%20gene%20conversion%20affects%20patterns%20of%20codon%20usage%20and%20amino%20acid%20usage%20in%20the%20Saccharomyces%20sensu%20stricto%20group%20of%20yeasts&amp;author=R.J.%20Harrison&amp;author=B.%20Charlesworth&amp;publication_year=2011&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=28&amp;pages=117-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msq191" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsq191" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsq191"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20656793" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hill</div> <div class="given-names">W.G.</div></div>,  <div class="name"><div class="surname">Robertson</div> <div class="given-names">A.</div></div>. </div><div class="article-title">The effect of linkage on limits to artificial selection</div>, <div class="source">Genet. Res.</div> , <div class="year">1966</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">269</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20effect%20of%20linkage%20on%20limits%20to%20artificial%20selection&amp;author=W.G.%20Hill&amp;author=A.%20Robertson&amp;publication_year=1966&amp;journal=Genet.%20Res.&amp;volume=8&amp;pages=269-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1017/S0016672300010156" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1017%2FS0016672300010156" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1017%2FS0016672300010156"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5980116" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McInerney</div> <div class="given-names">J.O.</div></div>. </div><div class="article-title">Replicational and transcriptional selection on codon usage in <em>Borrelia burgdorferi</em></div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1998</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">10698</div>-<div class="lpage">703</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Replicational%20and%20transcriptional%20selection%20on%20codon%20usage%20in%20Borrelia%20burgdorferi&amp;author=J.O.%20McInerney&amp;publication_year=1998&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=95&amp;pages=10698-703" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.18.10698" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.18.10698" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.18.10698"> </span></div> </div></div></div></div></div><div content-id="DSS039C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">S.L.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Hottes</div> <div class="given-names">A.K.</div></div>,  <div class="name"><div class="surname">Shapiro</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">McAdams</div> <div class="given-names">H.H.</div></div>. </div><div class="article-title">Codon usage between genomes is constrained by genome-wide mutational processes</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">3480</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20between%20genomes%20is%20constrained%20by%20genome-wide%20mutational%20processes&amp;author=S.L.%20Chen&amp;author=W.%20Lee&amp;author=A.K.%20Hottes&amp;author=L.%20Shapiro&amp;author=H.H.%20McAdams&amp;publication_year=2004&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=3480-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0307827100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0307827100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0307827100"> </span></div> </div></div></div></div></div><div content-id="DSS039C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Qin</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Biao</div> <div class="given-names">W.W.</div></div>,  <div class="name"><div class="surname">Comeron</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Kreitman</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">W.H.</div></div>. </div><div class="article-title">Intragenic spatial patterns of codon usage bias in prokaryotic and eukaryotic genomes</div>, <div class="source">Genetics</div> , <div class="year">2004</div>, vol. <div class="volume">168</div> (pg. <div class="fpage">2245</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intragenic%20spatial%20patterns%20of%20codon%20usage%20bias%20in%20prokaryotic%20and%20eukaryotic%20genomes&amp;author=H.%20Qin&amp;author=W.W.%20Biao&amp;author=J.M.%20Comeron&amp;author=M.%20Kreitman&amp;author=W.H.%20Li&amp;publication_year=2004&amp;journal=Genetics&amp;volume=168&amp;pages=2245-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.104.030866" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.104.030866" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.104.030866"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15611189" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stoletzki</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Eyre-Walker</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Synonymous codon usage in <em>Escherichia coli</em>: selection for translational accuracy</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2007</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">374</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synonymous%20codon%20usage%20in%20Escherichia%20coli%3A%20selection%20for%20translational%20accuracy&amp;author=N.%20Stoletzki&amp;author=A.%20Eyre-Walker&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=24&amp;pages=374-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msl166" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsl166" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsl166"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17101719" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lin</div> <div class="given-names">Y.S.</div></div>,  <div class="name"><div class="surname">Byrnes</div> <div class="given-names">J.K.</div></div>,  <div class="name"><div class="surname">Hwang</div> <div class="given-names">J.K.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">W.H.</div></div>. </div><div class="article-title">Codon-usage bias versus gene conversion in the evolution of yeast duplicate genes</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2006</div>, vol. <div class="volume">103</div> (pg. <div class="fpage">14412</div>-<div class="lpage">16</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon-usage%20bias%20versus%20gene%20conversion%20in%20the%20evolution%20of%20yeast%20duplicate%20genes&amp;author=Y.S.%20Lin&amp;author=J.K.%20Byrnes&amp;author=J.K.%20Hwang&amp;author=W.H.%20Li&amp;publication_year=2006&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=103&amp;pages=14412-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0606348103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0606348103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0606348103"> </span></div> </div></div></div></div></div><div content-id="DSS039C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhou</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sun</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Lu</div> <div class="given-names">Z.</div></div>. </div><div class="article-title">Synonymous codon usage in environmental <em>Chlamydia</em> UWE25 reflects an evolutional divergence from pathogenic <em>Chlamydiae</em></div>, <div class="source">Gene</div> , <div class="year">2006</div>, vol. <div class="volume">368</div> (pg. <div class="fpage">117</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synonymous%20codon%20usage%20in%20environmental%20Chlamydia%20UWE25%20reflects%20an%20evolutional%20divergence%20from%20pathogenic%20Chlamydiae&amp;author=T.%20Zhou&amp;author=X.%20Sun&amp;author=Z.%20Lu&amp;publication_year=2006&amp;journal=Gene&amp;volume=368&amp;pages=117-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2005.10.035" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2005.10.035" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2005.10.035"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16380221" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharp</div> <div class="given-names">P.M.</div></div>,  <div class="name"><div class="surname">Emery</div> <div class="given-names">L.R.</div></div>,  <div class="name"><div class="surname">Zeng</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Forces that influence the evolution of codon bias</div>, <div class="source">Philos. Trans. R. Soc. Lond. B Biol. Sci.</div> , <div class="year">2010</div>, vol. <div class="volume">365</div> (pg. <div class="fpage">1203</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Forces%20that%20influence%20the%20evolution%20of%20codon%20bias&amp;author=P.M.%20Sharp&amp;author=L.R.%20Emery&amp;author=K.%20Zeng&amp;publication_year=2010&amp;journal=Philos.%20Trans.%20R.%20Soc.%20Lond.%20B%20Biol.%20Sci.&amp;volume=365&amp;pages=1203-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1098/rstb.2009.0305" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frstb.2009.0305" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1098%2Frstb.2009.0305"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20308095" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wright</div> <div class="given-names">F.</div></div>. </div><div class="article-title">The ‘effective number of codons’ used in a gene</div>, <div class="source">Gene</div> , <div class="year">1990</div>, vol. <div class="volume">87</div> (pg. <div class="fpage">23</div>-<div class="lpage">29</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20%E2%80%98effective%20number%20of%20codons%E2%80%99%20used%20in%20a%20gene&amp;author=F.%20Wright&amp;publication_year=1990&amp;journal=Gene&amp;volume=87&amp;pages=23-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119(90)90491-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119(90)90491-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0378-1119(90)90491-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2110097" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wan</div> <div class="given-names">X.F.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Kleinhofs</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Zhou</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Quantitative relationship between synonymous bias and GC composition across unicellular genomes</div>, <div class="source">BMC Evol. Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">4</div> pg. <div class="fpage">19</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20relationship%20between%20synonymous%20bias%20and%20GC%20composition%20across%20unicellular%20genomes&amp;author=X.F.%20Wan&amp;author=D.%20Xu&amp;author=A.%20Kleinhofs&amp;author=J.%20Zhou&amp;publication_year=2004&amp;journal=BMC%20Evol.%20Biol.&amp;volume=4&amp;pages=19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-4-19" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-4-19" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-4-19"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15222899" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xu</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">L.L.</div></div>,  <div class="name"><div class="surname">Ruan</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">The draft genome of sweet orange (Citrus sinensis)</div>, <div class="source">Nat. Genet.</div> , <div class="year">2012</div><div class="pub-id-doi"><a href="https://doi.org/10.1038/ng.2472" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/ng.2472</a></div></div></div></div></div><div content-id="DSS039C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Madan</div> <div class="given-names">A.</div></div>. </div><div class="article-title">CAP3: a DNA sequence assembly program</div>, <div class="source">Genome Res.</div> , <div class="year">1999</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">868</div>-<div class="lpage">77</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CAP3%3A%20a%20DNA%20sequence%20assembly%20program&amp;author=X.%20Huang&amp;author=A.%20Madan&amp;publication_year=1999&amp;journal=Genome%20Res.&amp;volume=9&amp;pages=868-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.9.9.868" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.9.9.868" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.9.9.868"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10508846" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gouzy</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Carrere</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Schiex</div> <div class="given-names">T.</div></div>. </div><div class="article-title">FrameDP: sensitive peptide detection on noisy matured sequences</div>, <div class="source">Bioinformatics</div> , <div class="year">2009</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">670</div>-<div class="lpage">71</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=FrameDP%3A%20sensitive%20peptide%20detection%20on%20noisy%20matured%20sequences&amp;author=J.%20Gouzy&amp;author=S.%20Carrere&amp;author=T.%20Schiex&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=670-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19153134" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schiex</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Gouzy</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Moisan</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">de Oliveira</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">FrameD: a flexible program for quality check and gene prediction in prokaryotic genomes and noisy matured eukaryotic sequences</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">373</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=FrameD%3A%20a%20flexible%20program%20for%20quality%20check%20and%20gene%20prediction%20in%20prokaryotic%20genomes%20and%20noisy%20matured%20eukaryotic%20sequences&amp;author=T.%20Schiex&amp;author=J.%20Gouzy&amp;author=A.%20Moisan&amp;author=Y.%20de%20Oliveira&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=373-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg610" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg610" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg610"> </span></div> </div></div></div></div></div><div content-id="DSS039C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Madden</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Schäffer</div> <div class="given-names">A.A.</div></div>, et al. </div><div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3389</div>-<div class="lpage">402</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.F.%20Altschul&amp;author=T.L.%20Madden&amp;author=A.A.%20Sch%C3%A4ffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Greenacre</div> <div class="given-names">M.J.</div></div>. </div>, <div class="source">Theory and Application of Correspondence Analysis</div> , <div class="year">1984</div><div class="publisher-loc">London</div><div class="publisher-name">Academic Press</div>pg. <div class="fpage">223</div> </div></div></div></div><div content-id="DSS039C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharp</div> <div class="given-names">P.M.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">W.H.</div></div>. </div><div class="article-title">An evolutionary perspective on synonymous codon usage in unicellular organisms</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1986</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">28</div>-<div class="lpage">38</div>)</div></div></div></div><div content-id="DSS039C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kyte</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Doolittle</div> <div class="given-names">R.</div></div>. </div><div class="article-title">A simple method for displaying the hydropathic character of a protein</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1982</div>, vol. <div class="volume">157</div> (pg. <div class="fpage">105</div>-<div class="lpage">32</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20simple%20method%20for%20displaying%20the%20hydropathic%20character%20of%20a%20protein&amp;author=J.%20Kyte&amp;author=R.%20Doolittle&amp;publication_year=1982&amp;journal=J.%20Mol.%20Biol.&amp;volume=157&amp;pages=105-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(82)90515-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(82)90515-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(82)90515-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7108955" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Codon usage and tRNA content in unicellular and multicellular organisms</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1985</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">13</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20and%20tRNA%20content%20in%20unicellular%20and%20multicellular%20organisms&amp;author=T.%20Ikemura&amp;publication_year=1985&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=2&amp;pages=13-34" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3916708" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sablok</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Nayak</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Vazquez</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Tatarinova</div> <div class="given-names">T.V.</div></div>. </div><div class="article-title">Synonymous codon usage, GC3 and evolutionary patterns across plastomes of three pooid model species – emerging grass genome models for monocots</div>, <div class="source">Mol. Biotechnol.</div> , <div class="year">2011</div>, vol. <div class="volume">49</div> (pg. <div class="fpage">116</div>-<div class="lpage">28</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synonymous%20codon%20usage%2C%20GC3%20and%20evolutionary%20patterns%20across%20plastomes%20of%20three%20pooid%20model%20species%20%E2%80%93%20emerging%20grass%20genome%20models%20for%20monocots&amp;author=G.%20Sablok&amp;author=K.%20Nayak&amp;author=F.%20Vazquez&amp;author=T.V.%20Tatarinova&amp;publication_year=2011&amp;journal=Mol.%20Biotechnol.&amp;volume=49&amp;pages=116-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s12033-011-9383-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs12033-011-9383-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs12033-011-9383-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21308422" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lechner</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Findeiss</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Steiner</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Marz</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Stadler</div> <div class="given-names">P.F.</div></div>,  <div class="name"><div class="surname">Prohaska</div> <div class="given-names">S.J.</div></div>. </div><div class="article-title">Proteinortho: detection of (co-) orthologs in large-scale analysis</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">124</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Proteinortho%3A%20detection%20of%20(co-)%20orthologs%20in%20large-scale%20analysis&amp;author=M.%20Lechner&amp;author=S.%20Findeiss&amp;author=L.%20Steiner&amp;author=M.%20Marz&amp;author=P.F.%20Stadler&amp;author=S.J.%20Prohaska&amp;publication_year=2011&amp;journal=BMC%20Bioinformatics&amp;volume=12&amp;pages=124" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-12-124" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-12-124" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-12-124"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21526987" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Foster</div> <div class="given-names">P.G.</div></div>,  <div class="name"><div class="surname">Jermiin</div> <div class="given-names">L.S.</div></div>,  <div class="name"><div class="surname">Hickey</div> <div class="given-names">D.A.</div></div>. </div><div class="article-title">Nucleotide composition bias affects amino acid content in proteins coded by animal mitochondria</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">1997</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">282</div>-<div class="lpage">88</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nucleotide%20composition%20bias%20affects%20amino%20acid%20content%20in%20proteins%20coded%20by%20animal%20mitochondria&amp;author=P.G.%20Foster&amp;author=L.S.%20Jermiin&amp;author=D.A.%20Hickey&amp;publication_year=1997&amp;journal=J.%20Mol.%20Evol.&amp;volume=44&amp;pages=282-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00006145" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00006145" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00006145"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9060394" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Singer</div> <div class="given-names">G.A.C.</div></div>,  <div class="name"><div class="surname">Hickey</div> <div class="given-names">D.A.</div></div>. </div><div class="article-title">Nucleotide bias causes a genomewide bias in the amino acid composition of proteins</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2000</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">1581</div>-<div class="lpage">88</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nucleotide%20bias%20causes%20a%20genomewide%20bias%20in%20the%20amino%20acid%20composition%20of%20proteins&amp;author=G.A.C.%20Singer&amp;author=D.A.%20Hickey&amp;publication_year=2000&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=17&amp;pages=1581-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026257" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a026257" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a026257"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11070046" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hu</div> <div class="given-names">G.B.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">S.L.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">C.J.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">S.Q.</div></div>. </div><div class="article-title">Analysis of codon usage in <em>Citrus</em></div>, <div class="source">J. Fruit Sci.</div> , <div class="year">2006</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">479</div>-<div class="lpage">85</div>)</div></div></div></div><div content-id="DSS039C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sueoka</div> <div class="given-names">N.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Bryson</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Vogel</div> <div class="given-names">H.J.</div></div>. </div><div class="article-title">On the evolution of informational macromolecules</div>, <div class="source">Evolving Genes and Proteins</div> , <div class="year">1964</div><div class="publisher-loc">New York</div><div class="publisher-name">Academic Press</div>(pg. <div class="fpage">479</div>-<div class="lpage">96</div>)</div></div></div></div><div content-id="DSS039C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sueoka</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Kawanishi</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">DNA G+ C content of the third codon position and codon usage biases of human genes</div>, <div class="source">Gene</div> , <div class="year">2000</div>, vol. <div class="volume">261</div> (pg. <div class="fpage">53</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20G%2B%20C%20content%20of%20the%20third%20codon%20position%20and%20codon%20usage%20biases%20of%20human%20genes&amp;author=N.%20Sueoka&amp;author=Y.%20Kawanishi&amp;publication_year=2000&amp;journal=Gene&amp;volume=261&amp;pages=53-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(00)00480-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(00)00480-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(00)00480-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11164037" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sueoka</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Directional mutation pressure and neutral molecular evolution</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1988</div>, vol. <div class="volume">85</div> (pg. <div class="fpage">2653</div>-<div class="lpage">57</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Directional%20mutation%20pressure%20and%20neutral%20molecular%20evolution&amp;author=N.%20Sueoka&amp;publication_year=1988&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=85&amp;pages=2653-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.85.8.2653" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.85.8.2653" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.85.8.2653"> </span></div> </div></div></div></div></div><div content-id="DSS039C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">H.C.</div></div>,  <div class="name"><div class="surname">Hickey</div> <div class="given-names">D.A.</div></div>. </div><div class="article-title">Rapid divergence of codon usage patterns within the rice genome</div>, <div class="source">BMC Evol. Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">7</div> <div class="issue">Suppl. 1</div>pg. <div class="fpage">S6</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20divergence%20of%20codon%20usage%20patterns%20within%20the%20rice%20genome&amp;author=H.C.%20Wang&amp;author=D.A.%20Hickey&amp;publication_year=2007&amp;journal=BMC%20Evol.%20Biol.&amp;volume=7&amp;pages=S6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-7-S1-S6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-7-S1-S6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-7-S1-S6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17288579" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gupta</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Bhattacharyya</div> <div class="given-names">T.K.</div></div>,  <div class="name"><div class="surname">Ghosh</div> <div class="given-names">T.C.</div></div>. </div><div class="article-title">Synonymous codon usage in <em>Lactococcus lactis</em>: mutational bias versus translational selection</div>, <div class="source">J. Biomol. Struct. Dyn.</div> , <div class="year">2004</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">527</div>-<div class="lpage">36</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synonymous%20codon%20usage%20in%20Lactococcus%20lactis%3A%20mutational%20bias%20versus%20translational%20selection&amp;author=S.K.%20Gupta&amp;author=T.K.%20Bhattacharyya&amp;author=T.C.%20Ghosh&amp;publication_year=2004&amp;journal=J.%20Biomol.%20Struct.%20Dyn.&amp;volume=21&amp;pages=527-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/07391102.2004.10506946" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F07391102.2004.10506946" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F07391102.2004.10506946"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14692797" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wright</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Bibb</div> <div class="given-names">M.J.</div></div>. </div><div class="article-title">Codon usage in the G+C-rich Streptomyces genome</div>, <div class="source">Gene</div> , <div class="year">1992</div>, vol. <div class="volume">113</div> (pg. <div class="fpage">55</div>-<div class="lpage">65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20in%20the%20G%2BC-rich%20Streptomyces%20genome&amp;author=F.%20Wright&amp;author=M.J.%20Bibb&amp;publication_year=1992&amp;journal=Gene&amp;volume=113&amp;pages=55-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119(92)90669-G" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119(92)90669-G" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0378-1119(92)90669-G"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1563633" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tatarinova</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Alexandrov</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Bouck</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Feldmann</div> <div class="given-names">K.</div></div>. </div><div class="article-title">GC<sub>3</sub> biology in corn, rice, sorghum and other grasses</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">308</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GC3%20biology%20in%20corn%2C%20rice%2C%20sorghum%20and%20other%20grasses&amp;author=T.%20Tatarinova&amp;author=N.%20Alexandrov&amp;author=J.%20Bouck&amp;author=K.%20Feldmann&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=308" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-308" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-308" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-308"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20470436" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharp</div> <div class="given-names">P.M.</div></div>,  <div class="name"><div class="surname">Cowe</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Higgins</div> <div class="given-names">D.G.</div></div>,  <div class="name"><div class="surname">Shields</div> <div class="given-names">D.C.</div></div>,  <div class="name"><div class="surname">Wolfe</div> <div class="given-names">K.H.</div></div>,  <div class="name"><div class="surname">Wright</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Codon usage in <em>Escherichia coli</em>, <em>Bacillus subtilis</em>, <em>Saccharomyces cerevisiae</em>, <em>Schizosaccharomyces pombe</em>, <em>Drosophila melanogaster</em> and <em>Homo sapiens</em>; a review of the considerable within-species diversity</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1988</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">8207</div>-<div class="lpage">11</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20in%20Escherichia%20coli%2C%20Bacillus%20subtilis%2C%20Saccharomyces%20cerevisiae%2C%20Schizosaccharomyces%20pombe%2C%20Drosophila%20melanogaster%20and%20Homo%20sapiens%3B%20a%20review%20of%20the%20considerable%20within-species%20diversity&amp;author=P.M.%20Sharp&amp;author=E.%20Cowe&amp;author=D.G.%20Higgins&amp;author=D.C.%20Shields&amp;author=K.H.%20Wolfe&amp;author=F.%20Wright&amp;publication_year=1988&amp;journal=Nucleic%20Acids%20Res.&amp;volume=16&amp;pages=8207-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/16.17.8207" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F16.17.8207" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F16.17.8207"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3138659" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Andersson</div> <div class="given-names">S.G.E.</div></div>,  <div class="name"><div class="surname">Kurland</div> <div class="given-names">C.G.</div></div>. </div><div class="article-title">Codon preferences in free-living microorganisms</div>, <div class="source">Microbiol. Rev.</div> , <div class="year">1990</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">198</div>-<div class="lpage">210</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20preferences%20in%20free-living%20microorganisms&amp;author=S.G.E.%20Andersson&amp;author=C.G.%20Kurland&amp;publication_year=1990&amp;journal=Microbiol.%20Rev.&amp;volume=54&amp;pages=198-210" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2194095" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Qian</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">J-R.</div></div>,  <div class="name"><div class="surname">Pearson</div> <div class="given-names">N.M.</div></div>,  <div class="name"><div class="surname">Maclean</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Balanced codon usage optimizes eukaryotic translational efficiency</div>, <div class="source">PLoS Genet.</div> , <div class="year">2012</div>, vol. <div class="volume">8</div> pg. <div class="fpage">e1002603</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Balanced%20codon%20usage%20optimizes%20eukaryotic%20translational%20efficiency&amp;author=W.%20Qian&amp;author=J-R.%20Yang&amp;author=N.M.%20Pearson&amp;author=C.%20Maclean&amp;author=J.%20Zhang&amp;publication_year=2012&amp;journal=PLoS%20Genet.&amp;volume=8&amp;pages=e1002603" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1002603" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1002603" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1002603"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22479199" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sørensen</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Pedersen</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Absolute <em>in vivo</em> translation rates of individual codons in <em>Escherichia coli</em>: the two glutamic acid codons GAA and GAG are translated with a threefold difference in rate</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1991</div>, vol. <div class="volume">222</div> (pg. <div class="fpage">265</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Absolute%20in%20vivo%20translation%20rates%20of%20individual%20codons%20in%20Escherichia%20coli%3A%20the%20two%20glutamic%20acid%20codons%20GAA%20and%20GAG%20are%20translated%20with%20a%20threefold%20difference%20in%20rate&amp;author=M.A.%20S%C3%B8rensen&amp;author=S.%20Pedersen&amp;publication_year=1991&amp;journal=J.%20Mol.%20Biol.&amp;volume=222&amp;pages=265-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(91)90211-N" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(91)90211-N" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(91)90211-N"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1960727" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kudla</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Murray</div> <div class="given-names">A.W.</div></div>,  <div class="name"><div class="surname">Tollervey</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Plotkin</div> <div class="given-names">J.B.</div></div>. </div><div class="article-title">Coding-sequence determinants of gene expression in <em>Escherichia coli</em></div>, <div class="source">Science</div> , <div class="year">2009</div>, vol. <div class="volume">324</div> (pg. <div class="fpage">255</div>-<div class="lpage">58</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Coding-sequence%20determinants%20of%20gene%20expression%20in%20Escherichia%20coli&amp;author=G.%20Kudla&amp;author=A.W.%20Murray&amp;author=D.%20Tollervey&amp;author=J.B.%20Plotkin&amp;publication_year=2009&amp;journal=Science&amp;volume=324&amp;pages=255-58" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1170160" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1170160" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1170160"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19359587" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Audic</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Claverie</div> <div class="given-names">J.M.</div></div>. </div><div class="article-title">The significance of digital gene expression profiles</div>, <div class="source">Genome Res.</div> , <div class="year">1997</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">986</div>-<div class="lpage">95</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20significance%20of%20digital%20gene%20expression%20profiles&amp;author=S.%20Audic&amp;author=J.M.%20Claverie&amp;publication_year=1997&amp;journal=Genome%20Res.&amp;volume=7&amp;pages=986-95" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9331369" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Munoz</div> <div class="given-names">E.T.</div></div>,  <div class="name"><div class="surname">Bogarad</div> <div class="given-names">L.D.</div></div>,  <div class="name"><div class="surname">Deem</div> <div class="given-names">M.W.</div></div>. </div><div class="article-title">Microarray and EST database estimates of mRNA expression levels differ: the protein length versus expression curve for <em>C. elegans</em></div>, <div class="source">BMC Genomics</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> pg. <div class="fpage">30</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microarray%20and%20EST%20database%20estimates%20of%20mRNA%20expression%20levels%20differ%3A%20the%20protein%20length%20versus%20expression%20curve%20for%20C.%20elegans&amp;author=E.T.%20Munoz&amp;author=L.D.%20Bogarad&amp;author=M.W.%20Deem&amp;publication_year=2004&amp;journal=BMC%20Genomics&amp;volume=5&amp;pages=30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-5-30" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-5-30" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-5-30"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15134588" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cutter</div> <div class="given-names">A.D.</div></div>,  <div class="name"><div class="surname">Wasmuth</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Washington</div> <div class="given-names">N.L.</div></div>. </div><div class="article-title">Patterns of molecular evolution in <em>Caenorhabditis</em> preclude ancient origins of selfing</div>, <div class="source">Genetics</div> , <div class="year">2008</div>, vol. <div class="volume">178</div> (pg. <div class="fpage">2093</div>-<div class="lpage">104</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Patterns%20of%20molecular%20evolution%20in%20Caenorhabditis%20preclude%20ancient%20origins%20of%20selfing&amp;author=A.D.%20Cutter&amp;author=J.D.%20Wasmuth&amp;author=N.L.%20Washington&amp;publication_year=2008&amp;journal=Genetics&amp;volume=178&amp;pages=2093-104" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.107.085787" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.107.085787" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.107.085787"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18430935" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ingvarsson</div> <div class="given-names">P.D.</div></div>. </div><div class="article-title">Molecular evolution of synonymous codon usage in <em>Populus</em></div>, <div class="source">BMC Evol. Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">8</div> pg. <div class="fpage">307</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20evolution%20of%20synonymous%20codon%20usage%20in%20Populus&amp;author=P.D.%20Ingvarsson&amp;publication_year=2008&amp;journal=BMC%20Evol.%20Biol.&amp;volume=8&amp;pages=307" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-8-307" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-8-307" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-8-307"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18983655" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Whittle</div> <div class="given-names">C.A.</div></div>,  <div class="name"><div class="surname">Sun</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Johannesson</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Evolution of synonymous codon usage in <em>Neurospora tetrasperma</em> and <em>Neurospora discrete</em></div>, <div class="source">Genome Biol. Evol.</div> , <div class="year">2011</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">332</div>-<div class="lpage">43</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20synonymous%20codon%20usage%20in%20Neurospora%20tetrasperma%20and%20Neurospora%20discrete&amp;author=C.A.%20Whittle&amp;author=Y.%20Sun&amp;author=H.%20Johannesson&amp;publication_year=2011&amp;journal=Genome%20Biol.%20Evol.&amp;volume=3&amp;pages=332-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evr018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevr018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevr018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21402862" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Knight</div> <div class="given-names">R.D.</div></div>,  <div class="name"><div class="surname">Freeland</div> <div class="given-names">S.J.</div></div>,  <div class="name"><div class="surname">Landweber</div> <div class="given-names">L.F.</div></div>. </div><div class="article-title">A simple model based on mutation and selection explains trends in codon and amino acid usage and GC composition within and across genomes</div>, <div class="source">Genome Biol.</div> , <div class="year">2001</div>, vol. <div class="volume">2</div> pg. <div class="fpage">research0010</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20simple%20model%20based%20on%20mutation%20and%20selection%20explains%20trends%20in%20codon%20and%20amino%20acid%20usage%20and%20GC%20composition%20within%20and%20across%20genomes&amp;author=R.D.%20Knight&amp;author=S.J.%20Freeland&amp;author=L.F.%20Landweber&amp;publication_year=2001&amp;journal=Genome%20Biol.&amp;volume=2&amp;pages=research0010" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11305938" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C57" class="ref-content "><div class="label -label">57</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mitreva</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Wendl</div> <div class="given-names">M.C.</div></div>,  <div class="name"><div class="surname">Martin</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Codon usage patterns in Nematoda: analysis based on over 25 million codons in thirty-two species</div>, <div class="source">Genome Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">R75</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20patterns%20in%20Nematoda%3A%20analysis%20based%20on%20over%2025%20million%20codons%20in%20thirty-two%20species&amp;author=M.%20Mitreva&amp;author=M.C.%20Wendl&amp;author=J.%20Martin&amp;publication_year=2006&amp;journal=Genome%20Biol.&amp;volume=7&amp;pages=R75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2006-7-8-r75" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2006-7-8-r75" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2006-7-8-r75"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26271136" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C58" class="ref-content "><div class="label -label">58</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Palidwor</div> <div class="given-names">G.A.</div></div>,  <div class="name"><div class="surname">Perkins</div> <div class="given-names">T.J.</div></div>,  <div class="name"><div class="surname">Xia</div> <div class="given-names">X.</div></div>. </div><div class="article-title">A general model of codon bias due to GC mutational bias</div>, <div class="source">PLoS ONE</div> , <div class="year">2010</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e13431</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20general%20model%20of%20codon%20bias%20due%20to%20GC%20mutational%20bias&amp;author=G.A.%20Palidwor&amp;author=T.J.%20Perkins&amp;author=X.%20Xia&amp;publication_year=2010&amp;journal=PLoS%20ONE&amp;volume=5&amp;pages=e13431" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0013431" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0013431" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0013431"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21048949" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C59" class="ref-content "><div class="label -label">59</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hiwasa-Tanase</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nyarubona</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hirai</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kato</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ichikawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ezura</div> <div class="given-names">H.</div></div>. </div><div class="article-title">High-level accumulation of recombinant miraculin protein in transgenic tomatoes expressing a synthetic miraculin gene with optimized codon usage terminated by the native miraculin terminator</div>, <div class="source">Plant Cell Rep.</div> , <div class="year">2011</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">113</div>-<div class="lpage">24</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-level%20accumulation%20of%20recombinant%20miraculin%20protein%20in%20transgenic%20tomatoes%20expressing%20a%20synthetic%20miraculin%20gene%20with%20optimized%20codon%20usage%20terminated%20by%20the%20native%20miraculin%20terminator&amp;author=K.%20Hiwasa-Tanase&amp;author=M.%20Nyarubona&amp;author=T.%20Hirai&amp;author=K.%20Kato&amp;author=T.%20Ichikawa&amp;author=H.%20Ezura&amp;publication_year=2011&amp;journal=Plant%20Cell%20Rep.&amp;volume=30&amp;pages=113-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00299-010-0949-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00299-010-0949-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00299-010-0949-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21076835" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C60" class="ref-content "><div class="label -label">60</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mendez</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Fritsche</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Porto</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Bastolla</div> <div class="given-names">U.</div></div>. </div><div class="article-title">Mutation bias favors protein folding stability in the evolution of small populations</div>, <div class="source">PLoS Comput. Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">6</div> pg. <div class="fpage">e1000767</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutation%20bias%20favors%20protein%20folding%20stability%20in%20the%20evolution%20of%20small%20populations&amp;author=R.%20Mendez&amp;author=M.%20Fritsche&amp;author=M.%20Porto&amp;author=U.%20Bastolla&amp;publication_year=2010&amp;journal=PLoS%20Comput.%20Biol.&amp;volume=6&amp;pages=e1000767" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1000767" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1000767" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1000767"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20463869" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C61" class="ref-content "><div class="label -label">61</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jiang</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Fergusona</div> <div class="given-names">A.A.</div></div>,  <div class="name"><div class="surname">Slotkinb</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Lischc</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Pack-mutator-like transposable elements (Pack-MULEs) induce directional modification of genes through biased insertion and DNA acquisition</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2011</div>, vol. <div class="volume">108</div> (pg. <div class="fpage">1537</div>-<div class="lpage">42</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pack-mutator-like%20transposable%20elements%20(Pack-MULEs)%20induce%20directional%20modification%20of%20genes%20through%20biased%20insertion%20and%20DNA%20acquisition&amp;author=N.%20Jiang&amp;author=A.A.%20Fergusona&amp;author=R.K.%20Slotkinb&amp;author=D.%20Lischc&amp;publication_year=2011&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=108&amp;pages=1537-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1010814108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1010814108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1010814108"> </span></div> </div></div></div></div></div><div content-id="DSS039C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C62" class="ref-content "><div class="label -label">62</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stayssman</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Nejman</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Roberts</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">Developmental programming of CpG island methylation profiles in the human genome</div>, <div class="source">Nat. Struct. Mol. Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">564</div>-<div class="lpage">71</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Developmental%20programming%20of%20CpG%20island%20methylation%20profiles%20in%20the%20human%20genome&amp;author=R.%20Stayssman&amp;author=D.%20Nejman&amp;author=D.%20Roberts&amp;publication_year=2009&amp;journal=Nat.%20Struct.%20Mol.%20Biol.&amp;volume=16&amp;pages=564-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb.1594" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb.1594" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb.1594"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19377480" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS039C63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS039C63" class="ref-content "><div class="label -label">63</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gupta</div> <div class="given-names">A.B.</div></div>,  <div class="name"><div class="surname">Sankararamakrishnan</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Genome-wide analysis of major intrinsic proteins in the tree plant <em>Populus trichocarpa</em>: characterization of XIP subfamily of aquaporins from evolutionary perspective</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">9</div> pg. <div class="fpage">134</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20major%20intrinsic%20proteins%20in%20the%20tree%20plant%20Populus%20trichocarpa%3A%20characterization%20of%20XIP%20subfamily%20of%20aquaporins%20from%20evolutionary%20perspective&amp;author=A.B.%20Gupta&amp;author=R.%20Sankararamakrishnan&amp;publication_year=2009&amp;journal=BMC%20Plant%20Biol.&amp;volume=9&amp;pages=134" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-9-134" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-9-134" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-9-134"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19930558" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Prof. Kenta Nakai</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=333376&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss039/2/dss039-dss039supp.pdf?Expires=2147483647&amp;Signature=W2OiFo~C04LCj5IQf-kXi2vfhhn5g5t3rjolyUGxzjHd-n26EWNFp5CA6Um~umtDrJHbxbxQGXu2tOqCxfdxYRwjGzkddKJF2X9wzB~b2rc3n27NyWZI5SSFUV0gpsJdA1f10sefPeqeasvMoeYhc29xc0BTIOlOZ9vhP9vbLK7vwbef9wjQu5vRADidRkkErOnlujySXy9aohwGYJG37G4HNRjDjiWPCuTaRCLOztHNj~2NRyeNGVx0CWyLR8Upd5TzogqC41rKZI7CiofTH0aRCPrTsNN4NEf9wpJ2-HHJhQ016q~lg1cCjkvAbnFuBvv6D7oMas7LD0blBUy26w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">274</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000318584600003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">13</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dss039"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="333376"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000318584600003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f20%2f2%2f135%2f333376">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000318584600003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (13)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F20%2F2%2F135%2F333376" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/333376'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Evaluation of Codon Biology in Citrus and Poncirus trifoliata Based on Genomic Features and Frame Corrected Expressed Sequence Tags" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dss039");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f20%2f2%2f135%2f333376">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

