

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/15/5/267/359884","@type":"ScholarlyArticle","name":"Detecting Horizontally Transferred and Essential Genes Based on Dinucleotide Relative Abundance","datePublished":"2008-09-16","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/15/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2008-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsn021","keywords":["amelioration","dinucleotide frequency","essential genes","horizontal transfer","molecular evolution"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/15/5/267/359884","author":[{"name":"Baran, Robert H.","affiliation":"Department of Electronics and Computer Engineering ,  Korea University ,  Anam-dong, Sungbuk-ku, Seoul 136-702 ,  South Korea","@type":"Person"},{"name":"Ko, Hanseok","affiliation":"Department of Electronics and Computer Engineering ,  Korea University ,  Anam-dong, Sungbuk-ku, Seoul 136-702 ,  South Korea","@type":"Person"}],"description":"Abstract.  Various methods have been developed to detect horizontal gene transfer in bacteria, based on anomalous nucleotide composition, assuming that composit","pageStart":"267","pageEnd":"276","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/5/1/m_cover.gif?Expires=2147483647&Signature=w0UhL1170rQznOI4EBmUQXu1ZOwiDy5D9yvI-xItfA5gEGy2mkrpZiqzDFtXeoJ~7GciRt1Jb~wykd96BMcXNK2oJMQKCvan9OGOux6ILlOeg1peDGvihwkEZeSlIEQRWsuy-Os5b2zaUJLN2EMIjSPATvDcEQeiR8VJm~tHNehbKW~jZGwxEfbLVfvvbsN-HCrfd6zxxRmciZJddcZFQjaH25Oc8zwhd9TcubBub~UwfgV64C6hjwxGjWayYeyC6503xComjsunKItnp20PuervJsOqbPEquDoZ8SVxa8ULqEXM-U2u8h~RKynzXZGyUfUdPZZhfeDp4GDC653uKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Detecting Horizontally Transferred and Essential Genes Based on Dinucleotide Relative Abundance","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/5/1/m_cover.gif?Expires=2147483647&Signature=w0UhL1170rQznOI4EBmUQXu1ZOwiDy5D9yvI-xItfA5gEGy2mkrpZiqzDFtXeoJ~7GciRt1Jb~wykd96BMcXNK2oJMQKCvan9OGOux6ILlOeg1peDGvihwkEZeSlIEQRWsuy-Os5b2zaUJLN2EMIjSPATvDcEQeiR8VJm~tHNehbKW~jZGwxEfbLVfvvbsN-HCrfd6zxxRmciZJddcZFQjaH25Oc8zwhd9TcubBub~UwfgV64C6hjwxGjWayYeyC6503xComjsunKItnp20PuervJsOqbPEquDoZ8SVxa8ULqEXM-U2u8h~RKynzXZGyUfUdPZZhfeDp4GDC653uKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Detecting Horizontally Transferred and Essential Genes Based on Dinucleotide Relative Abundance" />
    <meta property="og:description" content="Abstract.  Various methods have been developed to detect horizontal gene transfer in bacteria, based on anomalous nucleotide composition, assuming that composit" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/15/5/267/359884" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2008-09-16" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Baran, Robert H." /><meta name="citation_author_institution" content="Department of Electronics and Computer Engineering, Korea University, Anam-dong, Sungbuk-ku, Seoul 136-702, South Korea" /><meta name="citation_author" content="Ko, Hanseok" /><meta name="citation_author_institution" content="Department of Electronics and Computer Engineering, Korea University, Anam-dong, Sungbuk-ku, Seoul 136-702, South Korea" /><meta name="citation_title" content="Detecting Horizontally Transferred and Essential Genes Based on Dinucleotide Relative Abundance" /><meta name="citation_firstpage" content="267" /><meta name="citation_lastpage" content="276" /><meta name="citation_doi" content="10.1093/dnares/dsn021" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="15" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2008/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=The guanine and cytosine content of genomic DNA and bacterial evolution; Proc. Natl. Acad. Sci. USA;  citation_year=1987;  citation_volume=84;  citation_pages=166-169; " /><meta name="citation_reference" content="citation_title=Construction of Escherichia coli K-12 in-frame single-gene knockout mutants: the Keio collection; citation_text=2006.0008; Molec. Systems Biol.;  citation_year=2006;  citation_volume=2; " /><meta name="citation_reference" content="citation_title=A new computational method for detection of horizontal gene transfer events; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=922-923; " /><meta name="citation_reference" content="citation_title=Gene cluster analysis method identifies horizontally transferred genes with high reliability and indicates that they provide the main mechanism of operon gain in 8 species of gamma-Proteobacteria; Mol. Biol. Evol.;  citation_year=2007;  citation_volume=24;  citation_pages=805-813; " /><meta name="citation_reference" content="citation_title=Amelioration of bacterial genomes: rates of change and exchange; J. Molec. Evol.;  citation_year=1997;  citation_volume=44;  citation_pages=383-397; " /><meta name="citation_reference" content="citation_title=DarkHorse: a method for genome-wide prediction of horizontal gene transfer; Genome Biol.;  citation_year=2007;  citation_volume=8;  citation_pages=R16" /><meta name="citation_reference" content="citation_title=Molecular archaeology of the Escherichia coli genome; Proc. Natl. Acad. Sci. USA;  citation_year=1997;  citation_volume=95;  citation_pages=9413-9417; " /><meta name="citation_reference" content="citation_title=Comparison of synonymous codon distribution patterns of bacteriophage and host genomes; DNA Res.;  citation_year=1998;  citation_volume=5;  citation_pages=319-326; " /><meta name="citation_reference" content="citation_title=Detecting laterally transferred genes: use of entropic clustering methods and genome position; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=4629-4639; " /><meta name="citation_reference" content="citation_title=Codon usage and tRNA content in unicellular and multicellular organisms; Mol. Biol. Evol.;  citation_year=1985;  citation_volume=2;  citation_pages=13-34; " /><meta name="citation_reference" content="citation_title=Codon and amino acid usage in two major human pathogens of genus Bartonella—optimization between replicational-transcriptional selection, translational control and cost minimization; DNA Res.;  citation_year=2005;  citation_volume=12;  citation_pages=91-102; " /><meta name="citation_reference" content="citation_title=Evolutionary implications of microbial genome tetranucleotide frequency bias; Genome Res.;  citation_year=2003;  citation_volume=13;  citation_pages=145-158; " /><meta name="citation_reference" content="citation_title=Quantifying the species-specificity in genomic signatures, synonymous codon choice, amino acid usage and G + C content; Gene;  citation_year=2003;  citation_volume=311;  citation_pages=35-42; " /><meta name="citation_reference" content="citation_title=Detection and classification of horizontal transfers in prokaryotes using genomic signature; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=e6" /><meta name="citation_reference" content="citation_title=Differences in dinucleotide frequencies of human, yeast, and Escherichia coli genes; DNA Res.;  citation_year=1997;  citation_volume=4;  citation_pages=185-192; " /><meta name="citation_reference" content="citation_title=Genes from nine genomes are separated into their organisms in the dinucleotide composition space; DNA Res.;  citation_year=1998;  citation_volume=5;  citation_pages=251-259; " /><meta name="citation_reference" content="citation_title=Dinucleotide relative abundance extremes: a genomic signature; Trends Genet.;  citation_year=1995;  citation_volume=11;  citation_pages=283-290; " /><meta name="citation_reference" content="citation_title=Compositional biases of bacterial genomes and evolutionary implications; J. Bacteriol.;  citation_year=1997;  citation_volume=179;  citation_pages=3899-3913; " /><meta name="citation_reference" content="citation_title=The reach of the genome signature in prokaryotes; BMC Evol. Biol.;  citation_year=2006;  citation_volume=6;  citation_pages=84" /><meta name="citation_reference" content="citation_title=Capturing whole-genome characteristics in short sequences using a naive Bayesian classifier; Genome Res.;  citation_year=2001;  citation_volume=11;  citation_pages=1404-1409; " /><meta name="citation_reference" content="citation_publisher=Sinauer Associates, Sunderland, MA; Genetic Data Analysis.;  citation_year=1990; " /><meta name="citation_reference" content="citation_publisher=Wiley, New York; Categorical Data Analysis.;  citation_year=1990; " /><meta name="citation_reference" content="citation_title=Pervasive properties of the genomic signature; BMC Genom.;  citation_year=2002;  citation_volume=3;  citation_pages=23" /><meta name="citation_reference" content="citation_title=Similarity of the general designs of protochordates and invertebrates; Nature;  citation_year=1977;  citation_volume=266;  citation_pages=533-536; " /><meta name="citation_reference" content="citation_title=Testing lumpability in Markov chains; Stat. Prob. Lit.;  citation_year=2002;  citation_volume=64;  citation_pages=17-23; " /><meta name="citation_reference" content="citation_title=Base compositional structure of genomes; Genomics;  citation_year=1992;  citation_volume=13;  citation_pages=1056-1064; " /><meta name="citation_reference" content="citation_title=Are isochore sequences homogeneous?; Comput. Biol. Chem.;  citation_year=2003;  citation_volume=27;  citation_pages=5-10; " /><meta name="citation_reference" content="citation_title=Strand compositional asymmetry in bacterial and large viral genomes; Proc. Natl. Acad. Sci. USA;  citation_year=1998;  citation_volume=95;  citation_pages=3720-3725; " /><meta name="citation_reference" content="citation_title=EcoCyc: a comprehensive database resource for Escherichia coli; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=D334-D337; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-3402; " /><meta name="citation_reference" content="citation_title=Methods for assessing the statistical significance of molecular sequence features by using general scoring schemes; Proc. Natl. Acad. Sci. USA;  citation_year=1990;  citation_volume=87;  citation_pages=2264-2268; " /><meta name="citation_reference" content="citation_title=Rhs elements of Escherichia coli K12; J. Bacteriol.;  citation_year=1993;  citation_volume=175;  citation_pages=2799-2808; " /><meta name="citation_reference" content="citation_publisher=Wiley, New York; Pattern Classification: A Unified View of Statistical and Neural Approaches.;  citation_year=1996; " /><meta name="citation_reference" content="citation_title=Directional mutation pressure and neutral molecular evolution; Proc. Natl. Acad. Sci. USA;  citation_year=1988;  citation_volume=85;  citation_pages=2653-2658; " /><meta name="citation_reference" content="citation_title=Codon bias and base composition are poor indicators of horizontally transferred genes; Mol. Biol. Evol.;  citation_year=2001;  citation_volume=18;  citation_pages=404-412; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/15/5/267/1030312/dsn021.pdf" /><meta name="description" content="Abstract.  Various methods have been developed to detect horizontal gene transfer in bacteria, based on anomalous nucleotide composition, assuming that composit" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/15/5/267/359884" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/15/5/267/359884" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/15/5/267/359884");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsn021';
    </script>
<title>
	Detecting Horizontally Transferred and Essential Genes Based on Dinucleotide Relative Abundance | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./359884" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BbhHPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJCYXJhbiwgUm9iZXJ0IEguIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBFbGVjdHJvbmljcyBhbmQgQ29tcHV0ZXIgRW5naW5lZXJpbmcsIEtvcmVhIFVuaXZlcnNpdHksIEFuYW0tZG9uZywgU3VuZ2J1ay1rdSwgU2VvdWwgMTM2LTcwMiwgU291dGggS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLbywgSGFuc2VvayIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgRWxlY3Ryb25pY3MgYW5kIENvbXB1dGVyIEVuZ2luZWVyaW5nLCBLb3JlYSBVbml2ZXJzaXR5LCBBbmFtLWRvbmcsIFN1bmdidWsta3UsIFNlb3VsIDEzNi03MDIsIFNvdXRoIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJEZXRlY3RpbmcgSG9yaXpvbnRhbGx5IFRyYW5zZmVycmVkIGFuZCBFc3NlbnRpYWwgR2VuZXMgQmFzZWQgb24gRGludWNsZW90aWRlIFJlbGF0aXZlIEFidW5kYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjI2NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMjc2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNuMDIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMDgvMTAvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ3VhbmluZSBhbmQgY3l0b3NpbmUgY29udGVudCBvZiBnZW5vbWljIEROQSBhbmQgYmFjdGVyaWFsIGV2b2x1dGlvbjsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5ODc7ICBjaXRhdGlvbl92b2x1bWU9ODQ7ICBjaXRhdGlvbl9wYWdlcz0xNjYtMTY5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgRXNjaGVyaWNoaWEgY29saSBLLTEyIGluLWZyYW1lIHNpbmdsZS1nZW5lIGtub2Nrb3V0IG11dGFudHM6IHRoZSBLZWlvIGNvbGxlY3Rpb247IGNpdGF0aW9uX3RleHQ9MjAwNi4wMDA4OyBNb2xlYy4gU3lzdGVtcyBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5ldyBjb21wdXRhdGlvbmFsIG1ldGhvZCBmb3IgZGV0ZWN0aW9uIG9mIGhvcml6b250YWwgZ2VuZSB0cmFuc2ZlciBldmVudHM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTkyMi05MjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgY2x1c3RlciBhbmFseXNpcyBtZXRob2QgaWRlbnRpZmllcyBob3Jpem9udGFsbHkgdHJhbnNmZXJyZWQgZ2VuZXMgd2l0aCBoaWdoIHJlbGlhYmlsaXR5IGFuZCBpbmRpY2F0ZXMgdGhhdCB0aGV5IHByb3ZpZGUgdGhlIG1haW4gbWVjaGFuaXNtIG9mIG9wZXJvbiBnYWluIGluIDggc3BlY2llcyBvZiBnYW1tYS1Qcm90ZW9iYWN0ZXJpYTsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTgwNS04MTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFtZWxpb3JhdGlvbiBvZiBiYWN0ZXJpYWwgZ2Vub21lczogcmF0ZXMgb2YgY2hhbmdlIGFuZCBleGNoYW5nZTsgSi4gTW9sZWMuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTQ0OyAgY2l0YXRpb25fcGFnZXM9MzgzLTM5NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGFya0hvcnNlOiBhIG1ldGhvZCBmb3IgZ2Vub21lLXdpZGUgcHJlZGljdGlvbiBvZiBob3Jpem9udGFsIGdlbmUgdHJhbnNmZXI7IEdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9UjE2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGFyY2hhZW9sb2d5IG9mIHRoZSBFc2NoZXJpY2hpYSBjb2xpIGdlbm9tZTsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9OTU7ICBjaXRhdGlvbl9wYWdlcz05NDEzLTk0MTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmlzb24gb2Ygc3lub255bW91cyBjb2RvbiBkaXN0cmlidXRpb24gcGF0dGVybnMgb2YgYmFjdGVyaW9waGFnZSBhbmQgaG9zdCBnZW5vbWVzOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MzE5LTMyNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV0ZWN0aW5nIGxhdGVyYWxseSB0cmFuc2ZlcnJlZCBnZW5lczogdXNlIG9mIGVudHJvcGljIGNsdXN0ZXJpbmcgbWV0aG9kcyBhbmQgZ2Vub21lIHBvc2l0aW9uOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz00NjI5LTQ2Mzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9uIHVzYWdlIGFuZCB0Uk5BIGNvbnRlbnQgaW4gdW5pY2VsbHVsYXIgYW5kIG11bHRpY2VsbHVsYXIgb3JnYW5pc21zOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTg1OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0xMy0zNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gYW5kIGFtaW5vIGFjaWQgdXNhZ2UgaW4gdHdvIG1ham9yIGh1bWFuIHBhdGhvZ2VucyBvZiBnZW51cyBCYXJ0b25lbGxh4oCUb3B0aW1pemF0aW9uIGJldHdlZW4gcmVwbGljYXRpb25hbC10cmFuc2NyaXB0aW9uYWwgc2VsZWN0aW9uLCB0cmFuc2xhdGlvbmFsIGNvbnRyb2wgYW5kIGNvc3QgbWluaW1pemF0aW9uOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTkxLTEwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uYXJ5IGltcGxpY2F0aW9ucyBvZiBtaWNyb2JpYWwgZ2Vub21lIHRldHJhbnVjbGVvdGlkZSBmcmVxdWVuY3kgYmlhczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0xNDUtMTU4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1RdWFudGlmeWluZyB0aGUgc3BlY2llcy1zcGVjaWZpY2l0eSBpbiBnZW5vbWljIHNpZ25hdHVyZXMsIHN5bm9ueW1vdXMgY29kb24gY2hvaWNlLCBhbWlubyBhY2lkIHVzYWdlIGFuZCBHICsgQyBjb250ZW50OyBHZW5lOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMxMTsgIGNpdGF0aW9uX3BhZ2VzPTM1LTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXRlY3Rpb24gYW5kIGNsYXNzaWZpY2F0aW9uIG9mIGhvcml6b250YWwgdHJhbnNmZXJzIGluIHByb2thcnlvdGVzIHVzaW5nIGdlbm9taWMgc2lnbmF0dXJlOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz1lNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpZmZlcmVuY2VzIGluIGRpbnVjbGVvdGlkZSBmcmVxdWVuY2llcyBvZiBodW1hbiwgeWVhc3QsIGFuZCBFc2NoZXJpY2hpYSBjb2xpIGdlbmVzOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MTg1LTE5MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXMgZnJvbSBuaW5lIGdlbm9tZXMgYXJlIHNlcGFyYXRlZCBpbnRvIHRoZWlyIG9yZ2FuaXNtcyBpbiB0aGUgZGludWNsZW90aWRlIGNvbXBvc2l0aW9uIHNwYWNlOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MjUxLTI1OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGludWNsZW90aWRlIHJlbGF0aXZlIGFidW5kYW5jZSBleHRyZW1lczogYSBnZW5vbWljIHNpZ25hdHVyZTsgVHJlbmRzIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTI4My0yOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBvc2l0aW9uYWwgYmlhc2VzIG9mIGJhY3RlcmlhbCBnZW5vbWVzIGFuZCBldm9sdXRpb25hcnkgaW1wbGljYXRpb25zOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTE3OTsgIGNpdGF0aW9uX3BhZ2VzPTM4OTktMzkxMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHJlYWNoIG9mIHRoZSBnZW5vbWUgc2lnbmF0dXJlIGluIHByb2thcnlvdGVzOyBCTUMgRXZvbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTg0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2FwdHVyaW5nIHdob2xlLWdlbm9tZSBjaGFyYWN0ZXJpc3RpY3MgaW4gc2hvcnQgc2VxdWVuY2VzIHVzaW5nIGEgbmFpdmUgQmF5ZXNpYW4gY2xhc3NpZmllcjsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xNDA0LTE0MDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1TaW5hdWVyIEFzc29jaWF0ZXMsIFN1bmRlcmxhbmQsIE1BOyBHZW5ldGljIERhdGEgQW5hbHlzaXMuOyAgY2l0YXRpb25feWVhcj0xOTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9V2lsZXksIE5ldyBZb3JrOyBDYXRlZ29yaWNhbCBEYXRhIEFuYWx5c2lzLjsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGVydmFzaXZlIHByb3BlcnRpZXMgb2YgdGhlIGdlbm9taWMgc2lnbmF0dXJlOyBCTUMgR2Vub20uOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0yMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpbWlsYXJpdHkgb2YgdGhlIGdlbmVyYWwgZGVzaWducyBvZiBwcm90b2Nob3JkYXRlcyBhbmQgaW52ZXJ0ZWJyYXRlczsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0xOTc3OyAgY2l0YXRpb25fdm9sdW1lPTI2NjsgIGNpdGF0aW9uX3BhZ2VzPTUzMy01MzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRlc3RpbmcgbHVtcGFiaWxpdHkgaW4gTWFya292IGNoYWluczsgU3RhdC4gUHJvYi4gTGl0LjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT02NDsgIGNpdGF0aW9uX3BhZ2VzPTE3LTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYXNlIGNvbXBvc2l0aW9uYWwgc3RydWN0dXJlIG9mIGdlbm9tZXM7IEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MTA1Ni0xMDY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BcmUgaXNvY2hvcmUgc2VxdWVuY2VzIGhvbW9nZW5lb3VzPzsgQ29tcHV0LiBCaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTUtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cmFuZCBjb21wb3NpdGlvbmFsIGFzeW1tZXRyeSBpbiBiYWN0ZXJpYWwgYW5kIGxhcmdlIHZpcmFsIGdlbm9tZXM7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTk1OyAgY2l0YXRpb25fcGFnZXM9MzcyMC0zNzI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FY29DeWM6IGEgY29tcHJlaGVuc2l2ZSBkYXRhYmFzZSByZXNvdXJjZSBmb3IgRXNjaGVyaWNoaWEgY29saTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9RDMzNC1EMzM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HYXBwZWQgQkxBU1QgYW5kIFBTSS1CTEFTVDogYSBuZXcgZ2VuZXJhdGlvbiBvZiBwcm90ZWluIGRhdGFiYXNlIHNlYXJjaCBwcm9ncmFtczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MzM4OS0zNDAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZXRob2RzIGZvciBhc3Nlc3NpbmcgdGhlIHN0YXRpc3RpY2FsIHNpZ25pZmljYW5jZSBvZiBtb2xlY3VsYXIgc2VxdWVuY2UgZmVhdHVyZXMgYnkgdXNpbmcgZ2VuZXJhbCBzY29yaW5nIHNjaGVtZXM7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTg3OyAgY2l0YXRpb25fcGFnZXM9MjI2NC0yMjY4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SaHMgZWxlbWVudHMgb2YgRXNjaGVyaWNoaWEgY29saSBLMTI7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9MTc1OyAgY2l0YXRpb25fcGFnZXM9Mjc5OS0yODA4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9V2lsZXksIE5ldyBZb3JrOyBQYXR0ZXJuIENsYXNzaWZpY2F0aW9uOiBBIFVuaWZpZWQgVmlldyBvZiBTdGF0aXN0aWNhbCBhbmQgTmV1cmFsIEFwcHJvYWNoZXMuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXJlY3Rpb25hbCBtdXRhdGlvbiBwcmVzc3VyZSBhbmQgbmV1dHJhbCBtb2xlY3VsYXIgZXZvbHV0aW9uOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk4ODsgIGNpdGF0aW9uX3ZvbHVtZT04NTsgIGNpdGF0aW9uX3BhZ2VzPTI2NTMtMjY1ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gYmlhcyBhbmQgYmFzZSBjb21wb3NpdGlvbiBhcmUgcG9vciBpbmRpY2F0b3JzIG9mIGhvcml6b250YWxseSB0cmFuc2ZlcnJlZCBnZW5lczsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTQwNC00MTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xNS81LzI2Ny8xMDMwMzEyL2RzbjAyMS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgVmFyaW91cyBtZXRob2RzIGhhdmUgYmVlbiBkZXZlbG9wZWQgdG8gZGV0ZWN0IGhvcml6b250YWwgZ2VuZSB0cmFuc2ZlciBpbiBiYWN0ZXJpYSwgYmFzZWQgb24gYW5vbWFsb3VzIG51Y2xlb3RpZGUgY29tcG9zaXRpb24sIGFzc3VtaW5nIHRoYXQgY29tcG9zaXQiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM1OTg4NCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGSng7mcd5As7NZccRb/Yvwt8xZ5PA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f5%2f267%2f359884" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f5%2f267%2f359884" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13976" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/15/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/5/1/m_cover.gif?Expires=2147483647&amp;Signature=w0UhL1170rQznOI4EBmUQXu1ZOwiDy5D9yvI-xItfA5gEGy2mkrpZiqzDFtXeoJ~7GciRt1Jb~wykd96BMcXNK2oJMQKCvan9OGOux6ILlOeg1peDGvihwkEZeSlIEQRWsuy-Os5b2zaUJLN2EMIjSPATvDcEQeiR8VJm~tHNehbKW~jZGwxEfbLVfvvbsN-HCrfd6zxxRmciZJddcZFQjaH25Oc8zwhd9TcubBub~UwfgV64C6hjwxGjWayYeyC6503xComjsunKItnp20PuervJsOqbPEquDoZ8SVxa8ULqEXM-U2u8h~RKynzXZGyUfUdPZZhfeDp4GDC653uKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/15/5">
                        <div class="volume trailing-comma">Volume 15</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2008        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5191433">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5191433">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5191435">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5191435">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5191443">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5191443">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5191467">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5191467">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5191491">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5191491">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5191493">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5191493">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/15/5/277/359806">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Detecting Horizontally Transferred and Essential Genes Based on Dinucleotide Relative Abundance
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Robert H. Baran</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Department of Electronics and Computer Engineering</div>, <div class="insititution">Korea University</div>, <div class="addr-line">Anam-dong, Sungbuk-ku, Seoul 136-702</div>, <div class="country">South Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Robert+H.+Baran">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Baran R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Baran R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Robert H. Baran</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hanseok Ko</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Department of Electronics and Computer Engineering</div>, <div class="insititution">Korea University</div>, <div class="addr-line">Anam-dong, Sungbuk-ku, Seoul 136-702</div>, <div class="country">South Korea</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +82 2-3290-3239. Fax. <div class="fax">+82 2-3291-2450</div>. E-mail: <a href="mailto:hsko@korea.ac.kr" target="_blank">hsko@korea.ac.kr</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hanseok+Ko">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ko H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ko H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hanseok Ko</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Katsumi Isnono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 15, Issue 5, 1 October 2008, Pages 267–276, <a href='https://doi.org/10.1093/dnares/dsn021'>https://doi.org/10.1093/dnares/dsn021</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">16 September 2008</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">12 December 2007</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">05 August 2008</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="359884" href="/dnaresearch/article-pdf/15/5/267/1030312/dsn021.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Robert H. Baran, Hanseok Ko;  Detecting Horizontally Transferred and Essential Genes Based on Dinucleotide Relative Abundance, <em>DNA Research</em>, Volume 15, Issue 5, 1 October 2008, Pages 267–276, <a href='https://doi.org/10.1093/dnares/dsn021'>https://doi.org/10.1093/dnares/dsn021</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/359884?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/359884?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/359884?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/359884?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/359884?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/359884?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/359884?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Detecting%20Horizontally%20Transferred%20and%20Essential%20Genes%20Based%20on%20Dinucleotide%20Relative%20Abundance&amp;publicationDate=2008-09-16&amp;volumeNum=15&amp;issueNum=5&amp;author=Baran%2C%20Robert%20H.%3B%20Ko%2C%20Hanseok&amp;startPage=267&amp;endPage=276&amp;contentId=10.1093%2Fdnares%2Fdsn021&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f5%2f267%2f359884" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5191433 id="5191433" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Various methods have been developed to detect horizontal gene transfer in bacteria, based on anomalous nucleotide composition, assuming that compositional features undergo amelioration in the host genome. Evolutionary theory predicts the inevitability of false positives when essential sequences are strongly conserved. Foreign genes could become more detectable on the basis of their higher order compositions if such features ameliorate more rapidly and uniformly than lower order features. This possibility is tested by comparing the heterogeneities of bacterial genomes with respect to strand-independent first- and second-order features, (i) G + C content and (ii) dinucleotide relative abundance, in 1 kb segments. Although statistical analysis confirms that (ii) is less inhomogeneous than (i) in all 12 species examined, extreme anomalies with respect to (ii) in the <em>Escherichia coli</em> K12 genome are typically co-located with essential genes.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="amelioration">amelioration</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;dinucleotide frequency&quot;">dinucleotide frequency</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;essential genes&quot;">essential genes</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;horizontal transfer&quot;">horizontal transfer</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;molecular evolution&quot;">molecular evolution</a></div>                            <h2 scrollto-destination=5191435 id="5191435" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>If biased mutational pressure is exerted globally and uniformly on all parts of the genome, then local differences in evolution rates can be explained by the negative selection principle of the neutral theory, which implies that ‘functionally less important parts in the genome evolve faster than more important ones’.<sup><a reveal-id="DSN021C1" data-open="DSN021C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Recent progress in construction of single-gene knockout mutant collections makes it possible to determine the essentiality of each gene in any bacterium that can be cultured.<sup><a reveal-id="DSN021C2" data-open="DSN021C2" class="link link-ref link-reveal xref-bibr">2</a></sup> If evolution rates could be estimated in a defensible and consistent manner, the implication of the neutral theory might be tested by comparing estimates conditioned on essentiality. But if the evolutionary histories of bacterial genes were so transparent, then it would be easy to identify the ones acquired from foreign donors by horizontal gene transfer (HGT).</p><p>HGT detection has been pursued by phylogenetic methods, based on sequence alignment, and by composition-based methods<sup><a reveal-id="DSN021C3" data-open="DSN021C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSN021C4" data-open="DSN021C4" class="link link-ref link-reveal xref-bibr">4</a></sup> that involve alignment-free features such as G + C content, synonymous codon usage (SCU) and the frequencies of overlapping short oligomers. These species-specific features undergo amelioration, the process by which a foreign gene, acquired through HGT, evolves toward the composition of the host genome.<sup><a reveal-id="DSN021C5" data-open="DSN021C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Anomalous composition will be found in recently acquired genes, especially when they originate in distantly related species, but it will also be observed where amelioration has been retarded. Therefore, if genes are ranked according to dissimilarity with respect to global composition, the highest ranks will be awarded both to recent transfers and to relatively ancient coding sequences that have resisted mutational pressure. Composition-based methods have been criticized for falsely indicating HGT when highly conserved proteins have atypical amino acid compositions.<sup><a reveal-id="DSN021C6" data-open="DSN021C6" class="link link-ref link-reveal xref-bibr">6</a></sup> The coding sequences of such proteins can undergo amelioration to the extent permitted by SCU—but not much farther. Therefore, we expect to find essential genes among false positives produced by HGT detection methods that operate on compositional features other than SCU.</p><p>Seminal investigations HGT in <em>Escherichia coli</em> examined G + C content and SCU to identify the ∼17.6% of genes most probably acquired since divergence from <em>Salmonella</em> ∼100 million years ago<sup><a reveal-id="DSN021C7" data-open="DSN021C7" class="link link-ref link-reveal xref-bibr">7</a></sup> and to distinguish native genes from mobile elements (prophages and transposons).<sup><a reveal-id="DSN021C8" data-open="DSN021C8" class="link link-ref link-reveal xref-bibr">8</a></sup> After refining the mathematical treatment of context-dependent SCU, Azad and Lawrence<sup><a reveal-id="DSN021C9" data-open="DSN021C9" class="link link-ref link-reveal xref-bibr">9</a></sup> recently predicted 639 foreign genes in <em>E</em>. <em>coli</em>, reporting a false positive rate of 190/639 = 29.7% that was lower than for any competing method of comparable sensitivity. (False positives were genes &gt;300 bp with homologs in <em>S</em>. <em>enterica</em> LT2.) SCU reflects species-specific preferences that are strongly correlated with gross cellular amounts of isoaccepting tRNAs,<sup><a reveal-id="DSN021C10" data-open="DSN021C10" class="link link-ref link-reveal xref-bibr">10</a></sup> especially in highly expressed genes,<sup><a reveal-id="DSN021C8" data-open="DSN021C8" class="link link-ref link-reveal xref-bibr">8</a></sup> without regard to function. Though driven by genome-wide mutational pressure on G + C, especially at the third codon position, these preferences are not selection-neutral,<sup><a reveal-id="DSN021C11" data-open="DSN021C11" class="link link-ref link-reveal xref-bibr">11</a></sup> and false positives may reflect the retarded amelioration of SCU in lowly expressed genes.</p><p>It has been theorized that higher order compositional features are inherently more species-specific than lower order features<sup><a reveal-id="DSN021C12  DSN021C14" data-open="DSN021C12  DSN021C14" class="link link-ref link-reveal xref-bibr">12–14</a></sup> and hence that tetranucleotide frequencies, computed without reference to a reading frame, convey more useful information than G + C content and SCU for purposes like fragment classification and HGT detection.<sup><a reveal-id="DSN021C3" data-open="DSN021C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSN021C12  DSN021C14" data-open="DSN021C12  DSN021C14" class="link link-ref link-reveal xref-bibr">12–14</a></sup> This claim could be reconciled with the implication of the neutral theory if higher order features ameliorate more rapidly and uniformly than lower order features. For example, dinucleotide composition can be mathematically decomposed into two parts: (i) the mononucleotide composition and (ii) the matrix of ‘odds ratios’ that compare the observed proportions of the individual dinucleotides to their expectations under the assumption of pure randomness.<sup><a reveal-id="DSN021C15" data-open="DSN021C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Nakashima et al.<sup><a reveal-id="DSN021C16" data-open="DSN021C16" class="link link-ref link-reveal xref-bibr">16</a></sup> examined 10 complete genomes and concluded that part (ii) reflects phylogenetic relations better than part (i). The strand-invariant form of this feature is the dinucleotide relative abundance (DRA) profile which was called a ‘genomic signature’ by Karlin and Burge<sup><a reveal-id="DSN021C17" data-open="DSN021C17" class="link link-ref link-reveal xref-bibr">17</a></sup> because an organism can generally be identified by computing it from any 50 kb or longer segment.<sup><a reveal-id="DSN021C18" data-open="DSN021C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p><p>DRA ratios thus exhibit species-specificity, approaching their global values in all sufficiently long segments of any single genome, but varying greatly between distant relatives. This phenomenon is unexplained<sup><a reveal-id="DSN021C19" data-open="DSN021C19" class="link link-ref link-reveal xref-bibr">19</a></sup> but it could obviously be a consequence of the relatively rapid and uniform amelioration of genes with respect to DRA. The possibility of inferring HGT from anomalous dinucleotide composition was tested early on the path toward development of better methods and subsequent investigations have used this feature to establish a performance baseline.<sup><a reveal-id="DSN021C3" data-open="DSN021C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSN021C9" data-open="DSN021C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSN021C20" data-open="DSN021C20" class="link link-ref link-reveal xref-bibr">20</a></sup> We propose a new statistical procedure to assess the possibility that DRA ratios approach their global means more uniformly than G + C proportions in segments of bacterial genomes. If a compositional feature ameliorates uniformly, then its intra-genomic variability will be bounded within certain limits embodied in a null hypothesis of homogeneity as formulated below. After testing this hypothesis in several bacterial species, the most anomalous segments of the <em>E</em>. <em>coli</em> K12 sequence are subjected to a phylogenetic analysis to identify false positives (by the noted criteria<sup><a reveal-id="DSN021C10" data-open="DSN021C10" class="link link-ref link-reveal xref-bibr">10</a></sup>) which are subsequently checked for essentiality with reference to published data.<sup><a reveal-id="DSN021C2" data-open="DSN021C2" class="link link-ref link-reveal xref-bibr">2</a></sup></p>                            <h2 scrollto-destination=5191443 id="5191443" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
<div class="block-child-p">Let <em>n</em><sub><em>xy</em></sub> denote the number of overlapping <em>xy</em> dinucleotides in a coding sequence of length <em>n</em> + 1 = ΣΣ<em>n</em><sub><em>xy</em></sub> + 1 so that <em>f</em><sub><em>xy</em></sub> = <em>n</em><sub><em>xy</em></sub><em>n</em> is the corresponding (normalized) fraction. The mononucleotide count in the first place is <em>n</em><sub><em>x</em></sub> and the corresponding proportion is <em>f</em><sub><em>x</em></sub>. Symbol <em>f</em> is replaced by φ when the whole genome is considered. The dinucleotide odds ratios for the whole genome are defined as <a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn021/2/dsn02104.gif?Expires=2147483647&Signature=b8GgetFAj6K3G3NoKqC4JHMDMcXQToxlebkQFvEtjIGksz4RR0WiwuZmq9e-qH-9981e6GEfi6Z9Y7PVFDoFFHrwwRXzQXLwkOi6GEQy5-lojK-RiT0y46ddv0GRiPhyko3zR~0YRqyMjVKDe7cM53jRXER9C3LNvE6HZWtRcseshqj-VGheZmvSQUWHoG~hQx7EhoyDa8cCbLIFm~QT~wIoDXrUnt4fDdBiY1UCm1yY-b3v4HHvufNyLhlTJWEpzm3gCzTjRN~9otcYtogKLYOoRaMwDAx0SQ-Pc7r0ikuFWE4pKspoVkcgjpwUFHchWypAB~bqaoyNfRW8Zhl9AA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsn02104.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn021/2/m_dsn02104.gif?Expires=2147483647&Signature=LQQWdYBFypnyxH028D46g~ThVv2uE3eKig16PBPivmZUcbGedQYHD3RArwzD4tnplJ7LLDdlHAsZXEbhBYqEtGlNn0mRthljSpFsXN9oHyGrD3cBsJTfLZ6p9~pDcPgUwdvftWoxsjRW6pyNIkkzMCbP6bzrIT1EJP6HR0PRuH4AIIlNMsT6gQS83ROVwn~fhdcL~kIUMeIyMLf88L0rHv-7jgrMNHnX6j99-bvROTIgDbmBbstibQABIma1MBnzMe6C5J6JRkXXHgVi9UIqBJjstYrkWiWuHMACfsPmB4dm2ml7-UJE11Xdy9YukrMK1985EbN1KiCQh70n2UDdUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsn02104.gif" /></a></div></div></div> The local value of this ratio (e.g. for a single gene) is denoted <em>ρ</em><sub><em>xy</em></sub> and obtained by substituting <em>f</em> for φ.</div><div class="block-child-p">It is understood that the genomes are not perfectly homogeneous with respect to oligonucleotide compositions of any order. The present problem is to quantify the inhomogeneity with respect to dinucleotide composition and then apportion this total amount between parts (i) and (ii) defined above.<sup><a reveal-id="DSN021C16" data-open="DSN021C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Beginning with the mathematical identity <a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsn02105.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn021/2/m_dsn02105.gif?Expires=2147483647&amp;Signature=4WsZdWyneu8agYZX-GhTya1JDjpLfe3AzGsiUgmv14bPGBySPSr~tKVlBHjfXqYySp09tsUM0w4o1RYOJPJBRUnRV7v9lvgoJRluyV8SMhOKL37LmgNfEhTW5FuCDoBVx3U-JVn0Q92~iNIqQuKAWfRhR03bwYkR7gmmoopLZt~ZEpHKlw5BOpQ4s3hg2NoQK9zMUxh-GpksDcdRLw5EvuC8aN6nYDg2YA5WZx1Xn7cD1LumoE7wp~JFQlfLFkB-FLAKbVgEojFRGs~OKMEDuRSv5F6QtAUpyN4EtwsAy2pY-SpK8AgeMvJgsl02WB5hP7bJaVJ-R0U4uunSikBT9w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsn02105.gif" /></a></div></div></div> first take logarithms, then multiply through by <em>n</em><sub><em>xy</em></sub> and sum over all dinucleotides. The result is <a id="DSN021M1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSN021M1"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsn02106.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn021/2/m_dsn02106.gif?Expires=2147483647&amp;Signature=2jjRHkn4JIfZ6aoMlxWGAKdY7tvO18xB6MAqlZof-kPqNdotXuhrrYjz73l7QkFjaR80c-Y57feG8FNNaBx7TCGwsJwykjIe-8ikVduztxdOX6o-TLDurSTm2iHz6QZJOCdNjOV0QnjZfMAlElNVjiqNVzQyiniKTorOBkoRxEj3ZRrXqX~gMIpja8K56RAV8ObQBJYU~dB~vkj6eZcqX1Mu7VpSrQddYwkdmc~3BDpLUPZDzRpF8cbXo1nyYUavXO2TpWsI3AFENPcxUCeFNyaKwWgG-l1kmSzvuwm-DAcmFok4aYfv~1U2r8cBmrCo1KuPrSQktZVEhhvBdbY2kA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsn02106.gif" /></a></div></div><div class="label">1</div></div> in which each term is recognized as a deviance statistic that has a chi-squared (<em>χ</em><sup>2</sup>) distribution for sufficiently large <em>n</em> under a particular hypothesis of compositional homogeneity.</div><div class="block-child-p">In accordance with standard practice in genetics and many other fields,<sup><a reveal-id="DSN021C21" data-open="DSN021C21" class="link link-ref link-reveal xref-bibr">21</a></sup> the deviance is replaced by its second order approximation, the <em>χ</em><sup>2</sup> statistic, here denoted <em>X</em>, which converges in distribution more rapidly and monotonically to its large sample limit, subject to the usual proviso that each expected count is at least 5.<sup><a reveal-id="DSN021C22" data-open="DSN021C22" class="link link-ref link-reveal xref-bibr">22</a></sup> This will almost guarantee a <em>χ</em><sup>2</sup> distribution for <em>n</em> ≈ 1000 bp. On the left side of Equation (<a href="#DSN021M1" class="sectionLink xref-sec">1</a>), the deviance is <a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsn02107.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn021/2/m_dsn02107.gif?Expires=2147483647&amp;Signature=dnvcM68wLdq0Ws5~4~9qfYaq383VlQJN4WT130~90-dGK4KjANQkSWoTLvvJVs63Y2Kzu79EoutJfS0VCd3Ign9DcBVpzGAAhjSVLDwTv3vRcOh20I2TeiOWoH6bS~JRqPfLWxtYPGOEQz2cWG-KeOFeLG9VBsmoCrXEWzM~M69G7Jr4y5SfpOvdFN7Yyn7GNcPGHSfl17w8wYoGo2F0sFuqzBSg~pZgx5qLWEgQdg65hXYHz~cZWDjv6E~qdzIlpjWe~XvOL01BCV9kMsZYn3B5JDD3Ng1v9OOn7cp2D4PipYs21Sat8AjbEkSWf8D24PgpRHbynAa6Td3prcrF9Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsn02107.gif" /></a></div></div></div> where the arrow says that it approaches <em>χ</em><sup>2</sup> with 15 degrees of freedom under the hypothesis of (H<sub>3</sub>:) homogeneity with respect to dinucleotide composition. On the right side of Equation (<a href="#DSN021M1" class="sectionLink xref-sec">1</a>) we have <a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsn02108.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn021/2/m_dsn02108.gif?Expires=2147483647&amp;Signature=Gw3LUL-8lITo5ngFSy-omeek2aRuZcLDdx0HLX5z8nOPCRxb6NBLcwCFXOxxon9eBTEh0C2n4BRIKXnjkMwB2P-8B7Etc~xWMdqhIODcvRRJSQCkS-~i7ciLlYJxEnTP34D70DRAjpxLelPS8sBEbBYExWfCpBD1QfJUJ52GilUYoeOrU3LBJWnDnDKYgwng14eqUbWBDNiX2dVv~yGeymfm-ev~79lWTcmtL2F~SLkN8GS6HZ0G177Ejc1WCb73MKAjkCb-5VVJtDAo9i1N5RhxeNnu1s29tqgezvi2anykQSIsDOVh2QTI1gVpBSmKD3IYRs0~2GnUtWccrWLmbQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsn02108.gif" /></a></div></div></div> where <a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsn02109.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn021/2/m_dsn02109.gif?Expires=2147483647&amp;Signature=pjNKQk1VwOx~C78SxfP-1Z0WNOexf3A0MCjFpg~EyHRXLzEEcaJD3V3GY7H4oNCwzoxNFZD2khmoa4wKBcX9SYFggM39llZmPieR4X9ze7bIGb1VvlQ-hCTzIfeXCrpvrL6MOVlrtXEk-UD7ZnrYGfYuHgjiQ4wT1s7q09dXxqcmaLhQiG6~cKeNYR9OY0pZCBtYXne5cYi-nSCR4PRQ9aXgzxEEml8R0A28nWTuZ~id4JvFXP6VdrSht1xnu6P3NRGB5y09tpLu0f5Iw8jJWsTKPS18LMGQREya5VGm0CXNyZhL4MrR-vOoUFyIRrYm7wO4rJGXmRWxHDFvzY3jFg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsn02109.gif" /></a></div></div></div> is the expected value of <em>n</em><sub><em>xy</em></sub> under the hypothesis of (H<sub>2</sub>:) homogeneity with respect to DRA. This hypothesis conforms to the saturated log-linear model defined by Agresti (chapter 5)<sup><a reveal-id="DSN021C22" data-open="DSN021C22" class="link link-ref link-reveal xref-bibr">22</a></sup> in treating two-way contingency tables of Poisson-distributed counts. The last two terms on the right side of Equation (<a href="#DSN021M1" class="sectionLink xref-sec">1</a>) are added to obtain <a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsn021010.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn021/2/m_dsn021010.gif?Expires=2147483647&amp;Signature=agijtwgO1Xp7klpZWmKfcces1rgFKA~MY7vYGy-7Ic58BIQhOGA-whZjv7ekORg1r0jy19MRUTLkAp7ZcNvpsF-huwKuNx0V4vZJMwOBHtadNuvQ1ZCaDN6PFuapiqEgaK0ioOisQPMw71g47k3T~E2In8uSjtPs7vU-r6lUiw3hTh8dTSHeVwAYs22czECv6jSHgLv1TfNGOFSRcJaU-yWnckCQCLwQymLXmp7giJisSKRCg5FXa7M35V3tAFhcFidkBJFYt8me1f46gsMFo8Ih4CZ2KJ8AmztYmkZfMYw7lCgtKnv7EPt-qquWQUwZVhv0lDmp8--sDcy~zlof0A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsn021010.gif" /></a></div></div></div> under the hypothesis of (H<sub>1</sub>:) homogeneity with respect to mononucleotide composition. Note that H<sub>3</sub> implies H<sub>2</sub> and H<sub>1</sub> (and conversely). Thus, Equation (<a href="#DSN021M1" class="sectionLink xref-sec">1</a>) becomes <em>D</em><sub>3</sub> = <em>D</em><sub>2</sub> + <em>D</em><sub>1</sub>, which leads to approximately the equivalent statement <a id="DSN021M2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSN021M2"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsn021011.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn021/2/m_dsn021011.gif?Expires=2147483647&amp;Signature=CIJaFpG63NjT6OE9Jz-Y4L6LX0fseXnPKmzsIUpSf2HzzmhDWjxGGaYuYVlJDBBi1pRjyL5iduZ-9fmWkZh1rjOkaV6hJHyo~gQUkMC4nTN8qTPgc0En1inKu4kfdplEBSzRAVxgSfc-HlnUo6hylKbAuQs4-9vsN3EnjkImJUSMrcdk3Mz0FRLdh1Erp4HhzjracPsAXgjZr~dnlbek1yR8-0cqQJcgKQN-Yzf3YRup4xmV833iTRKa0GNkx2F1NnszLtRRaHw9zFlZaDsIooQawW16MPd9M3a6rxVC4b0kOFUT85MLB5dPlYVa-pwckSfnxg346oCtrK~T~drJ6w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsn021011.gif" /></a></div></div><div class="label">2</div></div> in which the degrees of freedom add correctly (15 = 9 + 6).</div><div class="block-child-p">Arbitrary segments (which may be intergenic or overlap more than one gene) can be handled similarly by averaging the dinucleotide counts of the sequence and its reverse complement. This step nullifies strand dependence and imposes counter-diagonal symmetry on the 4 × 4 matrix of dinucleotide proportions. The symmetrized dinucleotide odds ratios comprise the DRA profile, denoted <em>ρ</em>* by Karlin et al.,<sup><a reveal-id="DSN021C17" data-open="DSN021C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSN021C18" data-open="DSN021C18" class="link link-ref link-reveal xref-bibr">18</a></sup> who measured the distance between sequences <em>g</em> and <em>h</em> as <a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn021/2/dsn021012.gif?Expires=2147483647&Signature=NtCJDgGJGovhKxQJ-gP4L-Ya0MzDKAbrPjwnkcX1ZOVLfb6Wn0FV-1RMCTLV4QqDZAc4p-bTbhUQ0R~LhuFW6bcW5ULEHUWuG~DVraQ9MQ3O9dRfKI5LFP2kkyDagj3X-ExfdAn1UzBu8cXDyjw4OD3Wkqxjn-NrYUGcZlpzWNYGO5drZm6GVz7yJnofEsdGWTw4FyDDa7-xqOKM3tKOH6kkTH6n6LcSiOlLL-v7N0dopdBs6aJ8LVZMMJ9wwfnOEqApDZ~7WVWW-6iRVj0eo7IQ02pb4HhJ~Mcn2Cgskf957nNLT753i45BVFRP6ScU5ZZkrm0rEjSWVjzb0oVnGw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsn021012.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn021/2/m_dsn021012.gif?Expires=2147483647&Signature=FRPSkXDo7nJAQcD4AIZKgWW3QWKwP94meik3vqDcFayPcwxW5iBeAwUJ-otvI3HodT8mvW6SDDNnl~jzxEQqs2D7ERxNbMzd4bWZ1TIHKf88YMsfObv8u59vNALvP27a3QG2Pjn-20I38MXJ0PLsrkbUIpXE079BLnsRv3bMlII1X4c5WsFcH2vE3aZJdyZ58TNaCW~f2Lqt~3rVV18kMLIuYAF1xa75HovFoHXsF-dicuFeOK29sAtSyZh98bg8O1YqUYfNjYIwrzEObgs77Add8XeS5no~xBua1RLh33k0pprf7QJkv7et85QEYz5QR2TF42tGQZiA2DGHNMPl0Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsn021012.gif" /></a></div></div></div> which they called delta-distance. When <em>h</em> is a segment of genome <em>g</em>, the intra-genomic delta-distances are log-normally distributed, with means approaching zero slightly slower than reciprocal square root of segment length.<sup><a reveal-id="DSN021C23" data-open="DSN021C23" class="link link-ref link-reveal xref-bibr">23</a></sup> But the statistical significance of large local delta-distances cannot be rigorously assessed.</div><div class="block-child-p">Each quantity in Equation (<a href="#DSN021M1" class="sectionLink xref-sec">1</a>) has a strand-invariant analog, beginning with the symmetrized dinucleotide composition <em>f</em>*<sub><em>xy</em></sub>, and the asterisk (*) can be appended to every one of the statistics in Equation (<a href="#DSN021M2" class="sectionLink xref-sec">2</a>) to indicate these substitutions. The degrees of freedom are changed as follows: <a id="DSN021M3" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSN021M3"><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn021/2/dsn021013.gif?Expires=2147483647&Signature=DOOWas3escs37xfRHRCVe0aAWGT12NO9fYAtP0kmLSt1aMwJbqvSWjsrkNGt2CpxMKVOfRZel-71UaTLel31asghzjLoopyUXyVQw3Xhdfnzzohc~U9g-Y921jO6QdY3omQco2aNffH9tgSgc1QivVS0tS3dVX-3yPjOyVSyDQlzE123saRR-Q0ygj8Y-jGd1caMjocJ9hN4FVne6jqAwHd7CtS2LZ4nHg9T~6qilELdSIEjOgwz1gVAcXMGZR4O0y9Oq6yMpIwkqlZOsbCXzb~Tkr-i-nAzwjUa84w6~PzzAQSge8FZdwe45h-O~Bgc~jgEa-wHqOwHvWxYl0GZ8w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsn021013.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn021/2/m_dsn021013.gif?Expires=2147483647&Signature=no7o0XJd77avuZ109T9WdRmTbNCBQpVgQu7UCpudzBgA26XJdrzKCRc6iJAWiLOSrGSw0drGJgp77auKeMbWqoxj1oY1jOig64njvB6S1n2f5Ykh44025vwfUEL2Si9T~xUXIPgUVpx4m4I09tiWblCKUgFv53zaepKpBH5RFtTEo1Sso3FGdZC1~IihK9-v8WJTt2yn7yUtLPun7TF-X1iQ5FZ7zsR3Qei5T6vcrJ0HgMTUOe5WzFOcmK17OML1IMxi6zWCVrjoGHJ5ZapB8NZoCI-wdluWCiRojmbKkvzoe1Pz33dxi4j6Z~nYYCFzg2X68osoeFRvZf7YW97AEA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsn021013.gif" /></a></div></div><div class="label">3</div></div></div><p>For example, the 10 non-redundant components of <em>ρ</em>* present 6 degrees of freedom, as remarked by Russell and Subak-Sharpe.<sup><a reveal-id="DSN021C24" data-open="DSN021C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Note that the symmetrized mononucleotide composition is completely determined by the G + C proportions. Since <em>χ</em><sup>2</sup>-tests are meaningless unless degrees of freedom are exact, we verified these claims [Equation (<a href="#DSN021M3" class="sectionLink xref-sec">3</a>)] by Monte Carlo calculations.<sup><a reveal-id="DSN021C25" data-open="DSN021C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Each trial begins with random matrix [φ<sub><em>xy</em></sub>], each count <em>n</em><sub><em>xy</em></sub> is a Poisson random number with expected value <em>n</em>φ<sub><em>xy</em></sub>, and the three <em>X</em>* statistics are computed. After 2000 trials, the average value of each <em>X</em>* statistic is within ±0.15 of the specified degrees of freedom, and the probability integral transformations yield numbers that are uniformly distributed on the unit interval.</p><p>Starting with the published sequence of a bacterial chromosome, a window of length 1000 bp (1 kb) is displaced in increments of 1 kb from left to right (5′ to 3′), but not wrapping around to overlap the origin on the last displacement. Each window produces three <em>X</em>* statistics which obey Equation (<a href="#DSN021M2" class="sectionLink xref-sec">2</a>), as illustrated for the <em>E</em>. <em>coli</em> K12 sequence (NCBI reference sequence number NC_000913.2) in Fig. <a reveal-id="DSN021F1" data-open="DSN021F1" class="link link-reveal link-table xref-fig">1</a>, where the sum of the two statistics on the right side of Equation (<a href="#DSN021M2" class="sectionLink xref-sec">2</a>) is evidently close to the value on the left, and the mean absolute difference (MAD) is only 3.4%. But each statistic commonly exceeds its null degrees of freedom (the expected value) by a large amount, as shown in Fig. <a reveal-id="DSN021F2" data-open="DSN021F2" class="link link-reveal link-table xref-fig">2</a>, and many of them exceed the 99.5-percentile point of the <em>χ</em><sup>2</sup>-distribution that describes the homogeneous case. Large values of <em>X</em>*<sub>2</sub> occur in segments with anomalous DRA. This statistic, henceforth called the DRA-divergence, exceeds the 99.5-percentile of its null distribution in 10.2% of segments. Large values of <em>X</em>*<sub>1</sub> occur in segments with anomalous G + C proportions. This statistic, the GC-divergence, exceeds the 99.5-percentile of its null distribution in 34.0% of segments. At the 95-percentile point, the exceedance rates (ERs) are 24.3% for the DRA-divergence and 46.4% for the GC-divergence. On this basis, we see that DRA is less inhomogeneous than G + C.</p>                        <a id="5191460" scrollto-destination="5191460"></a>
<div id="DSN021F1" data-id="DSN021F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn021/2/m_dsn02101.jpeg?Expires=2147483647&Signature=Ql2wO8qV1oUPxSKyZoB1rYIpCaxruVWeGZmlog0bfb4rlIIJE7SKt2lPA5b9MZ~VkDg~lUiD9r~XZjAv16aEoCDr-MOrf8yTioKek72-jUzmBJQHik07StPFL8F7wPpXWaD3WSPWbNJNTe-oRk0AsTOuL1uqwwvc0kkTACiVPqtGs2P24B6LRXySOM2Hngjf2OTzgUqGfeTaGexPf3wxtnEqksB5tWztgQ-~-fQDXjnUHYV1nw~J6r9KuiqKENJQY7U4hbXyP6DpcOTTVfsCljJuVWjrqAqP-Yq5wlhIlhfygUR-g-7K26LsI4VJkg617J5~OTtlrO3gpE~~eVwwbA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scatter plot of total chi-squared divergence, on the left side of Equation (2), versus the sum of the statistics on the right side, in 1 kb segments of the E. coli K12 genome. The line of unit slope through the origin is also drawn." path-from-xml="dsn02101.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5191460/dsn02101.jpeg" path-from-xml="dsn02101.jpeg" target="_blank">View large</a><a section="5191460" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn021/2/dsn02101.jpeg?Expires=2147483647&Signature=m0uzE05LoArZO1krPYzKScgHzklAPu43WpS-i6VsCJYRy9q83Fv~eMxyfkgaHRiiczbBCnLZIn46ig1Qa5HRZRFHbm9CYVeQZrNc2zvXozWwzjfF~amFQah4La1pJgldiOmktAFFC3TmlU3ISInfJ2q2q3Tp4RHHqWG2t6LVZN6cAfaW7HyB4tefCuDjAvGIOEfx~-8cq-eB1OqpV8bpKrKaYx74Afl5COWKDrPWPCW3gSr3Y-oij6-wPjW-9kF0YtYFlx8wmQxj0uj0QrgrDGm-o3uTWSrlLiUKL07zHPE9K2SCsfvJEyKmZ9C3dNy6pZM-azp0zJM5lmYhurBkhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5191460&ar=359884&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02101.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scatter plot of total chi-squared divergence, on the left side of Equation (2), versus the sum of the statistics on the right side, in 1 kb segments of the <em>E</em>. <em>coli</em> K12 genome. The line of unit slope through the origin is also drawn.</p></div></div><div content-id="DSN021F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn021/2/m_dsn02101.jpeg?Expires=2147483647&Signature=Ql2wO8qV1oUPxSKyZoB1rYIpCaxruVWeGZmlog0bfb4rlIIJE7SKt2lPA5b9MZ~VkDg~lUiD9r~XZjAv16aEoCDr-MOrf8yTioKek72-jUzmBJQHik07StPFL8F7wPpXWaD3WSPWbNJNTe-oRk0AsTOuL1uqwwvc0kkTACiVPqtGs2P24B6LRXySOM2Hngjf2OTzgUqGfeTaGexPf3wxtnEqksB5tWztgQ-~-fQDXjnUHYV1nw~J6r9KuiqKENJQY7U4hbXyP6DpcOTTVfsCljJuVWjrqAqP-Yq5wlhIlhfygUR-g-7K26LsI4VJkg617J5~OTtlrO3gpE~~eVwwbA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scatter plot of total chi-squared divergence, on the left side of Equation (2), versus the sum of the statistics on the right side, in 1 kb segments of the E. coli K12 genome. The line of unit slope through the origin is also drawn." path-from-xml="dsn02101.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5191460/dsn02101.jpeg" path-from-xml="dsn02101.jpeg" target="_blank">View large</a><a section="5191460" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn021/2/dsn02101.jpeg?Expires=2147483647&Signature=m0uzE05LoArZO1krPYzKScgHzklAPu43WpS-i6VsCJYRy9q83Fv~eMxyfkgaHRiiczbBCnLZIn46ig1Qa5HRZRFHbm9CYVeQZrNc2zvXozWwzjfF~amFQah4La1pJgldiOmktAFFC3TmlU3ISInfJ2q2q3Tp4RHHqWG2t6LVZN6cAfaW7HyB4tefCuDjAvGIOEfx~-8cq-eB1OqpV8bpKrKaYx74Afl5COWKDrPWPCW3gSr3Y-oij6-wPjW-9kF0YtYFlx8wmQxj0uj0QrgrDGm-o3uTWSrlLiUKL07zHPE9K2SCsfvJEyKmZ9C3dNy6pZM-azp0zJM5lmYhurBkhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5191460&ar=359884&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02101.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scatter plot of total chi-squared divergence, on the left side of Equation (2), versus the sum of the statistics on the right side, in 1 kb segments of the <em>E</em>. <em>coli</em> K12 genome. The line of unit slope through the origin is also drawn.</p></div></div>                        <a id="5191461" scrollto-destination="5191461"></a>
<div id="DSN021F2" data-id="DSN021F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn021/2/m_dsn02102.jpeg?Expires=2147483647&Signature=TkHarqqAY67TouML39cSd2ZCXXHp48E-5902W09QGB~I3D3MrTxWzc6q6NsliEg7unnNMSeFM2IEXS6HxUlmKsFB9RqiAhkEp4OkF~nAwCbn559bVhwAb0FgtoNvvef7OsNU954yialMb8P2Cv36~aBP2TSWthUgu4qFVnlpUAGoQP0nlc0lmuP7zAAJFK9XvMgFwwC16W0zbdYAngtISob3wE34iC~X4Fy9c7uYozQwLZLRbaAYlSDhPo3ii-sm8UFhvckYZeHmSKTKdT0c9QfhFKg6qcbJ-RClo3VUmLuYsfkvQvMEZD~8wsZ2-PSgM~Kn5-NjPoePifCc-DYKZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scatter plot of GC-divergence, describing fluctuations from the gross genomic G + C proportion, versus divergence with respect to DRA, in 1 kb segments of the E. coli K12 genome. The line of unit slope through the origin is also drawn." path-from-xml="dsn02102.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5191461/dsn02102.jpeg" path-from-xml="dsn02102.jpeg" target="_blank">View large</a><a section="5191461" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn021/2/dsn02102.jpeg?Expires=2147483647&Signature=QhnNFWP6rVUW1WkL6nykUvQpWXrFD1hS-Bll19Zfq3c8WQl53MKWTYl4wNsqwYT~-lDSedfWUxmCkc9uU5M3Z9~IjU3UhIUxscDXcfREgXdIZNfCmMaMAX2wmepOV4ehxqrDumfPp5Id-txHicL1ve31zmgXz8SzaOuBsQ2xiLtmwh86OCJ45ZK~oXaHDMLoc9Q2eX14AHQyTP3kDO5DoEbeCDXRjNzcmkHdUFC9uFi6dF~dGlMMImejuUWVooFZvn2jHs4wBygCZwXurLV4cqYG5ZG38WHSuz0kmjluHBwsOCXxckTN1tDdn-j3DJRLM3AymPPvDx90-GNROYYkTA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5191461&ar=359884&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02102.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scatter plot of GC-divergence, describing fluctuations from the gross genomic G + C proportion, versus divergence with respect to DRA, in 1 kb segments of the <em>E</em>. <em>coli</em> K12 genome. The line of unit slope through the origin is also drawn.</p></div></div><div content-id="DSN021F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn021/2/m_dsn02102.jpeg?Expires=2147483647&Signature=TkHarqqAY67TouML39cSd2ZCXXHp48E-5902W09QGB~I3D3MrTxWzc6q6NsliEg7unnNMSeFM2IEXS6HxUlmKsFB9RqiAhkEp4OkF~nAwCbn559bVhwAb0FgtoNvvef7OsNU954yialMb8P2Cv36~aBP2TSWthUgu4qFVnlpUAGoQP0nlc0lmuP7zAAJFK9XvMgFwwC16W0zbdYAngtISob3wE34iC~X4Fy9c7uYozQwLZLRbaAYlSDhPo3ii-sm8UFhvckYZeHmSKTKdT0c9QfhFKg6qcbJ-RClo3VUmLuYsfkvQvMEZD~8wsZ2-PSgM~Kn5-NjPoePifCc-DYKZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scatter plot of GC-divergence, describing fluctuations from the gross genomic G + C proportion, versus divergence with respect to DRA, in 1 kb segments of the E. coli K12 genome. The line of unit slope through the origin is also drawn." path-from-xml="dsn02102.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5191461/dsn02102.jpeg" path-from-xml="dsn02102.jpeg" target="_blank">View large</a><a section="5191461" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn021/2/dsn02102.jpeg?Expires=2147483647&Signature=QhnNFWP6rVUW1WkL6nykUvQpWXrFD1hS-Bll19Zfq3c8WQl53MKWTYl4wNsqwYT~-lDSedfWUxmCkc9uU5M3Z9~IjU3UhIUxscDXcfREgXdIZNfCmMaMAX2wmepOV4ehxqrDumfPp5Id-txHicL1ve31zmgXz8SzaOuBsQ2xiLtmwh86OCJ45ZK~oXaHDMLoc9Q2eX14AHQyTP3kDO5DoEbeCDXRjNzcmkHdUFC9uFi6dF~dGlMMImejuUWVooFZvn2jHs4wBygCZwXurLV4cqYG5ZG38WHSuz0kmjluHBwsOCXxckTN1tDdn-j3DJRLM3AymPPvDx90-GNROYYkTA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5191461&ar=359884&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02102.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scatter plot of GC-divergence, describing fluctuations from the gross genomic G + C proportion, versus divergence with respect to DRA, in 1 kb segments of the <em>E</em>. <em>coli</em> K12 genome. The line of unit slope through the origin is also drawn.</p></div></div><p>A classic investigation of base compositional structure drew conclusions from two genomes, <em>E</em>. <em>coli</em> and human, and concluded that a viable stochastic model of both would probably fit ‘many or all of genomes’.<sup><a reveal-id="DSN021C26" data-open="DSN021C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Subsequent research has shown that bacterial genomes have highly variable and idiosyncratic compositions. Thus it is important to examine some other species by the same methods. <em>Escherichia</em><em>coli</em> is exceptional in having a nearly uniform mononucleotide composition. We select the linear chromosome of the spirochete <em>Borrelia burgdorferi</em> because it was found most homogeneous in G + C when viewed through windows of a certain size.<sup><a reveal-id="DSN021C27" data-open="DSN021C27" class="link link-ref link-reveal xref-bibr">27</a></sup> The cyanobacterium <em>Synechocystis</em> was judged to exhibit no detectable replication-associated strand bias.<sup><a reveal-id="DSN021C28" data-open="DSN021C28" class="link link-ref link-reveal xref-bibr">28</a></sup><em>Mycobacterium tuberculosis</em> is included because it is AT-rich. Except for <em>S</em>. <em>enterica</em>, the other eight genomes in Table <a reveal-id="DSN021TB1" data-open="DSN021TB1" class="link link-reveal link-table xref-fig">1</a> are <em>ad hoc</em> choices. Summary statistics for these 12 bacteria are presented in Table <a reveal-id="DSN021TB2" data-open="DSN021TB2" class="link link-reveal link-table xref-fig">2</a>.</p>                        <a id="5191463" scrollto-destination="5191463"></a>
<div content-id="DSN021TB1" class="table-modal"><div class="table-wrap"><div id="DSN021TB1" class="table-wrap-title" data-id="DSN021TB1"><div class="label -label">Table 1</div><div class="caption"><p>Twelve selected bacterial genomes indexed by serial number (SN), identified by species, strain and reference sequence number (Ref.Seq.No)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>SN </th><th>Species </th><th>Strain </th><th>NCBI Ref.Seq.No. </th><th>Length (kb) </th><th>G + C (%) </th></tr></thead><tbody><tr><td>1 </td><td><em>Bacillus subtilis</em> </td><td>168 </td><td>000964.2 </td><td>4214 </td><td>43 </td></tr><tr><td>2 </td><td><em>Borrelia burgdorferi</em> </td><td>B31 </td><td>001318.1 </td><td>910 </td><td>28 </td></tr><tr><td>3 </td><td><em>Campylobacter jejuni</em> </td><td>RM221 </td><td>003912.7 </td><td>1777 </td><td>30 </td></tr><tr><td>4 </td><td><em>Chlamydophilia pneumoniae</em> </td><td>J138 </td><td>002491.1 </td><td>1226 </td><td>42 </td></tr><tr><td>5 </td><td><em>Escherichia coli</em> </td><td>K12 </td><td>000913.2 </td><td>4639 </td><td>50 </td></tr><tr><td>6 </td><td><em>Haemophilus influenzae</em> </td><td>Rd KW20 </td><td>000907.1 </td><td>1830 </td><td>38 </td></tr><tr><td>7 </td><td><em>Helicobacter pylori</em> </td><td>J99 </td><td>000921.1 </td><td>1643 </td><td>39 </td></tr><tr><td>8 </td><td><em>Mycobacterium tuberculosis</em> </td><td>CDC1551 </td><td>002755.2 </td><td>4403 </td><td>65 </td></tr><tr><td>9 </td><td><em>Mycoplasma genitalium</em> </td><td>G37 </td><td>000908.2 </td><td>580 </td><td>31 </td></tr><tr><td>10 </td><td><em>Salmonella enterica</em> </td><td>CT18 </td><td>003198.1 </td><td>4809 </td><td>52 </td></tr><tr><td>11 </td><td><em>Staphylococcus aureus</em> </td><td>N315 </td><td>002745.2 </td><td>2814 </td><td>32 </td></tr><tr><td>12 </td><td><em>Synechocystis sp</em>. </td><td>PC6803 </td><td>000911.1 </td><td>3573 </td><td>47 </td></tr></tbody></table></div><div content-id="DSN021TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>SN </th><th>Species </th><th>Strain </th><th>NCBI Ref.Seq.No. </th><th>Length (kb) </th><th>G + C (%) </th></tr></thead><tbody><tr><td>1 </td><td><em>Bacillus subtilis</em> </td><td>168 </td><td>000964.2 </td><td>4214 </td><td>43 </td></tr><tr><td>2 </td><td><em>Borrelia burgdorferi</em> </td><td>B31 </td><td>001318.1 </td><td>910 </td><td>28 </td></tr><tr><td>3 </td><td><em>Campylobacter jejuni</em> </td><td>RM221 </td><td>003912.7 </td><td>1777 </td><td>30 </td></tr><tr><td>4 </td><td><em>Chlamydophilia pneumoniae</em> </td><td>J138 </td><td>002491.1 </td><td>1226 </td><td>42 </td></tr><tr><td>5 </td><td><em>Escherichia coli</em> </td><td>K12 </td><td>000913.2 </td><td>4639 </td><td>50 </td></tr><tr><td>6 </td><td><em>Haemophilus influenzae</em> </td><td>Rd KW20 </td><td>000907.1 </td><td>1830 </td><td>38 </td></tr><tr><td>7 </td><td><em>Helicobacter pylori</em> </td><td>J99 </td><td>000921.1 </td><td>1643 </td><td>39 </td></tr><tr><td>8 </td><td><em>Mycobacterium tuberculosis</em> </td><td>CDC1551 </td><td>002755.2 </td><td>4403 </td><td>65 </td></tr><tr><td>9 </td><td><em>Mycoplasma genitalium</em> </td><td>G37 </td><td>000908.2 </td><td>580 </td><td>31 </td></tr><tr><td>10 </td><td><em>Salmonella enterica</em> </td><td>CT18 </td><td>003198.1 </td><td>4809 </td><td>52 </td></tr><tr><td>11 </td><td><em>Staphylococcus aureus</em> </td><td>N315 </td><td>002745.2 </td><td>2814 </td><td>32 </td></tr><tr><td>12 </td><td><em>Synechocystis sp</em>. </td><td>PC6803 </td><td>000911.1 </td><td>3573 </td><td>47 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5191463" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSN021TB1" class="table-wrap-title" data-id="DSN021TB1"><div class="label -label">Table 1</div><div class="caption"><p>Twelve selected bacterial genomes indexed by serial number (SN), identified by species, strain and reference sequence number (Ref.Seq.No)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>SN </th><th>Species </th><th>Strain </th><th>NCBI Ref.Seq.No. </th><th>Length (kb) </th><th>G + C (%) </th></tr></thead><tbody><tr><td>1 </td><td><em>Bacillus subtilis</em> </td><td>168 </td><td>000964.2 </td><td>4214 </td><td>43 </td></tr><tr><td>2 </td><td><em>Borrelia burgdorferi</em> </td><td>B31 </td><td>001318.1 </td><td>910 </td><td>28 </td></tr><tr><td>3 </td><td><em>Campylobacter jejuni</em> </td><td>RM221 </td><td>003912.7 </td><td>1777 </td><td>30 </td></tr><tr><td>4 </td><td><em>Chlamydophilia pneumoniae</em> </td><td>J138 </td><td>002491.1 </td><td>1226 </td><td>42 </td></tr><tr><td>5 </td><td><em>Escherichia coli</em> </td><td>K12 </td><td>000913.2 </td><td>4639 </td><td>50 </td></tr><tr><td>6 </td><td><em>Haemophilus influenzae</em> </td><td>Rd KW20 </td><td>000907.1 </td><td>1830 </td><td>38 </td></tr><tr><td>7 </td><td><em>Helicobacter pylori</em> </td><td>J99 </td><td>000921.1 </td><td>1643 </td><td>39 </td></tr><tr><td>8 </td><td><em>Mycobacterium tuberculosis</em> </td><td>CDC1551 </td><td>002755.2 </td><td>4403 </td><td>65 </td></tr><tr><td>9 </td><td><em>Mycoplasma genitalium</em> </td><td>G37 </td><td>000908.2 </td><td>580 </td><td>31 </td></tr><tr><td>10 </td><td><em>Salmonella enterica</em> </td><td>CT18 </td><td>003198.1 </td><td>4809 </td><td>52 </td></tr><tr><td>11 </td><td><em>Staphylococcus aureus</em> </td><td>N315 </td><td>002745.2 </td><td>2814 </td><td>32 </td></tr><tr><td>12 </td><td><em>Synechocystis sp</em>. </td><td>PC6803 </td><td>000911.1 </td><td>3573 </td><td>47 </td></tr></tbody></table></div><div content-id="DSN021TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>SN </th><th>Species </th><th>Strain </th><th>NCBI Ref.Seq.No. </th><th>Length (kb) </th><th>G + C (%) </th></tr></thead><tbody><tr><td>1 </td><td><em>Bacillus subtilis</em> </td><td>168 </td><td>000964.2 </td><td>4214 </td><td>43 </td></tr><tr><td>2 </td><td><em>Borrelia burgdorferi</em> </td><td>B31 </td><td>001318.1 </td><td>910 </td><td>28 </td></tr><tr><td>3 </td><td><em>Campylobacter jejuni</em> </td><td>RM221 </td><td>003912.7 </td><td>1777 </td><td>30 </td></tr><tr><td>4 </td><td><em>Chlamydophilia pneumoniae</em> </td><td>J138 </td><td>002491.1 </td><td>1226 </td><td>42 </td></tr><tr><td>5 </td><td><em>Escherichia coli</em> </td><td>K12 </td><td>000913.2 </td><td>4639 </td><td>50 </td></tr><tr><td>6 </td><td><em>Haemophilus influenzae</em> </td><td>Rd KW20 </td><td>000907.1 </td><td>1830 </td><td>38 </td></tr><tr><td>7 </td><td><em>Helicobacter pylori</em> </td><td>J99 </td><td>000921.1 </td><td>1643 </td><td>39 </td></tr><tr><td>8 </td><td><em>Mycobacterium tuberculosis</em> </td><td>CDC1551 </td><td>002755.2 </td><td>4403 </td><td>65 </td></tr><tr><td>9 </td><td><em>Mycoplasma genitalium</em> </td><td>G37 </td><td>000908.2 </td><td>580 </td><td>31 </td></tr><tr><td>10 </td><td><em>Salmonella enterica</em> </td><td>CT18 </td><td>003198.1 </td><td>4809 </td><td>52 </td></tr><tr><td>11 </td><td><em>Staphylococcus aureus</em> </td><td>N315 </td><td>002745.2 </td><td>2814 </td><td>32 </td></tr><tr><td>12 </td><td><em>Synechocystis sp</em>. </td><td>PC6803 </td><td>000911.1 </td><td>3573 </td><td>47 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5191463" target="_blank">View Large</a></div></div>                        <a id="5191464" scrollto-destination="5191464"></a>
<div content-id="DSN021TB2" class="table-modal"><div class="table-wrap"><div id="DSN021TB2" class="table-wrap-title" data-id="DSN021TB2"><div class="label -label">Table 2</div><div class="caption"><p>Summary statistics for 12 selected bacterial genomes indexed by serial number (SN) as in Table <a reveal-id="DSN021TB1" data-open="DSN021TB1" class="link link-reveal link-table xref-fig">1</a></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>SN </th><th colspan="3">Mean divergence<hr /> </th><th>MAD </th><th colspan="3">ER 95%<hr /> </th><th colspan="3">ER 99.5%<hr /> </th></tr><tr><th> </th><th>Total </th><th>DRA </th><th>G + C </th><th> </th><th>Total </th><th>DRA </th><th>G + C </th><th>TOTAL </th><th>DRA </th><th>G + C </th></tr></thead><tbody><tr><td>1 </td><td>25.8 </td><td>10.8 </td><td>15.9 </td><td>2.9 </td><td>47.2 </td><td>27.9 </td><td>43.2 </td><td>32.4 </td><td>13.4 </td><td>32.2 </td></tr><tr><td>2 </td><td>21.3 </td><td>9.7 </td><td>12.0 </td><td>3.9 </td><td>42.2 </td><td>22.7 </td><td>39.6 </td><td>25.2 </td><td>7.8 </td><td>28.4 </td></tr><tr><td>3 </td><td>21.7 </td><td>10.4 </td><td>12.0 </td><td>3.9 </td><td>45.1 </td><td>26.2 </td><td>40.2 </td><td>28.0 </td><td>10.9 </td><td>27.4 </td></tr><tr><td>4 </td><td>15.6 </td><td>9.6 </td><td>5.9 </td><td>1.7 </td><td>34.8 </td><td>22.7 </td><td>26.4 </td><td>17.2 </td><td>8.1 </td><td>14.0 </td></tr><tr><td>5 </td><td>27.1 </td><td>9.8 </td><td>18.4 </td><td>3.4 </td><td>48.0 </td><td>24.3 </td><td>46.4 </td><td>33.5 </td><td>10.2 </td><td>34.0 </td></tr><tr><td>5(n) </td><td>27.5 </td><td>9.7 </td><td>18.9 </td><td>3.4 </td><td>48.2 </td><td>24.2 </td><td>46.4 </td><td>33.5 </td><td>10.2 </td><td>33.9 </td></tr><tr><td>5(e) </td><td>20.4 </td><td>11.4 </td><td>9.5 </td><td>2.9 </td><td>45.5 </td><td>14.6 </td><td>37.5 </td><td>21.6 </td><td>4.3 </td><td>23.2 </td></tr><tr><td>6 </td><td>19.9 </td><td>10.6 </td><td>10.2 </td><td>3.6 </td><td>37.0 </td><td>27.1 </td><td>31.0 </td><td>23.2 </td><td>11.6 </td><td>20.3 </td></tr><tr><td>7 </td><td>19.1 </td><td>11.4 </td><td>9.2 </td><td>5.5 </td><td>39.7 </td><td>29.4 </td><td>32.0 </td><td>25.0 </td><td>14.3 </td><td>21.7 </td></tr><tr><td>8 </td><td>20.2 </td><td>9.5 </td><td>10.3 </td><td>2.4 </td><td>32.1 </td><td>20.1 </td><td>27.8 </td><td>19.1 </td><td>8.4 </td><td>18.4 </td></tr><tr><td>9 </td><td>26.2 </td><td>11.3 </td><td>14.0 </td><td>3.4 </td><td>49.3 </td><td>28.6 </td><td>41.4 </td><td>33.6 </td><td>13.9 </td><td>30.7 </td></tr><tr><td>10 </td><td>30.4 </td><td>10.8 </td><td>21.9 </td><td>5.2 </td><td>52.8 </td><td>28.3 </td><td>49.3 </td><td>38.2 </td><td>12.8 </td><td>38.2 </td></tr><tr><td>11 </td><td>21.7 </td><td>11.3 </td><td>10.2 </td><td>2.6 </td><td>40.7 </td><td>28.6 </td><td>32.9 </td><td>25.6 </td><td>12.9 </td><td>21.2 </td></tr><tr><td>12 </td><td>24.4 </td><td>10.0 </td><td>16.6 </td><td>5.7 </td><td>43.8 </td><td>22.2 </td><td>44.6 </td><td>28.6 </td><td>9.8 </td><td>32.4 </td></tr><tr><td>Average </td><td>22.8 </td><td>10.4 </td><td>13.0 </td><td>3.7 </td><td>42.7 </td><td>25.7 </td><td>37.9 </td><td>27.5 </td><td>11.2 </td><td>26.6 </td></tr></tbody></table></div><div content-id="DSN021TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>SN </th><th colspan="3">Mean divergence<hr /> </th><th>MAD </th><th colspan="3">ER 95%<hr /> </th><th colspan="3">ER 99.5%<hr /> </th></tr><tr><th> </th><th>Total </th><th>DRA </th><th>G + C </th><th> </th><th>Total </th><th>DRA </th><th>G + C </th><th>TOTAL </th><th>DRA </th><th>G + C </th></tr></thead><tbody><tr><td>1 </td><td>25.8 </td><td>10.8 </td><td>15.9 </td><td>2.9 </td><td>47.2 </td><td>27.9 </td><td>43.2 </td><td>32.4 </td><td>13.4 </td><td>32.2 </td></tr><tr><td>2 </td><td>21.3 </td><td>9.7 </td><td>12.0 </td><td>3.9 </td><td>42.2 </td><td>22.7 </td><td>39.6 </td><td>25.2 </td><td>7.8 </td><td>28.4 </td></tr><tr><td>3 </td><td>21.7 </td><td>10.4 </td><td>12.0 </td><td>3.9 </td><td>45.1 </td><td>26.2 </td><td>40.2 </td><td>28.0 </td><td>10.9 </td><td>27.4 </td></tr><tr><td>4 </td><td>15.6 </td><td>9.6 </td><td>5.9 </td><td>1.7 </td><td>34.8 </td><td>22.7 </td><td>26.4 </td><td>17.2 </td><td>8.1 </td><td>14.0 </td></tr><tr><td>5 </td><td>27.1 </td><td>9.8 </td><td>18.4 </td><td>3.4 </td><td>48.0 </td><td>24.3 </td><td>46.4 </td><td>33.5 </td><td>10.2 </td><td>34.0 </td></tr><tr><td>5(n) </td><td>27.5 </td><td>9.7 </td><td>18.9 </td><td>3.4 </td><td>48.2 </td><td>24.2 </td><td>46.4 </td><td>33.5 </td><td>10.2 </td><td>33.9 </td></tr><tr><td>5(e) </td><td>20.4 </td><td>11.4 </td><td>9.5 </td><td>2.9 </td><td>45.5 </td><td>14.6 </td><td>37.5 </td><td>21.6 </td><td>4.3 </td><td>23.2 </td></tr><tr><td>6 </td><td>19.9 </td><td>10.6 </td><td>10.2 </td><td>3.6 </td><td>37.0 </td><td>27.1 </td><td>31.0 </td><td>23.2 </td><td>11.6 </td><td>20.3 </td></tr><tr><td>7 </td><td>19.1 </td><td>11.4 </td><td>9.2 </td><td>5.5 </td><td>39.7 </td><td>29.4 </td><td>32.0 </td><td>25.0 </td><td>14.3 </td><td>21.7 </td></tr><tr><td>8 </td><td>20.2 </td><td>9.5 </td><td>10.3 </td><td>2.4 </td><td>32.1 </td><td>20.1 </td><td>27.8 </td><td>19.1 </td><td>8.4 </td><td>18.4 </td></tr><tr><td>9 </td><td>26.2 </td><td>11.3 </td><td>14.0 </td><td>3.4 </td><td>49.3 </td><td>28.6 </td><td>41.4 </td><td>33.6 </td><td>13.9 </td><td>30.7 </td></tr><tr><td>10 </td><td>30.4 </td><td>10.8 </td><td>21.9 </td><td>5.2 </td><td>52.8 </td><td>28.3 </td><td>49.3 </td><td>38.2 </td><td>12.8 </td><td>38.2 </td></tr><tr><td>11 </td><td>21.7 </td><td>11.3 </td><td>10.2 </td><td>2.6 </td><td>40.7 </td><td>28.6 </td><td>32.9 </td><td>25.6 </td><td>12.9 </td><td>21.2 </td></tr><tr><td>12 </td><td>24.4 </td><td>10.0 </td><td>16.6 </td><td>5.7 </td><td>43.8 </td><td>22.2 </td><td>44.6 </td><td>28.6 </td><td>9.8 </td><td>32.4 </td></tr><tr><td>Average </td><td>22.8 </td><td>10.4 </td><td>13.0 </td><td>3.7 </td><td>42.7 </td><td>25.7 </td><td>37.9 </td><td>27.5 </td><td>11.2 </td><td>26.6 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Mean values of the chi-squared divergence statistics in Equation (2) are shown with the MAD. ERs at two percentile points are listed. Averages across all species are shown in the last row.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5191464" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSN021TB2" class="table-wrap-title" data-id="DSN021TB2"><div class="label -label">Table 2</div><div class="caption"><p>Summary statistics for 12 selected bacterial genomes indexed by serial number (SN) as in Table <a reveal-id="DSN021TB1" data-open="DSN021TB1" class="link link-reveal link-table xref-fig">1</a></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>SN </th><th colspan="3">Mean divergence<hr /> </th><th>MAD </th><th colspan="3">ER 95%<hr /> </th><th colspan="3">ER 99.5%<hr /> </th></tr><tr><th> </th><th>Total </th><th>DRA </th><th>G + C </th><th> </th><th>Total </th><th>DRA </th><th>G + C </th><th>TOTAL </th><th>DRA </th><th>G + C </th></tr></thead><tbody><tr><td>1 </td><td>25.8 </td><td>10.8 </td><td>15.9 </td><td>2.9 </td><td>47.2 </td><td>27.9 </td><td>43.2 </td><td>32.4 </td><td>13.4 </td><td>32.2 </td></tr><tr><td>2 </td><td>21.3 </td><td>9.7 </td><td>12.0 </td><td>3.9 </td><td>42.2 </td><td>22.7 </td><td>39.6 </td><td>25.2 </td><td>7.8 </td><td>28.4 </td></tr><tr><td>3 </td><td>21.7 </td><td>10.4 </td><td>12.0 </td><td>3.9 </td><td>45.1 </td><td>26.2 </td><td>40.2 </td><td>28.0 </td><td>10.9 </td><td>27.4 </td></tr><tr><td>4 </td><td>15.6 </td><td>9.6 </td><td>5.9 </td><td>1.7 </td><td>34.8 </td><td>22.7 </td><td>26.4 </td><td>17.2 </td><td>8.1 </td><td>14.0 </td></tr><tr><td>5 </td><td>27.1 </td><td>9.8 </td><td>18.4 </td><td>3.4 </td><td>48.0 </td><td>24.3 </td><td>46.4 </td><td>33.5 </td><td>10.2 </td><td>34.0 </td></tr><tr><td>5(n) </td><td>27.5 </td><td>9.7 </td><td>18.9 </td><td>3.4 </td><td>48.2 </td><td>24.2 </td><td>46.4 </td><td>33.5 </td><td>10.2 </td><td>33.9 </td></tr><tr><td>5(e) </td><td>20.4 </td><td>11.4 </td><td>9.5 </td><td>2.9 </td><td>45.5 </td><td>14.6 </td><td>37.5 </td><td>21.6 </td><td>4.3 </td><td>23.2 </td></tr><tr><td>6 </td><td>19.9 </td><td>10.6 </td><td>10.2 </td><td>3.6 </td><td>37.0 </td><td>27.1 </td><td>31.0 </td><td>23.2 </td><td>11.6 </td><td>20.3 </td></tr><tr><td>7 </td><td>19.1 </td><td>11.4 </td><td>9.2 </td><td>5.5 </td><td>39.7 </td><td>29.4 </td><td>32.0 </td><td>25.0 </td><td>14.3 </td><td>21.7 </td></tr><tr><td>8 </td><td>20.2 </td><td>9.5 </td><td>10.3 </td><td>2.4 </td><td>32.1 </td><td>20.1 </td><td>27.8 </td><td>19.1 </td><td>8.4 </td><td>18.4 </td></tr><tr><td>9 </td><td>26.2 </td><td>11.3 </td><td>14.0 </td><td>3.4 </td><td>49.3 </td><td>28.6 </td><td>41.4 </td><td>33.6 </td><td>13.9 </td><td>30.7 </td></tr><tr><td>10 </td><td>30.4 </td><td>10.8 </td><td>21.9 </td><td>5.2 </td><td>52.8 </td><td>28.3 </td><td>49.3 </td><td>38.2 </td><td>12.8 </td><td>38.2 </td></tr><tr><td>11 </td><td>21.7 </td><td>11.3 </td><td>10.2 </td><td>2.6 </td><td>40.7 </td><td>28.6 </td><td>32.9 </td><td>25.6 </td><td>12.9 </td><td>21.2 </td></tr><tr><td>12 </td><td>24.4 </td><td>10.0 </td><td>16.6 </td><td>5.7 </td><td>43.8 </td><td>22.2 </td><td>44.6 </td><td>28.6 </td><td>9.8 </td><td>32.4 </td></tr><tr><td>Average </td><td>22.8 </td><td>10.4 </td><td>13.0 </td><td>3.7 </td><td>42.7 </td><td>25.7 </td><td>37.9 </td><td>27.5 </td><td>11.2 </td><td>26.6 </td></tr></tbody></table></div><div content-id="DSN021TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>SN </th><th colspan="3">Mean divergence<hr /> </th><th>MAD </th><th colspan="3">ER 95%<hr /> </th><th colspan="3">ER 99.5%<hr /> </th></tr><tr><th> </th><th>Total </th><th>DRA </th><th>G + C </th><th> </th><th>Total </th><th>DRA </th><th>G + C </th><th>TOTAL </th><th>DRA </th><th>G + C </th></tr></thead><tbody><tr><td>1 </td><td>25.8 </td><td>10.8 </td><td>15.9 </td><td>2.9 </td><td>47.2 </td><td>27.9 </td><td>43.2 </td><td>32.4 </td><td>13.4 </td><td>32.2 </td></tr><tr><td>2 </td><td>21.3 </td><td>9.7 </td><td>12.0 </td><td>3.9 </td><td>42.2 </td><td>22.7 </td><td>39.6 </td><td>25.2 </td><td>7.8 </td><td>28.4 </td></tr><tr><td>3 </td><td>21.7 </td><td>10.4 </td><td>12.0 </td><td>3.9 </td><td>45.1 </td><td>26.2 </td><td>40.2 </td><td>28.0 </td><td>10.9 </td><td>27.4 </td></tr><tr><td>4 </td><td>15.6 </td><td>9.6 </td><td>5.9 </td><td>1.7 </td><td>34.8 </td><td>22.7 </td><td>26.4 </td><td>17.2 </td><td>8.1 </td><td>14.0 </td></tr><tr><td>5 </td><td>27.1 </td><td>9.8 </td><td>18.4 </td><td>3.4 </td><td>48.0 </td><td>24.3 </td><td>46.4 </td><td>33.5 </td><td>10.2 </td><td>34.0 </td></tr><tr><td>5(n) </td><td>27.5 </td><td>9.7 </td><td>18.9 </td><td>3.4 </td><td>48.2 </td><td>24.2 </td><td>46.4 </td><td>33.5 </td><td>10.2 </td><td>33.9 </td></tr><tr><td>5(e) </td><td>20.4 </td><td>11.4 </td><td>9.5 </td><td>2.9 </td><td>45.5 </td><td>14.6 </td><td>37.5 </td><td>21.6 </td><td>4.3 </td><td>23.2 </td></tr><tr><td>6 </td><td>19.9 </td><td>10.6 </td><td>10.2 </td><td>3.6 </td><td>37.0 </td><td>27.1 </td><td>31.0 </td><td>23.2 </td><td>11.6 </td><td>20.3 </td></tr><tr><td>7 </td><td>19.1 </td><td>11.4 </td><td>9.2 </td><td>5.5 </td><td>39.7 </td><td>29.4 </td><td>32.0 </td><td>25.0 </td><td>14.3 </td><td>21.7 </td></tr><tr><td>8 </td><td>20.2 </td><td>9.5 </td><td>10.3 </td><td>2.4 </td><td>32.1 </td><td>20.1 </td><td>27.8 </td><td>19.1 </td><td>8.4 </td><td>18.4 </td></tr><tr><td>9 </td><td>26.2 </td><td>11.3 </td><td>14.0 </td><td>3.4 </td><td>49.3 </td><td>28.6 </td><td>41.4 </td><td>33.6 </td><td>13.9 </td><td>30.7 </td></tr><tr><td>10 </td><td>30.4 </td><td>10.8 </td><td>21.9 </td><td>5.2 </td><td>52.8 </td><td>28.3 </td><td>49.3 </td><td>38.2 </td><td>12.8 </td><td>38.2 </td></tr><tr><td>11 </td><td>21.7 </td><td>11.3 </td><td>10.2 </td><td>2.6 </td><td>40.7 </td><td>28.6 </td><td>32.9 </td><td>25.6 </td><td>12.9 </td><td>21.2 </td></tr><tr><td>12 </td><td>24.4 </td><td>10.0 </td><td>16.6 </td><td>5.7 </td><td>43.8 </td><td>22.2 </td><td>44.6 </td><td>28.6 </td><td>9.8 </td><td>32.4 </td></tr><tr><td>Average </td><td>22.8 </td><td>10.4 </td><td>13.0 </td><td>3.7 </td><td>42.7 </td><td>25.7 </td><td>37.9 </td><td>27.5 </td><td>11.2 </td><td>26.6 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Mean values of the chi-squared divergence statistics in Equation (2) are shown with the MAD. ERs at two percentile points are listed. Averages across all species are shown in the last row.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5191464" target="_blank">View Large</a></div></div><p>Beyond statistical analysis by the method just described, we want to know if coding sequences undergo amelioration with respect to DRA at a rate that is relatively unaffected by essentiality. Accordingly, the 1 kb segments of the <em>E</em>. <em>coli</em> K12 sequence that are most anomalous in DRA-divergence are subjected to a phylogenetic analysis. The 20 highest ranking segments (largest DRA-divergences) are placed in serial order in Table <a reveal-id="DSN021TB3" data-open="DSN021TB3" class="link link-reveal link-table xref-fig">3</a>. Each segment overlaps zero, one or two genes in the protein table of the current annotation. (We suppose that HGT typically involves segments longer than 1 kb.) Horizontal transfer (HT) is indicated by ‘M’ when the gene is pro-phage, phage-like, an insertion element or other documented Mobile element. Otherwise, the translated coding sequence is retrieved from EcoCyc<sup><a reveal-id="DSN021C29" data-open="DSN021C29" class="link link-ref link-reveal xref-bibr">29</a></sup> and its best alignment in <em>S</em>. <em>enterica</em> (all complete genomes) is found by tBLASTn (via the National Center for Biotechnology website with default parameters) and the corresponding <em>E</em>-value is noted.<sup><a reveal-id="DSN021C30" data-open="DSN021C30" class="link link-ref link-reveal xref-bibr">30</a>,<a reveal-id="DSN021C31" data-open="DSN021C31" class="link link-ref link-reveal xref-bibr">31</a></sup> If <em>E</em> &gt; 0.1, the alignment is attributed to chance, and ‘+1’ is entered in the HT column to indicate true positive. If <em>E</em> ≤ 0.1 then the alignment is significant, the gene is assumed to have been inherited from the common ancestor, and the symbol ‘−1’ is entered in the HT column to indicate false positive. False positives are checked against the list of 302 essential gene candidates that were located on the chromosome of substrain MG1655 by testing knockout mutants.<sup><a reveal-id="DSN021C2" data-open="DSN021C2" class="link link-ref link-reveal xref-bibr">2</a></sup> When a false positive is an essential gene, ‘−1’ in the HT column is replaced by the abbreviated gene name (locus).</p>                        <a id="5191466" scrollto-destination="5191466"></a>
<div content-id="DSN021TB3" class="table-modal"><div class="table-wrap"><div id="DSN021TB3" class="table-wrap-title" data-id="DSN021TB3"><div class="label -label">Table 3</div><div class="caption"><p>Characterizing the 20 most anomalous 1 kb segments of the <em>E</em>. <em>coli</em> K12 genome based on dinucleotide signature dissimilarity as measured by (a) DRA-divergence, (b) delta-distance, (c) Euclidean distance and (d) the quadratic discriminant</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th># </th><th colspan="2">(a) chi-square<hr /> </th><th colspan="2">(b) delta-distance<hr /> </th><th colspan="2">(c) Euclidean<hr /> </th><th colspan="2">(d) quadratic<hr /> </th><th colspan="2">(e) G + C<hr /> </th></tr><tr><th> </th><th>loc. </th><th>HT </th><th>loc. </th><th>HT </th><th>loc. </th><th>HT </th><th>loc. </th><th>HT </th><th>loc. </th><th>HT </th></tr></thead><tbody><tr><td>1 </td><td>200 </td><td>yaeT </td><td>151 </td><td>−1 </td><td>151 </td><td>−1 </td><td>284 </td><td>M </td><td>583 </td><td>M </td></tr><tr><td>2 </td><td>227 </td><td>0 </td><td>227 </td><td>0 </td><td>274 </td><td>M </td><td>525 </td><td>M </td><td>584 </td><td>M </td></tr><tr><td>3 </td><td>393 </td><td>M </td><td>394 </td><td>−1 </td><td>575 </td><td>M </td><td>526 </td><td>M </td><td>1212 </td><td>−1, −1 </td></tr><tr><td>4 </td><td>394 </td><td>−1 </td><td>526 </td><td>M </td><td>777 </td><td>−1 </td><td>575 </td><td>M </td><td>1636 </td><td>M </td></tr><tr><td>5 </td><td>526 </td><td>M </td><td>777 </td><td>−1 </td><td>978 </td><td>mukB </td><td>777 </td><td>−1 </td><td>2102 </td><td>−1 </td></tr><tr><td>6 </td><td>777 </td><td>−1 </td><td>1287 </td><td>−1 </td><td>1142 </td><td>rne </td><td>1142 </td><td>rne </td><td>2105 </td><td>+1 </td></tr><tr><td>7 </td><td>978 </td><td>mukB </td><td>1427 </td><td>M </td><td>1395 </td><td>M </td><td>1427 </td><td>M </td><td>2468 </td><td>M </td></tr><tr><td>8 </td><td>1142 </td><td>rne </td><td>1465 </td><td>0 </td><td>1427 </td><td>M </td><td>1465 </td><td>0 </td><td>2773 </td><td>M </td></tr><tr><td>9 </td><td>1465 </td><td>0 </td><td>1527 </td><td>0 </td><td>1465 </td><td>0 </td><td>1527 </td><td>0 </td><td>2783 </td><td>0 </td></tr><tr><td>10 </td><td>1527 </td><td>0 </td><td>1707 </td><td>−1 </td><td>1527 </td><td>0 </td><td>1707 </td><td>−1 </td><td>2785 </td><td>0 </td></tr><tr><td>11 </td><td>1707 </td><td>−1 </td><td>2071 </td><td>M </td><td>2101 </td><td>M </td><td>2104 </td><td>+1 </td><td>2989 </td><td>+1 </td></tr><tr><td>12 </td><td>2071 </td><td>M </td><td>2072 </td><td>M </td><td>2104 </td><td>+1 </td><td>2105 </td><td>+1 </td><td>2990 </td><td>−1 </td></tr><tr><td>13 </td><td>2072 </td><td>M </td><td>2104 </td><td>+1 </td><td>2105 </td><td>+1 </td><td>2989 </td><td>+1 </td><td>2994 </td><td>0 </td></tr><tr><td>14 </td><td>2104 </td><td>+1 </td><td>2994 </td><td>0 </td><td>2994 </td><td>0 </td><td>2990 </td><td>−1 </td><td>3267 </td><td>+1, +1 </td></tr><tr><td>15 </td><td>3312 </td><td>−1, infB </td><td>3312 </td><td>−1, infB </td><td>3314 </td><td>−1 </td><td>2992 </td><td>−1 </td><td>3581 </td><td>+1 </td></tr><tr><td>16 </td><td>3450 </td><td>rplWD </td><td>3602 </td><td>ftsY </td><td>3450 </td><td>rplWD </td><td>2994 </td><td>0 </td><td>3797 </td><td>+1,+1 </td></tr><tr><td>17 </td><td>3602 </td><td>ftsY </td><td>3915 </td><td>−1 </td><td>3602 </td><td>ftsY </td><td>3602 </td><td>ftsY </td><td>3798 </td><td>−1 </td></tr><tr><td>18 </td><td>3915 </td><td>−1 </td><td>4058 </td><td>−1 </td><td>4121 </td><td>−1 </td><td>3620 </td><td>M </td><td>3803 </td><td>−1, +1 </td></tr><tr><td>19 </td><td>4058 </td><td>−1 </td><td>4187 </td><td>rpoC </td><td>4503 </td><td>M </td><td>4503 </td><td>M </td><td>4267 </td><td>−1 </td></tr><tr><td>20 </td><td>4181 </td><td>rpoB </td><td>4474 </td><td>−1, +1 </td><td>4504 </td><td>M </td><td>4504 </td><td>M, M </td><td>4475 </td><td>+1 </td></tr><tr><td>Genes </td><td colspan="2">19 </td><td colspan="2">18 </td><td colspan="2">18 </td><td colspan="2">18 </td><td colspan="2">21 </td></tr><tr><td>Errors </td><td colspan="2">14(8) </td><td colspan="2">12(3) </td><td colspan="2">9(5) </td><td colspan="2">6(2) </td><td colspan="2">7(0) </td></tr></tbody></table></div><div content-id="DSN021TB3" class="table-modal"><table><thead name="thead" align=""><tr><th># </th><th colspan="2">(a) chi-square<hr /> </th><th colspan="2">(b) delta-distance<hr /> </th><th colspan="2">(c) Euclidean<hr /> </th><th colspan="2">(d) quadratic<hr /> </th><th colspan="2">(e) G + C<hr /> </th></tr><tr><th> </th><th>loc. </th><th>HT </th><th>loc. </th><th>HT </th><th>loc. </th><th>HT </th><th>loc. </th><th>HT </th><th>loc. </th><th>HT </th></tr></thead><tbody><tr><td>1 </td><td>200 </td><td>yaeT </td><td>151 </td><td>−1 </td><td>151 </td><td>−1 </td><td>284 </td><td>M </td><td>583 </td><td>M </td></tr><tr><td>2 </td><td>227 </td><td>0 </td><td>227 </td><td>0 </td><td>274 </td><td>M </td><td>525 </td><td>M </td><td>584 </td><td>M </td></tr><tr><td>3 </td><td>393 </td><td>M </td><td>394 </td><td>−1 </td><td>575 </td><td>M </td><td>526 </td><td>M </td><td>1212 </td><td>−1, −1 </td></tr><tr><td>4 </td><td>394 </td><td>−1 </td><td>526 </td><td>M </td><td>777 </td><td>−1 </td><td>575 </td><td>M </td><td>1636 </td><td>M </td></tr><tr><td>5 </td><td>526 </td><td>M </td><td>777 </td><td>−1 </td><td>978 </td><td>mukB </td><td>777 </td><td>−1 </td><td>2102 </td><td>−1 </td></tr><tr><td>6 </td><td>777 </td><td>−1 </td><td>1287 </td><td>−1 </td><td>1142 </td><td>rne </td><td>1142 </td><td>rne </td><td>2105 </td><td>+1 </td></tr><tr><td>7 </td><td>978 </td><td>mukB </td><td>1427 </td><td>M </td><td>1395 </td><td>M </td><td>1427 </td><td>M </td><td>2468 </td><td>M </td></tr><tr><td>8 </td><td>1142 </td><td>rne </td><td>1465 </td><td>0 </td><td>1427 </td><td>M </td><td>1465 </td><td>0 </td><td>2773 </td><td>M </td></tr><tr><td>9 </td><td>1465 </td><td>0 </td><td>1527 </td><td>0 </td><td>1465 </td><td>0 </td><td>1527 </td><td>0 </td><td>2783 </td><td>0 </td></tr><tr><td>10 </td><td>1527 </td><td>0 </td><td>1707 </td><td>−1 </td><td>1527 </td><td>0 </td><td>1707 </td><td>−1 </td><td>2785 </td><td>0 </td></tr><tr><td>11 </td><td>1707 </td><td>−1 </td><td>2071 </td><td>M </td><td>2101 </td><td>M </td><td>2104 </td><td>+1 </td><td>2989 </td><td>+1 </td></tr><tr><td>12 </td><td>2071 </td><td>M </td><td>2072 </td><td>M </td><td>2104 </td><td>+1 </td><td>2105 </td><td>+1 </td><td>2990 </td><td>−1 </td></tr><tr><td>13 </td><td>2072 </td><td>M </td><td>2104 </td><td>+1 </td><td>2105 </td><td>+1 </td><td>2989 </td><td>+1 </td><td>2994 </td><td>0 </td></tr><tr><td>14 </td><td>2104 </td><td>+1 </td><td>2994 </td><td>0 </td><td>2994 </td><td>0 </td><td>2990 </td><td>−1 </td><td>3267 </td><td>+1, +1 </td></tr><tr><td>15 </td><td>3312 </td><td>−1, infB </td><td>3312 </td><td>−1, infB </td><td>3314 </td><td>−1 </td><td>2992 </td><td>−1 </td><td>3581 </td><td>+1 </td></tr><tr><td>16 </td><td>3450 </td><td>rplWD </td><td>3602 </td><td>ftsY </td><td>3450 </td><td>rplWD </td><td>2994 </td><td>0 </td><td>3797 </td><td>+1,+1 </td></tr><tr><td>17 </td><td>3602 </td><td>ftsY </td><td>3915 </td><td>−1 </td><td>3602 </td><td>ftsY </td><td>3602 </td><td>ftsY </td><td>3798 </td><td>−1 </td></tr><tr><td>18 </td><td>3915 </td><td>−1 </td><td>4058 </td><td>−1 </td><td>4121 </td><td>−1 </td><td>3620 </td><td>M </td><td>3803 </td><td>−1, +1 </td></tr><tr><td>19 </td><td>4058 </td><td>−1 </td><td>4187 </td><td>rpoC </td><td>4503 </td><td>M </td><td>4503 </td><td>M </td><td>4267 </td><td>−1 </td></tr><tr><td>20 </td><td>4181 </td><td>rpoB </td><td>4474 </td><td>−1, +1 </td><td>4504 </td><td>M </td><td>4504 </td><td>M, M </td><td>4475 </td><td>+1 </td></tr><tr><td>Genes </td><td colspan="2">19 </td><td colspan="2">18 </td><td colspan="2">18 </td><td colspan="2">18 </td><td colspan="2">21 </td></tr><tr><td>Errors </td><td colspan="2">14(8) </td><td colspan="2">12(3) </td><td colspan="2">9(5) </td><td colspan="2">6(2) </td><td colspan="2">7(0) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Each segment, indexed by location (loc.) in kb from the published origin, overlaps zero, one or two genes in the protein table. Intergenic segments are classified as ‘0’. HT is indicated by ‘+1’ or by ‘M’ (if the gene is a mobile element). False positives are indicated by gene locus (if essential) or by ‘−1’ (if not). False positives account for total errors (bottom row) and the number of essential genes is given (in parentheses). The analysis is repeated for the 20 most anomalous segments with respect to (e) GC-divergence.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5191466" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSN021TB3" class="table-wrap-title" data-id="DSN021TB3"><div class="label -label">Table 3</div><div class="caption"><p>Characterizing the 20 most anomalous 1 kb segments of the <em>E</em>. <em>coli</em> K12 genome based on dinucleotide signature dissimilarity as measured by (a) DRA-divergence, (b) delta-distance, (c) Euclidean distance and (d) the quadratic discriminant</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th># </th><th colspan="2">(a) chi-square<hr /> </th><th colspan="2">(b) delta-distance<hr /> </th><th colspan="2">(c) Euclidean<hr /> </th><th colspan="2">(d) quadratic<hr /> </th><th colspan="2">(e) G + C<hr /> </th></tr><tr><th> </th><th>loc. </th><th>HT </th><th>loc. </th><th>HT </th><th>loc. </th><th>HT </th><th>loc. </th><th>HT </th><th>loc. </th><th>HT </th></tr></thead><tbody><tr><td>1 </td><td>200 </td><td>yaeT </td><td>151 </td><td>−1 </td><td>151 </td><td>−1 </td><td>284 </td><td>M </td><td>583 </td><td>M </td></tr><tr><td>2 </td><td>227 </td><td>0 </td><td>227 </td><td>0 </td><td>274 </td><td>M </td><td>525 </td><td>M </td><td>584 </td><td>M </td></tr><tr><td>3 </td><td>393 </td><td>M </td><td>394 </td><td>−1 </td><td>575 </td><td>M </td><td>526 </td><td>M </td><td>1212 </td><td>−1, −1 </td></tr><tr><td>4 </td><td>394 </td><td>−1 </td><td>526 </td><td>M </td><td>777 </td><td>−1 </td><td>575 </td><td>M </td><td>1636 </td><td>M </td></tr><tr><td>5 </td><td>526 </td><td>M </td><td>777 </td><td>−1 </td><td>978 </td><td>mukB </td><td>777 </td><td>−1 </td><td>2102 </td><td>−1 </td></tr><tr><td>6 </td><td>777 </td><td>−1 </td><td>1287 </td><td>−1 </td><td>1142 </td><td>rne </td><td>1142 </td><td>rne </td><td>2105 </td><td>+1 </td></tr><tr><td>7 </td><td>978 </td><td>mukB </td><td>1427 </td><td>M </td><td>1395 </td><td>M </td><td>1427 </td><td>M </td><td>2468 </td><td>M </td></tr><tr><td>8 </td><td>1142 </td><td>rne </td><td>1465 </td><td>0 </td><td>1427 </td><td>M </td><td>1465 </td><td>0 </td><td>2773 </td><td>M </td></tr><tr><td>9 </td><td>1465 </td><td>0 </td><td>1527 </td><td>0 </td><td>1465 </td><td>0 </td><td>1527 </td><td>0 </td><td>2783 </td><td>0 </td></tr><tr><td>10 </td><td>1527 </td><td>0 </td><td>1707 </td><td>−1 </td><td>1527 </td><td>0 </td><td>1707 </td><td>−1 </td><td>2785 </td><td>0 </td></tr><tr><td>11 </td><td>1707 </td><td>−1 </td><td>2071 </td><td>M </td><td>2101 </td><td>M </td><td>2104 </td><td>+1 </td><td>2989 </td><td>+1 </td></tr><tr><td>12 </td><td>2071 </td><td>M </td><td>2072 </td><td>M </td><td>2104 </td><td>+1 </td><td>2105 </td><td>+1 </td><td>2990 </td><td>−1 </td></tr><tr><td>13 </td><td>2072 </td><td>M </td><td>2104 </td><td>+1 </td><td>2105 </td><td>+1 </td><td>2989 </td><td>+1 </td><td>2994 </td><td>0 </td></tr><tr><td>14 </td><td>2104 </td><td>+1 </td><td>2994 </td><td>0 </td><td>2994 </td><td>0 </td><td>2990 </td><td>−1 </td><td>3267 </td><td>+1, +1 </td></tr><tr><td>15 </td><td>3312 </td><td>−1, infB </td><td>3312 </td><td>−1, infB </td><td>3314 </td><td>−1 </td><td>2992 </td><td>−1 </td><td>3581 </td><td>+1 </td></tr><tr><td>16 </td><td>3450 </td><td>rplWD </td><td>3602 </td><td>ftsY </td><td>3450 </td><td>rplWD </td><td>2994 </td><td>0 </td><td>3797 </td><td>+1,+1 </td></tr><tr><td>17 </td><td>3602 </td><td>ftsY </td><td>3915 </td><td>−1 </td><td>3602 </td><td>ftsY </td><td>3602 </td><td>ftsY </td><td>3798 </td><td>−1 </td></tr><tr><td>18 </td><td>3915 </td><td>−1 </td><td>4058 </td><td>−1 </td><td>4121 </td><td>−1 </td><td>3620 </td><td>M </td><td>3803 </td><td>−1, +1 </td></tr><tr><td>19 </td><td>4058 </td><td>−1 </td><td>4187 </td><td>rpoC </td><td>4503 </td><td>M </td><td>4503 </td><td>M </td><td>4267 </td><td>−1 </td></tr><tr><td>20 </td><td>4181 </td><td>rpoB </td><td>4474 </td><td>−1, +1 </td><td>4504 </td><td>M </td><td>4504 </td><td>M, M </td><td>4475 </td><td>+1 </td></tr><tr><td>Genes </td><td colspan="2">19 </td><td colspan="2">18 </td><td colspan="2">18 </td><td colspan="2">18 </td><td colspan="2">21 </td></tr><tr><td>Errors </td><td colspan="2">14(8) </td><td colspan="2">12(3) </td><td colspan="2">9(5) </td><td colspan="2">6(2) </td><td colspan="2">7(0) </td></tr></tbody></table></div><div content-id="DSN021TB3" class="table-modal"><table><thead name="thead" align=""><tr><th># </th><th colspan="2">(a) chi-square<hr /> </th><th colspan="2">(b) delta-distance<hr /> </th><th colspan="2">(c) Euclidean<hr /> </th><th colspan="2">(d) quadratic<hr /> </th><th colspan="2">(e) G + C<hr /> </th></tr><tr><th> </th><th>loc. </th><th>HT </th><th>loc. </th><th>HT </th><th>loc. </th><th>HT </th><th>loc. </th><th>HT </th><th>loc. </th><th>HT </th></tr></thead><tbody><tr><td>1 </td><td>200 </td><td>yaeT </td><td>151 </td><td>−1 </td><td>151 </td><td>−1 </td><td>284 </td><td>M </td><td>583 </td><td>M </td></tr><tr><td>2 </td><td>227 </td><td>0 </td><td>227 </td><td>0 </td><td>274 </td><td>M </td><td>525 </td><td>M </td><td>584 </td><td>M </td></tr><tr><td>3 </td><td>393 </td><td>M </td><td>394 </td><td>−1 </td><td>575 </td><td>M </td><td>526 </td><td>M </td><td>1212 </td><td>−1, −1 </td></tr><tr><td>4 </td><td>394 </td><td>−1 </td><td>526 </td><td>M </td><td>777 </td><td>−1 </td><td>575 </td><td>M </td><td>1636 </td><td>M </td></tr><tr><td>5 </td><td>526 </td><td>M </td><td>777 </td><td>−1 </td><td>978 </td><td>mukB </td><td>777 </td><td>−1 </td><td>2102 </td><td>−1 </td></tr><tr><td>6 </td><td>777 </td><td>−1 </td><td>1287 </td><td>−1 </td><td>1142 </td><td>rne </td><td>1142 </td><td>rne </td><td>2105 </td><td>+1 </td></tr><tr><td>7 </td><td>978 </td><td>mukB </td><td>1427 </td><td>M </td><td>1395 </td><td>M </td><td>1427 </td><td>M </td><td>2468 </td><td>M </td></tr><tr><td>8 </td><td>1142 </td><td>rne </td><td>1465 </td><td>0 </td><td>1427 </td><td>M </td><td>1465 </td><td>0 </td><td>2773 </td><td>M </td></tr><tr><td>9 </td><td>1465 </td><td>0 </td><td>1527 </td><td>0 </td><td>1465 </td><td>0 </td><td>1527 </td><td>0 </td><td>2783 </td><td>0 </td></tr><tr><td>10 </td><td>1527 </td><td>0 </td><td>1707 </td><td>−1 </td><td>1527 </td><td>0 </td><td>1707 </td><td>−1 </td><td>2785 </td><td>0 </td></tr><tr><td>11 </td><td>1707 </td><td>−1 </td><td>2071 </td><td>M </td><td>2101 </td><td>M </td><td>2104 </td><td>+1 </td><td>2989 </td><td>+1 </td></tr><tr><td>12 </td><td>2071 </td><td>M </td><td>2072 </td><td>M </td><td>2104 </td><td>+1 </td><td>2105 </td><td>+1 </td><td>2990 </td><td>−1 </td></tr><tr><td>13 </td><td>2072 </td><td>M </td><td>2104 </td><td>+1 </td><td>2105 </td><td>+1 </td><td>2989 </td><td>+1 </td><td>2994 </td><td>0 </td></tr><tr><td>14 </td><td>2104 </td><td>+1 </td><td>2994 </td><td>0 </td><td>2994 </td><td>0 </td><td>2990 </td><td>−1 </td><td>3267 </td><td>+1, +1 </td></tr><tr><td>15 </td><td>3312 </td><td>−1, infB </td><td>3312 </td><td>−1, infB </td><td>3314 </td><td>−1 </td><td>2992 </td><td>−1 </td><td>3581 </td><td>+1 </td></tr><tr><td>16 </td><td>3450 </td><td>rplWD </td><td>3602 </td><td>ftsY </td><td>3450 </td><td>rplWD </td><td>2994 </td><td>0 </td><td>3797 </td><td>+1,+1 </td></tr><tr><td>17 </td><td>3602 </td><td>ftsY </td><td>3915 </td><td>−1 </td><td>3602 </td><td>ftsY </td><td>3602 </td><td>ftsY </td><td>3798 </td><td>−1 </td></tr><tr><td>18 </td><td>3915 </td><td>−1 </td><td>4058 </td><td>−1 </td><td>4121 </td><td>−1 </td><td>3620 </td><td>M </td><td>3803 </td><td>−1, +1 </td></tr><tr><td>19 </td><td>4058 </td><td>−1 </td><td>4187 </td><td>rpoC </td><td>4503 </td><td>M </td><td>4503 </td><td>M </td><td>4267 </td><td>−1 </td></tr><tr><td>20 </td><td>4181 </td><td>rpoB </td><td>4474 </td><td>−1, +1 </td><td>4504 </td><td>M </td><td>4504 </td><td>M, M </td><td>4475 </td><td>+1 </td></tr><tr><td>Genes </td><td colspan="2">19 </td><td colspan="2">18 </td><td colspan="2">18 </td><td colspan="2">18 </td><td colspan="2">21 </td></tr><tr><td>Errors </td><td colspan="2">14(8) </td><td colspan="2">12(3) </td><td colspan="2">9(5) </td><td colspan="2">6(2) </td><td colspan="2">7(0) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Each segment, indexed by location (loc.) in kb from the published origin, overlaps zero, one or two genes in the protein table. Intergenic segments are classified as ‘0’. HT is indicated by ‘+1’ or by ‘M’ (if the gene is a mobile element). False positives are indicated by gene locus (if essential) or by ‘−1’ (if not). False positives account for total errors (bottom row) and the number of essential genes is given (in parentheses). The analysis is repeated for the 20 most anomalous segments with respect to (e) GC-divergence.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5191466" target="_blank">View Large</a></div></div>                            <h2 scrollto-destination=5191467 id="5191467" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=5191468 id="5191468" class="section-title" data-legacy-id=s3a>3.1. Statistical analysis of compositional divergence in 12 bacterial species</h3>
<p>Statistical analysis of 1 kb segments of the <em>E</em>. <em>coli</em> K12 sequence showed that DRA is more homogeneous than G + C. The same conclusion holds for the 11 other species in Table <a reveal-id="DSN021TB1" data-open="DSN021TB1" class="link link-reveal link-table xref-fig">1</a>, as shown in Table <a reveal-id="DSN021TB2" data-open="DSN021TB2" class="link link-reveal link-table xref-fig">2</a>, in so far as ERs at the 95- and 99.5-percentile points of the <em>χ</em><sup>2</sup>-distributions are higher for GC-divergence than for DRA-divergence. This contrast holds in all 12 species despite the fact that mean GC-divergence is less than mean DRA-divergence in four of them. In every case, the <em>χ</em><sup>2</sup> approximation of Equation (<a href="#DSN021M1" class="sectionLink xref-sec">1</a>) is defensible, as the MAD between the left and right sides of Equation (<a href="#DSN021M2" class="sectionLink xref-sec">2</a>) is &lt;6%. But the variance of these results is wide and our sample is too small to argue that the contrast holds for all bacteria. By extending the analysis to a small, quasi-random sample of species, we have merely verified that the <em>E</em>. <em>coli</em> genome is not exceptional in this respect before exploring it more closely.</p>                            <h3 scrollto-destination=5191470 id="5191470" class="section-title" data-legacy-id=s3b>3.2. Phylogenetic analysis of extreme compositional anomalies in E. coli K12</h3>
<p>The 20 segments having greatest DRA-divergences are characterized in the first pair (a) of data columns in Table <a reveal-id="DSN021TB3" data-open="DSN021TB3" class="link link-reveal link-table xref-fig">3</a>. They overlap 19 genes of which 4 are mobile elements and 14 are false positives. The mobile elements include rhsD at location 256.<sup><a reveal-id="DSN021C32" data-open="DSN021C32" class="link link-ref link-reveal xref-bibr">32</a></sup> (Location is the endpoint in kb from the start of the published sequence.) The one additional true positive, at location 2104, is the β-1,6-galactofuranosyltransferase gene wbbI (locus tag b2034) which leads transcription unit wbbIJK. This gene lacks a homolog in 283 completed genomes of other proteobacteria. The 14 false positives include eight essential genes: yaeT, mukB, rne, infB, rplW, rplD, ftsY and rpoB. The probability of finding one essential gene by chance in a sample of size 1 is 302/4639 ≈ 6.5%; but the binomial probability of finding eight or more essential genes in a sample of 19 is ∼10<sup>−5</sup>. Large DRA-divergence is thus a strong indicator of essentiality. On the other hand, if 17.6% of <em>E</em>. <em>coli</em> genes are foreign,<sup><a reveal-id="DSN021C7" data-open="DSN021C7" class="link link-ref link-reveal xref-bibr">7</a></sup> the probability of finding five or more of these (four mobile elements plus one) by chance is roughly 10%, and large DRA-divergence is a weak indicator of HGT. Before attempting to reconcile this outcome with the conclusion drawn from compositional statistics, it seems important to consider some other mathematical discriminants as measures of DRA anomaly.</p><p>Delta-distance, introduced by Karlin et al. and defined above, has been used to establish a performance baseline as noted in the introduction. The 20 segments having greatest delta-distances from the global signature are characterized in the second pair (b) of data columns of Table <a reveal-id="DSN021TB3" data-open="DSN021TB3" class="link link-reveal link-table xref-fig">3</a>. They overlap 18 genes of which 4 are mobile elements and 12 are false positives. An apparent contradiction arises in the 20th segment, ending 4474 kb from sequence start, which overlaps (by &gt;300 bp each) two genes, yjgK (b4252) and yjgL (b4253), which encode hypothetical proteins. Classified as a conserved protein in EcoCyc, yjgK has homologs in <em>S</em>. <em>enterica</em> (<em>E</em> &lt; 10<sup>−65</sup>) and proteobacteria outside the gamma subdivision (<em>E</em> &lt; 10<sup>−10</sup>); but yjgL is a true positive with no good alignments in other proteobacteria except three species of <em>Shigella</em> which we suppose to be descended from <em>E</em>. <em>coli</em>. The 12 false positives include three essential genes that were previously identified by large DRA-divergence. The probability of finding three or more essential genes in a random sample of size 18 is ∼11%.</p><p>Euclidean distance in 16-dimensional DRA space has been employed in several investigations. When Euclidean distance is computed in the 10-dimensional space of the non-redundant components of the strand-invariant DRA profile, the 20 most anomalous segments are listed in the third pair (c) of data columns in Table <a reveal-id="DSN021TB3" data-open="DSN021TB3" class="link link-reveal link-table xref-fig">3</a>. They overlap 18 genes of which six are mobile elements and nine are false positives. Five false positives are essential genes that were previously identified by large DRA-divergence and the corresponding binomial <em>P</em>-value is ∼0.5%.</p><p>The species-specificity of a minimum Euclidean distance classifier was slightly enhanced by appropriately scaling the components.<sup><a reveal-id="DSN021C16" data-open="DSN021C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Applied to anomaly detection, a two-way classification problem, the effect of such scaling is to replace a spherical acceptance region with an ellipsoid. General principles of pattern recognition theory suggest that even greater specificity can be achieved by a quadratic discriminant classifier that inscribes more flexible boundaries in 10-dimensional DRA space. We constructed the anomaly detector using standard algorithms to form a quadratic discriminant<sup><a reveal-id="DSN021C33" data-open="DSN021C33" class="link link-ref link-reveal xref-bibr">33</a></sup> based on the logarithms of the strand-invariant DRA values (log-odds ratios). The 20 most anomalous segments, listed in the fourth pair (d) of data columns in Table <a reveal-id="DSN021TB3" data-open="DSN021TB3" class="link link-reveal link-table xref-fig">3</a>, overlap 18 genes of which nine are mobile elements and six are false positives. Only two false positives are essential genes and the binomial <em>P</em>-value is 33%. Gene ftsY, which is essential for septation, is found anomalous by this and every discriminant considered previously. The true positives include adjacent, co-oriented genes wbbI and rfc and also yqeK (b2849) which was not discovered previously. Thus, the false positive rate of the most powerful discriminant is 33%, and 33% of false positives are essential. The 50 most anomalous segments identified by the quadratic discriminant contain 5 intergenic regions, 25 mobile elements and 8 essential genes including mukB, rplBWD, rfaK/waaU, ftsN and the two just noted. The binomial <em>P</em>-value for eight essential genes in 45 is &lt;1%.</p><p>Last, the 20 highest ranking segments with respect to GC-divergence are analyzed. These segments, listed in the right-most pair (e) of columns in Table <a reveal-id="DSN021TB3" data-open="DSN021TB3" class="link link-reveal link-table xref-fig">3</a>, overlap 21 genes of which five are mobile elements and six are false positives. Segment 4267 overlaps tyrB (b4054), a false positive and an RNA gene (b4621) which is not counted. Segment 3803 produces a contradiction, overlapping rfaB (false positive) and rfaS (true positive). (This is only the second such contradiction in all cases examined.) Segment 3798 contains false positive rfaZ in the AT-rich rfaQGPSBIJYZ-waaU transcription unit; but segment 3797 overlaps true positives waaU and rfaL that are oppositely oriented. The sole essential gene on this list is waaU (also known as rfaK) which has significant alignments in several proteobacteria outside the gamma subdivision. No essential genes are found among the six false positives. Large values of the GC-divergence thus predict HGT with the same false positive rate as the most powerful (quadratic) discriminant based on DRA-divergence but do not mistake essentiality for foreignness so often.</p>                            <h3 scrollto-destination=5191476 id="5191476" class="section-title" data-legacy-id=s3c>3.3. Compositional divergences in segments overlapping the essential genes</h3>
<p>The 20 largest values of DRA-divergence in 1 kb segments of the <em>E</em>. <em>coli</em> K12 chromosome pointed to a significant number of essential gene candidates. Is DRA-divergence generally elevated in segments that contain essential genes? To answer this question, the mid-points of the coding sequences of essential gene candidates<sup><a reveal-id="DSN021C2" data-open="DSN021C2" class="link link-ref link-reveal xref-bibr">2</a></sup> are calculated, and each mid-point identifies a single segment. Mean divergences in these 302 ‘essential segments’ are shown in Table <a reveal-id="DSN021TB4" data-open="DSN021TB4" class="link link-reveal link-table xref-fig">4</a> and compared with corresponding genome-wide averages for all segments and for non-essential segments. Mean DRA-divergence is elevated, whereas the others are depressed, with reference to the third row where differences (<em>n</em> − <em>e</em>) are noted. The same pattern is followed by ER-95% and ER-99.5%. These ERs where higher for GC-divergence than for DRA-divergence, when all segments were considered, for all 12 species in Table <a reveal-id="DSN021TB2" data-open="DSN021TB2" class="link link-reveal link-table xref-fig">2</a>; but this contrast disappears for the essential segments of <em>E</em>. <em>coli</em>.</p>                        <a id="5191478" scrollto-destination="5191478"></a>
<div content-id="DSN021TB4" class="table-modal"><div class="table-wrap"><div id="DSN021TB4" class="table-wrap-title" data-id="DSN021TB4"><div class="label -label">Table 4</div><div class="caption"><p>Comparing essential segments of the <em>E</em>. <em>coli</em> sequence to all segments and to non-essential segments</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Segments </th><th colspan="3">Mean divergence<hr /> </th><th colspan="3">ER 95%<hr /> </th><th colspan="3">ER 99.5%<hr /> </th></tr><tr><th> </th><th>Total </th><th>DRA </th><th>G + C </th><th>Total </th><th>DRA </th><th>G + C </th><th>Total </th><th>DRA </th><th>G + C </th></tr></thead><tbody><tr><td>All segments </td><td>27.1 </td><td>9.8 </td><td>18.4 </td><td>48.0 </td><td>24.3 </td><td>46.4 </td><td>33.5 </td><td>10.2 </td><td>34.0 </td></tr><tr><td>Non-essential (<em>n</em>) </td><td>27.6 </td><td>9.7 </td><td>19.0 </td><td>48.2 </td><td>23.7 </td><td>47.2 </td><td>34.1 </td><td>9.8 </td><td>35.0 </td></tr><tr><td>Essential (<em>e</em>) </td><td>19.9 </td><td>12.1 </td><td>8.5 </td><td>45.5 </td><td>34.1 </td><td>33.1 </td><td>25.8 </td><td>18.4 </td><td>18.7 </td></tr><tr><td><em>n</em> − e </td><td>8.6 </td><td>−2.4 </td><td>10.5 </td><td>2.7 </td><td>−10.4 </td><td>14.1 </td><td>8.3 </td><td>−8.6 </td><td>16.3 </td></tr><tr><td> </td><td colspan="3">MSR </td><td colspan="3">(MSR − 0.5)/SD </td><td colspan="3"><em>P</em>-value </td></tr><tr><td>Essential </td><td>0.462 </td><td>0.559 </td><td>0.400 </td><td>−2.29 </td><td>+3.55 </td><td>−6.02 </td><td>0.011 </td><td>0.0002 </td><td>&lt;10<sup>−9</sup> </td></tr></tbody></table></div><div content-id="DSN021TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Segments </th><th colspan="3">Mean divergence<hr /> </th><th colspan="3">ER 95%<hr /> </th><th colspan="3">ER 99.5%<hr /> </th></tr><tr><th> </th><th>Total </th><th>DRA </th><th>G + C </th><th>Total </th><th>DRA </th><th>G + C </th><th>Total </th><th>DRA </th><th>G + C </th></tr></thead><tbody><tr><td>All segments </td><td>27.1 </td><td>9.8 </td><td>18.4 </td><td>48.0 </td><td>24.3 </td><td>46.4 </td><td>33.5 </td><td>10.2 </td><td>34.0 </td></tr><tr><td>Non-essential (<em>n</em>) </td><td>27.6 </td><td>9.7 </td><td>19.0 </td><td>48.2 </td><td>23.7 </td><td>47.2 </td><td>34.1 </td><td>9.8 </td><td>35.0 </td></tr><tr><td>Essential (<em>e</em>) </td><td>19.9 </td><td>12.1 </td><td>8.5 </td><td>45.5 </td><td>34.1 </td><td>33.1 </td><td>25.8 </td><td>18.4 </td><td>18.7 </td></tr><tr><td><em>n</em> − e </td><td>8.6 </td><td>−2.4 </td><td>10.5 </td><td>2.7 </td><td>−10.4 </td><td>14.1 </td><td>8.3 </td><td>−8.6 </td><td>16.3 </td></tr><tr><td> </td><td colspan="3">MSR </td><td colspan="3">(MSR − 0.5)/SD </td><td colspan="3"><em>P</em>-value </td></tr><tr><td>Essential </td><td>0.462 </td><td>0.559 </td><td>0.400 </td><td>−2.29 </td><td>+3.55 </td><td>−6.02 </td><td>0.011 </td><td>0.0002 </td><td>&lt;10<sup>−9</sup> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Mean values and ERs of the divergence statistics are computed as in Table <a reveal-id="DSN021TB2" data-open="DSN021TB2" class="link link-reveal link-table xref-fig">2</a>. The MSR of the divergence in essential segments is tested for significant departure from expected value (0.5) in the bottom row.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5191478" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSN021TB4" class="table-wrap-title" data-id="DSN021TB4"><div class="label -label">Table 4</div><div class="caption"><p>Comparing essential segments of the <em>E</em>. <em>coli</em> sequence to all segments and to non-essential segments</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Segments </th><th colspan="3">Mean divergence<hr /> </th><th colspan="3">ER 95%<hr /> </th><th colspan="3">ER 99.5%<hr /> </th></tr><tr><th> </th><th>Total </th><th>DRA </th><th>G + C </th><th>Total </th><th>DRA </th><th>G + C </th><th>Total </th><th>DRA </th><th>G + C </th></tr></thead><tbody><tr><td>All segments </td><td>27.1 </td><td>9.8 </td><td>18.4 </td><td>48.0 </td><td>24.3 </td><td>46.4 </td><td>33.5 </td><td>10.2 </td><td>34.0 </td></tr><tr><td>Non-essential (<em>n</em>) </td><td>27.6 </td><td>9.7 </td><td>19.0 </td><td>48.2 </td><td>23.7 </td><td>47.2 </td><td>34.1 </td><td>9.8 </td><td>35.0 </td></tr><tr><td>Essential (<em>e</em>) </td><td>19.9 </td><td>12.1 </td><td>8.5 </td><td>45.5 </td><td>34.1 </td><td>33.1 </td><td>25.8 </td><td>18.4 </td><td>18.7 </td></tr><tr><td><em>n</em> − e </td><td>8.6 </td><td>−2.4 </td><td>10.5 </td><td>2.7 </td><td>−10.4 </td><td>14.1 </td><td>8.3 </td><td>−8.6 </td><td>16.3 </td></tr><tr><td> </td><td colspan="3">MSR </td><td colspan="3">(MSR − 0.5)/SD </td><td colspan="3"><em>P</em>-value </td></tr><tr><td>Essential </td><td>0.462 </td><td>0.559 </td><td>0.400 </td><td>−2.29 </td><td>+3.55 </td><td>−6.02 </td><td>0.011 </td><td>0.0002 </td><td>&lt;10<sup>−9</sup> </td></tr></tbody></table></div><div content-id="DSN021TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Segments </th><th colspan="3">Mean divergence<hr /> </th><th colspan="3">ER 95%<hr /> </th><th colspan="3">ER 99.5%<hr /> </th></tr><tr><th> </th><th>Total </th><th>DRA </th><th>G + C </th><th>Total </th><th>DRA </th><th>G + C </th><th>Total </th><th>DRA </th><th>G + C </th></tr></thead><tbody><tr><td>All segments </td><td>27.1 </td><td>9.8 </td><td>18.4 </td><td>48.0 </td><td>24.3 </td><td>46.4 </td><td>33.5 </td><td>10.2 </td><td>34.0 </td></tr><tr><td>Non-essential (<em>n</em>) </td><td>27.6 </td><td>9.7 </td><td>19.0 </td><td>48.2 </td><td>23.7 </td><td>47.2 </td><td>34.1 </td><td>9.8 </td><td>35.0 </td></tr><tr><td>Essential (<em>e</em>) </td><td>19.9 </td><td>12.1 </td><td>8.5 </td><td>45.5 </td><td>34.1 </td><td>33.1 </td><td>25.8 </td><td>18.4 </td><td>18.7 </td></tr><tr><td><em>n</em> − e </td><td>8.6 </td><td>−2.4 </td><td>10.5 </td><td>2.7 </td><td>−10.4 </td><td>14.1 </td><td>8.3 </td><td>−8.6 </td><td>16.3 </td></tr><tr><td> </td><td colspan="3">MSR </td><td colspan="3">(MSR − 0.5)/SD </td><td colspan="3"><em>P</em>-value </td></tr><tr><td>Essential </td><td>0.462 </td><td>0.559 </td><td>0.400 </td><td>−2.29 </td><td>+3.55 </td><td>−6.02 </td><td>0.011 </td><td>0.0002 </td><td>&lt;10<sup>−9</sup> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Mean values and ERs of the divergence statistics are computed as in Table <a reveal-id="DSN021TB2" data-open="DSN021TB2" class="link link-reveal link-table xref-fig">2</a>. The MSR of the divergence in essential segments is tested for significant departure from expected value (0.5) in the bottom row.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5191478" target="_blank">View Large</a></div></div><p>The statistical significance of the differences (<em>n</em> − <em>e</em>) in Table <a reveal-id="DSN021TB4" data-open="DSN021TB4" class="link link-reveal link-table xref-fig">4</a> is assessed by a non-parametric test procedure. All segments are ranked according to divergence. Under the null hypothesis, essentiality does not influence divergence, and hence the essential segments are uniformly distributed from rank 1 to 4639. Then the mean rank of essential segments is divided by 4639 to obtain the mean scaled rank (MSR) which is nearly normal with expected value equal to 1/2 and standard deviation (SD) = [(1/12)/302]<sup>1/2</sup> = 0.0166. The last row of Table <a reveal-id="DSN021TB4" data-open="DSN021TB4" class="link link-reveal link-table xref-fig">4</a> shows that MSR &gt; 0.5 for DRA-divergence but &lt;0.5 for the other two components. These differences are expressed in SDs from expected value and the normal <em>P</em>-values for one-sided tests are given. On this basis, DRA-divergence is significantly elevated, but GC-divergence is very significantly depressed, in the essential sample. This conclusion is reinforced by Fig. <a reveal-id="DSN021F3" data-open="DSN021F3" class="link link-reveal link-table xref-fig">3</a>, which portrays the distributions of the divergence components on logarithmic axes. The non-essential sample is described by solid squares that mark the means, medians and labeled percentile points of the empirical distributions (not referred to <em>χ</em><sup>2</sup>), whereas the essential sample is described by solid circles. As each circle lies above and to the left of the corresponding square, the essential sample distribution is evidently shifted lower with respect to GC-divergence, but higher with respect to DRA-divergence.</p>                        <a id="5191480" scrollto-destination="5191480"></a>
<div id="DSN021F3" data-id="DSN021F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn021/2/m_dsn02103.jpeg?Expires=2147483647&Signature=KTQ4OFg6LiuBNixBRkxBHaoApjzEoT~DGXTXEyI9PYIiJxhq0HJ~GA6UPCa8Wt-ibDhZ~xbAc4z-crCcSNlcLmPdUF4Ts5oF1Vch1IxMt5Q5tSzefCt1E~UhSSj45hHmR9dBPHDi~pXNFD-Uta3ww~b0iLagr4CmG3qBSf7XUYexEVqx5InCSwjTr4lj4pj7nhYnOroVboM5uooUvaSvjAR0JWE3NcxLITO3~hYMAhjdeCVVJtzlgW7~4k~9K-D-4hCZTtD7E3s5v6h2RKFuvgE~NhJZzzOVTXY8HQrPUcBh4wmOyzlfzJW1mZicr4l7UwaFpKdadKZlohba4kMXng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparing observed distributions of compositional divergence components in segments of the E. coli K12 genome. Means, medians, and indicated percentile points are plotted for (circles) essential and (squares) non-essential segments. Arrows depict the inferred time course of amelioration as explained in Section 3.4." path-from-xml="dsn02103.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5191480/dsn02103.jpeg" path-from-xml="dsn02103.jpeg" target="_blank">View large</a><a section="5191480" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn021/2/dsn02103.jpeg?Expires=2147483647&Signature=1wKPqo6tKm~jZWWfufJnAP7fNfxO2G3Dn1aTZYh8o0lIi~9LUtZIMMDitKVW8Wv52O~MLeS3ZmgvCiMY175XWzdbLUmaAEWwWOtPLFZh0PrqJgy7fmbYWVC8EZKulnXHsHshsvqvOH66CrK-63nRJKiaQ9~yjU8GyAsp6adkwbiINj8g8GJkFcAGh6AfN9GGFtDdbTJ9BbzN4f2lYqnkDXrRVIbHSPqpywLmAAt0B9wkqaSw9wdoID8Uz-nFIkbWGk-CVOoN1OV-X2ygjC3rDB0~ivErFuAPkX-uzn9bUO6p5yNw5JdUsbrZnlz7AMWLu1EA9zD~3-Xg0ErTpgKyrw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5191480&ar=359884&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02103.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparing observed distributions of compositional divergence components in segments of the <em>E</em>. <em>coli</em> K12 genome. Means, medians, and indicated percentile points are plotted for (circles) essential and (squares) non-essential segments. Arrows depict the inferred time course of amelioration as explained in Section 3.4.</p></div></div><div content-id="DSN021F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn021/2/m_dsn02103.jpeg?Expires=2147483647&Signature=KTQ4OFg6LiuBNixBRkxBHaoApjzEoT~DGXTXEyI9PYIiJxhq0HJ~GA6UPCa8Wt-ibDhZ~xbAc4z-crCcSNlcLmPdUF4Ts5oF1Vch1IxMt5Q5tSzefCt1E~UhSSj45hHmR9dBPHDi~pXNFD-Uta3ww~b0iLagr4CmG3qBSf7XUYexEVqx5InCSwjTr4lj4pj7nhYnOroVboM5uooUvaSvjAR0JWE3NcxLITO3~hYMAhjdeCVVJtzlgW7~4k~9K-D-4hCZTtD7E3s5v6h2RKFuvgE~NhJZzzOVTXY8HQrPUcBh4wmOyzlfzJW1mZicr4l7UwaFpKdadKZlohba4kMXng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparing observed distributions of compositional divergence components in segments of the E. coli K12 genome. Means, medians, and indicated percentile points are plotted for (circles) essential and (squares) non-essential segments. Arrows depict the inferred time course of amelioration as explained in Section 3.4." path-from-xml="dsn02103.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5191480/dsn02103.jpeg" path-from-xml="dsn02103.jpeg" target="_blank">View large</a><a section="5191480" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn021/2/dsn02103.jpeg?Expires=2147483647&Signature=1wKPqo6tKm~jZWWfufJnAP7fNfxO2G3Dn1aTZYh8o0lIi~9LUtZIMMDitKVW8Wv52O~MLeS3ZmgvCiMY175XWzdbLUmaAEWwWOtPLFZh0PrqJgy7fmbYWVC8EZKulnXHsHshsvqvOH66CrK-63nRJKiaQ9~yjU8GyAsp6adkwbiINj8g8GJkFcAGh6AfN9GGFtDdbTJ9BbzN4f2lYqnkDXrRVIbHSPqpywLmAAt0B9wkqaSw9wdoID8Uz-nFIkbWGk-CVOoN1OV-X2ygjC3rDB0~ivErFuAPkX-uzn9bUO6p5yNw5JdUsbrZnlz7AMWLu1EA9zD~3-Xg0ErTpgKyrw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5191480&ar=359884&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02103.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparing observed distributions of compositional divergence components in segments of the <em>E</em>. <em>coli</em> K12 genome. Means, medians, and indicated percentile points are plotted for (circles) essential and (squares) non-essential segments. Arrows depict the inferred time course of amelioration as explained in Section 3.4.</p></div></div>                            <h3 scrollto-destination=5191481 id="5191481" class="section-title" data-legacy-id=s3d>3.4. Amelioration of compositional divergences</h3>
<p>Proceeding from the initial proposition that higher order features ameliorate more rapidly and uniformly than lower order features, we predicted that bacterial genomes are more homogeneous with respect to (second order) DRA than with respect to (first order) G + C when viewed through 1 kb windows. After statistical analysis supported the prediction, we expected to detect recent HGT in segments of the <em>E</em>. <em>coli</em> K12 sequence that are most anomalous with respect to DRA. Instead, large DRA-divergence was a better indicator of essentiality. Can these results be reconciled with the initial proposition?</p><div class="block-child-p">Lawrence and Ochman<sup><a reveal-id="DSN021C5" data-open="DSN021C5" class="link link-ref link-reveal xref-bibr">5</a></sup> modeled the amelioration of G + C proportions at the three codon positions in accordance with the theory of mutation-selection equilibria.<sup><a reveal-id="DSN021C34" data-open="DSN021C34" class="link link-ref link-reveal xref-bibr">34</a></sup> Their assumption of constant substitution rates was reflected in the exponential time course of amelioration toward the equilibrium proportion. Amelioration rates are typically lower at the first two positions where mutations are more likely to degrade essential functionality. When the G + C proportions are averaged over three positions in a coding sequence, the average proportion, denoted <em>P</em>, approaches the gross genomic G + C proportion, <em>Q</em>. Then the logarithm of the squared difference <em>D</em> = (<em>P</em> − <em>Q</em>)<sup>2</sup> declines in three quasi-linear stages. After sufficiently long time, its rate of decrease falls to the lowest position-specific amelioration rate. GC-divergence, as derived above, evolves this way because <a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn021/2/dsn021014.gif?Expires=2147483647&Signature=K0gHhPKpKTwOioxfzdmevcoCUUeVYUStdpE-NApNuWZiAsEePwtLJI4aqdb6AGIZ-FTd8jZt9Hzh6q9PsYgA-KGHvWgaXQQ13y7v3JCoWho7BAXXcOFkBBGPYmxPyZD4--KByhqzyzIcYYiOu5-eFbwLCozIuLdih5MbXri47~agR3Oh~3VDz1BFuE5QlITbPwQDaHsJsGBqjbf3JIjEVgkLIYeAWHADcUVfQQtBgWMEG8FxV0axEkrc9m1j2IAQ7yN6z4~WnmhTvam~qDC2jeZGzPuqb4-daKIkuLUbATX5FWBSmAjh0d1ytM1-TzGBhDruiC8rlwnh3tJ2zxKDHA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsn021014.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn021/2/m_dsn021014.gif?Expires=2147483647&Signature=WzGoFMiCNlZjzHg1wd4poC49nNVJjaqDuO4bEOWmCJX4leozTsBN2tDLOyCytDkISUTkWo4i1wAJ6MAsupMs7BzraxZTXZIaLz-8qwtawf5YX~HhF0a~T6yR7xGfYou6HhOejrXYwFNii4Wm8Re-vbSc5iTlhTRVojXJRdHuPo4ox1dJbTPR9~FzYL4IoE5qD3vU1FtJbKsWqnOwZv0vk8y5Lgi7Bjrol-3rvkdVx-mOd5iYnyX-PpsR6-esafT3y05N8kKuDA7rp9iQEKH3RlpqdathgkTvvD-kuNKfd1WzSV~fdtJPyyVvAAxdRvy5Yk0~aa5BihgmBre9krmCWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsn021014.gif" /></a></div></div></div> Relatively ancient coding sequences will have ameliorated to the extent that further reduction of GC-divergence is retarded by selective forces acting on the least neutral codon position.</div><div class="block-child-p">DRA ratios approach species-specific constants in a manner that is formally analogous to the law of mass action by which the reversible reaction <em>X</em> + <em>Y</em> ↔ <em>XY</em> gives rise to chemical equilibrium with <a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn021/2/dsn021015.gif?Expires=2147483647&Signature=ra1WpyJ8~RNU5ORp91x~IoUJ7UtyfHYp7VxR5PgkrpdlYC8I6-o5TsooP6xTmZYNDRPZ57HcZNkOTe1Ws5AKlI3bVK2vT3Gljt5v-pExul3DI84ac7yy081MNXJlGaWG7TZxl6MhompFRqb1d7sOPU0qD418e~aFIsGnzjhri459alWf7RPze8Z6FwiyDvbJFShoZIjMAE3i4g~JtXBV1ErCrebqu58JMytjo9DeXCV2d2VgF53ZT3K0FIPqRg8Ht4Z88UhjeGpEEYR21dGEjT5dCGXVxjLuhGdtYHdBNkaPQs88PIDaFP4aWrOqsofnTHIiOM~iBUVGfUbVHL6YUA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsn021015.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn021/2/m_dsn021015.gif?Expires=2147483647&Signature=5KevdELmU8CTLoI6exruRtk3VTXWp9eANbJWvqTPTEJRXnPt2Z~su4cwk3nYFDt0pjGzWtJePQOuKahL8VRNLvNkMWihwrFbHdCioPX3i0YFRgrSB0pm0rebii2U5TTMSS1cO-6emqWdos5yW5hqjm5oViF~7PfGNEbpdClYwiXzAxNorOwCku-LlJ8V66FyIvwIE8hboCbdC7LLQ3Ck-~FkWA69fH5IQTma8JkY0z7MVlCX-hSX-BuBnade7bV-Fr4lsxQUeSnolscGVEwDLlC4ga-qGVnqmNOuKMN7-sHZT6cHbeLDw1k--yV4xhrYB-2gHrdT-vxVe7-QXRoFIA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsn021015.gif" /></a></div></div></div> The [bracketed] concentrations are analogous to the normalized frequencies and each whole genome DRA ratio plays the role of a formation constant. We suppose this analogy would be illuminated by a mathematical extension of the theory of mutation-selection equilibrium that has not been attempted. DRA-divergence is a weighted sum of squared differences between current values of the ratios and their ‘constant’ limits. If each term in the sum follows an exponential time course then DRA-divergence will obey the general rule stated above, ultimately declining at the rate of its least selection-neutral term; but now the minimum is taken from 10 terms instead of three. In old, essential coding sequences, this minimum may be extremely small, and DRA-divergence will decline more slowly than GC-divergence.</div><p>This argument is sketched on Fig. <a reveal-id="DSN021F3" data-open="DSN021F3" class="link link-reveal link-table xref-fig">3</a> where arrows extend from point <em>T</em><sub>0</sub>, representing the divergence coordinates at some time in the past, to points <em>E</em> (essential) and <em>N</em> (non-essential) with the same GC-divergence. In extreme cases, at least one DRA ratio resists mutational pressure so strongly that divergence drops to a floor level and goes no lower. This possibility is represented on Fig. <a reveal-id="DSN021F3" data-open="DSN021F3" class="link link-reveal link-table xref-fig">3</a> by the arrow branching to point <em>E</em> from <em>T</em><sub>1</sub> that is attained after the less resistant ratios reach equilibrium. Extending this branch much farther will produce a coding sequence that closely matches gross genomic G + C but still exceeds the median of DRA-divergence. Gene ftsY fits this description in so far as its segment (3602) has small GC-divergence (<em>X</em>*<sub>1</sub>=0.42) and its large DRA-divergence (<em>X</em>*<sub>2</sub>=61.8) is half accounted for by three terms (for dinucleotides AT, TA and AG + CT).</p><p>Thus, the simplest explanation of our findings is that G + C typically ameliorates more rapidly than DRA, especially in older and essential parts of the genome, contrary to initial proposition. Finding DRA more homogeneous than G + C appears to contradict this explanation only if it is assumed DRA and G + C are equally divergent between donors and recipients of HT. In this re-interpretation of the data, we argue that the smaller intra-genomic range of DRA merely reflects its smaller inter-genomic range. For example, gross G + C fractions in <em>E</em>. <em>coli</em> and <em>S</em>. <em>enterica</em> differ by ∼2%, and segments of <em>E</em>. <em>coli</em> exhibit 19.9 average GC-divergence from the G + C fraction of <em>S</em>. <em>enterica</em>; but the average DRA-divergence relative to the genomic signature of <em>S</em>. <em>enterica</em> is only 12.4. These genomes appear to have diverged more rapidly in G + C than DRA.</p>                            <h3 scrollto-destination=5191487 id="5191487" class="section-title" data-legacy-id=s3e>3.5. Predicting essential genes based on dinucleotide composition</h3>
<p>DRA-divergence was introduced to assess the statistical significance of local deviations from the genomic signature. Other discriminants found fewer essential genes among the 20 most anomalous segments, but none found as few as divergence with respect to G + C, which is not a reliable indicator of horizontal transmission.<sup><a reveal-id="DSN021C35" data-open="DSN021C35" class="link link-ref link-reveal xref-bibr">35</a></sup> It remains to be explained how HGT detection methods based on higher order signatures can overcome the failure mode predicted by the implication of the neutral theory. The difficulty is implicitly acknowledged when genes belonging to certain functional classes, such as ribosomal proteins, are ‘manually removed’<sup><a reveal-id="DSN021C6" data-open="DSN021C6" class="link link-ref link-reveal xref-bibr">6</a></sup> from lists of HGT candidates produced by higher order methods.<sup><a reveal-id="DSN021C3" data-open="DSN021C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Our negative result highlights the challenge of reconciling such methods, which have advanced without explicitly addressing how essential genes impact performance, with principles of molecular evolution. Although we lack an <em>a priori</em> basis for extrapolating our result to higher order, it is evident that the performance level of a method based on DRA is an easy benchmark to exceed. Approaching the problem from the opposite perspective, it could be useful to screen out essential genes before proceeding with the search for HGT. Our analysis may suggest that DRA-divergence could serve this purpose. When all 1 kb segments of the <em>E</em>. <em>coli</em> K12 sequence are ranked by DRA-divergence, 42 essential segments are among the 302 highest ranks. The expected number is only (302)<sup>2</sup>/4639 ≈ 20 and the binomial <em>P</em>-value is ∼10<sup>−6</sup>. But the success rate of 42/302 ≈ 14% seems disappointing even though it is highly significant.</p><p>Various refinements could raise the success rate. The divergence quotient (DRA divided by GC) could be a stronger indicator of essentiality. A sliding window should improve the compositional contrast between essential and non-essential segments. (Our analysis used fixed length, non-overlapping windows to simplify statistical comparisons.) Essential coding sequences could become more detectable based on the 9 degrees of freedom divergence statistic (denoted by <em>X</em><sub>2</sub> in Section 2) computed from the 16 strand-dependent odds ratios.</p><p>These suggestions aside, we doubt that any single composition-based method can reliably predict essential genes. DRA ratios are determined by the amino acid sequence and SCU. Our methods and results do not illuminate how much of the central tendency of DRA is explained by species-specific codon usage patterns nor how much of the variance is driven by differing expression levels that are independent of essentiality. Predicting essential genes may be as difficult as detecting HTs, but progress can be measured more directly, as new experimental methods identify essential genes without revealing their evolutionary histories.</p>                            <h2 scrollto-destination=5191491 id="5191491" class="section-title" data-legacy-id=s4>Funding</h2>
<p>This research was supported by the MKE (Ministry of Knowledge Economy), Korea, under the ITFSIP supervised by the IITA.</p>                            <h2 scrollto-destination=5191493 id="5191493" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSN021C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Muto</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Osawa</div> <div class="given-names">S</div></div>. </div><div class="article-title">The guanine and cytosine content of genomic DNA and bacterial evolution</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1987</div>, vol. <div class="volume">84</div> (pg. <div class="fpage">166</div>-<div class="lpage">169</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20guanine%20and%20cytosine%20content%20of%20genomic%20DNA%20and%20bacterial%20evolution&amp;author=A.%20Muto&amp;author=S%20Osawa&amp;publication_year=1987&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=84&amp;pages=166-169" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.84.1.166" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.84.1.166" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.84.1.166"> </span></div> </div></div></div></div></div><div content-id="DSN021C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Baba</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Takeshi</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hasegawa</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Construction of <em>Escherichia coli</em> K-12 in-frame single-gene knockout mutants: the Keio collection</div>, <div class="source">Molec. Systems Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">2</div>  <div class="comment">2006.0008</div></div></div></div></div><div content-id="DSN021C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tsirigos</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Rigoutsos</div> <div class="given-names">I</div></div>. </div><div class="article-title">A new computational method for detection of horizontal gene transfer events</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">922</div>-<div class="lpage">923</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20computational%20method%20for%20detection%20of%20horizontal%20gene%20transfer%20events&amp;author=A.%20Tsirigos&amp;author=I%20Rigoutsos&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=922-923" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki187" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki187" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki187"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15716310" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN021C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Homma</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Fukuchi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nakamura</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Gojobori</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Nishikawa</div> <div class="given-names">K</div></div>. </div><div class="article-title">Gene cluster analysis method identifies horizontally transferred genes with high reliability and indicates that they provide the main mechanism of operon gain in 8 species of gamma-Proteobacteria</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2007</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">805</div>-<div class="lpage">813</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20cluster%20analysis%20method%20identifies%20horizontally%20transferred%20genes%20with%20high%20reliability%20and%20indicates%20that%20they%20provide%20the%20main%20mechanism%20of%20operon%20gain%20in%208%20species%20of%20gamma-Proteobacteria&amp;author=K.%20Homma&amp;author=S.%20Fukuchi&amp;author=Y.%20Nakamura&amp;author=T.%20Gojobori&amp;author=K%20Nishikawa&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=24&amp;pages=805-813" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msl206" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsl206" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsl206"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17185745" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN021C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lawrence</div> <div class="given-names">J. G.</div></div>,  <div class="name"><div class="surname">Ochman</div> <div class="given-names">H</div></div>. </div><div class="article-title">Amelioration of bacterial genomes: rates of change and exchange</div>, <div class="source">J. Molec. Evol.</div> , <div class="year">1997</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">383</div>-<div class="lpage">397</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Amelioration%20of%20bacterial%20genomes%3A%20rates%20of%20change%20and%20exchange&amp;author=J.%20G.%20Lawrence&amp;author=H%20Ochman&amp;publication_year=1997&amp;journal=J.%20Molec.%20Evol.&amp;volume=44&amp;pages=383-397" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00006158" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00006158" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00006158"> </span></div> </div></div></div></div></div><div content-id="DSN021C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Podell</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Gaasterland</div> <div class="given-names">T</div></div>. </div><div class="article-title">DarkHorse: a method for genome-wide prediction of horizontal gene transfer</div>, <div class="source">Genome Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">R16</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DarkHorse%3A%20a%20method%20for%20genome-wide%20prediction%20of%20horizontal%20gene%20transfer&amp;author=S.%20Podell&amp;author=T%20Gaasterland&amp;publication_year=2007&amp;journal=Genome%20Biol.&amp;volume=8&amp;pages=R16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2007-8-2-r16" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2007-8-2-r16" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2007-8-2-r16"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17274820" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN021C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lawrence</div> <div class="given-names">J. G.</div></div>,  <div class="name"><div class="surname">Ochman</div> <div class="given-names">H</div></div>. </div><div class="article-title">Molecular archaeology of the <em>Escherichia coli</em> genome</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1997</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">9413</div>-<div class="lpage">9417</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20archaeology%20of%20the%20Escherichia%20coli%20genome&amp;author=J.%20G.%20Lawrence&amp;author=H%20Ochman&amp;publication_year=1997&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=95&amp;pages=9413-9417" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.16.9413" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.16.9413" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.16.9413"> </span></div> </div></div></div></div></div><div content-id="DSN021C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kunisawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kanaya</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kutter</div> <div class="given-names">E</div></div>. </div><div class="article-title">Comparison of synonymous codon distribution patterns of bacteriophage and host genomes</div>, <div class="source">DNA Res.</div> , <div class="year">1998</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">319</div>-<div class="lpage">326</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20synonymous%20codon%20distribution%20patterns%20of%20bacteriophage%20and%20host%20genomes&amp;author=T.%20Kunisawa&amp;author=S.%20Kanaya&amp;author=E%20Kutter&amp;publication_year=1998&amp;journal=DNA%20Res.&amp;volume=5&amp;pages=319-326" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/5.6.319" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F5.6.319" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F5.6.319"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10048480" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN021C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Azad</div> <div class="given-names">R. K.</div></div>,  <div class="name"><div class="surname">Lawrence</div> <div class="given-names">J. G</div></div>. </div><div class="article-title">Detecting laterally transferred genes: use of entropic clustering methods and genome position</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">4629</div>-<div class="lpage">4639</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detecting%20laterally%20transferred%20genes%3A%20use%20of%20entropic%20clustering%20methods%20and%20genome%20position&amp;author=R.%20K.%20Azad&amp;author=J.%20G%20Lawrence&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=4629-4639" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm204" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm204" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm204"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17591616" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN021C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ikemura</div> <div class="given-names">T</div></div>. </div><div class="article-title">Codon usage and tRNA content in unicellular and multicellular organisms</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1985</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">13</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20and%20tRNA%20content%20in%20unicellular%20and%20multicellular%20organisms&amp;author=T%20Ikemura&amp;publication_year=1985&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=2&amp;pages=13-34" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3916708" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN021C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Das</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Paul</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Chatterjee</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Dutta</div> <div class="given-names">C</div></div>. </div><div class="article-title">Codon and amino acid usage in two major human pathogens of genus <em>Bartonella</em>—optimization between replicational-transcriptional selection, translational control and cost minimization</div>, <div class="source">DNA Res.</div> , <div class="year">2005</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">91</div>-<div class="lpage">102</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20and%20amino%20acid%20usage%20in%20two%20major%20human%20pathogens%20of%20genus%20Bartonella%E2%80%94optimization%20between%20replicational-transcriptional%20selection%2C%20translational%20control%20and%20cost%20minimization&amp;author=S.%20Das&amp;author=S.%20Paul&amp;author=S.%20Chatterjee&amp;author=C%20Dutta&amp;publication_year=2005&amp;journal=DNA%20Res.&amp;volume=12&amp;pages=91-102" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/12.2.91" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F12.2.91" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F12.2.91"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16303741" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN021C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pride</div> <div class="given-names">D. T.</div></div>,  <div class="name"><div class="surname">Meinersmann</div> <div class="given-names">R. J.</div></div>,  <div class="name"><div class="surname">Wassenaar</div> <div class="given-names">T. M</div></div>. </div><div class="article-title">Evolutionary implications of microbial genome tetranucleotide frequency bias</div>, <div class="source">Genome Res.</div> , <div class="year">2003</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">145</div>-<div class="lpage">158</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20implications%20of%20microbial%20genome%20tetranucleotide%20frequency%20bias&amp;author=D.%20T.%20Pride&amp;author=R.%20J.%20Meinersmann&amp;author=T.%20M%20Wassenaar&amp;publication_year=2003&amp;journal=Genome%20Res.&amp;volume=13&amp;pages=145-158" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.335003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.335003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.335003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12566393" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN021C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sandberg</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Bränden</div> <div class="given-names">C.-I.</div></div>,  <div class="name"><div class="surname">Ernberg</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Cöster</div> <div class="given-names">J</div></div>. </div><div class="article-title">Quantifying the species-specificity in genomic signatures, synonymous codon choice, amino acid usage and G + C content</div>, <div class="source">Gene</div> , <div class="year">2003</div>, vol. <div class="volume">311</div> (pg. <div class="fpage">35</div>-<div class="lpage">42</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantifying%20the%20species-specificity%20in%20genomic%20signatures%2C%20synonymous%20codon%20choice%2C%20amino%20acid%20usage%20and%20G%20%2B%20C%20content&amp;author=S.%20Sandberg&amp;author=C.-I.%20Br%C3%A4nden&amp;author=I.%20Ernberg&amp;author=J%20C%C3%B6ster&amp;publication_year=2003&amp;journal=Gene&amp;volume=311&amp;pages=35-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(03)00581-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(03)00581-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(03)00581-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12853136" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN021C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dufraigne</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Fertil</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Lespinats</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Giron</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Deschavanne</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Detection and classification of horizontal transfers in prokaryotes using genomic signature</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> pg. <div class="fpage">e6</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20and%20classification%20of%20horizontal%20transfers%20in%20prokaryotes%20using%20genomic%20signature&amp;author=C.%20Dufraigne&amp;author=B.%20Fertil&amp;author=S.%20Lespinats&amp;author=A.%20Giron&amp;author=P.%20Deschavanne&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=e6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gni004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgni004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgni004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15653627" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN021C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakashima</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Nishikawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ooi</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Differences in dinucleotide frequencies of human, yeast, and <em>Escherichia coli</em> genes</div>, <div class="source">DNA Res.</div> , <div class="year">1997</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">185</div>-<div class="lpage">192</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differences%20in%20dinucleotide%20frequencies%20of%20human%2C%20yeast%2C%20and%20Escherichia%20coli%20genes&amp;author=H.%20Nakashima&amp;author=K.%20Nishikawa&amp;author=T.%20Ooi&amp;publication_year=1997&amp;journal=DNA%20Res.&amp;volume=4&amp;pages=185-192" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/4.3.185" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F4.3.185" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F4.3.185"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9330906" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN021C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakashima</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ota</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Nishikawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ooi</div> <div class="given-names">T</div></div>. </div><div class="article-title">Genes from nine genomes are separated into their organisms in the dinucleotide composition space</div>, <div class="source">DNA Res.</div> , <div class="year">1998</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">251</div>-<div class="lpage">259</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genes%20from%20nine%20genomes%20are%20separated%20into%20their%20organisms%20in%20the%20dinucleotide%20composition%20space&amp;author=H.%20Nakashima&amp;author=M.%20Ota&amp;author=K.%20Nishikawa&amp;author=T%20Ooi&amp;publication_year=1998&amp;journal=DNA%20Res.&amp;volume=5&amp;pages=251-259" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/5.5.251" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F5.5.251" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F5.5.251"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9872449" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN021C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Karlin</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Burge</div> <div class="given-names">C</div></div>. </div><div class="article-title">Dinucleotide relative abundance extremes: a genomic signature</div>, <div class="source">Trends Genet.</div> , <div class="year">1995</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">283</div>-<div class="lpage">290</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dinucleotide%20relative%20abundance%20extremes%3A%20a%20genomic%20signature&amp;author=S.%20Karlin&amp;author=C%20Burge&amp;publication_year=1995&amp;journal=Trends%20Genet.&amp;volume=11&amp;pages=283-290" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(00)89076-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(00)89076-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(00)89076-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7482779" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN021C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Karlin</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Mrazek</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Campbell</div> <div class="given-names">A. M</div></div>. </div><div class="article-title">Compositional biases of bacterial genomes and evolutionary implications</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1997</div>, vol. <div class="volume">179</div> (pg. <div class="fpage">3899</div>-<div class="lpage">3913</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Compositional%20biases%20of%20bacterial%20genomes%20and%20evolutionary%20implications&amp;author=S.%20Karlin&amp;author=J.%20Mrazek&amp;author=A.%20M%20Campbell&amp;publication_year=1997&amp;journal=J.%20Bacteriol.&amp;volume=179&amp;pages=3899-3913" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9190805" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN021C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">van Passel</div> <div class="given-names">M. W. J.</div></div>,  <div class="name"><div class="surname">Kuramae</div> <div class="given-names">E. E.</div></div>,  <div class="name"><div class="surname">Luyf</div> <div class="given-names">A. C. M.</div></div>, et al. </div><div class="article-title">The reach of the genome signature in prokaryotes</div>, <div class="source">BMC Evol. Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">6</div> pg. <div class="fpage">84</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20reach%20of%20the%20genome%20signature%20in%20prokaryotes&amp;author=M.%20W.%20J.%20van%20Passel&amp;author=E.%20E.%20Kuramae&amp;author=A.%20C.%20M.%20Luyf&amp;publication_year=2006&amp;journal=BMC%20Evol.%20Biol.&amp;volume=6&amp;pages=84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-6-84" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-6-84" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-6-84"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17040564" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN021C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sandberg</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Winberg</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Branden</div> <div class="given-names">C.-I.</div></div>, et al. </div><div class="article-title">Capturing whole-genome characteristics in short sequences using a naive Bayesian classifier</div>, <div class="source">Genome Res.</div> , <div class="year">2001</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">1404</div>-<div class="lpage">1409</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Capturing%20whole-genome%20characteristics%20in%20short%20sequences%20using%20a%20naive%20Bayesian%20classifier&amp;author=R.%20Sandberg&amp;author=G.%20Winberg&amp;author=C.-I.%20Branden&amp;publication_year=2001&amp;journal=Genome%20Res.&amp;volume=11&amp;pages=1404-1409" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.186401" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.186401" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.186401"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11483581" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN021C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Weir</div> <div class="given-names">B. S</div></div>. </div>, <div class="source">Genetic Data Analysis.</div> , <div class="year">1990</div><div class="publisher-loc">Sunderland, MA</div><div class="publisher-name">Sinauer Associates</div></div></div></div></div><div content-id="DSN021C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Agresti</div> <div class="given-names">A</div></div>. </div>, <div class="source">Categorical Data Analysis.</div> , <div class="year">1990</div><div class="publisher-loc">New York</div><div class="publisher-name">Wiley</div></div></div></div></div><div content-id="DSN021C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jernigan</div> <div class="given-names">R. W.</div></div>,  <div class="name"><div class="surname">Baran</div> <div class="given-names">R. H</div></div>. </div><div class="article-title">Pervasive properties of the genomic signature</div>, <div class="source">BMC Genom.</div> , <div class="year">2002</div>, vol. <div class="volume">3</div> pg. <div class="fpage">23</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pervasive%20properties%20of%20the%20genomic%20signature&amp;author=R.%20W.%20Jernigan&amp;author=R.%20H%20Baran&amp;publication_year=2002&amp;journal=BMC%20Genom.&amp;volume=3&amp;pages=23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-3-23" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-3-23" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-3-23"> </span></div> </div></div></div></div></div><div content-id="DSN021C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Russell</div> <div class="given-names">G. J.</div></div>,  <div class="name"><div class="surname">Subak-Sharpe</div> <div class="given-names">J. H</div></div>. </div><div class="article-title">Similarity of the general designs of protochordates and invertebrates</div>, <div class="source">Nature</div> , <div class="year">1977</div>, vol. <div class="volume">266</div> (pg. <div class="fpage">533</div>-<div class="lpage">536</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Similarity%20of%20the%20general%20designs%20of%20protochordates%20and%20invertebrates&amp;author=G.%20J.%20Russell&amp;author=J.%20H%20Subak-Sharpe&amp;publication_year=1977&amp;journal=Nature&amp;volume=266&amp;pages=533-536" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/266533a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F266533a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F266533a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/558523" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN021C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Baran</div> <div class="given-names">R. H.</div></div>,  <div class="name"><div class="surname">Jernigan</div> <div class="given-names">R. W</div></div>. </div><div class="article-title">Testing lumpability in Markov chains</div>, <div class="source">Stat. Prob. Lit.</div> , <div class="year">2002</div>, vol. <div class="volume">64</div> (pg. <div class="fpage">17</div>-<div class="lpage">23</div>)</div></div></div></div><div content-id="DSN021C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fickett</div> <div class="given-names">J. W.</div></div>,  <div class="name"><div class="surname">Torney</div> <div class="given-names">D. C.</div></div>,  <div class="name"><div class="surname">Wolf</div> <div class="given-names">D. R</div></div>. </div><div class="article-title">Base compositional structure of genomes</div>, <div class="source">Genomics</div> , <div class="year">1992</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">1056</div>-<div class="lpage">1064</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Base%20compositional%20structure%20of%20genomes&amp;author=J.%20W.%20Fickett&amp;author=D.%20C.%20Torney&amp;author=D.%20R%20Wolf&amp;publication_year=1992&amp;journal=Genomics&amp;volume=13&amp;pages=1056-1064" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0888-7543(92)90019-O" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0888-7543(92)90019-O" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0888-7543(92)90019-O"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1505943" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN021C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Are isochore sequences homogeneous?</div>, <div class="source">Comput. Biol. Chem.</div> , <div class="year">2003</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">5</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Are%20isochore%20sequences%20homogeneous%3F&amp;author=W.%20Li&amp;publication_year=2003&amp;journal=Comput.%20Biol.%20Chem.&amp;volume=27&amp;pages=5-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1476-9271(02)00090-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1476-9271(02)00090-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1476-9271(02)00090-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12798034" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN021C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mrázek</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Karlin</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Strand compositional asymmetry in bacterial and large viral genomes</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1998</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">3720</div>-<div class="lpage">3725</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Strand%20compositional%20asymmetry%20in%20bacterial%20and%20large%20viral%20genomes&amp;author=J.%20Mr%C3%A1zek&amp;author=S.%20Karlin&amp;publication_year=1998&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=95&amp;pages=3720-3725" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.7.3720" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.7.3720" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.7.3720"> </span></div> </div></div></div></div></div><div content-id="DSN021C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Keseler</div> <div class="given-names">I. M.</div></div>,  <div class="name"><div class="surname">Collado-Vides</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Gama-Castro</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">EcoCyc: a comprehensive database resource for <em>Escherichia coli</em></div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">D334</div>-<div class="lpage">D337</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EcoCyc%3A%20a%20comprehensive%20database%20resource%20for%20Escherichia%20coli&amp;author=I.%20M.%20Keseler&amp;author=J.%20Collado-Vides&amp;author=J.%20Gama-Castro&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=D334-D337" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15608210" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN021C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S. F.</div></div>,  <div class="name"><div class="surname">Madden</div> <div class="given-names">T. L.</div></div>,  <div class="name"><div class="surname">Schäffer</div> <div class="given-names">A. A.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Miller</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Lipman</div> <div class="given-names">D. J</div></div>. </div><div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3389</div>-<div class="lpage">3402</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.%20F.%20Altschul&amp;author=T.%20L.%20Madden&amp;author=A.%20A.%20Sch%C3%A4ffer&amp;author=J.%20Zhang&amp;author=Z.%20Zhang&amp;author=W.%20Miller&amp;author=D.%20J%20Lipman&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-3402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN021C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Karlin</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Altschul</div> <div class="given-names">S. F.</div></div>. </div><div class="article-title">Methods for assessing the statistical significance of molecular sequence features by using general scoring schemes</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1990</div>, vol. <div class="volume">87</div> (pg. <div class="fpage">2264</div>-<div class="lpage">2268</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Methods%20for%20assessing%20the%20statistical%20significance%20of%20molecular%20sequence%20features%20by%20using%20general%20scoring%20schemes&amp;author=S.%20Karlin&amp;author=S.%20F.%20Altschul&amp;publication_year=1990&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=87&amp;pages=2264-2268" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.87.6.2264" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.87.6.2264" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.87.6.2264"> </span></div> </div></div></div></div></div><div content-id="DSN021C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhao</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sandt</div> <div class="given-names">C. H.</div></div>,  <div class="name"><div class="surname">Feulner</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">Rhs elements of <em>Escherichia coli</em> K12</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1993</div>, vol. <div class="volume">175</div> (pg. <div class="fpage">2799</div>-<div class="lpage">2808</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rhs%20elements%20of%20Escherichia%20coli%20K12&amp;author=S.%20Zhao&amp;author=C.%20H.%20Sandt&amp;author=G.%20Feulner&amp;publication_year=1993&amp;journal=J.%20Bacteriol.&amp;volume=175&amp;pages=2799-2808" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8387990" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN021C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schürmann</div> <div class="given-names">J.</div></div>. </div>, <div class="source">Pattern Classification: A Unified View of Statistical and Neural Approaches.</div> , <div class="year">1996</div><div class="publisher-loc">New York</div><div class="publisher-name">Wiley</div></div></div></div></div><div content-id="DSN021C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sueoka</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Directional mutation pressure and neutral molecular evolution</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1988</div>, vol. <div class="volume">85</div> (pg. <div class="fpage">2653</div>-<div class="lpage">2658</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Directional%20mutation%20pressure%20and%20neutral%20molecular%20evolution&amp;author=N.%20Sueoka&amp;publication_year=1988&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=85&amp;pages=2653-2658" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.85.8.2653" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.85.8.2653" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.85.8.2653"> </span></div> </div></div></div></div></div><div content-id="DSN021C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN021C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Koski</div> <div class="given-names">L. B.</div></div>,  <div class="name"><div class="surname">Morton</div> <div class="given-names">R. A.</div></div>,  <div class="name"><div class="surname">Golding</div> <div class="given-names">B. G</div></div>. </div><div class="article-title">Codon bias and base composition are poor indicators of horizontally transferred genes</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2001</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">404</div>-<div class="lpage">412</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20bias%20and%20base%20composition%20are%20poor%20indicators%20of%20horizontally%20transferred%20genes&amp;author=L.%20B.%20Koski&amp;author=R.%20A.%20Morton&amp;author=B.%20G%20Golding&amp;publication_year=2001&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=18&amp;pages=404-412" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003816" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a003816" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a003816"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11230541" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2008. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=359884&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">63</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000260843400001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">11</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsn021"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="359884"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000260843400001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f15%2f5%2f267%2f359884">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000260843400001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (11)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F15%2F5%2F267%2F359884" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/359884'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Detecting Horizontally Transferred and Essential Genes Based on Dinucleotide Relative Abundance" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsn021");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f15%2f5%2f267%2f359884">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

