

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/18/5/305/448180","@type":"ScholarlyArticle","name":"Genic Microsatellite Markers in  Brassica rapa : Development, Characterization, Mapping, and Their Utility in Other Cultivated and Wild  Brassica  Relatives","datePublished":"2011-07-17","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/18/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2011-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsr017","keywords":["<italic>Brassica rapa</italic>","expressed sequence-derived SSRs","integrated map","polymorphism information content","transferability"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/18/5/305/448180","author":[{"name":"Ramchiary, Nirala","affiliation":"1  Molecular Genetics and Genomics Lab, Department of Horticulture ,  Chungnam National University ,  Gung-Dong, Yuseong-Gu, Daejeon 305-764 ,  Republic of Korea","@type":"Person"},{"name":"Nguyen, Van Dan","affiliation":"1  Molecular Genetics and Genomics Lab, Department of Horticulture ,  Chungnam National University ,  Gung-Dong, Yuseong-Gu, Daejeon 305-764 ,  Republic of Korea","@type":"Person"},{"name":"Li, Xiaonan","affiliation":"1  Molecular Genetics and Genomics Lab, Department of Horticulture ,  Chungnam National University ,  Gung-Dong, Yuseong-Gu, Daejeon 305-764 ,  Republic of Korea","@type":"Person"},{"name":"Hong, Chang Pyo","affiliation":"2  Department of Life Science, and BK21 Program of Bio-Molecular Function ,  Pohang University of Science and Technology ,  Pohang 790-784 ,  Republic of Korea","@type":"Person"},{"name":"Dhandapani, Vignesh","affiliation":"1  Molecular Genetics and Genomics Lab, Department of Horticulture ,  Chungnam National University ,  Gung-Dong, Yuseong-Gu, Daejeon 305-764 ,  Republic of Korea","@type":"Person"},{"name":"Choi, Su Ryun","affiliation":"1  Molecular Genetics and Genomics Lab, Department of Horticulture ,  Chungnam National University ,  Gung-Dong, Yuseong-Gu, Daejeon 305-764 ,  Republic of Korea","@type":"Person"},{"name":"Yu, Ge","affiliation":"3  Department of Vegetable Genetics and Breeding, College of Horticulture ,  Shenyang Agricultural University ,  Shenyang 110 161, Liaoning ,  PR China","@type":"Person"},{"name":"Piao, Zhong Yun","affiliation":"3  Department of Vegetable Genetics and Breeding, College of Horticulture ,  Shenyang Agricultural University ,  Shenyang 110 161, Liaoning ,  PR China","@type":"Person"},{"name":"Lim, Yong Pyo","affiliation":"1  Molecular Genetics and Genomics Lab, Department of Horticulture ,  Chungnam National University ,  Gung-Dong, Yuseong-Gu, Daejeon 305-764 ,  Republic of Korea","@type":"Person"}],"description":"Abstract.  Genic microsatellite markers, also known as functional markers, are preferred over anonymous markers as they reveal the variation in transcribed gene","pageStart":"305","pageEnd":"320","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/5/1/m_cover.gif?Expires=2147483647&Signature=fZbX~i0F4YmMgemzi~KYwJoeRvQcK3vESx0rzJw5EUkahhhho4dt2jCm2s4eb4lcFKiJ3IkaVuW~mFnQ0N-xsBlJ-atzuGchQMdBMlEaTp9QoaKOLQdSjpbwxtx3IKeAKX9MAFcEHgiyPIddtsEKqmKyu2yRzRpJfM6fHjheCB8GOlWvmsNX-mG7yzjfRRAgK2-xl55jEo9vwfkLAfBWwh9slsk8CoXZ~kVyHDiYgXlvdg5m4LybAGcWIMBm9UYCeJTV~chzGeImI3yLbjJOa6lXjiaaTcisKEGSS9lHedM-W~GtV9CboT5Ke43Hf0WuKYesrRRMqe~T4KpwdHK-gg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genic Microsatellite Markers in  Brassica rapa : Development, Characterization, Mapping, and Their Utility in Other Cultivated and Wild  Brassica  Relatives","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/5/1/m_cover.gif?Expires=2147483647&Signature=fZbX~i0F4YmMgemzi~KYwJoeRvQcK3vESx0rzJw5EUkahhhho4dt2jCm2s4eb4lcFKiJ3IkaVuW~mFnQ0N-xsBlJ-atzuGchQMdBMlEaTp9QoaKOLQdSjpbwxtx3IKeAKX9MAFcEHgiyPIddtsEKqmKyu2yRzRpJfM6fHjheCB8GOlWvmsNX-mG7yzjfRRAgK2-xl55jEo9vwfkLAfBWwh9slsk8CoXZ~kVyHDiYgXlvdg5m4LybAGcWIMBm9UYCeJTV~chzGeImI3yLbjJOa6lXjiaaTcisKEGSS9lHedM-W~GtV9CboT5Ke43Hf0WuKYesrRRMqe~T4KpwdHK-gg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genic Microsatellite Markers in  Brassica rapa : Development, Characterization, Mapping, and Their Utility in Other Cultivated and Wild  Brassica  Relatives" />
    <meta property="og:description" content="Abstract.  Genic microsatellite markers, also known as functional markers, are preferred over anonymous markers as they reveal the variation in transcribed gene" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/18/5/305/448180" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2011-07-17" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Ramchiary, Nirala" /><meta name="citation_author_institution" content="Molecular Genetics and Genomics Lab, Department of Horticulture, Chungnam National University, Gung-Dong, Yuseong-Gu, Daejeon 305-764, Republic of Korea" /><meta name="citation_author" content="Nguyen, Van Dan" /><meta name="citation_author_institution" content="Molecular Genetics and Genomics Lab, Department of Horticulture, Chungnam National University, Gung-Dong, Yuseong-Gu, Daejeon 305-764, Republic of Korea" /><meta name="citation_author" content="Li, Xiaonan" /><meta name="citation_author_institution" content="Molecular Genetics and Genomics Lab, Department of Horticulture, Chungnam National University, Gung-Dong, Yuseong-Gu, Daejeon 305-764, Republic of Korea" /><meta name="citation_author" content="Hong, Chang Pyo" /><meta name="citation_author_institution" content="Department of Life Science, and BK21 Program of Bio-Molecular Function, Pohang University of Science and Technology, Pohang 790-784, Republic of Korea" /><meta name="citation_author" content="Dhandapani, Vignesh" /><meta name="citation_author_institution" content="Molecular Genetics and Genomics Lab, Department of Horticulture, Chungnam National University, Gung-Dong, Yuseong-Gu, Daejeon 305-764, Republic of Korea" /><meta name="citation_author" content="Choi, Su Ryun" /><meta name="citation_author_institution" content="Molecular Genetics and Genomics Lab, Department of Horticulture, Chungnam National University, Gung-Dong, Yuseong-Gu, Daejeon 305-764, Republic of Korea" /><meta name="citation_author" content="Yu, Ge" /><meta name="citation_author_institution" content="Department of Vegetable Genetics and Breeding, College of Horticulture, Shenyang Agricultural University, Shenyang 110 161, Liaoning, PR China" /><meta name="citation_author" content="Piao, Zhong Yun" /><meta name="citation_author_institution" content="Department of Vegetable Genetics and Breeding, College of Horticulture, Shenyang Agricultural University, Shenyang 110 161, Liaoning, PR China" /><meta name="citation_author" content="Lim, Yong Pyo" /><meta name="citation_author_institution" content="Molecular Genetics and Genomics Lab, Department of Horticulture, Chungnam National University, Gung-Dong, Yuseong-Gu, Daejeon 305-764, Republic of Korea" /><meta name="citation_title" content="Genic Microsatellite Markers in Brassica rapa: Development, Characterization, Mapping, and Their Utility in Other Cultivated and Wild Brassica Relatives" /><meta name="citation_firstpage" content="305" /><meta name="citation_lastpage" content="320" /><meta name="citation_doi" content="10.1093/dnares/dsr017" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="18" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2011/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Genome analysis in Brassica with special reference to the experimental formation of B. napus and peculiar mode of fertilization; citation_author=UN; Jpn. J. Bot.;  citation_year=1935;  citation_volume=7;  citation_pages=389-452; " /><meta name="citation_reference" content="citation_title=Mapping of RFLP and quantitative trait loci in Brassica rapa and comparison to the linkage maps of B. napus, B. oleracea and Arabidopsis thaliana; Theor. Appl. Genet.;  citation_year=1994;  citation_volume=89;  citation_pages=885-94; " /><meta name="citation_reference" content="citation_title=Molecular markers in Brassica oilseed breeding: current status and future possibilities; Plant Breed;  citation_year=2004;  citation_volume=123;  citation_pages=1-8; " /><meta name="citation_reference" content="citation_title=Segmental structure of the Brassica napus genome based on comparative analysis with Arabidopsis thaliana; Genetics;  citation_year=2005;  citation_volume=171;  citation_pages=765-81; " /><meta name="citation_reference" content="citation_title=The reference genetic linkage map for the multinational Brassica rapa genome sequencing project; Theor. Appl. Genet.;  citation_year=2007;  citation_volume=115;  citation_pages=777-92; " /><meta name="citation_reference" content="citation_title=Comparative mapping of Brassica juncea and Arabidopsis thaliana using Intron Polymorphism (IP) markers: homoeologous relationships, diversification and evolution of the A, B and C Brassica genomes; BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=113" /><meta name="citation_reference" content="citation_title=Sequenced BAC anchored reference genetic map that reconciles the ten individual chromosomes of Brassica rapa; BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=432" /><meta name="citation_reference" content="citation_title=Mapping of yield influencing QTL in Brassica juncea: implications for breeding of a major oilseed crop of dryland areas; Theor. Appl. Genet.;  citation_year=2007;  citation_volume=115;  citation_pages=807-17; " /><meta name="citation_reference" content="citation_title=QTL analysis reveals context-dependent loci for seed glucosinolate trait in the oilseed Brassica juncea: importance of recurrent selection backcross scheme for the identification of ‘true’ QTL; Theor. Appl. Genet.;  citation_year=2007;  citation_volume=116;  citation_pages=77-85; " /><meta name="citation_reference" content="citation_title=Quantitative trait loci for flowering time and morphological traits in multiple populations of Brassica rapa; J. Exp. Bot.;  citation_year=2007;  citation_volume=58;  citation_pages=4005-16; " /><meta name="citation_reference" content="citation_title=Quantitative trait loci for glucosinolate accumulation in Brassica rapa leaves; New Phytol.;  citation_year=2008;  citation_volume=179;  citation_pages=1017-32; " /><meta name="citation_reference" content="citation_title=Mapping QTLs for root morphological traits in Brassica rapa L. based on AFLP and RAPD markers; J. Appl. Genet.;  citation_year=2008;  citation_volume=49;  citation_pages=23-31; " /><meta name="citation_reference" content="citation_title=A Brassica rapa linkage map of EST-based SNP markers for identification of candidate genes controlling flowering time and leaf morphological traits; DNA Res.;  citation_year=2009;  citation_volume=16;  citation_pages=311-23; " /><meta name="citation_reference" content="citation_title=Map based cloning and characterization of a gene controlling hairiness and seed coat color traits in Brassica rapa; Plant Mol. Biol.;  citation_year=2009;  citation_volume=69;  citation_pages=553-63; " /><meta name="citation_reference" content="citation_title=Characterization and effects of the replicated time gene FLC in Brassica rapa; Genetics;  citation_year=2002;  citation_volume=162;  citation_pages=1457-68; " /><meta name="citation_reference" content="citation_title=A high-density consensus map of barley to compare the distribution of QTLs for partial resistance to Puccinia hordei and of defence gene homologues; Theor. Appl. Genet.;  citation_year=2007;  citation_volume=114;  citation_pages=487-500; " /><meta name="citation_reference" content="citation_title=An integrated molecular linkage map of diploid wheat based on a Triticum boeoticum × T. monococcum RIL population; Theor. Appl. Genet.;  citation_year=2007;  citation_volume=115;  citation_pages=301-12; " /><meta name="citation_reference" content="citation_title=A high density barley microsatellite consensus map with 775 SSR loci; Theor. Appl. Genet.;  citation_year=2007;  citation_volume=114;  citation_pages=1091-103; " /><meta name="citation_reference" content="citation_title=A reference integrated map for cultivated grapevine (Vitis vinifera L.) from three crosses, based on 283 SSR and 501 SNP-based markers; Theor. Appl. Genet.;  citation_year=2008;  citation_volume=117;  citation_pages=499-511; " /><meta name="citation_reference" content="citation_title=High-density integrated linkage map based on SSR markers in soybean; DNA Res.;  citation_year=2009;  citation_volume=16;  citation_pages=213-25; " /><meta name="citation_reference" content="citation_title=A consensus genetic map of sorghum that integrates multiple component maps and high-throughput Diversity Array Technology (DArT) markers; BMC Plant Biol.;  citation_year=2009;  citation_volume=9;  citation_pages=13" /><meta name="citation_reference" content="citation_title=A SSR-based composite genetic linkage map for the cultivated peanut (Arachis hypogaea L.) genome; BMC Plant Biol.;  citation_year=2010;  citation_volume=10;  citation_pages=17" /><meta name="citation_reference" content="citation_title=Unigene derived microsatellite markers for the cereal genomes; Theor. Appl. Genet.;  citation_year=2006;  citation_volume=112;  citation_pages=808-17; " /><meta name="citation_reference" content="citation_title=Microsatellites in Brassica unigenes: relative abundance,marker design, and use in comparative physical mapping and genome analysis; Genome;  citation_year=2010;  citation_volume=53;  citation_pages=55-67; " /><meta name="citation_reference" content="citation_title=Identification, characterization and utilization of unigene derived microsatellite markers in tea (Camellia sinensis L.); BMC Plant Biol.;  citation_year=2009;  citation_volume=9;  citation_pages=53" /><meta name="citation_reference" content="citation_title=Microsatellites are preferentially associated with nonrepetitive DNA in plant genomes; Nat Genet.;  citation_year=2002;  citation_volume=30;  citation_pages=194-200; " /><meta name="citation_reference" content="citation_title=Genetic microsatellite markers in plants: features and applications; Trends Biotech.;  citation_year=2005;  citation_volume=23;  citation_pages=48-55; " /><meta name="citation_reference" content="citation_title=Transferability and polymorphism of barley EST-SSR markers used for phylogenetic analysis in Hordeum chilense; BMC Plant Biol.;  citation_year=2008;  citation_volume=8;  citation_pages=97" /><meta name="citation_reference" content="citation_title=Localization of anchor loci representing five hundred annotated rice genes to wheat chromosomes using PLUG markers; Theor. Appl. Genet.;  citation_year=2008;  citation_volume=118;  citation_pages=499-514; " /><meta name="citation_reference" content="citation_title=Development and mapping of simple sequence repeat markers for pearl millet from data mining of expressed sequence tags; BMC Plant Biol.;  citation_year=2008;  citation_volume=8;  citation_pages=119" /><meta name="citation_reference" content="citation_title=Genetic map construction and QTL mapping of resistance to blackleg (Leptosphaeria maculans) disease in Australian canola (Brassica napus L.) cultivars; Theor. Appl. Genet.;  citation_year=2009;  citation_volume=120;  citation_pages=71-83; " /><meta name="citation_reference" content="citation_title=Gene-based microsatellites for cassava (Manihot esculenta Crantz): prevalence, polymorphisms, and cross-taxa utility; BMC Plant Biol.;  citation_year=2009;  citation_volume=9;  citation_pages=118" /><meta name="citation_reference" content="citation_title=Development of a high density integrated reference genetic linkage map for the multinational Brassica rapa Genome Sequencing Project; Genome;  citation_year=2010;  citation_volume=53;  citation_pages=939-47; " /><meta name="citation_reference" content="citation_title=CAP3: a DNA sequence assembly program; Genome Res.;  citation_year=1999;  citation_volume=9;  citation_pages=868-77; " /><meta name="citation_reference" content="citation_title=Genomic distribution of simple sequence repeats in Brassica rapa; Mol. Cells;  citation_year=2007;  citation_volume=23;  citation_pages=349-56; " /><meta name="citation_reference" content="citation_title=Primer3 on the WWW for general users and for biologist programmers; Meth. Mol. Biol.;  citation_year=2000;  citation_volume=132;  citation_pages=365-86; " /><meta name="citation_reference" content="citation_title=The sequence manipulation suite: JavaScript programs for analyzing and formatting protein and DNA sequences; Biotechniques;  citation_year=2000;  citation_volume=28;  citation_pages=1102-4; " /><meta name="citation_reference" content="citation_title=The abundance of various polymorphic microsatellite motifs differs between plants and vertebrates; Nucleic Acids Res.;  citation_year=1993;  citation_volume=21;  citation_pages=1111-5; " /><meta name="citation_reference" content="citation_title=Abundance, polymorphism and genetic mapping of microsatellites in oilseed rape (Brassica napus L.); Plant Breed.;  citation_year=1999;  citation_volume=118;  citation_pages=323-6; " /><meta name="citation_reference" content="citation_title=Transferability and genome specificity of a new set of microsatellite primers among Brassica species of the U triangle; Mol. Ecol. Notes;  citation_year=2002;  citation_volume=2;  citation_pages=7-11; " /><meta name="citation_reference" content="citation_title=Efficient large-scale development of microsatellites for marker and mapping applications in Brassica crop species; Theor. Appl. Genet.;  citation_year=2004;  citation_volume=108;  citation_pages=1103-12; " /><meta name="citation_reference" content="citation_title=Isolation and characterization of microsatellites in Brassica rapa L; Theor. Appl. Genet.;  citation_year=2002;  citation_volume=104;  citation_pages=1092-8; " /><meta name="citation_reference" content="citation_title=Simple sequence repeat-based comparative genomics between Brassica rapa and Arabidopsis thaliana: the genetic origin of clubroot resistance; Genetics;  citation_year=2006;  citation_volume=173;  citation_pages=309-19; " /><meta name="citation_reference" content="citation_title=Construction of an oilseed rape (Brassica napus L.) genetic map with SSR markers; Theor. Appl. Genet.;  citation_year=2005;  citation_volume=111;  citation_pages=1514-23; " /><meta name="citation_reference" content="citation_title=A sequence-tagged linkage map of Brassica rapa; Genetics;  citation_year=2006;  citation_volume=174;  citation_pages=29-39; " /><meta name="citation_reference" content="citation_title=Development of a set of public SSR markers derived from genomic sequence of a rapid cycling Brassica oleracea L. genotype; Theor. Appl. Genet.;  citation_year=2008;  citation_volume=117;  citation_pages=977-85; " /><meta name="citation_reference" content="citation_title=Development and genetic mapping of microsatellite markers from genome survey sequences in Brassica napus; Theor. Appl. Genet.;  citation_year=2009;  citation_volume=118;  citation_pages=1121-31; " /><meta name="citation_reference" content="citation_title=Construction of integrated genetic linkage maps by means of a new computer package: JoinMap; Plant J.;  citation_year=1993;  citation_volume=3;  citation_pages=739-44; " /><meta name="citation_reference" content="citation_publisher=Plant Research International, Wageningen, The Netherlands; JoinMap® Version 3.0: Software for the Calculation of Genetic Linkage Map;  citation_year=2001; " /><meta name="citation_reference" content="citation_title=The estimation of map distance from recombination values; Ann. Eugen.;  citation_year=1944;  citation_volume=12;  citation_pages=172-5; " /><meta name="citation_reference" content="citation_title=MapChart: software for the graphical presentation of linkage maps and QTLs; J. Hered.;  citation_year=2002;  citation_volume=93;  citation_pages=77-8; " /><meta name="citation_reference" content="citation_title=Construction of a genetic linkage map in man using restriction fragment length polymorphisms; Am. J. Hum. Genet.;  citation_year=1980;  citation_volume=32;  citation_pages=314-31; " /><meta name="citation_reference" content="citation_title=The ABC's of comparative genomics in the Brassicaceae: building blocks of crucifer genomes; Trends Plant Sci.;  citation_year=2006;  citation_volume=11;  citation_pages=535-42; " /><meta name="citation_reference" content="citation_title=A consensus linkage map for rapeseed (Brassica napus L.): construction and integration of three individual maps from DH populations; Theor. Appl. Genet.;  citation_year=2001;  citation_volume=103;  citation_pages=491-507; " /><meta name="citation_reference" content="citation_title=An integrated genetic/RFLP map of the Arabidopsis; Plant J.;  citation_year=1993;  citation_volume=3;  citation_pages=745-54; " /><meta name="citation_reference" content="citation_title=A consensus map for loblolly pine (Pinus taeda L.). I. construction and integration of individual linkage maps from two outbred three-generation pedigrees; Genetics;  citation_year=1999;  citation_volume=151;  citation_pages=321-30; " /><meta name="citation_reference" content="citation_title=An integrated AFLP and RFLP Brassica oleracea linkage map from two morphologically distinct doubled-haploid mapping populations; Theor. Appl. Genet.;  citation_year=2000;  citation_volume=100;  citation_pages=75-81; " /><meta name="citation_reference" content="citation_title=An integrated interspecific AFLP map of lettuce (Lactuca) based on two L.sativa times L. saligna F2 populations; Theor. Appl. Genet.;  citation_year=2001;  citation_volume=103;  citation_pages=638-47; " /><meta name="citation_reference" content="citation_title=A consensus linkage map for Sugi (Cryptomeria japonica) from two pedigrees, based on microsatellites and expressed sequence tags; Genetics;  citation_year=2003;  citation_volume=165;  citation_pages=1551-68; " /><meta name="citation_reference" content="citation_title=A consensus map of rye integrating mapping data from five mapping populations; Theor. Appl. Genet.;  citation_year=2009;  citation_volume=118;  citation_pages=793-800; " /><meta name="citation_reference" content="citation_title=Characterization and identification of (CT)n microsatellites in soybean using sheared genomic libraries; DNA Res.;  citation_year=2000;  citation_volume=7;  citation_pages=103-10; " /><meta name="citation_reference" content="citation_title=Characterization of the soybean genome using EST derived microsatellite markers; DNA Res.;  citation_year=2007;  citation_volume=14;  citation_pages=271-81; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/18/5/305/1225525/dsr017.pdf" /><meta name="description" content="Abstract.  Genic microsatellite markers, also known as functional markers, are preferred over anonymous markers as they reveal the variation in transcribed gene" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/18/5/305/448180" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/18/5/305/448180" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/18/5/305/448180");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsr017';
    </script>
<title>
	Genic Microsatellite Markers in Brassica rapa: Development, Characterization, Mapping, and Their Utility in Other Cultivated and Wild Brassica Relatives | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./448180" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BYKTATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUmFtY2hpYXJ5LCBOaXJhbGEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNb2xlY3VsYXIgR2VuZXRpY3MgYW5kIEdlbm9taWNzIExhYiwgRGVwYXJ0bWVudCBvZiBIb3J0aWN1bHR1cmUsIENodW5nbmFtIE5hdGlvbmFsIFVuaXZlcnNpdHksIEd1bmctRG9uZywgWXVzZW9uZy1HdSwgRGFlamVvbiAzMDUtNzY0LCBSZXB1YmxpYyBvZiBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik5ndXllbiwgVmFuIERhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik1vbGVjdWxhciBHZW5ldGljcyBhbmQgR2Vub21pY3MgTGFiLCBEZXBhcnRtZW50IG9mIEhvcnRpY3VsdHVyZSwgQ2h1bmduYW0gTmF0aW9uYWwgVW5pdmVyc2l0eSwgR3VuZy1Eb25nLCBZdXNlb25nLUd1LCBEYWVqZW9uIDMwNS03NjQsIFJlcHVibGljIG9mIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGksIFhpYW9uYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNb2xlY3VsYXIgR2VuZXRpY3MgYW5kIEdlbm9taWNzIExhYiwgRGVwYXJ0bWVudCBvZiBIb3J0aWN1bHR1cmUsIENodW5nbmFtIE5hdGlvbmFsIFVuaXZlcnNpdHksIEd1bmctRG9uZywgWXVzZW9uZy1HdSwgRGFlamVvbiAzMDUtNzY0LCBSZXB1YmxpYyBvZiBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkhvbmcsIENoYW5nIFB5byIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTGlmZSBTY2llbmNlLCBhbmQgQksyMSBQcm9ncmFtIG9mIEJpby1Nb2xlY3VsYXIgRnVuY3Rpb24sIFBvaGFuZyBVbml2ZXJzaXR5IG9mIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIFBvaGFuZyA3OTAtNzg0LCBSZXB1YmxpYyBvZiBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkRoYW5kYXBhbmksIFZpZ25lc2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNb2xlY3VsYXIgR2VuZXRpY3MgYW5kIEdlbm9taWNzIExhYiwgRGVwYXJ0bWVudCBvZiBIb3J0aWN1bHR1cmUsIENodW5nbmFtIE5hdGlvbmFsIFVuaXZlcnNpdHksIEd1bmctRG9uZywgWXVzZW9uZy1HdSwgRGFlamVvbiAzMDUtNzY0LCBSZXB1YmxpYyBvZiBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNob2ksIFN1IFJ5dW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNb2xlY3VsYXIgR2VuZXRpY3MgYW5kIEdlbm9taWNzIExhYiwgRGVwYXJ0bWVudCBvZiBIb3J0aWN1bHR1cmUsIENodW5nbmFtIE5hdGlvbmFsIFVuaXZlcnNpdHksIEd1bmctRG9uZywgWXVzZW9uZy1HdSwgRGFlamVvbiAzMDUtNzY0LCBSZXB1YmxpYyBvZiBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ill1LCBHZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgVmVnZXRhYmxlIEdlbmV0aWNzIGFuZCBCcmVlZGluZywgQ29sbGVnZSBvZiBIb3J0aWN1bHR1cmUsIFNoZW55YW5nIEFncmljdWx0dXJhbCBVbml2ZXJzaXR5LCBTaGVueWFuZyAxMTAgMTYxLCBMaWFvbmluZywgUFIgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJQaWFvLCBaaG9uZyBZdW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFZlZ2V0YWJsZSBHZW5ldGljcyBhbmQgQnJlZWRpbmcsIENvbGxlZ2Ugb2YgSG9ydGljdWx0dXJlLCBTaGVueWFuZyBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgU2hlbnlhbmcgMTEwIDE2MSwgTGlhb25pbmcsIFBSIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGltLCBZb25nIFB5byIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik1vbGVjdWxhciBHZW5ldGljcyBhbmQgR2Vub21pY3MgTGFiLCBEZXBhcnRtZW50IG9mIEhvcnRpY3VsdHVyZSwgQ2h1bmduYW0gTmF0aW9uYWwgVW5pdmVyc2l0eSwgR3VuZy1Eb25nLCBZdXNlb25nLUd1LCBEYWVqZW9uIDMwNS03NjQsIFJlcHVibGljIG9mIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJHZW5pYyBNaWNyb3NhdGVsbGl0ZSBNYXJrZXJzIGluIEJyYXNzaWNhIHJhcGE6IERldmVsb3BtZW50LCBDaGFyYWN0ZXJpemF0aW9uLCBNYXBwaW5nLCBhbmQgVGhlaXIgVXRpbGl0eSBpbiBPdGhlciBDdWx0aXZhdGVkIGFuZCBXaWxkIEJyYXNzaWNhIFJlbGF0aXZlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjMwNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMzIwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNyMDE3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTEvMTAvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgYW5hbHlzaXMgaW4gQnJhc3NpY2Egd2l0aCBzcGVjaWFsIHJlZmVyZW5jZSB0byB0aGUgZXhwZXJpbWVudGFsIGZvcm1hdGlvbiBvZiBCLiBuYXB1cyBhbmQgcGVjdWxpYXIgbW9kZSBvZiBmZXJ0aWxpemF0aW9uOyBjaXRhdGlvbl9hdXRob3I9VU47IEpwbi4gSi4gQm90LjsgIGNpdGF0aW9uX3llYXI9MTkzNTsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9Mzg5LTQ1MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFwcGluZyBvZiBSRkxQIGFuZCBxdWFudGl0YXRpdmUgdHJhaXQgbG9jaSBpbiBCcmFzc2ljYSByYXBhIGFuZCBjb21wYXJpc29uIHRvIHRoZSBsaW5rYWdlIG1hcHMgb2YgQi4gbmFwdXMsIEIuIG9sZXJhY2VhIGFuZCBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT04OTsgIGNpdGF0aW9uX3BhZ2VzPTg4NS05NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIG1hcmtlcnMgaW4gQnJhc3NpY2Egb2lsc2VlZCBicmVlZGluZzogY3VycmVudCBzdGF0dXMgYW5kIGZ1dHVyZSBwb3NzaWJpbGl0aWVzOyBQbGFudCBCcmVlZDsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjM7ICBjaXRhdGlvbl9wYWdlcz0xLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlZ21lbnRhbCBzdHJ1Y3R1cmUgb2YgdGhlIEJyYXNzaWNhIG5hcHVzIGdlbm9tZSBiYXNlZCBvbiBjb21wYXJhdGl2ZSBhbmFseXNpcyB3aXRoIEFyYWJpZG9wc2lzIHRoYWxpYW5hOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNzE7ICBjaXRhdGlvbl9wYWdlcz03NjUtODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSByZWZlcmVuY2UgZ2VuZXRpYyBsaW5rYWdlIG1hcCBmb3IgdGhlIG11bHRpbmF0aW9uYWwgQnJhc3NpY2EgcmFwYSBnZW5vbWUgc2VxdWVuY2luZyBwcm9qZWN0OyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTExNTsgIGNpdGF0aW9uX3BhZ2VzPTc3Ny05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgbWFwcGluZyBvZiBCcmFzc2ljYSBqdW5jZWEgYW5kIEFyYWJpZG9wc2lzIHRoYWxpYW5hIHVzaW5nIEludHJvbiBQb2x5bW9ycGhpc20gKElQKSBtYXJrZXJzOiBob21vZW9sb2dvdXMgcmVsYXRpb25zaGlwcywgZGl2ZXJzaWZpY2F0aW9uIGFuZCBldm9sdXRpb24gb2YgdGhlIEEsIEIgYW5kIEMgQnJhc3NpY2EgZ2Vub21lczsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0xMTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jZWQgQkFDIGFuY2hvcmVkIHJlZmVyZW5jZSBnZW5ldGljIG1hcCB0aGF0IHJlY29uY2lsZXMgdGhlIHRlbiBpbmRpdmlkdWFsIGNocm9tb3NvbWVzIG9mIEJyYXNzaWNhIHJhcGE7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTQzMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcHBpbmcgb2YgeWllbGQgaW5mbHVlbmNpbmcgUVRMIGluIEJyYXNzaWNhIGp1bmNlYTogaW1wbGljYXRpb25zIGZvciBicmVlZGluZyBvZiBhIG1ham9yIG9pbHNlZWQgY3JvcCBvZiBkcnlsYW5kIGFyZWFzOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTExNTsgIGNpdGF0aW9uX3BhZ2VzPTgwNy0xNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UVRMIGFuYWx5c2lzIHJldmVhbHMgY29udGV4dC1kZXBlbmRlbnQgbG9jaSBmb3Igc2VlZCBnbHVjb3Npbm9sYXRlIHRyYWl0IGluIHRoZSBvaWxzZWVkIEJyYXNzaWNhIGp1bmNlYTogaW1wb3J0YW5jZSBvZiByZWN1cnJlbnQgc2VsZWN0aW9uIGJhY2tjcm9zcyBzY2hlbWUgZm9yIHRoZSBpZGVudGlmaWNhdGlvbiBvZiDigJh0cnVl4oCZIFFUTDsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTY7ICBjaXRhdGlvbl9wYWdlcz03Ny04NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgZm9yIGZsb3dlcmluZyB0aW1lIGFuZCBtb3JwaG9sb2dpY2FsIHRyYWl0cyBpbiBtdWx0aXBsZSBwb3B1bGF0aW9ucyBvZiBCcmFzc2ljYSByYXBhOyBKLiBFeHAuIEJvdC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NTg7ICBjaXRhdGlvbl9wYWdlcz00MDA1LTE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1RdWFudGl0YXRpdmUgdHJhaXQgbG9jaSBmb3IgZ2x1Y29zaW5vbGF0ZSBhY2N1bXVsYXRpb24gaW4gQnJhc3NpY2EgcmFwYSBsZWF2ZXM7IE5ldyBQaHl0b2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE3OTsgIGNpdGF0aW9uX3BhZ2VzPTEwMTctMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcHBpbmcgUVRMcyBmb3Igcm9vdCBtb3JwaG9sb2dpY2FsIHRyYWl0cyBpbiBCcmFzc2ljYSByYXBhIEwuIGJhc2VkIG9uIEFGTFAgYW5kIFJBUEQgbWFya2VyczsgSi4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ5OyAgY2l0YXRpb25fcGFnZXM9MjMtMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgQnJhc3NpY2EgcmFwYSBsaW5rYWdlIG1hcCBvZiBFU1QtYmFzZWQgU05QIG1hcmtlcnMgZm9yIGlkZW50aWZpY2F0aW9uIG9mIGNhbmRpZGF0ZSBnZW5lcyBjb250cm9sbGluZyBmbG93ZXJpbmcgdGltZSBhbmQgbGVhZiBtb3JwaG9sb2dpY2FsIHRyYWl0czsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0zMTEtMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcCBiYXNlZCBjbG9uaW5nIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIGEgZ2VuZSBjb250cm9sbGluZyBoYWlyaW5lc3MgYW5kIHNlZWQgY29hdCBjb2xvciB0cmFpdHMgaW4gQnJhc3NpY2EgcmFwYTsgUGxhbnQgTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT02OTsgIGNpdGF0aW9uX3BhZ2VzPTU1My02MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhcmFjdGVyaXphdGlvbiBhbmQgZWZmZWN0cyBvZiB0aGUgcmVwbGljYXRlZCB0aW1lIGdlbmUgRkxDIGluIEJyYXNzaWNhIHJhcGE7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTE2MjsgIGNpdGF0aW9uX3BhZ2VzPTE0NTctNjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgaGlnaC1kZW5zaXR5IGNvbnNlbnN1cyBtYXAgb2YgYmFybGV5IHRvIGNvbXBhcmUgdGhlIGRpc3RyaWJ1dGlvbiBvZiBRVExzIGZvciBwYXJ0aWFsIHJlc2lzdGFuY2UgdG8gUHVjY2luaWEgaG9yZGVpIGFuZCBvZiBkZWZlbmNlIGdlbmUgaG9tb2xvZ3VlczsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTQ7ICBjaXRhdGlvbl9wYWdlcz00ODctNTAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBpbnRlZ3JhdGVkIG1vbGVjdWxhciBsaW5rYWdlIG1hcCBvZiBkaXBsb2lkIHdoZWF0IGJhc2VkIG9uIGEgVHJpdGljdW0gYm9lb3RpY3VtIMOXIFQuIG1vbm9jb2NjdW0gUklMIHBvcHVsYXRpb247IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTE1OyAgY2l0YXRpb25fcGFnZXM9MzAxLTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGhpZ2ggZGVuc2l0eSBiYXJsZXkgbWljcm9zYXRlbGxpdGUgY29uc2Vuc3VzIG1hcCB3aXRoIDc3NSBTU1IgbG9jaTsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTQ7ICBjaXRhdGlvbl9wYWdlcz0xMDkxLTEwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSByZWZlcmVuY2UgaW50ZWdyYXRlZCBtYXAgZm9yIGN1bHRpdmF0ZWQgZ3JhcGV2aW5lIChWaXRpcyB2aW5pZmVyYSBMLikgZnJvbSB0aHJlZSBjcm9zc2VzLCBiYXNlZCBvbiAyODMgU1NSIGFuZCA1MDEgU05QLWJhc2VkIG1hcmtlcnM7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTE3OyAgY2l0YXRpb25fcGFnZXM9NDk5LTUxMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC1kZW5zaXR5IGludGVncmF0ZWQgbGlua2FnZSBtYXAgYmFzZWQgb24gU1NSIG1hcmtlcnMgaW4gc295YmVhbjsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0yMTMtMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY29uc2Vuc3VzIGdlbmV0aWMgbWFwIG9mIHNvcmdodW0gdGhhdCBpbnRlZ3JhdGVzIG11bHRpcGxlIGNvbXBvbmVudCBtYXBzIGFuZCBoaWdoLXRocm91Z2hwdXQgRGl2ZXJzaXR5IEFycmF5IFRlY2hub2xvZ3kgKERBclQpIG1hcmtlcnM7IEJNQyBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIFNTUi1iYXNlZCBjb21wb3NpdGUgZ2VuZXRpYyBsaW5rYWdlIG1hcCBmb3IgdGhlIGN1bHRpdmF0ZWQgcGVhbnV0IChBcmFjaGlzIGh5cG9nYWVhIEwuKSBnZW5vbWU7IEJNQyBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTE3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VW5pZ2VuZSBkZXJpdmVkIG1pY3Jvc2F0ZWxsaXRlIG1hcmtlcnMgZm9yIHRoZSBjZXJlYWwgZ2Vub21lczsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTI7ICBjaXRhdGlvbl9wYWdlcz04MDgtMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3Jvc2F0ZWxsaXRlcyBpbiBCcmFzc2ljYSB1bmlnZW5lczogcmVsYXRpdmUgYWJ1bmRhbmNlLG1hcmtlciBkZXNpZ24sIGFuZCB1c2UgaW4gY29tcGFyYXRpdmUgcGh5c2ljYWwgbWFwcGluZyBhbmQgZ2Vub21lIGFuYWx5c2lzOyBHZW5vbWU7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NTM7ICBjaXRhdGlvbl9wYWdlcz01NS02NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24sIGNoYXJhY3Rlcml6YXRpb24gYW5kIHV0aWxpemF0aW9uIG9mIHVuaWdlbmUgZGVyaXZlZCBtaWNyb3NhdGVsbGl0ZSBtYXJrZXJzIGluIHRlYSAoQ2FtZWxsaWEgc2luZW5zaXMgTC4pOyBCTUMgUGxhbnQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTUzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9zYXRlbGxpdGVzIGFyZSBwcmVmZXJlbnRpYWxseSBhc3NvY2lhdGVkIHdpdGggbm9ucmVwZXRpdGl2ZSBETkEgaW4gcGxhbnQgZ2Vub21lczsgTmF0IEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTE5NC0yMDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgbWljcm9zYXRlbGxpdGUgbWFya2VycyBpbiBwbGFudHM6IGZlYXR1cmVzIGFuZCBhcHBsaWNhdGlvbnM7IFRyZW5kcyBCaW90ZWNoLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTQ4LTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2ZlcmFiaWxpdHkgYW5kIHBvbHltb3JwaGlzbSBvZiBiYXJsZXkgRVNULVNTUiBtYXJrZXJzIHVzZWQgZm9yIHBoeWxvZ2VuZXRpYyBhbmFseXNpcyBpbiBIb3JkZXVtIGNoaWxlbnNlOyBCTUMgUGxhbnQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTk3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TG9jYWxpemF0aW9uIG9mIGFuY2hvciBsb2NpIHJlcHJlc2VudGluZyBmaXZlIGh1bmRyZWQgYW5ub3RhdGVkIHJpY2UgZ2VuZXMgdG8gd2hlYXQgY2hyb21vc29tZXMgdXNpbmcgUExVRyBtYXJrZXJzOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTExODsgIGNpdGF0aW9uX3BhZ2VzPTQ5OS01MTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IGFuZCBtYXBwaW5nIG9mIHNpbXBsZSBzZXF1ZW5jZSByZXBlYXQgbWFya2VycyBmb3IgcGVhcmwgbWlsbGV0IGZyb20gZGF0YSBtaW5pbmcgb2YgZXhwcmVzc2VkIHNlcXVlbmNlIHRhZ3M7IEJNQyBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MTE5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBtYXAgY29uc3RydWN0aW9uIGFuZCBRVEwgbWFwcGluZyBvZiByZXNpc3RhbmNlIHRvIGJsYWNrbGVnIChMZXB0b3NwaGFlcmlhIG1hY3VsYW5zKSBkaXNlYXNlIGluIEF1c3RyYWxpYW4gY2Fub2xhIChCcmFzc2ljYSBuYXB1cyBMLikgY3VsdGl2YXJzOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEyMDsgIGNpdGF0aW9uX3BhZ2VzPTcxLTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lLWJhc2VkIG1pY3Jvc2F0ZWxsaXRlcyBmb3IgY2Fzc2F2YSAoTWFuaWhvdCBlc2N1bGVudGEgQ3JhbnR6KTogcHJldmFsZW5jZSwgcG9seW1vcnBoaXNtcywgYW5kIGNyb3NzLXRheGEgdXRpbGl0eTsgQk1DIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0xMTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBvZiBhIGhpZ2ggZGVuc2l0eSBpbnRlZ3JhdGVkIHJlZmVyZW5jZSBnZW5ldGljIGxpbmthZ2UgbWFwIGZvciB0aGUgbXVsdGluYXRpb25hbCBCcmFzc2ljYSByYXBhIEdlbm9tZSBTZXF1ZW5jaW5nIFByb2plY3Q7IEdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT01MzsgIGNpdGF0aW9uX3BhZ2VzPTkzOS00NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q0FQMzogYSBETkEgc2VxdWVuY2UgYXNzZW1ibHkgcHJvZ3JhbTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTg2OC03NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBkaXN0cmlidXRpb24gb2Ygc2ltcGxlIHNlcXVlbmNlIHJlcGVhdHMgaW4gQnJhc3NpY2EgcmFwYTsgTW9sLiBDZWxsczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTM0OS01NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJpbWVyMyBvbiB0aGUgV1dXIGZvciBnZW5lcmFsIHVzZXJzIGFuZCBmb3IgYmlvbG9naXN0IHByb2dyYW1tZXJzOyBNZXRoLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTEzMjsgIGNpdGF0aW9uX3BhZ2VzPTM2NS04NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHNlcXVlbmNlIG1hbmlwdWxhdGlvbiBzdWl0ZTogSmF2YVNjcmlwdCBwcm9ncmFtcyBmb3IgYW5hbHl6aW5nIGFuZCBmb3JtYXR0aW5nIHByb3RlaW4gYW5kIEROQSBzZXF1ZW5jZXM7IEJpb3RlY2huaXF1ZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz0xMTAyLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBhYnVuZGFuY2Ugb2YgdmFyaW91cyBwb2x5bW9ycGhpYyBtaWNyb3NhdGVsbGl0ZSBtb3RpZnMgZGlmZmVycyBiZXR3ZWVuIHBsYW50cyBhbmQgdmVydGVicmF0ZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTExMTEtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWJ1bmRhbmNlLCBwb2x5bW9ycGhpc20gYW5kIGdlbmV0aWMgbWFwcGluZyBvZiBtaWNyb3NhdGVsbGl0ZXMgaW4gb2lsc2VlZCByYXBlIChCcmFzc2ljYSBuYXB1cyBMLik7IFBsYW50IEJyZWVkLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTg7ICBjaXRhdGlvbl9wYWdlcz0zMjMtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNmZXJhYmlsaXR5IGFuZCBnZW5vbWUgc3BlY2lmaWNpdHkgb2YgYSBuZXcgc2V0IG9mIG1pY3Jvc2F0ZWxsaXRlIHByaW1lcnMgYW1vbmcgQnJhc3NpY2Egc3BlY2llcyBvZiB0aGUgVSB0cmlhbmdsZTsgTW9sLiBFY29sLiBOb3RlczsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9Ny0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RWZmaWNpZW50IGxhcmdlLXNjYWxlIGRldmVsb3BtZW50IG9mIG1pY3Jvc2F0ZWxsaXRlcyBmb3IgbWFya2VyIGFuZCBtYXBwaW5nIGFwcGxpY2F0aW9ucyBpbiBCcmFzc2ljYSBjcm9wIHNwZWNpZXM7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTA4OyAgY2l0YXRpb25fcGFnZXM9MTEwMy0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNvbGF0aW9uIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIG1pY3Jvc2F0ZWxsaXRlcyBpbiBCcmFzc2ljYSByYXBhIEw7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTA0OyAgY2l0YXRpb25fcGFnZXM9MTA5Mi04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW1wbGUgc2VxdWVuY2UgcmVwZWF0LWJhc2VkIGNvbXBhcmF0aXZlIGdlbm9taWNzIGJldHdlZW4gQnJhc3NpY2EgcmFwYSBhbmQgQXJhYmlkb3BzaXMgdGhhbGlhbmE6IHRoZSBnZW5ldGljIG9yaWdpbiBvZiBjbHVicm9vdCByZXNpc3RhbmNlOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNzM7ICBjaXRhdGlvbl9wYWdlcz0zMDktMTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBvZiBhbiBvaWxzZWVkIHJhcGUgKEJyYXNzaWNhIG5hcHVzIEwuKSBnZW5ldGljIG1hcCB3aXRoIFNTUiBtYXJrZXJzOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTExMTsgIGNpdGF0aW9uX3BhZ2VzPTE1MTQtMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgc2VxdWVuY2UtdGFnZ2VkIGxpbmthZ2UgbWFwIG9mIEJyYXNzaWNhIHJhcGE7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE3NDsgIGNpdGF0aW9uX3BhZ2VzPTI5LTM5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBvZiBhIHNldCBvZiBwdWJsaWMgU1NSIG1hcmtlcnMgZGVyaXZlZCBmcm9tIGdlbm9taWMgc2VxdWVuY2Ugb2YgYSByYXBpZCBjeWNsaW5nIEJyYXNzaWNhIG9sZXJhY2VhIEwuIGdlbm90eXBlOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTExNzsgIGNpdGF0aW9uX3BhZ2VzPTk3Ny04NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgYW5kIGdlbmV0aWMgbWFwcGluZyBvZiBtaWNyb3NhdGVsbGl0ZSBtYXJrZXJzIGZyb20gZ2Vub21lIHN1cnZleSBzZXF1ZW5jZXMgaW4gQnJhc3NpY2EgbmFwdXM7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTE4OyAgY2l0YXRpb25fcGFnZXM9MTEyMS0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIG9mIGludGVncmF0ZWQgZ2VuZXRpYyBsaW5rYWdlIG1hcHMgYnkgbWVhbnMgb2YgYSBuZXcgY29tcHV0ZXIgcGFja2FnZTogSm9pbk1hcDsgUGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTczOS00NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPVBsYW50IFJlc2VhcmNoIEludGVybmF0aW9uYWwsIFdhZ2VuaW5nZW4sIFRoZSBOZXRoZXJsYW5kczsgSm9pbk1hcMKuIFZlcnNpb24gMy4wOiBTb2Z0d2FyZSBmb3IgdGhlIENhbGN1bGF0aW9uIG9mIEdlbmV0aWMgTGlua2FnZSBNYXA7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBlc3RpbWF0aW9uIG9mIG1hcCBkaXN0YW5jZSBmcm9tIHJlY29tYmluYXRpb24gdmFsdWVzOyBBbm4uIEV1Z2VuLjsgIGNpdGF0aW9uX3llYXI9MTk0NDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTE3Mi01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXBDaGFydDogc29mdHdhcmUgZm9yIHRoZSBncmFwaGljYWwgcHJlc2VudGF0aW9uIG9mIGxpbmthZ2UgbWFwcyBhbmQgUVRMczsgSi4gSGVyZWQuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTkzOyAgY2l0YXRpb25fcGFnZXM9NzctODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIG9mIGEgZ2VuZXRpYyBsaW5rYWdlIG1hcCBpbiBtYW4gdXNpbmcgcmVzdHJpY3Rpb24gZnJhZ21lbnQgbGVuZ3RoIHBvbHltb3JwaGlzbXM7IEFtLiBKLiBIdW0uIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk4MDsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPTMxNC0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEFCQydzIG9mIGNvbXBhcmF0aXZlIGdlbm9taWNzIGluIHRoZSBCcmFzc2ljYWNlYWU6IGJ1aWxkaW5nIGJsb2NrcyBvZiBjcnVjaWZlciBnZW5vbWVzOyBUcmVuZHMgUGxhbnQgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTUzNS00MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb25zZW5zdXMgbGlua2FnZSBtYXAgZm9yIHJhcGVzZWVkIChCcmFzc2ljYSBuYXB1cyBMLik6IGNvbnN0cnVjdGlvbiBhbmQgaW50ZWdyYXRpb24gb2YgdGhyZWUgaW5kaXZpZHVhbCBtYXBzIGZyb20gREggcG9wdWxhdGlvbnM7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTAzOyAgY2l0YXRpb25fcGFnZXM9NDkxLTUwNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gaW50ZWdyYXRlZCBnZW5ldGljL1JGTFAgbWFwIG9mIHRoZSBBcmFiaWRvcHNpczsgUGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTc0NS01NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb25zZW5zdXMgbWFwIGZvciBsb2Jsb2xseSBwaW5lIChQaW51cyB0YWVkYSBMLikuIEkuIGNvbnN0cnVjdGlvbiBhbmQgaW50ZWdyYXRpb24gb2YgaW5kaXZpZHVhbCBsaW5rYWdlIG1hcHMgZnJvbSB0d28gb3V0YnJlZCB0aHJlZS1nZW5lcmF0aW9uIHBlZGlncmVlczsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTUxOyAgY2l0YXRpb25fcGFnZXM9MzIxLTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBpbnRlZ3JhdGVkIEFGTFAgYW5kIFJGTFAgQnJhc3NpY2Egb2xlcmFjZWEgbGlua2FnZSBtYXAgZnJvbSB0d28gbW9ycGhvbG9naWNhbGx5IGRpc3RpbmN0IGRvdWJsZWQtaGFwbG9pZCBtYXBwaW5nIHBvcHVsYXRpb25zOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTEwMDsgIGNpdGF0aW9uX3BhZ2VzPTc1LTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBpbnRlZ3JhdGVkIGludGVyc3BlY2lmaWMgQUZMUCBtYXAgb2YgbGV0dHVjZSAoTGFjdHVjYSkgYmFzZWQgb24gdHdvIEwuc2F0aXZhIHRpbWVzIEwuIHNhbGlnbmEgRjIgcG9wdWxhdGlvbnM7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTAzOyAgY2l0YXRpb25fcGFnZXM9NjM4LTQ3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNvbnNlbnN1cyBsaW5rYWdlIG1hcCBmb3IgU3VnaSAoQ3J5cHRvbWVyaWEgamFwb25pY2EpIGZyb20gdHdvIHBlZGlncmVlcywgYmFzZWQgb24gbWljcm9zYXRlbGxpdGVzIGFuZCBleHByZXNzZWQgc2VxdWVuY2UgdGFnczsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTY1OyAgY2l0YXRpb25fcGFnZXM9MTU1MS02ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb25zZW5zdXMgbWFwIG9mIHJ5ZSBpbnRlZ3JhdGluZyBtYXBwaW5nIGRhdGEgZnJvbSBmaXZlIG1hcHBpbmcgcG9wdWxhdGlvbnM7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTE4OyAgY2l0YXRpb25fcGFnZXM9NzkzLTgwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhcmFjdGVyaXphdGlvbiBhbmQgaWRlbnRpZmljYXRpb24gb2YgKENUKW4gbWljcm9zYXRlbGxpdGVzIGluIHNveWJlYW4gdXNpbmcgc2hlYXJlZCBnZW5vbWljIGxpYnJhcmllczsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTEwMy0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhcmFjdGVyaXphdGlvbiBvZiB0aGUgc295YmVhbiBnZW5vbWUgdXNpbmcgRVNUIGRlcml2ZWQgbWljcm9zYXRlbGxpdGUgbWFya2VyczsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0yNzEtODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xOC81LzMwNS8xMjI1NTI1L2RzcjAxNy5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgR2VuaWMgbWljcm9zYXRlbGxpdGUgbWFya2VycywgYWxzbyBrbm93biBhcyBmdW5jdGlvbmFsIG1hcmtlcnMsIGFyZSBwcmVmZXJyZWQgb3ZlciBhbm9ueW1vdXMgbWFya2VycyBhcyB0aGV5IHJldmVhbCB0aGUgdmFyaWF0aW9uIGluIHRyYW5zY3JpYmVkIGdlbmUiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQ0ODE4MCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGRDPmzVroq7H9s2pSMliw76c8oHSQ==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f5%2f305%2f448180" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f5%2f305%2f448180" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=17545" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/18/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/5/1/m_cover.gif?Expires=2147483647&amp;Signature=fZbX~i0F4YmMgemzi~KYwJoeRvQcK3vESx0rzJw5EUkahhhho4dt2jCm2s4eb4lcFKiJ3IkaVuW~mFnQ0N-xsBlJ-atzuGchQMdBMlEaTp9QoaKOLQdSjpbwxtx3IKeAKX9MAFcEHgiyPIddtsEKqmKyu2yRzRpJfM6fHjheCB8GOlWvmsNX-mG7yzjfRRAgK2-xl55jEo9vwfkLAfBWwh9slsk8CoXZ~kVyHDiYgXlvdg5m4LybAGcWIMBm9UYCeJTV~chzGeImI3yLbjJOa6lXjiaaTcisKEGSS9lHedM-W~GtV9CboT5Ke43Hf0WuKYesrRRMqe~T4KpwdHK-gg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/18/5">
                        <div class="volume trailing-comma">Volume 18</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2011        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6714113">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6714113">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6714115">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6714115">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6714121">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6714121">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6714131">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6714131">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6714160">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6714160">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6714169">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6714169">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6714171">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6714171">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6714173">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6714173">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6714175">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6714175">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/18/5/321/448380">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genic Microsatellite Markers in <em>Brassica rapa</em>: Development, Characterization, Mapping, and Their Utility in Other Cultivated and Wild <em>Brassica</em> Relatives
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nirala Ramchiary</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Molecular Genetics and Genomics Lab, Department of Horticulture</div>, <div class="insititution">Chungnam National University</div>, <div class="addr-line">Gung-Dong, Yuseong-Gu, Daejeon 305-764</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nirala+Ramchiary">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ramchiary N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ramchiary N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Nirala Ramchiary</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Van Dan Nguyen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Molecular Genetics and Genomics Lab, Department of Horticulture</div>, <div class="insititution">Chungnam National University</div>, <div class="addr-line">Gung-Dong, Yuseong-Gu, Daejeon 305-764</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Van+Dan+Nguyen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nguyen V">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nguyen V&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Van Dan Nguyen</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xiaonan Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Molecular Genetics and Genomics Lab, Department of Horticulture</div>, <div class="insititution">Chungnam National University</div>, <div class="addr-line">Gung-Dong, Yuseong-Gu, Daejeon 305-764</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xiaonan+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Li X&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Xiaonan Li</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chang Pyo Hong</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Life Science, and BK21 Program of Bio-Molecular Function</div>, <div class="insititution">Pohang University of Science and Technology</div>, <div class="addr-line">Pohang 790-784</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chang+Pyo+Hong">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hong C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hong C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Chang Pyo Hong</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Vignesh Dhandapani</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Molecular Genetics and Genomics Lab, Department of Horticulture</div>, <div class="insititution">Chungnam National University</div>, <div class="addr-line">Gung-Dong, Yuseong-Gu, Daejeon 305-764</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Vignesh+Dhandapani">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Dhandapani V">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Dhandapani V&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Vignesh Dhandapani</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Su Ryun Choi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Molecular Genetics and Genomics Lab, Department of Horticulture</div>, <div class="insititution">Chungnam National University</div>, <div class="addr-line">Gung-Dong, Yuseong-Gu, Daejeon 305-764</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Su+Ryun+Choi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Choi S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Choi S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Su Ryun Choi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ge Yu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Vegetable Genetics and Breeding, College of Horticulture</div>, <div class="insititution">Shenyang Agricultural University</div>, <div class="addr-line">Shenyang 110 161, Liaoning</div>, <div class="country">PR China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ge+Yu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yu G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yu G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ge Yu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Zhong Yun Piao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Vegetable Genetics and Breeding, College of Horticulture</div>, <div class="insititution">Shenyang Agricultural University</div>, <div class="addr-line">Shenyang 110 161, Liaoning</div>, <div class="country">PR China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Zhong+Yun+Piao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Piao Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Piao Z&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Zhong Yun Piao</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yong Pyo Lim</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Molecular Genetics and Genomics Lab, Department of Horticulture</div>, <div class="insititution">Chungnam National University</div>, <div class="addr-line">Gung-Dong, Yuseong-Gu, Daejeon 305-764</div>, <div class="country">Republic of Korea</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +82 42-821-5739. Fax. +82 42-821-8847. E-mail: <a href="mailto:yplim@cnu.ac.kr" target="_blank">yplim@cnu.ac.kr</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yong+Pyo+Lim">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lim Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lim Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yong Pyo Lim</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Satoshi Tabata</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 18, Issue 5, 1 October 2011, Pages 305–320, <a href='https://doi.org/10.1093/dnares/dsr017'>https://doi.org/10.1093/dnares/dsr017</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">17 July 2011</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">31 March 2011</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">02 June 2011</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="448180" href="/dnaresearch/article-pdf/18/5/305/1225525/dsr017.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Nirala Ramchiary, Van Dan Nguyen, Xiaonan Li, Chang Pyo Hong, Vignesh Dhandapani, Su Ryun Choi, Ge Yu, Zhong Yun Piao, Yong Pyo Lim;  Genic Microsatellite Markers in <em>Brassica rapa</em>: Development, Characterization, Mapping, and Their Utility in Other Cultivated and Wild <em>Brassica</em> Relatives, <em>DNA Research</em>, Volume 18, Issue 5, 1 October 2011, Pages 305–320, <a href='https://doi.org/10.1093/dnares/dsr017'>https://doi.org/10.1093/dnares/dsr017</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/448180?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/448180?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/448180?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/448180?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/448180?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/448180?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/448180?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genic%20Microsatellite%20Markers%20in%20%26lt%3Bem%26gt%3BBrassica%20rapa%26lt%3B%2Fem%26gt%3B%3A%20Development%2C%20Characterization%2C%20Mapping%2C%20and%20Their%20Utility%20in%20Other%20Cultivated%20and%20Wild%20%26lt%3Bem%26gt%3BBrassica%26lt%3B%2Fem%26gt%3B%20Relatives&amp;publicationDate=2011-07-17&amp;volumeNum=18&amp;issueNum=5&amp;author=Ramchiary%2C%20Nirala%3B%20Nguyen%2C%20Van%20Dan&amp;startPage=305&amp;endPage=320&amp;contentId=10.1093%2Fdnares%2Fdsr017&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f5%2f305%2f448180" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6714113 id="6714113" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Genic microsatellite markers, also known as functional markers, are preferred over anonymous markers as they reveal the variation in transcribed genes among individuals. In this study, we developed a total of 707 expressed sequence tag-derived simple sequence repeat markers (EST-SSRs) and used for development of a high-density integrated map using four individual mapping populations of <em>B. rapa</em>. This map contains a total of 1426 markers, consisting of 306 EST-SSRs, 153 intron polymorphic markers, 395 bacterial artificial chromosome-derived SSRs (BAC-SSRs), and 572 public SSRs and other markers covering a total distance of 1245.9 cM of the <em>B. rapa</em> genome. Analysis of allelic diversity in 24 <em>B. rapa</em> germplasm using 234 mapped EST-SSR markers showed amplification of 2 alleles by majority of EST-SSRs, although amplification of alleles ranging from 2 to 8 was found. Transferability analysis of 167 EST-SSRs in 35 species belonging to cultivated and wild <em>brassica</em> relatives showed 42.51% (<em>Sysimprium leteum</em>) to 100% (<em>B. carinata, B. juncea</em>, and <em>B. napus</em>) amplification. Our newly developed EST-SSRs and high-density linkage map based on highly transferable genic markers would facilitate the molecular mapping of quantitative trait loci and the positional cloning of specific genes, in addition to marker-assisted selection and comparative genomic studies of <em>B. rapa</em> with other related species.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Brassica rapa&quot;"><em>Brassica rapa</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;expressed sequence-derived SSRs&quot;">expressed sequence-derived SSRs</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;integrated map&quot;">integrated map</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;polymorphism information content&quot;">polymorphism information content</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="transferability">transferability</a></div>                            <h2 scrollto-destination=6714115 id="6714115" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p><em>Brassica rapa</em> (AA, 2<em>n</em> = 20) is an important diploid <em>Brassica</em> crop mainly grown for as a vegetable foodstuff, and to some extent for producing oilseed and fodder crops. Among the six cultivated <em>Brassica</em> species [the other five are the two diploids <em>Brassica nigra</em> (BB, <em>n</em> = 8) and <em>B. oleracea</em> (CC, <em>n</em> = 9) and the three amphidiploids <em>B. juncea</em> (AABB, <em>n</em> = 18), <em>B. carinata</em> (BBCC, <em>n</em> = 17), and <em>B. napus</em> (AACC, <em>n</em> = 19)], <em>B. rapa</em> has a comparatively small genome size (529 Mb), and has the second largest morphological and genetic diversity after <em>B. oleracea</em>. It is also one of the progenitor parents which contributed the A genome to the widely cultivated amphidiploid oilseed crops <em>B. juncea</em> and <em>B. napus</em>, as beautifully shown by U's triangle.<sup><a reveal-id="DSR017C1" data-open="DSR017C1" class="link link-ref link-reveal xref-bibr">1</a></sup></p><p>During the last two decades, several genetic maps with conventional anonymous molecular markers, such as amplified fragment length polymorphisms (AFLPs), restriction fragment length polymorphisms (RFLPs), and genomic simple sequence repeats (SSRs), have been constructed in <em>Brassica</em> species, including <em>B. rapa</em>.<sup><a reveal-id="DSR017C2  DSR017C7" data-open="DSR017C2  DSR017C7" class="link link-ref link-reveal xref-bibr">2–7</a></sup> These maps have been used to map, tag, and clone genetic loci [genes and/or quantitative trait loci (QTL)] that are associated with economically important traits, such as leaf traits, glucosinolates, seed coat colour, and other important agronomic traits.<sup><a reveal-id="DSR017C3" data-open="DSR017C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSR017C8  DSR017C14" data-open="DSR017C8  DSR017C14" class="link link-ref link-reveal xref-bibr">8–14</a></sup> Further, the construction of a detailed genetic map helped to study comparative genome organization, evolution, and conservation among the <em>Brassica</em> species and with <em>Arabidopsis thaliana</em>, the closest related model plant to the Brassicaceae family.<sup><a reveal-id="DSR017C4" data-open="DSR017C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSR017C6" data-open="DSR017C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Comparative mapping between <em>B. rapa</em> and <em>A. thaliana</em> was used to identify and clone candidate genes at the QTL regions for flowering time,<sup><a reveal-id="DSR017C13" data-open="DSR017C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSR017C15" data-open="DSR017C15" class="link link-ref link-reveal xref-bibr">15</a></sup> leaf hairiness,<sup><a reveal-id="DSR017C13" data-open="DSR017C13" class="link link-ref link-reveal xref-bibr">13</a></sup> and other traits.<sup><a reveal-id="DSR017C14" data-open="DSR017C14" class="link link-ref link-reveal xref-bibr">14</a></sup></p><p>However, a detailed high-density integrated genetic map combining many genetic maps developed from different populations and marker types has not been generated for <em>B. rapa</em>. The importance of high-density genetic maps in the understanding of genome organization, evolution, and the mapping and tagging of important QTL for molecular breeding and map-based cloning of economically important trait-related genes has created widespread interest for their development in many crop plants.<sup><a reveal-id="DSR017C16  DSR017C22" data-open="DSR017C16  DSR017C22" class="link link-ref link-reveal xref-bibr">16–22</a></sup> Further, previously developed conventional markers are anonymous, laborious to genotype (e.g. AFLPs and RFLPs), less reproducible (e.g. random amplification of polymorphic DNA, RAPD), require more time for development (e.g. genomic SSRs), and, more importantly, are less transferable between species. Due to these disadvantages, these conventional markers are being replaced by SSRs or single nucleotide polymorphisms (SNPs) isolated from transcribed regions [such as complementary DNA, messenger RNA, and expressed sequence tags (ESTs)]. Recent advances in plant functional genomics projects are producing enormous amounts of ESTs which have been deposited in the National Center for Biotechnology Information (NCBI) database. These sequences from transcribed genes are assembled into unique gene sequences and used to design SSRs from genes with a unique identity and position in the genome.<sup><a reveal-id="DSR017C23  DSR017C25" data-open="DSR017C23  DSR017C25" class="link link-ref link-reveal xref-bibr">23–25</a></sup> The co-dominant, multi-allelic, and high reproducibility nature besides amenable for high throughput marker analysis led to the rapid and economical expressed sequence tag-derived simple sequence repeat markers (EST-SSRs) development in several plant species since a large number of SSRs are found in coding regions.<sup><a reveal-id="DSR017C22  DSR017C24" data-open="DSR017C22  DSR017C24" class="link link-ref link-reveal xref-bibr">22–24</a>,<a reveal-id="DSR017C26  DSR017C32" data-open="DSR017C26  DSR017C32" class="link link-ref link-reveal xref-bibr">26–32</a></sup></p><p>Although the draft genome sequence of the euchromatic regions of <em>B. rapa</em> is expected to become available soon from the Multinational <em>Brassica rapa</em> Genome Sequencing Project (MBrGSP), the development and mapping of more uniformly spaced high-density genic markers, such as unigene-derived microsatellites (UGMS) and intron polymorphic (IP) markers along with bacterial artificial chromosome (BAC)-derived SSRs, would facilitate the mapping of important traits and their utilization in molecular breeding. In addition, a high-density map of anonymous and genic markers would help in the correct alignment of gene-rich euchromatic and repetitive heterochromatic sequences in the <em>B. rapa</em> genome because the soon-to-be released draft <em>B. rapa</em> genome sequence covers only 384 Mb of the 529 Mb <em>Brassica</em> A genome (personal communication from MBrGSP).</p><p>In <em>B. rapa</em>, Parida <em>et al</em>.<sup><a reveal-id="DSR017C24" data-open="DSR017C24" class="link link-ref link-reveal xref-bibr">24</a></sup> recently developed 347 unigene-derived SSR markers, suggesting that there are many more unidentified genic SSRs that would allow for the complete coverage of the <em>B. rapa</em> genome. This would help uniformly select genic SSR markers covering the total genome and facilitate the mapping, tagging, and identification of economically important genetic loci. Further, uniformly distributed markers would be useful for comparative mapping and evolutionary studies with other closely related <em>Brassica</em> species. Hence, the objectives of this study were to develop more EST-SSR markers, map the newly developed EST-SSRs along with the previously mapped BAC-derived SSRs, IP markers, and publicly available SSR markers into the <em>Brassica rapa</em> genome to construct a high-density gene-based updated integrated map, and transferability analysis of the mapped EST-SSRs markers in other <em>Brassica</em> relatives so that these markers could be used for comparative mapping between them.</p>                            <h2 scrollto-destination=6714121 id="6714121" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=6714122 id="6714122" class="section-title" data-legacy-id=s2a>2.1. Plant materials</h3>
<p>We used four <em>B. rapa</em> mapping populations to develop an integrated linkage map: CKDH, CRF<sub>2</sub>, PF<sub>2</sub>, and CSKF<sub>2</sub>. The CKDH population consisted of 78 double haploid lines derived from a cross between ‘Chiifu-401’ and ‘Kenshin’, which were earlier used to construct a reference genetic linkage map of <em>B. rapa</em>.<sup><a reveal-id="DSR017C5" data-open="DSR017C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSR017C7" data-open="DSR017C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSR017C33" data-open="DSR017C33" class="link link-ref link-reveal xref-bibr">33</a></sup> The CRF<sub>2</sub> population consisted of 190 F<sub>2</sub> individuals derived from crossing the ‘Chiifu-401’ and ‘Rapid cycling <em>B. rapa</em> (RCBr)’ parental lines, and was previously used by Li <em>et al</em>.<sup><a reveal-id="DSR017C33" data-open="DSR017C33" class="link link-ref link-reveal xref-bibr">33</a></sup> to construct a linkage map. The CSKF<sub>2</sub> population consisted of 94 individual lines derived from crossing between the clubroot resistance cultivar ‘CR Shinki’ and a susceptible cultivar ‘94SK’. The fourth population, PF<sub>2</sub>, consisted of 144 F<sub>2</sub> populations that were derived from crossing the diverse Chinese cabbage inbred lines ‘501’ with a large head and ‘601’ with a small head. In order to detect the polymorphic level and allelic frequencies of the newly mapped EST-SSRs, we selected 24 different <em>B. rapa</em> cultivars belonging to different sub-species and morphophytes which included Chinese cabbage, pak choi, and oil yielding types from the Korea <em>Brassica</em> Genome Resource Bank (Table <a reveal-id="DSR017TB1" data-open="DSR017TB1" class="link link-reveal link-table xref-fig">1</a>, serial number 1–24). Different <em>Brassica</em> species and wild relatives collected from Centre for Genetic Manipulation of Crop Plants, Delhi University South Campus, India; Korea <em>Brassica</em> Genome Research Bank, Korea; and Leibniz institute of Plant Genetics and Crop Plant Research, Gatersleben, Germany were used for marker transferability analysis (Table <a reveal-id="DSR017TB1" data-open="DSR017TB1" class="link link-reveal link-table xref-fig">1</a>). </p>                        <a id="6714124" scrollto-destination="6714124"></a>
<div content-id="DSR017TB1" class="table-modal"><div class="table-wrap"><div id="DSR017TB1" class="table-wrap-title" data-id="DSR017TB1"><div class="label -label">Table 1.</div><div class="caption"><p>List of different <em>Brassica</em> species and wild relatives used for allelic diversity and transferability analysis</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>SL No </th><th>Name/accession number </th><th>Species </th><th>Source </th><th>SL no. </th><th>Name/accession number </th><th>Species </th><th>Source </th></tr></thead><tbody><tr><td>1 </td><td>Chiifu-401 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>36 </td><td>HC-17 </td><td><em>B. carinata</em> </td><td>CGMCP </td></tr><tr><td>2 </td><td>Kenshin </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>37 </td><td>Sangam </td><td><em>B. nigra</em> </td><td>CGMCP </td></tr><tr><td>3 </td><td>94sk </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>38 </td><td>94029 </td><td><em>B. nigra</em> </td><td>CGMCP </td></tr><tr><td>4 </td><td>24020 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>39 </td><td>28407 </td><td><em>B. tourniforti</em> </td><td>IPK </td></tr><tr><td>5 </td><td>26021 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>40 </td><td>BRA 2850 </td><td><em>B. balearica</em> </td><td>IPK </td></tr><tr><td>6 </td><td>26022 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>41 </td><td>BRA 1877 </td><td><em>B. barreilieri</em> </td><td>IPK </td></tr><tr><td>7 </td><td>26028 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>42 </td><td>BRA 2922 </td><td><em>B. biovisina</em> </td><td>IPK </td></tr><tr><td>8 </td><td>28053 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>43 </td><td>K 9825 </td><td><em>B. bourgeai</em> </td><td>IPK </td></tr><tr><td>9 </td><td>28055 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>44 </td><td>K 6631 </td><td><em>B. cretica</em> </td><td>IPK </td></tr><tr><td>10 </td><td>cnu-28020 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>45 </td><td>BRA 2919 </td><td><em>B. desnottesii</em> </td><td>IPK </td></tr><tr><td>11 </td><td>cnu-28065 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>46 </td><td>K 9402 </td><td><em>B. depranensis</em> </td><td>IPK </td></tr><tr><td>12 </td><td>cnu-28072 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>47 </td><td>BRA 1039 </td><td><em>B. frutoiculosa</em> </td><td>IPK </td></tr><tr><td>13 </td><td>25082 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>48 </td><td>BRA 1810 </td><td><em>B. frutoiculosa</em> </td><td>IPK </td></tr><tr><td>14 </td><td>25083 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>49 </td><td>BRA 1169 </td><td><em>B. gravinae</em> </td><td>IPK </td></tr><tr><td>15 </td><td>25084 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>50 </td><td>BRA 2856 </td><td><em>B. incana</em> </td><td>IPK </td></tr><tr><td>16 </td><td>25095 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>51 </td><td>K 5997 </td><td><em>B. insularis</em> </td><td>IPK </td></tr><tr><td>17 </td><td>25103 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>52 </td><td>K 7635 </td><td><em>B. macrocarpa</em> </td><td>IPK </td></tr><tr><td>18 </td><td>25106 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>53 </td><td>K9242 </td><td><em>B. maurorum</em> </td><td>IPK </td></tr><tr><td>19 </td><td>25110 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>54 </td><td>BRA 1645 </td><td><em>B. repanda</em> </td><td>IPK </td></tr><tr><td>20 </td><td>26109 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>55 </td><td>K 6877 </td><td><em>B. rupestris</em> </td><td>IPK </td></tr><tr><td>21 </td><td>27081 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>56 </td><td>K 8823 </td><td><em>B. spinescens</em> </td><td>IPK </td></tr><tr><td>22 </td><td>Tetralocular </td><td><em>B. rapa</em> ssp <em>oleifera</em> </td><td>CGMCP </td><td>57 </td><td>BRA 1896 </td><td><em>B. villosa</em> </td><td>IPK </td></tr><tr><td>23 </td><td>YSPB </td><td><em>B. rapa</em> ssp <em>oleifera</em> </td><td>CGMCP </td><td>58 </td><td>LET </td><td><em>A. thaliana  LE ecootype</em> </td><td>KBGRB </td></tr><tr><td>24 </td><td>Candle </td><td><em>B. rapa</em> ssp <em>oleifera</em> </td><td>CGMCP </td><td>59 </td><td>ETR </td><td><em>A. thaliana Col ecotype</em> </td><td>KBGRB </td></tr><tr><td>25 </td><td>Pusa kalyanai </td><td><em>B. rapa</em> ssp <em>oleifera</em> </td><td>CGMCP </td><td>60 </td><td>28697 </td><td><em>Camelina sativa</em> </td><td>KBGRB </td></tr><tr><td>26 </td><td>RCBr </td><td><em>B. rapa</em> (rapid cycling) </td><td>KBGRB </td><td>61 </td><td>26165 </td><td><em>Herba  cichori</em> </td><td>KBGRB </td></tr><tr><td>27 </td><td>CNU 28003 </td><td><em>B</em>. <em>oleracea</em> ssp <em>capitata</em> </td><td>KBGRB </td><td>62 </td><td>2659 </td><td><em>Diplotaxis muralis</em> </td><td>KBGRB </td></tr><tr><td>28 </td><td>CNU 28004 </td><td><em>B</em>. <em>oleracea</em> ssp <em>capitata</em> </td><td>KBGRB </td><td>63 </td><td>28614 </td><td><em>Eruca  staiva</em> </td><td>KBGRB </td></tr><tr><td>29 </td><td>Varuna </td><td><em>B. juncea</em> </td><td>CGMCP </td><td>64 </td><td>28699 </td><td><em>Hesperis matronalis</em> </td><td>KBGRB </td></tr><tr><td>30 </td><td>Heera </td><td><em>B. juncea</em> </td><td>CGMCP </td><td>65 </td><td>26056 </td><td><em>Moricandia arvensis</em> </td><td>KBGRB </td></tr><tr><td>31 </td><td>Donskaja </td><td><em>B. juncea</em> </td><td>CGMCP </td><td>66 </td><td>28672 </td><td><em>Sinapis alba</em> </td><td>KBGRB </td></tr><tr><td>32 </td><td>TM 4 </td><td><em>B. juncea</em> </td><td>CGMCP </td><td>67 </td><td>28597 </td><td><em>Raphanus staivus</em> </td><td>KBGRB </td></tr><tr><td>33 </td><td>Tapidor </td><td><em>B. napus</em> </td><td>China </td><td>68 </td><td>26080 </td><td><em>Sisymprium leteum</em> </td><td>KBGRB </td></tr><tr><td>34 </td><td>Ningyou-7 </td><td><em>B. napus</em> </td><td>China </td><td>69 </td><td>26093 </td><td><em>Lepidium apetalum</em> </td><td>KBGRB </td></tr><tr><td>35 </td><td>Car6 </td><td><em>B. carinata</em> </td><td>CGMCP </td><td>70 </td><td> </td><td> </td><td> </td></tr></tbody></table></div><div content-id="DSR017TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>SL No </th><th>Name/accession number </th><th>Species </th><th>Source </th><th>SL no. </th><th>Name/accession number </th><th>Species </th><th>Source </th></tr></thead><tbody><tr><td>1 </td><td>Chiifu-401 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>36 </td><td>HC-17 </td><td><em>B. carinata</em> </td><td>CGMCP </td></tr><tr><td>2 </td><td>Kenshin </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>37 </td><td>Sangam </td><td><em>B. nigra</em> </td><td>CGMCP </td></tr><tr><td>3 </td><td>94sk </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>38 </td><td>94029 </td><td><em>B. nigra</em> </td><td>CGMCP </td></tr><tr><td>4 </td><td>24020 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>39 </td><td>28407 </td><td><em>B. tourniforti</em> </td><td>IPK </td></tr><tr><td>5 </td><td>26021 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>40 </td><td>BRA 2850 </td><td><em>B. balearica</em> </td><td>IPK </td></tr><tr><td>6 </td><td>26022 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>41 </td><td>BRA 1877 </td><td><em>B. barreilieri</em> </td><td>IPK </td></tr><tr><td>7 </td><td>26028 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>42 </td><td>BRA 2922 </td><td><em>B. biovisina</em> </td><td>IPK </td></tr><tr><td>8 </td><td>28053 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>43 </td><td>K 9825 </td><td><em>B. bourgeai</em> </td><td>IPK </td></tr><tr><td>9 </td><td>28055 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>44 </td><td>K 6631 </td><td><em>B. cretica</em> </td><td>IPK </td></tr><tr><td>10 </td><td>cnu-28020 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>45 </td><td>BRA 2919 </td><td><em>B. desnottesii</em> </td><td>IPK </td></tr><tr><td>11 </td><td>cnu-28065 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>46 </td><td>K 9402 </td><td><em>B. depranensis</em> </td><td>IPK </td></tr><tr><td>12 </td><td>cnu-28072 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>47 </td><td>BRA 1039 </td><td><em>B. frutoiculosa</em> </td><td>IPK </td></tr><tr><td>13 </td><td>25082 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>48 </td><td>BRA 1810 </td><td><em>B. frutoiculosa</em> </td><td>IPK </td></tr><tr><td>14 </td><td>25083 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>49 </td><td>BRA 1169 </td><td><em>B. gravinae</em> </td><td>IPK </td></tr><tr><td>15 </td><td>25084 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>50 </td><td>BRA 2856 </td><td><em>B. incana</em> </td><td>IPK </td></tr><tr><td>16 </td><td>25095 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>51 </td><td>K 5997 </td><td><em>B. insularis</em> </td><td>IPK </td></tr><tr><td>17 </td><td>25103 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>52 </td><td>K 7635 </td><td><em>B. macrocarpa</em> </td><td>IPK </td></tr><tr><td>18 </td><td>25106 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>53 </td><td>K9242 </td><td><em>B. maurorum</em> </td><td>IPK </td></tr><tr><td>19 </td><td>25110 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>54 </td><td>BRA 1645 </td><td><em>B. repanda</em> </td><td>IPK </td></tr><tr><td>20 </td><td>26109 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>55 </td><td>K 6877 </td><td><em>B. rupestris</em> </td><td>IPK </td></tr><tr><td>21 </td><td>27081 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>56 </td><td>K 8823 </td><td><em>B. spinescens</em> </td><td>IPK </td></tr><tr><td>22 </td><td>Tetralocular </td><td><em>B. rapa</em> ssp <em>oleifera</em> </td><td>CGMCP </td><td>57 </td><td>BRA 1896 </td><td><em>B. villosa</em> </td><td>IPK </td></tr><tr><td>23 </td><td>YSPB </td><td><em>B. rapa</em> ssp <em>oleifera</em> </td><td>CGMCP </td><td>58 </td><td>LET </td><td><em>A. thaliana  LE ecootype</em> </td><td>KBGRB </td></tr><tr><td>24 </td><td>Candle </td><td><em>B. rapa</em> ssp <em>oleifera</em> </td><td>CGMCP </td><td>59 </td><td>ETR </td><td><em>A. thaliana Col ecotype</em> </td><td>KBGRB </td></tr><tr><td>25 </td><td>Pusa kalyanai </td><td><em>B. rapa</em> ssp <em>oleifera</em> </td><td>CGMCP </td><td>60 </td><td>28697 </td><td><em>Camelina sativa</em> </td><td>KBGRB </td></tr><tr><td>26 </td><td>RCBr </td><td><em>B. rapa</em> (rapid cycling) </td><td>KBGRB </td><td>61 </td><td>26165 </td><td><em>Herba  cichori</em> </td><td>KBGRB </td></tr><tr><td>27 </td><td>CNU 28003 </td><td><em>B</em>. <em>oleracea</em> ssp <em>capitata</em> </td><td>KBGRB </td><td>62 </td><td>2659 </td><td><em>Diplotaxis muralis</em> </td><td>KBGRB </td></tr><tr><td>28 </td><td>CNU 28004 </td><td><em>B</em>. <em>oleracea</em> ssp <em>capitata</em> </td><td>KBGRB </td><td>63 </td><td>28614 </td><td><em>Eruca  staiva</em> </td><td>KBGRB </td></tr><tr><td>29 </td><td>Varuna </td><td><em>B. juncea</em> </td><td>CGMCP </td><td>64 </td><td>28699 </td><td><em>Hesperis matronalis</em> </td><td>KBGRB </td></tr><tr><td>30 </td><td>Heera </td><td><em>B. juncea</em> </td><td>CGMCP </td><td>65 </td><td>26056 </td><td><em>Moricandia arvensis</em> </td><td>KBGRB </td></tr><tr><td>31 </td><td>Donskaja </td><td><em>B. juncea</em> </td><td>CGMCP </td><td>66 </td><td>28672 </td><td><em>Sinapis alba</em> </td><td>KBGRB </td></tr><tr><td>32 </td><td>TM 4 </td><td><em>B. juncea</em> </td><td>CGMCP </td><td>67 </td><td>28597 </td><td><em>Raphanus staivus</em> </td><td>KBGRB </td></tr><tr><td>33 </td><td>Tapidor </td><td><em>B. napus</em> </td><td>China </td><td>68 </td><td>26080 </td><td><em>Sisymprium leteum</em> </td><td>KBGRB </td></tr><tr><td>34 </td><td>Ningyou-7 </td><td><em>B. napus</em> </td><td>China </td><td>69 </td><td>26093 </td><td><em>Lepidium apetalum</em> </td><td>KBGRB </td></tr><tr><td>35 </td><td>Car6 </td><td><em>B. carinata</em> </td><td>CGMCP </td><td>70 </td><td> </td><td> </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>KBGRB—Korea <em>Brassica</em> Genome Resource Bank, Daejeon, Korea; CGMCP—Centre for Genetic Manipulation of Crop Plants, Delhi, India, IPK—Leibniz Institute of Plant Genetics and Crop Plant Research, Gatersleben, Germany; LET<em>— A. thaliana  landsberg erecta ecotype</em>; ETR— <em>A. thaliana  columbia ecotype</em>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6714124" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR017TB1" class="table-wrap-title" data-id="DSR017TB1"><div class="label -label">Table 1.</div><div class="caption"><p>List of different <em>Brassica</em> species and wild relatives used for allelic diversity and transferability analysis</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>SL No </th><th>Name/accession number </th><th>Species </th><th>Source </th><th>SL no. </th><th>Name/accession number </th><th>Species </th><th>Source </th></tr></thead><tbody><tr><td>1 </td><td>Chiifu-401 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>36 </td><td>HC-17 </td><td><em>B. carinata</em> </td><td>CGMCP </td></tr><tr><td>2 </td><td>Kenshin </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>37 </td><td>Sangam </td><td><em>B. nigra</em> </td><td>CGMCP </td></tr><tr><td>3 </td><td>94sk </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>38 </td><td>94029 </td><td><em>B. nigra</em> </td><td>CGMCP </td></tr><tr><td>4 </td><td>24020 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>39 </td><td>28407 </td><td><em>B. tourniforti</em> </td><td>IPK </td></tr><tr><td>5 </td><td>26021 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>40 </td><td>BRA 2850 </td><td><em>B. balearica</em> </td><td>IPK </td></tr><tr><td>6 </td><td>26022 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>41 </td><td>BRA 1877 </td><td><em>B. barreilieri</em> </td><td>IPK </td></tr><tr><td>7 </td><td>26028 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>42 </td><td>BRA 2922 </td><td><em>B. biovisina</em> </td><td>IPK </td></tr><tr><td>8 </td><td>28053 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>43 </td><td>K 9825 </td><td><em>B. bourgeai</em> </td><td>IPK </td></tr><tr><td>9 </td><td>28055 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>44 </td><td>K 6631 </td><td><em>B. cretica</em> </td><td>IPK </td></tr><tr><td>10 </td><td>cnu-28020 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>45 </td><td>BRA 2919 </td><td><em>B. desnottesii</em> </td><td>IPK </td></tr><tr><td>11 </td><td>cnu-28065 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>46 </td><td>K 9402 </td><td><em>B. depranensis</em> </td><td>IPK </td></tr><tr><td>12 </td><td>cnu-28072 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>47 </td><td>BRA 1039 </td><td><em>B. frutoiculosa</em> </td><td>IPK </td></tr><tr><td>13 </td><td>25082 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>48 </td><td>BRA 1810 </td><td><em>B. frutoiculosa</em> </td><td>IPK </td></tr><tr><td>14 </td><td>25083 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>49 </td><td>BRA 1169 </td><td><em>B. gravinae</em> </td><td>IPK </td></tr><tr><td>15 </td><td>25084 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>50 </td><td>BRA 2856 </td><td><em>B. incana</em> </td><td>IPK </td></tr><tr><td>16 </td><td>25095 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>51 </td><td>K 5997 </td><td><em>B. insularis</em> </td><td>IPK </td></tr><tr><td>17 </td><td>25103 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>52 </td><td>K 7635 </td><td><em>B. macrocarpa</em> </td><td>IPK </td></tr><tr><td>18 </td><td>25106 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>53 </td><td>K9242 </td><td><em>B. maurorum</em> </td><td>IPK </td></tr><tr><td>19 </td><td>25110 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>54 </td><td>BRA 1645 </td><td><em>B. repanda</em> </td><td>IPK </td></tr><tr><td>20 </td><td>26109 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>55 </td><td>K 6877 </td><td><em>B. rupestris</em> </td><td>IPK </td></tr><tr><td>21 </td><td>27081 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>56 </td><td>K 8823 </td><td><em>B. spinescens</em> </td><td>IPK </td></tr><tr><td>22 </td><td>Tetralocular </td><td><em>B. rapa</em> ssp <em>oleifera</em> </td><td>CGMCP </td><td>57 </td><td>BRA 1896 </td><td><em>B. villosa</em> </td><td>IPK </td></tr><tr><td>23 </td><td>YSPB </td><td><em>B. rapa</em> ssp <em>oleifera</em> </td><td>CGMCP </td><td>58 </td><td>LET </td><td><em>A. thaliana  LE ecootype</em> </td><td>KBGRB </td></tr><tr><td>24 </td><td>Candle </td><td><em>B. rapa</em> ssp <em>oleifera</em> </td><td>CGMCP </td><td>59 </td><td>ETR </td><td><em>A. thaliana Col ecotype</em> </td><td>KBGRB </td></tr><tr><td>25 </td><td>Pusa kalyanai </td><td><em>B. rapa</em> ssp <em>oleifera</em> </td><td>CGMCP </td><td>60 </td><td>28697 </td><td><em>Camelina sativa</em> </td><td>KBGRB </td></tr><tr><td>26 </td><td>RCBr </td><td><em>B. rapa</em> (rapid cycling) </td><td>KBGRB </td><td>61 </td><td>26165 </td><td><em>Herba  cichori</em> </td><td>KBGRB </td></tr><tr><td>27 </td><td>CNU 28003 </td><td><em>B</em>. <em>oleracea</em> ssp <em>capitata</em> </td><td>KBGRB </td><td>62 </td><td>2659 </td><td><em>Diplotaxis muralis</em> </td><td>KBGRB </td></tr><tr><td>28 </td><td>CNU 28004 </td><td><em>B</em>. <em>oleracea</em> ssp <em>capitata</em> </td><td>KBGRB </td><td>63 </td><td>28614 </td><td><em>Eruca  staiva</em> </td><td>KBGRB </td></tr><tr><td>29 </td><td>Varuna </td><td><em>B. juncea</em> </td><td>CGMCP </td><td>64 </td><td>28699 </td><td><em>Hesperis matronalis</em> </td><td>KBGRB </td></tr><tr><td>30 </td><td>Heera </td><td><em>B. juncea</em> </td><td>CGMCP </td><td>65 </td><td>26056 </td><td><em>Moricandia arvensis</em> </td><td>KBGRB </td></tr><tr><td>31 </td><td>Donskaja </td><td><em>B. juncea</em> </td><td>CGMCP </td><td>66 </td><td>28672 </td><td><em>Sinapis alba</em> </td><td>KBGRB </td></tr><tr><td>32 </td><td>TM 4 </td><td><em>B. juncea</em> </td><td>CGMCP </td><td>67 </td><td>28597 </td><td><em>Raphanus staivus</em> </td><td>KBGRB </td></tr><tr><td>33 </td><td>Tapidor </td><td><em>B. napus</em> </td><td>China </td><td>68 </td><td>26080 </td><td><em>Sisymprium leteum</em> </td><td>KBGRB </td></tr><tr><td>34 </td><td>Ningyou-7 </td><td><em>B. napus</em> </td><td>China </td><td>69 </td><td>26093 </td><td><em>Lepidium apetalum</em> </td><td>KBGRB </td></tr><tr><td>35 </td><td>Car6 </td><td><em>B. carinata</em> </td><td>CGMCP </td><td>70 </td><td> </td><td> </td><td> </td></tr></tbody></table></div><div content-id="DSR017TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>SL No </th><th>Name/accession number </th><th>Species </th><th>Source </th><th>SL no. </th><th>Name/accession number </th><th>Species </th><th>Source </th></tr></thead><tbody><tr><td>1 </td><td>Chiifu-401 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>36 </td><td>HC-17 </td><td><em>B. carinata</em> </td><td>CGMCP </td></tr><tr><td>2 </td><td>Kenshin </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>37 </td><td>Sangam </td><td><em>B. nigra</em> </td><td>CGMCP </td></tr><tr><td>3 </td><td>94sk </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>38 </td><td>94029 </td><td><em>B. nigra</em> </td><td>CGMCP </td></tr><tr><td>4 </td><td>24020 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>39 </td><td>28407 </td><td><em>B. tourniforti</em> </td><td>IPK </td></tr><tr><td>5 </td><td>26021 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>40 </td><td>BRA 2850 </td><td><em>B. balearica</em> </td><td>IPK </td></tr><tr><td>6 </td><td>26022 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>41 </td><td>BRA 1877 </td><td><em>B. barreilieri</em> </td><td>IPK </td></tr><tr><td>7 </td><td>26028 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>42 </td><td>BRA 2922 </td><td><em>B. biovisina</em> </td><td>IPK </td></tr><tr><td>8 </td><td>28053 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>43 </td><td>K 9825 </td><td><em>B. bourgeai</em> </td><td>IPK </td></tr><tr><td>9 </td><td>28055 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>44 </td><td>K 6631 </td><td><em>B. cretica</em> </td><td>IPK </td></tr><tr><td>10 </td><td>cnu-28020 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>45 </td><td>BRA 2919 </td><td><em>B. desnottesii</em> </td><td>IPK </td></tr><tr><td>11 </td><td>cnu-28065 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>46 </td><td>K 9402 </td><td><em>B. depranensis</em> </td><td>IPK </td></tr><tr><td>12 </td><td>cnu-28072 </td><td><em>B. rapa</em> ssp <em>pekinensis</em> </td><td>KBGRB </td><td>47 </td><td>BRA 1039 </td><td><em>B. frutoiculosa</em> </td><td>IPK </td></tr><tr><td>13 </td><td>25082 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>48 </td><td>BRA 1810 </td><td><em>B. frutoiculosa</em> </td><td>IPK </td></tr><tr><td>14 </td><td>25083 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>49 </td><td>BRA 1169 </td><td><em>B. gravinae</em> </td><td>IPK </td></tr><tr><td>15 </td><td>25084 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>50 </td><td>BRA 2856 </td><td><em>B. incana</em> </td><td>IPK </td></tr><tr><td>16 </td><td>25095 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>51 </td><td>K 5997 </td><td><em>B. insularis</em> </td><td>IPK </td></tr><tr><td>17 </td><td>25103 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>52 </td><td>K 7635 </td><td><em>B. macrocarpa</em> </td><td>IPK </td></tr><tr><td>18 </td><td>25106 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>53 </td><td>K9242 </td><td><em>B. maurorum</em> </td><td>IPK </td></tr><tr><td>19 </td><td>25110 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>54 </td><td>BRA 1645 </td><td><em>B. repanda</em> </td><td>IPK </td></tr><tr><td>20 </td><td>26109 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>55 </td><td>K 6877 </td><td><em>B. rupestris</em> </td><td>IPK </td></tr><tr><td>21 </td><td>27081 </td><td><em>B. rapa</em> ssp <em>chinensis</em> </td><td>KBGRB </td><td>56 </td><td>K 8823 </td><td><em>B. spinescens</em> </td><td>IPK </td></tr><tr><td>22 </td><td>Tetralocular </td><td><em>B. rapa</em> ssp <em>oleifera</em> </td><td>CGMCP </td><td>57 </td><td>BRA 1896 </td><td><em>B. villosa</em> </td><td>IPK </td></tr><tr><td>23 </td><td>YSPB </td><td><em>B. rapa</em> ssp <em>oleifera</em> </td><td>CGMCP </td><td>58 </td><td>LET </td><td><em>A. thaliana  LE ecootype</em> </td><td>KBGRB </td></tr><tr><td>24 </td><td>Candle </td><td><em>B. rapa</em> ssp <em>oleifera</em> </td><td>CGMCP </td><td>59 </td><td>ETR </td><td><em>A. thaliana Col ecotype</em> </td><td>KBGRB </td></tr><tr><td>25 </td><td>Pusa kalyanai </td><td><em>B. rapa</em> ssp <em>oleifera</em> </td><td>CGMCP </td><td>60 </td><td>28697 </td><td><em>Camelina sativa</em> </td><td>KBGRB </td></tr><tr><td>26 </td><td>RCBr </td><td><em>B. rapa</em> (rapid cycling) </td><td>KBGRB </td><td>61 </td><td>26165 </td><td><em>Herba  cichori</em> </td><td>KBGRB </td></tr><tr><td>27 </td><td>CNU 28003 </td><td><em>B</em>. <em>oleracea</em> ssp <em>capitata</em> </td><td>KBGRB </td><td>62 </td><td>2659 </td><td><em>Diplotaxis muralis</em> </td><td>KBGRB </td></tr><tr><td>28 </td><td>CNU 28004 </td><td><em>B</em>. <em>oleracea</em> ssp <em>capitata</em> </td><td>KBGRB </td><td>63 </td><td>28614 </td><td><em>Eruca  staiva</em> </td><td>KBGRB </td></tr><tr><td>29 </td><td>Varuna </td><td><em>B. juncea</em> </td><td>CGMCP </td><td>64 </td><td>28699 </td><td><em>Hesperis matronalis</em> </td><td>KBGRB </td></tr><tr><td>30 </td><td>Heera </td><td><em>B. juncea</em> </td><td>CGMCP </td><td>65 </td><td>26056 </td><td><em>Moricandia arvensis</em> </td><td>KBGRB </td></tr><tr><td>31 </td><td>Donskaja </td><td><em>B. juncea</em> </td><td>CGMCP </td><td>66 </td><td>28672 </td><td><em>Sinapis alba</em> </td><td>KBGRB </td></tr><tr><td>32 </td><td>TM 4 </td><td><em>B. juncea</em> </td><td>CGMCP </td><td>67 </td><td>28597 </td><td><em>Raphanus staivus</em> </td><td>KBGRB </td></tr><tr><td>33 </td><td>Tapidor </td><td><em>B. napus</em> </td><td>China </td><td>68 </td><td>26080 </td><td><em>Sisymprium leteum</em> </td><td>KBGRB </td></tr><tr><td>34 </td><td>Ningyou-7 </td><td><em>B. napus</em> </td><td>China </td><td>69 </td><td>26093 </td><td><em>Lepidium apetalum</em> </td><td>KBGRB </td></tr><tr><td>35 </td><td>Car6 </td><td><em>B. carinata</em> </td><td>CGMCP </td><td>70 </td><td> </td><td> </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>KBGRB—Korea <em>Brassica</em> Genome Resource Bank, Daejeon, Korea; CGMCP—Centre for Genetic Manipulation of Crop Plants, Delhi, India, IPK—Leibniz Institute of Plant Genetics and Crop Plant Research, Gatersleben, Germany; LET<em>— A. thaliana  landsberg erecta ecotype</em>; ETR— <em>A. thaliana  columbia ecotype</em>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6714124" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=6714125 id="6714125" class="section-title" data-legacy-id=s2b>2.2. Searching for SSR-containing sequences and primer design</h3>
<p>We downloaded a total of 182 703 <em>B. rapa</em> EST sequences from NCBI database (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov">http://www.ncbi.nlm.nih.gov</a>) and assembled using CAP3<sup><a reveal-id="DSR017C34" data-open="DSR017C34" class="link link-ref link-reveal xref-bibr">34</a></sup> to identify unigenes. The unigene sequences (singlets and contigs) were then searched for the presence of SSR motifs using the MIcro SAtellite identification tool (MISA) available at <a class="link link-uri" href="http://pgrc.ipk-gatersleben.de/misa/misa.html">http://pgrc.ipk-gatersleben.de/misa/misa.html</a> and sputnik software following the criteria described earlier by Hong <em>et al</em>.<sup><a reveal-id="DSR017C35" data-open="DSR017C35" class="link link-ref link-reveal xref-bibr">35</a></sup> The contig or singleton sequences were used to design primers flanking the putative SSRs using Primer 3.<sup><a reveal-id="DSR017C36" data-open="DSR017C36" class="link link-ref link-reveal xref-bibr">36</a></sup> The primer designing conditions were: 58–60°C melting temperature with a difference of only 1°C between each forward and reverse primer, 40–60% GC content, and 19–21 bp primer length and an estimated amplicon size of 150–400 bp. ORF Finder<sup><a reveal-id="DSR017C37" data-open="DSR017C37" class="link link-ref link-reveal xref-bibr">37</a></sup> (<a class="link link-uri" href="http://bioinformatics.org/sms/orf_find.html">http://bioinformatics.org/sms/orf_find.html</a>) and UTRScan (<a class="link link-uri" href="http://utrdb.ba.itb.cnr.it/tool/utrscan">http://utrdb.ba.itb.cnr.it/tool/utrscan</a>) were used to find the location of repeat motifs in coding region and untranslated regions (5′UTR and 3′TR) or in open reading frames.</p>                            <h3 scrollto-destination=6714127 id="6714127" class="section-title" data-legacy-id=s2c>2.3. DNA extraction, marker analysis, and cloning of PCR amplicon</h3>
<p>DNA was extracted from young expanded leaf samples collected from greenhouse-grown plants using an RBC Genomic DNA Extraction Kit (Real Biotech Corporation, Taipei, Taiwan). A total of 707 newly developed UGMS markers (prefixed by ACMP, hereafter referred to as EST-SSRs, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>) were used for a polymorphism survey between the parental lines Chiifu-401, Kenshin, and RCBr. A total of 999 BAC-derived SSRs, which were previously developed in our laboratory (designated by ‘cnu’, ‘nia’, and ‘BRPGM’),<sup><a reveal-id="DSR017C7" data-open="DSR017C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSR017C33" data-open="DSR017C33" class="link link-ref link-reveal xref-bibr">33</a></sup> 272 IP markers developed and mapped by Panjabi <em>et al</em>.,<sup><a reveal-id="DSR017C6" data-open="DSR017C6" class="link link-ref link-reveal xref-bibr">6</a></sup> and 707 new EST-SSRs were screened between CR Shinki and 94SK. In another experiment, a total of 450 newly developed unigene-derived SSR markers (prefixed by ‘sau_um’, unpublished), which were developed in Shenyang Agricultural University, and 651 public SSR markers<sup><a reveal-id="DSR017C5" data-open="DSR017C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSR017C38  DSR017C47" data-open="DSR017C38  DSR017C47" class="link link-ref link-reveal xref-bibr">38–47</a></sup> were screened for polymorphisms between the parental lines ‘501’ and ‘601’ of the PF<sub>2</sub> population. The PCR reaction conditions used by Li <em>et al</em>.<sup><a reveal-id="DSR017C33" data-open="DSR017C33" class="link link-ref link-reveal xref-bibr">33</a></sup> were followed for the BAC-derived SSRs, IP markers, and the newly developed EST-SSRs. The PCR conditions for the EST-derived SSR markers were as follows: 5 min at 95°C; 36 cycles of 45 s at 95°C, 45 s at 55°C, and 45 s at 72°C; with a final step of 10 min at 72°C. PCR products were resolved in 8% polyacrylamide gel electrophoresis as described by Kim <em>et al</em>.<sup><a reveal-id="DSR017C7" data-open="DSR017C7" class="link link-ref link-reveal xref-bibr">7</a></sup> The PCR amplicon from different species was cloned in pGEM-T Easy cloning vector (<a class="link link-uri" href="http://www.promega.com">www.promega.com</a>) according to the manufacturer's instruction and at least one clone were sequenced two times from each <em>Brassica</em> species.</p>                            <h3 scrollto-destination=6714129 id="6714129" class="section-title" data-legacy-id=s2d>2.4. Construction of linkage maps and diversity analysis</h3>
<p>The four individual maps and the integrated genetic map were constructed with Joinmap version 4.0<sup><a reveal-id="DSR017C48" data-open="DSR017C48" class="link link-ref link-reveal xref-bibr">48</a>,<a reveal-id="DSR017C49" data-open="DSR017C49" class="link link-ref link-reveal xref-bibr">49</a></sup> using the same parameters as described by Li <em>et al</em>.<sup><a reveal-id="DSR017C33" data-open="DSR017C33" class="link link-ref link-reveal xref-bibr">33</a></sup> The Kosambi mapping function was used to calculate map distances.<sup><a reveal-id="DSR017C50" data-open="DSR017C50" class="link link-ref link-reveal xref-bibr">50</a></sup> Logarithm of the odds (LOD) scores of 4.0–8.0 was used to group markers. A recombination frequency &lt;0.4 and a LOD score &gt;1.0 were used to arrange the marker order. Common markers were used as ‘bridge’ markers to integrate the four maps using the function ‘Combine the Groups for Map Integration’. We used two approaches to integrate the four maps: first, we used the CKDH map as the reference map and sequentially integrated the other three maps in the order CRF<sub>2</sub>, PF<sub>2</sub>, and CSKF<sub>2</sub>; and secondly by simultaneously integrating the four maps to identify consistency. The final integrated linkage map was drawn using MapChart.<sup><a reveal-id="DSR017C51" data-open="DSR017C51" class="link link-ref link-reveal xref-bibr">51</a></sup> Power Marker 3.1 was used to calculate the polymorphic information content (PIC) value and gene diversity. The PIC value was estimated according to the method of Botstein <em>et al</em>.<sup><a reveal-id="DSR017C52" data-open="DSR017C52" class="link link-ref link-reveal xref-bibr">52</a></sup> The crucifer building blocks proposed by Schranz <em>et al</em>.<sup><a reveal-id="DSR017C53" data-open="DSR017C53" class="link link-ref link-reveal xref-bibr">53</a></sup> were identified in <em>B. rapa</em> genetic map based on homology search of primers pairs against the Arabidopsis genome sequence as described previously by Kim <em>et al</em>.<sup><a reveal-id="DSR017C7" data-open="DSR017C7" class="link link-ref link-reveal xref-bibr">7</a></sup></p>                            <h2 scrollto-destination=6714131 id="6714131" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=6714132 id="6714132" class="section-title" data-legacy-id=s3a>3.1. Development of UGMS markers</h3>
<p>We downloaded a total of 182 703 <em>B. rapa</em> EST sequences from NCBI database in April 2010 and alignment of these EST sequences gave 19 497 (18 931 contigs and 566 singlets) unigenes. Analyses in these unigenes identified 4174 microsatellite motifs in 3037 genes. Of these many unigenes containing one or more SSRs, we designed a total of 707 EST-SSR markers (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Among the primer pairs designed, trinucleotide repeats were the highest (573, 81.05%) followed by di- (126, 17.82%) and tetranucleotide repeats (8, 1.13%), respectively (Table <a reveal-id="DSR017TB2" data-open="DSR017TB2" class="link link-reveal link-table xref-fig">2</a>). Analysis of the location of the 707 SSR motifs in the sequence used to design primers showed that majority of them was located in the coding region (CDS, 491) compared to 5′UTR (107) and 3′UTR (109) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Of the 707 EST-SSR primer pairs, 691 (97.74%) produced repeatable and reliable amplifications of expected size in at least one line of the five <em>B. rapa</em> parental lines (Chiifu 401, Kenshin, Rapid cycling B. rapa, 94 SK and CR Shiki) screened, while 16 (2.26%) primer pairs either completely failed or led to weak amplifications and thus were excluded from further analysis. </p>                        <a id="6714134" scrollto-destination="6714134"></a>
<div content-id="DSR017TB2" class="table-modal"><div class="table-wrap"><div id="DSR017TB2" class="table-wrap-title" data-id="DSR017TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Frequency and distribution of different repeat types used to design EST-SSR primer pairs in <em>B. rapa</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Repeats </th><th colspan="11">Number of repeat units<hr /> </th><th>Total </th></tr><tr><th> </th><th>4 </th><th>5 </th><th>6 </th><th>7 </th><th>8 </th><th>9 </th><th>10 </th><th>11 </th><th>12 </th><th>13 </th><th>≥14 </th><th> </th></tr></thead><tbody><tr><td>AC/GT </td><td> </td><td> </td><td>1 </td><td> </td><td> </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>2 </td></tr><tr><td>AG/CT </td><td> </td><td> </td><td>17 </td><td>14 </td><td>5 </td><td>2 </td><td>5 </td><td>3 </td><td> </td><td> </td><td> </td><td>46 </td></tr><tr><td>AT/TA </td><td> </td><td> </td><td>9 </td><td>3 </td><td>2 </td><td>1 </td><td>2 </td><td> </td><td> </td><td>1 </td><td>2 </td><td>20 </td></tr><tr><td>CA/TG </td><td> </td><td> </td><td>4 </td><td>3 </td><td>2 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>10 </td></tr><tr><td>GA/TC </td><td> </td><td> </td><td>19 </td><td>7 </td><td>8 </td><td>5 </td><td>3 </td><td>1 </td><td>1 </td><td> </td><td>4 </td><td>48 </td></tr><tr><td>AAC/GTT </td><td>4 </td><td>3 </td><td>4 </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>13 </td></tr><tr><td>AAG/CTT </td><td>41 </td><td>13 </td><td>6 </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>62 </td></tr><tr><td>ACA/TGT </td><td>10 </td><td>3 </td><td>2 </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>17 </td></tr><tr><td>ACC/GGT </td><td>9 </td><td>7 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>16 </td></tr><tr><td>ACT/AGT </td><td>2 </td><td>2 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>5 </td></tr><tr><td>AGA/TCT </td><td>31 </td><td>7 </td><td>2 </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>42 </td></tr><tr><td>AGC/GCT </td><td>13 </td><td>6 </td><td>2 </td><td> </td><td> </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>22 </td></tr><tr><td>AGG/CCT </td><td>25 </td><td>14 </td><td>5 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>45 </td></tr><tr><td>ATA/ATT </td><td>3 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>3 </td></tr><tr><td>ATC/GAT </td><td>22 </td><td>5 </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>29 </td></tr><tr><td>ATG/CAT </td><td>11 </td><td>3 </td><td>2 </td><td> </td><td> </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td>18 </td></tr><tr><td>CAA/TTG </td><td>11 </td><td>5 </td><td>4 </td><td> </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>21 </td></tr><tr><td>CAC/GTG </td><td>3 </td><td>3 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>7 </td></tr><tr><td>CAG/CTG </td><td>8 </td><td>3 </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>1 </td><td> </td><td>14 </td></tr><tr><td>CCA/TGG </td><td>17 </td><td>7 </td><td> </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>26 </td></tr><tr><td>CCG/CGG </td><td>11 </td><td>3 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>15 </td></tr><tr><td>CGA/TCG </td><td>6 </td><td>2 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>9 </td></tr><tr><td>CGC/GCG </td><td>5 </td><td>3 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>8 </td></tr><tr><td>CGT/GAC </td><td>3 </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>5 </td></tr><tr><td>CTA/TAG </td><td>3 </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>5 </td></tr><tr><td>CTC/GAG </td><td>19 </td><td>6 </td><td>4 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>29 </td></tr><tr><td>GAA/TTC </td><td>32 </td><td>10 </td><td>5 </td><td>1 </td><td>1 </td><td>2 </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td>53 </td></tr><tr><td>GCA/TGC </td><td>14 </td><td>3 </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>19 </td></tr><tr><td>GCC/GGC </td><td>11 </td><td>3 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>15 </td></tr><tr><td>GGA/TCC </td><td>19 </td><td>10 </td><td>4 </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>35 </td></tr><tr><td>GTA/TAC </td><td>2 </td><td> </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>4 </td></tr><tr><td>TAA/TTA </td><td>4 </td><td> </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>5 </td></tr><tr><td>TCA/TGA </td><td>17 </td><td>11 </td><td>2 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>31 </td></tr><tr><td>AAGG/CTTC </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>2 </td></tr><tr><td>AGAA/TTAT </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>2 </td></tr><tr><td>GTGC/TGGT </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>2 </td></tr><tr><td>TACC/TGTT </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>2 </td></tr><tr><td>Total </td><td>364 </td><td>134 </td><td>107 </td><td>39 </td><td>23 </td><td>14 </td><td>12 </td><td>5 </td><td>1 </td><td>2 </td><td>6 </td><td>707 </td></tr></tbody></table></div><div content-id="DSR017TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Repeats </th><th colspan="11">Number of repeat units<hr /> </th><th>Total </th></tr><tr><th> </th><th>4 </th><th>5 </th><th>6 </th><th>7 </th><th>8 </th><th>9 </th><th>10 </th><th>11 </th><th>12 </th><th>13 </th><th>≥14 </th><th> </th></tr></thead><tbody><tr><td>AC/GT </td><td> </td><td> </td><td>1 </td><td> </td><td> </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>2 </td></tr><tr><td>AG/CT </td><td> </td><td> </td><td>17 </td><td>14 </td><td>5 </td><td>2 </td><td>5 </td><td>3 </td><td> </td><td> </td><td> </td><td>46 </td></tr><tr><td>AT/TA </td><td> </td><td> </td><td>9 </td><td>3 </td><td>2 </td><td>1 </td><td>2 </td><td> </td><td> </td><td>1 </td><td>2 </td><td>20 </td></tr><tr><td>CA/TG </td><td> </td><td> </td><td>4 </td><td>3 </td><td>2 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>10 </td></tr><tr><td>GA/TC </td><td> </td><td> </td><td>19 </td><td>7 </td><td>8 </td><td>5 </td><td>3 </td><td>1 </td><td>1 </td><td> </td><td>4 </td><td>48 </td></tr><tr><td>AAC/GTT </td><td>4 </td><td>3 </td><td>4 </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>13 </td></tr><tr><td>AAG/CTT </td><td>41 </td><td>13 </td><td>6 </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>62 </td></tr><tr><td>ACA/TGT </td><td>10 </td><td>3 </td><td>2 </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>17 </td></tr><tr><td>ACC/GGT </td><td>9 </td><td>7 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>16 </td></tr><tr><td>ACT/AGT </td><td>2 </td><td>2 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>5 </td></tr><tr><td>AGA/TCT </td><td>31 </td><td>7 </td><td>2 </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>42 </td></tr><tr><td>AGC/GCT </td><td>13 </td><td>6 </td><td>2 </td><td> </td><td> </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>22 </td></tr><tr><td>AGG/CCT </td><td>25 </td><td>14 </td><td>5 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>45 </td></tr><tr><td>ATA/ATT </td><td>3 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>3 </td></tr><tr><td>ATC/GAT </td><td>22 </td><td>5 </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>29 </td></tr><tr><td>ATG/CAT </td><td>11 </td><td>3 </td><td>2 </td><td> </td><td> </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td>18 </td></tr><tr><td>CAA/TTG </td><td>11 </td><td>5 </td><td>4 </td><td> </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>21 </td></tr><tr><td>CAC/GTG </td><td>3 </td><td>3 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>7 </td></tr><tr><td>CAG/CTG </td><td>8 </td><td>3 </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>1 </td><td> </td><td>14 </td></tr><tr><td>CCA/TGG </td><td>17 </td><td>7 </td><td> </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>26 </td></tr><tr><td>CCG/CGG </td><td>11 </td><td>3 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>15 </td></tr><tr><td>CGA/TCG </td><td>6 </td><td>2 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>9 </td></tr><tr><td>CGC/GCG </td><td>5 </td><td>3 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>8 </td></tr><tr><td>CGT/GAC </td><td>3 </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>5 </td></tr><tr><td>CTA/TAG </td><td>3 </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>5 </td></tr><tr><td>CTC/GAG </td><td>19 </td><td>6 </td><td>4 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>29 </td></tr><tr><td>GAA/TTC </td><td>32 </td><td>10 </td><td>5 </td><td>1 </td><td>1 </td><td>2 </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td>53 </td></tr><tr><td>GCA/TGC </td><td>14 </td><td>3 </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>19 </td></tr><tr><td>GCC/GGC </td><td>11 </td><td>3 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>15 </td></tr><tr><td>GGA/TCC </td><td>19 </td><td>10 </td><td>4 </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>35 </td></tr><tr><td>GTA/TAC </td><td>2 </td><td> </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>4 </td></tr><tr><td>TAA/TTA </td><td>4 </td><td> </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>5 </td></tr><tr><td>TCA/TGA </td><td>17 </td><td>11 </td><td>2 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>31 </td></tr><tr><td>AAGG/CTTC </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>2 </td></tr><tr><td>AGAA/TTAT </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>2 </td></tr><tr><td>GTGC/TGGT </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>2 </td></tr><tr><td>TACC/TGTT </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>2 </td></tr><tr><td>Total </td><td>364 </td><td>134 </td><td>107 </td><td>39 </td><td>23 </td><td>14 </td><td>12 </td><td>5 </td><td>1 </td><td>2 </td><td>6 </td><td>707 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6714134" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR017TB2" class="table-wrap-title" data-id="DSR017TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Frequency and distribution of different repeat types used to design EST-SSR primer pairs in <em>B. rapa</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Repeats </th><th colspan="11">Number of repeat units<hr /> </th><th>Total </th></tr><tr><th> </th><th>4 </th><th>5 </th><th>6 </th><th>7 </th><th>8 </th><th>9 </th><th>10 </th><th>11 </th><th>12 </th><th>13 </th><th>≥14 </th><th> </th></tr></thead><tbody><tr><td>AC/GT </td><td> </td><td> </td><td>1 </td><td> </td><td> </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>2 </td></tr><tr><td>AG/CT </td><td> </td><td> </td><td>17 </td><td>14 </td><td>5 </td><td>2 </td><td>5 </td><td>3 </td><td> </td><td> </td><td> </td><td>46 </td></tr><tr><td>AT/TA </td><td> </td><td> </td><td>9 </td><td>3 </td><td>2 </td><td>1 </td><td>2 </td><td> </td><td> </td><td>1 </td><td>2 </td><td>20 </td></tr><tr><td>CA/TG </td><td> </td><td> </td><td>4 </td><td>3 </td><td>2 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>10 </td></tr><tr><td>GA/TC </td><td> </td><td> </td><td>19 </td><td>7 </td><td>8 </td><td>5 </td><td>3 </td><td>1 </td><td>1 </td><td> </td><td>4 </td><td>48 </td></tr><tr><td>AAC/GTT </td><td>4 </td><td>3 </td><td>4 </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>13 </td></tr><tr><td>AAG/CTT </td><td>41 </td><td>13 </td><td>6 </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>62 </td></tr><tr><td>ACA/TGT </td><td>10 </td><td>3 </td><td>2 </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>17 </td></tr><tr><td>ACC/GGT </td><td>9 </td><td>7 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>16 </td></tr><tr><td>ACT/AGT </td><td>2 </td><td>2 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>5 </td></tr><tr><td>AGA/TCT </td><td>31 </td><td>7 </td><td>2 </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>42 </td></tr><tr><td>AGC/GCT </td><td>13 </td><td>6 </td><td>2 </td><td> </td><td> </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>22 </td></tr><tr><td>AGG/CCT </td><td>25 </td><td>14 </td><td>5 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>45 </td></tr><tr><td>ATA/ATT </td><td>3 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>3 </td></tr><tr><td>ATC/GAT </td><td>22 </td><td>5 </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>29 </td></tr><tr><td>ATG/CAT </td><td>11 </td><td>3 </td><td>2 </td><td> </td><td> </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td>18 </td></tr><tr><td>CAA/TTG </td><td>11 </td><td>5 </td><td>4 </td><td> </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>21 </td></tr><tr><td>CAC/GTG </td><td>3 </td><td>3 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>7 </td></tr><tr><td>CAG/CTG </td><td>8 </td><td>3 </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>1 </td><td> </td><td>14 </td></tr><tr><td>CCA/TGG </td><td>17 </td><td>7 </td><td> </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>26 </td></tr><tr><td>CCG/CGG </td><td>11 </td><td>3 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>15 </td></tr><tr><td>CGA/TCG </td><td>6 </td><td>2 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>9 </td></tr><tr><td>CGC/GCG </td><td>5 </td><td>3 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>8 </td></tr><tr><td>CGT/GAC </td><td>3 </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>5 </td></tr><tr><td>CTA/TAG </td><td>3 </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>5 </td></tr><tr><td>CTC/GAG </td><td>19 </td><td>6 </td><td>4 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>29 </td></tr><tr><td>GAA/TTC </td><td>32 </td><td>10 </td><td>5 </td><td>1 </td><td>1 </td><td>2 </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td>53 </td></tr><tr><td>GCA/TGC </td><td>14 </td><td>3 </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>19 </td></tr><tr><td>GCC/GGC </td><td>11 </td><td>3 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>15 </td></tr><tr><td>GGA/TCC </td><td>19 </td><td>10 </td><td>4 </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>35 </td></tr><tr><td>GTA/TAC </td><td>2 </td><td> </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>4 </td></tr><tr><td>TAA/TTA </td><td>4 </td><td> </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>5 </td></tr><tr><td>TCA/TGA </td><td>17 </td><td>11 </td><td>2 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>31 </td></tr><tr><td>AAGG/CTTC </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>2 </td></tr><tr><td>AGAA/TTAT </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>2 </td></tr><tr><td>GTGC/TGGT </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>2 </td></tr><tr><td>TACC/TGTT </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>2 </td></tr><tr><td>Total </td><td>364 </td><td>134 </td><td>107 </td><td>39 </td><td>23 </td><td>14 </td><td>12 </td><td>5 </td><td>1 </td><td>2 </td><td>6 </td><td>707 </td></tr></tbody></table></div><div content-id="DSR017TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Repeats </th><th colspan="11">Number of repeat units<hr /> </th><th>Total </th></tr><tr><th> </th><th>4 </th><th>5 </th><th>6 </th><th>7 </th><th>8 </th><th>9 </th><th>10 </th><th>11 </th><th>12 </th><th>13 </th><th>≥14 </th><th> </th></tr></thead><tbody><tr><td>AC/GT </td><td> </td><td> </td><td>1 </td><td> </td><td> </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>2 </td></tr><tr><td>AG/CT </td><td> </td><td> </td><td>17 </td><td>14 </td><td>5 </td><td>2 </td><td>5 </td><td>3 </td><td> </td><td> </td><td> </td><td>46 </td></tr><tr><td>AT/TA </td><td> </td><td> </td><td>9 </td><td>3 </td><td>2 </td><td>1 </td><td>2 </td><td> </td><td> </td><td>1 </td><td>2 </td><td>20 </td></tr><tr><td>CA/TG </td><td> </td><td> </td><td>4 </td><td>3 </td><td>2 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>10 </td></tr><tr><td>GA/TC </td><td> </td><td> </td><td>19 </td><td>7 </td><td>8 </td><td>5 </td><td>3 </td><td>1 </td><td>1 </td><td> </td><td>4 </td><td>48 </td></tr><tr><td>AAC/GTT </td><td>4 </td><td>3 </td><td>4 </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>13 </td></tr><tr><td>AAG/CTT </td><td>41 </td><td>13 </td><td>6 </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>62 </td></tr><tr><td>ACA/TGT </td><td>10 </td><td>3 </td><td>2 </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>17 </td></tr><tr><td>ACC/GGT </td><td>9 </td><td>7 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>16 </td></tr><tr><td>ACT/AGT </td><td>2 </td><td>2 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>5 </td></tr><tr><td>AGA/TCT </td><td>31 </td><td>7 </td><td>2 </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>42 </td></tr><tr><td>AGC/GCT </td><td>13 </td><td>6 </td><td>2 </td><td> </td><td> </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>22 </td></tr><tr><td>AGG/CCT </td><td>25 </td><td>14 </td><td>5 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>45 </td></tr><tr><td>ATA/ATT </td><td>3 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>3 </td></tr><tr><td>ATC/GAT </td><td>22 </td><td>5 </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>29 </td></tr><tr><td>ATG/CAT </td><td>11 </td><td>3 </td><td>2 </td><td> </td><td> </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td>18 </td></tr><tr><td>CAA/TTG </td><td>11 </td><td>5 </td><td>4 </td><td> </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>21 </td></tr><tr><td>CAC/GTG </td><td>3 </td><td>3 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>7 </td></tr><tr><td>CAG/CTG </td><td>8 </td><td>3 </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>1 </td><td> </td><td>14 </td></tr><tr><td>CCA/TGG </td><td>17 </td><td>7 </td><td> </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>26 </td></tr><tr><td>CCG/CGG </td><td>11 </td><td>3 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>15 </td></tr><tr><td>CGA/TCG </td><td>6 </td><td>2 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>9 </td></tr><tr><td>CGC/GCG </td><td>5 </td><td>3 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>8 </td></tr><tr><td>CGT/GAC </td><td>3 </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>5 </td></tr><tr><td>CTA/TAG </td><td>3 </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>5 </td></tr><tr><td>CTC/GAG </td><td>19 </td><td>6 </td><td>4 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>29 </td></tr><tr><td>GAA/TTC </td><td>32 </td><td>10 </td><td>5 </td><td>1 </td><td>1 </td><td>2 </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td>53 </td></tr><tr><td>GCA/TGC </td><td>14 </td><td>3 </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>19 </td></tr><tr><td>GCC/GGC </td><td>11 </td><td>3 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>15 </td></tr><tr><td>GGA/TCC </td><td>19 </td><td>10 </td><td>4 </td><td>1 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>35 </td></tr><tr><td>GTA/TAC </td><td>2 </td><td> </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>4 </td></tr><tr><td>TAA/TTA </td><td>4 </td><td> </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>5 </td></tr><tr><td>TCA/TGA </td><td>17 </td><td>11 </td><td>2 </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>31 </td></tr><tr><td>AAGG/CTTC </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>2 </td></tr><tr><td>AGAA/TTAT </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>2 </td></tr><tr><td>GTGC/TGGT </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>2 </td></tr><tr><td>TACC/TGTT </td><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>2 </td></tr><tr><td>Total </td><td>364 </td><td>134 </td><td>107 </td><td>39 </td><td>23 </td><td>14 </td><td>12 </td><td>5 </td><td>1 </td><td>2 </td><td>6 </td><td>707 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6714134" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=6714135 id="6714135" class="section-title" data-legacy-id=s3b>3.2. Construction of individual maps</h3>
                            <h4 scrollto-destination=6714136 id="6714136" class="section-title" data-legacy-id=s3b1>3.2.1. Updating the CKDH reference genetic map</h4>
<p>The CKDH linkage map was adopted as the reference genetic linkage map by the MBrGSP. The version I CKDH reference genetic map of <em>B. rapa</em> was constructed by Choi <em>et al</em>.,<sup><a reveal-id="DSR017C5" data-open="DSR017C5" class="link link-ref link-reveal xref-bibr">5</a></sup> while Kim <em>et al</em>.<sup><a reveal-id="DSR017C7" data-open="DSR017C7" class="link link-ref link-reveal xref-bibr">7</a></sup> generated version II by incorporating more BAC-anchored SSRs, and this was further updated with the inclusion of 95 gene-based IP markers by Li <em>et al</em>.<sup><a reveal-id="DSR017C33" data-open="DSR017C33" class="link link-ref link-reveal xref-bibr">33</a></sup> We screened 707 newly developed EST-SSR markers (prefix ACMP) for polymorphisms between the parental lines of the CKDH mapping population, Chiifu-401 and Kenshin, in the present study. However, only 99 (14%) of these markers were polymorphic between the 2 parental lines, and finally 95 EST-SSR markers were mapped to the 10 linkage groups of <em>B. rapa</em>. These 95 EST-SSR markers were distributed in all of the 10 <em>B. rapa</em> linkage groups, except in A2, and the number of markers ranged from 4 in A10 to 19 in A5. After adding the new EST-SSR markers, most of the previous markers were assigned in the same order without any major changes with respect to their position. The total length of the updated CKDH map was 1217.6 cM, which is 42.5 cM larger than the earlier map (1175.1 cM; Li <em>et al</em>.<sup><a reveal-id="DSR017C33" data-open="DSR017C33" class="link link-ref link-reveal xref-bibr">33</a></sup>). The linkage groups A3 (145.8 cM) and A6 (165.2 cM) increased by 10 cM compared to the map of Li <em>et al</em>.<sup><a reveal-id="DSR017C33" data-open="DSR017C33" class="link link-ref link-reveal xref-bibr">33</a></sup> because of the addition of new EST markers. The average distance between adjacent markers decreased from 1.45 to 1.34 cM. The updated EST-SSRs CKDH map consisted of a total of 907 markers with 190 BAC-derived SSRs, 95 new EST-derived SSRs, 94 IP markers, and 528 other markers (Choi <em>et al</em>.,<sup><a reveal-id="DSR017C5" data-open="DSR017C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Table <a reveal-id="DSR017TB2" data-open="DSR017TB2" class="link link-reveal link-table xref-fig">2</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>).</p>                            <h4 scrollto-destination=6714138 id="6714138" class="section-title" data-legacy-id=s3b2>3.2.2. Updating the CRF<sub>2</sub> linkage map</h4>
<p>The CRF<sub>2</sub> map was initially developed by Li <em>et al</em>.<sup><a reveal-id="DSR017C33" data-open="DSR017C33" class="link link-ref link-reveal xref-bibr">33</a></sup> using BAC-derived SSR and IP markers. Of the 707 EST-SSRs screened for polymorphisms between the parental lines, Chiifu-401 and RCBr, we identified 144 pairs of polymorphic EST-SSRs, of which only 142 pairs could be used for genotyping and the construction of the linkage map. After excluding the distorted and ungrouped markers, a total of 129 new EST-SSRs were successfully integrated into the 10 linkage groups of <em>B. rapa</em>, giving a total length of 1119.5 cM. The distribution of the newly mapped EST-SSRs varied from 7 in linkage group A10 to 19 in A3. Compared to the previous CRF<sub>2</sub> map,<sup><a reveal-id="DSR017C33" data-open="DSR017C33" class="link link-ref link-reveal xref-bibr">33</a></sup> most of the BAC-SSRs and IP markers remained in the same order; however, two SSR loci and one IP locus in linkage group A4, one IP locus in A5, and one SSR locus in A9, all distorted markers, were deleted from the map as they reshuffled the marker order in these two linkage groups. The updated CRF<sub>2</sub> linkage map now contains a total of 444 markers with 129 new EST-SSRs, 249 BAC-derived SSRs, and 66 IP loci (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). The length of the individual linkage groups ranged from 79.9 cM in A06 to 170.1 cM in A9. Although the total length of the genetic map did not significantly increase compared to the previous CRF<sub>2</sub> map,<sup><a reveal-id="DSR017C33" data-open="DSR017C33" class="link link-ref link-reveal xref-bibr">33</a></sup> the average distance between adjacent markers decreased from 3.47 to 2.53 cM.</p>                            <h4 scrollto-destination=6714140 id="6714140" class="section-title" data-legacy-id=s3b3>3.2.3. Construction of the CSKF<sub>2</sub> linkage map</h4>
<p>The new CSKF<sub>2</sub> population was developed by crossing the clubroot resistant cultivar CR Shinki and the susceptible 94SK line. For the construction of the genetic linkage map, we screened SSRs and IP markers for polymorphisms between the parental lines. Of the 707 EST-SSRs, 250 BRPGM-SSRs, 50 cnu_SSRs, 30 nia_SSRs, and 272 IP markers screened between the parental lines, and only 99 EST-SSRs, 28 BRPGM-SSRs, 19 cnu_SSRs, 6 nia_SSRs, and 19 IP makers were polymorphic. All markers, except the EST-SSRs, were previously described and some of them have already been mapped in the <em>B. rapa</em> genome.<sup><a reveal-id="DSR017C7" data-open="DSR017C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSR017C33" data-open="DSR017C33" class="link link-ref link-reveal xref-bibr">33</a></sup> The previously mapped markers enabled us to align and integrate the different maps in the present study. We used the genotype data from a total of 173 markers to construct the linkage map, although only 161 markers [93 EST-SSR, 49 BAC-SSR, 17 IP, and 2 sequence characterized amplified region (SCAR) markers] could be grouped and assigned to the 10 linkage groups, covering 602.3 cM of the <em>B. rapa</em> genome (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). The number of markers in each linkage group ranged from 7 in A1 to 33 in A3, with an average distance of 3.74 cM between adjacent markers.</p>                            <h4 scrollto-destination=6714142 id="6714142" class="section-title" data-legacy-id=s3b4>3.2.4. PF<sub>2</sub> linkage map</h4>
<p>The PF<sub>2</sub> genetic map was constructed using 144 F<sub>2</sub> lines derived from crossing between two diverse Chinese cabbage inbred lines, 501 with a large head and 601 with a small head, at Shenyang Agricultural University, China. This genetic map of <em>B. rapa</em> contains a total of 277 markers (72 EST-SSR loci, 154 genomic SSRs, and 1 leaf hairiness phenotypic marker) in the 10 linkage groups with total genome coverage of 908.4 cM. The average distance between adjacent markers was 4.02 cM (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S1 and Table S5</a>) and the shortest and longest linkage groups were A8 (74.9 cM) and A9 (122.0 cM).</p>                            <h3 scrollto-destination=6714144 id="6714144" class="section-title" data-legacy-id=s3c>3.3. Construction of the updated consensus genetic map of B. rapa</h3>
<p>The four individual maps were integrated using the commonly mapped markers as bridge markers. In total, 241 bridge markers (including 66 EST-SSRs) were identified among at least two of the mapping populations. The distribution of bridge markers was 22 in A1, 12 in A2, 38 in A3, 13 in A4, 37 in A5, 19 in A6, 25 in A7, 17 in A8, 42 in A9, and 16 in A10, respectively. SSR and IP markers with more than one polymorphic locus in the same linkage groups were correctly identified for size and order before designating the common loci between the maps. A total of 1426 markers were mapped on the 10 linkage groups of <em>B. rapa</em>. The number of markers in the integrated map ranged from 97 markers in linkage group A2 to 209 markers in A3, and the length of the linkage groups varied from 95.5 cM in A10 to 160.0 cM in A9. The total length of the integrated consensus map was 1245.9 cM, with an average distance between adjacent markers of 0.87 cM (Table <a reveal-id="DSR017TB3" data-open="DSR017TB3" class="link link-reveal link-table xref-fig">3</a> and Fig. <a reveal-id="DSR017F1" data-open="DSR017F1" class="link link-reveal link-table xref-fig">1</a>). The comparison of the four individual maps and the integrated map revealed a similar order of the markers, even though changes of order and the position of a few markers were observed within a 5 cM distance in some of the linkage groups, with the exception of five linkage groups (A3, A5, A7, A9, and A10) where an inversion of more than 10 cM was observed. The integrated map contained 306 new EST-derived SSR markers (prefixed by ACMP and sau-um), 395 BAC-derived SSRs, 153 IP markers, and 572 other markers (Table <a reveal-id="DSR017TB3" data-open="DSR017TB3" class="link link-reveal link-table xref-fig">3</a> and Fig. <a reveal-id="DSR017F1" data-open="DSR017F1" class="link link-reveal link-table xref-fig">1</a>). The number of EST-derived SSRs mapped in the 10 linkage groups of <em>B. rapa</em> ranged from 16 in linkage group A2 to 50 in A3. The density of the updated EST-SSR-rich integrated map increased compared to the previously integrated CKDH and CRF<sub>2</sub> maps by Li <em>et al</em>.<sup><a reveal-id="DSR017C33" data-open="DSR017C33" class="link link-ref link-reveal xref-bibr">33</a></sup> since the average distance between marker loci decreased from 1.24 to 0.87 cM. The length of the integrated linkage groups was similar to the corresponding longest linkage groups of the component maps with a slight increase in map length, except for the linkage groups A1, A6, A8, and A10 where decreases of approximately 20, 13.1, 8.4, and 10 cM, respectively, were observed in comparison to the maximum lengths of the individual maps. The EST-SSR marker ACMP00682, which mapped to the top portion of the A08 linkage group, increased the map length by 4.8 cM. The large gaps observed in individual maps were reduced due to the increased marker density, and the final consensus integrated map had only one large gap (&gt;10 cM) in the A2 linkage group. </p>                        <a id="6714146" scrollto-destination="6714146"></a>
<div content-id="DSR017TB3" class="table-modal"><div class="table-wrap"><div id="DSR017TB3" class="table-wrap-title" data-id="DSR017TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Characteristics of the updated <em>Brassica rapa</em> integrated linkage map developed using four mapping populations</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Linkage group </th><th colspan="5">Number of different marker types<hr /> </th><th>Total length (cM) </th><th>Average distance (cM) </th></tr><tr><th> </th><th>EST-derived SSRs<sup>a</sup> </th><th>BAC-derived SSRs<sup>b</sup> </th><th>IP<sup>c</sup> </th><th>Others<sup>d</sup> </th><th>Total </th><th> </th><th> </th></tr></thead><tbody><tr><td>A1 </td><td>21 </td><td>43 </td><td>12 </td><td>52 </td><td>128 </td><td>119.0 </td><td>0.93 </td></tr><tr><td>A2 </td><td>16 </td><td>25 </td><td>15 </td><td>41 </td><td>97 </td><td>122.3 </td><td>1.26 </td></tr><tr><td>A3 </td><td>50 </td><td>55 </td><td>31 </td><td>73 </td><td>209 </td><td>147.4 </td><td>0.71 </td></tr><tr><td>A4 </td><td>22 </td><td>25 </td><td>9 </td><td>29 </td><td>85 </td><td>104.7 </td><td>1.23 </td></tr><tr><td>A5 </td><td>42 </td><td>45 </td><td>12 </td><td>73 </td><td>172 </td><td>133.1 </td><td>0.77 </td></tr><tr><td>A6 </td><td>41 </td><td>19 </td><td>10 </td><td>79 </td><td>149 </td><td>152.1 </td><td>1.02 </td></tr><tr><td>A7 </td><td>34 </td><td>55 </td><td>14 </td><td>80 </td><td>183 </td><td>106.8 </td><td>0.58 </td></tr><tr><td>A8 </td><td>20 </td><td>30 </td><td>11 </td><td>41 </td><td>102 </td><td>105.0 </td><td>1.03 </td></tr><tr><td>A9 </td><td>41 </td><td>74 </td><td>29 </td><td>59 </td><td>203 </td><td>160.0 </td><td>0.79 </td></tr><tr><td>A10 </td><td>19 </td><td>24 </td><td>10 </td><td>45 </td><td>98 </td><td>95.5 </td><td>0.97 </td></tr><tr><td>Total </td><td>306 </td><td>395 </td><td>153 </td><td>572 </td><td>1426 </td><td>1245.9 </td><td>0.87 </td></tr></tbody></table></div><div content-id="DSR017TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Linkage group </th><th colspan="5">Number of different marker types<hr /> </th><th>Total length (cM) </th><th>Average distance (cM) </th></tr><tr><th> </th><th>EST-derived SSRs<sup>a</sup> </th><th>BAC-derived SSRs<sup>b</sup> </th><th>IP<sup>c</sup> </th><th>Others<sup>d</sup> </th><th>Total </th><th> </th><th> </th></tr></thead><tbody><tr><td>A1 </td><td>21 </td><td>43 </td><td>12 </td><td>52 </td><td>128 </td><td>119.0 </td><td>0.93 </td></tr><tr><td>A2 </td><td>16 </td><td>25 </td><td>15 </td><td>41 </td><td>97 </td><td>122.3 </td><td>1.26 </td></tr><tr><td>A3 </td><td>50 </td><td>55 </td><td>31 </td><td>73 </td><td>209 </td><td>147.4 </td><td>0.71 </td></tr><tr><td>A4 </td><td>22 </td><td>25 </td><td>9 </td><td>29 </td><td>85 </td><td>104.7 </td><td>1.23 </td></tr><tr><td>A5 </td><td>42 </td><td>45 </td><td>12 </td><td>73 </td><td>172 </td><td>133.1 </td><td>0.77 </td></tr><tr><td>A6 </td><td>41 </td><td>19 </td><td>10 </td><td>79 </td><td>149 </td><td>152.1 </td><td>1.02 </td></tr><tr><td>A7 </td><td>34 </td><td>55 </td><td>14 </td><td>80 </td><td>183 </td><td>106.8 </td><td>0.58 </td></tr><tr><td>A8 </td><td>20 </td><td>30 </td><td>11 </td><td>41 </td><td>102 </td><td>105.0 </td><td>1.03 </td></tr><tr><td>A9 </td><td>41 </td><td>74 </td><td>29 </td><td>59 </td><td>203 </td><td>160.0 </td><td>0.79 </td></tr><tr><td>A10 </td><td>19 </td><td>24 </td><td>10 </td><td>45 </td><td>98 </td><td>95.5 </td><td>0.97 </td></tr><tr><td>Total </td><td>306 </td><td>395 </td><td>153 </td><td>572 </td><td>1426 </td><td>1245.9 </td><td>0.87 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>EST-derived SSRs that include ‘ACMP’ and ‘Sau-um’ EST-SSRs.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>SSR markers designed from BAC end sequences prefixed by ‘cnu’, ‘nia’, ‘BRPGM’, and PC11 marker.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>IP markers from <em>B.juncea</em>.<sup><a reveal-id="DSR017C6" data-open="DSR017C6" class="link link-ref link-reveal xref-bibr">6</a></sup></p></div></div><div content-id=""><div class="fn"><p><sup>d</sup>AFLP, RAPD, RFLP, STS, ESTP,CAPS,<sup><a reveal-id="DSR017C5" data-open="DSR017C5" class="link link-ref link-reveal xref-bibr">5</a></sup> and public SSR markers.<sup><a reveal-id="DSR017C38  DSR017C47" data-open="DSR017C38  DSR017C47" class="link link-ref link-reveal xref-bibr">38–47</a></sup></p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6714146" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR017TB3" class="table-wrap-title" data-id="DSR017TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Characteristics of the updated <em>Brassica rapa</em> integrated linkage map developed using four mapping populations</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Linkage group </th><th colspan="5">Number of different marker types<hr /> </th><th>Total length (cM) </th><th>Average distance (cM) </th></tr><tr><th> </th><th>EST-derived SSRs<sup>a</sup> </th><th>BAC-derived SSRs<sup>b</sup> </th><th>IP<sup>c</sup> </th><th>Others<sup>d</sup> </th><th>Total </th><th> </th><th> </th></tr></thead><tbody><tr><td>A1 </td><td>21 </td><td>43 </td><td>12 </td><td>52 </td><td>128 </td><td>119.0 </td><td>0.93 </td></tr><tr><td>A2 </td><td>16 </td><td>25 </td><td>15 </td><td>41 </td><td>97 </td><td>122.3 </td><td>1.26 </td></tr><tr><td>A3 </td><td>50 </td><td>55 </td><td>31 </td><td>73 </td><td>209 </td><td>147.4 </td><td>0.71 </td></tr><tr><td>A4 </td><td>22 </td><td>25 </td><td>9 </td><td>29 </td><td>85 </td><td>104.7 </td><td>1.23 </td></tr><tr><td>A5 </td><td>42 </td><td>45 </td><td>12 </td><td>73 </td><td>172 </td><td>133.1 </td><td>0.77 </td></tr><tr><td>A6 </td><td>41 </td><td>19 </td><td>10 </td><td>79 </td><td>149 </td><td>152.1 </td><td>1.02 </td></tr><tr><td>A7 </td><td>34 </td><td>55 </td><td>14 </td><td>80 </td><td>183 </td><td>106.8 </td><td>0.58 </td></tr><tr><td>A8 </td><td>20 </td><td>30 </td><td>11 </td><td>41 </td><td>102 </td><td>105.0 </td><td>1.03 </td></tr><tr><td>A9 </td><td>41 </td><td>74 </td><td>29 </td><td>59 </td><td>203 </td><td>160.0 </td><td>0.79 </td></tr><tr><td>A10 </td><td>19 </td><td>24 </td><td>10 </td><td>45 </td><td>98 </td><td>95.5 </td><td>0.97 </td></tr><tr><td>Total </td><td>306 </td><td>395 </td><td>153 </td><td>572 </td><td>1426 </td><td>1245.9 </td><td>0.87 </td></tr></tbody></table></div><div content-id="DSR017TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Linkage group </th><th colspan="5">Number of different marker types<hr /> </th><th>Total length (cM) </th><th>Average distance (cM) </th></tr><tr><th> </th><th>EST-derived SSRs<sup>a</sup> </th><th>BAC-derived SSRs<sup>b</sup> </th><th>IP<sup>c</sup> </th><th>Others<sup>d</sup> </th><th>Total </th><th> </th><th> </th></tr></thead><tbody><tr><td>A1 </td><td>21 </td><td>43 </td><td>12 </td><td>52 </td><td>128 </td><td>119.0 </td><td>0.93 </td></tr><tr><td>A2 </td><td>16 </td><td>25 </td><td>15 </td><td>41 </td><td>97 </td><td>122.3 </td><td>1.26 </td></tr><tr><td>A3 </td><td>50 </td><td>55 </td><td>31 </td><td>73 </td><td>209 </td><td>147.4 </td><td>0.71 </td></tr><tr><td>A4 </td><td>22 </td><td>25 </td><td>9 </td><td>29 </td><td>85 </td><td>104.7 </td><td>1.23 </td></tr><tr><td>A5 </td><td>42 </td><td>45 </td><td>12 </td><td>73 </td><td>172 </td><td>133.1 </td><td>0.77 </td></tr><tr><td>A6 </td><td>41 </td><td>19 </td><td>10 </td><td>79 </td><td>149 </td><td>152.1 </td><td>1.02 </td></tr><tr><td>A7 </td><td>34 </td><td>55 </td><td>14 </td><td>80 </td><td>183 </td><td>106.8 </td><td>0.58 </td></tr><tr><td>A8 </td><td>20 </td><td>30 </td><td>11 </td><td>41 </td><td>102 </td><td>105.0 </td><td>1.03 </td></tr><tr><td>A9 </td><td>41 </td><td>74 </td><td>29 </td><td>59 </td><td>203 </td><td>160.0 </td><td>0.79 </td></tr><tr><td>A10 </td><td>19 </td><td>24 </td><td>10 </td><td>45 </td><td>98 </td><td>95.5 </td><td>0.97 </td></tr><tr><td>Total </td><td>306 </td><td>395 </td><td>153 </td><td>572 </td><td>1426 </td><td>1245.9 </td><td>0.87 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>EST-derived SSRs that include ‘ACMP’ and ‘Sau-um’ EST-SSRs.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>SSR markers designed from BAC end sequences prefixed by ‘cnu’, ‘nia’, ‘BRPGM’, and PC11 marker.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>IP markers from <em>B.juncea</em>.<sup><a reveal-id="DSR017C6" data-open="DSR017C6" class="link link-ref link-reveal xref-bibr">6</a></sup></p></div></div><div content-id=""><div class="fn"><p><sup>d</sup>AFLP, RAPD, RFLP, STS, ESTP,CAPS,<sup><a reveal-id="DSR017C5" data-open="DSR017C5" class="link link-ref link-reveal xref-bibr">5</a></sup> and public SSR markers.<sup><a reveal-id="DSR017C38  DSR017C47" data-open="DSR017C38  DSR017C47" class="link link-ref link-reveal xref-bibr">38–47</a></sup></p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6714146" target="_blank">View Large</a></div></div>                        <a id="6714147" scrollto-destination="6714147"></a>
<div id="DSR017F1" data-id="DSR017F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr017/2/m_dsr01701a.jpeg?Expires=2147483647&amp;Signature=KWsu-gBIZUnI~I-NmHjjFyFcE-gQseNqFLROtyqVGJuPyVJwJvWroeS5gzh7wkCMwd7YPu2xGVFoiJVZfLS70f-R8k1HdbV~eRsW4pqTIpspWf~4tZtpUcmQ8Fn2Cvza3BM4VbyTnMAnFKsx03jTKqj~hSaaZk6cHvJuJI80MWtcwGGVRbZO7BHleX8NzeyzcCvVP17ewRdBmk3QOOaa3A9pnDV6QTJZH9VqKInF0FrC-ygd-4P82fWWDaR3WQb1XGLAYFvyRAi0RnPQxq3I5GGkFH0gZvBGMjMnLJqlGQxVGWCtSLaCO6XNGMZDSQLSysjbroS3HzLrEssUnZdmhw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The distribution of EST–SSRs and other markers in the 10 integrated linkage groups (A1–A10) of Brassica rapa. BAC-derived SSR markers,7 EST–SSRs and IP markers6 containing representative blocks were highlighted by bold strokes. New EST-derived SSR markers are underlined. The colorful rectangular bars on left of the integrated linkage map indicates the crucifer building blocks homologous to the Arabidopsis thaliana (At) chromosomes (C1–C5).53" path-from-xml="dsr01701a.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6714147/dsr01701a.jpeg" path-from-xml="dsr01701a.jpeg" target="_blank">View large</a><a section="6714147" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr017/2/dsr01701a.jpeg?Expires=2147483647&amp;Signature=xnwWY-UV-GG61Z1-G5kwzscVl4tADYwVXmHw726AJ5QO-A3je15IBNV9NNi~H3m-iM1g61A4ct6YbLr~YeyuVLgkhb1gE24mV3N9Fnse3EzzTUnGgbq0n8nxSM6g2dO3MIKe3iJub4kMB7-NKoi3Y~uWm0vCK4GHmNBiPb4saubFpJgXgRPMxzR1u52Q1j9fAM1yVQBO5Znf7W1C6dsIBnpDImgSutkl8etq0OfT0x1xFEQHrEjIrgp3r1-FjYMP9ctxyydfBbeRtIx-yyuKYd~Bo3YaYd-psqBmgLtQFFGyZ1rl0EElghM7Xng8FNnHW~cpLI3SBqszy~R~gj5Y1A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=6714147&amp;ar=448180&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsr01701a.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr017/2/m_dsr01701b.jpeg?Expires=2147483647&amp;Signature=Gz66vR8vLRbzzC2bflqiZjRv~HQhm4gU2F~mPrRU-DxjfvVjCE0f7I~tezmA6p8IRFZNpKEkg6uq4ezhgArbabcGZFyUN3I3YtxN0MGWnSkbjcSkIxBUO7iVBjSEo9Ze3avQqiSfEhaDfpCTN7hB0fPLclTLBdxgIow7dGLvDILGATCJrCMMLIXm-4gDOKKeStn3ZOReBIZtskO7FL13EA2MiniA2dIRuzqO9-~hXYyqwy-kZvPzzGYJIltR5SFCsK39Fh6jaBJ3rLyFYfXA8UPsc7rwlCfEXMcbw0XAQ5HCsoqUgvZkUWSdbEheYsBl1YbsZ7Vr~hJeU3zmCfKzXA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The distribution of EST–SSRs and other markers in the 10 integrated linkage groups (A1–A10) of Brassica rapa. BAC-derived SSR markers,7 EST–SSRs and IP markers6 containing representative blocks were highlighted by bold strokes. New EST-derived SSR markers are underlined. The colorful rectangular bars on left of the integrated linkage map indicates the crucifer building blocks homologous to the Arabidopsis thaliana (At) chromosomes (C1–C5).53" path-from-xml="dsr01701b.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6714147/dsr01701b.jpeg" path-from-xml="dsr01701b.jpeg" target="_blank">View large</a><a section="6714147" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr017/2/dsr01701b.jpeg?Expires=2147483647&amp;Signature=K9G-V7apmR6KmeG3SMsiZ9qScXJg3PKFqEonWZfSZTCT-tdHYmIisTmUslVz3Tl7eFPDrQ0roG5i0xyF5sLKIG5TVhq~0Z8ERnVisBLTFt6brTjadA5Xmnig0s8RcYTzDhQ4bsdgQTkpNyHHPS7aai7t-EoY2HZRDNhCIKCETtWMNeO1fIAN-oOhFzGpFQLnXwiKIxxTAvjuDcDId0y5TBNrcgXHBPHIDM15geBY2GPX8RTEkXirMJa3fXhZqLybyF0Y9Q0B1wJpCm~V6Uuq5DvjZN6HEP6a-9mrXFO4LreS5~8YYCg9PgVRAZjx-7N33g4RtR8Ebhy1R5tv0qW8Pw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=6714147&amp;ar=448180&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsr01701b.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr017/2/m_dsr01701c.jpeg?Expires=2147483647&amp;Signature=iNQgG9gtxI22z41hzQFvK99q1dN0HXkuI1swI588XBlOVprlLFLb6lIeDmRqdyqzPMVBzsStfCgTKfzxORh5HZXtdRAFuQDwGigEu4JGj6060qbcWo1ytCxTR6ePN~iOPwM55AMtx35ZD-ucLL1Ek0k5RtRMnlWVBPJ7ZqGrEli1Gas5f8tbPXJKRqEX3keNEPWiH3ItiJzAWtAQ3UzxI~Nbl9mVEXqQjrjvF9mge4Rc2ydUPGYLaENE-cHabUE7diEsSMXkWZERznaVQWQhn66J50XHJguTdk-Ct9a7HSqnfrB2E1MocQh-T5L1A1efOF-zizpjMIc7hHXlBkJofw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The distribution of EST–SSRs and other markers in the 10 integrated linkage groups (A1–A10) of Brassica rapa. BAC-derived SSR markers,7 EST–SSRs and IP markers6 containing representative blocks were highlighted by bold strokes. New EST-derived SSR markers are underlined. The colorful rectangular bars on left of the integrated linkage map indicates the crucifer building blocks homologous to the Arabidopsis thaliana (At) chromosomes (C1–C5).53" path-from-xml="dsr01701c.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6714147/dsr01701c.jpeg" path-from-xml="dsr01701c.jpeg" target="_blank">View large</a><a section="6714147" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr017/2/dsr01701c.jpeg?Expires=2147483647&amp;Signature=EVqwTivz744geIJwQE5fXLX~mO89Wygct3Wq-YhAUDpOPSbLxDRVwjhFwM~N84hkf4f8lMF2euFAKSkCQS3a1Tl0uYcaMwvHbGUm2qsmgWZRyfTEWjL5Rp00~P4HZU8X3Mt5Jofqh3VRDFQQ1dyeQ-Kpc37jw~pGg4Ug1cf-4wch8nC~WeILcDBjEDIGj~g0RSj~ZIkkTJIn32xsIQfw1zDZRkV6yZvsB7msQWSU9bhbmbP-JCrPecVFPOa7MyMv5xs~jfQJk5YFY~h84wOGsDAOLjBS-z01Tu-pPSwSWxIc57b8IJmLLcIR1TAWjEg-9IBvX1hwCNFre9b4pMiNlg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=6714147&amp;ar=448180&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsr01701c.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The distribution of EST–SSRs and other markers in the 10 integrated linkage groups (A1–A10) of <em>Brassica rapa</em>. BAC-derived SSR markers,<sup><a reveal-id="DSR017C7" data-open="DSR017C7" class="link link-ref link-reveal xref-bibr">7</a></sup> EST–SSRs and IP markers<sup><a reveal-id="DSR017C6" data-open="DSR017C6" class="link link-ref link-reveal xref-bibr">6</a></sup> containing representative blocks were highlighted by bold strokes. New EST-derived SSR markers are underlined. The colorful rectangular bars on left of the integrated linkage map indicates the crucifer building blocks homologous to the <em>Arabidopsis thaliana</em> (At) chromosomes (C1–C5).<sup><a reveal-id="DSR017C53" data-open="DSR017C53" class="link link-ref link-reveal xref-bibr">53</a></sup></p></div></div><div content-id="DSR017F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr017/2/m_dsr01701a.jpeg?Expires=2147483647&amp;Signature=KWsu-gBIZUnI~I-NmHjjFyFcE-gQseNqFLROtyqVGJuPyVJwJvWroeS5gzh7wkCMwd7YPu2xGVFoiJVZfLS70f-R8k1HdbV~eRsW4pqTIpspWf~4tZtpUcmQ8Fn2Cvza3BM4VbyTnMAnFKsx03jTKqj~hSaaZk6cHvJuJI80MWtcwGGVRbZO7BHleX8NzeyzcCvVP17ewRdBmk3QOOaa3A9pnDV6QTJZH9VqKInF0FrC-ygd-4P82fWWDaR3WQb1XGLAYFvyRAi0RnPQxq3I5GGkFH0gZvBGMjMnLJqlGQxVGWCtSLaCO6XNGMZDSQLSysjbroS3HzLrEssUnZdmhw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The distribution of EST–SSRs and other markers in the 10 integrated linkage groups (A1–A10) of Brassica rapa. BAC-derived SSR markers,7 EST–SSRs and IP markers6 containing representative blocks were highlighted by bold strokes. New EST-derived SSR markers are underlined. The colorful rectangular bars on left of the integrated linkage map indicates the crucifer building blocks homologous to the Arabidopsis thaliana (At) chromosomes (C1–C5).53" path-from-xml="dsr01701a.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6714147/dsr01701a.jpeg" path-from-xml="dsr01701a.jpeg" target="_blank">View large</a><a section="6714147" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr017/2/dsr01701a.jpeg?Expires=2147483647&amp;Signature=xnwWY-UV-GG61Z1-G5kwzscVl4tADYwVXmHw726AJ5QO-A3je15IBNV9NNi~H3m-iM1g61A4ct6YbLr~YeyuVLgkhb1gE24mV3N9Fnse3EzzTUnGgbq0n8nxSM6g2dO3MIKe3iJub4kMB7-NKoi3Y~uWm0vCK4GHmNBiPb4saubFpJgXgRPMxzR1u52Q1j9fAM1yVQBO5Znf7W1C6dsIBnpDImgSutkl8etq0OfT0x1xFEQHrEjIrgp3r1-FjYMP9ctxyydfBbeRtIx-yyuKYd~Bo3YaYd-psqBmgLtQFFGyZ1rl0EElghM7Xng8FNnHW~cpLI3SBqszy~R~gj5Y1A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=6714147&amp;ar=448180&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsr01701a.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr017/2/m_dsr01701b.jpeg?Expires=2147483647&amp;Signature=Gz66vR8vLRbzzC2bflqiZjRv~HQhm4gU2F~mPrRU-DxjfvVjCE0f7I~tezmA6p8IRFZNpKEkg6uq4ezhgArbabcGZFyUN3I3YtxN0MGWnSkbjcSkIxBUO7iVBjSEo9Ze3avQqiSfEhaDfpCTN7hB0fPLclTLBdxgIow7dGLvDILGATCJrCMMLIXm-4gDOKKeStn3ZOReBIZtskO7FL13EA2MiniA2dIRuzqO9-~hXYyqwy-kZvPzzGYJIltR5SFCsK39Fh6jaBJ3rLyFYfXA8UPsc7rwlCfEXMcbw0XAQ5HCsoqUgvZkUWSdbEheYsBl1YbsZ7Vr~hJeU3zmCfKzXA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The distribution of EST–SSRs and other markers in the 10 integrated linkage groups (A1–A10) of Brassica rapa. BAC-derived SSR markers,7 EST–SSRs and IP markers6 containing representative blocks were highlighted by bold strokes. New EST-derived SSR markers are underlined. The colorful rectangular bars on left of the integrated linkage map indicates the crucifer building blocks homologous to the Arabidopsis thaliana (At) chromosomes (C1–C5).53" path-from-xml="dsr01701b.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6714147/dsr01701b.jpeg" path-from-xml="dsr01701b.jpeg" target="_blank">View large</a><a section="6714147" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr017/2/dsr01701b.jpeg?Expires=2147483647&amp;Signature=K9G-V7apmR6KmeG3SMsiZ9qScXJg3PKFqEonWZfSZTCT-tdHYmIisTmUslVz3Tl7eFPDrQ0roG5i0xyF5sLKIG5TVhq~0Z8ERnVisBLTFt6brTjadA5Xmnig0s8RcYTzDhQ4bsdgQTkpNyHHPS7aai7t-EoY2HZRDNhCIKCETtWMNeO1fIAN-oOhFzGpFQLnXwiKIxxTAvjuDcDId0y5TBNrcgXHBPHIDM15geBY2GPX8RTEkXirMJa3fXhZqLybyF0Y9Q0B1wJpCm~V6Uuq5DvjZN6HEP6a-9mrXFO4LreS5~8YYCg9PgVRAZjx-7N33g4RtR8Ebhy1R5tv0qW8Pw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=6714147&amp;ar=448180&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsr01701b.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr017/2/m_dsr01701c.jpeg?Expires=2147483647&amp;Signature=iNQgG9gtxI22z41hzQFvK99q1dN0HXkuI1swI588XBlOVprlLFLb6lIeDmRqdyqzPMVBzsStfCgTKfzxORh5HZXtdRAFuQDwGigEu4JGj6060qbcWo1ytCxTR6ePN~iOPwM55AMtx35ZD-ucLL1Ek0k5RtRMnlWVBPJ7ZqGrEli1Gas5f8tbPXJKRqEX3keNEPWiH3ItiJzAWtAQ3UzxI~Nbl9mVEXqQjrjvF9mge4Rc2ydUPGYLaENE-cHabUE7diEsSMXkWZERznaVQWQhn66J50XHJguTdk-Ct9a7HSqnfrB2E1MocQh-T5L1A1efOF-zizpjMIc7hHXlBkJofw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The distribution of EST–SSRs and other markers in the 10 integrated linkage groups (A1–A10) of Brassica rapa. BAC-derived SSR markers,7 EST–SSRs and IP markers6 containing representative blocks were highlighted by bold strokes. New EST-derived SSR markers are underlined. The colorful rectangular bars on left of the integrated linkage map indicates the crucifer building blocks homologous to the Arabidopsis thaliana (At) chromosomes (C1–C5).53" path-from-xml="dsr01701c.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6714147/dsr01701c.jpeg" path-from-xml="dsr01701c.jpeg" target="_blank">View large</a><a section="6714147" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr017/2/dsr01701c.jpeg?Expires=2147483647&amp;Signature=EVqwTivz744geIJwQE5fXLX~mO89Wygct3Wq-YhAUDpOPSbLxDRVwjhFwM~N84hkf4f8lMF2euFAKSkCQS3a1Tl0uYcaMwvHbGUm2qsmgWZRyfTEWjL5Rp00~P4HZU8X3Mt5Jofqh3VRDFQQ1dyeQ-Kpc37jw~pGg4Ug1cf-4wch8nC~WeILcDBjEDIGj~g0RSj~ZIkkTJIn32xsIQfw1zDZRkV6yZvsB7msQWSU9bhbmbP-JCrPecVFPOa7MyMv5xs~jfQJk5YFY~h84wOGsDAOLjBS-z01Tu-pPSwSWxIc57b8IJmLLcIR1TAWjEg-9IBvX1hwCNFre9b4pMiNlg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=6714147&amp;ar=448180&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsr01701c.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The distribution of EST–SSRs and other markers in the 10 integrated linkage groups (A1–A10) of <em>Brassica rapa</em>. BAC-derived SSR markers,<sup><a reveal-id="DSR017C7" data-open="DSR017C7" class="link link-ref link-reveal xref-bibr">7</a></sup> EST–SSRs and IP markers<sup><a reveal-id="DSR017C6" data-open="DSR017C6" class="link link-ref link-reveal xref-bibr">6</a></sup> containing representative blocks were highlighted by bold strokes. New EST-derived SSR markers are underlined. The colorful rectangular bars on left of the integrated linkage map indicates the crucifer building blocks homologous to the <em>Arabidopsis thaliana</em> (At) chromosomes (C1–C5).<sup><a reveal-id="DSR017C53" data-open="DSR017C53" class="link link-ref link-reveal xref-bibr">53</a></sup></p></div></div>                            <h3 scrollto-destination=6714148 id="6714148" class="section-title" data-legacy-id=s3d>3.4. Identification of crucifer building blocks</h3>
<p>With the addition of more markers, we could accurately resolve the <em>B. rapa</em> genome for collinearity blocks with <em>A. thaliana</em> chromosomes.<sup><a reveal-id="DSR017C53" data-open="DSR017C53" class="link link-ref link-reveal xref-bibr">53</a></sup> Blast analysis of <em>B. rapa</em> BAC and EST sequences, wherefrom the SSRs were identified, helped to identify homologous <em>Arabidopsis</em> chromosomal blocks which, in turn, helped to identify conserved ancestral crucifer building blocks in the <em>B. rapa</em> genome (Fig. <a reveal-id="DSR017F1" data-open="DSR017F1" class="link link-reveal link-table xref-fig">1</a>).<sup><a reveal-id="DSR017C53" data-open="DSR017C53" class="link link-ref link-reveal xref-bibr">53</a></sup> We could confidently establish homologous blocks that were previously identified with slightly less stringent criteria due to low marker density,<sup><a reveal-id="DSR017C33" data-open="DSR017C33" class="link link-ref link-reveal xref-bibr">33</a></sup> such as W and E blocks in linkage group A2, the B and C blocks in A5, the Q block in A6, the T block in A8, and the W block in A10. Furthermore, as the blocks were already identified in the corresponding homologous A genome chromosomes of <em>B. juncea</em> and <em>B. napus,</em> we could identify new blocks containing mostly one marker in the <em>B. rapa</em> A genome, and these could be regarded as probable blocks since the available evidence supports the highly conserved nature of the A chromosomes at gross level among the three species, despite their divergence a long time ago.<sup><a reveal-id="DSR017C4" data-open="DSR017C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSR017C6" data-open="DSR017C6" class="link link-ref link-reveal xref-bibr">6</a></sup> The new probable conserved blocks identified in this study are the M block in linkage group A1, the V block in A2, the X block in A6 and A9, and the I block in A7 (Fig. <a reveal-id="DSR017F1" data-open="DSR017F1" class="link link-reveal link-table xref-fig">1</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). However, further addition of markers is necessary to accurately resolve these blocks, even though we used already established evidence from the corresponding A genome chromosome blocks in <em>B. napus</em> and <em>B. juncea</em> while considering these blocks.</p>                            <h3 scrollto-destination=6714150 id="6714150" class="section-title" data-legacy-id=s3e>3.5. Evaluation of the mapped EST-derived SSR markers for allelic diversity</h3>
<p>We selected 238 EST-SSRs (234 ACMP and 4 sau-um) that mapped to different <em>B. rapa</em> linkage groups to study their allelic diversity in 24 <em>B. rapa</em> genotypes belonging to the sub-species <em>oleifera</em> (oil type), <em>pekinensis</em> (Chinese cabbage), and <em>chinensis</em> (Pakchoi) type. <em>Brassica rapa</em> plants belonging to these sub-species are morphologically different with respect to leaf types, heading habits, and overall plant morphology. The PCR successfully amplified all of the 238 EST-SSRs markers in the 24 <em>B. rapa</em> genomes. The number of alleles amplified per EST-SSR ranged from 2 to 8, with an average of 2.9 (Fig. <a reveal-id="DSR017F2" data-open="DSR017F2" class="link link-reveal link-table xref-fig">2</a>, Table <a reveal-id="DSR017TB4" data-open="DSR017TB4" class="link link-reveal link-table xref-fig">4</a>). The majority of SSRs consisted of two alleles. The PIC value, which is a measure of allelic diversity, varied from 0.08 to  0.65, with an average of 0.40. The most commonly observed PIC values were between 0.4 and 0.5 (Table <a reveal-id="DSR017TB4" data-open="DSR017TB4" class="link link-reveal link-table xref-fig">4</a>). The highest average PIC value was detected in the linkage group A1 (0.45) and the lowest one was in A9 (0.36). </p>                        <a id="6714152" scrollto-destination="6714152"></a>
<div content-id="DSR017TB4" class="table-modal"><div class="table-wrap"><div id="DSR017TB4" class="table-wrap-title" data-id="DSR017TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Average number of alleles and the PIC values calculated for the mapped EST-derived SSRs in 10 linkage groups of <em>B. rapa</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Linkage group </th><th>EST-markers number </th><th>Average PIC value<sup>a</sup> </th><th>Average number of alleles </th></tr></thead><tbody><tr><td>A1 </td><td>17 </td><td>0.45 </td><td>3.7 </td></tr><tr><td>A2 </td><td>12 </td><td>0.40 </td><td>3.0 </td></tr><tr><td>A3 </td><td>40 </td><td>0.37 </td><td>2.8 </td></tr><tr><td>A4 </td><td>22 </td><td>0.40 </td><td>3.0 </td></tr><tr><td>A5 </td><td>33 </td><td>0.42 </td><td>2.7 </td></tr><tr><td>A6 </td><td>29 </td><td>0.41 </td><td>2.7 </td></tr><tr><td>A7 </td><td>28 </td><td>0.40 </td><td>2.6 </td></tr><tr><td>A8 </td><td>13 </td><td>0.37 </td><td>2.8 </td></tr><tr><td>A9 </td><td>31 </td><td>0.36 </td><td>2.5 </td></tr><tr><td>A10 </td><td>13 </td><td>0.44 </td><td>2.8 </td></tr><tr><td>Total </td><td>238 </td><td>0.40 </td><td>2.9 </td></tr></tbody></table></div><div content-id="DSR017TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Linkage group </th><th>EST-markers number </th><th>Average PIC value<sup>a</sup> </th><th>Average number of alleles </th></tr></thead><tbody><tr><td>A1 </td><td>17 </td><td>0.45 </td><td>3.7 </td></tr><tr><td>A2 </td><td>12 </td><td>0.40 </td><td>3.0 </td></tr><tr><td>A3 </td><td>40 </td><td>0.37 </td><td>2.8 </td></tr><tr><td>A4 </td><td>22 </td><td>0.40 </td><td>3.0 </td></tr><tr><td>A5 </td><td>33 </td><td>0.42 </td><td>2.7 </td></tr><tr><td>A6 </td><td>29 </td><td>0.41 </td><td>2.7 </td></tr><tr><td>A7 </td><td>28 </td><td>0.40 </td><td>2.6 </td></tr><tr><td>A8 </td><td>13 </td><td>0.37 </td><td>2.8 </td></tr><tr><td>A9 </td><td>31 </td><td>0.36 </td><td>2.5 </td></tr><tr><td>A10 </td><td>13 </td><td>0.44 </td><td>2.8 </td></tr><tr><td>Total </td><td>238 </td><td>0.40 </td><td>2.9 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Polymorphic information content (PIC).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6714152" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR017TB4" class="table-wrap-title" data-id="DSR017TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Average number of alleles and the PIC values calculated for the mapped EST-derived SSRs in 10 linkage groups of <em>B. rapa</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Linkage group </th><th>EST-markers number </th><th>Average PIC value<sup>a</sup> </th><th>Average number of alleles </th></tr></thead><tbody><tr><td>A1 </td><td>17 </td><td>0.45 </td><td>3.7 </td></tr><tr><td>A2 </td><td>12 </td><td>0.40 </td><td>3.0 </td></tr><tr><td>A3 </td><td>40 </td><td>0.37 </td><td>2.8 </td></tr><tr><td>A4 </td><td>22 </td><td>0.40 </td><td>3.0 </td></tr><tr><td>A5 </td><td>33 </td><td>0.42 </td><td>2.7 </td></tr><tr><td>A6 </td><td>29 </td><td>0.41 </td><td>2.7 </td></tr><tr><td>A7 </td><td>28 </td><td>0.40 </td><td>2.6 </td></tr><tr><td>A8 </td><td>13 </td><td>0.37 </td><td>2.8 </td></tr><tr><td>A9 </td><td>31 </td><td>0.36 </td><td>2.5 </td></tr><tr><td>A10 </td><td>13 </td><td>0.44 </td><td>2.8 </td></tr><tr><td>Total </td><td>238 </td><td>0.40 </td><td>2.9 </td></tr></tbody></table></div><div content-id="DSR017TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Linkage group </th><th>EST-markers number </th><th>Average PIC value<sup>a</sup> </th><th>Average number of alleles </th></tr></thead><tbody><tr><td>A1 </td><td>17 </td><td>0.45 </td><td>3.7 </td></tr><tr><td>A2 </td><td>12 </td><td>0.40 </td><td>3.0 </td></tr><tr><td>A3 </td><td>40 </td><td>0.37 </td><td>2.8 </td></tr><tr><td>A4 </td><td>22 </td><td>0.40 </td><td>3.0 </td></tr><tr><td>A5 </td><td>33 </td><td>0.42 </td><td>2.7 </td></tr><tr><td>A6 </td><td>29 </td><td>0.41 </td><td>2.7 </td></tr><tr><td>A7 </td><td>28 </td><td>0.40 </td><td>2.6 </td></tr><tr><td>A8 </td><td>13 </td><td>0.37 </td><td>2.8 </td></tr><tr><td>A9 </td><td>31 </td><td>0.36 </td><td>2.5 </td></tr><tr><td>A10 </td><td>13 </td><td>0.44 </td><td>2.8 </td></tr><tr><td>Total </td><td>238 </td><td>0.40 </td><td>2.9 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Polymorphic information content (PIC).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6714152" target="_blank">View Large</a></div></div>                        <a id="6714153" scrollto-destination="6714153"></a>
<div id="DSR017F2" data-id="DSR017F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr017/2/m_dsr01702.jpeg?Expires=2147483647&Signature=zljO9A3ntJxZlv0o8R2zHrowQo8XP-VwaColecr6bxtq~0T0KZepVboIp~kQG38JKDLyX2UkR9~23812yVkho55m3XO~OTVsCmIbljdcQ8nrfteZpqr3sg8WwKfcfXuY2tqO24XrWxhm0VDoOM3wn3uWakvpo8m00d3Ue5SiCG-hA8N9eFxp3rTQujbny7xQTcrc5RH9kte3ozkR86YzuhUWlq-sBm9aL0KaqVZSeK6FOdgzCnoy6QUgJHpB3XXPT9r8BErhCEaW2hmwOtCpJjpytI31U859u9MuT2sKQh9e85xsh4Cx2XA-1nb~f7fRKioObXslHCCuvB8DcuSwvQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The distribution of PIC values and allele frequencies calculated from 238 EST-SSRs in 24 B. rapa germplasm." path-from-xml="dsr01702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6714153/dsr01702.jpeg" path-from-xml="dsr01702.jpeg" target="_blank">View large</a><a section="6714153" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr017/2/dsr01702.jpeg?Expires=2147483647&Signature=yClGTsx4wfodEcJgCV0XCvZVRXNM8Ptw1bL~GJQxcpW1wi9~uBB~AYWq8qhPsSY1xUj-s9-QA0c3aub4NKL1fhsfbTxXUZdo4HFCLFppTCXdzoBKHbxqTw0CO~20FBw3e1CpWvRlo3GK3h5wIAYFtSJWOM5D~cQHS~gtT7zxwRtLXSNGig0UVZOGIZfCNDtIWMxTWBUmNRFA6eMONboI0xtpk03pBodnmgSWKFU4bHrBruSbKSMm2Wdwx12BxW6KYSSya7gGrc-w5saYPMGB7MCYchITeWkAP9X9xfz6OlO26tKkdG3YcI~Y83HWyPAPea6LD2K1DxQR7GItiZbo8A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6714153&ar=448180&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The distribution of PIC values and allele frequencies calculated from 238 EST-SSRs in 24 <em>B. rapa</em> germplasm.</p></div></div><div content-id="DSR017F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr017/2/m_dsr01702.jpeg?Expires=2147483647&Signature=zljO9A3ntJxZlv0o8R2zHrowQo8XP-VwaColecr6bxtq~0T0KZepVboIp~kQG38JKDLyX2UkR9~23812yVkho55m3XO~OTVsCmIbljdcQ8nrfteZpqr3sg8WwKfcfXuY2tqO24XrWxhm0VDoOM3wn3uWakvpo8m00d3Ue5SiCG-hA8N9eFxp3rTQujbny7xQTcrc5RH9kte3ozkR86YzuhUWlq-sBm9aL0KaqVZSeK6FOdgzCnoy6QUgJHpB3XXPT9r8BErhCEaW2hmwOtCpJjpytI31U859u9MuT2sKQh9e85xsh4Cx2XA-1nb~f7fRKioObXslHCCuvB8DcuSwvQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The distribution of PIC values and allele frequencies calculated from 238 EST-SSRs in 24 B. rapa germplasm." path-from-xml="dsr01702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6714153/dsr01702.jpeg" path-from-xml="dsr01702.jpeg" target="_blank">View large</a><a section="6714153" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr017/2/dsr01702.jpeg?Expires=2147483647&Signature=yClGTsx4wfodEcJgCV0XCvZVRXNM8Ptw1bL~GJQxcpW1wi9~uBB~AYWq8qhPsSY1xUj-s9-QA0c3aub4NKL1fhsfbTxXUZdo4HFCLFppTCXdzoBKHbxqTw0CO~20FBw3e1CpWvRlo3GK3h5wIAYFtSJWOM5D~cQHS~gtT7zxwRtLXSNGig0UVZOGIZfCNDtIWMxTWBUmNRFA6eMONboI0xtpk03pBodnmgSWKFU4bHrBruSbKSMm2Wdwx12BxW6KYSSya7gGrc-w5saYPMGB7MCYchITeWkAP9X9xfz6OlO26tKkdG3YcI~Y83HWyPAPea6LD2K1DxQR7GItiZbo8A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6714153&ar=448180&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The distribution of PIC values and allele frequencies calculated from 238 EST-SSRs in 24 <em>B. rapa</em> germplasm.</p></div></div>                            <h3 scrollto-destination=6714154 id="6714154" class="section-title" data-legacy-id=s3f>3.6. Transferability of EST-SSRs in other cultivated and wild Brassica relatives</h3>
<p>To assess the utility of EST-SSR loci across other cultivated and wild relatives of <em>Brassica</em> species, 167 EST-SSRs (146 EST-SSRs mapped and 21 EST-SSRs that were not mapped in the 10 linkage groups of <em>B. rapa</em>) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a><strong>)</strong> were used to amplify 48 germplasm belonging to 35 species of Brassicaceae family which includes five genotypes from <em>B. rapa</em>, two each from <em>B. oleracea</em>, <em>B. nigra</em>, <em>B. napus, B. carinata</em> and <em>Arabidopsis thaliana</em>, five from <em>B. juncea</em> and wild relatives (Table <a reveal-id="DSR017TB1" data-open="DSR017TB1" class="link link-reveal link-table xref-fig">1</a> and Fig. <a reveal-id="DSR017F3" data-open="DSR017F3" class="link link-reveal link-table xref-fig">3</a>). Of the 167 primer pairs used to amplify in germplasm, cultivated <em>Brassica</em> species other than <em>B. rapa</em> showing 100% amplification were <em>B. carinata, B. juncea</em> and <em>B. napus</em>. Only one primer pair was not amplified in <em>B. oleracea</em>, while 17 primer pairs did not amplify in <em>B. nigra</em>. In <em>Arabidopsis thaliana</em>, 52 primer pairs did not show amplification. Among the wild species under <em>Brassica</em> genus, highest number of transferability was found in <em>B. incana</em> (amplified 161 out of 167 primer pairs) followed by <em>B. bourgeaui</em> and <em>B. insularis</em> (158 amplified in both the species). <em>Brassica balearica</em> DNA showed lowest number of primer pair amplification suggesting less cross-species transferable among them. Among the wild relative species, <em>Herba chichori</em> showed the highest number of amplification (163) and the lowest number for amplification was shown by <em>Sysimprium leteum</em> (71 primer pairs) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). </p>                        <a id="6714156" scrollto-destination="6714156"></a>
<div id="DSR017F3" data-id="DSR017F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr017/2/m_dsr01703.jpeg?Expires=2147483647&Signature=1X60NcV43kHZdUsaM76LD6RntEoJgEskkvSlcvluLyPWhuCDLfD-2H-ys208RZ5KY22Xai~7CPwTlNyPJLUv3mwjQR4VdJwl5EpCMSWpX3g24s8a8kuBKqXV-Hhba6Ji~OUxHpgN33Wg~GIOkv~Qe-u5AftwlvcQHoXO2CHWtaP6Bz1QLEhVsp4JybaWhxbCkIelNTnngR0wCNBOvYjUh~nPqQs~TlZYOuufuGmPSe8c3AXaY49QYt6F6zpJsfWfj8ilV~NxeWGzqCzbDTUwu-4dDe4hG1HGEM1XetUXtZWJEdb6haln2EVJfSbZAFuvJuXP-1NxsZJkV4YT80AI3w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Representative polyacrylamide gel picture of different alleles amplified with the primer ACMP00321 in 48 genotypes belonging to 35 different species. Serial numbers: 1–5, B. rapa; 6–7, B. oleracea; 8–9, B. nigra; 10–11, B. carinata; 12–13, B. napus, 14–17, B. juncea; 18–19, A. thaliana; 20, B. tourniforti; 21, B. balearica; 22, B. barreilieri; 23, B. bivoniana; 24, B. bourgeaui; 25, B. cretica; 26, B. desnottesii; 27, B. drepanensis; 28–29, B. fruticulosa; 30, B. gravinae; 31, B. incana; 32, B. insularis; 33, B. macrocarpa; 34, B. mourorum; 35, B. repanda; 36, B. rupestris; 37, B. spinescens; 38, B. villosa; 39, Camelina sativa; 40, Herba cichori; 41, Diplotaxis muralis; 42, Eruca sativa; 43, Hesperis matronalis; 44, Moricandia arvensis; 45, Sinapis alba; 46, Raphanus sativus; 47, Sisymprium leteum; 48, Lepidium apetalum. All species are listed in Table 1." path-from-xml="dsr01703.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6714156/dsr01703.jpeg" path-from-xml="dsr01703.jpeg" target="_blank">View large</a><a section="6714156" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr017/2/dsr01703.jpeg?Expires=2147483647&Signature=H1i90f~qsep0IQaD5y20P4KxVLeksygmVV0as6ANm6k~Fk8rdfNt6KGCY64Shm-~U9iwupG-RdLNpLLVBS-ZCc96DBP9DiS59~e-G8GAUsueNJFJlvvxJmmNx7k1jZRA3BJMMFe1wRlRxLJNQOU3ilXzN1YEFcc48qXP8UZPlyTu9egMy-PHyN6p7Fwo7xSYWZskzWyR5pHh-fldXcG5lnH5W6Kq-rKMNELvqBdCtFnwJPxanbbDP~pm~nQ1siNMralrihjLczkR9ID0jE7-QiPxqn~x6kU93JrLZqM2-QT026Xia13Cw0uExtIdWLIkVspgZrdoogU3t75h-7Mhqg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6714156&ar=448180&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01703.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Representative polyacrylamide gel picture of different alleles amplified with the primer ACMP00321 in 48 genotypes belonging to 35 different species. Serial numbers: 1–5, <em>B. rapa</em>; 6–7, <em>B. oleracea</em>; 8–9, <em>B. nigra;</em> 10–11, <em>B. carinata</em>; 12–13, <em>B. napus,</em> 14–17, <em>B. juncea</em>; 18–19, <em>A. thaliana</em>; 20, <em>B. tourniforti</em>; 21, <em>B. balearica</em>; 22, <em>B. barreilieri</em>; 23, <em>B. bivoniana</em>; 24<em>, B. bourgeaui</em>; 25, <em>B. cretica</em>; 26<em>, B. desnottesii</em>; 27, <em>B. drepanensis</em>; 28–29, <em>B. fruticulosa</em>; 30, <em>B. gravinae</em>; 31, <em>B. incana</em>; 32, <em>B. insularis</em>; 33, <em>B. macrocarpa</em>; 34, <em>B. mourorum</em>; 35, <em>B. repanda</em>; 36, <em>B. rupestris</em>; 37, <em>B. spinescens</em>; 38, <em>B. villosa</em>; 39, <em>Camelina sativa</em>; 40, <em>Herba cichori</em>; 41<em>, Diplotaxis muralis</em>; 42, <em>Eruca sativa</em>; 43, <em>Hesperis matronalis</em>; 44, <em>Moricandia arvensis</em>; 45, <em>Sinapis alba</em>; 46, <em>Raphanus sativus</em>; 47, <em>Sisymprium leteum</em>; 48, <em>Lepidium apetalum.</em> All species are listed in Table <a reveal-id="DSR017TB1" data-open="DSR017TB1" class="link link-reveal link-table xref-fig">1</a>.</p></div></div><div content-id="DSR017F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr017/2/m_dsr01703.jpeg?Expires=2147483647&Signature=1X60NcV43kHZdUsaM76LD6RntEoJgEskkvSlcvluLyPWhuCDLfD-2H-ys208RZ5KY22Xai~7CPwTlNyPJLUv3mwjQR4VdJwl5EpCMSWpX3g24s8a8kuBKqXV-Hhba6Ji~OUxHpgN33Wg~GIOkv~Qe-u5AftwlvcQHoXO2CHWtaP6Bz1QLEhVsp4JybaWhxbCkIelNTnngR0wCNBOvYjUh~nPqQs~TlZYOuufuGmPSe8c3AXaY49QYt6F6zpJsfWfj8ilV~NxeWGzqCzbDTUwu-4dDe4hG1HGEM1XetUXtZWJEdb6haln2EVJfSbZAFuvJuXP-1NxsZJkV4YT80AI3w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Representative polyacrylamide gel picture of different alleles amplified with the primer ACMP00321 in 48 genotypes belonging to 35 different species. Serial numbers: 1–5, B. rapa; 6–7, B. oleracea; 8–9, B. nigra; 10–11, B. carinata; 12–13, B. napus, 14–17, B. juncea; 18–19, A. thaliana; 20, B. tourniforti; 21, B. balearica; 22, B. barreilieri; 23, B. bivoniana; 24, B. bourgeaui; 25, B. cretica; 26, B. desnottesii; 27, B. drepanensis; 28–29, B. fruticulosa; 30, B. gravinae; 31, B. incana; 32, B. insularis; 33, B. macrocarpa; 34, B. mourorum; 35, B. repanda; 36, B. rupestris; 37, B. spinescens; 38, B. villosa; 39, Camelina sativa; 40, Herba cichori; 41, Diplotaxis muralis; 42, Eruca sativa; 43, Hesperis matronalis; 44, Moricandia arvensis; 45, Sinapis alba; 46, Raphanus sativus; 47, Sisymprium leteum; 48, Lepidium apetalum. All species are listed in Table 1." path-from-xml="dsr01703.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6714156/dsr01703.jpeg" path-from-xml="dsr01703.jpeg" target="_blank">View large</a><a section="6714156" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr017/2/dsr01703.jpeg?Expires=2147483647&Signature=H1i90f~qsep0IQaD5y20P4KxVLeksygmVV0as6ANm6k~Fk8rdfNt6KGCY64Shm-~U9iwupG-RdLNpLLVBS-ZCc96DBP9DiS59~e-G8GAUsueNJFJlvvxJmmNx7k1jZRA3BJMMFe1wRlRxLJNQOU3ilXzN1YEFcc48qXP8UZPlyTu9egMy-PHyN6p7Fwo7xSYWZskzWyR5pHh-fldXcG5lnH5W6Kq-rKMNELvqBdCtFnwJPxanbbDP~pm~nQ1siNMralrihjLczkR9ID0jE7-QiPxqn~x6kU93JrLZqM2-QT026Xia13Cw0uExtIdWLIkVspgZrdoogU3t75h-7Mhqg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6714156&ar=448180&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01703.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Representative polyacrylamide gel picture of different alleles amplified with the primer ACMP00321 in 48 genotypes belonging to 35 different species. Serial numbers: 1–5, <em>B. rapa</em>; 6–7, <em>B. oleracea</em>; 8–9, <em>B. nigra;</em> 10–11, <em>B. carinata</em>; 12–13, <em>B. napus,</em> 14–17, <em>B. juncea</em>; 18–19, <em>A. thaliana</em>; 20, <em>B. tourniforti</em>; 21, <em>B. balearica</em>; 22, <em>B. barreilieri</em>; 23, <em>B. bivoniana</em>; 24<em>, B. bourgeaui</em>; 25, <em>B. cretica</em>; 26<em>, B. desnottesii</em>; 27, <em>B. drepanensis</em>; 28–29, <em>B. fruticulosa</em>; 30, <em>B. gravinae</em>; 31, <em>B. incana</em>; 32, <em>B. insularis</em>; 33, <em>B. macrocarpa</em>; 34, <em>B. mourorum</em>; 35, <em>B. repanda</em>; 36, <em>B. rupestris</em>; 37, <em>B. spinescens</em>; 38, <em>B. villosa</em>; 39, <em>Camelina sativa</em>; 40, <em>Herba cichori</em>; 41<em>, Diplotaxis muralis</em>; 42, <em>Eruca sativa</em>; 43, <em>Hesperis matronalis</em>; 44, <em>Moricandia arvensis</em>; 45, <em>Sinapis alba</em>; 46, <em>Raphanus sativus</em>; 47, <em>Sisymprium leteum</em>; 48, <em>Lepidium apetalum.</em> All species are listed in Table <a reveal-id="DSR017TB1" data-open="DSR017TB1" class="link link-reveal link-table xref-fig">1</a>.</p></div></div><p>A total of 1326 SSR alleles were scored from 167 EST-SSRs primer pairs with an average of 7.94 alleles per primer pair across all 48 genotypes representing 35 species. The primer pair of ACMP00904 produced the highest number of alleles (17 bands), while two primer pairs namely ACMP00561 and ACMP00832 produced the lowest number of alleles (three bands) among the 48 germplasm. Most of the primer pairs showed the amplification of 5–6 alleles followed by primer pairs amplifying 7–8 and 9–10 alleles, respectively. There was a wide variation among species in the average number of alleles per primer pairs.</p>                            <h4 scrollto-destination=6714158 id="6714158" class="section-title" data-legacy-id=s3f1>3.7. Sequence level comparison of SSR locus</h4>
<p>To validate the sequence level conservation of SSRs across the species, at least one amplified product of EST-SSR ACMP00321 from 31 different species were cloned and sequenced. The EST-SSR primer pair for ACMP00321 was designed flanking four tri-nucleotide repeats (GAA). Although sequence alignment showed high conservation of nucleotide sequences from 31 different species, several single nucleotide substitutions and InDel polymorphism of 3-bp to 11-bp flanking the microsatellite repeats were observed besides variation in number of repeat motifs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). Variations of repeat numbers were observed from four repeats to a maximum of six repeats. <em>Brassica barreilieri</em> and <em>B. bourgeaui</em> showed six numbers of repeats, while <em>B. creteca, B. incana</em>, and <em>Camelina Sativa</em> showed five numbers of repeats. Remaining species showed conserved four repeats.</p>                            <h2 scrollto-destination=6714160 id="6714160" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>EST-SSRs are highly useful in molecular plant breeding and evolutionary studies since these markers are developed from transcribed region of the genes and are highly cross-species transferable. Although, recent study reported the development EST-SSR markers in <em>B. rapa</em>,<sup><a reveal-id="DSR017C24" data-open="DSR017C24" class="link link-ref link-reveal xref-bibr">24</a></sup> the exact genomic location of the developed markers in the <em>B. rapa</em> genome were not determined by mapping, and there remained many more EST-SSRs to be identified owing to the large number of genes present in <em>B. rapa</em> genome (expected gene number is over 40 000, MBrGSP, personal communication). Therefore, in this study, we developed 707 new EST-SSRs, of which 691 (97.73%) primer pairs were successfully amplified the DNA fragments of expected sizes in the <em>B. rapa</em> genome. The majority of these EST-SSR markers were single locus markers, while only a few of them were duplicated and mapped to different linkage groups.</p><p>The EST-SSR markers were less polymorphic than the genomic SSRs as they were derived from genic regions. Of the 707 EST-SSRs, only 99 (14%) were polymorphic between the parental lines Chiifu-401 and Kenshin, whereas 311 SSRs (41.5%) from 749 BAC-SSRs were polymorphic.<sup><a reveal-id="DSR017C7" data-open="DSR017C7" class="link link-ref link-reveal xref-bibr">7</a></sup> For the CRF<sub>2</sub> population, 20% of the EST-SSRs were polymorphic between the parental lines Chiifu-401 and RCBr, which is also lower than for genomic SSRs (67%). Similarly, reduced levels of polymorphisms were detected between CR Shinki and 94SK (14%, the parental lines of the CSKF<sub>2</sub> population) and between 501 and 601 (15.56%, the parental lines of the PF<sub>2</sub> population). Similar results were also reported in pearl millet<sup><a reveal-id="DSR017C30" data-open="DSR017C30" class="link link-ref link-reveal xref-bibr">30</a></sup> and soybean.<sup><a reveal-id="DSR017C20" data-open="DSR017C20" class="link link-ref link-reveal xref-bibr">20</a></sup> These findings indicate that the SSRs located in coding regions were relatively more conserved than those in non-coding regions. Despite showing lower levels of polymorphisms, the EST-derived SSRs have much more potential than genomic SSRs to reveal the functional variation between individuals.</p><p>The updated high-density integrated map of <em>B. rapa</em> now contains a total of 1426 markers that include 414 new markers (306 EST-SSRs, 55 BAC-SSR, 10 IP, and 40 public SSR markers, 1 phenotypic marker, and 2 SCAR markers) and covers a total length of 1245.9 cM, which is almost similar to the earlier map (1262.0 cM; Li <em>et al</em>.<sup><a reveal-id="DSR017C33" data-open="DSR017C33" class="link link-ref link-reveal xref-bibr">33</a></sup>). We found a slight decrease in the map length due to the addition of more markers, thereby increasing the marker density from 1.27 cM in the previous map<sup><a reveal-id="DSR017C33" data-open="DSR017C33" class="link link-ref link-reveal xref-bibr">33</a></sup> to 0.87 cM in the present one. The majority of the mapped EST-SSR markers were randomly distributed in the 10 linkage groups; however, a few were clustered in narrow regions of a few linkage groups, e.g. A3 (20–28 cM), A5 (59–62 cM), A8 (17–24 cM), and A10 (25–29 cM). These EST-SSR clusters may be indicative of gene-rich regions, although more gene-based markers are needed to fully characterize these regions.</p><p>We compared the updated integrated map and the component maps and found the overall order and positions of the markers to be same, despite observing minor local inversions within a 5 cM distance in many linkage groups, with the exception of a few markers with longer distance inversions (in A2, A3, A4, and A9). These kinds of local inversions were reported earlier in map integration in <em>Arabidopsis</em>, <em>B. oleracea</em>, lettuce, rapeseed, and many other plant species.<sup><a reveal-id="DSR017C54  DSR017C60" data-open="DSR017C54  DSR017C60" class="link link-ref link-reveal xref-bibr">54–60</a></sup> The various reasons cited were: (i) mapping inaccuracies resulted from small mapping populations,<sup><a reveal-id="DSR017C55" data-open="DSR017C55" class="link link-ref link-reveal xref-bibr">55</a></sup> (ii) closely spaced markers in one population,<sup><a reveal-id="DSR017C21" data-open="DSR017C21" class="link link-ref link-reveal xref-bibr">21</a></sup> (iii) distorted segregation of markers, and (iv) real inversions. We believe that the addition of more common markers would help solve up to certain extent the local discrepancies observed in few linkage groups due to any one or combination of above-cited reasons. However, while comparing the mapped IP markers in our map with the <em>B. juncea</em> map,<sup><a reveal-id="DSR017C6" data-open="DSR017C6" class="link link-ref link-reveal xref-bibr">6</a></sup> we observed the overall conservation of A genome chromosomes (data not shown) between the two species, which further supports the correct alignment of our map.</p><p>The addition of new BAC-SSRs, EST-SSRs, and IP markers in the updated consensus genetic map helped to confirm the accuracy of the previously identified cruciferous building blocks<sup><a reveal-id="DSR017C53" data-open="DSR017C53" class="link link-ref link-reveal xref-bibr">53</a></sup> and to identify previously unidentified probable blocks<sup><a reveal-id="DSR017C33" data-open="DSR017C33" class="link link-ref link-reveal xref-bibr">33</a></sup> in our linkage map by blast analysis against the <em>A. thaliana</em> genome sequence. We identified an additional five novel probable blocks (M in linkage group A1, V in A2, X in A6 and A9, and I in A7) compared to <em>B. napus</em><sup><a reveal-id="DSR017C4" data-open="DSR017C4" class="link link-ref link-reveal xref-bibr">4</a></sup> and <em>B. juncea</em>.<sup><a reveal-id="DSR017C6" data-open="DSR017C6" class="link link-ref link-reveal xref-bibr">6</a></sup> However, additional markers are necessary to confirm these blocks since only one or a few markers were identified and the blocks were designated on the basis of the corresponding A chromosomes of <em>B. juncea</em><sup><a reveal-id="DSR017C6" data-open="DSR017C6" class="link link-ref link-reveal xref-bibr">6</a></sup> and <em>B. napus</em>.<sup><a reveal-id="DSR017C4" data-open="DSR017C4" class="link link-ref link-reveal xref-bibr">4</a></sup></p><p>The analysis of allelic variation of the 238 mapped EST-SSRs in 24 <em>B. rapa</em> germplasm belonging to the sub-species <em>oleifera</em>, <em>chinensis</em>, and <em>pekinensis</em> revealed that the number of alleles ranged from 2 to 8. It is known that the observation of different allele frequencies of any given marker is closely related to its transferability among germplasm, as well as to the degree of variability within the marker locus.<sup><a reveal-id="DSR017C61" data-open="DSR017C61" class="link link-ref link-reveal xref-bibr">61</a></sup> However, the most frequent number of alleles amplified per EST-SSRs was 2, suggesting that these markers were probably generated by insertion–deletion polymorphisms, as previously observed in soybean.<sup><a reveal-id="DSR017C20" data-open="DSR017C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSR017C62" data-open="DSR017C62" class="link link-ref link-reveal xref-bibr">62</a></sup> We observed that the average PIC values of the mapped markers in the 10 linkage groups of <em>B. rapa</em> were 0.40 (range, 0.36–0.45), similar to the PIC of 0.40 observed in soybean by Hossain <em>et al</em>.<sup><a reveal-id="DSR017C61" data-open="DSR017C61" class="link link-ref link-reveal xref-bibr">61</a></sup> and Hisano <em>et al</em>.;<sup><a reveal-id="DSR017C62" data-open="DSR017C62" class="link link-ref link-reveal xref-bibr">62</a></sup> furthermore, they reported that the average PIC values of EST-derived markers were lower than for random genomic DNA-derived markers because of the highly conserved nature of the coding regions. We also expect lower PIC values for the EST-SSRs used in the present study than for the random and BAC-derived SSRs, even though we did not examine their PIC values, as the BAC-SSRs were more polymorphic than the EST-SSRs in the parental lines.</p><p>The advantage of high cross-species transferability of EST-SSRs can be widely exploited in comparative mapping studies to see the conservation and diversification of gene order in the related species and isolate the candidate genes from the target species using candidate genes. Cross-species transferability experiment analysis of newly developed EST-SSRs in 35 other cultivated and wild <em>Brassica</em> relatives showed varying number of amplification of EST-SSRs ranging from 100% in <em>B. napus, B. juncea</em>, and <em>B. carinata</em> to 45.50% in <em>Sysimprium leteum.</em> Varying rates of transferability of EST-SSR markers among related species or genera have been demonstrated in several studies.<sup><a reveal-id="DSR017C28" data-open="DSR017C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSR017C32" data-open="DSR017C32" class="link link-ref link-reveal xref-bibr">32</a></sup> For example, all of the 61 EST-SSRs markers developed in <em>Camellia sinensis</em> were fully transferable to <em>Camellia assamica</em> and <em>Camellia assamica</em> ssp. <em>Lasiocalyx</em>, while it showed various rates of transferability to <em>C. lutescens</em>; <em>C. irrawadiensis</em>, and <em>C. japonic</em>a.<sup><a reveal-id="DSR017C25" data-open="DSR017C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Raji <em>et al</em>.<sup><a reveal-id="DSR017C32" data-open="DSR017C32" class="link link-ref link-reveal xref-bibr">32</a></sup> found cross-species amplification of 94% of cassava EST-SSRs in related wild species, while amplification of up to 100% of EST-SSRs designed in barley was observed in <em>Hordeum chilense</em>, and 76–100% amplification in different <em>Triticum</em> species.<sup><a reveal-id="DSR017C28" data-open="DSR017C28" class="link link-ref link-reveal xref-bibr">28</a></sup> Further, comparison of DNA sequence from 31 species showed highly identical nature of the DNA sequences with only few SNPs and Indels among the species suggesting the highly conserved nature of the gene sequences even among the distantly related species. Although variation in repeat motifs was observed, Indels surrounding the SSR motifs were the main reason for getting sequence polymorphism between the species beside few SNPs observed throughout the sequences.</p><p>The development of an updated high-density integrated <em>B. rapa</em> genetic linkage map based on highly cross-species transferable gene-based markers such as EST-SSRs and IP markers would allow comparative genetic analysis between <em>B. rapa</em> and other <em>Brassica</em> crops. Further, the mapping of previously developed public SSRs in our genetic map would also facilitate the identification of candidate genes through the fine mapping of candidate QTL/gene regions for important traits as our map contains BAC-SSRs, EST-SSRs, and IP markers adjacent along with those markers. These could then be used for the marker-assisted selection of economically important trait loci/QTL in the respective populations.<sup><a reveal-id="DSR017C41" data-open="DSR017C41" class="link link-ref link-reveal xref-bibr">41</a>,<a reveal-id="DSR017C43" data-open="DSR017C43" class="link link-ref link-reveal xref-bibr">43</a>,<a reveal-id="DSR017C44" data-open="DSR017C44" class="link link-ref link-reveal xref-bibr">44</a>,<a reveal-id="DSR017C47" data-open="DSR017C47" class="link link-ref link-reveal xref-bibr">47</a></sup> The soon-to-be available draft genome sequence of the euchromatic regions of the <em>B. rapa</em> genome does not diminish the importance of genetic linkage maps, as our map will facilitate the alignment of gene-rich euchromatic regions with less gene-dense repetitive regions, thereby helping to confirm and refine the integration of the genetic map with genomic sequences. The large number of gene-based markers mapped in the present study will promote the understanding of the comparative analysis of the <em>Brassica</em> genome at the structural and functional levels.</p>                            <h2 scrollto-destination=6714169 id="6714169" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=6714171 id="6714171" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by grants from the Technology Development Program for Agriculture and Forestry, Ministry for Food, Agriculture, Forestry, and Fisheries (grant no. 607003-05), Republic of Korea; and National Natural Science Foundation of China (Grant No. 30771468). N.R. was partially supported by the National Research Foundation, Republic of Korea.</p>                            <h2 scrollto-destination=6714173 id="6714173" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We are thankful to the Korea <em>Brassica</em> Genome Resource Bank for providing the CKDH, CRF<sub>2</sub>, and CSKF<sub>2</sub> mapping populations, Centre for Genetic Manipulation of Crop Plants, Delhi University South Campus, India; and Leibniz Institute of Plant Genetics and Crop Plant Research, Gatersleben, Germany, for different <em>Brassica</em> and wild relatives germplasm; and Ms Kyeong Ahn Lee for technical support.</p>                            <h2 scrollto-destination=6714175 id="6714175" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSR017C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="collab">UN</div><div class="article-title">Genome analysis in <em>Brassica</em> with special reference to the experimental formation of <em>B. napus</em> and peculiar mode of fertilization</div>, <div class="source">Jpn. J. Bot.</div> , <div class="year">1935</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">389</div>-<div class="lpage">452</div>)</div></div></div></div><div content-id="DSR017C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Teutonico</div> <div class="given-names">R.A.</div></div>,  <div class="name"><div class="surname">Osborn</div> <div class="given-names">T.C.</div></div>. </div><div class="article-title">Mapping of RFLP and quantitative trait loci in <em>Brassica rapa</em> and comparison to the linkage maps of <em>B. napus, B. oleracea</em> and <em>Arabidopsis thaliana</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">1994</div>, vol. <div class="volume">89</div> (pg. <div class="fpage">885</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20of%20RFLP%20and%20quantitative%20trait%20loci%20in%20Brassica%20rapa%20and%20comparison%20to%20the%20linkage%20maps%20of%20B.%20napus%2C%20B.%20oleracea%20and%20Arabidopsis%20thaliana&amp;author=R.A.%20Teutonico&amp;author=T.C.%20Osborn&amp;publication_year=1994&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=89&amp;pages=885-94" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24178100" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Snowdon</div> <div class="given-names">R.J.</div></div>,  <div class="name"><div class="surname">Friedt</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Molecular markers in Brassica oilseed breeding: current status and future possibilities</div>, <div class="source">Plant Breed</div> , <div class="year">2004</div>, vol. <div class="volume">123</div> (pg. <div class="fpage">1</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20markers%20in%20Brassica%20oilseed%20breeding%3A%20current%20status%20and%20future%20possibilities&amp;author=R.J.%20Snowdon&amp;author=W.%20Friedt&amp;publication_year=2004&amp;journal=Plant%20Breed&amp;volume=123&amp;pages=1-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbr.2004.123.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbr.2004.123.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbr.2004.123.issue-1"> </span></div> </div></div></div></div></div><div content-id="DSR017C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parkin</div> <div class="given-names">I.A.P.</div></div>,  <div class="name"><div class="surname">Gulden</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Sharpe</div> <div class="given-names">A.G.</div></div>, et al. </div><div class="article-title">Segmental structure of the <em>Brassica napus</em> genome based on comparative analysis with <em>Arabidopsis thaliana</em></div>, <div class="source">Genetics</div> , <div class="year">2005</div>, vol. <div class="volume">171</div> (pg. <div class="fpage">765</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Segmental%20structure%20of%20the%20Brassica%20napus%20genome%20based%20on%20comparative%20analysis%20with%20Arabidopsis%20thaliana&amp;author=I.A.P.%20Parkin&amp;author=S.M.%20Gulden&amp;author=A.G.%20Sharpe&amp;publication_year=2005&amp;journal=Genetics&amp;volume=171&amp;pages=765-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.105.042093" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.105.042093" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.105.042093"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16020789" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Choi</div> <div class="given-names">S.R.</div></div>,  <div class="name"><div class="surname">Teakle</div> <div class="given-names">G.R.</div></div>,  <div class="name"><div class="surname">Plaha</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">The reference genetic linkage map for the multinational <em>Brassica rapa</em> genome sequencing project</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2007</div>, vol. <div class="volume">115</div> (pg. <div class="fpage">777</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20reference%20genetic%20linkage%20map%20for%20the%20multinational%20Brassica%20rapa%20genome%20sequencing%20project&amp;author=S.R.%20Choi&amp;author=G.R.%20Teakle&amp;author=P.%20Plaha&amp;publication_year=2007&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=115&amp;pages=777-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-007-0608-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-007-0608-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-007-0608-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17646962" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Panjabi</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Jagannath</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Bisht</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Comparative mapping of <em>Brassica juncea</em> and <em>Arabidopsis thaliana</em> using Intron Polymorphism (IP) markers: homoeologous relationships, diversification and evolution of the A, B and C Brassica genomes</div>, <div class="source">BMC Genomics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">113</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20mapping%20of%20Brassica%20juncea%20and%20Arabidopsis%20thaliana%20using%20Intron%20Polymorphism%20(IP)%20markers%3A%20homoeologous%20relationships%2C%20diversification%20and%20evolution%20of%20the%20A%2C%20B%20and%20C%20Brassica%20genomes&amp;author=P.%20Panjabi&amp;author=A.%20Jagannath&amp;author=N.%20Bisht&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=113" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18315867" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kim</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Choi</div> <div class="given-names">S.R.</div></div>,  <div class="name"><div class="surname">Bae</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Sequenced BAC anchored reference genetic map that reconciles the ten individual chromosomes of <em>Brassica rapa</em></div>, <div class="source">BMC Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">432</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequenced%20BAC%20anchored%20reference%20genetic%20map%20that%20reconciles%20the%20ten%20individual%20chromosomes%20of%20Brassica%20rapa&amp;author=H.%20Kim&amp;author=S.R.%20Choi&amp;author=J.%20Bae&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=432" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-432" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-432" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-432"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19751531" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ramchiary</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Padmaja</div> <div class="given-names">K.L.</div></div>,  <div class="name"><div class="surname">Sharma</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Mapping of yield influencing QTL in <em>Brassica juncea</em>: implications for breeding of a major oilseed crop of dryland areas</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2007</div>, vol. <div class="volume">115</div> (pg. <div class="fpage">807</div>-<div class="lpage">17</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20of%20yield%20influencing%20QTL%20in%20Brassica%20juncea%3A%20implications%20for%20breeding%20of%20a%20major%20oilseed%20crop%20of%20dryland%20areas&amp;author=N.%20Ramchiary&amp;author=K.L.%20Padmaja&amp;author=S.%20Sharma&amp;publication_year=2007&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=115&amp;pages=807-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-007-0610-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-007-0610-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-007-0610-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17646960" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ramchiary</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Bisht</div> <div class="given-names">N.C.</div></div>,  <div class="name"><div class="surname">Gupta</div> <div class="given-names">V.</div></div>, et al. </div><div class="article-title">QTL analysis reveals context-dependent loci for seed glucosinolate trait in the oilseed <em>Brassica juncea</em>: importance of recurrent selection backcross scheme for the identification of ‘true’ QTL</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2007</div>, vol. <div class="volume">116</div> (pg. <div class="fpage">77</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=QTL%20analysis%20reveals%20context-dependent%20loci%20for%20seed%20glucosinolate%20trait%20in%20the%20oilseed%20Brassica%20juncea%3A%20importance%20of%20recurrent%20selection%20backcross%20scheme%20for%20the%20identification%20of%20%E2%80%98true%E2%80%99%20QTL&amp;author=N.%20Ramchiary&amp;author=N.C.%20Bisht&amp;author=V.%20Gupta&amp;publication_year=2007&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=116&amp;pages=77-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-007-0648-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-007-0648-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-007-0648-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17898985" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lou</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">J.J.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">J.S.</div></div>, et al. </div><div class="article-title">Quantitative trait loci for flowering time and morphological traits in multiple populations of <em>Brassica rapa</em></div>, <div class="source">J. Exp. Bot.</div> , <div class="year">2007</div>, vol. <div class="volume">58</div> (pg. <div class="fpage">4005</div>-<div class="lpage">16</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20trait%20loci%20for%20flowering%20time%20and%20morphological%20traits%20in%20multiple%20populations%20of%20Brassica%20rapa&amp;author=P.%20Lou&amp;author=J.J.%20Zhao&amp;author=J.S.%20Kim&amp;publication_year=2007&amp;journal=J.%20Exp.%20Bot.&amp;volume=58&amp;pages=4005-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/erm255" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferm255" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferm255"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18048374" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lou</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">He</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Quantitative trait loci for glucosinolate accumulation in <em>Brassic</em>a rapa leaves</div>, <div class="source">New Phytol.</div> , <div class="year">2008</div>, vol. <div class="volume">179</div> (pg. <div class="fpage">1017</div>-<div class="lpage">32</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20trait%20loci%20for%20glucosinolate%20accumulation%20in%20Brassica%20rapa%20leaves&amp;author=P.%20Lou&amp;author=J.%20Zhao&amp;author=H.%20He&amp;publication_year=2008&amp;journal=New%20Phytol.&amp;volume=179&amp;pages=1017-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.2008.179.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.2008.179.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.2008.179.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18564302" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lu</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Cao</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Yu</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Xiang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Mapping QTLs for root morphological traits in <em>Brassica rapa</em> L. based on AFLP and RAPD markers</div>, <div class="source">J. Appl. Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">49</div> (pg. <div class="fpage">23</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20QTLs%20for%20root%20morphological%20traits%20in%20Brassica%20rapa%20L.%20based%20on%20AFLP%20and%20RAPD%20markers&amp;author=G.%20Lu&amp;author=J.%20Cao&amp;author=X.%20Yu&amp;author=X.%20Xiang&amp;author=H.%20Chen&amp;publication_year=2008&amp;journal=J.%20Appl.%20Genet.&amp;volume=49&amp;pages=23-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF03195245" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF03195245" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF03195245"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18263966" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Kitashiba</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Inaba</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nishio</div> <div class="given-names">T.</div></div>. </div><div class="article-title">A <em>Brassica rapa</em> linkage map of EST-based SNP markers for identification of candidate genes controlling flowering time and leaf morphological traits</div>, <div class="source">DNA Res.</div> , <div class="year">2009</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">311</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20Brassica%20rapa%20linkage%20map%20of%20EST-based%20SNP%20markers%20for%20identification%20of%20candidate%20genes%20controlling%20flowering%20time%20and%20leaf%20morphological%20traits&amp;author=F.%20Li&amp;author=H.%20Kitashiba&amp;author=K.%20Inaba&amp;author=T.%20Nishio&amp;publication_year=2009&amp;journal=DNA%20Res.&amp;volume=16&amp;pages=311-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsp020" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsp020" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsp020"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19884167" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Lu</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Yuan</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Map based cloning and characterization of a gene controlling hairiness and seed coat color traits in <em>Brassica rapa</em></div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">69</div> (pg. <div class="fpage">553</div>-<div class="lpage">63</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Map%20based%20cloning%20and%20characterization%20of%20a%20gene%20controlling%20hairiness%20and%20seed%20coat%20color%20traits%20in%20Brassica%20rapa&amp;author=J.%20Zhang&amp;author=Y.%20Lu&amp;author=Y.%20Yuan&amp;publication_year=2009&amp;journal=Plant%20Mol.%20Biol.&amp;volume=69&amp;pages=553-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11103-008-9437-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11103-008-9437-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11103-008-9437-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19039665" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schranz</div> <div class="given-names">M.E.</div></div>,  <div class="name"><div class="surname">Quijada</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Sung</div> <div class="given-names">S.B.</div></div>,  <div class="name"><div class="surname">Lukens</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Amasino</div> <div class="given-names">R.M.</div></div>,  <div class="name"><div class="surname">Osborn</div> <div class="given-names">T.C.</div></div>. </div><div class="article-title">Characterization and effects of the replicated time gene <em>FLC</em> in <em>Brassica rapa</em></div>, <div class="source">Genetics</div> , <div class="year">2002</div>, vol. <div class="volume">162</div> (pg. <div class="fpage">1457</div>-<div class="lpage">68</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20and%20effects%20of%20the%20replicated%20time%20gene%20FLC%20in%20Brassica%20rapa&amp;author=M.E.%20Schranz&amp;author=P.%20Quijada&amp;author=S.B.%20Sung&amp;author=L.%20Lukens&amp;author=R.M.%20Amasino&amp;author=T.C.%20Osborn&amp;publication_year=2002&amp;journal=Genetics&amp;volume=162&amp;pages=1457-68" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12454088" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Marcel</div> <div class="given-names">T.C.</div></div>,  <div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Barbieri</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">A high-density consensus map of barley to compare the distribution of QTLs for partial resistance to <em>Puccinia hordei</em> and of defence gene homologues</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2007</div>, vol. <div class="volume">114</div> (pg. <div class="fpage">487</div>-<div class="lpage">500</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high-density%20consensus%20map%20of%20barley%20to%20compare%20the%20distribution%20of%20QTLs%20for%20partial%20resistance%20to%20Puccinia%20hordei%20and%20of%20defence%20gene%20homologues&amp;author=T.C.%20Marcel&amp;author=R.K.%20Varshney&amp;author=M.%20Barbieri&amp;publication_year=2007&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=114&amp;pages=487-500" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-006-0448-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-006-0448-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-006-0448-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17115126" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Singh</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ghai</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Garg</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">An integrated molecular linkage map of diploid wheat based on a <em>Triticum boeoticum</em> × <em>T. monococcum</em> RIL population</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2007</div>, vol. <div class="volume">115</div> (pg. <div class="fpage">301</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20integrated%20molecular%20linkage%20map%20of%20diploid%20wheat%20based%20on%20a%20Triticum%20boeoticum%20%C3%97%20T.%20monococcum%20RIL%20population&amp;author=K.%20Singh&amp;author=M.%20Ghai&amp;author=M.%20Garg&amp;publication_year=2007&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=115&amp;pages=301-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-007-0543-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-007-0543-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-007-0543-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17565482" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Marcel</div> <div class="given-names">T.C.</div></div>,  <div class="name"><div class="surname">Ramsay</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">A high density barley microsatellite consensus map with 775 SSR loci</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2007</div>, vol. <div class="volume">114</div> (pg. <div class="fpage">1091</div>-<div class="lpage">103</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high%20density%20barley%20microsatellite%20consensus%20map%20with%20775%20SSR%20loci&amp;author=R.K.%20Varshney&amp;author=T.C.%20Marcel&amp;author=L.%20Ramsay&amp;publication_year=2007&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=114&amp;pages=1091-103" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-007-0503-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-007-0503-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-007-0503-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17345060" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vezzulli</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Troggio</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Coppola</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">A reference integrated map for cultivated grapevine (<em>Vitis vinifera</em> L.) from three crosses, based on 283 SSR and 501 SNP-based markers</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">117</div> (pg. <div class="fpage">499</div>-<div class="lpage">511</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20reference%20integrated%20map%20for%20cultivated%20grapevine%20(Vitis%20vinifera%20L.)%20from%20three%20crosses%2C%20based%20on%20283%20SSR%20and%20501%20SNP-based%20markers&amp;author=S.%20Vezzulli&amp;author=M.%20Troggio&amp;author=G.%20Coppola&amp;publication_year=2008&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=117&amp;pages=499-511" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-008-0794-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-008-0794-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-008-0794-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18504538" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hwang</div> <div class="given-names">T.Y.</div></div>,  <div class="name"><div class="surname">Sayama</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Takahashi</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">High-density integrated linkage map based on SSR markers in soybean</div>, <div class="source">DNA Res.</div> , <div class="year">2009</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">213</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-density%20integrated%20linkage%20map%20based%20on%20SSR%20markers%20in%20soybean&amp;author=T.Y.%20Hwang&amp;author=T.%20Sayama&amp;author=M.%20Takahashi&amp;publication_year=2009&amp;journal=DNA%20Res.&amp;volume=16&amp;pages=213-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsp010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsp010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsp010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19531560" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mace</div> <div class="given-names">E.S.</div></div>,  <div class="name"><div class="surname">Rami</div> <div class="given-names">J.-F.</div></div>,  <div class="name"><div class="surname">Bouchet</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">A consensus genetic map of sorghum that integrates multiple component maps and high-throughput Diversity Array Technology (DArT) markers</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">9</div> pg. <div class="fpage">13</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20consensus%20genetic%20map%20of%20sorghum%20that%20integrates%20multiple%20component%20maps%20and%20high-throughput%20Diversity%20Array%20Technology%20(DArT)%20markers&amp;author=E.S.%20Mace&amp;author=J.-F.%20Rami&amp;author=S.%20Bouchet&amp;publication_year=2009&amp;journal=BMC%20Plant%20Biol.&amp;volume=9&amp;pages=13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-9-13" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-9-13" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-9-13"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19171067" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hong</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Liang</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">A SSR-based composite genetic linkage map for the cultivated peanut (<em>Arachis hypogaea</em> L.) genome</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">10</div> pg. <div class="fpage">17</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20SSR-based%20composite%20genetic%20linkage%20map%20for%20the%20cultivated%20peanut%20(Arachis%20hypogaea%20L.)%20genome&amp;author=Y.%20Hong&amp;author=X.%20Chen&amp;author=X.%20Liang&amp;publication_year=2010&amp;journal=BMC%20Plant%20Biol.&amp;volume=10&amp;pages=17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-10-17" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-10-17" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-10-17"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20105299" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parida</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Anand Raj Kumar</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Dalal</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Singh</div> <div class="given-names">N.K.</div></div>,  <div class="name"><div class="surname">Mohapatra</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Unigene derived microsatellite markers for the cereal genomes</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">112</div> (pg. <div class="fpage">808</div>-<div class="lpage">17</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unigene%20derived%20microsatellite%20markers%20for%20the%20cereal%20genomes&amp;author=S.K.%20Parida&amp;author=K.%20Anand%20Raj%20Kumar&amp;author=V.%20Dalal&amp;author=N.K.%20Singh&amp;author=T.%20Mohapatra&amp;publication_year=2006&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=112&amp;pages=808-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-005-0182-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-005-0182-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-005-0182-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16429310" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parida</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Yadava</div> <div class="given-names">D.K.</div></div>,  <div class="name"><div class="surname">Mohapatra</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Microsatellites in <em>Brassica unigenes</em>: relative abundance,marker design, and use in comparative physical mapping and genome analysis</div>, <div class="source">Genome</div> , <div class="year">2010</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">55</div>-<div class="lpage">67</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellites%20in%20Brassica%20unigenes%3A%20relative%20abundance%2Cmarker%20design%2C%20and%20use%20in%20comparative%20physical%20mapping%20and%20genome%20analysis&amp;author=S.K.%20Parida&amp;author=D.K.%20Yadava&amp;author=T.%20Mohapatra&amp;publication_year=2010&amp;journal=Genome&amp;volume=53&amp;pages=55-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/G09-084" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2FG09-084" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2FG09-084"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20130749" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharma</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Bhardwaj</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Negi</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Mohapatra</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ahuja</div> <div class="given-names">P.S.</div></div>. </div><div class="article-title">Identification, characterization and utilization of unigene derived microsatellite markers in tea (<em>Camellia sinensis L.</em>)</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">9</div> pg. <div class="fpage">53</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%2C%20characterization%20and%20utilization%20of%20unigene%20derived%20microsatellite%20markers%20in%20tea%20(Camellia%20sinensis%20L.)&amp;author=R.K.%20Sharma&amp;author=P.%20Bhardwaj&amp;author=R.%20Negi&amp;author=T.%20Mohapatra&amp;author=P.S.%20Ahuja&amp;publication_year=2009&amp;journal=BMC%20Plant%20Biol.&amp;volume=9&amp;pages=53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-9-53" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-9-53" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-9-53"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19426565" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Morgante</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hanafey</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Powell</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Microsatellites are preferentially associated with nonrepetitive DNA in plant genomes</div>, <div class="source">Nat Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">194</div>-<div class="lpage">200</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellites%20are%20preferentially%20associated%20with%20nonrepetitive%20DNA%20in%20plant%20genomes&amp;author=M.%20Morgante&amp;author=M.%20Hanafey&amp;author=W.%20Powell&amp;publication_year=2002&amp;journal=Nat%20Genet.&amp;volume=30&amp;pages=194-200" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng822" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng822" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng822"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11799393" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Graner</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sorrells</div> <div class="given-names">M.E.</div></div>. </div><div class="article-title">Genetic microsatellite markers in plants: features and applications</div>, <div class="source">Trends Biotech.</div> , <div class="year">2005</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">48</div>-<div class="lpage">55</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20microsatellite%20markers%20in%20plants%3A%20features%20and%20applications&amp;author=R.K.%20Varshney&amp;author=A.%20Graner&amp;author=M.E.%20Sorrells&amp;publication_year=2005&amp;journal=Trends%20Biotech.&amp;volume=23&amp;pages=48-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tibtech.2004.11.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tibtech.2004.11.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tibtech.2004.11.005"> </span></div> </div></div></div></div></div><div content-id="DSR017C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Castillo</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Budak</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Dorado</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Graner</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hernandez</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Transferability and polymorphism of barley EST-SSR markers used for phylogenetic analysis in <em>Hordeum chilense</em></div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">8</div> pg. <div class="fpage">97</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transferability%20and%20polymorphism%20of%20barley%20EST-SSR%20markers%20used%20for%20phylogenetic%20analysis%20in%20Hordeum%20chilense&amp;author=A.%20Castillo&amp;author=H.%20Budak&amp;author=R.K.%20Varshney&amp;author=G.%20Dorado&amp;author=A.%20Graner&amp;author=P.%20Hernandez&amp;publication_year=2008&amp;journal=BMC%20Plant%20Biol.&amp;volume=8&amp;pages=97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-8-97" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-8-97" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-8-97"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18822176" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ishikawa</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Nakamura</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ashida</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Localization of anchor loci representing five hundred annotated rice genes to wheat chromosomes using PLUG markers</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">118</div> (pg. <div class="fpage">499</div>-<div class="lpage">514</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Localization%20of%20anchor%20loci%20representing%20five%20hundred%20annotated%20rice%20genes%20to%20wheat%20chromosomes%20using%20PLUG%20markers&amp;author=G.%20Ishikawa&amp;author=T.%20Nakamura&amp;author=T.%20Ashida&amp;publication_year=2008&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=118&amp;pages=499-514" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-008-0916-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-008-0916-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-008-0916-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19057889" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Senthilvel</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Jayashree</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Mahalakshmi</div> <div class="given-names">V.</div></div>, et al. </div><div class="article-title">Development and mapping of simple sequence repeat markers for pearl millet from data mining of expressed sequence tags</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">8</div> pg. <div class="fpage">119</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20mapping%20of%20simple%20sequence%20repeat%20markers%20for%20pearl%20millet%20from%20data%20mining%20of%20expressed%20sequence%20tags&amp;author=S.%20Senthilvel&amp;author=B.%20Jayashree&amp;author=V.%20Mahalakshmi&amp;publication_year=2008&amp;journal=BMC%20Plant%20Biol.&amp;volume=8&amp;pages=119" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-8-119" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-8-119" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-8-119"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19038016" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kaur</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Cogan</div> <div class="given-names">N. O. I.</div></div>,  <div class="name"><div class="surname">Ye</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">Genetic map construction and QTL mapping of resistance to blackleg (<em>Leptosphaeria maculans</em>) disease in Australian canola (<em>Brassica napus</em> L.) cultivars</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">120</div> (pg. <div class="fpage">71</div>-<div class="lpage">83</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20map%20construction%20and%20QTL%20mapping%20of%20resistance%20to%20blackleg%20(Leptosphaeria%20maculans)%20disease%20in%20Australian%20canola%20(Brassica%20napus%20L.)%20cultivars&amp;author=S.%20Kaur&amp;author=N.%20O.%20I.%20Cogan&amp;author=G.%20Ye&amp;publication_year=2009&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=120&amp;pages=71-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-009-1160-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-009-1160-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-009-1160-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19821065" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Raji</div> <div class="given-names">A.A.</div></div>,  <div class="name"><div class="surname">Anderson</div> <div class="given-names">J.V.</div></div>,  <div class="name"><div class="surname">Kolade1</div> <div class="given-names">O.A.</div></div>,  <div class="name"><div class="surname">Ugwu1</div> <div class="given-names">C.D.</div></div>,  <div class="name"><div class="surname">Dixon</div> <div class="given-names">A.G.</div></div>,  <div class="name"><div class="surname">Ingelbrecht</div> <div class="given-names">I.L.</div></div>. </div><div class="article-title">Gene-based microsatellites for cassava (Manihot esculenta Crantz): prevalence, polymorphisms, and cross-taxa utility</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">9</div> pg. <div class="fpage">118</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene-based%20microsatellites%20for%20cassava%20(Manihot%20esculenta%20Crantz)%3A%20prevalence%2C%20polymorphisms%2C%20and%20cross-taxa%20utility&amp;author=A.A.%20Raji&amp;author=J.V.%20Anderson&amp;author=O.A.%20Kolade1&amp;author=C.D.%20Ugwu1&amp;author=A.G.%20Dixon&amp;author=I.L.%20Ingelbrecht&amp;publication_year=2009&amp;journal=BMC%20Plant%20Biol.&amp;volume=9&amp;pages=118" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-9-118" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-9-118" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-9-118"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19747391" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Ramchiary</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Choi</div> <div class="given-names">S.R.</div></div>, et al. </div><div class="article-title">Development of a high density integrated reference genetic linkage map for the multinational <em>Brassica rapa</em> Genome Sequencing Project</div>, <div class="source">Genome</div> , <div class="year">2010</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">939</div>-<div class="lpage">47</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20a%20high%20density%20integrated%20reference%20genetic%20linkage%20map%20for%20the%20multinational%20Brassica%20rapa%20Genome%20Sequencing%20Project&amp;author=X.%20Li&amp;author=N.%20Ramchiary&amp;author=S.R.%20Choi&amp;publication_year=2010&amp;journal=Genome&amp;volume=53&amp;pages=939-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/G10-054" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2FG10-054" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2FG10-054"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21076509" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Madan</div> <div class="given-names">A.</div></div>. </div><div class="article-title">CAP3: a DNA sequence assembly program</div>, <div class="source">Genome Res.</div> , <div class="year">1999</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">868</div>-<div class="lpage">77</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CAP3%3A%20a%20DNA%20sequence%20assembly%20program&amp;author=X.%20Huang&amp;author=A.%20Madan&amp;publication_year=1999&amp;journal=Genome%20Res.&amp;volume=9&amp;pages=868-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.9.9.868" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.9.9.868" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.9.9.868"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10508846" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hong</div> <div class="given-names">C.P.</div></div>,  <div class="name"><div class="surname">Piao</div> <div class="given-names">Z.Y.</div></div>,  <div class="name"><div class="surname">Kang</div> <div class="given-names">T.W.</div></div>, et al. </div><div class="article-title">Genomic distribution of simple sequence repeats in <em>Brassica rapa</em></div>, <div class="source">Mol. Cells</div> , <div class="year">2007</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">349</div>-<div class="lpage">56</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20distribution%20of%20simple%20sequence%20repeats%20in%20Brassica%20rapa&amp;author=C.P.%20Hong&amp;author=Z.Y.%20Piao&amp;author=T.W.%20Kang&amp;publication_year=2007&amp;journal=Mol.%20Cells&amp;volume=23&amp;pages=349-56" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17646709" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rozen</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Skaletsky</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Primer3 on the WWW for general users and for biologist programmers</div>, <div class="source">Meth. Mol. Biol.</div> , <div class="year">2000</div>, vol. <div class="volume">132</div> (pg. <div class="fpage">365</div>-<div class="lpage">86</div>)</div></div></div></div><div content-id="DSR017C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stothard</div> <div class="given-names">P.</div></div>. </div><div class="article-title">The sequence manipulation suite: JavaScript programs for analyzing and formatting protein and DNA sequences</div>, <div class="source">Biotechniques</div> , <div class="year">2000</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">1102</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20sequence%20manipulation%20suite%3A%20JavaScript%20programs%20for%20analyzing%20and%20formatting%20protein%20and%20DNA%20sequences&amp;author=P.%20Stothard&amp;publication_year=2000&amp;journal=Biotechniques&amp;volume=28&amp;pages=1102-4" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10868275" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lagercrantz</div> <div class="given-names">U.</div></div>,  <div class="name"><div class="surname">Ellegren</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Andersson</div> <div class="given-names">L.</div></div>. </div><div class="article-title">The abundance of various polymorphic microsatellite motifs differs between plants and vertebrates</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1993</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">1111</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20abundance%20of%20various%20polymorphic%20microsatellite%20motifs%20differs%20between%20plants%20and%20vertebrates&amp;author=U.%20Lagercrantz&amp;author=H.%20Ellegren&amp;author=L.%20Andersson&amp;publication_year=1993&amp;journal=Nucleic%20Acids%20Res.&amp;volume=21&amp;pages=1111-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/21.5.1111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F21.5.1111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F21.5.1111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8464696" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Uzunova</div> <div class="given-names">M.I.</div></div>,  <div class="name"><div class="surname">Ecke</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Abundance, polymorphism and genetic mapping of microsatellites in oilseed rape (<em>Brassica napus L.</em>)</div>, <div class="source">Plant Breed.</div> , <div class="year">1999</div>, vol. <div class="volume">118</div> (pg. <div class="fpage">323</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Abundance%2C%20polymorphism%20and%20genetic%20mapping%20of%20microsatellites%20in%20oilseed%20rape%20(Brassica%20napus%20L.)&amp;author=M.I.%20Uzunova&amp;author=W.%20Ecke&amp;publication_year=1999&amp;journal=Plant%20Breed.&amp;volume=118&amp;pages=323-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1439-0523.1999.00371.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1439-0523.1999.00371.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1439-0523.1999.00371.x"> </span></div> </div></div></div></div></div><div content-id="DSR017C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lowe</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">Jones</div> <div class="given-names">A.E.</div></div>,  <div class="name"><div class="surname">Raybould</div> <div class="given-names">A.F.</div></div>,  <div class="name"><div class="surname">Trick</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Moule</div> <div class="given-names">C.L.</div></div>,  <div class="name"><div class="surname">Edwards</div> <div class="given-names">K.J.</div></div>. </div><div class="article-title">Transferability and genome specificity of a new set of microsatellite primers among <em>Brassica</em> species of the U triangle</div>, <div class="source">Mol. Ecol. Notes</div> , <div class="year">2002</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">7</div>-<div class="lpage">11</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transferability%20and%20genome%20specificity%20of%20a%20new%20set%20of%20microsatellite%20primers%20among%20Brassica%20species%20of%20the%20U%20triangle&amp;author=A.J.%20Lowe&amp;author=A.E.%20Jones&amp;author=A.F.%20Raybould&amp;author=M.%20Trick&amp;author=C.L.%20Moule&amp;author=K.J.%20Edwards&amp;publication_year=2002&amp;journal=Mol.%20Ecol.%20Notes&amp;volume=2&amp;pages=7-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1471-8286.2002.00126.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1471-8286.2002.00126.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1471-8286.2002.00126.x"> </span></div> </div></div></div></div></div><div content-id="DSR017C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lowe</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">Moule</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Trick</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Edwards</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Efficient large-scale development of microsatellites for marker and mapping applications in <em>Brassica</em> crop species</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">108</div> (pg. <div class="fpage">1103</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efficient%20large-scale%20development%20of%20microsatellites%20for%20marker%20and%20mapping%20applications%20in%20Brassica%20crop%20species&amp;author=A.J.%20Lowe&amp;author=C.%20Moule&amp;author=M.%20Trick&amp;author=K.%20Edwards&amp;publication_year=2004&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=108&amp;pages=1103-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-003-1522-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-003-1522-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-003-1522-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15067397" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suwabe</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Iketani</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Nunome</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kage</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Hirai</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Isolation and characterization of microsatellites in <em>Brassica rapa</em> L</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">104</div> (pg. <div class="fpage">1092</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20characterization%20of%20microsatellites%20in%20Brassica%20rapa%20L&amp;author=K.%20Suwabe&amp;author=H.%20Iketani&amp;author=T.%20Nunome&amp;author=T.%20Kage&amp;author=M.%20Hirai&amp;publication_year=2002&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=104&amp;pages=1092-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-002-0875-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-002-0875-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-002-0875-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12582617" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suwabe</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Tsukazaki</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Iketani</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Simple sequence repeat-based comparative genomics between <em>Brassica rapa</em> and <em>Arabidopsis thaliana</em>: the genetic origin of clubroot resistance</div>, <div class="source">Genetics</div> , <div class="year">2006</div>, vol. <div class="volume">173</div> (pg. <div class="fpage">309</div>-<div class="lpage">19</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simple%20sequence%20repeat-based%20comparative%20genomics%20between%20Brassica%20rapa%20and%20Arabidopsis%20thaliana%3A%20the%20genetic%20origin%20of%20clubroot%20resistance&amp;author=K.%20Suwabe&amp;author=H.%20Tsukazaki&amp;author=H.%20Iketani&amp;publication_year=2006&amp;journal=Genetics&amp;volume=173&amp;pages=309-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.104.038968" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.104.038968" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.104.038968"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16723420" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Piquemal</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Cinquin</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Couton</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">Construction of an oilseed rape (<em>Brassica napus</em> L.) genetic map with SSR markers</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">111</div> (pg. <div class="fpage">1514</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20an%20oilseed%20rape%20(Brassica%20napus%20L.)%20genetic%20map%20with%20SSR%20markers&amp;author=J.%20Piquemal&amp;author=E.%20Cinquin&amp;author=F.%20Couton&amp;publication_year=2005&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=111&amp;pages=1514-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-005-0080-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-005-0080-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-005-0080-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16187118" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kim</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">Chung</div> <div class="given-names">T.Y.</div></div>,  <div class="name"><div class="surname">King</div> <div class="given-names">G.J.</div></div>, et al. </div><div class="article-title">A sequence-tagged linkage map of <em>Brassica rapa</em></div>, <div class="source">Genetics</div> , <div class="year">2006</div>, vol. <div class="volume">174</div> (pg. <div class="fpage">29</div>-<div class="lpage">39</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20sequence-tagged%20linkage%20map%20of%20Brassica%20rapa&amp;author=J.S.%20Kim&amp;author=T.Y.%20Chung&amp;author=G.J.%20King&amp;publication_year=2006&amp;journal=Genetics&amp;volume=174&amp;pages=29-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.106.060152" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.106.060152" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.106.060152"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16988107" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Iniguez-Luy</div> <div class="given-names">F.L.</div></div>,  <div class="name"><div class="surname">Vande Voort</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Osborn</div> <div class="given-names">T.C.</div></div>. </div><div class="article-title">Development of a set of public SSR markers derived from genomic sequence of a rapid cycling <em>Brassica oleracea</em> L. genotype</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">117</div> (pg. <div class="fpage">977</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20a%20set%20of%20public%20SSR%20markers%20derived%20from%20genomic%20sequence%20of%20a%20rapid%20cycling%20Brassica%20oleracea%20L.%20genotype&amp;author=F.L.%20Iniguez-Luy&amp;author=A.%20Vande%20Voort&amp;author=T.C.%20Osborn&amp;publication_year=2008&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=117&amp;pages=977-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-008-0837-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-008-0837-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-008-0837-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18651126" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cheng</div> <div class="given-names">X.M.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Xia</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Development and genetic mapping of microsatellite markers from genome survey sequences in <em>Brassica napus</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">118</div> (pg. <div class="fpage">1121</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20genetic%20mapping%20of%20microsatellite%20markers%20from%20genome%20survey%20sequences%20in%20Brassica%20napus&amp;author=X.M.%20Cheng&amp;author=J.%20Xu&amp;author=S.%20Xia&amp;publication_year=2009&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=118&amp;pages=1121-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-009-0967-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-009-0967-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-009-0967-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19190889" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stam</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Construction of integrated genetic linkage maps by means of a new computer package: JoinMap</div>, <div class="source">Plant J.</div> , <div class="year">1993</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">739</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20integrated%20genetic%20linkage%20maps%20by%20means%20of%20a%20new%20computer%20package%3A%20JoinMap&amp;author=P.%20Stam&amp;publication_year=1993&amp;journal=Plant%20J.&amp;volume=3&amp;pages=739-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.1993.3.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.1993.3.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.1993.3.issue-5"> </span></div> </div></div></div></div></div><div content-id="DSR017C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Van Ooijen</div> <div class="given-names">J.W.</div></div>,  <div class="name"><div class="surname">Voorrips</div> <div class="given-names">R.E.</div></div>. </div>, <div class="source">JoinMap® Version 3.0: Software for the Calculation of Genetic Linkage Map</div> , <div class="year">2001</div><div class="publisher-loc">Wageningen, The Netherlands</div><div class="publisher-name">Plant Research International</div></div></div></div></div><div content-id="DSR017C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kosambi</div> <div class="given-names">D.D.</div></div>. </div><div class="article-title">The estimation of map distance from recombination values</div>, <div class="source">Ann. Eugen.</div> , <div class="year">1944</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">172</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20estimation%20of%20map%20distance%20from%20recombination%20values&amp;author=D.D.%20Kosambi&amp;publication_year=1944&amp;journal=Ann.%20Eugen.&amp;volume=12&amp;pages=172-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-1809.1943.tb02321.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-1809.1943.tb02321.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-1809.1943.tb02321.x"> </span></div> </div></div></div></div></div><div content-id="DSR017C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Voorrips</div> <div class="given-names">R.E.</div></div>. </div><div class="article-title">MapChart: software for the graphical presentation of linkage maps and QTLs</div>, <div class="source">J. Hered.</div> , <div class="year">2002</div>, vol. <div class="volume">93</div> (pg. <div class="fpage">77</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MapChart%3A%20software%20for%20the%20graphical%20presentation%20of%20linkage%20maps%20and%20QTLs&amp;author=R.E.%20Voorrips&amp;publication_year=2002&amp;journal=J.%20Hered.&amp;volume=93&amp;pages=77-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jhered/93.1.77" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjhered%2F93.1.77" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjhered%2F93.1.77"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12011185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Botstein</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">White</div> <div class="given-names">R.L.</div></div>,  <div class="name"><div class="surname">Skolnick</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Davis</div> <div class="given-names">R.W.</div></div>. </div><div class="article-title">Construction of a genetic linkage map in man using restriction fragment length polymorphisms</div>, <div class="source">Am. J. Hum. Genet.</div> , <div class="year">1980</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">314</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20genetic%20linkage%20map%20in%20man%20using%20restriction%20fragment%20length%20polymorphisms&amp;author=D.%20Botstein&amp;author=R.L.%20White&amp;author=M.%20Skolnick&amp;author=R.W.%20Davis&amp;publication_year=1980&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=32&amp;pages=314-31" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6247908" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schranz</div> <div class="given-names">M.E.</div></div>,  <div class="name"><div class="surname">Lysak</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Mitchell-Olds</div> <div class="given-names">T.</div></div>. </div><div class="article-title">The ABC's of comparative genomics in the <em>Brassicaceae</em>: building blocks of crucifer genomes</div>, <div class="source">Trends Plant Sci.</div> , <div class="year">2006</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">535</div>-<div class="lpage">42</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20ABC's%20of%20comparative%20genomics%20in%20the%20Brassicaceae%3A%20building%20blocks%20of%20crucifer%20genomes&amp;author=M.E.%20Schranz&amp;author=M.A.%20Lysak&amp;author=T.%20Mitchell-Olds&amp;publication_year=2006&amp;journal=Trends%20Plant%20Sci.&amp;volume=11&amp;pages=535-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2006.09.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2006.09.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2006.09.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17029932" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lombard</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Delourme</div> <div class="given-names">R.</div></div>. </div><div class="article-title">A consensus linkage map for rapeseed (<em>Brassica napus</em> L.): construction and integration of three individual maps from DH populations</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2001</div>, vol. <div class="volume">103</div> (pg. <div class="fpage">491</div>-<div class="lpage">507</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20consensus%20linkage%20map%20for%20rapeseed%20(Brassica%20napus%20L.)%3A%20construction%20and%20integration%20of%20three%20individual%20maps%20from%20DH%20populations&amp;author=V.%20Lombard&amp;author=R.%20Delourme&amp;publication_year=2001&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=103&amp;pages=491-507" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220100560" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220100560" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220100560"> </span></div> </div></div></div></div></div><div content-id="DSR017C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hauge</div> <div class="given-names">B.M.</div></div>,  <div class="name"><div class="surname">Hanley</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Cartinhour</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Cherry</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Goodman</div> <div class="given-names">H.M.</div></div>. </div><div class="article-title">An integrated genetic/RFLP map of the <em>Arabidopsis</em></div>, <div class="source">Plant J.</div> , <div class="year">1993</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">745</div>-<div class="lpage">54</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20integrated%20genetic%2FRFLP%20map%20of%20the%20Arabidopsis&amp;author=B.M.%20Hauge&amp;author=S.M.%20Hanley&amp;author=S.%20Cartinhour&amp;author=J.M.%20Cherry&amp;author=H.M.%20Goodman&amp;publication_year=1993&amp;journal=Plant%20J.&amp;volume=3&amp;pages=745-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.1993.3.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.1993.3.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.1993.3.issue-5"> </span></div> </div></div></div></div></div><div content-id="DSR017C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sewell</div> <div class="given-names">M.M.</div></div>,  <div class="name"><div class="surname">Sherman</div> <div class="given-names">B.K.</div></div>,  <div class="name"><div class="surname">Neale</div> <div class="given-names">D.B.</div></div>. </div><div class="article-title">A consensus map for loblolly pine (<em>Pinus taeda</em> L.). I. construction and integration of individual linkage maps from two outbred three-generation pedigrees</div>, <div class="source">Genetics</div> , <div class="year">1999</div>, vol. <div class="volume">151</div> (pg. <div class="fpage">321</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20consensus%20map%20for%20loblolly%20pine%20(Pinus%20taeda%20L.).%20I.%20construction%20and%20integration%20of%20individual%20linkage%20maps%20from%20two%20outbred%20three-generation%20pedigrees&amp;author=M.M.%20Sewell&amp;author=B.K.%20Sherman&amp;author=D.B.%20Neale&amp;publication_year=1999&amp;journal=Genetics&amp;volume=151&amp;pages=321-30" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9872970" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C57" class="ref-content "><div class="label -label">57</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sebastian</div> <div class="given-names">R.L.</div></div>,  <div class="name"><div class="surname">Howell</div> <div class="given-names">E.C.</div></div>,  <div class="name"><div class="surname">King</div> <div class="given-names">G.J.</div></div>,  <div class="name"><div class="surname">Marshall</div> <div class="given-names">D.F.</div></div>,  <div class="name"><div class="surname">Kearsey</div> <div class="given-names">M.J.</div></div>. </div><div class="article-title">An integrated AFLP and RFLP <em>Brassica oleracea</em> linkage map from two morphologically distinct doubled-haploid mapping populations</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2000</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">75</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20integrated%20AFLP%20and%20RFLP%20Brassica%20oleracea%20linkage%20map%20from%20two%20morphologically%20distinct%20doubled-haploid%20mapping%20populations&amp;author=R.L.%20Sebastian&amp;author=E.C.%20Howell&amp;author=G.J.%20King&amp;author=D.F.%20Marshall&amp;author=M.J.%20Kearsey&amp;publication_year=2000&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=100&amp;pages=75-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220050011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220050011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220050011"> </span></div> </div></div></div></div></div><div content-id="DSR017C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C58" class="ref-content "><div class="label -label">58</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jeuken</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Wijk</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Peleman</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Lindhout</div> <div class="given-names">P.</div></div>. </div><div class="article-title">An integrated interspecific AFLP map of lettuce (<em>Lactuca</em>) based on two <em>L.sativa</em> times <em>L. saligna</em> F<sub>2</sub> populations</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2001</div>, vol. <div class="volume">103</div> (pg. <div class="fpage">638</div>-<div class="lpage">47</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20integrated%20interspecific%20AFLP%20map%20of%20lettuce%20(Lactuca)%20based%20on%20two%20L.sativa%20times%20L.%20saligna%20F2%20populations&amp;author=M.%20Jeuken&amp;author=R.%20Wijk&amp;author=J.%20Peleman&amp;author=P.%20Lindhout&amp;publication_year=2001&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=103&amp;pages=638-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220100657" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220100657" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220100657"> </span></div> </div></div></div></div></div><div content-id="DSR017C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C59" class="ref-content "><div class="label -label">59</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tani</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Takahashi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Iwata</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">A consensus linkage map for Sugi (<em>Cryptomeria japonica</em>) from two pedigrees, based on microsatellites and expressed sequence tags</div>, <div class="source">Genetics</div> , <div class="year">2003</div>, vol. <div class="volume">165</div> (pg. <div class="fpage">1551</div>-<div class="lpage">68</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20consensus%20linkage%20map%20for%20Sugi%20(Cryptomeria%20japonica)%20from%20two%20pedigrees%2C%20based%20on%20microsatellites%20and%20expressed%20sequence%20tags&amp;author=N.%20Tani&amp;author=T.%20Takahashi&amp;author=H.%20Iwata&amp;publication_year=2003&amp;journal=Genetics&amp;volume=165&amp;pages=1551-68" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14668402" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C60" class="ref-content "><div class="label -label">60</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gustafson</div> <div class="given-names">J.P.</div></div>,  <div class="name"><div class="surname">Ma</div> <div class="given-names">X.F.</div></div>,  <div class="name"><div class="surname">Korzun</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Snape</div> <div class="given-names">J.W.</div></div>. </div><div class="article-title">A consensus map of rye integrating mapping data from five mapping populations</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">118</div> (pg. <div class="fpage">793</div>-<div class="lpage">800</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20consensus%20map%20of%20rye%20integrating%20mapping%20data%20from%20five%20mapping%20populations&amp;author=J.P.%20Gustafson&amp;author=X.F.%20Ma&amp;author=V.%20Korzun&amp;author=J.W.%20Snape&amp;publication_year=2009&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=118&amp;pages=793-800" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-008-0939-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-008-0939-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-008-0939-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19066841" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C61" class="ref-content "><div class="label -label">61</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hossain</div> <div class="given-names">K.G.</div></div>,  <div class="name"><div class="surname">Kawai</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Hayashi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hoshi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Yamanaka</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Harada</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Characterization and identification of (CT)n microsatellites in soybean using sheared genomic libraries</div>, <div class="source">DNA Res.</div> , <div class="year">2000</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">103</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20and%20identification%20of%20(CT)n%20microsatellites%20in%20soybean%20using%20sheared%20genomic%20libraries&amp;author=K.G.%20Hossain&amp;author=H.%20Kawai&amp;author=M.%20Hayashi&amp;author=M.%20Hoshi&amp;author=N.%20Yamanaka&amp;author=K.%20Harada&amp;publication_year=2000&amp;journal=DNA%20Res.&amp;volume=7&amp;pages=103-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/7.2.103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F7.2.103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F7.2.103"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10819325" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR017C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR017C62" class="ref-content "><div class="label -label">62</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hisano</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Isobe</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Characterization of the soybean genome using EST derived microsatellite markers</div>, <div class="source">DNA Res.</div> , <div class="year">2007</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">271</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20the%20soybean%20genome%20using%20EST%20derived%20microsatellite%20markers&amp;author=H.%20Hisano&amp;author=S.%20Sato&amp;author=S.%20Isobe&amp;publication_year=2007&amp;journal=DNA%20Res.&amp;volume=14&amp;pages=271-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsm025" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsm025" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsm025"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18192281" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2011. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=448180&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr017/2/dsr017-dsr017supp_table6.doc?Expires=2147483647&amp;Signature=ahkOqoVBAtmAfrXDOlfTvpkp-Ul-PQyiB4NQfE6BtFLcpMyxvPILVYHcG8pplJUBJU5~HdruTLX1u5AM40lYZ-sEaMoJRnUkoiMIqiJ9GYHWcsRAXygsoz0coj8Y~dZEVEVCRxWrKm1T3tzhFRXdRbGjgH1n5ACn2VLiAXBS-6JrwWCww3xtCWRqS7wMRXRH0KigrWwmGP4sJp5KM3fglV5-aAUZsrsLk6aGWeSHiObveDX3XvOhQB9amLlNw9uYVRQ1WA693wLiut5Z-E-BPUKsS1PYLGVZoCKgw3Egaa5kfFeFvcj5vwEoqE4g9n24HO3mP4lzBob0f7k1V4ZaGw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr017/2/dsr017-dsr017supp_table2.doc?Expires=2147483647&amp;Signature=F~okxdYHwHvY99jPN8OBUSpAxJD1nh3250r-w7aK6KFs3nC7Uym6ksBVrqq2Nu7r0K28~--9ZISHG8QH5PRB0K52F-yKiYLefVPmzio7acUNdGWSoKibMcMnBLe1CTFXo~eTFu-G2EtisdVDnUkMSOuk2iLAne9Hj~wXzly5Da2P9QMZQHg~To88PxWG4zh-ky0tI2lr9STTtm~VdiEulGlCNqWEW~5m5oHcDHlz6-fhDazoKTgvpO4ExU5VcT11qIS0Vbc~L~rqnxdEbU6eFqUovSNoNAQYHEEW2B5OEBKbD0pRN6IqLzeJhcUwI8TBDI-WcEhCaPOLjWwnFj-Alg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr017/2/dsr017-dsr017supp_table1.xls?Expires=2147483647&amp;Signature=LAJ0FyEvclg6DVp9U1oCPMLbpqY7N7h8J2F7xuAIIARMV1zjPtCU6qwGP5KamiBD0eZysaTfWoU4jMvHT849Ik1pXWG7hjGzWuTNKf1n0bJT4qOYjF1RCbHAyC2AMKE3xaMtwRFZ~lD9LeHqxbRBmSHBTlRWQicsjTRWVV4tjpSyoqgbaBTUHqGr03O1wrSXKQ4rOcwyO9l~qA7RglBXnVB6QldLF4DLuk4DdmyrSooPu9Ocj0vRRwtfA2qbxAFq~W9tULLpk3-TCDP4oBqpgABj7gAtQ4ENXxX0HoOUqlRgtY3QOHM94d5nb9sfhPOr1~K79LgCwtbjZRXrcw19Ow__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr017/2/dsr017-dsr017supp_fig2.ppt?Expires=2147483647&amp;Signature=uVOOibnoPDFY9cenx4rVlxexGHUpetUayWoePPSZCObP9bp~HFuWdEWdK0y2oclkw8XxlsEa8YfYaBuvTs2uL0YvMOodI52Y6Q2cv9xcuM8gH9SnkiKGPW3ocHj2z5Iy~2RGweRhOchFNeCJkzLuSl32knP29tck2bppHFiUT3I-dX265u0DV96viN~0L9sTMph3-rVNCGjDtELpMueyd-LoiwDNJC~msECfK8nhR9vf4WWa7uFR6OqL4uQqVK8ielwKiXCrBobjKToH-XQko0Q-5RspgInj6uB~f1Z-nTdxl8K~kpNhKrESo19ga1TasQrgCsnbYbcOP8W6RENo8Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr017/2/dsr017-dsr017supp_fig1.ppt?Expires=2147483647&amp;Signature=XEbbx~NktmJXhn8kTSUyFzVCYdlP9afGFyvgIMWHUUIi8zNr-zFaFe6N68OcuYFxWAdZfw~TbMv7OwhDNZDypH7oBuGAwWpyZjePxwni4M8~9Xz0nYfrN2rEkibgIk2RVd8iBMq499ntL2tMfhOG8WYDFnG1E9J7LW83sSiJUhqn-tHQ~xbMuho9XasbWO82OzNiIK4ZBPgioUj42q3x-Ba9EtnziuiTKy1Q--IcAYw7H8iff9YKrAlFGGywLQVWNRwdMe9Dja2kTcMTelj8lPqseiQtXLNnsf1nVarhMcynlT6HiNs1ql2CGKq0L3GNVQAwiv3t09hXd5P7tfMcVg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - ppt file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">394</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000296708600001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">59</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsr017"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="448180"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000296708600001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f18%2f5%2f305%2f448180">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000296708600001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (59)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F18%2F5%2F305%2F448180" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/448180'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genic Microsatellite Markers in Brassica rapa: Development, Characterization, Mapping, and Their Utility in Other Cultivated and Wild Brassica Relatives" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsr017");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f18%2f5%2f305%2f448180">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

