

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/4/419/3752474","@type":"ScholarlyArticle","name":"Integrated analysis of individual codon contribution to protein biosynthesis reveals a new approach to improving the basis of rational gene design","datePublished":"2017-04-24","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2017-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx014","keywords":["codon usage bias","microbial biotechnology","position-dependent codon usage","rational gene design","yeast genomics"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/4/419/3752474","author":[{"name":"Villada, Juan C.","affiliation":"1 Department of Microbiology, Universidade Federal de Viçosa, Viçosa 36570-900, Brazil","@type":"Person"},{"name":"Brustolini, Otávio José Bernardes","affiliation":"2 Department of Biochemistry and Molecular Biology, Universidade Federal de Viçosa, Viçosa 36570-900, Brazil","@type":"Person"},{"name":"Batista da Silveira, Wendel","affiliation":"1 Department of Microbiology, Universidade Federal de Viçosa, Viçosa 36570-900, Brazil","@type":"Person"}],"description":"Abstract.  Gene codon optimization may be impaired by the misinterpretation of frequency and optimality of codons. Although recent studies have revealed the eff","pageStart":"419","pageEnd":"434","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/4/1/m_dnares_24_4cover.png?Expires=2147483647&Signature=Bn68nrHzMnl3Hf0TBxDUSpIiE3ALYvws4lUuOsXgZQeZw~qHN97KSJf53QRVRMw9UIEwsJhU~oIqlbc~bNDrFxzLTkzmyUjnAZ7sS1Ope7v8mL4P5X8GIjyJTBx96xx3yzaFdQ9im8bglRhXw7RCd3PHFDaTYDb24rkCGDLkkLUBVi~BntzWxZrnH-LI5Yr~cYB3mJnIO~ISuJYAH~ZtK9rwdBnTT~rg4DIDb~I6Y2wsrkFjsIdVm-sIz91ZlsrHMWugLRT8lXkXDxrnR3~Cwmkt8-yfu-WWcXNP8U4bmwXopL-J4wp6s4jITl~IopCYMRAqzqwzkZxKkX6zWcYGMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Integrated analysis of individual codon contribution to protein biosynthesis reveals a new approach to improving the basis of rational gene design","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/4/1/m_dnares_24_4cover.png?Expires=2147483647&Signature=Bn68nrHzMnl3Hf0TBxDUSpIiE3ALYvws4lUuOsXgZQeZw~qHN97KSJf53QRVRMw9UIEwsJhU~oIqlbc~bNDrFxzLTkzmyUjnAZ7sS1Ope7v8mL4P5X8GIjyJTBx96xx3yzaFdQ9im8bglRhXw7RCd3PHFDaTYDb24rkCGDLkkLUBVi~BntzWxZrnH-LI5Yr~cYB3mJnIO~ISuJYAH~ZtK9rwdBnTT~rg4DIDb~I6Y2wsrkFjsIdVm-sIz91ZlsrHMWugLRT8lXkXDxrnR3~Cwmkt8-yfu-WWcXNP8U4bmwXopL-J4wp6s4jITl~IopCYMRAqzqwzkZxKkX6zWcYGMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Integrated analysis of individual codon contribution to protein biosynthesis reveals a new approach to improving the basis of rational gene design" />
    <meta property="og:description" content="Abstract.  Gene codon optimization may be impaired by the misinterpretation of frequency and optimality of codons. Although recent studies have revealed the eff" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/4/419/3752474" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-04-24" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Villada, Juan C." /><meta name="citation_author_institution" content="Department of Microbiology, Universidade Federal de Viçosa, Viçosa 36570-900, Brazil" /><meta name="citation_author" content="Brustolini, Otávio José Bernardes" /><meta name="citation_author_institution" content="Department of Biochemistry and Molecular Biology, Universidade Federal de Viçosa, Viçosa 36570-900, Brazil" /><meta name="citation_author" content="Batista da Silveira, Wendel" /><meta name="citation_author_institution" content="Department of Microbiology, Universidade Federal de Viçosa, Viçosa 36570-900, Brazil" /><meta name="citation_title" content="Integrated analysis of individual codon contribution to protein biosynthesis reveals a new approach to improving the basis of rational gene design" /><meta name="citation_firstpage" content="419" /><meta name="citation_lastpage" content="434" /><meta name="citation_doi" content="10.1093/dnares/dsx014" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2017/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Correlation between the abundance of Escherichia coli transfer RNAs and the occurrence of the respective codons in its protein genes,;  citation_journal_title=J. Mol. Biol;  citation_year=1981;  citation_volume=146;  citation_pages=1-21; " /><meta name="citation_reference" content="citation_title=The codon adaptation index-a measure of directional synonymous codon usage bias, and its potential applications,;  citation_journal_title=Nucleic Acids Res;  citation_year=1987;  citation_volume=15;  citation_pages=1281-95; " /><meta name="citation_reference" content="citation_title=Co-variation of tRNA abundance and codon usage in Escherichia coli at different growth rates,;  citation_journal_title=J. Mol. Biol;  citation_year=1996;  citation_volume=260;  citation_pages=649-63; " /><meta name="citation_reference" content="citation_title=Evolutionary genomics: codon bias and selection on single genomes,;  citation_journal_title=Nature;  citation_year=2005;  citation_volume=433;  citation_pages=E5-6; " /><meta name="citation_reference" content="citation_title=Solving the riddle of codon usage preferences: a test for translational selection,;  citation_journal_title=Nucleic Acids Res;  citation_year=2004;  citation_volume=32;  citation_pages=5036-44; " /><meta name="citation_reference" content="citation_title=Determinants of translation efficiency and accuracy,;  citation_journal_title=Mol. Syst. Biol;  citation_year=2011;  citation_volume=7;  citation_pages=481" /><meta name="citation_reference" content="citation_title=Studies of codon usage and tRNA genes of 18 unicellular organisms and quantification of Bacillus subtilis tRNAs: gene expression level and species-specific diversity of codon usage based on multivariate analysis,;  citation_journal_title=Gene;  citation_year=1999;  citation_volume=238;  citation_pages=143-55; " /><meta name="citation_reference" content="citation_title=Codon influence on protein expression in E. coli correlates with mRNA levels,;  citation_journal_title=Nature;  citation_year=2016;  citation_volume=529;  citation_pages=358-63; " /><meta name="citation_reference" content="citation_title=Universal function-specificity of codon usage,;  citation_journal_title=Nucleic Acids Res;  citation_year=2009;  citation_volume=37;  citation_pages=7014-23; " /><meta name="citation_reference" content="citation_title=Multiple roles of the coding sequence 5′ end in gene expression regulation,;  citation_journal_title=Nucleic Acids Res;  citation_year=2015;  citation_volume=43;  citation_pages=13-28; " /><meta name="citation_reference" content="citation_title=Speeding with control: codon usage, tRNAs, and ribosomes,;  citation_journal_title=Trends Genet;  citation_year=2012;  citation_volume=28;  citation_pages=574-81; " /><meta name="citation_reference" content="citation_title=Weak 5′-mRNA secondary structures in short eukaryotic genes,;  citation_journal_title=Genome Biol. Evol;  citation_year=2012;  citation_volume=4;  citation_pages=1046-53; " /><meta name="citation_reference" content="citation_title=Translation efficiency is determined by both codon bias and folding energy,;  citation_journal_title=Proc. Natl. Acad. Sci. U.S.A;  citation_year=2010;  citation_volume=107;  citation_pages=3645-50; " /><meta name="citation_reference" content="citation_title=Strong association between mRNA folding strength and protein abundance in S. cerevisiae,;  citation_journal_title=EMBO Rep;  citation_year=2012;  citation_volume=13;  citation_pages=272-7; " /><meta name="citation_reference" content="citation_title=Codon optimality is a major determinant of mRNA stability,;  citation_journal_title=Cell;  citation_year=2015;  citation_volume=160;  citation_pages=1111-24; " /><meta name="citation_reference" content="citation_title=A universal trend of reduced mRNA stability near the translation-initiation site in prokaryotes and eukaryotes,;  citation_journal_title=PLoS Comput. Biol;  citation_year=2010;  citation_volume=6;  citation_pages=e1000664" /><meta name="citation_reference" content="citation_title=Universally increased mRNA stability downstream of the translation initiation site in eukaryotes and prokaryotes,;  citation_journal_title=Gene;  citation_year=2013;  citation_volume=517;  citation_pages=230-5; " /><meta name="citation_reference" content="citation_title=Roles for synonymous codon usage in protein biogenesis,;  citation_journal_title=Annu. Rev. Biophys;  citation_year=2014;  citation_volume=44;  citation_pages=143-66; " /><meta name="citation_reference" content="citation_title=Evidence for codon bias selection at the pre-mRNA level in eukaryotes,;  citation_journal_title=Trends Genet;  citation_year=2004;  citation_volume=20;  citation_pages=534-8; " /><meta name="citation_reference" content="citation_title=Biological basis of miRNA action when their targets are located in human protein coding region,;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e63403" /><meta name="citation_reference" content="citation_title=Selection on synonymous sites for increased accessibility around miRNA binding sites in plants,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2012;  citation_volume=29;  citation_pages=3037-44; " /><meta name="citation_reference" content="citation_title=Codon usage influences the local rate of translation elongation to regulate co-translational protein folding,;  citation_journal_title=Mol. Cell;  citation_year=2015;  citation_volume=59;  citation_pages=744-54; " /><meta name="citation_reference" content="citation_title=The imprint of codons on protein structure,;  citation_journal_title=Biotechnol. J;  citation_year=2011;  citation_volume=6;  citation_pages=641-9; " /><meta name="citation_reference" content="citation_title=Synonymous mutations and ribosome stalling can lead to altered folding pathways and distinct minima,;  citation_journal_title=J. Mol. Biol;  citation_year=2008;  citation_volume=383;  citation_pages=281-91; " /><meta name="citation_reference" content="citation_title=Timing is everything: unifying codon translation rates and nascent proteome behavior,;  citation_journal_title=J. Am. Chem. Soc;  citation_year=2014;  citation_volume=136;  citation_pages=17892-8; " /><meta name="citation_reference" content="citation_title=Local slowdown of translation by nonoptimal codons promotes nascent-chain recognition by SRP in vivo,;  citation_journal_title=Nat. Struct. Mol. Biol;  citation_year=2014;  citation_volume=21;  citation_pages=1100-5; " /><meta name="citation_reference" content="citation_title=Evolutionary conservation of codon optimality reveals hidden signatures of cotranslational folding;  citation_journal_title=Nat. Struct. Mol. Biol;  citation_year=2013;  citation_volume=20;  citation_pages=237-43; " /><meta name="citation_reference" content="citation_title=Codon bias and heterologous protein expression,;  citation_journal_title=Trends Biotechnol;  citation_year=2004;  citation_volume=22;  citation_pages=346-53; " /><meta name="citation_reference" content="citation_title=An evolutionary perspective on synonymous codon usage in unicellular organisms,;  citation_journal_title=J. Mol. Evol;  citation_year=1986;  citation_volume=24;  citation_pages=28-38; " /><meta name="citation_reference" content="citation_title=An evolutionarily conserved mechanism for controlling the efficiency of protein translation,;  citation_journal_title=Cell;  citation_year=2010;  citation_volume=141;  citation_pages=344-54; " /><meta name="citation_reference" content="citation_title=Improving the accuracy of recombinant protein production through integration of bioinformatics, statistical and mass spectrometry methodologies,;  citation_journal_title=FEBS J;  citation_year=2015;  citation_volume=282;  citation_pages=769-87; " /><meta name="citation_reference" content="citation_title=Codon optimization, promoter and expression system selection that achieved high-level production of Yarrowia lipolytica lipase in Pichia pastoris,;  citation_journal_title=Enzyme Microb. Technol;  citation_year=2015;  citation_volume=71;  citation_pages=66-72; " /><meta name="citation_reference" content="citation_title=Codon optimization significantly improves the expression level of α-amylase gene from Bacillus licheniformis in Pichia pastoris,;  citation_journal_title=Biomed Res. Int;  citation_year=2015;  citation_volume=2015;  citation_pages=248680" /><meta name="citation_reference" content="citation_title=Codon optimisation improves the expression of Trichoderma viride sp. endochitinase in Pichia pastoris,;  citation_journal_title=Sci. Rep;  citation_year=2013;  citation_volume=3;  citation_pages=3043" /><meta name="citation_reference" content="citation_title=Good codons, bad transcript: large reductions in gene expression and fitness arising from synonymous mutations in a key enzyme,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2012;  citation_volume=30;  citation_pages=549-60; " /><meta name="citation_reference" content="citation_title=Rare codon content affects the solubility of recombinant proteins in a codon bias-adjusted Escherichia coli strain,;  citation_journal_title=Microb. Cell Fact;  citation_year=2009;  citation_volume=8;  citation_pages=41" /><meta name="citation_reference" content="citation_title=Silent substitutions predictably alter translation elongation rates and protein folding efficiencies,;  citation_journal_title=J. Mol. Biol;  citation_year=2012;  citation_volume=422;  citation_pages=328-35; " /><meta name="citation_reference" content="citation_title=Impact of rare codons and the functional coproduction of rate-limiting tRNAs on recombinant protein production in Bacillus megaterium,;  citation_journal_title=Appl. Microbiol. Biotechnol;  citation_year=2015;  citation_volume=99;  citation_pages=8999-9010; " /><meta name="citation_reference" content="citation_title=Enhancing functional expression of heterologous proteins through random substitution of genetic codes in the 5′ coding region,;  citation_journal_title=Biotechnol. Bioeng;  citation_year=2015;  citation_volume=112;  citation_pages=822-6; " /><meta name="citation_reference" content="citation_title=Quo vadis? The challenges of recombinant protein folding and secretion in Pichia pastoris,;  citation_journal_title=Appl. Microbiol. Biotechnol;  citation_year=2015;  citation_volume=99;  citation_pages=2925-38; " /><meta name="citation_reference" content="citation_title=Limitations of codon adaptation index and other coding DNA‐based features for prediction of protein expression in Saccharomyces cerevisiae,;  citation_journal_title=Yeast;  citation_year=2004;  citation_volume=21;  citation_pages=1083-93; " /><meta name="citation_reference" content="citation_title=Quantifying position-dependent codon usage bias,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2014;  citation_volume=31;  citation_pages=1880-93; " /><meta name="citation_reference" content="citation_title=Biased codon usage in signal peptides: a role in protein export,;  citation_journal_title=Trends Microbiol;  citation_year=2009;  citation_volume=17;  citation_pages=146-50; " /><meta name="citation_reference" content="citation_title=Whole genome analysis reveals a high incidence of non-optimal codons in secretory signal sequences of Escherichia coli,;  citation_journal_title=Biochem. Biophys. Res. Commun;  citation_year=2004;  citation_volume=322;  citation_pages=1038-44; " /><meta name="citation_reference" content="citation_title=Locating proteins in the cell using TargetP, SignalP and related tools,;  citation_journal_title=Nat. Protoc;  citation_year=2007;  citation_volume=2;  citation_pages=953-71; " /><meta name="citation_reference" content="citation_title=EMBOSS: the European Molecular Biology Open Software Suite,;  citation_journal_title=Trends Genet;  citation_year=2000;  citation_volume=16;  citation_pages=276-7; " /><meta name="citation_reference" content="citation_title=Codon usage in Caenorhabditis elegans: delineation of translational selection and mutational biases,;  citation_journal_title=Nucleic Acids Res;  citation_year=1994;  citation_volume=22;  citation_pages=2437-46; " /><meta name="citation_reference" content="citation_title=Genomic analysis of codon usage shows influence of mutation pressure, natural selection, and host features on Marburg virus evolution,;  citation_journal_title=BMC Evol. Biol;  citation_year=2015;  citation_volume=15;  citation_pages=174" /><meta name="citation_reference" content="citation_title=The tRNAscan-SE, snoscan and snoGPS web servers for the detection of tRNAs and snoRNAs,;  citation_journal_title=Nucleic Acids Res;  citation_year=2005;  citation_volume=33(suppl 2);  citation_pages=W686-9; " /><meta name="citation_reference" content="citation_title=The effect of tRNA levels on decoding times of mRNA codons,;  citation_journal_title=Nucleic Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=9171-81; " /><meta name="citation_reference" content="citation_publisher=Springer, Berlin, Heidelberg;  citation_title=Structural approaches to sequence evolution;  citation_year=2007;  citation_pages=207-32; " /><meta name="citation_reference" content="  Pages H., Aboyoun P., Gentleman R., DebRoy S.  2009 , String objects representing biological sequences, and matching algorithms , R package version. 2 , 2. " /><meta name="citation_reference" content="citation_title=UniProt: a hub for protein information; citation_author=UniProt Consortium;  citation_journal_title=Nucleic Acids Res;  citation_year=2015;  citation_volume=43;  citation_pages=D204-12; " /><meta name="citation_reference" content="citation_title=Clustal W and Clustal X version 2.0;  citation_journal_title=Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=2947-8; " /><meta name="citation_reference" content="citation_title=MEGA6: Molecular evolutionary genetics analysis version 6.0,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2013;  citation_volume=30;  citation_pages=2725-9; " /><meta name="citation_reference" content="citation_title=MMDB and VAST+: tracking structural similarities between macromolecular complexes,;  citation_journal_title=Nucleic Acids Res;  citation_year=2013;  citation_volume=42;  citation_pages=D297-303; " /><meta name="citation_reference" content="citation_title=ViennaRNA Package 2.0;  citation_journal_title=Algorithms Mol. Biol;  citation_year=2011;  citation_volume=6;  citation_pages=1" /><meta name="citation_reference" content="citation_title=Gene length and codon usage bias in Drosophila melanogaster, Saccharomyces cerevisiae and Escherichia coli,;  citation_journal_title=Nucleic Acids Res;  citation_year=1998;  citation_volume=26;  citation_pages=3188-93; " /><meta name="citation_reference" content="citation_title=Expression pattern and, surprisingly, gene length shape codon usage in Caenorhabditis, Drosophila, and Arabidopsis,;  citation_journal_title=Proc. Natl. Acad. Sci. U.S.A;  citation_year=1999;  citation_volume=96;  citation_pages=4482-7; " /><meta name="citation_reference" content="citation_title=Analysis of synonymous codon usage patterns in seven different citrus species,;  citation_journal_title=Evol. Bioinf. Online;  citation_year=2013;  citation_volume=9;  citation_pages=215-28; " /><meta name="citation_reference" content="citation_title=Analysis of codon usage bias of mitochondrial genome in Bombyx mori and its relation to evolution,;  citation_journal_title=BMC Evol. Biol;  citation_year=2014;  citation_volume=14;  citation_pages=262" /><meta name="citation_reference" content="citation_title=Non-uniqueness of factors constraint on the codon usage in Bombyx mori,;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=356" /><meta name="citation_reference" content="citation_title=Gene expression levels are correlated with synonymous codon usage, amino acid composition, and gene architecture in the red flour beetle, Tribolium castaneum;  citation_journal_title=Mol. Biol. Evol;  citation_year=2012;  citation_volume=29;  citation_pages=3755-66; " /><meta name="citation_reference" content="citation_title=Increased incidence of rare codon clusters at 5′ and 3′ gene termini: implications for function,;  citation_journal_title=BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=118" /><meta name="citation_reference" content="citation_title=Properties and determinants of codon decoding time distributions;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=S13" /><meta name="citation_reference" content="citation_title=Whole genome analysis of non-optimal codon usage in secretory signal sequences of Streptomyces coelicolor,;  citation_journal_title=BioSystems;  citation_year=2006;  citation_volume=85;  citation_pages=225-30; " /><meta name="citation_reference" content="citation_title=Codon bias as a means to fine-tune gene expression,;  citation_journal_title=Mol. Cell;  citation_year=2015;  citation_volume=59;  citation_pages=149-61; " /><meta name="citation_reference" content="citation_title=Synonymous but not the same: the causes and consequences of codon bias,;  citation_journal_title=Nat. Rev. Genet;  citation_year=2010;  citation_volume=12;  citation_pages=32-42; " /><meta name="citation_reference" content="citation_title=Rate-limiting steps in yeast protein translation,;  citation_journal_title=Cell;  citation_year=2013;  citation_volume=153;  citation_pages=1589-601; " /><meta name="citation_reference" content="citation_title=The role of codon selection in regulation of translation efficiency deduced from synthetic libraries,;  citation_journal_title=Genome Biol;  citation_year=2011;  citation_volume=12;  citation_pages=R12" /><meta name="citation_reference" content="citation_title=Causes and effects of N-terminal codon bias in bacterial genes;  citation_journal_title=Science;  citation_year=2013;  citation_volume=342;  citation_pages=475-9; " /><meta name="citation_reference" content="citation_title=In-silico determination of Pichia pastoris signal peptides for extracellular recombinant protein production,;  citation_journal_title=J. Theor. Biol;  citation_year=2015;  citation_volume=364;  citation_pages=179-88; " /><meta name="citation_reference" content="citation_title=Engineering protein folding and translocation improves heterologous protein secretion in Saccharomyces cerevisiae,;  citation_journal_title=Biotechnol. Bioeng;  citation_year=2015;  citation_volume=112;  citation_pages=1872-82; " /><meta name="citation_reference" content="citation_title=Synthetic signal sequences that enable efficient secretory protein production in the yeast Kluyveromyces marxianus,;  citation_journal_title=Microb. Cell Fact;  citation_year=2015;  citation_volume=14;  citation_pages=1" /><meta name="citation_reference" content="citation_publisher=Oxford University Press, ;  citation_title=Nucleic Acids Symposium Series;  citation_year=2003;  citation_volume=3;  citation_pages=261-2; " /><meta name="citation_reference" content="citation_title=Large-scale analysis of conserved rare codon clusters suggests an involvement in co-translational molecular recognition events,;  citation_journal_title=Bioinformatics;  citation_year=2012;  citation_volume=28;  citation_pages=1438-45; " /><meta name="citation_reference" content="citation_title=Sounds of silence: synonymous nucleotides as a key to biological regulation and complexity,;  citation_journal_title=Nucleic Acids Res;  citation_year=2013;  citation_volume=41;  citation_pages=2073-94; " /><meta name="citation_reference" content="citation_title=Cotranslational protein folding inside the ribosome exit tunnel,;  citation_journal_title=Cell Rep;  citation_year=2015;  citation_volume=12;  citation_pages=1533-40; " /><meta name="citation_reference" content="citation_title=The ribosomal tunnel as a functional environment for nascent polypeptide folding and translational stalling,;  citation_journal_title=Curr. Opin. Struct. Biol;  citation_year=2011;  citation_volume=21;  citation_pages=274-82; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/4/419/22893264/dsx014.pdf" /><meta name="description" content="Abstract.  Gene codon optimization may be impaired by the misinterpretation of frequency and optimality of codons. Although recent studies have revealed the eff" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/4/419/3752474" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/4/419/3752474" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/4/419/3752474");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsx014';
    </script>
<title>
	Integrated analysis of individual codon contribution to protein biosynthesis reveals a new approach to improving the basis of rational gene design | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./3752474" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZCsATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVmlsbGFkYSwgSnVhbiBDLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTWljcm9iaW9sb2d5LCBVbml2ZXJzaWRhZGUgRmVkZXJhbCBkZSBWacOnb3NhLCBWacOnb3NhIDM2NTcwLTkwMCwgQnJhemlsIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQnJ1c3RvbGluaSwgT3TDoXZpbyBKb3PDqSBCZXJuYXJkZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2NoZW1pc3RyeSBhbmQgTW9sZWN1bGFyIEJpb2xvZ3ksIFVuaXZlcnNpZGFkZSBGZWRlcmFsIGRlIFZpw6dvc2EsIFZpw6dvc2EgMzY1NzAtOTAwLCBCcmF6aWwiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJCYXRpc3RhIGRhIFNpbHZlaXJhLCBXZW5kZWwiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE1pY3JvYmlvbG9neSwgVW5pdmVyc2lkYWRlIEZlZGVyYWwgZGUgVmnDp29zYSwgVmnDp29zYSAzNjU3MC05MDAsIEJyYXppbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iSW50ZWdyYXRlZCBhbmFseXNpcyBvZiBpbmRpdmlkdWFsIGNvZG9uIGNvbnRyaWJ1dGlvbiB0byBwcm90ZWluIGJpb3N5bnRoZXNpcyByZXZlYWxzIGEgbmV3IGFwcHJvYWNoIHRvIGltcHJvdmluZyB0aGUgYmFzaXMgb2YgcmF0aW9uYWwgZ2VuZSBkZXNpZ24iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI0MTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjQzNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzeDAxNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE3LzA4LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29ycmVsYXRpb24gYmV0d2VlbiB0aGUgYWJ1bmRhbmNlIG9mIEVzY2hlcmljaGlhIGNvbGkgdHJhbnNmZXIgUk5BcyBhbmQgdGhlIG9jY3VycmVuY2Ugb2YgdGhlIHJlc3BlY3RpdmUgY29kb25zIGluIGl0cyBwcm90ZWluIGdlbmVzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0xOTgxOyAgY2l0YXRpb25fdm9sdW1lPTE0NjsgIGNpdGF0aW9uX3BhZ2VzPTEtMjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb2RvbiBhZGFwdGF0aW9uIGluZGV4LWEgbWVhc3VyZSBvZiBkaXJlY3Rpb25hbCBzeW5vbnltb3VzIGNvZG9uIHVzYWdlIGJpYXMsIGFuZCBpdHMgcG90ZW50aWFsIGFwcGxpY2F0aW9ucyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MTI4MS05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q28tdmFyaWF0aW9uIG9mIHRSTkEgYWJ1bmRhbmNlIGFuZCBjb2RvbiB1c2FnZSBpbiBFc2NoZXJpY2hpYSBjb2xpIGF0IGRpZmZlcmVudCBncm93dGggcmF0ZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MjYwOyAgY2l0YXRpb25fcGFnZXM9NjQ5LTYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb25hcnkgZ2Vub21pY3M6IGNvZG9uIGJpYXMgYW5kIHNlbGVjdGlvbiBvbiBzaW5nbGUgZ2Vub21lcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00MzM7ICBjaXRhdGlvbl9wYWdlcz1FNS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Tb2x2aW5nIHRoZSByaWRkbGUgb2YgY29kb24gdXNhZ2UgcHJlZmVyZW5jZXM6IGEgdGVzdCBmb3IgdHJhbnNsYXRpb25hbCBzZWxlY3Rpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPTUwMzYtNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldGVybWluYW50cyBvZiB0cmFuc2xhdGlvbiBlZmZpY2llbmN5IGFuZCBhY2N1cmFjeSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gU3lzdC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9NDgxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3R1ZGllcyBvZiBjb2RvbiB1c2FnZSBhbmQgdFJOQSBnZW5lcyBvZiAxOCB1bmljZWxsdWxhciBvcmdhbmlzbXMgYW5kIHF1YW50aWZpY2F0aW9uIG9mIEJhY2lsbHVzIHN1YnRpbGlzIHRSTkFzOiBnZW5lIGV4cHJlc3Npb24gbGV2ZWwgYW5kIHNwZWNpZXMtc3BlY2lmaWMgZGl2ZXJzaXR5IG9mIGNvZG9uIHVzYWdlIGJhc2VkIG9uIG11bHRpdmFyaWF0ZSBhbmFseXNpcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmU7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MjM4OyAgY2l0YXRpb25fcGFnZXM9MTQzLTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiBpbmZsdWVuY2Ugb24gcHJvdGVpbiBleHByZXNzaW9uIGluIEUuIGNvbGkgY29ycmVsYXRlcyB3aXRoIG1STkEgbGV2ZWxzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTUyOTsgIGNpdGF0aW9uX3BhZ2VzPTM1OC02MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VW5pdmVyc2FsIGZ1bmN0aW9uLXNwZWNpZmljaXR5IG9mIGNvZG9uIHVzYWdlLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz03MDE0LTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdWx0aXBsZSByb2xlcyBvZiB0aGUgY29kaW5nIHNlcXVlbmNlIDXigLIgZW5kIGluIGdlbmUgZXhwcmVzc2lvbiByZWd1bGF0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz0xMy0yODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3BlZWRpbmcgd2l0aCBjb250cm9sOiBjb2RvbiB1c2FnZSwgdFJOQXMsIGFuZCByaWJvc29tZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz01NzQtODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdlYWsgNeKAsi1tUk5BIHNlY29uZGFyeSBzdHJ1Y3R1cmVzIGluIHNob3J0IGV1a2FyeW90aWMgZ2VuZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MTA0Ni01MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNsYXRpb24gZWZmaWNpZW5jeSBpcyBkZXRlcm1pbmVkIGJ5IGJvdGggY29kb24gYmlhcyBhbmQgZm9sZGluZyBlbmVyZ3ksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFUuUy5BOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEwNzsgIGNpdGF0aW9uX3BhZ2VzPTM2NDUtNTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cm9uZyBhc3NvY2lhdGlvbiBiZXR3ZWVuIG1STkEgZm9sZGluZyBzdHJlbmd0aCBhbmQgcHJvdGVpbiBhYnVuZGFuY2UgaW4gUy4gY2VyZXZpc2lhZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVNQk8gUmVwOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MjcyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9uIG9wdGltYWxpdHkgaXMgYSBtYWpvciBkZXRlcm1pbmFudCBvZiBtUk5BIHN0YWJpbGl0eSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTYwOyAgY2l0YXRpb25fcGFnZXM9MTExMS0yNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSB1bml2ZXJzYWwgdHJlbmQgb2YgcmVkdWNlZCBtUk5BIHN0YWJpbGl0eSBuZWFyIHRoZSB0cmFuc2xhdGlvbi1pbml0aWF0aW9uIHNpdGUgaW4gcHJva2FyeW90ZXMgYW5kIGV1a2FyeW90ZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIENvbXB1dC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDA2NjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Vbml2ZXJzYWxseSBpbmNyZWFzZWQgbVJOQSBzdGFiaWxpdHkgZG93bnN0cmVhbSBvZiB0aGUgdHJhbnNsYXRpb24gaW5pdGlhdGlvbiBzaXRlIGluIGV1a2FyeW90ZXMgYW5kIHByb2thcnlvdGVzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT01MTc7ICBjaXRhdGlvbl9wYWdlcz0yMzAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Um9sZXMgZm9yIHN5bm9ueW1vdXMgY29kb24gdXNhZ2UgaW4gcHJvdGVpbiBiaW9nZW5lc2lzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5udS4gUmV2LiBCaW9waHlzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQ0OyAgY2l0YXRpb25fcGFnZXM9MTQzLTY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdmlkZW5jZSBmb3IgY29kb24gYmlhcyBzZWxlY3Rpb24gYXQgdGhlIHByZS1tUk5BIGxldmVsIGluIGV1a2FyeW90ZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz01MzQtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmlvbG9naWNhbCBiYXNpcyBvZiBtaVJOQSBhY3Rpb24gd2hlbiB0aGVpciB0YXJnZXRzIGFyZSBsb2NhdGVkIGluIGh1bWFuIHByb3RlaW4gY29kaW5nIHJlZ2lvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lNjM0MDMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZWxlY3Rpb24gb24gc3lub255bW91cyBzaXRlcyBmb3IgaW5jcmVhc2VkIGFjY2Vzc2liaWxpdHkgYXJvdW5kIG1pUk5BIGJpbmRpbmcgc2l0ZXMgaW4gcGxhbnRzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9MzAzNy00NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gdXNhZ2UgaW5mbHVlbmNlcyB0aGUgbG9jYWwgcmF0ZSBvZiB0cmFuc2xhdGlvbiBlbG9uZ2F0aW9uIHRvIHJlZ3VsYXRlIGNvLXRyYW5zbGF0aW9uYWwgcHJvdGVpbiBmb2xkaW5nLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTU5OyAgY2l0YXRpb25fcGFnZXM9NzQ0LTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgaW1wcmludCBvZiBjb2RvbnMgb24gcHJvdGVpbiBzdHJ1Y3R1cmUsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW90ZWNobm9sLiBKOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz02NDEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3lub255bW91cyBtdXRhdGlvbnMgYW5kIHJpYm9zb21lIHN0YWxsaW5nIGNhbiBsZWFkIHRvIGFsdGVyZWQgZm9sZGluZyBwYXRod2F5cyBhbmQgZGlzdGluY3QgbWluaW1hLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM4MzsgIGNpdGF0aW9uX3BhZ2VzPTI4MS05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGltaW5nIGlzIGV2ZXJ5dGhpbmc6IHVuaWZ5aW5nIGNvZG9uIHRyYW5zbGF0aW9uIHJhdGVzIGFuZCBuYXNjZW50IHByb3Rlb21lIGJlaGF2aW9yLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQW0uIENoZW0uIFNvYzsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMzY7ICBjaXRhdGlvbl9wYWdlcz0xNzg5Mi04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Mb2NhbCBzbG93ZG93biBvZiB0cmFuc2xhdGlvbiBieSBub25vcHRpbWFsIGNvZG9ucyBwcm9tb3RlcyBuYXNjZW50LWNoYWluIHJlY29nbml0aW9uIGJ5IFNSUCBpbiB2aXZvLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBTdHJ1Y3QuIE1vbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTExMDAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uYXJ5IGNvbnNlcnZhdGlvbiBvZiBjb2RvbiBvcHRpbWFsaXR5IHJldmVhbHMgaGlkZGVuIHNpZ25hdHVyZXMgb2YgY290cmFuc2xhdGlvbmFsIGZvbGRpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gU3RydWN0LiBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0yMzctNDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9uIGJpYXMgYW5kIGhldGVyb2xvZ291cyBwcm90ZWluIGV4cHJlc3Npb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgQmlvdGVjaG5vbDsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTM0Ni01MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gZXZvbHV0aW9uYXJ5IHBlcnNwZWN0aXZlIG9uIHN5bm9ueW1vdXMgY29kb24gdXNhZ2UgaW4gdW5pY2VsbHVsYXIgb3JnYW5pc21zLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0xOTg2OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MjgtMzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGV2b2x1dGlvbmFyaWx5IGNvbnNlcnZlZCBtZWNoYW5pc20gZm9yIGNvbnRyb2xsaW5nIHRoZSBlZmZpY2llbmN5IG9mIHByb3RlaW4gdHJhbnNsYXRpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE0MTsgIGNpdGF0aW9uX3BhZ2VzPTM0NC01NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW1wcm92aW5nIHRoZSBhY2N1cmFjeSBvZiByZWNvbWJpbmFudCBwcm90ZWluIHByb2R1Y3Rpb24gdGhyb3VnaCBpbnRlZ3JhdGlvbiBvZiBiaW9pbmZvcm1hdGljcywgc3RhdGlzdGljYWwgYW5kIG1hc3Mgc3BlY3Ryb21ldHJ5IG1ldGhvZG9sb2dpZXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GRUJTIEo7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MjgyOyAgY2l0YXRpb25fcGFnZXM9NzY5LTg3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiBvcHRpbWl6YXRpb24sIHByb21vdGVyIGFuZCBleHByZXNzaW9uIHN5c3RlbSBzZWxlY3Rpb24gdGhhdCBhY2hpZXZlZCBoaWdoLWxldmVsIHByb2R1Y3Rpb24gb2YgWWFycm93aWEgbGlwb2x5dGljYSBsaXBhc2UgaW4gUGljaGlhIHBhc3RvcmlzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RW56eW1lIE1pY3JvYi4gVGVjaG5vbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT03MTsgIGNpdGF0aW9uX3BhZ2VzPTY2LTcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiBvcHRpbWl6YXRpb24gc2lnbmlmaWNhbnRseSBpbXByb3ZlcyB0aGUgZXhwcmVzc2lvbiBsZXZlbCBvZiDOsS1hbXlsYXNlIGdlbmUgZnJvbSBCYWNpbGx1cyBsaWNoZW5pZm9ybWlzIGluIFBpY2hpYSBwYXN0b3Jpcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb21lZCBSZXMuIEludDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMDE1OyAgY2l0YXRpb25fcGFnZXM9MjQ4NjgwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gb3B0aW1pc2F0aW9uIGltcHJvdmVzIHRoZSBleHByZXNzaW9uIG9mIFRyaWNob2Rlcm1hIHZpcmlkZSBzcC4gZW5kb2NoaXRpbmFzZSBpbiBQaWNoaWEgcGFzdG9yaXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2kuIFJlcDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MzA0MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdvb2QgY29kb25zLCBiYWQgdHJhbnNjcmlwdDogbGFyZ2UgcmVkdWN0aW9ucyBpbiBnZW5lIGV4cHJlc3Npb24gYW5kIGZpdG5lc3MgYXJpc2luZyBmcm9tIHN5bm9ueW1vdXMgbXV0YXRpb25zIGluIGEga2V5IGVuenltZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTU0OS02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmFyZSBjb2RvbiBjb250ZW50IGFmZmVjdHMgdGhlIHNvbHViaWxpdHkgb2YgcmVjb21iaW5hbnQgcHJvdGVpbnMgaW4gYSBjb2RvbiBiaWFzLWFkanVzdGVkIEVzY2hlcmljaGlhIGNvbGkgc3RyYWluLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWljcm9iLiBDZWxsIEZhY3Q7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTQxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2lsZW50IHN1YnN0aXR1dGlvbnMgcHJlZGljdGFibHkgYWx0ZXIgdHJhbnNsYXRpb24gZWxvbmdhdGlvbiByYXRlcyBhbmQgcHJvdGVpbiBmb2xkaW5nIGVmZmljaWVuY2llcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIE1vbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00MjI7ICBjaXRhdGlvbl9wYWdlcz0zMjgtMzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUltcGFjdCBvZiByYXJlIGNvZG9ucyBhbmQgdGhlIGZ1bmN0aW9uYWwgY29wcm9kdWN0aW9uIG9mIHJhdGUtbGltaXRpbmcgdFJOQXMgb24gcmVjb21iaW5hbnQgcHJvdGVpbiBwcm9kdWN0aW9uIGluIEJhY2lsbHVzIG1lZ2F0ZXJpdW0sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BcHBsLiBNaWNyb2Jpb2wuIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9OTk7ICBjaXRhdGlvbl9wYWdlcz04OTk5LTkwMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVuaGFuY2luZyBmdW5jdGlvbmFsIGV4cHJlc3Npb24gb2YgaGV0ZXJvbG9nb3VzIHByb3RlaW5zIHRocm91Z2ggcmFuZG9tIHN1YnN0aXR1dGlvbiBvZiBnZW5ldGljIGNvZGVzIGluIHRoZSA14oCyIGNvZGluZyByZWdpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW90ZWNobm9sLiBCaW9lbmc7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTEyOyAgY2l0YXRpb25fcGFnZXM9ODIyLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVF1byB2YWRpcz8gVGhlIGNoYWxsZW5nZXMgb2YgcmVjb21iaW5hbnQgcHJvdGVpbiBmb2xkaW5nIGFuZCBzZWNyZXRpb24gaW4gUGljaGlhIHBhc3RvcmlzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QXBwbC4gTWljcm9iaW9sLiBCaW90ZWNobm9sOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTk5OyAgY2l0YXRpb25fcGFnZXM9MjkyNS0zODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGltaXRhdGlvbnMgb2YgY29kb24gYWRhcHRhdGlvbiBpbmRleCBhbmQgb3RoZXIgY29kaW5nIEROQeKAkGJhc2VkIGZlYXR1cmVzIGZvciBwcmVkaWN0aW9uIG9mIHByb3RlaW4gZXhwcmVzc2lvbiBpbiBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWUsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ZZWFzdDsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTEwODMtOTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVF1YW50aWZ5aW5nIHBvc2l0aW9uLWRlcGVuZGVudCBjb2RvbiB1c2FnZSBiaWFzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MTg4MC05MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Qmlhc2VkIGNvZG9uIHVzYWdlIGluIHNpZ25hbCBwZXB0aWRlczogYSByb2xlIGluIHByb3RlaW4gZXhwb3J0LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTE0Ni01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2hvbGUgZ2Vub21lIGFuYWx5c2lzIHJldmVhbHMgYSBoaWdoIGluY2lkZW5jZSBvZiBub24tb3B0aW1hbCBjb2RvbnMgaW4gc2VjcmV0b3J5IHNpZ25hbCBzZXF1ZW5jZXMgb2YgRXNjaGVyaWNoaWEgY29saSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2NoZW0uIEJpb3BoeXMuIFJlcy4gQ29tbXVuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMyMjsgIGNpdGF0aW9uX3BhZ2VzPTEwMzgtNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxvY2F0aW5nIHByb3RlaW5zIGluIHRoZSBjZWxsIHVzaW5nIFRhcmdldFAsIFNpZ25hbFAgYW5kIHJlbGF0ZWQgdG9vbHMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFByb3RvYzsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9OTUzLTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FTUJPU1M6IHRoZSBFdXJvcGVhbiBNb2xlY3VsYXIgQmlvbG9neSBPcGVuIFNvZnR3YXJlIFN1aXRlLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9Mjc2LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9uIHVzYWdlIGluIENhZW5vcmhhYmRpdGlzIGVsZWdhbnM6IGRlbGluZWF0aW9uIG9mIHRyYW5zbGF0aW9uYWwgc2VsZWN0aW9uIGFuZCBtdXRhdGlvbmFsIGJpYXNlcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MjQzNy00NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBhbmFseXNpcyBvZiBjb2RvbiB1c2FnZSBzaG93cyBpbmZsdWVuY2Ugb2YgbXV0YXRpb24gcHJlc3N1cmUsIG5hdHVyYWwgc2VsZWN0aW9uLCBhbmQgaG9zdCBmZWF0dXJlcyBvbiBNYXJidXJnIHZpcnVzIGV2b2x1dGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBFdm9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MTc0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHRSTkFzY2FuLVNFLCBzbm9zY2FuIGFuZCBzbm9HUFMgd2ViIHNlcnZlcnMgZm9yIHRoZSBkZXRlY3Rpb24gb2YgdFJOQXMgYW5kIHNub1JOQXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMyhzdXBwbCAyKTsgIGNpdGF0aW9uX3BhZ2VzPVc2ODYtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGVmZmVjdCBvZiB0Uk5BIGxldmVscyBvbiBkZWNvZGluZyB0aW1lcyBvZiBtUk5BIGNvZG9ucyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9OTE3MS04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPVNwcmluZ2VyLCBCZXJsaW4sIEhlaWRlbGJlcmc7ICBjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmFsIGFwcHJvYWNoZXMgdG8gc2VxdWVuY2UgZXZvbHV0aW9uOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fcGFnZXM9MjA3LTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSIgIFBhZ2VzIEguLCBBYm95b3VuIFAuLCBHZW50bGVtYW4gUi4sIERlYlJveSBTLiAgMjAwOSAsIFN0cmluZyBvYmplY3RzIHJlcHJlc2VudGluZyBiaW9sb2dpY2FsIHNlcXVlbmNlcywgYW5kIG1hdGNoaW5nIGFsZ29yaXRobXMgLCBSIHBhY2thZ2UgdmVyc2lvbi4gMiAsIDIuICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVuaVByb3Q6IGEgaHViIGZvciBwcm90ZWluIGluZm9ybWF0aW9uOyBjaXRhdGlvbl9hdXRob3I9VW5pUHJvdCBDb25zb3J0aXVtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPUQyMDQtMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNsdXN0YWwgVyBhbmQgQ2x1c3RhbCBYIHZlcnNpb24gMi4wOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTI5NDctODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUVHQTY6IE1vbGVjdWxhciBldm9sdXRpb25hcnkgZ2VuZXRpY3MgYW5hbHlzaXMgdmVyc2lvbiA2LjAsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0yNzI1LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1NREIgYW5kIFZBU1QrOiB0cmFja2luZyBzdHJ1Y3R1cmFsIHNpbWlsYXJpdGllcyBiZXR3ZWVuIG1hY3JvbW9sZWN1bGFyIGNvbXBsZXhlcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9RDI5Ny0zMDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZpZW5uYVJOQSBQYWNrYWdlIDIuMDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QWxnb3JpdGhtcyBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIGxlbmd0aCBhbmQgY29kb24gdXNhZ2UgYmlhcyBpbiBEcm9zb3BoaWxhIG1lbGFub2dhc3RlciwgU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlIGFuZCBFc2NoZXJpY2hpYSBjb2xpLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz0zMTg4LTkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHByZXNzaW9uIHBhdHRlcm4gYW5kLCBzdXJwcmlzaW5nbHksIGdlbmUgbGVuZ3RoIHNoYXBlIGNvZG9uIHVzYWdlIGluIENhZW5vcmhhYmRpdGlzLCBEcm9zb3BoaWxhLCBhbmQgQXJhYmlkb3BzaXMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFUuUy5BOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTk2OyAgY2l0YXRpb25fcGFnZXM9NDQ4Mi03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiBzeW5vbnltb3VzIGNvZG9uIHVzYWdlIHBhdHRlcm5zIGluIHNldmVuIGRpZmZlcmVudCBjaXRydXMgc3BlY2llcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUV2b2wuIEJpb2luZi4gT25saW5lOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0yMTUtMjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIGNvZG9uIHVzYWdlIGJpYXMgb2YgbWl0b2Nob25kcmlhbCBnZW5vbWUgaW4gQm9tYnl4IG1vcmkgYW5kIGl0cyByZWxhdGlvbiB0byBldm9sdXRpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgRXZvbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTI2MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5vbi11bmlxdWVuZXNzIG9mIGZhY3RvcnMgY29uc3RyYWludCBvbiB0aGUgY29kb24gdXNhZ2UgaW4gQm9tYnl4IG1vcmksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0zNTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIGV4cHJlc3Npb24gbGV2ZWxzIGFyZSBjb3JyZWxhdGVkIHdpdGggc3lub255bW91cyBjb2RvbiB1c2FnZSwgYW1pbm8gYWNpZCBjb21wb3NpdGlvbiwgYW5kIGdlbmUgYXJjaGl0ZWN0dXJlIGluIHRoZSByZWQgZmxvdXIgYmVldGxlLCBUcmlib2xpdW0gY2FzdGFuZXVtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz0zNzU1LTY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmNyZWFzZWQgaW5jaWRlbmNlIG9mIHJhcmUgY29kb24gY2x1c3RlcnMgYXQgNeKAsiBhbmQgM+KAsiBnZW5lIHRlcm1pbmk6IGltcGxpY2F0aW9ucyBmb3IgZnVuY3Rpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xMTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm9wZXJ0aWVzIGFuZCBkZXRlcm1pbmFudHMgb2YgY29kb24gZGVjb2RpbmcgdGltZSBkaXN0cmlidXRpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz1TMTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaG9sZSBnZW5vbWUgYW5hbHlzaXMgb2Ygbm9uLW9wdGltYWwgY29kb24gdXNhZ2UgaW4gc2VjcmV0b3J5IHNpZ25hbCBzZXF1ZW5jZXMgb2YgU3RyZXB0b215Y2VzIGNvZWxpY29sb3IsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9TeXN0ZW1zOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTg1OyAgY2l0YXRpb25fcGFnZXM9MjI1LTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiBiaWFzIGFzIGEgbWVhbnMgdG8gZmluZS10dW5lIGdlbmUgZXhwcmVzc2lvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT01OTsgIGNpdGF0aW9uX3BhZ2VzPTE0OS02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3lub255bW91cyBidXQgbm90IHRoZSBzYW1lOiB0aGUgY2F1c2VzIGFuZCBjb25zZXF1ZW5jZXMgb2YgY29kb24gYmlhcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTMyLTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SYXRlLWxpbWl0aW5nIHN0ZXBzIGluIHllYXN0IHByb3RlaW4gdHJhbnNsYXRpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE1MzsgIGNpdGF0aW9uX3BhZ2VzPTE1ODktNjAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcm9sZSBvZiBjb2RvbiBzZWxlY3Rpb24gaW4gcmVndWxhdGlvbiBvZiB0cmFuc2xhdGlvbiBlZmZpY2llbmN5IGRlZHVjZWQgZnJvbSBzeW50aGV0aWMgbGlicmFyaWVzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz1SMTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DYXVzZXMgYW5kIGVmZmVjdHMgb2YgTi10ZXJtaW5hbCBjb2RvbiBiaWFzIGluIGJhY3RlcmlhbCBnZW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zNDI7ICBjaXRhdGlvbl9wYWdlcz00NzUtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW4tc2lsaWNvIGRldGVybWluYXRpb24gb2YgUGljaGlhIHBhc3RvcmlzIHNpZ25hbCBwZXB0aWRlcyBmb3IgZXh0cmFjZWxsdWxhciByZWNvbWJpbmFudCBwcm90ZWluIHByb2R1Y3Rpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBUaGVvci4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0zNjQ7ICBjaXRhdGlvbl9wYWdlcz0xNzktODg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVuZ2luZWVyaW5nIHByb3RlaW4gZm9sZGluZyBhbmQgdHJhbnNsb2NhdGlvbiBpbXByb3ZlcyBoZXRlcm9sb2dvdXMgcHJvdGVpbiBzZWNyZXRpb24gaW4gU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvdGVjaG5vbC4gQmlvZW5nOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTExMjsgIGNpdGF0aW9uX3BhZ2VzPTE4NzItODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN5bnRoZXRpYyBzaWduYWwgc2VxdWVuY2VzIHRoYXQgZW5hYmxlIGVmZmljaWVudCBzZWNyZXRvcnkgcHJvdGVpbiBwcm9kdWN0aW9uIGluIHRoZSB5ZWFzdCBLbHV5dmVyb215Y2VzIG1hcnhpYW51cyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1pY3JvYi4gQ2VsbCBGYWN0OyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1PeGZvcmQgVW5pdmVyc2l0eSBQcmVzcywgOyAgY2l0YXRpb25fdGl0bGU9TnVjbGVpYyBBY2lkcyBTeW1wb3NpdW0gU2VyaWVzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0yNjEtMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGFyZ2Utc2NhbGUgYW5hbHlzaXMgb2YgY29uc2VydmVkIHJhcmUgY29kb24gY2x1c3RlcnMgc3VnZ2VzdHMgYW4gaW52b2x2ZW1lbnQgaW4gY28tdHJhbnNsYXRpb25hbCBtb2xlY3VsYXIgcmVjb2duaXRpb24gZXZlbnRzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz0xNDM4LTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Tb3VuZHMgb2Ygc2lsZW5jZTogc3lub255bW91cyBudWNsZW90aWRlcyBhcyBhIGtleSB0byBiaW9sb2dpY2FsIHJlZ3VsYXRpb24gYW5kIGNvbXBsZXhpdHksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00MTsgIGNpdGF0aW9uX3BhZ2VzPTIwNzMtOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvdHJhbnNsYXRpb25hbCBwcm90ZWluIGZvbGRpbmcgaW5zaWRlIHRoZSByaWJvc29tZSBleGl0IHR1bm5lbCw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGwgUmVwOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTUzMy00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHJpYm9zb21hbCB0dW5uZWwgYXMgYSBmdW5jdGlvbmFsIGVudmlyb25tZW50IGZvciBuYXNjZW50IHBvbHlwZXB0aWRlIGZvbGRpbmcgYW5kIHRyYW5zbGF0aW9uYWwgc3RhbGxpbmcsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBPcGluLiBTdHJ1Y3QuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0yNzQtODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yNC80LzQxOS8yMjg5MzI2NC9kc3gwMTQucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIEdlbmUgY29kb24gb3B0aW1pemF0aW9uIG1heSBiZSBpbXBhaXJlZCBieSB0aGUgbWlzaW50ZXJwcmV0YXRpb24gb2YgZnJlcXVlbmN5IGFuZCBvcHRpbWFsaXR5IG9mIGNvZG9ucy4gQWx0aG91Z2ggcmVjZW50IHN0dWRpZXMgaGF2ZSByZXZlYWxlZCB0aGUgZWZmIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zNzUyNDc0JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZNfC3kW0uGpGAmTvVlcXCphu2E5Y" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f4%2f419%2f3752474" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f4%2f419%2f3752474" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=98073" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/4/1/m_dnares_24_4cover.png?Expires=2147483647&amp;Signature=Bn68nrHzMnl3Hf0TBxDUSpIiE3ALYvws4lUuOsXgZQeZw~qHN97KSJf53QRVRMw9UIEwsJhU~oIqlbc~bNDrFxzLTkzmyUjnAZ7sS1Ope7v8mL4P5X8GIjyJTBx96xx3yzaFdQ9im8bglRhXw7RCd3PHFDaTYDb24rkCGDLkkLUBVi~BntzWxZrnH-LI5Yr~cYB3mJnIO~ISuJYAH~ZtK9rwdBnTT~rg4DIDb~I6Y2wsrkFjsIdVm-sIz91ZlsrHMWugLRT8lXkXDxrnR3~Cwmkt8-yfu-WWcXNP8U4bmwXopL-J4wp6s4jITl~IopCYMRAqzqwzkZxKkX6zWcYGMA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/4">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="106357434">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106357434">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106357436">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106357436">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106357442">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106357442">2. Material and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106357474">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106357474">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106357537">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106357537">Availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106357539">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106357539">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106357541">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106357541">Author contribution</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106357543">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106357543">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106357547">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106357547">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/4/407/3744590">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/4/435/3744591">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Integrated analysis of individual codon contribution to protein biosynthesis reveals a new approach to improving the basis of rational gene design
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Juan C. Villada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Microbiology, Universidade Federal de Viçosa, Viçosa 36570-900, Brazil</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Juan+C.+Villada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Villada J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Villada J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Juan C. Villada</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ot&#225;vio Jos&#233; Bernardes Brustolini</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Department of Biochemistry and Molecular Biology, Universidade Federal de Viçosa, Viçosa 36570-900, Brazil</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ot%c3%a1vio+Jos%c3%a9+Bernardes+Brustolini">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Brustolini O">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Brustolini O&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Otávio José Bernardes Brustolini</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wendel Batista da Silveira</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Microbiology, Universidade Federal de Viçosa, Viçosa 36570-900, Brazil</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx014-cor1" data-open="dsx014-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel: +55 31 3899 2957. Fax: +55 31 38992573. Email: <a href="mailto:wendel.silveira@ufv.br" target="_blank">wendel.silveira@ufv.br</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wendel+Batista+da+Silveira">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Batista da Silveira W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Batista da Silveira W&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Wendel Batista da Silveira</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx014-FM1"><div class="fn"><div class="footnote-content">Edited by Prof. Kenta Nakai</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 4, 1 August 2017, Pages 419–434, <a href='https://doi.org/10.1093/dnares/dsx014'>https://doi.org/10.1093/dnares/dsx014</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">24 April 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">24 July 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Received:</div>
                                <div class="wi-date">22 March 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">23 March 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="3752474" href="/dnaresearch/article-pdf/24/4/419/22893264/dsx014.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Juan C. Villada, Otávio José Bernardes Brustolini, Wendel Batista da Silveira;  Integrated analysis of individual codon contribution to protein biosynthesis reveals a new approach to improving the basis of rational gene design, <em>DNA Research</em>, Volume 24, Issue 4, 1 August 2017, Pages 419–434, <a href='https://doi.org/10.1093/dnares/dsx014'>https://doi.org/10.1093/dnares/dsx014</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/3752474?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/3752474?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/3752474?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/3752474?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/3752474?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/3752474?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/3752474?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Integrated%20analysis%20of%20individual%20codon%20contribution%20to%20protein%20biosynthesis%20reveals%20a%20new%20approach%20to%20improving%20the%20basis%20of%20rational%20gene%20design&amp;publicationDate=2017-04-24&amp;volumeNum=24&amp;issueNum=4&amp;author=Villada%2C%20Juan%20C.%3B%20Brustolini%2C%20Ot%C3%A1vio%20Jos%C3%A9%20Bernardes&amp;startPage=419&amp;endPage=434&amp;contentId=10.1093%2Fdnares%2Fdsx014&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f4%2f419%2f3752474" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=106357434 id="106357434" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Gene codon optimization may be impaired by the misinterpretation of frequency and optimality of codons. Although recent studies have revealed the effects of codon usage bias (CUB) on protein biosynthesis, an integrated perspective of the biological role of individual codons remains unknown. Unlike other previous studies, we show, through an integrated framework that attributes of codons such as frequency, optimality and positional dependency should be combined to unveil individual codon contribution for protein biosynthesis. We designed a codon quantification method for assessing CUB as a function of position within genes with a novel constraint: the relativity of position-dependent codon usage shaped by coding sequence length. Thus, we propose a new way of identifying the enrichment, depletion and non-uniform positional distribution of codons in different regions of yeast genes. We clustered codons that shared attributes of frequency and optimality. The cluster of non-optimal codons with rare occurrence displayed two remarkable characteristics: higher codon decoding time than frequent–non-optimal cluster and enrichment at the 5′-end region, where optimal codons with the highest frequency are depleted. Interestingly, frequent codons with non-optimal adaptation to tRNAs are uniformly distributed in the <em>Saccharomyces cerevisiae</em> genes, suggesting their determinant role as a speed regulator in protein elongation.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;codon usage bias&quot;">codon usage bias</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;microbial biotechnology&quot;">microbial biotechnology</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;position-dependent codon usage&quot;">position-dependent codon usage</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;rational gene design&quot;">rational gene design</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;yeast genomics&quot;">yeast genomics</a></div>                            <h2 scrollto-destination=106357436 id="106357436" class="section-title" >1. Introduction</h2>
<p>Codon usage bias (CUB) has been a wide-ranging field of research in the last few decades, revealing the importance of codons in many biological processes from cell physiology to the evolution of genes and organisms.<a reveal-id="dsx014-B1" data-open="dsx014-B1" class="link link-ref link-reveal xref-bibr"><sup>1–9</sup></a> A number of them are related to genic regulation,<a reveal-id="dsx014-B10" data-open="dsx014-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsx014-B11" data-open="dsx014-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> folding energy of mRNA secondary structures,<a reveal-id="dsx014-B12" data-open="dsx014-B12" class="link link-ref link-reveal xref-bibr"><sup>12–14</sup></a> mRNA stability,<a reveal-id="dsx014-B15" data-open="dsx014-B15" class="link link-ref link-reveal xref-bibr"><sup>15–17</sup></a> alternative splicing,<a reveal-id="dsx014-B18" data-open="dsx014-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a><sup>,</sup><a reveal-id="dsx014-B19" data-open="dsx014-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> miRNA–mRNA interaction,<a reveal-id="dsx014-B20" data-open="dsx014-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a><sup>,</sup><a reveal-id="dsx014-B21" data-open="dsx014-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> and protein aggregation or co-translational folding.<a reveal-id="dsx014-B22" data-open="dsx014-B22" class="link link-ref link-reveal xref-bibr"><sup>22–27</sup></a> Consequently, synonymous mutations affect significantly both gene expression and protein level, which are determinant steps in heterologous expression.<a reveal-id="dsx014-B28" data-open="dsx014-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a></p><p>A number of traditional approaches have been widely used for the computational assessment of CUB based on measuring codon occurrence, such as Relative Synonymous Codon Usage (RSCU<a reveal-id="dsx014-B29" data-open="dsx014-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a>) which is an index for evaluating the relative frequency of codons and determining when a codon is preferentially selected over its synonymous codons. The Codon Adaptation Index (CAI<sup>2</sup>), is another method used for determining the potential adaptability of a gene to a host genome based on frequency of codon usage. On the other hand, the tRNA Adaptation Index (tAI<sup>5</sup>) is based on translation efficiency as measured by the codon–tRNA interaction, which calculates the optimality of a codon to the cognate tRNAs. This method has been broadly used to describe the ‘ramp theory’ of translation efficiency.<a reveal-id="dsx014-B30" data-open="dsx014-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> Thus, many studies have been based on codon indexes to improve protein production through codon optimization in microorganisms.<a reveal-id="dsx014-B31" data-open="dsx014-B31" class="link link-ref link-reveal xref-bibr"><sup>31–34</sup></a> Nevertheless, the introduction of synonymous mutations has sometimes resulted in a decrease in protein production<a reveal-id="dsx014-B35" data-open="dsx014-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> or loss of protein function due to instability, misfolding or aggregation.<a reveal-id="dsx014-B24" data-open="dsx014-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a><sup>,</sup><a reveal-id="dsx014-B36" data-open="dsx014-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a><sup>,</sup><a reveal-id="dsx014-B37" data-open="dsx014-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> Contrary to earlier codon optimization studies, recent works have demonstrated enhanced protein production by insertion of rare codons,<a reveal-id="dsx014-B38" data-open="dsx014-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> and, indeed, by random codon substitutions at the 5′-end region of genes.<a reveal-id="dsx014-B39" data-open="dsx014-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a></p><p>Despite significant advances in CUB research, the understanding of the role of individual codons remains a challenging problem in the production of recombinant proteins,<a reveal-id="dsx014-B40" data-open="dsx014-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> especially in yeast, where the limitations of codon optimization indexes have been reported years ago.<a reveal-id="dsx014-B41" data-open="dsx014-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> We believe that this drawback is related mainly to a lack of integrated studies that could combine different approaches into one integral explanation of the effects of individual codons on protein biosynthesis. Although considerable advances have been made in accounting for the role of codons in certain processes, an integrated study is still required to explain the implications of codons at different biological levels and integrate them into a framework that could improve in depth the understanding of CUB and its basis in rational gene design.</p><p>Another important question to be addressed in yeast CUB research is the determination of which codons are under evolutionary selective pressures at different positions in the genes. Recently, Hockenberry et al.<a reveal-id="dsx014-B42" data-open="dsx014-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> developed a method of codon quantification which was implemented in the <em>Escherichia coli</em> genome to illustrate codon deviations from uniformity as a function of their position within genes. This approach is a major step to providing support for the ramp theory and codon selection at the 5′-end of endogenous genes. However, we observed that this method is not accurate enough for comparing codons at regions distant from the start codon. To overcome this drawback, we developed a method for quantifying all the genome regions based on a binning scheme of codon quantification relative to coding sequence (CDS) length. Thus, beyond the integrated scheme to evaluate CUB as a function of the position in yeast CDSs, the present study proposes the use of a different approach to the quantification of codons in yeasts using genome-scale data.</p><p>Further, we examine various features of individual codons for five yeasts widely used in heterologous expression and value-added chemical production. We integrate the results to show the potential effects of each codon in protein biosynthesis, evidencing that position-dependent CUB is a governing rule over endogenous yeast CDSs. We show the interdependencies between the genome-scale frequency of codons and their optimality to translation efficiency. Additionally, we demonstrate that there are very specific non-optimal codons that are, necessarily, uniformly distributed in intragenic regions of genes, while other kinds of non-optimal codons—having higher decoding times—are enriched at the beginning of genes and depleted in distant regions. Finally, we exemplify the potential impacts of individual codons in protein secondary structures. Taken together, our framework and the subsequent analyses present a meaningful advance in improving the basis of rational gene design for heterologous expression and yeast synthetic biology.</p>                            <h2 scrollto-destination=106357442 id="106357442" class="section-title" >2. Material and methods</h2>
                            <h3 scrollto-destination=106357443 id="106357443" class="section-title" >2.1. Criteria for yeast selection</h3>
<p>Because we were interested in analysing the CUB in yeasts related to biotechnological processes, principally as cell factories, we counted the yeast species names reported in the ‘Yeast biotechnology’ article collection (<a class="link link-uri" href="http://www.microbialcellfactories.com/series/Yeast%20Biotechnology" target="_blank">http://www.microbialcellfactories.com/series/Yeast%20Biotechnology</a>), available in the Microbial Cell Factories journal edited by Prof. Diethard Mattanovich (last updated on 8 May 2015). The yeasts which were chosen for subsequent analysis were those with the most related publications. The yeasts selected were <em>Kluyveromyces lactis</em>, <em>Kluyveromyces marxianus</em>, <em>Pichia pastoris</em>, <em>Saccharomyces cerevisiae</em> and <em>Yarrowia lipolytica</em>.</p>                            <h3 scrollto-destination=106357445 id="106357445" class="section-title" >2.2. Genome sequences</h3>
<p>The genomic sequences of <em>Kluyveromyces lactis</em> (NRRL Y-1140: NC_006037–NC_006042), <em>Pichia pastoris</em> (GS115: NC_012963–NC_012966), <em>Saccharomyces cerevisiae</em> (S288c: NC_001133–NC_001148) and <em>Yarrowia</em> lipolytica (CLIB122: NC_006067–NC_006072) were obtained from the Fungi section of the NCBI ftp server (accessed in January 2015). The genomic sequence of <em>Kluyveromyces marxianus</em> (CCT 7735: CP009303–CP009310<em>)</em> and <em>Escherichia coli</em> (K-12: NC_000913) were downloaded from the Nucleotide database of the NCBI (accessed in February 2015).</p><p>It has been demonstrated in previous studies that signal peptides (SPs) introduce a different bias of codon usage in the 5′-end gene region.<a reveal-id="dsx014-B26" data-open="dsx014-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a><sup>,</sup><a reveal-id="dsx014-B43" data-open="dsx014-B43" class="link link-ref link-reveal xref-bibr"><sup>43–44</sup></a> Based on this, we decided to analyse CDSs containing SPs separately from those lacking SPs. Accordingly, the presence or lack of proteins with SPs were determined using SignalP-4.1<a reveal-id="dsx014-B45" data-open="dsx014-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> with the parameter <em>t = euk</em> for yeast sequences and, <em>t = gram-</em> for <em>E. coli</em>. Default values were used for other parameters.</p>                            <h3 scrollto-destination=106357448 id="106357448" class="section-title" >2.3. Characterization of codons</h3>
                            <h4 scrollto-destination=106357449 id="106357449" class="section-title" >2.3.1. Codon adaptation</h4>
<p>In order to analyse the adaptation of each CDS to the codon usage of its respective genome, we used the Codon Adaptation Index (CAI<sup>2</sup>). First, the Codon Usage Table (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_material_dsx014.pdf?Expires=2147483647&Signature=yrjqF1m4pc3Nh7lf8ZdqxQMqJLie-IFYrxEBJLcpqDMU9-oV5zGeyuesVkp5VQyxlzhU3zRsQ5V7s0CE4tzXC9VLZwjVNYLbCg71yM~MUu4INxBSLKGJz-v5sKsGC2imGQ4TSce61gCHMWq4eaQ-D76mDKsHmzubvAB-a2bBBxivZocHwwtUzcM4OJaXpsknqlOW6x-RXwQBYjVSIChlumeabOAPtOUxiQhOnoeMUW2xXepqOP3dagDP3chbKxvFxJPV7gB8t5xpDAGZBv7vi-2N6gKuBo-Iw7Rd5OQigzF-bhrHnjqYDtEm~gR1DIfN8gxdwbPU1O1xQZBFPTKMNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S3–S7</a></span>) of each genome was calculated by using the <em>cusp</em> function in the local version of EMBOSS suite v.6.6.0.0.<a reveal-id="dsx014-B46" data-open="dsx014-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> Then, using the <em>cai</em> function of EMBOSS suite, the CAI was computed for each coding sequence with the correspondent codon usage table of its genome.</p>                            <h4 scrollto-destination=106357451 id="106357451" class="section-title" >2.3.2. The frequency of codon usage</h4>
<div class="block-child-p">The relative synonymous codon usage (RSCU) is significant to the analysis of codon bias in terms of frequency. An important advantage of this index is its independence from amino acid composition bias.<a reveal-id="dsx014-B47" data-open="dsx014-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> Then, the RSCU value of each codon was calculated for all organisms. The RSCU value was determined by using <a href="#E1" class="sectionLink xref-sec">Equation (1)</a> from a script in R as follows: <a id="E1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E1"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>R</mi><mi>S</mi><mi>C</mi><mi>U</mi><mo>=</mo><mfrac><mrow><msub><mi>O</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow><mrow><mrow><mo stretchy="true">[</mo><mrow><msubsup><mo>∑</mo><mi>j</mi><mrow><mi>N</mi><mi>i</mi></mrow></msubsup><mrow><msub><mi>O</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></mrow><mo stretchy="true">]</mo></mrow><mo>×</mo><mfrac><mn>1</mn><mrow><mi>N</mi><mi>i</mi></mrow></mfrac></mrow></mfrac></mrow></math></div></div><div class="label">(1)</div></div> where <em>O<sub>ij</sub></em> is the frequency of the <em>j</em>th codon for the <em>i</em>th amino acid, and <em>Ni</em> the total number of synonymous codons coding for the <em>i</em>th amino acid. Hence, as designated by Nasrullah et al.,<a reveal-id="dsx014-B48" data-open="dsx014-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> a frequent codon will have an RSCU ≥ 1 and codons with RSCU &lt; 1 are qualified as rare.</div>                            <h4 scrollto-destination=106357453 id="106357453" class="section-title" >2.3.3. Translation efficiency and codon decoding time</h4>
<p>The translation efficiency of a codon was obtained from the tRNA Adaptation Index (tAI<a reveal-id="dsx014-B5" data-open="dsx014-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a>), where codon relative adaptiveness (<em>w</em>) is the adaptation of a codon to the pool of available cognate tRNAs in the genome which incorporates the different possible tRNA and the wobble pairing rules. Based on reported methods,<a reveal-id="dsx014-B5" data-open="dsx014-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> we calculated the values for <em>K. marxianus</em> and <em>P. pastoris</em> of the relative adaptiveness of a codon using CodonR including the parameters for eukaryotic microorganism analysis. The datasets of relative adaptiveness of each codon of <em>S. cerevisiae</em> were obtained from reported data.<a reveal-id="dsx014-B30" data-open="dsx014-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> In the case of <em>K. lactis</em> and <em>Y. lipolytica</em> they were retrieved from a different experiment.<a reveal-id="dsx014-B27" data-open="dsx014-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> When needed, tRNA counts were predicted locally using tRNAscan 1.4.<a reveal-id="dsx014-B49" data-open="dsx014-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> As described by Pechmann and Frydman,<a reveal-id="dsx014-B27" data-open="dsx014-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> codon relative adaptiveness (<em>w</em>) characterizes a codon as optimal if <em>w</em> ≥ 0.4 or non-optimal if <em>w </em>&lt; 0.4.</p><div class="block-child-p">Codon decoding time (CDT) is a value related to the abstract time required to translate a codon in relation to the codon relative adaptiveness of the tAI index. Thus, as defined by Dana and Tuller,<a reveal-id="dsx014-B50" data-open="dsx014-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> codons with low relative adaptiveness to the tRNA pool are more slowly translated. The CDT for each codon was calculated using <a href="#E2" class="sectionLink xref-sec">Equation (2)</a> and is defined as: <a id="E2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E2"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>C</mi><mi>D</mi><msub><mi>T</mi><mi>i</mi></msub><mo>=</mo><mfrac><mn>1</mn><mrow><msub><mi>w</mi><mi>i</mi></msub></mrow></mfrac></mrow></math></div></div><div class="label">(2)</div></div></div>                            <h3 scrollto-destination=106357456 id="106357456" class="section-title" >2.4. Genome simulations</h3>
<p>For testing spatial deviation from uniformity in codon bias, 1200 complete genomes in total, 200 for each one of the 6 organisms in this study, were simulated. We developed an R script (<em>SyMuGS</em>—see Availability section and Methods Appendix for details) under the <em>seqinr</em> package<a reveal-id="dsx014-B51" data-open="dsx014-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> to generate genomes containing coding sequences with the same codon usage, coding the same protein sequence as the original genome but scrambling the codon position in each coding sequence. This provides a robust null model to test the deviations of the original genome against the random distribution of codons in the simulated genomes.<a reveal-id="dsx014-B42" data-open="dsx014-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a></p>                            <h3 scrollto-destination=106357458 id="106357458" class="section-title" >2.5. Quantification of position-dependent codon usage bias</h3>
<p>We developed two R scripts (<em>QuantiCUB</em> and <em>ExVar3D</em>, see Availability section and Methods Appendix) under R/Bioconductor (<a class="link link-uri" href="http://www.R-project.org/" target="_blank">http://www.R-project.org/</a>) and Biostrings<a reveal-id="dsx014-B52" data-open="dsx014-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> to count the codon occurrence under a relative-to-CDSs-length binning scheme (<a reveal-id="dsx014-F1" data-open="dsx014-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). We noticed that the codon quantification method proposed in a previous work<a reveal-id="dsx014-B42" data-open="dsx014-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> is an appropriate approach to testing deviations from uniformity at initial intragenic regions but not to test subsequent positions, because of the exponential growth of the bin sizes in regions far from the start codon. When the codon position is more distant, the procedure becomes imprecise as regards retrieving codon quantification in regions in the intermediate and the 3′-end region of CDSs. Hence, it is not possible to keep codon quantification values in the same bin as they are part of different intragenic region of CDSs with dissimilar lengths. Thus, we decided to quantify the CUB by binning codons as a function of the position relative to the length of the CDSs. We integrated all the scripts developed into a toolset named ‘<em>CodG: Analyzing codon positional dependency from genome-scale data</em>’ (See Methods Appendix in <span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_material_dsx014.pdf?Expires=2147483647&Signature=yrjqF1m4pc3Nh7lf8ZdqxQMqJLie-IFYrxEBJLcpqDMU9-oV5zGeyuesVkp5VQyxlzhU3zRsQ5V7s0CE4tzXC9VLZwjVNYLbCg71yM~MUu4INxBSLKGJz-v5sKsGC2imGQ4TSce61gCHMWq4eaQ-D76mDKsHmzubvAB-a2bBBxivZocHwwtUzcM4OJaXpsknqlOW6x-RXwQBYjVSIChlumeabOAPtOUxiQhOnoeMUW2xXepqOP3dagDP3chbKxvFxJPV7gB8t5xpDAGZBv7vi-2N6gKuBo-Iw7Rd5OQigzF-bhrHnjqYDtEm~gR1DIfN8gxdwbPU1O1xQZBFPTKMNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> and <a class="link link-uri" href="https://github.com/juanvillada/CodG" target="_blank">https://github.com/juanvillada/CodG</a>). </p>                        <a id="106357460" scrollto-destination="106357460"></a>
<div id="dsx014-F1" data-id="dsx014-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/m_dsx014f1.png?Expires=2147483647&Signature=pWOx~mti6ghWT~Mll-wkLyz1pcbi-MZWPrzPheat8fTnsWmRluw~423m3tK~E0L-cZJgmufNEylmMlR8~pJ09MxHxUTOQuW~1pgWXjFW3w6W7ROWUnwZqUmjxt5ab0pgirqU8KKycP4CCqfBgJb1A2phGvxy-scDpXZhAJelEp6cNsHJmLM7QDOQjxajpanME9FgBVhOa7bKt~GDYns2pcpHIf8qN56CiWivI5W65vBAUFTpWlKwCqjY~FgYapLnLwC3yY4BWsbH3TyjUCzANPVzP8e6tKkZbdvsJYgSeIqCH1QHCD1sb4wXHOaxmkTNnzXsfM3vq-QejBvTQBEdgw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Quantification strategy of codon usage bias by bins relative to coding sequences’ length. From each original genome, two datasets were arranged, the first including only CDSs lacking signal peptides, and the second comprising only CDSs codifying proteins with signal peptides. Then, for both datasets, all CDSs were divided into ten parts (bins) and their codons quantified and saved in bins (Observed Matrix). Synonymous mutations were introduced, conserving the original codon usage of the genome but scrambling codon positions within genes, thus generating 200 whole simulated genomes per yeast. Codons were quantified by bins as stated in the ‘Material and methods’ section, and a 3D matrix of 10 × 59 × 200 was generated in order to retrieve the expected value and standard deviation for each codon after codon position alteration." path-from-xml="dsx014f1.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357460/dsx014f1.png" path-from-xml="dsx014f1.png" target="_blank">View large</a><a section="106357460" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/dsx014f1.png?Expires=2147483647&Signature=EM5tOjLS-KkzDPr6~r1jyvc6GLKkXyZRW3051sFGBx2VLzmNeTfy2g57hQnnyHNUFHBd9Ju51I9ZnbuUiGQ4hSzOVX0OFk8wEWWcrmzomMthdYwGZ1A7YPKJIt02GIjY887JBxteEHQAJjjnhB3CgtN3OEI-UlFepYYYz99~iHsKi6oKrYXdBELkx0mYe80yLtQ6Bp3qD9RUD2MNxbMl5fDLFjRXB06acYnjxnozAp0aluQeasPAl8Xkdhn7Ju09wrfl~LsQFC-rl49Lw9iQUgOVaYyGFkWxbTkqmYPWWg~e1qGSIKI6JszXV13fOtXCd04Eh86vaRrnQn4xiFCCzQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357460&ar=3752474&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx014f1.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Quantification strategy of codon usage bias by bins relative to coding sequences’ length. From each original genome, two datasets were arranged, the first including only CDSs lacking signal peptides, and the second comprising only CDSs codifying proteins with signal peptides. Then, for both datasets, all CDSs were divided into ten parts (bins) and their codons quantified and saved in bins (Observed Matrix). Synonymous mutations were introduced, conserving the original codon usage of the genome but scrambling codon positions within genes, thus generating 200 whole simulated genomes per yeast. Codons were quantified by bins as stated in the ‘Material and methods’ section, and a 3D matrix of 10 × 59 × 200 was generated in order to retrieve the expected value and standard deviation for each codon after codon position alteration.</p></div></div><div content-id="dsx014-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/m_dsx014f1.png?Expires=2147483647&Signature=pWOx~mti6ghWT~Mll-wkLyz1pcbi-MZWPrzPheat8fTnsWmRluw~423m3tK~E0L-cZJgmufNEylmMlR8~pJ09MxHxUTOQuW~1pgWXjFW3w6W7ROWUnwZqUmjxt5ab0pgirqU8KKycP4CCqfBgJb1A2phGvxy-scDpXZhAJelEp6cNsHJmLM7QDOQjxajpanME9FgBVhOa7bKt~GDYns2pcpHIf8qN56CiWivI5W65vBAUFTpWlKwCqjY~FgYapLnLwC3yY4BWsbH3TyjUCzANPVzP8e6tKkZbdvsJYgSeIqCH1QHCD1sb4wXHOaxmkTNnzXsfM3vq-QejBvTQBEdgw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Quantification strategy of codon usage bias by bins relative to coding sequences’ length. From each original genome, two datasets were arranged, the first including only CDSs lacking signal peptides, and the second comprising only CDSs codifying proteins with signal peptides. Then, for both datasets, all CDSs were divided into ten parts (bins) and their codons quantified and saved in bins (Observed Matrix). Synonymous mutations were introduced, conserving the original codon usage of the genome but scrambling codon positions within genes, thus generating 200 whole simulated genomes per yeast. Codons were quantified by bins as stated in the ‘Material and methods’ section, and a 3D matrix of 10 × 59 × 200 was generated in order to retrieve the expected value and standard deviation for each codon after codon position alteration." path-from-xml="dsx014f1.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357460/dsx014f1.png" path-from-xml="dsx014f1.png" target="_blank">View large</a><a section="106357460" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/dsx014f1.png?Expires=2147483647&Signature=EM5tOjLS-KkzDPr6~r1jyvc6GLKkXyZRW3051sFGBx2VLzmNeTfy2g57hQnnyHNUFHBd9Ju51I9ZnbuUiGQ4hSzOVX0OFk8wEWWcrmzomMthdYwGZ1A7YPKJIt02GIjY887JBxteEHQAJjjnhB3CgtN3OEI-UlFepYYYz99~iHsKi6oKrYXdBELkx0mYe80yLtQ6Bp3qD9RUD2MNxbMl5fDLFjRXB06acYnjxnozAp0aluQeasPAl8Xkdhn7Ju09wrfl~LsQFC-rl49Lw9iQUgOVaYyGFkWxbTkqmYPWWg~e1qGSIKI6JszXV13fOtXCd04Eh86vaRrnQn4xiFCCzQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357460&ar=3752474&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx014f1.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Quantification strategy of codon usage bias by bins relative to coding sequences’ length. From each original genome, two datasets were arranged, the first including only CDSs lacking signal peptides, and the second comprising only CDSs codifying proteins with signal peptides. Then, for both datasets, all CDSs were divided into ten parts (bins) and their codons quantified and saved in bins (Observed Matrix). Synonymous mutations were introduced, conserving the original codon usage of the genome but scrambling codon positions within genes, thus generating 200 whole simulated genomes per yeast. Codons were quantified by bins as stated in the ‘Material and methods’ section, and a 3D matrix of 10 × 59 × 200 was generated in order to retrieve the expected value and standard deviation for each codon after codon position alteration.</p></div></div><p>First, the start and stop codons were removed from all sequences and, in order to maintain uniformity in the length distribution of the different CDS sequences between the five yeasts, sequences with less than 40 codons were excluded from the dataset (i.e. sequences excluded for <em>K. lactis</em> = 0.02%, <em>K. marxianus</em> = 0%, <em>P. pastoris</em> = 0%, <em>S. cerevisiae</em> = 0.7% and <em>Y. lipolytica</em> = 0.1%). Thus, following the results of a recent report,<a reveal-id="dsx014-B42" data-open="dsx014-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> where it was demonstrated that no significant differences were detected when using different bin sizes, and guided by the result that shows ten as the best number of sections for visual exploration of the codon quantification, our method divides each gene into ten parts and saves the information of codon quantity per each tenth part of the CDSs thereby establishing ten bins with the same total number of codons. For each genome, the algorithm generated a matrix of 59 codons as rows (excluding the start, stop, and tryptophan codon counts) and 10 columns as bins.</p><p>Finally, to test the effectiveness of our method, we used the <em>E. coli</em> genome as a positive control (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_material_dsx014.pdf?Expires=2147483647&Signature=yrjqF1m4pc3Nh7lf8ZdqxQMqJLie-IFYrxEBJLcpqDMU9-oV5zGeyuesVkp5VQyxlzhU3zRsQ5V7s0CE4tzXC9VLZwjVNYLbCg71yM~MUu4INxBSLKGJz-v5sKsGC2imGQ4TSce61gCHMWq4eaQ-D76mDKsHmzubvAB-a2bBBxivZocHwwtUzcM4OJaXpsknqlOW6x-RXwQBYjVSIChlumeabOAPtOUxiQhOnoeMUW2xXepqOP3dagDP3chbKxvFxJPV7gB8t5xpDAGZBv7vi-2N6gKuBo-Iw7Rd5OQigzF-bhrHnjqYDtEm~gR1DIfN8gxdwbPU1O1xQZBFPTKMNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>), comparing it to previous reports.<a reveal-id="dsx014-B42" data-open="dsx014-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> As expected, comparable quantification values at the 5′-end were retrieved, and additionally, deviations from uniformity were detected at the 3′-end, a result which had not been observed by using a previous binning scheme.<a reveal-id="dsx014-B42" data-open="dsx014-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a></p>                            <h3 scrollto-destination=106357463 id="106357463" class="section-title" >2.6. Test for uniformity in CUB</h3>
<div class="block-child-p">After constructing the matrix of codons per bins, we tested the uniformity of distribution of each codon by calculating the χ<sup>2</sup> value<a reveal-id="dsx014-B42" data-open="dsx014-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> from <a href="#E3" class="sectionLink xref-sec">Equation (3)</a> as follows: <a id="E3" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E3"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msup><mi>χ</mi><mn>2</mn></msup><mo>=</mo><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>n</mi></munderover><mrow><msup><mi>Z</mi><mn>2</mn></msup></mrow><mo>=</mo><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>n</mi></munderover><mrow><mfrac><mrow><msup><mrow><mrow><mo stretchy="true">(</mo><mrow><mi>O</mi><mo>−</mo><mi>E</mi></mrow><mo stretchy="true">)</mo></mrow></mrow><mn>2</mn></msup></mrow><mrow><msup><mi>σ</mi><mn>2</mn></msup></mrow></mfrac></mrow></mrow></math></div></div><div class="label">(3)</div></div> where <em>O</em> is the observed count per bin in the original genome, <em>E</em> the expected value and σ the standard deviation of codon counts per bin obtained from the 200 simulated genomes, <em>n</em> the number of bins and <em>z</em> the <em>z</em>-score of each codon per bin. The squared <em>z</em>-score is the value related to deviations from uniformity, it tests the selection for or against uniformity in codon distribution as a function of intragenic position.</div><div class="block-child-p">In order to test the enrichment or depletion of codons as a function of the position within a CDS, we used the Y-value as described mathematically in <a href="#E4" class="sectionLink xref-sec">Equation 4</a>: <a id="E4" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E4"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>Y</mi><mo>=</mo><mi>L</mi><mi>o</mi><msub><mi>g</mi><mn>2</mn></msub><mrow><mo stretchy="true">(</mo><mrow><mfrac><mi>O</mi><mi>E</mi></mfrac></mrow><mo stretchy="true">)</mo></mrow></mrow></math></div></div><div class="label">(4)</div></div> when <em>O</em> (observed value) is higher than <em>E</em> (expected value), it indicates codon enrichment at that intragenic position and the resulting value is positive. On the contrary, a depletion is given by negative values.</div>                            <h3 scrollto-destination=106357466 id="106357466" class="section-title" >2.7. Codon conservation in protein structures</h3>
<p>To understand how a similar CAI of a gene could affect translation optimization, we analysed the sites where codon categories are conserved in the structure of proteins in a similar way as that reported in the literature.<a reveal-id="dsx014-B27" data-open="dsx014-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> We examined, as a proof of concept, two proteins conserved in the five yeasts which have similar values of CAI, but different expression levels under physiological conditions, namely, the elongation factor 1-alpha (TEF1a) which is highly expressed, and the heat-shock protein SSE1 which has low expression level. We retrieved the protein homologous sequences from UniProt<a reveal-id="dsx014-B53" data-open="dsx014-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> where available, or determined them by <em>blast</em> when not. The 3D structures were obtained from the Protein Data Bank, 1F60 structure for EF1A and 2QXL structure for SSE1. The structures were visualized, and their colours were edited using PyMOL (The PyMOL Molecular Graphics System, Version 1.7.4 Schrödinger, LLC.). A multiple sequence alignment of amino acids and codons was formulated using ClustalW<a reveal-id="dsx014-B54" data-open="dsx014-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> and edited in MEGA 6<a reveal-id="dsx014-B55" data-open="dsx014-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> in order to avoid gaps and maintain the integrity of the <em>S. cerevisiae</em> sequence as the model. The datasets P02994 for EF1A and P32589 for SSE1 from UniProtKB were used as a reference for the secondary structure motifs and 3D structural features.</p><p>To test whether codon conservation in protein secondary structures is a generalized phenomenon, we used the data from the PDB web site database (<a class="link link-uri" href="http://www.rcsb.org/pdb" target="_blank">http://www.rcsb.org/pdb</a>). The files were retrieved by advanced search using <em>Saccharomyces cerevisiae</em> as the organism source. The proteins were ordered according to their resolution and we recovered the 300 best resolution files (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_dataset_s2_dsx014.xlsx?Expires=2147483647&Signature=UvPuY4OGA0v6iQlgw~~~bWwee6dvmCeDA5fIGl7w9962MwPV3qmsBLJsHkA0rOeBXAoOaO~qSR4sYrCPkfVcLaFIVvJJqZX4xRXpUXq8Bsbzhx7ri5c45EhCMQ-zc0U-aNZ9o~S6CDSL8sGYCwiH60A7VsbY4noqyjAVtxdf6Qjm0q2qP2eDbLHA1TVjflf8UwEq4l3qGodD~CcXEIPnen4RUB8xK1jpqXoIcQzfOXN27lKEWuQDmkoZrHWkWjUV6lVR2anKuuYZFL-Kx~BCYvtSUYTK-xDtag90oypHSDNejyePca-8DGre4ZE8vGPegoOI6LHTMSAkcq5UXs5dQA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Dataset S2—PDB IDs/SGDIDs</a></span>). Then we removed the redundancy of these files applying the program VAST: Vector Alignment Search Tool.<a reveal-id="dsx014-B56" data-open="dsx014-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> The set-up parameters were the <em>P</em>-value of 10e−7 and display of “PDB codes only”. The outcome was 138 PDB files/proteins tagged as non-redundant PDB data set (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_dataset_s2_dsx014.xlsx?Expires=2147483647&Signature=UvPuY4OGA0v6iQlgw~~~bWwee6dvmCeDA5fIGl7w9962MwPV3qmsBLJsHkA0rOeBXAoOaO~qSR4sYrCPkfVcLaFIVvJJqZX4xRXpUXq8Bsbzhx7ri5c45EhCMQ-zc0U-aNZ9o~S6CDSL8sGYCwiH60A7VsbY4noqyjAVtxdf6Qjm0q2qP2eDbLHA1TVjflf8UwEq4l3qGodD~CcXEIPnen4RUB8xK1jpqXoIcQzfOXN27lKEWuQDmkoZrHWkWjUV6lVR2anKuuYZFL-Kx~BCYvtSUYTK-xDtag90oypHSDNejyePca-8DGre4ZE8vGPegoOI6LHTMSAkcq5UXs5dQA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Dataset S2—VAST</a></span>). To automatize this process we used an in-house script (Struct_<em>nonredund.cc</em>, see Availability section).</p><p>In the PDB file format documentation (<a class="link link-uri" href="http://www.wwpdb.org/documentation/file-format" target="_blank">http://www.wwpdb.org/documentation/file-format</a> (January 2017, date last accessed)), there are only two secondary directly specified structures: Helix and Sheet. Thus, the analysis of codon proportion homogeneity within helix and sheet structures was performed by Chi-square test using two frequency tables, one for Helix (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_dataset_s2_dsx014.xlsx?Expires=2147483647&Signature=UvPuY4OGA0v6iQlgw~~~bWwee6dvmCeDA5fIGl7w9962MwPV3qmsBLJsHkA0rOeBXAoOaO~qSR4sYrCPkfVcLaFIVvJJqZX4xRXpUXq8Bsbzhx7ri5c45EhCMQ-zc0U-aNZ9o~S6CDSL8sGYCwiH60A7VsbY4noqyjAVtxdf6Qjm0q2qP2eDbLHA1TVjflf8UwEq4l3qGodD~CcXEIPnen4RUB8xK1jpqXoIcQzfOXN27lKEWuQDmkoZrHWkWjUV6lVR2anKuuYZFL-Kx~BCYvtSUYTK-xDtag90oypHSDNejyePca-8DGre4ZE8vGPegoOI6LHTMSAkcq5UXs5dQA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Dataset S2—Helix</a></span>) and another for Sheet (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_dataset_s2_dsx014.xlsx?Expires=2147483647&Signature=UvPuY4OGA0v6iQlgw~~~bWwee6dvmCeDA5fIGl7w9962MwPV3qmsBLJsHkA0rOeBXAoOaO~qSR4sYrCPkfVcLaFIVvJJqZX4xRXpUXq8Bsbzhx7ri5c45EhCMQ-zc0U-aNZ9o~S6CDSL8sGYCwiH60A7VsbY4noqyjAVtxdf6Qjm0q2qP2eDbLHA1TVjflf8UwEq4l3qGodD~CcXEIPnen4RUB8xK1jpqXoIcQzfOXN27lKEWuQDmkoZrHWkWjUV6lVR2anKuuYZFL-Kx~BCYvtSUYTK-xDtag90oypHSDNejyePca-8DGre4ZE8vGPegoOI6LHTMSAkcq5UXs5dQA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Dataset S2—Sheet</a></span>) created by our in-house C ++ script (<em>Struct_frequencies.cc</em>, see Availability section). Finally, we tested codon proportionality through a position-dependent scheme which analysed the occurrence of codon categories at the first, second and third positions within each protein structure using an in-house script (<em>Struct_freq2.cc</em>). Next, the same analysis was applied to the <em>n</em> − 2, <em>n</em> − 1 and <em>n</em>, where <em>n</em> is the last codon of each structure per protein. Finally, as in PDB documentation there is not a defined coil region, we performed the coil analysis using the regions which do not belong to Helix and Sheet coordinates present in the non-redundant PDB files. The chi-square tests were also applied to the data retrieved from the coil region, i.e. loops, which connects helices and sheets. The scripts used for coil analyses can be found in Availability section (<em>Struct_freq2_coil.cc</em> and <em>Struct_freq3_coil.cc</em>).</p>                            <h3 scrollto-destination=106357470 id="106357470" class="section-title" >2.8. Computation of mRNA minimum free energy</h3>
<p>To evaluate the positional dependency of minimum free energy (MFE) in the mRNA secondary structures, we calculated the MFE in different positions of the genes by using the RNAfold program of the ViennaRNA package 2.0.<a reveal-id="dsx014-B57" data-open="dsx014-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> The MFE was computed for all the CDSs of each yeast (n<em><sub>K.</sub><sub>lactis</sub></em>= 5065, n<em><sub>K.</sub></em><sub><em>marxianus</em></sub> = 4774, n<em><sub>P.</sub></em><sub><em>pastoris</em></sub> = 5019, n<em><sub>S.</sub><sub>cerevisiae</sub> = </em>5786, <em>n<sub>Y.</sub></em><sub><em>lipolytica</em></sub> = 6413). Ribosomal genes from each yeast genome were used as a subset of highly translated transcripts.</p>                            <h3 scrollto-destination=106357472 id="106357472" class="section-title" >2.9. Statistical tests</h3>
<p>The <em>P-</em>values (<a reveal-id="dsx014-F2" data-open="dsx014-F2" class="link link-reveal link-table xref-fig">Figs 2B and 5A</a>) are the probabilities related to the Wilcoxon rank sum test (Mann–Whitney test), a non-parametric test for expected values with a two-sided alternative hypothesis for independent samples. For χ<a reveal-id="dsx014-B2" data-open="dsx014-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> calculations, we report the significance of each codon by a <em>P-</em>value &lt; 0.00017 after a Bonferroni correction for 59 tests where <em>P-</em>value = 0.01 contrasting them with the χ<a reveal-id="dsx014-B2" data-open="dsx014-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> distributions with <em>n</em> − 1 degrees of freedom.</p>                            <h2 scrollto-destination=106357474 id="106357474" class="section-title" >3. Results and discussion</h2>
                            <h3 scrollto-destination=106357475 id="106357475" class="section-title" >3.1. Yeast cell factories</h3>
<p>The metabolic diversity encoded in the yeast’s genome has led the scientific community to explore its capability at the industrial level as cell factories to produce value-added chemicals and enzymes. To determine the codon usage particularities in yeasts of biotechnological relevance, we used data mining to retrieve the most researched yeast species using as database all the articles in the collection of “Yeast Biotechnology” from the Microbial Cell Factories journal. It was observed that 104 papers, relating to 15 different yeast species, have been published in the last four years (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_material_dsx014.pdf?Expires=2147483647&Signature=yrjqF1m4pc3Nh7lf8ZdqxQMqJLie-IFYrxEBJLcpqDMU9-oV5zGeyuesVkp5VQyxlzhU3zRsQ5V7s0CE4tzXC9VLZwjVNYLbCg71yM~MUu4INxBSLKGJz-v5sKsGC2imGQ4TSce61gCHMWq4eaQ-D76mDKsHmzubvAB-a2bBBxivZocHwwtUzcM4OJaXpsknqlOW6x-RXwQBYjVSIChlumeabOAPtOUxiQhOnoeMUW2xXepqOP3dagDP3chbKxvFxJPV7gB8t5xpDAGZBv7vi-2N6gKuBo-Iw7Rd5OQigzF-bhrHnjqYDtEm~gR1DIfN8gxdwbPU1O1xQZBFPTKMNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). However, most of them were limited to five species: <em>Kluyveromyces lactis</em>, <em>Kluyveromyces marxianus</em>, <em>Pichia pastoris</em>, <em>Saccharomyces cerevisiae</em> and <em>Yarrowia lipolytica</em>. It should be noted that there was an increase in the proportion of works published on non-conventional yeasts compared with the conventional <em>S. cerevisiae</em> (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_material_dsx014.pdf?Expires=2147483647&Signature=yrjqF1m4pc3Nh7lf8ZdqxQMqJLie-IFYrxEBJLcpqDMU9-oV5zGeyuesVkp5VQyxlzhU3zRsQ5V7s0CE4tzXC9VLZwjVNYLbCg71yM~MUu4INxBSLKGJz-v5sKsGC2imGQ4TSce61gCHMWq4eaQ-D76mDKsHmzubvAB-a2bBBxivZocHwwtUzcM4OJaXpsknqlOW6x-RXwQBYjVSIChlumeabOAPtOUxiQhOnoeMUW2xXepqOP3dagDP3chbKxvFxJPV7gB8t5xpDAGZBv7vi-2N6gKuBo-Iw7Rd5OQigzF-bhrHnjqYDtEm~gR1DIfN8gxdwbPU1O1xQZBFPTKMNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). This being the case, we decided to work with the aforementioned five species.</p>                            <h3 scrollto-destination=106357477 id="106357477" class="section-title" >3.2. Strong signals of non-uniform distribution of codons relative to CDS length</h3>
<p>We evaluated the positional dependency of CUB, integrating recent findings which state that gene expression levels in eukaryotes are shaped by CDS length and CUB,<a reveal-id="dsx014-B58" data-open="dsx014-B58" class="link link-ref link-reveal xref-bibr"><sup>58–63</sup></a> that also supported our scheme in order to overcome the drawback experienced by using a previous method,<a reveal-id="dsx014-B42" data-open="dsx014-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> i.e. codon comparison between genic regions distant from the start codon. We decided to create a binning scheme based on the quantification of codons as a function of their relative position within genes and in relation to the length of their CDSs, forming ten bins by position (<a reveal-id="dsx014-F1" data-open="dsx014-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>) to identify increased signals of deviations from uniformity [squared <em>z</em>-score from <a href="#E3" class="sectionLink xref-sec">Equation (3)</a>] as a function of the relative intragenic position.</p><p>Previous studies have evaluated CUB, using bins with fixed lengths to analyse genes of both prokaryotes<a reveal-id="dsx014-B42" data-open="dsx014-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> and eukaryotes.<a reveal-id="dsx014-B30" data-open="dsx014-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> These studies reported that the 5′-end region presented unusual CUB behaviour. A detailed inspection of the results presented in the Tuller <em>et al.</em> manuscript,<a reveal-id="dsx014-B30" data-open="dsx014-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> shows that the <em>E. coli</em> genome had substantial variation in standard deviations in the 3′-end of the averaged translation efficiency (as measured by the tRNA adaptation index). However, in a recent study, significant signals of deviations from uniformity (given by the <em>z</em>-score of the chi-square test) were not detected in the 3′-end of <em>E. coli</em>.<a reveal-id="dsx014-B42" data-open="dsx014-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> Even though each is a different measurement of CUB, we would expect that, under a positional dependency approach, deviations from uniformity would also be significant at the 3′-end region as codons contribute differentially to the translation efficiency.</p><p>We consider that this drawback may be due to the use of the fixed length of bins for deviations from uniformity computation, since the algorithm that uses the fixed length of bins is not capable of comparing the equivalent region of two CDSs (i.e. neither the middle nor the 3′-end) when the difference in CDS length is great. For example, if we have one CDS (CDS-A) with 200 codons and another with 100 codons (CDS-B), in a fixed configuration, we will be doing a comparison between the middle region of CDS-A and the 3′-end region of CDS-B which will give an inconsistent result. It is important to point out that our proposed binning scheme is aimed at overcoming this drawback.</p><p>Bearing this in mind, we hypothesized that, based on the correlation of both CUB and CDS length, increased selection signals against the uniformity of codon distribution at different positions on the genes would be observed, taking into account the relative position of codons by gene, instead of an absolute pre-established quantity of codons per bin. In order to test this, we implemented our method and used published results<a reveal-id="dsx014-B42" data-open="dsx014-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> as the positive control, drawing on research by authors who analysed the <em>Escherichia coli</em> genome and quantified high selection values against the uniformity of many codons at the 5′-end of the genes. As expected, by using our proposed codon quantification scheme on the same <em>E. coli</em> genome, we found comparable patterns of codon deviations from uniformity at the 5′-end (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_material_dsx014.pdf?Expires=2147483647&Signature=yrjqF1m4pc3Nh7lf8ZdqxQMqJLie-IFYrxEBJLcpqDMU9-oV5zGeyuesVkp5VQyxlzhU3zRsQ5V7s0CE4tzXC9VLZwjVNYLbCg71yM~MUu4INxBSLKGJz-v5sKsGC2imGQ4TSce61gCHMWq4eaQ-D76mDKsHmzubvAB-a2bBBxivZocHwwtUzcM4OJaXpsknqlOW6x-RXwQBYjVSIChlumeabOAPtOUxiQhOnoeMUW2xXepqOP3dagDP3chbKxvFxJPV7gB8t5xpDAGZBv7vi-2N6gKuBo-Iw7Rd5OQigzF-bhrHnjqYDtEm~gR1DIfN8gxdwbPU1O1xQZBFPTKMNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). Nevertheless, it is important to point out that we also obtained significant deviations from uniformity in a number of codons at the 3′-end of genes, which had not been detected by the previously reported binning scheme. Thus, as shown in the positive control, our method allows for the detection of deviations from uniformity at both the 5′- and 3′-gene extremities.</p><p>The strong selection against uniformity detected at both gene extremities contrasts with the absence thereof in intragenic regions (distant from the extremities) of the CDSs. Considering that selection forces on CUB are determinant features of protein biosynthesis,<a reveal-id="dsx014-B64" data-open="dsx014-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a> our proposed method is useful for describing the codon distribution patterns in endogenous genes and consequently further codon optimization. These results are explained principally by the accuracy of our binning scheme which is able to retrieve the same quantity of codons per bin, without binning bias at the 3′-, 5′- and all other intragenic positions. Therefore, our method exposes the differences, in terms of codon bias, between both ends and the inner regions of the CDSs, supporting recent findings described through ribosomal-protected footprint counts,<a reveal-id="dsx014-B65" data-open="dsx014-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a> where it was observed that codons have different footprint count distributions in a position-dependent profile, conserved along intragenic regions but divergent at the 5′- and 3′-end. Notably, a similar profile was observed when our genome-scale codon quantification method on the <em>E. coli</em> genome (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_material_dsx014.pdf?Expires=2147483647&Signature=yrjqF1m4pc3Nh7lf8ZdqxQMqJLie-IFYrxEBJLcpqDMU9-oV5zGeyuesVkp5VQyxlzhU3zRsQ5V7s0CE4tzXC9VLZwjVNYLbCg71yM~MUu4INxBSLKGJz-v5sKsGC2imGQ4TSce61gCHMWq4eaQ-D76mDKsHmzubvAB-a2bBBxivZocHwwtUzcM4OJaXpsknqlOW6x-RXwQBYjVSIChlumeabOAPtOUxiQhOnoeMUW2xXepqOP3dagDP3chbKxvFxJPV7gB8t5xpDAGZBv7vi-2N6gKuBo-Iw7Rd5OQigzF-bhrHnjqYDtEm~gR1DIfN8gxdwbPU1O1xQZBFPTKMNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>) and on five yeast species (<a reveal-id="dsx014-F2" data-open="dsx014-F2" class="link link-reveal link-table xref-fig">Fig. 2A and B</a>) was applied. </p>                        <a id="106357483" scrollto-destination="106357483"></a>
<div id="dsx014-F2" data-id="dsx014-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/m_dsx014f2.png?Expires=2147483647&Signature=2ICpCTqRNtNiCpWNlNuGrSjOeX4X36PdeFZBuNnXdR-yD9p~dxxGw6HyJ1TAOVpcj092ReDWWuAOq6s~s3B47JmtdUL2~rA-fGT~AXjzmrg77~1wbbUPHsWLscrq5R2hM9SQBD53dH0Fj7uXXVcOprw1OG2dRddbpHK6ZbEpGQdH7mT2VDoPcY-3P~pEfgt6XwDcAboyRYAvatB04iTSMGXIqWnGn9BxwGfUFGnRdHiNOFpoffgSC4ryVDIR6hPqLmdw~pcHaaYWtNvRSOevghBHNC0QTGWWH5L5pciWS~CC~m6i5AOHp4B~nzj4hbx0Sn-8gEhPm1yEyR7XtNeecA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The positional dependency of codon usage bias relative to coding sequences length. The figure shows the analysis applied to the set of coding sequences lacking signal peptides. (A) Deviations from uniformity (squared z-scores) are reported graphically to illustrate the codon usage deviations from uniformity at different intragenic regions, concentrated mainly at the 5′-end close to the start codon. Squared z-scores are presented according to the quadratically scaled bar. (B) Distribution of deviations from uniformity by bin, illustrating the differences between first bin (5′-end region) and subsequent bins. The Mann–Whitney test was applied to determine the significance of the differences." path-from-xml="dsx014f2.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357483/dsx014f2.png" path-from-xml="dsx014f2.png" target="_blank">View large</a><a section="106357483" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/dsx014f2.png?Expires=2147483647&Signature=CDD9crFaHWlAtQptWq2w4qar6R~nLPK5Zm3k2H5exuMHPTctLuiSoQAuUkQqA5UEoxTPKwduqtVaOzyjqsFQ9fm7hW0tuy2RJDcFpHPv2v2CQi~KPB7OcV7nB-YZu3BdsQVhes-7HhEBr5sM3I2ZPXGUlINDKVBjdM4ohqr2KlH4o30EHILmhjlSWQ4aFmcY25j3Zei4S2~lYtW0MFLqWS9JZ~YHS0nvxpxKQXY4oWj~f~dBIPGI5yF144XrKdNKKiKrX8gTe7IZfuclPlFnuU9JX-1o0sD4ZnKLSkMegu7qiktPo8Vk99DvkUIkUW0CCTQhDbgPx8~w4U-WVaHuAg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357483&ar=3752474&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx014f2.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The positional dependency of codon usage bias relative to coding sequences length. The figure shows the analysis applied to the set of coding sequences lacking signal peptides. (A) Deviations from uniformity (squared <em>z</em>-scores) are reported graphically to illustrate the codon usage deviations from uniformity at different intragenic regions, concentrated mainly at the 5′-end close to the start codon. Squared <em>z</em>-scores are presented according to the quadratically scaled bar. (B) Distribution of deviations from uniformity by bin, illustrating the differences between first bin (5′-end region) and subsequent bins. The Mann–Whitney test was applied to determine the significance of the differences.</p></div></div><div content-id="dsx014-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/m_dsx014f2.png?Expires=2147483647&Signature=2ICpCTqRNtNiCpWNlNuGrSjOeX4X36PdeFZBuNnXdR-yD9p~dxxGw6HyJ1TAOVpcj092ReDWWuAOq6s~s3B47JmtdUL2~rA-fGT~AXjzmrg77~1wbbUPHsWLscrq5R2hM9SQBD53dH0Fj7uXXVcOprw1OG2dRddbpHK6ZbEpGQdH7mT2VDoPcY-3P~pEfgt6XwDcAboyRYAvatB04iTSMGXIqWnGn9BxwGfUFGnRdHiNOFpoffgSC4ryVDIR6hPqLmdw~pcHaaYWtNvRSOevghBHNC0QTGWWH5L5pciWS~CC~m6i5AOHp4B~nzj4hbx0Sn-8gEhPm1yEyR7XtNeecA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The positional dependency of codon usage bias relative to coding sequences length. The figure shows the analysis applied to the set of coding sequences lacking signal peptides. (A) Deviations from uniformity (squared z-scores) are reported graphically to illustrate the codon usage deviations from uniformity at different intragenic regions, concentrated mainly at the 5′-end close to the start codon. Squared z-scores are presented according to the quadratically scaled bar. (B) Distribution of deviations from uniformity by bin, illustrating the differences between first bin (5′-end region) and subsequent bins. The Mann–Whitney test was applied to determine the significance of the differences." path-from-xml="dsx014f2.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357483/dsx014f2.png" path-from-xml="dsx014f2.png" target="_blank">View large</a><a section="106357483" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/dsx014f2.png?Expires=2147483647&Signature=CDD9crFaHWlAtQptWq2w4qar6R~nLPK5Zm3k2H5exuMHPTctLuiSoQAuUkQqA5UEoxTPKwduqtVaOzyjqsFQ9fm7hW0tuy2RJDcFpHPv2v2CQi~KPB7OcV7nB-YZu3BdsQVhes-7HhEBr5sM3I2ZPXGUlINDKVBjdM4ohqr2KlH4o30EHILmhjlSWQ4aFmcY25j3Zei4S2~lYtW0MFLqWS9JZ~YHS0nvxpxKQXY4oWj~f~dBIPGI5yF144XrKdNKKiKrX8gTe7IZfuclPlFnuU9JX-1o0sD4ZnKLSkMegu7qiktPo8Vk99DvkUIkUW0CCTQhDbgPx8~w4U-WVaHuAg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357483&ar=3752474&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx014f2.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The positional dependency of codon usage bias relative to coding sequences length. The figure shows the analysis applied to the set of coding sequences lacking signal peptides. (A) Deviations from uniformity (squared <em>z</em>-scores) are reported graphically to illustrate the codon usage deviations from uniformity at different intragenic regions, concentrated mainly at the 5′-end close to the start codon. Squared <em>z</em>-scores are presented according to the quadratically scaled bar. (B) Distribution of deviations from uniformity by bin, illustrating the differences between first bin (5′-end region) and subsequent bins. The Mann–Whitney test was applied to determine the significance of the differences.</p></div></div>                            <h3 scrollto-destination=106357484 id="106357484" class="section-title" >3.3. The positional dependency of the genome-scale CUB in yeast</h3>
<p>Using our codon quantification method, which had been previously tested on the <em>E. coli</em> genome, we evaluated the position-dependent CUB in the yeast cell factory genomes. For this, a chi-square test per codon [<a href="#E3" class="sectionLink xref-sec">Equation (3)</a>] was conducted after filtering out the genes encoding proteins with signal peptides (SPs) in order to avoid biases attributable to their particular CUB.<a reveal-id="dsx014-B26" data-open="dsx014-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a><sup>,</sup><a reveal-id="dsx014-B43" data-open="dsx014-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a><sup>,</sup><a reveal-id="dsx014-B44" data-open="dsx014-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a><sup>,</sup><a reveal-id="dsx014-B66" data-open="dsx014-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> In a subsequent experiment, the same equation and methods were used to detect deviations from uniformity in the CDSs containing SPs. The occurrence of codons by bin was counted in the original (observed) genome and then compared with a null model formed by the expected value and standard deviation of 200 simulated genomes, where each genome conserved overall gene CUB but is produced with random codon positional distribution (see details in Material and Methods). This allowed us to determine the significance of the original CUB genome against a null model, calculating the squared <em>z</em>-score (deviations from uniformity) as a function of codon position and CDS length.</p><p>Out of 59 redundant codons (<a reveal-id="dsx014-F2" data-open="dsx014-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>), we found 37 in <em>K. lactis</em>, 39 in <em>K. marxianus</em>, 14 in <em>P. pastoris</em>, 16 in <em>S. cerevisiae</em> and 50 in <em>Y. lipolytica</em> with significant non-uniform distribution in terms of their position (<em>P-</em>value &lt; 0.00017, chi-square test after Bonferroni correction; <span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_material_dsx014.pdf?Expires=2147483647&Signature=yrjqF1m4pc3Nh7lf8ZdqxQMqJLie-IFYrxEBJLcpqDMU9-oV5zGeyuesVkp5VQyxlzhU3zRsQ5V7s0CE4tzXC9VLZwjVNYLbCg71yM~MUu4INxBSLKGJz-v5sKsGC2imGQ4TSce61gCHMWq4eaQ-D76mDKsHmzubvAB-a2bBBxivZocHwwtUzcM4OJaXpsknqlOW6x-RXwQBYjVSIChlumeabOAPtOUxiQhOnoeMUW2xXepqOP3dagDP3chbKxvFxJPV7gB8t5xpDAGZBv7vi-2N6gKuBo-Iw7Rd5OQigzF-bhrHnjqYDtEm~gR1DIfN8gxdwbPU1O1xQZBFPTKMNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). These codons are highlighted with an asterisk symbol in <a reveal-id="dsx014-F2" data-open="dsx014-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>. Analyzing graphically the squared <em>z</em>-scores [<a href="#E3" class="sectionLink xref-sec">Equation (3)</a>] of the chi-square statistics, it was possible to observe greater uniformity deviations at both gene extremities, mainly at the 5′-end, and low values, close to zero, in intermediate gene positions. In addition, a Mann–Whitney test was applied because we were interested in testing the meaningful difference of the <em>z</em>-score distribution at the 5′-end bin to evaluate the statistical significance of the differences between deviations from uniformity by position as a whole.</p><p>Remarkably, using our quantification method constrained by the relativity of CDS length, the statistical test revealed that there are strong differences in codon selection against uniformity between the first part and subsequent position within genes. Moreover, when we tested the uniformity deviations at different positions within CDSs, we found significant differences (<em>P</em>-value ≤ 0.002) between the first and second bin for all yeasts, except for <em>K. marxianus</em> in which higher significance was found between the second and third bin, although we found higher squared <em>z</em>-score values at the 5′-end of CDSs compared with <em>K. lactis</em>, <em>P. pastoris</em> and <em>S. cerevisiae</em>. Accordingly, when we divided all the CDSs into an equal amount of parts (ten bins in this case), all yeasts had the highest deviations from uniformity at the 5′-end of genes. Furthermore, it should be noted that we found high squared <em>z</em>-scores at the 3′-end (<a reveal-id="dsx014-F2" data-open="dsx014-F2" class="link link-reveal link-table xref-fig">Fig. 2A and B</a>), suggesting that certain codons also present strong selection against uniformity at this extremity in agreement with the results obtained in a recent study.<a reveal-id="dsx014-B65" data-open="dsx014-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a> Consequently, the results reported here denote the contribution of individual codons to the generation of strong deviations from uniformity at the extremities of CDSs, showing how the ramp theory is also presented as a function of CDS length at the evolutionary level in yeast.</p>                            <h3 scrollto-destination=106357488 id="106357488" class="section-title" >3.4. Genome-scale signals of frequent codons with low optimality</h3>
<p>Because we were interested in determining the characteristics of frequency and optimality of the translation efficiency of each codon, we calculated two broadly used indexes: the Relative Synonymous Codon Usage (RSCU) using the codon usage table (CUSP) of each genome (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_material_dsx014.pdf?Expires=2147483647&Signature=yrjqF1m4pc3Nh7lf8ZdqxQMqJLie-IFYrxEBJLcpqDMU9-oV5zGeyuesVkp5VQyxlzhU3zRsQ5V7s0CE4tzXC9VLZwjVNYLbCg71yM~MUu4INxBSLKGJz-v5sKsGC2imGQ4TSce61gCHMWq4eaQ-D76mDKsHmzubvAB-a2bBBxivZocHwwtUzcM4OJaXpsknqlOW6x-RXwQBYjVSIChlumeabOAPtOUxiQhOnoeMUW2xXepqOP3dagDP3chbKxvFxJPV7gB8t5xpDAGZBv7vi-2N6gKuBo-Iw7Rd5OQigzF-bhrHnjqYDtEm~gR1DIfN8gxdwbPU1O1xQZBFPTKMNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S3–S7</a></span>) and the codon optimality (<em>w</em>, formally defined as the codon relative adaptiveness) to the tRNA pool using published data,<a reveal-id="dsx014-B30" data-open="dsx014-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> and when required, we predicted the tRNA counts (see Materials and Methods, tRNA counts in <span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_material_dsx014.pdf?Expires=2147483647&Signature=yrjqF1m4pc3Nh7lf8ZdqxQMqJLie-IFYrxEBJLcpqDMU9-oV5zGeyuesVkp5VQyxlzhU3zRsQ5V7s0CE4tzXC9VLZwjVNYLbCg71yM~MUu4INxBSLKGJz-v5sKsGC2imGQ4TSce61gCHMWq4eaQ-D76mDKsHmzubvAB-a2bBBxivZocHwwtUzcM4OJaXpsknqlOW6x-RXwQBYjVSIChlumeabOAPtOUxiQhOnoeMUW2xXepqOP3dagDP3chbKxvFxJPV7gB8t5xpDAGZBv7vi-2N6gKuBo-Iw7Rd5OQigzF-bhrHnjqYDtEm~gR1DIfN8gxdwbPU1O1xQZBFPTKMNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). We used the RSCU values [<a href="#E1" class="sectionLink xref-sec">Equation (1)</a>] to characterize a codon as frequent or rare (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_material_dsx014.pdf?Expires=2147483647&Signature=yrjqF1m4pc3Nh7lf8ZdqxQMqJLie-IFYrxEBJLcpqDMU9-oV5zGeyuesVkp5VQyxlzhU3zRsQ5V7s0CE4tzXC9VLZwjVNYLbCg71yM~MUu4INxBSLKGJz-v5sKsGC2imGQ4TSce61gCHMWq4eaQ-D76mDKsHmzubvAB-a2bBBxivZocHwwtUzcM4OJaXpsknqlOW6x-RXwQBYjVSIChlumeabOAPtOUxiQhOnoeMUW2xXepqOP3dagDP3chbKxvFxJPV7gB8t5xpDAGZBv7vi-2N6gKuBo-Iw7Rd5OQigzF-bhrHnjqYDtEm~gR1DIfN8gxdwbPU1O1xQZBFPTKMNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>) and the codon adaptiveness (<em>w</em>) to characterize codons as optimal when <em>w</em> ≥ 0.4 or non-optimal if <em>w </em>&lt; 0.4 (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_material_dsx014.pdf?Expires=2147483647&Signature=yrjqF1m4pc3Nh7lf8ZdqxQMqJLie-IFYrxEBJLcpqDMU9-oV5zGeyuesVkp5VQyxlzhU3zRsQ5V7s0CE4tzXC9VLZwjVNYLbCg71yM~MUu4INxBSLKGJz-v5sKsGC2imGQ4TSce61gCHMWq4eaQ-D76mDKsHmzubvAB-a2bBBxivZocHwwtUzcM4OJaXpsknqlOW6x-RXwQBYjVSIChlumeabOAPtOUxiQhOnoeMUW2xXepqOP3dagDP3chbKxvFxJPV7gB8t5xpDAGZBv7vi-2N6gKuBo-Iw7Rd5OQigzF-bhrHnjqYDtEm~gR1DIfN8gxdwbPU1O1xQZBFPTKMNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>) in agreement with a previous study, in which <em>w</em> had been assessed in yeast genomes.<a reveal-id="dsx014-B27" data-open="dsx014-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a></p><p>Based on principles of codon optimization for heterologous protein expression in microorganisms, we expected to observe a high frequency of optimal codons, and complementarily, a low frequency of non-optimal codons. Visual inspection of the frequency values (RSCU in <a reveal-id="dsx014-F3" data-open="dsx014-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>), reveals that they were very similar among <em>K. lactis</em>, <em>K. marxianus</em>, <em>P. pastoris</em> and <em>S. cerevisiae</em>; however, the optimality of codons (<em>w</em> values) showed significant variations (<a reveal-id="dsx014-F3" data-open="dsx014-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>). On the other hand, we observed variations in <em>Y. lipolytica</em> in both codon frequency and optimality when compared with the other species. Intriguingly, we found for each genome heterogeneous correspondences between frequency/rarity and optimality/non-optimality. In contrast to the expected, we observed that several rarely used codons have optimal translation efficiency whereas frequently used codons have non-optimal translation efficiency. </p>                        <a id="106357491" scrollto-destination="106357491"></a>
<div id="dsx014-F3" data-id="dsx014-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/m_dsx014f3.png?Expires=2147483647&Signature=KuB2btbBVnCoIO2oubcNKzaJgCOJVfInfPcG4Sx4dy850d5UFJ9GlswqHalUh2E7y3rbaRjiLh6KOEZONikhW3Hei~HYyswJ~CPxYiK2w9zDvF1IF8zRLXwsaQIMpEAXqaoYmPxgGBEpeNWz3al6II7a~2J40LLB8D5nc9-FBQ5xu9qdd0xKU~Kt0pGVIPLoK68tgasIRbU72hEfUPxUC0Ijrml0aadH-EcntC2~DecDvRR9CuItDr-PAgZYqJHLyxK7vcISsX4nhrqBAl5T3uOVt7RW4BHJV18797FULQkADPpD5UyEDMTLBmmocSvs4Y8ePanHofCPflnWgZ1L5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Heterogeneous correspondences between codon frequency and codon optimality metrics. (A) The values of the Relative Synonymous Codon Usage (RSCU) are shown here in order to characterize each codon in terms of frequency or rarity. Values near to the centre and inside the shaded region with black borderline are frequent codons, those outside the circle are rare. (B) Values of Codon Adaptiveness (w) to cognate tRNAs from the tRNA Adaptation Index (tAI) are represented to illustrate the translation efficiency of each codon. Points inside the shaded region, far apart from the centre of the plot represent “optimal” codons, opposite points near to the centre and inside the white region indicate non-optimal codons." path-from-xml="dsx014f3.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357491/dsx014f3.png" path-from-xml="dsx014f3.png" target="_blank">View large</a><a section="106357491" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/dsx014f3.png?Expires=2147483647&Signature=GRkSwzgWXjyj-cgmyWSN5OUNQnie6oemxe90XyIN-n9Vqhw8BEWDRdDxtHuk1T9TI1yNZsCwIJGNvZcbv3ZdlEYG0BV5qB7dL-RXImqaF2Ac7zbjq6hoxocDVNbYjuFSbu~rb8GFr9rXQMdXQwR4U5KmO4rQI9lYYAkNcugZnJhStcAQS2BVfsEHSH4-9dVmQt4-Dd~JobgN7xy1vGAfAwpsH5BKze2-SN~AEyYkROYkx~jHiiB~BvhErcgfECsZdGo53irRWMbc-zVPAlrTkIgPLK-2NsUDF0XkE3~WxGiA-4b5CKboQut-ujlhFecd~Cpkpaor1xbuakzjteCUfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357491&ar=3752474&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx014f3.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Heterogeneous correspondences between codon frequency and codon optimality metrics. (A) The values of the Relative Synonymous Codon Usage (RSCU) are shown here in order to characterize each codon in terms of frequency or rarity. Values near to the centre and inside the shaded region with black borderline are frequent codons, those outside the circle are rare. (B) Values of Codon Adaptiveness (<em>w</em>) to cognate tRNAs from the tRNA Adaptation Index (tAI) are represented to illustrate the translation efficiency of each codon. Points inside the shaded region, far apart from the centre of the plot represent “optimal” codons, opposite points near to the centre and inside the white region indicate non-optimal codons.</p></div></div><div content-id="dsx014-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/m_dsx014f3.png?Expires=2147483647&Signature=KuB2btbBVnCoIO2oubcNKzaJgCOJVfInfPcG4Sx4dy850d5UFJ9GlswqHalUh2E7y3rbaRjiLh6KOEZONikhW3Hei~HYyswJ~CPxYiK2w9zDvF1IF8zRLXwsaQIMpEAXqaoYmPxgGBEpeNWz3al6II7a~2J40LLB8D5nc9-FBQ5xu9qdd0xKU~Kt0pGVIPLoK68tgasIRbU72hEfUPxUC0Ijrml0aadH-EcntC2~DecDvRR9CuItDr-PAgZYqJHLyxK7vcISsX4nhrqBAl5T3uOVt7RW4BHJV18797FULQkADPpD5UyEDMTLBmmocSvs4Y8ePanHofCPflnWgZ1L5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Heterogeneous correspondences between codon frequency and codon optimality metrics. (A) The values of the Relative Synonymous Codon Usage (RSCU) are shown here in order to characterize each codon in terms of frequency or rarity. Values near to the centre and inside the shaded region with black borderline are frequent codons, those outside the circle are rare. (B) Values of Codon Adaptiveness (w) to cognate tRNAs from the tRNA Adaptation Index (tAI) are represented to illustrate the translation efficiency of each codon. Points inside the shaded region, far apart from the centre of the plot represent “optimal” codons, opposite points near to the centre and inside the white region indicate non-optimal codons." path-from-xml="dsx014f3.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357491/dsx014f3.png" path-from-xml="dsx014f3.png" target="_blank">View large</a><a section="106357491" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/dsx014f3.png?Expires=2147483647&Signature=GRkSwzgWXjyj-cgmyWSN5OUNQnie6oemxe90XyIN-n9Vqhw8BEWDRdDxtHuk1T9TI1yNZsCwIJGNvZcbv3ZdlEYG0BV5qB7dL-RXImqaF2Ac7zbjq6hoxocDVNbYjuFSbu~rb8GFr9rXQMdXQwR4U5KmO4rQI9lYYAkNcugZnJhStcAQS2BVfsEHSH4-9dVmQt4-Dd~JobgN7xy1vGAfAwpsH5BKze2-SN~AEyYkROYkx~jHiiB~BvhErcgfECsZdGo53irRWMbc-zVPAlrTkIgPLK-2NsUDF0XkE3~WxGiA-4b5CKboQut-ujlhFecd~Cpkpaor1xbuakzjteCUfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357491&ar=3752474&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx014f3.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Heterogeneous correspondences between codon frequency and codon optimality metrics. (A) The values of the Relative Synonymous Codon Usage (RSCU) are shown here in order to characterize each codon in terms of frequency or rarity. Values near to the centre and inside the shaded region with black borderline are frequent codons, those outside the circle are rare. (B) Values of Codon Adaptiveness (<em>w</em>) to cognate tRNAs from the tRNA Adaptation Index (tAI) are represented to illustrate the translation efficiency of each codon. Points inside the shaded region, far apart from the centre of the plot represent “optimal” codons, opposite points near to the centre and inside the white region indicate non-optimal codons.</p></div></div><p>From an evolutionary perspective, the evidence of codons which are frequently used but have non-optimal translation efficiency seems to indicate that there are selective pressures acting positively to conserve and use non-optimal codons, probably to guarantee the effectiveness of protein biosynthesis as elongation speed regulators. Therefore, we show that non-optimal and rare codons have different characteristics, and that optimal codons are different from frequent ones. Although they are referred to as synonymous in the literature,<a reveal-id="dsx014-B66" data-open="dsx014-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> these characteristics of frequency and optimality could be positively correlated for certain codons but not in all cases. Optimality refers to the adaptation to the cognate tRNA isoacceptors and frequency/rarity refers to the occurrence of a codon in the genome. Mixed characteristics of frequency and optimality should be considered in detail because they are different approaches used in the characterization of individual codons which could impact several molecular processes. Subsequently, we decided to explore the features of individual codons in depth.</p>                            <h3 scrollto-destination=106357493 id="106357493" class="section-title" >3.5. Characterization between the Frequency and Optimality of codons</h3>
<p>In order to characterize the different correspondences between the frequency and optimality of codons, we decided to study each codon in an integrated framework (<a reveal-id="dsx014-F4" data-open="dsx014-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>) using the codon frequency and individual codon optimality for translation efficiency and then compare them to the data of the CUB positional dependency (<a reveal-id="dsx014-F2" data-open="dsx014-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). It allowed us to define four categories for the classification of codons: (i) FreO, given by the intersection of high frequency and optimality; (ii) RareNO, given by low frequency (rare) and non-optimality; (iii) RareO, given by low frequency and optimality; and (iv) FreNO, given by high frequency and non-optimality. </p>                        <a id="106357495" scrollto-destination="106357495"></a>
<div id="dsx014-F4" data-id="dsx014-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/m_dsx014f4.png?Expires=2147483647&Signature=FwPkfFZvjqnRA-GLIKMXX02rMn8wq1wl1AitcuIofRCBI1ed27Vyipm4YicJsyk4aTmiE538iHh5FZ8~q8ff2Frp0YVmTKIX5k8zPB7Y~EyXmB~1-nYhijW0AWticr2-TM7b2Qm3HYdmR2zx5cibhO1JjY3mj2UgEKABIF1XlsS~TANMAlXNJWWXx7ZdtJtriIBiYTs4n3H7wnjnJJXMNzYYmmAtJ~dJ7P8zk8OAlVHLS7u-H7bsegAL2~r9h96IRZUZR3wxhFTP1M2zXOHFkLNOhTWK62vj3vTqtmkFw5ezpFcKQKWpgdWO6hFL3~4TjX-~JUNEW75N~IqF0s58NA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Integrated characterization of individual codons. (A) Each codon is characterized regarding its frequency and optimality. (B) Percentage of the four codon categories in the 59 redundant codons of each yeast genome. (C) The third-base composition of the four codon categories illustrates the differences between Yarrowia lipolytica and the other yeasts." path-from-xml="dsx014f4.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357495/dsx014f4.png" path-from-xml="dsx014f4.png" target="_blank">View large</a><a section="106357495" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/dsx014f4.png?Expires=2147483647&Signature=RYdu8Ht-LNCe99kOBIy-MQ2SExrFBDbuE~SxJM6oKsIptwATpSi5Q~ccd1SJ3vqkqR0gKAtUOEH84V9fgns06qqxvirvuBSab9BEdzzuehNjmOjA3ecYfvFKuDQiL4F8fcwBPNmWp3eAlXyTPF0AYD6PlQtJXPm-qlJ28UWICb05SAHsv4wfST89hg2N9V2hlKs8OsyJdJAjZRMC1hkuBEXhYsgyh6sIXVFhpWNab2zj-v5joTHAAGEIMurHtCxxmvlwNoW0rOv1QAXdDa3ZtYkDXaS7JuzFA5hOl3HhRPjBeJMdOwQGO-9NWJnpE1S~aYmdVUpq4LAySNmYdgVeug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357495&ar=3752474&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx014f4.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Integrated characterization of individual codons. (A) Each codon is characterized regarding its frequency and optimality. (B) Percentage of the four codon categories in the 59 redundant codons of each yeast genome. (C) The third-base composition of the four codon categories illustrates the differences between <em>Yarrowia lipolytica</em> and the other yeasts.</p></div></div><div content-id="dsx014-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/m_dsx014f4.png?Expires=2147483647&Signature=FwPkfFZvjqnRA-GLIKMXX02rMn8wq1wl1AitcuIofRCBI1ed27Vyipm4YicJsyk4aTmiE538iHh5FZ8~q8ff2Frp0YVmTKIX5k8zPB7Y~EyXmB~1-nYhijW0AWticr2-TM7b2Qm3HYdmR2zx5cibhO1JjY3mj2UgEKABIF1XlsS~TANMAlXNJWWXx7ZdtJtriIBiYTs4n3H7wnjnJJXMNzYYmmAtJ~dJ7P8zk8OAlVHLS7u-H7bsegAL2~r9h96IRZUZR3wxhFTP1M2zXOHFkLNOhTWK62vj3vTqtmkFw5ezpFcKQKWpgdWO6hFL3~4TjX-~JUNEW75N~IqF0s58NA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Integrated characterization of individual codons. (A) Each codon is characterized regarding its frequency and optimality. (B) Percentage of the four codon categories in the 59 redundant codons of each yeast genome. (C) The third-base composition of the four codon categories illustrates the differences between Yarrowia lipolytica and the other yeasts." path-from-xml="dsx014f4.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357495/dsx014f4.png" path-from-xml="dsx014f4.png" target="_blank">View large</a><a section="106357495" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/dsx014f4.png?Expires=2147483647&Signature=RYdu8Ht-LNCe99kOBIy-MQ2SExrFBDbuE~SxJM6oKsIptwATpSi5Q~ccd1SJ3vqkqR0gKAtUOEH84V9fgns06qqxvirvuBSab9BEdzzuehNjmOjA3ecYfvFKuDQiL4F8fcwBPNmWp3eAlXyTPF0AYD6PlQtJXPm-qlJ28UWICb05SAHsv4wfST89hg2N9V2hlKs8OsyJdJAjZRMC1hkuBEXhYsgyh6sIXVFhpWNab2zj-v5joTHAAGEIMurHtCxxmvlwNoW0rOv1QAXdDa3ZtYkDXaS7JuzFA5hOl3HhRPjBeJMdOwQGO-9NWJnpE1S~aYmdVUpq4LAySNmYdgVeug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357495&ar=3752474&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx014f4.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Integrated characterization of individual codons. (A) Each codon is characterized regarding its frequency and optimality. (B) Percentage of the four codon categories in the 59 redundant codons of each yeast genome. (C) The third-base composition of the four codon categories illustrates the differences between <em>Yarrowia lipolytica</em> and the other yeasts.</p></div></div><p>Our integrated framework revealed the remarkable features of individual codons. For example, cysteine is coded by both the TGC and TGT codons, presenting uniform distribution with regard to the position within genes (<a reveal-id="dsx014-F2" data-open="dsx014-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). In general, they have non-optimal adaptation to the tRNAs of each yeast genome (<a reveal-id="dsx014-F3" data-open="dsx014-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>), except for <em>P. pastoris.</em> In this yeast, the TGC codon is optimal for the tRNA pool (<a reveal-id="dsx014-F3" data-open="dsx014-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>); however, it is rarely used (<a reveal-id="dsx014-F3" data-open="dsx014-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>). On the other hand, codons of glutamine present a conserved non-uniform distribution as a function of the position within genes, possibly contributing to the formation of a selection ramp (<a reveal-id="dsx014-F2" data-open="dsx014-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>), with strong selection values against uniformity at the 5′-end in all yeasts. The CAA codon is frequent and optimal (<a reveal-id="dsx014-F4" data-open="dsx014-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>) for all the yeasts with the exception of <em>Y. lipolytica</em>, in which this codon is rare and non-optimal. In contrast, the CAG codon is rare and non-optimal in <em>K. lactis, K. marxianus</em> and <em>S. cerevisiae</em>, but frequent and optimal in <em>Y. lipolytica</em> (<a reveal-id="dsx014-F4" data-open="dsx014-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>). Another relevant feature is that, although CAG is rarely used and is non-optimal in the genomes of <em>K. lactis, K. marxianus</em> and <em>S. cerevisiae</em>, it is optimal in <em>P. pastoris</em> (<a reveal-id="dsx014-F4" data-open="dsx014-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>)<em>.</em> Thus, this is a characteristic that is particular to <em>P. pastoris.</em> Correspondingly, <em>P. pastoris</em> has a higher quantity of codons that are optimal but rarely used (<a reveal-id="dsx014-F4" data-open="dsx014-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a>), which seems to be an advantage for the production of recombinant proteins taking into account that these optimal codons, rarely used in the expression of endogenous genes, might be used more frequently in the expression of heterologous genes.</p><p>On the other hand, it was possible to observe the different properties in the codon selection features found in the genome of <em>Y. lipolytica</em> (<a reveal-id="dsx014-F4" data-open="dsx014-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a>) when compared with the other yeasts. Although it is broadly known that <em>Y. lipolytica</em> has a very different codon usage, this microorganism seems to be under strong positive pressure from optimal codons to use them frequently and, at the same time, a strong negative selection over non-optimal codons to use them rarely.</p><p>Finally, we found important differences when comparing the third base composition of the four codon categories. <em>Y. lipolytica</em> presents very low content of AT<sub>3</sub> in its frequent codons and high content in the rare ones. On the other hand, all the other yeasts showed high AT<sub>3</sub> content of frequent codons and very low, or zero content, in rare codons (<a reveal-id="dsx014-F4" data-open="dsx014-F4" class="link link-reveal link-table xref-fig">Fig. 4C</a>). It should be pointed out that codon optimality seems to be an inaccurate characteristic to yeasts categorization considering the third base composition (<a reveal-id="dsx014-F4" data-open="dsx014-F4" class="link link-reveal link-table xref-fig">Fig. 4C</a>). It is important to clarify that data integration through this approach has been achieved by using genome sequences of microorganisms, which exhibit codon usage bias positively associated with translation efficiency. Therefore, the application of this method should consider these features before applying it to future research in non-translationally efficient organisms. Nevertheless, with the properties highlighted here it is possible to define the contribution of each codon in terms of evolutionary selection, translation efficiency and whole genome occurrences for each yeast cell factory. This integrated framework stands as a valuable tool that contributes to the improvement of knowledge on the different roles of individual codons in endogenous genes.</p>                            <h3 scrollto-destination=106357499 id="106357499" class="section-title" >3.6. Rare–non-optimal codons have higher decoding time and are enriched at the beginning of CDSs</h3>
<p>The characterization of codons led us to address the contribution of each codon category to the protein elongation process. Based on this, and the following three main statements: first, codons are translated at different rates;<a reveal-id="dsx014-B26" data-open="dsx014-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a><sup>,</sup><a reveal-id="dsx014-B50" data-open="dsx014-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a><sup>,</sup><a reveal-id="dsx014-B65" data-open="dsx014-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a><sup>,</sup><a reveal-id="dsx014-B67" data-open="dsx014-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> second, we observed there are strong evolutionary selective pressures acting differentially on redundant codons to use them rarely or frequently; and third, codons are also differentially adapted to the tRNA pool in terms of efficiency, we hypothesized that it would be possible to find significant differences in the contribution of the four different categories of codons (FreNO, RareNO, etc.) to the kinetics of the elongation process.</p><p>In order to test this, for each codon we computed the codon decoding time [CDT, <a href="#E2" class="sectionLink xref-sec">Equation (2)</a>] as described previously<a reveal-id="dsx014-B65" data-open="dsx014-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a> and subsequently we formed four clusters of codons according to the categories revealed by the integrated framework. Finally, we analysed the distribution of CDT values, a measurement aimed at addressing the time required to decode each codon.</p><p>Confirming our hypothesis (<a reveal-id="dsx014-F5" data-open="dsx014-F5" class="link link-reveal link-table xref-fig">Fig. 5A</a>), rarely used non-optimal codons (RareNO) have significantly higher CDTs (<em>P </em>&lt; 0.05, Wilcoxon rank sum test) than frequent non-optimal (FreNO) codons, except in the case of <em>K. marxianus</em>. In this yeast, the statistical test was not significant. It should be noted (<a reveal-id="dsx014-F5" data-open="dsx014-F5" class="link link-reveal link-table xref-fig">Fig. 5A</a>) that certain RareNO codons of <em>K. marxianus</em> have CDT values higher than the highest CDT values of FreNO codons. However, there was no significant difference between the RareNO and FreNO groups in this yeast. This particularity could be due to difficulties in the <em>w</em> value computation used for calculating CDTs. Thus, it is expected that the future availability of <em>K. marxianus</em> protein expression data will allow for constraining the computation of their respective <em>w</em> values, which in turn will contribute to improving their CDT calculation. </p>                        <a id="106357503" scrollto-destination="106357503"></a>
<div id="dsx014-F5" data-id="dsx014-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/m_dsx014f5.png?Expires=2147483647&amp;Signature=QrJ-zibD8d7UQ04vbG8x-FSCuTAwnyrywoVnER1bqKnCyu3OR-2nPcous7wTCzSDOBi7wlLBs8E-4olz179PdXWedm2AjneaBOgzl0gaNJy13Q6QpkDZcrq2tyInJOnoTjfllGMsik86kSrgYQQwOG6z6yjsYCIgZ8hwinmzbgUH9B6NaT1h7f8~M6tOdjQGinJS11eH4EeHGTnz4BCNqPm~GSon75NdTrLBFhgd4SRRMRfXb8qjRaOcgrBtSeWiV2lqjPbyBdAIn6gN70vW12G69CjQjVIw5Eot3vnyIMGsEIRiPiDaJA6VedugbFeCwpMyBDnBDYJQOD77OCdBqw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Codon decoding time (CDT), the average of Y-values and the mRNA free folding energy are presented to indicate the features of each codon category. (A) shows the significant differences found when CDT values of Rare–Non-optimal (RareNO) are compared with Frequent–Optimal (FreNO) codons. The translation rate is the slowest when decoding RareNO codons. No significant differences between the optimal ones (FreO and RareO) were found. (B) RareNO codons seem to be enriched at regions near to start codons in K. lactis and other yeasts (Supplementary Fig. S8), while Frequent–Optimal (FreO) codons contrast that profile. FreNO codons are not significantly enriched or depleted as a function of position within genes, they are uniformly distributed inside genes, probably to guarantee the accuracy in protein biosynthesis as translation speed regulators (see Supplementary Figs S3–S7 for individual values of codons per bin). (C) mRNA structure is the lowest in the 5′-end region, where RareNO codons are enriched. The MFE was computed for all the CDSs of each yeast (nK. lactis= 5065, nK. marxianus = 4774, nP. pastoris = 5019, nS. cerevisiae = 5786, nY. lipolytica = 6413)." path-from-xml="dsx014f5.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357503/dsx014f5.png" path-from-xml="dsx014f5.png" target="_blank">View large</a><a section="106357503" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/dsx014f5.png?Expires=2147483647&amp;Signature=v2L~99oq3~6B6vNlyncGMjpysxEJZcQmRFrQg2H3WsxKUBdox~NoEg5blqm-KHNANPQ6DK39MUAg6LZfkKlpVynTeuqpX51rPEvbfEi4IT2DLqIKiaglSsNkDlWULxoI4Q-oG018dLrdou-AZdFsaQp~-F4wps-1UzijQPl9qr7Lm8GWpH70J7i15sc6n8ExBo3~nKXknSBS0bCgD4ZcADk7ThxjRPQZp2DHW7OlA2ji9cIXYZmNU6SLrmqYYoWnFIZEHDhIBJmqBqU-qgo~TGzQE4xKrJVbEuL8ahgHBijiEi6Ngyl5W9cVAFOl70zS9XqnO2Tpuz1FpzZC8grQUA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=106357503&amp;ar=3752474&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsx014f5.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Codon decoding time (CDT), the average of Y-values and the mRNA free folding energy are presented to indicate the features of each codon category. (A) shows the significant differences found when CDT values of Rare–Non-optimal (RareNO) are compared with Frequent–Optimal (FreNO) codons. The translation rate is the slowest when decoding RareNO codons. No significant differences between the optimal ones (FreO and RareO) were found. (B) RareNO codons seem to be enriched at regions near to start codons in <em>K. lactis</em> and other yeasts (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_material_dsx014.pdf?Expires=2147483647&amp;Signature=yrjqF1m4pc3Nh7lf8ZdqxQMqJLie-IFYrxEBJLcpqDMU9-oV5zGeyuesVkp5VQyxlzhU3zRsQ5V7s0CE4tzXC9VLZwjVNYLbCg71yM~MUu4INxBSLKGJz-v5sKsGC2imGQ4TSce61gCHMWq4eaQ-D76mDKsHmzubvAB-a2bBBxivZocHwwtUzcM4OJaXpsknqlOW6x-RXwQBYjVSIChlumeabOAPtOUxiQhOnoeMUW2xXepqOP3dagDP3chbKxvFxJPV7gB8t5xpDAGZBv7vi-2N6gKuBo-Iw7Rd5OQigzF-bhrHnjqYDtEm~gR1DIfN8gxdwbPU1O1xQZBFPTKMNA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>), while Frequent–Optimal (FreO) codons contrast that profile. FreNO codons are not significantly enriched or depleted as a function of position within genes, they are uniformly distributed inside genes, probably to guarantee the accuracy in protein biosynthesis as translation speed regulators (see <span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_material_dsx014.pdf?Expires=2147483647&amp;Signature=yrjqF1m4pc3Nh7lf8ZdqxQMqJLie-IFYrxEBJLcpqDMU9-oV5zGeyuesVkp5VQyxlzhU3zRsQ5V7s0CE4tzXC9VLZwjVNYLbCg71yM~MUu4INxBSLKGJz-v5sKsGC2imGQ4TSce61gCHMWq4eaQ-D76mDKsHmzubvAB-a2bBBxivZocHwwtUzcM4OJaXpsknqlOW6x-RXwQBYjVSIChlumeabOAPtOUxiQhOnoeMUW2xXepqOP3dagDP3chbKxvFxJPV7gB8t5xpDAGZBv7vi-2N6gKuBo-Iw7Rd5OQigzF-bhrHnjqYDtEm~gR1DIfN8gxdwbPU1O1xQZBFPTKMNA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S3–S7</a></span> for individual values of codons per bin). (C) mRNA structure is the lowest in the 5′-end region, where RareNO codons are enriched. The MFE was computed for all the CDSs of each yeast (<em>n<sub>K. lactis</sub></em>= 5065, <em>n<sub>K. marxianus</sub></em> = 4774, <em>n<sub>P. pastoris</sub></em> = 5019, <em>n<sub>S. cerevisiae</sub> = </em>5786, <em>n<sub>Y. lipolytica</sub></em> = 6413).</p></div></div><div content-id="dsx014-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/m_dsx014f5.png?Expires=2147483647&amp;Signature=QrJ-zibD8d7UQ04vbG8x-FSCuTAwnyrywoVnER1bqKnCyu3OR-2nPcous7wTCzSDOBi7wlLBs8E-4olz179PdXWedm2AjneaBOgzl0gaNJy13Q6QpkDZcrq2tyInJOnoTjfllGMsik86kSrgYQQwOG6z6yjsYCIgZ8hwinmzbgUH9B6NaT1h7f8~M6tOdjQGinJS11eH4EeHGTnz4BCNqPm~GSon75NdTrLBFhgd4SRRMRfXb8qjRaOcgrBtSeWiV2lqjPbyBdAIn6gN70vW12G69CjQjVIw5Eot3vnyIMGsEIRiPiDaJA6VedugbFeCwpMyBDnBDYJQOD77OCdBqw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Codon decoding time (CDT), the average of Y-values and the mRNA free folding energy are presented to indicate the features of each codon category. (A) shows the significant differences found when CDT values of Rare–Non-optimal (RareNO) are compared with Frequent–Optimal (FreNO) codons. The translation rate is the slowest when decoding RareNO codons. No significant differences between the optimal ones (FreO and RareO) were found. (B) RareNO codons seem to be enriched at regions near to start codons in K. lactis and other yeasts (Supplementary Fig. S8), while Frequent–Optimal (FreO) codons contrast that profile. FreNO codons are not significantly enriched or depleted as a function of position within genes, they are uniformly distributed inside genes, probably to guarantee the accuracy in protein biosynthesis as translation speed regulators (see Supplementary Figs S3–S7 for individual values of codons per bin). (C) mRNA structure is the lowest in the 5′-end region, where RareNO codons are enriched. The MFE was computed for all the CDSs of each yeast (nK. lactis= 5065, nK. marxianus = 4774, nP. pastoris = 5019, nS. cerevisiae = 5786, nY. lipolytica = 6413)." path-from-xml="dsx014f5.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357503/dsx014f5.png" path-from-xml="dsx014f5.png" target="_blank">View large</a><a section="106357503" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/dsx014f5.png?Expires=2147483647&amp;Signature=v2L~99oq3~6B6vNlyncGMjpysxEJZcQmRFrQg2H3WsxKUBdox~NoEg5blqm-KHNANPQ6DK39MUAg6LZfkKlpVynTeuqpX51rPEvbfEi4IT2DLqIKiaglSsNkDlWULxoI4Q-oG018dLrdou-AZdFsaQp~-F4wps-1UzijQPl9qr7Lm8GWpH70J7i15sc6n8ExBo3~nKXknSBS0bCgD4ZcADk7ThxjRPQZp2DHW7OlA2ji9cIXYZmNU6SLrmqYYoWnFIZEHDhIBJmqBqU-qgo~TGzQE4xKrJVbEuL8ahgHBijiEi6Ngyl5W9cVAFOl70zS9XqnO2Tpuz1FpzZC8grQUA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=106357503&amp;ar=3752474&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsx014f5.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Codon decoding time (CDT), the average of Y-values and the mRNA free folding energy are presented to indicate the features of each codon category. (A) shows the significant differences found when CDT values of Rare–Non-optimal (RareNO) are compared with Frequent–Optimal (FreNO) codons. The translation rate is the slowest when decoding RareNO codons. No significant differences between the optimal ones (FreO and RareO) were found. (B) RareNO codons seem to be enriched at regions near to start codons in <em>K. lactis</em> and other yeasts (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_material_dsx014.pdf?Expires=2147483647&amp;Signature=yrjqF1m4pc3Nh7lf8ZdqxQMqJLie-IFYrxEBJLcpqDMU9-oV5zGeyuesVkp5VQyxlzhU3zRsQ5V7s0CE4tzXC9VLZwjVNYLbCg71yM~MUu4INxBSLKGJz-v5sKsGC2imGQ4TSce61gCHMWq4eaQ-D76mDKsHmzubvAB-a2bBBxivZocHwwtUzcM4OJaXpsknqlOW6x-RXwQBYjVSIChlumeabOAPtOUxiQhOnoeMUW2xXepqOP3dagDP3chbKxvFxJPV7gB8t5xpDAGZBv7vi-2N6gKuBo-Iw7Rd5OQigzF-bhrHnjqYDtEm~gR1DIfN8gxdwbPU1O1xQZBFPTKMNA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>), while Frequent–Optimal (FreO) codons contrast that profile. FreNO codons are not significantly enriched or depleted as a function of position within genes, they are uniformly distributed inside genes, probably to guarantee the accuracy in protein biosynthesis as translation speed regulators (see <span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_material_dsx014.pdf?Expires=2147483647&amp;Signature=yrjqF1m4pc3Nh7lf8ZdqxQMqJLie-IFYrxEBJLcpqDMU9-oV5zGeyuesVkp5VQyxlzhU3zRsQ5V7s0CE4tzXC9VLZwjVNYLbCg71yM~MUu4INxBSLKGJz-v5sKsGC2imGQ4TSce61gCHMWq4eaQ-D76mDKsHmzubvAB-a2bBBxivZocHwwtUzcM4OJaXpsknqlOW6x-RXwQBYjVSIChlumeabOAPtOUxiQhOnoeMUW2xXepqOP3dagDP3chbKxvFxJPV7gB8t5xpDAGZBv7vi-2N6gKuBo-Iw7Rd5OQigzF-bhrHnjqYDtEm~gR1DIfN8gxdwbPU1O1xQZBFPTKMNA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S3–S7</a></span> for individual values of codons per bin). (C) mRNA structure is the lowest in the 5′-end region, where RareNO codons are enriched. The MFE was computed for all the CDSs of each yeast (<em>n<sub>K. lactis</sub></em>= 5065, <em>n<sub>K. marxianus</sub></em> = 4774, <em>n<sub>P. pastoris</sub></em> = 5019, <em>n<sub>S. cerevisiae</sub> = </em>5786, <em>n<sub>Y. lipolytica</sub></em> = 6413).</p></div></div><p>Additionally, the detection of higher CDTs conserved in RareNO codons seems to explain their low frequency in intermediate gene regions where the protein biosynthesis process should be highly efficient. This reveals that the decoding time of certain codons is a special feature that constrains codon selection, which explains why certain different non-optimal codons are used more frequently than others in yeast. Thus, we conclude that the insertion of RareNO codons in genes can facilitate slower protein elongation. In fact, RareNO codons introduce lower translation rates than FreNO codons, an important characteristic of note in the synthetic gene design.</p><p>The results described above also led us to hypothesize that, whether the ramp theory applies to translation efficiency or not,<a reveal-id="dsx014-B10" data-open="dsx014-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsx014-B30" data-open="dsx014-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a><sup>,</sup><a reveal-id="dsx014-B67" data-open="dsx014-B67" class="link link-ref link-reveal xref-bibr"><sup>67–70</sup></a> we should consider these signals in terms of richness or depletion by codon category as a function of position within CDSs. Hence, a higher occurrence of RareNO codons should be expected at the beginning of the genes which will modulate the initiation of elongation. At the same time, we would expect its absence in intragenic regions where the elongation process must be more efficient. On the other hand, it is reasonable to consider that FreNO codons, which are equally non-optimal as RareNO codons, but have lower decoding times, should be found at different positions on the genes. It is probable that they act as elongation speed regulators for protein co-translational folding,<a reveal-id="dsx014-B22" data-open="dsx014-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> and that elongation will not be slowed as much as when decoded by RareNO codons. To test this, we determined if the different categories of codons are enriched or depleted as a function of position in the ten bins. We calculated the Y-value [<a href="#E4" class="sectionLink xref-sec">Equation (4)</a>] as the ratio of observed (<em>O</em>) values to expected values (<em>E</em>) on a logarithmic scale, to determine the fold-change of observed codons against expected codon occurrences in simulated genomes with scrambled position of codons in their CDSs. The computation was carried out for each yeast genome (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_material_dsx014.pdf?Expires=2147483647&Signature=yrjqF1m4pc3Nh7lf8ZdqxQMqJLie-IFYrxEBJLcpqDMU9-oV5zGeyuesVkp5VQyxlzhU3zRsQ5V7s0CE4tzXC9VLZwjVNYLbCg71yM~MUu4INxBSLKGJz-v5sKsGC2imGQ4TSce61gCHMWq4eaQ-D76mDKsHmzubvAB-a2bBBxivZocHwwtUzcM4OJaXpsknqlOW6x-RXwQBYjVSIChlumeabOAPtOUxiQhOnoeMUW2xXepqOP3dagDP3chbKxvFxJPV7gB8t5xpDAGZBv7vi-2N6gKuBo-Iw7Rd5OQigzF-bhrHnjqYDtEm~gR1DIfN8gxdwbPU1O1xQZBFPTKMNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S3–S7</a></span>).</p><p>Interestingly, we found that RareNO codons are predominantly enriched at regions close to the start codon (<a reveal-id="dsx014-F5" data-open="dsx014-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a>) and depleted in distant regions. In general, FreO codons are predominantly depleted at the beginning of genes. In agreement with recent data,<a reveal-id="dsx014-B30" data-open="dsx014-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> we add important evidence to the theory of the translation ramp, with two remarkable additions: first, we suggests the existence of the ramp at the evolutionary level in yeast, and second, we found a specific group of codons (RareNO codons) enriched at the beginning of genes, which are responsible for the lowest translation rates. Moreover, we observed a conserving tendency in FreNO codons to avoid either enrichment or depletion. They were found uniformly distributed through the genomes of <em>K. lactis</em>, <em>K. marxianus</em>, <em>P. pastoris</em> and <em>S. cerevisiae</em> (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_material_dsx014.pdf?Expires=2147483647&amp;Signature=yrjqF1m4pc3Nh7lf8ZdqxQMqJLie-IFYrxEBJLcpqDMU9-oV5zGeyuesVkp5VQyxlzhU3zRsQ5V7s0CE4tzXC9VLZwjVNYLbCg71yM~MUu4INxBSLKGJz-v5sKsGC2imGQ4TSce61gCHMWq4eaQ-D76mDKsHmzubvAB-a2bBBxivZocHwwtUzcM4OJaXpsknqlOW6x-RXwQBYjVSIChlumeabOAPtOUxiQhOnoeMUW2xXepqOP3dagDP3chbKxvFxJPV7gB8t5xpDAGZBv7vi-2N6gKuBo-Iw7Rd5OQigzF-bhrHnjqYDtEm~gR1DIfN8gxdwbPU1O1xQZBFPTKMNA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>), but strongly enriched at the beginning of genes in <em>Y. lipolytica</em>, pointing out that this yeast also has the lowest percentage of FreNO codons in comparison to the other yeasts (<a reveal-id="dsx014-F4" data-open="dsx014-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a>). Another interesting point worthy of note is that our finding related to the enrichment of RareNO codons in the 5′-end region of endogenous genes in yeast could be associated with the maintaining of a decreased local mRNA structure (<a reveal-id="dsx014-F5" data-open="dsx014-F5" class="link link-reveal link-table xref-fig">Fig. 5C</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_dataset_s1_dsx014.xlsx?Expires=2147483647&amp;Signature=yn7ec37o6w~qyUL2X3g5iDJYPjZJ-UekkspdLw~wG9pMLv8LP7-VmOr3~JmsoUI5j-AEQ7ixLD3SSBktaex5CTjFnqyvch22~Y6KknPvypS8hUybL-t4LcS-2I9iKS752oDo-XuqsC1e6n3MaOkrBAoYUxSQBlBbjbxKZ5aPCRRgUqX7KUvXF5~re6xf2ZTH5-wrZx5pqS4x6WCf1FRqyhI0F8hJ7XaeKEctsOuTyEd6Zu~-A2x0zp8aFyO9hIDIB0REHmO4WVDkGqOI19Pco9gjbx2YlG2vNTOqA4H6~x95sXhsrRnp11EA1YfDbYmEnp1eKfcyqGsikaEkl1Hh~A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Dataset S1</a></span>). A similar result has been described recently in <em>E. coli</em>,<a reveal-id="dsx014-B42" data-open="dsx014-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> suggesting that rare codons assist to preserve reduced mRNA structure.</p><p>In our work, the local mRNA minimum folding energy (MFE) at the beginning of genes showed the lowest local mRNA structure when compared with intragenic positions distant from the 5′-end region. Our calculation indicates that it is a conserved characteristic when applied to the whole gene dataset of yeasts and to the highly expressed genes such as ribosomal proteins. Accordingly, this observation among mRNA, MFE and codon positional dependency in yeast suggests a similar phenomenon, as determined experimentally in bacteria,<a reveal-id="dsx014-B71" data-open="dsx014-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a> where rare codons reduced the secondary structure of mRNAs. Therefore, our data reinforce, with novel genome-scale characteristics, the ramp theory of codon usage bias.<a reveal-id="dsx014-B10" data-open="dsx014-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsx014-B30" data-open="dsx014-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a><sup>,</sup><a reveal-id="dsx014-B70" data-open="dsx014-B70" class="link link-ref link-reveal xref-bibr"><sup>70</sup></a> Additionally, the aforementioned findings validate the existence of the ramp at the evolutionary level and allow us to specify which codons are really contributing to slower translation speeds, illustrating the orchestration of selective constraints, translation efficiency, codon occurrence, local mRNA folding energies and the determinant contributions of individual codons, especially the imperative leadership of non-optimal codons, to the protein biosynthesis.</p>                            <h3 scrollto-destination=106357508 id="106357508" class="section-title" >3.7. Genes with signal peptides use preferentially RareNO codons at the 5′-end</h3>
<p>Signal peptide (SP) design and its synthetic construction is an effective method used in industrial biotechnology to improve heterologous protein production and biocatalysis.<a reveal-id="dsx014-B72" data-open="dsx014-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a><sup>,</sup><a reveal-id="dsx014-B73" data-open="dsx014-B73" class="link link-ref link-reveal xref-bibr"><sup>73</sup></a> Thus, investigation of the differential codon composition of genes with SPs has been an objective of various studies despite the different approaches employed.<a reveal-id="dsx014-B26" data-open="dsx014-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a><sup>,</sup><a reveal-id="dsx014-B43" data-open="dsx014-B43" class="link link-ref link-reveal xref-bibr"><sup>43–44</sup></a> We used our integrated framework, to analyse the CDSs containing SPs for each yeast genome, by computing the Y-value (<a reveal-id="dsx014-F6" data-open="dsx014-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>) and the deviations from uniformity (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_material_dsx014.pdf?Expires=2147483647&Signature=yrjqF1m4pc3Nh7lf8ZdqxQMqJLie-IFYrxEBJLcpqDMU9-oV5zGeyuesVkp5VQyxlzhU3zRsQ5V7s0CE4tzXC9VLZwjVNYLbCg71yM~MUu4INxBSLKGJz-v5sKsGC2imGQ4TSce61gCHMWq4eaQ-D76mDKsHmzubvAB-a2bBBxivZocHwwtUzcM4OJaXpsknqlOW6x-RXwQBYjVSIChlumeabOAPtOUxiQhOnoeMUW2xXepqOP3dagDP3chbKxvFxJPV7gB8t5xpDAGZBv7vi-2N6gKuBo-Iw7Rd5OQigzF-bhrHnjqYDtEm~gR1DIfN8gxdwbPU1O1xQZBFPTKMNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span>). </p>                        <a id="106357510" scrollto-destination="106357510"></a>
<div id="dsx014-F6" data-id="dsx014-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/m_dsx014f6.png?Expires=2147483647&Signature=ACGV-Vdofix2~kev53AUA8QUItDFCQqnd9HQFZUYPyYCoVn6e8MJJaLqu7Z3O~UNCedN7qKHPMeDiVzUMYKNQNoe-KwOuvKRaFnI-~hJx-fVbUO0X~ifUduxxxHBoFVCxCm8SzeydB2OjiQVDbm4Z6T-TKWe6sZb5p5GgtyHBBwcWmU2h24R1a8P-Q-Mp38LjX4N3PFeJkhHuYUXnLNW27ijspnUgGSOnN-Gm8bH2l1QF3UIJaHNN5ppF7eZe~sfB8ur9Rva8BehwqtN5m1Ps697FPahfooRF7IbKJ9IX3Xksj9BogeIPJ2baM2zY1WNJXy6ArtAUKvLWhZyAfwNWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The codon positional dependency in genes coding proteins with signal peptide. The figure shows the analysis applied to the set of CDSs which have signal peptides. The Y-value (see Materials and methods) is reported graphically to illustrate the codon enrichment or depletion at different intragenic positions." path-from-xml="dsx014f6.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357510/dsx014f6.png" path-from-xml="dsx014f6.png" target="_blank">View large</a><a section="106357510" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/dsx014f6.png?Expires=2147483647&Signature=sRVC7jnRxyClY7AWiSByj5ohEmPC~-BEvjVPhFf5LCgmzlZkCwZb8X0PPAdgRXlXr6t5ffqCIrO8cy0Jnnn0DfdcglA3S1stxH88OwQif1twqlRSA2TBUW9JP4tQQPFXNvtKSLrQ7qrSPlC9Kq3qnCy6kQkdOln3w5wOeMtHY2U0F0QJSyZpAOdOORufCP8Ma-zVr8e6OTIXHc528EwA91dboX7QR3DGBn1LZOeNaGRBfUqBXhmKlh0qInbHulZdtzq~dWBzg-wlVwXcGqqfj35mM3~~2u3v3fnW0TJM1c34OwcowWEUO1KeNzbAj2lDVmmCgYTxR63y1UyWI-Gyzw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357510&ar=3752474&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx014f6.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The codon positional dependency in genes coding proteins with signal peptide. The figure shows the analysis applied to the set of CDSs which have signal peptides. The Y-value (see Materials and methods) is reported graphically to illustrate the codon enrichment or depletion at different intragenic positions.</p></div></div><div content-id="dsx014-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/m_dsx014f6.png?Expires=2147483647&Signature=ACGV-Vdofix2~kev53AUA8QUItDFCQqnd9HQFZUYPyYCoVn6e8MJJaLqu7Z3O~UNCedN7qKHPMeDiVzUMYKNQNoe-KwOuvKRaFnI-~hJx-fVbUO0X~ifUduxxxHBoFVCxCm8SzeydB2OjiQVDbm4Z6T-TKWe6sZb5p5GgtyHBBwcWmU2h24R1a8P-Q-Mp38LjX4N3PFeJkhHuYUXnLNW27ijspnUgGSOnN-Gm8bH2l1QF3UIJaHNN5ppF7eZe~sfB8ur9Rva8BehwqtN5m1Ps697FPahfooRF7IbKJ9IX3Xksj9BogeIPJ2baM2zY1WNJXy6ArtAUKvLWhZyAfwNWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The codon positional dependency in genes coding proteins with signal peptide. The figure shows the analysis applied to the set of CDSs which have signal peptides. The Y-value (see Materials and methods) is reported graphically to illustrate the codon enrichment or depletion at different intragenic positions." path-from-xml="dsx014f6.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357510/dsx014f6.png" path-from-xml="dsx014f6.png" target="_blank">View large</a><a section="106357510" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/dsx014f6.png?Expires=2147483647&Signature=sRVC7jnRxyClY7AWiSByj5ohEmPC~-BEvjVPhFf5LCgmzlZkCwZb8X0PPAdgRXlXr6t5ffqCIrO8cy0Jnnn0DfdcglA3S1stxH88OwQif1twqlRSA2TBUW9JP4tQQPFXNvtKSLrQ7qrSPlC9Kq3qnCy6kQkdOln3w5wOeMtHY2U0F0QJSyZpAOdOORufCP8Ma-zVr8e6OTIXHc528EwA91dboX7QR3DGBn1LZOeNaGRBfUqBXhmKlh0qInbHulZdtzq~dWBzg-wlVwXcGqqfj35mM3~~2u3v3fnW0TJM1c34OwcowWEUO1KeNzbAj2lDVmmCgYTxR63y1UyWI-Gyzw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357510&ar=3752474&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx014f6.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The codon positional dependency in genes coding proteins with signal peptide. The figure shows the analysis applied to the set of CDSs which have signal peptides. The Y-value (see Materials and methods) is reported graphically to illustrate the codon enrichment or depletion at different intragenic positions.</p></div></div><p>Because SPs are composed mainly of a hydrophobic core of amino acids<a reveal-id="dsx014-B74" data-open="dsx014-B74" class="link link-ref link-reveal xref-bibr"><sup>74</sup></a> such as glycine (Gly), alanine (Ala), valine (Val), leucine (Leu), isoleucine (Ile), proline (Pro) and phenylalanine (Phe), we hypothesized that if translation efficiency controls the SPs, the aforementioned amino acids would be codified preferentially by the less efficient (non-optimal) codons, and then we would expect their enrichment in the 5′-end region of CDSs.</p><p>In agreement with our hypothesis, it is possible to observe that in the first bin (<a reveal-id="dsx014-F6" data-open="dsx014-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>) Leu is codified preferentially by CTG in <em>K. lactis</em> (<em>Kl</em>), CTG in <em>K. marxianus</em> (<em>Km</em>), CTC in <em>P. pastoris</em> (<em>Pp</em>), CTC in <em>S. cerevisiae</em> (<em>Sc</em>) and TTA in <em>Y. lipolytica</em> (<em>Yl</em>). Although they are different codons, all of them are RareNO codons. On the other hand, for the same amino acid, the most depleted codon is TTG in <em>Kl</em>, TTG in <em>Km</em>, TTA in <em>Pp</em>, TTG in <em>Sc</em> and CTT in <em>Yl</em>, which are optimal codons except for <em>Pp</em> that does not present optimal codons for this amino acid. In other cases, Val is preferentially codified by GTA in <em>Kl</em>, GTA in <em>Km</em>, GTG in <em>Pp</em>, GTG in <em>Sc</em> and GTA in <em>Yl</em>, being all RareNO codons. Phe is preferentially codified by TTT in <em>Kl</em>, <em>Km</em> and <em>Pp</em>, which is a non-optimal codon, and at the same time avoids TTC which is optimal in all cases; <em>Sc</em> and <em>Yl</em> do not present codon bias for this amino acid. Gly is preferentially codified by GGG in <em>Kl</em>, GGA in <em>Km</em>, GGG in <em>Sc</em> and GGG in <em>Yl</em>, which are all RareNO codons. <em>Pp</em> does not present codon bias for this amino acid, while <em>Km</em>, <em>Sc</em> and <em>Yl</em> avoid the use of the GGT codon which is FreO in all cases.</p><p>Interestingly, a number of researchers reported significant improvement in yeast protein secretion when Arg was inserted exactly in the N-terminal of synthetics SPs<a reveal-id="dsx014-B74" data-open="dsx014-B74" class="link link-ref link-reveal xref-bibr"><sup>74</sup></a><sup>,</sup><a reveal-id="dsx014-B75" data-open="dsx014-B75" class="link link-ref link-reveal xref-bibr"><sup>75</sup></a> and, as shown in <a reveal-id="dsx014-F6" data-open="dsx014-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>, the amino acid with the highest bias (conserved characteristic among the yeasts) is indeed Arg. We checked the features of its codons and found that Arg is preferentially codified in the 5′-end region by CGG in <em>Kl</em>, CGA in <em>Km</em>, CGA in <em>Sc</em> and CGC in <em>Yl</em>, which are RareNO codons, while <em>Pp</em> uses preferentially CGC (RareO) and CGG (RareNO). Therefore, this is an important feature to be considered in experimental designs of synthetic SPs.</p><p>In general, we observed in this study a conserved feature: amino acids that are part of the SPs are codified preferentially by non-optimal codons, most of them being RareNO codons. We also verified that this genic region is commonly depleted in optimal codons, demonstrating through this integrated approach, that positional dependency governs the codon arrangement in SPs, which seems to provide slow translation rates at the 5′-end and maintains the ramp of translation efficiency of genes,<a reveal-id="dsx014-B30" data-open="dsx014-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> to guarantee co-translational interaction between SPs and the signal recognition particle.<a reveal-id="dsx014-B26" data-open="dsx014-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a></p>                            <h3 scrollto-destination=106357515 id="106357515" class="section-title" >3.8. Codon category contribution to protein structure and the necessity of new optimization methods</h3>
<p>Protein co-translational folding is one of the most important features of individual codons in protein biosynthesis.<a reveal-id="dsx014-B22" data-open="dsx014-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><a reveal-id="dsx014-B67" data-open="dsx014-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a><sup>,</sup><a reveal-id="dsx014-B76" data-open="dsx014-B76" class="link link-ref link-reveal xref-bibr"><sup>76</sup></a><sup>,</sup><a reveal-id="dsx014-B77" data-open="dsx014-B77" class="link link-ref link-reveal xref-bibr"><sup>77</sup></a> We analysed the relationships between different categories of codons and the protein tertiary structure, as well as important issues related to the use of the classical method of codon optimization such as the Codon Adaptation Index (CAI).</p><p>We assessed in each yeast genome a conserved CDS with a similar CAI value, but with different protein expression levels under similar physiological conditions. In this way, we analysed the TEF1a (Translation elongation factor 1 alpha) and SSE1 (coding a heat-shock protein) genes which have similar CAI values in each host yeast genome (<a reveal-id="dsx014-F7" data-open="dsx014-F7" class="link link-reveal link-table xref-fig">Fig. 7A</a>). Subsequently, we performed codon multiple alignments as described in a previous study<a reveal-id="dsx014-B27" data-open="dsx014-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> and then we considered the corresponding codon category and determined the levels of conservation among the 5 yeasts (<a reveal-id="dsx014-F7" data-open="dsx014-F7" class="link link-reveal link-table xref-fig">Fig. 7B</a>). We represented the codon-category conserved sites in the 3D structure of both proteins, which allowed us to illustrate conserved evolutionary types of translation efficiency and co-translational folding over two differently expressed genes, despite having close CAI values. </p>                        <a id="106357518" scrollto-destination="106357518"></a>
<div id="dsx014-F7" data-id="dsx014-F7" class="fig fig-section"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/m_dsx014f7.png?Expires=2147483647&Signature=DC9RMnRSGAJl1281sdwL7JtzTj6vASIaoC88ZxatNTY57e0Zel13107mOJ~0dyLBjci7Uz7WWxz7cbU3G28vVuQQWxARZ1a1IBs1Gpp5J1XhnMy5dYBA09Cv6UO44DlSSMRqsHkcBOZsUQQbdMfmlrn7BS4WFTQlYYIrq8~-4e9qnILNKGEZi~BPPJX28a~t0aklYL7CxR1p6SpjTOiStTkYe~2G046bKrD-YHm0smd4PHmrciU7O2rWQ6qBY9P2OQG6MV172b3MoUntzBRRNBPRfqpbsu9mj33~ZhMPtey-LVUcqUA4QHsrK3HZZaaPqXIC50Tb1nP~ASFYCZbPqw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Proof of concept on codon category contribution to structural features of two different proteins. (A) Similar values of Codon Adaptation Index (CAI) were found for the SSE1 (expressed as a response of heat-shock) and the TEF1a proteins (highly and constitutively expressed). As both present similar CAI values to their host genomes, and have very different expression rates in physiological conditions, they are an interesting example to test our concept. CAI, a commonly used index, seems to be a non-well fitted method for predicting expression levels of proteins and thus is not recommended for the secure optimization of yeast genes. (B) Method for multiple sequence alignment based on codons to determine the conservation of codon categories in coding sequences. If a codon category is conserved at least in four out of five yeasts then the codon is defined as conserved and its category is illustrated with its correspondent colour in the protein structure. (C) Conserved codon categories are highlighted in SSE1 protein structure. Motifs of conserved Frequent–Optimal codons (FreO) at determinant positions in alpha-helices. (D) Conserved codon categories are highlighted in TEF1a protein structure. Positions with conserved Frequent–Non-optimal codons (FreNO), illustrating conserved non-optimal sites within genes at different positions to possibly regulate the efficiency and accuracy of the protein elongation process, being a potential requirement to the co-translational folding of proteins. (E) Changes in codon category composition when both proteins were optimized by CAI algorithm. CAI optimization introduces silent mutations that could impact translation rates by insertion of FreO and FreNO codons." path-from-xml="dsx014f7.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357518/dsx014f7.png" path-from-xml="dsx014f7.png" target="_blank">View large</a><a section="106357518" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/dsx014f7.png?Expires=2147483647&Signature=cbVhKUJylqJ45RWOEV0vWpgdMVAb3zGn-c~0zL0qLqQ2KaK9ijr4JQTbi-lLmJPTHrrfeB4kbF0csHPGXXNar3JRf0GX5iFkR4d8iqoj9kBB41XJOmH9w2JRcnEv62O-ZPy0Di7Y~5xux92~W42f9WefRcz9HsKlnjR3RFrT0ASKpu6zCxEM5RItO9TMIkpQyHWwCUdSrgvblvMZjYuRWS9gwpXhVAN7AKg9eSddF-OPwMdgTbf2b8AK7hDSUiaYaImigq05zGaaIpM6qQAV10aUOhgIZrc2zHJ1SI-yzy1Dh5ZRVOw9rDx-L4nzJQmfsUS-g0GB1ydXQzbUfKQ5Cw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357518&ar=3752474&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx014f7.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Proof of concept on codon category contribution to structural features of two different proteins. (A) Similar values of Codon Adaptation Index (CAI) were found for the SSE1 (expressed as a response of heat-shock) and the TEF1a proteins (highly and constitutively expressed). As both present similar CAI values to their host genomes, and have very different expression rates in physiological conditions, they are an interesting example to test our concept. CAI, a commonly used index, seems to be a non-well fitted method for predicting expression levels of proteins and thus is not recommended for the secure optimization of yeast genes. (B) Method for multiple sequence alignment based on codons to determine the conservation of codon categories in coding sequences. If a codon category is conserved at least in four out of five yeasts then the codon is defined as conserved and its category is illustrated with its correspondent colour in the protein structure. (C) Conserved codon categories are highlighted in SSE1 protein structure. Motifs of conserved Frequent–Optimal codons (FreO) at determinant positions in alpha-helices. (D) Conserved codon categories are highlighted in TEF1a protein structure. Positions with conserved Frequent–Non-optimal codons (FreNO), illustrating conserved non-optimal sites within genes at different positions to possibly regulate the efficiency and accuracy of the protein elongation process, being a potential requirement to the co-translational folding of proteins. (E) Changes in codon category composition when both proteins were optimized by CAI algorithm. CAI optimization introduces silent mutations that could impact translation rates by insertion of FreO and FreNO codons.</p></div></div><div content-id="dsx014-F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/m_dsx014f7.png?Expires=2147483647&Signature=DC9RMnRSGAJl1281sdwL7JtzTj6vASIaoC88ZxatNTY57e0Zel13107mOJ~0dyLBjci7Uz7WWxz7cbU3G28vVuQQWxARZ1a1IBs1Gpp5J1XhnMy5dYBA09Cv6UO44DlSSMRqsHkcBOZsUQQbdMfmlrn7BS4WFTQlYYIrq8~-4e9qnILNKGEZi~BPPJX28a~t0aklYL7CxR1p6SpjTOiStTkYe~2G046bKrD-YHm0smd4PHmrciU7O2rWQ6qBY9P2OQG6MV172b3MoUntzBRRNBPRfqpbsu9mj33~ZhMPtey-LVUcqUA4QHsrK3HZZaaPqXIC50Tb1nP~ASFYCZbPqw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Proof of concept on codon category contribution to structural features of two different proteins. (A) Similar values of Codon Adaptation Index (CAI) were found for the SSE1 (expressed as a response of heat-shock) and the TEF1a proteins (highly and constitutively expressed). As both present similar CAI values to their host genomes, and have very different expression rates in physiological conditions, they are an interesting example to test our concept. CAI, a commonly used index, seems to be a non-well fitted method for predicting expression levels of proteins and thus is not recommended for the secure optimization of yeast genes. (B) Method for multiple sequence alignment based on codons to determine the conservation of codon categories in coding sequences. If a codon category is conserved at least in four out of five yeasts then the codon is defined as conserved and its category is illustrated with its correspondent colour in the protein structure. (C) Conserved codon categories are highlighted in SSE1 protein structure. Motifs of conserved Frequent–Optimal codons (FreO) at determinant positions in alpha-helices. (D) Conserved codon categories are highlighted in TEF1a protein structure. Positions with conserved Frequent–Non-optimal codons (FreNO), illustrating conserved non-optimal sites within genes at different positions to possibly regulate the efficiency and accuracy of the protein elongation process, being a potential requirement to the co-translational folding of proteins. (E) Changes in codon category composition when both proteins were optimized by CAI algorithm. CAI optimization introduces silent mutations that could impact translation rates by insertion of FreO and FreNO codons." path-from-xml="dsx014f7.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357518/dsx014f7.png" path-from-xml="dsx014f7.png" target="_blank">View large</a><a section="106357518" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/dsx014f7.png?Expires=2147483647&Signature=cbVhKUJylqJ45RWOEV0vWpgdMVAb3zGn-c~0zL0qLqQ2KaK9ijr4JQTbi-lLmJPTHrrfeB4kbF0csHPGXXNar3JRf0GX5iFkR4d8iqoj9kBB41XJOmH9w2JRcnEv62O-ZPy0Di7Y~5xux92~W42f9WefRcz9HsKlnjR3RFrT0ASKpu6zCxEM5RItO9TMIkpQyHWwCUdSrgvblvMZjYuRWS9gwpXhVAN7AKg9eSddF-OPwMdgTbf2b8AK7hDSUiaYaImigq05zGaaIpM6qQAV10aUOhgIZrc2zHJ1SI-yzy1Dh5ZRVOw9rDx-L4nzJQmfsUS-g0GB1ydXQzbUfKQ5Cw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357518&ar=3752474&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx014f7.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Proof of concept on codon category contribution to structural features of two different proteins. (A) Similar values of Codon Adaptation Index (CAI) were found for the SSE1 (expressed as a response of heat-shock) and the TEF1a proteins (highly and constitutively expressed). As both present similar CAI values to their host genomes, and have very different expression rates in physiological conditions, they are an interesting example to test our concept. CAI, a commonly used index, seems to be a non-well fitted method for predicting expression levels of proteins and thus is not recommended for the secure optimization of yeast genes. (B) Method for multiple sequence alignment based on codons to determine the conservation of codon categories in coding sequences. If a codon category is conserved at least in four out of five yeasts then the codon is defined as conserved and its category is illustrated with its correspondent colour in the protein structure. (C) Conserved codon categories are highlighted in SSE1 protein structure. Motifs of conserved Frequent–Optimal codons (FreO) at determinant positions in alpha-helices. (D) Conserved codon categories are highlighted in TEF1a protein structure. Positions with conserved Frequent–Non-optimal codons (FreNO), illustrating conserved non-optimal sites within genes at different positions to possibly regulate the efficiency and accuracy of the protein elongation process, being a potential requirement to the co-translational folding of proteins. (E) Changes in codon category composition when both proteins were optimized by CAI algorithm. CAI optimization introduces silent mutations that could impact translation rates by insertion of FreO and FreNO codons.</p></div></div><p>First, it is important to highlight that by using our codon categorization, a potential impact on essential structural features was uncovered. Our novel categorization scheme allowed us to identify patterns of conserved optimal codons in agreement with previous findings,<a reveal-id="dsx014-B27" data-open="dsx014-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> but going beyond the categorizing of individual codons by optimality properties, the proposed method herein takes into account the fact that these optimal codons are also frequent codons in a genome-scale analysis. Thus, we observed FreO codons conserved in positions 1 and 4 from the helix start, and additionally, the enrichment of conserved FreO codons in the alpha-helices (<a reveal-id="dsx014-F7" data-open="dsx014-F7" class="link link-reveal link-table xref-fig">Fig. 7C and D</a>). One important point to be noted is the absence of FreNO conserved sites in the structure of the SSE1 protein (<a reveal-id="dsx014-F7" data-open="dsx014-F7" class="link link-reveal link-table xref-fig">Fig. 7C</a>), whose translation is low, but presents a conserved RareNO site at the beginning of the gene in the second codon/amino acid position. However, it was not possible to illustrate it in the protein structure due to lack of information of this amino acid in the original 3D protein model. In addition, it is important to observe (<a reveal-id="dsx014-F7" data-open="dsx014-F7" class="link link-reveal link-table xref-fig">Fig. 7C and D</a>) that even different codons, or indeed, when different amino acids are being coded, the codon category is conserved among the yeasts, suggesting the importance of the properties of individual codons to the conformation of structural features in proteins.</p><p>In the case of TEF1a, a highly expressed protein, we found optimal (FreO and RareO) and FreNO conserved sites in important structural locations (<a reveal-id="dsx014-F7" data-open="dsx014-F7" class="link link-reveal link-table xref-fig">Fig. 7D</a>), such as the last position of beta-sheets and alpha-helices. It is reasonable to claim that this FreNO conservation provides the control needed over the protein elongation rate. It seems that the ribosome suffers a decrease in translation speed to retain inside its tunnel the previously translated amino acids in order to fold the structural motif. The conserved FreNO sites are found in general at ∼20–30 codons downstream of complete alpha-helices or beta-sheets, fitting perfectly to the length of the ribosomal tunnel (∼30 amino acids), the region responsible for protein co-translational folding.<a reveal-id="dsx014-B26" data-open="dsx014-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a><sup>,</sup><a reveal-id="dsx014-B78" data-open="dsx014-B78" class="link link-ref link-reveal xref-bibr"><sup>78</sup></a><sup>,</sup><a reveal-id="dsx014-B79" data-open="dsx014-B79" class="link link-ref link-reveal xref-bibr"><sup>79</sup></a> After they have been analysed, we focused our investigation on a more generalized perspective about the role of the four categories of codons in secondary protein structures.</p><p>First, we hypothesized that codon features affect the translation efficiency of the ribosome, and assist protein secondary structure conformation. Therefore, we expected to observe that the secondary structures would be translated at different rates and their codon composition would not be equally distributed between the cluster of frequently used codons (FreO and FreNO) and the cluster of rarely used codons (RareO and RareNO). To test this, we retrieved 300 structures from the RSCB Protein Data Bank and filtered out the redundant ones, resulting in 138 non-redundant protein structures. Next, we evaluated the occurrence of the four codon categories inside all their alpha-helices, beta-sheets and coil regions (coil was assumed to be the regions which do not belong to helix and sheet coordinates in PDB files).</p><p>We found that the codon composition of alpha-helices and beta-sheets is similar (∼41% FreO, ∼16% FreNO, ∼18% RareNO and ∼23% RareO; <span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_material_dsx014.pdf?Expires=2147483647&amp;Signature=yrjqF1m4pc3Nh7lf8ZdqxQMqJLie-IFYrxEBJLcpqDMU9-oV5zGeyuesVkp5VQyxlzhU3zRsQ5V7s0CE4tzXC9VLZwjVNYLbCg71yM~MUu4INxBSLKGJz-v5sKsGC2imGQ4TSce61gCHMWq4eaQ-D76mDKsHmzubvAB-a2bBBxivZocHwwtUzcM4OJaXpsknqlOW6x-RXwQBYjVSIChlumeabOAPtOUxiQhOnoeMUW2xXepqOP3dagDP3chbKxvFxJPV7gB8t5xpDAGZBv7vi-2N6gKuBo-Iw7Rd5OQigzF-bhrHnjqYDtEm~gR1DIfN8gxdwbPU1O1xQZBFPTKMNA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10A</a></span>), also ∼65% of the codons were optimal and ∼35% non-optimal, suggesting that a substantial proportion of the secondary structures is translated slowly. Contrary, coil regions presented a notable difference in codon composition (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_material_dsx014.pdf?Expires=2147483647&amp;Signature=yrjqF1m4pc3Nh7lf8ZdqxQMqJLie-IFYrxEBJLcpqDMU9-oV5zGeyuesVkp5VQyxlzhU3zRsQ5V7s0CE4tzXC9VLZwjVNYLbCg71yM~MUu4INxBSLKGJz-v5sKsGC2imGQ4TSce61gCHMWq4eaQ-D76mDKsHmzubvAB-a2bBBxivZocHwwtUzcM4OJaXpsknqlOW6x-RXwQBYjVSIChlumeabOAPtOUxiQhOnoeMUW2xXepqOP3dagDP3chbKxvFxJPV7gB8t5xpDAGZBv7vi-2N6gKuBo-Iw7Rd5OQigzF-bhrHnjqYDtEm~gR1DIfN8gxdwbPU1O1xQZBFPTKMNA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10A</a></span>). We observed that the composition of RareNO codons in coils was the highest compared with the other two structures, approximately 26% of its total composition. This result indicates that, at least for the proteins analysed, coils have numerous points with the slowest translation efficiency as determined by codon decoding times in <a reveal-id="dsx014-F5" data-open="dsx014-F5" class="link link-reveal link-table xref-fig">Fig. 5A</a>. Moreover, codon optimality distribution in coil regions is particularly characterized by an increased presence of non-optimal codons (∼44% non-optimal and ∼56% optimal; <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_dataset_s2_dsx014.xlsx?Expires=2147483647&amp;Signature=UvPuY4OGA0v6iQlgw~~~bWwee6dvmCeDA5fIGl7w9962MwPV3qmsBLJsHkA0rOeBXAoOaO~qSR4sYrCPkfVcLaFIVvJJqZX4xRXpUXq8Bsbzhx7ri5c45EhCMQ-zc0U-aNZ9o~S6CDSL8sGYCwiH60A7VsbY4noqyjAVtxdf6Qjm0q2qP2eDbLHA1TVjflf8UwEq4l3qGodD~CcXEIPnen4RUB8xK1jpqXoIcQzfOXN27lKEWuQDmkoZrHWkWjUV6lVR2anKuuYZFL-Kx~BCYvtSUYTK-xDtag90oypHSDNejyePca-8DGre4ZE8vGPegoOI6LHTMSAkcq5UXs5dQA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Dataset S2</a></span>).</p><p>We believe that these observations are attributed to the dynamics of the protein elongation process. As an example, we consider a simplified part of a protein composed by a helix followed downstream by a coil. When the region corresponding to the helix has been already (optimally) translated, it would be fitted inside the ribosome tunnel where would occur the co-translational folding.<a reveal-id="dsx014-B26" data-open="dsx014-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a><sup>,</sup><a reveal-id="dsx014-B78" data-open="dsx014-B78" class="link link-ref link-reveal xref-bibr"><sup>78</sup></a><sup>,</sup><a reveal-id="dsx014-B79" data-open="dsx014-B79" class="link link-ref link-reveal xref-bibr"><sup>79</sup></a> Probably, the time required to take place this folding is provided by the decrease of translation rate in coil region, which would be associated to the presence of RareNO codons (enriched in the coil). Another possible explanation to this observation could be attributed to the evolutionary pressures that act over the sheet and helix structures, as they determine the correct function of proteins one will expect that these secondary structures tend to be adapted for efficient translation while coils could accumulate different silent mutations. This reasoning is still a theoretic approximation to the observations and further approaches should be applied to unveil the mechanisms behind these intriguing results. We suggest that synthetic optimization of codons in those structures should consider that changes in local translation rates could impact the protein co-translational folding and ultimately its function.</p><p>Finally, we were interested in examining the codon composition at the extremities of the secondary structures which seems to be specifically under higher selective pressures considering the previous TEF1a/SSE1 inspection. To analyse the positional dependency of the structure-codon correspondence we studied the differences in the codon composition of the four codon categories at different positions. Because the secondary structures of proteins present a high degree of variation in length, we decided to constrain our analysis using the first three and the last three codon positions. Notably, the codon composition of FreNOs presented the highest increment in the first codon of helices when compared with the overall (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_material_dsx014.pdf?Expires=2147483647&amp;Signature=yrjqF1m4pc3Nh7lf8ZdqxQMqJLie-IFYrxEBJLcpqDMU9-oV5zGeyuesVkp5VQyxlzhU3zRsQ5V7s0CE4tzXC9VLZwjVNYLbCg71yM~MUu4INxBSLKGJz-v5sKsGC2imGQ4TSce61gCHMWq4eaQ-D76mDKsHmzubvAB-a2bBBxivZocHwwtUzcM4OJaXpsknqlOW6x-RXwQBYjVSIChlumeabOAPtOUxiQhOnoeMUW2xXepqOP3dagDP3chbKxvFxJPV7gB8t5xpDAGZBv7vi-2N6gKuBo-Iw7Rd5OQigzF-bhrHnjqYDtEm~gR1DIfN8gxdwbPU1O1xQZBFPTKMNA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10B</a></span>). Although FreO codons are preferentially found at this first position (∼39%; <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_dataset_s2_dsx014.xlsx?Expires=2147483647&amp;Signature=UvPuY4OGA0v6iQlgw~~~bWwee6dvmCeDA5fIGl7w9962MwPV3qmsBLJsHkA0rOeBXAoOaO~qSR4sYrCPkfVcLaFIVvJJqZX4xRXpUXq8Bsbzhx7ri5c45EhCMQ-zc0U-aNZ9o~S6CDSL8sGYCwiH60A7VsbY4noqyjAVtxdf6Qjm0q2qP2eDbLHA1TVjflf8UwEq4l3qGodD~CcXEIPnen4RUB8xK1jpqXoIcQzfOXN27lKEWuQDmkoZrHWkWjUV6lVR2anKuuYZFL-Kx~BCYvtSUYTK-xDtag90oypHSDNejyePca-8DGre4ZE8vGPegoOI6LHTMSAkcq5UXs5dQA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Dataset S2</a></span>), FreNO codons presented their highest occurrence (∼21%) at this position when compared with all other positions. At subsequent positions (Codons 2 and 3) the FreNO composition is decreased and similar to the overall, in the third codon position FreO codons were found to be relatively increased. At the last position on the helices (Codon n), FreO codons decreased their occurrence and the non-optimal codons presented a relative increment. Also, we observed an increased occurrence of RareO codons (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_material_dsx014.pdf?Expires=2147483647&amp;Signature=yrjqF1m4pc3Nh7lf8ZdqxQMqJLie-IFYrxEBJLcpqDMU9-oV5zGeyuesVkp5VQyxlzhU3zRsQ5V7s0CE4tzXC9VLZwjVNYLbCg71yM~MUu4INxBSLKGJz-v5sKsGC2imGQ4TSce61gCHMWq4eaQ-D76mDKsHmzubvAB-a2bBBxivZocHwwtUzcM4OJaXpsknqlOW6x-RXwQBYjVSIChlumeabOAPtOUxiQhOnoeMUW2xXepqOP3dagDP3chbKxvFxJPV7gB8t5xpDAGZBv7vi-2N6gKuBo-Iw7Rd5OQigzF-bhrHnjqYDtEm~gR1DIfN8gxdwbPU1O1xQZBFPTKMNA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10A</a></span>). In general, our analysis indicates that helices are relatively depleted in optimal codons at the first and last position of the structure (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_material_dsx014.pdf?Expires=2147483647&amp;Signature=yrjqF1m4pc3Nh7lf8ZdqxQMqJLie-IFYrxEBJLcpqDMU9-oV5zGeyuesVkp5VQyxlzhU3zRsQ5V7s0CE4tzXC9VLZwjVNYLbCg71yM~MUu4INxBSLKGJz-v5sKsGC2imGQ4TSce61gCHMWq4eaQ-D76mDKsHmzubvAB-a2bBBxivZocHwwtUzcM4OJaXpsknqlOW6x-RXwQBYjVSIChlumeabOAPtOUxiQhOnoeMUW2xXepqOP3dagDP3chbKxvFxJPV7gB8t5xpDAGZBv7vi-2N6gKuBo-Iw7Rd5OQigzF-bhrHnjqYDtEm~gR1DIfN8gxdwbPU1O1xQZBFPTKMNA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S11</a></span>).</p><p>These results denote that the extremities of the helices, could serve as a transition point in the translation rate to start or terminate structure translation. This transition point in beta-sheets is characterized, in contrast to helices, by a decrease in FreO codons occurrence at the end of beta-sheets (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_material_dsx014.pdf?Expires=2147483647&Signature=yrjqF1m4pc3Nh7lf8ZdqxQMqJLie-IFYrxEBJLcpqDMU9-oV5zGeyuesVkp5VQyxlzhU3zRsQ5V7s0CE4tzXC9VLZwjVNYLbCg71yM~MUu4INxBSLKGJz-v5sKsGC2imGQ4TSce61gCHMWq4eaQ-D76mDKsHmzubvAB-a2bBBxivZocHwwtUzcM4OJaXpsknqlOW6x-RXwQBYjVSIChlumeabOAPtOUxiQhOnoeMUW2xXepqOP3dagDP3chbKxvFxJPV7gB8t5xpDAGZBv7vi-2N6gKuBo-Iw7Rd5OQigzF-bhrHnjqYDtEm~gR1DIfN8gxdwbPU1O1xQZBFPTKMNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10B</a></span>), where FreNO codons are relatively enriched, and by a relative depletion of non-optimal codons at the beginning of the structure (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_material_dsx014.pdf?Expires=2147483647&Signature=yrjqF1m4pc3Nh7lf8ZdqxQMqJLie-IFYrxEBJLcpqDMU9-oV5zGeyuesVkp5VQyxlzhU3zRsQ5V7s0CE4tzXC9VLZwjVNYLbCg71yM~MUu4INxBSLKGJz-v5sKsGC2imGQ4TSce61gCHMWq4eaQ-D76mDKsHmzubvAB-a2bBBxivZocHwwtUzcM4OJaXpsknqlOW6x-RXwQBYjVSIChlumeabOAPtOUxiQhOnoeMUW2xXepqOP3dagDP3chbKxvFxJPV7gB8t5xpDAGZBv7vi-2N6gKuBo-Iw7Rd5OQigzF-bhrHnjqYDtEm~gR1DIfN8gxdwbPU1O1xQZBFPTKMNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S11</a></span>). In the case of the coil regions, we observed that the first three positions presented a remarkable enrichment of RareNO codons (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_material_dsx014.pdf?Expires=2147483647&Signature=yrjqF1m4pc3Nh7lf8ZdqxQMqJLie-IFYrxEBJLcpqDMU9-oV5zGeyuesVkp5VQyxlzhU3zRsQ5V7s0CE4tzXC9VLZwjVNYLbCg71yM~MUu4INxBSLKGJz-v5sKsGC2imGQ4TSce61gCHMWq4eaQ-D76mDKsHmzubvAB-a2bBBxivZocHwwtUzcM4OJaXpsknqlOW6x-RXwQBYjVSIChlumeabOAPtOUxiQhOnoeMUW2xXepqOP3dagDP3chbKxvFxJPV7gB8t5xpDAGZBv7vi-2N6gKuBo-Iw7Rd5OQigzF-bhrHnjqYDtEm~gR1DIfN8gxdwbPU1O1xQZBFPTKMNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10B</a></span>), specifically, the position corresponding to the second codon presented the highest increment of RareNO codons in all the structures, suggesting that the translation at the beginning of coil regions is characterized by a very slow rate. Even though the occurrence of non-optimal codons in coil regions is mainly concentrated at the beginning of coils, we also found them increased at the end of the structure (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_material_dsx014.pdf?Expires=2147483647&Signature=yrjqF1m4pc3Nh7lf8ZdqxQMqJLie-IFYrxEBJLcpqDMU9-oV5zGeyuesVkp5VQyxlzhU3zRsQ5V7s0CE4tzXC9VLZwjVNYLbCg71yM~MUu4INxBSLKGJz-v5sKsGC2imGQ4TSce61gCHMWq4eaQ-D76mDKsHmzubvAB-a2bBBxivZocHwwtUzcM4OJaXpsknqlOW6x-RXwQBYjVSIChlumeabOAPtOUxiQhOnoeMUW2xXepqOP3dagDP3chbKxvFxJPV7gB8t5xpDAGZBv7vi-2N6gKuBo-Iw7Rd5OQigzF-bhrHnjqYDtEm~gR1DIfN8gxdwbPU1O1xQZBFPTKMNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S11</a></span>). Although codon overall composition of protein secondary structures seems not very dissimilar (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_material_dsx014.pdf?Expires=2147483647&Signature=yrjqF1m4pc3Nh7lf8ZdqxQMqJLie-IFYrxEBJLcpqDMU9-oV5zGeyuesVkp5VQyxlzhU3zRsQ5V7s0CE4tzXC9VLZwjVNYLbCg71yM~MUu4INxBSLKGJz-v5sKsGC2imGQ4TSce61gCHMWq4eaQ-D76mDKsHmzubvAB-a2bBBxivZocHwwtUzcM4OJaXpsknqlOW6x-RXwQBYjVSIChlumeabOAPtOUxiQhOnoeMUW2xXepqOP3dagDP3chbKxvFxJPV7gB8t5xpDAGZBv7vi-2N6gKuBo-Iw7Rd5OQigzF-bhrHnjqYDtEm~gR1DIfN8gxdwbPU1O1xQZBFPTKMNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10A</a></span>), it is possible to perceive that the three protein secondary structures presented contrasting profiles in the extremities (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_material_dsx014.pdf?Expires=2147483647&Signature=yrjqF1m4pc3Nh7lf8ZdqxQMqJLie-IFYrxEBJLcpqDMU9-oV5zGeyuesVkp5VQyxlzhU3zRsQ5V7s0CE4tzXC9VLZwjVNYLbCg71yM~MUu4INxBSLKGJz-v5sKsGC2imGQ4TSce61gCHMWq4eaQ-D76mDKsHmzubvAB-a2bBBxivZocHwwtUzcM4OJaXpsknqlOW6x-RXwQBYjVSIChlumeabOAPtOUxiQhOnoeMUW2xXepqOP3dagDP3chbKxvFxJPV7gB8t5xpDAGZBv7vi-2N6gKuBo-Iw7Rd5OQigzF-bhrHnjqYDtEm~gR1DIfN8gxdwbPU1O1xQZBFPTKMNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S11</a></span>). In general, this intriguing outcome is a theoretical evidence that protein secondary structures could be ruled by the codon positional dependency like the observed properties of coding sequences at genome-scale, having potential implications in the understanding of protein evolution as well as biotechnological applications. However, this is an unexpected result and further research should be aimed at collecting more evidence on the transition points presence and the position-dependent translation efficiency coupled to the co-translational folding in inner regions of protein secondary structures.</p><p>Although we have reported these correspondences between codon categories and protein secondary structures of 138 non-redundant proteins, we suggest that codon-structure correspondence analysis, aimed at heterologous expression experiments, should be performed on a gene-by-gene basis, to identify the codon category in relevant protein structures such as the N-terminus (which in helices seems to be generally enriched in FreNO and depleted in optimal codons) and the structural transition points. Accordingly, both optimal and non-optimal codons have molecular tasks important to cell development. Furthermore, the high complexity of individual codon contribution to protein biosynthesis in yeast cell factories should be considered in codon optimization.</p><p>This reasoning led us to show the potential effects of traditional codon optimization methods such as the Codon Adaptation Index (CAI) approach for optimization of genes. In our experiment (<a reveal-id="dsx014-F7" data-open="dsx014-F7" class="link link-reveal link-table xref-fig">Fig. 7E</a>), we obtained genes with a significant number of frequent codons which should not be considered as optimal; thus, it may be considered normal to obtain a sequence enriched in FreNO codons after gene optimization using the CAI. Then, these FreNO codons could affect protein translation elongation by decreasing the local translation rate and possibly generating unexpected protein qualities.</p><p>Consequently, individual codons have different features that contribute in diverse ways to protein biosynthesis. It depends on several characteristics such as the selective forces that define which codons are enriched and depleted as a function of position, the frequency of codon usage, the adaptation of each codon to the cognate tRNAs available in the cell, the GC content, and the mRNA folding energy. Additionally, each codon has a different role as a speed regulator in protein translation, property that orchestrates the accuracy, efficiency and co-translational folding in protein biosynthesis. Hence, simplistic manipulation of codons for heterologous expression does not take into account the complex biological implications of individual codons.</p><p>We propose that the systematic overview presented in this work must be considered when designing synthetic genes. We define this proposed approach as Codon Transliteration, in which a gene to be used in heterologous expression needs to be studied in depth.</p><p>In this proposed approach, each codon that encodes the same amino acid in the original organism must have its properties characterized. Subsequently, each codon and its features in the new host must be identified. As an example, we consider a gene from <em>K. marxianus</em> to be expressed in <em>Y. lipolytica</em>. The codon TGT of <em>K. marxianus</em> is categorized as FreNO (<a reveal-id="dsx014-F7" data-open="dsx014-F7" class="link link-reveal link-table xref-fig">Fig. 7D</a>), that is, it has features such as non-optimal codon and frequent occurrence in different positions within genes, a slow translation rate, but a codon decoding time lower than RareNO codons. Nevertheless, the same codon (TGT) is RareNO in <em>Y. lipolytica</em> and for this reason, it should not be chosen for expression in this yeast because despite encoding the same amino acid (cysteine), it contributes differently to the protein elongation process. Therefore, the introduction of this codon will lead to a decrease in the elongation rate, and generate unexpected results related to protein co-translational folding. On the other hand, the TGC codon also encodes cysteine in <em>Y. lipolytica</em> and it is categorized as FreNO codon, and conserves the same features found in <em>K. marxianus</em>. For these reasons, the TGC codon should be chosen in <em>Y. lipolytica</em>.</p><p>On the other hand, we expect that genes which are not highly expressed could be efficiently expressed in heterologous hosts by transliterating their codon features (FreNO, RareNO, etc.) to the new host. Thus, in order to optimize individual codons to the new host in terms of their role in the protein secondary structure, we suggest that the category of codons encoding amino acids inside alpha-helices and structural transition points should not be modified. Nevertheless, those that contribute to other structures could be optimized with a view to improving the translation rate of ribosomes.</p><p>In conclusion, by using traditional approaches, synonymous mutations can affect the protein synthesis in terms of quantity, folding and function (<a reveal-id="dsx014-F8" data-open="dsx014-F8" class="link link-reveal link-table xref-fig">Fig. 8</a>). As described through this integrated analysis, each codon seems to play a determinant role in gene translation efficiency and co-translational protein folding, and different impacts could be generated when a gene sequence is manipulated without the details deserved. </p>                        <a id="106357533" scrollto-destination="106357533"></a>
<div id="dsx014-F8" data-id="dsx014-F8" class="fig fig-section"><div class="label fig-label">Figure 8</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/m_dsx014f8.png?Expires=2147483647&Signature=omm4fQ3fk7gnhGRVqcCzwiGKtARWtnCjlF0VdjucQeCRlyqzPU-0M2IAaiyFhAwXND3BKxq0~YzPvk3wZUfVUz~B9keJKI9abO2fbc0fWxjgBResj5HuPD~a53E-U4h0BCp0rU5SJ2-VeFShZrvxII0~ldAc32Lk1TsLnXB8DfqF96M9I~GiXs-yCL8SX901KJ6EmrPaY4y~Xn~Cq0MMybjab1ouqSiNlzmyjZumNqykDohIgfwANM7Ikt8Ee0KJfOuOD3lGaYnYg6lqTSltMWg04oOBw7riLCmR-aq10qBC3057imEP6dki~P4npqoMn7mczYn03tRV9J1GW9IFcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Theoretical scheme of codon contribution to translation efficiency and co-translational protein folding. The figure shows the potential impacts in quantity, folding and function of proteins by insertion of synonymous (silent) mutations. The four categories of codons determined in the present work have been used to illustrate each codon contribution for protein biosynthesis. (A) A hypothetical protein-coding gene shaped by different codons category. (B) The gene has been modified in order to improve its optimality for available tRNA isoacceptors. Translation speed is enhanced, positively affecting protein quantity. On the other hand, this modification can affect negatively the co-translational folding and with that the protein function. (C) An optimized version in terms of frequency (e.g. by CAI) of the wild-type gene. The insertion of FreNO codons affects negatively the quantity of protein produced, but on the other hand supports the maintenance of a slower speed in the translation steps and thereby assists in co-translational protein folding. (D) Silent mutation inserting RareNO codons (slowest translated codons). It affects the translation rate, decreases protein quantity and leads to unknown features in co-translational folding and function. Abbreviations: FreO: Frequent and Optimal codons; RareO: Rare and Optimal; FreNO: Frequent and Non-optimal; RareNO: Rare and Non-optimal." path-from-xml="dsx014f8.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357533/dsx014f8.png" path-from-xml="dsx014f8.png" target="_blank">View large</a><a section="106357533" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/dsx014f8.png?Expires=2147483647&Signature=gZ5o22UAazN3rZlnKvJISY79Mt9ftFxgCaluI8uB7LMoC~cQb2jTZPOrKxFn9FGPQXzhKowf1dLKBHFcb3Hd1VasTRWgjPBrYxOFMehzYJoRSakSHFpjeoKvLPKtm8EKh76M6IedKsL22M6C-eIfmQ7CLsRNYv87xKaTNgzxpqgfldAkn3P0NHRkgE6zMBQ5ztO0BhOO1j~94bndq7VzlYK5W0NIX~d3UJSPNETlkFuZ64N2q9ZKHD3g5t7pQ-yrpDEgMMyyVcuy-aYgRZrexN-rWM5X-oyL~QIURzP97Qsqvu1lvRIvfPENX3Awf9DnYDsQpfFX6GZ5ITuZSZqgKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357533&ar=3752474&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx014f8.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Theoretical scheme of codon contribution to translation efficiency and co-translational protein folding. The figure shows the potential impacts in quantity, folding and function of proteins by insertion of synonymous (silent) mutations. The four categories of codons determined in the present work have been used to illustrate each codon contribution for protein biosynthesis. (A) A hypothetical protein-coding gene shaped by different codons category. (B) The gene has been modified in order to improve its optimality for available tRNA isoacceptors. Translation speed is enhanced, positively affecting protein quantity. On the other hand, this modification can affect negatively the co-translational folding and with that the protein function. (C) An optimized version in terms of frequency (e.g. by CAI) of the wild-type gene. The insertion of FreNO codons affects negatively the quantity of protein produced, but on the other hand supports the maintenance of a slower speed in the translation steps and thereby assists in co-translational protein folding. (D) Silent mutation inserting RareNO codons (slowest translated codons). It affects the translation rate, decreases protein quantity and leads to unknown features in co-translational folding and function. Abbreviations: FreO: Frequent and Optimal codons; RareO: Rare and Optimal; FreNO: Frequent and Non-optimal; RareNO: Rare and Non-optimal.</p></div></div><div content-id="dsx014-F8" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 8</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/m_dsx014f8.png?Expires=2147483647&Signature=omm4fQ3fk7gnhGRVqcCzwiGKtARWtnCjlF0VdjucQeCRlyqzPU-0M2IAaiyFhAwXND3BKxq0~YzPvk3wZUfVUz~B9keJKI9abO2fbc0fWxjgBResj5HuPD~a53E-U4h0BCp0rU5SJ2-VeFShZrvxII0~ldAc32Lk1TsLnXB8DfqF96M9I~GiXs-yCL8SX901KJ6EmrPaY4y~Xn~Cq0MMybjab1ouqSiNlzmyjZumNqykDohIgfwANM7Ikt8Ee0KJfOuOD3lGaYnYg6lqTSltMWg04oOBw7riLCmR-aq10qBC3057imEP6dki~P4npqoMn7mczYn03tRV9J1GW9IFcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Theoretical scheme of codon contribution to translation efficiency and co-translational protein folding. The figure shows the potential impacts in quantity, folding and function of proteins by insertion of synonymous (silent) mutations. The four categories of codons determined in the present work have been used to illustrate each codon contribution for protein biosynthesis. (A) A hypothetical protein-coding gene shaped by different codons category. (B) The gene has been modified in order to improve its optimality for available tRNA isoacceptors. Translation speed is enhanced, positively affecting protein quantity. On the other hand, this modification can affect negatively the co-translational folding and with that the protein function. (C) An optimized version in terms of frequency (e.g. by CAI) of the wild-type gene. The insertion of FreNO codons affects negatively the quantity of protein produced, but on the other hand supports the maintenance of a slower speed in the translation steps and thereby assists in co-translational protein folding. (D) Silent mutation inserting RareNO codons (slowest translated codons). It affects the translation rate, decreases protein quantity and leads to unknown features in co-translational folding and function. Abbreviations: FreO: Frequent and Optimal codons; RareO: Rare and Optimal; FreNO: Frequent and Non-optimal; RareNO: Rare and Non-optimal." path-from-xml="dsx014f8.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106357533/dsx014f8.png" path-from-xml="dsx014f8.png" target="_blank">View large</a><a section="106357533" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/dsx014f8.png?Expires=2147483647&Signature=gZ5o22UAazN3rZlnKvJISY79Mt9ftFxgCaluI8uB7LMoC~cQb2jTZPOrKxFn9FGPQXzhKowf1dLKBHFcb3Hd1VasTRWgjPBrYxOFMehzYJoRSakSHFpjeoKvLPKtm8EKh76M6IedKsL22M6C-eIfmQ7CLsRNYv87xKaTNgzxpqgfldAkn3P0NHRkgE6zMBQ5ztO0BhOO1j~94bndq7VzlYK5W0NIX~d3UJSPNETlkFuZ64N2q9ZKHD3g5t7pQ-yrpDEgMMyyVcuy-aYgRZrexN-rWM5X-oyL~QIURzP97Qsqvu1lvRIvfPENX3Awf9DnYDsQpfFX6GZ5ITuZSZqgKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106357533&ar=3752474&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx014f8.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Theoretical scheme of codon contribution to translation efficiency and co-translational protein folding. The figure shows the potential impacts in quantity, folding and function of proteins by insertion of synonymous (silent) mutations. The four categories of codons determined in the present work have been used to illustrate each codon contribution for protein biosynthesis. (A) A hypothetical protein-coding gene shaped by different codons category. (B) The gene has been modified in order to improve its optimality for available tRNA isoacceptors. Translation speed is enhanced, positively affecting protein quantity. On the other hand, this modification can affect negatively the co-translational folding and with that the protein function. (C) An optimized version in terms of frequency (e.g. by CAI) of the wild-type gene. The insertion of FreNO codons affects negatively the quantity of protein produced, but on the other hand supports the maintenance of a slower speed in the translation steps and thereby assists in co-translational protein folding. (D) Silent mutation inserting RareNO codons (slowest translated codons). It affects the translation rate, decreases protein quantity and leads to unknown features in co-translational folding and function. Abbreviations: FreO: Frequent and Optimal codons; RareO: Rare and Optimal; FreNO: Frequent and Non-optimal; RareNO: Rare and Non-optimal.</p></div></div><p>By insertion of synonymous (silent) mutations (<a reveal-id="dsx014-F8" data-open="dsx014-F8" class="link link-reveal link-table xref-fig">Fig. 8A</a>), a hypothetical protein-coding gene, shaped by different codon category emerges. Its translation efficiency is maintained by the absence of slower translated codons (RareNO, see <a reveal-id="dsx014-F5" data-open="dsx014-F5" class="link link-reveal link-table xref-fig">Fig. 5A</a> for details), and the presence of both FreO codons, that enhance the translation speed, and FreNO codons, which assist protein co-translational folding which slow the translation rate in order to support the polypeptide folding inside the ribosome tunnel. This arrangement of a different category of codons allows for normal protein production in terms of functionality, quantity and folding. However, when the gene has been modified (<a reveal-id="dsx014-F8" data-open="dsx014-F8" class="link link-reveal link-table xref-fig">Fig. 8B</a>) in order to improve its optimality to available tRNA isoacceptors (e.g. by tAI), the insertion of synonymous mutations of FreO and RareO codons leads to the enhancement of translation speed which affects the protein quantity positively, but the co-translational folding and, consequently, its function negatively.</p><p>In another case, an ‘optimized’ version, in terms of codon frequency (e.g. by CAI), of the wild-type gene is presented (<a reveal-id="dsx014-F8" data-open="dsx014-F8" class="link link-reveal link-table xref-fig">Fig. 8C</a>). The insertion of FreNO codons affects the quantity of protein synthesized negatively, but, on the other hand, maintains a slow speed in the translational periods, and by doing this, assists co-translational protein folding. However, excessive insertion of FreNO codons implies more sites at which the ribosome decreases its translational speed and generates potential misfolded structures in the protein with a different function to the wild-type. Finally, when silent mutations are created in a gene, containing more RareNO codons (<a reveal-id="dsx014-F8" data-open="dsx014-F8" class="link link-reveal link-table xref-fig">Fig. 8D</a>) than the wild-type gene, they could affect its translation rate, decrease protein quantity and lead to unknown features in co-translational folding and function.</p><p>In summary, it seems that codons, through a positional dependency function, are arranged so as to exert control over different molecular events such as the initiation of translation by means of mRNA folding energy, the translation per se by the codon adaptiveness to the tRNA isoaceptors, the yield protein production by the codon translation efficiency, and the protein function in terms of their different roles in co-translational folding. The approach developed in this study, based on the integrated analysis of individual codons, contributes to an in-depth understanding of the biological roles of codons, and features an improvement in the basis of rational gene design in heterologous expression and synthetic biology.</p>                            <h2 scrollto-destination=106357537 id="106357537" class="section-title" >Availability</h2>
<p>Algorithms description, codes and tutorial: <a class="link link-uri" href="https://github.com/juanvillada/CodG" target="_blank">https://github.com/juanvillada/CodG</a></p>                            <h2 scrollto-destination=106357539 id="106357539" class="section-title" >Supplementary data</h2>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_dataset_s1_dsx014.xlsx?Expires=2147483647&Signature=yn7ec37o6w~qyUL2X3g5iDJYPjZJ-UekkspdLw~wG9pMLv8LP7-VmOr3~JmsoUI5j-AEQ7ixLD3SSBktaex5CTjFnqyvch22~Y6KknPvypS8hUybL-t4LcS-2I9iKS752oDo-XuqsC1e6n3MaOkrBAoYUxSQBlBbjbxKZ5aPCRRgUqX7KUvXF5~re6xf2ZTH5-wrZx5pqS4x6WCf1FRqyhI0F8hJ7XaeKEctsOuTyEd6Zu~-A2x0zp8aFyO9hIDIB0REHmO4WVDkGqOI19Pco9gjbx2YlG2vNTOqA4H6~x95sXhsrRnp11EA1YfDbYmEnp1eKfcyqGsikaEkl1Hh~A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>DNARES</em> Online.</p>                            <h2 scrollto-destination=106357541 id="106357541" class="section-title" >Author contribution</h2>
<p>JCV and WBDS designed the research; JCVA and OJBB performed the experiments; JCV, OJBB, WBDS analysed the data, and JCVA and WBDS wrote the manuscript.</p>                            <h2 scrollto-destination=106357543 id="106357543" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Maria Fernanda Duran Meneses and Professor Flavia Maria Lopes Passos from Laboratorio de Fisiologia de Microrganismos at the UFV for research design support; Professor Denise Mara Soares Bazzoli from Laboratorio de Genetica Molecular de Bactérias and Professor Hilario Cuqueto Mantovani from Laboratorio de Microbiologia do Rumen e de Anaeróbios at UFV for their critical review of the manuscript. This study was supported by the Coordination for the Improvement of Higher Education Personnel (CAPES), the National Science and Technology Development Council (CNPq) and the Foundation for Research Support of the State of Minas Gerais (FAPEMIG). Support for JCV was obtained through the Organization of American States (OAS) and the Coimbra Group of Brazilian Universities (GCUB) Scholarship Award.</p>                            <h3 scrollto-destination=106357545 id="106357545" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h2 scrollto-destination=106357547 id="106357547" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx014-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ikemura</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="year">1981</div>, 
<div class="article-title">Correlation between the abundance of <em>Escherichia coli</em> transfer RNAs and the occurrence of the respective codons in its protein genes,</div><div class="source">J. Mol. Biol</div> ., <div class="volume">146</div>, <div class="fpage">1</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Correlation%20between%20the%20abundance%20of%20Escherichia%20coli%20transfer%20RNAs%20and%20the%20occurrence%20of%20the%20respective%20codons%20in%20its%20protein%20genes%2C&amp;author=T.%20Ikemura&amp;publication_year=1981&amp;journal=J.%20Mol.%20Biol&amp;volume=146&amp;pages=1-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(81)90363-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(81)90363-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(81)90363-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6167728" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sharp</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">W.-H</div></div></div>. <div class="year">1987</div>, 
<div class="article-title">The codon adaptation index-a measure of directional synonymous codon usage bias, and its potential applications,</div><div class="source">Nucleic Acids Res</div> ., <div class="volume">15</div>, <div class="fpage">1281</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20codon%20adaptation%20index-a%20measure%20of%20directional%20synonymous%20codon%20usage%20bias%2C%20and%20its%20potential%20applications%2C&amp;author=P.%20Sharp&amp;author=W.-H%20Li&amp;publication_year=1987&amp;journal=Nucleic%20Acids%20Res&amp;volume=15&amp;pages=1281-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/15.3.1281" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F15.3.1281" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F15.3.1281"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3547335" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dong</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Nilsson</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Kurland</div> <div class="given-names">C.G.</div></div></div>
          <div class="year">1996</div>, 
<div class="article-title">Co-variation of tRNA abundance and codon usage in <em>Escherichia coli</em> at different growth rates,</div><div class="source">J. Mol. Biol</div> ., <div class="volume">260</div>, <div class="fpage">649</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Co-variation%20of%20tRNA%20abundance%20and%20codon%20usage%20in%20Escherichia%20coli%20at%20different%20growth%20rates%2C&amp;author=H.%20Dong&amp;author=L.%20Nilsson&amp;author=C.G.%20Kurland&amp;publication_year=1996&amp;journal=J.%20Mol.%20Biol&amp;volume=260&amp;pages=649-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.1996.0428" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.1996.0428" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.1996.0428"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8709146" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hahn</div>
               <div class="given-names">M.W.</div>
            </div>, <div class="name string-name"><div class="surname">Mezey</div> <div class="given-names">J.G.</div></div>, <div class="name string-name"><div class="surname">Begun</div> <div class="given-names">D.J.</div></div></div>, et al.  <div class="year">2005</div>, 
<div class="article-title">Evolutionary genomics: codon bias and selection on single genomes,</div><div class="source">Nature</div> , <div class="volume">433</div>, <div class="fpage">E5</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20genomics%3A%20codon%20bias%20and%20selection%20on%20single%20genomes%2C&amp;author=M.W.%20Hahn&amp;author=J.G.%20Mezey&amp;author=D.J.%20Begun&amp;publication_year=2005&amp;journal=Nature&amp;volume=433&amp;pages=E5-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature03221" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature03221" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature03221"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15662370" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dos Reis</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Savva</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Wernisch</div> <div class="given-names">L.</div></div></div>
          <div class="year">2004</div>, 
<div class="article-title">Solving the riddle of codon usage preferences: a test for translational selection,</div><div class="source">Nucleic Acids Res</div> ., <div class="volume">32</div>, <div class="fpage">5036</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Solving%20the%20riddle%20of%20codon%20usage%20preferences%3A%20a%20test%20for%20translational%20selection%2C&amp;author=M.%20Dos%20Reis&amp;author=R.%20Savva&amp;author=L.%20Wernisch&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res&amp;volume=32&amp;pages=5036-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh834" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh834" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh834"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15448185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gingold</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Pilpel</div> <div class="given-names">Y.</div></div></div>
          <div class="year">2011</div>, 
<div class="article-title">Determinants of translation efficiency and accuracy,</div><div class="source">Mol. Syst. Biol</div> ., <div class="volume">7</div>, <div class="fpage">481</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Determinants%20of%20translation%20efficiency%20and%20accuracy%2C&amp;author=H.%20Gingold&amp;author=Y.%20Pilpel&amp;publication_year=2011&amp;journal=Mol.%20Syst.%20Biol&amp;volume=7&amp;pages=481" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/msb.2011.14" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fmsb.2011.14" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fmsb.2011.14"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21487400" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kanaya</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Yamada</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Kudo</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div></div> (<div class="year">1999</div>) 
<div class="article-title">Studies of codon usage and tRNA genes of 18 unicellular organisms and quantification of <em>Bacillus subtilis</em> tRNAs: gene expression level and species-specific diversity of codon usage based on multivariate analysis,</div><div class="source">Gene</div> , <div class="volume">238</div>, <div class="fpage">143</div>–<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Studies%20of%20codon%20usage%20and%20tRNA%20genes%20of%2018%20unicellular%20organisms%20and%20quantification%20of%20Bacillus%20subtilis%20tRNAs%3A%20gene%20expression%20level%20and%20species-specific%20diversity%20of%20codon%20usage%20based%20on%20multivariate%20analysis%2C&amp;author=S.%20Kanaya&amp;author=Y.%20Yamada&amp;author=Y.%20Kudo&amp;author=T.%20Ikemura&amp;publication_year=1999&amp;journal=Gene&amp;volume=238&amp;pages=143-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(99)00225-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(99)00225-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(99)00225-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10570992" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Boël</div>
               <div class="given-names">G.</div>
            </div>, <div class="name string-name"><div class="surname">Letso</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Neely</div> <div class="given-names">H.</div></div></div>, et al.  <div class="year">2016</div>, 
<div class="article-title">Codon influence on protein expression in <em>E. coli</em> correlates with mRNA levels,</div><div class="source">Nature</div> , <div class="volume">529</div>, <div class="fpage">358</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20influence%20on%20protein%20expression%20in%20E.%20coli%20correlates%20with%20mRNA%20levels%2C&amp;author=G.%20Bo%C3%ABl&amp;author=R.%20Letso&amp;author=H.%20Neely&amp;publication_year=2016&amp;journal=Nature&amp;volume=529&amp;pages=358-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature16509" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature16509" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature16509"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26760206" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Najafabadi</div>
               <div class="given-names">H.S.</div>
            </div>, <div class="name string-name"><div class="surname">Goodarzi</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Salavati</div> <div class="given-names">R.</div></div></div>
          <div class="year">2009</div>, 
<div class="article-title">Universal function-specificity of codon usage,</div><div class="source">Nucleic Acids Res</div> ., <div class="volume">37</div>, <div class="fpage">7014</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Universal%20function-specificity%20of%20codon%20usage%2C&amp;author=H.S.%20Najafabadi&amp;author=H.%20Goodarzi&amp;author=R.%20Salavati&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res&amp;volume=37&amp;pages=7014-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkp792" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkp792" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkp792"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19773421" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tuller</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Zur</div> <div class="given-names">H.</div></div></div>
          <div class="year">2015</div>, 
<div class="article-title">Multiple roles of the coding sequence 5′ end in gene expression regulation,</div><div class="source">Nucleic Acids Res</div> ., <div class="volume">43</div>, <div class="fpage">13</div>–<div class="lpage">28</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20roles%20of%20the%20coding%20sequence%205%E2%80%B2%20end%20in%20gene%20expression%20regulation%2C&amp;author=T.%20Tuller&amp;author=H.%20Zur&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res&amp;volume=43&amp;pages=13-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku1313" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku1313" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku1313"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25505165" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Novoa</div>
               <div class="given-names">E.M.</div>
            </div>, <div class="name string-name"><div class="surname">de Pouplana</div> <div class="given-names">L.R.</div></div></div>
          <div class="year">2012</div>, 
<div class="article-title">Speeding with control: codon usage, tRNAs, and ribosomes,</div><div class="source">Trends Genet</div> ., <div class="volume">28</div>, <div class="fpage">574</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Speeding%20with%20control%3A%20codon%20usage%2C%20tRNAs%2C%20and%20ribosomes%2C&amp;author=E.M.%20Novoa&amp;author=L.R.%20de%20Pouplana&amp;publication_year=2012&amp;journal=Trends%20Genet&amp;volume=28&amp;pages=574-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2012.07.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2012.07.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2012.07.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22921354" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ding</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Shah</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Plotkin</div> <div class="given-names">J.B.</div></div></div>
          <div class="year">2012</div>, 
<div class="article-title">Weak 5′-mRNA secondary structures in short eukaryotic genes,</div><div class="source">Genome Biol. Evol</div> ., <div class="volume">4</div>, <div class="fpage">1046</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Weak%205%E2%80%B2-mRNA%20secondary%20structures%20in%20short%20eukaryotic%20genes%2C&amp;author=Y.%20Ding&amp;author=P.%20Shah&amp;author=J.B.%20Plotkin&amp;publication_year=2012&amp;journal=Genome%20Biol.%20Evol&amp;volume=4&amp;pages=1046-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evs082" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevs082" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevs082"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23034215" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tuller</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Waldman</div> <div class="given-names">Y.Y.</div></div>, <div class="name string-name"><div class="surname">Kupiec</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Ruppin</div> <div class="given-names">E.</div></div></div>
          <div class="year">2010</div>, 
<div class="article-title">Translation efficiency is determined by both codon bias and folding energy,</div><div class="source">Proc. Natl. Acad. Sci. U.S.A</div> ., <div class="volume">107</div>, <div class="fpage">3645</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translation%20efficiency%20is%20determined%20by%20both%20codon%20bias%20and%20folding%20energy%2C&amp;author=T.%20Tuller&amp;author=Y.Y.%20Waldman&amp;author=M.%20Kupiec&amp;author=E.%20Ruppin&amp;publication_year=2010&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&amp;volume=107&amp;pages=3645-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0909910107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0909910107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0909910107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20133581" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zur</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Tuller</div> <div class="given-names">T.</div></div></div>
          <div class="year">2012</div>, 
<div class="article-title">Strong association between mRNA folding strength and protein abundance in <em>S. cerevisiae</em>,</div><div class="source">EMBO Rep</div> ., <div class="volume">13</div>, <div class="fpage">272</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Strong%20association%20between%20mRNA%20folding%20strength%20and%20protein%20abundance%20in%20S.%20cerevisiae%2C&amp;author=H.%20Zur&amp;author=T.%20Tuller&amp;publication_year=2012&amp;journal=EMBO%20Rep&amp;volume=13&amp;pages=272-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/embor.2011.262" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fembor.2011.262" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fembor.2011.262"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22249164" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Presnyak</div>
               <div class="given-names">V.</div>
            </div>, <div class="name string-name"><div class="surname">Alhusaini</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">Y.H.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Codon optimality is a major determinant of mRNA stability,</div><div class="source">Cell</div> , <div class="volume">160</div>, <div class="fpage">1111</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20optimality%20is%20a%20major%20determinant%20of%20mRNA%20stability%2C&amp;author=V.%20Presnyak&amp;author=N.%20Alhusaini&amp;author=Y.H.%20Chen&amp;publication_year=2015&amp;journal=Cell&amp;volume=160&amp;pages=1111-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2015.02.029" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2015.02.029" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2015.02.029"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25768907" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gu</div>
               <div class="given-names">W.</div>
            </div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Wilke</div> <div class="given-names">C.O.</div></div></div>
          <div class="year">2010</div>, 
<div class="article-title">A universal trend of reduced mRNA stability near the translation-initiation site in prokaryotes and eukaryotes,</div><div class="source">PLoS Comput. Biol</div> ., <div class="volume">6</div>, <div class="fpage">e1000664</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20universal%20trend%20of%20reduced%20mRNA%20stability%20near%20the%20translation-initiation%20site%20in%20prokaryotes%20and%20eukaryotes%2C&amp;author=W.%20Gu&amp;author=T.%20Zhou&amp;author=C.O.%20Wilke&amp;publication_year=2010&amp;journal=PLoS%20Comput.%20Biol&amp;volume=6&amp;pages=e1000664" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1000664" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1000664" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1000664"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20140241" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mao</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Cheng</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Tao</div> <div class="given-names">S.</div></div></div>
          <div class="year">2013</div>, 
<div class="article-title">Universally increased mRNA stability downstream of the translation initiation site in eukaryotes and prokaryotes,</div><div class="source">Gene</div> , <div class="volume">517</div>, <div class="fpage">230</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Universally%20increased%20mRNA%20stability%20downstream%20of%20the%20translation%20initiation%20site%20in%20eukaryotes%20and%20prokaryotes%2C&amp;author=Y.%20Mao&amp;author=W.%20Wang&amp;author=N.%20Cheng&amp;author=Q.%20Li&amp;author=S.%20Tao&amp;publication_year=2013&amp;journal=Gene&amp;volume=517&amp;pages=230-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2012.12.062" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2012.12.062" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2012.12.062"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23313297" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Chaney</div>
               <div class="given-names">J.L.</div>
            </div>, <div class="name string-name"><div class="surname">Clark</div> <div class="given-names">P.L.</div></div></div>
          <div class="year">2014</div>, 
<div class="article-title">Roles for synonymous codon usage in protein biogenesis,</div><div class="source">Annu. Rev. Biophys</div> ., <div class="volume">44</div>, <div class="fpage">143</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Roles%20for%20synonymous%20codon%20usage%20in%20protein%20biogenesis%2C&amp;author=J.L.%20Chaney&amp;author=P.L.%20Clark&amp;publication_year=2014&amp;journal=Annu.%20Rev.%20Biophys&amp;volume=44&amp;pages=143-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-biophys-060414-034333" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-biophys-060414-034333" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-biophys-060414-034333"> </span></div> </div></div></div></div></div><div content-id="dsx014-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Willie</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Majewski</div> <div class="given-names">J.</div></div></div>
          <div class="year">2004</div>, 
<div class="article-title">Evidence for codon bias selection at the pre-mRNA level in eukaryotes,</div><div class="source">Trends Genet</div> ., <div class="volume">20</div>, <div class="fpage">534</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20codon%20bias%20selection%20at%20the%20pre-mRNA%20level%20in%20eukaryotes%2C&amp;author=E.%20Willie&amp;author=J.%20Majewski&amp;publication_year=2004&amp;journal=Trends%20Genet&amp;volume=20&amp;pages=534-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2004.08.014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2004.08.014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2004.08.014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15475111" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gu</div>
               <div class="given-names">W.</div>
            </div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Zhai</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Xie</div> <div class="given-names">X.</div></div></div>
          <div class="year">2013</div>, 
<div class="article-title">Biological basis of miRNA action when their targets are located in human protein coding region,</div><div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e63403</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biological%20basis%20of%20miRNA%20action%20when%20their%20targets%20are%20located%20in%20human%20protein%20coding%20region%2C&amp;author=W.%20Gu&amp;author=X.%20Wang&amp;author=C.%20Zhai&amp;author=T.%20Zhou&amp;author=X.%20Xie&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e63403" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0063403" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0063403" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0063403"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23671676" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gu</div>
               <div class="given-names">W.</div>
            </div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Zhai</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Xie</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">T.</div></div></div>
          <div class="year">2012</div>, 
<div class="article-title">Selection on synonymous sites for increased accessibility around miRNA binding sites in plants,</div><div class="source">Mol. Biol. Evol</div> ., <div class="volume">29</div>, <div class="fpage">3037</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selection%20on%20synonymous%20sites%20for%20increased%20accessibility%20around%20miRNA%20binding%20sites%20in%20plants%2C&amp;author=W.%20Gu&amp;author=X.%20Wang&amp;author=C.%20Zhai&amp;author=X.%20Xie&amp;author=T.%20Zhou&amp;publication_year=2012&amp;journal=Mol.%20Biol.%20Evol&amp;volume=29&amp;pages=3037-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/mss109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmss109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmss109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22490819" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yu</div>
               <div class="given-names">C.H.</div>
            </div>, <div class="name string-name"><div class="surname">Dang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">Z.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Codon usage influences the local rate of translation elongation to regulate co-translational protein folding,</div><div class="source">Mol. Cell</div> , <div class="volume">59</div>, <div class="fpage">744</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20influences%20the%20local%20rate%20of%20translation%20elongation%20to%20regulate%20co-translational%20protein%20folding%2C&amp;author=C.H.%20Yu&amp;author=Y.%20Dang&amp;author=Z.%20Zhou&amp;publication_year=2015&amp;journal=Mol.%20Cell&amp;volume=59&amp;pages=744-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2015.07.018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2015.07.018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2015.07.018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26321254" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Deane</div>
               <div class="given-names">C.M.</div>
            </div>, <div class="name string-name"><div class="surname">Saunders</div> <div class="given-names">R.</div></div></div>
          <div class="year">2011</div>, 
<div class="article-title">The imprint of codons on protein structure,</div><div class="source">Biotechnol. J</div> ., <div class="volume">6</div>, <div class="fpage">641</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20imprint%20of%20codons%20on%20protein%20structure%2C&amp;author=C.M.%20Deane&amp;author=R.%20Saunders&amp;publication_year=2011&amp;journal=Biotechnol.%20J&amp;volume=6&amp;pages=641-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/biot.201000329" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fbiot.201000329" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fbiot.201000329"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21567957" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tsai</div>
               <div class="given-names">C.J.</div>
            </div>, <div class="name string-name"><div class="surname">Sauna</div> <div class="given-names">Z.E.</div></div>, <div class="name string-name"><div class="surname">Kimchi-Sarfaty</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Ambudkar</div> <div class="given-names">S.V.</div></div>, <div class="name string-name"><div class="surname">Gottesman</div> <div class="given-names">M.M.</div></div>, <div class="name string-name"><div class="surname">Nussinov</div> <div class="given-names">R.</div></div></div>
          <div class="year">2008</div>, 
<div class="article-title">Synonymous mutations and ribosome stalling can lead to altered folding pathways and distinct minima,</div><div class="source">J. Mol. Biol</div> ., <div class="volume">383</div>, <div class="fpage">281</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synonymous%20mutations%20and%20ribosome%20stalling%20can%20lead%20to%20altered%20folding%20pathways%20and%20distinct%20minima%2C&amp;author=C.J.%20Tsai&amp;author=Z.E.%20Sauna&amp;author=C.%20Kimchi-Sarfaty&amp;author=S.V.%20Ambudkar&amp;author=M.M.%20Gottesman&amp;author=R.%20Nussinov&amp;publication_year=2008&amp;journal=J.%20Mol.%20Biol&amp;volume=383&amp;pages=281-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jmb.2008.08.012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jmb.2008.08.012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jmb.2008.08.012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18722384" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nissley</div>
               <div class="given-names">D.A.</div>
            </div>, <div class="name string-name"><div class="surname">O’Brien</div> <div class="given-names">E.P.</div></div></div>
          <div class="year">2014</div>, 
<div class="article-title">Timing is everything: unifying codon translation rates and nascent proteome behavior,</div><div class="source">J. Am. Chem. Soc</div> ., <div class="volume">136</div>, <div class="fpage">17892</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Timing%20is%20everything%3A%20unifying%20codon%20translation%20rates%20and%20nascent%20proteome%20behavior%2C&amp;author=D.A.%20Nissley&amp;author=E.P.%20O%E2%80%99Brien&amp;publication_year=2014&amp;journal=J.%20Am.%20Chem.%20Soc&amp;volume=136&amp;pages=17892-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/ja510082j" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fja510082j" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fja510082j"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25486504" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Pechmann</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Chartron</div> <div class="given-names">J.W.</div></div>, <div class="name string-name"><div class="surname">Frydman</div> <div class="given-names">J.</div></div></div>
          <div class="year">2014</div>, 
<div class="article-title">Local slowdown of translation by nonoptimal codons promotes nascent-chain recognition by SRP in vivo,</div><div class="source">Nat. Struct. Mol. Biol</div> ., <div class="volume">21</div>, <div class="fpage">1100</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Local%20slowdown%20of%20translation%20by%20nonoptimal%20codons%20promotes%20nascent-chain%20recognition%20by%20SRP%20in%20vivo%2C&amp;author=S.%20Pechmann&amp;author=J.W.%20Chartron&amp;author=J.%20Frydman&amp;publication_year=2014&amp;journal=Nat.%20Struct.%20Mol.%20Biol&amp;volume=21&amp;pages=1100-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb.2919" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb.2919" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb.2919"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25420103" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Pechmann</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Frydman</div> <div class="given-names">J.</div></div></div>
          <div class="year">2013</div>, 
<div class="article-title">Evolutionary conservation of codon optimality reveals hidden signatures of cotranslational folding</div>, <div class="source">Nat. Struct. Mol. Biol</div> <em>.</em>, <div class="volume">20</div>, <div class="fpage">237</div>–<div class="lpage">43</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20conservation%20of%20codon%20optimality%20reveals%20hidden%20signatures%20of%20cotranslational%20folding&amp;author=S.%20Pechmann&amp;author=J.%20Frydman&amp;publication_year=2013&amp;journal=Nat.%20Struct.%20Mol.%20Biol&amp;volume=20&amp;pages=237-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb.2466" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb.2466" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb.2466"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23262490" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gustafsson</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Govindarajan</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Minshull</div> <div class="given-names">J.</div></div></div>
          <div class="year">2004</div>, 
<div class="article-title">Codon bias and heterologous protein expression,</div><div class="source">Trends Biotechnol</div> ., <div class="volume">22</div>, <div class="fpage">346</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20bias%20and%20heterologous%20protein%20expression%2C&amp;author=C.%20Gustafsson&amp;author=S.%20Govindarajan&amp;author=J.%20Minshull&amp;publication_year=2004&amp;journal=Trends%20Biotechnol&amp;volume=22&amp;pages=346-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tibtech.2004.04.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tibtech.2004.04.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tibtech.2004.04.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15245907" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sharp</div>
               <div class="given-names">P.M.</div>
            </div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">W.H.</div></div></div>
          <div class="year">1986</div>, 
<div class="article-title">An evolutionary perspective on synonymous codon usage in unicellular organisms,</div><div class="source">J. Mol. Evol</div> ., <div class="volume">24</div>, <div class="fpage">28</div>–<div class="lpage">38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20evolutionary%20perspective%20on%20synonymous%20codon%20usage%20in%20unicellular%20organisms%2C&amp;author=P.M.%20Sharp&amp;author=W.H.%20Li&amp;publication_year=1986&amp;journal=J.%20Mol.%20Evol&amp;volume=24&amp;pages=28-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02099948" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02099948" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02099948"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3104616" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tuller</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Carmi</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Vestsigian</div> <div class="given-names">K.</div></div></div>, et al.  <div class="year">2010</div>, 
<div class="article-title">An evolutionarily conserved mechanism for controlling the efficiency of protein translation,</div><div class="source">Cell</div> , <div class="volume">141</div>, <div class="fpage">344</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20evolutionarily%20conserved%20mechanism%20for%20controlling%20the%20efficiency%20of%20protein%20translation%2C&amp;author=T.%20Tuller&amp;author=A.%20Carmi&amp;author=K.%20Vestsigian&amp;publication_year=2010&amp;journal=Cell&amp;volume=141&amp;pages=344-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2010.03.031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2010.03.031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2010.03.031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20403328" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ragionieri</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Vitorino</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Frommlet</div> <div class="given-names">J.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Improving the accuracy of recombinant protein production through integration of bioinformatics, statistical and mass spectrometry methodologies,</div><div class="source">FEBS J</div> ., <div class="volume">282</div>, <div class="fpage">769</div>–<div class="lpage">87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improving%20the%20accuracy%20of%20recombinant%20protein%20production%20through%20integration%20of%20bioinformatics%2C%20statistical%20and%20mass%20spectrometry%20methodologies%2C&amp;author=L.%20Ragionieri&amp;author=R.%20Vitorino&amp;author=J.%20Frommlet&amp;publication_year=2015&amp;journal=FEBS%20J&amp;volume=282&amp;pages=769-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/febs.13181" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffebs.13181" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffebs.13181"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25532829" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhou</div>
               <div class="given-names">W.J.</div>
            </div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">J.K.</div></div>, <div class="name string-name"><div class="surname">Mao</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Miao</div> <div class="given-names">L.H.</div></div></div>
          <div class="year">2015</div>, 
<div class="article-title">Codon optimization, promoter and expression system selection that achieved high-level production of <em>Yarrowia lipolytica</em> lipase in <em>Pichia pastoris</em>,</div><div class="source">Enzyme Microb. Technol</div> ., <div class="volume">71</div>, <div class="fpage">66</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20optimization%2C%20promoter%20and%20expression%20system%20selection%20that%20achieved%20high-level%20production%20of%20Yarrowia%20lipolytica%20lipase%20in%20Pichia%20pastoris%2C&amp;author=W.J.%20Zhou&amp;author=J.K.%20Yang&amp;author=L.%20Mao&amp;author=L.H.%20Miao&amp;publication_year=2015&amp;journal=Enzyme%20Microb.%20Technol&amp;volume=71&amp;pages=66-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.enzmictec.2014.10.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.enzmictec.2014.10.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.enzmictec.2014.10.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25765312" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">J.R.</div>
            </div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">Y.Y.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">D.N.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Codon optimization significantly improves the expression level of α-amylase gene from <em>Bacillus licheniformis</em> in <em>Pichia pastoris</em>,</div><div class="source">Biomed Res. Int</div> ., <div class="volume">2015</div>, <div class="fpage">248680</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20optimization%20significantly%20improves%20the%20expression%20level%20of%20%CE%B1-amylase%20gene%20from%20Bacillus%20licheniformis%20in%20Pichia%20pastoris%2C&amp;author=J.R.%20Wang&amp;author=Y.Y.%20Li&amp;author=D.N.%20Liu&amp;publication_year=2015&amp;journal=Biomed%20Res.%20Int&amp;volume=2015&amp;pages=248680" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26171389" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yu</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Yan</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Gu</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">X.</div></div></div>
          <div class="year">2013</div>, 
<div class="article-title">Codon optimisation improves the expression of <em>Trichoderma viride</em> sp. endochitinase in <em>Pichia pastoris</em>,</div><div class="source">Sci. Rep</div> ., <div class="volume">3</div>, <div class="fpage">3043</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20optimisation%20improves%20the%20expression%20of%20Trichoderma%20viride%20sp.%20endochitinase%20in%20Pichia%20pastoris%2C&amp;author=P.%20Yu&amp;author=Y.%20Yan&amp;author=Q.%20Gu&amp;author=X.%20Wang&amp;publication_year=2013&amp;journal=Sci.%20Rep&amp;volume=3&amp;pages=3043" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep03043" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep03043" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep03043"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24154717" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Agashe</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Martinez-Gomez</div> <div class="given-names">N.C.</div></div>, <div class="name string-name"><div class="surname">Drummond</div> <div class="given-names">D.A.</div></div>, <div class="name string-name"><div class="surname">Marx</div> <div class="given-names">C.J.</div></div></div>
          <div class="year">2012</div>, 
<div class="article-title">Good codons, bad transcript: large reductions in gene expression and fitness arising from synonymous mutations in a key enzyme,</div><div class="source">Mol. Biol. Evol</div> ., <div class="volume">30</div>, <div class="fpage">549</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Good%20codons%2C%20bad%20transcript%3A%20large%20reductions%20in%20gene%20expression%20and%20fitness%20arising%20from%20synonymous%20mutations%20in%20a%20key%20enzyme%2C&amp;author=D.%20Agashe&amp;author=N.C.%20Martinez-Gomez&amp;author=D.A.%20Drummond&amp;author=C.J.%20Marx&amp;publication_year=2012&amp;journal=Mol.%20Biol.%20Evol&amp;volume=30&amp;pages=549-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/mss273" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmss273" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmss273"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23223712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rosano</div>
               <div class="given-names">G.L.</div>
            </div>, <div class="name string-name"><div class="surname">Ceccarelli</div> <div class="given-names">E.A.</div></div></div>
          <div class="year">2009</div>, 
<div class="article-title">Rare codon content affects the solubility of recombinant proteins in a codon bias-adjusted <em>Escherichia coli</em> strain,</div><div class="source">Microb. Cell Fact</div> ., <div class="volume">8</div>, <div class="fpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rare%20codon%20content%20affects%20the%20solubility%20of%20recombinant%20proteins%20in%20a%20codon%20bias-adjusted%20Escherichia%20coli%20strain%2C&amp;author=G.L.%20Rosano&amp;author=E.A.%20Ceccarelli&amp;publication_year=2009&amp;journal=Microb.%20Cell%20Fact&amp;volume=8&amp;pages=41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1475-2859-8-41" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1475-2859-8-41" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1475-2859-8-41"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19630980" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Spencer</div>
               <div class="given-names">P.S.</div>
            </div>, <div class="name string-name"><div class="surname">Siller</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Anderson</div> <div class="given-names">J.F.</div></div>, <div class="name string-name"><div class="surname">Barral</div> <div class="given-names">J.M.</div></div></div>
          <div class="year">2012</div>, 
<div class="article-title">Silent substitutions predictably alter translation elongation rates and protein folding efficiencies,</div><div class="source">J. Mol. Biol</div> ., <div class="volume">422</div>, <div class="fpage">328</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Silent%20substitutions%20predictably%20alter%20translation%20elongation%20rates%20and%20protein%20folding%20efficiencies%2C&amp;author=P.S.%20Spencer&amp;author=E.%20Siller&amp;author=J.F.%20Anderson&amp;author=J.M.%20Barral&amp;publication_year=2012&amp;journal=J.%20Mol.%20Biol&amp;volume=422&amp;pages=328-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jmb.2012.06.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jmb.2012.06.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jmb.2012.06.010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22705285" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Finger</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Gamer</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Klunkelfuß</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Bunk</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Biedendieck</div> <div class="given-names">R.</div></div></div>
          <div class="year">2015</div>, 
<div class="article-title">Impact of rare codons and the functional coproduction of rate-limiting tRNAs on recombinant protein production in <em>Bacillus megaterium</em>,</div><div class="source">Appl. Microbiol. Biotechnol</div> ., <div class="volume">99</div>, <div class="fpage">8999</div>–<div class="lpage">9010</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Impact%20of%20rare%20codons%20and%20the%20functional%20coproduction%20of%20rate-limiting%20tRNAs%20on%20recombinant%20protein%20production%20in%20Bacillus%20megaterium%2C&amp;author=C.%20Finger&amp;author=M.%20Gamer&amp;author=S.%20Klunkelfu%C3%9F&amp;author=B.%20Bunk&amp;author=R.%20Biedendieck&amp;publication_year=2015&amp;journal=Appl.%20Microbiol.%20Biotechnol&amp;volume=99&amp;pages=8999-9010" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00253-015-6744-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00253-015-6744-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00253-015-6744-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26138251" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cheong</div>
               <div class="given-names">D.E.</div>
            </div>, <div class="name string-name"><div class="surname">Ko</div> <div class="given-names">K.C.</div></div>, <div class="name string-name"><div class="surname">Han</div> <div class="given-names">Y.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Enhancing functional expression of heterologous proteins through random substitution of genetic codes in the 5′ coding region,</div><div class="source">Biotechnol. Bioeng</div> ., <div class="volume">112</div>, <div class="fpage">822</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Enhancing%20functional%20expression%20of%20heterologous%20proteins%20through%20random%20substitution%20of%20genetic%20codes%20in%20the%205%E2%80%B2%20coding%20region%2C&amp;author=D.E.%20Cheong&amp;author=K.C.%20Ko&amp;author=Y.%20Han&amp;publication_year=2015&amp;journal=Biotechnol.%20Bioeng&amp;volume=112&amp;pages=822-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/bit.v112.4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fbit.v112.4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fbit.v112.4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25323933" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Puxbaum</div>
               <div class="given-names">V.</div>
            </div>, <div class="name string-name"><div class="surname">Mattanovich</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Gasser</div> <div class="given-names">B.</div></div></div>
          <div class="year">2015</div>, 
<div class="article-title">Quo vadis? The challenges of recombinant protein folding and secretion in <em>Pichia pastoris</em>,</div><div class="source">Appl. Microbiol. Biotechnol</div> ., <div class="volume">99</div>, <div class="fpage">2925</div>–<div class="lpage">38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quo%20vadis%3F%20The%20challenges%20of%20recombinant%20protein%20folding%20and%20secretion%20in%20Pichia%20pastoris%2C&amp;author=V.%20Puxbaum&amp;author=D.%20Mattanovich&amp;author=B.%20Gasser&amp;publication_year=2015&amp;journal=Appl.%20Microbiol.%20Biotechnol&amp;volume=99&amp;pages=2925-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00253-015-6470-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00253-015-6470-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00253-015-6470-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25722021" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Friberg</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">von Rohr</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Gonnet</div> <div class="given-names">G.</div></div></div>
          <div class="year">2004</div>, 
<div class="article-title">Limitations of codon adaptation index and other coding DNA‐based features for prediction of protein expression in <em>Saccharomyces cerevisiae</em>,</div><div class="source">Yeast</div> , <div class="volume">21</div>, <div class="fpage">1083</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Limitations%20of%20codon%20adaptation%20index%20and%20other%20coding%20DNA%E2%80%90based%20features%20for%20prediction%20of%20protein%20expression%20in%20Saccharomyces%20cerevisiae%2C&amp;author=M.%20Friberg&amp;author=P.%20von%20Rohr&amp;author=G.%20Gonnet&amp;publication_year=2004&amp;journal=Yeast&amp;volume=21&amp;pages=1083-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0061" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0061" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0061"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15484285" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hockenberry</div>
               <div class="given-names">A.J.</div>
            </div>, <div class="name string-name"><div class="surname">Sirer</div> <div class="given-names">M.I.</div></div>, <div class="name string-name"><div class="surname">Amaral</div> <div class="given-names">L.A.N.</div></div>, <div class="name string-name"><div class="surname">Jewett</div> <div class="given-names">M.C.</div></div></div>
          <div class="year">2014</div>, 
<div class="article-title">Quantifying position-dependent codon usage bias,</div><div class="source">Mol. Biol. Evol</div> ., <div class="volume">31</div>, <div class="fpage">1880</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantifying%20position-dependent%20codon%20usage%20bias%2C&amp;author=A.J.%20Hockenberry&amp;author=M.I.%20Sirer&amp;author=L.A.N.%20Amaral&amp;author=M.C.%20Jewett&amp;publication_year=2014&amp;journal=Mol.%20Biol.%20Evol&amp;volume=31&amp;pages=1880-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msu126" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsu126" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsu126"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24710515" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zalucki</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Beacham</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Jennings</div> <div class="given-names">M.</div></div></div>
          <div class="year">2009</div>, 
<div class="article-title">Biased codon usage in signal peptides: a role in protein export,</div><div class="source">Trends Microbiol</div> ., <div class="volume">17</div>, <div class="fpage">146</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biased%20codon%20usage%20in%20signal%20peptides%3A%20a%20role%20in%20protein%20export%2C&amp;author=Y.%20Zalucki&amp;author=I.%20Beacham&amp;author=M.%20Jennings&amp;publication_year=2009&amp;journal=Trends%20Microbiol&amp;volume=17&amp;pages=146-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tim.2009.01.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tim.2009.01.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tim.2009.01.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19307122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Power</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Jones</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Beacham</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Bucholtz</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Jennings</div> <div class="given-names">M.</div></div></div>
          <div class="year">2004</div>, 
<div class="article-title">Whole genome analysis reveals a high incidence of non-optimal codons in secretory signal sequences of <em>Escherichia coli</em>,</div><div class="source">Biochem. Biophys. Res. Commun</div> ., <div class="volume">322</div>, <div class="fpage">1038</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole%20genome%20analysis%20reveals%20a%20high%20incidence%20of%20non-optimal%20codons%20in%20secretory%20signal%20sequences%20of%20Escherichia%20coli%2C&amp;author=P.%20Power&amp;author=R.%20Jones&amp;author=I.%20Beacham&amp;author=C.%20Bucholtz&amp;author=M.%20Jennings&amp;publication_year=2004&amp;journal=Biochem.%20Biophys.%20Res.%20Commun&amp;volume=322&amp;pages=1038-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbrc.2004.08.022" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbrc.2004.08.022" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbrc.2004.08.022"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15336569" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Emanuelsson</div>
               <div class="given-names">O.</div>
            </div>, <div class="name string-name"><div class="surname">Brunak</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">von Heijne</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Nielsen</div> <div class="given-names">H.</div></div></div>
          <div class="year">2007</div>, 
<div class="article-title">Locating proteins in the cell using TargetP, SignalP and related tools,</div><div class="source">Nat. Protoc</div> ., <div class="volume">2</div>, <div class="fpage">953</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Locating%20proteins%20in%20the%20cell%20using%20TargetP%2C%20SignalP%20and%20related%20tools%2C&amp;author=O.%20Emanuelsson&amp;author=S.%20Brunak&amp;author=G.%20von%20Heijne&amp;author=H.%20Nielsen&amp;publication_year=2007&amp;journal=Nat.%20Protoc&amp;volume=2&amp;pages=953-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2007.131" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2007.131" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2007.131"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17446895" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rice</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Longden</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Bleasby</div> <div class="given-names">A.</div></div></div>
          <div class="year">2000</div>, 
<div class="article-title">EMBOSS: the European Molecular Biology Open Software Suite,</div><div class="source">Trends Genet</div> ., <div class="volume">16</div>, <div class="fpage">276</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EMBOSS%3A%20the%20European%20Molecular%20Biology%20Open%20Software%20Suite%2C&amp;author=P.%20Rice&amp;author=I.%20Longden&amp;author=A.%20Bleasby&amp;publication_year=2000&amp;journal=Trends%20Genet&amp;volume=16&amp;pages=276-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(00)02024-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(00)02024-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(00)02024-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10827456" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Stenico</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Lloyd</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Sharp</div> <div class="given-names">P.</div></div></div>
          <div class="year">1994</div>, 
<div class="article-title">Codon usage in <em>Caenorhabditis elegans</em>: delineation of translational selection and mutational biases,</div><div class="source">Nucleic Acids Res</div> ., <div class="volume">22</div>, <div class="fpage">2437</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20in%20Caenorhabditis%20elegans%3A%20delineation%20of%20translational%20selection%20and%20mutational%20biases%2C&amp;author=M.%20Stenico&amp;author=A.%20Lloyd&amp;author=P.%20Sharp&amp;publication_year=1994&amp;journal=Nucleic%20Acids%20Res&amp;volume=22&amp;pages=2437-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/22.13.2437" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F22.13.2437" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F22.13.2437"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8041603" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nasrullah</div>
               <div class="given-names">I.</div>
            </div>, <div class="name string-name"><div class="surname">Butt</div> <div class="given-names">A.M.</div></div>, <div class="name string-name"><div class="surname">Tahir</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Idrees</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Tong</div> <div class="given-names">Y.</div></div></div>
          <div class="year">2015</div>, 
<div class="article-title">Genomic analysis of codon usage shows influence of mutation pressure, natural selection, and host features on Marburg virus evolution,</div><div class="source">BMC Evol. Biol</div> ., <div class="volume">15</div>, <div class="fpage">174</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20analysis%20of%20codon%20usage%20shows%20influence%20of%20mutation%20pressure%2C%20natural%20selection%2C%20and%20host%20features%20on%20Marburg%20virus%20evolution%2C&amp;author=I.%20Nasrullah&amp;author=A.M.%20Butt&amp;author=S.%20Tahir&amp;author=M.%20Idrees&amp;author=Y.%20Tong&amp;publication_year=2015&amp;journal=BMC%20Evol.%20Biol&amp;volume=15&amp;pages=174" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12862-015-0456-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12862-015-0456-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12862-015-0456-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26306510" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Schattner</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Brooks</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Lowe</div> <div class="given-names">T.</div></div></div>
          <div class="year">2005</div>, 
<div class="article-title">The tRNAscan-SE, snoscan and snoGPS web servers for the detection of tRNAs and snoRNAs,</div><div class="source">Nucleic Acids Res</div> ., <div class="volume">33(suppl 2)</div>, <div class="fpage">W686</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20tRNAscan-SE%2C%20snoscan%20and%20snoGPS%20web%20servers%20for%20the%20detection%20of%20tRNAs%20and%20snoRNAs%2C&amp;author=P.%20Schattner&amp;author=A.%20Brooks&amp;author=T.%20Lowe&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res&amp;volume=33(suppl 2)&amp;pages=W686-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki366" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki366" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki366"> </span></div> </div></div></div></div></div><div content-id="dsx014-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dana</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Tuller</div> <div class="given-names">T.</div></div></div>
          <div class="year">2014</div>, 
<div class="article-title">The effect of tRNA levels on decoding times of mRNA codons,</div><div class="source">Nucleic Acids Res</div> ., <div class="volume">42</div>, <div class="fpage">9171</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20effect%20of%20tRNA%20levels%20on%20decoding%20times%20of%20mRNA%20codons%2C&amp;author=A.%20Dana&amp;author=T.%20Tuller&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;pages=9171-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku646" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku646" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku646"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25056313" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Charif</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Lobry</div> <div class="given-names">J.</div></div></div>
          <div class="year">2007</div>, SeqinR 1.0-2: a contributed package to the R project for statistical computing devoted to biological sequences retrieval and analysis, In: <div class="person-group"><div class="name string-name"><div class="surname">Bastolla</div> <div class="given-names">U.</div></div>, <div class="name string-name"><div class="surname">Porto</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Roman</div> <div class="given-names">H.E.</div></div>, <div class="name string-name"><div class="surname">Vendruscolo</div> <div class="given-names">M.</div></div></div>, editors, <div class="source">Structural approaches to sequence evolution</div> , 
<div class="publisher-name">Springer</div>: 
<div class="publisher-loc">Berlin, Heidelberg</div>, pp. <div class="fpage">207</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20approaches%20to%20sequence%20evolution&amp;author=D.%20Charif&amp;author=J.%20Lobry&amp;author=U.%20Bastolla&amp;author=M.%20Porto&amp;author=H.E.%20Roman&amp;author=M.%20Vendruscolo&amp;publication_year=2007&amp;book=Structural%20approaches%20to%20sequence%20evolution" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/978-3-540-35306-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2F978-3-540-35306-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2F978-3-540-35306-5"> </span></div> </div></div></div></div></div><div content-id="dsx014-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Pages</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Aboyoun</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Gentleman</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">DebRoy</div> <div class="given-names">S.</div></div></div>
          <div class="year">2009</div>, 
<div class="article-title">String objects representing biological sequences, and matching algorithms</div>, <div class="source">R package version</div> . <div class="volume">2</div>, <div class="fpage">2</div>.</div></div></div></div><div content-id="dsx014-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation">
          <div class="collab">UniProt Consortium</div>. <div class="year">2015</div>, 
<div class="article-title">UniProt: a hub for protein information</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">43</div>, <div class="fpage">D204</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku989" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku989" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku989"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25348405" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Larkin</div>
               <div class="given-names">M.A.</div>
            </div>, <div class="name string-name"><div class="surname">Blackshields</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Brown</div> <div class="given-names">N.P.</div></div></div>, et al.  <div class="year">2007</div>, 
<div class="article-title">Clustal W and Clustal X version 2.0</div>, <div class="source">Bioinformatics</div> , <div class="volume">23</div>, <div class="fpage">2947</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Clustal%20W%20and%20Clustal%20X%20version%202.0&amp;author=M.A.%20Larkin&amp;author=G.%20Blackshields&amp;author=N.P.%20Brown&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=2947-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm404" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm404" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm404"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17846036" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tamura</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Stecher</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Peterson</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Filipski</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div></div>
          <div class="year">2013</div>, 
<div class="article-title">MEGA6: Molecular evolutionary genetics analysis version 6.0,</div><div class="source">Mol. Biol. Evol</div> ., <div class="volume">30</div>, <div class="fpage">2725</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA6%3A%20Molecular%20evolutionary%20genetics%20analysis%20version%206.0%2C&amp;author=K.%20Tamura&amp;author=G.%20Stecher&amp;author=D.%20Peterson&amp;author=A.%20Filipski&amp;author=S.%20Kumar&amp;publication_year=2013&amp;journal=Mol.%20Biol.%20Evol&amp;volume=30&amp;pages=2725-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/mst197" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmst197" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmst197"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24132122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Madej</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Lanczycki</div> <div class="given-names">C.J.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">D.</div></div></div>, et al.  <div class="year">2013</div>, 
<div class="article-title">MMDB and VAST+: tracking structural similarities between macromolecular complexes,</div><div class="source">Nucleic Acids Res</div> ., <div class="volume">42</div>, <div class="fpage">D297</div>–<div class="lpage">303</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MMDB%20and%20VAST%2B%3A%20tracking%20structural%20similarities%20between%20macromolecular%20complexes%2C&amp;author=T.%20Madej&amp;author=C.J.%20Lanczycki&amp;author=D.%20Zhang&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;pages=D297-303" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt1208" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt1208" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt1208"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24319143" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lorenz</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Bernhart</div> <div class="given-names">S.H.</div></div>, <div class="name string-name"><div class="surname">Zu Siederdissen</div> <div class="given-names">C.</div></div></div>, et al.  <div class="year">2011</div>, 
<div class="article-title">ViennaRNA Package 2.0</div>. <div class="source">Algorithms Mol. Biol</div> ., <div class="volume">6</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ViennaRNA%20Package%202.0&amp;author=R.%20Lorenz&amp;author=S.H.%20Bernhart&amp;author=C.%20Zu%20Siederdissen&amp;publication_year=2011&amp;journal=Algorithms%20Mol.%20Biol&amp;volume=6&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1748-7188-6-26" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1748-7188-6-26" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1748-7188-6-26"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21235792" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Moriyama</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Powell</div> <div class="given-names">J.</div></div></div>
          <div class="year">1998</div>, 
<div class="article-title">Gene length and codon usage bias in <em>Drosophila melanogaster</em>, <em>Saccharomyces cerevisiae</em> and <em>Escherichia coli</em>,</div><div class="source">Nucleic Acids Res</div> ., <div class="volume">26</div>, <div class="fpage">3188</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20length%20and%20codon%20usage%20bias%20in%20Drosophila%20melanogaster%2C%20Saccharomyces%20cerevisiae%20and%20Escherichia%20coli%2C&amp;author=E.%20Moriyama&amp;author=J.%20Powell&amp;publication_year=1998&amp;journal=Nucleic%20Acids%20Res&amp;volume=26&amp;pages=3188-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/26.13.3188" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F26.13.3188" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F26.13.3188"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9628917" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Duret</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Mouchiroud</div> <div class="given-names">D.</div></div></div>
          <div class="year">1999</div>, 
<div class="article-title">Expression pattern and, surprisingly, gene length shape codon usage in Caenorhabditis, Drosophila, and Arabidopsis,</div><div class="source">Proc. Natl. Acad. Sci. U.S.A</div> ., <div class="volume">96</div>, <div class="fpage">4482</div>–<div class="lpage">7</div><em>.</em><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20pattern%20and%2C%20surprisingly%2C%20gene%20length%20shape%20codon%20usage%20in%20Caenorhabditis%2C%20Drosophila%2C%20and%20Arabidopsis%2C&amp;author=L.%20Duret&amp;author=D.%20Mouchiroud&amp;publication_year=1999&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&amp;volume=96&amp;pages=4482-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.96.8.4482" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.96.8.4482" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.96.8.4482"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10200288" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Xu</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Dong</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Tong</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Gong</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Wen</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Zhuge</div> <div class="given-names">Q.</div></div></div>
          <div class="year">2013</div>, 
<div class="article-title">Analysis of synonymous codon usage patterns in seven different citrus species,</div><div class="source">Evol. Bioinf. Online</div> , <div class="volume">9</div>, <div class="fpage">215</div>–<div class="lpage">28</div>.</div></div></div></div><div content-id="dsx014-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wei</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">He</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Jia</div> <div class="given-names">X.</div></div></div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Analysis of codon usage bias of mitochondrial genome in <em>Bombyx mori</em> and its relation to evolution,</div><div class="source">BMC Evol. Biol</div> ., <div class="volume">14</div>, <div class="fpage">262</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20codon%20usage%20bias%20of%20mitochondrial%20genome%20in%20Bombyx%20mori%20and%20its%20relation%20to%20evolution%2C&amp;author=L.%20Wei&amp;author=J.%20He&amp;author=X.%20Jia&amp;publication_year=2014&amp;journal=BMC%20Evol.%20Biol&amp;volume=14&amp;pages=262" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12862-014-0262-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12862-014-0262-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12862-014-0262-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25515024" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jia</div>
               <div class="given-names">X.</div>
            </div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Zheng</div> <div class="given-names">H.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Non-uniqueness of factors constraint on the codon usage in <em>Bombyx mori</em>,</div><div class="source">BMC Genomics</div> , <div class="volume">16</div>, <div class="fpage">356</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Non-uniqueness%20of%20factors%20constraint%20on%20the%20codon%20usage%20in%20Bombyx%20mori%2C&amp;author=X.%20Jia&amp;author=S.%20Liu&amp;author=H.%20Zheng&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=356" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-1596-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-1596-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-1596-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25943559" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Williford</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Demuth</div> <div class="given-names">J.</div></div></div>
          <div class="year">2012</div>, 
<div class="article-title">Gene expression levels are correlated with synonymous codon usage, amino acid composition, and gene architecture in the red flour beetle, <em>Tribolium castaneum</em></div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">29</div>, <div class="fpage">3755</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20expression%20levels%20are%20correlated%20with%20synonymous%20codon%20usage%2C%20amino%20acid%20composition%2C%20and%20gene%20architecture%20in%20the%20red%20flour%20beetle%2C%20Tribolium%20castaneum&amp;author=A.%20Williford&amp;author=J.%20Demuth&amp;publication_year=2012&amp;journal=Mol.%20Biol.%20Evol&amp;volume=29&amp;pages=3755-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/mss184" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmss184" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmss184"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22826459" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Clarke</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Clark</div> <div class="given-names">P.</div></div></div>
          <div class="year">2010</div>, 
<div class="article-title">Increased incidence of rare codon clusters at 5′ and 3′ gene termini: implications for function,</div><div class="source">BMC Genomics</div> , <div class="volume">11</div>, <div class="fpage">118</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Increased%20incidence%20of%20rare%20codon%20clusters%20at%205%E2%80%B2%20and%203%E2%80%B2%20gene%20termini%3A%20implications%20for%20function%2C&amp;author=T.%20Clarke&amp;author=P.%20Clark&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=118" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-118" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-118" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-118"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20167116" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dana</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Tuller</div> <div class="given-names">T.</div></div></div>
          <div class="year">2014</div>, 
<div class="article-title">Properties and determinants of codon decoding time distributions</div>. <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">S13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Properties%20and%20determinants%20of%20codon%20decoding%20time%20distributions&amp;author=A.%20Dana&amp;author=T.%20Tuller&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=S13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-S6-S13" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-S6-S13" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-S6-S13"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25572668" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">Y.D.</div>
            </div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">Y.Q.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">J.S.</div></div>, <div class="name string-name"><div class="surname">Dong</div> <div class="given-names">H.J.</div></div>, <div class="name string-name"><div class="surname">Guan</div> <div class="given-names">W.J.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">H.</div></div></div>
          <div class="year">2006</div>, 
<div class="article-title">Whole genome analysis of non-optimal codon usage in secretory signal sequences of <em>Streptomyces coelicolor</em>,</div><div class="source">BioSystems</div> , <div class="volume">85</div>, <div class="fpage">225</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole%20genome%20analysis%20of%20non-optimal%20codon%20usage%20in%20secretory%20signal%20sequences%20of%20Streptomyces%20coelicolor%2C&amp;author=Y.D.%20Li&amp;author=Y.Q.%20Li&amp;author=J.S.%20Chen&amp;author=H.J.%20Dong&amp;author=W.J.%20Guan&amp;author=H.%20Zhou&amp;publication_year=2006&amp;journal=BioSystems&amp;volume=85&amp;pages=225-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.biosystems.2006.02.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.biosystems.2006.02.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.biosystems.2006.02.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16644095" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Quax</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Claassens</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Söll</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">van der Oost</div> <div class="given-names">J.</div></div></div>
          <div class="year">2015</div>, 
<div class="article-title">Codon bias as a means to fine-tune gene expression,</div><div class="source">Mol. Cell</div> , <div class="volume">59</div>, <div class="fpage">149</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20bias%20as%20a%20means%20to%20fine-tune%20gene%20expression%2C&amp;author=T.%20Quax&amp;author=N.%20Claassens&amp;author=D.%20S%C3%B6ll&amp;author=J.%20van%20der%20Oost&amp;publication_year=2015&amp;journal=Mol.%20Cell&amp;volume=59&amp;pages=149-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2015.05.035" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2015.05.035" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2015.05.035"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26186290" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Plotkin</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Kudla</div> <div class="given-names">G.</div></div></div>
          <div class="year">2010</div>, 
<div class="article-title">Synonymous but not the same: the causes and consequences of codon bias,</div><div class="source">Nat. Rev. Genet</div> ., <div class="volume">12</div>, <div class="fpage">32</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synonymous%20but%20not%20the%20same%3A%20the%20causes%20and%20consequences%20of%20codon%20bias%2C&amp;author=J.%20Plotkin&amp;author=G.%20Kudla&amp;publication_year=2010&amp;journal=Nat.%20Rev.%20Genet&amp;volume=12&amp;pages=32-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2899" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2899" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2899"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21102527" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Shah</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Ding</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Niemczyk</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kudla</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Plotkin</div> <div class="given-names">J.B.</div></div></div>
          <div class="year">2013</div>, 
<div class="article-title">Rate-limiting steps in yeast protein translation,</div><div class="source">Cell</div> , <div class="volume">153</div>, <div class="fpage">1589</div>–<div class="lpage">601</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rate-limiting%20steps%20in%20yeast%20protein%20translation%2C&amp;author=P.%20Shah&amp;author=Y.%20Ding&amp;author=M.%20Niemczyk&amp;author=G.%20Kudla&amp;author=J.B.%20Plotkin&amp;publication_year=2013&amp;journal=Cell&amp;volume=153&amp;pages=1589-601" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2013.05.049" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2013.05.049" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2013.05.049"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23791185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Navon</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Pilpel</div> <div class="given-names">Y.</div></div></div>
          <div class="year">2011</div>, 
<div class="article-title">The role of codon selection in regulation of translation efficiency deduced from synthetic libraries,</div><div class="source">Genome Biol</div> ., <div class="volume">12</div>, <div class="fpage">R12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20role%20of%20codon%20selection%20in%20regulation%20of%20translation%20efficiency%20deduced%20from%20synthetic%20libraries%2C&amp;author=S.%20Navon&amp;author=Y.%20Pilpel&amp;publication_year=2011&amp;journal=Genome%20Biol&amp;volume=12&amp;pages=R12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2011-12-2-r12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2011-12-2-r12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2011-12-2-r12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21284851" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Goodman</div>
               <div class="given-names">D.B.</div>
            </div>, <div class="name string-name"><div class="surname">Church</div> <div class="given-names">G.M.</div></div>, <div class="name string-name"><div class="surname">Kosuri</div> <div class="given-names">S.</div></div></div>
          <div class="year">2013</div>, 
<div class="article-title">Causes and effects of N-terminal codon bias in bacterial genes</div>. <div class="source">Science</div> , <div class="volume">342</div>, <div class="fpage">475</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Causes%20and%20effects%20of%20N-terminal%20codon%20bias%20in%20bacterial%20genes&amp;author=D.B.%20Goodman&amp;author=G.M.%20Church&amp;author=S.%20Kosuri&amp;publication_year=2013&amp;journal=Science&amp;volume=342&amp;pages=475-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1241934" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1241934" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1241934"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24072823" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B72" class="ref-content "><div class="label -label">72</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Massahi</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Çalık</div> <div class="given-names">P.</div></div></div>
          <div class="year">2015</div>, 
<div class="article-title">In-silico determination of <em>Pichia pastoris</em> signal peptides for extracellular recombinant protein production,</div><div class="source">J. Theor. Biol</div> ., <div class="volume">364</div>, <div class="fpage">179</div>–<div class="lpage">88</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In-silico%20determination%20of%20Pichia%20pastoris%20signal%20peptides%20for%20extracellular%20recombinant%20protein%20production%2C&amp;author=A.%20Massahi&amp;author=P.%20%C3%87al%C4%B1k&amp;publication_year=2015&amp;journal=J.%20Theor.%20Biol&amp;volume=364&amp;pages=179-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jtbi.2014.08.048" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jtbi.2014.08.048" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jtbi.2014.08.048"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25218497" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B73" class="ref-content "><div class="label -label">73</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tang</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Bao</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Shen</div> <div class="given-names">Y.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Engineering protein folding and translocation improves heterologous protein secretion in <em>Saccharomyces cerevisiae</em>,</div><div class="source">Biotechnol. Bioeng</div> ., <div class="volume">112</div>, <div class="fpage">1872</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Engineering%20protein%20folding%20and%20translocation%20improves%20heterologous%20protein%20secretion%20in%20Saccharomyces%20cerevisiae%2C&amp;author=H.%20Tang&amp;author=X.%20Bao&amp;author=Y.%20Shen&amp;publication_year=2015&amp;journal=Biotechnol.%20Bioeng&amp;volume=112&amp;pages=1872-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/bit.25596" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fbit.25596" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fbit.25596"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25850421" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B74" class="ref-content "><div class="label -label">74</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yarimizu</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Nakamura</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Hoshida</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Akada</div> <div class="given-names">R.</div></div></div>
          <div class="year">2015</div>, 
<div class="article-title">Synthetic signal sequences that enable efficient secretory protein production in the yeast <em>Kluyveromyces marxianus</em>,</div><div class="source">Microb. Cell Fact</div> ., <div class="volume">14</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synthetic%20signal%20sequences%20that%20enable%20efficient%20secretory%20protein%20production%20in%20the%20yeast%20Kluyveromyces%20marxianus%2C&amp;author=T.%20Yarimizu&amp;author=M.%20Nakamura&amp;author=H.%20Hoshida&amp;author=R.%20Akada&amp;publication_year=2015&amp;journal=Microb.%20Cell%20Fact&amp;volume=14&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12934-015-0203-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12934-015-0203-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12934-015-0203-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25567661" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B75" class="ref-content "><div class="label -label">75</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tsuchiya</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Morioka</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Shirai</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Yokomizo</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Yoshida</div> <div class="given-names">K.</div></div></div>
          <div class="year">2003</div>, Gene design of signal sequence for effective secretion of protein, In: <div class="source">Nucleic Acids Symposium Series</div> , Vol. <div class="volume">3</div>. 
<div class="publisher-name">Oxford University Press</div>, pp. <div class="fpage">261</div>–<div class="lpage">2</div>.</div></div></div></div><div content-id="dsx014-B76" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B76" class="ref-content "><div class="label -label">76</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Chartier</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Gaudreault</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Najmanovich</div> <div class="given-names">R.</div></div></div>
          <div class="year">2012</div>, 
<div class="article-title">Large-scale analysis of conserved rare codon clusters suggests an involvement in co-translational molecular recognition events,</div><div class="source">Bioinformatics</div> , <div class="volume">28</div>, <div class="fpage">1438</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20analysis%20of%20conserved%20rare%20codon%20clusters%20suggests%20an%20involvement%20in%20co-translational%20molecular%20recognition%20events%2C&amp;author=M.%20Chartier&amp;author=F.%20Gaudreault&amp;author=R.%20Najmanovich&amp;publication_year=2012&amp;journal=Bioinformatics&amp;volume=28&amp;pages=1438-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bts149" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbts149" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbts149"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22467916" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B77" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B77" class="ref-content "><div class="label -label">77</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Shabalina</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Spiridonov</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Kashina</div> <div class="given-names">A.</div></div></div>
          <div class="year">2013</div>, 
<div class="article-title">Sounds of silence: synonymous nucleotides as a key to biological regulation and complexity,</div><div class="source">Nucleic Acids Res</div> ., <div class="volume">41</div>, <div class="fpage">2073</div>–<div class="lpage">94</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sounds%20of%20silence%3A%20synonymous%20nucleotides%20as%20a%20key%20to%20biological%20regulation%20and%20complexity%2C&amp;author=S.%20Shabalina&amp;author=N.%20Spiridonov&amp;author=A.%20Kashina&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;pages=2073-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks1205" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks1205" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks1205"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23293005" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B78" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B78" class="ref-content "><div class="label -label">78</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nilsson</div>
               <div class="given-names">O.B.</div>
            </div>, <div class="name string-name"><div class="surname">Hedman</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Marino</div> <div class="given-names">J.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Cotranslational protein folding inside the ribosome exit tunnel,</div><div class="source">Cell Rep</div> ., <div class="volume">12</div>, <div class="fpage">1533</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cotranslational%20protein%20folding%20inside%20the%20ribosome%20exit%20tunnel%2C&amp;author=O.B.%20Nilsson&amp;author=R.%20Hedman&amp;author=J.%20Marino&amp;publication_year=2015&amp;journal=Cell%20Rep&amp;volume=12&amp;pages=1533-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.celrep.2015.07.065" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.celrep.2015.07.065" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.celrep.2015.07.065"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26321634" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx014-B79" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx014-B79" class="ref-content "><div class="label -label">79</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wilson</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Beckmann</div> <div class="given-names">R.</div></div></div>
          <div class="year">2011</div>, 
<div class="article-title">The ribosomal tunnel as a functional environment for nascent polypeptide folding and translational stalling,</div><div class="source">Curr. Opin. Struct. Biol</div> ., <div class="volume">21</div>, <div class="fpage">274</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20ribosomal%20tunnel%20as%20a%20functional%20environment%20for%20nascent%20polypeptide%20folding%20and%20translational%20stalling%2C&amp;author=D.%20Wilson&amp;author=R.%20Beckmann&amp;publication_year=2011&amp;journal=Curr.%20Opin.%20Struct.%20Biol&amp;volume=21&amp;pages=274-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.sbi.2011.01.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.sbi.2011.01.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.sbi.2011.01.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21316217" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=3752474&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_dataset_s1_dsx014.xlsx?Expires=2147483647&amp;Signature=yn7ec37o6w~qyUL2X3g5iDJYPjZJ-UekkspdLw~wG9pMLv8LP7-VmOr3~JmsoUI5j-AEQ7ixLD3SSBktaex5CTjFnqyvch22~Y6KknPvypS8hUybL-t4LcS-2I9iKS752oDo-XuqsC1e6n3MaOkrBAoYUxSQBlBbjbxKZ5aPCRRgUqX7KUvXF5~re6xf2ZTH5-wrZx5pqS4x6WCf1FRqyhI0F8hJ7XaeKEctsOuTyEd6Zu~-A2x0zp8aFyO9hIDIB0REHmO4WVDkGqOI19Pco9gjbx2YlG2vNTOqA4H6~x95sXhsrRnp11EA1YfDbYmEnp1eKfcyqGsikaEkl1Hh~A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Dataset S1</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_dataset_s2_dsx014.xlsx?Expires=2147483647&amp;Signature=UvPuY4OGA0v6iQlgw~~~bWwee6dvmCeDA5fIGl7w9962MwPV3qmsBLJsHkA0rOeBXAoOaO~qSR4sYrCPkfVcLaFIVvJJqZX4xRXpUXq8Bsbzhx7ri5c45EhCMQ-zc0U-aNZ9o~S6CDSL8sGYCwiH60A7VsbY4noqyjAVtxdf6Qjm0q2qP2eDbLHA1TVjflf8UwEq4l3qGodD~CcXEIPnen4RUB8xK1jpqXoIcQzfOXN27lKEWuQDmkoZrHWkWjUV6lVR2anKuuYZFL-Kx~BCYvtSUYTK-xDtag90oypHSDNejyePca-8DGre4ZE8vGPegoOI6LHTMSAkcq5UXs5dQA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Dataset S2</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx014/2/suppl_material_dsx014.pdf?Expires=2147483647&amp;Signature=yrjqF1m4pc3Nh7lf8ZdqxQMqJLie-IFYrxEBJLcpqDMU9-oV5zGeyuesVkp5VQyxlzhU3zRsQ5V7s0CE4tzXC9VLZwjVNYLbCg71yM~MUu4INxBSLKGJz-v5sKsGC2imGQ4TSce61gCHMWq4eaQ-D76mDKsHmzubvAB-a2bBBxivZocHwwtUzcM4OJaXpsknqlOW6x-RXwQBYjVSIChlumeabOAPtOUxiQhOnoeMUW2xXepqOP3dagDP3chbKxvFxJPV7gB8t5xpDAGZBv7vi-2N6gKuBo-Iw7Rd5OQigzF-bhrHnjqYDtEm~gR1DIfN8gxdwbPU1O1xQZBFPTKMNA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Material</div></a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,209</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx014"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="3752474"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f4%2f419%2f3752474">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F4%2F419%2F3752474" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/3752474'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Integrated analysis of individual codon contribution to protein biosynthesis reveals a new approach to improving the basis of rational gene design" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx014");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f4%2f419%2f3752474">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

