

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/18/6/413/476838","@type":"ScholarlyArticle","name":" Mycoplasma hyopneumoniae  Transcription Unit Organization: Genome Survey and Prediction","datePublished":"2011-11-14","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/18/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2011-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsr028","keywords":["ORF cluster","intergenic regions","cotranscription","transcriptional units"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/18/6/413/476838","author":[{"name":"Siqueira, Franciele Maboni","affiliation":"Programa de Pós-Graduação em Biologia Molecular e Celular, Departamento de Biologia Molecular e Biotecnologia, Centro de Biotecnologia, Universidade Federal do Rio Grande do Sul, Porto Alegre ,  Brazil","@type":"Person"},{"name":"Schrank, Augusto","affiliation":"Programa de Pós-Graduação em Biologia Molecular e Celular, Departamento de Biologia Molecular e Biotecnologia, Centro de Biotecnologia, Universidade Federal do Rio Grande do Sul, Porto Alegre ,  Brazil","@type":"Person"},{"name":"Schrank, Irene Silveira","affiliation":"Programa de Pós-Graduação em Biologia Molecular e Celular, Departamento de Biologia Molecular e Biotecnologia, Centro de Biotecnologia, Universidade Federal do Rio Grande do Sul, Porto Alegre ,  Brazil","@type":"Person"}],"description":"Abstract.  Mycoplasma hyopneumoniae is associated with swine respiratory diseases. Although gene organization and regulation are well known in many prokaryotic ","pageStart":"413","pageEnd":"422","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/6/1/m_cover.gif?Expires=2147483647&Signature=qL9inMhepM2zFUhwbU8mgSKjtZVxpg~PmNu8fbxg-9GXAGTJaQrhv6qnZRDeMk7qu~VNHreHxR8Oz-VBGUn7klik4EpJ16UF40231HSss3FWSUANYRXx6t042t9kZJxg5tVYoWTO8VtSK1XYiJ1~Hb4XzfVloDJNBsISl1abmgO6rLxcEn6ZsFnsGHwCs2gNKgd6or0KznZwXt8UzZ8lxgCsA3E4dHjQXfOwyJhuIWr89hghDvrxBGqblSwMEBtNAQaF~zOz6WymUDfO7ME02xpTq6iZ5QfAMCaAMZasP1hnDFuY5z9kxUJ-TfMyLTyjyLEQfP4lam1wVdxuX5tJEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":" Mycoplasma hyopneumoniae  Transcription Unit Organization: Genome Survey and Prediction","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/6/1/m_cover.gif?Expires=2147483647&Signature=qL9inMhepM2zFUhwbU8mgSKjtZVxpg~PmNu8fbxg-9GXAGTJaQrhv6qnZRDeMk7qu~VNHreHxR8Oz-VBGUn7klik4EpJ16UF40231HSss3FWSUANYRXx6t042t9kZJxg5tVYoWTO8VtSK1XYiJ1~Hb4XzfVloDJNBsISl1abmgO6rLxcEn6ZsFnsGHwCs2gNKgd6or0KznZwXt8UzZ8lxgCsA3E4dHjQXfOwyJhuIWr89hghDvrxBGqblSwMEBtNAQaF~zOz6WymUDfO7ME02xpTq6iZ5QfAMCaAMZasP1hnDFuY5z9kxUJ-TfMyLTyjyLEQfP4lam1wVdxuX5tJEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content=" Mycoplasma hyopneumoniae  Transcription Unit Organization: Genome Survey and Prediction" />
    <meta property="og:description" content="Abstract.  Mycoplasma hyopneumoniae is associated with swine respiratory diseases. Although gene organization and regulation are well known in many prokaryotic " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/18/6/413/476838" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2011-11-14" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Siqueira, Franciele Maboni" /><meta name="citation_author_institution" content="Programa de Pós-Graduação em Biologia Molecular e Celular, Departamento de Biologia Molecular e Biotecnologia, Centro de Biotecnologia, Universidade Federal do Rio Grande do Sul, Porto Alegre, Brazil" /><meta name="citation_author" content="Schrank, Augusto" /><meta name="citation_author_institution" content="Programa de Pós-Graduação em Biologia Molecular e Celular, Departamento de Biologia Molecular e Biotecnologia, Centro de Biotecnologia, Universidade Federal do Rio Grande do Sul, Porto Alegre, Brazil" /><meta name="citation_author" content="Schrank, Irene Silveira" /><meta name="citation_author_institution" content="Programa de Pós-Graduação em Biologia Molecular e Celular, Departamento de Biologia Molecular e Biotecnologia, Centro de Biotecnologia, Universidade Federal do Rio Grande do Sul, Porto Alegre, Brazil" /><meta name="citation_title" content="Mycoplasma hyopneumoniae Transcription Unit Organization: Genome Survey and Prediction" /><meta name="citation_firstpage" content="413" /><meta name="citation_lastpage" content="422" /><meta name="citation_doi" content="10.1093/dnares/dsr028" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="18" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2011/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Mycoplasmal disease; citation_publisher=Iowa State University Press, Ames; Diseases of Swine;  citation_year=1992;  citation_pages=537-51; " /><meta name="citation_reference" content="citation_title=The genome sequence of strain 232, the agent of swine mycoplasmosis; J. Bacteriol.;  citation_year=2004;  citation_volume=186;  citation_pages=7123-33; " /><meta name="citation_reference" content="citation_title=Swine and poultry pathogens: the complete genome sequences of two strains of Mycoplasma hyopneumoniae and a strain of Mycoplasma synoviae; J. Bacteriol.;  citation_year=2005;  citation_volume=187;  citation_pages=5568-77; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of Mycoplasma hyopneumoniae strain 168; J. Bacteriol.;  citation_year=2011;  citation_volume=193;  citation_pages=1016-7; " /><meta name="citation_reference" content="citation_title=Analysis of the nucleotide sequence of the P1 operon of Mycoplasma pneumoniae; Gene;  citation_year=1988;  citation_volume=73;  citation_pages=175-83; " /><meta name="citation_reference" content="citation_title=Comparative analysis of the genomes of the bacteria Mycoplasma pneumoniae and Mycoplasma genitalium; Nucl. Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=701-12; " /><meta name="citation_reference" content="citation_title=Transcriptional analysis of the hmw gene cluster of Mycoplasma pneumoniae; J. Bacteriol.;  citation_year=1999;  citation_volume=181;  citation_pages=4978-85; " /><meta name="citation_reference" content="citation_title=Transcriptional starts for cytadherence-related operons of Mycoplasma genitalium; FEMS Microbiol. Lett.;  citation_year=2003;  citation_volume=229;  citation_pages=73-81; " /><meta name="citation_reference" content="citation_title=In vivo expression analysis of the P97 and P102 paralog families of Mycoplasma hyopneumoniae; Infect. Immun.;  citation_year=2005;  citation_volume=73;  citation_pages=7784-7; " /><meta name="citation_reference" content="citation_title=Transcriptional analysis of the conserved ftsZ gene cluster in Mycoplasma genitalium and Mycoplasma pneumoniae; J. Bacteriol.;  citation_year=2005;  citation_volume=187;  citation_pages=4542-51; " /><meta name="citation_reference" content="citation_title=Synthesis, stability, and function of cytadhesin P1 and accessory protein B/C complex of Mycoplasma pneumoniae; J. Bacteriol.;  citation_year=2006;  citation_volume=188;  citation_pages=569-75; " /><meta name="citation_reference" content="citation_title=Transcriptome complexity in a genome-reduced bacterium; Science;  citation_year=2009;  citation_volume=326;  citation_pages=1268-71; " /><meta name="citation_reference" content="citation_title=The guanine and cytosine content of genomic DNA and bacterial evolution; Proc. Natl. Acad. Sci. USA;  citation_year=1987;  citation_volume=84;  citation_pages=166-9; " /><meta name="citation_reference" content="citation_title=Transcription in Mycoplasma pneumoniae; Nucl. Acids Res.;  citation_year=2000;  citation_volume=28;  citation_pages=4488-96; " /><meta name="citation_reference" content="citation_title=Construction and use of derivatives of transposon Tn4001 that function in Mycoplasma pulmonis and Mycoplasma arthritidis; J. Bacteriol.;  citation_year=2000;  citation_volume=182;  citation_pages=4343-7; " /><meta name="citation_reference" content="citation_title=Probing in vivo promoter activities in Mycoplasma pneumoniae: a system for generation of single-copy reporter constructs; Appl. Environ. Microbiol.;  citation_year=2006;  citation_volume=72;  citation_pages=1696-9; " /><meta name="citation_reference" content="citation_title=Versatile use of oriC plasmids for functional genomics of Mycoplasma capricolum subsp. capricolum; Appl. Environ. Microbiol.;  citation_year=2005;  citation_volume=71;  citation_pages=2888-93; " /><meta name="citation_reference" content="citation_title=Detection and Quantification of intergenic transcription in Mycoplasma hyopneumoniae; Microbiology;  citation_year=2010;  citation_volume=156;  citation_pages=2305-15; " /><meta name="citation_reference" content="citation_title=Analysis of complete genomes suggests that many prokaryotes do not rely on hairpin formation in transcription termination; Nucl. Acids Res.;  citation_year=1998;  citation_volume=26;  citation_pages=5456-63; " /><meta name="citation_reference" content="citation_title=Prediction of transcriptional terminators in Bacillus subtilis and related species; PLoS Comput. Biol.;  citation_year=2005;  citation_volume=1;  citation_pages=212-21; " /><meta name="citation_reference" content="citation_title=Regulation of gene expression in mycoplasmas: contribution from Mycoplasma hyopneumoniae and Mycoplasma synoviae genome sequences; Genet. Mol. Biol.;  citation_year=2007;  citation_volume=30;  citation_pages=277-82; " /><meta name="citation_reference" content="citation_title=Artemis: sequence visualization and annotation; Bioinformatics;  citation_year=2000;  citation_volume=16;  citation_pages=944-5; " /><meta name="citation_reference" content="citation_title=Artemis and ACT: viewing, annotating and comparing sequences stored in a relational database; Bioinformatics;  citation_year=2008;  citation_volume=24;  citation_pages=2672-6; " /><meta name="citation_reference" content="citation_title=Some recommendations concerning primary isolation of Mycoplasma suipneumoniae and Mycoplasma flocculare a survey; Nordisk Veterinaer. Medicin.;  citation_year=1975;  citation_volume=27;  citation_pages=337-9; " /><meta name="citation_reference" content="citation_publisher=Cold Spring Harbor Laboratory Press, New York; Molecular Cloning a Laboratory Manual;  citation_year=2001; " /><meta name="citation_reference" content="citation_title=Connected gene neighborhoods in prokaryotic genomes; Nucl. Acids Res.;  citation_year=2002;  citation_volume=30;  citation_pages=2212-23; " /><meta name="citation_reference" content="citation_title=Operon prediction by comparative genomics: an application to the Synechococcus sp. WH8102 genome; Nucl. Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=2147-57; " /><meta name="citation_reference" content="citation_title=A novel method for accurate operon predictions in all sequenced prokaryotes; Nucl. Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=880-92; " /><meta name="citation_reference" content="citation_title=A predicted operon map for mycobacterium tuberculosis; Nucl. Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=5085-95; " /><meta name="citation_reference" content="citation_title=The transcription unit architecture of the Escherichia coli genome; Nat. Biotechnol.;  citation_year=2009;  citation_volume=27;  citation_pages=1043-9; " /><meta name="citation_reference" content="citation_title=Characterization of relationships between transcriptional units and operon structures in Bacillus subtilis and Escherichia coli; BMC Genomics;  citation_year=2007;  citation_volume=8;  citation_pages=48" /><meta name="citation_reference" content="citation_title=Suboperonic regulatory signals; citation_text=, 1–7; Sci. STKE;  citation_year=2003;  citation_volume=185;  citation_pages=pe22" /><meta name="citation_reference" content="citation_title=The life-cycle of operons; PLoS Genet.;  citation_year=2006;  citation_volume=2;  citation_pages=859-73; " /><meta name="citation_reference" content="citation_title=Large-scale analysis of gene clustering in bacteria; Genome Res.;  citation_year=2008;  citation_volume=15;  citation_pages=949-56; " /><meta name="citation_reference" content="citation_title=DOOR: a database for prokaryotic operons; Nucl. Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=459-63; " /><meta name="citation_reference" content="citation_title=Operon prediction for sequenced bacterial genomes without experimental information; Appl. Environ. Microbiol.;  citation_year=2007;  citation_volume=73;  citation_pages=846-54; " /><meta name="citation_reference" content="citation_title=Prediction of prokaryotic promoters based on prediction of transcriptional units; Acta Biochimica et Biophysica Sinica;  citation_year=2003;  citation_volume=35;  citation_pages=317-24; " /><meta name="citation_reference" content="citation_title=A powerful nonhomology method for the prediction of operons in prokaryotes; Bioinformatics;  citation_year=2002;  citation_volume=18;  citation_pages=329-36; " /><meta name="citation_reference" content="citation_title=The relative value of operon predictions; Brief Bioinform.;  citation_year=2008;  citation_volume=9;  citation_pages=367-75; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/18/6/413/1289401/dsr028.pdf" /><meta name="description" content="Abstract.  Mycoplasma hyopneumoniae is associated with swine respiratory diseases. Although gene organization and regulation are well known in many prokaryotic " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/18/6/413/476838" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/18/6/413/476838" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/18/6/413/476838");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsr028';
    </script>
<title>
	Mycoplasma hyopneumoniae Transcription Unit Organization: Genome Survey and Prediction | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./476838" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BY5UPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTaXF1ZWlyYSwgRnJhbmNpZWxlIE1hYm9uaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlByb2dyYW1hIGRlIFDDs3MtR3JhZHVhw6fDo28gZW0gQmlvbG9naWEgTW9sZWN1bGFyIGUgQ2VsdWxhciwgRGVwYXJ0YW1lbnRvIGRlIEJpb2xvZ2lhIE1vbGVjdWxhciBlIEJpb3RlY25vbG9naWEsIENlbnRybyBkZSBCaW90ZWNub2xvZ2lhLCBVbml2ZXJzaWRhZGUgRmVkZXJhbCBkbyBSaW8gR3JhbmRlIGRvIFN1bCwgUG9ydG8gQWxlZ3JlLCBCcmF6aWwiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTY2hyYW5rLCBBdWd1c3RvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUHJvZ3JhbWEgZGUgUMOzcy1HcmFkdWHDp8OjbyBlbSBCaW9sb2dpYSBNb2xlY3VsYXIgZSBDZWx1bGFyLCBEZXBhcnRhbWVudG8gZGUgQmlvbG9naWEgTW9sZWN1bGFyIGUgQmlvdGVjbm9sb2dpYSwgQ2VudHJvIGRlIEJpb3RlY25vbG9naWEsIFVuaXZlcnNpZGFkZSBGZWRlcmFsIGRvIFJpbyBHcmFuZGUgZG8gU3VsLCBQb3J0byBBbGVncmUsIEJyYXppbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNjaHJhbmssIElyZW5lIFNpbHZlaXJhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUHJvZ3JhbWEgZGUgUMOzcy1HcmFkdWHDp8OjbyBlbSBCaW9sb2dpYSBNb2xlY3VsYXIgZSBDZWx1bGFyLCBEZXBhcnRhbWVudG8gZGUgQmlvbG9naWEgTW9sZWN1bGFyIGUgQmlvdGVjbm9sb2dpYSwgQ2VudHJvIGRlIEJpb3RlY25vbG9naWEsIFVuaXZlcnNpZGFkZSBGZWRlcmFsIGRvIFJpbyBHcmFuZGUgZG8gU3VsLCBQb3J0byBBbGVncmUsIEJyYXppbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iTXljb3BsYXNtYSBoeW9wbmV1bW9uaWFlIFRyYW5zY3JpcHRpb24gVW5pdCBPcmdhbml6YXRpb246IEdlbm9tZSBTdXJ2ZXkgYW5kIFByZWRpY3Rpb24iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI0MTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjQyMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzcjAyOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDExLzEyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXljb3BsYXNtYWwgZGlzZWFzZTsgY2l0YXRpb25fcHVibGlzaGVyPUlvd2EgU3RhdGUgVW5pdmVyc2l0eSBQcmVzcywgQW1lczsgRGlzZWFzZXMgb2YgU3dpbmU7ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl9wYWdlcz01MzctNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWUgc2VxdWVuY2Ugb2Ygc3RyYWluIDIzMiwgdGhlIGFnZW50IG9mIHN3aW5lIG15Y29wbGFzbW9zaXM7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTg2OyAgY2l0YXRpb25fcGFnZXM9NzEyMy0zMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3dpbmUgYW5kIHBvdWx0cnkgcGF0aG9nZW5zOiB0aGUgY29tcGxldGUgZ2Vub21lIHNlcXVlbmNlcyBvZiB0d28gc3RyYWlucyBvZiBNeWNvcGxhc21hIGh5b3BuZXVtb25pYWUgYW5kIGEgc3RyYWluIG9mIE15Y29wbGFzbWEgc3lub3ZpYWU7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTg3OyAgY2l0YXRpb25fcGFnZXM9NTU2OC03NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGxldGUgZ2Vub21lIHNlcXVlbmNlIG9mIE15Y29wbGFzbWEgaHlvcG5ldW1vbmlhZSBzdHJhaW4gMTY4OyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE5MzsgIGNpdGF0aW9uX3BhZ2VzPTEwMTYtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgdGhlIG51Y2xlb3RpZGUgc2VxdWVuY2Ugb2YgdGhlIFAxIG9wZXJvbiBvZiBNeWNvcGxhc21hIHBuZXVtb25pYWU7IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTE5ODg7ICBjaXRhdGlvbl92b2x1bWU9NzM7ICBjaXRhdGlvbl9wYWdlcz0xNzUtODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIGFuYWx5c2lzIG9mIHRoZSBnZW5vbWVzIG9mIHRoZSBiYWN0ZXJpYSBNeWNvcGxhc21hIHBuZXVtb25pYWUgYW5kIE15Y29wbGFzbWEgZ2VuaXRhbGl1bTsgTnVjbC4gQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTcwMS0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdGlvbmFsIGFuYWx5c2lzIG9mIHRoZSBobXcgZ2VuZSBjbHVzdGVyIG9mIE15Y29wbGFzbWEgcG5ldW1vbmlhZTsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0xODE7ICBjaXRhdGlvbl9wYWdlcz00OTc4LTg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0aW9uYWwgc3RhcnRzIGZvciBjeXRhZGhlcmVuY2UtcmVsYXRlZCBvcGVyb25zIG9mIE15Y29wbGFzbWEgZ2VuaXRhbGl1bTsgRkVNUyBNaWNyb2Jpb2wuIExldHQuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTIyOTsgIGNpdGF0aW9uX3BhZ2VzPTczLTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbiB2aXZvIGV4cHJlc3Npb24gYW5hbHlzaXMgb2YgdGhlIFA5NyBhbmQgUDEwMiBwYXJhbG9nIGZhbWlsaWVzIG9mIE15Y29wbGFzbWEgaHlvcG5ldW1vbmlhZTsgSW5mZWN0LiBJbW11bi47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NzM7ICBjaXRhdGlvbl9wYWdlcz03Nzg0LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRpb25hbCBhbmFseXNpcyBvZiB0aGUgY29uc2VydmVkIGZ0c1ogZ2VuZSBjbHVzdGVyIGluIE15Y29wbGFzbWEgZ2VuaXRhbGl1bSBhbmQgTXljb3BsYXNtYSBwbmV1bW9uaWFlOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE4NzsgIGNpdGF0aW9uX3BhZ2VzPTQ1NDItNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN5bnRoZXNpcywgc3RhYmlsaXR5LCBhbmQgZnVuY3Rpb24gb2YgY3l0YWRoZXNpbiBQMSBhbmQgYWNjZXNzb3J5IHByb3RlaW4gQi9DIGNvbXBsZXggb2YgTXljb3BsYXNtYSBwbmV1bW9uaWFlOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE4ODsgIGNpdGF0aW9uX3BhZ2VzPTU2OS03NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdG9tZSBjb21wbGV4aXR5IGluIGEgZ2Vub21lLXJlZHVjZWQgYmFjdGVyaXVtOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTMyNjsgIGNpdGF0aW9uX3BhZ2VzPTEyNjgtNzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBndWFuaW5lIGFuZCBjeXRvc2luZSBjb250ZW50IG9mIGdlbm9taWMgRE5BIGFuZCBiYWN0ZXJpYWwgZXZvbHV0aW9uOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT04NDsgIGNpdGF0aW9uX3BhZ2VzPTE2Ni05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0aW9uIGluIE15Y29wbGFzbWEgcG5ldW1vbmlhZTsgTnVjbC4gQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTQ0ODgtOTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBhbmQgdXNlIG9mIGRlcml2YXRpdmVzIG9mIHRyYW5zcG9zb24gVG40MDAxIHRoYXQgZnVuY3Rpb24gaW4gTXljb3BsYXNtYSBwdWxtb25pcyBhbmQgTXljb3BsYXNtYSBhcnRocml0aWRpczsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xODI7ICBjaXRhdGlvbl9wYWdlcz00MzQzLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb2JpbmcgaW4gdml2byBwcm9tb3RlciBhY3Rpdml0aWVzIGluIE15Y29wbGFzbWEgcG5ldW1vbmlhZTogYSBzeXN0ZW0gZm9yIGdlbmVyYXRpb24gb2Ygc2luZ2xlLWNvcHkgcmVwb3J0ZXIgY29uc3RydWN0czsgQXBwbC4gRW52aXJvbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT03MjsgIGNpdGF0aW9uX3BhZ2VzPTE2OTYtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VmVyc2F0aWxlIHVzZSBvZiBvcmlDIHBsYXNtaWRzIGZvciBmdW5jdGlvbmFsIGdlbm9taWNzIG9mIE15Y29wbGFzbWEgY2Fwcmljb2x1bSBzdWJzcC4gY2Fwcmljb2x1bTsgQXBwbC4gRW52aXJvbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT03MTsgIGNpdGF0aW9uX3BhZ2VzPTI4ODgtOTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldGVjdGlvbiBhbmQgUXVhbnRpZmljYXRpb24gb2YgaW50ZXJnZW5pYyB0cmFuc2NyaXB0aW9uIGluIE15Y29wbGFzbWEgaHlvcG5ldW1vbmlhZTsgTWljcm9iaW9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE1NjsgIGNpdGF0aW9uX3BhZ2VzPTIzMDUtMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIGNvbXBsZXRlIGdlbm9tZXMgc3VnZ2VzdHMgdGhhdCBtYW55IHByb2thcnlvdGVzIGRvIG5vdCByZWx5IG9uIGhhaXJwaW4gZm9ybWF0aW9uIGluIHRyYW5zY3JpcHRpb24gdGVybWluYXRpb247IE51Y2wuIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz01NDU2LTYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVkaWN0aW9uIG9mIHRyYW5zY3JpcHRpb25hbCB0ZXJtaW5hdG9ycyBpbiBCYWNpbGx1cyBzdWJ0aWxpcyBhbmQgcmVsYXRlZCBzcGVjaWVzOyBQTG9TIENvbXB1dC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTIxMi0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVndWxhdGlvbiBvZiBnZW5lIGV4cHJlc3Npb24gaW4gbXljb3BsYXNtYXM6IGNvbnRyaWJ1dGlvbiBmcm9tIE15Y29wbGFzbWEgaHlvcG5ldW1vbmlhZSBhbmQgTXljb3BsYXNtYSBzeW5vdmlhZSBnZW5vbWUgc2VxdWVuY2VzOyBHZW5ldC4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTI3Ny04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXJ0ZW1pczogc2VxdWVuY2UgdmlzdWFsaXphdGlvbiBhbmQgYW5ub3RhdGlvbjsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz05NDQtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXJ0ZW1pcyBhbmQgQUNUOiB2aWV3aW5nLCBhbm5vdGF0aW5nIGFuZCBjb21wYXJpbmcgc2VxdWVuY2VzIHN0b3JlZCBpbiBhIHJlbGF0aW9uYWwgZGF0YWJhc2U7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MjY3Mi02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Tb21lIHJlY29tbWVuZGF0aW9ucyBjb25jZXJuaW5nIHByaW1hcnkgaXNvbGF0aW9uIG9mIE15Y29wbGFzbWEgc3VpcG5ldW1vbmlhZSBhbmQgTXljb3BsYXNtYSBmbG9jY3VsYXJlIGEgc3VydmV5OyBOb3JkaXNrIFZldGVyaW5hZXIuIE1lZGljaW4uOyAgY2l0YXRpb25feWVhcj0xOTc1OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MzM3LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1Db2xkIFNwcmluZyBIYXJib3IgTGFib3JhdG9yeSBQcmVzcywgTmV3IFlvcms7IE1vbGVjdWxhciBDbG9uaW5nIGEgTGFib3JhdG9yeSBNYW51YWw7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbm5lY3RlZCBnZW5lIG5laWdoYm9yaG9vZHMgaW4gcHJva2FyeW90aWMgZ2Vub21lczsgTnVjbC4gQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTIyMTItMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9wZXJvbiBwcmVkaWN0aW9uIGJ5IGNvbXBhcmF0aXZlIGdlbm9taWNzOiBhbiBhcHBsaWNhdGlvbiB0byB0aGUgU3luZWNob2NvY2N1cyBzcC4gV0g4MTAyIGdlbm9tZTsgTnVjbC4gQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPTIxNDctNTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbm92ZWwgbWV0aG9kIGZvciBhY2N1cmF0ZSBvcGVyb24gcHJlZGljdGlvbnMgaW4gYWxsIHNlcXVlbmNlZCBwcm9rYXJ5b3RlczsgTnVjbC4gQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTg4MC05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBwcmVkaWN0ZWQgb3Blcm9uIG1hcCBmb3IgbXljb2JhY3Rlcml1bSB0dWJlcmN1bG9zaXM7IE51Y2wuIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz01MDg1LTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgdHJhbnNjcmlwdGlvbiB1bml0IGFyY2hpdGVjdHVyZSBvZiB0aGUgRXNjaGVyaWNoaWEgY29saSBnZW5vbWU7IE5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0xMDQzLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgcmVsYXRpb25zaGlwcyBiZXR3ZWVuIHRyYW5zY3JpcHRpb25hbCB1bml0cyBhbmQgb3Blcm9uIHN0cnVjdHVyZXMgaW4gQmFjaWxsdXMgc3VidGlsaXMgYW5kIEVzY2hlcmljaGlhIGNvbGk7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9NDgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdWJvcGVyb25pYyByZWd1bGF0b3J5IHNpZ25hbHM7IGNpdGF0aW9uX3RleHQ9LCAx4oCTNzsgU2NpLiBTVEtFOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE4NTsgIGNpdGF0aW9uX3BhZ2VzPXBlMjIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbGlmZS1jeWNsZSBvZiBvcGVyb25zOyBQTG9TIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9ODU5LTczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MYXJnZS1zY2FsZSBhbmFseXNpcyBvZiBnZW5lIGNsdXN0ZXJpbmcgaW4gYmFjdGVyaWE7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9OTQ5LTU2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ET09SOiBhIGRhdGFiYXNlIGZvciBwcm9rYXJ5b3RpYyBvcGVyb25zOyBOdWNsLiBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9NDU5LTYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcGVyb24gcHJlZGljdGlvbiBmb3Igc2VxdWVuY2VkIGJhY3RlcmlhbCBnZW5vbWVzIHdpdGhvdXQgZXhwZXJpbWVudGFsIGluZm9ybWF0aW9uOyBBcHBsLiBFbnZpcm9uLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTczOyAgY2l0YXRpb25fcGFnZXM9ODQ2LTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVkaWN0aW9uIG9mIHByb2thcnlvdGljIHByb21vdGVycyBiYXNlZCBvbiBwcmVkaWN0aW9uIG9mIHRyYW5zY3JpcHRpb25hbCB1bml0czsgQWN0YSBCaW9jaGltaWNhIGV0IEJpb3BoeXNpY2EgU2luaWNhOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9MzE3LTI0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHBvd2VyZnVsIG5vbmhvbW9sb2d5IG1ldGhvZCBmb3IgdGhlIHByZWRpY3Rpb24gb2Ygb3Blcm9ucyBpbiBwcm9rYXJ5b3RlczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0zMjktMzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSByZWxhdGl2ZSB2YWx1ZSBvZiBvcGVyb24gcHJlZGljdGlvbnM7IEJyaWVmIEJpb2luZm9ybS47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTM2Ny03NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE4LzYvNDEzLzEyODk0MDEvZHNyMDI4LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBNeWNvcGxhc21hIGh5b3BuZXVtb25pYWUgaXMgYXNzb2NpYXRlZCB3aXRoIHN3aW5lIHJlc3BpcmF0b3J5IGRpc2Vhc2VzLiBBbHRob3VnaCBnZW5lIG9yZ2FuaXphdGlvbiBhbmQgcmVndWxhdGlvbiBhcmUgd2VsbCBrbm93biBpbiBtYW55IHByb2thcnlvdGljICIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDc2ODM4JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZGhnxEUMWZCgZ945ds9aq+M+mst7" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f6%2f413%2f476838" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f6%2f413%2f476838" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=18719" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/18/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/6/1/m_cover.gif?Expires=2147483647&amp;Signature=qL9inMhepM2zFUhwbU8mgSKjtZVxpg~PmNu8fbxg-9GXAGTJaQrhv6qnZRDeMk7qu~VNHreHxR8Oz-VBGUn7klik4EpJ16UF40231HSss3FWSUANYRXx6t042t9kZJxg5tVYoWTO8VtSK1XYiJ1~Hb4XzfVloDJNBsISl1abmgO6rLxcEn6ZsFnsGHwCs2gNKgd6or0KznZwXt8UzZ8lxgCsA3E4dHjQXfOwyJhuIWr89hghDvrxBGqblSwMEBtNAQaF~zOz6WymUDfO7ME02xpTq6iZ5QfAMCaAMZasP1hnDFuY5z9kxUJ-TfMyLTyjyLEQfP4lam1wVdxuX5tJEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/18/6">
                        <div class="volume trailing-comma">Volume 18</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2011        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="7200191">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7200191">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7200193">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7200193">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7200199">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7200199">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7200212">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7200212">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7200235">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7200235">4. Conclusions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7200241">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7200241">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7200243">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7200243">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/18/6/423/477017">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main"><em>Mycoplasma hyopneumoniae</em> Transcription Unit Organization: Genome Survey and Prediction
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Franciele Maboni Siqueira</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Programa de Pós-Graduação em Biologia Molecular e Celular, Departamento de Biologia Molecular e Biotecnologia, Centro de Biotecnologia, Universidade Federal do Rio Grande do Sul, Porto Alegre</div>, <div class="country">Brazil</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Franciele+Maboni+Siqueira">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Siqueira F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Siqueira F&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Franciele Maboni Siqueira</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Augusto Schrank</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Programa de Pós-Graduação em Biologia Molecular e Celular, Departamento de Biologia Molecular e Biotecnologia, Centro de Biotecnologia, Universidade Federal do Rio Grande do Sul, Porto Alegre</div>, <div class="country">Brazil</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Augusto+Schrank">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Schrank A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Schrank A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Augusto Schrank</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Irene Silveira Schrank</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Programa de Pós-Graduação em Biologia Molecular e Celular, Departamento de Biologia Molecular e Biotecnologia, Centro de Biotecnologia, Universidade Federal do Rio Grande do Sul, Porto Alegre</div>, <div class="country">Brazil</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +55 51-3308-6055. Fax. +55 51-3308-7309. Email: <a href="mailto:irene@cbiot.ufrgs.br" target="_blank">irene@cbiot.ufrgs.br</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Irene+Silveira+Schrank">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Schrank I">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Schrank I&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Irene Silveira Schrank</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Naotake Ogasawara</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 18, Issue 6, 1 December 2011, Pages 413–422, <a href='https://doi.org/10.1093/dnares/dsr028'>https://doi.org/10.1093/dnares/dsr028</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">14 November 2011</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">23 March 2011</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">24 July 2011</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="476838" href="/dnaresearch/article-pdf/18/6/413/1289401/dsr028.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Franciele Maboni Siqueira, Augusto Schrank, Irene Silveira Schrank;  <em>Mycoplasma hyopneumoniae</em> Transcription Unit Organization: Genome Survey and Prediction, <em>DNA Research</em>, Volume 18, Issue 6, 1 December 2011, Pages 413–422, <a href='https://doi.org/10.1093/dnares/dsr028'>https://doi.org/10.1093/dnares/dsr028</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/476838?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/476838?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/476838?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/476838?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/476838?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/476838?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/476838?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=%26lt%3Bem%26gt%3BMycoplasma%20hyopneumoniae%26lt%3B%2Fem%26gt%3B%20Transcription%20Unit%20Organization%3A%20Genome%20Survey%20and%20Prediction&amp;publicationDate=2011-11-14&amp;volumeNum=18&amp;issueNum=6&amp;author=Siqueira%2C%20Franciele%20Maboni%3B%20Schrank%2C%20Augusto&amp;startPage=413&amp;endPage=422&amp;contentId=10.1093%2Fdnares%2Fdsr028&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f6%2f413%2f476838" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=7200191 id="7200191" class="abstract-title" >Abstract</h2>
<section class="abstract"><p><em>Mycoplasma hyopneumoniae</em> is associated with swine respiratory diseases. Although gene organization and regulation are well known in many prokaryotic organisms, knowledge on mycoplasma is limited. This study performed a comparative analysis of three strains of <em>M. hyopneumoniae</em> (7448, J and 232), with a focus on genome organization and gene comparison for open read frame (ORF) cluster (OC) identification. An <em>in silico</em> analysis of gene organization demonstrated 117 OCs and 34 single ORFs in <em>M. hyopneumoniae</em> 7448 and J, while 116 OCs and 36 single ORFs were identified in <em>M. hyopneumoniae</em> 232. Genomic comparison revealed high synteny and conservation of gene order between the OCs defined for 7448 and J strains as well as for 7448 and 232 strains. Twenty-one OCs were chosen and experimentally confirmed by reverse transcription–PCR from <em>M. hyopneumoniae</em> 7448 genome, validating our prediction. A subset of the ORFs within an OC could be independently transcribed due to the presence of internal promoters. Our results suggest that transcription occurs in ‘run-on’ from an upstream promoter in <em>M. hyopneumoniae</em>, thus forming large ORF clusters (from 2 to 29 ORFs in the same orientation) and indicating a complex transcriptional organization.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;ORF cluster&quot;">ORF cluster</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;intergenic regions&quot;">intergenic regions</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="cotranscription">cotranscription</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;transcriptional units&quot;">transcriptional units</a></div>                            <h2 scrollto-destination=7200193 id="7200193" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p><em>Mycoplasma hyopneumoniae</em> is the key aetiological agent of porcine enzootic pneumonia (EP) and is a major contributor to porcine respiratory disease complex.<sup><a reveal-id="DSR028C1" data-open="DSR028C1" class="link link-ref link-reveal xref-bibr">1</a></sup> EP is a chronic respiratory disease that mainly affects finishing pigs. Although major efforts to control <em>M. hyopneumoniae</em> infection and its detrimental effects have been made, swine production has suffered from significant economic loss.</p><p>Like other mycoplasmas, <em>M. hyopneumoniae</em> has a small genome with limited biosynthetic potential.<sup><a reveal-id="DSR028C2  DSR028C4" data-open="DSR028C2  DSR028C4" class="link link-ref link-reveal xref-bibr">2–4</a></sup> Up until now, four <em>M. hyopneumoniae</em> genome strains have been sequenced.<sup><a reveal-id="DSR028C2  DSR028C4" data-open="DSR028C2  DSR028C4" class="link link-ref link-reveal xref-bibr">2–4</a></sup> Despite the significant amount of data produced by genome sequencing, information on its open read frame (ORF) cluster (OC) organization, transcriptional unit (TU) formation and transcriptional regulation is very limited. Genes organized in an ORF cluster (OC) or TU are arranged in tandem in the genomic sequence, being delimited by the location of the upstream promoter and the downstream transcriptional terminator. TU identification is commonly based on bioinformatics approaches combined with experimental data. Mycoplasma studies have demonstrated the occurrence of polycistronic mRNA and TUs in some genomic regions.<sup><a reveal-id="DSR028C5  DSR028C12" data-open="DSR028C5  DSR028C12" class="link link-ref link-reveal xref-bibr">5–12</a></sup> However, information on characterized TUs is available for exceptional cases, making it difficult to judge the accuracy of current <em>in silico</em> TU or OC prediction methods in mycoplasma genome sequences.</p><p>Prediction and recognition of mycoplasma promoter elements are poorly developed. The unusual A + T content of intergenic regions (IRs) in mycoplasma<sup><a reveal-id="DSR028C13" data-open="DSR028C13" class="link link-ref link-reveal xref-bibr">13</a></sup> and the weak −35 consensus<sup><a reveal-id="DSR028C14" data-open="DSR028C14" class="link link-ref link-reveal xref-bibr">14</a></sup> prevent efficient prediction of promoters by current bioinformatics approaches. Moreover, in spite of the development of artificial transformation and other genetic tools for some species of <em>Mycoplasma</em>,<sup><a reveal-id="DSR028C14  DSR028C17" data-open="DSR028C14  DSR028C17" class="link link-ref link-reveal xref-bibr">14–17</a></sup> these methodologies are yet to be developed for <em>M. hyopneumoniae</em>. Therefore, strategies for the mapping of genome organization and gene transcription in <em>M. hyopneumoniae</em> can help understand regulatory mechanisms and form the basis for future work on mapping promoters and terminators.</p><p>Information regarding transcription promoter or terminator sequences have yet to be defined in <em>M. hyopneumoniae</em>. Therefore, any analyses at the level of transcription regulation are difficult. Recently, Gardner and Minion<sup><a reveal-id="DSR028C18" data-open="DSR028C18" class="link link-ref link-reveal xref-bibr">18</a></sup> demonstrated that transcription occurs in IRs of <em>M. hyopneumoniae</em>, indicating that <em>M. hyopneumoniae</em> does not strictly control transcription termination. An analysis of the average RNA folding energy near stop codons suggested that no stem-loops are formed in <em>Mycoplasma genitalium</em> and <em>Mycoplasma pneumoniae</em> TUs, indicating the existence of qualitatively different and uncharacterized mechanisms for transcription termination.<sup><a reveal-id="DSR028C19" data-open="DSR028C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Defining a transcription termination region in mycoplasmas has revealed some contradictory results. In some operons of <em>M. genitalium</em>,<sup><a reveal-id="DSR028C8" data-open="DSR028C8" class="link link-ref link-reveal xref-bibr">8</a></sup> the presence of stem-loop structures have been found, and results from Hoon <em>et al</em>.<sup><a reveal-id="DSR028C20" data-open="DSR028C20" class="link link-ref link-reveal xref-bibr">20</a></sup> suggest that the Rho-independent terminator represents the main mode of transcriptional termination in mycoplasmas. However, it has been suggested that genes are arranged in long clusters in mycoplasmas, indicating the possibility that the starting point of transcription occurs in the first gene and stops at low frequencies due to the absence of specific regions for transcription termination.<sup><a reveal-id="DSR028C21" data-open="DSR028C21" class="link link-ref link-reveal xref-bibr">21</a></sup></p><p>Although gene co-regulation occurs in mycoplasmas, to date only little information is available on ORF organization in the <em>M. hyopneumoniae</em> genome, as well as its relationship with pathogenicity. In order to establish differences between the gene organization of pathogenic and non-pathogenic <em>M. hyopneumoniae</em> and also to relate this gene organization to transcription regulatory mechanisms, we systematically analysed the synteny and conservation of ORF order among three <em>M. hyopneumoniae</em> genomes (two pathogenic strains: 7448 and 232; one non-pathogenic strain: J). We then performed the experimental characterization of some OCs predicted in the <em>M. hyopneumoniae</em> 7448 strain. This is the first report where gene organization analysis was systematically performed and compared in three <em>M. hyopneumoniae</em> genomes, two of which are pathogenic to swines and a significant number of predicted OCs are experimentally demonstrated and named TUs. These results could contribute to promote the understanding of <em>M. hyopneumoniae</em> transcription regulation.</p>                            <h2 scrollto-destination=7200199 id="7200199" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=7200200 id="7200200" class="section-title" data-legacy-id=s2a>2.1. <em>In silico</em> analysis of TUs</h3>
<p>OC prediction was performed by Artemis Release 10.5.2 software.<sup><a reveal-id="DSR028C22" data-open="DSR028C22" class="link link-ref link-reveal xref-bibr">22</a></sup> The criteria for the manual examination of possible OCs in <em>M. hyopneumoniae</em> genomes (MHP7448_GenBank: NC_007332; MHP232_GenBank: NC_006360 and MHPJ_GenBank: NC_007295) were established on the occurrence of clusters with two or more genes in tandem in the same DNA strand. This was performed by systematic annotation comparison of the protein sequences encoded in all ORFs from the analysed genomes. According to the complexity of adjacent ORF rearrangements, two groups were created: OCs characterized by the presence of two or more ORFs in the same DNA strand until the occurrence of ORFs in the opposite strand, and monocistronic (mC) group representing single ORFs. Differences in the annotation among the genomes were evaluated by comparing protein sequences using the NCBI/BLASTP program.</p><p>The comparative analysis of OC organization found among the three <em>M. hyopneumoniae</em> genomes (7448, 232 and J) was accomplished based on the <em>M. hyopneumoniae</em> 7448 genome results by Artemis Comparison tools Release 9.0.<sup><a reveal-id="DSR028C23" data-open="DSR028C23" class="link link-ref link-reveal xref-bibr">23</a></sup></p>                            <h3 scrollto-destination=7200203 id="7200203" class="section-title" data-legacy-id=s2b>2.2. Primer design</h3>
<p>The primers were designed in Vector NTI Advance 10 (Invitrogen) based on the <em>M. hyopneumoniae</em> 7448 genome sequence (GenBank: NC_007332). Primer pairs were chosen to enable synthesis by reverse transcriptase–PCR (RT–PCR), whose products span each IR, demonstrating the occurrence of polycistronic mRNA and confirming the prediction previously made. The OC<sub>7448</sub>111 (Fig. <a reveal-id="DSR028F1" data-open="DSR028F1" class="link link-reveal link-table xref-fig">1</a>) was used as an example where three primer pairs were designed for utilization in the RT–PCR analysis. As shown in Fig. <a reveal-id="DSR028F1" data-open="DSR028F1" class="link link-reveal link-table xref-fig">1</a>, primer pairs were distributed in the predicted OC and used in the following combination in the RT–PCRs: 1°F/1°R; 2°F/2°R; 3°F/3°R; 1°F/2°R and 2°F/3°R. To generate experimental data, a total of 71 primer pairs were projected and employed with this methodology (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). </p>                        <a id="7200205" scrollto-destination="7200205"></a>
<div id="DSR028F1" data-id="DSR028F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr028/2/m_dsr02801.jpeg?Expires=2147483647&Signature=PpCeNg-Neb1yFdi6TBfMMXxF~tz9~7QQSr4gW7sJFMbQ6nzvUr~o4RasWrvtDKydvHBVLTuAlVDOfypad-eqrK2PuUhxsy-idN35sRq4x6s6xGfnO~S1MDiIGgGrKp83q7Y4e62v6R8PTTVeWeqmiu4U4~31Z1Ye62M96D4mVfoOkS7caG92J-EW9OgYW81D6q9wwBshettQdkMholl-gpF0uvgS2oizDNsMt2dtSA-AKZECtiUwoicxpiEcdp9-F9gbIcomu13kpDMOXLJF51rVE4zFT1sHcgqga4Py08sCK7nN0fPOSliPNe5T3fansJUdfqP3qp2-fWfFRf811A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="OCs in M. hyopneumoniae. A series of RT–PCRs using ORF-specific primers are designed to produce the products shown at the centre of the figure (black lines with the product size in bp). The black arrows 1°F/1°R, 2°F/2°R and 3°F/3°R correspond to primers 111 1°F, 111 1°R, 111 2°F, 111 2°R, 111 3°F and 111 3°R, respectively, and represent a schematic localization of primers pairs in OC7448111 designed to span the IRs between ORFs. The ORFs and tRNA30 nomenclature is in agreement with the GenBank NC_007332 and are represent by grey arrows." path-from-xml="dsr02801.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7200205/dsr02801.jpeg" path-from-xml="dsr02801.jpeg" target="_blank">View large</a><a section="7200205" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr028/2/dsr02801.jpeg?Expires=2147483647&Signature=3DtWCurX10qpRFxWifMJnh873S428IZVfmLPhDAkBJvo~I0mVYynAyJYSWWqOAdEeSnhg0wicfdQe76POHDzVQTSRGyIx7~Nx36t-nS1mzsexTQishAU8TOfEBquNjQXVnWAiNHV-vzg4-FK8UFrJXtnptEHXirimFH092Aqv-h77To1nC18oo7XSOZtvTTy9MBMX3EU~udBjn0cAw6DJLW716ZFtHxGPg-G9C9s7tPZ6WO~niHAd3BEbGV9vcXXQnr-zXLwopUCWyMbOwXgLqPu7gRyoJ1XSojIBjEQCxV0ceVn-x5XenbV3MoPdJ9LlqlJ5DFyWBZo0a55RY7lPw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7200205&ar=476838&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02801.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>OCs in <em>M. hyopneumoniae</em>. A series of RT–PCRs using ORF-specific primers are designed to produce the products shown at the centre of the figure (black lines with the product size in bp). The black arrows 1°F/1°R, 2°F/2°R and 3°F/3°R correspond to primers 111 1°F, 111 1°R, 111 2°F, 111 2°R, 111 3°F and 111 3°R, respectively, and represent a schematic localization of primers pairs in OC<sub>7448</sub>111 designed to span the IRs between ORFs. The ORFs and tRNA30 nomenclature is in agreement with the GenBank NC_007332 and are represent by grey arrows.</p></div></div><div content-id="DSR028F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr028/2/m_dsr02801.jpeg?Expires=2147483647&Signature=PpCeNg-Neb1yFdi6TBfMMXxF~tz9~7QQSr4gW7sJFMbQ6nzvUr~o4RasWrvtDKydvHBVLTuAlVDOfypad-eqrK2PuUhxsy-idN35sRq4x6s6xGfnO~S1MDiIGgGrKp83q7Y4e62v6R8PTTVeWeqmiu4U4~31Z1Ye62M96D4mVfoOkS7caG92J-EW9OgYW81D6q9wwBshettQdkMholl-gpF0uvgS2oizDNsMt2dtSA-AKZECtiUwoicxpiEcdp9-F9gbIcomu13kpDMOXLJF51rVE4zFT1sHcgqga4Py08sCK7nN0fPOSliPNe5T3fansJUdfqP3qp2-fWfFRf811A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="OCs in M. hyopneumoniae. A series of RT–PCRs using ORF-specific primers are designed to produce the products shown at the centre of the figure (black lines with the product size in bp). The black arrows 1°F/1°R, 2°F/2°R and 3°F/3°R correspond to primers 111 1°F, 111 1°R, 111 2°F, 111 2°R, 111 3°F and 111 3°R, respectively, and represent a schematic localization of primers pairs in OC7448111 designed to span the IRs between ORFs. The ORFs and tRNA30 nomenclature is in agreement with the GenBank NC_007332 and are represent by grey arrows." path-from-xml="dsr02801.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7200205/dsr02801.jpeg" path-from-xml="dsr02801.jpeg" target="_blank">View large</a><a section="7200205" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr028/2/dsr02801.jpeg?Expires=2147483647&Signature=3DtWCurX10qpRFxWifMJnh873S428IZVfmLPhDAkBJvo~I0mVYynAyJYSWWqOAdEeSnhg0wicfdQe76POHDzVQTSRGyIx7~Nx36t-nS1mzsexTQishAU8TOfEBquNjQXVnWAiNHV-vzg4-FK8UFrJXtnptEHXirimFH092Aqv-h77To1nC18oo7XSOZtvTTy9MBMX3EU~udBjn0cAw6DJLW716ZFtHxGPg-G9C9s7tPZ6WO~niHAd3BEbGV9vcXXQnr-zXLwopUCWyMbOwXgLqPu7gRyoJ1XSojIBjEQCxV0ceVn-x5XenbV3MoPdJ9LlqlJ5DFyWBZo0a55RY7lPw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7200205&ar=476838&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02801.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>OCs in <em>M. hyopneumoniae</em>. A series of RT–PCRs using ORF-specific primers are designed to produce the products shown at the centre of the figure (black lines with the product size in bp). The black arrows 1°F/1°R, 2°F/2°R and 3°F/3°R correspond to primers 111 1°F, 111 1°R, 111 2°F, 111 2°R, 111 3°F and 111 3°R, respectively, and represent a schematic localization of primers pairs in OC<sub>7448</sub>111 designed to span the IRs between ORFs. The ORFs and tRNA30 nomenclature is in agreement with the GenBank NC_007332 and are represent by grey arrows.</p></div></div>                            <h3 scrollto-destination=7200206 id="7200206" class="section-title" data-legacy-id=s2c>2.3. DNA and RNA manipulations</h3>
<p><em>Mycoplasma hyopneumoniae</em> strain 7448 was grown in 5 and 25 ml of Friis broth<sup><a reveal-id="DSR028C24" data-open="DSR028C24" class="link link-ref link-reveal xref-bibr">24</a></sup> at 37°C for 48 h for DNA extraction and RNA isolation, respectively. Genomic DNA of <em>M. hyopneumoniae</em> was isolated using standard protocols.<sup><a reveal-id="DSR028C25" data-open="DSR028C25" class="link link-ref link-reveal xref-bibr">25</a></sup> The quality of the isolated DNA was verified by gel electrophoresis and displayed pure high-molecular-weight DNA.</p><p>Total RNA of <em>M. hyopneumoniae</em> extraction was performed with Trizol<sup>®</sup> (Invitrogen) following the manufacturer's guidelines, including 1 U/μg DNase/RNAse free (Promega) treatment, and then the extracted RNA was quantified in Qubit<sup>™</sup> system (Invitrogen) and also analysed by gel electrophoresis. The RNA extraction was stored at −70°C.</p>                            <h3 scrollto-destination=7200209 id="7200209" class="section-title" data-legacy-id=s2d>2.4. Reverse transcriptase–PCR</h3>
<p>A first-strand cDNA synthesis reaction was conducted by adding 1 µg of total RNA to 132.5 ng of pd(N)<sub>6</sub> random hexamer (GE Healthcare) and 10 mM deoxynucleotide triphosphates. The mixture was heated to 65°C for 5 min and then incubated on ice for 1 min. First-strand buffer (250 mM Tris–HCl, pH 8.3, 375 mM KCl, 15 mM MgCl<sub>2</sub>), 0.1 M dithiothreitol, 40 U RNase inhibitor (Invitrogen) and 200 U M-MLV RT (Moloney Murine Leukemia Virus Reverse Transcriptase—Invitrogen) was then added to a total volume of 20 µl. The reaction mixture was incubated at 25°C for 10 min and at 37°C for 50 min followed by 15 min at 70°C. Negative control was prepared in parallel, differing only by the absence of the RT enzyme.</p><p>PCRs included 1 U <em>Taq</em> DNA polymerase (Cenbiot Enzimas), 10× buffer (100 mM Tris–HCl, pH 8.5, 500 mM KCl), 1 mM magnesium chloride, 100 mM deoxynucleotide triphosphates, 10 pmol of each primer (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>) and 1 µl of the first-strand cDNA reaction in a final volume of 25 µl. Negative control of RT–PCR was prepared in parallel, which differed only by the absence of cDNA, and no genomic DNA was added to the reaction mixture for the PCR control. PCR positive control was prepared using the genomic DNA of <em>M. hyopneumoniae</em> as a template. The PCR conditions were as follows: 1 cycle at 94°C for 5 min was followed by 30 cycles of 94°C for 30 s; denaturation and extension temperature and time varied according to each primer pair (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The final extension step was at 72°C for 10 min. Reaction products were analysed in 1.2% agarose gels and fragments of the expected size were precipitated with tRNA (Invitrogen) as follows. To each RT–PCR (22 µl), 1 µg of tRNA and 2.5 V of absolute ethanol (4°C) were added and then incubated for 16 h at −20°C. After 30 min of centrifugation and wash with 70% ethanol, the pellets were resuspended in 15 µl of ultrapure water and sequenced with the DYEnamic ET Dye Terminator Cycle Sequencing Kit (Amersham Biosciences) in the MegaBACE 1000, as recommended by the manufacturer.</p>                            <h2 scrollto-destination=7200212 id="7200212" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=7200213 id="7200213" class="section-title" data-legacy-id=s3a>3.1. Genes in <em>M. hyopneumoniae</em> are preferably organized in OCs</h3>
<p>This study represents a global assessment of OC organization with <em>in silico</em> and <em>in vitro</em> analysis of the <em>M. hyopneumoniae</em> genome. Our detailed analysis indicated that genes are organized preferably in OCs. The gene-by-gene genome organization of the three strains of <em>M. hyopneumoniae</em> was analysed, gene localization was compared to detect ORFs with order conservation and two groups were created as described in Materials and methods: the OC group (named OC<sub>7448</sub>1–OC<sub>7448</sub>117 in the 7448 strain; OC<sub>J</sub>1–OC<sub>J</sub>117 in the J strain and OC<sub>232</sub>1–OC<sub>232</sub>116 in the 232 strain) and the mC group (named mC<sub>7448</sub>1–mC<sub>7448</sub>34 in the 7448 strain; mC<sub>J</sub>1–mC<sub>J</sub>34 in the J strain and mC<sub>232</sub>1–mC<sub>232</sub>36 in the 232 strain). In summary, 117 OCs and 34 mCs (strains 7448 and J) and 116 OCs and 36 mCs (strain 232) were detected (Table <a reveal-id="DSR028TB1" data-open="DSR028TB1" class="link link-reveal link-table xref-fig">1</a>). The mC ORFs of the three <em>M. hyopneumoniae</em> strains are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>. Surprisingly, just 5% of the ORFs in the three genomes were mC. Therefore, the majority (95%) of <em>M. hyopneumoniae</em> ORFs are transcribed in long polycistronic mRNAs, thus forming extensive OCs. Moreover, the ORF numbers that constitute each OC was very variable (Table <a reveal-id="DSR028TB2" data-open="DSR028TB2" class="link link-reveal link-table xref-fig">2</a>). As shown in Table <a reveal-id="DSR028TB2" data-open="DSR028TB2" class="link link-reveal link-table xref-fig">2</a>, the majority of the OCs were composed of two ORFs (22%), with increasing numbers of up to 29 ORFs. Interestingly, in the three genomes analysed, approximately 85% of the OCs were composed of two to eight ORFs, demonstrating the presence of OCs with fewer numbers of ORFs. </p>                        <a id="7200215" scrollto-destination="7200215"></a>
<div content-id="DSR028TB1" class="table-modal"><div class="table-wrap"><div id="DSR028TB1" class="table-wrap-title" data-id="DSR028TB1"><div class="label -label">Table 1.</div><div class="caption"><p>General feature of the OCs’ organization in <em>M.hyopneumoniae</em> genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Feature </th><th>MHP7448 </th><th>MHP J </th><th>MHP232 </th></tr></thead><tbody><tr><td>Total length (base pairs) </td><td>920 079 </td><td>897 405 </td><td>892 758 </td></tr><tr><td>Total no. protein coding ORFs </td><td>657 </td><td>657 </td><td>691 </td></tr><tr><td>Total no. of predicted OCs </td><td>117 </td><td>117 </td><td>116 </td></tr><tr><td>Total no. of mC (% compared with total ORFs) </td><td>34 (5.1%) </td><td>34 (5.1%) </td><td>36 (5.2%) </td></tr><tr><td> Valid ORFs </td><td>17 </td><td>12 </td><td>17 </td></tr><tr><td> Conserved hypothetical ORFs </td><td>8 </td><td>7 </td><td>10 </td></tr><tr><td> Hypothetical ORFs </td><td>9 </td><td>15 </td><td>9 </td></tr><tr><td>OCs organization similarity (%)<sup>a</sup> </td><td>100% </td><td>84% </td><td>71.00% </td></tr><tr><td>OCs organization similarity (%) without RIs<sup>b</sup> </td><td>100% </td><td>85.0% </td><td>82.0% </td></tr><tr><td>Location of region with inverted sequence (bp) </td><td>101 560–354 533 </td><td>Not present </td><td>104 691–344 079 </td></tr><tr><td>Transposases + ICEH-II<sup>c</sup> </td><td>9 + 1 </td><td>7 + 0 </td><td>5 + 1 </td></tr><tr><td>Validated predicted OCs (% compared with total predicted OCs) </td><td>21 (18%) </td><td>0 </td><td>0 </td></tr></tbody></table></div><div content-id="DSR028TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Feature </th><th>MHP7448 </th><th>MHP J </th><th>MHP232 </th></tr></thead><tbody><tr><td>Total length (base pairs) </td><td>920 079 </td><td>897 405 </td><td>892 758 </td></tr><tr><td>Total no. protein coding ORFs </td><td>657 </td><td>657 </td><td>691 </td></tr><tr><td>Total no. of predicted OCs </td><td>117 </td><td>117 </td><td>116 </td></tr><tr><td>Total no. of mC (% compared with total ORFs) </td><td>34 (5.1%) </td><td>34 (5.1%) </td><td>36 (5.2%) </td></tr><tr><td> Valid ORFs </td><td>17 </td><td>12 </td><td>17 </td></tr><tr><td> Conserved hypothetical ORFs </td><td>8 </td><td>7 </td><td>10 </td></tr><tr><td> Hypothetical ORFs </td><td>9 </td><td>15 </td><td>9 </td></tr><tr><td>OCs organization similarity (%)<sup>a</sup> </td><td>100% </td><td>84% </td><td>71.00% </td></tr><tr><td>OCs organization similarity (%) without RIs<sup>b</sup> </td><td>100% </td><td>85.0% </td><td>82.0% </td></tr><tr><td>Location of region with inverted sequence (bp) </td><td>101 560–354 533 </td><td>Not present </td><td>104 691–344 079 </td></tr><tr><td>Transposases + ICEH-II<sup>c</sup> </td><td>9 + 1 </td><td>7 + 0 </td><td>5 + 1 </td></tr><tr><td>Validated predicted OCs (% compared with total predicted OCs) </td><td>21 (18%) </td><td>0 </td><td>0 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Similarity evaluated in relationship with MHP7448. The percentage represents the total OCs conservation (equality ORFs and RIs).</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Similarity evaluated in relationship with MHP7448. The percentage represents the total OCs conservation leave out RIs differences.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Probably relationeted with sites of recombination in comparative analysis of OC organization between 7448 strain and 232 or J strains.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7200215" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR028TB1" class="table-wrap-title" data-id="DSR028TB1"><div class="label -label">Table 1.</div><div class="caption"><p>General feature of the OCs’ organization in <em>M.hyopneumoniae</em> genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Feature </th><th>MHP7448 </th><th>MHP J </th><th>MHP232 </th></tr></thead><tbody><tr><td>Total length (base pairs) </td><td>920 079 </td><td>897 405 </td><td>892 758 </td></tr><tr><td>Total no. protein coding ORFs </td><td>657 </td><td>657 </td><td>691 </td></tr><tr><td>Total no. of predicted OCs </td><td>117 </td><td>117 </td><td>116 </td></tr><tr><td>Total no. of mC (% compared with total ORFs) </td><td>34 (5.1%) </td><td>34 (5.1%) </td><td>36 (5.2%) </td></tr><tr><td> Valid ORFs </td><td>17 </td><td>12 </td><td>17 </td></tr><tr><td> Conserved hypothetical ORFs </td><td>8 </td><td>7 </td><td>10 </td></tr><tr><td> Hypothetical ORFs </td><td>9 </td><td>15 </td><td>9 </td></tr><tr><td>OCs organization similarity (%)<sup>a</sup> </td><td>100% </td><td>84% </td><td>71.00% </td></tr><tr><td>OCs organization similarity (%) without RIs<sup>b</sup> </td><td>100% </td><td>85.0% </td><td>82.0% </td></tr><tr><td>Location of region with inverted sequence (bp) </td><td>101 560–354 533 </td><td>Not present </td><td>104 691–344 079 </td></tr><tr><td>Transposases + ICEH-II<sup>c</sup> </td><td>9 + 1 </td><td>7 + 0 </td><td>5 + 1 </td></tr><tr><td>Validated predicted OCs (% compared with total predicted OCs) </td><td>21 (18%) </td><td>0 </td><td>0 </td></tr></tbody></table></div><div content-id="DSR028TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Feature </th><th>MHP7448 </th><th>MHP J </th><th>MHP232 </th></tr></thead><tbody><tr><td>Total length (base pairs) </td><td>920 079 </td><td>897 405 </td><td>892 758 </td></tr><tr><td>Total no. protein coding ORFs </td><td>657 </td><td>657 </td><td>691 </td></tr><tr><td>Total no. of predicted OCs </td><td>117 </td><td>117 </td><td>116 </td></tr><tr><td>Total no. of mC (% compared with total ORFs) </td><td>34 (5.1%) </td><td>34 (5.1%) </td><td>36 (5.2%) </td></tr><tr><td> Valid ORFs </td><td>17 </td><td>12 </td><td>17 </td></tr><tr><td> Conserved hypothetical ORFs </td><td>8 </td><td>7 </td><td>10 </td></tr><tr><td> Hypothetical ORFs </td><td>9 </td><td>15 </td><td>9 </td></tr><tr><td>OCs organization similarity (%)<sup>a</sup> </td><td>100% </td><td>84% </td><td>71.00% </td></tr><tr><td>OCs organization similarity (%) without RIs<sup>b</sup> </td><td>100% </td><td>85.0% </td><td>82.0% </td></tr><tr><td>Location of region with inverted sequence (bp) </td><td>101 560–354 533 </td><td>Not present </td><td>104 691–344 079 </td></tr><tr><td>Transposases + ICEH-II<sup>c</sup> </td><td>9 + 1 </td><td>7 + 0 </td><td>5 + 1 </td></tr><tr><td>Validated predicted OCs (% compared with total predicted OCs) </td><td>21 (18%) </td><td>0 </td><td>0 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Similarity evaluated in relationship with MHP7448. The percentage represents the total OCs conservation (equality ORFs and RIs).</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Similarity evaluated in relationship with MHP7448. The percentage represents the total OCs conservation leave out RIs differences.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Probably relationeted with sites of recombination in comparative analysis of OC organization between 7448 strain and 232 or J strains.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7200215" target="_blank">View Large</a></div></div>                        <a id="7200216" scrollto-destination="7200216"></a>
<div content-id="DSR028TB2" class="table-modal"><div class="table-wrap"><div id="DSR028TB2" class="table-wrap-title" data-id="DSR028TB2"><div class="label -label">Table 2.</div><div class="caption"><p>ORF composition in OCs of <em>M. hyopneumoniae</em> genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Feature </th><th>MHP7448 </th><th>MHP J </th><th>MHP232 </th></tr></thead><tbody><tr><td>Total no. of OCs </td><td>117 </td><td>117 </td><td>116 </td></tr><tr><td>OCs composed of two ORFs </td><td>26 </td><td>26 </td><td>25 </td></tr><tr><td>OCs composed of three ORFs </td><td>19 </td><td>18 </td><td>16 </td></tr><tr><td>OCs composed of four ORFs </td><td>16 </td><td>14 </td><td>16 </td></tr><tr><td>OCs composed of five ORFs </td><td>15 </td><td>16 </td><td>13 </td></tr><tr><td>OCs composed of six ORFs </td><td>9 </td><td>12 </td><td>9 </td></tr><tr><td>OCs composed of seven ORFs </td><td>11 </td><td>10 </td><td>10 </td></tr><tr><td>OCs composed of eight ORFs </td><td>5 </td><td>5 </td><td>5 </td></tr><tr><td>OCs composed of nine ORFs </td><td>2 </td><td>1 </td><td>4 </td></tr><tr><td>OCs composed of 10 ORFs </td><td>3 </td><td>4 </td><td>6 </td></tr><tr><td>OCs composed of 11 ORFs </td><td>3 </td><td>4 </td><td>2 </td></tr><tr><td>OCs composed of 12 ORFs </td><td>2 </td><td>2 </td><td>2 </td></tr><tr><td>OCs composed of 13 ORFs </td><td>1 </td><td>1 </td><td>3 </td></tr><tr><td>OCs composed of 15 ORFs </td><td>1 </td><td>1 </td><td>2 </td></tr><tr><td>OCs composed of 20 ORFs </td><td>1 </td><td>0 </td><td>0 </td></tr><tr><td>OCs composed of 21 ORFs </td><td>0 </td><td>0 </td><td>1 </td></tr><tr><td>OCs composed of 22 ORFs </td><td>1 </td><td>0 </td><td>0 </td></tr><tr><td>OCs composed of 24 ORFs </td><td>0 </td><td>1 </td><td>0 </td></tr><tr><td>OCs composed of 26 ORFs </td><td>1 </td><td>1 </td><td>0 </td></tr><tr><td>OCs composed of 27 ORFs </td><td>0 </td><td>0 </td><td>1 </td></tr><tr><td>OCs composed of 29 ORFs </td><td>1 </td><td>1 </td><td>1 </td></tr></tbody></table></div><div content-id="DSR028TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Feature </th><th>MHP7448 </th><th>MHP J </th><th>MHP232 </th></tr></thead><tbody><tr><td>Total no. of OCs </td><td>117 </td><td>117 </td><td>116 </td></tr><tr><td>OCs composed of two ORFs </td><td>26 </td><td>26 </td><td>25 </td></tr><tr><td>OCs composed of three ORFs </td><td>19 </td><td>18 </td><td>16 </td></tr><tr><td>OCs composed of four ORFs </td><td>16 </td><td>14 </td><td>16 </td></tr><tr><td>OCs composed of five ORFs </td><td>15 </td><td>16 </td><td>13 </td></tr><tr><td>OCs composed of six ORFs </td><td>9 </td><td>12 </td><td>9 </td></tr><tr><td>OCs composed of seven ORFs </td><td>11 </td><td>10 </td><td>10 </td></tr><tr><td>OCs composed of eight ORFs </td><td>5 </td><td>5 </td><td>5 </td></tr><tr><td>OCs composed of nine ORFs </td><td>2 </td><td>1 </td><td>4 </td></tr><tr><td>OCs composed of 10 ORFs </td><td>3 </td><td>4 </td><td>6 </td></tr><tr><td>OCs composed of 11 ORFs </td><td>3 </td><td>4 </td><td>2 </td></tr><tr><td>OCs composed of 12 ORFs </td><td>2 </td><td>2 </td><td>2 </td></tr><tr><td>OCs composed of 13 ORFs </td><td>1 </td><td>1 </td><td>3 </td></tr><tr><td>OCs composed of 15 ORFs </td><td>1 </td><td>1 </td><td>2 </td></tr><tr><td>OCs composed of 20 ORFs </td><td>1 </td><td>0 </td><td>0 </td></tr><tr><td>OCs composed of 21 ORFs </td><td>0 </td><td>0 </td><td>1 </td></tr><tr><td>OCs composed of 22 ORFs </td><td>1 </td><td>0 </td><td>0 </td></tr><tr><td>OCs composed of 24 ORFs </td><td>0 </td><td>1 </td><td>0 </td></tr><tr><td>OCs composed of 26 ORFs </td><td>1 </td><td>1 </td><td>0 </td></tr><tr><td>OCs composed of 27 ORFs </td><td>0 </td><td>0 </td><td>1 </td></tr><tr><td>OCs composed of 29 ORFs </td><td>1 </td><td>1 </td><td>1 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7200216" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR028TB2" class="table-wrap-title" data-id="DSR028TB2"><div class="label -label">Table 2.</div><div class="caption"><p>ORF composition in OCs of <em>M. hyopneumoniae</em> genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Feature </th><th>MHP7448 </th><th>MHP J </th><th>MHP232 </th></tr></thead><tbody><tr><td>Total no. of OCs </td><td>117 </td><td>117 </td><td>116 </td></tr><tr><td>OCs composed of two ORFs </td><td>26 </td><td>26 </td><td>25 </td></tr><tr><td>OCs composed of three ORFs </td><td>19 </td><td>18 </td><td>16 </td></tr><tr><td>OCs composed of four ORFs </td><td>16 </td><td>14 </td><td>16 </td></tr><tr><td>OCs composed of five ORFs </td><td>15 </td><td>16 </td><td>13 </td></tr><tr><td>OCs composed of six ORFs </td><td>9 </td><td>12 </td><td>9 </td></tr><tr><td>OCs composed of seven ORFs </td><td>11 </td><td>10 </td><td>10 </td></tr><tr><td>OCs composed of eight ORFs </td><td>5 </td><td>5 </td><td>5 </td></tr><tr><td>OCs composed of nine ORFs </td><td>2 </td><td>1 </td><td>4 </td></tr><tr><td>OCs composed of 10 ORFs </td><td>3 </td><td>4 </td><td>6 </td></tr><tr><td>OCs composed of 11 ORFs </td><td>3 </td><td>4 </td><td>2 </td></tr><tr><td>OCs composed of 12 ORFs </td><td>2 </td><td>2 </td><td>2 </td></tr><tr><td>OCs composed of 13 ORFs </td><td>1 </td><td>1 </td><td>3 </td></tr><tr><td>OCs composed of 15 ORFs </td><td>1 </td><td>1 </td><td>2 </td></tr><tr><td>OCs composed of 20 ORFs </td><td>1 </td><td>0 </td><td>0 </td></tr><tr><td>OCs composed of 21 ORFs </td><td>0 </td><td>0 </td><td>1 </td></tr><tr><td>OCs composed of 22 ORFs </td><td>1 </td><td>0 </td><td>0 </td></tr><tr><td>OCs composed of 24 ORFs </td><td>0 </td><td>1 </td><td>0 </td></tr><tr><td>OCs composed of 26 ORFs </td><td>1 </td><td>1 </td><td>0 </td></tr><tr><td>OCs composed of 27 ORFs </td><td>0 </td><td>0 </td><td>1 </td></tr><tr><td>OCs composed of 29 ORFs </td><td>1 </td><td>1 </td><td>1 </td></tr></tbody></table></div><div content-id="DSR028TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Feature </th><th>MHP7448 </th><th>MHP J </th><th>MHP232 </th></tr></thead><tbody><tr><td>Total no. of OCs </td><td>117 </td><td>117 </td><td>116 </td></tr><tr><td>OCs composed of two ORFs </td><td>26 </td><td>26 </td><td>25 </td></tr><tr><td>OCs composed of three ORFs </td><td>19 </td><td>18 </td><td>16 </td></tr><tr><td>OCs composed of four ORFs </td><td>16 </td><td>14 </td><td>16 </td></tr><tr><td>OCs composed of five ORFs </td><td>15 </td><td>16 </td><td>13 </td></tr><tr><td>OCs composed of six ORFs </td><td>9 </td><td>12 </td><td>9 </td></tr><tr><td>OCs composed of seven ORFs </td><td>11 </td><td>10 </td><td>10 </td></tr><tr><td>OCs composed of eight ORFs </td><td>5 </td><td>5 </td><td>5 </td></tr><tr><td>OCs composed of nine ORFs </td><td>2 </td><td>1 </td><td>4 </td></tr><tr><td>OCs composed of 10 ORFs </td><td>3 </td><td>4 </td><td>6 </td></tr><tr><td>OCs composed of 11 ORFs </td><td>3 </td><td>4 </td><td>2 </td></tr><tr><td>OCs composed of 12 ORFs </td><td>2 </td><td>2 </td><td>2 </td></tr><tr><td>OCs composed of 13 ORFs </td><td>1 </td><td>1 </td><td>3 </td></tr><tr><td>OCs composed of 15 ORFs </td><td>1 </td><td>1 </td><td>2 </td></tr><tr><td>OCs composed of 20 ORFs </td><td>1 </td><td>0 </td><td>0 </td></tr><tr><td>OCs composed of 21 ORFs </td><td>0 </td><td>0 </td><td>1 </td></tr><tr><td>OCs composed of 22 ORFs </td><td>1 </td><td>0 </td><td>0 </td></tr><tr><td>OCs composed of 24 ORFs </td><td>0 </td><td>1 </td><td>0 </td></tr><tr><td>OCs composed of 26 ORFs </td><td>1 </td><td>1 </td><td>0 </td></tr><tr><td>OCs composed of 27 ORFs </td><td>0 </td><td>0 </td><td>1 </td></tr><tr><td>OCs composed of 29 ORFs </td><td>1 </td><td>1 </td><td>1 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7200216" target="_blank">View Large</a></div></div><p>Genome minimization can be explained by reducing the number of genes or by packing them into strings of clustered genes. In the genus Mycoplasma, during genome reduction both processes seem to have occurred. In <em>M. hyopneumoniae</em>, the genome has been reduced by a loss of genes as those related with the ability to formylate Met-tRNAi<sup><a reveal-id="DSR028C3" data-open="DSR028C3" class="link link-ref link-reveal xref-bibr">3</a></sup> among others. Moreover, as shown in Table <a reveal-id="DSR028TB2" data-open="DSR028TB2" class="link link-reveal link-table xref-fig">2</a>, the density of genes in OC<sub>7448</sub>7 and OC<sub>7448</sub>102 could be explained by the genome compaction. Our results in Table <a reveal-id="DSR028TB2" data-open="DSR028TB2" class="link link-reveal link-table xref-fig">2</a> are in agreement with those reported by Rogozin <em>et al</em>.,<sup><a reveal-id="DSR028C26" data-open="DSR028C26" class="link link-ref link-reveal xref-bibr">26</a></sup> suggesting that, in diverse organisms, the majority of gene clusters are formed by a strings of two to four genes. Therefore, this arrangement found in the three <em>M. hyopneumoniae</em> strains could be the result of the maintenance of the size of the gene clusters during genome reduction.</p>                            <h3 scrollto-destination=7200218 id="7200218" class="section-title" data-legacy-id=s3b>3.2. High level of synteny of the OCs is found among <em>M. hyopneumoniae</em> strains</h3>
<p>Detailed analysis of each OC revealed a high level of synteny between the OCs defined for the 7448 and J strains as well as for the 7448 and 232 strains. Gene order was conserved with 85% similarity when comparing the 7448 and J strains, and 82% when comparing the 7448 and 232 strains (Table <a reveal-id="DSR028TB1" data-open="DSR028TB1" class="link link-reveal link-table xref-fig">1</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S3–S5</a>). Previous comparative analysis of these three genomes was performed at a genomic scale and strain-specific differences were demonstrated, but high rates of conservation were detected in regions of rearrangements, which were probably involved in pathogenesis.<sup><a reveal-id="DSR028C3" data-open="DSR028C3" class="link link-ref link-reveal xref-bibr">3</a></sup> In agreement with Vasconcelos <em>et al</em>.,<sup><a reveal-id="DSR028C3" data-open="DSR028C3" class="link link-ref link-reveal xref-bibr">3</a></sup> our work also demonstrated that the prevalence of ORFs transcribed in the DNA strand with conserved OCs reflected the high level of synteny among the strains' genomes. This occurs not only due to the same arrangement of ORFs but, more interestingly, also due to the presence of IRs of the same size. Although most of the strains display high synteny, low-degree conservation regions also occur. The analysis of genome regions with lower synteny evidences differences in ORF organization and size of intergenic distance. Some cases with common differences are shown in Fig. <a reveal-id="DSR028F2" data-open="DSR028F2" class="link link-reveal link-table xref-fig">2</a> and are related to: (Fig. <a reveal-id="DSR028F2" data-open="DSR028F2" class="link link-reveal link-table xref-fig">2</a>A) ORFs that are separated into different OCs (for example OC<sub>232</sub>46, OC<sub>7448</sub>10 and OC<sub>7448</sub>47, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S3 and S5</a>); (Fig. <a reveal-id="DSR028F2" data-open="DSR028F2" class="link link-reveal link-table xref-fig">2</a>B) addition or removal of OCs (see OC<sub>232</sub>02 and mC<sub>232</sub>01, mC<sub>7448</sub>01 and mC<sub>7448</sub>02, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S2, S3 and S5</a>); (Fig. <a reveal-id="DSR028F2" data-open="DSR028F2" class="link link-reveal link-table xref-fig">2</a>C) OCs with addition or removal of ORFs, generally encoding hypothetic proteins (for example OC<sub>7448</sub>06 and OC<sub>232</sub>07 or OC<sub>7448</sub>100 and OC<sub>J</sub>100, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S3–S5</a>). </p>                        <a id="7200220" scrollto-destination="7200220"></a>
<div id="DSR028F2" data-id="DSR028F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr028/2/m_dsr02802.jpeg?Expires=2147483647&Signature=R1zadox~kKEV7jft3Izwu72uy~AfknPvW0i0J-BLOsYIUJihRGJlqSzwh9D06GSoggj1tp3UYDE2~OTWrfwXq53eZ910RLxClVVCGsRGS~mdSKtbggAAWNLPr9fgUHR5IOmVzYemekc70kno0pL9PFK8E8YRr59j5iGEsE0zlwNm78t8gw6wiRLdAyunGqnshbSSBgpXd9RHSn0~SwX3VJUL7XAgI9yyhyIww9t8jv~r05jhomArBTMltjVjSoiJQlJA-K9usNNZNexsKZXhNg5rWj1oOCfEOrxdNArlVCnbUOMrZTEa8NIupGdWvb1ZZ1waLCcThAuWUOiEqQSqig__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of the OCs’ organization in three M. hyopneumoniae strains. (A) Homologous ORFs that are separated into different OCs in MHP7448 strain. (B) Addition or removal of OCs: the homologous ORF is mC in 7448 strain and is polycistronic in 232 strain. (C) OCs with addition or removal of ORFs, generally encoding hypothetic proteins. The bars connecting the arrows represent close matches. White arrowheads represent the ORFs and indicate the direction of transcription." path-from-xml="dsr02802.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7200220/dsr02802.jpeg" path-from-xml="dsr02802.jpeg" target="_blank">View large</a><a section="7200220" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr028/2/dsr02802.jpeg?Expires=2147483647&Signature=svnt41DEBrUKOrPttGqmKAEtEGHGRrH~AQCvKsAsqJqGQaXRSm1LWMS9qtgJ6F8BozcCn1hH1CeHIe-JDKTJmZvL3O4qC~-TY-UABmm3M1yK1CICLUUsmcfJ4nmwk5hGQfagnkZFTHXFb4q18qbHXgOOTtbMPzxveeKjxZ8nEOD2MAATyWJ26vbQrsq4O5ZXdHj5rYck3xDdAzqjEHRdxiDqdAzNm~R44l27kKwnhPKCO3ATYhUMwmOsue8-TiYC-r0eJoJrmOe27lasSHpErPM0okyaL0Al4gRPCKtVhh1C~~qxhtLeeTJZ0YZsVMpRMkYH9YJMmSuugEPBsTSjLw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7200220&ar=476838&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02802.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of the OCs’ organization in three <em>M. hyopneumoniae</em> strains. (<strong>A</strong>) Homologous ORFs that are separated into different OCs in MHP7448 strain. (<strong>B</strong>) Addition or removal of OCs: the homologous ORF is mC in 7448 strain and is polycistronic in 232 strain. (<strong>C</strong>) OCs with addition or removal of ORFs, generally encoding hypothetic proteins. The bars connecting the arrows represent close matches. White arrowheads represent the ORFs and indicate the direction of transcription.</p></div></div><div content-id="DSR028F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr028/2/m_dsr02802.jpeg?Expires=2147483647&Signature=R1zadox~kKEV7jft3Izwu72uy~AfknPvW0i0J-BLOsYIUJihRGJlqSzwh9D06GSoggj1tp3UYDE2~OTWrfwXq53eZ910RLxClVVCGsRGS~mdSKtbggAAWNLPr9fgUHR5IOmVzYemekc70kno0pL9PFK8E8YRr59j5iGEsE0zlwNm78t8gw6wiRLdAyunGqnshbSSBgpXd9RHSn0~SwX3VJUL7XAgI9yyhyIww9t8jv~r05jhomArBTMltjVjSoiJQlJA-K9usNNZNexsKZXhNg5rWj1oOCfEOrxdNArlVCnbUOMrZTEa8NIupGdWvb1ZZ1waLCcThAuWUOiEqQSqig__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of the OCs’ organization in three M. hyopneumoniae strains. (A) Homologous ORFs that are separated into different OCs in MHP7448 strain. (B) Addition or removal of OCs: the homologous ORF is mC in 7448 strain and is polycistronic in 232 strain. (C) OCs with addition or removal of ORFs, generally encoding hypothetic proteins. The bars connecting the arrows represent close matches. White arrowheads represent the ORFs and indicate the direction of transcription." path-from-xml="dsr02802.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7200220/dsr02802.jpeg" path-from-xml="dsr02802.jpeg" target="_blank">View large</a><a section="7200220" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr028/2/dsr02802.jpeg?Expires=2147483647&Signature=svnt41DEBrUKOrPttGqmKAEtEGHGRrH~AQCvKsAsqJqGQaXRSm1LWMS9qtgJ6F8BozcCn1hH1CeHIe-JDKTJmZvL3O4qC~-TY-UABmm3M1yK1CICLUUsmcfJ4nmwk5hGQfagnkZFTHXFb4q18qbHXgOOTtbMPzxveeKjxZ8nEOD2MAATyWJ26vbQrsq4O5ZXdHj5rYck3xDdAzqjEHRdxiDqdAzNm~R44l27kKwnhPKCO3ATYhUMwmOsue8-TiYC-r0eJoJrmOe27lasSHpErPM0okyaL0Al4gRPCKtVhh1C~~qxhtLeeTJZ0YZsVMpRMkYH9YJMmSuugEPBsTSjLw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7200220&ar=476838&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02802.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of the OCs’ organization in three <em>M. hyopneumoniae</em> strains. (<strong>A</strong>) Homologous ORFs that are separated into different OCs in MHP7448 strain. (<strong>B</strong>) Addition or removal of OCs: the homologous ORF is mC in 7448 strain and is polycistronic in 232 strain. (<strong>C</strong>) OCs with addition or removal of ORFs, generally encoding hypothetic proteins. The bars connecting the arrows represent close matches. White arrowheads represent the ORFs and indicate the direction of transcription.</p></div></div><p>In <em>M. hyopneumoniae</em>, the overall distribution of ORFs within the OCs is highly variable in the number of ORFs and size of the intergenic distances, as well as in the functional categories of the encoded products. Approximately 92% of the OCs display some variations related to these parameters (see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S3–S5</a>). However, some of the OCs are composed of ORFs whose products are related functionally, some examples are: OC<sub>7448</sub>22, OC<sub>7448</sub>43, OC<sub>7448</sub>86, OC<sub>7448</sub>105 and OC<sub>7448</sub>115, all OCs with orthologs in the 232 and J strains (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S3–S5</a>). Taken together, these results suggest that the classical definition of operon cannot be applied to <em>M. hyopneumoniae</em> genomes.</p><p>Despite the high level of synteny between the genomes of <em>M. hyopneumoniae</em> 7448, J and 232, strain-specific differences, such as inverted regions and rearrangements, were demonstrated, being potentially related to pathogenesis.<sup><a reveal-id="DSR028C3" data-open="DSR028C3" class="link link-ref link-reveal xref-bibr">3</a></sup> The inverted segment present in the 232 strain was further analysed and compared with the cognate 7448 region, taking into consideration the ORF string organization and OC distribution (Fig. <a reveal-id="DSR028F3" data-open="DSR028F3" class="link link-reveal link-table xref-fig">3</a>). In <em>M. hyopneumoniae</em> 232, this region corresponds to genome positions from nucleotides 104 691–353 186 bp (OC<sub>232</sub>11–OC<sub>232</sub>46), with a total length of 248 495 bp. In <em>M. hyopneumoniae</em> 7448, the equivalent location is from nucleotides 101 560–354 533 bp (OC<sub>7448</sub>10–OC<sub>7448</sub>46), with a size of 252 973 bp. In this location, the overall conservation of gene order and OC distribution were preserved (Fig. <a reveal-id="DSR028F3" data-open="DSR028F3" class="link link-reveal link-table xref-fig">3</a>). Typical examples of this organization are OC<sub>7448</sub>28 and OC<sub>232</sub>28, each containing 29 ORFs. Other examples are OC<sub>232</sub>15 and OC<sub>7448</sub>41 with 12 conserved ORFs each (see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S3 and S5</a>). A closer inspection of the regions bordering these inverted genome segments showed that, in each case, OC conservation was also preserved (see OC<sub>232</sub>10 and OC<sub>7448</sub>9). Therefore, the genome was inverted but the ORF organization was maintained. Moreover, at the right border side, the distribution of the ORF order was maintained, even when OC<sub>232</sub>46 split from OC<sub>7448</sub>10 and OC<sub>7448</sub>47. It should be noted that the inversion event did not change the relative ORF order in the OCs. Taken together, these results demonstrate the similarity in ORF strings among the three strains, further supporting the hypotheses of Chen <em>et al</em>.<sup><a reveal-id="DSR028C27" data-open="DSR028C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Price <em>et al</em>.<sup><a reveal-id="DSR028C28" data-open="DSR028C28" class="link link-ref link-reveal xref-bibr">28</a></sup> and Roback <em>et al</em>.<sup><a reveal-id="DSR028C29" data-open="DSR028C29" class="link link-ref link-reveal xref-bibr">29</a></sup> that the conservation in OC or TU organization in two or more phylogenetically related species is an important criterion for OC and after TU determination. Therefore, our results suggest that comparative genomics is a plausible approach for predicting OCs in mycoplasma genomes. </p>                        <a id="7200223" scrollto-destination="7200223"></a>
<div id="DSR028F3" data-id="DSR028F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr028/2/m_dsr02803.jpeg?Expires=2147483647&Signature=ETZI6UgtZE6VcC87JqUFS0dCxlahPbvVoznLC4~ofZ2U8SAY6zOwSPeVnRR0LMJhRWkbnDHLSIoy~~GZf4YW7RwgYc9ldngJOJNQVBIhBCzSYPx8ivzx5R5TJCjCsUlbk7d18XYUXR0jx0hOIOg4yhO9PGZYZWFde-AUDyUnWUxfaRegZ8OOxLFMy4MgPrfcQYzdKr4wFLAapHDFtFQq-C2TSAPqAJEdr1~Qf3-dVAVv9AjUJLakO8aPVc5oNZ6B2qIrXwvrMewZWHoNpv1OFcP6aWbAyX7yGolT4hFmncV30I7gnW5RoAmKcVStPY7T33xwx4CVqG12fophk~aEWg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A schematic diagram of the inverted region and rearrangement from M. hyopneumoniae 232 and 7448 strain. The bars connecting the maps represent close matches. The genomic limits of these regions are shown by arrows (OC23211, OC23246, OC744810 and OC744846). The other detached OCs are discussed in the text. Visualization by the software Artemis Compare." path-from-xml="dsr02803.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7200223/dsr02803.jpeg" path-from-xml="dsr02803.jpeg" target="_blank">View large</a><a section="7200223" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr028/2/dsr02803.jpeg?Expires=2147483647&Signature=roQ0l-vVI-eyzjnVWSrk3-9sfPuZz90fuxhVOIRdBnZRQIUOJkZzz0lJ-UfjLcqsbIrz43GB6mROnflU~7O1AS-7ySULlY5CFsopxnQOoPE7GumQufB4ewaMUV8KIS4Gc3uZeMFuRsV5yGfiN4tfteYIDSYzfkya3ffr6WkFgOCmE7oQE9bj4IsFOMTcoYW~H8EAuYDcZlCS2krLfthwwfPVVDr5yAwOuBS7W0RqpIMl1YQcJzKDCr6ONpqLiSrUdled8h6x-F3k6V1UwqT5VuqY-zdeb9dWF1w2rzyUIUwMYyrXGXhT4WiQu2kQQZNLW5ik9Z7jeOXsoH0AsYmD7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7200223&ar=476838&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02803.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A schematic diagram of the inverted region and rearrangement from <em>M. hyopneumoniae</em> 232 and 7448 strain. The bars connecting the maps represent close matches. The genomic limits of these regions are shown by arrows (OC<sub>232</sub>11, OC<sub>232</sub>46, OC<sub>7448</sub>10 and OC<sub>7448</sub>46). The other detached OCs are discussed in the text. Visualization by the software Artemis Compare.</p></div></div><div content-id="DSR028F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr028/2/m_dsr02803.jpeg?Expires=2147483647&Signature=ETZI6UgtZE6VcC87JqUFS0dCxlahPbvVoznLC4~ofZ2U8SAY6zOwSPeVnRR0LMJhRWkbnDHLSIoy~~GZf4YW7RwgYc9ldngJOJNQVBIhBCzSYPx8ivzx5R5TJCjCsUlbk7d18XYUXR0jx0hOIOg4yhO9PGZYZWFde-AUDyUnWUxfaRegZ8OOxLFMy4MgPrfcQYzdKr4wFLAapHDFtFQq-C2TSAPqAJEdr1~Qf3-dVAVv9AjUJLakO8aPVc5oNZ6B2qIrXwvrMewZWHoNpv1OFcP6aWbAyX7yGolT4hFmncV30I7gnW5RoAmKcVStPY7T33xwx4CVqG12fophk~aEWg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A schematic diagram of the inverted region and rearrangement from M. hyopneumoniae 232 and 7448 strain. The bars connecting the maps represent close matches. The genomic limits of these regions are shown by arrows (OC23211, OC23246, OC744810 and OC744846). The other detached OCs are discussed in the text. Visualization by the software Artemis Compare." path-from-xml="dsr02803.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7200223/dsr02803.jpeg" path-from-xml="dsr02803.jpeg" target="_blank">View large</a><a section="7200223" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr028/2/dsr02803.jpeg?Expires=2147483647&Signature=roQ0l-vVI-eyzjnVWSrk3-9sfPuZz90fuxhVOIRdBnZRQIUOJkZzz0lJ-UfjLcqsbIrz43GB6mROnflU~7O1AS-7ySULlY5CFsopxnQOoPE7GumQufB4ewaMUV8KIS4Gc3uZeMFuRsV5yGfiN4tfteYIDSYzfkya3ffr6WkFgOCmE7oQE9bj4IsFOMTcoYW~H8EAuYDcZlCS2krLfthwwfPVVDr5yAwOuBS7W0RqpIMl1YQcJzKDCr6ONpqLiSrUdled8h6x-F3k6V1UwqT5VuqY-zdeb9dWF1w2rzyUIUwMYyrXGXhT4WiQu2kQQZNLW5ik9Z7jeOXsoH0AsYmD7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7200223&ar=476838&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02803.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A schematic diagram of the inverted region and rearrangement from <em>M. hyopneumoniae</em> 232 and 7448 strain. The bars connecting the maps represent close matches. The genomic limits of these regions are shown by arrows (OC<sub>232</sub>11, OC<sub>232</sub>46, OC<sub>7448</sub>10 and OC<sub>7448</sub>46). The other detached OCs are discussed in the text. Visualization by the software Artemis Compare.</p></div></div><p>Some of the differences in ORF strings found among the three <em>M. hyopneumoniae</em> strains could be related to the presence of transposases or to the integrative conjugative element.<sup><a reveal-id="DSR028C3" data-open="DSR028C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Although the number of transposases is almost similar in all the three genomes (Table <a reveal-id="DSR028TB1" data-open="DSR028TB1" class="link link-reveal link-table xref-fig">1</a>), their localization in some cases, probably, could be involved in the generation of instable regions where most changes occur, leaving the rest of the genome stable. The integrative conjugative element (ICEH), related to genome rearrangement and probably pathogenesis, is present in the genomes of <em>M. hyopneumoniae</em> 7448 and 232 strains but have a different organization.<sup><a reveal-id="DSR028C3" data-open="DSR028C3" class="link link-ref link-reveal xref-bibr">3</a></sup> The ICEH is located within OC<sub>7448</sub>70, together with ORFs usually associated with bacterial conjugative elements and ORFs encoding hypothetical products, and within OC<sub>232</sub>86 with a similar organization.</p>                            <h3 scrollto-destination=7200225 id="7200225" class="section-title" data-legacy-id=s3c>3.3. Experimental analysis of OCs cotranscription</h3>
<p>Aiming to validate some of the OCs predicted by our <em>in silico</em> analysis, 21 OCs encompassing 115 ORFs from <em>M. hyopneumoniae</em> 7448 were further characterized, corresponding to 17% of the total ORFs in the genome and 18% of the predicted OCs (Table <a reveal-id="DSR028TB1" data-open="DSR028TB1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). The selection of these OCs was initially based on functional category variation among the ORFs and OCs with ORFs with long IRs. Usually, experimental validation of TUs is often established by measuring the length of mRNAs through Northern analysis. However, this approach failed to yield reliable transcript sizes in our experiments (data not shown). This experimental difficulty was also reported for other mycoplasmas and is probably due to the large size and instability of the transcripts in this genus.<sup><a reveal-id="DSR028C7" data-open="DSR028C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSR028C8" data-open="DSR028C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Therefore, we performed gene-specific RT–PCR strategy to amplify the IRs between the ORFs of the predicted OC<sub>7448</sub> for experimental validation. Briefly, as shown in Fig. <a reveal-id="DSR028F1" data-open="DSR028F1" class="link link-reveal link-table xref-fig">1</a>, we designed primer pairs that would only amplify a product if adjacent genes could be found on a single mRNA molecule. In the example shown in Fig. <a reveal-id="DSR028F1" data-open="DSR028F1" class="link link-reveal link-table xref-fig">1</a>, OC<sub>7448</sub>111 encompassed four ORFs and one tRNA gene and amplicons were generated from: MHP7448_0653 and <em>ksgA</em> (amplicon 263 bp), <em>ksgA</em> and <em>rpsL</em> (amplicon 706 bp), <em>rpsL</em> and tRNA30 (amplicon 357 bp), MHP7448_0653 and <em>rpsL</em> (amplicon 1555 bp), <em>ksgA</em> and tRNA30 (amplicon 1157 bp), indicating that a polycistronic mRNA spanning from MHP7448_0653 to tRNA30 is produced.</p><p>The experimental data from RT–PCR analysis of <em>M. hyopneumoniae</em> 7448, using primer pairs to each gene spanning the junctions between genes, demonstrated that the ORFs localized within each OC were linked. Therefore, 18% of the OCs characterized <em>in silico</em> in the 7448 strain were validated, corresponding to 21 of 117 OCs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). To better demonstrate these results, three segments of the <em>M. hyopneumoniae</em> 7448 genome encompassing 63.2 kb (Fig. <a reveal-id="DSR028F4" data-open="DSR028F4" class="link link-reveal link-table xref-fig">4</a>A), 27.5 kb (Fig. <a reveal-id="DSR028F4" data-open="DSR028F4" class="link link-reveal link-table xref-fig">4</a>B) and 21.4 kb (Fig. <a reveal-id="DSR028F4" data-open="DSR028F4" class="link link-reveal link-table xref-fig">4</a>C) are shown. The first genome region analysed contained nine OCs (OC<sub>7448</sub>109–OC<sub>7448</sub>117) with a total of 44 ORFs (Fig. <a reveal-id="DSR028F4" data-open="DSR028F4" class="link link-reveal link-table xref-fig">4</a>A), representing the typical ORF string distribution as previously described: (i) ORF number: two ORFs in OC<sub>7448</sub>110, OC<sub>7448</sub>113, OC<sub>7448</sub>114, OC<sub>7448</sub>115 up until 11 ORFs in OC<sub>7448</sub>109; (ii) ORF product: OC<sub>7448</sub>114 products from the same functional category and OC<sub>7448</sub>112 products not functionally related and (iii) ORF string conservation: OC<sub>7448</sub>112 with conserved gene order in OC<sub>J</sub>112 and OC<sub>232</sub>111; OC<sub>7448</sub>113 with same gene order as OC<sub>J</sub>113 and OC<sub>232</sub>112 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S4–S6</a>). </p>                        <a id="7200228" scrollto-destination="7200228"></a>
<div id="DSR028F4" data-id="DSR028F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr028/2/m_dsr02804.jpeg?Expires=2147483647&Signature=x5MtfIRI4mFGJEZxkqOgdcswxT-AOWwq~Jc6fAAwGO~VKUAaUDpOlgn~h0O8N7yHvOm3wvMZRd-JxrjqLWEFrDEFXHn5IEZxmrE5d78CNC-wyR-~atnTyPpVBWDaUOr3p5apd6b7FAAlIt~N6JDcMo8mSr1RLaPVj71fUdukSjFcvglvqGH8Y9aqsVJzUPLmrKnALPiu0~qZXtibOHT8JuIxM85752wAc1VrHJTTJPUVhr~cf3ymXe-pr~dNcFtGLv3vgcq9aXfqg2q3BmOujsiSnxGy6G6oSqX-QFgNvccY5HvsTL2fxL~Ozd5pCBLb2GoMdz~eedoQUHorVeESxw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Representation of experimentally analyzed OCs in M. hyopneumoniae genome. Some regions experimentally tested (A–C) are detached from the overall picture of the genome (labelled from 1 to 920 079 bp). The position on the chromosomal sequence is indicated in bp below both termini of the bars, with white arrowheads indicating the direction of transcription. (B and C) The size of some intergenic regions are marked (IR: followed by the number). (A) Genome coverage of 63.2 kb showing nine OCs. (B) Genome coverage of 27.5 kb showing three OCs. Two intergenic regions (IR: 711 bp and IR: 2757 bp) are indicated. (C) Genome coverage of 21.2 kb showing three OCs and two mC. The intergenic region (IR: 299 bp) in OC744890 represent the sequence between tuf and lon genes. Visualization by the software Artemis." path-from-xml="dsr02804.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7200228/dsr02804.jpeg" path-from-xml="dsr02804.jpeg" target="_blank">View large</a><a section="7200228" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr028/2/dsr02804.jpeg?Expires=2147483647&Signature=KsWprc066ZdLT7bSeXET9szNB66fL2yZKFweAR92pMz~bajWWm5MC3NZ7eB5ZPyog4klRbHK~66Uyrpt9K39FYrVNMqo1Pabi74clbaJ1-1GhFxTE0zYj~F1ZmTRguf1IheijtULjD4INTRxU6h~Hyr7VzGJz1tditEd7o~-dGdlfIIpM6HPX7uo7HHwKJjLUVv-NVwHXL1-MzkBVJt8laGwHsMhnLpW0q8SFBhjXEvDbmOzWN5euJrUvLcypJGkLEi-lZpoMSHrk7N5tPMMYDtfDHuMXf51-T4ZbMtvf2UGCbn7Q5wYLt0t5Yxw27PtcPaeIeo0QI3pwjiFPcoBGA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7200228&ar=476838&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02804.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Representation of experimentally analyzed OCs in <em>M. hyopneumoniae</em> genome. Some regions experimentally tested (<strong>A</strong>–<strong>C</strong>) are detached from the overall picture of the genome (labelled from 1 to 920 079 bp). The position on the chromosomal sequence is indicated in bp below both termini of the bars, with white arrowheads indicating the direction of transcription. (<strong>B</strong> and <strong>C</strong>) The size of some intergenic regions are marked (IR: followed by the number). (<strong>A</strong>) Genome coverage of 63.2 kb showing nine OCs. (<strong>B</strong>) Genome coverage of 27.5 kb showing three OCs. Two intergenic regions (IR: 711 bp and IR: 2757 bp) are indicated. (<strong>C</strong>) Genome coverage of 21.2 kb showing three OCs and two mC. The intergenic region (IR: 299 bp) in OC<sub>7448</sub>90 represent the sequence between <em>tuf</em> and <em>lon</em> genes. Visualization by the software Artemis.</p></div></div><div content-id="DSR028F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr028/2/m_dsr02804.jpeg?Expires=2147483647&Signature=x5MtfIRI4mFGJEZxkqOgdcswxT-AOWwq~Jc6fAAwGO~VKUAaUDpOlgn~h0O8N7yHvOm3wvMZRd-JxrjqLWEFrDEFXHn5IEZxmrE5d78CNC-wyR-~atnTyPpVBWDaUOr3p5apd6b7FAAlIt~N6JDcMo8mSr1RLaPVj71fUdukSjFcvglvqGH8Y9aqsVJzUPLmrKnALPiu0~qZXtibOHT8JuIxM85752wAc1VrHJTTJPUVhr~cf3ymXe-pr~dNcFtGLv3vgcq9aXfqg2q3BmOujsiSnxGy6G6oSqX-QFgNvccY5HvsTL2fxL~Ozd5pCBLb2GoMdz~eedoQUHorVeESxw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Representation of experimentally analyzed OCs in M. hyopneumoniae genome. Some regions experimentally tested (A–C) are detached from the overall picture of the genome (labelled from 1 to 920 079 bp). The position on the chromosomal sequence is indicated in bp below both termini of the bars, with white arrowheads indicating the direction of transcription. (B and C) The size of some intergenic regions are marked (IR: followed by the number). (A) Genome coverage of 63.2 kb showing nine OCs. (B) Genome coverage of 27.5 kb showing three OCs. Two intergenic regions (IR: 711 bp and IR: 2757 bp) are indicated. (C) Genome coverage of 21.2 kb showing three OCs and two mC. The intergenic region (IR: 299 bp) in OC744890 represent the sequence between tuf and lon genes. Visualization by the software Artemis." path-from-xml="dsr02804.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7200228/dsr02804.jpeg" path-from-xml="dsr02804.jpeg" target="_blank">View large</a><a section="7200228" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr028/2/dsr02804.jpeg?Expires=2147483647&Signature=KsWprc066ZdLT7bSeXET9szNB66fL2yZKFweAR92pMz~bajWWm5MC3NZ7eB5ZPyog4klRbHK~66Uyrpt9K39FYrVNMqo1Pabi74clbaJ1-1GhFxTE0zYj~F1ZmTRguf1IheijtULjD4INTRxU6h~Hyr7VzGJz1tditEd7o~-dGdlfIIpM6HPX7uo7HHwKJjLUVv-NVwHXL1-MzkBVJt8laGwHsMhnLpW0q8SFBhjXEvDbmOzWN5euJrUvLcypJGkLEi-lZpoMSHrk7N5tPMMYDtfDHuMXf51-T4ZbMtvf2UGCbn7Q5wYLt0t5Yxw27PtcPaeIeo0QI3pwjiFPcoBGA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7200228&ar=476838&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02804.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Representation of experimentally analyzed OCs in <em>M. hyopneumoniae</em> genome. Some regions experimentally tested (<strong>A</strong>–<strong>C</strong>) are detached from the overall picture of the genome (labelled from 1 to 920 079 bp). The position on the chromosomal sequence is indicated in bp below both termini of the bars, with white arrowheads indicating the direction of transcription. (<strong>B</strong> and <strong>C</strong>) The size of some intergenic regions are marked (IR: followed by the number). (<strong>A</strong>) Genome coverage of 63.2 kb showing nine OCs. (<strong>B</strong>) Genome coverage of 27.5 kb showing three OCs. Two intergenic regions (IR: 711 bp and IR: 2757 bp) are indicated. (<strong>C</strong>) Genome coverage of 21.2 kb showing three OCs and two mC. The intergenic region (IR: 299 bp) in OC<sub>7448</sub>90 represent the sequence between <em>tuf</em> and <em>lon</em> genes. Visualization by the software Artemis.</p></div></div><p>Some reports have suggested that genes co-transcribed as TUs or operons are likely to be compactly arranged on the genome with short IRs, although there are cases where regulatory elements can cause variations in the length of TUs.<sup><a reveal-id="DSR028C30" data-open="DSR028C30" class="link link-ref link-reveal xref-bibr">30</a>,<a reveal-id="DSR028C31" data-open="DSR028C31" class="link link-ref link-reveal xref-bibr">31</a></sup><em>Mycoplasma hyopneumoniae</em> has a reduced genome with limited biosynthetic potential, and the presence of short intergenic sequences would help us to allow efficient transcription. Interestingly, the size of the IRs was extremely variable within different OCs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S3–S5</a>). RT–PCRs with specific primers were applied to amplify all the IRs present in the 21 OCs and cotranscription was observed, even with large IRs (Fig. <a reveal-id="DSR028F4" data-open="DSR028F4" class="link link-reveal link-table xref-fig">4</a>B, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). The OC<sub>7448</sub>33 contained six ORFs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>) with a significant variation in the IR: (i) IR length of 2757 bp between <em>glyA</em> and <em>nrdF</em> genes; (ii) IR length of 2 bp between <em>nrdF</em> and <em>nrdI</em> genes; (iii) IR length of 76 bp between <em>nrdI</em> and <em>nrdE</em> genes; (iv) IR length of 89 bp between <em>nrdE</em> gene and an ORF encoding a hypothetical protein (MHP7448_0220) and (v) IR length of 711 bp between two ORFs encoding hypothetical proteins (MHP7448_0220 and MHP7448_0219).</p>                            <h3 scrollto-destination=7200230 id="7200230" class="section-title" data-legacy-id=s3d>3.4. Cotranscription of ORF strings in the same DNA strand</h3>
<p>The genome region detailed in Fig. <a reveal-id="DSR028F4" data-open="DSR028F4" class="link link-reveal link-table xref-fig">4</a>C illustrates the analysis to validate the criteria used <em>in silico</em> for OC definition, based on the presence of ORF strings in the same DNA strand until the occurrence of ORFs in the opposite strand. The ORF strings in OC<sub>7448</sub>90 and OC<sub>7448</sub>91 were interrupted by the presence of mC<sub>7448</sub>25 in the opposite strand. For the RT–PCRs, two primer pairs were designed to evaluate the presence of a polycistronic transcript corresponding to the ORFs strings in OC<sub>7448</sub>90 plus OC<sub>7448</sub>91. The first primer pair (gyrF/gyrR, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>) amplified a 941 bp DNA fragment of the MHP7448_0528_gyrA (mC<sub>7448</sub>25). The other primer pair was localized within the coding sequence of first ORF (<em>deo</em>) of OC<sub>7448</sub>90 and in the IR between OC<sub>7448</sub>90 and OC<sub>7448</sub>91 (gyr/deoC, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>), potentially generating an amplification of 1338 bp. Although the presence of mRNA for the <em>gyrA</em> ORF was demonstrated, no transcript (single polycistronic mRNA) was observed, including both OC (OC<sub>7448</sub>90 plus OC<sub>7448</sub>91) ORF strings. Therefore, these results support the criteria used <em>in silico</em> for defining the OCs characterized by the presence of two or more ORFs in the same DNA strand until the occurrence of ORFs in the opposite strand. Taken together, these findings demonstrated that the typical gene organization present in other bacteria, as <em>Escherichia coli</em> or <em>Bacillus subtilis</em>, cannot be applied to <em>M. hyopneumoniae</em> genome.</p><p>Although in <em>M. hyopneumoniae</em> genes are arranged in long clusters, suggesting that the starting point of transcription occurs in the first gene, the transcription termination regions have yet to be defined. Aiming to contribute to the definition of the terminator sequences, one IR between OC<sub>7448</sub>112 and OC<sub>7448</sub>113 (see Fig. <a reveal-id="DSR028F1" data-open="DSR028F1" class="link link-reveal link-table xref-fig">1</a>A) was analysed by RT–PCR. Primers pairs were designed to amplify the region between the ends of each OC (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). Our results demonstrate the presence of transcripts in this region, suggesting the absence of specific regions for transcription termination. These findings are supported by the observation of Gardner and Minion<sup><a reveal-id="DSR028C18" data-open="DSR028C18" class="link link-ref link-reveal xref-bibr">18</a></sup> that transcription occurs in IRs of <em>M. hyopneumoniae</em>, and that transcription run off the end of the last ORF.</p><p>The gene encoding the elongation factor Tu (<em>tuf</em>) is the last of five ORFs to constitute an OC (OC<sub>7448</sub>90), expressed from a promoter upstream of the first ORF (<em>deoC</em>) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>, Fig. <a reveal-id="DSR028F4" data-open="DSR028F4" class="link link-reveal link-table xref-fig">4</a>C). However, in the 299 bp IR between <em>tuf</em> and <em>lon</em>, a promoter-like sequence was found by primer extension analysis (data not shown). The OC<sub>7448</sub>90 represents a typical OC in <em>M. hyopneumoniae</em>, with ORFs encoding products not related functionally as <em>deoC</em> (deoxyribose-phosphate aldolase), <em>upp</em> (uracil phosphoribosyltransferase), MHP7448_0525 (hypothetical protein), <em>lon</em> (heat shock ATP-dependent protease) and <em>tuf</em> (elongation factor Tu, EF-Tu).</p><p>EF-Tu is necessary for protein synthesis in metabolically active cells, being possibly involved in the synthesis of alternative transcripts in the <em>M. hyopneumoniae</em> 7448 strain. It is already known that, in TUs, discoordinate expression occurs when the product of an individual gene needs to be expressed apart from the others to mediate a different role.<sup><a reveal-id="DSR028C31" data-open="DSR028C31" class="link link-ref link-reveal xref-bibr">31</a>,<a reveal-id="DSR028C32" data-open="DSR028C32" class="link link-ref link-reveal xref-bibr">32</a></sup> The OC<sub>7448</sub>90 has upstream and internal promoters resulting in two different sizes of clusters, indicating that the entire OC could be expressed from a common promoter or a subset of the OC (<em>tuf</em>) is separately transcribed in response to a specific signal. Previous reports have suggested a relationship between the expanded IRs and presence of internal regulatory elements in highly expressed TUs.<sup><a reveal-id="DSR028C31" data-open="DSR028C31" class="link link-ref link-reveal xref-bibr">31</a>,<a reveal-id="DSR028C33" data-open="DSR028C33" class="link link-ref link-reveal xref-bibr">33</a></sup> Güell <em>et al</em>.<sup><a reveal-id="DSR028C12" data-open="DSR028C12" class="link link-ref link-reveal xref-bibr">12</a></sup> in an <em>M. pneumoniae</em> transcriptome analysis demonstrated many alternative transcripts. The experimental evidence of the presence of regulatory elements such as internal promoters in the expanded IRs provided by the analysis of the <em>tuf</em> gene (IR of 299 bp between adjacent genes) and the variable size of the <em>M. hyopneumoniae</em> IRs analysed by RT–PCR suggest the presence of internal promoters in the IRs of <em>M. hyopneumoniae</em> OCs and a complexity in transcription regulation.</p>                            <h2 scrollto-destination=7200235 id="7200235" class="section-title" data-legacy-id=s4>4. Conclusions</h2>
<p>Different approaches have been used to analyse gene organization in bacterial genomes through the development of new algorithm<sup><a reveal-id="DSR028C34" data-open="DSR028C34" class="link link-ref link-reveal xref-bibr">34</a></sup> or prediction programs,<sup><a reveal-id="DSR028C35" data-open="DSR028C35" class="link link-ref link-reveal xref-bibr">35</a>,<a reveal-id="DSR028C36" data-open="DSR028C36" class="link link-ref link-reveal xref-bibr">36</a></sup> which are based on characteristics such as intergenic distance, closely related gene function, conservation of gene order in phylogenetically close genomes, or even the prediction of promoter region<sup><a reveal-id="DSR028C37" data-open="DSR028C37" class="link link-ref link-reveal xref-bibr">37</a></sup> and transcriptional terminators.<sup><a reveal-id="DSR028C19" data-open="DSR028C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSR028C20" data-open="DSR028C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Although these algorithms have shown a great promise in terms of being able to predict operon structure with a high degree of specificity and sensitivity, the data that they rely on are available only for a small number of bacterial species<sup><a reveal-id="DSR028C36" data-open="DSR028C36" class="link link-ref link-reveal xref-bibr">36</a></sup> and their application to all bacterial species could present some difficulties. Furthermore, combinations of these prediction analyses with experimental data are scarce. Not surprisingly, analysis of OC prediction followed by experimental confirmation in mycoplasma genomes was not conducted prior to this study. Moreover, experimental data on TUs are also scarce.</p><p>Similar to <em>M. hyopneumoniae</em> genes, some <em>M. pneumoniae</em> genes are arranged in long OCs but only with short or almost no intergenic sequences.<sup><a reveal-id="DSR028C12" data-open="DSR028C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Analysis of P97 and P102 paralog families have also suggested that a distance of up to 54 bp is an upper limit for defining the cotranscript structure in <em>M. hyopneumoniae</em>.<sup><a reveal-id="DSR028C9" data-open="DSR028C9" class="link link-ref link-reveal xref-bibr">9</a></sup> However, we have systematically mapped <em>in silico</em> all the OCs in three different strains of <em>M. hyopneumoniae</em> and experimentally validated 115 ORFs in 21 OCs with variably sized IRs in the 7448 strain. It has been proposed that intergenic distances between ORFs in OCs are similar in prokaryotes and can be used to predict OCs and to estimate the total number of OCs in genomes.<sup><a reveal-id="DSR028C26" data-open="DSR028C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSR028C28" data-open="DSR028C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSR028C29" data-open="DSR028C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DSR028C38" data-open="DSR028C38" class="link link-ref link-reveal xref-bibr">38</a>,<a reveal-id="DSR028C39" data-open="DSR028C39" class="link link-ref link-reveal xref-bibr">39</a></sup> However, according to our analysis, the distance models do not seem to be applicable for OC prediction in <em>M. hyopneumoniae</em> genomes. Furthermore, the variability in size of intergenic distances within an OC can be related to the complexity of mechanisms of gene regulation. Therefore, the unusual distribution of intergenic distances between ORFs within OCs in <em>M. hyopneumoniae</em> may reflect a biological difference in the genome structure as previous suggested for <em>Synechocystis</em>.<sup><a reveal-id="DSR028C28" data-open="DSR028C28" class="link link-ref link-reveal xref-bibr">28</a></sup> Furthermore, transcription analysis across the IRs of the <em>M. hyopneumoniae</em> 232 genome demonstrated the presence of transcripts in the IRs and that transcription run off the end of the last ORF and RNA polymerase is gradually released from template.<sup><a reveal-id="DSR028C18" data-open="DSR028C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p><p>Taken together, these findings suggest that OCs are continuously transcribed until reaching the next OC or ORF present in the opposite strand. However, a subset of the OCs could be independently transcribed due to the presence of internal promoters. These new data suggest complex transcriptional organization in <em>M. hyopneumoniae</em> genomes and the probability of the occurrence of other not-yet-known mechanisms that are involved in transcription. Furthermore, the high synteny of gene organization between pathogenic and non-pathogenic strains indicates that pathogenicity is not related to gene organization. The accelerating pace of molecular research into this important pathogen is certain to provide additional data to refine the results described in this work, forming a solid empirical foundation for our future understanding of the relationships between gene organization and transcription regulation in <em>M. hyopneumoniae</em>.</p><p><strong>Supplementary data:</strong><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=7200241 id="7200241" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by grants from the Brazilian National Research Council (CNPq).</p>                            <h2 scrollto-destination=7200243 id="7200243" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSR028C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ross</div> <div class="given-names">R.F.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Leman</div> <div class="given-names">A.D.</div></div>,  <div class="name"><div class="surname">Straw</div> <div class="given-names">B.E.</div></div>,  <div class="name"><div class="surname">Mengeling</div> <div class="given-names">W.L.</div></div>,  <div class="name"><div class="surname">D'Allaire</div> <div class="given-names">S.,</div></div>,  <div class="name"><div class="surname">Taylor</div> <div class="given-names">D.J.</div></div>. </div><div class="article-title">Mycoplasmal disease</div>, <div class="source">Diseases of Swine</div> , <div class="year">1992</div><div class="publisher-loc">Ames</div><div class="publisher-name">Iowa State University Press</div>(pg. <div class="fpage">537</div>-<div class="lpage">51</div>)</div></div></div></div><div content-id="DSR028C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Minion</div> <div class="given-names">F.C.</div></div>,  <div class="name"><div class="surname">Lefkowitz</div> <div class="given-names">E.L.</div></div>,  <div class="name"><div class="surname">Madsen</div> <div class="given-names">M.L.</div></div>, et al. </div><div class="article-title">The genome sequence of strain 232, the agent of swine mycoplasmosis</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2004</div>, vol. <div class="volume">186</div> (pg. <div class="fpage">7123</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20sequence%20of%20strain%20232%2C%20the%20agent%20of%20swine%20mycoplasmosis&amp;author=F.C.%20Minion&amp;author=E.L.%20Lefkowitz&amp;author=M.L.%20Madsen&amp;publication_year=2004&amp;journal=J.%20Bacteriol.&amp;volume=186&amp;pages=7123-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.186.21.7123-7133.2004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.186.21.7123-7133.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.186.21.7123-7133.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15489423" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR028C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vasconcelos</div> <div class="given-names">A.T.</div></div>,  <div class="name"><div class="surname">Ferreira</div> <div class="given-names">H.B.</div></div>,  <div class="name"><div class="surname">Bizarro</div> <div class="given-names">C.V.</div></div>, et al. </div><div class="article-title">Swine and poultry pathogens: the complete genome sequences of two strains of <em>Mycoplasma hyopneumoniae</em> and a strain of <em>Mycoplasma synoviae</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2005</div>, vol. <div class="volume">187</div> (pg. <div class="fpage">5568</div>-<div class="lpage">77</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Swine%20and%20poultry%20pathogens%3A%20the%20complete%20genome%20sequences%20of%20two%20strains%20of%20Mycoplasma%20hyopneumoniae%20and%20a%20strain%20of%20Mycoplasma%20synoviae&amp;author=A.T.%20Vasconcelos&amp;author=H.B.%20Ferreira&amp;author=C.V.%20Bizarro&amp;publication_year=2005&amp;journal=J.%20Bacteriol.&amp;volume=187&amp;pages=5568-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.187.16.5568-5577.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.187.16.5568-5577.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.187.16.5568-5577.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16077101" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR028C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Feng</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Fang</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">Complete genome sequence of <em>Mycoplasma hyopneumoniae</em> strain 168</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2011</div>, vol. <div class="volume">193</div> (pg. <div class="fpage">1016</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20Mycoplasma%20hyopneumoniae%20strain%20168&amp;author=W.%20Liu&amp;author=Z.%20Feng&amp;author=L.%20Fang&amp;publication_year=2011&amp;journal=J.%20Bacteriol.&amp;volume=193&amp;pages=1016-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.01305-10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.01305-10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.01305-10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21148737" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR028C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Inamine</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Loechel</div> <div class="given-names">S.,</div></div>,  <div class="name"><div class="surname">Hu</div> <div class="given-names">P.C.</div></div>. </div><div class="article-title">Analysis of the nucleotide sequence of the P1 operon of <em>Mycoplasma pneumoniae</em></div>, <div class="source">Gene</div> , <div class="year">1988</div>, vol. <div class="volume">73</div> (pg. <div class="fpage">175</div>-<div class="lpage">83</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20the%20nucleotide%20sequence%20of%20the%20P1%20operon%20of%20Mycoplasma%20pneumoniae&amp;author=J.M.%20Inamine&amp;author=S.%2C%20Loechel&amp;author=P.C.%20Hu&amp;publication_year=1988&amp;journal=Gene&amp;volume=73&amp;pages=175-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119(88)90323-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119(88)90323-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0378-1119(88)90323-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2468577" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR028C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Himmelreich</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Plagens</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Hilbert</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Reiner</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Herrmann</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Comparative analysis of the genomes of the bacteria <em>Mycoplasma pneumoniae</em> and <em>Mycoplasma genitalium</em></div>, <div class="source">Nucl. Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">701</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20the%20genomes%20of%20the%20bacteria%20Mycoplasma%20pneumoniae%20and%20Mycoplasma%20genitalium&amp;author=R.%20Himmelreich&amp;author=H.%20Plagens&amp;author=H.%20Hilbert&amp;author=B.%20Reiner&amp;author=R.%20Herrmann&amp;publication_year=1997&amp;journal=Nucl.%20Acids%20Res.&amp;volume=25&amp;pages=701-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.4.701" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.4.701" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.4.701"> </span></div> </div></div></div></div></div><div content-id="DSR028C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Waldo</div> <div class="given-names">R.H.</div>III</div>,  <div class="name"><div class="surname">Popham</div> <div class="given-names">P.L.</div></div>,  <div class="name"><div class="surname">Romero-Arroyo</div> <div class="given-names">C.E.</div></div>,  <div class="name"><div class="surname">Mothershed</div> <div class="given-names">E.A.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">K.K.,</div></div>,  <div class="name"><div class="surname">Krause</div> <div class="given-names">D.C.</div></div>. </div><div class="article-title">Transcriptional analysis of the <em>hmw</em> gene cluster of <em>Mycoplasma pneumoniae</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">1999</div>, vol. <div class="volume">181</div> (pg. <div class="fpage">4978</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20analysis%20of%20the%20hmw%20gene%20cluster%20of%20Mycoplasma%20pneumoniae&amp;author=R.H.%20Waldo&amp;author=P.L.%20Popham&amp;author=C.E.%20Romero-Arroyo&amp;author=E.A.%20Mothershed&amp;author=K.K.%2C%20Lee&amp;author=D.C.%20Krause&amp;publication_year=1999&amp;journal=J.%20Bacteriol.&amp;volume=181&amp;pages=4978-85" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10438770" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR028C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Musatovova</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Dhandayuthapani</div> <div class="given-names">S.,</div></div>,  <div class="name"><div class="surname">Baseman</div> <div class="given-names">J.B.</div></div>. </div><div class="article-title">Transcriptional starts for cytadherence-related operons of <em>Mycoplasma genitalium</em></div>, <div class="source">FEMS Microbiol. Lett.</div> , <div class="year">2003</div>, vol. <div class="volume">229</div> (pg. <div class="fpage">73</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20starts%20for%20cytadherence-related%20operons%20of%20Mycoplasma%20genitalium&amp;author=O.%20Musatovova&amp;author=S.%2C%20Dhandayuthapani&amp;author=J.B.%20Baseman&amp;publication_year=2003&amp;journal=FEMS%20Microbiol.%20Lett.&amp;volume=229&amp;pages=73-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1097(03)00789-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1097(03)00789-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1097(03)00789-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14659545" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR028C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Adams</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Pitzer</div> <div class="given-names">J.,</div></div>,  <div class="name"><div class="surname">Minion</div> <div class="given-names">F.C.</div></div>. </div><div class="article-title"><em>In vivo</em> expression analysis of the P97 and P102 paralog families of <em>Mycoplasma hyopneumoniae</em></div>, <div class="source">Infect. Immun.</div> , <div class="year">2005</div>, vol. <div class="volume">73</div> (pg. <div class="fpage">7784</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20vivo%20expression%20analysis%20of%20the%20P97%20and%20P102%20paralog%20families%20of%20Mycoplasma%20hyopneumoniae&amp;author=C.%20Adams&amp;author=J.%2C%20Pitzer&amp;author=F.C.%20Minion&amp;publication_year=2005&amp;journal=Infect.%20Immun.&amp;volume=73&amp;pages=7784-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/IAI.73.11.7784-7787.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FIAI.73.11.7784-7787.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FIAI.73.11.7784-7787.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16239586" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR028C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Benders</div> <div class="given-names">G.A.</div></div>,  <div class="name"><div class="surname">Powell</div> <div class="given-names">B.C.,</div></div>,  <div class="name"><div class="surname">Hutchison Iii</div> <div class="given-names">C.A.</div></div>. </div><div class="article-title">Transcriptional analysis of the conserved <em>fts</em>Z gene cluster in <em>Mycoplasma genitalium</em> and <em>Mycoplasma pneumoniae</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2005</div>, vol. <div class="volume">187</div> (pg. <div class="fpage">4542</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20analysis%20of%20the%20conserved%20ftsZ%20gene%20cluster%20in%20Mycoplasma%20genitalium%20and%20Mycoplasma%20pneumoniae&amp;author=G.A.%20Benders&amp;author=B.C.%2C%20Powell&amp;author=C.A.%20Hutchison%20Iii&amp;publication_year=2005&amp;journal=J.%20Bacteriol.&amp;volume=187&amp;pages=4542-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.187.13.4542-4551.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.187.13.4542-4551.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.187.13.4542-4551.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15968065" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR028C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Waldo</div> <div class="given-names">R.H.</div>III</div>,  <div class="name"><div class="surname">Krause</div> <div class="given-names">D.C.</div></div>. </div><div class="article-title">Synthesis, stability, and function of cytadhesin P1 and accessory protein B/C complex of <em>Mycoplasma pneumoniae</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2006</div>, vol. <div class="volume">188</div> (pg. <div class="fpage">569</div>-<div class="lpage">75</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synthesis%2C%20stability%2C%20and%20function%20of%20cytadhesin%20P1%20and%20accessory%20protein%20B%2FC%20complex%20of%20Mycoplasma%20pneumoniae&amp;author=R.H.%20Waldo&amp;author=D.C.%20Krause&amp;publication_year=2006&amp;journal=J.%20Bacteriol.&amp;volume=188&amp;pages=569-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.188.2.569-575.2006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.188.2.569-575.2006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.188.2.569-575.2006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16385047" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR028C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Güell</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Noort</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Yus</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Transcriptome complexity in a genome-reduced bacterium</div>, <div class="source">Science</div> , <div class="year">2009</div>, vol. <div class="volume">326</div> (pg. <div class="fpage">1268</div>-<div class="lpage">71</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20complexity%20in%20a%20genome-reduced%20bacterium&amp;author=M.%20G%C3%BCell&amp;author=V.%20Noort&amp;author=E.%20Yus&amp;publication_year=2009&amp;journal=Science&amp;volume=326&amp;pages=1268-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1176951" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1176951" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1176951"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19965477" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR028C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Muto</div> <div class="given-names">A.,</div></div>,  <div class="name"><div class="surname">Osawa</div> <div class="given-names">S.</div></div>. </div><div class="article-title">The guanine and cytosine content of genomic DNA and bacterial evolution</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1987</div>, vol. <div class="volume">84</div> (pg. <div class="fpage">166</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20guanine%20and%20cytosine%20content%20of%20genomic%20DNA%20and%20bacterial%20evolution&amp;author=A.%2C%20Muto&amp;author=S.%20Osawa&amp;publication_year=1987&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=84&amp;pages=166-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.84.1.166" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.84.1.166" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.84.1.166"> </span></div> </div></div></div></div></div><div content-id="DSR028C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Weiner</div> <div class="given-names">J.</div>III</div>,  <div class="name"><div class="surname">Herrmann</div> <div class="given-names">R.,</div></div>,  <div class="name"><div class="surname">Browning</div> <div class="given-names">G.F.</div></div>. </div><div class="article-title">Transcription in <em>Mycoplasma pneumoniae</em></div>, <div class="source">Nucl. Acids Res.</div> , <div class="year">2000</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">4488</div>-<div class="lpage">96</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription%20in%20Mycoplasma%20pneumoniae&amp;author=J.%20Weiner&amp;author=R.%2C%20Herrmann&amp;author=G.F.%20Browning&amp;publication_year=2000&amp;journal=Nucl.%20Acids%20Res.&amp;volume=28&amp;pages=4488-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/28.22.4488" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F28.22.4488" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F28.22.4488"> </span></div> </div></div></div></div></div><div content-id="DSR028C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dybvig</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">French</div> <div class="given-names">C.T.,</div></div>,  <div class="name"><div class="surname">Voelker</div> <div class="given-names">L.L.</div></div>. </div><div class="article-title">Construction and use of derivatives of transposon Tn4001 that function in <em>Mycoplasma pulmonis</em> and <em>Mycoplasma arthritidis</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2000</div>, vol. <div class="volume">182</div> (pg. <div class="fpage">4343</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20and%20use%20of%20derivatives%20of%20transposon%20Tn4001%20that%20function%20in%20Mycoplasma%20pulmonis%20and%20Mycoplasma%20arthritidis&amp;author=K.%20Dybvig&amp;author=C.T.%2C%20French&amp;author=L.L.%20Voelker&amp;publication_year=2000&amp;journal=J.%20Bacteriol.&amp;volume=182&amp;pages=4343-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.182.15.4343-4347.2000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.182.15.4343-4347.2000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.182.15.4343-4347.2000"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10894746" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR028C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Halbedel</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Stulke</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Probing <em>in vivo</em> promoter activities in <em>Mycoplasma pneumoniae</em>: a system for generation of single-copy reporter constructs</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">2006</div>, vol. <div class="volume">72</div> (pg. <div class="fpage">1696</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Probing%20in%20vivo%20promoter%20activities%20in%20Mycoplasma%20pneumoniae%3A%20a%20system%20for%20generation%20of%20single-copy%20reporter%20constructs&amp;author=S.%20Halbedel&amp;author=J.%20Stulke&amp;publication_year=2006&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=72&amp;pages=1696-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.72.2.1696-1699.2006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.72.2.1696-1699.2006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.72.2.1696-1699.2006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16461731" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR028C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Janis</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Lartigue</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Frey</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Versatile use of <em>oriC</em> plasmids for functional genomics of <em>Mycoplasma capricolum</em> subsp. capricolum</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">2005</div>, vol. <div class="volume">71</div> (pg. <div class="fpage">2888</div>-<div class="lpage">93</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Versatile%20use%20of%20oriC%20plasmids%20for%20functional%20genomics%20of%20Mycoplasma%20capricolum%20subsp.%20capricolum&amp;author=C.%20Janis&amp;author=C.%20Lartigue&amp;author=J.%20Frey&amp;publication_year=2005&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=71&amp;pages=2888-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.71.6.2888-2893.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.71.6.2888-2893.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.71.6.2888-2893.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15932982" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR028C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gardner</div> <div class="given-names">S.W.,</div></div>,  <div class="name"><div class="surname">Minion</div> <div class="given-names">F.C.</div></div>. </div><div class="article-title">Detection and Quantification of intergenic transcription in <em>Mycoplasma hyopneumoniae</em></div>, <div class="source">Microbiology</div> , <div class="year">2010</div>, vol. <div class="volume">156</div> (pg. <div class="fpage">2305</div>-<div class="lpage">15</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20and%20Quantification%20of%20intergenic%20transcription%20in%20Mycoplasma%20hyopneumoniae&amp;author=S.W.%2C%20Gardner&amp;author=F.C.%20Minion&amp;publication_year=2010&amp;journal=Microbiology&amp;volume=156&amp;pages=2305-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/mic.0.038760-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.038760-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fmic.0.038760-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20430814" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR028C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Washio</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sasayama</div> <div class="given-names">J.,</div></div>,  <div class="name"><div class="surname">Tomita</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Analysis of complete genomes suggests that many prokaryotes do not rely on hairpin formation in transcription termination</div>, <div class="source">Nucl. Acids Res.</div> , <div class="year">1998</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">5456</div>-<div class="lpage">63</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20complete%20genomes%20suggests%20that%20many%20prokaryotes%20do%20not%20rely%20on%20hairpin%20formation%20in%20transcription%20termination&amp;author=T.%20Washio&amp;author=J.%2C%20Sasayama&amp;author=M.%20Tomita&amp;publication_year=1998&amp;journal=Nucl.%20Acids%20Res.&amp;volume=26&amp;pages=5456-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/26.23.5456" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F26.23.5456" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F26.23.5456"> </span></div> </div></div></div></div></div><div content-id="DSR028C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hoon</div> <div class="given-names">M.J.L.</div></div>,  <div class="name"><div class="surname">Makita</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Nakai</div> <div class="given-names">K.,</div></div>,  <div class="name"><div class="surname">Miyano</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Prediction of transcriptional terminators in <em>Bacillus subtilis</em> and related species</div>, <div class="source">PLoS Comput. Biol.</div> , <div class="year">2005</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">212</div>-<div class="lpage">21</div>)</div></div></div></div><div content-id="DSR028C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Madeira</div> <div class="given-names">H.M.F.,</div></div>,  <div class="name"><div class="surname">Gabriel</div> <div class="given-names">J.E.</div></div>. </div><div class="article-title">Regulation of gene expression in mycoplasmas: contribution from <em>Mycoplasma hyopneumoniae</em> and <em>Mycoplasma synoviae</em> genome sequences</div>, <div class="source">Genet. Mol. Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">277</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20gene%20expression%20in%20mycoplasmas%3A%20contribution%20from%20Mycoplasma%20hyopneumoniae%20and%20Mycoplasma%20synoviae%20genome%20sequences&amp;author=H.M.F.%2C%20Madeira&amp;author=J.E.%20Gabriel&amp;publication_year=2007&amp;journal=Genet.%20Mol.%20Biol.&amp;volume=30&amp;pages=277-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1590/S1415-47572007000200016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1590%2FS1415-47572007000200016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1590%2FS1415-47572007000200016"> </span></div> </div></div></div></div></div><div content-id="DSR028C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rutherford</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Parkhill</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Crook</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Artemis: sequence visualization and annotation</div>, <div class="source">Bioinformatics</div> , <div class="year">2000</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">944</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Artemis%3A%20sequence%20visualization%20and%20annotation&amp;author=K.%20Rutherford&amp;author=J.%20Parkhill&amp;author=J.%20Crook&amp;publication_year=2000&amp;journal=Bioinformatics&amp;volume=16&amp;pages=944-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/16.10.944" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F16.10.944" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F16.10.944"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11120685" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR028C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Carver</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Berriman</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tivey</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Artemis and ACT: viewing, annotating and comparing sequences stored in a relational database</div>, <div class="source">Bioinformatics</div> , <div class="year">2008</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">2672</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Artemis%20and%20ACT%3A%20viewing%2C%20annotating%20and%20comparing%20sequences%20stored%20in%20a%20relational%20database&amp;author=T.%20Carver&amp;author=M.%20Berriman&amp;author=A.%20Tivey&amp;publication_year=2008&amp;journal=Bioinformatics&amp;volume=24&amp;pages=2672-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btn529" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtn529" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtn529"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18845581" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR028C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Friis</div> <div class="given-names">N.F.</div></div>. </div><div class="article-title">Some recommendations concerning primary isolation of <em>Mycoplasma suipneumoniae</em> and <em>Mycoplasma flocculare</em> a survey</div>, <div class="source">Nordisk Veterinaer. Medicin.</div> , <div class="year">1975</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">337</div>-<div class="lpage">9</div>)</div></div></div></div><div content-id="DSR028C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sambrook</div> <div class="given-names">J.,</div></div>,  <div class="name"><div class="surname">Russell</div> <div class="given-names">D.W.</div></div>. </div>, <div class="source">Molecular Cloning a Laboratory Manual</div> , <div class="year">2001</div><div class="publisher-loc">New York</div><div class="publisher-name">Cold Spring Harbor Laboratory Press</div></div></div></div></div><div content-id="DSR028C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rogozin</div> <div class="given-names">I.B.</div></div>,  <div class="name"><div class="surname">Makarova</div> <div class="given-names">K.S.</div></div>,  <div class="name"><div class="surname">Murvai</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Czabarka</div> <div class="given-names">E.,</div></div>,  <div class="name"><div class="surname">Wolf</div> <div class="given-names">Y.I.</div></div>. </div><div class="article-title">Connected gene neighborhoods in prokaryotic genomes</div>, <div class="source">Nucl. Acids Res.</div> , <div class="year">2002</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">2212</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Connected%20gene%20neighborhoods%20in%20prokaryotic%20genomes&amp;author=I.B.%20Rogozin&amp;author=K.S.%20Makarova&amp;author=J.%20Murvai&amp;author=E.%2C%20Czabarka&amp;author=Y.I.%20Wolf&amp;publication_year=2002&amp;journal=Nucl.%20Acids%20Res.&amp;volume=30&amp;pages=2212-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/30.10.2212" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F30.10.2212" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F30.10.2212"> </span></div> </div></div></div></div></div><div content-id="DSR028C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Su</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Dam</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Palenik</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">Y.,</div></div>,  <div class="name"><div class="surname">Jianq</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Operon prediction by comparative genomics: an application to the Synechococcus sp. WH8102 genome</div>, <div class="source">Nucl. Acids Res.</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">2147</div>-<div class="lpage">57</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Operon%20prediction%20by%20comparative%20genomics%3A%20an%20application%20to%20the%20Synechococcus%20sp.%20WH8102%20genome&amp;author=X.%20Chen&amp;author=Z.%20Su&amp;author=P.%20Dam&amp;author=B.%20Palenik&amp;author=Y.%2C%20Xu&amp;author=T.%20Jianq&amp;publication_year=2004&amp;journal=Nucl.%20Acids%20Res.&amp;volume=32&amp;pages=2147-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh510" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh510" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh510"> </span></div> </div></div></div></div></div><div content-id="DSR028C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Price</div> <div class="given-names">M.N.</div></div>,  <div class="name"><div class="surname">Huang</div> <div class="given-names">K.H.</div></div>,  <div class="name"><div class="surname">Arkin</div> <div class="given-names">A.P.,</div></div>,  <div class="name"><div class="surname">Alm</div> <div class="given-names">E.J.</div></div>. </div><div class="article-title">A novel method for accurate operon predictions in all sequenced prokaryotes</div>, <div class="source">Nucl. Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">880</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20method%20for%20accurate%20operon%20predictions%20in%20all%20sequenced%20prokaryotes&amp;author=M.N.%20Price&amp;author=K.H.%20Huang&amp;author=A.P.%2C%20Arkin&amp;author=E.J.%20Alm&amp;publication_year=2005&amp;journal=Nucl.%20Acids%20Res.&amp;volume=33&amp;pages=880-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki232" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki232" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki232"> </span></div> </div></div></div></div></div><div content-id="DSR028C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Roback</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Beard</div> <div class="given-names">J.,</div></div>,  <div class="name"><div class="surname">Baumann</div> <div class="given-names">D.</div></div>. </div><div class="article-title">A predicted operon map for mycobacterium tuberculosis</div>, <div class="source">Nucl. Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">5085</div>-<div class="lpage">95</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20predicted%20operon%20map%20for%20mycobacterium%20tuberculosis&amp;author=P.%20Roback&amp;author=J.%2C%20Beard&amp;author=D.%20Baumann&amp;publication_year=2007&amp;journal=Nucl.%20Acids%20Res.&amp;volume=35&amp;pages=5085-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm518" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm518" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm518"> </span></div> </div></div></div></div></div><div content-id="DSR028C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cho</div> <div class="given-names">B.K.</div></div>,  <div class="name"><div class="surname">Zengler</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Qiu</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">The transcription unit architecture of the <em>Escherichia coli</em> genome</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2009</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">1043</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20transcription%20unit%20architecture%20of%20the%20Escherichia%20coli%20genome&amp;author=B.K.%20Cho&amp;author=K.%20Zengler&amp;author=Y.%20Qiu&amp;publication_year=2009&amp;journal=Nat.%20Biotechnol.&amp;volume=27&amp;pages=1043-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1582" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1582" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1582"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19881496" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR028C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Okuda</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kawashima</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kobayashi</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ogasawara</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Kanehisa</div> <div class="given-names">M.,</div></div>,  <div class="name"><div class="surname">Goto</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Characterization of relationships between transcriptional units and operon structures in <em>Bacillus subtilis</em> and <em>Escherichia coli</em></div>, <div class="source">BMC Genomics</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">48</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20relationships%20between%20transcriptional%20units%20and%20operon%20structures%20in%20Bacillus%20subtilis%20and%20Escherichia%20coli&amp;author=S.%20Okuda&amp;author=S.%20Kawashima&amp;author=K.%20Kobayashi&amp;author=N.%20Ogasawara&amp;author=M.%2C%20Kanehisa&amp;author=S.%20Goto&amp;publication_year=2007&amp;journal=BMC%20Genomics&amp;volume=8&amp;pages=48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-8-48" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-8-48" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-8-48"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17298663" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR028C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Adhya</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Suboperonic regulatory signals</div>, <div class="source">Sci. STKE</div> , <div class="year">2003</div>, vol. <div class="volume">185</div> pg. <div class="fpage">pe22</div>  <div class="comment">, 1–7</div></div></div></div></div><div content-id="DSR028C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Price</div> <div class="given-names">M.N.</div></div>,  <div class="name"><div class="surname">Arkin</div> <div class="given-names">A.P.,</div></div>,  <div class="name"><div class="surname">Alm</div> <div class="given-names">E.J.</div></div>. </div><div class="article-title">The life-cycle of operons</div>, <div class="source">PLoS Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">859</div>-<div class="lpage">73</div>)</div></div></div></div><div content-id="DSR028C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Sze</div> <div class="given-names">S.H.</div></div>. </div><div class="article-title">Large-scale analysis of gene clustering in bacteria</div>, <div class="source">Genome Res.</div> , <div class="year">2008</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">949</div>-<div class="lpage">56</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20analysis%20of%20gene%20clustering%20in%20bacteria&amp;author=Q.%20Yang&amp;author=S.H.%20Sze&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=15&amp;pages=949-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.072322.107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.072322.107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.072322.107"> </span></div> </div></div></div></div></div><div content-id="DSR028C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mao</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Dam</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Chou</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Olman</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">DOOR: a database for prokaryotic operons</div>, <div class="source">Nucl. Acids Res.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">459</div>-<div class="lpage">63</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DOOR%3A%20a%20database%20for%20prokaryotic%20operons&amp;author=F.%20Mao&amp;author=P.%20Dam&amp;author=J.%20Chou&amp;author=V.%20Olman&amp;author=Y.%20Xu&amp;publication_year=2009&amp;journal=Nucl.%20Acids%20Res.&amp;volume=37&amp;pages=459-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn757" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn757" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn757"> </span></div> </div></div></div></div></div><div content-id="DSR028C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bergman</div> <div class="given-names">N.H.</div></div>,  <div class="name"><div class="surname">Passalacqua</div> <div class="given-names">K.D.</div></div>,  <div class="name"><div class="surname">Hanna</div> <div class="given-names">P.C.,</div></div>,  <div class="name"><div class="surname">Qin</div> <div class="given-names">Z.S.</div></div>. </div><div class="article-title">Operon prediction for sequenced bacterial genomes without experimental information</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">2007</div>, vol. <div class="volume">73</div> (pg. <div class="fpage">846</div>-<div class="lpage">54</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Operon%20prediction%20for%20sequenced%20bacterial%20genomes%20without%20experimental%20information&amp;author=N.H.%20Bergman&amp;author=K.D.%20Passalacqua&amp;author=P.C.%2C%20Hanna&amp;author=Z.S.%20Qin&amp;publication_year=2007&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=73&amp;pages=846-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.01686-06" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.01686-06" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.01686-06"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17122389" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR028C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jian-Cheng</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Jin-Lin</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Jian-Hua</div> <div class="given-names">L.,</div></div>,  <div class="name"><div class="surname">Yi-Xue</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Prediction of prokaryotic promoters based on prediction of transcriptional units</div>, <div class="source">Acta Biochimica et Biophysica Sinica</div> , <div class="year">2003</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">317</div>-<div class="lpage">24</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20prokaryotic%20promoters%20based%20on%20prediction%20of%20transcriptional%20units&amp;author=L.%20Jian-Cheng&amp;author=X.%20Jin-Lin&amp;author=L.%2C%20Jian-Hua&amp;author=L.%20Yi-Xue&amp;publication_year=2003&amp;journal=Acta%20Biochimica%20et%20Biophysica%20Sinica&amp;volume=35&amp;pages=317-24" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12673384" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR028C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moreno-Hagelsieb</div> <div class="given-names">G.,</div></div>,  <div class="name"><div class="surname">Collado-Vides</div> <div class="given-names">J.</div></div>. </div><div class="article-title">A powerful nonhomology method for the prediction of operons in prokaryotes</div>, <div class="source">Bioinformatics</div> , <div class="year">2002</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">329</div>-<div class="lpage">36</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20powerful%20nonhomology%20method%20for%20the%20prediction%20of%20operons%20in%20prokaryotes&amp;author=G.%2C%20Moreno-Hagelsieb&amp;author=J.%20Collado-Vides&amp;publication_year=2002&amp;journal=Bioinformatics&amp;volume=18&amp;pages=329-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/18.suppl_1.S329" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F18.suppl_1.S329" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F18.suppl_1.S329"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11847085" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR028C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR028C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brouwer</div> <div class="given-names">R.W.W.</div></div>,  <div class="name"><div class="surname">Kuipers</div> <div class="given-names">O.P.,</div></div>,  <div class="name"><div class="surname">Van Hijum</div> <div class="given-names">S.A.F.T.</div></div>. </div><div class="article-title">The relative value of operon predictions</div>, <div class="source">Brief Bioinform.</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">367</div>-<div class="lpage">75</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20relative%20value%20of%20operon%20predictions&amp;author=R.W.W.%20Brouwer&amp;author=O.P.%2C%20Kuipers&amp;author=S.A.F.T.%20Van%20Hijum&amp;publication_year=2008&amp;journal=Brief%20Bioinform.&amp;volume=9&amp;pages=367-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bib/bbn019" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbib%2Fbbn019" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbib%2Fbbn019"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18420711" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2011. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=476838&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr028/2/dsr028-dsr028supp_table6.xls?Expires=2147483647&amp;Signature=4~l-D7Nl76eVcOeW8KNq1W36B-wP3mQgFILR8jtS4SNoKyvhoFre8eZcAx~K12gkvEjWrHNXPNebTC5QN2nRyViXH~0RK~8IAU~EpJvLEGy3YxkOPyJ0F6Z-lq3xIts132-SMkQUT~LPNVQ0yqE1L4US-rKcQ35DVNfzcjuBliHoAW4GpovGWCB03V-EL6dNyon7FUZ~qU9bI8A5hc4DospwcmPhfy-xLJCYs4OyZkGHj-X-OSn3xMpaxMbfrKVbPHFoUvVCgwrJRbyVtC9jZWpznPX6K5NVDESE2brhpMn0Af0FJ9J8SdUbZ2HR4ZvhhboB4PXKxL2t-yB7h8jRTg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr028/2/dsr028-dsr028supp_table5.xls?Expires=2147483647&amp;Signature=FLaIOH-GC4EKFCUn2iMwZ8CP0hIiorPJcSujHXyqp2kFEVHlD4OgXZ-HtTNRg1jw4swLyxOd53DnsdapGvu5piR-lN~B~WlrWi6aSKYDddc6G-pTsr4P4EV5EWUQmfFFpCsAf-sut2xNOOCoeiUSg51Iwk6MiOGEc6MPa7vEf~y8ctLETOSb~3yyFQ6vd6zrCQaB5mkzaSwM8cdQHghdo9xuR1FXLx5AiaCDI6tBV3DNNf5RElMMFTjjD6AzwYRwTzKBKVeajoxQau6DRCPXszrCz8ZuikAP35cb520A9n7rvlnN0xfrsSZjDbtwYwr2iR0~-IIFGxfDWlyha8B-vw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr028/2/dsr028-dsr028supp_table4.xls?Expires=2147483647&amp;Signature=c1y5AO2hihFXJkCyn0opdoPgghga1TI~5MwAuzeytUsjlgmXlAcLXrygrxE3gFqRk5nk9I6b30DUAC1zQgjvuL17uC2V4sppKopymPWNs6Awe1mBHUC-Sr3eLDv79Xk2VEtYIrRXfjycxrg1YcoCyCgFjQcnUfxesihDjIz7ng3pHIv2e-0E0qPtDUboObW7HQ7mUhm0VVny7zU77DvNhq-FMNVkM6KBVPyBfoMt8e58J52OYG6KkiYpo6NgKlFEj38pEFhCA9hX0ztPbQraMYUkKg1sqyPc8~-rFH7N6oyMwg~TekrwgQUFrIlpedPxbwvnrpShPuQaR5PTTwQAOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr028/2/dsr028-dsr028supp_table3.xls?Expires=2147483647&amp;Signature=PogBQHd91TXNlzdG4zAHa0ZoosPV1XywH08W98i6K8g2UDu~23muKUZiEMXtOuDUwUoz855~B4TndyjhqT13BXdU5hkV2oJMgl-55nafPg9RhdTR6AGyWyxCubau5XyzgehngFurjQlmsjI05-5GwbyqOZpZ8jqtYa30tJZa2ARYh7yfQxMij8~2gVsClH7RAtvLlSRCg~wKOqRChsiXiUbqk-yJclGZ9O9CB00ChHAUZzbE-3aTrMKc~tRwASBe9c8vaaUpQbhyAl8jCPaxZkHGM2e-Gid8pHPQ4gkbBLKYJy0UK0IyLYR4fl20KplFsTbzPv3nh5h4XzfEQ1zqTg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr028/2/dsr028-dsr028supp_table2.xls?Expires=2147483647&amp;Signature=Xouw6Rx3zSUTOGmhEgz6dID6vf9MihjyERfuTKFGdiJhaqMZKoPVRB~65CnctX2uChZ4ZudXgmAiG5N~XDLUXFZN4xdKt-7H-h8uqwYlfuRDSdbbMxusb-8QnneMlmvOy9b~tnQRVfDmKLkEAYa4okTeA3mn8PS46YT1JylkrzPG1ozEVZDVfYxcnGg0tVJ~bGYr4XsZ2BjA-97P-JZ8OQhhNcOtQo3Qr~5ZuniydOyWj1JKDlko-h11Wh1iHHLcCh3SbKquohErcH3cgbxG5Uldhh2ZhGUBMKaJY5nij~LR9~UFReiGjqa80iVKa2U0aLJaxKQymVgmvQQKw0fR3Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr028/2/dsr028-dsr028supp_table1.xls?Expires=2147483647&amp;Signature=e6BeoG4H~OqeptqqwxvNeogEVYoJcrlaBaj-vw-8mYwixYGo-1tllfz5K3qHr~oDv0DNgnyBcFSfXmAPTzyHQFvIWDRO9Vlk5c6gG1DPmeAWphXQirb2Uftc8d9H~oEY5w18kVJS4RKgEwcOncH4JgVFSr3OfAB1KIlth857Pf9dC73pPOO6u8I~uYCNCXCnd5Wk8QM0kNSwcM4enymnjAVZEuYoE9Wpi07Jbrh-Wv1ZKKm3xBfCcjDdgzZS4eAGlE39H0Vxdi4IXnkPUKyz~mRB90VFCk4oVt-Inr7KaXVZZQahdNj9XJ8ip2NUbze4xLRDKxQwJ3XI2r0WwISy9g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/6/10.1093/dnares/dsr028/2/dsr028-dsr028supp.doc?Expires=2147483647&amp;Signature=v0MN06i2Cwqhc7c2ksC~VJz0H6TnDapRwYrhEpFr8yu79oigc2ZtDHnsmm78zEi-0cWODEIvj7ihgPt2L1cVzE2CeKA7vQGv7aMX8CGxt5v-M-uBik56is94xO3KuoFFsTE5MfEkx59S2mNAKpt~Ecf4ZZpBoBSczkr3dQpG9eMWOggVBD~wU6wjbuOTwOCxHPxe-DKBmIL7krdHhopMbqjwT~JUEtNmL8LpwxyiAnK8hgQavwS~cISNfKRgjC6MDj6sImCPyXIc1J~Q8lU5ChRHO0BydtTRg6~mIn~~rdKwtVKxBVO9cvd4PMt30PkGJfb-Skic~veysC1OAz3YZg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">111</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000297990700001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">9</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsr028"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="476838"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000297990700001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f18%2f6%2f413%2f476838">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000297990700001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (9)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F18%2F6%2F413%2F476838" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/476838'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Mycoplasma hyopneumoniae Transcription Unit Organization: Genome Survey and Prediction" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsr028");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f18%2f6%2f413%2f476838">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

