

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/6/649/4077064","@type":"ScholarlyArticle","name":"Isolation of highly thermostable β-xylosidases from a hot spring soil microbial community using a metagenomic approach","datePublished":"2017-08-07","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2017-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx032","keywords":["thermostable enzyme, β<bold>-</bold>xylosidase, metagenome, xylan-degradation"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/6/649/4077064","author":[{"name":"Sato, Masaru","affiliation":"Department of Biotechnology Research and Development, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan","@type":"Person"},{"name":"Suda, Migiwa","affiliation":"Department of Biotechnology Research and Development, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan","@type":"Person"},{"name":"Okuma, Jiro","affiliation":"Honda Research Institute Japan Co., Ltd., Wako, Saitama 351-0188, Japan","@type":"Person"},{"name":"Kato, Tomohiko","affiliation":"Department of Biotechnology Research and Development, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan","@type":"Person"},{"name":"Hirose, Yoshitsugu","affiliation":"Honda Research Institute Japan Co., Ltd., Wako, Saitama 351-0188, Japan","@type":"Person"},{"name":"Nishimura, Asuka","affiliation":"Honda Research Institute Japan Co., Ltd., Wako, Saitama 351-0188, Japan","@type":"Person"},{"name":"Kondo, Yasuhiko","affiliation":"Honda Research Institute Japan Co., Ltd., Wako, Saitama 351-0188, Japan","@type":"Person"},{"name":"Shibata, Daisuke","affiliation":"Department of Biotechnology Research and Development, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan","@type":"Person"}],"description":"Abstract.  The DNA extracted from a high-temperature environment in which micro-organisms are living will be a good source for the isolation of thermostable enz","pageStart":"649","pageEnd":"656","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/6/2/m_dnares_24_6cover.png?Expires=2147483647&Signature=z4yCzT-6enr7Fof9QqkUJvtPQAXDRV5XYB7R7L2HBbQZ2iaySlHxBG~hDePAa-rY6Gn-PH9W8mIlJwAyG9R2RE6Vd3RTDct~I7biyNGgXk9CP54tGn91x8VYIMO24kwuQBqahsZQMv3NxRiQKhbGI0yEDanozvBgG~-9FpaC12JFLE6A4BT6rBzjsJLh9UJqvGv0F2hWW2jLvdITi~5PoATpRIsbIK-S1wcvqp49OSN~oRepZtr6iz~107ho8gZAxoaG8nb4DvXKrvfnCuhhmbRD41NP3K2zFyztHicWtTYb1xunX52RqodNq-M3zVvUit2rSw5-mRSvm-jXb-p6nw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Isolation of highly thermostable β-xylosidases from a hot spring soil microbial community using a metagenomic approach","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/6/2/m_dnares_24_6cover.png?Expires=2147483647&Signature=z4yCzT-6enr7Fof9QqkUJvtPQAXDRV5XYB7R7L2HBbQZ2iaySlHxBG~hDePAa-rY6Gn-PH9W8mIlJwAyG9R2RE6Vd3RTDct~I7biyNGgXk9CP54tGn91x8VYIMO24kwuQBqahsZQMv3NxRiQKhbGI0yEDanozvBgG~-9FpaC12JFLE6A4BT6rBzjsJLh9UJqvGv0F2hWW2jLvdITi~5PoATpRIsbIK-S1wcvqp49OSN~oRepZtr6iz~107ho8gZAxoaG8nb4DvXKrvfnCuhhmbRD41NP3K2zFyztHicWtTYb1xunX52RqodNq-M3zVvUit2rSw5-mRSvm-jXb-p6nw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Isolation of highly thermostable β-xylosidases from a hot spring soil microbial community using a metagenomic approach" />
    <meta property="og:description" content="Abstract.  The DNA extracted from a high-temperature environment in which micro-organisms are living will be a good source for the isolation of thermostable enz" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/6/649/4077064" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-08-07" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Sato, Masaru" /><meta name="citation_author_institution" content="Department of Biotechnology Research and Development, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Suda, Migiwa" /><meta name="citation_author_institution" content="Department of Biotechnology Research and Development, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Okuma, Jiro" /><meta name="citation_author_institution" content="Honda Research Institute Japan Co., Ltd., Wako, Saitama 351-0188, Japan " /><meta name="citation_author" content="Kato, Tomohiko" /><meta name="citation_author_institution" content="Department of Biotechnology Research and Development, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Hirose, Yoshitsugu" /><meta name="citation_author_institution" content="Honda Research Institute Japan Co., Ltd., Wako, Saitama 351-0188, Japan " /><meta name="citation_author" content="Nishimura, Asuka" /><meta name="citation_author_institution" content="Honda Research Institute Japan Co., Ltd., Wako, Saitama 351-0188, Japan " /><meta name="citation_author" content="Kondo, Yasuhiko" /><meta name="citation_author_institution" content="Honda Research Institute Japan Co., Ltd., Wako, Saitama 351-0188, Japan " /><meta name="citation_author" content="Shibata, Daisuke" /><meta name="citation_author_institution" content="Department of Biotechnology Research and Development, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_title" content="Isolation of highly thermostable β-xylosidases from a hot spring soil microbial community using a metagenomic approach" /><meta name="citation_firstpage" content="649" /><meta name="citation_lastpage" content="656" /><meta name="citation_doi" content="10.1093/dnares/dsx032" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2017/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=A review of lignocellulose bioconversion using enzymatic hydrolysis and synergistic cooperation between enzymes-Factors affecting enzymes, conversion and synergy;  citation_journal_title=Biotechnol. Adv;  citation_year=2011;  citation_volume=30;  citation_pages=1458-80; " /><meta name="citation_reference" content="citation_title=BRENDA in 2013: integrated reactions, kinetic data, enzyme function data, improved disease classification: new options and contents in BRENDA,;  citation_journal_title=Nucleic Acids Res;  citation_year=2013;  citation_volume=41;  citation_pages=D764-72; " /><meta name="citation_reference" content="citation_title=Developments in industrially important thermostable enzymes: a review,;  citation_journal_title=Bioresour. Technol;  citation_year=2003;  citation_volume=89;  citation_pages=17-34; " /><meta name="citation_reference" content="citation_title=Expression and characterization of a thermostable beta-xylosidase from the hyperthermophile, Thermotoga maritima,;  citation_journal_title=Biotechnol. Lett;  citation_year=2004;  citation_volume=26;  citation_pages=1511-5; " /><meta name="citation_reference" content="citation_title=Biochemical properties of a novel thermostable and highly xylose-tolerant β-xylosidase/α-arabinosidase from Thermotoga thermarum,;  citation_journal_title=Biotechnol. Biofuels;  citation_year=2013;  citation_volume=6;  citation_pages=27" /><meta name="citation_reference" content="citation_title=Reconstitution of a thermostable xylan-degrading enzyme mixture from the bacterium Caldicellulosiruptor bescii,;  citation_journal_title=Appl. Environ. Microbiol;  citation_year=2013;  citation_volume=79;  citation_pages=1481-90; " /><meta name="citation_reference" content="citation_title=Purification and characterization of a thermostable β-xylosidase from Thermoanaerobacter ethanolicus,;  citation_journal_title=J. Bacteriol;  citation_year=1992;  citation_volume=174;  citation_pages=5848-53; " /><meta name="citation_reference" content="citation_title=Gene cloning and expression in Escherichia coli of a bi-functional beta-D-xylosidase/alpha-L-arabinosidase from Sulfolobus solfataricus involved in xylan degradation,;  citation_journal_title=Extremophiles;  citation_year=2007;  citation_volume=11;  citation_pages=123-32; " /><meta name="citation_reference" content="citation_title=Purification and characterization of β-xylosidase from Aureobasidium pullulans,;  citation_journal_title=Appl. Microbiol. Biotechnol;  citation_year=1991;  citation_volume=35;  citation_pages=210-5; " /><meta name="citation_reference" content="citation_title=Metagenomics: a promissing approach to assess enzymes biocatalyst for biofuel production,;  citation_journal_title=Asian J. Biotechnol;  citation_year=2013;  citation_volume=5;  citation_pages=33-50; " /><meta name="citation_reference" content="citation_title=Targeted discovery of glycoside hydrolases from a switchgrass-adapted compost community,;  citation_journal_title=PLoS One;  citation_year=2010;  citation_volume=5;  citation_pages=e8812" /><meta name="citation_reference" content="citation_title=Cloning and characterization of two β-glucosidase/xylosidase enzymes from yak rumen metagenome,;  citation_journal_title=Appl. Biochem. Biotechnol;  citation_year=2012;  citation_volume=166;  citation_pages=72-86; " /><meta name="citation_reference" content="citation_title=MetaGeneAnnotator: detecting species-specific patterns of ribosomal binding site for precise gene prediction in anonymous prokaryotic and phage genomes,;  citation_journal_title=DNA Res;  citation_year=2008;  citation_volume=15;  citation_pages=387-96; " /><meta name="citation_reference" content="citation_title=Clustal W and Clustal X version 2.0,;  citation_journal_title=Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=2947-8; " /><meta name="citation_reference" content="citation_title=Purification and characterization of a GH43 β-xylosidase from Enterobacter sp. identified and cloned from forest soil bacteria;  citation_journal_title=Microbiol. Res;  citation_year=2014;  citation_volume=169;  citation_pages=213-20; " /><meta name="citation_reference" content="citation_title=Effective induction, purification and characterization of Trichoderma koningii G-39 beta-xylosidase with high transferase activity;  citation_journal_title=Biotechnol. Appl. Biochem;  citation_year=2000;  citation_volume=31;  citation_pages=119-25; " /><meta name="citation_reference" content="citation_title=Functional metagenomics reveals novel β-galactosidases not predictable from gene sequences;  citation_journal_title=PLoS One;  citation_year=2017;  citation_volume=12;  citation_pages=e0172545." /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/6/649/22295860/dsx032.pdf" /><meta name="description" content="Abstract.  The DNA extracted from a high-temperature environment in which micro-organisms are living will be a good source for the isolation of thermostable enz" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/6/649/4077064" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/6/649/4077064" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/6/649/4077064");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsx032';
    </script>
<title>
	Isolation of highly thermostable β-xylosidases from a hot spring soil microbial community using a metagenomic approach | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./4077064" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bb46PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYXRvLCBNYXNhcnUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb3RlY2hub2xvZ3kgUmVzZWFyY2ggYW5kIERldmVsb3BtZW50LCBLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlN1ZGEsIE1pZ2l3YSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvdGVjaG5vbG9neSBSZXNlYXJjaCBhbmQgRGV2ZWxvcG1lbnQsIEthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iT2t1bWEsIEppcm8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJIb25kYSBSZXNlYXJjaCBJbnN0aXR1dGUgSmFwYW4gQ28uLCBMdGQuLCBXYWtvLCBTYWl0YW1hIDM1MS0wMTg4LCBKYXBhbiAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLYXRvLCBUb21vaGlrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvdGVjaG5vbG9neSBSZXNlYXJjaCBhbmQgRGV2ZWxvcG1lbnQsIEthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGlyb3NlLCBZb3NoaXRzdWd1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSG9uZGEgUmVzZWFyY2ggSW5zdGl0dXRlIEphcGFuIENvLiwgTHRkLiwgV2FrbywgU2FpdGFtYSAzNTEtMDE4OCwgSmFwYW4gIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmlzaGltdXJhLCBBc3VrYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkhvbmRhIFJlc2VhcmNoIEluc3RpdHV0ZSBKYXBhbiBDby4sIEx0ZC4sIFdha28sIFNhaXRhbWEgMzUxLTAxODgsIEphcGFuICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IktvbmRvLCBZYXN1aGlrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkhvbmRhIFJlc2VhcmNoIEluc3RpdHV0ZSBKYXBhbiBDby4sIEx0ZC4sIFdha28sIFNhaXRhbWEgMzUxLTAxODgsIEphcGFuICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNoaWJhdGEsIERhaXN1a2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb3RlY2hub2xvZ3kgUmVzZWFyY2ggYW5kIERldmVsb3BtZW50LCBLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iSXNvbGF0aW9uIG9mIGhpZ2hseSB0aGVybW9zdGFibGUgzrIteHlsb3NpZGFzZXMgZnJvbSBhIGhvdCBzcHJpbmcgc29pbCBtaWNyb2JpYWwgY29tbXVuaXR5IHVzaW5nIGEgbWV0YWdlbm9taWMgYXBwcm9hY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI2NDkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjY1NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzeDAzMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE3LzEyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSByZXZpZXcgb2YgbGlnbm9jZWxsdWxvc2UgYmlvY29udmVyc2lvbiB1c2luZyBlbnp5bWF0aWMgaHlkcm9seXNpcyBhbmQgc3luZXJnaXN0aWMgY29vcGVyYXRpb24gYmV0d2VlbiBlbnp5bWVzLUZhY3RvcnMgYWZmZWN0aW5nIGVuenltZXMsIGNvbnZlcnNpb24gYW5kIHN5bmVyZ3k7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb3RlY2hub2wuIEFkdjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTE0NTgtODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJSRU5EQSBpbiAyMDEzOiBpbnRlZ3JhdGVkIHJlYWN0aW9ucywga2luZXRpYyBkYXRhLCBlbnp5bWUgZnVuY3Rpb24gZGF0YSwgaW1wcm92ZWQgZGlzZWFzZSBjbGFzc2lmaWNhdGlvbjogbmV3IG9wdGlvbnMgYW5kIGNvbnRlbnRzIGluIEJSRU5EQSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9RDc2NC03MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnRzIGluIGluZHVzdHJpYWxseSBpbXBvcnRhbnQgdGhlcm1vc3RhYmxlIGVuenltZXM6IGEgcmV2aWV3LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvcmVzb3VyLiBUZWNobm9sOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTg5OyAgY2l0YXRpb25fcGFnZXM9MTctMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cHJlc3Npb24gYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgYSB0aGVybW9zdGFibGUgYmV0YS14eWxvc2lkYXNlIGZyb20gdGhlIGh5cGVydGhlcm1vcGhpbGUsIFRoZXJtb3RvZ2EgbWFyaXRpbWEsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW90ZWNobm9sLiBMZXR0OyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9MTUxMS01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CaW9jaGVtaWNhbCBwcm9wZXJ0aWVzIG9mIGEgbm92ZWwgdGhlcm1vc3RhYmxlIGFuZCBoaWdobHkgeHlsb3NlLXRvbGVyYW50IM6yLXh5bG9zaWRhc2UvzrEtYXJhYmlub3NpZGFzZSBmcm9tIFRoZXJtb3RvZ2EgdGhlcm1hcnVtLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvdGVjaG5vbC4gQmlvZnVlbHM7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTI3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVjb25zdGl0dXRpb24gb2YgYSB0aGVybW9zdGFibGUgeHlsYW4tZGVncmFkaW5nIGVuenltZSBtaXh0dXJlIGZyb20gdGhlIGJhY3Rlcml1bSBDYWxkaWNlbGx1bG9zaXJ1cHRvciBiZXNjaWksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BcHBsLiBFbnZpcm9uLiBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9Nzk7ICBjaXRhdGlvbl9wYWdlcz0xNDgxLTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QdXJpZmljYXRpb24gYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgYSB0aGVybW9zdGFibGUgzrIteHlsb3NpZGFzZSBmcm9tIFRoZXJtb2FuYWVyb2JhY3RlciBldGhhbm9saWN1cyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEJhY3RlcmlvbDsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT0xNzQ7ICBjaXRhdGlvbl9wYWdlcz01ODQ4LTUzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIGNsb25pbmcgYW5kIGV4cHJlc3Npb24gaW4gRXNjaGVyaWNoaWEgY29saSBvZiBhIGJpLWZ1bmN0aW9uYWwgYmV0YS1ELXh5bG9zaWRhc2UvYWxwaGEtTC1hcmFiaW5vc2lkYXNlIGZyb20gU3VsZm9sb2J1cyBzb2xmYXRhcmljdXMgaW52b2x2ZWQgaW4geHlsYW4gZGVncmFkYXRpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FeHRyZW1vcGhpbGVzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MTIzLTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QdXJpZmljYXRpb24gYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgzrIteHlsb3NpZGFzZSBmcm9tIEF1cmVvYmFzaWRpdW0gcHVsbHVsYW5zLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QXBwbC4gTWljcm9iaW9sLiBCaW90ZWNobm9sOyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9MjEwLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1ldGFnZW5vbWljczogYSBwcm9taXNzaW5nIGFwcHJvYWNoIHRvIGFzc2VzcyBlbnp5bWVzIGJpb2NhdGFseXN0IGZvciBiaW9mdWVsIHByb2R1Y3Rpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bc2lhbiBKLiBCaW90ZWNobm9sOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0zMy01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGFyZ2V0ZWQgZGlzY292ZXJ5IG9mIGdseWNvc2lkZSBoeWRyb2xhc2VzIGZyb20gYSBzd2l0Y2hncmFzcy1hZGFwdGVkIGNvbXBvc3QgY29tbXVuaXR5LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPWU4ODEyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2xvbmluZyBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiB0d28gzrItZ2x1Y29zaWRhc2UveHlsb3NpZGFzZSBlbnp5bWVzIGZyb20geWFrIHJ1bWVuIG1ldGFnZW5vbWUsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BcHBsLiBCaW9jaGVtLiBCaW90ZWNobm9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE2NjsgIGNpdGF0aW9uX3BhZ2VzPTcyLTg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZXRhR2VuZUFubm90YXRvcjogZGV0ZWN0aW5nIHNwZWNpZXMtc3BlY2lmaWMgcGF0dGVybnMgb2Ygcmlib3NvbWFsIGJpbmRpbmcgc2l0ZSBmb3IgcHJlY2lzZSBnZW5lIHByZWRpY3Rpb24gaW4gYW5vbnltb3VzIHByb2thcnlvdGljIGFuZCBwaGFnZSBnZW5vbWVzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTM4Ny05NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2x1c3RhbCBXIGFuZCBDbHVzdGFsIFggdmVyc2lvbiAyLjAsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTI5NDctODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHVyaWZpY2F0aW9uIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIGEgR0g0MyDOsi14eWxvc2lkYXNlIGZyb20gRW50ZXJvYmFjdGVyIHNwLiBpZGVudGlmaWVkIGFuZCBjbG9uZWQgZnJvbSBmb3Jlc3Qgc29pbCBiYWN0ZXJpYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWljcm9iaW9sLiBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTY5OyAgY2l0YXRpb25fcGFnZXM9MjEzLTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FZmZlY3RpdmUgaW5kdWN0aW9uLCBwdXJpZmljYXRpb24gYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgVHJpY2hvZGVybWEga29uaW5naWkgRy0zOSBiZXRhLXh5bG9zaWRhc2Ugd2l0aCBoaWdoIHRyYW5zZmVyYXNlIGFjdGl2aXR5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW90ZWNobm9sLiBBcHBsLiBCaW9jaGVtOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MTE5LTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdW5jdGlvbmFsIG1ldGFnZW5vbWljcyByZXZlYWxzIG5vdmVsIM6yLWdhbGFjdG9zaWRhc2VzIG5vdCBwcmVkaWN0YWJsZSBmcm9tIGdlbmUgc2VxdWVuY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPWUwMTcyNTQ1LiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yNC82LzY0OS8yMjI5NTg2MC9kc3gwMzIucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFRoZSBETkEgZXh0cmFjdGVkIGZyb20gYSBoaWdoLXRlbXBlcmF0dXJlIGVudmlyb25tZW50IGluIHdoaWNoIG1pY3JvLW9yZ2FuaXNtcyBhcmUgbGl2aW5nIHdpbGwgYmUgYSBnb29kIHNvdXJjZSBmb3IgdGhlIGlzb2xhdGlvbiBvZiB0aGVybW9zdGFibGUgZW56IiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj00MDc3MDY0JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZMxXX6PmHUzq/LYjvTQFeYB9Ry2C" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f6%2f649%2f4077064" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f6%2f649%2f4077064" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=103821" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/6/2/m_dnares_24_6cover.png?Expires=2147483647&amp;Signature=z4yCzT-6enr7Fof9QqkUJvtPQAXDRV5XYB7R7L2HBbQZ2iaySlHxBG~hDePAa-rY6Gn-PH9W8mIlJwAyG9R2RE6Vd3RTDct~I7biyNGgXk9CP54tGn91x8VYIMO24kwuQBqahsZQMv3NxRiQKhbGI0yEDanozvBgG~-9FpaC12JFLE6A4BT6rBzjsJLh9UJqvGv0F2hWW2jLvdITi~5PoATpRIsbIK-S1wcvqp49OSN~oRepZtr6iz~107ho8gZAxoaG8nb4DvXKrvfnCuhhmbRD41NP3K2zFyztHicWtTYb1xunX52RqodNq-M3zVvUit2rSw5-mRSvm-jXb-p6nw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/6">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="104086788">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086788">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086790">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086790">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086794">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086794">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086814">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086814">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086836">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086836">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086842">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086842">Data availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086844">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086844">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086852">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086852">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/6/635/3979487">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Isolation of highly thermostable β-xylosidases from a hot spring soil microbial community using a metagenomic approach
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masaru Sato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Biotechnology Research and Development, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masaru+Sato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sato M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sato M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Masaru Sato</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Migiwa Suda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Biotechnology Research and Development, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Migiwa+Suda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Suda M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Suda M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Migiwa Suda</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jiro Okuma</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Honda Research Institute Japan Co., Ltd., Wako, Saitama 351-0188, Japan </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jiro+Okuma">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Okuma J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Okuma J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jiro Okuma</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tomohiko Kato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Biotechnology Research and Development, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tomohiko+Kato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kato T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kato T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tomohiko Kato</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshitsugu Hirose</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Honda Research Institute Japan Co., Ltd., Wako, Saitama 351-0188, Japan </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshitsugu+Hirose">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hirose Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hirose Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yoshitsugu Hirose</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Asuka Nishimura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Honda Research Institute Japan Co., Ltd., Wako, Saitama 351-0188, Japan </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Asuka+Nishimura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nishimura A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nishimura A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Asuka Nishimura</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yasuhiko Kondo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Honda Research Institute Japan Co., Ltd., Wako, Saitama 351-0188, Japan </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yasuhiko+Kondo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kondo Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kondo Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yasuhiko Kondo</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Daisuke Shibata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Biotechnology Research and Development, Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx032-cor1" data-open="dsx032-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. 81 438 52 3947. Fax: +81 438 52 3948. Email: <a href="mailto:shibata@kazusa.or.jp" target="_blank">shibata@kazusa.or.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Daisuke+Shibata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shibata D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shibata D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Daisuke Shibata</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx032-FM1"><div class="fn"><div class="footnote-content">Present address: Graduate School of Agricultural and Life Sciences, The University of Tokyo, 1-1-1 Yayoi, Bunkyo-ku, Tokyo 113-8657, Japan.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx032-FM2"><div class="fn"><div class="footnote-content">Edited by Dr. Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 6, 1 December 2017, Pages 649–656, <a href='https://doi.org/10.1093/dnares/dsx032'>https://doi.org/10.1093/dnares/dsx032</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">07 August 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">17 February 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">31 July 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="4077064" href="/dnaresearch/article-pdf/24/6/649/22295860/dsx032.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Masaru Sato, Migiwa Suda, Jiro Okuma, Tomohiko Kato, Yoshitsugu Hirose, Asuka Nishimura, Yasuhiko Kondo, Daisuke Shibata;  Isolation of highly thermostable β-xylosidases from a hot spring soil microbial community using a metagenomic approach, <em>DNA Research</em>, Volume 24, Issue 6, 1 December 2017, Pages 649–656, <a href='https://doi.org/10.1093/dnares/dsx032'>https://doi.org/10.1093/dnares/dsx032</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/4077064?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/4077064?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/4077064?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/4077064?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/4077064?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/4077064?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/4077064?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Isolation%20of%20highly%20thermostable%20%CE%B2-xylosidases%20from%20a%20hot%20spring%20soil%20microbial%20community%20using%20a%20metagenomic%20approach&amp;publicationDate=2017-08-07&amp;volumeNum=24&amp;issueNum=6&amp;author=Sato%2C%20Masaru%3B%20Suda%2C%20Migiwa&amp;startPage=649&amp;endPage=656&amp;contentId=10.1093%2Fdnares%2Fdsx032&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f6%2f649%2f4077064" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=104086788 id="104086788" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The DNA extracted from a high-temperature environment in which micro-organisms are living will be a good source for the isolation of thermostable enzymes. Using a metagenomic approach, we aimed to isolate thermostable β-xylosidases that will be exploited for biofuel production from lignocellulosic biomass. DNA samples obtained from the soil near a spout of a hot spring (70°C, pH7.2) were subjected to sequencing, which generated a total of 84.2 Gbp with 967,925 contigs of &gt;500 bp in length. Similarity search for β-xylosidase in the contigs revealed the presence of 168 candidate sequences, each of which may have arisen from more than one gene. Individual genes were amplified by PCR using sequence-specific primers. The resultant DNA fragments were cloned and introduced into <em>Escherichia coli</em> BL21 Star(DE3). Consequently, 269 proteins were successfully expressed in the <em>E. coli</em> cells and then examined for β-xylosidase activity. A total of 82 proteins exhibited β-xylosidase activity at 50°C, six of which retained the activity even at 90°C. Out of the six, three proteins were originated from a single candidate sequence, AR19M-311. An amino acid sequence comparison suggested the amino acid residues that appeared to be crucial for thermal stability of the enzymes.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;thermostable enzyme, β-xylosidase, metagenome, xylan-degradation&quot;">thermostable enzyme, β<strong>-</strong>xylosidase, metagenome, xylan-degradation</a></div>                            <h2 scrollto-destination=104086790 id="104086790" class="section-title" >1. Introduction</h2>
<p>β-Xylosidases, which cleave the β-1,4-linkage of xylan from the non-reducing terminus to release <span class="small-caps">d</span>-xylose, have received attention in the past decade with respect to biofuel production from lignocellulosic biomass due to their potential to degrade hemicellulose, the second most abundant lignocellulose biomass, coordinately with endoxylanases and some glycosidases, such as arabionosidases and gluclonidases.<a reveal-id="dsx032-B1" data-open="dsx032-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> Although the sequences of over 800 β-xylosidases have been registered in the enzyme database BRENDA,<a reveal-id="dsx032-B2" data-open="dsx032-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> there is interest in the identification of thermostable β-xylosidases, because enzymatic processing at high temperatures prevents contamination with saprophytic bacteria and lowers the viscosity of the reaction mixture, thus increasing the high reaction efficiency and reducing energy consumption during agitation of the mixture.<a reveal-id="dsx032-B3" data-open="dsx032-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> Thermostable β-xylosidases have been isolated from thermophilic micro-organisms from hot compost material and hot spring soils. The thermophilic bacteria <em>Thermotoga maritima</em> and <em>T. thermarum</em> produce β-xylosidases active at temperatures above 90 °C.<a reveal-id="dsx032-B4" data-open="dsx032-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx032-B5" data-open="dsx032-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> Thermostable β-xylosidases from <em>Caldicellulosiruptor bescii</em> and <em>Thermoanaerobacter ethanolicus</em> have also been reported.<a reveal-id="dsx032-B6" data-open="dsx032-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsx032-B7" data-open="dsx032-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> A thermostable β-xylosidase exhibiting an optimum reaction temperature of 85 °C was isolated from the archea <em>Sulfolobus solfataricus</em><em>.</em><a reveal-id="dsx032-B8" data-open="dsx032-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> A β-xylosidase isolated from the fungus <em>Aureobasidium pullulans</em> was reported to be active at 80 °C.<a reveal-id="dsx032-B9" data-open="dsx032-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> No amino acid sequence motifs clearly associated with the thermal stability of these thermostable enzymes have been identified, however. The characterization of additional thermostable β-xylosidases will hopefully enhance our understanding of the molecular features associated with thermal stability, which in turn could facilitate genetic engineering of hydrolytic enzymes suitable for industrial use.</p><p>Recent advancements in next-generation sequencing (NGS) technologies now enable researchers to more easily access the vast stores of DNA sequence data for mixed micro-organisms, known as the metagenome. As such, the use of metagenomic approaches to identify enzymes of interest has increased in recent years in many areas of biological research.<a reveal-id="dsx032-B10" data-open="dsx032-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> Although β-xylosidases have been isolated from sources of mixed DNA (such as compost or yak rumen microbes, for example) using traditional screening of genomic libraries,<a reveal-id="dsx032-B11" data-open="dsx032-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsx032-B12" data-open="dsx032-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> no attempts have been made, to our knowledge, to isolate thermostable β-xylosidases using an NGS-based metagenomic approach.</p><p>In this study, we employed NGS to obtain an 84.2-Gbp sequence from a microbial community inhabiting a 70 °C hot spring. The sequence contained 168 open reading frames (ORFs) encoding β-xylosidase/β-glucosidase enzymes, most of which were active when expressed in <em>Escherichia coli</em>. Among the isolated enzymes, six β-xylosidases, including ones originated from clones with single nucleotide polymorphisms (SNPs), were highly thermostable, exhibiting activity at 90 °C. Interestingly, one protein, AR19M-311-27, lost activity at 90 °C, while three other homologous proteins were active at 90 °C. Based on these data, amino acid residues important for maintaining thermal stability were tentatively identified.</p>                            <h2 scrollto-destination=104086794 id="104086794" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=104086795 id="104086795" class="section-title" >2.1. Metagenomic DNA extraction and sequencing</h3>
<p>The soil sample was collected by scratching the surface of a spout of a hot spring (70 °C and pH 7.2 when sampled) by a spatula in the Onikobe-onsen-kyo geothermal area at lat. 38˚48’ and at long. 140˚40’ (Miyagi, Japan) in June, 2009. Metagenomic DNA was extracted from microbes in 10 g of the soil using an ISOIL Large for Beads, ver. 2, DNA extraction kit (Nippon Gene, Tokyo, Japan). Five micrograms of the extracted DNA were sequenced using a GS FLX Titanium DNA sequencer (Roche Diagnostics, Basel, Switzerland) according to the supplier’s protocol. In addition, the genomic DNA was amplified using a GenomiPhi V2 DNA amplification kit (GE Healthcare, Little Chalfont, UK) and then sequenced using a Hiseq 2000 DNA sequencer (Illumina, San Diego, CA, USA) according to the supplier’s protocol. Sequences with a total length of 1.11 Gbp and average length of 396 bp for each read were obtained from the former sequence, and sequences with a total length of 83.1 Gbp and average length of 92.7 bp for each read were obtained from the latter sequence. Uncertain sequences were trimmed from the raw sequences based on the quality score defined, and the resulting sequences were used for <em>de novo</em> assembly using CLC Genomics Workbench software, ver. 4.0.</p>                            <h3 scrollto-destination=104086797 id="104086797" class="section-title" >2.2. Prediction of the genes encoding β-xylosidases</h3>
<p>Genes and their ORFs in the assembled contigs were extracted using MetaGeneAnnotator software,<a reveal-id="dsx032-B13" data-open="dsx032-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> with the multiple species option for metagenome analysis, as recommended by the software supplier. The homology to β-xylosidase (E.C. 3.2.1.37) and β-glucosidase (E.C. 3.2.1.21) amino acid sequences of the extracted ORFs were searched using BLAST (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/supplementarytable1_dsx032.xlsx?Expires=2147483647&Signature=oqMJ5OPbNto0Ip6xQDjOquzoVF8X58CfeEzBGrHxWs~QEjk1HfKCaQn3FiDtPOR7q1D6ypX9tHZ8X0bZVMOClegKao8t~6ZEzfvEYi-PGqB9Zennj8yaIGdk0Bi5bxHe~pt1RVMqGeIUtrGtHwvpiD8npDJdavPg2SLJmnZ6v8vOJIIrDfq5thnHDm5kP1Yp1BHeRZ8qttcXb0gvep9x6KrNMSPxucck3uphUTzFoVD2EJQY5MM2iC6nWyXlu6xBRhhBbjFW-a5vwOmzrn7dqA3CCmVLlWGlshME4AfMHBMvyaGua~BcnwaxDwe57hzzjgqX24K1Yb5vV755UHBUzA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>) with sequences downloaded from the UniProt database, release date November 2011 (<a class="link link-uri" href="http://www.uniprot.org" target="_blank">http://www.uniprot.org</a>). The list contained 2,140 amino acid sequences, although the latest release (2017_06) contain 13,585 sequences for the enzymes, which could be useful for future research. ORFs exhibiting homology to the amino acid sequences of β-glucosidases/β-xylosidases with an expected value &lt;1 × 10<sup>−20</sup> were categorized as glycosyl hydrolase (GH) family members, with the hydrolase motifs defined in the Pfam database, version 23.0 (<a class="link link-uri" href="http://pfam.sanger.ac.uk" target="_blank">http://pfam.sanger.ac.uk</a>), using HMMER software, version 2.3 (hmmer.org).</p>                            <h3 scrollto-destination=104086799 id="104086799" class="section-title" >2.3. Cloning of the predicted genes</h3>
<p>Predicted gene sequences were amplified from the metagenomic DNA using KOD-FX DNA polymerase (Toyobo, Osaka, Japan) with PCR primers shown in <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/supplementarytable2_dsx032.xlsx?Expires=2147483647&Signature=JNnlo4GHLlOUrfPhnvYGj~Az4X8XQrk0mk~g2AlYnNgcDH4G-2y0pVyFqdm7rGCXr0lt4oMacdPQf3HTEkO~M9~2r~i0zIjHRGEge7AwIqwbmcmABwdui~l3k4mvWd9NSbR1HxLO7blInZQ8eHTtHXtW7tA5aewnorlLAEwArZpbToKXdub5nLlUwHEclHENacTxDDEbOA2kN95JqwcumSlbKj3f2aQGbqwPUpafg7gGZtZDTb~aubM05S7Psw26-kwEl-jvTkYG12DqbDt4kYe0wHg2lrevvHEuAM86iJiwvQJRL8lpG4H7ChXMZO~-s5yIN5-uJWToTvvBc~oFTw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>. The PCR-amplified fragments with four additional bases (CACC) at the 5’-terminus for directional TOPO<sup>®</sup> cloning were then inserted into the pET101/D-TOPO vector (Thermo Fisher Scientific, Waltham, MA, USA) and introduced into <em>E. coli</em> TOP10, which was a host for the vector recommended by the supplier for efficient cloning. The recombinant plasmids were extracted from the <em>E. coli</em> culture using a Wizard plus SV Minipreps DNA Purification System (Promega, Fitchburg, WI, USA). The inserted sequences of the plasmids were analysed on an ABI 3730 DNA analyser (Thermo Fisher Scientific).</p>                            <h3 scrollto-destination=104086801 id="104086801" class="section-title" >2.4. Preparation of cell-free extracts</h3>
<p>To produce the proteins, the plasmids isolated from <em>E. coli</em> TOP10 were introduced into <em>E. coli</em> BL21 Star (DE3) (Thermo Fisher Scientific), which is designed to efficiently express foreign DNA. The <em>E. coli</em> colonies were inoculated into 10 ml of Luria-Bertani medium containing 100 µg/ml of ampicillin and cultivated until the optical density at 600 nm reached approximately 0.2 to 0.8. Isopropyl-β-<span class="small-caps">d</span>(−)-thiogalactopyranoside was then added to the culture to induce expression of the foreign DNA encoded by the vector, and cultivation was continued for 16 h. The cells were harvested from the cultures by centrifugation and suspended in 1 ml of 50 mM Tris-HCl buffer (pH 8). The cell suspensions were homogenized using an Astrason 3000 homogenizer (Misonix, Farmingdale, NY, USA) and filtered using a 0.45-µm pore-size filter to prepare cell-free extracts used for the initial assay shown in <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/supplementaryfigure2_dsx032.pdf?Expires=2147483647&Signature=GRClZFfOZJ3J1QExWkFVI~yBYcA8hn8e8x5rn5Oz2OPtAwRx1UaaqaYPAwcC4Ca5BEMa772t2u~urAY5R0rJwZhBYUB-utm9mxb44ckBwTQ1et~G7GErQxG70T-mfvMD4tBgioqSkBpmQl6LVMkWZdUbbLYdwmp-8iiSnDiZOGIsl9t1M7VldDJJskNxI0rNmC0UuSZTopyeLqJz0N67ZqznglNf28nCuOGYE~qc8NC7~nkeq10LlnAPFyEmvbrvx489qjDLMWSL~w9KmO8h2REKskVsJja57foEPpHWqCo03fGtsaf0pQi6ry-dCgr5rAs0nqyrNW35U4h~HVwF6A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>.</p><p>To assess the thermal stability of the all β-xylosidases, recombinant <em>E. coli</em> cells were inoculated into 1 ml of Overnight Express<sup>TM</sup> instant LB medium (Merck, Darmstadt, Germany) containing 50 µg/ml of carbenicillin and cultivated for 20 h at 30 °C with rotary shaking at 1,000 rpm. The cells were harvested by centrifugation at 3,000 ×g for 10 min, washed twice with 50 mM sodium acetate buffer (pH 5.0), and lysed by freeze-thaw cycling in 200 µl of the buffer. The resulting lysates were centrifuged at 3,000 ×<em>g</em> for 10 min to remove cell debris and used for the assay.</p><p>In the assay for determining the optimal reaction temperature and pH and thermal stability of the proteins originated from AR19M-311, recombinant <em>E. coli</em> cells were inoculated into 100 ml of Overnight Express<sup>TM</sup> instant LB medium containing 50 µg/ml of carbenicillin and cultivated for 20 h at 30 °C with rotary shaking at 180 rpm. The cells were then washed twice with 50 mM sodium acetate buffer (pH 5.0), homogenized using a Sonifier 250 homogenizer (Branson Ultrasonics, St. Louis, MO, USA), and heat-treated at 70 °C for 10 min. After centrifugation at 20,000 ×<em>g</em> for 30 min, the supernatants were subjected to the β-xylosidase activity assay.</p>                            <h3 scrollto-destination=104086805 id="104086805" class="section-title" >2.5. Enzyme purification by column chromatography</h3>
<p>The cell-free extract was fractionated by anion exchange chromatography using an ÄKTA FPLC (GE Healthcare) system equipped a HiTrap Q HP column (GE Healthcare) equilibrated with 50 mM Tris-HCl buffer. The active fractions were collected and applied onto a HiTrap Phenyl HP column (GE Healthcare) for hydrophobic interaction chromatography. The enzyme was finally desalted by gel permeation chromatography using a PD-10 column (GE Healthcare).</p>                            <h3 scrollto-destination=104086807 id="104086807" class="section-title" >2.6. Enzymatic activity assay</h3>
<p>Hydrolysis of <em>p</em>-nitrophenylxylopyranoside (<em>p</em>NPX) by cell-free extracts containing the recombinant proteins was evaluated by determining <em>p</em>-nitrophenol release by measuring the absorbance at 400 nm. The portion (50 µl) of cell-free extracts were mixed with 1.7 mM of <em>p</em>NPX in 200 μl of 50 mM acetate or sodium phosphate buffer (pH 5.5 or 6.8) and incubated for 20 min. After the reaction, the same volume of 200 mM Na<sub>2</sub>CO<sub>3</sub> was added to the reaction mixture, and the absorbance at 400 nm was determined.</p><p>Thermal stability was evaluated at 50°C and 90 °C in 50 mM sodium acetate buffer (pH 5.0). <em>p</em>NPX was used as the substrate to assay β-xylosidase activity in crude cell-free extract. Before the assay, the cell lysates were incubated at 50°C or 90°C for 15 min. The assay was performed by adding 50 µl of the cell lysates to 200 µl of buffer containing 1.0 mM <em>p</em>NPX at 50°C for 15 min. Following addition of the same volume of 200 mM Na<sub>2</sub>CO<sub>3</sub>, the release of <em>p</em>-nitrophenol was measured by determining the absorbance at 400 nm. Instead of the cell lysate, water was added to the reaction mixture for the blank assay. One unit of the enzyme activity was defined as the amount of the enzyme releasing 1 µmol of <em>p</em>-nitrophenol per minute.</p><p>The optimal reaction temperature (50° C to 90° C) and pH (2 to 10) of the proteins originated from AR19M-311 were assayed in 50 mM GTA buffer, which consisted of 50 mM 3,3-dimethylglutaric acid, 50 mM 2-amino-2-methyl-1,3-propanediol, and 50 mM tris(hydroxymethyl)aminomethane. After the reaction, the amount of <em>p</em>-nitrophenol released was monitored using the same method described above. For assays of the thermal stability of the proteins originated from AR19M-311, the enzymes were incubated at temperatures ranging from 50°C to 90° C for 0 to 60 min in 50 mM sodium acetate buffer (pH 5.0). After incubation, the residual activity was measured at 90°C for 15 min using <em>p</em>NPX as the substrate.</p><p>Concentrations of proteins were determined by Coomassie Plus (Bradford) Assay Kit (Thermo Fischer Scientific) with bovine serum albumin as a standard.</p>                            <h3 scrollto-destination=104086812 id="104086812" class="section-title" >2.7. Phylogenetic analysis</h3>
<p>The amino acid sequences of known β-xylosidases were obtained from either UniProt database entries assigned as β-xylosidases or the BRENDA database containing thermostable β-xylosidases, and the sequences were then reviewed. These sequences and that of AR19M-311 were aligned using ClustalX ver. 2.0<a reveal-id="dsx032-B14" data-open="dsx032-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> with default parameters; subsequently, the phylogenetic tree was generated. The phylogenetic tree was visualized using FigTree ver. 1.4.2 (<a class="link link-uri" href="http://tree.bio.ed.ac.uk/software/figtree/" target="_blank">http://tree.bio.ed.ac.uk/software/figtree/</a>).</p>                            <h2 scrollto-destination=104086814 id="104086814" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=104086815 id="104086815" class="section-title" >3.1. Sequencing the metagenomic DNA of a hot spring microbial community</h3>
<p>Metagenomic DNA was extracted from a microbial community isolated from the soil of a hot spring (70 °C, pH 7.2) in Miyagi, Japan, and then subjected to NGS using a GS FLX Titanium DNA sequencer. The average length of the reads produced from each DNA fragment was 396 bp. The assembled sequences, totaling 1.1 Gbp, were searched against a list of amino acid sequences of β-xylosidases/β-glucosidases downloaded from the UniProt database (Materials and methods), resulting in the identification of 54 ORF sequences homologous to β-xylosidases/β-glucosidases. To find more genes, we then sequenced a portion of the DNA amplified from the metagenomic DNA using a Hiseq 2000 DNA sequencer, which has a higher sequencing output than the former sequencer. The resulting sequences (83.1 Gbp) were assembled with the previously generated 1.1 Gbp sequence. A total of 967,925 contigs with a length &gt;500 bp were finally obtained, of which the longest contig was 287,641 bp.</p>                            <h3 scrollto-destination=104086817 id="104086817" class="section-title" >3.2. Isolation of a gene encoding a thermostable β-xylosidase</h3>
<p>From the assembled sequences, 168 genes potentially belonging to GH families 1, 3, 31, and 43 were obtained from ORF prediction using MetaGeneAnnotator, followed by homology searching of the ORFs using the programs BLAST and HAMMER. A total of 105 ORFs appeared to be full-length, and the remainder lacked 5’- and/or 3’-terminal sequences. The ORFs exhibited a wide distribution of amino acid sequence similarity with known sequences of β-xylosidases and β-glucosidases in the UniProt database, as shown in <a reveal-id="dsx032-F1" data-open="dsx032-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>. Most of the ORFs exhibited &lt;80% homology with known sequences, indicating that metagenomic approaches are useful for identifying novel sequences. </p>                        <a id="104086819" scrollto-destination="104086819"></a>
<div id="dsx032-F1" data-id="dsx032-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/m_dsx032f1.png?Expires=2147483647&Signature=noFzT-xkjCqWOtTzmWP4NTQ08mYBq~IODH-yMaPiI3uxWx3B~~sgQZYKdJWeTOt1PHa1DgZEEgkLx7Noo4VBWtgjFdUY80~MLY2uJrXeoY3DzXb3ILGatQ28aai1C3YO3k2bXgNXJTmJnSoK-lxGNh6TpuRIV7NDHIdgvhrY-ymEwg-r1B6OiqRuvgp9PkBijmzAe8nkRrcEcxovVfdQEBlMv6kLDmgj7I3Xne3Ou8I6vVBiAi6~dCsVPVdSQCOK5V3mb-GVn5A7--sJ5~NihHSqEFyX3TtFj7O-lUsA4VmIr6CZ7-JxPyQYiCoG0epuAyg0Kf8sg~yxnL7nJVVb3w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Frequency distribution of the 168 ORFs identified by identity searches against known β-xylosidases/β-glucosidases contained in the UniProt database." path-from-xml="dsx032f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086819/dsx032f1.tif" path-from-xml="dsx032f1.tif" target="_blank">View large</a><a section="104086819" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/dsx032f1.png?Expires=2147483647&Signature=YKIXSA~c3~Efx~PsJ9FVKl1Er-~pzCV1P~kf2VI2GZ9h9eVsJQSlbJ1kjaYhxDltr1FwA4BVhV4cDKJ5LBlDdXN305C8rXEBgqunY5-2N4z9E-HvGPia52bYcnGDecGRWj9aj7Tg4ra-ILOjNMSNrpkfopNnTxZX9sIDuIujSGVkk8q0IrbdfDXAFk9AhG2DVhV52vrRS2Ds647Nktp0ManptUmjEP0dQjAeSwlucsN3Wn9RMzkWo4MwJiZyz-aKRxlCuOYx0AYLU7mnzn7YSuKTu3e6HHxhRoJrjae7rE5SEqcAeYSDI0OFvn52Wrn871kLNHeENnRNI1NkklPa8w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086819&ar=4077064&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx032f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Frequency distribution of the 168 ORFs identified by identity searches against known β-xylosidases/β-glucosidases contained in the UniProt database.</p></div></div><div content-id="dsx032-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/m_dsx032f1.png?Expires=2147483647&Signature=noFzT-xkjCqWOtTzmWP4NTQ08mYBq~IODH-yMaPiI3uxWx3B~~sgQZYKdJWeTOt1PHa1DgZEEgkLx7Noo4VBWtgjFdUY80~MLY2uJrXeoY3DzXb3ILGatQ28aai1C3YO3k2bXgNXJTmJnSoK-lxGNh6TpuRIV7NDHIdgvhrY-ymEwg-r1B6OiqRuvgp9PkBijmzAe8nkRrcEcxovVfdQEBlMv6kLDmgj7I3Xne3Ou8I6vVBiAi6~dCsVPVdSQCOK5V3mb-GVn5A7--sJ5~NihHSqEFyX3TtFj7O-lUsA4VmIr6CZ7-JxPyQYiCoG0epuAyg0Kf8sg~yxnL7nJVVb3w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Frequency distribution of the 168 ORFs identified by identity searches against known β-xylosidases/β-glucosidases contained in the UniProt database." path-from-xml="dsx032f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086819/dsx032f1.tif" path-from-xml="dsx032f1.tif" target="_blank">View large</a><a section="104086819" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/dsx032f1.png?Expires=2147483647&Signature=YKIXSA~c3~Efx~PsJ9FVKl1Er-~pzCV1P~kf2VI2GZ9h9eVsJQSlbJ1kjaYhxDltr1FwA4BVhV4cDKJ5LBlDdXN305C8rXEBgqunY5-2N4z9E-HvGPia52bYcnGDecGRWj9aj7Tg4ra-ILOjNMSNrpkfopNnTxZX9sIDuIujSGVkk8q0IrbdfDXAFk9AhG2DVhV52vrRS2Ds647Nktp0ManptUmjEP0dQjAeSwlucsN3Wn9RMzkWo4MwJiZyz-aKRxlCuOYx0AYLU7mnzn7YSuKTu3e6HHxhRoJrjae7rE5SEqcAeYSDI0OFvn52Wrn871kLNHeENnRNI1NkklPa8w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086819&ar=4077064&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx032f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Frequency distribution of the 168 ORFs identified by identity searches against known β-xylosidases/β-glucosidases contained in the UniProt database.</p></div></div><p>Using PCR, we amplified DNAs corresponding to ORFs, and cloned the resulting products into the pET101/D-TOPO vector using <em>E. coli</em> TOP10 as a cloning host. The CACC sequence, necessary for TOPO<sup>®</sup> cloning, was added on the 5’-terminal of the forward primer. Each of the DNA primer set was designed to amplify the region from initial codon to terminal codon of each full-length ORF. However, when amplifying the ORFs lacking 5’- and/or 3’-terminal sequences, the DNA primer sets were designed to amplify the region of the longest in-frame sequences of the ORFs, and added lacking initial and/or terminal codons at the terminals. One to six independent clones for each ORF were isolated to confirm DNA insertion by partial sequencing. In total, 269 genes originating from 120 ORFs were successfully cloned. These 269 genes were consisted of the ones that were identical to the original ORF sequence and the others that possessed SNPs. Then we distinguished all of the genes by adding branch numbers. The host of the plasmids was then changed from <em>E. coli</em> TOP10 to <em>E. coli</em> BL21 Star (DE3), which was more suitable for protein production. Cell-free lysates of the new host were used for evaluating β-xylosidase activity using <em>p</em>NPX as the substrate (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/supplementarytable2_dsx032.xlsx?Expires=2147483647&Signature=JNnlo4GHLlOUrfPhnvYGj~Az4X8XQrk0mk~g2AlYnNgcDH4G-2y0pVyFqdm7rGCXr0lt4oMacdPQf3HTEkO~M9~2r~i0zIjHRGEge7AwIqwbmcmABwdui~l3k4mvWd9NSbR1HxLO7blInZQ8eHTtHXtW7tA5aewnorlLAEwArZpbToKXdub5nLlUwHEclHENacTxDDEbOA2kN95JqwcumSlbKj3f2aQGbqwPUpafg7gGZtZDTb~aubM05S7Psw26-kwEl-jvTkYG12DqbDt4kYe0wHg2lrevvHEuAM86iJiwvQJRL8lpG4H7ChXMZO~-s5yIN5-uJWToTvvBc~oFTw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). When assayed at 50 °C, 82 crude cell-free extracts showed activity against the substrate (<a reveal-id="dsx032-F2" data-open="dsx032-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). However, only six extracts retained the activity when assayed at 90 °C, indicating that most of the enzymes were not thermostable at higher temperatures (<a reveal-id="dsx032-F2" data-open="dsx032-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>). The highly thermostable enzymes included three proteins originated from AR19M-311 (AR19M-311-2, AR19M-311-11, and AR19M-311-21), each single protein originated from AR19M-319 (AR19M-319-21), AR19M-329 (AR19M-329-1), and AR19M-333 (AR19M-333-4). AR19M-311-27 originated from AR19M-311 was lost its activity at 90°C, while three other proteins were highly thermostable. Then, we further investigated the enzyme characteristics of proteins originated from AR19M-311 to reveal the relationship between thermostability and SNPs. </p>                        <a id="104086821" scrollto-destination="104086821"></a>
<div id="dsx032-F2" data-id="dsx032-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/m_dsx032f2.png?Expires=2147483647&Signature=3dXTRxBPGZx8QAVJU9LWFp1ZO7mVJJxalsY7Nh9ph2xVvG-ZXUFrrLlJ1aFsn5z8qPKEnzuO82tIbH1loO5LgdZPghPVwRe3Pf3BZMg4rqYO3UTh63~rjy55StQlU5HWrforsdN~gW-ug5DhogTVhcNgG2xtDRhYCB0PL8dCchKhgEVNr8g3xNIpVrdi2w6ba-OFhqvCVWU6ZGLn0xObOBB7VERKLsw7WfJilpx24x1YAYgnXfz9FePuHFJoGQmIlSdDAf~aReBgSEnn0qK2ybE6BeSZnOd9nbhJsotw2IDppKsx6Kh1sND92InIxeu0NgK0uqLL5ezMuS0ewDXL-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The amount of p-nitrophenol released from the hydrolysis of pNPX by the crude cell-free extracts with the 82 β−xylosidases isolated from the metagenomic DNA. The enzymes were incubated at 50 °C (A) or 90 °C (B) for 15 min, then subjected to the β-xylosidase assay using pNPX as the substrate. After the reaction, the amount of p-nitrophenol was measured to detect the residual activity." path-from-xml="dsx032f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086821/dsx032f2.tif" path-from-xml="dsx032f2.tif" target="_blank">View large</a><a section="104086821" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/dsx032f2.png?Expires=2147483647&Signature=p6kVEmdP-Jf5JJq4PgFG9nOak-OWqTz6DGlq22cDqLzimUKZI64gjTPUOPrp40u0Y6-TdfyOnrvyShYAvoQbyDFa6VJKeTe5Q07kxQqd2wjc5khvDgys36geiqZBs22jBszIKRq2t--vucWcYkLL8~YvvuIUUAk2YlT1Cdb~JwPY2oIlz7hv7~8XBxxpFfbkoFiEs4zYMPHnguBJs~wJl0gd60kKh6Nz1SVcXcgC-0xljOiKGQRBE0yZMUQ5iUhcZBaA~aHWCBOoUToBIipYmPPqIJgfpz2SImRulFcmq8zqiMIOY-fp7m4VqzPPlZE3~nxn1DCN~7CNd3JNKOQwUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086821&ar=4077064&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx032f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The amount of <em>p</em>-nitrophenol released from the hydrolysis of <em>p</em>NPX by the crude cell-free extracts with the 82 β−xylosidases isolated from the metagenomic DNA. The enzymes were incubated at 50 °C (A) or 90 °C (B) for 15 min, then subjected to the β-xylosidase assay using <em>p</em>NPX as the substrate. After the reaction, the amount of <em>p</em>-nitrophenol was measured to detect the residual activity.</p></div></div><div content-id="dsx032-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/m_dsx032f2.png?Expires=2147483647&Signature=3dXTRxBPGZx8QAVJU9LWFp1ZO7mVJJxalsY7Nh9ph2xVvG-ZXUFrrLlJ1aFsn5z8qPKEnzuO82tIbH1loO5LgdZPghPVwRe3Pf3BZMg4rqYO3UTh63~rjy55StQlU5HWrforsdN~gW-ug5DhogTVhcNgG2xtDRhYCB0PL8dCchKhgEVNr8g3xNIpVrdi2w6ba-OFhqvCVWU6ZGLn0xObOBB7VERKLsw7WfJilpx24x1YAYgnXfz9FePuHFJoGQmIlSdDAf~aReBgSEnn0qK2ybE6BeSZnOd9nbhJsotw2IDppKsx6Kh1sND92InIxeu0NgK0uqLL5ezMuS0ewDXL-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The amount of p-nitrophenol released from the hydrolysis of pNPX by the crude cell-free extracts with the 82 β−xylosidases isolated from the metagenomic DNA. The enzymes were incubated at 50 °C (A) or 90 °C (B) for 15 min, then subjected to the β-xylosidase assay using pNPX as the substrate. After the reaction, the amount of p-nitrophenol was measured to detect the residual activity." path-from-xml="dsx032f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086821/dsx032f2.tif" path-from-xml="dsx032f2.tif" target="_blank">View large</a><a section="104086821" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/dsx032f2.png?Expires=2147483647&Signature=p6kVEmdP-Jf5JJq4PgFG9nOak-OWqTz6DGlq22cDqLzimUKZI64gjTPUOPrp40u0Y6-TdfyOnrvyShYAvoQbyDFa6VJKeTe5Q07kxQqd2wjc5khvDgys36geiqZBs22jBszIKRq2t--vucWcYkLL8~YvvuIUUAk2YlT1Cdb~JwPY2oIlz7hv7~8XBxxpFfbkoFiEs4zYMPHnguBJs~wJl0gd60kKh6Nz1SVcXcgC-0xljOiKGQRBE0yZMUQ5iUhcZBaA~aHWCBOoUToBIipYmPPqIJgfpz2SImRulFcmq8zqiMIOY-fp7m4VqzPPlZE3~nxn1DCN~7CNd3JNKOQwUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086821&ar=4077064&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx032f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The amount of <em>p</em>-nitrophenol released from the hydrolysis of <em>p</em>NPX by the crude cell-free extracts with the 82 β−xylosidases isolated from the metagenomic DNA. The enzymes were incubated at 50 °C (A) or 90 °C (B) for 15 min, then subjected to the β-xylosidase assay using <em>p</em>NPX as the substrate. After the reaction, the amount of <em>p</em>-nitrophenol was measured to detect the residual activity.</p></div></div>                            <h3 scrollto-destination=104086822 id="104086822" class="section-title" >3.3. Effects of temperature and pH on the hydrolysis of <em>p</em>NPX by the β-xylosidases</h3>
<p>We characterized the proteins originated from AR19M-311 further after removal of host-derived proteins by heat treatment at 70 °C, which caused most proteins in the lysate, except the heat-stable enzymes, to precipitate. A protein, AR19M-311-27, lost activity during incubation at 70 °C. AR19M-311-2, AR19M-311-11, and AR19M-311-21 showed similar optimum temperatures (<a reveal-id="dsx032-F3" data-open="dsx032-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>). These enzymes showed an optimal pH of 5.0, but no activity was detected at pH 4.5 when assayed at 90 °C, whereas AR19M-311-27 showed negligible activity at all pH values examined (<a reveal-id="dsx032-F3" data-open="dsx032-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>). The drastic difference in activity over such a narrow pH range (5.0 to 4.5) suggests that there are structural differences between the pH range, and this will be an interesting target of research when the three-dimensional structures of the enzymes become available. Similar pH dependency was reported for a β-xylosidase from <em>Enterobacter</em> sp.<a reveal-id="dsx032-B15" data-open="dsx032-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a></p>                        <a id="104086824" scrollto-destination="104086824"></a>
<div id="dsx032-F3" data-id="dsx032-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/m_dsx032f3.png?Expires=2147483647&Signature=vj217q1W32y5Lof3ekHhNKELzSbdOAR8nYZhBsBHAP34kQWMFUsOvWaOM-FVr~GWa6Nl5w3s8W5O1D1DBP6S2Mb9rxlBlc-x-p0c9gNKFE4upI40ofp2S2CmPp0ghfwZdrp2QUlS4ptiy6uXKIhCMxGg1kbebTkQaz-fgPSDHn78Ki36YJukb73kNTJKPvxq1DNy2qqw4w2eVvsxEa91-JuSDtCClt619jEUP8IIXn6FyHQXiT8mhbqPVhwKihTo9eWxapsjy3ULlPdm5EJvrl5zRmU9tJu7T0470YGir0A~TW1sPBw9odnp60Sj-~WFTvnpz~eHwo~IlVV3JSRPbw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Optimal β-xylosidase activity reaction temperature (A) and pH (B) for the proteins originated from AR19M-311. β-Xylosidase activity was assayed at several temperatures in GTA buffer (pH 5.0) in (A) and in buffers adjusted to different pH values at 90 °C in (B). Activity was assessed by measuring the release of p-nitrophenol at 400 nm." path-from-xml="dsx032f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086824/dsx032f3.tif" path-from-xml="dsx032f3.tif" target="_blank">View large</a><a section="104086824" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/dsx032f3.png?Expires=2147483647&Signature=p63dxGNeQTFtVaOLeITL04ny3yeewNiLdl3S9QeTmDngJOJd3Uw0IFQALWTP6KE0-bNPar6KOWMHHXrzC46biVgFa0gIgIG3VQ11grsyqOuPwFLiHMNsAwnp2en9TvYyS7~x3~vMKFQvNxHN4ylkXQE8W~ycwheMkynvw73CH-qtewIU3Fgcgu0ES9RB-hPCxI~3x8waj7z23~7g5ioRp0z4nUJ335kjgKSKPknpyEwYwhEas6F7i4W6e7VMr5qeGhfI8FqEsYnF32KGAUo1MAkFr6C-2cILXqwloWYZG1LJ-bkSX-ohWe07sUqU~5hSLeH~Pv0ubAelNz8aYmw3Mg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086824&ar=4077064&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx032f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Optimal β-xylosidase activity reaction temperature (A) and pH (B) for the proteins originated from AR19M-311. β-Xylosidase activity was assayed at several temperatures in GTA buffer (pH 5.0) in (A) and in buffers adjusted to different pH values at 90 °C in (B). Activity was assessed by measuring the release of <em>p</em>-nitrophenol at 400 nm.</p></div></div><div content-id="dsx032-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/m_dsx032f3.png?Expires=2147483647&Signature=vj217q1W32y5Lof3ekHhNKELzSbdOAR8nYZhBsBHAP34kQWMFUsOvWaOM-FVr~GWa6Nl5w3s8W5O1D1DBP6S2Mb9rxlBlc-x-p0c9gNKFE4upI40ofp2S2CmPp0ghfwZdrp2QUlS4ptiy6uXKIhCMxGg1kbebTkQaz-fgPSDHn78Ki36YJukb73kNTJKPvxq1DNy2qqw4w2eVvsxEa91-JuSDtCClt619jEUP8IIXn6FyHQXiT8mhbqPVhwKihTo9eWxapsjy3ULlPdm5EJvrl5zRmU9tJu7T0470YGir0A~TW1sPBw9odnp60Sj-~WFTvnpz~eHwo~IlVV3JSRPbw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Optimal β-xylosidase activity reaction temperature (A) and pH (B) for the proteins originated from AR19M-311. β-Xylosidase activity was assayed at several temperatures in GTA buffer (pH 5.0) in (A) and in buffers adjusted to different pH values at 90 °C in (B). Activity was assessed by measuring the release of p-nitrophenol at 400 nm." path-from-xml="dsx032f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086824/dsx032f3.tif" path-from-xml="dsx032f3.tif" target="_blank">View large</a><a section="104086824" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/dsx032f3.png?Expires=2147483647&Signature=p63dxGNeQTFtVaOLeITL04ny3yeewNiLdl3S9QeTmDngJOJd3Uw0IFQALWTP6KE0-bNPar6KOWMHHXrzC46biVgFa0gIgIG3VQ11grsyqOuPwFLiHMNsAwnp2en9TvYyS7~x3~vMKFQvNxHN4ylkXQE8W~ycwheMkynvw73CH-qtewIU3Fgcgu0ES9RB-hPCxI~3x8waj7z23~7g5ioRp0z4nUJ335kjgKSKPknpyEwYwhEas6F7i4W6e7VMr5qeGhfI8FqEsYnF32KGAUo1MAkFr6C-2cILXqwloWYZG1LJ-bkSX-ohWe07sUqU~5hSLeH~Pv0ubAelNz8aYmw3Mg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086824&ar=4077064&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx032f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Optimal β-xylosidase activity reaction temperature (A) and pH (B) for the proteins originated from AR19M-311. β-Xylosidase activity was assayed at several temperatures in GTA buffer (pH 5.0) in (A) and in buffers adjusted to different pH values at 90 °C in (B). Activity was assessed by measuring the release of <em>p</em>-nitrophenol at 400 nm.</p></div></div><p>As the three enzymes AR19M-311-2, AR19M-311-11 and AR19M-311-21 showed almost the same optimum pH and temperature for the enzymatic reaction (<a reveal-id="dsx032-F3" data-open="dsx032-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>), only AR19M-311-2 was further purified by column chromatography using HiTrap Q HP and HiTrap Phenyl HP columns to determine the kinetic parameters under the optimal reaction conditions using <em>p</em>NPX as the substrate. The <em>k</em><sub>cat</sub> of AR19M-311-2 was 1.30 × 10<sup>2</sup> s<sup>−1</sup>, and the <em>K</em>m for <em>p</em>NPX was 0.872 mM. The lineweaver-Burk plot for the determination of the kinetic parameters was shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/supplementaryfigure1_dsx032.pdf?Expires=2147483647&Signature=M2HY~7Xd846OMGROjYCtl7ItRmUSq1w-V8-MfqosfQe7yFdyGCDwvyzGhEvTdjUDRPORe7~DeYuRDEjfUOnBG4GLov4~h0KZpuvZUFapu8mgKrntdv3fh1rP1jZASpgWuodsArIRvQz06aIEx2PlnuscQe0GrBdLAqjrMneklq9IEi5sFZ007QsnVM1eTXeMeOmuGQT7n01XXQRJaK8~gQLaoySLTQ4Fe-iYFnVnJRh~JPjT98nO1r-6wwlCnDS3i0Cuacx0dztGKpYJSbU5ZjDKmMGyt1vf927gqNLHJKxPjf3Z0ubeHjjVnAM7-FBMcO0a6OrprS5WeDz4CbgVKQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>.</p>                            <h3 scrollto-destination=104086826 id="104086826" class="section-title" >3.4. Thermostability of the β-xylosidases</h3>
<p>We analysed the thermal stability of AR19M-311-2, AR19M-311-11, and AR19M-311-21 by incubating the partially purified enzymes without substrate at 50°C, 60°C, 70°C, 80°C, and 90 °C and pH 5.0 for 0 to 60 min (<a reveal-id="dsx032-F4" data-open="dsx032-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). The enzymes retained above 80% of their initial activities at 50°C, 60°C, and 70°C for more than 60 min, except AR19M-311-2 whose activity was decreased to 50% after incubation at 70°C for 60 min. The three enzymes lost 50% of their activities within 10 min at 80°C. While the activity was immediately lost when the substrate <em>p</em>NPX was absent at 90°C, the hydrolysis proceeded at 90°C at least 15 min as shown in <a reveal-id="dsx032-F3" data-open="dsx032-F3" class="link link-reveal link-table xref-fig">Fig. 3</a> when the substrate existed. Hence, it suggests that the substrate stabilize these enzymes. When the crude cell-free extracts were used, the activities were retained after incubation at 90°C without the substrate as shown in <a reveal-id="dsx032-F2" data-open="dsx032-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>, indicating the enzymes were stabilized in the extract. </p>                        <a id="104086828" scrollto-destination="104086828"></a>
<div id="dsx032-F4" data-id="dsx032-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/m_dsx032f4.png?Expires=2147483647&Signature=VDGis0dYUKlMiofYNaj-QRzMjkniewjzfdfQ-Ztbsg1lih0TjcfJJ3YYRobMjinxrVEC8C~7q6Yb7YfBO1Po0ii6iuJGzwXX58MvKubP9CAjaqiNM6lVaJJuj47t8fZsH3KUHs5SvQVS-TEifxkCqet-ACt~DUJPnXuyAj4~38MYnCa1BHG-RpRbXEt0gik0H~K7Gjpri2Dwr8vVRemp8sEJP0eBIocET2UCrDAfcTCrmGmHbaqGg4~1Hj7-H~4piBOaV985Oklg-eJm2Er0y9pSA0vetWuxh-zlvEvunhzCaupPwYsUBFVq0g6S8YPg7jIs2tERblpYRfEfEC2kDg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Thermal stability of AR19M-311-2 (A), AR19M-311-11 (B), and AR19M-311-21 (C). The enzymes were preincubated at 70°C for 10 min to precipitate host-derived proteins, subsequently, the enzymes were incubated at 50, 60, 70, 80, or 90 °C for 0 to 60 min and then subjected to the β-xylosidase assay using pNPX as the substrate. Residual activity was assessed by measuring the release of p-nitrophenol at 400 nm." path-from-xml="dsx032f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086828/dsx032f4.tif" path-from-xml="dsx032f4.tif" target="_blank">View large</a><a section="104086828" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/dsx032f4.png?Expires=2147483647&Signature=LqZ8jYwu7uldYoLKr2LCziVQuSPTx~YQMmt8SfkoSr8ht2Cz6GObcR-qW2ox6KUnwgJmnvHzpNFwdunZfhMmMIy1cl9AtbQP1hgwCWDUHMjVkWXe5gH6EKLJPTyPgzXu7-8eVMvNnZluD0MiFGPS-Ay36Blyyriuu3l0zmmdl1-NWt1fEuY02~J8cce9ti8rgiX5lMvP3uqi013AdyAGHRRY1A3K2nRpGCXbFBVrdpQyIS~Pe20k9vkGfq44lAwJmTX5xiF4t3KaM9y8jbO55SKZc014uRpwBrKvqMj5NEx6bURvFSgLWEHBqxZ143la15K-Jfb96cSgUCirj4YKGA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086828&ar=4077064&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx032f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Thermal stability of AR19M-311-2 (A), AR19M-311-11 (B), and AR19M-311-21 (C). The enzymes were preincubated at 70°C for 10 min to precipitate host-derived proteins, subsequently, the enzymes were incubated at 50, 60, 70, 80, or 90 °C for 0 to 60 min and then subjected to the β-xylosidase assay using <em>p</em>NPX as the substrate. Residual activity was assessed by measuring the release of <em>p</em>-nitrophenol at 400 nm.</p></div></div><div content-id="dsx032-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/m_dsx032f4.png?Expires=2147483647&Signature=VDGis0dYUKlMiofYNaj-QRzMjkniewjzfdfQ-Ztbsg1lih0TjcfJJ3YYRobMjinxrVEC8C~7q6Yb7YfBO1Po0ii6iuJGzwXX58MvKubP9CAjaqiNM6lVaJJuj47t8fZsH3KUHs5SvQVS-TEifxkCqet-ACt~DUJPnXuyAj4~38MYnCa1BHG-RpRbXEt0gik0H~K7Gjpri2Dwr8vVRemp8sEJP0eBIocET2UCrDAfcTCrmGmHbaqGg4~1Hj7-H~4piBOaV985Oklg-eJm2Er0y9pSA0vetWuxh-zlvEvunhzCaupPwYsUBFVq0g6S8YPg7jIs2tERblpYRfEfEC2kDg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Thermal stability of AR19M-311-2 (A), AR19M-311-11 (B), and AR19M-311-21 (C). The enzymes were preincubated at 70°C for 10 min to precipitate host-derived proteins, subsequently, the enzymes were incubated at 50, 60, 70, 80, or 90 °C for 0 to 60 min and then subjected to the β-xylosidase assay using pNPX as the substrate. Residual activity was assessed by measuring the release of p-nitrophenol at 400 nm." path-from-xml="dsx032f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086828/dsx032f4.tif" path-from-xml="dsx032f4.tif" target="_blank">View large</a><a section="104086828" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/dsx032f4.png?Expires=2147483647&Signature=LqZ8jYwu7uldYoLKr2LCziVQuSPTx~YQMmt8SfkoSr8ht2Cz6GObcR-qW2ox6KUnwgJmnvHzpNFwdunZfhMmMIy1cl9AtbQP1hgwCWDUHMjVkWXe5gH6EKLJPTyPgzXu7-8eVMvNnZluD0MiFGPS-Ay36Blyyriuu3l0zmmdl1-NWt1fEuY02~J8cce9ti8rgiX5lMvP3uqi013AdyAGHRRY1A3K2nRpGCXbFBVrdpQyIS~Pe20k9vkGfq44lAwJmTX5xiF4t3KaM9y8jbO55SKZc014uRpwBrKvqMj5NEx6bURvFSgLWEHBqxZ143la15K-Jfb96cSgUCirj4YKGA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086828&ar=4077064&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx032f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Thermal stability of AR19M-311-2 (A), AR19M-311-11 (B), and AR19M-311-21 (C). The enzymes were preincubated at 70°C for 10 min to precipitate host-derived proteins, subsequently, the enzymes were incubated at 50, 60, 70, 80, or 90 °C for 0 to 60 min and then subjected to the β-xylosidase assay using <em>p</em>NPX as the substrate. Residual activity was assessed by measuring the release of <em>p</em>-nitrophenol at 400 nm.</p></div></div>                            <h3 scrollto-destination=104086829 id="104086829" class="section-title" >3.5. Classification of AR19M-311 and other β-xylosidases based on sequence homology</h3>
<p>We searched for proteins similar to the ones originated from AR19M-311, which consisted of 752 amino acid residues, in the National Center for Biotechnology Information amino acid sequence database. The highest identity (71%) was exhibited by a predicted xylosidase/arabinosidase encoded by the genome of the bacterium <em>Dictyoglomus thermophilum</em>, although there is no experimental evidence confirming that this protein is a hydrolase. Among the known thermostable β-xylosidases, the enzyme shared the highest homology (56%) with a β-xylosidase from the archaeon <em>S. solfataricus</em><a reveal-id="dsx032-B8" data-open="dsx032-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> and the lowest homology (29%) with a β-xylosidase from the fungus <em>A. pullulans</em><em>.</em><a reveal-id="dsx032-B9" data-open="dsx032-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a></p><p>The phylogenetic relationship between AR19M-311 and other β-xylosidases was determined using the ClustalX program (<a reveal-id="dsx032-F5" data-open="dsx032-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). AR19M-311 was classified in a thermostable β-xylosidase cluster that included like enzymes from <em>S. solfataricus</em> and the bacteria <em>T. thermarum</em>, <em>T. maritima</em>, <em>Thermoanaerobacter ethanolicus</em>, and <em>Caldicellulosiruptor bescii</em>. </p>                        <a id="104086832" scrollto-destination="104086832"></a>
<div id="dsx032-F5" data-id="dsx032-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/m_dsx032f5.png?Expires=2147483647&Signature=PldsZWl14XjAGRih21wChWB0xVXt2wU0tP2eUQ7zyGOpUBFnnHz7uqnkNWBWDSB2pBJK-crDrJoOc3arWQkPPCUQDnBYBwgcswJfaxFksIqRNagF1ZIn8jyWXChAbjBSSN2C9j9JGPMpkdaCig-5DBS8~Y~OY62m9-UB9iz3W5ZK~WwDWJMklwMGA17dZBw-G-46CJm82B7Q5MuCP4sK5FcyY9tEZU696vnwVNt2KBYHeL0D~mIX1UDQBXRsqXDaOCuZIa~APzBmQoxC4pVVBbbvthaEZlCLIDp3Up3Q3vgrU0UFbR1pI04pzFy27TuNw7rsRwiYqjmvlDACGbB~8g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic tree of the proteins originated from AR19M-311 and representatives of known β-xylosidases." path-from-xml="dsx032f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086832/dsx032f5.tif" path-from-xml="dsx032f5.tif" target="_blank">View large</a><a section="104086832" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/dsx032f5.png?Expires=2147483647&Signature=2cBt973vHXROCpY8Owb4IlbT8K6JE50atkGGgRWThiiM4zwQ9oqSjUzwaSf0WJvYkIHXoNBrxdMI2rnTdPKKwUVqwPnkdtb0U85Q2b1S9~HqAiQK70g3RASI8ONOjiqQTd-9Vi7ielqhjFqhTaibwkFVc01EIdsHGN75L2GhVCIBwzgtyX3AGmoQuBLCP4BK5Rv34lIwGDFi6yPq8mDlrTZZSowa3QIaMFvu9CgyywwwDcXehzm-falrM-E~i-SSIACrIG2CMMG3W-iN12-zYnmz4KxSiaiXT3gTC7IX1sUV3QQnBUQMXopEK93bfCiA8gH-P9SSYnSNOddnKtmjdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086832&ar=4077064&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx032f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree of the proteins originated from AR19M-311 and representatives of known β-xylosidases.</p></div></div><div content-id="dsx032-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/m_dsx032f5.png?Expires=2147483647&Signature=PldsZWl14XjAGRih21wChWB0xVXt2wU0tP2eUQ7zyGOpUBFnnHz7uqnkNWBWDSB2pBJK-crDrJoOc3arWQkPPCUQDnBYBwgcswJfaxFksIqRNagF1ZIn8jyWXChAbjBSSN2C9j9JGPMpkdaCig-5DBS8~Y~OY62m9-UB9iz3W5ZK~WwDWJMklwMGA17dZBw-G-46CJm82B7Q5MuCP4sK5FcyY9tEZU696vnwVNt2KBYHeL0D~mIX1UDQBXRsqXDaOCuZIa~APzBmQoxC4pVVBbbvthaEZlCLIDp3Up3Q3vgrU0UFbR1pI04pzFy27TuNw7rsRwiYqjmvlDACGbB~8g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic tree of the proteins originated from AR19M-311 and representatives of known β-xylosidases." path-from-xml="dsx032f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086832/dsx032f5.tif" path-from-xml="dsx032f5.tif" target="_blank">View large</a><a section="104086832" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/dsx032f5.png?Expires=2147483647&Signature=2cBt973vHXROCpY8Owb4IlbT8K6JE50atkGGgRWThiiM4zwQ9oqSjUzwaSf0WJvYkIHXoNBrxdMI2rnTdPKKwUVqwPnkdtb0U85Q2b1S9~HqAiQK70g3RASI8ONOjiqQTd-9Vi7ielqhjFqhTaibwkFVc01EIdsHGN75L2GhVCIBwzgtyX3AGmoQuBLCP4BK5Rv34lIwGDFi6yPq8mDlrTZZSowa3QIaMFvu9CgyywwwDcXehzm-falrM-E~i-SSIACrIG2CMMG3W-iN12-zYnmz4KxSiaiXT3gTC7IX1sUV3QQnBUQMXopEK93bfCiA8gH-P9SSYnSNOddnKtmjdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086832&ar=4077064&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx032f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree of the proteins originated from AR19M-311 and representatives of known β-xylosidases.</p></div></div>                            <h3 scrollto-destination=104086833 id="104086833" class="section-title" >3.6. Amino acid substitutions of proteins originated from AR19M-311</h3>
<p>The consensus amino acid sequence of AR19M-311 and those of the four proteins were compared, as shown in <a reveal-id="dsx032-F6" data-open="dsx032-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>. Amino acid substitutions were observed for 11 residues (Thr<sup>57</sup>, Arg<sup>187</sup>, Val<sup>194</sup>, Ala<sup>206</sup>, Thr<sup>311</sup>, Val<sup>355</sup>, Ala<sup>451</sup>, Gly<sup>519</sup>, Ile<sup>625</sup>, Val<sup>635</sup>, and Lys<sup>739</sup>) of the consensus sequence. Comparing the thermostable proteins, AR19M-311-2, AR19M-311-11, and AR19M-311-21, and the non-thermostable protein AR19M-311-27, there were five residues that were substituted only with non-thermostable protein (Thr<sup>57</sup> to Ile<sup>57</sup>, Ile<sup>311</sup> to Thr<sup>311</sup>, Ala<sup>451</sup> to Thr<sup>451</sup>, Gly<sup>519</sup> to Ser<sup>519</sup>, and Lys<sup>739</sup> to Asn<sup>739</sup>). AR19M-311-2 exhibited unique substitutions of Val<sup>355</sup> with Ala and Val<sup>635</sup> with Ile, but these substitutions might not be crucial for maintaining high activity because they were not found in the other thermostable proteins. Interestingly, Arg<sup>187</sup>, Val<sup>194</sup>, and Ala<sup>206</sup> were only seen in the consensus sequence and not in the four proteins. </p>                        <a id="104086835" scrollto-destination="104086835"></a>
<div id="dsx032-F6" data-id="dsx032-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/m_dsx032f6.png?Expires=2147483647&Signature=Y3gG1L2qpmm0hygsfOC870EC8xLz62M7VvYrZA8VSns3~xk-0MHLaRdqIk3zIL9OMHKiKhd1mqtqFJCEQFm95zsPIodbEs8DV5BegMuq6Fn4NRV8Io7dcK3ZQMOHrUPXDk~Zg0Gt4Tvx6j2~UMxLA0dGDm8bn5Xc-AxbFjFNM2v2AOxnRKOQ7skEB-W80Ns3UsIa-JgdZl73OWchBgpynWfxJts9ztHjcKqjfPheRQx4pV8CCEYbmUc21wJkJJLcLM6Rc2-ZdKHxtTyQ6r04pV12O2bx2gG6MpyFO7S1ZagnMpT2PHaX6CLZodE0x6GZDf1dklbFQyqIA1VB6MmpVg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sequence alignment of AR19M-311 and its clones. The alignment was created by ClustalX version 2.0 and the residues substituted in any of the clone were highlighted by black background color and the residue numbers were indicated at the top of the residues. AR19M311 is the consensus sequence obtained from the ORF search. AR19M-311-2, AR19M-311-11, and AR19M-311-21 are the thermostable proteins, and AR19M-311-27 is the non-thermostable protein." path-from-xml="dsx032f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086835/dsx032f6.tif" path-from-xml="dsx032f6.tif" target="_blank">View large</a><a section="104086835" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/dsx032f6.png?Expires=2147483647&Signature=sW~lgpRovML3DTKpQZKLCgwgtdg5MrX2PiNs6sC9epgOaEHfs7uCwLJgASgRAtV~7g6ZNLVM-YkXC93Knu0-ZZBvQQZHRCUPWcAod7YzfGs4JPWV0j5zPTyHG2jCsm1iQD6MOfIqW84K8T3m3oyPC0cEB0q0agdkcclkY8jKwlU86dZhka-66ucGtp~tNmcpOsSikSOOVnA24jVMHbWOlZ5EAVeF02emCNZyMe6rB1Csi2NdKkrCZG2bWQmC1sJIAYynSoQHN6M5Co~i~a2g1HrS-aOg-yflUT71sFhHvHfVI-k-MmhEE2Gm0~riU~9Nawgzo1MUhRpIrTYuGnN6iA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086835&ar=4077064&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx032f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sequence alignment of AR19M-311 and its clones. The alignment was created by ClustalX version 2.0 and the residues substituted in any of the clone were highlighted by black background color and the residue numbers were indicated at the top of the residues. AR19M311 is the consensus sequence obtained from the ORF search. AR19M-311-2, AR19M-311-11, and AR19M-311-21 are the thermostable proteins, and AR19M-311-27 is the non-thermostable protein.</p></div></div><div content-id="dsx032-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/m_dsx032f6.png?Expires=2147483647&Signature=Y3gG1L2qpmm0hygsfOC870EC8xLz62M7VvYrZA8VSns3~xk-0MHLaRdqIk3zIL9OMHKiKhd1mqtqFJCEQFm95zsPIodbEs8DV5BegMuq6Fn4NRV8Io7dcK3ZQMOHrUPXDk~Zg0Gt4Tvx6j2~UMxLA0dGDm8bn5Xc-AxbFjFNM2v2AOxnRKOQ7skEB-W80Ns3UsIa-JgdZl73OWchBgpynWfxJts9ztHjcKqjfPheRQx4pV8CCEYbmUc21wJkJJLcLM6Rc2-ZdKHxtTyQ6r04pV12O2bx2gG6MpyFO7S1ZagnMpT2PHaX6CLZodE0x6GZDf1dklbFQyqIA1VB6MmpVg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sequence alignment of AR19M-311 and its clones. The alignment was created by ClustalX version 2.0 and the residues substituted in any of the clone were highlighted by black background color and the residue numbers were indicated at the top of the residues. AR19M311 is the consensus sequence obtained from the ORF search. AR19M-311-2, AR19M-311-11, and AR19M-311-21 are the thermostable proteins, and AR19M-311-27 is the non-thermostable protein." path-from-xml="dsx032f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086835/dsx032f6.tif" path-from-xml="dsx032f6.tif" target="_blank">View large</a><a section="104086835" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/dsx032f6.png?Expires=2147483647&Signature=sW~lgpRovML3DTKpQZKLCgwgtdg5MrX2PiNs6sC9epgOaEHfs7uCwLJgASgRAtV~7g6ZNLVM-YkXC93Knu0-ZZBvQQZHRCUPWcAod7YzfGs4JPWV0j5zPTyHG2jCsm1iQD6MOfIqW84K8T3m3oyPC0cEB0q0agdkcclkY8jKwlU86dZhka-66ucGtp~tNmcpOsSikSOOVnA24jVMHbWOlZ5EAVeF02emCNZyMe6rB1Csi2NdKkrCZG2bWQmC1sJIAYynSoQHN6M5Co~i~a2g1HrS-aOg-yflUT71sFhHvHfVI-k-MmhEE2Gm0~riU~9Nawgzo1MUhRpIrTYuGnN6iA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086835&ar=4077064&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx032f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sequence alignment of AR19M-311 and its clones. The alignment was created by ClustalX version 2.0 and the residues substituted in any of the clone were highlighted by black background color and the residue numbers were indicated at the top of the residues. AR19M311 is the consensus sequence obtained from the ORF search. AR19M-311-2, AR19M-311-11, and AR19M-311-21 are the thermostable proteins, and AR19M-311-27 is the non-thermostable protein.</p></div></div>                            <h2 scrollto-destination=104086836 id="104086836" class="section-title" >4. Discussion</h2>
<p>Here, we carried out a comprehensive survey of thermostable β-xylosidases using the metagenomic DNA of a microbial community isolated from the soil of a hot spring, which was 70 °C and pH 7.2 when sampled. Using two types of NGS analysers, a total of 84.2 Gbp of sequences were obtained from the metagenomic DNA. From 967,925 contigs calculated from the sequences, 168 ORFs were identified, most of which shared homology of less than 80% with known β-xylosidases (<a reveal-id="dsx032-F1" data-open="dsx032-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). Of these 168 ORFs, 120 were successfully cloned from the metagenomic DNA. Including the SNPs that were identified by PCR-based cloning of the 120 ORFs, a total of 269 individual clones were obtained, and 82 proteins produced from these clones showed β-xylosidase activity at 50 °C, demonstrating the usefulness of metagenomic approaches for identifying novel β-xylosidases. Six genes, including three genes originated from AR19M-311, and genes originated from AR19M-319, AR19M-329, and AR19M-334, encoded enzymes showing thermal stability up to 90 °C (<a reveal-id="dsx032-F2" data-open="dsx032-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). This is a large-scaled survey for enzymatic activity of thermostable cellulases isolated from metagenomic DNA resources, which is, to our knowledge, the first report of thermostable β-xylosidases isolated by a metagenomic approach.</p><p>This study provided novel sequences of thermostable β-xylosidases for further consideration of structure-activity relationships. AR19M-311-2, AR19M-311-11, and AR19M311-21 showed the highest activity at 90 °C and were stable at temperatures up to 70 °C for at least 1 h. As the β-xylosidase from <em>T. ethanolicus</em> showed the highest optimal reaction temperature (93 °C),<a reveal-id="dsx032-B7" data-open="dsx032-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> it was concluded that the three proteins are highly thermostable β-xylosidases. Despite similarities in their thermal properties, the amino acid sequences shared 54% identity with <em>T. ethanolicus</em> β-xylosidase. The β-xylosidases from <em>S. solfataricus</em> and <em>T. maritima</em>, which have an optimal reaction temperature of 90 °C,<sup><a reveal-id="dsx032-B4" data-open="dsx032-B4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="dsx032-B8" data-open="dsx032-B8" class="link link-ref link-reveal xref-bibr">8</a></sup> are also considered highly thermostable β-xylosidases. However, the sequence identity between these β-xylosidases and the three proteins we describe was &lt;60%. The sequence differences between these highly thermostable β-xylosidases helped us identify a number of consensus and essential amino acids in the sequences of the highly thermostable β-xylosidases, as discussed below.</p><p>Analysis of the phylogenetic tree indicated that the prokaryotic thermostable β-xylosidase cluster is more closely related to eukaryotic β-xylosidases (which include those of plants such as <em>Arabidopsis thaliana</em> and <em>Medicago sativa</em> and those of the fungi <em>A. pullulans</em><a reveal-id="dsx032-B9" data-open="dsx032-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> and <em>Aspergillus</em> sp.) than to the prokaryotic β-xylosidases, which have not been reported as thermostable (<a reveal-id="dsx032-F5" data-open="dsx032-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). Therefore, comparative sequence analysis of these β-xylosidases could also provide information that could enhance our understanding of the mechanisms of high-temperature stability.</p><p>From an integrated consideration of both the complete comparative sequence analysis and the local effects of amino acid substitutions, we identified some of the amino acid residues essential for the thermal stability of β-xylosidases. Comparing the three thermostable proteins and the non-thermostable protein AR19M-311-27, the five residues were found as unique substitutions in AR19M-311-27 (Thr<sup>57</sup> to Ile<sup>57</sup>, Ile<sup>311</sup> to Thr<sup>311</sup>, Ala<sup>451</sup> to Thr<sup>451</sup>, Gly<sup>519</sup> to Ser<sup>519</sup>, and Lys<sup>739</sup> to Asn<sup>739</sup>). In alignment of the sequences of the thermostable prokaryotic β-xylosidases and eukaryotic β-xylosidases (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/supplementaryfigure2_dsx032.pdf?Expires=2147483647&Signature=GRClZFfOZJ3J1QExWkFVI~yBYcA8hn8e8x5rn5Oz2OPtAwRx1UaaqaYPAwcC4Ca5BEMa772t2u~urAY5R0rJwZhBYUB-utm9mxb44ckBwTQ1et~G7GErQxG70T-mfvMD4tBgioqSkBpmQl6LVMkWZdUbbLYdwmp-8iiSnDiZOGIsl9t1M7VldDJJskNxI0rNmC0UuSZTopyeLqJz0N67ZqznglNf28nCuOGYE~qc8NC7~nkeq10LlnAPFyEmvbrvx489qjDLMWSL~w9KmO8h2REKskVsJja57foEPpHWqCo03fGtsaf0pQi6ry-dCgr5rAs0nqyrNW35U4h~HVwF6A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>), the Thr<sup>57</sup> and the Ala<sup>451</sup> was conserved in all of the thermostable enzymes, and the Gly<sup>519</sup> was conserved in five of the six thermostable enzymes. Hence, the three substitutions of Thr<sup>57</sup> to Ile<sup>57</sup>, Ala<sup>451</sup> to Thr<sup>451</sup> and Gly<sup>519</sup> to Ser<sup>519</sup> were considered to affect the low thermostability of AR19M-311-27 at 90°C. The alignment also showed that insertion of two amino acids, including Thr<sup>311</sup> or the corresponding residues of other β-xylosidases, occurs only in the thermostable prokaryotic β-xylosidases and the moderately thermostable β-xylosidase from <em>Trichoderma koningii</em><em>.</em><a reveal-id="dsx032-B16" data-open="dsx032-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> The Thr residue is conserved in the four prokaryotic β-xylosidases and substituted with Ile in the others. Therefore, we suggest that the insertion of the two amino acids contributes to the thermal stability of β-xylosidases, which should be confirmed by future biochemical works. The contribution of the substitution of Lys<sup>739</sup> to Asn<sup>739</sup> on the thermostability is considered to be insignificant, because Lys<sup>739</sup> is not conserved in thermostable enzymes. AR19M-311-2 also possessed the unique substitutions of Val<sup>355</sup> with Ala and Val<sup>635</sup> with Ile, substitutions not found in AR19M-311-11 and AR19M-311-21. These substitutions apparently have only a slight influence on thermostability, as Val, Ala, and Ile are all hydrophobic residues with similar properties.</p><p>As shown by this study, PCR-based cloning of genes from metagenomic DNA is a useful method because it can lead to the identification of SNPs in consensus ORF sequences, which is advantageous for elucidating structure-activity relationships. There are two possible approaches to discover functional enzymes from metagenomic data. One is the homology search method demonstrated in this study, while the other is the activity-driven one. One of the advantages of the former method is that it could find highly homologous sequences with distinct activities from vast sequences, especially encoding active and non-active enzymes, which may lead speculation of function of each amino acid residue on the enzyme activity, as shown in our results of active forms, AR19M-311-2, AR19M-311-11, and AR19M-311-21 and a non-active form AR19M-311-27, which could not be obtained by the latter method alone. Recently, isolation of novel β-galactosidases not predictable from gene sequences was reported,<a reveal-id="dsx032-B17" data-open="dsx032-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> in which the enzymes were identified from a cosmid library with ∼33 kbp metagenomic DNA in surrogate hosts by the activity-driven method. They emphasize the importance of developing of multi-host systems for functional screening to substantially extend the breadth of gene discovery. Therefore, either method may be used according to the purpose of research. </p>                            <h2 scrollto-destination=104086842 id="104086842" class="section-title" >Data availability</h2>
<p>The sequences of AR19M metagenome were deposited in DDBJ Sequence Read Archive maintained by the DNA Data Bank of Japan under accession numbers of DRA005406 and DRA005407.</p>                            <h2 scrollto-destination=104086844 id="104086844" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Fumiko Fujii for her excellent technical support in DNA sequencing using the GS FLX Titanium sequencer and Kumiko Suzuki, Miyuki Inde, Masumi Nagaki, Akiko Kisen, Takahiro Matsuyama, and Satomi Saito for their technical support in DNA cloning and protein expression analyses.</p>                            <h3 scrollto-destination=104086846 id="104086846" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/supplementaryfigure1_dsx032.pdf?Expires=2147483647&Signature=M2HY~7Xd846OMGROjYCtl7ItRmUSq1w-V8-MfqosfQe7yFdyGCDwvyzGhEvTdjUDRPORe7~DeYuRDEjfUOnBG4GLov4~h0KZpuvZUFapu8mgKrntdv3fh1rP1jZASpgWuodsArIRvQz06aIEx2PlnuscQe0GrBdLAqjrMneklq9IEi5sFZ007QsnVM1eTXeMeOmuGQT7n01XXQRJaK8~gQLaoySLTQ4Fe-iYFnVnJRh~JPjT98nO1r-6wwlCnDS3i0Cuacx0dztGKpYJSbU5ZjDKmMGyt1vf927gqNLHJKxPjf3Z0ubeHjjVnAM7-FBMcO0a6OrprS5WeDz4CbgVKQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>DNARES</em> online.</p>                            <h3 scrollto-destination=104086848 id="104086848" class="section-title" >Funding</h3>
<p>This research was supported by Honda Research Institute Japan and the Kazusa DNA Research Foundation.</p>                            <h3 scrollto-destination=104086850 id="104086850" class="section-title" >Conflict of interest</h3>
<p>None declared. </p>                            <h2 scrollto-destination=104086852 id="104086852" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx032-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx032-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">van Dyk</div> <div class="given-names">J.S.</div></div>, <div class="name string-name"><div class="surname">Pletschke</div> <div class="given-names">B.I.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">A review of lignocellulose bioconversion using enzymatic hydrolysis and synergistic cooperation between enzymes-Factors affecting enzymes, conversion and synergy</div>, <div class="source">Biotechnol. Adv</div> ., <div class="volume">30</div>, <div class="fpage">1458</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20review%20of%20lignocellulose%20bioconversion%20using%20enzymatic%20hydrolysis%20and%20synergistic%20cooperation%20between%20enzymes-Factors%20affecting%20enzymes%2C%20conversion%20and%20synergy&amp;author=J.S.%20van%20Dyk&amp;author=B.I.%20Pletschke&amp;publication_year=2011&amp;journal=Biotechnol.%20Adv&amp;volume=30&amp;pages=1458-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.biotechadv.2012.03.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.biotechadv.2012.03.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.biotechadv.2012.03.002"> </span></div> </div></div></div></div></div><div content-id="dsx032-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx032-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schomburg</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Chang</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Placzek</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">BRENDA in 2013: integrated reactions, kinetic data, enzyme function data, improved disease classification: new options and contents in BRENDA,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">41</div>, <div class="fpage">D764</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BRENDA%20in%202013%3A%20integrated%20reactions%2C%20kinetic%20data%2C%20enzyme%20function%20data%2C%20improved%20disease%20classification%3A%20new%20options%20and%20contents%20in%20BRENDA%2C&amp;author=I.%20Schomburg&amp;author=A.%20Chang&amp;author=S.%20Placzek&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;pages=D764-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks1049" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks1049" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks1049"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23203881" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx032-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx032-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Haki</div> <div class="given-names">G.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Developments in industrially important thermostable enzymes: a review,</div> <div class="source">Bioresour. Technol</div> ., <div class="volume">89</div>, <div class="fpage">17</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Developments%20in%20industrially%20important%20thermostable%20enzymes%3A%20a%20review%2C&amp;author=G.%20Haki&amp;publication_year=2003&amp;journal=Bioresour.%20Technol&amp;volume=89&amp;pages=17-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0960-8524(03)00033-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0960-8524(03)00033-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0960-8524(03)00033-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12676497" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx032-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx032-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xue</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Shao</div> <div class="given-names">W.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Expression and characterization of a thermostable beta-xylosidase from the hyperthermophile, <em>Thermotoga maritima</em>,</div> <div class="source">Biotechnol. Lett</div> ., <div class="volume">26</div>, <div class="fpage">1511</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20and%20characterization%20of%20a%20thermostable%20beta-xylosidase%20from%20the%20hyperthermophile%2C%20Thermotoga%20maritima%2C&amp;author=Y.%20Xue&amp;author=W.%20Shao&amp;publication_year=2004&amp;journal=Biotechnol.%20Lett&amp;volume=26&amp;pages=1511-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/B:BILE.0000044454.70768.81" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FB:BILE.0000044454.70768.81" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FB:BILE.0000044454.70768.81"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15604789" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx032-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx032-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shi</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Gu</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Biochemical properties of a novel thermostable and highly xylose-tolerant β-xylosidase/α-arabinosidase from <em>Thermotoga thermarum</em>,</div> <div class="source">Biotechnol. Biofuels</div> , <div class="volume">6</div>, <div class="fpage">27</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biochemical%20properties%20of%20a%20novel%20thermostable%20and%20highly%20xylose-tolerant%20%CE%B2-xylosidase%2F%CE%B1-arabinosidase%20from%20Thermotoga%20thermarum%2C&amp;author=H.%20Shi&amp;author=X.%20Li&amp;author=H.%20Gu&amp;publication_year=2013&amp;journal=Biotechnol.%20Biofuels&amp;volume=6&amp;pages=27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1754-6834-6-27" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1754-6834-6-27" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1754-6834-6-27"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23422003" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx032-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx032-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Su</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Han</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Dodd</div> <div class="given-names">D.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Reconstitution of a thermostable xylan-degrading enzyme mixture from the bacterium <em>Caldicellulosiruptor bescii</em>,</div> <div class="source">Appl. Environ. Microbiol</div> ., <div class="volume">79</div>, <div class="fpage">1481</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reconstitution%20of%20a%20thermostable%20xylan-degrading%20enzyme%20mixture%20from%20the%20bacterium%20Caldicellulosiruptor%20bescii%2C&amp;author=X.%20Su&amp;author=Y.%20Han&amp;author=D.%20Dodd&amp;publication_year=2013&amp;journal=Appl.%20Environ.%20Microbiol&amp;volume=79&amp;pages=1481-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.03265-12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.03265-12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.03265-12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23263957" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx032-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx032-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shao</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Wiegel</div> <div class="given-names">J.</div></div></div> <div class="year">1992</div>, 
<div class="article-title">Purification and characterization of a thermostable β-xylosidase from <em>Thermoanaerobacter ethanolicus</em>,</div> <div class="source">J. Bacteriol</div> ., <div class="volume">174</div>, <div class="fpage">5848</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Purification%20and%20characterization%20of%20a%20thermostable%20%CE%B2-xylosidase%20from%20Thermoanaerobacter%20ethanolicus%2C&amp;author=W.%20Shao&amp;author=J.%20Wiegel&amp;publication_year=1992&amp;journal=J.%20Bacteriol&amp;volume=174&amp;pages=5848-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/jb.174.18.5848-5853.1992" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2Fjb.174.18.5848-5853.1992" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2Fjb.174.18.5848-5853.1992"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1522062" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx032-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx032-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Morana</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Paris</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Maurelli</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Rossi</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Cannio</div> <div class="given-names">R.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Gene cloning and expression in <em>Escherichia coli</em> of a bi-functional beta-D-xylosidase/alpha-L-arabinosidase from <em>Sulfolobus solfataricus</em> involved in xylan degradation,</div> <div class="source">Extremophiles</div> , <div class="volume">11</div>, <div class="fpage">123</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20cloning%20and%20expression%20in%20Escherichia%20coli%20of%20a%20bi-functional%20beta-D-xylosidase%2Falpha-L-arabinosidase%20from%20Sulfolobus%20solfataricus%20involved%20in%20xylan%20degradation%2C&amp;author=A.%20Morana&amp;author=O.%20Paris&amp;author=L.%20Maurelli&amp;author=M.%20Rossi&amp;author=R.%20Cannio&amp;publication_year=2007&amp;journal=Extremophiles&amp;volume=11&amp;pages=123-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00792-006-0020-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00792-006-0020-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00792-006-0020-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17033733" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx032-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx032-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dobberstein</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Emeis</div> <div class="given-names">C.C.</div></div></div> <div class="year">1991</div>, 
<div class="article-title">Purification and characterization of β-xylosidase from <em>Aureobasidium pullulans</em>,</div> <div class="source">Appl. Microbiol. Biotechnol</div> ., <div class="volume">35</div>, <div class="fpage">210</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Purification%20and%20characterization%20of%20%CE%B2-xylosidase%20from%20Aureobasidium%20pullulans%2C&amp;author=J.%20Dobberstein&amp;author=C.C.%20Emeis&amp;publication_year=1991&amp;journal=Appl.%20Microbiol.%20Biotechnol&amp;volume=35&amp;pages=210-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00184688" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00184688" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00184688"> </span></div> </div></div></div></div></div><div content-id="dsx032-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx032-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sebastian</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">J.Y.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">T.H.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">K.T.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Metagenomics: a promissing approach to assess enzymes biocatalyst for biofuel production,</div> <div class="source">Asian J. Biotechnol</div> ., <div class="volume">5</div>, <div class="fpage">33</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metagenomics%3A%20a%20promissing%20approach%20to%20assess%20enzymes%20biocatalyst%20for%20biofuel%20production%2C&amp;author=R.%20Sebastian&amp;author=J.Y.%20Kim&amp;author=T.H.%20Kim&amp;author=K.T.%20Lee&amp;publication_year=2013&amp;journal=Asian%20J.%20Biotechnol&amp;volume=5&amp;pages=33-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3923/ajbkr.2013.33.50" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3923%2Fajbkr.2013.33.50" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3923%2Fajbkr.2013.33.50"> </span></div> </div></div></div></div></div><div content-id="dsx032-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx032-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Allgaier</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Reddy</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Park</div> <div class="given-names">J.I.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Targeted discovery of glycoside hydrolases from a switchgrass-adapted compost community,</div> <div class="source">PLoS One</div> , <div class="volume">5</div>, <div class="fpage">e8812</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeted%20discovery%20of%20glycoside%20hydrolases%20from%20a%20switchgrass-adapted%20compost%20community%2C&amp;author=M.%20Allgaier&amp;author=A.%20Reddy&amp;author=J.I.%20Park&amp;publication_year=2010&amp;journal=PLoS%20One&amp;volume=5&amp;pages=e8812" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0008812" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0008812" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0008812"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20098679" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx032-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx032-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bao</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Chang</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Sun</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Lu</div> <div class="given-names">H.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Cloning and characterization of two β-glucosidase/xylosidase enzymes from yak rumen metagenome,</div> <div class="source">Appl. Biochem. Biotechnol</div> ., <div class="volume">166</div>, <div class="fpage">72</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cloning%20and%20characterization%20of%20two%20%CE%B2-glucosidase%2Fxylosidase%20enzymes%20from%20yak%20rumen%20metagenome%2C&amp;author=L.%20Bao&amp;author=Q.%20Huang&amp;author=L.%20Chang&amp;author=Q.%20Sun&amp;author=J.%20Zhou&amp;author=H.%20Lu&amp;publication_year=2012&amp;journal=Appl.%20Biochem.%20Biotechnol&amp;volume=166&amp;pages=72-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s12010-011-9405-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs12010-011-9405-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs12010-011-9405-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22020745" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx032-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx032-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Noguchi</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Taniguchi</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Itoh</div> <div class="given-names">T.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">MetaGeneAnnotator: detecting species-specific patterns of ribosomal binding site for precise gene prediction in anonymous prokaryotic and phage genomes,</div> <div class="source">DNA Res</div> ., <div class="volume">15</div>, <div class="fpage">387</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MetaGeneAnnotator%3A%20detecting%20species-specific%20patterns%20of%20ribosomal%20binding%20site%20for%20precise%20gene%20prediction%20in%20anonymous%20prokaryotic%20and%20phage%20genomes%2C&amp;author=H.%20Noguchi&amp;author=T.%20Taniguchi&amp;author=T.%20Itoh&amp;publication_year=2008&amp;journal=DNA%20Res&amp;volume=15&amp;pages=387-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsn027" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsn027" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsn027"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18940874" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx032-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx032-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Larkin</div> <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Blackshields</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Brown</div> <div class="given-names">N.P.</div></div></div>, et al.   <div class="year">2007</div>, 
<div class="article-title">Clustal W and Clustal X version 2.0,</div> <div class="source">Bioinformatics</div> , <div class="volume">23</div>, <div class="fpage">2947</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Clustal%20W%20and%20Clustal%20X%20version%202.0%2C&amp;author=M.A.%20Larkin&amp;author=G.%20Blackshields&amp;author=N.P.%20Brown&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=2947-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm404" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm404" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm404"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17846036" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx032-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx032-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Campos</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Alvarez</div> <div class="given-names">M.J.N.</div></div>, <div class="name string-name"><div class="surname">di Lorenzo</div> <div class="given-names">G.S.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Purification and characterization of a GH43 β-xylosidase from <em>Enterobacter</em> sp. identified and cloned from forest soil bacteria</div>. <div class="source">Microbiol. Res</div> ., <div class="volume">169</div>, <div class="fpage">213</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Purification%20and%20characterization%20of%20a%20GH43%20%CE%B2-xylosidase%20from%20Enterobacter%20sp.%20identified%20and%20cloned%20from%20forest%20soil%20bacteria&amp;author=E.%20Campos&amp;author=M.J.N.%20Alvarez&amp;author=G.S.%20di%20Lorenzo&amp;publication_year=2014&amp;journal=Microbiol.%20Res&amp;volume=169&amp;pages=213-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.micres.2013.06.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.micres.2013.06.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.micres.2013.06.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23838121" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx032-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx032-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">Y.K.</div></div>, <div class="name string-name"><div class="surname">Yao</div> <div class="given-names">H.J.</div></div>, <div class="name string-name"><div class="surname">Cho</div> <div class="given-names">Y.T.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Effective induction, purification and characterization of <em>Trichoderma koningii</em> G-39 beta-xylosidase with high transferase activity</div>. <div class="source">Biotechnol. Appl. Biochem</div> ., <div class="volume">31</div>, <div class="fpage">119</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effective%20induction%2C%20purification%20and%20characterization%20of%20Trichoderma%20koningii%20G-39%20beta-xylosidase%20with%20high%20transferase%20activity&amp;author=Y.K.%20Li&amp;author=H.J.%20Yao&amp;author=Y.T.%20Cho&amp;publication_year=2000&amp;journal=Biotechnol.%20Appl.%20Biochem&amp;volume=31&amp;pages=119-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1042/BA19990072" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1042%2FBA19990072" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1042%2FBA19990072"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10744957" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx032-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx032-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cheng</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Romantsov</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Engel</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">Functional metagenomics reveals novel β-galactosidases not predictable from gene sequences</div>. <div class="source">PLoS One</div> , <div class="volume">12</div>, <div class="fpage">e0172545.</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20metagenomics%20reveals%20novel%20%CE%B2-galactosidases%20not%20predictable%20from%20gene%20sequences&amp;author=J.%20Cheng&amp;author=T.%20Romantsov&amp;author=K.%20Engel&amp;publication_year=2017&amp;journal=PLoS%20One&amp;volume=12&amp;pages=e0172545." target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0172545" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0172545" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0172545"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28273103" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=4077064&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/supplementaryfigure1_dsx032.pdf?Expires=2147483647&amp;Signature=M2HY~7Xd846OMGROjYCtl7ItRmUSq1w-V8-MfqosfQe7yFdyGCDwvyzGhEvTdjUDRPORe7~DeYuRDEjfUOnBG4GLov4~h0KZpuvZUFapu8mgKrntdv3fh1rP1jZASpgWuodsArIRvQz06aIEx2PlnuscQe0GrBdLAqjrMneklq9IEi5sFZ007QsnVM1eTXeMeOmuGQT7n01XXQRJaK8~gQLaoySLTQ4Fe-iYFnVnJRh~JPjT98nO1r-6wwlCnDS3i0Cuacx0dztGKpYJSbU5ZjDKmMGyt1vf927gqNLHJKxPjf3Z0ubeHjjVnAM7-FBMcO0a6OrprS5WeDz4CbgVKQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S1</div></a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/supplementaryfigure2_dsx032.pdf?Expires=2147483647&amp;Signature=GRClZFfOZJ3J1QExWkFVI~yBYcA8hn8e8x5rn5Oz2OPtAwRx1UaaqaYPAwcC4Ca5BEMa772t2u~urAY5R0rJwZhBYUB-utm9mxb44ckBwTQ1et~G7GErQxG70T-mfvMD4tBgioqSkBpmQl6LVMkWZdUbbLYdwmp-8iiSnDiZOGIsl9t1M7VldDJJskNxI0rNmC0UuSZTopyeLqJz0N67ZqznglNf28nCuOGYE~qc8NC7~nkeq10LlnAPFyEmvbrvx489qjDLMWSL~w9KmO8h2REKskVsJja57foEPpHWqCo03fGtsaf0pQi6ry-dCgr5rAs0nqyrNW35U4h~HVwF6A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S2</div></a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/supplementarytable1_dsx032.xlsx?Expires=2147483647&amp;Signature=oqMJ5OPbNto0Ip6xQDjOquzoVF8X58CfeEzBGrHxWs~QEjk1HfKCaQn3FiDtPOR7q1D6ypX9tHZ8X0bZVMOClegKao8t~6ZEzfvEYi-PGqB9Zennj8yaIGdk0Bi5bxHe~pt1RVMqGeIUtrGtHwvpiD8npDJdavPg2SLJmnZ6v8vOJIIrDfq5thnHDm5kP1Yp1BHeRZ8qttcXb0gvep9x6KrNMSPxucck3uphUTzFoVD2EJQY5MM2iC6nWyXlu6xBRhhBbjFW-a5vwOmzrn7dqA3CCmVLlWGlshME4AfMHBMvyaGua~BcnwaxDwe57hzzjgqX24K1Yb5vV755UHBUzA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S1</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx032/1/supplementarytable2_dsx032.xlsx?Expires=2147483647&amp;Signature=JNnlo4GHLlOUrfPhnvYGj~Az4X8XQrk0mk~g2AlYnNgcDH4G-2y0pVyFqdm7rGCXr0lt4oMacdPQf3HTEkO~M9~2r~i0zIjHRGEge7AwIqwbmcmABwdui~l3k4mvWd9NSbR1HxLO7blInZQ8eHTtHXtW7tA5aewnorlLAEwArZpbToKXdub5nLlUwHEclHENacTxDDEbOA2kN95JqwcumSlbKj3f2aQGbqwPUpafg7gGZtZDTb~aubM05S7Psw26-kwEl-jvTkYG12DqbDt4kYe0wHg2lrevvHEuAM86iJiwvQJRL8lpG4H7ChXMZO~-s5yIN5-uJWToTvvBc~oFTw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S2</div></a> - xlsx file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">760</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx032"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="4077064"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f6%2f649%2f4077064">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F6%2F649%2F4077064" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/4077064'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Isolation of highly thermostable β-xylosidases from a hot spring soil microbial community using a metagenomic approach" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx032");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f6%2f649%2f4077064">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

