

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/21/5/459/408276","@type":"ScholarlyArticle","name":"Evolutionary Changes in Vertebrate Genome Signatures with Special Focus on Coelacanth","datePublished":"2014-05-06","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/21/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2014-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsu012","keywords":["big data","epigenetic","SOM","DNA methylation","CG suppression"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/21/5/459/408276","author":[{"name":"Iwasaki, Yuki","affiliation":"1  Department of Bioscience ,  Nagahama Institute of Bio-Science and Technology ,  Nagahama, Shiga 526-0829 ,  Japan","@type":"Person"},{"name":"Abe, Takashi","affiliation":"2  Department of Information Engineering, Faculty of Engineering ,  Institute of Science and Technology ,  Niigata University ,  Niigata-ken 950-2181 ,  Japan","@type":"Person"},{"name":"Okada, Norihiro","affiliation":"1  Department of Bioscience ,  Nagahama Institute of Bio-Science and Technology ,  Nagahama, Shiga 526-0829 ,  Japan    3  Faculty of Bioscience and Biotechnology ,  Tokyo Institute of Technology ,  Yokohama, Kanagawa 226 ,  Japan    4  Department of Life Sciences ,  National Cheng Kung University ,  Tainan 701 ,  Taiwan","@type":"Person"},{"name":"Wada, Kennosuke","affiliation":"1  Department of Bioscience ,  Nagahama Institute of Bio-Science and Technology ,  Nagahama, Shiga 526-0829 ,  Japan","@type":"Person"},{"name":"Wada, Yoshiko","affiliation":"1  Department of Bioscience ,  Nagahama Institute of Bio-Science and Technology ,  Nagahama, Shiga 526-0829 ,  Japan","@type":"Person"},{"name":"Ikemura, Toshimichi","affiliation":"1  Department of Bioscience ,  Nagahama Institute of Bio-Science and Technology ,  Nagahama, Shiga 526-0829 ,  Japan","@type":"Person"}],"description":"Abstract.  With a remarkable increase in genomic sequence data of a wide range of species, novel tools are needed for comprehensive analyses of the big sequence","pageStart":"459","pageEnd":"467","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/5/1/m_cover.gif?Expires=2147483647&Signature=gPiJSb93IIOcIMTSkMAi7kcKhziIjvuRKEzK8XVp8CTVu4eB811a4N7c5iRqAkFTvycBeFOL1qu9DtcvWUQE-HFcU90uhCx7UXftpGVaIU0r-58rykRAQjr7OyCGXhRRXufrlhLC-AFU8QeGWWE7goVK9s00kVi5KaqA8BdZOl4ZUh6UGigqEAXS9fKMJcyv7L69eSvyhk8fr0u2lpE6Z~kKXJ-zph2mYgOTE6TZsnpM7S63PaSYvqdIQyt~fHxa7pkm4binASfSKcJvbv9WOoMwYew88jtLZv495c~AkAPvlFj827Es8r960R62hvuI-oun7Y-AfhyLYD5usT~QKA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Evolutionary Changes in Vertebrate Genome Signatures with Special Focus on Coelacanth","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/5/1/m_cover.gif?Expires=2147483647&Signature=gPiJSb93IIOcIMTSkMAi7kcKhziIjvuRKEzK8XVp8CTVu4eB811a4N7c5iRqAkFTvycBeFOL1qu9DtcvWUQE-HFcU90uhCx7UXftpGVaIU0r-58rykRAQjr7OyCGXhRRXufrlhLC-AFU8QeGWWE7goVK9s00kVi5KaqA8BdZOl4ZUh6UGigqEAXS9fKMJcyv7L69eSvyhk8fr0u2lpE6Z~kKXJ-zph2mYgOTE6TZsnpM7S63PaSYvqdIQyt~fHxa7pkm4binASfSKcJvbv9WOoMwYew88jtLZv495c~AkAPvlFj827Es8r960R62hvuI-oun7Y-AfhyLYD5usT~QKA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Evolutionary Changes in Vertebrate Genome Signatures with Special Focus on Coelacanth" />
    <meta property="og:description" content="Abstract.  With a remarkable increase in genomic sequence data of a wide range of species, novel tools are needed for comprehensive analyses of the big sequence" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/21/5/459/408276" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2014-05-06" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Iwasaki, Yuki" /><meta name="citation_author_institution" content="Department of Bioscience, Nagahama Institute of Bio-Science and Technology, Nagahama, Shiga 526-0829, Japan" /><meta name="citation_author" content="Abe, Takashi" /><meta name="citation_author_institution" content="Department of Information Engineering, Faculty of Engineering, Institute of Science and Technology, Niigata University, Niigata-ken 950-2181, Japan" /><meta name="citation_author" content="Okada, Norihiro" /><meta name="citation_author_institution" content="Department of Bioscience, Nagahama Institute of Bio-Science and Technology, Nagahama, Shiga 526-0829, Japan" /><meta name="citation_author_institution" content="Faculty of Bioscience and Biotechnology, Tokyo Institute of Technology, Yokohama, Kanagawa 226, Japan" /><meta name="citation_author_institution" content="Department of Life Sciences, National Cheng Kung University, Tainan 701, Taiwan" /><meta name="citation_author" content="Wada, Kennosuke" /><meta name="citation_author_institution" content="Department of Bioscience, Nagahama Institute of Bio-Science and Technology, Nagahama, Shiga 526-0829, Japan" /><meta name="citation_author" content="Wada, Yoshiko" /><meta name="citation_author_institution" content="Department of Bioscience, Nagahama Institute of Bio-Science and Technology, Nagahama, Shiga 526-0829, Japan" /><meta name="citation_author" content="Ikemura, Toshimichi" /><meta name="citation_author_institution" content="Department of Bioscience, Nagahama Institute of Bio-Science and Technology, Nagahama, Shiga 526-0829, Japan" /><meta name="citation_title" content="Evolutionary Changes in Vertebrate Genome Signatures with Special Focus on Coelacanth" /><meta name="citation_firstpage" content="459" /><meta name="citation_lastpage" content="467" /><meta name="citation_doi" content="10.1093/dnares/dsu012" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="21" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2014/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Doublet frequencies in evolutionary distinct groups; Nucleic Acids Res.;  citation_year=1984;  citation_volume=12;  citation_pages=1749-63; " /><meta name="citation_reference" content="citation_title=Comparative DNA analysis across diverse genomes; Annu. Rev. Genet.;  citation_year=1998;  citation_volume=32;  citation_pages=185-225; " /><meta name="citation_reference" content="citation_title=Genome-scale compositional comparisons in eukaryotes; Genome Res.;  citation_year=2001;  citation_volume=11;  citation_pages=540-6; " /><meta name="citation_reference" content="citation_title=Self-organized formation of topologically correct feature maps; Biol. Cybern.;  citation_year=1982;  citation_volume=43;  citation_pages=59-69; " /><meta name="citation_reference" content="citation_title=Engineering applications of the self-organizing map; Proc. IEEE;  citation_year=1996;  citation_volume=84;  citation_pages=1358-84; " /><meta name="citation_reference" content="citation_title=Analysis of codon usage diversity of bacterial genes with a self-organizing map (SOM): characterization of horizontally transferred genes with emphasis on the E. coli O157 genome; Gene;  citation_year=2001;  citation_volume=276;  citation_pages=89-99; " /><meta name="citation_reference" content="citation_title=Informatics for unveiling hidden genome signatures; Genome Res.;  citation_year=2003;  citation_volume=13;  citation_pages=693-702; " /><meta name="citation_reference" content="citation_title=Novel phylogenetic studies of genomic sequence fragments derived from uncultured microbe mixtures in environmental and clinical samples; DNA Res.;  citation_year=2005;  citation_volume=12;  citation_pages=281-90; " /><meta name="citation_reference" content="citation_title=A large-scale self-organizing map (SOM) unveils sequence characteristics of a wide range of eukaryote genomes; Gene;  citation_year=2006;  citation_volume=365;  citation_pages=27-34; " /><meta name="citation_reference" content="citation_title=Coelacanth genome sequence reveals the evolutionary history of vertebrate genes; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=2397-405; " /><meta name="citation_reference" content="citation_title=The African coelacanth genome provides insights into tetrapod evolution; Nature;  citation_year=2013;  citation_volume=496;  citation_pages=311-6; " /><meta name="citation_reference" content="citation_title=Coelacanth genomes reveal signatures for evolutionary transition from water to land; Genome Res.;  citation_year=2013;  citation_volume=23;  citation_pages=1740-8; " /><meta name="citation_reference" content="citation_title=Replication-associated strand asymmetries in mammalian genomes: toward detection of replication origins; Proc. Natl. Acad. Sci. USA;  citation_year=2005;  citation_volume=102;  citation_pages=9836-41; " /><meta name="citation_reference" content="citation_title=Prediction of directional changes of influenza A virus genome sequences with emphasis on pandemic H1N1/09 as a model case; DNA Res.;  citation_year=2011;  citation_volume=18;  citation_pages=125-36; " /><meta name="citation_reference" content="citation_title=Genome-scale relationships between cytosine methylation and dinucleotide abundances in animals; Genomics;  citation_year=2008;  citation_volume=92;  citation_pages=33-40; " /><meta name="citation_reference" content="citation_title=The mosaic genome of warm-blooded vertebrates; Science;  citation_year=1985;  citation_volume=228;  citation_pages=953-8; " /><meta name="citation_reference" content="citation_title=Codon usage and transfer RNA content in unicellular and multicellular organisms; Mol. Biol. Evol.;  citation_year=1985;  citation_volume=2;  citation_pages=13-34; " /><meta name="citation_reference" content="citation_title=Global variation in G+C content along vertebrate genome DNA: possible correlation with chromosome band structures; J. Mol. Biol.;  citation_year=1988;  citation_volume=203;  citation_pages=1-13; " /><meta name="citation_reference" content="citation_title=Evident diversity of codon usage patterns of human genes with respect to chromosome banding patterns and chromosome numbers; relation between nucleotide sequence data and cytogenetic data; Nucl. Acids Res.;  citation_year=1991;  citation_volume=19;  citation_pages=4333-9; " /><meta name="citation_reference" content="citation_publisher=Elsevier, New York; Structural and Evolutionary Genomics: Natural Selection in Genome Evolution;  citation_year=2004; " /><meta name="citation_reference" content="citation_title=Cytosine methylation and mammalian development; Genes Dev.;  citation_year=1999;  citation_volume=13;  citation_pages=26-34; " /><meta name="citation_reference" content="citation_title=Cytosine methylation and the ecology of intragenomic parasites; Trends Genet.;  citation_year=1997;  citation_volume=13;  citation_pages=335-40; " /><meta name="citation_reference" content="citation_title=DNA of Drosophila melanogaster contains 5-methylcytosine; EMBO J.;  citation_year=2000;  citation_volume=24;  citation_pages=6918-23; " /><meta name="citation_reference" content="citation_title=DNA methylation in Drosophila melanogaster; Nature;  citation_year=2000;  citation_volume=408;  citation_pages=538-40; " /><meta name="citation_reference" content="citation_title=Identification of a gene required for de novo DNA methylation of the zebrafish no tail gene; Dev. Dyn.;  citation_year=2005;  citation_volume=233;  citation_pages=1509-16; " /><meta name="citation_reference" content="citation_title=Novel splice variants associated with one of the zebrafish dnmt3 genes; BMC Dev. Biol.;  citation_year=2005;  citation_volume=5;  citation_pages=23" /><meta name="citation_reference" content="citation_title=Expression of the dnmt3 genes in zebrafish development: similarity to Dnmt3a and Dnmt3b; Dev. Genes. Evol.;  citation_year=2011;  citation_volume=220;  citation_pages=347-53; " /><meta name="citation_reference" content="citation_title=MEGA6: molecular evolutionary genetics analysis version 6.0; Mol. Biol. Evol.;  citation_year=2013;  citation_volume=30;  citation_pages=2725-9; " /><meta name="citation_reference" content="citation_title=RNA-directed transcriptional gene silencing in mammals; Trends Genet.;  citation_year=2005;  citation_volume=21;  citation_pages=370-3; " /><meta name="citation_reference" content="citation_title=Notable clustering of transcription-factor-binding motifs in human pericentric regions and its biological significance; Chromosome Res.;  citation_year=2013;  citation_volume=21;  citation_pages=461-74; " /><meta name="citation_reference" content="citation_title=Sequences from almost all prokaryotic, eukaryotic, and viral genomes available could be classified according to genomes on a large-scale self-organizing map constructed with the earth simulator; J Earth Simulator;  citation_year=2006;  citation_volume=6;  citation_pages=17-23; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/21/5/459/1138043/dsu012.pdf" /><meta name="description" content="Abstract.  With a remarkable increase in genomic sequence data of a wide range of species, novel tools are needed for comprehensive analyses of the big sequence" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/21/5/459/408276" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/21/5/459/408276" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/21/5/459/408276");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsu012';
    </script>
<title>
	Evolutionary Changes in Vertebrate Genome Signatures with Special Focus on Coelacanth | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./408276" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bd5LPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJd2FzYWtpLCBZdWtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9zY2llbmNlLCBOYWdhaGFtYSBJbnN0aXR1dGUgb2YgQmlvLVNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIE5hZ2FoYW1hLCBTaGlnYSA1MjYtMDgyOSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBYmUsIFRha2FzaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEluZm9ybWF0aW9uIEVuZ2luZWVyaW5nLCBGYWN1bHR5IG9mIEVuZ2luZWVyaW5nLCBJbnN0aXR1dGUgb2YgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgTmlpZ2F0YSBVbml2ZXJzaXR5LCBOaWlnYXRhLWtlbiA5NTAtMjE4MSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPa2FkYSwgTm9yaWhpcm8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb3NjaWVuY2UsIE5hZ2FoYW1hIEluc3RpdHV0ZSBvZiBCaW8tU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgTmFnYWhhbWEsIFNoaWdhIDUyNi0wODI5LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkZhY3VsdHkgb2YgQmlvc2NpZW5jZSBhbmQgQmlvdGVjaG5vbG9neSwgVG9reW8gSW5zdGl0dXRlIG9mIFRlY2hub2xvZ3ksIFlva29oYW1hLCBLYW5hZ2F3YSAyMjYsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBMaWZlIFNjaWVuY2VzLCBOYXRpb25hbCBDaGVuZyBLdW5nIFVuaXZlcnNpdHksIFRhaW5hbiA3MDEsIFRhaXdhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IldhZGEsIEtlbm5vc3VrZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvc2NpZW5jZSwgTmFnYWhhbWEgSW5zdGl0dXRlIG9mIEJpby1TY2llbmNlIGFuZCBUZWNobm9sb2d5LCBOYWdhaGFtYSwgU2hpZ2EgNTI2LTA4MjksIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iV2FkYSwgWW9zaGlrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvc2NpZW5jZSwgTmFnYWhhbWEgSW5zdGl0dXRlIG9mIEJpby1TY2llbmNlIGFuZCBUZWNobm9sb2d5LCBOYWdhaGFtYSwgU2hpZ2EgNTI2LTA4MjksIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSWtlbXVyYSwgVG9zaGltaWNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvc2NpZW5jZSwgTmFnYWhhbWEgSW5zdGl0dXRlIG9mIEJpby1TY2llbmNlIGFuZCBUZWNobm9sb2d5LCBOYWdhaGFtYSwgU2hpZ2EgNTI2LTA4MjksIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJFdm9sdXRpb25hcnkgQ2hhbmdlcyBpbiBWZXJ0ZWJyYXRlIEdlbm9tZSBTaWduYXR1cmVzIHdpdGggU3BlY2lhbCBGb2N1cyBvbiBDb2VsYWNhbnRoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iNDU5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI0NjciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3UwMTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNC8xMC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURvdWJsZXQgZnJlcXVlbmNpZXMgaW4gZXZvbHV0aW9uYXJ5IGRpc3RpbmN0IGdyb3VwczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTg0OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTc0OS02MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgRE5BIGFuYWx5c2lzIGFjcm9zcyBkaXZlcnNlIGdlbm9tZXM7IEFubnUuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9MTg1LTIyNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXNjYWxlIGNvbXBvc2l0aW9uYWwgY29tcGFyaXNvbnMgaW4gZXVrYXJ5b3RlczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz01NDAtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VsZi1vcmdhbml6ZWQgZm9ybWF0aW9uIG9mIHRvcG9sb2dpY2FsbHkgY29ycmVjdCBmZWF0dXJlIG1hcHM7IEJpb2wuIEN5YmVybi47ICBjaXRhdGlvbl95ZWFyPTE5ODI7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz01OS02OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW5naW5lZXJpbmcgYXBwbGljYXRpb25zIG9mIHRoZSBzZWxmLW9yZ2FuaXppbmcgbWFwOyBQcm9jLiBJRUVFOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTg0OyAgY2l0YXRpb25fcGFnZXM9MTM1OC04NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgY29kb24gdXNhZ2UgZGl2ZXJzaXR5IG9mIGJhY3RlcmlhbCBnZW5lcyB3aXRoIGEgc2VsZi1vcmdhbml6aW5nIG1hcCAoU09NKTogY2hhcmFjdGVyaXphdGlvbiBvZiBob3Jpem9udGFsbHkgdHJhbnNmZXJyZWQgZ2VuZXMgd2l0aCBlbXBoYXNpcyBvbiB0aGUgRS4gY29saSBPMTU3IGdlbm9tZTsgR2VuZTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzY7ICBjaXRhdGlvbl9wYWdlcz04OS05OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5mb3JtYXRpY3MgZm9yIHVudmVpbGluZyBoaWRkZW4gZ2Vub21lIHNpZ25hdHVyZXM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9NjkzLTcwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm92ZWwgcGh5bG9nZW5ldGljIHN0dWRpZXMgb2YgZ2Vub21pYyBzZXF1ZW5jZSBmcmFnbWVudHMgZGVyaXZlZCBmcm9tIHVuY3VsdHVyZWQgbWljcm9iZSBtaXh0dXJlcyBpbiBlbnZpcm9ubWVudGFsIGFuZCBjbGluaWNhbCBzYW1wbGVzOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTI4MS05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBsYXJnZS1zY2FsZSBzZWxmLW9yZ2FuaXppbmcgbWFwIChTT00pIHVudmVpbHMgc2VxdWVuY2UgY2hhcmFjdGVyaXN0aWNzIG9mIGEgd2lkZSByYW5nZSBvZiBldWthcnlvdGUgZ2Vub21lczsgR2VuZTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNjU7ICBjaXRhdGlvbl9wYWdlcz0yNy0zNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29lbGFjYW50aCBnZW5vbWUgc2VxdWVuY2UgcmV2ZWFscyB0aGUgZXZvbHV0aW9uYXJ5IGhpc3Rvcnkgb2YgdmVydGVicmF0ZSBnZW5lczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0yMzk3LTQwNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEFmcmljYW4gY29lbGFjYW50aCBnZW5vbWUgcHJvdmlkZXMgaW5zaWdodHMgaW50byB0ZXRyYXBvZCBldm9sdXRpb247IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00OTY7ICBjaXRhdGlvbl9wYWdlcz0zMTEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29lbGFjYW50aCBnZW5vbWVzIHJldmVhbCBzaWduYXR1cmVzIGZvciBldm9sdXRpb25hcnkgdHJhbnNpdGlvbiBmcm9tIHdhdGVyIHRvIGxhbmQ7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MTc0MC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXBsaWNhdGlvbi1hc3NvY2lhdGVkIHN0cmFuZCBhc3ltbWV0cmllcyBpbiBtYW1tYWxpYW4gZ2Vub21lczogdG93YXJkIGRldGVjdGlvbiBvZiByZXBsaWNhdGlvbiBvcmlnaW5zOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDI7ICBjaXRhdGlvbl9wYWdlcz05ODM2LTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVkaWN0aW9uIG9mIGRpcmVjdGlvbmFsIGNoYW5nZXMgb2YgaW5mbHVlbnphIEEgdmlydXMgZ2Vub21lIHNlcXVlbmNlcyB3aXRoIGVtcGhhc2lzIG9uIHBhbmRlbWljIEgxTjEvMDkgYXMgYSBtb2RlbCBjYXNlOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTEyNS0zNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXNjYWxlIHJlbGF0aW9uc2hpcHMgYmV0d2VlbiBjeXRvc2luZSBtZXRoeWxhdGlvbiBhbmQgZGludWNsZW90aWRlIGFidW5kYW5jZXMgaW4gYW5pbWFsczsgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTI7ICBjaXRhdGlvbl9wYWdlcz0zMy00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG1vc2FpYyBnZW5vbWUgb2Ygd2FybS1ibG9vZGVkIHZlcnRlYnJhdGVzOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0xOTg1OyAgY2l0YXRpb25fdm9sdW1lPTIyODsgIGNpdGF0aW9uX3BhZ2VzPTk1My04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiB1c2FnZSBhbmQgdHJhbnNmZXIgUk5BIGNvbnRlbnQgaW4gdW5pY2VsbHVsYXIgYW5kIG11bHRpY2VsbHVsYXIgb3JnYW5pc21zOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTg1OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0xMy0zNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2xvYmFsIHZhcmlhdGlvbiBpbiBHK0MgY29udGVudCBhbG9uZyB2ZXJ0ZWJyYXRlIGdlbm9tZSBETkE6IHBvc3NpYmxlIGNvcnJlbGF0aW9uIHdpdGggY2hyb21vc29tZSBiYW5kIHN0cnVjdHVyZXM7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODg7ICBjaXRhdGlvbl92b2x1bWU9MjAzOyAgY2l0YXRpb25fcGFnZXM9MS0xMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZpZGVudCBkaXZlcnNpdHkgb2YgY29kb24gdXNhZ2UgcGF0dGVybnMgb2YgaHVtYW4gZ2VuZXMgd2l0aCByZXNwZWN0IHRvIGNocm9tb3NvbWUgYmFuZGluZyBwYXR0ZXJucyBhbmQgY2hyb21vc29tZSBudW1iZXJzOyByZWxhdGlvbiBiZXR3ZWVuIG51Y2xlb3RpZGUgc2VxdWVuY2UgZGF0YSBhbmQgY3l0b2dlbmV0aWMgZGF0YTsgTnVjbC4gQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5MTsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTQzMzMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUVsc2V2aWVyLCBOZXcgWW9yazsgU3RydWN0dXJhbCBhbmQgRXZvbHV0aW9uYXJ5IEdlbm9taWNzOiBOYXR1cmFsIFNlbGVjdGlvbiBpbiBHZW5vbWUgRXZvbHV0aW9uOyAgY2l0YXRpb25feWVhcj0yMDA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DeXRvc2luZSBtZXRoeWxhdGlvbiBhbmQgbWFtbWFsaWFuIGRldmVsb3BtZW50OyBHZW5lcyBEZXYuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MjYtMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUN5dG9zaW5lIG1ldGh5bGF0aW9uIGFuZCB0aGUgZWNvbG9neSBvZiBpbnRyYWdlbm9taWMgcGFyYXNpdGVzOyBUcmVuZHMgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MzM1LTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgb2YgRHJvc29waGlsYSBtZWxhbm9nYXN0ZXIgY29udGFpbnMgNS1tZXRoeWxjeXRvc2luZTsgRU1CTyBKLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTY5MTgtMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUROQSBtZXRoeWxhdGlvbiBpbiBEcm9zb3BoaWxhIG1lbGFub2dhc3RlcjsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTQwODsgIGNpdGF0aW9uX3BhZ2VzPTUzOC00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgYSBnZW5lIHJlcXVpcmVkIGZvciBkZSBub3ZvIEROQSBtZXRoeWxhdGlvbiBvZiB0aGUgemVicmFmaXNoIG5vIHRhaWwgZ2VuZTsgRGV2LiBEeW4uOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIzMzsgIGNpdGF0aW9uX3BhZ2VzPTE1MDktMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5vdmVsIHNwbGljZSB2YXJpYW50cyBhc3NvY2lhdGVkIHdpdGggb25lIG9mIHRoZSB6ZWJyYWZpc2ggZG5tdDMgZ2VuZXM7IEJNQyBEZXYuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0yMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cHJlc3Npb24gb2YgdGhlIGRubXQzIGdlbmVzIGluIHplYnJhZmlzaCBkZXZlbG9wbWVudDogc2ltaWxhcml0eSB0byBEbm10M2EgYW5kIERubXQzYjsgRGV2LiBHZW5lcy4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MjIwOyAgY2l0YXRpb25fcGFnZXM9MzQ3LTUzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NRUdBNjogbW9sZWN1bGFyIGV2b2x1dGlvbmFyeSBnZW5ldGljcyBhbmFseXNpcyB2ZXJzaW9uIDYuMDsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTI3MjUtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Uk5BLWRpcmVjdGVkIHRyYW5zY3JpcHRpb25hbCBnZW5lIHNpbGVuY2luZyBpbiBtYW1tYWxzOyBUcmVuZHMgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MzcwLTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5vdGFibGUgY2x1c3RlcmluZyBvZiB0cmFuc2NyaXB0aW9uLWZhY3Rvci1iaW5kaW5nIG1vdGlmcyBpbiBodW1hbiBwZXJpY2VudHJpYyByZWdpb25zIGFuZCBpdHMgYmlvbG9naWNhbCBzaWduaWZpY2FuY2U7IENocm9tb3NvbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTQ2MS03NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VxdWVuY2VzIGZyb20gYWxtb3N0IGFsbCBwcm9rYXJ5b3RpYywgZXVrYXJ5b3RpYywgYW5kIHZpcmFsIGdlbm9tZXMgYXZhaWxhYmxlIGNvdWxkIGJlIGNsYXNzaWZpZWQgYWNjb3JkaW5nIHRvIGdlbm9tZXMgb24gYSBsYXJnZS1zY2FsZSBzZWxmLW9yZ2FuaXppbmcgbWFwIGNvbnN0cnVjdGVkIHdpdGggdGhlIGVhcnRoIHNpbXVsYXRvcjsgSiBFYXJ0aCBTaW11bGF0b3I7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTE3LTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjEvNS80NTkvMTEzODA0My9kc3UwMTIucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFdpdGggYSByZW1hcmthYmxlIGluY3JlYXNlIGluIGdlbm9taWMgc2VxdWVuY2UgZGF0YSBvZiBhIHdpZGUgcmFuZ2Ugb2Ygc3BlY2llcywgbm92ZWwgdG9vbHMgYXJlIG5lZWRlZCBmb3IgY29tcHJlaGVuc2l2ZSBhbmFseXNlcyBvZiB0aGUgYmlnIHNlcXVlbmNlIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj00MDgyNzYmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2Rk3ZcMhdS+awXmp3j4n1okq6dHKcs=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f5%2f459%2f408276" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f5%2f459%2f408276" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=15969" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/21/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/5/1/m_cover.gif?Expires=2147483647&amp;Signature=gPiJSb93IIOcIMTSkMAi7kcKhziIjvuRKEzK8XVp8CTVu4eB811a4N7c5iRqAkFTvycBeFOL1qu9DtcvWUQE-HFcU90uhCx7UXftpGVaIU0r-58rykRAQjr7OyCGXhRRXufrlhLC-AFU8QeGWWE7goVK9s00kVi5KaqA8BdZOl4ZUh6UGigqEAXS9fKMJcyv7L69eSvyhk8fr0u2lpE6Z~kKXJ-zph2mYgOTE6TZsnpM7S63PaSYvqdIQyt~fHxa7pkm4binASfSKcJvbv9WOoMwYew88jtLZv495c~AkAPvlFj827Es8r960R62hvuI-oun7Y-AfhyLYD5usT~QKA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/21/5">
                        <div class="volume trailing-comma">Volume 21</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2014        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6045295">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6045295">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6045297">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6045297">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6045300">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6045300">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6045305">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6045305">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6045332">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6045332">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6045336">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6045336">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6045338">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6045338">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6045340">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6045340">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/21/5/469/2754536">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Evolutionary Changes in Vertebrate Genome Signatures with Special Focus on Coelacanth
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuki Iwasaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Bioscience</div>, <div class="insititution">Nagahama Institute of Bio-Science and Technology</div>, <div class="addr-line">Nagahama, Shiga 526-0829</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuki+Iwasaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Iwasaki Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Iwasaki Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yuki Iwasaki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takashi Abe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Information Engineering, Faculty of Engineering</div>, <div class="insititution">Institute of Science and Technology</div>, <div class="insititution">Niigata University</div>, <div class="addr-line">Niigata-ken 950-2181</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 25-262-6745. Fax. +81 25-262-6745. Email: <a href="mailto:takaabe@ie.niigata-u.ac.jp" target="_blank">takaabe@ie.niigata-u.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takashi+Abe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Abe T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Abe T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takashi Abe</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Norihiro Okada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Bioscience</div>, <div class="insititution">Nagahama Institute of Bio-Science and Technology</div>, <div class="addr-line">Nagahama, Shiga 526-0829</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Faculty of Bioscience and Biotechnology</div>, <div class="insititution">Tokyo Institute of Technology</div>, <div class="addr-line">Yokohama, Kanagawa 226</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">4</div><div class="addr-line">Department of Life Sciences</div>, <div class="insititution">National Cheng Kung University</div>, <div class="addr-line">Tainan 701</div>, <div class="country">Taiwan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Norihiro+Okada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Okada N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Okada N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Norihiro Okada</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kennosuke Wada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Bioscience</div>, <div class="insititution">Nagahama Institute of Bio-Science and Technology</div>, <div class="addr-line">Nagahama, Shiga 526-0829</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kennosuke+Wada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wada K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wada K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kennosuke Wada</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshiko Wada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Bioscience</div>, <div class="insititution">Nagahama Institute of Bio-Science and Technology</div>, <div class="addr-line">Nagahama, Shiga 526-0829</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshiko+Wada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wada Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wada Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yoshiko Wada</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Toshimichi Ikemura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Bioscience</div>, <div class="insititution">Nagahama Institute of Bio-Science and Technology</div>, <div class="addr-line">Nagahama, Shiga 526-0829</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Toshimichi+Ikemura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ikemura T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ikemura T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Toshimichi Ikemura</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="an1"><div class="fn"><div class="footnote-content">Edited by Prof. Kenta Nakai</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 21, Issue 5, 1 October 2014, Pages 459467, <a href='https://doi.org/10.1093/dnares/dsu012'>https://doi.org/10.1093/dnares/dsu012</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">06 May 2014</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">26 November 2013</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">31 March 2014</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="408276" href="/dnaresearch/article-pdf/21/5/459/1138043/dsu012.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Yuki Iwasaki, Takashi Abe, Norihiro Okada, Kennosuke Wada, Yoshiko Wada, Toshimichi Ikemura;  Evolutionary Changes in Vertebrate Genome Signatures with Special Focus on Coelacanth, <em>DNA Research</em>, Volume 21, Issue 5, 1 October 2014, Pages 459467, <a href='https://doi.org/10.1093/dnares/dsu012'>https://doi.org/10.1093/dnares/dsu012</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/408276?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/408276?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/408276?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/408276?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/408276?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/408276?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/408276?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Evolutionary%20Changes%20in%20Vertebrate%20Genome%20Signatures%20with%20Special%20Focus%20on%20Coelacanth&amp;publicationDate=2014-05-06&amp;volumeNum=21&amp;issueNum=5&amp;author=Iwasaki%2C%20Yuki%3B%20Abe%2C%20Takashi&amp;startPage=459&amp;endPage=467&amp;contentId=10.1093%2Fdnares%2Fdsu012&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f5%2f459%2f408276" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6045295 id="6045295" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>With a remarkable increase in genomic sequence data of a wide range of species, novel tools are needed for comprehensive analyses of the big sequence data. Self-organizing map (SOM) is a powerful tool for clustering high-dimensional data on one plane. For oligonucleotide compositions handled as high-dimensional data, we have previously modified the conventional SOM for genome informatics: BLSOM. In the present study, we constructed BLSOMs for oligonucleotide compositions in fragment sequences (e.g. 100 kb) from a wide range of vertebrates, including coelacanth, and found that the sequences were clustered primarily according to species without species information. As one of the nearest living relatives of tetrapod ancestors, coelacanth is believed to provide access to the phenotypic and genomic transitions leading to the emergence of tetrapods. The characteristic oligonucleotide composition found for coelacanth was connected with the lowest dinucleotide CG occurrence (i.e. the highest CG suppression) among fishes, which was rather equivalent to that of tetrapods. This evident CG suppression in coelacanth should reflect molecular evolutionary processes of epigenetic systems including DNA methylation during vertebrate evolution. Sequence of a <em>de novo</em> DNA methylase (Dntm3a) of coelacanth was found to be more closely related to that of tetrapods than that of other fishes.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;big data&quot;">big data</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="epigenetic">epigenetic</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="SOM">SOM</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;DNA methylation&quot;">DNA methylation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;CG suppression&quot;">CG suppression</a></div>                            <h2 scrollto-destination=6045297 id="6045297" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Oligonucleotide composition varies significantly among species even with the same genome G+C% and designated as a genome signature, which characterizes the genomic sequences of individual species.<sup><a reveal-id="DSU012C1  DSU012C3" data-open="DSU012C1  DSU012C3" class="link link-ref link-reveal xref-bibr">13</a></sup> Kohonen's self-organizing map (SOM) is a powerful tool for clustering high-dimensional complex data on one map.<sup><a reveal-id="DSU012C4" data-open="DSU012C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSU012C5" data-open="DSU012C5" class="link link-ref link-reveal xref-bibr">5</a></sup> For codon and oligonucleotide compositions handled as high-dimensional data, we have modified the conventional SOM to BLSOM.<sup><a reveal-id="DSU012C6" data-open="DSU012C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSU012C7" data-open="DSU012C7" class="link link-ref link-reveal xref-bibr">7</a></sup> BLSOMs for oligonucleotide compositions can classify genomic fragments (e.g. 100 kb) from a wide range of species according to species, without species information during the BLSOM calculation.<sup><a reveal-id="DSU012C8" data-open="DSU012C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSU012C9" data-open="DSU012C9" class="link link-ref link-reveal xref-bibr">9</a></sup> BLSOM can also visualize diagnostic oligonucleotides responsible for the species-specific clustering, supporting an efficient knowledge discovery from a big sequence data.</p><p>The present paper studies vertebrate genome signatures and their evolutionary changes with special focus on coelacanth. Coelacanth is one of the oldest known living <em>Sarcopterygii</em> and of the nearest living relatives of tetrapods.<sup><a reveal-id="DSU012C10" data-open="DSU012C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Therefore, analysis of this genome is believed to provide information concerning molecular evolutionary processes during the transition from fishes to tetrapods, and the genome of African coelacanth (<em>Latimeria chalumnae</em>) has recently been sequenced.<sup><a reveal-id="DSU012C11" data-open="DSU012C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSU012C12" data-open="DSU012C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Oligonucleotide BLSOM for genome sequences from a wide range of vertebrates, including the coelacanth, shows that CG suppression (i.e. under-representation of CG dinucleotide) is the highest among fishes and equivalent to that of tetrapods. The evident CG suppression in the mammalian genomes has been connected with the methylation at C in CG dinucleotide, which is a well-characterized epigenetic maker. Therefore, phylogenetic tree analyses on DNA methylases of fishes and tetrapods have been conducted. The classification and visualization power of the oligonucleotide BLSOM is very high and can efficiently unveil a wide range of genome characteristics from massive amounts of genomic sequences without prior knowledge, and therefore, this unsupervised data-mining method is a powerful and timely tool useful for genome studies in the era of big sequence data obtained from high-throughput DNA sequencers.</p>                            <h2 scrollto-destination=6045300 id="6045300" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=6045301 id="6045301" class="section-title" data-legacy-id=s2a>2.1. BLSOM</h3>
<p>SOM is an unsupervised clustering algorithm that non-linearly maps high-dimensional vectorial data onto a two-dimensional array of lattice points.<sup><a reveal-id="DSU012C4" data-open="DSU012C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSU012C5" data-open="DSU012C5" class="link link-ref link-reveal xref-bibr">5</a></sup> We previously modified the conventional SOM for genome informatics to make the learning process and resulting map independent of the order of data input, on the basis of batch-learning SOM (BLSOM).<sup><a reveal-id="DSU012C6" data-open="DSU012C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSU012C7" data-open="DSU012C7" class="link link-ref link-reveal xref-bibr">7</a></sup> The initial weight vectors were defined by principal component analysis instead of random values, and BLSOM learning for oligonucleotide composition was conducted as described previously.<sup><a reveal-id="DSU012C8" data-open="DSU012C8" class="link link-ref link-reveal xref-bibr">8</a></sup> The map size was chosen assuming that the average number of data per lattice point was 10. Diagnostic oligonucleotides for species-dependent separation were visualized as described previously.<sup><a reveal-id="DSU012C8" data-open="DSU012C8" class="link link-ref link-reveal xref-bibr">8</a></sup> The BLSOM programme was obtained from <a href="mailto:takaabe@ie.niigata-u.ac.jp" target="_blank">takaabe@ie.niigata-u.ac.jp</a>.</p>                            <h3 scrollto-destination=6045303 id="6045303" class="section-title" data-legacy-id=s2b>2.2. Sequences</h3>
<p>Coelacanth genome sequences were obtained from <a class="link link-uri" href="http://coelacanth.nig.ac.jp">http://coelacanth.nig.ac.jp</a>, and other genome sequences were obtained from the UCSC ftp site (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/genomes/">http://www.ncbi.nlm.nih.gov/genomes/</a>). When the number of undetermined nucleotides (Ns) in a fragment sequence (e.g. 100 kb) exceeded 20% of the sequence, the sequence was omitted from the analysis. In the case where the number of Ns was &lt;20%, the oligonucleotide frequencies were normalized to the length without Ns and included in the analysis. Amino acid sequences of coelacanth Dnmt3a, Dnmt3b and Dnmt1 (GENE ID: R03136, A01856 and A13991) were obtained from the Coelacanth Genome Project (<a class="link link-uri" href="http://coelacanth.nig.ac.jp/">http://coelacanth.nig.ac.jp/</a>).</p>                            <h2 scrollto-destination=6045305 id="6045305" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=6045306 id="6045306" class="section-title" data-legacy-id=s3a>3.1. BLSOM for six fish genomes</h3>
<p>To compare the coelacanth's genome signature with that of other fishes and to study molecular evolutionary mechanisms for the establishment of their signatures, we constructed BLSOMs with di-, tri- and tetranucleotide compositions in all 100-kb sequence fragments from six fish genomes, including the African coelacanth genome. The fishes other than coelacanth were chosen, since a major portion of their genomes was sequenced and registered in the International DNA Sequence Databanks (DDBJ/EMBL/GenBank) as complete or draft genome sequences. In the Databanks, only one strand of complementary sequences is registered, and the strand is chosen rather arbitrarily in the registration of fragment sequences (e.g. draft genome sequences). Our previous BLSOM analysis of a wide range of species<sup><a reveal-id="DSU012C7" data-open="DSU012C7" class="link link-ref link-reveal xref-bibr">7</a></sup> revealed that sequences from a single genome often gave a mirror-symmetrical split on BLSOM in the vertical direction, e.g. according to the replicational direction of genomic fragments.<sup><a reveal-id="DSU012C13" data-open="DSU012C13" class="link link-ref link-reveal xref-bibr">13</a></sup> When investigating the general characteristics of genome sequences such as genome signatures, differences between two complementary strands are not important. Furthermore, the obtained map should not be affected by the choice of strands registered to the Databanks. Therefore, we constructed a BLSOM, for which the frequencies of a pair of complementary oligonucleotides in each fragment were summed up (e.g. AAC and GTT; abbreviated as AAC+GTT).<sup><a reveal-id="DSU012C8" data-open="DSU012C8" class="link link-ref link-reveal xref-bibr">8</a></sup> The BLSOM for this degenerate set of a pair of complementary di-, tri- and tetranucleotides is abbreviated as DegDi, DegTri and DegTetra. The DegTri is listed in Fig.<a reveal-id="DSU012F1" data-open="DSU012F1" class="link link-reveal link-table xref-fig">1</a>A; other two are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1A and B</a>. Lattice points containing sequences from a single species are coloured to indicate the species, whereas those containing sequences from multiple species are indicated in black. With no information concerning species during BLSOM calculation, species-specific clustering (self-organization) of the fragment sequences is clear; i.e. sequences from individual species have formed their own territories. The number of black lattice points on DegTri and DegTetra is smaller than on DegDi, showing their higher clustering power; for details of their differences (Fig.<a reveal-id="DSU012F1" data-open="DSU012F1" class="link link-reveal link-table xref-fig">1</a>A). One of the basal characteristics of BLSOM separation for genomic sequence has been attributed previously to the G+C%.<sup><a reveal-id="DSU012C7" data-open="DSU012C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSU012C9" data-open="DSU012C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSU012C14" data-open="DSU012C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Sequences with higher or lower G+C% are located on the left or right side of the map (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figs S1Aii and S4Aii</a>), showing that the G+C% is reflected primarily in the horizontal direction. </p>                        <a id="6045308" scrollto-destination="6045308"></a>
<div id="DSU012F1" data-id="DSU012F1" class="fig fig-section"><div class="label fig-label">Figure1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01201.png?Expires=2147483647&amp;Signature=iY7jPpjU~NN2FH59fzOwlaJ1FH0ahEDVFLfqd~m2gkcpvREnuW4PUw7AxkVXrpPdNipqT6ond5Kxv-dpIfzGKYDFnU8EjdlOn8H3mg1FeObCI4ka7KvDldWCE6GnU~AylOHVrdmop99nUz4-q1iUiuFnyziV9j3D6cvgzxoYWij3QlDa9NFbZcIzkp87D7L9crsByT9-Vz-3220grGqaFjD1TZTSned7SaEi8hSVEWYplO9IV0JlhRLqgBsShwAiX5GBsVwB2LilIFnu9e-aaBh3Aoo7RAC2MmpSAe0SbXwpA0D8t~dqZDKUYeZp8NET6jXe50ZO6rGzd11Gd7gplw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Oligonucleotide BLSOM for 100-kb sequences. (Ai) DegTri for six fish genomes. Lattice points containing sequences from multiple species are indicated in black and those containing sequences from a single species are coloured as follows: coelacanth (), medaka (), stickleback (Gasterosteus aculeatus; ), fugu (), tetraodon () and zebrafish (). Lattice points containing no sequences after BLSOM calculation are indicated in white blank. 89, 95 and 97% of the sequences are located at the lattice points containing sequences from a single species (designated as pure lattice points) on DegDi, DegTri and DegTetra (see also Supplementary Fig. S1Ai and Bi). The separation of two closely related species, fugu and tetraodon, is rather poor especially on DegDi; therefore, black lattice points are evident for their territories (green for fugu and dark green for tetraodon). When these species belonging to Tetraodontidae are grouped into one category, 92, 97 and 98% of all fish sequences are located in the pure lattice points on DegDi, DegTri and DegTetra. BLSOM for 50-kb sequences also gave a clear species-specific separation (data not shown). (Aii) Examples of trinucleotides over-represented and evidently under-represented in the coelacanth genome. The occurrence of the trinucleotide for each lattice point has been calculated and normalized with the occurrence expected from the mononucleotide composition for each lattice point.8 This observed/expected ratio is indicated in colour presented at the bottom of the panel. (Bi) DegTri for 11 vertebrate genomes. Lattice points containing sequences from multiple species are indicated in black and those containing sequences from a single species are coloured. Fishes are marked as described in Ai, and tetrapods are as follows: chicken (), human (Homo sapiens; ), mouse (Mus musculus; ), lizard (Anolis carolinensis; ) and X. tropicalis (). About 89, 97 and 98% of the sequences are located at the lattice points containing sequences from a single species (pure lattice points) on DegDi, DegTri and DegTetra, respectively (see also Fig.2Ai and Supplementary Fig. S1Ci). When fugu and tetraodon are grouped into one category, 90, 98 and 99% of all fish sequences are located in the pure lattice points. (Bii) Lattice points containing sequences derived only from fish genomes are marked with the colours listed in Ai. (Biii) Examples of trinucleotides whose occurrence differs between coelacanth and other fishes. Lattice points are marked as described in Aii." path-from-xml="dsu01201.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6045308/dsu01201.tiff" path-from-xml="dsu01201.tiff" target="_blank">View large</a><a section="6045308" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/dsu01201.png?Expires=2147483647&amp;Signature=MdYQ57RoXi7ozKWHuTxm0YDrrwy6bjHeaRQHdUVz3cUDAbRrHAW6nKmVKh9yhGWlH2FjvbIC7nM-ynXzB3GhVXs0m206BPxCg9~TATUesZLThhaqNzE-5ZkGg36d7grhVGLS6PWh8iXr3t8jgwp1q39SryED0RQMSGK3gcpgZDp03ZZZo3kdlTA~Hz-lueDwmMMIQHK1KQuwyopmlV7J~EKaYVu5LDGJfDvlayd1s9a-Hn3IsydqUSIArwbXlioia~nD7eyg0k4S5CGKd2r7fAW4wx9mhOIvqPDxGISwH0IboRl9fCdZDNlU-thw6KJURoPCrIjSa1399QYLInUjuA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=6045308&amp;ar=408276&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu01201.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Oligonucleotide BLSOM for 100-kb sequences. (Ai) DegTri for six fish genomes. Lattice points containing sequences from multiple species are indicated in black and those containing sequences from a single species are coloured as follows: coelacanth (<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01207.gif?Expires=2147483647&amp;Signature=DoJ-y2MuthXmhRp8Bu0hhNS6JfgoCY-qB88ZOMO0hRhg2jNGKRUsNz1oWvV3YWQI1HU44vwBorAe~cDSpbeSvdvLSk9VVPlKx6PrmMGoAVkTNnM0-OYbsRACPSehrLrp~1cEFhzK-9OKFZYTumJ5tpaZhbdYP3r7zIaE5ylSpJCBbtvULzz-tN-8QFBCaTycqh6Pg~Yq~RpnGuFaIILu8rMTbGbwAstZvkMcCH-Jr93EHsO09~7WKi18O4sD6n9y3v18fIyOCEl5P6qY2LO3S7MwcuD~n-X6EtnxaoOj2QniVclJXk133YPGDuuzPzSxtHHQTuw4SP7M1s4ySabVFg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsu01207.gif" />), medaka (<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01208.gif?Expires=2147483647&amp;Signature=IB-NA~vZknTA419n28vlwRDelMIMhXO9jn2kFP~~Qaz2~GrEaGAjz0CkgZtIXXk6qMpbhJMoSdpyek-wq6wy5pPcxnk2lkLr-YBDgM~WQUX2~n5JUHJKuzObiMCTt89l3LQl6FDk~7O6sT6y-EJ5xJEetMai0~Fly-Qvve7X9WnTn93iW4w4B4QVuAR5bhgi8bpGP1zxQAV-lUfNVzcqNqqBlP8SZgDA3yrmci02sI3hLdKPmUqlB3cBlfstQc3R~WuTYRKg8SSELzHqFyWQvYQqcpoRzIBybae4A6TEjEcHToGpdsLBAc2hbSEthmyQvwYgRhVFFpr1uOpKcU07rw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsu01208.gif" />), stickleback (<em>Gasterosteus aculeatus</em>; <img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01209.gif?Expires=2147483647&amp;Signature=BclUqjvkFtfpLBK1G99xRKjt4iVsXPkP370XzLNqZkgGXMNoFrgq61-sFsK325cInTBY~KCGCldbl4RvAkT1BP1mI0aeq8ZQgAZ7WGa~0KkkoHLuTNPao6o5GIY1vEW4J2SoItLYQreEvkxTGQ4gykMNQXdzcAPQcotNwxOqIwwDys8bDk4p8eItmM~tbwI9QVnfrZ~qObaZmmjCXn6v0nftBnMajdAEEIcQViaAi9jXCwVPYBiojt3pDL~eEbtbnMPrKGgbqf2GLdIBDj3CHNLbw~2nLmglFt0dNmxIrk9gxSqpxM-QniZ2WrvwFDAzRXwulew0~F6vuOYf4MXvQQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsu01209.gif" />), fugu (<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01210.gif?Expires=2147483647&amp;Signature=LDT9AvOKUTWpnpCzQAN5EU2g2-2N-0qOJAciOJk8qB6V98HVOQFGZpVqHfey7jU72DuMxyLxnK1mVO~bC5FAaOdb8COIRQWhFreT9QHWlL4ExibYtkFc6t~stH5BkuhvsCFZEMqgOZCtJAjylp2-STz6jU4L7ASK~pLMa7x7MFmQw2jGs13HOohd5I3wQumZGLNOcanhNsP2BWvp9Nv6ityCzCLMEuL7qScOCSGOUoWBBpY18IhntVh2tRo2cHi31jyVl7~V~N4bpsKJ7Uk3A-ES1nQL9rhWM2VulZHO7mZbQNx3d~7aZJjFLaOY14gWHPDsYxlAEZPLbbhVWtHRWw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsu01210.gif" />), tetraodon (<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01211.gif?Expires=2147483647&amp;Signature=ic3v5nOVYlK6YqC~v1Rh1aEvDJmF2s6kXZg1gU88f7N8Bje9pnQg-weSqtxq~07LPYirS-wHob6jgIuQKyjZe3J571azQeyrYS7FuRVva2tk3TzkOm1JrxQGeVlHyM8Jo0isPJTs2h8VkKEBaOIAwFU47D2UnPq5iDKWnnPp2gEgscbmWmz1Br2u8R7Z3A8CMBITmn~BSu9l8wosLmJAx0lgA780EkXEW7vVkxJly2b4lUubT1JK-5f-d11MSt8IM7-tuwwj9AqtTUAxYKH3sPPV7c2WY~ZsSoaRAq-hMdzUnYG355OZ0UV9N-rnL8sM0WS4P2bT34hqh3BSLPBEKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsu01211.gif" />) and zebrafish (<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01212.gif?Expires=2147483647&amp;Signature=DPLvv0iRqS-QkW3GM98ofaXZdlB3IDlJH7f-E4mKW5NRG0cjUECSxuCHPhL2DklSbVcLRIGAAmHz04~pEolHAq4682VFmi2-w5AKAtnzwdUJvu6Blzq9dPs5yCp9skxzpiMPAJGOhuBI~9JD9S3SaOnBGXaJgc8-tLJuzW0X1wM3qRBVHCGctQl4-LM2TBny10tP9Y~bnXBZxLUy4lj-gllRMvhajjp2VRKv~0kiYd0FNZ2CfmOB1iTOjKnE8Om2UT3gA2Wx~X6SqY05I-QN1tKF-eGspU8TU8VC-heTrYb-zIrDd3bIadc-H5hBMTM6VDPqGOUCLxBQczks6m0lEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsu01212.gif" />). Lattice points containing no sequences after BLSOM calculation are indicated in white blank. 89, 95 and 97% of the sequences are located at the lattice points containing sequences from a single species (designated as pure lattice points) on DegDi, DegTri and DegTetra (see also <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1Ai and Bi</a>). The separation of two closely related species, fugu and tetraodon, is rather poor especially on DegDi; therefore, black lattice points are evident for their territories (green for fugu and dark green for tetraodon). When these species belonging to Tetraodontidae are grouped into one category, 92, 97 and 98% of all fish sequences are located in the pure lattice points on DegDi, DegTri and DegTetra. BLSOM for 50-kb sequences also gave a clear species-specific separation (data not shown). (Aii) Examples of trinucleotides over-represented and evidently under-represented in the coelacanth genome. The occurrence of the trinucleotide for each lattice point has been calculated and normalized with the occurrence expected from the mononucleotide composition for each lattice point.<sup><a reveal-id="DSU012C8" data-open="DSU012C8" class="link link-ref link-reveal xref-bibr">8</a></sup> This observed/expected ratio is indicated in colour presented at the bottom of the panel. (Bi) DegTri for 11 vertebrate genomes. Lattice points containing sequences from multiple species are indicated in black and those containing sequences from a single species are coloured. Fishes are marked as described in Ai, and tetrapods are as follows: chicken (<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01213.gif?Expires=2147483647&amp;Signature=pX9gYaMH9O9eV~-BBaTSFhnlyV9c1bLWzCBvsPjfW-xDnrhviFM031KuKa2alRL86tUKCr~8Et5JAH-DpKqpisyOxW01nO~aKqgqEn30IvtIvoF8CCyQ2D6ECLk1egRMTpoghmrZfxzUWHCnmtEDYmR9ePuOaVI4Tr1dVzLH5BiHW5ZJyqevazOurEWFG5vwnqC3~Sui7d9iJ7nMmrZjfS3zZcaXX5jvj46ksJGtQorPAFh9P0PTy8ET6~GXET90P9ionPwBsXOzJskQp48DT-z1Wz7tKY~TNavKuiQTgddkYCx-sIsfajHPbaLpjI8Q3AeTqbvveOicg7PEYuq2xQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsu01213.gif" />), human (<em>Homo sapiens</em>; <img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01214.gif?Expires=2147483647&amp;Signature=2GuoC9EYhlgkWH0dKxSkd13uREWZh7ETTHAXdHiFpjunWCslEOaSfUm~k7jqX5qoumlrjIhcq4XJ-TC9fkml2SuAir60xyHZ1kBSOIye8mW9F~KKz5rZJ7dxgYjJsEa6GGWCF0ruiMBTdQ2Hr4ZfmwKtHCw5e5i82Jq0PhMPewND9QL0D9MFsrpT13LSAqFaFlAIH9gRIiWwTv3JcynAPcmD92HQGVuxjc1rzZ4Jmz1sZYwzm9LmA8c-GfDL3blLh4pxlXjCf2snBYa3oqKvMGg9NtzsVTw4XU1SxV~tgv1jMluyn5-kWS6kDlwS9EzERLetuO6vuOgFVW-s7hHa0w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsu01214.gif" />), mouse (<em>Mus musculus</em>; <img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01215.gif?Expires=2147483647&amp;Signature=OIQP98mwjGIn5zgAOAedQHxzDDuY6oovHXsqyyD39r70oOSPQ0nVRpT829HApdn-arEA2HOX3ciRedigNgu5tXCusT-gF71XcEcRZeWFdfajQajQW6KYcwDWsN-bxU2bXm6haBhXiQJxV2yZ0FqopHqYb0ipUTM13nN8WZ5nHbprZCnkD7~7k2UWF0haqwysKqMVsicc5yBLi-VZ68NJnCnT4oK9XAMgjo3HmD0v8AH0TaKQ4UZvarkkSoJ3g9aKjYQTW5X1HSS6z9pWFUp3B230R-NxQkDz-ZaMR6Mm4JbY-ZL4jCqyIRv0nyTVa-SQ7sDiuZarUitA1tz5Zz~nHQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsu01215.gif" />), lizard (<em>Anolis carolinensis</em>; <img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01216.gif?Expires=2147483647&amp;Signature=hDbbeZj-lvHSQTjIPo7NEkXGQ4L1WHLke-csE0-NeWJnqggfLf2wCoTyVIHBjfz8K4X0dAsjoGbyXCqi0owlvaROKxRYipfnK-H4AHJKBP9uC1WN8U7BjrV567QQgSN5iGDoVgMVa-kA8pPQqoF8CDNGVspheGoSz5rjBWOIQCnNQ9YFUFXglZnqwrk4MFvBb-jZ28Sdq0Ow0Qc2qjObL4Dm1MJmJw9xhe9PkkCxVWGrhm3U3MfFxguZksNi4~F4kF1OFHfPoESpAm7gYhnAl-~SC0d6Ez7ofKgo~RtK~BVv-2-Q64ZiccFcM0GJOZeUR9LdYLzJso2KDE3hQsEa~w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsu01216.gif" />) and <em>X. tropicalis</em> (<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01217.gif?Expires=2147483647&amp;Signature=x17f4~q-S2b6mEFGxNGxo~MtOs2r0-lLj9a2oDci1e1uvEgP~zQ5gVlp-jEs4v0UT~Qxxuxh0KCIcn6454iNOBFA3Liza4UmGqcErGv5BjROc7iiLQVOQB8l0tX3UXL2fvqEnIXtx4sduWyFL3t9nfhJE0RTcdmP7WTucdypuN4Ii-5NvysM~oSdJGz0Pv4RzWB30L0wm4IoKvxgkZbtta1BB62lxrc3rnC6iZmIny2c0~j4uKxGoCOF7BREAl-zvNpAyhlMDqUSYmMxhjpOy7kAT1oa1B7hMMfgKtKG5hHJcG-7vpqj3RzG8MBGIJq0RGdsK8K7sNWM1BAhj0MCbQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsu01217.gif" />). About 89, 97 and 98% of the sequences are located at the lattice points containing sequences from a single species (pure lattice points) on DegDi, DegTri and DegTetra, respectively (see also Fig.<a reveal-id="DSU012F2" data-open="DSU012F2" class="link link-reveal link-table xref-fig">2</a>Ai and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1Ci</a>). When fugu and tetraodon are grouped into one category, 90, 98 and 99% of all fish sequences are located in the pure lattice points. (Bii) Lattice points containing sequences derived only from fish genomes are marked with the colours listed in Ai. (Biii) Examples of trinucleotides whose occurrence differs between coelacanth and other fishes. Lattice points are marked as described in Aii.</p></div></div><div content-id="DSU012F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01201.png?Expires=2147483647&amp;Signature=iY7jPpjU~NN2FH59fzOwlaJ1FH0ahEDVFLfqd~m2gkcpvREnuW4PUw7AxkVXrpPdNipqT6ond5Kxv-dpIfzGKYDFnU8EjdlOn8H3mg1FeObCI4ka7KvDldWCE6GnU~AylOHVrdmop99nUz4-q1iUiuFnyziV9j3D6cvgzxoYWij3QlDa9NFbZcIzkp87D7L9crsByT9-Vz-3220grGqaFjD1TZTSned7SaEi8hSVEWYplO9IV0JlhRLqgBsShwAiX5GBsVwB2LilIFnu9e-aaBh3Aoo7RAC2MmpSAe0SbXwpA0D8t~dqZDKUYeZp8NET6jXe50ZO6rGzd11Gd7gplw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Oligonucleotide BLSOM for 100-kb sequences. (Ai) DegTri for six fish genomes. Lattice points containing sequences from multiple species are indicated in black and those containing sequences from a single species are coloured as follows: coelacanth (), medaka (), stickleback (Gasterosteus aculeatus; ), fugu (), tetraodon () and zebrafish (). Lattice points containing no sequences after BLSOM calculation are indicated in white blank. 89, 95 and 97% of the sequences are located at the lattice points containing sequences from a single species (designated as pure lattice points) on DegDi, DegTri and DegTetra (see also Supplementary Fig. S1Ai and Bi). The separation of two closely related species, fugu and tetraodon, is rather poor especially on DegDi; therefore, black lattice points are evident for their territories (green for fugu and dark green for tetraodon). When these species belonging to Tetraodontidae are grouped into one category, 92, 97 and 98% of all fish sequences are located in the pure lattice points on DegDi, DegTri and DegTetra. BLSOM for 50-kb sequences also gave a clear species-specific separation (data not shown). (Aii) Examples of trinucleotides over-represented and evidently under-represented in the coelacanth genome. The occurrence of the trinucleotide for each lattice point has been calculated and normalized with the occurrence expected from the mononucleotide composition for each lattice point.8 This observed/expected ratio is indicated in colour presented at the bottom of the panel. (Bi) DegTri for 11 vertebrate genomes. Lattice points containing sequences from multiple species are indicated in black and those containing sequences from a single species are coloured. Fishes are marked as described in Ai, and tetrapods are as follows: chicken (), human (Homo sapiens; ), mouse (Mus musculus; ), lizard (Anolis carolinensis; ) and X. tropicalis (). About 89, 97 and 98% of the sequences are located at the lattice points containing sequences from a single species (pure lattice points) on DegDi, DegTri and DegTetra, respectively (see also Fig.2Ai and Supplementary Fig. S1Ci). When fugu and tetraodon are grouped into one category, 90, 98 and 99% of all fish sequences are located in the pure lattice points. (Bii) Lattice points containing sequences derived only from fish genomes are marked with the colours listed in Ai. (Biii) Examples of trinucleotides whose occurrence differs between coelacanth and other fishes. Lattice points are marked as described in Aii." path-from-xml="dsu01201.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6045308/dsu01201.tiff" path-from-xml="dsu01201.tiff" target="_blank">View large</a><a section="6045308" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/dsu01201.png?Expires=2147483647&amp;Signature=MdYQ57RoXi7ozKWHuTxm0YDrrwy6bjHeaRQHdUVz3cUDAbRrHAW6nKmVKh9yhGWlH2FjvbIC7nM-ynXzB3GhVXs0m206BPxCg9~TATUesZLThhaqNzE-5ZkGg36d7grhVGLS6PWh8iXr3t8jgwp1q39SryED0RQMSGK3gcpgZDp03ZZZo3kdlTA~Hz-lueDwmMMIQHK1KQuwyopmlV7J~EKaYVu5LDGJfDvlayd1s9a-Hn3IsydqUSIArwbXlioia~nD7eyg0k4S5CGKd2r7fAW4wx9mhOIvqPDxGISwH0IboRl9fCdZDNlU-thw6KJURoPCrIjSa1399QYLInUjuA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=6045308&amp;ar=408276&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu01201.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Oligonucleotide BLSOM for 100-kb sequences. (Ai) DegTri for six fish genomes. Lattice points containing sequences from multiple species are indicated in black and those containing sequences from a single species are coloured as follows: coelacanth (<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01207.gif?Expires=2147483647&amp;Signature=DoJ-y2MuthXmhRp8Bu0hhNS6JfgoCY-qB88ZOMO0hRhg2jNGKRUsNz1oWvV3YWQI1HU44vwBorAe~cDSpbeSvdvLSk9VVPlKx6PrmMGoAVkTNnM0-OYbsRACPSehrLrp~1cEFhzK-9OKFZYTumJ5tpaZhbdYP3r7zIaE5ylSpJCBbtvULzz-tN-8QFBCaTycqh6Pg~Yq~RpnGuFaIILu8rMTbGbwAstZvkMcCH-Jr93EHsO09~7WKi18O4sD6n9y3v18fIyOCEl5P6qY2LO3S7MwcuD~n-X6EtnxaoOj2QniVclJXk133YPGDuuzPzSxtHHQTuw4SP7M1s4ySabVFg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsu01207.gif" />), medaka (<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01208.gif?Expires=2147483647&amp;Signature=IB-NA~vZknTA419n28vlwRDelMIMhXO9jn2kFP~~Qaz2~GrEaGAjz0CkgZtIXXk6qMpbhJMoSdpyek-wq6wy5pPcxnk2lkLr-YBDgM~WQUX2~n5JUHJKuzObiMCTt89l3LQl6FDk~7O6sT6y-EJ5xJEetMai0~Fly-Qvve7X9WnTn93iW4w4B4QVuAR5bhgi8bpGP1zxQAV-lUfNVzcqNqqBlP8SZgDA3yrmci02sI3hLdKPmUqlB3cBlfstQc3R~WuTYRKg8SSELzHqFyWQvYQqcpoRzIBybae4A6TEjEcHToGpdsLBAc2hbSEthmyQvwYgRhVFFpr1uOpKcU07rw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsu01208.gif" />), stickleback (<em>Gasterosteus aculeatus</em>; <img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01209.gif?Expires=2147483647&amp;Signature=BclUqjvkFtfpLBK1G99xRKjt4iVsXPkP370XzLNqZkgGXMNoFrgq61-sFsK325cInTBY~KCGCldbl4RvAkT1BP1mI0aeq8ZQgAZ7WGa~0KkkoHLuTNPao6o5GIY1vEW4J2SoItLYQreEvkxTGQ4gykMNQXdzcAPQcotNwxOqIwwDys8bDk4p8eItmM~tbwI9QVnfrZ~qObaZmmjCXn6v0nftBnMajdAEEIcQViaAi9jXCwVPYBiojt3pDL~eEbtbnMPrKGgbqf2GLdIBDj3CHNLbw~2nLmglFt0dNmxIrk9gxSqpxM-QniZ2WrvwFDAzRXwulew0~F6vuOYf4MXvQQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsu01209.gif" />), fugu (<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01210.gif?Expires=2147483647&amp;Signature=LDT9AvOKUTWpnpCzQAN5EU2g2-2N-0qOJAciOJk8qB6V98HVOQFGZpVqHfey7jU72DuMxyLxnK1mVO~bC5FAaOdb8COIRQWhFreT9QHWlL4ExibYtkFc6t~stH5BkuhvsCFZEMqgOZCtJAjylp2-STz6jU4L7ASK~pLMa7x7MFmQw2jGs13HOohd5I3wQumZGLNOcanhNsP2BWvp9Nv6ityCzCLMEuL7qScOCSGOUoWBBpY18IhntVh2tRo2cHi31jyVl7~V~N4bpsKJ7Uk3A-ES1nQL9rhWM2VulZHO7mZbQNx3d~7aZJjFLaOY14gWHPDsYxlAEZPLbbhVWtHRWw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsu01210.gif" />), tetraodon (<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01211.gif?Expires=2147483647&amp;Signature=ic3v5nOVYlK6YqC~v1Rh1aEvDJmF2s6kXZg1gU88f7N8Bje9pnQg-weSqtxq~07LPYirS-wHob6jgIuQKyjZe3J571azQeyrYS7FuRVva2tk3TzkOm1JrxQGeVlHyM8Jo0isPJTs2h8VkKEBaOIAwFU47D2UnPq5iDKWnnPp2gEgscbmWmz1Br2u8R7Z3A8CMBITmn~BSu9l8wosLmJAx0lgA780EkXEW7vVkxJly2b4lUubT1JK-5f-d11MSt8IM7-tuwwj9AqtTUAxYKH3sPPV7c2WY~ZsSoaRAq-hMdzUnYG355OZ0UV9N-rnL8sM0WS4P2bT34hqh3BSLPBEKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsu01211.gif" />) and zebrafish (<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01212.gif?Expires=2147483647&amp;Signature=DPLvv0iRqS-QkW3GM98ofaXZdlB3IDlJH7f-E4mKW5NRG0cjUECSxuCHPhL2DklSbVcLRIGAAmHz04~pEolHAq4682VFmi2-w5AKAtnzwdUJvu6Blzq9dPs5yCp9skxzpiMPAJGOhuBI~9JD9S3SaOnBGXaJgc8-tLJuzW0X1wM3qRBVHCGctQl4-LM2TBny10tP9Y~bnXBZxLUy4lj-gllRMvhajjp2VRKv~0kiYd0FNZ2CfmOB1iTOjKnE8Om2UT3gA2Wx~X6SqY05I-QN1tKF-eGspU8TU8VC-heTrYb-zIrDd3bIadc-H5hBMTM6VDPqGOUCLxBQczks6m0lEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsu01212.gif" />). Lattice points containing no sequences after BLSOM calculation are indicated in white blank. 89, 95 and 97% of the sequences are located at the lattice points containing sequences from a single species (designated as pure lattice points) on DegDi, DegTri and DegTetra (see also <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1Ai and Bi</a>). The separation of two closely related species, fugu and tetraodon, is rather poor especially on DegDi; therefore, black lattice points are evident for their territories (green for fugu and dark green for tetraodon). When these species belonging to Tetraodontidae are grouped into one category, 92, 97 and 98% of all fish sequences are located in the pure lattice points on DegDi, DegTri and DegTetra. BLSOM for 50-kb sequences also gave a clear species-specific separation (data not shown). (Aii) Examples of trinucleotides over-represented and evidently under-represented in the coelacanth genome. The occurrence of the trinucleotide for each lattice point has been calculated and normalized with the occurrence expected from the mononucleotide composition for each lattice point.<sup><a reveal-id="DSU012C8" data-open="DSU012C8" class="link link-ref link-reveal xref-bibr">8</a></sup> This observed/expected ratio is indicated in colour presented at the bottom of the panel. (Bi) DegTri for 11 vertebrate genomes. Lattice points containing sequences from multiple species are indicated in black and those containing sequences from a single species are coloured. Fishes are marked as described in Ai, and tetrapods are as follows: chicken (<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01213.gif?Expires=2147483647&amp;Signature=pX9gYaMH9O9eV~-BBaTSFhnlyV9c1bLWzCBvsPjfW-xDnrhviFM031KuKa2alRL86tUKCr~8Et5JAH-DpKqpisyOxW01nO~aKqgqEn30IvtIvoF8CCyQ2D6ECLk1egRMTpoghmrZfxzUWHCnmtEDYmR9ePuOaVI4Tr1dVzLH5BiHW5ZJyqevazOurEWFG5vwnqC3~Sui7d9iJ7nMmrZjfS3zZcaXX5jvj46ksJGtQorPAFh9P0PTy8ET6~GXET90P9ionPwBsXOzJskQp48DT-z1Wz7tKY~TNavKuiQTgddkYCx-sIsfajHPbaLpjI8Q3AeTqbvveOicg7PEYuq2xQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsu01213.gif" />), human (<em>Homo sapiens</em>; <img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01214.gif?Expires=2147483647&amp;Signature=2GuoC9EYhlgkWH0dKxSkd13uREWZh7ETTHAXdHiFpjunWCslEOaSfUm~k7jqX5qoumlrjIhcq4XJ-TC9fkml2SuAir60xyHZ1kBSOIye8mW9F~KKz5rZJ7dxgYjJsEa6GGWCF0ruiMBTdQ2Hr4ZfmwKtHCw5e5i82Jq0PhMPewND9QL0D9MFsrpT13LSAqFaFlAIH9gRIiWwTv3JcynAPcmD92HQGVuxjc1rzZ4Jmz1sZYwzm9LmA8c-GfDL3blLh4pxlXjCf2snBYa3oqKvMGg9NtzsVTw4XU1SxV~tgv1jMluyn5-kWS6kDlwS9EzERLetuO6vuOgFVW-s7hHa0w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsu01214.gif" />), mouse (<em>Mus musculus</em>; <img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01215.gif?Expires=2147483647&amp;Signature=OIQP98mwjGIn5zgAOAedQHxzDDuY6oovHXsqyyD39r70oOSPQ0nVRpT829HApdn-arEA2HOX3ciRedigNgu5tXCusT-gF71XcEcRZeWFdfajQajQW6KYcwDWsN-bxU2bXm6haBhXiQJxV2yZ0FqopHqYb0ipUTM13nN8WZ5nHbprZCnkD7~7k2UWF0haqwysKqMVsicc5yBLi-VZ68NJnCnT4oK9XAMgjo3HmD0v8AH0TaKQ4UZvarkkSoJ3g9aKjYQTW5X1HSS6z9pWFUp3B230R-NxQkDz-ZaMR6Mm4JbY-ZL4jCqyIRv0nyTVa-SQ7sDiuZarUitA1tz5Zz~nHQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsu01215.gif" />), lizard (<em>Anolis carolinensis</em>; <img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01216.gif?Expires=2147483647&amp;Signature=hDbbeZj-lvHSQTjIPo7NEkXGQ4L1WHLke-csE0-NeWJnqggfLf2wCoTyVIHBjfz8K4X0dAsjoGbyXCqi0owlvaROKxRYipfnK-H4AHJKBP9uC1WN8U7BjrV567QQgSN5iGDoVgMVa-kA8pPQqoF8CDNGVspheGoSz5rjBWOIQCnNQ9YFUFXglZnqwrk4MFvBb-jZ28Sdq0Ow0Qc2qjObL4Dm1MJmJw9xhe9PkkCxVWGrhm3U3MfFxguZksNi4~F4kF1OFHfPoESpAm7gYhnAl-~SC0d6Ez7ofKgo~RtK~BVv-2-Q64ZiccFcM0GJOZeUR9LdYLzJso2KDE3hQsEa~w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsu01216.gif" />) and <em>X. tropicalis</em> (<img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01217.gif?Expires=2147483647&amp;Signature=x17f4~q-S2b6mEFGxNGxo~MtOs2r0-lLj9a2oDci1e1uvEgP~zQ5gVlp-jEs4v0UT~Qxxuxh0KCIcn6454iNOBFA3Liza4UmGqcErGv5BjROc7iiLQVOQB8l0tX3UXL2fvqEnIXtx4sduWyFL3t9nfhJE0RTcdmP7WTucdypuN4Ii-5NvysM~oSdJGz0Pv4RzWB30L0wm4IoKvxgkZbtta1BB62lxrc3rnC6iZmIny2c0~j4uKxGoCOF7BREAl-zvNpAyhlMDqUSYmMxhjpOy7kAT1oa1B7hMMfgKtKG5hHJcG-7vpqj3RzG8MBGIJq0RGdsK8K7sNWM1BAhj0MCbQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsu01217.gif" />). About 89, 97 and 98% of the sequences are located at the lattice points containing sequences from a single species (pure lattice points) on DegDi, DegTri and DegTetra, respectively (see also Fig.<a reveal-id="DSU012F2" data-open="DSU012F2" class="link link-reveal link-table xref-fig">2</a>Ai and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1Ci</a>). When fugu and tetraodon are grouped into one category, 90, 98 and 99% of all fish sequences are located in the pure lattice points. (Bii) Lattice points containing sequences derived only from fish genomes are marked with the colours listed in Ai. (Biii) Examples of trinucleotides whose occurrence differs between coelacanth and other fishes. Lattice points are marked as described in Aii.</p></div></div><p>BLSOM can visualize diagnostic oligonucleotides responsible for species-specific clustering (self-organization) and provides information concerning possible molecular evolutionary mechanisms for establishing the present genome signatures. For this visualization, we have previously developed a method to unveil diagnostic oligonucleotides in a way unaffected by a simple difference in G+C% of genomic sequences.<sup><a reveal-id="DSU012C8" data-open="DSU012C8" class="link link-ref link-reveal xref-bibr">8</a></sup> After calculating the frequency of each oligonucleotide expected from the mononucleotide composition for each lattice point, the observed/expected ratio is indicated in red (over-represented) or blue (under-represented) (Fig.<a reveal-id="DSU012F1" data-open="DSU012F1" class="link link-reveal link-table xref-fig">1</a>Aii and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1Aiii and Bii</a>).</p><p>We focus on the oligonucleotides whose occurrence in coelacanth (red in Fig.<a reveal-id="DSU012F1" data-open="DSU012F1" class="link link-reveal link-table xref-fig">1</a>Ai) differs from that in other fishes, i.e. the oligonucleotides, for which the transition between red and blue coincides with the border between the coelacanth's and the other fishes' territories in Fig.<a reveal-id="DSU012F1" data-open="DSU012F1" class="link link-reveal link-table xref-fig">1</a>Aii. CCC+GGG is over-represented (red) only in the coelacanth territory; and CG-possessing trinucleotides are more under-represented in the coelacanth territory (dark blue) than in the other fishes' territories (pale blue). Importantly, all other CG-possessing trinucleotides (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2A</a>), as well as CG (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1Aiii</a>) and all CG-possessing tetranucleotides (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2B</a>), are more under-represented in the coelacanth territory (dark blue) than in the other fishes' territories (pale blue). This shows higher CG suppression in coelacanth than in other fishes; a numerical analysis of the CG suppression will be shown later.</p>                            <h3 scrollto-destination=6045311 id="6045311" class="section-title" data-legacy-id=s3b>3.2. BLSOM for a broad phylogenetic range of vertebrate genomes</h3>
<p>Oligonucleotide BLSOM for fish genomes shows that a notable characteristic of coelacanth distinct from other fishes is the highest CG suppression. Importantly, this finding can be obtained with no prior knowledge, showing a prominent ability of this unsupervised data-mining method in knowledge discovery. Once this type of information is obtained, we can focus on detailed research of the respective genome characteristics. A major mechanism for the evident CG suppression in the mammalian genomes has been connected with methylation at C in CG dinucleotide, which is followed by spontaneous deamination. As a result, CG steadily mutates to TG, resulting in CG suppression and in CA and TG over-representation. In the case of fish genomes, by analysing a wide range of animal genomes from various aspects, Simmen<sup><a reveal-id="DSU012C15" data-open="DSU012C15" class="link link-ref link-reveal xref-bibr">15</a></sup> has found that their CG suppression is also accountable in terms of 5-methylcytosine mutation; in the study, the author has proposed the following four criteria which support the 5-methylcytosine mutation. (i) Genome-level CG suppression. (ii) Genome-level TG and CA excess. (iii) Positive correlation between the local CG relative abundance (i.e. the observed/expected ratio of CG) and the local G+C% level. This is because regions of high G+C content possess a higher DNA melting temperature than low G+C content regions, and therefore, these are less susceptible to strand separation, resulting in a lower rate of deamination and a lesser degree of CG suppression. (iv) Negative correlation between local relative abundance of CG and TG (CA). We have found that all four criteria are applicable to coelacanth (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>), showing that a major mechanism for the CG suppression in coelacanth is related to the 5-methylcytosine mutation, as previously reported for other fishes.<sup><a reveal-id="DSU012C15" data-open="DSU012C15" class="link link-ref link-reveal xref-bibr">15</a></sup></p><p>Methylated C is a well-characterized epigenetic marker, and therefore, the highest CG suppression in coelacanth among fishes should be of particular interest in terms of evolutionary changes in epigenetic systems. We next included five additional vertebrate genomes covering a broad phylogenetic range of tetrapods, for which complete or draft genome sequences were available. On DegTri and DegTetra (Figs<a reveal-id="DSU012F1" data-open="DSU012F1" class="link link-reveal link-table xref-fig">1</a>B and <a reveal-id="DSU012F2" data-open="DSU012F2" class="link link-reveal link-table xref-fig">2</a>A), as well as DegDi (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1C</a>), for 100-kb sequences from the 11 vertebrate genomes, species-specific clustering is apparent. In Figs<a reveal-id="DSU012F1" data-open="DSU012F1" class="link link-reveal link-table xref-fig">1</a>Bii and <a reveal-id="DSU012F2" data-open="DSU012F2" class="link link-reveal link-table xref-fig">2</a>Aii, we mark the lattice points corresponding only to fish territories, which are located primarily in the lower part of the map. </p>                        <a id="6045314" scrollto-destination="6045314"></a>
<div id="DSU012F2" data-id="DSU012F2" class="fig fig-section"><div class="label fig-label">Figure2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01202.png?Expires=2147483647&Signature=aTA32VdXQ-PgsOien3uB-HQunOHNNM8q9POGDADmDha8eZ12edf21XzISCo0Hyh2B6sI3o5JMv-b2X3J5KaWY~uakCFH1oWX6h-8L3OgGNbTXy~QCFRfLDrFJzCAKNhWLeurkQMDRMlZp63y7nSdM2hYBDbeTCVPwFNQ72fKlaSHBCmVQq592RFQoVO-OHzfn9krwHlDpEg1~a9KNXQ04KbkJOXdFnMtmYYmnBToyqaL49DapXxwnW5Adp4zM8HMV-pyFtQ2EzIR19mTsGrKDbvoGJn7kfbSVABLFVL2KbYGDaIlrubkLTKGYB2Sj1Wm~4vgt5mOPTsMppXMCSdtOA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="DegTetra for 100-kb sequences derived from 11 vertebrate genomes. (Ai and ii) Lattice points are marked as described in Fig.1Bi and ii. (Aiii) Examples of tetranucleotides whose occurrence differs between coelacanth and other fishes. Lattice points are marked as described in Fig.1Aii. (B) DegTri for 100-kb unique sequences from 11 vertebrate genomes. DegTri for repeat sequences also has shown the species-specific separation (data not shown). (Ci) DegTri constructed for unique plus repeat sequences. (Cii) Lattice points containing only unique sequences are marked. (Ciii) Lattice points containing only repeat sequences are marked." path-from-xml="dsu01202.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6045314/dsu01202.tiff" path-from-xml="dsu01202.tiff" target="_blank">View large</a><a section="6045314" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/dsu01202.png?Expires=2147483647&Signature=DShQ8fcN2jmB~xHZZM6TXOAo-zwJZOdRQqIYc45Yi39JcyZ9oJj5WgWRaoD5RH2nIbjPm~QZqQaRHWwcRs3-IwsDAxjqsIMQK1kAqcfWeHbnTYu7A3Azv1oY0Y9NnYdTRw~TIdkSKNIsX7YZnw783ZGNcBB4jeHcGh5CRdk79WmNUtIKro6Bj7l4hNKr~yIqyknqCMaC9CLKMkwkrAGQzqLIifX-Bqks4qyrErWoUX6VhhXsC7G4zs5nBa6G14bNfVR8cnnXjncChQ7nn4~CPVEUFEP11qbXlt8bRfbiREV6uiKHVP2GrKFzmdXo3OGLo1C6JyhGyOQsDPS1drk-JQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6045314&ar=408276&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu01202.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>DegTetra for 100-kb sequences derived from 11 vertebrate genomes. (Ai and ii) Lattice points are marked as described in Fig.<a reveal-id="DSU012F1" data-open="DSU012F1" class="link link-reveal link-table xref-fig">1</a>Bi and ii. (Aiii) Examples of tetranucleotides whose occurrence differs between coelacanth and other fishes. Lattice points are marked as described in Fig.<a reveal-id="DSU012F1" data-open="DSU012F1" class="link link-reveal link-table xref-fig">1</a>Aii. (B) DegTri for 100-kb unique sequences from 11 vertebrate genomes. DegTri for repeat sequences also has shown the species-specific separation (data not shown). (Ci) DegTri constructed for unique plus repeat sequences. (Cii) Lattice points containing only unique sequences are marked. (Ciii) Lattice points containing only repeat sequences are marked.</p></div></div><div content-id="DSU012F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01202.png?Expires=2147483647&Signature=aTA32VdXQ-PgsOien3uB-HQunOHNNM8q9POGDADmDha8eZ12edf21XzISCo0Hyh2B6sI3o5JMv-b2X3J5KaWY~uakCFH1oWX6h-8L3OgGNbTXy~QCFRfLDrFJzCAKNhWLeurkQMDRMlZp63y7nSdM2hYBDbeTCVPwFNQ72fKlaSHBCmVQq592RFQoVO-OHzfn9krwHlDpEg1~a9KNXQ04KbkJOXdFnMtmYYmnBToyqaL49DapXxwnW5Adp4zM8HMV-pyFtQ2EzIR19mTsGrKDbvoGJn7kfbSVABLFVL2KbYGDaIlrubkLTKGYB2Sj1Wm~4vgt5mOPTsMppXMCSdtOA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="DegTetra for 100-kb sequences derived from 11 vertebrate genomes. (Ai and ii) Lattice points are marked as described in Fig.1Bi and ii. (Aiii) Examples of tetranucleotides whose occurrence differs between coelacanth and other fishes. Lattice points are marked as described in Fig.1Aii. (B) DegTri for 100-kb unique sequences from 11 vertebrate genomes. DegTri for repeat sequences also has shown the species-specific separation (data not shown). (Ci) DegTri constructed for unique plus repeat sequences. (Cii) Lattice points containing only unique sequences are marked. (Ciii) Lattice points containing only repeat sequences are marked." path-from-xml="dsu01202.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6045314/dsu01202.tiff" path-from-xml="dsu01202.tiff" target="_blank">View large</a><a section="6045314" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/dsu01202.png?Expires=2147483647&Signature=DShQ8fcN2jmB~xHZZM6TXOAo-zwJZOdRQqIYc45Yi39JcyZ9oJj5WgWRaoD5RH2nIbjPm~QZqQaRHWwcRs3-IwsDAxjqsIMQK1kAqcfWeHbnTYu7A3Azv1oY0Y9NnYdTRw~TIdkSKNIsX7YZnw783ZGNcBB4jeHcGh5CRdk79WmNUtIKro6Bj7l4hNKr~yIqyknqCMaC9CLKMkwkrAGQzqLIifX-Bqks4qyrErWoUX6VhhXsC7G4zs5nBa6G14bNfVR8cnnXjncChQ7nn4~CPVEUFEP11qbXlt8bRfbiREV6uiKHVP2GrKFzmdXo3OGLo1C6JyhGyOQsDPS1drk-JQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6045314&ar=408276&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu01202.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>DegTetra for 100-kb sequences derived from 11 vertebrate genomes. (Ai and ii) Lattice points are marked as described in Fig.<a reveal-id="DSU012F1" data-open="DSU012F1" class="link link-reveal link-table xref-fig">1</a>Bi and ii. (Aiii) Examples of tetranucleotides whose occurrence differs between coelacanth and other fishes. Lattice points are marked as described in Fig.<a reveal-id="DSU012F1" data-open="DSU012F1" class="link link-reveal link-table xref-fig">1</a>Aii. (B) DegTri for 100-kb unique sequences from 11 vertebrate genomes. DegTri for repeat sequences also has shown the species-specific separation (data not shown). (Ci) DegTri constructed for unique plus repeat sequences. (Cii) Lattice points containing only unique sequences are marked. (Ciii) Lattice points containing only repeat sequences are marked.</p></div></div><p>The genomes of warm-blooded vertebrates are known to be composed of a long-range segmental G+C% distribution designated as isochore, which has been connected with chromosomal bands.<sup><a reveal-id="DSU012C16  DSU012C20" data-open="DSU012C16  DSU012C20" class="link link-ref link-reveal xref-bibr">1620</a></sup> A single genome, especially of a warm-blooded vertebrate, has multiple territories on DegTri [e.g. four chicken (<em>Gallus gallus</em>) territories marked in pink in Fig.<a reveal-id="DSU012F1" data-open="DSU012F1" class="link link-reveal link-table xref-fig">1</a>Bi]. As previously mentioned, sequences with high or low G+C% are located on the left or right side of the map, and therefore, the split into several sub-territories in the horizontal direction should most likely relate to isochore structures.</p><p>On DegTri and DegTetra, the suppression of CG-possessing oligonucleotides in coelacanth and tetrapods and the high occurrence of CC+GG-possessing oligonucleotides in coelacanth and most tetrapods except chicken are clear (Figs<a reveal-id="DSU012F1" data-open="DSU012F1" class="link link-reveal link-table xref-fig">1</a>Biii and <a reveal-id="DSU012F2" data-open="DSU012F2" class="link link-reveal link-table xref-fig">2</a>Aiii); other examples of diagnostic oligonucleotides for phylotype-dependent clustering are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>. It should also be noted that the oligonucleotides diagnostic for one species are primarily common among the sub-territories of the species, e.g. the characteristic over-representation of CCC+GGG in the tetrapods other than chicken is observed regardless of intra-genomic G+C% differences such as isochores of chicken (Fig.<a reveal-id="DSU012F1" data-open="DSU012F1" class="link link-reveal link-table xref-fig">1</a>Biii). In addition to the characteristic occurrence of CG- and CC+GG-possessing tetranucleotides, phylotype-dependent characteristics that cannot be explained by the characteristics of the dinucleotide composition are observed on DegTetra, e.g. under-representation of GTCA+TGAC in coelacanth and tetrapods (Fig.<a reveal-id="DSU012F2" data-open="DSU012F2" class="link link-reveal link-table xref-fig">2</a>Aiii). Such oligonucleotides' biological significance will be discussed later.</p>                            <h3 scrollto-destination=6045317 id="6045317" class="section-title" data-legacy-id=s3c>3.3. Distinction between unique and repeat sequences</h3>
<p>Higher eukaryote genomes are composed of unique and repeat sequences with different biological significances. Since repeat sequences usually have peculiar oligonucleotide composition, the composition in repeat sequences might be a major causal factor for genome signatures detected by the present oligonucleotide BLSOMs. This possibility is tested as follows. In the UCSC Genome Browser, repeat sequences identified by RepeatMasker and Tandem Repeats Finder (with a period of 12) are specified in lower-case letters to distinguish from unique sequences specified in upper-case letters. We thus separately constructed oligonucleotide BLSOMs for repeat and unique sequences of the 11 vertebrates; we first concatenated repeat and unique sequences separately and divided these artificially concatenated sequences into 100-kb fragments, followed by counting oligonucleotide composition. DegTri for unique sequences shows the clear species-specific separation (Fig.<a reveal-id="DSU012F2" data-open="DSU012F2" class="link link-reveal link-table xref-fig">2</a>B), proving that their genome signatures are not the peculiar characteristics of oligonucleotide compositions in repeat sequences; the same finding was obtained on DegDi and DegTetra (data not shown). In Fig.<a reveal-id="DSU012F2" data-open="DSU012F2" class="link link-reveal link-table xref-fig">2</a>Ci, unique plus repeat sequences have been analysed on one DegTri; using the same map, unique and repeat sequences are separately marked (Fig.<a reveal-id="DSU012F2" data-open="DSU012F2" class="link link-reveal link-table xref-fig">2</a>Cii and iii). Clear separation is observed among species and between unique and repeat sequences, showing differential oligonucleotide compositions among these different categories.</p>                            <h3 scrollto-destination=6045319 id="6045319" class="section-title" data-legacy-id=s3d>3.4. CG suppression in a wide phylogenetic range of vertebrates</h3>
<p>A difference in CG suppression observed among different lineages of vertebrates may relate to the differential features in epigenetic systems.<sup><a reveal-id="DSU012C21" data-open="DSU012C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSU012C22" data-open="DSU012C22" class="link link-ref link-reveal xref-bibr">22</a></sup> Considering the fact that biologically important genome characteristics tend to be maintained stably during evolution, we analysed the CG suppression of many more vertebrates than those analysed with BLSOMs, because CG suppression can be analysed even if their contiguous sequences are rather short (e.g. shorter than 100 kb). Specifically, we analysed CG suppression for 44 vertebrate genomes covering a broad phylogenetic range. The CG/GC ratio (often called the CpG/GpC ratio) has long been used to examine CG suppression for individual species, because this ratio is not affected by the genome G+C% difference. Strictly speaking, however, this ratio should be affected by the possible species-specific GC dinucleotide occurrence. Therefore, the observed/expected ratio of CG (i.e. normalization with the occurrence level expected from the mononucleotide composition) was analysed (Fig.<a reveal-id="DSU012F3" data-open="DSU012F3" class="link link-reveal link-table xref-fig">3</a>A); the species are arranged in descending order of the normalized CG level. Supporting the BLSOM results, the normalized CG level in coelacanth (0.31, arrowed) is clearly lower than in other fishes (indicated by a horizontal bar) and rather equivalent to that of a wide range of tetrapods, e.g. platypus (<em>Ornithorhynchus anatinus</em>; 0.38), <em>X. tropicalis</em> (<em>Xenopus tropicalis</em>; 0.35), cat (<em>Felis catus</em>; 0.33) and tenrec (<em>Echinops telfairii</em>; 0.30). It should also be mentioned that Marsupialia, such as tasmanian devil (<em>Sarcophilus harrisii</em>), wallaby (<em>Macropus eugenii</em>) and opossum (<em>Monodelphis domestica</em>), have a very low-CG occurrence (&lt;0.14). Since the CG suppression has been evolutionarily conserved well within individual phylogenetic lineages, the CG suppression is thought to relate to fundamental biological functions. Similar results are found for the aforementioned CG/GC ratio (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5A</a>). <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5A</a> also shows that the evident CG suppression of coelacanth among fishes is observed in both unique and repeat sequences. <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5B and C</a> shows that CA/AC and TG/GT ratios of coelacanth are higher than those of other fishes and equivalent to most of tetrapods, as expected from the mechanism responsible for the CG suppression; CA and TG over-representation. </p>                        <a id="6045321" scrollto-destination="6045321"></a>
<div id="DSU012F3" data-id="DSU012F3" class="fig fig-section"><div class="label fig-label">Figure3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01203.png?Expires=2147483647&Signature=urBFd9sSqmvnTa5lOBO~SViWVts7MrXBBN0-AC~yf5bD5Gf9qxYgDGTIyCBHYrh3ReDkoZSWeU3LcWTzcFYSzHiMSH86kkyVtgtIrWYkapW7U4F38-PyvlnmUHQybMKFByQlZIES2JmsZCh9-PTpqOxZruQBmw7I04z~GjxVrAhP4nrNZJ86flyQw-O7QYpYm7Jeh7wpi1CWAOzAhe~Ew9CYb78itWx8YmmHXIcLBo9ub4So2xEOTjjqjgLH13LsAVqgDr3gQVC8UYmRYNiIV7rFJK53qzieFpv8J-0mWg5qTH~pVkDadiIa4G~J2Kxs7Q~Aw~hpjOyqgtpTrGiHaQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Normalized CG and CC+GG levels for 44 vertebrate genomes: Armadillo (Dasypus novemcinctus), bushbaby (Otolemur garnettii), cat (F. catus), chicken (G. gallus), chimp (Pan troglodytes), coelacanth (L. chalumnae), cow (Bos taurus), dog (Canis lupus), elephant (Loxodonta africana), fugu (Fugu rubripes), Geospiza fortis (Geospiza fortis), Gibbon (Nomascus leucogenys), Gorilla (Gorilla gorilla), Guinea pig (Cavia porcellus), Hedgehog (Erinaceus europaeus), horse (Equus caballus), human (H. sapiens), lamprey (P. marinus), lizard (A. carolinensis), marmoset (Callithrix jacchus), medaka (O. latipes), micorbat (Myotis lucifugus), mole rat (Heterocephalus glaber), mouse (M. musculus), opposum (M. domestica), orangutan (Pongo pygmaeus), panda (Ailuropoda melanoleuca), pig (Sus scrofa), platypus (O. anatinus), rabbit (Oryctolagus cuniculus), rat (Rattus norvegicus), rhesus (Macaca mulatta), sheep (Ovis aries), shrew (Sorex araneus), stickleback (G. aculeatus), tasmanian devil (S. harrisii), tenrec (E. telfairii), tetraodon (Tetraodon nigroviridis), Turkey (M. gallopavo), turtle (Chrysemys picta), wallaby (M. eugenii), Xenopus (X. tropicalis), Zebra finch (Taeniopygia guttata), and Zebrafish (D. rerio). (A and B) The occurrence level of CG and of CC+GG normalized with the level expected from the mononucleotide composition. The species are arranged in descending order of the normalized CG level. The data for coelacanth is arrowed and those for other fishes are indicated by a horizontal bar." path-from-xml="dsu01203.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6045321/dsu01203.tiff" path-from-xml="dsu01203.tiff" target="_blank">View large</a><a section="6045321" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/dsu01203.png?Expires=2147483647&Signature=K0I328JRaFYHn8AO79enSPr6RB90CeE1V9ZmoSESqKUTRKf0EAEtNVSxyltniOV2lAO5ofrZwNUSmhUNsCAGIBTHyI0W~3miaAhPPD0fXnB0G-ZvKqXiuIM4yegTyRc75-u94zq33AYoasfE6HYI3H3D8rV97hThiBmkWPCTJvSZyDdzNl3emLRhVz7tQYu0ThpI0lxPmXw1~t3cbZ4k391YhkMSWZw9IgHhODbnZUmWyujClpLeN12qnGdzqwtdY4fcTgGSmTLKO9Z~8dv3b~cE7PWfgp3anFajFLNlRgadum7sqfEuHtDGDyPYH9F5lkoI-YDC8K~W73FsHCsQOw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6045321&ar=408276&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu01203.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Normalized CG and CC+GG levels for 44 vertebrate genomes: Armadillo (<em>Dasypus novemcinctus</em>), bushbaby (<em>Otolemur garnettii</em>), cat (<em>F. catus</em>), chicken (<em>G. gallus</em>), chimp (<em>Pan troglodytes</em>), coelacanth (<em>L. chalumnae</em>), cow (<em>Bos taurus</em>), dog (<em>Canis lupus</em>), elephant (<em>Loxodonta africana</em>), fugu (<em>Fugu rubripes</em>), Geospiza fortis (<em>Geospiza fortis</em>), Gibbon (<em>Nomascus leucogenys</em>), Gorilla (<em>Gorilla gorilla</em>), Guinea pig (<em>Cavia porcellus</em>), Hedgehog (<em>Erinaceus europaeus</em>), horse (<em>Equus caballus</em>), human (<em>H. sapiens</em>), lamprey (<em>P. marinus</em>), lizard (<em>A. carolinensis</em>), marmoset (<em>Callithrix jacchus</em>), medaka (<em>O. latipes</em>), micorbat (<em>Myotis lucifugus</em>), mole rat (<em>Heterocephalus glaber</em>), mouse (<em>M. musculus</em>), opposum (<em>M. domestica</em>), orangutan (<em>Pongo pygmaeus</em>), panda (<em>Ailuropoda melanoleuca</em>), pig (<em>Sus scrofa</em>), platypus (<em>O. anatinus</em>), rabbit (<em>Oryctolagus cuniculus</em>), rat (<em>Rattus norvegicus</em>), rhesus (<em>Macaca mulatta</em>), sheep (<em>Ovis aries</em>), shrew (<em>Sorex araneus</em>), stickleback (<em>G. aculeatus</em>), tasmanian devil (<em>S. harrisii</em>), tenrec (<em>E. telfairii</em>), tetraodon (<em>Tetraodon nigroviridis</em>), Turkey (<em>M. gallopavo</em>), turtle (<em>Chrysemys picta</em>), wallaby (<em>M. eugenii</em>), Xenopus (<em>X. tropicalis</em>), Zebra finch (<em>Taeniopygia guttata</em>), and Zebrafish (<em>D. rerio</em>). (A and B) The occurrence level of CG and of CC+GG normalized with the level expected from the mononucleotide composition. The species are arranged in descending order of the normalized CG level. The data for coelacanth is arrowed and those for other fishes are indicated by a horizontal bar.</p></div></div><div content-id="DSU012F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01203.png?Expires=2147483647&Signature=urBFd9sSqmvnTa5lOBO~SViWVts7MrXBBN0-AC~yf5bD5Gf9qxYgDGTIyCBHYrh3ReDkoZSWeU3LcWTzcFYSzHiMSH86kkyVtgtIrWYkapW7U4F38-PyvlnmUHQybMKFByQlZIES2JmsZCh9-PTpqOxZruQBmw7I04z~GjxVrAhP4nrNZJ86flyQw-O7QYpYm7Jeh7wpi1CWAOzAhe~Ew9CYb78itWx8YmmHXIcLBo9ub4So2xEOTjjqjgLH13LsAVqgDr3gQVC8UYmRYNiIV7rFJK53qzieFpv8J-0mWg5qTH~pVkDadiIa4G~J2Kxs7Q~Aw~hpjOyqgtpTrGiHaQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Normalized CG and CC+GG levels for 44 vertebrate genomes: Armadillo (Dasypus novemcinctus), bushbaby (Otolemur garnettii), cat (F. catus), chicken (G. gallus), chimp (Pan troglodytes), coelacanth (L. chalumnae), cow (Bos taurus), dog (Canis lupus), elephant (Loxodonta africana), fugu (Fugu rubripes), Geospiza fortis (Geospiza fortis), Gibbon (Nomascus leucogenys), Gorilla (Gorilla gorilla), Guinea pig (Cavia porcellus), Hedgehog (Erinaceus europaeus), horse (Equus caballus), human (H. sapiens), lamprey (P. marinus), lizard (A. carolinensis), marmoset (Callithrix jacchus), medaka (O. latipes), micorbat (Myotis lucifugus), mole rat (Heterocephalus glaber), mouse (M. musculus), opposum (M. domestica), orangutan (Pongo pygmaeus), panda (Ailuropoda melanoleuca), pig (Sus scrofa), platypus (O. anatinus), rabbit (Oryctolagus cuniculus), rat (Rattus norvegicus), rhesus (Macaca mulatta), sheep (Ovis aries), shrew (Sorex araneus), stickleback (G. aculeatus), tasmanian devil (S. harrisii), tenrec (E. telfairii), tetraodon (Tetraodon nigroviridis), Turkey (M. gallopavo), turtle (Chrysemys picta), wallaby (M. eugenii), Xenopus (X. tropicalis), Zebra finch (Taeniopygia guttata), and Zebrafish (D. rerio). (A and B) The occurrence level of CG and of CC+GG normalized with the level expected from the mononucleotide composition. The species are arranged in descending order of the normalized CG level. The data for coelacanth is arrowed and those for other fishes are indicated by a horizontal bar." path-from-xml="dsu01203.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6045321/dsu01203.tiff" path-from-xml="dsu01203.tiff" target="_blank">View large</a><a section="6045321" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/dsu01203.png?Expires=2147483647&Signature=K0I328JRaFYHn8AO79enSPr6RB90CeE1V9ZmoSESqKUTRKf0EAEtNVSxyltniOV2lAO5ofrZwNUSmhUNsCAGIBTHyI0W~3miaAhPPD0fXnB0G-ZvKqXiuIM4yegTyRc75-u94zq33AYoasfE6HYI3H3D8rV97hThiBmkWPCTJvSZyDdzNl3emLRhVz7tQYu0ThpI0lxPmXw1~t3cbZ4k391YhkMSWZw9IgHhODbnZUmWyujClpLeN12qnGdzqwtdY4fcTgGSmTLKO9Z~8dv3b~cE7PWfgp3anFajFLNlRgadum7sqfEuHtDGDyPYH9F5lkoI-YDC8K~W73FsHCsQOw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6045321&ar=408276&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu01203.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Normalized CG and CC+GG levels for 44 vertebrate genomes: Armadillo (<em>Dasypus novemcinctus</em>), bushbaby (<em>Otolemur garnettii</em>), cat (<em>F. catus</em>), chicken (<em>G. gallus</em>), chimp (<em>Pan troglodytes</em>), coelacanth (<em>L. chalumnae</em>), cow (<em>Bos taurus</em>), dog (<em>Canis lupus</em>), elephant (<em>Loxodonta africana</em>), fugu (<em>Fugu rubripes</em>), Geospiza fortis (<em>Geospiza fortis</em>), Gibbon (<em>Nomascus leucogenys</em>), Gorilla (<em>Gorilla gorilla</em>), Guinea pig (<em>Cavia porcellus</em>), Hedgehog (<em>Erinaceus europaeus</em>), horse (<em>Equus caballus</em>), human (<em>H. sapiens</em>), lamprey (<em>P. marinus</em>), lizard (<em>A. carolinensis</em>), marmoset (<em>Callithrix jacchus</em>), medaka (<em>O. latipes</em>), micorbat (<em>Myotis lucifugus</em>), mole rat (<em>Heterocephalus glaber</em>), mouse (<em>M. musculus</em>), opposum (<em>M. domestica</em>), orangutan (<em>Pongo pygmaeus</em>), panda (<em>Ailuropoda melanoleuca</em>), pig (<em>Sus scrofa</em>), platypus (<em>O. anatinus</em>), rabbit (<em>Oryctolagus cuniculus</em>), rat (<em>Rattus norvegicus</em>), rhesus (<em>Macaca mulatta</em>), sheep (<em>Ovis aries</em>), shrew (<em>Sorex araneus</em>), stickleback (<em>G. aculeatus</em>), tasmanian devil (<em>S. harrisii</em>), tenrec (<em>E. telfairii</em>), tetraodon (<em>Tetraodon nigroviridis</em>), Turkey (<em>M. gallopavo</em>), turtle (<em>Chrysemys picta</em>), wallaby (<em>M. eugenii</em>), Xenopus (<em>X. tropicalis</em>), Zebra finch (<em>Taeniopygia guttata</em>), and Zebrafish (<em>D. rerio</em>). (A and B) The occurrence level of CG and of CC+GG normalized with the level expected from the mononucleotide composition. The species are arranged in descending order of the normalized CG level. The data for coelacanth is arrowed and those for other fishes are indicated by a horizontal bar.</p></div></div><p>CpG islands, which belong to the unique sequence category, have been connected with transcriptional regulatory mechanisms. Figure<a reveal-id="DSU012F4" data-open="DSU012F4" class="link link-reveal link-table xref-fig">4</a> analysed the CG level in the sequences within and outside CpG islands, which were identified by newcpgreport (<a class="link link-uri" href="http://emboss.sourceforge.net/apps/cvs/emboss/apps/newcpgreport.htm">http://emboss.sourceforge.net/apps/cvs/emboss/apps/newcpgreport.htm</a>). The CG level outside CpG islands varies significantly among fishes and the coelacanth has the lowest CG level, as reflecting its global genome characteristics. In contrast, the CG level within CpG islands is rather similar among fishes including coelacanth. This level varies significantly among tetrapods; the level of lizard is similar to that of coelacanth, but those of chicken, mouse and human are clearly higher than that of coelacanth. This difference observed even between closely related species may reflect the difference in actual detailed molecular mechanisms in the transcriptional regulation. The higher CG suppression outside CpG islands (Fig.<a reveal-id="DSU012F4" data-open="DSU012F4" class="link link-reveal link-table xref-fig">4</a>B) is consistent with the hypothesis that hypermutation of methylated C is the main cause of higher CG suppression in coelacanth, because methylation of C generally occurs outside CpG islands. </p>                        <a id="6045323" scrollto-destination="6045323"></a>
<div id="DSU012F4" data-id="DSU012F4" class="fig fig-section"><div class="label fig-label">Figure4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01204.png?Expires=2147483647&Signature=MF3bKUSinepB7FfoHT40iAiAqsD~n1OtfmKgXpu50DvfzAyGYZwzaPCovjVwFYQzf4FVT6kz7agcD99pM-QnnJ6rY-HrM~iy67RJTmotiZZ-hITp-6CG2FpLlhUYn4FEMsJcZczUPJG~LPKy4sEB6CTMwns2fYbvodDGc-FJuZeCjAcyrJPL-RiFXx58gm1EdB2XDOnh3eit~Rxl70FjQ9F2~6qfK9hLtlQy5Njjgz-G1bbX8j1vfRE2mTQdP2QMscl~v-GN88~kgdTJX2D1BJQhchzDRCxAcPtml0eupamFgjBGXp5IW8OPQ6rNpZ3IUm4cxbqdBvaARrfR~SjFbw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="CG occurrence within and outside CpG islands for 11 vertebrate genomes analysed in Fig.2." path-from-xml="dsu01204.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6045323/dsu01204.tiff" path-from-xml="dsu01204.tiff" target="_blank">View large</a><a section="6045323" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/dsu01204.png?Expires=2147483647&Signature=pIU8pp8o5n0s1elh8aZxigIk2x0vTaVJfGL-9OylgmaP1cSrj40olhGpOqljsPpoTLWlNRlRbITjzA9AX3XahKo0XKGEA9ckZYRZHMTl8T9VSIfv63Taum2mXLa1OpFYCPKs8BTi5K7q4pHQkN6~je-h2MSPNwGGLRfrNCpL7HBqhQZyXQ1oHnD6xUO1qWTIdGZicmWQd8RQ9xX~TyFS1vQyoklyoKJloWQ555ahtWWcm2zjRBCJzyqW9hu0fgD~XSGzJmmEyC6uX1btzPGXBwpexncp3jVEson6TMpsOTAlso~p1rME5PFTEL49R4Bru7Gyjqm8mmb4GesclfhD7A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6045323&ar=408276&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu01204.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>CG occurrence within and outside CpG islands for 11 vertebrate genomes analysed in Fig.<a reveal-id="DSU012F2" data-open="DSU012F2" class="link link-reveal link-table xref-fig">2</a>.</p></div></div><div content-id="DSU012F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01204.png?Expires=2147483647&Signature=MF3bKUSinepB7FfoHT40iAiAqsD~n1OtfmKgXpu50DvfzAyGYZwzaPCovjVwFYQzf4FVT6kz7agcD99pM-QnnJ6rY-HrM~iy67RJTmotiZZ-hITp-6CG2FpLlhUYn4FEMsJcZczUPJG~LPKy4sEB6CTMwns2fYbvodDGc-FJuZeCjAcyrJPL-RiFXx58gm1EdB2XDOnh3eit~Rxl70FjQ9F2~6qfK9hLtlQy5Njjgz-G1bbX8j1vfRE2mTQdP2QMscl~v-GN88~kgdTJX2D1BJQhchzDRCxAcPtml0eupamFgjBGXp5IW8OPQ6rNpZ3IUm4cxbqdBvaARrfR~SjFbw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="CG occurrence within and outside CpG islands for 11 vertebrate genomes analysed in Fig.2." path-from-xml="dsu01204.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6045323/dsu01204.tiff" path-from-xml="dsu01204.tiff" target="_blank">View large</a><a section="6045323" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/dsu01204.png?Expires=2147483647&Signature=pIU8pp8o5n0s1elh8aZxigIk2x0vTaVJfGL-9OylgmaP1cSrj40olhGpOqljsPpoTLWlNRlRbITjzA9AX3XahKo0XKGEA9ckZYRZHMTl8T9VSIfv63Taum2mXLa1OpFYCPKs8BTi5K7q4pHQkN6~je-h2MSPNwGGLRfrNCpL7HBqhQZyXQ1oHnD6xUO1qWTIdGZicmWQd8RQ9xX~TyFS1vQyoklyoKJloWQ555ahtWWcm2zjRBCJzyqW9hu0fgD~XSGzJmmEyC6uX1btzPGXBwpexncp3jVEson6TMpsOTAlso~p1rME5PFTEL49R4Bru7Gyjqm8mmb4GesclfhD7A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6045323&ar=408276&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu01204.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>CG occurrence within and outside CpG islands for 11 vertebrate genomes analysed in Fig.<a reveal-id="DSU012F2" data-open="DSU012F2" class="link link-reveal link-table xref-fig">2</a>.</p></div></div>                            <h3 scrollto-destination=6045324 id="6045324" class="section-title" data-legacy-id=s3e>3.5. DNA methylase genes</h3>
<p>The CG occurrence in lamprey (<em>Petromyzon marinus</em>) is the highest (0.78) and rather similar to a level of the very slightly methylated genome of drosophila (<em>Drosophila melanogaster</em>),<sup><a reveal-id="DSU012C23" data-open="DSU012C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSU012C24" data-open="DSU012C24" class="link link-ref link-reveal xref-bibr">24</a></sup> but significantly differs from medaka (<em>Oryzias latipes</em>; 0.46) (Fig.<a reveal-id="DSU012F3" data-open="DSU012F3" class="link link-reveal link-table xref-fig">3</a>A). This wide range of CG suppression among fishes indicates that various changes in DNA-methylation and -demethylation systems have occurred during the fish lineage evolution. Taking this into account, we next consider the possible molecular mechanisms responsible for the highest CG suppression of coelacanth among fishes. Since CG suppression in vertebrate genomes is believed to relate to methylation at CG dinucleotide, we first compared <em>de novo</em> DNA methylases found in the coelacanth genome with those of other fishes and tetrapods; <em>de novo</em> DNA methylases are responsible for the new addition of a methyl group to DNA. As opposed to the two catalytically active <em>de novo</em> DNA methylase genes (<em>dnmt3a</em> and <em>dnmt3b</em>) of mammals, zebrafish (<em>Danio rerio</em>) is known to have six <em>de novo</em> DNA methylase genes and lacks <em>dnmt3l</em> which is catalytically inactive but required for the establishment of genomic imprinting.<sup><a reveal-id="DSU012C25" data-open="DSU012C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DSU012C26" data-open="DSU012C26" class="link link-ref link-reveal xref-bibr">26</a></sup> It is also known that the expression pattern of zebrafish <em>dnmt6</em> and <em>dnmt8</em>, which more closely resemble mammalian <em>dnmt3a</em> in amino acid sequence, more closely resembles that of mammalian <em>dnmt3a</em> than <em>dnmt3b</em>, but the expression pattern of the other four zebrafish genes (<em>dnmt3</em>, <em>dnmt4</em>, <em>dnmt5</em> and <em>dnmt7</em>) resembles the pattern of mammalian <em>dnmt3b</em>, i.e. <em>dnmt6</em> and <em>dnmt8</em> have been shown to more ubiquitously express in developing and adult tissues than the other four zebrafish genes.<sup><a reveal-id="DSU012C27" data-open="DSU012C27" class="link link-ref link-reveal xref-bibr">27</a></sup></p><p>In the coelacanth genome, we can find two <em>dnmt3</em> genes (<em>dnmt3a</em> and <em>dnmt3b</em>) as observed for mammals but no <em>dnmt3l</em> as observed for zebrafish. Using multiple sequence alignment (MUSCLE) in Mega6,<sup><a reveal-id="DSU012C28" data-open="DSU012C28" class="link link-ref link-reveal xref-bibr">28</a></sup> phylogenetic analyses of coelacanth Dnmt3a and Dnmt3b were conducted at a protein-sequence level (Fig.<a reveal-id="DSU012F5" data-open="DSU012F5" class="link link-reveal link-table xref-fig">5</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>). All three programmes (maximum likelihood, minimum evolution and neighbour-joining) show that the coelacanth Dnmt3a is clustered with tetrapods' Dnmt3a, but not with other fishes' Dnmt3a. In the case of Dnmt3b, maximum likelihood shows that the coelacanth Dnmt3b is clustered with tetrapods' Dnmt3b as observed for Dnmt3a (Fig.<a reveal-id="DSU012F5" data-open="DSU012F5" class="link link-reveal link-table xref-fig">5</a>), but minimum evolution and neighbour-joining show it to be outside of tetrapods' and other fishes' clusters (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>). In the case of Dnmt1, which preferentially methylates hemimethylated DNA for maintaining the methylation pattern in the newly synthesized strand, all three programmes show the coelacanth Dnmt1 to be outside of tetrapods' and other fishes' clusters (Fig.<a reveal-id="DSU012F6" data-open="DSU012F6" class="link link-reveal link-table xref-fig">6</a>). </p>                        <a id="6045327" scrollto-destination="6045327"></a>
<div id="DSU012F5" data-id="DSU012F5" class="fig fig-section"><div class="label fig-label">Figure5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01205.png?Expires=2147483647&Signature=3R9XlRdPGVZ7oyIcUbvSFJ66yxdBD5jCMMSO2efqlGb9twW2y4t2GnonTh95dO2Xhw8M6wUx6cru7V3zg4NSuFgHL4mRIIcjdwwimI7E1vIM91RqPwvk-kW7rLp~roHPaa2Hj72Vidl~ACh21VQkeVJtOuDz0EV0bcmoDw~z7YE7R6chfme-Q4YSJfyDgQljF5x2f7sZN0oyu9~GpruqhFlku8nI2fv-oGMjMyQvOJ3SI51nFY-ogddyGa9HWthUCGaxOq~pZ4ElOjfmFRHOktSJUKsq6mkCiz7tfeC7KTHpBlNjJb4KgWx~K4GdA8QUxE0UclhB3UxqVCL5vZPKdQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic tree of Dnmt3 constructed with maximum likelihood. Nine hundred and fifty amino acid sites are used for the tree inference and the tree is unrooted. A bootstrap value is presented above each branch. The scale is proportional to the number of substitutions per amino acid." path-from-xml="dsu01205.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6045327/dsu01205.tiff" path-from-xml="dsu01205.tiff" target="_blank">View large</a><a section="6045327" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/dsu01205.png?Expires=2147483647&Signature=zVoxd2dFL7audOIIWo-l0fHsGNCcObPqws1yAicMKz21YlOTTwV6XXiFVLOxCj8KEyJxfhMr9fGMHc1n7zEfiNvIs0S7dss1M6GIbxurciJEiUS-ZYQ~-nd8IXwNonLaXnAmsJBln~3uXsiNfUMXpoLHQd7kJNcPUP6ojM7JSzumHueQAd2yf93GKsx~2YCx9q8kuyyzt-gPtmQINlfiLnHsRZCHlbnRE0yzIe-YJgzUR-AcGEu31--C5VnRCGI4fuKoViUiv2ltFsM~U3bdqB7UDvpPva0za7Um0s4peWGFpWtnLw6586~WXbpug9PvemwsOaXu~DJnO~xzzfAKrA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6045327&ar=408276&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu01205.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree of Dnmt3 constructed with maximum likelihood. Nine hundred and fifty amino acid sites are used for the tree inference and the tree is unrooted. A bootstrap value is presented above each branch. The scale is proportional to the number of substitutions per amino acid.</p></div></div><div content-id="DSU012F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01205.png?Expires=2147483647&Signature=3R9XlRdPGVZ7oyIcUbvSFJ66yxdBD5jCMMSO2efqlGb9twW2y4t2GnonTh95dO2Xhw8M6wUx6cru7V3zg4NSuFgHL4mRIIcjdwwimI7E1vIM91RqPwvk-kW7rLp~roHPaa2Hj72Vidl~ACh21VQkeVJtOuDz0EV0bcmoDw~z7YE7R6chfme-Q4YSJfyDgQljF5x2f7sZN0oyu9~GpruqhFlku8nI2fv-oGMjMyQvOJ3SI51nFY-ogddyGa9HWthUCGaxOq~pZ4ElOjfmFRHOktSJUKsq6mkCiz7tfeC7KTHpBlNjJb4KgWx~K4GdA8QUxE0UclhB3UxqVCL5vZPKdQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic tree of Dnmt3 constructed with maximum likelihood. Nine hundred and fifty amino acid sites are used for the tree inference and the tree is unrooted. A bootstrap value is presented above each branch. The scale is proportional to the number of substitutions per amino acid." path-from-xml="dsu01205.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6045327/dsu01205.tiff" path-from-xml="dsu01205.tiff" target="_blank">View large</a><a section="6045327" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/dsu01205.png?Expires=2147483647&Signature=zVoxd2dFL7audOIIWo-l0fHsGNCcObPqws1yAicMKz21YlOTTwV6XXiFVLOxCj8KEyJxfhMr9fGMHc1n7zEfiNvIs0S7dss1M6GIbxurciJEiUS-ZYQ~-nd8IXwNonLaXnAmsJBln~3uXsiNfUMXpoLHQd7kJNcPUP6ojM7JSzumHueQAd2yf93GKsx~2YCx9q8kuyyzt-gPtmQINlfiLnHsRZCHlbnRE0yzIe-YJgzUR-AcGEu31--C5VnRCGI4fuKoViUiv2ltFsM~U3bdqB7UDvpPva0za7Um0s4peWGFpWtnLw6586~WXbpug9PvemwsOaXu~DJnO~xzzfAKrA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6045327&ar=408276&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu01205.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree of Dnmt3 constructed with maximum likelihood. Nine hundred and fifty amino acid sites are used for the tree inference and the tree is unrooted. A bootstrap value is presented above each branch. The scale is proportional to the number of substitutions per amino acid.</p></div></div>                        <a id="6045328" scrollto-destination="6045328"></a>
<div id="DSU012F6" data-id="DSU012F6" class="fig fig-section"><div class="label fig-label">Figure6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01206.png?Expires=2147483647&Signature=zxp-rWOrJujzYSzKyZ2GcgadhTTJzwVLIKuTOhp5Thv1A6rKVTumCkQJFcI14ZnorWpCwOZWo7xkmonDoI64LlKnopXgZHN6ny4AfaSRy1tUveW6eoHU59Twdj71od3~c2H6OOUhHjfEm7WvV3mbb4aL1fBrdmEcIg2YAmO5FEiRbBKqtZxOur-yXDH252DNd8TSDz5wMfjBIkq4awIB0M6ODGHg9bZutL44pbSS4zFhSvxXVsH1-Pb9fcTJ5nI6OFbgCzdoW3tpVKwisNBVDw2a41dO8Y9U5As~z2j36cPXCGKRiY7OyPX91V50NQZlvbkfI~6NsdlevjYprczTvg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic tree of Dnmt1 constructed with maximum likelihood. One thousand five hundred and twenty amino acid sites are used for the tree inference and the tree is unrooted. A similar tree was obtained with neighbour-joining and minimum evolution (data not shown)." path-from-xml="dsu01206.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6045328/dsu01206.tiff" path-from-xml="dsu01206.tiff" target="_blank">View large</a><a section="6045328" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/dsu01206.png?Expires=2147483647&Signature=XTsiBaFK2mED-WnOaf--5TBqBEws1KKkzKmrc1DZ6vuOPkT2xnE6BfKSo9a8mjdGeTI7yvRCT0oHs94a9wKRDf06stoRqiG6z3nqx0wDz7n-nfj7~NDJ-KRyh3iFt6ggitDMU0PazfDapBwM5G4T3JHRz4rbgpsUyD9Hoo5~M~GBZ8qdUrDTS0EugUP9UpD1uLrkaRt43vwnWeVgPHbNyhWaqQNoc17l-k8gPeS4JGwUMRJJQ0k5WYfX8aHuCP-k2ZGCm-d0wq0YsrtxnUGTKKH5AD0gkdOWgsTkVNcZG3XERrYGLHTJPsyeu1WHlv4IjHZi9jqfECzaRqSYi75J7A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6045328&ar=408276&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu01206.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree of Dnmt1 constructed with maximum likelihood. One thousand five hundred and twenty amino acid sites are used for the tree inference and the tree is unrooted. A similar tree was obtained with neighbour-joining and minimum evolution (data not shown).</p></div></div><div content-id="DSU012F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/m_dsu01206.png?Expires=2147483647&Signature=zxp-rWOrJujzYSzKyZ2GcgadhTTJzwVLIKuTOhp5Thv1A6rKVTumCkQJFcI14ZnorWpCwOZWo7xkmonDoI64LlKnopXgZHN6ny4AfaSRy1tUveW6eoHU59Twdj71od3~c2H6OOUhHjfEm7WvV3mbb4aL1fBrdmEcIg2YAmO5FEiRbBKqtZxOur-yXDH252DNd8TSDz5wMfjBIkq4awIB0M6ODGHg9bZutL44pbSS4zFhSvxXVsH1-Pb9fcTJ5nI6OFbgCzdoW3tpVKwisNBVDw2a41dO8Y9U5As~z2j36cPXCGKRiY7OyPX91V50NQZlvbkfI~6NsdlevjYprczTvg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic tree of Dnmt1 constructed with maximum likelihood. One thousand five hundred and twenty amino acid sites are used for the tree inference and the tree is unrooted. A similar tree was obtained with neighbour-joining and minimum evolution (data not shown)." path-from-xml="dsu01206.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6045328/dsu01206.tiff" path-from-xml="dsu01206.tiff" target="_blank">View large</a><a section="6045328" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/dsu01206.png?Expires=2147483647&Signature=XTsiBaFK2mED-WnOaf--5TBqBEws1KKkzKmrc1DZ6vuOPkT2xnE6BfKSo9a8mjdGeTI7yvRCT0oHs94a9wKRDf06stoRqiG6z3nqx0wDz7n-nfj7~NDJ-KRyh3iFt6ggitDMU0PazfDapBwM5G4T3JHRz4rbgpsUyD9Hoo5~M~GBZ8qdUrDTS0EugUP9UpD1uLrkaRt43vwnWeVgPHbNyhWaqQNoc17l-k8gPeS4JGwUMRJJQ0k5WYfX8aHuCP-k2ZGCm-d0wq0YsrtxnUGTKKH5AD0gkdOWgsTkVNcZG3XERrYGLHTJPsyeu1WHlv4IjHZi9jqfECzaRqSYi75J7A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6045328&ar=408276&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu01206.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree of Dnmt1 constructed with maximum likelihood. One thousand five hundred and twenty amino acid sites are used for the tree inference and the tree is unrooted. A similar tree was obtained with neighbour-joining and minimum evolution (data not shown).</p></div></div><p>The difference found among these three methylases, which have differential biological functions, may indicate the differential level of contribution to the CG suppression during evolution; the ubiquitously expressed <em>de novo</em> methylase Dnmt3a may have contributed most significantly to the evident CG suppression in coelacanth. For the adequate understanding of CG suppression, however, additional molecular mechanisms such as RNA-directed DNA methylation<sup><a reveal-id="DSU012C29" data-open="DSU012C29" class="link link-ref link-reveal xref-bibr">29</a></sup> and DNA-demethylation systems have to be considered. Since the CG-suppression level has stably been conserved phylogenetically (Fig.<a reveal-id="DSU012F3" data-open="DSU012F3" class="link link-reveal link-table xref-fig">3</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>), analyses of this level from various aspects may provide novel knowledge about fundamental biological mechanisms related to epigenetics.</p>                            <h3 scrollto-destination=6045330 id="6045330" class="section-title" data-legacy-id=s3f>3.6. Oligonucleotides unrelated to CG</h3>
<p>If we know in advance that a fundamental characteristic of the coelacanth genome distinct from other fishes is related to CG suppression, a simple analysis of dinucleotide composition should be enough to prove this characteristic. However, when we have not such background knowledge, an unsupervised data-mining method such as the present BLSOM becomes valuable because it can unveil a wide range of characteristics hidden in the genome sequence. Actually, the occurrence of various CG-non-possessing oligonucleotides in coelacanth was found distinct from that in other fishes, but equivalent to that in almost all tetrapods. A clear example is the over-representation of (CC+GG)-possessing tri-and tetranucleotides on DegTri and DegTetra (Figs<a reveal-id="DSU012F1" data-open="DSU012F1" class="link link-reveal link-table xref-fig">1</a>Biii and <a reveal-id="DSU012F2" data-open="DSU012F2" class="link link-reveal link-table xref-fig">2</a>Aiii). Figure<a reveal-id="DSU012F3" data-open="DSU012F3" class="link link-reveal link-table xref-fig">3</a>B presents the observed/expected ratio of CC+GG in 44 vertebrate genomes, and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S7A and B</a> presents the results for CCC+GGG and CCCC+GGGG. As observed on BLSOMs (Fig.<a reveal-id="DSU012F1" data-open="DSU012F1" class="link link-reveal link-table xref-fig">1</a>Aii and Biii, and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1Aiii, Bii and Ciii</a>), their occurrences in coelacanth are clearly higher than in other fishes. The same is true for most tetrapods, including <em>Marsupialia</em> but excluding chicken and turkey (<em>Meleagris gallopavo</em>). It should also be noted that the occurrence of CCCC+GGGG in coelacanth is the highest among the vertebrates analysed (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S7B</a>).</p>                            <h2 scrollto-destination=6045332 id="6045332" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>The genome characteristics maintained stably in a certain phylogenetic lineage should reflect its fundamental biological significance. An apparent causative factor for the genome signature is the context-dependent DNA mutation and repair mechanisms. It should also be mentioned that the oligonucleotides longer than tetranucleotides often represent motif sequences responsible for sequence-specific protein binding (e.g. transcription factor binding). The occurrence of such motif oligonucleotides should differ from the level expected from the mononucleotide composition in the respective genome and may differ among genomic portions within one genome. We have recently found that DegPenta- and DegHexa-BLSOMs for the human genome can effectively detect characteristic enrichment of many transcription factor-binding motifs in pericentric heterochromatin regions.<sup><a reveal-id="DSU012C30" data-open="DSU012C30" class="link link-ref link-reveal xref-bibr">30</a></sup> The present study has found evident CG suppression and CC+GG over-representation in coelacanth and tetrapods, by using DegDi, DegTri and DegTetra. If the BLSOM analysis extends to oligonucleotides longer than tetranucleotides, we may assign <em>in silico</em> the oligonucleotides with important biological functions such as transcription factor-binding in the genome of coelacanth and other species, for which experimental studies other than DNA sequencing have been poorly done.</p><p>BLSOM can analyse big sequence data such as millions of sequences simultaneously<sup><a reveal-id="DSU012C31" data-open="DSU012C31" class="link link-ref link-reveal xref-bibr">31</a></sup> and, therefore, can analyse genome sequences from a wide range of species and visualize characteristics of their oligonucleotide composition on a single map. This can support an efficient knowledge discovery from the big data with no prior knowledge. Pointing out the oligonucleotides whose occurrence is peculiar commonly (and thus stably) in a certain lineage should provide information concerning biological roles of these oligonucleotides in the lineage. In conclusion, BLSOM is a powerful and timely tool useful for genome studies in the era of big sequence data obtained from high-throughput DNA sequencers.</p><p><em>Conflict of Interest statement.</em> None declared.</p>                            <h2 scrollto-destination=6045336 id="6045336" class="section-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=6045338 id="6045338" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by Grant-in-Aid for Scientific Research (C) and for Young Scientists (B) from the Ministry of Education, Culture, Sports, Science and Technology of Japan and by the Grant-in-Aid for JSPS Fellows (JSPS KAKENHI Number 24-9979). The computation was done in part with the Earth Simulator of Japan Agency for Marine-Earth Science and Technology.</p>                            <h2 scrollto-destination=6045340 id="6045340" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSU012C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU012C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nussinov</div> <div class="given-names">R.</div></div>.</div><div class="article-title">Doublet frequencies in evolutionary distinct groups</div>,<div class="source">Nucleic Acids Res.</div>,<div class="year">1984</div>,vol.<div class="volume">12</div>(pg.<div class="fpage">1749</div>-<div class="lpage">63</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Doublet%20frequencies%20in%20evolutionary%20distinct%20groups&amp;author=R.%20Nussinov&amp;publication_year=1984&amp;journal=Nucleic%20Acids%20Res.&amp;volume=12&amp;pages=1749-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/12.3.1749" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F12.3.1749" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F12.3.1749"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6583663" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU012C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU012C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Karlin</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Campbell</div> <div class="given-names">A.M.</div></div>, <div class="name"><div class="surname">Mrazek</div> <div class="given-names">J.</div></div>.</div><div class="article-title">Comparative DNA analysis across diverse genomes</div>,<div class="source">Annu. Rev. Genet.</div>,<div class="year">1998</div>,vol.<div class="volume">32</div>(pg.<div class="fpage">185</div>-<div class="lpage">225</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20DNA%20analysis%20across%20diverse%20genomes&amp;author=S.%20Karlin&amp;author=A.M.%20Campbell&amp;author=J.%20Mrazek&amp;publication_year=1998&amp;journal=Annu.%20Rev.%20Genet.&amp;volume=32&amp;pages=185-225" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.genet.32.1.185" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genet.32.1.185" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.genet.32.1.185"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9928479" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU012C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU012C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gentles</div> <div class="given-names">A.J.</div></div>, <div class="name"><div class="surname">Karlin</div> <div class="given-names">S.</div></div>.</div><div class="article-title">Genome-scale compositional comparisons in eukaryotes</div>,<div class="source">Genome Res.</div>,<div class="year">2001</div>,vol.<div class="volume">11</div>(pg.<div class="fpage">540</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-scale%20compositional%20comparisons%20in%20eukaryotes&amp;author=A.J.%20Gentles&amp;author=S.%20Karlin&amp;publication_year=2001&amp;journal=Genome%20Res.&amp;volume=11&amp;pages=540-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.163101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.163101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.163101"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11282969" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU012C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU012C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kohonen</div> <div class="given-names">T.</div></div>.</div><div class="article-title">Self-organized formation of topologically correct feature maps</div>,<div class="source">Biol. Cybern.</div>,<div class="year">1982</div>,vol.<div class="volume">43</div>(pg.<div class="fpage">59</div>-<div class="lpage">69</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Self-organized%20formation%20of%20topologically%20correct%20feature%20maps&amp;author=T.%20Kohonen&amp;publication_year=1982&amp;journal=Biol.%20Cybern.&amp;volume=43&amp;pages=59-69" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00337288" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00337288" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00337288"> </span></div></div></div></div></div></div><div content-id="DSU012C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU012C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kohonen</div> <div class="given-names">T.</div></div>, <div class="name"><div class="surname">Oja</div> <div class="given-names">E.</div></div>, <div class="name"><div class="surname">Simula</div> <div class="given-names">O.</div></div>, <div class="name"><div class="surname">Visa</div> <div class="given-names">A.</div></div>, <div class="name"><div class="surname">Kangas</div> <div class="given-names">J.</div></div>.</div><div class="article-title">Engineering applications of the self-organizing map</div>,<div class="source">Proc. IEEE</div>,<div class="year">1996</div>,vol.<div class="volume">84</div>(pg.<div class="fpage">1358</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Engineering%20applications%20of%20the%20self-organizing%20map&amp;author=T.%20Kohonen&amp;author=E.%20Oja&amp;author=O.%20Simula&amp;author=A.%20Visa&amp;author=J.%20Kangas&amp;publication_year=1996&amp;journal=Proc.%20IEEE&amp;volume=84&amp;pages=1358-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1109/5.537105" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1109%2F5.537105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1109%2F5.537105"> </span></div></div></div></div></div></div><div content-id="DSU012C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU012C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kanaya</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Kinouchi</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Abe</div> <div class="given-names">T.</div></div>,et al.</div><div class="article-title">Analysis of codon usage diversity of bacterial genes with a self-organizing map (SOM): characterization of horizontally transferred genes with emphasis on the <em>E. coli</em> O157 genome</div>,<div class="source">Gene</div>,<div class="year">2001</div>,vol.<div class="volume">276</div>(pg.<div class="fpage">89</div>-<div class="lpage">99</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20codon%20usage%20diversity%20of%20bacterial%20genes%20with%20a%20self-organizing%20map%20(SOM)%3A%20characterization%20of%20horizontally%20transferred%20genes%20with%20emphasis%20on%20the%20E.%20coli%20O157%20genome&amp;author=S.%20Kanaya&amp;author=M.%20Kinouchi&amp;author=T.%20Abe&amp;publication_year=2001&amp;journal=Gene&amp;volume=276&amp;pages=89-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(01)00673-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(01)00673-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(01)00673-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11591475" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU012C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU012C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Abe</div> <div class="given-names">T.</div></div>, <div class="name"><div class="surname">Kanaya</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Kinouchi</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Ichiba</div> <div class="given-names">Y.</div></div>, <div class="name"><div class="surname">Kozuki</div> <div class="given-names">T.</div></div>, <div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>.</div><div class="article-title">Informatics for unveiling hidden genome signatures</div>,<div class="source">Genome Res.</div>,<div class="year">2003</div>,vol.<div class="volume">13</div>(pg.<div class="fpage">693</div>-<div class="lpage">702</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Informatics%20for%20unveiling%20hidden%20genome%20signatures&amp;author=T.%20Abe&amp;author=S.%20Kanaya&amp;author=M.%20Kinouchi&amp;author=Y.%20Ichiba&amp;author=T.%20Kozuki&amp;author=T.%20Ikemura&amp;publication_year=2003&amp;journal=Genome%20Res.&amp;volume=13&amp;pages=693-702" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.634603" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.634603" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.634603"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12671005" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU012C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU012C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Abe</div> <div class="given-names">T.</div></div>, <div class="name"><div class="surname">Sugawara</div> <div class="given-names">H.</div></div>, <div class="name"><div class="surname">Kinouchi</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Kanaya</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>.</div><div class="article-title">Novel phylogenetic studies of genomic sequence fragments derived from uncultured microbe mixtures in environmental and clinical samples</div>,<div class="source">DNA Res.</div>,<div class="year">2005</div>,vol.<div class="volume">12</div>(pg.<div class="fpage">281</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20phylogenetic%20studies%20of%20genomic%20sequence%20fragments%20derived%20from%20uncultured%20microbe%20mixtures%20in%20environmental%20and%20clinical%20samples&amp;author=T.%20Abe&amp;author=H.%20Sugawara&amp;author=M.%20Kinouchi&amp;author=S.%20Kanaya&amp;author=T.%20Ikemura&amp;publication_year=2005&amp;journal=DNA%20Res.&amp;volume=12&amp;pages=281-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsi015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsi015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsi015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16769690" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU012C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU012C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Abe</div> <div class="given-names">T.</div></div>, <div class="name"><div class="surname">Sugawara</div> <div class="given-names">H.</div></div>, <div class="name"><div class="surname">Kinouchi</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Kanaya</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>.</div><div class="article-title">A large-scale self-organizing map (SOM) unveils sequence characteristics of a wide range of eukaryote genomes</div>,<div class="source">Gene</div>,<div class="year">2006</div>,vol.<div class="volume">365</div>(pg.<div class="fpage">27</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20large-scale%20self-organizing%20map%20(SOM)%20unveils%20sequence%20characteristics%20of%20a%20wide%20range%20of%20eukaryote%20genomes&amp;author=T.%20Abe&amp;author=H.%20Sugawara&amp;author=M.%20Kinouchi&amp;author=S.%20Kanaya&amp;author=T.%20Ikemura&amp;publication_year=2006&amp;journal=Gene&amp;volume=365&amp;pages=27-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2005.09.040" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2005.09.040" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2005.09.040"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16364569" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU012C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU012C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Noonan</div> <div class="given-names">J.P.</div></div>, <div class="name"><div class="surname">Grimwood</div> <div class="given-names">J.</div></div>, <div class="name"><div class="surname">Danke</div> <div class="given-names">J.</div></div>,et al.</div><div class="article-title">Coelacanth genome sequence reveals the evolutionary history of vertebrate genes</div>,<div class="source">Genome Res.</div>,<div class="year">2004</div>,vol.<div class="volume">14</div>(pg.<div class="fpage">2397</div>-<div class="lpage">405</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Coelacanth%20genome%20sequence%20reveals%20the%20evolutionary%20history%20of%20vertebrate%20genes&amp;author=J.P.%20Noonan&amp;author=J.%20Grimwood&amp;author=J.%20Danke&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=2397-405" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.2972804" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.2972804" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.2972804"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15545497" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU012C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU012C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Amemiya</div> <div class="given-names">C.T.</div></div>, <div class="name"><div class="surname">Alfldi</div> <div class="given-names">J.</div></div>, <div class="name"><div class="surname">Lee</div> <div class="given-names">A.P.</div></div>,et al.</div><div class="article-title">The African coelacanth genome provides insights into tetrapod evolution</div>,<div class="source">Nature</div>,<div class="year">2013</div>,vol.<div class="volume">496</div>(pg.<div class="fpage">311</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20African%20coelacanth%20genome%20provides%20insights%20into%20tetrapod%20evolution&amp;author=C.T.%20Amemiya&amp;author=J.%20Alf%C3%B6ldi&amp;author=A.P.%20Lee&amp;publication_year=2013&amp;journal=Nature&amp;volume=496&amp;pages=311-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature12027" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature12027" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature12027"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23598338" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU012C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU012C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nikaido</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Noguchi</div> <div class="given-names">H.</div></div>, <div class="name"><div class="surname">Nishihara</div> <div class="given-names">H.</div></div>,et al.</div><div class="article-title">Coelacanth genomes reveal signatures for evolutionary transition from water to land</div>,<div class="source">Genome Res.</div>,<div class="year">2013</div>,vol.<div class="volume">23</div>(pg.<div class="fpage">1740</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Coelacanth%20genomes%20reveal%20signatures%20for%20evolutionary%20transition%20from%20water%20to%20land&amp;author=M.%20Nikaido&amp;author=H.%20Noguchi&amp;author=H.%20Nishihara&amp;publication_year=2013&amp;journal=Genome%20Res.&amp;volume=23&amp;pages=1740-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.158105.113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.158105.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.158105.113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23878157" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU012C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU012C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Touchon</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Nicolay</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Audit</div> <div class="given-names">B.</div></div>,et al.</div><div class="article-title">Replication-associated strand asymmetries in mammalian genomes: toward detection of replication origins</div>,<div class="source">Proc. Natl. Acad. Sci. USA</div>,<div class="year">2005</div>,vol.<div class="volume">102</div>(pg.<div class="fpage">9836</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Replication-associated%20strand%20asymmetries%20in%20mammalian%20genomes%3A%20toward%20detection%20of%20replication%20origins&amp;author=M.%20Touchon&amp;author=S.%20Nicolay&amp;author=B.%20Audit&amp;publication_year=2005&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=102&amp;pages=9836-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0500577102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0500577102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0500577102"> </span></div></div></div></div></div></div><div content-id="DSU012C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU012C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Iwasaki</div> <div class="given-names">Y.</div></div>, <div class="name"><div class="surname">Abe</div> <div class="given-names">T.</div></div>, <div class="name"><div class="surname">Wada</div> <div class="given-names">K.</div></div>, <div class="name"><div class="surname">Itoh</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>.</div><div class="article-title">Prediction of directional changes of influenza A virus genome sequences with emphasis on pandemic H1N1/09 as a model case</div>,<div class="source">DNA Res.</div>,<div class="year">2011</div>,vol.<div class="volume">18</div>(pg.<div class="fpage">125</div>-<div class="lpage">36</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20directional%20changes%20of%20influenza%20A%20virus%20genome%20sequences%20with%20emphasis%20on%20pandemic%20H1N1%2F09%20as%20a%20model%20case&amp;author=Y.%20Iwasaki&amp;author=T.%20Abe&amp;author=K.%20Wada&amp;author=M.%20Itoh&amp;author=T.%20Ikemura&amp;publication_year=2011&amp;journal=DNA%20Res.&amp;volume=18&amp;pages=125-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsr005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsr005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsr005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21444341" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU012C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU012C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Simmen</div> <div class="given-names">M.W.</div></div>.</div><div class="article-title">Genome-scale relationships between cytosine methylation and dinucleotide abundances in animals</div>,<div class="source">Genomics</div>,<div class="year">2008</div>,vol.<div class="volume">92</div>(pg.<div class="fpage">33</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-scale%20relationships%20between%20cytosine%20methylation%20and%20dinucleotide%20abundances%20in%20animals&amp;author=M.W.%20Simmen&amp;publication_year=2008&amp;journal=Genomics&amp;volume=92&amp;pages=33-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ygeno.2008.03.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ygeno.2008.03.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ygeno.2008.03.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18485662" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU012C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU012C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bernardi</div> <div class="given-names">G.</div></div>, <div class="name"><div class="surname">Olofsson</div> <div class="given-names">B.</div></div>, <div class="name"><div class="surname">Filipski</div> <div class="given-names">J.</div></div>,et al.</div><div class="article-title">The mosaic genome of warm-blooded vertebrates</div>,<div class="source">Science</div>,<div class="year">1985</div>,vol.<div class="volume">228</div>(pg.<div class="fpage">953</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20mosaic%20genome%20of%20warm-blooded%20vertebrates&amp;author=G.%20Bernardi&amp;author=B.%20Olofsson&amp;author=J.%20Filipski&amp;publication_year=1985&amp;journal=Science&amp;volume=228&amp;pages=953-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.4001930" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.4001930" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.4001930"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4001930" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU012C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU012C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>.</div><div class="article-title">Codon usage and transfer RNA content in unicellular and multicellular organisms</div>,<div class="source">Mol. Biol. Evol.</div>,<div class="year">1985</div>,vol.<div class="volume">2</div>(pg.<div class="fpage">13</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20and%20transfer%20RNA%20content%20in%20unicellular%20and%20multicellular%20organisms&amp;author=T.%20Ikemura&amp;publication_year=1985&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=2&amp;pages=13-34" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3916708" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU012C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU012C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>, <div class="name"><div class="surname">Aota</div> <div class="given-names">S.</div></div>.</div><div class="article-title">Global variation in G+C content along vertebrate genome DNA: possible correlation with chromosome band structures</div>,<div class="source">J. Mol. Biol.</div>,<div class="year">1988</div>,vol.<div class="volume">203</div>(pg.<div class="fpage">1</div>-<div class="lpage">13</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20variation%20in%20G%2BC%20content%20along%20vertebrate%20genome%20DNA%3A%20possible%20correlation%20with%20chromosome%20band%20structures&amp;author=T.%20Ikemura&amp;author=S.%20Aota&amp;publication_year=1988&amp;journal=J.%20Mol.%20Biol.&amp;volume=203&amp;pages=1-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(88)90086-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(88)90086-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(88)90086-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3054117" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU012C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU012C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>, <div class="name"><div class="surname">Wada</div> <div class="given-names">K.</div></div>.</div><div class="article-title">Evident diversity of codon usage patterns of human genes with respect to chromosome banding patterns and chromosome numbers; relation between nucleotide sequence data and cytogenetic data</div>,<div class="source">Nucl. Acids Res.</div>,<div class="year">1991</div>,vol.<div class="volume">19</div>(pg.<div class="fpage">4333</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evident%20diversity%20of%20codon%20usage%20patterns%20of%20human%20genes%20with%20respect%20to%20chromosome%20banding%20patterns%20and%20chromosome%20numbers%3B%20relation%20between%20nucleotide%20sequence%20data%20and%20cytogenetic%20data&amp;author=T.%20Ikemura&amp;author=K.%20Wada&amp;publication_year=1991&amp;journal=Nucl.%20Acids%20Res.&amp;volume=19&amp;pages=4333-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/19.16.4333" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F19.16.4333" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F19.16.4333"> </span></div></div></div></div></div></div><div content-id="DSU012C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU012C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bernardi</div> <div class="given-names">G.</div></div>.</div>,<div class="source">Structural and Evolutionary Genomics: Natural Selection in Genome Evolution</div>,<div class="year">2004</div><div class="publisher-loc">New York</div><div class="publisher-name">Elsevier</div></div></div></div></div><div content-id="DSU012C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU012C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Walsh</div> <div class="given-names">C.P.</div></div>, <div class="name"><div class="surname">Bestor</div> <div class="given-names">T.H.</div></div>.</div><div class="article-title">Cytosine methylation and mammalian development</div>,<div class="source">Genes Dev.</div>,<div class="year">1999</div>,vol.<div class="volume">13</div>(pg.<div class="fpage">26</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cytosine%20methylation%20and%20mammalian%20development&amp;author=C.P.%20Walsh&amp;author=T.H.%20Bestor&amp;publication_year=1999&amp;journal=Genes%20Dev.&amp;volume=13&amp;pages=26-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.13.1.26" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.13.1.26" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.13.1.26"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9887097" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU012C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU012C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yoder</div> <div class="given-names">J.A.</div></div>, <div class="name"><div class="surname">Walsh</div> <div class="given-names">C.P.</div></div>, <div class="name"><div class="surname">Bestor</div> <div class="given-names">T.H.</div></div>.</div><div class="article-title">Cytosine methylation and the ecology of intragenomic parasites</div>,<div class="source">Trends Genet.</div>,<div class="year">1997</div>,vol.<div class="volume">13</div>(pg.<div class="fpage">335</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cytosine%20methylation%20and%20the%20ecology%20of%20intragenomic%20parasites&amp;author=J.A.%20Yoder&amp;author=C.P.%20Walsh&amp;author=T.H.%20Bestor&amp;publication_year=1997&amp;journal=Trends%20Genet.&amp;volume=13&amp;pages=335-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(97)01181-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(97)01181-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(97)01181-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9260521" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU012C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU012C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gowher</div> <div class="given-names">H.</div></div>, <div class="name"><div class="surname">Leismann</div> <div class="given-names">O.</div></div>, <div class="name"><div class="surname">Jeltsch</div> <div class="given-names">A.</div></div>.</div><div class="article-title">DNA of <em>Drosophila melanogaster</em> contains 5-methylcytosine</div>,<div class="source">EMBO J.</div>,<div class="year">2000</div>,vol.<div class="volume">24</div>(pg.<div class="fpage">6918</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20of%20Drosophila%20melanogaster%20contains%205-methylcytosine&amp;author=H.%20Gowher&amp;author=O.%20Leismann&amp;author=A.%20Jeltsch&amp;publication_year=2000&amp;journal=EMBO%20J.&amp;volume=24&amp;pages=6918-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/emboj/19.24.6918" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Femboj%2F19.24.6918" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Femboj%2F19.24.6918"> </span></div></div></div></div></div></div><div content-id="DSU012C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU012C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lyko</div> <div class="given-names">F.</div></div>, <div class="name"><div class="surname">Ramsahoye</div> <div class="given-names">B.H.</div></div>, <div class="name"><div class="surname">Jaenisch</div> <div class="given-names">R.</div></div>.</div><div class="article-title">DNA methylation in <em>Drosophila melanogaster</em></div>,<div class="source">Nature</div>,<div class="year">2000</div>,vol.<div class="volume">408</div>(pg.<div class="fpage">538</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20methylation%20in%20Drosophila%20melanogaster&amp;author=F.%20Lyko&amp;author=B.H.%20Ramsahoye&amp;author=R.%20Jaenisch&amp;publication_year=2000&amp;journal=Nature&amp;volume=408&amp;pages=538-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35046205" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35046205" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35046205"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11117732" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU012C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU012C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shimoda</div> <div class="given-names">N.</div></div>, <div class="name"><div class="surname">Yamakoshi</div> <div class="given-names">K.</div></div>, <div class="name"><div class="surname">Miyake</div> <div class="given-names">A.</div></div>, <div class="name"><div class="surname">Takeda</div> <div class="given-names">H.</div></div>.</div><div class="article-title">Identification of a gene required for de novo DNA methylation of the zebrafish no tail gene</div>,<div class="source">Dev. Dyn.</div>,<div class="year">2005</div>,vol.<div class="volume">233</div>(pg.<div class="fpage">1509</div>-<div class="lpage">16</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20a%20gene%20required%20for%20de%20novo%20DNA%20methylation%20of%20the%20zebrafish%20no%20tail%20gene&amp;author=N.%20Shimoda&amp;author=K.%20Yamakoshi&amp;author=A.%20Miyake&amp;author=H.%20Takeda&amp;publication_year=2005&amp;journal=Dev.%20Dyn.&amp;volume=233&amp;pages=1509-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0177" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0177" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0177"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15937923" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU012C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU012C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Smith</div> <div class="given-names">T.H.</div></div>, <div class="name"><div class="surname">Dueck</div> <div class="given-names">C.C.</div></div>, <div class="name"><div class="surname">Mhanni</div> <div class="given-names">A.A.</div></div>, <div class="name"><div class="surname">McGowan</div> <div class="given-names">R.A.</div></div>.</div><div class="article-title">Novel splice variants associated with one of the zebrafish dnmt3 genes</div>,<div class="source">BMC Dev. Biol.</div>,<div class="year">2005</div>,vol.<div class="volume">5</div>pg.<div class="fpage">23</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20splice%20variants%20associated%20with%20one%20of%20the%20zebrafish%20dnmt3%20genes&amp;author=T.H.%20Smith&amp;author=C.C.%20Dueck&amp;author=A.A.%20Mhanni&amp;author=R.A.%20McGowan&amp;publication_year=2005&amp;journal=BMC%20Dev.%20Biol.&amp;volume=5&amp;pages=23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-213X-5-23" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-213X-5-23" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-213X-5-23"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16236173" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU012C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU012C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Smith</div> <div class="given-names">T.H.</div></div>, <div class="name"><div class="surname">Collins</div> <div class="given-names">T.M.</div></div>, <div class="name"><div class="surname">McGowan</div> <div class="given-names">R.A.</div></div>.</div><div class="article-title">Expression of the dnmt3 genes in zebrafish development: similarity to Dnmt3a and Dnmt3b</div>,<div class="source">Dev. Genes. Evol.</div>,<div class="year">2011</div>,vol.<div class="volume">220</div>(pg.<div class="fpage">347</div>-<div class="lpage">53</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20of%20the%20dnmt3%20genes%20in%20zebrafish%20development%3A%20similarity%20to%20Dnmt3a%20and%20Dnmt3b&amp;author=T.H.%20Smith&amp;author=T.M.%20Collins&amp;author=R.A.%20McGowan&amp;publication_year=2011&amp;journal=Dev.%20Genes.%20Evol.&amp;volume=220&amp;pages=347-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00427-010-0347-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00427-010-0347-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00427-010-0347-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21258815" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU012C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU012C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div>, <div class="name"><div class="surname">Stecher</div> <div class="given-names">G.</div></div>, <div class="name"><div class="surname">Peterson</div> <div class="given-names">D.</div></div>, <div class="name"><div class="surname">Filipski</div> <div class="given-names">A.</div></div>, <div class="name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div>.</div><div class="article-title">MEGA6: molecular evolutionary genetics analysis version 6.0</div>,<div class="source">Mol. Biol. Evol.</div>,<div class="year">2013</div>,vol.<div class="volume">30</div>(pg.<div class="fpage">2725</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA6%3A%20molecular%20evolutionary%20genetics%20analysis%20version%206.0&amp;author=K.%20Tamura&amp;author=G.%20Stecher&amp;author=D.%20Peterson&amp;author=A.%20Filipski&amp;author=S.%20Kumar&amp;publication_year=2013&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=30&amp;pages=2725-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/mst197" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmst197" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmst197"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24132122" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU012C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU012C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bayne</div> <div class="given-names">E.H.</div></div>, <div class="name"><div class="surname">Allshire</div> <div class="given-names">R.C.</div></div>.</div><div class="article-title">RNA-directed transcriptional gene silencing in mammals</div>,<div class="source">Trends Genet.</div>,<div class="year">2005</div>,vol.<div class="volume">21</div>(pg.<div class="fpage">370</div>-<div class="lpage">3</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA-directed%20transcriptional%20gene%20silencing%20in%20mammals&amp;author=E.H.%20Bayne&amp;author=R.C.%20Allshire&amp;publication_year=2005&amp;journal=Trends%20Genet.&amp;volume=21&amp;pages=370-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2005.05.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2005.05.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2005.05.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15908035" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU012C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU012C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Iwasaki</div> <div class="given-names">Y.</div></div>, <div class="name"><div class="surname">Wada</div> <div class="given-names">K.</div></div>, <div class="name"><div class="surname">Wada</div> <div class="given-names">Y.</div></div>, <div class="name"><div class="surname">Abe</div> <div class="given-names">T.</div></div>, <div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>.</div><div class="article-title">Notable clustering of transcription-factor-binding motifs in human pericentric regions and its biological significance</div>,<div class="source">Chromosome Res.</div>,<div class="year">2013</div>,vol.<div class="volume">21</div>(pg.<div class="fpage">461</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Notable%20clustering%20of%20transcription-factor-binding%20motifs%20in%20human%20pericentric%20regions%20and%20its%20biological%20significance&amp;author=Y.%20Iwasaki&amp;author=K.%20Wada&amp;author=Y.%20Wada&amp;author=T.%20Abe&amp;author=T.%20Ikemura&amp;publication_year=2013&amp;journal=Chromosome%20Res.&amp;volume=21&amp;pages=461-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10577-013-9371-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10577-013-9371-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10577-013-9371-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23896648" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU012C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU012C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Abe</div> <div class="given-names">T.</div></div>, <div class="name"><div class="surname">Sugawara</div> <div class="given-names">H.</div></div>, <div class="name"><div class="surname">Kanaya</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>.</div><div class="article-title">Sequences from almost all prokaryotic, eukaryotic, and viral genomes available could be classified according to genomes on a large-scale self-organizing map constructed with the earth simulator</div>,<div class="source">J Earth Simulator</div>,<div class="year">2006</div>,vol.<div class="volume">6</div>(pg.<div class="fpage">17</div>-<div class="lpage">23</div>)</div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement"> The Author 2014. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=408276&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/dsu012-dsu012supp_figs.ppt?Expires=2147483647&amp;Signature=Hlt1OSsbvctU0F3yM8~tu6iQEtTW3p1nqF~XPXR4A4bw4YRejZt3qdSCnqG9jXi7mN1ZeMAfub2vMnwBHRaJkkYiYVKZG82RqcyQI4RSO3HIkS1gpazquex6SKveH0YN-lhLExRxCff2QjqWJx0mdh~wsh1WkyNVtlaO57iLqztasWdn048AyrVwRs3RqmMhPg9KBsdOdrrm-fqICqePDlGhy7ipiV3sGFI--k2L-t-hfObBXRoP3CrzcGeNKUnXjbXOt6KyXTk3d6s6NECt58WFPnFRSNEzxav-RAcOsWFl-IDwa37k79QgVSatMtjnQHmEfRpG~O6K1k8TOL2qFQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093/dnares/dsu012/2/dsu012-dsu012supp.docx?Expires=2147483647&amp;Signature=KZCusTq4ydxPieeVEZ3mpYnww7Ed2ETkeDnsM5CIgJX6ic9dffgYWmw88uWf-M6Zc-Mf3EGYnoYYDi5nnqg1hCH0WitQgP1yrYtmuP1s7k5FNPHtu~nkLdjvYqGGDLLb7CwDl5kHTV8kQGKr2BCcYPwn3GwPu8pPcKzkI6XNkK1JoOeeroZPjvfuBQJX1BN-WxYAY8YSb3umFTcECSd~BYLEiRSbKFiv2Md-0uBPn-rjWdQ92i9IUq8dnLty5nGMaX38iVZ5wlgfQhuRwSdj63eiO7ezn2AUeYkDRYMSK3FO-b1Ufeet9RXKY9pvqsSfrUfzUyxWm7aUFLRqZjgrNw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - docx file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">258</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000344244900001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">2</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsu012"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="408276"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000344244900001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f21%2f5%2f459%2f408276">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000344244900001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (2)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F21%2F5%2F459%2F408276" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/408276'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transpositionduplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Evolutionary Changes in Vertebrate Genome Signatures with Special Focus on Coelacanth" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsu012");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f21%2f5%2f459%2f408276">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

