

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/14/5/207/351235","@type":"ScholarlyArticle","name":"A Large-scale Protein–protein Interaction Analysis in  Synechocystis  sp. PCC6803","datePublished":"2007-11-13","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/14/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2007-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsm021","keywords":["<italic>Synechocystis</italic> sp. strain PCC6803","protein–protein interactions","yeast two-hybrid system"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/14/5/207/351235","author":[{"name":"Sato, Shusei","affiliation":"Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Shimoda, Yoshikazu","affiliation":"Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Muraki, Akiko","affiliation":"Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kohara, Mitsuyo","affiliation":"Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Nakamura, Yasukazu","affiliation":"Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Tabata, Satoshi","affiliation":"Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"}],"description":"Abstract.  Protein–protein interactions (PPIs) play crucial roles in protein function for a variety of biological processes. Data from large-scale PPI screening","pageStart":"207","pageEnd":"216","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/5/1/m_cover.gif?Expires=2147483647&Signature=CiCBmlfLQhYr8Ppz1~EJgCGKsrf3rW9AbUbRBYaOZWAZvVVl9QF3HNtURbdC8ru4eUIF8tkvjuKwsWFjWRPmFNcZkuQdKkpX0KM9t5HNmqBbuzPO~bvHSPph~gq~VZYcc1BLelBTam6u-PocW8JG97aio20zzby~i9d0CqJlAzp0UX9zeF-GNmIQehmYG3~yAiwwYnqWwPa~qusME0QvBaq9V1FXjnxPuJ7qjVWvv8qb9I3UUMrT~8kslFRqa1bzDmA1GsY77h9aLm07bY9C4TaNVHl7iQGCRJh6j9xQVQA0enfZ4FCv6EBAxkAOzBdR9agEdxAXvTjVLRF2dnQczA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"A Large-scale Protein–protein Interaction Analysis in  Synechocystis  sp. PCC6803","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/5/1/m_cover.gif?Expires=2147483647&Signature=CiCBmlfLQhYr8Ppz1~EJgCGKsrf3rW9AbUbRBYaOZWAZvVVl9QF3HNtURbdC8ru4eUIF8tkvjuKwsWFjWRPmFNcZkuQdKkpX0KM9t5HNmqBbuzPO~bvHSPph~gq~VZYcc1BLelBTam6u-PocW8JG97aio20zzby~i9d0CqJlAzp0UX9zeF-GNmIQehmYG3~yAiwwYnqWwPa~qusME0QvBaq9V1FXjnxPuJ7qjVWvv8qb9I3UUMrT~8kslFRqa1bzDmA1GsY77h9aLm07bY9C4TaNVHl7iQGCRJh6j9xQVQA0enfZ4FCv6EBAxkAOzBdR9agEdxAXvTjVLRF2dnQczA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Large-scale Protein–protein Interaction Analysis in  Synechocystis  sp. PCC6803" />
    <meta property="og:description" content="Abstract.  Protein–protein interactions (PPIs) play crucial roles in protein function for a variety of biological processes. Data from large-scale PPI screening" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/14/5/207/351235" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2007-11-13" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Sato, Shusei" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Shimoda, Yoshikazu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Muraki, Akiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kohara, Mitsuyo" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Nakamura, Yasukazu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Tabata, Satoshi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_title" content="A Large-scale Protein–protein Interaction Analysis in Synechocystis sp. PCC6803" /><meta name="citation_firstpage" content="207" /><meta name="citation_lastpage" content="216" /><meta name="citation_doi" content="10.1093/dnares/dsm021" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="14" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2007/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Sequence analysis of the genome of the unicellular cyanobacterium Synechocystis sp. strain PCC6803. II. Sequence determination of the entire genome and assignment of potential protein-coding regions; DNA Res.;  citation_year=1996;  citation_volume=3;  citation_pages=109-136; " /><meta name="citation_reference" content="citation_title=Synechocystis sp. PCC 6803—a useful tool in the study of the genetics of cyanobacteria; Photosynth Res.;  citation_year=2001;  citation_volume=70;  citation_pages=73-83; " /><meta name="citation_reference" content="citation_title=DNA microarray analysis of cyanobacterial gene expression during acclimation to high light; Plant Cell;  citation_year=2001;  citation_volume=13;  citation_pages=793-806; " /><meta name="citation_reference" content="citation_title=Cyano2Dbase updated: linkage of 234 protein spots to corresponding genes through N-terminal microsequencing; Electrophoresis;  citation_year=1999;  citation_volume=20;  citation_pages=2160-2171; " /><meta name="citation_reference" content="citation_title=Extension of CyanoBase. CyanoMutants: repository of mutant information on Synechocystis sp. strain PCC6803; Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=66-68; " /><meta name="citation_reference" content="citation_title=CyanoBase, the genome database for Synechocystis sp. strain PCC6803: status for the year; citation_text=2000; Nucleic Acids Res.;  citation_year=2000;  citation_volume=28;  citation_pages=72" /><meta name="citation_reference" content="citation_title=A comprehensive analysis of protein-protein interactions in Saccharomyces cerevisiae; Nature;  citation_year=2000;  citation_volume=403;  citation_pages=623-627; " /><meta name="citation_reference" content="citation_title=A comprehensive two-hybrid analysis to explore the yeast protein interactome; Proc. Natl Acad. Sci. USA;  citation_year=2001;  citation_volume=98;  citation_pages=4569-4574; " /><meta name="citation_reference" content="citation_title=Protein interaction mapping in C. elegans using proteins involved in vulval development; Science;  citation_year=2000;  citation_volume=287;  citation_pages=116-122; " /><meta name="citation_reference" content="citation_title=The protein–protein interaction map of Helicobacter pylori; Nature;  citation_year=2001;  citation_volume=409;  citation_pages=211-215; " /><meta name="citation_reference" content="citation_title=A protein interaction map of Drosophila melanogaster; Science;  citation_year=2003;  citation_volume=302;  citation_pages=1727-1736; " /><meta name="citation_reference" content="citation_title=A protein interaction network of the malaria parasite Plasmodium falciparum; Nature;  citation_year=2005;  citation_volume=438;  citation_pages=103-107; " /><meta name="citation_reference" content="citation_title=A proteome-wide protein interaction map for Campylobacter jejuni; Genome Biol.;  citation_year=2007;  citation_volume=8;  citation_pages=R130" /><meta name="citation_reference" content="citation_title=Compilation of all genes encoding bacterial two-component signal transducers in the genome of the cyanobacterium, Synechocystis sp. strain PCC 6803; DNA Res.;  citation_year=1996;  citation_volume=3;  citation_pages=407-414; " /><meta name="citation_reference" content="citation_title=Exploitation of genomic sequences in a systematic analysis to access how cyanobacteria sense environmental stress; J. Exp. Bot.;  citation_year=2006;  citation_volume=57;  citation_pages=235-247; " /><meta name="citation_reference" content="citation_title=How many genes in Arabidopsis come from cyanobacteria? An estimate from 386 protein phylogenies; Trends Genet.;  citation_year=2000;  citation_volume=17;  citation_pages=113-120; " /><meta name="citation_reference" content="citation_title=Evolutionary analysis of Arabidopsis, cyanobacterial, and chloroplast genomes reveals plastid phylogeny and thousands of cyanobacterial genes in the nucleus; Proc. Natl Acad. Sci. USA;  citation_year=2002;  citation_volume=99;  citation_pages=12246-12251; " /><meta name="citation_reference" content="citation_title=Toward a functional analysis of the yeast genome through exhaustive two-hybrid screens; Nat. Genet.;  citation_year=1997;  citation_volume=16;  citation_pages=277-282; " /><meta name="citation_reference" content="citation_title=Transformation of intact yeast cells treated with alkali cations; J. Bacteriol.;  citation_year=1983;  citation_volume=153;  citation_pages=163-168; " /><meta name="citation_reference" content="citation_title=Systematic and large-scale two-hybrid screens; Curr. Opin. Microbiol.;  citation_year=2000;  citation_volume=3;  citation_pages=303-308; " /><meta name="citation_reference" content="citation_title=Interaction generality, a measurement to assess the reliability of a protein-protein interaction; Nucleic Acids Res.;  citation_year=2002;  citation_volume=30;  citation_pages=1163-1168; " /><meta name="citation_reference" content="citation_title=High-throughput two-hybrid analysis; citation_text=The promise and the peril; FEBS J.;  citation_year=2005;  citation_volume=272;  citation_pages=5391-5399; " /><meta name="citation_reference" content="citation_title=Protein–protein interaction panel using mouse full-length cDNAs; Genome Res.;  citation_year=2007;  citation_volume=11;  citation_pages=1758-1765; " /><meta name="citation_reference" content="citation_title=InterDom: a database of putative interacting protein domains for validating predicted protein interactions and complexes; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=251-254; " /><meta name="citation_reference" content="citation_title=Large-scale identification of protein-protein interaction of Escherichia coli K-12; Genome Res.;  citation_volume=16;  citation_pages=686-691; " /><meta name="citation_reference" content="citation_title=Cytoscape: a software environment for integrated models of biomolecular interaction networks; Genome Res.;  citation_volume=13;  citation_pages=2498-2504; " /><meta name="citation_reference" content="citation_title=The identification of functional modules from the genomic association of genes; Proc. Natl Acad. Sci.;  citation_year=2002;  citation_volume=99;  citation_pages=5890-5895; " /><meta name="citation_reference" content="citation_title=The use of gene clusters to infer functional coupling; Proc. Natl Acad. Sci.;  citation_year=1999;  citation_volume=96;  citation_pages=2896-2901; " /><meta name="citation_reference" content="citation_title=The extracytoplasmic function (ECF) sigma factors; Adv. Microb. Physiol.;  citation_year=2002;  citation_volume=46;  citation_pages=47-110; " /><meta name="citation_reference" content="citation_title=Binding properties and evolution of homodimers in protein–protein interaction networks; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=3629-3635; " /><meta name="citation_reference" content="citation_title=Protein histidine kinases and signal transduction in prokaryotes and eukaryotes; Trends Genet.;  citation_year=1994;  citation_volume=10;  citation_pages=133-138; " /><meta name="citation_reference" content="citation_title=Communication modules in bacterial signaling proteins; Annu. Rev. Genet.;  citation_year=1992;  citation_volume=26;  citation_pages=71-112; " /><meta name="citation_reference" content="citation_title=Is cross regulation by phosphorylation of two-component response regulator proteins important in bacteria; J. Bacteriol.;  citation_year=1992;  citation_volume=174;  citation_pages=2053-2058; " /><meta name="citation_reference" content="citation_title=The pathway for perception and transduction of low-temperature signals in Synechocystis; EMBO J.;  citation_year=2000;  citation_volume=19;  citation_pages=1327-1334; " /><meta name="citation_reference" content="citation_title=Five histidine kinases perceive osmotic stress and regulate distinct sets of genes in Synechocystis; J. Biol. Chem.;  citation_year=2004;  citation_volume=279;  citation_pages=53078-53086; " /><meta name="citation_reference" content="citation_title=Identical Hik-Rre systems are involved in perception and transduction of salt signals and hyperosmotic signals but regulate the expression of individual genes to different extents in Synechocystis; J. Biol. Chem.;  citation_year=2005;  citation_volume=280;  citation_pages=21531-21538; " /><meta name="citation_reference" content="citation_title=Plant acetyl-CoA carboxylase: Structure, biosynthesis, regulation, and gene manipulation for plant breeding; Biosci. Biotechnol. Biochem.;  citation_year=2004;  citation_volume=68;  citation_pages=1175-1184; " /><meta name="citation_reference" content="citation_title=Proteomic identification of all plastid-specific ribosomal proteins in higher plant chloroplast 30S ribosomal subunit; Eur. J. Biochem.;  citation_year=2003;  citation_volume=270;  citation_pages=190-205; " /><meta name="citation_reference" content="citation_title=Identification of PamA as a PII-binding membrane protein important in nitrogen-related and sugar-catabolic gene expression in Synechocystis sp. PCC 6803; J. Biol. Chem.;  citation_year=2005;  citation_volume=280;  citation_pages=34684-34690; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/14/5/207/17462675/dsm021.pdf" /><meta name="description" content="Abstract.  Protein–protein interactions (PPIs) play crucial roles in protein function for a variety of biological processes. Data from large-scale PPI screening" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/14/5/207/351235" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/14/5/207/351235" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/14/5/207/351235");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsm021';
    </script>
<title>
	Large-scale Protein–protein Interaction Analysis in Synechocystis sp. PCC6803 | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./351235" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BahaPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYXRvLCBTaHVzZWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLUthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2hpbW9kYSwgWW9zaGlrYXp1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1LYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik11cmFraSwgQWtpa28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLUthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS29oYXJhLCBNaXRzdXlvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1LYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik5ha2FtdXJhLCBZYXN1a2F6dSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2EtS2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYWJhdGEsIFNhdG9zaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLUthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJBIExhcmdlLXNjYWxlIFByb3RlaW7igJNwcm90ZWluIEludGVyYWN0aW9uIEFuYWx5c2lzIGluIFN5bmVjaG9jeXN0aXMgc3AuIFBDQzY4MDMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIyMDciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjIxNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzbTAyMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDA3LzAxLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VxdWVuY2UgYW5hbHlzaXMgb2YgdGhlIGdlbm9tZSBvZiB0aGUgdW5pY2VsbHVsYXIgY3lhbm9iYWN0ZXJpdW0gU3luZWNob2N5c3RpcyBzcC4gc3RyYWluIFBDQzY4MDMuIElJLiBTZXF1ZW5jZSBkZXRlcm1pbmF0aW9uIG9mIHRoZSBlbnRpcmUgZ2Vub21lIGFuZCBhc3NpZ25tZW50IG9mIHBvdGVudGlhbCBwcm90ZWluLWNvZGluZyByZWdpb25zOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MTA5LTEzNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3luZWNob2N5c3RpcyBzcC4gUENDIDY4MDPigJRhIHVzZWZ1bCB0b29sIGluIHRoZSBzdHVkeSBvZiB0aGUgZ2VuZXRpY3Mgb2YgY3lhbm9iYWN0ZXJpYTsgUGhvdG9zeW50aCBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTcwOyAgY2l0YXRpb25fcGFnZXM9NzMtODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUROQSBtaWNyb2FycmF5IGFuYWx5c2lzIG9mIGN5YW5vYmFjdGVyaWFsIGdlbmUgZXhwcmVzc2lvbiBkdXJpbmcgYWNjbGltYXRpb24gdG8gaGlnaCBsaWdodDsgUGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTc5My04MDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUN5YW5vMkRiYXNlIHVwZGF0ZWQ6IGxpbmthZ2Ugb2YgMjM0IHByb3RlaW4gc3BvdHMgdG8gY29ycmVzcG9uZGluZyBnZW5lcyB0aHJvdWdoIE4tdGVybWluYWwgbWljcm9zZXF1ZW5jaW5nOyBFbGVjdHJvcGhvcmVzaXM7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0yMTYwLTIxNzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4dGVuc2lvbiBvZiBDeWFub0Jhc2UuIEN5YW5vTXV0YW50czogcmVwb3NpdG9yeSBvZiBtdXRhbnQgaW5mb3JtYXRpb24gb24gU3luZWNob2N5c3RpcyBzcC4gc3RyYWluIFBDQzY4MDM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTY2LTY4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DeWFub0Jhc2UsIHRoZSBnZW5vbWUgZGF0YWJhc2UgZm9yIFN5bmVjaG9jeXN0aXMgc3AuIHN0cmFpbiBQQ0M2ODAzOiBzdGF0dXMgZm9yIHRoZSB5ZWFyOyBjaXRhdGlvbl90ZXh0PTIwMDA7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTcyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb21wcmVoZW5zaXZlIGFuYWx5c2lzIG9mIHByb3RlaW4tcHJvdGVpbiBpbnRlcmFjdGlvbnMgaW4gU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9NDAzOyAgY2l0YXRpb25fcGFnZXM9NjIzLTYyNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb21wcmVoZW5zaXZlIHR3by1oeWJyaWQgYW5hbHlzaXMgdG8gZXhwbG9yZSB0aGUgeWVhc3QgcHJvdGVpbiBpbnRlcmFjdG9tZTsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT05ODsgIGNpdGF0aW9uX3BhZ2VzPTQ1NjktNDU3NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvdGVpbiBpbnRlcmFjdGlvbiBtYXBwaW5nIGluIEMuIGVsZWdhbnMgdXNpbmcgcHJvdGVpbnMgaW52b2x2ZWQgaW4gdnVsdmFsIGRldmVsb3BtZW50OyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTI4NzsgIGNpdGF0aW9uX3BhZ2VzPTExNi0xMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBwcm90ZWlu4oCTcHJvdGVpbiBpbnRlcmFjdGlvbiBtYXAgb2YgSGVsaWNvYmFjdGVyIHB5bG9yaTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTQwOTsgIGNpdGF0aW9uX3BhZ2VzPTIxMS0yMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcHJvdGVpbiBpbnRlcmFjdGlvbiBtYXAgb2YgRHJvc29waGlsYSBtZWxhbm9nYXN0ZXI7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzAyOyAgY2l0YXRpb25fcGFnZXM9MTcyNy0xNzM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHByb3RlaW4gaW50ZXJhY3Rpb24gbmV0d29yayBvZiB0aGUgbWFsYXJpYSBwYXJhc2l0ZSBQbGFzbW9kaXVtIGZhbGNpcGFydW07IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00Mzg7ICBjaXRhdGlvbl9wYWdlcz0xMDMtMTA3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHByb3Rlb21lLXdpZGUgcHJvdGVpbiBpbnRlcmFjdGlvbiBtYXAgZm9yIENhbXB5bG9iYWN0ZXIgamVqdW5pOyBHZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPVIxMzAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21waWxhdGlvbiBvZiBhbGwgZ2VuZXMgZW5jb2RpbmcgYmFjdGVyaWFsIHR3by1jb21wb25lbnQgc2lnbmFsIHRyYW5zZHVjZXJzIGluIHRoZSBnZW5vbWUgb2YgdGhlIGN5YW5vYmFjdGVyaXVtLCBTeW5lY2hvY3lzdGlzIHNwLiBzdHJhaW4gUENDIDY4MDM7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz00MDctNDE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHBsb2l0YXRpb24gb2YgZ2Vub21pYyBzZXF1ZW5jZXMgaW4gYSBzeXN0ZW1hdGljIGFuYWx5c2lzIHRvIGFjY2VzcyBob3cgY3lhbm9iYWN0ZXJpYSBzZW5zZSBlbnZpcm9ubWVudGFsIHN0cmVzczsgSi4gRXhwLiBCb3QuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTU3OyAgY2l0YXRpb25fcGFnZXM9MjM1LTI0NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SG93IG1hbnkgZ2VuZXMgaW4gQXJhYmlkb3BzaXMgY29tZSBmcm9tIGN5YW5vYmFjdGVyaWE/IEFuIGVzdGltYXRlIGZyb20gMzg2IHByb3RlaW4gcGh5bG9nZW5pZXM7IFRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0xMTMtMTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb25hcnkgYW5hbHlzaXMgb2YgQXJhYmlkb3BzaXMsIGN5YW5vYmFjdGVyaWFsLCBhbmQgY2hsb3JvcGxhc3QgZ2Vub21lcyByZXZlYWxzIHBsYXN0aWQgcGh5bG9nZW55IGFuZCB0aG91c2FuZHMgb2YgY3lhbm9iYWN0ZXJpYWwgZ2VuZXMgaW4gdGhlIG51Y2xldXM7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9OTk7ICBjaXRhdGlvbl9wYWdlcz0xMjI0Ni0xMjI1MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VG93YXJkIGEgZnVuY3Rpb25hbCBhbmFseXNpcyBvZiB0aGUgeWVhc3QgZ2Vub21lIHRocm91Z2ggZXhoYXVzdGl2ZSB0d28taHlicmlkIHNjcmVlbnM7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9Mjc3LTI4MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNmb3JtYXRpb24gb2YgaW50YWN0IHllYXN0IGNlbGxzIHRyZWF0ZWQgd2l0aCBhbGthbGkgY2F0aW9uczsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk4MzsgIGNpdGF0aW9uX3ZvbHVtZT0xNTM7ICBjaXRhdGlvbl9wYWdlcz0xNjMtMTY4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeXN0ZW1hdGljIGFuZCBsYXJnZS1zY2FsZSB0d28taHlicmlkIHNjcmVlbnM7IEN1cnIuIE9waW4uIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTMwMy0zMDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVyYWN0aW9uIGdlbmVyYWxpdHksIGEgbWVhc3VyZW1lbnQgdG8gYXNzZXNzIHRoZSByZWxpYWJpbGl0eSBvZiBhIHByb3RlaW4tcHJvdGVpbiBpbnRlcmFjdGlvbjsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MTE2My0xMTY4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoLXRocm91Z2hwdXQgdHdvLWh5YnJpZCBhbmFseXNpczsgY2l0YXRpb25fdGV4dD1UaGUgcHJvbWlzZSBhbmQgdGhlIHBlcmlsOyBGRUJTIEouOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTI3MjsgIGNpdGF0aW9uX3BhZ2VzPTUzOTEtNTM5OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvdGVpbuKAk3Byb3RlaW4gaW50ZXJhY3Rpb24gcGFuZWwgdXNpbmcgbW91c2UgZnVsbC1sZW5ndGggY0ROQXM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MTc1OC0xNzY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlckRvbTogYSBkYXRhYmFzZSBvZiBwdXRhdGl2ZSBpbnRlcmFjdGluZyBwcm90ZWluIGRvbWFpbnMgZm9yIHZhbGlkYXRpbmcgcHJlZGljdGVkIHByb3RlaW4gaW50ZXJhY3Rpb25zIGFuZCBjb21wbGV4ZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTI1MS0yNTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxhcmdlLXNjYWxlIGlkZW50aWZpY2F0aW9uIG9mIHByb3RlaW4tcHJvdGVpbiBpbnRlcmFjdGlvbiBvZiBFc2NoZXJpY2hpYSBjb2xpIEstMTI7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9Njg2LTY5MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q3l0b3NjYXBlOiBhIHNvZnR3YXJlIGVudmlyb25tZW50IGZvciBpbnRlZ3JhdGVkIG1vZGVscyBvZiBiaW9tb2xlY3VsYXIgaW50ZXJhY3Rpb24gbmV0d29ya3M7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MjQ5OC0yNTA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgaWRlbnRpZmljYXRpb24gb2YgZnVuY3Rpb25hbCBtb2R1bGVzIGZyb20gdGhlIGdlbm9taWMgYXNzb2NpYXRpb24gb2YgZ2VuZXM7IFByb2MuIE5hdGwgQWNhZC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT05OTsgIGNpdGF0aW9uX3BhZ2VzPTU4OTAtNTg5NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHVzZSBvZiBnZW5lIGNsdXN0ZXJzIHRvIGluZmVyIGZ1bmN0aW9uYWwgY291cGxpbmc7IFByb2MuIE5hdGwgQWNhZC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT05NjsgIGNpdGF0aW9uX3BhZ2VzPTI4OTYtMjkwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGV4dHJhY3l0b3BsYXNtaWMgZnVuY3Rpb24gKEVDRikgc2lnbWEgZmFjdG9yczsgQWR2LiBNaWNyb2IuIFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTQ2OyAgY2l0YXRpb25fcGFnZXM9NDctMTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CaW5kaW5nIHByb3BlcnRpZXMgYW5kIGV2b2x1dGlvbiBvZiBob21vZGltZXJzIGluIHByb3RlaW7igJNwcm90ZWluIGludGVyYWN0aW9uIG5ldHdvcmtzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz0zNjI5LTM2MzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb3RlaW4gaGlzdGlkaW5lIGtpbmFzZXMgYW5kIHNpZ25hbCB0cmFuc2R1Y3Rpb24gaW4gcHJva2FyeW90ZXMgYW5kIGV1a2FyeW90ZXM7IFRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0xMzMtMTM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21tdW5pY2F0aW9uIG1vZHVsZXMgaW4gYmFjdGVyaWFsIHNpZ25hbGluZyBwcm90ZWluczsgQW5udS4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz03MS0xMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlzIGNyb3NzIHJlZ3VsYXRpb24gYnkgcGhvc3Bob3J5bGF0aW9uIG9mIHR3by1jb21wb25lbnQgcmVzcG9uc2UgcmVndWxhdG9yIHByb3RlaW5zIGltcG9ydGFudCBpbiBiYWN0ZXJpYTsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT0xNzQ7ICBjaXRhdGlvbl9wYWdlcz0yMDUzLTIwNTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBwYXRod2F5IGZvciBwZXJjZXB0aW9uIGFuZCB0cmFuc2R1Y3Rpb24gb2YgbG93LXRlbXBlcmF0dXJlIHNpZ25hbHMgaW4gU3luZWNob2N5c3RpczsgRU1CTyBKLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTEzMjctMTMzNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Rml2ZSBoaXN0aWRpbmUga2luYXNlcyBwZXJjZWl2ZSBvc21vdGljIHN0cmVzcyBhbmQgcmVndWxhdGUgZGlzdGluY3Qgc2V0cyBvZiBnZW5lcyBpbiBTeW5lY2hvY3lzdGlzOyBKLiBCaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yNzk7ICBjaXRhdGlvbl9wYWdlcz01MzA3OC01MzA4NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpY2FsIEhpay1ScmUgc3lzdGVtcyBhcmUgaW52b2x2ZWQgaW4gcGVyY2VwdGlvbiBhbmQgdHJhbnNkdWN0aW9uIG9mIHNhbHQgc2lnbmFscyBhbmQgaHlwZXJvc21vdGljIHNpZ25hbHMgYnV0IHJlZ3VsYXRlIHRoZSBleHByZXNzaW9uIG9mIGluZGl2aWR1YWwgZ2VuZXMgdG8gZGlmZmVyZW50IGV4dGVudHMgaW4gU3luZWNob2N5c3RpczsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjgwOyAgY2l0YXRpb25fcGFnZXM9MjE1MzEtMjE1Mzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBsYW50IGFjZXR5bC1Db0EgY2FyYm94eWxhc2U6IFN0cnVjdHVyZSwgYmlvc3ludGhlc2lzLCByZWd1bGF0aW9uLCBhbmQgZ2VuZSBtYW5pcHVsYXRpb24gZm9yIHBsYW50IGJyZWVkaW5nOyBCaW9zY2kuIEJpb3RlY2hub2wuIEJpb2NoZW0uOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTY4OyAgY2l0YXRpb25fcGFnZXM9MTE3NS0xMTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm90ZW9taWMgaWRlbnRpZmljYXRpb24gb2YgYWxsIHBsYXN0aWQtc3BlY2lmaWMgcmlib3NvbWFsIHByb3RlaW5zIGluIGhpZ2hlciBwbGFudCBjaGxvcm9wbGFzdCAzMFMgcmlib3NvbWFsIHN1YnVuaXQ7IEV1ci4gSi4gQmlvY2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MjcwOyAgY2l0YXRpb25fcGFnZXM9MTkwLTIwNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgUGFtQSBhcyBhIFBJSS1iaW5kaW5nIG1lbWJyYW5lIHByb3RlaW4gaW1wb3J0YW50IGluIG5pdHJvZ2VuLXJlbGF0ZWQgYW5kIHN1Z2FyLWNhdGFib2xpYyBnZW5lIGV4cHJlc3Npb24gaW4gU3luZWNob2N5c3RpcyBzcC4gUENDIDY4MDM7IEouIEJpb2wuIENoZW0uOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTI4MDsgIGNpdGF0aW9uX3BhZ2VzPTM0Njg0LTM0NjkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTQvNS8yMDcvMTc0NjI2NzUvZHNtMDIxLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBQcm90ZWlu4oCTcHJvdGVpbiBpbnRlcmFjdGlvbnMgKFBQSXMpIHBsYXkgY3J1Y2lhbCByb2xlcyBpbiBwcm90ZWluIGZ1bmN0aW9uIGZvciBhIHZhcmlldHkgb2YgYmlvbG9naWNhbCBwcm9jZXNzZXMuIERhdGEgZnJvbSBsYXJnZS1zY2FsZSBQUEkgc2NyZWVuaW5nIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zNTEyMzUmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkF9HJRRiFomQx12nh7cbGTlaM+hE=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f5%2f207%2f351235" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f5%2f207%2f351235" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13580" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/14/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/5/1/m_cover.gif?Expires=2147483647&amp;Signature=CiCBmlfLQhYr8Ppz1~EJgCGKsrf3rW9AbUbRBYaOZWAZvVVl9QF3HNtURbdC8ru4eUIF8tkvjuKwsWFjWRPmFNcZkuQdKkpX0KM9t5HNmqBbuzPO~bvHSPph~gq~VZYcc1BLelBTam6u-PocW8JG97aio20zzby~i9d0CqJlAzp0UX9zeF-GNmIQehmYG3~yAiwwYnqWwPa~qusME0QvBaq9V1FXjnxPuJ7qjVWvv8qb9I3UUMrT~8kslFRqa1bzDmA1GsY77h9aLm07bY9C4TaNVHl7iQGCRJh6j9xQVQA0enfZ4FCv6EBAxkAOzBdR9agEdxAXvTjVLRF2dnQczA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/14/5">
                        <div class="volume trailing-comma">Volume 14</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
2007        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87195641">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195641">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195643">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195643">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195650">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195650">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195661">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195661">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195696">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195696">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195698">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195698">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195701">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195701">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/14/5/201/350973">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/14/5/217/351025">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">A Large-scale Protein–protein Interaction Analysis in <em>Synechocystis</em> sp. PCC6803
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shusei Sato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81-438-52-3935. Fax. <div class="fax">+81-438-52-3934</div>. E-mail: <a href="mailto:ssato@kazusa.or.jp" target="_blank">ssato@kazusa.or.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shusei+Sato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sato S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sato S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shusei Sato</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshikazu Shimoda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshikazu+Shimoda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shimoda Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shimoda Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yoshikazu Shimoda</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akiko Muraki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akiko+Muraki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Muraki A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Muraki A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Akiko Muraki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mitsuyo Kohara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mitsuyo+Kohara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kohara M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kohara M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mitsuyo Kohara</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yasukazu Nakamura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yasukazu+Nakamura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakamura Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakamura Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yasukazu Nakamura</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoshi Tabata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoshi+Tabata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tabata S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tabata S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Satoshi Tabata</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Takashi Ito</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 14, Issue 5, 1 January 2007, Pages 207–216, <a href='https://doi.org/10.1093/dnares/dsm021'>https://doi.org/10.1093/dnares/dsm021</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">13 November 2007</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">05 September 2007</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">09 October 2007</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="351235" href="/dnaresearch/article-pdf/14/5/207/17462675/dsm021.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Shusei Sato, Yoshikazu Shimoda, Akiko Muraki, Mitsuyo Kohara, Yasukazu Nakamura, Satoshi Tabata;  A Large-scale Protein–protein Interaction Analysis in <em>Synechocystis</em> sp. PCC6803, <em>DNA Research</em>, Volume 14, Issue 5, 1 January 2007, Pages 207–216, <a href='https://doi.org/10.1093/dnares/dsm021'>https://doi.org/10.1093/dnares/dsm021</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/351235?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/351235?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/351235?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/351235?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/351235?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/351235?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/351235?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=A%20Large-scale%20Protein%E2%80%93protein%20Interaction%20Analysis%20in%20%26lt%3Bem%26gt%3BSynechocystis%26lt%3B%2Fem%26gt%3B%20sp.%20PCC6803&amp;publicationDate=2007-11-13&amp;volumeNum=14&amp;issueNum=5&amp;author=Sato%2C%20Shusei%3B%20Shimoda%2C%20Yoshikazu&amp;startPage=207&amp;endPage=216&amp;contentId=10.1093%2Fdnares%2Fdsm021&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f5%2f207%2f351235" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87195641 id="87195641" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Protein–protein interactions (PPIs) play crucial roles in protein function for a variety of biological processes. Data from large-scale PPI screening has contributed to understanding the function of a large number of predicted genes from fully sequenced genomes. Here, we report the systematic identification of protein interactions for the unicellular cyanobacterium <em>Synechocystis</em> sp. strain PCC6803. Using a modified high-throughput yeast two-hybrid assay, we screened 1825 genes selected primarily from (i) genes of two-component signal transducers of <em>Synechocystis</em>, (ii) <em>Synechocystis</em> genes whose homologues are conserved in the genome of <em>Arabidopsis thaliana</em>, and (iii) genes of unknown function on the <em>Synechocystis</em> chromosome. A total of 3236 independent two-hybrid interactions involving 1920 proteins (52% of the total protein coding genes) were identified and each interaction was evaluated using an interaction generality (IG) measure, as well as the general features of interacting partners. The interaction data obtained in this study should provide new insights and novel strategies for functional analyses of genes in <em>Synechocystis</em>, and, additionally, genes in other cyanobacteria and plant genes of cyanobacterial origin.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Synechocystis sp. strain PCC6803&quot;"><em>Synechocystis</em> sp. strain PCC6803</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;protein–protein interactions&quot;">protein–protein interactions</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;yeast two-hybrid system&quot;">yeast two-hybrid system</a></div>                            <h2 scrollto-destination=87195643 id="87195643" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The unicellular cyanobacterium <em>Synechocystis</em> sp. strain PCC6803 was the first phototrophic organism to be fully sequenced.<sup><a reveal-id="DSM021C1" data-open="DSM021C1" class="link link-ref link-reveal xref-bibr">1</a></sup> The availability of the complete genome sequence of <em>Synechocystis</em> has drastically changed the strategy for studying genetic systems in cyanobacteria.<sup><a reveal-id="DSM021C2" data-open="DSM021C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Gene identification and functional assignment have been accelerated utilizing the genome sequence. The genome sequence has also allowed systematic analyses of gene regulation and function on the genomic level. Microarray and proteome analyses have made it possible to monitor the expression of a substantial proportion of genes at both transcriptional and translational levels.<sup><a reveal-id="DSM021C3" data-open="DSM021C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSM021C4" data-open="DSM021C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Genetic analyses, such as targeted disruption or random tagging analysis, have suggested many functional links between gene products.<sup><a reveal-id="DSM021C2" data-open="DSM021C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Two genome databases, CyanoBase and CyanoMutants, have been established and act as centralized information resources.<sup><a reveal-id="DSM021C5" data-open="DSM021C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSM021C6" data-open="DSM021C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Based on these material and information resources, <em>Synechocystis</em> provides an ideal model system for genetic studies of photosynthetic organisms.</p><p>Although a large quantity of data has been accumulated through functional analyses of the genome, many of these analyses were driven by the predicted functions of the annotated genes. Therefore, a limited amount of information is available for unannotated genes. One of the effective approaches to obtain information regarding the function of these uncharacterized proteins is an analysis of protein–protein interactions (PPIs). Since nearly half of the predicted gene products of <em>Synechocystis</em> remain unannotated, a systematic approach for analysis of PPIs is needed.</p><p>We conducted a large-scale analysis of <em>Synechocystis</em> PPIs using a yeast two-hybrid (YTH) system. YTH analysis is one of the well-established methods to detect binary protein interactions and is feasible for large-scale analysis. Large-scale YTH screens have been conducted in a wide range of organisms and have provided several important biological and bioinformatics platforms for the study of protein networks in different organisms.<sup><a reveal-id="DSM021C7  DSM021C13" data-open="DSM021C7  DSM021C13" class="link link-ref link-reveal xref-bibr">7–13</a></sup> These analyses have also successfully placed functionally uncharacterized proteins in their biological context.</p><p>In this study, we applied the YTH approach to three major target gene groups. For the initial screening, we selected genes of two-component signal transducers in order to evaluate our YTH screening system, as well as to elucidate the two-component signal transduction pathways in <em>Synechocystis.</em> The two-component signal transduction system generally consists of a histidine kinase (Hik) and a response regulator (Rre), and signal transduction is achieved by phosphorylation as a result of the interaction between cognate pairs of Hiks and Rres. Since the interactions of Hiks and Rres are binary, these two-component signal transducers are suitable targets for evaluation using the YTH screening system. On the <em>Synechocystis</em> chromosome, 44 putative genes for Hiks and 42 genes for Rres are predicted.<sup><a reveal-id="DSM021C1" data-open="DSM021C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSM021C14" data-open="DSM021C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSM021C15" data-open="DSM021C15" class="link link-ref link-reveal xref-bibr">15</a></sup> In contrast to the <em>Escherichia coli</em> and <em>Bacillus subtilis</em> genomes, in which most of the genes for cognate pairs of Hiks and Rres are located close to each other, many of the genes for Hiks and Rres in <em>Synechocystis</em> are distributed randomly in the chromosome. Among 44 genes for Hiks on the <em>Synechocystis</em> chromosome, 14 are located in the vicinity of genes for potential cognate Rres, whereas other 30 are not located near any genes for Rres.<sup><a reveal-id="DSM021C15" data-open="DSM021C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Therefore, identification of PPIs between two-component signal transducers will provide valuable information to elucidate the cognate pairs of Hiks and Rres and signal transduction pathways in <em>Synechocystis</em>.</p><p>For the second screening group, we selected <em>Synechocystis</em> genes whose homologues are conserved in the <em>A. thaliana</em> genome with the aim to obtain interaction information that is applicable to plant genes of cyanobacterial origin as well as <em>Synechocystis</em> genes. Chloroplasts are descendants of free-living cyanobacteria that became endsymbionts. In the course of the evolutionary processes that transformed the cyanobacterial symbiont into an organelle, chloroplasts have donated many genes to nuclear chromosomes.<sup><a reveal-id="DSM021C16" data-open="DSM021C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Martin et al.<sup><a reveal-id="DSM021C17" data-open="DSM021C17" class="link link-ref link-reveal xref-bibr">17</a></sup> reported that approximately 1700 of 9369 <em>A. thaliana</em> genes that were investigated are of cyanobacterial origin and these encompass all functional categories. PPI information on <em>Synechocystis</em> genes conserved in plants would provide useful insights for functional analysis of plant genes of cyanobacterial origin, since <em>Synechocystis</em> has been used as a model for analysis of these plant genes.</p><p>For the third screening group, we selected genes of unknown function in order to obtain interaction data that could be used to predict function. Since approximately 60% of the <em>Synechocystis</em> genes of unknown function have putative orthologues in at least one of the cyanobacteria whose genome sequence is available, the data obtained from a screen of <em>Synechocystis</em> genes will be applicable to other cyanobacteria.</p>                            <h2 scrollto-destination=87195650 id="87195650" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87195651 id="87195651" class="section-title" data-legacy-id=s2a>2.1. Construction of bait clones</h3>
<p>The yeast genetic techniques and media used in this study have been described in previous reports.<sup><a reveal-id="DSM021C8" data-open="DSM021C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSM021C18" data-open="DSM021C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p><p>For the GAL4 DNA-binding domain-fusion (bait) vector, we constructed pAS2-1-<em>Asc</em> I by inserting an <em>Asc</em> I site between the <em>Nco</em> I and <em>Eco</em> RI sites of pAS2-1 (Clontech, Mountain View, CA, USA). Specific primer sets were designed for the target region of each <em>Synechocystis</em> gene, and an <em>Asc</em> I site was added to the 5′ end of each gene-specific forward primer. Target gene fragments were obtained by PCR amplification from cosmid clones or genomic DNA using the gene specific primer sets. To minimize potential misincorporation during PCR, the high-fidelity <em>pfu</em> DNA polymerase (Stratagene, La Jolla, CA, USA) was used. The resulting fragments were subjected to <em>Asc</em> I digestion and ligated into pAS2-1-<em>Asc</em> I predigested with <em>Asc</em> I and <em>Sma</em> I. The target genes cloned into pAS2-1-<em>Asc</em>I (bait clone) were transformed into the yeast strains CG1945 (<em>MATa, ura3-52, his3-200, ade2-101, lys2-801, trp1-901, leu2-3, 112, gal4-542, gal80-538, cyh<sup>r</sup>2, LYS2::GAL1</em><sub>UAS</sub><em>-GAL1</em><sub>TATA</sub><em>-HIS3, URA3::GAL4</em><sub>17-mers(x3)</sub><em>-CYC1</em><sub>TATA</sub><em>-lacZ</em>) or AH109 (<em>MATa, trp1-901, leu2-3, 112, ura3-52, his3-200, gal4Δ, gal80Δ, LYS2::GAL1</em><sub>UAS</sub><em>-GAL1</em><sub>TATA</sub><em>-HIS3, MEL1, GAL2</em><sub>UAS</sub><em>-GAL2</em><sub>TATA</sub><em>-ADE2, URA3::MEL1</em><sub>UAS</sub><em>-MEL1</em><sub>TATA</sub><em>-lacZ</em>) (Clontech, Mountain View, CA, USA).</p>                            <h3 scrollto-destination=87195654 id="87195654" class="section-title" data-legacy-id=s2b>2.2. Library construction</h3>
<p>The genomic DNA of <em>Synechocystis</em> sp. strain PCC6803 was subjected to sonication followed by size-fractionation by agarose gel electrophoresis. The 0.5−2.0 kb fraction was cloned into the <em>Sma</em> I digested activation domain-fusion (prey) vector, pACT2 (Clontech, Mountain View, CA, USA), and transformed into <em>E. coli</em> (XLI-Blue; Stratagene, La Jolla, CA, USA). The library DNA isolated from 10<sup>6</sup> independent clones was then introduced into the yeast strain Y187 (<em>MATα, ura3-52, his3-200, ade2-101, trp1-901, leu2-3, 112, gal4Δ, met</em><sup>−</sup><em>, gal80Δ, URA3::GAL1</em><sub>UAS</sub><em>-GAL1</em><sub>TATA</sub><em>-lacZ, MEL1</em>) using standard lithium acetate-mediated procedures.<sup><a reveal-id="DSM021C19" data-open="DSM021C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Three million independent yeast colonies were collected, pooled, and stored at −80°C.</p>                            <h3 scrollto-destination=87195656 id="87195656" class="section-title" data-legacy-id=s2c>2.3. Two-hybrid screening procedure</h3>
<p>Prior to screening, the self-activity of each bait clone was tested by mating with the Y187 strain harboring an empty pACT2 vector and then plating on SD/-His/-Leu/-Trp/ medium supplemented with 2.5, 5, or 10 mM 3-amino 1,2,4-triazole (3-AT). Each bait clone was then mated with the prey library containing approximately 1.6 × 10<sup>7</sup> clones and plated on SD/-His/-Leu/-Trp/ agar medium supplemented with the optimal concentration of 3-AT based on the results of the self-activity test. After 7 days of growth at 30°C, independent positive colonies were collected in 96-well plates and cultured for an additional 3 days at 30°C. Aliquots of cultured positive clones were used for β-galactosidase assays and DNA sequencing of the prey inserts, and the cultures were stored at −80°C. The collected positive clones were treated with a zymolyase solution [2.5 mg/mL Zymolyase-100T (Seikagaku America Inc., East Falmouth, MA, USA), 1.2 M sorbitol, 0.1 M Na phosphate, pH 7.4] for 30 min at 30°C, and then used as templates for direct amplification of the prey clone inserts using the following primers: T0017 (5′-TACCACTACAATGGATGATG-3′) and T0018 (5′- GGGGTTTTTCAGTATCTACG-3′). Amplified inserts were directly sequenced using T0017 (5′) and T0018 (3′) to obtain sequence tags. The sequence tags were subsequently subjected to a BLAST search against the sequence database in CyanoBase (<a class="link link-uri" href="http://www.kazusa.or.jp/cyanobase/">http://www.kazusa.or.jp/cyanobase/</a>) using a semi-automated data processing system. A web-based interface was created to display the search results and to support the submission of interaction data into a interaction database. Sequence tags corresponding to antisense or intergenic regions were discarded. Sequence tags corresponding to the sense strand of an ORF were selected as PPI candidates. The PPI candidates were selected not only from in-frame prey clones (phase 2 position) but also out-of-frame fusion positions for the following reasons. PPIs supported by both in-frame and out-of-frame fusion prey clones have occasionally been found, and translational frameshifting has been reported and biochemically confirmed in previous YTH analyses.<sup><a reveal-id="DSM021C18" data-open="DSM021C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSM021C20" data-open="DSM021C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Similarly, the prey sequence tags corresponding to the up-stream region of a gene (within 200 nucleotides) were selected as PPI candidates that may be translated from an upstream start site.</p>                            <h3 scrollto-destination=87195658 id="87195658" class="section-title" data-legacy-id=s2d>2.4. Paralogue search and calculation of IG</h3>
<p>Information on genes paralogous to <em>Synechocystis</em> genes was obtained from the CYORF database [<a class="link link-uri" href="http://cyano.genome.ad.jp/">http://cyano.genome.ad.jp/</a>]. The lower threshold of acceptability was set at a Smith–Waterman score of 200.</p><p>Calculation of IG values for each interaction was conducted using the methods described in the previous report.<sup><a reveal-id="DSM021C21" data-open="DSM021C21" class="link link-ref link-reveal xref-bibr">21</a></sup></p>                            <h2 scrollto-destination=87195661 id="87195661" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=87195662 id="87195662" class="section-title" data-legacy-id=s3a>3.1. Construction of a high throughput YTH system and selection of Synechocystis genes for screening</h3>
<p>One of the labor intensive steps of YTH analysis is designing the bait clone construction to establish directional and in-frame cloning. To overcome this point, we introduced an <em>Asc</em> I site into the multiple cloning site of a bait vector and used a simplified procedure for bait clone construction, as described in the Materials and Methods. This procedure can be applied to almost all genes in the <em>Synechocystis</em> genome, since there are only five <em>Asc</em> I sites in coding regions.<sup><a reveal-id="DSM021C1" data-open="DSM021C1" class="link link-ref link-reveal xref-bibr">1</a></sup></p><p>To increase the efficiency of screening, we used a yeast mating-based screen, as described in the Material and Methods, and used a GAL4 activation domain fusion library of random genomic fragments of <em>Synechocystis</em> sp strain PCC6803. By using the random genomic fragment library as prey clones, information on PPI pairs and interacting domains of the prey proteins can be obtained simultaneously. The selected positive clones were analyzed using a 96-well plate format to increase throughput. For data processing of the large number of sequence tags from positive clones, we developed a semi-automatic system of two programs for vector sequence trimming and blast search against the <em>Synechocystis</em> sequence database. We also created an interface to display the search results and to support submission of interaction data to an interaction database.</p><p>Using this high-throughput YTH system, we explored large-scale PPI analysis in the photoautotorophic cyanobacterium <em>Synechocystis</em> sp. PCC6803. For bait clones of YTH screening, we selected the following groups of <em>Synechocystis</em> genes: (i) genes of putative two-component signal transducers, (ii) genes whose homologues are conserved in the <em>A. thaliana</em> genome, and (iii) genes of unknown function.</p><p>For two-component signal transducers, 44 predicted genes for Hiks and 42 predicted genes for Rres were selected.<sup><a reveal-id="DSM021C14" data-open="DSM021C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSM021C15" data-open="DSM021C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Selection of <em>Synechocystis</em> genes with homologues in the <em>A. thaliana</em> genome was conducted as follows. First, all <em>A. thaliana</em> genes were compared with those of <em>Synechocystis</em>, <em>E. coli</em>, yeast, and <em>Caenorhabditis elegans</em>, and the <em>A. thaliana</em> genes with highest degree of sequence similarity to those of <em>Synechocystis</em> were identified. Then by assigning <em>Synechocystis</em> genes corresponding to the identified <em>A. thaliana</em> genes, a total of 657 genes were selected as candidate <em>Synechocystis</em> genes conserved in the <em>A. thaliana</em> genome. The selected genes accounted for 10–50% of each functional category in <em>Synechocystis</em>. In the <em>Synechocystis</em> chromosome, 1641 genes (49%) are still categorized as hypothetical or of unknown function. Among the genes of unknown function, 190 candidate genes were selected because homologues were present in the <em>A. thaliana</em> genome. In addition, 413 genes encoding proteins with multiple transmembrane domains were excluded, because of the difficulty in detecting PPI of transmembrane proteins using the YTH system. Thus, a total of 1038 genes of unknown function were selected as the third screening group. For approximately 70% of these selected function unknown genes, putative orthologues are present in at least one of the completely sequenced cyanobacteria. Altogether a total of 1825 genes, including 44 genes selected for other purposes, were selected as target genes in this study (Fig. <a reveal-id="DSM021F1" data-open="DSM021F1" class="link link-reveal link-table xref-fig">1</a>, Table <a reveal-id="DSM021TB1" data-open="DSM021TB1" class="link link-reveal link-table xref-fig">1</a>, Supplementary Table 1).</p>                        <a id="87195667" scrollto-destination="87195667"></a>
<div id="DSM021F1" data-id="DSM021F1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm021/1/m_dsm02101.jpeg?Expires=2147483647&Signature=4u78DenrMGgvIGcC~5dnK3ZQ7U-tsWsi29vmDgvJMB7crJ2~ORib0hZC7pAM8YN1zN8jtRcXHNkFh1vfhzhrytBOezkzs5HEYEuq4YjVqjvaVYtoKjb0zR8UD~LMdmvRdblslj4ezb6vjHOCW3vkhiQqX2lj8ie-nnUe~lv6HDpU4TR90TaQxasPYDkhJRVoIRZmeUck~4Ukt-VV4ECVaqBggTabvlAF8uruh31g551gRXQa-~jYjcEMOKX4~BQUakeTa2q1pRCVeIcbnIQOHZ-o0PG9s9szzo1zuNwZDOD4Bu0nZx9WsjB67wjInJOUYqP5jJgwVsj2RUU8YN6c7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Flow chart of the sequential steps used for identification of PPIs of Synechocystis. The details are given in the Materials and Methods. BD, GAL4 DNA-binding domain; AD, GAL4 activation domain." path-from-xml="dsm02101.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195667/dsm02101.jpeg" path-from-xml="dsm02101.jpeg" target="_blank">View large</a><a section="87195667" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm021/1/dsm02101.jpeg?Expires=2147483647&Signature=WWSXxfdTXEzvGnVIys62b~1aAgfVjSZ53C9zq8NN6bVFsvkaZAth05XEPYPoxxOWCwOCORZQwr9EdgPtWNSSZ2zGsSOymkBosN2Si9qY1mid1RnaeZJeNoqc3wlQPcVD8AZn4OXy4l7UDDZlV-DqVpNGihgVzjP9wTY1K0gW9bPUwxq2-EPXxl3bDhXLIyPeUPL79XkSJ7LCNB1kBux5tyh8flQrsUd63HfX4EEU7CBF1HBL3nYlnTobp-rcpty3TA4wGAswrjJa6yGr7k1z4sYJ3Ct1l3pqqkTOPzblfPmOU4XzkfuBdTASRW~PaNBxSxRImfgDcYV8v9Hc2NAW1w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195667&ar=351235&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm02101.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Flow chart of the sequential steps used for identification of PPIs of <em>Synechocystis</em>. The details are given in the Materials and Methods. BD, GAL4 DNA-binding domain; AD, GAL4 activation domain.</p></div></div><div content-id="DSM021F1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm021/1/m_dsm02101.jpeg?Expires=2147483647&Signature=4u78DenrMGgvIGcC~5dnK3ZQ7U-tsWsi29vmDgvJMB7crJ2~ORib0hZC7pAM8YN1zN8jtRcXHNkFh1vfhzhrytBOezkzs5HEYEuq4YjVqjvaVYtoKjb0zR8UD~LMdmvRdblslj4ezb6vjHOCW3vkhiQqX2lj8ie-nnUe~lv6HDpU4TR90TaQxasPYDkhJRVoIRZmeUck~4Ukt-VV4ECVaqBggTabvlAF8uruh31g551gRXQa-~jYjcEMOKX4~BQUakeTa2q1pRCVeIcbnIQOHZ-o0PG9s9szzo1zuNwZDOD4Bu0nZx9WsjB67wjInJOUYqP5jJgwVsj2RUU8YN6c7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Flow chart of the sequential steps used for identification of PPIs of Synechocystis. The details are given in the Materials and Methods. BD, GAL4 DNA-binding domain; AD, GAL4 activation domain." path-from-xml="dsm02101.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195667/dsm02101.jpeg" path-from-xml="dsm02101.jpeg" target="_blank">View large</a><a section="87195667" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm021/1/dsm02101.jpeg?Expires=2147483647&Signature=WWSXxfdTXEzvGnVIys62b~1aAgfVjSZ53C9zq8NN6bVFsvkaZAth05XEPYPoxxOWCwOCORZQwr9EdgPtWNSSZ2zGsSOymkBosN2Si9qY1mid1RnaeZJeNoqc3wlQPcVD8AZn4OXy4l7UDDZlV-DqVpNGihgVzjP9wTY1K0gW9bPUwxq2-EPXxl3bDhXLIyPeUPL79XkSJ7LCNB1kBux5tyh8flQrsUd63HfX4EEU7CBF1HBL3nYlnTobp-rcpty3TA4wGAswrjJa6yGr7k1z4sYJ3Ct1l3pqqkTOPzblfPmOU4XzkfuBdTASRW~PaNBxSxRImfgDcYV8v9Hc2NAW1w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195667&ar=351235&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm02101.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Flow chart of the sequential steps used for identification of PPIs of <em>Synechocystis</em>. The details are given in the Materials and Methods. BD, GAL4 DNA-binding domain; AD, GAL4 activation domain.</p></div></div>                        <a id="87195668" scrollto-destination="87195668"></a>
<div content-id="DSM021TB1" class="table-modal"><div class="table-wrap"><div id="DSM021TB1" class="table-wrap-title" data-id="DSM021TB1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p><em>Synechocystis</em> genes used for YTH analysis as baits</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Description </th><th>Number </th><th>Screened </th><th>Positive </th><th>No positive </th><th>Self-active </th></tr></thead><tbody><tr><td>Genes of two component signal transducer </td><td>86 </td><td>86 </td><td>53 </td><td>20 </td><td>13 </td></tr><tr><td>Genes conserved in the <em>A. thaliana</em> genome </td><td>657 </td><td>655 </td><td>380 </td><td>177 </td><td>98 </td></tr><tr><td>Genes of unknown function </td><td>1038 </td><td>1029 </td><td>571 </td><td>207 </td><td>251 </td></tr><tr><td>Others </td><td>44 </td><td>44 </td><td>33 </td><td>8 </td><td>3 </td></tr><tr><td>Total </td><td>1825 </td><td>1814 </td><td>1037 </td><td>412 </td><td>365 </td></tr></tbody></table></div><div content-id="DSM021TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Description </th><th>Number </th><th>Screened </th><th>Positive </th><th>No positive </th><th>Self-active </th></tr></thead><tbody><tr><td>Genes of two component signal transducer </td><td>86 </td><td>86 </td><td>53 </td><td>20 </td><td>13 </td></tr><tr><td>Genes conserved in the <em>A. thaliana</em> genome </td><td>657 </td><td>655 </td><td>380 </td><td>177 </td><td>98 </td></tr><tr><td>Genes of unknown function </td><td>1038 </td><td>1029 </td><td>571 </td><td>207 </td><td>251 </td></tr><tr><td>Others </td><td>44 </td><td>44 </td><td>33 </td><td>8 </td><td>3 </td></tr><tr><td>Total </td><td>1825 </td><td>1814 </td><td>1037 </td><td>412 </td><td>365 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195668" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM021TB1" class="table-wrap-title" data-id="DSM021TB1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p><em>Synechocystis</em> genes used for YTH analysis as baits</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Description </th><th>Number </th><th>Screened </th><th>Positive </th><th>No positive </th><th>Self-active </th></tr></thead><tbody><tr><td>Genes of two component signal transducer </td><td>86 </td><td>86 </td><td>53 </td><td>20 </td><td>13 </td></tr><tr><td>Genes conserved in the <em>A. thaliana</em> genome </td><td>657 </td><td>655 </td><td>380 </td><td>177 </td><td>98 </td></tr><tr><td>Genes of unknown function </td><td>1038 </td><td>1029 </td><td>571 </td><td>207 </td><td>251 </td></tr><tr><td>Others </td><td>44 </td><td>44 </td><td>33 </td><td>8 </td><td>3 </td></tr><tr><td>Total </td><td>1825 </td><td>1814 </td><td>1037 </td><td>412 </td><td>365 </td></tr></tbody></table></div><div content-id="DSM021TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Description </th><th>Number </th><th>Screened </th><th>Positive </th><th>No positive </th><th>Self-active </th></tr></thead><tbody><tr><td>Genes of two component signal transducer </td><td>86 </td><td>86 </td><td>53 </td><td>20 </td><td>13 </td></tr><tr><td>Genes conserved in the <em>A. thaliana</em> genome </td><td>657 </td><td>655 </td><td>380 </td><td>177 </td><td>98 </td></tr><tr><td>Genes of unknown function </td><td>1038 </td><td>1029 </td><td>571 </td><td>207 </td><td>251 </td></tr><tr><td>Others </td><td>44 </td><td>44 </td><td>33 </td><td>8 </td><td>3 </td></tr><tr><td>Total </td><td>1825 </td><td>1814 </td><td>1037 </td><td>412 </td><td>365 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195668" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=87195669 id="87195669" class="section-title" data-legacy-id=s3b>3.2. Classification and assessment of the protein interaction data</h3>
<p>Of the 1825 target genes, bait clones were successfully constructed for 1814 (Fig. <a reveal-id="DSM021F1" data-open="DSM021F1" class="link link-reveal link-table xref-fig">1</a>, Table <a reveal-id="DSM021TB1" data-open="DSM021TB1" class="link link-reveal link-table xref-fig">1</a>, Supplementary Table 1). The YTH screen detected 3236 independent PPIs from 57% (1037) of the tested bait clones. The PPI network that was obtained contains 1920 <em>Synechocystis</em> proteins (Fig. <a reveal-id="DSM021F1" data-open="DSM021F1" class="link link-reveal link-table xref-fig">1</a>, Supplementary Table 2).</p><p>One of the major concerns of large-scale interaction analysis is its reliability. Generally, PPI data sets obtained from YTH analysis, as well as other comprehensive analyses, contain many false positives. Therefore, the detected interactions need to be evaluated. In the case of YTH analysis, promiscuous interaction and self-activation by bait clones are two of the major causes of false positives.<sup><a reveal-id="DSM021C20  DSM021C23" data-open="DSM021C20  DSM021C23" class="link link-ref link-reveal xref-bibr">20–23</a></sup> In a large-scale YTH analysis in yeast, Ito et al.<sup><a reveal-id="DSM021C8" data-open="DSM021C8" class="link link-ref link-reveal xref-bibr">8</a></sup> classified the reliability of the PPI data by the number of interaction sequence tags (ISTs), the pair of tagged sequences, obtained from positive clones. They designated PPI data with more than three IST hits as the core data, and used this data set for construction of an interaction map and data comparison.<sup><a reveal-id="DSM021C8" data-open="DSM021C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Similar to this method, we classified all detected interactions into four categories (Category A–D), as an indicator of data reliability, based on how many positive prey clones supported the interaction. Category A and Category B comprised interactions supported by multiple positive prey clones with different (A) or identical (B) inserts. Category C comprised interactions supported by a single positive clone. PPI data supported by prey clones which were identified to have interactions with more than 18 different bait clones, i.e. 1% of tested bait clones, were considered to be promiscuous interactions and grouped in Category D. Most of the PPIs classified in Category D are likely to be false positives. Although there was no clear common feature among the 15 proteins classified as promiscuous prey, four of them had multiple trans-membrane domains and two contained a protein domain previously shown to interact promiscuously (Supplementary Table 3).<sup><a reveal-id="DSM021C24" data-open="DSM021C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSM021C25" data-open="DSM021C25" class="link link-ref link-reveal xref-bibr">25</a></sup> The proportion of genes in each category is shown in Table <a reveal-id="DSM021TB2" data-open="DSM021TB2" class="link link-reveal link-table xref-fig">2</a>.</p>                        <a id="87195672" scrollto-destination="87195672"></a>
<div content-id="DSM021TB2" class="table-modal"><div class="table-wrap"><div id="DSM021TB2" class="table-wrap-title" data-id="DSM021TB2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Classification of interaction category</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Description </th><th>Total interactions </th><th>Proportion (%) </th></tr></thead><tbody><tr><td>Category A (supported by multiple positive clones with different inserts ) </td><td>795 </td><td>25 </td></tr><tr><td>Category B (supported by multiple positive clones with same inserts) </td><td>269 </td><td>8 </td></tr><tr><td>Category C (supported by a single positive clone) </td><td>1768 </td><td>55 </td></tr><tr><td>Category D (interaction of putative promiscuous prey clones) </td><td>404 </td><td>12 </td></tr></tbody></table></div><div content-id="DSM021TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Description </th><th>Total interactions </th><th>Proportion (%) </th></tr></thead><tbody><tr><td>Category A (supported by multiple positive clones with different inserts ) </td><td>795 </td><td>25 </td></tr><tr><td>Category B (supported by multiple positive clones with same inserts) </td><td>269 </td><td>8 </td></tr><tr><td>Category C (supported by a single positive clone) </td><td>1768 </td><td>55 </td></tr><tr><td>Category D (interaction of putative promiscuous prey clones) </td><td>404 </td><td>12 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195672" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM021TB2" class="table-wrap-title" data-id="DSM021TB2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Classification of interaction category</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Description </th><th>Total interactions </th><th>Proportion (%) </th></tr></thead><tbody><tr><td>Category A (supported by multiple positive clones with different inserts ) </td><td>795 </td><td>25 </td></tr><tr><td>Category B (supported by multiple positive clones with same inserts) </td><td>269 </td><td>8 </td></tr><tr><td>Category C (supported by a single positive clone) </td><td>1768 </td><td>55 </td></tr><tr><td>Category D (interaction of putative promiscuous prey clones) </td><td>404 </td><td>12 </td></tr></tbody></table></div><div content-id="DSM021TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Description </th><th>Total interactions </th><th>Proportion (%) </th></tr></thead><tbody><tr><td>Category A (supported by multiple positive clones with different inserts ) </td><td>795 </td><td>25 </td></tr><tr><td>Category B (supported by multiple positive clones with same inserts) </td><td>269 </td><td>8 </td></tr><tr><td>Category C (supported by a single positive clone) </td><td>1768 </td><td>55 </td></tr><tr><td>Category D (interaction of putative promiscuous prey clones) </td><td>404 </td><td>12 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195672" target="_blank">View Large</a></div></div><p>To assess the validity of our categorizations, we evaluated all the detected interactions using an IG measure, a method for computationally assessing the reliability of PPI.<sup><a reveal-id="DSM021C21" data-open="DSM021C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Interactions with lower IG values are more likely to be reliable than interactions with higher IG values. The IG values for all detected interactions ranged from 1 to 17 and the average IG value was 2.4. When we extracted interactions having an IG value &lt;3, approximately 90% of interactions from Categories A, B, and C were included, whereas only 44% of of interactions from Category D were included (Supplementary Fig. <a reveal-id="DSM021F1" data-open="DSM021F1" class="link link-reveal link-table xref-fig">1</a>). The average IG values of Categories A (2.00 ± 0.16), B (1.87 ± 0.08), and C (2.01 ± 0.33) were significantly lower (<em>P</em> &lt; 0.01) than that of Category D (5.42 ± 0.08) when fifty independent interactions were selected randomly from each category and compared. This result suggests that the interaction Categories A–D appropriately indicate the reliability of the PPI data.</p><p>To minimize false positives caused by self-activation, we used multiple reporter genes driven by different GAL4-responsive promoters and carefully determined the level of self-activation of each bait clone (see Materials and Methods). As shown in Supplementary Table 4, a total of 363 bait clones displayed strong self-activation as they could not be suppressed by leaky <em>HIS3</em> reporter gene expression even in the presence of 10 mM 3-AT and these clones were eliminated from the screen. To examine the protein domains responsible for self-activation, we examined the protein domains of the genes that showed self-activation. Bacterial regulatory protein domains, such as the response regulator receiver (IPR001789) and TPR-1 (IPR001440), were frequently identified among the annotated proteins (Supplementary Table 4). In addition, several domains of unknown function, such as DUF29 (IPR002636) and UPF0150 (IPR005357), were occasionally observed among the proteins of unknown function that showed self-activation.</p><p>Another feature of large-scale YTH analyses is the high frequency of false negatives or missed interactions. The precise proportion of false-negative interactions can be assessed by comparison with published data. However, this approach is difficult to apply to <em>Synechocystis</em> PPI analysis because of the limited data available regarding protein interactions in <em>Synechocystis</em>. As an alternative, we estimated the proportion of false-negatives based on the interactions of two-component signal transducers whose interaction can be expected by the location of the corresponding genes in the genome. Among 14 Hiks located in the vicinity of genes for potentially cognate Rres in the <em>Synechocystis</em> genome, seven pairs were detected in our screening, indicating that about 50% of the interactions may not have been detected in our analysis. Considering that the two-component signal transducers were extensively analyzed during this study, the false-negative rate among other bait clones could be higher.</p>                            <h3 scrollto-destination=87195676 id="87195676" class="section-title" data-legacy-id=s3c>3.3. General features of protein interactions</h3>
<p>A global view of the protein interaction network is illustrated in Fig. <a reveal-id="DSM021F2" data-open="DSM021F2" class="link link-reveal link-table xref-fig">2</a>A using Cytoscape 2.2 (<a class="link link-uri" href="http://www.cytoscape.org/">http://www.cytoscape.org/</a>).<sup><a reveal-id="DSM021C26" data-open="DSM021C26" class="link link-ref link-reveal xref-bibr">26</a></sup> In Fig. <a reveal-id="DSM021F2" data-open="DSM021F2" class="link link-reveal link-table xref-fig">2</a>A, protein disks and interaction lines are color-coded according to functional categories and interaction categories, respectively. The 1920 proteins involved in this interaction network includes proteins from all functional categories in <em>Synechocystis</em> (Fig. <a reveal-id="DSM021F2" data-open="DSM021F2" class="link link-reveal link-table xref-fig">2</a>B).</p>                        <a id="87195678" scrollto-destination="87195678"></a>
<div id="DSM021F2" data-id="DSM021F2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm021/1/m_dsm02102.jpeg?Expires=2147483647&Signature=iR1SRKK15SqPOBvOma8d0KMxhJhi8Q8tUnlLRP50xvwc75PQ7gUEND03Tdv6SvkgsCkjnrvsjXPvFTCdcmcvi89inNjqlpSbRHoCYgNu4rtIqU9g1Mswo9ay30gTloK4Ft-v9ibddoxeVvFAJ7Qzt6R0sjS0m2z~cHj2J2aJ62QIHCs6i7bMRReE2~D17ntoyFy1byglHemyPPOhE0ArJ0eM6wzQsB71QHS07GGhY2ZZ~b-fBkdLMWzV9KfS180R77ZmjZHQNFRDK-t7YJ15ECmrVuP-XVwFhao4YgJYs73hdYE0j~cmusTs9C7PIKRP8jl7bkYAkEtAyKc2CUDnFQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Global view of the PPIs of Synechocystis. (A) All detected PPIs. Proteins (circles) are color-coded according to their functional category, as assigned in Kaneko et al.1 Interactions (lines) are color-coded by interaction category (A–D), which is based on the frequency of detection of identical pairs, as described in the text. (B) Number of identified interactions in each function category. The white bar indicates the total number of genes from each functional category assigned in the Synechocystis genome and the black bar indicates number of genes shown to have interactions. The red bar indicates the number of genes used as bait in the screen. The percentages represent the proportion of interacting genes in each function category." path-from-xml="dsm02102.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195678/dsm02102.jpeg" path-from-xml="dsm02102.jpeg" target="_blank">View large</a><a section="87195678" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm021/1/dsm02102.jpeg?Expires=2147483647&Signature=dRAAvd356f6ZJ2v1FvnERyS993LTxXpjIn-r0UaPQ8otFlEj0cmsqBds83ZFl~ByB9soXmYHcLdc3-4o8o5pyl86ytZOZsDohnAvpaNsXqzODQ1Hg7AQbmMBVNJARR3CEoZei-fqmTDQj2a06vuHnzRoGYbo2o0VvC31RIky8AgIuYEPHHHtc-HmiwvcrPC2K6z~mMgyIXwqQLmi~-~spDB7HjX2n5ulHJF2qBfjI8xRD5PeLRl34trJKXfwyHYPRJl8bR9bKd-M3OuqqzJ0oRPEXXM1H9JpepZ2NgiMQKp3zn3zlU4fFfi8jYfeOLXDSNa5psAWML47M5pgmgu6Qw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195678&ar=351235&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm02102.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Global view of the PPIs of <em>Synechocystis</em>. (<strong>A</strong>) All detected PPIs. Proteins (circles) are color-coded according to their functional category, as assigned in Kaneko et al.<sup>1</sup> Interactions (lines) are color-coded by interaction category (<strong>A</strong>–<strong>D</strong>), which is based on the frequency of detection of identical pairs, as described in the text. (<strong>B</strong>) Number of identified interactions in each function category. The white bar indicates the total number of genes from each functional category assigned in the <em>Synechocystis</em> genome and the black bar indicates number of genes shown to have interactions. The red bar indicates the number of genes used as bait in the screen. The percentages represent the proportion of interacting genes in each function category.</p></div></div><div content-id="DSM021F2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm021/1/m_dsm02102.jpeg?Expires=2147483647&Signature=iR1SRKK15SqPOBvOma8d0KMxhJhi8Q8tUnlLRP50xvwc75PQ7gUEND03Tdv6SvkgsCkjnrvsjXPvFTCdcmcvi89inNjqlpSbRHoCYgNu4rtIqU9g1Mswo9ay30gTloK4Ft-v9ibddoxeVvFAJ7Qzt6R0sjS0m2z~cHj2J2aJ62QIHCs6i7bMRReE2~D17ntoyFy1byglHemyPPOhE0ArJ0eM6wzQsB71QHS07GGhY2ZZ~b-fBkdLMWzV9KfS180R77ZmjZHQNFRDK-t7YJ15ECmrVuP-XVwFhao4YgJYs73hdYE0j~cmusTs9C7PIKRP8jl7bkYAkEtAyKc2CUDnFQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Global view of the PPIs of Synechocystis. (A) All detected PPIs. Proteins (circles) are color-coded according to their functional category, as assigned in Kaneko et al.1 Interactions (lines) are color-coded by interaction category (A–D), which is based on the frequency of detection of identical pairs, as described in the text. (B) Number of identified interactions in each function category. The white bar indicates the total number of genes from each functional category assigned in the Synechocystis genome and the black bar indicates number of genes shown to have interactions. The red bar indicates the number of genes used as bait in the screen. The percentages represent the proportion of interacting genes in each function category." path-from-xml="dsm02102.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195678/dsm02102.jpeg" path-from-xml="dsm02102.jpeg" target="_blank">View large</a><a section="87195678" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm021/1/dsm02102.jpeg?Expires=2147483647&Signature=dRAAvd356f6ZJ2v1FvnERyS993LTxXpjIn-r0UaPQ8otFlEj0cmsqBds83ZFl~ByB9soXmYHcLdc3-4o8o5pyl86ytZOZsDohnAvpaNsXqzODQ1Hg7AQbmMBVNJARR3CEoZei-fqmTDQj2a06vuHnzRoGYbo2o0VvC31RIky8AgIuYEPHHHtc-HmiwvcrPC2K6z~mMgyIXwqQLmi~-~spDB7HjX2n5ulHJF2qBfjI8xRD5PeLRl34trJKXfwyHYPRJl8bR9bKd-M3OuqqzJ0oRPEXXM1H9JpepZ2NgiMQKp3zn3zlU4fFfi8jYfeOLXDSNa5psAWML47M5pgmgu6Qw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195678&ar=351235&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm02102.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Global view of the PPIs of <em>Synechocystis</em>. (<strong>A</strong>) All detected PPIs. Proteins (circles) are color-coded according to their functional category, as assigned in Kaneko et al.<sup>1</sup> Interactions (lines) are color-coded by interaction category (<strong>A</strong>–<strong>D</strong>), which is based on the frequency of detection of identical pairs, as described in the text. (<strong>B</strong>) Number of identified interactions in each function category. The white bar indicates the total number of genes from each functional category assigned in the <em>Synechocystis</em> genome and the black bar indicates number of genes shown to have interactions. The red bar indicates the number of genes used as bait in the screen. The percentages represent the proportion of interacting genes in each function category.</p></div></div><p>Prokaryotic genes are generally organized into operons in which a gene cluster is transcribed as a polycistronic mRNA. Several previous works have demonstrated that genes encoded in the same operon are likely to have related functions.<sup><a reveal-id="DSM021C27" data-open="DSM021C27" class="link link-ref link-reveal xref-bibr">27</a>,<a reveal-id="DSM021C28" data-open="DSM021C28" class="link link-ref link-reveal xref-bibr">28</a></sup> In the PPI data obtained here, we found 44 interacting protein pairs that are encoded by genes mapped to adjacent loci in the <em>Synechocystis</em> genome (Supplementary Table 5). It is noteworthy that 31 of these 44 PPIs were classified in Category A and the frequency of Category A PPIs occurring between proteins encoded by adjacent genes is significantly higher than that of the entire network (Table <a reveal-id="DSM021TB2" data-open="DSM021TB2" class="link link-reveal link-table xref-fig">2</a>). This result supports the validity of our evaluation of the protein interaction data by interaction category. These interactions contained six interacting pairs between annotated proteins and proteins of unknown function mapping to adjacent loci. Two of these interactions were between extracytoplasmic function (ECF) sigma factors and their immediate downstream gene products, SigI (Sll0687)–Sll0688 and SigG (Slr1545)–Slr1546. ECF sigma factor is usually transcribed by one or more negative regulators (anti-sigma factors) which bind and inhibit the cognate sigma factor.<sup><a reveal-id="DSM021C29" data-open="DSM021C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Therefore, these PPI data suggest that the hypothetical proteins Sll0688 and Slr1546 may function as anti-sigma factors.</p><p>We detected 110 self-interacting proteins (Supplementary Table 6). Of these, 34 were proteins with functional annotations, and some are known to execute their function as homo-multimers or multi-protein complexes containing homo-dimers, such as the 10 kD chaperonin (Slr2075) and nitrogen regulatory protein P-II (Ssl0707). However, more than half of the self-interacting proteins detected in this screen were categorized as hypothetical or unknown. This information on the self-interaction of uncharacterized proteins will further our understanding of the function of these proteins. Domain level analysis of the uncharacterized self-interacting proteins revealed that five of the proteins have domains that are known to be involved in PPIs, such as ribbon–helix–helix (IPR10985) and the HxlR type helix–turn–helix, (IPR002577). Furthermore, a protein domain of unknown function, DUF820 (IPR008538), was found in nine of the self-interacting proteins and, thus, we hypothesize that this domain is involved in PPIs.</p><p>We also identified 15 hetero-dimeric interactions that occurred between two paralogous proteins (Smith–Waterman score &gt;200). Among these interactions, seven were between proteins of unknown function, and these proteins also tended to be self-interacting. For instance, Slr1152, which showed self-interaction, interacted with a paralogous protein Sll1446. Also among the proteins containing the DUF820 domain (IPR008538) described above, the paralogous interactions Sll1250–Sll0296 and Sll1250–Sll1355 were detected in addition to the self-interactions of Sll0296 and Sll1355. These results suggest that a duplication of self-interacting proteins can generate a pair of paralogous proteins that interact with each other. Similar tendencies have been identified in PPI networks of yeast and other eukaryotes.<sup><a reveal-id="DSM021C30" data-open="DSM021C30" class="link link-ref link-reveal xref-bibr">30</a></sup></p>                            <h3 scrollto-destination=87195682 id="87195682" class="section-title" data-legacy-id=s3d>3.4. Interactions of two-component signal transducers</h3>
<p>A total of 105 bait clones corresponding to 86 genes of two-component signal transducers were constructed, since genes longer than 2.5 kb were divided into two or more constructs. After removal of 23 self-active bait clones, 82 bait clones were used for screening. A total of 195 PPIs were obtained from 54 bait clones. Among these PPI data, 44 interactions were between two component signal transducers, including seven pairs of bidirectional interactions and six self-interactions (Fig. <a reveal-id="DSM021F3" data-open="DSM021F3" class="link link-reveal link-table xref-fig">3</a>). Of these, we successfully identified seven putative cognate pairs of Hiks and Rres predicted by genome location. Among the 10 PPIs detected between these cognate pairs, six were classified in Category A, and three putative cognate pairs were supported by bidirectional PPIs (Fig. <a reveal-id="DSM021F3" data-open="DSM021F3" class="link link-reveal link-table xref-fig">3</a>). Detection of putative cognate pairs of Hiks and Rres as relatively reliable interactions indicates the validity of this screening approach and the specificity of our YTH analysis. Moreover, 21 of the identified PPIs between Hiks and Rres corresponded to 19 Hik and Rre pairs that were localized to different regions of the <em>Synechocystis</em> genome (Fig. <a reveal-id="DSM021F3" data-open="DSM021F3" class="link link-reveal link-table xref-fig">3</a>). Since it is difficult to know the specific functional Hik and Rre partners from their sequence, the interaction pairs obtained by this study provide evidence to reinforce the functional relationships of the putative cognate pairs of Hiks and Rres and enabled us to identify the functional partners of Hiks and Rres that were localized to different regions of the <em>Synechocystis</em> genome.</p>                        <a id="87195684" scrollto-destination="87195684"></a>
<div id="DSM021F3" data-id="DSM021F3" class="fig fig-section"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm021/1/m_dsm02103.jpeg?Expires=2147483647&Signature=RF2mw51tidLjI0RMe0IRB8TkkxhsrSpvJhMka3pTkmb9Zv8Ax0Yu6e9xk2TrXK~15iuOzhDe9M7cuYiEcMHj2wZOwSkfgoTRAKuh2851oDVmLIgbxgFVq1GVCJ2~hve2XDrh--9GxaN5ALa7up9E76bizMrhkNcSSOvtK9STw85vCbqKMhox7OJeJr-Se0bw0VoFidZKny4-xfrfrNbf~BYSsQ4K6i4U-cUqdbChgcqkHi07R0v7IH0wmiUjrA4Mecy2alQgB0aA9p8qA9w1mcwmALV1WRYj94BtoWmE0g7H4pwpsa2-i6wV3IDyaU1q-LTbUggtZZ9~E1smqLiJOQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Interaction pairs of two-component signal transducers. Sensory histidine kinases, hybrid sensory kinases, and response regulators are shown by blue, purple, and orange boxes, respectively. Boxes marked with a red line indicate the interaction between Hik and Rre encoded by putative cognate pairs. The arrow in each interaction indicates the direction of bait protein to prey protein and the reliability of each interaction." path-from-xml="dsm02103.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195684/dsm02103.jpeg" path-from-xml="dsm02103.jpeg" target="_blank">View large</a><a section="87195684" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm021/1/dsm02103.jpeg?Expires=2147483647&Signature=zEgjt1QiJJB4IpJGuu44UBVL8ReZLW6I1WyPxTnytsKjfe6jDecxFJd25t9spR5PJ5eAZOPLB1Q0rKKJuZXAcDKQyGIb9zc~AEDtitw4Sv0BPzxFVBW0JW3LWtxRW8tDwXdG0HfBEmtsStKDqSIFUhaskN1-0ThQJz5QYjVm~IJr~OVMNMz3LbdaJ5ee9Udc7gQHkSfICkzZhcOHKe8rwp9qLHeWWpN4CajJaTT3ylZ62eaza2oxf1KcW03UzY3OPzGr7dT9y2wIuzT5K9~MzrEaoDq7GSQ6-KWYgJu8K~fdpSy8f985tnzPhjcMIJrmPXVdOoEyHDXxLbc~j~OqLg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195684&ar=351235&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm02103.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Interaction pairs of two-component signal transducers. Sensory histidine kinases, hybrid sensory kinases, and response regulators are shown by blue, purple, and orange boxes, respectively. Boxes marked with a red line indicate the interaction between Hik and Rre encoded by putative cognate pairs. The arrow in each interaction indicates the direction of bait protein to prey protein and the reliability of each interaction.</p></div></div><div content-id="DSM021F3" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm021/1/m_dsm02103.jpeg?Expires=2147483647&Signature=RF2mw51tidLjI0RMe0IRB8TkkxhsrSpvJhMka3pTkmb9Zv8Ax0Yu6e9xk2TrXK~15iuOzhDe9M7cuYiEcMHj2wZOwSkfgoTRAKuh2851oDVmLIgbxgFVq1GVCJ2~hve2XDrh--9GxaN5ALa7up9E76bizMrhkNcSSOvtK9STw85vCbqKMhox7OJeJr-Se0bw0VoFidZKny4-xfrfrNbf~BYSsQ4K6i4U-cUqdbChgcqkHi07R0v7IH0wmiUjrA4Mecy2alQgB0aA9p8qA9w1mcwmALV1WRYj94BtoWmE0g7H4pwpsa2-i6wV3IDyaU1q-LTbUggtZZ9~E1smqLiJOQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Interaction pairs of two-component signal transducers. Sensory histidine kinases, hybrid sensory kinases, and response regulators are shown by blue, purple, and orange boxes, respectively. Boxes marked with a red line indicate the interaction between Hik and Rre encoded by putative cognate pairs. The arrow in each interaction indicates the direction of bait protein to prey protein and the reliability of each interaction." path-from-xml="dsm02103.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195684/dsm02103.jpeg" path-from-xml="dsm02103.jpeg" target="_blank">View large</a><a section="87195684" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm021/1/dsm02103.jpeg?Expires=2147483647&Signature=zEgjt1QiJJB4IpJGuu44UBVL8ReZLW6I1WyPxTnytsKjfe6jDecxFJd25t9spR5PJ5eAZOPLB1Q0rKKJuZXAcDKQyGIb9zc~AEDtitw4Sv0BPzxFVBW0JW3LWtxRW8tDwXdG0HfBEmtsStKDqSIFUhaskN1-0ThQJz5QYjVm~IJr~OVMNMz3LbdaJ5ee9Udc7gQHkSfICkzZhcOHKe8rwp9qLHeWWpN4CajJaTT3ylZ62eaza2oxf1KcW03UzY3OPzGr7dT9y2wIuzT5K9~MzrEaoDq7GSQ6-KWYgJu8K~fdpSy8f985tnzPhjcMIJrmPXVdOoEyHDXxLbc~j~OqLg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195684&ar=351235&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm02103.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Interaction pairs of two-component signal transducers. Sensory histidine kinases, hybrid sensory kinases, and response regulators are shown by blue, purple, and orange boxes, respectively. Boxes marked with a red line indicate the interaction between Hik and Rre encoded by putative cognate pairs. The arrow in each interaction indicates the direction of bait protein to prey protein and the reliability of each interaction.</p></div></div><p>In addition to the 31 interactions between Hiks and Rres, nine interactions within Hiks were detected. Among the interactions within Hiks, five PPIs were obtained between a sensor kinase and hybrid sensory kinases, which contain both transmitter and receiver domains and are involved in multistep phospho-relays.<sup><a reveal-id="DSM021C14" data-open="DSM021C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSM021C31" data-open="DSM021C31" class="link link-ref link-reveal xref-bibr">31</a></sup> As a result, candidates for a putative multistep phospho-relay were identified as Hik15 (Sll1353)–Hik19 (Sll1905)–Rre41 (Slr1305).</p><p>Interactions between two-component transducers were detected not only as simple one-to-one interactions between Hiks and Rres, but also as interactions between multiple Hiks and a single Rre, and vice versa. These interactions could reflect cross-regulation occurring between multiple two-component signal transducers. The possibility of cross-regulation in two-component signal transduction systems has been proposed previously.<sup><a reveal-id="DSM021C32" data-open="DSM021C32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="DSM021C33" data-open="DSM021C33" class="link link-ref link-reveal xref-bibr">33</a></sup> In <em>Synechocystis</em>, the presence of cross-regulation has been demonstrated experimentally. Detailed analysis of Hiks using a series of gene-knockout libraries revealed that Hik33 (Sll0698) regulates several different sets of genes by interacting with deferent Rres in response to low-temperature stress, hyperosmotic stress, and salt stress signals.<sup><a reveal-id="DSM021C15" data-open="DSM021C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSM021C34  DSM021C36" data-open="DSM021C34  DSM021C36" class="link link-ref link-reveal xref-bibr">34–36</a></sup> Although we failed to identify interactions between Hik33 (Sll0698) and Rres, the PPI data obtained for multiple Hiks and Rres will help to elucidate novel cross-regulation in two-component systems in <em>Synechocystis</em>.</p>                            <h3 scrollto-destination=87195687 id="87195687" class="section-title" data-legacy-id=s3e>3.5. Interactions of proteins encoded by genes conserved in the plant genome</h3>
<p>Among 657 genes selected as candidates that are conserved in the <em>A. thaliana</em> genome, a total of 557 bait clones were used for screening (98 self-active clones and two failed constructs were removed). A total of 1281 PPIs were obtained from 380 bait clones. Among these interactions, 30% (389) were PPIs between the products of the selected 657 genes (Supplementary Fig. <a reveal-id="DSM021F2" data-open="DSM021F2" class="link link-reveal link-table xref-fig">2</a>). These PPI data between the products of candidate genes conserved in the <em>A. thaliana</em> genome will be useful in functional analysis of plant genes of endosymbiont cyanobacterial origin. An example of the applicable PPI information is the acetyl-CoA carboxylase (ACCase) complex. ACCase is essential for fatty acid synthesis in both <em>Synechocystis</em> and plants. Many plants have two types of ACCases in two subcellular locations, a eukaryotic homomeric form in the cytoplasm and a prokaryotic heteromeric form in the chloroplast.<sup><a reveal-id="DSM021C37" data-open="DSM021C37" class="link link-ref link-reveal xref-bibr">37</a></sup> The plant heteromeric form of ACCase is considered to be of cyanobacterial origin, and is composed of four subunits, a biotin carboxyl carrier protein (BCCP), biotin carboxylase (BC), and the α and β subunits of carboxyltransferase (CT). The plant heteromeric form of ACCase easily dissociates into two components, the α and β subunits of CT and a BC–BCCP complex.<sup><a reveal-id="DSM021C37" data-open="DSM021C37" class="link link-ref link-reveal xref-bibr">37</a></sup> Consistent with the physiological properties of the plant heteromeric ACCase, the PPIs in <em>Synechocystis</em> were found between AccB (Slr0435: encoding BCCP) and AccC (Sll0053: encoding BC), and between AccA (Sll0728: encoding the CT α subunit) and AccD (Sll0336: encoding the CT β subunit). These results demonstrate the applicability of the PPI data to the study of plant genes of cyanobacterial origin. Additionally, the results also indicate the advantage of YTH analysis in detecting basal binary interactions among multi-subunit complex components as well as the limitations in detecting secondary interactions in the multi-subunit complex.</p><p>An example of applicable PPI information was also obtained from that of conserved hypothetical proteins. A bidirectional interaction was detected between the hypothetical YCF65 protein (Slr0923) and the 30S ribosomal protein S1 (Slr1356). The <em>A. thaliana</em> protein PSRP-3 encoded by a homologue of the <em>ycf65</em> gene on the nuclear genome has previously been identified as a component of the 30S subunit of the chloroplast ribosome by comprehensive proteome analysis of the chloroplast ribosome.<sup><a reveal-id="DSM021C38" data-open="DSM021C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Therefore, the PPI obtained between YCF65 (Slr0923) and the 30S ribosomal protein S1 (Slr1356) provides important experimental evidence revealing the functional conservation between cyanobacteria and plants, and also supports the hypothesis that <em>Psrp-3</em>/<em>ycf65</em> may be a gene transferred from the organelle genome to the nuclear genome during chloroplast evolution.<sup><a reveal-id="DSM021C38" data-open="DSM021C38" class="link link-ref link-reveal xref-bibr">38</a></sup></p>                            <h3 scrollto-destination=87195690 id="87195690" class="section-title" data-legacy-id=s3f>3.6. Interactions of proteins of unknown function</h3>
<p>When 1228 genes of unknown function were screened (including 190 genes screened as conserved genes), 2029 PPIs were obtained from 682 bait clones (54%). Among these interactions, 1104 PPIs from 525 bait clones interacted with proteins of known function. PPI data on proteins of known function were also obtained as prey clones during the course of screening. In the entire PPI network obtained in this study, 1022 proteins of unknown function were included (Table <a reveal-id="DSM021TB3" data-open="DSM021TB3" class="link link-reveal link-table xref-fig">3</a>). Of these, 727 have at least one partner of known function and 104 showed interactions with two or more known proteins of the same functional category (Table <a reveal-id="DSM021TB3" data-open="DSM021TB3" class="link link-reveal link-table xref-fig">3</a>). For example, Slr0978 interacts with four distinct hybrid type Hiks (Hik22, Hik24, Hik38, and Hik40), which are paralogues with common domains, suggesting that Slr0978 may be involved in multiple two component signal transduction pathways. Sll0269 and Slr1636 interacted with two methyl-accepting chemotaxis proteins, Sll0041 and Slr1044, and Sll0041 and Sll1294, respectively. Thus Sll0269 and/or Slr1636 may play roles in the chemotaxis signaling pathway.</p>                        <a id="87195692" scrollto-destination="87195692"></a>
<div content-id="DSM021TB3" class="table-modal"><div class="table-wrap"><div id="DSM021TB3" class="table-wrap-title" data-id="DSM021TB3"><div class="label -label"><strong>Table 3</strong></div><div class="caption"><p>Summary of interaction with protein of unknown function</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Description </th><th> </th></tr></thead><tbody><tr><td>Number of assessed baits of function unknown protein </td><td>1228 </td></tr><tr><td>Number of baits of function unknown protein exhibiting interaction </td><td>682 </td></tr><tr><td>Number of function unknown protein in whole network </td><td>1022 </td></tr><tr><td>Number of function unknown protein interact with annotated gene </td><td>727 </td></tr><tr><td>Number of function unknown protein interact with two or more annotated genes of same function category </td><td>104 </td></tr></tbody></table></div><div content-id="DSM021TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Description </th><th> </th></tr></thead><tbody><tr><td>Number of assessed baits of function unknown protein </td><td>1228 </td></tr><tr><td>Number of baits of function unknown protein exhibiting interaction </td><td>682 </td></tr><tr><td>Number of function unknown protein in whole network </td><td>1022 </td></tr><tr><td>Number of function unknown protein interact with annotated gene </td><td>727 </td></tr><tr><td>Number of function unknown protein interact with two or more annotated genes of same function category </td><td>104 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195692" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM021TB3" class="table-wrap-title" data-id="DSM021TB3"><div class="label -label"><strong>Table 3</strong></div><div class="caption"><p>Summary of interaction with protein of unknown function</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Description </th><th> </th></tr></thead><tbody><tr><td>Number of assessed baits of function unknown protein </td><td>1228 </td></tr><tr><td>Number of baits of function unknown protein exhibiting interaction </td><td>682 </td></tr><tr><td>Number of function unknown protein in whole network </td><td>1022 </td></tr><tr><td>Number of function unknown protein interact with annotated gene </td><td>727 </td></tr><tr><td>Number of function unknown protein interact with two or more annotated genes of same function category </td><td>104 </td></tr></tbody></table></div><div content-id="DSM021TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Description </th><th> </th></tr></thead><tbody><tr><td>Number of assessed baits of function unknown protein </td><td>1228 </td></tr><tr><td>Number of baits of function unknown protein exhibiting interaction </td><td>682 </td></tr><tr><td>Number of function unknown protein in whole network </td><td>1022 </td></tr><tr><td>Number of function unknown protein interact with annotated gene </td><td>727 </td></tr><tr><td>Number of function unknown protein interact with two or more annotated genes of same function category </td><td>104 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195692" target="_blank">View Large</a></div></div><p>The obtained PPI information regarding genes of unknown function expands our knowledge of known pathways by identifying novel interacting components of the pathways. For instance, interaction between the nitrogen regulatory protein PII (GlnB; Ssl0707) and a function unknown protein (Sll0985, now designated PamA) indicates the need for further analysis of Sll0985 in the PII signaling pathway. A detailed analysis using a <em>pamA</em> deletion mutant revealed that PamA plays a role in the control of transcript abundance of nitrogen-related and sugar catabolic genes.<sup><a reveal-id="DSM021C39" data-open="DSM021C39" class="link link-ref link-reveal xref-bibr">39</a></sup> Thus, the PPI information obtained in this study will provide useful starting points for further detailed functional analyses to expand the knowledge of known pathways and to place functionally uncharacterized proteins in their biological context.</p><p>We have established a YTH screening system feasible for large-scale analysis of <em>Synechocystis</em> sp. strain PCC6803, and have identified 3236 interactions involving 1920 proteins using 1825 selected genes as bait in this screen. The obtained interaction network contains numbers of proteins encoded by genes conserved in plant genomes, as wells as many genes of unknown function. Thus, these data should provide new insights and novel strategies for functional analysis of genes in <em>Synechocystis</em>, and, additionally, genes in other cyanobacteria and plant genes of cyanobacterial origin. To make the interaction data publicly available, the obtained PPI data have been integrated into the cyanobacterial genome database, CyanoBase (<a class="link link-uri" href="http://www.kazusa.or.jp/cyanobase/">http://www.kazusa.or.jp/cyanobase/</a>). We have provided all the PPI data, as well as the interaction categories so that the user can select a data set based on particular analysis needs.</p><p>Although it will be essential to confirm and extend our findings by independent tests, we believe that the obtained PPI data should provide a useful starting point for studying the functions of uncharacterized proteins and for predicting functional pathways in <em>Synechocystis</em>, other cyanobacteria, and plant.</p>                            <h2 scrollto-destination=87195696 id="87195696" class="section-title" data-legacy-id=s4>Funding</h2>
<p>This work was supported by the Kazusa DNA Research Institute Foundation.</p>                            <h2 scrollto-destination=87195698 id="87195698" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We would like to thank S. Sasamoto for excellent technical assistance. We are also grateful to Drs T. Kaneko, S. Okamoto and M. Nakao for their help in data analysis and for valuable discussions.</p><p><strong>Supplementary Data:</strong> Supplementary data are available online at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=87195701 id="87195701" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSM021C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kaneko</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kotani</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Sequence analysis of the genome of the unicellular cyanobacterium <em>Synechocystis</em> sp. strain PCC6803. II. Sequence determination of the entire genome and assignment of potential protein-coding regions</div>, <div class="source">DNA Res.</div> , <div class="year">1996</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">109</div>-<div class="lpage">136</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20analysis%20of%20the%20genome%20of%20the%20unicellular%20cyanobacterium%20Synechocystis%20sp.%20strain%20PCC6803.%20II.%20Sequence%20determination%20of%20the%20entire%20genome%20and%20assignment%20of%20potential%20protein-coding%20regions&amp;author=T.%20Kaneko&amp;author=S.%20Sato&amp;author=H.%20Kotani&amp;publication_year=1996&amp;journal=DNA%20Res.&amp;volume=3&amp;pages=109-136" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/3.3.109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F3.3.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F3.3.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8905231" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM021C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ikeuchi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tabata</div> <div class="given-names">S.</div></div>. </div><div class="article-title"><em>Synechocystis</em> sp. PCC 6803—a useful tool in the study of the genetics of cyanobacteria</div>, <div class="source">Photosynth Res.</div> , <div class="year">2001</div>, vol. <div class="volume">70</div> (pg. <div class="fpage">73</div>-<div class="lpage">83</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synechocystis%20sp.%20PCC%206803%E2%80%94a%20useful%20tool%20in%20the%20study%20of%20the%20genetics%20of%20cyanobacteria&amp;author=M.%20Ikeuchi&amp;author=S.%20Tabata&amp;publication_year=2001&amp;journal=Photosynth%20Res.&amp;volume=70&amp;pages=73-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1013887908680" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1013887908680" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1013887908680"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16228363" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM021C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hihara</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kamei</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kanehisa</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kaplan</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Ikeuchi</div> <div class="given-names">M.</div></div>. </div><div class="article-title">DNA microarray analysis of cyanobacterial gene expression during acclimation to high light</div>, <div class="source">Plant Cell</div> , <div class="year">2001</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">793</div>-<div class="lpage">806</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20microarray%20analysis%20of%20cyanobacterial%20gene%20expression%20during%20acclimation%20to%20high%20light&amp;author=Y.%20Hihara&amp;author=A.%20Kamei&amp;author=M.%20Kanehisa&amp;author=A.%20Kaplan&amp;author=M.%20Ikeuchi&amp;publication_year=2001&amp;journal=Plant%20Cell&amp;volume=13&amp;pages=793-806" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.13.4.793" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.13.4.793" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.13.4.793"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11283337" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM021C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sazuka</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Yamaguchi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ohara</div> <div class="given-names">O.</div></div>. </div><div class="article-title">Cyano2Dbase updated: linkage of 234 protein spots to corresponding genes through N-terminal microsequencing</div>, <div class="source">Electrophoresis</div> , <div class="year">1999</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">2160</div>-<div class="lpage">2171</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cyano2Dbase%20updated%3A%20linkage%20of%20234%20protein%20spots%20to%20corresponding%20genes%20through%20N-terminal%20microsequencing&amp;author=T.%20Sazuka&amp;author=M.%20Yamaguchi&amp;author=O.%20Ohara&amp;publication_year=1999&amp;journal=Electrophoresis&amp;volume=20&amp;pages=2160-2171" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1522-2683" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1522-2683" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1522-2683"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10493121" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM021C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakamura</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kaneko</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Miyajima</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Tabata</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Extension of CyanoBase. CyanoMutants: repository of mutant information on <em>Synechocystis</em> sp. strain PCC6803</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1999</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">66</div>-<div class="lpage">68</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extension%20of%20CyanoBase.%20CyanoMutants%3A%20repository%20of%20mutant%20information%20on%20Synechocystis%20sp.%20strain%20PCC6803&amp;author=Y.%20Nakamura&amp;author=T.%20Kaneko&amp;author=N.%20Miyajima&amp;author=S.%20Tabata&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=66-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.1.66" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.1.66" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.1.66"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9847144" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM021C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakamura</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kaneko</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Tabata</div> <div class="given-names">S.</div></div>. </div><div class="article-title">CyanoBase, the genome database for <em>Synechocystis</em> sp. strain PCC6803: status for the year</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2000</div>, vol. <div class="volume">28</div> pg. <div class="fpage">72</div>  <div class="comment">2000</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CyanoBase%2C%20the%20genome%20database%20for%20Synechocystis%20sp.%20strain%20PCC6803%3A%20status%20for%20the%20year&amp;author=Y.%20Nakamura&amp;author=T.%20Kaneko&amp;author=S.%20Tabata&amp;publication_year=2000&amp;journal=Nucleic%20Acids%20Res.&amp;volume=28&amp;pages=72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/28.1.72" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F28.1.72" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F28.1.72"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10592184" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM021C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Uetz</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Giot</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Cagney</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Mansfield</div> <div class="given-names">T. A.</div></div>, et al. </div><div class="article-title">A comprehensive analysis of protein-protein interactions in <em>Saccharomyces cerevisiae</em></div>, <div class="source">Nature</div> , <div class="year">2000</div>, vol. <div class="volume">403</div> (pg. <div class="fpage">623</div>-<div class="lpage">627</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comprehensive%20analysis%20of%20protein-protein%20interactions%20in%20Saccharomyces%20cerevisiae&amp;author=P.%20Uetz&amp;author=L.%20Giot&amp;author=G.%20Cagney&amp;author=T.%20A.%20Mansfield&amp;publication_year=2000&amp;journal=Nature&amp;volume=403&amp;pages=623-627" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35001009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35001009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35001009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10688190" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM021C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ito</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Chiba</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ozawa</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Yoshida</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hattori</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Sakaki</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">A comprehensive two-hybrid analysis to explore the yeast protein interactome</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2001</div>, vol. <div class="volume">98</div> (pg. <div class="fpage">4569</div>-<div class="lpage">4574</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comprehensive%20two-hybrid%20analysis%20to%20explore%20the%20yeast%20protein%20interactome&amp;author=T.%20Ito&amp;author=T.%20Chiba&amp;author=R.%20Ozawa&amp;author=M.%20Yoshida&amp;author=M.%20Hattori&amp;author=Y.%20Sakaki&amp;publication_year=2001&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=98&amp;pages=4569-4574" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.061034498" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.061034498" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.061034498"> </span></div> </div></div></div></div></div><div content-id="DSM021C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Walhout</div> <div class="given-names">A. J.</div></div>,  <div class="name"><div class="surname">Sordella</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Lu</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">Protein interaction mapping in <em>C. elegans</em> using proteins involved in vulval development</div>, <div class="source">Science</div> , <div class="year">2000</div>, vol. <div class="volume">287</div> (pg. <div class="fpage">116</div>-<div class="lpage">122</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protein%20interaction%20mapping%20in%20C.%20elegans%20using%20proteins%20involved%20in%20vulval%20development&amp;author=A.%20J.%20Walhout&amp;author=R.%20Sordella&amp;author=X.%20Lu&amp;publication_year=2000&amp;journal=Science&amp;volume=287&amp;pages=116-122" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.287.5450.116" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.287.5450.116" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.287.5450.116"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10615043" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM021C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rain</div> <div class="given-names">J. C.</div></div>,  <div class="name"><div class="surname">Selig</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">De Reuse</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">The protein–protein interaction map of <em>Helicobacter pylori</em></div>, <div class="source">Nature</div> , <div class="year">2001</div>, vol. <div class="volume">409</div> (pg. <div class="fpage">211</div>-<div class="lpage">215</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20protein%E2%80%93protein%20interaction%20map%20of%20Helicobacter%20pylori&amp;author=J.%20C.%20Rain&amp;author=L.%20Selig&amp;author=H.%20De%20Reuse&amp;publication_year=2001&amp;journal=Nature&amp;volume=409&amp;pages=211-215" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35051615" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35051615" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35051615"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11196647" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM021C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Giot</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Bader</div> <div class="given-names">J. S.</div></div>,  <div class="name"><div class="surname">Brouwer</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">A protein interaction map of <em>Drosophila melanogaster</em></div>, <div class="source">Science</div> , <div class="year">2003</div>, vol. <div class="volume">302</div> (pg. <div class="fpage">1727</div>-<div class="lpage">1736</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20protein%20interaction%20map%20of%20Drosophila%20melanogaster&amp;author=L.%20Giot&amp;author=J.%20S.%20Bader&amp;author=C.%20Brouwer&amp;publication_year=2003&amp;journal=Science&amp;volume=302&amp;pages=1727-1736" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1090289" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1090289" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1090289"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14605208" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM021C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">LaCount</div> <div class="given-names">D. J.</div></div>,  <div class="name"><div class="surname">Vignali</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Chettier</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">A protein interaction network of the malaria parasite <em>Plasmodium falciparum</em></div>, <div class="source">Nature</div> , <div class="year">2005</div>, vol. <div class="volume">438</div> (pg. <div class="fpage">103</div>-<div class="lpage">107</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20protein%20interaction%20network%20of%20the%20malaria%20parasite%20Plasmodium%20falciparum&amp;author=D.%20J.%20LaCount&amp;author=M.%20Vignali&amp;author=R.%20Chettier&amp;publication_year=2005&amp;journal=Nature&amp;volume=438&amp;pages=103-107" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature04104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature04104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature04104"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16267556" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM021C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parrish</div> <div class="given-names">J. R.</div></div>,  <div class="name"><div class="surname">Yu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">A proteome-wide protein interaction map for <em>Campylobacter jejuni</em></div>, <div class="source">Genome Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">R130</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20proteome-wide%20protein%20interaction%20map%20for%20Campylobacter%20jejuni&amp;author=J.%20R.%20Parrish&amp;author=J.%20Yu&amp;author=G.%20Liu&amp;publication_year=2007&amp;journal=Genome%20Biol.&amp;volume=8&amp;pages=R130" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2007-8-7-r130" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2007-8-7-r130" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2007-8-7-r130"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17615063" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM021C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mizuno</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kaneko</div> <div class="given-names">T</div></div>,  <div class="name"><div class="surname">Tabata</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Compilation of all genes encoding bacterial two-component signal transducers in the genome of the cyanobacterium, <em>Synechocystis</em> sp. strain PCC 6803</div>, <div class="source">DNA Res.</div> , <div class="year">1996</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">407</div>-<div class="lpage">414</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Compilation%20of%20all%20genes%20encoding%20bacterial%20two-component%20signal%20transducers%20in%20the%20genome%20of%20the%20cyanobacterium%2C%20Synechocystis%20sp.%20strain%20PCC%206803&amp;author=T.%20Mizuno&amp;author=T%20Kaneko&amp;author=S.%20Tabata&amp;publication_year=1996&amp;journal=DNA%20Res.&amp;volume=3&amp;pages=407-414" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/3.6.407" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F3.6.407" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F3.6.407"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9097043" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM021C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Murata</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">I.</div></div>. </div><div class="article-title">Exploitation of genomic sequences in a systematic analysis to access how cyanobacteria sense environmental stress</div>, <div class="source">J. Exp. Bot.</div> , <div class="year">2006</div>, vol. <div class="volume">57</div> (pg. <div class="fpage">235</div>-<div class="lpage">247</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exploitation%20of%20genomic%20sequences%20in%20a%20systematic%20analysis%20to%20access%20how%20cyanobacteria%20sense%20environmental%20stress&amp;author=N.%20Murata&amp;author=I.%20Suzuki&amp;publication_year=2006&amp;journal=J.%20Exp.%20Bot.&amp;volume=57&amp;pages=235-247" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/erj005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferj005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferj005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16317040" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM021C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rujan</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Martin</div> <div class="given-names">W.</div></div>. </div><div class="article-title">How many genes in <em>Arabidopsis</em> come from cyanobacteria? An estimate from 386 protein phylogenies</div>, <div class="source">Trends Genet.</div> , <div class="year">2000</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">113</div>-<div class="lpage">120</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=How%20many%20genes%20in%20Arabidopsis%20come%20from%20cyanobacteria%3F%20An%20estimate%20from%20386%20protein%20phylogenies&amp;author=T.%20Rujan&amp;author=W.%20Martin&amp;publication_year=2000&amp;journal=Trends%20Genet.&amp;volume=17&amp;pages=113-120" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(00)02209-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(00)02209-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(00)02209-5"> </span></div> </div></div></div></div></div><div content-id="DSM021C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Martin</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Rujan</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Richly</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Evolutionary analysis of <em>Arabidopsis</em>, cyanobacterial, and chloroplast genomes reveals plastid phylogeny and thousands of cyanobacterial genes in the nucleus</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2002</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">12246</div>-<div class="lpage">12251</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20analysis%20of%20Arabidopsis%2C%20cyanobacterial%2C%20and%20chloroplast%20genomes%20reveals%20plastid%20phylogeny%20and%20thousands%20of%20cyanobacterial%20genes%20in%20the%20nucleus&amp;author=W.%20Martin&amp;author=T.%20Rujan&amp;author=E.%20Richly&amp;publication_year=2002&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=99&amp;pages=12246-12251" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.182432999" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.182432999" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.182432999"> </span></div> </div></div></div></div></div><div content-id="DSM021C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fromont-Racine</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Rain</div> <div class="given-names">J.-C.</div></div>,  <div class="name"><div class="surname">Legrain</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Toward a functional analysis of the yeast genome through exhaustive two-hybrid screens</div>, <div class="source">Nat. Genet.</div> , <div class="year">1997</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">277</div>-<div class="lpage">282</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Toward%20a%20functional%20analysis%20of%20the%20yeast%20genome%20through%20exhaustive%20two-hybrid%20screens&amp;author=M.%20Fromont-Racine&amp;author=J.-C.%20Rain&amp;author=P.%20Legrain&amp;publication_year=1997&amp;journal=Nat.%20Genet.&amp;volume=16&amp;pages=277-282" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng0797-277" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng0797-277" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng0797-277"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9207794" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM021C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ito</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Fukuda</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Murata</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kimura</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Transformation of intact yeast cells treated with alkali cations</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1983</div>, vol. <div class="volume">153</div> (pg. <div class="fpage">163</div>-<div class="lpage">168</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transformation%20of%20intact%20yeast%20cells%20treated%20with%20alkali%20cations&amp;author=H.%20Ito&amp;author=Y.%20Fukuda&amp;author=K.%20Murata&amp;author=A.%20Kimura&amp;publication_year=1983&amp;journal=J.%20Bacteriol.&amp;volume=153&amp;pages=163-168" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6336730" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM021C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Uetz</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Hughes</div> <div class="given-names">R. E.</div></div>. </div><div class="article-title">Systematic and large-scale two-hybrid screens</div>, <div class="source">Curr. Opin. Microbiol.</div> , <div class="year">2000</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">303</div>-<div class="lpage">308</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systematic%20and%20large-scale%20two-hybrid%20screens&amp;author=P.%20Uetz&amp;author=R.%20E.%20Hughes&amp;publication_year=2000&amp;journal=Curr.%20Opin.%20Microbiol.&amp;volume=3&amp;pages=303-308" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1369-5274(00)00094-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1369-5274(00)00094-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1369-5274(00)00094-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10851163" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM021C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Saito</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Hayashizaki</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Interaction generality, a measurement to assess the reliability of a protein-protein interaction</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2002</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">1163</div>-<div class="lpage">1168</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Interaction%20generality%2C%20a%20measurement%20to%20assess%20the%20reliability%20of%20a%20protein-protein%20interaction&amp;author=R.%20Saito&amp;author=H.%20Suzuki&amp;author=Y.%20Hayashizaki&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res.&amp;volume=30&amp;pages=1163-1168" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/30.5.1163" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F30.5.1163" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F30.5.1163"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11861907" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM021C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fields</div> <div class="given-names">S.</div></div>. </div><div class="article-title">High-throughput two-hybrid analysis</div>, <div class="source">FEBS J.</div> , <div class="year">2005</div>, vol. <div class="volume">272</div> (pg. <div class="fpage">5391</div>-<div class="lpage">5399</div>) <div class="comment">The promise and the peril</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20two-hybrid%20analysis&amp;author=S.%20Fields&amp;publication_year=2005&amp;journal=FEBS%20J.&amp;volume=272&amp;pages=5391-5399" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/ejb.2005.272.issue-21" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fejb.2005.272.issue-21" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fejb.2005.272.issue-21"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16262681" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM021C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suzuki</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Fukunishi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kagawa</div> <div class="given-names">I.</div></div>, et al. </div><div class="article-title">Protein–protein interaction panel using mouse full-length cDNAs</div>, <div class="source">Genome Res.</div> , <div class="year">2007</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">1758</div>-<div class="lpage">1765</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protein%E2%80%93protein%20interaction%20panel%20using%20mouse%20full-length%20cDNAs&amp;author=H.%20Suzuki&amp;author=Y.%20Fukunishi&amp;author=I.%20Kagawa&amp;publication_year=2007&amp;journal=Genome%20Res.&amp;volume=11&amp;pages=1758-1765" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.180101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.180101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.180101"> </span></div> </div></div></div></div></div><div content-id="DSM021C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ng</div> <div class="given-names">S. K.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Tan</div> <div class="given-names">S.H.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">K.</div></div>. </div><div class="article-title">InterDom: a database of putative interacting protein domains for validating predicted protein interactions and complexes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">251</div>-<div class="lpage">254</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=InterDom%3A%20a%20database%20of%20putative%20interacting%20protein%20domains%20for%20validating%20predicted%20protein%20interactions%20and%20complexes&amp;author=S.%20K.%20Ng&amp;author=Z.%20Zhang&amp;author=S.H.%20Tan&amp;author=K.%20Lin&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=251-254" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg079" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg079" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg079"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12519994" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM021C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Arifuzzaman</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Maeda</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Itoh</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Large-scale identification of protein-protein interaction of <em>Escherichia coli</em> K-12</div>, <div class="source">Genome Res.</div> , vol. <div class="volume">16</div> (pg. <div class="fpage">686</div>-<div class="lpage">691</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.4527806" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.4527806" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.4527806"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16606699" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM021C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shannon</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Markiel</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Ozier</div> <div class="given-names">O.</div></div>, et al. </div><div class="article-title">Cytoscape: a software environment for integrated models of biomolecular interaction networks</div>, <div class="source">Genome Res.</div> , vol. <div class="volume">13</div> (pg. <div class="fpage">2498</div>-<div class="lpage">2504</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1239303" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1239303" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1239303"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14597658" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM021C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Snel</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Bork</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Huynen</div> <div class="given-names">M. A.</div></div>. </div><div class="article-title">The identification of functional modules from the genomic association of genes</div>, <div class="source">Proc. Natl Acad. Sci.</div> , <div class="year">2002</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">5890</div>-<div class="lpage">5895</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20identification%20of%20functional%20modules%20from%20the%20genomic%20association%20of%20genes&amp;author=B.%20Snel&amp;author=P.%20Bork&amp;author=M.%20A.%20Huynen&amp;publication_year=2002&amp;journal=Proc.%20Natl%20Acad.%20Sci.&amp;volume=99&amp;pages=5890-5895" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.092632599" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.092632599" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.092632599"> </span></div> </div></div></div></div></div><div content-id="DSM021C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Overbeek</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Fonstein</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">D'Souza</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Pusch</div> <div class="given-names">G. D.</div></div>,  <div class="name"><div class="surname">Maltsev</div> <div class="given-names">N.</div></div>. </div><div class="article-title">The use of gene clusters to infer functional coupling</div>, <div class="source">Proc. Natl Acad. Sci.</div> , <div class="year">1999</div>, vol. <div class="volume">96</div> (pg. <div class="fpage">2896</div>-<div class="lpage">2901</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20use%20of%20gene%20clusters%20to%20infer%20functional%20coupling&amp;author=R.%20Overbeek&amp;author=M.%20Fonstein&amp;author=M.%20D'Souza&amp;author=G.%20D.%20Pusch&amp;author=N.%20Maltsev&amp;publication_year=1999&amp;journal=Proc.%20Natl%20Acad.%20Sci.&amp;volume=96&amp;pages=2896-2901" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.96.6.2896" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.96.6.2896" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.96.6.2896"> </span></div> </div></div></div></div></div><div content-id="DSM021C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Helmann</div> <div class="given-names">J. D.</div></div>. </div><div class="article-title">The extracytoplasmic function (ECF) sigma factors</div>, <div class="source">Adv. Microb. Physiol.</div> , <div class="year">2002</div>, vol. <div class="volume">46</div> (pg. <div class="fpage">47</div>-<div class="lpage">110</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20extracytoplasmic%20function%20(ECF)%20sigma%20factors&amp;author=J.%20D.%20Helmann&amp;publication_year=2002&amp;journal=Adv.%20Microb.%20Physiol.&amp;volume=46&amp;pages=47-110" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12073657" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM021C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ispolatov</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Yuryev</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Mazo</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Maslov</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Binding properties and evolution of homodimers in protein–protein interaction networks</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">3629</div>-<div class="lpage">3635</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Binding%20properties%20and%20evolution%20of%20homodimers%20in%20protein%E2%80%93protein%20interaction%20networks&amp;author=I.%20Ispolatov&amp;author=A.%20Yuryev&amp;author=I.%20Mazo&amp;author=S.%20Maslov&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=3629-3635" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki678" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki678" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki678"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15983135" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM021C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Alex</div> <div class="given-names">L. A.</div></div>,  <div class="name"><div class="surname">Simon</div> <div class="given-names">M. I.</div></div>. </div><div class="article-title">Protein histidine kinases and signal transduction in prokaryotes and eukaryotes</div>, <div class="source">Trends Genet.</div> , <div class="year">1994</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">133</div>-<div class="lpage">138</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protein%20histidine%20kinases%20and%20signal%20transduction%20in%20prokaryotes%20and%20eukaryotes&amp;author=L.%20A.%20Alex&amp;author=M.%20I.%20Simon&amp;publication_year=1994&amp;journal=Trends%20Genet.&amp;volume=10&amp;pages=133-138" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0168-9525(94)90215-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0168-9525(94)90215-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0168-9525(94)90215-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8029829" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM021C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parkinson</div> <div class="given-names">J. S.</div></div>,  <div class="name"><div class="surname">Kofoid</div> <div class="given-names">E. C.</div></div>. </div><div class="article-title">Communication modules in bacterial signaling proteins</div>, <div class="source">Annu. Rev. Genet.</div> , <div class="year">1992</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">71</div>-<div class="lpage">112</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Communication%20modules%20in%20bacterial%20signaling%20proteins&amp;author=J.%20S.%20Parkinson&amp;author=E.%20C.%20Kofoid&amp;publication_year=1992&amp;journal=Annu.%20Rev.%20Genet.&amp;volume=26&amp;pages=71-112" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.ge.26.120192.000443" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.ge.26.120192.000443" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.ge.26.120192.000443"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1482126" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM021C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wanner</div> <div class="given-names">B.L.</div></div>. </div><div class="article-title">Is cross regulation by phosphorylation of two-component response regulator proteins important in bacteria</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1992</div>, vol. <div class="volume">174</div> (pg. <div class="fpage">2053</div>-<div class="lpage">2058</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Is%20cross%20regulation%20by%20phosphorylation%20of%20two-component%20response%20regulator%20proteins%20important%20in%20bacteria&amp;author=B.L.%20Wanner&amp;publication_year=1992&amp;journal=J.%20Bacteriol.&amp;volume=174&amp;pages=2053-2058" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/jb.174.7.2053-2058.1992" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2Fjb.174.7.2053-2058.1992" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2Fjb.174.7.2053-2058.1992"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1551826" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM021C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suzuki</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Los</div> <div class="given-names">D. A.</div></div>,  <div class="name"><div class="surname">Kanesaki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Mikami</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Murata</div> <div class="given-names">N.</div></div>. </div><div class="article-title">The pathway for perception and transduction of low-temperature signals in <em>Synechocystis</em></div>, <div class="source">EMBO J.</div> , <div class="year">2000</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">1327</div>-<div class="lpage">1334</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20pathway%20for%20perception%20and%20transduction%20of%20low-temperature%20signals%20in%20Synechocystis&amp;author=I.%20Suzuki&amp;author=D.%20A.%20Los&amp;author=Y.%20Kanesaki&amp;author=K.%20Mikami&amp;author=N.%20Murata&amp;publication_year=2000&amp;journal=EMBO%20J.&amp;volume=19&amp;pages=1327-1334" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/emboj/19.6.1327" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Femboj%2F19.6.1327" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Femboj%2F19.6.1327"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10716932" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM021C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Paithoonrangsarid</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Shoumskaya</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Kanesaki</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Five histidine kinases perceive osmotic stress and regulate distinct sets of genes in <em>Synechocystis</em></div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2004</div>, vol. <div class="volume">279</div> (pg. <div class="fpage">53078</div>-<div class="lpage">53086</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Five%20histidine%20kinases%20perceive%20osmotic%20stress%20and%20regulate%20distinct%20sets%20of%20genes%20in%20Synechocystis&amp;author=K.%20Paithoonrangsarid&amp;author=M.%20A.%20Shoumskaya&amp;author=Y.%20Kanesaki&amp;publication_year=2004&amp;journal=J.%20Biol.%20Chem.&amp;volume=279&amp;pages=53078-53086" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M410162200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M410162200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M410162200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15471853" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM021C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shoumskaya</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Paithoonrangsarid</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kanesaki</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Identical Hik-Rre systems are involved in perception and transduction of salt signals and hyperosmotic signals but regulate the expression of individual genes to different extents in <em>Synechocystis</em></div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2005</div>, vol. <div class="volume">280</div> (pg. <div class="fpage">21531</div>-<div class="lpage">21538</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identical%20Hik-Rre%20systems%20are%20involved%20in%20perception%20and%20transduction%20of%20salt%20signals%20and%20hyperosmotic%20signals%20but%20regulate%20the%20expression%20of%20individual%20genes%20to%20different%20extents%20in%20Synechocystis&amp;author=M.%20A.%20Shoumskaya&amp;author=K.%20Paithoonrangsarid&amp;author=Y.%20Kanesaki&amp;publication_year=2005&amp;journal=J.%20Biol.%20Chem.&amp;volume=280&amp;pages=21531-21538" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M412174200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M412174200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M412174200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15805106" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM021C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sasaki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Nagano</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Plant acetyl-CoA carboxylase: Structure, biosynthesis, regulation, and gene manipulation for plant breeding</div>, <div class="source">Biosci. Biotechnol. Biochem.</div> , <div class="year">2004</div>, vol. <div class="volume">68</div> (pg. <div class="fpage">1175</div>-<div class="lpage">1184</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20acetyl-CoA%20carboxylase%3A%20Structure%2C%20biosynthesis%2C%20regulation%2C%20and%20gene%20manipulation%20for%20plant%20breeding&amp;author=Y.%20Sasaki&amp;author=Y.%20Nagano&amp;publication_year=2004&amp;journal=Biosci.%20Biotechnol.%20Biochem.&amp;volume=68&amp;pages=1175-1184" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1271/bbb.68.1175" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1271%2Fbbb.68.1175" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1271%2Fbbb.68.1175"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15215578" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM021C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamaguchi</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Subramanian</div> <div class="given-names">A. R.</div></div>. </div><div class="article-title">Proteomic identification of all plastid-specific ribosomal proteins in higher plant chloroplast 30S ribosomal subunit</div>, <div class="source">Eur. J. Biochem.</div> , <div class="year">2003</div>, vol. <div class="volume">270</div> (pg. <div class="fpage">190</div>-<div class="lpage">205</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Proteomic%20identification%20of%20all%20plastid-specific%20ribosomal%20proteins%20in%20higher%20plant%20chloroplast%2030S%20ribosomal%20subunit&amp;author=K.%20Yamaguchi&amp;author=A.%20R.%20Subramanian&amp;publication_year=2003&amp;journal=Eur.%20J.%20Biochem.&amp;volume=270&amp;pages=190-205" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1432-1033.2003.03359.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1432-1033.2003.03359.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1432-1033.2003.03359.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12605670" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM021C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM021C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Osanai</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Tabata</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Tanaka</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Identification of PamA as a PII-binding membrane protein important in nitrogen-related and sugar-catabolic gene expression in <em>Synechocystis</em> sp. PCC 6803</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2005</div>, vol. <div class="volume">280</div> (pg. <div class="fpage">34684</div>-<div class="lpage">34690</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20PamA%20as%20a%20PII-binding%20membrane%20protein%20important%20in%20nitrogen-related%20and%20sugar-catabolic%20gene%20expression%20in%20Synechocystis%20sp.%20PCC%206803&amp;author=T.%20Osanai&amp;author=S.%20Sato&amp;author=S.%20Tabata&amp;author=K.%20Tanaka&amp;publication_year=2005&amp;journal=J.%20Biol.%20Chem.&amp;volume=280&amp;pages=34684-34690" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M507489200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M507489200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M507489200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16109709" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2007. Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=351235&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/5/10.1093_dnares_dsm021/1/dsm021_Supplementary_Data.zip?Expires=2147483647&amp;Signature=2E6KVT8WJXAUMZp~Me9FAYdSddXswFIvKLiLjAEh016lRzlFHzF~Kn0Ze4y1YsRK5gy1No6lsoNvVl0W0Uoc4pYWm5daT6kOaALix63wZ29hsusE-Ksge8u1l2QE3f3bnttY35SsWL8Z9x9fROOGho0nLe9HeH-v~poxymrZ~mwlQsLRuEJ73Q7421c05NH7fiDosnxrooeFwZ9k4aVmbiuLbe7Uv-2~qviuLCTKgHWdneSwDd~WLa0JgIQ9sscuq5s-Sa5Jt~n8kp5k3vpTAi3MsnQUrwCP9ZVmi-sEUzHG0xPVFreVf~goF6OXlLykiQ0~LRhnvXOV26Ci017dsQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">409</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000251879300002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">110</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsm021"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="351235"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000251879300002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f14%2f5%2f207%2f351235">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000251879300002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (110)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F14%2F5%2F207%2F351235" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/351235'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Large-scale Protein–protein Interaction Analysis in Synechocystis sp. PCC6803" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsm021");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f14%2f5%2f207%2f351235">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

