

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/1/39/440428","@type":"ScholarlyArticle","name":"Construction of a high-density, high-resolution genetic map and its integration with BAC-based physical map in channel catfish","datePublished":"2014-11-26","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2015-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsu038","keywords":["SNP","catfish","linkage map","physical map","genome"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/1/39/440428","author":[{"name":"Li, Yun","affiliation":"1  The Fish Molecular Genetics and Biotechnology Laboratory, School of Fisheries, Aquaculture, and Aquatic Sciences and Program of Cell and Molecular Biosciences, Aquatic Genomics Unit ,  Auburn University ,  Auburn, AL 36849 ,  USA","@type":"Person"},{"name":"Liu, Shikai","affiliation":"1  The Fish Molecular Genetics and Biotechnology Laboratory, School of Fisheries, Aquaculture, and Aquatic Sciences and Program of Cell and Molecular Biosciences, Aquatic Genomics Unit ,  Auburn University ,  Auburn, AL 36849 ,  USA","@type":"Person"},{"name":"Qin, Zhenkui","affiliation":"1  The Fish Molecular Genetics and Biotechnology Laboratory, School of Fisheries, Aquaculture, and Aquatic Sciences and Program of Cell and Molecular Biosciences, Aquatic Genomics Unit ,  Auburn University ,  Auburn, AL 36849 ,  USA","@type":"Person"},{"name":"Waldbieser, Geoff","affiliation":"2  USDA-ARS Warmwater Aquaculture Research Unit ,  Stoneville, MS 38776 ,  USA","@type":"Person"},{"name":"Wang, Ruijia","affiliation":"1  The Fish Molecular Genetics and Biotechnology Laboratory, School of Fisheries, Aquaculture, and Aquatic Sciences and Program of Cell and Molecular Biosciences, Aquatic Genomics Unit ,  Auburn University ,  Auburn, AL 36849 ,  USA","@type":"Person"},{"name":"Sun, Luyang","affiliation":"1  The Fish Molecular Genetics and Biotechnology Laboratory, School of Fisheries, Aquaculture, and Aquatic Sciences and Program of Cell and Molecular Biosciences, Aquatic Genomics Unit ,  Auburn University ,  Auburn, AL 36849 ,  USA","@type":"Person"},{"name":"Bao, Lisui","affiliation":"1  The Fish Molecular Genetics and Biotechnology Laboratory, School of Fisheries, Aquaculture, and Aquatic Sciences and Program of Cell and Molecular Biosciences, Aquatic Genomics Unit ,  Auburn University ,  Auburn, AL 36849 ,  USA","@type":"Person"},{"name":"Danzmann, Roy G.","affiliation":"3  Department of Integrative Biology ,  University of Guelph ,  Guelph, Ontario N1G 2W1 ,  Canada","@type":"Person"},{"name":"Dunham, Rex","affiliation":"1  The Fish Molecular Genetics and Biotechnology Laboratory, School of Fisheries, Aquaculture, and Aquatic Sciences and Program of Cell and Molecular Biosciences, Aquatic Genomics Unit ,  Auburn University ,  Auburn, AL 36849 ,  USA","@type":"Person"},{"name":"Liu, Zhanjiang","affiliation":"1  The Fish Molecular Genetics and Biotechnology Laboratory, School of Fisheries, Aquaculture, and Aquatic Sciences and Program of Cell and Molecular Biosciences, Aquatic Genomics Unit ,  Auburn University ,  Auburn, AL 36849 ,  USA","@type":"Person"}],"description":"Abstract.  Construction of genetic linkage map is essential for genetic and genomic studies. Recent advances in sequencing and genotyping technologies made it p","pageStart":"39","pageEnd":"52","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/1/1/m_cover.gif?Expires=2147483647&Signature=Vhf6NToQrkVldazSweShdrlrfCnUbBpTokrfu8JXGVRMaoyIjioben46GoxM3BSEoRr~te73o-dfY7QEM91wVumB~VX8hAxN9ywAZ1zRAbXXmtU23LiP8gcw~y1-D8ACKe3CZXmetX85Oj4q~Op8xej8CbOs0MGBcAElTMf~GIynbEl8oPRbKlaZUEBETFvDqCw1PlymDq~5degwzATpcv5kHJhRgAnT8PjHfQEqeZxMFeCt-B-phqwpjq5AMA0hAnSz5a5U5SMfCC3Naxv-~~yWr1GxeMNVTp3iBMlBqhdGNwFRsnBBquWLbTiF47z2gcE0wubs8CNqU4CX2qcpAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Construction of a high-density, high-resolution genetic map and its integration with BAC-based physical map in channel catfish","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/1/1/m_cover.gif?Expires=2147483647&Signature=Vhf6NToQrkVldazSweShdrlrfCnUbBpTokrfu8JXGVRMaoyIjioben46GoxM3BSEoRr~te73o-dfY7QEM91wVumB~VX8hAxN9ywAZ1zRAbXXmtU23LiP8gcw~y1-D8ACKe3CZXmetX85Oj4q~Op8xej8CbOs0MGBcAElTMf~GIynbEl8oPRbKlaZUEBETFvDqCw1PlymDq~5degwzATpcv5kHJhRgAnT8PjHfQEqeZxMFeCt-B-phqwpjq5AMA0hAnSz5a5U5SMfCC3Naxv-~~yWr1GxeMNVTp3iBMlBqhdGNwFRsnBBquWLbTiF47z2gcE0wubs8CNqU4CX2qcpAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Construction of a high-density, high-resolution genetic map and its integration with BAC-based physical map in channel catfish" />
    <meta property="og:description" content="Abstract.  Construction of genetic linkage map is essential for genetic and genomic studies. Recent advances in sequencing and genotyping technologies made it p" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/1/39/440428" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2014-11-26" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Li, Yun" /><meta name="citation_author_institution" content="The Fish Molecular Genetics and Biotechnology Laboratory, School of Fisheries, Aquaculture, and Aquatic Sciences and Program of Cell and Molecular Biosciences, Aquatic Genomics Unit, Auburn University, Auburn, AL 36849, USA" /><meta name="citation_author" content="Liu, Shikai" /><meta name="citation_author_institution" content="The Fish Molecular Genetics and Biotechnology Laboratory, School of Fisheries, Aquaculture, and Aquatic Sciences and Program of Cell and Molecular Biosciences, Aquatic Genomics Unit, Auburn University, Auburn, AL 36849, USA" /><meta name="citation_author" content="Qin, Zhenkui" /><meta name="citation_author_institution" content="The Fish Molecular Genetics and Biotechnology Laboratory, School of Fisheries, Aquaculture, and Aquatic Sciences and Program of Cell and Molecular Biosciences, Aquatic Genomics Unit, Auburn University, Auburn, AL 36849, USA" /><meta name="citation_author" content="Waldbieser, Geoff" /><meta name="citation_author_institution" content="USDA-ARS Warmwater Aquaculture Research Unit, Stoneville, MS 38776, USA" /><meta name="citation_author" content="Wang, Ruijia" /><meta name="citation_author_institution" content="The Fish Molecular Genetics and Biotechnology Laboratory, School of Fisheries, Aquaculture, and Aquatic Sciences and Program of Cell and Molecular Biosciences, Aquatic Genomics Unit, Auburn University, Auburn, AL 36849, USA" /><meta name="citation_author" content="Sun, Luyang" /><meta name="citation_author_institution" content="The Fish Molecular Genetics and Biotechnology Laboratory, School of Fisheries, Aquaculture, and Aquatic Sciences and Program of Cell and Molecular Biosciences, Aquatic Genomics Unit, Auburn University, Auburn, AL 36849, USA" /><meta name="citation_author" content="Bao, Lisui" /><meta name="citation_author_institution" content="The Fish Molecular Genetics and Biotechnology Laboratory, School of Fisheries, Aquaculture, and Aquatic Sciences and Program of Cell and Molecular Biosciences, Aquatic Genomics Unit, Auburn University, Auburn, AL 36849, USA" /><meta name="citation_author" content="Danzmann, Roy G." /><meta name="citation_author_institution" content="Department of Integrative Biology, University of Guelph, Guelph, Ontario N1G 2W1, Canada" /><meta name="citation_author" content="Dunham, Rex" /><meta name="citation_author_institution" content="The Fish Molecular Genetics and Biotechnology Laboratory, School of Fisheries, Aquaculture, and Aquatic Sciences and Program of Cell and Molecular Biosciences, Aquatic Genomics Unit, Auburn University, Auburn, AL 36849, USA" /><meta name="citation_author" content="Liu, Zhanjiang" /><meta name="citation_author_institution" content="The Fish Molecular Genetics and Biotechnology Laboratory, School of Fisheries, Aquaculture, and Aquatic Sciences and Program of Cell and Molecular Biosciences, Aquatic Genomics Unit, Auburn University, Auburn, AL 36849, USA" /><meta name="citation_title" content="Construction of a high-density, high-resolution genetic map and its integration with BAC-based physical map in channel catfish" /><meta name="citation_firstpage" content="39" /><meta name="citation_lastpage" content="52" /><meta name="citation_doi" content="10.1093/dnares/dsu038" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2015/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=A fine-scale map of recombination rates and hotspots across the human genome; Science;  citation_year=2005;  citation_volume=310;  citation_pages=321-4; " /><meta name="citation_reference" content="citation_title=Construction of genetic linkage maps and comparative genome analysis of catfish using gene-associated markers; Genetics;  citation_year=2009;  citation_volume=181;  citation_pages=1649-60; " /><meta name="citation_reference" content="citation_title=Cloning of large segments of exogenous DNA into yeast by means of artificial chromosome vectors; Science;  citation_year=1987;  citation_volume=236;  citation_pages=806-12; " /><meta name="citation_reference" content="citation_title=A physical map of the human genome; Nature;  citation_year=2001;  citation_volume=409;  citation_pages=934-41; " /><meta name="citation_reference" content="citation_title=Use of an ordered cosmid library to deduce the genomic organization of Mycobacterium leprae; Mol. Microbiol.;  citation_year=1993;  citation_volume=7;  citation_pages=197-206; " /><meta name="citation_reference" content="citation_title=A BAC-based physical map of the channel catfish genome; Genomics;  citation_year=2007;  citation_volume=90;  citation_pages=380-8; " /><meta name="citation_reference" content="citation_title=A first generation BAC-based physical map of the channel catfish genome; BMC Genomics;  citation_year=2007;  citation_volume=8;  citation_pages=40" /><meta name="citation_reference" content="citation_title=Comparative analysis of catfish BAC end sequences with the zebrafish genome; BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=592" /><meta name="citation_reference" content="citation_title=Second-generation genetic linkage map of catfish and its integration with the BAC-based physical map; G3 (Bethesda);  citation_year=2012;  citation_volume=2;  citation_pages=1233-41; " /><meta name="citation_reference" content="citation_title=A second generation integrated map of the rainbow trout (Oncorhynchus mykiss) genome: analysis of conserved synteny with model fish genomes; Mar. Biotechnol. (NY);  citation_year=2012;  citation_volume=14;  citation_pages=343-57; " /><meta name="citation_reference" content="citation_title=A high-resolution linkage map for comparative genome analysis and QTL fine mapping in Asian seabass, Lates calcarifer; BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=174" /><meta name="citation_reference" content="citation_title=A dense SNP-based linkage map for Atlantic salmon (Salmo salar) reveals extended chromosome homeologies and striking differences in sex-specific recombination patterns; BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=615" /><meta name="citation_reference" content="citation_title=Construction of a high-density microsatellite genetic linkage map and mapping of sexual and growth-related traits in half-smooth tongue sole (Cynoglossus semilaevis); PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e52097" /><meta name="citation_reference" content="citation_title=A second generation genetic map for rainbow trout (Oncorhynchus mykiss); BMC Genetics;  citation_year=2008;  citation_volume=9;  citation_pages=74" /><meta name="citation_reference" content="citation_title=A dense genetic linkage map for common carp and its integration with a BAC-based physical map; PLoS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=e63928" /><meta name="citation_reference" content="citation_title=An AFLP-based genetic linkage map of channel catfish (Ictalurus punctatus) constructed by using an interspecific hybrid resource family; Genetics;  citation_year=2003;  citation_volume=165;  citation_pages=687-94; " /><meta name="citation_reference" content="citation_title=A microsatellite-based genetic linkage map for channel catfish, Ictalurus punctatus; Genetics;  citation_year=2001;  citation_volume=158;  citation_pages=727-34; " /><meta name="citation_reference" content="citation_title=BAC-based upgrading and physical integration of a genetic SNP map in Atlantic salmon; Anim. Genet.;  citation_year=2010;  citation_volume=41;  citation_pages=48-54; " /><meta name="citation_reference" content="citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers; PLoS ONE;  citation_year=2008;  citation_volume=3;  citation_pages=e3376" /><meta name="citation_reference" content="citation_title=Genome-wide genetic marker discovery and genotyping using next-generation sequencing; Nat. Rev. Genet.;  citation_year=2011;  citation_volume=12;  citation_pages=499-510; " /><meta name="citation_reference" content="citation_title=Rapid and cost-effective polymorphism identification and genotyping using restriction site associated DNA (RAD) markers; Genome Res.;  citation_year=2007;  citation_volume=17;  citation_pages=240-8; " /><meta name="citation_reference" content="citation_title=Genetic mapping of quantitative trait loci (QTL) for body-weight in Atlantic salmon (Salmo salar) using a 6.5K SNP array; Aquaculture;  citation_year=2012;  citation_volume=358;  citation_pages=61-70; " /><meta name="citation_reference" content="citation_title=A high density SNP array for the domestic horse and extant Perissodactyla: utility for association mapping, genetic diversity, and phylogeny studies; PLoS Genet.;  citation_year=2012;  citation_volume=8;  citation_pages=e1002451" /><meta name="citation_reference" content="citation_title=Development and validation of a high density SNP genotyping array for Atlantic salmon (Salmo salar); BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=90" /><meta name="citation_reference" content="citation_title=Multiple sex-associated regions and a putative sex chromosome in zebrafish revealed by RAD mapping and population genomics; PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e40701" /><meta name="citation_reference" content="citation_title=Linkage maps of the Atlantic salmon (Salmo salar) genome derived from RAD sequencing; BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=166" /><meta name="citation_reference" content="citation_title=Meiotic maps of sockeye salmon derived from massively parallel DNA sequencing; BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=521" /><meta name="citation_reference" content="citation_title=Mapping and validation of the major sex-determining region in Nile Tilapia (Oreochromis niloticus L.) using RAD sequencing; PLoS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=e68389" /><meta name="citation_reference" content="citation_title=Whole genome comparative analysis of channel catfish (Ictalurus punctatus) with four model fish species; BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=780" /><meta name="citation_reference" content="citation_title=Comparative genomic analysis of catfish linkage group 8 reveals two homologous chromosomes in zebrafish and other teleosts with extensive inter-chromosomal rearrangements; BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=387" /><meta name="citation_reference" content="citation_title=Generation of genome-scale gene-associated SNPs in catfish for the construction of a high-density SNP array; BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=53" /><meta name="citation_reference" content="citation_title=Identification and analysis of genome-wide SNPs provide insight into signatures of selection and domestication in catfish; PLoS ONE;  citation_year=2014;  citation_volume=9;  citation_pages=e109666" /><meta name="citation_reference" content="citation_title=Development of the catfish 250K SNP array for genome-wide association studies; BMC Res. Notes;  citation_year=2014;  citation_volume=7;  citation_pages=135" /><meta name="citation_reference" content="citation_publisher=Kyazma BV, Wageningen, The Netherlands; JoinMap 4. Software for the calculation of genetic linkage maps in experimental populations;  citation_year=2006; " /><meta name="citation_reference" content="citation_title=Construction of integrated genetic linkage maps by means of a new computer package: Join Map; Plant J.;  citation_year=1993;  citation_volume=3;  citation_pages=739-44; " /><meta name="citation_reference" content="citation_title=Accurate construction of consensus genetic maps via integer linear programming; IEEE/ACM Trans. Comput. Biol. Bioinform.;  citation_year=2011;  citation_volume=8;  citation_pages=381-94; " /><meta name="citation_reference" content="citation_title=MapChart: software for the graphical presentation of linkage maps and QTLs; J. Hered.;  citation_year=2002;  citation_volume=93;  citation_pages=77-8; " /><meta name="citation_reference" content="citation_publisher=Johns Hopkins Univ. Press, Baltimore, USA, ; Analysis of human genetic linkage;  citation_year=1999; " /><meta name="citation_reference" content="citation_title=On the use of the classical tests for detecting linkage; J. Hered.;  citation_year=1992;  citation_volume=83;  citation_pages=143-6; " /><meta name="citation_reference" content="citation_title=Channel catfish BAC-end sequences for marker development and assessment of syntenic conservation with other fish species; Anim. Genet.;  citation_year=2006;  citation_volume=37;  citation_pages=321-6; " /><meta name="citation_reference" content="citation_title=Generation of physical map contig-specific sequences useful for whole genome sequence scaffolding; PLoS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=e78872" /><meta name="citation_reference" content="citation_title=Two-phase analysis in consensus genetic mapping; G3-Genes Genom. Genet.;  citation_year=2012;  citation_volume=2;  citation_pages=537-49; " /><meta name="citation_reference" content="citation_title=Building reliable genetic maps: different mapping strategies may result in different maps; Nat. Sci.;  citation_year=2010;  citation_volume=2;  citation_pages=576" /><meta name="citation_reference" content="citation_title=The zebrafish reference genome sequence and its relationship to the human genome; Nature;  citation_year=2013;  citation_volume=496;  citation_pages=498-503; " /><meta name="citation_reference" content="citation_title=A genetic linkage map of a cichlid fish, the tilapia (Oreochromis niloticus); Genetics;  citation_year=1998;  citation_volume=148;  citation_pages=1225-32; " /><meta name="citation_reference" content="citation_title=A detailed linkage map of medaka, Oryzias latipes: comparative genomics and genome evolution; Genetics;  citation_year=2000;  citation_volume=154;  citation_pages=1773-84; " /><meta name="citation_reference" content="citation_title=A microsatellite linkage map of rainbow trout (Oncorhynchus mykiss) characterized by large sex-specific differences in recombination rates; Genetics;  citation_year=2000;  citation_volume=155;  citation_pages=1331-45; " /><meta name="citation_reference" content="citation_title=Why is the centromere so cold?; Genome Res.;  citation_year=1998;  citation_volume=8;  citation_pages=81-2; " /><meta name="citation_reference" content="citation_title=Heterogeneity in rates of recombination across the mouse genome; Genetics;  citation_year=1996;  citation_volume=142;  citation_pages=537-48; " /><meta name="citation_reference" content="citation_title=An evolutionary view of human recombination; Nat. Rev. Genet.;  citation_year=2007;  citation_volume=8;  citation_pages=23-34; " /><meta name="citation_reference" content="citation_title=A high density recombination map of the pig reveals a correlation between sex-specific recombination and GC content; BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=586" /><meta name="citation_reference" content="citation_title=A high-density SNP-based linkage map of the chicken genome reveals sequence features correlated with recombination rate; Genome Res.;  citation_year=2009;  citation_volume=19;  citation_pages=510-9; " /><meta name="citation_reference" content="citation_title=Comparative recombination rates in the rat, mouse, and human genomes; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=528-38; " /><meta name="citation_reference" content="citation_title=Meiotic recombination hot spots and cold spots; Nat. Rev. Genet.;  citation_year=2001;  citation_volume=2;  citation_pages=360-9; " /><meta name="citation_reference" content="citation_title=PRDM9 is a major determinant of meiotic recombination hotspots in humans and mice; Science;  citation_year=2010;  citation_volume=327;  citation_pages=836-40; " /><meta name="citation_reference" content="citation_title=Drive against hotspot motifs in primates implicates the PRDM9 gene in meiotic recombination; Science;  citation_year=2010;  citation_volume=327;  citation_pages=876-9; " /><meta name="citation_reference" content="citation_title=Prdm9 controls activation of mammalian recombination hotspots; Science;  citation_year=2010;  citation_volume=327;  citation_pages=835" /><meta name="citation_reference" content="citation_title=Sex-specific recombination rates in zebrafish (Danio rerio); Genetics;  citation_year=2002;  citation_volume=160;  citation_pages=649-57; " /><meta name="citation_reference" content="citation_title=A microsatellite linkage map of the European sea bass Dicentrarchus labrax L; Genetics;  citation_year=2005;  citation_volume=170;  citation_pages=1821-6; " /><meta name="citation_reference" content="citation_title=A second generation genetic linkage map for silver carp (Hypophthalmichehys molitrix) using microsatellite markers; Aquaculture;  citation_year=2013;  citation_volume=412;  citation_pages=97-106; " /><meta name="citation_reference" content="citation_title=A genetic linkage map for Arctic char (Salvelinus alpinus): evidence for higher recombination rates and segregation distortion in hybrid versus pure strain mapping parents; Genome;  citation_year=2004;  citation_volume=47;  citation_pages=304-15; " /><meta name="citation_reference" content="citation_title=A consensus linkage map of the grass carp (Ctenopharyngodon idella) based on microsatellites and SNPs; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=135" /><meta name="citation_reference" content="citation_title=Recombination difference between sexes: a role for haploid selection; PLoS Biol.;  citation_year=2005;  citation_volume=3;  citation_pages=396-403; " /><meta name="citation_reference" content="citation_title=Cytological studies of human meiosis: sex-specific differences in recombination originate at, or prior to, establishment of double-strand breaks; PLoS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=e85075" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/1/39/1208134/dsu038.pdf" /><meta name="description" content="Abstract.  Construction of genetic linkage map is essential for genetic and genomic studies. Recent advances in sequencing and genotyping technologies made it p" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/1/39/440428" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/1/39/440428" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/22/1/39/440428");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsu038';
    </script>
<title>
	Construction of a high-density, high-resolution genetic map and its integration with BAC-based physical map in channel catfish | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./440428" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZ6UATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGksIFl1biIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlRoZSBGaXNoIE1vbGVjdWxhciBHZW5ldGljcyBhbmQgQmlvdGVjaG5vbG9neSBMYWJvcmF0b3J5LCBTY2hvb2wgb2YgRmlzaGVyaWVzLCBBcXVhY3VsdHVyZSwgYW5kIEFxdWF0aWMgU2NpZW5jZXMgYW5kIFByb2dyYW0gb2YgQ2VsbCBhbmQgTW9sZWN1bGFyIEJpb3NjaWVuY2VzLCBBcXVhdGljIEdlbm9taWNzIFVuaXQsIEF1YnVybiBVbml2ZXJzaXR5LCBBdWJ1cm4sIEFMIDM2ODQ5LCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMaXUsIFNoaWthaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlRoZSBGaXNoIE1vbGVjdWxhciBHZW5ldGljcyBhbmQgQmlvdGVjaG5vbG9neSBMYWJvcmF0b3J5LCBTY2hvb2wgb2YgRmlzaGVyaWVzLCBBcXVhY3VsdHVyZSwgYW5kIEFxdWF0aWMgU2NpZW5jZXMgYW5kIFByb2dyYW0gb2YgQ2VsbCBhbmQgTW9sZWN1bGFyIEJpb3NjaWVuY2VzLCBBcXVhdGljIEdlbm9taWNzIFVuaXQsIEF1YnVybiBVbml2ZXJzaXR5LCBBdWJ1cm4sIEFMIDM2ODQ5LCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJRaW4sIFpoZW5rdWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJUaGUgRmlzaCBNb2xlY3VsYXIgR2VuZXRpY3MgYW5kIEJpb3RlY2hub2xvZ3kgTGFib3JhdG9yeSwgU2Nob29sIG9mIEZpc2hlcmllcywgQXF1YWN1bHR1cmUsIGFuZCBBcXVhdGljIFNjaWVuY2VzIGFuZCBQcm9ncmFtIG9mIENlbGwgYW5kIE1vbGVjdWxhciBCaW9zY2llbmNlcywgQXF1YXRpYyBHZW5vbWljcyBVbml0LCBBdWJ1cm4gVW5pdmVyc2l0eSwgQXVidXJuLCBBTCAzNjg0OSwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iV2FsZGJpZXNlciwgR2VvZmYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJVU0RBLUFSUyBXYXJtd2F0ZXIgQXF1YWN1bHR1cmUgUmVzZWFyY2ggVW5pdCwgU3RvbmV2aWxsZSwgTVMgMzg3NzYsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IldhbmcsIFJ1aWppYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlRoZSBGaXNoIE1vbGVjdWxhciBHZW5ldGljcyBhbmQgQmlvdGVjaG5vbG9neSBMYWJvcmF0b3J5LCBTY2hvb2wgb2YgRmlzaGVyaWVzLCBBcXVhY3VsdHVyZSwgYW5kIEFxdWF0aWMgU2NpZW5jZXMgYW5kIFByb2dyYW0gb2YgQ2VsbCBhbmQgTW9sZWN1bGFyIEJpb3NjaWVuY2VzLCBBcXVhdGljIEdlbm9taWNzIFVuaXQsIEF1YnVybiBVbml2ZXJzaXR5LCBBdWJ1cm4sIEFMIDM2ODQ5LCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTdW4sIEx1eWFuZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlRoZSBGaXNoIE1vbGVjdWxhciBHZW5ldGljcyBhbmQgQmlvdGVjaG5vbG9neSBMYWJvcmF0b3J5LCBTY2hvb2wgb2YgRmlzaGVyaWVzLCBBcXVhY3VsdHVyZSwgYW5kIEFxdWF0aWMgU2NpZW5jZXMgYW5kIFByb2dyYW0gb2YgQ2VsbCBhbmQgTW9sZWN1bGFyIEJpb3NjaWVuY2VzLCBBcXVhdGljIEdlbm9taWNzIFVuaXQsIEF1YnVybiBVbml2ZXJzaXR5LCBBdWJ1cm4sIEFMIDM2ODQ5LCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJCYW8sIExpc3VpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVGhlIEZpc2ggTW9sZWN1bGFyIEdlbmV0aWNzIGFuZCBCaW90ZWNobm9sb2d5IExhYm9yYXRvcnksIFNjaG9vbCBvZiBGaXNoZXJpZXMsIEFxdWFjdWx0dXJlLCBhbmQgQXF1YXRpYyBTY2llbmNlcyBhbmQgUHJvZ3JhbSBvZiBDZWxsIGFuZCBNb2xlY3VsYXIgQmlvc2NpZW5jZXMsIEFxdWF0aWMgR2Vub21pY3MgVW5pdCwgQXVidXJuIFVuaXZlcnNpdHksIEF1YnVybiwgQUwgMzY4NDksIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkRhbnptYW5uLCBSb3kgRy4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEludGVncmF0aXZlIEJpb2xvZ3ksIFVuaXZlcnNpdHkgb2YgR3VlbHBoLCBHdWVscGgsIE9udGFyaW8gTjFHIDJXMSwgQ2FuYWRhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRHVuaGFtLCBSZXgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJUaGUgRmlzaCBNb2xlY3VsYXIgR2VuZXRpY3MgYW5kIEJpb3RlY2hub2xvZ3kgTGFib3JhdG9yeSwgU2Nob29sIG9mIEZpc2hlcmllcywgQXF1YWN1bHR1cmUsIGFuZCBBcXVhdGljIFNjaWVuY2VzIGFuZCBQcm9ncmFtIG9mIENlbGwgYW5kIE1vbGVjdWxhciBCaW9zY2llbmNlcywgQXF1YXRpYyBHZW5vbWljcyBVbml0LCBBdWJ1cm4gVW5pdmVyc2l0eSwgQXVidXJuLCBBTCAzNjg0OSwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGl1LCBaaGFuamlhbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJUaGUgRmlzaCBNb2xlY3VsYXIgR2VuZXRpY3MgYW5kIEJpb3RlY2hub2xvZ3kgTGFib3JhdG9yeSwgU2Nob29sIG9mIEZpc2hlcmllcywgQXF1YWN1bHR1cmUsIGFuZCBBcXVhdGljIFNjaWVuY2VzIGFuZCBQcm9ncmFtIG9mIENlbGwgYW5kIE1vbGVjdWxhciBCaW9zY2llbmNlcywgQXF1YXRpYyBHZW5vbWljcyBVbml0LCBBdWJ1cm4gVW5pdmVyc2l0eSwgQXVidXJuLCBBTCAzNjg0OSwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJDb25zdHJ1Y3Rpb24gb2YgYSBoaWdoLWRlbnNpdHksIGhpZ2gtcmVzb2x1dGlvbiBnZW5ldGljIG1hcCBhbmQgaXRzIGludGVncmF0aW9uIHdpdGggQkFDLWJhc2VkIHBoeXNpY2FsIG1hcCBpbiBjaGFubmVsIGNhdGZpc2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIzOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3UwMzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNS8wMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZmluZS1zY2FsZSBtYXAgb2YgcmVjb21iaW5hdGlvbiByYXRlcyBhbmQgaG90c3BvdHMgYWNyb3NzIHRoZSBodW1hbiBnZW5vbWU7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzEwOyAgY2l0YXRpb25fcGFnZXM9MzIxLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBvZiBnZW5ldGljIGxpbmthZ2UgbWFwcyBhbmQgY29tcGFyYXRpdmUgZ2Vub21lIGFuYWx5c2lzIG9mIGNhdGZpc2ggdXNpbmcgZ2VuZS1hc3NvY2lhdGVkIG1hcmtlcnM7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE4MTsgIGNpdGF0aW9uX3BhZ2VzPTE2NDktNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNsb25pbmcgb2YgbGFyZ2Ugc2VnbWVudHMgb2YgZXhvZ2Vub3VzIEROQSBpbnRvIHllYXN0IGJ5IG1lYW5zIG9mIGFydGlmaWNpYWwgY2hyb21vc29tZSB2ZWN0b3JzOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTIzNjsgIGNpdGF0aW9uX3BhZ2VzPTgwNi0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBwaHlzaWNhbCBtYXAgb2YgdGhlIGh1bWFuIGdlbm9tZTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTQwOTsgIGNpdGF0aW9uX3BhZ2VzPTkzNC00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VXNlIG9mIGFuIG9yZGVyZWQgY29zbWlkIGxpYnJhcnkgdG8gZGVkdWNlIHRoZSBnZW5vbWljIG9yZ2FuaXphdGlvbiBvZiBNeWNvYmFjdGVyaXVtIGxlcHJhZTsgTW9sLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0xOTctMjA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIEJBQy1iYXNlZCBwaHlzaWNhbCBtYXAgb2YgdGhlIGNoYW5uZWwgY2F0ZmlzaCBnZW5vbWU7IEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTkwOyAgY2l0YXRpb25fcGFnZXM9MzgwLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZmlyc3QgZ2VuZXJhdGlvbiBCQUMtYmFzZWQgcGh5c2ljYWwgbWFwIG9mIHRoZSBjaGFubmVsIGNhdGZpc2ggZ2Vub21lOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTQwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgYW5hbHlzaXMgb2YgY2F0ZmlzaCBCQUMgZW5kIHNlcXVlbmNlcyB3aXRoIHRoZSB6ZWJyYWZpc2ggZ2Vub21lOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz01OTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZWNvbmQtZ2VuZXJhdGlvbiBnZW5ldGljIGxpbmthZ2UgbWFwIG9mIGNhdGZpc2ggYW5kIGl0cyBpbnRlZ3JhdGlvbiB3aXRoIHRoZSBCQUMtYmFzZWQgcGh5c2ljYWwgbWFwOyBHMyAoQmV0aGVzZGEpOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0xMjMzLTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHNlY29uZCBnZW5lcmF0aW9uIGludGVncmF0ZWQgbWFwIG9mIHRoZSByYWluYm93IHRyb3V0IChPbmNvcmh5bmNodXMgbXlraXNzKSBnZW5vbWU6IGFuYWx5c2lzIG9mIGNvbnNlcnZlZCBzeW50ZW55IHdpdGggbW9kZWwgZmlzaCBnZW5vbWVzOyBNYXIuIEJpb3RlY2hub2wuIChOWSk7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0zNDMtNTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgaGlnaC1yZXNvbHV0aW9uIGxpbmthZ2UgbWFwIGZvciBjb21wYXJhdGl2ZSBnZW5vbWUgYW5hbHlzaXMgYW5kIFFUTCBmaW5lIG1hcHBpbmcgaW4gQXNpYW4gc2VhYmFzcywgTGF0ZXMgY2FsY2FyaWZlcjsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTc0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBkZW5zZSBTTlAtYmFzZWQgbGlua2FnZSBtYXAgZm9yIEF0bGFudGljIHNhbG1vbiAoU2FsbW8gc2FsYXIpIHJldmVhbHMgZXh0ZW5kZWQgY2hyb21vc29tZSBob21lb2xvZ2llcyBhbmQgc3RyaWtpbmcgZGlmZmVyZW5jZXMgaW4gc2V4LXNwZWNpZmljIHJlY29tYmluYXRpb24gcGF0dGVybnM7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTYxNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBvZiBhIGhpZ2gtZGVuc2l0eSBtaWNyb3NhdGVsbGl0ZSBnZW5ldGljIGxpbmthZ2UgbWFwIGFuZCBtYXBwaW5nIG9mIHNleHVhbCBhbmQgZ3Jvd3RoLXJlbGF0ZWQgdHJhaXRzIGluIGhhbGYtc21vb3RoIHRvbmd1ZSBzb2xlIChDeW5vZ2xvc3N1cyBzZW1pbGFldmlzKTsgUExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWU1MjA5NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgc2Vjb25kIGdlbmVyYXRpb24gZ2VuZXRpYyBtYXAgZm9yIHJhaW5ib3cgdHJvdXQgKE9uY29yaHluY2h1cyBteWtpc3MpOyBCTUMgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTc0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBkZW5zZSBnZW5ldGljIGxpbmthZ2UgbWFwIGZvciBjb21tb24gY2FycCBhbmQgaXRzIGludGVncmF0aW9uIHdpdGggYSBCQUMtYmFzZWQgcGh5c2ljYWwgbWFwOyBQTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9ZTYzOTI4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gQUZMUC1iYXNlZCBnZW5ldGljIGxpbmthZ2UgbWFwIG9mIGNoYW5uZWwgY2F0ZmlzaCAoSWN0YWx1cnVzIHB1bmN0YXR1cykgY29uc3RydWN0ZWQgYnkgdXNpbmcgYW4gaW50ZXJzcGVjaWZpYyBoeWJyaWQgcmVzb3VyY2UgZmFtaWx5OyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNjU7ICBjaXRhdGlvbl9wYWdlcz02ODctOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbWljcm9zYXRlbGxpdGUtYmFzZWQgZ2VuZXRpYyBsaW5rYWdlIG1hcCBmb3IgY2hhbm5lbCBjYXRmaXNoLCBJY3RhbHVydXMgcHVuY3RhdHVzOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTg7ICBjaXRhdGlvbl9wYWdlcz03MjctMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJBQy1iYXNlZCB1cGdyYWRpbmcgYW5kIHBoeXNpY2FsIGludGVncmF0aW9uIG9mIGEgZ2VuZXRpYyBTTlAgbWFwIGluIEF0bGFudGljIHNhbG1vbjsgQW5pbS4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9NDgtNTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJhcGlkIFNOUCBkaXNjb3ZlcnkgYW5kIGdlbmV0aWMgbWFwcGluZyB1c2luZyBzZXF1ZW5jZWQgUkFEIG1hcmtlcnM7IFBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz1lMzM3NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGdlbmV0aWMgbWFya2VyIGRpc2NvdmVyeSBhbmQgZ2Vub3R5cGluZyB1c2luZyBuZXh0LWdlbmVyYXRpb24gc2VxdWVuY2luZzsgTmF0LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTQ5OS01MTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJhcGlkIGFuZCBjb3N0LWVmZmVjdGl2ZSBwb2x5bW9ycGhpc20gaWRlbnRpZmljYXRpb24gYW5kIGdlbm90eXBpbmcgdXNpbmcgcmVzdHJpY3Rpb24gc2l0ZSBhc3NvY2lhdGVkIEROQSAoUkFEKSBtYXJrZXJzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTI0MC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIG1hcHBpbmcgb2YgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgKFFUTCkgZm9yIGJvZHktd2VpZ2h0IGluIEF0bGFudGljIHNhbG1vbiAoU2FsbW8gc2FsYXIpIHVzaW5nIGEgNi41SyBTTlAgYXJyYXk7IEFxdWFjdWx0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTM1ODsgIGNpdGF0aW9uX3BhZ2VzPTYxLTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGhpZ2ggZGVuc2l0eSBTTlAgYXJyYXkgZm9yIHRoZSBkb21lc3RpYyBob3JzZSBhbmQgZXh0YW50IFBlcmlzc29kYWN0eWxhOiB1dGlsaXR5IGZvciBhc3NvY2lhdGlvbiBtYXBwaW5nLCBnZW5ldGljIGRpdmVyc2l0eSwgYW5kIHBoeWxvZ2VueSBzdHVkaWVzOyBQTG9TIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDI0NTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBhbmQgdmFsaWRhdGlvbiBvZiBhIGhpZ2ggZGVuc2l0eSBTTlAgZ2Vub3R5cGluZyBhcnJheSBmb3IgQXRsYW50aWMgc2FsbW9uIChTYWxtbyBzYWxhcik7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTkwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXVsdGlwbGUgc2V4LWFzc29jaWF0ZWQgcmVnaW9ucyBhbmQgYSBwdXRhdGl2ZSBzZXggY2hyb21vc29tZSBpbiB6ZWJyYWZpc2ggcmV2ZWFsZWQgYnkgUkFEIG1hcHBpbmcgYW5kIHBvcHVsYXRpb24gZ2Vub21pY3M7IFBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lNDA3MDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MaW5rYWdlIG1hcHMgb2YgdGhlIEF0bGFudGljIHNhbG1vbiAoU2FsbW8gc2FsYXIpIGdlbm9tZSBkZXJpdmVkIGZyb20gUkFEIHNlcXVlbmNpbmc7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTE2NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1laW90aWMgbWFwcyBvZiBzb2NrZXllIHNhbG1vbiBkZXJpdmVkIGZyb20gbWFzc2l2ZWx5IHBhcmFsbGVsIEROQSBzZXF1ZW5jaW5nOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz01MjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXBwaW5nIGFuZCB2YWxpZGF0aW9uIG9mIHRoZSBtYWpvciBzZXgtZGV0ZXJtaW5pbmcgcmVnaW9uIGluIE5pbGUgVGlsYXBpYSAoT3Jlb2Nocm9taXMgbmlsb3RpY3VzIEwuKSB1c2luZyBSQUQgc2VxdWVuY2luZzsgUExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPWU2ODM4OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdob2xlIGdlbm9tZSBjb21wYXJhdGl2ZSBhbmFseXNpcyBvZiBjaGFubmVsIGNhdGZpc2ggKEljdGFsdXJ1cyBwdW5jdGF0dXMpIHdpdGggZm91ciBtb2RlbCBmaXNoIHNwZWNpZXM7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTc4MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIGdlbm9taWMgYW5hbHlzaXMgb2YgY2F0ZmlzaCBsaW5rYWdlIGdyb3VwIDggcmV2ZWFscyB0d28gaG9tb2xvZ291cyBjaHJvbW9zb21lcyBpbiB6ZWJyYWZpc2ggYW5kIG90aGVyIHRlbGVvc3RzIHdpdGggZXh0ZW5zaXZlIGludGVyLWNocm9tb3NvbWFsIHJlYXJyYW5nZW1lbnRzOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0zODciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lcmF0aW9uIG9mIGdlbm9tZS1zY2FsZSBnZW5lLWFzc29jaWF0ZWQgU05QcyBpbiBjYXRmaXNoIGZvciB0aGUgY29uc3RydWN0aW9uIG9mIGEgaGlnaC1kZW5zaXR5IFNOUCBhcnJheTsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9NTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBhbmQgYW5hbHlzaXMgb2YgZ2Vub21lLXdpZGUgU05QcyBwcm92aWRlIGluc2lnaHQgaW50byBzaWduYXR1cmVzIG9mIHNlbGVjdGlvbiBhbmQgZG9tZXN0aWNhdGlvbiBpbiBjYXRmaXNoOyBQTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9ZTEwOTY2NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIHRoZSBjYXRmaXNoIDI1MEsgU05QIGFycmF5IGZvciBnZW5vbWUtd2lkZSBhc3NvY2lhdGlvbiBzdHVkaWVzOyBCTUMgUmVzLiBOb3RlczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MTM1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUt5YXptYSBCViwgV2FnZW5pbmdlbiwgVGhlIE5ldGhlcmxhbmRzOyBKb2luTWFwIDQuIFNvZnR3YXJlIGZvciB0aGUgY2FsY3VsYXRpb24gb2YgZ2VuZXRpYyBsaW5rYWdlIG1hcHMgaW4gZXhwZXJpbWVudGFsIHBvcHVsYXRpb25zOyAgY2l0YXRpb25feWVhcj0yMDA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgaW50ZWdyYXRlZCBnZW5ldGljIGxpbmthZ2UgbWFwcyBieSBtZWFucyBvZiBhIG5ldyBjb21wdXRlciBwYWNrYWdlOiBKb2luIE1hcDsgUGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTczOS00NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWNjdXJhdGUgY29uc3RydWN0aW9uIG9mIGNvbnNlbnN1cyBnZW5ldGljIG1hcHMgdmlhIGludGVnZXIgbGluZWFyIHByb2dyYW1taW5nOyBJRUVFL0FDTSBUcmFucy4gQ29tcHV0LiBCaW9sLiBCaW9pbmZvcm0uOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0zODEtOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcENoYXJ0OiBzb2Z0d2FyZSBmb3IgdGhlIGdyYXBoaWNhbCBwcmVzZW50YXRpb24gb2YgbGlua2FnZSBtYXBzIGFuZCBRVExzOyBKLiBIZXJlZC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9OTM7ICBjaXRhdGlvbl9wYWdlcz03Ny04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9Sm9obnMgSG9wa2lucyBVbml2LiBQcmVzcywgQmFsdGltb3JlLCBVU0EsIDsgQW5hbHlzaXMgb2YgaHVtYW4gZ2VuZXRpYyBsaW5rYWdlOyAgY2l0YXRpb25feWVhcj0xOTk5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PbiB0aGUgdXNlIG9mIHRoZSBjbGFzc2ljYWwgdGVzdHMgZm9yIGRldGVjdGluZyBsaW5rYWdlOyBKLiBIZXJlZC47ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9ODM7ICBjaXRhdGlvbl9wYWdlcz0xNDMtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhbm5lbCBjYXRmaXNoIEJBQy1lbmQgc2VxdWVuY2VzIGZvciBtYXJrZXIgZGV2ZWxvcG1lbnQgYW5kIGFzc2Vzc21lbnQgb2Ygc3ludGVuaWMgY29uc2VydmF0aW9uIHdpdGggb3RoZXIgZmlzaCBzcGVjaWVzOyBBbmltLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz0zMjEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXJhdGlvbiBvZiBwaHlzaWNhbCBtYXAgY29udGlnLXNwZWNpZmljIHNlcXVlbmNlcyB1c2VmdWwgZm9yIHdob2xlIGdlbm9tZSBzZXF1ZW5jZSBzY2FmZm9sZGluZzsgUExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPWU3ODg3MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVR3by1waGFzZSBhbmFseXNpcyBpbiBjb25zZW5zdXMgZ2VuZXRpYyBtYXBwaW5nOyBHMy1HZW5lcyBHZW5vbS4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz01MzctNDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJ1aWxkaW5nIHJlbGlhYmxlIGdlbmV0aWMgbWFwczogZGlmZmVyZW50IG1hcHBpbmcgc3RyYXRlZ2llcyBtYXkgcmVzdWx0IGluIGRpZmZlcmVudCBtYXBzOyBOYXQuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTU3NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSB6ZWJyYWZpc2ggcmVmZXJlbmNlIGdlbm9tZSBzZXF1ZW5jZSBhbmQgaXRzIHJlbGF0aW9uc2hpcCB0byB0aGUgaHVtYW4gZ2Vub21lOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDk2OyAgY2l0YXRpb25fcGFnZXM9NDk4LTUwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBnZW5ldGljIGxpbmthZ2UgbWFwIG9mIGEgY2ljaGxpZCBmaXNoLCB0aGUgdGlsYXBpYSAoT3Jlb2Nocm9taXMgbmlsb3RpY3VzKTsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTQ4OyAgY2l0YXRpb25fcGFnZXM9MTIyNS0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBkZXRhaWxlZCBsaW5rYWdlIG1hcCBvZiBtZWRha2EsIE9yeXppYXMgbGF0aXBlczogY29tcGFyYXRpdmUgZ2Vub21pY3MgYW5kIGdlbm9tZSBldm9sdXRpb247IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTE1NDsgIGNpdGF0aW9uX3BhZ2VzPTE3NzMtODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbWljcm9zYXRlbGxpdGUgbGlua2FnZSBtYXAgb2YgcmFpbmJvdyB0cm91dCAoT25jb3JoeW5jaHVzIG15a2lzcykgY2hhcmFjdGVyaXplZCBieSBsYXJnZSBzZXgtc3BlY2lmaWMgZGlmZmVyZW5jZXMgaW4gcmVjb21iaW5hdGlvbiByYXRlczsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTU1OyAgY2l0YXRpb25fcGFnZXM9MTMzMS00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2h5IGlzIHRoZSBjZW50cm9tZXJlIHNvIGNvbGQ/OyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9ODEtMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGV0ZXJvZ2VuZWl0eSBpbiByYXRlcyBvZiByZWNvbWJpbmF0aW9uIGFjcm9zcyB0aGUgbW91c2UgZ2Vub21lOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0xNDI7ICBjaXRhdGlvbl9wYWdlcz01MzctNDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGV2b2x1dGlvbmFyeSB2aWV3IG9mIGh1bWFuIHJlY29tYmluYXRpb247IE5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTIzLTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGhpZ2ggZGVuc2l0eSByZWNvbWJpbmF0aW9uIG1hcCBvZiB0aGUgcGlnIHJldmVhbHMgYSBjb3JyZWxhdGlvbiBiZXR3ZWVuIHNleC1zcGVjaWZpYyByZWNvbWJpbmF0aW9uIGFuZCBHQyBjb250ZW50OyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz01ODYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGhpZ2gtZGVuc2l0eSBTTlAtYmFzZWQgbGlua2FnZSBtYXAgb2YgdGhlIGNoaWNrZW4gZ2Vub21lIHJldmVhbHMgc2VxdWVuY2UgZmVhdHVyZXMgY29ycmVsYXRlZCB3aXRoIHJlY29tYmluYXRpb24gcmF0ZTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz01MTAtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgcmVjb21iaW5hdGlvbiByYXRlcyBpbiB0aGUgcmF0LCBtb3VzZSwgYW5kIGh1bWFuIGdlbm9tZXM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9NTI4LTM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZWlvdGljIHJlY29tYmluYXRpb24gaG90IHNwb3RzIGFuZCBjb2xkIHNwb3RzOyBOYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0zNjAtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UFJETTkgaXMgYSBtYWpvciBkZXRlcm1pbmFudCBvZiBtZWlvdGljIHJlY29tYmluYXRpb24gaG90c3BvdHMgaW4gaHVtYW5zIGFuZCBtaWNlOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTMyNzsgIGNpdGF0aW9uX3BhZ2VzPTgzNi00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RHJpdmUgYWdhaW5zdCBob3RzcG90IG1vdGlmcyBpbiBwcmltYXRlcyBpbXBsaWNhdGVzIHRoZSBQUkRNOSBnZW5lIGluIG1laW90aWMgcmVjb21iaW5hdGlvbjsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0zMjc7ICBjaXRhdGlvbl9wYWdlcz04NzYtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJkbTkgY29udHJvbHMgYWN0aXZhdGlvbiBvZiBtYW1tYWxpYW4gcmVjb21iaW5hdGlvbiBob3RzcG90czsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0zMjc7ICBjaXRhdGlvbl9wYWdlcz04MzUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXgtc3BlY2lmaWMgcmVjb21iaW5hdGlvbiByYXRlcyBpbiB6ZWJyYWZpc2ggKERhbmlvIHJlcmlvKTsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTYwOyAgY2l0YXRpb25fcGFnZXM9NjQ5LTU3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG1pY3Jvc2F0ZWxsaXRlIGxpbmthZ2UgbWFwIG9mIHRoZSBFdXJvcGVhbiBzZWEgYmFzcyBEaWNlbnRyYXJjaHVzIGxhYnJheCBMOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNzA7ICBjaXRhdGlvbl9wYWdlcz0xODIxLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgc2Vjb25kIGdlbmVyYXRpb24gZ2VuZXRpYyBsaW5rYWdlIG1hcCBmb3Igc2lsdmVyIGNhcnAgKEh5cG9waHRoYWxtaWNoZWh5cyBtb2xpdHJpeCkgdXNpbmcgbWljcm9zYXRlbGxpdGUgbWFya2VyczsgQXF1YWN1bHR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDEyOyAgY2l0YXRpb25fcGFnZXM9OTctMTA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGdlbmV0aWMgbGlua2FnZSBtYXAgZm9yIEFyY3RpYyBjaGFyIChTYWx2ZWxpbnVzIGFscGludXMpOiBldmlkZW5jZSBmb3IgaGlnaGVyIHJlY29tYmluYXRpb24gcmF0ZXMgYW5kIHNlZ3JlZ2F0aW9uIGRpc3RvcnRpb24gaW4gaHlicmlkIHZlcnN1cyBwdXJlIHN0cmFpbiBtYXBwaW5nIHBhcmVudHM7IEdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT00NzsgIGNpdGF0aW9uX3BhZ2VzPTMwNC0xNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb25zZW5zdXMgbGlua2FnZSBtYXAgb2YgdGhlIGdyYXNzIGNhcnAgKEN0ZW5vcGhhcnluZ29kb24gaWRlbGxhKSBiYXNlZCBvbiBtaWNyb3NhdGVsbGl0ZXMgYW5kIFNOUHM7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTEzNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlY29tYmluYXRpb24gZGlmZmVyZW5jZSBiZXR3ZWVuIHNleGVzOiBhIHJvbGUgZm9yIGhhcGxvaWQgc2VsZWN0aW9uOyBQTG9TIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0zOTYtNDAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DeXRvbG9naWNhbCBzdHVkaWVzIG9mIGh1bWFuIG1laW9zaXM6IHNleC1zcGVjaWZpYyBkaWZmZXJlbmNlcyBpbiByZWNvbWJpbmF0aW9uIG9yaWdpbmF0ZSBhdCwgb3IgcHJpb3IgdG8sIGVzdGFibGlzaG1lbnQgb2YgZG91YmxlLXN0cmFuZCBicmVha3M7IFBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lODUwNzUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjIvMS8zOS8xMjA4MTM0L2RzdTAzOC5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgQ29uc3RydWN0aW9uIG9mIGdlbmV0aWMgbGlua2FnZSBtYXAgaXMgZXNzZW50aWFsIGZvciBnZW5ldGljIGFuZCBnZW5vbWljIHN0dWRpZXMuIFJlY2VudCBhZHZhbmNlcyBpbiBzZXF1ZW5jaW5nIGFuZCBnZW5vdHlwaW5nIHRlY2hub2xvZ2llcyBtYWRlIGl0IHAiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQ0MDQyOCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGRKqQAmci2LOL5sVg/NDca1vGnmCg==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f39%2f440428" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f39%2f440428" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=17206" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/1/1/m_cover.gif?Expires=2147483647&amp;Signature=Vhf6NToQrkVldazSweShdrlrfCnUbBpTokrfu8JXGVRMaoyIjioben46GoxM3BSEoRr~te73o-dfY7QEM91wVumB~VX8hAxN9ywAZ1zRAbXXmtU23LiP8gcw~y1-D8ACKe3CZXmetX85Oj4q~Op8xej8CbOs0MGBcAElTMf~GIynbEl8oPRbKlaZUEBETFvDqCw1PlymDq~5degwzATpcv5kHJhRgAnT8PjHfQEqeZxMFeCt-B-phqwpjq5AMA0hAnSz5a5U5SMfCC3Naxv-~~yWr1GxeMNVTp3iBMlBqhdGNwFRsnBBquWLbTiF47z2gcE0wubs8CNqU4CX2qcpAA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/1">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6578173">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6578173">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6578178">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6578178">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6578204">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6578204">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6578242">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6578242">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6578279">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6578279">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6578285">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6578285">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6578287">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6578287">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6578289">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6578289">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6578291">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6578291">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/22/1/29/440058">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/22/1/53/441064">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Construction of a high-density, high-resolution genetic map and its integration with BAC-based physical map in channel catfish
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yun Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">The Fish Molecular Genetics and Biotechnology Laboratory, School of Fisheries, Aquaculture, and Aquatic Sciences and Program of Cell and Molecular Biosciences, Aquatic Genomics Unit</div>, <div class="insititution">Auburn University</div>, <div class="addr-line">Auburn, AL 36849</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yun+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Li Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yun Li</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shikai Liu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">The Fish Molecular Genetics and Biotechnology Laboratory, School of Fisheries, Aquaculture, and Aquatic Sciences and Program of Cell and Molecular Biosciences, Aquatic Genomics Unit</div>, <div class="insititution">Auburn University</div>, <div class="addr-line">Auburn, AL 36849</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shikai+Liu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liu S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Liu S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shikai Liu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Zhenkui Qin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">The Fish Molecular Genetics and Biotechnology Laboratory, School of Fisheries, Aquaculture, and Aquatic Sciences and Program of Cell and Molecular Biosciences, Aquatic Genomics Unit</div>, <div class="insititution">Auburn University</div>, <div class="addr-line">Auburn, AL 36849</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Zhenkui+Qin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Qin Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Qin Z&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Zhenkui Qin</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Geoff Waldbieser</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">USDA-ARS Warmwater Aquaculture Research Unit</div>, <div class="addr-line">Stoneville, MS 38776</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Geoff+Waldbieser">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Waldbieser G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Waldbieser G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Geoff Waldbieser</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ruijia Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">The Fish Molecular Genetics and Biotechnology Laboratory, School of Fisheries, Aquaculture, and Aquatic Sciences and Program of Cell and Molecular Biosciences, Aquatic Genomics Unit</div>, <div class="insititution">Auburn University</div>, <div class="addr-line">Auburn, AL 36849</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ruijia+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wang R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ruijia Wang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Luyang Sun</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">The Fish Molecular Genetics and Biotechnology Laboratory, School of Fisheries, Aquaculture, and Aquatic Sciences and Program of Cell and Molecular Biosciences, Aquatic Genomics Unit</div>, <div class="insititution">Auburn University</div>, <div class="addr-line">Auburn, AL 36849</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Luyang+Sun">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sun L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sun L&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Luyang Sun</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Lisui Bao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">The Fish Molecular Genetics and Biotechnology Laboratory, School of Fisheries, Aquaculture, and Aquatic Sciences and Program of Cell and Molecular Biosciences, Aquatic Genomics Unit</div>, <div class="insititution">Auburn University</div>, <div class="addr-line">Auburn, AL 36849</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Lisui+Bao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bao L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Bao L&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Lisui Bao</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Roy G. Danzmann</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Integrative Biology</div>, <div class="insititution">University of Guelph</div>, <div class="addr-line">Guelph, Ontario N1G 2W1</div>, <div class="country">Canada</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Roy+G.+Danzmann">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Danzmann R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Danzmann R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Roy G. Danzmann</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rex Dunham</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">The Fish Molecular Genetics and Biotechnology Laboratory, School of Fisheries, Aquaculture, and Aquatic Sciences and Program of Cell and Molecular Biosciences, Aquatic Genomics Unit</div>, <div class="insititution">Auburn University</div>, <div class="addr-line">Auburn, AL 36849</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Rex+Dunham">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Dunham R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Dunham R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Rex Dunham</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Zhanjiang Liu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">The Fish Molecular Genetics and Biotechnology Laboratory, School of Fisheries, Aquaculture, and Aquatic Sciences and Program of Cell and Molecular Biosciences, Aquatic Genomics Unit</div>, <div class="insititution">Auburn University</div>, <div class="addr-line">Auburn, AL 36849</div>, <div class="country">USA</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +1 334-844-4054. Fax. +1 334-844-9208. E-mail: <a href="mailto:liuzhan@auburn.edu" target="_blank">liuzhan@auburn.edu</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Zhanjiang+Liu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liu Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Liu Z&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Zhanjiang Liu</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN2"><div class="fn"><div class="footnote-content">Edited by Dr Toshihiko Shiroishi</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 1, 1 February 2015, Pages 39–52, <a href='https://doi.org/10.1093/dnares/dsu038'>https://doi.org/10.1093/dnares/dsu038</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">26 November 2014</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">19 September 2014</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">21 October 2014</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="440428" href="/dnaresearch/article-pdf/22/1/39/1208134/dsu038.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Yun Li, Shikai Liu, Zhenkui Qin, Geoff Waldbieser, Ruijia Wang, Luyang Sun, Lisui Bao, Roy G. Danzmann, Rex Dunham, Zhanjiang Liu;  Construction of a high-density, high-resolution genetic map and its integration with BAC-based physical map in channel catfish, <em>DNA Research</em>, Volume 22, Issue 1, 1 February 2015, Pages 39–52, <a href='https://doi.org/10.1093/dnares/dsu038'>https://doi.org/10.1093/dnares/dsu038</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/440428?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/440428?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/440428?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/440428?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/440428?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/440428?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/440428?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Construction%20of%20a%20high-density%2C%20high-resolution%20genetic%20map%20and%20its%20integration%20with%20BAC-based%20physical%20map%20in%20channel%20catfish&amp;publicationDate=2014-11-26&amp;volumeNum=22&amp;issueNum=1&amp;author=Li%2C%20Yun%3B%20Liu%2C%20Shikai&amp;startPage=39&amp;endPage=52&amp;contentId=10.1093%2Fdnares%2Fdsu038&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f39%2f440428" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6578173 id="6578173" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Construction of genetic linkage map is essential for genetic and genomic studies. Recent advances in sequencing and genotyping technologies made it possible to generate high-density and high-resolution genetic linkage maps, especially for the organisms lacking extensive genomic resources. In the present work, we constructed a high-density and high-resolution genetic map for channel catfish with three large resource families genotyped using the catfish 250K single-nucleotide polymorphism (SNP) array. A total of 54,342 SNPs were placed on the linkage map, which to our knowledge had the highest marker density among aquaculture species. The estimated genetic size was 3,505.4 cM with a resolution of 0.22 cM for sex-averaged genetic map. The sex-specific linkage maps spanned a total of 4,495.1 cM in females and 2,593.7 cM in males, presenting a ratio of 1.7 : 1 between female and male in recombination fraction. After integration with the previously established physical map, over 87% of physical map contigs were anchored to the linkage groups that covered a physical length of 867 Mb, accounting for ∼90% of the catfish genome. The integrated map provides a valuable tool for validating and improving the catfish whole-genome assembly and facilitates fine-scale QTL mapping and positional cloning of genes responsible for economically important traits.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="SNP">SNP</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="catfish">catfish</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;linkage map&quot;">linkage map</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;physical map&quot;">physical map</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="genome">genome</a></div>                            <h2 scrollto-destination=6578178 id="6578178" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Genetic linkage maps are essential for the understanding of genomic levels of organization of inheritance of traits.<sup><a reveal-id="DSU038C1" data-open="DSU038C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Construction of high-density and high-resolution genetic maps is a key step for fine mapping of quantitative trait loci (QTL) and marker-assisted selection. In addition, genetic linkage maps are valuable resources for the generation of chromosome-level assembly of whole-genome sequences and for comparative genome analysis.<sup><a reveal-id="DSU038C2" data-open="DSU038C2" class="link link-ref link-reveal xref-bibr">2</a></sup> In most of the recent whole-genome sequencing cases, whole-genome sequences are generated using next-generation sequencing (NGS). Short sequence reads are assembled into contigs. Such contigs are generally still relatively short although they vary in sizes from several kilobases to tens of kilobases. Increases in genome sequencing coverage and sequencing libraries can increase the quality of the assembly, allowing the sizes of contigs to be increased. However, NGS methods alone cannot provide the resources to assemble complex genomes at the chromosomal level. Genome sequence assemblies at this level require the assembly of tens of thousands to hundreds of thousands of contigs.</p><p>Highly segmented genome assemblies prohibit efficient genome analysis. Therefore, various genome resources have been created to reduce the segmentation of the genome assemblies. One of these resources is the large-insert-based physical maps. Historically, several types of large insert libraries have been used. These include yeast artificial chromosomes (YACs),<sup><a reveal-id="DSU038C3" data-open="DSU038C3" class="link link-ref link-reveal xref-bibr">3</a></sup> bacterial artificial chromosomes (BACs),<sup><a reveal-id="DSU038C4" data-open="DSU038C4" class="link link-ref link-reveal xref-bibr">4</a></sup> and cosmid-based libraries.<sup><a reveal-id="DSU038C5" data-open="DSU038C5" class="link link-ref link-reveal xref-bibr">5</a></sup> YACs have the largest capacity for cloning the large inserts, but they are relatively unstable; therefore, their use in genome studies has been limited. Cosmid libraries have the smallest capacity for cloning the large inserts; therefore, their use in large insert libraries has also been limited. BACs are the most popular large insert libraries as they are stable and can hold inserts of up to 200 kb. BAC-based physical maps can organize the entire genome into restriction fingerprint-based contigs. Such contigs are similar to the whole-genome sequencing contigs, but they are constructed using overlapping restriction enzyme fingerprints rather than overlapping sequences themselves for the whole-genome sequence assemblies. By analysis of restriction fingerprints of overlapping genomic clones of BAC inserts, the whole genome can be organized into a limited number of contigs, most often in thousands. For instance, the catfish physical maps had 3,307 contigs and 1,891 contigs.<sup><a reveal-id="DSU038C6" data-open="DSU038C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSU038C7" data-open="DSU038C7" class="link link-ref link-reveal xref-bibr">7</a></sup> The integrated catfish physical map included over 2,500 contigs (unpublished).</p><p>Integration of physical maps and whole-genome sequence contigs allows the relationship to be established between the sequence-based contigs and the restriction fingerprint-based contigs, thereby reducing the levels of segmentation of the genome. One of the major applications of genetic linkage map is to integrate physical maps and whole-genome assemblies. Integration of genetic map with physical map is useful for understanding genomes from different dimensions and is essential for comparative genome analysis,<sup><a reveal-id="DSU038C8" data-open="DSU038C8" class="link link-ref link-reveal xref-bibr">8</a></sup> fine-scale QTL mapping and positional cloning of genes responsible for performance and production traits.<sup><a reveal-id="DSU038C9" data-open="DSU038C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSU038C10" data-open="DSU038C10" class="link link-ref link-reveal xref-bibr">10</a></sup> In aquaculture fish species, genetic maps have been constructed in a few species, such as Asian seabass,<sup><a reveal-id="DSU038C11" data-open="DSU038C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Atlantic salmon,<sup><a reveal-id="DSU038C12" data-open="DSU038C12" class="link link-ref link-reveal xref-bibr">12</a></sup> half-smooth tongue sole,<sup><a reveal-id="DSU038C13" data-open="DSU038C13" class="link link-ref link-reveal xref-bibr">13</a></sup> rainbow trout,<sup><a reveal-id="DSU038C14" data-open="DSU038C14" class="link link-ref link-reveal xref-bibr">14</a></sup> common carp,<sup><a reveal-id="DSU038C15" data-open="DSU038C15" class="link link-ref link-reveal xref-bibr">15</a></sup> and catfish.<sup><a reveal-id="DSU038C2" data-open="DSU038C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSU038C9" data-open="DSU038C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSU038C16" data-open="DSU038C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSU038C17" data-open="DSU038C17" class="link link-ref link-reveal xref-bibr">17</a></sup> These maps harbour several hundred to a couple of thousands of markers, with which QTL for agriculturally important performance and production traits can only be mapped in large genomic regions.</p><p>Integrated maps have been developed in several aquaculture fish species using low-density genetic maps. In Atlantic salmon, in addition to 579 BAC contigs that were integrated into the linkage map using microsatellite markers, identification and mapping of new BAC-anchored single-nucleotide polymorphism (SNP) markers from BAC-end sequences placed 73 additional BAC contigs to Atlantic salmon linkage groups.<sup><a reveal-id="DSU038C18" data-open="DSU038C18" class="link link-ref link-reveal xref-bibr">18</a></sup> The second generation of rainbow trout integrated map anchored up to 265 contigs to the genetic map, covering ∼11% of the genome.<sup><a reveal-id="DSU038C10" data-open="DSU038C10" class="link link-ref link-reveal xref-bibr">10</a></sup> In common carp, a total of 463 physical map contigs and 88 single BACs were integrated into the genetic linkage map, which covered 30% of the common carp genome.<sup><a reveal-id="DSU038C15" data-open="DSU038C15" class="link link-ref link-reveal xref-bibr">15</a></sup> In catfish, 2,030 BAC-end sequence (BES)-derived microsatellites from 1,481 physical map contigs were used for linkage map construction, which anchored 44.8% of the catfish BAC physical map contigs, covering ∼52.8% of the genome.<sup><a reveal-id="DSU038C9" data-open="DSU038C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Apparently, the level of integration is dependent on the density and resolution of the genetic maps.</p><p>One objective of this study was to construct a high-density and high-resolution genetic linkage map by using a large number of molecular markers, covering the entire genome and a large resource panel for linkage mapping analysis. In addition to this primary goal, a secondary goal was to increase integration of the linkage and physical maps, using markers derived from BECs for genetic linkage analysis. Our previous genetic linkage analysis used mostly microsatellite markers. In spite of their high polymorphism, and low cost of single marker genotyping, an analysis of tens of thousands of microsatellites with a large number of mapping fish is extremely labourious and cost-prohibitive. SNPs overcome these difficulties by providing high efficiency and low-cost, large-scale genotyping. SNPs have become markers of choice because of their abundance, even genomic distribution, and easy adaptation to automation.<sup><a reveal-id="DSU038C19" data-open="DSU038C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Recent advances in NGS have allowed rapid discovery of genome-wide SNPs in any organism in a cost-effective manner.<sup><a reveal-id="DSU038C20" data-open="DSU038C20" class="link link-ref link-reveal xref-bibr">20</a></sup> With the availability of large numbers of SNPs, high-density SNP array platform can be developed for high-throughput and efficient genotyping. Alternatively, an NGS-based genotyping-by-sequencing (GBS)<sup><a reveal-id="DSU038C19  DSU038C21" data-open="DSU038C19  DSU038C21" class="link link-ref link-reveal xref-bibr">19–21</a></sup> is also highly efficient. SNP arrays and GBS have both been used to genotype large-scale SNPs for genetic mapping and association analyses.<sup><a reveal-id="DSU038C12" data-open="DSU038C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSU038C22  DSU038C28" data-open="DSU038C22  DSU038C28" class="link link-ref link-reveal xref-bibr">22–28</a></sup> Compared with GBS, SNP arrays are more cost-effective. In addition, they provide a greater level of genome coverage. For instance, in most cases, the total number of commonly analysed SNPs is limited to several thousands while millions of SNPs can be analysed by using very high-density SNP arrays. In addition, GBS of large resource panels for high-resolution maps is cost-prohibitive.</p><p>Channel catfish, <em>Ictalurus punctatus,</em> is the primary aquaculture species in the United States. Since the initiation of genome research over two decades ago, several genetic maps have been constructed with different types of molecular markers and various resource families.<sup><a reveal-id="DSU038C2" data-open="DSU038C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSU038C9" data-open="DSU038C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSU038C16" data-open="DSU038C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSU038C17" data-open="DSU038C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Up to date, the highest density map was developed to contain 2,030 microsatellites and 100 SNPs.<sup><a reveal-id="DSU038C9" data-open="DSU038C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Although this genetic map has been very useful for genetic and genomic analysis,<sup><a reveal-id="DSU038C29" data-open="DSU038C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DSU038C30" data-open="DSU038C30" class="link link-ref link-reveal xref-bibr">30</a></sup> marker density in this map was still fairly low and only facilitated integration to the physical map for ∼52% of catfish genome. Recently, following efforts to expand catfish genomic resources, we have identified millions of SNPs in channel catfish<sup><a reveal-id="DSU038C31" data-open="DSU038C31" class="link link-ref link-reveal xref-bibr">31</a>,<a reveal-id="DSU038C32" data-open="DSU038C32" class="link link-ref link-reveal xref-bibr">32</a></sup> and developed high-density SNP arrays,<sup><a reveal-id="DSU038C33" data-open="DSU038C33" class="link link-ref link-reveal xref-bibr">33</a></sup> which provided the opportunity to develop a high-density and high-resolution SNP-based genetic map. Here we report the construction of genetic linkage map with over 50,000 SNPs in channel catfish, which is, to the best of our knowledge, the highest density genetic map for any aquaculture species. Along with the high density of markers, the utilization of BAC-associated SNPs also allowed significant increase of the integration of the genetic linkage map with the BAC-based physical map in catfish, allowing over 90% of the catfish genome physical map contigs to be mapped to linkage groups. This genetic map should serve as a valuable framework for validating the reference whole-genome sequences, extensive comparative and functional genomic studies, and fine-scale QTL mapping and association studies in catfish.</p>                            <h2 scrollto-destination=6578204 id="6578204" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=6578206 id="6578206" class="section-title" data-legacy-id=s2a>2.1. Resource families and DNA preparation</h3>
<p>A total of 576 fish, with 192 full-sibling individuals from each of three full-sibling channel catfish families, were used for linkage mapping. The parent and grandparent samples were also obtained. All DNAs for these samples used in this study were provided by USDA-ARS Warmwater Aquaculture Research Unit, which were prepared following the procedures as previously described.<sup><a reveal-id="DSU038C17" data-open="DSU038C17" class="link link-ref link-reveal xref-bibr">17</a></sup></p>                            <h3 scrollto-destination=6578212 id="6578212" class="section-title" data-legacy-id=s2b>2.2. SNP genotyping</h3>
<p>DNA samples were arranged in 96-well microtitre plates and diluted to a final concentration of 50 ng/µl with the final volume of 10 µl. Genotyping was conducted by GeneSeek (Lincoln, NE, USA) using the catfish 250K SNP array.<sup><a reveal-id="DSU038C33" data-open="DSU038C33" class="link link-ref link-reveal xref-bibr">33</a></sup> Affymetrix CEL files were analysed using Affymetrix Genotyping Console software (version 4.0) for quality control analysis and SNP genotype calling using the Affymetrix AxiomGT1 algorithm. Samples passing the quality control (Dish value &gt; 0.85) and SNP call rates threshold (&gt;95%) were retained for analysis. Following genotyping, an R package, SNPolisher, was used to generate the genotyping outputs. The package produced SNP quality control metrics and divided SNPs into several classes based on the quality of genotype calls.<sup><a reveal-id="DSU038C33" data-open="DSU038C33" class="link link-ref link-reveal xref-bibr">33</a></sup> The polymorphic markers with high resolution of cluster separation (high genotyping quality) were remained for further analysis. The CHP files generated from the Affymetrix Genotyping Console were imported into SNP &amp; Variation Suite (SVS version 7, Golden Helix, Inc.) for further filtering to remove SNPs with missing genotypes &gt;10% and minor allele frequency &lt;5%.</p>                            <h3 scrollto-destination=6578217 id="6578217" class="section-title" data-legacy-id=s2c>2.3. Linkage map construction</h3>
<p>Only SNPs that had high quality of genotype calls and were heterozygous in at least one parent were retained for linkage analysis. Based on segregation patterns, SNPs were categorized into three types: 1 : 2 : 1 type (AB × AB, segregating in both parents), 1 : 1 type (AB × AA or AB × BB, segregating only in female), and 1 : 1 type (AA × AB or BB × AB, segregating only in male). The <em>χ</em><sup>2</sup> goodness-of-fit tests were performed to examine the segregation distortion. Markers with significant segregation distortion were excluded (<em>P</em> &lt; 0.001).</p><p>Linkage maps were initially developed independently for the three mapping families. Sex-specific maps were constructed for each parent using 1 : 1 segregation-type markers only (AB × AA or AB × BB for female map and AA × AB or BB × AB for male map). The markers of segregation type 1 : 2 : 1 (AB × AB) were used to build the sex-averaged map. The LINKMFEX software (version 2.4, <a class="link link-uri" href="http://www.uoguelph.ca/~rdanzman/software.htm">http://www.uoguelph.ca/~rdanzman/software.htm</a>) was used to perform linkage analysis for sex-specific markers, and OneMap (R package, version 2.0) was used for linkage analysis of sex-averaged markers. Linkage between markers was examined by estimating logarithm of the odds (LOD) scores for recombination fraction (<em>θ</em>). A LOD threshold of 8.0 was used, and a maximum <em>θ</em> of 0.35 was set to assign markers into linkage groups.</p><p>For markers falling into ‘zero recombination clusters', only one marker was picked that had the most informative meiosis as the representative marker of each cluster for linkage map construction to reduce the power required for computation of linkage. Genetic maps were constructed using JoinMap software version 4.0 with the regression mapping algorithm.<sup><a reveal-id="DSU038C34" data-open="DSU038C34" class="link link-ref link-reveal xref-bibr">34</a></sup> The Kosambi mapping function was used to convert the recombination frequencies into map distances (centiMorgans). The positions of markers were determined according to the sequential build-up of the map.<sup><a reveal-id="DSU038C35" data-open="DSU038C35" class="link link-ref link-reveal xref-bibr">35</a></sup> Briefly, a pair of markers was firstly selected, followed by sequential addition of other markers. The ‘ripple’ was performed each time after adding one marker. The best fitting position of the added markers was searched based on the goodness-of-fit test for the resulting map. When a marker generated a negative map distance or a large ‘jump’ value in goodness-of-fit test, the marker was removed, and map calculation was continued to construct a first-round map. Thereafter, the removed markers were added to the first-round map and again subjected to the goodness-of-fit test to generate an optimum order of markers. For the sex-specific markers, the linkage phases were inferred automatically, while for the markers with 1 : 2 : 1 segregation type (AB × AB), the linkage phase was deduced based on the genotypes of grandparents to assist the map construction.</p><p>The consensus maps were then established using the MergeMap<sup><a reveal-id="DSU038C36" data-open="DSU038C36" class="link link-ref link-reveal xref-bibr">36</a></sup> by integrating individual maps from three reference families through shared markers. Lastly, the markers falling into ‘zero recombination clusters’ that were excluded during map construction were anchored to the linkage map, based on the positions of their corresponding representative markers. All genetic linkage maps were drawn using MAPCHART 2.2.<sup><a reveal-id="DSU038C37" data-open="DSU038C37" class="link link-ref link-reveal xref-bibr">37</a></sup></p>                            <h3 scrollto-destination=6578230 id="6578230" class="section-title" data-legacy-id=s2d>2.4. Differences in recombination rates between families and sexes</h3>
<div class="block-child-p">To assess differences in recombination rates among the three resource families, the sex-averaged map was used following the M-test according to Ott's method.<sup><a reveal-id="DSU038C38" data-open="DSU038C38" class="link link-ref link-reveal xref-bibr">38</a></sup><a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msup><mi>X</mi><mn>2</mn></msup></mrow><mo xmlns="">=</mo><mn xmlns="">2</mn><mo xmlns="">×</mo><mtext xmlns="">ln(10)</mtext><mrow xmlns=""><mo>[</mo><mrow><mo>∑</mo><mrow><mrow><msub><mi>Z</mi><mi>i</mi></msub></mrow><mo stretchy="false">(</mo><mrow><msub><mrow><mrow><mover><mi>θ</mi><mo stretchy="false">ˆ</mo></mover></mrow></mrow><mi>i</mi></msub></mrow><mo stretchy="false">)</mo><mo>−</mo></mrow><mi>Z</mi><mo stretchy="false">(</mo><mrow><mover><mi>θ</mi><mo stretchy="false">ˆ</mo></mover></mrow><mo stretchy="false">)</mo></mrow><mo>]</mo></mrow></math></div></div></div> where <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>Z</mi><mi>i</mi></msub></mrow><mo stretchy="false" xmlns="">(</mo><mrow xmlns=""><msub><mrow><mover><mi>θ</mi><mo stretchy="false">ˆ</mo></mover></mrow><mi>i</mi></msub></mrow><mo stretchy="false" xmlns="">)</mo></math></span> represents the LOD scores of maximum-likelihood estimation (MLE) for the <em>i</em>th reference family for common marker pairs among the three families. The <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">Z</mi><mo stretchy="false" xmlns="">(</mo><mrow xmlns=""><mover><mi>θ</mi><mo stretchy="false">ˆ</mo></mover></mrow><mo stretchy="false" xmlns="">)</mo></math></span> represents the total LOD scores of MLE for all three families. The recombination fractions for all mapped locus intervals from each family were obtained from JoinMap.</div><p>To investigate sex-specific heterogeneity throughout each linkage group, common marker pairs were used to compare the locus intervals across the male-specific map and female-specific map using contingency G-test.<sup><a reveal-id="DSU038C39" data-open="DSU038C39" class="link link-ref link-reveal xref-bibr">39</a></sup></p>                            <h3 scrollto-destination=6578234 id="6578234" class="section-title" data-legacy-id=s2e>2.5. Integration of linkage map with physical map</h3>
<p>To integrate previously developed physical map<sup><a reveal-id="DSU038C40" data-open="DSU038C40" class="link link-ref link-reveal xref-bibr">40</a></sup> with the high-density linkage map constructed in the present work, all the mapped SNPs with 70-bp flanking sequences were aligned with BES<sup><a reveal-id="DSU038C8" data-open="DSU038C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSU038C40" data-open="DSU038C40" class="link link-ref link-reveal xref-bibr">40</a></sup> and BAC-based physical map contig-specific sequences (PMCSS)<sup><a reveal-id="DSU038C41" data-open="DSU038C41" class="link link-ref link-reveal xref-bibr">41</a></sup> using BLAST with the <em>E</em>-value cut-off 1E−10, minimal alignment length of 36, and minimal identity of 95%. The relationship of physical map and genetic map was built up based on the SNP-associated BES and PMCSS. The BAC physical map contigs that harboured BAC-derived BES and PMCSS were anchored onto the genetic map based on the SNP positions.</p>                            <h2 scrollto-destination=6578242 id="6578242" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=6578245 id="6578245" class="section-title" data-legacy-id=s3a>3.1. Selection of SNP markers</h3>
<p>As summarized in Table <a reveal-id="DSU038TB1" data-open="DSU038TB1" class="link link-reveal link-table xref-fig">1</a>, SNP genotypes were obtained from 576 samples of three mapping families (192 samples per family). According to the assessment of genotyping quality and polymorphism in all samples from the three mapping families, genotypes of a total of 121,521 SNPs were used (Table <a reveal-id="DSU038TB1" data-open="DSU038TB1" class="link link-reveal link-table xref-fig">1</a>). Owing to the low genotype calling rate (&lt;95%), nine individuals were excluded for analysis, including one individual from family 1, three individuals from family 2, and five individuals from family 3. After further filtering to remove SNPs with low calling rate (&lt;97%) and non-Mendelian inheritance (<em>P</em> &lt; 0.001), a total of 18,866 SNPs from family 1, 33,224 SNPs from family 2, and 35,093 SNPs from family 3 were retained for linkage mapping. Taken together, a total of 64,889 SNPs that were informative in at least one of the three mapping families were used. </p>                        <a id="6578249" scrollto-destination="6578249"></a>
<div content-id="DSU038TB1" class="table-modal"><div class="table-wrap"><div id="DSU038TB1" class="table-wrap-title" data-id="DSU038TB1"><div class="label -label">Table 1.</div><div class="caption"><p>SNPs selected for linkage mapping</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Item </th><th>Number </th></tr></thead><tbody><tr><td>SNPs on array </td><td>250,113 </td></tr><tr><td>SNPs polymorphic and with high genotyping quality </td><td>121,521 </td></tr><tr><td>SNPs used for linkage mapping after further filtering </td><td>64,889 </td></tr><tr><td>SNPs from family 1 used for linkage mapping </td><td>18,866 </td></tr><tr><td>SNPs from family 2 used for linkage mapping </td><td>33,224 </td></tr><tr><td>SNPs from family 3 used for linkage mapping </td><td>35,093 </td></tr></tbody></table></div><div content-id="DSU038TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Item </th><th>Number </th></tr></thead><tbody><tr><td>SNPs on array </td><td>250,113 </td></tr><tr><td>SNPs polymorphic and with high genotyping quality </td><td>121,521 </td></tr><tr><td>SNPs used for linkage mapping after further filtering </td><td>64,889 </td></tr><tr><td>SNPs from family 1 used for linkage mapping </td><td>18,866 </td></tr><tr><td>SNPs from family 2 used for linkage mapping </td><td>33,224 </td></tr><tr><td>SNPs from family 3 used for linkage mapping </td><td>35,093 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6578249" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU038TB1" class="table-wrap-title" data-id="DSU038TB1"><div class="label -label">Table 1.</div><div class="caption"><p>SNPs selected for linkage mapping</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Item </th><th>Number </th></tr></thead><tbody><tr><td>SNPs on array </td><td>250,113 </td></tr><tr><td>SNPs polymorphic and with high genotyping quality </td><td>121,521 </td></tr><tr><td>SNPs used for linkage mapping after further filtering </td><td>64,889 </td></tr><tr><td>SNPs from family 1 used for linkage mapping </td><td>18,866 </td></tr><tr><td>SNPs from family 2 used for linkage mapping </td><td>33,224 </td></tr><tr><td>SNPs from family 3 used for linkage mapping </td><td>35,093 </td></tr></tbody></table></div><div content-id="DSU038TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Item </th><th>Number </th></tr></thead><tbody><tr><td>SNPs on array </td><td>250,113 </td></tr><tr><td>SNPs polymorphic and with high genotyping quality </td><td>121,521 </td></tr><tr><td>SNPs used for linkage mapping after further filtering </td><td>64,889 </td></tr><tr><td>SNPs from family 1 used for linkage mapping </td><td>18,866 </td></tr><tr><td>SNPs from family 2 used for linkage mapping </td><td>33,224 </td></tr><tr><td>SNPs from family 3 used for linkage mapping </td><td>35,093 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6578249" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=6578251 id="6578251" class="section-title" data-legacy-id=s3b>3.2. Linkage mapping</h3>
<p>Linkage maps were first constructed for each of the three families, separately. For all three mapping families, 29 linkage groups (LGs) were obtained, which was consistent with the number of chromosomes of the catfish haploid genome. The consensus genetic linkage maps were obtained by merging separate maps from three mapping families. A total of 31,387 markers with distinct genetic positions (hereafter referred to as unique markers) were placed on the consensus linkage maps, which included 8,644 SNPs from family 1, 13,477 SNPs from family 2, and 14,343 SNPs from family 3 (Table <a reveal-id="DSU038TB2" data-open="DSU038TB2" class="link link-reveal link-table xref-fig">2</a>). After anchoring the previously excluded markers that fell into ‘zero recombination clusters’ based on the genetic positions of representative markers, a total of 54,342 SNP markers were placed onto the current linkage map (Table <a reveal-id="DSU038TB2" data-open="DSU038TB2" class="link link-reveal link-table xref-fig">2</a>). </p>                        <a id="6578255" scrollto-destination="6578255"></a>
<div content-id="DSU038TB2" class="table-modal"><div class="table-wrap"><div id="DSU038TB2" class="table-wrap-title" data-id="DSU038TB2"><div class="label -label">Table 2.</div><div class="caption"><p>SNPs placed on the linkage maps</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Item </th><th>Number </th></tr></thead><tbody><tr><td>Unique SNPs from family 1 mapped to linkage map </td><td>8,644 </td></tr><tr><td>Unique SNPs from family 2 mapped to linkage map </td><td>13,477 </td></tr><tr><td>Unique SNPs from family 3 mapped to linkage map </td><td>14,343 </td></tr><tr><td>Total unique SNPs mapped to linkage map </td><td>31,387 </td></tr><tr><td>Total SNPs mapped to linkage map </td><td>54,342 </td></tr></tbody></table></div><div content-id="DSU038TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Item </th><th>Number </th></tr></thead><tbody><tr><td>Unique SNPs from family 1 mapped to linkage map </td><td>8,644 </td></tr><tr><td>Unique SNPs from family 2 mapped to linkage map </td><td>13,477 </td></tr><tr><td>Unique SNPs from family 3 mapped to linkage map </td><td>14,343 </td></tr><tr><td>Total unique SNPs mapped to linkage map </td><td>31,387 </td></tr><tr><td>Total SNPs mapped to linkage map </td><td>54,342 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6578255" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU038TB2" class="table-wrap-title" data-id="DSU038TB2"><div class="label -label">Table 2.</div><div class="caption"><p>SNPs placed on the linkage maps</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Item </th><th>Number </th></tr></thead><tbody><tr><td>Unique SNPs from family 1 mapped to linkage map </td><td>8,644 </td></tr><tr><td>Unique SNPs from family 2 mapped to linkage map </td><td>13,477 </td></tr><tr><td>Unique SNPs from family 3 mapped to linkage map </td><td>14,343 </td></tr><tr><td>Total unique SNPs mapped to linkage map </td><td>31,387 </td></tr><tr><td>Total SNPs mapped to linkage map </td><td>54,342 </td></tr></tbody></table></div><div content-id="DSU038TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Item </th><th>Number </th></tr></thead><tbody><tr><td>Unique SNPs from family 1 mapped to linkage map </td><td>8,644 </td></tr><tr><td>Unique SNPs from family 2 mapped to linkage map </td><td>13,477 </td></tr><tr><td>Unique SNPs from family 3 mapped to linkage map </td><td>14,343 </td></tr><tr><td>Total unique SNPs mapped to linkage map </td><td>31,387 </td></tr><tr><td>Total SNPs mapped to linkage map </td><td>54,342 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6578255" target="_blank">View Large</a></div></div><p>The sex-specific maps were constructed using markers that were heterozygous in only female or male parent. The female genetic map consisted of 18,444 SNPs including 9,746 unique markers, with a total genetic length of 4,495.1 cM (Table <a reveal-id="DSU038TB3" data-open="DSU038TB3" class="link link-reveal link-table xref-fig">3</a>; Fig. <a reveal-id="DSU038F1" data-open="DSU038F1" class="link link-reveal link-table xref-fig">1</a>). The marker intervals estimated based on the unique marker positions ranged from 0.37 cM/marker pair in LG12 to 0.7 cM/marker pair in LG18 with the average marker interval of 0.46 cM/marker pair in female genetic map. The male genetic map comprised 15,148 SNPs with 7,250 unique markers and a total genetic length of 2,593.7 cM, which was much shorter than female genetic map. The marker intervals of male genetic map ranged from 0.25 cM/marker pair (LG18) to 0.55 cM/marker pair (LG1), with an average marker interval of 0.36 cM (Table <a reveal-id="DSU038TB3" data-open="DSU038TB3" class="link link-reveal link-table xref-fig">3</a>, Fig. <a reveal-id="DSU038F2" data-open="DSU038F2" class="link link-reveal link-table xref-fig">2</a>). It should be noted that these distances refer to intervals where recombination was detected and of course it should be recognized that in both sexes, the minimum distance observed was 0 cM for completely linked markers. The female- and male-specific maps are illustrated in Figs <a reveal-id="DSU038F1" data-open="DSU038F1" class="link link-reveal link-table xref-fig">1</a> and <a reveal-id="DSU038F2" data-open="DSU038F2" class="link link-reveal link-table xref-fig">2</a>, respectively. The detailed map information was provided in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data S1</a>. </p>                        <a id="6578258" scrollto-destination="6578258"></a>
<div content-id="DSU038TB3" class="table-modal"><div class="table-wrap"><div id="DSU038TB3" class="table-wrap-title" data-id="DSU038TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Summary of the sex-specific linkage maps of channel catfish</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Linkage group </th><th colspan="4">Female-specific map<hr /> </th><th colspan="4">Male-specific map<hr /> </th><th rowspan="2">F:M ratio </th></tr><tr><th>Mapped markers </th><th>Distinct positions </th><th>Genetic length (cM) </th><th>Marker interval (cM) </th><th>Mapped markers </th><th>Distinct positions </th><th>Genetic length (cM) </th><th>Marker interval (cM) </th></tr></thead><tbody><tr><td>1 </td><td>810 </td><td>397 </td><td>209.8 </td><td>0.53 </td><td>374 </td><td>153 </td><td>83.9 </td><td>0.55 </td><td>2.50 </td></tr><tr><td>2 </td><td>616 </td><td>370 </td><td>147.1 </td><td>0.40 </td><td>517 </td><td>251 </td><td>90.6 </td><td>0.36 </td><td>1.62 </td></tr><tr><td>3 </td><td>934 </td><td>481 </td><td>230.6 </td><td>0.48 </td><td>665 </td><td>342 </td><td>126.4 </td><td>0.37 </td><td>1.82 </td></tr><tr><td>4 </td><td>774 </td><td>380 </td><td>166.8 </td><td>0.44 </td><td>699 </td><td>313 </td><td>110 </td><td>0.35 </td><td>1.52 </td></tr><tr><td>5 </td><td>643 </td><td>353 </td><td>165.3 </td><td>0.47 </td><td>441 </td><td>199 </td><td>79.1 </td><td>0.40 </td><td>2.09 </td></tr><tr><td>6 </td><td>821 </td><td>470 </td><td>205.3 </td><td>0.44 </td><td>606 </td><td>300 </td><td>132.1 </td><td>0.44 </td><td>1.55 </td></tr><tr><td>7 </td><td>583 </td><td>313 </td><td>134.9 </td><td>0.43 </td><td>370 </td><td>196 </td><td>63.1 </td><td>0.32 </td><td>2.14 </td></tr><tr><td>8 </td><td>550 </td><td>326 </td><td>160.4 </td><td>0.49 </td><td>615 </td><td>297 </td><td>92.7 </td><td>0.31 </td><td>1.73 </td></tr><tr><td>9 </td><td>751 </td><td>399 </td><td>177.6 </td><td>0.45 </td><td>546 </td><td>291 </td><td>97.1 </td><td>0.33 </td><td>1.83 </td></tr><tr><td>10 </td><td>488 </td><td>238 </td><td>133.4 </td><td>0.56 </td><td>516 </td><td>228 </td><td>75.7 </td><td>0.33 </td><td>1.76 </td></tr><tr><td>11 </td><td>815 </td><td>434 </td><td>198.5 </td><td>0.46 </td><td>616 </td><td>273 </td><td>111.6 </td><td>0.41 </td><td>1.78 </td></tr><tr><td>12 </td><td>695 </td><td>361 </td><td>133.9 </td><td>0.37 </td><td>727 </td><td>311 </td><td>82.2 </td><td>0.26 </td><td>1.63 </td></tr><tr><td>13 </td><td>604 </td><td>311 </td><td>149.3 </td><td>0.48 </td><td>557 </td><td>285 </td><td>123.6 </td><td>0.43 </td><td>1.21 </td></tr><tr><td>14 </td><td>598 </td><td>340 </td><td>133.9 </td><td>0.39 </td><td>351 </td><td>150 </td><td>79.2 </td><td>0.53 </td><td>1.69 </td></tr><tr><td>15 </td><td>651 </td><td>332 </td><td>145.5 </td><td>0.44 </td><td>524 </td><td>248 </td><td>73.8 </td><td>0.30 </td><td>1.97 </td></tr><tr><td>16 </td><td>799 </td><td>385 </td><td>178.1 </td><td>0.46 </td><td>635 </td><td>337 </td><td>111.2 </td><td>0.33 </td><td>1.60 </td></tr><tr><td>17 </td><td>792 </td><td>420 </td><td>175.4 </td><td>0.42 </td><td>506 </td><td>264 </td><td>103.6 </td><td>0.39 </td><td>1.69 </td></tr><tr><td>18 </td><td>473 </td><td>230 </td><td>160.3 </td><td>0.70 </td><td>640 </td><td>290 </td><td>73.5 </td><td>0.25 </td><td>2.18 </td></tr><tr><td>19 </td><td>542 </td><td>302 </td><td>139.3 </td><td>0.46 </td><td>495 </td><td>256 </td><td>95.9 </td><td>0.37 </td><td>1.45 </td></tr><tr><td>20 </td><td>657 </td><td>344 </td><td>162.5 </td><td>0.47 </td><td>557 </td><td>244 </td><td>88.2 </td><td>0.36 </td><td>1.84 </td></tr><tr><td>21 </td><td>576 </td><td>318 </td><td>134.1 </td><td>0.42 </td><td>437 </td><td>217 </td><td>76.4 </td><td>0.35 </td><td>1.76 </td></tr><tr><td>22 </td><td>308 </td><td>153 </td><td>65.8 </td><td>0.43 </td><td>380 </td><td>169 </td><td>67.9 </td><td>0.40 </td><td>0.97 </td></tr><tr><td>23 </td><td>668 </td><td>319 </td><td>125.3 </td><td>0.39 </td><td>478 </td><td>245 </td><td>73.9 </td><td>0.30 </td><td>1.70 </td></tr><tr><td>24 </td><td>518 </td><td>296 </td><td>134.5 </td><td>0.45 </td><td>408 </td><td>195 </td><td>72.6 </td><td>0.37 </td><td>1.85 </td></tr><tr><td>25 </td><td>665 </td><td>348 </td><td>150.2 </td><td>0.43 </td><td>535 </td><td>269 </td><td>109.3 </td><td>0.41 </td><td>1.37 </td></tr><tr><td>26 </td><td>389 </td><td>181 </td><td>124.9 </td><td>0.69 </td><td>512 </td><td>242 </td><td>76.9 </td><td>0.32 </td><td>1.62 </td></tr><tr><td>27 </td><td>595 </td><td>327 </td><td>152.9 </td><td>0.47 </td><td>522 </td><td>233 </td><td>70.4 </td><td>0.30 </td><td>2.17 </td></tr><tr><td>28 </td><td>663 </td><td>334 </td><td>141.7 </td><td>0.42 </td><td>484 </td><td>237 </td><td>80.5 </td><td>0.34 </td><td>1.76 </td></tr><tr><td>29 </td><td>466 </td><td>284 </td><td>157.4 </td><td>0.55 </td><td>435 </td><td>215 </td><td>72.5 </td><td>0.34 </td><td>2.17 </td></tr><tr><td>Total </td><td>18,444 </td><td>9,746 </td><td>4,495.1 </td><td>0.46 </td><td>15,148 </td><td>7,250 </td><td>2,593.7 </td><td>0.36 </td><td>1.73 </td></tr></tbody></table></div><div content-id="DSU038TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Linkage group </th><th colspan="4">Female-specific map<hr /> </th><th colspan="4">Male-specific map<hr /> </th><th rowspan="2">F:M ratio </th></tr><tr><th>Mapped markers </th><th>Distinct positions </th><th>Genetic length (cM) </th><th>Marker interval (cM) </th><th>Mapped markers </th><th>Distinct positions </th><th>Genetic length (cM) </th><th>Marker interval (cM) </th></tr></thead><tbody><tr><td>1 </td><td>810 </td><td>397 </td><td>209.8 </td><td>0.53 </td><td>374 </td><td>153 </td><td>83.9 </td><td>0.55 </td><td>2.50 </td></tr><tr><td>2 </td><td>616 </td><td>370 </td><td>147.1 </td><td>0.40 </td><td>517 </td><td>251 </td><td>90.6 </td><td>0.36 </td><td>1.62 </td></tr><tr><td>3 </td><td>934 </td><td>481 </td><td>230.6 </td><td>0.48 </td><td>665 </td><td>342 </td><td>126.4 </td><td>0.37 </td><td>1.82 </td></tr><tr><td>4 </td><td>774 </td><td>380 </td><td>166.8 </td><td>0.44 </td><td>699 </td><td>313 </td><td>110 </td><td>0.35 </td><td>1.52 </td></tr><tr><td>5 </td><td>643 </td><td>353 </td><td>165.3 </td><td>0.47 </td><td>441 </td><td>199 </td><td>79.1 </td><td>0.40 </td><td>2.09 </td></tr><tr><td>6 </td><td>821 </td><td>470 </td><td>205.3 </td><td>0.44 </td><td>606 </td><td>300 </td><td>132.1 </td><td>0.44 </td><td>1.55 </td></tr><tr><td>7 </td><td>583 </td><td>313 </td><td>134.9 </td><td>0.43 </td><td>370 </td><td>196 </td><td>63.1 </td><td>0.32 </td><td>2.14 </td></tr><tr><td>8 </td><td>550 </td><td>326 </td><td>160.4 </td><td>0.49 </td><td>615 </td><td>297 </td><td>92.7 </td><td>0.31 </td><td>1.73 </td></tr><tr><td>9 </td><td>751 </td><td>399 </td><td>177.6 </td><td>0.45 </td><td>546 </td><td>291 </td><td>97.1 </td><td>0.33 </td><td>1.83 </td></tr><tr><td>10 </td><td>488 </td><td>238 </td><td>133.4 </td><td>0.56 </td><td>516 </td><td>228 </td><td>75.7 </td><td>0.33 </td><td>1.76 </td></tr><tr><td>11 </td><td>815 </td><td>434 </td><td>198.5 </td><td>0.46 </td><td>616 </td><td>273 </td><td>111.6 </td><td>0.41 </td><td>1.78 </td></tr><tr><td>12 </td><td>695 </td><td>361 </td><td>133.9 </td><td>0.37 </td><td>727 </td><td>311 </td><td>82.2 </td><td>0.26 </td><td>1.63 </td></tr><tr><td>13 </td><td>604 </td><td>311 </td><td>149.3 </td><td>0.48 </td><td>557 </td><td>285 </td><td>123.6 </td><td>0.43 </td><td>1.21 </td></tr><tr><td>14 </td><td>598 </td><td>340 </td><td>133.9 </td><td>0.39 </td><td>351 </td><td>150 </td><td>79.2 </td><td>0.53 </td><td>1.69 </td></tr><tr><td>15 </td><td>651 </td><td>332 </td><td>145.5 </td><td>0.44 </td><td>524 </td><td>248 </td><td>73.8 </td><td>0.30 </td><td>1.97 </td></tr><tr><td>16 </td><td>799 </td><td>385 </td><td>178.1 </td><td>0.46 </td><td>635 </td><td>337 </td><td>111.2 </td><td>0.33 </td><td>1.60 </td></tr><tr><td>17 </td><td>792 </td><td>420 </td><td>175.4 </td><td>0.42 </td><td>506 </td><td>264 </td><td>103.6 </td><td>0.39 </td><td>1.69 </td></tr><tr><td>18 </td><td>473 </td><td>230 </td><td>160.3 </td><td>0.70 </td><td>640 </td><td>290 </td><td>73.5 </td><td>0.25 </td><td>2.18 </td></tr><tr><td>19 </td><td>542 </td><td>302 </td><td>139.3 </td><td>0.46 </td><td>495 </td><td>256 </td><td>95.9 </td><td>0.37 </td><td>1.45 </td></tr><tr><td>20 </td><td>657 </td><td>344 </td><td>162.5 </td><td>0.47 </td><td>557 </td><td>244 </td><td>88.2 </td><td>0.36 </td><td>1.84 </td></tr><tr><td>21 </td><td>576 </td><td>318 </td><td>134.1 </td><td>0.42 </td><td>437 </td><td>217 </td><td>76.4 </td><td>0.35 </td><td>1.76 </td></tr><tr><td>22 </td><td>308 </td><td>153 </td><td>65.8 </td><td>0.43 </td><td>380 </td><td>169 </td><td>67.9 </td><td>0.40 </td><td>0.97 </td></tr><tr><td>23 </td><td>668 </td><td>319 </td><td>125.3 </td><td>0.39 </td><td>478 </td><td>245 </td><td>73.9 </td><td>0.30 </td><td>1.70 </td></tr><tr><td>24 </td><td>518 </td><td>296 </td><td>134.5 </td><td>0.45 </td><td>408 </td><td>195 </td><td>72.6 </td><td>0.37 </td><td>1.85 </td></tr><tr><td>25 </td><td>665 </td><td>348 </td><td>150.2 </td><td>0.43 </td><td>535 </td><td>269 </td><td>109.3 </td><td>0.41 </td><td>1.37 </td></tr><tr><td>26 </td><td>389 </td><td>181 </td><td>124.9 </td><td>0.69 </td><td>512 </td><td>242 </td><td>76.9 </td><td>0.32 </td><td>1.62 </td></tr><tr><td>27 </td><td>595 </td><td>327 </td><td>152.9 </td><td>0.47 </td><td>522 </td><td>233 </td><td>70.4 </td><td>0.30 </td><td>2.17 </td></tr><tr><td>28 </td><td>663 </td><td>334 </td><td>141.7 </td><td>0.42 </td><td>484 </td><td>237 </td><td>80.5 </td><td>0.34 </td><td>1.76 </td></tr><tr><td>29 </td><td>466 </td><td>284 </td><td>157.4 </td><td>0.55 </td><td>435 </td><td>215 </td><td>72.5 </td><td>0.34 </td><td>2.17 </td></tr><tr><td>Total </td><td>18,444 </td><td>9,746 </td><td>4,495.1 </td><td>0.46 </td><td>15,148 </td><td>7,250 </td><td>2,593.7 </td><td>0.36 </td><td>1.73 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6578258" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU038TB3" class="table-wrap-title" data-id="DSU038TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Summary of the sex-specific linkage maps of channel catfish</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Linkage group </th><th colspan="4">Female-specific map<hr /> </th><th colspan="4">Male-specific map<hr /> </th><th rowspan="2">F:M ratio </th></tr><tr><th>Mapped markers </th><th>Distinct positions </th><th>Genetic length (cM) </th><th>Marker interval (cM) </th><th>Mapped markers </th><th>Distinct positions </th><th>Genetic length (cM) </th><th>Marker interval (cM) </th></tr></thead><tbody><tr><td>1 </td><td>810 </td><td>397 </td><td>209.8 </td><td>0.53 </td><td>374 </td><td>153 </td><td>83.9 </td><td>0.55 </td><td>2.50 </td></tr><tr><td>2 </td><td>616 </td><td>370 </td><td>147.1 </td><td>0.40 </td><td>517 </td><td>251 </td><td>90.6 </td><td>0.36 </td><td>1.62 </td></tr><tr><td>3 </td><td>934 </td><td>481 </td><td>230.6 </td><td>0.48 </td><td>665 </td><td>342 </td><td>126.4 </td><td>0.37 </td><td>1.82 </td></tr><tr><td>4 </td><td>774 </td><td>380 </td><td>166.8 </td><td>0.44 </td><td>699 </td><td>313 </td><td>110 </td><td>0.35 </td><td>1.52 </td></tr><tr><td>5 </td><td>643 </td><td>353 </td><td>165.3 </td><td>0.47 </td><td>441 </td><td>199 </td><td>79.1 </td><td>0.40 </td><td>2.09 </td></tr><tr><td>6 </td><td>821 </td><td>470 </td><td>205.3 </td><td>0.44 </td><td>606 </td><td>300 </td><td>132.1 </td><td>0.44 </td><td>1.55 </td></tr><tr><td>7 </td><td>583 </td><td>313 </td><td>134.9 </td><td>0.43 </td><td>370 </td><td>196 </td><td>63.1 </td><td>0.32 </td><td>2.14 </td></tr><tr><td>8 </td><td>550 </td><td>326 </td><td>160.4 </td><td>0.49 </td><td>615 </td><td>297 </td><td>92.7 </td><td>0.31 </td><td>1.73 </td></tr><tr><td>9 </td><td>751 </td><td>399 </td><td>177.6 </td><td>0.45 </td><td>546 </td><td>291 </td><td>97.1 </td><td>0.33 </td><td>1.83 </td></tr><tr><td>10 </td><td>488 </td><td>238 </td><td>133.4 </td><td>0.56 </td><td>516 </td><td>228 </td><td>75.7 </td><td>0.33 </td><td>1.76 </td></tr><tr><td>11 </td><td>815 </td><td>434 </td><td>198.5 </td><td>0.46 </td><td>616 </td><td>273 </td><td>111.6 </td><td>0.41 </td><td>1.78 </td></tr><tr><td>12 </td><td>695 </td><td>361 </td><td>133.9 </td><td>0.37 </td><td>727 </td><td>311 </td><td>82.2 </td><td>0.26 </td><td>1.63 </td></tr><tr><td>13 </td><td>604 </td><td>311 </td><td>149.3 </td><td>0.48 </td><td>557 </td><td>285 </td><td>123.6 </td><td>0.43 </td><td>1.21 </td></tr><tr><td>14 </td><td>598 </td><td>340 </td><td>133.9 </td><td>0.39 </td><td>351 </td><td>150 </td><td>79.2 </td><td>0.53 </td><td>1.69 </td></tr><tr><td>15 </td><td>651 </td><td>332 </td><td>145.5 </td><td>0.44 </td><td>524 </td><td>248 </td><td>73.8 </td><td>0.30 </td><td>1.97 </td></tr><tr><td>16 </td><td>799 </td><td>385 </td><td>178.1 </td><td>0.46 </td><td>635 </td><td>337 </td><td>111.2 </td><td>0.33 </td><td>1.60 </td></tr><tr><td>17 </td><td>792 </td><td>420 </td><td>175.4 </td><td>0.42 </td><td>506 </td><td>264 </td><td>103.6 </td><td>0.39 </td><td>1.69 </td></tr><tr><td>18 </td><td>473 </td><td>230 </td><td>160.3 </td><td>0.70 </td><td>640 </td><td>290 </td><td>73.5 </td><td>0.25 </td><td>2.18 </td></tr><tr><td>19 </td><td>542 </td><td>302 </td><td>139.3 </td><td>0.46 </td><td>495 </td><td>256 </td><td>95.9 </td><td>0.37 </td><td>1.45 </td></tr><tr><td>20 </td><td>657 </td><td>344 </td><td>162.5 </td><td>0.47 </td><td>557 </td><td>244 </td><td>88.2 </td><td>0.36 </td><td>1.84 </td></tr><tr><td>21 </td><td>576 </td><td>318 </td><td>134.1 </td><td>0.42 </td><td>437 </td><td>217 </td><td>76.4 </td><td>0.35 </td><td>1.76 </td></tr><tr><td>22 </td><td>308 </td><td>153 </td><td>65.8 </td><td>0.43 </td><td>380 </td><td>169 </td><td>67.9 </td><td>0.40 </td><td>0.97 </td></tr><tr><td>23 </td><td>668 </td><td>319 </td><td>125.3 </td><td>0.39 </td><td>478 </td><td>245 </td><td>73.9 </td><td>0.30 </td><td>1.70 </td></tr><tr><td>24 </td><td>518 </td><td>296 </td><td>134.5 </td><td>0.45 </td><td>408 </td><td>195 </td><td>72.6 </td><td>0.37 </td><td>1.85 </td></tr><tr><td>25 </td><td>665 </td><td>348 </td><td>150.2 </td><td>0.43 </td><td>535 </td><td>269 </td><td>109.3 </td><td>0.41 </td><td>1.37 </td></tr><tr><td>26 </td><td>389 </td><td>181 </td><td>124.9 </td><td>0.69 </td><td>512 </td><td>242 </td><td>76.9 </td><td>0.32 </td><td>1.62 </td></tr><tr><td>27 </td><td>595 </td><td>327 </td><td>152.9 </td><td>0.47 </td><td>522 </td><td>233 </td><td>70.4 </td><td>0.30 </td><td>2.17 </td></tr><tr><td>28 </td><td>663 </td><td>334 </td><td>141.7 </td><td>0.42 </td><td>484 </td><td>237 </td><td>80.5 </td><td>0.34 </td><td>1.76 </td></tr><tr><td>29 </td><td>466 </td><td>284 </td><td>157.4 </td><td>0.55 </td><td>435 </td><td>215 </td><td>72.5 </td><td>0.34 </td><td>2.17 </td></tr><tr><td>Total </td><td>18,444 </td><td>9,746 </td><td>4,495.1 </td><td>0.46 </td><td>15,148 </td><td>7,250 </td><td>2,593.7 </td><td>0.36 </td><td>1.73 </td></tr></tbody></table></div><div content-id="DSU038TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Linkage group </th><th colspan="4">Female-specific map<hr /> </th><th colspan="4">Male-specific map<hr /> </th><th rowspan="2">F:M ratio </th></tr><tr><th>Mapped markers </th><th>Distinct positions </th><th>Genetic length (cM) </th><th>Marker interval (cM) </th><th>Mapped markers </th><th>Distinct positions </th><th>Genetic length (cM) </th><th>Marker interval (cM) </th></tr></thead><tbody><tr><td>1 </td><td>810 </td><td>397 </td><td>209.8 </td><td>0.53 </td><td>374 </td><td>153 </td><td>83.9 </td><td>0.55 </td><td>2.50 </td></tr><tr><td>2 </td><td>616 </td><td>370 </td><td>147.1 </td><td>0.40 </td><td>517 </td><td>251 </td><td>90.6 </td><td>0.36 </td><td>1.62 </td></tr><tr><td>3 </td><td>934 </td><td>481 </td><td>230.6 </td><td>0.48 </td><td>665 </td><td>342 </td><td>126.4 </td><td>0.37 </td><td>1.82 </td></tr><tr><td>4 </td><td>774 </td><td>380 </td><td>166.8 </td><td>0.44 </td><td>699 </td><td>313 </td><td>110 </td><td>0.35 </td><td>1.52 </td></tr><tr><td>5 </td><td>643 </td><td>353 </td><td>165.3 </td><td>0.47 </td><td>441 </td><td>199 </td><td>79.1 </td><td>0.40 </td><td>2.09 </td></tr><tr><td>6 </td><td>821 </td><td>470 </td><td>205.3 </td><td>0.44 </td><td>606 </td><td>300 </td><td>132.1 </td><td>0.44 </td><td>1.55 </td></tr><tr><td>7 </td><td>583 </td><td>313 </td><td>134.9 </td><td>0.43 </td><td>370 </td><td>196 </td><td>63.1 </td><td>0.32 </td><td>2.14 </td></tr><tr><td>8 </td><td>550 </td><td>326 </td><td>160.4 </td><td>0.49 </td><td>615 </td><td>297 </td><td>92.7 </td><td>0.31 </td><td>1.73 </td></tr><tr><td>9 </td><td>751 </td><td>399 </td><td>177.6 </td><td>0.45 </td><td>546 </td><td>291 </td><td>97.1 </td><td>0.33 </td><td>1.83 </td></tr><tr><td>10 </td><td>488 </td><td>238 </td><td>133.4 </td><td>0.56 </td><td>516 </td><td>228 </td><td>75.7 </td><td>0.33 </td><td>1.76 </td></tr><tr><td>11 </td><td>815 </td><td>434 </td><td>198.5 </td><td>0.46 </td><td>616 </td><td>273 </td><td>111.6 </td><td>0.41 </td><td>1.78 </td></tr><tr><td>12 </td><td>695 </td><td>361 </td><td>133.9 </td><td>0.37 </td><td>727 </td><td>311 </td><td>82.2 </td><td>0.26 </td><td>1.63 </td></tr><tr><td>13 </td><td>604 </td><td>311 </td><td>149.3 </td><td>0.48 </td><td>557 </td><td>285 </td><td>123.6 </td><td>0.43 </td><td>1.21 </td></tr><tr><td>14 </td><td>598 </td><td>340 </td><td>133.9 </td><td>0.39 </td><td>351 </td><td>150 </td><td>79.2 </td><td>0.53 </td><td>1.69 </td></tr><tr><td>15 </td><td>651 </td><td>332 </td><td>145.5 </td><td>0.44 </td><td>524 </td><td>248 </td><td>73.8 </td><td>0.30 </td><td>1.97 </td></tr><tr><td>16 </td><td>799 </td><td>385 </td><td>178.1 </td><td>0.46 </td><td>635 </td><td>337 </td><td>111.2 </td><td>0.33 </td><td>1.60 </td></tr><tr><td>17 </td><td>792 </td><td>420 </td><td>175.4 </td><td>0.42 </td><td>506 </td><td>264 </td><td>103.6 </td><td>0.39 </td><td>1.69 </td></tr><tr><td>18 </td><td>473 </td><td>230 </td><td>160.3 </td><td>0.70 </td><td>640 </td><td>290 </td><td>73.5 </td><td>0.25 </td><td>2.18 </td></tr><tr><td>19 </td><td>542 </td><td>302 </td><td>139.3 </td><td>0.46 </td><td>495 </td><td>256 </td><td>95.9 </td><td>0.37 </td><td>1.45 </td></tr><tr><td>20 </td><td>657 </td><td>344 </td><td>162.5 </td><td>0.47 </td><td>557 </td><td>244 </td><td>88.2 </td><td>0.36 </td><td>1.84 </td></tr><tr><td>21 </td><td>576 </td><td>318 </td><td>134.1 </td><td>0.42 </td><td>437 </td><td>217 </td><td>76.4 </td><td>0.35 </td><td>1.76 </td></tr><tr><td>22 </td><td>308 </td><td>153 </td><td>65.8 </td><td>0.43 </td><td>380 </td><td>169 </td><td>67.9 </td><td>0.40 </td><td>0.97 </td></tr><tr><td>23 </td><td>668 </td><td>319 </td><td>125.3 </td><td>0.39 </td><td>478 </td><td>245 </td><td>73.9 </td><td>0.30 </td><td>1.70 </td></tr><tr><td>24 </td><td>518 </td><td>296 </td><td>134.5 </td><td>0.45 </td><td>408 </td><td>195 </td><td>72.6 </td><td>0.37 </td><td>1.85 </td></tr><tr><td>25 </td><td>665 </td><td>348 </td><td>150.2 </td><td>0.43 </td><td>535 </td><td>269 </td><td>109.3 </td><td>0.41 </td><td>1.37 </td></tr><tr><td>26 </td><td>389 </td><td>181 </td><td>124.9 </td><td>0.69 </td><td>512 </td><td>242 </td><td>76.9 </td><td>0.32 </td><td>1.62 </td></tr><tr><td>27 </td><td>595 </td><td>327 </td><td>152.9 </td><td>0.47 </td><td>522 </td><td>233 </td><td>70.4 </td><td>0.30 </td><td>2.17 </td></tr><tr><td>28 </td><td>663 </td><td>334 </td><td>141.7 </td><td>0.42 </td><td>484 </td><td>237 </td><td>80.5 </td><td>0.34 </td><td>1.76 </td></tr><tr><td>29 </td><td>466 </td><td>284 </td><td>157.4 </td><td>0.55 </td><td>435 </td><td>215 </td><td>72.5 </td><td>0.34 </td><td>2.17 </td></tr><tr><td>Total </td><td>18,444 </td><td>9,746 </td><td>4,495.1 </td><td>0.46 </td><td>15,148 </td><td>7,250 </td><td>2,593.7 </td><td>0.36 </td><td>1.73 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6578258" target="_blank">View Large</a></div></div>                        <a id="6578260" scrollto-destination="6578260"></a>
<div id="DSU038F1" data-id="DSU038F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu038/2/m_dsu03801.png?Expires=2147483647&Signature=juq~T2aJuAoE6Xd50-BCzA2-PC-MbTK9SlOHpTzxj3Lycf5JNhMBBVocCYsWJnG3UVB8w1IKM6cltpMXyg5QFeYetW9zRDJ76zUHVwy187Qxe9vywg5Z54Obu8nxW67qxFBZ8BnO4Vt0AIvwNuegXGT1NzJfTeWqgl~faPREcYxZvkpVdBl1~EZlkGwkERwWKYCuOmBx0hYl6b89d2v6oABl06bZN-Se2RE1lyRz4OmY6hxw2-FutOKepOWv1M~RdHvqAN6LueDHrIk0bCt2ZIYexypPDXPaTi9ftvOcB3sh0W4EaGlZ8RmTMc8~QvyfRT~rmIKPVYX12-3B2RdERw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Illustration of female-specific linkage map." path-from-xml="dsu03801.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6578260/dsu03801.tif" path-from-xml="dsu03801.tif" target="_blank">View large</a><a section="6578260" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu038/2/dsu03801.png?Expires=2147483647&Signature=A~~L6NamwCOo6C4U~Z7ATrK7pk8BL6xyxQ9-3mS8m1bCT8pRoTQpAfoYTi4Deyq8FzZ-REtVC3SpOyYxlNmZHv8Djvfjt3UbUi6Sliv~b-07dcZW~7KdO2Y-CqYis1b~B7FK5LqmD3uy-d6UvQ79KKADM6BznYw7N8TGmymMC345RVwvpNVDwPDcIaTJLed06aL2Z2N4iVf9BtclodMMdQyyHgTBIee7-8A9sfcOKcCZzn7UjZCB-C1l1AimPVOeIX6xTFiYwB0pDxmakTdeflU~IKdnuVv45~FaaVjkhSJvg~~WVdze7eES2Eqwj5Tm8HCntU8iWfZo54sldRU1bQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6578260&ar=440428&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03801.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Illustration of female-specific linkage map.</p></div></div><div content-id="DSU038F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu038/2/m_dsu03801.png?Expires=2147483647&Signature=juq~T2aJuAoE6Xd50-BCzA2-PC-MbTK9SlOHpTzxj3Lycf5JNhMBBVocCYsWJnG3UVB8w1IKM6cltpMXyg5QFeYetW9zRDJ76zUHVwy187Qxe9vywg5Z54Obu8nxW67qxFBZ8BnO4Vt0AIvwNuegXGT1NzJfTeWqgl~faPREcYxZvkpVdBl1~EZlkGwkERwWKYCuOmBx0hYl6b89d2v6oABl06bZN-Se2RE1lyRz4OmY6hxw2-FutOKepOWv1M~RdHvqAN6LueDHrIk0bCt2ZIYexypPDXPaTi9ftvOcB3sh0W4EaGlZ8RmTMc8~QvyfRT~rmIKPVYX12-3B2RdERw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Illustration of female-specific linkage map." path-from-xml="dsu03801.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6578260/dsu03801.tif" path-from-xml="dsu03801.tif" target="_blank">View large</a><a section="6578260" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu038/2/dsu03801.png?Expires=2147483647&Signature=A~~L6NamwCOo6C4U~Z7ATrK7pk8BL6xyxQ9-3mS8m1bCT8pRoTQpAfoYTi4Deyq8FzZ-REtVC3SpOyYxlNmZHv8Djvfjt3UbUi6Sliv~b-07dcZW~7KdO2Y-CqYis1b~B7FK5LqmD3uy-d6UvQ79KKADM6BznYw7N8TGmymMC345RVwvpNVDwPDcIaTJLed06aL2Z2N4iVf9BtclodMMdQyyHgTBIee7-8A9sfcOKcCZzn7UjZCB-C1l1AimPVOeIX6xTFiYwB0pDxmakTdeflU~IKdnuVv45~FaaVjkhSJvg~~WVdze7eES2Eqwj5Tm8HCntU8iWfZo54sldRU1bQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6578260&ar=440428&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03801.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Illustration of female-specific linkage map.</p></div></div>                        <a id="6578261" scrollto-destination="6578261"></a>
<div id="DSU038F2" data-id="DSU038F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu038/2/m_dsu03802.png?Expires=2147483647&Signature=Clw4cDVzj~7m5Ku4t~Cj~GCLsXO8sM7wgfFqBc8fIqaJjcS7hdmLvqy4gVjH7sNP3-CrU45e4tTdkzSm9SiiunU1sfVbTRsmwhpOSf~pC~K8nuVtKF-JFvomcZJsjItunILaTQKD9DOrWv5-aKk~rCGFhoPfoO7Adqt1Tg~Kxv-6yLVYnz8QgUrosREB9y1F7f4BsNaKru~wdLWt312b8BRErtjrtCOubYoaj11o3eHhc1lygSCtvpL-E1PpDy1lphDXzzc~QVoxAKgPa-1VAEqIw78mbtXOtb7CzbeYjOdmSfkbxbYkVPPUVWHNZs1IKu20uGRinAWmD~Kk1DfU1A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Illustration of male-specific linkage map." path-from-xml="dsu03802.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6578261/dsu03802.tif" path-from-xml="dsu03802.tif" target="_blank">View large</a><a section="6578261" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu038/2/dsu03802.png?Expires=2147483647&Signature=aH2D9ldq1xURPg1QtqisPzGlorpGf9i3VV1~m39xZxTyyCHBaGC72TywbBeamB0P09phB~rZKyF22Lfg07QM5EjkoQP-T0a7NdRz3gTJhGilja4uRojqQRKABA6aYTl1R2MAnLTTdD4wwQ1duez4AS7dN-FnJJ8cqFILqn8D052G8f92QX8HwE9X1Ftxj8uWueLbQWx~zVED7CNx0iQIDSJusjWXby1oXrZGtYFYN00BSuR0BBYbvSHfMBenHf55vPL7TnGTWb7UB~IUTRGtRNMrcTqCXvMc4kVYCA9Utaqzq1eMA5u4LRfcPOYUH8PgHikh5Xecw1AA9GXzGhFAAQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6578261&ar=440428&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03802.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Illustration of male-specific linkage map.</p></div></div><div content-id="DSU038F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu038/2/m_dsu03802.png?Expires=2147483647&Signature=Clw4cDVzj~7m5Ku4t~Cj~GCLsXO8sM7wgfFqBc8fIqaJjcS7hdmLvqy4gVjH7sNP3-CrU45e4tTdkzSm9SiiunU1sfVbTRsmwhpOSf~pC~K8nuVtKF-JFvomcZJsjItunILaTQKD9DOrWv5-aKk~rCGFhoPfoO7Adqt1Tg~Kxv-6yLVYnz8QgUrosREB9y1F7f4BsNaKru~wdLWt312b8BRErtjrtCOubYoaj11o3eHhc1lygSCtvpL-E1PpDy1lphDXzzc~QVoxAKgPa-1VAEqIw78mbtXOtb7CzbeYjOdmSfkbxbYkVPPUVWHNZs1IKu20uGRinAWmD~Kk1DfU1A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Illustration of male-specific linkage map." path-from-xml="dsu03802.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6578261/dsu03802.tif" path-from-xml="dsu03802.tif" target="_blank">View large</a><a section="6578261" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu038/2/dsu03802.png?Expires=2147483647&Signature=aH2D9ldq1xURPg1QtqisPzGlorpGf9i3VV1~m39xZxTyyCHBaGC72TywbBeamB0P09phB~rZKyF22Lfg07QM5EjkoQP-T0a7NdRz3gTJhGilja4uRojqQRKABA6aYTl1R2MAnLTTdD4wwQ1duez4AS7dN-FnJJ8cqFILqn8D052G8f92QX8HwE9X1Ftxj8uWueLbQWx~zVED7CNx0iQIDSJusjWXby1oXrZGtYFYN00BSuR0BBYbvSHfMBenHf55vPL7TnGTWb7UB~IUTRGtRNMrcTqCXvMc4kVYCA9Utaqzq1eMA5u4LRfcPOYUH8PgHikh5Xecw1AA9GXzGhFAAQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6578261&ar=440428&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03802.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Illustration of male-specific linkage map.</p></div></div><p>The sex-averaged map was constructed using markers that were heterozygous in both parents. As summarized in Table <a reveal-id="DSU038TB4" data-open="DSU038TB4" class="link link-reveal link-table xref-fig">4</a>, a total of 29,081 SNPs were mapped, which consisted of 15,598 unique markers. The sex-averaged map spanned 3,505.4 cM with an average marker interval of 0.22 cM, ranging from 0.17 cM/marker (LG12) to 0.30 cM/marker (LG18). The genetic map is illustrated in Fig. <a reveal-id="DSU038F3" data-open="DSU038F3" class="link link-reveal link-table xref-fig">3</a>. The detailed map information was provided in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data S1</a>. </p>                        <a id="6578263" scrollto-destination="6578263"></a>
<div content-id="DSU038TB4" class="table-modal"><div class="table-wrap"><div id="DSU038TB4" class="table-wrap-title" data-id="DSU038TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Summary of the sex-averaged linkage map of channel catfish</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Linkage group </th><th colspan="4">Sex-averaged map<hr /> </th></tr><tr><th>Mapped markers </th><th>Distinct positions </th><th>Genetic length (cM) </th><th>Marker interval (cM) </th></tr></thead><tbody><tr><td>1 </td><td>1,039 </td><td>556 </td><td>139.5 </td><td>0.25 </td></tr><tr><td>2 </td><td>1,014 </td><td>557 </td><td>116.2 </td><td>0.21 </td></tr><tr><td>3 </td><td>1,212 </td><td>673 </td><td>169.7 </td><td>0.25 </td></tr><tr><td>4 </td><td>1,220 </td><td>653 </td><td>128.1 </td><td>0.20 </td></tr><tr><td>5 </td><td>1,002 </td><td>491 </td><td>110.2 </td><td>0.22 </td></tr><tr><td>6 </td><td>1,183 </td><td>680 </td><td>163.3 </td><td>0.24 </td></tr><tr><td>7 </td><td>861 </td><td>422 </td><td>102.1 </td><td>0.24 </td></tr><tr><td>8 </td><td>1,073 </td><td>557 </td><td>117.3 </td><td>0.21 </td></tr><tr><td>9 </td><td>1,100 </td><td>604 </td><td>146.9 </td><td>0.24 </td></tr><tr><td>10 </td><td>913 </td><td>475 </td><td>105.8 </td><td>0.22 </td></tr><tr><td>11 </td><td>999 </td><td>572 </td><td>152.7 </td><td>0.27 </td></tr><tr><td>12 </td><td>1,216 </td><td>625 </td><td>104.8 </td><td>0.17 </td></tr><tr><td>13 </td><td>1,176 </td><td>630 </td><td>131.1 </td><td>0.21 </td></tr><tr><td>14 </td><td>868 </td><td>472 </td><td>113.8 </td><td>0.24 </td></tr><tr><td>15 </td><td>1,047 </td><td>562 </td><td>115.7 </td><td>0.21 </td></tr><tr><td>16 </td><td>1,219 </td><td>695 </td><td>145.3 </td><td>0.21 </td></tr><tr><td>17 </td><td>1,178 </td><td>623 </td><td>138.2 </td><td>0.22 </td></tr><tr><td>18 </td><td>782 </td><td>388 </td><td>114.8 </td><td>0.30 </td></tr><tr><td>19 </td><td>914 </td><td>523 </td><td>118.9 </td><td>0.23 </td></tr><tr><td>20 </td><td>1,029 </td><td>550 </td><td>126 </td><td>0.23 </td></tr><tr><td>21 </td><td>968 </td><td>484 </td><td>110.3 </td><td>0.23 </td></tr><tr><td>22 </td><td>779 </td><td>364 </td><td>68.4 </td><td>0.19 </td></tr><tr><td>23 </td><td>929 </td><td>486 </td><td>93.3 </td><td>0.19 </td></tr><tr><td>24 </td><td>854 </td><td>492 </td><td>106.9 </td><td>0.22 </td></tr><tr><td>25 </td><td>1,097 </td><td>584 </td><td>127.6 </td><td>0.22 </td></tr><tr><td>26 </td><td>811 </td><td>427 </td><td>101.2 </td><td>0.24 </td></tr><tr><td>27 </td><td>925 </td><td>516 </td><td>113.7 </td><td>0.22 </td></tr><tr><td>28 </td><td>969 </td><td>516 </td><td>114.1 </td><td>0.22 </td></tr><tr><td>29 </td><td>704 </td><td>421 </td><td>109.5 </td><td>0.26 </td></tr><tr><td>Total </td><td>29,081 </td><td>15,598 </td><td>3,505.4 </td><td>0.22 </td></tr></tbody></table></div><div content-id="DSU038TB4" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Linkage group </th><th colspan="4">Sex-averaged map<hr /> </th></tr><tr><th>Mapped markers </th><th>Distinct positions </th><th>Genetic length (cM) </th><th>Marker interval (cM) </th></tr></thead><tbody><tr><td>1 </td><td>1,039 </td><td>556 </td><td>139.5 </td><td>0.25 </td></tr><tr><td>2 </td><td>1,014 </td><td>557 </td><td>116.2 </td><td>0.21 </td></tr><tr><td>3 </td><td>1,212 </td><td>673 </td><td>169.7 </td><td>0.25 </td></tr><tr><td>4 </td><td>1,220 </td><td>653 </td><td>128.1 </td><td>0.20 </td></tr><tr><td>5 </td><td>1,002 </td><td>491 </td><td>110.2 </td><td>0.22 </td></tr><tr><td>6 </td><td>1,183 </td><td>680 </td><td>163.3 </td><td>0.24 </td></tr><tr><td>7 </td><td>861 </td><td>422 </td><td>102.1 </td><td>0.24 </td></tr><tr><td>8 </td><td>1,073 </td><td>557 </td><td>117.3 </td><td>0.21 </td></tr><tr><td>9 </td><td>1,100 </td><td>604 </td><td>146.9 </td><td>0.24 </td></tr><tr><td>10 </td><td>913 </td><td>475 </td><td>105.8 </td><td>0.22 </td></tr><tr><td>11 </td><td>999 </td><td>572 </td><td>152.7 </td><td>0.27 </td></tr><tr><td>12 </td><td>1,216 </td><td>625 </td><td>104.8 </td><td>0.17 </td></tr><tr><td>13 </td><td>1,176 </td><td>630 </td><td>131.1 </td><td>0.21 </td></tr><tr><td>14 </td><td>868 </td><td>472 </td><td>113.8 </td><td>0.24 </td></tr><tr><td>15 </td><td>1,047 </td><td>562 </td><td>115.7 </td><td>0.21 </td></tr><tr><td>16 </td><td>1,219 </td><td>695 </td><td>145.3 </td><td>0.21 </td></tr><tr><td>17 </td><td>1,178 </td><td>623 </td><td>138.2 </td><td>0.22 </td></tr><tr><td>18 </td><td>782 </td><td>388 </td><td>114.8 </td><td>0.30 </td></tr><tr><td>19 </td><td>914 </td><td>523 </td><td>118.9 </td><td>0.23 </td></tr><tr><td>20 </td><td>1,029 </td><td>550 </td><td>126 </td><td>0.23 </td></tr><tr><td>21 </td><td>968 </td><td>484 </td><td>110.3 </td><td>0.23 </td></tr><tr><td>22 </td><td>779 </td><td>364 </td><td>68.4 </td><td>0.19 </td></tr><tr><td>23 </td><td>929 </td><td>486 </td><td>93.3 </td><td>0.19 </td></tr><tr><td>24 </td><td>854 </td><td>492 </td><td>106.9 </td><td>0.22 </td></tr><tr><td>25 </td><td>1,097 </td><td>584 </td><td>127.6 </td><td>0.22 </td></tr><tr><td>26 </td><td>811 </td><td>427 </td><td>101.2 </td><td>0.24 </td></tr><tr><td>27 </td><td>925 </td><td>516 </td><td>113.7 </td><td>0.22 </td></tr><tr><td>28 </td><td>969 </td><td>516 </td><td>114.1 </td><td>0.22 </td></tr><tr><td>29 </td><td>704 </td><td>421 </td><td>109.5 </td><td>0.26 </td></tr><tr><td>Total </td><td>29,081 </td><td>15,598 </td><td>3,505.4 </td><td>0.22 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6578263" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU038TB4" class="table-wrap-title" data-id="DSU038TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Summary of the sex-averaged linkage map of channel catfish</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Linkage group </th><th colspan="4">Sex-averaged map<hr /> </th></tr><tr><th>Mapped markers </th><th>Distinct positions </th><th>Genetic length (cM) </th><th>Marker interval (cM) </th></tr></thead><tbody><tr><td>1 </td><td>1,039 </td><td>556 </td><td>139.5 </td><td>0.25 </td></tr><tr><td>2 </td><td>1,014 </td><td>557 </td><td>116.2 </td><td>0.21 </td></tr><tr><td>3 </td><td>1,212 </td><td>673 </td><td>169.7 </td><td>0.25 </td></tr><tr><td>4 </td><td>1,220 </td><td>653 </td><td>128.1 </td><td>0.20 </td></tr><tr><td>5 </td><td>1,002 </td><td>491 </td><td>110.2 </td><td>0.22 </td></tr><tr><td>6 </td><td>1,183 </td><td>680 </td><td>163.3 </td><td>0.24 </td></tr><tr><td>7 </td><td>861 </td><td>422 </td><td>102.1 </td><td>0.24 </td></tr><tr><td>8 </td><td>1,073 </td><td>557 </td><td>117.3 </td><td>0.21 </td></tr><tr><td>9 </td><td>1,100 </td><td>604 </td><td>146.9 </td><td>0.24 </td></tr><tr><td>10 </td><td>913 </td><td>475 </td><td>105.8 </td><td>0.22 </td></tr><tr><td>11 </td><td>999 </td><td>572 </td><td>152.7 </td><td>0.27 </td></tr><tr><td>12 </td><td>1,216 </td><td>625 </td><td>104.8 </td><td>0.17 </td></tr><tr><td>13 </td><td>1,176 </td><td>630 </td><td>131.1 </td><td>0.21 </td></tr><tr><td>14 </td><td>868 </td><td>472 </td><td>113.8 </td><td>0.24 </td></tr><tr><td>15 </td><td>1,047 </td><td>562 </td><td>115.7 </td><td>0.21 </td></tr><tr><td>16 </td><td>1,219 </td><td>695 </td><td>145.3 </td><td>0.21 </td></tr><tr><td>17 </td><td>1,178 </td><td>623 </td><td>138.2 </td><td>0.22 </td></tr><tr><td>18 </td><td>782 </td><td>388 </td><td>114.8 </td><td>0.30 </td></tr><tr><td>19 </td><td>914 </td><td>523 </td><td>118.9 </td><td>0.23 </td></tr><tr><td>20 </td><td>1,029 </td><td>550 </td><td>126 </td><td>0.23 </td></tr><tr><td>21 </td><td>968 </td><td>484 </td><td>110.3 </td><td>0.23 </td></tr><tr><td>22 </td><td>779 </td><td>364 </td><td>68.4 </td><td>0.19 </td></tr><tr><td>23 </td><td>929 </td><td>486 </td><td>93.3 </td><td>0.19 </td></tr><tr><td>24 </td><td>854 </td><td>492 </td><td>106.9 </td><td>0.22 </td></tr><tr><td>25 </td><td>1,097 </td><td>584 </td><td>127.6 </td><td>0.22 </td></tr><tr><td>26 </td><td>811 </td><td>427 </td><td>101.2 </td><td>0.24 </td></tr><tr><td>27 </td><td>925 </td><td>516 </td><td>113.7 </td><td>0.22 </td></tr><tr><td>28 </td><td>969 </td><td>516 </td><td>114.1 </td><td>0.22 </td></tr><tr><td>29 </td><td>704 </td><td>421 </td><td>109.5 </td><td>0.26 </td></tr><tr><td>Total </td><td>29,081 </td><td>15,598 </td><td>3,505.4 </td><td>0.22 </td></tr></tbody></table></div><div content-id="DSU038TB4" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Linkage group </th><th colspan="4">Sex-averaged map<hr /> </th></tr><tr><th>Mapped markers </th><th>Distinct positions </th><th>Genetic length (cM) </th><th>Marker interval (cM) </th></tr></thead><tbody><tr><td>1 </td><td>1,039 </td><td>556 </td><td>139.5 </td><td>0.25 </td></tr><tr><td>2 </td><td>1,014 </td><td>557 </td><td>116.2 </td><td>0.21 </td></tr><tr><td>3 </td><td>1,212 </td><td>673 </td><td>169.7 </td><td>0.25 </td></tr><tr><td>4 </td><td>1,220 </td><td>653 </td><td>128.1 </td><td>0.20 </td></tr><tr><td>5 </td><td>1,002 </td><td>491 </td><td>110.2 </td><td>0.22 </td></tr><tr><td>6 </td><td>1,183 </td><td>680 </td><td>163.3 </td><td>0.24 </td></tr><tr><td>7 </td><td>861 </td><td>422 </td><td>102.1 </td><td>0.24 </td></tr><tr><td>8 </td><td>1,073 </td><td>557 </td><td>117.3 </td><td>0.21 </td></tr><tr><td>9 </td><td>1,100 </td><td>604 </td><td>146.9 </td><td>0.24 </td></tr><tr><td>10 </td><td>913 </td><td>475 </td><td>105.8 </td><td>0.22 </td></tr><tr><td>11 </td><td>999 </td><td>572 </td><td>152.7 </td><td>0.27 </td></tr><tr><td>12 </td><td>1,216 </td><td>625 </td><td>104.8 </td><td>0.17 </td></tr><tr><td>13 </td><td>1,176 </td><td>630 </td><td>131.1 </td><td>0.21 </td></tr><tr><td>14 </td><td>868 </td><td>472 </td><td>113.8 </td><td>0.24 </td></tr><tr><td>15 </td><td>1,047 </td><td>562 </td><td>115.7 </td><td>0.21 </td></tr><tr><td>16 </td><td>1,219 </td><td>695 </td><td>145.3 </td><td>0.21 </td></tr><tr><td>17 </td><td>1,178 </td><td>623 </td><td>138.2 </td><td>0.22 </td></tr><tr><td>18 </td><td>782 </td><td>388 </td><td>114.8 </td><td>0.30 </td></tr><tr><td>19 </td><td>914 </td><td>523 </td><td>118.9 </td><td>0.23 </td></tr><tr><td>20 </td><td>1,029 </td><td>550 </td><td>126 </td><td>0.23 </td></tr><tr><td>21 </td><td>968 </td><td>484 </td><td>110.3 </td><td>0.23 </td></tr><tr><td>22 </td><td>779 </td><td>364 </td><td>68.4 </td><td>0.19 </td></tr><tr><td>23 </td><td>929 </td><td>486 </td><td>93.3 </td><td>0.19 </td></tr><tr><td>24 </td><td>854 </td><td>492 </td><td>106.9 </td><td>0.22 </td></tr><tr><td>25 </td><td>1,097 </td><td>584 </td><td>127.6 </td><td>0.22 </td></tr><tr><td>26 </td><td>811 </td><td>427 </td><td>101.2 </td><td>0.24 </td></tr><tr><td>27 </td><td>925 </td><td>516 </td><td>113.7 </td><td>0.22 </td></tr><tr><td>28 </td><td>969 </td><td>516 </td><td>114.1 </td><td>0.22 </td></tr><tr><td>29 </td><td>704 </td><td>421 </td><td>109.5 </td><td>0.26 </td></tr><tr><td>Total </td><td>29,081 </td><td>15,598 </td><td>3,505.4 </td><td>0.22 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6578263" target="_blank">View Large</a></div></div>                        <a id="6578264" scrollto-destination="6578264"></a>
<div id="DSU038F3" data-id="DSU038F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu038/2/m_dsu03803.png?Expires=2147483647&Signature=D15wJGUhpCWgHZbKbiQDiY9cbk0sHn1nIPqFgSkssEfB1~WziTh5PDzUYCjTt8s2zpAMkH5XjKiTkjW-eKUrbKzI3los-vC6KVIr9puwHWBTxDKMjmNakNeCzvE5ygrBAMY~Kig32cwqxeIx8c7Id8gdjjN9HXbj2MFc8iIm~uYx~TuGSEphIy2rdj~m5r9pBADtoDdCRIgynX3EQzsw2~-bzdwWh1jrZCbrxQuqdi1~brVsMzAw71ehMia0fT6R9AsVgfWmTUg27LLHU3rAg-Lrcct7M1azIMCkxybYtF4zPnRg1nFWrzucaAd3rRglL~Ss3vukIn8MAgmOe2mZvA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Illustration of sex-averaged linkage map." path-from-xml="dsu03803.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6578264/dsu03803.tif" path-from-xml="dsu03803.tif" target="_blank">View large</a><a section="6578264" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu038/2/dsu03803.png?Expires=2147483647&Signature=gqB3pzsmSoHxoLsAlcISOTWv-T4A2rflV29ZFNUH6D4~qjAJCByrCGhue30RH1V6FI2PtDSg6Ku9mSIr~wwrliOpMxeRJleJEFeJuxJBDkvKlJ-REskLM-CnP7-9U4qb1vbi37qrgvoYVhTFGoc8x2Xt5oN6GygGi2rxpcFxaTmaR3z~KSg5NCSCJme7bFPrAJ7TN-vxlJGe2iNdT3mdgSVqYm~5g4km0kegRZLUQon3JrtoiizKRZVm6H4dCwA2AQCU6J27hcOj10~q1AZFjiUJJOX6jD9wjzzQtQQsZAHA95bvRNQe5Je~odOj-GzHpBFzBmXZnAdL3~zxD47hOA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6578264&ar=440428&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03803.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Illustration of sex-averaged linkage map.</p></div></div><div content-id="DSU038F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu038/2/m_dsu03803.png?Expires=2147483647&Signature=D15wJGUhpCWgHZbKbiQDiY9cbk0sHn1nIPqFgSkssEfB1~WziTh5PDzUYCjTt8s2zpAMkH5XjKiTkjW-eKUrbKzI3los-vC6KVIr9puwHWBTxDKMjmNakNeCzvE5ygrBAMY~Kig32cwqxeIx8c7Id8gdjjN9HXbj2MFc8iIm~uYx~TuGSEphIy2rdj~m5r9pBADtoDdCRIgynX3EQzsw2~-bzdwWh1jrZCbrxQuqdi1~brVsMzAw71ehMia0fT6R9AsVgfWmTUg27LLHU3rAg-Lrcct7M1azIMCkxybYtF4zPnRg1nFWrzucaAd3rRglL~Ss3vukIn8MAgmOe2mZvA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Illustration of sex-averaged linkage map." path-from-xml="dsu03803.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6578264/dsu03803.tif" path-from-xml="dsu03803.tif" target="_blank">View large</a><a section="6578264" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu038/2/dsu03803.png?Expires=2147483647&Signature=gqB3pzsmSoHxoLsAlcISOTWv-T4A2rflV29ZFNUH6D4~qjAJCByrCGhue30RH1V6FI2PtDSg6Ku9mSIr~wwrliOpMxeRJleJEFeJuxJBDkvKlJ-REskLM-CnP7-9U4qb1vbi37qrgvoYVhTFGoc8x2Xt5oN6GygGi2rxpcFxaTmaR3z~KSg5NCSCJme7bFPrAJ7TN-vxlJGe2iNdT3mdgSVqYm~5g4km0kegRZLUQon3JrtoiizKRZVm6H4dCwA2AQCU6J27hcOj10~q1AZFjiUJJOX6jD9wjzzQtQQsZAHA95bvRNQe5Je~odOj-GzHpBFzBmXZnAdL3~zxD47hOA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6578264&ar=440428&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03803.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Illustration of sex-averaged linkage map.</p></div></div>                            <h3 scrollto-destination=6578265 id="6578265" class="section-title" data-legacy-id=s3c>3.3. Analysis of recombination rates</h3>
<p>Within each linkage group, mild to strong localized specific recombination patterns were observed, whereby recombination rates were usually elevated towards the end and suppressed in the middle of the map (Fig. <a reveal-id="DSU038F4" data-open="DSU038F4" class="link link-reveal link-table xref-fig">4</a>). Clustered markers that fell into ‘zero recombination rate’ were observed in every linkage group, especially in positions close to the centromeres in contrast to the telomeres (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data S1</a>). </p>                        <a id="6578267" scrollto-destination="6578267"></a>
<div id="DSU038F4" data-id="DSU038F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu038/2/m_dsu03804.png?Expires=2147483647&Signature=0zYFrbgggSakWOqO5-bFK1vk7HM1895Jgwm65oaNds-ZQ6JzoerEyOuYxbri7-7BQWEeIsN4B9vBhMrftpPynXh-eHStlYuUk3tWp7vsXJjaybFOiXIJW-x8~OirdV5AD7xij7IQRIAFDvB2y~Y~vlOybgiTzGJCqIf9I9mc1rMdllRcS1kpDKQUlnyryv0-WNq7UDzMTyAKOEUYR2v7x-uXh89spWPj3OZeDgEwZFpfogL7sqyCxQt6p6yLz1lgy9ecEg9KXb4ZjuIVdRLSngSTfnZkOgYlPCO1PVXjxGprDy3Pqti66zlzQSRYD19ptWffPSOsXtEeMqngUm4Alg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The patterns of localized regional recombination rates along each linkage group." path-from-xml="dsu03804.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6578267/dsu03804.tif" path-from-xml="dsu03804.tif" target="_blank">View large</a><a section="6578267" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu038/2/dsu03804.png?Expires=2147483647&Signature=U0NSQoshDrmZaIChY4R6aCXmC5ZvKBTaGg7vLvlOzzUT2MGVdeCVw5lu1THu2D4lkXtGtngiIfljlHbj7MxbqS4WuQc0uPpfYjXJ4FnbIDsLO~A9dDDFTVWUoJPnPEfKtBG762~vu1Pk-GIm4ksNzUrZlb~WC~qOzI6jXQ55gKGl7-QlrvMOP1Bc6wOxJ5vJUR4ILTa2LozTmWrjbriVjmuds7Sh0Xwa-K2uwzGgyOtPpZOdYpQwgDGsJILtU7QWoMPqLdFqOizbCa0hq2XHBCJNr6bSnOYO09ZLsbuNEh1zh68fvBtfwJ3MNYTSSEb3LHEFvc4eL14D9pB0f6AlJw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6578267&ar=440428&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03804.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The patterns of localized regional recombination rates along each linkage group.</p></div></div><div content-id="DSU038F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu038/2/m_dsu03804.png?Expires=2147483647&Signature=0zYFrbgggSakWOqO5-bFK1vk7HM1895Jgwm65oaNds-ZQ6JzoerEyOuYxbri7-7BQWEeIsN4B9vBhMrftpPynXh-eHStlYuUk3tWp7vsXJjaybFOiXIJW-x8~OirdV5AD7xij7IQRIAFDvB2y~Y~vlOybgiTzGJCqIf9I9mc1rMdllRcS1kpDKQUlnyryv0-WNq7UDzMTyAKOEUYR2v7x-uXh89spWPj3OZeDgEwZFpfogL7sqyCxQt6p6yLz1lgy9ecEg9KXb4ZjuIVdRLSngSTfnZkOgYlPCO1PVXjxGprDy3Pqti66zlzQSRYD19ptWffPSOsXtEeMqngUm4Alg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The patterns of localized regional recombination rates along each linkage group." path-from-xml="dsu03804.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6578267/dsu03804.tif" path-from-xml="dsu03804.tif" target="_blank">View large</a><a section="6578267" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu038/2/dsu03804.png?Expires=2147483647&Signature=U0NSQoshDrmZaIChY4R6aCXmC5ZvKBTaGg7vLvlOzzUT2MGVdeCVw5lu1THu2D4lkXtGtngiIfljlHbj7MxbqS4WuQc0uPpfYjXJ4FnbIDsLO~A9dDDFTVWUoJPnPEfKtBG762~vu1Pk-GIm4ksNzUrZlb~WC~qOzI6jXQ55gKGl7-QlrvMOP1Bc6wOxJ5vJUR4ILTa2LozTmWrjbriVjmuds7Sh0Xwa-K2uwzGgyOtPpZOdYpQwgDGsJILtU7QWoMPqLdFqOizbCa0hq2XHBCJNr6bSnOYO09ZLsbuNEh1zh68fvBtfwJ3MNYTSSEb3LHEFvc4eL14D9pB0f6AlJw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6578267&ar=440428&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03804.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The patterns of localized regional recombination rates along each linkage group.</p></div></div><p>There is no significant difference in recombination rate among three mapping families based on the examination with common marker pairs among the three families (see Methods section). In contrast, significant higher recombination rates were observed in a majority of the linkage groups of the female genetic map than that of male genetic map (<em>P</em> &lt; 0.01). Overall, the female genetic map was 1,901.4 cM longer than male genetic map, with an average female-to-male ratio of 1.7 : 1 (Table <a reveal-id="DSU038TB3" data-open="DSU038TB3" class="link link-reveal link-table xref-fig">3</a>). The ratio varied by linkage groups, ranging from 0.97 : 1 in LG22 to 2.50 : 1 in LG1 (Table <a reveal-id="DSU038TB3" data-open="DSU038TB3" class="link link-reveal link-table xref-fig">3</a>). The largest differences in recombination rate between the female and male maps were observed in LG1, LG5, LG7, LG18, LG27, and LG29. Across all these linkage groups, the female : male recombination ratios exceeded 2.0. LG22 was unusual in that recombination rates were very similar between the sexes (i.e. 0.97 : 1). Within unique map positions, 504 SNPs were commonly shared between the two sexes, while significantly higher recombination rates were observed in majority of the common locus intervals (<em>P</em> &lt; 0.01) (Fig. <a reveal-id="DSU038F5" data-open="DSU038F5" class="link link-reveal link-table xref-fig">5</a>). </p>                        <a id="6578269" scrollto-destination="6578269"></a>
<div id="DSU038F5" data-id="DSU038F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu038/2/m_dsu03805.png?Expires=2147483647&Signature=RjW6OUBznEYA489drKI8nAycohoHVP6~B3DiPTBAsqctVdsbZEtZ7fZcyoTXYNcqr5EDRJ5a-ftI3jjPWH81CifWRms3Y7ixE-Tq1726mlwZRd2owI6IROcQhRjwUx9BDkUi51sVK3WgnDI3SWdGXMvYsjmO6PjDJ33GQCMJRVev89Fx7~ux6pBULRCCXtX6lrd0WkO25ny~rilHpJ56kmdyJww8R6gc7XiDjLFKQOSmFFSO6jEFKtCHpYV1WOac8hQFxDmpTwIZ0MT-CYJLCXH6FgamoktKXa-sVcSIt~W8x4uMkvqAt4L2pA-V4dHA-fVufDoydbWMXm8almvLXA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of the recombination rate between female and male. The inter-marker distances (cM) for all pairs of adjacent markers from both female- and male-specific maps were used. The diagonal line represents sex-equal recombination rates." path-from-xml="dsu03805.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6578269/dsu03805.tif" path-from-xml="dsu03805.tif" target="_blank">View large</a><a section="6578269" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu038/2/dsu03805.png?Expires=2147483647&Signature=FBzlNX0jFYG2TMh2IfJmf21xl09MHymR~aexnV-d9QvPtZXL02lDBhtXjxD17TNlCeHzYoPNIub2zyuuIiiWq0HFYy-AmpSiKcydzsMTP-5nXqGqO7s6B2~7Dbl0rUopKNpILug6rft5Viiq22CYka~W0w4h6o6DaMMRP0j24eXzMjm6bHyqPJ~ylL5xdQViy9ILlrlAuSin4HRPxMhjZZRkDtGD5n5IDG2iXqnuDFB1DjwzJIS7RNhh3Fkd3Pr~o4r-4HvYalcTpguVWOY4qTj--PhfooXI9lmlcIE10WxmY36ra8dw8sOq6Fy345qg4ahkDE8aRSuwMpzi1B~gQw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6578269&ar=440428&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03805.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of the recombination rate between female and male. The inter-marker distances (cM) for all pairs of adjacent markers from both female- and male-specific maps were used. The diagonal line represents sex-equal recombination rates.</p></div></div><div content-id="DSU038F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu038/2/m_dsu03805.png?Expires=2147483647&Signature=RjW6OUBznEYA489drKI8nAycohoHVP6~B3DiPTBAsqctVdsbZEtZ7fZcyoTXYNcqr5EDRJ5a-ftI3jjPWH81CifWRms3Y7ixE-Tq1726mlwZRd2owI6IROcQhRjwUx9BDkUi51sVK3WgnDI3SWdGXMvYsjmO6PjDJ33GQCMJRVev89Fx7~ux6pBULRCCXtX6lrd0WkO25ny~rilHpJ56kmdyJww8R6gc7XiDjLFKQOSmFFSO6jEFKtCHpYV1WOac8hQFxDmpTwIZ0MT-CYJLCXH6FgamoktKXa-sVcSIt~W8x4uMkvqAt4L2pA-V4dHA-fVufDoydbWMXm8almvLXA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of the recombination rate between female and male. The inter-marker distances (cM) for all pairs of adjacent markers from both female- and male-specific maps were used. The diagonal line represents sex-equal recombination rates." path-from-xml="dsu03805.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6578269/dsu03805.tif" path-from-xml="dsu03805.tif" target="_blank">View large</a><a section="6578269" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu038/2/dsu03805.png?Expires=2147483647&Signature=FBzlNX0jFYG2TMh2IfJmf21xl09MHymR~aexnV-d9QvPtZXL02lDBhtXjxD17TNlCeHzYoPNIub2zyuuIiiWq0HFYy-AmpSiKcydzsMTP-5nXqGqO7s6B2~7Dbl0rUopKNpILug6rft5Viiq22CYka~W0w4h6o6DaMMRP0j24eXzMjm6bHyqPJ~ylL5xdQViy9ILlrlAuSin4HRPxMhjZZRkDtGD5n5IDG2iXqnuDFB1DjwzJIS7RNhh3Fkd3Pr~o4r-4HvYalcTpguVWOY4qTj--PhfooXI9lmlcIE10WxmY36ra8dw8sOq6Fy345qg4ahkDE8aRSuwMpzi1B~gQw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6578269&ar=440428&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03805.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of the recombination rate between female and male. The inter-marker distances (cM) for all pairs of adjacent markers from both female- and male-specific maps were used. The diagonal line represents sex-equal recombination rates.</p></div></div>                            <h3 scrollto-destination=6578270 id="6578270" class="section-title" data-legacy-id=s3d>3.4. Integration and validation with physical map</h3>
<p>The integration of the genetic map with the BAC-based physical map anchored 2,728 (83%) of the 3,307 physical map contigs (Table <a reveal-id="DSU038TB5" data-open="DSU038TB5" class="link link-reveal link-table xref-fig">5</a>). Together with the 1,481 (44.8%) physical map contigs that were anchored previously,<sup><a reveal-id="DSU038C6" data-open="DSU038C6" class="link link-ref link-reveal xref-bibr">6</a></sup> we were able to anchor a total of 2,880 (87.1%) physical map contigs consisting of 28,416 BAC clones (92.9% of total BAC clones). The sizes of anchored physical map contigs varied from 66.0 kb to 2,005.8 kb (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data S2</a>), with an average size of 301 kb. Together, a total of 867.4 Mb of the physical map was integrated by genetic linkage map that accounted for ∼90% of the channel catfish genome (Table <a reveal-id="DSU038TB5" data-open="DSU038TB5" class="link link-reveal link-table xref-fig">5</a>). Detailed information regarding integration of linkage map and physical map is provided in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data S2</a>. </p>                        <a id="6578272" scrollto-destination="6578272"></a>
<div content-id="DSU038TB5" class="table-modal"><div class="table-wrap"><div id="DSU038TB5" class="table-wrap-title" data-id="DSU038TB5"><div class="label -label">Table 5.</div><div class="caption"><p>Integration of channel catfish linkage map with physical map</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Item </th><th>Number </th><th>Percentage </th></tr></thead><tbody><tr><td>Number of physical map contigs<sup>a</sup> </td><td>3,307 </td><td>100 </td></tr><tr><td>Number of physical map contigs with SNP markers </td><td>3,243 </td><td>98.1 </td></tr><tr><td>Number of physical map contigs anchored to linkage map in this work </td><td>2,728 </td><td>82.5 </td></tr><tr><td>Number of physical map contigs anchored in previous study<sup><sup>b</sup></sup> </td><td>1,481 </td><td>44.8 </td></tr><tr><td>Total physical map contigs mapped to linkage map </td><td>2,880 </td><td>87.1 </td></tr><tr><td>Total BAC clones in the catfish physical map<sup>a</sup> </td><td>30,582 </td><td>100 </td></tr><tr><td>Total BAC clones mapped on the linkage map </td><td>28,416 </td><td>92.9 </td></tr><tr><td>Total size of physical map contigs<sup>a</sup> </td><td>965,279 </td><td>100 </td></tr><tr><td>Total size of anchored physical map contigs </td><td>867,364 </td><td>89.9 </td></tr></tbody></table></div><div content-id="DSU038TB5" class="table-modal"><table><thead name="thead" align=""><tr><th>Item </th><th>Number </th><th>Percentage </th></tr></thead><tbody><tr><td>Number of physical map contigs<sup>a</sup> </td><td>3,307 </td><td>100 </td></tr><tr><td>Number of physical map contigs with SNP markers </td><td>3,243 </td><td>98.1 </td></tr><tr><td>Number of physical map contigs anchored to linkage map in this work </td><td>2,728 </td><td>82.5 </td></tr><tr><td>Number of physical map contigs anchored in previous study<sup><sup>b</sup></sup> </td><td>1,481 </td><td>44.8 </td></tr><tr><td>Total physical map contigs mapped to linkage map </td><td>2,880 </td><td>87.1 </td></tr><tr><td>Total BAC clones in the catfish physical map<sup>a</sup> </td><td>30,582 </td><td>100 </td></tr><tr><td>Total BAC clones mapped on the linkage map </td><td>28,416 </td><td>92.9 </td></tr><tr><td>Total size of physical map contigs<sup>a</sup> </td><td>965,279 </td><td>100 </td></tr><tr><td>Total size of anchored physical map contigs </td><td>867,364 </td><td>89.9 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><div class="label -label">a</div><p>Data obtained from Xu <em>et al.</em><sup><a reveal-id="DSU038C6" data-open="DSU038C6" class="link link-ref link-reveal xref-bibr">6</a></sup></p></div></div><div content-id="TN2"><div class="fn"><div class="label -label">b</div><p>Data obtained from Ninwichian <em>et al.</em><sup><a reveal-id="DSU038C9" data-open="DSU038C9" class="link link-ref link-reveal xref-bibr">9</a></sup></p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6578272" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU038TB5" class="table-wrap-title" data-id="DSU038TB5"><div class="label -label">Table 5.</div><div class="caption"><p>Integration of channel catfish linkage map with physical map</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Item </th><th>Number </th><th>Percentage </th></tr></thead><tbody><tr><td>Number of physical map contigs<sup>a</sup> </td><td>3,307 </td><td>100 </td></tr><tr><td>Number of physical map contigs with SNP markers </td><td>3,243 </td><td>98.1 </td></tr><tr><td>Number of physical map contigs anchored to linkage map in this work </td><td>2,728 </td><td>82.5 </td></tr><tr><td>Number of physical map contigs anchored in previous study<sup><sup>b</sup></sup> </td><td>1,481 </td><td>44.8 </td></tr><tr><td>Total physical map contigs mapped to linkage map </td><td>2,880 </td><td>87.1 </td></tr><tr><td>Total BAC clones in the catfish physical map<sup>a</sup> </td><td>30,582 </td><td>100 </td></tr><tr><td>Total BAC clones mapped on the linkage map </td><td>28,416 </td><td>92.9 </td></tr><tr><td>Total size of physical map contigs<sup>a</sup> </td><td>965,279 </td><td>100 </td></tr><tr><td>Total size of anchored physical map contigs </td><td>867,364 </td><td>89.9 </td></tr></tbody></table></div><div content-id="DSU038TB5" class="table-modal"><table><thead name="thead" align=""><tr><th>Item </th><th>Number </th><th>Percentage </th></tr></thead><tbody><tr><td>Number of physical map contigs<sup>a</sup> </td><td>3,307 </td><td>100 </td></tr><tr><td>Number of physical map contigs with SNP markers </td><td>3,243 </td><td>98.1 </td></tr><tr><td>Number of physical map contigs anchored to linkage map in this work </td><td>2,728 </td><td>82.5 </td></tr><tr><td>Number of physical map contigs anchored in previous study<sup><sup>b</sup></sup> </td><td>1,481 </td><td>44.8 </td></tr><tr><td>Total physical map contigs mapped to linkage map </td><td>2,880 </td><td>87.1 </td></tr><tr><td>Total BAC clones in the catfish physical map<sup>a</sup> </td><td>30,582 </td><td>100 </td></tr><tr><td>Total BAC clones mapped on the linkage map </td><td>28,416 </td><td>92.9 </td></tr><tr><td>Total size of physical map contigs<sup>a</sup> </td><td>965,279 </td><td>100 </td></tr><tr><td>Total size of anchored physical map contigs </td><td>867,364 </td><td>89.9 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><div class="label -label">a</div><p>Data obtained from Xu <em>et al.</em><sup><a reveal-id="DSU038C6" data-open="DSU038C6" class="link link-ref link-reveal xref-bibr">6</a></sup></p></div></div><div content-id="TN2"><div class="fn"><div class="label -label">b</div><p>Data obtained from Ninwichian <em>et al.</em><sup><a reveal-id="DSU038C9" data-open="DSU038C9" class="link link-ref link-reveal xref-bibr">9</a></sup></p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6578272" target="_blank">View Large</a></div></div><p>The integration of linkage map and physical map enabled the cross-validation of the quality of the physical map and the genetic map (Table <a reveal-id="DSU038TB6" data-open="DSU038TB6" class="link link-reveal link-table xref-fig">6</a>). A total of 1,467 physical map contigs containing 2,961 BES-associated SNPs were placed onto linkage maps. With 615 physical map contigs, two or more SNP markers were mapped that allowed the determination of orientation of the physical map contigs along the chromosome. With such contigs, the quality of physical map contigs was assessed based on the mapped markers. An illustration with linkage group 12 is provided in Fig. <a reveal-id="DSU038F6" data-open="DSU038F6" class="link link-reveal link-table xref-fig">6</a>. The vast majority of physical contigs were validated, but a small fraction of physical map contigs was found incorrectly assembled, or the ordering of the SNP markers onto the map was in error. </p>                        <a id="6578274" scrollto-destination="6578274"></a>
<div content-id="DSU038TB6" class="table-modal"><div class="table-wrap"><div id="DSU038TB6" class="table-wrap-title" data-id="DSU038TB6"><div class="label -label">Table 6.</div><div class="caption"><p>Cross-validation between linkage map and physical map</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Item </th><th>Number </th></tr></thead><tbody><tr><td>Number of physical map contigs with BES-associated SNPs </td><td>2,790 </td></tr><tr><td>Number of physical map contigs anchored through BES-associated SNPs </td><td>1,467 </td></tr><tr><td>Number of physical map contigs with at least two BES-associated SNPs </td><td>615 </td></tr><tr><td>Number of physical map contigs mapped onto different LGs </td><td>47 </td></tr><tr><td>Number of BAC clones with BES-associated SNPs </td><td>2,707 </td></tr><tr><td>Number of BAC clones with at least two BES-associated SNPs </td><td>238 </td></tr><tr><td>Number of BAC clones mapped onto different LGs </td><td>7 </td></tr></tbody></table></div><div content-id="DSU038TB6" class="table-modal"><table><thead name="thead" align=""><tr><th>Item </th><th>Number </th></tr></thead><tbody><tr><td>Number of physical map contigs with BES-associated SNPs </td><td>2,790 </td></tr><tr><td>Number of physical map contigs anchored through BES-associated SNPs </td><td>1,467 </td></tr><tr><td>Number of physical map contigs with at least two BES-associated SNPs </td><td>615 </td></tr><tr><td>Number of physical map contigs mapped onto different LGs </td><td>47 </td></tr><tr><td>Number of BAC clones with BES-associated SNPs </td><td>2,707 </td></tr><tr><td>Number of BAC clones with at least two BES-associated SNPs </td><td>238 </td></tr><tr><td>Number of BAC clones mapped onto different LGs </td><td>7 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6578274" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU038TB6" class="table-wrap-title" data-id="DSU038TB6"><div class="label -label">Table 6.</div><div class="caption"><p>Cross-validation between linkage map and physical map</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Item </th><th>Number </th></tr></thead><tbody><tr><td>Number of physical map contigs with BES-associated SNPs </td><td>2,790 </td></tr><tr><td>Number of physical map contigs anchored through BES-associated SNPs </td><td>1,467 </td></tr><tr><td>Number of physical map contigs with at least two BES-associated SNPs </td><td>615 </td></tr><tr><td>Number of physical map contigs mapped onto different LGs </td><td>47 </td></tr><tr><td>Number of BAC clones with BES-associated SNPs </td><td>2,707 </td></tr><tr><td>Number of BAC clones with at least two BES-associated SNPs </td><td>238 </td></tr><tr><td>Number of BAC clones mapped onto different LGs </td><td>7 </td></tr></tbody></table></div><div content-id="DSU038TB6" class="table-modal"><table><thead name="thead" align=""><tr><th>Item </th><th>Number </th></tr></thead><tbody><tr><td>Number of physical map contigs with BES-associated SNPs </td><td>2,790 </td></tr><tr><td>Number of physical map contigs anchored through BES-associated SNPs </td><td>1,467 </td></tr><tr><td>Number of physical map contigs with at least two BES-associated SNPs </td><td>615 </td></tr><tr><td>Number of physical map contigs mapped onto different LGs </td><td>47 </td></tr><tr><td>Number of BAC clones with BES-associated SNPs </td><td>2,707 </td></tr><tr><td>Number of BAC clones with at least two BES-associated SNPs </td><td>238 </td></tr><tr><td>Number of BAC clones mapped onto different LGs </td><td>7 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6578274" target="_blank">View Large</a></div></div>                        <a id="6578275" scrollto-destination="6578275"></a>
<div id="DSU038F6" data-id="DSU038F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu038/2/m_dsu03806.png?Expires=2147483647&Signature=j7Wm0Yc~kONjOOv1nVGMgHNHvI-JJz0nJLLwK1BHfJFD6YPBA6Pdiz9UxUeOOo4~XePLXTCwRFAwJqWxfXHN51JISEk~nLA15qdEuStTwbkw5sQlgdMDwhKujM-N2AIj7Vr1-QkNBajiuwKm-TLnH-DrDQYMCo3dP7EuATApDwpyqXhR792R2XW-z7B-9EbpNadhXP0cq-00SKHOh3MU~TQp4ljKBGJNDNF-8wIrhDv7-ommyqaP1DRd4ZqjaUSW5JkUEXG2~CjtpTd~W9ljg76KZCWxc1SOR~8y~sm8NCRJGGS11ZbXKKdBdn1TNOWQ8uiq~Ri8T0-psn57x2o95w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Illustration of integration of the linkage map (left) with the physical map (right). The linkage group 12 (LG12) was used for the illustration. Vertical bars represent physical map contigs containing at least two SNPs mapped onto the linkage map, and the dot spots indicate physical map contigs with only one SNP." path-from-xml="dsu03806.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6578275/dsu03806.tif" path-from-xml="dsu03806.tif" target="_blank">View large</a><a section="6578275" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu038/2/dsu03806.png?Expires=2147483647&Signature=h7U7l~K0MsOp9bJOY7IdOypOKkM9l7bWi8gtg8j6PHXANlfMK-VdCRo-MS76QPEee3XlDr0KZ41BORmXoWq6ql1cJFjk3d9s4JMrXpzkLEWuKht9i1P4mVDWHj-YWFffJX9ll6oMzmTrhCle-BqTZVdeg6AIqSpKT3JhoaTIAe50AQVP3SWqfS917-HW-CF7cgjDFXN8jnpXJGZ-ARKVX34Pi35Dz5Jc-aS8laEA6AUttQDkAUzMRH8a-zHSncm2RsWFOVeXh7qPrDphHS6dGreXaJlgyV9exuozOU0sfnHum395-z2~cGDld0vvAsuuEASPg220TwyqII09GgYWbQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6578275&ar=440428&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03806.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Illustration of integration of the linkage map (left) with the physical map (right). The linkage group 12 (LG12) was used for the illustration. Vertical bars represent physical map contigs containing at least two SNPs mapped onto the linkage map, and the dot spots indicate physical map contigs with only one SNP.</p></div></div><div content-id="DSU038F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu038/2/m_dsu03806.png?Expires=2147483647&Signature=j7Wm0Yc~kONjOOv1nVGMgHNHvI-JJz0nJLLwK1BHfJFD6YPBA6Pdiz9UxUeOOo4~XePLXTCwRFAwJqWxfXHN51JISEk~nLA15qdEuStTwbkw5sQlgdMDwhKujM-N2AIj7Vr1-QkNBajiuwKm-TLnH-DrDQYMCo3dP7EuATApDwpyqXhR792R2XW-z7B-9EbpNadhXP0cq-00SKHOh3MU~TQp4ljKBGJNDNF-8wIrhDv7-ommyqaP1DRd4ZqjaUSW5JkUEXG2~CjtpTd~W9ljg76KZCWxc1SOR~8y~sm8NCRJGGS11ZbXKKdBdn1TNOWQ8uiq~Ri8T0-psn57x2o95w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Illustration of integration of the linkage map (left) with the physical map (right). The linkage group 12 (LG12) was used for the illustration. Vertical bars represent physical map contigs containing at least two SNPs mapped onto the linkage map, and the dot spots indicate physical map contigs with only one SNP." path-from-xml="dsu03806.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6578275/dsu03806.tif" path-from-xml="dsu03806.tif" target="_blank">View large</a><a section="6578275" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu038/2/dsu03806.png?Expires=2147483647&Signature=h7U7l~K0MsOp9bJOY7IdOypOKkM9l7bWi8gtg8j6PHXANlfMK-VdCRo-MS76QPEee3XlDr0KZ41BORmXoWq6ql1cJFjk3d9s4JMrXpzkLEWuKht9i1P4mVDWHj-YWFffJX9ll6oMzmTrhCle-BqTZVdeg6AIqSpKT3JhoaTIAe50AQVP3SWqfS917-HW-CF7cgjDFXN8jnpXJGZ-ARKVX34Pi35Dz5Jc-aS8laEA6AUttQDkAUzMRH8a-zHSncm2RsWFOVeXh7qPrDphHS6dGreXaJlgyV9exuozOU0sfnHum395-z2~cGDld0vvAsuuEASPg220TwyqII09GgYWbQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6578275&ar=440428&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03806.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Illustration of integration of the linkage map (left) with the physical map (right). The linkage group 12 (LG12) was used for the illustration. Vertical bars represent physical map contigs containing at least two SNPs mapped onto the linkage map, and the dot spots indicate physical map contigs with only one SNP.</p></div></div><p>A total of 47 physical map contigs had markers that were mapped in different linkage groups, indicating the potential errors of the physical map assembly or the mapping errors of incorrectly assigned SNPs. These errors could be caused by SNPs falling into duplicated regions of genome. Moreover, of the 2,707 BAC clones with BES-associated SNPs, 238 BAC clones contained at least two BES-associated SNPs. Within this group, 7 BAC clones were mapped onto different LGs, suggesting the misplacement of those SNP markers onto the linkage groups. As an illustration, the integration of sex-averaged linkage map with physical map Contig72 and Contig534 is shown in Fig. <a reveal-id="DSU038F7" data-open="DSU038F7" class="link link-reveal link-table xref-fig">7</a>, presenting the correct and incorrect physical map assembly, respectively. The manual check of the potential assembly or mapping error warrants the correctness of both physical map and linkage map in the next step. </p>                        <a id="6578277" scrollto-destination="6578277"></a>
<div id="DSU038F7" data-id="DSU038F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu038/2/m_dsu03807.png?Expires=2147483647&Signature=EQ-sDqq59xZzobrb6m-uf-hfxymoqQ3pskHf4qtyZOXi4Sd0NAabLXcqJ9T0cbhnu4NQ57UszXCIfkmaQzl1FQGix1zc9sVjrK5elbfya2RQWOtQNqHJRvzSS8TJIUgp0w-VHcBPCfyXYFTRJXWVwMqc8q0FFRsYWe5XrA~XNFlfjNWwdcHQKyjXZcH3Pkz182wBLvJPQINktHmweTsFFkhqE2YvHa5-CXM-s8fOjsHAA0rZvP~KGjoDH3NEkUUe4sVjUVn92diiAouMp0zLwfEWwNExpBlU7pEZal-wDFRAJMtR1r~72JMzG63mDAPlOj-M7BnywBpxdFh6SyM0pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Example of validation of physical map using the linkage map. (A) shows the correct physical map contig using Contig72 as an example; (B) shows the incorrect physical map contig using Contig534 as an example." path-from-xml="dsu03807.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6578277/dsu03807.tif" path-from-xml="dsu03807.tif" target="_blank">View large</a><a section="6578277" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu038/2/dsu03807.png?Expires=2147483647&Signature=zRlRLdM6nvNDybLrqTTOPDWj~iINhFiWr7Dzpk1KPFGA7o~Eovk~SnXxX9Sd595GgduBL2pEHv-1SkkdtZDhKjsqBFz4kg-5az0y1SqzdGY7LVfJxwzhE2p5JiCRSgNh~LAFxfZsmkHjdqQxK9sxWzrdi5cJ0Z5G7Wf19G26cBEFXbNAqrVBf8FlrLh68PochR-VtYqaFwIp3Rfj34rWJ01rs1pMyg7O5Q8yXu8b3-8yYUUuqpTl~8Srh7NGTOofjidsT3cLyYiImR1y04mxagF0ZxizTKIosFKoPlD25NofRmTevEA6NNaC-uwDEDlySJpfRt5sPjpagEBQ77XdWw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6578277&ar=440428&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03807.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Example of validation of physical map using the linkage map. (A) shows the correct physical map contig using Contig72 as an example; (B) shows the incorrect physical map contig using Contig534 as an example.</p></div></div><div content-id="DSU038F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu038/2/m_dsu03807.png?Expires=2147483647&Signature=EQ-sDqq59xZzobrb6m-uf-hfxymoqQ3pskHf4qtyZOXi4Sd0NAabLXcqJ9T0cbhnu4NQ57UszXCIfkmaQzl1FQGix1zc9sVjrK5elbfya2RQWOtQNqHJRvzSS8TJIUgp0w-VHcBPCfyXYFTRJXWVwMqc8q0FFRsYWe5XrA~XNFlfjNWwdcHQKyjXZcH3Pkz182wBLvJPQINktHmweTsFFkhqE2YvHa5-CXM-s8fOjsHAA0rZvP~KGjoDH3NEkUUe4sVjUVn92diiAouMp0zLwfEWwNExpBlU7pEZal-wDFRAJMtR1r~72JMzG63mDAPlOj-M7BnywBpxdFh6SyM0pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Example of validation of physical map using the linkage map. (A) shows the correct physical map contig using Contig72 as an example; (B) shows the incorrect physical map contig using Contig534 as an example." path-from-xml="dsu03807.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6578277/dsu03807.tif" path-from-xml="dsu03807.tif" target="_blank">View large</a><a section="6578277" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu038/2/dsu03807.png?Expires=2147483647&Signature=zRlRLdM6nvNDybLrqTTOPDWj~iINhFiWr7Dzpk1KPFGA7o~Eovk~SnXxX9Sd595GgduBL2pEHv-1SkkdtZDhKjsqBFz4kg-5az0y1SqzdGY7LVfJxwzhE2p5JiCRSgNh~LAFxfZsmkHjdqQxK9sxWzrdi5cJ0Z5G7Wf19G26cBEFXbNAqrVBf8FlrLh68PochR-VtYqaFwIp3Rfj34rWJ01rs1pMyg7O5Q8yXu8b3-8yYUUuqpTl~8Srh7NGTOofjidsT3cLyYiImR1y04mxagF0ZxizTKIosFKoPlD25NofRmTevEA6NNaC-uwDEDlySJpfRt5sPjpagEBQ77XdWw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6578277&ar=440428&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03807.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Example of validation of physical map using the linkage map. (A) shows the correct physical map contig using Contig72 as an example; (B) shows the incorrect physical map contig using Contig534 as an example.</p></div></div><p>Physical distances were estimated based on the position of SNP markers within each physical map contig, which allowed for estimation of the ratios of physical distance to genetic distance (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data S3</a>). On the other hand, recombination frequencies can be calculated for physical map contigs with multiple SNP markers mapped onto the linkage map. With the sex-averaged linkage map, the genetic size of 3,505.4 cM and the physical size of 965,279 kb based on the physical map, we estimated the average genetic size as 3.6 cM per Mb. The ratios observed from the 250 physical map contigs that contained multiple SNPs mapped onto the linkage map ranged from 0 to 2.26 cM per Mb. The markers with higher ratios might indicate the potential loci of recombination hot spots.</p>                            <h2 scrollto-destination=6578279 id="6578279" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>In this work, we constructed a high-density genetic map for channel catfish. This map possesses the highest marker density among all the genetic linkage maps constructed for any aquaculture species. Taking advantage of the catfish 250K SNP array, we were able to efficiently and cost-effectively genotype 250,000 SNPs in the 576 fish from three mapping families (192 fish per family). Such a high-density linkage map should be a valuable resource for analysis and fine-scale QTL mapping in catfish.</p><p>Although the efficient SNP genotyping technology allowed the construction of high-density linkage map, the high-resolution was yet to be achieved. A large number of markers fell into ‘zero recombination clusters’ where no recombination events occur during meiosis between the markers among the fish used for mapping analysis. These represent closely arrayed adjacent SNP positions, and their ordering is likely best achieved by assessing their order within the aligned physical map contigs. Conversely, it was also observed that genetic recombination rates could be artificially inflated when genotyping errors occur or when markers with large amounts of missing data are included in the analysis.<sup><a reveal-id="DSU038C42" data-open="DSU038C42" class="link link-ref link-reveal xref-bibr">42</a>,<a reveal-id="DSU038C43" data-open="DSU038C43" class="link link-ref link-reveal xref-bibr">43</a></sup> Similar situations were observed previously in zebrafish during the construction of high-density genetic map. In zebrafish, the genetic sizes of initial map were over 1,000 cM per chromosome. After removal of genotyping errors, the genetic sizes were reduced to around 100 cM.<sup><a reveal-id="DSU038C44" data-open="DSU038C44" class="link link-ref link-reveal xref-bibr">44</a></sup> To reduce the effect of clustered markers on the map construction and reduce the computing load, we picked one representative anchor marker that had the most informative meiosis from each of these clusters in the linkage map construction. Afterwards, we rejoined the markers within the ‘zero recombination clusters’ that were excluded during the initial mapping steps. This procedure relocated the informative markers onto the linkage maps based on the positions of their representative anchor marker. Such clustered markers are still valuable for the chromosome-scale scaffolding.</p><p>With the availability of such high-density mapped SNPs, the patterns of marker distribution across chromosomes can be examined. It appears that clustered markers were commonly found in regions around centromeres while less frequently found around the telomeres. This was consistent with observations in various genetic maps previously developed in fish species, such as tilapia,<sup><a reveal-id="DSU038C45" data-open="DSU038C45" class="link link-ref link-reveal xref-bibr">45</a></sup> medaka,<sup><a reveal-id="DSU038C46" data-open="DSU038C46" class="link link-ref link-reveal xref-bibr">46</a></sup> rainbow trout,<sup><a reveal-id="DSU038C47" data-open="DSU038C47" class="link link-ref link-reveal xref-bibr">47</a></sup> Atlantic salmon,<sup><a reveal-id="DSU038C12" data-open="DSU038C12" class="link link-ref link-reveal xref-bibr">12</a></sup> and catfish.<sup><a reveal-id="DSU038C9" data-open="DSU038C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSU038C16" data-open="DSU038C16" class="link link-ref link-reveal xref-bibr">16</a></sup> One potential explanation is that the centromeres contain abundant tandemly repeated, heterochromatic DNA sequences.<sup><a reveal-id="DSU038C48" data-open="DSU038C48" class="link link-ref link-reveal xref-bibr">48</a></sup> As shown in Fig. <a reveal-id="DSU038F4" data-open="DSU038F4" class="link link-reveal link-table xref-fig">4</a>, the distribution of mapped SNP markers across chromosomes in channel catfish is consistent with the observation of telomere and centromere effects,<sup><a reveal-id="DSU038C49" data-open="DSU038C49" class="link link-ref link-reveal xref-bibr">49</a></sup> which would result in a higher recombination rate near the telomeres while a lower recombination rate near the centre of the chromosomes. The recombination rates might be positively correlated with GC content, as in human,<sup><a reveal-id="DSU038C50" data-open="DSU038C50" class="link link-ref link-reveal xref-bibr">50</a></sup> pig,<sup><a reveal-id="DSU038C51" data-open="DSU038C51" class="link link-ref link-reveal xref-bibr">51</a></sup> chicken,<sup><a reveal-id="DSU038C52" data-open="DSU038C52" class="link link-ref link-reveal xref-bibr">52</a></sup> rodent,<sup><a reveal-id="DSU038C53" data-open="DSU038C53" class="link link-ref link-reveal xref-bibr">53</a></sup> and yeast,<sup><a reveal-id="DSU038C54" data-open="DSU038C54" class="link link-ref link-reveal xref-bibr">54</a></sup> or correlated with some other genomic features such as gene density and the presence of genes determining recombination hotspots. The PRDM9 gene was recently found to determine the recombination hotspots in the mammalian genomes.<sup><a reveal-id="DSU038C55  DSU038C57" data-open="DSU038C55  DSU038C57" class="link link-ref link-reveal xref-bibr">55–57</a></sup> Future investigations on recombination landscape warrant the characterization of genomic features, affecting the meiotic recombination in catfish upon the availability of fully assembled genome sequences.</p><p>The suppression of recombination in males relative to the females observed in channel catfish (averaged female-to-male ratio is 1.7 : 1) was consistent with our previous observation<sup><a reveal-id="DSU038C2" data-open="DSU038C2" class="link link-ref link-reveal xref-bibr">2</a></sup> as well as many other studies. Recombination rates differ between the two sexes in many organisms where recombination occurs more frequently in the homogametic sex than in the heterogametic sex.<sup><a reveal-id="DSU038C50" data-open="DSU038C50" class="link link-ref link-reveal xref-bibr">50</a>,<a reveal-id="DSU038C58" data-open="DSU038C58" class="link link-ref link-reveal xref-bibr">58</a></sup> The similar phenomenon was reported in many other fish species such as Atlantic salmon (1.38 : 1), rainbow trout (1.68 : 1), European sea bass (1.60 : 1), Arctic char (1.69 : 1), and silver carp (1.52 : 1).<sup><a reveal-id="DSU038C12" data-open="DSU038C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSU038C14" data-open="DSU038C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSU038C58  DSU038C61" data-open="DSU038C58  DSU038C61" class="link link-ref link-reveal xref-bibr">58–61</a></sup> Striking recombination differences between the sexes were observed in zebrafish (2.74 : 1)<sup><a reveal-id="DSU038C58" data-open="DSU038C58" class="link link-ref link-reveal xref-bibr">58</a></sup> and grass carp (2.0 : 1).<sup><a reveal-id="DSU038C62" data-open="DSU038C62" class="link link-ref link-reveal xref-bibr">62</a></sup> These observations indicated that even closely related fish species differ in genetic sizes and the extent of sexual dimorphism for recombination fraction,<sup><a reveal-id="DSU038C50" data-open="DSU038C50" class="link link-ref link-reveal xref-bibr">50</a></sup> similarly as in mammals. Although the genetic basis for recombination bias between sexes remains unknown, several theories were proposed to explain this observation. One explanation from selection perspective suggests that selection pressure is stronger in male gametes than in female gametes during the haploid life stage. This difference could lead to male-specific selection to maintain beneficial haplotype to decrease the male recombination rate.<sup><a reveal-id="DSU038C58" data-open="DSU038C58" class="link link-ref link-reveal xref-bibr">58</a>,<a reveal-id="DSU038C63" data-open="DSU038C63" class="link link-ref link-reveal xref-bibr">63</a></sup> An alternative explanation is that the female recombination rate is higher to compensate for the apparently less stringent checkpoint for achiasmatic chromosomes compared with males.<sup><a reveal-id="DSU038C50" data-open="DSU038C50" class="link link-ref link-reveal xref-bibr">50</a></sup> In human, cytological studies suggested that sex-specific differences in recombination may derive from chromatin differences established prior to the onset of the recombination pathway.<sup><a reveal-id="DSU038C64" data-open="DSU038C64" class="link link-ref link-reveal xref-bibr">64</a></sup></p><p>The high-density and the relatively high-resolution genetic map provides valuable resource for integrating the physical map and whole-genome assemblies. In this study, we anchored the catfish physical map to genetic linkage map through BES and PMCSS that contain SNP markers. With this linkage map of channel catfish, we were able to anchor 87% of the catfish BAC physical map contigs, covering ∼92% of the catfish genome. This is a great improvement on map integration compared with our previous efforts that only anchored 44.8% of the catfish BAC physical map contigs accounting for 52.8% of the catfish genome. To our knowledge, this is also the highest percentage of physical maps integration with genetic maps that were obtained in any aquaculture species. The well-integrated map is useful for comprehensive comparative genomic analyses, fine-scale mapping of QTL, and positional cloning for candidate genes.<sup><a reveal-id="DSU038C10" data-open="DSU038C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Moreover, the integrated map can be used as an important resource for the validation of the catfish whole-genome assembly.</p>                            <h2 scrollto-destination=6578285 id="6578285" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at</a> www.dnaresearch.oxfordjournals.org.</p>                            <h2 scrollto-destination=6578287 id="6578287" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This project was supported by Agriculture and Food Research Initiative Competitive Grant (No. 2012-67015-19410) from the US Department of Agriculture (USDA) National Institute of Food and Agriculture (NIFA) and partially supported by USDA Aquaculture Research Program Competitive Grant (No. 2014-70007-22395) from USDA NIFA. Funding to pay the Open Access publication charges for this article was provided by a grant from USDA Aquaculture Research Program Competitive Grant no. 2014-70007-22395.</p>                            <h2 scrollto-destination=6578289 id="6578289" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Miyako Kodama (University of Washington) for discussion on linkage mapping. The authors are grateful for the technical assistance with lab work from Ludmilla Kaltenboeck and Dr Huseyin Kucuktas.</p>                            <h2 scrollto-destination=6578291 id="6578291" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSU038C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Myers</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Bottolo</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Freeman</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">McVean</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Donnelly</div> <div class="given-names">P.</div></div>. </div><div class="article-title">A fine-scale map of recombination rates and hotspots across the human genome</div>, <div class="source">Science</div> , <div class="year">2005</div>, vol. <div class="volume">310</div> (pg. <div class="fpage">321</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20fine-scale%20map%20of%20recombination%20rates%20and%20hotspots%20across%20the%20human%20genome&amp;author=S.%20Myers&amp;author=L.%20Bottolo&amp;author=C.%20Freeman&amp;author=G.%20McVean&amp;author=P.%20Donnelly&amp;publication_year=2005&amp;journal=Science&amp;volume=310&amp;pages=321-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1117196" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1117196" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1117196"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16224025" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kucuktas</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Construction of genetic linkage maps and comparative genome analysis of catfish using gene-associated markers</div>, <div class="source">Genetics</div> , <div class="year">2009</div>, vol. <div class="volume">181</div> (pg. <div class="fpage">1649</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20genetic%20linkage%20maps%20and%20comparative%20genome%20analysis%20of%20catfish%20using%20gene-associated%20markers&amp;author=H.%20Kucuktas&amp;author=S.%20Wang&amp;author=P.%20Li&amp;publication_year=2009&amp;journal=Genetics&amp;volume=181&amp;pages=1649-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.108.098855" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.108.098855" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.108.098855"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19171943" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Burke</div> <div class="given-names">D.T.</div></div>,  <div class="name"><div class="surname">Carle</div> <div class="given-names">G.F.</div></div>,  <div class="name"><div class="surname">Olson</div> <div class="given-names">M.V.</div></div>. </div><div class="article-title">Cloning of large segments of exogenous DNA into yeast by means of artificial chromosome vectors</div>, <div class="source">Science</div> , <div class="year">1987</div>, vol. <div class="volume">236</div> (pg. <div class="fpage">806</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cloning%20of%20large%20segments%20of%20exogenous%20DNA%20into%20yeast%20by%20means%20of%20artificial%20chromosome%20vectors&amp;author=D.T.%20Burke&amp;author=G.F.%20Carle&amp;author=M.V.%20Olson&amp;publication_year=1987&amp;journal=Science&amp;volume=236&amp;pages=806-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.3033825" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.3033825" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.3033825"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3033825" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McPherson</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Marra</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hillier</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">A physical map of the human genome</div>, <div class="source">Nature</div> , <div class="year">2001</div>, vol. <div class="volume">409</div> (pg. <div class="fpage">934</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20physical%20map%20of%20the%20human%20genome&amp;author=J.D.%20McPherson&amp;author=M.%20Marra&amp;author=L.%20Hillier&amp;publication_year=2001&amp;journal=Nature&amp;volume=409&amp;pages=934-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35057157" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35057157" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35057157"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11237014" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eiglmeier</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Honore</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Woods</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Caudron</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Cole</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Use of an ordered cosmid library to deduce the genomic organization of <em>Mycobacterium leprae</em></div>, <div class="source">Mol. Microbiol.</div> , <div class="year">1993</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">197</div>-<div class="lpage">206</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Use%20of%20an%20ordered%20cosmid%20library%20to%20deduce%20the%20genomic%20organization%20of%20Mycobacterium%20leprae&amp;author=K.%20Eiglmeier&amp;author=N.%20Honore&amp;author=S.%20Woods&amp;author=B.%20Caudron&amp;author=S.%20Cole&amp;publication_year=1993&amp;journal=Mol.%20Microbiol.&amp;volume=7&amp;pages=197-206" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.1993.7.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.1993.7.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.1993.7.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8446027" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xu</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Thorsen</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Kucuktas</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">Z.</div></div>. </div><div class="article-title">A BAC-based physical map of the channel catfish genome</div>, <div class="source">Genomics</div> , <div class="year">2007</div>, vol. <div class="volume">90</div> (pg. <div class="fpage">380</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20BAC-based%20physical%20map%20of%20the%20channel%20catfish%20genome&amp;author=P.%20Xu&amp;author=S.%20Wang&amp;author=L.%20Liu&amp;author=J.%20Thorsen&amp;author=H.%20Kucuktas&amp;author=Z.%20Liu&amp;publication_year=2007&amp;journal=Genomics&amp;volume=90&amp;pages=380-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ygeno.2007.05.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ygeno.2007.05.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ygeno.2007.05.008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17582737" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Quiniou</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Waldbieser</div> <div class="given-names">G.C.</div></div>,  <div class="name"><div class="surname">Duke</div> <div class="given-names">M.V.</div></div>. </div><div class="article-title">A first generation BAC-based physical map of the channel catfish genome</div>, <div class="source">BMC Genomics</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">40</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20first%20generation%20BAC-based%20physical%20map%20of%20the%20channel%20catfish%20genome&amp;author=S.M.%20Quiniou&amp;author=G.C.%20Waldbieser&amp;author=M.V.%20Duke&amp;publication_year=2007&amp;journal=BMC%20Genomics&amp;volume=8&amp;pages=40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-8-40" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-8-40" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-8-40"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17284319" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Jiang</div> <div class="given-names">Y.L.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">S.L.</div></div>, et al. </div><div class="article-title">Comparative analysis of catfish BAC end sequences with the zebrafish genome</div>, <div class="source">BMC Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">592</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20catfish%20BAC%20end%20sequences%20with%20the%20zebrafish%20genome&amp;author=H.%20Liu&amp;author=Y.L.%20Jiang&amp;author=S.L.%20Wang&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=592" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-592" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-592" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-592"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20003258" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ninwichian</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Peatman</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Second-generation genetic linkage map of catfish and its integration with the BAC-based physical map</div>, <div class="source">G3 (Bethesda)</div> , <div class="year">2012</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">1233</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Second-generation%20genetic%20linkage%20map%20of%20catfish%20and%20its%20integration%20with%20the%20BAC-based%20physical%20map&amp;author=P.%20Ninwichian&amp;author=E.%20Peatman&amp;author=H.%20Liu&amp;publication_year=2012&amp;journal=G3%20(Bethesda)&amp;volume=2&amp;pages=1233-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.112.003962" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.112.003962" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.112.003962"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23050234" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Palti</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Genet</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Gao</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">A second generation integrated map of the rainbow trout (<em>Oncorhynchus mykiss</em>) genome: analysis of conserved synteny with model fish genomes</div>, <div class="source">Mar. Biotechnol. (NY)</div> , <div class="year">2012</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">343</div>-<div class="lpage">57</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20second%20generation%20integrated%20map%20of%20the%20rainbow%20trout%20(Oncorhynchus%20mykiss)%20genome%3A%20analysis%20of%20conserved%20synteny%20with%20model%20fish%20genomes&amp;author=Y.%20Palti&amp;author=C.%20Genet&amp;author=G.%20Gao&amp;publication_year=2012&amp;journal=Mar.%20Biotechnol.%20(NY)&amp;volume=14&amp;pages=343-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10126-011-9418-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10126-011-9418-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10126-011-9418-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22101344" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">C.M.</div></div>,  <div class="name"><div class="surname">Bai</div> <div class="given-names">Z.Y.</div></div>,  <div class="name"><div class="surname">He</div> <div class="given-names">X.P.</div></div>, et al. </div><div class="article-title">A high-resolution linkage map for comparative genome analysis and QTL fine mapping in Asian seabass, <em>Lates calcarifer</em></div>, <div class="source">BMC Genomics</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">174</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high-resolution%20linkage%20map%20for%20comparative%20genome%20analysis%20and%20QTL%20fine%20mapping%20in%20Asian%20seabass%2C%20Lates%20calcarifer&amp;author=C.M.%20Wang&amp;author=Z.Y.%20Bai&amp;author=X.P.%20He&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=174" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-174" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-174" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-174"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21457569" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lien</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Gidskehaug</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Moen</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">A dense SNP-based linkage map for Atlantic salmon (<em>Salmo salar</em>) reveals extended chromosome homeologies and striking differences in sex-specific recombination patterns</div>, <div class="source">BMC Genomics</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">615</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20dense%20SNP-based%20linkage%20map%20for%20Atlantic%20salmon%20(Salmo%20salar)%20reveals%20extended%20chromosome%20homeologies%20and%20striking%20differences%20in%20sex-specific%20recombination%20patterns&amp;author=S.%20Lien&amp;author=L.%20Gidskehaug&amp;author=T.%20Moen&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=615" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-615" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-615" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-615"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22182215" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Song</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Construction of a high-density microsatellite genetic linkage map and mapping of sexual and growth-related traits in half-smooth tongue sole (<em>Cynoglossus semilaevis</em>)</div>, <div class="source">PLoS ONE</div> , <div class="year">2012</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e52097</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20high-density%20microsatellite%20genetic%20linkage%20map%20and%20mapping%20of%20sexual%20and%20growth-related%20traits%20in%20half-smooth%20tongue%20sole%20(Cynoglossus%20semilaevis)&amp;author=W.%20Song&amp;author=Y.%20Li&amp;author=Y.%20Zhao&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=e52097" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0052097" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0052097" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0052097"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23284884" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rexroad</div> <div class="given-names">C.E.</div>III</div>,  <div class="name"><div class="surname">Palti</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Gahr</div> <div class="given-names">S.A.</div></div>,  <div class="name"><div class="surname">Vallejo</div> <div class="given-names">R.L.</div></div>. </div><div class="article-title">A second generation genetic map for rainbow trout (<em>Oncorhynchus mykiss</em>)</div>, <div class="source">BMC Genetics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">74</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20second%20generation%20genetic%20map%20for%20rainbow%20trout%20(Oncorhynchus%20mykiss)&amp;author=C.E.%20Rexroad&amp;author=Y.%20Palti&amp;author=S.A.%20Gahr&amp;author=R.L.%20Vallejo&amp;publication_year=2008&amp;journal=BMC%20Genetics&amp;volume=9&amp;pages=74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2156-9-74" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2156-9-74" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2156-9-74"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19019240" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhao</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ji</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">A dense genetic linkage map for common carp and its integration with a BAC-based physical map</div>, <div class="source">PLoS ONE</div> , <div class="year">2013</div>, vol. <div class="volume">8</div> pg. <div class="fpage">e63928</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20dense%20genetic%20linkage%20map%20for%20common%20carp%20and%20its%20integration%20with%20a%20BAC-based%20physical%20map&amp;author=L.%20Zhao&amp;author=Y.%20Zhang&amp;author=P.%20Ji&amp;publication_year=2013&amp;journal=PLoS%20ONE&amp;volume=8&amp;pages=e63928" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0063928" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0063928" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0063928"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23704958" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Karsi</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Cao</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Dunham</div> <div class="given-names">R.</div></div>. </div><div class="article-title">An AFLP-based genetic linkage map of channel catfish (<em>Ictalurus punctatus</em>) constructed by using an interspecific hybrid resource family</div>, <div class="source">Genetics</div> , <div class="year">2003</div>, vol. <div class="volume">165</div> (pg. <div class="fpage">687</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20AFLP-based%20genetic%20linkage%20map%20of%20channel%20catfish%20(Ictalurus%20punctatus)%20constructed%20by%20using%20an%20interspecific%20hybrid%20resource%20family&amp;author=Z.%20Liu&amp;author=A.%20Karsi&amp;author=P.%20Li&amp;author=D.%20Cao&amp;author=R.%20Dunham&amp;publication_year=2003&amp;journal=Genetics&amp;volume=165&amp;pages=687-94" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14573480" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Waldbieser</div> <div class="given-names">G.C.</div></div>,  <div class="name"><div class="surname">Bosworth</div> <div class="given-names">B.G.</div></div>,  <div class="name"><div class="surname">Nonneman</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Wolters</div> <div class="given-names">W.R.</div></div>. </div><div class="article-title">A microsatellite-based genetic linkage map for channel catfish, <em>Ictalurus punctatus</em></div>, <div class="source">Genetics</div> , <div class="year">2001</div>, vol. <div class="volume">158</div> (pg. <div class="fpage">727</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20microsatellite-based%20genetic%20linkage%20map%20for%20channel%20catfish%2C%20Ictalurus%20punctatus&amp;author=G.C.%20Waldbieser&amp;author=B.G.%20Bosworth&amp;author=D.J.%20Nonneman&amp;author=W.R.%20Wolters&amp;publication_year=2001&amp;journal=Genetics&amp;volume=158&amp;pages=727-34" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11404336" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lorenz</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Brenna-Hansen</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Moen</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">BAC-based upgrading and physical integration of a genetic SNP map in Atlantic salmon</div>, <div class="source">Anim. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">41</div> (pg. <div class="fpage">48</div>-<div class="lpage">54</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BAC-based%20upgrading%20and%20physical%20integration%20of%20a%20genetic%20SNP%20map%20in%20Atlantic%20salmon&amp;author=S.%20Lorenz&amp;author=S.%20Brenna-Hansen&amp;author=T.%20Moen&amp;publication_year=2010&amp;journal=Anim.%20Genet.&amp;volume=41&amp;pages=48-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/age.2010.41.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fage.2010.41.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fage.2010.41.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19917045" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Baird</div> <div class="given-names">N.A.</div></div>,  <div class="name"><div class="surname">Etter</div> <div class="given-names">P.D.</div></div>,  <div class="name"><div class="surname">Atwood</div> <div class="given-names">T.S.</div></div>, et al. </div><div class="article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers</div>, <div class="source">PLoS ONE</div> , <div class="year">2008</div>, vol. <div class="volume">3</div> pg. <div class="fpage">e3376</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20SNP%20discovery%20and%20genetic%20mapping%20using%20sequenced%20RAD%20markers&amp;author=N.A.%20Baird&amp;author=P.D.%20Etter&amp;author=T.S.%20Atwood&amp;publication_year=2008&amp;journal=PLoS%20ONE&amp;volume=3&amp;pages=e3376" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0003376" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0003376" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0003376"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18852878" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Davey</div> <div class="given-names">J.W.</div></div>,  <div class="name"><div class="surname">Hohenlohe</div> <div class="given-names">P.A.</div></div>,  <div class="name"><div class="surname">Etter</div> <div class="given-names">P.D.</div></div>,  <div class="name"><div class="surname">Boone</div> <div class="given-names">J.Q.</div></div>,  <div class="name"><div class="surname">Catchen</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Blaxter</div> <div class="given-names">M.L.</div></div>. </div><div class="article-title">Genome-wide genetic marker discovery and genotyping using next-generation sequencing</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">499</div>-<div class="lpage">510</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20genetic%20marker%20discovery%20and%20genotyping%20using%20next-generation%20sequencing&amp;author=J.W.%20Davey&amp;author=P.A.%20Hohenlohe&amp;author=P.D.%20Etter&amp;author=J.Q.%20Boone&amp;author=J.M.%20Catchen&amp;author=M.L.%20Blaxter&amp;publication_year=2011&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=12&amp;pages=499-510" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21681211" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Miller</div> <div class="given-names">M.R.</div></div>,  <div class="name"><div class="surname">Dunham</div> <div class="given-names">J.P.</div></div>,  <div class="name"><div class="surname">Amores</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Cresko</div> <div class="given-names">W.A.</div></div>,  <div class="name"><div class="surname">Johnson</div> <div class="given-names">E.A.</div></div>. </div><div class="article-title">Rapid and cost-effective polymorphism identification and genotyping using restriction site associated DNA (RAD) markers</div>, <div class="source">Genome Res.</div> , <div class="year">2007</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">240</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20and%20cost-effective%20polymorphism%20identification%20and%20genotyping%20using%20restriction%20site%20associated%20DNA%20(RAD)%20markers&amp;author=M.R.%20Miller&amp;author=J.P.%20Dunham&amp;author=A.%20Amores&amp;author=W.A.%20Cresko&amp;author=E.A.%20Johnson&amp;publication_year=2007&amp;journal=Genome%20Res.&amp;volume=17&amp;pages=240-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.5681207" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.5681207" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.5681207"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17189378" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gutierrez</div> <div class="given-names">A.P.</div></div>,  <div class="name"><div class="surname">Lubieniecki</div> <div class="given-names">K.P.</div></div>,  <div class="name"><div class="surname">Davidson</div> <div class="given-names">E.A.</div></div>, et al. </div><div class="article-title">Genetic mapping of quantitative trait loci (QTL) for body-weight in Atlantic salmon (<em>Salmo salar</em>) using a 6.5K SNP array</div>, <div class="source">Aquaculture</div> , <div class="year">2012</div>, vol. <div class="volume">358</div> (pg. <div class="fpage">61</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20mapping%20of%20quantitative%20trait%20loci%20(QTL)%20for%20body-weight%20in%20Atlantic%20salmon%20(Salmo%20salar)%20using%20a%206.5K%20SNP%20array&amp;author=A.P.%20Gutierrez&amp;author=K.P.%20Lubieniecki&amp;author=E.A.%20Davidson&amp;publication_year=2012&amp;journal=Aquaculture&amp;volume=358&amp;pages=61-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.aquaculture.2012.06.017" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.aquaculture.2012.06.017" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.aquaculture.2012.06.017"> </span></div> </div></div></div></div></div><div content-id="DSU038C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McCue</div> <div class="given-names">M.E.</div></div>,  <div class="name"><div class="surname">Bannasch</div> <div class="given-names">D.L.</div></div>,  <div class="name"><div class="surname">Petersen</div> <div class="given-names">J.L.</div></div>, et al. </div><div class="article-title">A high density SNP array for the domestic horse and extant Perissodactyla: utility for association mapping, genetic diversity, and phylogeny studies</div>, <div class="source">PLoS Genet.</div> , <div class="year">2012</div>, vol. <div class="volume">8</div> pg. <div class="fpage">e1002451</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high%20density%20SNP%20array%20for%20the%20domestic%20horse%20and%20extant%20Perissodactyla%3A%20utility%20for%20association%20mapping%2C%20genetic%20diversity%2C%20and%20phylogeny%20studies&amp;author=M.E.%20McCue&amp;author=D.L.%20Bannasch&amp;author=J.L.%20Petersen&amp;publication_year=2012&amp;journal=PLoS%20Genet.&amp;volume=8&amp;pages=e1002451" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1002451" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1002451" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1002451"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22253606" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Houston</div> <div class="given-names">R.D.</div></div>,  <div class="name"><div class="surname">Taggart</div> <div class="given-names">J.B.</div></div>,  <div class="name"><div class="surname">Cezard</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Development and validation of a high density SNP genotyping array for Atlantic salmon (<em>Salmo salar</em>)</div>, <div class="source">BMC Genomics</div> , <div class="year">2014</div>, vol. <div class="volume">15</div> pg. <div class="fpage">90</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20validation%20of%20a%20high%20density%20SNP%20genotyping%20array%20for%20Atlantic%20salmon%20(Salmo%20salar)&amp;author=R.D.%20Houston&amp;author=J.B.%20Taggart&amp;author=T.%20Cezard&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-90" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-90" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-90"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24524230" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Anderson</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Mari</div> <div class="given-names">A.R.</div></div>,  <div class="name"><div class="surname">Braasch</div> <div class="given-names">I.</div></div>, et al. </div><div class="article-title">Multiple sex-associated regions and a putative sex chromosome in zebrafish revealed by RAD mapping and population genomics</div>, <div class="source">PLoS ONE</div> , <div class="year">2012</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e40701</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20sex-associated%20regions%20and%20a%20putative%20sex%20chromosome%20in%20zebrafish%20revealed%20by%20RAD%20mapping%20and%20population%20genomics&amp;author=J.L.%20Anderson&amp;author=A.R.%20Mari&amp;author=I.%20Braasch&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=e40701" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0040701" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0040701" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0040701"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22792396" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gonen</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Lowe</div> <div class="given-names">N.R.</div></div>,  <div class="name"><div class="surname">Cezard</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Gharbi</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Bishop</div> <div class="given-names">S.C.</div></div>,  <div class="name"><div class="surname">Houston</div> <div class="given-names">R.D.</div></div>. </div><div class="article-title">Linkage maps of the Atlantic salmon (<em>Salmo salar</em>) genome derived from RAD sequencing</div>, <div class="source">BMC Genomics</div> , <div class="year">2014</div>, vol. <div class="volume">15</div> pg. <div class="fpage">166</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Linkage%20maps%20of%20the%20Atlantic%20salmon%20(Salmo%20salar)%20genome%20derived%20from%20RAD%20sequencing&amp;author=S.%20Gonen&amp;author=N.R.%20Lowe&amp;author=T.%20Cezard&amp;author=K.%20Gharbi&amp;author=S.C.%20Bishop&amp;author=R.D.%20Houston&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=166" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-166" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-166" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-166"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24571138" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Everett</div> <div class="given-names">M.V.</div></div>,  <div class="name"><div class="surname">Miller</div> <div class="given-names">M.R.</div></div>,  <div class="name"><div class="surname">Seeb</div> <div class="given-names">J.E.</div></div>. </div><div class="article-title">Meiotic maps of sockeye salmon derived from massively parallel DNA sequencing</div>, <div class="source">BMC Genomics</div> , <div class="year">2012</div>, vol. <div class="volume">13</div> pg. <div class="fpage">521</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Meiotic%20maps%20of%20sockeye%20salmon%20derived%20from%20massively%20parallel%20DNA%20sequencing&amp;author=M.V.%20Everett&amp;author=M.R.%20Miller&amp;author=J.E.%20Seeb&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=521" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-521" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-521" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-521"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23031582" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Palaiokostas</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Bekaert</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Khan</div> <div class="given-names">M.G.Q.</div></div>, et al. </div><div class="article-title">Mapping and validation of the major sex-determining region in Nile Tilapia (<em>Oreochromis niloticus</em> L.) using RAD sequencing</div>, <div class="source">PLoS ONE</div> , <div class="year">2013</div>, vol. <div class="volume">8</div> pg. <div class="fpage">e68389</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20and%20validation%20of%20the%20major%20sex-determining%20region%20in%20Nile%20Tilapia%20(Oreochromis%20niloticus%20L.)%20using%20RAD%20sequencing&amp;author=C.%20Palaiokostas&amp;author=M.%20Bekaert&amp;author=M.G.Q.%20Khan&amp;publication_year=2013&amp;journal=PLoS%20ONE&amp;volume=8&amp;pages=e68389" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0068389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0068389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0068389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23874606" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jiang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Gao</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Whole genome comparative analysis of channel catfish (<em>Ictalurus punctatus</em>) with four model fish species</div>, <div class="source">BMC Genomics</div> , <div class="year">2013</div>, vol. <div class="volume">14</div> pg. <div class="fpage">780</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole%20genome%20comparative%20analysis%20of%20channel%20catfish%20(Ictalurus%20punctatus)%20with%20four%20model%20fish%20species&amp;author=Y.%20Jiang&amp;author=X.%20Gao&amp;author=S.%20Liu&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=780" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-780" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-780" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-780"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24215161" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Lu</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Comparative genomic analysis of catfish linkage group 8 reveals two homologous chromosomes in zebrafish and other teleosts with extensive inter-chromosomal rearrangements</div>, <div class="source">BMC Genomics</div> , <div class="year">2013</div>, vol. <div class="volume">14</div> pg. <div class="fpage">387</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genomic%20analysis%20of%20catfish%20linkage%20group%208%20reveals%20two%20homologous%20chromosomes%20in%20zebrafish%20and%20other%20teleosts%20with%20extensive%20inter-chromosomal%20rearrangements&amp;author=Y.%20Zhang&amp;author=S.%20Liu&amp;author=J.%20Lu&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=387" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-387" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-387" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-387"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23758806" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Zhou</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Lu</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Generation of genome-scale gene-associated SNPs in catfish for the construction of a high-density SNP array</div>, <div class="source">BMC Genomics</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">53</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Generation%20of%20genome-scale%20gene-associated%20SNPs%20in%20catfish%20for%20the%20construction%20of%20a%20high-density%20SNP%20array&amp;author=S.%20Liu&amp;author=Z.%20Zhou&amp;author=J.%20Lu&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-53" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-53" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-53"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21255432" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sun</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Identification and analysis of genome-wide SNPs provide insight into signatures of selection and domestication in catfish</div>, <div class="source">PLoS ONE</div> , <div class="year">2014</div>, vol. <div class="volume">9</div> pg. <div class="fpage">e109666</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20analysis%20of%20genome-wide%20SNPs%20provide%20insight%20into%20signatures%20of%20selection%20and%20domestication%20in%20catfish&amp;author=L.%20Sun&amp;author=S.%20Liu&amp;author=R.%20Wang&amp;publication_year=2014&amp;journal=PLoS%20ONE&amp;volume=9&amp;pages=e109666" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0109666" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0109666" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0109666"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25313648" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sun</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Development of the catfish 250K SNP array for genome-wide association studies</div>, <div class="source">BMC Res. Notes</div> , <div class="year">2014</div>, vol. <div class="volume">7</div> pg. <div class="fpage">135</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20the%20catfish%20250K%20SNP%20array%20for%20genome-wide%20association%20studies&amp;author=S.%20Liu&amp;author=L.%20Sun&amp;author=Y.%20Li&amp;publication_year=2014&amp;journal=BMC%20Res.%20Notes&amp;volume=7&amp;pages=135" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1756-0500-7-135" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1756-0500-7-135" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1756-0500-7-135"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24618043" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Van Ooijen</div> <div class="given-names">J</div></div>. </div>, <div class="source">JoinMap 4. Software for the calculation of genetic linkage maps in experimental populations</div> , <div class="year">2006</div><div class="publisher-loc">Wageningen, The Netherlands</div><div class="publisher-name">Kyazma BV</div></div></div></div></div><div content-id="DSU038C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stam</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Construction of integrated genetic linkage maps by means of a new computer package: Join Map</div>, <div class="source">Plant J.</div> , <div class="year">1993</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">739</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20integrated%20genetic%20linkage%20maps%20by%20means%20of%20a%20new%20computer%20package%3A%20Join%20Map&amp;author=P.%20Stam&amp;publication_year=1993&amp;journal=Plant%20J.&amp;volume=3&amp;pages=739-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.1993.3.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.1993.3.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.1993.3.issue-5"> </span></div> </div></div></div></div></div><div content-id="DSU038C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wu</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Close</div> <div class="given-names">T.J.</div></div>,  <div class="name"><div class="surname">Lonardi</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Accurate construction of consensus genetic maps via integer linear programming</div>, <div class="source">IEEE/ACM Trans. Comput. Biol. Bioinform.</div> , <div class="year">2011</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">381</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accurate%20construction%20of%20consensus%20genetic%20maps%20via%20integer%20linear%20programming&amp;author=Y.%20Wu&amp;author=T.J.%20Close&amp;author=S.%20Lonardi&amp;publication_year=2011&amp;journal=IEEE%2FACM%20Trans.%20Comput.%20Biol.%20Bioinform.&amp;volume=8&amp;pages=381-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1109/TCBB.2010.35" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1109%2FTCBB.2010.35" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1109%2FTCBB.2010.35"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20479505" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Voorrips</div> <div class="given-names">R.</div></div>. </div><div class="article-title">MapChart: software for the graphical presentation of linkage maps and QTLs</div>, <div class="source">J. Hered.</div> , <div class="year">2002</div>, vol. <div class="volume">93</div> (pg. <div class="fpage">77</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MapChart%3A%20software%20for%20the%20graphical%20presentation%20of%20linkage%20maps%20and%20QTLs&amp;author=R.%20Voorrips&amp;publication_year=2002&amp;journal=J.%20Hered.&amp;volume=93&amp;pages=77-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jhered/93.1.77" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjhered%2F93.1.77" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjhered%2F93.1.77"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12011185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ott</div> <div class="given-names">J</div></div>. </div>, <div class="source">Analysis of human genetic linkage</div> , <div class="year">1999</div><div class="publisher-name">Johns Hopkins Univ. Press, Baltimore, USA</div></div></div></div></div><div content-id="DSU038C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Garcia-Dorado</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Gallego</div> <div class="given-names">A.</div></div>. </div><div class="article-title">On the use of the classical tests for detecting linkage</div>, <div class="source">J. Hered.</div> , <div class="year">1992</div>, vol. <div class="volume">83</div> (pg. <div class="fpage">143</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=On%20the%20use%20of%20the%20classical%20tests%20for%20detecting%20linkage&amp;author=A.%20Garcia-Dorado&amp;author=A.%20Gallego&amp;publication_year=1992&amp;journal=J.%20Hered.&amp;volume=83&amp;pages=143-6" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1578105" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xu</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">Channel catfish BAC-end sequences for marker development and assessment of syntenic conservation with other fish species</div>, <div class="source">Anim. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">321</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Channel%20catfish%20BAC-end%20sequences%20for%20marker%20development%20and%20assessment%20of%20syntenic%20conservation%20with%20other%20fish%20species&amp;author=P.%20Xu&amp;author=S.%20Wang&amp;author=L.%20Liu&amp;publication_year=2006&amp;journal=Anim.%20Genet.&amp;volume=37&amp;pages=321-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/age.2006.37.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fage.2006.37.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fage.2006.37.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16879340" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jiang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ninwichian</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Generation of physical map contig-specific sequences useful for whole genome sequence scaffolding</div>, <div class="source">PLoS ONE</div> , <div class="year">2013</div>, vol. <div class="volume">8</div> pg. <div class="fpage">e78872</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Generation%20of%20physical%20map%20contig-specific%20sequences%20useful%20for%20whole%20genome%20sequence%20scaffolding&amp;author=Y.%20Jiang&amp;author=P.%20Ninwichian&amp;author=S.%20Liu&amp;publication_year=2013&amp;journal=PLoS%20ONE&amp;volume=8&amp;pages=e78872" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0078872" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0078872" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0078872"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24205335" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ronin</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Mester</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Minkov</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">Two-phase analysis in consensus genetic mapping</div>, <div class="source">G3-Genes Genom. Genet.</div> , <div class="year">2012</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">537</div>-<div class="lpage">49</div>)</div></div></div></div><div content-id="DSU038C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ronin</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Mester</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Minkov</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Korol</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Building reliable genetic maps: different mapping strategies may result in different maps</div>, <div class="source">Nat. Sci.</div> , <div class="year">2010</div>, vol. <div class="volume">2</div> pg. <div class="fpage">576</div> </div></div></div></div><div content-id="DSU038C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Howe</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Clark</div> <div class="given-names">M.D.</div></div>,  <div class="name"><div class="surname">Torroja</div> <div class="given-names">C.F.</div></div>, et al. </div><div class="article-title">The zebrafish reference genome sequence and its relationship to the human genome</div>, <div class="source">Nature</div> , <div class="year">2013</div>, vol. <div class="volume">496</div> (pg. <div class="fpage">498</div>-<div class="lpage">503</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20zebrafish%20reference%20genome%20sequence%20and%20its%20relationship%20to%20the%20human%20genome&amp;author=K.%20Howe&amp;author=M.D.%20Clark&amp;author=C.F.%20Torroja&amp;publication_year=2013&amp;journal=Nature&amp;volume=496&amp;pages=498-503" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature12111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature12111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature12111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23594743" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kocher</div> <div class="given-names">T.D.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">W.J.</div></div>,  <div class="name"><div class="surname">Sobolewska</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Penman</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">McAndrew</div> <div class="given-names">B.</div></div>. </div><div class="article-title">A genetic linkage map of a cichlid fish, the tilapia (<em>Oreochromis niloticus</em>)</div>, <div class="source">Genetics</div> , <div class="year">1998</div>, vol. <div class="volume">148</div> (pg. <div class="fpage">1225</div>-<div class="lpage">32</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genetic%20linkage%20map%20of%20a%20cichlid%20fish%2C%20the%20tilapia%20(Oreochromis%20niloticus)&amp;author=T.D.%20Kocher&amp;author=W.J.%20Lee&amp;author=H.%20Sobolewska&amp;author=D.%20Penman&amp;author=B.%20McAndrew&amp;publication_year=1998&amp;journal=Genetics&amp;volume=148&amp;pages=1225-32" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9539437" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Naruse</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Fukamachi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Mitani</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">A detailed linkage map of medaka, <em>Oryzias latipes</em>: comparative genomics and genome evolution</div>, <div class="source">Genetics</div> , <div class="year">2000</div>, vol. <div class="volume">154</div> (pg. <div class="fpage">1773</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20detailed%20linkage%20map%20of%20medaka%2C%20Oryzias%20latipes%3A%20comparative%20genomics%20and%20genome%20evolution&amp;author=K.%20Naruse&amp;author=S.%20Fukamachi&amp;author=H.%20Mitani&amp;publication_year=2000&amp;journal=Genetics&amp;volume=154&amp;pages=1773-84" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10747068" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sakamoto</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Danzmann</div> <div class="given-names">R.G.</div></div>,  <div class="name"><div class="surname">Gharbi</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">A microsatellite linkage map of rainbow trout (<em>Oncorhynchus mykiss</em>) characterized by large sex-specific differences in recombination rates</div>, <div class="source">Genetics</div> , <div class="year">2000</div>, vol. <div class="volume">155</div> (pg. <div class="fpage">1331</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20microsatellite%20linkage%20map%20of%20rainbow%20trout%20(Oncorhynchus%20mykiss)%20characterized%20by%20large%20sex-specific%20differences%20in%20recombination%20rates&amp;author=T.%20Sakamoto&amp;author=R.G.%20Danzmann&amp;author=K.%20Gharbi&amp;publication_year=2000&amp;journal=Genetics&amp;volume=155&amp;pages=1331-45" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10880492" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Choo</div> <div class="given-names">K.H.A.</div></div>. </div><div class="article-title">Why is the centromere so cold?</div>, <div class="source">Genome Res.</div> , <div class="year">1998</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">81</div>-<div class="lpage">2</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Why%20is%20the%20centromere%20so%20cold%3F&amp;author=K.H.A.%20Choo&amp;publication_year=1998&amp;journal=Genome%20Res.&amp;volume=8&amp;pages=81-2" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9477334" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nachman</div> <div class="given-names">M.W.</div></div>,  <div class="name"><div class="surname">Churchill</div> <div class="given-names">G.A.</div></div>. </div><div class="article-title">Heterogeneity in rates of recombination across the mouse genome</div>, <div class="source">Genetics</div> , <div class="year">1996</div>, vol. <div class="volume">142</div> (pg. <div class="fpage">537</div>-<div class="lpage">48</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Heterogeneity%20in%20rates%20of%20recombination%20across%20the%20mouse%20genome&amp;author=M.W.%20Nachman&amp;author=G.A.%20Churchill&amp;publication_year=1996&amp;journal=Genetics&amp;volume=142&amp;pages=537-48" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8852851" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Coop</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Przeworski</div> <div class="given-names">M.</div></div>. </div><div class="article-title">An evolutionary view of human recombination</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">23</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20evolutionary%20view%20of%20human%20recombination&amp;author=G.%20Coop&amp;author=M.%20Przeworski&amp;publication_year=2007&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=8&amp;pages=23-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1947" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1947" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1947"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17146469" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tortereau</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Servin</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Frantz</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">A high density recombination map of the pig reveals a correlation between sex-specific recombination and GC content</div>, <div class="source">BMC Genomics</div> , <div class="year">2012</div>, vol. <div class="volume">13</div> pg. <div class="fpage">586</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high%20density%20recombination%20map%20of%20the%20pig%20reveals%20a%20correlation%20between%20sex-specific%20recombination%20and%20GC%20content&amp;author=F.%20Tortereau&amp;author=B.%20Servin&amp;author=L.%20Frantz&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=586" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-586" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-586" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-586"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23152986" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Groenen</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Wahlberg</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Foglio</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">A high-density SNP-based linkage map of the chicken genome reveals sequence features correlated with recombination rate</div>, <div class="source">Genome Res.</div> , <div class="year">2009</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">510</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high-density%20SNP-based%20linkage%20map%20of%20the%20chicken%20genome%20reveals%20sequence%20features%20correlated%20with%20recombination%20rate&amp;author=M.A.%20Groenen&amp;author=P.%20Wahlberg&amp;author=M.%20Foglio&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=19&amp;pages=510-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.086538.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.086538.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.086538.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19088305" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jensen-Seaman</div> <div class="given-names">M.I.</div></div>,  <div class="name"><div class="surname">Furey</div> <div class="given-names">T.S.</div></div>,  <div class="name"><div class="surname">Payseur</div> <div class="given-names">B.A.</div></div>, et al. </div><div class="article-title">Comparative recombination rates in the rat, mouse, and human genomes</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">528</div>-<div class="lpage">38</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20recombination%20rates%20in%20the%20rat%2C%20mouse%2C%20and%20human%20genomes&amp;author=M.I.%20Jensen-Seaman&amp;author=T.S.%20Furey&amp;author=B.A.%20Payseur&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=528-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1970304" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1970304" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1970304"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15059993" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Petes</div> <div class="given-names">T.D.</div></div>. </div><div class="article-title">Meiotic recombination hot spots and cold spots</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2001</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">360</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Meiotic%20recombination%20hot%20spots%20and%20cold%20spots&amp;author=T.D.%20Petes&amp;publication_year=2001&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=2&amp;pages=360-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35072078" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35072078" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35072078"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11331902" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Baudat</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Buard</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Grey</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">PRDM9 is a major determinant of meiotic recombination hotspots in humans and mice</div>, <div class="source">Science</div> , <div class="year">2010</div>, vol. <div class="volume">327</div> (pg. <div class="fpage">836</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PRDM9%20is%20a%20major%20determinant%20of%20meiotic%20recombination%20hotspots%20in%20humans%20and%20mice&amp;author=F.%20Baudat&amp;author=J.%20Buard&amp;author=C.%20Grey&amp;publication_year=2010&amp;journal=Science&amp;volume=327&amp;pages=836-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1183439" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1183439" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1183439"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20044539" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Myers</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Bowden</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Tumian</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Drive against hotspot motifs in primates implicates the PRDM9 gene in meiotic recombination</div>, <div class="source">Science</div> , <div class="year">2010</div>, vol. <div class="volume">327</div> (pg. <div class="fpage">876</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Drive%20against%20hotspot%20motifs%20in%20primates%20implicates%20the%20PRDM9%20gene%20in%20meiotic%20recombination&amp;author=S.%20Myers&amp;author=R.%20Bowden&amp;author=A.%20Tumian&amp;publication_year=2010&amp;journal=Science&amp;volume=327&amp;pages=876-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1182363" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1182363" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1182363"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20044541" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C57" class="ref-content "><div class="label -label">57</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parvanov</div> <div class="given-names">E.D.</div></div>,  <div class="name"><div class="surname">Petkov</div> <div class="given-names">P.M.</div></div>,  <div class="name"><div class="surname">Paigen</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Prdm9 controls activation of mammalian recombination hotspots</div>, <div class="source">Science</div> , <div class="year">2010</div>, vol. <div class="volume">327</div> pg. <div class="fpage">835</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prdm9%20controls%20activation%20of%20mammalian%20recombination%20hotspots&amp;author=E.D.%20Parvanov&amp;author=P.M.%20Petkov&amp;author=K.%20Paigen&amp;publication_year=2010&amp;journal=Science&amp;volume=327&amp;pages=835" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1181495" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1181495" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1181495"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20044538" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C58" class="ref-content "><div class="label -label">58</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Singer</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Perlman</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Yan</div> <div class="given-names">Y.L.</div></div>, et al. </div><div class="article-title">Sex-specific recombination rates in zebrafish (<em>Danio rerio</em>)</div>, <div class="source">Genetics</div> , <div class="year">2002</div>, vol. <div class="volume">160</div> (pg. <div class="fpage">649</div>-<div class="lpage">57</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sex-specific%20recombination%20rates%20in%20zebrafish%20(Danio%20rerio)&amp;author=A.%20Singer&amp;author=H.%20Perlman&amp;author=Y.L.%20Yan&amp;publication_year=2002&amp;journal=Genetics&amp;volume=160&amp;pages=649-57" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11861568" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C59" class="ref-content "><div class="label -label">59</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chistiakov</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Hellemans</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Haley</div> <div class="given-names">C.S.</div></div>, et al. </div><div class="article-title">A microsatellite linkage map of the European sea bass <em>Dicentrarchus labrax</em> L</div>, <div class="source">Genetics</div> , <div class="year">2005</div>, vol. <div class="volume">170</div> (pg. <div class="fpage">1821</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20microsatellite%20linkage%20map%20of%20the%20European%20sea%20bass%20Dicentrarchus%20labrax%20L&amp;author=D.A.%20Chistiakov&amp;author=B.%20Hellemans&amp;author=C.S.%20Haley&amp;publication_year=2005&amp;journal=Genetics&amp;volume=170&amp;pages=1821-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.104.039719" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.104.039719" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.104.039719"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15937133" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C60" class="ref-content "><div class="label -label">60</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guo</div> <div class="given-names">W.J.</div></div>,  <div class="name"><div class="surname">Tong</div> <div class="given-names">J.G.</div></div>,  <div class="name"><div class="surname">Yu</div> <div class="given-names">X.M.</div></div>, et al. </div><div class="article-title">A second generation genetic linkage map for silver carp (<em>Hypophthalmichehys molitrix</em>) using microsatellite markers</div>, <div class="source">Aquaculture</div> , <div class="year">2013</div>, vol. <div class="volume">412</div> (pg. <div class="fpage">97</div>-<div class="lpage">106</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20second%20generation%20genetic%20linkage%20map%20for%20silver%20carp%20(Hypophthalmichehys%20molitrix)%20using%20microsatellite%20markers&amp;author=W.J.%20Guo&amp;author=J.G.%20Tong&amp;author=X.M.%20Yu&amp;publication_year=2013&amp;journal=Aquaculture&amp;volume=412&amp;pages=97-106" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.aquaculture.2013.06.027" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.aquaculture.2013.06.027" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.aquaculture.2013.06.027"> </span></div> </div></div></div></div></div><div content-id="DSU038C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C61" class="ref-content "><div class="label -label">61</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Woram</div> <div class="given-names">R.A.</div></div>,  <div class="name"><div class="surname">McGowan</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Stout</div> <div class="given-names">J.A.</div></div>, et al. </div><div class="article-title">A genetic linkage map for Arctic char (<em>Salvelinus alpinus</em>): evidence for higher recombination rates and segregation distortion in hybrid versus pure strain mapping parents</div>, <div class="source">Genome</div> , <div class="year">2004</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">304</div>-<div class="lpage">15</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genetic%20linkage%20map%20for%20Arctic%20char%20(Salvelinus%20alpinus)%3A%20evidence%20for%20higher%20recombination%20rates%20and%20segregation%20distortion%20in%20hybrid%20versus%20pure%20strain%20mapping%20parents&amp;author=R.A.%20Woram&amp;author=C.%20McGowan&amp;author=J.A.%20Stout&amp;publication_year=2004&amp;journal=Genome&amp;volume=47&amp;pages=304-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g03-127" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg03-127" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg03-127"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15060583" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C62" class="ref-content "><div class="label -label">62</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xia</div> <div class="given-names">J.H.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">Z.Y.</div></div>, et al. </div><div class="article-title">A consensus linkage map of the grass carp (<em>Ctenopharyngodon idella</em>) based on microsatellites and SNPs</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">135</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20consensus%20linkage%20map%20of%20the%20grass%20carp%20(Ctenopharyngodon%20idella)%20based%20on%20microsatellites%20and%20SNPs&amp;author=J.H.%20Xia&amp;author=F.%20Liu&amp;author=Z.Y.%20Zhu&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=135" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-135" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-135" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-135"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20181260" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU038C63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C63" class="ref-content "><div class="label -label">63</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lenormand</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Dutheil</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Recombination difference between sexes: a role for haploid selection</div>, <div class="source">PLoS Biol.</div> , <div class="year">2005</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">396</div>-<div class="lpage">403</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recombination%20difference%20between%20sexes%3A%20a%20role%20for%20haploid%20selection&amp;author=T.%20Lenormand&amp;author=J.%20Dutheil&amp;publication_year=2005&amp;journal=PLoS%20Biol.&amp;volume=3&amp;pages=396-403" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0030063" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0030063" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0030063"> </span></div> </div></div></div></div></div><div content-id="DSU038C64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU038C64" class="ref-content "><div class="label -label">64</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gruhn</div> <div class="given-names">J.R.</div></div>,  <div class="name"><div class="surname">Rubio</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Broman</div> <div class="given-names">K.W.</div></div>,  <div class="name"><div class="surname">Hunt</div> <div class="given-names">P.A.</div></div>,  <div class="name"><div class="surname">Hassold</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Cytological studies of human meiosis: sex-specific differences in recombination originate at, or prior to, establishment of double-strand breaks</div>, <div class="source">PLoS ONE</div> , <div class="year">2013</div>, vol. <div class="volume">8</div> pg. <div class="fpage">e85075</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cytological%20studies%20of%20human%20meiosis%3A%20sex-specific%20differences%20in%20recombination%20originate%20at%2C%20or%20prior%20to%2C%20establishment%20of%20double-strand%20breaks&amp;author=J.R.%20Gruhn&amp;author=C.%20Rubio&amp;author=K.W.%20Broman&amp;author=P.A.%20Hunt&amp;author=T.%20Hassold&amp;publication_year=2013&amp;journal=PLoS%20ONE&amp;volume=8&amp;pages=e85075" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0085075" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0085075" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0085075"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24376867" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2014. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=440428&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu038/2/dsu038-dsu038supp_data3.xlsx?Expires=2147483647&amp;Signature=x-f8y~4cATBjzElfWw5PX5mqolUMn0C5Ehv3xtlNhauSt63DLyLx1iERFSuBXVD9aWfh1OyczqG2~wUOpLDr8Cp3j2gGWaXvhxvTdk2fQd3BCFHeOjpvUDwYLi8j37in8g79YHubaMmdlNO27u~gfnx9Q7glYAaOoprzr3Fxdy7r4SO83atitDZdfLj0a~kaIQl94YiVD29ZMtgYws3M8sKTIqa5mHehT-T3L7FQVFd9po-OZbTK5IprotVUsG341D1GpPbUjFl8a1SUp~rGet8ph1~bdHmiCiEJIacE96U7WOKPDALLOWpQALEOTuy6u1FCZ6MOMHlKnCgz6r7f4Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu038/2/dsu038-dsu038supp_data2.xlsx?Expires=2147483647&amp;Signature=Qx-DFyOwee9iSkUtkX1txNrOIX4sQYZ1gSt4nfXOaJ9z83py33qmBhD4PKCTf-zrY5K~28LsjsdP9nT-~OocowscLRxC3pgsplr4Jg0dvY-P-AkJ9vrrFQnQeJFCnfHJaOBj3APLDR2Hpf2Zwl-FXMMTEyyE5fg1m0wTVdWLD7t264ThEKG3nTuJAaEGcr2MBH-dtCh1GIwtVVC4w8sXN47WJ-c0~62qI-CB82~IfKxTp3ZUfPXy4iAMEOTvKcTWyws7yaMGdscHMpdp7Bs5DTadty7Vwthy4omCuWgr~AJwn2~~CF6MIlnuBAXLpG3ftS2aJufmy94hzAA3f-rDRA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu038/2/dsu038-dsu038supp_data1.xlsx?Expires=2147483647&amp;Signature=rlsJOPAgoYVlRke0hPNRS~0q15iRaXCV9NywAJnlB4XsMP8QJBL0YWVwTbveEtbAL7gnRflt1YlLytlu0iKmNbA~6UZYm-zy172U5nBozgZgAfSJYomClAl8JdOx4F~dYbnBmJoG166T2UExtYd7t9tc9tdZWq~bnsBQ3kdx6-H-TQ05K29J~QEO9lLAMN8ouxQiOV3qpj-NRO-~XvFXmfk-3h8EAw0CL0Qri83-KFbHPBnf2G7Et-JOvNEpoSqnLCb-iIkjw~NrC6-zWEablrwNeocEvLkPU9y61pS6InehG-MFg96LAkD3Cg0b6f2EF81lIWEkF43Gl~1QKIB5~w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - xlsx file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">578</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000350442100005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">23</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsu038"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="440428"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000350442100005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f1%2f39%2f440428">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000350442100005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (23)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F1%2F39%2F440428" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/440428'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Construction of a high-density, high-resolution genetic map and its integration with BAC-based physical map in channel catfish" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsu038");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f39%2f440428">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

