

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/3/183/591017","@type":"ScholarlyArticle","name":"High-density genetic map construction and identification of a locus controlling weeping trait in an ornamental woody plant ( Prunus mume  Sieb. et Zucc)","datePublished":"2015-03-15","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2015-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsv003","keywords":["mei","high density","linkage mapping","SLAF-seq","weeping trait"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/3/183/591017","author":[{"name":"Zhang, Jie","affiliation":"1  Beijing Key Laboratory of Ornamental Plants Germplasm Innovation and Molecular Breeding, National Engineering Research Center for Floriculture ,  Beijing Laboratory of Urban and Rural Ecological Environment, College of Landscape Architecture, Beijing Forestry University ,  Beijing 100083 ,  China","@type":"Person"},{"name":"Zhang, Qixiang","affiliation":"1  Beijing Key Laboratory of Ornamental Plants Germplasm Innovation and Molecular Breeding, National Engineering Research Center for Floriculture ,  Beijing Laboratory of Urban and Rural Ecological Environment, College of Landscape Architecture, Beijing Forestry University ,  Beijing 100083 ,  China","@type":"Person"},{"name":"Cheng, Tangren","affiliation":"1  Beijing Key Laboratory of Ornamental Plants Germplasm Innovation and Molecular Breeding, National Engineering Research Center for Floriculture ,  Beijing Laboratory of Urban and Rural Ecological Environment, College of Landscape Architecture, Beijing Forestry University ,  Beijing 100083 ,  China","@type":"Person"},{"name":"Yang, Weiru","affiliation":"1  Beijing Key Laboratory of Ornamental Plants Germplasm Innovation and Molecular Breeding, National Engineering Research Center for Floriculture ,  Beijing Laboratory of Urban and Rural Ecological Environment, College of Landscape Architecture, Beijing Forestry University ,  Beijing 100083 ,  China","@type":"Person"},{"name":"Pan, Huitang","affiliation":"1  Beijing Key Laboratory of Ornamental Plants Germplasm Innovation and Molecular Breeding, National Engineering Research Center for Floriculture ,  Beijing Laboratory of Urban and Rural Ecological Environment, College of Landscape Architecture, Beijing Forestry University ,  Beijing 100083 ,  China","@type":"Person"},{"name":"Zhong, Junjun","affiliation":"1  Beijing Key Laboratory of Ornamental Plants Germplasm Innovation and Molecular Breeding, National Engineering Research Center for Floriculture ,  Beijing Laboratory of Urban and Rural Ecological Environment, College of Landscape Architecture, Beijing Forestry University ,  Beijing 100083 ,  China","@type":"Person"},{"name":"Huang, Long","affiliation":"2  Biomarker Technologies Corporation ,  Beijing ,  China","@type":"Person"},{"name":"Liu, Enze","affiliation":"2  Biomarker Technologies Corporation ,  Beijing ,  China","@type":"Person"}],"description":"Abstract.  High-density genetic map is a valuable tool for fine mapping locus controlling a specific trait especially for perennial woody plants. In this study,","pageStart":"183","pageEnd":"191","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/3/1/m_cover.gif?Expires=2147483647&Signature=o~ckdHMyoALozjazxFdJARrSll6HI0qZKNez1rp60DiaVqyzX7KX3gffSJBpNovfc7x0FBbw5M~q-oXkBI0BrEKD0pgvQT5eVNC4LZt8HYW1wCNn7TYsIJ~iamj5~PWBqwfQdRx7Lwp~5y0cfA6QZfzXvHT-c2y8KfWBVWCVg9vuL7VY9lDugsDIpyrHTWTMWH4KR~-txVxPM4XBFIVE2oVLkX6QYMzija1YcuQl4yisR1R~qhzzHGrjVvva6IGLptvPDIFfjhAoXGwaWNy8qGaYVTkt5JCmjQmDV1APqpGYq6f9-3DKz0Py3DzfqJ6xp8-SqVKAnV4V9jMcimV1eA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"High-density genetic map construction and identification of a locus controlling weeping trait in an ornamental woody plant ( Prunus mume  Sieb. et Zucc)","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/3/1/m_cover.gif?Expires=2147483647&Signature=o~ckdHMyoALozjazxFdJARrSll6HI0qZKNez1rp60DiaVqyzX7KX3gffSJBpNovfc7x0FBbw5M~q-oXkBI0BrEKD0pgvQT5eVNC4LZt8HYW1wCNn7TYsIJ~iamj5~PWBqwfQdRx7Lwp~5y0cfA6QZfzXvHT-c2y8KfWBVWCVg9vuL7VY9lDugsDIpyrHTWTMWH4KR~-txVxPM4XBFIVE2oVLkX6QYMzija1YcuQl4yisR1R~qhzzHGrjVvva6IGLptvPDIFfjhAoXGwaWNy8qGaYVTkt5JCmjQmDV1APqpGYq6f9-3DKz0Py3DzfqJ6xp8-SqVKAnV4V9jMcimV1eA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="High-density genetic map construction and identification of a locus controlling weeping trait in an ornamental woody plant ( Prunus mume  Sieb. et Zucc)" />
    <meta property="og:description" content="Abstract.  High-density genetic map is a valuable tool for fine mapping locus controlling a specific trait especially for perennial woody plants. In this study," />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/3/183/591017" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2015-03-15" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Zhang, Jie" /><meta name="citation_author_institution" content="Beijing Key Laboratory of Ornamental Plants Germplasm Innovation and Molecular Breeding, National Engineering Research Center for Floriculture, Beijing Laboratory of Urban and Rural Ecological Environment, College of Landscape Architecture, Beijing Forestry University, Beijing 100083, China" /><meta name="citation_author" content="Zhang, Qixiang" /><meta name="citation_author_institution" content="Beijing Key Laboratory of Ornamental Plants Germplasm Innovation and Molecular Breeding, National Engineering Research Center for Floriculture, Beijing Laboratory of Urban and Rural Ecological Environment, College of Landscape Architecture, Beijing Forestry University, Beijing 100083, China" /><meta name="citation_author" content="Cheng, Tangren" /><meta name="citation_author_institution" content="Beijing Key Laboratory of Ornamental Plants Germplasm Innovation and Molecular Breeding, National Engineering Research Center for Floriculture, Beijing Laboratory of Urban and Rural Ecological Environment, College of Landscape Architecture, Beijing Forestry University, Beijing 100083, China" /><meta name="citation_author" content="Yang, Weiru" /><meta name="citation_author_institution" content="Beijing Key Laboratory of Ornamental Plants Germplasm Innovation and Molecular Breeding, National Engineering Research Center for Floriculture, Beijing Laboratory of Urban and Rural Ecological Environment, College of Landscape Architecture, Beijing Forestry University, Beijing 100083, China" /><meta name="citation_author" content="Pan, Huitang" /><meta name="citation_author_institution" content="Beijing Key Laboratory of Ornamental Plants Germplasm Innovation and Molecular Breeding, National Engineering Research Center for Floriculture, Beijing Laboratory of Urban and Rural Ecological Environment, College of Landscape Architecture, Beijing Forestry University, Beijing 100083, China" /><meta name="citation_author" content="Zhong, Junjun" /><meta name="citation_author_institution" content="Beijing Key Laboratory of Ornamental Plants Germplasm Innovation and Molecular Breeding, National Engineering Research Center for Floriculture, Beijing Laboratory of Urban and Rural Ecological Environment, College of Landscape Architecture, Beijing Forestry University, Beijing 100083, China" /><meta name="citation_author" content="Huang, Long" /><meta name="citation_author_institution" content="Biomarker Technologies Corporation, Beijing, China" /><meta name="citation_author" content="Liu, Enze" /><meta name="citation_author_institution" content="Biomarker Technologies Corporation, Beijing, China" /><meta name="citation_title" content="High-density genetic map construction and identification of a locus controlling weeping trait in an ornamental woody plant (Prunus mume Sieb. et Zucc)" /><meta name="citation_firstpage" content="183" /><meta name="citation_lastpage" content="191" /><meta name="citation_doi" content="10.1093/dnares/dsv003" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2015/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=An SNP map of the human genome generated by reduced representation shotgun sequencing; citation_author=Altshuler  D.; citation_author=Pollara  V.J.; citation_author=Cowles  C.R.;  citation_journal_title=Nature;  citation_year=2000;  citation_volume=407;  citation_pages=513-6; " /><meta name="citation_reference" content="citation_title=High-throughput SNP discovery through deep resequencing of a reduced representation library to anchor and orient scaffolds in the soybean whole genome sequence; citation_author=Hyten  D.L.; citation_author=Cannon  S.B.; citation_author=Song  Q.;  citation_journal_title=BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=38" /><meta name="citation_reference" content="citation_title=Genetic analysis using genomic representations; citation_author=Lucito  R.; citation_author=Nakimura  M.; citation_author=West  J.A.;  citation_journal_title=Proc. Natl Acad. Sci.;  citation_year=1998;  citation_volume=95;  citation_pages=4487-92; " /><meta name="citation_reference" content="citation_title=SNP discovery and allele frequency estimation by deep sequencing of reduced representation libraries; citation_author=Van Tassell  C.P.; citation_author=Smith  T.P.; citation_author=Matukumalli  L.K.;  citation_journal_title=Nat. Methods;  citation_year=2008;  citation_volume=5;  citation_pages=247-52; " /><meta name="citation_reference" content="citation_title=Linkage mapping and comparative genomics using next-generation RAD sequencing of a non-model organism; citation_author=Baxter  S.W.; citation_author=Davey  J.W.; citation_author=Johnston  J.S.;  citation_journal_title=PloS ONE;  citation_year=2011;  citation_volume=6;  citation_pages=e19315" /><meta name="citation_reference" content="citation_title=Construction and application for QTL analysis of a Restriction Site Associated DNA (RAD) linkage map in barley; citation_author=Chutimanitsakun  Y.; citation_author=Nipper  R.W.; citation_author=Cuesta-Marcos  A.;  citation_journal_title=BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=4" /><meta name="citation_reference" content="citation_title=Mapping with RAD (restriction-site associated DNA) markers to rapidly identify QTL for stem rust resistance in Lolium perenne; citation_author=Pfender  W.; citation_author=Saha  M.; citation_author=Johnson  E.; citation_author=Slabaugh  M.;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2011;  citation_volume=122;  citation_pages=1467-80; " /><meta name="citation_reference" content="citation_title=Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach; citation_author=Poland  J.A.; citation_author=Brown  P.J.; citation_author=Sorrells  M.E.; citation_author=Jannink  J.-L.;  citation_journal_title=PloS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e32253" /><meta name="citation_reference" content="citation_title=The development of 7E chromosome-specific molecular markers for Thinopyrum elongatum based on SLAF-seq technology; citation_author=Chen  S.; citation_author=Huang  Z.; citation_author=Dai  Y.;  citation_journal_title=PloS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=e65122" /><meta name="citation_reference" content="citation_title=SLAF-seq: an efficient method of large-scale de novo SNP discovery and genotyping using high-throughput sequencing; citation_author=Sun  X.; citation_author=Liu  D.; citation_author=Zhang  X.;  citation_journal_title=PloS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=e58700" /><meta name="citation_reference" content="citation_title=Construction of a high-density genetic map for sesame based on large scale marker development by specific length amplified fragment (SLAF) sequencing; citation_author=Zhang  Y.; citation_author=Wang  L.; citation_author=Xin  H.;  citation_journal_title=BMC Plant Biol.;  citation_year=2013;  citation_volume=13;  citation_pages=141" /><meta name="citation_reference" content="citation_title=Genetic control of juvenile growth and botanical architecture in an ornamental woody plant,. Prunus mume Sieb. et Zucc. as revealed by a high-density linkage map; citation_author=Sun  L.; citation_author=Wang  Y.; citation_author=Yan  X.;  citation_journal_title=BMC Genet.;  citation_year=2014;  citation_volume=15;  citation_pages=S1" /><meta name="citation_reference" content="citation_author=Chen  J.; citation_publisher=Hainan Publishing House, Haikou, China;  citation_title=Chinese mei flowers;  citation_year=1996;  citation_pages=51-2; " /><meta name="citation_reference" content="citation_title=Inheritance of a weeping character and the low frequency of rooting from cuttings of the mulberry variety ‘Shidareguwa; citation_author=Yamanouchi  H.; citation_author=Koyama  A.; citation_author=Machii  H.; citation_author=Takyu  T.; citation_author=Muramatsu  N.;  citation_journal_title=Plant Breed.;  citation_year=2009;  citation_volume=128;  citation_pages=321-3; " /><meta name="citation_reference" content="citation_title=Gibberellin metabolism and its regulation; citation_author=Yamaguchi  S.;  citation_journal_title=Annu. Rev. Plant Biol.;  citation_year=2008;  citation_volume=59;  citation_pages=225-51; " /><meta name="citation_reference" content="citation_title=Genetic interactions of pillar and weeping peach genotypes; citation_author=Werner  D.J.; citation_author=Chaparro  J.X.;  citation_journal_title=Hort Sci.;  citation_year=2005;  citation_volume=40;  citation_pages=18-20; " /><meta name="citation_reference" content="citation_title=Molecular genetic mapping of peach; citation_author=Dirlewanger  E.; citation_author=Bodo  C.;  citation_journal_title=Euphytica;  citation_year=1994;  citation_volume=77;  citation_pages=101-3; " /><meta name="citation_reference" content="citation_author=Ymeng  L.; citation_publisher=College of Horticulture Science and Engineering, Shandong Agricultural University, Taian;  citation_title=Physiological characteristics of weeping phenomenon and its simple sequence repeats analysis in weeping peach (Prunus persica Var. Pendula);  citation_year=2006;  citation_pages=54" /><meta name="citation_reference" content="citation_title=SOAP: short oligonucleotide alignment program; citation_author=Li  R.; citation_author=Li  Y.; citation_author=Kristiansen  K.; citation_author=Wang  J.;  citation_journal_title=Bioinformatics;  citation_year=2008;  citation_volume=24;  citation_pages=713-4; " /><meta name="citation_reference" content="citation_title=Construction and analysis of high-density linkage map using high-throughput sequencing data; citation_author=Liu  D.; citation_author=Ma  C.; citation_author=Hong  W.;  citation_journal_title=PloS ONE;  citation_year=2014;  citation_volume=9;  citation_pages=e98855" /><meta name="citation_reference" content="citation_title=Constructing dense genetic linkage maps; citation_author=Jansen  J.; citation_author=De Jong  A.; citation_author=Van Ooijen  J.;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2001;  citation_volume=102;  citation_pages=1113-22; " /><meta name="citation_reference" content="citation_title=Multipoint maximum likelihood mapping in a full-sib family of an outbreeding species; citation_author=Van Ooijen  J.;  citation_journal_title=Genet. Res.;  citation_year=2011;  citation_volume=93;  citation_pages=343-9; " /><meta name="citation_reference" content="citation_title=SMOOTH: a statistical method for successful removal of genotyping errors from high-density genetic linkage data; citation_author=van Os  H.; citation_author=Stam  P.; citation_author=Visser  R.G.; citation_author=van Eck  H.J.;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2005;  citation_volume=112;  citation_pages=187-94; " /><meta name="citation_reference" content="citation_title=Genome-wide association study of flowering time and grain yield traits in a worldwide collection of rice germplasm; citation_author=Huang  X.; citation_author=Zhao  Y.; citation_author=Wei  X.;  citation_journal_title=Nat. Genet.;  citation_year=2012;  citation_volume=44;  citation_pages=32-9; " /><meta name="citation_reference" content="citation_title=Mapping quantitative trait loci using distorted markers; citation_author=Xu  S.; citation_author=Hu  Z.;  citation_journal_title=Int. J. Plant Genom;  citation_year=2010;  citation_volume=2009;  citation_pages=2-10; " /><meta name="citation_reference" content="citation_title=The estimation of map distances from recombination values; citation_author=Kosambi  D.;  citation_journal_title=Ann. Eugenics;  citation_year=1943;  citation_volume=12;  citation_pages=172-5; " /><meta name="citation_reference" content="citation_author=Broman  K.W.; citation_author=Sen  S.; citation_publisher=Springer, New York, ;  citation_title=A guide to QTL mapping with R/qtl;  citation_year=2009; " /><meta name="citation_reference" content="citation_title=Genome sequencing reveals agronomically important loci in rice using MutMap; citation_author=Abe  A.; citation_author=Kosugi  S.; citation_author=Yoshida  K.;  citation_journal_title=Nat. Biotechnol.;  citation_year=2012;  citation_volume=30;  citation_pages=174-8; " /><meta name="citation_reference" content="citation_title=Gene Ontology: tool for the unification of biology; citation_author=Ashburner  M.; citation_author=Ball  C.A.; citation_author=Blake  J.A.;  citation_journal_title=Nat. Genet.;  citation_year=2000;  citation_volume=25;  citation_pages=25-9; " /><meta name="citation_reference" content="citation_title=Chromosome sectioning of Prunus mume Siet. et Zucc all the year around; citation_author=Chen  J.; citation_author=Yang  B.; citation_author=Yang  W.; citation_author=Hao  R.; citation_author=Zhang  J.; citation_author=Zhang  Q.;  citation_journal_title=Acta Agrticulturae Boreali-Occidentalis Sinica;  citation_year=2013;  citation_volume=22;  citation_pages=150-4; " /><meta name="citation_reference" content="citation_title=Informativeness of human (dC-dA) n·(dG-dT) n polymorphisms; citation_author=Weber  J.L.;  citation_journal_title=Genomics;  citation_year=1990;  citation_volume=7;  citation_pages=524-30; " /><meta name="citation_reference" content="citation_title=Expression of cellulose synthase-like (Csl) genes in insect cells reveals that CslA family members encode mannan synthases; citation_author=Liepman  A.H.; citation_author=Wilkerson  C.G.; citation_author=Keegstra  K.;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2005;  citation_volume=102;  citation_pages=2221-6; " /><meta name="citation_reference" content="citation_title=A phylogenetic study of endo-beta-1, 4-glucanase in higher termites; citation_author=Bujang  N.; citation_author=Harrison  N.; citation_author=Su  N.-Y.;  citation_journal_title=Insectes Sociaux;  citation_year=2014;  citation_volume=61;  citation_pages=29-40; " /><meta name="citation_reference" content="citation_title=Two divergent endo-beta-1, 4-glucanase genes exhibit overlapping expression in ripening fruit and abscising flowers; citation_author=Lashbrook  C.C.; citation_author=Gonzalez-Bosch  C.; citation_author=Bennett  A.B.;  citation_journal_title=Plant Cell Online;  citation_year=1994;  citation_volume=6;  citation_pages=1485-93; " /><meta name="citation_reference" content="citation_title=Cell wall regeneration in Chlamydomonas: accumulation of mRNAs encoding cell wall hydroxyproline-rich glycoproteins; citation_author=Adair  W.S.; citation_author=Apt  K.E.;  citation_journal_title=Proc. Natl Acad. Sci.;  citation_year=1990;  citation_volume=87;  citation_pages=7355-9; " /><meta name="citation_reference" content="citation_title=Whole-genome comparison of leucine-rich repeat extensins in Arabidopsis and rice. A conserved family of cell wall proteins form a vegetative and a reproductive clade; citation_author=Baumberger  N.; citation_author=Doesseger  B.; citation_author=Guyot  R.;  citation_journal_title=Plant Physiol.;  citation_year=2003;  citation_volume=131;  citation_pages=1313-26; " /><meta name="citation_reference" content="citation_title=An auxin-inducible gene from loblolly pine (Pinus taeda L.) is differentially expressed in mature and juvenile-phase shoots and encodes a putative transmembrane protein; citation_author=Busov  V.B.; citation_author=Johannes  E.; citation_author=Whetten  R.W.;  citation_journal_title=Planta;  citation_year=2004;  citation_volume=218;  citation_pages=916-27; " /><meta name="citation_reference" content="citation_author=Andersson Gunnerås  S.; citation_publisher=Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Sciences, Umeå;  citation_title=Wood formation and transcript analysis with focus on tension wood and ethylene biology;  citation_year=2005;  citation_pages=3" /><meta name="citation_reference" content="citation_title=The cinnamyl alcohol dehydrogenase gene family in Populus: phylogeny, organization, and expression; citation_author=Barakat  A.; citation_author=Bagniewska-Zadworna  A.; citation_author=Choi  A.;  citation_journal_title=BMC Plant Biol.;  citation_year=2009;  citation_volume=9;  citation_pages=26" /><meta name="citation_reference" content="citation_title=NAC transcription factors, NST1 and NST3, are key regulators of the formation of secondary walls in woody tissues of Arabidopsis; citation_author=Mitsuda  N.; citation_author=Iwase  A.; citation_author=Yamamoto  H.;  citation_journal_title=Plant Cell Online;  citation_year=2007;  citation_volume=19;  citation_pages=270-80; " /><meta name="citation_reference" content="citation_title=Expression of gibberellin 3. BETA.-hydroxylase gene in a gravi-response mutant, weeping Japanese flowering cherry; citation_author=Sugano  M.; citation_author=Nakagawa  Y.; citation_author=Nyunoya  H.; citation_author=Nakamura  T.;  citation_journal_title=Biol. Sci. Space;  citation_year=2004;  citation_volume=18;  citation_pages=261-6; " /><meta name="citation_reference" content="citation_title=Loss of 26S proteasome function leads to increased cell size and decreased cell number in Arabidopsis shoot organs; citation_author=Kurepa  J.; citation_author=Wang  S.; citation_author=Li  Y.; citation_author=Zaitlin  D.; citation_author=Pierce  A.J.; citation_author=Smalle  J.A.;  citation_journal_title=Plant Physiol.;  citation_year=2009;  citation_volume=150;  citation_pages=178-89; " /><meta name="citation_reference" content="citation_title=The nodule inception-like protein 7 modulates nitrate sensing and metabolism in Arabidopsis; citation_author=Castaings  L.; citation_author=Camargo  A.; citation_author=Pocholle  D.;  citation_journal_title=Plant J.;  citation_year=2009;  citation_volume=57;  citation_pages=426-35; " /><meta name="citation_reference" content="citation_title=The basic helix-loop-helix transcription factor family in plants: a genome-wide study of protein structure and functional diversity; citation_author=Heim  M.A.; citation_author=Jakoby  M.; citation_author=Werber  M.; citation_author=Martin  C.; citation_author=Weisshaar  B.; citation_author=Bailey  P.C.;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2003;  citation_volume=20;  citation_pages=735-47; " /><meta name="citation_reference" content="citation_title=Intergenic transcription by RNA polymerase II coordinates Pol IV and Pol V in siRNA-directed transcriptional gene silencing in Arabidopsis; citation_author=Zheng  B.; citation_author=Wang  Z.; citation_author=Li  S.; citation_author=Yu  B.; citation_author=Liu  J.-Y.; citation_author=Chen  X.;  citation_journal_title=Genes Dev.;  citation_year=2009;  citation_volume=23;  citation_pages=2850-60; " /><meta name="citation_reference" content="citation_title=Genome-wide genetic marker discovery and genotyping using next-generation sequencing; citation_author=Davey  J.W.; citation_author=Hohenlohe  P.A.; citation_author=Etter  P.D.; citation_author=Boone  J.Q.; citation_author=Catchen  J.M.; citation_author=Blaxter  M.L.;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2011;  citation_volume=12;  citation_pages=499-510; " /><meta name="citation_reference" content="citation_title=Genome-wide characterization and linkage mapping of simple sequence repeats in Mei (Prunus mume Sieb. et Zucc.); citation_author=Sun  L.; citation_author=Yang  W.; citation_author=Zhang  Q.;  citation_journal_title=PloS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=e59562" /><meta name="citation_reference" content="citation_title=The genome of Prunus mume; citation_author=Zhang  Q.X.; citation_author=Chen  W.B.; citation_author=Sun  L.D.;  citation_journal_title=Nat. Commun.;  citation_year=2012;  citation_volume=3;  citation_pages=1318" /><meta name="citation_reference" content="citation_title=The peach genome; citation_author=Arús  P.; citation_author=Verde  I.; citation_author=Sosinski  B.; citation_author=Zhebentyayeva  T.; citation_author=Abbott  A.G.;  citation_journal_title=Tree Genet. Genom.;  citation_year=2012;  citation_volume=8;  citation_pages=531-47; " /><meta name="citation_reference" content="citation_title=Molecular mapping of segregation distortion loci in Aegilops tauschii; citation_author=Faris  J.; citation_author=Laddomada  B.; citation_author=Gill  B.;  citation_journal_title=Genetics;  citation_year=1998;  citation_volume=149;  citation_pages=319-27; " /><meta name="citation_reference" content="citation_title=Mapping segregation distortion loci and quantitative trait loci for spikelet sterility in rice (Oryza sativa L.); citation_author=Wang  C.; citation_author=Zhu  C.; citation_author=Zhai  H.; citation_author=Wan  J.;  citation_journal_title=Genet. Res.;  citation_year=2005;  citation_volume=86;  citation_pages=97-106; " /><meta name="citation_reference" content="citation_title=Effects of missing marker and segregation distortion on QTL mapping in F2 populations; citation_author=Zhang  L.; citation_author=Wang  S.; citation_author=Li  H.;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2010;  citation_volume=121;  citation_pages=1071-82; " /><meta name="citation_reference" content="citation_title=Quantitative trait locus mapping can benefit from segregation distortion; citation_author=Xu  S.;  citation_journal_title=Genetics;  citation_year=2008;  citation_volume=180;  citation_pages=2201-8; " /><meta name="citation_reference" content="citation_title=Peach breeding for growth habit; citation_author=Bassi  D.; citation_author=Rizzo  M.; citation_author=Geibel  M.; citation_author=Fischer  M.; citation_author=Fischer  C.;  citation_journal_title=Acta Hortic.;  citation_year=2000;  citation_volume=1;  citation_pages=411-4; " /><meta name="citation_reference" content="citation_title=Genetic study of the weeping habit in peach; citation_author=Monet  R.; citation_author=Bastard  Y.; citation_author=Gibault  B.;  citation_journal_title=Agronomie;  citation_year=1988;  citation_volume=8;  citation_pages=127-32; " /><meta name="citation_reference" content="citation_title=Genetic interactions of pillar (columnar), compact, and dwarf peach tree genotypes; citation_author=Scorza  R.; citation_author=Bassi  D.; citation_author=Liverani  A.;  citation_journal_title=J. Am. Soc. Hortic. Sci.;  citation_year=2002;  citation_volume=127;  citation_pages=254-61; " /><meta name="citation_reference" content="citation_title=Inheritance of some characteristics and breeding of new hybrids in flowering peaches; citation_author=Yamazaki  K.; citation_author=Okabe  M.; citation_author=Takahashi  E.;  citation_journal_title=Bull. Kanagawa Hort. Exp. Sta. (Japan);  citation_year=1987;  citation_volume=34;  citation_pages=46-53; " /><meta name="citation_reference" content="citation_title=The character display of Castanea mollissima var. pendula Sexual Progenies; citation_author=Tu  S.; citation_author=Qiu  F.;  citation_journal_title=Acta Hirticuoturae Sinica;  citation_year=2007;  citation_volume=34;  citation_pages=760-2; " /><meta name="citation_reference" content="citation_title=Comparison of mechanical properties of tension and opposite wood in Populus; citation_author=Coutand  C.; citation_author=Jeronimidis  G.; citation_author=Chanson  B.; citation_author=Loup  C.;  citation_journal_title=Wood Sci. Technol.;  citation_year=2004;  citation_volume=38;  citation_pages=11-24; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/3/183/1592679/dsv003.pdf" /><meta name="description" content="Abstract.  High-density genetic map is a valuable tool for fine mapping locus controlling a specific trait especially for perennial woody plants. In this study," />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/3/183/591017" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/3/183/591017" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/22/3/183/591017");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsv003';
    </script>
<title>
	High-density genetic map construction and identification of a locus controlling weeping trait in an ornamental woody plant (Prunus mume Sieb. et Zucc) | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./591017" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bcy9ATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWmhhbmcsIEppZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJlaWppbmcgS2V5IExhYm9yYXRvcnkgb2YgT3JuYW1lbnRhbCBQbGFudHMgR2VybXBsYXNtIElubm92YXRpb24gYW5kIE1vbGVjdWxhciBCcmVlZGluZywgTmF0aW9uYWwgRW5naW5lZXJpbmcgUmVzZWFyY2ggQ2VudGVyIGZvciBGbG9yaWN1bHR1cmUsIEJlaWppbmcgTGFib3JhdG9yeSBvZiBVcmJhbiBhbmQgUnVyYWwgRWNvbG9naWNhbCBFbnZpcm9ubWVudCwgQ29sbGVnZSBvZiBMYW5kc2NhcGUgQXJjaGl0ZWN0dXJlLCBCZWlqaW5nIEZvcmVzdHJ5IFVuaXZlcnNpdHksIEJlaWppbmcgMTAwMDgzLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlpoYW5nLCBRaXhpYW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmVpamluZyBLZXkgTGFib3JhdG9yeSBvZiBPcm5hbWVudGFsIFBsYW50cyBHZXJtcGxhc20gSW5ub3ZhdGlvbiBhbmQgTW9sZWN1bGFyIEJyZWVkaW5nLCBOYXRpb25hbCBFbmdpbmVlcmluZyBSZXNlYXJjaCBDZW50ZXIgZm9yIEZsb3JpY3VsdHVyZSwgQmVpamluZyBMYWJvcmF0b3J5IG9mIFVyYmFuIGFuZCBSdXJhbCBFY29sb2dpY2FsIEVudmlyb25tZW50LCBDb2xsZWdlIG9mIExhbmRzY2FwZSBBcmNoaXRlY3R1cmUsIEJlaWppbmcgRm9yZXN0cnkgVW5pdmVyc2l0eSwgQmVpamluZyAxMDAwODMsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2hlbmcsIFRhbmdyZW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCZWlqaW5nIEtleSBMYWJvcmF0b3J5IG9mIE9ybmFtZW50YWwgUGxhbnRzIEdlcm1wbGFzbSBJbm5vdmF0aW9uIGFuZCBNb2xlY3VsYXIgQnJlZWRpbmcsIE5hdGlvbmFsIEVuZ2luZWVyaW5nIFJlc2VhcmNoIENlbnRlciBmb3IgRmxvcmljdWx0dXJlLCBCZWlqaW5nIExhYm9yYXRvcnkgb2YgVXJiYW4gYW5kIFJ1cmFsIEVjb2xvZ2ljYWwgRW52aXJvbm1lbnQsIENvbGxlZ2Ugb2YgTGFuZHNjYXBlIEFyY2hpdGVjdHVyZSwgQmVpamluZyBGb3Jlc3RyeSBVbml2ZXJzaXR5LCBCZWlqaW5nIDEwMDA4MywgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJZYW5nLCBXZWlydSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJlaWppbmcgS2V5IExhYm9yYXRvcnkgb2YgT3JuYW1lbnRhbCBQbGFudHMgR2VybXBsYXNtIElubm92YXRpb24gYW5kIE1vbGVjdWxhciBCcmVlZGluZywgTmF0aW9uYWwgRW5naW5lZXJpbmcgUmVzZWFyY2ggQ2VudGVyIGZvciBGbG9yaWN1bHR1cmUsIEJlaWppbmcgTGFib3JhdG9yeSBvZiBVcmJhbiBhbmQgUnVyYWwgRWNvbG9naWNhbCBFbnZpcm9ubWVudCwgQ29sbGVnZSBvZiBMYW5kc2NhcGUgQXJjaGl0ZWN0dXJlLCBCZWlqaW5nIEZvcmVzdHJ5IFVuaXZlcnNpdHksIEJlaWppbmcgMTAwMDgzLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlBhbiwgSHVpdGFuZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJlaWppbmcgS2V5IExhYm9yYXRvcnkgb2YgT3JuYW1lbnRhbCBQbGFudHMgR2VybXBsYXNtIElubm92YXRpb24gYW5kIE1vbGVjdWxhciBCcmVlZGluZywgTmF0aW9uYWwgRW5naW5lZXJpbmcgUmVzZWFyY2ggQ2VudGVyIGZvciBGbG9yaWN1bHR1cmUsIEJlaWppbmcgTGFib3JhdG9yeSBvZiBVcmJhbiBhbmQgUnVyYWwgRWNvbG9naWNhbCBFbnZpcm9ubWVudCwgQ29sbGVnZSBvZiBMYW5kc2NhcGUgQXJjaGl0ZWN0dXJlLCBCZWlqaW5nIEZvcmVzdHJ5IFVuaXZlcnNpdHksIEJlaWppbmcgMTAwMDgzLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ilpob25nLCBKdW5qdW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCZWlqaW5nIEtleSBMYWJvcmF0b3J5IG9mIE9ybmFtZW50YWwgUGxhbnRzIEdlcm1wbGFzbSBJbm5vdmF0aW9uIGFuZCBNb2xlY3VsYXIgQnJlZWRpbmcsIE5hdGlvbmFsIEVuZ2luZWVyaW5nIFJlc2VhcmNoIENlbnRlciBmb3IgRmxvcmljdWx0dXJlLCBCZWlqaW5nIExhYm9yYXRvcnkgb2YgVXJiYW4gYW5kIFJ1cmFsIEVjb2xvZ2ljYWwgRW52aXJvbm1lbnQsIENvbGxlZ2Ugb2YgTGFuZHNjYXBlIEFyY2hpdGVjdHVyZSwgQmVpamluZyBGb3Jlc3RyeSBVbml2ZXJzaXR5LCBCZWlqaW5nIDEwMDA4MywgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIdWFuZywgTG9uZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJpb21hcmtlciBUZWNobm9sb2dpZXMgQ29ycG9yYXRpb24sIEJlaWppbmcsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGl1LCBFbnplIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvbWFya2VyIFRlY2hub2xvZ2llcyBDb3Jwb3JhdGlvbiwgQmVpamluZywgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkhpZ2gtZGVuc2l0eSBnZW5ldGljIG1hcCBjb25zdHJ1Y3Rpb24gYW5kIGlkZW50aWZpY2F0aW9uIG9mIGEgbG9jdXMgY29udHJvbGxpbmcgd2VlcGluZyB0cmFpdCBpbiBhbiBvcm5hbWVudGFsIHdvb2R5IHBsYW50IChQcnVudXMgbXVtZSBTaWViLiBldCBadWNjKSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjE4MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMTkxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN2MDAzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTUvMDYvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBTTlAgbWFwIG9mIHRoZSBodW1hbiBnZW5vbWUgZ2VuZXJhdGVkIGJ5IHJlZHVjZWQgcmVwcmVzZW50YXRpb24gc2hvdGd1biBzZXF1ZW5jaW5nOyBjaXRhdGlvbl9hdXRob3I9QWx0c2h1bGVyICBELjsgY2l0YXRpb25fYXV0aG9yPVBvbGxhcmEgIFYuSi47IGNpdGF0aW9uX2F1dGhvcj1Db3dsZXMgIEMuUi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT00MDc7ICBjaXRhdGlvbl9wYWdlcz01MTMtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC10aHJvdWdocHV0IFNOUCBkaXNjb3ZlcnkgdGhyb3VnaCBkZWVwIHJlc2VxdWVuY2luZyBvZiBhIHJlZHVjZWQgcmVwcmVzZW50YXRpb24gbGlicmFyeSB0byBhbmNob3IgYW5kIG9yaWVudCBzY2FmZm9sZHMgaW4gdGhlIHNveWJlYW4gd2hvbGUgZ2Vub21lIHNlcXVlbmNlOyBjaXRhdGlvbl9hdXRob3I9SHl0ZW4gIEQuTC47IGNpdGF0aW9uX2F1dGhvcj1DYW5ub24gIFMuQi47IGNpdGF0aW9uX2F1dGhvcj1Tb25nICBRLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGFuYWx5c2lzIHVzaW5nIGdlbm9taWMgcmVwcmVzZW50YXRpb25zOyBjaXRhdGlvbl9hdXRob3I9THVjaXRvICBSLjsgY2l0YXRpb25fYXV0aG9yPU5ha2ltdXJhICBNLjsgY2l0YXRpb25fYXV0aG9yPVdlc3QgIEouQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwgQWNhZC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT05NTsgIGNpdGF0aW9uX3BhZ2VzPTQ0ODctOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNOUCBkaXNjb3ZlcnkgYW5kIGFsbGVsZSBmcmVxdWVuY3kgZXN0aW1hdGlvbiBieSBkZWVwIHNlcXVlbmNpbmcgb2YgcmVkdWNlZCByZXByZXNlbnRhdGlvbiBsaWJyYXJpZXM7IGNpdGF0aW9uX2F1dGhvcj1WYW4gVGFzc2VsbCAgQy5QLjsgY2l0YXRpb25fYXV0aG9yPVNtaXRoICBULlAuOyBjaXRhdGlvbl9hdXRob3I9TWF0dWt1bWFsbGkgIEwuSy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MjQ3LTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MaW5rYWdlIG1hcHBpbmcgYW5kIGNvbXBhcmF0aXZlIGdlbm9taWNzIHVzaW5nIG5leHQtZ2VuZXJhdGlvbiBSQUQgc2VxdWVuY2luZyBvZiBhIG5vbi1tb2RlbCBvcmdhbmlzbTsgY2l0YXRpb25fYXV0aG9yPUJheHRlciAgUy5XLjsgY2l0YXRpb25fYXV0aG9yPURhdmV5ICBKLlcuOyBjaXRhdGlvbl9hdXRob3I9Sm9obnN0b24gIEouUy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz1lMTkzMTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gYW5kIGFwcGxpY2F0aW9uIGZvciBRVEwgYW5hbHlzaXMgb2YgYSBSZXN0cmljdGlvbiBTaXRlIEFzc29jaWF0ZWQgRE5BIChSQUQpIGxpbmthZ2UgbWFwIGluIGJhcmxleTsgY2l0YXRpb25fYXV0aG9yPUNodXRpbWFuaXRzYWt1biAgWS47IGNpdGF0aW9uX2F1dGhvcj1OaXBwZXIgIFIuVy47IGNpdGF0aW9uX2F1dGhvcj1DdWVzdGEtTWFyY29zICBBLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcHBpbmcgd2l0aCBSQUQgKHJlc3RyaWN0aW9uLXNpdGUgYXNzb2NpYXRlZCBETkEpIG1hcmtlcnMgdG8gcmFwaWRseSBpZGVudGlmeSBRVEwgZm9yIHN0ZW0gcnVzdCByZXNpc3RhbmNlIGluIExvbGl1bSBwZXJlbm5lOyBjaXRhdGlvbl9hdXRob3I9UGZlbmRlciAgVy47IGNpdGF0aW9uX2F1dGhvcj1TYWhhICBNLjsgY2l0YXRpb25fYXV0aG9yPUpvaG5zb24gIEUuOyBjaXRhdGlvbl9hdXRob3I9U2xhYmF1Z2ggIE0uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyMjsgIGNpdGF0aW9uX3BhZ2VzPTE0NjctODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIGhpZ2gtZGVuc2l0eSBnZW5ldGljIG1hcHMgZm9yIGJhcmxleSBhbmQgd2hlYXQgdXNpbmcgYSBub3ZlbCB0d28tZW56eW1lIGdlbm90eXBpbmctYnktc2VxdWVuY2luZyBhcHByb2FjaDsgY2l0YXRpb25fYXV0aG9yPVBvbGFuZCAgSi5BLjsgY2l0YXRpb25fYXV0aG9yPUJyb3duICBQLkouOyBjaXRhdGlvbl9hdXRob3I9U29ycmVsbHMgIE0uRS47IGNpdGF0aW9uX2F1dGhvcj1KYW5uaW5rICBKLi1MLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWUzMjI1MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBkZXZlbG9wbWVudCBvZiA3RSBjaHJvbW9zb21lLXNwZWNpZmljIG1vbGVjdWxhciBtYXJrZXJzIGZvciBUaGlub3B5cnVtIGVsb25nYXR1bSBiYXNlZCBvbiBTTEFGLXNlcSB0ZWNobm9sb2d5OyBjaXRhdGlvbl9hdXRob3I9Q2hlbiAgUy47IGNpdGF0aW9uX2F1dGhvcj1IdWFuZyAgWi47IGNpdGF0aW9uX2F1dGhvcj1EYWkgIFkuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9ZTY1MTIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U0xBRi1zZXE6IGFuIGVmZmljaWVudCBtZXRob2Qgb2YgbGFyZ2Utc2NhbGUgZGUgbm92byBTTlAgZGlzY292ZXJ5IGFuZCBnZW5vdHlwaW5nIHVzaW5nIGhpZ2gtdGhyb3VnaHB1dCBzZXF1ZW5jaW5nOyBjaXRhdGlvbl9hdXRob3I9U3VuICBYLjsgY2l0YXRpb25fYXV0aG9yPUxpdSAgRC47IGNpdGF0aW9uX2F1dGhvcj1aaGFuZyAgWC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lNTg3MDAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgYSBoaWdoLWRlbnNpdHkgZ2VuZXRpYyBtYXAgZm9yIHNlc2FtZSBiYXNlZCBvbiBsYXJnZSBzY2FsZSBtYXJrZXIgZGV2ZWxvcG1lbnQgYnkgc3BlY2lmaWMgbGVuZ3RoIGFtcGxpZmllZCBmcmFnbWVudCAoU0xBRikgc2VxdWVuY2luZzsgY2l0YXRpb25fYXV0aG9yPVpoYW5nICBZLjsgY2l0YXRpb25fYXV0aG9yPVdhbmcgIEwuOyBjaXRhdGlvbl9hdXRob3I9WGluICBILjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MTQxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBjb250cm9sIG9mIGp1dmVuaWxlIGdyb3d0aCBhbmQgYm90YW5pY2FsIGFyY2hpdGVjdHVyZSBpbiBhbiBvcm5hbWVudGFsIHdvb2R5IHBsYW50LC4gUHJ1bnVzIG11bWUgU2llYi4gZXQgWnVjYy4gYXMgcmV2ZWFsZWQgYnkgYSBoaWdoLWRlbnNpdHkgbGlua2FnZSBtYXA7IGNpdGF0aW9uX2F1dGhvcj1TdW4gIEwuOyBjaXRhdGlvbl9hdXRob3I9V2FuZyAgWS47IGNpdGF0aW9uX2F1dGhvcj1ZYW4gIFguOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9UzEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9hdXRob3I9Q2hlbiAgSi47IGNpdGF0aW9uX3B1Ymxpc2hlcj1IYWluYW4gUHVibGlzaGluZyBIb3VzZSwgSGFpa291LCBDaGluYTsgIGNpdGF0aW9uX3RpdGxlPUNoaW5lc2UgbWVpIGZsb3dlcnM7ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl9wYWdlcz01MS0yOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jbmhlcml0YW5jZSBvZiBhIHdlZXBpbmcgY2hhcmFjdGVyIGFuZCB0aGUgbG93IGZyZXF1ZW5jeSBvZiByb290aW5nIGZyb20gY3V0dGluZ3Mgb2YgdGhlIG11bGJlcnJ5IHZhcmlldHkg4oCYU2hpZGFyZWd1d2E7IGNpdGF0aW9uX2F1dGhvcj1ZYW1hbm91Y2hpICBILjsgY2l0YXRpb25fYXV0aG9yPUtveWFtYSAgQS47IGNpdGF0aW9uX2F1dGhvcj1NYWNoaWkgIEguOyBjaXRhdGlvbl9hdXRob3I9VGFreXUgIFQuOyBjaXRhdGlvbl9hdXRob3I9TXVyYW1hdHN1ICBOLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQnJlZWQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEyODsgIGNpdGF0aW9uX3BhZ2VzPTMyMS0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HaWJiZXJlbGxpbiBtZXRhYm9saXNtIGFuZCBpdHMgcmVndWxhdGlvbjsgY2l0YXRpb25fYXV0aG9yPVlhbWFndWNoaSAgUy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFubnUuIFJldi4gUGxhbnQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NTk7ICBjaXRhdGlvbl9wYWdlcz0yMjUtNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgaW50ZXJhY3Rpb25zIG9mIHBpbGxhciBhbmQgd2VlcGluZyBwZWFjaCBnZW5vdHlwZXM7IGNpdGF0aW9uX2F1dGhvcj1XZXJuZXIgIEQuSi47IGNpdGF0aW9uX2F1dGhvcj1DaGFwYXJybyAgSi5YLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SG9ydCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9MTgtMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBnZW5ldGljIG1hcHBpbmcgb2YgcGVhY2g7IGNpdGF0aW9uX2F1dGhvcj1EaXJsZXdhbmdlciAgRS47IGNpdGF0aW9uX2F1dGhvcj1Cb2RvICBDLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RXVwaHl0aWNhOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTc3OyAgY2l0YXRpb25fcGFnZXM9MTAxLTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX2F1dGhvcj1ZbWVuZyAgTC47IGNpdGF0aW9uX3B1Ymxpc2hlcj1Db2xsZWdlIG9mIEhvcnRpY3VsdHVyZSBTY2llbmNlIGFuZCBFbmdpbmVlcmluZywgU2hhbmRvbmcgQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIFRhaWFuOyAgY2l0YXRpb25fdGl0bGU9UGh5c2lvbG9naWNhbCBjaGFyYWN0ZXJpc3RpY3Mgb2Ygd2VlcGluZyBwaGVub21lbm9uIGFuZCBpdHMgc2ltcGxlIHNlcXVlbmNlIHJlcGVhdHMgYW5hbHlzaXMgaW4gd2VlcGluZyBwZWFjaCAoUHJ1bnVzIHBlcnNpY2EgVmFyLiBQZW5kdWxhKTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3BhZ2VzPTU0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U09BUDogc2hvcnQgb2xpZ29udWNsZW90aWRlIGFsaWdubWVudCBwcm9ncmFtOyBjaXRhdGlvbl9hdXRob3I9TGkgIFIuOyBjaXRhdGlvbl9hdXRob3I9TGkgIFkuOyBjaXRhdGlvbl9hdXRob3I9S3Jpc3RpYW5zZW4gIEsuOyBjaXRhdGlvbl9hdXRob3I9V2FuZyAgSi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9NzEzLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBhbmQgYW5hbHlzaXMgb2YgaGlnaC1kZW5zaXR5IGxpbmthZ2UgbWFwIHVzaW5nIGhpZ2gtdGhyb3VnaHB1dCBzZXF1ZW5jaW5nIGRhdGE7IGNpdGF0aW9uX2F1dGhvcj1MaXUgIEQuOyBjaXRhdGlvbl9hdXRob3I9TWEgIEMuOyBjaXRhdGlvbl9hdXRob3I9SG9uZyAgVy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz1lOTg4NTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3RpbmcgZGVuc2UgZ2VuZXRpYyBsaW5rYWdlIG1hcHM7IGNpdGF0aW9uX2F1dGhvcj1KYW5zZW4gIEouOyBjaXRhdGlvbl9hdXRob3I9RGUgSm9uZyAgQS47IGNpdGF0aW9uX2F1dGhvcj1WYW4gT29pamVuICBKLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDI7ICBjaXRhdGlvbl9wYWdlcz0xMTEzLTIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdWx0aXBvaW50IG1heGltdW0gbGlrZWxpaG9vZCBtYXBwaW5nIGluIGEgZnVsbC1zaWIgZmFtaWx5IG9mIGFuIG91dGJyZWVkaW5nIHNwZWNpZXM7IGNpdGF0aW9uX2F1dGhvcj1WYW4gT29pamVuICBKLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXQuIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9OTM7ICBjaXRhdGlvbl9wYWdlcz0zNDMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U01PT1RIOiBhIHN0YXRpc3RpY2FsIG1ldGhvZCBmb3Igc3VjY2Vzc2Z1bCByZW1vdmFsIG9mIGdlbm90eXBpbmcgZXJyb3JzIGZyb20gaGlnaC1kZW5zaXR5IGdlbmV0aWMgbGlua2FnZSBkYXRhOyBjaXRhdGlvbl9hdXRob3I9dmFuIE9zICBILjsgY2l0YXRpb25fYXV0aG9yPVN0YW0gIFAuOyBjaXRhdGlvbl9hdXRob3I9Vmlzc2VyICBSLkcuOyBjaXRhdGlvbl9hdXRob3I9dmFuIEVjayAgSC5KLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTI7ICBjaXRhdGlvbl9wYWdlcz0xODctOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFzc29jaWF0aW9uIHN0dWR5IG9mIGZsb3dlcmluZyB0aW1lIGFuZCBncmFpbiB5aWVsZCB0cmFpdHMgaW4gYSB3b3JsZHdpZGUgY29sbGVjdGlvbiBvZiByaWNlIGdlcm1wbGFzbTsgY2l0YXRpb25fYXV0aG9yPUh1YW5nICBYLjsgY2l0YXRpb25fYXV0aG9yPVpoYW8gIFkuOyBjaXRhdGlvbl9hdXRob3I9V2VpICBYLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDQ7ICBjaXRhdGlvbl9wYWdlcz0zMi05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXBwaW5nIHF1YW50aXRhdGl2ZSB0cmFpdCBsb2NpIHVzaW5nIGRpc3RvcnRlZCBtYXJrZXJzOyBjaXRhdGlvbl9hdXRob3I9WHUgIFMuOyBjaXRhdGlvbl9hdXRob3I9SHUgIFouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JbnQuIEouIFBsYW50IEdlbm9tOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTIwMDk7ICBjaXRhdGlvbl9wYWdlcz0yLTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZXN0aW1hdGlvbiBvZiBtYXAgZGlzdGFuY2VzIGZyb20gcmVjb21iaW5hdGlvbiB2YWx1ZXM7IGNpdGF0aW9uX2F1dGhvcj1Lb3NhbWJpICBELjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5uLiBFdWdlbmljczsgIGNpdGF0aW9uX3llYXI9MTk0MzsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTE3Mi01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9hdXRob3I9QnJvbWFuICBLLlcuOyBjaXRhdGlvbl9hdXRob3I9U2VuICBTLjsgY2l0YXRpb25fcHVibGlzaGVyPVNwcmluZ2VyLCBOZXcgWW9yaywgOyAgY2l0YXRpb25fdGl0bGU9QSBndWlkZSB0byBRVEwgbWFwcGluZyB3aXRoIFIvcXRsOyAgY2l0YXRpb25feWVhcj0yMDA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgc2VxdWVuY2luZyByZXZlYWxzIGFncm9ub21pY2FsbHkgaW1wb3J0YW50IGxvY2kgaW4gcmljZSB1c2luZyBNdXRNYXA7IGNpdGF0aW9uX2F1dGhvcj1BYmUgIEEuOyBjaXRhdGlvbl9hdXRob3I9S29zdWdpICBTLjsgY2l0YXRpb25fYXV0aG9yPVlvc2hpZGEgIEsuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MTc0LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgT250b2xvZ3k6IHRvb2wgZm9yIHRoZSB1bmlmaWNhdGlvbiBvZiBiaW9sb2d5OyBjaXRhdGlvbl9hdXRob3I9QXNoYnVybmVyICBNLjsgY2l0YXRpb25fYXV0aG9yPUJhbGwgIEMuQS47IGNpdGF0aW9uX2F1dGhvcj1CbGFrZSAgSi5BLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0yNS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaHJvbW9zb21lIHNlY3Rpb25pbmcgb2YgUHJ1bnVzIG11bWUgU2lldC4gZXQgWnVjYyBhbGwgdGhlIHllYXIgYXJvdW5kOyBjaXRhdGlvbl9hdXRob3I9Q2hlbiAgSi47IGNpdGF0aW9uX2F1dGhvcj1ZYW5nICBCLjsgY2l0YXRpb25fYXV0aG9yPVlhbmcgIFcuOyBjaXRhdGlvbl9hdXRob3I9SGFvICBSLjsgY2l0YXRpb25fYXV0aG9yPVpoYW5nICBKLjsgY2l0YXRpb25fYXV0aG9yPVpoYW5nICBRLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QWN0YSBBZ3J0aWN1bHR1cmFlIEJvcmVhbGktT2NjaWRlbnRhbGlzIFNpbmljYTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTE1MC00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmZvcm1hdGl2ZW5lc3Mgb2YgaHVtYW4gKGRDLWRBKSBuwrcoZEctZFQpIG4gcG9seW1vcnBoaXNtczsgY2l0YXRpb25fYXV0aG9yPVdlYmVyICBKLkwuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9NTI0LTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHByZXNzaW9uIG9mIGNlbGx1bG9zZSBzeW50aGFzZS1saWtlIChDc2wpIGdlbmVzIGluIGluc2VjdCBjZWxscyByZXZlYWxzIHRoYXQgQ3NsQSBmYW1pbHkgbWVtYmVycyBlbmNvZGUgbWFubmFuIHN5bnRoYXNlczsgY2l0YXRpb25fYXV0aG9yPUxpZXBtYW4gIEEuSC47IGNpdGF0aW9uX2F1dGhvcj1XaWxrZXJzb24gIEMuRy47IGNpdGF0aW9uX2F1dGhvcj1LZWVnc3RyYSAgSy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTAyOyAgY2l0YXRpb25fcGFnZXM9MjIyMS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHBoeWxvZ2VuZXRpYyBzdHVkeSBvZiBlbmRvLWJldGEtMSwgNC1nbHVjYW5hc2UgaW4gaGlnaGVyIHRlcm1pdGVzOyBjaXRhdGlvbl9hdXRob3I9QnVqYW5nICBOLjsgY2l0YXRpb25fYXV0aG9yPUhhcnJpc29uICBOLjsgY2l0YXRpb25fYXV0aG9yPVN1ICBOLi1ZLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SW5zZWN0ZXMgU29jaWF1eDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT02MTsgIGNpdGF0aW9uX3BhZ2VzPTI5LTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ud28gZGl2ZXJnZW50IGVuZG8tYmV0YS0xLCA0LWdsdWNhbmFzZSBnZW5lcyBleGhpYml0IG92ZXJsYXBwaW5nIGV4cHJlc3Npb24gaW4gcmlwZW5pbmcgZnJ1aXQgYW5kIGFic2Npc2luZyBmbG93ZXJzOyBjaXRhdGlvbl9hdXRob3I9TGFzaGJyb29rICBDLkMuOyBjaXRhdGlvbl9hdXRob3I9R29uemFsZXotQm9zY2ggIEMuOyBjaXRhdGlvbl9hdXRob3I9QmVubmV0dCAgQS5CLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbCBPbmxpbmU7ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTE0ODUtOTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNlbGwgd2FsbCByZWdlbmVyYXRpb24gaW4gQ2hsYW15ZG9tb25hczogYWNjdW11bGF0aW9uIG9mIG1STkFzIGVuY29kaW5nIGNlbGwgd2FsbCBoeWRyb3h5cHJvbGluZS1yaWNoIGdseWNvcHJvdGVpbnM7IGNpdGF0aW9uX2F1dGhvcj1BZGFpciAgVy5TLjsgY2l0YXRpb25fYXV0aG9yPUFwdCAgSy5FLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bCBBY2FkLiBTY2kuOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTg3OyAgY2l0YXRpb25fcGFnZXM9NzM1NS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaG9sZS1nZW5vbWUgY29tcGFyaXNvbiBvZiBsZXVjaW5lLXJpY2ggcmVwZWF0IGV4dGVuc2lucyBpbiBBcmFiaWRvcHNpcyBhbmQgcmljZS4gQSBjb25zZXJ2ZWQgZmFtaWx5IG9mIGNlbGwgd2FsbCBwcm90ZWlucyBmb3JtIGEgdmVnZXRhdGl2ZSBhbmQgYSByZXByb2R1Y3RpdmUgY2xhZGU7IGNpdGF0aW9uX2F1dGhvcj1CYXVtYmVyZ2VyICBOLjsgY2l0YXRpb25fYXV0aG9yPURvZXNzZWdlciAgQi47IGNpdGF0aW9uX2F1dGhvcj1HdXlvdCAgUi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEzMTsgIGNpdGF0aW9uX3BhZ2VzPTEzMTMtMjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGF1eGluLWluZHVjaWJsZSBnZW5lIGZyb20gbG9ibG9sbHkgcGluZSAoUGludXMgdGFlZGEgTC4pIGlzIGRpZmZlcmVudGlhbGx5IGV4cHJlc3NlZCBpbiBtYXR1cmUgYW5kIGp1dmVuaWxlLXBoYXNlIHNob290cyBhbmQgZW5jb2RlcyBhIHB1dGF0aXZlIHRyYW5zbWVtYnJhbmUgcHJvdGVpbjsgY2l0YXRpb25fYXV0aG9yPUJ1c292ICBWLkIuOyBjaXRhdGlvbl9hdXRob3I9Sm9oYW5uZXMgIEUuOyBjaXRhdGlvbl9hdXRob3I9V2hldHRlbiAgUi5XLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnRhOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTIxODsgIGNpdGF0aW9uX3BhZ2VzPTkxNi0yNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fYXV0aG9yPUFuZGVyc3NvbiBHdW5uZXLDpXMgIFMuOyBjaXRhdGlvbl9wdWJsaXNoZXI9RGVwYXJ0bWVudCBvZiBGb3Jlc3QgR2VuZXRpY3MgYW5kIFBsYW50IFBoeXNpb2xvZ3ksIFN3ZWRpc2ggVW5pdmVyc2l0eSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIFVtZcOlOyAgY2l0YXRpb25fdGl0bGU9V29vZCBmb3JtYXRpb24gYW5kIHRyYW5zY3JpcHQgYW5hbHlzaXMgd2l0aCBmb2N1cyBvbiB0ZW5zaW9uIHdvb2QgYW5kIGV0aHlsZW5lIGJpb2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl9wYWdlcz0zIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGNpbm5hbXlsIGFsY29ob2wgZGVoeWRyb2dlbmFzZSBnZW5lIGZhbWlseSBpbiBQb3B1bHVzOiBwaHlsb2dlbnksIG9yZ2FuaXphdGlvbiwgYW5kIGV4cHJlc3Npb247IGNpdGF0aW9uX2F1dGhvcj1CYXJha2F0ICBBLjsgY2l0YXRpb25fYXV0aG9yPUJhZ25pZXdza2EtWmFkd29ybmEgIEEuOyBjaXRhdGlvbl9hdXRob3I9Q2hvaSAgQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MjYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OQUMgdHJhbnNjcmlwdGlvbiBmYWN0b3JzLCBOU1QxIGFuZCBOU1QzLCBhcmUga2V5IHJlZ3VsYXRvcnMgb2YgdGhlIGZvcm1hdGlvbiBvZiBzZWNvbmRhcnkgd2FsbHMgaW4gd29vZHkgdGlzc3VlcyBvZiBBcmFiaWRvcHNpczsgY2l0YXRpb25fYXV0aG9yPU1pdHN1ZGEgIE4uOyBjaXRhdGlvbl9hdXRob3I9SXdhc2UgIEEuOyBjaXRhdGlvbl9hdXRob3I9WWFtYW1vdG8gIEguOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsIE9ubGluZTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTI3MC04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwcmVzc2lvbiBvZiBnaWJiZXJlbGxpbiAzLiBCRVRBLi1oeWRyb3h5bGFzZSBnZW5lIGluIGEgZ3JhdmktcmVzcG9uc2UgbXV0YW50LCB3ZWVwaW5nIEphcGFuZXNlIGZsb3dlcmluZyBjaGVycnk7IGNpdGF0aW9uX2F1dGhvcj1TdWdhbm8gIE0uOyBjaXRhdGlvbl9hdXRob3I9TmFrYWdhd2EgIFkuOyBjaXRhdGlvbl9hdXRob3I9Tnl1bm95YSAgSC47IGNpdGF0aW9uX2F1dGhvcj1OYWthbXVyYSAgVC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2wuIFNjaS4gU3BhY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0yNjEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TG9zcyBvZiAyNlMgcHJvdGVhc29tZSBmdW5jdGlvbiBsZWFkcyB0byBpbmNyZWFzZWQgY2VsbCBzaXplIGFuZCBkZWNyZWFzZWQgY2VsbCBudW1iZXIgaW4gQXJhYmlkb3BzaXMgc2hvb3Qgb3JnYW5zOyBjaXRhdGlvbl9hdXRob3I9S3VyZXBhICBKLjsgY2l0YXRpb25fYXV0aG9yPVdhbmcgIFMuOyBjaXRhdGlvbl9hdXRob3I9TGkgIFkuOyBjaXRhdGlvbl9hdXRob3I9WmFpdGxpbiAgRC47IGNpdGF0aW9uX2F1dGhvcj1QaWVyY2UgIEEuSi47IGNpdGF0aW9uX2F1dGhvcj1TbWFsbGUgIEouQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE1MDsgIGNpdGF0aW9uX3BhZ2VzPTE3OC04OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG5vZHVsZSBpbmNlcHRpb24tbGlrZSBwcm90ZWluIDcgbW9kdWxhdGVzIG5pdHJhdGUgc2Vuc2luZyBhbmQgbWV0YWJvbGlzbSBpbiBBcmFiaWRvcHNpczsgY2l0YXRpb25fYXV0aG9yPUNhc3RhaW5ncyAgTC47IGNpdGF0aW9uX2F1dGhvcj1DYW1hcmdvICBBLjsgY2l0YXRpb25fYXV0aG9yPVBvY2hvbGxlICBELjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NTc7ICBjaXRhdGlvbl9wYWdlcz00MjYtMzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBiYXNpYyBoZWxpeC1sb29wLWhlbGl4IHRyYW5zY3JpcHRpb24gZmFjdG9yIGZhbWlseSBpbiBwbGFudHM6IGEgZ2Vub21lLXdpZGUgc3R1ZHkgb2YgcHJvdGVpbiBzdHJ1Y3R1cmUgYW5kIGZ1bmN0aW9uYWwgZGl2ZXJzaXR5OyBjaXRhdGlvbl9hdXRob3I9SGVpbSAgTS5BLjsgY2l0YXRpb25fYXV0aG9yPUpha29ieSAgTS47IGNpdGF0aW9uX2F1dGhvcj1XZXJiZXIgIE0uOyBjaXRhdGlvbl9hdXRob3I9TWFydGluICBDLjsgY2l0YXRpb25fYXV0aG9yPVdlaXNzaGFhciAgQi47IGNpdGF0aW9uX2F1dGhvcj1CYWlsZXkgIFAuQy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz03MzUtNDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVyZ2VuaWMgdHJhbnNjcmlwdGlvbiBieSBSTkEgcG9seW1lcmFzZSBJSSBjb29yZGluYXRlcyBQb2wgSVYgYW5kIFBvbCBWIGluIHNpUk5BLWRpcmVjdGVkIHRyYW5zY3JpcHRpb25hbCBnZW5lIHNpbGVuY2luZyBpbiBBcmFiaWRvcHNpczsgY2l0YXRpb25fYXV0aG9yPVpoZW5nICBCLjsgY2l0YXRpb25fYXV0aG9yPVdhbmcgIFouOyBjaXRhdGlvbl9hdXRob3I9TGkgIFMuOyBjaXRhdGlvbl9hdXRob3I9WXUgIEIuOyBjaXRhdGlvbl9hdXRob3I9TGl1ICBKLi1ZLjsgY2l0YXRpb25fYXV0aG9yPUNoZW4gIFguOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5lcyBEZXYuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9Mjg1MC02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgZ2VuZXRpYyBtYXJrZXIgZGlzY292ZXJ5IGFuZCBnZW5vdHlwaW5nIHVzaW5nIG5leHQtZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nOyBjaXRhdGlvbl9hdXRob3I9RGF2ZXkgIEouVy47IGNpdGF0aW9uX2F1dGhvcj1Ib2hlbmxvaGUgIFAuQS47IGNpdGF0aW9uX2F1dGhvcj1FdHRlciAgUC5ELjsgY2l0YXRpb25fYXV0aG9yPUJvb25lICBKLlEuOyBjaXRhdGlvbl9hdXRob3I9Q2F0Y2hlbiAgSi5NLjsgY2l0YXRpb25fYXV0aG9yPUJsYXh0ZXIgIE0uTC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz00OTktNTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBjaGFyYWN0ZXJpemF0aW9uIGFuZCBsaW5rYWdlIG1hcHBpbmcgb2Ygc2ltcGxlIHNlcXVlbmNlIHJlcGVhdHMgaW4gTWVpIChQcnVudXMgbXVtZSBTaWViLiBldCBadWNjLik7IGNpdGF0aW9uX2F1dGhvcj1TdW4gIEwuOyBjaXRhdGlvbl9hdXRob3I9WWFuZyAgVy47IGNpdGF0aW9uX2F1dGhvcj1aaGFuZyAgUS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lNTk1NjIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIG9mIFBydW51cyBtdW1lOyBjaXRhdGlvbl9hdXRob3I9WmhhbmcgIFEuWC47IGNpdGF0aW9uX2F1dGhvcj1DaGVuICBXLkIuOyBjaXRhdGlvbl9hdXRob3I9U3VuICBMLkQuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIENvbW11bi47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTEzMTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcGVhY2ggZ2Vub21lOyBjaXRhdGlvbl9hdXRob3I9QXLDunMgIFAuOyBjaXRhdGlvbl9hdXRob3I9VmVyZGUgIEkuOyBjaXRhdGlvbl9hdXRob3I9U29zaW5za2kgIEIuOyBjaXRhdGlvbl9hdXRob3I9WmhlYmVudHlheWV2YSAgVC47IGNpdGF0aW9uX2F1dGhvcj1BYmJvdHQgIEEuRy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZWUgR2VuZXQuIEdlbm9tLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9NTMxLTQ3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgbWFwcGluZyBvZiBzZWdyZWdhdGlvbiBkaXN0b3J0aW9uIGxvY2kgaW4gQWVnaWxvcHMgdGF1c2NoaWk7IGNpdGF0aW9uX2F1dGhvcj1GYXJpcyAgSi47IGNpdGF0aW9uX2F1dGhvcj1MYWRkb21hZGEgIEIuOyBjaXRhdGlvbl9hdXRob3I9R2lsbCAgQi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTE0OTsgIGNpdGF0aW9uX3BhZ2VzPTMxOS0yNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFwcGluZyBzZWdyZWdhdGlvbiBkaXN0b3J0aW9uIGxvY2kgYW5kIHF1YW50aXRhdGl2ZSB0cmFpdCBsb2NpIGZvciBzcGlrZWxldCBzdGVyaWxpdHkgaW4gcmljZSAoT3J5emEgc2F0aXZhIEwuKTsgY2l0YXRpb25fYXV0aG9yPVdhbmcgIEMuOyBjaXRhdGlvbl9hdXRob3I9Wmh1ICBDLjsgY2l0YXRpb25fYXV0aG9yPVpoYWkgIEguOyBjaXRhdGlvbl9hdXRob3I9V2FuICBKLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXQuIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9ODY7ICBjaXRhdGlvbl9wYWdlcz05Ny0xMDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVmZmVjdHMgb2YgbWlzc2luZyBtYXJrZXIgYW5kIHNlZ3JlZ2F0aW9uIGRpc3RvcnRpb24gb24gUVRMIG1hcHBpbmcgaW4gRjIgcG9wdWxhdGlvbnM7IGNpdGF0aW9uX2F1dGhvcj1aaGFuZyAgTC47IGNpdGF0aW9uX2F1dGhvcj1XYW5nICBTLjsgY2l0YXRpb25fYXV0aG9yPUxpICBILjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjE7ICBjaXRhdGlvbl9wYWdlcz0xMDcxLTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1RdWFudGl0YXRpdmUgdHJhaXQgbG9jdXMgbWFwcGluZyBjYW4gYmVuZWZpdCBmcm9tIHNlZ3JlZ2F0aW9uIGRpc3RvcnRpb247IGNpdGF0aW9uX2F1dGhvcj1YdSAgUy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE4MDsgIGNpdGF0aW9uX3BhZ2VzPTIyMDEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGVhY2ggYnJlZWRpbmcgZm9yIGdyb3d0aCBoYWJpdDsgY2l0YXRpb25fYXV0aG9yPUJhc3NpICBELjsgY2l0YXRpb25fYXV0aG9yPVJpenpvICBNLjsgY2l0YXRpb25fYXV0aG9yPUdlaWJlbCAgTS47IGNpdGF0aW9uX2F1dGhvcj1GaXNjaGVyICBNLjsgY2l0YXRpb25fYXV0aG9yPUZpc2NoZXIgIEMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BY3RhIEhvcnRpYy47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTQxMS00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIHN0dWR5IG9mIHRoZSB3ZWVwaW5nIGhhYml0IGluIHBlYWNoOyBjaXRhdGlvbl9hdXRob3I9TW9uZXQgIFIuOyBjaXRhdGlvbl9hdXRob3I9QmFzdGFyZCAgWS47IGNpdGF0aW9uX2F1dGhvcj1HaWJhdWx0ICBCLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QWdyb25vbWllOyAgY2l0YXRpb25feWVhcj0xOTg4OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0xMjctMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgaW50ZXJhY3Rpb25zIG9mIHBpbGxhciAoY29sdW1uYXIpLCBjb21wYWN0LCBhbmQgZHdhcmYgcGVhY2ggdHJlZSBnZW5vdHlwZXM7IGNpdGF0aW9uX2F1dGhvcj1TY29yemEgIFIuOyBjaXRhdGlvbl9hdXRob3I9QmFzc2kgIEQuOyBjaXRhdGlvbl9hdXRob3I9TGl2ZXJhbmkgIEEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBBbS4gU29jLiBIb3J0aWMuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTI3OyAgY2l0YXRpb25fcGFnZXM9MjU0LTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jbmhlcml0YW5jZSBvZiBzb21lIGNoYXJhY3RlcmlzdGljcyBhbmQgYnJlZWRpbmcgb2YgbmV3IGh5YnJpZHMgaW4gZmxvd2VyaW5nIHBlYWNoZXM7IGNpdGF0aW9uX2F1dGhvcj1ZYW1hemFraSAgSy47IGNpdGF0aW9uX2F1dGhvcj1Pa2FiZSAgTS47IGNpdGF0aW9uX2F1dGhvcj1UYWthaGFzaGkgIEUuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CdWxsLiBLYW5hZ2F3YSBIb3J0LiBFeHAuIFN0YS4gKEphcGFuKTsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIGNpdGF0aW9uX3BhZ2VzPTQ2LTUzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgY2hhcmFjdGVyIGRpc3BsYXkgb2YgQ2FzdGFuZWEgbW9sbGlzc2ltYSB2YXIuIHBlbmR1bGEgU2V4dWFsIFByb2dlbmllczsgY2l0YXRpb25fYXV0aG9yPVR1ICBTLjsgY2l0YXRpb25fYXV0aG9yPVFpdSAgRi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFjdGEgSGlydGljdW90dXJhZSBTaW5pY2E7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz03NjAtMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyaXNvbiBvZiBtZWNoYW5pY2FsIHByb3BlcnRpZXMgb2YgdGVuc2lvbiBhbmQgb3Bwb3NpdGUgd29vZCBpbiBQb3B1bHVzOyBjaXRhdGlvbl9hdXRob3I9Q291dGFuZCAgQy47IGNpdGF0aW9uX2F1dGhvcj1KZXJvbmltaWRpcyAgRy47IGNpdGF0aW9uX2F1dGhvcj1DaGFuc29uICBCLjsgY2l0YXRpb25fYXV0aG9yPUxvdXAgIEMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Xb29kIFNjaS4gVGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz0xMS0yNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIyLzMvMTgzLzE1OTI2NzkvZHN2MDAzLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBIaWdoLWRlbnNpdHkgZ2VuZXRpYyBtYXAgaXMgYSB2YWx1YWJsZSB0b29sIGZvciBmaW5lIG1hcHBpbmcgbG9jdXMgY29udHJvbGxpbmcgYSBzcGVjaWZpYyB0cmFpdCBlc3BlY2lhbGx5IGZvciBwZXJlbm5pYWwgd29vZHkgcGxhbnRzLiBJbiB0aGlzIHN0dWR5LCIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NTkxMDE3JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZDaB2ohqVfOU51CwcxK4QZinFADn" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f3%2f183%2f591017" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f3%2f183%2f591017" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=23905" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/3/1/m_cover.gif?Expires=2147483647&amp;Signature=o~ckdHMyoALozjazxFdJARrSll6HI0qZKNez1rp60DiaVqyzX7KX3gffSJBpNovfc7x0FBbw5M~q-oXkBI0BrEKD0pgvQT5eVNC4LZt8HYW1wCNn7TYsIJ~iamj5~PWBqwfQdRx7Lwp~5y0cfA6QZfzXvHT-c2y8KfWBVWCVg9vuL7VY9lDugsDIpyrHTWTMWH4KR~-txVxPM4XBFIVE2oVLkX6QYMzija1YcuQl4yisR1R~qhzzHGrjVvva6IGLptvPDIFfjhAoXGwaWNy8qGaYVTkt5JCmjQmDV1APqpGYq6f9-3DKz0Py3DzfqJ6xp8-SqVKAnV4V9jMcimV1eA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/3">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="9624223">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9624223">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="9624225">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9624225">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="9624230">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9624230">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="9624247">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9624247">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="9624275">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9624275">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="9624290">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9624290">Conflict of Interest statement</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="9624292">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9624292">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="9624294">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9624294">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="9624296">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9624296">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="9624298">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9624298">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/22/3/193/591209">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">High-density genetic map construction and identification of a locus controlling weeping trait in an ornamental woody plant (<em>Prunus mume</em> Sieb. et Zucc)
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jie Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Beijing Key Laboratory of Ornamental Plants Germplasm Innovation and Molecular Breeding, National Engineering Research Center for Floriculture</div>, <div class="insititution">Beijing Laboratory of Urban and Rural Ecological Environment, College of Landscape Architecture, Beijing Forestry University</div>, <div class="addr-line">Beijing 100083</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jie+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhang J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jie Zhang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Qixiang Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Beijing Key Laboratory of Ornamental Plants Germplasm Innovation and Molecular Breeding, National Engineering Research Center for Floriculture</div>, <div class="insititution">Beijing Laboratory of Urban and Rural Ecological Environment, College of Landscape Architecture, Beijing Forestry University</div>, <div class="addr-line">Beijing 100083</div>, <div class="country">China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86 010-62336321. Fax. +86 010-62336321. E-mail: <a href="mailto:zqxbjfu@126.com" target="_blank">zqxbjfu@126.com</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Qixiang+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang Q">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhang Q&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Qixiang Zhang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tangren Cheng</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Beijing Key Laboratory of Ornamental Plants Germplasm Innovation and Molecular Breeding, National Engineering Research Center for Floriculture</div>, <div class="insititution">Beijing Laboratory of Urban and Rural Ecological Environment, College of Landscape Architecture, Beijing Forestry University</div>, <div class="addr-line">Beijing 100083</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tangren+Cheng">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cheng T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Cheng T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tangren Cheng</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Weiru Yang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Beijing Key Laboratory of Ornamental Plants Germplasm Innovation and Molecular Breeding, National Engineering Research Center for Floriculture</div>, <div class="insititution">Beijing Laboratory of Urban and Rural Ecological Environment, College of Landscape Architecture, Beijing Forestry University</div>, <div class="addr-line">Beijing 100083</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Weiru+Yang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yang W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yang W&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Weiru Yang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Huitang Pan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Beijing Key Laboratory of Ornamental Plants Germplasm Innovation and Molecular Breeding, National Engineering Research Center for Floriculture</div>, <div class="insititution">Beijing Laboratory of Urban and Rural Ecological Environment, College of Landscape Architecture, Beijing Forestry University</div>, <div class="addr-line">Beijing 100083</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Huitang+Pan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pan H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Pan H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Huitang Pan</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Junjun Zhong</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Beijing Key Laboratory of Ornamental Plants Germplasm Innovation and Molecular Breeding, National Engineering Research Center for Floriculture</div>, <div class="insititution">Beijing Laboratory of Urban and Rural Ecological Environment, College of Landscape Architecture, Beijing Forestry University</div>, <div class="addr-line">Beijing 100083</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Junjun+Zhong">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhong J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhong J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Junjun Zhong</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Long Huang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Biomarker Technologies Corporation</div>, <div class="addr-line">Beijing</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Long+Huang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Huang L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Huang L&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Long Huang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Enze Liu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Biomarker Technologies Corporation</div>, <div class="addr-line">Beijing</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Enze+Liu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liu E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Liu E&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Enze Liu</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="footnote-content">Edited by Dr Satoshi Tabata</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 3, 1 June 2015, Pages 183–191, <a href='https://doi.org/10.1093/dnares/dsv003'>https://doi.org/10.1093/dnares/dsv003</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">15 March 2015</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">12 December 2014</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">21 February 2015</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="591017" href="/dnaresearch/article-pdf/22/3/183/1592679/dsv003.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Jie Zhang, Qixiang Zhang, Tangren Cheng, Weiru Yang, Huitang Pan, Junjun Zhong, Long Huang, Enze Liu;  High-density genetic map construction and identification of a locus controlling weeping trait in an ornamental woody plant (<em>Prunus mume</em> Sieb. et Zucc), <em>DNA Research</em>, Volume 22, Issue 3, 1 June 2015, Pages 183–191, <a href='https://doi.org/10.1093/dnares/dsv003'>https://doi.org/10.1093/dnares/dsv003</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/591017?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/591017?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/591017?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/591017?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/591017?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/591017?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/591017?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=High-density%20genetic%20map%20construction%20and%20identification%20of%20a%20locus%20controlling%20weeping%20trait%20in%20an%20ornamental%20woody%20plant%20(%26lt%3Bem%26gt%3BPrunus%20mume%26lt%3B%2Fem%26gt%3B%20Sieb.%20et%20Zucc)&amp;publicationDate=2015-03-15&amp;volumeNum=22&amp;issueNum=3&amp;author=Zhang%2C%20Jie%3B%20Zhang%2C%20Qixiang&amp;startPage=183&amp;endPage=191&amp;contentId=10.1093%2Fdnares%2Fdsv003&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f3%2f183%2f591017" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=9624223 id="9624223" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>High-density genetic map is a valuable tool for fine mapping locus controlling a specific trait especially for perennial woody plants. In this study, we firstly constructed a high-density genetic map of mei (<em>Prunus mume</em>) using SLAF markers, developed by specific locus amplified fragment sequencing (SLAF-seq). The linkage map contains 8,007 markers, with a mean marker distance of 0.195 cM, making it the densest genetic map for the genus <em>Prunus</em>. Though weeping trees are used worldwide as landscape plants, little is known about weeping controlling gene(s) (<em>Pl</em>). To test the utility of the high-density genetic map, we did fine-scale mapping of this important ornamental trait. In total, three statistic methods were performed progressively based on the result of inheritance analysis. Quantitative trait loci (QTL) analysis initially revealed that a locus on linkage group 7 was strongly responsible for weeping trait. Mutmap-like strategy and extreme linkage analysis were then applied to fine map this locus within 1.14 cM. Bioinformatics analysis of the locus identified some candidate genes. The successful localization of weeping trait strongly indicates that the high-density map constructed using SLAF markers is a worthy reference for mapping important traits for woody plants.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="mei">mei</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;high density&quot;">high density</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;linkage mapping&quot;">linkage mapping</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="SLAF-seq">SLAF-seq</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;weeping trait&quot;">weeping trait</a></div>                            <h2 scrollto-destination=9624225 id="9624225" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>High-density linkage maps are exceptionally valuable tools in many genetic and genomic applications, especially for fine-scale mapping and map-based cloning of trait-controlled genes. A key prerequisite for constructing a high-density linkage map is the availability of plenty genetic markers, which has been feasible due to the advances in next-generation sequencing (NGS) technology for large-scale marker discovery.</p><p>Reduced representation genome sequencing (RRGS) is a rapid and cost-effective strategy for massive SNP discovery and genotyping.<sup><a reveal-id="DSV003C1  DSV003C4" data-open="DSV003C1  DSV003C4" class="link link-ref link-reveal xref-bibr">1–4</a></sup> Various methods of RRGS have been developed along with the spread of NGS technology.<sup><a reveal-id="DSV003C5  DSV003C8" data-open="DSV003C5  DSV003C8" class="link link-ref link-reveal xref-bibr">5–8</a></sup> SLAF-seq is a recently developed approach based on RRGS and high-throughput pair-end sequencing. It has been applied in many species for genetic map construction.<sup><a reveal-id="DSV003C9  DSV003C11" data-open="DSV003C9  DSV003C11" class="link link-ref link-reveal xref-bibr">9–11</a></sup> With barcode multiplexed sequencing, large populations with massive loci can be genotyped simultaneously. Besides, a pre-designed scheme can optimize marker efficiency, and stringent genotyping procedure can ensure the accuracy of genotypes.</p><p>Mei (Japanese apricot) is a popular ornamental plant widely cultivated in the entire East Asia and, as an important member of genus <em>Prunus</em>, plays a pivotal role in systematic studies of the Rosaceae.<sup><a reveal-id="DSV003C12" data-open="DSV003C12" class="link link-ref link-reveal xref-bibr">12</a></sup> It owned many prominent ornamental features, such as colourful corollas, pleasant fragrance and weeping habit.<sup><a reveal-id="DSV003C13" data-open="DSV003C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Weeping trees characterized by soft, pendant branches are used worldwide as landscape plants due to their graceful and intriguing forms. Meanwhile, being a specific tree architecture, weeping trait has received increasing attention during the past years.<sup><a reveal-id="DSV003C14  DSV003C16" data-open="DSV003C14  DSV003C16" class="link link-ref link-reveal xref-bibr">14–16</a></sup> In the peach, a set of random amplified polymorphic DNA (RAPD) markers were developed in an F2 population, and the <em>Pl</em> was located preliminarily on the second linkage group at comparatively large distances (11.4 and 17.2 cM) between two RAPD markers.<sup><a reveal-id="DSV003C17" data-open="DSV003C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Subsequently, the simple sequence repeat (SSR) marker CPPCT029 was found to co-segregate with <em>Pl</em> in peach.<sup><a reveal-id="DSV003C18" data-open="DSV003C18" class="link link-ref link-reveal xref-bibr">18</a></sup> However, RAPD markers are difficult to transfer to other mapping population. Besides, the located region of <em>Pl</em> was just an estimate and no candidate genes had been detected.</p><p>In the current study, a high-density linkage map of mei was constructed by applying SLAF-seq method and HighMap linkage mapping strategy for the first time. To test the utility of this map, fine-scale mapping of weeping trait was conducted. Gene annotation results suggest the accuracy of the localization. It is for the first time that weeping trait was fine-mapped, which gives a typical example for further QTL fine mapping of important traits in perennial woody plants.</p>                            <h2 scrollto-destination=9624230 id="9624230" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=9624231 id="9624231" class="section-title" data-legacy-id=s2a>2.1. Mapping population</h3>
<p>The pedigree used for QTL mapping was generated by the female ‘LiuBan’ (upright type) from QingDao, China (36.20169°N, 120.4162°E) and the male ‘FenTaiChuiZhi’ (weeping type) from WuHan, China (30.54526°N, 114.39511°E), which were selected based on their phenotypic divergence in tree architecture, flower shape and colourful corollas (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). In spring 2013, a clonally replicated plantation of this entire pedigree was established in a randomized complete block design with three replicates and two tree plots at Hangzhou, China (30.566389°N, 119.879582°E).</p>                            <h3 scrollto-destination=9624233 id="9624233" class="section-title" data-legacy-id=s2b>2.2. SLAF library construction and high-throughput sequencing</h3>
<p>An improved SLAF-seq strategy was utilized in our experiment. Firstly, reference genome of mei was used to design marker discovery experiments by simulating <em>in silico</em> the number of markers produced by different enzymes. Next, a SLAF pilot experiment was performed, and the SLAF library was conducted in accordance using the pre-designed scheme. For the mei F1 population, two enzymes (HaeIII and Hpy166II, New England Biolabs, NEB, USA) were used to digest the genomic DNA. A single nucleotide (A) overhang was added subsequently to the digested fragments using Klenow Fragment (3′ → 5′ exo<sup>−</sup>) (NEB) and dATP at 37°C. Duplex tag-labelled sequencing adapters (PAGE-purified, Life Technologies, USA) were then ligated to the A-tailed fragments using T4 DNA ligase. Polymerase chain reaction (PCR) was performed using diluted restriction-ligation DNA samples, dNTP, Q5<sup>®</sup> High-Fidelity DNA Polymerase and PCR primers (Forward primer: 5′-AATGATACGGCGACCACCGA-3′, reverse primer: 5′-CAAGCAGAAGACGGCATACG-3′) (PAGE-purified, Life Technologies). PCR products were then purified using Agencourt AMPure XP beads (Beckman Coulter, High Wycombe, UK) and pooled. Pooled samples were separated by 2% agarose gel electrophoresis. Fragments ranging from 214 to 294 base pairs (with indexes and adaptors) in size were excised and purified using a QIAquick gel extraction kit (Qiagen, Hilden, Germany). Gel-purified products were then diluted. And pair-end sequencing (each end 100 bp) was performed on an Illumina HiSeq 2500 system (Illumina, Inc., San Diego, CA, USA) according to the manufacturer's recommendations.</p>                            <h3 scrollto-destination=9624235 id="9624235" class="section-title" data-legacy-id=s2c>2.3. Sequence data grouping and genotyping</h3>
<p>SLAF marker identification and genotyping were performed using procedures described by Sun <em>et al.</em><sup><a reveal-id="DSV003C10" data-open="DSV003C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Briefly, low-quality reads (quality score &lt;20e) were filtered out and then raw reads were sorted to each progeny according to duplex barcode sequences. After the barcodes and the terminal 5-bp positions were trimmed from each high-quality reads, clean reads from the same sample were mapped onto the mei genome sequence using SOAP software.<sup><a reveal-id="DSV003C19" data-open="DSV003C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Sequences mapping to the same position with over 95% identity were defined as one SLAF locus.<sup><a reveal-id="DSV003C11" data-open="DSV003C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Allele tags of each SLAF locus were then detected according to parental reads with sequence depth &gt;20-fold and &gt;30% integrity in the offspring. Single nucleotide polymorphisms (SNPs) were detected according to the allele tags and SLAFs with &gt;3 SNPs were filtered out firstly. For diploid species, one SLAF locus can contain no more than four allele tags, so SLAF loci with more than four alleles were defined as repetitive SLAFs and discarded subsequently. Only SLAFs with two to four alleles were identified as polymorphic and considered potential markers. All polymorphism SLAFs loci were genotyped with consistency in the parental and offspring SNP loci. The marker code of the polymorphic SLAFs was analysed according to the population type CP (cross pollinators), which consist of five segregation types (ab × cd, ef × eg, hk × hk, lm × ll, nn × np). The segregation type ‘abxcd’ represents that the two alleles of one marker are different in both parents.</p><p>Genotype scoring was then performed using a Bayesian approach to further ensure the genotyping quality.<sup><a reveal-id="DSV003C10" data-open="DSV003C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Briefly, a posteriori conditional probability was calculated using the coverage of each allele and the number of SNPs. Then, genotyping quality score translated from the probability was used to select qualified markers for subsequent analysis, which was a dynamic optimization process.<sup><a reveal-id="DSV003C10" data-open="DSV003C10" class="link link-ref link-reveal xref-bibr">10</a></sup> When the average genotype quality scores of all SLAF markers reached the cut-off value, we ceased this process. After that, high-quality SLAF markers for the genetic mapping were filtered by the following criteria. First, average sequence depths should be &gt;7-fold in each progeny and &gt;80-fold in the parents. Second, markers with &gt;30% missing data were filtered. Third, the <em>χ</em><sup>2</sup> test was performed to examine the segregation distortion. Markers with significant segregation distortion were initially excluded from the map construction and were then added later as accessory markers.</p>                            <h3 scrollto-destination=9624238 id="9624238" class="section-title" data-legacy-id=s2d>2.4. Linkage map construction</h3>
<p>Marker loci were partitioned primarily into linkage groups (LGs) based on their locations on mei genome. Next, the modified logarithm of odds (MLOD) scores between markers were calculated to further confirm the robustness of markers for each LGs. Markers with MLOD scores &lt;5 were filtered prior to ordering. To ensure efficient construction of the high-density and high-quality map, a newly developed HighMap strategy was utilized to order the SLAF markers and correct genotyping errors within LGs.<sup><a reveal-id="DSV003C20" data-open="DSV003C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Firstly, recombinant frequencies and LOD scores were calculated by two-point analysis, which were applied to infer linkage phases. Then, enhanced gibbs sampling, spatial sampling and simulated annealing algorithms (GSS) were combined to conduct an iterative process of marker ordering.<sup><a reveal-id="DSV003C21" data-open="DSV003C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSV003C22" data-open="DSV003C22" class="link link-ref link-reveal xref-bibr">22</a></sup> Briefly, in the first stage of the ordering procedure, SLAF markers were selected using spatial sampling. One marker was taken randomly in a priority order of pseudo-testcross, and markers with a recombination frequency smaller than a given sampling radius are excluded from the marker set. Subsequently, simulated annealing was applied to searching for the best map order. Summation of adjacent recombination fractions was calculated as illustrated by Liu <em>et al.</em><sup><a reveal-id="DSV003C20" data-open="DSV003C20" class="link link-ref link-reveal xref-bibr">20</a></sup> The annealing system continued until, in a number of successive steps, the newly generated map order is rejected. Blocked Gibbs sampling was employed to estimate multipoint recombination frequencies of the parents after the optimal map order of sample markers was obtained.<sup><a reveal-id="DSV003C20" data-open="DSV003C20" class="link link-ref link-reveal xref-bibr">20</a></sup> The updated recombination frequencies were used to integrate the two parental maps, which optimize the map order in the next cycle of simulated annealing. Once a stable map order was obtained after 3–4 cycles, we turned to the next map building round. A subset of currently unmapped markers was selected and added to the previous sample with decreased sample radius. The mapping algorithm repeats until all the markers were mapped appropriately. The error correction strategy of SMOOTH was then conducted according to parental contribution of genotypes,<sup><a reveal-id="DSV003C23" data-open="DSV003C23" class="link link-ref link-reveal xref-bibr">23</a></sup> and a k-nearest neighbor algorithm was applied to impute missing genotypes.<sup><a reveal-id="DSV003C24" data-open="DSV003C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Skewed markers were then added into this map by applying a multipoint method of maximum likelihood.<sup><a reveal-id="DSV003C25" data-open="DSV003C25" class="link link-ref link-reveal xref-bibr">25</a></sup></p><p>The sex-specific maps were constructed using markers that were heterozygous in female or male parent, while the consensus map was established by integrating the parental maps through the anchor markers (markers that were heterozygous in both parents).<sup><a reveal-id="DSV003C22" data-open="DSV003C22" class="link link-ref link-reveal xref-bibr">22</a></sup> For anchor markers, map distances were calculated as the averages over the two parental distances. The remaining markers segregating in only one of the parents were placed on the consensus map by interpolation or extrapolation according to the relative position between the flanking anchor markers on the relevant parental map. Map distances were then estimated using the Kosambi mapping function.<sup><a reveal-id="DSV003C26" data-open="DSV003C26" class="link link-ref link-reveal xref-bibr">26</a></sup></p>                            <h3 scrollto-destination=9624241 id="9624241" class="section-title" data-legacy-id=s2e>2.5. Marker-trait association analysis</h3>
<p>Firstly, QTL analysis was performed using the R/QTL package.<sup><a reveal-id="DSV003C27" data-open="DSV003C27" class="link link-ref link-reveal xref-bibr">27</a></sup> A binary option was used to detect the weeping QTL. The significance thresholds were determined using 1,000 permutations. Results from the interval mapping analysis were used to construct the QTLs, and their positions were used in a default model.</p><p>For the Mutmap-like strategy, high-quality SLAF markers from each individual were first divided into two pools, weeping and upright, according to the phenotypic segregation of the F1 population. To examine the relative genotype contribution from each parent, the difference in SLAF allele frequencies between the weeping and upright pools was evaluated using SNP-index-like methods<sup><a reveal-id="DSV003C28" data-open="DSV003C28" class="link link-ref link-reveal xref-bibr">28</a></sup> and, hereafter, will be referred to as the SLAF-index. The difference in proportions [Δ(SLAF-index)] of one genotype in the weeping pool was compared with the upright pool. The proportion of one genotype derived from the male parent (M<sub>g</sub>) corresponding to the two parental genotypes (N<sub>g</sub>) was defined as the SLAF-index. If the proportion of M<sub>g</sub> in N<sub>g</sub> was equal in the two pools, then the Δ(SLAF-index) was equal to 0. A Δ(SLAF-index) equal to 1 suggests that one genotype almost entirely belongs to the weeping pool, indicating that the associate SLAF markers are linked closely to the weeping trait. Accordingly, the Δ(SLAF-index) was calculated for all high-quality SLAF markers between parents, which was then used as an important indicator of the relationship between marker and weeping trait.</p><p>Then, the weeping trait was considered a marker (marker0), with parents' genotype being nnxnp. Accordingly, we got the genotype of each individual based on their phenotype (‘weeping’ is ‘nn’, while ‘upright’ is ‘np’). Recombination rates between marker 0 and markers of other genotypes were then calculated.</p>                            <h3 scrollto-destination=9624245 id="9624245" class="section-title" data-legacy-id=s2f>2.6. Candidate gene selection and annotation</h3>
<p>We initially detected the genes in the causal region according to the functional annotation of the mei genome (<a class="link link-uri" href="http://prunusmumegenome.bjfu.edu.cn">http://prunusmumegenome.bjfu.edu.cn</a>). The gene annotation list was then screened for polymorphism at the amino acid level between the resequenced ‘LiuBan’ genome and ‘FenTaiChuiZhi’ genome (unpublished data). Genes showing at least one change at the amino acid level were considered candidates. For a more detailed functional annotation, the candidates were compared with the NR protein sequences available at UniProt database using the BLASTX algorithm, with an <em>E</em>-value threshold of 10<sup>−01</sup>. The associated hits were then searched for their respective Gene Ontology (GO) terms at <a class="link link-uri" href="http://www.geneontology.org">www.geneontology.org</a>.<sup><a reveal-id="DSV003C29" data-open="DSV003C29" class="link link-ref link-reveal xref-bibr">29</a></sup></p>                            <h2 scrollto-destination=9624247 id="9624247" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=9624248 id="9624248" class="section-title" data-legacy-id=s3a>3.1. SLAF sequencing and genotyping</h3>
<p>Based on the results of SLAF pilot experiment, HaeIII and Hpy166II were used for SLAF libraries construction. In total, 472.66 M pair-end reads were obtained for both parents and 387 progeny, with each descendant sharing an average of 1.11 M reads. The raw data of SLAF-seq have all been deposited in NCBI SRA database under project accession PRJNA273338. Among them, the sequence depth of parents had more impact on alleles calling, since they were the basis for genotyping for each marker. The parents obtained 43.09 M reads, a rate of 9.31%, which was higher than average. A total of 230,584 SLAF loci were detected after reads clustering. The average sequence depths of these SLAFs were 35.15-fold for parents and 4.00-fold for each individual. The extreme depths of the locus-specific sequences suggested the accuracy of the SLAF marker discovery. Repetitive SLAFs were then disregarded, and polymorphic SLAFs accounted for 40.35% of all SLAFs (Table <a reveal-id="DSV003TB1" data-open="DSV003TB1" class="link link-reveal link-table xref-fig">1</a>). All polymorphic SLAFs were then genotyped separately for both parents and all individuals. </p>                        <a id="9624250" scrollto-destination="9624250"></a>
<div content-id="DSV003TB1" class="table-modal"><div class="table-wrap"><div id="DSV003TB1" class="table-wrap-title" data-id="DSV003TB1"><div class="label -label">Table 1.</div><div class="caption"><p>SLAF-seq data summary for mei F1 population</p></div> </div><div class="table-overflow"><table><tbody><tr><td colspan="2">Total reads </td></tr><tr><td> No. of reads </td><td>472,655,544.00 </td></tr><tr><td> Reads in high-quality SLAFs </td><td>183,454,384.00 </td></tr><tr><td> Reads in repeat SLAFs </td><td>13,619,794.00 </td></tr><tr><td> Reads in low depth SLAFs </td><td>8,778,576.00 </td></tr><tr><td colspan="2">High-quality SLAFs </td></tr><tr><td> No. of SLAFs </td><td>230,584.00 </td></tr><tr><td> Average SLAF depth </td><td>795.61 </td></tr><tr><td> Average depth in parents </td><td>35.13 </td></tr><tr><td> Average depth in individuals </td><td>4.00 </td></tr><tr><td colspan="2">Polymorphic SLAFs </td></tr><tr><td> No. of polymorphic SLAFs </td><td>93,031.00 </td></tr><tr><td> Average depth in parents </td><td>127.31 </td></tr><tr><td> Average depth in individuals </td><td>7.85 </td></tr><tr><td> No. of SNPs </td><td>14,388.00 </td></tr><tr><td> SNP ratio per kb </td><td>6.02 </td></tr><tr><td colspan="2">High-quality SLAF markers </td></tr><tr><td> No. of high-quality SLAF markers </td><td>9,412 </td></tr></tbody></table></div><div content-id="DSV003TB1" class="table-modal"><table><tbody><tr><td colspan="2">Total reads </td></tr><tr><td> No. of reads </td><td>472,655,544.00 </td></tr><tr><td> Reads in high-quality SLAFs </td><td>183,454,384.00 </td></tr><tr><td> Reads in repeat SLAFs </td><td>13,619,794.00 </td></tr><tr><td> Reads in low depth SLAFs </td><td>8,778,576.00 </td></tr><tr><td colspan="2">High-quality SLAFs </td></tr><tr><td> No. of SLAFs </td><td>230,584.00 </td></tr><tr><td> Average SLAF depth </td><td>795.61 </td></tr><tr><td> Average depth in parents </td><td>35.13 </td></tr><tr><td> Average depth in individuals </td><td>4.00 </td></tr><tr><td colspan="2">Polymorphic SLAFs </td></tr><tr><td> No. of polymorphic SLAFs </td><td>93,031.00 </td></tr><tr><td> Average depth in parents </td><td>127.31 </td></tr><tr><td> Average depth in individuals </td><td>7.85 </td></tr><tr><td> No. of SNPs </td><td>14,388.00 </td></tr><tr><td> SNP ratio per kb </td><td>6.02 </td></tr><tr><td colspan="2">High-quality SLAF markers </td></tr><tr><td> No. of high-quality SLAF markers </td><td>9,412 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><p><strong>N</strong>o: number; SLAF: specific length amplified fragment.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/9624250" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV003TB1" class="table-wrap-title" data-id="DSV003TB1"><div class="label -label">Table 1.</div><div class="caption"><p>SLAF-seq data summary for mei F1 population</p></div> </div><div class="table-overflow"><table><tbody><tr><td colspan="2">Total reads </td></tr><tr><td> No. of reads </td><td>472,655,544.00 </td></tr><tr><td> Reads in high-quality SLAFs </td><td>183,454,384.00 </td></tr><tr><td> Reads in repeat SLAFs </td><td>13,619,794.00 </td></tr><tr><td> Reads in low depth SLAFs </td><td>8,778,576.00 </td></tr><tr><td colspan="2">High-quality SLAFs </td></tr><tr><td> No. of SLAFs </td><td>230,584.00 </td></tr><tr><td> Average SLAF depth </td><td>795.61 </td></tr><tr><td> Average depth in parents </td><td>35.13 </td></tr><tr><td> Average depth in individuals </td><td>4.00 </td></tr><tr><td colspan="2">Polymorphic SLAFs </td></tr><tr><td> No. of polymorphic SLAFs </td><td>93,031.00 </td></tr><tr><td> Average depth in parents </td><td>127.31 </td></tr><tr><td> Average depth in individuals </td><td>7.85 </td></tr><tr><td> No. of SNPs </td><td>14,388.00 </td></tr><tr><td> SNP ratio per kb </td><td>6.02 </td></tr><tr><td colspan="2">High-quality SLAF markers </td></tr><tr><td> No. of high-quality SLAF markers </td><td>9,412 </td></tr></tbody></table></div><div content-id="DSV003TB1" class="table-modal"><table><tbody><tr><td colspan="2">Total reads </td></tr><tr><td> No. of reads </td><td>472,655,544.00 </td></tr><tr><td> Reads in high-quality SLAFs </td><td>183,454,384.00 </td></tr><tr><td> Reads in repeat SLAFs </td><td>13,619,794.00 </td></tr><tr><td> Reads in low depth SLAFs </td><td>8,778,576.00 </td></tr><tr><td colspan="2">High-quality SLAFs </td></tr><tr><td> No. of SLAFs </td><td>230,584.00 </td></tr><tr><td> Average SLAF depth </td><td>795.61 </td></tr><tr><td> Average depth in parents </td><td>35.13 </td></tr><tr><td> Average depth in individuals </td><td>4.00 </td></tr><tr><td colspan="2">Polymorphic SLAFs </td></tr><tr><td> No. of polymorphic SLAFs </td><td>93,031.00 </td></tr><tr><td> Average depth in parents </td><td>127.31 </td></tr><tr><td> Average depth in individuals </td><td>7.85 </td></tr><tr><td> No. of SNPs </td><td>14,388.00 </td></tr><tr><td> SNP ratio per kb </td><td>6.02 </td></tr><tr><td colspan="2">High-quality SLAF markers </td></tr><tr><td> No. of high-quality SLAF markers </td><td>9,412 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><p><strong>N</strong>o: number; SLAF: specific length amplified fragment.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/9624250" target="_blank">View Large</a></div></div><p>Sequencing depth alone cannot ensure the genotyping quality; thus, genotyping quality scores (details see methods) were used to select qualified markers. The worst low-quality marker was discarded during each cycle. This dynamic process was repeated until the average genotype quality score of all SLAF markers reached the cut-off value of 30. Finally, 9,412 SLAFs were defined as high-quality SLAF markers with over 80-fold parental sequence depth, over 7-fold progeny sequence depth and &gt;70% integrity among the F1 population.</p>                            <h3 scrollto-destination=9624252 id="9624252" class="section-title" data-legacy-id=s3b>3.2. High-density linkage map construction</h3>
<p>The 9,412 high-quality markers were distributed into eight LGs according to their locations on mei genome and the MLOD scores with other markers (at least one MLOD score &gt;5). A total of 8,007 markers containing skewed markers (segregation and genotype sequence data are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S1 and S2</a>) were used to construct the final linkage map (Fig. <a reveal-id="DSV003F1" data-open="DSV003F1" class="link link-reveal link-table xref-fig">1</a>). The information of all markers on the map is organized in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>, which includes marker names, LGs, genetic position and physical location in mei genome. Following linkage analysis, coverage of the markers was 155.77-fold in the male parent, 98.21-fold in the female parent and 8.18-fold in each F1 individual (on average). The integrity of each marker among the 387 F1 individuals was also a key parameter to control map quality. All markers on the map demonstrated 96% integrity (on average). The final map was 1,550.62 cM in length with an average distance of 0.195 cM between adjacent markers. As shown in Table <a reveal-id="DSV003TB2" data-open="DSV003TB2" class="link link-reveal link-table xref-fig">2</a>, the largest LG was LG2, with 1,722 SLAF markers and a 0.15-cM marker interval, while LG6 was the smallest with 698 SLAF markers and a 0.20-cM marker interval (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). The uneven length of LGs is identical with the karyotypic parameters of mei chromosomes,<sup><a reveal-id="DSV003C30" data-open="DSV003C30" class="link link-ref link-reveal xref-bibr">30</a></sup> and the distribution of SLAF markers on LGs is shown in Fig. <a reveal-id="DSV003F2" data-open="DSV003F2" class="link link-reveal link-table xref-fig">2</a>. The largest gap was 24.13 cM in LG1, followed by 12.79 cM in LG6. 3.50% skewed markers with <em>P</em> &lt; 0.05 and 4.98% with <em>P</em> &lt; 0.01 have been identified on the final map. Here, we defined an area containing four skewed markers as segregation distortion regions (SDRs), and one SDR was detected in LG2, while three other SDRs were located in LG6. These SDRs may be related with preferential selection and will not affect the accuracy of the genetic map.<sup><a reveal-id="DSV003C31" data-open="DSV003C31" class="link link-ref link-reveal xref-bibr">31</a></sup></p>                        <a id="9624254" scrollto-destination="9624254"></a>
<div content-id="DSV003TB2" class="table-modal"><div class="table-wrap"><div id="DSV003TB2" class="table-wrap-title" data-id="DSV003TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Basic characteristics of mei linkage groups</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>LGID </th><th>Total marker </th><th>Total distance (cM) </th><th>Average distance (cM) </th><th>Max gap </th><th>Segregation SLAF (<em>P</em> &lt; 0.05) </th><th>Segregation SLAF (<em>P</em> &lt; 0.01) </th><th>SDR number </th></tr></thead><tbody><tr><td>LG1 </td><td>1,139 </td><td>238.14 </td><td>0.21 </td><td>24.13 </td><td>10 </td><td>49 </td><td>0 </td></tr><tr><td>LG2 </td><td>1,722 </td><td>263.84 </td><td>0.15 </td><td>2.95 </td><td>156 </td><td>181 </td><td>1 </td></tr><tr><td>LG3 </td><td>1,131 </td><td>216.97 </td><td>0.19 </td><td>2.86 </td><td>3 </td><td>9 </td><td>0 </td></tr><tr><td>LG4 </td><td>917 </td><td>205.79 </td><td>0.22 </td><td>5.53 </td><td>1 </td><td>1 </td><td>0 </td></tr><tr><td>LG5 </td><td>972 </td><td>231.69 </td><td>0.24 </td><td>3.4 </td><td>32 </td><td>39 </td><td>0 </td></tr><tr><td>LG6 </td><td>698 </td><td>142.48 </td><td>0.2 </td><td>12.79 </td><td>78 </td><td>105 </td><td>3 </td></tr><tr><td>LG7 </td><td>704 </td><td>129.8 </td><td>0.18 </td><td>6.93 </td><td>0 </td><td>15 </td><td>0 </td></tr><tr><td>LG8 </td><td>724 </td><td>121.9 </td><td>0.17 </td><td>3.13 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>Max group </td><td>1,722 </td><td>263.84 </td><td>0.24 </td><td>24.13 </td><td>156 </td><td>181 </td><td>3 </td></tr><tr><td>Min group </td><td>698 </td><td>121.9 </td><td>0.15 </td><td>2.86 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>Total </td><td>8,007 </td><td>1,550.62 </td><td>1.56 </td><td>61.72 </td><td>280 </td><td>399 </td><td>4 </td></tr></tbody></table></div><div content-id="DSV003TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>LGID </th><th>Total marker </th><th>Total distance (cM) </th><th>Average distance (cM) </th><th>Max gap </th><th>Segregation SLAF (<em>P</em> &lt; 0.05) </th><th>Segregation SLAF (<em>P</em> &lt; 0.01) </th><th>SDR number </th></tr></thead><tbody><tr><td>LG1 </td><td>1,139 </td><td>238.14 </td><td>0.21 </td><td>24.13 </td><td>10 </td><td>49 </td><td>0 </td></tr><tr><td>LG2 </td><td>1,722 </td><td>263.84 </td><td>0.15 </td><td>2.95 </td><td>156 </td><td>181 </td><td>1 </td></tr><tr><td>LG3 </td><td>1,131 </td><td>216.97 </td><td>0.19 </td><td>2.86 </td><td>3 </td><td>9 </td><td>0 </td></tr><tr><td>LG4 </td><td>917 </td><td>205.79 </td><td>0.22 </td><td>5.53 </td><td>1 </td><td>1 </td><td>0 </td></tr><tr><td>LG5 </td><td>972 </td><td>231.69 </td><td>0.24 </td><td>3.4 </td><td>32 </td><td>39 </td><td>0 </td></tr><tr><td>LG6 </td><td>698 </td><td>142.48 </td><td>0.2 </td><td>12.79 </td><td>78 </td><td>105 </td><td>3 </td></tr><tr><td>LG7 </td><td>704 </td><td>129.8 </td><td>0.18 </td><td>6.93 </td><td>0 </td><td>15 </td><td>0 </td></tr><tr><td>LG8 </td><td>724 </td><td>121.9 </td><td>0.17 </td><td>3.13 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>Max group </td><td>1,722 </td><td>263.84 </td><td>0.24 </td><td>24.13 </td><td>156 </td><td>181 </td><td>3 </td></tr><tr><td>Min group </td><td>698 </td><td>121.9 </td><td>0.15 </td><td>2.86 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>Total </td><td>8,007 </td><td>1,550.62 </td><td>1.56 </td><td>61.72 </td><td>280 </td><td>399 </td><td>4 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN2"><div class="fn"><p>LG: linkage group; SDR: segregation distortion region; cM: centiMorgan.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/9624254" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV003TB2" class="table-wrap-title" data-id="DSV003TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Basic characteristics of mei linkage groups</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>LGID </th><th>Total marker </th><th>Total distance (cM) </th><th>Average distance (cM) </th><th>Max gap </th><th>Segregation SLAF (<em>P</em> &lt; 0.05) </th><th>Segregation SLAF (<em>P</em> &lt; 0.01) </th><th>SDR number </th></tr></thead><tbody><tr><td>LG1 </td><td>1,139 </td><td>238.14 </td><td>0.21 </td><td>24.13 </td><td>10 </td><td>49 </td><td>0 </td></tr><tr><td>LG2 </td><td>1,722 </td><td>263.84 </td><td>0.15 </td><td>2.95 </td><td>156 </td><td>181 </td><td>1 </td></tr><tr><td>LG3 </td><td>1,131 </td><td>216.97 </td><td>0.19 </td><td>2.86 </td><td>3 </td><td>9 </td><td>0 </td></tr><tr><td>LG4 </td><td>917 </td><td>205.79 </td><td>0.22 </td><td>5.53 </td><td>1 </td><td>1 </td><td>0 </td></tr><tr><td>LG5 </td><td>972 </td><td>231.69 </td><td>0.24 </td><td>3.4 </td><td>32 </td><td>39 </td><td>0 </td></tr><tr><td>LG6 </td><td>698 </td><td>142.48 </td><td>0.2 </td><td>12.79 </td><td>78 </td><td>105 </td><td>3 </td></tr><tr><td>LG7 </td><td>704 </td><td>129.8 </td><td>0.18 </td><td>6.93 </td><td>0 </td><td>15 </td><td>0 </td></tr><tr><td>LG8 </td><td>724 </td><td>121.9 </td><td>0.17 </td><td>3.13 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>Max group </td><td>1,722 </td><td>263.84 </td><td>0.24 </td><td>24.13 </td><td>156 </td><td>181 </td><td>3 </td></tr><tr><td>Min group </td><td>698 </td><td>121.9 </td><td>0.15 </td><td>2.86 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>Total </td><td>8,007 </td><td>1,550.62 </td><td>1.56 </td><td>61.72 </td><td>280 </td><td>399 </td><td>4 </td></tr></tbody></table></div><div content-id="DSV003TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>LGID </th><th>Total marker </th><th>Total distance (cM) </th><th>Average distance (cM) </th><th>Max gap </th><th>Segregation SLAF (<em>P</em> &lt; 0.05) </th><th>Segregation SLAF (<em>P</em> &lt; 0.01) </th><th>SDR number </th></tr></thead><tbody><tr><td>LG1 </td><td>1,139 </td><td>238.14 </td><td>0.21 </td><td>24.13 </td><td>10 </td><td>49 </td><td>0 </td></tr><tr><td>LG2 </td><td>1,722 </td><td>263.84 </td><td>0.15 </td><td>2.95 </td><td>156 </td><td>181 </td><td>1 </td></tr><tr><td>LG3 </td><td>1,131 </td><td>216.97 </td><td>0.19 </td><td>2.86 </td><td>3 </td><td>9 </td><td>0 </td></tr><tr><td>LG4 </td><td>917 </td><td>205.79 </td><td>0.22 </td><td>5.53 </td><td>1 </td><td>1 </td><td>0 </td></tr><tr><td>LG5 </td><td>972 </td><td>231.69 </td><td>0.24 </td><td>3.4 </td><td>32 </td><td>39 </td><td>0 </td></tr><tr><td>LG6 </td><td>698 </td><td>142.48 </td><td>0.2 </td><td>12.79 </td><td>78 </td><td>105 </td><td>3 </td></tr><tr><td>LG7 </td><td>704 </td><td>129.8 </td><td>0.18 </td><td>6.93 </td><td>0 </td><td>15 </td><td>0 </td></tr><tr><td>LG8 </td><td>724 </td><td>121.9 </td><td>0.17 </td><td>3.13 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>Max group </td><td>1,722 </td><td>263.84 </td><td>0.24 </td><td>24.13 </td><td>156 </td><td>181 </td><td>3 </td></tr><tr><td>Min group </td><td>698 </td><td>121.9 </td><td>0.15 </td><td>2.86 </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>Total </td><td>8,007 </td><td>1,550.62 </td><td>1.56 </td><td>61.72 </td><td>280 </td><td>399 </td><td>4 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN2"><div class="fn"><p>LG: linkage group; SDR: segregation distortion region; cM: centiMorgan.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/9624254" target="_blank">View Large</a></div></div>                        <a id="9624255" scrollto-destination="9624255"></a>
<div id="DSV003F1" data-id="DSV003F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv003/2/m_dsv00301.png?Expires=2147483647&Signature=JIITOzmiIJBHiGIhGXW~AbHanWaPCitb90jhx02s5DzFCMkSjat28QtgAinILKeAZh7goU34B-rcMIT2PqDw4r8BRXXMwBfb8BGD0kSwdUS1myxSnUBgqHw6Ai2J4KmJ~g05X57Ykn5vDtcha8nLpr9fiw3k3FprVr1MW5Yq~AIUU3gyrp7w0yZyYtppU2J77LkW9JX17V6Q1mSqW3gaFFgUv3vfmANdUi30vHBlhL8HlCNTwK8ZHgN1Z17rfkX~BHvFHFvU0H6yYfl6BpkwwHppJhUOENVoUYvJULm4~e4IZnELb5-h63LTTnKeB-HQjuHNdGZ0Jk4dldYTxyryww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Numbers of each marker segregation type on the linkage map of mei. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv00301.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9624255/dsv00301.tif" path-from-xml="dsv00301.tif" target="_blank">View large</a><a section="9624255" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv003/2/dsv00301.png?Expires=2147483647&Signature=KWphMRueBZPHrsJOOzHZqtpjHTFDn~OkYlZwK1rriuooYvgid7JydV2GLTSl5rgDIH3mYldIxuJ-i9vDk80vN6CVkFXa5H6TR1GfhocUTHvDVpTIThOO4jVWe6fBMd3RVbJ7pzPPWPnA83RXsNrkM43s19-ser9liLtrE9x7xzBJI6r0fVEziVshXAEvdGdYIwcthu48ZVSfGOWE2koqnglYvsoHOqAMKwUAZGeotHWK20FcI1HJ5tNmju2OP6ffZ8THN-3SHCkzisx6hMcY-xrvHhzOgoLsTKCmMmbJqFIAXiiAdmN9YmfStny7oXvVcQMI6CA~66bm2u4-ULtnIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9624255&ar=591017&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00301.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Numbers of each marker segregation type on the linkage map of mei. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV003F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv003/2/m_dsv00301.png?Expires=2147483647&Signature=JIITOzmiIJBHiGIhGXW~AbHanWaPCitb90jhx02s5DzFCMkSjat28QtgAinILKeAZh7goU34B-rcMIT2PqDw4r8BRXXMwBfb8BGD0kSwdUS1myxSnUBgqHw6Ai2J4KmJ~g05X57Ykn5vDtcha8nLpr9fiw3k3FprVr1MW5Yq~AIUU3gyrp7w0yZyYtppU2J77LkW9JX17V6Q1mSqW3gaFFgUv3vfmANdUi30vHBlhL8HlCNTwK8ZHgN1Z17rfkX~BHvFHFvU0H6yYfl6BpkwwHppJhUOENVoUYvJULm4~e4IZnELb5-h63LTTnKeB-HQjuHNdGZ0Jk4dldYTxyryww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Numbers of each marker segregation type on the linkage map of mei. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv00301.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9624255/dsv00301.tif" path-from-xml="dsv00301.tif" target="_blank">View large</a><a section="9624255" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv003/2/dsv00301.png?Expires=2147483647&Signature=KWphMRueBZPHrsJOOzHZqtpjHTFDn~OkYlZwK1rriuooYvgid7JydV2GLTSl5rgDIH3mYldIxuJ-i9vDk80vN6CVkFXa5H6TR1GfhocUTHvDVpTIThOO4jVWe6fBMd3RVbJ7pzPPWPnA83RXsNrkM43s19-ser9liLtrE9x7xzBJI6r0fVEziVshXAEvdGdYIwcthu48ZVSfGOWE2koqnglYvsoHOqAMKwUAZGeotHWK20FcI1HJ5tNmju2OP6ffZ8THN-3SHCkzisx6hMcY-xrvHhzOgoLsTKCmMmbJqFIAXiiAdmN9YmfStny7oXvVcQMI6CA~66bm2u4-ULtnIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9624255&ar=591017&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00301.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Numbers of each marker segregation type on the linkage map of mei. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                        <a id="9624256" scrollto-destination="9624256"></a>
<div id="DSV003F2" data-id="DSV003F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv003/2/m_dsv00302.png?Expires=2147483647&Signature=UZTAhzoyj5BHKRsPh5-Otg3AYkRSD9hkkuZJxQm4upR3-rPv7Ujye3mYWNAzcTNgv5GzAl2zHZPN5dHD1V9NAO-74HZwlEgZlKkZRvIxuTeCKx64bXVXkmIkrsEbllGNn-0Z2EY-JEchwCIdM0qK3gckO0XiKQPZ-PmmdfdxNrnQBM37Pa8MFO9mZe-9pz8BXzvotBolNv1VwaxTXd3rlQfxOsGvwL~cHirO9LDxFOtl8itzEGAwRm~DaEyjKOCF5lD5ZsIwuabvGFrG2lLpHSTCZnhEwoFGSYCXL7u2foy2Qy~4pTYL5Cqid7B09mf454Rq5r6G-C~rQX-HuSkf3Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of SLAF markers on eight linkage groups of mei. A black bar indicates a SLAF marker. The x-axis represents linkage group number and the y-axis indicates genetic distance (centiMorgan as unit)." path-from-xml="dsv00302.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9624256/dsv00302.tif" path-from-xml="dsv00302.tif" target="_blank">View large</a><a section="9624256" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv003/2/dsv00302.png?Expires=2147483647&Signature=kcNdPOtgznUk4Td-GqsOeatm4fPMsOtSUiAzuJLbB3rccxr3KTl6PfWhR2v6iFN1NIn5IOKExvTq1ccmk7LGNd5cH--bcxO2O~H07IkiakjMGIoTgnXbp~R1i1ElidYlrVoKFHMyt4OmGGIrqcBr-lHBavUCH5ZDcg20zOKmp96uEso99rBuNr~ZxPlfNYjeQ7eLabROFNdnf-A2O3LW3BkdVGsJoxbfZaV1OKdiktRsTatQpBapPNG1hcnBqSgrXmq3IEZOuDdAWYHXsbqtwgkeJRh6UViyfXUeS1OcGLw31qHfeT7Ft8a6WJrAZqlIJUkLj9l1t1Ny8oeT9NPIMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9624256&ar=591017&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00302.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of SLAF markers on eight linkage groups of mei. A black bar indicates a SLAF marker. The <em>x</em>-axis represents linkage group number and the <em>y</em>-axis indicates genetic distance (centiMorgan as unit).</p></div></div><div content-id="DSV003F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv003/2/m_dsv00302.png?Expires=2147483647&Signature=UZTAhzoyj5BHKRsPh5-Otg3AYkRSD9hkkuZJxQm4upR3-rPv7Ujye3mYWNAzcTNgv5GzAl2zHZPN5dHD1V9NAO-74HZwlEgZlKkZRvIxuTeCKx64bXVXkmIkrsEbllGNn-0Z2EY-JEchwCIdM0qK3gckO0XiKQPZ-PmmdfdxNrnQBM37Pa8MFO9mZe-9pz8BXzvotBolNv1VwaxTXd3rlQfxOsGvwL~cHirO9LDxFOtl8itzEGAwRm~DaEyjKOCF5lD5ZsIwuabvGFrG2lLpHSTCZnhEwoFGSYCXL7u2foy2Qy~4pTYL5Cqid7B09mf454Rq5r6G-C~rQX-HuSkf3Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of SLAF markers on eight linkage groups of mei. A black bar indicates a SLAF marker. The x-axis represents linkage group number and the y-axis indicates genetic distance (centiMorgan as unit)." path-from-xml="dsv00302.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9624256/dsv00302.tif" path-from-xml="dsv00302.tif" target="_blank">View large</a><a section="9624256" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv003/2/dsv00302.png?Expires=2147483647&Signature=kcNdPOtgznUk4Td-GqsOeatm4fPMsOtSUiAzuJLbB3rccxr3KTl6PfWhR2v6iFN1NIn5IOKExvTq1ccmk7LGNd5cH--bcxO2O~H07IkiakjMGIoTgnXbp~R1i1ElidYlrVoKFHMyt4OmGGIrqcBr-lHBavUCH5ZDcg20zOKmp96uEso99rBuNr~ZxPlfNYjeQ7eLabROFNdnf-A2O3LW3BkdVGsJoxbfZaV1OKdiktRsTatQpBapPNG1hcnBqSgrXmq3IEZOuDdAWYHXsbqtwgkeJRh6UViyfXUeS1OcGLw31qHfeT7Ft8a6WJrAZqlIJUkLj9l1t1Ny8oeT9NPIMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9624256&ar=591017&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00302.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of SLAF markers on eight linkage groups of mei. A black bar indicates a SLAF marker. The <em>x</em>-axis represents linkage group number and the <em>y</em>-axis indicates genetic distance (centiMorgan as unit).</p></div></div>                            <h3 scrollto-destination=9624257 id="9624257" class="section-title" data-legacy-id=s3c>3.3. Comparative analysis of the high-density linkage map</h3>
<p>To evaluate the quality of the genetic map, we firstly mapped the SLAF markers to the mei genome. An excellent collinearity between physical distances and genetic distances of all SLAF markers in eight LGs is presented in Fig. <a reveal-id="DSV003F3" data-open="DSV003F3" class="link link-reveal link-table xref-fig">3</a>. All consecutive curves generated from the eight LGs indicate two facts: the mei genome was sufficiently covered with SLAF markers, and SLAF markers were placed accurately within each LG. Most parts of these curves represent a falling trend, suggesting that their genetic and physical positions follow an identical order. Meanwhile, two possible recombination hotspots can be observed intuitively on pseudo-chromosome 1 and pseudo-chromosome 4, and the relative positions in each LG can be seen in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>. </p>                        <a id="9624259" scrollto-destination="9624259"></a>
<div id="DSV003F3" data-id="DSV003F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv003/2/m_dsv00303.png?Expires=2147483647&Signature=vxdBlAW0C2-K-roX6202B71YwL9AudU3qu0cf6ZJdYnq0HJFnoE942B01GeSW6V2XC7DZGQ~QgIPMSpJyER205Xp9G7jP9BR~ZmPKmh4Ta5G873nb5faC54Q4cEZtZY13E2aUIWBzCLAPLEczdMnEzkmyXMZJN3hckf81aNeUv~GA7RCPv2IDulB89WYvE-CC2BDDgoZneSsgIJOWq1LGu1JK6veg1uO1O5aol6J8dsMkF53DTEvQtwz3hhLYrgIbHcWstng378252KaCmWT7xJ0uc4WzV0s8WuJquQwLiKyQ1lCD0hJ-hQagLCEr1Loz1uSaL5bBAdhM1emwlIzRw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Collinearity analysis of all mei linkage groups with mei genome sequence. The x-axis scales genetic distance of mei LGs accordingly, while the y-axis represents the position of mei peseudo-chromosomes. SLAF markers in these LGs are plotted as dots on the Figure. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv00303.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9624259/dsv00303.tif" path-from-xml="dsv00303.tif" target="_blank">View large</a><a section="9624259" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv003/2/dsv00303.png?Expires=2147483647&Signature=qwV5CwuBQQZZE1lpG2HRlhWG4eZ3Q1VKC8UwBKM~SBbBSxJWsjIBBIzKDTuOKZD5L7VAbLudXsnN~ZNXt5WH0fNtt5MbPIfmacA-gedEEycRknmelwe060Dq3ys227d31htUyxgQvOrbph7ROHl0wLQcF0jdWuXO6nu-dBIHZeSFCbSpurDvAnjhXy7uJbbCJ6w9JbRoC2ae2bJNOQ820i1D89gERlMFYshya9FuW2GFi4SA3AQFZ8MIlflyrWTXJpt5JepHFZfVqBXELE7sjEtNfzZGqRfaIRcHWXUESryFZEU4sdVlSpWzSt3e~lRgl64DtbOKZcUM~gzJO47FhA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9624259&ar=591017&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00303.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Collinearity analysis of all mei linkage groups with mei genome sequence. The <em>x</em>-axis scales genetic distance of mei LGs accordingly, while the <em>y</em>-axis represents the position of mei peseudo-chromosomes. SLAF markers in these LGs are plotted as dots on the Figure. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV003F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv003/2/m_dsv00303.png?Expires=2147483647&Signature=vxdBlAW0C2-K-roX6202B71YwL9AudU3qu0cf6ZJdYnq0HJFnoE942B01GeSW6V2XC7DZGQ~QgIPMSpJyER205Xp9G7jP9BR~ZmPKmh4Ta5G873nb5faC54Q4cEZtZY13E2aUIWBzCLAPLEczdMnEzkmyXMZJN3hckf81aNeUv~GA7RCPv2IDulB89WYvE-CC2BDDgoZneSsgIJOWq1LGu1JK6veg1uO1O5aol6J8dsMkF53DTEvQtwz3hhLYrgIbHcWstng378252KaCmWT7xJ0uc4WzV0s8WuJquQwLiKyQ1lCD0hJ-hQagLCEr1Loz1uSaL5bBAdhM1emwlIzRw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Collinearity analysis of all mei linkage groups with mei genome sequence. The x-axis scales genetic distance of mei LGs accordingly, while the y-axis represents the position of mei peseudo-chromosomes. SLAF markers in these LGs are plotted as dots on the Figure. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv00303.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9624259/dsv00303.tif" path-from-xml="dsv00303.tif" target="_blank">View large</a><a section="9624259" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv003/2/dsv00303.png?Expires=2147483647&Signature=qwV5CwuBQQZZE1lpG2HRlhWG4eZ3Q1VKC8UwBKM~SBbBSxJWsjIBBIzKDTuOKZD5L7VAbLudXsnN~ZNXt5WH0fNtt5MbPIfmacA-gedEEycRknmelwe060Dq3ys227d31htUyxgQvOrbph7ROHl0wLQcF0jdWuXO6nu-dBIHZeSFCbSpurDvAnjhXy7uJbbCJ6w9JbRoC2ae2bJNOQ820i1D89gERlMFYshya9FuW2GFi4SA3AQFZ8MIlflyrWTXJpt5JepHFZfVqBXELE7sjEtNfzZGqRfaIRcHWXUESryFZEU4sdVlSpWzSt3e~lRgl64DtbOKZcUM~gzJO47FhA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9624259&ar=591017&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00303.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Collinearity analysis of all mei linkage groups with mei genome sequence. The <em>x</em>-axis scales genetic distance of mei LGs accordingly, while the <em>y</em>-axis represents the position of mei peseudo-chromosomes. SLAF markers in these LGs are plotted as dots on the Figure. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>Macro-collinearity was revealed by aligning the SLAF markers on the mei high-density map to the peach genome sequence. Using 90% identity as the standard, 6,837 SLAF markers were mapped onto the peach genome (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). LG1 and LG6 appeared to have adverse orientations with the peach chromosomes (Ch6 and Ch8), which were nominal since, there was no stringent direction of the markers during linkage mapping. LG5 was syntenic to two peach chromosomes, Ch2 and Ch4, suggesting that fission or translocation events have occurred between the two <em>Prunus</em> species.</p>                            <h3 scrollto-destination=9624261 id="9624261" class="section-title" data-legacy-id=s3d>3.4. Inheritance mode of weeping trait of mei</h3>
<p>As a prerequisite to mapping the loci responsible for weeping, the inheritance of the weeping trait was evaluated. In this study, two F1 segregation populations were obtained, of which the parents differed remarkably in tree architecture (upright cultivar × weeping cultivar). Of the 320 genotypes in the ‘LiuBanDan’ × ‘ShuangBi’ population, 143 were classified as weeping and 177 as upright. A <em>χ</em><sup>2</sup> test indicated that the growth habit segregation fit the expected ratio of 1 : 1. In the ‘LiuBan’ × ‘FenTaiChuiZhi’ mapping population (387 individuals), the growth habits of 165 and 222 trees were ranked as weeping (W) and upright (U), respectively (Table <a reveal-id="DSV003TB3" data-open="DSV003TB3" class="link link-reveal link-table xref-fig">3</a>). The growth habit segregation in this population was slightly distorted from the expected 1(W) : 1(U) ratio due to an excess of uprights. Meanwhile, diverse branching habit that determined different tree architecture of weeping progeny was observed. Combined with former molecular basis of weeping trait, we suggested that weeping trait of mei may be controlled by a major gene together with one or more modifiers to account for a deficiency of weeping seedlings in the ‘LiuBan’ × ‘FenTaiChuiZhi’ population. </p>                        <a id="9624263" scrollto-destination="9624263"></a>
<div content-id="DSV003TB3" class="table-modal"><div class="table-wrap"><div id="DSV003TB3" class="table-wrap-title" data-id="DSV003TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Segregation ratios of progenies with weeping and upright shape in mei F1 populations</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Population </th><th>Total </th><th>Weeping </th><th>Upright </th><th>Expected ration </th><th><em>χ</em><sup>2</sup> </th><th>Significance </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>χ</mi><mrow><msub><mi> </mi><mrow><mn>0.05</mn></mrow></msub></mrow><mn>2</mn></msubsup></math></span> </th></tr></thead><tbody><tr><td>LB <strong>×</strong>FT </td><td>387 </td><td>165 </td><td>222 </td><td>1 : 1 </td><td>4.20 </td><td>Significance </td><td>3.87 </td></tr><tr><td>LBD <strong>×</strong>SB </td><td>320 </td><td>143 </td><td>177 </td><td>1 : 1 </td><td>3.61 </td><td>No significance </td><td>3.87 </td></tr></tbody></table></div><div content-id="DSV003TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Population </th><th>Total </th><th>Weeping </th><th>Upright </th><th>Expected ration </th><th><em>χ</em><sup>2</sup> </th><th>Significance </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>χ</mi><mrow><msub><mi> </mi><mrow><mn>0.05</mn></mrow></msub></mrow><mn>2</mn></msubsup></math></span> </th></tr></thead><tbody><tr><td>LB <strong>×</strong>FT </td><td>387 </td><td>165 </td><td>222 </td><td>1 : 1 </td><td>4.20 </td><td>Significance </td><td>3.87 </td></tr><tr><td>LBD <strong>×</strong>SB </td><td>320 </td><td>143 </td><td>177 </td><td>1 : 1 </td><td>3.61 </td><td>No significance </td><td>3.87 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN3"><div class="fn"><p>Cross combinations: LB × FT: ‘LiuBan’ × ‘FenTaiChuiZhi’; LBD × SB: ‘LiuBanDan’ × ‘ShuangBi’.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/9624263" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV003TB3" class="table-wrap-title" data-id="DSV003TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Segregation ratios of progenies with weeping and upright shape in mei F1 populations</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Population </th><th>Total </th><th>Weeping </th><th>Upright </th><th>Expected ration </th><th><em>χ</em><sup>2</sup> </th><th>Significance </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>χ</mi><mrow><msub><mi> </mi><mrow><mn>0.05</mn></mrow></msub></mrow><mn>2</mn></msubsup></math></span> </th></tr></thead><tbody><tr><td>LB <strong>×</strong>FT </td><td>387 </td><td>165 </td><td>222 </td><td>1 : 1 </td><td>4.20 </td><td>Significance </td><td>3.87 </td></tr><tr><td>LBD <strong>×</strong>SB </td><td>320 </td><td>143 </td><td>177 </td><td>1 : 1 </td><td>3.61 </td><td>No significance </td><td>3.87 </td></tr></tbody></table></div><div content-id="DSV003TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Population </th><th>Total </th><th>Weeping </th><th>Upright </th><th>Expected ration </th><th><em>χ</em><sup>2</sup> </th><th>Significance </th><th><span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mi>χ</mi><mrow><msub><mi> </mi><mrow><mn>0.05</mn></mrow></msub></mrow><mn>2</mn></msubsup></math></span> </th></tr></thead><tbody><tr><td>LB <strong>×</strong>FT </td><td>387 </td><td>165 </td><td>222 </td><td>1 : 1 </td><td>4.20 </td><td>Significance </td><td>3.87 </td></tr><tr><td>LBD <strong>×</strong>SB </td><td>320 </td><td>143 </td><td>177 </td><td>1 : 1 </td><td>3.61 </td><td>No significance </td><td>3.87 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN3"><div class="fn"><p>Cross combinations: LB × FT: ‘LiuBan’ × ‘FenTaiChuiZhi’; LBD × SB: ‘LiuBanDan’ × ‘ShuangBi’.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/9624263" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=9624264 id="9624264" class="section-title" data-legacy-id=s3e>3.5. Fine mapping of loci conferring weeping trait</h3>
<p>First, QTL analysis was performed using R/qtl based on the high-density genetic map and phenotypic separation among progeny.<sup><a reveal-id="DSV003C27" data-open="DSV003C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Threshold of significance (<em>P</em> = 0.05) for each marker after 1,000 permutation was set to 4.35, which resulted in the identification of a single locus at 71.88 cM on pseudo-chromosome 7 (Fig. <a reveal-id="DSV003F4" data-open="DSV003F4" class="link link-reveal link-table xref-fig">4</a>). On the linkage map, three markers (marker353041, marker437413 and marker321918) located at 71.88 cM. The LOD scores on pseudo-chromosome 7 were much higher than those on other pseudo-chromosomes, with the LOD score of the strongest QTL peak (peak LOD score = 87.65) being the highest. This phenomenon suggested that the locus on pseudo-chromosome 7 was strongly associated with weeping trait. However, the 95% confidence intervals for the QTL position were 4.80–87.65 cM, almost covered the whole pseudo-chromosome 7. To conduct finer mapping of the loci controlling weeping trait, a Mutmap-like strategy was initially applied to identify polymorphisms of the SLAF markers. Mutmap was developed on the basis of bulked segregation analysis (BSA) by calculating a parameter (SNP-index) to judge the contribution of an allele to the mutant phenotype.<sup><a reveal-id="DSV003C28" data-open="DSV003C28" class="link link-ref link-reveal xref-bibr">28</a></sup> Here, we considered weeping as the mutant trait and calculated Δ(SLAF-index) for each SLAF marker on the genome. The relationship between Δ(SLAF-index) and SLAF position on the genome was graphically represented (Fig. <a reveal-id="DSV003F5" data-open="DSV003F5" class="link link-reveal link-table xref-fig">5</a>). As expected, Δ(SLAF-index) values were distributed randomly around 0 for most of the genome, with the exception of pseudo-chromosome 7. An obvious peak was observed when the Δ(SLAF-index) threshold was set to 0.85. Five markers (marker334902, marker430976, marker446598, marker431969 and marker348136) above the threshold value with an average Δ(SLAF-index) equal to 0.9 were considered to be tightly linked markers to the weeping trait, and ranged from 69.63 to 75.52 cM on LG7. The above results strongly suggested that a major locus controlling the weeping trait was located on pseudo-chromosome 7 of mei. Thus, we assumed that weeping trait was a completely monogenetic trait to further confirm this localization. On this basis, we considered weeping trait as a marker (marker0) and calculated the recombination rate of marker0 with other markers. Eventually, the locus conferring weeping trait was determined to be located between two markers, whose recombination rate with marker0 was the lowest in LG7. The two flanking markers were positioned at 76.83 cM (marker315769) and 82.99 cM (marker418421), respectively. </p>                        <a id="9624266" scrollto-destination="9624266"></a>
<div id="DSV003F4" data-id="DSV003F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv003/2/m_dsv00304.png?Expires=2147483647&Signature=wdCYV8QNkiz2SS~178UHEI23e9hFqgEwyLBK2f4GLFKAev-5KW7Kkpswc-NxQ9LQ0hQ-WiYau2eUJh7O4LwNVDCYIsoTgkkgZ0fyaik1kRYSB7PfOFy3AmKEPH-hu0yxo0Zu-Hc2-vH3z231Bf-ex7lue6ARUAn3AiFFAvOhE2s~lmpSuUvI7eI4RRX-yyZ3HdmzhCR6JUeiT3hv2sn6JixdhRg0Rp59atSgxBRajc4IeT7pMuzrabp0KAlyZdnvlW8~66kjmiLl8nLoQaG3r5RK4xQ-j3VyuIZl-kfifOB77CgtG-PH12YJxy3lrYBTemvsicvcxL72-X7~DVw3IA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="R/QTL analysis of weeping trait. The x-axis scales genetic distance of mei LGs accordingly, while the y-axis represents the LOD scores. The dashed line represents the significant threshold value of 4.35 (P = 0.05). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv00304.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9624266/dsv00304.tif" path-from-xml="dsv00304.tif" target="_blank">View large</a><a section="9624266" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv003/2/dsv00304.png?Expires=2147483647&Signature=iLFILuOyi9-3CAEzvZemZNGdIpdDRMerJaaxdZS-9xDpe78-spQkqcDo1HFsLSaXCM0oUk0SQxSZpcEWTF1slsqIRIO1qPiqNtenqRcE9HaMtIdlA8VLqJ8zjpQe0wfioB7yup227YoD2z4-RZTXR-D37pjoKO6peHAm7w8xYReEAX2jqlJ3f~6~0ZYiJo4-LxIn9THWAgT8KPPx4btRcNlqV5zcJnYJUm7y06OFUNHgnUnfIA9XIsvw1G1IezwXtg3MGB75hHIDDrFDgt789i62MD3CqY237SdofWJ8GdRNTPAv2Z-r3R57a3K3gUla-9PJ4ttcLd2TJm~U8UGJpw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9624266&ar=591017&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00304.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>R/QTL analysis of weeping trait. The <em>x</em>-axis scales genetic distance of mei LGs accordingly, while the <em>y</em>-axis represents the LOD scores. The dashed line represents the significant threshold value of 4.35 (<em>P</em> = 0.05). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV003F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv003/2/m_dsv00304.png?Expires=2147483647&Signature=wdCYV8QNkiz2SS~178UHEI23e9hFqgEwyLBK2f4GLFKAev-5KW7Kkpswc-NxQ9LQ0hQ-WiYau2eUJh7O4LwNVDCYIsoTgkkgZ0fyaik1kRYSB7PfOFy3AmKEPH-hu0yxo0Zu-Hc2-vH3z231Bf-ex7lue6ARUAn3AiFFAvOhE2s~lmpSuUvI7eI4RRX-yyZ3HdmzhCR6JUeiT3hv2sn6JixdhRg0Rp59atSgxBRajc4IeT7pMuzrabp0KAlyZdnvlW8~66kjmiLl8nLoQaG3r5RK4xQ-j3VyuIZl-kfifOB77CgtG-PH12YJxy3lrYBTemvsicvcxL72-X7~DVw3IA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="R/QTL analysis of weeping trait. The x-axis scales genetic distance of mei LGs accordingly, while the y-axis represents the LOD scores. The dashed line represents the significant threshold value of 4.35 (P = 0.05). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv00304.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9624266/dsv00304.tif" path-from-xml="dsv00304.tif" target="_blank">View large</a><a section="9624266" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv003/2/dsv00304.png?Expires=2147483647&Signature=iLFILuOyi9-3CAEzvZemZNGdIpdDRMerJaaxdZS-9xDpe78-spQkqcDo1HFsLSaXCM0oUk0SQxSZpcEWTF1slsqIRIO1qPiqNtenqRcE9HaMtIdlA8VLqJ8zjpQe0wfioB7yup227YoD2z4-RZTXR-D37pjoKO6peHAm7w8xYReEAX2jqlJ3f~6~0ZYiJo4-LxIn9THWAgT8KPPx4btRcNlqV5zcJnYJUm7y06OFUNHgnUnfIA9XIsvw1G1IezwXtg3MGB75hHIDDrFDgt789i62MD3CqY237SdofWJ8GdRNTPAv2Z-r3R57a3K3gUla-9PJ4ttcLd2TJm~U8UGJpw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9624266&ar=591017&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00304.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>R/QTL analysis of weeping trait. The <em>x</em>-axis scales genetic distance of mei LGs accordingly, while the <em>y</em>-axis represents the LOD scores. The dashed line represents the significant threshold value of 4.35 (<em>P</em> = 0.05). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                        <a id="9624267" scrollto-destination="9624267"></a>
<div id="DSV003F5" data-id="DSV003F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv003/2/m_dsv00305.png?Expires=2147483647&Signature=mWoU-RscTGrnlnXz9oqKn3NM90xjsfQMzVhaDkFnI0MphTL0rIJCOihFlvjJOnkFh98TrUFLTFHUkYJtvjouLmuDK2K5hB39bB9XP8uSuiRhO7Aj2vDio7lzlHdPbZMeyX0lc40v30jfK9H7C6yFMLGvHDi~TSQIy-xA2tN-BzIRVLTpK4cqrVIpy5ZEvob4VVn8kHsMffftuxSBpfyJjOApnzggKa1Gtp~VCwY8ug0W-uT~iAAuW9c5BXTMZICoEa8VLTB20MCUVrFiq2wGLlhf9xjUe6iDvSiIwQ7e6LXVTOx5YbCRVLartFZftd6XvBcA0DtsgFXBRcWBT~UXig__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Mutmap-like analysis of weeping trait in mei. The x-axis represents the position of pseudo-chromosomes, whereas the y-axis shows the value of △(SLAF-index). SLAF markers unique to weeping (i.e. not present in ‘LiuBan’ and upright individuals) are shown in grey, whereas those unique to upright (i.e. not present in ‘FenTaiChuiZhi’ and weeping individuals) are in black. The dashed line represents the threshold value of 0.85. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv00305.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9624267/dsv00305.tif" path-from-xml="dsv00305.tif" target="_blank">View large</a><a section="9624267" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv003/2/dsv00305.png?Expires=2147483647&Signature=PB~FZqjlozUs29xA2a2r9bdw2B7IvhidJY2djqfJjOtLZQGVuze4bPF~NQweUOzuqISfsYoqetyvKz2vOsLdM8VScuGumldDPDaRPBAXPmpuideW6fQf0~lQcfj5AUhVRUsKuk7~OgJZ-oXCWkhZpMUhaRsvr-HUeA52hJd5hlkxgq2EvVjGtYDr3JQmXI7S76LnNNKkZQC9SeWdIUHTpBgLQ0X7cp0~Fdj4P1UlitpGzPW4754mXFQqnRLrBSYpzoBmuZo8gXmQj~aDSpF-jOF1Uk3IOwXcSWNCwIiKw0Ax3cX4xwG4c9wvGwvkIdkk8h2UtC1AV1TmAwqPvrMu7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9624267&ar=591017&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00305.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Mutmap-like analysis of weeping trait in mei. The <em>x</em>-axis represents the position of pseudo-chromosomes, whereas the <em>y</em>-axis shows the value of △(SLAF-index). SLAF markers unique to weeping (i.e. not present in ‘LiuBan’ and upright individuals) are shown in grey, whereas those unique to upright (i.e. not present in ‘FenTaiChuiZhi’ and weeping individuals) are in black. The dashed line represents the threshold value of 0.85. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV003F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv003/2/m_dsv00305.png?Expires=2147483647&Signature=mWoU-RscTGrnlnXz9oqKn3NM90xjsfQMzVhaDkFnI0MphTL0rIJCOihFlvjJOnkFh98TrUFLTFHUkYJtvjouLmuDK2K5hB39bB9XP8uSuiRhO7Aj2vDio7lzlHdPbZMeyX0lc40v30jfK9H7C6yFMLGvHDi~TSQIy-xA2tN-BzIRVLTpK4cqrVIpy5ZEvob4VVn8kHsMffftuxSBpfyJjOApnzggKa1Gtp~VCwY8ug0W-uT~iAAuW9c5BXTMZICoEa8VLTB20MCUVrFiq2wGLlhf9xjUe6iDvSiIwQ7e6LXVTOx5YbCRVLartFZftd6XvBcA0DtsgFXBRcWBT~UXig__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Mutmap-like analysis of weeping trait in mei. The x-axis represents the position of pseudo-chromosomes, whereas the y-axis shows the value of △(SLAF-index). SLAF markers unique to weeping (i.e. not present in ‘LiuBan’ and upright individuals) are shown in grey, whereas those unique to upright (i.e. not present in ‘FenTaiChuiZhi’ and weeping individuals) are in black. The dashed line represents the threshold value of 0.85. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv00305.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9624267/dsv00305.tif" path-from-xml="dsv00305.tif" target="_blank">View large</a><a section="9624267" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv003/2/dsv00305.png?Expires=2147483647&Signature=PB~FZqjlozUs29xA2a2r9bdw2B7IvhidJY2djqfJjOtLZQGVuze4bPF~NQweUOzuqISfsYoqetyvKz2vOsLdM8VScuGumldDPDaRPBAXPmpuideW6fQf0~lQcfj5AUhVRUsKuk7~OgJZ-oXCWkhZpMUhaRsvr-HUeA52hJd5hlkxgq2EvVjGtYDr3JQmXI7S76LnNNKkZQC9SeWdIUHTpBgLQ0X7cp0~Fdj4P1UlitpGzPW4754mXFQqnRLrBSYpzoBmuZo8gXmQj~aDSpF-jOF1Uk3IOwXcSWNCwIiKw0Ax3cX4xwG4c9wvGwvkIdkk8h2UtC1AV1TmAwqPvrMu7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9624267&ar=591017&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00305.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Mutmap-like analysis of weeping trait in mei. The <em>x</em>-axis represents the position of pseudo-chromosomes, whereas the <em>y</em>-axis shows the value of △(SLAF-index). SLAF markers unique to weeping (i.e. not present in ‘LiuBan’ and upright individuals) are shown in grey, whereas those unique to upright (i.e. not present in ‘FenTaiChuiZhi’ and weeping individuals) are in black. The dashed line represents the threshold value of 0.85. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>To avoid loss of candidate genes, we integrated the results of the analyses. In total, 10 markers were considered to be tightly linked to weeping trait. Based on the positions of the SLAF markers on the fully sequence genome of mei, the candidate genomic region for weeping trait was located at 10.54–11.68 Mb on pseudo-chromosome 7 of mei (Fig. <a reveal-id="DSV003F6" data-open="DSV003F6" class="link link-reveal link-table xref-fig">6</a>A). </p>                        <a id="9624269" scrollto-destination="9624269"></a>
<div id="DSV003F6" data-id="DSV003F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv003/2/m_dsv00306.png?Expires=2147483647&Signature=KPEpskcRZwCLdcqAZ4EFA2HxjQTPva8r1KWxWnMuFhZ5BY6WCrF6uAT17tKkPVmrrzVS~c9occ37-oCZ2wG9F3TI0AGY~mVRs1SklURrs~R4KyUVAzT5HJThs3gnCTuUc5OUoN2s2zaJNhbRErYbJbAjV8i1jbCL--iiIqbBRuj-jqGo1c13cEclAtv2irKpN9lOQ5KQ9E5hKKB0Q62s7Q-HOBarfdA-4dj99ej8P7VdV6xTXB~Mu-YGBcohK2EuP278R6dfS04oFHZ10iM2mrDbmnQnwYebGGgTOA4gRD38vQDt~GhL-wCtzwRAt45~3mrqhnkvugTY25eV5dVmXg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of tightly linked markers and plausible candidate genes of weeping on Pseudo-chromosome 7 of mei. (A) Candidate genomic region on pseudo-chromosome 7. (B) Schematic diagram of the order of the 10 tightly linked markers and the 18 plausible candidate genes of weeping based on gene annotation. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv00306.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9624269/dsv00306.tif" path-from-xml="dsv00306.tif" target="_blank">View large</a><a section="9624269" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv003/2/dsv00306.png?Expires=2147483647&Signature=ehGcZxiCnSAhFGtrGr7~kFsFbGlLD-WJKB93TaRo8OBQevEL~u~RY-cxOiJAbvdSd7ywd~q77EreWSmSEicFbbNDta0lbldy4mIbC3YXR2icIvTv0r5gvw83tZYYVts-V0uz6dNaRc11oHa-D5eL6Hda~EF9xhVg4CzZD1y5FGfDHsSpBscEwEQjkFFb1KcZKl2-E6zdhW7x4LKeZpD0N3Gc~pY9SqdRsTm~1bMpccROeT6FkHZdub0JvXpOtS3U5jIKtth~Xei0PE0PcwcPXztRTiXI1hat4Cgrs-b2LJ7qXFk8hKMtl-J0q6uId4PKJHK2RQ-nUWfrrpWwy2aW4g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9624269&ar=591017&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00306.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of tightly linked markers and plausible candidate genes of weeping on Pseudo-chromosome 7 of mei. (A) Candidate genomic region on pseudo-chromosome 7. (B) Schematic diagram of the order of the 10 tightly linked markers and the 18 plausible candidate genes of weeping based on gene annotation. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV003F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv003/2/m_dsv00306.png?Expires=2147483647&Signature=KPEpskcRZwCLdcqAZ4EFA2HxjQTPva8r1KWxWnMuFhZ5BY6WCrF6uAT17tKkPVmrrzVS~c9occ37-oCZ2wG9F3TI0AGY~mVRs1SklURrs~R4KyUVAzT5HJThs3gnCTuUc5OUoN2s2zaJNhbRErYbJbAjV8i1jbCL--iiIqbBRuj-jqGo1c13cEclAtv2irKpN9lOQ5KQ9E5hKKB0Q62s7Q-HOBarfdA-4dj99ej8P7VdV6xTXB~Mu-YGBcohK2EuP278R6dfS04oFHZ10iM2mrDbmnQnwYebGGgTOA4gRD38vQDt~GhL-wCtzwRAt45~3mrqhnkvugTY25eV5dVmXg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of tightly linked markers and plausible candidate genes of weeping on Pseudo-chromosome 7 of mei. (A) Candidate genomic region on pseudo-chromosome 7. (B) Schematic diagram of the order of the 10 tightly linked markers and the 18 plausible candidate genes of weeping based on gene annotation. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv00306.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9624269/dsv00306.tif" path-from-xml="dsv00306.tif" target="_blank">View large</a><a section="9624269" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv003/2/dsv00306.png?Expires=2147483647&Signature=ehGcZxiCnSAhFGtrGr7~kFsFbGlLD-WJKB93TaRo8OBQevEL~u~RY-cxOiJAbvdSd7ywd~q77EreWSmSEicFbbNDta0lbldy4mIbC3YXR2icIvTv0r5gvw83tZYYVts-V0uz6dNaRc11oHa-D5eL6Hda~EF9xhVg4CzZD1y5FGfDHsSpBscEwEQjkFFb1KcZKl2-E6zdhW7x4LKeZpD0N3Gc~pY9SqdRsTm~1bMpccROeT6FkHZdub0JvXpOtS3U5jIKtth~Xei0PE0PcwcPXztRTiXI1hat4Cgrs-b2LJ7qXFk8hKMtl-J0q6uId4PKJHK2RQ-nUWfrrpWwy2aW4g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9624269&ar=591017&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00306.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of tightly linked markers and plausible candidate genes of weeping on Pseudo-chromosome 7 of mei. (A) Candidate genomic region on pseudo-chromosome 7. (B) Schematic diagram of the order of the 10 tightly linked markers and the 18 plausible candidate genes of weeping based on gene annotation. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=9624270 id="9624270" class="section-title" data-legacy-id=s3f>3.6. Annotated genes within the candidate region</h3>
<p>The interval of the candidate region is 1.14 Mb in length and contains 159 predicted protein encoding genes. Genes within the candidate region were then detected based on annotation of the mei genome (<a class="link link-uri" href="http://prunusmumegenome.bjfu.edu.cn./">http://prunusmumegenome.bjfu.edu.cn./</a>). All 159 predicted genes were first screened for predicted amino acid polymorphisms between the ‘LiuBan’ and ‘FenTaiChuiZhi’ alleles according to their whole-genome resequencing data. This selection resulted in 69 genes. Among them, 19 could not be annotated using the known annotation databases. A complete list of the remaining 50 genes is provided in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>.</p><p>Some of the genes are conspicuous. Nine were obviously associated with tension wood (TW) formation and development, which are related with weeping trait. Two members of the cellulose synthase-like (CSL) family, Pm024150 and Pm024152, are important for cell wall development and growth.<sup><a reveal-id="DSV003C32" data-open="DSV003C32" class="link link-ref link-reveal xref-bibr">32</a></sup> Pm024254 that encodes endo-beta-1,4-glucanase is believed to function in cell wall changes associated with diverse growth processes and cell elongation.<sup><a reveal-id="DSV003C33" data-open="DSV003C33" class="link link-ref link-reveal xref-bibr">33</a>,<a reveal-id="DSV003C34" data-open="DSV003C34" class="link link-ref link-reveal xref-bibr">34</a></sup> Two predicted genes (Pm024195 and Pm024255) have been suggested to be involved in cell wall formation and assembly.<sup><a reveal-id="DSV003C35" data-open="DSV003C35" class="link link-ref link-reveal xref-bibr">35</a>,<a reveal-id="DSV003C36" data-open="DSV003C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Auxin-induced protein 5NG4-like (Pm024277), a positive regulator of auxin metabolism, may be involved in secondary cell wall biogenesis.<sup><a reveal-id="DSV003C37" data-open="DSV003C37" class="link link-ref link-reveal xref-bibr">37</a></sup> Also, genes involved in lignin biosynthesis that affect fibre development may result in TW abnormalities.<sup><a reveal-id="DSV003C38" data-open="DSV003C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Pm024278 is a probable cinnamyl alcohol dehydrogenase 1, which catalyzes the final specific step for lignin monomer production,<sup><a reveal-id="DSV003C39" data-open="DSV003C39" class="link link-ref link-reveal xref-bibr">39</a></sup> while Pm024136 is predicted to be involved in xylan metabolism. NAC domain-containing protein 43 (Pm024277) may regulate the secondary cell wall lignification of tissues.<sup><a reveal-id="DSV003C40" data-open="DSV003C40" class="link link-ref link-reveal xref-bibr">40</a></sup></p><p>Weeping trait may be controlled by a regulatory gene causing differential expression of the downstream network.<sup><a reveal-id="DSV003C41" data-open="DSV003C41" class="link link-ref link-reveal xref-bibr">41</a></sup> Our annotation analysis revealed nine genes that may regulate gene expression at the transcriptional level or post-transcriptional level. 26S proteasome (Pm024160) deserves close attention, since it may play an important role in balancing cell expansion with cell proliferation during shoot development.<sup><a reveal-id="DSV003C42" data-open="DSV003C42" class="link link-ref link-reveal xref-bibr">42</a></sup> Twenty-one Protein NLP6 (Pm024165) is an uncharacterized transcription factor, while 22 Protein NLP7 (Pm024166) is a transcription factor involved in the regulation of nitrate assimilation and nitrate signal transduction.<sup><a reveal-id="DSV003C43" data-open="DSV003C43" class="link link-ref link-reveal xref-bibr">43</a></sup> BHLH155 (Pm024214) is a transcription factor that may regulate root development, while growth-regulating factor 8 (Pm024257) may act as a transcriptional activator that plays a role in the regulation of cell expansion in meristem tissues.<sup><a reveal-id="DSV003C44" data-open="DSV003C44" class="link link-ref link-reveal xref-bibr">44</a></sup> Four genes (Pm024270, Pm024271, Pm024275 and Pm024123) with significant function variation in the ‘LiuBan’ and ‘FenTaiChuiZhi’ genome were predicted to be DNA-directed RNA polymerase II (Pol II) subunit RPB1, a structure to promote translocation of Pol II, which serves as a platform for the assembly of factors that regulate transcription initiation, elongation, termination and mRNA processing.<sup><a reveal-id="DSV003C45" data-open="DSV003C45" class="link link-ref link-reveal xref-bibr">45</a></sup></p><p>In addition, some genes are related to enzymatic activities. Among these genes, Pm024200 corresponds to the tightly linked marker437413 and is suggested to participate in coenzyme transport and metabolism. It has been suggested that ethylene promotes cell elongation in cotton.<sup><a reveal-id="DSV003C38" data-open="DSV003C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Pm024219, which encodes serine/threonine-protein kinase WNK8, and Pm024247, which encodes subtilisin-like protease (precursor), may both contribute to cell elongation, since a conformational change in the serine/threonine protein kinase activates the downstream signalling of ethylene.<sup><a reveal-id="DSV003C38" data-open="DSV003C38" class="link link-ref link-reveal xref-bibr">38</a></sup> N6-adenosine-methyltransferase MT-A70-like (Pm024237), which methylates adenosine residues of some mRNAs, may also play a role in the efficiency of mRNA splicing, transport or translation.</p>                            <h2 scrollto-destination=9624275 id="9624275" class="section-title" data-legacy-id=s4>4. Discussion</h2>
                            <h3 scrollto-destination=9624276 id="9624276" class="section-title" data-legacy-id=s4a>4.1. Characteristics of SLAF-seq strategy for large-scale marker development</h3>
<p>The SLAF-seq strategy, a combination of locus-specific amplification and high-throughput sequencing, is an effective method for large-scale SNP discovery and genotyping, which has been applied successfully in various species.<sup><a reveal-id="DSV003C9  DSV003C11" data-open="DSV003C9  DSV003C11" class="link link-ref link-reveal xref-bibr">9–11</a></sup></p><p>Compared with conventional methods of marker development, the SLAF-seq method owned some unique superiority. First, since copy number uniformity among fragments can reduce bias sequencing result, a pre-designed scheme was conducted to ensure the density, uniformity and efficiency of marker development. The well-assembled reference genome of mei was analysed beforehand, considering the information on genomic GC content, repeat conditions and genetic characteristics. After that, a pilot experiment was performed to check the reduced representation library features in targeted length range (214–294 bp), which was supposed to include fragments with similar amplification features on the gel. When non-specific amplified bands appeared, the pre-design step would be repeated. SLAF markers were designed such that an even distribution throughout the genome was established and repeats were avoided. Second, sequence depth together with the genotyping quality scores enhanced the genotyping accuracy. Since lower coverage per locus always lead to a lower confidence in each genotype call and each newly discovered marker,<sup><a reveal-id="DSV003C46" data-open="DSV003C46" class="link link-ref link-reveal xref-bibr">46</a></sup> sequencing depth is an important consideration in sequencing-based genotyping. It is suggested that 6-fold is a minimal sequencing depth for each individual when SLAF-seq strategy is used.<sup><a reveal-id="DSV003C10" data-open="DSV003C10" class="link link-ref link-reveal xref-bibr">10</a></sup> A sequencing depth of over 80-fold for parents and over 7-fold for progenies of our study can provide sufficiently high genotyping accuracy. Sequencing depth alone cannot ensure the genotyping quality; thus, genotyping quality scores were used to detect suspicious markers. In this study, suspicious marker was discarded during each cycle of the dynamic process, until the average genotype quality score of all SLAF markers reached the cut-off value of 30. Third, successive stringent principles were used for high-quality SLAF marker selection. All these features mentioned above make the SLAF-seq strategy an effective new tool for large-scale genotyping and rapidly markers development.</p><p>This study provides the first marker development on large scale for mei; in total, 9,412 high-quality SLAF markers were developed, and 8,007 polymorphic markers were finally identified for genetic linkage map construction. The integrity and accuracy of the SLAF markers were high, while the quality and quantity of them also met the requirements for construction of a high-quality and high-density genetic map. It is for the first time that SLAF-seq strategy was successfully applied in developing markers in ornamental woody plants.</p>                            <h3 scrollto-destination=9624280 id="9624280" class="section-title" data-legacy-id=s4b>4.2. The densest genetic map of mei</h3>
<p>The genetic map in this study was the first mei genetic map constructed using over 8,000 SLAF markers developed from SLAF-seq strategy. The earliest genetic map was of the Chinese mei cultivars ‘Fenban’ and ‘Kouzi’, using 144 SSR markers.<sup><a reveal-id="DSV003C47" data-open="DSV003C47" class="link link-ref link-reveal xref-bibr">47</a></sup> Later, by applying the RAD-seq strategy, 779 SNP markers were developed for the same mapping population, which were then used in anchoring and orienting scaffolds for mei genome assembly.<sup><a reveal-id="DSV003C48" data-open="DSV003C48" class="link link-ref link-reveal xref-bibr">48</a></sup> However, the resolutions of these maps and the sizes of the mapping populations were not enough for fine-scare QTL localization. In this study, the linkage map contained 8,007 markers, a marker density that has, to our knowledge, never been reached for mei and even any other species of <em>Prunus</em>.<sup><a reveal-id="DSV003C12" data-open="DSV003C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSV003C49" data-open="DSV003C49" class="link link-ref link-reveal xref-bibr">49</a></sup> This linkage map covered nearly the whole genome with a resolution of 0.195 cM.</p><p>To increase the quantity of markers and genomic coverage on the final genetic map, skewed markers were not discarded. In total, 8.5% skewed markers were inserted in the genetic map at the last step. Marker distortion may be caused by preferential selection or gametic/zygotic selection,<sup><a reveal-id="DSV003C31" data-open="DSV003C31" class="link link-ref link-reveal xref-bibr">31</a>,<a reveal-id="DSV003C50" data-open="DSV003C50" class="link link-ref link-reveal xref-bibr">50</a>,<a reveal-id="DSV003C51" data-open="DSV003C51" class="link link-ref link-reveal xref-bibr">51</a></sup> which always resulted in higher genetic variance.<sup><a reveal-id="DSV003C52" data-open="DSV003C52" class="link link-ref link-reveal xref-bibr">52</a></sup> It is reported that skewed markers can be applied for QTL mapping without detrimental effect and can be beneficial if used properly, as areas containing skewed markers may be recombination hotspots for specific traits.<sup><a reveal-id="DSV003C25" data-open="DSV003C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DSV003C53" data-open="DSV003C53" class="link link-ref link-reveal xref-bibr">53</a></sup></p><p>Besides, a HighMap strategy (the development of HighMap software is under way) was utilized to construct the high-density genetic map in this study. By exploiting an iterative ordering and error correction strategy as well as optimized GSS algorithm, HighMap was efficient for constructing linkage maps using markers developed from NGS. First, the marker order and the map distance were relatively accurate for data with genotyping errors, since an error correction strategy was performed. Second, HighMap greatly curbed the genetic map distance, while the iterative ordering strategy ensured an accurate marker order.<sup><a reveal-id="DSV003C20" data-open="DSV003C20" class="link link-ref link-reveal xref-bibr">20</a></sup> The excellent collinearity between physical distance and genetic distance of all SLAF markers in eight LGs strongly indicates the high quality of our map. Meanwhile, high macro-collinearity is also observed among the SLAF markers of our map and the peach genome, which is identical with former researches.<sup><a reveal-id="DSV003C47" data-open="DSV003C47" class="link link-ref link-reveal xref-bibr">47</a>,<a reveal-id="DSV003C48" data-open="DSV003C48" class="link link-ref link-reveal xref-bibr">48</a></sup></p><p>The mapping population should segregate for flower shape, corolla colour and cold tolerance, since the parents of the F1 population show obvious differences in these traits (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). As the mei whole-genome sequence has been released,<sup><a reveal-id="DSV003C48" data-open="DSV003C48" class="link link-ref link-reveal xref-bibr">48</a></sup> it can be used in combination with our map for fine-scale mapping of genes related to important ornamental traits, since the sequences of each marker is available and genes around the marker are easy to obtain. As the species of <em>Prunus</em> are closely related, this high-quality and high-density genetic map will also accelerate the molecular breeding for perennial woody plants in <em>Prunus</em>.</p>                            <h3 scrollto-destination=9624285 id="9624285" class="section-title" data-legacy-id=s4c>4.3. Location of locus conferring weeping trait utilizing the high-density map</h3>
<p>Previous studies have suggested that the weeping trait was controlled by a single recessive gene both in peach and mulberry.<sup><a reveal-id="DSV003C14" data-open="DSV003C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSV003C16" data-open="DSV003C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSV003C54  DSV003C57" data-open="DSV003C54  DSV003C57" class="link link-ref link-reveal xref-bibr">54–57</a></sup> However, Tu <em>et al.</em> suggested that weeping trait in <em>Castanea mollissima</em> was controlled by a major recessive gene together with some modifier genes. Although self-cross of a weeping parent yielded 1(W) : 3(U) ratio of the progeny, different branching drooping degrees were observed among the weeping offspring.<sup><a reveal-id="DSV003C58" data-open="DSV003C58" class="link link-ref link-reveal xref-bibr">58</a></sup> In terms of our mapping population, the segregation ratio was slightly distorted from the expected ratio, while no skewed markers had been detected around the flanking regions of the weeping locus. Besides, the branching drooping degree of the progeny was also diverse. The inheritance pattern of mei was similar to <em>C. mollissima.</em> Thus, we supposed that weeping trait of mei was controlled by a major recessive gene together with some modifier genes account for an excess of upright seedlings.</p><p>Because of the complexity of tree architecture, it is difficult to define a methodology for describing branching habit in an accurate quantitative way on tree descendants. However, the distinction between the weeping and upright forms of the mei progeny was sharply clear according to their branching directions.<sup><a reveal-id="DSV003C13" data-open="DSV003C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Thus, the progeny were scored quantitatively according to their phenotypes, and QTL analysis was then performed to map loci conferring weeping trait. According to the confidence intervals analysis, the candidate region almost covered the whole pseudo-chromosome 7. Then a Mutmap-like strategy was performed to fine map the weeping loci. The results of the two methods both suggested that the major locus controlling weeping trait was located on mei pseudo-chromosome 7. To confirm this conclusion, an extreme linkage analysis was conducted considering weeping trait as a complete recessive monogenetic trait. And the supposed weeping gene was also located on mei pseudo-chromosome 7. The results of these three methods can be mutual authentication and supplement, which could increase the reliability of the candidate locus of weeping trait.</p><p>Since the use of more highly annotated genomic databases could narrow down the selection of plausible candidate genes, it was worthwhile to determine the function of the candidate genes in the causal region. Combining the whole-genome resequencing data of the parents with the highly annotated genomic databases, we successfully narrowed down the number of plausible candidates to 69. However, due to the poor genetic background of weeping trait, it was difficult to determine the weeping controlling gene directly. Nine candidate structural genes and enzymes were predicted to be involved in cell wall and cellulose synthesis/degradation. These genes were supposed to be linked with weeping trait, since TW contains higher cellulose content and lower lignin, which may cause the drooping phenomenon.<sup><a reveal-id="DSV003C38" data-open="DSV003C38" class="link link-ref link-reveal xref-bibr">38</a>,<a reveal-id="DSV003C59" data-open="DSV003C59" class="link link-ref link-reveal xref-bibr">59</a></sup> Besides, nine genes were predicted to be related to transcriptional regulation, which were preferred for further experimental verification. In total, 18 candidates were considered as the most plausible genes of weeping in the casual region (Fig. <a reveal-id="DSV003F6" data-open="DSV003F6" class="link link-reveal link-table xref-fig">6</a>B). The annotation results also confirm that the localization of weeping locus is precise which suggested the high quality of our genetic map.</p><p>In summary, the present study demonstrates that SLAF-seq strategy is a powerful method for marker discovery and high-density linkage map construction, and our study resulted in the densest linkage map for mei. Comparative analysis and fine-scale mapping of the weeping trait suggest the high quality of this genetic map. A locus conferring weeping trait was detected on a 1.14-Mb interval on pseudo-chromosome 7 of mei for the first time, which is important for a better understanding of genetic mechanism of weeping trait. The precise localization of weeping trait strongly indicates that the high-density genetic map should contribute to the breeding of important ornamental traits in mei, along with the genus <em>prunus</em>.</p>                            <h2 scrollto-destination=9624290 id="9624290" class="section-title" data-legacy-id=s5>Conflict of Interest statement</h2>
<p>The authors declared that they have no competing interests.</p>                            <h2 scrollto-destination=9624292 id="9624292" class="section-title" data-legacy-id=s6>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=9624294 id="9624294" class="section-title" data-legacy-id=s7>Funding</h2>
<p>This work was supported by the Fundamental Research Funds for the Ministry of Science and Technology (Grant No. 2011AA100207), the Fundamental Research Funds for Central Universities (No. BLYJ201407) and Special Fund for Beijing Common Construction Project. Funding to pay the Open Access publication charges for this article was provided by Fundamental Research Funds for the Ministry of Science and Technology (Grant No. 2011AA100207).</p>                            <h2 scrollto-destination=9624296 id="9624296" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We really appreciate Dr Satoshi Tabata and the reviewers for the precious comments and suggestions in improving our manuscript.</p>                            <h2 scrollto-destination=9624298 id="9624298" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSV003C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Altshuler</div> <div class="given-names">D.</div></div> <div class="name"><div class="surname">Pollara</div> <div class="given-names">V.J.</div></div> <div class="name"><div class="surname">Cowles</div> <div class="given-names">C.R.</div></div>et al.  </div>, <div class="year">2000</div>, <div class="article-title">An SNP map of the human genome generated by reduced representation shotgun sequencing</div>, <div class="source">Nature</div> , <div class="volume">407</div>, <div class="fpage">513</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20SNP%20map%20of%20the%20human%20genome%20generated%20by%20reduced%20representation%20shotgun%20sequencing&amp;author=D.%20Altshuler&amp;author=V.J.%20Pollara&amp;author=C.R.%20Cowles&amp;publication_year=2000&amp;journal=Nature&amp;volume=407&amp;pages=513-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35035083" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35035083" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35035083"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11029002" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Hyten</div> <div class="given-names">D.L.</div></div> <div class="name"><div class="surname">Cannon</div> <div class="given-names">S.B.</div></div> <div class="name"><div class="surname">Song</div> <div class="given-names">Q.</div></div>et al.  </div>, <div class="year">2010</div>, <div class="article-title">High-throughput SNP discovery through deep resequencing of a reduced representation library to anchor and orient scaffolds in the soybean whole genome sequence</div>, <div class="source">BMC Genomics</div> , <div class="volume">11</div>, <div class="fpage">38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20SNP%20discovery%20through%20deep%20resequencing%20of%20a%20reduced%20representation%20library%20to%20anchor%20and%20orient%20scaffolds%20in%20the%20soybean%20whole%20genome%20sequence&amp;author=D.L.%20Hyten&amp;author=S.B.%20Cannon&amp;author=Q.%20Song&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-38" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-38" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-38"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20078886" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Lucito</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Nakimura</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">West</div> <div class="given-names">J.A.</div></div>et al.  </div>, <div class="year">1998</div>, <div class="article-title">Genetic analysis using genomic representations</div>, <div class="source">Proc. Natl Acad. Sci.</div> , <div class="volume">95</div>, <div class="fpage">4487</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20analysis%20using%20genomic%20representations&amp;author=R.%20Lucito&amp;author=M.%20Nakimura&amp;author=J.A.%20West&amp;publication_year=1998&amp;journal=Proc.%20Natl%20Acad.%20Sci.&amp;volume=95&amp;pages=4487-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.8.4487" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.8.4487" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.8.4487"> </span></div> </div></div></div></div></div><div content-id="DSV003C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Van Tassell</div> <div class="given-names">C.P.</div></div> <div class="name"><div class="surname">Smith</div> <div class="given-names">T.P.</div></div> <div class="name"><div class="surname">Matukumalli</div> <div class="given-names">L.K.</div></div>et al.  </div>, <div class="year">2008</div>, <div class="article-title">SNP discovery and allele frequency estimation by deep sequencing of reduced representation libraries</div>, <div class="source">Nat. Methods</div> , <div class="volume">5</div>, <div class="fpage">247</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNP%20discovery%20and%20allele%20frequency%20estimation%20by%20deep%20sequencing%20of%20reduced%20representation%20libraries&amp;author=C.P.%20Van%20Tassell&amp;author=T.P.%20Smith&amp;author=L.K.%20Matukumalli&amp;publication_year=2008&amp;journal=Nat.%20Methods&amp;volume=5&amp;pages=247-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1185" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1185" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1185"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18297082" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Baxter</div> <div class="given-names">S.W.</div></div> <div class="name"><div class="surname">Davey</div> <div class="given-names">J.W.</div></div> <div class="name"><div class="surname">Johnston</div> <div class="given-names">J.S.</div></div>et al.  </div>, <div class="year">2011</div>, <div class="article-title">Linkage mapping and comparative genomics using next-generation RAD sequencing of a non-model organism</div>, <div class="source">PloS ONE</div> , <div class="volume">6</div>, <div class="fpage">e19315</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Linkage%20mapping%20and%20comparative%20genomics%20using%20next-generation%20RAD%20sequencing%20of%20a%20non-model%20organism&amp;author=S.W.%20Baxter&amp;author=J.W.%20Davey&amp;author=J.S.%20Johnston&amp;publication_year=2011&amp;journal=PloS%20ONE&amp;volume=6&amp;pages=e19315" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0019315" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0019315" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0019315"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21541297" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Chutimanitsakun</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Nipper</div> <div class="given-names">R.W.</div></div> <div class="name"><div class="surname">Cuesta-Marcos</div> <div class="given-names">A.</div></div>et al.  </div>, <div class="year">2011</div>, <div class="article-title">Construction and application for QTL analysis of a Restriction Site Associated DNA (RAD) linkage map in barley</div>, <div class="source">BMC Genomics</div> , <div class="volume">12</div>, <div class="fpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20and%20application%20for%20QTL%20analysis%20of%20a%20Restriction%20Site%20Associated%20DNA%20(RAD)%20linkage%20map%20in%20barley&amp;author=Y.%20Chutimanitsakun&amp;author=R.W.%20Nipper&amp;author=A.%20Cuesta-Marcos&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21205322" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Pfender</div> <div class="given-names">W.</div></div> <div class="name"><div class="surname">Saha</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Johnson</div> <div class="given-names">E.</div></div> <div class="name"><div class="surname">Slabaugh</div> <div class="given-names">M.</div></div></div> <div class="year">2011</div>, <div class="article-title">Mapping with RAD (restriction-site associated DNA) markers to rapidly identify QTL for stem rust resistance in Lolium perenne</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">122</div>, <div class="fpage">1467</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20with%20RAD%20(restriction-site%20associated%20DNA)%20markers%20to%20rapidly%20identify%20QTL%20for%20stem%20rust%20resistance%20in%20Lolium%20perenne&amp;author=W.%20Pfender&amp;author=M.%20Saha&amp;author=E.%20Johnson&amp;author=M.%20Slabaugh&amp;publication_year=2011&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=122&amp;pages=1467-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-011-1546-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-011-1546-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-011-1546-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21344184" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Poland</div> <div class="given-names">J.A.</div></div> <div class="name"><div class="surname">Brown</div> <div class="given-names">P.J.</div></div> <div class="name"><div class="surname">Sorrells</div> <div class="given-names">M.E.</div></div> <div class="name"><div class="surname">Jannink</div> <div class="given-names">J.-L.</div></div></div> <div class="year">2012</div>, <div class="article-title">Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach</div>, <div class="source">PloS ONE</div> , <div class="volume">7</div>, <div class="fpage">e32253</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20high-density%20genetic%20maps%20for%20barley%20and%20wheat%20using%20a%20novel%20two-enzyme%20genotyping-by-sequencing%20approach&amp;author=J.A.%20Poland&amp;author=P.J.%20Brown&amp;author=M.E.%20Sorrells&amp;author=J.-L.%20Jannink&amp;publication_year=2012&amp;journal=PloS%20ONE&amp;volume=7&amp;pages=e32253" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0032253" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0032253" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0032253"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22389690" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Huang</div> <div class="given-names">Z.</div></div> <div class="name"><div class="surname">Dai</div> <div class="given-names">Y.</div></div>et al.  </div>, <div class="year">2013</div>, <div class="article-title">The development of 7E chromosome-specific molecular markers for Thinopyrum elongatum based on SLAF-seq technology</div>, <div class="source">PloS ONE</div> , <div class="volume">8</div>, <div class="fpage">e65122</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20development%20of%207E%20chromosome-specific%20molecular%20markers%20for%20Thinopyrum%20elongatum%20based%20on%20SLAF-seq%20technology&amp;author=S.%20Chen&amp;author=Z.%20Huang&amp;author=Y.%20Dai&amp;publication_year=2013&amp;journal=PloS%20ONE&amp;volume=8&amp;pages=e65122" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0065122" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0065122" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0065122"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23762296" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Sun</div> <div class="given-names">X.</div></div> <div class="name"><div class="surname">Liu</div> <div class="given-names">D.</div></div> <div class="name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div>et al.  </div>, <div class="year">2013</div>, <div class="article-title">SLAF-seq: an efficient method of large-scale de novo SNP discovery and genotyping using high-throughput sequencing</div>, <div class="source">PloS ONE</div> , <div class="volume">8</div>, <div class="fpage">e58700</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SLAF-seq%3A%20an%20efficient%20method%20of%20large-scale%20de%20novo%20SNP%20discovery%20and%20genotyping%20using%20high-throughput%20sequencing&amp;author=X.%20Sun&amp;author=D.%20Liu&amp;author=X.%20Zhang&amp;publication_year=2013&amp;journal=PloS%20ONE&amp;volume=8&amp;pages=e58700" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0058700" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0058700" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0058700"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23527008" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Wang</div> <div class="given-names">L.</div></div> <div class="name"><div class="surname">Xin</div> <div class="given-names">H.</div></div>et al.  </div>, <div class="year">2013</div>, <div class="article-title">Construction of a high-density genetic map for sesame based on large scale marker development by specific length amplified fragment (SLAF) sequencing</div>, <div class="source">BMC Plant Biol.</div> , <div class="volume">13</div>, <div class="fpage">141</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20high-density%20genetic%20map%20for%20sesame%20based%20on%20large%20scale%20marker%20development%20by%20specific%20length%20amplified%20fragment%20(SLAF)%20sequencing&amp;author=Y.%20Zhang&amp;author=L.%20Wang&amp;author=H.%20Xin&amp;publication_year=2013&amp;journal=BMC%20Plant%20Biol.&amp;volume=13&amp;pages=141" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-13-141" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-13-141" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-13-141"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24060091" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Sun</div> <div class="given-names">L.</div></div> <div class="name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Yan</div> <div class="given-names">X.</div></div>et al.  </div>, <div class="year">2014</div>, <div class="article-title">Genetic control of juvenile growth and botanical architecture in an ornamental woody plant,. Prunus mume Sieb. et Zucc. as revealed by a high-density linkage map</div>, <div class="source">BMC Genet.</div> , <div class="volume">15</div>, <div class="fpage">S1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20control%20of%20juvenile%20growth%20and%20botanical%20architecture%20in%20an%20ornamental%20woody%20plant%2C.%20Prunus%20mume%20Sieb.%20et%20Zucc.%20as%20revealed%20by%20a%20high-density%20linkage%20map&amp;author=L.%20Sun&amp;author=Y.%20Wang&amp;author=X.%20Yan&amp;publication_year=2014&amp;journal=BMC%20Genet.&amp;volume=15&amp;pages=S1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2156-15-73" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2156-15-73" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2156-15-73"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25078672" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">J.</div></div></div> <div class="year">1996</div>, <div class="source">Chinese mei flowers</div> , <div class="publisher-loc">Haikou, China</div>: <div class="publisher-name">Hainan Publishing House</div>, pp.<div class="fpage">51</div>–<div class="lpage">2</div>.</div></div></div></div><div content-id="DSV003C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Yamanouchi</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Koyama</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Machii</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Takyu</div> <div class="given-names">T.</div></div> <div class="name"><div class="surname">Muramatsu</div> <div class="given-names">N.</div></div></div> <div class="year">2009</div>, <div class="article-title">Inheritance of a weeping character and the low frequency of rooting from cuttings of the mulberry variety ‘Shidareguwa</div>’, <div class="source">Plant Breed.</div> , <div class="volume">128</div>, <div class="fpage">321</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inheritance%20of%20a%20weeping%20character%20and%20the%20low%20frequency%20of%20rooting%20from%20cuttings%20of%20the%20mulberry%20variety%20%E2%80%98Shidareguwa&amp;author=H.%20Yamanouchi&amp;author=A.%20Koyama&amp;author=H.%20Machii&amp;author=T.%20Takyu&amp;author=N.%20Muramatsu&amp;publication_year=2009&amp;journal=Plant%20Breed.&amp;volume=128&amp;pages=321-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbr.2009.128.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbr.2009.128.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbr.2009.128.issue-3"> </span></div> </div></div></div></div></div><div content-id="DSV003C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Yamaguchi</div> <div class="given-names">S.</div></div></div> <div class="year">2008</div>, <div class="article-title">Gibberellin metabolism and its regulation</div>, <div class="source">Annu. Rev. Plant Biol.</div> , <div class="volume">59</div>, <div class="fpage">225</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gibberellin%20metabolism%20and%20its%20regulation&amp;author=S.%20Yamaguchi&amp;publication_year=2008&amp;journal=Annu.%20Rev.%20Plant%20Biol.&amp;volume=59&amp;pages=225-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.arplant.59.032607.092804" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.arplant.59.032607.092804" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.arplant.59.032607.092804"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18173378" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Werner</div> <div class="given-names">D.J.</div></div> <div class="name"><div class="surname">Chaparro</div> <div class="given-names">J.X.</div></div></div> <div class="year">2005</div>, <div class="article-title">Genetic interactions of pillar and weeping peach genotypes</div>, <div class="source">Hort Sci.</div> , <div class="volume">40</div>, <div class="fpage">18</div>–<div class="lpage">20</div>.</div></div></div></div><div content-id="DSV003C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Dirlewanger</div> <div class="given-names">E.</div></div> <div class="name"><div class="surname">Bodo</div> <div class="given-names">C.</div></div></div> <div class="year">1994</div>, <div class="article-title">Molecular genetic mapping of peach</div>, <div class="source">Euphytica</div> , <div class="volume">77</div>, <div class="fpage">101</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20genetic%20mapping%20of%20peach&amp;author=E.%20Dirlewanger&amp;author=C.%20Bodo&amp;publication_year=1994&amp;journal=Euphytica&amp;volume=77&amp;pages=101-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02551470" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02551470" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02551470"> </span></div> </div></div></div></div></div><div content-id="DSV003C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Ymeng</div> <div class="given-names">L.</div></div></div> <div class="year">2006</div>, <div class="source">Physiological characteristics of weeping phenomenon and its simple sequence repeats analysis in weeping peach <em>(</em>Prunus persica <em>Var. Pendula)</em></div> , <div class="publisher-loc">Taian</div>: <div class="publisher-name">College of Horticulture Science and Engineering, Shandong Agricultural University</div>, pp.<div class="fpage">54</div>.</div></div></div></div><div content-id="DSV003C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Li</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Kristiansen</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div></div> <div class="year">2008</div>, <div class="article-title">SOAP: short oligonucleotide alignment program</div>, <div class="source">Bioinformatics</div> , <div class="volume">24</div>, <div class="fpage">713</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SOAP%3A%20short%20oligonucleotide%20alignment%20program&amp;author=R.%20Li&amp;author=Y.%20Li&amp;author=K.%20Kristiansen&amp;author=J.%20Wang&amp;publication_year=2008&amp;journal=Bioinformatics&amp;volume=24&amp;pages=713-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btn025" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtn025" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtn025"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18227114" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">D.</div></div> <div class="name"><div class="surname">Ma</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Hong</div> <div class="given-names">W.</div></div>et al.  </div>, <div class="year">2014</div>, <div class="article-title">Construction and analysis of high-density linkage map using high-throughput sequencing data</div>, <div class="source">PloS ONE</div> , <div class="volume">9</div>, <div class="fpage">e98855</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20and%20analysis%20of%20high-density%20linkage%20map%20using%20high-throughput%20sequencing%20data&amp;author=D.%20Liu&amp;author=C.%20Ma&amp;author=W.%20Hong&amp;publication_year=2014&amp;journal=PloS%20ONE&amp;volume=9&amp;pages=e98855" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0098855" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0098855" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0098855"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24905985" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Jansen</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">De Jong</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Van Ooijen</div> <div class="given-names">J.</div></div></div> <div class="year">2001</div>, <div class="article-title">Constructing dense genetic linkage maps</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">102</div>, <div class="fpage">1113</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Constructing%20dense%20genetic%20linkage%20maps&amp;author=J.%20Jansen&amp;author=A.%20De%20Jong&amp;author=J.%20Van%20Ooijen&amp;publication_year=2001&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=102&amp;pages=1113-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220000489" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220000489" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220000489"> </span></div> </div></div></div></div></div><div content-id="DSV003C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Van Ooijen</div> <div class="given-names">J.</div></div></div> <div class="year">2011</div>, <div class="article-title">Multipoint maximum likelihood mapping in a full-sib family of an outbreeding species</div>, <div class="source">Genet. Res.</div> , <div class="volume">93</div>, <div class="fpage">343</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multipoint%20maximum%20likelihood%20mapping%20in%20a%20full-sib%20family%20of%20an%20outbreeding%20species&amp;author=J.%20Van%20Ooijen&amp;publication_year=2011&amp;journal=Genet.%20Res.&amp;volume=93&amp;pages=343-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1017/S0016672311000279" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1017%2FS0016672311000279" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1017%2FS0016672311000279"> </span></div> </div></div></div></div></div><div content-id="DSV003C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">van Os</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Stam</div> <div class="given-names">P.</div></div> <div class="name"><div class="surname">Visser</div> <div class="given-names">R.G.</div></div> <div class="name"><div class="surname">van Eck</div> <div class="given-names">H.J.</div></div></div> <div class="year">2005</div>, <div class="article-title">SMOOTH: a statistical method for successful removal of genotyping errors from high-density genetic linkage data</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">112</div>, <div class="fpage">187</div>–<div class="lpage">94</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SMOOTH%3A%20a%20statistical%20method%20for%20successful%20removal%20of%20genotyping%20errors%20from%20high-density%20genetic%20linkage%20data&amp;author=H.%20van%20Os&amp;author=P.%20Stam&amp;author=R.G.%20Visser&amp;author=H.J.%20van%20Eck&amp;publication_year=2005&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=112&amp;pages=187-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-005-0124-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-005-0124-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-005-0124-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16258753" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">X.</div></div> <div class="name"><div class="surname">Zhao</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Wei</div> <div class="given-names">X.</div></div>et al.  </div>, <div class="year">2012</div>, <div class="article-title">Genome-wide association study of flowering time and grain yield traits in a worldwide collection of rice germplasm</div>, <div class="source">Nat. Genet.</div> , <div class="volume">44</div>, <div class="fpage">32</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20study%20of%20flowering%20time%20and%20grain%20yield%20traits%20in%20a%20worldwide%20collection%20of%20rice%20germplasm&amp;author=X.%20Huang&amp;author=Y.%20Zhao&amp;author=X.%20Wei&amp;publication_year=2012&amp;journal=Nat.%20Genet.&amp;volume=44&amp;pages=32-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.1018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.1018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.1018"> </span></div> </div></div></div></div></div><div content-id="DSV003C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Xu</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Hu</div> <div class="given-names">Z.</div></div></div> <div class="year">2010</div>, <div class="article-title">Mapping quantitative trait loci using distorted markers</div>, <div class="source">Int. J. Plant Genom</div> , <div class="volume">2009</div>, <div class="fpage">2</div>–<div class="lpage">10</div>.</div></div></div></div><div content-id="DSV003C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Kosambi</div> <div class="given-names">D.</div></div></div> <div class="year">1943</div>, <div class="article-title">The estimation of map distances from recombination values</div>, <div class="source">Ann. Eugenics</div> , <div class="volume">12</div>, <div class="fpage">172</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20estimation%20of%20map%20distances%20from%20recombination%20values&amp;author=D.%20Kosambi&amp;publication_year=1943&amp;journal=Ann.%20Eugenics&amp;volume=12&amp;pages=172-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-1809.1943.tb02321.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-1809.1943.tb02321.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-1809.1943.tb02321.x"> </span></div> </div></div></div></div></div><div content-id="DSV003C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Broman</div> <div class="given-names">K.W.</div></div> <div class="name"><div class="surname">Sen</div> <div class="given-names">S.</div></div></div> <div class="year">2009</div>, <div class="source">A guide to QTL mapping with R/qtl</div> . <div class="publisher-name">Springer, New York</div>.</div></div></div></div><div content-id="DSV003C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Abe</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Kosugi</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Yoshida</div> <div class="given-names">K.</div></div>et al.  </div>, <div class="year">2012</div>, <div class="article-title">Genome sequencing reveals agronomically important loci in rice using MutMap</div>, <div class="source">Nat. Biotechnol.</div> , <div class="volume">30</div>, <div class="fpage">174</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequencing%20reveals%20agronomically%20important%20loci%20in%20rice%20using%20MutMap&amp;author=A.%20Abe&amp;author=S.%20Kosugi&amp;author=K.%20Yoshida&amp;publication_year=2012&amp;journal=Nat.%20Biotechnol.&amp;volume=30&amp;pages=174-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2095" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2095" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2095"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22267009" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Ashburner</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Ball</div> <div class="given-names">C.A.</div></div> <div class="name"><div class="surname">Blake</div> <div class="given-names">J.A.</div></div>et al.  </div>, <div class="year">2000</div>, <div class="article-title">Gene Ontology: tool for the unification of biology</div>, <div class="source">Nat. Genet.</div> , <div class="volume">25</div>, <div class="fpage">25</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20Ontology%3A%20tool%20for%20the%20unification%20of%20biology&amp;author=M.%20Ashburner&amp;author=C.A.%20Ball&amp;author=J.A.%20Blake&amp;publication_year=2000&amp;journal=Nat.%20Genet.&amp;volume=25&amp;pages=25-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/75556" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F75556" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F75556"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10802651" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Yang</div> <div class="given-names">B.</div></div> <div class="name"><div class="surname">Yang</div> <div class="given-names">W.</div></div> <div class="name"><div class="surname">Hao</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Zhang</div> <div class="given-names">Q.</div></div></div> <div class="year">2013</div>, <div class="article-title">Chromosome sectioning of Prunus mume Siet. et Zucc all the year around</div>, <div class="source">Acta Agrticulturae Boreali-Occidentalis Sinica</div> , <div class="volume">22</div>, <div class="fpage">150</div>–<div class="lpage">4</div>.</div></div></div></div><div content-id="DSV003C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Weber</div> <div class="given-names">J.L.</div></div></div> <div class="year">1990</div>, <div class="article-title">Informativeness of human (dC-dA) n·(dG-dT) n polymorphisms</div>, <div class="source">Genomics</div> , <div class="volume">7</div>, <div class="fpage">524</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Informativeness%20of%20human%20(dC-dA)%20n%C2%B7(dG-dT)%20n%20polymorphisms&amp;author=J.L.%20Weber&amp;publication_year=1990&amp;journal=Genomics&amp;volume=7&amp;pages=524-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0888-7543(90)90195-Z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0888-7543(90)90195-Z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0888-7543(90)90195-Z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1974878" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Liepman</div> <div class="given-names">A.H.</div></div> <div class="name"><div class="surname">Wilkerson</div> <div class="given-names">C.G.</div></div> <div class="name"><div class="surname">Keegstra</div> <div class="given-names">K.</div></div></div> <div class="year">2005</div>, <div class="article-title">Expression of cellulose synthase-like (Csl) genes in insect cells reveals that CslA family members encode mannan synthases</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">102</div>, <div class="fpage">2221</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20of%20cellulose%20synthase-like%20(Csl)%20genes%20in%20insect%20cells%20reveals%20that%20CslA%20family%20members%20encode%20mannan%20synthases&amp;author=A.H.%20Liepman&amp;author=C.G.%20Wilkerson&amp;author=K.%20Keegstra&amp;publication_year=2005&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=102&amp;pages=2221-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0409179102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0409179102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0409179102"> </span></div> </div></div></div></div></div><div content-id="DSV003C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Bujang</div> <div class="given-names">N.</div></div> <div class="name"><div class="surname">Harrison</div> <div class="given-names">N.</div></div> <div class="name"><div class="surname">Su</div> <div class="given-names">N.-Y.</div></div></div> <div class="year">2014</div>, <div class="article-title">A phylogenetic study of endo-beta-1, 4-glucanase in higher termites</div>, <div class="source">Insectes Sociaux</div> , <div class="volume">61</div>, <div class="fpage">29</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20phylogenetic%20study%20of%20endo-beta-1%2C%204-glucanase%20in%20higher%20termites&amp;author=N.%20Bujang&amp;author=N.%20Harrison&amp;author=N.-Y.%20Su&amp;publication_year=2014&amp;journal=Insectes%20Sociaux&amp;volume=61&amp;pages=29-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00040-013-0321-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00040-013-0321-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00040-013-0321-7"> </span></div> </div></div></div></div></div><div content-id="DSV003C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Lashbrook</div> <div class="given-names">C.C.</div></div> <div class="name"><div class="surname">Gonzalez-Bosch</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Bennett</div> <div class="given-names">A.B.</div></div></div> <div class="year">1994</div>, <div class="article-title">Two divergent endo-beta-1, 4-glucanase genes exhibit overlapping expression in ripening fruit and abscising flowers</div>, <div class="source">Plant Cell Online</div> , <div class="volume">6</div>, <div class="fpage">1485</div>–<div class="lpage">93</div>.</div></div></div></div><div content-id="DSV003C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Adair</div> <div class="given-names">W.S.</div></div> <div class="name"><div class="surname">Apt</div> <div class="given-names">K.E.</div></div></div> <div class="year">1990</div>, <div class="article-title">Cell wall regeneration in Chlamydomonas: accumulation of mRNAs encoding cell wall hydroxyproline-rich glycoproteins</div>, <div class="source">Proc. Natl Acad. Sci.</div> , <div class="volume">87</div>, <div class="fpage">7355</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cell%20wall%20regeneration%20in%20Chlamydomonas%3A%20accumulation%20of%20mRNAs%20encoding%20cell%20wall%20hydroxyproline-rich%20glycoproteins&amp;author=W.S.%20Adair&amp;author=K.E.%20Apt&amp;publication_year=1990&amp;journal=Proc.%20Natl%20Acad.%20Sci.&amp;volume=87&amp;pages=7355-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.87.19.7355" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.87.19.7355" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.87.19.7355"> </span></div> </div></div></div></div></div><div content-id="DSV003C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Baumberger</div> <div class="given-names">N.</div></div> <div class="name"><div class="surname">Doesseger</div> <div class="given-names">B.</div></div> <div class="name"><div class="surname">Guyot</div> <div class="given-names">R.</div></div>et al.  </div>, <div class="year">2003</div>, <div class="article-title">Whole-genome comparison of leucine-rich repeat extensins in Arabidopsis and rice. A conserved family of cell wall proteins form a vegetative and a reproductive clade</div>, <div class="source">Plant Physiol.</div> , <div class="volume">131</div>, <div class="fpage">1313</div>–<div class="lpage">26</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20comparison%20of%20leucine-rich%20repeat%20extensins%20in%20Arabidopsis%20and%20rice.%20A%20conserved%20family%20of%20cell%20wall%20proteins%20form%20a%20vegetative%20and%20a%20reproductive%20clade&amp;author=N.%20Baumberger&amp;author=B.%20Doesseger&amp;author=R.%20Guyot&amp;publication_year=2003&amp;journal=Plant%20Physiol.&amp;volume=131&amp;pages=1313-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.102.014928" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.102.014928" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.102.014928"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12644681" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Busov</div> <div class="given-names">V.B.</div></div> <div class="name"><div class="surname">Johannes</div> <div class="given-names">E.</div></div> <div class="name"><div class="surname">Whetten</div> <div class="given-names">R.W.</div></div>et al.  </div>, <div class="year">2004</div>, <div class="article-title">An auxin-inducible gene from loblolly pine (Pinus taeda L.) is differentially expressed in mature and juvenile-phase shoots and encodes a putative transmembrane protein</div>, <div class="source">Planta</div> , <div class="volume">218</div>, <div class="fpage">916</div>–<div class="lpage">27</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20auxin-inducible%20gene%20from%20loblolly%20pine%20(Pinus%20taeda%20L.)%20is%20differentially%20expressed%20in%20mature%20and%20juvenile-phase%20shoots%20and%20encodes%20a%20putative%20transmembrane%20protein&amp;author=V.B.%20Busov&amp;author=E.%20Johannes&amp;author=R.W.%20Whetten&amp;publication_year=2004&amp;journal=Planta&amp;volume=218&amp;pages=916-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00425-003-1175-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00425-003-1175-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00425-003-1175-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14722770" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Andersson Gunnerås</div> <div class="given-names">S.</div></div></div> <div class="year">2005</div>, <div class="source">Wood formation and transcript analysis with focus on tension wood and ethylene biology</div> , <div class="publisher-loc">Umeå</div>: <div class="publisher-name">Department of Forest Genetics and Plant Physiology, Swedish University of Agricultural Sciences</div>, pp.<div class="fpage">3</div>.</div></div></div></div><div content-id="DSV003C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Barakat</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Bagniewska-Zadworna</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Choi</div> <div class="given-names">A.</div></div>et al.  </div>, <div class="year">2009</div>, <div class="article-title">The cinnamyl alcohol dehydrogenase gene family in Populus: phylogeny, organization, and expression</div>, <div class="source">BMC Plant Biol.</div> , <div class="volume">9</div>, <div class="fpage">26</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20cinnamyl%20alcohol%20dehydrogenase%20gene%20family%20in%20Populus%3A%20phylogeny%2C%20organization%2C%20and%20expression&amp;author=A.%20Barakat&amp;author=A.%20Bagniewska-Zadworna&amp;author=A.%20Choi&amp;publication_year=2009&amp;journal=BMC%20Plant%20Biol.&amp;volume=9&amp;pages=26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-9-26" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-9-26" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-9-26"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19267902" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Mitsuda</div> <div class="given-names">N.</div></div> <div class="name"><div class="surname">Iwase</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Yamamoto</div> <div class="given-names">H.</div></div>et al.  </div>, <div class="year">2007</div>, <div class="article-title">NAC transcription factors, NST1 and NST3, are key regulators of the formation of secondary walls in woody tissues of Arabidopsis</div>, <div class="source">Plant Cell Online</div> , <div class="volume">19</div>, <div class="fpage">270</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NAC%20transcription%20factors%2C%20NST1%20and%20NST3%2C%20are%20key%20regulators%20of%20the%20formation%20of%20secondary%20walls%20in%20woody%20tissues%20of%20Arabidopsis&amp;author=N.%20Mitsuda&amp;author=A.%20Iwase&amp;author=H.%20Yamamoto&amp;publication_year=2007&amp;journal=Plant%20Cell%20Online&amp;volume=19&amp;pages=270-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.106.047043" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.106.047043" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.106.047043"> </span></div> </div></div></div></div></div><div content-id="DSV003C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Sugano</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Nakagawa</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Nyunoya</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Nakamura</div> <div class="given-names">T.</div></div></div> <div class="year">2004</div>, <div class="article-title">Expression of gibberellin 3. BETA.-hydroxylase gene in a gravi-response mutant, weeping Japanese flowering cherry</div>, <div class="source">Biol. Sci. Space</div> , <div class="volume">18</div>, <div class="fpage">261</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20of%20gibberellin%203.%20BETA.-hydroxylase%20gene%20in%20a%20gravi-response%20mutant%2C%20weeping%20Japanese%20flowering%20cherry&amp;author=M.%20Sugano&amp;author=Y.%20Nakagawa&amp;author=H.%20Nyunoya&amp;author=T.%20Nakamura&amp;publication_year=2004&amp;journal=Biol.%20Sci.%20Space&amp;volume=18&amp;pages=261-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2187/bss.18.261" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2187%2Fbss.18.261" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2187%2Fbss.18.261"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15858394" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Kurepa</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Wang</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Li</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Zaitlin</div> <div class="given-names">D.</div></div> <div class="name"><div class="surname">Pierce</div> <div class="given-names">A.J.</div></div> <div class="name"><div class="surname">Smalle</div> <div class="given-names">J.A.</div></div></div> <div class="year">2009</div>, <div class="article-title">Loss of 26S proteasome function leads to increased cell size and decreased cell number in Arabidopsis shoot organs</div>, <div class="source">Plant Physiol.</div> , <div class="volume">150</div>, <div class="fpage">178</div>–<div class="lpage">89</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Loss%20of%2026S%20proteasome%20function%20leads%20to%20increased%20cell%20size%20and%20decreased%20cell%20number%20in%20Arabidopsis%20shoot%20organs&amp;author=J.%20Kurepa&amp;author=S.%20Wang&amp;author=Y.%20Li&amp;author=D.%20Zaitlin&amp;author=A.J.%20Pierce&amp;author=J.A.%20Smalle&amp;publication_year=2009&amp;journal=Plant%20Physiol.&amp;volume=150&amp;pages=178-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.109.135970" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.109.135970" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.109.135970"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19321709" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Castaings</div> <div class="given-names">L.</div></div> <div class="name"><div class="surname">Camargo</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Pocholle</div> <div class="given-names">D.</div></div>et al.  </div>, <div class="year">2009</div>, <div class="article-title">The nodule inception-like protein 7 modulates nitrate sensing and metabolism in Arabidopsis</div>, <div class="source">Plant J.</div> , <div class="volume">57</div>, <div class="fpage">426</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20nodule%20inception-like%20protein%207%20modulates%20nitrate%20sensing%20and%20metabolism%20in%20Arabidopsis&amp;author=L.%20Castaings&amp;author=A.%20Camargo&amp;author=D.%20Pocholle&amp;publication_year=2009&amp;journal=Plant%20J.&amp;volume=57&amp;pages=426-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2009.57.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2009.57.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2009.57.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18826430" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Heim</div> <div class="given-names">M.A.</div></div> <div class="name"><div class="surname">Jakoby</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Werber</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Martin</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Weisshaar</div> <div class="given-names">B.</div></div> <div class="name"><div class="surname">Bailey</div> <div class="given-names">P.C.</div></div></div> <div class="year">2003</div>, <div class="article-title">The basic helix-loop-helix transcription factor family in plants: a genome-wide study of protein structure and functional diversity</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="volume">20</div>, <div class="fpage">735</div>–<div class="lpage">47</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20basic%20helix-loop-helix%20transcription%20factor%20family%20in%20plants%3A%20a%20genome-wide%20study%20of%20protein%20structure%20and%20functional%20diversity&amp;author=M.A.%20Heim&amp;author=M.%20Jakoby&amp;author=M.%20Werber&amp;author=C.%20Martin&amp;author=B.%20Weisshaar&amp;author=P.C.%20Bailey&amp;publication_year=2003&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=20&amp;pages=735-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msg088" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsg088" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsg088"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12679534" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Zheng</div> <div class="given-names">B.</div></div> <div class="name"><div class="surname">Wang</div> <div class="given-names">Z.</div></div> <div class="name"><div class="surname">Li</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Yu</div> <div class="given-names">B.</div></div> <div class="name"><div class="surname">Liu</div> <div class="given-names">J.-Y.</div></div> <div class="name"><div class="surname">Chen</div> <div class="given-names">X.</div></div></div> <div class="year">2009</div>, <div class="article-title">Intergenic transcription by RNA polymerase II coordinates Pol IV and Pol V in siRNA-directed transcriptional gene silencing in Arabidopsis</div>, <div class="source">Genes Dev.</div> , <div class="volume">23</div>, <div class="fpage">2850</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intergenic%20transcription%20by%20RNA%20polymerase%20II%20coordinates%20Pol%20IV%20and%20Pol%20V%20in%20siRNA-directed%20transcriptional%20gene%20silencing%20in%20Arabidopsis&amp;author=B.%20Zheng&amp;author=Z.%20Wang&amp;author=S.%20Li&amp;author=B.%20Yu&amp;author=J.-Y.%20Liu&amp;author=X.%20Chen&amp;publication_year=2009&amp;journal=Genes%20Dev.&amp;volume=23&amp;pages=2850-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.1868009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.1868009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.1868009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19948763" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Davey</div> <div class="given-names">J.W.</div></div> <div class="name"><div class="surname">Hohenlohe</div> <div class="given-names">P.A.</div></div> <div class="name"><div class="surname">Etter</div> <div class="given-names">P.D.</div></div> <div class="name"><div class="surname">Boone</div> <div class="given-names">J.Q.</div></div> <div class="name"><div class="surname">Catchen</div> <div class="given-names">J.M.</div></div> <div class="name"><div class="surname">Blaxter</div> <div class="given-names">M.L.</div></div></div> <div class="year">2011</div>, <div class="article-title">Genome-wide genetic marker discovery and genotyping using next-generation sequencing</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">12</div>, <div class="fpage">499</div>–<div class="lpage">510</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20genetic%20marker%20discovery%20and%20genotyping%20using%20next-generation%20sequencing&amp;author=J.W.%20Davey&amp;author=P.A.%20Hohenlohe&amp;author=P.D.%20Etter&amp;author=J.Q.%20Boone&amp;author=J.M.%20Catchen&amp;author=M.L.%20Blaxter&amp;publication_year=2011&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=12&amp;pages=499-510" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21681211" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Sun</div> <div class="given-names">L.</div></div> <div class="name"><div class="surname">Yang</div> <div class="given-names">W.</div></div> <div class="name"><div class="surname">Zhang</div> <div class="given-names">Q.</div></div>et al.  </div>, <div class="year">2013</div>, <div class="article-title">Genome-wide characterization and linkage mapping of simple sequence repeats in Mei (Prunus mume Sieb. et Zucc.)</div>, <div class="source">PloS ONE</div> , <div class="volume">8</div>, <div class="fpage">e59562</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20characterization%20and%20linkage%20mapping%20of%20simple%20sequence%20repeats%20in%20Mei%20(Prunus%20mume%20Sieb.%20et%20Zucc.)&amp;author=L.%20Sun&amp;author=W.%20Yang&amp;author=Q.%20Zhang&amp;publication_year=2013&amp;journal=PloS%20ONE&amp;volume=8&amp;pages=e59562" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0059562" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0059562" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0059562"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23555708" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">Q.X.</div></div> <div class="name"><div class="surname">Chen</div> <div class="given-names">W.B.</div></div> <div class="name"><div class="surname">Sun</div> <div class="given-names">L.D.</div></div>et al.  </div>, <div class="year">2012</div>, <div class="article-title">The genome of Prunus mume</div>, <div class="source">Nat. Commun.</div> , <div class="volume">3</div>, <div class="fpage">1318</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20Prunus%20mume&amp;author=Q.X.%20Zhang&amp;author=W.B.%20Chen&amp;author=L.D.%20Sun&amp;publication_year=2012&amp;journal=Nat.%20Commun.&amp;volume=3&amp;pages=1318" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms2290" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms2290" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms2290"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23271652" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Arús</div> <div class="given-names">P.</div></div> <div class="name"><div class="surname">Verde</div> <div class="given-names">I.</div></div> <div class="name"><div class="surname">Sosinski</div> <div class="given-names">B.</div></div> <div class="name"><div class="surname">Zhebentyayeva</div> <div class="given-names">T.</div></div> <div class="name"><div class="surname">Abbott</div> <div class="given-names">A.G.</div></div></div> <div class="year">2012</div>, <div class="article-title">The peach genome</div>, <div class="source">Tree Genet. Genom.</div> , <div class="volume">8</div>, <div class="fpage">531</div>–<div class="lpage">47</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20peach%20genome&amp;author=P.%20Ar%C3%BAs&amp;author=I.%20Verde&amp;author=B.%20Sosinski&amp;author=T.%20Zhebentyayeva&amp;author=A.G.%20Abbott&amp;publication_year=2012&amp;journal=Tree%20Genet.%20Genom.&amp;volume=8&amp;pages=531-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11295-012-0493-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11295-012-0493-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11295-012-0493-8"> </span></div> </div></div></div></div></div><div content-id="DSV003C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Faris</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Laddomada</div> <div class="given-names">B.</div></div> <div class="name"><div class="surname">Gill</div> <div class="given-names">B.</div></div></div> <div class="year">1998</div>, <div class="article-title">Molecular mapping of segregation distortion loci in Aegilops tauschii</div>, <div class="source">Genetics</div> , <div class="volume">149</div>, <div class="fpage">319</div>–<div class="lpage">27</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20mapping%20of%20segregation%20distortion%20loci%20in%20Aegilops%20tauschii&amp;author=J.%20Faris&amp;author=B.%20Laddomada&amp;author=B.%20Gill&amp;publication_year=1998&amp;journal=Genetics&amp;volume=149&amp;pages=319-27" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9584106" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Zhu</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Zhai</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Wan</div> <div class="given-names">J.</div></div></div> <div class="year">2005</div>, <div class="article-title">Mapping segregation distortion loci and quantitative trait loci for spikelet sterility in rice (Oryza sativa L.)</div>, <div class="source">Genet. Res.</div> , <div class="volume">86</div>, <div class="fpage">97</div>–<div class="lpage">106</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20segregation%20distortion%20loci%20and%20quantitative%20trait%20loci%20for%20spikelet%20sterility%20in%20rice%20(Oryza%20sativa%20L.)&amp;author=C.%20Wang&amp;author=C.%20Zhu&amp;author=H.%20Zhai&amp;author=J.%20Wan&amp;publication_year=2005&amp;journal=Genet.%20Res.&amp;volume=86&amp;pages=97-106" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1017/S0016672305007779" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1017%2FS0016672305007779" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1017%2FS0016672305007779"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16356283" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">L.</div></div> <div class="name"><div class="surname">Wang</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Li</div> <div class="given-names">H.</div></div>et al.  </div>, <div class="year">2010</div>, <div class="article-title">Effects of missing marker and segregation distortion on QTL mapping in F2 populations</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">121</div>, <div class="fpage">1071</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effects%20of%20missing%20marker%20and%20segregation%20distortion%20on%20QTL%20mapping%20in%20F2%20populations&amp;author=L.%20Zhang&amp;author=S.%20Wang&amp;author=H.%20Li&amp;publication_year=2010&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=121&amp;pages=1071-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-010-1372-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-010-1372-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-010-1372-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20535442" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Xu</div> <div class="given-names">S.</div></div></div> <div class="year">2008</div>, <div class="article-title">Quantitative trait locus mapping can benefit from segregation distortion</div>, <div class="source">Genetics</div> , <div class="volume">180</div>, <div class="fpage">2201</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20trait%20locus%20mapping%20can%20benefit%20from%20segregation%20distortion&amp;author=S.%20Xu&amp;publication_year=2008&amp;journal=Genetics&amp;volume=180&amp;pages=2201-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.108.090688" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.108.090688" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.108.090688"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18957707" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV003C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Bassi</div> <div class="given-names">D.</div></div> <div class="name"><div class="surname">Rizzo</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Geibel</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Fischer</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Fischer</div> <div class="given-names">C.</div></div></div> <div class="year">2000</div>, <div class="article-title">Peach breeding for growth habit</div>, <div class="source">Acta Hortic.</div> , <div class="volume">1</div>, <div class="fpage">411</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Peach%20breeding%20for%20growth%20habit&amp;author=D.%20Bassi&amp;author=M.%20Rizzo&amp;author=M.%20Geibel&amp;author=M.%20Fischer&amp;author=C.%20Fischer&amp;publication_year=2000&amp;journal=Acta%20Hortic.&amp;volume=1&amp;pages=411-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.17660/ActaHortic.2000.538.72" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.17660%2FActaHortic.2000.538.72" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.17660%2FActaHortic.2000.538.72"> </span></div> </div></div></div></div></div><div content-id="DSV003C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Monet</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Bastard</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Gibault</div> <div class="given-names">B.</div></div></div> <div class="year">1988</div>, <div class="article-title">Genetic study of the weeping habit in peach</div>, <div class="source">Agronomie</div> , <div class="volume">8</div>, <div class="fpage">127</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20study%20of%20the%20weeping%20habit%20in%20peach&amp;author=R.%20Monet&amp;author=Y.%20Bastard&amp;author=B.%20Gibault&amp;publication_year=1988&amp;journal=Agronomie&amp;volume=8&amp;pages=127-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1051/agro:19880205" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1051%2Fagro:19880205" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1051%2Fagro:19880205"> </span></div> </div></div></div></div></div><div content-id="DSV003C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Scorza</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Bassi</div> <div class="given-names">D.</div></div> <div class="name"><div class="surname">Liverani</div> <div class="given-names">A.</div></div></div> <div class="year">2002</div>, <div class="article-title">Genetic interactions of pillar (columnar), compact, and dwarf peach tree genotypes</div>, <div class="source">J. Am. Soc. Hortic. Sci.</div> , <div class="volume">127</div>, <div class="fpage">254</div>–<div class="lpage">61</div>.</div></div></div></div><div content-id="DSV003C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Yamazaki</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Okabe</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Takahashi</div> <div class="given-names">E.</div></div></div> <div class="year">1987</div>, <div class="article-title">Inheritance of some characteristics and breeding of new hybrids in flowering peaches</div>. <div class="source">Bull. Kanagawa Hort. Exp. Sta. (Japan)</div> , <div class="volume">34</div>, <div class="fpage">46</div>–<div class="lpage">53</div>.</div></div></div></div><div content-id="DSV003C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Tu</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Qiu</div> <div class="given-names">F.</div></div></div> <div class="year">2007</div>, <div class="article-title">The character display of <em>Castanea mollissima</em> var. pendula Sexual Progenies</div>, <div class="source">Acta Hirticuoturae Sinica</div> , <div class="volume">34</div>, <div class="fpage">760</div>–<div class="lpage">2</div>.</div></div></div></div><div content-id="DSV003C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV003C59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Coutand</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Jeronimidis</div> <div class="given-names">G.</div></div> <div class="name"><div class="surname">Chanson</div> <div class="given-names">B.</div></div> <div class="name"><div class="surname">Loup</div> <div class="given-names">C.</div></div></div> <div class="year">2004</div>, <div class="article-title">Comparison of mechanical properties of tension and opposite wood in Populus</div>, <div class="source">Wood Sci. Technol.</div> , <div class="volume">38</div>, <div class="fpage">11</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20mechanical%20properties%20of%20tension%20and%20opposite%20wood%20in%20Populus&amp;author=C.%20Coutand&amp;author=G.%20Jeronimidis&amp;author=B.%20Chanson&amp;author=C.%20Loup&amp;publication_year=2004&amp;journal=Wood%20Sci.%20Technol.&amp;volume=38&amp;pages=11-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00226-003-0194-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00226-003-0194-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00226-003-0194-4"> </span></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2015. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/4.0/), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=591017&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv003/2/dsv003-dsv003supp_table4.doc?Expires=2147483647&amp;Signature=f44LHxo9mXxh9HzAqOBjcvO8WO6TT0JDbrw-~YINap0RrxdpQ-zFXlVjvJCe4ljFtQVBWqCk0kPvWYrEO19NDmv~4hsQ2IE5FCCO~Am01eZ4ngCDNAJi2tzefdPxcXC3VsskjmCWJT~bdTRh2wUABoLewhyBaOhg37aEywLPiuyS2KtG5AuMXRVfizTJLo4J4Ubcb0bmXxHwqNxoxCRH2RvfH9mwSpjY1XQrQGaqgGvDCxSUVNGabY2YwW8iZw2Z7HsdksWuSZM6MHFGo4NqpkmqRpWmjK6CiJ3KnFZzn6hrql13QgxWHivwyejFW9tctrbu3BmN8o4Z7tScuQ31MA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv003/2/dsv003-dsv003supp_table3.xls?Expires=2147483647&amp;Signature=gVb6Ik9aTkbVY0SYZEOfBzad-AwTGjBW502z3wXAKZhySBehYDQWcnb5ZnaT5bcIaCFf5l0H12Nn0vLoDjkaKXDHZdZR-Vgn3skj2JM4HtW4OwE0RCJjNThKNYfens-iGKToa4KGvsPH8cdixDEl0Pdig9-Oo-9fjOjVDVW9USKhGPM76olKSXxatuW3ByjGGL3Y~dGYl3a-HTRuk4brrQeNiXu7N32LyBUL5abh00S0gMyESqPlII9kagNJkN360iOile441rQ4MtJnvc45b29hS3NJn2u7kYgUpxwduhXUAMsun7P6dj2hZ4iRoEkJK5bsc2hkQS7M1InXvDLO~A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv003/2/dsv003-dsv003supp_table2.xls?Expires=2147483647&amp;Signature=Z8baRk-A9LkWQKi4rFmOiIjVgg-Tv~GHwVGdOxJWUzM28DXoI8J8qywYXyaLZKzR~2iKvsa8VG5WzjLBXWbdIpxsAsjTh3K5Vp2gngiyGs8HxRLT~axcc~SiaOIh3Rw3R0vvUoJz1Z3eK6P1k5nzF3dIj0zlL5RL3YaskCsg4VroW-6GP3yRmcRXBmPPQb-eYg8~WLg5Nc4z34w7ybOsrZ0l2t0bamnvEGd-7bEqlF-3XTuvR9lHVJWUEKAZjWSGX307~2-V3N0aQjjwz23IqZHSni2cbonrVL62WOgQGMUFMG~ENKsabTakWWNHKPoMy0tVpH0uSV5tkfqnWm543A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv003/2/dsv003-dsv003supp_table1.xlsx?Expires=2147483647&amp;Signature=3EkMAPetqcIp07Hv4WgcQuf2j4zhuKIdazEBQUOTNbIHopimJ9x5IklUw5zGfhSS~AcQrvWX-h3jVYmTGp8RFha4ndAss6vizG8p8VDINwLE8d75Ug0fmdDqAT5zHrSjAv3w0-IZN9sa5npsOFrKzy5TNquCIsgwJyyT5qjhFlhLZ1lMMWrbeCibvhqfsoYdg6X3nE2tauJe8JtOQPXNV5dBa36aG9a-zfdhGflyI31zUIPeDmKNqQ31Fw-lxu7QlOJacbbN4Rio3T9uXEKxV4Fk3qiF9MFHSHXiua~9y7fHLeWzS-5q3v01MK3vLgGFSBIoIZ28Kdj0bBmMmtY51Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv003/2/dsv003-dsv003supp_figures.doc?Expires=2147483647&amp;Signature=KKpQj9-H2PA9SIHpzF41AE~NwNDKT5cCsOVmA2YABRcKGv~2cafuGgDyXGCvYIHrjK6kqoD9D04au6wJOH0gxF-Us5jr9kWgqU13~fs-HzrUyZvMqL9mKW-4ao4Q1KsQUgaJNEhz0j25S7bf3aYfeWvxKGudNLhY-FqFZL9PljRTVmSp1UdUq43uYMZ2Qlx8QoMPF1GySfW8w5gDltwKXryWOBwlg17QUc3B5zvIBFPwC0OwoHmC~NmzWFe2lW72FeekEmIM2zOhuuBK6ZgqgQY3W2GdVSQkDyjglJNp0tWdOIGUWGgj1TLMQFx~D176Tx4mWZs4iblvXFsdfc3FHA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,044</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000356658900001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">35</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsv003"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="591017"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000356658900001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f3%2f183%2f591017">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000356658900001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (35)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F3%2F183%2F591017" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/591017'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="High-density genetic map construction and identification of a locus controlling weeping trait in an ornamental woody plant (Prunus mume Sieb. et Zucc)" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsv003");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f3%2f183%2f591017">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

