

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/15/3/137/464067","@type":"ScholarlyArticle","name":"Exploration of Human ORFeome: High-Throughput Preparation of ORF Clones and Efficient Characterization of Their Protein Products","datePublished":"2008-03-02","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/15/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2008-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsn004","keywords":["cDNA","HaloTag","proteomics","Flexi cloning","ORFeome"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/15/3/137/464067","author":[{"name":"Nagase, Takahiro","affiliation":"1  Department of Human Genome Research ,  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Yamakawa, Hisashi","affiliation":"1  Department of Human Genome Research ,  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Tadokoro, Shinichi","affiliation":"2  Laboratory of Pharmacogenomics, Graduate School of Pharmaceutical Sciences ,  Chiba University ,  1-8-1 Inohana, Chuo-ku, Chiba 260-8675 ,  Japan","@type":"Person"},{"name":"Nakajima, Daisuke","affiliation":"1  Department of Human Genome Research ,  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Inoue, Shinichi","affiliation":"1  Department of Human Genome Research ,  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Yamaguchi, Kei","affiliation":"3  Graduate School of Science and Technology ,  Chiba University ,  648 Matsudo, Matsudo, Chiba 271-8510 ,  Japan","@type":"Person"},{"name":"Itokawa, Yasuhide","affiliation":"4  Graduate School of Science ,  Toho University ,  2-2-1 Miyama, Funabashi, Chiba 274-8510 ,  Japan","@type":"Person"},{"name":"Kikuno, Reiko F.","affiliation":"1  Department of Human Genome Research ,  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Koga, Hisashi","affiliation":"1  Department of Human Genome Research ,  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Ohara, Osamu","affiliation":"1  Department of Human Genome Research ,  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan    2  Laboratory of Pharmacogenomics, Graduate School of Pharmaceutical Sciences ,  Chiba University ,  1-8-1 Inohana, Chuo-ku, Chiba 260-8675 ,  Japan    5  RIKEN Research Center for Allergy and Immunology ,  1-7-22 Suehiro, Tsurumi-ku, Yokohama, Kanagawa 230-0045 ,  Japan","@type":"Person"}],"description":"Abstract.  In this study, we established new systematic protocols for the preparation of cDNA clones, conventionally termed open reading frame (ORF) clones, sui","pageStart":"137","pageEnd":"149","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/3/1/m_cover.gif?Expires=2147483647&Signature=Wofi61EvCcdWxJvtysqPZG0U~KWWJOmThdMZDP-AglxAwDtnmCnY2MpChppyQDDpv6NdMaeLAZ~nLGaX7cJuWBVCGvKy3i2prPhQHBCH-vHV3CnTYh7LKJ6XOvu-5EP8bX5lf-70Wga4tfffe84AGB6pX1KlziPK7gdEs5QnVkZBbwLfiRPt-I1COdPZHhK91Imi3IuGiPl9ftt~EYfFKFyDk2nT8pASRTr325Ma~vy8IX6K~kWiKpA09pXB8RYEn5ztSYMyc20h7L3oQyBZ3j8qfpO0VRwhgradKgQz7QarN5Dl5qIio9OjPV0kSCdBe3E3RBSZsEr31pWrM0u5RQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Exploration of Human ORFeome: High-Throughput Preparation of ORF Clones and Efficient Characterization of Their Protein Products","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/3/1/m_cover.gif?Expires=2147483647&Signature=Wofi61EvCcdWxJvtysqPZG0U~KWWJOmThdMZDP-AglxAwDtnmCnY2MpChppyQDDpv6NdMaeLAZ~nLGaX7cJuWBVCGvKy3i2prPhQHBCH-vHV3CnTYh7LKJ6XOvu-5EP8bX5lf-70Wga4tfffe84AGB6pX1KlziPK7gdEs5QnVkZBbwLfiRPt-I1COdPZHhK91Imi3IuGiPl9ftt~EYfFKFyDk2nT8pASRTr325Ma~vy8IX6K~kWiKpA09pXB8RYEn5ztSYMyc20h7L3oQyBZ3j8qfpO0VRwhgradKgQz7QarN5Dl5qIio9OjPV0kSCdBe3E3RBSZsEr31pWrM0u5RQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Exploration of Human ORFeome: High-Throughput Preparation of ORF Clones and Efficient Characterization of Their Protein Products" />
    <meta property="og:description" content="Abstract.  In this study, we established new systematic protocols for the preparation of cDNA clones, conventionally termed open reading frame (ORF) clones, sui" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/15/3/137/464067" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2008-03-02" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Nagase, Takahiro" /><meta name="citation_author_institution" content="Department of Human Genome Research, Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Yamakawa, Hisashi" /><meta name="citation_author_institution" content="Department of Human Genome Research, Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Tadokoro, Shinichi" /><meta name="citation_author_institution" content="Laboratory of Pharmacogenomics, Graduate School of Pharmaceutical Sciences, Chiba University, 1-8-1 Inohana, Chuo-ku, Chiba 260-8675, Japan" /><meta name="citation_author" content="Nakajima, Daisuke" /><meta name="citation_author_institution" content="Department of Human Genome Research, Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Inoue, Shinichi" /><meta name="citation_author_institution" content="Department of Human Genome Research, Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Yamaguchi, Kei" /><meta name="citation_author_institution" content="Graduate School of Science and Technology, Chiba University, 648 Matsudo, Matsudo, Chiba 271-8510, Japan" /><meta name="citation_author" content="Itokawa, Yasuhide" /><meta name="citation_author_institution" content="Graduate School of Science, Toho University, 2-2-1 Miyama, Funabashi, Chiba 274-8510, Japan" /><meta name="citation_author" content="Kikuno, Reiko F." /><meta name="citation_author_institution" content="Department of Human Genome Research, Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Koga, Hisashi" /><meta name="citation_author_institution" content="Department of Human Genome Research, Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Ohara, Osamu" /><meta name="citation_author_institution" content="Department of Human Genome Research, Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author_institution" content="Laboratory of Pharmacogenomics, Graduate School of Pharmaceutical Sciences, Chiba University, 1-8-1 Inohana, Chuo-ku, Chiba 260-8675, Japan" /><meta name="citation_author_institution" content="RIKEN Research Center for Allergy and Immunology, 1-7-22 Suehiro, Tsurumi-ku, Yokohama, Kanagawa 230-0045, Japan" /><meta name="citation_title" content="Exploration of Human ORFeome: High-Throughput Preparation of ORF Clones and Efficient Characterization of Their Protein Products" /><meta name="citation_firstpage" content="137" /><meta name="citation_lastpage" content="149" /><meta name="citation_doi" content="10.1093/dnares/dsn004" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="15" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2008/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Genome annotation past, present, and future: how to define an ORF at each locus; Genome Res.;  citation_year=2006;  citation_volume=15;  citation_pages=1777-1786; " /><meta name="citation_reference" content="citation_title=Ensembl 2007; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=D610-D617; " /><meta name="citation_reference" content="citation_title=ORFeome projects: gateway between genomics and omics; Curr. Opin. Chem. Biol.;  citation_year=2004;  citation_volume=8;  citation_pages=20-25; " /><meta name="citation_reference" content="citation_title=The FLEXGene repository: exploiting the fruits of the genome projects by creating a needed resource to face the challenges of the post-genomic era; Arch. Med. Res.;  citation_year=2002;  citation_volume=33;  citation_pages=318-324; " /><meta name="citation_reference" content="citation_title=From genome to proteome: developing expression clone resources for the human genome; Hum. Mol. Genet.;  citation_year=2006;  citation_volume=15;  citation_pages=R31-R43; " /><meta name="citation_reference" content="citation_title=GATEWAY recombinational cloning: application to the cloning of large numbers of open reading frames or ORFeomes; Methods Enzymol.;  citation_year=2000;  citation_volume=328;  citation_pages=575-592; " /><meta name="citation_reference" content="citation_title=Many paths to many clones: a comparative look at high-throughput cloning methods; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=2020-2028; " /><meta name="citation_reference" content="citation_title=High efficiency single step production of expression plasmids from cDNA clones using the Flexi Vector cloning system; Protein Expr. Purif.;  citation_year=2006;  citation_volume=47;  citation_pages=562-570; " /><meta name="citation_reference" content="citation_title=Preparation of a set of expression-ready clones of mammalian long cDNAs encoding large proteins by the ORF trap cloning method; DNA Res.;  citation_year=2005;  citation_volume=12;  citation_pages=257-267; " /><meta name="citation_reference" content="citation_title=Methods in Molecular Biology; citation_publisher=Human Press, Totowa;  citation_year=2007;  citation_volume=356;  citation_pages=195-208; " /><meta name="citation_reference" content="citation_title=A general method for the covalent labeling of fusion proteins with small molecules in vivo; Nat. Biotechnol.;  citation_year=2003;  citation_volume=21;  citation_pages=86-89; " /><meta name="citation_reference" content="citation_title=Adding value to fusion proteins through covalent labeling; Curr. Opin. Biotechnol.;  citation_year=2005;  citation_volume=16;  citation_pages=453-458; " /><meta name="citation_reference" content="citation_title=A variant of yellow fluorescent protein with fast and efficient maturation for cell-biological applications; Nat. Biotechnol.;  citation_year=2002;  citation_volume=20;  citation_pages=87-90; " /><meta name="citation_reference" content="citation_title=Construction and characterization of human brain cDNA libraries suitable for analysis of cDNA clones encoding relatively large proteins; DNA Res.;  citation_year=1997;  citation_volume=4;  citation_pages=53-59; " /><meta name="citation_reference" content="citation_title=Kazusa mammalian cDNA resources: towards functional characterization of KIAA gene products; Brief. Funct. Genomic Proteomic.;  citation_year=2006;  citation_volume=5;  citation_pages=4-7; " /><meta name="citation_reference" content="citation_title=Genetic analysis of the RecE pathway of genetic recombination in Escherichia coli K-12; J. Bacteriol.;  citation_year=1981;  citation_volume=145;  citation_pages=521-532; " /><meta name="citation_reference" content="citation_title=In vivo cloning of PCR products in E. coli; Nucleic Acids Res.;  citation_year=1993;  citation_volume=21;  citation_pages=5192-5197; " /><meta name="citation_reference" content="citation_title=A new logic for DNA engineering using recombination in Escherichia coli; Nature Genet.;  citation_year=1998;  citation_volume=20;  citation_pages=123-128; " /><meta name="citation_reference" content="citation_title=Performance of expression vector, pTD1, in insect cell-free translation system; J. Biosci. Bioengineer.;  citation_year=2006;  citation_volume=102;  citation_pages=69-71; " /><meta name="citation_reference" content="citation_title=Rapid amplification of plasmid and phage DNA using Phi29 DNA polymerase and multiply-primed rolling circle amplification; Genome Res.;  citation_year=2001;  citation_volume=11;  citation_pages=1095-1099; " /><meta name="citation_reference" content="citation_title=Posttranslational membrane attachment and dynamic fatty acylation of a neuronal growth cone protein, GAP-43; J. Cell Biol.;  citation_year=1989;  citation_volume=108;  citation_pages=613-624; " /><meta name="citation_reference" content="citation_title=Induction of nuclear transport with a synthetic peptide homologous to the SV40 T antigen transport signal; Cell;  citation_year=1986;  citation_volume=46;  citation_pages=575-582; " /><meta name="citation_reference" content="citation_title=Molecular cloning of the high affinity calcium-binding protein (calreticulin) of skeletal muscle sarcoplasmic reticulum; J. Biol. Chem.;  citation_year=1989;  citation_volume=264;  citation_pages=21522-21528; " /><meta name="citation_reference" content="citation_title=Measurement of cytosolic, mitochondrial, and Golgi pH in single living cells with green fluorescent proteins; Proc. Natl. Acad. Sci. USA;  citation_year=1998;  citation_volume=95;  citation_pages=6803-6808; " /><meta name="citation_reference" content="citation_title=Chimeric green fluorescent protein as a tool for visualizing subcellular organelles in living cells; Curr. Biol;  citation_year=1995;  citation_volume=5;  citation_pages=635-642; " /><meta name="citation_reference" content="citation_title=The universal protein resource (UniProt); citation_author=The UniProt Consortium; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=D193-D197; " /><meta name="citation_reference" content="citation_title=hORFeome v3.1: a resource of human open reading frames representing over 10,000 human genes; Genomics;  citation_year=2007;  citation_volume=89;  citation_pages=307-315; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/15/3/137/1258594/dsn004.pdf" /><meta name="description" content="Abstract.  In this study, we established new systematic protocols for the preparation of cDNA clones, conventionally termed open reading frame (ORF) clones, sui" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/15/3/137/464067" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/15/3/137/464067" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/15/3/137/464067");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsn004';
    </script>
<title>
	Exploration of Human ORFeome: High-Throughput Preparation of ORF Clones and Efficient Characterization of Their Protein Products | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./464067" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BaRPPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOYWdhc2UsIFRha2FoaXJvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBIdW1hbiBHZW5vbWUgUmVzZWFyY2gsIEthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2EtS2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJZYW1ha2F3YSwgSGlzYXNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgSHVtYW4gR2Vub21lIFJlc2VhcmNoLCBLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLUthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFkb2tvcm8sIFNoaW5pY2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTGFib3JhdG9yeSBvZiBQaGFybWFjb2dlbm9taWNzLCBHcmFkdWF0ZSBTY2hvb2wgb2YgUGhhcm1hY2V1dGljYWwgU2NpZW5jZXMsIENoaWJhIFVuaXZlcnNpdHksIDEtOC0xIElub2hhbmEsIENodW8ta3UsIENoaWJhIDI2MC04Njc1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik5ha2FqaW1hLCBEYWlzdWtlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBIdW1hbiBHZW5vbWUgUmVzZWFyY2gsIEthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2EtS2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJbm91ZSwgU2hpbmljaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEh1bWFuIEdlbm9tZSBSZXNlYXJjaCwgS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1LYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IllhbWFndWNoaSwgS2VpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIENoaWJhIFVuaXZlcnNpdHksIDY0OCBNYXRzdWRvLCBNYXRzdWRvLCBDaGliYSAyNzEtODUxMCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJdG9rYXdhLCBZYXN1aGlkZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBTY2llbmNlLCBUb2hvIFVuaXZlcnNpdHksIDItMi0xIE1peWFtYSwgRnVuYWJhc2hpLCBDaGliYSAyNzQtODUxMCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLaWt1bm8sIFJlaWtvIEYuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBIdW1hbiBHZW5vbWUgUmVzZWFyY2gsIEthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2EtS2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLb2dhLCBIaXNhc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBIdW1hbiBHZW5vbWUgUmVzZWFyY2gsIEthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2EtS2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPaGFyYSwgT3NhbXUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEh1bWFuIEdlbm9tZSBSZXNlYXJjaCwgS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1LYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgUGhhcm1hY29nZW5vbWljcywgR3JhZHVhdGUgU2Nob29sIG9mIFBoYXJtYWNldXRpY2FsIFNjaWVuY2VzLCBDaGliYSBVbml2ZXJzaXR5LCAxLTgtMSBJbm9oYW5hLCBDaHVvLWt1LCBDaGliYSAyNjAtODY3NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJSSUtFTiBSZXNlYXJjaCBDZW50ZXIgZm9yIEFsbGVyZ3kgYW5kIEltbXVub2xvZ3ksIDEtNy0yMiBTdWVoaXJvLCBUc3VydW1pLWt1LCBZb2tvaGFtYSwgS2FuYWdhd2EgMjMwLTAwNDUsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJFeHBsb3JhdGlvbiBvZiBIdW1hbiBPUkZlb21lOiBIaWdoLVRocm91Z2hwdXQgUHJlcGFyYXRpb24gb2YgT1JGIENsb25lcyBhbmQgRWZmaWNpZW50IENoYXJhY3Rlcml6YXRpb24gb2YgVGhlaXIgUHJvdGVpbiBQcm9kdWN0cyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjEzNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMTQ5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNuMDA0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMDgvMDYvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgYW5ub3RhdGlvbiBwYXN0LCBwcmVzZW50LCBhbmQgZnV0dXJlOiBob3cgdG8gZGVmaW5lIGFuIE9SRiBhdCBlYWNoIGxvY3VzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTE3NzctMTc4NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW5zZW1ibCAyMDA3OyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz1ENjEwLUQ2MTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9SRmVvbWUgcHJvamVjdHM6IGdhdGV3YXkgYmV0d2VlbiBnZW5vbWljcyBhbmQgb21pY3M7IEN1cnIuIE9waW4uIENoZW0uIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0yMC0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEZMRVhHZW5lIHJlcG9zaXRvcnk6IGV4cGxvaXRpbmcgdGhlIGZydWl0cyBvZiB0aGUgZ2Vub21lIHByb2plY3RzIGJ5IGNyZWF0aW5nIGEgbmVlZGVkIHJlc291cmNlIHRvIGZhY2UgdGhlIGNoYWxsZW5nZXMgb2YgdGhlIHBvc3QtZ2Vub21pYyBlcmE7IEFyY2guIE1lZC4gUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTMxOC0zMjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZyb20gZ2Vub21lIHRvIHByb3Rlb21lOiBkZXZlbG9waW5nIGV4cHJlc3Npb24gY2xvbmUgcmVzb3VyY2VzIGZvciB0aGUgaHVtYW4gZ2Vub21lOyBIdW0uIE1vbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9UjMxLVI0MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R0FURVdBWSByZWNvbWJpbmF0aW9uYWwgY2xvbmluZzogYXBwbGljYXRpb24gdG8gdGhlIGNsb25pbmcgb2YgbGFyZ2UgbnVtYmVycyBvZiBvcGVuIHJlYWRpbmcgZnJhbWVzIG9yIE9SRmVvbWVzOyBNZXRob2RzIEVuenltb2wuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTMyODsgIGNpdGF0aW9uX3BhZ2VzPTU3NS01OTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hbnkgcGF0aHMgdG8gbWFueSBjbG9uZXM6IGEgY29tcGFyYXRpdmUgbG9vayBhdCBoaWdoLXRocm91Z2hwdXQgY2xvbmluZyBtZXRob2RzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTIwMjAtMjAyODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaCBlZmZpY2llbmN5IHNpbmdsZSBzdGVwIHByb2R1Y3Rpb24gb2YgZXhwcmVzc2lvbiBwbGFzbWlkcyBmcm9tIGNETkEgY2xvbmVzIHVzaW5nIHRoZSBGbGV4aSBWZWN0b3IgY2xvbmluZyBzeXN0ZW07IFByb3RlaW4gRXhwci4gUHVyaWYuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTQ3OyAgY2l0YXRpb25fcGFnZXM9NTYyLTU3MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJlcGFyYXRpb24gb2YgYSBzZXQgb2YgZXhwcmVzc2lvbi1yZWFkeSBjbG9uZXMgb2YgbWFtbWFsaWFuIGxvbmcgY0ROQXMgZW5jb2RpbmcgbGFyZ2UgcHJvdGVpbnMgYnkgdGhlIE9SRiB0cmFwIGNsb25pbmcgbWV0aG9kOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTI1Ny0yNjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1ldGhvZHMgaW4gTW9sZWN1bGFyIEJpb2xvZ3k7IGNpdGF0aW9uX3B1Ymxpc2hlcj1IdW1hbiBQcmVzcywgVG90b3dhOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM1NjsgIGNpdGF0aW9uX3BhZ2VzPTE5NS0yMDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ2VuZXJhbCBtZXRob2QgZm9yIHRoZSBjb3ZhbGVudCBsYWJlbGluZyBvZiBmdXNpb24gcHJvdGVpbnMgd2l0aCBzbWFsbCBtb2xlY3VsZXMgaW4gdml2bzsgTmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTg2LTg5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZGRpbmcgdmFsdWUgdG8gZnVzaW9uIHByb3RlaW5zIHRocm91Z2ggY292YWxlbnQgbGFiZWxpbmc7IEN1cnIuIE9waW4uIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9NDUzLTQ1ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSB2YXJpYW50IG9mIHllbGxvdyBmbHVvcmVzY2VudCBwcm90ZWluIHdpdGggZmFzdCBhbmQgZWZmaWNpZW50IG1hdHVyYXRpb24gZm9yIGNlbGwtYmlvbG9naWNhbCBhcHBsaWNhdGlvbnM7IE5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz04Ny05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIGh1bWFuIGJyYWluIGNETkEgbGlicmFyaWVzIHN1aXRhYmxlIGZvciBhbmFseXNpcyBvZiBjRE5BIGNsb25lcyBlbmNvZGluZyByZWxhdGl2ZWx5IGxhcmdlIHByb3RlaW5zOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9NTMtNTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUthenVzYSBtYW1tYWxpYW4gY0ROQSByZXNvdXJjZXM6IHRvd2FyZHMgZnVuY3Rpb25hbCBjaGFyYWN0ZXJpemF0aW9uIG9mIEtJQUEgZ2VuZSBwcm9kdWN0czsgQnJpZWYuIEZ1bmN0LiBHZW5vbWljIFByb3Rlb21pYy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTQtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBhbmFseXNpcyBvZiB0aGUgUmVjRSBwYXRod2F5IG9mIGdlbmV0aWMgcmVjb21iaW5hdGlvbiBpbiBFc2NoZXJpY2hpYSBjb2xpIEstMTI7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODE7ICBjaXRhdGlvbl92b2x1bWU9MTQ1OyAgY2l0YXRpb25fcGFnZXM9NTIxLTUzMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW4gdml2byBjbG9uaW5nIG9mIFBDUiBwcm9kdWN0cyBpbiBFLiBjb2xpOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz01MTkyLTUxOTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbmV3IGxvZ2ljIGZvciBETkEgZW5naW5lZXJpbmcgdXNpbmcgcmVjb21iaW5hdGlvbiBpbiBFc2NoZXJpY2hpYSBjb2xpOyBOYXR1cmUgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MTIzLTEyODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGVyZm9ybWFuY2Ugb2YgZXhwcmVzc2lvbiB2ZWN0b3IsIHBURDEsIGluIGluc2VjdCBjZWxsLWZyZWUgdHJhbnNsYXRpb24gc3lzdGVtOyBKLiBCaW9zY2kuIEJpb2VuZ2luZWVyLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDI7ICBjaXRhdGlvbl9wYWdlcz02OS03MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmFwaWQgYW1wbGlmaWNhdGlvbiBvZiBwbGFzbWlkIGFuZCBwaGFnZSBETkEgdXNpbmcgUGhpMjkgRE5BIHBvbHltZXJhc2UgYW5kIG11bHRpcGx5LXByaW1lZCByb2xsaW5nIGNpcmNsZSBhbXBsaWZpY2F0aW9uOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTEwOTUtMTA5OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG9zdHRyYW5zbGF0aW9uYWwgbWVtYnJhbmUgYXR0YWNobWVudCBhbmQgZHluYW1pYyBmYXR0eSBhY3lsYXRpb24gb2YgYSBuZXVyb25hbCBncm93dGggY29uZSBwcm90ZWluLCBHQVAtNDM7IEouIENlbGwgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODk7ICBjaXRhdGlvbl92b2x1bWU9MTA4OyAgY2l0YXRpb25fcGFnZXM9NjEzLTYyNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5kdWN0aW9uIG9mIG51Y2xlYXIgdHJhbnNwb3J0IHdpdGggYSBzeW50aGV0aWMgcGVwdGlkZSBob21vbG9nb3VzIHRvIHRoZSBTVjQwIFQgYW50aWdlbiB0cmFuc3BvcnQgc2lnbmFsOyBDZWxsOyAgY2l0YXRpb25feWVhcj0xOTg2OyAgY2l0YXRpb25fdm9sdW1lPTQ2OyAgY2l0YXRpb25fcGFnZXM9NTc1LTU4MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGNsb25pbmcgb2YgdGhlIGhpZ2ggYWZmaW5pdHkgY2FsY2l1bS1iaW5kaW5nIHByb3RlaW4gKGNhbHJldGljdWxpbikgb2Ygc2tlbGV0YWwgbXVzY2xlIHNhcmNvcGxhc21pYyByZXRpY3VsdW07IEouIEJpb2wuIENoZW0uOyAgY2l0YXRpb25feWVhcj0xOTg5OyAgY2l0YXRpb25fdm9sdW1lPTI2NDsgIGNpdGF0aW9uX3BhZ2VzPTIxNTIyLTIxNTI4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZWFzdXJlbWVudCBvZiBjeXRvc29saWMsIG1pdG9jaG9uZHJpYWwsIGFuZCBHb2xnaSBwSCBpbiBzaW5nbGUgbGl2aW5nIGNlbGxzIHdpdGggZ3JlZW4gZmx1b3Jlc2NlbnQgcHJvdGVpbnM7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTk1OyAgY2l0YXRpb25fcGFnZXM9NjgwMy02ODA4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGltZXJpYyBncmVlbiBmbHVvcmVzY2VudCBwcm90ZWluIGFzIGEgdG9vbCBmb3IgdmlzdWFsaXppbmcgc3ViY2VsbHVsYXIgb3JnYW5lbGxlcyBpbiBsaXZpbmcgY2VsbHM7IEN1cnIuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTYzNS02NDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSB1bml2ZXJzYWwgcHJvdGVpbiByZXNvdXJjZSAoVW5pUHJvdCk7IGNpdGF0aW9uX2F1dGhvcj1UaGUgVW5pUHJvdCBDb25zb3J0aXVtOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz1EMTkzLUQxOTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPWhPUkZlb21lIHYzLjE6IGEgcmVzb3VyY2Ugb2YgaHVtYW4gb3BlbiByZWFkaW5nIGZyYW1lcyByZXByZXNlbnRpbmcgb3ZlciAxMCwwMDAgaHVtYW4gZ2VuZXM7IEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTg5OyAgY2l0YXRpb25fcGFnZXM9MzA3LTMxNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE1LzMvMTM3LzEyNTg1OTQvZHNuMDA0LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBJbiB0aGlzIHN0dWR5LCB3ZSBlc3RhYmxpc2hlZCBuZXcgc3lzdGVtYXRpYyBwcm90b2NvbHMgZm9yIHRoZSBwcmVwYXJhdGlvbiBvZiBjRE5BIGNsb25lcywgY29udmVudGlvbmFsbHkgdGVybWVkIG9wZW4gcmVhZGluZyBmcmFtZSAoT1JGKSBjbG9uZXMsIHN1aSIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDY0MDY3JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZDffv2bROM3GsmE/CWxW3j0+GTzn" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f3%2f137%2f464067" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f3%2f137%2f464067" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=18181" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/15/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/3/1/m_cover.gif?Expires=2147483647&amp;Signature=Wofi61EvCcdWxJvtysqPZG0U~KWWJOmThdMZDP-AglxAwDtnmCnY2MpChppyQDDpv6NdMaeLAZ~nLGaX7cJuWBVCGvKy3i2prPhQHBCH-vHV3CnTYh7LKJ6XOvu-5EP8bX5lf-70Wga4tfffe84AGB6pX1KlziPK7gdEs5QnVkZBbwLfiRPt-I1COdPZHhK91Imi3IuGiPl9ftt~EYfFKFyDk2nT8pASRTr325Ma~vy8IX6K~kWiKpA09pXB8RYEn5ztSYMyc20h7L3oQyBZ3j8qfpO0VRwhgradKgQz7QarN5Dl5qIio9OjPV0kSCdBe3E3RBSZsEr31pWrM0u5RQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/15/3">
                        <div class="volume trailing-comma">Volume 15</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2008        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6967023">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6967023">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6967025">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6967025">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6967028">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6967028">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6967041">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6967041">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6967059">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6967059">Supplementary material</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6967061">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6967061">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6967063">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6967063">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6967065">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6967065">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/15/3/123/464521">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/15/3/151/464397">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Exploration of Human ORFeome: High-Throughput Preparation of ORF Clones and Efficient Characterization of Their Protein Products
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takahiro Nagase</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Human Genome Research</div>, <div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 438-52-3930. Fax. <div class="fax">+81 438-52-3931</div>. E-mail: <a href="mailto:nagase@kazusa.or.jp" target="_blank">nagase@kazusa.or.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takahiro+Nagase">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nagase T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nagase T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takahiro Nagase</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hisashi Yamakawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Human Genome Research</div>, <div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hisashi+Yamakawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamakawa H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yamakawa H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hisashi Yamakawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shinichi Tadokoro</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Laboratory of Pharmacogenomics, Graduate School of Pharmaceutical Sciences</div>, <div class="insititution">Chiba University</div>, <div class="addr-line">1-8-1 Inohana, Chuo-ku, Chiba 260-8675</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shinichi+Tadokoro">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tadokoro S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tadokoro S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shinichi Tadokoro</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Daisuke Nakajima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Human Genome Research</div>, <div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Daisuke+Nakajima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakajima D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakajima D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Daisuke Nakajima</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shinichi Inoue</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Human Genome Research</div>, <div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shinichi+Inoue">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Inoue S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Inoue S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shinichi Inoue</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kei Yamaguchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Graduate School of Science and Technology</div>, <div class="insititution">Chiba University</div>, <div class="addr-line">648 Matsudo, Matsudo, Chiba 271-8510</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kei+Yamaguchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamaguchi K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yamaguchi K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kei Yamaguchi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yasuhide Itokawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Graduate School of Science</div>, <div class="insititution">Toho University</div>, <div class="addr-line">2-2-1 Miyama, Funabashi, Chiba 274-8510</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yasuhide+Itokawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Itokawa Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Itokawa Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yasuhide Itokawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Reiko F. Kikuno</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Human Genome Research</div>, <div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Reiko+F.+Kikuno">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kikuno R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kikuno R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Reiko F. Kikuno</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hisashi Koga</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Human Genome Research</div>, <div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hisashi+Koga">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Koga H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Koga H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hisashi Koga</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Osamu Ohara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Human Genome Research</div>, <div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Laboratory of Pharmacogenomics, Graduate School of Pharmaceutical Sciences</div>, <div class="insititution">Chiba University</div>, <div class="addr-line">1-8-1 Inohana, Chuo-ku, Chiba 260-8675</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">5</div><div class="insititution">RIKEN Research Center for Allergy and Immunology</div>, <div class="addr-line">1-7-22 Suehiro, Tsurumi-ku, Yokohama, Kanagawa 230-0045</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Osamu+Ohara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ohara O">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ohara O&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Osamu Ohara</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 15, Issue 3, 1 June 2008, Pages 137–149, <a href='https://doi.org/10.1093/dnares/dsn004'>https://doi.org/10.1093/dnares/dsn004</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">02 March 2008</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">10 January 2008</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">29 January 2008</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="464067" href="/dnaresearch/article-pdf/15/3/137/1258594/dsn004.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Takahiro Nagase, Hisashi Yamakawa, Shinichi Tadokoro, Daisuke Nakajima, Shinichi Inoue, Kei Yamaguchi, Yasuhide Itokawa, Reiko F. Kikuno, Hisashi Koga, Osamu Ohara;  Exploration of Human ORFeome: High-Throughput Preparation of ORF Clones and Efficient Characterization of Their Protein Products, <em>DNA Research</em>, Volume 15, Issue 3, 1 June 2008, Pages 137–149, <a href='https://doi.org/10.1093/dnares/dsn004'>https://doi.org/10.1093/dnares/dsn004</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/464067?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/464067?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/464067?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/464067?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/464067?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/464067?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/464067?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Exploration%20of%20Human%20ORFeome%3A%20High-Throughput%20Preparation%20of%20ORF%20Clones%20and%20Efficient%20Characterization%20of%20Their%20Protein%20Products&amp;publicationDate=2008-03-02&amp;volumeNum=15&amp;issueNum=3&amp;author=Nagase%2C%20Takahiro%3B%20Yamakawa%2C%20Hisashi&amp;startPage=137&amp;endPage=149&amp;contentId=10.1093%2Fdnares%2Fdsn004&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f3%2f137%2f464067" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6967023 id="6967023" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>In this study, we established new systematic protocols for the preparation of cDNA clones, conventionally termed open reading frame (ORF) clones, suitable for characterization of their gene products by adopting a restriction-enzyme-assisted cloning method using the Flexi<sup>®</sup> cloning system. The system has following advantages: (1) preparation of ORF clones and their transfer into other vectors can be achieved efficiently and at lower cost; (2) the system provides a seamless connection to the versatile HaloTag<sup>®</sup> labeling system, in which a single fusion tag can be used for various proteomic analyses; and (3) the resultant ORF clones show higher expression levels both <em>in vitro</em> and <em>in vivo</em>. With this system, we prepared ORF clones encoding 1929 human genes and characterized the HaloTag-fusion proteins of its subset that are expressed <em>in vitro</em> or in mammalian cells. Results thus obtained have demonstrated that our Flexi<sup>®</sup> ORF clones are efficient for the production of HaloTag-fusion proteins that can provide a new versatile set for a variety of functional analyses of human genes.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="cDNA">cDNA</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="HaloTag">HaloTag</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="proteomics">proteomics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Flexi cloning&quot;">Flexi cloning</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="ORFeome">ORFeome</a></div>                            <h2 scrollto-destination=6967025 id="6967025" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The entire human genome sequence has allowed us to create a provisional catalog of human transcripts and proteins by a combination of computational predictions and information from experimentally collected cDNA sequences.<sup><a reveal-id="DSN004C1" data-open="DSN004C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Although approximately 23 000 protein-coding genes have been assigned to the human genome by the Ensembl database (NCBI built 36, Ensembl 48; <a class="link link-uri" href="http://www.ensembl.org/Homo_sapiens/index.html">http://www.ensembl.org/Homo_sapiens/index.html</a>),<sup><a reveal-id="DSN004C2" data-open="DSN004C2" class="link link-ref link-reveal xref-bibr">2</a></sup> the functional roles of the respective gene products remains to be experimentally explored. Because of the importance of these studies, many groups have started to collect a set of protein-coding sequences, conventionally termed open reading frames (ORFs), in a genome-wide manner, so-called ‘ORFeome cloning’. ORF clones serve as versatile reagents for the functional and structural studies of proteins.<sup><a reveal-id="DSN004C3  DSN004C5" data-open="DSN004C3  DSN004C5" class="link link-ref link-reveal xref-bibr">3–5</a></sup> Therefore, various systems have been developed to enable efficient large-scale cloning and expression of ORF clones. Currently available systems rely on either site-specific recombinases or rare-cutting restriction enzymes for the transfer of ORFs between expression vectors. These systems allow the correct orientation and reading frame to be maintained after transfer.<sup><a reveal-id="DSN004C6  DSN004C8" data-open="DSN004C6  DSN004C8" class="link link-ref link-reveal xref-bibr">6–8</a></sup> The Gateway and Creator cloning systems, which are representative of the recombinase-based systems, transfer ORFs by λ-att recombination using the 25-bp <em>att</em>B site, and Cre-<em>loxP</em> recombination using the 34-bp lox-P site upstream of the ORF, respectively.<sup><a reveal-id="DSN004C7" data-open="DSN004C7" class="link link-ref link-reveal xref-bibr">7</a></sup> On the other hand, restriction-enzyme-based systems are less frequently used at present, although a commercial system has recently emerged that is as efficient as those based on recombinases. The Flexi<sup>®</sup> cloning system can manipulate ORFs using rare-cutting restriction enzymes, <em>Sgf</em>I and <em>Pme</em>I, after the addition of these 8-bp restriction sites to the flanking 5′ and 3′ ends of the ORFs, respectively.<sup><a reveal-id="DSN004C8" data-open="DSN004C8" class="link link-ref link-reveal xref-bibr">8</a></sup> We previously prepared a set of ORF clones for human KIAA genes using a Gateway-type vector.<sup><a reveal-id="DSN004C9" data-open="DSN004C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Furthermore, we created the ORF clones as fusion proteins with green-fluorescent protein to analyze the subcellular localization of the proteins in cultured cells and to evaluate the <em>in vitro</em> expression of ORF clones.<sup><a reveal-id="DSN004C9" data-open="DSN004C9" class="link link-ref link-reveal xref-bibr">9</a></sup> However, when we wish to carry out alternative functional studies beyond bioimaging, the Gateway system requires re-construction of appropriate expression plasmids for the creation of differently tagged proteins. This creates a serious bottleneck in the use of our large ORF clone set for functional genomics analysis. Thus, we started to look for a more versatile tag system that would allow us to carry out both bioimaging and biochemical experiments using a single tag. Within this context, we considered a recently emerging technology, commercially designated as HaloTag<sup>®</sup> technology.<sup><a reveal-id="DSN004C10" data-open="DSN004C10" class="link link-ref link-reveal xref-bibr">10</a></sup> This technology is quite attractive since HaloTag can be used not only for bioimaging, but also for various proteomic applications involving fusion protein immobilization. Although similar technologies have been developed, such as SNAP-tag which uses a modified human O<sup><a reveal-id="DSN004C6" data-open="DSN004C6" class="link link-ref link-reveal xref-bibr">6</a></sup>-alkylguanine DNA alkyl transferase as the tag,<sup><a reveal-id="DSN004C11" data-open="DSN004C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSN004C12" data-open="DSN004C12" class="link link-ref link-reveal xref-bibr">12</a></sup> we preferred HaloTag<sup>®</sup> technology because we felt it may be more suitable for studies in mammals. Haloalkane dehalogenase, the enzyme from which HaloTag is derived, does not exist in mammalian cells, unlike O<sup><a reveal-id="DSN004C6" data-open="DSN004C6" class="link link-ref link-reveal xref-bibr">6</a></sup>-alkylguanine DNA alkyl transferase. Since HaloTag<sup>®</sup> technology was available in the Flexi<sup>®</sup> cloning system, we were strongly motivated to examine whether the Flexi<sup>®</sup> cloning method was suitable for preparation of a large number of ORF clones, and whether it would be useful for downstream functional characterization.</p><p>In this study, we prepared nearly 2000 ORF clones using the Flexi<sup>®</sup> cloning method, and the resultant ORF clones were examined with respect to production in cell-free protein synthesis systems as HaloTag-fusion proteins. The results indicated that the Flexi<sup>®</sup> cloning method was as efficient as the Gateway cloning method and that the Flexi ORF clones consistently produced equal or larger amounts of HaloTag-fusion proteins than Gateway clones in <em>in vitro</em> protein production systems. We also examined the subcellular localizations of 40 HaloTag-fusion proteins in HEK293 cells and confirmed that the results were equivalent to those of the corresponding Monster Green<sup>®</sup> Fluorescent Protein (MGFP)-fusion proteins. Taken together, these results indicate that the Flexi ORF clone set can perform efficient functional analyses of human genes and provide an alternative resource for the human ORFeome project.</p>                            <h2 scrollto-destination=6967028 id="6967028" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=6967029 id="6967029" class="section-title" data-legacy-id=s2a>2.1. Materials</h3>
<p>pTD1 plasmid was obtained from Shimadzu (Kyoto, Japan). pF1Kof vector, which was constructed by flipping the <em>Acc</em>III–<em>Age</em>I 832-bp DNA fragment including the origin of replication in the pF1KT7 (Promega, Madison, USA) vector, was kindly provided by Promega Corporation. pF3AHT vector was constructed by inserting the blunt-ended <em>Eco</em>ICRI–<em>Sal</em>I 946-bp DNA fragment from the pFC8A (HT) vector (Promega) into the <em>Eco</em>ICRI site of the pF3A WG (BYDV) (Promega) vector backbone in the appropriate order for production of a C-terminal HaloTag-fusion protein. pFC8A (MGFP) was constructed by replacing the HaloTag gene with the MGFP gene from the phMGFP vector (Promega). For construction of HaloTag-fusion KIAA protein expression clones in the pTD2-Gateway and pTD2-Flexi vectors, KIAA ORFs with the HaloTag sequence were extracted from pF3AHT ORF clones. The ORFs for the synthetic firefly <em>luciferase</em> gene (<em>luc2</em>), HaloTag, and Venus were obtained from pGL4.10[<em>luc2</em>], pFC8A vector (Promega), and (EYFP-F46L/V68L/M153T/V163A/S175G)/pCS2 kindly provided by Dr. Atsushi Miyawaki (Brain Science Institute, RIKEN),<sup><a reveal-id="DSN004C13" data-open="DSN004C13" class="link link-ref link-reveal xref-bibr">13</a></sup> respectively, by PCR and were subcloned into the pF1KT7 Flexi vector. Each ORF was transferred to seven different kinds of modified pTD2 expression vectors whose cloning sites were flanked at both sides with specified sequences. The D5 vector, which carried a 10-bp sequence (AATCGAATTC) in place of the 5′-polyhedrin enhancer sequence, was also used in the preparation of the <em>luc2</em> expression plasmid. pFC8KHT-Memb, pFC8KHT-NLS, pFC8KHT-ER, pFC8KHT-Golgi, and pFC8KHT-Mito were constructed by ligation of DNA fragments containing localization signal sequences, with the pFC8K (HaloTag) vector digested by <em>Sgf</em>I and <em>Eco</em>ICRI (Carboxy Flexi<sup>®</sup> System, Promega). The localization signal fragments (abbreviated as Memb, NLS, ER, Golgi, and Mito) were amplified by PCR from the following expression clones (BD, Franklin Lakes, USA) with each primer appended by <em>Sgf</em>I or <em>Pme</em>I sites at the 5′ end: Memb, a membrane-localized signal from N-terminal sequence of neuromodulin (pEYFP-Mem); NLS, a nuclear localization signal from a triple repeat of NLS from SV40 large T-antigen (pDsRed2-Nuc); ER, an endoplasmic reticulum (ER) targeting sequence from calreticulin (pDsRed2-ER); Golgi, a Golgi-localized signal from beta 1,4-galactosyltransferase (pEYFP-Golgi); and Mito, a mitochondrial targeting sequence from subunit VIII of cytochrome c oxidase (pDsRed2-Mito). HaloTag Cy5 ligand was prepared by the coupling of HaloTag Amine (O4) ligand (Promega) and Cy5 Mono-Reactive Dye Protein Array Grade (Amersham Biosciences, Piscataway, USA) according to the manufacturer's instructions. Anti-HaloTag IgG was generously provided by Promega Corporation. <em>Escherichia coli</em> strain JC8679 (<em>recB21, recC22, sbcA23, thr-1, leuB6, phi-1, lacY1, galK2, ara-14, xyl-5, mtl-1, proA2, his-4, argE3, rpsL31, tsx-33, supE44, his-328</em>) was obtained from the Health Science Research Resources Bank (HSRRB) of the Japan Health Sciences Foundation.</p>                            <h3 scrollto-destination=6967031 id="6967031" class="section-title" data-legacy-id=s2b>2.2. Preparation of Flexi ORF clones</h3>
<p>The ORF of interest was amplified from 1 to 4 ng of a plasmid DNA using 0.5 units of PrimeSTAR™ HS DNA polymerase (Takara Bio, Shiga, Japan) in a 20-μL PCR reaction mixture that included 0.2 mM each of four dNTPs and 0.5 μM each of the gene-specific primers (5′-CCCCGCGATCGCCATG N<sub>17</sub>-3′ and 5′-CCCCGTTTAAAC N<sub>20</sub>-3′, where N<sub>17</sub> indicates a 17 nt-sense sequence downstream from the start methionine codon of ORF and N<sub>20</sub> indicates a 20 nt-anti-sense sequence upstream from the stop codon), under the following PCR conditions: 95°C for 2 min/30 cycles of 98°C for 10 s and 68°C for 3 min/68°C for 5 min. In some cases, 0.5 units of KOD-Plus-DNA Polymerase (Toyobo, Osaka, Japan) was used in a 25-μL PCR reaction mixture that included 0.2 mM each of four dNTPs, 1 mM MgCl<sub>2</sub> and 0.3 mM each of the gene-specific primers under the following PCR conditions: 94°C for 2 min/30 cycles of 98°C for 10 s, 55°C for 30 s/68°C for 3 min. One-third of the resultant amplified PCR products was digested by <em>Sgf</em>I and <em>Pme</em>I in a 10‐μL reaction mixture of Flexi<sup>®</sup> buffer with 1× Flexi<sup>®</sup> Enzyme Blend (<em>Sgf</em>I and <em>Pme</em>I) (Promega) for 30 min at 37°C after purification of the PCR products using Wizard SV 96 PCR Clean-Up System (Promega). After inactivation of the restriction enzymes by incubation for 20 min at 65°C, one-sixth of the digested PCR product was ligated with 25 ng of the <em>Sgf</em>I- and <em>Pme</em>I-digested pF1K T7 vector in a 10‐μL reaction of Flexi<sup>®</sup> Ligase buffer with 10 units of T4 DNA Ligase (Promega) for 60 min at 25°C. One microliter of the ligation reaction was used to transform 10 μL of JM109 Competent Cells (&gt;10<sup>8</sup> cfu/μg, Promega), and the transformants were selected on LB agar plates containing 50 μg/mL of kanamycin. A randomly chosen clone was first examined using the CloneChecker™ System (Invitrogen, Carlsbad, USA) by estimating the size of the supercoiled plasmid DNA. For ORFs cloned by the PCR cloning method, entire sequences were determined with the BigDye Terminator v3.1 Cycle Sequencing kit (Applied Biosystems, Foster City, USA), with sequencing primers designed every approximate 400 bp in a single strand of ORF. The sequence data were obtained with ABI PRISM 3700 or 3130, and analyzed with SEQUENCHER™ sequence assembly software (Gene Codes Co., Ann Arbor, USA). For ORF Trap cloning, the ORF of interest was transferred to pF1Kof vector using a homologous recombination system in <em>E. coli</em> (JC8679), according to the method previously described.<sup><a reveal-id="DSN004C9" data-open="DSN004C9" class="link link-ref link-reveal xref-bibr">9</a></sup> For Flexi<sup>®</sup> Vector cloning using the pENT entry clones we had previously constructed,<sup><a reveal-id="DSN004C9" data-open="DSN004C9" class="link link-ref link-reveal xref-bibr">9</a></sup> the ORF of interest was recovered by digestion with <em>Bst</em>BI and <em>Sna</em>BI restriction endonucleases whose recognition sites were located upstream and downstream of the ORF, respectively, in the vector portion. It was then inserted between the <em>Bst</em>BI and <em>Sna</em>BI sites of the pSP73Flexi-1 vector, which contains <em>Sgf</em>I and <em>Pme</em>I sites upstream and downstream of the <em>Bst</em>BI and <em>Sna</em>BI sites, respectively. After cloning the ORFs into the pSP73Flexi-1 vector, they were cut with <em>Sgf</em>I and <em>Pme</em>I restriction endonucleases and then re-cloned between the <em>Sgf</em>I and <em>Pme</em>I sites of the pF1K vector.</p>                            <h3 scrollto-destination=6967033 id="6967033" class="section-title" data-legacy-id=s2c>2.3. In vitro protein production</h3>
<p>For <em>in vitro</em> translation systems using Wheat Germ Extract Plus (WGEP) (Promega) and a TransDirect insect cell lysate (TD) (Shimadzu, Japan), RNA synthesis was performed using the T7 RiboMAX Express Large Scale RNA Production System (Promega) with linearized template DNAs, according to the supplier's instructions. After purification with the RNeasy Mini Kit (QIAGEN, Hilden, Germany), RNAs were concentrated by ethanol precipitation and dissolved in 15 μL of DEPC-treated water. <em>In vitro</em> translation reactions were performed in a reaction volume of 10 μL using 2.4 and 3.2 μg of the RNA for the WGEP and the TD systems, respectively, following the supplier's recommendation unless otherwise stated. For the T<span class="small-caps">n</span>T SP6 High-Yield Protein Expression (HYPE) System (Promega), approximately 400 ng of TempliPhi-amplified pF3AHT HaloTag-fusion protein expression clones were used as a template in a 10-μL reaction, according to the manufacturer's instructions. For production of radiolabeled proteins, 2.86 µCi of <sup>35</sup>S-labeled Redivue Promix (a mixture of <span class="small-caps">l</span>-[<sup>35</sup>S]methionine and <span class="small-caps">l</span>-[<sup>35</sup>S] cysteine) (GE Healthcare, USA) was added in the reaction. The TempliPhi (GE Healthcare) reaction was conducted by denaturation of 10–30 ng of DNAs with 5 μL of Denature buffer for 3 min at 95ºC followed by polymerization for 16 h at 30ºC. For the HaloTag tetramethylrhodamine (TMR) ligand labeling of the <em>in vitro</em> produced HaloTag-harboring proteins, 2.3 μL of 1 μM HaloTag-TMR ligand (Promega) was added after the <em>in vitro</em> reaction (10 μL) and the mixture was incubated for 5–10 min at room temperature. After the addition of 4.2 μL of 4× Laemmli's SDS sample buffer, the products were resolved on a MDG-267 Real Gel Plate (concentration gradient: 5–10%; Biocraft, Japan) by polyacrylamide gel electrophoresis in the presence of SDS (SDS–PAGE).</p>                            <h3 scrollto-destination=6967035 id="6967035" class="section-title" data-legacy-id=s2d>2.4. Examination of the effects of the flanking sequences of ORF on in vitro production of luciferase, Venus, and HaloTag</h3>
<p>For <em>in vitro luc2</em> production, 1.4 μg of RNAs were used for WGEP and 3.8 μg for TD in a reaction volume of 12 μL. The activities of <em>luc2</em> in an aliquot of the reaction mixture (10 μL) were assayed using the Dual-Glo Luciferase Assay System (Promega) according to the supplier's instructions. Luminescence was detected in a GloMAX 96 Microplate Luminometer (Promega). <em>In vitro</em> syntheses of Venus proteins were performed in 20 μL using 4.8 and 6.4 μg of the template RNA in the WGEP and the TD systems, respectively, after removing turbidity by centrifugation at 15 000 rpm for 2 min. Fluorescent signals were detected in the Applied Biosystems 7500 Fast Real-Time PCR System after 2 and 5 h reactions in the WGEP and TD systems, respectively. HaloTag-TMR ligand binding was monitored for quantification of the active HaloTag protein. Two microliters of 5 μM HaloTag-TMR ligand was added to an 8-μL reaction mixture, and the mixture was incubated at 25°C for 5 min. The binding reaction was stopped by adding 2 μL of 6× Laemmli's SDS sample buffer and then boiling at 100°C for 5 min. The solution (7.5 μL) was analyzed on 12.5% SDS–PAGE. Fluorescent signals from TMR were detected and quantified with FLA-3000 (Fujifilm, Tokyo, Japan) and MultiGauge image analyzing software (Fujifilm).</p>                            <h3 scrollto-destination=6967037 id="6967037" class="section-title" data-legacy-id=s2e>2.5. Western blotting</h3>
<p>For Western blot analysis of HaloTag-fusion proteins, the proteins separated by SDS–PAGE were electrophoretically transferred onto a PVDF membrane (FluoroTrans W; PALL, Portsmouth, UK) using the transfer buffer [25 mM Tris–Cl (pH 8.3), 192 mM glycine, 20% (V/V) methanol] with the BIOCRAFT BE-300 semidry transfer device (Biocraft, Tokyo, Japan). HaloTag-fusion proteins were detected as follows. The resultant PVDF membrane was washed with TBS [20 mM Tris–Cl (pH 7.5) and 150 mM NaCl] including 0.05% Tween-20 (TBST) for 10 min with gentle agitation. After pre-incubation of the membrane with TBST containing 5% skim milk for 60 min, the membrane was incubated with 1:30 000 rabbit anti-HaloTag IgG antibody (Promega) in TBST containing 1% skim milk for 60 min. After washing with TBST for 5 min four times, the membrane was further incubated with 1:30 000 horseradish-peroxidase-conjugated anti-rabbit IgG antibody (Promega) in TBST containing 1% skim milk for 60 min. After washing the membrane with TBST, HaloTag-fusion proteins were finally detected using ECL plus (GE Healthcare) according to the manufacturer's instructions. The luminescent images were recorded by a Luminescent Image Analyzer LAS3000 (Fujifilm). MagicMark XP Western Protein Standard (Invitrogen) was used for estimation of the apparent molecular masses of the HaloTag-fusion proteins.</p>                            <h3 scrollto-destination=6967039 id="6967039" class="section-title" data-legacy-id=s2f>2.6. Examination of the subcellular localization of HaloTag-fusion proteins</h3>
<p>The human embryonic kidney (HEK) 293 cell line was obtained from HSRRB of the Japan Health Sciences Foundation. HEK293 and COS-7 cell lines were grown in DMEM supplemented with 10% of Tet System Approved Fetal Bovine Serum (BD) and 1× Antibiotic–Antimycotic reagent (100 U/mL penicillin + 100 μg/mL streptomycin + 0.25 μg/mL amphotericin B; Invitrogen). HEK293 and COS-7 were transfected with expression clones using FuGENE6 Transfection Reagent (Roche Diagnostics, Basel, Switzerland) in an 8-well chambered coverglass (Thermo Fisher Scientific, Rochester, USA) according to the supplier's instructions. Cells (200 μL, 4 × 10<sup>4</sup>) were plated 24 h before transfection. After 16–24 h of transfection, HaloTag-fusion proteins and DNA were labeled with medium with 1 μM of HaloTag-TMR ligand and Hoechst (33342, 3.3 μg/mL, final concentration; Sigma-Aldrich, St. Louis, USA) for 15 min in a CO<sub>2</sub> (5%) incubator at 37ºC. After washing four times with 300 μL of medium, the cells were incubated for 0.5–24 h in a CO<sub>2</sub> incubator and the subcellular localization of HaloTag- or MGFP-fusion proteins were observed with a Biozero fluorescence microscopy system (Keyence, Osaka, Japan) with 605/55 and 535/50 nm filters. Photos labeled with Hoechst33342 were merged with those of MGFP- or HaloTag-fusion proteins and the subcellular localizations of MGFP- and HaloTag-fusion proteins were compared.</p>                            <h2 scrollto-destination=6967041 id="6967041" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=6967042 id="6967042" class="section-title" data-legacy-id=s3a>3.1. Construction of Flexi ORF clones</h3>
<p>Led by the motivation described in Section 1, we have initiated preparation of Flexi<sup>®</sup> Vector ORF clones using human cDNA clones accumulated by the Kazusa cDNA project.<sup><a reveal-id="DSN004C14" data-open="DSN004C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSN004C15" data-open="DSN004C15" class="link link-ref link-reveal xref-bibr">15</a></sup> So far, 1929 Flexi ORF clones have been constructed in a pF1K vector format. The list of the Flexi ORF clones is accessible at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org">www.dnaresearch.oxfordjournals.org</a> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). These Flexi ORF clones contained relatively large ORFs derived from 1163 KIAA and 766 known genes, the average sizes of which are 2.8 and 2.4 kb, respectively. ORFs of 341 pF1K clones were PCR-cloned using PrimeStar HS or KOD plus high-fidelity DNA polymerases, as described in Section 2. All ORF clones obtained by PCR cloning were sequence-verified and we found that the PCR mutation occurrence rate was less than 5 × 10<sup>−5</sup> mutations per residue under the conditions employed in this study. ORF clones with large ORFs are inevitably at higher risk of having artificial missense mutation(s) introduced by PCR, thus we previously developed and applied the ORF Tap cloning method for the construction of Flexi ORF clones for some relatively long ORFs (mainly those &gt;2 kb).<sup><a reveal-id="DSN004C9" data-open="DSN004C9" class="link link-ref link-reveal xref-bibr">9</a></sup> The method is based on a homologous recombination in <em>E. coli</em> JC8679 and thus is very unlikely to introduce mutation(s) into ORFs during cloning. We therefore considered that this method was also highly suited to the preparation of Flexi ORF clones because reconfirmation of the ORF sequence could be avoided.<sup><a reveal-id="DSN004C16" data-open="DSN004C16" class="link link-ref link-reveal xref-bibr">16</a>–<a reveal-id="DSN004C18" data-open="DSN004C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Within this context, 842 pF1K clones were constructed by the ORF Trap cloning method. The remaining 749 pF1K clones, termed Flexi_RBS type, were constructed by transferring ORF sequences from the Gateway entry clones we had previously prepared to the pF1K vector, because the 5′- and 3′-untranslated sequences (UTRs) have been already trimmed out in the Gateway entry clones. These clones contained a 19-bp fragment, including a ribosome-binding site (RBS), between the <em>Sgf</em>I site and a translational initiation codon. These Flexi clones were distinguished from others by adding ‘SD’, representing a Shine-Dalgarno sequence as an indication of the presence of an RBS, to their clone IDs, whereas the other pF1K clones were simply termed Flexi type. The ORF flanking sequences in these Flexi clones are shown in Fig. <a reveal-id="DSN004F1" data-open="DSN004F1" class="link link-reveal link-table xref-fig">1</a>A. The <em>Sgf</em>I site was placed one base upstream of the initiation codon, which allowed the production of recombinant proteins with the native translational initiation site or with N-terminal tags using appropriate Flexi<sup>®</sup> vectors. On the other hand, the <em>Pme</em>I site was placed just downstream from the original stop codon of the ORF, which resulted in an attachment of the Val in Flexi type or the Tyr–Val–Val in Flexi_RBS type to the carboxy end of the native ORF. When an ORF sequence flanked by <em>Sgf</em>I and <em>Pme</em>I was cloned between the <em>Sgf</em>I and <em>Eco</em>ICRI sites of a Flexi<sup>®</sup> vector, the translational stop codon was destroyed and thus the protein was expressed as a carboxy-terminal fusion. ORFs in pF1K clones thus prepared can be easily transferred to other types of Flexi<sup>®</sup> vectors following digestion with <em>Sgf</em>I and <em>Pme</em>I (Fig. <a reveal-id="DSN004F1" data-open="DSN004F1" class="link link-reveal link-table xref-fig">1</a>B).</p>                        <a id="6967044" scrollto-destination="6967044"></a>
<div id="DSN004F1" data-id="DSN004F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn004/2/m_dsn00401.jpeg?Expires=2147483647&Signature=BhB8iCCmBE0BrlBQ5CVBfyUrtOavdScaOGaYNt-BAXIaZHVvb-4Z73rAUYeLZxOIeDo~cqjkCy4IiURXLkPJvwHJr0QK5Jc2kbv3UyolRS0dVZcmeCtYyevUlSS9E5p5J3hWbudPpqdFGl-OKQ-oN9zkTHUmx~DQmHGJZKVxw9hH2493FjwTNmPlc1guczilSKPVrDs1mp1Cte6CgQsYfYrmPBfEQW~YWh1Xt75vqX~XTI~beG~3G7d4aITD2epVHNiZ3R54kzurB8VzT9n6mS5HTiCcsL3KHB5-~~v0FeXKgSg8h8tWAD~mme21iazqJOdpfrWtEOJdwjfwztFalA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="ORF transfer in the Flexi® Vector cloning system. (A) Flanking sequences of ORF in Flexi clones. Recognition sequences of SgfI and PmeI are indicated as green and red characters, respectively. The nucleotide sequence corresponding to the ribosomal binding site is underlined. The amino acid sequence encoded in the frame in the flanking regions of the ORF is indicated as a three-letter code. Recognition sequences of BstBI and SnaBI, arising in the vector of Flexi_RBS type are indicated as blue characters. (B) Transfer of the ORF from the pF1K clone to multiple expression vectors. The ORF sequence in the pF1K clone can be easily transferred to a variety of other expression vectors with the correct orientation after digestion by SgfI and PmeI. For construction of a C-terminal tag-fusion clone, SgfI–PmeI ORF sequence must be cloned into SgfI and EcoICRI sites of the expression vector to omit a stop codon arising in the PmeI site. The appropriate promoter is indicated as an orange arrow in the vectors." path-from-xml="dsn00401.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6967044/dsn00401.jpeg" path-from-xml="dsn00401.jpeg" target="_blank">View large</a><a section="6967044" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn004/2/dsn00401.jpeg?Expires=2147483647&Signature=tejhG~e6BvBg21560ewyjMcEFf-mryqm18utxFOnl9NNaO~u2aPko9mFEMPicfLEoaUCZ32JLqlrZDHcQur2h8ygT2SbxooAUencQXuoc6bXlUyHGqKjqFKEHbWZqaxTlQlj034Vx8ZRGfuXCL5gocrxa9wHvhN1JdvLbm61JqV5LZOYgODZfIwwbmEaNOejpEd-k3I43DQMTcU86SsBYMV-PHS3qGsOQE8kMh6-eCKfk98KMdm47Yv0FG8Bw1oGgExchulkLVBEb7mho5r50XypjQu-ujG0bK~qDl7VT62ngKuxoPEDzuGRAEBtFEf6Ine9U1Bes8NpMAwht-njXg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6967044&ar=464067&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn00401.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>ORF transfer in the Flexi<sup>®</sup> Vector cloning system. (<strong>A</strong>) Flanking sequences of ORF in Flexi clones. Recognition sequences of <em>Sgf</em>I and <em>Pme</em>I are indicated as green and red characters, respectively. The nucleotide sequence corresponding to the ribosomal binding site is underlined. The amino acid sequence encoded in the frame in the flanking regions of the ORF is indicated as a three-letter code. Recognition sequences of <em>Bst</em>BI and <em>Sna</em>BI, arising in the vector of Flexi_RBS type are indicated as blue characters. (<strong>B</strong>) Transfer of the ORF from the pF1K clone to multiple expression vectors. The ORF sequence in the pF1K clone can be easily transferred to a variety of other expression vectors with the correct orientation after digestion by <em>Sgf</em>I and <em>Pme</em>I. For construction of a C-terminal tag-fusion clone, <em>Sgf</em>I–<em>Pme</em>I ORF sequence must be cloned into <em>Sgf</em>I and <em>Eco</em>ICRI sites of the expression vector to omit a stop codon arising in the <em>Pme</em>I site. The appropriate promoter is indicated as an orange arrow in the vectors.</p></div></div><div content-id="DSN004F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn004/2/m_dsn00401.jpeg?Expires=2147483647&Signature=BhB8iCCmBE0BrlBQ5CVBfyUrtOavdScaOGaYNt-BAXIaZHVvb-4Z73rAUYeLZxOIeDo~cqjkCy4IiURXLkPJvwHJr0QK5Jc2kbv3UyolRS0dVZcmeCtYyevUlSS9E5p5J3hWbudPpqdFGl-OKQ-oN9zkTHUmx~DQmHGJZKVxw9hH2493FjwTNmPlc1guczilSKPVrDs1mp1Cte6CgQsYfYrmPBfEQW~YWh1Xt75vqX~XTI~beG~3G7d4aITD2epVHNiZ3R54kzurB8VzT9n6mS5HTiCcsL3KHB5-~~v0FeXKgSg8h8tWAD~mme21iazqJOdpfrWtEOJdwjfwztFalA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="ORF transfer in the Flexi® Vector cloning system. (A) Flanking sequences of ORF in Flexi clones. Recognition sequences of SgfI and PmeI are indicated as green and red characters, respectively. The nucleotide sequence corresponding to the ribosomal binding site is underlined. The amino acid sequence encoded in the frame in the flanking regions of the ORF is indicated as a three-letter code. Recognition sequences of BstBI and SnaBI, arising in the vector of Flexi_RBS type are indicated as blue characters. (B) Transfer of the ORF from the pF1K clone to multiple expression vectors. The ORF sequence in the pF1K clone can be easily transferred to a variety of other expression vectors with the correct orientation after digestion by SgfI and PmeI. For construction of a C-terminal tag-fusion clone, SgfI–PmeI ORF sequence must be cloned into SgfI and EcoICRI sites of the expression vector to omit a stop codon arising in the PmeI site. The appropriate promoter is indicated as an orange arrow in the vectors." path-from-xml="dsn00401.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6967044/dsn00401.jpeg" path-from-xml="dsn00401.jpeg" target="_blank">View large</a><a section="6967044" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn004/2/dsn00401.jpeg?Expires=2147483647&Signature=tejhG~e6BvBg21560ewyjMcEFf-mryqm18utxFOnl9NNaO~u2aPko9mFEMPicfLEoaUCZ32JLqlrZDHcQur2h8ygT2SbxooAUencQXuoc6bXlUyHGqKjqFKEHbWZqaxTlQlj034Vx8ZRGfuXCL5gocrxa9wHvhN1JdvLbm61JqV5LZOYgODZfIwwbmEaNOejpEd-k3I43DQMTcU86SsBYMV-PHS3qGsOQE8kMh6-eCKfk98KMdm47Yv0FG8Bw1oGgExchulkLVBEb7mho5r50XypjQu-ujG0bK~qDl7VT62ngKuxoPEDzuGRAEBtFEf6Ine9U1Bes8NpMAwht-njXg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6967044&ar=464067&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn00401.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>ORF transfer in the Flexi<sup>®</sup> Vector cloning system. (<strong>A</strong>) Flanking sequences of ORF in Flexi clones. Recognition sequences of <em>Sgf</em>I and <em>Pme</em>I are indicated as green and red characters, respectively. The nucleotide sequence corresponding to the ribosomal binding site is underlined. The amino acid sequence encoded in the frame in the flanking regions of the ORF is indicated as a three-letter code. Recognition sequences of <em>Bst</em>BI and <em>Sna</em>BI, arising in the vector of Flexi_RBS type are indicated as blue characters. (<strong>B</strong>) Transfer of the ORF from the pF1K clone to multiple expression vectors. The ORF sequence in the pF1K clone can be easily transferred to a variety of other expression vectors with the correct orientation after digestion by <em>Sgf</em>I and <em>Pme</em>I. For construction of a C-terminal tag-fusion clone, <em>Sgf</em>I–<em>Pme</em>I ORF sequence must be cloned into <em>Sgf</em>I and <em>Eco</em>ICRI sites of the expression vector to omit a stop codon arising in the <em>Pme</em>I site. The appropriate promoter is indicated as an orange arrow in the vectors.</p></div></div><p>Information about the Flexi ORF clones (i.e. ORF nucleotide sequences, the predicted primary sequences of the gene products, corresponding Ensembl gene ID, and gene description<sup><a reveal-id="DSN004C2" data-open="DSN004C2" class="link link-ref link-reveal xref-bibr">2</a></sup> in addition to results of computer-assisted analyses of the predicted primary amino acid sequences) is available through our KOP database (<a class="link link-uri" href="http://www.kazusa.or.jp/kop/">http://www.kazusa.or.jp/kop/</a>).</p>                            <h3 scrollto-destination=6967046 id="6967046" class="section-title" data-legacy-id=s3b>3.2. Comparison of the productivities of HaloTag-fusion proteins from Flexi ORF clones and Gateway ORF clones in two in vitro protein production systems</h3>
<p>Since our focus is on the use of HaloTag<sup>®</sup> technology, we initially prepared a set of Flexi ORF clones as HaloTag-fusions using the Flexi<sup>®</sup> cloning method. To examine the utility of HaloTag<sup>®</sup> technology, HaloTag-fusion products were produced in <em>in vitro</em> translation systems. The expressed proteins were analyzed on SDS–PAGE after labeling with a HaloTag-TMR ligand. This simple labeling procedure enabled us to fluorometrically detect only proteins carrying a HaloTag portion on SDS–PAGE, and thereby made it easy to check whether the ORF encoded an appropriately sized protein. Using this analysis, quantities of HaloTag-fusion proteins with ligand-binding activity could be directly estimated from the fluorescence intensities of protein bands on SDS–PAGE. In these experiments, we also compared the performance of two <em>in vitro</em> translation systems: one derived from a wheat germ extract and the second from an insect cell extract [commercially named WGEP and TransDirect (TD), respectively]. We used these systems because these eukaryotic cell extracts are known to produce larger amounts of proteins than conventional rabbit reticulocyte lysates. In parallel, we also compared the efficiency of protein synthesis by Gateway-type (pTD2-Gw) and Flexi-type (pTD2-Flx) expression constructs, where ORFs were flanked with different appended sequences that are associated with the respective cloning methods. These appended sequences may affect the translational activity of the HaloTag-fusion constructs <em>in vitro</em>. For this purpose, we constructed 13 sets of HaloTag-fusion protein expression plasmids using pTD2-Gw and pTD2-Flx vectors. pTD2-Flx is a simple Flexi-type HaloTag-fusion protein expression vector derived from the pTD1 vector for the TD system.<sup><a reveal-id="DSN004C19" data-open="DSN004C19" class="link link-ref link-reveal xref-bibr">19</a></sup> This vector has 8-bp restriction enzyme recognition sites (<em>Sgf</em>I and <em>Pme</em>I sites) flanked by the baculovirus polyhedrin 5′- and 3′-UTRs, respectively. pTD2-Gw is a hybrid expression vector derived from pTD2-Flx, which has 25-bp Gateway recombination sites, <em>att</em>B1 or <em>att</em>B2, between polyhedrin 5′-UTR and <em>Sgf</em>I, or between <em>Pme</em>I and 3′-UTR, respectively (Fig. <a reveal-id="DSN004F2" data-open="DSN004F2" class="link link-reveal link-table xref-fig">2</a>A). In both systems, protein production was directed by <em>in-vitro</em>-synthesized RNAs. Many fluorescent bands, probably resulting from proteolytic degradation, were observed in most samples. The fluorescence signal intensities of the largest bands were the expected size for the respective protein products, and each HaloTag-fusion protein was recorded for comparing the protein productivities of pTD2-Flx and pTD2-Gw (Fig. <a reveal-id="DSN004F2" data-open="DSN004F2" class="link link-reveal link-table xref-fig">2</a>B, indicated by an arrow). In the TD system, the fluorescence intensities of 8 out of 13 HaloTag-fusion proteins in the pTD2-Gw format were less than 60% as intense as those derived from pTD2-Flx clones (Fig. <a reveal-id="DSN004F2" data-open="DSN004F2" class="link link-reveal link-table xref-fig">2</a>B, upper lanes: 2, 3, 5, 8–11, and 13). On the other hand, in the WGEP expression system, the fluorescence intensities of HaloTag-fusion proteins from five fusion constructs in the pTD2-Gw vector were slightly reduced compared with those from pTD2-Flx fusion constructs (Fig. <a reveal-id="DSN004F2" data-open="DSN004F2" class="link link-reveal link-table xref-fig">2</a>B, lower lanes: 3, 4, 8–10). Interestingly enough, the fluorescent band patterns of HaloTag-fusions produced in the TD and WGEP systems were considerably different from each other, although the same <em>in-vitro-</em>synthesized RNAs were subjected to both <em>in vitro</em> protein production systems. These results indicated that (<a reveal-id="DSN004C1" data-open="DSN004C1" class="link link-ref link-reveal xref-bibr">1</a>) the protein productivity of both the TD and WGEP systems varied widely from gene to gene, (<a reveal-id="DSN004C2" data-open="DSN004C2" class="link link-ref link-reveal xref-bibr">2</a>) TD and WGEP had different preferences for genes in terms of protein production, and (<a reveal-id="DSN004C3" data-open="DSN004C3" class="link link-ref link-reveal xref-bibr">3</a>) the Gateway recombination site(s) apparently had an inhibitory effect on translational activity, which was more prominent in the TD systems.</p>                        <a id="6967048" scrollto-destination="6967048"></a>
<div id="DSN004F2" data-id="DSN004F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn004/2/m_dsn00402.jpeg?Expires=2147483647&Signature=DHspYCYXpv0jkH3~mu0b1LfnUmWduHWSeoUsGsxou-2HrhZj67ihxXtSm3OV6Cc~mFr~SZuEmGh9EmIq-Nss1YfuU17lnehS02FW9cLJ~9ZSgixn9YpDZ8fCfKVU4tkyEvk0RRUdx3x-oohETmoHd9VK-TCHZmQNdG02FB8gw9iDjssbhAe7XeaStAioFa1h0R-SPBOC6EIM5VTcinO854ID6PKuPQ4vKYcEazHHyJX4aGXOfSUotLOaQM-hTDOY7tq5IHahxJAKToHgzDZZ3K2yEwyrkrxHKVOywJysoER2FNBhKgA3LRslJam9hpRHg0Sa7sDT~mCoZyAWCJfZFg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Effects of appended sequences surrounding the ORF on translational activity in cell-free protein synthesis systems. (A) Schematic representation of HaloTag-fusion protein expression vectors, pTD2-Flx (Flexi-type), and pTD2-Gw (Gateway-type). Functional elements are indicated in the open boxes. attB1 and attB2 represent the Gateway sequences, where specific recombination reactions occur. (B) Two types of in vitro protein synthesis experiments were carried out using Flexi-type and Gateway-type clones. ORFs from the 13 different genes listed below were cloned into the two different vectors shown in A and used for in vitro RNA synthesis, followed by in vitro protein synthesis. Products labeled with HaloTag-TMR ligand and their fluorescence intensities were detected on SDS–PAGE using FluoroImager FLA3000 (indicated by arrows). Upper panel shows the results of the TransDirect insect cell (TD) and lower panel shows those of the WGEP. In each panel, protein products from the Flexi-type (F) and Gateway-type vectors (G) are alternately shown. Ratios of the fluorescence intensities of Gateway products to Flexi products, expressed as percentages, are shown beneath each panel. An asterisk indicates that protein products were under detection limits. The 13 genes examined here and the numbers of amino acid residues without HaloTag are as follows: KIAA0409 (lane 1, 457 a.a.); KIAA1248 (lane 2, 358 a.a.); KIAA0491 (lane 3, 366 a.a.); KIAA1238 (lane 4, 387 a.a.); KIAA1788 (lane 5, 412 a.a.); KIAA1604 (lane 6, 909 a.a.); KIAA1649 (lane 7, 422 a.a.); KIAA1390 (lane 8, 470 a.a.); KIAA2021 (lane 9, 637 a.a.); KIAA0018 (lane 10, 517 a.a.); KIAA1346 (lane 11, 968 a.a.); KIAA1584 (lane 12, 967 a.a.); and KIAA2015 (lane 13, 992 a.a.). (C) Schematic representation of luc2, HaloTag, and Venus expression clones. Functional elements are indicated in the open boxes. Additional sequences inserted after/before polyhedrin enhancers are shown. Km1 and Km2 were derived from a kanamycin-resistant gene. RBS represents a ribosome-binding sequence. N–N is equivalent to pTDFlexi. (D) Results of in vitro protein products assay using luc2, HaloTag, and Venus expression clones. Their activities were measured as described in Section 2. In the expression clone D5, the polyhedrin 5′-UTR 46-bp sequence was replaced with a 10-bp sequence (AATCGAATTC). Each set of signal intensities was normalized by dividing by that of the N–N, and the averages of three (Venus and HaloTag), six (TD of luc2), and seven (WGEP of luc2) independent experiments were calculated. Filled bars represent luc2 activity, gray bars represent the HaloTag-TMR ligand signal, and open bars indicate Venus fluorescence intensity." path-from-xml="dsn00402.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6967048/dsn00402.jpeg" path-from-xml="dsn00402.jpeg" target="_blank">View large</a><a section="6967048" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn004/2/dsn00402.jpeg?Expires=2147483647&Signature=sEGT6aAurxgQT4pN1ll-~sW3M3RkY3jX2gdUJQQwEOTo3NkXiNFEOnOs0VwlCjxUGbQUx6NrvWt9pDg41BT8s271LBSX4gDMbQ5HPZBVwx6EC6A664uCwMhrH3tXhDdo1nnC-wUH4EelgHFx3elSb2GoHJGQ3Odt50Fh8UY9rAvYjNWhRTxmJIY6JNnMNnjms4mb8oKnVus0TkeC5LYNjj8XuxtCBSFVzp0-KESeMiMhI1cx-Qbrm8tU22-pV3M3ZpveuM7Gq0JXihGbDaQESEELO62Zhg~BELlXMf9DZa~RzkwCffnBoLNzgRWZa4jYL5E3ptjdt5~SJ8xCa~68BA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6967048&ar=464067&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn00402.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Effects of appended sequences surrounding the ORF on translational activity in cell-free protein synthesis systems. (<strong>A</strong>) Schematic representation of HaloTag-fusion protein expression vectors, pTD2-Flx (Flexi-type), and pTD2-Gw (Gateway-type). Functional elements are indicated in the open boxes. <em>att</em>B1 and <em>att</em>B2 represent the Gateway sequences, where specific recombination reactions occur. (<strong>B</strong>) Two types of <em>in vitro</em> protein synthesis experiments were carried out using Flexi-type and Gateway-type clones. ORFs from the 13 different genes listed below were cloned into the two different vectors shown in A and used for <em>in vitro</em> RNA synthesis, followed by <em>in vitro</em> protein synthesis. Products labeled with HaloTag-TMR ligand and their fluorescence intensities were detected on SDS–PAGE using FluoroImager FLA3000 (indicated by arrows). Upper panel shows the results of the TransDirect insect cell (TD) and lower panel shows those of the WGEP. In each panel, protein products from the Flexi-type (F) and Gateway-type vectors (G) are alternately shown. Ratios of the fluorescence intensities of Gateway products to Flexi products, expressed as percentages, are shown beneath each panel. An asterisk indicates that protein products were under detection limits. The 13 genes examined here and the numbers of amino acid residues without HaloTag are as follows: KIAA0409 (lane 1, 457 a.a.); KIAA1248 (lane 2, 358 a.a.); KIAA0491 (lane 3, 366 a.a.); KIAA1238 (lane 4, 387 a.a.); KIAA1788 (lane 5, 412 a.a.); KIAA1604 (lane 6, 909 a.a.); KIAA1649 (lane 7, 422 a.a.); KIAA1390 (lane 8, 470 a.a.); KIAA2021 (lane 9, 637 a.a.); KIAA0018 (lane 10, 517 a.a.); KIAA1346 (lane 11, 968 a.a.); KIAA1584 (lane 12, 967 a.a.); and KIAA2015 (lane 13, 992 a.a.). (<strong>C</strong>) Schematic representation of <em>luc2</em>, HaloTag, and Venus expression clones. Functional elements are indicated in the open boxes. Additional sequences inserted after/before polyhedrin enhancers are shown. Km1 and Km2 were derived from a kanamycin-resistant gene. RBS represents a ribosome-binding sequence. N–N is equivalent to pTDFlexi. (<strong>D</strong>) Results of <em>in vitro</em> protein products assay using <em>luc2</em>, HaloTag, and Venus expression clones. Their activities were measured as described in Section 2. In the expression clone D5, the polyhedrin 5′-UTR 46-bp sequence was replaced with a 10-bp sequence (AATCGAATTC). Each set of signal intensities was normalized by dividing by that of the N–N, and the averages of three (Venus and HaloTag), six (TD of <em>luc2</em>), and seven (WGEP of <em>luc2</em>) independent experiments were calculated. Filled bars represent <em>luc2</em> activity, gray bars represent the HaloTag-TMR ligand signal, and open bars indicate Venus fluorescence intensity.</p></div></div><div content-id="DSN004F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn004/2/m_dsn00402.jpeg?Expires=2147483647&Signature=DHspYCYXpv0jkH3~mu0b1LfnUmWduHWSeoUsGsxou-2HrhZj67ihxXtSm3OV6Cc~mFr~SZuEmGh9EmIq-Nss1YfuU17lnehS02FW9cLJ~9ZSgixn9YpDZ8fCfKVU4tkyEvk0RRUdx3x-oohETmoHd9VK-TCHZmQNdG02FB8gw9iDjssbhAe7XeaStAioFa1h0R-SPBOC6EIM5VTcinO854ID6PKuPQ4vKYcEazHHyJX4aGXOfSUotLOaQM-hTDOY7tq5IHahxJAKToHgzDZZ3K2yEwyrkrxHKVOywJysoER2FNBhKgA3LRslJam9hpRHg0Sa7sDT~mCoZyAWCJfZFg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Effects of appended sequences surrounding the ORF on translational activity in cell-free protein synthesis systems. (A) Schematic representation of HaloTag-fusion protein expression vectors, pTD2-Flx (Flexi-type), and pTD2-Gw (Gateway-type). Functional elements are indicated in the open boxes. attB1 and attB2 represent the Gateway sequences, where specific recombination reactions occur. (B) Two types of in vitro protein synthesis experiments were carried out using Flexi-type and Gateway-type clones. ORFs from the 13 different genes listed below were cloned into the two different vectors shown in A and used for in vitro RNA synthesis, followed by in vitro protein synthesis. Products labeled with HaloTag-TMR ligand and their fluorescence intensities were detected on SDS–PAGE using FluoroImager FLA3000 (indicated by arrows). Upper panel shows the results of the TransDirect insect cell (TD) and lower panel shows those of the WGEP. In each panel, protein products from the Flexi-type (F) and Gateway-type vectors (G) are alternately shown. Ratios of the fluorescence intensities of Gateway products to Flexi products, expressed as percentages, are shown beneath each panel. An asterisk indicates that protein products were under detection limits. The 13 genes examined here and the numbers of amino acid residues without HaloTag are as follows: KIAA0409 (lane 1, 457 a.a.); KIAA1248 (lane 2, 358 a.a.); KIAA0491 (lane 3, 366 a.a.); KIAA1238 (lane 4, 387 a.a.); KIAA1788 (lane 5, 412 a.a.); KIAA1604 (lane 6, 909 a.a.); KIAA1649 (lane 7, 422 a.a.); KIAA1390 (lane 8, 470 a.a.); KIAA2021 (lane 9, 637 a.a.); KIAA0018 (lane 10, 517 a.a.); KIAA1346 (lane 11, 968 a.a.); KIAA1584 (lane 12, 967 a.a.); and KIAA2015 (lane 13, 992 a.a.). (C) Schematic representation of luc2, HaloTag, and Venus expression clones. Functional elements are indicated in the open boxes. Additional sequences inserted after/before polyhedrin enhancers are shown. Km1 and Km2 were derived from a kanamycin-resistant gene. RBS represents a ribosome-binding sequence. N–N is equivalent to pTDFlexi. (D) Results of in vitro protein products assay using luc2, HaloTag, and Venus expression clones. Their activities were measured as described in Section 2. In the expression clone D5, the polyhedrin 5′-UTR 46-bp sequence was replaced with a 10-bp sequence (AATCGAATTC). Each set of signal intensities was normalized by dividing by that of the N–N, and the averages of three (Venus and HaloTag), six (TD of luc2), and seven (WGEP of luc2) independent experiments were calculated. Filled bars represent luc2 activity, gray bars represent the HaloTag-TMR ligand signal, and open bars indicate Venus fluorescence intensity." path-from-xml="dsn00402.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6967048/dsn00402.jpeg" path-from-xml="dsn00402.jpeg" target="_blank">View large</a><a section="6967048" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn004/2/dsn00402.jpeg?Expires=2147483647&Signature=sEGT6aAurxgQT4pN1ll-~sW3M3RkY3jX2gdUJQQwEOTo3NkXiNFEOnOs0VwlCjxUGbQUx6NrvWt9pDg41BT8s271LBSX4gDMbQ5HPZBVwx6EC6A664uCwMhrH3tXhDdo1nnC-wUH4EelgHFx3elSb2GoHJGQ3Odt50Fh8UY9rAvYjNWhRTxmJIY6JNnMNnjms4mb8oKnVus0TkeC5LYNjj8XuxtCBSFVzp0-KESeMiMhI1cx-Qbrm8tU22-pV3M3ZpveuM7Gq0JXihGbDaQESEELO62Zhg~BELlXMf9DZa~RzkwCffnBoLNzgRWZa4jYL5E3ptjdt5~SJ8xCa~68BA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6967048&ar=464067&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn00402.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Effects of appended sequences surrounding the ORF on translational activity in cell-free protein synthesis systems. (<strong>A</strong>) Schematic representation of HaloTag-fusion protein expression vectors, pTD2-Flx (Flexi-type), and pTD2-Gw (Gateway-type). Functional elements are indicated in the open boxes. <em>att</em>B1 and <em>att</em>B2 represent the Gateway sequences, where specific recombination reactions occur. (<strong>B</strong>) Two types of <em>in vitro</em> protein synthesis experiments were carried out using Flexi-type and Gateway-type clones. ORFs from the 13 different genes listed below were cloned into the two different vectors shown in A and used for <em>in vitro</em> RNA synthesis, followed by <em>in vitro</em> protein synthesis. Products labeled with HaloTag-TMR ligand and their fluorescence intensities were detected on SDS–PAGE using FluoroImager FLA3000 (indicated by arrows). Upper panel shows the results of the TransDirect insect cell (TD) and lower panel shows those of the WGEP. In each panel, protein products from the Flexi-type (F) and Gateway-type vectors (G) are alternately shown. Ratios of the fluorescence intensities of Gateway products to Flexi products, expressed as percentages, are shown beneath each panel. An asterisk indicates that protein products were under detection limits. The 13 genes examined here and the numbers of amino acid residues without HaloTag are as follows: KIAA0409 (lane 1, 457 a.a.); KIAA1248 (lane 2, 358 a.a.); KIAA0491 (lane 3, 366 a.a.); KIAA1238 (lane 4, 387 a.a.); KIAA1788 (lane 5, 412 a.a.); KIAA1604 (lane 6, 909 a.a.); KIAA1649 (lane 7, 422 a.a.); KIAA1390 (lane 8, 470 a.a.); KIAA2021 (lane 9, 637 a.a.); KIAA0018 (lane 10, 517 a.a.); KIAA1346 (lane 11, 968 a.a.); KIAA1584 (lane 12, 967 a.a.); and KIAA2015 (lane 13, 992 a.a.). (<strong>C</strong>) Schematic representation of <em>luc2</em>, HaloTag, and Venus expression clones. Functional elements are indicated in the open boxes. Additional sequences inserted after/before polyhedrin enhancers are shown. Km1 and Km2 were derived from a kanamycin-resistant gene. RBS represents a ribosome-binding sequence. N–N is equivalent to pTDFlexi. (<strong>D</strong>) Results of <em>in vitro</em> protein products assay using <em>luc2</em>, HaloTag, and Venus expression clones. Their activities were measured as described in Section 2. In the expression clone D5, the polyhedrin 5′-UTR 46-bp sequence was replaced with a 10-bp sequence (AATCGAATTC). Each set of signal intensities was normalized by dividing by that of the N–N, and the averages of three (Venus and HaloTag), six (TD of <em>luc2</em>), and seven (WGEP of <em>luc2</em>) independent experiments were calculated. Filled bars represent <em>luc2</em> activity, gray bars represent the HaloTag-TMR ligand signal, and open bars indicate Venus fluorescence intensity.</p></div></div>                            <h3 scrollto-destination=6967049 id="6967049" class="section-title" data-legacy-id=s3c>3.3. Effects of ORF-flanking sequences on translational activity in cell-free protein synthesis systems</h3>
<p>While the different performances of the TD and WGEP systems were anticipated to some extent, it was not anticipated that the Gateway recombination sites would have an inhibitory effect on <em>in vitro</em> protein production. Thus, we further pursued the cause of this apparent inhibitory effect of the Gateway recombination sites on <em>in vitro</em> protein production. To systematically examine the effects of the appended sequences flanking the ORFs on <em>in vitro</em> protein production, we constructed systematically modified versions of expression clones for <em>luc2</em>, HaloTag, and Venus autofluorescent proteins, with different flanking sequences for each ORF (Fig. <a reveal-id="DSN004F2" data-open="DSN004F2" class="link link-reveal link-table xref-fig">2</a>C). Protein production in the TD and WGEP systems was monitored by measuring luciferase activity, HaloTag-TMR ligand-binding activity, or the fluorescence intensity of Venus protein (Fig. <a reveal-id="DSN004F2" data-open="DSN004F2" class="link link-reveal link-table xref-fig">2</a>D). As shown in Fig. <a reveal-id="DSN004F2" data-open="DSN004F2" class="link link-reveal link-table xref-fig">2</a>C, these clones included flanking sequences derived from the Gateway <em>att</em>B sites (B1 and/or B2), or parts of the kanamycin-resistant gene (K1: ACTGGGCACAACAGACAATCGGCTG, or K2: CCTGTCATCTCACCTTGCTCCTGCC, as examples of non-specific sequences) between the translational enhancer (polyhedrin 5′-UTR) and the initiation ATG codon, and/or between the stop codon and the polyhedrin 3′-UTR. N–N, an expression clone without any additional sequences, was the control clone. N–B2 and N–K2 carry additional sequences (B2 and K2, respectively) only at the 3′-flanking site, whereas B1–N and K1–N have additional sequences (B1 and K1, respectively) only at the 5′-flanking site. K1 and K2 were spacer sequences with equal sizes as B1 and B2 and were not expected to be involved with the Gateway recombination. We also compared protein production from the Gateway-type expression vector (B1–B2) and the modified vector (B1–K2) in parallel. In total, we examined seven different expression vectors in these experiments. Figure <a reveal-id="DSN004F2" data-open="DSN004F2" class="link link-reveal link-table xref-fig">2</a>D shows the relative activities of <em>luc2</em>, HaloTag, and Venus proteins produced in the TD and WGEP systems compared with the N–N type of pTDFlexi vector. Under these experimental conditions, the luciferase expression clones with <em>att</em>B1 sequence upstream of the ORFs (Fig. <a reveal-id="DSN004F2" data-open="DSN004F2" class="link link-reveal link-table xref-fig">2</a>, B1–N, B1–B2, and B1–K2) appeared to exhibit only a slight inhibitory effect in the TD and WGEP systems in comparison with the pTDFlexi-<em>luc2</em> construct (N–N in Fig. <a reveal-id="DSN004F2" data-open="DSN004F2" class="link link-reveal link-table xref-fig">2</a>C). This reduction in protein productivity seemed to depend on the distance (39 nucleotides) between the 5′ translational enhancer and the initiation ATG but not the specific nucleotide sequence, at least in insect TD systems. Expression clones with short nucleotide sequences of the same length derived from the kanamycin-resistant gene also reduced the production of <em>luciferase</em> (Fig. <a reveal-id="DSN004F2" data-open="DSN004F2" class="link link-reveal link-table xref-fig">2</a>D, upper, K1–N). Such an inhibitory effect was more evident for HaloTag and Venus protein production in the TD system. In fact, in the TD system, both HaloTag-TMR-ligand-binding activity and the fluorescence intensity of Venus decreased to less than 50% when the additional sequences were located upstream of the ORF (Fig. <a reveal-id="DSN004F2" data-open="DSN004F2" class="link link-reveal link-table xref-fig">2</a>D, upper, B1–N, K1–N, B1–B2, and B1–K2). These results indicated that protein productivity in the TD system was affected by the increase in distance between the polyhedrin 5′-UTR and the start ATG, rather than by the flanking nucleotide sequence of the spacer.</p><p>It was reported that the polyhedrin 5′-UTR in pTD1 displayed a strong translational enhancing activity, even in the wheat germ extract and rabbit reticulocyte lysate systems.<sup><a reveal-id="DSN004C19" data-open="DSN004C19" class="link link-ref link-reveal xref-bibr">19</a></sup> The inhibitory effect of sequences upstream of the ORF was prominent in the TD system but only minor in the WGEP system (Fig. <a reveal-id="DSN004F2" data-open="DSN004F2" class="link link-reveal link-table xref-fig">2</a>D, lower). Our results also showed that the expression clone lacking 5′-UTR considerably reduced the productivity of <em>luciferase</em> in both the TD and WGEP systems, indicating that the translational enhancer of 5′-UTR in pTD1 was functional in each system (Fig. <a reveal-id="DSN004F2" data-open="DSN004F2" class="link link-reveal link-table xref-fig">2</a>D, D5). On the other hand, translational activity was largely unaffected by the spacing between the 3′-translational enhancer and the translational stop codon in all experiments using <em>luc2</em>, HaloTag, and Venus proteins (Fig. <a reveal-id="DSN004F2" data-open="DSN004F2" class="link link-reveal link-table xref-fig">2</a>D, N–B2, N–K2). Taking all these results together, we concluded that the spacing between the 5′-translational enhancer and the translational initiation codon affected protein productivity, at least <em>in vitro</em>. Although the mechanism of the inhibitory effect on translational activity by spacing remains to be elucidated, it appears that nucleotide spacing between the enhancer and the initiation codon can sometimes affect translational activity. Because the distance between the 5′-translational enhancer and the initiation codon is shorter in the Flexi ORF clone than in the Gateway ORF clone, these results suggest that the Flexi ORF clone could be the clone of choice to consistently obtain high protein productivity in both the TD and WGEP systems. We consider this as another benefit of using the Flexi ORF clones, since production level is a serious concern, particularly when a recombinant protein is synthesized <em>in vitro</em>.</p>                            <h3 scrollto-destination=6967052 id="6967052" class="section-title" data-legacy-id=s3d>3.4. Quality check of ORF clones from the viewpoint of in vitro protein production</h3>
<p>Based on the results described above, we confirmed the functionality of a number of Flexi ORF clones produced as HaloTag-fusions in <em>in vitro</em> protein production systems. For this purpose, we used a transcription/translation coupled wheat-germ-cell-free protein synthesis system [T<span class="small-caps">n</span>T SP6 High Yield Protein Expression System, hereafter abbreviated as (T<span class="small-caps">n</span>T SP6 HYPES)] because the coupled system is easier and faster to operate and gives a higher yield of recombinant protein production than other uncoupled cell-free protein synthesis systems. Eight hundred and fourteen ORF sequences in pF1K clones were transferred to pF3AHT, C-terminal HaloTag-fusion protein expression vectors, by Flexi<sup>®</sup> cloning methods. One practical problem we encountered was that the coupled system required a large amount of expression plasmids to obtain the best results. Since we planned to use a small volume of <em>E. coli</em> culture (about 1 mL) to produce the plasmids, we were concerned that we would not be able to produce enough DNA to obtain good results in the T<span class="small-caps">n</span>T system. To solve this problem, we took advantage of <em>in vitro</em> amplification of expression plasmids. We found that DNA amplified with a TempliPhi rolling circle amplification kit could be used as template DNA in the T<span class="small-caps">n</span>T SP6 HYPES without any purification.<sup><a reveal-id="DSN004C20" data-open="DSN004C20" class="link link-ref link-reveal xref-bibr">20</a></sup> This enabled us to examine the production of a number of HaloTag-fusions in a high-throughput manner. As a result, medium-sized and even relatively large proteins, composed of more than 1000 amino acid residues, were effectively produced and detected by Western blot analysis using anti-HaloTag IgG, with only small amounts of degraded or inappropriately initiated proteins (Fig. <a reveal-id="DSN004F3" data-open="DSN004F3" class="link link-reveal link-table xref-fig">3</a>A). Moreover, most of the HaloTag-fusion proteins retained HaloTag ligand-binding activities (Fig. <a reveal-id="DSN004F3" data-open="DSN004F3" class="link link-reveal link-table xref-fig">3</a>B). Of the 814 pF3AHT clones tested in the T<span class="small-caps">n</span>T system, 97% produced recombinant proteins that could be detected by TMR HaloTag ligand labeling and Western blot analysis using anti-HaloTag antibody. The apparent molecular masses of the recombinant proteins were estimated by SDS–PAGE followed by Western blot analysis using anti-HaloTag antibodies. Most of the proteins had apparent molecular masses that were consistent with the numbers of amino acid residues of the fused proteins (data not shown). Although the fluorescence intensity of TMR varied widely from protein to protein in a range of arbitrary fluorescence units from 10 to 10 000, HaloTag-fusions that retained the functional characteristics of TMR ligand-binding were efficiently produced. However, we did observe a tendency of HaloTag-TMR ligand binding to display decreases in fluorescence intensity with increases in protein size (Fig. <a reveal-id="DSN004F3" data-open="DSN004F3" class="link link-reveal link-table xref-fig">3</a>C). These results are most likely explained by decreases in the production levels of larger HaloTag-fusions. However, it could also be explained on the assumption that the partner proteins may affect HaloTag-TMR ligand-binding activities. To address this, we examined the relationship between protein yields and their HaloTag ligand-binding activities for 53 HaloTag-fusions. In this experiment, the HaloTag-fusion protein yields were expressed by the radioactivity of [<sup>35</sup>S]methionine and [<sup>35</sup>S]cysteine incorporated into the proteins and divided by the number that contained methionine and cysteine, while the HaloTag ligand-binding activities were estimated from the fluorescence intensity of Cy5-labeled HaloTag ligand-bound HaloTag-fusion proteins on SDS–PAGE. The HaloTag-fusion protein yields and the HaloTag ligand-binding activities are plotted against the number of their amino acid residues in Fig. <a reveal-id="DSN004F3" data-open="DSN004F3" class="link link-reveal link-table xref-fig">3</a>D and E, respectively. In Fig. <a reveal-id="DSN004F3" data-open="DSN004F3" class="link link-reveal link-table xref-fig">3</a>F, the ratios of the HaloTag ligand-binding activities to the yields of the HaloTag-fusion proteins are plotted against the number of amino acid residues. The results indicated that the molecular HaloTag ligand-binding activities did not vary widely from fusion to fusion. Based on this result, we consider that the low ligand-binding activities observed resulted from low production levels of HaloTag-fusion proteins, although there may have been some exceptions.</p>                        <a id="6967054" scrollto-destination="6967054"></a>
<div id="DSN004F3" data-id="DSN004F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn004/2/m_dsn00402.jpeg?Expires=2147483647&Signature=DHspYCYXpv0jkH3~mu0b1LfnUmWduHWSeoUsGsxou-2HrhZj67ihxXtSm3OV6Cc~mFr~SZuEmGh9EmIq-Nss1YfuU17lnehS02FW9cLJ~9ZSgixn9YpDZ8fCfKVU4tkyEvk0RRUdx3x-oohETmoHd9VK-TCHZmQNdG02FB8gw9iDjssbhAe7XeaStAioFa1h0R-SPBOC6EIM5VTcinO854ID6PKuPQ4vKYcEazHHyJX4aGXOfSUotLOaQM-hTDOY7tq5IHahxJAKToHgzDZZ3K2yEwyrkrxHKVOywJysoER2FNBhKgA3LRslJam9hpRHg0Sa7sDT~mCoZyAWCJfZFg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression of recombinant HaloTag-fusion proteins in an in vitro transcription/translation coupled protein synthesis system. Example of HaloTag-fusion proteins expressed in vitro. TempliPhi-amplified 48 pF3AHT DNA clones were used for an in vitro transcription/translation system using wheat germ extract (TnT SP6 HYPE System). The recombinant HaloTag-fusion proteins were separated by SDS–PAGE. (A) The recombinant proteins were detected by Western blot analysis using anti-HaloTag antibody. Three bands indicated by asterisks are the signals derived from wheat germ endogenous proteins. (B) The recombinant proteins were labeled by the HaloTag-TMR ligand after the reaction and detected with FLA3000. The corresponding gene numbers for the HaloTag-fusion recombinant proteins in each lane of (A) and (B) are indicated as follows: 1, KIAA1231; 2, KIAA0720; 3, KIAA1426; 4, KIAA0386; 5, KIAA0651; 6, KIAA0010; 7, KIAA0543; 8, KIAA0987; 9, KIAA1386; 10, KIAA1902; 11, KIAA0594; 12, KIAA1450; 13, KIAA1973; 14, KIAA0055; 15, KIAA0835; 16, KIAA1531; 17, KIAA0750; 18, KIAA1714; 19, KIAA1738; 20, KIAA1106; 21, KIAA1525; 22, KIAA0229; 23, KIAA0182; 24, KIAA0954; 25, KIAA1904; 26, KIAA0020; 27, KIAA0028; 28, KIAA0076; 29, KIAA0150; 30, KIAA0152; 31, KIAA0153; 32, KIAA0168; 33, KIAA0186; 34, KIAA0205; 35, KIAA0223; 36, KIAA0278; 37, KIAA0680; 38, KIAA1184; 39, KIAA1245; 40, KIAA1399; 41, KIAA1871; 42, KIAA1857; 43, KIAA1912; 44, KIAA1916; 45, KIAA1917; 46, KIAA1942; 47, KIAA1949; and 48, KIAA1957. The positions of MagicMark XP Western Protein Standard (M) are indicated in kDa on the left. Relationship between TMR-ligand-binding activities and sizes of HaloTag-fusion proteins synthesized in the TnT SP6 HYPE system. (C) Fluorescence intensities of the HaloTag-TMR ligand covalently bound to the recombinant proteins (vertical axis) are plotted against the number of amino acid residues without HaloTag (horizontal axis). (D) Yields of the HaloTag-fusion proteins were estimated as the radioactivity of [35S]methionine and [35S]cysteine-labeled proteins calculated by BAS2000 imaging analyzer after SDS–PAGE and the value was divided by the number that contained methionine and cysteine. Yields (vertical axis) are plotted against the number of amino acid residues (horizontal axis) (closed circles). (E) HaloTag ligand-binding activity was estimated by measuring the fluorescence intensities of the HaloTag Cy5 ligand covalently bound to the HaloTag-fusion proteins after SDS–PAGE by FluoroImager FLA3000. The activities (vertical axis) are plotted against the number of amino acid residues (horizontal axis) (gray circles). (F) Ratios of HaloTag ligand-binding activities to HaloTag-fusion protein yields (vertical axis) are plotted against the number of amino acid residues (horizontal axis) (open circles). The 53 human genes used for protein production in (D), (E), and (F) are as follows: KIAA0024, KIAA0027, KIAA0057, KIAA0137, KIAA0151, KIAA0199, KIAA0230, KIAA0238, KIAA0352, KIAA0412, KIAA0416, KIAA0422, KIAA0441, KIAA0550, KIAA0557, KIAA0606, KIAA0623, KIAA0644, KIAA0722, KIAA0798, KIAA0813, KIAA0848, KIAA0936, KIAA0939, KIAA0968, KIAA1020, KIAA1048, KIAA1142, KIAA1198, KIAA1225, KIAA1227, KIAA1246, KIAA1253, KIAA1264, KIAA1337, KIAA1369, KIAA1382, KIAA1431, KIAA1469, KIAA1477, KIAA1559, KIAA1572, KIAA1580, KIAA1851, KIAA1852, KIAA1874, KIAA1910, KIAA2003, ABCA3, PAK6, STK38, PINK1, and GPRC5." path-from-xml="dsn00402.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6967054/dsn00402.jpeg" path-from-xml="dsn00402.jpeg" target="_blank">View large</a><a section="6967054" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn004/2/dsn00402.jpeg?Expires=2147483647&Signature=sEGT6aAurxgQT4pN1ll-~sW3M3RkY3jX2gdUJQQwEOTo3NkXiNFEOnOs0VwlCjxUGbQUx6NrvWt9pDg41BT8s271LBSX4gDMbQ5HPZBVwx6EC6A664uCwMhrH3tXhDdo1nnC-wUH4EelgHFx3elSb2GoHJGQ3Odt50Fh8UY9rAvYjNWhRTxmJIY6JNnMNnjms4mb8oKnVus0TkeC5LYNjj8XuxtCBSFVzp0-KESeMiMhI1cx-Qbrm8tU22-pV3M3ZpveuM7Gq0JXihGbDaQESEELO62Zhg~BELlXMf9DZa~RzkwCffnBoLNzgRWZa4jYL5E3ptjdt5~SJ8xCa~68BA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6967054&ar=464067&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn00402.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression of recombinant HaloTag-fusion proteins in an <em>in vitro</em> transcription/translation coupled protein synthesis system. Example of HaloTag-fusion proteins expressed <em>in vitro</em>. TempliPhi-amplified 48 pF3AHT DNA clones were used for an <em>in vitro</em> transcription/translation system using wheat germ extract (T<span class="small-caps">n</span>T SP6 HYPE System). The recombinant HaloTag-fusion proteins were separated by SDS–PAGE. (<strong>A</strong>) The recombinant proteins were detected by Western blot analysis using anti-HaloTag antibody. Three bands indicated by asterisks are the signals derived from wheat germ endogenous proteins. (<strong>B</strong>) The recombinant proteins were labeled by the HaloTag-TMR ligand after the reaction and detected with FLA3000. The corresponding gene numbers for the HaloTag-fusion recombinant proteins in each lane of (A) and (B) are indicated as follows: 1, KIAA1231; 2, KIAA0720; 3, KIAA1426; 4, KIAA0386; 5, KIAA0651; 6, KIAA0010; 7, KIAA0543; 8, KIAA0987; 9, KIAA1386; 10, KIAA1902; 11, KIAA0594; 12, KIAA1450; 13, KIAA1973; 14, KIAA0055; 15, KIAA0835; 16, KIAA1531; 17, KIAA0750; 18, KIAA1714; 19, KIAA1738; 20, KIAA1106; 21, KIAA1525; 22, KIAA0229; 23, KIAA0182; 24, KIAA0954; 25, KIAA1904; 26, KIAA0020; 27, KIAA0028; 28, KIAA0076; 29, KIAA0150; 30, KIAA0152; 31, KIAA0153; 32, KIAA0168; 33, KIAA0186; 34, KIAA0205; 35, KIAA0223; 36, KIAA0278; 37, KIAA0680; 38, KIAA1184; 39, KIAA1245; 40, KIAA1399; 41, KIAA1871; 42, KIAA1857; 43, KIAA1912; 44, KIAA1916; 45, KIAA1917; 46, KIAA1942; 47, KIAA1949; and 48, KIAA1957. The positions of MagicMark XP Western Protein Standard (M) are indicated in kDa on the left. Relationship between TMR-ligand-binding activities and sizes of HaloTag-fusion proteins synthesized in the T<span class="small-caps">n</span>T SP6 HYPE system. (<strong>C</strong>) Fluorescence intensities of the HaloTag-TMR ligand covalently bound to the recombinant proteins (vertical axis) are plotted against the number of amino acid residues without HaloTag (horizontal axis). (<strong>D</strong>) Yields of the HaloTag-fusion proteins were estimated as the radioactivity of [<sup>35</sup>S]methionine and [<sup>35</sup>S]cysteine-labeled proteins calculated by BAS2000 imaging analyzer after SDS–PAGE and the value was divided by the number that contained methionine and cysteine. Yields (vertical axis) are plotted against the number of amino acid residues (horizontal axis) (closed circles). (<strong>E</strong>) HaloTag ligand-binding activity was estimated by measuring the fluorescence intensities of the HaloTag Cy5 ligand covalently bound to the HaloTag-fusion proteins after SDS–PAGE by FluoroImager FLA3000. The activities (vertical axis) are plotted against the number of amino acid residues (horizontal axis) (gray circles). (<strong>F</strong>) Ratios of HaloTag ligand-binding activities to HaloTag-fusion protein yields (vertical axis) are plotted against the number of amino acid residues (horizontal axis) (open circles). The 53 human genes used for protein production in (D), (E), and (F) are as follows: KIAA0024, KIAA0027, KIAA0057, KIAA0137, KIAA0151, KIAA0199, KIAA0230, KIAA0238, KIAA0352, KIAA0412, KIAA0416, KIAA0422, KIAA0441, KIAA0550, KIAA0557, KIAA0606, KIAA0623, KIAA0644, KIAA0722, KIAA0798, KIAA0813, KIAA0848, KIAA0936, KIAA0939, KIAA0968, KIAA1020, KIAA1048, KIAA1142, KIAA1198, KIAA1225, KIAA1227, KIAA1246, KIAA1253, KIAA1264, KIAA1337, KIAA1369, KIAA1382, KIAA1431, KIAA1469, KIAA1477, KIAA1559, KIAA1572, KIAA1580, KIAA1851, KIAA1852, KIAA1874, KIAA1910, KIAA2003, ABCA3, PAK6, STK38, PINK1, and GPRC5.</p></div></div><div content-id="DSN004F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn004/2/m_dsn00402.jpeg?Expires=2147483647&Signature=DHspYCYXpv0jkH3~mu0b1LfnUmWduHWSeoUsGsxou-2HrhZj67ihxXtSm3OV6Cc~mFr~SZuEmGh9EmIq-Nss1YfuU17lnehS02FW9cLJ~9ZSgixn9YpDZ8fCfKVU4tkyEvk0RRUdx3x-oohETmoHd9VK-TCHZmQNdG02FB8gw9iDjssbhAe7XeaStAioFa1h0R-SPBOC6EIM5VTcinO854ID6PKuPQ4vKYcEazHHyJX4aGXOfSUotLOaQM-hTDOY7tq5IHahxJAKToHgzDZZ3K2yEwyrkrxHKVOywJysoER2FNBhKgA3LRslJam9hpRHg0Sa7sDT~mCoZyAWCJfZFg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression of recombinant HaloTag-fusion proteins in an in vitro transcription/translation coupled protein synthesis system. Example of HaloTag-fusion proteins expressed in vitro. TempliPhi-amplified 48 pF3AHT DNA clones were used for an in vitro transcription/translation system using wheat germ extract (TnT SP6 HYPE System). The recombinant HaloTag-fusion proteins were separated by SDS–PAGE. (A) The recombinant proteins were detected by Western blot analysis using anti-HaloTag antibody. Three bands indicated by asterisks are the signals derived from wheat germ endogenous proteins. (B) The recombinant proteins were labeled by the HaloTag-TMR ligand after the reaction and detected with FLA3000. The corresponding gene numbers for the HaloTag-fusion recombinant proteins in each lane of (A) and (B) are indicated as follows: 1, KIAA1231; 2, KIAA0720; 3, KIAA1426; 4, KIAA0386; 5, KIAA0651; 6, KIAA0010; 7, KIAA0543; 8, KIAA0987; 9, KIAA1386; 10, KIAA1902; 11, KIAA0594; 12, KIAA1450; 13, KIAA1973; 14, KIAA0055; 15, KIAA0835; 16, KIAA1531; 17, KIAA0750; 18, KIAA1714; 19, KIAA1738; 20, KIAA1106; 21, KIAA1525; 22, KIAA0229; 23, KIAA0182; 24, KIAA0954; 25, KIAA1904; 26, KIAA0020; 27, KIAA0028; 28, KIAA0076; 29, KIAA0150; 30, KIAA0152; 31, KIAA0153; 32, KIAA0168; 33, KIAA0186; 34, KIAA0205; 35, KIAA0223; 36, KIAA0278; 37, KIAA0680; 38, KIAA1184; 39, KIAA1245; 40, KIAA1399; 41, KIAA1871; 42, KIAA1857; 43, KIAA1912; 44, KIAA1916; 45, KIAA1917; 46, KIAA1942; 47, KIAA1949; and 48, KIAA1957. The positions of MagicMark XP Western Protein Standard (M) are indicated in kDa on the left. Relationship between TMR-ligand-binding activities and sizes of HaloTag-fusion proteins synthesized in the TnT SP6 HYPE system. (C) Fluorescence intensities of the HaloTag-TMR ligand covalently bound to the recombinant proteins (vertical axis) are plotted against the number of amino acid residues without HaloTag (horizontal axis). (D) Yields of the HaloTag-fusion proteins were estimated as the radioactivity of [35S]methionine and [35S]cysteine-labeled proteins calculated by BAS2000 imaging analyzer after SDS–PAGE and the value was divided by the number that contained methionine and cysteine. Yields (vertical axis) are plotted against the number of amino acid residues (horizontal axis) (closed circles). (E) HaloTag ligand-binding activity was estimated by measuring the fluorescence intensities of the HaloTag Cy5 ligand covalently bound to the HaloTag-fusion proteins after SDS–PAGE by FluoroImager FLA3000. The activities (vertical axis) are plotted against the number of amino acid residues (horizontal axis) (gray circles). (F) Ratios of HaloTag ligand-binding activities to HaloTag-fusion protein yields (vertical axis) are plotted against the number of amino acid residues (horizontal axis) (open circles). The 53 human genes used for protein production in (D), (E), and (F) are as follows: KIAA0024, KIAA0027, KIAA0057, KIAA0137, KIAA0151, KIAA0199, KIAA0230, KIAA0238, KIAA0352, KIAA0412, KIAA0416, KIAA0422, KIAA0441, KIAA0550, KIAA0557, KIAA0606, KIAA0623, KIAA0644, KIAA0722, KIAA0798, KIAA0813, KIAA0848, KIAA0936, KIAA0939, KIAA0968, KIAA1020, KIAA1048, KIAA1142, KIAA1198, KIAA1225, KIAA1227, KIAA1246, KIAA1253, KIAA1264, KIAA1337, KIAA1369, KIAA1382, KIAA1431, KIAA1469, KIAA1477, KIAA1559, KIAA1572, KIAA1580, KIAA1851, KIAA1852, KIAA1874, KIAA1910, KIAA2003, ABCA3, PAK6, STK38, PINK1, and GPRC5." path-from-xml="dsn00402.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6967054/dsn00402.jpeg" path-from-xml="dsn00402.jpeg" target="_blank">View large</a><a section="6967054" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn004/2/dsn00402.jpeg?Expires=2147483647&Signature=sEGT6aAurxgQT4pN1ll-~sW3M3RkY3jX2gdUJQQwEOTo3NkXiNFEOnOs0VwlCjxUGbQUx6NrvWt9pDg41BT8s271LBSX4gDMbQ5HPZBVwx6EC6A664uCwMhrH3tXhDdo1nnC-wUH4EelgHFx3elSb2GoHJGQ3Odt50Fh8UY9rAvYjNWhRTxmJIY6JNnMNnjms4mb8oKnVus0TkeC5LYNjj8XuxtCBSFVzp0-KESeMiMhI1cx-Qbrm8tU22-pV3M3ZpveuM7Gq0JXihGbDaQESEELO62Zhg~BELlXMf9DZa~RzkwCffnBoLNzgRWZa4jYL5E3ptjdt5~SJ8xCa~68BA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6967054&ar=464067&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn00402.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression of recombinant HaloTag-fusion proteins in an <em>in vitro</em> transcription/translation coupled protein synthesis system. Example of HaloTag-fusion proteins expressed <em>in vitro</em>. TempliPhi-amplified 48 pF3AHT DNA clones were used for an <em>in vitro</em> transcription/translation system using wheat germ extract (T<span class="small-caps">n</span>T SP6 HYPE System). The recombinant HaloTag-fusion proteins were separated by SDS–PAGE. (<strong>A</strong>) The recombinant proteins were detected by Western blot analysis using anti-HaloTag antibody. Three bands indicated by asterisks are the signals derived from wheat germ endogenous proteins. (<strong>B</strong>) The recombinant proteins were labeled by the HaloTag-TMR ligand after the reaction and detected with FLA3000. The corresponding gene numbers for the HaloTag-fusion recombinant proteins in each lane of (A) and (B) are indicated as follows: 1, KIAA1231; 2, KIAA0720; 3, KIAA1426; 4, KIAA0386; 5, KIAA0651; 6, KIAA0010; 7, KIAA0543; 8, KIAA0987; 9, KIAA1386; 10, KIAA1902; 11, KIAA0594; 12, KIAA1450; 13, KIAA1973; 14, KIAA0055; 15, KIAA0835; 16, KIAA1531; 17, KIAA0750; 18, KIAA1714; 19, KIAA1738; 20, KIAA1106; 21, KIAA1525; 22, KIAA0229; 23, KIAA0182; 24, KIAA0954; 25, KIAA1904; 26, KIAA0020; 27, KIAA0028; 28, KIAA0076; 29, KIAA0150; 30, KIAA0152; 31, KIAA0153; 32, KIAA0168; 33, KIAA0186; 34, KIAA0205; 35, KIAA0223; 36, KIAA0278; 37, KIAA0680; 38, KIAA1184; 39, KIAA1245; 40, KIAA1399; 41, KIAA1871; 42, KIAA1857; 43, KIAA1912; 44, KIAA1916; 45, KIAA1917; 46, KIAA1942; 47, KIAA1949; and 48, KIAA1957. The positions of MagicMark XP Western Protein Standard (M) are indicated in kDa on the left. Relationship between TMR-ligand-binding activities and sizes of HaloTag-fusion proteins synthesized in the T<span class="small-caps">n</span>T SP6 HYPE system. (<strong>C</strong>) Fluorescence intensities of the HaloTag-TMR ligand covalently bound to the recombinant proteins (vertical axis) are plotted against the number of amino acid residues without HaloTag (horizontal axis). (<strong>D</strong>) Yields of the HaloTag-fusion proteins were estimated as the radioactivity of [<sup>35</sup>S]methionine and [<sup>35</sup>S]cysteine-labeled proteins calculated by BAS2000 imaging analyzer after SDS–PAGE and the value was divided by the number that contained methionine and cysteine. Yields (vertical axis) are plotted against the number of amino acid residues (horizontal axis) (closed circles). (<strong>E</strong>) HaloTag ligand-binding activity was estimated by measuring the fluorescence intensities of the HaloTag Cy5 ligand covalently bound to the HaloTag-fusion proteins after SDS–PAGE by FluoroImager FLA3000. The activities (vertical axis) are plotted against the number of amino acid residues (horizontal axis) (gray circles). (<strong>F</strong>) Ratios of HaloTag ligand-binding activities to HaloTag-fusion protein yields (vertical axis) are plotted against the number of amino acid residues (horizontal axis) (open circles). The 53 human genes used for protein production in (D), (E), and (F) are as follows: KIAA0024, KIAA0027, KIAA0057, KIAA0137, KIAA0151, KIAA0199, KIAA0230, KIAA0238, KIAA0352, KIAA0412, KIAA0416, KIAA0422, KIAA0441, KIAA0550, KIAA0557, KIAA0606, KIAA0623, KIAA0644, KIAA0722, KIAA0798, KIAA0813, KIAA0848, KIAA0936, KIAA0939, KIAA0968, KIAA1020, KIAA1048, KIAA1142, KIAA1198, KIAA1225, KIAA1227, KIAA1246, KIAA1253, KIAA1264, KIAA1337, KIAA1369, KIAA1382, KIAA1431, KIAA1469, KIAA1477, KIAA1559, KIAA1572, KIAA1580, KIAA1851, KIAA1852, KIAA1874, KIAA1910, KIAA2003, ABCA3, PAK6, STK38, PINK1, and GPRC5.</p></div></div>                            <h3 scrollto-destination=6967055 id="6967055" class="section-title" data-legacy-id=s3e>3.5. Subcellular localization of HaloTag-fusion proteins and MGFP-fusion proteins</h3>
<p>In general, information on the subcellular localization of proteins provides us with an important clue on potential protein function. Thus, we accumulated data of the subcellular localizations using green fluorescent protein (GFP)–ORF fusions constructed using the Gateway system. It was important for us to confirm that the subcellular localization revealed by HaloTag-fusions was consistent with the subcellular localization observed for GFP-fusions. First, we examined whether or not the HaloTag protein could be located in an expected subcellular compartment upon fusion of localization signals in COS-7 cells. To do this, we prepared pFC8A expression clones for the HaloTag protein containing signal sequences, such as membane-localized signal (Memb),<sup><a reveal-id="DSN004C21" data-open="DSN004C21" class="link link-ref link-reveal xref-bibr">21</a></sup> nuclear localization signal (NLS),<sup><a reveal-id="DSN004C22" data-open="DSN004C22" class="link link-ref link-reveal xref-bibr">22</a></sup> endoplasmic reticulum targeting sequence (ER),<sup><a reveal-id="DSN004C23" data-open="DSN004C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Golgi-localized signal (Golgi),<sup><a reveal-id="DSN004C24" data-open="DSN004C24" class="link link-ref link-reveal xref-bibr">24</a></sup> and mitochondrial targeting sequence (Mito).<sup><a reveal-id="DSN004C25" data-open="DSN004C25" class="link link-ref link-reveal xref-bibr">25</a></sup> These sequences were all placed at the N-terminal region of the HaloTag proteins. Eighteen hours after transfection of these constructs into COS-7 cells, the HaloTag-fusion proteins were labeled with the TMR ligand and observed with a fluorescence microscopy system. Figure <a reveal-id="DSN004F4" data-open="DSN004F4" class="link link-reveal link-table xref-fig">4</a>A shows that all fused proteins were detected in an expected subcellular compartment, indicating that HaloTag proteins do not inhibit the function of these subcellular localization signals. Next, we directly evaluated whether HaloTag-fusion proteins were co-localized with the same GFP-fusion proteins when both fusions were expressed in the same cells. C-terminal HaloTag- and MGFP-fusion protein expression clones were prepared by the Flexi<sup>®</sup> cloning system using the Flexi ORF clones we prepared and pFC8A (HaloTag) and pFC8A (MGFP) vectors. After co-transfection of HEK293 cells with the same ORF clone fused to the two different fusion tags, HaloTag-fusion proteins were labeled with the TMR ligand and then the fluorescence images were obtained for TMR- (red) and MGFP-labeled (green) proteins for 40 ORFs. Examples of the cellular localization of the fusion proteins are shown in Fig. <a reveal-id="DSN004F4" data-open="DSN004F4" class="link link-reveal link-table xref-fig">4</a>B. Among them, subcellular locations of glucocorticoid modulatory element-binding protein 2 (GMEB-2/KIAA1269), phosphatidylserine synthase 1 (PTDSS1/KIAA0024), membrane-bound transcription factor site-1 protease (MBTPS1/KIAA0091), and transmembrane protein 127 (TMEM127/KIBB2508) are reported in the UniProt Knowledgebase as nucleus/cytoplasm, membrane, endoplasmic reticulum/Golgi apparatus, and membrane, respectively (<a class="link link-uri" href="http://ca.expasy.org/sprot/">http://ca.expasy.org/sprot/</a>).<sup><a reveal-id="DSN004C26" data-open="DSN004C26" class="link link-ref link-reveal xref-bibr">26</a></sup> In contrast, the subcellular location of Mesoderm development candidate 2 (MESDC2/KIAA0081) is not mentioned and Secernin-1 (SCRN1/KIAA0193) is predicted to exist in the cytoplasm from similarities found in the database. The subcellular localizations of the 40 HaloTag-fusion proteins analyzed here were the same as those for MGFP-fusion proteins. These results indicate that HaloTag-fusion proteins can be used as an alternative for subcellular localization analysis in place of conventional autofluorescent proteins.</p>                        <a id="6967057" scrollto-destination="6967057"></a>
<div id="DSN004F4" data-id="DSN004F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn004/2/m_dsn00403.jpeg?Expires=2147483647&Signature=ykIUyfOun3R3B3p-~q6k93OOL-DZto3lAPZmVe7nNprYLzOmeXM-a5ZiPXnVVqh0cUCTiMEldfEOHLywnOQlRSN1wASmxyKHbP62s-v19gkz4XFGXRofQBAnYXFhxLUHByrpGPiZqricmVjMt-f69jnHrSBkBZM2InteQROFzibQ4WjHGWbU9OJaMBGo6SQ~orMSo-z473a4iNhH7ohKmKIq9iLwlFxpXfPNcxOrP33mZiBFGEbox15wNZGvu30jNTjZIq-pnEcuTbIXlZuUuuQEx-LZrusKiW-E9Urk0sAe3sV1S2tlASt2eJDRxQoP0x3Xi7IRh5cv2IFa8w-2vA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Subcellular localization of HaloTag-fusion proteins in cultured cells. (A) Subcellular localization of transiently expressed HaloTag proteins with various signal sequences. COS-7 cells were transfected with pFC8A expression clones for HaloTag proteins containing signal sequences, such as plasma-membrane-localized signal (Memb), nuclear localization signal (NLS), ER targeting sequence (ER), Golgi-localized signal (Golgi), and mitochondria-localized signal (Mito) and HaloTag pHT2 vector which expresses HaloTag ORF (HaloTag). HaloTag-fusion proteins were labeled with medium containing 1-mM HaloTag-TMR ligand (red) for 15 min, washed and incubated for 30 min. Nuclei were stained with Hoechst33342 (blue). Cells were observed by BioZERO fluorescence microscope (TMR). Photos labeled with Hoechst33342 and TMR ligand are merged (TMR + Hoechst). (B) Comparison of subcellular localizations between HaloTag- and MGFP-fusion proteins. HEK293 cells were simultaneously transfected with HaloTag-fusion (red) and MGFP-fusion protein (green) expression clones. The HaloTag-fusion proteins (red) and nuclei (blue) were observed as described above, except that the final incubation before observation was 24 h. Photos labeled with Hoechst33342 are merged with those of MGFP- or HaloTag-fusion proteins (MGFP or HaloTag). Gene symbols and alias names are indicated beneath the photos." path-from-xml="dsn00403.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6967057/dsn00403.jpeg" path-from-xml="dsn00403.jpeg" target="_blank">View large</a><a section="6967057" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn004/2/dsn00403.jpeg?Expires=2147483647&Signature=Kl24T4WE7RJ1hNw-glm-4VVDYsjOR54tgGxjNsH8E0DYxbn9nacD8ZwKnkMUt1VA53zErQ-2DHYPqtuEK2N8y9BfyMGVjGE06hG5yj8OouknGUIU0ecg6SG-tmVGV31CyNT0IE48yt5Qscc7qZ~rJJbz4IxMRQVOaGzCrbmsgkJn5aANGeoLMI43xNNSlzZuQ95cAGpaAuvphjkeOKa-fhHVRAfMzKKckLoWx8ByM8JkKB5fWt7iIvUSDcZ1bVEm91gISP-tifBQ~4L2HmRNbZbXGLv~drUXTkrizBeJigqwlH8VW50dP3SZfB~lckuGnco~7rMXEE1Rj-6sxQPLJQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6967057&ar=464067&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn00403.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Subcellular localization of HaloTag-fusion proteins in cultured cells. (<strong>A</strong>) Subcellular localization of transiently expressed HaloTag proteins with various signal sequences. COS-7 cells were transfected with pFC8A expression clones for HaloTag proteins containing signal sequences, such as plasma-membrane-localized signal (Memb), nuclear localization signal (NLS), ER targeting sequence (ER), Golgi-localized signal (Golgi), and mitochondria-localized signal (Mito) and HaloTag pHT2 vector which expresses HaloTag ORF (HaloTag). HaloTag-fusion proteins were labeled with medium containing 1-mM HaloTag-TMR ligand (red) for 15 min, washed and incubated for 30 min. Nuclei were stained with Hoechst33342 (blue). Cells were observed by BioZERO fluorescence microscope (TMR). Photos labeled with Hoechst33342 and TMR ligand are merged (TMR + Hoechst). (<strong>B</strong>) Comparison of subcellular localizations between HaloTag- and MGFP-fusion proteins. HEK293 cells were simultaneously transfected with HaloTag-fusion (red) and MGFP-fusion protein (green) expression clones. The HaloTag-fusion proteins (red) and nuclei (blue) were observed as described above, except that the final incubation before observation was 24 h. Photos labeled with Hoechst33342 are merged with those of MGFP- or HaloTag-fusion proteins (MGFP or HaloTag). Gene symbols and alias names are indicated beneath the photos.</p></div></div><div content-id="DSN004F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn004/2/m_dsn00403.jpeg?Expires=2147483647&Signature=ykIUyfOun3R3B3p-~q6k93OOL-DZto3lAPZmVe7nNprYLzOmeXM-a5ZiPXnVVqh0cUCTiMEldfEOHLywnOQlRSN1wASmxyKHbP62s-v19gkz4XFGXRofQBAnYXFhxLUHByrpGPiZqricmVjMt-f69jnHrSBkBZM2InteQROFzibQ4WjHGWbU9OJaMBGo6SQ~orMSo-z473a4iNhH7ohKmKIq9iLwlFxpXfPNcxOrP33mZiBFGEbox15wNZGvu30jNTjZIq-pnEcuTbIXlZuUuuQEx-LZrusKiW-E9Urk0sAe3sV1S2tlASt2eJDRxQoP0x3Xi7IRh5cv2IFa8w-2vA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Subcellular localization of HaloTag-fusion proteins in cultured cells. (A) Subcellular localization of transiently expressed HaloTag proteins with various signal sequences. COS-7 cells were transfected with pFC8A expression clones for HaloTag proteins containing signal sequences, such as plasma-membrane-localized signal (Memb), nuclear localization signal (NLS), ER targeting sequence (ER), Golgi-localized signal (Golgi), and mitochondria-localized signal (Mito) and HaloTag pHT2 vector which expresses HaloTag ORF (HaloTag). HaloTag-fusion proteins were labeled with medium containing 1-mM HaloTag-TMR ligand (red) for 15 min, washed and incubated for 30 min. Nuclei were stained with Hoechst33342 (blue). Cells were observed by BioZERO fluorescence microscope (TMR). Photos labeled with Hoechst33342 and TMR ligand are merged (TMR + Hoechst). (B) Comparison of subcellular localizations between HaloTag- and MGFP-fusion proteins. HEK293 cells were simultaneously transfected with HaloTag-fusion (red) and MGFP-fusion protein (green) expression clones. The HaloTag-fusion proteins (red) and nuclei (blue) were observed as described above, except that the final incubation before observation was 24 h. Photos labeled with Hoechst33342 are merged with those of MGFP- or HaloTag-fusion proteins (MGFP or HaloTag). Gene symbols and alias names are indicated beneath the photos." path-from-xml="dsn00403.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6967057/dsn00403.jpeg" path-from-xml="dsn00403.jpeg" target="_blank">View large</a><a section="6967057" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn004/2/dsn00403.jpeg?Expires=2147483647&Signature=Kl24T4WE7RJ1hNw-glm-4VVDYsjOR54tgGxjNsH8E0DYxbn9nacD8ZwKnkMUt1VA53zErQ-2DHYPqtuEK2N8y9BfyMGVjGE06hG5yj8OouknGUIU0ecg6SG-tmVGV31CyNT0IE48yt5Qscc7qZ~rJJbz4IxMRQVOaGzCrbmsgkJn5aANGeoLMI43xNNSlzZuQ95cAGpaAuvphjkeOKa-fhHVRAfMzKKckLoWx8ByM8JkKB5fWt7iIvUSDcZ1bVEm91gISP-tifBQ~4L2HmRNbZbXGLv~drUXTkrizBeJigqwlH8VW50dP3SZfB~lckuGnco~7rMXEE1Rj-6sxQPLJQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6967057&ar=464067&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn00403.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Subcellular localization of HaloTag-fusion proteins in cultured cells. (<strong>A</strong>) Subcellular localization of transiently expressed HaloTag proteins with various signal sequences. COS-7 cells were transfected with pFC8A expression clones for HaloTag proteins containing signal sequences, such as plasma-membrane-localized signal (Memb), nuclear localization signal (NLS), ER targeting sequence (ER), Golgi-localized signal (Golgi), and mitochondria-localized signal (Mito) and HaloTag pHT2 vector which expresses HaloTag ORF (HaloTag). HaloTag-fusion proteins were labeled with medium containing 1-mM HaloTag-TMR ligand (red) for 15 min, washed and incubated for 30 min. Nuclei were stained with Hoechst33342 (blue). Cells were observed by BioZERO fluorescence microscope (TMR). Photos labeled with Hoechst33342 and TMR ligand are merged (TMR + Hoechst). (<strong>B</strong>) Comparison of subcellular localizations between HaloTag- and MGFP-fusion proteins. HEK293 cells were simultaneously transfected with HaloTag-fusion (red) and MGFP-fusion protein (green) expression clones. The HaloTag-fusion proteins (red) and nuclei (blue) were observed as described above, except that the final incubation before observation was 24 h. Photos labeled with Hoechst33342 are merged with those of MGFP- or HaloTag-fusion proteins (MGFP or HaloTag). Gene symbols and alias names are indicated beneath the photos.</p></div></div><p>In conclusion, a complete set of ORF resources in various expression-ready clone formats clearly can serve as versatile sets of reagents for functional genomics research. The rapid, easy, and assured transfer of ORF sequences into different expression vectors is highly critical for this purpose, since many ORF clones are used in combination with protein fusion tags that allow functional analysis. Within this context, we selected a HaloTag protein as a fusion partner because we considered that HaloTag<sup>®</sup> technology would provide a wide range of utilities that include detection and purification as well as adaptability in functional studies. In fact, we have successfully used HaloTag-fusion proteins in other studies for pulse-chase labeling, protein–protein interaction, and chromosomal immunoprecipitation-like analyses along with bioimaging (unpublished data). In this report, we have shown that the Flexi<sup>®</sup> cloning method is suitable for the preparation of a large number of ORF clones into the pF1K vector and allows seamless implementation of HaloTag<sup>®</sup> interchangeable technology. Moreover, we have demonstrated that most human proteins were efficiently expressed as HaloTag-fusions in an <em>in vitro</em> coupled protein expression system, as well as in cultured mammalian cells. The resultant HaloTag-fusion proteins were biochemically active and successfully used for analyses of subcellular localization as an alternative to conventional autofluorescent proteins. Based on these results, we will accumulate a number of human ORF clones in the form of expression plasmids for HaloTag-fusion proteins, either C- or N-terminal fusions, in our ORFeome project since they will be directly used for functional studies, as well as for further transfer to another vector. HaloTag<sup>®</sup> technology provides a single fusion tag that can be utilized for a variety of structural and functional proteomic studies. Although there are a number of ORFeome resources for human genes that have been generated by Gateway recombinational cloning,<sup><a reveal-id="DSN004C27" data-open="DSN004C27" class="link link-ref link-reveal xref-bibr">27</a></sup> a set of human Flexi ORF clones prepared as described would provide the research community with an important alternative for large-scale proteomic studies and a tool for individual research laboratories.</p>                            <h2 scrollto-destination=6967059 id="6967059" class="section-title" data-legacy-id=s4>Supplementary material</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available online at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=6967061 id="6967061" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This project was supported in part by a grant for Promega‐Kazusa scientific collaboration from Promega Corporation and by a special grant for acceleration of the practical application of biotechnology from Chiba Prefectural Government.</p>                            <h2 scrollto-destination=6967063 id="6967063" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We would like to thank J. Hartnett and M. Slater for their gift of the pF1Kof vector and A. Miyawaki for providing us with venus/pCS2 plasmid. We are grateful to B. Bulleit for his valuable comments. We also thank K. Ozawa, K. Sumi, T. Watanabe, K. Yamada, S. Yokoyama, M. Takazawa, N. Kashima, M. Tamura, H. Kinoshita, E. Suzuki, and C. Mori for their technical assistance.</p>                            <h2 scrollto-destination=6967065 id="6967065" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSN004C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN004C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brent</div> <div class="given-names">M. R</div></div>. </div><div class="article-title">Genome annotation past, present, and future: how to define an ORF at each locus</div>, <div class="source">Genome Res.</div> , <div class="year">2006</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">1777</div>-<div class="lpage">1786</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20annotation%20past%2C%20present%2C%20and%20future%3A%20how%20to%20define%20an%20ORF%20at%20each%20locus&amp;author=M.%20R%20Brent&amp;publication_year=2006&amp;journal=Genome%20Res.&amp;volume=15&amp;pages=1777-1786" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.3866105" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.3866105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.3866105"> </span></div> </div></div></div></div></div><div content-id="DSN004C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN004C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hubbard</div> <div class="given-names">T. J. P.</div></div>,  <div class="name"><div class="surname">Aken</div> <div class="given-names">B. L.</div></div>,  <div class="name"><div class="surname">Beal</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Ensembl 2007</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">D610</div>-<div class="lpage">D617</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ensembl%202007&amp;author=T.%20J.%20P.%20Hubbard&amp;author=B.%20L.%20Aken&amp;author=K.%20Beal&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=D610-D617" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl996" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl996" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl996"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17148474" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN004C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN004C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rual</div> <div class="given-names">J. F.</div></div>,  <div class="name"><div class="surname">Hill</div> <div class="given-names">D. E.</div></div>,  <div class="name"><div class="surname">Vidal</div> <div class="given-names">M.</div></div>. </div><div class="article-title">ORFeome projects: gateway between genomics and omics</div>, <div class="source">Curr. Opin. Chem. Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">20</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ORFeome%20projects%3A%20gateway%20between%20genomics%20and%20omics&amp;author=J.%20F.%20Rual&amp;author=D.%20E.%20Hill&amp;author=M.%20Vidal&amp;publication_year=2004&amp;journal=Curr.%20Opin.%20Chem.%20Biol.&amp;volume=8&amp;pages=20-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cbpa.2003.12.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cbpa.2003.12.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cbpa.2003.12.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15036152" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN004C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN004C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brizuela</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Richardson</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Marsischky</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Labaer</div> <div class="given-names">J.</div></div>. </div><div class="article-title">The FLEXGene repository: exploiting the fruits of the genome projects by creating a needed resource to face the challenges of the post-genomic era</div>, <div class="source">Arch. Med. Res.</div> , <div class="year">2002</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">318</div>-<div class="lpage">324</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20FLEXGene%20repository%3A%20exploiting%20the%20fruits%20of%20the%20genome%20projects%20by%20creating%20a%20needed%20resource%20to%20face%20the%20challenges%20of%20the%20post-genomic%20era&amp;author=L.%20Brizuela&amp;author=A.%20Richardson&amp;author=G.%20Marsischky&amp;author=J.%20Labaer&amp;publication_year=2002&amp;journal=Arch.%20Med.%20Res.&amp;volume=33&amp;pages=318-324" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0188-4409(02)00372-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0188-4409(02)00372-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0188-4409(02)00372-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12234520" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN004C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN004C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Temple</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Lamesch</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Milstein</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">From genome to proteome: developing expression clone resources for the human genome</div>, <div class="source">Hum. Mol. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">R31</div>-<div class="lpage">R43</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=From%20genome%20to%20proteome%3A%20developing%20expression%20clone%20resources%20for%20the%20human%20genome&amp;author=G.%20Temple&amp;author=P.%20Lamesch&amp;author=S.%20Milstein&amp;publication_year=2006&amp;journal=Hum.%20Mol.%20Genet.&amp;volume=15&amp;pages=R31-R43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/hmg/ddl048" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fhmg%2Fddl048" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fhmg%2Fddl048"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16651367" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN004C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN004C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Walhout</div> <div class="given-names">A. J.</div></div>,  <div class="name"><div class="surname">Temple</div> <div class="given-names">G. F.</div></div>,  <div class="name"><div class="surname">Brasch</div> <div class="given-names">M. A.</div></div>, et al. </div><div class="article-title">GATEWAY recombinational cloning: application to the cloning of large numbers of open reading frames or ORFeomes</div>, <div class="source">Methods Enzymol.</div> , <div class="year">2000</div>, vol. <div class="volume">328</div> (pg. <div class="fpage">575</div>-<div class="lpage">592</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GATEWAY%20recombinational%20cloning%3A%20application%20to%20the%20cloning%20of%20large%20numbers%20of%20open%20reading%20frames%20or%20ORFeomes&amp;author=A.%20J.%20Walhout&amp;author=G.%20F.%20Temple&amp;author=M.%20A.%20Brasch&amp;publication_year=2000&amp;journal=Methods%20Enzymol.&amp;volume=328&amp;pages=575-592" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11075367" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN004C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN004C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Marsischky</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">LaBaer</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Many paths to many clones: a comparative look at high-throughput cloning methods</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">2020</div>-<div class="lpage">2028</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Many%20paths%20to%20many%20clones%3A%20a%20comparative%20look%20at%20high-throughput%20cloning%20methods&amp;author=G.%20Marsischky&amp;author=J.%20LaBaer&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=2020-2028" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.2528804" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.2528804" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.2528804"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15489321" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN004C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN004C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Blommel</div> <div class="given-names">P. G.</div></div>,  <div class="name"><div class="surname">Martin</div> <div class="given-names">P. A.</div></div>,  <div class="name"><div class="surname">Wrobel</div> <div class="given-names">R. L.</div></div>,  <div class="name"><div class="surname">Steffen</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Fox</div> <div class="given-names">B. G.</div></div>. </div><div class="article-title">High efficiency single step production of expression plasmids from cDNA clones using the Flexi Vector cloning system</div>, <div class="source">Protein Expr. Purif.</div> , <div class="year">2006</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">562</div>-<div class="lpage">570</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High%20efficiency%20single%20step%20production%20of%20expression%20plasmids%20from%20cDNA%20clones%20using%20the%20Flexi%20Vector%20cloning%20system&amp;author=P.%20G.%20Blommel&amp;author=P.%20A.%20Martin&amp;author=R.%20L.%20Wrobel&amp;author=E.%20Steffen&amp;author=B.%20G.%20Fox&amp;publication_year=2006&amp;journal=Protein%20Expr.%20Purif.&amp;volume=47&amp;pages=562-570" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pep.2005.11.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pep.2005.11.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pep.2005.11.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16377204" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN004C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN004C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakajima</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Saito</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yamakawa</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Preparation of a set of expression-ready clones of mammalian long cDNAs encoding large proteins by the ORF trap cloning method</div>, <div class="source">DNA Res.</div> , <div class="year">2005</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">257</div>-<div class="lpage">267</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Preparation%20of%20a%20set%20of%20expression-ready%20clones%20of%20mammalian%20long%20cDNAs%20encoding%20large%20proteins%20by%20the%20ORF%20trap%20cloning%20method&amp;author=D.%20Nakajima&amp;author=K.%20Saito&amp;author=H.%20Yamakawa&amp;publication_year=2005&amp;journal=DNA%20Res.&amp;volume=12&amp;pages=257-267" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsi010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsi010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsi010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16769688" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN004C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN004C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Los</div> <div class="given-names">V. G.</div></div>,  <div class="name"><div class="surname">Wood</div> <div class="given-names">K.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Taylor</div> <div class="given-names">D. L.</div></div>,  <div class="name"><div class="surname">Haskins</div> <div class="given-names">J. R.</div></div>,  <div class="name"><div class="surname">Giuliano</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Methods in Molecular Biology</div>, <div class="year">2007</div>, vol. <div class="volume">356</div> <div class="publisher-loc">Totowa</div><div class="publisher-name">Human Press</div>(pg. <div class="fpage">195</div>-<div class="lpage">208</div>)</div></div></div></div><div content-id="DSN004C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN004C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Keppler</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Gendreizig</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Gronemeyer</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Pick</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Vogel</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Johnsson</div> <div class="given-names">K.</div></div>. </div><div class="article-title">A general method for the covalent labeling of fusion proteins with small molecules <em>in vivo</em></div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2003</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">86</div>-<div class="lpage">89</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20general%20method%20for%20the%20covalent%20labeling%20of%20fusion%20proteins%20with%20small%20molecules%20in%20vivo&amp;author=A.%20Keppler&amp;author=S.%20Gendreizig&amp;author=T.%20Gronemeyer&amp;author=H.%20Pick&amp;author=H.%20Vogel&amp;author=K.%20Johnsson&amp;publication_year=2003&amp;journal=Nat.%20Biotechnol.&amp;volume=21&amp;pages=86-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt765" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt765" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt765"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12469133" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN004C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN004C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gronemeyer</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Godin</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Johnsson</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Adding value to fusion proteins through covalent labeling</div>, <div class="source">Curr. Opin. Biotechnol.</div> , <div class="year">2005</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">453</div>-<div class="lpage">458</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Adding%20value%20to%20fusion%20proteins%20through%20covalent%20labeling&amp;author=T.%20Gronemeyer&amp;author=G.%20Godin&amp;author=K.%20Johnsson&amp;publication_year=2005&amp;journal=Curr.%20Opin.%20Biotechnol.&amp;volume=16&amp;pages=453-458" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.copbio.2005.06.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.copbio.2005.06.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.copbio.2005.06.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15967656" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN004C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN004C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nagai</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ibata</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Park</div> <div class="given-names">E. S.</div></div>,  <div class="name"><div class="surname">Kubota</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Mikoshiba</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Miyawaki</div> <div class="given-names">A.</div></div>. </div><div class="article-title">A variant of yellow fluorescent protein with fast and efficient maturation for cell-biological applications</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2002</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">87</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20variant%20of%20yellow%20fluorescent%20protein%20with%20fast%20and%20efficient%20maturation%20for%20cell-biological%20applications&amp;author=T.%20Nagai&amp;author=T.%20Ibata&amp;author=E.%20S.%20Park&amp;author=M.%20Kubota&amp;author=K.%20Mikoshiba&amp;author=A.%20Miyawaki&amp;publication_year=2002&amp;journal=Nat.%20Biotechnol.&amp;volume=20&amp;pages=87-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt0102-87" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt0102-87" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt0102-87"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11753368" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN004C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN004C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohara</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Nagase</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ishikawa</div> <div class="given-names">K.-I.</div></div>, et al. </div><div class="article-title">Construction and characterization of human brain cDNA libraries suitable for analysis of cDNA clones encoding relatively large proteins</div>, <div class="source">DNA Res.</div> , <div class="year">1997</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">53</div>-<div class="lpage">59</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20and%20characterization%20of%20human%20brain%20cDNA%20libraries%20suitable%20for%20analysis%20of%20cDNA%20clones%20encoding%20relatively%20large%20proteins&amp;author=O.%20Ohara&amp;author=T.%20Nagase&amp;author=K.-I.%20Ishikawa&amp;publication_year=1997&amp;journal=DNA%20Res.&amp;volume=4&amp;pages=53-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/4.1.53" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F4.1.53" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F4.1.53"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9179496" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN004C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN004C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nagase</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Koga</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ohara</div> <div class="given-names">O</div></div>. </div><div class="article-title">Kazusa mammalian cDNA resources: towards functional characterization of KIAA gene products</div>, <div class="source">Brief. Funct. Genomic Proteomic.</div> , <div class="year">2006</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">4</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Kazusa%20mammalian%20cDNA%20resources%3A%20towards%20functional%20characterization%20of%20KIAA%20gene%20products&amp;author=T.%20Nagase&amp;author=H.%20Koga&amp;author=O%20Ohara&amp;publication_year=2006&amp;journal=Brief.%20Funct.%20Genomic%20Proteomic.&amp;volume=5&amp;pages=4-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bfgp/ell005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbfgp%2Fell005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbfgp%2Fell005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16769670" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN004C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN004C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gillen</div> <div class="given-names">J. R.</div></div>,  <div class="name"><div class="surname">Willis</div> <div class="given-names">D. K.</div></div>,  <div class="name"><div class="surname">Clark</div> <div class="given-names">A. J.</div></div>. </div><div class="article-title">Genetic analysis of the RecE pathway of genetic recombination in <em>Escherichia coli</em> K-12</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1981</div>, vol. <div class="volume">145</div> (pg. <div class="fpage">521</div>-<div class="lpage">532</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20analysis%20of%20the%20RecE%20pathway%20of%20genetic%20recombination%20in%20Escherichia%20coli%20K-12&amp;author=J.%20R.%20Gillen&amp;author=D.%20K.%20Willis&amp;author=A.%20J.%20Clark&amp;publication_year=1981&amp;journal=J.%20Bacteriol.&amp;volume=145&amp;pages=521-532" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6257642" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN004C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN004C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Oliner</div> <div class="given-names">J. D.</div></div>,  <div class="name"><div class="surname">Kinzler</div> <div class="given-names">K. W.</div></div>,  <div class="name"><div class="surname">Vogelstein</div> <div class="given-names">B.</div></div>. </div><div class="article-title"><em>In vivo</em> cloning of PCR products in <em>E. coli</em></div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1993</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">5192</div>-<div class="lpage">5197</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20vivo%20cloning%20of%20PCR%20products%20in%20E.%20coli&amp;author=J.%20D.%20Oliner&amp;author=K.%20W.%20Kinzler&amp;author=B.%20Vogelstein&amp;publication_year=1993&amp;journal=Nucleic%20Acids%20Res.&amp;volume=21&amp;pages=5192-5197" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/21.22.5192" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F21.22.5192" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F21.22.5192"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8255776" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN004C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN004C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Buchholz</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Muyrers</div> <div class="given-names">J. P.</div></div>,  <div class="name"><div class="surname">Stewart</div> <div class="given-names">A. F.</div></div>. </div><div class="article-title">A new logic for DNA engineering using recombination in <em>Escherichia coli</em></div>, <div class="source">Nature Genet.</div> , <div class="year">1998</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">123</div>-<div class="lpage">128</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20logic%20for%20DNA%20engineering%20using%20recombination%20in%20Escherichia%20coli&amp;author=Y.%20Zhang&amp;author=F.%20Buchholz&amp;author=J.%20P.%20Muyrers&amp;author=A.%20F.%20Stewart&amp;publication_year=1998&amp;journal=Nature%20Genet.&amp;volume=20&amp;pages=123-128" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/2417" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F2417" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F2417"> </span></div> </div></div></div></div></div><div content-id="DSN004C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN004C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suzuki</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ito</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ezure</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Performance of expression vector, pTD1, in insect cell-free translation system</div>, <div class="source">J. Biosci. Bioengineer.</div> , <div class="year">2006</div>, vol. <div class="volume">102</div> (pg. <div class="fpage">69</div>-<div class="lpage">71</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Performance%20of%20expression%20vector%2C%20pTD1%2C%20in%20insect%20cell-free%20translation%20system&amp;author=T.%20Suzuki&amp;author=M.%20Ito&amp;author=T.%20Ezure&amp;publication_year=2006&amp;journal=J.%20Biosci.%20Bioengineer.&amp;volume=102&amp;pages=69-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1263/jbb.102.69" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1263%2Fjbb.102.69" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1263%2Fjbb.102.69"> </span></div> </div></div></div></div></div><div content-id="DSN004C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN004C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dean</div> <div class="given-names">F. B.</div></div>,  <div class="name"><div class="surname">Nelson</div> <div class="given-names">J. R.</div></div>,  <div class="name"><div class="surname">Giesler</div> <div class="given-names">T. L.</div></div>,  <div class="name"><div class="surname">Lasken</div> <div class="given-names">R. S.</div></div>. </div><div class="article-title">Rapid amplification of plasmid and phage DNA using Phi29 DNA polymerase and multiply-primed rolling circle amplification</div>, <div class="source">Genome Res.</div> , <div class="year">2001</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">1095</div>-<div class="lpage">1099</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20amplification%20of%20plasmid%20and%20phage%20DNA%20using%20Phi29%20DNA%20polymerase%20and%20multiply-primed%20rolling%20circle%20amplification&amp;author=F.%20B.%20Dean&amp;author=J.%20R.%20Nelson&amp;author=T.%20L.%20Giesler&amp;author=R.%20S.%20Lasken&amp;publication_year=2001&amp;journal=Genome%20Res.&amp;volume=11&amp;pages=1095-1099" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.180501" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.180501" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.180501"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11381035" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN004C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN004C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Skene</div> <div class="given-names">J. H.</div></div>,  <div class="name"><div class="surname">Virag</div> <div class="given-names">I.</div></div>. </div><div class="article-title">Posttranslational membrane attachment and dynamic fatty acylation of a neuronal growth cone protein, GAP-43</div>, <div class="source">J. Cell Biol.</div> , <div class="year">1989</div>, vol. <div class="volume">108</div> (pg. <div class="fpage">613</div>-<div class="lpage">624</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Posttranslational%20membrane%20attachment%20and%20dynamic%20fatty%20acylation%20of%20a%20neuronal%20growth%20cone%20protein%2C%20GAP-43&amp;author=J.%20H.%20Skene&amp;author=I.%20Virag&amp;publication_year=1989&amp;journal=J.%20Cell%20Biol.&amp;volume=108&amp;pages=613-624" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1083/jcb.108.2.613" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1083%2Fjcb.108.2.613" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1083%2Fjcb.108.2.613"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2918027" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN004C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN004C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lanford</div> <div class="given-names">R. E.</div></div>,  <div class="name"><div class="surname">Kanda</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Kennedy</div> <div class="given-names">R. C.</div></div>. </div><div class="article-title">Induction of nuclear transport with a synthetic peptide homologous to the SV40 T antigen transport signal</div>, <div class="source">Cell</div> , <div class="year">1986</div>, vol. <div class="volume">46</div> (pg. <div class="fpage">575</div>-<div class="lpage">582</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Induction%20of%20nuclear%20transport%20with%20a%20synthetic%20peptide%20homologous%20to%20the%20SV40%20T%20antigen%20transport%20signal&amp;author=R.%20E.%20Lanford&amp;author=P.%20Kanda&amp;author=R.%20C.%20Kennedy&amp;publication_year=1986&amp;journal=Cell&amp;volume=46&amp;pages=575-582" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0092-8674(86)90883-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0092-8674(86)90883-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0092-8674(86)90883-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3015419" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN004C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN004C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fliegel</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Burns</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">MacLennan</div> <div class="given-names">D. H.</div></div>,  <div class="name"><div class="surname">Reithmeier</div> <div class="given-names">R. A.</div></div>,  <div class="name"><div class="surname">Michalak</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Molecular cloning of the high affinity calcium-binding protein (calreticulin) of skeletal muscle sarcoplasmic reticulum</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1989</div>, vol. <div class="volume">264</div> (pg. <div class="fpage">21522</div>-<div class="lpage">21528</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20cloning%20of%20the%20high%20affinity%20calcium-binding%20protein%20(calreticulin)%20of%20skeletal%20muscle%20sarcoplasmic%20reticulum&amp;author=L.%20Fliegel&amp;author=K.%20Burns&amp;author=D.%20H.%20MacLennan&amp;author=R.%20A.%20Reithmeier&amp;author=M.%20Michalak&amp;publication_year=1989&amp;journal=J.%20Biol.%20Chem.&amp;volume=264&amp;pages=21522-21528" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2600080" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN004C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN004C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Llopis</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">McCaffery</div> <div class="given-names">J. M.</div></div>,  <div class="name"><div class="surname">Miyawaki</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Farquhar</div> <div class="given-names">M. G.</div></div>,  <div class="name"><div class="surname">Tsien</div> <div class="given-names">R. Y.</div></div>. </div><div class="article-title">Measurement of cytosolic, mitochondrial, and Golgi pH in single living cells with green fluorescent proteins</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1998</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">6803</div>-<div class="lpage">6808</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Measurement%20of%20cytosolic%2C%20mitochondrial%2C%20and%20Golgi%20pH%20in%20single%20living%20cells%20with%20green%20fluorescent%20proteins&amp;author=J.%20Llopis&amp;author=J.%20M.%20McCaffery&amp;author=A.%20Miyawaki&amp;author=M.%20G.%20Farquhar&amp;author=R.%20Y.%20Tsien&amp;publication_year=1998&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=95&amp;pages=6803-6808" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.12.6803" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.12.6803" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.12.6803"> </span></div> </div></div></div></div></div><div content-id="DSN004C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN004C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rizzuto</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Brini</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Pizzo</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Murgia</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Pozzan</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Chimeric green fluorescent protein as a tool for visualizing subcellular organelles in living cells</div>, <div class="source">Curr. Biol</div> , <div class="year">1995</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">635</div>-<div class="lpage">642</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chimeric%20green%20fluorescent%20protein%20as%20a%20tool%20for%20visualizing%20subcellular%20organelles%20in%20living%20cells&amp;author=R.%20Rizzuto&amp;author=M.%20Brini&amp;author=P.%20Pizzo&amp;author=M.%20Murgia&amp;author=T.%20Pozzan&amp;publication_year=1995&amp;journal=Curr.%20Biol&amp;volume=5&amp;pages=635-642" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0960-9822(95)00128-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0960-9822(95)00128-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0960-9822(95)00128-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7552174" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN004C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN004C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="collab">The UniProt Consortium</div><div class="article-title">The universal protein resource (UniProt)</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">D193</div>-<div class="lpage">D197</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl929" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl929" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl929"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17142230" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN004C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN004C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lamesch</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Milstein</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">hORFeome v3.1: a resource of human open reading frames representing over 10,000 human genes</div>, <div class="source">Genomics</div> , <div class="year">2007</div>, vol. <div class="volume">89</div> (pg. <div class="fpage">307</div>-<div class="lpage">315</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=hORFeome%20v3.1%3A%20a%20resource%20of%20human%20open%20reading%20frames%20representing%20over%2010%2C000%20human%20genes&amp;author=P.%20Lamesch&amp;author=N.%20Li&amp;author=S.%20Milstein&amp;publication_year=2007&amp;journal=Genomics&amp;volume=89&amp;pages=307-315" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ygeno.2006.11.012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ygeno.2006.11.012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ygeno.2006.11.012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17207965" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2008. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=464067&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/3/10.1093/dnares/dsn004/2/dsn004-dsn004supp_table.xls?Expires=2147483647&amp;Signature=jJ2puLWh3W8xR2D4W1~E~YTWhXgBeBd8AunjAziFCU62aFvMyV-yC0kdFcyGx~WJ0IuIm9697OP6tTSwT0gfl6PVY0wAxads7ypOb5gyDa4rWlbxJWxnUb2KhMW41WgKcagc3~lc~IyAk5HdVXZDTnjIhg0Ka6E9sB3PBY4egcGfY3ZfIlYcad8NcyllFDJZJXr3bh0-Hzz4BaAEJ1IV0~45CS-ZEzrCC6k-6TqtsbwzWDG91rEL0CUTdyyjUj3YGWEgosDUIT7~uWZpmiB2JALDIy7aPvhkzM6NtMrGGlquPPDA6UfHnPjR0tXsw8nCrWrWSdaWgQAElJltIG5e~A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - xls file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">212</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000258424600003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">32</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsn004"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="464067"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000258424600003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f15%2f3%2f137%2f464067">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000258424600003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (32)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F15%2F3%2F137%2F464067" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/464067'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Exploration of Human ORFeome: High-Throughput Preparation of ORF Clones and Efficient Characterization of Their Protein Products" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsn004");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f15%2f3%2f137%2f464067">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

