

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/20/3/255/368679","@type":"ScholarlyArticle","name":"Development and Characterization of cDNA Resources for the Common Marmoset: One of the Experimental Primate Models","datePublished":"2013-03-29","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/20/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2013-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dst007","keywords":["common marmoset","cDNA","gene resource"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/20/3/255/368679","author":[{"name":"Tatsumoto, Shoji","affiliation":"1  RIKEN Genomic Sciences Center ,  Suehiro-cho 1-7-22, Tsurumi-ku, Yokohama 230-0045 ,  Japan","@type":"Person"},{"name":"Adati, Naoki","affiliation":"1  RIKEN Genomic Sciences Center ,  Suehiro-cho 1-7-22, Tsurumi-ku, Yokohama 230-0045 ,  Japan","@type":"Person"},{"name":"Tohtoki, Yasushi","affiliation":"1  RIKEN Genomic Sciences Center ,  Suehiro-cho 1-7-22, Tsurumi-ku, Yokohama 230-0045 ,  Japan","@type":"Person"},{"name":"Sakaki, Yoshiyuki","affiliation":"1  RIKEN Genomic Sciences Center ,  Suehiro-cho 1-7-22, Tsurumi-ku, Yokohama 230-0045 ,  Japan","@type":"Person"},{"name":"Boroviak, Thorsten","affiliation":"2  Centre for Stem Cell Research, Wellcome Trust and Medical Research Council Stem Cell Institute ,  University of Cambridge ,  Tennis Court Road, Cambridge CB2 1QR ,  UK","@type":"Person"},{"name":"Habu, Sonoko","affiliation":"3  Department of Immunology ,  Juntendo University School of Medicine ,  Hongo 2-1-1, Bunkyo-ku, Tokyo 113-8421 ,  Japan","@type":"Person"},{"name":"Okano, Hideyuki","affiliation":"4  Department of Physiology, School of Medicine ,  Keio University ,  Shinano-machi 35, Shinjyuku-ku, Tokyo 160-8582 ,  Japan","@type":"Person"},{"name":"Suemizu, Hiroshi","affiliation":"5  Biomedical Research Department ,  Central Institute for Experimental Animals ,  Tonomachi 3-25-12, Kawasaki-ku ,  Kawasaki 210-0821 ,  Japan","@type":"Person"},{"name":"Sasaki, Erika","affiliation":"6  Department of Applied Developmental Biology ,  Central Institute for Experimental Animals ,  Tonomachi 3-25-12, Kawasaki-ku, Kawasaki 210-0821 ,  Japan","@type":"Person"},{"name":"Satake, Masanobu","affiliation":"7  Institute of Development, Aging and Cancer ,  Tohoku University ,  Seiryo-machi 4-1, Aoba-ku, Sendai 980-8575 ,  Japan","@type":"Person"}],"description":"Abstract.  The common marmoset is a new world monkey, which has become a valuable experimental animal for biomedical research. This study developed cDNA librari","pageStart":"255","pageEnd":"262","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/3/1/m_cover.gif?Expires=2147483647&Signature=3JNT-3uWcoFSRSDjpyRS683cAWcv-uPBeo5zUjK0TFHyw5pdGPApuAnjwKw9hziMynOyeupAu8qXTBcWNNZ5zwCIK3wDotNETRf1C0gT1NeaWwrIBlHGIKu6bYD8JmeQgULL4qaQcRAHnyo7FyqKjoT8e0y4GLL7JPBIG8ClvgMIpmjvKRKJfK-rkhDpE9pXuYo8ex4C6o9vQhlNIWM8gIl4ygPsB2y8BMTIUXCEDS8KZ8W94~SHycoNAwx1VF1wN~gGUhCSNthMU3E6MsUONjgAq-FWmQ8kuAdsyKdxNSJy8mhuDD9SBTDoUvN5zzz~5gwbw0AeOX0S-zArQW9D7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Development and Characterization of cDNA Resources for the Common Marmoset: One of the Experimental Primate Models","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/3/1/m_cover.gif?Expires=2147483647&Signature=3JNT-3uWcoFSRSDjpyRS683cAWcv-uPBeo5zUjK0TFHyw5pdGPApuAnjwKw9hziMynOyeupAu8qXTBcWNNZ5zwCIK3wDotNETRf1C0gT1NeaWwrIBlHGIKu6bYD8JmeQgULL4qaQcRAHnyo7FyqKjoT8e0y4GLL7JPBIG8ClvgMIpmjvKRKJfK-rkhDpE9pXuYo8ex4C6o9vQhlNIWM8gIl4ygPsB2y8BMTIUXCEDS8KZ8W94~SHycoNAwx1VF1wN~gGUhCSNthMU3E6MsUONjgAq-FWmQ8kuAdsyKdxNSJy8mhuDD9SBTDoUvN5zzz~5gwbw0AeOX0S-zArQW9D7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Development and Characterization of cDNA Resources for the Common Marmoset: One of the Experimental Primate Models" />
    <meta property="og:description" content="Abstract.  The common marmoset is a new world monkey, which has become a valuable experimental animal for biomedical research. This study developed cDNA librari" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/20/3/255/368679" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2013-03-29" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Tatsumoto, Shoji" /><meta name="citation_author_institution" content="RIKEN Genomic Sciences Center, Suehiro-cho 1-7-22, Tsurumi-ku, Yokohama 230-0045, Japan" /><meta name="citation_author" content="Adati, Naoki" /><meta name="citation_author_institution" content="RIKEN Genomic Sciences Center, Suehiro-cho 1-7-22, Tsurumi-ku, Yokohama 230-0045, Japan" /><meta name="citation_author" content="Tohtoki, Yasushi" /><meta name="citation_author_institution" content="RIKEN Genomic Sciences Center, Suehiro-cho 1-7-22, Tsurumi-ku, Yokohama 230-0045, Japan" /><meta name="citation_author" content="Sakaki, Yoshiyuki" /><meta name="citation_author_institution" content="RIKEN Genomic Sciences Center, Suehiro-cho 1-7-22, Tsurumi-ku, Yokohama 230-0045, Japan" /><meta name="citation_author" content="Boroviak, Thorsten" /><meta name="citation_author_institution" content="Centre for Stem Cell Research, Wellcome Trust and Medical Research Council Stem Cell Institute, University of Cambridge, Tennis Court Road, Cambridge CB2 1QR, UK" /><meta name="citation_author" content="Habu, Sonoko" /><meta name="citation_author_institution" content="Department of Immunology, Juntendo University School of Medicine, Hongo 2-1-1, Bunkyo-ku, Tokyo 113-8421, Japan" /><meta name="citation_author" content="Okano, Hideyuki" /><meta name="citation_author_institution" content="Department of Physiology, School of Medicine, Keio University, Shinano-machi 35, Shinjyuku-ku, Tokyo 160-8582, Japan" /><meta name="citation_author" content="Suemizu, Hiroshi" /><meta name="citation_author_institution" content="Biomedical Research Department, Central Institute for Experimental Animals, Tonomachi 3-25-12, Kawasaki-ku, Kawasaki 210-0821, Japan" /><meta name="citation_author" content="Sasaki, Erika" /><meta name="citation_author_institution" content="Department of Applied Developmental Biology, Central Institute for Experimental Animals, Tonomachi 3-25-12, Kawasaki-ku, Kawasaki 210-0821, Japan" /><meta name="citation_author" content="Satake, Masanobu" /><meta name="citation_author_institution" content="Institute of Development, Aging and Cancer, Tohoku University, Seiryo-machi 4-1, Aoba-ku, Sendai 980-8575, Japan" /><meta name="citation_title" content="Development and Characterization of cDNA Resources for the Common Marmoset: One of the Experimental Primate Models" /><meta name="citation_firstpage" content="255" /><meta name="citation_lastpage" content="262" /><meta name="citation_doi" content="10.1093/dnares/dst007" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="20" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2013/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Marmoset models commonly used in biomedical research; Comp. Med.;  citation_year=2003;  citation_volume=53;  citation_pages=383-92; " /><meta name="citation_reference" content="citation_title=Active and passively induced experimental autoimmune encephalomyelitis in common marmosets: a new model for multiple sclerosis; Ann. Neurol.;  citation_year=1995;  citation_volume=37;  citation_pages=519-30; " /><meta name="citation_reference" content="citation_title=Experimental allergic encephalomyelitis in the new world monkey Callithrix jacchus; Immunol. Rev.;  citation_year=2001;  citation_volume=183;  citation_pages=159-92; " /><meta name="citation_reference" content="citation_title=Modeling of multiple sclerosis: lessons learned in a non-human primate; Lancet Neurol.;  citation_year=2004;  citation_volume=3;  citation_pages=588-97; " /><meta name="citation_reference" content="citation_title=Generation of transgenic nonhuman primates with germline transmission; Nature;  citation_year=2009;  citation_volume=459;  citation_pages=515-6; " /><meta name="citation_reference" content="citation_title=Hematopoietic activity of common marmoset CD34 cells isolated by a novel monoclonal antibody MA24; Exp. Hematol.;  citation_year=2004;  citation_volume=32;  citation_pages=843-51; " /><meta name="citation_reference" content="citation_title=Novel monoclonal antibodies recognizing different subsets of lymphocytes from the common marmoset (Callithrix jacchus); Immunol. Lett.;  citation_year=2008;  citation_volume=121;  citation_pages=116-22; " /><meta name="citation_reference" content="citation_title=Development of monoclonal antibodies for analyzing immune and hematopoietic systems of common marmoset; Exp. Hematol.;  citation_year=2009;  citation_volume=37;  citation_pages=1318-29; " /><meta name="citation_reference" content="citation_title=An extensive monoclonal antibody panel for the phenotyping of leukocyte subsets in the common marmoset and the cotton-top tamarin; Cytometry;  citation_year=2001;  citation_volume=45;  citation_pages=294-303; " /><meta name="citation_reference" content="citation_title=Comparison of 30 immunity-related genes from the common marmoset with orthologues from human and mouse; Tohoku J. Exp. Med.;  citation_year=2008;  citation_volume=215;  citation_pages=167-80; " /><meta name="citation_reference" content="citation_title=Establishment of novel embryonic stem cell lines derived from the common marmoset (Callithrix jacchus); Stem Cells;  citation_year=2005;  citation_volume=23;  citation_pages=1304-13; " /><meta name="citation_reference" content="citation_title=Determination of the capped site sequence of mRNA based on the detection of cap-dependent nucleotide addition using an anchor ligation method; DNA Res.;  citation_year=2004;  citation_volume=11;  citation_pages=305-9; " /><meta name="citation_reference" content="citation_title=Basic local alignment search tool; J. Mol. Biol.;  citation_year=1990;  citation_volume=215;  citation_pages=403-10; " /><meta name="citation_reference" content="citation_title=BLAT—the BLAST-like alignment tool; Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=656-64; " /><meta name="citation_reference" content="citation_title=Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences; Bioinformatics;  citation_year=2006;  citation_volume=22;  citation_pages=1658-9; " /><meta name="citation_reference" content="citation_title=CAP3: a DNA sequence assembly program; Genome. Res.;  citation_year=1999;  citation_volume=9;  citation_pages=868-77; " /><meta name="citation_reference" content="citation_title=Base-calling of automated sequencer traces using phred. II. Error probabilities; Genome. Res.;  citation_year=1998;  citation_volume=8;  citation_pages=186-94; " /><meta name="citation_reference" content="citation_title=EMBOSS: the European Molecular Biology Open Software Suite; Trends in Genet.;  citation_year=2000;  citation_volume=16;  citation_pages=276-7; " /><meta name="citation_reference" content="citation_title=InterProScan: protein domains identifier; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=116-20; " /><meta name="citation_reference" content="citation_title=Analysis of 5′-end sequences of chimpanzee cDNAs; Genome. Res.;  citation_year=2003;  citation_volume=13;  citation_pages=1022-6; " /><meta name="citation_reference" content="citation_title=Analysis of 10,000 ESTs from lymphocytes of the cynomolgus monkey to improve our understanding of its immune system; BMC Genomics;  citation_year=2006;  citation_volume=7;  citation_pages=82" /><meta name="citation_reference" content="citation_title=Expressed sequence tags from cynomolgus monkey (Macaca fascicularis) liver: a systematic identification of drug-metabolizing enzymes; FEBS Lett.;  citation_year=2008;  citation_volume=582;  citation_pages=351-8; " /><meta name="citation_reference" content="citation_title=Collection of Macaca fascicularis cDNAs derived from bone marrow, kidney, liver, pancreas, spleen, and thymus; BMC Res. Notes;  citation_year=2009;  citation_volume=2;  citation_pages=199" /><meta name="citation_reference" content="citation_title=Expression sequence tag library derived from peripheral blood mononuclear cells of the chlorocebus sabaeus; BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=279" /><meta name="citation_reference" content="citation_title=Induced pluripotent stem cell lines derived from human somatic cells; Science;  citation_year=2007;  citation_volume=318;  citation_pages=1917-20; " /><meta name="citation_reference" content="citation_title=The homeoprotein Nanog is required for maintenance of pluripotency in mouse epiblast and ES cells; Cell;  citation_year=2003;  citation_volume=113;  citation_pages=631-42; " /><meta name="citation_reference" content="citation_title=Genetic analysis of the role of the reprogramming gene LIN-28 in human embryonic stem cells; Stem Cells;  citation_year=2009;  citation_volume=27;  citation_pages=352-62; " /><meta name="citation_reference" content="citation_title=Lin28 modulates cell growth and associates with a subset of cell cycle regulator mRNAs in mouse embryonic stem cells; RNA;  citation_year=2009;  citation_volume=15;  citation_pages=357-61; " /><meta name="citation_reference" content="citation_title=Identification of developmental pluripotency associated 5 expression in human pluripotent stem cells; Stem Cells;  citation_year=2005;  citation_volume=23;  citation_pages=458-62; " /><meta name="citation_reference" content="citation_title=Atypical structure and phylogenomic evolution of the new eutherian oocyte- and embryo-expressed KHDC1/DPPA5/ECAT1/OOEP gene family; Genomics;  citation_year=2007;  citation_volume=90;  citation_pages=583-94; " /><meta name="citation_reference" content="citation_title=Alternative isoform regulation in human tissue transcriptomes; Nature;  citation_year=2008;  citation_volume=456;  citation_pages=470-6; " /><meta name="citation_reference" content="citation_title=Integrative annotation of human large intergenic noncoding RNAs reveals global properties and specific subclasses; Genes Dev.;  citation_year=2011;  citation_volume=25;  citation_pages=1915-27; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/20/3/255/1050644/dst007.pdf" /><meta name="description" content="Abstract.  The common marmoset is a new world monkey, which has become a valuable experimental animal for biomedical research. This study developed cDNA librari" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/20/3/255/368679" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/20/3/255/368679" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/20/3/255/368679");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dst007';
    </script>
<title>
	Development and Characterization of cDNA Resources for the Common Marmoset: One of the Experimental Primate Models | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./368679" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BapTPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYXRzdW1vdG8sIFNob2ppIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUklLRU4gR2Vub21pYyBTY2llbmNlcyBDZW50ZXIsIFN1ZWhpcm8tY2hvIDEtNy0yMiwgVHN1cnVtaS1rdSwgWW9rb2hhbWEgMjMwLTAwNDUsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQWRhdGksIE5hb2tpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUklLRU4gR2Vub21pYyBTY2llbmNlcyBDZW50ZXIsIFN1ZWhpcm8tY2hvIDEtNy0yMiwgVHN1cnVtaS1rdSwgWW9rb2hhbWEgMjMwLTAwNDUsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVG9odG9raSwgWWFzdXNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlJJS0VOIEdlbm9taWMgU2NpZW5jZXMgQ2VudGVyLCBTdWVoaXJvLWNobyAxLTctMjIsIFRzdXJ1bWkta3UsIFlva29oYW1hIDIzMC0wMDQ1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNha2FraSwgWW9zaGl5dWtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUklLRU4gR2Vub21pYyBTY2llbmNlcyBDZW50ZXIsIFN1ZWhpcm8tY2hvIDEtNy0yMiwgVHN1cnVtaS1rdSwgWW9rb2hhbWEgMjMwLTAwNDUsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQm9yb3ZpYWssIFRob3JzdGVuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2VudHJlIGZvciBTdGVtIENlbGwgUmVzZWFyY2gsIFdlbGxjb21lIFRydXN0IGFuZCBNZWRpY2FsIFJlc2VhcmNoIENvdW5jaWwgU3RlbSBDZWxsIEluc3RpdHV0ZSwgVW5pdmVyc2l0eSBvZiBDYW1icmlkZ2UsIFRlbm5pcyBDb3VydCBSb2FkLCBDYW1icmlkZ2UgQ0IyIDFRUiwgVUsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIYWJ1LCBTb25va28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEltbXVub2xvZ3ksIEp1bnRlbmRvIFVuaXZlcnNpdHkgU2Nob29sIG9mIE1lZGljaW5lLCBIb25nbyAyLTEtMSwgQnVua3lvLWt1LCBUb2t5byAxMTMtODQyMSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPa2FubywgSGlkZXl1a2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFBoeXNpb2xvZ3ksIFNjaG9vbCBvZiBNZWRpY2luZSwgS2VpbyBVbml2ZXJzaXR5LCBTaGluYW5vLW1hY2hpIDM1LCBTaGluanl1a3Uta3UsIFRva3lvIDE2MC04NTgyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlN1ZW1penUsIEhpcm9zaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCaW9tZWRpY2FsIFJlc2VhcmNoIERlcGFydG1lbnQsIENlbnRyYWwgSW5zdGl0dXRlIGZvciBFeHBlcmltZW50YWwgQW5pbWFscywgVG9ub21hY2hpIDMtMjUtMTIsIEthd2FzYWtpLWt1LCBLYXdhc2FraSAyMTAtMDgyMSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYXNha2ksIEVyaWthIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBBcHBsaWVkIERldmVsb3BtZW50YWwgQmlvbG9neSwgQ2VudHJhbCBJbnN0aXR1dGUgZm9yIEV4cGVyaW1lbnRhbCBBbmltYWxzLCBUb25vbWFjaGkgMy0yNS0xMiwgS2F3YXNha2kta3UsIEthd2FzYWtpIDIxMC0wODIxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNhdGFrZSwgTWFzYW5vYnUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgb2YgRGV2ZWxvcG1lbnQsIEFnaW5nIGFuZCBDYW5jZXIsIFRvaG9rdSBVbml2ZXJzaXR5LCBTZWlyeW8tbWFjaGkgNC0xLCBBb2JhLWt1LCBTZW5kYWkgOTgwLTg1NzUsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJEZXZlbG9wbWVudCBhbmQgQ2hhcmFjdGVyaXphdGlvbiBvZiBjRE5BIFJlc291cmNlcyBmb3IgdGhlIENvbW1vbiBNYXJtb3NldDogT25lIG9mIHRoZSBFeHBlcmltZW50YWwgUHJpbWF0ZSBNb2RlbHMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIyNTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjI2MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdDAwNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDEzLzA2LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFybW9zZXQgbW9kZWxzIGNvbW1vbmx5IHVzZWQgaW4gYmlvbWVkaWNhbCByZXNlYXJjaDsgQ29tcC4gTWVkLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT01MzsgIGNpdGF0aW9uX3BhZ2VzPTM4My05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWN0aXZlIGFuZCBwYXNzaXZlbHkgaW5kdWNlZCBleHBlcmltZW50YWwgYXV0b2ltbXVuZSBlbmNlcGhhbG9teWVsaXRpcyBpbiBjb21tb24gbWFybW9zZXRzOiBhIG5ldyBtb2RlbCBmb3IgbXVsdGlwbGUgc2NsZXJvc2lzOyBBbm4uIE5ldXJvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz01MTktMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cGVyaW1lbnRhbCBhbGxlcmdpYyBlbmNlcGhhbG9teWVsaXRpcyBpbiB0aGUgbmV3IHdvcmxkIG1vbmtleSBDYWxsaXRocml4IGphY2NodXM7IEltbXVub2wuIFJldi47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTgzOyAgY2l0YXRpb25fcGFnZXM9MTU5LTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2RlbGluZyBvZiBtdWx0aXBsZSBzY2xlcm9zaXM6IGxlc3NvbnMgbGVhcm5lZCBpbiBhIG5vbi1odW1hbiBwcmltYXRlOyBMYW5jZXQgTmV1cm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9NTg4LTk3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lcmF0aW9uIG9mIHRyYW5zZ2VuaWMgbm9uaHVtYW4gcHJpbWF0ZXMgd2l0aCBnZXJtbGluZSB0cmFuc21pc3Npb247IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT00NTk7ICBjaXRhdGlvbl9wYWdlcz01MTUtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGVtYXRvcG9pZXRpYyBhY3Rpdml0eSBvZiBjb21tb24gbWFybW9zZXQgQ0QzNCBjZWxscyBpc29sYXRlZCBieSBhIG5vdmVsIG1vbm9jbG9uYWwgYW50aWJvZHkgTUEyNDsgRXhwLiBIZW1hdG9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPTg0My01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm92ZWwgbW9ub2Nsb25hbCBhbnRpYm9kaWVzIHJlY29nbml6aW5nIGRpZmZlcmVudCBzdWJzZXRzIG9mIGx5bXBob2N5dGVzIGZyb20gdGhlIGNvbW1vbiBtYXJtb3NldCAoQ2FsbGl0aHJpeCBqYWNjaHVzKTsgSW1tdW5vbC4gTGV0dC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTIxOyAgY2l0YXRpb25fcGFnZXM9MTE2LTIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBvZiBtb25vY2xvbmFsIGFudGlib2RpZXMgZm9yIGFuYWx5emluZyBpbW11bmUgYW5kIGhlbWF0b3BvaWV0aWMgc3lzdGVtcyBvZiBjb21tb24gbWFybW9zZXQ7IEV4cC4gSGVtYXRvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz0xMzE4LTI5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBleHRlbnNpdmUgbW9ub2Nsb25hbCBhbnRpYm9keSBwYW5lbCBmb3IgdGhlIHBoZW5vdHlwaW5nIG9mIGxldWtvY3l0ZSBzdWJzZXRzIGluIHRoZSBjb21tb24gbWFybW9zZXQgYW5kIHRoZSBjb3R0b24tdG9wIHRhbWFyaW47IEN5dG9tZXRyeTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT00NTsgIGNpdGF0aW9uX3BhZ2VzPTI5NC0zMDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmlzb24gb2YgMzAgaW1tdW5pdHktcmVsYXRlZCBnZW5lcyBmcm9tIHRoZSBjb21tb24gbWFybW9zZXQgd2l0aCBvcnRob2xvZ3VlcyBmcm9tIGh1bWFuIGFuZCBtb3VzZTsgVG9ob2t1IEouIEV4cC4gTWVkLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yMTU7ICBjaXRhdGlvbl9wYWdlcz0xNjctODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVzdGFibGlzaG1lbnQgb2Ygbm92ZWwgZW1icnlvbmljIHN0ZW0gY2VsbCBsaW5lcyBkZXJpdmVkIGZyb20gdGhlIGNvbW1vbiBtYXJtb3NldCAoQ2FsbGl0aHJpeCBqYWNjaHVzKTsgU3RlbSBDZWxsczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTEzMDQtMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldGVybWluYXRpb24gb2YgdGhlIGNhcHBlZCBzaXRlIHNlcXVlbmNlIG9mIG1STkEgYmFzZWQgb24gdGhlIGRldGVjdGlvbiBvZiBjYXAtZGVwZW5kZW50IG51Y2xlb3RpZGUgYWRkaXRpb24gdXNpbmcgYW4gYW5jaG9yIGxpZ2F0aW9uIG1ldGhvZDsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0zMDUtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFzaWMgbG9jYWwgYWxpZ25tZW50IHNlYXJjaCB0b29sOyBKLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTIxNTsgIGNpdGF0aW9uX3BhZ2VzPTQwMy0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QkxBVOKAlHRoZSBCTEFTVC1saWtlIGFsaWdubWVudCB0b29sOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTY1Ni02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2QtaGl0OiBhIGZhc3QgcHJvZ3JhbSBmb3IgY2x1c3RlcmluZyBhbmQgY29tcGFyaW5nIGxhcmdlIHNldHMgb2YgcHJvdGVpbiBvciBudWNsZW90aWRlIHNlcXVlbmNlczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0xNjU4LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNBUDM6IGEgRE5BIHNlcXVlbmNlIGFzc2VtYmx5IHByb2dyYW07IEdlbm9tZS4gUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9ODY4LTc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYXNlLWNhbGxpbmcgb2YgYXV0b21hdGVkIHNlcXVlbmNlciB0cmFjZXMgdXNpbmcgcGhyZWQuIElJLiBFcnJvciBwcm9iYWJpbGl0aWVzOyBHZW5vbWUuIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTE4Ni05NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RU1CT1NTOiB0aGUgRXVyb3BlYW4gTW9sZWN1bGFyIEJpb2xvZ3kgT3BlbiBTb2Z0d2FyZSBTdWl0ZTsgVHJlbmRzIGluIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTI3Ni03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlclByb1NjYW46IHByb3RlaW4gZG9tYWlucyBpZGVudGlmaWVyOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz0xMTYtMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIDXigLItZW5kIHNlcXVlbmNlcyBvZiBjaGltcGFuemVlIGNETkFzOyBHZW5vbWUuIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0xMDIyLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIDEwLDAwMCBFU1RzIGZyb20gbHltcGhvY3l0ZXMgb2YgdGhlIGN5bm9tb2xndXMgbW9ua2V5IHRvIGltcHJvdmUgb3VyIHVuZGVyc3RhbmRpbmcgb2YgaXRzIGltbXVuZSBzeXN0ZW07IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ODIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHByZXNzZWQgc2VxdWVuY2UgdGFncyBmcm9tIGN5bm9tb2xndXMgbW9ua2V5IChNYWNhY2EgZmFzY2ljdWxhcmlzKSBsaXZlcjogYSBzeXN0ZW1hdGljIGlkZW50aWZpY2F0aW9uIG9mIGRydWctbWV0YWJvbGl6aW5nIGVuenltZXM7IEZFQlMgTGV0dC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NTgyOyAgY2l0YXRpb25fcGFnZXM9MzUxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbGxlY3Rpb24gb2YgTWFjYWNhIGZhc2NpY3VsYXJpcyBjRE5BcyBkZXJpdmVkIGZyb20gYm9uZSBtYXJyb3csIGtpZG5leSwgbGl2ZXIsIHBhbmNyZWFzLCBzcGxlZW4sIGFuZCB0aHltdXM7IEJNQyBSZXMuIE5vdGVzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0xOTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHByZXNzaW9uIHNlcXVlbmNlIHRhZyBsaWJyYXJ5IGRlcml2ZWQgZnJvbSBwZXJpcGhlcmFsIGJsb29kIG1vbm9udWNsZWFyIGNlbGxzIG9mIHRoZSBjaGxvcm9jZWJ1cyBzYWJhZXVzOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0yNzkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmR1Y2VkIHBsdXJpcG90ZW50IHN0ZW0gY2VsbCBsaW5lcyBkZXJpdmVkIGZyb20gaHVtYW4gc29tYXRpYyBjZWxsczsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTg7ICBjaXRhdGlvbl9wYWdlcz0xOTE3LTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgaG9tZW9wcm90ZWluIE5hbm9nIGlzIHJlcXVpcmVkIGZvciBtYWludGVuYW5jZSBvZiBwbHVyaXBvdGVuY3kgaW4gbW91c2UgZXBpYmxhc3QgYW5kIEVTIGNlbGxzOyBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTExMzsgIGNpdGF0aW9uX3BhZ2VzPTYzMS00MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBhbmFseXNpcyBvZiB0aGUgcm9sZSBvZiB0aGUgcmVwcm9ncmFtbWluZyBnZW5lIExJTi0yOCBpbiBodW1hbiBlbWJyeW9uaWMgc3RlbSBjZWxsczsgU3RlbSBDZWxsczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTM1Mi02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGluMjggbW9kdWxhdGVzIGNlbGwgZ3Jvd3RoIGFuZCBhc3NvY2lhdGVzIHdpdGggYSBzdWJzZXQgb2YgY2VsbCBjeWNsZSByZWd1bGF0b3IgbVJOQXMgaW4gbW91c2UgZW1icnlvbmljIHN0ZW0gY2VsbHM7IFJOQTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTM1Ny02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgZGV2ZWxvcG1lbnRhbCBwbHVyaXBvdGVuY3kgYXNzb2NpYXRlZCA1IGV4cHJlc3Npb24gaW4gaHVtYW4gcGx1cmlwb3RlbnQgc3RlbSBjZWxsczsgU3RlbSBDZWxsczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTQ1OC02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXR5cGljYWwgc3RydWN0dXJlIGFuZCBwaHlsb2dlbm9taWMgZXZvbHV0aW9uIG9mIHRoZSBuZXcgZXV0aGVyaWFuIG9vY3l0ZS0gYW5kIGVtYnJ5by1leHByZXNzZWQgS0hEQzEvRFBQQTUvRUNBVDEvT09FUCBnZW5lIGZhbWlseTsgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9OTA7ICBjaXRhdGlvbl9wYWdlcz01ODMtOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsdGVybmF0aXZlIGlzb2Zvcm0gcmVndWxhdGlvbiBpbiBodW1hbiB0aXNzdWUgdHJhbnNjcmlwdG9tZXM7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00NTY7ICBjaXRhdGlvbl9wYWdlcz00NzAtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZWdyYXRpdmUgYW5ub3RhdGlvbiBvZiBodW1hbiBsYXJnZSBpbnRlcmdlbmljIG5vbmNvZGluZyBSTkFzIHJldmVhbHMgZ2xvYmFsIHByb3BlcnRpZXMgYW5kIHNwZWNpZmljIHN1YmNsYXNzZXM7IEdlbmVzIERldi47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xOTE1LTI3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjAvMy8yNTUvMTA1MDY0NC9kc3QwMDcucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFRoZSBjb21tb24gbWFybW9zZXQgaXMgYSBuZXcgd29ybGQgbW9ua2V5LCB3aGljaCBoYXMgYmVjb21lIGEgdmFsdWFibGUgZXhwZXJpbWVudGFsIGFuaW1hbCBmb3IgYmlvbWVkaWNhbCByZXNlYXJjaC4gVGhpcyBzdHVkeSBkZXZlbG9wZWQgY0ROQSBsaWJyYXJpIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zNjg2NzkmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2Rki7fMnWf0k2N2i/lzCDEI03f2Hs8=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f3%2f255%2f368679" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f3%2f255%2f368679" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=14337" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/20/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/3/1/m_cover.gif?Expires=2147483647&amp;Signature=3JNT-3uWcoFSRSDjpyRS683cAWcv-uPBeo5zUjK0TFHyw5pdGPApuAnjwKw9hziMynOyeupAu8qXTBcWNNZ5zwCIK3wDotNETRf1C0gT1NeaWwrIBlHGIKu6bYD8JmeQgULL4qaQcRAHnyo7FyqKjoT8e0y4GLL7JPBIG8ClvgMIpmjvKRKJfK-rkhDpE9pXuYo8ex4C6o9vQhlNIWM8gIl4ygPsB2y8BMTIUXCEDS8KZ8W94~SHycoNAwx1VF1wN~gGUhCSNthMU3E6MsUONjgAq-FWmQ8kuAdsyKdxNSJy8mhuDD9SBTDoUvN5zzz~5gwbw0AeOX0S-zArQW9D7g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/20/3">
                        <div class="volume trailing-comma">Volume 20</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2013        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5353437">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5353437">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5353441">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5353441">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5353453">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5353453">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5353468">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5353468">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5353505">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5353505">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5353507">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5353507">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5353509">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5353509">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5353511">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5353511">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/20/3/241/368334">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/20/3/263/368846">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Development and Characterization of cDNA Resources for the Common Marmoset: One of the Experimental Primate Models
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shoji Tatsumoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">RIKEN Genomic Sciences Center</div>, <div class="addr-line">Suehiro-cho 1-7-22, Tsurumi-ku, Yokohama 230-0045</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shoji+Tatsumoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tatsumoto S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tatsumoto S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shoji Tatsumoto</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Naoki Adati</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">RIKEN Genomic Sciences Center</div>, <div class="addr-line">Suehiro-cho 1-7-22, Tsurumi-ku, Yokohama 230-0045</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Naoki+Adati">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Adati N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Adati N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Naoki Adati</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yasushi Tohtoki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">RIKEN Genomic Sciences Center</div>, <div class="addr-line">Suehiro-cho 1-7-22, Tsurumi-ku, Yokohama 230-0045</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yasushi+Tohtoki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tohtoki Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tohtoki Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yasushi Tohtoki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshiyuki Sakaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">RIKEN Genomic Sciences Center</div>, <div class="addr-line">Suehiro-cho 1-7-22, Tsurumi-ku, Yokohama 230-0045</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshiyuki+Sakaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sakaki Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sakaki Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yoshiyuki Sakaki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Thorsten Boroviak</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Centre for Stem Cell Research, Wellcome Trust and Medical Research Council Stem Cell Institute</div>, <div class="insititution">University of Cambridge</div>, <div class="addr-line">Tennis Court Road, Cambridge CB2 1QR</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Thorsten+Boroviak">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Boroviak T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Boroviak T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Thorsten Boroviak</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sonoko Habu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Immunology</div>, <div class="insititution">Juntendo University School of Medicine</div>, <div class="addr-line">Hongo 2-1-1, Bunkyo-ku, Tokyo 113-8421</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sonoko+Habu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Habu S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Habu S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sonoko Habu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideyuki Okano</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Department of Physiology, School of Medicine</div>, <div class="insititution">Keio University</div>, <div class="addr-line">Shinano-machi 35, Shinjyuku-ku, Tokyo 160-8582</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideyuki+Okano">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Okano H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Okano H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hideyuki Okano</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroshi Suemizu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">Biomedical Research Department</div>, <div class="insititution">Central Institute for Experimental Animals</div>, <div class="addr-line">Tonomachi 3-25-12, Kawasaki-ku</div>, <div class="addr-line">Kawasaki 210-0821</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroshi+Suemizu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Suemizu H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Suemizu H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hiroshi Suemizu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Erika Sasaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="addr-line">Department of Applied Developmental Biology</div>, <div class="insititution">Central Institute for Experimental Animals</div>, <div class="addr-line">Tonomachi 3-25-12, Kawasaki-ku, Kawasaki 210-0821</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Erika+Sasaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sasaki E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sasaki E&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Erika Sasaki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masanobu Satake</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">7</div><div class="addr-line">Institute of Development, Aging and Cancer</div>, <div class="insititution">Tohoku University</div>, <div class="addr-line">Seiryo-machi 4-1, Aoba-ku, Sendai 980-8575</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81-22-717-8477. Fax. +81-22-717-8482. Email: <a href="mailto:satake@idac.tohoku.ac.jp" target="_blank">satake@idac.tohoku.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masanobu+Satake">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Satake M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Satake M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Masanobu Satake</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">Present address: National Institute of Genetics, Yata 1111, Mishima 411-8540, Japan.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN2"><div class="fn"><div class="label -label">‡</div><div class="footnote-content">Present address: Research Equipment Center, Hamamatsu University School of Medicine, Handayama 1-20-1, Higashi-ku, Hamamatsu 431-3192, Japan.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN3"><div class="fn"><div class="label -label">¶</div><div class="footnote-content">Present address: Division of Cancer Genomics, National Cancer Center Research Institute, Tsukiji 5-1-1, Chuo-ku, Tokyo 104-0045, Japan.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN4"><div class="fn"><div class="label -label">§</div><div class="footnote-content">Present address: Toyohashi University of Technology, Hibarigaoka 1-1, Tenpaku-cho, Toyohashi 441-8580, Japan.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 20, Issue 3, 1 June 2013, Pages 255–262, <a href='https://doi.org/10.1093/dnares/dst007'>https://doi.org/10.1093/dnares/dst007</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">29 March 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">09 May 2012</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">01 March 2013</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="368679" href="/dnaresearch/article-pdf/20/3/255/1050644/dst007.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Shoji Tatsumoto, Naoki Adati, Yasushi Tohtoki, Yoshiyuki Sakaki, Thorsten Boroviak, Sonoko Habu, Hideyuki Okano, Hiroshi Suemizu, Erika Sasaki, Masanobu Satake;  Development and Characterization of cDNA Resources for the Common Marmoset: One of the Experimental Primate Models, <em>DNA Research</em>, Volume 20, Issue 3, 1 June 2013, Pages 255–262, <a href='https://doi.org/10.1093/dnares/dst007'>https://doi.org/10.1093/dnares/dst007</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/368679?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/368679?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/368679?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/368679?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/368679?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/368679?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/368679?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Development%20and%20Characterization%20of%20cDNA%20Resources%20for%20the%20Common%20Marmoset%3A%20One%20of%20the%20Experimental%20Primate%20Models&amp;publicationDate=2013-03-29&amp;volumeNum=20&amp;issueNum=3&amp;author=Tatsumoto%2C%20Shoji%3B%20Adati%2C%20Naoki&amp;startPage=255&amp;endPage=262&amp;contentId=10.1093%2Fdnares%2Fdst007&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f3%2f255%2f368679" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5353437 id="5353437" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The common marmoset is a new world monkey, which has become a valuable experimental animal for biomedical research. This study developed cDNA libraries for the common marmoset from five different tissues. A total of 290 426 high-quality EST sequences were obtained, where 251 587 sequences (86.5%) had homology (1E<sup>−100</sup>) with the Refseqs of six different primate species, including human and marmoset. In parallel, 270 673 sequences (93.2%) were aligned to the human genome. When 247 090 sequences were assembled into 17 232 contigs, most of the sequences (218 857 or 15 089 contigs) were located in exonic regions, indicating that these genes are expressed in human and marmoset. The other 5578 sequences (or 808 contigs) mapping to the human genome were not located in exonic regions, suggesting that they are not expressed in human. Furthermore, a different set of 118 potential coding sequences were not similar to any Refseqs in any species, and, thus, may represent unknown genes. The cDNA libraries developed in this study are available through RIKEN Bio Resource Center. A Web server for the marmoset cDNAs is available at <a class="link link-uri" href="http://marmoset.nig.ac.jp/index.html">http://marmoset.nig.ac.jp/index.html</a>, where each marmoset EST sequence has been annotated by reference to the human genome. These new libraries will be a useful genetic resource to facilitate research in the common marmoset.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;common marmoset&quot;">common marmoset</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="cDNA">cDNA</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gene resource&quot;">gene resource</a></div>                            <h2 scrollto-destination=5353441 id="5353441" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The mouse is a widely used and well-studied model animal for biomedical research. Many techniques for sophisticated genetic manipulations to model diseases have been established and researchers hope that the results obtained can be extrapolated to humans. Although this assumption is true in some cases, there are several areas of biomedical research where this proves to be more difficult. These areas include neuroscience, behavioural research, toxicology, drug development, and infectious diseases.<sup><a reveal-id="DST007C1" data-open="DST007C1" class="link link-ref link-reveal xref-bibr">1</a></sup> To overcome these limitations, efforts have been made to carry out biomedical studies in non-human primate model organisms, as the latter are more closely related to humans.</p><p>One of these established non-human primate model organisms is <em>Callithrix jacchus</em> (common marmoset). The marmoset is a small new world monkey and offers many advantages as an experimental animal over other non-human primates. It is small in size, which makes it comparatively easy to handle. Furthermore, it has been bred in captivity and its progeny have been maintained for &gt;30 years in laboratory environments. Also, it does not harbour or transmit hazardous infectious agents. Therefore, the common marmoset is increasingly used in biomedical research worldwide. For example, models of autoimmune diseases involving the central nervous system have been developed in the common marmoset and it has been used extensively as a primate model.<sup><a reveal-id="DST007C2  DST007C4" data-open="DST007C2  DST007C4" class="link link-ref link-reveal xref-bibr">2–4</a></sup> More recently, genetically modified common marmosets have been produced successfully and their transgenes have been transmitted through the germ line.<sup><a reveal-id="DST007C5" data-open="DST007C5" class="link link-ref link-reveal xref-bibr">5</a></sup> In the future, it would be very useful to develop transgenic marmosets as models of human diseases.</p><p>Intensive efforts have been made to develop research tools for using the common marmoset as an experimental animal. For instance, several lines of monoclonal antibodies have been prepared, which are directed against immunity-related antigens of the marmoset.<sup><a reveal-id="DST007C6  DST007C8" data-open="DST007C6  DST007C8" class="link link-ref link-reveal xref-bibr">6–8</a></sup> Many, but not all anti-human antigen antibodies cross-reacted with the corresponding marmoset antigens, so it was necessary to establish marmoset-specific antibodies.<sup><a reveal-id="DST007C9" data-open="DST007C9" class="link link-ref link-reveal xref-bibr">9</a></sup> A pilot gene analysis study reported cDNA sequencing of immunity-related genes.<sup><a reveal-id="DST007C10" data-open="DST007C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Based on genome-wide analyses, a draft sequence of the common marmoset, known as caljac3, was produced and made available to the public via the genome browser of the University of California Santa Cruz (<a class="link link-uri" href="http://genome.ucsc.edu/">http://genome.ucsc.edu/</a>).</p><p>The current study describes the preparation of cDNA libraries for the common marmoset using five different cell types/tissues, which resulted in the identification of 290 426 high-quality EST sequences. These sequences were characterized by comparison with the sequences of six primate species, including humans. Overall, the EST sequences transcribed in the marmoset shared many common features with those from humans, whereas a small fraction was found to be unique to the marmoset.</p>                            <h2 scrollto-destination=5353453 id="5353453" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5353456 id="5353456" class="section-title" data-legacy-id=s2a>2.1. RNA extraction and library construction</h3>
<p>Cytoplasmic RNA was extracted from the liver (MLI), brain and spinal cord (MSC), spleen (MSP), testis (MTE), and embryonic stem (ES) cells (MES) of the common marmoset using Trizol reagent. Marmoset ES cells were cultured as described previously.<sup><a reveal-id="DST007C11" data-open="DST007C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Full-length cDNA libraries were constructed from the total RNAs of the aforementioned tissues/cells using a vector-capping method.<sup><a reveal-id="DST007C12" data-open="DST007C12" class="link link-ref link-reveal xref-bibr">12</a></sup> cDNAs generated from MLI, MSC, MSP, MTE, and MES were ligated into pGCAP1, pGCAPzf3, pGCAPzf3, pGCAP10, and pGCAP10 vectors, respectively. Colonies of <em>Escherichia coli</em> transformants were picked randomly, inoculated into 384-well plates using a Flexys colony picker (Genomic Solutions Ltd., Cambridgeshire, UK), and stored at −80°C.</p>                            <h3 scrollto-destination=5353459 id="5353459" class="section-title" data-legacy-id=s2b>2.2. EST sequencing</h3>
<p>Colonies were picked from 99 plates for MSC; 200 plates each for MES, MSP, and MTE; and 201 plates for MLI (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Sequencing templates were prepared using a TempliPhi DNA Amplification Kit (GE Healthcare UK Ltd., Buckinghamshire, UK). The sequencing reactions for the 5′-end directional ESTs were conducted using a BigDye Terminator v3.1 Cycle Sequencing Kit (Applied Biosystems, Inc., CA, USA). The sequence primers used for pGCAP1, pGCAP10, and pGCAPzf3 were 5′-AGGCCTGTACGGAAGTGT-3′, 5′-AGGCCTGTACGGAAGTGT-3′, and 5′-CAAGGCGATTAAGTTGGGT-3′, respectively. The sequencing reaction products were purified by ethanol precipitation and loaded onto 3730 DNA Analyzers (Applied Biosystems Inc.).</p>                            <h3 scrollto-destination=5353461 id="5353461" class="section-title" data-legacy-id=s2c>2.3. Selection of high-quality EST data</h3>
<p>The raw sequence data were basecalled using the KB basecaller program, which identified 345 600 sequences. A cross-match program was applied to the raw data to remove sequences derived from vectors and those added as caps during plasmid construction (−minimach 10, −minscore 20). Low-quality sequences [quality value (QV) = one for each nucleotide ± 3 neighbouring nucleotides measuring &lt;105 in length] were masked by <em>N</em> (average QV &gt; 15). If <em>N</em> was longer than 10 nucleotides, sequences located 3′ of these <em>N</em> were removed. Finally, only high-quality sequences longer than 100 nucleotides were selected and used in further analyses (see the high quality column, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). There were 290 426 high-quality ESTs, which represented 84% of the raw sequence data. High-quality EST sequences were obtained frequently and were relatively abundant (93%) in MES and MTE, whereas their frequencies were comparatively low (72–80%) in MLI, MSC, and MSP.</p><p>The sequence length was higher in MES and MTE (702–706 nucleotides), than in MLI, MSC, and MSP (514–551 nucleotides; see the length column in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The differences in the average length of readable sequences and the differences in the frequency of high-quality ESTs probably reflected the quality of each cDNA library. The cDNA libraries were also evaluated for transcription initiation sites by searching for the presence or absence of cap structure-derived guanine nucleotides at the extreme 5′ end of each EST (see the G-cap column, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The frequency of G-cap-positive sequences was &gt;80% in all five cDNA libraries, indicating that the synthesis of most cDNAs was initiated at the 5′ end.</p>                            <h3 scrollto-destination=5353464 id="5353464" class="section-title" data-legacy-id=s2d>2.4. Registration of the sequences</h3>
<p>All of the EST sequences were deposited in the DNA Databank of Japan. Their accession numbers are as follows; HX373156 to HX444163 for MES cDNAs, HX444164 to HX500395 for MLI cDNAs, HX500396 to HX529651 for MSC cDNAs, HX529652 to HX591448 for MSP cDNAs, and HX591449 to HX663542 for MTE cDNAs.</p>                            <h3 scrollto-destination=5353466 id="5353466" class="section-title" data-legacy-id=s2e>2.5. Tools used for sequence analyses</h3>
<p>The following tools were used for sequence analysis: BLAST (The Basic Local Alignment Search Tool) finds regions of local similarity between sequences. The program compares nucleotide or protein sequences to sequence databases and calculates the statistical significance of matches),<sup><a reveal-id="DST007C13" data-open="DST007C13" class="link link-ref link-reveal xref-bibr">13</a></sup> BLAT (The BLAST-Like Alignment Tool uses the index to find regions in the genome likely to be homologous to the query sequence),<sup><a reveal-id="DST007C14" data-open="DST007C14" class="link link-ref link-reveal xref-bibr">14</a></sup> CD-HIT (a widely used program for clustering and comparing protein or nucleotide sequences. CD-HIT helps to significantly reduce the computational and manual efforts in many sequence analysis tasks and aids in understanding the data structure and correct the bias within a dataset),<sup><a reveal-id="DST007C15" data-open="DST007C15" class="link link-ref link-reveal xref-bibr">15</a></sup> CAP3 (a DNA sequence assembly program),<sup><a reveal-id="DST007C16" data-open="DST007C16" class="link link-ref link-reveal xref-bibr">16</a></sup> cross_match (this tool uses cross_match to mask vector/adapter sequences and optimally trim vector sequence and/or polyA/T trail. It takes a set of sequences to be masked, and a set of vectors/adapters used to perform masking),<sup><a reveal-id="DST007C17" data-open="DST007C17" class="link link-ref link-reveal xref-bibr">17</a></sup> EMBOSS (getorf finds and extracts open reading frames),<sup><a reveal-id="DST007C18" data-open="DST007C18" class="link link-ref link-reveal xref-bibr">18</a></sup> and InterProScan (a protein domains identifier).<sup><a reveal-id="DST007C19" data-open="DST007C19" class="link link-ref link-reveal xref-bibr">19</a></sup></p>                            <h2 scrollto-destination=5353468 id="5353468" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=5353469 id="5353469" class="section-title" data-legacy-id=s3a>3.1. EST clustering and assembly</h3>
<p>The 290 426 sequences obtained by normalization of ESTs as in Section 2.3 were clustered and/or assembled to estimate how many genes/transcriptional units were read as ESTs (Table <a reveal-id="DST007TB1" data-open="DST007TB1" class="link link-reveal link-table xref-fig">1</a>). Overlapping sequences were clustered within the longest sequence using the CD-HIT program, whereas overlapping sequences were assembled and extended into a contig using the CAP3 program. According to the CD-HIT program, the number of clusters varied from 10 010 in MLI to 29 028 in MTE. The summed number of contigs and singletons ranged from 8831 in MLI to 25 909 in MTE. According to both programs, MTE had the highest number of clusters/assemblies, suggesting that the testis had the greatest number of expressed transcriptional units. Notably, there were some more clusters than assemblies in each cDNA library. In fact, the total and non-redundant numbers in the five different libraries were 62 210 clusters and 60 568 assemblies (Table <a reveal-id="DST007TB1" data-open="DST007TB1" class="link link-reveal link-table xref-fig">1</a>). This small difference (62 210 versus 60 568) suggests that most of the EST sequences obtained correspond to the mRNA 5′ end (as for another indication of similar performance of CD-HIT and CAP3, see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). </p>                        <a id="5353471" scrollto-destination="5353471"></a>
<div content-id="DST007TB1" class="table-modal"><div class="table-wrap"><div id="DST007TB1" class="table-wrap-title" data-id="DST007TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Clustering of ESTs by CD-HIT and assembly of ESTs by CAP3</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Libraries </th><th>Number of ESTs </th><th>Number of clusters by CD-HIT </th><th>Number of contigs and singlets assembled by CAP3 </th></tr></thead><tbody><tr><td>MES </td><td>71 009 </td><td>17 467 </td><td>15 837 = 5519 (contig) + 10 318 (singlet) </td></tr><tr><td>MLI </td><td>56 232 </td><td>10 010 </td><td>8831 = 3319 (contig) + 5512 (singlet) </td></tr><tr><td>MSC </td><td>29 258 </td><td>12 309 </td><td>10 617 = 3764 (contig) + 6853 (singlet) </td></tr><tr><td>MSP </td><td>61 831 </td><td>16 600 </td><td>14 268 = 5086 (contig) + 9182 (singlet) </td></tr><tr><td>MTE </td><td>72 096 </td><td>29 028 </td><td>25 909 = 8044 (contig) + 17 865 (singlet) </td></tr><tr><td>All </td><td>290 426 </td><td>62 210 </td><td>60 568 = 17 232 (contig) + 43 336 (singlet) </td></tr></tbody></table></div><div content-id="DST007TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Libraries </th><th>Number of ESTs </th><th>Number of clusters by CD-HIT </th><th>Number of contigs and singlets assembled by CAP3 </th></tr></thead><tbody><tr><td>MES </td><td>71 009 </td><td>17 467 </td><td>15 837 = 5519 (contig) + 10 318 (singlet) </td></tr><tr><td>MLI </td><td>56 232 </td><td>10 010 </td><td>8831 = 3319 (contig) + 5512 (singlet) </td></tr><tr><td>MSC </td><td>29 258 </td><td>12 309 </td><td>10 617 = 3764 (contig) + 6853 (singlet) </td></tr><tr><td>MSP </td><td>61 831 </td><td>16 600 </td><td>14 268 = 5086 (contig) + 9182 (singlet) </td></tr><tr><td>MTE </td><td>72 096 </td><td>29 028 </td><td>25 909 = 8044 (contig) + 17 865 (singlet) </td></tr><tr><td>All </td><td>290 426 </td><td>62 210 </td><td>60 568 = 17 232 (contig) + 43 336 (singlet) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Parameters used in CD-HIT and CAP3 programs were default.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5353471" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST007TB1" class="table-wrap-title" data-id="DST007TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Clustering of ESTs by CD-HIT and assembly of ESTs by CAP3</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Libraries </th><th>Number of ESTs </th><th>Number of clusters by CD-HIT </th><th>Number of contigs and singlets assembled by CAP3 </th></tr></thead><tbody><tr><td>MES </td><td>71 009 </td><td>17 467 </td><td>15 837 = 5519 (contig) + 10 318 (singlet) </td></tr><tr><td>MLI </td><td>56 232 </td><td>10 010 </td><td>8831 = 3319 (contig) + 5512 (singlet) </td></tr><tr><td>MSC </td><td>29 258 </td><td>12 309 </td><td>10 617 = 3764 (contig) + 6853 (singlet) </td></tr><tr><td>MSP </td><td>61 831 </td><td>16 600 </td><td>14 268 = 5086 (contig) + 9182 (singlet) </td></tr><tr><td>MTE </td><td>72 096 </td><td>29 028 </td><td>25 909 = 8044 (contig) + 17 865 (singlet) </td></tr><tr><td>All </td><td>290 426 </td><td>62 210 </td><td>60 568 = 17 232 (contig) + 43 336 (singlet) </td></tr></tbody></table></div><div content-id="DST007TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Libraries </th><th>Number of ESTs </th><th>Number of clusters by CD-HIT </th><th>Number of contigs and singlets assembled by CAP3 </th></tr></thead><tbody><tr><td>MES </td><td>71 009 </td><td>17 467 </td><td>15 837 = 5519 (contig) + 10 318 (singlet) </td></tr><tr><td>MLI </td><td>56 232 </td><td>10 010 </td><td>8831 = 3319 (contig) + 5512 (singlet) </td></tr><tr><td>MSC </td><td>29 258 </td><td>12 309 </td><td>10 617 = 3764 (contig) + 6853 (singlet) </td></tr><tr><td>MSP </td><td>61 831 </td><td>16 600 </td><td>14 268 = 5086 (contig) + 9182 (singlet) </td></tr><tr><td>MTE </td><td>72 096 </td><td>29 028 </td><td>25 909 = 8044 (contig) + 17 865 (singlet) </td></tr><tr><td>All </td><td>290 426 </td><td>62 210 </td><td>60 568 = 17 232 (contig) + 43 336 (singlet) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Parameters used in CD-HIT and CAP3 programs were default.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5353471" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=5353472 id="5353472" class="section-title" data-legacy-id=s3b>3.2. Assignment of marmoset ESTs to the Refseq mRNAs of primates</h3>
<p>All 290 426 sequences were examined by comparing them with the known sequences registered at the NCBI as Refseq mRNAs (<a class="link link-uri" href="http://ncbi.nlm.nih.gov/RefSeq/">http://ncbi.nlm.nih.gov/RefSeq/</a>). Refseqs from six different primates were used in the reference dataset, including <em>Homo sapiens</em>, <em>Pan troglodytes</em>, <em>Pongo abelii</em>, <em>Macaca mulatta</em>, <em>Nomascus leucogenys</em>, and <em>C. jacchus</em>. It should be noted that the human Refseqs were based on cDNA sequences, whereas the Refseqs from other primate species were based mainly on the predictions of genomic sequences. The search program used was BLASTn and the threshold of significant homology was set to 1E<sup>−100</sup>, which was a very strict criterion.</p><p>A total of 239 920 and 231 084 sequences shared homology with human and marmoset Refseqs, respectively (Table <a reveal-id="DST007TB2" data-open="DST007TB2" class="link link-reveal link-table xref-fig">2</a>). Homology with the combined Refseqs from the six primates was found for 251 587 sequences (85.6% of 290 426). Therefore, these 251 587 sequences were designated as primate homologues. The average length of the homologous sequences was 528 nucleotides. Of these 251 587 sequences, 4974 sequences were identical to their corresponding Refseq sequences, whereas 94 102 sequences shared 100% nucleotide sequence identity only with the aligned homologous regions. Additionally, 931 sequences shared homology with the Refseqs of non-primates. Most of these sequences were homologous to mouse Refseqs, suggesting that they were probably derived from mouse cells that were used as a feeder layer to culture ES cells. </p>                        <a id="5353475" scrollto-destination="5353475"></a>
<div content-id="DST007TB2" class="table-modal"><div class="table-wrap"><div id="DST007TB2" class="table-wrap-title" data-id="DST007TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Assignment of common marmoset ESTs to primates’ Refseq</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species derivation of Refseq </th><th>Number of ESTs homologous to Refseq (number of homologous genes) </th></tr></thead><tbody><tr><td><em>Homo sapiens</em> </td><td>239 920 (13 825) </td></tr><tr><td><em>Pan troglodytes</em> </td><td>233 913 (14 372) </td></tr><tr><td><em>Callithrix jacchus</em> </td><td>231 084 (13 499) </td></tr><tr><td><em>Pongo abelii</em> </td><td>231 354 (13 898) </td></tr><tr><td><em>Macaca mulatta</em> </td><td>229 151 (13 677) </td></tr><tr><td><em>Nomascus leucogenys</em> </td><td>228 749 (13 296) </td></tr><tr><td>Six primates </td><td>251 587 </td></tr><tr><td>Non-primates </td><td>931 </td></tr></tbody></table></div><div content-id="DST007TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Species derivation of Refseq </th><th>Number of ESTs homologous to Refseq (number of homologous genes) </th></tr></thead><tbody><tr><td><em>Homo sapiens</em> </td><td>239 920 (13 825) </td></tr><tr><td><em>Pan troglodytes</em> </td><td>233 913 (14 372) </td></tr><tr><td><em>Callithrix jacchus</em> </td><td>231 084 (13 499) </td></tr><tr><td><em>Pongo abelii</em> </td><td>231 354 (13 898) </td></tr><tr><td><em>Macaca mulatta</em> </td><td>229 151 (13 677) </td></tr><tr><td><em>Nomascus leucogenys</em> </td><td>228 749 (13 296) </td></tr><tr><td>Six primates </td><td>251 587 </td></tr><tr><td>Non-primates </td><td>931 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>EST sequences of common marmoset (total 290 426) were referred to primates’ Refseq mRNA that are registered at NCBI. Homology was searched using BLASTn and judged significant at &lt;1E<sup>−100</sup>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5353475" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST007TB2" class="table-wrap-title" data-id="DST007TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Assignment of common marmoset ESTs to primates’ Refseq</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species derivation of Refseq </th><th>Number of ESTs homologous to Refseq (number of homologous genes) </th></tr></thead><tbody><tr><td><em>Homo sapiens</em> </td><td>239 920 (13 825) </td></tr><tr><td><em>Pan troglodytes</em> </td><td>233 913 (14 372) </td></tr><tr><td><em>Callithrix jacchus</em> </td><td>231 084 (13 499) </td></tr><tr><td><em>Pongo abelii</em> </td><td>231 354 (13 898) </td></tr><tr><td><em>Macaca mulatta</em> </td><td>229 151 (13 677) </td></tr><tr><td><em>Nomascus leucogenys</em> </td><td>228 749 (13 296) </td></tr><tr><td>Six primates </td><td>251 587 </td></tr><tr><td>Non-primates </td><td>931 </td></tr></tbody></table></div><div content-id="DST007TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Species derivation of Refseq </th><th>Number of ESTs homologous to Refseq (number of homologous genes) </th></tr></thead><tbody><tr><td><em>Homo sapiens</em> </td><td>239 920 (13 825) </td></tr><tr><td><em>Pan troglodytes</em> </td><td>233 913 (14 372) </td></tr><tr><td><em>Callithrix jacchus</em> </td><td>231 084 (13 499) </td></tr><tr><td><em>Pongo abelii</em> </td><td>231 354 (13 898) </td></tr><tr><td><em>Macaca mulatta</em> </td><td>229 151 (13 677) </td></tr><tr><td><em>Nomascus leucogenys</em> </td><td>228 749 (13 296) </td></tr><tr><td>Six primates </td><td>251 587 </td></tr><tr><td>Non-primates </td><td>931 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>EST sequences of common marmoset (total 290 426) were referred to primates’ Refseq mRNA that are registered at NCBI. Homology was searched using BLASTn and judged significant at &lt;1E<sup>−100</sup>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5353475" target="_blank">View Large</a></div></div><p>Out of the above described 251 587 primate homologues, only sequences that shared homology with the Refseqs of all six primates were extracted, and that yielded 199 511 sequences. Then, for each of 199 511 sequences, the alignments between the EST and Refseq with the highest score among the six primates were selected, and 199 511 sequences were grouped into six for each primate species. Finally, the average identities and coverage were calculated using the alignments for each primate species (Table <a reveal-id="DST007TB3" data-open="DST007TB3" class="link link-reveal link-table xref-fig">3</a>). As expected, the highest sequence identity (99%) was between the marmoset EST and marmoset Refseq. The average sequence identities between the marmoset EST and the Refseqs of the other five primate species were in a similar range (94–95%). Thus, the sequences differed by ∼5% between the marmoset and the other five primates. The coverage was highest between the marmoset EST and human Refseq (91%), whereas it was lowest between the marmoset EST and marmoset Refseq (84%). This difference of 6% corresponded to 25 nucleotides, so it is likely that the alignment of the EST and marmoset Refseqs started 25 nucleotides downstream from the 5′ end of the marmoset Refseq. This might suggest that the 5′UTR of the marmoset Refseq was not predicted precisely from its genomic sequence. </p>                        <a id="5353477" scrollto-destination="5353477"></a>
<div content-id="DST007TB3" class="table-modal"><div class="table-wrap"><div id="DST007TB3" class="table-wrap-title" data-id="DST007TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Identity and coverage between homologous marmoset ESTs and primates’ Refseq</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species derivation of Refseq </th><th>Identity for Refseq (for 9879 HomoloGenes) </th><th>Coverage for Refseq (for 9879 HomoloGenes) </th></tr></thead><tbody><tr><td><em>Homo sapiens</em> </td><td>94.88% (94.54%) </td><td>91.14% (93.91%) </td></tr><tr><td><em>Pan troglodytes</em> </td><td>94.86% (94.54%) </td><td>88.84% (92.11%) </td></tr><tr><td><em>Callithrix jacchu</em>s </td><td>99.55% </td><td>84.70% </td></tr><tr><td><em>Pongo abelii</em> </td><td>94.77% </td><td>87.36% </td></tr><tr><td><em>Macaca mulatta</em> </td><td>94.73% (94.44%) </td><td>87.20% (89.66%) </td></tr><tr><td><em>Nomascus leucogenys</em> </td><td>94.72% </td><td>87.80% </td></tr></tbody></table></div><div content-id="DST007TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Species derivation of Refseq </th><th>Identity for Refseq (for 9879 HomoloGenes) </th><th>Coverage for Refseq (for 9879 HomoloGenes) </th></tr></thead><tbody><tr><td><em>Homo sapiens</em> </td><td>94.88% (94.54%) </td><td>91.14% (93.91%) </td></tr><tr><td><em>Pan troglodytes</em> </td><td>94.86% (94.54%) </td><td>88.84% (92.11%) </td></tr><tr><td><em>Callithrix jacchu</em>s </td><td>99.55% </td><td>84.70% </td></tr><tr><td><em>Pongo abelii</em> </td><td>94.77% </td><td>87.36% </td></tr><tr><td><em>Macaca mulatta</em> </td><td>94.73% (94.44%) </td><td>87.20% (89.66%) </td></tr><tr><td><em>Nomascus leucogenys</em> </td><td>94.72% </td><td>87.80% </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Identity (%) represents a degree of identity between the aligned two sequences of high-scoring segment pairs, whereas coverage (%) represents a ratio of aligned sequence length over an entire length of EST. See the text as for the details how identity and coverage were calculated.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5353477" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST007TB3" class="table-wrap-title" data-id="DST007TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Identity and coverage between homologous marmoset ESTs and primates’ Refseq</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species derivation of Refseq </th><th>Identity for Refseq (for 9879 HomoloGenes) </th><th>Coverage for Refseq (for 9879 HomoloGenes) </th></tr></thead><tbody><tr><td><em>Homo sapiens</em> </td><td>94.88% (94.54%) </td><td>91.14% (93.91%) </td></tr><tr><td><em>Pan troglodytes</em> </td><td>94.86% (94.54%) </td><td>88.84% (92.11%) </td></tr><tr><td><em>Callithrix jacchu</em>s </td><td>99.55% </td><td>84.70% </td></tr><tr><td><em>Pongo abelii</em> </td><td>94.77% </td><td>87.36% </td></tr><tr><td><em>Macaca mulatta</em> </td><td>94.73% (94.44%) </td><td>87.20% (89.66%) </td></tr><tr><td><em>Nomascus leucogenys</em> </td><td>94.72% </td><td>87.80% </td></tr></tbody></table></div><div content-id="DST007TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Species derivation of Refseq </th><th>Identity for Refseq (for 9879 HomoloGenes) </th><th>Coverage for Refseq (for 9879 HomoloGenes) </th></tr></thead><tbody><tr><td><em>Homo sapiens</em> </td><td>94.88% (94.54%) </td><td>91.14% (93.91%) </td></tr><tr><td><em>Pan troglodytes</em> </td><td>94.86% (94.54%) </td><td>88.84% (92.11%) </td></tr><tr><td><em>Callithrix jacchu</em>s </td><td>99.55% </td><td>84.70% </td></tr><tr><td><em>Pongo abelii</em> </td><td>94.77% </td><td>87.36% </td></tr><tr><td><em>Macaca mulatta</em> </td><td>94.73% (94.44%) </td><td>87.20% (89.66%) </td></tr><tr><td><em>Nomascus leucogenys</em> </td><td>94.72% </td><td>87.80% </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Identity (%) represents a degree of identity between the aligned two sequences of high-scoring segment pairs, whereas coverage (%) represents a ratio of aligned sequence length over an entire length of EST. See the text as for the details how identity and coverage were calculated.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5353477" target="_blank">View Large</a></div></div><p>In the above assignment of marmoset ESTs to primates’ Refseqs in Table <a reveal-id="DST007TB2" data-open="DST007TB2" class="link link-reveal link-table xref-fig">2</a>, the Refseqs used as references can be re-classified into non-overlapping, distinct gene entities. Homologous sequences of 239 920 and 231 084 corresponded to 13 825 human and 13 499 marmoset genes, respectively (see the numbers in parenthesis in Table <a reveal-id="DST007TB2" data-open="DST007TB2" class="link link-reveal link-table xref-fig">2</a>). In addition, we searched for HomoloGenes (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/homologene">http://www.ncbi.nlm.nih.gov/homologene</a>) that are found in common among <em>Homo sapiens</em>, <em>Pan troglodytes</em>, and <em>Macaca mulatta</em>, and detected 9879 Homologenes. When identities and coverage between marmoset ESTs and primates’ Refseq were recalculated for these 9879 Homologenes (see the numbers in parenthesis in Table <a reveal-id="DST007TB3" data-open="DST007TB3" class="link link-reveal link-table xref-fig">3</a>), coverage increased by 2–3%, whereas identity between marmoset and human remained the same (94–95%). Thus, it appears again plausible that the sequences diverge by 5% between marmoset and other primates including human.</p>                            <h3 scrollto-destination=5353479 id="5353479" class="section-title" data-legacy-id=s3c>3.3. Mapping marmoset ESTs to the human genome</h3>
<p>Each marmoset EST sequence (total 290 426 reads) was mapped to the human genome (hg19) using the BLAT search program. This showed that 99.1% of the ESTs (287 849 reads) mapped to the human genome (Table <a reveal-id="DST007TB4" data-open="DST007TB4" class="link link-reveal link-table xref-fig">4</a>). This mapping result was filtered further using the UCSC Genome Browser and a pslCDnaFilter. Finally, 93.2% (270 673 reads) of the marmoset ESTs were assigned specifically and exactly to the human genome. </p>                        <a id="5353481" scrollto-destination="5353481"></a>
<div content-id="DST007TB4" class="table-modal"><div class="table-wrap"><div id="DST007TB4" class="table-wrap-title" data-id="DST007TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Mapping of ESTs on the human genome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Libraries </th><th>Number of marmoset ESTs </th><th>Number of ESTs mapped on the human genome (raw data) </th><th>Number of ESTs mapped on the human genome (filtered) </th></tr></thead><tbody><tr><td>MES </td><td>71 009 </td><td>70 375 </td><td>66 894 (94.2%) </td></tr><tr><td>MLI </td><td>56 232 </td><td>55 602 </td><td>52 405 (93.2%) </td></tr><tr><td>MSC </td><td>29 258 </td><td>28 931 </td><td>27 253 (93.1%) </td></tr><tr><td>MSP </td><td>61 831 </td><td>61 300 </td><td>58 170 (94.1%) </td></tr><tr><td>MTE </td><td>72 096 </td><td>71 641 </td><td>65 951 (91.5%) </td></tr><tr><td>All </td><td>290 426 </td><td>287 849 (99.1%) </td><td>270 673 (93.2%) </td></tr></tbody></table></div><div content-id="DST007TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Libraries </th><th>Number of marmoset ESTs </th><th>Number of ESTs mapped on the human genome (raw data) </th><th>Number of ESTs mapped on the human genome (filtered) </th></tr></thead><tbody><tr><td>MES </td><td>71 009 </td><td>70 375 </td><td>66 894 (94.2%) </td></tr><tr><td>MLI </td><td>56 232 </td><td>55 602 </td><td>52 405 (93.2%) </td></tr><tr><td>MSC </td><td>29 258 </td><td>28 931 </td><td>27 253 (93.1%) </td></tr><tr><td>MSP </td><td>61 831 </td><td>61 300 </td><td>58 170 (94.1%) </td></tr><tr><td>MTE </td><td>72 096 </td><td>71 641 </td><td>65 951 (91.5%) </td></tr><tr><td>All </td><td>290 426 </td><td>287 849 (99.1%) </td><td>270 673 (93.2%) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Marmoset ESTs (290 426) were mapped on the human genome (hg19) by using a BLAT search program (−stepSize = 5, −minScore = 50, −minIdentity = 80, −repMatch = 2253). This initial mapping gave a number of 287 849 (99.1%) ESTs. These ESTs were then filtered, following a UCSC Genome Browser and using a pslCDnaFilter (−minId = 0.85, −minCover = 0.75, −globalNearBest = 0.0025, −minQSize = 20, −minNonRepSize = 16, −ignoreNs, −bestOveralp). Basis for adopting this filtering condition was as follows. A use of the condition such as (−minId = 0.95, −minCover = 0.25) selected only 159 309 ESTs (54.9%), indicating −minId = 0.95 to be extremely strict in the exactness. Therefore, we lowered the −minId to 0.85 (and −minCover = 0.25) and found the reasonably selected numbers of exact ESTs. Thus, under this −minId of 0.85, we then tried to improve the specificity by increasing −minCover to 0.75 (since 0.90 appeared too strict, 0.90 was not used). This −minCover number of 0.75 is roughly equal to the expected coverage of coding sequence [the average length of EST was 619 nt, and the average length of 5′UTR of human transcripts is 170 nt, therefore, an expected coverage between coding sequences and ESTs would be (619–170)/619 = 0.73]. Eventually, a condition of −minId = 0.85, −minCover = 0.75 filtered 270 673 ESTs (93.2%) as exact and specific.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5353481" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST007TB4" class="table-wrap-title" data-id="DST007TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Mapping of ESTs on the human genome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Libraries </th><th>Number of marmoset ESTs </th><th>Number of ESTs mapped on the human genome (raw data) </th><th>Number of ESTs mapped on the human genome (filtered) </th></tr></thead><tbody><tr><td>MES </td><td>71 009 </td><td>70 375 </td><td>66 894 (94.2%) </td></tr><tr><td>MLI </td><td>56 232 </td><td>55 602 </td><td>52 405 (93.2%) </td></tr><tr><td>MSC </td><td>29 258 </td><td>28 931 </td><td>27 253 (93.1%) </td></tr><tr><td>MSP </td><td>61 831 </td><td>61 300 </td><td>58 170 (94.1%) </td></tr><tr><td>MTE </td><td>72 096 </td><td>71 641 </td><td>65 951 (91.5%) </td></tr><tr><td>All </td><td>290 426 </td><td>287 849 (99.1%) </td><td>270 673 (93.2%) </td></tr></tbody></table></div><div content-id="DST007TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Libraries </th><th>Number of marmoset ESTs </th><th>Number of ESTs mapped on the human genome (raw data) </th><th>Number of ESTs mapped on the human genome (filtered) </th></tr></thead><tbody><tr><td>MES </td><td>71 009 </td><td>70 375 </td><td>66 894 (94.2%) </td></tr><tr><td>MLI </td><td>56 232 </td><td>55 602 </td><td>52 405 (93.2%) </td></tr><tr><td>MSC </td><td>29 258 </td><td>28 931 </td><td>27 253 (93.1%) </td></tr><tr><td>MSP </td><td>61 831 </td><td>61 300 </td><td>58 170 (94.1%) </td></tr><tr><td>MTE </td><td>72 096 </td><td>71 641 </td><td>65 951 (91.5%) </td></tr><tr><td>All </td><td>290 426 </td><td>287 849 (99.1%) </td><td>270 673 (93.2%) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Marmoset ESTs (290 426) were mapped on the human genome (hg19) by using a BLAT search program (−stepSize = 5, −minScore = 50, −minIdentity = 80, −repMatch = 2253). This initial mapping gave a number of 287 849 (99.1%) ESTs. These ESTs were then filtered, following a UCSC Genome Browser and using a pslCDnaFilter (−minId = 0.85, −minCover = 0.75, −globalNearBest = 0.0025, −minQSize = 20, −minNonRepSize = 16, −ignoreNs, −bestOveralp). Basis for adopting this filtering condition was as follows. A use of the condition such as (−minId = 0.95, −minCover = 0.25) selected only 159 309 ESTs (54.9%), indicating −minId = 0.95 to be extremely strict in the exactness. Therefore, we lowered the −minId to 0.85 (and −minCover = 0.25) and found the reasonably selected numbers of exact ESTs. Thus, under this −minId of 0.85, we then tried to improve the specificity by increasing −minCover to 0.75 (since 0.90 appeared too strict, 0.90 was not used). This −minCover number of 0.75 is roughly equal to the expected coverage of coding sequence [the average length of EST was 619 nt, and the average length of 5′UTR of human transcripts is 170 nt, therefore, an expected coverage between coding sequences and ESTs would be (619–170)/619 = 0.73]. Eventually, a condition of −minId = 0.85, −minCover = 0.75 filtered 270 673 ESTs (93.2%) as exact and specific.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5353481" target="_blank">View Large</a></div></div><p>The nucleotide sequence identity between aligned sequences was calculated after each EST had been mapped to the genome (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). The majority of ESTs shared 90–96% identity with the human genome. The relatively low identity between the ESTs and human sequences was analogous to the identity between the EST and Refseq shown in Table <a reveal-id="DST007TB3" data-open="DST007TB3" class="link link-reveal link-table xref-fig">3</a>. It should be noted that the degree of mapping to the human genome did not differ significantly among the five cDNA libraries (data not shown).</p>                            <h4 scrollto-destination=5353483 id="5353483" class="section-title" data-legacy-id=s3c1>3.3.1. ESTs that were mapped to exonic regions of the human genome</h4>
<p>The CAP3 method identified 17 232 contigs (Table <a reveal-id="DST007TB1" data-open="DST007TB1" class="link link-reveal link-table xref-fig">1</a>). The contig sequences represented those that were read multiple times in the overlapping regions and they were considered more reliable than single-read singletons. These 17 232 contigs were mapped onto the human genome according to the filtering method shown in Table <a reveal-id="DST007TB4" data-open="DST007TB4" class="link link-reveal link-table xref-fig">4</a>. Furthermore, information from human Refseqs was used to determine whether the mapped contigs corresponded to exonic sequences of the human gene. This approach identified 15 089 contigs (88% of 17 232) that mapped onto the human genome and that corresponded to human Refseqs. These 15 089 contigs, which consisted of 218 857 ESTs, were considered to be representative of the genes that are commonly transcribed in marmosets and humans.</p><p>Of these 15 089 contigs, ESTs that were present in only one type of cDNA library were identified. Of these tissue-specific contigs, those with the highest numbers of constitutive ESTs are shown in Table <a reveal-id="DST007TB5" data-open="DST007TB5" class="link link-reveal link-table xref-fig">5</a>. The contigs with &gt;1000 ESTs were <em>ALB</em>, <em>HPR</em>, <em>ORM2</em>, and <em>ORM1</em> from MLI, and <em>PRM1</em> from MTE. Most of the genes listed in Table <a reveal-id="DST007TB5" data-open="DST007TB5" class="link link-reveal link-table xref-fig">5</a> are known to be characteristic of each specific cell type/tissue. </p>                        <a id="5353486" scrollto-destination="5353486"></a>
<div content-id="DST007TB5" class="table-modal"><div class="table-wrap"><div id="DST007TB5" class="table-wrap-title" data-id="DST007TB5"><div class="label -label">Table 5.</div><div class="caption"><p>Top five genes expressed abundantly in each cDNA library</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Libraries </th><th>Gene symbols </th><th>Descriptions </th><th>Number of ESTs </th></tr></thead><tbody><tr><td>MES </td><td><em>PYY</em> </td><td>Peptide YY </td><td>123 </td></tr><tr><td>MES </td><td><em>LIN28A</em> </td><td>Lin-28 homologue A </td><td>50 </td></tr><tr><td>MES </td><td><em>C6orf221</em> </td><td>Chromosome 6 open reading frame 221 </td><td>46 </td></tr><tr><td>MES </td><td><em>NANOG</em> </td><td>Nanog homeobox </td><td>45 </td></tr><tr><td>MES </td><td><em>ERVMER34-1</em> </td><td>Endogenous retrovirus group MER34, member 1 </td><td>28 </td></tr><tr><td>MLI </td><td><em>ALB</em> </td><td>Albumin </td><td>4492 </td></tr><tr><td>MLI </td><td><em>HPR</em> </td><td>Haptoglobin-related protein </td><td>1537 </td></tr><tr><td>MLI </td><td><em>ORM2</em> </td><td>Orosomucoid 2 </td><td>1227 </td></tr><tr><td>MLI </td><td><em>ORM1</em> </td><td>Orosomucoid 1 </td><td>1221 </td></tr><tr><td>MLI </td><td><em>APOA2</em> </td><td>Apolipoprotein A-II </td><td>673 </td></tr><tr><td>MSC </td><td><em>SNAP25</em> </td><td>Synaptosomal-associated protein, 25 kDa </td><td>109 </td></tr><tr><td>MSC </td><td><em>PLP1</em> </td><td>Proteolipid protein 1 </td><td>58 </td></tr><tr><td>MSC </td><td><em>CALCA</em> </td><td>Calcitonin-related polypeptide alpha </td><td>51 </td></tr><tr><td>MSC </td><td><em>STMN2</em> </td><td>Stathmin-like 2 </td><td>30 </td></tr><tr><td>MSC </td><td><em>THY1</em> </td><td>Thy-1 cell surface antigen </td><td>28 </td></tr><tr><td>MSP </td><td><em>MS4A1</em> </td><td>Membrane-spanning 4-domains, subfamily A, member 1 </td><td>62 </td></tr><tr><td>MSP </td><td><em>ITGB2</em> </td><td>Integrin, beta 2 </td><td>29 </td></tr><tr><td>MSP </td><td><em>HLA-DPA1</em> </td><td>Major histocompatibility complex, class II, DP alpha 1 </td><td>26 </td></tr><tr><td>MSP </td><td><em>CLEC4F</em> </td><td>C-type lectin domain family 4, member F </td><td>20 </td></tr><tr><td>MSP </td><td><em>CD53</em> </td><td>CD53 molecule </td><td>18 </td></tr><tr><td>MTE </td><td><em>PRM1</em> </td><td>Protamine 1 </td><td>1,198 </td></tr><tr><td>MTE </td><td><em>TNP1</em> </td><td>Transition protein 1 </td><td>161 </td></tr><tr><td>MTE </td><td><em>HMGB4</em> </td><td>High mobility group box 4 </td><td>139 </td></tr><tr><td>MTE </td><td><em>PHF7</em> </td><td>PHD finger protein 7 </td><td>138 </td></tr><tr><td>MTE </td><td><em>DKKL1</em> </td><td>Dickkopf-like 1 </td><td>107 </td></tr></tbody></table></div><div content-id="DST007TB5" class="table-modal"><table><thead name="thead" align=""><tr><th>Libraries </th><th>Gene symbols </th><th>Descriptions </th><th>Number of ESTs </th></tr></thead><tbody><tr><td>MES </td><td><em>PYY</em> </td><td>Peptide YY </td><td>123 </td></tr><tr><td>MES </td><td><em>LIN28A</em> </td><td>Lin-28 homologue A </td><td>50 </td></tr><tr><td>MES </td><td><em>C6orf221</em> </td><td>Chromosome 6 open reading frame 221 </td><td>46 </td></tr><tr><td>MES </td><td><em>NANOG</em> </td><td>Nanog homeobox </td><td>45 </td></tr><tr><td>MES </td><td><em>ERVMER34-1</em> </td><td>Endogenous retrovirus group MER34, member 1 </td><td>28 </td></tr><tr><td>MLI </td><td><em>ALB</em> </td><td>Albumin </td><td>4492 </td></tr><tr><td>MLI </td><td><em>HPR</em> </td><td>Haptoglobin-related protein </td><td>1537 </td></tr><tr><td>MLI </td><td><em>ORM2</em> </td><td>Orosomucoid 2 </td><td>1227 </td></tr><tr><td>MLI </td><td><em>ORM1</em> </td><td>Orosomucoid 1 </td><td>1221 </td></tr><tr><td>MLI </td><td><em>APOA2</em> </td><td>Apolipoprotein A-II </td><td>673 </td></tr><tr><td>MSC </td><td><em>SNAP25</em> </td><td>Synaptosomal-associated protein, 25 kDa </td><td>109 </td></tr><tr><td>MSC </td><td><em>PLP1</em> </td><td>Proteolipid protein 1 </td><td>58 </td></tr><tr><td>MSC </td><td><em>CALCA</em> </td><td>Calcitonin-related polypeptide alpha </td><td>51 </td></tr><tr><td>MSC </td><td><em>STMN2</em> </td><td>Stathmin-like 2 </td><td>30 </td></tr><tr><td>MSC </td><td><em>THY1</em> </td><td>Thy-1 cell surface antigen </td><td>28 </td></tr><tr><td>MSP </td><td><em>MS4A1</em> </td><td>Membrane-spanning 4-domains, subfamily A, member 1 </td><td>62 </td></tr><tr><td>MSP </td><td><em>ITGB2</em> </td><td>Integrin, beta 2 </td><td>29 </td></tr><tr><td>MSP </td><td><em>HLA-DPA1</em> </td><td>Major histocompatibility complex, class II, DP alpha 1 </td><td>26 </td></tr><tr><td>MSP </td><td><em>CLEC4F</em> </td><td>C-type lectin domain family 4, member F </td><td>20 </td></tr><tr><td>MSP </td><td><em>CD53</em> </td><td>CD53 molecule </td><td>18 </td></tr><tr><td>MTE </td><td><em>PRM1</em> </td><td>Protamine 1 </td><td>1,198 </td></tr><tr><td>MTE </td><td><em>TNP1</em> </td><td>Transition protein 1 </td><td>161 </td></tr><tr><td>MTE </td><td><em>HMGB4</em> </td><td>High mobility group box 4 </td><td>139 </td></tr><tr><td>MTE </td><td><em>PHF7</em> </td><td>PHD finger protein 7 </td><td>138 </td></tr><tr><td>MTE </td><td><em>DKKL1</em> </td><td>Dickkopf-like 1 </td><td>107 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Shown are the contigs that were detected only in one out of five cDNA libraries and possessed larger numbers of constituting ESTs.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5353486" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST007TB5" class="table-wrap-title" data-id="DST007TB5"><div class="label -label">Table 5.</div><div class="caption"><p>Top five genes expressed abundantly in each cDNA library</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Libraries </th><th>Gene symbols </th><th>Descriptions </th><th>Number of ESTs </th></tr></thead><tbody><tr><td>MES </td><td><em>PYY</em> </td><td>Peptide YY </td><td>123 </td></tr><tr><td>MES </td><td><em>LIN28A</em> </td><td>Lin-28 homologue A </td><td>50 </td></tr><tr><td>MES </td><td><em>C6orf221</em> </td><td>Chromosome 6 open reading frame 221 </td><td>46 </td></tr><tr><td>MES </td><td><em>NANOG</em> </td><td>Nanog homeobox </td><td>45 </td></tr><tr><td>MES </td><td><em>ERVMER34-1</em> </td><td>Endogenous retrovirus group MER34, member 1 </td><td>28 </td></tr><tr><td>MLI </td><td><em>ALB</em> </td><td>Albumin </td><td>4492 </td></tr><tr><td>MLI </td><td><em>HPR</em> </td><td>Haptoglobin-related protein </td><td>1537 </td></tr><tr><td>MLI </td><td><em>ORM2</em> </td><td>Orosomucoid 2 </td><td>1227 </td></tr><tr><td>MLI </td><td><em>ORM1</em> </td><td>Orosomucoid 1 </td><td>1221 </td></tr><tr><td>MLI </td><td><em>APOA2</em> </td><td>Apolipoprotein A-II </td><td>673 </td></tr><tr><td>MSC </td><td><em>SNAP25</em> </td><td>Synaptosomal-associated protein, 25 kDa </td><td>109 </td></tr><tr><td>MSC </td><td><em>PLP1</em> </td><td>Proteolipid protein 1 </td><td>58 </td></tr><tr><td>MSC </td><td><em>CALCA</em> </td><td>Calcitonin-related polypeptide alpha </td><td>51 </td></tr><tr><td>MSC </td><td><em>STMN2</em> </td><td>Stathmin-like 2 </td><td>30 </td></tr><tr><td>MSC </td><td><em>THY1</em> </td><td>Thy-1 cell surface antigen </td><td>28 </td></tr><tr><td>MSP </td><td><em>MS4A1</em> </td><td>Membrane-spanning 4-domains, subfamily A, member 1 </td><td>62 </td></tr><tr><td>MSP </td><td><em>ITGB2</em> </td><td>Integrin, beta 2 </td><td>29 </td></tr><tr><td>MSP </td><td><em>HLA-DPA1</em> </td><td>Major histocompatibility complex, class II, DP alpha 1 </td><td>26 </td></tr><tr><td>MSP </td><td><em>CLEC4F</em> </td><td>C-type lectin domain family 4, member F </td><td>20 </td></tr><tr><td>MSP </td><td><em>CD53</em> </td><td>CD53 molecule </td><td>18 </td></tr><tr><td>MTE </td><td><em>PRM1</em> </td><td>Protamine 1 </td><td>1,198 </td></tr><tr><td>MTE </td><td><em>TNP1</em> </td><td>Transition protein 1 </td><td>161 </td></tr><tr><td>MTE </td><td><em>HMGB4</em> </td><td>High mobility group box 4 </td><td>139 </td></tr><tr><td>MTE </td><td><em>PHF7</em> </td><td>PHD finger protein 7 </td><td>138 </td></tr><tr><td>MTE </td><td><em>DKKL1</em> </td><td>Dickkopf-like 1 </td><td>107 </td></tr></tbody></table></div><div content-id="DST007TB5" class="table-modal"><table><thead name="thead" align=""><tr><th>Libraries </th><th>Gene symbols </th><th>Descriptions </th><th>Number of ESTs </th></tr></thead><tbody><tr><td>MES </td><td><em>PYY</em> </td><td>Peptide YY </td><td>123 </td></tr><tr><td>MES </td><td><em>LIN28A</em> </td><td>Lin-28 homologue A </td><td>50 </td></tr><tr><td>MES </td><td><em>C6orf221</em> </td><td>Chromosome 6 open reading frame 221 </td><td>46 </td></tr><tr><td>MES </td><td><em>NANOG</em> </td><td>Nanog homeobox </td><td>45 </td></tr><tr><td>MES </td><td><em>ERVMER34-1</em> </td><td>Endogenous retrovirus group MER34, member 1 </td><td>28 </td></tr><tr><td>MLI </td><td><em>ALB</em> </td><td>Albumin </td><td>4492 </td></tr><tr><td>MLI </td><td><em>HPR</em> </td><td>Haptoglobin-related protein </td><td>1537 </td></tr><tr><td>MLI </td><td><em>ORM2</em> </td><td>Orosomucoid 2 </td><td>1227 </td></tr><tr><td>MLI </td><td><em>ORM1</em> </td><td>Orosomucoid 1 </td><td>1221 </td></tr><tr><td>MLI </td><td><em>APOA2</em> </td><td>Apolipoprotein A-II </td><td>673 </td></tr><tr><td>MSC </td><td><em>SNAP25</em> </td><td>Synaptosomal-associated protein, 25 kDa </td><td>109 </td></tr><tr><td>MSC </td><td><em>PLP1</em> </td><td>Proteolipid protein 1 </td><td>58 </td></tr><tr><td>MSC </td><td><em>CALCA</em> </td><td>Calcitonin-related polypeptide alpha </td><td>51 </td></tr><tr><td>MSC </td><td><em>STMN2</em> </td><td>Stathmin-like 2 </td><td>30 </td></tr><tr><td>MSC </td><td><em>THY1</em> </td><td>Thy-1 cell surface antigen </td><td>28 </td></tr><tr><td>MSP </td><td><em>MS4A1</em> </td><td>Membrane-spanning 4-domains, subfamily A, member 1 </td><td>62 </td></tr><tr><td>MSP </td><td><em>ITGB2</em> </td><td>Integrin, beta 2 </td><td>29 </td></tr><tr><td>MSP </td><td><em>HLA-DPA1</em> </td><td>Major histocompatibility complex, class II, DP alpha 1 </td><td>26 </td></tr><tr><td>MSP </td><td><em>CLEC4F</em> </td><td>C-type lectin domain family 4, member F </td><td>20 </td></tr><tr><td>MSP </td><td><em>CD53</em> </td><td>CD53 molecule </td><td>18 </td></tr><tr><td>MTE </td><td><em>PRM1</em> </td><td>Protamine 1 </td><td>1,198 </td></tr><tr><td>MTE </td><td><em>TNP1</em> </td><td>Transition protein 1 </td><td>161 </td></tr><tr><td>MTE </td><td><em>HMGB4</em> </td><td>High mobility group box 4 </td><td>139 </td></tr><tr><td>MTE </td><td><em>PHF7</em> </td><td>PHD finger protein 7 </td><td>138 </td></tr><tr><td>MTE </td><td><em>DKKL1</em> </td><td>Dickkopf-like 1 </td><td>107 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Shown are the contigs that were detected only in one out of five cDNA libraries and possessed larger numbers of constituting ESTs.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5353486" target="_blank">View Large</a></div></div>                            <h4 scrollto-destination=5353487 id="5353487" class="section-title" data-legacy-id=s3c2>3.3.2. ES cell-specific transcripts</h4>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a> shows previously reported cDNA/EST studies of primates other than the marmoset.<sup><a reveal-id="DST007C20  DST007C24" data-open="DST007C20  DST007C24" class="link link-ref link-reveal xref-bibr">20–24</a></sup> These species include <em>Pan troglodytes</em>, <em>Macaca fascicularis</em>, and <em>Chlorocebus sabaeus</em>, while the tissues included the brain, skin, liver, B lymphocytes, bone marrow, pancreas, spleen, thymus, and peripheral blood mononuclear cells. As the current study is the first example of the use of monkey-derived ES cells in EST studies, the ES-specific transcripts are mentioned briefly (see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a> where the contigs containing &gt;5 ESTs are listed).</p><p>The most notable were <em>LIN28A</em>, <em>NANOG</em>, and <em>SOX2</em> because, together with <em>OCT4</em>, they are known to reprogram human somatic cells to induced pluripotent stem cells.<sup><a reveal-id="DST007C25" data-open="DST007C25" class="link link-ref link-reveal xref-bibr">25</a></sup><em>LIN28A</em> and <em>NANOG</em> contribute to the maintenance of pluripotency in stem cells.<sup><a reveal-id="DST007C26  DST007C28" data-open="DST007C26  DST007C28" class="link link-ref link-reveal xref-bibr">26–28</a></sup><em>C6orf221</em> (also known as <em>ECAT1</em>, ES cell-associated transcript 1) and <em>DPPA5</em> belong to the same gene family, and they are expressed specifically in human ES cells.<sup><a reveal-id="DST007C29" data-open="DST007C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DST007C30" data-open="DST007C30" class="link link-ref link-reveal xref-bibr">30</a></sup> The frequent appearance of the claudin family (see <em>CLDN6</em>, <em>CLDN9</em>, <em>CLDN7</em>, and <em>CLDN4</em>) was also noted, although its biological significance is not known.</p>                            <h4 scrollto-destination=5353490 id="5353490" class="section-title" data-legacy-id=s3c3>3.3.3. ESTs that were mapped onto the human genome outside exonic regions</h4>
<p>We noted that there was another category of contigs that was mapped onto the human genome, but not located in exonic regions (i.e. contigs that shared no homology with human Refseqs). These comprised 808 contigs (4.7% of 17 232) with 5578 ESTs. These 808 contigs represented transcribed genes in the marmoset. Although they were conserved in the human genome, they did not appear to be transcribed as genes; hence, they were not characterized as human genes.</p><p>Next, the number of marmoset cDNA libraries, in which each contig was expressed, was determined. The number of contigs with ESTs detected in libraries 1 (i.e. only MES), 2 (i.e. MES and MTE), 3, 4, and 5 (all of MES, MLI, MSC, MSP, and MTE) were 562, 195, 28, 7, and 6, respectively (808 in total). Of the 562 contigs, 29 were found in MES, 31 in MLI, 28 in MSC, 87 in MSP, and 387 in MTE (562 in total). It appeared that the genes expressed in the marmoset testes (387/562 = 69%) had an increased likelihood of not being expressed in humans compared with the genes expressed in other tissues (31%/4 = 7.8%).</p><p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a> shows the 20 contigs whose constituting ESTs’ numbers were the largest among the 808 contigs. Interestingly, the five contigs that had the highest numbers of ESTs (ID: 198, 15591, 1258, 1567, and 3721) also had ESTs in all five libraries. In the previous paragraph, it was noted that six contigs were expressed in all five libraries. It was found that five out of these six contigs were widely expressed at a high level in the marmoset. This suggests that actively expressed genes in the marmoset may even be non-transcribed as genes in humans. Many of the remaining genes in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a> were expressed in a combination of tissues e.g. MLI and MSP, or MLI and MSC, or only in MTE.</p>                            <h4 scrollto-destination=5353494 id="5353494" class="section-title" data-legacy-id=s3c4>3.3.4. Characterization of 808 contigs in non-coding RNA or silent regions of the human genome</h4>
<p>The 808 contigs described above were mapped onto the human genome, but did not have any corresponding human Refseqs. The human genome browser at UCSC also supports mapping of RNA-seq and large intergenic non-coding RNAs (linc RNA).<sup><a reveal-id="DST007C31" data-open="DST007C31" class="link link-ref link-reveal xref-bibr">31</a>,<a reveal-id="DST007C32" data-open="DST007C32" class="link link-ref link-reveal xref-bibr">32</a></sup> The locations of these RNA-seq/linc RNAs were checked in relation to the marmoset ESTs mapped onto the human genome. The tissues that are used in common in RNA-seq/linc RNA studies<sup><a reveal-id="DST007C31" data-open="DST007C31" class="link link-ref link-reveal xref-bibr">31</a>,<a reveal-id="DST007C32" data-open="DST007C32" class="link link-ref link-reveal xref-bibr">32</a></sup> and the current study are the liver, brain, and testis; hence, 138 contigs expressed in MLI, 125 in MSC, and 516 in MTE (total 779) were used for the analyses (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). This showed that 500 contigs (64% of 779) matched RNA-seqs, while 98 contigs had corresponding sequences in RNA-seq and linc RNA. However, 259 contigs (33% of 779) did not have any counterparts in RNA-seq or linc RNA, suggesting that they were not expressed as RNA.</p><p>In summary, of the 808 contigs that did not have a corresponding human Refseq, two-thirds were probably expressed as non-coding RNA, whereas the other third appeared to be silent. This potentially suggests that these 808 may have lost their characteristics as genes during human evolution.</p>                            <h4 scrollto-destination=5353497 id="5353497" class="section-title" data-legacy-id=s3c5>3.3.5. Identification of so far unknown genes based on probable full-length cDNA sequences</h4>
<p>The degree of conservation and/or divergence in marmoset and human genes was analysed using the following approach as well. <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>A shows a flow chart of the method used for selecting ‘unknown genes.’ First, getorf in EMBOSS was applied to extract 60 568 unigenes (see Table <a reveal-id="DST007TB1" data-open="DST007TB1" class="link link-reveal link-table xref-fig">1</a> where 60 568 represents the sum of contigs and singlets). Thus, sequences were selected that contain an open-reading frame spanning an initiating methionine through to a stop codon, with a poly(A) tail at the 3′ end. The G-cap was preserved in most ESTs, so the selected sequences probably represented ‘full-length’ cDNAs. This approach selected 3151 unigenes.</p><p>Each unigene was mapped onto the human genome. In total, 2595 sequences were mapped onto the human genome. Of the 556 marmoset sequences that could not be mapped onto the human genome, 127 were annotated and identified using the Refseq information of all living species, while 311 (306 + 5) shared homology with sequences registered in EST/nr sequence databases. The remaining 118 unigenes could not be annotated, suggesting that they represent potentially unknown genes.</p><p>Notably, only 1 sequence was derived from MLI and MSC, 3 from MSP, 14 from MES, while 104 from MTE. This might indicate comparatively abundant expression of unknown genes in the marmoset testis. The length of the polypeptides encoded by the ‘unknown genes’ ranged from 11 to 131 amino acids, with an average of 47 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>). It should be noted that no known domains/motifs were detected in these ‘unknown’ amino acid sequences using the INTERPRO program.</p>                            <h4 scrollto-destination=5353501 id="5353501" class="section-title" data-legacy-id=s3c6>3.3.6. Further characterization of the 118 ‘unknown genes’</h4>
<p>It is possible that the 118 genes with ORFs might not encode polypeptides but may represent non-coding RNAs. In fact, five shared homology (1e-5) with non-coding RNAs in a comprehensive database (<a class="link link-uri" href="http://www.ncrna.org/frnadb/">http://www.ncrna.org/frnadb/</a>) and a marmoset-derived non-coding RNA database (<a class="link link-uri" href="http://ftp://ftp.ensembl.org/pub/release70/fasta/callithrix_jacchus/ncrna/">ftp://ftp.ensembl.org/pub/release70/fasta/callithrix_jacchus/ncrna/</a>). Furthermore, 66 out of 118 genes could be aligned with the human genome if a 75% sequence similarity was employed, and each of these alignments covered more than half of each sequence. Therefore, a significant portion of the 118 genes had features of both polypeptide-coding genes and non-coding RNAs. <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>B and its legend provide further details on the unbiased approach used to identify unknown genes.</p>                            <h3 scrollto-destination=5353503 id="5353503" class="section-title" data-legacy-id=s3d>3.4. Availability of the resources</h3>
<p>A Web server of marmoset cDNAs has been constructed (<a class="link link-uri" href="http://marmoset.nig.ac.jp/index.html">http://marmoset.nig.ac.jp/index.html</a>), in which each marmoset EST sequence is mapped onto the human genome. Information accessible via the human genome browser can be obtained on this server, including the Ensembl gene annotation, Refseqs, and RNA-seq/linc RNA. This Web server can be used as a search engine, so the marmoset EST sequences can be BLASTed and the results displayed. The cDNAs libraries and/or the clones are available upon request from the DNA Bank, RIKEN BioResource Center (RDB no. 6388–6392). These deposited resources are expected to be valuable for future studies that use the common marmoset as an experimental animal model.</p>                            <h2 scrollto-destination=5353505 id="5353505" class="section-title" data-legacy-id=s4>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data</a> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5353507 id="5353507" class="section-title" data-legacy-id=s5>Funding</h2>
<p>Sequencing work was supported by a Special Fund for RIKEN Genomic Sciences Center.</p>                            <h2 scrollto-destination=5353509 id="5353509" class="backacknowledgements-title" >Acknowledgements</h2>
<p>S.T. and M.S. are very grateful to Atsushi Toyoda (National Institute of Genetics, Mishima, Japan) for his continuous encouragement, which helped us to accomplish this work. We thank members of Sequence Technology Team, RIKEN Genomic Sciences Center for the generation of marmoset ESTs.</p>                            <h2 scrollto-destination=5353511 id="5353511" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DST007C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST007C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mansfield</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Marmoset models commonly used in biomedical research</div>, <div class="source">Comp. Med.</div> , <div class="year">2003</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">383</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Marmoset%20models%20commonly%20used%20in%20biomedical%20research&amp;author=K.%20Mansfield&amp;publication_year=2003&amp;journal=Comp.%20Med.&amp;volume=53&amp;pages=383-92" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14524414" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST007C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST007C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Massacesi</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Genain</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Lee-Parritz</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">Active and passively induced experimental autoimmune encephalomyelitis in common marmosets: a new model for multiple sclerosis</div>, <div class="source">Ann. Neurol.</div> , <div class="year">1995</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">519</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Active%20and%20passively%20induced%20experimental%20autoimmune%20encephalomyelitis%20in%20common%20marmosets%3A%20a%20new%20model%20for%20multiple%20sclerosis&amp;author=L.%20Massacesi&amp;author=C.%20Genain&amp;author=D.%20Lee-Parritz&amp;publication_year=1995&amp;journal=Ann.%20Neurol.&amp;volume=37&amp;pages=519-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1531-8249" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1531-8249" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1531-8249"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7717689" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST007C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST007C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Genain</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Hauser</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Experimental allergic encephalomyelitis in the new world monkey Callithrix jacchus</div>, <div class="source">Immunol. Rev.</div> , <div class="year">2001</div>, vol. <div class="volume">183</div> (pg. <div class="fpage">159</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Experimental%20allergic%20encephalomyelitis%20in%20the%20new%20world%20monkey%20Callithrix%20jacchus&amp;author=C.%20Genain&amp;author=S.%20Hauser&amp;publication_year=2001&amp;journal=Immunol.%20Rev.&amp;volume=183&amp;pages=159-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1034/j.1600-065x.2001.1830113.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1034%2Fj.1600-065x.2001.1830113.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1034%2Fj.1600-065x.2001.1830113.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11782255" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST007C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST007C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">'t Hart</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Laman</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bauer</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Modeling of multiple sclerosis: lessons learned in a non-human primate</div>, <div class="source">Lancet Neurol.</div> , <div class="year">2004</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">588</div>-<div class="lpage">97</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Modeling%20of%20multiple%20sclerosis%3A%20lessons%20learned%20in%20a%20non-human%20primate&amp;author=B.%20't%20Hart&amp;author=J.%20Laman&amp;author=J.%20Bauer&amp;publication_year=2004&amp;journal=Lancet%20Neurol.&amp;volume=3&amp;pages=588-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1474-4422(04)00879-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1474-4422(04)00879-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1474-4422(04)00879-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15380155" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST007C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST007C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sasaki</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Suemizu</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Shimada</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Generation of transgenic nonhuman primates with germline transmission</div>, <div class="source">Nature</div> , <div class="year">2009</div>, vol. <div class="volume">459</div> (pg. <div class="fpage">515</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Generation%20of%20transgenic%20nonhuman%20primates%20with%20germline%20transmission&amp;author=E.%20Sasaki&amp;author=H.%20Suemizu&amp;author=A.%20Shimada&amp;publication_year=2009&amp;journal=Nature&amp;volume=459&amp;pages=515-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08090" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08090" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08090"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19478771" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST007C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST007C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Izawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Tani</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nakazaki</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Hematopoietic activity of common marmoset CD34 cells isolated by a novel monoclonal antibody MA24</div>, <div class="source">Exp. Hematol.</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">843</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hematopoietic%20activity%20of%20common%20marmoset%20CD34%20cells%20isolated%20by%20a%20novel%20monoclonal%20antibody%20MA24&amp;author=K.%20Izawa&amp;author=K.%20Tani&amp;author=Y.%20Nakazaki&amp;publication_year=2004&amp;journal=Exp.%20Hematol.&amp;volume=32&amp;pages=843-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.exphem.2004.06.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.exphem.2004.06.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.exphem.2004.06.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15345286" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST007C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST007C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ito</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Maekawa</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kawai</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Novel monoclonal antibodies recognizing different subsets of lymphocytes from the common marmoset (Callithrix jacchus)</div>, <div class="source">Immunol. Lett.</div> , <div class="year">2008</div>, vol. <div class="volume">121</div> (pg. <div class="fpage">116</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20monoclonal%20antibodies%20recognizing%20different%20subsets%20of%20lymphocytes%20from%20the%20common%20marmoset%20(Callithrix%20jacchus)&amp;author=R.%20Ito&amp;author=S.%20Maekawa&amp;author=K.%20Kawai&amp;publication_year=2008&amp;journal=Immunol.%20Lett.&amp;volume=121&amp;pages=116-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.imlet.2008.09.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.imlet.2008.09.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.imlet.2008.09.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18977390" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST007C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST007C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kametani</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Kohu</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Development of monoclonal antibodies for analyzing immune and hematopoietic systems of common marmoset</div>, <div class="source">Exp. Hematol.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">1318</div>-<div class="lpage">29</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20monoclonal%20antibodies%20for%20analyzing%20immune%20and%20hematopoietic%20systems%20of%20common%20marmoset&amp;author=Y.%20Kametani&amp;author=D.%20Suzuki&amp;author=K.%20Kohu&amp;publication_year=2009&amp;journal=Exp.%20Hematol.&amp;volume=37&amp;pages=1318-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.exphem.2009.08.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.exphem.2009.08.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.exphem.2009.08.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19715740" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST007C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST007C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brok</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Hornby</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Griffiths</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">An extensive monoclonal antibody panel for the phenotyping of leukocyte subsets in the common marmoset and the cotton-top tamarin</div>, <div class="source">Cytometry</div> , <div class="year">2001</div>, vol. <div class="volume">45</div> (pg. <div class="fpage">294</div>-<div class="lpage">303</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20extensive%20monoclonal%20antibody%20panel%20for%20the%20phenotyping%20of%20leukocyte%20subsets%20in%20the%20common%20marmoset%20and%20the%20cotton-top%20tamarin&amp;author=H.%20Brok&amp;author=R.%20Hornby&amp;author=G.%20Griffiths&amp;publication_year=2001&amp;journal=Cytometry&amp;volume=45&amp;pages=294-303" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0320" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0320" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0320"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11746099" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST007C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST007C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kohu</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yamabe</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Matsuzawa</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Comparison of 30 immunity-related genes from the common marmoset with orthologues from human and mouse</div>, <div class="source">Tohoku J. Exp. Med.</div> , <div class="year">2008</div>, vol. <div class="volume">215</div> (pg. <div class="fpage">167</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%2030%20immunity-related%20genes%20from%20the%20common%20marmoset%20with%20orthologues%20from%20human%20and%20mouse&amp;author=K.%20Kohu&amp;author=E.%20Yamabe&amp;author=A.%20Matsuzawa&amp;publication_year=2008&amp;journal=Tohoku%20J.%20Exp.%20Med.&amp;volume=215&amp;pages=167-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1620/tjem.215.167" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1620%2Ftjem.215.167" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1620%2Ftjem.215.167"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18577846" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST007C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST007C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sasaki</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Hanazawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kurita</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Establishment of novel embryonic stem cell lines derived from the common marmoset (<em>Callithrix jacchus</em>)</div>, <div class="source">Stem Cells</div> , <div class="year">2005</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">1304</div>-<div class="lpage">13</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Establishment%20of%20novel%20embryonic%20stem%20cell%20lines%20derived%20from%20the%20common%20marmoset%20(Callithrix%20jacchus)&amp;author=E.%20Sasaki&amp;author=K.%20Hanazawa&amp;author=R.%20Kurita&amp;publication_year=2005&amp;journal=Stem%20Cells&amp;volume=23&amp;pages=1304-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1634/stemcells.2004-0366" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1634%2Fstemcells.2004-0366" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1634%2Fstemcells.2004-0366"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16109758" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST007C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST007C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohtake</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ohtoko</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ishimaru</div> <div class="given-names">Y.,</div></div>,  <div class="name"><div class="surname">Kato</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Determination of the capped site sequence of mRNA based on the detection of cap-dependent nucleotide addition using an anchor ligation method</div>, <div class="source">DNA Res.</div> , <div class="year">2004</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">305</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Determination%20of%20the%20capped%20site%20sequence%20of%20mRNA%20based%20on%20the%20detection%20of%20cap-dependent%20nucleotide%20addition%20using%20an%20anchor%20ligation%20method&amp;author=H.%20Ohtake&amp;author=K.%20Ohtoko&amp;author=Y.%2C%20Ishimaru&amp;author=S.%20Kato&amp;publication_year=2004&amp;journal=DNA%20Res.&amp;volume=11&amp;pages=305-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/11.4.305" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F11.4.305" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F11.4.305"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15500255" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST007C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST007C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Gish</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Miller</div> <div class="given-names">W.</div></div>, et al. </div><div class="article-title">Basic local alignment search tool</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1990</div>, vol. <div class="volume">215</div> (pg. <div class="fpage">403</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=S.F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol.&amp;volume=215&amp;pages=403-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST007C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST007C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kent</div> <div class="given-names">W.J.</div></div>. </div><div class="article-title">BLAT—the BLAST-like alignment tool</div>, <div class="source">Genome Res.</div> , <div class="year">2002</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">656</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BLAT%E2%80%94the%20BLAST-like%20alignment%20tool&amp;author=W.J.%20Kent&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=656-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.229202. Article published online before March 2002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.229202. Article published online before March 2002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.229202. Article published online before March 2002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11932250" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST007C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST007C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Godzik</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences</div>, <div class="source">Bioinformatics</div> , <div class="year">2006</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">1658</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cd-hit%3A%20a%20fast%20program%20for%20clustering%20and%20comparing%20large%20sets%20of%20protein%20or%20nucleotide%20sequences&amp;author=W.%20Li&amp;author=A.%20Godzik&amp;publication_year=2006&amp;journal=Bioinformatics&amp;volume=22&amp;pages=1658-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btl158" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtl158" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtl158"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16731699" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST007C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST007C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Madan</div> <div class="given-names">A.</div></div>. </div><div class="article-title">CAP3: a DNA sequence assembly program</div>, <div class="source">Genome. Res.</div> , <div class="year">1999</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">868</div>-<div class="lpage">77</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CAP3%3A%20a%20DNA%20sequence%20assembly%20program&amp;author=X.%20Huang&amp;author=A.%20Madan&amp;publication_year=1999&amp;journal=Genome.%20Res.&amp;volume=9&amp;pages=868-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.9.9.868" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.9.9.868" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.9.9.868"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10508846" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST007C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST007C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ewing</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Green</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Base-calling of automated sequencer traces using phred. II. Error probabilities</div>, <div class="source">Genome. Res.</div> , <div class="year">1998</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">186</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Base-calling%20of%20automated%20sequencer%20traces%20using%20phred.%20II.%20Error%20probabilities&amp;author=B.%20Ewing&amp;author=P.%20Green&amp;publication_year=1998&amp;journal=Genome.%20Res.&amp;volume=8&amp;pages=186-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.8.3.186" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.8.3.186" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.8.3.186"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9521922" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST007C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST007C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rice</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Longden</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Bleasby</div> <div class="given-names">A.</div></div>. </div><div class="article-title">EMBOSS: the European Molecular Biology Open Software Suite</div>, <div class="source">Trends in Genet.</div> , <div class="year">2000</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">276</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EMBOSS%3A%20the%20European%20Molecular%20Biology%20Open%20Software%20Suite&amp;author=P.%20Rice&amp;author=I.%20Longden&amp;author=A.%20Bleasby&amp;publication_year=2000&amp;journal=Trends%20in%20Genet.&amp;volume=16&amp;pages=276-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(00)02024-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(00)02024-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(00)02024-2"> </span></div> </div></div></div></div></div><div content-id="DST007C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST007C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Quevillon</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Silventoinen</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Pillai</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">InterProScan: protein domains identifier</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">116</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=InterProScan%3A%20protein%20domains%20identifier&amp;author=E.%20Quevillon&amp;author=V.%20Silventoinen&amp;author=S.%20Pillai&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=116-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki442" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki442" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki442"> </span></div> </div></div></div></div></div><div content-id="DST007C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST007C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sakate</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Osada</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Hida</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Analysis of 5′-end sequences of chimpanzee cDNAs</div>, <div class="source">Genome. Res.</div> , <div class="year">2003</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">1022</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%205%E2%80%B2-end%20sequences%20of%20chimpanzee%20cDNAs&amp;author=R.%20Sakate&amp;author=N.%20Osada&amp;author=M.%20Hida&amp;publication_year=2003&amp;journal=Genome.%20Res.&amp;volume=13&amp;pages=1022-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.783103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.783103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.783103"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12727913" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST007C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST007C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">W.H.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">X.X.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">W.</div></div>, et al. </div><div class="article-title">Analysis of 10,000 ESTs from lymphocytes of the cynomolgus monkey to improve our understanding of its immune system</div>, <div class="source">BMC Genomics</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">82</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%2010%2C000%20ESTs%20from%20lymphocytes%20of%20the%20cynomolgus%20monkey%20to%20improve%20our%20understanding%20of%20its%20immune%20system&amp;author=W.H.%20Chen&amp;author=X.X.%20Wang&amp;author=W.%20Lin&amp;publication_year=2006&amp;journal=BMC%20Genomics&amp;volume=7&amp;pages=82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-7-82" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-7-82" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-7-82"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16618371" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST007C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST007C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Uno</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Wakaguri</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Expressed sequence tags from cynomolgus monkey (<em>Macaca fascicularis</em>) liver: a systematic identification of drug-metabolizing enzymes</div>, <div class="source">FEBS Lett.</div> , <div class="year">2008</div>, vol. <div class="volume">582</div> (pg. <div class="fpage">351</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expressed%20sequence%20tags%20from%20cynomolgus%20monkey%20(Macaca%20fascicularis)%20liver%3A%20a%20systematic%20identification%20of%20drug-metabolizing%20enzymes&amp;author=Y.%20Uno&amp;author=Y.%20Suzuki&amp;author=H.%20Wakaguri&amp;publication_year=2008&amp;journal=FEBS%20Lett.&amp;volume=582&amp;pages=351-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.febslet.2007.12.031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.febslet.2007.12.031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.febslet.2007.12.031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18166157" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST007C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST007C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Osada</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Hirata</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tanuma</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Collection of <em>Macaca fascicularis</em> cDNAs derived from bone marrow, kidney, liver, pancreas, spleen, and thymus</div>, <div class="source">BMC Res. Notes</div> , <div class="year">2009</div>, vol. <div class="volume">2</div> pg. <div class="fpage">199</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Collection%20of%20Macaca%20fascicularis%20cDNAs%20derived%20from%20bone%20marrow%2C%20kidney%2C%20liver%2C%20pancreas%2C%20spleen%2C%20and%20thymus&amp;author=N.%20Osada&amp;author=M.%20Hirata&amp;author=R.%20Tanuma&amp;publication_year=2009&amp;journal=BMC%20Res.%20Notes&amp;volume=2&amp;pages=199" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1756-0500-2-199" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1756-0500-2-199" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1756-0500-2-199"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19785770" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST007C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST007C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tchitchek</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Jacquelin</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Winker</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Expression sequence tag library derived from peripheral blood mononuclear cells of the chlorocebus sabaeus</div>, <div class="source">BMC Genomics</div> , <div class="year">2012</div>, vol. <div class="volume">13</div> pg. <div class="fpage">279</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20sequence%20tag%20library%20derived%20from%20peripheral%20blood%20mononuclear%20cells%20of%20the%20chlorocebus%20sabaeus&amp;author=N.%20Tchitchek&amp;author=B.%20Jacquelin&amp;author=P.%20Winker&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=279" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-279" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-279" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-279"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22726727" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST007C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST007C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Vodyanik</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Smuga-Otto</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Induced pluripotent stem cell lines derived from human somatic cells</div>, <div class="source">Science</div> , <div class="year">2007</div>, vol. <div class="volume">318</div> (pg. <div class="fpage">1917</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Induced%20pluripotent%20stem%20cell%20lines%20derived%20from%20human%20somatic%20cells&amp;author=J.%20Yu&amp;author=M.A.%20Vodyanik&amp;author=K.%20Smuga-Otto&amp;publication_year=2007&amp;journal=Science&amp;volume=318&amp;pages=1917-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1151526" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1151526" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1151526"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18029452" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST007C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST007C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mitsui</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Tokuzawa</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Itoh</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">The homeoprotein Nanog is required for maintenance of pluripotency in mouse epiblast and ES cells</div>, <div class="source">Cell</div> , <div class="year">2003</div>, vol. <div class="volume">113</div> (pg. <div class="fpage">631</div>-<div class="lpage">42</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20homeoprotein%20Nanog%20is%20required%20for%20maintenance%20of%20pluripotency%20in%20mouse%20epiblast%20and%20ES%20cells&amp;author=K.%20Mitsui&amp;author=Y.%20Tokuzawa&amp;author=H.%20Itoh&amp;publication_year=2003&amp;journal=Cell&amp;volume=113&amp;pages=631-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0092-8674(03)00393-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0092-8674(03)00393-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0092-8674(03)00393-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12787504" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST007C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST007C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Darr</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Benvenisty</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Genetic analysis of the role of the reprogramming gene <em>LIN-28</em> in human embryonic stem cells</div>, <div class="source">Stem Cells</div> , <div class="year">2009</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">352</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20analysis%20of%20the%20role%20of%20the%20reprogramming%20gene%20LIN-28%20in%20human%20embryonic%20stem%20cells&amp;author=H.%20Darr&amp;author=N.%20Benvenisty&amp;publication_year=2009&amp;journal=Stem%20Cells&amp;volume=27&amp;pages=352-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1634/stemcells.2008-0720" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1634%2Fstemcells.2008-0720" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1634%2Fstemcells.2008-0720"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19038789" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST007C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST007C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xu</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Huang</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Lin28 modulates cell growth and associates with a subset of cell cycle regulator mRNAs in mouse embryonic stem cells</div>, <div class="source">RNA</div> , <div class="year">2009</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">357</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lin28%20modulates%20cell%20growth%20and%20associates%20with%20a%20subset%20of%20cell%20cycle%20regulator%20mRNAs%20in%20mouse%20embryonic%20stem%20cells&amp;author=B.%20Xu&amp;author=K.%20Zhang&amp;author=Y.%20Huang&amp;publication_year=2009&amp;journal=RNA&amp;volume=15&amp;pages=357-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.1368009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.1368009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.1368009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19147696" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST007C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST007C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kim</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Suh</div> <div class="given-names">M.R.</div></div>,  <div class="name"><div class="surname">Yoon</div> <div class="given-names">H.S.</div></div>, et al. </div><div class="article-title">Identification of developmental pluripotency associated 5 expression in human pluripotent stem cells</div>, <div class="source">Stem Cells</div> , <div class="year">2005</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">458</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20developmental%20pluripotency%20associated%205%20expression%20in%20human%20pluripotent%20stem%20cells&amp;author=S.K.%20Kim&amp;author=M.R.%20Suh&amp;author=H.S.%20Yoon&amp;publication_year=2005&amp;journal=Stem%20Cells&amp;volume=23&amp;pages=458-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1634/stemcells.2004-0245" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1634%2Fstemcells.2004-0245" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1634%2Fstemcells.2004-0245"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15790765" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST007C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST007C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pierre</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Gautier</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Callebaut</div> <div class="given-names">I.</div></div>, et al. </div><div class="article-title">Atypical structure and phylogenomic evolution of the new eutherian oocyte- and embryo-expressed <em>KHDC1/DPPA5/ECAT1/OOEP</em> gene family</div>, <div class="source">Genomics</div> , <div class="year">2007</div>, vol. <div class="volume">90</div> (pg. <div class="fpage">583</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Atypical%20structure%20and%20phylogenomic%20evolution%20of%20the%20new%20eutherian%20oocyte-%20and%20embryo-expressed%20KHDC1%2FDPPA5%2FECAT1%2FOOEP%20gene%20family&amp;author=A.%20Pierre&amp;author=M.%20Gautier&amp;author=I.%20Callebaut&amp;publication_year=2007&amp;journal=Genomics&amp;volume=90&amp;pages=583-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ygeno.2007.06.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ygeno.2007.06.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ygeno.2007.06.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17913455" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST007C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST007C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">E.T.</div></div>,  <div class="name"><div class="surname">Sandberg</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Luo</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Alternative isoform regulation in human tissue transcriptomes</div>, <div class="source">Nature</div> , <div class="year">2008</div>, vol. <div class="volume">456</div> (pg. <div class="fpage">470</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20isoform%20regulation%20in%20human%20tissue%20transcriptomes&amp;author=E.T.%20Wang&amp;author=R.%20Sandberg&amp;author=S.%20Luo&amp;publication_year=2008&amp;journal=Nature&amp;volume=456&amp;pages=470-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature07509" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature07509" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature07509"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18978772" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST007C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST007C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cabili</div> <div class="given-names">M.N.</div></div>,  <div class="name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Goff</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">Integrative annotation of human large intergenic noncoding RNAs reveals global properties and specific subclasses</div>, <div class="source">Genes Dev.</div> , <div class="year">2011</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">1915</div>-<div class="lpage">27</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrative%20annotation%20of%20human%20large%20intergenic%20noncoding%20RNAs%20reveals%20global%20properties%20and%20specific%20subclasses&amp;author=M.N.%20Cabili&amp;author=C.%20Trapnell&amp;author=L.%20Goff&amp;publication_year=2011&amp;journal=Genes%20Dev.&amp;volume=25&amp;pages=1915-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.17446611" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.17446611" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.17446611"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21890647" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Dr Toshihiko Shiroishi</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by-nc/3.0/), which permits non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=368679&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst007/2/dst007-dst007supp_tableS1-S5.xlsx?Expires=2147483647&amp;Signature=BXh8X~-HxrL3N90ZhQy~0I~2tk3mWRBHrpSW-8IPMUiOOIdb-n~j4sKrUSwYbFZNGsDU8EqbSjeL7EkH3QZEvKIaaI19TuaVDYMXKSUMi2Od10wkL9y04K3E3ECbr852GKgiFzu9iKX9d~w~TG3lvO-gs5xrs6dY7Br8mMI5eYkOvuUyOed6CIGyECp2Gv1BAgPNxcgX-R1jFJ4CVpwNpRnuNxEF0NllU3zih-HcSz~PdOH-7n0NbzdRCweTEcdvINh5ep3M0nvKN2Fzf4nDJfeWrvCZSyQsrVpfzDKiVKsICN04Ex9F4PQDeZnRiMcs0smaAUCrwXWNZk5lJaSgBw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst007/2/dst007-dst007supp_fig2-4.pptx?Expires=2147483647&amp;Signature=KBSVAUzTRlM1gWdYDHHsu2lfcqEV3xNHgNwL0gmKgj3aKWTO7xSPeYyhDLkJj4N5h43lzwGHf2SSa737jGa07bJMJXuqVxXKVVUEsc43Io-B1E6HtnKifg765CvlSvIdxgU3MLIyEZGx17n9mW50Taa--rL6CK~viFJ7lwXARG2LvORmhYcn1Ncv0zSA9cJ3gHEm4-R4wEcklh0TKSMbrd5gjOXVFGh1bNJrZbNd6t~KGCKdmlKZzBNdPsVhp4tImRe7Sl8wR4HZX9HOHV4q6tYEqqCOsw7QSvBnPw42mGyCf82hpbF9OaaDydr6ZKb8nTVD8CdN6GJI2mLemT6I7w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - pptx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst007/2/dst007-dst007supp_fig1.pptx?Expires=2147483647&amp;Signature=2~5~f4lC3eEWOCHqWYGjunlhqtPUTrAAh9QFKOJ~hxQ3u6w7AOCv0NExADQY~WeZEo7OKFm5IHVXT0GfyBqiiAxaf6MGRg3~Z6lF6eZY6VazkBnN~SiuinK2SmUd4viGZPavOlwPz70nUfZ7k~Y9Q6yO49Q8GmyRCZFztEPTo5SQe4qK7g59dTeQYgxl1C0fQhnUDYsM7482xuEp7DtxwvtdgoTayJgckoSpWKSgIo5tbgx0OYVgfExWfmB8lNGc9VlnZbR5E~7jo-GVedJiAng6585BvJW9qhs8U0tRkNJDv~8Vz3puv3fs6Wf3BygJD9epJ4pI4heN8NbE620xRQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - pptx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst007/2/dst007-dst007supp.docx?Expires=2147483647&amp;Signature=XuzLjI0pRnZddNdU1Ff69HGEYgVDTNtdzPPSpoYyxdF9i41Cz0N2a16Glkh6gHEdd0g76NIKKmP9Y6seMhsmQs~sts1nsOx33VZMEPU7PGpbs4fQBY5vaqMrfCcwaVzBkEb-RmjdW7isqCaPgIWHuHEHVa843Q5duzbu6MRmHFqvAm6f~HGnv9txvhlem9F5wN8m8KDMlJYUQfEKNKiY2jCV3VxspAEiGNga2Ktmh6srYUEbamhTqZuk0lP7GCpcg-YtPf9Bjo5cI7m~OfgcErkbR0lc1cFcG0IlbdA56P2dAYnN2~lkl1owVuJxVHBJc-1SIhBL1znUf--GIEttBA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - docx file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">140</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000321404200005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">6</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dst007"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="368679"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000321404200005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f20%2f3%2f255%2f368679">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000321404200005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (6)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F20%2F3%2F255%2F368679" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/368679'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Development and Characterization of cDNA Resources for the Common Marmoset: One of the Experimental Primate Models" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dst007");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f20%2f3%2f255%2f368679">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

