

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/6/461/2462501","@type":"ScholarlyArticle","name":"New insights into the interplay between codon bias determinants in plants","datePublished":"2015-11-05","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2015-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsv027","keywords":["codon bias","mutational bias","translational selection","plant genetics"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/6/461/2462501","author":[{"name":"Camiolo, S.","affiliation":"Dipartimento di Agraria, SACEG ,  Università degli Studi di Sassari ,  Sassari ,  Italy","@type":"Person"},{"name":"Melito, S.","affiliation":"Dipartimento di Agraria, SACEG ,  Università degli Studi di Sassari ,  Sassari ,  Italy","@type":"Person"},{"name":"Porceddu, A.","affiliation":"Dipartimento di Agraria, SACEG ,  Università degli Studi di Sassari ,  Sassari ,  Italy","@type":"Person"}],"description":"Abstract.  Codon bias is the non-random use of synonymous codons, a phenomenon that has been observed in species as diverse as bacteria, plants and mammals. The","pageStart":"461","pageEnd":"470","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/6/1/m_cover.gif?Expires=2147483647&Signature=083aE4wPido26a6~EOoRT3Nb4fPpp5HV1MZzrjjay94a3evfhfRTvESRMhc7ROqyA2UIz2umFef89XL3Lu7hUZFod2Ix~QtkT4CP~8MDkYY61O0I1KLQQcYNpXVti4-0uwuVQWc0Ik7X46YCFzJAu2cTqwHZ2B0OBVFR-OZWKIwha86xHbqGB8R5L83t0zl8dqx1CkCPppI95mus6SUAdvZ0-wLyrVhsfEErb~4S7v8v0rjWvz1OZ1A2b6wWGyo1cjX3sRRxlgjkSlblmiowk2fs-R4agAPGw2bGiz2-0n1EnfsgKN8CSyFRqrRfdmccMFKJr5L5jh5C1s7L6OJB0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"New insights into the interplay between codon bias determinants in plants","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/6/1/m_cover.gif?Expires=2147483647&Signature=083aE4wPido26a6~EOoRT3Nb4fPpp5HV1MZzrjjay94a3evfhfRTvESRMhc7ROqyA2UIz2umFef89XL3Lu7hUZFod2Ix~QtkT4CP~8MDkYY61O0I1KLQQcYNpXVti4-0uwuVQWc0Ik7X46YCFzJAu2cTqwHZ2B0OBVFR-OZWKIwha86xHbqGB8R5L83t0zl8dqx1CkCPppI95mus6SUAdvZ0-wLyrVhsfEErb~4S7v8v0rjWvz1OZ1A2b6wWGyo1cjX3sRRxlgjkSlblmiowk2fs-R4agAPGw2bGiz2-0n1EnfsgKN8CSyFRqrRfdmccMFKJr5L5jh5C1s7L6OJB0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="New insights into the interplay between codon bias determinants in plants" />
    <meta property="og:description" content="Abstract.  Codon bias is the non-random use of synonymous codons, a phenomenon that has been observed in species as diverse as bacteria, plants and mammals. The" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/6/461/2462501" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2015-11-05" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Camiolo, S." /><meta name="citation_author_institution" content="Dipartimento di Agraria, SACEG, Università degli Studi di Sassari, Sassari, Italy" /><meta name="citation_author" content="Melito, S." /><meta name="citation_author_institution" content="Dipartimento di Agraria, SACEG, Università degli Studi di Sassari, Sassari, Italy" /><meta name="citation_author" content="Porceddu, A." /><meta name="citation_author_institution" content="Dipartimento di Agraria, SACEG, Università degli Studi di Sassari, Sassari, Italy" /><meta name="citation_title" content="New insights into the interplay between codon bias determinants in plants" /><meta name="citation_firstpage" content="461" /><meta name="citation_lastpage" content="470" /><meta name="citation_doi" content="10.1093/dnares/dsv027" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2015/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Hearing silence: non-neutral evolution at synonymous sites in mammals;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2006;  citation_volume=7;  citation_pages=98-108; " /><meta name="citation_reference" content="citation_title=Evolution of synonymous codon usage in metazoans;  citation_journal_title=Curr. Opin. Genet. Dev.;  citation_year=2002;  citation_volume=12;  citation_pages=640-9; " /><meta name="citation_reference" content="citation_title=Codon usage and selection on proteins;  citation_journal_title=J. Mol. Evol.;  citation_year=2006;  citation_volume=63;  citation_pages=635-53; " /><meta name="citation_reference" content="citation_title=Synonymous but not the same: the causes and consequences of codon bias;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2011;  citation_volume=12;  citation_pages=32-42; " /><meta name="citation_reference" content="citation_title=Correlation between the abundance of Escherichia coli transfer RNAs and the occurrence of the respective codons in its protein genes: a proposal for a synonymous codon choice that is optimal for the E. coli translational system;  citation_journal_title=J. Mol. Biol.;  citation_year=1981;  citation_volume=151;  citation_pages=389-409; " /><meta name="citation_reference" content="citation_title=Synonymous codon usage in Drosophila melanogaster: natural selection and translational accuracy;  citation_journal_title=Genetics;  citation_year=1994;  citation_volume=136;  citation_pages=927-35; " /><meta name="citation_reference" content="citation_title=The selection-mutation-drift theory of synonymous codon usage;  citation_journal_title=Genetics;  citation_year=1991;  citation_volume=129;  citation_pages=897-907; " /><meta name="citation_reference" content="citation_title=Codon usage patterns in Escherichia coli, Bacillus subtilis, Saccharomyces cerevisiae, Schizosaccharomyces pombe, Drosophila melanogaster and Homo sapiens; a review of the considerable within-species diversity;  citation_journal_title=Nucleic Acids Res.;  citation_year=1988;  citation_volume=16;  citation_pages=8207-11; " /><meta name="citation_reference" content="citation_title=Universal function-specificity of codon usage;  citation_journal_title=Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=7014-23; " /><meta name="citation_reference" content="citation_title=The relation of codon bias to tissue-specific gene expression in Arabidopsis thaliana;  citation_journal_title=Genetics;  citation_year=2012;  citation_volume=192;  citation_pages=641-9; " /><meta name="citation_reference" content="citation_title=Compositional gradients in Gramineae genes;  citation_journal_title=Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=851-6; " /><meta name="citation_reference" content="citation_title=The mosaic genome of warm-blooded vertebrates;  citation_journal_title=Science;  citation_year=1985;  citation_volume=228;  citation_pages=953-8; " /><meta name="citation_reference" content="citation_title=Two classes of genes in plants;  citation_journal_title=Genetics;  citation_year=2000;  citation_volume=154;  citation_pages=1819-25; " /><meta name="citation_reference" content="citation_title=Mutational bias and translational selection shaping the codon usage pattern of tissue-specific genes in rice;  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e48295" /><meta name="citation_reference" content="citation_title=Codon bias in Escherichia coli: the influence of codon context on mutation and selection;  citation_journal_title=Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=1397-404; " /><meta name="citation_reference" content="citation_title=Intercodon dinucleotides affect codon choice in plant genes;  citation_journal_title=Nucleic Acids Res.;  citation_year=2000;  citation_volume=28;  citation_pages=3339-45; " /><meta name="citation_reference" content="citation_title=What drives codon choices in human genes?;  citation_journal_title=J. Mol. Biol.;  citation_year=1996;  citation_volume=262;  citation_pages=459-72; " /><meta name="citation_reference" content="citation_title=Spatial analyses of mono, di and trinucleotide trends in plant genes;  citation_journal_title=PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e22855" /><meta name="citation_reference" content="citation_title=General rules for optimal codon choice;  citation_journal_title=PLoS Genet.;  citation_year=2009;  citation_volume=5;  citation_pages=e1000556" /><meta name="citation_reference" content="citation_title=Selective constraints on codon usage of nuclear genes from Arabidopsis thaliana;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2007;  citation_volume=24;  citation_pages=122-9; " /><meta name="citation_reference" content="citation_title=Phytozome: a comparative platform for green plant genomics;  citation_journal_title=Nucleic Acids Res.;  citation_year=2012;  citation_volume=40;  citation_pages=D1178-86; " /><meta name="citation_reference" content="citation_title=gff2sequence, a new user friendly tool for the generation of genomic sequences;  citation_journal_title=BioData Min.;  citation_year=2013;  citation_volume=6;  citation_pages=15" /><meta name="citation_reference" content="citation_title=PLEXdb: gene expression resources for plants and plant pathogens;  citation_journal_title=Nucleic Acids Res.;  citation_year=2012;  citation_volume=40;  citation_pages=D1194-201; " /><meta name="citation_reference" content="citation_title=Mutational biases and selective forces shaping the structure of Arabidopsis genes;  citation_journal_title=PLoS One;  citation_year=2009;  citation_volume=4;  citation_pages=e6356" /><meta name="citation_reference" content="citation_title=The “effective number of codons” used in a gene;  citation_journal_title=Gene;  citation_year=1990;  citation_volume=87;  citation_pages=23-9; " /><meta name="citation_reference" content="citation_title=An evolutionary perspective on synonymous codon usage in unicellular organisms;  citation_journal_title=J. Mol. Evol.;  citation_year=1986;  citation_volume=24;  citation_pages=28-38; " /><meta name="citation_reference" content="citation_title=An evolutionarily conserved mechanism for controlling the efficiency of protein translation;  citation_journal_title=Cell;  citation_year=2010;  citation_volume=141;  citation_pages=344-54; " /><meta name="citation_reference" content="citation_title=Evaluation of codon biology in citrus and Poncirus trifoliata based on genomic features and frame corrected expressed sequence tags;  citation_journal_title=DNA Res.;  citation_year=2013;  citation_volume=20;  citation_pages=135-50; " /><meta name="citation_reference" content="citation_title=September 21, The ribosomial ramp and the regulation of translation in plants;  citation_journal_title=J. Bioinforma. Res. Stud.;  citation_year=2014;  citation_volume=1;  citation_pages=2" /><meta name="citation_reference" content="citation_title=A general model of codon bias due to GC mutational bias;  citation_journal_title=PLoS One;  citation_year=2010;  citation_volume=5;  citation_pages=e13431" /><meta name="citation_reference" content="citation_title=Global dinucleotide signatures and analysis of genomic heterogeneity;  citation_journal_title=Curr. Opin. Microbiol.;  citation_year=1998;  citation_volume=1;  citation_pages=598-610; " /><meta name="citation_reference" content="citation_title=Assessment of compositional heterogeneity within and between eukaryotic genomes;  citation_journal_title=Genome Res.;  citation_year=2000;  citation_volume=10;  citation_pages=1986-95; " /><meta name="citation_reference" content="citation_title=Mutational bias is the driving force for shaping the synonymous codon usage pattern of alternatively spliced genes in rice (Oryza sativa L.);  citation_journal_title=Mol. Genet. Genomics;  citation_year=2015;  citation_volume=290;  citation_pages=649-60; " /><meta name="citation_reference" content="citation_title=Transcription-associated mutational asymmetry in mammalian evolution;  citation_journal_title=Nat. Genet.;  citation_year=2003;  citation_volume=33;  citation_pages=514-7; " /><meta name="citation_reference" content="citation_title=The rate and molecular spectrum of spontaneous mutations in Arabidopsis thaliana;  citation_journal_title=Science;  citation_year=2010;  citation_volume=327;  citation_pages=92-4; " /><meta name="citation_reference" content="citation_title=Effects of ultra-violet alone and simulated solar ultra-violet radiation on the leaves of higher plants;  citation_journal_title=Nature;  citation_year=1966;  citation_volume=211;  citation_pages=484-6; " /><meta name="citation_reference" content="citation_title=Mutational bias and gene conversion affect the intraspecific nitrogen stoichiometry of the Arabidopsis thaliana transcriptome;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2013;  citation_volume=30;  citation_pages=561-8; " /><meta name="citation_reference" content="citation_title=Structural rearrangements and insertions of dispersed elements in pericentromeric alpha satellites occur preferably at kinkable DNA sites;  citation_journal_title=J. Mol. Biol.;  citation_year=2001;  citation_volume=305;  citation_pages=33-48; " /><meta name="citation_reference" content="citation_title=Evolution of the genome and the genetic code: selection at the dinucleotide level by methylation and polyribonucleotide cleavage;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=1989;  citation_volume=86;  citation_pages=192-6; " /><meta name="citation_reference" content="citation_title=Reduced synonymous substitution rate at the start of enterobacterial genes;  citation_journal_title=Nucleic Acids Res.;  citation_year=1993;  citation_volume=21;  citation_pages=4599-603; " /><meta name="citation_reference" content="citation_title=A novel theory on the origin of the genetic code: a GNC-SNS hypothesis;  citation_journal_title=J. Mol. Evol.;  citation_year=2002;  citation_volume=54;  citation_pages=530-8; " /><meta name="citation_reference" content="citation_title=Effects of gene expression on molecular evolution in Arabidopsis thaliana and Arabidopsis lyrata;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2004;  citation_volume=21;  citation_pages=1719-26; " /><meta name="citation_reference" content="citation_title=The signatures of selection for translational accuracy in plant genes;  citation_journal_title=Genome Biol. Evol.;  citation_year=2013;  citation_volume=5;  citation_pages=1117-26; " /><meta name="citation_reference" content="citation_title=In plants, expression breadth and expression level distinctly and non-linearly correlate with gene structure;  citation_journal_title=Biol. Direct.;  citation_year=2009;  citation_volume=4;  citation_pages=45" /><meta name="citation_reference" content="citation_title=GC-content evolution in mammalian genomes: the biased gene conversion hypothesis;  citation_journal_title=Genetics;  citation_year=2001;  citation_volume=159;  citation_pages=907-11; " /><meta name="citation_reference" content="citation_title=Corn and humans: recombination and linkage disequilibrium in two genomes of similar size;  citation_journal_title=Trends Genet.;  citation_year=2004;  citation_volume=20;  citation_pages=103-11; " /><meta name="citation_reference" content="citation_title=Patterns and evolution of nucleotide landscapes in seed plants;  citation_journal_title=Plant Cell;  citation_year=2012;  citation_volume=24;  citation_pages=1379-97; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/6/461/17463342/dsv027.pdf" /><meta name="description" content="Abstract.  Codon bias is the non-random use of synonymous codons, a phenomenon that has been observed in species as diverse as bacteria, plants and mammals. The" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/6/461/2462501" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/6/461/2462501" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/22/6/461/2462501");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsv027';
    </script>
<title>
	New insights into the interplay between codon bias determinants in plants | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2462501" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bf9nPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDYW1pb2xvLCBTLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRpcGFydGltZW50byBkaSBBZ3JhcmlhLCBTQUNFRywgVW5pdmVyc2l0w6AgZGVnbGkgU3R1ZGkgZGkgU2Fzc2FyaSwgU2Fzc2FyaSwgSXRhbHkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNZWxpdG8sIFMuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGlwYXJ0aW1lbnRvIGRpIEFncmFyaWEsIFNBQ0VHLCBVbml2ZXJzaXTDoCBkZWdsaSBTdHVkaSBkaSBTYXNzYXJpLCBTYXNzYXJpLCBJdGFseSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlBvcmNlZGR1LCBBLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRpcGFydGltZW50byBkaSBBZ3JhcmlhLCBTQUNFRywgVW5pdmVyc2l0w6AgZGVnbGkgU3R1ZGkgZGkgU2Fzc2FyaSwgU2Fzc2FyaSwgSXRhbHkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9Ik5ldyBpbnNpZ2h0cyBpbnRvIHRoZSBpbnRlcnBsYXkgYmV0d2VlbiBjb2RvbiBiaWFzIGRldGVybWluYW50cyBpbiBwbGFudHMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI0NjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjQ3MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdjAyNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE1LzEyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGVhcmluZyBzaWxlbmNlOiBub24tbmV1dHJhbCBldm9sdXRpb24gYXQgc3lub255bW91cyBzaXRlcyBpbiBtYW1tYWxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz05OC0xMDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbiBvZiBzeW5vbnltb3VzIGNvZG9uIHVzYWdlIGluIG1ldGF6b2FuczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gT3Bpbi4gR2VuZXQuIERldi47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz02NDAtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gdXNhZ2UgYW5kIHNlbGVjdGlvbiBvbiBwcm90ZWluczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT02MzsgIGNpdGF0aW9uX3BhZ2VzPTYzNS01MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3lub255bW91cyBidXQgbm90IHRoZSBzYW1lOiB0aGUgY2F1c2VzIGFuZCBjb25zZXF1ZW5jZXMgb2YgY29kb24gYmlhczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTMyLTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db3JyZWxhdGlvbiBiZXR3ZWVuIHRoZSBhYnVuZGFuY2Ugb2YgRXNjaGVyaWNoaWEgY29saSB0cmFuc2ZlciBSTkFzIGFuZCB0aGUgb2NjdXJyZW5jZSBvZiB0aGUgcmVzcGVjdGl2ZSBjb2RvbnMgaW4gaXRzIHByb3RlaW4gZ2VuZXM6IGEgcHJvcG9zYWwgZm9yIGEgc3lub255bW91cyBjb2RvbiBjaG9pY2UgdGhhdCBpcyBvcHRpbWFsIGZvciB0aGUgRS4gY29saSB0cmFuc2xhdGlvbmFsIHN5c3RlbTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk4MTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTE7ICBjaXRhdGlvbl9wYWdlcz0zODktNDA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeW5vbnltb3VzIGNvZG9uIHVzYWdlIGluIERyb3NvcGhpbGEgbWVsYW5vZ2FzdGVyOiBuYXR1cmFsIHNlbGVjdGlvbiBhbmQgdHJhbnNsYXRpb25hbCBhY2N1cmFjeTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MTM2OyAgY2l0YXRpb25fcGFnZXM9OTI3LTM1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgc2VsZWN0aW9uLW11dGF0aW9uLWRyaWZ0IHRoZW9yeSBvZiBzeW5vbnltb3VzIGNvZG9uIHVzYWdlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MTk5MTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjk7ICBjaXRhdGlvbl9wYWdlcz04OTctOTA3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiB1c2FnZSBwYXR0ZXJucyBpbiBFc2NoZXJpY2hpYSBjb2xpLCBCYWNpbGx1cyBzdWJ0aWxpcywgU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlLCBTY2hpem9zYWNjaGFyb215Y2VzIHBvbWJlLCBEcm9zb3BoaWxhIG1lbGFub2dhc3RlciBhbmQgSG9tbyBzYXBpZW5zOyBhIHJldmlldyBvZiB0aGUgY29uc2lkZXJhYmxlIHdpdGhpbi1zcGVjaWVzIGRpdmVyc2l0eTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTg4OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9ODIwNy0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VW5pdmVyc2FsIGZ1bmN0aW9uLXNwZWNpZmljaXR5IG9mIGNvZG9uIHVzYWdlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz03MDE0LTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcmVsYXRpb24gb2YgY29kb24gYmlhcyB0byB0aXNzdWUtc3BlY2lmaWMgZ2VuZSBleHByZXNzaW9uIGluIEFyYWJpZG9wc2lzIHRoYWxpYW5hOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xOTI7ICBjaXRhdGlvbl9wYWdlcz02NDEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcG9zaXRpb25hbCBncmFkaWVudHMgaW4gR3JhbWluZWFlIGdlbmVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTg1MS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbW9zYWljIGdlbm9tZSBvZiB3YXJtLWJsb29kZWQgdmVydGVicmF0ZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTE5ODU7ICBjaXRhdGlvbl92b2x1bWU9MjI4OyAgY2l0YXRpb25fcGFnZXM9OTUzLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVR3byBjbGFzc2VzIG9mIGdlbmVzIGluIHBsYW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTU0OyAgY2l0YXRpb25fcGFnZXM9MTgxOS0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXV0YXRpb25hbCBiaWFzIGFuZCB0cmFuc2xhdGlvbmFsIHNlbGVjdGlvbiBzaGFwaW5nIHRoZSBjb2RvbiB1c2FnZSBwYXR0ZXJuIG9mIHRpc3N1ZS1zcGVjaWZpYyBnZW5lcyBpbiByaWNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTQ4Mjk1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gYmlhcyBpbiBFc2NoZXJpY2hpYSBjb2xpOiB0aGUgaW5mbHVlbmNlIG9mIGNvZG9uIGNvbnRleHQgb24gbXV0YXRpb24gYW5kIHNlbGVjdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MTM5Ny00MDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVyY29kb24gZGludWNsZW90aWRlcyBhZmZlY3QgY29kb24gY2hvaWNlIGluIHBsYW50IGdlbmVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz0zMzM5LTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaGF0IGRyaXZlcyBjb2RvbiBjaG9pY2VzIGluIGh1bWFuIGdlbmVzPzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0yNjI7ICBjaXRhdGlvbl9wYWdlcz00NTktNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNwYXRpYWwgYW5hbHlzZXMgb2YgbW9ubywgZGkgYW5kIHRyaW51Y2xlb3RpZGUgdHJlbmRzIGluIHBsYW50IGdlbmVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTIyODU1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXJhbCBydWxlcyBmb3Igb3B0aW1hbCBjb2RvbiBjaG9pY2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMDU1NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlbGVjdGl2ZSBjb25zdHJhaW50cyBvbiBjb2RvbiB1c2FnZSBvZiBudWNsZWFyIGdlbmVzIGZyb20gQXJhYmlkb3BzaXMgdGhhbGlhbmE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xMjItOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5dG96b21lOiBhIGNvbXBhcmF0aXZlIHBsYXRmb3JtIGZvciBncmVlbiBwbGFudCBnZW5vbWljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9RDExNzgtODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPWdmZjJzZXF1ZW5jZSwgYSBuZXcgdXNlciBmcmllbmRseSB0b29sIGZvciB0aGUgZ2VuZXJhdGlvbiBvZiBnZW5vbWljIHNlcXVlbmNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvRGF0YSBNaW4uOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0xNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBMRVhkYjogZ2VuZSBleHByZXNzaW9uIHJlc291cmNlcyBmb3IgcGxhbnRzIGFuZCBwbGFudCBwYXRob2dlbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPUQxMTk0LTIwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXV0YXRpb25hbCBiaWFzZXMgYW5kIHNlbGVjdGl2ZSBmb3JjZXMgc2hhcGluZyB0aGUgc3RydWN0dXJlIG9mIEFyYWJpZG9wc2lzIGdlbmVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9ZTYzNTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUg4oCcZWZmZWN0aXZlIG51bWJlciBvZiBjb2RvbnPigJ0gdXNlZCBpbiBhIGdlbmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmU7ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9ODc7ICBjaXRhdGlvbl9wYWdlcz0yMy05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBldm9sdXRpb25hcnkgcGVyc3BlY3RpdmUgb24gc3lub255bW91cyBjb2RvbiB1c2FnZSBpbiB1bmljZWxsdWxhciBvcmdhbmlzbXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIE1vbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODY7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0yOC0zODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gZXZvbHV0aW9uYXJpbHkgY29uc2VydmVkIG1lY2hhbmlzbSBmb3IgY29udHJvbGxpbmcgdGhlIGVmZmljaWVuY3kgb2YgcHJvdGVpbiB0cmFuc2xhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDE7ICBjaXRhdGlvbl9wYWdlcz0zNDQtNTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2YWx1YXRpb24gb2YgY29kb24gYmlvbG9neSBpbiBjaXRydXMgYW5kIFBvbmNpcnVzIHRyaWZvbGlhdGEgYmFzZWQgb24gZ2Vub21pYyBmZWF0dXJlcyBhbmQgZnJhbWUgY29ycmVjdGVkIGV4cHJlc3NlZCBzZXF1ZW5jZSB0YWdzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTEzNS01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VwdGVtYmVyIDIxLCBUaGUgcmlib3NvbWlhbCByYW1wIGFuZCB0aGUgcmVndWxhdGlvbiBvZiB0cmFuc2xhdGlvbiBpbiBwbGFudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEJpb2luZm9ybWEuIFJlcy4gU3R1ZC47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGdlbmVyYWwgbW9kZWwgb2YgY29kb24gYmlhcyBkdWUgdG8gR0MgbXV0YXRpb25hbCBiaWFzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9ZTEzNDMxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2xvYmFsIGRpbnVjbGVvdGlkZSBzaWduYXR1cmVzIGFuZCBhbmFseXNpcyBvZiBnZW5vbWljIGhldGVyb2dlbmVpdHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN1cnIuIE9waW4uIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTU5OC02MTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzc2Vzc21lbnQgb2YgY29tcG9zaXRpb25hbCBoZXRlcm9nZW5laXR5IHdpdGhpbiBhbmQgYmV0d2VlbiBldWthcnlvdGljIGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MTk4Ni05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXV0YXRpb25hbCBiaWFzIGlzIHRoZSBkcml2aW5nIGZvcmNlIGZvciBzaGFwaW5nIHRoZSBzeW5vbnltb3VzIGNvZG9uIHVzYWdlIHBhdHRlcm4gb2YgYWx0ZXJuYXRpdmVseSBzcGxpY2VkIGdlbmVzIGluIHJpY2UgKE9yeXphIHNhdGl2YSBMLik7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gR2VuZXQuIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTI5MDsgIGNpdGF0aW9uX3BhZ2VzPTY0OS02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdGlvbi1hc3NvY2lhdGVkIG11dGF0aW9uYWwgYXN5bW1ldHJ5IGluIG1hbW1hbGlhbiBldm9sdXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9NTE0LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSByYXRlIGFuZCBtb2xlY3VsYXIgc3BlY3RydW0gb2Ygc3BvbnRhbmVvdXMgbXV0YXRpb25zIGluIEFyYWJpZG9wc2lzIHRoYWxpYW5hOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTMyNzsgIGNpdGF0aW9uX3BhZ2VzPTkyLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVmZmVjdHMgb2YgdWx0cmEtdmlvbGV0IGFsb25lIGFuZCBzaW11bGF0ZWQgc29sYXIgdWx0cmEtdmlvbGV0IHJhZGlhdGlvbiBvbiB0aGUgbGVhdmVzIG9mIGhpZ2hlciBwbGFudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MTk2NjsgIGNpdGF0aW9uX3ZvbHVtZT0yMTE7ICBjaXRhdGlvbl9wYWdlcz00ODQtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXV0YXRpb25hbCBiaWFzIGFuZCBnZW5lIGNvbnZlcnNpb24gYWZmZWN0IHRoZSBpbnRyYXNwZWNpZmljIG5pdHJvZ2VuIHN0b2ljaGlvbWV0cnkgb2YgdGhlIEFyYWJpZG9wc2lzIHRoYWxpYW5hIHRyYW5zY3JpcHRvbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz01NjEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RydWN0dXJhbCByZWFycmFuZ2VtZW50cyBhbmQgaW5zZXJ0aW9ucyBvZiBkaXNwZXJzZWQgZWxlbWVudHMgaW4gcGVyaWNlbnRyb21lcmljIGFscGhhIHNhdGVsbGl0ZXMgb2NjdXIgcHJlZmVyYWJseSBhdCBraW5rYWJsZSBETkEgc2l0ZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MzA1OyAgY2l0YXRpb25fcGFnZXM9MzMtNDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbiBvZiB0aGUgZ2Vub21lIGFuZCB0aGUgZ2VuZXRpYyBjb2RlOiBzZWxlY3Rpb24gYXQgdGhlIGRpbnVjbGVvdGlkZSBsZXZlbCBieSBtZXRoeWxhdGlvbiBhbmQgcG9seXJpYm9udWNsZW90aWRlIGNsZWF2YWdlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTg5OyAgY2l0YXRpb25fdm9sdW1lPTg2OyAgY2l0YXRpb25fcGFnZXM9MTkyLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlZHVjZWQgc3lub255bW91cyBzdWJzdGl0dXRpb24gcmF0ZSBhdCB0aGUgc3RhcnQgb2YgZW50ZXJvYmFjdGVyaWFsIGdlbmVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz00NTk5LTYwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBub3ZlbCB0aGVvcnkgb24gdGhlIG9yaWdpbiBvZiB0aGUgZ2VuZXRpYyBjb2RlOiBhIEdOQy1TTlMgaHlwb3RoZXNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT01NDsgIGNpdGF0aW9uX3BhZ2VzPTUzMC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FZmZlY3RzIG9mIGdlbmUgZXhwcmVzc2lvbiBvbiBtb2xlY3VsYXIgZXZvbHV0aW9uIGluIEFyYWJpZG9wc2lzIHRoYWxpYW5hIGFuZCBBcmFiaWRvcHNpcyBseXJhdGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0xNzE5LTI2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgc2lnbmF0dXJlcyBvZiBzZWxlY3Rpb24gZm9yIHRyYW5zbGF0aW9uYWwgYWNjdXJhY3kgaW4gcGxhbnQgZ2VuZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MTExNy0yNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW4gcGxhbnRzLCBleHByZXNzaW9uIGJyZWFkdGggYW5kIGV4cHJlc3Npb24gbGV2ZWwgZGlzdGluY3RseSBhbmQgbm9uLWxpbmVhcmx5IGNvcnJlbGF0ZSB3aXRoIGdlbmUgc3RydWN0dXJlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9sLiBEaXJlY3QuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz00NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdDLWNvbnRlbnQgZXZvbHV0aW9uIGluIG1hbW1hbGlhbiBnZW5vbWVzOiB0aGUgYmlhc2VkIGdlbmUgY29udmVyc2lvbiBoeXBvdGhlc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTk7ICBjaXRhdGlvbl9wYWdlcz05MDctMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvcm4gYW5kIGh1bWFuczogcmVjb21iaW5hdGlvbiBhbmQgbGlua2FnZSBkaXNlcXVpbGlicml1bSBpbiB0d28gZ2Vub21lcyBvZiBzaW1pbGFyIHNpemU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0xMDMtMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBhdHRlcm5zIGFuZCBldm9sdXRpb24gb2YgbnVjbGVvdGlkZSBsYW5kc2NhcGVzIGluIHNlZWQgcGxhbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MTM3OS05NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIyLzYvNDYxLzE3NDYzMzQyL2RzdjAyNy5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgQ29kb24gYmlhcyBpcyB0aGUgbm9uLXJhbmRvbSB1c2Ugb2Ygc3lub255bW91cyBjb2RvbnMsIGEgcGhlbm9tZW5vbiB0aGF0IGhhcyBiZWVuIG9ic2VydmVkIGluIHNwZWNpZXMgYXMgZGl2ZXJzZSBhcyBiYWN0ZXJpYSwgcGxhbnRzIGFuZCBtYW1tYWxzLiBUaGUiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTI0NjI1MDEmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkJEogkYUXNfCALBmO5M1BdyA5fbM=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f6%2f461%2f2462501" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f6%2f461%2f2462501" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=83834" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/6/1/m_cover.gif?Expires=2147483647&amp;Signature=083aE4wPido26a6~EOoRT3Nb4fPpp5HV1MZzrjjay94a3evfhfRTvESRMhc7ROqyA2UIz2umFef89XL3Lu7hUZFod2Ix~QtkT4CP~8MDkYY61O0I1KLQQcYNpXVti4-0uwuVQWc0Ik7X46YCFzJAu2cTqwHZ2B0OBVFR-OZWKIwha86xHbqGB8R5L83t0zl8dqx1CkCPppI95mus6SUAdvZ0-wLyrVhsfEErb~4S7v8v0rjWvz1OZ1A2b6wWGyo1cjX3sRRxlgjkSlblmiowk2fs-R4agAPGw2bGiz2-0n1EnfsgKN8CSyFRqrRfdmccMFKJr5L5jh5C1s7L6OJB0g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/6">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87200886">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200886">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200888">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200888">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200909">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200909">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200945">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200945">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200969">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200969">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200989">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200989">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200991">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200991">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/22/6/451/2462493">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/22/6/471/2462531">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">New insights into the interplay between codon bias determinants in plants
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">S. Camiolo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Dipartimento di Agraria, SACEG</div>, <div class="insititution">Università degli Studi di Sassari</div>, <div class="addr-line">Sassari</div>, <div class="country">Italy</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=S.+Camiolo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Camiolo S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Camiolo S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">S. Camiolo</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">S. Melito</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Dipartimento di Agraria, SACEG</div>, <div class="insititution">Università degli Studi di Sassari</div>, <div class="addr-line">Sassari</div>, <div class="country">Italy</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=S.+Melito">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Melito S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Melito S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">S. Melito</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">A. Porceddu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Dipartimento di Agraria, SACEG</div>, <div class="insititution">Università degli Studi di Sassari</div>, <div class="addr-line">Sassari</div>, <div class="country">Italy</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +39 079 229224. Fax. +39 079 229222. E-mail: <a href="mailto:aporceddu@uniss.it" target="_blank">aporceddu@uniss.it</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=A.+Porceddu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Porceddu A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Porceddu A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">A. Porceddu</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="footnote-content">Edited by Dr Satoshi Tabata</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 6, 1 December 2015, Pages 461–470, <a href='https://doi.org/10.1093/dnares/dsv027'>https://doi.org/10.1093/dnares/dsv027</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">05 November 2015</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">30 June 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">01 October 2015</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2462501" href="/dnaresearch/article-pdf/22/6/461/17463342/dsv027.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>S. Camiolo, S. Melito, A. Porceddu;  New insights into the interplay between codon bias determinants in plants, <em>DNA Research</em>, Volume 22, Issue 6, 1 December 2015, Pages 461–470, <a href='https://doi.org/10.1093/dnares/dsv027'>https://doi.org/10.1093/dnares/dsv027</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2462501?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462501?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462501?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462501?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462501?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462501?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462501?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=New%20insights%20into%20the%20interplay%20between%20codon%20bias%20determinants%20in%20plants&amp;publicationDate=2015-11-05&amp;volumeNum=22&amp;issueNum=6&amp;author=Camiolo%2C%20S.%3B%20Melito%2C%20S.&amp;startPage=461&amp;endPage=470&amp;contentId=10.1093%2Fdnares%2Fdsv027&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f6%2f461%2f2462501" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87200886 id="87200886" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Codon bias is the non-random use of synonymous codons, a phenomenon that has been observed in species as diverse as bacteria, plants and mammals. The preferential use of particular synonymous codons may reflect neutral mechanisms (e.g. mutational bias, G|C-biased gene conversion, genetic drift) and/or selection for mRNA stability, translational efficiency and accuracy. The extent to which these different factors influence codon usage is unknown, so we dissected the contribution of mutational bias and selection towards codon bias in genes from 15 eudicots, 4 monocots and 2 mosses. We analysed the frequency of mononucleotides, dinucleotides and trinucleotides and investigated whether the compositional genomic background could account for the observed codon usage profiles. Neutral forces such as mutational pressure and G|C-biased gene conversion appeared to underlie most of the observed codon bias, although there was also evidence for the selection of optimal translational efficiency and mRNA folding. Our data confirmed the compositional differences between monocots and dicots, with the former featuring in general a lower background compositional bias but a higher overall codon bias.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;codon bias&quot;">codon bias</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;mutational bias&quot;">mutational bias</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;translational selection&quot;">translational selection</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;plant genetics&quot;">plant genetics</a></div>                            <h2 scrollto-destination=87200888 id="87200888" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The genetic code is redundant, with most amino acids encoded by two or more synonymous codons.<sup><a reveal-id="DSV027C1  DSV027C4" data-open="DSV027C1  DSV027C4" class="link link-ref link-reveal xref-bibr">1–4</a></sup> The non-random use of synonymous codons is known as codon bias, and it may reflect several underlying factors, including mutational bias in the genome and translational selection. The possibility that mutational bias affects codon usage has led to the neutralist model, in which codon identity is mainly determined by nucleotide substitution patterns in the genome. In contrast, the possibility of translational selection has led to the selective model, in which the choice of synonymous codons reflects tRNA abundance<sup><a reveal-id="DSV027C5" data-open="DSV027C5" class="link link-ref link-reveal xref-bibr">5</a></sup> to optimize the efficiency<sup><a reveal-id="DSV027C4" data-open="DSV027C4" class="link link-ref link-reveal xref-bibr">4</a></sup> and accuracy<sup><a reveal-id="DSV027C6" data-open="DSV027C6" class="link link-ref link-reveal xref-bibr">6</a></sup> of translation. These models are not mutually exclusive, i.e. the choice among synonymous codons may reflect a balance between selective and mutational pressures.<sup><a reveal-id="DSV027C7" data-open="DSV027C7" class="link link-ref link-reveal xref-bibr">7</a></sup></p><p>Species-dependent differences in codon usage are well known,<sup><a reveal-id="DSV027C8" data-open="DSV027C8" class="link link-ref link-reveal xref-bibr">8</a></sup> but recent studies have identified variations within species that must also be addressed by the neutralist and selective models. For example, the abundance of tRNA may vary during development and in response to external stimuli,<sup><a reveal-id="DSV027C9" data-open="DSV027C9" class="link link-ref link-reveal xref-bibr">9</a></sup> suggesting that codon bias may represent an adaptive response to tRNA levels that differ among plant tissues.<sup><a reveal-id="DSV027C10" data-open="DSV027C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Nucleotide substitution patterns are also unequally distributed in the genome,<sup><a reveal-id="DSV027C11" data-open="DSV027C11" class="link link-ref link-reveal xref-bibr">11</a></sup> e.g. there are large homogeneous blocks G + C rich sequences known as isochores in the genomes of warm-blooded vertebrates.<sup><a reveal-id="DSV027C12" data-open="DSV027C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSV027C13" data-open="DSV027C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Likewise, differences in nucleotide substitution patterns have been observed in rice (<em>Oryza sativa</em>), with genes expressed in the roots being predominantly G + C rich and genes expressed in seeds and leaves being predominantly A + T rich.<sup><a reveal-id="DSV027C14" data-open="DSV027C14" class="link link-ref link-reveal xref-bibr">14</a></sup></p><p>The compositional context can also influence synonymous codon selection, a phenomenon known as context-dependent codon bias (CCDB). In mammals, bacteria and plants, the first nucleotide after each codon drives synonymous codon choice, because several dinucleotide sequences such as CG, GA and TA are under-represented.<sup><a reveal-id="DSV027C15  DSV027C17" data-open="DSV027C15  DSV027C17" class="link link-ref link-reveal xref-bibr">15–17</a></sup> In plant genomes, there is also a general bias in the use of specific dinucleotides and trinucleotides in different genomic regions.<sup><a reveal-id="DSV027C11" data-open="DSV027C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSV027C18" data-open="DSV027C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p><p>The composition of coding sequences is determined by a complex series of interacting factors, so it is difficult to identify the relative impact of different components. A model to determine the influence of nucleotide substitution patterns on codon bias has been developed by building a new set of sequences in which the third codon position in the coding sequence is replaced with a random nucleotide from the neighbouring intergenic region.<sup><a reveal-id="DSV027C19" data-open="DSV027C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Such intergenic corrected coding sequences (ICCSs) retain the same amino acid sequence while mirroring the background nucleotide substitution pattern of the genome. Comparing the codon bias between the original coding sequence (CS) and ICCS datasets can therefore highlight the influence of the background nucleotide substitution pattern on the coding sequence composition.<sup><a reveal-id="DSV027C19" data-open="DSV027C19" class="link link-ref link-reveal xref-bibr">19</a></sup></p><p>Although the selective model has been studied in several plant species, the impact of background composition on gene structure has been largely overlooked. Codon bias in <em>Arabidopsis thaliana</em> (Arabidopsis) tends to be associated with the composition of the 3′ flanking region in both strongly and weakly expressed genes,<sup><a reveal-id="DSV027C20" data-open="DSV027C20" class="link link-ref link-reveal xref-bibr">20</a></sup> although the impact of selection on both classes of genes has also been recognized. Here we used the Hershberg and Petrov approach<sup><a reveal-id="DSV027C19" data-open="DSV027C19" class="link link-ref link-reveal xref-bibr">19</a></sup> to determine the effect of background composition on the codon bias of 21 plant species while also accounting for bias in the frequencies of dinucleotides and trinucleotides. We discuss in detail the impact of these multiple factors and others influencing codon bias in plants.</p>                            <h2 scrollto-destination=87200909 id="87200909" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87200915 id="87200915" class="section-title" data-legacy-id=s2a>2.1. Sequence data</h3>
<p>The genomic sequences and annotation data for 21 plant species were downloaded from Phytozome (<a class="link link-uri" href="http://www.phytozome.net">http://www.phytozome.net</a>).<sup><a reveal-id="DSV027C21" data-open="DSV027C21" class="link link-ref link-reveal xref-bibr">21</a></sup> We analysed the sequences of 4 monocots [<em>Brachypodium distachyon</em> (BD), <em>O. sativa</em> (OS), <em>Sorghum bicolor</em> (SB) and <em>Zea mays</em> (ZM)], 15 dicots [<em>Arabidopsis lyrata</em> (AL)<em>, A. thaliana</em> (AT)<em>, Brassica rapa</em> (BR)<em>, Citrus clementina</em> (CC)<em>, Citrus sinensis</em> (CS)<em>, Eucalyptus grandis</em> (EG)<em>, Glycine max</em> (GM)<em>, Linum usitatissimum</em> (LU)<em>, Medicago truncatula</em> (MT)<em>, Populus trichocarpa</em> (PopT)<em>, Phaseolus vulgaris</em> (PV)<em>, Solanum lycopersicum</em> (SL)<em>, Solanum tuberosum</em> (ST)<em>, Thellungiella halophila</em> (TH) and <em>Vitis vinifer</em>a (VV)] and 2 mosses [<em>Physcomitrella patens</em> (PP) and <em>Selaginella moellendorffii</em> (SM)].</p><p>We used gff2sequence<sup><a reveal-id="DSV027C22" data-open="DSV027C22" class="link link-ref link-reveal xref-bibr">22</a></sup> to identify coding sequences, proteins, introns and intergenic sequences. Coding sequences featuring non-canonical bases (other than A, C, G or T), missing stop codons or incomplete triplets were excluded. Finally, the longest splicing variant was chosen when multiple transcripts representing the same gene were annotated if not otherwise specified. Monocot coding sequences were divided into three subsets for analysis: (i) the entire genome, (ii) high G + C (HGC) content genes, &gt;60% and (iii) low G + C (LGC) content genes, ≤60%.</p>                            <h3 scrollto-destination=87200924 id="87200924" class="section-title" data-legacy-id=s2b>2.2. Expression data</h3>
<p>Gene expression data for Arabidopsis and rice were downloaded from the Plexdb database (<a class="link link-uri" href="http://www.plexdb.org/">http://www.plexdb.org/</a>).<sup><a reveal-id="DSV027C23" data-open="DSV027C23" class="link link-ref link-reveal xref-bibr">23</a></sup> We chose the expression atlases representing Arabidopsis dataset AT40 (<a class="link link-uri" href="http://www.plexdb.org/modules/PD_browse/experiment_browser.php?experiment=AT40">http://www.plexdb.org/modules/PD_browse/experiment_browser.php?experiment=AT40</a>) and rice dataset OS9 (<a class="link link-uri" href="http://www.plexdb.org/modules/PD_browse/experiment_browser.php?experiment=OS9">http://www.plexdb.org/modules/PD_browse/experiment_browser.php?experiment=OS9</a>). All the expression data were normalized by robust multi-array averaging (RMA). An expression value was calculated for each gene by averaging the replicates within each experiment and then computing a mean value over all the experiments in which the corresponding gene was expressed.<sup><a reveal-id="DSV027C24" data-open="DSV027C24" class="link link-ref link-reveal xref-bibr">24</a></sup></p>                            <h3 scrollto-destination=87200931 id="87200931" class="section-title" data-legacy-id=s2c>2.3. Intergenic controlled coding sequences</h3>
<p>For each gene, the first 100 two-fold and four-fold degenerate codons were used to create the coding sequence for analysis. Transcripts with fewer degenerate codons were excluded. The ICCSs were generated to estimate the influence of the background composition on coding sequence codon bias. Upstream and downstream sequences longer than 50 bp were extracted from the leading strand and joined together to form a set of concatenated intergenic sequences (CISs).</p><p>Four different background controls were used to generate the ICCS datasets, beginning with the mononucleotide composition as originally used in the Hershberg and Petrov method.<sup><a reveal-id="DSV027C19" data-open="DSV027C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Briefly, a subsequence of 100 consecutive base pairs was randomly selected from the CIS, and the third base of each codon in the coding sequence was replaced with a nucleotide from this CIS subsequence (Fig. <a reveal-id="DSV027F1" data-open="DSV027F1" class="link link-reveal link-table xref-fig">1</a>). This yielded a new data set called monoICCS. The intergenic dinucleotide composition was used to generate a second class of ICCS (dinuICCS) by choosing a random subsequence of 200 consecutive nucleotides from the CIS and picking the second base of each coding sequence codon randomly from within that subsequence. The adjacent base was then selected as the third codon position in the ICCS (Fig. <a reveal-id="DSV027F1" data-open="DSV027F1" class="link link-reveal link-table xref-fig">1</a>). If the CIS was shorter than 100 bp, it was excluded from the monoICCS, and if it was shorter than 200 bp, it was excluded from the dinuICCS. Genes were also excluded from further analysis if any base in the CIS occurred fewer than four times. </p>                        <a id="87200935" scrollto-destination="87200935"></a>
<div id="DSV027F1" data-id="DSV027F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/m_dsv02701.png?Expires=2147483647&Signature=itegEPVLjvrTRTf~s4Uj2O-r5oc16z3JbfuBKbPmJArfxoRP2rcgTqyZLp4-bEU9SMzYwtaM7YBHxaC~H6dNYv0eATpDwJ2UGvZbfrWD0SdkH1wo3atko~wHowIdDdDToGjkqLxXYQwPmYx5sYMOOx5MUV3C3W4GvvU3TZOH4mJ~1TX1FDu9K7u0WLoTwPJ3g86DQNA0wLIqrEZs~FObdhDVRi51uZoXjH1kAMCSpmd0qMDE8icZqCN6QdKw0T6BfykhgiW2QFsyiqTgcyZJg6w5lNhpdSYhsa9c~lb1RnEIZ2Kr4KhGJ2YEJXa6os6K3gIyBDa8po0O~U2xb3Y7Jw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Method used for the construction of the mononucleotide ICCS (monoICCS), dinucleotide ICCS (dinuICCS) and trinucleotide ICCS (trinuICCS) data sets. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02701.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200935/dsv02701.tiff" path-from-xml="dsv02701.tiff" target="_blank">View large</a><a section="87200935" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/dsv02701.png?Expires=2147483647&Signature=Zr6smivujaRQjj0wDb9cCldb5vS3FpGkNjQmxK0e8w7oqQ8UQFWo6KiB54pYdiAEbum7RdDv2bUwljIMsQXyvD-wm0Vc9ag6nEPe7WaTpMNjJRGsCYiHZSdRJBb0IQOrs60psJS2imqM-sjlncs61PA3f6teg1oNqkiI~rOS-e-2uGfyE5O6GDfeOm3d1rwHXGC81NrOa-m0AY-Y4ixuvKWyhhNweU9nwEDM8msmv7~dCU0hzG9slUkhPqpmac8Uv5ZE54jwgHK41C3PL8YKSEgJHYUQWdxfOD4PYlJ~UisN248HrcKlQwwomiSYXV5f8cDEt04bef6T0HT7leTHag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200935&ar=2462501&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02701.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Method used for the construction of the mononucleotide ICCS (monoICCS), dinucleotide ICCS (dinuICCS) and trinucleotide ICCS (trinuICCS) data sets. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV027F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/m_dsv02701.png?Expires=2147483647&Signature=itegEPVLjvrTRTf~s4Uj2O-r5oc16z3JbfuBKbPmJArfxoRP2rcgTqyZLp4-bEU9SMzYwtaM7YBHxaC~H6dNYv0eATpDwJ2UGvZbfrWD0SdkH1wo3atko~wHowIdDdDToGjkqLxXYQwPmYx5sYMOOx5MUV3C3W4GvvU3TZOH4mJ~1TX1FDu9K7u0WLoTwPJ3g86DQNA0wLIqrEZs~FObdhDVRi51uZoXjH1kAMCSpmd0qMDE8icZqCN6QdKw0T6BfykhgiW2QFsyiqTgcyZJg6w5lNhpdSYhsa9c~lb1RnEIZ2Kr4KhGJ2YEJXa6os6K3gIyBDa8po0O~U2xb3Y7Jw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Method used for the construction of the mononucleotide ICCS (monoICCS), dinucleotide ICCS (dinuICCS) and trinucleotide ICCS (trinuICCS) data sets. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02701.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200935/dsv02701.tiff" path-from-xml="dsv02701.tiff" target="_blank">View large</a><a section="87200935" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/dsv02701.png?Expires=2147483647&Signature=Zr6smivujaRQjj0wDb9cCldb5vS3FpGkNjQmxK0e8w7oqQ8UQFWo6KiB54pYdiAEbum7RdDv2bUwljIMsQXyvD-wm0Vc9ag6nEPe7WaTpMNjJRGsCYiHZSdRJBb0IQOrs60psJS2imqM-sjlncs61PA3f6teg1oNqkiI~rOS-e-2uGfyE5O6GDfeOm3d1rwHXGC81NrOa-m0AY-Y4ixuvKWyhhNweU9nwEDM8msmv7~dCU0hzG9slUkhPqpmac8Uv5ZE54jwgHK41C3PL8YKSEgJHYUQWdxfOD4PYlJ~UisN248HrcKlQwwomiSYXV5f8cDEt04bef6T0HT7leTHag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200935&ar=2462501&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02701.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Method used for the construction of the mononucleotide ICCS (monoICCS), dinucleotide ICCS (dinuICCS) and trinucleotide ICCS (trinuICCS) data sets. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>Intergenic trinucleotide controlled coding sequences (trinuICCS) were produced by randomly picking the first dinucleotide for each coding sequence codon from within the CIS and selecting the adjacent nucleotide as the third base for the ICCS (Fig. <a reveal-id="DSV027F1" data-open="DSV027F1" class="link link-reveal link-table xref-fig">1</a>). Finally, CDCB was estimated by randomly selecting an interrupted dinucleotide comprising the second base of each coding sequence codon and the first base of the subsequent codon. The intervening nucleotide was then selected as the third codon base in the CDCB intergenic controlled coding sequence (cdcbICCS). Genes in the trinuICCS and cdcbICCS data sets were excluded from further analysis if the corresponding dinucleotide appeared fewer than four times in the CIS.</p><p>To maintain the coding sequence amino acid structure, two-fold degenerate codons ending in A|G produced an ICCS ending in A when the corresponding intergenic position was A or T, and otherwise the ICCS ended in G. Similarly, two-fold degenerate codons ending in T|C produced an ICCS featuring T at the third codon position when the corresponding CIS nucleotide was A or T, and otherwise the ICCS ended in C.</p>                            <h3 scrollto-destination=87200938 id="87200938" class="section-title" data-legacy-id=s2d>2.4. Codon bias measurements</h3>
<p>The effective number of codons (Nc)<sup><a reveal-id="DSV027C25" data-open="DSV027C25" class="link link-ref link-reveal xref-bibr">25</a></sup> was used to estimate the overall codon bias for each gene in the CS and ICCS data sets. The Nc index gave higher values for less-biased genes (theoretical values between 21 and 61). To explore the contribution of each individual codon, relative synonymous codon usage (RSCU) values were computed for informative two-fold and four-fold degenerate codons (i.e. excluding methionine, tryptophan, stop codons, as well as three-fold and six-fold degenerate codons to avoid uneven nucleotide representation at the third position within the same codon family). RSCU values were calculated as the ratio of the observed and expected codon frequencies, i.e. the random use of all codons within a specific degenerate family.<sup><a reveal-id="DSV027C26" data-open="DSV027C26" class="link link-ref link-reveal xref-bibr">26</a></sup></p>                            <h3 scrollto-destination=87200940 id="87200940" class="section-title" data-legacy-id=s2e>2.5. Signature of selection</h3>
<div class="block-child-p">For the 38 codons with two-fold or four-fold degeneracy, differences between the average RSCU values in the CS and ICCS data sets were calculated to highlight over-representation and under-representation in the coding sequences. For each codon <em>cod</em>, the deviation from background was calculated as follows:<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi mathvariant="normal" xmlns="">Δ</mi><mtext xmlns="">RSC</mtext><mrow xmlns=""><msup><mtext>U</mtext><mrow><mrow><mi mathvariant="normal">cod</mi></mrow></mrow></msup></mrow><mo xmlns="">=</mo><mstyle xmlns=""><mrow><mfrac><mrow><msubsup><mtext>RSCU</mtext><mrow><mrow><mi mathvariant="normal">CS</mi></mrow></mrow><mrow><mrow><mi mathvariant="normal">cod</mi></mrow></mrow></msubsup><mo>−</mo><msubsup><mtext>RSCU</mtext><mrow><mrow><mi mathvariant="normal">ICCS</mi></mrow></mrow><mrow><mrow><mi mathvariant="normal">cod</mi></mrow></mrow></msubsup></mrow><mrow><mtext>De</mtext><mrow><msup><mtext>g</mtext><mrow><mrow><mi mathvariant="normal">cod</mi></mrow></mrow></msup></mrow></mrow></mfrac></mrow></mstyle></math></div></div></div> where Deg<sup>cod</sup> is the degeneracy of the codon family. The significance of highlighted differences was tested using a paired Wilcoxon test. The same statistical analysis was applied to highlight differences in the effective number of codons between the CS and ICCS datasets.</div><p>The association between the RSCU values of the CS and ICCS datasets and the gene expression levels was investigated in Arabidopsis and rice by sorting genes on the basis of their expression levels into 20 bins containing the same number of genes. The bin rank was then plotted against the average CS|ICCS RSCU value within the bin using either the intergenic or intron portion for the construction of the ICCS data set. In the latter case, all introns within the same gene were concatenated. In rice, this analysis was also carried out separately on the HGC and LGC gene sets.</p><p>Differences in codon bias between the CS and ICCS data sets were also calculated in several different portions of the coding sequences in a subset of plant species (AT, BR, OS, BD, PV, MT and SM). The unequal distribution of codon bias within individual transcripts is a well-known phenomenon that may cause different levels of deviation from the background.<sup><a reveal-id="DSV027C27  DSV027C29" data-open="DSV027C27  DSV027C29" class="link link-ref link-reveal xref-bibr">27–29</a></sup> To investigate this phenomenon, we filtered coding sequences &gt;1,500 bp in length and split them into three portions: the 5′ region comprising the first 501 nucleotides, the 3′ region the last 501 nucleotides and the middle region containing the remaining part of the coding sequence. The ΔRSCU value and its significance were calculated for each of the three portions.</p><p>The analysis was repeated with the Arabidopsis and rice ICCS data sets using intergenic regions that were masked (e.g. repetitive elements and pseudogenes were removed according to data retrieved from the TAIR and Phytozome websites) and trimmed by 200 bp at both ends. The differences between the CS and ICCS data sets in both species were also analysed separately on genes showing regular or alternative splicing. Finally, the impact of the intergenic distance was investigated by analysing two Arabidopsis and rice data sets, one featuring long intergenic sequences (&gt;1,000 bp for AT and &gt;2,200 bp for OS) and the other featuring short intergenic sequences (≤1,000 bp for AT and ≤2,200 bp for OS).</p>                            <h2 scrollto-destination=87200945 id="87200945" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=87200946 id="87200946" class="section-title" data-legacy-id=s3a>3.1. Codon bias in the ICCSs</h3>
<p>Mutational bias is one of the main forces affecting synonymous codon choice in plants.<sup><a reveal-id="DSV027C30" data-open="DSV027C30" class="link link-ref link-reveal xref-bibr">30</a></sup> In theory, if no additional forces act on the coding sequences, the third base of each codon should reflect the background nucleotide frequency in the genome. However, the direction and strength of the codon bias should be investigated in the local genomic context to correct for intragenomic compositional heterogeneity. Several authors have reported the non-random usage of specific dinucleotides<sup><a reveal-id="DSV027C31" data-open="DSV027C31" class="link link-ref link-reveal xref-bibr">31</a></sup> and trinucleotides<sup><a reveal-id="DSV027C11" data-open="DSV027C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSV027C18" data-open="DSV027C18" class="link link-ref link-reveal xref-bibr">18</a></sup> in plant genomes, and thus, the analysis of mutational bias should also take into account these compositional units.</p><p>We used the composition of intergenic regions as a proxy for background bias according to the Hershberg and Petrov method.<sup><a reveal-id="DSV027C19" data-open="DSV027C19" class="link link-ref link-reveal xref-bibr">19</a></sup> The portion of codon bias caused by background composition was measured by generating several ICCS data sets taking into account the mononucleotide, dinucleotide, trinucleotide and context-dependent intergenic composition of each plant species.</p><p>We investigated the direction of mutational bias by analysing the codon bias among all the ICCS data sets. If the four nucleotides are equally and randomly distributed in the background, then there should be no codon bias in any of the ICCS data sets. In contrast, we observed the opposite trend, i.e. Nc values compatible with a multilevel background compositional bias. Interestingly, the lowest Nc values were found in the dinuICCS data set, revealing the strongest bias for all species (Fig. <a reveal-id="DSV027F2" data-open="DSV027F2" class="link link-reveal link-table xref-fig">2</a>). The intensity of codon bias distinguished monocots and dicots. Whereas eudicots and particularly the legumes (<em>P. vulgaris</em>, <em>M. truncatula</em> and <em>G. max</em>) showed the highest ICCS codon bias at all levels, the opposite trend was observed for the monocots (rice, <em>Z. mays</em>, <em>S. bicolor</em> and <em>B. distachyon</em>) as shown in Fig. <a reveal-id="DSV027F2" data-open="DSV027F2" class="link link-reveal link-table xref-fig">2</a>. </p>                        <a id="87200950" scrollto-destination="87200950"></a>
<div id="DSV027F2" data-id="DSV027F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/m_dsv02702.png?Expires=2147483647&Signature=hH~vim3wdmWzeCr~GTyodYspwoiDHBNHlAfO8n5HrVmeSF6Fg7KTjrr~oEgun0eUwc5KLBuZAeW2DcoIU4j34NAAgbpb2i11fP5FZKD~RU869EvO6r-9~5XDUm2Vyt6e2kQkpJtY4N3hTsEJCTmvZqalGth6-WjxzOL8RiazykWbd2a3YRFPQVq6rizuilEpSY9bVagCxBP03yfiRMNuu~j55P3WMZTASKErF-8zaF3CvG2vF3~ONIXgmBPm-8rCopfhLHPhRPKcAtNfXFqByGk4BFqGsJ3RG4lf0BaBRI8RRKMU0HXZtGfqxzC7CWKN3TK8NWVFSvWz-yFqLtzubg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Heat map showing the average Nc values for the CS and ICCS data sets. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02702.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200950/dsv02702.tiff" path-from-xml="dsv02702.tiff" target="_blank">View large</a><a section="87200950" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/dsv02702.png?Expires=2147483647&Signature=m-hDujszUKG-zZCMLVKle9XYKuJlLS64uuy4WH20knI1zBr0LpOYCfmxnVdOPyObYc8u4bmMBy6DYhSTD0q5lgCYSHS1eKKBUbKdre~48-egPZS7JSSga3OkqLsEpPgI9EihfRJZQFtpSghIKxmWnTx07u0G1qmoTI9IEG4wNm6uG7D9Y8lS5YlpbTnnykJgDr2jrY6pDLuUE6GUKTMfNkoohoNecc-6z0IxqGky12iSiL1y2WIKV0p3jmPE-w5jM8k2yZkQZGKzAO6EUbBVkyNtcon2SH-THDjtDaFjU2A4hzxrW3Y07wJIvZ2r6sAv0CzA6x8-u~KIEB6Tdg-s7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200950&ar=2462501&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02702.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Heat map showing the average Nc values for the CS and ICCS data sets. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV027F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/m_dsv02702.png?Expires=2147483647&Signature=hH~vim3wdmWzeCr~GTyodYspwoiDHBNHlAfO8n5HrVmeSF6Fg7KTjrr~oEgun0eUwc5KLBuZAeW2DcoIU4j34NAAgbpb2i11fP5FZKD~RU869EvO6r-9~5XDUm2Vyt6e2kQkpJtY4N3hTsEJCTmvZqalGth6-WjxzOL8RiazykWbd2a3YRFPQVq6rizuilEpSY9bVagCxBP03yfiRMNuu~j55P3WMZTASKErF-8zaF3CvG2vF3~ONIXgmBPm-8rCopfhLHPhRPKcAtNfXFqByGk4BFqGsJ3RG4lf0BaBRI8RRKMU0HXZtGfqxzC7CWKN3TK8NWVFSvWz-yFqLtzubg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Heat map showing the average Nc values for the CS and ICCS data sets. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02702.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200950/dsv02702.tiff" path-from-xml="dsv02702.tiff" target="_blank">View large</a><a section="87200950" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/dsv02702.png?Expires=2147483647&Signature=m-hDujszUKG-zZCMLVKle9XYKuJlLS64uuy4WH20knI1zBr0LpOYCfmxnVdOPyObYc8u4bmMBy6DYhSTD0q5lgCYSHS1eKKBUbKdre~48-egPZS7JSSga3OkqLsEpPgI9EihfRJZQFtpSghIKxmWnTx07u0G1qmoTI9IEG4wNm6uG7D9Y8lS5YlpbTnnykJgDr2jrY6pDLuUE6GUKTMfNkoohoNecc-6z0IxqGky12iSiL1y2WIKV0p3jmPE-w5jM8k2yZkQZGKzAO6EUbBVkyNtcon2SH-THDjtDaFjU2A4hzxrW3Y07wJIvZ2r6sAv0CzA6x8-u~KIEB6Tdg-s7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200950&ar=2462501&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02702.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Heat map showing the average Nc values for the CS and ICCS data sets. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>Nc values provide a snapshot of overall codon bias within genes but do not reveal the specific contributions of each codon. ICCS RSCU values were therefore calculated for all codons with two-fold or four-fold degeneracy (Fig. <a reveal-id="DSV027F3" data-open="DSV027F3" class="link link-reveal link-table xref-fig">3</a>). We observed a significant over-representation of codons ending in A|T in the monoICCS data set of the eudicots, particularly <em>V. vinifera</em>, <em>Solanum</em> spp<em>.</em> and the legumes, and likewise in monocots and <em>S. moellendorffii</em> but to a much lesser extent. </p>                        <a id="87200952" scrollto-destination="87200952"></a>
<div id="DSV027F3" data-id="DSV027F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/m_dsv02703.png?Expires=2147483647&Signature=AyzypgnQKQ8Gr2BUmyQMMbJveQZs-GkIBXXVvZKmJV~bIYQjt5stbqcZZsItwoPFFRIAHTYZ4PvdALzrejFS5qsXx-E33xqUGzhDSx6dsB7afxoxy~kLOBLzN8QYRt4pt7ZmwdkKqBWFB1MBONv7SxsZoabiXduxtTiMY5Tsrm9udK4aR1H1z3qFARZ0CCQqITkoffUPyz5ezIoaKvQfbea4Tekq5r93khSnNvVj3eAFGaD-8MJc7wjPOqF1iRCD2SSK-UMhmEddjwzzjxbl7FKeFtzRhYaKikKLOIeNDCH3OeTvZsa2q1sxJeOeWHT7IH4b2MNPtnvp3bcmKpRCGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="RSCU values calculated for the monoICCS, dinuICCS and trinuICCS data sets (only codons ending in G|C with two-fold degeneracy are shown). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02703.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200952/dsv02703.tiff" path-from-xml="dsv02703.tiff" target="_blank">View large</a><a section="87200952" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/dsv02703.png?Expires=2147483647&Signature=cBjH4dMwN8~7wyEP94FNzLVQSs7rzlNWef2mhqzmmrkAEA9UrUWg2F-PJAiMy73zrlQMVoVKmnSPpUt~602UwWyXODLrsbbDtD-5BGn2xTgROMUOGySALYfOfYZkd40wfK0GxXUV3nuhUQQANmMooRrJZnAahHMI-pNmO91B8broPU7wle7tcWqSu2vmX77eufOes5AlpmHr1hiLx8zyjo6RmKYfI6TTWw8L1qd7OorTyXSIO3MzJy-DXZM6DRoJmGBNS68yZqLw4dCtzPwwkYcF79Qo~jOHVPPtU8TXx29QKvq-dhr5aumWHyfJHzIqbFhIGjuY6Cdw7sWNtPUilw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200952&ar=2462501&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02703.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>RSCU values calculated for the monoICCS, dinuICCS and trinuICCS data sets (only codons ending in G|C with two-fold degeneracy are shown). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV027F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/m_dsv02703.png?Expires=2147483647&Signature=AyzypgnQKQ8Gr2BUmyQMMbJveQZs-GkIBXXVvZKmJV~bIYQjt5stbqcZZsItwoPFFRIAHTYZ4PvdALzrejFS5qsXx-E33xqUGzhDSx6dsB7afxoxy~kLOBLzN8QYRt4pt7ZmwdkKqBWFB1MBONv7SxsZoabiXduxtTiMY5Tsrm9udK4aR1H1z3qFARZ0CCQqITkoffUPyz5ezIoaKvQfbea4Tekq5r93khSnNvVj3eAFGaD-8MJc7wjPOqF1iRCD2SSK-UMhmEddjwzzjxbl7FKeFtzRhYaKikKLOIeNDCH3OeTvZsa2q1sxJeOeWHT7IH4b2MNPtnvp3bcmKpRCGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="RSCU values calculated for the monoICCS, dinuICCS and trinuICCS data sets (only codons ending in G|C with two-fold degeneracy are shown). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02703.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200952/dsv02703.tiff" path-from-xml="dsv02703.tiff" target="_blank">View large</a><a section="87200952" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/dsv02703.png?Expires=2147483647&Signature=cBjH4dMwN8~7wyEP94FNzLVQSs7rzlNWef2mhqzmmrkAEA9UrUWg2F-PJAiMy73zrlQMVoVKmnSPpUt~602UwWyXODLrsbbDtD-5BGn2xTgROMUOGySALYfOfYZkd40wfK0GxXUV3nuhUQQANmMooRrJZnAahHMI-pNmO91B8broPU7wle7tcWqSu2vmX77eufOes5AlpmHr1hiLx8zyjo6RmKYfI6TTWw8L1qd7OorTyXSIO3MzJy-DXZM6DRoJmGBNS68yZqLw4dCtzPwwkYcF79Qo~jOHVPPtU8TXx29QKvq-dhr5aumWHyfJHzIqbFhIGjuY6Cdw7sWNtPUilw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200952&ar=2462501&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02703.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>RSCU values calculated for the monoICCS, dinuICCS and trinuICCS data sets (only codons ending in G|C with two-fold degeneracy are shown). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>A similar picture emerged from the analysis of the dinuICCS and trinuICCS RSCU values although several compositional signatures also emerged. The codons ending in A|T generally showed higher RSCU values at the expense of those ending in G|C, but the extent of the bias was variable. Indeed, codons ending in CG, AG and AC tended to be suppressed more strongly in the dinuICCS data set, e.g. these dinucleotides were under-represented in the intergenic regions of all the plants whereas codons ending in CA showed the opposite trend (Fig. <a reveal-id="DSV027F3" data-open="DSV027F3" class="link link-reveal link-table xref-fig">3</a>). Only marginal differences were observed between the dinuICCS and trinuICCS data sets, indicating that the intergenic trinucleotide bias is predominantly caused by bias in the frequency of the underlying dinucleotides. Even less difference was observed between the cdcbICCS and trinuICCS datasets, indicating that CDCB does not reflect an analogous compositional pattern at a genomic level.</p><p>Differences in codon bias were distributed along the chromosomes of both monocots and dicots for all ICCS data sets (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/dsv027_Supplementary_Data.zip?Expires=2147483647&Signature=o4Yq-5GBDQo03KxFpLSJSloKTczliBPl7Y6bH5NKBv3MiWaj2LAb4Jv0ucEJ1yodO~9Ht7JfUypLsXNM0eCXlcX9AW7e91dJpdtfp-vCCatiBSJij7KQBsnEWod4wn2gj5PPJEYu9HEmeHgBY-ykpVHV2AbcRKp5uTC5rWlJRF40MwC5Yzz1c4PMdw5-3e-pNKqGcabhbMjVSCnHc6H~RCnEL5HoX--9XXZYh768xTuS00xjK94IgEoYJ3XO8G-9wJoCX~JVyjJLDKe1QiF1ojPBxub4hnpXbKH1dDyx-EG7u3TS82lkUWoBTmdVREYMrmkpaaDzk2l7X11lVlEW7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>), but spatial autocorrelation analysis failed to detect a significant effect on the ICCS RSCU values (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/dsv027_Supplementary_Data.zip?Expires=2147483647&Signature=o4Yq-5GBDQo03KxFpLSJSloKTczliBPl7Y6bH5NKBv3MiWaj2LAb4Jv0ucEJ1yodO~9Ht7JfUypLsXNM0eCXlcX9AW7e91dJpdtfp-vCCatiBSJij7KQBsnEWod4wn2gj5PPJEYu9HEmeHgBY-ykpVHV2AbcRKp5uTC5rWlJRF40MwC5Yzz1c4PMdw5-3e-pNKqGcabhbMjVSCnHc6H~RCnEL5HoX--9XXZYh768xTuS00xjK94IgEoYJ3XO8G-9wJoCX~JVyjJLDKe1QiF1ojPBxub4hnpXbKH1dDyx-EG7u3TS82lkUWoBTmdVREYMrmkpaaDzk2l7X11lVlEW7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>).<sup><a reveal-id="DSV027C32" data-open="DSV027C32" class="link link-ref link-reveal xref-bibr">32</a></sup></p>                            <h3 scrollto-destination=87200955 id="87200955" class="section-title" data-legacy-id=s3b>3.2. Codon bias in the coding sequences</h3>
<p>If synonymous codon choice solely reflects the distribution of nucleotides in the genomic background, there should be no differences in Nc value between the CS and ICCS data sets. However, our results revealed significant differences (Wilcoxon paired sample test) between these data sets, although the direction and extent of divergence were species dependent. Coding sequences were more biased in monocots than in dicots (Fig. <a reveal-id="DSV027F2" data-open="DSV027F2" class="link link-reveal link-table xref-fig">2</a>) in sharp contrast to the comparison of ICCS data sets. Interestingly, the coding sequences were even less biased than the corresponding ICCSs in some species (i.e. GM and PV), with the trend most noticeable when comparing the CS and dinuICCS data sets. These data suggest that additional forces shape the coding sequences and oppose mutational bias, e.g. resulting in G|C enrichment despite background A|T enrichment.</p><p>Differences in RSCU values between the CS and ICCS data sets allowed us to focus on codons whose frequency cannot be explained by background bias alone. Comparisons between the CS data set and the four ICCS data sets led to similar results (Fig. <a reveal-id="DSV027F4" data-open="DSV027F4" class="link link-reveal link-table xref-fig">4</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/dsv027_Supplementary_Data.zip?Expires=2147483647&Signature=o4Yq-5GBDQo03KxFpLSJSloKTczliBPl7Y6bH5NKBv3MiWaj2LAb4Jv0ucEJ1yodO~9Ht7JfUypLsXNM0eCXlcX9AW7e91dJpdtfp-vCCatiBSJij7KQBsnEWod4wn2gj5PPJEYu9HEmeHgBY-ykpVHV2AbcRKp5uTC5rWlJRF40MwC5Yzz1c4PMdw5-3e-pNKqGcabhbMjVSCnHc6H~RCnEL5HoX--9XXZYh768xTuS00xjK94IgEoYJ3XO8G-9wJoCX~JVyjJLDKe1QiF1ojPBxub4hnpXbKH1dDyx-EG7u3TS82lkUWoBTmdVREYMrmkpaaDzk2l7X11lVlEW7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). All codons ending in A|T were under-represented in the coding sequences, with GTA showing the greatest depletion. In contrast, several codons ending in G|C were over-represented in the coding sequence, with certain species-dependent exceptions. Codons CCC and GGG were suppressed in many species, together with codons CCG, GCG and ACG. The under-representation of these codons together with GTA was more striking when comparing the CS and monoICCS data sets and less pronounced when comparing the CS and dinuICCS data sets, suggesting that such a bias in the coding sequences in part reflects the general suppression of the TA|CG dinucleotide in the intergenic regions (Fig. <a reveal-id="DSV027F3" data-open="DSV027F3" class="link link-reveal link-table xref-fig">3</a>). Some differences between the CS and ICCS data sets were also taxon dependent, e.g. the preference for G|C at the third codon position was more apparent in monocots than in eudicots, particularly <em>M. truncatula, V. vinifera</em> and <em>Solanum</em> spp., where there was little evidence for preference (Fig. <a reveal-id="DSV027F4" data-open="DSV027F4" class="link link-reveal link-table xref-fig">4</a>). </p>                        <a id="87200958" scrollto-destination="87200958"></a>
<div id="DSV027F4" data-id="DSV027F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/m_dsv02704.png?Expires=2147483647&Signature=fjcfCQtJaOeVGahXSSgnCUEpZxkkbb~RMwrdic9r8UH8bD6LVxohA~KB4Vr3m~4OXtSCNcOWUC84uCME3BFKa4vgduCbfsSxHXGd0j9ixoQqCRHtUOVmwhagS1wTPsBPeQU0ibA2ICmYhWfDb7v8KaeRVHAAnz80dyLUgHxsXCBaOpAwpyxW~966jy37XKk6w7IDMMLNbD4u3aSiWjQh2eLPJAyP2yiF28dJQoYVFjukBvJ3t3Xf4R~uM77jM1XKJwJ9SfEaGr7d54Jsb7iOyUIJZR1aXU5iGN1Plr2Y4Sd8lXmRsVB~Ghms7Tk5NHrhlFFMmL32Cd~2IsdzOIslhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Calculation of ΔRSCU values as the standardized differences between the CS RSCU and the monoICCS (right) and the dinuICCS RSCUs (left). Black crosses in white squares show insignificant differences between the data sets. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02704.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200958/dsv02704.tiff" path-from-xml="dsv02704.tiff" target="_blank">View large</a><a section="87200958" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/dsv02704.png?Expires=2147483647&Signature=N6WmOsga5E1PTGSzxfwsj3EnZsXSZjnJOgZjoYKsf84piOwiLANauuYvcqdA-8h-S5rLTGaIMxCJJnW0Qgs5KnlHHNADLy9ooGRuJtuYZ8RVRAx7wSCtMRSCZXMXMKH6nnYFv8II~ZSPfu77nCWzf32wffTkrtZprB89mVOeluJwW~XDWprD18fy7jSpCIvmsB49gzljNkUrG1CZVIeUJ5vjcLPlK5yHg4iUdb~wJE1E6QoV4lMXI~nR3DwB9aJngSqWtJBpC4NEBsISv8m5iUOpKRMql-l78CI5s7jmAO0fbL8KsM3wKainnx-VNgxcn-DfZn3Lb3kBCpAhaGOccQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200958&ar=2462501&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02704.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Calculation of ΔRSCU values as the standardized differences between the CS RSCU and the monoICCS (right) and the dinuICCS RSCUs (left). Black crosses in white squares show insignificant differences between the data sets. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV027F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/m_dsv02704.png?Expires=2147483647&Signature=fjcfCQtJaOeVGahXSSgnCUEpZxkkbb~RMwrdic9r8UH8bD6LVxohA~KB4Vr3m~4OXtSCNcOWUC84uCME3BFKa4vgduCbfsSxHXGd0j9ixoQqCRHtUOVmwhagS1wTPsBPeQU0ibA2ICmYhWfDb7v8KaeRVHAAnz80dyLUgHxsXCBaOpAwpyxW~966jy37XKk6w7IDMMLNbD4u3aSiWjQh2eLPJAyP2yiF28dJQoYVFjukBvJ3t3Xf4R~uM77jM1XKJwJ9SfEaGr7d54Jsb7iOyUIJZR1aXU5iGN1Plr2Y4Sd8lXmRsVB~Ghms7Tk5NHrhlFFMmL32Cd~2IsdzOIslhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Calculation of ΔRSCU values as the standardized differences between the CS RSCU and the monoICCS (right) and the dinuICCS RSCUs (left). Black crosses in white squares show insignificant differences between the data sets. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02704.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200958/dsv02704.tiff" path-from-xml="dsv02704.tiff" target="_blank">View large</a><a section="87200958" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/dsv02704.png?Expires=2147483647&Signature=N6WmOsga5E1PTGSzxfwsj3EnZsXSZjnJOgZjoYKsf84piOwiLANauuYvcqdA-8h-S5rLTGaIMxCJJnW0Qgs5KnlHHNADLy9ooGRuJtuYZ8RVRAx7wSCtMRSCZXMXMKH6nnYFv8II~ZSPfu77nCWzf32wffTkrtZprB89mVOeluJwW~XDWprD18fy7jSpCIvmsB49gzljNkUrG1CZVIeUJ5vjcLPlK5yHg4iUdb~wJE1E6QoV4lMXI~nR3DwB9aJngSqWtJBpC4NEBsISv8m5iUOpKRMql-l78CI5s7jmAO0fbL8KsM3wKainnx-VNgxcn-DfZn3Lb3kBCpAhaGOccQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200958&ar=2462501&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02704.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Calculation of ΔRSCU values as the standardized differences between the CS RSCU and the monoICCS (right) and the dinuICCS RSCUs (left). Black crosses in white squares show insignificant differences between the data sets. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>An unequal distribution of codon bias within individual transcripts has been observed in many species<sup><a reveal-id="DSV027C27  DSV027C29" data-open="DSV027C27  DSV027C29" class="link link-ref link-reveal xref-bibr">27–29</a></sup> so we also studied the differences between the CS and ICCS data sets in three different portions of the coding sequences. Accordingly, we observed a variation in ΔRSCU values along the transcript, with the 5′ regions featuring the greatest differences (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/dsv027_Supplementary_Data.zip?Expires=2147483647&Signature=o4Yq-5GBDQo03KxFpLSJSloKTczliBPl7Y6bH5NKBv3MiWaj2LAb4Jv0ucEJ1yodO~9Ht7JfUypLsXNM0eCXlcX9AW7e91dJpdtfp-vCCatiBSJij7KQBsnEWod4wn2gj5PPJEYu9HEmeHgBY-ykpVHV2AbcRKp5uTC5rWlJRF40MwC5Yzz1c4PMdw5-3e-pNKqGcabhbMjVSCnHc6H~RCnEL5HoX--9XXZYh768xTuS00xjK94IgEoYJ3XO8G-9wJoCX~JVyjJLDKe1QiF1ojPBxub4hnpXbKH1dDyx-EG7u3TS82lkUWoBTmdVREYMrmkpaaDzk2l7X11lVlEW7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>).</p><p>The influence of recurrent motifs within the intergenic regions was tested by using a hard-masked version of the genome and trimming the intergenic portions by 200 bp at each end. Additionally, the impact of intergenic length on the ΔRSCU value was investigated by splitting the original data set into two subsets representing short and long intergenic sequences. In both cases, no differences were observed compared with the original data set (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/dsv027_Supplementary_Data.zip?Expires=2147483647&Signature=o4Yq-5GBDQo03KxFpLSJSloKTczliBPl7Y6bH5NKBv3MiWaj2LAb4Jv0ucEJ1yodO~9Ht7JfUypLsXNM0eCXlcX9AW7e91dJpdtfp-vCCatiBSJij7KQBsnEWod4wn2gj5PPJEYu9HEmeHgBY-ykpVHV2AbcRKp5uTC5rWlJRF40MwC5Yzz1c4PMdw5-3e-pNKqGcabhbMjVSCnHc6H~RCnEL5HoX--9XXZYh768xTuS00xjK94IgEoYJ3XO8G-9wJoCX~JVyjJLDKe1QiF1ojPBxub4hnpXbKH1dDyx-EG7u3TS82lkUWoBTmdVREYMrmkpaaDzk2l7X11lVlEW7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S5 and S6</a></span>).</p><p>Finally, a recent report has suggested that rice genes experience differential selection depending on whether they undergo regular invariant splicing or alternative splicing.<sup><a reveal-id="DSV027C33" data-open="DSV027C33" class="link link-ref link-reveal xref-bibr">33</a></sup> We therefore calculated the differences between the CS and ICCS data sets representing each set of genes separately and observed a strong correlation between the results of each analysis (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/dsv027_Supplementary_Data.zip?Expires=2147483647&Signature=o4Yq-5GBDQo03KxFpLSJSloKTczliBPl7Y6bH5NKBv3MiWaj2LAb4Jv0ucEJ1yodO~9Ht7JfUypLsXNM0eCXlcX9AW7e91dJpdtfp-vCCatiBSJij7KQBsnEWod4wn2gj5PPJEYu9HEmeHgBY-ykpVHV2AbcRKp5uTC5rWlJRF40MwC5Yzz1c4PMdw5-3e-pNKqGcabhbMjVSCnHc6H~RCnEL5HoX--9XXZYh768xTuS00xjK94IgEoYJ3XO8G-9wJoCX~JVyjJLDKe1QiF1ojPBxub4hnpXbKH1dDyx-EG7u3TS82lkUWoBTmdVREYMrmkpaaDzk2l7X11lVlEW7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span>).</p>                            <h3 scrollto-destination=87200962 id="87200962" class="section-title" data-legacy-id=s3c>3.3. Correlation between codon bias and gene expression in Arabidopsis and rice</h3>
<p>Codon bias in the species we investigated was not fully explained by background compositional differences so we investigated the influence of gene expression on synonymous codon usage in the CS and ICCS data sets. Codons that are translated more rapidly or accurately should be preferentially found in the coding sequences of genes, particularly those expressed at high levels. We chose Arabidopsis and rice as representative dicot and monocot species and assigned genes to 20 bins based on expression levels, and then mapped the RSCU values of the CS and the ICCS data sets onto these bins (Fig. <a reveal-id="DSV027F5" data-open="DSV027F5" class="link link-reveal link-table xref-fig">5</a>). This strategy should also reveal additional phenomena other than translational selection. For example, a codon whose frequency is positively associated with expression level may be under selection to optimize translation efficiency and/or accuracy. However, if genes with comparable expression levels share the same composition as the intergenic sequences, this would also generate variation in their ICCS RSCU values. Furthermore, if the RSCU values in the CS and ICCS data sets differ regardless of the expression level, then forces other than the translational selection are likely to be responsible. </p>                        <a id="87200964" scrollto-destination="87200964"></a>
<div id="DSV027F5" data-id="DSV027F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/m_dsv02705.png?Expires=2147483647&Signature=ujgCWN0Ur0QV~1f3TTsGGB5KZjJY8HHchhDd935I5Xg07HM06QXePLfbcIuiCiDCPHr-4c132VQ6wgK3xx0CLCbXkZvFTMs5YpIc1le2~oFXHCMpGpEL~nytE5j~ER614Q1JKANyvSD3ebP6Jjs1UF0gS~yrh5nEul2U7s0QjB1viuJej0NjHM3utmQ7Qv1HKovRYhmZhEQuPeIAkTx6A74m28nlrbYjkSqJ41rgwHldJqkGpCfn9NnrdzuX13~cXA5de3-MOASiyUxjW1LhB97omtAUwffIJ1LRROj3-dD-fk8-2NObWbcB-exPNCUuUKnzf0hqECZ8OT41Xdc59Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="RSCU values of (A) Arabidopsis and (B) rice genes after splitting the data sets into 20 expression bins. Intergenic sequences were used for the construction of the ICCS data sets. Symbol codes: square, CS; circle, monoICCS; triangle, dinuICCS; diamond, trinuICCS; cross, cdcbICCS. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02705.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200964/dsv02705.tiff" path-from-xml="dsv02705.tiff" target="_blank">View large</a><a section="87200964" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/dsv02705.png?Expires=2147483647&Signature=P3FnHFKq5ZG-8nyywKycvv927EdvmRdSJCy04ItH3LWKH9s26FkepoP3LBOY1G8KTphKiaSYK6Dnqg2Bgec-De6QEf6-5On9iJzbPlQzYNWIBvGR5Ku8f~QlCFf6SiLSv8K0w-loUAktaPJImCYY~blGwhmzGfqyHOw9uTCGcDm2WJ~46-feBuZYg1Mm~CAB8p3o4SxuKvbILYuwUHLMDYB6DDSZttSWmpR2l4-tVNkdOmUkdZy-TZytjKJ4IPIMZcLXSxkAzgigS2pH0oh6DqVC8F1hzEVvmxdWioJBakxVB~A1aT0b~6hmtBVycpjXI99YVtgPt2YtZD-0Hv9P1g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200964&ar=2462501&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02705.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>RSCU values of (A) Arabidopsis and (B) rice genes after splitting the data sets into 20 expression bins. Intergenic sequences were used for the construction of the ICCS data sets. Symbol codes: square, CS; circle, monoICCS; triangle, dinuICCS; diamond, trinuICCS; cross, cdcbICCS. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV027F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/m_dsv02705.png?Expires=2147483647&Signature=ujgCWN0Ur0QV~1f3TTsGGB5KZjJY8HHchhDd935I5Xg07HM06QXePLfbcIuiCiDCPHr-4c132VQ6wgK3xx0CLCbXkZvFTMs5YpIc1le2~oFXHCMpGpEL~nytE5j~ER614Q1JKANyvSD3ebP6Jjs1UF0gS~yrh5nEul2U7s0QjB1viuJej0NjHM3utmQ7Qv1HKovRYhmZhEQuPeIAkTx6A74m28nlrbYjkSqJ41rgwHldJqkGpCfn9NnrdzuX13~cXA5de3-MOASiyUxjW1LhB97omtAUwffIJ1LRROj3-dD-fk8-2NObWbcB-exPNCUuUKnzf0hqECZ8OT41Xdc59Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="RSCU values of (A) Arabidopsis and (B) rice genes after splitting the data sets into 20 expression bins. Intergenic sequences were used for the construction of the ICCS data sets. Symbol codes: square, CS; circle, monoICCS; triangle, dinuICCS; diamond, trinuICCS; cross, cdcbICCS. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02705.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200964/dsv02705.tiff" path-from-xml="dsv02705.tiff" target="_blank">View large</a><a section="87200964" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/dsv02705.png?Expires=2147483647&Signature=P3FnHFKq5ZG-8nyywKycvv927EdvmRdSJCy04ItH3LWKH9s26FkepoP3LBOY1G8KTphKiaSYK6Dnqg2Bgec-De6QEf6-5On9iJzbPlQzYNWIBvGR5Ku8f~QlCFf6SiLSv8K0w-loUAktaPJImCYY~blGwhmzGfqyHOw9uTCGcDm2WJ~46-feBuZYg1Mm~CAB8p3o4SxuKvbILYuwUHLMDYB6DDSZttSWmpR2l4-tVNkdOmUkdZy-TZytjKJ4IPIMZcLXSxkAzgigS2pH0oh6DqVC8F1hzEVvmxdWioJBakxVB~A1aT0b~6hmtBVycpjXI99YVtgPt2YtZD-0Hv9P1g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200964&ar=2462501&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02705.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>RSCU values of (A) Arabidopsis and (B) rice genes after splitting the data sets into 20 expression bins. Intergenic sequences were used for the construction of the ICCS data sets. Symbol codes: square, CS; circle, monoICCS; triangle, dinuICCS; diamond, trinuICCS; cross, cdcbICCS. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>We found that two-fold degenerate codons ending in G|C are more frequent in the coding sequences of rice and Arabidopsis, although in the latter case the aspartic acid codon GAC was an exception. There was no association between expression level and the RSCU values of the ICCSs, suggesting that expression-dependent variation in codon bias does not reflect differences in the background composition of the Arabidopsis genome. Weaker trends were observed in Arabidopsis when the four-fold degenerate codons were analysed both in terms of deviation from the background and variation with expression. Indeed, the frequency of many codons mirrored the background composition of the entire set of bins, so that the curves generated by the CS and ICCS data sets could be superimposed. However, there was a positive association between the RSCU values and expression levels of codons ending in C, and the opposite trend was observed for codons ending in A. Interestingly, the frequency of several codons ending in T was positively associated with the expression level, particularly the alanine codon CGT.</p><p>A more complex picture emerged from the analysis of rice genes. The correlation between RSCU values and gene expression was non-monotonic for codons with two-fold degeneracy, although as in Arabidopsis the codons ending in G|C were still found more frequently in the coding sequences. However, an initial reduction in the frequency of such codons was complemented by an increase in RSCU values at higher expression levels. If we exclude the effect of the background composition, which again does not vary with the expression level, such non-monotonic trends may reflect the coexistence of contrasting forces whose strength could be dependent on the expression level. Alternatively, the different expression bins in rice may be populated with genes that do not experience the same selective pressure. Indeed, it is well known that monocots have two classes of genes that differ in G + C content.<sup><a reveal-id="DSV027C13" data-open="DSV027C13" class="link link-ref link-reveal xref-bibr">13</a></sup> For this reason, the above analysis was repeated focusing specifically on LGC and HGC genes. The LGC genes accounted for most of the dataset and mirrored the overall trends discussed above, but the HGC genes showed a general positive association between the expression level and the RSCU values of two-fold degenerate codons ending in G|C.</p><p>In rice, four-fold degenerate codons ending in G|C were always used more frequently in the coding sequence than the ICCS, whereas the opposite trend was observed for codons ending in A|T. Furthermore, codons ending in G|C were generally used less frequently in strongly expressed genes, whereas there was a positive association between codons ending in T and expression level. As previously reported, such trends are often non-monotonic, either disappearing or changing direction when HGC and LGC genes are analysed separately (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/dsv027_Supplementary_Data.zip?Expires=2147483647&Signature=o4Yq-5GBDQo03KxFpLSJSloKTczliBPl7Y6bH5NKBv3MiWaj2LAb4Jv0ucEJ1yodO~9Ht7JfUypLsXNM0eCXlcX9AW7e91dJpdtfp-vCCatiBSJij7KQBsnEWod4wn2gj5PPJEYu9HEmeHgBY-ykpVHV2AbcRKp5uTC5rWlJRF40MwC5Yzz1c4PMdw5-3e-pNKqGcabhbMjVSCnHc6H~RCnEL5HoX--9XXZYh768xTuS00xjK94IgEoYJ3XO8G-9wJoCX~JVyjJLDKe1QiF1ojPBxub4hnpXbKH1dDyx-EG7u3TS82lkUWoBTmdVREYMrmkpaaDzk2l7X11lVlEW7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>).</p><p>The positive association between the RSCU value and expression level of codons ending in T is reminiscent of transcription-associated mutational bias (TAMB), a well-known repair system that increases bias towards G|T rather than C|A in the pre-mRNA sequence.<sup><a reveal-id="DSV027C34" data-open="DSV027C34" class="link link-ref link-reveal xref-bibr">34</a></sup> For this reason, we repeated our analysis by constructing a new set of ICCSs using intron sequences as a proxy for the background composition. If codons ending in T become more frequent in strongly expressed genes possibly due to TAMB, the same trend should be observed in intron-corrected coding sequences. Our results revealed no such association (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/dsv027_Supplementary_Data.zip?Expires=2147483647&Signature=o4Yq-5GBDQo03KxFpLSJSloKTczliBPl7Y6bH5NKBv3MiWaj2LAb4Jv0ucEJ1yodO~9Ht7JfUypLsXNM0eCXlcX9AW7e91dJpdtfp-vCCatiBSJij7KQBsnEWod4wn2gj5PPJEYu9HEmeHgBY-ykpVHV2AbcRKp5uTC5rWlJRF40MwC5Yzz1c4PMdw5-3e-pNKqGcabhbMjVSCnHc6H~RCnEL5HoX--9XXZYh768xTuS00xjK94IgEoYJ3XO8G-9wJoCX~JVyjJLDKe1QiF1ojPBxub4hnpXbKH1dDyx-EG7u3TS82lkUWoBTmdVREYMrmkpaaDzk2l7X11lVlEW7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span>), indicating that TAMB is not responsible for the positive association between gene expression level and the frequency of codons ending in T.</p>                            <h2 scrollto-destination=87200969 id="87200969" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>Several ICCS data sets were generated to investigate whether codon bias in the coding sequence reflected the genomic background nucleotide composition of the plant species included in this study. Four ICCS data sets constructed using different strategies were analysed to determine the effective number of codons and RSCU.</p>                            <h3 scrollto-destination=87200971 id="87200971" class="section-title" data-legacy-id=s4a>4.1. Background composition</h3>
<p>The RSCU values of the monoICCS data set revealed the over-representation of A|T compared with G|C although to a different extent in each species. The legumes and solanaceous species showed the greatest difference in the representation of A|T and G|C, whereas there was a weaker distinction in the monocot species, and the brassicas showed intermediate values. G|C to A|T mutations may be more frequent in Arabidopsis, leading to A|T enrichment in the genome.<sup><a reveal-id="DSV027C35" data-open="DSV027C35" class="link link-ref link-reveal xref-bibr">35</a></sup> The deamination of methylated cytosine residues at CG dinucleotide motifs and the UV-induced mutagenesis of dipyrimidines (CC and TC) were proposed as the main mechanisms to explain this phenomenon.<sup><a reveal-id="DSV027C35" data-open="DSV027C35" class="link link-ref link-reveal xref-bibr">35</a></sup> Several factors may account for the observed differences in the background nucleotide composition of plants. In monocots, the vertical leaf orientation, protective basal sheath and concealed apical meristem make the interception of solar radiation less efficient,<sup><a reveal-id="DSV027C36" data-open="DSV027C36" class="link link-ref link-reveal xref-bibr">36</a></sup> thus reducing the prevalence of AT enrichment induced by UV light. Domestication may also contribute to this bias, because A|T pairs contain seven nitrogen atoms compared with the eight present in G|C pairs, which may drive A|T enrichment in non-cultivated plants.<sup><a reveal-id="DSV027C37" data-open="DSV027C37" class="link link-ref link-reveal xref-bibr">37</a></sup> However, the cluster analysis of ICCS RSCU values revealed a pattern that is consistent with the genomic composition of plants solely depending on phylogeny (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/dsv027_Supplementary_Data.zip?Expires=2147483647&Signature=o4Yq-5GBDQo03KxFpLSJSloKTczliBPl7Y6bH5NKBv3MiWaj2LAb4Jv0ucEJ1yodO~9Ht7JfUypLsXNM0eCXlcX9AW7e91dJpdtfp-vCCatiBSJij7KQBsnEWod4wn2gj5PPJEYu9HEmeHgBY-ykpVHV2AbcRKp5uTC5rWlJRF40MwC5Yzz1c4PMdw5-3e-pNKqGcabhbMjVSCnHc6H~RCnEL5HoX--9XXZYh768xTuS00xjK94IgEoYJ3XO8G-9wJoCX~JVyjJLDKe1QiF1ojPBxub4hnpXbKH1dDyx-EG7u3TS82lkUWoBTmdVREYMrmkpaaDzk2l7X11lVlEW7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10</a></span>) rather than the contrast between wild and cultivated plants.</p><p>A strong dinucleotide compositional bias was evident in the ICCS data set of all species as shown by the higher levels of codon bias (i.e. lower Nc values) in the dinuICCS data set compared with the others (Fig. <a reveal-id="DSV027F2" data-open="DSV027F2" class="link link-reveal link-table xref-fig">2</a>). We found that codons ending in CG were among the most under-represented in the dinuICCS data set, whereas codons ending in CA (the reverse complement of the C deamination product in CG) were among the most over-represented, as recently reported in Arabidopsis and rice.<sup><a reveal-id="DSV027C31" data-open="DSV027C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Codons ending in CC and TC were also suppressed in the dinuICCS data set, confirming the under-representation of these dinucleotides in non-coding sequences.<sup><a reveal-id="DSV027C18" data-open="DSV027C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p>                            <h3 scrollto-destination=87200974 id="87200974" class="section-title" data-legacy-id=s4b>4.2. Differences between the CS and ICCS data sets</h3>
<p>The compositional pattern of several genomes has been investigated at different levels, including the whole genome and the genic portion.<sup><a reveal-id="DSV027C18" data-open="DSV027C18" class="link link-ref link-reveal xref-bibr">18</a></sup> However, to the best of our knowledge, this is the first time that coding and adjacent non-coding sequences have been directly compared in terms of the mononucleotide, dinucleotide and trinucleotide frequencies as well as CDCB. Paired Wilcoxon tests generally revealed significant differences in codon usage between the CS and ICCS data sets in terms of both the Nc and RSCU values. The observed differences were similar in magnitude when comparing the CS data set with all four methods for the construction of ICCS data sets. This finding validates the Hershberg and Petrov method for the identification of optimal codons in plants even though this only considers the mononucleotide composition of the background sequences. The optimal codon data sets calculated using this method could be almost perfectly superimposed over the data sets generated using the other three methods (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/dsv027_Supplementary_Data.zip?Expires=2147483647&Signature=o4Yq-5GBDQo03KxFpLSJSloKTczliBPl7Y6bH5NKBv3MiWaj2LAb4Jv0ucEJ1yodO~9Ht7JfUypLsXNM0eCXlcX9AW7e91dJpdtfp-vCCatiBSJij7KQBsnEWod4wn2gj5PPJEYu9HEmeHgBY-ykpVHV2AbcRKp5uTC5rWlJRF40MwC5Yzz1c4PMdw5-3e-pNKqGcabhbMjVSCnHc6H~RCnEL5HoX--9XXZYh768xTuS00xjK94IgEoYJ3XO8G-9wJoCX~JVyjJLDKe1QiF1ojPBxub4hnpXbKH1dDyx-EG7u3TS82lkUWoBTmdVREYMrmkpaaDzk2l7X11lVlEW7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S11</a></span>). The comparison between CS and ICCS allowed us to distinguish codon bias caused by genomic compositional features from codon bias related to the expressed portion of the genome.</p><p>The comparison of CS and ICCS data sets suggested that there was enrichment for codons ending with G or C particularly in monocots, although among the codons ending in G|C the frequency of the complementary codons GGG and CCC was close to the genomic background. This suggests the existence of selection against codons that promote the formation of complex mRNA tertiary structures, and the prevalence of this phenomenon in monocots with their higher overall G + C content may emphasize such an underlying mechanism.</p><p>Codons ACG, CCG and GCG were marginally over-represented in the coding sequences of some species and under-represented in others. This supports the observation that the CG dinucleotide is suppressed, in part due to deleterious methylation/deamination events. However, our analysis suggested that this phenomenon reflects an underlying genomic tendency, because the effects were not seen when comparing the CS and dinuICCS data sets (Fig. <a reveal-id="DSV027F4" data-open="DSV027F4" class="link link-reveal link-table xref-fig">4</a>).</p><p>Finally, the general suppression of codon GTA was one of the most conserved features among the plant species we investigated. This is not surprising, because the codon ends with dinucleotide TA, which is known to be suppressed in the coding sequences of several plant species<sup><a reveal-id="DSV027C18" data-open="DSV027C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSV027C31" data-open="DSV027C31" class="link link-ref link-reveal xref-bibr">31</a></sup> possibly to discourage insertion events that target the TA dinucleotide,<sup><a reveal-id="DSV027C38" data-open="DSV027C38" class="link link-ref link-reveal xref-bibr">38</a></sup> to reduce the likelihood of mutations leading to stop codons and to prevent attacks by TA-specific RNases.<sup><a reveal-id="DSV027C39" data-open="DSV027C39" class="link link-ref link-reveal xref-bibr">39</a></sup> The less striking divergence between the CS and dinuICCS data sets in terms of GTA preference suggests that TA suppression is also a general genomic signature (Fig. <a reveal-id="DSV027F3" data-open="DSV027F3" class="link link-reveal link-table xref-fig">3</a>).</p><p>We also demonstrated that the differences between the CS and ICCS vary along the transcript (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/dsv027_Supplementary_Data.zip?Expires=2147483647&Signature=o4Yq-5GBDQo03KxFpLSJSloKTczliBPl7Y6bH5NKBv3MiWaj2LAb4Jv0ucEJ1yodO~9Ht7JfUypLsXNM0eCXlcX9AW7e91dJpdtfp-vCCatiBSJij7KQBsnEWod4wn2gj5PPJEYu9HEmeHgBY-ykpVHV2AbcRKp5uTC5rWlJRF40MwC5Yzz1c4PMdw5-3e-pNKqGcabhbMjVSCnHc6H~RCnEL5HoX--9XXZYh768xTuS00xjK94IgEoYJ3XO8G-9wJoCX~JVyjJLDKe1QiF1ojPBxub4hnpXbKH1dDyx-EG7u3TS82lkUWoBTmdVREYMrmkpaaDzk2l7X11lVlEW7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>), although the background composition cannot explain the observed codon bias in all three portions of the coding sequence that were analysed individually. Differences between CS and ICCS were stronger in the first portion of the transcript in agreement with previous studies reporting stronger selection in the proximal region of the transcript.<sup><a reveal-id="DSV027C40" data-open="DSV027C40" class="link link-ref link-reveal xref-bibr">40</a></sup></p>                            <h3 scrollto-destination=87200980 id="87200980" class="section-title" data-legacy-id=s4c>4.3. Mutational bias in coding sequences</h3>
<p>The background nucleotide pattern alone cannot explain the observed codon bias in the coding sequences, so additional forces must be involved. Indeed, modestly frequent amino acids (or under-represented amino acids in the proteins encoded by strongly expressed genes) may be under weaker selection, and for this reason, their codons may better tolerate changes driven by mutational bias. For example, histidine and cysteine are among the less abundant amino acids in the Arabidopsis proteome, and there is a significant negative correlation between their frequency and the expression level of the corresponding genes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/dsv027_Supplementary_Data.zip?Expires=2147483647&Signature=o4Yq-5GBDQo03KxFpLSJSloKTczliBPl7Y6bH5NKBv3MiWaj2LAb4Jv0ucEJ1yodO~9Ht7JfUypLsXNM0eCXlcX9AW7e91dJpdtfp-vCCatiBSJij7KQBsnEWod4wn2gj5PPJEYu9HEmeHgBY-ykpVHV2AbcRKp5uTC5rWlJRF40MwC5Yzz1c4PMdw5-3e-pNKqGcabhbMjVSCnHc6H~RCnEL5HoX--9XXZYh768xTuS00xjK94IgEoYJ3XO8G-9wJoCX~JVyjJLDKe1QiF1ojPBxub4hnpXbKH1dDyx-EG7u3TS82lkUWoBTmdVREYMrmkpaaDzk2l7X11lVlEW7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). The frequency of codons CAC (His) and TGC (Cys) in Arabidopsis is similar in the CS and ICCS data sets (Fig. <a reveal-id="DSV027F5" data-open="DSV027F5" class="link link-reveal link-table xref-fig">5</a>). The codons GAC (Asp), GGC (Gly) and GCC (Ala) also revealed biases that mirrored the background composition. Interestingly, these codons feature the generic sequence CNG that has been proposed as the minimal building block for globular proteins and, for this reason, may represent the core of the primitive genetic code.<sup><a reveal-id="DSV027C41" data-open="DSV027C41" class="link link-ref link-reveal xref-bibr">41</a></sup> Such a trend may therefore reflect the more prolonged effect of mutational bias on the most ancient codons. In contrast, there appeared to be little mutational bias in monocots, where other factors increase the frequency codons ending in G|C.</p>                            <h3 scrollto-destination=87200982 id="87200982" class="section-title" data-legacy-id=s4d>4.4. Translational selection and mRNA stability</h3>
<p>Genes were assigned to 20 expression bins whose average RSCU values were plotted for the CS data set and all four ICCS data sets for Arabidopsis and rice, representing the eudicots and monocots, respectively (Fig. <a reveal-id="DSV027F5" data-open="DSV027F5" class="link link-reveal link-table xref-fig">5</a>). ICCS RSCU values were not associated with the expression level, suggesting that RSCU variation within coding sequences cannot be explained by the background composition.</p><p>In Arabidopsis, we observed a positive correlation between gene expression level and the frequency of optimal codons<sup><a reveal-id="DSV027C42" data-open="DSV027C42" class="link link-ref link-reveal xref-bibr">42</a>,<a reveal-id="DSV027C43" data-open="DSV027C43" class="link link-ref link-reveal xref-bibr">43</a></sup> ending in G|C, mainly for genes assigned to the last few expression bins, underlining the marginal effect of translational selection in this species.<sup><a reveal-id="DSV027C42" data-open="DSV027C42" class="link link-ref link-reveal xref-bibr">42</a></sup> A more complex scenario emerged in rice, where non-monotonic trends were observed for codons ending in G|C, i.e. a reduction in frequency in the first expression bins changing to an increase in frequency in the last few. This behaviour may be typical of G + C rich monocot genomes and may indicate a compromise between the advantage of using optimal codons and the avoidance of tightly packed mRNA tertiary structures. Indeed a significant negative correlation was found between the expression level and both the coding sequence length and the G + C content of rice genes.<sup><a reveal-id="DSV027C44" data-open="DSV027C44" class="link link-ref link-reveal xref-bibr">44</a></sup> Taken together, these data suggest that weakly expressed genes are longer and have a lower G + C content at the third codon position than strongly expressed genes. Longer transcripts are more likely to form strongly packed mRNA tertiary structures if they are enriched in optimal codons ending in G|C, so the accumulation of such codons may be counterselected in genes expressed at low or moderate levels. Such a trend would diminish in shorter genes expressed at high levels. This hypothesis was confirmed by analysing HGC and LGC rice genes separately. We found that short HGC genes were generally characterized by a monotonic positive association between the expression level and the frequency of optimal codons ending in G|C.</p>                            <h3 scrollto-destination=87200985 id="87200985" class="section-title" data-legacy-id=s4e>4.5. Additional factors</h3>
<p>TAMB is a well-known DNA repair system that influences the composition of primary transcripts (both exons and introns) by increasing the prevalence of G|T over C|A. Despite this general G + C enrichment, we observed the enrichment of four-fold degenerate codons ending in T, suggesting that TAMB may affect codon bias in Arabidopsis and rice, so we repeated our analysis using introns rather than intergenic DNA as an index of background composition. However, we found no differences in the frequency of codons ending in T when we used introns rather than intergenic DNA (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/dsv027_Supplementary_Data.zip?Expires=2147483647&Signature=o4Yq-5GBDQo03KxFpLSJSloKTczliBPl7Y6bH5NKBv3MiWaj2LAb4Jv0ucEJ1yodO~9Ht7JfUypLsXNM0eCXlcX9AW7e91dJpdtfp-vCCatiBSJij7KQBsnEWod4wn2gj5PPJEYu9HEmeHgBY-ykpVHV2AbcRKp5uTC5rWlJRF40MwC5Yzz1c4PMdw5-3e-pNKqGcabhbMjVSCnHc6H~RCnEL5HoX--9XXZYh768xTuS00xjK94IgEoYJ3XO8G-9wJoCX~JVyjJLDKe1QiF1ojPBxub4hnpXbKH1dDyx-EG7u3TS82lkUWoBTmdVREYMrmkpaaDzk2l7X11lVlEW7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span>).</p><p>The increasing G + C content of plant genomes may have been driven by G + C-biased gene conversion (gBGC) during double-strand break repair followed by a recombination event.<sup><a reveal-id="DSV027C45" data-open="DSV027C45" class="link link-ref link-reveal xref-bibr">45</a></sup> This phenomenon is accompanied by the correction of eventual mismatches between two paired DNA strands featuring high-sequence similarity, with such a correction being biased towards the placement of either G or C. Given that gBGC is not restricted to genes, the differences in codon bias between the CS and ICCS data sets would not highlight gBGC events. However, synonymous substitutions caused by gBGC are more likely to be fixed in the coding sequence, because codons ending in G|C are known to mirror the most abundant tRNAs in most plant species. There is also evidence that recombination in plant genomes occurs mainly within genes.<sup><a reveal-id="DSV027C46" data-open="DSV027C46" class="link link-ref link-reveal xref-bibr">46</a></sup></p><p>Our data provided several lines of evidence supporting the occurrence of gBGC in plants. Indeed if gBGC rather than selection is considered to be the main source of G|C enrichment at the third codon position, such an effect should be evident in all the transcripts regardless of the expression level. As shown in Fig. <a reveal-id="DSV027F5" data-open="DSV027F5" class="link link-reveal link-table xref-fig">5</a>, this was the case for the majority of the codons and expression bins in both Arabidopsis and rice, with the latter featuring wider divergence from the background in accordance with previous reports of gBGC in grasses.<sup><a reveal-id="DSV027C47" data-open="DSV027C47" class="link link-ref link-reveal xref-bibr">47</a></sup></p>                            <h2 scrollto-destination=87200989 id="87200989" class="section-title" data-legacy-id=s6>Funding</h2>
<p>We would like to thank the Università di Sassari for funding. Funding to pay the Open Access publication charges for this article was also provided by the Università di Sassari.</p>                            <h2 scrollto-destination=87200991 id="87200991" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSV027C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Chamary</div>
               <div class="given-names">J.V.</div>
            </div>, <div class="name string-name"><div class="surname">Parmley</div> <div class="given-names">J.L.</div></div>, <div class="name string-name"><div class="surname">Hurst</div> <div class="given-names">L.D.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">Hearing silence: non-neutral evolution at synonymous sites in mammals</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">7</div>, <div class="fpage">98</div>–<div class="lpage">108</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hearing%20silence%3A%20non-neutral%20evolution%20at%20synonymous%20sites%20in%20mammals&amp;author=J.V.%20Chamary&amp;author=J.L.%20Parmley&amp;author=L.D.%20Hurst&amp;publication_year=2006&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=7&amp;pages=98-108" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1770" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1770" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1770"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16418745" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Duret</div>
               <div class="given-names">L.</div>
            </div>
          </div>
          <div class="year">2002</div>, <div class="article-title">Evolution of synonymous codon usage in metazoans</div>, <div class="source">Curr. Opin. Genet. Dev.</div> , <div class="volume">12</div>, <div class="fpage">640</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20synonymous%20codon%20usage%20in%20metazoans&amp;author=L.%20Duret&amp;publication_year=2002&amp;journal=Curr.%20Opin.%20Genet.%20Dev.&amp;volume=12&amp;pages=640-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0959-437X(02)00353-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0959-437X(02)00353-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0959-437X(02)00353-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12433576" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Plotkin</div>
               <div class="given-names">J.B.</div>
            </div>, <div class="name string-name"><div class="surname">Dushoff</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Desai</div> <div class="given-names">M.M.</div></div>, <div class="name string-name"><div class="surname">Fraser</div> <div class="given-names">H.B.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">Codon usage and selection on proteins</div>, <div class="source">J. Mol. Evol.</div> , <div class="volume">63</div>, <div class="fpage">635</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20and%20selection%20on%20proteins&amp;author=J.B.%20Plotkin&amp;author=J.%20Dushoff&amp;author=M.M.%20Desai&amp;author=H.B.%20Fraser&amp;publication_year=2006&amp;journal=J.%20Mol.%20Evol.&amp;volume=63&amp;pages=635-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00239-005-0233-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00239-005-0233-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00239-005-0233-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17043750" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Plotkin</div>
               <div class="given-names">J.B.</div>
            </div>, <div class="name string-name"><div class="surname">Kudla</div> <div class="given-names">G.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Synonymous but not the same: the causes and consequences of codon bias</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">12</div>, <div class="fpage">32</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synonymous%20but%20not%20the%20same%3A%20the%20causes%20and%20consequences%20of%20codon%20bias&amp;author=J.B.%20Plotkin&amp;author=G.%20Kudla&amp;publication_year=2011&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=12&amp;pages=32-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2899" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2899" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2899"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21102527" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ikemura</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="year">1981</div>, <div class="article-title">Correlation between the abundance of Escherichia coli transfer RNAs and the occurrence of the respective codons in its protein genes: a proposal for a synonymous codon choice that is optimal for the E. coli translational system</div>, <div class="source">J. Mol. Biol.</div> , <div class="volume">151</div>, <div class="fpage">389</div>–<div class="lpage">409</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Correlation%20between%20the%20abundance%20of%20Escherichia%20coli%20transfer%20RNAs%20and%20the%20occurrence%20of%20the%20respective%20codons%20in%20its%20protein%20genes%3A%20a%20proposal%20for%20a%20synonymous%20codon%20choice%20that%20is%20optimal%20for%20the%20E.%20coli%20translational%20system&amp;author=T.%20Ikemura&amp;publication_year=1981&amp;journal=J.%20Mol.%20Biol.&amp;volume=151&amp;pages=389-409" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(81)90003-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(81)90003-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(81)90003-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6175758" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Akashi</div>
               <div class="given-names">H.</div>
            </div>
          </div>
          <div class="year">1994</div>, <div class="article-title">Synonymous codon usage in Drosophila melanogaster: natural selection and translational accuracy</div>, <div class="source">Genetics</div> , <div class="volume">136</div>, <div class="fpage">927</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synonymous%20codon%20usage%20in%20Drosophila%20melanogaster%3A%20natural%20selection%20and%20translational%20accuracy&amp;author=H.%20Akashi&amp;publication_year=1994&amp;journal=Genetics&amp;volume=136&amp;pages=927-35" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8005445" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bulmer</div>
               <div class="given-names">M.</div>
            </div>
          </div>
          <div class="year">1991</div>, <div class="article-title">The selection-mutation-drift theory of synonymous codon usage</div>, <div class="source">Genetics</div> , <div class="volume">129</div>, <div class="fpage">897</div>–<div class="lpage">907</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20selection-mutation-drift%20theory%20of%20synonymous%20codon%20usage&amp;author=M.%20Bulmer&amp;publication_year=1991&amp;journal=Genetics&amp;volume=129&amp;pages=897-907" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1752426" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sharp</div>
               <div class="given-names">P.M.</div>
            </div>, <div class="name string-name"><div class="surname">Cowe</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Higgins</div> <div class="given-names">D.G.</div></div>, <div class="name string-name"><div class="surname">Shields</div> <div class="given-names">D.C.</div></div>, <div class="name string-name"><div class="surname">Wolfe</div> <div class="given-names">K.H.</div></div>, <div class="name string-name"><div class="surname">Wright</div> <div class="given-names">F.</div></div></div>
          <div class="year">1988</div>, <div class="article-title">Codon usage patterns in Escherichia coli, Bacillus subtilis, Saccharomyces cerevisiae, Schizosaccharomyces pombe, Drosophila melanogaster and Homo sapiens; a review of the considerable within-species diversity</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">16</div>, <div class="fpage">8207</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20patterns%20in%20Escherichia%20coli%2C%20Bacillus%20subtilis%2C%20Saccharomyces%20cerevisiae%2C%20Schizosaccharomyces%20pombe%2C%20Drosophila%20melanogaster%20and%20Homo%20sapiens%3B%20a%20review%20of%20the%20considerable%20within-species%20diversity&amp;author=P.M.%20Sharp&amp;author=E.%20Cowe&amp;author=D.G.%20Higgins&amp;author=D.C.%20Shields&amp;author=K.H.%20Wolfe&amp;author=F.%20Wright&amp;publication_year=1988&amp;journal=Nucleic%20Acids%20Res.&amp;volume=16&amp;pages=8207-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/16.17.8207" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F16.17.8207" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F16.17.8207"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3138659" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Najafabadi</div>
               <div class="given-names">H.S.</div>
            </div>, <div class="name string-name"><div class="surname">Goodarzi</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Salavati</div> <div class="given-names">R.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Universal function-specificity of codon usage</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">37</div>, <div class="fpage">7014</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Universal%20function-specificity%20of%20codon%20usage&amp;author=H.S.%20Najafabadi&amp;author=H.%20Goodarzi&amp;author=R.%20Salavati&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=7014-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkp792" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkp792" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkp792"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19773421" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Camiolo</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Farina</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Porceddu</div> <div class="given-names">A.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">The relation of codon bias to tissue-specific gene expression in Arabidopsis thaliana</div>, <div class="source">Genetics</div> , <div class="volume">192</div>, <div class="fpage">641</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20relation%20of%20codon%20bias%20to%20tissue-specific%20gene%20expression%20in%20Arabidopsis%20thaliana&amp;author=S.%20Camiolo&amp;author=L.%20Farina&amp;author=A.%20Porceddu&amp;publication_year=2012&amp;journal=Genetics&amp;volume=192&amp;pages=641-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.112.143677" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.112.143677" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.112.143677"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22865738" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wong</div>
               <div class="given-names">G.K.-S.</div>
            </div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Tao</div> <div class="given-names">L.</div></div>et al.  </div>. <div class="year">2002</div>, <div class="article-title">Compositional gradients in Gramineae genes</div>, <div class="source">Genome Res.</div> , <div class="volume">12</div>, <div class="fpage">851</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Compositional%20gradients%20in%20Gramineae%20genes&amp;author=G.K.-S.%20Wong&amp;author=J.%20Wang&amp;author=L.%20Tao&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=851-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.189102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.189102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.189102"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12045139" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bernardi</div>
               <div class="given-names">G.</div>
            </div>, <div class="name string-name"><div class="surname">Olofsson</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Filipski</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">1985</div>, <div class="article-title">The mosaic genome of warm-blooded vertebrates</div>, <div class="source">Science</div> , <div class="volume">228</div>, <div class="fpage">953</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20mosaic%20genome%20of%20warm-blooded%20vertebrates&amp;author=G.%20Bernardi&amp;author=B.%20Olofsson&amp;author=J.%20Filipski&amp;publication_year=1985&amp;journal=Science&amp;volume=228&amp;pages=953-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.4001930" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.4001930" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.4001930"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4001930" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Carels</div>
               <div class="given-names">N.</div>
            </div>, <div class="name string-name"><div class="surname">Bernardi</div> <div class="given-names">G.</div></div></div>
          <div class="year">2000</div>, <div class="article-title">Two classes of genes in plants</div>, <div class="source">Genetics</div> , <div class="volume">154</div>, <div class="fpage">1819</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Two%20classes%20of%20genes%20in%20plants&amp;author=N.%20Carels&amp;author=G.%20Bernardi&amp;publication_year=2000&amp;journal=Genetics&amp;volume=154&amp;pages=1819-25" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10747072" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Liu</div>
               <div class="given-names">Q.</div>
            </div>
          </div>
          <div class="year">2012</div>, <div class="article-title">Mutational bias and translational selection shaping the codon usage pattern of tissue-specific genes in rice</div>. <div class="comment">Robinson-Rechavi, M., (ed.)</div>, <div class="source">PLoS One</div> , <div class="volume">7</div>, <div class="fpage">e48295</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutational%20bias%20and%20translational%20selection%20shaping%20the%20codon%20usage%20pattern%20of%20tissue-specific%20genes%20in%20rice&amp;author=Q.%20Liu&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e48295" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0048295" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0048295" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0048295"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23144748" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Berg</div>
               <div class="given-names">O.G.</div>
            </div>, <div class="name string-name"><div class="surname">Silva</div> <div class="given-names">P.J.</div></div></div>
          <div class="year">1997</div>, <div class="article-title">Codon bias in Escherichia coli: the influence of codon context on mutation and selection</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">25</div>, <div class="fpage">1397</div>–<div class="lpage">404</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20bias%20in%20Escherichia%20coli%3A%20the%20influence%20of%20codon%20context%20on%20mutation%20and%20selection&amp;author=O.G.%20Berg&amp;author=P.J.%20Silva&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=1397-404" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.7.1397" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.7.1397" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.7.1397"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9060435" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">De Amicis</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Marchetti</div> <div class="given-names">S.</div></div></div>
          <div class="year">2000</div>, <div class="article-title">Intercodon dinucleotides affect codon choice in plant genes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">28</div>, <div class="fpage">3339</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intercodon%20dinucleotides%20affect%20codon%20choice%20in%20plant%20genes&amp;author=F.%20De%20Amicis&amp;author=S.%20Marchetti&amp;publication_year=2000&amp;journal=Nucleic%20Acids%20Res.&amp;volume=28&amp;pages=3339-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/28.17.3339" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F28.17.3339" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F28.17.3339"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10954603" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Karlin</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Mrázek</div> <div class="given-names">J.</div></div></div>
          <div class="year">1996</div>, <div class="article-title">What drives codon choices in human genes?</div><div class="source">J. Mol. Biol.</div> , <div class="volume">262</div>, <div class="fpage">459</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=What%20drives%20codon%20choices%20in%20human%20genes%3F&amp;author=S.%20Karlin&amp;author=J.%20Mr%C3%A1zek&amp;publication_year=1996&amp;journal=J.%20Mol.%20Biol.&amp;volume=262&amp;pages=459-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.1996.0528" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.1996.0528" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.1996.0528"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8893856" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Porceddu</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Camiolo</div> <div class="given-names">S.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Spatial analyses of mono, di and trinucleotide trends in plant genes</div>, <div class="source">PLoS One</div> , <div class="volume">6</div>, <div class="fpage">e22855</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Spatial%20analyses%20of%20mono%2C%20di%20and%20trinucleotide%20trends%20in%20plant%20genes&amp;author=A.%20Porceddu&amp;author=S.%20Camiolo&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e22855" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0022855" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0022855" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0022855"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21829660" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hershberg</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Petrov</div> <div class="given-names">D.A.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">General rules for optimal codon choice</div>, <div class="source">PLoS Genet.</div> , <div class="volume">5</div>, <div class="fpage">e1000556</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=General%20rules%20for%20optimal%20codon%20choice&amp;author=R.%20Hershberg&amp;author=D.A.%20Petrov&amp;publication_year=2009&amp;journal=PLoS%20Genet.&amp;volume=5&amp;pages=e1000556" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000556" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000556" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000556"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19593368" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Morton</div>
               <div class="given-names">B.R.</div>
            </div>, <div class="name string-name"><div class="surname">Wright</div> <div class="given-names">S.I.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">Selective constraints on codon usage of nuclear genes from Arabidopsis thaliana</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="volume">24</div>, <div class="fpage">122</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selective%20constraints%20on%20codon%20usage%20of%20nuclear%20genes%20from%20Arabidopsis%20thaliana&amp;author=B.R.%20Morton&amp;author=S.I.%20Wright&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=24&amp;pages=122-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msl139" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsl139" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsl139"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17021276" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Goodstein</div>
               <div class="given-names">D.M.</div>
            </div>, <div class="name string-name"><div class="surname">Shu</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Howson</div> <div class="given-names">R.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Phytozome: a comparative platform for green plant genomics</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">40</div>, <div class="fpage">D1178</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phytozome%3A%20a%20comparative%20platform%20for%20green%20plant%20genomics&amp;author=D.M.%20Goodstein&amp;author=S.%20Shu&amp;author=R.%20Howson&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=D1178-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr944" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr944" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr944"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22110026" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Camiolo</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Porceddu</div> <div class="given-names">A.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">gff2sequence, a new user friendly tool for the generation of genomic sequences</div>, <div class="source">BioData Min.</div> , <div class="volume">6</div>, <div class="fpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=gff2sequence%2C%20a%20new%20user%20friendly%20tool%20for%20the%20generation%20of%20genomic%20sequences&amp;author=S.%20Camiolo&amp;author=A.%20Porceddu&amp;publication_year=2013&amp;journal=BioData%20Min.&amp;volume=6&amp;pages=15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1756-0381-6-15" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1756-0381-6-15" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1756-0381-6-15"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24020993" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dash</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Van Hemert</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Hong</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Wise</div> <div class="given-names">R.P.</div></div>, <div class="name string-name"><div class="surname">Dickerson</div> <div class="given-names">J.A.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">PLEXdb: gene expression resources for plants and plant pathogens</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">40</div>, <div class="fpage">D1194</div>–<div class="lpage">201</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PLEXdb%3A%20gene%20expression%20resources%20for%20plants%20and%20plant%20pathogens&amp;author=S.%20Dash&amp;author=J.%20Van%20Hemert&amp;author=L.%20Hong&amp;author=R.P.%20Wise&amp;author=J.A.%20Dickerson&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=D1194-201" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr938" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr938" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr938"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22084198" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Camiolo</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Rau</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Porceddu</div> <div class="given-names">A.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Mutational biases and selective forces shaping the structure of Arabidopsis genes</div>, <div class="source">PLoS One</div> , <div class="volume">4</div>, <div class="fpage">e6356</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutational%20biases%20and%20selective%20forces%20shaping%20the%20structure%20of%20Arabidopsis%20genes&amp;author=S.%20Camiolo&amp;author=D.%20Rau&amp;author=A.%20Porceddu&amp;publication_year=2009&amp;journal=PLoS%20One&amp;volume=4&amp;pages=e6356" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0006356" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0006356" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0006356"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19633720" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wright</div>
               <div class="given-names">F.</div>
            </div>
          </div>
          <div class="year">1990</div>, <div class="article-title">The “effective number of codons” used in a gene</div>, <div class="source">Gene</div> , <div class="volume">87</div>, <div class="fpage">23</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20%E2%80%9Ceffective%20number%20of%20codons%E2%80%9D%20used%20in%20a%20gene&amp;author=F.%20Wright&amp;publication_year=1990&amp;journal=Gene&amp;volume=87&amp;pages=23-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119(90)90491-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119(90)90491-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0378-1119(90)90491-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2110097" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sharp</div>
               <div class="given-names">P.M.</div>
            </div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">W.H.</div></div></div>
          <div class="year">1986</div>, <div class="article-title">An evolutionary perspective on synonymous codon usage in unicellular organisms</div>, <div class="source">J. Mol. Evol.</div> , <div class="volume">24</div>, <div class="fpage">28</div>–<div class="lpage">38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20evolutionary%20perspective%20on%20synonymous%20codon%20usage%20in%20unicellular%20organisms&amp;author=P.M.%20Sharp&amp;author=W.H.%20Li&amp;publication_year=1986&amp;journal=J.%20Mol.%20Evol.&amp;volume=24&amp;pages=28-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02099948" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02099948" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02099948"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3104616" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tuller</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Carmi</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Vestsigian</div> <div class="given-names">K.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">An evolutionarily conserved mechanism for controlling the efficiency of protein translation</div>, <div class="source">Cell</div> , <div class="volume">141</div>, <div class="fpage">344</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20evolutionarily%20conserved%20mechanism%20for%20controlling%20the%20efficiency%20of%20protein%20translation&amp;author=T.%20Tuller&amp;author=A.%20Carmi&amp;author=K.%20Vestsigian&amp;publication_year=2010&amp;journal=Cell&amp;volume=141&amp;pages=344-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2010.03.031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2010.03.031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2010.03.031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20403328" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ahmad</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Sablok</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Tatarinova</div> <div class="given-names">T.V.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Deng</div> <div class="given-names">X.-X.</div></div>, <div class="name string-name"><div class="surname">Guo</div> <div class="given-names">W.-W.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Evaluation of codon biology in citrus and Poncirus trifoliata based on genomic features and frame corrected expressed sequence tags</div>, <div class="source">DNA Res.</div> , <div class="volume">20</div>, <div class="fpage">135</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evaluation%20of%20codon%20biology%20in%20citrus%20and%20Poncirus%20trifoliata%20based%20on%20genomic%20features%20and%20frame%20corrected%20expressed%20sequence%20tags&amp;author=T.%20Ahmad&amp;author=G.%20Sablok&amp;author=T.V.%20Tatarinova&amp;author=Q.%20Xu&amp;author=X.-X.%20Deng&amp;author=W.-W.%20Guo&amp;publication_year=2013&amp;journal=DNA%20Res.&amp;volume=20&amp;pages=135-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dss039" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdss039" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdss039"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23315666" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Camiolo</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Melito</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Milia</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Porceddu</div> <div class="given-names">A.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">September 21, The ribosomial ramp and the regulation of translation in plants</div>, <div class="source">J. Bioinforma. Res. Stud.</div> , <div class="volume">1</div>, <div class="fpage">2</div>.</div></div></div></div><div content-id="DSV027C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Palidwor</div>
               <div class="given-names">G.A.</div>
            </div>, <div class="name string-name"><div class="surname">Perkins</div> <div class="given-names">T.J.</div></div>, <div class="name string-name"><div class="surname">Xia</div> <div class="given-names">X.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">A general model of codon bias due to GC mutational bias</div>, <div class="source">PLoS One</div> , <div class="volume">5</div>, <div class="fpage">e13431</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20general%20model%20of%20codon%20bias%20due%20to%20GC%20mutational%20bias&amp;author=G.A.%20Palidwor&amp;author=T.J.%20Perkins&amp;author=X.%20Xia&amp;publication_year=2010&amp;journal=PLoS%20One&amp;volume=5&amp;pages=e13431" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0013431" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0013431" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0013431"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21048949" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Karlin</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="year">1998</div>, <div class="article-title">Global dinucleotide signatures and analysis of genomic heterogeneity</div>, <div class="source">Curr. Opin. Microbiol.</div> , <div class="volume">1</div>, <div class="fpage">598</div>–<div class="lpage">610</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20dinucleotide%20signatures%20and%20analysis%20of%20genomic%20heterogeneity&amp;author=S.%20Karlin&amp;publication_year=1998&amp;journal=Curr.%20Opin.%20Microbiol.&amp;volume=1&amp;pages=598-610" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1369-5274(98)80095-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1369-5274(98)80095-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1369-5274(98)80095-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10066522" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nekrutenko</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">W.H.</div></div></div>
          <div class="year">2000</div>, <div class="article-title">Assessment of compositional heterogeneity within and between eukaryotic genomes</div>, <div class="source">Genome Res.</div> , <div class="volume">10</div>, <div class="fpage">1986</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessment%20of%20compositional%20heterogeneity%20within%20and%20between%20eukaryotic%20genomes&amp;author=A.%20Nekrutenko&amp;author=W.H.%20Li&amp;publication_year=2000&amp;journal=Genome%20Res.&amp;volume=10&amp;pages=1986-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.10.12.1986" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.10.12.1986" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.10.12.1986"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11116093" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Liu</div>
               <div class="given-names">Q.</div>
            </div>, <div class="name string-name"><div class="surname">Hu</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">H.</div></div></div>
          <div class="year">2015</div>, <div class="article-title">Mutational bias is the driving force for shaping the synonymous codon usage pattern of alternatively spliced genes in rice (Oryza sativa L.)</div>, <div class="source">Mol. Genet. Genomics</div> , <div class="volume">290</div>, <div class="fpage">649</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutational%20bias%20is%20the%20driving%20force%20for%20shaping%20the%20synonymous%20codon%20usage%20pattern%20of%20alternatively%20spliced%20genes%20in%20rice%20(Oryza%20sativa%20L.)&amp;author=Q.%20Liu&amp;author=H.%20Hu&amp;author=H.%20Wang&amp;publication_year=2015&amp;journal=Mol.%20Genet.%20Genomics&amp;volume=290&amp;pages=649-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00438-014-0951-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00438-014-0951-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00438-014-0951-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25407289" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Green</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Ewing</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Miller</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Thomas</div> <div class="given-names">P.J.</div></div>, <div class="name string-name"><div class="surname">Green</div> <div class="given-names">E.D.</div></div></div>
          <div class="year">2003</div>, <div class="article-title">Transcription-associated mutational asymmetry in mammalian evolution</div>, <div class="source">Nat. Genet.</div> , <div class="volume">33</div>, <div class="fpage">514</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription-associated%20mutational%20asymmetry%20in%20mammalian%20evolution&amp;author=P.%20Green&amp;author=B.%20Ewing&amp;author=W.%20Miller&amp;author=P.J.%20Thomas&amp;author=E.D.%20Green&amp;publication_year=2003&amp;journal=Nat.%20Genet.&amp;volume=33&amp;pages=514-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1103"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12612582" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ossowski</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Schneeberger</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Lucas-Lledó</div> <div class="given-names">J.I.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">The rate and molecular spectrum of spontaneous mutations in Arabidopsis thaliana</div>, <div class="source">Science</div> , <div class="volume">327</div>, <div class="fpage">92</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20rate%20and%20molecular%20spectrum%20of%20spontaneous%20mutations%20in%20Arabidopsis%20thaliana&amp;author=S.%20Ossowski&amp;author=K.%20Schneeberger&amp;author=J.I.%20Lucas-Lled%C3%B3&amp;publication_year=2010&amp;journal=Science&amp;volume=327&amp;pages=92-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1180677" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1180677" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1180677"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20044577" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cline</div>
               <div class="given-names">M.G.</div>
            </div>, <div class="name string-name"><div class="surname">Salisbury</div> <div class="given-names">F.B.</div></div></div>
          <div class="year">1966</div>, <div class="article-title">Effects of ultra-violet alone and simulated solar ultra-violet radiation on the leaves of higher plants</div>, <div class="source">Nature</div> , <div class="volume">211</div>, <div class="fpage">484</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effects%20of%20ultra-violet%20alone%20and%20simulated%20solar%20ultra-violet%20radiation%20on%20the%20leaves%20of%20higher%20plants&amp;author=M.G.%20Cline&amp;author=F.B.%20Salisbury&amp;publication_year=1966&amp;journal=Nature&amp;volume=211&amp;pages=484-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/211484a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F211484a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F211484a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5967488" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Günther</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Lampei</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Schmid</div> <div class="given-names">K.J.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Mutational bias and gene conversion affect the intraspecific nitrogen stoichiometry of the Arabidopsis thaliana transcriptome</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="volume">30</div>, <div class="fpage">561</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutational%20bias%20and%20gene%20conversion%20affect%20the%20intraspecific%20nitrogen%20stoichiometry%20of%20the%20Arabidopsis%20thaliana%20transcriptome&amp;author=T.%20G%C3%BCnther&amp;author=C.%20Lampei&amp;author=K.J.%20Schmid&amp;publication_year=2013&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=30&amp;pages=561-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/mss249" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmss249" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmss249"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23115321" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mashkova</div>
               <div class="given-names">T.D.</div>
            </div>, <div class="name string-name"><div class="surname">Oparina</div> <div class="given-names">N.Y.</div></div>, <div class="name string-name"><div class="surname">Lacroix</div> <div class="given-names">M.H.</div></div>et al.  </div>. <div class="year">2001</div>, <div class="article-title">Structural rearrangements and insertions of dispersed elements in pericentromeric alpha satellites occur preferably at kinkable DNA sites</div>, <div class="source">J. Mol. Biol.</div> , <div class="volume">305</div>, <div class="fpage">33</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20rearrangements%20and%20insertions%20of%20dispersed%20elements%20in%20pericentromeric%20alpha%20satellites%20occur%20preferably%20at%20kinkable%20DNA%20sites&amp;author=T.D.%20Mashkova&amp;author=N.Y.%20Oparina&amp;author=M.H.%20Lacroix&amp;publication_year=2001&amp;journal=J.%20Mol.%20Biol.&amp;volume=305&amp;pages=33-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.2000.4270" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.2000.4270" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.2000.4270"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11114245" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Beutler</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Gelbart</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Han</div> <div class="given-names">J.H.</div></div>, <div class="name string-name"><div class="surname">Koziol</div> <div class="given-names">J.A.</div></div>, <div class="name string-name"><div class="surname">Beutler</div> <div class="given-names">B.</div></div></div>
          <div class="year">1989</div>, <div class="article-title">Evolution of the genome and the genetic code: selection at the dinucleotide level by methylation and polyribonucleotide cleavage</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">86</div>, <div class="fpage">192</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20the%20genome%20and%20the%20genetic%20code%3A%20selection%20at%20the%20dinucleotide%20level%20by%20methylation%20and%20polyribonucleotide%20cleavage&amp;author=E.%20Beutler&amp;author=T.%20Gelbart&amp;author=J.H.%20Han&amp;author=J.A.%20Koziol&amp;author=B.%20Beutler&amp;publication_year=1989&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=86&amp;pages=192-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.86.1.192" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.86.1.192" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.86.1.192"> </span></div> </div></div></div></div></div><div content-id="DSV027C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Eyre-Walker</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Bulmer</div> <div class="given-names">M.</div></div></div>
          <div class="year">1993</div>, <div class="article-title">Reduced synonymous substitution rate at the start of enterobacterial genes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">21</div>, <div class="fpage">4599</div>–<div class="lpage">603</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reduced%20synonymous%20substitution%20rate%20at%20the%20start%20of%20enterobacterial%20genes&amp;author=A.%20Eyre-Walker&amp;author=M.%20Bulmer&amp;publication_year=1993&amp;journal=Nucleic%20Acids%20Res.&amp;volume=21&amp;pages=4599-603" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/21.19.4599" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F21.19.4599" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F21.19.4599"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8233796" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ikehara</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Omori</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Arai</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Hirose</div> <div class="given-names">A.</div></div></div>
          <div class="year">2002</div>, <div class="article-title">A novel theory on the origin of the genetic code: a GNC-SNS hypothesis</div>, <div class="source">J. Mol. Evol.</div> , <div class="volume">54</div>, <div class="fpage">530</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20theory%20on%20the%20origin%20of%20the%20genetic%20code%3A%20a%20GNC-SNS%20hypothesis&amp;author=K.%20Ikehara&amp;author=Y.%20Omori&amp;author=R.%20Arai&amp;author=A.%20Hirose&amp;publication_year=2002&amp;journal=J.%20Mol.%20Evol.&amp;volume=54&amp;pages=530-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00239-001-0053-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00239-001-0053-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00239-001-0053-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11956691" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wright</div>
               <div class="given-names">S.I.</div>
            </div>, <div class="name string-name"><div class="surname">Yau</div> <div class="given-names">C.B.K.</div></div>, <div class="name string-name"><div class="surname">Looseley</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Meyers</div> <div class="given-names">B.C.</div></div></div>
          <div class="year">2004</div>, <div class="article-title">Effects of gene expression on molecular evolution in Arabidopsis thaliana and Arabidopsis lyrata</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="volume">21</div>, <div class="fpage">1719</div>–<div class="lpage">26</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effects%20of%20gene%20expression%20on%20molecular%20evolution%20in%20Arabidopsis%20thaliana%20and%20Arabidopsis%20lyrata&amp;author=S.I.%20Wright&amp;author=C.B.K.%20Yau&amp;author=M.%20Looseley&amp;author=B.C.%20Meyers&amp;publication_year=2004&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=21&amp;pages=1719-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msh191" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsh191" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsh191"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15201397" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Porceddu</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Zenoni</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Camiolo</div> <div class="given-names">S.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">The signatures of selection for translational accuracy in plant genes</div>, <div class="source">Genome Biol. Evol.</div> , <div class="volume">5</div>, <div class="fpage">1117</div>–<div class="lpage">26</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20signatures%20of%20selection%20for%20translational%20accuracy%20in%20plant%20genes&amp;author=A.%20Porceddu&amp;author=S.%20Zenoni&amp;author=S.%20Camiolo&amp;publication_year=2013&amp;journal=Genome%20Biol.%20Evol.&amp;volume=5&amp;pages=1117-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evt078" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevt078" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevt078"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23695187" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yang</div>
               <div class="given-names">H.</div>
            </div>
          </div>
          <div class="year">2009</div>, <div class="article-title">In plants, expression breadth and expression level distinctly and non-linearly correlate with gene structure</div>, <div class="source">Biol. Direct.</div> , <div class="volume">4</div>, <div class="fpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20plants%2C%20expression%20breadth%20and%20expression%20level%20distinctly%20and%20non-linearly%20correlate%20with%20gene%20structure&amp;author=H.%20Yang&amp;publication_year=2009&amp;journal=Biol.%20Direct.&amp;volume=4&amp;pages=45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1745-6150-4-45" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1745-6150-4-45" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1745-6150-4-45"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19930585" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Galtier</div>
               <div class="given-names">N.</div>
            </div>, <div class="name string-name"><div class="surname">Piganeau</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Mouchiroud</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Duret</div> <div class="given-names">L.</div></div></div>
          <div class="year">2001</div>, <div class="article-title">GC-content evolution in mammalian genomes: the biased gene conversion hypothesis</div>, <div class="source">Genetics</div> , <div class="volume">159</div>, <div class="fpage">907</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GC-content%20evolution%20in%20mammalian%20genomes%3A%20the%20biased%20gene%20conversion%20hypothesis&amp;author=N.%20Galtier&amp;author=G.%20Piganeau&amp;author=D.%20Mouchiroud&amp;author=L.%20Duret&amp;publication_year=2001&amp;journal=Genetics&amp;volume=159&amp;pages=907-11" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11693127" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rafalski</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Morgante</div> <div class="given-names">M.</div></div></div>
          <div class="year">2004</div>, <div class="article-title">Corn and humans: recombination and linkage disequilibrium in two genomes of similar size</div>, <div class="source">Trends Genet.</div> , <div class="volume">20</div>, <div class="fpage">103</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Corn%20and%20humans%3A%20recombination%20and%20linkage%20disequilibrium%20in%20two%20genomes%20of%20similar%20size&amp;author=A.%20Rafalski&amp;author=M.%20Morgante&amp;publication_year=2004&amp;journal=Trends%20Genet.&amp;volume=20&amp;pages=103-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2003.12.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2003.12.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2003.12.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14746992" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV027C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV027C47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Serres-Giardi</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Belkhir</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">David</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Glémin</div> <div class="given-names">S.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Patterns and evolution of nucleotide landscapes in seed plants</div>, <div class="source">Plant Cell</div> , <div class="volume">24</div>, <div class="fpage">1379</div>–<div class="lpage">97</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Patterns%20and%20evolution%20of%20nucleotide%20landscapes%20in%20seed%20plants&amp;author=L.%20Serres-Giardi&amp;author=K.%20Belkhir&amp;author=J.%20David&amp;author=S.%20Gl%C3%A9min&amp;publication_year=2012&amp;journal=Plant%20Cell&amp;volume=24&amp;pages=1379-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.111.093674" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.111.093674" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.111.093674"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22492812" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2015. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="link link-uri" href="http://creativecommons.org/licenses/by/4.0/" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2462501&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv027/1/dsv027_Supplementary_Data.zip?Expires=2147483647&amp;Signature=o4Yq-5GBDQo03KxFpLSJSloKTczliBPl7Y6bH5NKBv3MiWaj2LAb4Jv0ucEJ1yodO~9Ht7JfUypLsXNM0eCXlcX9AW7e91dJpdtfp-vCCatiBSJij7KQBsnEWod4wn2gj5PPJEYu9HEmeHgBY-ykpVHV2AbcRKp5uTC5rWlJRF40MwC5Yzz1c4PMdw5-3e-pNKqGcabhbMjVSCnHc6H~RCnEL5HoX--9XXZYh768xTuS00xjK94IgEoYJ3XO8G-9wJoCX~JVyjJLDKe1QiF1ojPBxub4hnpXbKH1dDyx-EG7u3TS82lkUWoBTmdVREYMrmkpaaDzk2l7X11lVlEW7g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">263</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000366631700007&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">6</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsv027"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2462501"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000366631700007&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f6%2f461%2f2462501">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000366631700007&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (6)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F6%2F461%2F2462501" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2462501'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="New insights into the interplay between codon bias determinants in plants" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsv027");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f6%2f461%2f2462501">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

