

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":0,"applicationTime":1522,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/20/2/185/333880","@type":"ScholarlyArticle","name":"Gene Capture Coupled to High-Throughput Sequencing as a Strategy for Targeted Metagenome Exploration","datePublished":"2013-01-30","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/20/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2013-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dst001","keywords":["α-subunit of the methyl-coenzyme M reductase","metagenomics","sequence capture","454 pyrosequencing","microbial diversity"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/20/2/185/333880","author":[{"name":"Denonfoux, Jérémie","affiliation":"1  Centre de Recherche en Nutrition Humaine Auvergne ,  Clermont Université, Université d'Auvergne,  EA 4678, Conception, Ingénierie et Développement de l'Aliment et du Médicament, BP 10448, Clermont-Ferrand 63000 ,  France    2  Clermont Université, Université Blaise Pascal ,  Clermont-Ferrand 63000 ,  France    3  UMR CNRS 6023, Université Blaise Pascal ,  Clermont-Ferrand 63000 ,  France","@type":"Person"},{"name":"Parisot, Nicolas","affiliation":"1  Centre de Recherche en Nutrition Humaine Auvergne ,  Clermont Université, Université d'Auvergne,  EA 4678, Conception, Ingénierie et Développement de l'Aliment et du Médicament, BP 10448, Clermont-Ferrand 63000 ,  France    2  Clermont Université, Université Blaise Pascal ,  Clermont-Ferrand 63000 ,  France    3  UMR CNRS 6023, Université Blaise Pascal ,  Clermont-Ferrand 63000 ,  France","@type":"Person"},{"name":"Dugat-Bony, Eric","affiliation":"1  Centre de Recherche en Nutrition Humaine Auvergne ,  Clermont Université, Université d'Auvergne,  EA 4678, Conception, Ingénierie et Développement de l'Aliment et du Médicament, BP 10448, Clermont-Ferrand 63000 ,  France    4  UFR Pharmacie ,  Clermont Université, Université d'Auvergne,  Clermont-Ferrand 63000 ,  France","@type":"Person"},{"name":"Biderre-Petit, Corinne","affiliation":"3  UMR CNRS 6023, Université Blaise Pascal ,  Clermont-Ferrand 63000 ,  France    5  Laboratoire Microorganismes: Génome et Environnement ,  Clermont Université, Université Blaise Pascal ,  BP 10448, Clermont-Ferrand 63000 ,  France","@type":"Person"},{"name":"Boucher, Delphine","affiliation":"1  Centre de Recherche en Nutrition Humaine Auvergne ,  Clermont Université, Université d'Auvergne,  EA 4678, Conception, Ingénierie et Développement de l'Aliment et du Médicament, BP 10448, Clermont-Ferrand 63000 ,  France    4  UFR Pharmacie ,  Clermont Université, Université d'Auvergne,  Clermont-Ferrand 63000 ,  France","@type":"Person"},{"name":"Morgavi, Diego P.","affiliation":"6  INRA ,  UMR1213 Herbivores, F-63122 Saint-Genès-Champanelle and Clermont Université, VetAgro Sup, UMR Herbivores, BP 10448, F-63000, Clermont-Ferrand ,  France","@type":"Person"},{"name":"Le Paslier, Denis","affiliation":"7  CEA, DSV, Institut de Génomique ,  Genoscope, 2 rue Gaston Crémieux, Evry 91057 ,  France    8  CNRS, UMR8030 ,  Evry 91057 ,  France    9  UEVE, Université d'Evry ,  Evry 91057 ,  France","@type":"Person"},{"name":"Peyretaillade, Eric","affiliation":"1  Centre de Recherche en Nutrition Humaine Auvergne ,  Clermont Université, Université d'Auvergne,  EA 4678, Conception, Ingénierie et Développement de l'Aliment et du Médicament, BP 10448, Clermont-Ferrand 63000 ,  France    4  UFR Pharmacie ,  Clermont Université, Université d'Auvergne,  Clermont-Ferrand 63000 ,  France","@type":"Person"},{"name":"Peyret, Pierre","affiliation":"1  Centre de Recherche en Nutrition Humaine Auvergne ,  Clermont Université, Université d'Auvergne,  EA 4678, Conception, Ingénierie et Développement de l'Aliment et du Médicament, BP 10448, Clermont-Ferrand 63000 ,  France    4  UFR Pharmacie ,  Clermont Université, Université d'Auvergne,  Clermont-Ferrand 63000 ,  France","@type":"Person"}],"description":"Abstract.  Next-generation sequencing (NGS) allows faster acquisition of metagenomic data, but complete exploration of complex ecosystems is hindered by the ext","pageStart":"185","pageEnd":"196","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/2/1/m_cover.gif?Expires=2147483647&Signature=Yq7dbhq6hb-Iyo5ewEGt-Y1T99brR4Xh8FvVnCI2Bg1SIlPGztlGCk-Bm0A9A72SaqDKcetVlEQehNtxyAGT~T55ZFFD19A3hKk~A1XbnQ-JfXYKj2-gYHz-VirOlvLe-8XkC6qvT4h2PniinfIJH6d85JsIf3rhRRtN086VVGvi2qNVR0THPJOeMpQvElVQEEN-4bpo3wEYmq80uwX58y5ScJ0pcJt7sV40xw1Fd6mp92v-su9vXuGT~IIVjyHRDyuiGt~ZiUzcXtor6WX4MT1zcCCf5ySC3Wm7DfaZruIvVsbtJxHUYCwePrgmcTq1vhdeOWHam2svM-jBZyoDoA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Gene Capture Coupled to High-Throughput Sequencing as a Strategy for Targeted Metagenome Exploration","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/2/1/m_cover.gif?Expires=2147483647&Signature=Yq7dbhq6hb-Iyo5ewEGt-Y1T99brR4Xh8FvVnCI2Bg1SIlPGztlGCk-Bm0A9A72SaqDKcetVlEQehNtxyAGT~T55ZFFD19A3hKk~A1XbnQ-JfXYKj2-gYHz-VirOlvLe-8XkC6qvT4h2PniinfIJH6d85JsIf3rhRRtN086VVGvi2qNVR0THPJOeMpQvElVQEEN-4bpo3wEYmq80uwX58y5ScJ0pcJt7sV40xw1Fd6mp92v-su9vXuGT~IIVjyHRDyuiGt~ZiUzcXtor6WX4MT1zcCCf5ySC3Wm7DfaZruIvVsbtJxHUYCwePrgmcTq1vhdeOWHam2svM-jBZyoDoA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Gene Capture Coupled to High-Throughput Sequencing as a Strategy for Targeted Metagenome Exploration" />
    <meta property="og:description" content="Abstract.  Next-generation sequencing (NGS) allows faster acquisition of metagenomic data, but complete exploration of complex ecosystems is hindered by the ext" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/20/2/185/333880" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2013-01-30" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Denonfoux, Jérémie" /><meta name="citation_author_institution" content="Centre de Recherche en Nutrition Humaine Auvergne, Clermont Université, Université d'Auvergne,EA 4678, Conception, Ingénierie et Développement de l'Aliment et du Médicament, BP 10448, Clermont-Ferrand 63000, France" /><meta name="citation_author_institution" content="Clermont Université, Université Blaise Pascal, Clermont-Ferrand 63000, France" /><meta name="citation_author_institution" content="UMR CNRS 6023, Université Blaise Pascal, Clermont-Ferrand 63000, France" /><meta name="citation_author" content="Parisot, Nicolas" /><meta name="citation_author_institution" content="Centre de Recherche en Nutrition Humaine Auvergne, Clermont Université, Université d'Auvergne,EA 4678, Conception, Ingénierie et Développement de l'Aliment et du Médicament, BP 10448, Clermont-Ferrand 63000, France" /><meta name="citation_author_institution" content="Clermont Université, Université Blaise Pascal, Clermont-Ferrand 63000, France" /><meta name="citation_author_institution" content="UMR CNRS 6023, Université Blaise Pascal, Clermont-Ferrand 63000, France" /><meta name="citation_author" content="Dugat-Bony, Eric" /><meta name="citation_author_institution" content="Centre de Recherche en Nutrition Humaine Auvergne, Clermont Université, Université d'Auvergne,EA 4678, Conception, Ingénierie et Développement de l'Aliment et du Médicament, BP 10448, Clermont-Ferrand 63000, France" /><meta name="citation_author_institution" content="UFR Pharmacie, Clermont Université, Université d'Auvergne,Clermont-Ferrand 63000, France" /><meta name="citation_author" content="Biderre-Petit, Corinne" /><meta name="citation_author_institution" content="UMR CNRS 6023, Université Blaise Pascal, Clermont-Ferrand 63000, France" /><meta name="citation_author_institution" content="Laboratoire Microorganismes: Génome et Environnement, Clermont Université, Université Blaise Pascal, BP 10448, Clermont-Ferrand 63000, France" /><meta name="citation_author" content="Boucher, Delphine" /><meta name="citation_author_institution" content="Centre de Recherche en Nutrition Humaine Auvergne, Clermont Université, Université d'Auvergne,EA 4678, Conception, Ingénierie et Développement de l'Aliment et du Médicament, BP 10448, Clermont-Ferrand 63000, France" /><meta name="citation_author_institution" content="UFR Pharmacie, Clermont Université, Université d'Auvergne,Clermont-Ferrand 63000, France" /><meta name="citation_author" content="Morgavi, Diego P." /><meta name="citation_author_institution" content="INRA, UMR1213 Herbivores, F-63122 Saint-Genès-Champanelle and Clermont Université, VetAgro Sup, UMR Herbivores, BP 10448, F-63000, Clermont-Ferrand, France" /><meta name="citation_author" content="Le Paslier, Denis" /><meta name="citation_author_institution" content="CEA, DSV, Institut de Génomique, Genoscope, 2 rue Gaston Crémieux, Evry 91057, France" /><meta name="citation_author_institution" content="CNRS, UMR8030, Evry 91057, France" /><meta name="citation_author_institution" content="UEVE, Université d'Evry, Evry 91057, France" /><meta name="citation_author" content="Peyretaillade, Eric" /><meta name="citation_author_institution" content="Centre de Recherche en Nutrition Humaine Auvergne, Clermont Université, Université d'Auvergne,EA 4678, Conception, Ingénierie et Développement de l'Aliment et du Médicament, BP 10448, Clermont-Ferrand 63000, France" /><meta name="citation_author_institution" content="UFR Pharmacie, Clermont Université, Université d'Auvergne,Clermont-Ferrand 63000, France" /><meta name="citation_author" content="Peyret, Pierre" /><meta name="citation_author_institution" content="Centre de Recherche en Nutrition Humaine Auvergne, Clermont Université, Université d'Auvergne,EA 4678, Conception, Ingénierie et Développement de l'Aliment et du Médicament, BP 10448, Clermont-Ferrand 63000, France" /><meta name="citation_author_institution" content="UFR Pharmacie, Clermont Université, Université d'Auvergne,Clermont-Ferrand 63000, France" /><meta name="citation_title" content="Gene Capture Coupled to High-Throughput Sequencing as a Strategy for Targeted Metagenome Exploration" /><meta name="citation_firstpage" content="185" /><meta name="citation_lastpage" content="196" /><meta name="citation_doi" content="10.1093/dnares/dst001" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="20" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2013/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Prokaryotes: the unseen majority; Proc. Natl. Acad. Sci. USA;  citation_year=1998;  citation_volume=95;  citation_pages=6578-83; " /><meta name="citation_reference" content="citation_title=What is the extent of prokaryotic diversity?; Philos. Trans. R. Soc. Lond. B. Biol. Sci.;  citation_year=2006;  citation_volume=361;  citation_pages=2023-37; " /><meta name="citation_reference" content="citation_title=Phylogenetic identification and in situ detection of individual microbial cells without cultivation; Microbiol. Rev.;  citation_year=1995;  citation_volume=59;  citation_pages=143-69; " /><meta name="citation_reference" content="citation_title=Environmental shotgun sequencing: its potential and challenges for studying the hidden world of microbes; PLoS Biol.;  citation_year=2007;  citation_volume=5;  citation_pages=e82" /><meta name="citation_reference" content="citation_title=Molecular biological access to the chemistry of unknown soil microbes: a new frontier for natural products; Chem. Biol.;  citation_year=1998;  citation_volume=5;  citation_pages=R245-249; " /><meta name="citation_reference" content="citation_title=Metagenomic signatures of the Peru Margin subseafloor biosphere show a genetically distinct environment; Proc. Natl. Acad. Sci. USA;  citation_year=2008;  citation_volume=105;  citation_pages=10583-88; " /><meta name="citation_reference" content="citation_title=Comparative metagenomics of microbial communities; Science;  citation_year=2005;  citation_volume=308;  citation_pages=554-7; " /><meta name="citation_reference" content="citation_title=Metagenomics: genomic analysis of microbial communities; Annu. Rev. Genet.;  citation_year=2004;  citation_volume=38;  citation_pages=525-52; " /><meta name="citation_reference" content="citation_title=Targeted metagenomics: a high-resolution metagenomics approach for specific gene clusters in complex microbial communities; Environ. Microbiol.;  citation_year=2011;  citation_volume=14;  citation_pages=13-22; " /><meta name="citation_reference" content="citation_title=Using pyrosequencing to shed light on deep mine microbial ecology; BMC Genomics;  citation_year=2006;  citation_volume=7;  citation_pages=57" /><meta name="citation_reference" content="citation_title=The impact of next-generation sequencing technology on genetics; Trends Genet.;  citation_year=2008;  citation_volume=24;  citation_pages=133-41; " /><meta name="citation_reference" content="citation_title=The rational exploration of microbial diversity; ISME J.;  citation_year=2008;  citation_volume=2;  citation_pages=997-1006; " /><meta name="citation_reference" content="citation_title=The effect of sequencing errors on metagenomic gene prediction; BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=520" /><meta name="citation_reference" content="citation_title=Enabling technologies of genomic-scale sequence enrichment for targeted high-throughput sequencing; Genomics;  citation_year=2009;  citation_volume=94;  citation_pages=363-8; " /><meta name="citation_reference" content="citation_title=Direct selection of human genomic loci by microarray hybridization; Nat. Methods;  citation_year=2007;  citation_volume=4;  citation_pages=903-5; " /><meta name="citation_reference" content="citation_title=Microarray-based genomic selection for high-throughput resequencing; Nat. Methods;  citation_year=2007;  citation_volume=4;  citation_pages=907-9; " /><meta name="citation_reference" content="citation_title=Accurate SNP and mutation detection by targeted custom microarray-based genomic enrichment of short-fragment sequencing libraries; Nucleic Acids Res.;  citation_year=2010;  citation_volume=38;  citation_pages=e116" /><meta name="citation_reference" content="citation_title=Enrichment of sequencing targets from the human genome by solution hybridization; Genome Biol.;  citation_year=2009;  citation_volume=10;  citation_pages=R116" /><meta name="citation_reference" content="citation_title=Solution hybrid selection with ultra-long oligonucleotides for massively parallel targeted sequencing; Nat. Biotechnol.;  citation_year=2009;  citation_volume=27;  citation_pages=182-9; " /><meta name="citation_reference" content="citation_title=Gene-targeted-metagenomics reveals extensive diversity of aromatic dioxygenase genes in the environment; ISME J.;  citation_year=2010;  citation_volume=4;  citation_pages=279-85; " /><meta name="citation_reference" content="citation_title=Detecting variants with Metabolic Design, a new software tool to design probes for explorative functional DNA microarray development; BMC Bioinformatics;  citation_year=2010;  citation_volume=11;  citation_pages=478" /><meta name="citation_reference" content="citation_title=Detecting unknown sequences with DNA microarrays: explorative probe design strategies; Environ. Microbiol.;  citation_year=2011;  citation_volume=14;  citation_pages=356-371; " /><meta name="citation_reference" content="citation_title=Bias caused by template annealing in the amplification of mixtures of 16S rRNA genes by PCR; Appl. Environ. Microbiol.;  citation_year=1996;  citation_volume=62;  citation_pages=625-30; " /><meta name="citation_reference" content="citation_title=Polymerase chain reaction primers miss half of rRNA microbial diversity; ISME J.;  citation_year=2009;  citation_volume=3;  citation_pages=1365-73; " /><meta name="citation_reference" content="citation_title=Methane emissions from lakes: dependence of lake characteristics, two regional assessments, and a global estimate; citation_text=GB4009, doi:10.1029/2004GB002238; Global Biogeochem. Cycles;  citation_year=2004;  citation_volume=18; " /><meta name="citation_reference" content="citation_title=Accumulation of mantle gases in a permanently stratified volcanic lake (Lake Pavin, France); Geochim. Cosmochim. Acta;  citation_year=1999;  citation_volume=63;  citation_pages=3357-72; " /><meta name="citation_reference" content="citation_title=Identification of microbial communities involved in the methane cycle of a freshwater meromictic lake; FEMS Microbiol. Ecol.;  citation_year=2011;  citation_volume=77;  citation_pages=533-45; " /><meta name="citation_reference" content="citation_title=Molecular biology of methanogens; Annu. Rev. Microbiol.;  citation_year=1992;  citation_volume=46;  citation_pages=165-91; " /><meta name="citation_reference" content="citation_title=Comparative analysis of genes encoding methyl coenzyme M reductase in methanogenic bacteria; Mol. Gen. Genet.;  citation_year=1988;  citation_volume=213;  citation_pages=409-20; " /><meta name="citation_reference" content="citation_title=HiSpOD: probe design for functional DNA microarrays; Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=641-8; " /><meta name="citation_reference" content="citation_title=A putative new order of methanogenic Archaea inhabiting the human gut, as revealed by molecular analyses of the mcrA gene; Res. Microbiol.;  citation_year=2008;  citation_volume=159;  citation_pages=516-21; " /><meta name="citation_reference" content="citation_title=The Staden sequence analysis package; Mol. Biotechnol.;  citation_year=1996;  citation_volume=5;  citation_pages=233-41; " /><meta name="citation_reference" content="citation_title=Quality control and preprocessing of metagenomic datasets; Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=863-4; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-402; " /><meta name="citation_reference" content="citation_title=UCHIME improves sensitivity and speed of chimera detection; Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=2194-200; " /><meta name="citation_reference" content="citation_title=Clustal W and Clustal X version 2.0; Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=2947-8; " /><meta name="citation_reference" content="citation_title=SeaView version 4: a multiplatform graphical user interface for sequence alignment and phylogenetic tree building; Mol. Biol. Evol.;  citation_year=2009;  citation_volume=27;  citation_pages=221-4; " /><meta name="citation_reference" content="citation_title=Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences; Bioinformatics;  citation_year=2006;  citation_volume=22;  citation_pages=1658-9; " /><meta name="citation_reference" content="citation_title=The mcrA gene as an alternative to 16S rRNA in the phylogenetic analysis of methanogen populations in landfill; Microbiology;  citation_year=2002;  citation_volume=148;  citation_pages=3521-30; " /><meta name="citation_reference" content="citation_title=A note on the neighbor-joining algorithm of Saitou and Nei; Mol. Biol. Evol.;  citation_year=1988;  citation_volume=5;  citation_pages=729-31; " /><meta name="citation_reference" content="citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees; Mol. Biol. Evol.;  citation_year=1987;  citation_volume=4;  citation_pages=406-25; " /><meta name="citation_reference" content="citation_title=MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods; Mol. Biol. Evol.;  citation_year=2011;  citation_volume=28;  citation_pages=2731-9; " /><meta name="citation_reference" content="citation_title=Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) method; Methods;  citation_year=2001;  citation_volume=25;  citation_pages=402-8; " /><meta name="citation_reference" content="citation_title=Basic local alignment search tool; J. Mol. Biol.;  citation_year=1990;  citation_volume=215;  citation_pages=403-10; " /><meta name="citation_reference" content="citation_title=Isolation of a novel acidiphilic methanogen from an acidic peat bog; Nature;  citation_year=2006;  citation_volume=442;  citation_pages=192-4; " /><meta name="citation_reference" content="citation_title=Methanomassiliicoccus luminyensis gen. nov. sp. nov. a methanogenic archaeon isolated from human faeces; Int. J. Syst. Evol. Microbiol.;  citation_year=2012;  citation_volume=62;  citation_pages=1902-7; " /><meta name="citation_reference" content="citation_title=Genome sequence of ‘Candidatus Methanomethylophilus alvus’ Mx1201, a methanogenic archaeon from the human gut belonging to a seventh order of methanogens; J. Bacteriol.;  citation_year=2012;  citation_volume=194;  citation_pages=6944-5; " /><meta name="citation_reference" content="citation_title=Rapid quantitative profiling of complex microbial populations; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=e5" /><meta name="citation_reference" content="citation_title=The genome sequence of Methanosphaera stadtmanae reveals why this human intestinal archaeon is restricted to methanol and H2 for methane formation and ATP synthesis; J. Bacteriol.;  citation_year=2005;  citation_volume=188;  citation_pages=642-58; " /><meta name="citation_reference" content="citation_title=Methanobacterium lacus sp. nov., a novel hydrogenotrophic methanogen from the deep cold sediment of a meromictic lake; Int. J. Syst. Evol. Microbiol.;  citation_year=2011;  citation_volume=62;  citation_pages=1625-1629; " /><meta name="citation_reference" content="citation_title=Microarray-based multicycle-enrichment of genomic subsets for targeted next-generation sequencing; Genome Res.;  citation_year=2009;  citation_volume=19;  citation_pages=1616-21; " /><meta name="citation_reference" content="citation_title=Anaerobic microbial communities in Lake Pavin, a unique meromictic lake in France; Appl. Environ. Microbiol.;  citation_year=2005;  citation_volume=71;  citation_pages=7389-400; " /><meta name="citation_reference" content="citation_title=Environmental rRNA inventories miss over half of protistan diversity; BMC Microbiol.;  citation_year=2008;  citation_volume=8;  citation_pages=222" /><meta name="citation_reference" content="citation_title=PhylArray: phylogenetic probe design algorithm for microarray; Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=2550-7; " /><meta name="citation_reference" content="citation_title=KASpOD – a web service for highly specific and explorative oligonucleotide design; Bioinformatics;  citation_year=2012;  citation_volume=28;  citation_pages=3161-3162; " /><meta name="citation_reference" content="citation_title=Third generation DNA sequencing: pacific biosciences’ single molecule real time technology; Chem. Biol.;  citation_year=2010;  citation_volume=17;  citation_pages=675-6; " /><meta name="citation_reference" content="citation_title=A window into third-generation sequencing; Hum. Mol. Genet.;  citation_year=2010;  citation_volume=19;  citation_pages=R227-240; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/20/2/185/955416/dst001.pdf" /><meta name="description" content="Abstract.  Next-generation sequencing (NGS) allows faster acquisition of metagenomic data, but complete exploration of complex ecosystems is hindered by the ext" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/20/2/185/333880" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/20/2/185/333880" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/20/2/185/333880");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dst001';
    </script>
<title>
	Gene Capture Coupled to High-Throughput Sequencing as a Strategy for Targeted Metagenome Exploration | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./333880" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BdCOATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRGVub25mb3V4LCBKw6lyw6ltaWUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDZW50cmUgZGUgUmVjaGVyY2hlIGVuIE51dHJpdGlvbiBIdW1haW5lIEF1dmVyZ25lLCBDbGVybW9udCBVbml2ZXJzaXTDqSwgVW5pdmVyc2l0w6kgZCdBdXZlcmduZSxFQSA0Njc4LCBDb25jZXB0aW9uLCBJbmfDqW5pZXJpZSBldCBEw6l2ZWxvcHBlbWVudCBkZSBsJ0FsaW1lbnQgZXQgZHUgTcOpZGljYW1lbnQsIEJQIDEwNDQ4LCBDbGVybW9udC1GZXJyYW5kIDYzMDAwLCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDbGVybW9udCBVbml2ZXJzaXTDqSwgVW5pdmVyc2l0w6kgQmxhaXNlIFBhc2NhbCwgQ2xlcm1vbnQtRmVycmFuZCA2MzAwMCwgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVU1SIENOUlMgNjAyMywgVW5pdmVyc2l0w6kgQmxhaXNlIFBhc2NhbCwgQ2xlcm1vbnQtRmVycmFuZCA2MzAwMCwgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUGFyaXNvdCwgTmljb2xhcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNlbnRyZSBkZSBSZWNoZXJjaGUgZW4gTnV0cml0aW9uIEh1bWFpbmUgQXV2ZXJnbmUsIENsZXJtb250IFVuaXZlcnNpdMOpLCBVbml2ZXJzaXTDqSBkJ0F1dmVyZ25lLEVBIDQ2NzgsIENvbmNlcHRpb24sIEluZ8OpbmllcmllIGV0IETDqXZlbG9wcGVtZW50IGRlIGwnQWxpbWVudCBldCBkdSBNw6lkaWNhbWVudCwgQlAgMTA0NDgsIENsZXJtb250LUZlcnJhbmQgNjMwMDAsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNsZXJtb250IFVuaXZlcnNpdMOpLCBVbml2ZXJzaXTDqSBCbGFpc2UgUGFzY2FsLCBDbGVybW9udC1GZXJyYW5kIDYzMDAwLCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJVTVIgQ05SUyA2MDIzLCBVbml2ZXJzaXTDqSBCbGFpc2UgUGFzY2FsLCBDbGVybW9udC1GZXJyYW5kIDYzMDAwLCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJEdWdhdC1Cb255LCBFcmljIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2VudHJlIGRlIFJlY2hlcmNoZSBlbiBOdXRyaXRpb24gSHVtYWluZSBBdXZlcmduZSwgQ2xlcm1vbnQgVW5pdmVyc2l0w6ksIFVuaXZlcnNpdMOpIGQnQXV2ZXJnbmUsRUEgNDY3OCwgQ29uY2VwdGlvbiwgSW5nw6luaWVyaWUgZXQgRMOpdmVsb3BwZW1lbnQgZGUgbCdBbGltZW50IGV0IGR1IE3DqWRpY2FtZW50LCBCUCAxMDQ0OCwgQ2xlcm1vbnQtRmVycmFuZCA2MzAwMCwgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVUZSIFBoYXJtYWNpZSwgQ2xlcm1vbnQgVW5pdmVyc2l0w6ksIFVuaXZlcnNpdMOpIGQnQXV2ZXJnbmUsQ2xlcm1vbnQtRmVycmFuZCA2MzAwMCwgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQmlkZXJyZS1QZXRpdCwgQ29yaW5uZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlVNUiBDTlJTIDYwMjMsIFVuaXZlcnNpdMOpIEJsYWlzZSBQYXNjYWwsIENsZXJtb250LUZlcnJhbmQgNjMwMDAsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvaXJlIE1pY3Jvb3JnYW5pc21lczogR8Opbm9tZSBldCBFbnZpcm9ubmVtZW50LCBDbGVybW9udCBVbml2ZXJzaXTDqSwgVW5pdmVyc2l0w6kgQmxhaXNlIFBhc2NhbCwgQlAgMTA0NDgsIENsZXJtb250LUZlcnJhbmQgNjMwMDAsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkJvdWNoZXIsIERlbHBoaW5lIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2VudHJlIGRlIFJlY2hlcmNoZSBlbiBOdXRyaXRpb24gSHVtYWluZSBBdXZlcmduZSwgQ2xlcm1vbnQgVW5pdmVyc2l0w6ksIFVuaXZlcnNpdMOpIGQnQXV2ZXJnbmUsRUEgNDY3OCwgQ29uY2VwdGlvbiwgSW5nw6luaWVyaWUgZXQgRMOpdmVsb3BwZW1lbnQgZGUgbCdBbGltZW50IGV0IGR1IE3DqWRpY2FtZW50LCBCUCAxMDQ0OCwgQ2xlcm1vbnQtRmVycmFuZCA2MzAwMCwgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVUZSIFBoYXJtYWNpZSwgQ2xlcm1vbnQgVW5pdmVyc2l0w6ksIFVuaXZlcnNpdMOpIGQnQXV2ZXJnbmUsQ2xlcm1vbnQtRmVycmFuZCA2MzAwMCwgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTW9yZ2F2aSwgRGllZ28gUC4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJTlJBLCBVTVIxMjEzIEhlcmJpdm9yZXMsIEYtNjMxMjIgU2FpbnQtR2Vuw6hzLUNoYW1wYW5lbGxlIGFuZCBDbGVybW9udCBVbml2ZXJzaXTDqSwgVmV0QWdybyBTdXAsIFVNUiBIZXJiaXZvcmVzLCBCUCAxMDQ0OCwgRi02MzAwMCwgQ2xlcm1vbnQtRmVycmFuZCwgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGUgUGFzbGllciwgRGVuaXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDRUEsIERTViwgSW5zdGl0dXQgZGUgR8Opbm9taXF1ZSwgR2Vub3Njb3BlLCAyIHJ1ZSBHYXN0b24gQ3LDqW1pZXV4LCBFdnJ5IDkxMDU3LCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDTlJTLCBVTVI4MDMwLCBFdnJ5IDkxMDU3LCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJVRVZFLCBVbml2ZXJzaXTDqSBkJ0V2cnksIEV2cnkgOTEwNTcsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlBleXJldGFpbGxhZGUsIEVyaWMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDZW50cmUgZGUgUmVjaGVyY2hlIGVuIE51dHJpdGlvbiBIdW1haW5lIEF1dmVyZ25lLCBDbGVybW9udCBVbml2ZXJzaXTDqSwgVW5pdmVyc2l0w6kgZCdBdXZlcmduZSxFQSA0Njc4LCBDb25jZXB0aW9uLCBJbmfDqW5pZXJpZSBldCBEw6l2ZWxvcHBlbWVudCBkZSBsJ0FsaW1lbnQgZXQgZHUgTcOpZGljYW1lbnQsIEJQIDEwNDQ4LCBDbGVybW9udC1GZXJyYW5kIDYzMDAwLCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJVRlIgUGhhcm1hY2llLCBDbGVybW9udCBVbml2ZXJzaXTDqSwgVW5pdmVyc2l0w6kgZCdBdXZlcmduZSxDbGVybW9udC1GZXJyYW5kIDYzMDAwLCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJQZXlyZXQsIFBpZXJyZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNlbnRyZSBkZSBSZWNoZXJjaGUgZW4gTnV0cml0aW9uIEh1bWFpbmUgQXV2ZXJnbmUsIENsZXJtb250IFVuaXZlcnNpdMOpLCBVbml2ZXJzaXTDqSBkJ0F1dmVyZ25lLEVBIDQ2NzgsIENvbmNlcHRpb24sIEluZ8OpbmllcmllIGV0IETDqXZlbG9wcGVtZW50IGRlIGwnQWxpbWVudCBldCBkdSBNw6lkaWNhbWVudCwgQlAgMTA0NDgsIENsZXJtb250LUZlcnJhbmQgNjMwMDAsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlVGUiBQaGFybWFjaWUsIENsZXJtb250IFVuaXZlcnNpdMOpLCBVbml2ZXJzaXTDqSBkJ0F1dmVyZ25lLENsZXJtb250LUZlcnJhbmQgNjMwMDAsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iR2VuZSBDYXB0dXJlIENvdXBsZWQgdG8gSGlnaC1UaHJvdWdocHV0IFNlcXVlbmNpbmcgYXMgYSBTdHJhdGVneSBmb3IgVGFyZ2V0ZWQgTWV0YWdlbm9tZSBFeHBsb3JhdGlvbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjE4NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMTk2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN0MDAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTMvMDQvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm9rYXJ5b3RlczogdGhlIHVuc2VlbiBtYWpvcml0eTsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9OTU7ICBjaXRhdGlvbl9wYWdlcz02NTc4LTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaGF0IGlzIHRoZSBleHRlbnQgb2YgcHJva2FyeW90aWMgZGl2ZXJzaXR5PzsgUGhpbG9zLiBUcmFucy4gUi4gU29jLiBMb25kLiBCLiBCaW9sLiBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM2MTsgIGNpdGF0aW9uX3BhZ2VzPTIwMjMtMzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoeWxvZ2VuZXRpYyBpZGVudGlmaWNhdGlvbiBhbmQgaW4gc2l0dSBkZXRlY3Rpb24gb2YgaW5kaXZpZHVhbCBtaWNyb2JpYWwgY2VsbHMgd2l0aG91dCBjdWx0aXZhdGlvbjsgTWljcm9iaW9sLiBSZXYuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTU5OyAgY2l0YXRpb25fcGFnZXM9MTQzLTY5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbnZpcm9ubWVudGFsIHNob3RndW4gc2VxdWVuY2luZzogaXRzIHBvdGVudGlhbCBhbmQgY2hhbGxlbmdlcyBmb3Igc3R1ZHlpbmcgdGhlIGhpZGRlbiB3b3JsZCBvZiBtaWNyb2JlczsgUExvUyBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9ZTgyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGJpb2xvZ2ljYWwgYWNjZXNzIHRvIHRoZSBjaGVtaXN0cnkgb2YgdW5rbm93biBzb2lsIG1pY3JvYmVzOiBhIG5ldyBmcm9udGllciBmb3IgbmF0dXJhbCBwcm9kdWN0czsgQ2hlbS4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPVIyNDUtMjQ5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZXRhZ2Vub21pYyBzaWduYXR1cmVzIG9mIHRoZSBQZXJ1IE1hcmdpbiBzdWJzZWFmbG9vciBiaW9zcGhlcmUgc2hvdyBhIGdlbmV0aWNhbGx5IGRpc3RpbmN0IGVudmlyb25tZW50OyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xMDU7ICBjaXRhdGlvbl9wYWdlcz0xMDU4My04ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgbWV0YWdlbm9taWNzIG9mIG1pY3JvYmlhbCBjb21tdW5pdGllczsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMDg7ICBjaXRhdGlvbl9wYWdlcz01NTQtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWV0YWdlbm9taWNzOiBnZW5vbWljIGFuYWx5c2lzIG9mIG1pY3JvYmlhbCBjb21tdW5pdGllczsgQW5udS4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz01MjUtNTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRhcmdldGVkIG1ldGFnZW5vbWljczogYSBoaWdoLXJlc29sdXRpb24gbWV0YWdlbm9taWNzIGFwcHJvYWNoIGZvciBzcGVjaWZpYyBnZW5lIGNsdXN0ZXJzIGluIGNvbXBsZXggbWljcm9iaWFsIGNvbW11bml0aWVzOyBFbnZpcm9uLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MTMtMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVzaW5nIHB5cm9zZXF1ZW5jaW5nIHRvIHNoZWQgbGlnaHQgb24gZGVlcCBtaW5lIG1pY3JvYmlhbCBlY29sb2d5OyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTU3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGltcGFjdCBvZiBuZXh0LWdlbmVyYXRpb24gc2VxdWVuY2luZyB0ZWNobm9sb2d5IG9uIGdlbmV0aWNzOyBUcmVuZHMgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MTMzLTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcmF0aW9uYWwgZXhwbG9yYXRpb24gb2YgbWljcm9iaWFsIGRpdmVyc2l0eTsgSVNNRSBKLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9OTk3LTEwMDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBlZmZlY3Qgb2Ygc2VxdWVuY2luZyBlcnJvcnMgb24gbWV0YWdlbm9taWMgZ2VuZSBwcmVkaWN0aW9uOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz01MjAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbmFibGluZyB0ZWNobm9sb2dpZXMgb2YgZ2Vub21pYy1zY2FsZSBzZXF1ZW5jZSBlbnJpY2htZW50IGZvciB0YXJnZXRlZCBoaWdoLXRocm91Z2hwdXQgc2VxdWVuY2luZzsgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9OTQ7ICBjaXRhdGlvbl9wYWdlcz0zNjMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlyZWN0IHNlbGVjdGlvbiBvZiBodW1hbiBnZW5vbWljIGxvY2kgYnkgbWljcm9hcnJheSBoeWJyaWRpemF0aW9uOyBOYXQuIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTkwMy01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb2FycmF5LWJhc2VkIGdlbm9taWMgc2VsZWN0aW9uIGZvciBoaWdoLXRocm91Z2hwdXQgcmVzZXF1ZW5jaW5nOyBOYXQuIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTkwNy05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BY2N1cmF0ZSBTTlAgYW5kIG11dGF0aW9uIGRldGVjdGlvbiBieSB0YXJnZXRlZCBjdXN0b20gbWljcm9hcnJheS1iYXNlZCBnZW5vbWljIGVucmljaG1lbnQgb2Ygc2hvcnQtZnJhZ21lbnQgc2VxdWVuY2luZyBsaWJyYXJpZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0zODsgIGNpdGF0aW9uX3BhZ2VzPWUxMTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbnJpY2htZW50IG9mIHNlcXVlbmNpbmcgdGFyZ2V0cyBmcm9tIHRoZSBodW1hbiBnZW5vbWUgYnkgc29sdXRpb24gaHlicmlkaXphdGlvbjsgR2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9UjExNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNvbHV0aW9uIGh5YnJpZCBzZWxlY3Rpb24gd2l0aCB1bHRyYS1sb25nIG9saWdvbnVjbGVvdGlkZXMgZm9yIG1hc3NpdmVseSBwYXJhbGxlbCB0YXJnZXRlZCBzZXF1ZW5jaW5nOyBOYXQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MTgyLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUtdGFyZ2V0ZWQtbWV0YWdlbm9taWNzIHJldmVhbHMgZXh0ZW5zaXZlIGRpdmVyc2l0eSBvZiBhcm9tYXRpYyBkaW94eWdlbmFzZSBnZW5lcyBpbiB0aGUgZW52aXJvbm1lbnQ7IElTTUUgSi47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTI3OS04NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV0ZWN0aW5nIHZhcmlhbnRzIHdpdGggTWV0YWJvbGljIERlc2lnbiwgYSBuZXcgc29mdHdhcmUgdG9vbCB0byBkZXNpZ24gcHJvYmVzIGZvciBleHBsb3JhdGl2ZSBmdW5jdGlvbmFsIEROQSBtaWNyb2FycmF5IGRldmVsb3BtZW50OyBCTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz00NzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXRlY3RpbmcgdW5rbm93biBzZXF1ZW5jZXMgd2l0aCBETkEgbWljcm9hcnJheXM6IGV4cGxvcmF0aXZlIHByb2JlIGRlc2lnbiBzdHJhdGVnaWVzOyBFbnZpcm9uLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MzU2LTM3MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmlhcyBjYXVzZWQgYnkgdGVtcGxhdGUgYW5uZWFsaW5nIGluIHRoZSBhbXBsaWZpY2F0aW9uIG9mIG1peHR1cmVzIG9mIDE2UyByUk5BIGdlbmVzIGJ5IFBDUjsgQXBwbC4gRW52aXJvbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT02MjsgIGNpdGF0aW9uX3BhZ2VzPTYyNS0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG9seW1lcmFzZSBjaGFpbiByZWFjdGlvbiBwcmltZXJzIG1pc3MgaGFsZiBvZiByUk5BIG1pY3JvYmlhbCBkaXZlcnNpdHk7IElTTUUgSi47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTEzNjUtNzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1ldGhhbmUgZW1pc3Npb25zIGZyb20gbGFrZXM6IGRlcGVuZGVuY2Ugb2YgbGFrZSBjaGFyYWN0ZXJpc3RpY3MsIHR3byByZWdpb25hbCBhc3Nlc3NtZW50cywgYW5kIGEgZ2xvYmFsIGVzdGltYXRlOyBjaXRhdGlvbl90ZXh0PUdCNDAwOSwgZG9pOjEwLjEwMjkvMjAwNEdCMDAyMjM4OyBHbG9iYWwgQmlvZ2VvY2hlbS4gQ3ljbGVzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BY2N1bXVsYXRpb24gb2YgbWFudGxlIGdhc2VzIGluIGEgcGVybWFuZW50bHkgc3RyYXRpZmllZCB2b2xjYW5pYyBsYWtlIChMYWtlIFBhdmluLCBGcmFuY2UpOyBHZW9jaGltLiBDb3Ntb2NoaW0uIEFjdGE7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9NjM7ICBjaXRhdGlvbl9wYWdlcz0zMzU3LTcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBtaWNyb2JpYWwgY29tbXVuaXRpZXMgaW52b2x2ZWQgaW4gdGhlIG1ldGhhbmUgY3ljbGUgb2YgYSBmcmVzaHdhdGVyIG1lcm9taWN0aWMgbGFrZTsgRkVNUyBNaWNyb2Jpb2wuIEVjb2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTc3OyAgY2l0YXRpb25fcGFnZXM9NTMzLTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgYmlvbG9neSBvZiBtZXRoYW5vZ2VuczsgQW5udS4gUmV2LiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTQ2OyAgY2l0YXRpb25fcGFnZXM9MTY1LTkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBhbmFseXNpcyBvZiBnZW5lcyBlbmNvZGluZyBtZXRoeWwgY29lbnp5bWUgTSByZWR1Y3Rhc2UgaW4gbWV0aGFub2dlbmljIGJhY3RlcmlhOyBNb2wuIEdlbi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTg4OyAgY2l0YXRpb25fdm9sdW1lPTIxMzsgIGNpdGF0aW9uX3BhZ2VzPTQwOS0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlTcE9EOiBwcm9iZSBkZXNpZ24gZm9yIGZ1bmN0aW9uYWwgRE5BIG1pY3JvYXJyYXlzOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTY0MS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHB1dGF0aXZlIG5ldyBvcmRlciBvZiBtZXRoYW5vZ2VuaWMgQXJjaGFlYSBpbmhhYml0aW5nIHRoZSBodW1hbiBndXQsIGFzIHJldmVhbGVkIGJ5IG1vbGVjdWxhciBhbmFseXNlcyBvZiB0aGUgbWNyQSBnZW5lOyBSZXMuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTU5OyAgY2l0YXRpb25fcGFnZXM9NTE2LTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgU3RhZGVuIHNlcXVlbmNlIGFuYWx5c2lzIHBhY2thZ2U7IE1vbC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTIzMy00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UXVhbGl0eSBjb250cm9sIGFuZCBwcmVwcm9jZXNzaW5nIG9mIG1ldGFnZW5vbWljIGRhdGFzZXRzOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTg2My00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HYXBwZWQgQkxBU1QgYW5kIFBTSS1CTEFTVDogYSBuZXcgZ2VuZXJhdGlvbiBvZiBwcm90ZWluIGRhdGFiYXNlIHNlYXJjaCBwcm9ncmFtczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MzM4OS00MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVDSElNRSBpbXByb3ZlcyBzZW5zaXRpdml0eSBhbmQgc3BlZWQgb2YgY2hpbWVyYSBkZXRlY3Rpb247IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MjE5NC0yMDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNsdXN0YWwgVyBhbmQgQ2x1c3RhbCBYIHZlcnNpb24gMi4wOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTI5NDctODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VhVmlldyB2ZXJzaW9uIDQ6IGEgbXVsdGlwbGF0Zm9ybSBncmFwaGljYWwgdXNlciBpbnRlcmZhY2UgZm9yIHNlcXVlbmNlIGFsaWdubWVudCBhbmQgcGh5bG9nZW5ldGljIHRyZWUgYnVpbGRpbmc7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0yMjEtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2QtaGl0OiBhIGZhc3QgcHJvZ3JhbSBmb3IgY2x1c3RlcmluZyBhbmQgY29tcGFyaW5nIGxhcmdlIHNldHMgb2YgcHJvdGVpbiBvciBudWNsZW90aWRlIHNlcXVlbmNlczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0xNjU4LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBtY3JBIGdlbmUgYXMgYW4gYWx0ZXJuYXRpdmUgdG8gMTZTIHJSTkEgaW4gdGhlIHBoeWxvZ2VuZXRpYyBhbmFseXNpcyBvZiBtZXRoYW5vZ2VuIHBvcHVsYXRpb25zIGluIGxhbmRmaWxsOyBNaWNyb2Jpb2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTQ4OyAgY2l0YXRpb25fcGFnZXM9MzUyMS0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBub3RlIG9uIHRoZSBuZWlnaGJvci1qb2luaW5nIGFsZ29yaXRobSBvZiBTYWl0b3UgYW5kIE5laTsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk4ODsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9NzI5LTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbmVpZ2hib3Itam9pbmluZyBtZXRob2Q6IGEgbmV3IG1ldGhvZCBmb3IgcmVjb25zdHJ1Y3RpbmcgcGh5bG9nZW5ldGljIHRyZWVzOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz00MDYtMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1FR0E1OiBtb2xlY3VsYXIgZXZvbHV0aW9uYXJ5IGdlbmV0aWNzIGFuYWx5c2lzIHVzaW5nIG1heGltdW0gbGlrZWxpaG9vZCwgZXZvbHV0aW9uYXJ5IGRpc3RhbmNlLCBhbmQgbWF4aW11bSBwYXJzaW1vbnkgbWV0aG9kczsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTI3MzEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgcmVsYXRpdmUgZ2VuZSBleHByZXNzaW9uIGRhdGEgdXNpbmcgcmVhbC10aW1lIHF1YW50aXRhdGl2ZSBQQ1IgYW5kIHRoZSAyKC1EZWx0YSBEZWx0YSBDKFQpKSBtZXRob2Q7IE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz00MDItODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFzaWMgbG9jYWwgYWxpZ25tZW50IHNlYXJjaCB0b29sOyBKLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTIxNTsgIGNpdGF0aW9uX3BhZ2VzPTQwMy0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNvbGF0aW9uIG9mIGEgbm92ZWwgYWNpZGlwaGlsaWMgbWV0aGFub2dlbiBmcm9tIGFuIGFjaWRpYyBwZWF0IGJvZzsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTQ0MjsgIGNpdGF0aW9uX3BhZ2VzPTE5Mi00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZXRoYW5vbWFzc2lsaWljb2NjdXMgbHVtaW55ZW5zaXMgZ2VuLiBub3YuIHNwLiBub3YuIGEgbWV0aGFub2dlbmljIGFyY2hhZW9uIGlzb2xhdGVkIGZyb20gaHVtYW4gZmFlY2VzOyBJbnQuIEouIFN5c3QuIEV2b2wuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NjI7ICBjaXRhdGlvbl9wYWdlcz0xOTAyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jZSBvZiDigJhDYW5kaWRhdHVzIE1ldGhhbm9tZXRoeWxvcGhpbHVzIGFsdnVz4oCZIE14MTIwMSwgYSBtZXRoYW5vZ2VuaWMgYXJjaGFlb24gZnJvbSB0aGUgaHVtYW4gZ3V0IGJlbG9uZ2luZyB0byBhIHNldmVudGggb3JkZXIgb2YgbWV0aGFub2dlbnM7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTk0OyAgY2l0YXRpb25fcGFnZXM9Njk0NC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SYXBpZCBxdWFudGl0YXRpdmUgcHJvZmlsaW5nIG9mIGNvbXBsZXggbWljcm9iaWFsIHBvcHVsYXRpb25zOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz1lNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWUgc2VxdWVuY2Ugb2YgTWV0aGFub3NwaGFlcmEgc3RhZHRtYW5hZSByZXZlYWxzIHdoeSB0aGlzIGh1bWFuIGludGVzdGluYWwgYXJjaGFlb24gaXMgcmVzdHJpY3RlZCB0byBtZXRoYW5vbCBhbmQgSDIgZm9yIG1ldGhhbmUgZm9ybWF0aW9uIGFuZCBBVFAgc3ludGhlc2lzOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE4ODsgIGNpdGF0aW9uX3BhZ2VzPTY0Mi01ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWV0aGFub2JhY3Rlcml1bSBsYWN1cyBzcC4gbm92LiwgYSBub3ZlbCBoeWRyb2dlbm90cm9waGljIG1ldGhhbm9nZW4gZnJvbSB0aGUgZGVlcCBjb2xkIHNlZGltZW50IG9mIGEgbWVyb21pY3RpYyBsYWtlOyBJbnQuIEouIFN5c3QuIEV2b2wuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NjI7ICBjaXRhdGlvbl9wYWdlcz0xNjI1LTE2Mjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3JvYXJyYXktYmFzZWQgbXVsdGljeWNsZS1lbnJpY2htZW50IG9mIGdlbm9taWMgc3Vic2V0cyBmb3IgdGFyZ2V0ZWQgbmV4dC1nZW5lcmF0aW9uIHNlcXVlbmNpbmc7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MTYxNi0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hZXJvYmljIG1pY3JvYmlhbCBjb21tdW5pdGllcyBpbiBMYWtlIFBhdmluLCBhIHVuaXF1ZSBtZXJvbWljdGljIGxha2UgaW4gRnJhbmNlOyBBcHBsLiBFbnZpcm9uLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTcxOyAgY2l0YXRpb25fcGFnZXM9NzM4OS00MDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVudmlyb25tZW50YWwgclJOQSBpbnZlbnRvcmllcyBtaXNzIG92ZXIgaGFsZiBvZiBwcm90aXN0YW4gZGl2ZXJzaXR5OyBCTUMgTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MjIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5bEFycmF5OiBwaHlsb2dlbmV0aWMgcHJvYmUgZGVzaWduIGFsZ29yaXRobSBmb3IgbWljcm9hcnJheTsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0yNTUwLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUtBU3BPRCDigJMgYSB3ZWIgc2VydmljZSBmb3IgaGlnaGx5IHNwZWNpZmljIGFuZCBleHBsb3JhdGl2ZSBvbGlnb251Y2xlb3RpZGUgZGVzaWduOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTMxNjEtMzE2MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhpcmQgZ2VuZXJhdGlvbiBETkEgc2VxdWVuY2luZzogcGFjaWZpYyBiaW9zY2llbmNlc+KAmSBzaW5nbGUgbW9sZWN1bGUgcmVhbCB0aW1lIHRlY2hub2xvZ3k7IENoZW0uIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9Njc1LTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgd2luZG93IGludG8gdGhpcmQtZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nOyBIdW0uIE1vbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9UjIyNy0yNDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMC8yLzE4NS85NTU0MTYvZHN0MDAxLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBOZXh0LWdlbmVyYXRpb24gc2VxdWVuY2luZyAoTkdTKSBhbGxvd3MgZmFzdGVyIGFjcXVpc2l0aW9uIG9mIG1ldGFnZW5vbWljIGRhdGEsIGJ1dCBjb21wbGV0ZSBleHBsb3JhdGlvbiBvZiBjb21wbGV4IGVjb3N5c3RlbXMgaXMgaGluZGVyZWQgYnkgdGhlIGV4dCIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MzMzODgwJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZDBP1BArIvbF67Vs1ZQral3yQjfv" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f2%2f185%2f333880" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f2%2f185%2f333880" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=12698" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/20/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/2/1/m_cover.gif?Expires=2147483647&amp;Signature=Yq7dbhq6hb-Iyo5ewEGt-Y1T99brR4Xh8FvVnCI2Bg1SIlPGztlGCk-Bm0A9A72SaqDKcetVlEQehNtxyAGT~T55ZFFD19A3hKk~A1XbnQ-JfXYKj2-gYHz-VirOlvLe-8XkC6qvT4h2PniinfIJH6d85JsIf3rhRRtN086VVGvi2qNVR0THPJOeMpQvElVQEEN-4bpo3wEYmq80uwX58y5ScJ0pcJt7sV40xw1Fd6mp92v-su9vXuGT~IIVjyHRDyuiGt~ZiUzcXtor6WX4MT1zcCCf5ySC3Wm7DfaZruIvVsbtJxHUYCwePrgmcTq1vhdeOWHam2svM-jBZyoDoA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/20/2">
                        <div class="volume trailing-comma">Volume 20</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2013        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="4627772">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4627772">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4627774">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4627774">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4627782">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4627782">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4627808">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4627808">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4627834">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4627834">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4627841">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4627841">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4627843">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4627843">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4627845">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4627845">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4627847">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4627847">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/20/2/173/333781">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/20/2/197/334072">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
<span class="article-flag"><a href="https://academic.oup.com/dnaresearch/search-results?f_OUPSeries=Editor's+Choice">Editor's Choice</a></span>            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Gene Capture Coupled to High-Throughput Sequencing as a Strategy for Targeted Metagenome Exploration
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">J&#233;r&#233;mie Denonfoux</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Centre de Recherche en Nutrition Humaine Auvergne</div>, <div class="insititution">Clermont Université, Université d'Auvergne,</div><div class="addr-line">EA 4678, Conception, Ingénierie et Développement de l'Aliment et du Médicament, BP 10448, Clermont-Ferrand 63000</div>, <div class="country">France</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">Clermont Université, Université Blaise Pascal</div>, <div class="addr-line">Clermont-Ferrand 63000</div>, <div class="country">France</div></div><div class="aff"><div class="label -label">3</div><div class="insititution">UMR CNRS 6023, Université Blaise Pascal</div>, <div class="addr-line">Clermont-Ferrand 63000</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=J%c3%a9r%c3%a9mie+Denonfoux">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Denonfoux J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Denonfoux J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jérémie Denonfoux</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nicolas Parisot</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Centre de Recherche en Nutrition Humaine Auvergne</div>, <div class="insititution">Clermont Université, Université d'Auvergne,</div><div class="addr-line">EA 4678, Conception, Ingénierie et Développement de l'Aliment et du Médicament, BP 10448, Clermont-Ferrand 63000</div>, <div class="country">France</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">Clermont Université, Université Blaise Pascal</div>, <div class="addr-line">Clermont-Ferrand 63000</div>, <div class="country">France</div></div><div class="aff"><div class="label -label">3</div><div class="insititution">UMR CNRS 6023, Université Blaise Pascal</div>, <div class="addr-line">Clermont-Ferrand 63000</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nicolas+Parisot">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Parisot N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Parisot N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Nicolas Parisot</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Eric Dugat-Bony</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Centre de Recherche en Nutrition Humaine Auvergne</div>, <div class="insititution">Clermont Université, Université d'Auvergne,</div><div class="addr-line">EA 4678, Conception, Ingénierie et Développement de l'Aliment et du Médicament, BP 10448, Clermont-Ferrand 63000</div>, <div class="country">France</div></div><div class="aff"><div class="label -label">4</div><div class="addr-line">UFR Pharmacie</div>, <div class="insititution">Clermont Université, Université d'Auvergne,</div><div class="addr-line">Clermont-Ferrand 63000</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Eric+Dugat-Bony">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Dugat-Bony E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Dugat-Bony E&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Eric Dugat-Bony</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Corinne Biderre-Petit</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">UMR CNRS 6023, Université Blaise Pascal</div>, <div class="addr-line">Clermont-Ferrand 63000</div>, <div class="country">France</div></div><div class="aff"><div class="label -label">5</div><div class="addr-line">Laboratoire Microorganismes: Génome et Environnement</div>, <div class="insititution">Clermont Université, Université Blaise Pascal</div>, <div class="addr-line">BP 10448, Clermont-Ferrand 63000</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Corinne+Biderre-Petit">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Biderre-Petit C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Biderre-Petit C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Corinne Biderre-Petit</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Delphine Boucher</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Centre de Recherche en Nutrition Humaine Auvergne</div>, <div class="insititution">Clermont Université, Université d'Auvergne,</div><div class="addr-line">EA 4678, Conception, Ingénierie et Développement de l'Aliment et du Médicament, BP 10448, Clermont-Ferrand 63000</div>, <div class="country">France</div></div><div class="aff"><div class="label -label">4</div><div class="addr-line">UFR Pharmacie</div>, <div class="insititution">Clermont Université, Université d'Auvergne,</div><div class="addr-line">Clermont-Ferrand 63000</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Delphine+Boucher">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Boucher D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Boucher D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Delphine Boucher</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Diego P. Morgavi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="insititution">INRA</div>, <div class="addr-line">UMR1213 Herbivores, F-63122 Saint-Genès-Champanelle and Clermont Université, VetAgro Sup, UMR Herbivores, BP 10448, F-63000, Clermont-Ferrand</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Diego+P.+Morgavi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Morgavi D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Morgavi D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Diego P. Morgavi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Denis Le Paslier</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">7</div><div class="insititution">CEA, DSV, Institut de Génomique</div>, <div class="addr-line">Genoscope, 2 rue Gaston Crémieux, Evry 91057</div>, <div class="country">France</div></div><div class="aff"><div class="label -label">8</div><div class="insititution">CNRS, UMR8030</div>, <div class="addr-line">Evry 91057</div>, <div class="country">France</div></div><div class="aff"><div class="label -label">9</div><div class="insititution">UEVE, Université d'Evry</div>, <div class="addr-line">Evry 91057</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Denis+Le+Paslier">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Le Paslier D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Le Paslier D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Denis Le Paslier</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Eric Peyretaillade</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Centre de Recherche en Nutrition Humaine Auvergne</div>, <div class="insititution">Clermont Université, Université d'Auvergne,</div><div class="addr-line">EA 4678, Conception, Ingénierie et Développement de l'Aliment et du Médicament, BP 10448, Clermont-Ferrand 63000</div>, <div class="country">France</div></div><div class="aff"><div class="label -label">4</div><div class="addr-line">UFR Pharmacie</div>, <div class="insititution">Clermont Université, Université d'Auvergne,</div><div class="addr-line">Clermont-Ferrand 63000</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Eric+Peyretaillade">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Peyretaillade E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Peyretaillade E&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Eric Peyretaillade</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Pierre Peyret</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Centre de Recherche en Nutrition Humaine Auvergne</div>, <div class="insititution">Clermont Université, Université d'Auvergne,</div><div class="addr-line">EA 4678, Conception, Ingénierie et Développement de l'Aliment et du Médicament, BP 10448, Clermont-Ferrand 63000</div>, <div class="country">France</div></div><div class="aff"><div class="label -label">4</div><div class="addr-line">UFR Pharmacie</div>, <div class="insititution">Clermont Université, Université d'Auvergne,</div><div class="addr-line">Clermont-Ferrand 63000</div>, <div class="country">France</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. EA4678 CIDAM, 28 place Henri Dunant, 63001 Clermont-Ferrand. Tel. +33 47-3178-308. Fax. +33 47-3275-624. Email: <a href="mailto:pierre.peyret@udamail.fr" target="_blank">pierre.peyret@udamail.fr</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Pierre+Peyret">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Peyret P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Peyret P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Pierre Peyret</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this study.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 20, Issue 2, 1 April 2013, Pages 185–196, <a href='https://doi.org/10.1093/dnares/dst001'>https://doi.org/10.1093/dnares/dst001</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">30 January 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">10 October 2012</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">09 January 2013</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="333880" href="/dnaresearch/article-pdf/20/2/185/955416/dst001.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Jérémie Denonfoux, Nicolas Parisot, Eric Dugat-Bony, Corinne Biderre-Petit, Delphine Boucher, Diego P. Morgavi, Denis Le Paslier, Eric Peyretaillade, Pierre Peyret;  Gene Capture Coupled to High-Throughput Sequencing as a Strategy for Targeted Metagenome Exploration, <em>DNA Research</em>, Volume 20, Issue 2, 1 April 2013, Pages 185–196, <a href='https://doi.org/10.1093/dnares/dst001'>https://doi.org/10.1093/dnares/dst001</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/333880?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/333880?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/333880?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/333880?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/333880?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/333880?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/333880?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Gene%20Capture%20Coupled%20to%20High-Throughput%20Sequencing%20as%20a%20Strategy%20for%20Targeted%20Metagenome%20Exploration&amp;publicationDate=2013-01-30&amp;volumeNum=20&amp;issueNum=2&amp;author=Denonfoux%2C%20J%C3%A9r%C3%A9mie%3B%20Parisot%2C%20Nicolas&amp;startPage=185&amp;endPage=196&amp;contentId=10.1093%2Fdnares%2Fdst001&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f2%2f185%2f333880" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=4627772 id="4627772" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Next-generation sequencing (NGS) allows faster acquisition of metagenomic data, but complete exploration of complex ecosystems is hindered by the extraordinary diversity of microorganisms. To reduce the environmental complexity, we created an innovative solution hybrid selection (SHS) method that is combined with NGS to characterize large DNA fragments harbouring biomarkers of interest. The quality of enrichment was evaluated after fragments containing the methyl coenzyme M reductase subunit A gene (<em>mcrA</em>), the biomarker of methanogenesis, were captured from a <em>Methanosarcina</em> strain and a metagenomic sample from a meromictic lake. The methanogen diversity was compared with direct metagenome and <em>mcrA</em>-based amplicon pyrosequencing strategies<em>.</em> The SHS approach resulted in the capture of DNA fragments up to 2.5 kb with an enrichment efficiency between 41 and 100%, depending on the sample complexity. Compared with direct metagenome and amplicons sequencing, SHS detected broader <em>mcrA</em> diversity, and it allowed efficient sampling of the rare biosphere and unknown sequences. In contrast to amplicon-based strategies, SHS is less biased and GC independent, and it recovered complete biomarker sequences in addition to conserved regions. Because this method can also isolate the regions flanking the target sequences, it could facilitate operon reconstructions.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;α-subunit of the methyl-coenzyme M reductase&quot;">α-subunit of the methyl-coenzyme M reductase</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="metagenomics">metagenomics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;sequence capture&quot;">sequence capture</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;454 pyrosequencing&quot;">454 pyrosequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;microbial diversity&quot;">microbial diversity</a></div>                            <h2 scrollto-destination=4627774 id="4627774" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Microorganisms are extremely diverse and crucial for healthy, functioning biospheres.<sup><a reveal-id="DST001C1" data-open="DST001C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DST001C2" data-open="DST001C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Although studies of isolated species have produced a great deal of information about microbial genetics, physiology, biotechnology and molecular biology, the diversity and structure of complex microbial communities are still poorly understood. This deficiency results from the inability to culture most microorganisms using standard microbiological techniques.<sup><a reveal-id="DST001C1" data-open="DST001C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DST001C3" data-open="DST001C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Consequently, although there are most likely millions of bacterial species on the planet, only a few thousand have been formally described.<sup><a reveal-id="DST001C4" data-open="DST001C4" class="link link-ref link-reveal xref-bibr">4</a></sup></p><p>Culture-independent techniques, such as metagenomics,<sup><a reveal-id="DST001C5" data-open="DST001C5" class="link link-ref link-reveal xref-bibr">5</a></sup> circumvent the problem of unculturability and transcend previous studies on individual organisms to focus on microbial communities present in an environment. Metagenomics has enriched our knowledge of environmental microbiology through the structural (gene/species richness and distribution)<sup><a reveal-id="DST001C6" data-open="DST001C6" class="link link-ref link-reveal xref-bibr">6</a></sup> and functional (metabolic)<sup><a reveal-id="DST001C7" data-open="DST001C7" class="link link-ref link-reveal xref-bibr">7</a></sup> profiling of complex environmental microbial communities. Based on unselective (shotgun) or targeted (activity driven and sequence driven) methods, metagenomics links genome information with structure and function relationships within microbial populations.<sup><a reveal-id="DST001C8" data-open="DST001C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DST001C9" data-open="DST001C9" class="link link-ref link-reveal xref-bibr">9</a></sup></p><p>Recently developed next-generation sequencing (NGS) technologies recover genetic materials from environmental samples without the preparation of metagenomic clone libraries.<sup><a reveal-id="DST001C10" data-open="DST001C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Furthermore, they explore a greater amount of sequence information because they have higher throughput and lower costs than other methods.<sup><a reveal-id="DST001C11" data-open="DST001C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Nevertheless, Quince <em>et al.</em><sup><a reveal-id="DST001C12" data-open="DST001C12" class="link link-ref link-reveal xref-bibr">12</a></sup> showed that covering 90% of the species richness in some hyper-diverse environments could require 10—1000-fold increases in the current NGS sequencing efforts. In addition, the massive amount of short metagenomic sequence reads (between 20 and 700 bases depending on the platform) can be problematic for assembling and identifying complete coding DNA sequence and/or operon structure.<sup><a reveal-id="DST001C13" data-open="DST001C13" class="link link-ref link-reveal xref-bibr">13</a></sup> One promising alternative is to reduce the environmental sample complexity by enriching the desired genomic target before sequencing.</p><p>Currently, several strategies of genomic-scale sequence enrichment have been reported.<sup><a reveal-id="DST001C14" data-open="DST001C14" class="link link-ref link-reveal xref-bibr">14</a></sup> The more efficient methods rely on complementary nucleic acid capture probes that hybridize to the targeted DNA sequences. Two hybridization methods—solid phase<sup><a reveal-id="DST001C15  DST001C17" data-open="DST001C15  DST001C17" class="link link-ref link-reveal xref-bibr">15–17</a></sup> and solution phase, also known as solution hybrid selection (SHS)<sup><a reveal-id="DST001C18" data-open="DST001C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DST001C19" data-open="DST001C19" class="link link-ref link-reveal xref-bibr">19</a></sup>—can be used to ascertain genetic variation by specifically enriching and resequencing regions from complex eukaryotic genomes.</p><p>To the best of our knowledge, only high-throughput enrichment methods based on polymerase chain reaction (PCR) have been applied to target functional genes in complex environments.<sup><a reveal-id="DST001C20" data-open="DST001C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Because no current methods use oligonucleotide capture probes to specifically enrich targeted genes from a complex environmental genomic DNA (gDNA), we applied this methodology in the context of microbial ecology (Fig. <a reveal-id="DST001F1" data-open="DST001F1" class="link link-reveal link-table xref-fig">1</a>A) to specifically capture DNA fragments harbouring known or unknown genetic biomarkers of interest (Fig. <a reveal-id="DST001F1" data-open="DST001F1" class="link link-reveal link-table xref-fig">1</a>B). We hypothesized that the use of variant specific and explorative probes<sup><a reveal-id="DST001C21" data-open="DST001C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DST001C22" data-open="DST001C22" class="link link-ref link-reveal xref-bibr">22</a></sup> would more accurately define the overall biomarker diversity (including the rare biosphere and unknown sequences) and would facilitate the discovery of genes linked to the target sequences <em>via</em> the reconstruction of adjacent DNA regions. This method should lead to better diversity coverage that is not influenced by PCR biases, as generally occurs in amplicon sequencing.<sup><a reveal-id="DST001C23" data-open="DST001C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DST001C24" data-open="DST001C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Because it is not limited to a specific DNA region (as in PCR enrichment), this strategy will increase the sequence coverage over target regions and lower the cost per target when compared with shotgun sequencing. </p>                        <a id="4627780" scrollto-destination="4627780"></a>
<div id="DST001F1" data-id="DST001F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst001/2/m_dst00101.jpeg?Expires=2147483647&Signature=zytrl-9FqQZZWOplFFk0iAIrxAe3BXRPXq-7PaT8Ifv5RHQiqyyB3KJotsVmD~MvXw30iTRKJdy74Lzcy9ef3TVW3nFQ9-hStUpDCU7vG6HR2S3Bv898lFAu-7yULQoQmqpB53znY0H-NW4tF1yUIKSCD0HbzP~oCHH3iAH3zv~1faKyjbrfDRw56g6drw~66VT9HirEbw4HpmLSuLlBAKT~kG1RuH9tsPsTXrjNwLZUqqtgCTVD9FzC53tW8fAx4czB4lDE7pF-hLumofFEah8kLWPqott7i7izqDCOME2tTfNVBv8vD7uFPyody7Nh3td0p8pBLg4TSMMX6gs1ww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic comparison of targeted capture methods applied to classical direct selection method of individual genomic loci (human for instance) (A) and our new approach for metagenomics targeting (B). The enrichment through microarray and the SHS of large genomic regions within complex eukaryotic genomes, as described in A, uses specific tiling probes to target resequencing genomic loci for copy number variation (CNV) and single nucleotide polymorphism detection. Our SHS method (B) uses the design of specific variants and explorative probes across a targeted biomarker to specifically enrich large DNA fragments from complex metagenomic DNA. Captured DNA fragments are sequenced to explore biomarker diversity and adjacent flanking regions. The red rectangles indicate the targeted regions." path-from-xml="dst00101.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4627780/dst00101.jpeg" path-from-xml="dst00101.jpeg" target="_blank">View large</a><a section="4627780" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst001/2/dst00101.jpeg?Expires=2147483647&Signature=hK9Y04TdLK3Gr5qv-VBAs6nNyL29NtKP1OZn1r7Y3qEajN7j6viiBZrFnlRzJix-TXkjN1rVKbipktOdPo56g40L~XWP7Zy7VtK7QdtcJNT~IiEdq2nPgGNz4esr7I5pwPiMuEmRXvButj1PUU684PV~qp6DbMiKovKh7Krr8R1YdR94ca~9vebByXnWGuCbcKrvhqbukXrivoQ5Ymwe4MLLwClNai9yC3l8LE6N3xFQdgf3gqAOZSrW1lDCIxkKpoLQQy3hsh5E9lWQyXsZN~qzg46arGixDG7XJDVGFneqcCnP6OPKaxASsvL2xpoG3Ow13Cug2yYb4ij3mSFuJQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4627780&ar=333880&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00101.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic comparison of targeted capture methods applied to classical direct selection method of individual genomic loci (human for instance) (A) and our new approach for metagenomics targeting (B). The enrichment through microarray and the SHS of large genomic regions within complex eukaryotic genomes, as described in A, uses specific tiling probes to target resequencing genomic loci for copy number variation (CNV) and single nucleotide polymorphism detection. Our SHS method (B) uses the design of specific variants and explorative probes across a targeted biomarker to specifically enrich large DNA fragments from complex metagenomic DNA. Captured DNA fragments are sequenced to explore biomarker diversity and adjacent flanking regions. The red rectangles indicate the targeted regions.</p></div></div><div content-id="DST001F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst001/2/m_dst00101.jpeg?Expires=2147483647&Signature=zytrl-9FqQZZWOplFFk0iAIrxAe3BXRPXq-7PaT8Ifv5RHQiqyyB3KJotsVmD~MvXw30iTRKJdy74Lzcy9ef3TVW3nFQ9-hStUpDCU7vG6HR2S3Bv898lFAu-7yULQoQmqpB53znY0H-NW4tF1yUIKSCD0HbzP~oCHH3iAH3zv~1faKyjbrfDRw56g6drw~66VT9HirEbw4HpmLSuLlBAKT~kG1RuH9tsPsTXrjNwLZUqqtgCTVD9FzC53tW8fAx4czB4lDE7pF-hLumofFEah8kLWPqott7i7izqDCOME2tTfNVBv8vD7uFPyody7Nh3td0p8pBLg4TSMMX6gs1ww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic comparison of targeted capture methods applied to classical direct selection method of individual genomic loci (human for instance) (A) and our new approach for metagenomics targeting (B). The enrichment through microarray and the SHS of large genomic regions within complex eukaryotic genomes, as described in A, uses specific tiling probes to target resequencing genomic loci for copy number variation (CNV) and single nucleotide polymorphism detection. Our SHS method (B) uses the design of specific variants and explorative probes across a targeted biomarker to specifically enrich large DNA fragments from complex metagenomic DNA. Captured DNA fragments are sequenced to explore biomarker diversity and adjacent flanking regions. The red rectangles indicate the targeted regions." path-from-xml="dst00101.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4627780/dst00101.jpeg" path-from-xml="dst00101.jpeg" target="_blank">View large</a><a section="4627780" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst001/2/dst00101.jpeg?Expires=2147483647&Signature=hK9Y04TdLK3Gr5qv-VBAs6nNyL29NtKP1OZn1r7Y3qEajN7j6viiBZrFnlRzJix-TXkjN1rVKbipktOdPo56g40L~XWP7Zy7VtK7QdtcJNT~IiEdq2nPgGNz4esr7I5pwPiMuEmRXvButj1PUU684PV~qp6DbMiKovKh7Krr8R1YdR94ca~9vebByXnWGuCbcKrvhqbukXrivoQ5Ymwe4MLLwClNai9yC3l8LE6N3xFQdgf3gqAOZSrW1lDCIxkKpoLQQy3hsh5E9lWQyXsZN~qzg46arGixDG7XJDVGFneqcCnP6OPKaxASsvL2xpoG3Ow13Cug2yYb4ij3mSFuJQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4627780&ar=333880&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00101.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic comparison of targeted capture methods applied to classical direct selection method of individual genomic loci (human for instance) (A) and our new approach for metagenomics targeting (B). The enrichment through microarray and the SHS of large genomic regions within complex eukaryotic genomes, as described in A, uses specific tiling probes to target resequencing genomic loci for copy number variation (CNV) and single nucleotide polymorphism detection. Our SHS method (B) uses the design of specific variants and explorative probes across a targeted biomarker to specifically enrich large DNA fragments from complex metagenomic DNA. Captured DNA fragments are sequenced to explore biomarker diversity and adjacent flanking regions. The red rectangles indicate the targeted regions.</p></div></div><p>In the present study, we describe the first adaptation of the SHS capture method for the selective enrichment of a target-specific biomarker from a complex environmental metagenome. Methane (CH<sub>4</sub>) is an important radiative trace gas responsible for the greenhouse effect, and a significant proportion (6–16%) of the global natural methane emissions are released from freshwater lakes.<sup><a reveal-id="DST001C25" data-open="DST001C25" class="link link-ref link-reveal xref-bibr">25</a></sup> We surveyed the methanogen diversity in a permanently stratified crater lake located in the French Massif Central (Lake Pavin). This original freshwater ecosystem is composed of an anoxic deep water layer (monimolimnion, ∼60–90 m depth) separated from the oxygenated upper layer (mixolimnion) by an intermediate layer (mesolimnion),<sup><a reveal-id="DST001C26" data-open="DST001C26" class="link link-ref link-reveal xref-bibr">26</a></sup> where both the sediments and the anoxic water column contribute to methane production.<sup><a reveal-id="DST001C27" data-open="DST001C27" class="link link-ref link-reveal xref-bibr">27</a></sup> We targeted the gene coding for the α-subunit of the methyl coenzyme M reductase (<em>mcrA</em>) that is involved in the final step of methanogenesis. This gene is arranged in a single transcriptional unit—the <em>mcr</em> operon—that is highly conserved among all methanogens.<sup><a reveal-id="DST001C28" data-open="DST001C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DST001C29" data-open="DST001C29" class="link link-ref link-reveal xref-bibr">29</a></sup> To highlight the broad benefits of the gene capture approach when compared with the more classical sequencing methods, three methods were used for pyrosequencing of an environmental sample: the SHS method, a classical random-shotgun metagenomic approach and an <em>mcrA</em>-targeted amplicon sequencing survey.</p>                            <h2 scrollto-destination=4627782 id="4627782" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=4627783 id="4627783" class="section-title" data-legacy-id=s2a>2.1. Capture probe design and synthesis</h3>
<p>Two sets of capture probes were designed. The first set targeted the <em>mcrA</em> gene from the <em>Methanosarcina acetivorans</em> C2A genome (GenBank accession no. AE010299), and the second set targeted the <em>mcrA</em> sequences pooled from environmental samples. The first set of capture probes consisted of six high specific 50-mer probes (P1–P6) targeting six distinct regions of the <em>M. acetivorans</em> C2A <em>mcrA</em> gene (Fig. <a reveal-id="DST001F2" data-open="DST001F2" class="link link-reveal link-table xref-fig">2</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). These probes were designed with HiSpOD software.<sup><a reveal-id="DST001C30" data-open="DST001C30" class="link link-ref link-reveal xref-bibr">30</a></sup> Adaptor sequences were added at each end, resulting in 80-mer hybrid probes consisting of 5′-ATCGCACCAGCGTGT(X)<sub>50</sub>CACTGCGGCTCCTCA-3′, with X<sub>50</sub> indicating the specific capture probe. </p>                        <a id="4627785" scrollto-destination="4627785"></a>
<div id="DST001F2" data-id="DST001F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst001/2/m_dst00102.jpeg?Expires=2147483647&Signature=c2KmW3FV63m90gB1vu-MmZ0bA4JlFytbUwTiZQJHQzwLr8kMKmeN-nO96qVII0pKdop2ZLC0m4th1Sae77j4ci1lXsDweVV-gzM7tJBvg1nqxejUyJTPaP57t-YOMYaJ4kQ6VtXaWiwI4Vi5oaF-kDlzPg5wQI5TM1SftsTuXpirUB6BTJ3WrH8nzcbdtRbILreOeJIiPAtTXCaS1ocJqlUBji6JacXi0kWzFz0gCd-6T1Y0iGOCSZcBxKI2Ygm~X7kvh75gM3sj8btpWpbB3uQU2gqig5hiv53f22~ttRbrAZJRNNjTxh8YwbTTXvwKCNIyCGvHayIpXHJa9tBdcA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic representation of mcr operon fragments on (A) M. acetivorans C2A gDNA and (B) Lake Pavin metagenomic DNA. Primer pairs used for fmdA (1) and mcrA (2) quantification as well as mcrA-fmdC region (3) amplification are symbolized. Dashed arrows indicate the sequence coverage of each of the five clones retrieved from the environmental sample (B). P1–P6: Positions of the six capture probes in the mcrA gene of M. acetivorans (see Supplementary Table S1 for probe sequences)." path-from-xml="dst00102.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4627785/dst00102.jpeg" path-from-xml="dst00102.jpeg" target="_blank">View large</a><a section="4627785" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst001/2/dst00102.jpeg?Expires=2147483647&Signature=YLOyU6L5lZq~FN~t2yxgPmhNi6gCqqvtPsYIfZi~nHjH8bAHHs~cGkZyI9BLybZLHZ26DAeazn9As6~BZcBxEiCh4DuaVw44s8U~-F5FKrsLaV6~mHukwujhXKtBjTkVBwy-50RFJtXmmN8rZolCKMkATwu08BVZ32h6fErp0i-zOmjsP5iT39u5eiDFofgALKarBCIEDcw4BQPjBz1UdF016wW9rru649dtsvty1ybVW0L47M5sqQyP9O8FonhBclV0bImhxkbSuV4Z~JJnQtyy0~3IljR4pNsXNf-PTUZSdTvfbO6noswG8YI7~tOCjmewvinoBdgBw2zhsv7nfw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4627785&ar=333880&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00102.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic representation of <em>mcr</em> operon fragments on (A) <em>M. acetivorans</em> C2A gDNA and (B) Lake Pavin metagenomic DNA. Primer pairs used for <em>fmdA</em> (1) and <em>mcrA</em> (2) quantification as well as <em>mcrA</em>-<em>fmdC</em> region (3) amplification are symbolized. Dashed arrows indicate the sequence coverage of each of the five clones retrieved from the environmental sample (B). P1–P6: Positions of the six capture probes in the <em>mcrA</em> gene of <em>M. acetivorans</em> (see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a> for probe sequences).</p></div></div><div content-id="DST001F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst001/2/m_dst00102.jpeg?Expires=2147483647&Signature=c2KmW3FV63m90gB1vu-MmZ0bA4JlFytbUwTiZQJHQzwLr8kMKmeN-nO96qVII0pKdop2ZLC0m4th1Sae77j4ci1lXsDweVV-gzM7tJBvg1nqxejUyJTPaP57t-YOMYaJ4kQ6VtXaWiwI4Vi5oaF-kDlzPg5wQI5TM1SftsTuXpirUB6BTJ3WrH8nzcbdtRbILreOeJIiPAtTXCaS1ocJqlUBji6JacXi0kWzFz0gCd-6T1Y0iGOCSZcBxKI2Ygm~X7kvh75gM3sj8btpWpbB3uQU2gqig5hiv53f22~ttRbrAZJRNNjTxh8YwbTTXvwKCNIyCGvHayIpXHJa9tBdcA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic representation of mcr operon fragments on (A) M. acetivorans C2A gDNA and (B) Lake Pavin metagenomic DNA. Primer pairs used for fmdA (1) and mcrA (2) quantification as well as mcrA-fmdC region (3) amplification are symbolized. Dashed arrows indicate the sequence coverage of each of the five clones retrieved from the environmental sample (B). P1–P6: Positions of the six capture probes in the mcrA gene of M. acetivorans (see Supplementary Table S1 for probe sequences)." path-from-xml="dst00102.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4627785/dst00102.jpeg" path-from-xml="dst00102.jpeg" target="_blank">View large</a><a section="4627785" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst001/2/dst00102.jpeg?Expires=2147483647&Signature=YLOyU6L5lZq~FN~t2yxgPmhNi6gCqqvtPsYIfZi~nHjH8bAHHs~cGkZyI9BLybZLHZ26DAeazn9As6~BZcBxEiCh4DuaVw44s8U~-F5FKrsLaV6~mHukwujhXKtBjTkVBwy-50RFJtXmmN8rZolCKMkATwu08BVZ32h6fErp0i-zOmjsP5iT39u5eiDFofgALKarBCIEDcw4BQPjBz1UdF016wW9rru649dtsvty1ybVW0L47M5sqQyP9O8FonhBclV0bImhxkbSuV4Z~JJnQtyy0~3IljR4pNsXNf-PTUZSdTvfbO6noswG8YI7~tOCjmewvinoBdgBw2zhsv7nfw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4627785&ar=333880&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00102.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic representation of <em>mcr</em> operon fragments on (A) <em>M. acetivorans</em> C2A gDNA and (B) Lake Pavin metagenomic DNA. Primer pairs used for <em>fmdA</em> (1) and <em>mcrA</em> (2) quantification as well as <em>mcrA</em>-<em>fmdC</em> region (3) amplification are symbolized. Dashed arrows indicate the sequence coverage of each of the five clones retrieved from the environmental sample (B). P1–P6: Positions of the six capture probes in the <em>mcrA</em> gene of <em>M. acetivorans</em> (see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a> for probe sequences).</p></div></div><p>The second set of capture probes was 26 oligos (1 49-mer and 25 50-mers) designed to target <em>mcrA</em> and <em>mrtA</em> (encoding the α-subunit of the methyl coenzyme M reductase isoform II), but not the <em>mcrA</em> of anaerobic methanotrophs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>).</p><p>Oligonucleotides were purchased from Eurogentec S.A. (Belgium). The RNA probe was prepared as described by Gnirke <em>et al.</em><sup><a reveal-id="DST001C19" data-open="DST001C19" class="link link-ref link-reveal xref-bibr">19</a></sup></p>                            <h3 scrollto-destination=4627788 id="4627788" class="section-title" data-legacy-id=s2b>2.2. Preparation of biological samples and libraries</h3>
<p>The two biological models used in this study were the <em>M. acetivorans</em> C2A strain (DSM 2834) and Lake Pavin, located in the French Massif Central (45°29′74″N, 2°53′28″E). The <em>M. acetivorans</em> C2A strain was cultivated using the medium 304 (<a class="link link-uri" href="http://www.dsmz.de/microorganisms/medium/pdf/DSMZ_Medium304.pdf">http://www.dsmz.de/microorganisms/medium/pdf/DSMZ_Medium304.pdf</a>) according to the manufacturer's instructions. gDNA from the strain was extracted using the Easy DNA kit (Invitrogen), whereas environmental DNA was extracted from 350 ml of freshwater collected from Lake Pavin at a 90-m depth, as described by Dugat-Bony <em>et al.</em><sup><a reveal-id="DST001C30" data-open="DST001C30" class="link link-ref link-reveal xref-bibr">30</a></sup></p><p>Libraries were prepared using Roche's GS FLX Titanium General Library Preparation Kit (Roche Applied Science) according to the manufacturer's instructions. First, 5 µg of DNA was sheared by nebulization. DNA fragments were size selected with AMPure beads (Beckman Coulter Genomics). After purification, fragment end polishing, adaptor ligation (A and B adapter keys; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>) and fill-in reactions, the libraries were PCR amplified with the 454 Ti-A and 454 Ti-B primers (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The cycle conditions were 3 min at 93°C followed by 20 cycles of 15 s at 93°C, 1 min at 58°C and 8 min at 68°C and a final elongation step at 68°C for 6 min. The amplified libraries were purified with AMPure beads and stored at −20°C until use.</p><p>For the amplicon library, <em>mcrA</em> fragments were PCR amplified from total community DNA with the <em>mcrA</em>-specific primer pair MM_01/MM_02<sup><a reveal-id="DST001C31" data-open="DST001C31" class="link link-ref link-reveal xref-bibr">31</a></sup> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The amplicon was run on a 2% (wt/vol) agarose gel, and the ∼500 bp product was purified with a QIAquick gel extraction kit (Qiagen) and AMPure beads. Each DNA library was quantified by fluorometry with a Quant-iT PicoGreen dsDNA assay kit (Invitrogen). The DNA quality and size distribution were assessed on an Agilent Bioanalyzer High Sensitivity DNA chip (Agilent Technologies).</p>                            <h3 scrollto-destination=4627792 id="4627792" class="section-title" data-legacy-id=s2c>2.3. Hybridization capture and elution</h3>
<p>For each SHS-capture method library, 2.5 µg of salmon sperm DNA (Ambion) and 500 ng of DNA library were mixed (7 µl final volume), denatured for 5 min at 95°C, incubated for 5 min at 65°C before adding 13 µl of prewarmed (65°C) hybridization buffer (10X SSPE, 10X Denhardt's Solution, 10 mM EDTA and 0.2% SDS) and 6 µl freshly prepared, prewarmed (2 min at 65°C) biotinylated RNA probes (500 ng). After 24 h at 65°C, 500 ng of washed M-280 Dynabeads coated with streptavidin (Invitrogen) were added to the hybridization mix that was incubated for 30 min at room temperature (RT). The beads were precipitated with a magnetic stand (Ambion) and washed once for 15 min at RT with 500 µl 1X SSC/0.1% SDS and three times for 10 min at 65°C with 500 µl prewarmed 0.1X SSC/0.1% SDS. The captured DNA was eluted with 50 µl 0.1 M NaOH for 10 min at RT. After magnetic bead precipitation, the DNA supernatant was transferred to a sterile tube containing 70 µl of 1 M Tris–HCl pH 7.5, purified on a QIAquick column (Qiagen) and eluted in a final volume of 20 µl. A 2.5 µl aliquot was subjected to 15 cycles of PCR amplification using the 454 Ti-A and Ti-B primers as described above. After purification, a second round of capture was performed from each first-round PCR product. To increase the DNA yield, a final PCR amplification consisting of 20 cycles was performed. The final product was purified on a QIAquick column (Qiagen) and quantified with a NanoDrop ND-1000 spectrophotometer (NanoDrop Technologies).</p>                            <h3 scrollto-destination=4627794 id="4627794" class="section-title" data-legacy-id=s2d>2.4. Sanger sequencing and data analysis</h3>
<p>PCR products were cloned using the TOPO TA cloning kit (Invitrogen). Plasmids were screened for high-size inserts by digestion with <em>Eco</em>RI, and positive clones were Sanger sequenced at MWG DNA sequencing services (Ebersberg, Germany). Sequences were processed and joined using the Staden package program,<sup><a reveal-id="DST001C32" data-open="DST001C32" class="link link-ref link-reveal xref-bibr">32</a></sup> and primer sequences were removed from paired-end consensus sequences. The <em>mcr</em> sequence data retrieved from Lake Pavin by the SHS method were deposited in the GenBank database under accession numbers JQ404494, JQ404495, JQ404496, JQ404497 and JQ404498, and the sequence of the <em>mcrA</em>-<em>fmd</em> region-spanning fragment was deposited under accession number JQ425691.</p>                            <h3 scrollto-destination=4627796 id="4627796" class="section-title" data-legacy-id=s2e>2.5. 454 GS FLX Titanium sequencing and data analysis</h3>
<p>DNA samples were sequenced using the GS FLX Titanium system on the ‘GINA’ platform (part of the GENTYANE platform, labelled IBISA since 2009; BP 392, 63 011 Clermont-Ferrand, France) at the Centre Jean Perrin, according to the manufacturer's specifications. For quality filtering and de-replication of reads, sequences were trimmed with the PRINSEQ-lite PERL script<sup><a reveal-id="DST001C33" data-open="DST001C33" class="link link-ref link-reveal xref-bibr">33</a></sup> using the parameters described in the preprocessing chart (<a class="link link-uri" href="http://prinseq.sourceforge.net/Preprocessing_454_SFF_chart.pdf">http://prinseq.sourceforge.net/Preprocessing_454_SFF_chart.pdf</a>).</p><p>Functional assignment and enrichment were assessed with a BLASTX query<sup><a reveal-id="DST001C34" data-open="DST001C34" class="link link-ref link-reveal xref-bibr">34</a></sup> against a database containing 12 603 McrA protein sequences downloaded from the Genbank database (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/">http://www.ncbi.nlm.nih.gov/</a>), using WWW-Query (<a class="link link-uri" href="http://pbil.univ-lyon1.fr/search/query_fam.php">http://pbil.univ-lyon1.fr/search/query_fam.php</a>) to perform an advanced keyword search. Reads showing &gt;40% identity over 100 or more amino acids were classified as McrA sequences. Chimaera detection was performed with the UCHIME program<sup><a reveal-id="DST001C35" data-open="DST001C35" class="link link-ref link-reveal xref-bibr">35</a></sup> with a stringent threshold score of five. Sequences containing possible frameshifts were identified with the ‘–w 20’ BLAST option and disabled low complexity filters. Amino acid sequences without frameshifts were extracted from the BLAST results, and only the sequences that passed this filter were chosen for further phylogenetic analysis.</p><p>The sequence data were deposited in the NCBI as a Short Read Archive (SRA) project under accession no. SRA049219.</p>                            <h3 scrollto-destination=4627800 id="4627800" class="section-title" data-legacy-id=s2f>2.6. Phylogenetic analysis and tree construction</h3>
<p>All McrA sequences derived from the SHS method and metagenomic libraries were aligned to a sequence obtained from the amplicon library. The amino acid alignment used the ClustalW2 alignment method<sup><a reveal-id="DST001C36" data-open="DST001C36" class="link link-ref link-reveal xref-bibr">36</a></sup> driven by the Seaview version 4 program<sup><a reveal-id="DST001C37" data-open="DST001C37" class="link link-ref link-reveal xref-bibr">37</a></sup> to select the reads having at least 100 amino acids in common with this reference sequence. The overlapping regions of the remaining amino acid sequences, all amplicon pyrosequences and 29 McrA sequences previously identified from the same sampling depth and downloaded from GenBank (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/genbank/">http://www.ncbi.nlm.nih.gov/genbank/</a>) were fed to CD-HIT<sup><a reveal-id="DST001C38" data-open="DST001C38" class="link link-ref link-reveal xref-bibr">38</a></sup> that assigned them to operational taxonomic units (OTUs) using a complete linkage clustering method at a 91% cut-off value.<sup><a reveal-id="DST001C27" data-open="DST001C27" class="link link-ref link-reveal xref-bibr">27</a>,<a reveal-id="DST001C39" data-open="DST001C39" class="link link-ref link-reveal xref-bibr">39</a></sup></p><p>One representative sequence of each OTU was chosen to build a phylogenetic tree (Seaview 4)<sup><a reveal-id="DST001C37" data-open="DST001C37" class="link link-ref link-reveal xref-bibr">37</a></sup> using the neighbour-joining method<sup><a reveal-id="DST001C40" data-open="DST001C40" class="link link-ref link-reveal xref-bibr">40</a>,<a reveal-id="DST001C41" data-open="DST001C41" class="link link-ref link-reveal xref-bibr">41</a></sup> and 1000 bootstrapped trials. Closely related sequences available from GenBank (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/">http://www.ncbi.nlm.nih.gov/</a>) were included in the phylogenetic trees to decipher the microbial community diversity. A final tree was drawn in MEGA version 5.<sup><a reveal-id="DST001C42" data-open="DST001C42" class="link link-ref link-reveal xref-bibr">42</a></sup></p>                            <h3 scrollto-destination=4627803 id="4627803" class="section-title" data-legacy-id=s2g>2.7. qPCR assays for enrichment and methanogen abundance</h3>
<p>The assays were conducted in 20 µl with 5 µl of DNA sample or <em>mcrA</em> PCR product standards (covering a dynamic range of 5 × 10<sup>7</sup> to 50 copies), 10 µl of 2X MESA Green quantitative Polymerase Chain Reaction (qPCR) for SYBR assay mixture (Eurogentec S.A) and 0.2 μM forward and reverse primers. The thermo cycling protocol included an initial step of 95°C for 5 min, followed by 40 cycles of denaturation at 95°C for 15 s, annealing at the melting temperature of each primer set for 15 s and elongation at 68°C for 30 s. The samples and each point of the standard curve were quantified in triplicate. The primer sets are described in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>. The data were analysed with Realplex software version 1.5 (Eppendorf Inc.) and MxPro qPCR software 4.10d (Agilent technologies). Based on the ΔΔCt method,<sup><a reveal-id="DST001C43" data-open="DST001C43" class="link link-ref link-reveal xref-bibr">43</a></sup> relative enrichments (<em>R</em>) were calculated according to <em>R</em> = 2<sup>−ΔΔCt</sup>. The relative quantification method established a mean Ct value comparison (ΔCt) between <em>mcrA</em> (target gene) and <em>fmdA</em> (non-target gene 500 kb upstream from <em>mcrA</em>). The relative capture enrichment was determined by the comparison of ΔCt before and after capture, and this result described the fold change or ΔΔCt.</p>                            <h3 scrollto-destination=4627805 id="4627805" class="section-title" data-legacy-id=s2h>2.8. SHS <em>de novo</em> read assembly</h3>
<p>The filtered SHS reads were assembled with Newbler version 2.6 (Roche Applied Science) using stringent assembly parameters (60 bases overlap and 95% overlap identity) and the ‘- rip’ option that forces Newbler to place each read into one unique contig. The functional assignment of contigs and singletons was performed by a BLASTX query<sup><a reveal-id="DST001C34" data-open="DST001C34" class="link link-ref link-reveal xref-bibr">34</a></sup> against our database containing 12 603 McrA protein sequences. Chimaeras were detected in the <em>mcrA</em> contigs and singletons with the UCHIME program<sup><a reveal-id="DST001C35" data-open="DST001C35" class="link link-ref link-reveal xref-bibr">35</a></sup> and a stringent threshold score of five. Prediction of the <em>mcrA</em> gene location within contigs and singletons was performed by BLASTN<sup><a reveal-id="DST001C44" data-open="DST001C44" class="link link-ref link-reveal xref-bibr">44</a></sup> against the reference genomes of <em>Candidatus Methanoregula boonei</em> 6A8 (<em>Methanomicrobiales</em> order, accession no. NC_009712), <em>Methanosaeta concilii</em> GP-6 (<em>Methanosarcinales</em> order, accession no. CP002565) and <em>Methanosphaera stadtmanae</em> DSM 3091 (<em>Methanobacteriales</em> order, accession no. CP000102). Contigs extending at least 100 nucleotides beyond <em>mcrA</em> were segregated for BLASTX<sup><a reveal-id="DST001C34" data-open="DST001C34" class="link link-ref link-reveal xref-bibr">34</a></sup> analysis against the non-redundant (nr) protein sequences database to identify putative open-reading frames within the flanking regions.</p><p>The sequence data from homologous <em>mcrA</em> contigs (without chimaeras or frameshifts) were deposited in the GenBank database under accession no. KC184908 to KC185399.</p>                            <h2 scrollto-destination=4627808 id="4627808" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=4627809 id="4627809" class="section-title" data-legacy-id=s3a>3.1. Development of an SHS method for genomic-scale sequence enrichment</h3>
                            <h4 scrollto-destination=4627810 id="4627810" class="section-title" data-legacy-id=s3a1>3.1.1. Method validation: mcrA gene enrichment from <em>M. acetivorans</em> C2A gDNA</h4>
<p>We performed the initial validation of our enrichment strategy by capturing the <em>mcrA</em> gene from a 1 to 3 kb fragment library of the completely sequenced methanogenic <em>M. acetivorans</em> C2A strain. The minimal probe set spanned different non-overlapping regions of the gene (Fig. <a reveal-id="DST001F2" data-open="DST001F2" class="link link-reveal link-table xref-fig">2</a>A). The qPCR reactions revealed a 461-fold relative enrichment of <em>mcrA</em> sequences after the first cycle of capture and at least 175 365-fold enrichment after the second cycle. Furthermore, as the <em>M. acetivorans</em> C2A genome consists of 5751 kb with a single <em>mcrA</em> gene copy, the probability of randomly sequencing this gene from a 1 to 3 kb fragment size clone library is 0.02–0.05%. Using our solution-based DNA capture-enrichment method and working on an isolated species, the likelihood increased from 7.8 to 23% after the first cycle and could reach 100% after the second.</p><p>The DNA sequence of fragments retrieved after the second cycle of capture was controlled by the cloning-sequencing method. Six clones were sequenced, and all had a perfect correspondence to the <em>mcrA</em> gene from <em>M. acetivorans</em> C2A, reinforcing the efficiency of the two iterative cycles of capture. The captured fragments were assembled into a 1834-bp contig containing the nearly complete <em>mcrA</em> gene (1645 bp) and its 3′ non-coding region (189 bp). After validating this approach, we further tested the performance of the method by enriching <em>mcrA</em> sequences from a complex methanogenic freshwater environment.</p>                            <h4 scrollto-destination=4627813 id="4627813" class="section-title" data-legacy-id=s3a2>3.1.2. Environmental application: mcrA sequence enrichment from a methanogenic lacustrine environment (Lake Pavin)</h4>
<p>The freshwater sample was collected in the anoxic zone at 90 m depth, where the highest methanogen diversity was available in the lacustrine environment.<sup><a reveal-id="DST001C27" data-open="DST001C27" class="link link-ref link-reveal xref-bibr">27</a></sup> An improved <em>mcrA</em> probe set included all known <em>mcrA</em> sequences and targeted new variants with explorative probes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). The efficiency of the <em>mcrA</em> enrichment was determined by cloning and sequencing the second capture product. Five out of the ten clones with large inserts (2041–2493 bp) included <em>mcrA</em> sequences. All positive clones had a ∼1500 bp common zone corresponding to the <em>mcrA</em> gene, but they also harboured upstream or downstream regions containing other genes (Fig. <a reveal-id="DST001F2" data-open="DST001F2" class="link link-reveal link-table xref-fig">2</a>B). BLAST analysis of the cloned sequences revealed that they are very similar (99% similarity) to <em>mcrA</em> sequences previously retrieved from this ecosystem (accession nos. GQ389949, GQ389912 and GQ389806).<sup><a reveal-id="DST001C27" data-open="DST001C27" class="link link-ref link-reveal xref-bibr">27</a></sup> The closest relative to the <em>mcrA</em>, <em>mcrG</em> and partial <em>mcrC</em> sequences were from a cultured methanogen, <em>Candidatus Methanoregula boonei</em> 6A8 (&gt;85, 84 and 81% similarity, respectively). This hydrogenotrophic species belongs to the <em>Methanomicrobiales</em> order, and it was isolated from an acidic peat bog.<sup><a reveal-id="DST001C45" data-open="DST001C45" class="link link-ref link-reveal xref-bibr">45</a></sup> Furthermore, the <em>fmdC</em> gene fragment identified 821 bp downstream the target gene (Fig. <a reveal-id="DST001F2" data-open="DST001F2" class="link link-reveal link-table xref-fig">2</a>B) that shared 77% identity with subunit C of the formyl methanofuran dehydrogenase gene of this species. This gene has been located in the reference genome (GenBank: CP000780.1) at almost 300 kb from the <em>mcr</em> operon. It should be noted that this genome organization—with the <em>fmd</em> operon located just downstream from the <em>mcr</em> operon—has not been described previously in methanogens. To exclude the possibility of chimaera formation during metagenomic library amplification, a PCR fragment spanning the <em>mcrA</em>–<em>fmdC</em> region was obtained directly from the initial metagenomic DNA sample, using two specific primers (Fig. <a reveal-id="DST001F2" data-open="DST001F2" class="link link-reveal link-table xref-fig">2</a>B, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The sequenced 821 bp PCR product (JQ425691) confirmed the organization revealed by the SHS method (100% identity with the captured DNA fragment).</p><p>Our results showed that the capture method not only efficiently enriched targets out of a complex environmental genomic mixture, but also recovered sequences adjacent to the targeted biomarker gene. Additionally, the SHS method was coupled with NGS technologies to assess the coverage of archaeal <em>mcrA</em> diversity in a complex ecosystem.</p>                            <h3 scrollto-destination=4627816 id="4627816" class="section-title" data-legacy-id=s3b>3.2. Metagenome exploration with genome-scale sequence enrichment and NGS</h3>
<p>The benefit of the SHS method in terms of diversity coverage, when compared with more classical approaches, was further examined by sequencing the SHS capture products. A new random-shotgun DNA metagenomic library adapted for pyrosequencing (fragment sizes ∼500 bp) was prepared for the SHS products and for direct sequencing (shotgun metagenomics approach). From the same metagenomic DNA sample, <em>mcrA</em> PCR products were also amplified with the primer set MM_01-MM_02,<sup><a reveal-id="DST001C31" data-open="DST001C31" class="link link-ref link-reveal xref-bibr">31</a></sup> generating amplicons of ∼500 bp. Sequencing (captured DNA fragments, metagenome and amplicons) was performed with the 454 GS FLX Titanium technology, generating a slightly different amount of raw data with an average read length of 414–471 bases. After pre-processing, sequencing datasets from all three approaches had nearly equivalent numbers of reads (Table <a reveal-id="DST001TB1" data-open="DST001TB1" class="link link-reveal link-table xref-fig">1</a>). </p>                        <a id="4627818" scrollto-destination="4627818"></a>
<div content-id="DST001TB1" class="table-modal"><div class="table-wrap"><div id="DST001TB1" class="table-wrap-title" data-id="DST001TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Summary statistics from 454 pyrosequencing</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Metagenome </th><th>Amplicons </th><th>SHS </th></tr></thead><tbody><tr><td>Total number of raw reads </td><td>136 256 </td><td>121 665 </td><td>177 977 </td></tr><tr><td>Number of reads after pre-processing </td><td>116 365 </td><td>119 437 </td><td>122 772 </td></tr><tr><td>Average length of cleaned reads (bases) </td><td>471 </td><td>414 </td><td>454 </td></tr><tr><td><em>mcrA</em> homologous sequences<sup>a</sup> </td><td>3 </td><td>119 409 </td><td>50 727 </td></tr><tr><td>Enrichment performance (%) </td><td>0.003 </td><td>99.98 </td><td>41.32 </td></tr><tr><td>Number of chimaeras </td><td>0 </td><td>150 </td><td>30 </td></tr><tr><td>Number of reads containing frameshifts </td><td>1 </td><td>80 390 </td><td>21 855 </td></tr><tr><td>Number of high-quality <em>mcrA</em> homologous sequences (without chimaera and frameshifts) </td><td>2 </td><td>38 869 </td><td>28 842 </td></tr><tr><td><em>McrA</em> sequences used for methanogenic diversity and abundance (comparison of a common region) </td><td>1 </td><td>38 807 </td><td>11 442 </td></tr><tr><td>Number of OTUs </td><td>1 </td><td>40 </td><td>44 </td></tr><tr><td><em>McrA</em> sequences related to OTUs </td><td>1 </td><td>38 784<sup>b</sup> </td><td>11 324<sup>b</sup> </td></tr><tr><td>Relative abundance of <em>mcrA</em> sequences affiliated with <em>Methanomicrobiales</em> (%) </td><td>0 </td><td>98.57 </td><td>98.82 </td></tr><tr><td>Relative abundance of <em>mcrA</em> sequences affiliated with <em>Methanosarcinales</em> (%) </td><td>0 </td><td>0.005 </td><td>0.86 </td></tr><tr><td>Relative abundance of <em>mcrA</em> sequences affiliated with the Novel Order (%) </td><td>100 </td><td>1.43 </td><td>0.13 </td></tr><tr><td>Relative abundance of <em>mcrA</em> sequences affiliated with <em>Methanobacteriales</em> (%) </td><td>0 </td><td>0 </td><td>0.19 </td></tr></tbody></table></div><div content-id="DST001TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Metagenome </th><th>Amplicons </th><th>SHS </th></tr></thead><tbody><tr><td>Total number of raw reads </td><td>136 256 </td><td>121 665 </td><td>177 977 </td></tr><tr><td>Number of reads after pre-processing </td><td>116 365 </td><td>119 437 </td><td>122 772 </td></tr><tr><td>Average length of cleaned reads (bases) </td><td>471 </td><td>414 </td><td>454 </td></tr><tr><td><em>mcrA</em> homologous sequences<sup>a</sup> </td><td>3 </td><td>119 409 </td><td>50 727 </td></tr><tr><td>Enrichment performance (%) </td><td>0.003 </td><td>99.98 </td><td>41.32 </td></tr><tr><td>Number of chimaeras </td><td>0 </td><td>150 </td><td>30 </td></tr><tr><td>Number of reads containing frameshifts </td><td>1 </td><td>80 390 </td><td>21 855 </td></tr><tr><td>Number of high-quality <em>mcrA</em> homologous sequences (without chimaera and frameshifts) </td><td>2 </td><td>38 869 </td><td>28 842 </td></tr><tr><td><em>McrA</em> sequences used for methanogenic diversity and abundance (comparison of a common region) </td><td>1 </td><td>38 807 </td><td>11 442 </td></tr><tr><td>Number of OTUs </td><td>1 </td><td>40 </td><td>44 </td></tr><tr><td><em>McrA</em> sequences related to OTUs </td><td>1 </td><td>38 784<sup>b</sup> </td><td>11 324<sup>b</sup> </td></tr><tr><td>Relative abundance of <em>mcrA</em> sequences affiliated with <em>Methanomicrobiales</em> (%) </td><td>0 </td><td>98.57 </td><td>98.82 </td></tr><tr><td>Relative abundance of <em>mcrA</em> sequences affiliated with <em>Methanosarcinales</em> (%) </td><td>0 </td><td>0.005 </td><td>0.86 </td></tr><tr><td>Relative abundance of <em>mcrA</em> sequences affiliated with the Novel Order (%) </td><td>100 </td><td>1.43 </td><td>0.13 </td></tr><tr><td>Relative abundance of <em>mcrA</em> sequences affiliated with <em>Methanobacteriales</em> (%) </td><td>0 </td><td>0 </td><td>0.19 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>BLASTX parameters: percentage of identity: 40%; E-value cut-off: 10.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>McrA sequences related to OTUs containing more than one sequence.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4627818" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST001TB1" class="table-wrap-title" data-id="DST001TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Summary statistics from 454 pyrosequencing</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Metagenome </th><th>Amplicons </th><th>SHS </th></tr></thead><tbody><tr><td>Total number of raw reads </td><td>136 256 </td><td>121 665 </td><td>177 977 </td></tr><tr><td>Number of reads after pre-processing </td><td>116 365 </td><td>119 437 </td><td>122 772 </td></tr><tr><td>Average length of cleaned reads (bases) </td><td>471 </td><td>414 </td><td>454 </td></tr><tr><td><em>mcrA</em> homologous sequences<sup>a</sup> </td><td>3 </td><td>119 409 </td><td>50 727 </td></tr><tr><td>Enrichment performance (%) </td><td>0.003 </td><td>99.98 </td><td>41.32 </td></tr><tr><td>Number of chimaeras </td><td>0 </td><td>150 </td><td>30 </td></tr><tr><td>Number of reads containing frameshifts </td><td>1 </td><td>80 390 </td><td>21 855 </td></tr><tr><td>Number of high-quality <em>mcrA</em> homologous sequences (without chimaera and frameshifts) </td><td>2 </td><td>38 869 </td><td>28 842 </td></tr><tr><td><em>McrA</em> sequences used for methanogenic diversity and abundance (comparison of a common region) </td><td>1 </td><td>38 807 </td><td>11 442 </td></tr><tr><td>Number of OTUs </td><td>1 </td><td>40 </td><td>44 </td></tr><tr><td><em>McrA</em> sequences related to OTUs </td><td>1 </td><td>38 784<sup>b</sup> </td><td>11 324<sup>b</sup> </td></tr><tr><td>Relative abundance of <em>mcrA</em> sequences affiliated with <em>Methanomicrobiales</em> (%) </td><td>0 </td><td>98.57 </td><td>98.82 </td></tr><tr><td>Relative abundance of <em>mcrA</em> sequences affiliated with <em>Methanosarcinales</em> (%) </td><td>0 </td><td>0.005 </td><td>0.86 </td></tr><tr><td>Relative abundance of <em>mcrA</em> sequences affiliated with the Novel Order (%) </td><td>100 </td><td>1.43 </td><td>0.13 </td></tr><tr><td>Relative abundance of <em>mcrA</em> sequences affiliated with <em>Methanobacteriales</em> (%) </td><td>0 </td><td>0 </td><td>0.19 </td></tr></tbody></table></div><div content-id="DST001TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Metagenome </th><th>Amplicons </th><th>SHS </th></tr></thead><tbody><tr><td>Total number of raw reads </td><td>136 256 </td><td>121 665 </td><td>177 977 </td></tr><tr><td>Number of reads after pre-processing </td><td>116 365 </td><td>119 437 </td><td>122 772 </td></tr><tr><td>Average length of cleaned reads (bases) </td><td>471 </td><td>414 </td><td>454 </td></tr><tr><td><em>mcrA</em> homologous sequences<sup>a</sup> </td><td>3 </td><td>119 409 </td><td>50 727 </td></tr><tr><td>Enrichment performance (%) </td><td>0.003 </td><td>99.98 </td><td>41.32 </td></tr><tr><td>Number of chimaeras </td><td>0 </td><td>150 </td><td>30 </td></tr><tr><td>Number of reads containing frameshifts </td><td>1 </td><td>80 390 </td><td>21 855 </td></tr><tr><td>Number of high-quality <em>mcrA</em> homologous sequences (without chimaera and frameshifts) </td><td>2 </td><td>38 869 </td><td>28 842 </td></tr><tr><td><em>McrA</em> sequences used for methanogenic diversity and abundance (comparison of a common region) </td><td>1 </td><td>38 807 </td><td>11 442 </td></tr><tr><td>Number of OTUs </td><td>1 </td><td>40 </td><td>44 </td></tr><tr><td><em>McrA</em> sequences related to OTUs </td><td>1 </td><td>38 784<sup>b</sup> </td><td>11 324<sup>b</sup> </td></tr><tr><td>Relative abundance of <em>mcrA</em> sequences affiliated with <em>Methanomicrobiales</em> (%) </td><td>0 </td><td>98.57 </td><td>98.82 </td></tr><tr><td>Relative abundance of <em>mcrA</em> sequences affiliated with <em>Methanosarcinales</em> (%) </td><td>0 </td><td>0.005 </td><td>0.86 </td></tr><tr><td>Relative abundance of <em>mcrA</em> sequences affiliated with the Novel Order (%) </td><td>100 </td><td>1.43 </td><td>0.13 </td></tr><tr><td>Relative abundance of <em>mcrA</em> sequences affiliated with <em>Methanobacteriales</em> (%) </td><td>0 </td><td>0 </td><td>0.19 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>BLASTX parameters: percentage of identity: 40%; E-value cut-off: 10.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>McrA sequences related to OTUs containing more than one sequence.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4627818" target="_blank">View Large</a></div></div>                            <h4 scrollto-destination=4627819 id="4627819" class="section-title" data-legacy-id=s3b1>3.2.1. Functional assignment and enrichment performance</h4>
<p>Only three reads (0.003% of total reads) from the random-shotgun sequencing approach corresponded to the <em>mcrA</em> gene. For the SHS method, 50 727 reads were identified as <em>mcrA</em> sequences (41.32%), and almost all the amplicon approach sequences were from <em>mcrA</em> (119 409 reads, 99.98%).</p><p>For <em>mcrA</em> diversity evaluation, however, we only analysed high-quality sequences (no chimaeras or frameshifts), and all the problematic reads were subsequently excluded.</p>                            <h4 scrollto-destination=4627822 id="4627822" class="section-title" data-legacy-id=s3b2>3.2.2. Methanogen diversity and abundance</h4>
<p>The phylogeny of the methanogenic McrA protein sequences was investigated and compared for each of the three approaches. We used ClustalW2<sup><a reveal-id="DST001C36" data-open="DST001C36" class="link link-ref link-reveal xref-bibr">36</a></sup> to determine a common reference region of 143 amino acids shared by the largest number of McrA sequences retrieved from the 3 approaches. All McrA sequences that included this region were truncated so that at least 100 amino acids aligned with this reference. The resulting sequences, which included 1 read from the shotgun library, 11 442 reads from the SHS method library and 38 807 reads from the amplicon library, were used for further analysis. Furthermore, 29 additional sequences (referred to as Pavin90m) from a previous study<sup><a reveal-id="DST001C27" data-open="DST001C27" class="link link-ref link-reveal xref-bibr">27</a></sup> were included in the analysis.</p><p>Following the clustering method, 127 distinct OTUs (longer than 300 bp) were observed, and the 58 OTUs that contained more than 1 sequence were included in a more detailed phylogenetic analysis. The shotgun library sequence, which contained a single final read, was also included. Among these 58 OTUs, 44 were detected from the SHS method, 40 from the amplicon approach, 1 from the metagenomic shotgun library and 3 from Pavin90m sequences. The SHS method and amplicons shared 27 OTUs, including 3 from the Pavin90m sequences (Fig. <a reveal-id="DST001F3" data-open="DST001F3" class="link link-reveal link-table xref-fig">3</a>). The remaining 31 OTUs were specific to a single method, with 1 for the metagenome, 17 for the SHS and 13 for the amplicons (Fig. <a reveal-id="DST001F3" data-open="DST001F3" class="link link-reveal link-table xref-fig">3</a>). </p>                        <a id="4627825" scrollto-destination="4627825"></a>
<div id="DST001F3" data-id="DST001F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst001/2/m_dst00103.jpeg?Expires=2147483647&Signature=27vitSpoeqrJ15aiDCAB6tEI2-yd7d6HWJyiyCuv2d1~~qNZ9lpxslvKMUQXVT3bgWzSG~M1dAyClUc961C7pTHTb1eZQTl~opowcPcsLMX6cisFGOkpGFEv4d5lzsteYr~QhEcugOFbfsRRrLkBx1YovMogAGAQkHDkJFbYFWobBZwo0Tg1m9ffcCgmQ5zl2o9Snx1fxgOjO2asQ8Jqmqm0gBELdr5Ud6GCMKmPy1jhK~sfrl~pASelZXAqb4y1t3dZMxVbNlvRx8FG4YS0g-M5LqQcceKLY3jgGBqG4Wxv2x2HICYrZ~mnY0W0R4h4RuWj4pjJAcgRHOMNgMlxNw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Venn diagram showing the number of unique and shared OTUs for the in-solution capture method (SHS), PCR-based strategy (Amplicons) and sequences isolated at 90 m depth from a previous PCR-based study of Lake Pavin (Pavin90m).27 The Venn diagram was generated with Venny (http://bioinfogp.cnb.csic.es/tools/venny/index.html)." path-from-xml="dst00103.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4627825/dst00103.jpeg" path-from-xml="dst00103.jpeg" target="_blank">View large</a><a section="4627825" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst001/2/dst00103.jpeg?Expires=2147483647&Signature=uQ6WN3lBP1qi7GfQVUG~InqNmh8UbC0rF44M4CQRr2koYW7sWyh0qfnZrzOuWgNn5Hd78q2sDwGbfzO-uJq4-PkYArfSj1d9PNVTKh68gecy4HANjsgZh9FkeBFv3jDV~TEOL1h3cUpMVKq4ooYQZEKzUHvUaXCBxlVa8OQmMwandBL8ua2refCKokJki9Db1NJRh4cTe5yIG5b30mwSssHp1FmkZKsCMpmrCVJs1TaxDPaprk9omHfgLkhXDaCBF1ZbcqK2YXKTCCNr5aXA8Z1IIHr7P~huZ7JE0nJl1gzXXi~aV5pO6BmggrmqObIpVgeK2uRXwxstPWvQ38F00Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4627825&ar=333880&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00103.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Venn diagram showing the number of unique and shared OTUs for the in-solution capture method (SHS), PCR-based strategy (Amplicons) and sequences isolated at 90 m depth from a previous PCR-based study of Lake Pavin (Pavin90m).<sup><a reveal-id="DST001C27" data-open="DST001C27" class="link link-ref link-reveal xref-bibr">27</a></sup> The Venn diagram was generated with Venny (<a class="link link-uri" href="http://bioinfogp.cnb.csic.es/tools/venny/index.html">http://bioinfogp.cnb.csic.es/tools/venny/index.html</a>).</p></div></div><div content-id="DST001F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst001/2/m_dst00103.jpeg?Expires=2147483647&Signature=27vitSpoeqrJ15aiDCAB6tEI2-yd7d6HWJyiyCuv2d1~~qNZ9lpxslvKMUQXVT3bgWzSG~M1dAyClUc961C7pTHTb1eZQTl~opowcPcsLMX6cisFGOkpGFEv4d5lzsteYr~QhEcugOFbfsRRrLkBx1YovMogAGAQkHDkJFbYFWobBZwo0Tg1m9ffcCgmQ5zl2o9Snx1fxgOjO2asQ8Jqmqm0gBELdr5Ud6GCMKmPy1jhK~sfrl~pASelZXAqb4y1t3dZMxVbNlvRx8FG4YS0g-M5LqQcceKLY3jgGBqG4Wxv2x2HICYrZ~mnY0W0R4h4RuWj4pjJAcgRHOMNgMlxNw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Venn diagram showing the number of unique and shared OTUs for the in-solution capture method (SHS), PCR-based strategy (Amplicons) and sequences isolated at 90 m depth from a previous PCR-based study of Lake Pavin (Pavin90m).27 The Venn diagram was generated with Venny (http://bioinfogp.cnb.csic.es/tools/venny/index.html)." path-from-xml="dst00103.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4627825/dst00103.jpeg" path-from-xml="dst00103.jpeg" target="_blank">View large</a><a section="4627825" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst001/2/dst00103.jpeg?Expires=2147483647&Signature=uQ6WN3lBP1qi7GfQVUG~InqNmh8UbC0rF44M4CQRr2koYW7sWyh0qfnZrzOuWgNn5Hd78q2sDwGbfzO-uJq4-PkYArfSj1d9PNVTKh68gecy4HANjsgZh9FkeBFv3jDV~TEOL1h3cUpMVKq4ooYQZEKzUHvUaXCBxlVa8OQmMwandBL8ua2refCKokJki9Db1NJRh4cTe5yIG5b30mwSssHp1FmkZKsCMpmrCVJs1TaxDPaprk9omHfgLkhXDaCBF1ZbcqK2YXKTCCNr5aXA8Z1IIHr7P~huZ7JE0nJl1gzXXi~aV5pO6BmggrmqObIpVgeK2uRXwxstPWvQ38F00Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4627825&ar=333880&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00103.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Venn diagram showing the number of unique and shared OTUs for the in-solution capture method (SHS), PCR-based strategy (Amplicons) and sequences isolated at 90 m depth from a previous PCR-based study of Lake Pavin (Pavin90m).<sup><a reveal-id="DST001C27" data-open="DST001C27" class="link link-ref link-reveal xref-bibr">27</a></sup> The Venn diagram was generated with Venny (<a class="link link-uri" href="http://bioinfogp.cnb.csic.es/tools/venny/index.html">http://bioinfogp.cnb.csic.es/tools/venny/index.html</a>).</p></div></div><p>The 58 OTUs covered four lineages including <em>Methanobacteriales</em>, <em>Methanomicrobiales</em>, <em>Methanosarcinales</em> and a putative fourth lineage called ‘Novel Order’. Most OTUs were closely related to the <em>Methanomicrobiales</em> order (48 OTUs, 98.6% of the total input sequences). OTU3, OTU10 and OTU17 formed a distinct branch within this cluster (Fig. <a reveal-id="DST001F4" data-open="DST001F4" class="link link-reveal link-table xref-fig">4</a>A), and they were closely related to cultured methanogenic species that also have an insertion in their McrA protein sequence (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). Both the SHS and amplicon strategies clustered sequences in the most abundant OTUs (Fig. <a reveal-id="DST001F5" data-open="DST001F5" class="link link-reveal link-table xref-fig">5</a>). These abundant OTUs represented 94 and 98%, respectively, of the total sequences for each approach. The <em>Methanosarcinales</em> (two OTUs; Fig. <a reveal-id="DST001F4" data-open="DST001F4" class="link link-reveal link-table xref-fig">4</a>B) grouped into two distinct branches were related to the reference acetoclastic species <em>M. concilii</em> GP6 (85 and 87% similarity with OTU9 and OTU18, respectively). The most abundant cluster was OTU9 that represented 0.83% of the total SHS reads and 0.005% for the total amplicon reads (Fig. <a reveal-id="DST001F5" data-open="DST001F5" class="link link-reveal link-table xref-fig">5</a>). In contrast, the putative Novel Order (five OTUs; Fig. <a reveal-id="DST001F4" data-open="DST001F4" class="link link-reveal link-table xref-fig">4</a>C) was dominated by OTU13 clustering with 1.39% of the total amplicons sequences, but only 0.13% of the total SHS reads (Fig. <a reveal-id="DST001F5" data-open="DST001F5" class="link link-reveal link-table xref-fig">5</a>). Even if we did not include the more recently described sequences of <em>Methanomassiliicoccus luminyensis</em><sup><a reveal-id="DST001C46" data-open="DST001C46" class="link link-ref link-reveal xref-bibr">46</a></sup> and <em>Candidatus Methanomethylophilus alvus</em><sup><a reveal-id="DST001C47" data-open="DST001C47" class="link link-ref link-reveal xref-bibr">47</a></sup> belonging to the novel order for the probe design, distant sequences could be captured with probes by a mismatched nucleotide pairing. We cannot exclude that the sequences captured by specific probes allow indirect hybridization of other <em>mcr</em>A sequences as described for DNA microarrays experiments and referred to as ‘hitchhiking’.<sup><a reveal-id="DST001C48" data-open="DST001C48" class="link link-ref link-reveal xref-bibr">48</a></sup> Despite the substantial sequencing effort for amplicons, no sequences belonging to the <em>Methanobacteriales</em> order were recovered from this approach. These sequences were obtained only from the SHS sample (Fig. <a reveal-id="DST001F4" data-open="DST001F4" class="link link-reveal link-table xref-fig">4</a>D), and they were clustered in three OTUs such that one was 90% similar to MrtA (MCR isoenzyme encoded by the <em>mrt</em> operon) from <em>M. stadtmanae</em> DSM 3091<sup><a reveal-id="DST001C49" data-open="DST001C49" class="link link-ref link-reveal xref-bibr">49</a></sup> and the remaining two were 77 and 79% identical to MrtA sequences from <em>Methanobacterium lacus</em> that is in the <em>Methanobacteriales</em> order and has been isolated from Lake Pavin sediments.<sup><a reveal-id="DST001C50" data-open="DST001C50" class="link link-ref link-reveal xref-bibr">50</a></sup> These sequences represented 0.19% of total SHS <em>mcrA</em>-related sequences, with the most abundant OTU78 clustering 0.11% of the total SHS reads (Fig. <a reveal-id="DST001F5" data-open="DST001F5" class="link link-reveal link-table xref-fig">5</a>). </p>                        <a id="4627827" scrollto-destination="4627827"></a>
<div id="DST001F4" data-id="DST001F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst001/2/m_dst00104.jpeg?Expires=2147483647&Signature=a904wiD~Fy9lPWYna4xcybnVKuJhsK~cLri0dVCYdm0K5ozbkAjwHB~GAcBs3EvK0D8PTHObjsUuXUM~SWqELkUsqxapgrLefNqeK~3g2Vr~8BpsPsn3asgYERMCEhgINY2hlGQeT3-n4JEc5B1~uH~z2KIU0e5zX6UpoD4dRM1hOYzW~0pRg53nMxwzfqssxQI23hLE834Xu1qIh3kAjBJMs~9GEgzs9sCRLTl8Npj~MRrFWVsz5rXjGTAWNjABQ7wIpcSvA3eLYtpmHMB3myZR2AoLXn4rHDk0cDavXMKNPPZiIyttIgCKDDmfR2QBRiZ6bO8AoRTVTs9Wr8tf2Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic analysis of deduced McrA amino acid sequences obtained from the PCR, SHS and Pavin90m datasets showing evolutionary distances within the orders Methanomicrobiales (A), Methanosarcinales (B), Novel Order (C) and Methanobacteriales (D). Evolutionary history was inferred using the neighbour-joining method40,41 (Poisson distance model) using Seaview software.37 The final tree was drawn in MEGA 5.42 The bars represent a 5% sequence divergence. Numbers at the nodes represent bootstrap values &gt;60% (1000 resamplings). The number of amino acid sequences assigned to each OTU is given in brackets, together with the name of the strategies for obtaining them. McrA amino acid sequence from Methanosarcina barkeri (AAZ69867), uncultured methanogenic archaeon clone Lak19-ML (CAH68744) and Methanobrevibacter smithii (ABQ87220) were used as outgroups, and Methanopyrus kandleri (AAM01870) was an outgroup for rooting the tree. Bold arrows indicate dominant OTUs." path-from-xml="dst00104.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4627827/dst00104.jpeg" path-from-xml="dst00104.jpeg" target="_blank">View large</a><a section="4627827" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst001/2/dst00104.jpeg?Expires=2147483647&Signature=Fi7uXi4Hz0o43dszxZUPPDoxWObzeps9msLtvVNKFSZ63j9DRfSzrsHumMCIeOzjD2-gSk91rhXxBJSNRz9M9MHmcSqmqdaeF4d3~CZl4SlEk8-6nBVpbvi~T9Qf440bA747NiP6eEWpdkyHCANA-9V1lDwy~7~gBbhyEdZJFetqulJ5UAZ5unZIgrrVdWqWrIb3xvaE8mcZMCnzpNBz~93cjWCNj8CKaOtwwpqJsvGuhkW9l71PPr-SGp4ajhtGhEp8fem5aCM4dBmlMGez9-j~IjByc8uAi2E8onrshW~wSam5PnCvc2yS6-A8NAOgnLQrrhlr1PB5iwQt4fBoTg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4627827&ar=333880&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00104.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic analysis of deduced McrA amino acid sequences obtained from the PCR, SHS and Pavin90m datasets showing evolutionary distances within the orders <em>Methanomicrobiales</em> (A), <em>Methanosarcinales</em> (B), Novel Order (C) and <em>Methanobacteriales</em> (D). Evolutionary history was inferred using the neighbour-joining method<sup><a reveal-id="DST001C40" data-open="DST001C40" class="link link-ref link-reveal xref-bibr">40</a>,<a reveal-id="DST001C41" data-open="DST001C41" class="link link-ref link-reveal xref-bibr">41</a></sup> (Poisson distance model) using Seaview software.<sup><a reveal-id="DST001C37" data-open="DST001C37" class="link link-ref link-reveal xref-bibr">37</a></sup> The final tree was drawn in MEGA 5.<sup><a reveal-id="DST001C42" data-open="DST001C42" class="link link-ref link-reveal xref-bibr">42</a></sup> The bars represent a 5% sequence divergence. Numbers at the nodes represent bootstrap values &gt;60% (1000 resamplings). The number of amino acid sequences assigned to each OTU is given in brackets, together with the name of the strategies for obtaining them. McrA amino acid sequence from <em>Methanosarcina barkeri</em> (AAZ69867), uncultured methanogenic archaeon clone Lak19-ML (CAH68744) and <em>Methanobrevibacter smithii</em> (ABQ87220) were used as outgroups, and <em>Methanopyrus kandleri</em> (AAM01870) was an outgroup for rooting the tree. Bold arrows indicate dominant OTUs.</p></div></div><div content-id="DST001F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst001/2/m_dst00104.jpeg?Expires=2147483647&Signature=a904wiD~Fy9lPWYna4xcybnVKuJhsK~cLri0dVCYdm0K5ozbkAjwHB~GAcBs3EvK0D8PTHObjsUuXUM~SWqELkUsqxapgrLefNqeK~3g2Vr~8BpsPsn3asgYERMCEhgINY2hlGQeT3-n4JEc5B1~uH~z2KIU0e5zX6UpoD4dRM1hOYzW~0pRg53nMxwzfqssxQI23hLE834Xu1qIh3kAjBJMs~9GEgzs9sCRLTl8Npj~MRrFWVsz5rXjGTAWNjABQ7wIpcSvA3eLYtpmHMB3myZR2AoLXn4rHDk0cDavXMKNPPZiIyttIgCKDDmfR2QBRiZ6bO8AoRTVTs9Wr8tf2Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic analysis of deduced McrA amino acid sequences obtained from the PCR, SHS and Pavin90m datasets showing evolutionary distances within the orders Methanomicrobiales (A), Methanosarcinales (B), Novel Order (C) and Methanobacteriales (D). Evolutionary history was inferred using the neighbour-joining method40,41 (Poisson distance model) using Seaview software.37 The final tree was drawn in MEGA 5.42 The bars represent a 5% sequence divergence. Numbers at the nodes represent bootstrap values &gt;60% (1000 resamplings). The number of amino acid sequences assigned to each OTU is given in brackets, together with the name of the strategies for obtaining them. McrA amino acid sequence from Methanosarcina barkeri (AAZ69867), uncultured methanogenic archaeon clone Lak19-ML (CAH68744) and Methanobrevibacter smithii (ABQ87220) were used as outgroups, and Methanopyrus kandleri (AAM01870) was an outgroup for rooting the tree. Bold arrows indicate dominant OTUs." path-from-xml="dst00104.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4627827/dst00104.jpeg" path-from-xml="dst00104.jpeg" target="_blank">View large</a><a section="4627827" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst001/2/dst00104.jpeg?Expires=2147483647&Signature=Fi7uXi4Hz0o43dszxZUPPDoxWObzeps9msLtvVNKFSZ63j9DRfSzrsHumMCIeOzjD2-gSk91rhXxBJSNRz9M9MHmcSqmqdaeF4d3~CZl4SlEk8-6nBVpbvi~T9Qf440bA747NiP6eEWpdkyHCANA-9V1lDwy~7~gBbhyEdZJFetqulJ5UAZ5unZIgrrVdWqWrIb3xvaE8mcZMCnzpNBz~93cjWCNj8CKaOtwwpqJsvGuhkW9l71PPr-SGp4ajhtGhEp8fem5aCM4dBmlMGez9-j~IjByc8uAi2E8onrshW~wSam5PnCvc2yS6-A8NAOgnLQrrhlr1PB5iwQt4fBoTg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4627827&ar=333880&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00104.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic analysis of deduced McrA amino acid sequences obtained from the PCR, SHS and Pavin90m datasets showing evolutionary distances within the orders <em>Methanomicrobiales</em> (A), <em>Methanosarcinales</em> (B), Novel Order (C) and <em>Methanobacteriales</em> (D). Evolutionary history was inferred using the neighbour-joining method<sup><a reveal-id="DST001C40" data-open="DST001C40" class="link link-ref link-reveal xref-bibr">40</a>,<a reveal-id="DST001C41" data-open="DST001C41" class="link link-ref link-reveal xref-bibr">41</a></sup> (Poisson distance model) using Seaview software.<sup><a reveal-id="DST001C37" data-open="DST001C37" class="link link-ref link-reveal xref-bibr">37</a></sup> The final tree was drawn in MEGA 5.<sup><a reveal-id="DST001C42" data-open="DST001C42" class="link link-ref link-reveal xref-bibr">42</a></sup> The bars represent a 5% sequence divergence. Numbers at the nodes represent bootstrap values &gt;60% (1000 resamplings). The number of amino acid sequences assigned to each OTU is given in brackets, together with the name of the strategies for obtaining them. McrA amino acid sequence from <em>Methanosarcina barkeri</em> (AAZ69867), uncultured methanogenic archaeon clone Lak19-ML (CAH68744) and <em>Methanobrevibacter smithii</em> (ABQ87220) were used as outgroups, and <em>Methanopyrus kandleri</em> (AAM01870) was an outgroup for rooting the tree. Bold arrows indicate dominant OTUs.</p></div></div>                        <a id="4627828" scrollto-destination="4627828"></a>
<div id="DST001F5" data-id="DST001F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst001/2/m_dst00105.jpeg?Expires=2147483647&Signature=mBE4Dlcxq2GBJ4txLZPN2ZeZkzn2fOuXJCURJ9aEyrEBK9ZltvoVv61OYpG7wA4yazsxbtrLb1A~vcOpchh-6woUVoCOKdgrLfXgEnhFjnYrHsKrqaoU2rVf6Ab2Ji0ZFEux4UC5FuyzJGOzPGdjzeLXCuHY~p~ynA~DxdyKiFaXgY8hUfxMNGNqzrW6aBSNya8zMJ8dLRj4fLpzf8BBonJB~bnmzvejx8M-Idz4ZYH~kLPfKIIfFti2df4cvm7hEXxMiQboNHZuo1yzU1qTdhdSdConGDJ62JkVu5xWFTriFbidh1BYLZmzF~U71XC8Eog6XK1qM6bW2MYFOx5w8g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The relative abundances of dominant OTUs from four methanogenic bacterial orders identified by the targeted capture method (SHS), PCR-based strategy (amplicons) and qPCR experiments (qPCR). The relative abundances calculated by qPCR were computed using mcrA copy number as reference obtained using a primers pair targeting all OTUs (Supplementary Table S3)." path-from-xml="dst00105.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4627828/dst00105.jpeg" path-from-xml="dst00105.jpeg" target="_blank">View large</a><a section="4627828" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst001/2/dst00105.jpeg?Expires=2147483647&Signature=eLTnesEa9RWkRXl-4zB1e7ueAvVioNlP2OWnWtfntv2~SNoedggvHJR8gezzXcshr4T9VY5Ztf~AvyQ7hmZ0qTUk0vbYq4QYe7xT0EMQuOM5PHmP4xDt8YQRrJyqF~8AxwFBonvPLhrrUA1thhxCdIENyJdvVL1bdrukJJb8iRIVto~TIUSAg6W4Cl9iCQ61mMYUZP82zGMNLkbquOi6E-o1ElJaGjAQMbC2Cb654k2UcImnC4AT7MdXW~6vx4kcTnn2qO9RS-zq0Zmb8v5nUKKAdm0mrws0~3ivi5CEsf3BHApGcthDHp7WAOrQZxlDUKll1WlqQBI86K7he7GtRw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4627828&ar=333880&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00105.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The relative abundances of dominant OTUs from four methanogenic bacterial orders identified by the targeted capture method (SHS), PCR-based strategy (amplicons) and qPCR experiments (qPCR). The relative abundances calculated by qPCR were computed using <em>mcrA</em> copy number as reference obtained using a primers pair targeting all OTUs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>).</p></div></div><div content-id="DST001F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst001/2/m_dst00105.jpeg?Expires=2147483647&Signature=mBE4Dlcxq2GBJ4txLZPN2ZeZkzn2fOuXJCURJ9aEyrEBK9ZltvoVv61OYpG7wA4yazsxbtrLb1A~vcOpchh-6woUVoCOKdgrLfXgEnhFjnYrHsKrqaoU2rVf6Ab2Ji0ZFEux4UC5FuyzJGOzPGdjzeLXCuHY~p~ynA~DxdyKiFaXgY8hUfxMNGNqzrW6aBSNya8zMJ8dLRj4fLpzf8BBonJB~bnmzvejx8M-Idz4ZYH~kLPfKIIfFti2df4cvm7hEXxMiQboNHZuo1yzU1qTdhdSdConGDJ62JkVu5xWFTriFbidh1BYLZmzF~U71XC8Eog6XK1qM6bW2MYFOx5w8g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The relative abundances of dominant OTUs from four methanogenic bacterial orders identified by the targeted capture method (SHS), PCR-based strategy (amplicons) and qPCR experiments (qPCR). The relative abundances calculated by qPCR were computed using mcrA copy number as reference obtained using a primers pair targeting all OTUs (Supplementary Table S3)." path-from-xml="dst00105.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4627828/dst00105.jpeg" path-from-xml="dst00105.jpeg" target="_blank">View large</a><a section="4627828" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst001/2/dst00105.jpeg?Expires=2147483647&Signature=eLTnesEa9RWkRXl-4zB1e7ueAvVioNlP2OWnWtfntv2~SNoedggvHJR8gezzXcshr4T9VY5Ztf~AvyQ7hmZ0qTUk0vbYq4QYe7xT0EMQuOM5PHmP4xDt8YQRrJyqF~8AxwFBonvPLhrrUA1thhxCdIENyJdvVL1bdrukJJb8iRIVto~TIUSAg6W4Cl9iCQ61mMYUZP82zGMNLkbquOi6E-o1ElJaGjAQMbC2Cb654k2UcImnC4AT7MdXW~6vx4kcTnn2qO9RS-zq0Zmb8v5nUKKAdm0mrws0~3ivi5CEsf3BHApGcthDHp7WAOrQZxlDUKll1WlqQBI86K7he7GtRw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4627828&ar=333880&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00105.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The relative abundances of dominant OTUs from four methanogenic bacterial orders identified by the targeted capture method (SHS), PCR-based strategy (amplicons) and qPCR experiments (qPCR). The relative abundances calculated by qPCR were computed using <em>mcrA</em> copy number as reference obtained using a primers pair targeting all OTUs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>).</p></div></div><p>The GC content of the <em>mcrA</em> genes ranged from 50.4 to 61.1% for amplicons and from 37 to 63.2% for SHS. In the <em>mcrA</em> database, the GC content ranges from 36.2 to 67.2%, indicating that the SHS method is most likely less affected by GC composition than PCR approaches. Furthermore, we evaluated the presence of mismatch residues between PCR primers and probes on <em>mcrA</em> genes in both SHS and amplicon approaches. We identified 99.10, 0.77 and 0.13% of <em>mcrA</em> sequences for amplicons versus 37.68, 50.22 and 12.10% for SHS with 0, 1 and 2 mismatch residues, respectively, between probes (or primers) and sequences. This trend highlights the potential advantage of the SHS approach with long capture probes that tolerate more mismatches, allowing access to new <em>mcrA</em> gene variants.</p><p>In parallel, qPCR was used to precisely describe the methanogen abundance in Lake Pavin with regard to the most abundant OTUs and bacterial orders (primers are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). The results were compared with the relative sequence abundance calculated previously for the selected OTUs with amplicons and SHS (Fig. <a reveal-id="DST001F5" data-open="DST001F5" class="link link-reveal link-table xref-fig">5</a>). The abundance of OTU2, which included the <em>Methanomicrobiales</em> order, was similar in qPCR and amplicons (33.5 and 28.62%), but not SHS (11.87%). In contrast, the second <em>Methanomicrobiales</em> OTU (OTU7) was more abundant in SHS (8.02%) and qPCR (3.6%), but not amplicons (0.05%). The same trend was observed for OTU9 (<em>Methanosarcinales</em>). No significant difference was observed for OTU13 (Novel Order). Finally, no qPCR amplification of OTU78 (<em>Methanobacteriales</em>) occurred<em>.</em> However, we validated the presence of this OTU in Lake Pavin by successive PCR cycles, cloning and sequencing (100% identity). This result indicates that <em>Methanobacteriales</em> are rare in this ecosystem.</p>                            <h4 scrollto-destination=4627831 id="4627831" class="section-title" data-legacy-id=s3b3>3.2.3. <em>De novo</em> assembly of SHS reads</h4>
<p>To reconstruct contigs with sequences flanking the targeted <em>mcrA</em> gene, <em>de novo</em> assembly was performed using the pyrosequencing reads obtained by the SHS method (Table <a reveal-id="DST001TB2" data-open="DST001TB2" class="link link-reveal link-table xref-fig">2</a>). We identified 691 contigs (301–1639 bases) with <em>mcrA</em> sequences. By mapping these sequences to complete reference genomes for the <em>Methanomicrobiales</em>, <em>Methanosarcinales</em> and <em>Methanobacteriales</em> orders (no genome was available for the Novel Order), we identified contigs extending into the <em>mcrA</em> flanking regions. The upstream sequences were all part of the <em>mcrG</em> gene. We also characterized two adjacent ORFs located at 200 bases downstream from the <em>mcrA</em> gene and in the same orientation; these ORFs encoded a DtxR family iron (metal)-dependent repressor and a DOMON domain-containing protein<em>.</em> The DtxR sequences were closely related (76–83% identity) to <em>Methanosphaerula palustris</em> E1-9C (accession no. ACL16981) of the <em>Methanomicrobiales</em> order. In the reference genome of this species, the gene is located ∼700 kb downstream of the <em>mcr</em> operon. The sequences of DOMON domain-containing protein are closely related (74–80% identity) to <em>M. concilii</em> GP-6 (accession no. AEB67518) that belongs to the <em>Methanosarcinales</em> order. In the reference genome of this species, the gene is located ∼50 kb downstream of the <em>mcr</em> operon. </p>                        <a id="4627833" scrollto-destination="4627833"></a>
<div content-id="DST001TB2" class="table-modal"><div class="table-wrap"><div id="DST001TB2" class="table-wrap-title" data-id="DST001TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Summary statistics from <em>de novo</em> assembly</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Newbler version 2.6 </th><th>SHS </th></tr></thead><tbody><tr><td>No. of reads used for assembly </td><td>122 772 </td></tr><tr><td>No. of reads assembled into contigs </td><td>53 307 </td></tr><tr><td>No. of singletons </td><td>56 834 </td></tr><tr><td>Outliers<sup>a</sup> </td><td>12 631 </td></tr><tr><td>No. of contigs assembled </td><td>1916 </td></tr><tr><td>N<sub>50</sub> contig size (bases) </td><td>820 </td></tr><tr><td>No. of <em>mcrA</em> homologous contigs </td><td>693 </td></tr><tr><td>No. of <em>mcrA</em> homologous singletons </td><td>1142 </td></tr><tr><td>Number of chimaeras </td><td>5 </td></tr><tr><td>Number of high-quality <em>mcrA</em> homologous contigs (without chimaeras) </td><td>691 </td></tr><tr><td>Number of high-quality <em>mcrA</em> homologous singletons (without chimaeras) </td><td>1139 </td></tr><tr><td>Average <em>mcrA</em> homologous contig length (bases) </td><td>589 </td></tr><tr><td>Largest <em>mcrA</em> homologous contig length (bases) </td><td>1639 </td></tr></tbody></table></div><div content-id="DST001TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Newbler version 2.6 </th><th>SHS </th></tr></thead><tbody><tr><td>No. of reads used for assembly </td><td>122 772 </td></tr><tr><td>No. of reads assembled into contigs </td><td>53 307 </td></tr><tr><td>No. of singletons </td><td>56 834 </td></tr><tr><td>Outliers<sup>a</sup> </td><td>12 631 </td></tr><tr><td>No. of contigs assembled </td><td>1916 </td></tr><tr><td>N<sub>50</sub> contig size (bases) </td><td>820 </td></tr><tr><td>No. of <em>mcrA</em> homologous contigs </td><td>693 </td></tr><tr><td>No. of <em>mcrA</em> homologous singletons </td><td>1142 </td></tr><tr><td>Number of chimaeras </td><td>5 </td></tr><tr><td>Number of high-quality <em>mcrA</em> homologous contigs (without chimaeras) </td><td>691 </td></tr><tr><td>Number of high-quality <em>mcrA</em> homologous singletons (without chimaeras) </td><td>1139 </td></tr><tr><td>Average <em>mcrA</em> homologous contig length (bases) </td><td>589 </td></tr><tr><td>Largest <em>mcrA</em> homologous contig length (bases) </td><td>1639 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Reads were discarded due to quality control by Newbler.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4627833" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST001TB2" class="table-wrap-title" data-id="DST001TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Summary statistics from <em>de novo</em> assembly</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Newbler version 2.6 </th><th>SHS </th></tr></thead><tbody><tr><td>No. of reads used for assembly </td><td>122 772 </td></tr><tr><td>No. of reads assembled into contigs </td><td>53 307 </td></tr><tr><td>No. of singletons </td><td>56 834 </td></tr><tr><td>Outliers<sup>a</sup> </td><td>12 631 </td></tr><tr><td>No. of contigs assembled </td><td>1916 </td></tr><tr><td>N<sub>50</sub> contig size (bases) </td><td>820 </td></tr><tr><td>No. of <em>mcrA</em> homologous contigs </td><td>693 </td></tr><tr><td>No. of <em>mcrA</em> homologous singletons </td><td>1142 </td></tr><tr><td>Number of chimaeras </td><td>5 </td></tr><tr><td>Number of high-quality <em>mcrA</em> homologous contigs (without chimaeras) </td><td>691 </td></tr><tr><td>Number of high-quality <em>mcrA</em> homologous singletons (without chimaeras) </td><td>1139 </td></tr><tr><td>Average <em>mcrA</em> homologous contig length (bases) </td><td>589 </td></tr><tr><td>Largest <em>mcrA</em> homologous contig length (bases) </td><td>1639 </td></tr></tbody></table></div><div content-id="DST001TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Newbler version 2.6 </th><th>SHS </th></tr></thead><tbody><tr><td>No. of reads used for assembly </td><td>122 772 </td></tr><tr><td>No. of reads assembled into contigs </td><td>53 307 </td></tr><tr><td>No. of singletons </td><td>56 834 </td></tr><tr><td>Outliers<sup>a</sup> </td><td>12 631 </td></tr><tr><td>No. of contigs assembled </td><td>1916 </td></tr><tr><td>N<sub>50</sub> contig size (bases) </td><td>820 </td></tr><tr><td>No. of <em>mcrA</em> homologous contigs </td><td>693 </td></tr><tr><td>No. of <em>mcrA</em> homologous singletons </td><td>1142 </td></tr><tr><td>Number of chimaeras </td><td>5 </td></tr><tr><td>Number of high-quality <em>mcrA</em> homologous contigs (without chimaeras) </td><td>691 </td></tr><tr><td>Number of high-quality <em>mcrA</em> homologous singletons (without chimaeras) </td><td>1139 </td></tr><tr><td>Average <em>mcrA</em> homologous contig length (bases) </td><td>589 </td></tr><tr><td>Largest <em>mcrA</em> homologous contig length (bases) </td><td>1639 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Reads were discarded due to quality control by Newbler.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4627833" target="_blank">View Large</a></div></div>                            <h2 scrollto-destination=4627834 id="4627834" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>We captured specific target DNA from a complex environmental metagenome using a novel SHS capture method and NGS. We showed that the relative enrichment of the target sequence was increased to 175 365-fold with 2 cycles of capture, and this result was superior to previous studies using a single cycle<sup><a reveal-id="DST001C18" data-open="DST001C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DST001C19" data-open="DST001C19" class="link link-ref link-reveal xref-bibr">19</a></sup> and microarray-based capture.<sup><a reveal-id="DST001C51" data-open="DST001C51" class="link link-ref link-reveal xref-bibr">51</a></sup> We applied this strategy to the anoxic layer of Lake Pavin, where <em>Archaea</em> account for 17% of 4,6-diamidino-2-phenylindole-stained cells<sup><a reveal-id="DST001C52" data-open="DST001C52" class="link link-ref link-reveal xref-bibr">52</a></sup> and only a fraction of these microbes are methanogens. Our SHS strategy specifically enriched <em>mcrA</em> sequences from the environmental sample. In comparison with the random-shotgun metagenomic approach (0.003% recovery of <em>mcrA</em> sequences), the SHS method was superior (41.32% <em>mcrA</em> sequence enrichment). However, the capture efficiency is also likely influenced by the number of probes used per region and the mismatched residues between the probes and their targets. Consequently, two rounds of capture and multiple long RNA probes are advantageous for efficient enrichment.</p><p>With a random-shotgun metagenomics approach, many hundreds of thousands of additional single reads would have been necessary to estimate the biodiversity of the methanogen community in this environment. The SHS experiment contained much more <em>mcrA</em> data and provided a solid taxonomic basis for studying methanogens diversity. Finally, PCR was the most effective enrichment approach; with ∼100% of the amplicons corresponding to the biomarker, the primers used were very specific and efficient.<sup><a reveal-id="DST001C31" data-open="DST001C31" class="link link-ref link-reveal xref-bibr">31</a></sup></p><p>The SHS and amplicon strategies both revealed similar patterns in methanogen communities such as the high abundance and diversity of <em>Methanomicrobiales</em> sequences (more than 98% of the total sequences representing 48 OTUs). These data confirm a previous study by Biderre-Petit <em>et al.</em><sup><a reveal-id="DST001C27" data-open="DST001C27" class="link link-ref link-reveal xref-bibr">27</a></sup> High-throughput sequencing, however, reveals that methanogen diversity is much higher than previously estimated by amplicon libraries and Sanger sequencing.<sup><a reveal-id="DST001C27" data-open="DST001C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Importantly, the amplicon sequencing approach missed all the <em>Methanobacteriales</em> taxonomic groups and some <em>Methanosarcinales</em>, possibly due to <em>mcrA</em> primer bias. PCR undersampling often leads to significant underestimation of true community diversity.<sup><a reveal-id="DST001C24" data-open="DST001C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DST001C53" data-open="DST001C53" class="link link-ref link-reveal xref-bibr">53</a></sup> SHS efficiently targets rare sequences, as demonstrated for <em>Methanobacteriales</em>, and does not appear to be influenced by GC content. As previously demonstrated for microarray approaches,<sup><a reveal-id="DST001C21" data-open="DST001C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DST001C22" data-open="DST001C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DST001C54" data-open="DST001C54" class="link link-ref link-reveal xref-bibr">54</a></sup> more extensive explorative capture probe sets could recover rare sequences, leading to the detection of many uncharacterized microbial populations. Moreover, the SHS and amplicon library results were correlated by qPCR.</p><p>We also used <em>de novo</em> assembly of SHS sequence reads to explore the regions flanking the target gene, and we identified two ORFs (<em>dtxR</em> and DOMON domain) at previously unknown positions downstream of <em>mcrA</em>. Because this genomic organization may link methanogenesis to electron transfer and Fe homeostasis in organisms living in the anoxic layer of the Lake Pavin, it could reflect adaptation to this particular environment. More experiments are needed, however, to validate this hypothesis.</p><p>In this study, we present a novel enrichment method that, when coupled to NGS, expands our knowledge of the diversity of a target gene within a complex microbial community. The method was successfully applied to a lacustrine environment using the <em>mcrA</em> gene, and it revealed higher methanogen community diversity than observed with other methods. To some extent, this method could be applied to phylogenetic studies to explore the diversity of commonly conserved genes such as the 16S rRNA biomarker. The main limitation is the design of high quality probes sets to expect a full coverage of 16S rDNA sequences as complete as possible. New algorithms, such as KASpOD,<sup><a reveal-id="DST001C55" data-open="DST001C55" class="link link-ref link-reveal xref-bibr">55</a></sup> can be used to design highly specific and explorative probes (i.e. targeting sequences not already included in databases) based on oligonucleotide <em>k-mer</em> signatures. These probe designs would be extremely suitable and beneficial to the SHS approach.</p><p>With the emergence of third generation sequencing platforms and the capability to sequence longer DNA sequences without library construction,<sup><a reveal-id="DST001C56" data-open="DST001C56" class="link link-ref link-reveal xref-bibr">56</a>,<a reveal-id="DST001C57" data-open="DST001C57" class="link link-ref link-reveal xref-bibr">57</a></sup> the SHS strategy could link genomic structure and function in microbial communities.</p>                            <h2 scrollto-destination=4627841 id="4627841" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at</a><a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=4627843 id="4627843" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by the ANR-09-EBIO-009 project (Agence Nationale de la Recherche). J.D. was supported by a studentship from the Centre National de la Recherche Scientifique (CNRS, grant number 163588) and the Région Auvergne. N.P. was funded by Direction Générale de l'Armement (DGA).</p>                            <h2 scrollto-destination=4627845 id="4627845" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We would like to thank Yannick Bidet and Maud Privat from the Centre Jean Perrin for their help with sample processing on the 454 GS FLX pyrosequencing platform. We also thank Sarah Orlhac and Nicolas Gallois for their efficient technical assistance and David Tottey for reviewing the English version of the manuscript.</p>                            <h2 scrollto-destination=4627847 id="4627847" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DST001C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Whitman</div> <div class="given-names">W.B.</div></div>,  <div class="name"><div class="surname">Coleman</div> <div class="given-names">D.C.</div></div>,  <div class="name"><div class="surname">Wiebe</div> <div class="given-names">W.J.</div></div>. </div><div class="article-title">Prokaryotes: the unseen majority</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1998</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">6578</div>-<div class="lpage">83</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prokaryotes%3A%20the%20unseen%20majority&amp;author=W.B.%20Whitman&amp;author=D.C.%20Coleman&amp;author=W.J.%20Wiebe&amp;publication_year=1998&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=95&amp;pages=6578-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.12.6578" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.12.6578" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.12.6578"> </span></div> </div></div></div></div></div><div content-id="DST001C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Curtis</div> <div class="given-names">T.P.</div></div>,  <div class="name"><div class="surname">Head</div> <div class="given-names">I.M.</div></div>,  <div class="name"><div class="surname">Lunn</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Woodcock</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Schloss</div> <div class="given-names">P.D.</div></div>,  <div class="name"><div class="surname">Sloan</div> <div class="given-names">W.T.</div></div>. </div><div class="article-title">What is the extent of prokaryotic diversity?</div>, <div class="source">Philos. Trans. R. Soc. Lond. B. Biol. Sci.</div> , <div class="year">2006</div>, vol. <div class="volume">361</div> (pg. <div class="fpage">2023</div>-<div class="lpage">37</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=What%20is%20the%20extent%20of%20prokaryotic%20diversity%3F&amp;author=T.P.%20Curtis&amp;author=I.M.%20Head&amp;author=M.%20Lunn&amp;author=S.%20Woodcock&amp;author=P.D.%20Schloss&amp;author=W.T.%20Sloan&amp;publication_year=2006&amp;journal=Philos.%20Trans.%20R.%20Soc.%20Lond.%20B.%20Biol.%20Sci.&amp;volume=361&amp;pages=2023-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1098/rstb.2006.1921" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frstb.2006.1921" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1098%2Frstb.2006.1921"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17028084" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Amann</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Ludwig</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Schleifer</div> <div class="given-names">K.-H.</div></div>. </div><div class="article-title">Phylogenetic identification and in situ detection of individual microbial cells without cultivation</div>, <div class="source">Microbiol. Rev.</div> , <div class="year">1995</div>, vol. <div class="volume">59</div> (pg. <div class="fpage">143</div>-<div class="lpage">69</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20identification%20and%20in%20situ%20detection%20of%20individual%20microbial%20cells%20without%20cultivation&amp;author=R.%20Amann&amp;author=W.%20Ludwig&amp;author=K.-H.%20Schleifer&amp;publication_year=1995&amp;journal=Microbiol.%20Rev.&amp;volume=59&amp;pages=143-69" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7535888" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eisen</div> <div class="given-names">J.A.</div></div>. </div><div class="article-title">Environmental shotgun sequencing: its potential and challenges for studying the hidden world of microbes</div>, <div class="source">PLoS Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e82</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Environmental%20shotgun%20sequencing%3A%20its%20potential%20and%20challenges%20for%20studying%20the%20hidden%20world%20of%20microbes&amp;author=J.A.%20Eisen&amp;publication_year=2007&amp;journal=PLoS%20Biol.&amp;volume=5&amp;pages=e82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0050082" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0050082" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0050082"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17355177" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Handelsman</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Rondon</div> <div class="given-names">M.R.</div></div>,  <div class="name"><div class="surname">Brady</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Clardy</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Goodman</div> <div class="given-names">R.M.</div></div>. </div><div class="article-title">Molecular biological access to the chemistry of unknown soil microbes: a new frontier for natural products</div>, <div class="source">Chem. Biol.</div> , <div class="year">1998</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">R245</div>-<div class="lpage">249</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20biological%20access%20to%20the%20chemistry%20of%20unknown%20soil%20microbes%3A%20a%20new%20frontier%20for%20natural%20products&amp;author=J.%20Handelsman&amp;author=M.R.%20Rondon&amp;author=S.F.%20Brady&amp;author=J.%20Clardy&amp;author=R.M.%20Goodman&amp;publication_year=1998&amp;journal=Chem.%20Biol.&amp;volume=5&amp;pages=R245-249" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1074-5521(98)90108-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1074-5521(98)90108-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1074-5521(98)90108-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9818143" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Biddle</div> <div class="given-names">J.F.</div></div>,  <div class="name"><div class="surname">Fitz-Gibbon</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Schuster</div> <div class="given-names">S.C.</div></div>,  <div class="name"><div class="surname">Brenchley</div> <div class="given-names">J.E.</div></div>,  <div class="name"><div class="surname">House</div> <div class="given-names">C.H.</div></div>. </div><div class="article-title">Metagenomic signatures of the Peru Margin subseafloor biosphere show a genetically distinct environment</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2008</div>, vol. <div class="volume">105</div> (pg. <div class="fpage">10583</div>-<div class="lpage">88</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metagenomic%20signatures%20of%20the%20Peru%20Margin%20subseafloor%20biosphere%20show%20a%20genetically%20distinct%20environment&amp;author=J.F.%20Biddle&amp;author=S.%20Fitz-Gibbon&amp;author=S.C.%20Schuster&amp;author=J.E.%20Brenchley&amp;author=C.H.%20House&amp;publication_year=2008&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=105&amp;pages=10583-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0709942105" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0709942105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0709942105"> </span></div> </div></div></div></div></div><div content-id="DST001C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tringe</div> <div class="given-names">S.G.</div></div>,  <div class="name"><div class="surname">von Mering</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Kobayashi</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Comparative metagenomics of microbial communities</div>, <div class="source">Science</div> , <div class="year">2005</div>, vol. <div class="volume">308</div> (pg. <div class="fpage">554</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20metagenomics%20of%20microbial%20communities&amp;author=S.G.%20Tringe&amp;author=C.%20von%20Mering&amp;author=A.%20Kobayashi&amp;publication_year=2005&amp;journal=Science&amp;volume=308&amp;pages=554-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1107851" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1107851" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1107851"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15845853" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Riesenfeld</div> <div class="given-names">C.S.</div></div>,  <div class="name"><div class="surname">Schloss</div> <div class="given-names">P.D.</div></div>,  <div class="name"><div class="surname">Handelsman</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Metagenomics: genomic analysis of microbial communities</div>, <div class="source">Annu. Rev. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">525</div>-<div class="lpage">52</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metagenomics%3A%20genomic%20analysis%20of%20microbial%20communities&amp;author=C.S.%20Riesenfeld&amp;author=P.D.%20Schloss&amp;author=J.%20Handelsman&amp;publication_year=2004&amp;journal=Annu.%20Rev.%20Genet.&amp;volume=38&amp;pages=525-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.genet.38.072902.091216" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genet.38.072902.091216" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.genet.38.072902.091216"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15568985" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suenaga</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Targeted metagenomics: a high-resolution metagenomics approach for specific gene clusters in complex microbial communities</div>, <div class="source">Environ. Microbiol.</div> , <div class="year">2011</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">13</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeted%20metagenomics%3A%20a%20high-resolution%20metagenomics%20approach%20for%20specific%20gene%20clusters%20in%20complex%20microbial%20communities&amp;author=H.%20Suenaga&amp;publication_year=2011&amp;journal=Environ.%20Microbiol.&amp;volume=14&amp;pages=13-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1462-2920.2011.02438.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1462-2920.2011.02438.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1462-2920.2011.02438.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21366818" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Edwards</div> <div class="given-names">R.A.</div></div>,  <div class="name"><div class="surname">Rodriguez-Brito</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Wegley</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">Using pyrosequencing to shed light on deep mine microbial ecology</div>, <div class="source">BMC Genomics</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">57</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20pyrosequencing%20to%20shed%20light%20on%20deep%20mine%20microbial%20ecology&amp;author=R.A.%20Edwards&amp;author=B.%20Rodriguez-Brito&amp;author=L.%20Wegley&amp;publication_year=2006&amp;journal=BMC%20Genomics&amp;volume=7&amp;pages=57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-7-57" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-7-57" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-7-57"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16549033" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mardis</div> <div class="given-names">E.R.</div></div>. </div><div class="article-title">The impact of next-generation sequencing technology on genetics</div>, <div class="source">Trends Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">133</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20impact%20of%20next-generation%20sequencing%20technology%20on%20genetics&amp;author=E.R.%20Mardis&amp;publication_year=2008&amp;journal=Trends%20Genet.&amp;volume=24&amp;pages=133-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2007.12.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2007.12.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2007.12.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18262675" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Quince</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Curtis</div> <div class="given-names">T.P.</div></div>,  <div class="name"><div class="surname">Sloan</div> <div class="given-names">W.T.</div></div>. </div><div class="article-title">The rational exploration of microbial diversity</div>, <div class="source">ISME J.</div> , <div class="year">2008</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">997</div>-<div class="lpage">1006</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20rational%20exploration%20of%20microbial%20diversity&amp;author=C.%20Quince&amp;author=T.P.%20Curtis&amp;author=W.T.%20Sloan&amp;publication_year=2008&amp;journal=ISME%20J.&amp;volume=2&amp;pages=997-1006" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ismej.2008.69" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2008.69" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fismej.2008.69"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18650928" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hoff</div> <div class="given-names">K.J.</div></div>. </div><div class="article-title">The effect of sequencing errors on metagenomic gene prediction</div>, <div class="source">BMC Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">520</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20effect%20of%20sequencing%20errors%20on%20metagenomic%20gene%20prediction&amp;author=K.J.%20Hoff&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=520" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-520" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-520" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-520"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19909532" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Summerer</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Enabling technologies of genomic-scale sequence enrichment for targeted high-throughput sequencing</div>, <div class="source">Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">94</div> (pg. <div class="fpage">363</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Enabling%20technologies%20of%20genomic-scale%20sequence%20enrichment%20for%20targeted%20high-throughput%20sequencing&amp;author=D.%20Summerer&amp;publication_year=2009&amp;journal=Genomics&amp;volume=94&amp;pages=363-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ygeno.2009.08.012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ygeno.2009.08.012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ygeno.2009.08.012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19720138" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Albert</div> <div class="given-names">T.J.</div></div>,  <div class="name"><div class="surname">Molla</div> <div class="given-names">M.N.</div></div>,  <div class="name"><div class="surname">Muzny</div> <div class="given-names">D.M.</div></div>, et al. </div><div class="article-title">Direct selection of human genomic loci by microarray hybridization</div>, <div class="source">Nat. Methods</div> , <div class="year">2007</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">903</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Direct%20selection%20of%20human%20genomic%20loci%20by%20microarray%20hybridization&amp;author=T.J.%20Albert&amp;author=M.N.%20Molla&amp;author=D.M.%20Muzny&amp;publication_year=2007&amp;journal=Nat.%20Methods&amp;volume=4&amp;pages=903-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth1111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth1111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth1111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17934467" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Okou</div> <div class="given-names">D.T.</div></div>,  <div class="name"><div class="surname">Steinberg</div> <div class="given-names">K.M.</div></div>,  <div class="name"><div class="surname">Middle</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Cutler</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Albert</div> <div class="given-names">T.J.</div></div>,  <div class="name"><div class="surname">Zwick</div> <div class="given-names">M.E.</div></div>. </div><div class="article-title">Microarray-based genomic selection for high-throughput resequencing</div>, <div class="source">Nat. Methods</div> , <div class="year">2007</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">907</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microarray-based%20genomic%20selection%20for%20high-throughput%20resequencing&amp;author=D.T.%20Okou&amp;author=K.M.%20Steinberg&amp;author=C.%20Middle&amp;author=D.J.%20Cutler&amp;author=T.J.%20Albert&amp;author=M.E.%20Zwick&amp;publication_year=2007&amp;journal=Nat.%20Methods&amp;volume=4&amp;pages=907-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth1109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth1109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth1109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17934469" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mokry</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Feitsma</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Nijman</div> <div class="given-names">I.J.</div></div>, et al. </div><div class="article-title">Accurate SNP and mutation detection by targeted custom microarray-based genomic enrichment of short-fragment sequencing libraries</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2010</div>, vol. <div class="volume">38</div> pg. <div class="fpage">e116</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accurate%20SNP%20and%20mutation%20detection%20by%20targeted%20custom%20microarray-based%20genomic%20enrichment%20of%20short-fragment%20sequencing%20libraries&amp;author=M.%20Mokry&amp;author=H.%20Feitsma&amp;author=I.J.%20Nijman&amp;publication_year=2010&amp;journal=Nucleic%20Acids%20Res.&amp;volume=38&amp;pages=e116" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq072" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq072" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq072"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20164091" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tewhey</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Nakano</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">Enrichment of sequencing targets from the human genome by solution hybridization</div>, <div class="source">Genome Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">R116</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Enrichment%20of%20sequencing%20targets%20from%20the%20human%20genome%20by%20solution%20hybridization&amp;author=R.%20Tewhey&amp;author=M.%20Nakano&amp;author=X.%20Wang&amp;publication_year=2009&amp;journal=Genome%20Biol.&amp;volume=10&amp;pages=R116" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2009-10-10-r116" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2009-10-10-r116" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2009-10-10-r116"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19835619" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gnirke</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Melnikov</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Maguire</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Solution hybrid selection with ultra-long oligonucleotides for massively parallel targeted sequencing</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2009</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">182</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Solution%20hybrid%20selection%20with%20ultra-long%20oligonucleotides%20for%20massively%20parallel%20targeted%20sequencing&amp;author=A.%20Gnirke&amp;author=A.%20Melnikov&amp;author=J.%20Maguire&amp;publication_year=2009&amp;journal=Nat.%20Biotechnol.&amp;volume=27&amp;pages=182-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1523" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1523" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1523"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19182786" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Iwai</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Chai</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Sul</div> <div class="given-names">W.J.</div></div>,  <div class="name"><div class="surname">Cole</div> <div class="given-names">J.R.</div></div>,  <div class="name"><div class="surname">Hashsham</div> <div class="given-names">S.A.</div></div>,  <div class="name"><div class="surname">Tiedje</div> <div class="given-names">J.M.</div></div>. </div><div class="article-title">Gene-targeted-metagenomics reveals extensive diversity of aromatic dioxygenase genes in the environment</div>, <div class="source">ISME J.</div> , <div class="year">2010</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">279</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene-targeted-metagenomics%20reveals%20extensive%20diversity%20of%20aromatic%20dioxygenase%20genes%20in%20the%20environment&amp;author=S.%20Iwai&amp;author=B.%20Chai&amp;author=W.J.%20Sul&amp;author=J.R.%20Cole&amp;author=S.A.%20Hashsham&amp;author=J.M.%20Tiedje&amp;publication_year=2010&amp;journal=ISME%20J.&amp;volume=4&amp;pages=279-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ismej.2009.104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2009.104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fismej.2009.104"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19776767" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Terrat</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Peyretaillade</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Goncalves</div> <div class="given-names">O.</div></div>, et al. </div><div class="article-title">Detecting variants with Metabolic Design, a new software tool to design probes for explorative functional DNA microarray development</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">478</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detecting%20variants%20with%20Metabolic%20Design%2C%20a%20new%20software%20tool%20to%20design%20probes%20for%20explorative%20functional%20DNA%20microarray%20development&amp;author=S.%20Terrat&amp;author=E.%20Peyretaillade&amp;author=O.%20Goncalves&amp;publication_year=2010&amp;journal=BMC%20Bioinformatics&amp;volume=11&amp;pages=478" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-11-478" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-11-478" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-11-478"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20860850" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dugat-Bony</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Peyretaillade</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Parisot</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Detecting unknown sequences with DNA microarrays: explorative probe design strategies</div>, <div class="source">Environ. Microbiol.</div> , <div class="year">2011</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">356</div>-<div class="lpage">371</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detecting%20unknown%20sequences%20with%20DNA%20microarrays%3A%20explorative%20probe%20design%20strategies&amp;author=E.%20Dugat-Bony&amp;author=E.%20Peyretaillade&amp;author=N.%20Parisot&amp;publication_year=2011&amp;journal=Environ.%20Microbiol.&amp;volume=14&amp;pages=356-371" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1462-2920.2011.02559.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1462-2920.2011.02559.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1462-2920.2011.02559.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21895914" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suzuki</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Giovannoni</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Bias caused by template annealing in the amplification of mixtures of 16S rRNA genes by PCR</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">1996</div>, vol. <div class="volume">62</div> (pg. <div class="fpage">625</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bias%20caused%20by%20template%20annealing%20in%20the%20amplification%20of%20mixtures%20of%2016S%20rRNA%20genes%20by%20PCR&amp;author=M.%20Suzuki&amp;author=S.%20Giovannoni&amp;publication_year=1996&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=62&amp;pages=625-30" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8593063" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hong</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Bunge</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Leslin</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Jeon</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Epstein</div> <div class="given-names">S.S.</div></div>. </div><div class="article-title">Polymerase chain reaction primers miss half of rRNA microbial diversity</div>, <div class="source">ISME J.</div> , <div class="year">2009</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">1365</div>-<div class="lpage">73</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Polymerase%20chain%20reaction%20primers%20miss%20half%20of%20rRNA%20microbial%20diversity&amp;author=S.%20Hong&amp;author=J.%20Bunge&amp;author=C.%20Leslin&amp;author=S.%20Jeon&amp;author=S.S.%20Epstein&amp;publication_year=2009&amp;journal=ISME%20J.&amp;volume=3&amp;pages=1365-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ismej.2009.89" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2009.89" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fismej.2009.89"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19693101" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bastviken</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Cole</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Pace</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tranvik</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Methane emissions from lakes: dependence of lake characteristics, two regional assessments, and a global estimate</div>, <div class="source">Global Biogeochem. Cycles</div> , <div class="year">2004</div>, vol. <div class="volume">18</div>  <div class="comment">GB4009, doi:10.1029/2004GB002238</div></div></div></div></div><div content-id="DST001C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Aeschbach-Hertig</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Hofer</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kipfer</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Imboden</div> <div class="given-names">D.M.</div></div>,  <div class="name"><div class="surname">Wieler</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Accumulation of mantle gases in a permanently stratified volcanic lake (Lake Pavin, France)</div>, <div class="source">Geochim. Cosmochim. Acta</div> , <div class="year">1999</div>, vol. <div class="volume">63</div> (pg. <div class="fpage">3357</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accumulation%20of%20mantle%20gases%20in%20a%20permanently%20stratified%20volcanic%20lake%20(Lake%20Pavin%2C%20France)&amp;author=W.%20Aeschbach-Hertig&amp;author=M.%20Hofer&amp;author=R.%20Kipfer&amp;author=D.M.%20Imboden&amp;author=R.%20Wieler&amp;publication_year=1999&amp;journal=Geochim.%20Cosmochim.%20Acta&amp;volume=63&amp;pages=3357-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0016-7037(99)00257-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0016-7037(99)00257-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0016-7037(99)00257-4"> </span></div> </div></div></div></div></div><div content-id="DST001C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Biderre-Petit</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Jezequel</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Dugat-Bony</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Identification of microbial communities involved in the methane cycle of a freshwater meromictic lake</div>, <div class="source">FEMS Microbiol. Ecol.</div> , <div class="year">2011</div>, vol. <div class="volume">77</div> (pg. <div class="fpage">533</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20microbial%20communities%20involved%20in%20the%20methane%20cycle%20of%20a%20freshwater%20meromictic%20lake&amp;author=C.%20Biderre-Petit&amp;author=D.%20Jezequel&amp;author=E.%20Dugat-Bony&amp;publication_year=2011&amp;journal=FEMS%20Microbiol.%20Ecol.&amp;volume=77&amp;pages=533-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1574-6941.2011.01134.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1574-6941.2011.01134.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1574-6941.2011.01134.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21595728" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Reeve</div> <div class="given-names">J.N.</div></div>. </div><div class="article-title">Molecular biology of methanogens</div>, <div class="source">Annu. Rev. Microbiol.</div> , <div class="year">1992</div>, vol. <div class="volume">46</div> (pg. <div class="fpage">165</div>-<div class="lpage">91</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20biology%20of%20methanogens&amp;author=J.N.%20Reeve&amp;publication_year=1992&amp;journal=Annu.%20Rev.%20Microbiol.&amp;volume=46&amp;pages=165-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.mi.46.100192.001121" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.mi.46.100192.001121" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.mi.46.100192.001121"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1444253" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Klein</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Allmansberger</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Bokranz</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Knaub</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Müller</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Muth</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Comparative analysis of genes encoding methyl coenzyme M reductase in methanogenic bacteria</div>, <div class="source">Mol. Gen. Genet.</div> , <div class="year">1988</div>, vol. <div class="volume">213</div> (pg. <div class="fpage">409</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20genes%20encoding%20methyl%20coenzyme%20M%20reductase%20in%20methanogenic%20bacteria&amp;author=A.%20Klein&amp;author=R.%20Allmansberger&amp;author=M.%20Bokranz&amp;author=S.%20Knaub&amp;author=B.%20M%C3%BCller&amp;author=E.%20Muth&amp;publication_year=1988&amp;journal=Mol.%20Gen.%20Genet.&amp;volume=213&amp;pages=409-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00339610" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00339610" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00339610"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3185509" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dugat-Bony</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Missaoui</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Peyretaillade</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">HiSpOD: probe design for functional DNA microarrays</div>, <div class="source">Bioinformatics</div> , <div class="year">2011</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">641</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=HiSpOD%3A%20probe%20design%20for%20functional%20DNA%20microarrays&amp;author=E.%20Dugat-Bony&amp;author=M.%20Missaoui&amp;author=E.%20Peyretaillade&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=641-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq712" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq712" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtq712"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21216777" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mihajlovski</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Alric</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Brugere</div> <div class="given-names">J.F.</div></div>. </div><div class="article-title">A putative new order of methanogenic Archaea inhabiting the human gut, as revealed by molecular analyses of the mcrA gene</div>, <div class="source">Res. Microbiol.</div> , <div class="year">2008</div>, vol. <div class="volume">159</div> (pg. <div class="fpage">516</div>-<div class="lpage">21</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20putative%20new%20order%20of%20methanogenic%20Archaea%20inhabiting%20the%20human%20gut%2C%20as%20revealed%20by%20molecular%20analyses%20of%20the%20mcrA%20gene&amp;author=A.%20Mihajlovski&amp;author=M.%20Alric&amp;author=J.F.%20Brugere&amp;publication_year=2008&amp;journal=Res.%20Microbiol.&amp;volume=159&amp;pages=516-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.resmic.2008.06.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.resmic.2008.06.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.resmic.2008.06.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18644435" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Staden</div> <div class="given-names">R.</div></div>. </div><div class="article-title">The Staden sequence analysis package</div>, <div class="source">Mol. Biotechnol.</div> , <div class="year">1996</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">233</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Staden%20sequence%20analysis%20package&amp;author=R.%20Staden&amp;publication_year=1996&amp;journal=Mol.%20Biotechnol.&amp;volume=5&amp;pages=233-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02900361" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02900361" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02900361"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8837029" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schmieder</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Edwards</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Quality control and preprocessing of metagenomic datasets</div>, <div class="source">Bioinformatics</div> , <div class="year">2011</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">863</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quality%20control%20and%20preprocessing%20of%20metagenomic%20datasets&amp;author=R.%20Schmieder&amp;author=R.%20Edwards&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=863-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr026" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr026" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr026"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21278185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Madden</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Schäffer</div> <div class="given-names">A.A.</div></div>, et al. </div><div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3389</div>-<div class="lpage">402</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.F.%20Altschul&amp;author=T.L.%20Madden&amp;author=A.A.%20Sch%C3%A4ffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Edgar</div> <div class="given-names">R.C.</div></div>,  <div class="name"><div class="surname">Haas</div> <div class="given-names">B.J.</div></div>,  <div class="name"><div class="surname">Clemente</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Quince</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Knight</div> <div class="given-names">R.</div></div>. </div><div class="article-title">UCHIME improves sensitivity and speed of chimera detection</div>, <div class="source">Bioinformatics</div> , <div class="year">2011</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">2194</div>-<div class="lpage">200</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=UCHIME%20improves%20sensitivity%20and%20speed%20of%20chimera%20detection&amp;author=R.C.%20Edgar&amp;author=B.J.%20Haas&amp;author=J.C.%20Clemente&amp;author=C.%20Quince&amp;author=R.%20Knight&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=2194-200" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr381" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr381" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr381"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21700674" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Larkin</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Blackshields</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Brown</div> <div class="given-names">N.P.</div></div>, et al. </div><div class="article-title">Clustal W and Clustal X version 2.0</div>, <div class="source">Bioinformatics</div> , <div class="year">2007</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">2947</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Clustal%20W%20and%20Clustal%20X%20version%202.0&amp;author=M.A.%20Larkin&amp;author=G.%20Blackshields&amp;author=N.P.%20Brown&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=2947-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm404" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm404" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm404"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17846036" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gouy</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Guindon</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Gascuel</div> <div class="given-names">O.</div></div>. </div><div class="article-title">SeaView version 4: a multiplatform graphical user interface for sequence alignment and phylogenetic tree building</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2009</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">221</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SeaView%20version%204%3A%20a%20multiplatform%20graphical%20user%20interface%20for%20sequence%20alignment%20and%20phylogenetic%20tree%20building&amp;author=M.%20Gouy&amp;author=S.%20Guindon&amp;author=O.%20Gascuel&amp;publication_year=2009&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=27&amp;pages=221-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msp259" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsp259" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsp259"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19854763" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Godzik</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences</div>, <div class="source">Bioinformatics</div> , <div class="year">2006</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">1658</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cd-hit%3A%20a%20fast%20program%20for%20clustering%20and%20comparing%20large%20sets%20of%20protein%20or%20nucleotide%20sequences&amp;author=W.%20Li&amp;author=A.%20Godzik&amp;publication_year=2006&amp;journal=Bioinformatics&amp;volume=22&amp;pages=1658-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btl158" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtl158" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtl158"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16731699" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Luton</div> <div class="given-names">P.E.</div></div>,  <div class="name"><div class="surname">Wayne</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Sharp</div> <div class="given-names">R.J.</div></div>,  <div class="name"><div class="surname">Riley</div> <div class="given-names">P.W.</div></div>. </div><div class="article-title">The mcrA gene as an alternative to 16S rRNA in the phylogenetic analysis of methanogen populations in landfill</div>, <div class="source">Microbiology</div> , <div class="year">2002</div>, vol. <div class="volume">148</div> (pg. <div class="fpage">3521</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20mcrA%20gene%20as%20an%20alternative%20to%2016S%20rRNA%20in%20the%20phylogenetic%20analysis%20of%20methanogen%20populations%20in%20landfill&amp;author=P.E.%20Luton&amp;author=J.M.%20Wayne&amp;author=R.J.%20Sharp&amp;author=P.W.%20Riley&amp;publication_year=2002&amp;journal=Microbiology&amp;volume=148&amp;pages=3521-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/00221287-148-11-3521" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2F00221287-148-11-3521" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2F00221287-148-11-3521"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12427943" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Studier</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Keppler</div> <div class="given-names">K.</div></div>. </div><div class="article-title">A note on the neighbor-joining algorithm of Saitou and Nei</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1988</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">729</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20note%20on%20the%20neighbor-joining%20algorithm%20of%20Saitou%20and%20Nei&amp;author=J.%20Studier&amp;author=K.%20Keppler&amp;publication_year=1988&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=5&amp;pages=729-31" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3221794" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Saitou</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>. </div><div class="article-title">The neighbor-joining method: a new method for reconstructing phylogenetic trees</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1987</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">406</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20neighbor-joining%20method%3A%20a%20new%20method%20for%20reconstructing%20phylogenetic%20trees&amp;author=N.%20Saitou&amp;author=M.%20Nei&amp;publication_year=1987&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=4&amp;pages=406-25" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3447015" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Peterson</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Peterson</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Stecher</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div>. </div><div class="article-title">MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2011</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">2731</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA5%3A%20molecular%20evolutionary%20genetics%20analysis%20using%20maximum%20likelihood%2C%20evolutionary%20distance%2C%20and%20maximum%20parsimony%20methods&amp;author=K.%20Tamura&amp;author=D.%20Peterson&amp;author=N.%20Peterson&amp;author=G.%20Stecher&amp;author=M.%20Nei&amp;author=S.%20Kumar&amp;publication_year=2011&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=28&amp;pages=2731-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msr121" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsr121" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsr121"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21546353" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Livak</div> <div class="given-names">K.J.</div></div>,  <div class="name"><div class="surname">Schmittgen</div> <div class="given-names">T.D.</div></div>. </div><div class="article-title">Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) method</div>, <div class="source">Methods</div> , <div class="year">2001</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">402</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20relative%20gene%20expression%20data%20using%20real-time%20quantitative%20PCR%20and%20the%202(-Delta%20Delta%20C(T))%20method&amp;author=K.J.%20Livak&amp;author=T.D.%20Schmittgen&amp;publication_year=2001&amp;journal=Methods&amp;volume=25&amp;pages=402-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/meth.2001.1262" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fmeth.2001.1262" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fmeth.2001.1262"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11846609" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Gish</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Miller</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Myers</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Lipman</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Basic local alignment search tool</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1990</div>, vol. <div class="volume">215</div> (pg. <div class="fpage">403</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=S.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.%20Myers&amp;author=D.%20Lipman&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol.&amp;volume=215&amp;pages=403-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brauer</div> <div class="given-names">S.L.</div></div>,  <div class="name"><div class="surname">Cadillo-Quiroz</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Yashiro</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Yavitt</div> <div class="given-names">J.B.</div></div>,  <div class="name"><div class="surname">Zinder</div> <div class="given-names">S.H.</div></div>. </div><div class="article-title">Isolation of a novel acidiphilic methanogen from an acidic peat bog</div>, <div class="source">Nature</div> , <div class="year">2006</div>, vol. <div class="volume">442</div> (pg. <div class="fpage">192</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20of%20a%20novel%20acidiphilic%20methanogen%20from%20an%20acidic%20peat%20bog&amp;author=S.L.%20Brauer&amp;author=H.%20Cadillo-Quiroz&amp;author=E.%20Yashiro&amp;author=J.B.%20Yavitt&amp;author=S.H.%20Zinder&amp;publication_year=2006&amp;journal=Nature&amp;volume=442&amp;pages=192-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature04810" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature04810" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature04810"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16699521" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dridi</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Fardeau</div> <div class="given-names">M.L.</div></div>,  <div class="name"><div class="surname">Ollivier</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Raoult</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Drancourt</div> <div class="given-names">M.</div></div>. </div><div class="article-title"><em>Methanomassiliicoccus luminyensis</em> gen. nov. sp. nov. a methanogenic archaeon isolated from human faeces</div>, <div class="source">Int. J. Syst. Evol. Microbiol.</div> , <div class="year">2012</div>, vol. <div class="volume">62</div> (pg. <div class="fpage">1902</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/1910.1099/ijs.1900.033712-033710" class="link link-pub-id-doi" target="_blank">https://doi.org/1910.1099/ijs.1900.033712-033710</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Methanomassiliicoccus%20luminyensis%20gen.%20nov.%20sp.%20nov.%20a%20methanogenic%20archaeon%20isolated%20from%20human%20faeces&amp;author=B.%20Dridi&amp;author=M.L.%20Fardeau&amp;author=B.%20Ollivier&amp;author=D.%20Raoult&amp;author=M.%20Drancourt&amp;publication_year=2012&amp;journal=Int.%20J.%20Syst.%20Evol.%20Microbiol.&amp;volume=62&amp;pages=1902-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/ijs.0.033712-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fijs.0.033712-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fijs.0.033712-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22859731" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Borrel</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Harris</div> <div class="given-names">H.M.B.</div></div>,  <div class="name"><div class="surname">Tottey</div> <div class="given-names">W.</div></div>, et al. </div><div class="article-title">Genome sequence of ‘<em>Candidatus Methanomethylophilus alvus</em>’ Mx1201, a methanogenic archaeon from the human gut belonging to a seventh order of methanogens</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2012</div>, vol. <div class="volume">194</div> (pg. <div class="fpage">6944</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20%E2%80%98Candidatus%20Methanomethylophilus%20alvus%E2%80%99%20Mx1201%2C%20a%20methanogenic%20archaeon%20from%20the%20human%20gut%20belonging%20to%20a%20seventh%20order%20of%20methanogens&amp;author=G.%20Borrel&amp;author=H.M.B.%20Harris&amp;author=W.%20Tottey&amp;publication_year=2012&amp;journal=J.%20Bacteriol.&amp;volume=194&amp;pages=6944-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.01867-12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.01867-12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.01867-12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23209209" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Palmer</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Rapid quantitative profiling of complex microbial populations</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> pg. <div class="fpage">e5</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20quantitative%20profiling%20of%20complex%20microbial%20populations&amp;author=C.%20Palmer&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=e5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gnj007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgnj007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgnj007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16407321" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fricke</div> <div class="given-names">W.F.</div></div>,  <div class="name"><div class="surname">Seedorf</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Henne</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">The genome sequence of <em>Methanosphaera stadtmanae</em> reveals why this human intestinal archaeon is restricted to methanol and H2 for methane formation and ATP synthesis</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2005</div>, vol. <div class="volume">188</div> (pg. <div class="fpage">642</div>-<div class="lpage">58</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20sequence%20of%20Methanosphaera%20stadtmanae%20reveals%20why%20this%20human%20intestinal%20archaeon%20is%20restricted%20to%20methanol%20and%20H2%20for%20methane%20formation%20and%20ATP%20synthesis&amp;author=W.F.%20Fricke&amp;author=H.%20Seedorf&amp;author=A.%20Henne&amp;publication_year=2005&amp;journal=J.%20Bacteriol.&amp;volume=188&amp;pages=642-58" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.188.2.642-658.2006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.188.2.642-658.2006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.188.2.642-658.2006"> </span></div> </div></div></div></div></div><div content-id="DST001C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Borrel</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Joblin</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Guedon</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title"><em>Methanobacterium lacus</em> sp. nov., a novel hydrogenotrophic methanogen from the deep cold sediment of a meromictic lake</div>, <div class="source">Int. J. Syst. Evol. Microbiol.</div> , <div class="year">2011</div>, vol. <div class="volume">62</div> (pg. <div class="fpage">1625</div>-<div class="lpage">1629</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Methanobacterium%20lacus%20sp.%20nov.%2C%20a%20novel%20hydrogenotrophic%20methanogen%20from%20the%20deep%20cold%20sediment%20of%20a%20meromictic%20lake&amp;author=G.%20Borrel&amp;author=K.%20Joblin&amp;author=A.%20Guedon&amp;publication_year=2011&amp;journal=Int.%20J.%20Syst.%20Evol.%20Microbiol.&amp;volume=62&amp;pages=1625-1629" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/ijs.0.034538-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fijs.0.034538-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fijs.0.034538-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21890730" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Summerer</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Haase</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">Microarray-based multicycle-enrichment of genomic subsets for targeted next-generation sequencing</div>, <div class="source">Genome Res.</div> , <div class="year">2009</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">1616</div>-<div class="lpage">21</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microarray-based%20multicycle-enrichment%20of%20genomic%20subsets%20for%20targeted%20next-generation%20sequencing&amp;author=D.%20Summerer&amp;author=H.%20Wu&amp;author=B.%20Haase&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=19&amp;pages=1616-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.091942.109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.091942.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.091942.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19638418" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lehours</div> <div class="given-names">A.C.</div></div>,  <div class="name"><div class="surname">Bardot</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Thenot</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Debroas</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Fonty</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Anaerobic microbial communities in Lake Pavin, a unique meromictic lake in France</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">2005</div>, vol. <div class="volume">71</div> (pg. <div class="fpage">7389</div>-<div class="lpage">400</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Anaerobic%20microbial%20communities%20in%20Lake%20Pavin%2C%20a%20unique%20meromictic%20lake%20in%20France&amp;author=A.C.%20Lehours&amp;author=C.%20Bardot&amp;author=A.%20Thenot&amp;author=D.%20Debroas&amp;author=G.%20Fonty&amp;publication_year=2005&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=71&amp;pages=7389-400" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.71.11.7389-7400.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.71.11.7389-7400.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.71.11.7389-7400.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16269781" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jeon</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Bunge</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Leslin</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Stoeck</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Hong</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Epstein</div> <div class="given-names">S.S.</div></div>. </div><div class="article-title">Environmental rRNA inventories miss over half of protistan diversity</div>, <div class="source">BMC Microbiol.</div> , <div class="year">2008</div>, vol. <div class="volume">8</div> pg. <div class="fpage">222</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Environmental%20rRNA%20inventories%20miss%20over%20half%20of%20protistan%20diversity&amp;author=S.%20Jeon&amp;author=J.%20Bunge&amp;author=C.%20Leslin&amp;author=T.%20Stoeck&amp;author=S.%20Hong&amp;author=S.S.%20Epstein&amp;publication_year=2008&amp;journal=BMC%20Microbiol.&amp;volume=8&amp;pages=222" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2180-8-222" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2180-8-222" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2180-8-222"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19087295" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Militon</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Rimour</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Missaoui</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">PhylArray: phylogenetic probe design algorithm for microarray</div>, <div class="source">Bioinformatics</div> , <div class="year">2007</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">2550</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PhylArray%3A%20phylogenetic%20probe%20design%20algorithm%20for%20microarray&amp;author=C.%20Militon&amp;author=S.%20Rimour&amp;author=M.%20Missaoui&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=2550-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm392" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm392" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm392"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17698494" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parisot</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Denonfoux</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Dugat-Bony</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Peyret</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Peyretaillade</div> <div class="given-names">E.</div></div>. </div><div class="article-title">KASpOD – a web service for highly specific and explorative oligonucleotide design</div>, <div class="source">Bioinformatics</div> , <div class="year">2012</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">3161</div>-<div class="lpage">3162</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=KASpOD%20%E2%80%93%20a%20web%20service%20for%20highly%20specific%20and%20explorative%20oligonucleotide%20design&amp;author=N.%20Parisot&amp;author=J.%20Denonfoux&amp;author=E.%20Dugat-Bony&amp;author=P.%20Peyret&amp;author=E.%20Peyretaillade&amp;publication_year=2012&amp;journal=Bioinformatics&amp;volume=28&amp;pages=3161-3162" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bts597" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbts597" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbts597"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23047560" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McCarthy</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Third generation DNA sequencing: pacific biosciences’ single molecule real time technology</div>, <div class="source">Chem. Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">675</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Third%20generation%20DNA%20sequencing%3A%20pacific%20biosciences%E2%80%99%20single%20molecule%20real%20time%20technology&amp;author=A.%20McCarthy&amp;publication_year=2010&amp;journal=Chem.%20Biol.&amp;volume=17&amp;pages=675-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.chembiol.2010.07.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.chembiol.2010.07.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.chembiol.2010.07.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20659677" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST001C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST001C57" class="ref-content "><div class="label -label">57</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schadt</div> <div class="given-names">E.E.</div></div>,  <div class="name"><div class="surname">Turner</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kasarskis</div> <div class="given-names">A.</div></div>. </div><div class="article-title">A window into third-generation sequencing</div>, <div class="source">Hum. Mol. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">R227</div>-<div class="lpage">240</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20window%20into%20third-generation%20sequencing&amp;author=E.E.%20Schadt&amp;author=S.%20Turner&amp;author=A.%20Kasarskis&amp;publication_year=2010&amp;journal=Hum.%20Mol.%20Genet.&amp;volume=19&amp;pages=R227-240" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/hmg/ddq416" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fhmg%2Fddq416" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fhmg%2Fddq416"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20858600" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Prof. Masahira Hattori</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by-nc/3.0/), which permits non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=333880&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst001/2/dst001-dst001supp_tables.ppt?Expires=2147483647&amp;Signature=4IcLxMQW3n-NV5-7CL36fUI4tvFrJishNpVqgNWdC32nOu5qHxEHnANcMAezCVInvcFiBNVv9i1YHQzcF5VFEkeCPcC04hoKDLHV1KM~x10cDQ-bbDBupQCl6Y4yhHlcbJCWg52NvxlWyV81XEa3KtouBIc3QYf6mpSc9Q0g~G1-aMlozuthYAkW9PPpxyUjEQa3kp05ZK6h~HPbV0RwftG6DO93mk319irWAzcc1LHQLOp3oY1sYbwnFMFq0YYmwP6mSGpq~ttdqDdvInWaDglbVy7MGV-zEDLMwvmzxL3xGGmwFiO6TG~oPiCdQbofiSAc-azliQEb-8ccgtDu3A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst001/2/dst001-dst001supp_fig2.ppt?Expires=2147483647&amp;Signature=VNb6VqBup7m8FGNoRkcObBfNC9bVZc5S7e9S1oFrpRH~8ru5m6iHub7QUqEe34hp7LPvGlM5UP6zzCP1HFZbp2UYVfLxKjpl3LXs8CYSUmzNc1kBeeobgrXjhf-hSNEspj5bCdhtYwr~oZwjx8Pym8WIphn-JI6Jv5kyUStnXT-RsEmaz3GHz9YOFe7VtDIZLci8lWWs75mnM8Adr-Evdl4dWI~oKQI3AnERv3PEGL9IQ~tz5HDmD6tmK-vQK9XHxoU8bicaaKNrIfhoHE7PvFQHP55rvzEhhCpykcLcCa4af7jxSeDmKRMcFlh4dVWklOgPl9-488MIq41a6nsJqQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dst001/2/dst001-dst001supp_fig1.ppt?Expires=2147483647&amp;Signature=OnQyIPqS4E2JhaT4MnwLVqz2BNRSevvVPwcUBcruE3wK28dcw3hl9F64mHUQbGuJT6WM8eoNBKzEs5v4vVlBy6VEZ1iTdu7tLtFjD9OBvYIS4InpmGTzZEYeZpPTWcHofYFct6V~hy5qWT-8QpJCKNgJZwd0mYBKqkTWOy7xVjEhpw6QbnlOUgHZI~VgOX3XAisDhmvhtGrJZC4NrqXdVq6qjowj-IpWlgV3pkaMmGCe7o9aFAOGaA5xFAVB6zg9nUAHO8RdI0EXBm5fQo8Jo3l7g6~2oYbvo5nqd~o3pmDgPSpw5IzWEwZU0uCO0~GO4Z6q8avup73twwzZ79ONrA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - ppt file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">420</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000318584600007&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">30</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dst001"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="333880"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000318584600007&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f20%2f2%2f185%2f333880">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000318584600007&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (30)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F20%2F2%2F185%2F333880" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/333880'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Gene Capture Coupled to High-Throughput Sequencing as a Strategy for Targeted Metagenome Exploration" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dst001");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f20%2f2%2f185%2f333880">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

