

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/23/6/517/2647442","@type":"ScholarlyArticle","name":"Correlation between genome reduction and bacterial growth","datePublished":"2016-07-03","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/23/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2016-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw035","keywords":["growth rate","dispensable genomic sequence","genome reduction","accessory gene"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/23/6/517/2647442","author":[{"name":"Kurokawa, Masaomi","affiliation":"1 Graduate School of Life and Environmental Sciences, University of Tsukuba, Ibaraki 305-8572, Japan","@type":"Person"},{"name":"Seno, Shigeto","affiliation":"2 Graduate School of Information Science and Technology, Osaka University, Osaka 565-0871, Japan","@type":"Person"},{"name":"Matsuda, Hideo","affiliation":"2 Graduate School of Information Science and Technology, Osaka University, Osaka 565-0871, Japan","@type":"Person"},{"name":"Ying, Bei-Wen","affiliation":"1 Graduate School of Life and Environmental Sciences, University of Tsukuba, Ibaraki 305-8572, Japan","@type":"Person"}],"description":"Genome reduction by removing dispensable genomic sequences in bacteria is commonly used in both fundamental and applied studies to determine the minimal genetic","pageStart":"517","pageEnd":"525","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/6/0/m_dnares_23_6cover.png?Expires=2147483647&Signature=Max3WHFhEi-UCkzx6P3aYqgGeApUTTF7rFn5PtyKoLTwqYHjx5CL~q9P-cecyPnMaWMfAiuwjwIq4FX0zeusJJeQovtOXJf6HCEN1nyt6gGT-V0qXZbGd9kAS2xoR8~AhSm3sDF8oaeSvJqsJf7X0O3CSXDRO0TzM8dBxL8950DJw1dPLvl7nHrUNq2p9yI4l0hMXt0UcvxAxqNgAki~K8SZjGpu3wP2yBtmQAG5iZnvNTuV7drgxbF2S717hI9S~I3x6JRIOVuEraI2uq8N7rEjIvQ14-5VJbutut2EpRpGHZf7SQYkJkdSrHr028n8qO7shpq47NBQIX2Fd8XLBg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Correlation between genome reduction and bacterial growth","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/6/0/m_dnares_23_6cover.png?Expires=2147483647&Signature=Max3WHFhEi-UCkzx6P3aYqgGeApUTTF7rFn5PtyKoLTwqYHjx5CL~q9P-cecyPnMaWMfAiuwjwIq4FX0zeusJJeQovtOXJf6HCEN1nyt6gGT-V0qXZbGd9kAS2xoR8~AhSm3sDF8oaeSvJqsJf7X0O3CSXDRO0TzM8dBxL8950DJw1dPLvl7nHrUNq2p9yI4l0hMXt0UcvxAxqNgAki~K8SZjGpu3wP2yBtmQAG5iZnvNTuV7drgxbF2S717hI9S~I3x6JRIOVuEraI2uq8N7rEjIvQ14-5VJbutut2EpRpGHZf7SQYkJkdSrHr028n8qO7shpq47NBQIX2Fd8XLBg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Correlation between genome reduction and bacterial growth" />
    <meta property="og:description" content="Genome reduction by removing dispensable genomic sequences in bacteria is commonly used in both fundamental and applied studies to determine the minimal genetic" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/23/6/517/2647442" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-07-03" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Kurokawa, Masaomi" /><meta name="citation_author_institution" content="Graduate School of Life and Environmental Sciences, University of Tsukuba, Ibaraki 305-8572, Japan" /><meta name="citation_author" content="Seno, Shigeto" /><meta name="citation_author_institution" content="Graduate School of Information Science and Technology, Osaka University, Osaka 565-0871, Japan" /><meta name="citation_author" content="Matsuda, Hideo" /><meta name="citation_author_institution" content="Graduate School of Information Science and Technology, Osaka University, Osaka 565-0871, Japan" /><meta name="citation_author" content="Ying, Bei-Wen" /><meta name="citation_author_institution" content="Graduate School of Life and Environmental Sciences, University of Tsukuba, Ibaraki 305-8572, Japan" /><meta name="citation_title" content="Correlation between genome reduction and bacterial growth" /><meta name="citation_firstpage" content="517" /><meta name="citation_lastpage" content="525" /><meta name="citation_doi" content="10.1093/dnares/dsw035" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="23" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2016/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Minimizing the genome of Escherichia coli. Motivation and strategy,;  citation_journal_title=Ann. N. Y. Acad. Sci;  citation_year=1994;  citation_volume=745;  citation_pages=1-3; " /><meta name="citation_reference" content="citation_title=The complete genome sequence of Escherichia coli K-12,;  citation_journal_title=Science;  citation_year=1997;  citation_volume=277;  citation_pages=1453-62; " /><meta name="citation_reference" content="citation_title=Engineering a reduced Escherichia coli genome,;  citation_journal_title=Genome Res;  citation_year=2002;  citation_volume=12;  citation_pages=640-7; " /><meta name="citation_reference" content="citation_title=Minimization of the Escherichia coli genome using a Tn5-targeted Cre/loxP excision system,;  citation_journal_title=Nat. Biotechnol;  citation_year=2002;  citation_volume=20;  citation_pages=1018-23; " /><meta name="citation_reference" content="citation_title=Scarless engineering of the Escherichia coli genome,;  citation_journal_title=Methods Mol. Biol;  citation_year=2008;  citation_volume=416;  citation_pages=251-9; " /><meta name="citation_reference" content="citation_title=A genome-scale metabolic reconstruction for Escherichia coli K-12 MG1655 that accounts for 1260 ORFs and thermodynamic information,;  citation_journal_title=Mol. Syst. Biol;  citation_year=2007;  citation_volume=3;  citation_pages=121" /><meta name="citation_reference" content="citation_title=Escherichia coli minimum genome factory,;  citation_journal_title=Biotechnol. Appl. Biochem;  citation_year=2007;  citation_volume=46;  citation_pages=157-67; " /><meta name="citation_reference" content="citation_title=Design and synthesis of a minimal bacterial genome,;  citation_journal_title=Science;  citation_year=2016;  citation_volume=351;  citation_pages=aad6253" /><meta name="citation_reference" content="citation_title=Systems biology perspectives on minimal and simpler cells,;  citation_journal_title=Microbiol. Mol. Biol. Rev;  citation_year=2014;  citation_volume=78;  citation_pages=487-509; " /><meta name="citation_reference" content="citation_title=Construction of Escherichia coli K-12 in-frame, single-gene knockout mutants: the Keio collection,;  citation_journal_title=Mol. Syst. Biol;  citation_year=2006;  citation_volume=2;  citation_pages=2006 0008" /><meta name="citation_reference" content="citation_title=Update on the Keio collection of Escherichia coli single-gene deletion mutants,;  citation_journal_title=Mol. Syst. Biol;  citation_year=2009;  citation_volume=5;  citation_pages=335" /><meta name="citation_reference" content="citation_title=Emergent properties of reduced-genome Escherichia coli,;  citation_journal_title=Science;  citation_year=2006;  citation_volume=312;  citation_pages=1044-6; " /><meta name="citation_reference" content="citation_title=Construction of consecutive deletions of the Escherichia coli chromosome,;  citation_journal_title=Mol. Syst. Biol;  citation_year=2007;  citation_volume=3;  citation_pages=132" /><meta name="citation_reference" content="citation_title=Superpositioning of deletions promotes growth of Escherichia coli with a reduced genome,;  citation_journal_title=DNA Res;  citation_year=2008;  citation_volume=15;  citation_pages=277-84; " /><meta name="citation_reference" content="citation_title=Creation of a bacterial cell controlled by a chemically synthesized genome,;  citation_journal_title=Science;  citation_year=2010;  citation_volume=329;  citation_pages=52-6; " /><meta name="citation_reference" content="citation_title=Systematic genome reductions: theoretical and experimental approaches,;  citation_journal_title=Chem. Rev;  citation_year=2007;  citation_volume=107;  citation_pages=3498-513; " /><meta name="citation_reference" content="citation_title=The dawn of evolutionary genome engineering,;  citation_journal_title=Nat. Rev. Genet;  citation_year=2014;  citation_volume=15;  citation_pages=504-12; " /><meta name="citation_reference" content="citation_title=Impact of genome reduction on bacterial metabolism and its regulation,;  citation_journal_title=Science;  citation_year=2009;  citation_volume=326;  citation_pages=1263-8; " /><meta name="citation_reference" content="citation_title=Expression of two recombinant chloramphenicol acetyltransferase variants in highly reduced genome Escherichia coli strains,;  citation_journal_title=Biotechnol. Bioeng;  citation_year=2007;  citation_volume=98;  citation_pages=1056-70; " /><meta name="citation_reference" content="citation_title=Enhanced recombinant protein productivity by genome reduction in Bacillus subtilis,;  citation_journal_title=DNA Res;  citation_year=2008;  citation_volume=15;  citation_pages=73-81; " /><meta name="citation_reference" content="citation_title=Metabolic engineering of a reduced-genome strain of Escherichia coli for L-threonine production,;  citation_journal_title=Microb. Cell Factories;  citation_year=2009;  citation_volume=8;  citation_pages=2" /><meta name="citation_reference" content="citation_title=A reduced genome decreases the host carrying capacity for foreign DNA,;  citation_journal_title=Microb. Cell Factories;  citation_year=2014;  citation_volume=13;  citation_pages=49" /><meta name="citation_reference" content="citation_title=Low-mutation-rate, reduced-genome Escherichia coli: an improved host for faithful maintenance of engineered genetic constructs,;  citation_journal_title=Microb. Cell Factories;  citation_year=2012;  citation_volume=11;  citation_pages=11" /><meta name="citation_reference" content="citation_title=Reduced evolvability of Escherichia coli MDS42, an IS-less cellular chassis for molecular and synthetic biology applications,;  citation_journal_title=Microb. Cell Factories;  citation_year=2010;  citation_volume=9;  citation_pages=38" /><meta name="citation_reference" content="citation_title=Growth temperature and genome size in bacteria are negatively correlated, suggesting genomic streamlining during thermal adaptation,;  citation_journal_title=Genome Biol. Evol;  citation_year=2013;  citation_volume=5;  citation_pages=966-77; " /><meta name="citation_reference" content="citation_title=Indispensability of horizontally transferred genes and its impact on bacterial genome streamlining,;  citation_journal_title=Mol. Biol. Evol;  citation_year=2016;  citation_volume=33;  citation_pages=1257-69; " /><meta name="citation_reference" content="citation_title=Specific growth rate dependent transcriptome profiling of Escherichia coli K12 MG1655 in accelerostat cultures,;  citation_journal_title=J. Biotechnol;  citation_year=2010;  citation_volume=145;  citation_pages=60-5; " /><meta name="citation_reference" content="citation_title=The Escherichia coli transcriptome linked to growth fitness,;  citation_journal_title=Genomics Data;  citation_year=2016;  citation_volume=7;  citation_pages=1-3; " /><meta name="citation_reference" content="citation_title=Global coordination in adaptation to gene rewiring,;  citation_journal_title=Nucleic Acids Res;  citation_year=2015;  citation_volume=43;  citation_pages=1304-16; " /><meta name="citation_reference" content="citation_title=Growth rate-coordinated transcriptome reorganization in bacteria,;  citation_journal_title=BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=808" /><meta name="citation_reference" content="citation_title=Bacterial transcriptome reorganization in thermal adaptive evolution,;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=802" /><meta name="citation_reference" content="citation_title=Cell size and nucleoid organization of engineered Escherichia coli cells with a reduced genome,;  citation_journal_title=Mol. Microbial;  citation_year=2005;  citation_volume=55;  citation_pages=137-49; " /><meta name="citation_reference" content="citation_title=Multilevel comparative analysis of the contributions of genome reduction and heat shock to the Escherichia coli transcriptome,;  citation_journal_title=BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=25" /><meta name="citation_reference" content="citation_title=Nutrient-dependent growth defects and mutability of mutators in Escherichia coli,;  citation_journal_title=Genes Cells;  citation_year=2015;  citation_volume=20;  citation_pages=68-76; " /><meta name="citation_reference" content="citation_title=A new look at the statistical model identification,;  citation_journal_title=IEEE Trans. Automatic Control;  citation_year=1974;  citation_volume=19;  citation_pages=8" /><meta name="citation_reference" content="citation_title=Model selection for ecologists: the worldviews of AIC and BIC,;  citation_journal_title=Ecology;  citation_year=2014;  citation_volume=95;  citation_pages=631-6; " /><meta name="citation_reference" content="citation_title=R: a language for data analysis and graphics,;  citation_journal_title=J. Comput. Graph. Stat;  citation_year=1996;  citation_volume=5;  citation_pages=299-314; " /><meta name="citation_reference" content="citation_publisher=Sage, Thousand Oaks CA;  citation_title=An R Companion to Applied Regression;  citation_year=2011; " /><meta name="citation_reference" content="citation_title=Escherichia coli K-12: a cooperatively developed annotation snapshot–2005,;  citation_journal_title=Nucleic Acids Res;  citation_year=2006;  citation_volume=34;  citation_pages=1-9; " /><meta name="citation_reference" content="citation_title=BLAST Ring Image Generator (BRIG): simple prokaryote genome comparisons,;  citation_journal_title=BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=402" /><meta name="citation_reference" content="citation_title=Molecular archaeology of the Escherichia coli genome,;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=1998;  citation_volume=95;  citation_pages=9413-7; " /><meta name="citation_reference" content="citation_title=Genomes in turmoil: quantification of genome dynamics in prokaryote supergenomes,;  citation_journal_title=BMC Biol;  citation_year=2014;  citation_volume=12;  citation_pages=66" /><meta name="citation_reference" content="citation_title=Genome-wide molecular clock and horizontal gene transfer in bacterial evolution,;  citation_journal_title=J. Bacteriol;  citation_year=2004;  citation_volume=186;  citation_pages=6575-85; " /><meta name="citation_reference" content="citation_title=Genomics of bacteria and archaea: the emerging dynamic view of the prokaryotic world,;  citation_journal_title=Nucleic Acids Res;  citation_year=2008;  citation_volume=36;  citation_pages=6688-719; " /><meta name="citation_reference" content="citation_title=Horizontal gene transfer: a critical view,;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2003;  citation_volume=100;  citation_pages=9658-62; " /><meta name="citation_reference" content="citation_publisher=Wiley-VCH, Weinheim;  citation_title=Stress and Environmental Control of Gene Expression in Bacteria;  citation_year=2016; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/23/6/517/8325641/dsw035.pdf" /><meta name="description" content="Genome reduction by removing dispensable genomic sequences in bacteria is commonly used in both fundamental and applied studies to determine the minimal genetic" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/23/6/517/2647442" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/23/6/517/2647442" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/23/6/517/2647442");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw035';
    </script>
<title>
	Correlation between genome reduction and bacterial growth | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2647442" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZ5lPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLdXJva2F3YSwgTWFzYW9taSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBMaWZlIGFuZCBFbnZpcm9ubWVudGFsIFNjaWVuY2VzLCBVbml2ZXJzaXR5IG9mIFRzdWt1YmEsIEliYXJha2kgMzA1LTg1NzIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2VubywgU2hpZ2V0byIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBJbmZvcm1hdGlvbiBTY2llbmNlIGFuZCBUZWNobm9sb2d5LCBPc2FrYSBVbml2ZXJzaXR5LCBPc2FrYSA1NjUtMDg3MSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNYXRzdWRhLCBIaWRlbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBJbmZvcm1hdGlvbiBTY2llbmNlIGFuZCBUZWNobm9sb2d5LCBPc2FrYSBVbml2ZXJzaXR5LCBPc2FrYSA1NjUtMDg3MSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJZaW5nLCBCZWktV2VuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIExpZmUgYW5kIEVudmlyb25tZW50YWwgU2NpZW5jZXMsIFVuaXZlcnNpdHkgb2YgVHN1a3ViYSwgSWJhcmFraSAzMDUtODU3MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkNvcnJlbGF0aW9uIGJldHdlZW4gZ2Vub21lIHJlZHVjdGlvbiBhbmQgYmFjdGVyaWFsIGdyb3d0aCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjUxNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNTI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN3MDM1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTYvMTIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaW5pbWl6aW5nIHRoZSBnZW5vbWUgb2YgRXNjaGVyaWNoaWEgY29saS4gTW90aXZhdGlvbiBhbmQgc3RyYXRlZ3ksOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm4uIE4uIFkuIEFjYWQuIFNjaTsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT03NDU7ICBjaXRhdGlvbl9wYWdlcz0xLTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb21wbGV0ZSBnZW5vbWUgc2VxdWVuY2Ugb2YgRXNjaGVyaWNoaWEgY29saSBLLTEyLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNzc7ICBjaXRhdGlvbl9wYWdlcz0xNDUzLTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbmdpbmVlcmluZyBhIHJlZHVjZWQgRXNjaGVyaWNoaWEgY29saSBnZW5vbWUsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9NjQwLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pbmltaXphdGlvbiBvZiB0aGUgRXNjaGVyaWNoaWEgY29saSBnZW5vbWUgdXNpbmcgYSBUbjUtdGFyZ2V0ZWQgQ3JlL2xveFAgZXhjaXNpb24gc3lzdGVtLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBCaW90ZWNobm9sOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MTAxOC0yMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2Nhcmxlc3MgZW5naW5lZXJpbmcgb2YgdGhlIEVzY2hlcmljaGlhIGNvbGkgZ2Vub21lLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWV0aG9kcyBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NDE2OyAgY2l0YXRpb25fcGFnZXM9MjUxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ2Vub21lLXNjYWxlIG1ldGFib2xpYyByZWNvbnN0cnVjdGlvbiBmb3IgRXNjaGVyaWNoaWEgY29saSBLLTEyIE1HMTY1NSB0aGF0IGFjY291bnRzIGZvciAxMjYwIE9SRnMgYW5kIHRoZXJtb2R5bmFtaWMgaW5mb3JtYXRpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIFN5c3QuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTEyMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVzY2hlcmljaGlhIGNvbGkgbWluaW11bSBnZW5vbWUgZmFjdG9yeSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb3RlY2hub2wuIEFwcGwuIEJpb2NoZW07ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NDY7ICBjaXRhdGlvbl9wYWdlcz0xNTctNjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlc2lnbiBhbmQgc3ludGhlc2lzIG9mIGEgbWluaW1hbCBiYWN0ZXJpYWwgZ2Vub21lLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNTE7ICBjaXRhdGlvbl9wYWdlcz1hYWQ2MjUzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3lzdGVtcyBiaW9sb2d5IHBlcnNwZWN0aXZlcyBvbiBtaW5pbWFsIGFuZCBzaW1wbGVyIGNlbGxzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWljcm9iaW9sLiBNb2wuIEJpb2wuIFJldjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT03ODsgIGNpdGF0aW9uX3BhZ2VzPTQ4Ny01MDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBvZiBFc2NoZXJpY2hpYSBjb2xpIEstMTIgaW4tZnJhbWUsIHNpbmdsZS1nZW5lIGtub2Nrb3V0IG11dGFudHM6IHRoZSBLZWlvIGNvbGxlY3Rpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIFN5c3QuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTIwMDYgMDAwOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVwZGF0ZSBvbiB0aGUgS2VpbyBjb2xsZWN0aW9uIG9mIEVzY2hlcmljaGlhIGNvbGkgc2luZ2xlLWdlbmUgZGVsZXRpb24gbXV0YW50cyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gU3lzdC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MzM1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW1lcmdlbnQgcHJvcGVydGllcyBvZiByZWR1Y2VkLWdlbm9tZSBFc2NoZXJpY2hpYSBjb2xpLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zMTI7ICBjaXRhdGlvbl9wYWdlcz0xMDQ0LTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBvZiBjb25zZWN1dGl2ZSBkZWxldGlvbnMgb2YgdGhlIEVzY2hlcmljaGlhIGNvbGkgY2hyb21vc29tZSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gU3lzdC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MTMyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3VwZXJwb3NpdGlvbmluZyBvZiBkZWxldGlvbnMgcHJvbW90ZXMgZ3Jvd3RoIG9mIEVzY2hlcmljaGlhIGNvbGkgd2l0aCBhIHJlZHVjZWQgZ2Vub21lLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTI3Ny04NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q3JlYXRpb24gb2YgYSBiYWN0ZXJpYWwgY2VsbCBjb250cm9sbGVkIGJ5IGEgY2hlbWljYWxseSBzeW50aGVzaXplZCBnZW5vbWUsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTMyOTsgIGNpdGF0aW9uX3BhZ2VzPTUyLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN5c3RlbWF0aWMgZ2Vub21lIHJlZHVjdGlvbnM6IHRoZW9yZXRpY2FsIGFuZCBleHBlcmltZW50YWwgYXBwcm9hY2hlcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNoZW0uIFJldjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDc7ICBjaXRhdGlvbl9wYWdlcz0zNDk4LTUxMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGRhd24gb2YgZXZvbHV0aW9uYXJ5IGdlbm9tZSBlbmdpbmVlcmluZyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTUwNC0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW1wYWN0IG9mIGdlbm9tZSByZWR1Y3Rpb24gb24gYmFjdGVyaWFsIG1ldGFib2xpc20gYW5kIGl0cyByZWd1bGF0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zMjY7ICBjaXRhdGlvbl9wYWdlcz0xMjYzLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cHJlc3Npb24gb2YgdHdvIHJlY29tYmluYW50IGNobG9yYW1waGVuaWNvbCBhY2V0eWx0cmFuc2ZlcmFzZSB2YXJpYW50cyBpbiBoaWdobHkgcmVkdWNlZCBnZW5vbWUgRXNjaGVyaWNoaWEgY29saSBzdHJhaW5zLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvdGVjaG5vbC4gQmlvZW5nOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTk4OyAgY2l0YXRpb25fcGFnZXM9MTA1Ni03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW5oYW5jZWQgcmVjb21iaW5hbnQgcHJvdGVpbiBwcm9kdWN0aXZpdHkgYnkgZ2Vub21lIHJlZHVjdGlvbiBpbiBCYWNpbGx1cyBzdWJ0aWxpcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz03My04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWV0YWJvbGljIGVuZ2luZWVyaW5nIG9mIGEgcmVkdWNlZC1nZW5vbWUgc3RyYWluIG9mIEVzY2hlcmljaGlhIGNvbGkgZm9yIEwtdGhyZW9uaW5lIHByb2R1Y3Rpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1NaWNyb2IuIENlbGwgRmFjdG9yaWVzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0yIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSByZWR1Y2VkIGdlbm9tZSBkZWNyZWFzZXMgdGhlIGhvc3QgY2FycnlpbmcgY2FwYWNpdHkgZm9yIGZvcmVpZ24gRE5BLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWljcm9iLiBDZWxsIEZhY3RvcmllczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTQ5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TG93LW11dGF0aW9uLXJhdGUsIHJlZHVjZWQtZ2Vub21lIEVzY2hlcmljaGlhIGNvbGk6IGFuIGltcHJvdmVkIGhvc3QgZm9yIGZhaXRoZnVsIG1haW50ZW5hbmNlIG9mIGVuZ2luZWVyZWQgZ2VuZXRpYyBjb25zdHJ1Y3RzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWljcm9iLiBDZWxsIEZhY3RvcmllczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTExIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVkdWNlZCBldm9sdmFiaWxpdHkgb2YgRXNjaGVyaWNoaWEgY29saSBNRFM0MiwgYW4gSVMtbGVzcyBjZWxsdWxhciBjaGFzc2lzIGZvciBtb2xlY3VsYXIgYW5kIHN5bnRoZXRpYyBiaW9sb2d5IGFwcGxpY2F0aW9ucyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1pY3JvYi4gQ2VsbCBGYWN0b3JpZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R3Jvd3RoIHRlbXBlcmF0dXJlIGFuZCBnZW5vbWUgc2l6ZSBpbiBiYWN0ZXJpYSBhcmUgbmVnYXRpdmVseSBjb3JyZWxhdGVkLCBzdWdnZXN0aW5nIGdlbm9taWMgc3RyZWFtbGluaW5nIGR1cmluZyB0aGVybWFsIGFkYXB0YXRpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9OTY2LTc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmRpc3BlbnNhYmlsaXR5IG9mIGhvcml6b250YWxseSB0cmFuc2ZlcnJlZCBnZW5lcyBhbmQgaXRzIGltcGFjdCBvbiBiYWN0ZXJpYWwgZ2Vub21lIHN0cmVhbWxpbmluZyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTEyNTctNjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNwZWNpZmljIGdyb3d0aCByYXRlIGRlcGVuZGVudCB0cmFuc2NyaXB0b21lIHByb2ZpbGluZyBvZiBFc2NoZXJpY2hpYSBjb2xpIEsxMiBNRzE2NTUgaW4gYWNjZWxlcm9zdGF0IGN1bHR1cmVzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmlvdGVjaG5vbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDU7ICBjaXRhdGlvbl9wYWdlcz02MC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgRXNjaGVyaWNoaWEgY29saSB0cmFuc2NyaXB0b21lIGxpbmtlZCB0byBncm93dGggZml0bmVzcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9taWNzIERhdGE7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTEtMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2xvYmFsIGNvb3JkaW5hdGlvbiBpbiBhZGFwdGF0aW9uIHRvIGdlbmUgcmV3aXJpbmcsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTEzMDQtMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdyb3d0aCByYXRlLWNvb3JkaW5hdGVkIHRyYW5zY3JpcHRvbWUgcmVvcmdhbml6YXRpb24gaW4gYmFjdGVyaWEsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz04MDgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYWN0ZXJpYWwgdHJhbnNjcmlwdG9tZSByZW9yZ2FuaXphdGlvbiBpbiB0aGVybWFsIGFkYXB0aXZlIGV2b2x1dGlvbiw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTgwMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNlbGwgc2l6ZSBhbmQgbnVjbGVvaWQgb3JnYW5pemF0aW9uIG9mIGVuZ2luZWVyZWQgRXNjaGVyaWNoaWEgY29saSBjZWxscyB3aXRoIGEgcmVkdWNlZCBnZW5vbWUsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIE1pY3JvYmlhbDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT01NTsgIGNpdGF0aW9uX3BhZ2VzPTEzNy00OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXVsdGlsZXZlbCBjb21wYXJhdGl2ZSBhbmFseXNpcyBvZiB0aGUgY29udHJpYnV0aW9ucyBvZiBnZW5vbWUgcmVkdWN0aW9uIGFuZCBoZWF0IHNob2NrIHRvIHRoZSBFc2NoZXJpY2hpYSBjb2xpIHRyYW5zY3JpcHRvbWUsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0yNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU51dHJpZW50LWRlcGVuZGVudCBncm93dGggZGVmZWN0cyBhbmQgbXV0YWJpbGl0eSBvZiBtdXRhdG9ycyBpbiBFc2NoZXJpY2hpYSBjb2xpLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXMgQ2VsbHM7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz02OC03NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBuZXcgbG9vayBhdCB0aGUgc3RhdGlzdGljYWwgbW9kZWwgaWRlbnRpZmljYXRpb24sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JRUVFIFRyYW5zLiBBdXRvbWF0aWMgQ29udHJvbDsgIGNpdGF0aW9uX3llYXI9MTk3NDsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2RlbCBzZWxlY3Rpb24gZm9yIGVjb2xvZ2lzdHM6IHRoZSB3b3JsZHZpZXdzIG9mIEFJQyBhbmQgQklDLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RWNvbG9neTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT05NTsgIGNpdGF0aW9uX3BhZ2VzPTYzMS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SOiBhIGxhbmd1YWdlIGZvciBkYXRhIGFuYWx5c2lzIGFuZCBncmFwaGljcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIENvbXB1dC4gR3JhcGguIFN0YXQ7ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTI5OS0zMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1TYWdlLCBUaG91c2FuZCBPYWtzIENBOyAgY2l0YXRpb25fdGl0bGU9QW4gUiBDb21wYW5pb24gdG8gQXBwbGllZCBSZWdyZXNzaW9uOyAgY2l0YXRpb25feWVhcj0yMDExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fc2NoZXJpY2hpYSBjb2xpIEstMTI6IGEgY29vcGVyYXRpdmVseSBkZXZlbG9wZWQgYW5ub3RhdGlvbiBzbmFwc2hvdOKAkzIwMDUsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIGNpdGF0aW9uX3BhZ2VzPTEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QkxBU1QgUmluZyBJbWFnZSBHZW5lcmF0b3IgKEJSSUcpOiBzaW1wbGUgcHJva2FyeW90ZSBnZW5vbWUgY29tcGFyaXNvbnMsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz00MDIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgYXJjaGFlb2xvZ3kgb2YgdGhlIEVzY2hlcmljaGlhIGNvbGkgZ2Vub21lLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVLiBTLiBBOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTk1OyAgY2l0YXRpb25fcGFnZXM9OTQxMy03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWVzIGluIHR1cm1vaWw6IHF1YW50aWZpY2F0aW9uIG9mIGdlbm9tZSBkeW5hbWljcyBpbiBwcm9rYXJ5b3RlIHN1cGVyZ2Vub21lcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9NjYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBtb2xlY3VsYXIgY2xvY2sgYW5kIGhvcml6b250YWwgZ2VuZSB0cmFuc2ZlciBpbiBiYWN0ZXJpYWwgZXZvbHV0aW9uLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmFjdGVyaW9sOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE4NjsgIGNpdGF0aW9uX3BhZ2VzPTY1NzUtODU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWNzIG9mIGJhY3RlcmlhIGFuZCBhcmNoYWVhOiB0aGUgZW1lcmdpbmcgZHluYW1pYyB2aWV3IG9mIHRoZSBwcm9rYXJ5b3RpYyB3b3JsZCw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9NjY4OC03MTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhvcml6b250YWwgZ2VuZSB0cmFuc2ZlcjogYSBjcml0aWNhbCB2aWV3LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVLiBTLiBBOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEwMDsgIGNpdGF0aW9uX3BhZ2VzPTk2NTgtNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1XaWxleS1WQ0gsIFdlaW5oZWltOyAgY2l0YXRpb25fdGl0bGU9U3RyZXNzIGFuZCBFbnZpcm9ubWVudGFsIENvbnRyb2wgb2YgR2VuZSBFeHByZXNzaW9uIGluIEJhY3RlcmlhOyAgY2l0YXRpb25feWVhcj0yMDE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjMvNi81MTcvODMyNTY0MS9kc3cwMzUucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJHZW5vbWUgcmVkdWN0aW9uIGJ5IHJlbW92aW5nIGRpc3BlbnNhYmxlIGdlbm9taWMgc2VxdWVuY2VzIGluIGJhY3RlcmlhIGlzIGNvbW1vbmx5IHVzZWQgaW4gYm90aCBmdW5kYW1lbnRhbCBhbmQgYXBwbGllZCBzdHVkaWVzIHRvIGRldGVybWluZSB0aGUgbWluaW1hbCBnZW5ldGljIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0yNjQ3NDQyJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZCBPJryZuaW1jrUuY75qbPyHJznV" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f6%2f517%2f2647442" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f6%2f517%2f2647442" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=89339" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/23/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/6/0/m_dnares_23_6cover.png?Expires=2147483647&amp;Signature=Max3WHFhEi-UCkzx6P3aYqgGeApUTTF7rFn5PtyKoLTwqYHjx5CL~q9P-cecyPnMaWMfAiuwjwIq4FX0zeusJJeQovtOXJf6HCEN1nyt6gGT-V0qXZbGd9kAS2xoR8~AhSm3sDF8oaeSvJqsJf7X0O3CSXDRO0TzM8dBxL8950DJw1dPLvl7nHrUNq2p9yI4l0hMXt0UcvxAxqNgAki~K8SZjGpu3wP2yBtmQAG5iZnvNTuV7drgxbF2S717hI9S~I3x6JRIOVuEraI2uq8N7rEjIvQ14-5VJbutut2EpRpGHZf7SQYkJkdSrHr028n8qO7shpq47NBQIX2Fd8XLBg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/23/6">
                        <div class="volume trailing-comma">Volume 23</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="48004809">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#48004809">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="48004814">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#48004814">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="48004830">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#48004830">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="48004852">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#48004852">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="48004860">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#48004860">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/23/6/507/2647441">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/23/6/527/2647443">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Correlation between genome reduction and bacterial growth
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masaomi Kurokawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Graduate School of Life and Environmental Sciences, University of Tsukuba, Ibaraki 305-8572, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masaomi+Kurokawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kurokawa M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kurokawa M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Masaomi Kurokawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shigeto Seno</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Graduate School of Information Science and Technology, Osaka University, Osaka 565-0871, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shigeto+Seno">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Seno S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Seno S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shigeto Seno</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideo Matsuda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Graduate School of Information Science and Technology, Osaka University, Osaka 565-0871, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideo+Matsuda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Matsuda H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Matsuda H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hideo Matsuda</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Bei-Wen Ying</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Graduate School of Life and Environmental Sciences, University of Tsukuba, Ibaraki 305-8572, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw035-cor1" data-open="dsw035-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*To</div> whom correspondence should be addressed. Tel/Fax. +81 29 853 6633. Email: <a href="mailto:ying.beiwen.gf@u.tsukuba.ac.jp" target="_blank">ying.beiwen.gf@u.tsukuba.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Bei-Wen+Ying">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ying B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ying B&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Bei-Wen Ying</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw035-FM2"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this study.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw035-FM1"><div class="fn"><div class="footnote-content">Edited by Dr Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 23, Issue 6, 1 December 2016, Pages 517–525, <a href='https://doi.org/10.1093/dnares/dsw035'>https://doi.org/10.1093/dnares/dsw035</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">03 July 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">19 April 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">08 June 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2647442" href="/dnaresearch/article-pdf/23/6/517/8325641/dsw035.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Masaomi Kurokawa, Shigeto Seno, Hideo Matsuda, Bei-Wen Ying;  Correlation between genome reduction and bacterial growth, <em>DNA Research</em>, Volume 23, Issue 6, 1 December 2016, Pages 517–525, <a href='https://doi.org/10.1093/dnares/dsw035'>https://doi.org/10.1093/dnares/dsw035</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2647442?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2647442?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2647442?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2647442?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2647442?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2647442?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2647442?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Correlation%20between%20genome%20reduction%20and%20bacterial%20growth&amp;publicationDate=2016-07-03&amp;volumeNum=23&amp;issueNum=6&amp;author=Kurokawa%2C%20Masaomi%3B%20Seno%2C%20Shigeto&amp;startPage=517&amp;endPage=525&amp;contentId=10.1093%2Fdnares%2Fdsw035&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f6%2f517%2f2647442" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





<section class="abstract"><p>Genome reduction by removing dispensable genomic sequences in bacteria is commonly used in both fundamental and applied studies to determine the minimal genetic requirements for a living system or to develop highly efficient bioreactors. Nevertheless, whether and how the accumulative loss of dispensable genomic sequences disturbs bacterial growth remains unclear. To investigate the relationship between genome reduction and growth, a series of <em>Escherichia coli</em> strains carrying genomes reduced in a stepwise manner were used. Intensive growth analyses revealed that the accumulation of multiple genomic deletions caused decreases in the exponential growth rate and the saturated cell density in a deletion-length-dependent manner as well as gradual changes in the patterns of growth dynamics, regardless of the growth media. Accordingly, a perspective growth model linking genome evolution to genome engineering was proposed. This study provides the first demonstration of a quantitative connection between genomic sequence and bacterial growth, indicating that growth rate is potentially associated with dispensable genomic sequences.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;growth rate&quot;">growth rate</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;dispensable genomic sequence&quot;">dispensable genomic sequence</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome reduction&quot;">genome reduction</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;accessory gene&quot;">accessory gene</a></div>                            <h2 scrollto-destination=48004809 id="48004809" class="section-title" data-legacy-id=SEC1>1. Introduction</h2>
<p>As a collection of genetic information, genomes are thought to be sequence redundant.<a reveal-id="dsw035-B1" data-open="dsw035-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> Since the first genome sequence was reported in bacteria,<a reveal-id="dsw035-B2" data-open="dsw035-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> experimental and theoretical studies<a reveal-id="dsw035-B3 dsw035-B4 dsw035-B5 dsw035-B6 dsw035-B7 dsw035-B8" data-open="dsw035-B3 dsw035-B4 dsw035-B5 dsw035-B6 dsw035-B7 dsw035-B8" class="link link-ref link-reveal xref-bibr"><sup>3–8</sup></a> have attempted to determine the minimal sets of genes essential for cellular life and to define what constitutes a minimal genome.<a reveal-id="dsw035-B9" data-open="dsw035-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> A number of sound experiments have succeeded in constructing <em>Escherichia coli</em> libraries either of knockout strains, by deleting particular functional genes,<a reveal-id="dsw035-B10" data-open="dsw035-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsw035-B11" data-open="dsw035-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> or of a variety of genome sizes, by removing the non-essential genomic sequences.<a reveal-id="dsw035-B12 dsw035-B13 dsw035-B14" data-open="dsw035-B12 dsw035-B13 dsw035-B14" class="link link-ref link-reveal xref-bibr"><sup>12–14</sup></a> Recently, thought-provoking experiments involving synthetic genomes harbouring the smallest gene sets and exhibiting high replication efficiency<a reveal-id="dsw035-B8" data-open="dsw035-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsw035-B15" data-open="dsw035-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> have further demonstrated that native genomic sequences are redundant in terms of the capacity for life. As a consequence, genetically engineered genomes lacking dispensable sequences are commonly used in both basic science and applied research.<a reveal-id="dsw035-B16" data-open="dsw035-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsw035-B17" data-open="dsw035-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a></p><p>Reduced genomes have both advantages and disadvantages regarding cellular functions,<a reveal-id="dsw035-B18" data-open="dsw035-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> although few studies have commented on whether the deleted dispensable genomic sequences contribute to biological fitness. Intensive studies have shown that reduced genomes increase productivity in generating gene products,<a reveal-id="dsw035-B19 dsw035-B20 dsw035-B21" data-open="dsw035-B19 dsw035-B20 dsw035-B21" class="link link-ref link-reveal xref-bibr"><sup>19–21</sup></a> which is a major advantage in various applications, and reduce the capacity to take up foreign DNA<sup>22</sup> as well as mutability,<a reveal-id="dsw035-B23" data-open="dsw035-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a><sup>,</sup><a reveal-id="dsw035-B24" data-open="dsw035-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> which are likely to be disadvantageous to evolving living organisms. These results imply that the deletion of the “junk” regions of the genome increases the efficiency of biological processes but decreases cellular plasticity in living organisms.<a reveal-id="dsw035-B22" data-open="dsw035-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> Bioinformatic analysis has successfully linked the genome size to growth conditions.<a reveal-id="dsw035-B25" data-open="dsw035-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> However, insight into the relation between genome size and growth fitness is lacking. Although genome reduction has been found not to affect the capacity for growth,<a reveal-id="dsw035-B12 dsw035-B13 dsw035-B14" data-open="dsw035-B12 dsw035-B13 dsw035-B14" class="link link-ref link-reveal xref-bibr"><sup>12–14</sup></a><sup>,</sup><a reveal-id="dsw035-B26" data-open="dsw035-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> experimental and theoretical studies of the global effects of genome reduction on cellular life are required.</p><p>Considering the reduced genomes as growing systems, the accumulated deletion of dispensable genomic sequences is assumed to be associated with growth rate, a common global parameter that represents the overall activity (output of the genomic and metabolic reactions) of a living cell. Such a global effect of bacterial growth has been demonstrated by the correlation between transcriptome and growth fitness<a reveal-id="dsw035-B27" data-open="dsw035-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a><sup>,</sup><a reveal-id="dsw035-B28" data-open="dsw035-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> in both adaptation<a reveal-id="dsw035-B29" data-open="dsw035-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a><sup>,</sup><a reveal-id="dsw035-B30" data-open="dsw035-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> and evolution.<a reveal-id="dsw035-B31" data-open="dsw035-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> However, studies constructing reduced genomes have reported diverse evidence regarding growth. Rough measurements of a series of reduced genomes (constructed from W3110) have not indicated any significant differences in the growth rate on the basis of optical density (OD),<a reveal-id="dsw035-B14" data-open="dsw035-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> whereas a closely related study has reported increased cell division time in another reduced genome library (constructed from MG1655)<a reveal-id="dsw035-B32" data-open="dsw035-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a>. In addition, the well-known “clean” genome (MDS42)<a reveal-id="dsw035-B12" data-open="dsw035-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> has a growth rate comparable to that of its parent wild type genome (MG1655) in minimal medium,<a reveal-id="dsw035-B22" data-open="dsw035-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><a reveal-id="dsw035-B33" data-open="dsw035-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> and no general trend has been observed between genome size and growth in rich medium.<a reveal-id="dsw035-B26" data-open="dsw035-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> Whether and how the genome reduction links to the bacterial growth is still in argument, due to the different thinking on growth cost and/or genome evolution. Thus, the biological impact of genome reduction remains to be determined under different conditions in varied point views. Considering these various findings and assumptions, we sought to quantitatively evaluate the relation between cumulatively reduced genomic sequences and the changes in growth under nutritional variation.</p><p>In the present study, we addressed the question of whether and how the cumulative deletion of dispensable genomic sequences contributes to the population growth of <em>E. coli</em> in varied conditions. We used a previously constructed <em>E. coli</em> library comprising a series of genomes that were reduced in a stepwise manner,<a reveal-id="dsw035-B14" data-open="dsw035-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> and easy access to their genetic backgrounds was crucial for the analysis. Evaluation of the contribution of genomic sequences dispensable for bacterial growth was achieved through comprehensive growth assays of these <em>E. coli</em> strains. The results offer a perspective opinion of the relationship between genome size and growth fitness as a growth mode that is potentially involved in genome evolution and is applicable to genome engineering.</p>                            <h2 scrollto-destination=48004814 id="48004814" class="section-title" data-legacy-id=SEC2>2. Materials and methods</h2>
                            <h3 scrollto-destination=48004815 id="48004815" class="section-title" data-legacy-id=SEC2.1>2.1 Strains and media</h3>
<p>A total of 28 <em>E. coli</em> strains with reduced genomes (KHK collection) and the wild type genome W3110 were obtained from the National BioResource Project, National Institute of Genetics, Shizuoka, Japan. The strains, which were initially cultured and stored in complete LB medium (Miller), were inoculated into M63 minimal medium and re-cultured until the optical density (OD<sub>600</sub>) reached 0.01∼0.1, representing the exponential growth phase. The cell culture was subsequently stored in 20 tubes in small aliquots (100 μl per tube) for future use. <em>Escherichia coli</em> culture stocks (newly prepared single tubes) were used once, and the remainder was discarded. The compositions of the three different media, the complete medium (LB), the minimal medium (M63), and the minimal medium supplemented with 20 amino acids (MAA), used in the present study were described previously.<a reveal-id="dsw035-B34" data-open="dsw035-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a></p>                            <h3 scrollto-destination=48004817 id="48004817" class="section-title" data-legacy-id=SEC2.2>2.2 Acquisition of growth curves</h3>
<p>The <em>E. coli</em> culture stocks were completely dissolved and diluted 10- to 100-fold with fresh media in a test tube. Three types of media, LB, MAA, and M63, were used, which represent the rich, supplementary, and poor growth conditions, respectively. The diluted cell mixtures of multiple dilution rates were subsequently loaded into a 96-well plate (Costar) in three to six wells per dilution rate per strain, with 200 μl of diluted cell mixture applied in each well. The 96-well plate was incubated in a plate reader (Epoch2, BioTek) with a rotation rate of 600 rpm at 37 °C. Cell growth was detected at an absorbance of 600 nm, with readings obtained at 30-min or 1-h intervals for 24 to 48 h. The growth curves were obtained for each well (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035_Supp.pdf?Expires=2147483647&Signature=yZ2sO9VP2Z3TYSkUXP~QOTiGs27MRweE6AWrjAkM9HNiC9Gvjlx8POMjF8qk8LzqIpRL-Nvx1OEyGZYpsf72-JrDcHVfi~vBdLUCmHfcPfCwtACeE9Y5b-eu~lNdQ5TNotgue4AFDZPtd2UKgPwc88ZFLFmM-6Fim-Rh6qebLui7ixMu0rZrlaWQ--ByMqKk6u5Btmr0WkLvRP-M3PbG8EJEsFDGSlUtKadVKMtkFO~-uRbgucy-VgNpi3L2sG4g5ynwG7dcp-7r1xkolCwxGosoelpENGXM6PfKgknd8ZmhUlKCiAUcCDUPq6snL6ckRnM0rSqta1nBlMMlcQ2XHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2A</a></span>). Repeated tests were performed, which resulted in 11 to 30 growth curves used for further calculations of growth rate and population density for each strain at each growth condition (medium). Growth curves were acquired in three different media: LB, M63 and MAA.</p>                            <h3 scrollto-destination=48004819 id="48004819" class="section-title" data-legacy-id=SEC2.3>2.3 Calculation of exponential growth rate and saturated population density</h3>
<div class="block-child-p">The growth rate (<em>μ</em>) during exponential phase and population density during stationary phase was evaluated according to the growth curves. The growth rates were calculated from every two continuous reading points in a growth curve, according to the following equation (<a href="#E1" class="sectionLink xref-sec">Equation 1</a>). <a id="E1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E1"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>μ</mi><mo>=</mo><mi>L</mi><mi>N</mi><mo stretchy="false">(</mo><mfrac><mrow><msub><mi>C</mi><mi>j</mi></msub></mrow><mrow><msub><mi>C</mi><mi>i</mi></msub></mrow></mfrac><mo stretchy="false">)</mo><mo>/</mo><mo stretchy="false">(</mo><msub><mi>t</mi><mi>j</mi></msub><mo>−</mo><msub><mi>t</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow></math></div></div><div class="label">(1)</div></div></div><p>Here, <em>C<sub>i</sub></em> and <em>C<sub>j</sub></em> represent the two reads of OD<sub>600</sub> values at two continuous time points of <em>t<sub>j</sub></em> and <em>t<sub>i</sub></em>, which were at intervals of either 0.5 or 1 (h) in the present study. Every four to five continuous growth rates (<em>μ</em>) that exhibited the largest mean and the smallest standard deviation was averaged and taken to be the exponential growth rate of this growth curve (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035_Supp.pdf?Expires=2147483647&Signature=yZ2sO9VP2Z3TYSkUXP~QOTiGs27MRweE6AWrjAkM9HNiC9Gvjlx8POMjF8qk8LzqIpRL-Nvx1OEyGZYpsf72-JrDcHVfi~vBdLUCmHfcPfCwtACeE9Y5b-eu~lNdQ5TNotgue4AFDZPtd2UKgPwc88ZFLFmM-6Fim-Rh6qebLui7ixMu0rZrlaWQ--ByMqKk6u5Btmr0WkLvRP-M3PbG8EJEsFDGSlUtKadVKMtkFO~-uRbgucy-VgNpi3L2sG4g5ynwG7dcp-7r1xkolCwxGosoelpENGXM6PfKgknd8ZmhUlKCiAUcCDUPq6snL6ckRnM0rSqta1nBlMMlcQ2XHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). The mean of the exponential growth rates resulting from 11 to 30 replicated growth curves was determined as the exponential growth rate (<em>r</em>) of the corresponding strain and used for the correlation analyses (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035_Supp.pdf?Expires=2147483647&Signature=yZ2sO9VP2Z3TYSkUXP~QOTiGs27MRweE6AWrjAkM9HNiC9Gvjlx8POMjF8qk8LzqIpRL-Nvx1OEyGZYpsf72-JrDcHVfi~vBdLUCmHfcPfCwtACeE9Y5b-eu~lNdQ5TNotgue4AFDZPtd2UKgPwc88ZFLFmM-6Fim-Rh6qebLui7ixMu0rZrlaWQ--ByMqKk6u5Btmr0WkLvRP-M3PbG8EJEsFDGSlUtKadVKMtkFO~-uRbgucy-VgNpi3L2sG4g5ynwG7dcp-7r1xkolCwxGosoelpENGXM6PfKgknd8ZmhUlKCiAUcCDUPq6snL6ckRnM0rSqta1nBlMMlcQ2XHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3A</a></span>). The population density was calculated directly from the growth curve by averaging five continuous reads at OD<sub>600</sub> that exhibited the largest means (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035_Supp.pdf?Expires=2147483647&Signature=yZ2sO9VP2Z3TYSkUXP~QOTiGs27MRweE6AWrjAkM9HNiC9Gvjlx8POMjF8qk8LzqIpRL-Nvx1OEyGZYpsf72-JrDcHVfi~vBdLUCmHfcPfCwtACeE9Y5b-eu~lNdQ5TNotgue4AFDZPtd2UKgPwc88ZFLFmM-6Fim-Rh6qebLui7ixMu0rZrlaWQ--ByMqKk6u5Btmr0WkLvRP-M3PbG8EJEsFDGSlUtKadVKMtkFO~-uRbgucy-VgNpi3L2sG4g5ynwG7dcp-7r1xkolCwxGosoelpENGXM6PfKgknd8ZmhUlKCiAUcCDUPq6snL6ckRnM0rSqta1nBlMMlcQ2XHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). Similarly, the mean of the population densities that result from 11 to 30 replicated growth curves was determined as the saturated population density (<em>K</em>) of the corresponding strain and used for the correlation analyses (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035_Supp.pdf?Expires=2147483647&Signature=yZ2sO9VP2Z3TYSkUXP~QOTiGs27MRweE6AWrjAkM9HNiC9Gvjlx8POMjF8qk8LzqIpRL-Nvx1OEyGZYpsf72-JrDcHVfi~vBdLUCmHfcPfCwtACeE9Y5b-eu~lNdQ5TNotgue4AFDZPtd2UKgPwc88ZFLFmM-6Fim-Rh6qebLui7ixMu0rZrlaWQ--ByMqKk6u5Btmr0WkLvRP-M3PbG8EJEsFDGSlUtKadVKMtkFO~-uRbgucy-VgNpi3L2sG4g5ynwG7dcp-7r1xkolCwxGosoelpENGXM6PfKgknd8ZmhUlKCiAUcCDUPq6snL6ckRnM0rSqta1nBlMMlcQ2XHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3B</a></span>).</p>                            <h3 scrollto-destination=48004822 id="48004822" class="section-title" data-legacy-id=SEC2.4>2.4 Measurements of cell concentration and size</h3>
<p>The <em>E. coli</em> culture stocks were inoculated in 3 ml of M63 minimal medium at a 100-fold dilution and were cultured in a bioshaker (Taitec) with a rotation rate of 200 rpm at 37 °C. Each cell culture was temporally sampled for size measurement from the late exponential to stationary phases. The cell size (diameter) and the corresponding cell concentration were measured using a cell counter (Multisizer™ 4 coulter counter, Beckman) equipped with a 20-μm aperture. Every 50 μl of cell culture, a 5- to 10-fold dilution, if required, was mixed with 10 ml of COULTER ISOTON II DILUENT (Beckman) in a 25-ml disposable vial (Accuvette ST, Beckman) for measurement. The distribution of cell sizes, including the mean, medium and standard deviation of the distribution, were acquired automatically at each measurement (30,000∼100,000 cells). Repeated experiments (from cell culture to size measurement) were performed for each strain. The average of multiple values of the measured medium was taken to be the representative cell size of the cell population. The stationary cell sizes were determined as the cell size acquired at concentrations &gt;1 × 10<sup>9</sup> cells/ml, individually. Six to eight measurements during stationary phase were obtained for each strain, and the average of these measurements was applied to the analysis.</p>                            <h3 scrollto-destination=48004824 id="48004824" class="section-title" data-legacy-id=SEC2.5>2.5 Evaluation of growth dynamics</h3>
<p>A total of 1,060 growth curves were rescaled to be comparable (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035_Supp.pdf?Expires=2147483647&Signature=yZ2sO9VP2Z3TYSkUXP~QOTiGs27MRweE6AWrjAkM9HNiC9Gvjlx8POMjF8qk8LzqIpRL-Nvx1OEyGZYpsf72-JrDcHVfi~vBdLUCmHfcPfCwtACeE9Y5b-eu~lNdQ5TNotgue4AFDZPtd2UKgPwc88ZFLFmM-6Fim-Rh6qebLui7ixMu0rZrlaWQ--ByMqKk6u5Btmr0WkLvRP-M3PbG8EJEsFDGSlUtKadVKMtkFO~-uRbgucy-VgNpi3L2sG4g5ynwG7dcp-7r1xkolCwxGosoelpENGXM6PfKgknd8ZmhUlKCiAUcCDUPq6snL6ckRnM0rSqta1nBlMMlcQ2XHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). For all growth curves, the intervals between the start of growth over the measurement under-limit and the time required to achieve the maximum OD<sub>600</sub> value were standardized to a value ranging from 0 to 100. The start of the growth phase was defined as the time when the increase of OD<sub>600</sub> was observed in five consecutive reading points. In addition, OD<sub>600</sub> values were also standardized to a value ranging from 0 to 1, by dividing each value on a curve by individual maximum OD<sub>600</sub> value. The rescaled growth rates were calculated by taking differences between consecutive two reading points on rescaled growth curves, and the timing of the maximal growth rate was regarded as the standardized timing of the maximal growth rate. This value indicates when the cells most grew most efficiently, and comparing distributions of the values between strains provides insights into the differences in growth characteristics.</p>                            <h3 scrollto-destination=48004826 id="48004826" class="section-title" data-legacy-id=SEC2.6>2.6 Multiple linear regression analysis</h3>
<p>The data set comprising the growth rate, the length of accumulated deletions, the population size, the angle from <em>ori</em> to <em>dif</em>, and the growth medium was subjected to multiple linear regression analysis. In this data set, only the growth media (<em>M</em>) was a categorical variable (i.e. LB, MAA and M63) and was included in the multiple regression model by dummy coding. Thus, an artificial variable, as a dummy variable (or indicator variable), was created to represent the variation of <em>M</em> (= LB, MAA or M63) and was introduced to the multiple regression analysis. To increase the regression efficiency, regression variable selection was performed on the basis of AIC (Akaike Information Criteria),<a reveal-id="dsw035-B35" data-open="dsw035-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> which evaluated the relative quality of regression models for the data set.<a reveal-id="dsw035-B36" data-open="dsw035-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> Computational analyses were performed with R,<a reveal-id="dsw035-B37" data-open="dsw035-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> and a 3D-plot was created by using the car package.<a reveal-id="dsw035-B38" data-open="dsw035-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a></p>                            <h3 scrollto-destination=48004828 id="48004828" class="section-title" data-legacy-id=SEC2.7>2.7 Organization and graphics of reduced genomes</h3>
<p>Information on the deleted genomic sequences (genomic position and deletion ID) was obtained from the website <a class="link link-uri" href="http://shigen.nig.ac.jp/ecoli/strain/" target="_blank">http://shigen.nig.ac.jp/ecoli/strain/</a>, which distributed the <em>E. coli</em> strains (KHK collection). The accumulated length of deleted genomic sequences was calculated. The numbers of deleted genes and the corresponding gene categories,<a reveal-id="dsw035-B39" data-open="dsw035-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> which is the only gene function classification that covering the complete gene set across the whole <em>E. coli</em> genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035_Supp.pdf?Expires=2147483647&Signature=yZ2sO9VP2Z3TYSkUXP~QOTiGs27MRweE6AWrjAkM9HNiC9Gvjlx8POMjF8qk8LzqIpRL-Nvx1OEyGZYpsf72-JrDcHVfi~vBdLUCmHfcPfCwtACeE9Y5b-eu~lNdQ5TNotgue4AFDZPtd2UKgPwc88ZFLFmM-6Fim-Rh6qebLui7ixMu0rZrlaWQ--ByMqKk6u5Btmr0WkLvRP-M3PbG8EJEsFDGSlUtKadVKMtkFO~-uRbgucy-VgNpi3L2sG4g5ynwG7dcp-7r1xkolCwxGosoelpENGXM6PfKgknd8ZmhUlKCiAUcCDUPq6snL6ckRnM0rSqta1nBlMMlcQ2XHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>, 100% assigned function), were determined (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035_Supp.pdf?Expires=2147483647&Signature=yZ2sO9VP2Z3TYSkUXP~QOTiGs27MRweE6AWrjAkM9HNiC9Gvjlx8POMjF8qk8LzqIpRL-Nvx1OEyGZYpsf72-JrDcHVfi~vBdLUCmHfcPfCwtACeE9Y5b-eu~lNdQ5TNotgue4AFDZPtd2UKgPwc88ZFLFmM-6Fim-Rh6qebLui7ixMu0rZrlaWQ--ByMqKk6u5Btmr0WkLvRP-M3PbG8EJEsFDGSlUtKadVKMtkFO~-uRbgucy-VgNpi3L2sG4g5ynwG7dcp-7r1xkolCwxGosoelpENGXM6PfKgknd8ZmhUlKCiAUcCDUPq6snL6ckRnM0rSqta1nBlMMlcQ2XHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>) and subjected to the correlation analysis, as previously described.<a reveal-id="dsw035-B30" data-open="dsw035-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> Graphical representations of the reduced genomes were created using the BRIG (BLAST Ring Image Generator) tool.<a reveal-id="dsw035-B40" data-open="dsw035-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a></p>                            <h2 scrollto-destination=48004830 id="48004830" class="section-title" data-legacy-id=SEC3>3. Results and discussion</h2>
                            <h3 scrollto-destination=48004831 id="48004831" class="section-title" data-legacy-id=SEC3.1>3.1 Reduced genomes: a stepwise loss of dispensable genomic sequences</h3>
<div class="block-child-p">An assortment of <em>E. coli</em> strains with reduced genomes was used to evaluate the contribution of dispensable genomic sequences to cell growth. A total of 29 <em>E. coli</em> strains, assigned as No. 0 for the wild type genome W3110 and Nos. 1 to 28 for the reduced genomes constructed in a stepwise manner from W3110,<a reveal-id="dsw035-B14" data-open="dsw035-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> were investigated. Note that the reduction of genome size was in a complete cumulative mode, i.e. the strain No. 2 maintained the deletion in No. 1, and No. 3 remained the deletions in No. 2, and so on, as illustrated (<a reveal-id="dsw035-F1" data-open="dsw035-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>A). According to the patterns of multiple deletions that occurred in the reduced genomes, the length of multiple deletions accumulated in each genome was calculated. The results revealed a gradual increase in the length of deleted genomic sequences from ∼48 to 982 kb (<a reveal-id="dsw035-F1" data-open="dsw035-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>B). In addition, the genes located in the deleted regions were clustered according to the gene categories (summarized in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035_Supp.pdf?Expires=2147483647&amp;Signature=yZ2sO9VP2Z3TYSkUXP~QOTiGs27MRweE6AWrjAkM9HNiC9Gvjlx8POMjF8qk8LzqIpRL-Nvx1OEyGZYpsf72-JrDcHVfi~vBdLUCmHfcPfCwtACeE9Y5b-eu~lNdQ5TNotgue4AFDZPtd2UKgPwc88ZFLFmM-6Fim-Rh6qebLui7ixMu0rZrlaWQ--ByMqKk6u5Btmr0WkLvRP-M3PbG8EJEsFDGSlUtKadVKMtkFO~-uRbgucy-VgNpi3L2sG4g5ynwG7dcp-7r1xkolCwxGosoelpENGXM6PfKgknd8ZmhUlKCiAUcCDUPq6snL6ckRnM0rSqta1nBlMMlcQ2XHQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>) and subjected to enrichment analysis to identify the significant loss of gene function in each strain (reduced genomes), as previously reported.<a reveal-id="dsw035-B29" data-open="dsw035-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a><sup>,</sup><a reveal-id="dsw035-B30" data-open="dsw035-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a><sup>,</sup><a reveal-id="dsw035-B31" data-open="dsw035-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> Bimodal tests revealed that the deleted genomic sequences were most significantly enriched in the gene category of phage/IS-related function (h) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035_Supp.pdf?Expires=2147483647&amp;Signature=yZ2sO9VP2Z3TYSkUXP~QOTiGs27MRweE6AWrjAkM9HNiC9Gvjlx8POMjF8qk8LzqIpRL-Nvx1OEyGZYpsf72-JrDcHVfi~vBdLUCmHfcPfCwtACeE9Y5b-eu~lNdQ5TNotgue4AFDZPtd2UKgPwc88ZFLFmM-6Fim-Rh6qebLui7ixMu0rZrlaWQ--ByMqKk6u5Btmr0WkLvRP-M3PbG8EJEsFDGSlUtKadVKMtkFO~-uRbgucy-VgNpi3L2sG4g5ynwG7dcp-7r1xkolCwxGosoelpENGXM6PfKgknd8ZmhUlKCiAUcCDUPq6snL6ckRnM0rSqta1nBlMMlcQ2XHQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>), thus verifying that the deleted sequences were mostly non-essential. Note that all deleted genes are designated as the accessory genes in the present study, to avoid the debated definitions of essential and non-essential. Whether and how the dispensable genomic sequences contribute to cell growth in both poor and rich conditions were subsequently investigated. <div data-id="dsw035-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/m_dsw035f1p.png?Expires=2147483647&amp;Signature=EOCfnLGGfQMYlUWgFPrsmAvsoUBE1X-Hs4uDmk7YUPq8XVBaGEsAjPy~UeoewrajO-r~8kwB7nNpGwU44BK7d-BZEuAii~jsU8~xdv7L2eONu~POkPacn1wAyJmwVsRw0dkoFVdafWzk~RjKfOgU9Ag0C~DfQ7laVlKG0A0eKCET3u1AYMghEJv-g~6ACwGOOLN-p6OhdtRStxcYhITh-vILHtNLbFRCke-KSCCAlADfL21Tz0ztt1NiMf4zwvi8A3Et~SKvLmq2PtvpKB9uU~vYx-cJgtpcDKy0G7w7uKzkVpvOLVJPeZuI3UJSJ2Ydoms3umdeAPL3zfhhub5lvA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Reduced genomes. (A) Genomes of a total 29 E. coli strains. The inner bold lined ring indicates the wild type genome W3110, and black bands on the ring indicate the areas to be deleted. These deletions are also indicated by the deletion ID numbers (D1-D49) at the outside of rings. The truncated grey rings represent the reduced genomes of Nos. 1 to 28 from inside to outside, respectively. The replication initiation and termination sites are indicated by “Ori” and “Dif”. (B) Accumulated length of deleted sequences. The sum of the deleted genomic length in the strains of reduced genomes (Nos. 1-28) was calculated. " path-from-xml="dsw035f1p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48004832/dsw035f1p.tif" path-from-xml="dsw035f1p.tif" target="_blank">View large</a><a section="48004832" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035f1p.png?Expires=2147483647&amp;Signature=q1IuZ34OgLCNCX3k9Kvw0rcJi6~mjKMb7hhFWYsGTcXBIgUB1srdQdFK2Aid4EZGeTB-853lQC9POn5AtYos6xiC-pSFQIbpcA~9uaw9QFIu5DLbB6Y2U~Jal4K3DhfihAsm0vHnDpNCTWcpGTr~j5CDSyH4RmM0zi2~FJwd~9nOJ-JYvZkuTWPTjtN9PgID6k24-~tDVC1YAeN25mpK7mWEc7180YnjeYibpRFChuK5Ae9eInEZILatfu4R~3KUEW-GghVlnlQOzT305Mh6df9nh76kMI1UAsKC8H~8WNTpB9KdTYVxsw7P3bh7~PGRrI-tkjt~DSRxJ29HqcZyCA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48004832&amp;ar=2647442&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw035f1p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Reduced genomes. (A) Genomes of a total 29 <em>E. coli</em> strains. The inner bold lined ring indicates the wild type genome W3110, and black bands on the ring indicate the areas to be deleted. These deletions are also indicated by the deletion ID numbers (D1-D49) at the outside of rings. The truncated grey rings represent the reduced genomes of Nos. 1 to 28 from inside to outside, respectively. The replication initiation and termination sites are indicated by “Ori” and “Dif”. (B) Accumulated length of deleted sequences. The sum of the deleted genomic length in the strains of reduced genomes (Nos. 1-28) was calculated. </p></div></div><div content-id="dsw035-F1" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/m_dsw035f1p.png?Expires=2147483647&amp;Signature=EOCfnLGGfQMYlUWgFPrsmAvsoUBE1X-Hs4uDmk7YUPq8XVBaGEsAjPy~UeoewrajO-r~8kwB7nNpGwU44BK7d-BZEuAii~jsU8~xdv7L2eONu~POkPacn1wAyJmwVsRw0dkoFVdafWzk~RjKfOgU9Ag0C~DfQ7laVlKG0A0eKCET3u1AYMghEJv-g~6ACwGOOLN-p6OhdtRStxcYhITh-vILHtNLbFRCke-KSCCAlADfL21Tz0ztt1NiMf4zwvi8A3Et~SKvLmq2PtvpKB9uU~vYx-cJgtpcDKy0G7w7uKzkVpvOLVJPeZuI3UJSJ2Ydoms3umdeAPL3zfhhub5lvA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Reduced genomes. (A) Genomes of a total 29 E. coli strains. The inner bold lined ring indicates the wild type genome W3110, and black bands on the ring indicate the areas to be deleted. These deletions are also indicated by the deletion ID numbers (D1-D49) at the outside of rings. The truncated grey rings represent the reduced genomes of Nos. 1 to 28 from inside to outside, respectively. The replication initiation and termination sites are indicated by “Ori” and “Dif”. (B) Accumulated length of deleted sequences. The sum of the deleted genomic length in the strains of reduced genomes (Nos. 1-28) was calculated. " path-from-xml="dsw035f1p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48004832/dsw035f1p.tif" path-from-xml="dsw035f1p.tif" target="_blank">View large</a><a section="48004832" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035f1p.png?Expires=2147483647&amp;Signature=q1IuZ34OgLCNCX3k9Kvw0rcJi6~mjKMb7hhFWYsGTcXBIgUB1srdQdFK2Aid4EZGeTB-853lQC9POn5AtYos6xiC-pSFQIbpcA~9uaw9QFIu5DLbB6Y2U~Jal4K3DhfihAsm0vHnDpNCTWcpGTr~j5CDSyH4RmM0zi2~FJwd~9nOJ-JYvZkuTWPTjtN9PgID6k24-~tDVC1YAeN25mpK7mWEc7180YnjeYibpRFChuK5Ae9eInEZILatfu4R~3KUEW-GghVlnlQOzT305Mh6df9nh76kMI1UAsKC8H~8WNTpB9KdTYVxsw7P3bh7~PGRrI-tkjt~DSRxJ29HqcZyCA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48004832&amp;ar=2647442&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw035f1p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Reduced genomes. (A) Genomes of a total 29 <em>E. coli</em> strains. The inner bold lined ring indicates the wild type genome W3110, and black bands on the ring indicate the areas to be deleted. These deletions are also indicated by the deletion ID numbers (D1-D49) at the outside of rings. The truncated grey rings represent the reduced genomes of Nos. 1 to 28 from inside to outside, respectively. The replication initiation and termination sites are indicated by “Ori” and “Dif”. (B) Accumulated length of deleted sequences. The sum of the deleted genomic length in the strains of reduced genomes (Nos. 1-28) was calculated. </p></div></div></div>                            <h3 scrollto-destination=48004833 id="48004833" class="section-title" data-legacy-id=SEC3.2>3.2 Accumulated deletions caused correlated changes in population growth</h3>
<div class="block-child-p">A negative correlation between population growth and the accumulated deletions was the most salient finding in the present study. Repeated measurements of the <em>E. coli</em> strains that were grown under three different nutritional conditions, resulted in 1,060 independent growth curves. The growth rates during exponential phase and the saturated cell densities during stationary phase were manually calculated for each growth curve (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035_Supp.pdf?Expires=2147483647&amp;Signature=yZ2sO9VP2Z3TYSkUXP~QOTiGs27MRweE6AWrjAkM9HNiC9Gvjlx8POMjF8qk8LzqIpRL-Nvx1OEyGZYpsf72-JrDcHVfi~vBdLUCmHfcPfCwtACeE9Y5b-eu~lNdQ5TNotgue4AFDZPtd2UKgPwc88ZFLFmM-6Fim-Rh6qebLui7ixMu0rZrlaWQ--ByMqKk6u5Btmr0WkLvRP-M3PbG8EJEsFDGSlUtKadVKMtkFO~-uRbgucy-VgNpi3L2sG4g5ynwG7dcp-7r1xkolCwxGosoelpENGXM6PfKgknd8ZmhUlKCiAUcCDUPq6snL6ckRnM0rSqta1nBlMMlcQ2XHQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). A data set of the exponential growth rates and saturated cell densities associated with reduced genomes was acquired (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035_Supp.pdf?Expires=2147483647&amp;Signature=yZ2sO9VP2Z3TYSkUXP~QOTiGs27MRweE6AWrjAkM9HNiC9Gvjlx8POMjF8qk8LzqIpRL-Nvx1OEyGZYpsf72-JrDcHVfi~vBdLUCmHfcPfCwtACeE9Y5b-eu~lNdQ5TNotgue4AFDZPtd2UKgPwc88ZFLFmM-6Fim-Rh6qebLui7ixMu0rZrlaWQ--ByMqKk6u5Btmr0WkLvRP-M3PbG8EJEsFDGSlUtKadVKMtkFO~-uRbgucy-VgNpi3L2sG4g5ynwG7dcp-7r1xkolCwxGosoelpENGXM6PfKgknd8ZmhUlKCiAUcCDUPq6snL6ckRnM0rSqta1nBlMMlcQ2XHQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). Statistical tests indicated a highly significant correlation (<em>P </em>&lt; 0.001) between the exponential growth rate and the length of cumulatively deleted genomic sequences in all three nutritional conditions (<a reveal-id="dsw035-F2" data-open="dsw035-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>A). Genome reduction-induced growth decreases were clearly detected. In addition, the mode of growth decrease was nutritional dependent. In comparison to a continuous gradual decrease in the poor condition (M63), a stepwise decrease in the rich condition (LB) was detected, close to the finding reported recently.<a reveal-id="dsw035-B26" data-open="dsw035-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> The variation in mode of growth decrease reflected the degree of requirement of dispensable genomic sequences for efficient growth. <div data-id="dsw035-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/m_dsw035f2p.png?Expires=2147483647&amp;Signature=VvxzUE4YWI89ixCiIgMD2lo2hYDrtK9PTpFx9QpGa9m9IMjslqsuFconUx~Nn~qt7Ubb6j4lCgVL0eN8GwDh5eo6I7kf4uDqawLoPHkayTB-FcNr146hGNENTeS8BDLGPtnbkkfbbc4o~J2UXNmd5OiVhb33kmAWWbeaBQV1zpU29o6jsznJWjb1GdrZqYlT7I1BX-exbXHJ7yycZa8lKttAPP2PcFuZVYxJDlN8PZDK9sgiZFN-NJFXDeVoras5Z4pSRRWdMXtMzZlw-ZVg9BnEa6mAYVwz~2yB60RjzjtGsyTeMnOTpvl0AHlZ5fTcMY1WWwr2Pg-ouaiXfm5Isw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Accumulated deletion-dependent changes in population growth. (A) Correlated changes in growth rate. The calculated exponential growth rates are plotted against the length of accumulated deletions. (B) Correlated changes in saturated cell density. The calculated maximal OD600 values are plotted against the length of accumulated deletions. Standard errors (N = 11∼30), growth media, correlation coefficients and P values are indicated. M63, MAA and LB indicate the growth media. (C) Nutritional dependent changes in population growth. The left and right panels represent the normalized decease in growth rate and the normalized decease in saturated cell density, respectively. Red, green and blue represent the growth in LB, MAA and M63, respectively." path-from-xml="dsw035f2p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48004834/dsw035f2p.tif" path-from-xml="dsw035f2p.tif" target="_blank">View large</a><a section="48004834" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035f2p.png?Expires=2147483647&amp;Signature=J7ce7oXzHAmN1TcyZIPuKNPkiGwx~tg1isE8s23RIkdH9bihVvJ3eH-0lTUofUHKGVT6ZIzeUZGX~oxCFh1V2KXUJmDBAqeRHKmNO~w9uU-IPS1y8mb72AZASHQCNh6wESFrPS-rYKic2kEyWdOabC2Qodn5rXWSAGZ4cjQ8xzrJoMO1PwPMlZm0VG16VeVXK52ZTGodQnk2p0iQEnLVX-3dkscLSihf1KJN5ssho1l384BRts0Zo23BtWWuHDUyzIA146UCdealmgfDqmlYAFXef1nhDIH4aCSCr3kiOhYOjPyeFVNEv~VUwqX-EQdztk~-Hdf71VfmxT0E~VpXvQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48004834&amp;ar=2647442&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw035f2p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Accumulated deletion-dependent changes in population growth. (A) Correlated changes in growth rate. The calculated exponential growth rates are plotted against the length of accumulated deletions. (B) Correlated changes in saturated cell density. The calculated maximal OD<sub>600</sub> values are plotted against the length of accumulated deletions. Standard errors (<em>N</em> = 11∼30), growth media, correlation coefficients and <em>P</em> values are indicated. M63, MAA and LB indicate the growth media. (C) Nutritional dependent changes in population growth. The left and right panels represent the normalized decease in growth rate and the normalized decease in saturated cell density, respectively. Red, green and blue represent the growth in LB, MAA and M63, respectively.</p></div></div><div content-id="dsw035-F2" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/m_dsw035f2p.png?Expires=2147483647&amp;Signature=VvxzUE4YWI89ixCiIgMD2lo2hYDrtK9PTpFx9QpGa9m9IMjslqsuFconUx~Nn~qt7Ubb6j4lCgVL0eN8GwDh5eo6I7kf4uDqawLoPHkayTB-FcNr146hGNENTeS8BDLGPtnbkkfbbc4o~J2UXNmd5OiVhb33kmAWWbeaBQV1zpU29o6jsznJWjb1GdrZqYlT7I1BX-exbXHJ7yycZa8lKttAPP2PcFuZVYxJDlN8PZDK9sgiZFN-NJFXDeVoras5Z4pSRRWdMXtMzZlw-ZVg9BnEa6mAYVwz~2yB60RjzjtGsyTeMnOTpvl0AHlZ5fTcMY1WWwr2Pg-ouaiXfm5Isw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Accumulated deletion-dependent changes in population growth. (A) Correlated changes in growth rate. The calculated exponential growth rates are plotted against the length of accumulated deletions. (B) Correlated changes in saturated cell density. The calculated maximal OD600 values are plotted against the length of accumulated deletions. Standard errors (N = 11∼30), growth media, correlation coefficients and P values are indicated. M63, MAA and LB indicate the growth media. (C) Nutritional dependent changes in population growth. The left and right panels represent the normalized decease in growth rate and the normalized decease in saturated cell density, respectively. Red, green and blue represent the growth in LB, MAA and M63, respectively." path-from-xml="dsw035f2p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48004834/dsw035f2p.tif" path-from-xml="dsw035f2p.tif" target="_blank">View large</a><a section="48004834" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035f2p.png?Expires=2147483647&amp;Signature=J7ce7oXzHAmN1TcyZIPuKNPkiGwx~tg1isE8s23RIkdH9bihVvJ3eH-0lTUofUHKGVT6ZIzeUZGX~oxCFh1V2KXUJmDBAqeRHKmNO~w9uU-IPS1y8mb72AZASHQCNh6wESFrPS-rYKic2kEyWdOabC2Qodn5rXWSAGZ4cjQ8xzrJoMO1PwPMlZm0VG16VeVXK52ZTGodQnk2p0iQEnLVX-3dkscLSihf1KJN5ssho1l384BRts0Zo23BtWWuHDUyzIA146UCdealmgfDqmlYAFXef1nhDIH4aCSCr3kiOhYOjPyeFVNEv~VUwqX-EQdztk~-Hdf71VfmxT0E~VpXvQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48004834&amp;ar=2647442&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw035f2p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Accumulated deletion-dependent changes in population growth. (A) Correlated changes in growth rate. The calculated exponential growth rates are plotted against the length of accumulated deletions. (B) Correlated changes in saturated cell density. The calculated maximal OD<sub>600</sub> values are plotted against the length of accumulated deletions. Standard errors (<em>N</em> = 11∼30), growth media, correlation coefficients and <em>P</em> values are indicated. M63, MAA and LB indicate the growth media. (C) Nutritional dependent changes in population growth. The left and right panels represent the normalized decease in growth rate and the normalized decease in saturated cell density, respectively. Red, green and blue represent the growth in LB, MAA and M63, respectively.</p></div></div></div><div class="block-child-p">A correlation was also observed between the saturated cell density and accumulated deletions (<em>P </em>&lt; 0.05); namely, an increase in deleted length was correlated with a decrease in cell density (<a reveal-id="dsw035-F2" data-open="dsw035-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>B). The reduced cell density appeared to be attributed to the reduced number of cells; however, the loss of dispensable genomic sequences influenced the cell size at the saturated concentration (<a reveal-id="dsw035-F3" data-open="dsw035-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). Because the cell density was evaluated through OD measurements, the number or the size of the cells could contribute to the changes in turbidity. Measurement of the saturated culture in M63, in which the most significant decrease in turbidity was observed, revealed that the cumulative deletions triggered the decrease in cell concentration (<a reveal-id="dsw035-F3" data-open="dsw035-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>B) and the increase in cell size (<a reveal-id="dsw035-F3" data-open="dsw035-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>C) at the saturated phase. Nevertheless, a positive correlation between the cell density evaluated by optical measurement and the cell concentration according to the numbers (particles) of cells (<a reveal-id="dsw035-F3" data-open="dsw035-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>A) confirmed the association “between accumulative genomic deletion and population decrease. <div data-id="dsw035-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/m_dsw035f3p.png?Expires=2147483647&Signature=OW0vr7rECBJHYl6jfazg-c9W6jpGcD7oASqdTUByFzH4-obCey2uidu-2pvmOJLTVinIQvh1q3cPH9-PY-djtOYpJkdXlbN3JU54dQA4Z1Pr26aE0r09bWyx4yezVVTXUVfQo-uLlnPP7Ns7wCR9FkHKWzhrKU0RJ9Y2lENpIqsWnB~-2QUKS1qaBkR9qli68Sy7WT4ui-u6sNa3PjOR7QxE8nXQYXHj5jRRK33-jMGtyBFIXN~PDK9ysKrKThZSckrgQtCkugxKS4kMpreN7kGu4mbhdlVeZ3NHwA3g1n5u9xPHGvFfx1DRr5eFuwb0sqB9lTB2M-jg71-FH~wvEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Changes in cell concentration and cell size. (A) Correlation between cell concentration and cell density. The saturated cell density calculated from optical measurement is plotted against the saturated cell concentration as measured by cell counter. (B) Correlations with changes in cell concentration. The saturated cell concentration is plotted against the length of accumulated deletions. (C) Correlations with changes in cell size. The mean cell size of the saturated population is plotted against the length of accumulated deletions. The cell size is represented by the diameter of the cell particle. Standard errors, correlation coefficients and statistical significance are indicated." path-from-xml="dsw035f3p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48004835/dsw035f3p.tif" path-from-xml="dsw035f3p.tif" target="_blank">View large</a><a section="48004835" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035f3p.png?Expires=2147483647&Signature=g0kLd0Zen8HDKUA97ZTHjmcM36x3sT4F6oXC1WfDrY0B5E0WVOVTLP7TMKe2NeTAL3Rs1x~ewPhW8~vKhn6wVuu4CD66PcHAJTMkdjB3unXBbhdY1A7ss2wgQ-1WY19acbF-BDT9IZE4rCOQoqJqjvOjTHkgZm2DsKDcrOE~hmaj~TyjEO7hJAhRoTV~QStnyE1xDe-aqugQtWqUaF8pg3xmAjwZzaC3sgVW4BXH2sPJu8jzXB1yB45skRELDhiuqzhqU32yJByXNktzyy0JgP29MI3xJaGZre8rlFB5eW8PDW3UAVC-asuP1uF~H9LhPlVxGulq-vpPHSDZIDL1lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=48004835&ar=2647442&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw035f3p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Changes in cell concentration and cell size. (A) Correlation between cell concentration and cell density. The saturated cell density calculated from optical measurement is plotted against the saturated cell concentration as measured by cell counter. (B) Correlations with changes in cell concentration. The saturated cell concentration is plotted against the length of accumulated deletions. (C) Correlations with changes in cell size. The mean cell size of the saturated population is plotted against the length of accumulated deletions. The cell size is represented by the diameter of the cell particle. Standard errors, correlation coefficients and statistical significance are indicated.</p></div></div><div content-id="dsw035-F3" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/m_dsw035f3p.png?Expires=2147483647&Signature=OW0vr7rECBJHYl6jfazg-c9W6jpGcD7oASqdTUByFzH4-obCey2uidu-2pvmOJLTVinIQvh1q3cPH9-PY-djtOYpJkdXlbN3JU54dQA4Z1Pr26aE0r09bWyx4yezVVTXUVfQo-uLlnPP7Ns7wCR9FkHKWzhrKU0RJ9Y2lENpIqsWnB~-2QUKS1qaBkR9qli68Sy7WT4ui-u6sNa3PjOR7QxE8nXQYXHj5jRRK33-jMGtyBFIXN~PDK9ysKrKThZSckrgQtCkugxKS4kMpreN7kGu4mbhdlVeZ3NHwA3g1n5u9xPHGvFfx1DRr5eFuwb0sqB9lTB2M-jg71-FH~wvEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Changes in cell concentration and cell size. (A) Correlation between cell concentration and cell density. The saturated cell density calculated from optical measurement is plotted against the saturated cell concentration as measured by cell counter. (B) Correlations with changes in cell concentration. The saturated cell concentration is plotted against the length of accumulated deletions. (C) Correlations with changes in cell size. The mean cell size of the saturated population is plotted against the length of accumulated deletions. The cell size is represented by the diameter of the cell particle. Standard errors, correlation coefficients and statistical significance are indicated." path-from-xml="dsw035f3p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48004835/dsw035f3p.tif" path-from-xml="dsw035f3p.tif" target="_blank">View large</a><a section="48004835" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035f3p.png?Expires=2147483647&Signature=g0kLd0Zen8HDKUA97ZTHjmcM36x3sT4F6oXC1WfDrY0B5E0WVOVTLP7TMKe2NeTAL3Rs1x~ewPhW8~vKhn6wVuu4CD66PcHAJTMkdjB3unXBbhdY1A7ss2wgQ-1WY19acbF-BDT9IZE4rCOQoqJqjvOjTHkgZm2DsKDcrOE~hmaj~TyjEO7hJAhRoTV~QStnyE1xDe-aqugQtWqUaF8pg3xmAjwZzaC3sgVW4BXH2sPJu8jzXB1yB45skRELDhiuqzhqU32yJByXNktzyy0JgP29MI3xJaGZre8rlFB5eW8PDW3UAVC-asuP1uF~H9LhPlVxGulq-vpPHSDZIDL1lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=48004835&ar=2647442&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw035f3p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Changes in cell concentration and cell size. (A) Correlation between cell concentration and cell density. The saturated cell density calculated from optical measurement is plotted against the saturated cell concentration as measured by cell counter. (B) Correlations with changes in cell concentration. The saturated cell concentration is plotted against the length of accumulated deletions. (C) Correlations with changes in cell size. The mean cell size of the saturated population is plotted against the length of accumulated deletions. The cell size is represented by the diameter of the cell particle. Standard errors, correlation coefficients and statistical significance are indicated.</p></div></div></div><p>Together, our findings indicated that longer accumulative lengths of deleted dispensable genomic sequences not only led to the decrease in growth fitness during exponential phase but also decreased the final population size during stationary phase. Although the dispensable genomic sequences were non-essential for life, the length-dependent changes illustrated the slight but significant contribution of the accumulated loss of these sequences to bacterial growth.</p>                            <h3 scrollto-destination=48004837 id="48004837" class="section-title" data-legacy-id=SEC3.3>3.3 The degree of change in population growth was nutritional dependent</h3>
<div class="block-child-p">A quantitative association between the accumulated length of multiple deletions and bacterial growth was constructed. Normalization of both the exponential growth rate and the saturated cell density indicated gradual changes in the magnitude of the decrease (<a reveal-id="dsw035-F2" data-open="dsw035-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>C). Here, the exponential growth rate and the saturated cell density of the wild type genome (Strain No. 0) were considered as one unit. Linear regression was simply applied to fit the following equations (<a href="#E2" class="sectionLink xref-sec">Equations 2</a> and <a href="#E4" class="sectionLink xref-sec">3</a>). <a id="E2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E2"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>r</mi><mo>=</mo><mn>1</mn><mo>−</mo><mi>α</mi><msub><mi>L</mi><mi>d</mi></msub></mrow></math></div></div><div class="label">(2)</div></div><a id="E3" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E3"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>K</mi><mo>=</mo><mn>1</mn><mo>−</mo><mi>β</mi><msub><mi>L</mi><mi>d</mi></msub></mrow></math></div></div><div class="label">(3)</div></div></div><p>Here, <em>L<sub>d</sub></em>, <em>r</em> and <em>K</em> represent the length of the accumulated deletions (kb), the normalized growth rate and the cell density, respectively. The constants α and β indicate the magnitudes of the decrease in population growth in relation to the length of multiple deletions. In three different media, the slopes of α<sub>M63</sub>, α<sub>MAA</sub> and α<sub>LB</sub> were 5e-4, 1e-4 and 9e-5 per kb, respectively (<a reveal-id="dsw035-F2" data-open="dsw035-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>C, left). The slopes of β<sub>M63</sub>, β<sub>MAA</sub> and β<sub>LB</sub> were 2e-4, 1e-4 and 7e-5 per kb, respectively (<a reveal-id="dsw035-F2" data-open="dsw035-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>C, right). Thus, every 1-kb deletion would cause a 0.05%, 0.01%, and 0.009% fitness decrease and a 0.02%, 0.01%, and 0.007% population reduction in the conditions of M63, MAA, and LB, respectively. Both α and β exhibited gradual changes in the order of the richness of growth media, thus indicating that the extent of the length-dependent changes in population growth was nutritional dependent. The deleted genomic sequences affected both the growth fitness (<em>r</em>) and the environmental capacity (<em>K</em>), and this effect was more substantial in poor environments.</p><div class="block-child-p">Combining the two equations (<a href="#E2" class="sectionLink xref-sec">Equations 2</a> and <a href="#E3" class="sectionLink xref-sec">3</a>) resulted in the following equation (<a href="#E4" class="sectionLink xref-sec">Equation 4</a>), and further transformation made the coordinated relationship between <em>r</em> and <em>K</em> clearer (<a href="#E5" class="sectionLink xref-sec">Equation 5</a>). <a id="E4" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E4"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>K</mi><mo>=</mo><mn>1</mn><mo>−</mo><mfrac><mi>β</mi><mi>α</mi></mfrac><mo stretchy="false">(</mo><mn>1</mn><mo>−</mo><mi>r</mi><mo stretchy="false">)</mo></mrow></math></div></div><div class="label">(4)</div></div><a id="E5" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E5"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>K</mi><mo>=</mo><msub><mi>C</mi><mi>M</mi></msub><mo>+</mo><msub><mi>k</mi><mi>M</mi></msub><mi>r</mi></mrow></math></div></div><div class="label">(5)</div></div></div><p>Here, <em>C<sub>M</sub></em> and <em>k<sub>M</sub></em> represent the medium-dependent population capacity and correlation, respectively. Although a positive correlation between population size and growth fitness could be estimated (<em>k<sub>M </sub></em>&gt; 0), significant correlations were identified in the conditions of M63 and LB (<em>P </em>&lt; 0.01) but not MAA (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035_Supp.pdf?Expires=2147483647&Signature=yZ2sO9VP2Z3TYSkUXP~QOTiGs27MRweE6AWrjAkM9HNiC9Gvjlx8POMjF8qk8LzqIpRL-Nvx1OEyGZYpsf72-JrDcHVfi~vBdLUCmHfcPfCwtACeE9Y5b-eu~lNdQ5TNotgue4AFDZPtd2UKgPwc88ZFLFmM-6Fim-Rh6qebLui7ixMu0rZrlaWQ--ByMqKk6u5Btmr0WkLvRP-M3PbG8EJEsFDGSlUtKadVKMtkFO~-uRbgucy-VgNpi3L2sG4g5ynwG7dcp-7r1xkolCwxGosoelpENGXM6PfKgknd8ZmhUlKCiAUcCDUPq6snL6ckRnM0rSqta1nBlMMlcQ2XHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). It might because of the relatively low significance in the correlation between the saturated cell density and genome reduction in MAA (<a reveal-id="dsw035-F2" data-open="dsw035-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>B). The results suggest that the nutritionally differentiated bias in the relationships among growth parameters is likely to be related to genomic deletions.</p>                            <h3 scrollto-destination=48004842 id="48004842" class="section-title" data-legacy-id=SEC3.4>3.4 Accumulated deletions also caused correlated changes in growth dynamics</h3>
<div class="block-child-p">In addition to the speed (<em>r</em>) and the maximum (<em>K</em>) of population growth, the patterns of growth dynamics were also observed in correlated changes (<a reveal-id="dsw035-F4" data-open="dsw035-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>) evaluated by rescaling the growth curves (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035_Supp.pdf?Expires=2147483647&amp;Signature=yZ2sO9VP2Z3TYSkUXP~QOTiGs27MRweE6AWrjAkM9HNiC9Gvjlx8POMjF8qk8LzqIpRL-Nvx1OEyGZYpsf72-JrDcHVfi~vBdLUCmHfcPfCwtACeE9Y5b-eu~lNdQ5TNotgue4AFDZPtd2UKgPwc88ZFLFmM-6Fim-Rh6qebLui7ixMu0rZrlaWQ--ByMqKk6u5Btmr0WkLvRP-M3PbG8EJEsFDGSlUtKadVKMtkFO~-uRbgucy-VgNpi3L2sG4g5ynwG7dcp-7r1xkolCwxGosoelpENGXM6PfKgknd8ZmhUlKCiAUcCDUPq6snL6ckRnM0rSqta1nBlMMlcQ2XHQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). The saturated cell density was rescaled in a common unit such that it had an increased effect on the low <em>K</em> and <em>r</em> in the reduced genomes. If the growth dynamics was similar, the rescaling would lead to the overlapped growth curves of all strains despite the variation in the raw growth curves (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035_Supp.pdf?Expires=2147483647&amp;Signature=yZ2sO9VP2Z3TYSkUXP~QOTiGs27MRweE6AWrjAkM9HNiC9Gvjlx8POMjF8qk8LzqIpRL-Nvx1OEyGZYpsf72-JrDcHVfi~vBdLUCmHfcPfCwtACeE9Y5b-eu~lNdQ5TNotgue4AFDZPtd2UKgPwc88ZFLFmM-6Fim-Rh6qebLui7ixMu0rZrlaWQ--ByMqKk6u5Btmr0WkLvRP-M3PbG8EJEsFDGSlUtKadVKMtkFO~-uRbgucy-VgNpi3L2sG4g5ynwG7dcp-7r1xkolCwxGosoelpENGXM6PfKgknd8ZmhUlKCiAUcCDUPq6snL6ckRnM0rSqta1nBlMMlcQ2XHQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). That is, the time required to achieve the maximal growth rate (<em>τ</em>), which was considered a representative parameter of growth dynamics, should be equivalent. However, the analysis resulted in high variation in the timing and the negative correlations between the averaged timing (<em>τ</em>) and the deleted length (<em>L<sub>d</sub></em>) in all three media (<em>P </em>&lt; 0.005) (<a reveal-id="dsw035-F4" data-open="dsw035-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). The accumulative deletion caused the changes in growth patterns; i.e. the time from the growth initiation to the highest growth efficiency was increased, and the time from the maximal growth rate to the population saturation was decreased. Because growth is thought to be controlled by metabolic reactions, the global biochemical reactions achieved the highest efficacy more slowly and shut down more rapidly. These findings implied that the deleted sequences participated in metabolic efficiency in a dose-dependent manner. Hence, the qualities of life processes (e.g. growth fitness, metabolic activity) are associated with the quantity of genetic material (i.e. genomic length). <div data-id="dsw035-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/m_dsw035f4p.png?Expires=2147483647&amp;Signature=Gl~a196mbFyS2BELtFgjuU1ds~e0rv3369xD80yeBAmQj0aQEuGhc7P7LaGDOZ2rn9Qawbd1n8sBJeif600tpp5jkDzYe711mBfP3x5TIF0COCcCe-hFQw9HqQHl0AVkkQzZes-lihcVo~gUnhRp3CpChLkJEujrZBde1rawebs~~5qteBB1nxNro5CeVaZwKwLgsUWSR3~yFYsYCbbfJLY9Ibn5uwlXHrzeLPqTeArqib0vohfJT-xA8rJrQYLdLkIRGMrS0NwcX038Stga1QT0TIJVeGpRAiEvmLql9JvS6lFKb-jbHOkjDpiFyTt5xCQ7k2RKaMLeuHXckQe7xw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Rescaled timing of the maximal growth rate. The time required for the cell populations to achieve the best growth performance was calculated following the rescaling. Box plots represent every 11 to 30 repeated measurements for each condition. Panels from top to bottom represent growth in LB, MAA and M63, respectively. The genomes (Strain No.) from Nos. 0 to 28 are indicated in order from the top to bottom in each panel. Spearman’s rank correlation coefficients between the mean timing and the length of accumulated deletions are indicated along with the statistical significance." path-from-xml="dsw035f4p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48004843/dsw035f4p.tif" path-from-xml="dsw035f4p.tif" target="_blank">View large</a><a section="48004843" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035f4p.png?Expires=2147483647&amp;Signature=vB49tFCvvt8MNnMx082I4n7OsMK7QSrI3Vs3c4VF5BpAmfIfe5-k6i8YSgTUoEmN5uSWkii5sB4RV4cRl0OpW92uI6F9fvgwoQGhiAR3SkwuEve~Ll4N9wdKWx5B0ChElLb4sIolrSKkiiLUugfzvAbnbxf6X6jb98jaDUCTR7wqlfVdu3ZwHBdwbh1bsYGZC-Zifw7u9aHoJm5ZFj-MSQcvNaN7mW2OOklxEVOj0cG-7W6BdQljs48oTuNJg7QZBMjYUCt97KMkMeh1~FciLxs8fSOHqE7iPAQu6zB8o8MbOTmNaZhpsk1cufE0FKukVx1enlCh9RDQSrkOtWreIA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48004843&amp;ar=2647442&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw035f4p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Rescaled timing of the maximal growth rate. The time required for the cell populations to achieve the best growth performance was calculated following the rescaling. Box plots represent every 11 to 30 repeated measurements for each condition. Panels from top to bottom represent growth in LB, MAA and M63, respectively. The genomes (Strain No.) from Nos. 0 to 28 are indicated in order from the top to bottom in each panel. Spearman’s rank correlation coefficients between the mean timing and the length of accumulated deletions are indicated along with the statistical significance.</p></div></div><div content-id="dsw035-F4" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/m_dsw035f4p.png?Expires=2147483647&amp;Signature=Gl~a196mbFyS2BELtFgjuU1ds~e0rv3369xD80yeBAmQj0aQEuGhc7P7LaGDOZ2rn9Qawbd1n8sBJeif600tpp5jkDzYe711mBfP3x5TIF0COCcCe-hFQw9HqQHl0AVkkQzZes-lihcVo~gUnhRp3CpChLkJEujrZBde1rawebs~~5qteBB1nxNro5CeVaZwKwLgsUWSR3~yFYsYCbbfJLY9Ibn5uwlXHrzeLPqTeArqib0vohfJT-xA8rJrQYLdLkIRGMrS0NwcX038Stga1QT0TIJVeGpRAiEvmLql9JvS6lFKb-jbHOkjDpiFyTt5xCQ7k2RKaMLeuHXckQe7xw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Rescaled timing of the maximal growth rate. The time required for the cell populations to achieve the best growth performance was calculated following the rescaling. Box plots represent every 11 to 30 repeated measurements for each condition. Panels from top to bottom represent growth in LB, MAA and M63, respectively. The genomes (Strain No.) from Nos. 0 to 28 are indicated in order from the top to bottom in each panel. Spearman’s rank correlation coefficients between the mean timing and the length of accumulated deletions are indicated along with the statistical significance." path-from-xml="dsw035f4p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48004843/dsw035f4p.tif" path-from-xml="dsw035f4p.tif" target="_blank">View large</a><a section="48004843" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035f4p.png?Expires=2147483647&amp;Signature=vB49tFCvvt8MNnMx082I4n7OsMK7QSrI3Vs3c4VF5BpAmfIfe5-k6i8YSgTUoEmN5uSWkii5sB4RV4cRl0OpW92uI6F9fvgwoQGhiAR3SkwuEve~Ll4N9wdKWx5B0ChElLb4sIolrSKkiiLUugfzvAbnbxf6X6jb98jaDUCTR7wqlfVdu3ZwHBdwbh1bsYGZC-Zifw7u9aHoJm5ZFj-MSQcvNaN7mW2OOklxEVOj0cG-7W6BdQljs48oTuNJg7QZBMjYUCt97KMkMeh1~FciLxs8fSOHqE7iPAQu6zB8o8MbOTmNaZhpsk1cufE0FKukVx1enlCh9RDQSrkOtWreIA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48004843&amp;ar=2647442&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw035f4p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Rescaled timing of the maximal growth rate. The time required for the cell populations to achieve the best growth performance was calculated following the rescaling. Box plots represent every 11 to 30 repeated measurements for each condition. Panels from top to bottom represent growth in LB, MAA and M63, respectively. The genomes (Strain No.) from Nos. 0 to 28 are indicated in order from the top to bottom in each panel. Spearman’s rank correlation coefficients between the mean timing and the length of accumulated deletions are indicated along with the statistical significance.</p></div></div></div>                            <h3 scrollto-destination=48004844 id="48004844" class="section-title" data-legacy-id=SEC3.5>3.5 Growth prediction verified that genome reduction is the main factor underlying decreased growth</h3>
<div class="block-child-p">To determine the main factors involved in the growth decrease, growth prediction by multiple regression analysis was performed, in which five parameters were used:, the length of accumulated deletions (<em>L<sub>d</sub></em>), the population size (<em>K</em>), the culture media (<em>M</em>), the timing of growth maximal (<em>τ</em>) and the <em>ori-dif</em> angle (<em>θ</em>), a newly introduced parameter representing the symmetry in genome replication from <em>ori</em> to <em>dif</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035_Supp.pdf?Expires=2147483647&amp;Signature=yZ2sO9VP2Z3TYSkUXP~QOTiGs27MRweE6AWrjAkM9HNiC9Gvjlx8POMjF8qk8LzqIpRL-Nvx1OEyGZYpsf72-JrDcHVfi~vBdLUCmHfcPfCwtACeE9Y5b-eu~lNdQ5TNotgue4AFDZPtd2UKgPwc88ZFLFmM-6Fim-Rh6qebLui7ixMu0rZrlaWQ--ByMqKk6u5Btmr0WkLvRP-M3PbG8EJEsFDGSlUtKadVKMtkFO~-uRbgucy-VgNpi3L2sG4g5ynwG7dcp-7r1xkolCwxGosoelpENGXM6PfKgknd8ZmhUlKCiAUcCDUPq6snL6ckRnM0rSqta1nBlMMlcQ2XHQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). Multiple deletions might disturb the symmetry of genome replication and cause growth changes. To avoid over regression, model selection among the five predictors (<em>L<sub>d</sub></em>, <em>K</em>, <em>M</em>, <em>τ</em>, <em>θ</em>) according to Akaike Information Criteria<a reveal-id="dsw035-B35" data-open="dsw035-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> was applied. The results showed that the most efficient regression (adjusted <em>R</em><sup>2</sup>, 0.9818) was achieved by choosing <em>L<sub>d</sub></em>, <em>M</em> and <em>θ</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035_Supp.pdf?Expires=2147483647&amp;Signature=yZ2sO9VP2Z3TYSkUXP~QOTiGs27MRweE6AWrjAkM9HNiC9Gvjlx8POMjF8qk8LzqIpRL-Nvx1OEyGZYpsf72-JrDcHVfi~vBdLUCmHfcPfCwtACeE9Y5b-eu~lNdQ5TNotgue4AFDZPtd2UKgPwc88ZFLFmM-6Fim-Rh6qebLui7ixMu0rZrlaWQ--ByMqKk6u5Btmr0WkLvRP-M3PbG8EJEsFDGSlUtKadVKMtkFO~-uRbgucy-VgNpi3L2sG4g5ynwG7dcp-7r1xkolCwxGosoelpENGXM6PfKgknd8ZmhUlKCiAUcCDUPq6snL6ckRnM0rSqta1nBlMMlcQ2XHQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). Rejection of <em>K</em> and <em>τ</em> from the growth prediction suggested the multicollinearity among <em>K</em>, <em>τ</em> and <em>L<sub>d</sub></em>, consistently with the previous discussion (<a href="#E2" class="sectionLink xref-sec">Equations 2∼5</a> and <a reveal-id="dsw035-F5" data-open="dsw035-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). <div data-id="dsw035-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/m_dsw035f5p.png?Expires=2147483647&amp;Signature=sMjGN~bmHCTE7lxxutCds6lL3XbDkg4o3neGUv8fs-Kqcwq1Je82UtWyHuSKnbmhSybw2ULseKm6PMWf~O9x9bBlLQ4h5V~1jbMQ7RxJufho6wZpP59-gnOXDsM8UUPl71foRsR6c3fXXiW-G5qASeuR9bmMYzLAr2BdaKo8o0~ACfx8trQJ2JCvTN0zddHpHv5GvW271zGoyQA94CpbJjVj3Zr-xIerXh7WcFnEmRVogx4Jciu1dnpNAaGaMfpzv3XAk8D~LRYwI51aaodrWhQjlUcswX4KNfoia0MN1qOIrim6zAi6zbpxHvphxJIZX1ULPTqP7-w6brYm-bSTCQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Multiple linear regression analysis of growth rate. Best prediction model of the growth rate (h −1, y-axis) with the length of accumulated deletions (Kb, x-axis), the asymmetric angle (°, z-axis) and the growth media (LB, MAA and M63) is presented. Colour planes and spheres represent the predicted and the experimental values, respectively. Residuals between the predicted and the experimental values are represented by the lines from spheres to the corresponding plane." path-from-xml="dsw035f5p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48004845/dsw035f5p.tif" path-from-xml="dsw035f5p.tif" target="_blank">View large</a><a section="48004845" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035f5p.png?Expires=2147483647&amp;Signature=4GJm1wy~7gi1nS8PAxOOkMivIkbghQPKORU3xLF26TLaycM172sjKAEk1n4x-0e5mxsTbhTkiyJxAnGTbYL7z5GBH9DzXjApCBw1d4VMBKgddhHZuPk5oHyUce-u4GbSVLiIu3kvDr-VcOHb8mQgVWX78K4iC19KjnZEtztoVTGcBjYL45mmW1CsUwJ9Eixlbwsl9tcdjQjQVXw1nkE-hu5KwaVXx3zVd-cuoLdh9dtV-lMl39Ye5yboFVKYl2bTSWAj6tzF06qay0GvlAPvzv7PEP3Cu8XEYBzzyG00dM82IoV2p6EZM9WVSpbqp2Q7RpQMJ9B4W-WW25UhuTzYhA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48004845&amp;ar=2647442&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw035f5p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Multiple linear regression analysis of growth rate. Best prediction model of the growth rate (h <sup>−</sup><sup>1</sup>, <em>y</em>-axis) with the length of accumulated deletions (Kb, <em>x</em>-axis), the asymmetric angle (°, <em>z</em>-axis) and the growth media (LB, MAA and M63) is presented. Colour planes and spheres represent the predicted and the experimental values, respectively. Residuals between the predicted and the experimental values are represented by the lines from spheres to the corresponding plane.</p></div></div><div content-id="dsw035-F5" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/m_dsw035f5p.png?Expires=2147483647&amp;Signature=sMjGN~bmHCTE7lxxutCds6lL3XbDkg4o3neGUv8fs-Kqcwq1Je82UtWyHuSKnbmhSybw2ULseKm6PMWf~O9x9bBlLQ4h5V~1jbMQ7RxJufho6wZpP59-gnOXDsM8UUPl71foRsR6c3fXXiW-G5qASeuR9bmMYzLAr2BdaKo8o0~ACfx8trQJ2JCvTN0zddHpHv5GvW271zGoyQA94CpbJjVj3Zr-xIerXh7WcFnEmRVogx4Jciu1dnpNAaGaMfpzv3XAk8D~LRYwI51aaodrWhQjlUcswX4KNfoia0MN1qOIrim6zAi6zbpxHvphxJIZX1ULPTqP7-w6brYm-bSTCQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Multiple linear regression analysis of growth rate. Best prediction model of the growth rate (h −1, y-axis) with the length of accumulated deletions (Kb, x-axis), the asymmetric angle (°, z-axis) and the growth media (LB, MAA and M63) is presented. Colour planes and spheres represent the predicted and the experimental values, respectively. Residuals between the predicted and the experimental values are represented by the lines from spheres to the corresponding plane." path-from-xml="dsw035f5p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48004845/dsw035f5p.tif" path-from-xml="dsw035f5p.tif" target="_blank">View large</a><a section="48004845" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035f5p.png?Expires=2147483647&amp;Signature=4GJm1wy~7gi1nS8PAxOOkMivIkbghQPKORU3xLF26TLaycM172sjKAEk1n4x-0e5mxsTbhTkiyJxAnGTbYL7z5GBH9DzXjApCBw1d4VMBKgddhHZuPk5oHyUce-u4GbSVLiIu3kvDr-VcOHb8mQgVWX78K4iC19KjnZEtztoVTGcBjYL45mmW1CsUwJ9Eixlbwsl9tcdjQjQVXw1nkE-hu5KwaVXx3zVd-cuoLdh9dtV-lMl39Ye5yboFVKYl2bTSWAj6tzF06qay0GvlAPvzv7PEP3Cu8XEYBzzyG00dM82IoV2p6EZM9WVSpbqp2Q7RpQMJ9B4W-WW25UhuTzYhA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=48004845&amp;ar=2647442&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw035f5p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Multiple linear regression analysis of growth rate. Best prediction model of the growth rate (h <sup>−</sup><sup>1</sup>, <em>y</em>-axis) with the length of accumulated deletions (Kb, <em>x</em>-axis), the asymmetric angle (°, <em>z</em>-axis) and the growth media (LB, MAA and M63) is presented. Colour planes and spheres represent the predicted and the experimental values, respectively. Residuals between the predicted and the experimental values are represented by the lines from spheres to the corresponding plane.</p></div></div></div><div class="block-child-p">Thus, the best regression for global effect on cell growth was defined as follows (<a href="#E6" class="sectionLink xref-sec">Equation 6</a>). <a id="E6" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E6"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>r</mi><mo>=</mo><mo stretchy="false">(</mo><mo>−</mo><mn>2.00</mn><mi>e</mi><mo>−</mo><mn>4</mn><mo stretchy="false">)</mo><msub><mi>L</mi><mi>d</mi></msub><mo>+</mo><mn>0.005</mn><mi>θ</mi><mo>+</mo><mrow><mo stretchy="true">{</mo><mtable columnalign="left"><mtr><mtd><mn>1.741</mn><mo>,</mo><mtable><mtr><mtd><mrow></mrow></mtd><mtd><mrow></mrow></mtd><mtd><mrow></mrow></mtd></mtr></mtable><mi>M</mi><mo>=</mo><mo>"</mo><mi>L</mi><mi>B</mi><mo>"</mo></mtd></mtr><mtr><mtd><mn>1.741</mn><mo>−</mo><mn>0.809</mn><mo>,</mo><mi>M</mi><mo>=</mo><mo>"</mo><mi>M</mi><mi>A</mi><mi>A</mi><mo>"</mo></mtd></mtr><mtr><mtd><mn>1.741</mn><mo>−</mo><mn>1.158</mn><mo>,</mo><mi>M</mi><mo>=</mo><mo>"</mo><mi>M</mi><mn>63</mn><mo>"</mo></mtd></mtr></mtable></mrow></mrow></math></div></div><div class="label">(6)</div></div></div><p>The predicted growth rates were similar to those acquired experimentally (<a reveal-id="dsw035-F4" data-open="dsw035-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). Notably, the additional introduction of the angle to the regression model improved the prediction accuracy (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035_Supp.pdf?Expires=2147483647&Signature=yZ2sO9VP2Z3TYSkUXP~QOTiGs27MRweE6AWrjAkM9HNiC9Gvjlx8POMjF8qk8LzqIpRL-Nvx1OEyGZYpsf72-JrDcHVfi~vBdLUCmHfcPfCwtACeE9Y5b-eu~lNdQ5TNotgue4AFDZPtd2UKgPwc88ZFLFmM-6Fim-Rh6qebLui7ixMu0rZrlaWQ--ByMqKk6u5Btmr0WkLvRP-M3PbG8EJEsFDGSlUtKadVKMtkFO~-uRbgucy-VgNpi3L2sG4g5ynwG7dcp-7r1xkolCwxGosoelpENGXM6PfKgknd8ZmhUlKCiAUcCDUPq6snL6ckRnM0rSqta1nBlMMlcQ2XHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). However, the direct comparison between the angle and growth changes did not detect any significant correlation (<em>P </em>&gt; 0.1, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035_Supp.pdf?Expires=2147483647&Signature=yZ2sO9VP2Z3TYSkUXP~QOTiGs27MRweE6AWrjAkM9HNiC9Gvjlx8POMjF8qk8LzqIpRL-Nvx1OEyGZYpsf72-JrDcHVfi~vBdLUCmHfcPfCwtACeE9Y5b-eu~lNdQ5TNotgue4AFDZPtd2UKgPwc88ZFLFmM-6Fim-Rh6qebLui7ixMu0rZrlaWQ--ByMqKk6u5Btmr0WkLvRP-M3PbG8EJEsFDGSlUtKadVKMtkFO~-uRbgucy-VgNpi3L2sG4g5ynwG7dcp-7r1xkolCwxGosoelpENGXM6PfKgknd8ZmhUlKCiAUcCDUPq6snL6ckRnM0rSqta1nBlMMlcQ2XHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). The slight but positive contribution of <em>θ</em> to <em>r</em> appeared to increase the prediction accuracy. Thus, the deterministic factors for growth fitness included genome reduction and growth conditions. Although the mechanism of the predictable growth changes was unclear and the theoretical relation might be strain- or reduction-mode-dependent, the present findings reveal a novel quantitative principle involved in the bacterial genome as a propagating machine.</p>                            <h3 scrollto-destination=48004848 id="48004848" class="section-title" data-legacy-id=SEC3.6>3.6 <em>A hypothesis connecting genome reduction with growth changes</em></h3>
<div class="block-child-p">Given that there are dispensable regions in bacterial genomes, harbouring the accessory genes involved in genome evolution,<a reveal-id="dsw035-B41" data-open="dsw035-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a><sup>,</sup><a reveal-id="dsw035-B42" data-open="dsw035-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> we propose a hypothesis to explain the correlation between accumulative genomic reduction and gradual growth changes, particularly in severe environments (<a reveal-id="dsw035-F6" data-open="dsw035-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>A). First, the length-dependent changes may be explained by considering genome evolution as a molecular clock.<a reveal-id="dsw035-B43" data-open="dsw035-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> Genome evolution should promote an enlarged genome size by incorporating non-essential accessory genes (e.g. horizontal gene transfer, HGT),<a reveal-id="dsw035-B44" data-open="dsw035-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a><sup>,</sup><a reveal-id="dsw035-B45" data-open="dsw035-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> which might be disadvantageous for the growth fitness, given the additional cost for the replication and expression of the newly acquired sequences. Nevertheless, the evolutionary selection on genomic sequences was based on growth fitness. The enlarged genome size was subsequently optimized (e.g. mutagenesis) to generate novel genetic interactions for beneficial cooperation from the new sequences (i.e. accessory genes) to achieve the growth recovery (fitness increase). Thus, the wild type genome was the evolutionary consequence representing a balance between the additional cost and the novel cooperation, as well as a balanced distribution of accessory genes in genome. Given that genomic sequence elongation correlates with the evolutionary period (clock), the deletion-length dependent fitness decrease is reasonable. Second, multiple deletions of the evolved genomic sequence in the laboratory clearly disturbed the cooperative accessory networks formed during evolution, thus leading to decreased fitness regardless of the remaining growing capacity. Additional steps for sequence optimization or complete systematic reduction of accessory networks was required to retain the growth fitness of the small genome, thus potentially explaining why the reduced genome MDS42 grew as fast as the wild type genome did in minimal medium.<a reveal-id="dsw035-B12" data-open="dsw035-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><a reveal-id="dsw035-B22" data-open="dsw035-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><a reveal-id="dsw035-B33" data-open="dsw035-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> Because MDS42 exhibited a fixed periodicity for efficient growth in the transcriptome<a reveal-id="dsw035-B33" data-open="dsw035-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> and the sequence differences from the MG1655 counterpart ranging from single nucleotide substitutions to complete deletions,<a reveal-id="dsw035-B46" data-open="dsw035-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> we assumed that either a successful systematic loss of accessory networks or the sequence optimization owing to the serial transfer was achieved. <div data-id="dsw035-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/m_dsw035f6p.png?Expires=2147483647&Signature=UIN~MM7AFbMhdtru1XKp7M~0Jg5BcwdWlV7ThSIT6G8Cfmn6e2blMAidNERczo5GAEHRidjcR3Gy4HmQ~rVRwlBkEhrk~3PMcVegZwoTTwhjVOTAfj3re7DMwCwrdZifnboNWik~vAI~bBtdFgbGBztvpjqGF9Cwne~kVedl~Gdvucrzi6BZhCcqq0SIyS8SqPlt5Qql0udsKI6v-~h4kjpVYFj2DcYNWrFdwEZ1SQBwP0jnZPvHI0T58Kb75rbEqw2Idt914s63~wYKxJ9fXNWChAKWUiVsYOxIH-t1ny~WDGBVq2HOiCr~XQEA~RWPlf9azMMqCbrgvJNfOzN47Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Hypothesis for genome reduction-correlated growth changes. (A) Perspective of genome size. Left and right flowcharts illustrate the processes of genome evolution and genome reduction, respectively. Upward and downward bold arrows indicate the increased and decreased changes in growth fitness, respectively. A detailed explanation is provided in the main text. (B) Correlation between the growth rate and the number of deleted genes. The deleted genes were clustered according to the gene categories. Statistical significance of the correlation coefficients between the growth rates and the numbers of deleted genes in each gene categories is presented as a heat map with a logarithmic scale. The progression from dark to light represents the significance from high to low, and the white colour represents no significance. M63, MAA and LB indicate the growth media. Gene categories are noted in both full names and abbreviations." path-from-xml="dsw035f6p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48004849/dsw035f6p.tif" path-from-xml="dsw035f6p.tif" target="_blank">View large</a><a section="48004849" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035f6p.png?Expires=2147483647&Signature=pkxlGic12FixnKA~B2E0UbM-3-yY~raz0wYc5Sio8-kRdfJyU9fDgk5gZTKuz3qRO348zY8al7YNgYjFfJZIZjEhV652lfPz4auDPB~sYJeU989Ya7ZMDMIMKa1DeRp5KvPTXtVjY6p~OxbmHWC~desT5FpsyIroudJNmoq2BKZeBBAiN4Now~3RwGrb1n6j3SfTh6NuswgChZUzbYWxgpm~fk10YEp7mQNA0--LoUkw-YKF3hcc6eOPfXHL7tud5NTvfP1yNkKGT1gkg9aPDNnicPQEeSTeMIueyfsijye5KWr1q6uNEwPsM4D2ct5HDKuJM-tYsb94sXPlhItmdw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=48004849&ar=2647442&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw035f6p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Hypothesis for genome reduction-correlated growth changes. (A) Perspective of genome size. Left and right flowcharts illustrate the processes of genome evolution and genome reduction, respectively. Upward and downward bold arrows indicate the increased and decreased changes in growth fitness, respectively. A detailed explanation is provided in the main text. (B) Correlation between the growth rate and the number of deleted genes. The deleted genes were clustered according to the gene categories. Statistical significance of the correlation coefficients between the growth rates and the numbers of deleted genes in each gene categories is presented as a heat map with a logarithmic scale. The progression from dark to light represents the significance from high to low, and the white colour represents no significance. M63, MAA and LB indicate the growth media. Gene categories are noted in both full names and abbreviations.</p></div></div><div content-id="dsw035-F6" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/m_dsw035f6p.png?Expires=2147483647&Signature=UIN~MM7AFbMhdtru1XKp7M~0Jg5BcwdWlV7ThSIT6G8Cfmn6e2blMAidNERczo5GAEHRidjcR3Gy4HmQ~rVRwlBkEhrk~3PMcVegZwoTTwhjVOTAfj3re7DMwCwrdZifnboNWik~vAI~bBtdFgbGBztvpjqGF9Cwne~kVedl~Gdvucrzi6BZhCcqq0SIyS8SqPlt5Qql0udsKI6v-~h4kjpVYFj2DcYNWrFdwEZ1SQBwP0jnZPvHI0T58Kb75rbEqw2Idt914s63~wYKxJ9fXNWChAKWUiVsYOxIH-t1ny~WDGBVq2HOiCr~XQEA~RWPlf9azMMqCbrgvJNfOzN47Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Hypothesis for genome reduction-correlated growth changes. (A) Perspective of genome size. Left and right flowcharts illustrate the processes of genome evolution and genome reduction, respectively. Upward and downward bold arrows indicate the increased and decreased changes in growth fitness, respectively. A detailed explanation is provided in the main text. (B) Correlation between the growth rate and the number of deleted genes. The deleted genes were clustered according to the gene categories. Statistical significance of the correlation coefficients between the growth rates and the numbers of deleted genes in each gene categories is presented as a heat map with a logarithmic scale. The progression from dark to light represents the significance from high to low, and the white colour represents no significance. M63, MAA and LB indicate the growth media. Gene categories are noted in both full names and abbreviations." path-from-xml="dsw035f6p.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/48004849/dsw035f6p.tif" path-from-xml="dsw035f6p.tif" target="_blank">View large</a><a section="48004849" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035f6p.png?Expires=2147483647&Signature=pkxlGic12FixnKA~B2E0UbM-3-yY~raz0wYc5Sio8-kRdfJyU9fDgk5gZTKuz3qRO348zY8al7YNgYjFfJZIZjEhV652lfPz4auDPB~sYJeU989Ya7ZMDMIMKa1DeRp5KvPTXtVjY6p~OxbmHWC~desT5FpsyIroudJNmoq2BKZeBBAiN4Now~3RwGrb1n6j3SfTh6NuswgChZUzbYWxgpm~fk10YEp7mQNA0--LoUkw-YKF3hcc6eOPfXHL7tud5NTvfP1yNkKGT1gkg9aPDNnicPQEeSTeMIueyfsijye5KWr1q6uNEwPsM4D2ct5HDKuJM-tYsb94sXPlhItmdw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=48004849&ar=2647442&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw035f6p.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Hypothesis for genome reduction-correlated growth changes. (A) Perspective of genome size. Left and right flowcharts illustrate the processes of genome evolution and genome reduction, respectively. Upward and downward bold arrows indicate the increased and decreased changes in growth fitness, respectively. A detailed explanation is provided in the main text. (B) Correlation between the growth rate and the number of deleted genes. The deleted genes were clustered according to the gene categories. Statistical significance of the correlation coefficients between the growth rates and the numbers of deleted genes in each gene categories is presented as a heat map with a logarithmic scale. The progression from dark to light represents the significance from high to low, and the white colour represents no significance. M63, MAA and LB indicate the growth media. Gene categories are noted in both full names and abbreviations.</p></div></div></div><p>A supportive line of evidence was that the gene function correlation analysis did not detect any single gene category that was specifically responsible for the changes in population growth (<a reveal-id="dsw035-F6" data-open="dsw035-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>B). We analysed whether any gene categories specifically contributed to the decrease in growth rate. The gene categories comprising &gt;10 deleted genes in strain No. 28 (independent of the significance evaluated in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035_Supp.pdf?Expires=2147483647&Signature=yZ2sO9VP2Z3TYSkUXP~QOTiGs27MRweE6AWrjAkM9HNiC9Gvjlx8POMjF8qk8LzqIpRL-Nvx1OEyGZYpsf72-JrDcHVfi~vBdLUCmHfcPfCwtACeE9Y5b-eu~lNdQ5TNotgue4AFDZPtd2UKgPwc88ZFLFmM-6Fim-Rh6qebLui7ixMu0rZrlaWQ--ByMqKk6u5Btmr0WkLvRP-M3PbG8EJEsFDGSlUtKadVKMtkFO~-uRbgucy-VgNpi3L2sG4g5ynwG7dcp-7r1xkolCwxGosoelpENGXM6PfKgknd8ZmhUlKCiAUcCDUPq6snL6ckRnM0rSqta1nBlMMlcQ2XHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>) were subjected to the evaluation. The correlations between the decreasing growth rates from strain No. 1 to 28 (<a reveal-id="dsw035-F2" data-open="dsw035-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>A) and the increasing numbers of deleted genes in each gene categories (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035_Supp.pdf?Expires=2147483647&Signature=yZ2sO9VP2Z3TYSkUXP~QOTiGs27MRweE6AWrjAkM9HNiC9Gvjlx8POMjF8qk8LzqIpRL-Nvx1OEyGZYpsf72-JrDcHVfi~vBdLUCmHfcPfCwtACeE9Y5b-eu~lNdQ5TNotgue4AFDZPtd2UKgPwc88ZFLFmM-6Fim-Rh6qebLui7ixMu0rZrlaWQ--ByMqKk6u5Btmr0WkLvRP-M3PbG8EJEsFDGSlUtKadVKMtkFO~-uRbgucy-VgNpi3L2sG4g5ynwG7dcp-7r1xkolCwxGosoelpENGXM6PfKgknd8ZmhUlKCiAUcCDUPq6snL6ckRnM0rSqta1nBlMMlcQ2XHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>, 14 out of 23 gene categories) were evaluated. High significance (<em>P </em>&lt; 0.001) was detected in all 14 gene categories under the conditions of M63 and LB, thus indicating that all gene categories contributed to the growth decrease in both rich and poor nutritional conditions. The addition of 20 amino acids (MAA) decreased the significance of such correlation, thus suggesting that these amino acids compensated for the loss of the genes broadly classified in not only the central gene categories of Enzyme (e), Structural component (s), and Regulator (r), but also the accessory gene categories of Unknown function (o), Phage/IS (h), Pseudogenes (su) and predicted functions (pf, pr). Although a single accessory gene might have an undetectable contribution to the growth (∼0.03–0.001%), a cluster of these genes might reach a quantitatively predictive level regardless of the variation in gene categories and/or functional mechanisms. Note that other gene function analyses failed to acquire any significant correlated contribution, which was largely due to the missing functional assignment of the deleted accessory genes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035_Supp.pdf?Expires=2147483647&Signature=yZ2sO9VP2Z3TYSkUXP~QOTiGs27MRweE6AWrjAkM9HNiC9Gvjlx8POMjF8qk8LzqIpRL-Nvx1OEyGZYpsf72-JrDcHVfi~vBdLUCmHfcPfCwtACeE9Y5b-eu~lNdQ5TNotgue4AFDZPtd2UKgPwc88ZFLFmM-6Fim-Rh6qebLui7ixMu0rZrlaWQ--ByMqKk6u5Btmr0WkLvRP-M3PbG8EJEsFDGSlUtKadVKMtkFO~-uRbgucy-VgNpi3L2sG4g5ynwG7dcp-7r1xkolCwxGosoelpENGXM6PfKgknd8ZmhUlKCiAUcCDUPq6snL6ckRnM0rSqta1nBlMMlcQ2XHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). The broad but not specific correlations between the gene category and the growth rate not only explained the finding of that the growth rate could be predicted on the basis of the accumulatively deleted length alone but also supported the assumption in which evolutionary sequence optimization can build the functional interactions involving accessory genes and thus result in increased fitness.</p><p>In summary, the accumulative loss of dispensable genomic sequences contributes to bacterial growth in a dose-dependent manner, significantly in poor conditions. The data sets and the theoretical underpinnings suggest a quantitative linkage between genomic sequences and growth fitness and should be valuable in understanding genome evolution and constructing the desired genome for a growing system.</p>                            <h2 scrollto-destination=48004852 id="48004852" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank the National BioResource Project and the National Institute of Genetics (Shizuoka, Japan) for providing the <em>E. coli</em> strains and our lab members for fruitful discussions.</p>                            <h3 scrollto-destination=48004854 id="48004854" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h3 scrollto-destination=48004856 id="48004856" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035_Supp.pdf?Expires=2147483647&Signature=yZ2sO9VP2Z3TYSkUXP~QOTiGs27MRweE6AWrjAkM9HNiC9Gvjlx8POMjF8qk8LzqIpRL-Nvx1OEyGZYpsf72-JrDcHVfi~vBdLUCmHfcPfCwtACeE9Y5b-eu~lNdQ5TNotgue4AFDZPtd2UKgPwc88ZFLFmM-6Fim-Rh6qebLui7ixMu0rZrlaWQ--ByMqKk6u5Btmr0WkLvRP-M3PbG8EJEsFDGSlUtKadVKMtkFO~-uRbgucy-VgNpi3L2sG4g5ynwG7dcp-7r1xkolCwxGosoelpENGXM6PfKgknd8ZmhUlKCiAUcCDUPq6snL6ckRnM0rSqta1nBlMMlcQ2XHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h3 scrollto-destination=48004858 id="48004858" class="section-title" >Funding</h3>
<p>This work was partially supported by a Grant-in-Aid for Scientific Research (C) no. 26506003 (to BWY) and a Grant-in-Aid for Scientific Research on Innovative Areas no. 26119702 (to BWY) from the Ministry of Education, Culture, Sports, Science and Technology, Japan. Funding to pay the Open Access publication charges for this article was provided by the Ministry of Education, Culture, Sports, Science and Technology, Japan.</p>                            <h2 scrollto-destination=48004860 id="48004860" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw035-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Koob</div> <div class="given-names">M.D.</div></div><div class="name string-name"><div class="surname">Shaw</div> <div class="given-names">A.J.</div></div><div class="name string-name"><div class="surname">Cameron</div> <div class="given-names">D.C.</div></div></div> <div class="year">1994</div>, 
<div class="article-title">Minimizing the genome of <em>Escherichia coli</em>. Motivation and strategy,</div> <div class="source">Ann. N. Y. Acad. Sci</div> ., <div class="volume">745</div>, <div class="fpage">1</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Minimizing%20the%20genome%20of%20Escherichia%20coli.%20Motivation%20and%20strategy%2C&amp;author=M.D.%20Koob&amp;author=A.J.%20Shaw&amp;author=D.C.%20Cameron&amp;publication_year=1994&amp;journal=Ann.%20N.%20Y.%20Acad.%20Sci&amp;volume=745&amp;pages=1-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nyas.1994.745.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnyas.1994.745.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnyas.1994.745.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7832501" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Blattner</div> <div class="given-names">F.R.</div></div><div class="name string-name"><div class="surname">Plunkett</div> <div class="given-names">G.</div>3rd</div><div class="name string-name"><div class="surname">Bloch</div> <div class="given-names">C.A.</div></div></div>, et al.  . <div class="year">1997</div>, 
<div class="article-title">The complete genome sequence of <em>Escherichia coli</em> K-12,</div> <div class="source">Science</div> , <div class="volume">277</div>, <div class="fpage">1453</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complete%20genome%20sequence%20of%20Escherichia%20coli%20K-12%2C&amp;author=F.R.%20Blattner&amp;author=G.%20Plunkett&amp;author=C.A.%20Bloch&amp;publication_year=1997&amp;journal=Science&amp;volume=277&amp;pages=1453-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.277.5331.1453" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.277.5331.1453" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.277.5331.1453"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9278503" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kolisnychenko</div> <div class="given-names">V.</div></div><div class="name string-name"><div class="surname">Plunkett</div> <div class="given-names">G.</div>3rd</div><div class="name string-name"><div class="surname">Herring</div> <div class="given-names">C.D.</div></div></div>, et al.  . <div class="year">2002</div>, 
<div class="article-title">Engineering a reduced <em>Escherichia coli</em> genome,</div> <div class="source">Genome Res</div> ., <div class="volume">12</div>, <div class="fpage">640</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Engineering%20a%20reduced%20Escherichia%20coli%20genome%2C&amp;author=V.%20Kolisnychenko&amp;author=G.%20Plunkett&amp;author=C.D.%20Herring&amp;publication_year=2002&amp;journal=Genome%20Res&amp;volume=12&amp;pages=640-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.217202" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.217202" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.217202"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11932248" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yu</div> <div class="given-names">B.J.</div></div><div class="name string-name"><div class="surname">Sung</div> <div class="given-names">B.H.</div></div><div class="name string-name"><div class="surname">Koob</div> <div class="given-names">M.D.</div></div></div>, et al.  . <div class="year">2002</div>, 
<div class="article-title">Minimization of the <em>Escherichia coli</em> genome using a Tn5-targeted Cre/loxP excision system,</div> <div class="source">Nat. Biotechnol</div> ., <div class="volume">20</div>, <div class="fpage">1018</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Minimization%20of%20the%20Escherichia%20coli%20genome%20using%20a%20Tn5-targeted%20Cre%2FloxP%20excision%20system%2C&amp;author=B.J.%20Yu&amp;author=B.H.%20Sung&amp;author=M.D.%20Koob&amp;publication_year=2002&amp;journal=Nat.%20Biotechnol&amp;volume=20&amp;pages=1018-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt740" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt740" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt740"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12244329" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Feher</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Karcagi</div> <div class="given-names">I.</div></div><div class="name string-name"><div class="surname">Gyorfy</div> <div class="given-names">Z.</div></div><div class="name string-name"><div class="surname">Umenhoffer</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Csorgo</div> <div class="given-names">B.</div></div><div class="name string-name"><div class="surname">Posfai</div> <div class="given-names">G.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Scarless engineering of the <em>Escherichia coli</em> genome,</div> <div class="source">Methods Mol. Biol</div> ., <div class="volume">416</div>, <div class="fpage">251</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Scarless%20engineering%20of%20the%20Escherichia%20coli%20genome%2C&amp;author=T.%20Feher&amp;author=I.%20Karcagi&amp;author=Z.%20Gyorfy&amp;author=K.%20Umenhoffer&amp;author=B.%20Csorgo&amp;author=G.%20Posfai&amp;publication_year=2008&amp;journal=Methods%20Mol.%20Biol&amp;volume=416&amp;pages=251-9" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18392972" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Feist</div> <div class="given-names">A.M.</div></div><div class="name string-name"><div class="surname">Henry</div> <div class="given-names">C.S.</div></div><div class="name string-name"><div class="surname">Reed</div> <div class="given-names">J.L.</div></div></div>, et al.  . <div class="year">2007</div>, 
<div class="article-title">A genome-scale metabolic reconstruction for <em>Escherichia coli</em> K-12 MG1655 that accounts for 1260 ORFs and thermodynamic information,</div> <div class="source">Mol. Syst. Biol</div> ., <div class="volume">3</div>, <div class="fpage">121</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genome-scale%20metabolic%20reconstruction%20for%20Escherichia%20coli%20K-12%20MG1655%20that%20accounts%20for%201260%20ORFs%20and%20thermodynamic%20information%2C&amp;author=A.M.%20Feist&amp;author=C.S.%20Henry&amp;author=J.L.%20Reed&amp;publication_year=2007&amp;journal=Mol.%20Syst.%20Biol&amp;volume=3&amp;pages=121" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/msb4100155" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fmsb4100155" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fmsb4100155"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17593909" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mizoguchi</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Mori</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Fujio</div> <div class="given-names">T.</div></div></div> <div class="year">2007</div>, 
<div class="article-title"><em>Escherichia coli</em> minimum genome factory,</div> <div class="source">Biotechnol. Appl. Biochem</div> ., <div class="volume">46</div>, <div class="fpage">157</div>–<div class="lpage">67</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Escherichia%20coli%20minimum%20genome%20factory%2C&amp;author=H.%20Mizoguchi&amp;author=H.%20Mori&amp;author=T.%20Fujio&amp;publication_year=2007&amp;journal=Biotechnol.%20Appl.%20Biochem&amp;volume=46&amp;pages=157-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1042/BA20060107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1042%2FBA20060107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1042%2FBA20060107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17300222" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hutchison</div> <div class="given-names">C.A.</div>3rd</div><div class="name string-name"><div class="surname">Chuang</div> <div class="given-names">R.Y.</div></div><div class="name string-name"><div class="surname">Noskov</div> <div class="given-names">V.N.</div></div></div>, et al.  . <div class="year">2016</div>, 
<div class="article-title">Design and synthesis of a minimal bacterial genome,</div> <div class="source">Science</div> , <div class="volume">351</div>, <div class="fpage">aad6253</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Design%20and%20synthesis%20of%20a%20minimal%20bacterial%20genome%2C&amp;author=C.A.%20Hutchison&amp;author=R.Y.%20Chuang&amp;author=V.N.%20Noskov&amp;publication_year=2016&amp;journal=Science&amp;volume=351&amp;pages=aad6253" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.aad6253" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.aad6253" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.aad6253"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27013737" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xavier</div> <div class="given-names">J.C.</div></div><div class="name string-name"><div class="surname">Patil</div> <div class="given-names">K.R.</div></div><div class="name string-name"><div class="surname">Rocha</div> <div class="given-names">I.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Systems biology perspectives on minimal and simpler cells,</div> <div class="source">Microbiol. Mol. Biol. Rev</div> ., <div class="volume">78</div>, <div class="fpage">487</div>–<div class="lpage">509</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systems%20biology%20perspectives%20on%20minimal%20and%20simpler%20cells%2C&amp;author=J.C.%20Xavier&amp;author=K.R.%20Patil&amp;author=I.%20Rocha&amp;publication_year=2014&amp;journal=Microbiol.%20Mol.%20Biol.%20Rev&amp;volume=78&amp;pages=487-509" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MMBR.00050-13" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMMBR.00050-13" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMMBR.00050-13"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25184563" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Baba</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Ara</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Hasegawa</div> <div class="given-names">M.</div></div></div>, et al.  . <div class="year">2006</div>, 
<div class="article-title">Construction of <em>Escherichia coli</em> K-12 in-frame, single-gene knockout mutants: the Keio collection,</div> <div class="source">Mol. Syst. Biol</div> ., <div class="volume">2</div>, <div class="fpage">2006 0008</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20Escherichia%20coli%20K-12%20in-frame%2C%20single-gene%20knockout%20mutants%3A%20the%20Keio%20collection%2C&amp;author=T.%20Baba&amp;author=T.%20Ara&amp;author=M.%20Hasegawa&amp;publication_year=2006&amp;journal=Mol.%20Syst.%20Biol&amp;volume=2&amp;pages=2006 0008" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/msb4100050" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fmsb4100050" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fmsb4100050"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16738554" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yamamoto</div> <div class="given-names">N.</div></div><div class="name string-name"><div class="surname">Nakahigashi</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Nakamichi</div> <div class="given-names">T.</div></div></div>, et al.  . <div class="year">2009</div>, 
<div class="article-title">Update on the Keio collection of <em>Escherichia coli</em> single-gene deletion mutants,</div> <div class="source">Mol. Syst. Biol</div> ., <div class="volume">5</div>, <div class="fpage">335</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Update%20on%20the%20Keio%20collection%20of%20Escherichia%20coli%20single-gene%20deletion%20mutants%2C&amp;author=N.%20Yamamoto&amp;author=K.%20Nakahigashi&amp;author=T.%20Nakamichi&amp;publication_year=2009&amp;journal=Mol.%20Syst.%20Biol&amp;volume=5&amp;pages=335" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/msb.2009.92" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fmsb.2009.92" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fmsb.2009.92"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20029369" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Posfai</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Plunkett</div> <div class="given-names">G.</div>3rd</div><div class="name string-name"><div class="surname">Feher</div> <div class="given-names">T.</div></div></div>, et al.  . <div class="year">2006</div>, 
<div class="article-title">Emergent properties of reduced-genome <em>Escherichia coli</em>,</div> <div class="source">Science</div> , <div class="volume">312</div>, <div class="fpage">1044</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Emergent%20properties%20of%20reduced-genome%20Escherichia%20coli%2C&amp;author=G.%20Posfai&amp;author=G.%20Plunkett&amp;author=T.%20Feher&amp;publication_year=2006&amp;journal=Science&amp;volume=312&amp;pages=1044-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1126439" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1126439" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1126439"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16645050" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kato</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Hashimoto</div> <div class="given-names">M.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Construction of consecutive deletions of the <em>Escherichia coli</em> chromosome,</div> <div class="source">Mol. Syst. Biol</div> ., <div class="volume">3</div>, <div class="fpage">132</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20consecutive%20deletions%20of%20the%20Escherichia%20coli%20chromosome%2C&amp;author=J.%20Kato&amp;author=M.%20Hashimoto&amp;publication_year=2007&amp;journal=Mol.%20Syst.%20Biol&amp;volume=3&amp;pages=132" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/msb4100174" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fmsb4100174" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fmsb4100174"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17700540" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mizoguchi</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Sawano</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Kato</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Mori</div> <div class="given-names">H.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Superpositioning of deletions promotes growth of <em>Escherichia coli</em> with a reduced genome,</div> <div class="source">DNA Res</div> ., <div class="volume">15</div>, <div class="fpage">277</div>–<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Superpositioning%20of%20deletions%20promotes%20growth%20of%20Escherichia%20coli%20with%20a%20reduced%20genome%2C&amp;author=H.%20Mizoguchi&amp;author=Y.%20Sawano&amp;author=J.%20Kato&amp;author=H.%20Mori&amp;publication_year=2008&amp;journal=DNA%20Res&amp;volume=15&amp;pages=277-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsn019" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsn019" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsn019"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18753290" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gibson</div> <div class="given-names">D.G.</div></div><div class="name string-name"><div class="surname">Glass</div> <div class="given-names">J.I.</div></div><div class="name string-name"><div class="surname">Lartigue</div> <div class="given-names">C.</div></div></div>, et al.  . <div class="year">2010</div>, 
<div class="article-title">Creation of a bacterial cell controlled by a chemically synthesized genome,</div> <div class="source">Science</div> , <div class="volume">329</div>, <div class="fpage">52</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Creation%20of%20a%20bacterial%20cell%20controlled%20by%20a%20chemically%20synthesized%20genome%2C&amp;author=D.G.%20Gibson&amp;author=J.I.%20Glass&amp;author=C.%20Lartigue&amp;publication_year=2010&amp;journal=Science&amp;volume=329&amp;pages=52-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1190719" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1190719" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1190719"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20488990" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Feher</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Papp</div> <div class="given-names">B.</div></div><div class="name string-name"><div class="surname">Pal</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Posfai</div> <div class="given-names">G.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Systematic genome reductions: theoretical and experimental approaches,</div> <div class="source">Chem. Rev</div> ., <div class="volume">107</div>, <div class="fpage">3498</div>–<div class="lpage">513</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systematic%20genome%20reductions%3A%20theoretical%20and%20experimental%20approaches%2C&amp;author=T.%20Feher&amp;author=B.%20Papp&amp;author=C.%20Pal&amp;author=G.%20Posfai&amp;publication_year=2007&amp;journal=Chem.%20Rev&amp;volume=107&amp;pages=3498-513" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/cr0683111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fcr0683111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fcr0683111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17636890" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pal</div> <div class="given-names">C.</div></div><div class="name string-name"><div class="surname">Papp</div> <div class="given-names">B.</div></div><div class="name string-name"><div class="surname">Posfai</div> <div class="given-names">G.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">The dawn of evolutionary genome engineering,</div> <div class="source">Nat. Rev. Genet</div> ., <div class="volume">15</div>, <div class="fpage">504</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20dawn%20of%20evolutionary%20genome%20engineering%2C&amp;author=C.%20Pal&amp;author=B.%20Papp&amp;author=G.%20Posfai&amp;publication_year=2014&amp;journal=Nat.%20Rev.%20Genet&amp;volume=15&amp;pages=504-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3746" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3746" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3746"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24866756" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yus</div> <div class="given-names">E.</div></div><div class="name string-name"><div class="surname">Maier</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Michalodimitrakis</div> <div class="given-names">K.</div></div></div>, et al.  . <div class="year">2009</div>, 
<div class="article-title">Impact of genome reduction on bacterial metabolism and its regulation,</div> <div class="source">Science</div> , <div class="volume">326</div>, <div class="fpage">1263</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Impact%20of%20genome%20reduction%20on%20bacterial%20metabolism%20and%20its%20regulation%2C&amp;author=E.%20Yus&amp;author=T.%20Maier&amp;author=K.%20Michalodimitrakis&amp;publication_year=2009&amp;journal=Science&amp;volume=326&amp;pages=1263-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1177263" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1177263" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1177263"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19965476" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sharma</div> <div class="given-names">S.S.</div></div><div class="name string-name"><div class="surname">Campbell</div> <div class="given-names">J.W.</div></div><div class="name string-name"><div class="surname">Frisch</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Blattner</div> <div class="given-names">F.R.</div></div><div class="name string-name"><div class="surname">Harcum</div> <div class="given-names">S.W.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Expression of two recombinant chloramphenicol acetyltransferase variants in highly reduced genome <em>Escherichia coli</em> strains,</div> <div class="source">Biotechnol. Bioeng</div> ., <div class="volume">98</div>, <div class="fpage">1056</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20of%20two%20recombinant%20chloramphenicol%20acetyltransferase%20variants%20in%20highly%20reduced%20genome%20Escherichia%20coli%20strains%2C&amp;author=S.S.%20Sharma&amp;author=J.W.%20Campbell&amp;author=D.%20Frisch&amp;author=F.R.%20Blattner&amp;author=S.W.%20Harcum&amp;publication_year=2007&amp;journal=Biotechnol.%20Bioeng&amp;volume=98&amp;pages=1056-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0290" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0290" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0290"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17497738" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Morimoto</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Kadoya</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Endo</div> <div class="given-names">K.</div></div></div>, et al.  . <div class="year">2008</div>, 
<div class="article-title">Enhanced recombinant protein productivity by genome reduction in <em>Bacillus subtilis</em>,</div> <div class="source">DNA Res</div> ., <div class="volume">15</div>, <div class="fpage">73</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Enhanced%20recombinant%20protein%20productivity%20by%20genome%20reduction%20in%20Bacillus%20subtilis%2C&amp;author=T.%20Morimoto&amp;author=R.%20Kadoya&amp;author=K.%20Endo&amp;publication_year=2008&amp;journal=DNA%20Res&amp;volume=15&amp;pages=73-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsn002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsn002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsn002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18334513" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lee</div> <div class="given-names">J.H.</div></div><div class="name string-name"><div class="surname">Sung</div> <div class="given-names">B.H.</div></div><div class="name string-name"><div class="surname">Kim</div> <div class="given-names">M.S.</div></div></div>, et al.  . <div class="year">2009</div>, 
<div class="article-title">Metabolic engineering of a reduced-genome strain of <em>Escherichia coli</em> for L-threonine production,</div> <div class="source">Microb. Cell Factories</div> , <div class="volume">8</div>, <div class="fpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metabolic%20engineering%20of%20a%20reduced-genome%20strain%20of%20Escherichia%20coli%20for%20L-threonine%20production%2C&amp;author=J.H.%20Lee&amp;author=B.H.%20Sung&amp;author=M.S.%20Kim&amp;publication_year=2009&amp;journal=Microb.%20Cell%20Factories&amp;volume=8&amp;pages=2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1475-2859-8-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1475-2859-8-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1475-2859-8-2"> </span></div> </div></div></div></div></div><div content-id="dsw035-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Akeno</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Ying</div> <div class="given-names">B.W.</div></div><div class="name string-name"><div class="surname">Tsuru</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Yomo</div> <div class="given-names">T.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">A reduced genome decreases the host carrying capacity for foreign DNA,</div> <div class="source">Microb. Cell Factories</div> , <div class="volume">13</div>, <div class="fpage">49</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20reduced%20genome%20decreases%20the%20host%20carrying%20capacity%20for%20foreign%20DNA%2C&amp;author=Y.%20Akeno&amp;author=B.W.%20Ying&amp;author=S.%20Tsuru&amp;author=T.%20Yomo&amp;publication_year=2014&amp;journal=Microb.%20Cell%20Factories&amp;volume=13&amp;pages=49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1475-2859-13-49" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1475-2859-13-49" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1475-2859-13-49"> </span></div> </div></div></div></div></div><div content-id="dsw035-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Csorgo</div> <div class="given-names">B.</div></div><div class="name string-name"><div class="surname">Feher</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Timar</div> <div class="given-names">E.</div></div><div class="name string-name"><div class="surname">Blattner</div> <div class="given-names">F.R.</div></div><div class="name string-name"><div class="surname">Posfai</div> <div class="given-names">G.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Low-mutation-rate, reduced-genome <em>Escherichia coli</em>: an improved host for faithful maintenance of engineered genetic constructs,</div> <div class="source">Microb. Cell Factories</div> , <div class="volume">11</div>, <div class="fpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Low-mutation-rate%2C%20reduced-genome%20Escherichia%20coli%3A%20an%20improved%20host%20for%20faithful%20maintenance%20of%20engineered%20genetic%20constructs%2C&amp;author=B.%20Csorgo&amp;author=T.%20Feher&amp;author=E.%20Timar&amp;author=F.R.%20Blattner&amp;author=G.%20Posfai&amp;publication_year=2012&amp;journal=Microb.%20Cell%20Factories&amp;volume=11&amp;pages=11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1475-2859-11-11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1475-2859-11-11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1475-2859-11-11"> </span></div> </div></div></div></div></div><div content-id="dsw035-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Umenhoffer</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Feher</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Baliko</div> <div class="given-names">G.</div></div></div>, et al.  . <div class="year">2010</div>, 
<div class="article-title">Reduced evolvability of <em>Escherichia coli</em> MDS42, an IS-less cellular chassis for molecular and synthetic biology applications,</div> <div class="source">Microb. Cell Factories</div> , <div class="volume">9</div>, <div class="fpage">38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reduced%20evolvability%20of%20Escherichia%20coli%20MDS42%2C%20an%20IS-less%20cellular%20chassis%20for%20molecular%20and%20synthetic%20biology%20applications%2C&amp;author=K.%20Umenhoffer&amp;author=T.%20Feher&amp;author=G.%20Baliko&amp;publication_year=2010&amp;journal=Microb.%20Cell%20Factories&amp;volume=9&amp;pages=38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1475-2859-9-38" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1475-2859-9-38" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1475-2859-9-38"> </span></div> </div></div></div></div></div><div content-id="dsw035-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sabath</div> <div class="given-names">N.</div></div><div class="name string-name"><div class="surname">Ferrada</div> <div class="given-names">E.</div></div><div class="name string-name"><div class="surname">Barve</div> <div class="given-names">A.</div></div><div class="name string-name"><div class="surname">Wagner</div> <div class="given-names">A.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Growth temperature and genome size in bacteria are negatively correlated, suggesting genomic streamlining during thermal adaptation,</div> <div class="source">Genome Biol. Evol</div> ., <div class="volume">5</div>, <div class="fpage">966</div>–<div class="lpage">77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Growth%20temperature%20and%20genome%20size%20in%20bacteria%20are%20negatively%20correlated%2C%20suggesting%20genomic%20streamlining%20during%20thermal%20adaptation%2C&amp;author=N.%20Sabath&amp;author=E.%20Ferrada&amp;author=A.%20Barve&amp;author=A.%20Wagner&amp;publication_year=2013&amp;journal=Genome%20Biol.%20Evol&amp;volume=5&amp;pages=966-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evt050" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevt050" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevt050"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23563968" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Karcagi</div> <div class="given-names">I.</div></div><div class="name string-name"><div class="surname">Draskovits</div> <div class="given-names">G.</div></div><div class="name string-name"><div class="surname">Umenhoffer</div> <div class="given-names">K.</div></div></div>, et al.  . <div class="year">2016</div>, 
<div class="article-title">Indispensability of horizontally transferred genes and its impact on bacterial genome streamlining,</div> <div class="source">Mol. Biol. Evol</div> , <div class="volume">33</div>, <div class="fpage">1257</div>–<div class="lpage">69</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Indispensability%20of%20horizontally%20transferred%20genes%20and%20its%20impact%20on%20bacterial%20genome%20streamlining%2C&amp;author=I.%20Karcagi&amp;author=G.%20Draskovits&amp;author=K.%20Umenhoffer&amp;publication_year=2016&amp;journal=Mol.%20Biol.%20Evol&amp;volume=33&amp;pages=1257-69" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msw009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsw009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsw009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26769030" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nahku</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Valgepea</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Lahtvee</div> <div class="given-names">P.J.</div></div></div>, et al.  . <div class="year">2010</div>, 
<div class="article-title">Specific growth rate dependent transcriptome profiling of <em>Escherichia coli</em> K12 MG1655 in accelerostat cultures,</div> <div class="source">J. Biotechnol</div> ., <div class="volume">145</div>, <div class="fpage">60</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Specific%20growth%20rate%20dependent%20transcriptome%20profiling%20of%20Escherichia%20coli%20K12%20MG1655%20in%20accelerostat%20cultures%2C&amp;author=R.%20Nahku&amp;author=K.%20Valgepea&amp;author=P.J.%20Lahtvee&amp;publication_year=2010&amp;journal=J.%20Biotechnol&amp;volume=145&amp;pages=60-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jbiotec.2009.10.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jbiotec.2009.10.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jbiotec.2009.10.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19861135" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ying</div> <div class="given-names">B.W.</div></div><div class="name string-name"><div class="surname">Yama</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Kitahara</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Yomo</div> <div class="given-names">T.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">The <em>Escherichia coli</em> transcriptome linked to growth fitness,</div> <div class="source">Genomics Data</div> , <div class="volume">7</div>, <div class="fpage">1</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Escherichia%20coli%20transcriptome%20linked%20to%20growth%20fitness%2C&amp;author=B.W.%20Ying&amp;author=K.%20Yama&amp;author=K.%20Kitahara&amp;author=T.%20Yomo&amp;publication_year=2016&amp;journal=Genomics%20Data&amp;volume=7&amp;pages=1-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gdata.2015.11.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gdata.2015.11.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gdata.2015.11.011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26981347" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Murakami</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Matsumoto</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Tsuru</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Ying</div> <div class="given-names">B.W.</div></div><div class="name string-name"><div class="surname">Yomo</div> <div class="given-names">T.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Global coordination in adaptation to gene rewiring,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">43</div>, <div class="fpage">1304</div>–<div class="lpage">16</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20coordination%20in%20adaptation%20to%20gene%20rewiring%2C&amp;author=Y.%20Murakami&amp;author=Y.%20Matsumoto&amp;author=S.%20Tsuru&amp;author=B.W.%20Ying&amp;author=T.%20Yomo&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res&amp;volume=43&amp;pages=1304-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku1366" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku1366" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku1366"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25564530" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Matsumoto</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Murakami</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Tsuru</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Ying</div> <div class="given-names">B.W.</div></div><div class="name string-name"><div class="surname">Yomo</div> <div class="given-names">T.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Growth rate-coordinated transcriptome reorganization in bacteria,</div> <div class="source">BMC Genomics</div> , <div class="volume">14</div>, <div class="fpage">808</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Growth%20rate-coordinated%20transcriptome%20reorganization%20in%20bacteria%2C&amp;author=Y.%20Matsumoto&amp;author=Y.%20Murakami&amp;author=S.%20Tsuru&amp;author=B.W.%20Ying&amp;author=T.%20Yomo&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=808" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-808" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-808" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-808"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24252326" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ying</div> <div class="given-names">B.W.</div></div><div class="name string-name"><div class="surname">Matsumoto</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Kitahara</div> <div class="given-names">K.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">Bacterial transcriptome reorganization in thermal adaptive evolution,</div> <div class="source">BMC Genomics</div> , <div class="volume">16</div>, <div class="fpage">802</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacterial%20transcriptome%20reorganization%20in%20thermal%20adaptive%20evolution%2C&amp;author=B.W.%20Ying&amp;author=Y.%20Matsumoto&amp;author=K.%20Kitahara&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=802" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-1999-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-1999-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-1999-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26474851" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hashimoto</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Ichimura</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Mizoguchi</div> <div class="given-names">H.</div></div></div>, et al.  . <div class="year">2005</div>, 
<div class="article-title">Cell size and nucleoid organization of engineered <em>Escherichia coli</em> cells with a reduced genome,</div> <div class="source">Mol. Microbial</div> ., <div class="volume">55</div>, <div class="fpage">137</div>–<div class="lpage">49</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cell%20size%20and%20nucleoid%20organization%20of%20engineered%20Escherichia%20coli%20cells%20with%20a%20reduced%20genome%2C&amp;author=M.%20Hashimoto&amp;author=T.%20Ichimura&amp;author=H.%20Mizoguchi&amp;publication_year=2005&amp;journal=Mol.%20Microbial&amp;volume=55&amp;pages=137-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-2958.2004.04386.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-2958.2004.04386.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-2958.2004.04386.x"> </span></div> </div></div></div></div></div><div content-id="dsw035-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ying</div> <div class="given-names">B.W.</div></div><div class="name string-name"><div class="surname">Seno</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Kaneko</div> <div class="given-names">F.</div></div><div class="name string-name"><div class="surname">Matsuda</div> <div class="given-names">H.</div></div><div class="name string-name"><div class="surname">Yomo</div> <div class="given-names">T.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Multilevel comparative analysis of the contributions of genome reduction and heat shock to the <em>Escherichia coli</em> transcriptome,</div> <div class="source">BMC Genomics</div> , <div class="volume">14</div>, <div class="fpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multilevel%20comparative%20analysis%20of%20the%20contributions%20of%20genome%20reduction%20and%20heat%20shock%20to%20the%20Escherichia%20coli%20transcriptome%2C&amp;author=B.W.%20Ying&amp;author=S.%20Seno&amp;author=F.%20Kaneko&amp;author=H.%20Matsuda&amp;author=T.%20Yomo&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-25" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-25" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-25"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23324527" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ishizawa</div> <div class="given-names">Y.</div></div><div class="name string-name"><div class="surname">Ying</div> <div class="given-names">B.W.</div></div><div class="name string-name"><div class="surname">Tsuru</div> <div class="given-names">S.</div></div><div class="name string-name"><div class="surname">Yomo</div> <div class="given-names">T.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Nutrient-dependent growth defects and mutability of mutators in <em>Escherichia coli</em>,</div> <div class="source">Genes Cells</div> , <div class="volume">20</div>, <div class="fpage">68</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nutrient-dependent%20growth%20defects%20and%20mutability%20of%20mutators%20in%20Escherichia%20coli%2C&amp;author=Y.%20Ishizawa&amp;author=B.W.%20Ying&amp;author=S.%20Tsuru&amp;author=T.%20Yomo&amp;publication_year=2015&amp;journal=Genes%20Cells&amp;volume=20&amp;pages=68-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/gtc.2015.20.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fgtc.2015.20.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fgtc.2015.20.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25378049" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Akaike</div> <div class="given-names">H.</div></div></div> <div class="year">1974</div>, 
<div class="article-title">A new look at the statistical model identification,</div> <div class="source">IEEE Trans. Automatic Control</div> , <div class="volume">19</div>, <div class="fpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20look%20at%20the%20statistical%20model%20identification%2C&amp;author=H.%20Akaike&amp;publication_year=1974&amp;journal=IEEE%20Trans.%20Automatic%20Control&amp;volume=19&amp;pages=8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1109/TAC.1974.1100483" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1109%2FTAC.1974.1100483" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1109%2FTAC.1974.1100483"> </span></div> </div></div></div></div></div><div content-id="dsw035-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Aho</div> <div class="given-names">K.</div></div><div class="name string-name"><div class="surname">Derryberry</div> <div class="given-names">D.</div></div><div class="name string-name"><div class="surname">Peterson</div> <div class="given-names">T.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Model selection for ecologists: the worldviews of AIC and BIC,</div> <div class="source">Ecology</div> , <div class="volume">95</div>, <div class="fpage">631</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Model%20selection%20for%20ecologists%3A%20the%20worldviews%20of%20AIC%20and%20BIC%2C&amp;author=K.%20Aho&amp;author=D.%20Derryberry&amp;author=T.%20Peterson&amp;publication_year=2014&amp;journal=Ecology&amp;volume=95&amp;pages=631-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1890/13-1452.1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1890%2F13-1452.1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1890%2F13-1452.1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24804445" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ihaka</div> <div class="given-names">R.</div></div><div class="name string-name"><div class="surname">Gentleman</div> <div class="given-names">R.</div></div></div> <div class="year">1996</div>, 
<div class="article-title">R: a language for data analysis and graphics,</div> <div class="source">J. Comput. Graph. Stat</div> ., <div class="volume">5</div>, <div class="fpage">299</div>–<div class="lpage">314</div>.</div></div></div></div><div content-id="dsw035-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fox</div> <div class="given-names">J.</div></div><div class="name string-name"><div class="surname">Weisberg</div> <div class="given-names">S.</div></div></div> <div class="year">2011</div>, <div class="source">An R Companion to Applied Regression</div> , 
<div class="publisher-name">Sage</div>, 
<div class="publisher-loc">Thousand Oaks CA</div>.</div></div></div></div><div content-id="dsw035-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Riley</div> <div class="given-names">M.</div></div><div class="name string-name"><div class="surname">Abe</div> <div class="given-names">T.</div></div><div class="name string-name"><div class="surname">Arnaud</div> <div class="given-names">M.B.</div></div></div>, et al.  . <div class="year">2006</div>, 
<div class="article-title"><em>Escherichia coli</em> K-12: a cooperatively developed annotation snapshot–2005,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">34</div>, <div class="fpage">1</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Escherichia%20coli%20K-12%3A%20a%20cooperatively%20developed%20annotation%20snapshot%E2%80%932005%2C&amp;author=M.%20Riley&amp;author=T.%20Abe&amp;author=M.B.%20Arnaud&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res&amp;volume=34&amp;pages=1-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkj405" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkj405" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkj405"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16397293" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Alikhan</div> <div class="given-names">N.F.</div></div><div class="name string-name"><div class="surname">Petty</div> <div class="given-names">N.K.</div></div><div class="name string-name"><div class="surname">Ben Zakour</div> <div class="given-names">N.L.</div></div><div class="name string-name"><div class="surname">Beatson</div> <div class="given-names">S.A.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">BLAST Ring Image Generator (BRIG): simple prokaryote genome comparisons,</div> <div class="source">BMC Genomics</div> , <div class="volume">12</div>, <div class="fpage">402</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BLAST%20Ring%20Image%20Generator%20(BRIG)%3A%20simple%20prokaryote%20genome%20comparisons%2C&amp;author=N.F.%20Alikhan&amp;author=N.K.%20Petty&amp;author=N.L.%20Ben%20Zakour&amp;author=S.A.%20Beatson&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-402" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-402" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-402"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21824423" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lawrence</div> <div class="given-names">J.G.</div></div><div class="name string-name"><div class="surname">Ochman</div> <div class="given-names">H.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">Molecular archaeology of the <em>Escherichia coli</em> genome,</div> <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">95</div>, <div class="fpage">9413</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20archaeology%20of%20the%20Escherichia%20coli%20genome%2C&amp;author=J.G.%20Lawrence&amp;author=H.%20Ochman&amp;publication_year=1998&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=95&amp;pages=9413-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.16.9413" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.16.9413" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.16.9413"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9689094" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Puigbo</div> <div class="given-names">P.</div></div><div class="name string-name"><div class="surname">Lobkovsky</div> <div class="given-names">A.E.</div></div><div class="name string-name"><div class="surname">Kristensen</div> <div class="given-names">D.M.</div></div><div class="name string-name"><div class="surname">Wolf</div> <div class="given-names">Y.I.</div></div><div class="name string-name"><div class="surname">Koonin</div> <div class="given-names">E.V.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Genomes in turmoil: quantification of genome dynamics in prokaryote supergenomes,</div> <div class="source">BMC Biol</div> ., <div class="volume">12</div>, <div class="fpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomes%20in%20turmoil%3A%20quantification%20of%20genome%20dynamics%20in%20prokaryote%20supergenomes%2C&amp;author=P.%20Puigbo&amp;author=A.E.%20Lobkovsky&amp;author=D.M.%20Kristensen&amp;author=Y.I.%20Wolf&amp;author=E.V.%20Koonin&amp;publication_year=2014&amp;journal=BMC%20Biol&amp;volume=12&amp;pages=66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12915-014-0066-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12915-014-0066-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12915-014-0066-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25141959" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Novichkov</div> <div class="given-names">P.S.</div></div><div class="name string-name"><div class="surname">Omelchenko</div> <div class="given-names">M.V.</div></div><div class="name string-name"><div class="surname">Gelfand</div> <div class="given-names">M.S.</div></div><div class="name string-name"><div class="surname">Mironov</div> <div class="given-names">A.A.</div></div><div class="name string-name"><div class="surname">Wolf</div> <div class="given-names">Y.I.</div></div><div class="name string-name"><div class="surname">Koonin</div> <div class="given-names">E.V.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Genome-wide molecular clock and horizontal gene transfer in bacterial evolution,</div> <div class="source">J. Bacteriol</div> ., <div class="volume">186</div>, <div class="fpage">6575</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20molecular%20clock%20and%20horizontal%20gene%20transfer%20in%20bacterial%20evolution%2C&amp;author=P.S.%20Novichkov&amp;author=M.V.%20Omelchenko&amp;author=M.S.%20Gelfand&amp;author=A.A.%20Mironov&amp;author=Y.I.%20Wolf&amp;author=E.V.%20Koonin&amp;publication_year=2004&amp;journal=J.%20Bacteriol&amp;volume=186&amp;pages=6575-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.186.19.6575-6585.2004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.186.19.6575-6585.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.186.19.6575-6585.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15375139" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Koonin</div> <div class="given-names">E.V.</div></div><div class="name string-name"><div class="surname">Wolf</div> <div class="given-names">Y.I.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Genomics of bacteria and archaea: the emerging dynamic view of the prokaryotic world,</div> <div class="source">Nucleic Acids Res</div> ., <div class="volume">36</div>, <div class="fpage">6688</div>–<div class="lpage">719</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomics%20of%20bacteria%20and%20archaea%3A%20the%20emerging%20dynamic%20view%20of%20the%20prokaryotic%20world%2C&amp;author=E.V.%20Koonin&amp;author=Y.I.%20Wolf&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res&amp;volume=36&amp;pages=6688-719" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn668" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn668" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn668"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18948295" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kurland</div> <div class="given-names">C.G.</div></div><div class="name string-name"><div class="surname">Canback</div> <div class="given-names">B.</div></div><div class="name string-name"><div class="surname">Berg</div> <div class="given-names">O.G.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Horizontal gene transfer: a critical view,</div> <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">100</div>, <div class="fpage">9658</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Horizontal%20gene%20transfer%3A%20a%20critical%20view%2C&amp;author=C.G.%20Kurland&amp;author=B.%20Canback&amp;author=O.G.%20Berg&amp;publication_year=2003&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=100&amp;pages=9658-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1632870100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1632870100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1632870100"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12902542" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw035-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw035-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ying</div> <div class="given-names">B.W.</div></div><div class="name string-name"><div class="surname">Yomo</div> <div class="given-names">T.</div></div></div> <div class="year">2016</div>, Comparative analyses of bacterial transcriptome reorganisation in response to temperature increase. In: <div class="person-group"><div class="name string-name"><div class="surname">Bruijn</div> <div class="given-names">F.D.</div></div></div> (ed.), <div class="source">Stress and Environmental Control of Gene Expression in Bacteria</div> , 
<div class="publisher-name">Wiley-VCH</div>, 
<div class="publisher-loc">Weinheim</div>.</div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2647442&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/6/10.1093_dnares_dsw035/2/dsw035_Supp.pdf?Expires=2147483647&amp;Signature=yZ2sO9VP2Z3TYSkUXP~QOTiGs27MRweE6AWrjAkM9HNiC9Gvjlx8POMjF8qk8LzqIpRL-Nvx1OEyGZYpsf72-JrDcHVfi~vBdLUCmHfcPfCwtACeE9Y5b-eu~lNdQ5TNotgue4AFDZPtd2UKgPwc88ZFLFmM-6Fim-Rh6qebLui7ixMu0rZrlaWQ--ByMqKk6u5Btmr0WkLvRP-M3PbG8EJEsFDGSlUtKadVKMtkFO~-uRbgucy-VgNpi3L2sG4g5ynwG7dcp-7r1xkolCwxGosoelpENGXM6PfKgknd8ZmhUlKCiAUcCDUPq6snL6ckRnM0rSqta1nBlMMlcQ2XHQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,193</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000392723300002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">8</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw035"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2647442"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000392723300002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f23%2f6%2f517%2f2647442">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000392723300002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (8)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F23%2F6%2F517%2F2647442" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2647442'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Correlation between genome reduction and bacterial growth" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw035");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f23%2f6%2f517%2f2647442">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

