

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/15/5/297/359537","@type":"ScholarlyArticle","name":"Construction of Signature-tagged Mutant Library in  Mesorhizobium loti  as a Powerful Tool for Functional Genomics","datePublished":"2008-07-25","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/15/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2008-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsn017","keywords":["<italic>Mesorhizobium loti</italic>","signature-tagged mutagenesis","mutant library","reverse genetics"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/15/5/297/359537","author":[{"name":"Shimoda, Yoshikazu","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Mitsui, Hisayuki","affiliation":"2  Graduate School of Life Sciences ,  Tohoku University, Katahira, Aoba-ku ,  Sendai 980-8577 ,  Japan","@type":"Person"},{"name":"Kamimatsuse, Hiroko","affiliation":"2  Graduate School of Life Sciences ,  Tohoku University, Katahira, Aoba-ku ,  Sendai 980-8577 ,  Japan","@type":"Person"},{"name":"Minamisawa, Kiwamu","affiliation":"2  Graduate School of Life Sciences ,  Tohoku University, Katahira, Aoba-ku ,  Sendai 980-8577 ,  Japan","@type":"Person"},{"name":"Nishiyama, Eri","affiliation":"2  Graduate School of Life Sciences ,  Tohoku University, Katahira, Aoba-ku ,  Sendai 980-8577 ,  Japan","@type":"Person"},{"name":"Ohtsubo, Yoshiyuki","affiliation":"2  Graduate School of Life Sciences ,  Tohoku University, Katahira, Aoba-ku ,  Sendai 980-8577 ,  Japan","@type":"Person"},{"name":"Nagata, Yuji","affiliation":"2  Graduate School of Life Sciences ,  Tohoku University, Katahira, Aoba-ku ,  Sendai 980-8577 ,  Japan","@type":"Person"},{"name":"Tsuda, Masataka","affiliation":"2  Graduate School of Life Sciences ,  Tohoku University, Katahira, Aoba-ku ,  Sendai 980-8577 ,  Japan","@type":"Person"},{"name":"Shinpo, Sayaka","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Watanabe, Akiko","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kohara, Mitsuyo","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Yamada, Manabu","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Nakamura, Yasukazu","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Tabata, Satoshi","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Sato, Shusei","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"}],"description":"Abstract.  Rhizobia are nitrogen-fixing soil bacteria that establish endosymbiosis with some leguminous plants. The completion of several rhizobial genome seque","pageStart":"297","pageEnd":"308","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/5/1/m_cover.gif?Expires=2147483647&Signature=w0UhL1170rQznOI4EBmUQXu1ZOwiDy5D9yvI-xItfA5gEGy2mkrpZiqzDFtXeoJ~7GciRt1Jb~wykd96BMcXNK2oJMQKCvan9OGOux6ILlOeg1peDGvihwkEZeSlIEQRWsuy-Os5b2zaUJLN2EMIjSPATvDcEQeiR8VJm~tHNehbKW~jZGwxEfbLVfvvbsN-HCrfd6zxxRmciZJddcZFQjaH25Oc8zwhd9TcubBub~UwfgV64C6hjwxGjWayYeyC6503xComjsunKItnp20PuervJsOqbPEquDoZ8SVxa8ULqEXM-U2u8h~RKynzXZGyUfUdPZZhfeDp4GDC653uKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Construction of Signature-tagged Mutant Library in  Mesorhizobium loti  as a Powerful Tool for Functional Genomics","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/5/1/m_cover.gif?Expires=2147483647&Signature=w0UhL1170rQznOI4EBmUQXu1ZOwiDy5D9yvI-xItfA5gEGy2mkrpZiqzDFtXeoJ~7GciRt1Jb~wykd96BMcXNK2oJMQKCvan9OGOux6ILlOeg1peDGvihwkEZeSlIEQRWsuy-Os5b2zaUJLN2EMIjSPATvDcEQeiR8VJm~tHNehbKW~jZGwxEfbLVfvvbsN-HCrfd6zxxRmciZJddcZFQjaH25Oc8zwhd9TcubBub~UwfgV64C6hjwxGjWayYeyC6503xComjsunKItnp20PuervJsOqbPEquDoZ8SVxa8ULqEXM-U2u8h~RKynzXZGyUfUdPZZhfeDp4GDC653uKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Construction of Signature-tagged Mutant Library in  Mesorhizobium loti  as a Powerful Tool for Functional Genomics" />
    <meta property="og:description" content="Abstract.  Rhizobia are nitrogen-fixing soil bacteria that establish endosymbiosis with some leguminous plants. The completion of several rhizobial genome seque" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/15/5/297/359537" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2008-07-25" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Shimoda, Yoshikazu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Mitsui, Hisayuki" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, Katahira, Aoba-ku, Sendai 980-8577, Japan" /><meta name="citation_author" content="Kamimatsuse, Hiroko" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, Katahira, Aoba-ku, Sendai 980-8577, Japan" /><meta name="citation_author" content="Minamisawa, Kiwamu" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, Katahira, Aoba-ku, Sendai 980-8577, Japan" /><meta name="citation_author" content="Nishiyama, Eri" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, Katahira, Aoba-ku, Sendai 980-8577, Japan" /><meta name="citation_author" content="Ohtsubo, Yoshiyuki" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, Katahira, Aoba-ku, Sendai 980-8577, Japan" /><meta name="citation_author" content="Nagata, Yuji" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, Katahira, Aoba-ku, Sendai 980-8577, Japan" /><meta name="citation_author" content="Tsuda, Masataka" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, Katahira, Aoba-ku, Sendai 980-8577, Japan" /><meta name="citation_author" content="Shinpo, Sayaka" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Watanabe, Akiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kohara, Mitsuyo" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Yamada, Manabu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Nakamura, Yasukazu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Tabata, Satoshi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Sato, Shusei" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_title" content="Construction of Signature-tagged Mutant Library in Mesorhizobium loti as a Powerful Tool for Functional Genomics" /><meta name="citation_firstpage" content="297" /><meta name="citation_lastpage" content="308" /><meta name="citation_doi" content="10.1093/dnares/dsn017" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="15" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2008/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Complete genome structure of the nitrogen-fixing symbiotic bacterium Mesorhizobium loti; DNA Res.;  citation_year=2000;  citation_volume=7;  citation_pages=331-338; " /><meta name="citation_reference" content="citation_title=Complete genomic sequence of nitrogen-fixing symbiotic bacterium Bradyrhizobium japonicum USDA110; DNA Res.;  citation_year=2002;  citation_volume=9;  citation_pages=189-197; " /><meta name="citation_reference" content="citation_title=The composite genome of the legume symbiont Sinorhizobium meliloti; Science;  citation_year=2001;  citation_volume=293;  citation_pages=668-672; " /><meta name="citation_reference" content="citation_title=The partitioned Rhizobium etli genome: genetic and metabolic redundancy in seven interacting replicons; Proc. Natl. Acad. Sci. USA;  citation_year=2006;  citation_volume=103;  citation_pages=3834-3839; " /><meta name="citation_reference" content="citation_title=The genome of Rhizobium leguminosarum has recognizable core and accessory components; Genome Biol.;  citation_year=2006;  citation_volume=7;  citation_pages=R34" /><meta name="citation_reference" content="citation_title=Legumes symbioses: absence of Nod genes in photosynthetic bradyrhizobia; Science;  citation_year=2007;  citation_volume=316;  citation_pages=1307-1312; " /><meta name="citation_reference" content="citation_title=Development of Sinorhizobium meliloti pilot macroarrays for transcriptome analysis; Appl. Environ. Microbiol.;  citation_year=2003;  citation_volume=69;  citation_pages=1214-1219; " /><meta name="citation_reference" content="citation_title=A global analysis of protein expression profiles in Sinorhizobium meliloti: discovery of new genes for nodule occupancy and stress adaptation; Mol. Plant Microbe. Interact.;  citation_year=2003;  citation_volume=16;  citation_pages=508-524; " /><meta name="citation_reference" content="citation_title=A dual-genome Symbiosis Chip for coordinate study of signal exchange and development in a prokaryote–host interaction; Proc. Natl. Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=16636-16641; " /><meta name="citation_reference" content="citation_title=Characterization of bacteroid proteins in soybean nodules formed with Bradyrhizobium japonicum USDA110; Microbiol. Environ.;  citation_year=2004;  citation_volume=19;  citation_pages=71-75; " /><meta name="citation_reference" content="citation_title=Expression islands clustered on symbiosis island of Mesorhizobium loti genome; J. Bacteriol.;  citation_year=2004;  citation_volume=45;  citation_pages=2439-2448; " /><meta name="citation_reference" content="citation_title=Global protein expression pattern of Bradyrhizobium japonicum bacteroids: a prelude to functional proteomics; Proteomics;  citation_year=2005;  citation_volume=5;  citation_pages=4170-4184; " /><meta name="citation_reference" content="citation_title=A large scale analysis of protein–protein interactions in the nitrogen-fixing bacterium Mesorhizobium loti; DNA Res.;  citation_year=2008;  citation_volume=15;  citation_pages=13-23; " /><meta name="citation_reference" content="citation_title=Global transposon mutagenesis and a minimal mycoplasma genome; Science;  citation_year=1999;  citation_volume=286;  citation_pages=2165-2169; " /><meta name="citation_reference" content="citation_title=Essential Bacillus subtilis genes; Proc. Natl. Acad. Sci. USA;  citation_year=2003;  citation_volume=100;  citation_pages=4678-4683; " /><meta name="citation_reference" content="citation_title=Systematic mutagenesis of the Escherichia coli genome; J. Bacteriol.;  citation_year=2004;  citation_volume=186;  citation_pages=4921-4930; " /><meta name="citation_reference" content="citation_title=Construction of Escherichia coli K-12 in-frame, single-gene knockout mutants: the Keio collection; Mol. Syst. Biol.;  citation_year=2006;  citation_volume=2;  citation_pages=0008" /><meta name="citation_reference" content="citation_title=Essential genes of a minimal bacterium; Proc. Natl. Acad. Sci. USA;  citation_year=2006;  citation_volume=103;  citation_pages=425-430; " /><meta name="citation_reference" content="citation_title=Signature-tagged mutagenesis: technical advances in a negative selection method for virulence gene identification; Curr. Opin. Microbiol.;  citation_year=2005;  citation_volume=8;  citation_pages=612-619; " /><meta name="citation_reference" content="citation_title=Construction of a large signature-tagged Mini-Tn5 transposon library and its application to mutagenesis of Sinorhizobium meliloti; Appl. Environ. Microbiol.;  citation_year=2006;  citation_volume=72;  citation_pages=4329-4337; " /><meta name="citation_reference" content="citation_title=Identification of genes relevant to symbiosis and competitiveness in Sinorhizobium meliloti using signature-tagged mutants; Mol. Plant Microbe. Interact.;  citation_year=2008;  citation_volume=21;  citation_pages=219-231; " /><meta name="citation_reference" content="citation_title=Surface polysaccharide involvement in establishing the rhizobium-legume symbiosis; FEBS J.;  citation_year=2003;  citation_volume=270;  citation_pages=1365-1380; " /><meta name="citation_reference" content="citation_title=Nodule development induced by Mesorhizobium loti mutant strains affected in polysaccharide synthesis; Mol. Plant Microbe. Interact.;  citation_year=2005;  citation_volume=18;  citation_pages=446-457; " /><meta name="citation_reference" content="citation_title=The Mesorhizobium loti purB gene is involved in infection thread formation and nodule development in Lotus japonicus; J. Bacteriol.;  citation_year=2007;  citation_volume=189;  citation_pages=8347-8352; " /><meta name="citation_reference" content="citation_title=Evolutionary, structural and functional relationships revealed by comparative analysis of syntenic genes in Rhizobiales; BMC Evol. Biol.;  citation_year=2005;  citation_volume=5;  citation_pages=55" /><meta name="citation_reference" content="citation_title=Second symbiotic megaplasmid in Rhizobium meliloti carrying exopolysaccharide and thiamine synthesis genes; J. Bacteriol.;  citation_year=1986;  citation_volume=167;  citation_pages=66-72; " /><meta name="citation_reference" content="citation_title=R factor transfer in Rhizobium leguminosarum; J. Gen. Microbiol.;  citation_year=1974;  citation_volume=84;  citation_pages=188-198; " /><meta name="citation_reference" content="citation_title=Plasposon: modular self-clonig minitransposon derivatives for rapid genetic analysis of Gram-negative bacterial genomes; Appl. Environ. Microbiol.;  citation_year=1998;  citation_volume=64;  citation_pages=2710-2715; " /><meta name="citation_reference" content="citation_title=In vitro insertional mutagenesis with a selectable DNA fragment; Gene;  citation_year=1984;  citation_volume=29;  citation_pages=303-313; " /><meta name="citation_reference" content="citation_title=A broad host range mobilization system for in vivo genetic engineering: transposon mutagenesis in gram negative bacteria; Bio/Technology;  citation_year=1983;  citation_volume=1;  citation_pages=784-791; " /><meta name="citation_reference" content="citation_title=Identification of Burkholderia cenocepacia genes required for bacterial survival in vivo; Infect. Immun.;  citation_year=2004;  citation_volume=72;  citation_pages=4010-4022; " /><meta name="citation_reference" content="citation_title=Lessons from signature-tagged mutagenesis on the infectious mechanisms of pathogenic bacteria; FEMS Microbiol. Rev.;  citation_year=2005;  citation_volume=29;  citation_pages=703-717; " /><meta name="citation_reference" content="citation_title=Tn5 as a model for understanding DNA transposition; Mol. Microbiol.;  citation_year=2003;  citation_volume=47;  citation_pages=1199-1206; " /><meta name="citation_reference" content="citation_title=A method adapting microarray technology for signature-tagged mutagenesis of Desulfovibrio desulfuricans G20 and Shewanella oneidensis MR-1 in anaerobic sediment survival experiments; Appl. Environ. Microbiol.;  citation_year=2005;  citation_volume=71;  citation_pages=7064-7074; " /><meta name="citation_reference" content="citation_title=High-throughput transposon mutagenesis of Corynebacterium glutamicum and construction of a single-gene disruptant mutant library; Appl. Environ. Microbiol.;  citation_year=2006;  citation_volume=72;  citation_pages=3750-3755; " /><meta name="citation_reference" content="citation_title=A comprehensive transposon mutant library of Francisella novicida, a bioweapon surrogate; Proc. Natl. Acad. Sci. USA;  citation_year=2007;  citation_volume=104;  citation_pages=1009-1014; " /><meta name="citation_reference" content="citation_title=Random mutagenesis in Corynebacterium glutamicum ATCC 13032 using an IS6100-based transposon vector identified the last unknown gene in the histidine biosynthesis pathway; BMC Genomics;  citation_year=2006;  citation_volume=7;  citation_pages=205" /><meta name="citation_reference" content="citation_title=Combinatorial pattern discovery in biological sequences: the TEIRESIAS algorithm; Bioinformatics;  citation_year=1998;  citation_volume=14;  citation_pages=55-67; " /><meta name="citation_reference" content="citation_title=Comprehensive transposon mutant library of Pseudomonas aeruginosa; Proc. Natl. Acad. Sci. USA;  citation_year=2003;  citation_volume=100;  citation_pages=14339-14344; " /><meta name="citation_reference" content="citation_title=An ordered, nonredundant library of Pseudomonas aeruginosa strain PA14 transposon insertion mutants; Proc. Natl. Acad. Sci. USA;  citation_year=2006;  citation_volume=103;  citation_pages=2833-2838; " /><meta name="citation_reference" content="citation_title=A complete collection of single-gene deletion mutants of Acinetobacter baylyi ADP1; Mol. Syst. Biol.;  citation_year=2008;  citation_volume=4;  citation_pages=174" /><meta name="citation_reference" content="citation_title=Experimental determination and system level analysis of essential genes in Escherichia coli MG1655; J. Bacteriol.;  citation_year=2003;  citation_volume=185;  citation_pages=5673-5684; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/15/5/297/1029210/dsn017.pdf" /><meta name="description" content="Abstract.  Rhizobia are nitrogen-fixing soil bacteria that establish endosymbiosis with some leguminous plants. The completion of several rhizobial genome seque" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/15/5/297/359537" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/15/5/297/359537" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/15/5/297/359537");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsn017';
    </script>
<title>
	Construction of Signature-tagged Mutant Library in Mesorhizobium loti as a Powerful Tool for Functional Genomics | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./359537" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BeFvPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTaGltb2RhLCBZb3NoaWthenUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTWl0c3VpLCBIaXNheXVraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBMaWZlIFNjaWVuY2VzLCBUb2hva3UgVW5pdmVyc2l0eSwgS2F0YWhpcmEsIEFvYmEta3UsIFNlbmRhaSA5ODAtODU3NywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLYW1pbWF0c3VzZSwgSGlyb2tvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIExpZmUgU2NpZW5jZXMsIFRvaG9rdSBVbml2ZXJzaXR5LCBLYXRhaGlyYSwgQW9iYS1rdSwgU2VuZGFpIDk4MC04NTc3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1pbmFtaXNhd2EsIEtpd2FtdSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBMaWZlIFNjaWVuY2VzLCBUb2hva3UgVW5pdmVyc2l0eSwgS2F0YWhpcmEsIEFvYmEta3UsIFNlbmRhaSA5ODAtODU3NywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOaXNoaXlhbWEsIEVyaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBMaWZlIFNjaWVuY2VzLCBUb2hva3UgVW5pdmVyc2l0eSwgS2F0YWhpcmEsIEFvYmEta3UsIFNlbmRhaSA5ODAtODU3NywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPaHRzdWJvLCBZb3NoaXl1a2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgTGlmZSBTY2llbmNlcywgVG9ob2t1IFVuaXZlcnNpdHksIEthdGFoaXJhLCBBb2JhLWt1LCBTZW5kYWkgOTgwLTg1NzcsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmFnYXRhLCBZdWppIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIExpZmUgU2NpZW5jZXMsIFRvaG9rdSBVbml2ZXJzaXR5LCBLYXRhaGlyYSwgQW9iYS1rdSwgU2VuZGFpIDk4MC04NTc3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRzdWRhLCBNYXNhdGFrYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBMaWZlIFNjaWVuY2VzLCBUb2hva3UgVW5pdmVyc2l0eSwgS2F0YWhpcmEsIEFvYmEta3UsIFNlbmRhaSA5ODAtODU3NywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTaGlucG8sIFNheWFrYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXYXRhbmFiZSwgQWtpa28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS29oYXJhLCBNaXRzdXlvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IllhbWFkYSwgTWFuYWJ1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik5ha2FtdXJhLCBZYXN1a2F6dSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYWJhdGEsIFNhdG9zaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2F0bywgU2h1c2VpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iQ29uc3RydWN0aW9uIG9mIFNpZ25hdHVyZS10YWdnZWQgTXV0YW50IExpYnJhcnkgaW4gTWVzb3JoaXpvYml1bSBsb3RpIGFzIGEgUG93ZXJmdWwgVG9vbCBmb3IgRnVuY3Rpb25hbCBHZW5vbWljcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjI5NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMzA4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNuMDE3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMDgvMTAvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGV0ZSBnZW5vbWUgc3RydWN0dXJlIG9mIHRoZSBuaXRyb2dlbi1maXhpbmcgc3ltYmlvdGljIGJhY3Rlcml1bSBNZXNvcmhpem9iaXVtIGxvdGk7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0zMzEtMzM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGV0ZSBnZW5vbWljIHNlcXVlbmNlIG9mIG5pdHJvZ2VuLWZpeGluZyBzeW1iaW90aWMgYmFjdGVyaXVtIEJyYWR5cmhpem9iaXVtIGphcG9uaWN1bSBVU0RBMTEwOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MTg5LTE5NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGNvbXBvc2l0ZSBnZW5vbWUgb2YgdGhlIGxlZ3VtZSBzeW1iaW9udCBTaW5vcmhpem9iaXVtIG1lbGlsb3RpOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI5MzsgIGNpdGF0aW9uX3BhZ2VzPTY2OC02NzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBwYXJ0aXRpb25lZCBSaGl6b2JpdW0gZXRsaSBnZW5vbWU6IGdlbmV0aWMgYW5kIG1ldGFib2xpYyByZWR1bmRhbmN5IGluIHNldmVuIGludGVyYWN0aW5nIHJlcGxpY29uczsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTAzOyAgY2l0YXRpb25fcGFnZXM9MzgzNC0zODM5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIG9mIFJoaXpvYml1bSBsZWd1bWlub3NhcnVtIGhhcyByZWNvZ25pemFibGUgY29yZSBhbmQgYWNjZXNzb3J5IGNvbXBvbmVudHM7IEdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9UjM0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGVndW1lcyBzeW1iaW9zZXM6IGFic2VuY2Ugb2YgTm9kIGdlbmVzIGluIHBob3Rvc3ludGhldGljIGJyYWR5cmhpem9iaWE7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzE2OyAgY2l0YXRpb25fcGFnZXM9MTMwNy0xMzEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBvZiBTaW5vcmhpem9iaXVtIG1lbGlsb3RpIHBpbG90IG1hY3JvYXJyYXlzIGZvciB0cmFuc2NyaXB0b21lIGFuYWx5c2lzOyBBcHBsLiBFbnZpcm9uLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTY5OyAgY2l0YXRpb25fcGFnZXM9MTIxNC0xMjE5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGdsb2JhbCBhbmFseXNpcyBvZiBwcm90ZWluIGV4cHJlc3Npb24gcHJvZmlsZXMgaW4gU2lub3JoaXpvYml1bSBtZWxpbG90aTogZGlzY292ZXJ5IG9mIG5ldyBnZW5lcyBmb3Igbm9kdWxlIG9jY3VwYW5jeSBhbmQgc3RyZXNzIGFkYXB0YXRpb247IE1vbC4gUGxhbnQgTWljcm9iZS4gSW50ZXJhY3QuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9NTA4LTUyNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBkdWFsLWdlbm9tZSBTeW1iaW9zaXMgQ2hpcCBmb3IgY29vcmRpbmF0ZSBzdHVkeSBvZiBzaWduYWwgZXhjaGFuZ2UgYW5kIGRldmVsb3BtZW50IGluIGEgcHJva2FyeW90ZeKAk2hvc3QgaW50ZXJhY3Rpb247IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwMTsgIGNpdGF0aW9uX3BhZ2VzPTE2NjM2LTE2NjQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpemF0aW9uIG9mIGJhY3Rlcm9pZCBwcm90ZWlucyBpbiBzb3liZWFuIG5vZHVsZXMgZm9ybWVkIHdpdGggQnJhZHlyaGl6b2JpdW0gamFwb25pY3VtIFVTREExMTA7IE1pY3JvYmlvbC4gRW52aXJvbi47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz03MS03NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwcmVzc2lvbiBpc2xhbmRzIGNsdXN0ZXJlZCBvbiBzeW1iaW9zaXMgaXNsYW5kIG9mIE1lc29yaGl6b2JpdW0gbG90aSBnZW5vbWU7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NDU7ICBjaXRhdGlvbl9wYWdlcz0yNDM5LTI0NDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdsb2JhbCBwcm90ZWluIGV4cHJlc3Npb24gcGF0dGVybiBvZiBCcmFkeXJoaXpvYml1bSBqYXBvbmljdW0gYmFjdGVyb2lkczogYSBwcmVsdWRlIHRvIGZ1bmN0aW9uYWwgcHJvdGVvbWljczsgUHJvdGVvbWljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9NDE3MC00MTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGxhcmdlIHNjYWxlIGFuYWx5c2lzIG9mIHByb3RlaW7igJNwcm90ZWluIGludGVyYWN0aW9ucyBpbiB0aGUgbml0cm9nZW4tZml4aW5nIGJhY3Rlcml1bSBNZXNvcmhpem9iaXVtIGxvdGk7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MTMtMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdsb2JhbCB0cmFuc3Bvc29uIG11dGFnZW5lc2lzIGFuZCBhIG1pbmltYWwgbXljb3BsYXNtYSBnZW5vbWU7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9Mjg2OyAgY2l0YXRpb25fcGFnZXM9MjE2NS0yMTY5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fc3NlbnRpYWwgQmFjaWxsdXMgc3VidGlsaXMgZ2VuZXM7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEwMDsgIGNpdGF0aW9uX3BhZ2VzPTQ2NzgtNDY4MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3lzdGVtYXRpYyBtdXRhZ2VuZXNpcyBvZiB0aGUgRXNjaGVyaWNoaWEgY29saSBnZW5vbWU7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTg2OyAgY2l0YXRpb25fcGFnZXM9NDkyMS00OTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgRXNjaGVyaWNoaWEgY29saSBLLTEyIGluLWZyYW1lLCBzaW5nbGUtZ2VuZSBrbm9ja291dCBtdXRhbnRzOiB0aGUgS2VpbyBjb2xsZWN0aW9uOyBNb2wuIFN5c3QuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0wMDA4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXNzZW50aWFsIGdlbmVzIG9mIGEgbWluaW1hbCBiYWN0ZXJpdW07IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTEwMzsgIGNpdGF0aW9uX3BhZ2VzPTQyNS00MzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpZ25hdHVyZS10YWdnZWQgbXV0YWdlbmVzaXM6IHRlY2huaWNhbCBhZHZhbmNlcyBpbiBhIG5lZ2F0aXZlIHNlbGVjdGlvbiBtZXRob2QgZm9yIHZpcnVsZW5jZSBnZW5lIGlkZW50aWZpY2F0aW9uOyBDdXJyLiBPcGluLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz02MTItNjE5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgYSBsYXJnZSBzaWduYXR1cmUtdGFnZ2VkIE1pbmktVG41IHRyYW5zcG9zb24gbGlicmFyeSBhbmQgaXRzIGFwcGxpY2F0aW9uIHRvIG11dGFnZW5lc2lzIG9mIFNpbm9yaGl6b2JpdW0gbWVsaWxvdGk7IEFwcGwuIEVudmlyb24uIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NzI7ICBjaXRhdGlvbl9wYWdlcz00MzI5LTQzMzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIGdlbmVzIHJlbGV2YW50IHRvIHN5bWJpb3NpcyBhbmQgY29tcGV0aXRpdmVuZXNzIGluIFNpbm9yaGl6b2JpdW0gbWVsaWxvdGkgdXNpbmcgc2lnbmF0dXJlLXRhZ2dlZCBtdXRhbnRzOyBNb2wuIFBsYW50IE1pY3JvYmUuIEludGVyYWN0LjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTIxOS0yMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN1cmZhY2UgcG9seXNhY2NoYXJpZGUgaW52b2x2ZW1lbnQgaW4gZXN0YWJsaXNoaW5nIHRoZSByaGl6b2JpdW0tbGVndW1lIHN5bWJpb3NpczsgRkVCUyBKLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0yNzA7ICBjaXRhdGlvbl9wYWdlcz0xMzY1LTEzODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5vZHVsZSBkZXZlbG9wbWVudCBpbmR1Y2VkIGJ5IE1lc29yaGl6b2JpdW0gbG90aSBtdXRhbnQgc3RyYWlucyBhZmZlY3RlZCBpbiBwb2x5c2FjY2hhcmlkZSBzeW50aGVzaXM7IE1vbC4gUGxhbnQgTWljcm9iZS4gSW50ZXJhY3QuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9NDQ2LTQ1NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIE1lc29yaGl6b2JpdW0gbG90aSBwdXJCIGdlbmUgaXMgaW52b2x2ZWQgaW4gaW5mZWN0aW9uIHRocmVhZCBmb3JtYXRpb24gYW5kIG5vZHVsZSBkZXZlbG9wbWVudCBpbiBMb3R1cyBqYXBvbmljdXM7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTg5OyAgY2l0YXRpb25fcGFnZXM9ODM0Ny04MzUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb25hcnksIHN0cnVjdHVyYWwgYW5kIGZ1bmN0aW9uYWwgcmVsYXRpb25zaGlwcyByZXZlYWxlZCBieSBjb21wYXJhdGl2ZSBhbmFseXNpcyBvZiBzeW50ZW5pYyBnZW5lcyBpbiBSaGl6b2JpYWxlczsgQk1DIEV2b2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz01NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlY29uZCBzeW1iaW90aWMgbWVnYXBsYXNtaWQgaW4gUmhpem9iaXVtIG1lbGlsb3RpIGNhcnJ5aW5nIGV4b3BvbHlzYWNjaGFyaWRlIGFuZCB0aGlhbWluZSBzeW50aGVzaXMgZ2VuZXM7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODY7ICBjaXRhdGlvbl92b2x1bWU9MTY3OyAgY2l0YXRpb25fcGFnZXM9NjYtNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVIgZmFjdG9yIHRyYW5zZmVyIGluIFJoaXpvYml1bSBsZWd1bWlub3NhcnVtOyBKLiBHZW4uIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5NzQ7ICBjaXRhdGlvbl92b2x1bWU9ODQ7ICBjaXRhdGlvbl9wYWdlcz0xODgtMTk4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QbGFzcG9zb246IG1vZHVsYXIgc2VsZi1jbG9uaWcgbWluaXRyYW5zcG9zb24gZGVyaXZhdGl2ZXMgZm9yIHJhcGlkIGdlbmV0aWMgYW5hbHlzaXMgb2YgR3JhbS1uZWdhdGl2ZSBiYWN0ZXJpYWwgZ2Vub21lczsgQXBwbC4gRW52aXJvbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT02NDsgIGNpdGF0aW9uX3BhZ2VzPTI3MTAtMjcxNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW4gdml0cm8gaW5zZXJ0aW9uYWwgbXV0YWdlbmVzaXMgd2l0aCBhIHNlbGVjdGFibGUgRE5BIGZyYWdtZW50OyBHZW5lOyAgY2l0YXRpb25feWVhcj0xOTg0OyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9MzAzLTMxMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBicm9hZCBob3N0IHJhbmdlIG1vYmlsaXphdGlvbiBzeXN0ZW0gZm9yIGluIHZpdm8gZ2VuZXRpYyBlbmdpbmVlcmluZzogdHJhbnNwb3NvbiBtdXRhZ2VuZXNpcyBpbiBncmFtIG5lZ2F0aXZlIGJhY3RlcmlhOyBCaW8vVGVjaG5vbG9neTsgIGNpdGF0aW9uX3llYXI9MTk4MzsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9Nzg0LTc5MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgQnVya2hvbGRlcmlhIGNlbm9jZXBhY2lhIGdlbmVzIHJlcXVpcmVkIGZvciBiYWN0ZXJpYWwgc3Vydml2YWwgaW4gdml2bzsgSW5mZWN0LiBJbW11bi47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NzI7ICBjaXRhdGlvbl9wYWdlcz00MDEwLTQwMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxlc3NvbnMgZnJvbSBzaWduYXR1cmUtdGFnZ2VkIG11dGFnZW5lc2lzIG9uIHRoZSBpbmZlY3Rpb3VzIG1lY2hhbmlzbXMgb2YgcGF0aG9nZW5pYyBiYWN0ZXJpYTsgRkVNUyBNaWNyb2Jpb2wuIFJldi47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz03MDMtNzE3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UbjUgYXMgYSBtb2RlbCBmb3IgdW5kZXJzdGFuZGluZyBETkEgdHJhbnNwb3NpdGlvbjsgTW9sLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTQ3OyAgY2l0YXRpb25fcGFnZXM9MTE5OS0xMjA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG1ldGhvZCBhZGFwdGluZyBtaWNyb2FycmF5IHRlY2hub2xvZ3kgZm9yIHNpZ25hdHVyZS10YWdnZWQgbXV0YWdlbmVzaXMgb2YgRGVzdWxmb3ZpYnJpbyBkZXN1bGZ1cmljYW5zIEcyMCBhbmQgU2hld2FuZWxsYSBvbmVpZGVuc2lzIE1SLTEgaW4gYW5hZXJvYmljIHNlZGltZW50IHN1cnZpdmFsIGV4cGVyaW1lbnRzOyBBcHBsLiBFbnZpcm9uLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTcxOyAgY2l0YXRpb25fcGFnZXM9NzA2NC03MDc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoLXRocm91Z2hwdXQgdHJhbnNwb3NvbiBtdXRhZ2VuZXNpcyBvZiBDb3J5bmViYWN0ZXJpdW0gZ2x1dGFtaWN1bSBhbmQgY29uc3RydWN0aW9uIG9mIGEgc2luZ2xlLWdlbmUgZGlzcnVwdGFudCBtdXRhbnQgbGlicmFyeTsgQXBwbC4gRW52aXJvbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT03MjsgIGNpdGF0aW9uX3BhZ2VzPTM3NTAtMzc1NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb21wcmVoZW5zaXZlIHRyYW5zcG9zb24gbXV0YW50IGxpYnJhcnkgb2YgRnJhbmNpc2VsbGEgbm92aWNpZGEsIGEgYmlvd2VhcG9uIHN1cnJvZ2F0ZTsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTA0OyAgY2l0YXRpb25fcGFnZXM9MTAwOS0xMDE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SYW5kb20gbXV0YWdlbmVzaXMgaW4gQ29yeW5lYmFjdGVyaXVtIGdsdXRhbWljdW0gQVRDQyAxMzAzMiB1c2luZyBhbiBJUzYxMDAtYmFzZWQgdHJhbnNwb3NvbiB2ZWN0b3IgaWRlbnRpZmllZCB0aGUgbGFzdCB1bmtub3duIGdlbmUgaW4gdGhlIGhpc3RpZGluZSBiaW9zeW50aGVzaXMgcGF0aHdheTsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0yMDUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21iaW5hdG9yaWFsIHBhdHRlcm4gZGlzY292ZXJ5IGluIGJpb2xvZ2ljYWwgc2VxdWVuY2VzOiB0aGUgVEVJUkVTSUFTIGFsZ29yaXRobTsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz01NS02NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcHJlaGVuc2l2ZSB0cmFuc3Bvc29uIG11dGFudCBsaWJyYXJ5IG9mIFBzZXVkb21vbmFzIGFlcnVnaW5vc2E7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEwMDsgIGNpdGF0aW9uX3BhZ2VzPTE0MzM5LTE0MzQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBvcmRlcmVkLCBub25yZWR1bmRhbnQgbGlicmFyeSBvZiBQc2V1ZG9tb25hcyBhZXJ1Z2lub3NhIHN0cmFpbiBQQTE0IHRyYW5zcG9zb24gaW5zZXJ0aW9uIG11dGFudHM7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTEwMzsgIGNpdGF0aW9uX3BhZ2VzPTI4MzMtMjgzODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb21wbGV0ZSBjb2xsZWN0aW9uIG9mIHNpbmdsZS1nZW5lIGRlbGV0aW9uIG11dGFudHMgb2YgQWNpbmV0b2JhY3RlciBiYXlseWkgQURQMTsgTW9sLiBTeXN0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MTc0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwZXJpbWVudGFsIGRldGVybWluYXRpb24gYW5kIHN5c3RlbSBsZXZlbCBhbmFseXNpcyBvZiBlc3NlbnRpYWwgZ2VuZXMgaW4gRXNjaGVyaWNoaWEgY29saSBNRzE2NTU7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTg1OyAgY2l0YXRpb25fcGFnZXM9NTY3My01Njg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTUvNS8yOTcvMTAyOTIxMC9kc24wMTcucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFJoaXpvYmlhIGFyZSBuaXRyb2dlbi1maXhpbmcgc29pbCBiYWN0ZXJpYSB0aGF0IGVzdGFibGlzaCBlbmRvc3ltYmlvc2lzIHdpdGggc29tZSBsZWd1bWlub3VzIHBsYW50cy4gVGhlIGNvbXBsZXRpb24gb2Ygc2V2ZXJhbCByaGl6b2JpYWwgZ2Vub21lIHNlcXVlIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zNTk1MzcmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2Rkk2Pa/OYaIAFfWjFFRcB9ovL5658=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f5%2f297%2f359537" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f5%2f297%2f359537" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13976" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/15/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/5/1/m_cover.gif?Expires=2147483647&amp;Signature=w0UhL1170rQznOI4EBmUQXu1ZOwiDy5D9yvI-xItfA5gEGy2mkrpZiqzDFtXeoJ~7GciRt1Jb~wykd96BMcXNK2oJMQKCvan9OGOux6ILlOeg1peDGvihwkEZeSlIEQRWsuy-Os5b2zaUJLN2EMIjSPATvDcEQeiR8VJm~tHNehbKW~jZGwxEfbLVfvvbsN-HCrfd6zxxRmciZJddcZFQjaH25Oc8zwhd9TcubBub~UwfgV64C6hjwxGjWayYeyC6503xComjsunKItnp20PuervJsOqbPEquDoZ8SVxa8ULqEXM-U2u8h~RKynzXZGyUfUdPZZhfeDp4GDC653uKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/15/5">
                        <div class="volume trailing-comma">Volume 15</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2008        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5183309">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5183309">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5183311">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5183311">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5183317">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5183317">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5183334">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5183334">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5183364">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5183364">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5183366">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5183366">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5183368">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5183368">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/15/5/285/359655">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/15/5/309/359388">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Construction of Signature-tagged Mutant Library in <em>Mesorhizobium loti</em> as a Powerful Tool for Functional Genomics
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshikazu Shimoda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshikazu+Shimoda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shimoda Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shimoda Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yoshikazu Shimoda</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hisayuki Mitsui</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Graduate School of Life Sciences</div>, <div class="insititution">Tohoku University, Katahira, Aoba-ku</div>, <div class="addr-line">Sendai 980-8577</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hisayuki+Mitsui">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mitsui H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mitsui H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hisayuki Mitsui</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroko Kamimatsuse</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Graduate School of Life Sciences</div>, <div class="insititution">Tohoku University, Katahira, Aoba-ku</div>, <div class="addr-line">Sendai 980-8577</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroko+Kamimatsuse">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kamimatsuse H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kamimatsuse H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hiroko Kamimatsuse</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kiwamu Minamisawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Graduate School of Life Sciences</div>, <div class="insititution">Tohoku University, Katahira, Aoba-ku</div>, <div class="addr-line">Sendai 980-8577</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kiwamu+Minamisawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Minamisawa K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Minamisawa K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kiwamu Minamisawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Eri Nishiyama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Graduate School of Life Sciences</div>, <div class="insititution">Tohoku University, Katahira, Aoba-ku</div>, <div class="addr-line">Sendai 980-8577</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Eri+Nishiyama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nishiyama E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nishiyama E&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Eri Nishiyama</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshiyuki Ohtsubo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Graduate School of Life Sciences</div>, <div class="insititution">Tohoku University, Katahira, Aoba-ku</div>, <div class="addr-line">Sendai 980-8577</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshiyuki+Ohtsubo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ohtsubo Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ohtsubo Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yoshiyuki Ohtsubo</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuji Nagata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Graduate School of Life Sciences</div>, <div class="insititution">Tohoku University, Katahira, Aoba-ku</div>, <div class="addr-line">Sendai 980-8577</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuji+Nagata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nagata Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nagata Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yuji Nagata</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masataka Tsuda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Graduate School of Life Sciences</div>, <div class="insititution">Tohoku University, Katahira, Aoba-ku</div>, <div class="addr-line">Sendai 980-8577</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masataka+Tsuda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tsuda M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tsuda M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Masataka Tsuda</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sayaka Shinpo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sayaka+Shinpo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shinpo S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shinpo S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Sayaka Shinpo</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akiko Watanabe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akiko+Watanabe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Watanabe A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Watanabe A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Akiko Watanabe</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mitsuyo Kohara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mitsuyo+Kohara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kohara M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kohara M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Mitsuyo Kohara</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Manabu Yamada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Manabu+Yamada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamada M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yamada M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Manabu Yamada</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yasukazu Nakamura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yasukazu+Nakamura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakamura Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakamura Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yasukazu Nakamura</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoshi Tabata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoshi+Tabata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tabata S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tabata S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Satoshi Tabata</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shusei Sato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 438-52-3935. Fax. <div class="fax">+81 438-52-3934</div>. E-mail: <a href="mailto:ssato@kazusa.or.jp" target="_blank">ssato@kazusa.or.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shusei+Sato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sato S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sato S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Shusei Sato</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Naotake Ogasawara</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 15, Issue 5, 1 October 2008, Pages 297–308, <a href='https://doi.org/10.1093/dnares/dsn017'>https://doi.org/10.1093/dnares/dsn017</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">25 July 2008</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">30 April 2008</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">25 June 2008</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="359537" href="/dnaresearch/article-pdf/15/5/297/1029210/dsn017.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Yoshikazu Shimoda, Hisayuki Mitsui, Hiroko Kamimatsuse, Kiwamu Minamisawa, Eri Nishiyama, Yoshiyuki Ohtsubo, Yuji Nagata, Masataka Tsuda, Sayaka Shinpo, Akiko Watanabe, Mitsuyo Kohara, Manabu Yamada, Yasukazu Nakamura, Satoshi Tabata, Shusei Sato;  Construction of Signature-tagged Mutant Library in <em>Mesorhizobium loti</em> as a Powerful Tool for Functional Genomics, <em>DNA Research</em>, Volume 15, Issue 5, 1 October 2008, Pages 297–308, <a href='https://doi.org/10.1093/dnares/dsn017'>https://doi.org/10.1093/dnares/dsn017</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/359537?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/359537?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/359537?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/359537?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/359537?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/359537?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/359537?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Construction%20of%20Signature-tagged%20Mutant%20Library%20in%20%26lt%3Bem%26gt%3BMesorhizobium%20loti%26lt%3B%2Fem%26gt%3B%20as%20a%20Powerful%20Tool%20for%20Functional%20Genomics&amp;publicationDate=2008-07-25&amp;volumeNum=15&amp;issueNum=5&amp;author=Shimoda%2C%20Yoshikazu%3B%20Mitsui%2C%20Hisayuki&amp;startPage=297&amp;endPage=308&amp;contentId=10.1093%2Fdnares%2Fdsn017&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f5%2f297%2f359537" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5183309 id="5183309" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Rhizobia are nitrogen-fixing soil bacteria that establish endosymbiosis with some leguminous plants. The completion of several rhizobial genome sequences provides opportunities for genome-wide functional studies of the physiological roles of many rhizobial genes. In order to carry out genome-wide phenotypic screenings, we have constructed a large mutant library of the nitrogen-fixing symbiotic bacterium, <em>Mesorhizobium loti</em>, by transposon mutagenesis. Transposon insertion mutants were generated using the signature-tagged mutagenesis (STM) technique and a total of 29 330 independent mutants were obtained. Along with the collection of transposon mutants, we have determined the transposon insertion sites for 7892 clones, and confirmed insertions in 3680 non-redundant <em>M</em>. <em>loti</em> genes (50.5% of the total number of <em>M</em>. <em>loti</em> genes). Transposon insertions were randomly distributed throughout the <em>M</em>. <em>loti</em> genome without any bias toward G+C contents of insertion target sites and transposon plasmids used for the mutagenesis. We also show the utility of STM mutants by examining the specificity of signature tags and test screenings for growth- and nodulation-deficient mutants. This defined mutant library allows for genome-wide forward- and reverse-genetic functional studies of <em>M</em>. <em>loti</em> and will serve as an invaluable resource for researchers to further our understanding of rhizobial biology.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Mesorhizobium loti&quot;"><em>Mesorhizobium loti</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;signature-tagged mutagenesis&quot;">signature-tagged mutagenesis</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;mutant library&quot;">mutant library</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;reverse genetics&quot;">reverse genetics</a></div>                            <h2 scrollto-destination=5183311 id="5183311" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Nitrogen-fixing rhizobia are of great agronomic benefit, allowing many leguminous crops to be grown without nitrogen fertilizer by forming endosymbiotic relationships with the plants. In the course of the symbiosis, rhizobia elicit formation of specialized organs, ‘root nodules’, on the roots of compatible host legumes. Inside the nodules, the bacteria convert inert atmospheric dinitrogen into biologically usable ammonia. Since this symbiotic relationship is established by the results of highly regulated molecular dialogues between host plant and rhizobia, the system of symbiotic nitrogen fixation is therefore a very suitable model for studies of plant–microbe interactions.</p><p>The agronomic and biological importance of the rhizobia has accelerated the determination of the full genome sequences of several rhizobial species.<sup><a reveal-id="DSN017C1  DSN017C6" data-open="DSN017C1  DSN017C6" class="link link-ref link-reveal xref-bibr">1–6</a></sup> The availability of complete genome sequences has drastically changed the strategy for studying rhizobial genetics. Gene identification and functional assignment have been accelerated by utilizing the genome sequences. Simultaneously with the completions of rhizobial genome sequences, post-genomic researches on several rhizobia have rapidly progressed. Comprehenzsive transcriptome or proteome analyses have been conducted to examine physiological states of rhizobia under a variety of conditions, including symbiosis with the host legume or nutrient-depleted conditions.<sup><a reveal-id="DSN017C7  DSN017C12" data-open="DSN017C7  DSN017C12" class="link link-ref link-reveal xref-bibr">7–12</a></sup> In addition, we previously conducted a large-scale interactome analysis in order to propose functional relationships between known and unknown rhizobial genes.<sup><a reveal-id="DSN017C13" data-open="DSN017C13" class="link link-ref link-reveal xref-bibr">13</a></sup> These advances in rhizobial genetics have generated a vast amount of information about gene function and regulation; however, the advances have simultaneously raised the need for development of novel molecular tools which make use of the results of these functional analyses.</p><p>One of the most utilized genetic approaches to reveal gene function is the disruption of particular genes and subsequent phenotypic characterization of the mutants. Comprehensive collections of single gene disruptants have been systematically constructed in several model organisms by transposon random mutagenesis or targeted gene disruption approaches.<sup><a reveal-id="DSN017C14  DSN017C18" data-open="DSN017C14  DSN017C18" class="link link-ref link-reveal xref-bibr">14–18</a></sup> These analyses have contributed to the discovery of essential genes of the organisms and have provided fundamental tools for functional studies of many known and unknown genes. In rhizobia, recent genome-wide analyses have provided various intriguing and useful starting points for more detailed functional studies;<sup><a reveal-id="DSN017C7  DSN017C13" data-open="DSN017C7  DSN017C13" class="link link-ref link-reveal xref-bibr">7–13</a></sup> however, there are still large gaps between such systematic analyses and the availability of the corresponding gene mutants. Such gaps have resulted primarily because targeted gene disruption achieved by homologous recombination is not suitable for systematic generation of mutants in rhizobia because of its low efficiency. In addition, screening of mutants in some test conditions is problematic because phenotypes of individual mutants have to be evaluated one-by-one. Therefore, more effective approaches to generate large mutant collections and allow simultaneous screening of many mutants are needed.</p><p>In this study, we conducted the first large-scale random mutagenesis of <em>Mesorhizobium loti</em>, an endosymbiont of the model legume <em>Lotus japonicus</em>, using a signature-tagged mutagenesis (STM) technique. STM is based on transposon insertional mutagenesis that allows large numbers of mutants to be analyzed simultaneously. This is accomplished by tagging each mutant with a unique short DNA sequence so that it can subsequently be distinguished from other mutants by detecting unique signature tags. This system has been used with a variety of bacteria, and, in particular, has been applied to discovery of virulence genes in several microbial pathogens.<sup><a reveal-id="DSN017C19" data-open="DSN017C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Recently, a large STM library was constructed in <em>Sinorhizobium meliloti</em> strain 1021,<sup><a reveal-id="DSN017C20" data-open="DSN017C20" class="link link-ref link-reveal xref-bibr">20</a></sup> a symbiont of <em>Medicago truncatula</em> and contributed to the discovery of many genes relevant to symbiosis and competitiveness.<sup><a reveal-id="DSN017C21" data-open="DSN017C21" class="link link-ref link-reveal xref-bibr">21</a></sup> However, genome-wide generation of disruption mutants in <em>M</em>. <em>loti</em>, in addition to the set of <em>S</em>. <em>meliloti</em>, is also very valuable for several reasons. First, <em>M</em>. <em>loti</em> and <em>S</em>. <em>meliloti</em> exhibit different aspects as symbiont for legumes. They naturally exhibit distinct host-specificity and form different types of nodules (determinate and indeterminate nodules) on their respective model legume hosts, <em>L</em>. <em>japonicus</em> and <em>M</em>. <em>truncatula</em>. With respect to this symbiosis, previous studies have discovered several rhizobial factors and metabolic cycles required for nodulation and nitrogen-fixation.<sup><a reveal-id="DSN017C22  DSN017C24" data-open="DSN017C22  DSN017C24" class="link link-ref link-reveal xref-bibr">22–24</a></sup> However, comparative verifications of the significance of these factors on respective symbioses have not been fully explored. Second, comparative genomics analyses have revealed the existence of genes that are conserved, as well as many that are specific to each species,<sup><a reveal-id="DSN017C2" data-open="DSN017C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSN017C25" data-open="DSN017C25" class="link link-ref link-reveal xref-bibr">25</a></sup> but the molecular tools available for the functional comparisons of these genes are extremely limited. Therefore, large collections of disruption mutants on the individual species are necessary to reveal common and unique biological aspects of the two species.</p><p>In this report, we developed a large signature-tagged mutant library of <em>M</em>. <em>loti</em> in order to provide fundamental tools for functional studies of <em>M</em>. <em>loti</em> genes and characterized it with the gene coverage and transposon insertion distribution of the library. Sequencing of transposon insertion sites enabled us to collect defined sets of transposon mutants. Furthermore, pilot experiments showed the utility of the mutants and provide basic experimental conditions that can be used for future mutant screenings. This large signature-tagged mutant library will be an important and powerful resource for future functional genomics in <em>M</em>. <em>loti</em>.</p>                            <h2 scrollto-destination=5183317 id="5183317" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5183318 id="5183318" class="section-title" data-legacy-id=s2a>2.1. Bacterial strains and growth conditions</h3>
<p><em>Escherichia coli</em> strains DH5α and MT616 were used as hosts for the transposon plasmids and pRK600 helper plasmid,<sup><a reveal-id="DSN017C26" data-open="DSN017C26" class="link link-ref link-reveal xref-bibr">26</a></sup> respectively. Transposon insertion mutants were generated in the wild-type <em>M</em>. <em>loti</em> strain MAFF303099. <em>Mesorhizobium loti</em> was grown at 30°C in tryptone yeast (TY) extract medium<sup><a reveal-id="DSN017C27" data-open="DSN017C27" class="link link-ref link-reveal xref-bibr">27</a></sup> supplemented with phosphomycin (100 µg/ml). <em>Escherichia coli</em> was grown at 37°C in Luria-Bertani (LB) medium. LB medium was supplemented with streptomycin (100 µg/ml) and spectinomycin (100 µg/ml) for DH5α donor cells harboring transposon plasmids and with chloramphenicol (20 µg/ml) for MT616 helper cells. Transposon insertion mutants were selected on TY medium supplemented with streptomycin, spectinomycin and phosphomycin at the same concentration as above.</p>                            <h3 scrollto-destination=5183320 id="5183320" class="section-title" data-legacy-id=s2b>2.2. Construction of signature-tagged transposon plasmids</h3>
<p>The backbone of the transposon plasmid was p<em>TnMod</em>-OGm.<sup><a reveal-id="DSN017C28" data-open="DSN017C28" class="link link-ref link-reveal xref-bibr">28</a></sup> The Ω interposon cassette was excised from pHP45Ω<sup><a reveal-id="DSN017C29" data-open="DSN017C29" class="link link-ref link-reveal xref-bibr">29</a></sup> and inserted by replacing a <em>Sac</em>I fragment containing the gentamicin resistance gene cassette of p<em>TnMod</em>-OGm. The resultant construct was designated pTn<em>Mod</em>-OΩ. Twenty-seven different 21-base oligonucleotides (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>) were annealed to complementary molecules to make double-strand tags. Each oligonucleotide tag was inserted individually into the blunted <em>Kpn</em>I site of p<em>TnMod</em>-OΩ (Fig. <a reveal-id="DSN017F1" data-open="DSN017F1" class="link link-reveal link-table xref-fig">1</a>). These 27 signature-tagged constructs were designated pTn<em>Mod</em>-OΩ-<em>tag</em>1 to pTn<em>Mod</em>-OΩ-<em>tag</em>41.</p>                        <a id="5183322" scrollto-destination="5183322"></a>
<div id="DSN017F1" data-id="DSN017F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn017/2/m_dsn01701.jpeg?Expires=2147483647&Signature=0f6j27fkYvDYoXzylqD~43y~5bCjTM-xjIZ2be6CXxdfzLQVxK2k9lKD~nWwuV4eE079mspjmjHAsVWtb5Qu71YzevtHK-3wzEHUvsCp5sK-13gqlA6BOiCd-XsooFAaK-OoeU-zLgzcxheM0jrqBAio-l8-u6HBMT9sWe5w-2IleQ0BPkxdw8edvpPZf-l5fuoYgXu98UdSXCJi083YBh-YT5n74dlVCyRYg3oc3brX6g2fW4AvXT0iTHAXp5Jt8FTh3JqdR3OZpoK0xI4LU-YRHe8ohRAgfLF~HtVFg9Ql6HW7-r3g~RCrwjMQSKlefu1kcSkYjX4eomUtgE7N3w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Vector construct used for STM of M. loti. The tagged transposon plasmids were constructed from the backbone of pTnMod-OGm.28 The omega-interposon cassette (Spr/Smr) was excised from pHP45Ω29 and inserted into the SacI site of pTnMod-OGm. Oligonucleotide tags (21 bp) were incorporated into the KpnI site. Arrows indicate the locations of the STM common primer and the tag-specific primer (Supplementary Table S1). OriT is an RP4 origin of transfer. IR and tnp represent the inverted repeat and Tn5 transposase, respectively." path-from-xml="dsn01701.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5183322/dsn01701.jpeg" path-from-xml="dsn01701.jpeg" target="_blank">View large</a><a section="5183322" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn017/2/dsn01701.jpeg?Expires=2147483647&Signature=NL2FkxevdJoqm52TzhflMfri~r-~ASrEljDCAt2U7h-kmF2pldDLgwzPy~z3Z6TKtnK0w2w5gdT6Gml6HoWCjFnp8fPks-SisDwIUB~5gpcefZ-vbcNAd7TbI6DqNA5fWpDPLdF3DTM2H~47HdFRyJnkveuhnUK~FgEvXWFby6m6CBEACBu8SF62G~8L-ovQVHfBCCDiA2TxozRyJfPiTttKnrwC99rqCMv~SSRC3wXQlEm7-oci4YET-okMt~eqDp8Q9cGGcP9TPuQ85maUREj3MHyUUZxRptBBqkpH0J88-2ehOO-dAkjIelJ1C8YTX3lB2JY3hzPNIaSOEbfOhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5183322&ar=359537&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01701.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Vector construct used for STM of <em>M</em>. <em>loti</em>. The tagged transposon plasmids were constructed from the backbone of pTn<em>Mod</em>-OGm.<sup><a reveal-id="DSN017C28" data-open="DSN017C28" class="link link-ref link-reveal xref-bibr">28</a></sup> The omega-interposon cassette (Sp<sup>r</sup>/Sm<sup>r</sup>) was excised from pHP45Ω<sup><a reveal-id="DSN017C29" data-open="DSN017C29" class="link link-ref link-reveal xref-bibr">29</a></sup> and inserted into the <em>Sac</em>I site of pTn<em>Mod</em>-OGm. Oligonucleotide tags (21 bp) were incorporated into the <em>Kpn</em>I site. Arrows indicate the locations of the STM common primer and the tag-specific primer (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). OriT is an RP4 origin of transfer. IR and tnp represent the inverted repeat and Tn5 transposase, respectively.</p></div></div><div content-id="DSN017F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn017/2/m_dsn01701.jpeg?Expires=2147483647&Signature=0f6j27fkYvDYoXzylqD~43y~5bCjTM-xjIZ2be6CXxdfzLQVxK2k9lKD~nWwuV4eE079mspjmjHAsVWtb5Qu71YzevtHK-3wzEHUvsCp5sK-13gqlA6BOiCd-XsooFAaK-OoeU-zLgzcxheM0jrqBAio-l8-u6HBMT9sWe5w-2IleQ0BPkxdw8edvpPZf-l5fuoYgXu98UdSXCJi083YBh-YT5n74dlVCyRYg3oc3brX6g2fW4AvXT0iTHAXp5Jt8FTh3JqdR3OZpoK0xI4LU-YRHe8ohRAgfLF~HtVFg9Ql6HW7-r3g~RCrwjMQSKlefu1kcSkYjX4eomUtgE7N3w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Vector construct used for STM of M. loti. The tagged transposon plasmids were constructed from the backbone of pTnMod-OGm.28 The omega-interposon cassette (Spr/Smr) was excised from pHP45Ω29 and inserted into the SacI site of pTnMod-OGm. Oligonucleotide tags (21 bp) were incorporated into the KpnI site. Arrows indicate the locations of the STM common primer and the tag-specific primer (Supplementary Table S1). OriT is an RP4 origin of transfer. IR and tnp represent the inverted repeat and Tn5 transposase, respectively." path-from-xml="dsn01701.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5183322/dsn01701.jpeg" path-from-xml="dsn01701.jpeg" target="_blank">View large</a><a section="5183322" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn017/2/dsn01701.jpeg?Expires=2147483647&Signature=NL2FkxevdJoqm52TzhflMfri~r-~ASrEljDCAt2U7h-kmF2pldDLgwzPy~z3Z6TKtnK0w2w5gdT6Gml6HoWCjFnp8fPks-SisDwIUB~5gpcefZ-vbcNAd7TbI6DqNA5fWpDPLdF3DTM2H~47HdFRyJnkveuhnUK~FgEvXWFby6m6CBEACBu8SF62G~8L-ovQVHfBCCDiA2TxozRyJfPiTttKnrwC99rqCMv~SSRC3wXQlEm7-oci4YET-okMt~eqDp8Q9cGGcP9TPuQ85maUREj3MHyUUZxRptBBqkpH0J88-2ehOO-dAkjIelJ1C8YTX3lB2JY3hzPNIaSOEbfOhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5183322&ar=359537&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01701.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Vector construct used for STM of <em>M</em>. <em>loti</em>. The tagged transposon plasmids were constructed from the backbone of pTn<em>Mod</em>-OGm.<sup><a reveal-id="DSN017C28" data-open="DSN017C28" class="link link-ref link-reveal xref-bibr">28</a></sup> The omega-interposon cassette (Sp<sup>r</sup>/Sm<sup>r</sup>) was excised from pHP45Ω<sup><a reveal-id="DSN017C29" data-open="DSN017C29" class="link link-ref link-reveal xref-bibr">29</a></sup> and inserted into the <em>Sac</em>I site of pTn<em>Mod</em>-OGm. Oligonucleotide tags (21 bp) were incorporated into the <em>Kpn</em>I site. Arrows indicate the locations of the STM common primer and the tag-specific primer (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). OriT is an RP4 origin of transfer. IR and tnp represent the inverted repeat and Tn5 transposase, respectively.</p></div></div>                            <h3 scrollto-destination=5183323 id="5183323" class="section-title" data-legacy-id=s2c>2.3. Transposon mutagenesis</h3>
<p>Introduction of tagged transposon plasmids into <em>M</em>. <em>loti</em> was carried out by tri-parental mating.<sup><a reveal-id="DSN017C30" data-open="DSN017C30" class="link link-ref link-reveal xref-bibr">30</a></sup><em>Escherichia coli</em> strains DH5α and MT616 (containing the pRK600 helper plasmid) were used as the conjugation donor and helper, respectively. An overnight culture of donor and helper (0.5 ml each) was mixed with 1 ml of a 2-day culture of <em>M</em>. <em>loti</em>. After washing several times with sterilized water, the cell mixture was suspended in 50 µl of sterilized water and spotted onto a sterilized membrane filter (MILLIPORE, Billerica, MA, USA, pore size 0.45 µm, Cat. No. HAWG047S3) that was placed on TY agar medium without antibiotics. After an 8 h incubation on the membrane filter at 30°C, all the cells were collected and suspended in 1 ml of sterilized water. Suspended cells were plated on TY selection medium supplemented with phosphomycin, spectinomycin and streptomycin, and grown further until positive colonies appeared. Finally, individual positive colonies were picked up randomly in 96-well plates and stored at −80°C.</p>                            <h3 scrollto-destination=5183325 id="5183325" class="section-title" data-legacy-id=s2d>2.4. Determination of transposon insertion sites</h3>
<p>Transposon insertion sites were determined by direct sequencing of transposon-genome flanking region. Cell culture and genomic DNA isolation were carried out in 96-well plate format. Five microlitter of stocked mutant clones were inoculated into 1 ml of TY medium containing phosphomycin (100 µg/ml), streptomycin (100 µg/ml) and spectinomycin (100 µg/ml), and the cultures were grown in 96-well deep-well plate (Axygen Bioscience, Union City, CA, USA) at 30°C for 2 days. After centrifugation of cultured cells at 4000 rpm for 15 min, genomic DNA of mutants was prepared using AquaPure Genomic DNA isolation kit (Bio-Rad Laboratories, Tokyo, Japan). Collected cells in each well were suspended in 150 µl of Genomic DNA Lysis Solution and were incubated at 80°C for 15 min. RNaseA solution (0.75 µl) was added to cell lysate, and the mixture was incubated at 37°C for 60 min. After the incubation on ice for 15 min, 50 µl of protein precipitation solution was added to each well and the samples were mixed well by vortex. Samples were then transferred individually to 1.5 ml microcentrifuge tube and were centrifuged at 15 000 rpm for 15 min. Supernatants (100–125 µl) were transferred to new 96-well PCR plate containing 100 µl of isopropanol and were mixed well by inverting plate. Genomic DNA precipitated by centrifugation (4000 rpm, 30 min) was rinsed three times with 200 µl of 70% ethanol. Genomic DNA was then dried and dissolved in 20 µl of DNA hydration solution. After heating the samples at 65°C for 60 min, concentration of genomic DNA was measured and ∼2 µg of genomic DNA was used for direct sequence. Direct sequencing was performed with the STM-seq primer (5′-TTTGCTGGCCTTTTGCTCACATGTTCTTTC-3′) using the following PCR conditions: 98°C for 3 min, followed by 70 cycles of 97°C for 15 s and 60°C for 4 min. The composition of reaction mixture for direct sequence was as follows. Two micrograms of genomic DNA, 1 µl of STM-seq primer (3.2 µM), 1 µl of BigDye Terminator premix (Applied Biosystem, Foster City, CA, USA), 3.5 µl of 5× sequencing buffer (Applied Biosystems) and sterilized water were added up to 20 µl.</p><p>For some clones which were not successfully identified by direct sequencing, we determined the transposon integration sites by two-round inverse PCR. For inverse PCR, genomic DNA (∼1 µg) isolated from each mutant was digested with <em>Sal</em>I and <em>Xho</em>I, and then self-ligated using a DNA ligation kit (TAKARA Bio, Shiga, Japan). Using a self-ligated DNA template and a primer pair (5′-TTCGCCACCTCTGACTTGAGCGTCG-3′ and 5′-GAATTGATCCGGTGGATGAC-3′), first round PCR was performed using the following cycling conditions: 98°C for 2 min, followed by 35 cycles of 97°C for 15 s, 55°C for 45 s, 72°C for 3 min and a final extension at 72°C for 10 min. A 1 µl aliquot from the first round PCR product was added to a secondary PCR reaction mixture containing a nested primer pair (5′-TTTGCTGGCCTTTTGCTCACATGTTCTTTC-3′ and 5′-ACGGTTTACAAGCATAAAGC-3′). Second round PCR was performed using the following PCR conditions: 98°C for 2 min, followed by 35 cycles of 97°C for 15 s, 57°C for 45 s, 72°C for 3 min and a final extension at 72°C for 10 min. After the secondary PCR, products were purified by polyethylene glycol precipitation and suspended in 10 µl distilled water. The composition of sequence reaction mixture with amplified PCR product was as follows. Two microliters of PCR product, 1 µl of sequencing primer (5′-ACGGTTTACAAGCATAAAGC-3′) (3.2 µM), 1 µl of BigDye Terminator premix (Applied Biosystems), 3.5 µl of 5× sequencing buffer (Applied Biosystems) and 12.5 µl of sterilized water. Cycle sequencing was performed at 96°C for 1 min and then at 96°C for 10 s, 50°C for 5 s and 60°C for 4 min for 25 cycles. All sequencings were analyzed by ABI 3730 autosequencer. The resulting sequences were subsequently subjected to BlastN search against the <em>M</em>. <em>loti</em> genome sequence (<a class="link link-uri" href="http://www.kazusa.or.jp/rhizobase/Mesorhizobium/index.html">http://www.kazusa.or.jp/rhizobase/Mesorhizobium/index.html</a>) to determine the transposon insertion sites.</p>                            <h3 scrollto-destination=5183328 id="5183328" class="section-title" data-legacy-id=s2e>2.5. PCR and real-time PCR</h3>
<p>The specificity of oligonucleotide tags was examined by PCR. Tag-containing DNA fragments (160 bp) were amplified from genomic DNA using the Herculase II Fusion DNA Polymerase (STRATAGENE, La Jolla, CA, USA), and forward (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>) and reverse (STM-common, 5′-TTCGCCACCTCTGACTTGAGCGTCG-3′) primers were used for amplification. PCR reactions were conducted in GeneAmp PCR system 9700 (Applied Biosystems) using the following conditions: initial denaturation at 98°C for 2 min, followed by 25 cycles of 97°C for 15 s, 68°C for 1 min and a final extension at 72°C for 10 min. Amplified products were analyzed on 2% agarose gels.</p><p>The population of growth-deficient mutant within a mutant pool was examined by real-time PCR. Genomic DNA isolated from input and output pools was used as the DNA template. Real-time PCR was performed using a DyNAmo HS SYBR Green qPCR kit (Finnzyme, Espoo, Finland) and reactions were detected by a DNA Engine Opticon system (Bio-Rad). The conditions used for real-time PCR are as follows: 15 min at 95°C, followed by 35 cycles of 94°C for 30 s, 58°C for 30 s and 72°C for 30 s and a final extension at 72°C for 10 min. Quantitative and specificity (melting curve) analyses of amplified PCR products were carried out using associated software in the Opticon system (Opticon monitor3) by following the manufacture's protocol.</p>                            <h3 scrollto-destination=5183331 id="5183331" class="section-title" data-legacy-id=s2f>2.6. Detection of signature tags from plant samples</h3>
<p>Seeds of <em>L</em>. <em>japonicus</em> (MG-20 Miyakojima) were sterilized and germinated on half-strength B&amp;D agar medium. Six-day-old seedlings were transferred to B&amp;D agar medium or inoculation pots filled with vermiculite and were then inoculated with 1 × 10<sup>7</sup> cell/ml of mutant mixture.</p><p>For tag detection from nodules, detached nodules were washed twice with a solution of 0.5%-SDS-100 mM NaCl and then soaked in 70% ethanol for 5 min. After five times washes with sterilized water, total DNA was isolated using a DNeasy plant mini kit (Qiagen, Hilden, Germany). For amplification of a tag-containing fragment from a single nodule, nodule was washed as above and crushed in 700 µl sterilized water. After boiling the samples at 95°C for 5 min, 5 µl of supernatant was used for PCR. PCR reactions with nodule samples used the following conditions: initial denaturation at 98°Cfor 2 min, followed by 25 cycles of 97°C for 15 s, 68°C for 1 min and a final extension at 72°C for 10 min. Amplified products were analyzed on 2% agarose gels.</p>                            <h2 scrollto-destination=5183334 id="5183334" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=5183335 id="5183335" class="section-title" data-legacy-id=s3a>3.1. Construction of a signature-tagged mutant library</h3>
<p>In this study, we adapted transposon-based STM techniques to genome-wide mutagenesis of the symbiotic bacterium, <em>M</em>. <em>loti</em>. As a transposon construct, the plasposon plasmid pTn<em>Mod</em>-OGm which carries a Tn5-based mini-transposon and conditional origin of replication<sup><a reveal-id="DSN017C28" data-open="DSN017C28" class="link link-ref link-reveal xref-bibr">28</a></sup> was used for mutagenesis (Fig. <a reveal-id="DSN017F1" data-open="DSN017F1" class="link link-reveal link-table xref-fig">1</a>). In order to screen for genes which were inactivated by transposon insertion and those affected polarity effect of the inserted transposon, we introduced the Ω interposon cassette by replacing the gentamycin resistance gene (aacC1) of pTn<em>Mod</em>-OGm (Fig. <a reveal-id="DSN017F1" data-open="DSN017F1" class="link link-reveal link-table xref-fig">1</a>). Since pTn<em>Mod</em> is a broad host range suicide vector and the insertion of the Ω fragment confers spectinomycin and streptomycin resistance, only mutants in which the transposon fragment is integrated into the <em>M</em>. <em>loti</em> genome can be selected effectively by their antibiotic resistance.</p><p>In order to distinguish individual transposon mutants, we introduced 21 bp short oligonucleotide tags that have been used in a previous study.<sup><a reveal-id="DSN017C31" data-open="DSN017C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Since these oligonucleotide tags were designed to have similar G+C content and melting temperature, they are feasible for PCR-based mutant screening. Considering the isolation of symbiotic mutants from root nodules in which reduction of variation was expected, number of tag variation need to be restricted to obtain reliable results of screening from a reasonable number of plant materials (root nodules). Therefore, we further selected tags from the original 41 tags<sup><a reveal-id="DSN017C31" data-open="DSN017C31" class="link link-ref link-reveal xref-bibr">31</a></sup> by analyzing their specificity of PCR amplification. We selected a total of 29 unique tags based on the reproducibility of PCR amplification and the plasposon plasmids tagged with each of 29 tags (pTn<em>Mod</em>-OΩ-<em>tag</em>n) were then individually mobilized into wild-type <em>M</em>. <em>loti</em> by tri-parental mating. Among the 29 tags, two tag plasmids which yielded significantly small numbers of transposon mutant colonies were eliminated, and the remaining 27 tags (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>) were used for further analysis (name of tags are designated according to Hunt et al.<sup><a reveal-id="DSN017C31" data-open="DSN017C31" class="link link-ref link-reveal xref-bibr">31</a></sup>). In order to construct independent mutant libraries, tri-parental mating and collection of transposon mutants were carried out separately for each oligonucleotide tag. As the result of 63 independent tri-parental mating experiments, 29 330 independent transposon mutants were obtained (Table <a reveal-id="DSN017TB1" data-open="DSN017TB1" class="link link-reveal link-table xref-fig">1</a>). Individual mutants were stored in 96-well plates and mutant names (ID) were assigned according to their position in the plate with prefix indicating tag number followed by ‘T’ (i.e. 07T02d09 is a mutant integrated with tag7 plasmid).</p>                        <a id="5183338" scrollto-destination="5183338"></a>
<div content-id="DSN017TB1" class="table-modal"><div class="table-wrap"><div id="DSN017TB1" class="table-wrap-title" data-id="DSN017TB1"><div class="label -label">Table 1</div><div class="caption"><p>Summary of experimental results</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Mutants collected </td><td>29 330 </td></tr><tr><td>Mutants sequenced for mapping of transposon insertion site </td><td>9344 </td></tr><tr><td>Mapped insertion locations </td><td>7892 </td></tr><tr><td>Unique insertion locations </td><td>7586 </td></tr><tr><td>Insertion between ORFs (intergenic region) </td><td>1156 </td></tr><tr><td> Chromosome </td><td>1018 </td></tr><tr><td> pMLa </td><td>80 </td></tr><tr><td> pMLb </td><td>58 </td></tr><tr><td>Insertion inside ORFs </td><td>6430 </td></tr><tr><td> Chromosome </td><td>6000 </td></tr><tr><td> pMLa </td><td>318 </td></tr><tr><td> pMLb </td><td>112 </td></tr><tr><td>Insertion inside ORFs (non-redundant) </td><td>3680 </td></tr><tr><td>Genes supported by more than two mutant alleles </td><td>1592 </td></tr></tbody></table></div><div content-id="DSN017TB1" class="table-modal"><table><tbody><tr><td>Mutants collected </td><td>29 330 </td></tr><tr><td>Mutants sequenced for mapping of transposon insertion site </td><td>9344 </td></tr><tr><td>Mapped insertion locations </td><td>7892 </td></tr><tr><td>Unique insertion locations </td><td>7586 </td></tr><tr><td>Insertion between ORFs (intergenic region) </td><td>1156 </td></tr><tr><td> Chromosome </td><td>1018 </td></tr><tr><td> pMLa </td><td>80 </td></tr><tr><td> pMLb </td><td>58 </td></tr><tr><td>Insertion inside ORFs </td><td>6430 </td></tr><tr><td> Chromosome </td><td>6000 </td></tr><tr><td> pMLa </td><td>318 </td></tr><tr><td> pMLb </td><td>112 </td></tr><tr><td>Insertion inside ORFs (non-redundant) </td><td>3680 </td></tr><tr><td>Genes supported by more than two mutant alleles </td><td>1592 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5183338" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSN017TB1" class="table-wrap-title" data-id="DSN017TB1"><div class="label -label">Table 1</div><div class="caption"><p>Summary of experimental results</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Mutants collected </td><td>29 330 </td></tr><tr><td>Mutants sequenced for mapping of transposon insertion site </td><td>9344 </td></tr><tr><td>Mapped insertion locations </td><td>7892 </td></tr><tr><td>Unique insertion locations </td><td>7586 </td></tr><tr><td>Insertion between ORFs (intergenic region) </td><td>1156 </td></tr><tr><td> Chromosome </td><td>1018 </td></tr><tr><td> pMLa </td><td>80 </td></tr><tr><td> pMLb </td><td>58 </td></tr><tr><td>Insertion inside ORFs </td><td>6430 </td></tr><tr><td> Chromosome </td><td>6000 </td></tr><tr><td> pMLa </td><td>318 </td></tr><tr><td> pMLb </td><td>112 </td></tr><tr><td>Insertion inside ORFs (non-redundant) </td><td>3680 </td></tr><tr><td>Genes supported by more than two mutant alleles </td><td>1592 </td></tr></tbody></table></div><div content-id="DSN017TB1" class="table-modal"><table><tbody><tr><td>Mutants collected </td><td>29 330 </td></tr><tr><td>Mutants sequenced for mapping of transposon insertion site </td><td>9344 </td></tr><tr><td>Mapped insertion locations </td><td>7892 </td></tr><tr><td>Unique insertion locations </td><td>7586 </td></tr><tr><td>Insertion between ORFs (intergenic region) </td><td>1156 </td></tr><tr><td> Chromosome </td><td>1018 </td></tr><tr><td> pMLa </td><td>80 </td></tr><tr><td> pMLb </td><td>58 </td></tr><tr><td>Insertion inside ORFs </td><td>6430 </td></tr><tr><td> Chromosome </td><td>6000 </td></tr><tr><td> pMLa </td><td>318 </td></tr><tr><td> pMLb </td><td>112 </td></tr><tr><td>Insertion inside ORFs (non-redundant) </td><td>3680 </td></tr><tr><td>Genes supported by more than two mutant alleles </td><td>1592 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5183338" target="_blank">View Large</a></div></div><p>The coverage of the collected mutants is approximately fourfold with respect to the 7281 ORFs of <em>M</em>. <em>loti</em>.<sup><a reveal-id="DSN017C1" data-open="DSN017C1" class="link link-ref link-reveal xref-bibr">1</a></sup> If the transposon insertions are evenly distributed throughout the <em>M</em>. <em>loti</em> genome, the mean distance between independent insertions is estimated to be 259 bp, and thus 91% of <em>M</em>. <em>loti</em> genes are expected to carry at least one insertion since <em>M</em>. <em>loti</em> genes shorter than 259 bp account for 9% of all genes.</p>                            <h3 scrollto-destination=5183340 id="5183340" class="section-title" data-legacy-id=s3b>3.2. Utility of STM mutants</h3>
<p>The STM technique allows for the simultaneous screening of multiple mutants in mixed populations by detecting unique DNA tags. However, previous studies have reported that non-specific amplification or failure in tag amplification from mutant chromosomal DNA is a major inherent problem that affects the efficiency and reproducibility of the screening procedure.<sup><a reveal-id="DSN017C31" data-open="DSN017C31" class="link link-ref link-reveal xref-bibr">31</a>,<a reveal-id="DSN017C32" data-open="DSN017C32" class="link link-ref link-reveal xref-bibr">32</a></sup> Therefore, the specificity and efficiency of tag amplification after integration into the genome needs to be tested in the respective targeted organisms. To investigate the possibility of cross-detection among the 27 signature tags used in this study, we conducted test amplifications by PCR. Fig. <a reveal-id="DSN017F2" data-open="DSN017F2" class="link link-reveal link-table xref-fig">2</a>A shows representative results in which PCR was performed with genomic DNA from three individual tagged mutants (Tag2, Tag9 and Tag11). When a tag-specific primer (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>) and the STM common primer (5′-TTCGCCACCTCTGACTTGAGCGTCG-3′) that anneals 120 bp downstream of the integrated oligonucleotide tag were used for PCR, specific amplification was observed only in reactions containing the appropriate tag-specific primer. A series of PCR reactions performed on the other oligonucleotide tags produced the same results as Fig. <a reveal-id="DSN017F2" data-open="DSN017F2" class="link link-reveal link-table xref-fig">2</a>A, indicating that there are no cross-amplifications among the designed oligonucleotide tags. In addition, all 27 oligonucleotide tags were amplified specifically when genomic DNA from a mixture of mutants was used as the template (Fig. <a reveal-id="DSN017F2" data-open="DSN017F2" class="link link-reveal link-table xref-fig">2</a>B). These test amplifications demonstrated that the applied oligonucleotide tags can be recovered specifically from the <em>M</em>. <em>loti</em> genome and that the 27 individually tagged mutants can be distinguished from one another by detecting these signature tags.</p>                        <a id="5183342" scrollto-destination="5183342"></a>
<div id="DSN017F2" data-id="DSN017F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn017/2/m_dsn01702.jpeg?Expires=2147483647&Signature=Oy2o9qdg10GRhcYhEMdJsQTKw6latEhu90v17I-cCZvgfAVKZK5XZa0ebiig~DYjBfTeLKUkne1~U3lihsSeFZC2ip7ux31Re5F4hApnvoELUG59uR4Dc119MjDTCXJDB5btfGcMSEBRmlLUEQKnzhe3WclN0Rk2FUc5jLLCeievYhS~uKDhJBamnhnwazL0Hyux8kw6rSYODLOYgHLILgR~DHnqGBnVzL7-JLZTyOADu~ld0wejXNJITBhcfS-fpb0Hl3TQ23JUEFMssd0i0un8K3YBCDrMjAfehybeDrqo33YcYBpYdmZSzsDGCK-gtczWRIrlEj2SX75u~LgB3Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Representative gels of control experiments testing the specificity of signature oligonucleotide tags. PCR was performed with genomic DNA isolated from mutants with pTnMod-OΩ-Tag2 (upper), pTnMod-OΩ-Tag9 (middle) or pTnMod-OΩ-Tag11 (bottom) alone (A), and using genomic DNA isolated from cultures containing a mixture of 27 mutants (B). The STM-common primer was used with one of the 27 different tag-specific primers (Supplementary Table S1) in each reaction. The numbers indicate the combination of primers." path-from-xml="dsn01702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5183342/dsn01702.jpeg" path-from-xml="dsn01702.jpeg" target="_blank">View large</a><a section="5183342" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn017/2/dsn01702.jpeg?Expires=2147483647&Signature=mbzjnEqGpU4uUkh0TSCMiAAFmlQ0cjOafQrVt7TWKbuhMNeEj49pj7ZM8cLx-JnMlg1QZ4lYO7TAsD3td3umBQWKQWvPxEz8ppfG7q0zjbd8bxWWTz4TMKdZog5ureEZvgOwik9Ftr6xCnSBJ5j1v2Ia05oU3UqGuasrOqMOj-axpJTxG5we71xDuRB62GXJmVTlZCx~UDTMb54eCasHqLoh5af2cegbkzqSM5-DPa65Sw~UdTtLh0SeeblAYiX-fu0HyR13~Lmf1TXB5mWwXqbgfyF0xs5kf10oj3RfSdDoToxC7I7T9A4KZoCTZIZpCOZV0OSr7cQwOvj3Jdtzyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5183342&ar=359537&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Representative gels of control experiments testing the specificity of signature oligonucleotide tags. PCR was performed with genomic DNA isolated from mutants with pTn<em>Mod</em>-OΩ-<em>Tag</em>2 (upper), pTn<em>Mod</em>-OΩ-<em>Tag</em>9 (middle) or pTn<em>Mod</em>-OΩ-<em>Tag</em>11 (bottom) alone (<strong>A</strong>), and using genomic DNA isolated from cultures containing a mixture of 27 mutants (<strong>B</strong>). The STM-common primer was used with one of the 27 different tag-specific primers (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>) in each reaction. The numbers indicate the combination of primers.</p></div></div><div content-id="DSN017F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn017/2/m_dsn01702.jpeg?Expires=2147483647&Signature=Oy2o9qdg10GRhcYhEMdJsQTKw6latEhu90v17I-cCZvgfAVKZK5XZa0ebiig~DYjBfTeLKUkne1~U3lihsSeFZC2ip7ux31Re5F4hApnvoELUG59uR4Dc119MjDTCXJDB5btfGcMSEBRmlLUEQKnzhe3WclN0Rk2FUc5jLLCeievYhS~uKDhJBamnhnwazL0Hyux8kw6rSYODLOYgHLILgR~DHnqGBnVzL7-JLZTyOADu~ld0wejXNJITBhcfS-fpb0Hl3TQ23JUEFMssd0i0un8K3YBCDrMjAfehybeDrqo33YcYBpYdmZSzsDGCK-gtczWRIrlEj2SX75u~LgB3Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Representative gels of control experiments testing the specificity of signature oligonucleotide tags. PCR was performed with genomic DNA isolated from mutants with pTnMod-OΩ-Tag2 (upper), pTnMod-OΩ-Tag9 (middle) or pTnMod-OΩ-Tag11 (bottom) alone (A), and using genomic DNA isolated from cultures containing a mixture of 27 mutants (B). The STM-common primer was used with one of the 27 different tag-specific primers (Supplementary Table S1) in each reaction. The numbers indicate the combination of primers." path-from-xml="dsn01702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5183342/dsn01702.jpeg" path-from-xml="dsn01702.jpeg" target="_blank">View large</a><a section="5183342" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn017/2/dsn01702.jpeg?Expires=2147483647&Signature=mbzjnEqGpU4uUkh0TSCMiAAFmlQ0cjOafQrVt7TWKbuhMNeEj49pj7ZM8cLx-JnMlg1QZ4lYO7TAsD3td3umBQWKQWvPxEz8ppfG7q0zjbd8bxWWTz4TMKdZog5ureEZvgOwik9Ftr6xCnSBJ5j1v2Ia05oU3UqGuasrOqMOj-axpJTxG5we71xDuRB62GXJmVTlZCx~UDTMb54eCasHqLoh5af2cegbkzqSM5-DPa65Sw~UdTtLh0SeeblAYiX-fu0HyR13~Lmf1TXB5mWwXqbgfyF0xs5kf10oj3RfSdDoToxC7I7T9A4KZoCTZIZpCOZV0OSr7cQwOvj3Jdtzyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5183342&ar=359537&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Representative gels of control experiments testing the specificity of signature oligonucleotide tags. PCR was performed with genomic DNA isolated from mutants with pTn<em>Mod</em>-OΩ-<em>Tag</em>2 (upper), pTn<em>Mod</em>-OΩ-<em>Tag</em>9 (middle) or pTn<em>Mod</em>-OΩ-<em>Tag</em>11 (bottom) alone (<strong>A</strong>), and using genomic DNA isolated from cultures containing a mixture of 27 mutants (<strong>B</strong>). The STM-common primer was used with one of the 27 different tag-specific primers (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>) in each reaction. The numbers indicate the combination of primers.</p></div></div><p>In order to test the utility of our mutant sets, we carried out additional pilot experiments using both in vitro and in vivo conditions. In vitro experiments were conducted with a pool of mutants that contains a mutant with an attenuated growth on rich medium. To make a mutant set for the test experiment, one mutant was selected randomly from each 27-tag library and cultured individually. One growth-deficient clone (07T02d09) was included in the pool (Fig. <a reveal-id="DSN017F3" data-open="DSN017F3" class="link link-reveal link-table xref-fig">3</a>A). Independently cultured clones were then mixed at same concentration to make an input pool. As for an output pool, the input pool was diluted 500-fold and cultured further for 2 days. After isolation of genomic DNA from both pools, the tag population was examined by real-time PCR. Real-time PCR was performed using the SYBR Green detection system with the same primer set used to test the cross-reactivity (Fig. <a reveal-id="DSN017F2" data-open="DSN017F2" class="link link-reveal link-table xref-fig">2</a>). Fig. <a reveal-id="DSN017F3" data-open="DSN017F3" class="link link-reveal link-table xref-fig">3</a> illustrates the results of real-time PCR of the input (Fig. <a reveal-id="DSN017F3" data-open="DSN017F3" class="link link-reveal link-table xref-fig">3</a>B) and output (Fig. <a reveal-id="DSN017F3" data-open="DSN017F3" class="link link-reveal link-table xref-fig">3</a>C) pool samples. Consistent with the growth on agar medium (Fig. <a reveal-id="DSN017F3" data-open="DSN017F3" class="link link-reveal link-table xref-fig">3</a>A), the quantity of PCR product derived from the growth-deficient mutant (07T02d09) was significantly lower than the other tags which were all amplified to similar levels (Fig. <a reveal-id="DSN017F3" data-open="DSN017F3" class="link link-reveal link-table xref-fig">3</a>B and C). In addition, after each round of PCR amplification, a melting curve was generated for each run of 27 samples (Fig. <a reveal-id="DSN017F3" data-open="DSN017F3" class="link link-reveal link-table xref-fig">3</a>D). The melting curve of all reactions exhibited a sharp peak at ∼83.5°C and a low melting peak was observed in the reactions with the tag7 primer (Fig <a reveal-id="DSN017F3" data-open="DSN017F3" class="link link-reveal link-table xref-fig">3</a>D arrow), indicating that all tag-containing fragments were amplified specifically and that the quantity of tag7-containing fragment was lower than the other tags. This test experiment demonstrated that we can distinguish differences in populations of each tagged mutant within pools by quantifying the signature tags.</p>                        <a id="5183344" scrollto-destination="5183344"></a>
<div id="DSN017F3" data-id="DSN017F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn017/2/m_dsn01703.jpeg?Expires=2147483647&Signature=3E8lpd3Q~BAXe83YCjLRQo7wbRc8nA-Pwzhzets1x25ryQ7QOh1UU61oGbTr8jRtxSceZw-8ep8bIMnsDUZ7ZJ0KMJQkz8OfbVUJn1jmKrkMguktJj7blhzk3HIQ4APi~gj5CyyjLcgDOdSKDUXQ5S7aDTmyrJZKI7HuW3vep09~d~Z6h9dL6BysOGmlyFrwXDFncMxsbH35xAM8qnADmlMvpNE0AIe5hlNWopGJNzpH8PFVmfT6dFz6veKNOz5gLct2k6d82trFnEv8WzqPLyY6CO2B~ZuIlmBPzfSA6THt1zpzsiDPEDVYG1Jt3ONQR-5KRN-C06AGjQksDOJtUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Test experiments for identifying a growth-deficient mutant within a pool of mutants. (A) Growth of 27-tagged mutants on rich TY medium. Same concentration of individual mutants was spotted on TY broth medium. Mutant 07T2d09, integrated with pTnMod-OΩ-Tag7, exhibited reduced growth compared with other mutants. Quantitation of PCR amplifications of the input pool (B) and output pool (C). X and Y axes show the number of PCR cycles and log fluorescence of PCR products, respectively. Arrow in C indicates the quantitative curve from PCR reaction with Tag7 primer. The amounts of amplified products in all reactions of input pool were similar (ranging from 0.37- to 1.7-fold compare with average). In output pool, the amount of amplified products of all but tag7 were similar to that of input pool (ranging from 0.25- to 1.6-fold compared with that of input pool), whereas the amount in reaction of tag7 primer was 90-fold lower than that of input pool. (D) Melting curve of the output pool showing the change in fluorescence from 72°C to 91°C. All melting curves are shown as relative intensities among all reactions. Arrow in D indicates the melting curve of PCR reaction with Tag7 primer." path-from-xml="dsn01703.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5183344/dsn01703.jpeg" path-from-xml="dsn01703.jpeg" target="_blank">View large</a><a section="5183344" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn017/2/dsn01703.jpeg?Expires=2147483647&Signature=iejCYI5Cdad46kh0MCxsWFl6WoZHcS55Zk354vqcEqb9tfnKtZZVEqhHIFKsqjWKdW9hJ8BR6-uE9MzPihli-2MKOzHdmei-S0nZBtuYHC38UU9f-FVyyhLsg2oDmpqMjjFpTsQLi6TkdmG41O5WP6gVP-DnIqek0noaCwTMa9~uwBjDJteno3Ph-dFvMBLDIr15tn-4KoFnw8nRwPMO4ui-nwG0Y3wJqTPIX8iXv9bz32td9MvMpxfPO6J5wnQ91Li92zc0fnFgoaKYNb3vHrpMdS3Ym7tz3x0PY5c9JVAazf5XJrf3I-th4lAWQRXtnUgsWrLsVGBsWNotVr3Xpg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5183344&ar=359537&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01703.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Test experiments for identifying a growth-deficient mutant within a pool of mutants. (<strong>A</strong>) Growth of 27-tagged mutants on rich TY medium. Same concentration of individual mutants was spotted on TY broth medium. Mutant 07T2d09, integrated with pTn<em>Mod</em>-OΩ-Tag7, exhibited reduced growth compared with other mutants. Quantitation of PCR amplifications of the input pool (<strong>B</strong>) and output pool (<strong>C</strong>). <em>X</em> and <em>Y</em> axes show the number of PCR cycles and log fluorescence of PCR products, respectively. Arrow in C indicates the quantitative curve from PCR reaction with Tag7 primer. The amounts of amplified products in all reactions of input pool were similar (ranging from 0.37- to 1.7-fold compare with average). In output pool, the amount of amplified products of all but tag7 were similar to that of input pool (ranging from 0.25- to 1.6-fold compared with that of input pool), whereas the amount in reaction of tag7 primer was 90-fold lower than that of input pool. (<strong>D</strong>) Melting curve of the output pool showing the change in fluorescence from 72°C to 91°C. All melting curves are shown as relative intensities among all reactions. Arrow in D indicates the melting curve of PCR reaction with Tag7 primer.</p></div></div><div content-id="DSN017F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn017/2/m_dsn01703.jpeg?Expires=2147483647&Signature=3E8lpd3Q~BAXe83YCjLRQo7wbRc8nA-Pwzhzets1x25ryQ7QOh1UU61oGbTr8jRtxSceZw-8ep8bIMnsDUZ7ZJ0KMJQkz8OfbVUJn1jmKrkMguktJj7blhzk3HIQ4APi~gj5CyyjLcgDOdSKDUXQ5S7aDTmyrJZKI7HuW3vep09~d~Z6h9dL6BysOGmlyFrwXDFncMxsbH35xAM8qnADmlMvpNE0AIe5hlNWopGJNzpH8PFVmfT6dFz6veKNOz5gLct2k6d82trFnEv8WzqPLyY6CO2B~ZuIlmBPzfSA6THt1zpzsiDPEDVYG1Jt3ONQR-5KRN-C06AGjQksDOJtUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Test experiments for identifying a growth-deficient mutant within a pool of mutants. (A) Growth of 27-tagged mutants on rich TY medium. Same concentration of individual mutants was spotted on TY broth medium. Mutant 07T2d09, integrated with pTnMod-OΩ-Tag7, exhibited reduced growth compared with other mutants. Quantitation of PCR amplifications of the input pool (B) and output pool (C). X and Y axes show the number of PCR cycles and log fluorescence of PCR products, respectively. Arrow in C indicates the quantitative curve from PCR reaction with Tag7 primer. The amounts of amplified products in all reactions of input pool were similar (ranging from 0.37- to 1.7-fold compare with average). In output pool, the amount of amplified products of all but tag7 were similar to that of input pool (ranging from 0.25- to 1.6-fold compared with that of input pool), whereas the amount in reaction of tag7 primer was 90-fold lower than that of input pool. (D) Melting curve of the output pool showing the change in fluorescence from 72°C to 91°C. All melting curves are shown as relative intensities among all reactions. Arrow in D indicates the melting curve of PCR reaction with Tag7 primer." path-from-xml="dsn01703.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5183344/dsn01703.jpeg" path-from-xml="dsn01703.jpeg" target="_blank">View large</a><a section="5183344" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn017/2/dsn01703.jpeg?Expires=2147483647&Signature=iejCYI5Cdad46kh0MCxsWFl6WoZHcS55Zk354vqcEqb9tfnKtZZVEqhHIFKsqjWKdW9hJ8BR6-uE9MzPihli-2MKOzHdmei-S0nZBtuYHC38UU9f-FVyyhLsg2oDmpqMjjFpTsQLi6TkdmG41O5WP6gVP-DnIqek0noaCwTMa9~uwBjDJteno3Ph-dFvMBLDIr15tn-4KoFnw8nRwPMO4ui-nwG0Y3wJqTPIX8iXv9bz32td9MvMpxfPO6J5wnQ91Li92zc0fnFgoaKYNb3vHrpMdS3Ym7tz3x0PY5c9JVAazf5XJrf3I-th4lAWQRXtnUgsWrLsVGBsWNotVr3Xpg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5183344&ar=359537&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01703.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Test experiments for identifying a growth-deficient mutant within a pool of mutants. (<strong>A</strong>) Growth of 27-tagged mutants on rich TY medium. Same concentration of individual mutants was spotted on TY broth medium. Mutant 07T2d09, integrated with pTn<em>Mod</em>-OΩ-Tag7, exhibited reduced growth compared with other mutants. Quantitation of PCR amplifications of the input pool (<strong>B</strong>) and output pool (<strong>C</strong>). <em>X</em> and <em>Y</em> axes show the number of PCR cycles and log fluorescence of PCR products, respectively. Arrow in C indicates the quantitative curve from PCR reaction with Tag7 primer. The amounts of amplified products in all reactions of input pool were similar (ranging from 0.37- to 1.7-fold compare with average). In output pool, the amount of amplified products of all but tag7 were similar to that of input pool (ranging from 0.25- to 1.6-fold compared with that of input pool), whereas the amount in reaction of tag7 primer was 90-fold lower than that of input pool. (<strong>D</strong>) Melting curve of the output pool showing the change in fluorescence from 72°C to 91°C. All melting curves are shown as relative intensities among all reactions. Arrow in D indicates the melting curve of PCR reaction with Tag7 primer.</p></div></div><p>In consideration of future screening experiments that would be explored <em>in planta</em>, we next examined whether tagged mutants can also be detected from plant samples (root nodules). For these test experiments, one mutant from each of the 27 tag libraries was mixed in roughly equal quantities and inoculated onto <em>L</em>. <em>japonicus</em> roots to form nodules. Four weeks after inoculation, the nodules were detached and used for mutant detection. In order to determine how many mutants exist in a single nodule, we examined 22 nodules formed by the inoculation of two different pools of mutants. This confirmation enables us to estimate how many nodules should be analyzed for mutant screenings. When the presence of mutants in a single nodule was examined by the procedures described in the Materials and methods section, most nodules (18 of 22 tested nodules) were occupied by a mutant with a single tag (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). This result was consistent with a previous study in <em>S</em>. <em>meliloti</em>,<sup><a reveal-id="DSN017C21" data-open="DSN017C21" class="link link-ref link-reveal xref-bibr">21</a></sup> indicating that the colonization or infection process may have similar features for both determinate and indeterminate nodules, and competition of infection may already occur at the colonization stage on host root hairs. And, as far as we tested, all tags were recovered in a screen of 70 nodules, suggesting that at least 70 nodules should be analyzed for mutant screenings in nodules.</p><p>Additionally, we tested whether nodulation mutants can be screened properly for in nodules formed by pools of mutants. Two mutants (02T05e06 and 22T04b06) that exhibit defective in nodulation (Fig. <a reveal-id="DSN017F4" data-open="DSN017F4" class="link link-reveal link-table xref-fig">4</a>A) were included in two distinct test sets (input samples) and inoculated onto <em>L</em>. <em>japonicus</em> roots. When we examined the presence of mutants in the total DNA isolated from 120 nodules (output samples), tag2 and tag22 containing fragments were not recovered, whereas all the other tags were detected (Fig. <a reveal-id="DSN017F4" data-open="DSN017F4" class="link link-reveal link-table xref-fig">4</a>B). These results are consistent with the defective nodulation phenotype of the mutants (02T05e06 and 22T04b06), as tested individually (Fig. <a reveal-id="DSN017F4" data-open="DSN017F4" class="link link-reveal link-table xref-fig">4</a>A). During the infection process, integrated transposon elements were stable since identical amplification patterns were obtained before and after infection (Fig. <a reveal-id="DSN017F4" data-open="DSN017F4" class="link link-reveal link-table xref-fig">4</a>C, three representative results are shown).</p>                        <a id="5183347" scrollto-destination="5183347"></a>
<div id="DSN017F4" data-id="DSN017F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn017/2/m_dsn01704.jpeg?Expires=2147483647&Signature=vkPT0OSUf0WQ4aBarNxOMSpiK05InFb0FxTklQPv1T9r4AElxIzC2lDt8nec9eEqSGD90i5fGeQT1aGARVHZjfkK8ISduUykLVFqJTsJwu7-HvzPkDPoHGwrduKlhqdrsksT7qHt6W2JVzbL8z-98rLcmnHYIEl-fmHxEZqEdEwzZEcCP2ohGN7d3Bz3uZMRh8mA095Jk8~1SQqQD7yVT~i~P-04F8eLifMOKSC6eEPu6sOLgyvwuiwr8N5YY0Ga05~52fomvytB7YHhf6tRJd9I5sR7m7zB5LAug0riZK9IApLipHZZ-c39vx6BT2VdhJZ6q6ls~Nm2SgVPjlKwlQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Pilot experiments identifying mutants with a nodulation defective phenotype. (A) Phenotype of L. japonicus after inoculation with wild-type M. loti or with two nodulation-deficient mutants (02T05e06 and 22T04b06). Arrowheads indicate the nodules formed on the roots of L. japonicus and the inset show the enlarged image of root and nodules. (B) Tag amplification from genomic DNA isolated from input pool (upper) and output pool (middle and bottom). The numbers indicate the combination of primers used. The arrows indicate the absence of PCR products in reactions with the Tag2 or Tag22 primer. (C) Representative PCR amplification showing the stability of the integrated transposon fragment in three different mutants before (−) and after (+) inoculation. Arrowheads indicate the fragment containing the integrated transposon element. W, PCR amplification with genomic DNA from wild-type M. loti." path-from-xml="dsn01704.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5183347/dsn01704.jpeg" path-from-xml="dsn01704.jpeg" target="_blank">View large</a><a section="5183347" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn017/2/dsn01704.jpeg?Expires=2147483647&Signature=B1gDc6Vci40O6wwrUqj4E2cxzMoSKy4tJ8cQ6Y-3qZ3AQw6fbWanmdc4iHqx3JvWE0fk3SM9lFhZdgs5NKg4sxpWl8Sh6uI7y~Zt-x4hkyqff6itoDBYZk8Zyhis38AkYnipDHZ7TW2dmd78ZHydkrG9TKRnqLqQEwITQcDf1I3CX1Rw7N6li2Swl257VfmP9x0SaAU-6VrNN7AtdrXV0snMtO~y1lmqKMudVwRaQSVeky1mKCisbxr~tbXF0ZaK4xGEfedIGFp1J5zmGYBPkxJjKqn~QPHwUEwYL3kuAIRwexEGyRTI~71CUWwaG5-VWfDEPW8ixiTDL11ubJ42wg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5183347&ar=359537&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01704.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Pilot experiments identifying mutants with a nodulation defective phenotype. (<strong>A</strong>) Phenotype of <em>L</em>. <em>japonicus</em> after inoculation with wild-type <em>M</em>. <em>loti</em> or with two nodulation-deficient mutants (02T05e06 and 22T04b06). Arrowheads indicate the nodules formed on the roots of <em>L</em>. <em>japonicus</em> and the inset show the enlarged image of root and nodules. (<strong>B</strong>) Tag amplification from genomic DNA isolated from input pool (upper) and output pool (middle and bottom). The numbers indicate the combination of primers used. The arrows indicate the absence of PCR products in reactions with the Tag2 or Tag22 primer. (<strong>C</strong>) Representative PCR amplification showing the stability of the integrated transposon fragment in three different mutants before (−) and after (+) inoculation. Arrowheads indicate the fragment containing the integrated transposon element. W, PCR amplification with genomic DNA from wild-type <em>M</em>. <em>loti</em>.</p></div></div><div content-id="DSN017F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn017/2/m_dsn01704.jpeg?Expires=2147483647&Signature=vkPT0OSUf0WQ4aBarNxOMSpiK05InFb0FxTklQPv1T9r4AElxIzC2lDt8nec9eEqSGD90i5fGeQT1aGARVHZjfkK8ISduUykLVFqJTsJwu7-HvzPkDPoHGwrduKlhqdrsksT7qHt6W2JVzbL8z-98rLcmnHYIEl-fmHxEZqEdEwzZEcCP2ohGN7d3Bz3uZMRh8mA095Jk8~1SQqQD7yVT~i~P-04F8eLifMOKSC6eEPu6sOLgyvwuiwr8N5YY0Ga05~52fomvytB7YHhf6tRJd9I5sR7m7zB5LAug0riZK9IApLipHZZ-c39vx6BT2VdhJZ6q6ls~Nm2SgVPjlKwlQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Pilot experiments identifying mutants with a nodulation defective phenotype. (A) Phenotype of L. japonicus after inoculation with wild-type M. loti or with two nodulation-deficient mutants (02T05e06 and 22T04b06). Arrowheads indicate the nodules formed on the roots of L. japonicus and the inset show the enlarged image of root and nodules. (B) Tag amplification from genomic DNA isolated from input pool (upper) and output pool (middle and bottom). The numbers indicate the combination of primers used. The arrows indicate the absence of PCR products in reactions with the Tag2 or Tag22 primer. (C) Representative PCR amplification showing the stability of the integrated transposon fragment in three different mutants before (−) and after (+) inoculation. Arrowheads indicate the fragment containing the integrated transposon element. W, PCR amplification with genomic DNA from wild-type M. loti." path-from-xml="dsn01704.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5183347/dsn01704.jpeg" path-from-xml="dsn01704.jpeg" target="_blank">View large</a><a section="5183347" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn017/2/dsn01704.jpeg?Expires=2147483647&Signature=B1gDc6Vci40O6wwrUqj4E2cxzMoSKy4tJ8cQ6Y-3qZ3AQw6fbWanmdc4iHqx3JvWE0fk3SM9lFhZdgs5NKg4sxpWl8Sh6uI7y~Zt-x4hkyqff6itoDBYZk8Zyhis38AkYnipDHZ7TW2dmd78ZHydkrG9TKRnqLqQEwITQcDf1I3CX1Rw7N6li2Swl257VfmP9x0SaAU-6VrNN7AtdrXV0snMtO~y1lmqKMudVwRaQSVeky1mKCisbxr~tbXF0ZaK4xGEfedIGFp1J5zmGYBPkxJjKqn~QPHwUEwYL3kuAIRwexEGyRTI~71CUWwaG5-VWfDEPW8ixiTDL11ubJ42wg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5183347&ar=359537&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01704.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Pilot experiments identifying mutants with a nodulation defective phenotype. (<strong>A</strong>) Phenotype of <em>L</em>. <em>japonicus</em> after inoculation with wild-type <em>M</em>. <em>loti</em> or with two nodulation-deficient mutants (02T05e06 and 22T04b06). Arrowheads indicate the nodules formed on the roots of <em>L</em>. <em>japonicus</em> and the inset show the enlarged image of root and nodules. (<strong>B</strong>) Tag amplification from genomic DNA isolated from input pool (upper) and output pool (middle and bottom). The numbers indicate the combination of primers used. The arrows indicate the absence of PCR products in reactions with the Tag2 or Tag22 primer. (<strong>C</strong>) Representative PCR amplification showing the stability of the integrated transposon fragment in three different mutants before (−) and after (+) inoculation. Arrowheads indicate the fragment containing the integrated transposon element. W, PCR amplification with genomic DNA from wild-type <em>M</em>. <em>loti</em>.</p></div></div><p>From the results of these pilot experiments, we concluded that the signature-tagged mutants can be used for future STM screenings both in vitro and in vivo using our experimental conditions.</p>                            <h3 scrollto-destination=5183349 id="5183349" class="section-title" data-legacy-id=s3c>3.3. Mapping of transposon insertion sites</h3>
<p>Along with the collection of transposon mutants, we determined the transposon insertion sites of individual mutants. Transposon insertion sites were determined by sequencing of transposon-flanking genomic regions, followed by comparison with the reference genome sequence. Determination of transposon insertions revealed that the inverted repeat of both the left and right ends of the integrated transposon were followed by the individual 9 bp direct repeat of the target site, indicating that the transposition were occurred properly by Tn5-transposon integration.<sup><a reveal-id="DSN017C33" data-open="DSN017C33" class="link link-ref link-reveal xref-bibr">33</a></sup></p><p>The distribution of transposon insertion sites in the <em>M</em>. <em>loti</em> genome is illustrated in Fig. <a reveal-id="DSN017F5" data-open="DSN017F5" class="link link-reveal link-table xref-fig">5</a>. Among 9344 mutants examined, insertion sites of 7892 were determined (success rate, 84.5%) that were mapped to 7586 unique insertion sites (Table <a reveal-id="DSN017TB1" data-open="DSN017TB1" class="link link-reveal link-table xref-fig">1</a>). This proportion indicates that ∼96% of mutants carry a unique transposon insertion in their genome and there is no significant redundancy of a clone arising from identical parent clones in the same tri-parental mating experiment. Among the unique insertions, 7018 (92.5%), 398 (5.2%) and 170 (2.3%) were mapped onto the chromosome and two plasmids of <em>M</em>. <em>loti</em> (pMLa and pMLb), respectively (all insertion sites are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2 and S3</a>). The proportion of transposon distribution was consistent with the proportion of total size of each <em>M. loti</em> replicon (Chr, 92.6%, pMLa 4.6% and pMLb 2.7%). Furthermore, among the unique insertion sites, 6430 (84.8%) were within ORFs, corresponding to the overall proportion (85.6%) of coding sequences within the entire <em>M</em>. <em>loti</em> genome.<sup><a reveal-id="DSN017C1" data-open="DSN017C1" class="link link-ref link-reveal xref-bibr">1</a></sup></p>                        <a id="5183352" scrollto-destination="5183352"></a>
<div id="DSN017F5" data-id="DSN017F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn017/2/m_dsn01705.jpeg?Expires=2147483647&Signature=5P3kQa4mcaHrwRjMq0CmBKACtDYZDZyK0s6TK-IxdMOmAkmcY2e0zV0mBOOWf3JWOn5Xw3WXuYSYq5vJ2ZdRN8A9XpGkBeN59PNFXuAb0y~3Ispux4Bxc~YzJ7l4WkjjRCthc0oVx1M3I7hV6c7qP0~NOs0ScltrvxXkCtN2IsYC243CTVA4rIeyI1saFpGjbZFVcP7knIn-QoGqHqyWvlGIlPOCPc2qt-X6ijik6Qc7iU9376MqX87JWOnaBlUhRWe8Syh7EuZa2Sr~vYXxAPyJ25vCJLsEStfD~E8TgKUlvyDveunm5g9IXdOb466HDaBor64xU70132ISv51~Tw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of transposon insertions in the M. loti genome. The bars in the first and second circles show the position of transposon insertion sites within ORFs (red bars) and within intergenic regions (blue bars), respectively. The innermost circles represent the average G+C content calculated in each 10 kb. A positive deviation in G+C content from the average (62.7%) is shown by black bars pointing outward and a negative deviation by red bars pointing inward. Grey bar in chromosome indicates the region of symbiosis island." path-from-xml="dsn01705.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5183352/dsn01705.jpeg" path-from-xml="dsn01705.jpeg" target="_blank">View large</a><a section="5183352" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn017/2/dsn01705.jpeg?Expires=2147483647&Signature=YNU63AvhXrLsRByCtSi7OHhyzjuSOldlE6m8PqO3NLP604pndUb5ipvHlkkqnmkR22pJYeTcQADmSlPn1BBiU3C3r9p7kNNHZxhSsHY7cmmCyP7xlPjYGFi1JuT33ZF-ATAbbGt2hzMYjhIEDTwzlwr9Hs7B93XuhOugb3QQM7tI6HdwGSMxHM3eC4Ou82O5VsqlcSQVsIlUAAaKWfmWnI1IbDZV0SkufGLWbSgpEvVNlqQ-aVNSOh9KLndGZCtSy0pPXV~aReNm0Bx1iaurPIoGtVvUTSom5ZuxPHObpn6chQzn4hrTddQTdsuKhPtZ7k9BWnpnDgJMX7MlzbLzug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5183352&ar=359537&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01705.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of transposon insertions in the <em>M</em>. <em>loti</em> genome. The bars in the first and second circles show the position of transposon insertion sites within ORFs (red bars) and within intergenic regions (blue bars), respectively. The innermost circles represent the average G+C content calculated in each 10 kb. A positive deviation in G+C content from the average (62.7%) is shown by black bars pointing outward and a negative deviation by red bars pointing inward. Grey bar in chromosome indicates the region of symbiosis island.</p></div></div><div content-id="DSN017F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn017/2/m_dsn01705.jpeg?Expires=2147483647&Signature=5P3kQa4mcaHrwRjMq0CmBKACtDYZDZyK0s6TK-IxdMOmAkmcY2e0zV0mBOOWf3JWOn5Xw3WXuYSYq5vJ2ZdRN8A9XpGkBeN59PNFXuAb0y~3Ispux4Bxc~YzJ7l4WkjjRCthc0oVx1M3I7hV6c7qP0~NOs0ScltrvxXkCtN2IsYC243CTVA4rIeyI1saFpGjbZFVcP7knIn-QoGqHqyWvlGIlPOCPc2qt-X6ijik6Qc7iU9376MqX87JWOnaBlUhRWe8Syh7EuZa2Sr~vYXxAPyJ25vCJLsEStfD~E8TgKUlvyDveunm5g9IXdOb466HDaBor64xU70132ISv51~Tw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of transposon insertions in the M. loti genome. The bars in the first and second circles show the position of transposon insertion sites within ORFs (red bars) and within intergenic regions (blue bars), respectively. The innermost circles represent the average G+C content calculated in each 10 kb. A positive deviation in G+C content from the average (62.7%) is shown by black bars pointing outward and a negative deviation by red bars pointing inward. Grey bar in chromosome indicates the region of symbiosis island." path-from-xml="dsn01705.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5183352/dsn01705.jpeg" path-from-xml="dsn01705.jpeg" target="_blank">View large</a><a section="5183352" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn017/2/dsn01705.jpeg?Expires=2147483647&Signature=YNU63AvhXrLsRByCtSi7OHhyzjuSOldlE6m8PqO3NLP604pndUb5ipvHlkkqnmkR22pJYeTcQADmSlPn1BBiU3C3r9p7kNNHZxhSsHY7cmmCyP7xlPjYGFi1JuT33ZF-ATAbbGt2hzMYjhIEDTwzlwr9Hs7B93XuhOugb3QQM7tI6HdwGSMxHM3eC4Ou82O5VsqlcSQVsIlUAAaKWfmWnI1IbDZV0SkufGLWbSgpEvVNlqQ-aVNSOh9KLndGZCtSy0pPXV~aReNm0Bx1iaurPIoGtVvUTSom5ZuxPHObpn6chQzn4hrTddQTdsuKhPtZ7k9BWnpnDgJMX7MlzbLzug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5183352&ar=359537&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01705.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of transposon insertions in the <em>M</em>. <em>loti</em> genome. The bars in the first and second circles show the position of transposon insertion sites within ORFs (red bars) and within intergenic regions (blue bars), respectively. The innermost circles represent the average G+C content calculated in each 10 kb. A positive deviation in G+C content from the average (62.7%) is shown by black bars pointing outward and a negative deviation by red bars pointing inward. Grey bar in chromosome indicates the region of symbiosis island.</p></div></div><p>Previous studies have demonstrated that the distribution patterns of transposon insertion into genomes vary depending on the types of transposon and organisms used for mutagenesis.<sup><a reveal-id="DSN017C34  DSN017C36" data-open="DSN017C34  DSN017C36" class="link link-ref link-reveal xref-bibr">34–36</a></sup> To examine whether the transposon insertions were distributed randomly throughout the <em>M</em>. <em>loti</em> genome, we first investigated potential target site preferences for transposon insertion. Transposon insertions occurred in both intragenic and intergenic regions and the orientations of integrated transposons, with respect to the ORF directions of transcription, are present in nearly equal proportions (3274 clones in the same direction as transcription and 3397 clones in the opposite direction) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). In addition, according to a previous report,<sup><a reveal-id="DSN017C37" data-open="DSN017C37" class="link link-ref link-reveal xref-bibr">37</a></sup> G+C content was calculated from the 9 nucleotides of the insertion-target sites and 10 nucleotides up- and down-stream of target site of 50 randomly selected clones. It showed a high variance ranging from 30% to 88% (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>), and the average G+C content of the target site (62.7%) was consistent with that of <em>M</em>. <em>loti</em> chromosome (62.7%).<sup><a reveal-id="DSN017C1" data-open="DSN017C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Furthermore, 50 randomly selected target sites showed no obvious sequence pattern, including palindromic or symmetrical sequences, when these were analyzed by TEIRESIAS, the sequence pattern search algorithm (<a class="link link-uri" href="http://cbcsrv.watson.ibm.com/Tspd.html">http://cbcsrv.watson.ibm.com/Tspd.html</a>).<sup><a reveal-id="DSN017C38" data-open="DSN017C38" class="link link-ref link-reveal xref-bibr">38</a></sup> These features indicate that there are no clear preferences in insertion with regard to G+C content or structural features of the target sites.</p><p>In order to verify the random distribution of transposon insertion, we additionally investigated the relationship between the number of genes carrying an inserted transposon and the size of the corresponding genes. Fig. <a reveal-id="DSN017F6" data-open="DSN017F6" class="link link-reveal link-table xref-fig">6</a> shows the distribution of transposon insertion within ORFs. As expected, more transposon insertions occurred in larger ORFs and the number of disrupted genes clearly correlate with the number of <em>M</em>. <em>loti</em> ORFs in each size range. In addition, as shown in Fig. <a reveal-id="DSN017F5" data-open="DSN017F5" class="link link-reveal link-table xref-fig">5</a>, there is no obvious correlation between the distribution of transposon insertion and local G+C content of the <em>M</em>. <em>loti</em> genome. The symbiosis island, which exhibits features of horizontally transferred genomic element of low G+C content, also contained transposon insertions at rates similar to other chromosomal region (Fig. <a reveal-id="DSN017F5" data-open="DSN017F5" class="link link-reveal link-table xref-fig">5</a> grey line). Furthermore, this random distribution was observed for all tagged plasmid constructs.</p>                        <a id="5183355" scrollto-destination="5183355"></a>
<div id="DSN017F6" data-id="DSN017F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn017/2/m_dsn01706.jpeg?Expires=2147483647&Signature=WO2DFE4amswXOIM6tupre7Jycz6V3FZGW8QlOZfTtnaifWTWcDs8x~3kxZzHNEODuaLLcueU69YgkXLpu-Z8ySVxDCGgVapC6bc264qxzBJS4tvOpcZFVjuxzbdAvT4UN9N212MQAftt8X4-7CHfrw8yfESHvV5WuKgGrmnV~8Vc6b~slZUX2hGF2Mg6HEIBhlgcazHIcMEQVrIhl1Xf23XMmUp0Px~OUB9bY88eiFsI8OpdjklKB-W5eNUvnmh0ToFSsPdQY00CBSXcyzTP8rahZ3iNcHgdsoIWn8DWV1cQuWFTd-bZ0dh3mAzCWXz9GfWbt0Jm-ejwT1~sMCfVgQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of transposon insertion within M. loti ORFs. The white bars represent the number of M. loti genes in each size range and the grey bars represent the number of genes with transposon insertions. The line chart represents the average number of insertion in genes of each size range with standard deviation. Percentages in each bar indicate the coverage of disrupted genes in each size range." path-from-xml="dsn01706.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5183355/dsn01706.jpeg" path-from-xml="dsn01706.jpeg" target="_blank">View large</a><a section="5183355" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn017/2/dsn01706.jpeg?Expires=2147483647&Signature=pO~7j3q9OXfbLe6I-mhMqdl5Z1IPDUBXQgvP03oK3YD6ZoGBu~h6o1LxOqgW3B6RPACfdvTmGnw~nZhHXg94UKj~iIecUwzeR9oTYD0rO0ucTmPXjKpdREEgnXVGoVdRhCV5XnOfTvFowhNmBPxAVkHmEs5xbz0qaHczHq1mJmJ2i~bRe5Ajs~F82g5CSHl5QDnZJNz9ALvVWnsZHUjvJUaKe9t7hEVftiH~81JBrDqyupFi7AwN7eB-ADS506VuUmooHTWE2EGNXbpp7qFDr86kQVHta8Mz9OsXUmYVFSjVQmJeYmm2Wb6YbBR7OD0oonL7K8muGNupH0C~b62NTA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5183355&ar=359537&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01706.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of transposon insertion within <em>M</em>. <em>loti</em> ORFs. The white bars represent the number of <em>M</em>. <em>loti</em> genes in each size range and the grey bars represent the number of genes with transposon insertions. The line chart represents the average number of insertion in genes of each size range with standard deviation. Percentages in each bar indicate the coverage of disrupted genes in each size range.</p></div></div><div content-id="DSN017F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn017/2/m_dsn01706.jpeg?Expires=2147483647&Signature=WO2DFE4amswXOIM6tupre7Jycz6V3FZGW8QlOZfTtnaifWTWcDs8x~3kxZzHNEODuaLLcueU69YgkXLpu-Z8ySVxDCGgVapC6bc264qxzBJS4tvOpcZFVjuxzbdAvT4UN9N212MQAftt8X4-7CHfrw8yfESHvV5WuKgGrmnV~8Vc6b~slZUX2hGF2Mg6HEIBhlgcazHIcMEQVrIhl1Xf23XMmUp0Px~OUB9bY88eiFsI8OpdjklKB-W5eNUvnmh0ToFSsPdQY00CBSXcyzTP8rahZ3iNcHgdsoIWn8DWV1cQuWFTd-bZ0dh3mAzCWXz9GfWbt0Jm-ejwT1~sMCfVgQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of transposon insertion within M. loti ORFs. The white bars represent the number of M. loti genes in each size range and the grey bars represent the number of genes with transposon insertions. The line chart represents the average number of insertion in genes of each size range with standard deviation. Percentages in each bar indicate the coverage of disrupted genes in each size range." path-from-xml="dsn01706.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5183355/dsn01706.jpeg" path-from-xml="dsn01706.jpeg" target="_blank">View large</a><a section="5183355" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn017/2/dsn01706.jpeg?Expires=2147483647&Signature=pO~7j3q9OXfbLe6I-mhMqdl5Z1IPDUBXQgvP03oK3YD6ZoGBu~h6o1LxOqgW3B6RPACfdvTmGnw~nZhHXg94UKj~iIecUwzeR9oTYD0rO0ucTmPXjKpdREEgnXVGoVdRhCV5XnOfTvFowhNmBPxAVkHmEs5xbz0qaHczHq1mJmJ2i~bRe5Ajs~F82g5CSHl5QDnZJNz9ALvVWnsZHUjvJUaKe9t7hEVftiH~81JBrDqyupFi7AwN7eB-ADS506VuUmooHTWE2EGNXbpp7qFDr86kQVHta8Mz9OsXUmYVFSjVQmJeYmm2Wb6YbBR7OD0oonL7K8muGNupH0C~b62NTA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5183355&ar=359537&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01706.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of transposon insertion within <em>M</em>. <em>loti</em> ORFs. The white bars represent the number of <em>M</em>. <em>loti</em> genes in each size range and the grey bars represent the number of genes with transposon insertions. The line chart represents the average number of insertion in genes of each size range with standard deviation. Percentages in each bar indicate the coverage of disrupted genes in each size range.</p></div></div><p>Several previous studies have provided sets of essential genes of organisms predicted from comprehensive random mutagenesis or targeted gene disruption.<sup><a reveal-id="DSN017C36" data-open="DSN017C36" class="link link-ref link-reveal xref-bibr">36</a>,<a reveal-id="DSN017C39" data-open="DSN017C39" class="link link-ref link-reveal xref-bibr">39</a>,<a reveal-id="DSN017C40" data-open="DSN017C40" class="link link-ref link-reveal xref-bibr">40</a></sup> Although here we collected a large number of transposon mutants of <em>M</em>. <em>loti</em>, it is currently impossible to estimate the precise number of <em>M</em>. <em>loti</em> essential genes because the number of mutants for which transposon insertion sites have been determined is not sufficient to cover all non-essential genes. However, we can predict essential gene candidates among the relatively large size <em>M</em>. <em>loti</em> genes since large size <em>M</em>. <em>loti</em> genes (&gt;3000 bp) were disrupted at high coverage (83.1%). Among 77 genes whose size is larger than 3000 bp, transposon insertions were not detected in 13 genes, whereas an average of 4.7 insertions occurred in other genes in this size range (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). Considering the random distribution of transposon insertion, these 13 genes are candidate <em>M</em>. <em>loti</em> essential genes required for growth on rich media. Among the genes, 11 genes (excluding mlr9704 and mll1090) are highly conserved in the genome of completely sequenced rhizobia and four genes, mlr8250 (isoleucyl-tRNA synthetase), mll0870 (DNA polymerase III alpha subunit), mlr0276 (RNA polymerase beta subunit) and mlr0277 (RNA polymerase, beta prime subunit), showed high similarity to essential genes of <em>E</em>. <em>coli</em> (<em>ileS</em>, <em>dnaE</em>, <em>rpoB</em> and <em>rpoC</em>, respectively). Furthermore, mlr2517 is orthologous to the essential gene of <em>Acinetobacter baylyi</em> ADP1 (<em>carB</em>, carbamoyl-phosphate synthase, large subunit).<sup><a reveal-id="DSN017C41" data-open="DSN017C41" class="link link-ref link-reveal xref-bibr">41</a></sup> Other essential gene candidates could be those with insertions only near 3′-terminus of the ORF. Mutants for mll1426, mlr0796 and mlr0585 could be the candidate of essential genes since relatively small number of insertions, all of which were in their 3′-terminus, were detected as listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>.</p><p>In bacteria, essential genes are more evolutionarily conserved than non-essential genes.<sup><a reveal-id="DSN017C42" data-open="DSN017C42" class="link link-ref link-reveal xref-bibr">42</a></sup> We compared <em>E</em>. <em>coli</em> essential genes and <em>M</em>. <em>loti</em> genes [BlastP search was carried out against <em>E</em>. <em>coli</em> essential genes listed in the PEC data base (<a class="link link-uri" href="http://www.shigen.nig.ac.jp/ecoli/pec/index.jsp">www.shigen.nig.ac.jp/ecoli/pec/index.jsp</a>)] and revealed that a total of 196 <em>M</em>. <em>loti</em> ORFs showed high similarity to <em>E</em>. <em>coli</em> essential genes. Transposon mutants were obtained for only 21 of these 196 genes (10.7%). This ratio is significantly low since the result of sequence analysis transposon insertion sites identified mutants for ∼50% of <em>M</em>. <em>loti</em> genes. Among the 21 genes with transposon insertion, eight genes have putative palarogous genes which are considered to complement the function of the disrupted genes. These results indicate that significant overlap exists among sets of essential genes in <em>E</em>. <em>coli</em> and <em>M</em>. <em>loti</em>. However, saturating collections of transposon mutants under different selective conditions are, of course, needed to confirm these genes cannot actually be disrupted.</p>                            <h3 scrollto-destination=5183358 id="5183358" class="section-title" data-legacy-id=s3d>3.4. Mutant database</h3>
<p>To provide resources for depositing and retrieving information on <em>M</em>. <em>loti</em> mutants, we developed a relational database, RhizoGenes (<a class="link link-uri" href="http://www.kazusa.or.jp/rhizobase/Mesorhizobium/genes2/index.html">http://www.kazusa.or.jp/rhizobase/Mesorhizobium/genes2/index.html</a>). This database operates as a public web database that provides general information about mutants, e.g. mutant IDs, transposon insertion site, list of mutant alleles and experimental results (which can be deposited by users). In this database, individual mutants can be retrieved through keyword search by clone ID, name and functional annotation of disrupted genes. Therefore, when genes have attracted interest on the basis of bioinformatic or functional genomic analyses, appropriate mutants can be searched immediately. The transposon mutants collected in this study will be available from the National Bioresource project of Japan (<a class="link link-uri" href="http://www.legumebase.agr.miyazaki-u.ac.jp/index.jsp">http://www.legumebase.agr.miyazaki-u.ac.jp/index.jsp</a>).</p><p>Furthermore, we have integrated this database into the genome information database, RhizoBase (<a class="link link-uri" href="http://bacteria.kazusa.or.jp/rhizobase/Mesorhizobium/index.html">http://bacteria.kazusa.or.jp/rhizobase/Mesorhizobium/index.html</a>), so as to link directly to more detailed information about <em>M</em>. <em>loti</em> ORFs, such as operon structure, predicted protein domains and orthologous (paralogous) protein groups.</p>                            <h3 scrollto-destination=5183361 id="5183361" class="section-title" data-legacy-id=s3e>3.5. Concluding remarks</h3>
<p>In the present study, we have described a large-scale transposon mutagenesis in the nitrogen-fixing symbiotic bacterium, <em>M</em>. <em>loti</em>. The STM technique was adapted to barcode individual transposon mutants and a large mutant library containing &gt;29 000 independent clones was created using a set of 27 signature tags. Sequencing of the transposon insertion sites of one-third of the mutants revealed a random distribution of transposon insertion and ∼50% of <em>M</em>. <em>loti</em> genes have been confirmed to be disrupted. Since such a large number of disruption mutants have not previously been available for <em>M</em>. <em>loti</em> and the low efficiency of gene-disruption mediated by homologous recombination has been a serious technical bottleneck for functional studies of this symbiont, our mutant collections should be a powerful tool for future functional studies of <em>M</em>. <em>loti</em> genes.</p><p>Applied STM technique and PCR-based screening allow for the simultaneous discovery of mutants with subtle phenotypic differences, such as reduced competitiveness or survival under various conditions. Additionally, random distribution of transposon insertion ensures the screenings of mutants over the entire <em>M</em>. <em>loti</em> genome and reduces the number of clones to be screened. Determination of transposon insertion sites also enables users to select defined, non-redundant sets of mutants depending on individual needs and facilitates rapid identification of causal genes of the mutant phenotypes. Furthermore, individual transposon mutants can be used as single gene disruptants. The availability of single gene disruptants allows functional verifications of individual or paralogous <em>M</em>. <em>loti</em> genes as well as comparative functional studies by examining with the orthologous genes of other rhizobium species. Additionally, our resource will be useful for <em>in planta</em> studies in the host legume. Actually, our STM mutants have recently contributed to reveal the importance of purine metabolisms in <em>Lotus-M. loti</em> symbiosis.<sup><a reveal-id="DSN017C24" data-open="DSN017C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Use of the mutant collections in <em>Lotus-M. loti</em> symbiosis will contribute to discovery of novel rhizobial factors or host plant recognition systems required for the symbiosis. By providing this mutant resource to the research communities, we hope to contribute to various functional analyses that lead to novel findings and understandings of rhizobial biology.</p>                            <h2 scrollto-destination=5183364 id="5183364" class="section-title" data-legacy-id=s4>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5183366 id="5183366" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was supported by KAKENHI (Grant-in-Aid for Scientific Research) on Priority Areas “Comparative Genomics” from the Ministry of Education, Culture, Sports, Science, and Technology of Japan, and the Kazusa DNA Research Institute Foundation.</p>                            <h2 scrollto-destination=5183368 id="5183368" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSN017C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kaneko</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Nakamura</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">S.,</div></div>, et al. </div><div class="article-title">Complete genome structure of the nitrogen-fixing symbiotic bacterium <em>Mesorhizobium loti</em></div>, <div class="source">DNA Res.</div> , <div class="year">2000</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">331</div>-<div class="lpage">338</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20structure%20of%20the%20nitrogen-fixing%20symbiotic%20bacterium%20Mesorhizobium%20loti&amp;author=T.%20Kaneko&amp;author=Y.%20Nakamura&amp;author=S.%2C%20Sato&amp;publication_year=2000&amp;journal=DNA%20Res.&amp;volume=7&amp;pages=331-338" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/7.6.331" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F7.6.331" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F7.6.331"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11214968" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN017C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kaneko</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Nakamura</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">S.,</div></div>, et al. </div><div class="article-title">Complete genomic sequence of nitrogen-fixing symbiotic bacterium <em>Bradyrhizobium japonicum</em> USDA110</div>, <div class="source">DNA Res.</div> , <div class="year">2002</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">189</div>-<div class="lpage">197</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genomic%20sequence%20of%20nitrogen-fixing%20symbiotic%20bacterium%20Bradyrhizobium%20japonicum%20USDA110&amp;author=T.%20Kaneko&amp;author=Y.%20Nakamura&amp;author=S.%2C%20Sato&amp;publication_year=2002&amp;journal=DNA%20Res.&amp;volume=9&amp;pages=189-197" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/9.6.189" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F9.6.189" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F9.6.189"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12597275" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN017C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Galibert</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Finan</div> <div class="given-names">T. M.</div></div>,  <div class="name"><div class="surname">Long</div> <div class="given-names">S. R.,</div></div>, et al. </div><div class="article-title">The composite genome of the legume symbiont <em>Sinorhizobium meliloti</em></div>, <div class="source">Science</div> , <div class="year">2001</div>, vol. <div class="volume">293</div> (pg. <div class="fpage">668</div>-<div class="lpage">672</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20composite%20genome%20of%20the%20legume%20symbiont%20Sinorhizobium%20meliloti&amp;author=F.%20Galibert&amp;author=T.%20M.%20Finan&amp;author=S.%20R.%2C%20Long&amp;publication_year=2001&amp;journal=Science&amp;volume=293&amp;pages=668-672" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1060966" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1060966" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1060966"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11474104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN017C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">González</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Santamaría</div> <div class="given-names">R. I.</div></div>,  <div class="name"><div class="surname">Bustos</div> <div class="given-names">P.,</div></div>, et al. </div><div class="article-title">The partitioned <em>Rhizobium etli</em> genome: genetic and metabolic redundancy in seven interacting replicons</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2006</div>, vol. <div class="volume">103</div> (pg. <div class="fpage">3834</div>-<div class="lpage">3839</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20partitioned%20Rhizobium%20etli%20genome%3A%20genetic%20and%20metabolic%20redundancy%20in%20seven%20interacting%20replicons&amp;author=V.%20Gonz%C3%A1lez&amp;author=R.%20I.%20Santamar%C3%ADa&amp;author=P.%2C%20Bustos&amp;publication_year=2006&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=103&amp;pages=3834-3839" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0508502103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0508502103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0508502103"> </span></div> </div></div></div></div></div><div content-id="DSN017C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Young</div> <div class="given-names">J. P.</div></div>,  <div class="name"><div class="surname">Crossman</div> <div class="given-names">L. C.</div></div>,  <div class="name"><div class="surname">Johnston</div> <div class="given-names">A. W.,</div></div>, et al. </div><div class="article-title">The genome of <em>Rhizobium leguminosarum</em> has recognizable core and accessory components</div>, <div class="source">Genome Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">R34</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20Rhizobium%20leguminosarum%20has%20recognizable%20core%20and%20accessory%20components&amp;author=J.%20P.%20Young&amp;author=L.%20C.%20Crossman&amp;author=A.%20W.%2C%20Johnston&amp;publication_year=2006&amp;journal=Genome%20Biol.&amp;volume=7&amp;pages=R34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2006-7-4-r34" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2006-7-4-r34" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2006-7-4-r34"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16640791" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN017C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Giraud</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Moulin</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Vallenet</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Barbe</div> <div class="given-names">V.,</div></div>, et al. </div><div class="article-title">Legumes symbioses: absence of Nod genes in photosynthetic bradyrhizobia</div>, <div class="source">Science</div> , <div class="year">2007</div>, vol. <div class="volume">316</div> (pg. <div class="fpage">1307</div>-<div class="lpage">1312</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Legumes%20symbioses%3A%20absence%20of%20Nod%20genes%20in%20photosynthetic%20bradyrhizobia&amp;author=E.%20Giraud&amp;author=L.%20Moulin&amp;author=D.%20Vallenet&amp;author=V.%2C%20Barbe&amp;publication_year=2007&amp;journal=Science&amp;volume=316&amp;pages=1307-1312" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1139548" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1139548" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1139548"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17540897" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN017C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bergès</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Lauber</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Liebe</div> <div class="given-names">C.,</div></div>, et al. </div><div class="article-title">Development of <em>Sinorhizobium meliloti</em> pilot macroarrays for transcriptome analysis</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">2003</div>, vol. <div class="volume">69</div> (pg. <div class="fpage">1214</div>-<div class="lpage">1219</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20Sinorhizobium%20meliloti%20pilot%20macroarrays%20for%20transcriptome%20analysis&amp;author=H.%20Berg%C3%A8s&amp;author=E.%20Lauber&amp;author=C.%2C%20Liebe&amp;publication_year=2003&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=69&amp;pages=1214-1219" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.69.2.1214-1219.2003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.69.2.1214-1219.2003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.69.2.1214-1219.2003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12571049" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN017C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Djordjevic</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">H. C.</div></div>,  <div class="name"><div class="surname">Natera</div> <div class="given-names">S.,</div></div>, et al. </div><div class="article-title">A global analysis of protein expression profiles in <em>Sinorhizobium meliloti</em>: discovery of new genes for nodule occupancy and stress adaptation</div>, <div class="source">Mol. Plant Microbe. Interact.</div> , <div class="year">2003</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">508</div>-<div class="lpage">524</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20global%20analysis%20of%20protein%20expression%20profiles%20in%20Sinorhizobium%20meliloti%3A%20discovery%20of%20new%20genes%20for%20nodule%20occupancy%20and%20stress%20adaptation&amp;author=M.%20A.%20Djordjevic&amp;author=H.%20C.%20Chen&amp;author=S.%2C%20Natera&amp;publication_year=2003&amp;journal=Mol.%20Plant%20Microbe.%20Interact.&amp;volume=16&amp;pages=508-524" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI.2003.16.6.508" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI.2003.16.6.508" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI.2003.16.6.508"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12795377" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN017C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barnett</div> <div class="given-names">M. J.</div></div>,  <div class="name"><div class="surname">Toman</div> <div class="given-names">C. J.</div></div>,  <div class="name"><div class="surname">Fisher</div> <div class="given-names">R. F.</div></div>,  <div class="name"><div class="surname">Long</div> <div class="given-names">S. R</div></div>. </div><div class="article-title">A dual-genome Symbiosis Chip for coordinate study of signal exchange and development in a prokaryote–host interaction</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">16636</div>-<div class="lpage">16641</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20dual-genome%20Symbiosis%20Chip%20for%20coordinate%20study%20of%20signal%20exchange%20and%20development%20in%20a%20prokaryote%E2%80%93host%20interaction&amp;author=M.%20J.%20Barnett&amp;author=C.%20J.%20Toman&amp;author=R.%20F.%20Fisher&amp;author=S.%20R%20Long&amp;publication_year=2004&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=16636-16641" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0407269101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0407269101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0407269101"> </span></div> </div></div></div></div></div><div content-id="DSN017C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hoa</div> <div class="given-names">L. T. P.</div></div>,  <div class="name"><div class="surname">Nomura</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tajima</div> <div class="given-names">S</div></div>. </div><div class="article-title">Characterization of bacteroid proteins in soybean nodules formed with <em>Bradyrhizobium japonicum</em> USDA110</div>, <div class="source">Microbiol. Environ.</div> , <div class="year">2004</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">71</div>-<div class="lpage">75</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20bacteroid%20proteins%20in%20soybean%20nodules%20formed%20with%20Bradyrhizobium%20japonicum%20USDA110&amp;author=L.%20T.%20P.%20Hoa&amp;author=M.%20Nomura&amp;author=S%20Tajima&amp;publication_year=2004&amp;journal=Microbiol.%20Environ.&amp;volume=19&amp;pages=71-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1264/jsme2.19.71" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1264%2Fjsme2.19.71" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1264%2Fjsme2.19.71"> </span></div> </div></div></div></div></div><div content-id="DSN017C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Uchiumi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ohwada</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Itakura</div> <div class="given-names">M.,</div></div>, et al. </div><div class="article-title">Expression islands clustered on symbiosis island of <em>Mesorhizobium loti</em> genome</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2004</div>, vol. <div class="volume">45</div> (pg. <div class="fpage">2439</div>-<div class="lpage">2448</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20islands%20clustered%20on%20symbiosis%20island%20of%20Mesorhizobium%20loti%20genome&amp;author=T.%20Uchiumi&amp;author=T.%20Ohwada&amp;author=M.%2C%20Itakura&amp;publication_year=2004&amp;journal=J.%20Bacteriol.&amp;volume=45&amp;pages=2439-2448" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.186.8.2439-2448.2004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.186.8.2439-2448.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.186.8.2439-2448.2004"> </span></div> </div></div></div></div></div><div content-id="DSN017C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sarma</div> <div class="given-names">A. D.</div></div>,  <div class="name"><div class="surname">Emerich</div> <div class="given-names">D. W</div></div>. </div><div class="article-title">Global protein expression pattern of <em>Bradyrhizobium japonicum</em> bacteroids: a prelude to functional proteomics</div>, <div class="source">Proteomics</div> , <div class="year">2005</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">4170</div>-<div class="lpage">4184</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20protein%20expression%20pattern%20of%20Bradyrhizobium%20japonicum%20bacteroids%3A%20a%20prelude%20to%20functional%20proteomics&amp;author=A.%20D.%20Sarma&amp;author=D.%20W%20Emerich&amp;publication_year=2005&amp;journal=Proteomics&amp;volume=5&amp;pages=4170-4184" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1615-9861" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1615-9861" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1615-9861"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16254929" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN017C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shimoda</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Shinpo</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kohara</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Nakamura</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Tabata</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">S</div></div>. </div><div class="article-title">A large scale analysis of protein–protein interactions in the nitrogen-fixing bacterium <em>Mesorhizobium loti</em></div>, <div class="source">DNA Res.</div> , <div class="year">2008</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">13</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20large%20scale%20analysis%20of%20protein%E2%80%93protein%20interactions%20in%20the%20nitrogen-fixing%20bacterium%20Mesorhizobium%20loti&amp;author=Y.%20Shimoda&amp;author=S.%20Shinpo&amp;author=M.%20Kohara&amp;author=Y.%20Nakamura&amp;author=S.%20Tabata&amp;author=S%20Sato&amp;publication_year=2008&amp;journal=DNA%20Res.&amp;volume=15&amp;pages=13-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsm028" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsm028" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsm028"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18192278" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN017C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hutchison</div> <div class="given-names">C. A.</div></div>,  <div class="name"><div class="surname">Peterson</div> <div class="given-names">S. N.</div></div>,  <div class="name"><div class="surname">Gill</div> <div class="given-names">S. R.,</div></div>, et al. </div><div class="article-title">Global transposon mutagenesis and a minimal mycoplasma genome</div>, <div class="source">Science</div> , <div class="year">1999</div>, vol. <div class="volume">286</div> (pg. <div class="fpage">2165</div>-<div class="lpage">2169</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20transposon%20mutagenesis%20and%20a%20minimal%20mycoplasma%20genome&amp;author=C.%20A.%20Hutchison&amp;author=S.%20N.%20Peterson&amp;author=S.%20R.%2C%20Gill&amp;publication_year=1999&amp;journal=Science&amp;volume=286&amp;pages=2165-2169" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.286.5447.2165" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.286.5447.2165" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.286.5447.2165"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10591650" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN017C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kobayashi</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ehrlich</div> <div class="given-names">S. D.</div></div>,  <div class="name"><div class="surname">Albertini</div> <div class="given-names">A.,</div></div>, et al. </div><div class="article-title">Essential <em>Bacillus subtilis</em> genes</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2003</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">4678</div>-<div class="lpage">4683</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Essential%20Bacillus%20subtilis%20genes&amp;author=K.%20Kobayashi&amp;author=S.%20D.%20Ehrlich&amp;author=A.%2C%20Albertini&amp;publication_year=2003&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=100&amp;pages=4678-4683" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0730515100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0730515100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0730515100"> </span></div> </div></div></div></div></div><div content-id="DSN017C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Durfee</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Glasner</div> <div class="given-names">J. D.,</div></div>, et al. </div><div class="article-title">Systematic mutagenesis of the <em>Escherichia coli</em> genome</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2004</div>, vol. <div class="volume">186</div> (pg. <div class="fpage">4921</div>-<div class="lpage">4930</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systematic%20mutagenesis%20of%20the%20Escherichia%20coli%20genome&amp;author=Y.%20Kang&amp;author=T.%20Durfee&amp;author=J.%20D.%2C%20Glasner&amp;publication_year=2004&amp;journal=J.%20Bacteriol.&amp;volume=186&amp;pages=4921-4930" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.186.15.4921-4930.2004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.186.15.4921-4930.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.186.15.4921-4930.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15262929" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN017C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Baba</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ara</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Hasegawa</div> <div class="given-names">M.,</div></div>, et al. </div><div class="article-title">Construction of <em>Escherichia coli</em> K-12 in-frame, single-gene knockout mutants: the Keio collection</div>, <div class="source">Mol. Syst. Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">2</div> pg. <div class="fpage">0008</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20Escherichia%20coli%20K-12%20in-frame%2C%20single-gene%20knockout%20mutants%3A%20the%20Keio%20collection&amp;author=T.%20Baba&amp;author=T.%20Ara&amp;author=M.%2C%20Hasegawa&amp;publication_year=2006&amp;journal=Mol.%20Syst.%20Biol.&amp;volume=2&amp;pages=0008" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/msb4100050" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fmsb4100050" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fmsb4100050"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16738554" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN017C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Glass</div> <div class="given-names">J. I.</div></div>,  <div class="name"><div class="surname">Assad-Garcia</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Alperovich</div> <div class="given-names">N.,</div></div>, et al. </div><div class="article-title">Essential genes of a minimal bacterium</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2006</div>, vol. <div class="volume">103</div> (pg. <div class="fpage">425</div>-<div class="lpage">430</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Essential%20genes%20of%20a%20minimal%20bacterium&amp;author=J.%20I.%20Glass&amp;author=N.%20Assad-Garcia&amp;author=N.%2C%20Alperovich&amp;publication_year=2006&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=103&amp;pages=425-430" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0510013103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0510013103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0510013103"> </span></div> </div></div></div></div></div><div content-id="DSN017C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Saenz</div> <div class="given-names">H. L.</div></div>,  <div class="name"><div class="surname">Dehio</div> <div class="given-names">C</div></div>. </div><div class="article-title">Signature-tagged mutagenesis: technical advances in a negative selection method for virulence gene identification</div>, <div class="source">Curr. Opin. Microbiol.</div> , <div class="year">2005</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">612</div>-<div class="lpage">619</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Signature-tagged%20mutagenesis%3A%20technical%20advances%20in%20a%20negative%20selection%20method%20for%20virulence%20gene%20identification&amp;author=H.%20L.%20Saenz&amp;author=C%20Dehio&amp;publication_year=2005&amp;journal=Curr.%20Opin.%20Microbiol.&amp;volume=8&amp;pages=612-619" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mib.2005.08.013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mib.2005.08.013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mib.2005.08.013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16126452" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN017C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pobigaylo</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Wetter</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Szymczak</div> <div class="given-names">S.,</div></div>, et al. </div><div class="article-title">Construction of a large signature-tagged Mini-Tn5 transposon library and its application to mutagenesis of <em>Sinorhizobium meliloti</em></div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">2006</div>, vol. <div class="volume">72</div> (pg. <div class="fpage">4329</div>-<div class="lpage">4337</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20large%20signature-tagged%20Mini-Tn5%20transposon%20library%20and%20its%20application%20to%20mutagenesis%20of%20Sinorhizobium%20meliloti&amp;author=N.%20Pobigaylo&amp;author=D.%20Wetter&amp;author=S.%2C%20Szymczak&amp;publication_year=2006&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=72&amp;pages=4329-4337" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.03072-05" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.03072-05" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.03072-05"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16751548" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN017C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pobigaylo</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Szymczak</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nattkemper</div> <div class="given-names">T. M.</div></div>,  <div class="name"><div class="surname">Becker</div> <div class="given-names">A</div></div>. </div><div class="article-title">Identification of genes relevant to symbiosis and competitiveness in <em>Sinorhizobium meliloti</em> using signature-tagged mutants</div>, <div class="source">Mol. Plant Microbe. Interact.</div> , <div class="year">2008</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">219</div>-<div class="lpage">231</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20genes%20relevant%20to%20symbiosis%20and%20competitiveness%20in%20Sinorhizobium%20meliloti%20using%20signature-tagged%20mutants&amp;author=N.%20Pobigaylo&amp;author=S.%20Szymczak&amp;author=T.%20M.%20Nattkemper&amp;author=A%20Becker&amp;publication_year=2008&amp;journal=Mol.%20Plant%20Microbe.%20Interact.&amp;volume=21&amp;pages=219-231" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI-21-2-0219" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI-21-2-0219" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI-21-2-0219"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18184066" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN017C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fraysse</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">François</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Véréna</div> <div class="given-names">P</div></div>. </div><div class="article-title">Surface polysaccharide involvement in establishing the rhizobium-legume symbiosis</div>, <div class="source">FEBS J.</div> , <div class="year">2003</div>, vol. <div class="volume">270</div> (pg. <div class="fpage">1365</div>-<div class="lpage">1380</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Surface%20polysaccharide%20involvement%20in%20establishing%20the%20rhizobium-legume%20symbiosis&amp;author=N.%20Fraysse&amp;author=C.%20Fran%C3%A7ois&amp;author=P%20V%C3%A9r%C3%A9na&amp;publication_year=2003&amp;journal=FEBS%20J.&amp;volume=270&amp;pages=1365-1380" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1432-1033.2003.03492.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1432-1033.2003.03492.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1432-1033.2003.03492.x"> </span></div> </div></div></div></div></div><div content-id="DSN017C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">D'Antuono</div> <div class="given-names">A. L.</div></div>,  <div class="name"><div class="surname">Casabuono</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Couto</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Ugalde</div> <div class="given-names">R. A.</div></div>,  <div class="name"><div class="surname">Lepek</div> <div class="given-names">V. C</div></div>. </div><div class="article-title">Nodule development induced by <em>Mesorhizobium loti</em> mutant strains affected in polysaccharide synthesis</div>, <div class="source">Mol. Plant Microbe. Interact.</div> , <div class="year">2005</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">446</div>-<div class="lpage">457</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nodule%20development%20induced%20by%20Mesorhizobium%20loti%20mutant%20strains%20affected%20in%20polysaccharide%20synthesis&amp;author=A.%20L.%20D'Antuono&amp;author=A.%20Casabuono&amp;author=A.%20Couto&amp;author=R.%20A.%20Ugalde&amp;author=V.%20C%20Lepek&amp;publication_year=2005&amp;journal=Mol.%20Plant%20Microbe.%20Interact.&amp;volume=18&amp;pages=446-457" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI-18-0446" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI-18-0446" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI-18-0446"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15915643" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN017C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Okazaki</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Hattori</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Saeki</div> <div class="given-names">K</div></div>. </div><div class="article-title">The <em>Mesorhizobium loti</em> purB gene is involved in infection thread formation and nodule development in <em>Lotus japonicus</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2007</div>, vol. <div class="volume">189</div> (pg. <div class="fpage">8347</div>-<div class="lpage">8352</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Mesorhizobium%20loti%20purB%20gene%20is%20involved%20in%20infection%20thread%20formation%20and%20nodule%20development%20in%20Lotus%20japonicus&amp;author=S.%20Okazaki&amp;author=Y.%20Hattori&amp;author=K%20Saeki&amp;publication_year=2007&amp;journal=J.%20Bacteriol.&amp;volume=189&amp;pages=8347-8352" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.00788-07" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.00788-07" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.00788-07"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17827288" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN017C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guerrero</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Peralta</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Aguilar</div> <div class="given-names">A.,</div></div>, et al. </div><div class="article-title">Evolutionary, structural and functional relationships revealed by comparative analysis of syntenic genes in Rhizobiales</div>, <div class="source">BMC Evol. Biol.</div> , <div class="year">2005</div>, vol. <div class="volume">5</div> pg. <div class="fpage">55</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%2C%20structural%20and%20functional%20relationships%20revealed%20by%20comparative%20analysis%20of%20syntenic%20genes%20in%20Rhizobiales&amp;author=G.%20Guerrero&amp;author=H.%20Peralta&amp;author=A.%2C%20Aguilar&amp;publication_year=2005&amp;journal=BMC%20Evol.%20Biol.&amp;volume=5&amp;pages=55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-5-55" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-5-55" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-5-55"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16229745" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN017C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Finan</div> <div class="given-names">T. M.</div></div>,  <div class="name"><div class="surname">Kunkel</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">De Vos</div> <div class="given-names">G. F.</div></div>,  <div class="name"><div class="surname">Signer</div> <div class="given-names">E. R</div></div>. </div><div class="article-title">Second symbiotic megaplasmid in <em>Rhizobium meliloti</em> carrying exopolysaccharide and thiamine synthesis genes</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1986</div>, vol. <div class="volume">167</div> (pg. <div class="fpage">66</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Second%20symbiotic%20megaplasmid%20in%20Rhizobium%20meliloti%20carrying%20exopolysaccharide%20and%20thiamine%20synthesis%20genes&amp;author=T.%20M.%20Finan&amp;author=B.%20Kunkel&amp;author=G.%20F.%20De%20Vos&amp;author=E.%20R%20Signer&amp;publication_year=1986&amp;journal=J.%20Bacteriol.&amp;volume=167&amp;pages=66-72" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3013840" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN017C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Beringer</div> <div class="given-names">J. E</div></div>. </div><div class="article-title">R factor transfer in <em>Rhizobium leguminosarum</em></div>, <div class="source">J. Gen. Microbiol.</div> , <div class="year">1974</div>, vol. <div class="volume">84</div> (pg. <div class="fpage">188</div>-<div class="lpage">198</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=R%20factor%20transfer%20in%20Rhizobium%20leguminosarum&amp;author=J.%20E%20Beringer&amp;publication_year=1974&amp;journal=J.%20Gen.%20Microbiol.&amp;volume=84&amp;pages=188-198" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4612098" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN017C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dennis</div> <div class="given-names">J. J.</div></div>,  <div class="name"><div class="surname">Zylstra</div> <div class="given-names">G. J</div></div>. </div><div class="article-title">Plasposon: modular self-clonig minitransposon derivatives for rapid genetic analysis of Gram-negative bacterial genomes</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">1998</div>, vol. <div class="volume">64</div> (pg. <div class="fpage">2710</div>-<div class="lpage">2715</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plasposon%3A%20modular%20self-clonig%20minitransposon%20derivatives%20for%20rapid%20genetic%20analysis%20of%20Gram-negative%20bacterial%20genomes&amp;author=J.%20J.%20Dennis&amp;author=G.%20J%20Zylstra&amp;publication_year=1998&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=64&amp;pages=2710-2715" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9647854" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN017C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Prentki</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Kirsch</div> <div class="given-names">H. M</div></div>. </div><div class="article-title"><em>In vitro</em> insertional mutagenesis with a selectable DNA fragment</div>, <div class="source">Gene</div> , <div class="year">1984</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">303</div>-<div class="lpage">313</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20vitro%20insertional%20mutagenesis%20with%20a%20selectable%20DNA%20fragment&amp;author=P.%20Prentki&amp;author=H.%20M%20Kirsch&amp;publication_year=1984&amp;journal=Gene&amp;volume=29&amp;pages=303-313" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119(84)90059-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119(84)90059-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0378-1119(84)90059-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6237955" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN017C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Simon</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Priefer</div> <div class="given-names">U.</div></div>,  <div class="name"><div class="surname">Pühler</div> <div class="given-names">A</div></div>. </div><div class="article-title">A broad host range mobilization system for in vivo genetic engineering: transposon mutagenesis in gram negative bacteria</div>, <div class="source">Bio/Technology</div> , <div class="year">1983</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">784</div>-<div class="lpage">791</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20broad%20host%20range%20mobilization%20system%20for%20in%20vivo%20genetic%20engineering%3A%20transposon%20mutagenesis%20in%20gram%20negative%20bacteria&amp;author=R.%20Simon&amp;author=U.%20Priefer&amp;author=A%20P%C3%BChler&amp;publication_year=1983&amp;journal=Bio%2FTechnology&amp;volume=1&amp;pages=784-791" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1183-784" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1183-784" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1183-784"> </span></div> </div></div></div></div></div><div content-id="DSN017C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hunt</div> <div class="given-names">T. A.</div></div>,  <div class="name"><div class="surname">Kooi</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Sokol</div> <div class="given-names">P. A.</div></div>,  <div class="name"><div class="surname">Valvano</div> <div class="given-names">M. A</div></div>. </div><div class="article-title">Identification of <em>Burkholderia cenocepacia</em> genes required for bacterial survival in vivo</div>, <div class="source">Infect. Immun.</div> , <div class="year">2004</div>, vol. <div class="volume">72</div> (pg. <div class="fpage">4010</div>-<div class="lpage">4022</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20Burkholderia%20cenocepacia%20genes%20required%20for%20bacterial%20survival%20in%20vivo&amp;author=T.%20A.%20Hunt&amp;author=C.%20Kooi&amp;author=P.%20A.%20Sokol&amp;author=M.%20A%20Valvano&amp;publication_year=2004&amp;journal=Infect.%20Immun.&amp;volume=72&amp;pages=4010-4022" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/IAI.72.7.4010-4022.2004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FIAI.72.7.4010-4022.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FIAI.72.7.4010-4022.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15213146" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN017C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Autret</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Charbit</div> <div class="given-names">A</div></div>. </div><div class="article-title">Lessons from signature-tagged mutagenesis on the infectious mechanisms of pathogenic bacteria</div>, <div class="source">FEMS Microbiol. Rev.</div> , <div class="year">2005</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">703</div>-<div class="lpage">717</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lessons%20from%20signature-tagged%20mutagenesis%20on%20the%20infectious%20mechanisms%20of%20pathogenic%20bacteria&amp;author=A.%20Autret&amp;author=A%20Charbit&amp;publication_year=2005&amp;journal=FEMS%20Microbiol.%20Rev.&amp;volume=29&amp;pages=703-717" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.femsre.2004.10.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.femsre.2004.10.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.femsre.2004.10.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16102599" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN017C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Reznikoff</div> <div class="given-names">W. S</div></div>. </div><div class="article-title">Tn5 as a model for understanding DNA transposition</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2003</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">1199</div>-<div class="lpage">1206</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tn5%20as%20a%20model%20for%20understanding%20DNA%20transposition&amp;author=W.%20S%20Reznikoff&amp;publication_year=2003&amp;journal=Mol.%20Microbiol.&amp;volume=47&amp;pages=1199-1206" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.2003.03382.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.2003.03382.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.2003.03382.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12603728" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN017C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Groh</div> <div class="given-names">J. L.</div></div>,  <div class="name"><div class="surname">Luo</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Ballard</div> <div class="given-names">J. D.</div></div>,  <div class="name"><div class="surname">Krumholz</div> <div class="given-names">L. R</div></div>. </div><div class="article-title">A method adapting microarray technology for signature-tagged mutagenesis of <em>Desulfovibrio desulfuricans</em> G20 and <em>Shewanella oneidensis</em> MR-1 in anaerobic sediment survival experiments</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">2005</div>, vol. <div class="volume">71</div> (pg. <div class="fpage">7064</div>-<div class="lpage">7074</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20method%20adapting%20microarray%20technology%20for%20signature-tagged%20mutagenesis%20of%20Desulfovibrio%20desulfuricans%20G20%20and%20Shewanella%20oneidensis%20MR-1%20in%20anaerobic%20sediment%20survival%20experiments&amp;author=J.%20L.%20Groh&amp;author=Q.%20Luo&amp;author=J.%20D.%20Ballard&amp;author=L.%20R%20Krumholz&amp;publication_year=2005&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=71&amp;pages=7064-7074" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.71.11.7064-7074.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.71.11.7064-7074.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.71.11.7064-7074.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16269742" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN017C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suzuki</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Okai</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Nonaka</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Tsuge</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Inui</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Yukawa</div> <div class="given-names">H</div></div>. </div><div class="article-title">High-throughput transposon mutagenesis of <em>Corynebacterium glutamicum</em> and construction of a single-gene disruptant mutant library</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">2006</div>, vol. <div class="volume">72</div> (pg. <div class="fpage">3750</div>-<div class="lpage">3755</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20transposon%20mutagenesis%20of%20Corynebacterium%20glutamicum%20and%20construction%20of%20a%20single-gene%20disruptant%20mutant%20library&amp;author=N.%20Suzuki&amp;author=N.%20Okai&amp;author=H.%20Nonaka&amp;author=Y.%20Tsuge&amp;author=M.%20Inui&amp;author=H%20Yukawa&amp;publication_year=2006&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=72&amp;pages=3750-3755" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.72.5.3750-3755.2006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.72.5.3750-3755.2006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.72.5.3750-3755.2006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16672528" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN017C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gallagher</div> <div class="given-names">L. A.</div></div>,  <div class="name"><div class="surname">Ramage</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Jacobs</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Kaul</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Brittnacher</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Manoil</div> <div class="given-names">A</div></div>. </div><div class="article-title">A comprehensive transposon mutant library of <em>Francisella novicida</em>, a bioweapon surrogate</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2007</div>, vol. <div class="volume">104</div> (pg. <div class="fpage">1009</div>-<div class="lpage">1014</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comprehensive%20transposon%20mutant%20library%20of%20Francisella%20novicida%2C%20a%20bioweapon%20surrogate&amp;author=L.%20A.%20Gallagher&amp;author=E.%20Ramage&amp;author=M.%20A.%20Jacobs&amp;author=R.%20Kaul&amp;author=M.%20Brittnacher&amp;author=A%20Manoil&amp;publication_year=2007&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=104&amp;pages=1009-1014" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0606713104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0606713104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0606713104"> </span></div> </div></div></div></div></div><div content-id="DSN017C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mormann</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Lömker</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Rückert</div> <div class="given-names">C.,</div></div>, et al. </div><div class="article-title">Random mutagenesis in <em>Corynebacterium glutamicum</em> ATCC 13032 using an IS6100-based transposon vector identified the last unknown gene in the histidine biosynthesis pathway</div>, <div class="source">BMC Genomics</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">205</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Random%20mutagenesis%20in%20Corynebacterium%20glutamicum%20ATCC%2013032%20using%20an%20IS6100-based%20transposon%20vector%20identified%20the%20last%20unknown%20gene%20in%20the%20histidine%20biosynthesis%20pathway&amp;author=S.%20Mormann&amp;author=A.%20L%C3%B6mker&amp;author=C.%2C%20R%C3%BCckert&amp;publication_year=2006&amp;journal=BMC%20Genomics&amp;volume=7&amp;pages=205" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-7-205" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-7-205" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-7-205"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16901339" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN017C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rigoutsos</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Floratos</div> <div class="given-names">A</div></div>. </div><div class="article-title">Combinatorial pattern discovery in biological sequences: the TEIRESIAS algorithm</div>, <div class="source">Bioinformatics</div> , <div class="year">1998</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">55</div>-<div class="lpage">67</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Combinatorial%20pattern%20discovery%20in%20biological%20sequences%3A%20the%20TEIRESIAS%20algorithm&amp;author=I.%20Rigoutsos&amp;author=A%20Floratos&amp;publication_year=1998&amp;journal=Bioinformatics&amp;volume=14&amp;pages=55-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/14.1.55" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F14.1.55" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F14.1.55"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9520502" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN017C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jacobs</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Alwood</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Thaipisuttikul</div> <div class="given-names">I.,</div></div>, et al. </div><div class="article-title">Comprehensive transposon mutant library of <em>Pseudomonas aeruginosa</em></div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2003</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">14339</div>-<div class="lpage">14344</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20transposon%20mutant%20library%20of%20Pseudomonas%20aeruginosa&amp;author=M.%20A.%20Jacobs&amp;author=A.%20Alwood&amp;author=I.%2C%20Thaipisuttikul&amp;publication_year=2003&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=100&amp;pages=14339-14344" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.2036282100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.2036282100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.2036282100"> </span></div> </div></div></div></div></div><div content-id="DSN017C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liberati</div> <div class="given-names">N. T.</div></div>,  <div class="name"><div class="surname">Urbach</div> <div class="given-names">J. M.</div></div>,  <div class="name"><div class="surname">Miyata</div> <div class="given-names">S.,</div></div>, et al. </div><div class="article-title">An ordered, nonredundant library of <em>Pseudomonas aeruginosa</em> strain PA14 transposon insertion mutants</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2006</div>, vol. <div class="volume">103</div> (pg. <div class="fpage">2833</div>-<div class="lpage">2838</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20ordered%2C%20nonredundant%20library%20of%20Pseudomonas%20aeruginosa%20strain%20PA14%20transposon%20insertion%20mutants&amp;author=N.%20T.%20Liberati&amp;author=J.%20M.%20Urbach&amp;author=S.%2C%20Miyata&amp;publication_year=2006&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=103&amp;pages=2833-2838" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0511100103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0511100103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0511100103"> </span></div> </div></div></div></div></div><div content-id="DSN017C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">de Berardinis</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Vallenet</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Castelli</div> <div class="given-names">V.,</div></div>, et al. </div><div class="article-title">A complete collection of single-gene deletion mutants of <em>Acinetobacter baylyi</em> ADP1</div>, <div class="source">Mol. Syst. Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">4</div> pg. <div class="fpage">174</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20complete%20collection%20of%20single-gene%20deletion%20mutants%20of%20Acinetobacter%20baylyi%20ADP1&amp;author=V.%20de%20Berardinis&amp;author=D.%20Vallenet&amp;author=V.%2C%20Castelli&amp;publication_year=2008&amp;journal=Mol.%20Syst.%20Biol.&amp;volume=4&amp;pages=174" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/msb.2008.10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fmsb.2008.10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fmsb.2008.10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18319726" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN017C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN017C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gerdes</div> <div class="given-names">S. Y.</div></div>,  <div class="name"><div class="surname">Scholle</div> <div class="given-names">M. D.</div></div>,  <div class="name"><div class="surname">Campbell</div> <div class="given-names">J. W.,</div></div>, et al. </div><div class="article-title">Experimental determination and system level analysis of essential genes in <em>Escherichia coli</em> MG1655</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2003</div>, vol. <div class="volume">185</div> (pg. <div class="fpage">5673</div>-<div class="lpage">5684</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Experimental%20determination%20and%20system%20level%20analysis%20of%20essential%20genes%20in%20Escherichia%20coli%20MG1655&amp;author=S.%20Y.%20Gerdes&amp;author=M.%20D.%20Scholle&amp;author=J.%20W.%2C%20Campbell&amp;publication_year=2003&amp;journal=J.%20Bacteriol.&amp;volume=185&amp;pages=5673-5684" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.185.19.5673-5684.2003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.185.19.5673-5684.2003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.185.19.5673-5684.2003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/13129938" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2008. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=359537&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn017/2/dsn017-dnar-2008-029-File014.xls?Expires=2147483647&amp;Signature=DzUrjsN7EJc6-LHl2nAzKKMe5Q1mPSH-dUD5pcpctHW0D-YvhHN8MY-LQmED0d8UfH2p9GN2kV~J-nyUKu58M~ifh-54zykhZVDXrO~0gDMN0SV2MxcwiKLKhJ7kpUPyq2DrsgLG49WxLAORkf5u4Ch9-6WBZ7~MXd9n8obfF5Ct1D0mSLPvG2SSxdoDKbGrBKqbnjFOlpdMlZiSnO1IbCOST~cMjZEotn-wCd7GwZWmcqZU~qEG9ueQ6oYSvIfmWSGEMCPnBORRHSQpn5uietmAo9eJPW8U113IKkEvR~qW8qYuE11uFzVT7XEByzLkl~K8c48LSFJI474P08BvAA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn017/2/dsn017-dnar-2008-029-File013.xls?Expires=2147483647&amp;Signature=lkGX49j9T8oqf5uUc0HhXl8zEisFFgrxvqEOWsbL-NvYwlnaHBtS2lsmdbhUzvVyYW0jf9y33sHCQr0RFIygIAXwPoDzcY8~sQ8dsPqsrgzMS7DzByLlw0k1qwU6-zfC5KFZp5w0HhTmeflwg73H8WFc0OPvFtw8gcXCO-kGGa2WPEssYIiYUJbGKXoCQt~N8i1yZ2c0RvqiqEsE-XuAXsXglDEhs~rq5db1TVUgX83ZmSo5B72MEh3wuMzkTX~EFC4UOKY4UMPYURGdFLgqxuWAwEB9mbQCSLE6Jtxv4i4rc1NJZi8Yzdg2Cmvtk-1IkqlG8scRgvZtGJyXtPn7mw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn017/2/dsn017-dnar-2008-029-File012.xls?Expires=2147483647&amp;Signature=QHW7ZTdInHcZNdxbgkVW6HUlLqBsWOH6pQRPR3EuJXKuM5cfNaA7snjW6QrE0G1rg35RCbzngfJoWkDya3UC2tW2tdcZ0-d-9LRFcxOViQ8pH5BuMzibWblXZ9SR0enVQE7i~Dwkc5uqU2Zgd2T9ZgRk9TBkPF4GKtMOO8XapDIna9xeUeyyMF2sZjK-uA6cfa1nJJb4u4W0Q5yX3ayAU-qlJoJEXWDw2RkwYtFuXgJTrtDKIc-L60ZvEzkCrnmJCN3DAhLMZorsyWHptfz7T6LtsRWi0rslWSGP-0naHzvi9mG3Ek-nSKgr47jDYCZGdM28bg~wt1YUlj249Nxceg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn017/2/dsn017-dnar-2008-029-File011.xls?Expires=2147483647&amp;Signature=22qJIFhB2QrzCJsXHw68ygA3zH0UJP9~SsDPVIAJtgtyFkNO8zyGu6jrcyM-DS9ZFDsJ2OfFCMB7MvPrPERNz5gelypIoDguydz-HBGZAxJYkmtIpXfGY3pLBJSiabNQX6LmBGJ3GjxVTEzm84S7hFi4i3-29YQq7jfm~yQR9lOCEa2T7vff9PSJDaq4JMuYrwR72PKy73GXniALwWCaKAGmvsws1rYeVQ7kZKxFNDPWPcjZNic99JCI6mD-wr12P1RAwRbjrOTlxTmARX2~FdFjEX47-MKcIMD1zepJdFkV-EDpRIG8vqAGnUsknXGvyPt-7WJQ0wmTgrzf5gBkkg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn017/2/dsn017-dnar-2008-029-File010.xls?Expires=2147483647&amp;Signature=wFh4xtKIsnHGnV6WuyQbficAw9vi5syTJHMY8B-vmARTe~mcXIVUlhO5ir~SeL~xM5U27EKBg-yywCDQei6J0LZLyW7SEQcC2z3rAh3XlEMBykRfn85Vdgh2ewPiNNRX-8QiyMz~coGyQqu2j56FGZ8-BytwFGIA0WdxtDWI1BSwle7NpeoAccjBeLWObW7I8AhiR~wbJONEYZddE37yROokRt~H3daCK5CRLZWLaGlC5Yob5fWonSI-IqLmviPKjuo9-VBolXcQfdcBXgmK6TDxFWCApqVI6YSZm6ZU-Rx-JU5zyUL1i2Fl9BX43ZEwrX4aEilGZqt39pWUMa05nA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/5/10.1093/dnares/dsn017/2/dsn017-dnar-2008-029-File009.pdf?Expires=2147483647&amp;Signature=IAQYHcsQDcp9us-uxysb4EXbtJ16B1Fm7FovQ2T3~zr0mEyiAxiENqGNROwbdW-7NuUezgscXBXY2d-rkwxzawaH2k6kwC1ZzEm1MGShTDtbCVkWy78i0qU-v6vM5kAITnGWDK2hvkTDySC9yA5uUkBvGb1GjCKNZtw23wQEus5XAuAFljJK37Q86zBbDFmVqgMMOACuwmqmsw7XVthjcuj0QrQdEfOFuz9HWG-HeyL2tcY9nvUMU1ZWNQ0qGALj1ZWcczj14EUOjJ9PMKiOnhrWzHTRQGRogEIEa-wRMYKdLfJtmOQn9zA~JOa0altpWMHssr1g4TnRLPEqjPWAEw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">296</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000260843400004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">24</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsn017"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="359537"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000260843400004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f15%2f5%2f297%2f359537">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000260843400004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (24)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F15%2F5%2F297%2F359537" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/359537'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Construction of Signature-tagged Mutant Library in Mesorhizobium loti as a Powerful Tool for Functional Genomics" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsn017");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f15%2f5%2f297%2f359537">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

