

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/17/6/381/484555","@type":"ScholarlyArticle","name":"SNP Discovery and Linkage Map Construction in Cultivated Tomato","datePublished":"2010-11-02","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/17/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2010-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsq024","keywords":["DNA marker","linkage map","single nucleotide polymorphism","<italic>Solanum lycopersicum</italic>","tomato"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/17/6/381/484555","author":[{"name":"Shirasawa, Kenta","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Isobe, Sachiko","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Hirakawa, Hideki","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Asamizu, Erika","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Fukuoka, Hiroyuki","affiliation":"2  National Institute of Vegetable and Tea Science (NIVTS) ,  National Agriculture and Food Research Organization ,  360 Kusawa, Ano, Tsu, Mie 514-2392 ,  Japan","@type":"Person"},{"name":"Just, Daniel","affiliation":"3  Unité Mixte de Recherche 619, Biologie du Fruit, Institut National de la Recherche Agronomique (INRA Bordeaux), Universités de Bordeaux ,  33883 Villenave d'Ornon cedex ,  France","@type":"Person"},{"name":"Rothan, Christophe","affiliation":"3  Unité Mixte de Recherche 619, Biologie du Fruit, Institut National de la Recherche Agronomique (INRA Bordeaux), Universités de Bordeaux ,  33883 Villenave d'Ornon cedex ,  France","@type":"Person"},{"name":"Sasamoto, Shigemi","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Fujishiro, Tsunakazu","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kishida, Yoshie","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kohara, Mitsuyo","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Tsuruoka, Hisano","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Wada, Tsuyuko","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Nakamura, Yasukazu","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Sato, Shusei","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Tabata, Satoshi","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"}],"description":"Abstract.  Few intraspecific genetic linkage maps have been reported for cultivated tomato, mainly because genetic diversity within Solanum lycopersicum is much","pageStart":"381","pageEnd":"391","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/6/1/m_cover.gif?Expires=2147483647&Signature=GE4QDx7vPXS7wjUaoLIKdeQMVYvKpNXob0I6QlcCM7oe~eKCfP8E380kotmiw9z~axHxmBJESzx9vV88~C9g-CA5dZPWzHX3y0HSadJ2Jrf2k3i9guvDwVaIIvgF6cZI1iLndCWPhjSM1C0sC8Omes66kjDc9sn4S8VlQHryn-wE9mPjK2sZ-quW-rVkH966kcOElJ2hx~w7cpINeghmflNy5cmWJN-jp6Sp3iFBqCubFpEA3485f5KqRLMCbhc5nMYcpZWmmznkyRdy0CL0lxP-Xr1JDHnrIseUrn4fTYBDpMQY0Dv8f9-vnTHQwjT0Mz0eJIlyt6vd81Yiu4619w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"SNP Discovery and Linkage Map Construction in Cultivated Tomato","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/6/1/m_cover.gif?Expires=2147483647&Signature=GE4QDx7vPXS7wjUaoLIKdeQMVYvKpNXob0I6QlcCM7oe~eKCfP8E380kotmiw9z~axHxmBJESzx9vV88~C9g-CA5dZPWzHX3y0HSadJ2Jrf2k3i9guvDwVaIIvgF6cZI1iLndCWPhjSM1C0sC8Omes66kjDc9sn4S8VlQHryn-wE9mPjK2sZ-quW-rVkH966kcOElJ2hx~w7cpINeghmflNy5cmWJN-jp6Sp3iFBqCubFpEA3485f5KqRLMCbhc5nMYcpZWmmznkyRdy0CL0lxP-Xr1JDHnrIseUrn4fTYBDpMQY0Dv8f9-vnTHQwjT0Mz0eJIlyt6vd81Yiu4619w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="SNP Discovery and Linkage Map Construction in Cultivated Tomato" />
    <meta property="og:description" content="Abstract.  Few intraspecific genetic linkage maps have been reported for cultivated tomato, mainly because genetic diversity within Solanum lycopersicum is much" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/17/6/381/484555" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2010-11-02" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Shirasawa, Kenta" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Isobe, Sachiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Hirakawa, Hideki" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Asamizu, Erika" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Fukuoka, Hiroyuki" /><meta name="citation_author_institution" content="National Institute of Vegetable and Tea Science (NIVTS), National Agriculture and Food Research Organization, 360 Kusawa, Ano, Tsu, Mie 514-2392, Japan" /><meta name="citation_author" content="Just, Daniel" /><meta name="citation_author_institution" content="Unité Mixte de Recherche 619, Biologie du Fruit, Institut National de la Recherche Agronomique (INRA Bordeaux), Universités de Bordeaux, 33883 Villenave d'Ornon cedex, France" /><meta name="citation_author" content="Rothan, Christophe" /><meta name="citation_author_institution" content="Unité Mixte de Recherche 619, Biologie du Fruit, Institut National de la Recherche Agronomique (INRA Bordeaux), Universités de Bordeaux, 33883 Villenave d'Ornon cedex, France" /><meta name="citation_author" content="Sasamoto, Shigemi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Fujishiro, Tsunakazu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kishida, Yoshie" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kohara, Mitsuyo" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Tsuruoka, Hisano" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Wada, Tsuyuko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Nakamura, Yasukazu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Sato, Shusei" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Tabata, Satoshi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_title" content="SNP Discovery and Linkage Map Construction in Cultivated Tomato" /><meta name="citation_firstpage" content="381" /><meta name="citation_lastpage" content="391" /><meta name="citation_doi" content="10.1093/dnares/dsq024" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="17" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2010/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Tomato; citation_publisher=Springer, New York; Genome Mapping and Molecular Breeding in Plants, vol. 5;  citation_year=2007;  citation_pages=1-125; " /><meta name="citation_reference" content="citation_title=Resolution of quantitative traits into Mendelian factors by using a complete linkage map of restriction fragment length polymorphisms; Nature;  citation_year=1988;  citation_volume=335;  citation_pages=721-726; " /><meta name="citation_reference" content="citation_title=High density molecular linkage maps of the tomato and potato genomes; Genetics;  citation_year=1992;  citation_volume=132;  citation_pages=1141-1160; " /><meta name="citation_reference" content="citation_title=QTL analysis of transgressive segregation in an interspecific tomato cross; Genetics;  citation_year=1993;  citation_volume=134;  citation_pages=585-596; " /><meta name="citation_reference" content="citation_title=Recombinant inbred lines for genetic mapping in tomato; Theor. Appl. Genet.;  citation_year=1995;  citation_volume=90;  citation_pages=542-548; " /><meta name="citation_reference" content="citation_title=Introgression and DNA marker analysis of Lycopersicon peruvianum, a wild relative of the cultivated tomato, into Lycopersicon esculentum, followed through three successive backcross generations; Theor. Appl. Genet.;  citation_year=1997;  citation_volume=95;  citation_pages=895-902; " /><meta name="citation_reference" content="citation_title=An interspecific linkage map of SSR and intronic polymorphism markers in tomato; Theor. Appl. Genet.;  citation_year=2010;  citation_volume=121;  citation_pages=731-739; " /><meta name="citation_reference" content="citation_title=fw2.2: a quantitative trait locus key to the evolution of tomato fruit size; Science;  citation_year=2000;  citation_volume=289;  citation_pages=85-88; " /><meta name="citation_reference" content="citation_title=A new class of regulatory genes underlying the cause of pear-shaped tomato fruit; Proc. Natl Acad. Sci. USA;  citation_year=2002;  citation_volume=99;  citation_pages=13302-13306; " /><meta name="citation_reference" content="citation_title=Regulatory change in YABBY-like transcription factor led to evolution of extreme fruit size during tomato domestication; Nat. Genet.;  citation_year=2008;  citation_volume=40;  citation_pages=800-804; " /><meta name="citation_reference" content="citation_title=A retrotransposon-mediated gene duplication underlies morphological variation of tomato fruit; Science;  citation_year=2008;  citation_volume=319;  citation_pages=1527-1530; " /><meta name="citation_reference" content="citation_title=The tomato Cf-2 disease resistance locus comprises two functional genes encoding leucine-rich repeat proteins; Cell;  citation_year=1996;  citation_volume=84;  citation_pages=451-459; " /><meta name="citation_reference" content="citation_title=Map-based cloning of a protein kinase gene conferring disease resistance in tomato; Science;  citation_year=1993;  citation_volume=262;  citation_pages=1432-1436; " /><meta name="citation_reference" content="citation_title=Isolation of the tomato Cf-9 gene for resistance to Cladosporium fulvum by transposon tagging; Science;  citation_year=1994;  citation_volume=266;  citation_pages=789-793; " /><meta name="citation_reference" content="citation_title=Characterization of the tomato Cf-4 gene for resistance to Cladosporium fulvum identifies sequences that determine recognitional specificity in Cf-4 and Cf-9; Plant. Cell;  citation_year=1997;  citation_volume=9;  citation_pages=2209-2224; " /><meta name="citation_reference" content="citation_title=A recombination hotspot delimits a wild-species quantitative trait locus for tomato sugar content to 484 bp within an invertase gene; Proc. Natl. Acad. Sci. USA;  citation_year=2000;  citation_volume=97;  citation_pages=4718-4723; " /><meta name="citation_reference" content="citation_title=RFLP analysis of phylogenetic relationships and genetic variation in the genus Lycopersicon; Theor. Appl. Genet.;  citation_year=1990;  citation_volume=80;  citation_pages=437-448; " /><meta name="citation_reference" content="citation_title=Efficiency of RFLP, RAPD, and AFLP markers for the construction of an intraspecific map of the tomato genome; Genome;  citation_year=2000;  citation_volume=43;  citation_pages=29-40; " /><meta name="citation_reference" content="citation_title=Development of a set of PCR-based anchor markers encompassing the tomato genome and evaluation of their usefulness for genetics and breeding experiments; Theor. Appl. Genet.;  citation_year=2005;  citation_volume=111;  citation_pages=291-312; " /><meta name="citation_reference" content="citation_title=Characterization of tomato SSR markers developed using BAC-end and cDNA sequences from genome databases; Mol. Breed.;  citation_year=2009;  citation_volume=23;  citation_pages=685-691; " /><meta name="citation_reference" content="citation_title=Cross-species bacterial artificial chromosome-fluorescence in situ hybridization painting of the tomato and potato chromosome 6 reveals undescribed chromosomal rearrangements; Genetics;  citation_year=2008;  citation_volume=180;  citation_pages=1319-1328; " /><meta name="citation_reference" content="citation_title=Euchromatin and pericentromeric heterochromatin: comparative composition in the tomato genome; Genetics;  citation_year=2006;  citation_volume=172;  citation_pages=2529-2540; " /><meta name="citation_reference" content="citation_title=Array-based high-throughput DNA markers for crop improvement; Heredity;  citation_year=2008;  citation_volume=101;  citation_pages=5-18; " /><meta name="citation_reference" content="citation_title=Sequence diversity in three tomato species: SNPs, markers, and molecular evolution; BMC Plant Biol.;  citation_year=2009;  citation_volume=9;  citation_pages=85" /><meta name="citation_reference" content="citation_title=Tomato SNP discovery by EST mining and resequencing; Mol. Breed.;  citation_year=2005;  citation_volume=16;  citation_pages=343-349; " /><meta name="citation_reference" content="citation_title=Discovery of single nucleotide polymorphisms in Lycopersicon esculentum by computer aided analysis of expressed sequence tags; Mol. Breed.;  citation_year=2004;  citation_volume=14;  citation_pages=21-34; " /><meta name="citation_reference" content="citation_title=Expressed sequence tags from the laboratory-grown miniature tomato (Lycopersicon esculentum) cultivar Micro-Tom and mining for single nucleotide polymorphisms and insertions/deletions in tomato cultivars; Gene;  citation_year=2005;  citation_volume=15;  citation_pages=127-134; " /><meta name="citation_reference" content="citation_title=Tomato is a next-generation model plant for research and development; J. Jpn Soc. Hort. Sci.;  citation_year=2009;  citation_volume=78;  citation_pages=1-2; " /><meta name="citation_reference" content="citation_title=Micro-Tom—a miniature dwarf tomato; Florida Agr. Expt. Sta. Circ.;  citation_year=1989;  citation_volume=370;  citation_pages=1-6; " /><meta name="citation_reference" content="citation_title=A new model system for tomato genetics; Plant J.;  citation_year=1997;  citation_volume=12;  citation_pages=1465-1472; " /><meta name="citation_reference" content="citation_title=Generation of gamma irradiation-induced mutant lines of the miniature tomato (Solanum lycopersicum L.) cultivar ‘Micro-Tom; Plant Biotechnol.;  citation_year=2007;  citation_volume=24;  citation_pages=39-44; " /><meta name="citation_reference" content="citation_title=Ethylmethanesulfonate (EMS) mutagenesis of Solanum lycopersicum cv. Micro-Tom for large-scale mutant screens; Plant Biotechnol.;  citation_year=2007;  citation_volume=24;  citation_pages=33-38; " /><meta name="citation_reference" content="citation_title=MicroTom—a high-throughput model transformation system for functional genomics; Plant Cell Rep.;  citation_year=2006;  citation_volume=25;  citation_pages=432-441; " /><meta name="citation_reference" content="citation_title=A highly efficient transformation protocol for Micro-Tom, a model cultivar for tomato functional genomics; Plant Cell Physiol.;  citation_year=2006;  citation_volume=47;  citation_pages=426-431; " /><meta name="citation_reference" content="citation_title=Metabolite annotations based on the integration of mass spectral information; Plant J.;  citation_year=2008;  citation_volume=54;  citation_pages=949-962; " /><meta name="citation_reference" content="citation_title=Large-scale analysis of full-length cDNAs from the tomato (Solanum lycopersicum) cultivar Micro-Tom, a reference system for the Solanaceae genomics; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=210" /><meta name="citation_reference" content="citation_title=Using the miraEST assembler for reliable and automated mRNA transcript assembly and SNP detection in sequenced ESTs; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=1147-1159; " /><meta name="citation_reference" content="citation_title=Primer3 on the WWW for general users and for biologist programmers; Methods Mol. Biol.;  citation_year=2000;  citation_volume=132;  citation_pages=365-386; " /><meta name="citation_reference" content="citation_title=Comprehensive structural analysis of the genome of red clover (Trifolium pratense L.); DNA Res.;  citation_year=2005;  citation_volume=12;  citation_pages=301-364; " /><meta name="citation_reference" content="citation_publisher=Kyazma. BV, Wageningen, Netherlands; JoinMap®4, Software for the Calculation of Genetic Linkage Maps in Experimental Populations;  citation_year=2006; " /><meta name="citation_reference" content="citation_title=Genetic and physiological characterization of tomato cv. Micro-Tom; J. Exp. Bot.;  citation_year=2006;  citation_volume=57;  citation_pages=2037-2047; " /><meta name="citation_reference" content="citation_title=High-throughput genotyping with the GoldenGate assay in the complex genome of soybean; Theor. Appl. Genet.;  citation_year=2008;  citation_volume=116;  citation_pages=945-952; " /><meta name="citation_reference" content="citation_title=A consensus genetic map of cowpea [Vigna unguiculata (L) Walp.] and synteny based on EST-derived SNPs; Proc. Natl Acad. Sci. USA;  citation_year=2009;  citation_volume=106;  citation_pages=18159-18164; " /><meta name="citation_reference" content="citation_title=High-throughput SNP discovery and assay development in common bean; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=475" /><meta name="citation_reference" content="citation_title=Highly-multiplexed SNP genotyping for genetic mapping and germplasm diversity studies in pea; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=468" /><meta name="citation_reference" content="citation_title=Genetic and biosystematic studies on two new sibling species of Lycopersicon from interandean Peru; Theor. Appl. Genet.;  citation_year=1976;  citation_volume=47;  citation_pages=55-68; " /><meta name="citation_reference" content="citation_title=Catalog of Micro-Tom tomato responses to common fungal, bacterial, and viral pathogens; J. Gen. Plant Pathol.;  citation_year=2005;  citation_volume=71;  citation_pages=8-22; " /><meta name="citation_reference" content="citation_title=EST, COSII, and arbitrary gene markers give similar estimates of nucleotide diversity in cultivated tomato (Solanum lycopersicum L.); Theor. Appl. Genet.;  citation_year=2009;  citation_volume=118;  citation_pages=1005-1014; " /><meta name="citation_reference" content="citation_title=Diversity in conserved genes in tomato; BMC Genomics;  citation_year=2007;  citation_volume=8;  citation_pages=465" /><meta name="citation_reference" content="citation_title=Advances in cereal genomics and applications in crop breeding; Trends. Biotechnol.;  citation_year=2006;  citation_volume=24;  citation_pages=490-499; " /><meta name="citation_reference" content="citation_title=A snapshot of the emerging tomato genome sequence; Plant Genome;  citation_year=2009;  citation_volume=2;  citation_pages=78-92; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/17/6/381/1309928/dsq024.pdf" /><meta name="description" content="Abstract.  Few intraspecific genetic linkage maps have been reported for cultivated tomato, mainly because genetic diversity within Solanum lycopersicum is much" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/17/6/381/484555" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/17/6/381/484555" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/17/6/381/484555");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsq024';
    </script>
<title>
	SNP Discovery and Linkage Map Construction in Cultivated Tomato | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./484555" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BeSBATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2hpcmFzYXdhLCBLZW50YSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2EtS2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJc29iZSwgU2FjaGlrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2EtS2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIaXJha2F3YSwgSGlkZWtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1LYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkFzYW1penUsIEVyaWthIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1LYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkZ1a3Vva2EsIEhpcm95dWtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgSW5zdGl0dXRlIG9mIFZlZ2V0YWJsZSBhbmQgVGVhIFNjaWVuY2UgKE5JVlRTKSwgTmF0aW9uYWwgQWdyaWN1bHR1cmUgYW5kIEZvb2QgUmVzZWFyY2ggT3JnYW5pemF0aW9uLCAzNjAgS3VzYXdhLCBBbm8sIFRzdSwgTWllIDUxNC0yMzkyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikp1c3QsIERhbmllbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlVuaXTDqSBNaXh0ZSBkZSBSZWNoZXJjaGUgNjE5LCBCaW9sb2dpZSBkdSBGcnVpdCwgSW5zdGl0dXQgTmF0aW9uYWwgZGUgbGEgUmVjaGVyY2hlIEFncm9ub21pcXVlIChJTlJBIEJvcmRlYXV4KSwgVW5pdmVyc2l0w6lzIGRlIEJvcmRlYXV4LCAzMzg4MyBWaWxsZW5hdmUgZCdPcm5vbiBjZWRleCwgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUm90aGFuLCBDaHJpc3RvcGhlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVW5pdMOpIE1peHRlIGRlIFJlY2hlcmNoZSA2MTksIEJpb2xvZ2llIGR1IEZydWl0LCBJbnN0aXR1dCBOYXRpb25hbCBkZSBsYSBSZWNoZXJjaGUgQWdyb25vbWlxdWUgKElOUkEgQm9yZGVhdXgpLCBVbml2ZXJzaXTDqXMgZGUgQm9yZGVhdXgsIDMzODgzIFZpbGxlbmF2ZSBkJ09ybm9uIGNlZGV4LCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYXNhbW90bywgU2hpZ2VtaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2EtS2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGdWppc2hpcm8sIFRzdW5ha2F6dSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2EtS2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLaXNoaWRhLCBZb3NoaWUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLUthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS29oYXJhLCBNaXRzdXlvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1LYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRzdXJ1b2thLCBIaXNhbm8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLUthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iV2FkYSwgVHN1eXVrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2EtS2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOYWthbXVyYSwgWWFzdWthenUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLUthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2F0bywgU2h1c2VpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1LYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRhYmF0YSwgU2F0b3NoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2EtS2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IlNOUCBEaXNjb3ZlcnkgYW5kIExpbmthZ2UgTWFwIENvbnN0cnVjdGlvbiBpbiBDdWx0aXZhdGVkIFRvbWF0byIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjM4MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMzkxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNxMDI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTAvMTIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ub21hdG87IGNpdGF0aW9uX3B1Ymxpc2hlcj1TcHJpbmdlciwgTmV3IFlvcms7IEdlbm9tZSBNYXBwaW5nIGFuZCBNb2xlY3VsYXIgQnJlZWRpbmcgaW4gUGxhbnRzLCB2b2wuIDU7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl9wYWdlcz0xLTEyNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVzb2x1dGlvbiBvZiBxdWFudGl0YXRpdmUgdHJhaXRzIGludG8gTWVuZGVsaWFuIGZhY3RvcnMgYnkgdXNpbmcgYSBjb21wbGV0ZSBsaW5rYWdlIG1hcCBvZiByZXN0cmljdGlvbiBmcmFnbWVudCBsZW5ndGggcG9seW1vcnBoaXNtczsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0xOTg4OyAgY2l0YXRpb25fdm9sdW1lPTMzNTsgIGNpdGF0aW9uX3BhZ2VzPTcyMS03MjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2ggZGVuc2l0eSBtb2xlY3VsYXIgbGlua2FnZSBtYXBzIG9mIHRoZSB0b21hdG8gYW5kIHBvdGF0byBnZW5vbWVzOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzI7ICBjaXRhdGlvbl9wYWdlcz0xMTQxLTExNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVFUTCBhbmFseXNpcyBvZiB0cmFuc2dyZXNzaXZlIHNlZ3JlZ2F0aW9uIGluIGFuIGludGVyc3BlY2lmaWMgdG9tYXRvIGNyb3NzOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzQ7ICBjaXRhdGlvbl9wYWdlcz01ODUtNTk2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWNvbWJpbmFudCBpbmJyZWQgbGluZXMgZm9yIGdlbmV0aWMgbWFwcGluZyBpbiB0b21hdG87IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9OTA7ICBjaXRhdGlvbl9wYWdlcz01NDItNTQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRyb2dyZXNzaW9uIGFuZCBETkEgbWFya2VyIGFuYWx5c2lzIG9mIEx5Y29wZXJzaWNvbiBwZXJ1dmlhbnVtLCBhIHdpbGQgcmVsYXRpdmUgb2YgdGhlIGN1bHRpdmF0ZWQgdG9tYXRvLCBpbnRvIEx5Y29wZXJzaWNvbiBlc2N1bGVudHVtLCBmb2xsb3dlZCB0aHJvdWdoIHRocmVlIHN1Y2Nlc3NpdmUgYmFja2Nyb3NzIGdlbmVyYXRpb25zOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTk1OyAgY2l0YXRpb25fcGFnZXM9ODk1LTkwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gaW50ZXJzcGVjaWZpYyBsaW5rYWdlIG1hcCBvZiBTU1IgYW5kIGludHJvbmljIHBvbHltb3JwaGlzbSBtYXJrZXJzIGluIHRvbWF0bzsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjE7ICBjaXRhdGlvbl9wYWdlcz03MzEtNzM5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1mdzIuMjogYSBxdWFudGl0YXRpdmUgdHJhaXQgbG9jdXMga2V5IHRvIHRoZSBldm9sdXRpb24gb2YgdG9tYXRvIGZydWl0IHNpemU7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9Mjg5OyAgY2l0YXRpb25fcGFnZXM9ODUtODg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbmV3IGNsYXNzIG9mIHJlZ3VsYXRvcnkgZ2VuZXMgdW5kZXJseWluZyB0aGUgY2F1c2Ugb2YgcGVhci1zaGFwZWQgdG9tYXRvIGZydWl0OyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTk5OyAgY2l0YXRpb25fcGFnZXM9MTMzMDItMTMzMDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlZ3VsYXRvcnkgY2hhbmdlIGluIFlBQkJZLWxpa2UgdHJhbnNjcmlwdGlvbiBmYWN0b3IgbGVkIHRvIGV2b2x1dGlvbiBvZiBleHRyZW1lIGZydWl0IHNpemUgZHVyaW5nIHRvbWF0byBkb21lc3RpY2F0aW9uOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPTgwMC04MDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcmV0cm90cmFuc3Bvc29uLW1lZGlhdGVkIGdlbmUgZHVwbGljYXRpb24gdW5kZXJsaWVzIG1vcnBob2xvZ2ljYWwgdmFyaWF0aW9uIG9mIHRvbWF0byBmcnVpdDsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zMTk7ICBjaXRhdGlvbl9wYWdlcz0xNTI3LTE1MzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSB0b21hdG8gQ2YtMiBkaXNlYXNlIHJlc2lzdGFuY2UgbG9jdXMgY29tcHJpc2VzIHR3byBmdW5jdGlvbmFsIGdlbmVzIGVuY29kaW5nIGxldWNpbmUtcmljaCByZXBlYXQgcHJvdGVpbnM7IENlbGw7ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9ODQ7ICBjaXRhdGlvbl9wYWdlcz00NTEtNDU5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXAtYmFzZWQgY2xvbmluZyBvZiBhIHByb3RlaW4ga2luYXNlIGdlbmUgY29uZmVycmluZyBkaXNlYXNlIHJlc2lzdGFuY2UgaW4gdG9tYXRvOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTI2MjsgIGNpdGF0aW9uX3BhZ2VzPTE0MzItMTQzNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNvbGF0aW9uIG9mIHRoZSB0b21hdG8gQ2YtOSBnZW5lIGZvciByZXNpc3RhbmNlIHRvIENsYWRvc3Bvcml1bSBmdWx2dW0gYnkgdHJhbnNwb3NvbiB0YWdnaW5nOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTI2NjsgIGNpdGF0aW9uX3BhZ2VzPTc4OS03OTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgdGhlIHRvbWF0byBDZi00IGdlbmUgZm9yIHJlc2lzdGFuY2UgdG8gQ2xhZG9zcG9yaXVtIGZ1bHZ1bSBpZGVudGlmaWVzIHNlcXVlbmNlcyB0aGF0IGRldGVybWluZSByZWNvZ25pdGlvbmFsIHNwZWNpZmljaXR5IGluIENmLTQgYW5kIENmLTk7IFBsYW50LiBDZWxsOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0yMjA5LTIyMjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcmVjb21iaW5hdGlvbiBob3RzcG90IGRlbGltaXRzIGEgd2lsZC1zcGVjaWVzIHF1YW50aXRhdGl2ZSB0cmFpdCBsb2N1cyBmb3IgdG9tYXRvIHN1Z2FyIGNvbnRlbnQgdG8gNDg0IGJwIHdpdGhpbiBhbiBpbnZlcnRhc2UgZ2VuZTsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9OTc7ICBjaXRhdGlvbl9wYWdlcz00NzE4LTQ3MjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJGTFAgYW5hbHlzaXMgb2YgcGh5bG9nZW5ldGljIHJlbGF0aW9uc2hpcHMgYW5kIGdlbmV0aWMgdmFyaWF0aW9uIGluIHRoZSBnZW51cyBMeWNvcGVyc2ljb247IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9ODA7ICBjaXRhdGlvbl9wYWdlcz00MzctNDQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FZmZpY2llbmN5IG9mIFJGTFAsIFJBUEQsIGFuZCBBRkxQIG1hcmtlcnMgZm9yIHRoZSBjb25zdHJ1Y3Rpb24gb2YgYW4gaW50cmFzcGVjaWZpYyBtYXAgb2YgdGhlIHRvbWF0byBnZW5vbWU7IEdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTI5LTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBvZiBhIHNldCBvZiBQQ1ItYmFzZWQgYW5jaG9yIG1hcmtlcnMgZW5jb21wYXNzaW5nIHRoZSB0b21hdG8gZ2Vub21lIGFuZCBldmFsdWF0aW9uIG9mIHRoZWlyIHVzZWZ1bG5lc3MgZm9yIGdlbmV0aWNzIGFuZCBicmVlZGluZyBleHBlcmltZW50czsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTE7ICBjaXRhdGlvbl9wYWdlcz0yOTEtMzEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpemF0aW9uIG9mIHRvbWF0byBTU1IgbWFya2VycyBkZXZlbG9wZWQgdXNpbmcgQkFDLWVuZCBhbmQgY0ROQSBzZXF1ZW5jZXMgZnJvbSBnZW5vbWUgZGF0YWJhc2VzOyBNb2wuIEJyZWVkLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTY4NS02OTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNyb3NzLXNwZWNpZXMgYmFjdGVyaWFsIGFydGlmaWNpYWwgY2hyb21vc29tZS1mbHVvcmVzY2VuY2UgaW4gc2l0dSBoeWJyaWRpemF0aW9uIHBhaW50aW5nIG9mIHRoZSB0b21hdG8gYW5kIHBvdGF0byBjaHJvbW9zb21lIDYgcmV2ZWFscyB1bmRlc2NyaWJlZCBjaHJvbW9zb21hbCByZWFycmFuZ2VtZW50czsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTgwOyAgY2l0YXRpb25fcGFnZXM9MTMxOS0xMzI4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdWNocm9tYXRpbiBhbmQgcGVyaWNlbnRyb21lcmljIGhldGVyb2Nocm9tYXRpbjogY29tcGFyYXRpdmUgY29tcG9zaXRpb24gaW4gdGhlIHRvbWF0byBnZW5vbWU7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE3MjsgIGNpdGF0aW9uX3BhZ2VzPTI1MjktMjU0MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXJyYXktYmFzZWQgaGlnaC10aHJvdWdocHV0IEROQSBtYXJrZXJzIGZvciBjcm9wIGltcHJvdmVtZW50OyBIZXJlZGl0eTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xMDE7ICBjaXRhdGlvbl9wYWdlcz01LTE4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jZSBkaXZlcnNpdHkgaW4gdGhyZWUgdG9tYXRvIHNwZWNpZXM6IFNOUHMsIG1hcmtlcnMsIGFuZCBtb2xlY3VsYXIgZXZvbHV0aW9uOyBCTUMgUGxhbnQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTg1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VG9tYXRvIFNOUCBkaXNjb3ZlcnkgYnkgRVNUIG1pbmluZyBhbmQgcmVzZXF1ZW5jaW5nOyBNb2wuIEJyZWVkLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTM0My0zNDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc2NvdmVyeSBvZiBzaW5nbGUgbnVjbGVvdGlkZSBwb2x5bW9ycGhpc21zIGluIEx5Y29wZXJzaWNvbiBlc2N1bGVudHVtIGJ5IGNvbXB1dGVyIGFpZGVkIGFuYWx5c2lzIG9mIGV4cHJlc3NlZCBzZXF1ZW5jZSB0YWdzOyBNb2wuIEJyZWVkLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTIxLTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHByZXNzZWQgc2VxdWVuY2UgdGFncyBmcm9tIHRoZSBsYWJvcmF0b3J5LWdyb3duIG1pbmlhdHVyZSB0b21hdG8gKEx5Y29wZXJzaWNvbiBlc2N1bGVudHVtKSBjdWx0aXZhciBNaWNyby1Ub20gYW5kIG1pbmluZyBmb3Igc2luZ2xlIG51Y2xlb3RpZGUgcG9seW1vcnBoaXNtcyBhbmQgaW5zZXJ0aW9ucy9kZWxldGlvbnMgaW4gdG9tYXRvIGN1bHRpdmFyczsgR2VuZTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTEyNy0xMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRvbWF0byBpcyBhIG5leHQtZ2VuZXJhdGlvbiBtb2RlbCBwbGFudCBmb3IgcmVzZWFyY2ggYW5kIGRldmVsb3BtZW50OyBKLiBKcG4gU29jLiBIb3J0LiBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTc4OyAgY2l0YXRpb25fcGFnZXM9MS0yOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyby1Ub23igJRhIG1pbmlhdHVyZSBkd2FyZiB0b21hdG87IEZsb3JpZGEgQWdyLiBFeHB0LiBTdGEuIENpcmMuOyAgY2l0YXRpb25feWVhcj0xOTg5OyAgY2l0YXRpb25fdm9sdW1lPTM3MDsgIGNpdGF0aW9uX3BhZ2VzPTEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBuZXcgbW9kZWwgc3lzdGVtIGZvciB0b21hdG8gZ2VuZXRpY3M7IFBsYW50IEouOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTQ2NS0xNDcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lcmF0aW9uIG9mIGdhbW1hIGlycmFkaWF0aW9uLWluZHVjZWQgbXV0YW50IGxpbmVzIG9mIHRoZSBtaW5pYXR1cmUgdG9tYXRvIChTb2xhbnVtIGx5Y29wZXJzaWN1bSBMLikgY3VsdGl2YXIg4oCYTWljcm8tVG9tOyBQbGFudCBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTM5LTQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdGh5bG1ldGhhbmVzdWxmb25hdGUgKEVNUykgbXV0YWdlbmVzaXMgb2YgU29sYW51bSBseWNvcGVyc2ljdW0gY3YuIE1pY3JvLVRvbSBmb3IgbGFyZ2Utc2NhbGUgbXV0YW50IHNjcmVlbnM7IFBsYW50IEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MzMtMzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3JvVG9t4oCUYSBoaWdoLXRocm91Z2hwdXQgbW9kZWwgdHJhbnNmb3JtYXRpb24gc3lzdGVtIGZvciBmdW5jdGlvbmFsIGdlbm9taWNzOyBQbGFudCBDZWxsIFJlcC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz00MzItNDQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGhpZ2hseSBlZmZpY2llbnQgdHJhbnNmb3JtYXRpb24gcHJvdG9jb2wgZm9yIE1pY3JvLVRvbSwgYSBtb2RlbCBjdWx0aXZhciBmb3IgdG9tYXRvIGZ1bmN0aW9uYWwgZ2Vub21pY3M7IFBsYW50IENlbGwgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NDc7ICBjaXRhdGlvbl9wYWdlcz00MjYtNDMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZXRhYm9saXRlIGFubm90YXRpb25zIGJhc2VkIG9uIHRoZSBpbnRlZ3JhdGlvbiBvZiBtYXNzIHNwZWN0cmFsIGluZm9ybWF0aW9uOyBQbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT01NDsgIGNpdGF0aW9uX3BhZ2VzPTk0OS05NjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxhcmdlLXNjYWxlIGFuYWx5c2lzIG9mIGZ1bGwtbGVuZ3RoIGNETkFzIGZyb20gdGhlIHRvbWF0byAoU29sYW51bSBseWNvcGVyc2ljdW0pIGN1bHRpdmFyIE1pY3JvLVRvbSwgYSByZWZlcmVuY2Ugc3lzdGVtIGZvciB0aGUgU29sYW5hY2VhZSBnZW5vbWljczsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MjEwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VXNpbmcgdGhlIG1pcmFFU1QgYXNzZW1ibGVyIGZvciByZWxpYWJsZSBhbmQgYXV0b21hdGVkIG1STkEgdHJhbnNjcmlwdCBhc3NlbWJseSBhbmQgU05QIGRldGVjdGlvbiBpbiBzZXF1ZW5jZWQgRVNUczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0xMTQ3LTExNTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByaW1lcjMgb24gdGhlIFdXVyBmb3IgZ2VuZXJhbCB1c2VycyBhbmQgZm9yIGJpb2xvZ2lzdCBwcm9ncmFtbWVyczsgTWV0aG9kcyBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTEzMjsgIGNpdGF0aW9uX3BhZ2VzPTM2NS0zODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXByZWhlbnNpdmUgc3RydWN0dXJhbCBhbmFseXNpcyBvZiB0aGUgZ2Vub21lIG9mIHJlZCBjbG92ZXIgKFRyaWZvbGl1bSBwcmF0ZW5zZSBMLik7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MzAxLTM2NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUt5YXptYS4gQlYsIFdhZ2VuaW5nZW4sIE5ldGhlcmxhbmRzOyBKb2luTWFwwq40LCBTb2Z0d2FyZSBmb3IgdGhlIENhbGN1bGF0aW9uIG9mIEdlbmV0aWMgTGlua2FnZSBNYXBzIGluIEV4cGVyaW1lbnRhbCBQb3B1bGF0aW9uczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBhbmQgcGh5c2lvbG9naWNhbCBjaGFyYWN0ZXJpemF0aW9uIG9mIHRvbWF0byBjdi4gTWljcm8tVG9tOyBKLiBFeHAuIEJvdC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NTc7ICBjaXRhdGlvbl9wYWdlcz0yMDM3LTIwNDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2gtdGhyb3VnaHB1dCBnZW5vdHlwaW5nIHdpdGggdGhlIEdvbGRlbkdhdGUgYXNzYXkgaW4gdGhlIGNvbXBsZXggZ2Vub21lIG9mIHNveWJlYW47IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTE2OyAgY2l0YXRpb25fcGFnZXM9OTQ1LTk1MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb25zZW5zdXMgZ2VuZXRpYyBtYXAgb2YgY293cGVhIFtWaWduYSB1bmd1aWN1bGF0YSAoTCkgV2FscC5dIGFuZCBzeW50ZW55IGJhc2VkIG9uIEVTVC1kZXJpdmVkIFNOUHM7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA2OyAgY2l0YXRpb25fcGFnZXM9MTgxNTktMTgxNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2gtdGhyb3VnaHB1dCBTTlAgZGlzY292ZXJ5IGFuZCBhc3NheSBkZXZlbG9wbWVudCBpbiBjb21tb24gYmVhbjsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NDc1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaGx5LW11bHRpcGxleGVkIFNOUCBnZW5vdHlwaW5nIGZvciBnZW5ldGljIG1hcHBpbmcgYW5kIGdlcm1wbGFzbSBkaXZlcnNpdHkgc3R1ZGllcyBpbiBwZWE7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTQ2OCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgYW5kIGJpb3N5c3RlbWF0aWMgc3R1ZGllcyBvbiB0d28gbmV3IHNpYmxpbmcgc3BlY2llcyBvZiBMeWNvcGVyc2ljb24gZnJvbSBpbnRlcmFuZGVhbiBQZXJ1OyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTc2OyAgY2l0YXRpb25fdm9sdW1lPTQ3OyAgY2l0YXRpb25fcGFnZXM9NTUtNjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNhdGFsb2cgb2YgTWljcm8tVG9tIHRvbWF0byByZXNwb25zZXMgdG8gY29tbW9uIGZ1bmdhbCwgYmFjdGVyaWFsLCBhbmQgdmlyYWwgcGF0aG9nZW5zOyBKLiBHZW4uIFBsYW50IFBhdGhvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NzE7ICBjaXRhdGlvbl9wYWdlcz04LTIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FU1QsIENPU0lJLCBhbmQgYXJiaXRyYXJ5IGdlbmUgbWFya2VycyBnaXZlIHNpbWlsYXIgZXN0aW1hdGVzIG9mIG51Y2xlb3RpZGUgZGl2ZXJzaXR5IGluIGN1bHRpdmF0ZWQgdG9tYXRvIChTb2xhbnVtIGx5Y29wZXJzaWN1bSBMLik7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTE4OyAgY2l0YXRpb25fcGFnZXM9MTAwNS0xMDE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXZlcnNpdHkgaW4gY29uc2VydmVkIGdlbmVzIGluIHRvbWF0bzsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz00NjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZHZhbmNlcyBpbiBjZXJlYWwgZ2Vub21pY3MgYW5kIGFwcGxpY2F0aW9ucyBpbiBjcm9wIGJyZWVkaW5nOyBUcmVuZHMuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9NDkwLTQ5OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzbmFwc2hvdCBvZiB0aGUgZW1lcmdpbmcgdG9tYXRvIGdlbm9tZSBzZXF1ZW5jZTsgUGxhbnQgR2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz03OC05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE3LzYvMzgxLzEzMDk5MjgvZHNxMDI0LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBGZXcgaW50cmFzcGVjaWZpYyBnZW5ldGljIGxpbmthZ2UgbWFwcyBoYXZlIGJlZW4gcmVwb3J0ZWQgZm9yIGN1bHRpdmF0ZWQgdG9tYXRvLCBtYWlubHkgYmVjYXVzZSBnZW5ldGljIGRpdmVyc2l0eSB3aXRoaW4gU29sYW51bSBseWNvcGVyc2ljdW0gaXMgbXVjaCIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDg0NTU1JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZFJUP6mHmf2fLvXo0sqASb1t0w4T" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f6%2f381%2f484555" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f6%2f381%2f484555" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=19064" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/17/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/6/1/m_cover.gif?Expires=2147483647&amp;Signature=GE4QDx7vPXS7wjUaoLIKdeQMVYvKpNXob0I6QlcCM7oe~eKCfP8E380kotmiw9z~axHxmBJESzx9vV88~C9g-CA5dZPWzHX3y0HSadJ2Jrf2k3i9guvDwVaIIvgF6cZI1iLndCWPhjSM1C0sC8Omes66kjDc9sn4S8VlQHryn-wE9mPjK2sZ-quW-rVkH966kcOElJ2hx~w7cpINeghmflNy5cmWJN-jp6Sp3iFBqCubFpEA3485f5KqRLMCbhc5nMYcpZWmmznkyRdy0CL0lxP-Xr1JDHnrIseUrn4fTYBDpMQY0Dv8f9-vnTHQwjT0Mz0eJIlyt6vd81Yiu4619w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/17/6">
                        <div class="volume trailing-comma">Volume 17</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2010        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="7361950">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7361950">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7361954">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7361954">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7361965">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7361965">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7361996">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7361996">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7362016">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7362016">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7362025">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7362025">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7362027">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7362027">Availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7362029">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7362029">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7362031">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7362031">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7362033">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7362033">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/17/6/369/484698">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/17/6/393/484773">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">SNP Discovery and Linkage Map Construction in Cultivated Tomato
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kenta Shirasawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 438-52-3935. Fax. <div class="fax">+81 438-52-3934</div>. Email: <a href="mailto:shirasaw@kazusa.or.jp" target="_blank">shirasaw@kazusa.or.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kenta+Shirasawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shirasawa K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shirasawa K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kenta Shirasawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sachiko Isobe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sachiko+Isobe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Isobe S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Isobe S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Sachiko Isobe</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideki Hirakawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideki+Hirakawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hirakawa H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hirakawa H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hideki Hirakawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Erika Asamizu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Erika+Asamizu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Asamizu E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Asamizu E&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Erika Asamizu</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroyuki Fukuoka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">National Institute of Vegetable and Tea Science (NIVTS)</div>, <div class="insititution">National Agriculture and Food Research Organization</div>, <div class="addr-line">360 Kusawa, Ano, Tsu, Mie 514-2392</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroyuki+Fukuoka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fukuoka H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fukuoka H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hiroyuki Fukuoka</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Daniel Just</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Unité Mixte de Recherche 619, Biologie du Fruit, Institut National de la Recherche Agronomique (INRA Bordeaux), Universités de Bordeaux</div>, <div class="addr-line">33883 Villenave d'Ornon cedex</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Daniel+Just">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Just D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Just D&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Daniel Just</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Christophe Rothan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Unité Mixte de Recherche 619, Biologie du Fruit, Institut National de la Recherche Agronomique (INRA Bordeaux), Universités de Bordeaux</div>, <div class="addr-line">33883 Villenave d'Ornon cedex</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Christophe+Rothan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Rothan C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Rothan C&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Christophe Rothan</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shigemi Sasamoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shigemi+Sasamoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sasamoto S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sasamoto S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Shigemi Sasamoto</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tsunakazu Fujishiro</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tsunakazu+Fujishiro">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fujishiro T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fujishiro T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Tsunakazu Fujishiro</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshie Kishida</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshie+Kishida">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kishida Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kishida Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yoshie Kishida</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mitsuyo Kohara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mitsuyo+Kohara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kohara M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kohara M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Mitsuyo Kohara</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hisano Tsuruoka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hisano+Tsuruoka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tsuruoka H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tsuruoka H&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Hisano Tsuruoka</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tsuyuko Wada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tsuyuko+Wada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wada T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wada T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Tsuyuko Wada</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yasukazu Nakamura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yasukazu+Nakamura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakamura Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakamura Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yasukazu Nakamura</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shusei Sato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shusei+Sato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sato S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sato S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Shusei Sato</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoshi Tabata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoshi+Tabata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tabata S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tabata S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Satoshi Tabata</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">Present address: Gene Research Center, University of Tsukuba, 1-1-1 Tennodai, Tsukuba, Ibaraki 305-8572, Japan.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN2"><div class="fn"><div class="label -label">‡</div><div class="footnote-content">Present address: Center for Information Biology and DNA Data Bank of Japan, National Institute of Genetics, Research Organization for Information and Systems, Yata, Mishima, Shizuoka 411-8510, Japan.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Doil Choi</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 17, Issue 6, 1 December 2010, Pages 381–391, <a href='https://doi.org/10.1093/dnares/dsq024'>https://doi.org/10.1093/dnares/dsq024</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">02 November 2010</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">14 July 2010</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">21 September 2010</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="484555" href="/dnaresearch/article-pdf/17/6/381/1309928/dsq024.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Kenta Shirasawa, Sachiko Isobe, Hideki Hirakawa, Erika Asamizu, Hiroyuki Fukuoka, Daniel Just, Christophe Rothan, Shigemi Sasamoto, Tsunakazu Fujishiro, Yoshie Kishida, Mitsuyo Kohara, Hisano Tsuruoka, Tsuyuko Wada, Yasukazu Nakamura, Shusei Sato, Satoshi Tabata;  SNP Discovery and Linkage Map Construction in Cultivated Tomato, <em>DNA Research</em>, Volume 17, Issue 6, 1 December 2010, Pages 381–391, <a href='https://doi.org/10.1093/dnares/dsq024'>https://doi.org/10.1093/dnares/dsq024</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/484555?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/484555?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/484555?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/484555?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/484555?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/484555?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/484555?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=SNP%20Discovery%20and%20Linkage%20Map%20Construction%20in%20Cultivated%20Tomato&amp;publicationDate=2010-11-02&amp;volumeNum=17&amp;issueNum=6&amp;author=Shirasawa%2C%20Kenta%3B%20Isobe%2C%20Sachiko&amp;startPage=381&amp;endPage=391&amp;contentId=10.1093%2Fdnares%2Fdsq024&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f6%2f381%2f484555" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=7361950 id="7361950" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Few intraspecific genetic linkage maps have been reported for cultivated tomato, mainly because genetic diversity within <em>Solanum lycopersicum</em> is much less than that between tomato species. Single nucleotide polymorphisms (SNPs), the most abundant source of genomic variation, are the most promising source of polymorphisms for the construction of linkage maps for closely related intraspecific lines. In this study, we developed SNP markers based on expressed sequence tags for the construction of intraspecific linkage maps in tomato. Out of the 5607 SNP positions detected through <em>in silico</em> analysis, 1536 were selected for high-throughput genotyping of two mapping populations derived from crosses between ‘Micro-Tom’ and either ‘Ailsa Craig’ or ‘M82’. A total of 1137 markers, including 793 out of the 1338 successfully genotyped SNPs, along with 344 simple sequence repeat and intronic polymorphism markers, were mapped onto two linkage maps, which covered 1467.8 and 1422.7 cM, respectively. The SNP markers developed were then screened against cultivated tomato lines in order to estimate the transferability of these SNPs to other breeding materials. The molecular markers and linkage maps represent a milestone in the genomics and genetics, and are the first step toward molecular breeding of cultivated tomato. Information on the DNA markers, linkage maps, and SNP genotypes for these tomato lines is available at <a class="link link-uri" href="http://www.kazusa.or.jp/tomato/">http://www.kazusa.or.jp/tomato/</a>.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;DNA marker&quot;">DNA marker</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;linkage map&quot;">linkage map</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;single nucleotide polymorphism&quot;">single nucleotide polymorphism</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Solanum lycopersicum&quot;"><em>Solanum lycopersicum</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="tomato">tomato</a></div>                            <h2 scrollto-destination=7361954 id="7361954" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Genetics in tomato (<em>Solanum lycopersicum</em>) and its wild relatives, including <em>S. chilense</em>, <em>S. habrochaites</em>, <em>S. pimpinellifolium</em>, and <em>S. pennellii</em>, have been greatly advanced since molecular markers have become available.<sup><a reveal-id="DSQ024C1" data-open="DSQ024C1" class="link link-ref link-reveal xref-bibr">1</a></sup> During the past two decades, several genetic maps in tomato have been reported, with a total of more than 2000 loci detected by restriction fragment length polymorphism (RFLP), amplified fragment length polymorphism (AFLP), cleaved amplified polymorphic sequence (CAPS), and simple sequence repeat (SSR) markers based on the mapping of populations derived from crosses between tomato and related wild species.<sup><a reveal-id="DSQ024C2  DSQ024C6" data-open="DSQ024C2  DSQ024C6" class="link link-ref link-reveal xref-bibr">2–6</a></sup> Recently, 1282 novel SSR markers and 151 intronic polymorphic markers were mapped onto an interspecific map, ‘Tomato-EXPEN 2000’ derived from a cross between <em>S. lycopersicum</em> and <em>S. pennellii</em>.<sup><a reveal-id="DSQ024C7" data-open="DSQ024C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Such efforts have resulted in the identification of a number of quantitative trait loci (QTLs) and genes for fruit morphology,<sup><a reveal-id="DSQ024C8  DSQ024C11" data-open="DSQ024C8  DSQ024C11" class="link link-ref link-reveal xref-bibr">8–11</a></sup> disease resistance,<sup><a reveal-id="DSQ024C12  DSQ024C15" data-open="DSQ024C12  DSQ024C15" class="link link-ref link-reveal xref-bibr">12–15</a></sup> and other agronomical traits.<sup><a reveal-id="DSQ024C16" data-open="DSQ024C16" class="link link-ref link-reveal xref-bibr">16</a></sup> The identified genes, e.g. <em>Cf-4</em>, <em>Tm-2</em>, and <em>Sw-5</em>, have already been used for tomato breeding through advanced-backcross and introgression-line strategies using molecular markers.<sup><a reveal-id="DSQ024C1" data-open="DSQ024C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Though significant advances in molecular genetics and breeding have been reported in tomato, most of them were based on interspecific crosses because genetic diversity in the cultivated tomato is lower than in its wild relatives.<sup><a reveal-id="DSQ024C17" data-open="DSQ024C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Meanwhile, intraspecific maps are required to identify QTLs for agronomically important traits, which are the targets of practical breeding programs. However, only one intraspecific map, based on AFLP, RFLP, and random amplified polymorphic DNA (RAPD) markers, has been reported for <em>S. lycopersicum</em>.<sup><a reveal-id="DSQ024C18" data-open="DSQ024C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p><p>Single nucleotide polymorphisms (SNPs) are the most abundant source of variation in the genome for both intragenic and intergenic regions. They therefore represent a valuable basis for the development of molecular markers for identification of polymorphisms among closely related lines. Previous studies have suggested that DNA markers developed from intergenic regions tend to cluster in heterochromatic portions of chromosomes, while those derived from genic regions disperse along entire chromosomes.<sup><a reveal-id="DSQ024C7" data-open="DSQ024C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSQ024C19  DSQ024C22" data-open="DSQ024C19  DSQ024C22" class="link link-ref link-reveal xref-bibr">19–22</a></sup> Therefore, SNPs, especially those located in intragenic regions, are expected to distribute randomly along the whole genome. In addition, novel techniques based on the DNA microarray method allow high-throughput SNP genotyping.<sup><a reveal-id="DSQ024C23" data-open="DSQ024C23" class="link link-ref link-reveal xref-bibr">23</a></sup> For these reasons, SNP markers derived from intragenic regions are the most informative markers for genome-wide genetic analysis in intraspecific tomato populations. By comparing expressed sequence tags (ESTs) in tomato and related wild species, approximately 40 000 candidate SNPs have been identified.<sup><a reveal-id="DSQ024C24  DSQ024C27" data-open="DSQ024C24  DSQ024C27" class="link link-ref link-reveal xref-bibr">24–27</a></sup> Since then, the number of ESTs derived from several tomato cultivars has increased to approximately 300 000, all of which are available in the public DNA databases, e.g. DNA Data Bank of Japan (DDBJ: <a class="link link-uri" href="http://www.ddbj.nig.ac.jp/">http://www.ddbj.nig.ac.jp/</a>), Sol Genomics Network (SGN: <a class="link link-uri" href="http://solgenomics.net/">http://solgenomics.net/</a>), and MiBASE (<a class="link link-uri" href="http://www.pgb.kazusa.or.jp/mibase/">http://www.pgb.kazusa.or.jp/mibase/</a>).</p><p>The tomato is regarded as a model plant not only for the <em>Solanaceae</em> but also for other fruiting plants.<sup><a reveal-id="DSQ024C28" data-open="DSQ024C28" class="link link-ref link-reveal xref-bibr">28</a></sup> A miniature dwarf cultivar, ‘Micro-Tom’, originally bred for home gardening purposes,<sup><a reveal-id="DSQ024C29" data-open="DSQ024C29" class="link link-ref link-reveal xref-bibr">29</a></sup> has drawn attention as a model tomato line because of its small plant size, short life cycle, easy transformation, and availability of transposon-tagging systems for use in reverse genetics.<sup><a reveal-id="DSQ024C30" data-open="DSQ024C30" class="link link-ref link-reveal xref-bibr">30</a></sup> Various genomic and genetic resources have been developed for ‘Micro-Tom’. These include mutagenized lines,<sup><a reveal-id="DSQ024C31" data-open="DSQ024C31" class="link link-ref link-reveal xref-bibr">31</a>,<a reveal-id="DSQ024C32" data-open="DSQ024C32" class="link link-ref link-reveal xref-bibr">32</a></sup> effective transformation systems,<sup><a reveal-id="DSQ024C33" data-open="DSQ024C33" class="link link-ref link-reveal xref-bibr">33</a>,<a reveal-id="DSQ024C34" data-open="DSQ024C34" class="link link-ref link-reveal xref-bibr">34</a></sup> metabolite annotations,<sup><a reveal-id="DSQ024C35" data-open="DSQ024C35" class="link link-ref link-reveal xref-bibr">35</a></sup> full-length cDNAs,<sup><a reveal-id="DSQ024C36" data-open="DSQ024C36" class="link link-ref link-reveal xref-bibr">36</a></sup> and BAC-end sequences (Asamizu <em>et al.</em>, released in the public DNA database with accession numbers: FT227487–FT321168). ‘Micro-Tom’ seeds are available through two seed stock centers: the Tomato Genetics Resource Center at the University of California, Davis (USA, accession no. LA3911) and the National Bio-Resource Project at the University of Tsukuba (Japan, accession no. TOMJPF00001).</p><p>In this study, we developed SNP markers using publicly available ESTs from several tomato cultivars and designed an SNP-genotyping platform using the GoldenGate<sup>®</sup> assay (Illumina, San Diego, CA, USA) in order to accelerate genetic studies and molecular breeding in tomato. SNP markers, along with SSR markers and intronic polymorphic markers, which were developed and mapped onto the interspecific map Tomato-EXPEN 2000 by Shirasawa <em>et al.</em>,<sup><a reveal-id="DSQ024C7" data-open="DSQ024C7" class="link link-ref link-reveal xref-bibr">7</a></sup> were applied to create linkage maps using two mapping populations derived from crosses between ‘Micro-Tom’ and ‘Ailsa Craig’, a greenhouse-type tomato, and between ‘Micro-Tom’ and ‘M82’, a processing tomato. In addition, the polymorphism of the SNP markers was investigated in cultivated tomato lines in order to estimate the transferability of the SNPs to breeding materials.</p>                            <h2 scrollto-destination=7361965 id="7361965" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=7361968 id="7361968" class="section-title" data-legacy-id=s2a>2.1. Plant materials</h3>
<p>Two F<sub>2</sub> mapping populations, AMF2 and MMF2, each derived by crossing two <em>S. lycopersicum</em> lines, were used for the construction of the linkage maps. AMF2 (<em>n</em> = 120) was derived from a cross between the ‘Ailsa Craig’ and ‘Micro-Tom’ lines, while MMF2 (<em>n</em> = 135) was derived from a cross between the ‘M82’ and ‘Micro-Tom’ lines. AMF2 and MMF2 were generated in the National Institute of Vegetable and Tea Science, Japan, and in the Institut National de la Recherche Agronomique, France, respectively (Table <a reveal-id="DSQ024TB1" data-open="DSQ024TB1" class="link link-reveal link-table xref-fig">1</a>). To address potential residual heterozygosity in the parental ‘Micro-Tom’ lines used to create AMF2 and MMF2, they are distinguished in this study by the designations ‘Micro-Tom_AM’ and ‘Micro-Tom_MM’, respectively. Along with the four parental lines of the mapping populations, 22 lines, including 16 inbred and 6 hybrid tomato lines, and an <em>S. pennellii</em> line (‘LA716’) were used for polymorphic analysis of SNPs (Table <a reveal-id="DSQ024TB1" data-open="DSQ024TB1" class="link link-reveal link-table xref-fig">1</a>). Total DNA for each line was extracted using the DNeasy Plant Mini kit (Qiagen, Hilden, Germany). </p>                        <a id="7361973" scrollto-destination="7361973"></a>
<div content-id="DSQ024TB1" class="table-modal"><div class="table-wrap"><div id="DSQ024TB1" class="table-wrap-title" data-id="DSQ024TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Description of plant materials</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Line name </th><th>Note </th><th>Source<sup>a</sup> </th><th>Accession number </th><th>SNP validation<sup>b</sup> </th></tr></thead><tbody><tr><td colspan="5">Parental lines of mapping populations </td></tr><tr><td> Micro-Tom_AM </td><td>Inbred line </td><td>NIVTS </td><td> </td><td>Tested </td></tr><tr><td> Ailsa Craig </td><td>Inbred line </td><td>NBRP </td><td>TOMJPF00004 </td><td>Tested </td></tr><tr><td> Micro-Tom_MM </td><td>Inbred line </td><td>INRA </td><td> </td><td> </td></tr><tr><td> M82 </td><td>Inbred line </td><td>INRA </td><td> </td><td>Tested </td></tr><tr><td colspan="5">Tomato lines for SNP typing </td></tr><tr><td> Aichi First </td><td>Inbred line </td><td>NBRP </td><td>TOMJPF00003 </td><td> </td></tr><tr><td> Best of All </td><td>Inbred line </td><td>NIVTS </td><td>LS3908 </td><td> </td></tr><tr><td> Earliana </td><td>Inbred line </td><td>TGRC </td><td>LA3238 </td><td>Tested </td></tr><tr><td> Fruit </td><td>Inbred line </td><td>NIVTS </td><td>LS1100 </td><td> </td></tr><tr><td> Furikoma </td><td>Inbred line </td><td>NIVTS </td><td>LS3903 </td><td> </td></tr><tr><td> Heinz 1706-BG </td><td>Inbred line </td><td>NIVTS </td><td>LS461 </td><td>Tested </td></tr><tr><td> LA925 </td><td>Inbred line </td><td>Cornell University </td><td> </td><td>Tested </td></tr><tr><td> Marglobe </td><td>Inbred line </td><td>TGRC </td><td>LA0502 </td><td>Tested </td></tr><tr><td> Money Maker </td><td>Inbred line </td><td>TGRC </td><td>LA2706 </td><td>Tested </td></tr><tr><td> Ponderosa </td><td>Inbred line </td><td>NIVTS </td><td>LS1728 </td><td> </td></tr><tr><td> Rio Grande </td><td>Inbred line </td><td>TGRC </td><td>LA3343 </td><td>Tested </td></tr><tr><td> Rutgers </td><td>Inbred line </td><td>TGRC </td><td>LA1090 </td><td>Tested </td></tr><tr><td> San Marzano </td><td>Inbred line </td><td>NIVTS </td><td>LS4956 </td><td>Tested </td></tr><tr><td> Tomato Chuukanbohon Nou 9 </td><td>Inbred line </td><td>NIVTS </td><td> </td><td> </td></tr><tr><td> Tomato Chuukanbohon Nou 11 </td><td>Inbred line </td><td>NIVTS </td><td> </td><td> </td></tr><tr><td> Geronimo </td><td>F<sub>1</sub> hybrid </td><td>De Ruiter Seeds Co. </td><td> </td><td>Tested </td></tr><tr><td> Labell </td><td>F<sub>1</sub> hybrid </td><td>De Ruiter Seeds Co. </td><td> </td><td>Tested </td></tr><tr><td> Matrix </td><td>F<sub>1</sub> hybrid </td><td>De Ruiter Seeds Co. </td><td> </td><td>Tested </td></tr><tr><td> Momotaro 8 </td><td>F<sub>1</sub> hybrid </td><td>Takii Seeds Co. </td><td> </td><td>Tested </td></tr><tr><td> Reika </td><td>F<sub>1</sub> hybrid </td><td>Sakata Seeds Co. </td><td> </td><td>Tested </td></tr><tr><td> Regina </td><td>Inbred line, cherry type </td><td>Sakata Seeds Co. </td><td> </td><td> </td></tr><tr><td> Sweet100 </td><td>F<sub>1</sub> hybrid, cherry type </td><td>Vilmorin Seeds Co. </td><td> </td><td>Tested </td></tr><tr><td> LA716 </td><td>Inbred line, <em>S. pennellii</em> </td><td>Cornell University </td><td> </td><td> </td></tr></tbody></table></div><div content-id="DSQ024TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Line name </th><th>Note </th><th>Source<sup>a</sup> </th><th>Accession number </th><th>SNP validation<sup>b</sup> </th></tr></thead><tbody><tr><td colspan="5">Parental lines of mapping populations </td></tr><tr><td> Micro-Tom_AM </td><td>Inbred line </td><td>NIVTS </td><td> </td><td>Tested </td></tr><tr><td> Ailsa Craig </td><td>Inbred line </td><td>NBRP </td><td>TOMJPF00004 </td><td>Tested </td></tr><tr><td> Micro-Tom_MM </td><td>Inbred line </td><td>INRA </td><td> </td><td> </td></tr><tr><td> M82 </td><td>Inbred line </td><td>INRA </td><td> </td><td>Tested </td></tr><tr><td colspan="5">Tomato lines for SNP typing </td></tr><tr><td> Aichi First </td><td>Inbred line </td><td>NBRP </td><td>TOMJPF00003 </td><td> </td></tr><tr><td> Best of All </td><td>Inbred line </td><td>NIVTS </td><td>LS3908 </td><td> </td></tr><tr><td> Earliana </td><td>Inbred line </td><td>TGRC </td><td>LA3238 </td><td>Tested </td></tr><tr><td> Fruit </td><td>Inbred line </td><td>NIVTS </td><td>LS1100 </td><td> </td></tr><tr><td> Furikoma </td><td>Inbred line </td><td>NIVTS </td><td>LS3903 </td><td> </td></tr><tr><td> Heinz 1706-BG </td><td>Inbred line </td><td>NIVTS </td><td>LS461 </td><td>Tested </td></tr><tr><td> LA925 </td><td>Inbred line </td><td>Cornell University </td><td> </td><td>Tested </td></tr><tr><td> Marglobe </td><td>Inbred line </td><td>TGRC </td><td>LA0502 </td><td>Tested </td></tr><tr><td> Money Maker </td><td>Inbred line </td><td>TGRC </td><td>LA2706 </td><td>Tested </td></tr><tr><td> Ponderosa </td><td>Inbred line </td><td>NIVTS </td><td>LS1728 </td><td> </td></tr><tr><td> Rio Grande </td><td>Inbred line </td><td>TGRC </td><td>LA3343 </td><td>Tested </td></tr><tr><td> Rutgers </td><td>Inbred line </td><td>TGRC </td><td>LA1090 </td><td>Tested </td></tr><tr><td> San Marzano </td><td>Inbred line </td><td>NIVTS </td><td>LS4956 </td><td>Tested </td></tr><tr><td> Tomato Chuukanbohon Nou 9 </td><td>Inbred line </td><td>NIVTS </td><td> </td><td> </td></tr><tr><td> Tomato Chuukanbohon Nou 11 </td><td>Inbred line </td><td>NIVTS </td><td> </td><td> </td></tr><tr><td> Geronimo </td><td>F<sub>1</sub> hybrid </td><td>De Ruiter Seeds Co. </td><td> </td><td>Tested </td></tr><tr><td> Labell </td><td>F<sub>1</sub> hybrid </td><td>De Ruiter Seeds Co. </td><td> </td><td>Tested </td></tr><tr><td> Matrix </td><td>F<sub>1</sub> hybrid </td><td>De Ruiter Seeds Co. </td><td> </td><td>Tested </td></tr><tr><td> Momotaro 8 </td><td>F<sub>1</sub> hybrid </td><td>Takii Seeds Co. </td><td> </td><td>Tested </td></tr><tr><td> Reika </td><td>F<sub>1</sub> hybrid </td><td>Sakata Seeds Co. </td><td> </td><td>Tested </td></tr><tr><td> Regina </td><td>Inbred line, cherry type </td><td>Sakata Seeds Co. </td><td> </td><td> </td></tr><tr><td> Sweet100 </td><td>F<sub>1</sub> hybrid, cherry type </td><td>Vilmorin Seeds Co. </td><td> </td><td>Tested </td></tr><tr><td> LA716 </td><td>Inbred line, <em>S. pennellii</em> </td><td>Cornell University </td><td> </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>NBRP: University of Tsukuba in National Bio-Resource Project of MEXT, Japan; INRA: National Institute for Agricultural Research, France; NIVTS: National Institute of Vegetable and Tea Science, Japan; TGRC: Tomato Genetics Resource Center, University of California, Davis, USA.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Lines that used for validation of 82 eSNPs prior to design a SNP genotyping platform using Illumina GoldenGate<sup>®</sup> assay.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7361973" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ024TB1" class="table-wrap-title" data-id="DSQ024TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Description of plant materials</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Line name </th><th>Note </th><th>Source<sup>a</sup> </th><th>Accession number </th><th>SNP validation<sup>b</sup> </th></tr></thead><tbody><tr><td colspan="5">Parental lines of mapping populations </td></tr><tr><td> Micro-Tom_AM </td><td>Inbred line </td><td>NIVTS </td><td> </td><td>Tested </td></tr><tr><td> Ailsa Craig </td><td>Inbred line </td><td>NBRP </td><td>TOMJPF00004 </td><td>Tested </td></tr><tr><td> Micro-Tom_MM </td><td>Inbred line </td><td>INRA </td><td> </td><td> </td></tr><tr><td> M82 </td><td>Inbred line </td><td>INRA </td><td> </td><td>Tested </td></tr><tr><td colspan="5">Tomato lines for SNP typing </td></tr><tr><td> Aichi First </td><td>Inbred line </td><td>NBRP </td><td>TOMJPF00003 </td><td> </td></tr><tr><td> Best of All </td><td>Inbred line </td><td>NIVTS </td><td>LS3908 </td><td> </td></tr><tr><td> Earliana </td><td>Inbred line </td><td>TGRC </td><td>LA3238 </td><td>Tested </td></tr><tr><td> Fruit </td><td>Inbred line </td><td>NIVTS </td><td>LS1100 </td><td> </td></tr><tr><td> Furikoma </td><td>Inbred line </td><td>NIVTS </td><td>LS3903 </td><td> </td></tr><tr><td> Heinz 1706-BG </td><td>Inbred line </td><td>NIVTS </td><td>LS461 </td><td>Tested </td></tr><tr><td> LA925 </td><td>Inbred line </td><td>Cornell University </td><td> </td><td>Tested </td></tr><tr><td> Marglobe </td><td>Inbred line </td><td>TGRC </td><td>LA0502 </td><td>Tested </td></tr><tr><td> Money Maker </td><td>Inbred line </td><td>TGRC </td><td>LA2706 </td><td>Tested </td></tr><tr><td> Ponderosa </td><td>Inbred line </td><td>NIVTS </td><td>LS1728 </td><td> </td></tr><tr><td> Rio Grande </td><td>Inbred line </td><td>TGRC </td><td>LA3343 </td><td>Tested </td></tr><tr><td> Rutgers </td><td>Inbred line </td><td>TGRC </td><td>LA1090 </td><td>Tested </td></tr><tr><td> San Marzano </td><td>Inbred line </td><td>NIVTS </td><td>LS4956 </td><td>Tested </td></tr><tr><td> Tomato Chuukanbohon Nou 9 </td><td>Inbred line </td><td>NIVTS </td><td> </td><td> </td></tr><tr><td> Tomato Chuukanbohon Nou 11 </td><td>Inbred line </td><td>NIVTS </td><td> </td><td> </td></tr><tr><td> Geronimo </td><td>F<sub>1</sub> hybrid </td><td>De Ruiter Seeds Co. </td><td> </td><td>Tested </td></tr><tr><td> Labell </td><td>F<sub>1</sub> hybrid </td><td>De Ruiter Seeds Co. </td><td> </td><td>Tested </td></tr><tr><td> Matrix </td><td>F<sub>1</sub> hybrid </td><td>De Ruiter Seeds Co. </td><td> </td><td>Tested </td></tr><tr><td> Momotaro 8 </td><td>F<sub>1</sub> hybrid </td><td>Takii Seeds Co. </td><td> </td><td>Tested </td></tr><tr><td> Reika </td><td>F<sub>1</sub> hybrid </td><td>Sakata Seeds Co. </td><td> </td><td>Tested </td></tr><tr><td> Regina </td><td>Inbred line, cherry type </td><td>Sakata Seeds Co. </td><td> </td><td> </td></tr><tr><td> Sweet100 </td><td>F<sub>1</sub> hybrid, cherry type </td><td>Vilmorin Seeds Co. </td><td> </td><td>Tested </td></tr><tr><td> LA716 </td><td>Inbred line, <em>S. pennellii</em> </td><td>Cornell University </td><td> </td><td> </td></tr></tbody></table></div><div content-id="DSQ024TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Line name </th><th>Note </th><th>Source<sup>a</sup> </th><th>Accession number </th><th>SNP validation<sup>b</sup> </th></tr></thead><tbody><tr><td colspan="5">Parental lines of mapping populations </td></tr><tr><td> Micro-Tom_AM </td><td>Inbred line </td><td>NIVTS </td><td> </td><td>Tested </td></tr><tr><td> Ailsa Craig </td><td>Inbred line </td><td>NBRP </td><td>TOMJPF00004 </td><td>Tested </td></tr><tr><td> Micro-Tom_MM </td><td>Inbred line </td><td>INRA </td><td> </td><td> </td></tr><tr><td> M82 </td><td>Inbred line </td><td>INRA </td><td> </td><td>Tested </td></tr><tr><td colspan="5">Tomato lines for SNP typing </td></tr><tr><td> Aichi First </td><td>Inbred line </td><td>NBRP </td><td>TOMJPF00003 </td><td> </td></tr><tr><td> Best of All </td><td>Inbred line </td><td>NIVTS </td><td>LS3908 </td><td> </td></tr><tr><td> Earliana </td><td>Inbred line </td><td>TGRC </td><td>LA3238 </td><td>Tested </td></tr><tr><td> Fruit </td><td>Inbred line </td><td>NIVTS </td><td>LS1100 </td><td> </td></tr><tr><td> Furikoma </td><td>Inbred line </td><td>NIVTS </td><td>LS3903 </td><td> </td></tr><tr><td> Heinz 1706-BG </td><td>Inbred line </td><td>NIVTS </td><td>LS461 </td><td>Tested </td></tr><tr><td> LA925 </td><td>Inbred line </td><td>Cornell University </td><td> </td><td>Tested </td></tr><tr><td> Marglobe </td><td>Inbred line </td><td>TGRC </td><td>LA0502 </td><td>Tested </td></tr><tr><td> Money Maker </td><td>Inbred line </td><td>TGRC </td><td>LA2706 </td><td>Tested </td></tr><tr><td> Ponderosa </td><td>Inbred line </td><td>NIVTS </td><td>LS1728 </td><td> </td></tr><tr><td> Rio Grande </td><td>Inbred line </td><td>TGRC </td><td>LA3343 </td><td>Tested </td></tr><tr><td> Rutgers </td><td>Inbred line </td><td>TGRC </td><td>LA1090 </td><td>Tested </td></tr><tr><td> San Marzano </td><td>Inbred line </td><td>NIVTS </td><td>LS4956 </td><td>Tested </td></tr><tr><td> Tomato Chuukanbohon Nou 9 </td><td>Inbred line </td><td>NIVTS </td><td> </td><td> </td></tr><tr><td> Tomato Chuukanbohon Nou 11 </td><td>Inbred line </td><td>NIVTS </td><td> </td><td> </td></tr><tr><td> Geronimo </td><td>F<sub>1</sub> hybrid </td><td>De Ruiter Seeds Co. </td><td> </td><td>Tested </td></tr><tr><td> Labell </td><td>F<sub>1</sub> hybrid </td><td>De Ruiter Seeds Co. </td><td> </td><td>Tested </td></tr><tr><td> Matrix </td><td>F<sub>1</sub> hybrid </td><td>De Ruiter Seeds Co. </td><td> </td><td>Tested </td></tr><tr><td> Momotaro 8 </td><td>F<sub>1</sub> hybrid </td><td>Takii Seeds Co. </td><td> </td><td>Tested </td></tr><tr><td> Reika </td><td>F<sub>1</sub> hybrid </td><td>Sakata Seeds Co. </td><td> </td><td>Tested </td></tr><tr><td> Regina </td><td>Inbred line, cherry type </td><td>Sakata Seeds Co. </td><td> </td><td> </td></tr><tr><td> Sweet100 </td><td>F<sub>1</sub> hybrid, cherry type </td><td>Vilmorin Seeds Co. </td><td> </td><td>Tested </td></tr><tr><td> LA716 </td><td>Inbred line, <em>S. pennellii</em> </td><td>Cornell University </td><td> </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>NBRP: University of Tsukuba in National Bio-Resource Project of MEXT, Japan; INRA: National Institute for Agricultural Research, France; NIVTS: National Institute of Vegetable and Tea Science, Japan; TGRC: Tomato Genetics Resource Center, University of California, Davis, USA.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Lines that used for validation of 82 eSNPs prior to design a SNP genotyping platform using Illumina GoldenGate<sup>®</sup> assay.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7361973" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=7361975 id="7361975" class="section-title" data-legacy-id=s2b>2.2. Development of SNP markers and polymorphic analysis</h3>
<p>A total of 229 086 EST sequences from <em>S. lycopersicum</em>, retrieved from two public databases, SGN (<a class="link link-uri" href="http://solgenomics.net/">http://solgenomics.net/</a>) and MiBASE (<a class="link link-uri" href="http://www.pgb.kazusa.or.jp/mibase/">http://www.pgb.kazusa.or.jp/mibase/</a>), were used for identification of eSNPs, i.e. SNPs discovered <em>in silico</em>. The ESTs registered in MiBASE were derived only from ‘Micro-Tom’, while those registered in SGN were developed from 19 tomato lines including ‘Micro-Tom’. The retrieved EST sequences were assembled using the MIRA program.<sup><a reveal-id="DSQ024C37" data-open="DSQ024C37" class="link link-ref link-reveal xref-bibr">37</a></sup> The eSNPs were then selected according to the following three criteria: (i) only nucleotides with Phred scores of 15 or more were considered candidates for eSNPs, (ii) a nucleotide at an eSNP site should be identical among multiple sequences within a given line, and (iii) no other SNP candidates should be detected on the flanking sequences 10 bp upstream and downstream of a given candidate.</p><p>In order to validate the credibility of the identified eSNP, nucleotide sequences of PCR products containing the eSNP regions were determined by direct sequencing using a DNA sequencer (ABI-3730<em>xl</em>, Applied Biosystems, Foster City, CA, USA). A total of 82 primer pairs were designed in flanking regions of the randomly selected target eSNPs using the Primer3 program.<sup><a reveal-id="DSQ024C38" data-open="DSQ024C38" class="link link-ref link-reveal xref-bibr">38</a></sup> PCR was performed for 17 tomato lines listed in Table <a reveal-id="DSQ024TB1" data-open="DSQ024TB1" class="link link-reveal link-table xref-fig">1</a> in a 5-µl reaction mixture containing 0.5 ng genomic DNA, 1× PCR buffer (Bioline, London, UK), 3 mM MgCl<sub>2</sub>, 0.04 U BIOTAQ™ DNA polymerase (Bioline), 0.2 mM dNTPs, and 0.8 µM of each of the primers. The modified ‘touchdown PCR’ protocol was used as described previously.<sup><a reveal-id="DSQ024C39" data-open="DSQ024C39" class="link link-ref link-reveal xref-bibr">39</a></sup></p><p>After validation of the 82 eSNPs, a total of 1536 eSNPs were subjected to polymorphic analysis for the two mapping populations and the 23 tomato lines described above using the GoldenGate<sup>®</sup> assay system (Illumina). Allele- and locus-specific oligonucleotides were designed from the flanking sequences of the 1536 SNP sites using the iCom website (<a class="link link-uri" href="https://icom.illumina.com/">https://icom.illumina.com/</a>). Polymorphic analysis of the SNPs was performed according to the standard protocol of the GoldenGate<sup>®</sup> assay, and the data analysis was performed using GenomeStudio Data Analysis software (Illumina).</p><p>SNPs in <em>DWARF</em> (<em>D</em>) and <em>SELF-PRUNING</em> (<em>SP</em>) were analyzed using the dCAPS and CAPS methods, respectively. PCR was performed under the same conditions as described above. The primer sequences are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. The PCR products from the <em>D</em> and <em>SP</em> genes were digested with <em>Pst</em>I and <em>Mva</em>I, respectively, and were subjected to electrophoresis on native 10% polyacrylamide gels in 1× TBE buffer. The resulting DNA bands were then stained with ethidium bromide.</p>                            <h3 scrollto-destination=7361986 id="7361986" class="section-title" data-legacy-id=s2c>2.3. Mapping of SSR and intronic polymorphic markers on AMF2</h3>
<p>A total of 3510 tomato genomic SSR (TGS), 2047 tomato EST-SSR (TES), and 166 tomato EST-derived intronic polymorphic (TEI) markers, developed by Shirasawa <em>et al.</em>,<sup><a reveal-id="DSQ024C7" data-open="DSQ024C7" class="link link-ref link-reveal xref-bibr">7</a></sup> were used for segregation analysis of the AMF2 population (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The polymorphic analyses of the markers were performed as described previously.<sup><a reveal-id="DSQ024C7" data-open="DSQ024C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Primer information for the tested markers is available at <a class="link link-uri" href="http://www.kazusa.or.jp/tomato/">http://www.kazusa.or.jp/tomato/</a>.</p>                            <h3 scrollto-destination=7361991 id="7361991" class="section-title" data-legacy-id=s2d>2.4. Linkage analysis</h3>
<p>Linkage analysis was performed using the JoinMap<sup>®</sup> program, version 4.<sup><a reveal-id="DSQ024C40" data-open="DSQ024C40" class="link link-ref link-reveal xref-bibr">40</a></sup> The segregated data were classified into 12 linkage groups, which corresponded to the Tomato-EXPEN 2000 map,<sup><a reveal-id="DSQ024C7" data-open="DSQ024C7" class="link link-ref link-reveal xref-bibr">7</a></sup> using the grouping module of JoinMap<sup>®</sup> with LOD scores of 4.0–10.0. The marker order and relative genetic distances were calculated by the regression-mapping algorithm with the following parameters: Haldane's mapping function, recombination frequency ≤0.35, and LOD score ≥2.0.</p>                            <h2 scrollto-destination=7361996 id="7361996" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=7361998 id="7361998" class="section-title" data-legacy-id=s3a>3.1. In silico SNP mining and validation</h3>
<p>A total of 170 586 and 58 500 EST sequences available in SGN and MiBASE, respectively, along with data on their quality, were used for <em>in silico</em> SNP mining. The name of the original tomato line for each EST was obtained from the DDBJ database (<a class="link link-uri" href="http://www.ddbj.nig.ac.jp/">http://www.ddbj.nig.ac.jp/</a>). In total, 229 086 ESTs derived from 20 tomato cultivars, the average length of which was 497 bp, were used for assembly (Table <a reveal-id="DSQ024TB2" data-open="DSQ024TB2" class="link link-reveal link-table xref-fig">2</a>). </p>                        <a id="7362000" scrollto-destination="7362000"></a>
<div content-id="DSQ024TB2" class="table-modal"><div class="table-wrap"><div id="DSQ024TB2" class="table-wrap-title" data-id="DSQ024TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Number of ESTs and their original sources used for assembling</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Line name<sup>a</sup> </th><th>No. of ESTs </th></tr></thead><tbody><tr><td>TA496 </td><td>106 142 </td></tr><tr><td>Micro-Tom </td><td>101 157 </td></tr><tr><td>Rio Grande PtoR </td><td>8803 </td></tr><tr><td>R11-13 </td><td>5031 </td></tr><tr><td>R11-12 </td><td>4925 </td></tr><tr><td>TA492 </td><td>2120 </td></tr><tr><td>West Virginia 106 </td><td>861 </td></tr><tr><td>Money Maker </td><td>11 </td></tr><tr><td>Ailsa Craig </td><td>7 </td></tr><tr><td>VF36 </td><td>7 </td></tr><tr><td>Momotaro </td><td>4 </td></tr><tr><td>Zhongshu 4 </td><td>4 </td></tr><tr><td>Betterboy </td><td>3 </td></tr><tr><td>Vendor </td><td>3 </td></tr><tr><td>House Odoriko </td><td>2 </td></tr><tr><td>Rutgers </td><td>2 </td></tr><tr><td>M82 </td><td>1 </td></tr><tr><td>Pera </td><td>1 </td></tr><tr><td>Rio Grande </td><td>1 </td></tr><tr><td>UC82B </td><td>1 </td></tr><tr><td>Total </td><td>229 086 </td></tr></tbody></table></div><div content-id="DSQ024TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Line name<sup>a</sup> </th><th>No. of ESTs </th></tr></thead><tbody><tr><td>TA496 </td><td>106 142 </td></tr><tr><td>Micro-Tom </td><td>101 157 </td></tr><tr><td>Rio Grande PtoR </td><td>8803 </td></tr><tr><td>R11-13 </td><td>5031 </td></tr><tr><td>R11-12 </td><td>4925 </td></tr><tr><td>TA492 </td><td>2120 </td></tr><tr><td>West Virginia 106 </td><td>861 </td></tr><tr><td>Money Maker </td><td>11 </td></tr><tr><td>Ailsa Craig </td><td>7 </td></tr><tr><td>VF36 </td><td>7 </td></tr><tr><td>Momotaro </td><td>4 </td></tr><tr><td>Zhongshu 4 </td><td>4 </td></tr><tr><td>Betterboy </td><td>3 </td></tr><tr><td>Vendor </td><td>3 </td></tr><tr><td>House Odoriko </td><td>2 </td></tr><tr><td>Rutgers </td><td>2 </td></tr><tr><td>M82 </td><td>1 </td></tr><tr><td>Pera </td><td>1 </td></tr><tr><td>Rio Grande </td><td>1 </td></tr><tr><td>UC82B </td><td>1 </td></tr><tr><td>Total </td><td>229 086 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Names of tomato lines used for EST generation.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7362000" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ024TB2" class="table-wrap-title" data-id="DSQ024TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Number of ESTs and their original sources used for assembling</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Line name<sup>a</sup> </th><th>No. of ESTs </th></tr></thead><tbody><tr><td>TA496 </td><td>106 142 </td></tr><tr><td>Micro-Tom </td><td>101 157 </td></tr><tr><td>Rio Grande PtoR </td><td>8803 </td></tr><tr><td>R11-13 </td><td>5031 </td></tr><tr><td>R11-12 </td><td>4925 </td></tr><tr><td>TA492 </td><td>2120 </td></tr><tr><td>West Virginia 106 </td><td>861 </td></tr><tr><td>Money Maker </td><td>11 </td></tr><tr><td>Ailsa Craig </td><td>7 </td></tr><tr><td>VF36 </td><td>7 </td></tr><tr><td>Momotaro </td><td>4 </td></tr><tr><td>Zhongshu 4 </td><td>4 </td></tr><tr><td>Betterboy </td><td>3 </td></tr><tr><td>Vendor </td><td>3 </td></tr><tr><td>House Odoriko </td><td>2 </td></tr><tr><td>Rutgers </td><td>2 </td></tr><tr><td>M82 </td><td>1 </td></tr><tr><td>Pera </td><td>1 </td></tr><tr><td>Rio Grande </td><td>1 </td></tr><tr><td>UC82B </td><td>1 </td></tr><tr><td>Total </td><td>229 086 </td></tr></tbody></table></div><div content-id="DSQ024TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Line name<sup>a</sup> </th><th>No. of ESTs </th></tr></thead><tbody><tr><td>TA496 </td><td>106 142 </td></tr><tr><td>Micro-Tom </td><td>101 157 </td></tr><tr><td>Rio Grande PtoR </td><td>8803 </td></tr><tr><td>R11-13 </td><td>5031 </td></tr><tr><td>R11-12 </td><td>4925 </td></tr><tr><td>TA492 </td><td>2120 </td></tr><tr><td>West Virginia 106 </td><td>861 </td></tr><tr><td>Money Maker </td><td>11 </td></tr><tr><td>Ailsa Craig </td><td>7 </td></tr><tr><td>VF36 </td><td>7 </td></tr><tr><td>Momotaro </td><td>4 </td></tr><tr><td>Zhongshu 4 </td><td>4 </td></tr><tr><td>Betterboy </td><td>3 </td></tr><tr><td>Vendor </td><td>3 </td></tr><tr><td>House Odoriko </td><td>2 </td></tr><tr><td>Rutgers </td><td>2 </td></tr><tr><td>M82 </td><td>1 </td></tr><tr><td>Pera </td><td>1 </td></tr><tr><td>Rio Grande </td><td>1 </td></tr><tr><td>UC82B </td><td>1 </td></tr><tr><td>Total </td><td>229 086 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Names of tomato lines used for EST generation.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7362000" target="_blank">View Large</a></div></div><p>Assembly was performed using nucleotides with Phred scores ≥15. As a result, a total of 20 274 contigs, the average length of which was 775 bp, and 29 698 singletons were generated. From initial alignment data from all 20 274 contigs, a total of 5607 eSNP sites were identified in 2634 of these contigs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S2</a> and <a class="link link-uri" href="http://dnaresearch.oxfordjournals.org/cgi/content/full/dsq024/DC1" target="_blank">S3</a>). We gave an SNP code to each eSNP according to the following rule: contig name and position of the eSNP on the contig, linked with an underscore, e.g. the 112th position on contig 2758 was given the following SNP code: 2758_112.</p><p>Before designing the SNP genotyping platform (using the Illumina GoldenGate<sup>®</sup> assay), 82 randomly selected eSNPs were tested in 17 tomato lines (Table <a reveal-id="DSQ024TB1" data-open="DSQ024TB1" class="link link-reveal link-table xref-fig">1</a>) by direct sequencing of fragments amplified by PCR. As a result, 55 (67%) out of the 82 examined eSNP candidates were experimentally confirmed as SNPs at the predicted positions, indicating that approximately 67% of the 5607 eSNPs detected <em>in silico</em> represent true SNPs in the tomato lines used in the present study. In addition, 40 (49%) and 50 (61%) of the 82 eSNPs segregated between the two mapping parents for AMF2 and MMF2, respectively.</p><p>For SNP genotyping, a total of 1536 SNPs were selected from the 5607 eSNPs, as follows: (i) one eSNP was selected from each contig and the Selected-BAC-Mixture contig released from the Kazusa Tomato SBM &amp; Marker Database (<a class="link link-uri" href="http://www.kazusa.or.jp/tomato/">http://www.kazusa.or.jp/tomato/</a>); (ii) an SNP score of more than 0.6, as determined by the iCom website of Illumina (<a class="link link-uri" href="https://icom.illumina.com/">https://icom.illumina.com/</a>), was required for each of these eSNPs. As reported by the GoldenGate<sup>®</sup> assay, 1338 (87%) out of the 1536 SNPs could be properly genotyped in the 279 plants. These included the two mapping populations (AMF2 and MMF2) and 23 other tomato lines. The remaining 198 (13%) eSNPs failed to be genotyped because fluorescent signals for these eSNPs did not form clusters pursuant to the criteria required by the GenomeStudio Data Analysis software (Illumina).</p>                            <h3 scrollto-destination=7362004 id="7362004" class="section-title" data-legacy-id=s3b>3.2. Mapping of SNP, SSR, and intronic markers</h3>
<p>In the AMF2 population, 648 of the 1338 available SNPs (48.4%) generated segregation data, a similar ratio to that determined in the validation of the 82 eSNPs. Two SNP markers designed in the <em>D</em> and <em>SP</em> genes, for which ‘Micro-Tom’ has mutant alleles,<sup><a reveal-id="DSQ024C41" data-open="DSQ024C41" class="link link-ref link-reveal xref-bibr">41</a></sup> showed polymorphism between ‘Ailsa Craig’ and ‘Micro-Tom’. Along with the SNP markers, a total of 5723 previously reported markers, including 2047 EST-SSR (TES), 3510 genomic-SSR (TGS), and 166 intronic (TEI) markers, were used for the polymorphic analysis. As a result, 96 TES (4.7%), 223 TGS (6.3%), and 28 TEI (16.8%) markers exhibited polymorphism between the parental lines. In total, 997 markers were used to construct the AMF2 linkage map.</p><p>In the MMF2 population, 640 of the 1338 available SNPs (47.9%) segregated. This ratio was over 10% less than that determined in the validation of the 82 eSNPs, suggesting that the result of the eSNP validation was overestimated. The SNP on the <em>D</em> gene showed polymorphism in the MMF2 mapping population, while two parental lines detected the mutated <em>sp</em> allele for the <em>SP</em> gene. In total, 641 segregated markers were used to construct the MMF2 map.</p>                            <h3 scrollto-destination=7362007 id="7362007" class="section-title" data-legacy-id=s3c>3.3. Construction of linkage maps</h3>
<p>For AMF2, a total of 989 of the 997 segregated loci (99.2%) formed 12 linkage groups (LGs), while 637 of the 641 segregated loci (99.4%) formed 13 linkage groups for MMF2. The total sizes of the LGs of the AMF2 and MMF2 maps were 1467.8 and 1422.7 cM, respectively (Table <a reveal-id="DSQ024TB3" data-open="DSQ024TB3" class="link link-reveal link-table xref-fig">3</a>, Fig. <a reveal-id="DSQ024F1" data-open="DSQ024F1" class="link link-reveal link-table xref-fig">1</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Combining the two maps yielded a total of 1137 markers, including 793 SNP, 221 TGS, 93 TES, 28 TEI, and 2 gene markers, located on the intraspecific map. Among these, 488 SNP markers were commonly located on both linkage maps, while 157 and 148 marker loci were specific to the AMF2 map and the MMF2 map, respectively. Chromosome 7 (Chr07) of MMF2 divided into two linkage groups, Chr07p and Chr07q, which were located at the upper and the lower portions, respectively, of Chr07 of Tomato-EXPEN 2000. The average lengths of the intervals between two loci on the AMF2 and the MMF2 maps were calculated to be 1.5 and 2.2 cM, respectively. </p>                        <a id="7362009" scrollto-destination="7362009"></a>
<div content-id="DSQ024TB3" class="table-modal"><div class="table-wrap"><div id="DSQ024TB3" class="table-wrap-title" data-id="DSQ024TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Length, number of mapped loci, and segregation distortion of the AMF2 and MMF2 maps</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Chromosome </th><th colspan="8">AMF2<hr /> </th><th colspan="5">MMF2<hr /> </th></tr><tr><th> </th><th>Length (cM) </th><th colspan="6">Number of loci<hr /> </th><th>Segregation distortion ratio (%) </th><th>Length (cM) </th><th colspan="3">Number of loci<hr /> </th><th>Segregation distortion ratio (%) </th></tr><tr><th> </th><th> </th><th>TGS </th><th>TES </th><th>TEI </th><th>SNP </th><th>Gene </th><th>Total </th><th> </th><th> </th><th>SNP </th><th>Gene </th><th>Total </th><th> </th></tr></thead><tbody><tr><td>Chr01 </td><td>187.0 </td><td>18 </td><td>8 </td><td>5 </td><td>62 </td><td>0 </td><td>93 </td><td>0.0 </td><td>158.4 </td><td>57 </td><td>0 </td><td>57 </td><td>3.5 </td></tr><tr><td>Chr02 </td><td>141.9 </td><td>24 </td><td>6 </td><td>2 </td><td>39 </td><td>1 </td><td>72 </td><td>2.8 </td><td>105.1 </td><td>60 </td><td>1 </td><td>61 </td><td>9.8 </td></tr><tr><td>Chr03 </td><td>143.2 </td><td>29 </td><td>6 </td><td>3 </td><td>70 </td><td>0 </td><td>108 </td><td>11.1 </td><td>130.4 </td><td>100 </td><td>0 </td><td>100 </td><td>4.0 </td></tr><tr><td>Chr04 </td><td>132.0 </td><td>15 </td><td>15 </td><td>4 </td><td>114 </td><td>0 </td><td>148 </td><td>3.4 </td><td>125.3 </td><td>97 </td><td>0 </td><td>97 </td><td>4.1 </td></tr><tr><td>Chr05 </td><td>54.2 </td><td>45 </td><td>8 </td><td>1 </td><td>51 </td><td>0 </td><td>105 </td><td>2.9 </td><td>69.2 </td><td>70 </td><td>0 </td><td>70 </td><td>0.0 </td></tr><tr><td>Chr06 </td><td>95.5 </td><td>3 </td><td>0 </td><td>1 </td><td>9 </td><td>1 </td><td>14 </td><td>7.1 </td><td>95.1 </td><td>18 </td><td>0 </td><td>18 </td><td>16.7 </td></tr><tr><td>Chr07 </td><td>117.9 </td><td>8 </td><td>17 </td><td>1 </td><td>91 </td><td>0 </td><td>117 </td><td>3.4 </td><td>29.0<sup>a</sup> </td><td>54<sup>a</sup> </td><td>0 </td><td>54 </td><td>1.9 </td></tr><tr><td>Chr08 </td><td>124.0 </td><td>5 </td><td>5 </td><td>0 </td><td>22 </td><td>0 </td><td>32 </td><td>0.0 </td><td>92.8 </td><td>23 </td><td>0 </td><td>23 </td><td>0.0 </td></tr><tr><td>Chr09 </td><td>121.7 </td><td>13 </td><td>7 </td><td>7 </td><td>53 </td><td>0 </td><td>80 </td><td>15.0 </td><td>118.2 </td><td>53 </td><td>0 </td><td>53 </td><td>17.0 </td></tr><tr><td>Chr10 </td><td>97.3 </td><td>6 </td><td>3 </td><td>0 </td><td>22 </td><td>0 </td><td>31 </td><td>6.5 </td><td>87.2 </td><td>26 </td><td>0 </td><td>26 </td><td>11.5 </td></tr><tr><td>Chr11 </td><td>118.4 </td><td>25 </td><td>11 </td><td>3 </td><td>62 </td><td>0 </td><td>101 </td><td>55.4 </td><td>115.3 </td><td>43 </td><td>0 </td><td>43 </td><td>2.3 </td></tr><tr><td>Chr12 </td><td>134.7 </td><td>30 </td><td>7 </td><td>1 </td><td>50 </td><td>0 </td><td>88 </td><td>0.0 </td><td>104.0 </td><td>35 </td><td>0 </td><td>35 </td><td>2.9 </td></tr><tr><td>Total </td><td>1467.8 </td><td>221 </td><td>93 </td><td>28 </td><td>645 </td><td>2 </td><td>989 </td><td>9.8 </td><td>1422.7 </td><td>636 </td><td>1 </td><td>637 </td><td>5.3 </td></tr></tbody></table></div><div content-id="DSQ024TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Chromosome </th><th colspan="8">AMF2<hr /> </th><th colspan="5">MMF2<hr /> </th></tr><tr><th> </th><th>Length (cM) </th><th colspan="6">Number of loci<hr /> </th><th>Segregation distortion ratio (%) </th><th>Length (cM) </th><th colspan="3">Number of loci<hr /> </th><th>Segregation distortion ratio (%) </th></tr><tr><th> </th><th> </th><th>TGS </th><th>TES </th><th>TEI </th><th>SNP </th><th>Gene </th><th>Total </th><th> </th><th> </th><th>SNP </th><th>Gene </th><th>Total </th><th> </th></tr></thead><tbody><tr><td>Chr01 </td><td>187.0 </td><td>18 </td><td>8 </td><td>5 </td><td>62 </td><td>0 </td><td>93 </td><td>0.0 </td><td>158.4 </td><td>57 </td><td>0 </td><td>57 </td><td>3.5 </td></tr><tr><td>Chr02 </td><td>141.9 </td><td>24 </td><td>6 </td><td>2 </td><td>39 </td><td>1 </td><td>72 </td><td>2.8 </td><td>105.1 </td><td>60 </td><td>1 </td><td>61 </td><td>9.8 </td></tr><tr><td>Chr03 </td><td>143.2 </td><td>29 </td><td>6 </td><td>3 </td><td>70 </td><td>0 </td><td>108 </td><td>11.1 </td><td>130.4 </td><td>100 </td><td>0 </td><td>100 </td><td>4.0 </td></tr><tr><td>Chr04 </td><td>132.0 </td><td>15 </td><td>15 </td><td>4 </td><td>114 </td><td>0 </td><td>148 </td><td>3.4 </td><td>125.3 </td><td>97 </td><td>0 </td><td>97 </td><td>4.1 </td></tr><tr><td>Chr05 </td><td>54.2 </td><td>45 </td><td>8 </td><td>1 </td><td>51 </td><td>0 </td><td>105 </td><td>2.9 </td><td>69.2 </td><td>70 </td><td>0 </td><td>70 </td><td>0.0 </td></tr><tr><td>Chr06 </td><td>95.5 </td><td>3 </td><td>0 </td><td>1 </td><td>9 </td><td>1 </td><td>14 </td><td>7.1 </td><td>95.1 </td><td>18 </td><td>0 </td><td>18 </td><td>16.7 </td></tr><tr><td>Chr07 </td><td>117.9 </td><td>8 </td><td>17 </td><td>1 </td><td>91 </td><td>0 </td><td>117 </td><td>3.4 </td><td>29.0<sup>a</sup> </td><td>54<sup>a</sup> </td><td>0 </td><td>54 </td><td>1.9 </td></tr><tr><td>Chr08 </td><td>124.0 </td><td>5 </td><td>5 </td><td>0 </td><td>22 </td><td>0 </td><td>32 </td><td>0.0 </td><td>92.8 </td><td>23 </td><td>0 </td><td>23 </td><td>0.0 </td></tr><tr><td>Chr09 </td><td>121.7 </td><td>13 </td><td>7 </td><td>7 </td><td>53 </td><td>0 </td><td>80 </td><td>15.0 </td><td>118.2 </td><td>53 </td><td>0 </td><td>53 </td><td>17.0 </td></tr><tr><td>Chr10 </td><td>97.3 </td><td>6 </td><td>3 </td><td>0 </td><td>22 </td><td>0 </td><td>31 </td><td>6.5 </td><td>87.2 </td><td>26 </td><td>0 </td><td>26 </td><td>11.5 </td></tr><tr><td>Chr11 </td><td>118.4 </td><td>25 </td><td>11 </td><td>3 </td><td>62 </td><td>0 </td><td>101 </td><td>55.4 </td><td>115.3 </td><td>43 </td><td>0 </td><td>43 </td><td>2.3 </td></tr><tr><td>Chr12 </td><td>134.7 </td><td>30 </td><td>7 </td><td>1 </td><td>50 </td><td>0 </td><td>88 </td><td>0.0 </td><td>104.0 </td><td>35 </td><td>0 </td><td>35 </td><td>2.9 </td></tr><tr><td>Total </td><td>1467.8 </td><td>221 </td><td>93 </td><td>28 </td><td>645 </td><td>2 </td><td>989 </td><td>9.8 </td><td>1422.7 </td><td>636 </td><td>1 </td><td>637 </td><td>5.3 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Chr07 of the MMF2 were divided into two linkage groups, Chr07p and Chr07q. The numbers indicate total value of two linkage groups.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7362009" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ024TB3" class="table-wrap-title" data-id="DSQ024TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Length, number of mapped loci, and segregation distortion of the AMF2 and MMF2 maps</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Chromosome </th><th colspan="8">AMF2<hr /> </th><th colspan="5">MMF2<hr /> </th></tr><tr><th> </th><th>Length (cM) </th><th colspan="6">Number of loci<hr /> </th><th>Segregation distortion ratio (%) </th><th>Length (cM) </th><th colspan="3">Number of loci<hr /> </th><th>Segregation distortion ratio (%) </th></tr><tr><th> </th><th> </th><th>TGS </th><th>TES </th><th>TEI </th><th>SNP </th><th>Gene </th><th>Total </th><th> </th><th> </th><th>SNP </th><th>Gene </th><th>Total </th><th> </th></tr></thead><tbody><tr><td>Chr01 </td><td>187.0 </td><td>18 </td><td>8 </td><td>5 </td><td>62 </td><td>0 </td><td>93 </td><td>0.0 </td><td>158.4 </td><td>57 </td><td>0 </td><td>57 </td><td>3.5 </td></tr><tr><td>Chr02 </td><td>141.9 </td><td>24 </td><td>6 </td><td>2 </td><td>39 </td><td>1 </td><td>72 </td><td>2.8 </td><td>105.1 </td><td>60 </td><td>1 </td><td>61 </td><td>9.8 </td></tr><tr><td>Chr03 </td><td>143.2 </td><td>29 </td><td>6 </td><td>3 </td><td>70 </td><td>0 </td><td>108 </td><td>11.1 </td><td>130.4 </td><td>100 </td><td>0 </td><td>100 </td><td>4.0 </td></tr><tr><td>Chr04 </td><td>132.0 </td><td>15 </td><td>15 </td><td>4 </td><td>114 </td><td>0 </td><td>148 </td><td>3.4 </td><td>125.3 </td><td>97 </td><td>0 </td><td>97 </td><td>4.1 </td></tr><tr><td>Chr05 </td><td>54.2 </td><td>45 </td><td>8 </td><td>1 </td><td>51 </td><td>0 </td><td>105 </td><td>2.9 </td><td>69.2 </td><td>70 </td><td>0 </td><td>70 </td><td>0.0 </td></tr><tr><td>Chr06 </td><td>95.5 </td><td>3 </td><td>0 </td><td>1 </td><td>9 </td><td>1 </td><td>14 </td><td>7.1 </td><td>95.1 </td><td>18 </td><td>0 </td><td>18 </td><td>16.7 </td></tr><tr><td>Chr07 </td><td>117.9 </td><td>8 </td><td>17 </td><td>1 </td><td>91 </td><td>0 </td><td>117 </td><td>3.4 </td><td>29.0<sup>a</sup> </td><td>54<sup>a</sup> </td><td>0 </td><td>54 </td><td>1.9 </td></tr><tr><td>Chr08 </td><td>124.0 </td><td>5 </td><td>5 </td><td>0 </td><td>22 </td><td>0 </td><td>32 </td><td>0.0 </td><td>92.8 </td><td>23 </td><td>0 </td><td>23 </td><td>0.0 </td></tr><tr><td>Chr09 </td><td>121.7 </td><td>13 </td><td>7 </td><td>7 </td><td>53 </td><td>0 </td><td>80 </td><td>15.0 </td><td>118.2 </td><td>53 </td><td>0 </td><td>53 </td><td>17.0 </td></tr><tr><td>Chr10 </td><td>97.3 </td><td>6 </td><td>3 </td><td>0 </td><td>22 </td><td>0 </td><td>31 </td><td>6.5 </td><td>87.2 </td><td>26 </td><td>0 </td><td>26 </td><td>11.5 </td></tr><tr><td>Chr11 </td><td>118.4 </td><td>25 </td><td>11 </td><td>3 </td><td>62 </td><td>0 </td><td>101 </td><td>55.4 </td><td>115.3 </td><td>43 </td><td>0 </td><td>43 </td><td>2.3 </td></tr><tr><td>Chr12 </td><td>134.7 </td><td>30 </td><td>7 </td><td>1 </td><td>50 </td><td>0 </td><td>88 </td><td>0.0 </td><td>104.0 </td><td>35 </td><td>0 </td><td>35 </td><td>2.9 </td></tr><tr><td>Total </td><td>1467.8 </td><td>221 </td><td>93 </td><td>28 </td><td>645 </td><td>2 </td><td>989 </td><td>9.8 </td><td>1422.7 </td><td>636 </td><td>1 </td><td>637 </td><td>5.3 </td></tr></tbody></table></div><div content-id="DSQ024TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Chromosome </th><th colspan="8">AMF2<hr /> </th><th colspan="5">MMF2<hr /> </th></tr><tr><th> </th><th>Length (cM) </th><th colspan="6">Number of loci<hr /> </th><th>Segregation distortion ratio (%) </th><th>Length (cM) </th><th colspan="3">Number of loci<hr /> </th><th>Segregation distortion ratio (%) </th></tr><tr><th> </th><th> </th><th>TGS </th><th>TES </th><th>TEI </th><th>SNP </th><th>Gene </th><th>Total </th><th> </th><th> </th><th>SNP </th><th>Gene </th><th>Total </th><th> </th></tr></thead><tbody><tr><td>Chr01 </td><td>187.0 </td><td>18 </td><td>8 </td><td>5 </td><td>62 </td><td>0 </td><td>93 </td><td>0.0 </td><td>158.4 </td><td>57 </td><td>0 </td><td>57 </td><td>3.5 </td></tr><tr><td>Chr02 </td><td>141.9 </td><td>24 </td><td>6 </td><td>2 </td><td>39 </td><td>1 </td><td>72 </td><td>2.8 </td><td>105.1 </td><td>60 </td><td>1 </td><td>61 </td><td>9.8 </td></tr><tr><td>Chr03 </td><td>143.2 </td><td>29 </td><td>6 </td><td>3 </td><td>70 </td><td>0 </td><td>108 </td><td>11.1 </td><td>130.4 </td><td>100 </td><td>0 </td><td>100 </td><td>4.0 </td></tr><tr><td>Chr04 </td><td>132.0 </td><td>15 </td><td>15 </td><td>4 </td><td>114 </td><td>0 </td><td>148 </td><td>3.4 </td><td>125.3 </td><td>97 </td><td>0 </td><td>97 </td><td>4.1 </td></tr><tr><td>Chr05 </td><td>54.2 </td><td>45 </td><td>8 </td><td>1 </td><td>51 </td><td>0 </td><td>105 </td><td>2.9 </td><td>69.2 </td><td>70 </td><td>0 </td><td>70 </td><td>0.0 </td></tr><tr><td>Chr06 </td><td>95.5 </td><td>3 </td><td>0 </td><td>1 </td><td>9 </td><td>1 </td><td>14 </td><td>7.1 </td><td>95.1 </td><td>18 </td><td>0 </td><td>18 </td><td>16.7 </td></tr><tr><td>Chr07 </td><td>117.9 </td><td>8 </td><td>17 </td><td>1 </td><td>91 </td><td>0 </td><td>117 </td><td>3.4 </td><td>29.0<sup>a</sup> </td><td>54<sup>a</sup> </td><td>0 </td><td>54 </td><td>1.9 </td></tr><tr><td>Chr08 </td><td>124.0 </td><td>5 </td><td>5 </td><td>0 </td><td>22 </td><td>0 </td><td>32 </td><td>0.0 </td><td>92.8 </td><td>23 </td><td>0 </td><td>23 </td><td>0.0 </td></tr><tr><td>Chr09 </td><td>121.7 </td><td>13 </td><td>7 </td><td>7 </td><td>53 </td><td>0 </td><td>80 </td><td>15.0 </td><td>118.2 </td><td>53 </td><td>0 </td><td>53 </td><td>17.0 </td></tr><tr><td>Chr10 </td><td>97.3 </td><td>6 </td><td>3 </td><td>0 </td><td>22 </td><td>0 </td><td>31 </td><td>6.5 </td><td>87.2 </td><td>26 </td><td>0 </td><td>26 </td><td>11.5 </td></tr><tr><td>Chr11 </td><td>118.4 </td><td>25 </td><td>11 </td><td>3 </td><td>62 </td><td>0 </td><td>101 </td><td>55.4 </td><td>115.3 </td><td>43 </td><td>0 </td><td>43 </td><td>2.3 </td></tr><tr><td>Chr12 </td><td>134.7 </td><td>30 </td><td>7 </td><td>1 </td><td>50 </td><td>0 </td><td>88 </td><td>0.0 </td><td>104.0 </td><td>35 </td><td>0 </td><td>35 </td><td>2.9 </td></tr><tr><td>Total </td><td>1467.8 </td><td>221 </td><td>93 </td><td>28 </td><td>645 </td><td>2 </td><td>989 </td><td>9.8 </td><td>1422.7 </td><td>636 </td><td>1 </td><td>637 </td><td>5.3 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Chr07 of the MMF2 were divided into two linkage groups, Chr07p and Chr07q. The numbers indicate total value of two linkage groups.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7362009" target="_blank">View Large</a></div></div>                        <a id="7362010" scrollto-destination="7362010"></a>
<div id="DSQ024F1" data-id="DSQ024F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq024/2/m_dsq02401.jpeg?Expires=2147483647&Signature=aitqfYtX~Makv0u16Dr4jwskI8vfVcL3bgxzlRAink~6-e719eeR~GNfXA7jHEmng8vnjzIGdua5rFmpwcXuPsQ0re0A6az28y7-NLhNdj7OmjIf9~YzQrUcPt9mJLTYYH8j95m1hguQq5c195GPZtbr2pnL-Iv4HI5DRD6raUdu08YqZzp0ztjvzQeUI~8oDxkrHLWuMh3lZ6QZ8v6U0E7x6gysf-7TUe46Udq7tqdG9B0pM2w7zWYrbLSoHmMR~w3P4hJv3IHMec6Ff~R8jDc50ayh4LgkrbRGgaQJK~GzCm0Z06UyINOQtRugmMrPdZuteexw~3maT7pabd5Rmw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Intraspecific linkage maps for tomato. The middle and right bars of each linkage group represent maps of the AMF2 and MMF2 lines, respectively. The left bar represents a linkage map of Tomato-EXPEN 2000, which is an interspecific map constructed in our previous study.7 The lines on each linkage group show SNP (green), TGS (red), TES (blue), TEI (yellow), and the other available (black) markers. Common markers between two maps are connected with black lines. Boxes on the Tomato-EXPEN 2000 map indicate heterochromatic regions. Detailed information on the markers, including marker positions, is shown in Supplementary Table S1, and is also available at http://www.kazusa.or.jp/tomato/." path-from-xml="dsq02401.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7362010/dsq02401.jpeg" path-from-xml="dsq02401.jpeg" target="_blank">View large</a><a section="7362010" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq024/2/dsq02401.jpeg?Expires=2147483647&Signature=C2t9YKXPAne27plz~NTpKF-rjsUqoYHADlTIUdhwf5h4zohryAjKed7YWRCvwGxlgEH~nonmC0Gr91~vyUuWchp5ZDy3QWiin7nLaJNb3cIt-oAt4AEAliyuUtmpQQgdiHXDGrmXNjOtcKKv1oNTtOoxJuZUAl5Ty0qqwO3PBfmU3rqkMUXOxP4Czw90Sp2FrljlY6bvb-EaaS8r5wIZFjz2bnQ2LmqMk8fomqjJqT7l1-VBkq1ALmArYiNF8A9F~TTMx2QEZc4h942ZduozZNIMtrjyGAl06aMVfxyNIE6lWE5IJBivur5jOBxnDkPJKzNPYvnbMfqKf5AT~asdRg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7362010&ar=484555&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02401.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Intraspecific linkage maps for tomato. The middle and right bars of each linkage group represent maps of the AMF2 and MMF2 lines, respectively. The left bar represents a linkage map of Tomato-EXPEN 2000, which is an interspecific map constructed in our previous study.<sup><a reveal-id="DSQ024C7" data-open="DSQ024C7" class="link link-ref link-reveal xref-bibr">7</a></sup> The lines on each linkage group show SNP (green), TGS (red), TES (blue), TEI (yellow), and the other available (black) markers. Common markers between two maps are connected with black lines. Boxes on the Tomato-EXPEN 2000 map indicate heterochromatic regions. Detailed information on the markers, including marker positions, is shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>, and is also available at <a class="link link-uri" href="http://www.kazusa.or.jp/tomato/">http://www.kazusa.or.jp/tomato/</a>.</p></div></div><div content-id="DSQ024F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq024/2/m_dsq02401.jpeg?Expires=2147483647&Signature=aitqfYtX~Makv0u16Dr4jwskI8vfVcL3bgxzlRAink~6-e719eeR~GNfXA7jHEmng8vnjzIGdua5rFmpwcXuPsQ0re0A6az28y7-NLhNdj7OmjIf9~YzQrUcPt9mJLTYYH8j95m1hguQq5c195GPZtbr2pnL-Iv4HI5DRD6raUdu08YqZzp0ztjvzQeUI~8oDxkrHLWuMh3lZ6QZ8v6U0E7x6gysf-7TUe46Udq7tqdG9B0pM2w7zWYrbLSoHmMR~w3P4hJv3IHMec6Ff~R8jDc50ayh4LgkrbRGgaQJK~GzCm0Z06UyINOQtRugmMrPdZuteexw~3maT7pabd5Rmw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Intraspecific linkage maps for tomato. The middle and right bars of each linkage group represent maps of the AMF2 and MMF2 lines, respectively. The left bar represents a linkage map of Tomato-EXPEN 2000, which is an interspecific map constructed in our previous study.7 The lines on each linkage group show SNP (green), TGS (red), TES (blue), TEI (yellow), and the other available (black) markers. Common markers between two maps are connected with black lines. Boxes on the Tomato-EXPEN 2000 map indicate heterochromatic regions. Detailed information on the markers, including marker positions, is shown in Supplementary Table S1, and is also available at http://www.kazusa.or.jp/tomato/." path-from-xml="dsq02401.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7362010/dsq02401.jpeg" path-from-xml="dsq02401.jpeg" target="_blank">View large</a><a section="7362010" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq024/2/dsq02401.jpeg?Expires=2147483647&Signature=C2t9YKXPAne27plz~NTpKF-rjsUqoYHADlTIUdhwf5h4zohryAjKed7YWRCvwGxlgEH~nonmC0Gr91~vyUuWchp5ZDy3QWiin7nLaJNb3cIt-oAt4AEAliyuUtmpQQgdiHXDGrmXNjOtcKKv1oNTtOoxJuZUAl5Ty0qqwO3PBfmU3rqkMUXOxP4Czw90Sp2FrljlY6bvb-EaaS8r5wIZFjz2bnQ2LmqMk8fomqjJqT7l1-VBkq1ALmArYiNF8A9F~TTMx2QEZc4h942ZduozZNIMtrjyGAl06aMVfxyNIE6lWE5IJBivur5jOBxnDkPJKzNPYvnbMfqKf5AT~asdRg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7362010&ar=484555&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02401.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Intraspecific linkage maps for tomato. The middle and right bars of each linkage group represent maps of the AMF2 and MMF2 lines, respectively. The left bar represents a linkage map of Tomato-EXPEN 2000, which is an interspecific map constructed in our previous study.<sup><a reveal-id="DSQ024C7" data-open="DSQ024C7" class="link link-ref link-reveal xref-bibr">7</a></sup> The lines on each linkage group show SNP (green), TGS (red), TES (blue), TEI (yellow), and the other available (black) markers. Common markers between two maps are connected with black lines. Boxes on the Tomato-EXPEN 2000 map indicate heterochromatic regions. Detailed information on the markers, including marker positions, is shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>, and is also available at <a class="link link-uri" href="http://www.kazusa.or.jp/tomato/">http://www.kazusa.or.jp/tomato/</a>.</p></div></div><p>Segregation distortions were observed in the two maps. In the AMF2 map, 9.8% of the marker loci showed segregation distortions, ranging from 0.0% for Chr01, Chr08, and Chr12, to 55.4% for Chr11 (Table <a reveal-id="DSQ024TB3" data-open="DSQ024TB3" class="link link-reveal link-table xref-fig">3</a>). In the MMF2 map, 5.3% of the marker loci were distorted, ranging from 0.0% for Chr05 and Chr08, to 17.0% for Chr09 (Table <a reveal-id="DSQ024TB3" data-open="DSQ024TB3" class="link link-reveal link-table xref-fig">3</a>). The linkage groups harboring severe segregation distortions were different between the two mapping populations, especially between Chr11 of AMF2 (55.4%) and that of MMF2 (2.3%), suggesting Chr11 of ‘Ailsa Craig’ might have transmission ratio distorters.</p>                            <h3 scrollto-destination=7362012 id="7362012" class="section-title" data-legacy-id=s3d>3.4. Polymorphic analysis of the SNP markers in tomato cultivars and <em>S. pennellii</em></h3>
<p>A total of 916 (68.5%) out of the 1338 SNP markers showed polymorphisms in at least one line among the 27 tomato lines listed in Table <a reveal-id="DSQ024TB1" data-open="DSQ024TB1" class="link link-reveal link-table xref-fig">1</a> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). The polymorphic ratio was similar to the ratio determined during the PCR-based validation of the 82 eSNPs. In ‘LA719’ (<em>S. pennellii</em>) and ‘Sweet 100’, no data were obtained for 229 (17.1%) and one SNP markers, respectively. The polymorphic ratios differed according to the combination of tomato lines (Fig. <a reveal-id="DSQ024F2" data-open="DSQ024F2" class="link link-reveal link-table xref-fig">2</a>), and the number of segregated SNPs between any two lines among the 27 lines was 255.0 (19.1%) on average. A total of 608.2 SNPs (45.5%) were identified between ‘Micro-Tom’ and the other inbred lines, on average, while only 80.8 SNPs (6.0%) were identified among the 17 inbred tomato lines. Within the 17 inbred tomato lines, ‘M82’ showed the highest number of polymorphisms: 176.3 SNPs (13.2%) on average, which was twice as high as that of the other lines. SNPs between the F<sub>1</sub> hybrid cultivars and the inbred lines were found at 190.6 loci (14.2%) on average. The two cherry-type tomato cultivars showed higher polymorphisms than the inbred tomato lines, with 310.1 (23.2%) SNPs on average. When 26 <em>S. lycopersicum</em> lines were compared with <em>S. pennellii</em>, on average, 618.5 out of the 1338 loci (46.2%) were polymorphic. Heterozygosity was observed at multiple SNP sites in all six F<sub>1</sub> hybrid cultivars, ranging from 69 (5.2%) in ‘Matrix’ to 229 (17.1%) in ‘Sweet 100’. In the inbred line ‘Rio Grande’, 25 (1.9%) heterozygous SNPs were identified. </p>                        <a id="7362014" scrollto-destination="7362014"></a>
<div id="DSQ024F2" data-id="DSQ024F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq024/2/m_dsq02402.jpeg?Expires=2147483647&Signature=xYfRG2a5Jj5ThlbmpZ5oJttfuJ9WcZDikfmppWsEzJPE0LAB3fveBlpAB3hUll50qQgJhttkq3a3CIn0WFkXf3l~c6i9EcVkEHjC8ktoOdybDEsWBFDQ-CMFbGoTgXOqVYjjKWtSOUw-WSw4NbbDZrLY5H0d9aPKoctwYTPtgbUXwZz8UDN-RyW~kTgbF-RrQMnbEG8NKcgwTnOJNvnPiOqJzlr8yHfCsZsRhR6aRbVXUXW6HJXCOcG7l30zZAYJ3SubIVlBjSyALJOHrcO7sQnG2Zeaje2K5BMi2DHYe4VFbK~qcbxfndCc7codP2abT3PPDU7PZJek2ljav7dvFw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Number of SNP markers identified by the GoldenGate® assay among 27 tomato and S. pennellii lines. The numbers show the loci of the 1338 SNPs genotyped in this study. The colors in each cell represent a continuum of polymorphic ratios: lower ratios are represented by green color, higher ratios by red color, and middle ratios by yellow color." path-from-xml="dsq02402.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7362014/dsq02402.jpeg" path-from-xml="dsq02402.jpeg" target="_blank">View large</a><a section="7362014" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq024/2/dsq02402.jpeg?Expires=2147483647&Signature=fphlHsEwpc3m~bQLaediSEUFQs7LafbIq5evpQD~-Z59yFxdhKKoCzaBF--cny9ierivB8bv1BEhwhAq8vQm2uj~QN1oIghEY3DSccV77Qthpm94UYIUGJu8SxFLpJX1S2KnFEJdksjRou68uQVy7lPDYGHLG5WueY43LmqqnJuS79-f4RR~oEHC4d2~vj3QCTLah~jBcDnP2V1mXCMfZ~O~9Y22TWeXZcvJ1z3H09NBKTmxnXRQAi-cxJqnFbuFzFDbGzM~1KuT1xT~0zoJd32mFAAZLnHCr-j36el25gFCH0ZD3xlLCFtatpApB-WBYJlyKfTerlp3y-NnVaXYZA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7362014&ar=484555&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02402.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Number of SNP markers identified by the GoldenGate<sup>®</sup> assay among 27 tomato and <em>S. pennellii</em> lines. The numbers show the loci of the 1338 SNPs genotyped in this study. The colors in each cell represent a continuum of polymorphic ratios: lower ratios are represented by green color, higher ratios by red color, and middle ratios by yellow color.</p></div></div><div content-id="DSQ024F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq024/2/m_dsq02402.jpeg?Expires=2147483647&Signature=xYfRG2a5Jj5ThlbmpZ5oJttfuJ9WcZDikfmppWsEzJPE0LAB3fveBlpAB3hUll50qQgJhttkq3a3CIn0WFkXf3l~c6i9EcVkEHjC8ktoOdybDEsWBFDQ-CMFbGoTgXOqVYjjKWtSOUw-WSw4NbbDZrLY5H0d9aPKoctwYTPtgbUXwZz8UDN-RyW~kTgbF-RrQMnbEG8NKcgwTnOJNvnPiOqJzlr8yHfCsZsRhR6aRbVXUXW6HJXCOcG7l30zZAYJ3SubIVlBjSyALJOHrcO7sQnG2Zeaje2K5BMi2DHYe4VFbK~qcbxfndCc7codP2abT3PPDU7PZJek2ljav7dvFw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Number of SNP markers identified by the GoldenGate® assay among 27 tomato and S. pennellii lines. The numbers show the loci of the 1338 SNPs genotyped in this study. The colors in each cell represent a continuum of polymorphic ratios: lower ratios are represented by green color, higher ratios by red color, and middle ratios by yellow color." path-from-xml="dsq02402.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7362014/dsq02402.jpeg" path-from-xml="dsq02402.jpeg" target="_blank">View large</a><a section="7362014" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq024/2/dsq02402.jpeg?Expires=2147483647&Signature=fphlHsEwpc3m~bQLaediSEUFQs7LafbIq5evpQD~-Z59yFxdhKKoCzaBF--cny9ierivB8bv1BEhwhAq8vQm2uj~QN1oIghEY3DSccV77Qthpm94UYIUGJu8SxFLpJX1S2KnFEJdksjRou68uQVy7lPDYGHLG5WueY43LmqqnJuS79-f4RR~oEHC4d2~vj3QCTLah~jBcDnP2V1mXCMfZ~O~9Y22TWeXZcvJ1z3H09NBKTmxnXRQAi-cxJqnFbuFzFDbGzM~1KuT1xT~0zoJd32mFAAZLnHCr-j36el25gFCH0ZD3xlLCFtatpApB-WBYJlyKfTerlp3y-NnVaXYZA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7362014&ar=484555&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02402.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Number of SNP markers identified by the GoldenGate<sup>®</sup> assay among 27 tomato and <em>S. pennellii</em> lines. The numbers show the loci of the 1338 SNPs genotyped in this study. The colors in each cell represent a continuum of polymorphic ratios: lower ratios are represented by green color, higher ratios by red color, and middle ratios by yellow color.</p></div></div><p>It is noteworthy that 136 SNPs (10.2%) were identified between ‘Micro-Tom_AM’ and ‘Micro-Tom_MM’, the parental lines of AMF2 and MMF2, respectively (Fig. <a reveal-id="DSQ024F2" data-open="DSQ024F2" class="link link-reveal link-table xref-fig">2</a>). Out of these 136 SNP loci, 134 mapped onto the AMF2 and/or the MMF2 maps, mainly on Chr04 (44 loci), Chr07 (38 loci), and Chr12 (36 loci) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). ‘Micro-Tom_AM’ had a higher number of polymorphisms, in comparison with the other 25 examined lines, than ‘Micro-Tom_MM’ (Fig. <a reveal-id="DSQ024F2" data-open="DSQ024F2" class="link link-reveal link-table xref-fig">2</a>). For example, 738 loci (55.2%) showed polymorphisms between ‘Micro-Tom_AM’ and ‘M82’, while only 640 SNPs (47.8%) were found between ‘Micro-Tom_MM’ and ‘M82’. It is likely that this difference resulted in an overestimation of the number of segregated loci between ‘Micro-Tom’ and ‘M82’ in the 82-eSNP PCR-based validation.</p>                            <h2 scrollto-destination=7362016 id="7362016" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>To our knowledge, the two genetic linkage maps presented here are the first intraspecific maps for <em>S. lycopersicum</em> with SNPs and other PCR-based co-dominant markers. The AMF2 and MMF2 genetic linkage maps comprise a total of 989 and 637 DNA marker loci, respectively, including SNP, SSR, and intron polymorphic and gene markers. Because the SNP markers developed in this study showed a higher degree of polymorphism among the tomato cultivars than SSR markers, SNP information is greatly important to be utilized for genetic analyses in cultivated tomato, including gene mapping, QTL analysis, population genetics, and marker-assisted breeding. In addition, the genomic tools developed in this study will be valuable for exploiting the extensive artificially induced genetic variability created by ethyl methane sulfonate (EMS) mutagenesis in ‘Micro-Tom’ mutant collections. For example, they could allow, by forward genetic approaches, the identification of the causal mutations for remarkable fruit and plant phenotypes.</p><p>In the SNP genotyping by the GoldenGate<sup>®</sup> assay, 1338 (87%) of the 1536 SNPs could be successfully genotyped. In other crops, successful ratio of SNP genotyping by the GoldenGate<sup>®</sup> assay is reported to be raging from 79 to 92%,<sup><a reveal-id="DSQ024C42  DSQ024C45" data-open="DSQ024C42  DSQ024C45" class="link link-ref link-reveal xref-bibr">42–45</a></sup> which fits to the result of the present study. In order to improve the ratio, we suggest additional three criteria to select SNPs for genotyping. The first is elimination of SNP positioned near junction site of intron and exon, because intron inhibits hybridization of allele- and locus-specific oligonucleotide to the target sequence based on EST. It can be achieved by comparing the sequences of EST with those of genome, if available. Next is avoidance designing SNP markers on multi-copy genes, which disrupts the fluorescent-signal clusters on the GenomeStudio Data Analysis software. Selection of accurate SNP site, e.g. with high-quality value and/or with highly coverage of sequence fragments, is also important. Large scale of genome analysis by massively parallel DNA sequencers would be convenient to overcome these matters.</p><p>In interspecific linkage maps of tomato and its relatives, markers derived from ESTs tend to distribute randomly along the genome, while markers derived from random genomic regions, e.g. RAPD, AFLP, and genomic SSRs, tend to form clusters in heterochromatic regions.<sup><a reveal-id="DSQ024C7" data-open="DSQ024C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSQ024C19  DSQ024C22" data-open="DSQ024C19  DSQ024C22" class="link link-ref link-reveal xref-bibr">19–22</a></sup> In this study, however, the marker loci did not disperse along the two linkage maps derived from AMF2 and MMF2, despite the fact that most markers were developed from ESTs. Comparison between the maps of Tomato-EXPEN 2000 and the two intraspecific mapping populations did not indicate any evidence of an obvious relationship between the marker clusters on the maps and chromosome structures, i.e. the heterochromatic and euchromatic regions (Fig. <a reveal-id="DSQ024F1" data-open="DSQ024F1" class="link link-reveal link-table xref-fig">1</a>). It can be assumed that the marker clusters correspond to probable integration regions originating from ‘<em>Lycopersicon minutum</em>’, an ancestral line of ‘Micro-Tom’,<sup><a reveal-id="DSQ024C29" data-open="DSQ024C29" class="link link-ref link-reveal xref-bibr">29</a></sup> which belongs to the <em>S. chmielewskii</em> and <em>S. neorickii</em> complex.<sup><a reveal-id="DSQ024C46" data-open="DSQ024C46" class="link link-ref link-reveal xref-bibr">46</a></sup> Such regions are expected to show higher frequencies of polymorphism than the other regions, which originate from cultivated lines.</p><p>Out of the 989 SNP markers on the AMF2 map, 489 were located on the two intraspecific maps generated in this study and 155 had already been located on the interspecific map generated from Tomato-EXPEN 2000.<sup><a reveal-id="DSQ024C7" data-open="DSQ024C7" class="link link-ref link-reveal xref-bibr">7</a></sup> The common markers for the three maps allow the alignment and connection of these maps as shown in Fig. <a reveal-id="DSQ024F1" data-open="DSQ024F1" class="link link-reveal link-table xref-fig">1</a>. Significant translocation and inversion of chromosome were not observed between intra- and interspecific maps, meaning the order of genes would be conserved in two species, and the genome sequence of tomato (<em>S. lycopersicum</em>) could be used as a reference genome of <em>S. pennellii</em>. In addition, it is likely that the AMF2 and MMF2 maps cover the whole tomato genome except for the middle part of Chr07 on the MMF2 map, and that the marker order is mostly conserved in the three maps. One possible approach to connecting the two linkage groups of Chr07 on the MMF2 map would be to develop a novel mapping population between ‘M82’ and ‘Micro-Tom_AM’ instead of ‘Micro-Tom_MM’ because SNPs on Chr07 segregated more frequently between ‘M82’ and ‘Micro-Tom_AM’.</p><p>Indeed, we found that 136 of the 1338 tested SNP markers (10.2%) showed polymorphisms between ‘Micro-Tom_AM’ and ‘Micro-Tom_MM’ (Fig. <a reveal-id="DSQ024F2" data-open="DSQ024F2" class="link link-reveal link-table xref-fig">2</a>), indicating possible residual heterosis in ‘Micro-Tom’. It is assumed that these loci had not been fixed at the time of ‘Micro-Tom’ being released, although ‘Micro-Tom’ seeds are propagated and distributed in the F<sub>12</sub> generation after a crossing.<sup><a reveal-id="DSQ024C29" data-open="DSQ024C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Theoretically, the heterozygosity of genome in the F<sub>12</sub> generation is calculated to be 0.05% [=(1/2)<sup>12-1</sup>] in self-pollinating plants, which means that most of the genomic regions are expected to be homozygous. Most of the polymorphic markers between the two ‘Micro-Tom’ lines were mapped on Chr04, Chr07, and Chr12. This result suggested that ‘Micro-Tom’ might have been bred under natural and/or artificial selection pressure from the regions under the influence of heterosis or crossing incompatibilities between <em>S. lycopersicum</em> and <em>L. minutum</em>. Alternatively, multiple lines might have been selected as ‘Micro-Tom’ from the breeding population before the complete fixation of the genotypes of each plant.</p><p>Though ‘Micro-Tom’ itself, bred as an ornamental plant, has little agricultural value, its genes may be of great value to agriculture. ‘Micro-Tom’ has resistance to several diseases, caused by <em>Alternaria alternata</em>, <em>Corynespora cassiicola</em>, <em>Fusarium oxysporum</em>, and <em>Pseudomonas syringae</em>.<sup><a reveal-id="DSQ024C47" data-open="DSQ024C47" class="link link-ref link-reveal xref-bibr">47</a></sup> Moreover, a large number of mutant lines have been developed using ‘Micro-Tom’.<sup><a reveal-id="DSQ024C31" data-open="DSQ024C31" class="link link-ref link-reveal xref-bibr">31</a>,<a reveal-id="DSQ024C32" data-open="DSQ024C32" class="link link-ref link-reveal xref-bibr">32</a></sup> The markers and maps developed in this study may therefore be useful for introgression breeding for disease resistance or targeted genes identified in ‘Micro-Tom’ or its mutant lines. Indeed, ‘Micro-Tom’ mutant lines carry mutated alleles that may confer high agricultural value to tomato, e.g. alleles causing large variations in fruit color, shape, size, and composition. Mutants may also help to decipher the mechanisms controlling specific traits in tomato.</p><p>In this study, we demonstrated the validity of the strategy of combining large-scale eSNP discovery with high-throughput SNP genotyping assays. Comparison of sequence data from tomato cultivars has been reported as an efficient strategy for developing a large number of SNP markers for tomato cultivars.<sup><a reveal-id="DSQ024C24" data-open="DSQ024C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSQ024C48" data-open="DSQ024C48" class="link link-ref link-reveal xref-bibr">48</a>,<a reveal-id="DSQ024C49" data-open="DSQ024C49" class="link link-ref link-reveal xref-bibr">49</a></sup> Today, extensive amounts of sequence data from crop genomes can be easily collected using massively parallel DNA sequencers.<sup><a reveal-id="DSQ024C50" data-open="DSQ024C50" class="link link-ref link-reveal xref-bibr">50</a></sup> In addition, genomic sequences from The International Tomato Genome Sequencing Consortium of SGN will soon become available.<sup><a reveal-id="DSQ024C51" data-open="DSQ024C51" class="link link-ref link-reveal xref-bibr">51</a></sup> The accumulating genome sequences can be used to develop custom SNP markers within cultivated tomato. The molecular markers and genetic linkage maps developed in the present study represent one of the initial milestones in the fusion of genomics, genetics, and molecular breeding in cultivated tomato.</p>                            <h2 scrollto-destination=7362025 id="7362025" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=7362027 id="7362027" class="section-title" data-legacy-id=s6>Availability</h2>
<p>Information on the SNP and SSR markers, the AMF2 and MMF2 linkage maps, and the SNP genotypes for the tomato lines investigated in the current study are available at <a class="link link-uri" href="http://www.kazusa.or.jp/tomato/">http://www.kazusa.or.jp/tomato/</a>.</p>                            <h2 scrollto-destination=7362029 id="7362029" class="section-title" data-legacy-id=s7>Funding</h2>
<p>This work was supported by the Kazusa DNA Research Institute Foundation and the Ministry of Agriculture, Forestry, and Fisheries of Japan with the cooperation of the Genomics for Agricultural Innovation Foundation (DD-4010).</p>                            <h2 scrollto-destination=7362031 id="7362031" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We are grateful to Dr K. Aoki (Kazusa DNA Research Institute, Japan) for providing the EST data for Micro-Tom. Plant materials were provided by Dr S. D. Tanksley (Cornell University, USA), Dr T. Ariizumi (University of Tsukuba through the National Bio-Resource Project of the Ministry of Education, Culture, Sports, Science and Technology, Japan), Dr T. Saito (National Institute of Vegetable and Tea Science, Japan), and Dr S. M. Tam (Tomato Genomic Resource Center, University of California, Davis, USA).</p>                            <h2 scrollto-destination=7362033 id="7362033" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSQ024C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Labate</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Grandillo</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Fulton</div> <div class="given-names">T.</div></div>, et al. </div><div class="person-group"><div class="name"><div class="surname">Cole</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Tomato</div>, <div class="source">Genome Mapping and Molecular Breeding in Plants, vol. 5</div> , <div class="year">2007</div><div class="publisher-loc">New York</div><div class="publisher-name">Springer</div>(pg. <div class="fpage">1</div>-<div class="lpage">125</div>)</div></div></div></div><div content-id="DSQ024C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Paterson</div> <div class="given-names">A.H.</div></div>,  <div class="name"><div class="surname">Lander</div> <div class="given-names">E.S.</div></div>,  <div class="name"><div class="surname">Hewitt</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Peterson</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Lincoln</div> <div class="given-names">S.E.</div></div>,  <div class="name"><div class="surname">Tanksley</div> <div class="given-names">S.D.</div></div>. </div><div class="article-title">Resolution of quantitative traits into Mendelian factors by using a complete linkage map of restriction fragment length polymorphisms</div>, <div class="source">Nature</div> , <div class="year">1988</div>, vol. <div class="volume">335</div> (pg. <div class="fpage">721</div>-<div class="lpage">726</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Resolution%20of%20quantitative%20traits%20into%20Mendelian%20factors%20by%20using%20a%20complete%20linkage%20map%20of%20restriction%20fragment%20length%20polymorphisms&amp;author=A.H.%20Paterson&amp;author=E.S.%20Lander&amp;author=J.D.%20Hewitt&amp;author=S.%20Peterson&amp;author=S.E.%20Lincoln&amp;author=S.D.%20Tanksley&amp;publication_year=1988&amp;journal=Nature&amp;volume=335&amp;pages=721-726" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/335721a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F335721a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F335721a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2902517" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tanksley</div> <div class="given-names">S.D.</div></div>,  <div class="name"><div class="surname">Ganal</div> <div class="given-names">M.W.</div></div>,  <div class="name"><div class="surname">Prince</div> <div class="given-names">J.P.</div></div>, et al. </div><div class="article-title">High density molecular linkage maps of the tomato and potato genomes</div>, <div class="source">Genetics</div> , <div class="year">1992</div>, vol. <div class="volume">132</div> (pg. <div class="fpage">1141</div>-<div class="lpage">1160</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High%20density%20molecular%20linkage%20maps%20of%20the%20tomato%20and%20potato%20genomes&amp;author=S.D.%20Tanksley&amp;author=M.W.%20Ganal&amp;author=J.P.%20Prince&amp;publication_year=1992&amp;journal=Genetics&amp;volume=132&amp;pages=1141-1160" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1360934" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">de Vicente</div> <div class="given-names">M.C.</div></div>,  <div class="name"><div class="surname">Tanksley</div> <div class="given-names">S.D.</div></div>. </div><div class="article-title">QTL analysis of transgressive segregation in an interspecific tomato cross</div>, <div class="source">Genetics</div> , <div class="year">1993</div>, vol. <div class="volume">134</div> (pg. <div class="fpage">585</div>-<div class="lpage">596</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=QTL%20analysis%20of%20transgressive%20segregation%20in%20an%20interspecific%20tomato%20cross&amp;author=M.C.%20de%20Vicente&amp;author=S.D.%20Tanksley&amp;publication_year=1993&amp;journal=Genetics&amp;volume=134&amp;pages=585-596" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8100788" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Paran</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Goldman</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Tanksley</div> <div class="given-names">S.D.</div></div>,  <div class="name"><div class="surname">Zamir</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Recombinant inbred lines for genetic mapping in tomato</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">1995</div>, vol. <div class="volume">90</div> (pg. <div class="fpage">542</div>-<div class="lpage">548</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recombinant%20inbred%20lines%20for%20genetic%20mapping%20in%20tomato&amp;author=I.%20Paran&amp;author=I.%20Goldman&amp;author=S.D.%20Tanksley&amp;author=D.%20Zamir&amp;publication_year=1995&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=90&amp;pages=542-548" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00222001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00222001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00222001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24173949" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fulton</div> <div class="given-names">T.M.</div></div>,  <div class="name"><div class="surname">Nelson</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Tanksley</div> <div class="given-names">S.D.</div></div>. </div><div class="article-title">Introgression and DNA marker analysis of <em>Lycopersicon peruvianum</em>, a wild relative of the cultivated tomato, into <em>Lycopersicon esculentum</em>, followed through three successive backcross generations</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">1997</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">895</div>-<div class="lpage">902</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Introgression%20and%20DNA%20marker%20analysis%20of%20Lycopersicon%20peruvianum%2C%20a%20wild%20relative%20of%20the%20cultivated%20tomato%2C%20into%20Lycopersicon%20esculentum%2C%20followed%20through%20three%20successive%20backcross%20generations&amp;author=T.M.%20Fulton&amp;author=J.C.%20Nelson&amp;author=S.D.%20Tanksley&amp;publication_year=1997&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=95&amp;pages=895-902" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220050640" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220050640" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220050640"> </span></div> </div></div></div></div></div><div content-id="DSQ024C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shirasawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Asamizu</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Fukuoka</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">An interspecific linkage map of SSR and intronic polymorphism markers in tomato</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">121</div> (pg. <div class="fpage">731</div>-<div class="lpage">739</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20interspecific%20linkage%20map%20of%20SSR%20and%20intronic%20polymorphism%20markers%20in%20tomato&amp;author=K.%20Shirasawa&amp;author=E.%20Asamizu&amp;author=H.%20Fukuoka&amp;publication_year=2010&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=121&amp;pages=731-739" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-010-1344-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-010-1344-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-010-1344-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20431859" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Frary</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Nesbitt</div> <div class="given-names">T.C.</div></div>,  <div class="name"><div class="surname">Grandillo</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title"><em>fw2.2</em>: a quantitative trait locus key to the evolution of tomato fruit size</div>, <div class="source">Science</div> , <div class="year">2000</div>, vol. <div class="volume">289</div> (pg. <div class="fpage">85</div>-<div class="lpage">88</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=fw2.2%3A%20a%20quantitative%20trait%20locus%20key%20to%20the%20evolution%20of%20tomato%20fruit%20size&amp;author=A.%20Frary&amp;author=T.C.%20Nesbitt&amp;author=S.%20Grandillo&amp;publication_year=2000&amp;journal=Science&amp;volume=289&amp;pages=85-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.289.5476.85" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.289.5476.85" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.289.5476.85"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10884229" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Van Eck</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Cong</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Tanksley</div> <div class="given-names">S.D.</div></div>. </div><div class="article-title">A new class of regulatory genes underlying the cause of pear-shaped tomato fruit</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2002</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">13302</div>-<div class="lpage">13306</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20class%20of%20regulatory%20genes%20underlying%20the%20cause%20of%20pear-shaped%20tomato%20fruit&amp;author=J.%20Liu&amp;author=J.%20Van%20Eck&amp;author=B.%20Cong&amp;author=S.D.%20Tanksley&amp;publication_year=2002&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=99&amp;pages=13302-13306" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.162485999" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.162485999" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.162485999"> </span></div> </div></div></div></div></div><div content-id="DSQ024C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cong</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Barrero</div> <div class="given-names">L.S.</div></div>,  <div class="name"><div class="surname">Tanksley</div> <div class="given-names">S.D.</div></div>. </div><div class="article-title">Regulatory change in YABBY-like transcription factor led to evolution of extreme fruit size during tomato domestication</div>, <div class="source">Nat. Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">800</div>-<div class="lpage">804</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulatory%20change%20in%20YABBY-like%20transcription%20factor%20led%20to%20evolution%20of%20extreme%20fruit%20size%20during%20tomato%20domestication&amp;author=B.%20Cong&amp;author=L.S.%20Barrero&amp;author=S.D.%20Tanksley&amp;publication_year=2008&amp;journal=Nat.%20Genet.&amp;volume=40&amp;pages=800-804" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.144" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.144" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.144"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18469814" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xiao</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Jiang</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Schaffner</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Stockinger</div> <div class="given-names">E.J.</div></div>,  <div class="name"><div class="surname">van der Knaap</div> <div class="given-names">E.</div></div>. </div><div class="article-title">A retrotransposon-mediated gene duplication underlies morphological variation of tomato fruit</div>, <div class="source">Science</div> , <div class="year">2008</div>, vol. <div class="volume">319</div> (pg. <div class="fpage">1527</div>-<div class="lpage">1530</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20retrotransposon-mediated%20gene%20duplication%20underlies%20morphological%20variation%20of%20tomato%20fruit&amp;author=H.%20Xiao&amp;author=N.%20Jiang&amp;author=E.%20Schaffner&amp;author=E.J.%20Stockinger&amp;author=E.%20van%20der%20Knaap&amp;publication_year=2008&amp;journal=Science&amp;volume=319&amp;pages=1527-1530" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1153040" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1153040" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1153040"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18339939" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dixon</div> <div class="given-names">M.S.</div></div>,  <div class="name"><div class="surname">Jones</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Keddie</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">Thomas</div> <div class="given-names">C.M.</div></div>,  <div class="name"><div class="surname">Harrison</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Jones</div> <div class="given-names">J.D.</div></div>. </div><div class="article-title">The tomato <em>Cf-2</em> disease resistance locus comprises two functional genes encoding leucine-rich repeat proteins</div>, <div class="source">Cell</div> , <div class="year">1996</div>, vol. <div class="volume">84</div> (pg. <div class="fpage">451</div>-<div class="lpage">459</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20tomato%20Cf-2%20disease%20resistance%20locus%20comprises%20two%20functional%20genes%20encoding%20leucine-rich%20repeat%20proteins&amp;author=M.S.%20Dixon&amp;author=D.A.%20Jones&amp;author=J.S.%20Keddie&amp;author=C.M.%20Thomas&amp;author=K.%20Harrison&amp;author=J.D.%20Jones&amp;publication_year=1996&amp;journal=Cell&amp;volume=84&amp;pages=451-459" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0092-8674(00)81290-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0092-8674(00)81290-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0092-8674(00)81290-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8608599" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Martin</div> <div class="given-names">G.B.</div></div>,  <div class="name"><div class="surname">Brommonschenkel</div> <div class="given-names">S.H.</div></div>,  <div class="name"><div class="surname">Chunwongse</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Map-based cloning of a protein kinase gene conferring disease resistance in tomato</div>, <div class="source">Science</div> , <div class="year">1993</div>, vol. <div class="volume">262</div> (pg. <div class="fpage">1432</div>-<div class="lpage">1436</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Map-based%20cloning%20of%20a%20protein%20kinase%20gene%20conferring%20disease%20resistance%20in%20tomato&amp;author=G.B.%20Martin&amp;author=S.H.%20Brommonschenkel&amp;author=J.%20Chunwongse&amp;publication_year=1993&amp;journal=Science&amp;volume=262&amp;pages=1432-1436" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.7902614" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.7902614" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.7902614"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7902614" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jones</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Thomas</div> <div class="given-names">C.M.</div></div>,  <div class="name"><div class="surname">Hammond-Kosack</div> <div class="given-names">K.E.</div></div>,  <div class="name"><div class="surname">Balint-Kurti</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Jones</div> <div class="given-names">J.D.</div></div>. </div><div class="article-title">Isolation of the tomato <em>Cf-9</em> gene for resistance to <em>Cladosporium fulvum</em> by transposon tagging</div>, <div class="source">Science</div> , <div class="year">1994</div>, vol. <div class="volume">266</div> (pg. <div class="fpage">789</div>-<div class="lpage">793</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20of%20the%20tomato%20Cf-9%20gene%20for%20resistance%20to%20Cladosporium%20fulvum%20by%20transposon%20tagging&amp;author=D.A.%20Jones&amp;author=C.M.%20Thomas&amp;author=K.E.%20Hammond-Kosack&amp;author=P.J.%20Balint-Kurti&amp;author=J.D.%20Jones&amp;publication_year=1994&amp;journal=Science&amp;volume=266&amp;pages=789-793" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.7973631" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.7973631" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.7973631"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7973631" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thomas</div> <div class="given-names">C.M.</div></div>,  <div class="name"><div class="surname">Jones</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Parniske</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Characterization of the tomato <em>Cf-4</em> gene for resistance to <em>Cladosporium fulvum</em> identifies sequences that determine recognitional specificity in Cf-4 and Cf-9</div>, <div class="source">Plant. Cell</div> , <div class="year">1997</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">2209</div>-<div class="lpage">2224</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20the%20tomato%20Cf-4%20gene%20for%20resistance%20to%20Cladosporium%20fulvum%20identifies%20sequences%20that%20determine%20recognitional%20specificity%20in%20Cf-4%20and%20Cf-9&amp;author=C.M.%20Thomas&amp;author=D.A.%20Jones&amp;author=M.%20Parniske&amp;publication_year=1997&amp;journal=Plant.%20Cell&amp;volume=9&amp;pages=2209-2224" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.9.12.2209" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.9.12.2209" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.9.12.2209"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9437864" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fridman</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Pleban</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Zamir</div> <div class="given-names">D.</div></div>. </div><div class="article-title">A recombination hotspot delimits a wild-species quantitative trait locus for tomato sugar content to 484 bp within an invertase gene</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2000</div>, vol. <div class="volume">97</div> (pg. <div class="fpage">4718</div>-<div class="lpage">4723</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20recombination%20hotspot%20delimits%20a%20wild-species%20quantitative%20trait%20locus%20for%20tomato%20sugar%20content%20to%20484%20bp%20within%20an%20invertase%20gene&amp;author=E.%20Fridman&amp;author=T.%20Pleban&amp;author=D.%20Zamir&amp;publication_year=2000&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=97&amp;pages=4718-4723" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.97.9.4718" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.97.9.4718" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.97.9.4718"> </span></div> </div></div></div></div></div><div content-id="DSQ024C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Miller</div> <div class="given-names">J C.</div></div>,  <div class="name"><div class="surname">Tanksley</div> <div class="given-names">S.D.</div></div>. </div><div class="article-title">RFLP analysis of phylogenetic relationships and genetic variation in the genus <em>Lycopersicon</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">1990</div>, vol. <div class="volume">80</div> (pg. <div class="fpage">437</div>-<div class="lpage">448</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RFLP%20analysis%20of%20phylogenetic%20relationships%20and%20genetic%20variation%20in%20the%20genus%20Lycopersicon&amp;author=J%20C.%20Miller&amp;author=S.D.%20Tanksley&amp;publication_year=1990&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=80&amp;pages=437-448" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24221000" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Saliba-Colombani</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Causse</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Gervais</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Philouze</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Efficiency of RFLP, RAPD, and AFLP markers for the construction of an intraspecific map of the tomato genome</div>, <div class="source">Genome</div> , <div class="year">2000</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">29</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efficiency%20of%20RFLP%2C%20RAPD%2C%20and%20AFLP%20markers%20for%20the%20construction%20of%20an%20intraspecific%20map%20of%20the%20tomato%20genome&amp;author=V.%20Saliba-Colombani&amp;author=M.%20Causse&amp;author=L.%20Gervais&amp;author=J.%20Philouze&amp;publication_year=2000&amp;journal=Genome&amp;volume=43&amp;pages=29-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g99-096" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg99-096" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg99-096"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10701110" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Frary</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Mitchell</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Tedeschi</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Tanksley</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Development of a set of PCR-based anchor markers encompassing the tomato genome and evaluation of their usefulness for genetics and breeding experiments</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">111</div> (pg. <div class="fpage">291</div>-<div class="lpage">312</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20a%20set%20of%20PCR-based%20anchor%20markers%20encompassing%20the%20tomato%20genome%20and%20evaluation%20of%20their%20usefulness%20for%20genetics%20and%20breeding%20experiments&amp;author=A.%20Frary&amp;author=Y.%20Xu&amp;author=J.%20Liu&amp;author=S.%20Mitchell&amp;author=E.%20Tedeschi&amp;author=S.%20Tanksley&amp;publication_year=2005&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=111&amp;pages=291-312" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-005-2023-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-005-2023-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-005-2023-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15926074" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohyama</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Asamizu</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Negoro</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Characterization of tomato SSR markers developed using BAC-end and cDNA sequences from genome databases</div>, <div class="source">Mol. Breed.</div> , <div class="year">2009</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">685</div>-<div class="lpage">691</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20tomato%20SSR%20markers%20developed%20using%20BAC-end%20and%20cDNA%20sequences%20from%20genome%20databases&amp;author=A.%20Ohyama&amp;author=E.%20Asamizu&amp;author=S.%20Negoro&amp;publication_year=2009&amp;journal=Mol.%20Breed.&amp;volume=23&amp;pages=685-691" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-009-9265-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-009-9265-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-009-9265-z"> </span></div> </div></div></div></div></div><div content-id="DSQ024C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Szinay</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Lang</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Cross-species bacterial artificial chromosome-fluorescence <em>in situ</em> hybridization painting of the tomato and potato chromosome 6 reveals undescribed chromosomal rearrangements</div>, <div class="source">Genetics</div> , <div class="year">2008</div>, vol. <div class="volume">180</div> (pg. <div class="fpage">1319</div>-<div class="lpage">1328</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cross-species%20bacterial%20artificial%20chromosome-fluorescence%20in%20situ%20hybridization%20painting%20of%20the%20tomato%20and%20potato%20chromosome%206%20reveals%20undescribed%20chromosomal%20rearrangements&amp;author=X.%20Tang&amp;author=D.%20Szinay&amp;author=C.%20Lang&amp;publication_year=2008&amp;journal=Genetics&amp;volume=180&amp;pages=1319-1328" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.108.093211" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.108.093211" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.108.093211"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18791231" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Tang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Cheng</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Mueller</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Giovannoni</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Tanksley</div> <div class="given-names">S.D.</div></div>. </div><div class="article-title">Euchromatin and pericentromeric heterochromatin: comparative composition in the tomato genome</div>, <div class="source">Genetics</div> , <div class="year">2006</div>, vol. <div class="volume">172</div> (pg. <div class="fpage">2529</div>-<div class="lpage">2540</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Euchromatin%20and%20pericentromeric%20heterochromatin%3A%20comparative%20composition%20in%20the%20tomato%20genome&amp;author=Y.%20Wang&amp;author=X.%20Tang&amp;author=Z.%20Cheng&amp;author=L.%20Mueller&amp;author=J.%20Giovannoni&amp;author=S.D.%20Tanksley&amp;publication_year=2006&amp;journal=Genetics&amp;volume=172&amp;pages=2529-2540" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.106.055772" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.106.055772" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.106.055772"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16489216" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gupta</div> <div class="given-names">P.K.</div></div>,  <div class="name"><div class="surname">Rustgi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Mir</div> <div class="given-names">R.R.</div></div>. </div><div class="article-title">Array-based high-throughput DNA markers for crop improvement</div>, <div class="source">Heredity</div> , <div class="year">2008</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">5</div>-<div class="lpage">18</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Array-based%20high-throughput%20DNA%20markers%20for%20crop%20improvement&amp;author=P.K.%20Gupta&amp;author=S.%20Rustgi&amp;author=R.R.%20Mir&amp;publication_year=2008&amp;journal=Heredity&amp;volume=101&amp;pages=5-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/hdy.2008.35" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fhdy.2008.35" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fhdy.2008.35"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18461083" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jimenez-Gomez</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Maloof</div> <div class="given-names">J.N.</div></div>. </div><div class="article-title">Sequence diversity in three tomato species: SNPs, markers, and molecular evolution</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">9</div> pg. <div class="fpage">85</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20diversity%20in%20three%20tomato%20species%3A%20SNPs%2C%20markers%2C%20and%20molecular%20evolution&amp;author=J.M.%20Jimenez-Gomez&amp;author=J.N.%20Maloof&amp;publication_year=2009&amp;journal=BMC%20Plant%20Biol.&amp;volume=9&amp;pages=85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-9-85" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-9-85" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-9-85"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19575805" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Labate</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Baldo</div> <div class="given-names">A.M.</div></div>. </div><div class="article-title">Tomato SNP discovery by EST mining and resequencing</div>, <div class="source">Mol. Breed.</div> , <div class="year">2005</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">343</div>-<div class="lpage">349</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tomato%20SNP%20discovery%20by%20EST%20mining%20and%20resequencing&amp;author=J.A.%20Labate&amp;author=A.M.%20Baldo&amp;publication_year=2005&amp;journal=Mol.%20Breed.&amp;volume=16&amp;pages=343-349" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-005-1911-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-005-1911-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-005-1911-5"> </span></div> </div></div></div></div></div><div content-id="DSQ024C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Bai</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Kabelka</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Discovery of single nucleotide polymorphisms in <em>Lycopersicon esculentum</em> by computer aided analysis of expressed sequence tags</div>, <div class="source">Mol. Breed.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">21</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Discovery%20of%20single%20nucleotide%20polymorphisms%20in%20Lycopersicon%20esculentum%20by%20computer%20aided%20analysis%20of%20expressed%20sequence%20tags&amp;author=W.%20Yang&amp;author=X.%20Bai&amp;author=E.%20Kabelka&amp;publication_year=2004&amp;journal=Mol.%20Breed.&amp;volume=14&amp;pages=21-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/B:MOLB.0000037992.03731.a5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FB:MOLB.0000037992.03731.a5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FB:MOLB.0000037992.03731.a5"> </span></div> </div></div></div></div></div><div content-id="DSQ024C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamamoto</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Tsugane</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Watanabe</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Expressed sequence tags from the laboratory-grown miniature tomato (<em>Lycopersicon esculentum</em>) cultivar Micro-Tom and mining for single nucleotide polymorphisms and insertions/deletions in tomato cultivars</div>, <div class="source">Gene</div> , <div class="year">2005</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">127</div>-<div class="lpage">134</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expressed%20sequence%20tags%20from%20the%20laboratory-grown%20miniature%20tomato%20(Lycopersicon%20esculentum)%20cultivar%20Micro-Tom%20and%20mining%20for%20single%20nucleotide%20polymorphisms%20and%20insertions%2Fdeletions%20in%20tomato%20cultivars&amp;author=N.%20Yamamoto&amp;author=T.%20Tsugane&amp;author=M.%20Watanabe&amp;publication_year=2005&amp;journal=Gene&amp;volume=15&amp;pages=127-134" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2005.04.026" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2005.04.026" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2005.04.026"> </span></div> </div></div></div></div></div><div content-id="DSQ024C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ezura</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Tomato is a next-generation model plant for research and development</div>, <div class="source">J. Jpn Soc. Hort. Sci.</div> , <div class="year">2009</div>, vol. <div class="volume">78</div> (pg. <div class="fpage">1</div>-<div class="lpage">2</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tomato%20is%20a%20next-generation%20model%20plant%20for%20research%20and%20development&amp;author=H.%20Ezura&amp;publication_year=2009&amp;journal=J.%20Jpn%20Soc.%20Hort.%20Sci.&amp;volume=78&amp;pages=1-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2503/jjshs1.78.1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2503%2Fjjshs1.78.1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2503%2Fjjshs1.78.1"> </span></div> </div></div></div></div></div><div content-id="DSQ024C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Scott</div> <div class="given-names">J.W.</div></div>,  <div class="name"><div class="surname">Harbaugh</div> <div class="given-names">B.K.</div></div>. </div><div class="article-title">Micro-Tom—a miniature dwarf tomato</div>, <div class="source">Florida Agr. Expt. Sta. Circ.</div> , <div class="year">1989</div>, vol. <div class="volume">370</div> (pg. <div class="fpage">1</div>-<div class="lpage">6</div>)</div></div></div></div><div content-id="DSQ024C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Meissner</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Jacobson</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Melamed</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">A new model system for tomato genetics</div>, <div class="source">Plant J.</div> , <div class="year">1997</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">1465</div>-<div class="lpage">1472</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20model%20system%20for%20tomato%20genetics&amp;author=R.%20Meissner&amp;author=Y.%20Jacobson&amp;author=S.%20Melamed&amp;publication_year=1997&amp;journal=Plant%20J.&amp;volume=12&amp;pages=1465-1472" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313x.1997.12061465.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313x.1997.12061465.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313x.1997.12061465.x"> </span></div> </div></div></div></div></div><div content-id="DSQ024C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Matsukura</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Yamaguchi</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Inamura</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Generation of gamma irradiation-induced mutant lines of the miniature tomato (<em>Solanum lycopersicum</em> L.) cultivar ‘Micro-Tom</div>, <div class="source">Plant Biotechnol.</div> , <div class="year">2007</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">39</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Generation%20of%20gamma%20irradiation-induced%20mutant%20lines%20of%20the%20miniature%20tomato%20(Solanum%20lycopersicum%20L.)%20cultivar%20%E2%80%98Micro-Tom&amp;author=C.%20Matsukura&amp;author=I.%20Yamaguchi&amp;author=M.%20Inamura&amp;publication_year=2007&amp;journal=Plant%20Biotechnol.&amp;volume=24&amp;pages=39-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.5511/plantbiotechnology.24.39" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.5511%2Fplantbiotechnology.24.39" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.5511%2Fplantbiotechnology.24.39"> </span></div> </div></div></div></div></div><div content-id="DSQ024C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Watanabe</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Mizoguchi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Aoki</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Ethylmethanesulfonate (EMS) mutagenesis of <em>Solanum lycopersicum</em> cv. Micro-Tom for large-scale mutant screens</div>, <div class="source">Plant Biotechnol.</div> , <div class="year">2007</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">33</div>-<div class="lpage">38</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ethylmethanesulfonate%20(EMS)%20mutagenesis%20of%20Solanum%20lycopersicum%20cv.%20Micro-Tom%20for%20large-scale%20mutant%20screens&amp;author=S.%20Watanabe&amp;author=T.%20Mizoguchi&amp;author=K.%20Aoki&amp;publication_year=2007&amp;journal=Plant%20Biotechnol.&amp;volume=24&amp;pages=33-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.5511/plantbiotechnology.24.33" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.5511%2Fplantbiotechnology.24.33" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.5511%2Fplantbiotechnology.24.33"> </span></div> </div></div></div></div></div><div content-id="DSQ024C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dan</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Yan</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Munyikwa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Dong</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Armstrong</div> <div class="given-names">C.L.</div></div>. </div><div class="article-title">MicroTom—a high-throughput model transformation system for functional genomics</div>, <div class="source">Plant Cell Rep.</div> , <div class="year">2006</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">432</div>-<div class="lpage">441</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MicroTom%E2%80%94a%20high-throughput%20model%20transformation%20system%20for%20functional%20genomics&amp;author=Y.%20Dan&amp;author=H.%20Yan&amp;author=T.%20Munyikwa&amp;author=J.%20Dong&amp;author=Y.%20Zhang&amp;author=C.L.%20Armstrong&amp;publication_year=2006&amp;journal=Plant%20Cell%20Rep.&amp;volume=25&amp;pages=432-441" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00299-005-0084-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00299-005-0084-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00299-005-0084-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16341726" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sun</div> <div class="given-names">H.J.</div></div>,  <div class="name"><div class="surname">Uchii</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Watanabe</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ezura</div> <div class="given-names">H.</div></div>. </div><div class="article-title">A highly efficient transformation protocol for Micro-Tom, a model cultivar for tomato functional genomics</div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2006</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">426</div>-<div class="lpage">431</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20highly%20efficient%20transformation%20protocol%20for%20Micro-Tom%2C%20a%20model%20cultivar%20for%20tomato%20functional%20genomics&amp;author=H.J.%20Sun&amp;author=S.%20Uchii&amp;author=S.%20Watanabe&amp;author=H.%20Ezura&amp;publication_year=2006&amp;journal=Plant%20Cell%20Physiol.&amp;volume=47&amp;pages=426-431" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pci251" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpci251" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpci251"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16381658" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Iijima</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Nakamura</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ogata</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Metabolite annotations based on the integration of mass spectral information</div>, <div class="source">Plant J.</div> , <div class="year">2008</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">949</div>-<div class="lpage">962</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metabolite%20annotations%20based%20on%20the%20integration%20of%20mass%20spectral%20information&amp;author=Y.%20Iijima&amp;author=Y.%20Nakamura&amp;author=Y.%20Ogata&amp;publication_year=2008&amp;journal=Plant%20J.&amp;volume=54&amp;pages=949-962" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-313X.2008.03434.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-313X.2008.03434.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-313X.2008.03434.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18266924" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Aoki</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yano</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Large-scale analysis of full-length cDNAs from the tomato (Solanum lycopersicum) cultivar Micro-Tom, a reference system for the Solanaceae genomics</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">210</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20analysis%20of%20full-length%20cDNAs%20from%20the%20tomato%20(Solanum%20lycopersicum)%20cultivar%20Micro-Tom%2C%20a%20reference%20system%20for%20the%20Solanaceae%20genomics&amp;author=K.%20Aoki&amp;author=K.%20Yano&amp;author=A.%20Suzuki&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=210" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-210" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-210" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-210"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20350329" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chevreux</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Pfisterer</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Drescher</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">Using the miraEST assembler for reliable and automated mRNA transcript assembly and SNP detection in sequenced ESTs</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">1147</div>-<div class="lpage">1159</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20the%20miraEST%20assembler%20for%20reliable%20and%20automated%20mRNA%20transcript%20assembly%20and%20SNP%20detection%20in%20sequenced%20ESTs&amp;author=B.%20Chevreux&amp;author=T.%20Pfisterer&amp;author=B.%20Drescher&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=1147-1159" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1917404" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1917404" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1917404"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15140833" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rozen</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Skaletsky</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Primer3 on the WWW for general users and for biologist programmers</div>, <div class="source">Methods Mol. Biol.</div> , <div class="year">2000</div>, vol. <div class="volume">132</div> (pg. <div class="fpage">365</div>-<div class="lpage">386</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Primer3%20on%20the%20WWW%20for%20general%20users%20and%20for%20biologist%20programmers&amp;author=S.%20Rozen&amp;author=H.%20Skaletsky&amp;publication_year=2000&amp;journal=Methods%20Mol.%20Biol.&amp;volume=132&amp;pages=365-386" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10547847" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Isobe</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Asamizu</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Comprehensive structural analysis of the genome of red clover (<em>Trifolium pratense</em> L.)</div>, <div class="source">DNA Res.</div> , <div class="year">2005</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">301</div>-<div class="lpage">364</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20structural%20analysis%20of%20the%20genome%20of%20red%20clover%20(Trifolium%20pratense%20L.)&amp;author=S.%20Sato&amp;author=S.%20Isobe&amp;author=E.%20Asamizu&amp;publication_year=2005&amp;journal=DNA%20Res.&amp;volume=12&amp;pages=301-364" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsi018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsi018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsi018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16769692" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Van Ooijen</div> <div class="given-names">J.W.</div></div>. </div>, <div class="source">JoinMap<sup>®</sup>4, Software for the Calculation of Genetic Linkage Maps in Experimental Populations</div> , <div class="year">2006</div><div class="publisher-loc">Wageningen, Netherlands</div><div class="publisher-name">Kyazma. BV</div></div></div></div></div><div content-id="DSQ024C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C41" class="ref-content "><div class="label -label">4!</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Marti</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Gisbert</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Bishop</div> <div class="given-names">G.J.</div></div>,  <div class="name"><div class="surname">Dixon</div> <div class="given-names">M.S.</div></div>,  <div class="name"><div class="surname">Garcia-Martinez</div> <div class="given-names">J.L.</div></div>. </div><div class="article-title">Genetic and physiological characterization of tomato cv. Micro-Tom</div>, <div class="source">J. Exp. Bot.</div> , <div class="year">2006</div>, vol. <div class="volume">57</div> (pg. <div class="fpage">2037</div>-<div class="lpage">2047</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20and%20physiological%20characterization%20of%20tomato%20cv.%20Micro-Tom&amp;author=E.%20Marti&amp;author=C.%20Gisbert&amp;author=G.J.%20Bishop&amp;author=M.S.%20Dixon&amp;author=J.L.%20Garcia-Martinez&amp;publication_year=2006&amp;journal=J.%20Exp.%20Bot.&amp;volume=57&amp;pages=2037-2047" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/erj154" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferj154" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferj154"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16687436" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hyten</div> <div class="given-names">D.L.</div></div>,  <div class="name"><div class="surname">Song</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Choi</div> <div class="given-names">I.-K.</div></div>, et al. </div><div class="article-title">High-throughput genotyping with the GoldenGate assay in the complex genome of soybean</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">116</div> (pg. <div class="fpage">945</div>-<div class="lpage">952</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20genotyping%20with%20the%20GoldenGate%20assay%20in%20the%20complex%20genome%20of%20soybean&amp;author=D.L.%20Hyten&amp;author=Q.%20Song&amp;author=I.-K.%20Choi&amp;publication_year=2008&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=116&amp;pages=945-952" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-008-0726-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-008-0726-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-008-0726-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18278477" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Muchero</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Diop</div> <div class="given-names">N.N.</div></div>,  <div class="name"><div class="surname">Bhat</div> <div class="given-names">P.R.</div></div>, et al. </div><div class="article-title">A consensus genetic map of cowpea [<em>Vigna unguiculata</em> (L) Walp.] and synteny based on EST-derived SNPs</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2009</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">18159</div>-<div class="lpage">18164</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20consensus%20genetic%20map%20of%20cowpea%20%5BVigna%20unguiculata%20(L)%20Walp.%5D%20and%20synteny%20based%20on%20EST-derived%20SNPs&amp;author=W.%20Muchero&amp;author=N.N.%20Diop&amp;author=P.R.%20Bhat&amp;publication_year=2009&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=106&amp;pages=18159-18164" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0905886106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0905886106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0905886106"> </span></div> </div></div></div></div></div><div content-id="DSQ024C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hyten</div> <div class="given-names">D.L.</div></div>,  <div class="name"><div class="surname">Song</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Fickus</div> <div class="given-names">E.W.</div></div>, et al. </div><div class="article-title">High-throughput SNP discovery and assay development in common bean</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">475</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20SNP%20discovery%20and%20assay%20development%20in%20common%20bean&amp;author=D.L.%20Hyten&amp;author=Q.%20Song&amp;author=E.W.%20Fickus&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=475" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-475" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-475" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-475"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20712881" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Deulvot</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Charrel</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Marty</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Highly-multiplexed SNP genotyping for genetic mapping and germplasm diversity studies in pea</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">468</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Highly-multiplexed%20SNP%20genotyping%20for%20genetic%20mapping%20and%20germplasm%20diversity%20studies%20in%20pea&amp;author=C.%20Deulvot&amp;author=H.%20Charrel&amp;author=A.%20Marty&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=468" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-468" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-468" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-468"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20701750" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rick</div> <div class="given-names">C.M.</div></div>,  <div class="name"><div class="surname">Kesicki</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Fobes</div> <div class="given-names">J.F.</div></div>,  <div class="name"><div class="surname">Holle</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Genetic and biosystematic studies on two new sibling species of <em>Lycopersicon</em> from interandean Peru</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">1976</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">55</div>-<div class="lpage">68</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20and%20biosystematic%20studies%20on%20two%20new%20sibling%20species%20of%20Lycopersicon%20from%20interandean%20Peru&amp;author=C.M.%20Rick&amp;author=E.%20Kesicki&amp;author=J.F.%20Fobes&amp;author=M.%20Holle&amp;publication_year=1976&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=47&amp;pages=55-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00281917" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00281917" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00281917"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24414383" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Takahashi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Shimizu</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Arie</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Catalog of Micro-Tom tomato responses to common fungal, bacterial, and viral pathogens</div>, <div class="source">J. Gen. Plant Pathol.</div> , <div class="year">2005</div>, vol. <div class="volume">71</div> (pg. <div class="fpage">8</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Catalog%20of%20Micro-Tom%20tomato%20responses%20to%20common%20fungal%2C%20bacterial%2C%20and%20viral%20pathogens&amp;author=H.%20Takahashi&amp;author=A.%20Shimizu&amp;author=T.%20Arie&amp;publication_year=2005&amp;journal=J.%20Gen.%20Plant%20Pathol.&amp;volume=71&amp;pages=8-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10327-004-0168-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10327-004-0168-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10327-004-0168-x"> </span></div> </div></div></div></div></div><div content-id="DSQ024C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Labate</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Robertson</div> <div class="given-names">L.D.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Tanksley</div> <div class="given-names">S.D.</div></div>,  <div class="name"><div class="surname">Baldo</div> <div class="given-names">A.M.</div></div>. </div><div class="article-title">EST, COSII, and arbitrary gene markers give similar estimates of nucleotide diversity in cultivated tomato (<em>Solanum lycopersicum</em> L.)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">118</div> (pg. <div class="fpage">1005</div>-<div class="lpage">1014</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EST%2C%20COSII%2C%20and%20arbitrary%20gene%20markers%20give%20similar%20estimates%20of%20nucleotide%20diversity%20in%20cultivated%20tomato%20(Solanum%20lycopersicum%20L.)&amp;author=J.A.%20Labate&amp;author=L.D.%20Robertson&amp;author=F.%20Wu&amp;author=S.D.%20Tanksley&amp;author=A.M.%20Baldo&amp;publication_year=2009&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=118&amp;pages=1005-1014" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-008-0957-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-008-0957-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-008-0957-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19153710" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Van Deynze</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Stoffel</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Buell</div> <div class="given-names">C.R.</div></div>, et al. </div><div class="article-title">Diversity in conserved genes in tomato</div>, <div class="source">BMC Genomics</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">465</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversity%20in%20conserved%20genes%20in%20tomato&amp;author=A.%20Van%20Deynze&amp;author=K.%20Stoffel&amp;author=C.R.%20Buell&amp;publication_year=2007&amp;journal=BMC%20Genomics&amp;volume=8&amp;pages=465" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-8-465" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-8-465" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-8-465"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18088428" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Hoisington</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Tyagi</div> <div class="given-names">A.K.</div></div>. </div><div class="article-title">Advances in cereal genomics and applications in crop breeding</div>, <div class="source">Trends. Biotechnol.</div> , <div class="year">2006</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">490</div>-<div class="lpage">499</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Advances%20in%20cereal%20genomics%20and%20applications%20in%20crop%20breeding&amp;author=R.K.%20Varshney&amp;author=D.A.%20Hoisington&amp;author=A.K.%20Tyagi&amp;publication_year=2006&amp;journal=Trends.%20Biotechnol.&amp;volume=24&amp;pages=490-499" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tibtech.2006.08.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tibtech.2006.08.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tibtech.2006.08.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16956681" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ024C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ024C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mueller</div> <div class="given-names">L.A.</div></div>,  <div class="name"><div class="surname">Lankhorst</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Tanksley</div> <div class="given-names">S.D.</div></div>, et al. </div><div class="article-title">A snapshot of the emerging tomato genome sequence</div>, <div class="source">Plant Genome</div> , <div class="year">2009</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">78</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20snapshot%20of%20the%20emerging%20tomato%20genome%20sequence&amp;author=L.A.%20Mueller&amp;author=R.K.%20Lankhorst&amp;author=S.D.%20Tanksley&amp;publication_year=2009&amp;journal=Plant%20Genome&amp;volume=2&amp;pages=78-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3835/plantgenome2008.08.0005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3835%2Fplantgenome2008.08.0005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3835%2Fplantgenome2008.08.0005"> </span></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2010. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=484555&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq024/2/dsq024-dsq024supp_table4.xls?Expires=2147483647&amp;Signature=Jl-mg-thS4IF6oUsl4OXAO6FWIcjpWgzDh00ykcTBkm3DaYD5f885l6BFWUQNoqziixHdtskPUN1ereFcQjbkNk5UxhQvfEQaAJX5h7NkMvJZ3ChWmaavu5AepPLZGKSACFquz3EvS-gNMyIjfb9MVsACoOnPO4QiBkYp76UvjdLO1SjjZED7CvswToZTJWpB3~EK1hE7~yvnMmPKS1lw5tiyxnIUQdEBDsA6OSGkgb5sl1YcyfMGjkWh-ahQHG0O1UhwRdNhJ0tbGvmRppVHETJfkk1nFszOv8OQSENzxjMYh4kjD406qIuQf2THUKljqqsCFRNnD~PJumiRL1b5g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq024/2/dsq024-dsq024supp_table3.xls?Expires=2147483647&amp;Signature=Jz7GcgAc8s3btByBn4BDo7c3w~N0waWgawUVH1PiW4HqGrqO6vnDn-7r53m4szxItnIP~8GjJ98vZhRJEBwvN~A8ARfZfqvKvvQhUp5rGHB6vUkhuRyLRoaLaIHRkwZeNf7YLoTIDV6CpcXJZHMeMxZP0gLNWs7Ca9y2eH81GF2DXKNQB-3hS6lY3f2dcBBuvuWhc3p~wTcK8GUpd0Q9tHZgUYXtq8MlTD6DWcuy6cbjJBVovbRvgCjBMfYFjo6ulSZyR1UL24RLfMygizSe8Yr3oq3fHsZh6dY58RzQLgZBpcbw2XFFQ5RWlSJnU8oYc-~gah6RsuUbY8599GQ4~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq024/2/dsq024-dsq024supp_table2.xls?Expires=2147483647&amp;Signature=FL1-IpSArE1vlZIpa2VYPXUi0mIJacpNDSRv4xp6fv~Y8mQADkwwcZOk3RxXcKdXLyQJ3XN6ZNpXvubMfT7M5cx96Is4-CoAVa~JNtmiRjaRzcsJkk77ILLmkcLw3A~RHvMn3G81NOOFHnsXe6OWGJQbiD9vVGn4CjluFR~vRdMH~KxPaRroK64pHWIK9OltTn075E50iSuPbopk9LIQYb6pBaAQK7V9lLDgLdQya~JlvEPI47F1T5dXWJ8N9CBjMXwSwl88JE3kIrPj421rSEhED7zaPCmjU2U-C8Oau3uizyM029LtDTHG4rn7b~5qj-COQWIuT0JRT9z-MYhXvg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq024/2/dsq024-dsq024supp_table1.xls?Expires=2147483647&amp;Signature=0hfkEh8~-pKxYWNognfQ-iWvDmKDsOyfWonqexS~96oElglLF7cn8Yt67JlB5vkDIUbvxq-CmKqtKDIggAiWTW7FYc-ziyoyWJrStfZcnBg0MEWO~FA4FR73wvR9~1k4Ysbdw2ytLPA4S6SfGurY5K7hpVmjMHhs-BWUZq3DESRw4a3reFP89W6rlqvl0gS-AjCpPBdK~Kma~-twNyjvrnrIkh8gdQbZobSI2ibVvNSzJ0t-4HFIfpOa~UHE6tHx9jmW7fQAjfAVAJX4cyBOPlcn7VpN0Lbo7HYcPMhmlsl621p~ojdpmuympNhrKXMVTv0Q6ZgAVeNvtTdskM4NGw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - xls file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">417</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000285800200005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">61</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsq024"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="484555"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000285800200005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f17%2f6%2f381%2f484555">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000285800200005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (61)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F17%2F6%2F381%2F484555" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/484555'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="SNP Discovery and Linkage Map Construction in Cultivated Tomato" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsq024");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f17%2f6%2f381%2f484555">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

