

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/21/4/341/444964","@type":"ScholarlyArticle","name":"Sma3s: A Three-Step Modular Annotator for Large Sequence Datasets","datePublished":"2014-02-05","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/21/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2014-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsu001","keywords":["functional annotation","genome annotation","transcriptome annotation","bioinformatic tool"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/21/4/341/444964","author":[{"name":"Muñoz-Mérida, Antonio","affiliation":"1  Integrated Bioinformatics, National Institute for Bioinformatics ,  University of Málaga ,  Campus de Teatinos ,  Spain","@type":"Person"},{"name":"Viguera, Enrique","affiliation":"2  Cellular Biology, Genetics and Physiology Department ,  University of Málaga ,  Campus de Teatinos ,  Spain","@type":"Person"},{"name":"Claros, M. Gonzalo","affiliation":"3  Molecular Biology and Biochemistry Department ,  University of Málaga ,  Campus de Teatinos ,  Spain","@type":"Person"},{"name":"Trelles, Oswaldo","affiliation":"1  Integrated Bioinformatics, National Institute for Bioinformatics ,  University of Málaga ,  Campus de Teatinos ,  Spain    4  Computer Architecture Department ,  University of Málaga ,  Campus de Teatinos ,  Spain","@type":"Person"},{"name":"Pérez-Pulido, Antonio J.","affiliation":"5  Centro Andaluz de Biología del Desarrollo (CABD, UPO-CSIC-JA), Facultad de Ciencias Experimentales (Área de Genética) ,  Universidad Pablo de Olavide ,  Sevilla 41013 ,  Spain","@type":"Person"}],"description":"Abstract.  Automatic sequence annotation is an essential component of modern ‘omics’ studies, which aim to extract information from large collections of sequenc","pageStart":"341","pageEnd":"353","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/4/1/m_cover.gif?Expires=2147483647&Signature=u1K0F1kMa-64NqKa4J2VQTk-ZgDNBNdy7xP9K68v94JN5WFnh6Of~dA8F~vuNuY-DWKv8sgKc6Tx8dCh~u-HxATm4iXS709rS-0HkL4VrV-yHYOvTEJMgBGMYLmkCZLHLnqJ4CE8~N9vWk3MlqGsd6CiX6Nq4~C6vim6BhKuTZhFWadSb~4gUcfc5RIhx0e6adpzN-8ic5w6tmizeNhHnsbot9fC0b5HPTtgY4T0sWivhNwSa56rOcmWTxrX6Ct6aeNXnMXGfCSv4nXY9sUZJB1uoWiHPNj1cxffLQO70T71y1MESr63u0ogFEZ8T3x1cUhck9QSbq-jL~iCjW~OcQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Sma3s: A Three-Step Modular Annotator for Large Sequence Datasets","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/4/1/m_cover.gif?Expires=2147483647&Signature=u1K0F1kMa-64NqKa4J2VQTk-ZgDNBNdy7xP9K68v94JN5WFnh6Of~dA8F~vuNuY-DWKv8sgKc6Tx8dCh~u-HxATm4iXS709rS-0HkL4VrV-yHYOvTEJMgBGMYLmkCZLHLnqJ4CE8~N9vWk3MlqGsd6CiX6Nq4~C6vim6BhKuTZhFWadSb~4gUcfc5RIhx0e6adpzN-8ic5w6tmizeNhHnsbot9fC0b5HPTtgY4T0sWivhNwSa56rOcmWTxrX6Ct6aeNXnMXGfCSv4nXY9sUZJB1uoWiHPNj1cxffLQO70T71y1MESr63u0ogFEZ8T3x1cUhck9QSbq-jL~iCjW~OcQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Sma3s: A Three-Step Modular Annotator for Large Sequence Datasets" />
    <meta property="og:description" content="Abstract.  Automatic sequence annotation is an essential component of modern ‘omics’ studies, which aim to extract information from large collections of sequenc" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/21/4/341/444964" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2014-02-05" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Muñoz-Mérida, Antonio" /><meta name="citation_author_institution" content="Integrated Bioinformatics, National Institute for Bioinformatics, University of Málaga, Campus de Teatinos, Spain" /><meta name="citation_author" content="Viguera, Enrique" /><meta name="citation_author_institution" content="Cellular Biology, Genetics and Physiology Department, University of Málaga, Campus de Teatinos, Spain" /><meta name="citation_author" content="Claros, M. Gonzalo" /><meta name="citation_author_institution" content="Molecular Biology and Biochemistry Department, University of Málaga, Campus de Teatinos, Spain" /><meta name="citation_author" content="Trelles, Oswaldo" /><meta name="citation_author_institution" content="Integrated Bioinformatics, National Institute for Bioinformatics, University of Málaga, Campus de Teatinos, Spain" /><meta name="citation_author_institution" content="Computer Architecture Department, University of Málaga, Campus de Teatinos, Spain" /><meta name="citation_author" content="Pérez-Pulido, Antonio J." /><meta name="citation_author_institution" content="Centro Andaluz de Biología del Desarrollo (CABD, UPO-CSIC-JA), Facultad de Ciencias Experimentales (Área de Genética), Universidad Pablo de Olavide, Sevilla 41013, Spain" /><meta name="citation_title" content="Sma3s: A Three-Step Modular Annotator for Large Sequence Datasets" /><meta name="citation_firstpage" content="341" /><meta name="citation_lastpage" content="353" /><meta name="citation_doi" content="10.1093/dnares/dsu001" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="21" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2014/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Automatic prediction of protein function; Cell. Mol. Life Sci.;  citation_year=2003;  citation_volume=60;  citation_pages=2637-50; " /><meta name="citation_reference" content="citation_title=Protein function annotation by homology-based inference; Genome Biol.;  citation_year=2009;  citation_volume=10;  citation_pages=207" /><meta name="citation_reference" content="citation_title=Automatic annotation of protein function based on family identification; Proteins;  citation_year=2003;  citation_volume=53;  citation_pages=683-92; " /><meta name="citation_reference" content="citation_title=Sequence-based feature prediction and annotation of proteins; Genome Biol.;  citation_year=2009;  citation_volume=10;  citation_pages=206" /><meta name="citation_reference" content="citation_title=EDITtoTrEMBL: a distributed approach to high-quality automated protein sequence annotation; Bioinformatics;  citation_year=1999;  citation_volume=15;  citation_pages=219-27; " /><meta name="citation_reference" content="citation_title=GENECODIS: a web-based tool for finding significant concurrent annotations in gene lists; Genome Biol.;  citation_year=2007;  citation_volume=8;  citation_pages=R3" /><meta name="citation_reference" content="citation_title=A biclustering algorithm for extracting bit-patterns from binary datasets; Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=2738-45; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-402; " /><meta name="citation_reference" content="citation_title=Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research; Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=3674-6; " /><meta name="citation_reference" content="citation_title=Gene ontology: tool for the unification of biology, The Gene Ontology Consortium; Nat. Genet.;  citation_year=2000;  citation_volume=25;  citation_pages=25-9; " /><meta name="citation_reference" content="citation_title=AutoFACT: an automatic functional annotation and classification tool; BMC Bioinformatics;  citation_year=2005;  citation_volume=6;  citation_pages=151" /><meta name="citation_reference" content="citation_title=GOtcha: a new method for prediction of protein function assessed by the annotation of seven genomes; BMC Bioinformatics;  citation_year=2004;  citation_volume=5;  citation_pages=178" /><meta name="citation_reference" content="citation_title=ESTAnnotator: a tool for high throughput EST annotation; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=3716-9; " /><meta name="citation_reference" content="citation_title=EST-PAC a web package for EST annotation and protein sequence prediction; Source Code Biol. Med.;  citation_year=2006;  citation_volume=1;  citation_pages=2" /><meta name="citation_reference" content="citation_title=BLANNOTATOR: enhanced homology-based function prediction of bacterial proteins; BMC Bioinformatics;  citation_year=2012;  citation_volume=13;  citation_pages=33" /><meta name="citation_reference" content="citation_title=Babelomics: an integrative platform for the analysis of transcriptomics, proteomics and genomic data with advanced functional profiling; Nucleic Acids Res.;  citation_year=2010;  citation_volume=38;  citation_pages=W210-213; " /><meta name="citation_reference" content="citation_title=Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources; Nat. Protoc.;  citation_year=2009;  citation_volume=4;  citation_pages=44-57; " /><meta name="citation_reference" content="citation_title=BioMart: driving a paradigm change in biological data management; Database (Oxford);  citation_year=2011;  citation_volume=2011;  citation_pages=bar049" /><meta name="citation_reference" content="citation_title=How well is enzyme function conserved as a function of pairwise sequence identity?; J. Mol. Biol.;  citation_year=2003;  citation_volume=333;  citation_pages=863-82; " /><meta name="citation_reference" content="citation_title=The Universal Protein Resource (UniProt); citation_author=The UniProt Consortium; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=D193-7; " /><meta name="citation_reference" content="citation_title=New developments in the InterPro database; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=D224-8; " /><meta name="citation_reference" content="citation_title=Database of homology-derived protein structures and the structural meaning of sequence alignment; Proteins;  citation_year=1991;  citation_volume=9;  citation_pages=56-68; " /><meta name="citation_reference" content="citation_title=Twilight zone of protein sequence alignments; Protein Eng.;  citation_year=1999;  citation_volume=12;  citation_pages=85-94; " /><meta name="citation_reference" content="citation_title=Assessing annotation transfer for genomics: quantifying the relations between protein sequence, structure and function through traditional and probabilistic scores; J. Mol. Biol.;  citation_year=2000;  citation_volume=297;  citation_pages=233-49; " /><meta name="citation_reference" content="citation_title=Discovering biological connections between experimental conditions based on common patterns of differential gene expression; BMC Bioinformatics;  citation_year=2011;  citation_volume=12;  citation_pages=381" /><meta name="citation_reference" content="citation_title=Transcriptome analysis of the venom glands of the Chinese wolf spider Lycosa singoriensis; Zoology (Jena);  citation_year=2010;  citation_volume=113;  citation_pages=10-8; " /><meta name="citation_reference" content="citation_title=Effects of the Escherichia coli sfsA gene on mal genes expression and a DNA binding activity of SfsA; Biosci. Biotechnol. Biochem.;  citation_year=2001;  citation_volume=65;  citation_pages=213-7; " /><meta name="citation_reference" content="citation_title=MOWServ: a web client for integration of bioinformatic resources; Nucleic Acids Res.;  citation_year=2010;  citation_volume=38;  citation_pages=W671-676; " /><meta name="citation_reference" content="citation_title=De novo assembly and functional annotation of the olive (Olea europaea) transcriptome; DNA Res.;  citation_year=2013;  citation_volume=20;  citation_pages=93-108; " /><meta name="citation_reference" content="citation_title=De novo assembly of maritime pine transcriptome: implications for forest breeding and biotechnology; citation_text=[Epub ahead of print] PMID: 24256179; Plant Biotechnol. J.;  citation_year=2013; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/21/4/341/1217453/dsu001.pdf" /><meta name="description" content="Abstract.  Automatic sequence annotation is an essential component of modern ‘omics’ studies, which aim to extract information from large collections of sequenc" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/21/4/341/444964" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/21/4/341/444964" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/21/4/341/444964");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsu001';
    </script>
<title>
	Sma3s: A Three-Step Modular Annotator for Large Sequence Datasets | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./444964" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BaZHPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNdcOxb3otTcOpcmlkYSwgQW50b25pbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkludGVncmF0ZWQgQmlvaW5mb3JtYXRpY3MsIE5hdGlvbmFsIEluc3RpdHV0ZSBmb3IgQmlvaW5mb3JtYXRpY3MsIFVuaXZlcnNpdHkgb2YgTcOhbGFnYSwgQ2FtcHVzIGRlIFRlYXRpbm9zLCBTcGFpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlZpZ3VlcmEsIEVucmlxdWUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDZWxsdWxhciBCaW9sb2d5LCBHZW5ldGljcyBhbmQgUGh5c2lvbG9neSBEZXBhcnRtZW50LCBVbml2ZXJzaXR5IG9mIE3DoWxhZ2EsIENhbXB1cyBkZSBUZWF0aW5vcywgU3BhaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDbGFyb3MsIE0uIEdvbnphbG8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNb2xlY3VsYXIgQmlvbG9neSBhbmQgQmlvY2hlbWlzdHJ5IERlcGFydG1lbnQsIFVuaXZlcnNpdHkgb2YgTcOhbGFnYSwgQ2FtcHVzIGRlIFRlYXRpbm9zLCBTcGFpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRyZWxsZXMsIE9zd2FsZG8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnRlZ3JhdGVkIEJpb2luZm9ybWF0aWNzLCBOYXRpb25hbCBJbnN0aXR1dGUgZm9yIEJpb2luZm9ybWF0aWNzLCBVbml2ZXJzaXR5IG9mIE3DoWxhZ2EsIENhbXB1cyBkZSBUZWF0aW5vcywgU3BhaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDb21wdXRlciBBcmNoaXRlY3R1cmUgRGVwYXJ0bWVudCwgVW5pdmVyc2l0eSBvZiBNw6FsYWdhLCBDYW1wdXMgZGUgVGVhdGlub3MsIFNwYWluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUMOpcmV6LVB1bGlkbywgQW50b25pbyBKLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNlbnRybyBBbmRhbHV6IGRlIEJpb2xvZ8OtYSBkZWwgRGVzYXJyb2xsbyAoQ0FCRCwgVVBPLUNTSUMtSkEpLCBGYWN1bHRhZCBkZSBDaWVuY2lhcyBFeHBlcmltZW50YWxlcyAow4FyZWEgZGUgR2Vuw6l0aWNhKSwgVW5pdmVyc2lkYWQgUGFibG8gZGUgT2xhdmlkZSwgU2V2aWxsYSA0MTAxMywgU3BhaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IlNtYTNzOiBBIFRocmVlLVN0ZXAgTW9kdWxhciBBbm5vdGF0b3IgZm9yIExhcmdlIFNlcXVlbmNlIERhdGFzZXRzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMzQxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIzNTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3UwMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNC8wOC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUF1dG9tYXRpYyBwcmVkaWN0aW9uIG9mIHByb3RlaW4gZnVuY3Rpb247IENlbGwuIE1vbC4gTGlmZSBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTYwOyAgY2l0YXRpb25fcGFnZXM9MjYzNy01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvdGVpbiBmdW5jdGlvbiBhbm5vdGF0aW9uIGJ5IGhvbW9sb2d5LWJhc2VkIGluZmVyZW5jZTsgR2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MjA3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXV0b21hdGljIGFubm90YXRpb24gb2YgcHJvdGVpbiBmdW5jdGlvbiBiYXNlZCBvbiBmYW1pbHkgaWRlbnRpZmljYXRpb247IFByb3RlaW5zOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTUzOyAgY2l0YXRpb25fcGFnZXM9NjgzLTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jZS1iYXNlZCBmZWF0dXJlIHByZWRpY3Rpb24gYW5kIGFubm90YXRpb24gb2YgcHJvdGVpbnM7IEdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTIwNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVESVR0b1RyRU1CTDogYSBkaXN0cmlidXRlZCBhcHByb2FjaCB0byBoaWdoLXF1YWxpdHkgYXV0b21hdGVkIHByb3RlaW4gc2VxdWVuY2UgYW5ub3RhdGlvbjsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0yMTktMjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdFTkVDT0RJUzogYSB3ZWItYmFzZWQgdG9vbCBmb3IgZmluZGluZyBzaWduaWZpY2FudCBjb25jdXJyZW50IGFubm90YXRpb25zIGluIGdlbmUgbGlzdHM7IEdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9UjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGJpY2x1c3RlcmluZyBhbGdvcml0aG0gZm9yIGV4dHJhY3RpbmcgYml0LXBhdHRlcm5zIGZyb20gYmluYXJ5IGRhdGFzZXRzOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTI3MzgtNDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdhcHBlZCBCTEFTVCBhbmQgUFNJLUJMQVNUOiBhIG5ldyBnZW5lcmF0aW9uIG9mIHByb3RlaW4gZGF0YWJhc2Ugc2VhcmNoIHByb2dyYW1zOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0zMzg5LTQwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Qmxhc3QyR086IGEgdW5pdmVyc2FsIHRvb2wgZm9yIGFubm90YXRpb24sIHZpc3VhbGl6YXRpb24gYW5kIGFuYWx5c2lzIGluIGZ1bmN0aW9uYWwgZ2Vub21pY3MgcmVzZWFyY2g7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MzY3NC02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIG9udG9sb2d5OiB0b29sIGZvciB0aGUgdW5pZmljYXRpb24gb2YgYmlvbG9neSwgVGhlIEdlbmUgT250b2xvZ3kgQ29uc29ydGl1bTsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0yNS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BdXRvRkFDVDogYW4gYXV0b21hdGljIGZ1bmN0aW9uYWwgYW5ub3RhdGlvbiBhbmQgY2xhc3NpZmljYXRpb24gdG9vbDsgQk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0xNTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HT3RjaGE6IGEgbmV3IG1ldGhvZCBmb3IgcHJlZGljdGlvbiBvZiBwcm90ZWluIGZ1bmN0aW9uIGFzc2Vzc2VkIGJ5IHRoZSBhbm5vdGF0aW9uIG9mIHNldmVuIGdlbm9tZXM7IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MTc4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RVNUQW5ub3RhdG9yOiBhIHRvb2wgZm9yIGhpZ2ggdGhyb3VnaHB1dCBFU1QgYW5ub3RhdGlvbjsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MzcxNi05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FU1QtUEFDIGEgd2ViIHBhY2thZ2UgZm9yIEVTVCBhbm5vdGF0aW9uIGFuZCBwcm90ZWluIHNlcXVlbmNlIHByZWRpY3Rpb247IFNvdXJjZSBDb2RlIEJpb2wuIE1lZC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CTEFOTk9UQVRPUjogZW5oYW5jZWQgaG9tb2xvZ3ktYmFzZWQgZnVuY3Rpb24gcHJlZGljdGlvbiBvZiBiYWN0ZXJpYWwgcHJvdGVpbnM7IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTMzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFiZWxvbWljczogYW4gaW50ZWdyYXRpdmUgcGxhdGZvcm0gZm9yIHRoZSBhbmFseXNpcyBvZiB0cmFuc2NyaXB0b21pY3MsIHByb3Rlb21pY3MgYW5kIGdlbm9taWMgZGF0YSB3aXRoIGFkdmFuY2VkIGZ1bmN0aW9uYWwgcHJvZmlsaW5nOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz1XMjEwLTIxMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3lzdGVtYXRpYyBhbmQgaW50ZWdyYXRpdmUgYW5hbHlzaXMgb2YgbGFyZ2UgZ2VuZSBsaXN0cyB1c2luZyBEQVZJRCBiaW9pbmZvcm1hdGljcyByZXNvdXJjZXM7IE5hdC4gUHJvdG9jLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9NDQtNTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJpb01hcnQ6IGRyaXZpbmcgYSBwYXJhZGlnbSBjaGFuZ2UgaW4gYmlvbG9naWNhbCBkYXRhIG1hbmFnZW1lbnQ7IERhdGFiYXNlIChPeGZvcmQpOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTIwMTE7ICBjaXRhdGlvbl9wYWdlcz1iYXIwNDkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ib3cgd2VsbCBpcyBlbnp5bWUgZnVuY3Rpb24gY29uc2VydmVkIGFzIGEgZnVuY3Rpb24gb2YgcGFpcndpc2Ugc2VxdWVuY2UgaWRlbnRpdHk/OyBKLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMzMzsgIGNpdGF0aW9uX3BhZ2VzPTg2My04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFVuaXZlcnNhbCBQcm90ZWluIFJlc291cmNlIChVbmlQcm90KTsgY2l0YXRpb25fYXV0aG9yPVRoZSBVbmlQcm90IENvbnNvcnRpdW07IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPUQxOTMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TmV3IGRldmVsb3BtZW50cyBpbiB0aGUgSW50ZXJQcm8gZGF0YWJhc2U7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPUQyMjQtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGF0YWJhc2Ugb2YgaG9tb2xvZ3ktZGVyaXZlZCBwcm90ZWluIHN0cnVjdHVyZXMgYW5kIHRoZSBzdHJ1Y3R1cmFsIG1lYW5pbmcgb2Ygc2VxdWVuY2UgYWxpZ25tZW50OyBQcm90ZWluczsgIGNpdGF0aW9uX3llYXI9MTk5MTsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9NTYtNjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVR3aWxpZ2h0IHpvbmUgb2YgcHJvdGVpbiBzZXF1ZW5jZSBhbGlnbm1lbnRzOyBQcm90ZWluIEVuZy47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz04NS05NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNzZXNzaW5nIGFubm90YXRpb24gdHJhbnNmZXIgZm9yIGdlbm9taWNzOiBxdWFudGlmeWluZyB0aGUgcmVsYXRpb25zIGJldHdlZW4gcHJvdGVpbiBzZXF1ZW5jZSwgc3RydWN0dXJlIGFuZCBmdW5jdGlvbiB0aHJvdWdoIHRyYWRpdGlvbmFsIGFuZCBwcm9iYWJpbGlzdGljIHNjb3JlczsgSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yOTc7ICBjaXRhdGlvbl9wYWdlcz0yMzMtNDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc2NvdmVyaW5nIGJpb2xvZ2ljYWwgY29ubmVjdGlvbnMgYmV0d2VlbiBleHBlcmltZW50YWwgY29uZGl0aW9ucyBiYXNlZCBvbiBjb21tb24gcGF0dGVybnMgb2YgZGlmZmVyZW50aWFsIGdlbmUgZXhwcmVzc2lvbjsgQk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MzgxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdG9tZSBhbmFseXNpcyBvZiB0aGUgdmVub20gZ2xhbmRzIG9mIHRoZSBDaGluZXNlIHdvbGYgc3BpZGVyIEx5Y29zYSBzaW5nb3JpZW5zaXM7IFpvb2xvZ3kgKEplbmEpOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExMzsgIGNpdGF0aW9uX3BhZ2VzPTEwLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVmZmVjdHMgb2YgdGhlIEVzY2hlcmljaGlhIGNvbGkgc2ZzQSBnZW5lIG9uIG1hbCBnZW5lcyBleHByZXNzaW9uIGFuZCBhIEROQSBiaW5kaW5nIGFjdGl2aXR5IG9mIFNmc0E7IEJpb3NjaS4gQmlvdGVjaG5vbC4gQmlvY2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9NjU7ICBjaXRhdGlvbl9wYWdlcz0yMTMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TU9XU2VydjogYSB3ZWIgY2xpZW50IGZvciBpbnRlZ3JhdGlvbiBvZiBiaW9pbmZvcm1hdGljIHJlc291cmNlczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTM4OyAgY2l0YXRpb25fcGFnZXM9VzY3MS02NzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlIG5vdm8gYXNzZW1ibHkgYW5kIGZ1bmN0aW9uYWwgYW5ub3RhdGlvbiBvZiB0aGUgb2xpdmUgKE9sZWEgZXVyb3BhZWEpIHRyYW5zY3JpcHRvbWU7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9OTMtMTA4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZSBub3ZvIGFzc2VtYmx5IG9mIG1hcml0aW1lIHBpbmUgdHJhbnNjcmlwdG9tZTogaW1wbGljYXRpb25zIGZvciBmb3Jlc3QgYnJlZWRpbmcgYW5kIGJpb3RlY2hub2xvZ3k7IGNpdGF0aW9uX3RleHQ9W0VwdWIgYWhlYWQgb2YgcHJpbnRdIFBNSUQ6IDI0MjU2MTc5OyBQbGFudCBCaW90ZWNobm9sLiBKLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIxLzQvMzQxLzEyMTc0NTMvZHN1MDAxLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBBdXRvbWF0aWMgc2VxdWVuY2UgYW5ub3RhdGlvbiBpcyBhbiBlc3NlbnRpYWwgY29tcG9uZW50IG9mIG1vZGVybiDigJhvbWljc+KAmSBzdHVkaWVzLCB3aGljaCBhaW0gdG8gZXh0cmFjdCBpbmZvcm1hdGlvbiBmcm9tIGxhcmdlIGNvbGxlY3Rpb25zIG9mIHNlcXVlbmMiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQ0NDk2NCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGQxSJFvTng+p1FGHKCNJLKcHfzXYA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f4%2f341%2f444964" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f4%2f341%2f444964" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=17419" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/21/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/4/1/m_cover.gif?Expires=2147483647&amp;Signature=u1K0F1kMa-64NqKa4J2VQTk-ZgDNBNdy7xP9K68v94JN5WFnh6Of~dA8F~vuNuY-DWKv8sgKc6Tx8dCh~u-HxATm4iXS709rS-0HkL4VrV-yHYOvTEJMgBGMYLmkCZLHLnqJ4CE8~N9vWk3MlqGsd6CiX6Nq4~C6vim6BhKuTZhFWadSb~4gUcfc5RIhx0e6adpzN-8ic5w6tmizeNhHnsbot9fC0b5HPTtgY4T0sWivhNwSa56rOcmWTxrX6Ct6aeNXnMXGfCSv4nXY9sUZJB1uoWiHPNj1cxffLQO70T71y1MESr63u0ogFEZ8T3x1cUhck9QSbq-jL~iCjW~OcQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/21/4">
                        <div class="volume trailing-comma">Volume 21</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2014        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6651732">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6651732">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6651734">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6651734">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6651743">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6651743">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6651778">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6651778">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6651815">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6651815">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6651833">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6651833">Authors' contributions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6651835">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6651835">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6651837">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6651837">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6651839">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6651839">Acknowledgement</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6651841">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6651841">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/21/4/355/445136">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Sma3s: A Three-Step Modular Annotator for Large Sequence Datasets
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Antonio Mu&#241;oz-M&#233;rida</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Integrated Bioinformatics, National Institute for Bioinformatics</div>, <div class="insititution">University of Málaga</div>, <div class="addr-line">Campus de Teatinos</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Antonio+Mu%c3%b1oz-M%c3%a9rida">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mu&#241;oz-M&#233;rida A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mu&#241;oz-M&#233;rida A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Antonio Muñoz-Mérida</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Enrique Viguera</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Cellular Biology, Genetics and Physiology Department</div>, <div class="insititution">University of Málaga</div>, <div class="addr-line">Campus de Teatinos</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Enrique+Viguera">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Viguera E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Viguera E&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Enrique Viguera</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">M. Gonzalo Claros</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Molecular Biology and Biochemistry Department</div>, <div class="insititution">University of Málaga</div>, <div class="addr-line">Campus de Teatinos</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=M.+Gonzalo+Claros">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Claros M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Claros M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">M. Gonzalo Claros</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Oswaldo Trelles</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Integrated Bioinformatics, National Institute for Bioinformatics</div>, <div class="insititution">University of Málaga</div>, <div class="addr-line">Campus de Teatinos</div>, <div class="country">Spain</div></div><div class="aff"><div class="label -label">4</div><div class="addr-line">Computer Architecture Department</div>, <div class="insititution">University of Málaga</div>, <div class="addr-line">Campus de Teatinos</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Oswaldo+Trelles">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Trelles O">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Trelles O&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Oswaldo Trelles</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Antonio J. P&#233;rez-Pulido</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">Centro Andaluz de Biología del Desarrollo (CABD, UPO-CSIC-JA), Facultad de Ciencias Experimentales (Área de Genética)</div>, <div class="insititution">Universidad Pablo de Olavide</div>, <div class="addr-line">Sevilla 41013</div>, <div class="country">Spain</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +34 954-348-652. Fax. +34 954-349-376. E-mail: <a href="mailto:ajperez@upo.es" target="_blank">ajperez@upo.es</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Antonio+J.+P%c3%a9rez-Pulido">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=P&#233;rez-Pulido A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;P&#233;rez-Pulido A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Antonio J. Pérez-Pulido</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="fn1"><div class="fn"><div class="footnote-content">Edited by Prof. Kenta Nakai</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 21, Issue 4, 1 August 2014, Pages 341–353, <a href='https://doi.org/10.1093/dnares/dsu001'>https://doi.org/10.1093/dnares/dsu001</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">05 February 2014</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">29 October 2013</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">06 January 2014</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="444964" href="/dnaresearch/article-pdf/21/4/341/1217453/dsu001.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Antonio Muñoz-Mérida, Enrique Viguera, M. Gonzalo Claros, Oswaldo Trelles, Antonio J. Pérez-Pulido;  Sma3s: A Three-Step Modular Annotator for Large Sequence Datasets, <em>DNA Research</em>, Volume 21, Issue 4, 1 August 2014, Pages 341–353, <a href='https://doi.org/10.1093/dnares/dsu001'>https://doi.org/10.1093/dnares/dsu001</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/444964?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/444964?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/444964?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/444964?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/444964?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/444964?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/444964?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Sma3s%3A%20A%20Three-Step%20Modular%20Annotator%20for%20Large%20Sequence%20Datasets&amp;publicationDate=2014-02-05&amp;volumeNum=21&amp;issueNum=4&amp;author=Mu%C3%B1oz-M%C3%A9rida%2C%20Antonio%3B%20Viguera%2C%20Enrique&amp;startPage=341&amp;endPage=353&amp;contentId=10.1093%2Fdnares%2Fdsu001&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f4%2f341%2f444964" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6651732 id="6651732" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Automatic sequence annotation is an essential component of modern ‘omics’ studies, which aim to extract information from large collections of sequence data. Most existing tools use sequence homology to establish evolutionary relationships and assign putative functions to sequences. However, it can be difficult to define a similarity threshold that achieves sufficient coverage without sacrificing annotation quality. Defining the correct configuration is critical and can be challenging for non-specialist users. Thus, the development of robust automatic annotation techniques that generate high-quality annotations without needing expert knowledge would be very valuable for the research community. We present Sma3s, a tool for automatically annotating very large collections of biological sequences from any kind of gene library or genome. Sma3s is composed of three modules that progressively annotate query sequences using either: (i) very similar homologues, (ii) orthologous sequences or (iii) terms enriched in groups of homologous sequences. We trained the system using several random sets of known sequences, demonstrating average sensitivity and specificity values of ∼85%. In conclusion, Sma3s is a versatile tool for high-throughput annotation of a wide variety of sequence datasets that outperforms the accuracy of other well-established annotation algorithms, and it can enrich existing database annotations and uncover previously hidden features. Importantly, Sma3s has already been used in the functional annotation of two published transcriptomes.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;functional annotation&quot;">functional annotation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome annotation&quot;">genome annotation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;transcriptome annotation&quot;">transcriptome annotation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;bioinformatic tool&quot;">bioinformatic tool</a></div>                            <h2 scrollto-destination=6651734 id="6651734" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Sequence annotation is the process of associating biological information to sequences of interest. Annotations can include the potential function, cellular localization, biological process or protein structure of a given sequence.<sup><a reveal-id="DSU001C1" data-open="DSU001C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Some sequences are annotated using direct experimental evidence, but most annotations are inferred from sequence similarities or conserved patterns associated with known characteristics.<sup><a reveal-id="DSU001C2  DSU001C5" data-open="DSU001C2  DSU001C5" class="link link-ref link-reveal xref-bibr">2–5</a></sup> Large publically accessible databases of annotated sequences make it possible to automatically annotate large collections of unknown sequences. This is especially valuable for the interpretation of large sequence datasets generated by genome and expressed sequence tag (EST) sequencing projects as well as gene and protein expression experiments, such as DNA microarrays, and many other emerging research areas.<sup><a reveal-id="DSU001C6" data-open="DSU001C6" class="link link-ref link-reveal xref-bibr">6</a></sup></p><p>Sequence annotation is also important in transcriptomic experiments that aim to identify gene clusters with similar expression patterns that are linked to a particular biological process or experimental condition. Biological function can then be inferred from annotations shared within these clusters.<sup><a reveal-id="DSU001C7" data-open="DSU001C7" class="link link-ref link-reveal xref-bibr">7</a></sup></p><p>Several methods have been developed for the annotation of the large datasets generated by sequencing projects. These methods typically perform homology-based searches and infer annotations from sequences with high levels of homology to the query sequence. To detect sequence similarity, most methods use BLAST, a fast heuristic algorithm for identifying homology between sequences based on local sequence alignments.<sup><a reveal-id="DSU001C8" data-open="DSU001C8" class="link link-ref link-reveal xref-bibr">8</a></sup></p><p>Blast2GO is one of the most widely used tools for genomic annotation.<sup><a reveal-id="DSU001C9" data-open="DSU001C9" class="link link-ref link-reveal xref-bibr">9</a></sup> It performs three steps: a BLAST search, a step that maps similar sequences to existing annotation associations, and finally term annotation. As its name suggests, Blast2GO mainly uses terms from the Gene Ontology (GO) controlled vocabulary for annotation,<sup><a reveal-id="DSU001C10" data-open="DSU001C10" class="link link-ref link-reveal xref-bibr">10</a></sup> although it also includes other annotation classes. However, Blast2GO is most commonly run online, which can make it more difficult to analyse large collections of sequences. Other methods can be installed locally, and so avoid problems associated with remote execution, such as sequence number limitations or resource thresholds. Blast2GO is also available in an offline form, but installation requires knowledge of relational databases that make it less suitable for routine use and difficult for non-specialist users. AutoFact<sup><a reveal-id="DSU001C11" data-open="DSU001C11" class="link link-ref link-reveal xref-bibr">11</a></sup> can perform several types of BLAST search using locally downloaded domain and motif databases. However, it uses databases with descriptive non-standard annotation entries rather than controlled vocabularies such as GO. The resulting lack of homogeneity in terms of vocabulary or syntax can make it much more difficult to automatically assign annotations or evaluate annotation quality.</p><p>The above methods can be applied to a wide range of different sequence types and organisms. Many other tools are specialized for specific applications. Some tools only deal with single annotation classes, as is the case for Gotcha, which only uses GO terms.<sup><a reveal-id="DSU001C12" data-open="DSU001C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Others are specialized for the annotation of specific sequence types, such as the EST-specific ESTAnnotator<sup><a reveal-id="DSU001C13" data-open="DSU001C13" class="link link-ref link-reveal xref-bibr">13</a></sup> and EST-PAC.<sup><a reveal-id="DSU001C14" data-open="DSU001C14" class="link link-ref link-reveal xref-bibr">14</a></sup> In addition, some tools are designed especially for the annotation of specific organisms. For example, BLANNANOTATOR assigns non-standard annotations to bacterial sequences.<sup><a reveal-id="DSU001C15" data-open="DSU001C15" class="link link-ref link-reveal xref-bibr">15</a></sup></p><p>Some platforms allow diverse data sources to be integrated for the purposes of sequence annotation and annotation analysis. For example, the Babelomics suite,<sup><a reveal-id="DSU001C16" data-open="DSU001C16" class="link link-ref link-reveal xref-bibr">16</a></sup> of which Blast2GO is part, contains tools for assigning interactions, pathways or even regulatory annotations to analysed sequences. Another widely used platform is DAVID,<sup><a reveal-id="DSU001C17" data-open="DSU001C17" class="link link-ref link-reveal xref-bibr">17</a></sup> which extracts functional annotations from a variety of public genome resources, and allows subsequent analysis by biological enrichment. One of the richest annotation sources is BioMart,<sup><a reveal-id="DSU001C18" data-open="DSU001C18" class="link link-ref link-reveal xref-bibr">18</a></sup> which retrieves sequence annotations from Ensembl and other databases.</p><p>All of these annotation methods use BLAST to detect and evaluate the similarity between a query sequence and putative homologues.<sup><a reveal-id="DSU001C8" data-open="DSU001C8" class="link link-ref link-reveal xref-bibr">8</a></sup> For each query sequence, BLAST attempts to create alignments with database sequences and uses these alignments to judge sequence similarity. Most annotation methods select the database sequence with the highest overall similarity score for each query sequence as the donor of annotation information. In addition, a minimum similarity threshold sets the limit below which sequences will be rejected even if they are the most similar sequence in the database. Defining the similarity threshold is not an easy task, as it must balance annotation quality and sequence coverage (SC). Moreover, it has been shown that the optimal similarity threshold can vary between different sequence types.<sup><a reveal-id="DSU001C19" data-open="DSU001C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Several other BLAST variables, such as how it scores alignment gaps, can also result in significantly different results. These factors can be difficult to configure, particularly for non-specialist users.</p><p>We have developed Sma3s, a fast, accurate and flexible annotation tool specifically designed for the annotation of large collections of sequences obtained from diverse gene libraries or coding sequence datasets. It is comprised of three modules that solve the annotation process with progressively less-stringent sequence similarity requirements, combined with methods to optimize specificity. Each module uses an initial exhaustive BLAST search as its starting point. The third Sma3s module enhances annotation quality using term enrichment to identify annotations shared by groups of similar sequences. We have defined optimal default Sma3s parameter values to minimize user intervention for most applications, thus aiding consistency and comparability. We show that Sma3s can rapidly produce high levels of prediction accuracy with minimal human supervision and modest computational resources.</p>                            <h2 scrollto-destination=6651743 id="6651743" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=6651744 id="6651744" class="section-title" data-legacy-id=s2a>2.1. Implementation</h3>
<p>Sma3s was written in Perl using modules of the BioPerl 1.6 project (<a class="link link-uri" href="http://www.bioperl.org">http://www.bioperl.org</a>) and is freely accessible from the Sma3s website (<a class="link link-uri" href="http://www.bioinfocabd.upo.es/sma3s/">http://www.bioinfocabd.upo.es/sma3s/</a>). Sma3s uses two programmes from the Blast package: blastp for amino acid sequences and blastx for nucleotide sequences, using an <em>E</em>-value threshold of 10<sup>−6</sup>, and blastclust for sequence clustering procedures. A Perl biological enrichment module, based on hypergeometric distribution, was used to calculate probability values (<a class="link link-uri" href="http://www.cse.huji.ac.il/course/2006/bioskill/Ex2/HyGe.pm">http://www.cse.huji.ac.il/course/2006/bioskill/Ex2/HyGe.pm</a>).</p><p>Sma3s uses the UniProt database in plain-text format files (<em>file</em>.dat), corresponding to the taxonomic division of the organism under study, as its main data source. More details about programme requirements and operation are available on the website.</p><p>Running on a dual-processor Intel Pentium 4 CPU 3.00 GHz with 4 Gb of RAM, the programme takes an average of 0.7 s to annotate each sequence (mean length = 400 amino acids) against a division of the Swiss-Prot database (∼45 000 sequences), using the three modules. In fact, the most time-consuming process (∼90%) is the initial exhaustive BLAST (blastp or blastx), whose results are then reused in subsequent parameter tuning and annotation refinement steps.</p>                            <h3 scrollto-destination=6651748 id="6651748" class="section-title" data-legacy-id=s2b>2.2. Annotation types</h3>
<p><em>UniProt</em><sup><a reveal-id="DSU001C20" data-open="DSU001C20" class="link link-ref link-reveal xref-bibr">20</a></sup> is arguably the most complete publically available protein database. Human-based curation ensures high-quality annotations, which is particularly evident in the Swiss-Prot section, compared with the automatically annotated TrEMBL section (also part of UniProt). Sma3s uses Swiss-Prot as its main source for extracting annotations. The UniProt fields used by Sma3s are:</p>                            <h4 scrollto-destination=6651750 id="6651750" class="section-title" data-legacy-id=s2b1>Gene ontology</h4>
<p>GO<sup><a reveal-id="DSU001C10" data-open="DSU001C10" class="link link-ref link-reveal xref-bibr">10</a></sup> provides a controlled vocabulary to describe genes and gene product attributes. It is organized into three biological ontologies: molecular function, biological process and cellular component. Standardized GO term annotations are included in the cross-reference (DR) field of UniProt.</p>                            <h4 scrollto-destination=6651752 id="6651752" class="section-title" data-legacy-id=s2b2>Interpro</h4>
<p><em>InterPro</em><sup><a reveal-id="DSU001C21" data-open="DSU001C21" class="link link-ref link-reveal xref-bibr">21</a></sup> is an integrated documentation resource of protein families, domains and sites. It combines complementary sequence pattern information from several databases. UniProt provides InterPro identifiers also from the DR field.</p>                            <h4 scrollto-destination=6651754 id="6651754" class="section-title" data-legacy-id=s2b3>Swiss-Prot keywords</h4>
<p>The <em>Swiss-Prot keywords</em> constitute a well-defined and controlled vocabulary of terms used to annotate a UniProt protein entry. These keywords (the KW field in UniProt) describe functions, biological processes, structure, cellular localization and other protein characteristics.</p>                            <h4 scrollto-destination=6651756 id="6651756" class="section-title" data-legacy-id=s2b4>Pathway annotation</h4>
<p>This annotation provides a description of the metabolic pathway(s) in which a protein is involved. It is obtained from the comment (CC) field containing generic and specific metabolic pathway descriptors (e.g.: -!- PATHWAY: Nucleotide metabolism; purine metabolism). Sma3s gathers annotations of the most generic level (‘Nucleotide metabolism’ in the previous example). This type of annotation is particularly useful for identifying co-expressed genes that are active in the same metabolic pathway.</p><p>New annotation types can be incorporated into Sma3s with only minor changes to its algorithms.</p>                            <h3 scrollto-destination=6651759 id="6651759" class="section-title" data-legacy-id=s2c>2.3. Threshold calculation for the selection of multiple homologous sequences</h3>
<div class="block-child-p">Module 3 uses a modified version of Sander's formula,<sup><a reveal-id="DSU001C22" data-open="DSU001C22" class="link link-ref link-reveal xref-bibr">22</a></sup> later updated by Rost<sup><a reveal-id="DSU001C23" data-open="DSU001C23" class="link link-ref link-reveal xref-bibr">23</a></sup> to select alignments based on both identity and alignment length. The original formulas describe the relationship between sequence identity and alignment length observed in sequences sharing structural similarities as a curve. In the Rost equation, the identity threshold (p) for an alignment of length <em>L</em> is defined as: <a id="DSU001M1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU001M1"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mspace width="thinmathspace"></mspace><msup><mi>p</mi><mi>l</mi></msup></mrow><mo stretchy="false" xmlns="">(</mo><mi xmlns="">n</mi><mo stretchy="false" xmlns="">)</mo><mo xmlns="">=</mo><mi xmlns="">n</mi><mo xmlns="">+</mo><mn xmlns="">480</mn><mo xmlns="">×</mo><mrow xmlns=""><msup><mi>L</mi><mrow><mo>−</mo><mn>0.32</mn><mo>×</mo><mo stretchy="false">(</mo><mn>1</mn><mo>+</mo><mrow><msup><mi>e</mi><mrow><mo>−</mo><mi>L</mi><mrow><mo>/</mo></mrow><mn>100</mn></mrow></msup></mrow><mo stretchy="false">)</mo></mrow></msup></mrow><mo xmlns="">,</mo></math></div></div><div class="label">(1)</div></div> where <em>n</em> describes the distance in percentage from the original curve. This value allows curve adjustment to increase stringency and to ensure at least 40% identity (<em>n</em> = 20) at any alignment length in accordance with the ∼40% threshold for functional conservation described by Wilson <em>et al.</em><sup><a reveal-id="DSU001C24" data-open="DSU001C24" class="link link-ref link-reveal xref-bibr">24</a></sup> and will be the default value used for Module 3 annotation.</div>                            <h3 scrollto-destination=6651761 id="6651761" class="section-title" data-legacy-id=s2d>2.4. Statistical significance analysis of annotations</h3>
<p>Each term assigned by Sma3s' prediction algorithm is assigned a <em>P</em>-value to indicate annotation quality. Annotations from Module 3 take <em>P</em>-values from a biological enrichment calculation, while those from Modules 1 and 2 are derived from the highest rated BLAST hit.</p><div class="block-child-p">The biological enrichment algorithm (see below) is based on hypergeometric distribution as follows: from <em>M</em> sequences in the database, <em>K</em> contains annotation <em>X</em>; with <em>m</em> sequences in the annotated cluster, the probability <em>P</em>(<em>x</em> = <em>k</em>) that <em>k</em> sequences will have annotation <em>X</em> is: <a id="DSU001M2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU001M2"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">P</mi><mo stretchy="false" xmlns="">(</mo><mi xmlns="">x</mi><mo xmlns="">=</mo><mi xmlns="">k</mi><mo stretchy="false" xmlns="">)</mo><mo xmlns="">=</mo><mrow xmlns=""><mfrac><mrow><mfenced close=")" open="("><mrow><mtable columnspacing="1em" rowspacing="4pt"><mtr><mtd><mi>K</mi></mtd></mtr><mtr><mtd><mi>k</mi></mtd></mtr></mtable></mrow></mfenced><mfenced close=")" open="("><mrow><mtable columnspacing="1em" rowspacing="4pt"><mtr><mtd><mi>M</mi><mspace width="-8pt"></mspace><mo>−</mo><mrow><mspace width="1.5pt"></mspace><mi>K</mi></mrow></mtd></mtr><mtr><mtd><mi>m</mi><mspace width="-1pc"></mspace><mo>−</mo><mrow><mspace width="1.5pt"></mspace><mi>k</mi></mrow></mtd></mtr></mtable></mrow></mfenced></mrow><mfenced close=")" open="("><mrow><mtable columnspacing="1em" rowspacing="4pt"><mtr><mtd><mi>M</mi></mtd></mtr><mtr><mtd><mi>m</mi></mtd></mtr></mtable></mrow></mfenced></mfrac></mrow></math></div></div><div class="label">(2)</div></div></div><div class="block-child-p">Assuming that <em>l</em> sequences were found with annotation <em>X</em>, the probability <em>P</em> that <em>l</em> can be adjusted to the null hypothesis is: <a id="DSU001M3" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU001M3"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">P</mi><mo xmlns="">=</mo><munderover xmlns=""><mo movablelimits="false">∑</mo><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mrow><mo movablelimits="true">min</mo><mo fence="false" stretchy="false">{</mo><mrow><mi>K</mi><mo>,</mo><mi>m</mi></mrow><mo fence="false" stretchy="false">}</mo></mrow></munderover><mrow xmlns=""><mi>P</mi><mo stretchy="false">(</mo><mi>x</mi><mo>=</mo><mi>k</mi><mo stretchy="false">)</mo></mrow><mo xmlns="">,</mo></math></div></div><div class="label">(3)</div></div></div><p>The cut-off <em>P</em>-value was determined experimentally to be ≤0.1 (see the Results section).</p>                            <h3 scrollto-destination=6651766 id="6651766" class="section-title" data-legacy-id=s2e>2.5. Accuracy estimation</h3>
<p>Sma3s assesses three different ratios to evaluate method accuracy taking into account test sets of previously annotated sequences. These ratios are based on TP (true positives: predicted terms corresponding to the real annotation), FP (false positives: predicted terms not corresponding to the real annotation) and FN (false negatives: non-predicted real terms).</p><div class="block-child-p">It is worth pointing out that (TP + FN) corresponds to the number of real annotations and (TP + FP) represents the number of predicted annotations. In this way, we can calculate both sensitivity and specificity indices. We can also calculate the Relative Error Quotient (REQ), which evaluates the overall prediction quality by considering both the sensitivity and specificity measurements as described in the GOtcha method.<sup><a reveal-id="DSU001C12" data-open="DSU001C12" class="link link-ref link-reveal xref-bibr">12</a></sup> REQ is calculated as:<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtext xmlns="">REQ = </mtext><mstyle xmlns=""><mrow><mfrac><mrow><mtext>FN</mtext><mo>×</mo><mi>W</mi><mtext>+ FP</mtext></mrow><mrow><mtext>TP</mtext><mo>×</mo><mo stretchy="false">(</mo><mn>1</mn><mo>+</mo><mi>W</mi><mo stretchy="false">)</mo></mrow></mfrac></mrow><mo>,</mo></mstyle></math></div></div><div class="label">(4)</div></div> where <em>W</em> is a weighting factor that moderates the influence of FN and FP. In this work, <em>W</em> has been set to 1 to make results comparable with the literature. Low REQ values indicate a low annotation error rate, whereas a high REQ indicates a higher proportion of errors. This measure of accuracy has the advantage that it encompasses TP, FP and FN, thus combining sensitivity and selectivity in a single value.</div><p>To evaluate prediction coverage percentages, two more values were calculated: the SC (the number of sequences with predicted annotations with respect to sequences with known annotations) and term coverage (TC, the number of predicted annotations with respect to known annotations).</p>                            <h3 scrollto-destination=6651770 id="6651770" class="section-title" data-legacy-id=s2f>2.6. Test datasets</h3>
                            <h4 scrollto-destination=6651771 id="6651771" class="section-title" data-legacy-id=s2f1>2.6.1. Random sequence datasets</h4>
<p>All the databases used in this work belong to the UniProt Knowledgebase Release 2011_08, which were obtained from the EBI ftp-site: <a class="link link-uri" href="http://ftp://ftp.ebi.ac.uk/pub/databases/uniprot/current_release/knowledgebase/taxonomic_divisions/">ftp://ftp.ebi.ac.uk/pub/databases/uniprot/current_release/knowledgebase/taxonomic_divisions/</a>.</p><p>We downloaded four taxonomic divisions (bacteria, plants, invertebrates and mammals) and built five independent random datasets of 1000 sequences (5000 in total) for each taxonomic division. These datasets were processed with Sma3s. Query sequences were removed from the database before each run to avoid self-annotation.</p>                            <h4 scrollto-destination=6651774 id="6651774" class="section-title" data-legacy-id=s2f2>2.6.2. Sequence sets from DNA arrays</h4>
<p>We extracted two independent DNA arrays from the GEO database:<sup><a reveal-id="DSU001C25" data-open="DSU001C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Affymetrix Murine 11K SubB Array (AC:GPL76) and the Affymetrix Arabidopsis ATH1 Genome Array (AC:GPL198). The nucleotide sequences from these arrays were downloaded, and sequences shorter than 150 were removed (to ensure a minimum amino acid sequence length of 50). Reference annotations were taken from both Affymetrix-assigned GO terms and GO terms extracted from the Swiss-Prot entries referred to in the Affymetrix annotation.</p>                            <h3 scrollto-destination=6651776 id="6651776" class="section-title" data-legacy-id=s2g>2.7. Blast2GO and Top-BLAST tests</h3>
<p>We opened Blast2GO as a Java application (<a class="link link-uri" href="http://www.blast2go.com/b2ghome">http://www.blast2go.com/b2ghome</a>), selecting the 2Gb memory version, as would be typical for non-expert users performing a search with default parameters. The test did not include graphical data mining or enrichment analysis Blast2GO options. For Top-BLAST annotation, the best hit from a default BLAST search was used to annotate every sequence in the dataset. Self-prediction in Blast2GO (annotation based on annotations from the query sequence) was unavoidable, since dataset sequences cannot be removed from the database.</p>                            <h2 scrollto-destination=6651778 id="6651778" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=6651779 id="6651779" class="section-title" data-legacy-id=s3a>3.1. Sma3s algorithm overview</h3>
<p>With Sma3s, we set out to develop a tool that can intelligently assign annotations to query sequences in a way that maximizes accuracy without sacrificing coverage, and works in a way that is robust enough to generate high-quality results without having to adjust the parameters.</p><p>To achieve these aims, we designed Sma3s as a modular system that processes query sequences in a series of discrete steps. The method starts with a BLAST similarity search, followed by a three-step annotation process which sequentially tests different degrees of similarity between query and database sequences (Fig. <a reveal-id="DSU001F1" data-open="DSU001F1" class="link link-reveal link-table xref-fig">1</a>). The first module annotates query sequences using database homologues with very high levels of similarity (Fig. <a reveal-id="DSU001F1" data-open="DSU001F1" class="link link-reveal link-table xref-fig">1</a>, M1). The second module performs a reciprocal BLAST on sequences unannotated in the first step to generate annotations from orthologous database sequences (Fig. <a reveal-id="DSU001F1" data-open="DSU001F1" class="link link-reveal link-table xref-fig">1</a>, M2). The third module uses a novel strategy (Fig. <a reveal-id="DSU001F1" data-open="DSU001F1" class="link link-reveal link-table xref-fig">1</a>, M3) that looks for more distantly related sequences and analyses these data for statistically significant sequence annotations. </p>                        <a id="6651782" scrollto-destination="6651782"></a>
<div id="DSU001F1" data-id="DSU001F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu001/2/m_dsu00101.png?Expires=2147483647&Signature=UZq3g9b8htqmvQGmgvS0TD41vRmeSe01LPrzNrWbbtcqy1smMzsrQT6P7h3508rHmX5olglHcjspaMCJDc5wcqaSNdscYeWoSZLZosYyjYbMgewiSd96t7AphsQ1oZ2f9bnBrl-OC4jzbW-O~QgLZEkB42xsHkbarsx0fH-s556KvaJwx5mn6vF3XklEbcLDBsWe6i43g8vofnd32LChSIxHo~zCNEMl47WYPSpEzN2AHrgJcWkUskBxfiiSOahE0v6FNhh957wa01pY9hS13Y-9LoEOm0RSHo9OMDNfqkd4z-AGuepLNfovx6spaBGsbxotbooSu3lgGHi~ucQlTQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sma3s workflow. The first M1 module derives annotations from highly similar sequences stored in the database, choosing sequences using the Top-BLAST method, which selected the highest similarity homologue from each BLAST search that meets the minimum similarity criteria. The remaining sequences are passed to the second module (M2), which performs reciprocal BLAST searches to identify orthologous sequences as annotation sources, also using the Top-BLAST method. Finally, the M3 module obtains annotations from a set of related sequences whose similarity is supported by statistically significant concentrations of similar annotations, filtered by clustering techniques to avoid over-representation from duplicated gene families. This figure appears in colour in the online version of DNA Research." path-from-xml="dsu00101.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6651782/dsu00101.tiff" path-from-xml="dsu00101.tiff" target="_blank">View large</a><a section="6651782" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu001/2/dsu00101.png?Expires=2147483647&Signature=d9ig4BZQFHwdfDhQFEXcUtEDFUKvEeNofY2QJx0EGoJkMiORx5QTrqaiuuxOSia7fikaFRCieZG-0i-0cYwBclXuqGZHt~b0VZEWV0yfM5OvMl7WMtOY88ZWqXIIgHoqeu6~zhX5vH9Kl657zgDM2iJjFmqntlQ4Gk-PdBMjq~4thIIXls0T9oKlq30l5oJpoSncC8zrP2G5ogOFClNT97Dquy3EcEsPQfs9kclUcfgy21k-Dgw5wAkl5c~gFuzf9rfj7aywPFoecNZWk3crjP0MNHAfeYvPf~7vuMewQztuW0TZ6DTKtEZ~86jYIggotjzWaE2qtVKY5Dks2G6Ghg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6651782&ar=444964&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00101.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sma3s workflow. The first M1 module derives annotations from highly similar sequences stored in the database, choosing sequences using the Top-BLAST method, which selected the highest similarity homologue from each BLAST search that meets the minimum similarity criteria. The remaining sequences are passed to the second module (M2), which performs reciprocal BLAST searches to identify orthologous sequences as annotation sources, also using the Top-BLAST method. Finally, the M3 module obtains annotations from a set of related sequences whose similarity is supported by statistically significant concentrations of similar annotations, filtered by clustering techniques to avoid over-representation from duplicated gene families. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><div content-id="DSU001F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu001/2/m_dsu00101.png?Expires=2147483647&Signature=UZq3g9b8htqmvQGmgvS0TD41vRmeSe01LPrzNrWbbtcqy1smMzsrQT6P7h3508rHmX5olglHcjspaMCJDc5wcqaSNdscYeWoSZLZosYyjYbMgewiSd96t7AphsQ1oZ2f9bnBrl-OC4jzbW-O~QgLZEkB42xsHkbarsx0fH-s556KvaJwx5mn6vF3XklEbcLDBsWe6i43g8vofnd32LChSIxHo~zCNEMl47WYPSpEzN2AHrgJcWkUskBxfiiSOahE0v6FNhh957wa01pY9hS13Y-9LoEOm0RSHo9OMDNfqkd4z-AGuepLNfovx6spaBGsbxotbooSu3lgGHi~ucQlTQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sma3s workflow. The first M1 module derives annotations from highly similar sequences stored in the database, choosing sequences using the Top-BLAST method, which selected the highest similarity homologue from each BLAST search that meets the minimum similarity criteria. The remaining sequences are passed to the second module (M2), which performs reciprocal BLAST searches to identify orthologous sequences as annotation sources, also using the Top-BLAST method. Finally, the M3 module obtains annotations from a set of related sequences whose similarity is supported by statistically significant concentrations of similar annotations, filtered by clustering techniques to avoid over-representation from duplicated gene families. This figure appears in colour in the online version of DNA Research." path-from-xml="dsu00101.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6651782/dsu00101.tiff" path-from-xml="dsu00101.tiff" target="_blank">View large</a><a section="6651782" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu001/2/dsu00101.png?Expires=2147483647&Signature=d9ig4BZQFHwdfDhQFEXcUtEDFUKvEeNofY2QJx0EGoJkMiORx5QTrqaiuuxOSia7fikaFRCieZG-0i-0cYwBclXuqGZHt~b0VZEWV0yfM5OvMl7WMtOY88ZWqXIIgHoqeu6~zhX5vH9Kl657zgDM2iJjFmqntlQ4Gk-PdBMjq~4thIIXls0T9oKlq30l5oJpoSncC8zrP2G5ogOFClNT97Dquy3EcEsPQfs9kclUcfgy21k-Dgw5wAkl5c~gFuzf9rfj7aywPFoecNZWk3crjP0MNHAfeYvPf~7vuMewQztuW0TZ6DTKtEZ~86jYIggotjzWaE2qtVKY5Dks2G6Ghg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6651782&ar=444964&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00101.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sma3s workflow. The first M1 module derives annotations from highly similar sequences stored in the database, choosing sequences using the Top-BLAST method, which selected the highest similarity homologue from each BLAST search that meets the minimum similarity criteria. The remaining sequences are passed to the second module (M2), which performs reciprocal BLAST searches to identify orthologous sequences as annotation sources, also using the Top-BLAST method. Finally, the M3 module obtains annotations from a set of related sequences whose similarity is supported by statistically significant concentrations of similar annotations, filtered by clustering techniques to avoid over-representation from duplicated gene families. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><p>Module 1: The first annotator uses a Top-BLAST strategy to check whether the query sequence (or one very similar to it) already exists in the UniProt database, and then directly assigns database annotations to the query sequence. We established a minimum sequence identity of 90% and a sequence overlap of at least 90% of the database sequence, although these parameters are customizable. In all cases, Sma3s uses annotations from Swiss-Prot in preference to TrEMBL when identical amino acid sequences are found in both databases (Table <a reveal-id="DSU001TB1" data-open="DSU001TB1" class="link link-reveal link-table xref-fig">1</a>, see the next section for details). </p>                        <a id="6651784" scrollto-destination="6651784"></a>
<div content-id="DSU001TB1" class="table-modal"><div class="table-wrap"><div id="DSU001TB1" class="table-wrap-title" data-id="DSU001TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Sma3s results with different source databases</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Sn </th><th>Sp </th><th>REQ </th><th>SC </th></tr></thead><tbody><tr><td>Swiss-Prot </td><td>0.83 ± 0.13 </td><td>0.87 ± 0.07 </td><td>0.19 ± 0.15 </td><td>0.87 ± 0.13 </td></tr><tr><td>TrEMBL </td><td>0.61 ± 0.15 </td><td>0.86 ± 0.08 </td><td>0.45 ± 0.25 </td><td>0.94 ± 0.06 </td></tr><tr><td>UniProt </td><td>0.68 ± 0.09 </td><td>0.87 ± 0.07 </td><td>0.32 ± 0.14 </td><td>0.94 ± 0.05 </td></tr></tbody></table></div><div content-id="DSU001TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Sn </th><th>Sp </th><th>REQ </th><th>SC </th></tr></thead><tbody><tr><td>Swiss-Prot </td><td>0.83 ± 0.13 </td><td>0.87 ± 0.07 </td><td>0.19 ± 0.15 </td><td>0.87 ± 0.13 </td></tr><tr><td>TrEMBL </td><td>0.61 ± 0.15 </td><td>0.86 ± 0.08 </td><td>0.45 ± 0.25 </td><td>0.94 ± 0.06 </td></tr><tr><td>UniProt </td><td>0.68 ± 0.09 </td><td>0.87 ± 0.07 </td><td>0.32 ± 0.14 </td><td>0.94 ± 0.05 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn1"><div class="fn"><p>Sequence coverage (SC), specificity (Sp), sensitivity (Sn) and REQ values are shown together with the corresponding standard deviation.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6651784" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU001TB1" class="table-wrap-title" data-id="DSU001TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Sma3s results with different source databases</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Sn </th><th>Sp </th><th>REQ </th><th>SC </th></tr></thead><tbody><tr><td>Swiss-Prot </td><td>0.83 ± 0.13 </td><td>0.87 ± 0.07 </td><td>0.19 ± 0.15 </td><td>0.87 ± 0.13 </td></tr><tr><td>TrEMBL </td><td>0.61 ± 0.15 </td><td>0.86 ± 0.08 </td><td>0.45 ± 0.25 </td><td>0.94 ± 0.06 </td></tr><tr><td>UniProt </td><td>0.68 ± 0.09 </td><td>0.87 ± 0.07 </td><td>0.32 ± 0.14 </td><td>0.94 ± 0.05 </td></tr></tbody></table></div><div content-id="DSU001TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Sn </th><th>Sp </th><th>REQ </th><th>SC </th></tr></thead><tbody><tr><td>Swiss-Prot </td><td>0.83 ± 0.13 </td><td>0.87 ± 0.07 </td><td>0.19 ± 0.15 </td><td>0.87 ± 0.13 </td></tr><tr><td>TrEMBL </td><td>0.61 ± 0.15 </td><td>0.86 ± 0.08 </td><td>0.45 ± 0.25 </td><td>0.94 ± 0.06 </td></tr><tr><td>UniProt </td><td>0.68 ± 0.09 </td><td>0.87 ± 0.07 </td><td>0.32 ± 0.14 </td><td>0.94 ± 0.05 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn1"><div class="fn"><p>Sequence coverage (SC), specificity (Sp), sensitivity (Sn) and REQ values are shown together with the corresponding standard deviation.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6651784" target="_blank">View Large</a></div></div><p>Module 2: This module annotates using information from putative orthologues with lower similarity. The second module of Sma3s is based on reciprocal BLAST searches. It detects lower similarity sequences with at least 75% of per-residue identity, covering at least 75% of database sequence length. The best hit is then used as an input seed in a blastp or tblastn search against the initial set of query sequences. If this second BLAST identifies the starting query sequence as the best hit, then an orthology relationship is established, and consequently, the annotation recovered. As in the previous step, Sma3s uses Swiss-Prot annotations in preference to TrEMBL (Table <a reveal-id="DSU001TB1" data-open="DSU001TB1" class="link link-reveal link-table xref-fig">1</a>, see the next section for details).</p><p>Module 3: Module 3 recovers annotation information from multiple homologous sequences using a novel multi-step approach. The first step is to determine which homologues have sufficient similarity to the query sequence. Work by Sander<sup><a reveal-id="DSU001C22" data-open="DSU001C22" class="link link-ref link-reveal xref-bibr">22</a></sup> and Rost<sup><a reveal-id="DSU001C23" data-open="DSU001C23" class="link link-ref link-reveal xref-bibr">23</a></sup> found that minimum alignment length was a better criterion than overall percentage sequence identity for estimating sequence and structural homology, especially when comparing pairs of structurally matched proteins with lower levels of similarity. Intuitively, for short sequence alignments, a high percentage identity is needed to establish statistically significant relationships. Conversely, long pairwise alignments require lower identity to be qualified as significant. In the context of protein structure, a minimum threshold of ∼20% was found to be a good predictor of protein structure, provided that the alignment had a length of at least 150 residues. Function is less well conserved than structure, but further studies have reported that biological function is typically conserved when two sequences exhibit 40% sequence identity.<sup><a reveal-id="DSU001C24" data-open="DSU001C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Thus, based on the hypothesis that function conservation is also related to sequence alignment length, we have used a modified form of Rost's equation (see Materials and Methods for details) that only selects sequences whose alignments have at least 40% identity at any alignment length.</p><p>Although the sequences selected by this method can have significantly lower similarities to those identified by Modules 1 and 2, the combined analyses of multiple sequences can be used to increase annotation reliability. This is based on the hypothesis that annotations shared by several homologous sequences are more likely to reflect functions shared by the query sequence. Module 3 uses biological enrichment to only select those terms which appear more frequently in the identified homologues than would be expected by chance in the source database (see ‘Statistical significance assessment of annotations’ in the Materials and Methods section for details).</p><p>The presence of several redundant sequences in the search results could bias the term enrichment algorithm. To reduce this possibility, very similar sequences are first combined by ‘clustering’. Redundant sequences are identified using blastclust (part of the Blast package), using 95% identity and alignment length as search criteria. Thus, when evaluating the frequency of annotation terms, each cluster counts only once, regardless of the number of sequences it contains.</p><p>In summary, Module 3 processes each query as follows (also see Fig. <a reveal-id="DSU001F1" data-open="DSU001F1" class="link link-reveal link-table xref-fig">1</a>, M3): (i) the BLAST report is scanned for statistically significant alignments according to Rost's equation; (ii) identification and grouping of clusters containing very similar members of the same family; (iii) a preliminary annotation set is formed by combining individual non-redundant annotations from each cluster and finally, (iv) biological term enrichment is used to select significantly overrepresented annotations in each group of homologous sequences. Owing to the modular implementation of the algorithm, each module can be run independently, allowing the user to easily customize annotation criteria.</p>                            <h3 scrollto-destination=6651790 id="6651790" class="section-title" data-legacy-id=s3b>3.2. Sma3s annotates random Swiss-Prot sequence collections with high accuracy</h3>
<p>We trained Sma3s using random datasets extracted from the four Swiss-Prot taxonomic divisions. Five independent random sets of 1000 sequences (5000 in total) were extracted for each division. These sequences were then annotated using the corresponding division from Swiss-Prot, and each query sequence removed from the database at each step to avoid self-annotation.</p><p>We evaluated the annotation of each random set of sequences for SC, specificity, sensitivity and REQ values (an accuracy measurement which combines both specificity and sensitivity, with 0 being the best possible value). The results obtained varied significantly between taxonomic divisions, but not between different datasets from the same division (Fig. <a reveal-id="DSU001F2" data-open="DSU001F2" class="link link-reveal link-table xref-fig">2</a>). </p>                        <a id="6651793" scrollto-destination="6651793"></a>
<div id="DSU001F2" data-id="DSU001F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu001/2/m_dsu00102.png?Expires=2147483647&Signature=XhD2gMrPJt959ic1TbtEv0tcO9JG3549OR~0gDh6xgkNcdLqTwWj9b~rrWBKIXvbGnhs9Y8Ql6l8N-U5X~AvJBR5khfQZrpJ9-mYb1LQyq42b69TJ1rOq5tb6LhmvAUzucygqsgY~jQx6rGVdWPvacFpUzd2-840nmY-LmoHXL5e81IDMlai4RV4uws~4mWZfSKf1KIP7z6GgQ7ejuExEem4dvhfoIZM6cD4dxMddd1xaiZyUrC-SMVfScrbBvibCF1y4fLBDbwuMkqifXlowuoWrH0oWy7pPsGOY2eGUEccPw78MLtHNmtaUVz~6AIH1WEQJ~Sy8TrjzJFg9oGcow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sma3s annotation of random Swiss-Prot test datasets. The sequence coverage (SC), specificity (Sp), sensitivity (Sn) and REQ values obtained by Sma3s are shown for (a) bacteria, (b) plants, (c) mammals, (d) invertebrates, (e) and average values, with standard deviation for all four datasets. This figure appears in colour in the online version of DNA Research." path-from-xml="dsu00102.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6651793/dsu00102.tiff" path-from-xml="dsu00102.tiff" target="_blank">View large</a><a section="6651793" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu001/2/dsu00102.png?Expires=2147483647&Signature=z-F3rLWdGTn6qtCsFl9l0APyNVD18KkqVNVLoXBYZxZmJw57BStVTsW1AWS5H-Ovclar1KCsgtgB9QqU5gIXwCW3i5a5pGTeHHmd8hyIiyvj1ibu7X0Z0cCL23TMMOPZoeBGkDEccW2a8DtQQ2ggO-CKnUJN61lJvL02Twaapi9xy0VYOnXekOc~Cf4B0cQKgZMV72rb21GfRdbKP6fjFTClB-hT7pVay49cjw9lO~nb3CuolexWnYCxH1iq0PlDE3-RJvUXFVV1iflnaVFZPK5BKxcPQgzR69USgrgSgmQUxOakFQjQShN-ZvK5gAXFz1fz1NSyWbGj0wbQgOvdqQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6651793&ar=444964&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00102.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sma3s annotation of random Swiss-Prot test datasets. The sequence coverage (SC), specificity (Sp), sensitivity (Sn) and REQ values obtained by Sma3s are shown for (a) bacteria, (b) plants, (c) mammals, (d) invertebrates, (e) and average values, with standard deviation for all four datasets. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><div content-id="DSU001F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu001/2/m_dsu00102.png?Expires=2147483647&Signature=XhD2gMrPJt959ic1TbtEv0tcO9JG3549OR~0gDh6xgkNcdLqTwWj9b~rrWBKIXvbGnhs9Y8Ql6l8N-U5X~AvJBR5khfQZrpJ9-mYb1LQyq42b69TJ1rOq5tb6LhmvAUzucygqsgY~jQx6rGVdWPvacFpUzd2-840nmY-LmoHXL5e81IDMlai4RV4uws~4mWZfSKf1KIP7z6GgQ7ejuExEem4dvhfoIZM6cD4dxMddd1xaiZyUrC-SMVfScrbBvibCF1y4fLBDbwuMkqifXlowuoWrH0oWy7pPsGOY2eGUEccPw78MLtHNmtaUVz~6AIH1WEQJ~Sy8TrjzJFg9oGcow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sma3s annotation of random Swiss-Prot test datasets. The sequence coverage (SC), specificity (Sp), sensitivity (Sn) and REQ values obtained by Sma3s are shown for (a) bacteria, (b) plants, (c) mammals, (d) invertebrates, (e) and average values, with standard deviation for all four datasets. This figure appears in colour in the online version of DNA Research." path-from-xml="dsu00102.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6651793/dsu00102.tiff" path-from-xml="dsu00102.tiff" target="_blank">View large</a><a section="6651793" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu001/2/dsu00102.png?Expires=2147483647&Signature=z-F3rLWdGTn6qtCsFl9l0APyNVD18KkqVNVLoXBYZxZmJw57BStVTsW1AWS5H-Ovclar1KCsgtgB9QqU5gIXwCW3i5a5pGTeHHmd8hyIiyvj1ibu7X0Z0cCL23TMMOPZoeBGkDEccW2a8DtQQ2ggO-CKnUJN61lJvL02Twaapi9xy0VYOnXekOc~Cf4B0cQKgZMV72rb21GfRdbKP6fjFTClB-hT7pVay49cjw9lO~nb3CuolexWnYCxH1iq0PlDE3-RJvUXFVV1iflnaVFZPK5BKxcPQgzR69USgrgSgmQUxOakFQjQShN-ZvK5gAXFz1fz1NSyWbGj0wbQgOvdqQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6651793&ar=444964&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00102.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sma3s annotation of random Swiss-Prot test datasets. The sequence coverage (SC), specificity (Sp), sensitivity (Sn) and REQ values obtained by Sma3s are shown for (a) bacteria, (b) plants, (c) mammals, (d) invertebrates, (e) and average values, with standard deviation for all four datasets. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><p>To identify the optimal default cut-off value, we tested Sma3s with different <em>P</em>-values using our random datasets as query sequences and the corresponding taxonomic divisions of the Swiss-Prot database as annotation sources. As can be seen in Fig. <a reveal-id="DSU001F2" data-open="DSU001F2" class="link link-reveal link-table xref-fig">2</a>, more restrictive <em>P</em>-values generated consistent results with low REQ values and higher specificity (Sp) but at the cost of reduced sensitivity (Sn). The highest accuracy, as defined by the lowest REQ values, was obtained when specificity and sensitivity had equal or similar values. With the exception of the invertebrate dataset, a <em>P</em>-value of 0.1 resulted in average sensitivity and specificity values &gt;0.8. These values compare favourably with those of other tools, such as Blast2GO<sup><a reveal-id="DSU001C9" data-open="DSU001C9" class="link link-ref link-reveal xref-bibr">9</a></sup> or Gotcha,<sup><a reveal-id="DSU001C12" data-open="DSU001C12" class="link link-ref link-reveal xref-bibr">12</a></sup> whose accuracy typically ranges between 0.65 and 0.7. On this basis, we established a default <em>P-</em>value of 0.1 for Sma3s.</p><p>When the same test datasets were annotated using the uncurated TrEMBL database or the complete UniProt database instead of Swiss-Prot, the average accuracy was reduced despite the high coverage obtained (Table <a reveal-id="DSU001TB1" data-open="DSU001TB1" class="link link-reveal link-table xref-fig">1</a>). This can be explained by the fact that the unreviewed TrEMBL sequence collection is much larger than the manually annotated Swiss-Prot set. Thus, a large proportion of annotations assigned from the UniProt database are derived from TrEMBL, even though UniProt also contains Swiss-Prot sequences.</p><p>Accuracy was also calculated separately for each of the four annotation types recovered by Sma3s: GO terms, SW-Keywords, InterPro and Pathways (Fig. <a reveal-id="DSU001F3" data-open="DSU001F3" class="link link-reveal link-table xref-fig">3</a>). Best results were obtained for Pathway and InterPro annotations, which had REQ values ≤0.2 for all except the invertebrate dataset. Keyword and GO annotations had poorer REQ values, but still achieved average sensitivity and specificity values of 0.8 (and as high as 0.97 for bacteria). The exception, again, was the invertebrate dataset for which Sma3s generated higher REQ values, associated with significantly lower sensitivity. The relatively poor results for the invertebrate dataset are probably due to the relatively small number of sequences present in the corresponding invertebrate database. Sequence length is also important. In fact, this dataset is the group with the highest proportion of very short sequences (10-40 amino acids), which are usually annotated with difficulty (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1a</a>). </p>                        <a id="6651797" scrollto-destination="6651797"></a>
<div id="DSU001F3" data-id="DSU001F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu001/2/m_dsu00103.png?Expires=2147483647&Signature=L2l6umgNFllLCZ9euP56kGPUQoRxdBhM8U2fodw3iqfBU~xyAMp6dN079GTJSwwwesyHP2FBKJ3Z5U6YIn5xRJEfZIBR7NJzGKANXvb54M2EKFK1imWOAIAvhXsK-77Xou-9GB1PwPnFZm33r66mLDsPSaM1aU5HByUhKXWkBStjSXVGVbU0DtmukxyG1kJAy5njN54N94bxXc3eHh6NfomqPCHlcQEXf7O0BYFWzUEz6kY7bBJttkcMIN3Dn6uA5YCiiOCBvweaXfwNcynu6xJBL7WSHKBOcOmPMqz--IY59MZkNsbMS3HON-ZT7WeDq0O~n0n5~siPYz2RUJLaCA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sensitivity, specificity and REQ index for different annotation types. (a) Sensitivity, (b) specificity and (c) REQ index calculated from the results obtained by Sma3s with the random datasets from different taxonomic divisions are shown, separated by the different annotation types under study. This figure appears in colour in the online version of DNA Research." path-from-xml="dsu00103.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6651797/dsu00103.tiff" path-from-xml="dsu00103.tiff" target="_blank">View large</a><a section="6651797" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu001/2/dsu00103.png?Expires=2147483647&Signature=jwXzQRzlohI8tiaKS07DKUKIFCcvrNLHndVpradwMH5ZQQN-3oc8kCPiyCAag4tXGzVxNjG8Qp4m7BoxpmbPHAGpgDz0sfDQcvaw11q-Vz0ZhgFxP9NOAfUtYCH7mryIdES-ISV9c3UZBRDozNW9mXU1G7ABSuF48dyYfHYjmHokgz3~VRUSrUxiZ6el~cZS2gbmWp1MziemK9R1PZ7rmel1KSMqluQrB0gPLstoNqBX9eRE1P5CB9gXIE4iEFbfKu~LZw2z95VKa8zv6IaAwStiN7jj1YtK3RucRlcE8L1u7SihTtyX31acBwKIMNTy6miB5WqVwc5Epo7JpqVK4g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6651797&ar=444964&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00103.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sensitivity, specificity and REQ index for different annotation types. (a) Sensitivity, (b) specificity and (c) REQ index calculated from the results obtained by Sma3s with the random datasets from different taxonomic divisions are shown, separated by the different annotation types under study. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><div content-id="DSU001F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu001/2/m_dsu00103.png?Expires=2147483647&Signature=L2l6umgNFllLCZ9euP56kGPUQoRxdBhM8U2fodw3iqfBU~xyAMp6dN079GTJSwwwesyHP2FBKJ3Z5U6YIn5xRJEfZIBR7NJzGKANXvb54M2EKFK1imWOAIAvhXsK-77Xou-9GB1PwPnFZm33r66mLDsPSaM1aU5HByUhKXWkBStjSXVGVbU0DtmukxyG1kJAy5njN54N94bxXc3eHh6NfomqPCHlcQEXf7O0BYFWzUEz6kY7bBJttkcMIN3Dn6uA5YCiiOCBvweaXfwNcynu6xJBL7WSHKBOcOmPMqz--IY59MZkNsbMS3HON-ZT7WeDq0O~n0n5~siPYz2RUJLaCA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sensitivity, specificity and REQ index for different annotation types. (a) Sensitivity, (b) specificity and (c) REQ index calculated from the results obtained by Sma3s with the random datasets from different taxonomic divisions are shown, separated by the different annotation types under study. This figure appears in colour in the online version of DNA Research." path-from-xml="dsu00103.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6651797/dsu00103.tiff" path-from-xml="dsu00103.tiff" target="_blank">View large</a><a section="6651797" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu001/2/dsu00103.png?Expires=2147483647&Signature=jwXzQRzlohI8tiaKS07DKUKIFCcvrNLHndVpradwMH5ZQQN-3oc8kCPiyCAag4tXGzVxNjG8Qp4m7BoxpmbPHAGpgDz0sfDQcvaw11q-Vz0ZhgFxP9NOAfUtYCH7mryIdES-ISV9c3UZBRDozNW9mXU1G7ABSuF48dyYfHYjmHokgz3~VRUSrUxiZ6el~cZS2gbmWp1MziemK9R1PZ7rmel1KSMqluQrB0gPLstoNqBX9eRE1P5CB9gXIE4iEFbfKu~LZw2z95VKa8zv6IaAwStiN7jj1YtK3RucRlcE8L1u7SihTtyX31acBwKIMNTy6miB5WqVwc5Epo7JpqVK4g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6651797&ar=444964&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00103.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sensitivity, specificity and REQ index for different annotation types. (a) Sensitivity, (b) specificity and (c) REQ index calculated from the results obtained by Sma3s with the random datasets from different taxonomic divisions are shown, separated by the different annotation types under study. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div>                            <h3 scrollto-destination=6651798 id="6651798" class="section-title" data-legacy-id=s3c>3.3. The M3 module can enrich the annotation of previously analysed sequences</h3>
<p>As described above, each dataset sequence is annotated by one of the three different modules in the Sma3s algorithm, which sequentially run from M1 to M3. To determine the effectiveness of each module, we calculated the percentage of random dataset sequences annotated by each module. On average, the M1 module annotated 53.76% of sequences (Table <a reveal-id="DSU001TB2" data-open="DSU001TB2" class="link link-reveal link-table xref-fig">2</a>) with M2 and M3 annotating the remainder (19.70 and 26.54%, respectively). As modules M2 and M3 only annotate query sequences that were not annotated by the previous module(s), the ∼26% of sequences annotated by M3 suggests that it significantly increases Sma3s annotation coverage. </p>                        <a id="6651800" scrollto-destination="6651800"></a>
<div content-id="DSU001TB2" class="table-modal"><div class="table-wrap"><div id="DSU001TB2" class="table-wrap-title" data-id="DSU001TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Number of annotations assigned by each Sma3s module</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Dataset </th><th>Module </th><th>Number of annotations </th><th>% </th></tr></thead><tbody><tr><td rowspan="3">Random </td><td>M1 </td><td>468.45 ± 164.25 </td><td>53.76 </td></tr><tr><td>M2 </td><td>171.70 ± 45.12 </td><td>19.70 </td></tr><tr><td>M3 </td><td>231.30 ± 80.72 </td><td>26.54 </td></tr><tr><td rowspan="3">DNA arrays </td><td>M1 </td><td>386.5 ± 369.82 </td><td>5.23 </td></tr><tr><td>M2 </td><td>462.5 ± 556.49 </td><td>6.26 </td></tr><tr><td>M3 </td><td>6536.5 ± 4531.85 </td><td>88.50 </td></tr></tbody></table></div><div content-id="DSU001TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Dataset </th><th>Module </th><th>Number of annotations </th><th>% </th></tr></thead><tbody><tr><td rowspan="3">Random </td><td>M1 </td><td>468.45 ± 164.25 </td><td>53.76 </td></tr><tr><td>M2 </td><td>171.70 ± 45.12 </td><td>19.70 </td></tr><tr><td>M3 </td><td>231.30 ± 80.72 </td><td>26.54 </td></tr><tr><td rowspan="3">DNA arrays </td><td>M1 </td><td>386.5 ± 369.82 </td><td>5.23 </td></tr><tr><td>M2 </td><td>462.5 ± 556.49 </td><td>6.26 </td></tr><tr><td>M3 </td><td>6536.5 ± 4531.85 </td><td>88.50 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn2"><div class="fn"><p>The average number of annotations identified by Sma3s is shown for the random and DNA arrays datasets, together with the corresponding standard deviation. Note that the percentages for each dataset group sum 100%, since Sma3s has applied its three modules consecutively (default configuration).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6651800" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU001TB2" class="table-wrap-title" data-id="DSU001TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Number of annotations assigned by each Sma3s module</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Dataset </th><th>Module </th><th>Number of annotations </th><th>% </th></tr></thead><tbody><tr><td rowspan="3">Random </td><td>M1 </td><td>468.45 ± 164.25 </td><td>53.76 </td></tr><tr><td>M2 </td><td>171.70 ± 45.12 </td><td>19.70 </td></tr><tr><td>M3 </td><td>231.30 ± 80.72 </td><td>26.54 </td></tr><tr><td rowspan="3">DNA arrays </td><td>M1 </td><td>386.5 ± 369.82 </td><td>5.23 </td></tr><tr><td>M2 </td><td>462.5 ± 556.49 </td><td>6.26 </td></tr><tr><td>M3 </td><td>6536.5 ± 4531.85 </td><td>88.50 </td></tr></tbody></table></div><div content-id="DSU001TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Dataset </th><th>Module </th><th>Number of annotations </th><th>% </th></tr></thead><tbody><tr><td rowspan="3">Random </td><td>M1 </td><td>468.45 ± 164.25 </td><td>53.76 </td></tr><tr><td>M2 </td><td>171.70 ± 45.12 </td><td>19.70 </td></tr><tr><td>M3 </td><td>231.30 ± 80.72 </td><td>26.54 </td></tr><tr><td rowspan="3">DNA arrays </td><td>M1 </td><td>386.5 ± 369.82 </td><td>5.23 </td></tr><tr><td>M2 </td><td>462.5 ± 556.49 </td><td>6.26 </td></tr><tr><td>M3 </td><td>6536.5 ± 4531.85 </td><td>88.50 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn2"><div class="fn"><p>The average number of annotations identified by Sma3s is shown for the random and DNA arrays datasets, together with the corresponding standard deviation. Note that the percentages for each dataset group sum 100%, since Sma3s has applied its three modules consecutively (default configuration).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6651800" target="_blank">View Large</a></div></div><p>M1 and M2 modules provide highly accurate results since annotations are obtained from verified orthologues with high degrees of identity between query and donor sequences. However, as both modules employ the traditional Top-BLAST approach, each query sequence is annotated from only one putative orthologue. In contrast, the M3 module typically extracts annotations from several homologous sequences identified in the BLAST report. To better compare the relative benefits of the different approaches used by Sma3s, we compared the annotation of our random datasets using the three modules as described above (M123) against annotation using Module 3 alone (M3). In the latter case, all sequences are annotated by Module 3, not just the sequences unannotated by Modules 1 and 2. Figure <a reveal-id="DSU001F4" data-open="DSU001F4" class="link link-reveal link-table xref-fig">4</a> shows that the mean specificity and sensitivity of the M3 module were slightly lower than the full Sma3s analysis, with the REQ value increasing from 0.19 to 0.32. However, M3 obtained more annotations (Fig. <a reveal-id="DSU001F4" data-open="DSU001F4" class="link link-reveal link-table xref-fig">4</a>, column TC) due to its ability to extract annotations from multiple homologues. M3 accuracy was also compared with combined M1 and M2 modules (M12), which represent more conventional annotation algorithms. M12 obtained higher specificity (0.97) since it annotates only using very similar sequences (Fig. <a reveal-id="DSU001F4" data-open="DSU001F4" class="link link-reveal link-table xref-fig">4</a>). However, sensitivity is rather less (0.47) with only 64% of dataset sequences annotated. This results in an REQ value of 0.71 for M12 versus the 0.32 for M3. Several examples illustrate this additional advantage of Sma3s M3 as a standalone annotation tool. </p>                        <a id="6651802" scrollto-destination="6651802"></a>
<div id="DSU001F4" data-id="DSU001F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu001/2/m_dsu00104.png?Expires=2147483647&Signature=vwPhIOylWg4lBjgzOJzjHhWpRkdFA0ybG1ha9dS18-z0JTNoWlHJjS~m7l-zXxwEbkBgSdd-k7tZoUnQQ4pIngFiavDN9CIR8aqLI9--dwm81uG5c7oOBhdkaKOLbalhwYE6RfiBswcxAegElwxmdGRfllTDFkHZKjziuIT6Le~FaT3FxyWc117v6484Gjl1UbUawICfRMo5DyDkyincnGQknF5r-NEmtMmpnnQmsehVlE4IojqDJIcPlnAY4vXmFvBrhYqiUIyzQ208VDyczcFEFTzFWRnUvwk7jQMuu3PLVyYgeQQ5agtAK8-G87PWYCClmOy05slKPCOI~0uNWg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of Sma3s performance using all modules (M123) or the M3 module alone. Corresponding sensitivity (Sn), specificity (Sp), sequence coverage (SC) and term coverage (TC) values appear above the bars. This figure appears in colour in the online version of DNA Research." path-from-xml="dsu00104.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6651802/dsu00104.tiff" path-from-xml="dsu00104.tiff" target="_blank">View large</a><a section="6651802" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu001/2/dsu00104.png?Expires=2147483647&Signature=nZEgM4pebknUkX6UEi~eZvnrvVhSrPpJf4lMuSUEoYBa4NDyhaDRfIJOBYbJvCTAIZxM4dvEUW-riYIpsluEDnrCKhX31E9yOPRzce4eFYNF7pLeEdMmOfZMOH50mK9XsmPXFNsDW7Wh61dLBvWpIdENVmRYnYgpVo0W2xTdrFIFX1ZzA32-Roda-PRqAEK47S60NyMW7vJZinSiAI5udHrm5~JVWfG~MlgT2vg1DNFnD1mZXs2E-hti4FAe9cq0gSZg52xSLM57ci1wCIK1eYBZ9mq2pHyPcbEpfep6ppsT1XUvNsXf2MzwgWb1EWUIWgvaDurIIJNLisrsjx7ilA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6651802&ar=444964&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00104.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of Sma3s performance using all modules (M123) or the M3 module alone. Corresponding sensitivity (Sn), specificity (Sp), sequence coverage (SC) and term coverage (TC) values appear above the bars. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><div content-id="DSU001F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu001/2/m_dsu00104.png?Expires=2147483647&Signature=vwPhIOylWg4lBjgzOJzjHhWpRkdFA0ybG1ha9dS18-z0JTNoWlHJjS~m7l-zXxwEbkBgSdd-k7tZoUnQQ4pIngFiavDN9CIR8aqLI9--dwm81uG5c7oOBhdkaKOLbalhwYE6RfiBswcxAegElwxmdGRfllTDFkHZKjziuIT6Le~FaT3FxyWc117v6484Gjl1UbUawICfRMo5DyDkyincnGQknF5r-NEmtMmpnnQmsehVlE4IojqDJIcPlnAY4vXmFvBrhYqiUIyzQ208VDyczcFEFTzFWRnUvwk7jQMuu3PLVyYgeQQ5agtAK8-G87PWYCClmOy05slKPCOI~0uNWg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of Sma3s performance using all modules (M123) or the M3 module alone. Corresponding sensitivity (Sn), specificity (Sp), sequence coverage (SC) and term coverage (TC) values appear above the bars. This figure appears in colour in the online version of DNA Research." path-from-xml="dsu00104.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6651802/dsu00104.tiff" path-from-xml="dsu00104.tiff" target="_blank">View large</a><a section="6651802" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu001/2/dsu00104.png?Expires=2147483647&Signature=nZEgM4pebknUkX6UEi~eZvnrvVhSrPpJf4lMuSUEoYBa4NDyhaDRfIJOBYbJvCTAIZxM4dvEUW-riYIpsluEDnrCKhX31E9yOPRzce4eFYNF7pLeEdMmOfZMOH50mK9XsmPXFNsDW7Wh61dLBvWpIdENVmRYnYgpVo0W2xTdrFIFX1ZzA32-Roda-PRqAEK47S60NyMW7vJZinSiAI5udHrm5~JVWfG~MlgT2vg1DNFnD1mZXs2E-hti4FAe9cq0gSZg52xSLM57ci1wCIK1eYBZ9mq2pHyPcbEpfep6ppsT1XUvNsXf2MzwgWb1EWUIWgvaDurIIJNLisrsjx7ilA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6651802&ar=444964&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00104.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of Sma3s performance using all modules (M123) or the M3 module alone. Corresponding sensitivity (Sn), specificity (Sp), sequence coverage (SC) and term coverage (TC) values appear above the bars. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><p>The best BLAST hit of the pig Qil1 protein (QIL1_PIG) in the mammalian dataset is the primate homologue (QIL1_MACFA). Because this protein entry does not contain any of the four annotation types used by Sma3s, pig Qil1 was not annotated by M1. However, the M3 module was able to annotate pig Qil1 localization using the GO term ‘Mitochondrion’, obtained from a more distant Qil1 mouse homologue (QIL1_MOUSE).</p><p>The Sma3s M3 module could be used even in cases where incomplete annotations are present in Swiss-Prot. For example, when M1 or M2 Sma3s modules are used with invertebrate datasets, the U3-lycotoxin-Ls1a protein from the wolf spider (TX308_LYCSI) only generates the GO term ‘extracellular region’. In contrast, the M3 module annotates the sequence with two additional GO terms: ‘pathogenesis’ and ‘calcium channel inhibitor activity’. Since this protein belongs to the spider toxin CSTX superfamily, which blocks mammalian neuronal calcium channels,<sup><a reveal-id="DSU001C26" data-open="DSU001C26" class="link link-ref link-reveal xref-bibr">26</a></sup> these terms should be present in the annotation of this protein.</p><p>Another example is the SfsA protein (SFSA_PSEF5) from <em>Pseudomonas fluorescens</em>. SfsA was annotated in bacterial datasets with the Swiss-Prot keyword ‘Complete proteome’ and no GO terms. However, M3 predicted the following Keywords and GO terms: DNA binding and carbohydrate metabolic process. These terms seem valid because the bacterial SfsA family includes sugar fermentation stimulation proteins that contain a helix-turn-helix motif that probably binds DNA at its C-terminus.<sup><a reveal-id="DSU001C27" data-open="DSU001C27" class="link link-ref link-reveal xref-bibr">27</a></sup></p>                            <h3 scrollto-destination=6651806 id="6651806" class="section-title" data-legacy-id=s3d>3.4. Accurate annotation of DNA array sequence sets</h3>
<p>To test Sma3s with an experimental set of sequences, we chose collections belonging to two different Affymetrix DNA arrays (one mouse and one <em>Arabidopsis</em>). The real annotations for these datasets were extracted from the Affymetrix web server (March, 2012) as GO terms linked to the sequences, or as GO terms extracted from Swiss-Prot IDs linked to the sequences. Sma3s was used to annotate these arrays using default parameters, and accuracy measured using the reference annotations. We compared these results with the traditional Top-BLAST method, and the widely used Blast2GO tool with default annotation settings (Fig. <a reveal-id="DSU001F5" data-open="DSU001F5" class="link link-reveal link-table xref-fig">5</a>). </p>                        <a id="6651808" scrollto-destination="6651808"></a>
<div id="DSU001F5" data-id="DSU001F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu001/2/m_dsu00105.png?Expires=2147483647&Signature=WaWhvx-RAF~dIOGKP2vKToQDIUl90b4C5H7e~vm9Z5hT1FOFpSYOdQUsJBRfrTw-khvMerT~xDbAu0jlnhfUZ3ENRkRIsZzRDj1dDNgqMFDm4rCK4TvsYMN22PGKMKxEXj3eVPIb8ejBVLimXgxOxWctWrYVK3-y9OGsDU9LlPKZFDH8sqIBosE6A78hUKpcHZupRBIm9qoLtYxNpj626yjty5Vr2-QQyv9JY7f8eaCUd79dbucSSzbZa3fOm0Wr43oXlgsOP52NbWVA6DZdixfsKikuh6jdvr0Z-eOEmhwIvpIr5FCQIMuX-5YXO5jHsV9-HEx3iqLGmTtMySdw1A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Annotation of two DNA arrays using Sma3s, Blast2GO and Top-BLAST. The annotation prediction results of mouse and Arabidopsis sequences are shown from Sma3s with all modules, Sma3s with the M3 module only, Blast2GO and Top-BLAST. The corresponding values appear above the bars. Sn, sensitivity; Sp, specificity; SC, sequence coverage; TC, term coverage. This figure appears in colour in the online version of DNA Research." path-from-xml="dsu00105.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6651808/dsu00105.tiff" path-from-xml="dsu00105.tiff" target="_blank">View large</a><a section="6651808" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu001/2/dsu00105.png?Expires=2147483647&Signature=loKa7dbvO6lQkdDiEks51u6PNabiNsv75kTi-ZMn1k-fMezvPgeqjXhaxTWaGQ-I~qHkabr2n-VEjqfRGpgjkKfgx~-CTYfAlq6z1nDMg~mlOj4sea3SBop9TfPUwgPFe03wgUdDHvb1eH-2rbtDFTswVxLU7w~uDf7cBVSav7L~TxCDM~PsSZvgEIjBIlkRiHqFAREdOOW2tRf3SXacGVJFQuFNtE8HZrmiRiUw8YreU-C5qregUVXk3tHeeT9MCrrXiqCgudGbOItb-w8uias~ywD63VflG4B4U-qQgezziLSTY68YGmgvVHC7wUMJacDGL401SljMXZxUbshDkg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6651808&ar=444964&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00105.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Annotation of two DNA arrays using Sma3s, Blast2GO and Top-BLAST. The annotation prediction results of mouse and Arabidopsis sequences are shown from Sma3s with all modules, Sma3s with the M3 module only, Blast2GO and Top-BLAST. The corresponding values appear above the bars. Sn, sensitivity; Sp, specificity; SC, sequence coverage; TC, term coverage. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><div content-id="DSU001F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu001/2/m_dsu00105.png?Expires=2147483647&Signature=WaWhvx-RAF~dIOGKP2vKToQDIUl90b4C5H7e~vm9Z5hT1FOFpSYOdQUsJBRfrTw-khvMerT~xDbAu0jlnhfUZ3ENRkRIsZzRDj1dDNgqMFDm4rCK4TvsYMN22PGKMKxEXj3eVPIb8ejBVLimXgxOxWctWrYVK3-y9OGsDU9LlPKZFDH8sqIBosE6A78hUKpcHZupRBIm9qoLtYxNpj626yjty5Vr2-QQyv9JY7f8eaCUd79dbucSSzbZa3fOm0Wr43oXlgsOP52NbWVA6DZdixfsKikuh6jdvr0Z-eOEmhwIvpIr5FCQIMuX-5YXO5jHsV9-HEx3iqLGmTtMySdw1A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Annotation of two DNA arrays using Sma3s, Blast2GO and Top-BLAST. The annotation prediction results of mouse and Arabidopsis sequences are shown from Sma3s with all modules, Sma3s with the M3 module only, Blast2GO and Top-BLAST. The corresponding values appear above the bars. Sn, sensitivity; Sp, specificity; SC, sequence coverage; TC, term coverage. This figure appears in colour in the online version of DNA Research." path-from-xml="dsu00105.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6651808/dsu00105.tiff" path-from-xml="dsu00105.tiff" target="_blank">View large</a><a section="6651808" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu001/2/dsu00105.png?Expires=2147483647&Signature=loKa7dbvO6lQkdDiEks51u6PNabiNsv75kTi-ZMn1k-fMezvPgeqjXhaxTWaGQ-I~qHkabr2n-VEjqfRGpgjkKfgx~-CTYfAlq6z1nDMg~mlOj4sea3SBop9TfPUwgPFe03wgUdDHvb1eH-2rbtDFTswVxLU7w~uDf7cBVSav7L~TxCDM~PsSZvgEIjBIlkRiHqFAREdOOW2tRf3SXacGVJFQuFNtE8HZrmiRiUw8YreU-C5qregUVXk3tHeeT9MCrrXiqCgudGbOItb-w8uias~ywD63VflG4B4U-qQgezziLSTY68YGmgvVHC7wUMJacDGL401SljMXZxUbshDkg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6651808&ar=444964&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00105.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Annotation of two DNA arrays using Sma3s, Blast2GO and Top-BLAST. The annotation prediction results of mouse and Arabidopsis sequences are shown from Sma3s with all modules, Sma3s with the M3 module only, Blast2GO and Top-BLAST. The corresponding values appear above the bars. Sn, sensitivity; Sp, specificity; SC, sequence coverage; TC, term coverage. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><p><em>Arabidopsis</em> and mouse array analyses were performed with either the Swiss-Prot (Fig. <a reveal-id="DSU001F5" data-open="DSU001F5" class="link link-reveal link-table xref-fig">5</a>a) or Affymetrix (Fig. <a reveal-id="DSU001F5" data-open="DSU001F5" class="link link-reveal link-table xref-fig">5</a>b) annotations as references. In the case of <em>Arabidopsis</em>, Sma3s showed the best accuracy (as measured by REQ, sensitivity and specificity), although Top-BLAST provided a better sequence and TC. However, there were striking differences between the Swiss-Prot and Affymetrix annotation sources. We observed a much bigger difference in REQ values between Sma3s (M123 and M3 alone) and the other methods with Swiss-Prot annotations.</p><p>In the case of the mouse dataset, Sma3s provided significantly better sensitivity albeit with slightly poorer specificity than Top-BLAST when using Swiss-Prot as an annotation source (Fig. <a reveal-id="DSU001F5" data-open="DSU001F5" class="link link-reveal link-table xref-fig">5</a>c). A similar pattern occurred when using Affymetrix sources (Fig. <a reveal-id="DSU001F5" data-open="DSU001F5" class="link link-reveal link-table xref-fig">5</a>d), but the differences were more marginal. In contrast to the <em>Arabidopsis</em> collection, Sma3s generated the highest TC values for mouse sequences with both annotation sources. Our results suggest that both the full and M3-only implementations of Sma3s analysis compare favourably with commonly used annotation tools, with equal or better REQ values in almost all cases. However, it is clear that the annotation results obtained were very sensitive to both the query dataset and the choice of annotation source.</p>                            <h3 scrollto-destination=6651811 id="6651811" class="section-title" data-legacy-id=s3e>3.5. Sma3s is fast and has low-memory usage</h3>
<p>In the post-genomic era, it is critical to have annotation tools that can be easily integrated into sequence analysis pipelines and applied quickly to large datasets. Therefore, it is important that annotation analyses run efficiently without excessive resource demands or sequence limitations. To this end, we estimated the speed of Sma3s analysis using the modest computer hardware (see Materials and Methods for details). We tested run times and memory usage using three datasets with progressively increasing sizes (Fig. <a reveal-id="DSU001F6" data-open="DSU001F6" class="link link-reveal link-table xref-fig">6</a>). Measurements were taken separately for the BLAST search and sequence annotation steps. Sma3s processing times increased in proportion to the number of sequences analysed (Fig. <a reveal-id="DSU001F6" data-open="DSU001F6" class="link link-reveal link-table xref-fig">6</a>a). Times were highly dependent on the initial BLAST step which increased in proportion to the sequence number. However, Sma3s annotation times were much lower and barely affected by increasing dataset sizes. Significantly, Sma3s run times were two orders of magnitude shorter than those of the Blast2GO method. </p>                        <a id="6651813" scrollto-destination="6651813"></a>
<div id="DSU001F6" data-id="DSU001F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu001/2/m_dsu00106.png?Expires=2147483647&Signature=kkLBjuP1aGYXcop3RUZsag3-kPb1otz0blbRJwcXB9vCfD34lTmDS94vYpxngYktaA1YMBmE5iwPS-dBdgke09TtohVndIwewjvtjtH6TbAa0dn7wxuQgh2JD5LfEBwjsZTbwX6AhQJgV8X6pWkl7AQ4J2BR6i07b7zl6sJ7w2Dw3Yvxg5XIRK9PfJPU6mx5h24vBpr4O8etdhZUbayUy-3ITzgj-aIPa8pDZomXkcNhjZNtNO8Wp-FNQD~t-5yhE-wMAHg2aiM71-ZRBGIKHKZFMMhketK1-xREdxBrNxp28uyd-4bikU3mEcAGdbjrNu2mewwlL79~qK6AZylWZg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Run times and memory usage of Sma3s versus Blast2GO. Tests were performed using datasets with different numbers of sequences (500, 1000 and 2000). The different steps in both tools are shown as separate bars. Blast2GO did not report results with datasets &gt;1000 sequences due to memory problems. (a) Sma3s and Blast2GO run times are shown in the first and second bars, respectively. (b) Sma3s and Blast2GO memory usage are shown in the first and second bars, respectively. Sma3s includes Blast and annotation (term assignment to query sequences) steps, and Blast2GO additionally includes the mapping step (term extraction from the obtained hits). This figure appears in colour in the online version of DNA Research." path-from-xml="dsu00106.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6651813/dsu00106.tiff" path-from-xml="dsu00106.tiff" target="_blank">View large</a><a section="6651813" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu001/2/dsu00106.png?Expires=2147483647&Signature=pxrSuGW0XX3qKXeeexyECnrDIgCSejQaXmxp1fzjSz~zranrTa6LYBTjoyHi7FmCBin6CqLK5UturElW5lIOUEKL6qPfcQCit5cEihCYPW7s1ENyTVcO3Rk6njbd1ROCu11D08e5cUP6cDyMafFiT6arhpku9KoeWUEoiFtrPb~LVEUxmnfdhMybRdqCrQOb9BTZSS4s~DplSYbCeoJUjQcZG6Qh6gRCpo1jhpTXVHhlFTbCEtsW8oZJF--08XU3DAHGJKlUteqqmMCXmmvgYE1KaKh5pw0k2vQu-mN8OCAA~QjODywdPxMroZEFkE76eku03O6s8WT~qEKLUmv24w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6651813&ar=444964&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00106.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Run times and memory usage of Sma3s versus Blast2GO. Tests were performed using datasets with different numbers of sequences (500, 1000 and 2000). The different steps in both tools are shown as separate bars. Blast2GO did not report results with datasets &gt;1000 sequences due to memory problems. (a) Sma3s and Blast2GO run times are shown in the first and second bars, respectively. (b) Sma3s and Blast2GO memory usage are shown in the first and second bars, respectively. Sma3s includes Blast and annotation (term assignment to query sequences) steps, and Blast2GO additionally includes the mapping step (term extraction from the obtained hits). This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><div content-id="DSU001F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu001/2/m_dsu00106.png?Expires=2147483647&Signature=kkLBjuP1aGYXcop3RUZsag3-kPb1otz0blbRJwcXB9vCfD34lTmDS94vYpxngYktaA1YMBmE5iwPS-dBdgke09TtohVndIwewjvtjtH6TbAa0dn7wxuQgh2JD5LfEBwjsZTbwX6AhQJgV8X6pWkl7AQ4J2BR6i07b7zl6sJ7w2Dw3Yvxg5XIRK9PfJPU6mx5h24vBpr4O8etdhZUbayUy-3ITzgj-aIPa8pDZomXkcNhjZNtNO8Wp-FNQD~t-5yhE-wMAHg2aiM71-ZRBGIKHKZFMMhketK1-xREdxBrNxp28uyd-4bikU3mEcAGdbjrNu2mewwlL79~qK6AZylWZg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Run times and memory usage of Sma3s versus Blast2GO. Tests were performed using datasets with different numbers of sequences (500, 1000 and 2000). The different steps in both tools are shown as separate bars. Blast2GO did not report results with datasets &gt;1000 sequences due to memory problems. (a) Sma3s and Blast2GO run times are shown in the first and second bars, respectively. (b) Sma3s and Blast2GO memory usage are shown in the first and second bars, respectively. Sma3s includes Blast and annotation (term assignment to query sequences) steps, and Blast2GO additionally includes the mapping step (term extraction from the obtained hits). This figure appears in colour in the online version of DNA Research." path-from-xml="dsu00106.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6651813/dsu00106.tiff" path-from-xml="dsu00106.tiff" target="_blank">View large</a><a section="6651813" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu001/2/dsu00106.png?Expires=2147483647&Signature=pxrSuGW0XX3qKXeeexyECnrDIgCSejQaXmxp1fzjSz~zranrTa6LYBTjoyHi7FmCBin6CqLK5UturElW5lIOUEKL6qPfcQCit5cEihCYPW7s1ENyTVcO3Rk6njbd1ROCu11D08e5cUP6cDyMafFiT6arhpku9KoeWUEoiFtrPb~LVEUxmnfdhMybRdqCrQOb9BTZSS4s~DplSYbCeoJUjQcZG6Qh6gRCpo1jhpTXVHhlFTbCEtsW8oZJF--08XU3DAHGJKlUteqqmMCXmmvgYE1KaKh5pw0k2vQu-mN8OCAA~QjODywdPxMroZEFkE76eku03O6s8WT~qEKLUmv24w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6651813&ar=444964&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu00106.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Run times and memory usage of Sma3s versus Blast2GO. Tests were performed using datasets with different numbers of sequences (500, 1000 and 2000). The different steps in both tools are shown as separate bars. Blast2GO did not report results with datasets &gt;1000 sequences due to memory problems. (a) Sma3s and Blast2GO run times are shown in the first and second bars, respectively. (b) Sma3s and Blast2GO memory usage are shown in the first and second bars, respectively. Sma3s includes Blast and annotation (term assignment to query sequences) steps, and Blast2GO additionally includes the mapping step (term extraction from the obtained hits). This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><p>Memory usage was also measured using the same datasets and methods (Fig. <a reveal-id="DSU001F6" data-open="DSU001F6" class="link link-reveal link-table xref-fig">6</a>b). Sma3s had an essentially constant memory usage in its two steps despite the sequence number increase, in contrast to the Blast2Go method. Our results show that Sma3s is able to efficiently annotate large sequence datasets using modest computational resources.</p>                            <h2 scrollto-destination=6651815 id="6651815" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>Sequencing projects produce hundreds of thousands of new biological sequences that need to be quickly and accurately annotated since this information constitutes the basis for subsequent analysis, such as pathway design or genomics and transcriptomics studies. Therefore, fast and user friendly methods are needed to support sequence annotation at this stage.</p><p>Sma3s uses three consecutive modules: M1 (which searches for very similar sequences in the database), M2 (which searches for orthologues) and M3 (which extracts information from several homologous sequences). The novel approach used by the M3 module selects several homologues for each query sequence, using a modified Rost equation, which takes into account both alignment length and percentage identity. Annotations from all the non-redundant sequences are pooled, but only annotations that are significantly enriched versus the expected background frequency are assigned to the query sequence. This has the advantage that useful annotation information present in several homologues can be extracted, rather than only from the sequence with the highest homology.</p><p>Our tests have shown that Sma3s is a fast annotation method requiring minimal human supervision and computational resources. Mean Sma3s accuracy using different annotation types (GO, Swiss-Prot keywords and pathways, and InterPro) was higher than 0.8 (except for the invertebrate dataset), and was even higher (0.9) for specific datasets such as bacteria. In addition, REQ values were consistently very low. InterPro annotation was highly specific and sensitive (see Fig. <a reveal-id="DSU001F3" data-open="DSU001F3" class="link link-reveal link-table xref-fig">3</a>), which can be explained by the fact that current InterPro annotation is strongly focused on sequence similarity, which has been shown to be effective for protein function prediction 26. On the other hand, it is difficult to judge Sma3s Swiss-Prot Pathway annotation given the limited coverage of this category in the UniProt database. Importantly, both InterPro and Pathway use short and controlled vocabularies with low term diversification.</p><p>Some annotation methods can assign sentences derived from the database description line 15. Sma3s can also assign this descriptor (data not shown). However, the accuracy of these non-standard annotations is difficult to measure, whereas controlled terms such as GO or Keywords are more useful for experiments using massive datasets. Sma3s can be easily adapted to use different annotation classes, and each term predicted by Sma3s can be assigned a corresponding probability value. For example, Sma3s also includes annotations for the most probable gene names for each annotated sequence (data not shown), which can be useful for non-specialized users who only want to rapidly identify their sequences. Furthermore, Sma3s could be used with other annotation sources since the annotation assignment step can easily be disengaged from the main algorithm and used with other databases. For example, the annotation component could be extended to use data from the BioMart project, which links annotations from heterogeneous databases to UniProt identifiers. If Sma3s was converted to a web service,<sup><a reveal-id="DSU001C28" data-open="DSU001C28" class="link link-ref link-reveal xref-bibr">28</a></sup> it could be directed to use BioMart public web services as an annotation source. This would also make Sma3s more versatile by dividing Blast and annotation steps into separate services.</p>                            <h3 scrollto-destination=6651820 id="6651820" class="section-title" data-legacy-id=s4a>4.1. The reference dataset and the BLAST results influence annotation accuracy</h3>
<p>We demonstrated the accuracy of Sma3s with tests using different random datasets (Fig. <a reveal-id="DSU001F2" data-open="DSU001F2" class="link link-reveal link-table xref-fig">2</a>). Moreover, Sma3s outperformed other widely used annotation methods when annotating DNA array datasets (Fig. <a reveal-id="DSU001F5" data-open="DSU001F5" class="link link-reveal link-table xref-fig">5</a>). The only exception was the mouse dataset using Swiss-Prot as a reference. The annotation accuracy of this dataset is strongly dependent on reference annotations, which in this case came from Affymetrix. These annotations are assigned using NetAffx, which is based on homology transfer from heterogeneous databases, including Swiss-Prot, but also Unigene and LocusLink 27. Furthermore, these annotations depend on the external database entries associated with individual Affymetrix sequences, which have been shown to contain frequent inaccuracies 28. Thus, selection of a suitably high-quality reference annotation set seems to be an important factor for testing this or any other method. Nevertheless, Sma3s produced the best results in virtually all cases.</p><p>Further evidence for the idea that database annotation influences annotation accuracy is presented in Table <a reveal-id="DSU001TB1" data-open="DSU001TB1" class="link link-reveal link-table xref-fig">1</a>, where the well-annotated Swiss-Prot division of UniProt provided the best accuracy (including specificity and sensitivity), while TrEMBL provided the lowest values. UniProt is the most widely used protein database since it contains rich information on each sequence. UniProt is comprised of Swiss-Prot, a curated section reviewed by skilled annotators and TrEMBL, an unreviewed section containing additional sequences that can be moved to Swiss-Prot once they have been manually reviewed. We have found that the best annotation accuracy is obtained when Swiss-Prot is used as the sole reference database (Table <a reveal-id="DSU001TB1" data-open="DSU001TB1" class="link link-reveal link-table xref-fig">1</a>). Conversely, accuracy was reduced when using either TrEMBL or the complete UniProt database, mainly due to lower sensitivity. These lower values are expected partly because the test datasets were extracted from Swiss-Prot, and partly because TrEMBL is a much larger database. Thus, when UniProt or TrEMBL are used as reference sources, a higher proportion of Sma3s annotations are TrEMBL-derived, which consequently lowers the average annotation accuracy.</p><p>Similarly, it is important to highlight that results were also dependent on the quality of annotated reference sequences for each taxonomic group. For example, Sma3s annotation of random invertebrate sequences was of a much lower quality than those of bacterial sequences. The main difference between the two Uniprot datasets is in the number of sequences they contain: 24 050 invertebrate sequences versus 326 570 bacterial sequences. In order for Module 3 to assign an annotation, it has to be statistically enriched within the homologous sequences. The probability of finding enriched annotations is more likely with the higher numbers of candidate sequences available in the bacterial dataset. In fact, we have found that a combination of TrEMBL and SWISS-PROT databases increases sensitivity when annotating weakly conserved genomes (unpublished observation). However, this approach results in a higher rate of false positives.</p><p>Sequence length may be an important factor for successful annotation. For example, short sequences occur most frequently in invertebrates and to a lesser extent in plants (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1a</a>). These sequences are rarely annotated due to the low alignment scores they produce. The M1 module, which requires very high levels of similarity, is typically the most effective Sma3s component for annotation of these sequences (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1b</a>). This effect is likely to be enhanced by the higher diversity of the invertebrate dataset versus the bacterial one.</p><p>Another key factor for successful annotation is the initial BLAST search. In order for a sequence to be annotated by any of the three Sma3s modules, the initial BLAST must identify sequence alignments. For example, of the 30% of mouse array sequences which were not annotated by Sma3s (SC = 0.7 in Fig. <a reveal-id="DSU001F5" data-open="DSU001F5" class="link link-reveal link-table xref-fig">5</a>), 86% lacked BLAST results, making annotation impossible. In the case of Arabidopsis array sequences, 40% lacked BLAST results thus limiting annotation coverage to a maximum of 60%. This dependency on BLAST results is shared with most annotation methods, including AutoFact 11, Blast2GO 9 and GOtcha 12. In the case of Sma3s, the effect of the BLAST search is especially important given that Sma3s was able to annotate 90% of array sequences with BLAST alignments, largely due to the novel M3 module (Table <a reveal-id="DSU001TB2" data-open="DSU001TB2" class="link link-reveal link-table xref-fig">2</a>).</p>                            <h3 scrollto-destination=6651826 id="6651826" class="section-title" data-legacy-id=s4b>4.2. The combined use of the three Sma3s modules generates accurate annotation, with enhanced coverage from the M3 module</h3>
<p>We have shown that Sma3s can annotate different dataset types using its three module structure. Modules M1 and M2 are able to identify closely related database sequences and extract specific annotations, albeit with low overall coverage (TC in Figs <a reveal-id="DSU001F4" data-open="DSU001F4" class="link link-reveal link-table xref-fig">4</a> and <a reveal-id="DSU001F5" data-open="DSU001F5" class="link link-reveal link-table xref-fig">5</a>). Thus, modules M1 and M2 provide high-quality annotations (with lower REQ values), increasing the overall sensitivity and specificity of Sma3s (Sn and Sp in Fig. <a reveal-id="DSU001F4" data-open="DSU001F4" class="link link-reveal link-table xref-fig">4</a>).</p><p>Our findings suggest that the M3 module can increase SC in some cases, especially where close orthologues lack database annotation. The initial BLAST result can be reused by the M3 module to obtain new annotations for sequences previously annotated by M1 or M2. M3 annotation is broader, allowing new terms to be found (Fig. <a reveal-id="DSU001F4" data-open="DSU001F4" class="link link-reveal link-table xref-fig">4</a>) but with lower specificity, thus increasing the chance of generating false positives. For most applications, using the three Sma3s modules sequentially balance the higher accuracy of Modules 1 and 2 with the identification of useful annotations from Module 3 for otherwise unannotated sequences.</p><p>Our results demonstrate that the Sma3s M3 module is able to generate new annotations for sequences that were unannotated in their original databases. Thus, the M3 module may be valuable as a database curation tool, adding new annotations to existing public databases. However, it is important to take into account that this module is likely to generate annotations with lower specificity (Sp in Figs <a reveal-id="DSU001F4" data-open="DSU001F4" class="link link-reveal link-table xref-fig">4</a> and <a reveal-id="DSU001F5" data-open="DSU001F5" class="link link-reveal link-table xref-fig">5</a>). Ideally, users would be able to choose between high sensitivity and high specificity according to the goal of the experiment.</p>                            <h3 scrollto-destination=6651830 id="6651830" class="section-title" data-legacy-id=s4c>4.3. Sma3s has low computing requirements</h3>
<p>The Sma3s algorithm comprises two different basic steps: similarity searching by BLAST and annotation by different strategies in each of the three modules. For high-throughput analyses, the BLAST itself accounts for most of the calculation time. Result execution times (Fig. <a reveal-id="DSU001F6" data-open="DSU001F6" class="link link-reveal link-table xref-fig">6</a>) demonstrate that parallel BLAST searches could accelerate the process without affecting the rest of the algorithm. In fact, both Sma3s and Blast2GO can use separately generated BLAST reports, which may allow accelerated analysis if the user was to execute similarity searches in parallel. However, the speed advantage of Sma3s over Blast2GO is due to more than the speed of the BLAST search. Another reason for the speed of Sma3s is that it uses Swiss-Prot, a relatively small high-quality database, which itself increases annotation accuracy (Figs <a reveal-id="DSU001F2" data-open="DSU001F2" class="link link-reveal link-table xref-fig">2</a>, <a reveal-id="DSU001F3" data-open="DSU001F3" class="link link-reveal link-table xref-fig">3</a> and <a reveal-id="DSU001F5" data-open="DSU001F5" class="link link-reveal link-table xref-fig">5</a>). These results together allow for a faster and more accurate annotation process when analysing large datasets. Thus, Sma3s can annotate a set of 20 000 sequences in ∼3 h, without human intervention. Finally, memory usage is low (Fig. <a reveal-id="DSU001F6" data-open="DSU001F6" class="link link-reveal link-table xref-fig">6</a>b), alleviating the need for high performance computing facilities.</p><p>In conclusion, Sma3s is a fast and accurate method for annotating massive nucleotide or amino acid datasets in a way that is readily accessible for non-expert users. In fact, Sma3s has already been used in the functional annotation of both the olive<sup><a reveal-id="DSU001C29" data-open="DSU001C29" class="link link-ref link-reveal xref-bibr">29</a></sup> and the maritime pine<sup><a reveal-id="DSU001C30" data-open="DSU001C30" class="link link-ref link-reveal xref-bibr">30</a></sup> transcriptomes, and is now being used for the annotation of both a new bacterial genome and an EST database of almost 300 plant datasets containing a total of 8.6 million sequences (data not published yet).</p>                            <h2 scrollto-destination=6651833 id="6651833" class="section-title" data-legacy-id=s5>Authors' contributions</h2>
<p>A.M.M. significantly helped in the design computational implementation, and testing of Sma3s algorithm. E.V. and M.G.C. contributed to computational design and rationale. O.T. managed and coordinated the initial stages, including the training. A.J.P.P. conceived the study, carried out the design, the tests and trainings, and wrote the computer program and the manuscript.</p>                            <h2 scrollto-destination=6651835 id="6651835" class="section-title" data-legacy-id=s6>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=6651837 id="6651837" class="section-title" data-legacy-id=s7>Funding</h2>
<p>This work has been partially financed by the National Institute for Bioinformatics (<a class="link link-uri" href="http://www.inab.org">www.inab.org</a>), a platform of Genoma España and the EC project ‘Advancing Clinico-Genomic Trials on Cancer’ (contract no. 026996).</p>                            <h2 scrollto-destination=6651839 id="6651839" class="backacknowledgements-title" >Acknowledgement</h2>
<p>We thank John Pearson for his critical reading of the manuscript.</p>                            <h2 scrollto-destination=6651841 id="6651841" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSU001C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU001C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rost</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Nair</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Wrzeszczynski</div> <div class="given-names">K.O.</div></div>,  <div class="name"><div class="surname">Ofran</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Automatic prediction of protein function</div>, <div class="source">Cell. Mol. Life Sci.</div> , <div class="year">2003</div>, vol. <div class="volume">60</div> (pg. <div class="fpage">2637</div>-<div class="lpage">50</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automatic%20prediction%20of%20protein%20function&amp;author=B.%20Rost&amp;author=J.%20Liu&amp;author=R.%20Nair&amp;author=K.O.%20Wrzeszczynski&amp;author=Y.%20Ofran&amp;publication_year=2003&amp;journal=Cell.%20Mol.%20Life%20Sci.&amp;volume=60&amp;pages=2637-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00018-003-3114-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00018-003-3114-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00018-003-3114-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14685688" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU001C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU001C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Loewenstein</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Raimondo</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Redfern</div> <div class="given-names">O.C.</div></div>, et al. </div><div class="article-title">Protein function annotation by homology-based inference</div>, <div class="source">Genome Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">207</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protein%20function%20annotation%20by%20homology-based%20inference&amp;author=Y.%20Loewenstein&amp;author=D.%20Raimondo&amp;author=O.C.%20Redfern&amp;publication_year=2009&amp;journal=Genome%20Biol.&amp;volume=10&amp;pages=207" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2009-10-2-207" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2009-10-2-207" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2009-10-2-207"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19226439" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU001C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU001C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Abascal</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Valencia</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Automatic annotation of protein function based on family identification</div>, <div class="source">Proteins</div> , <div class="year">2003</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">683</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automatic%20annotation%20of%20protein%20function%20based%20on%20family%20identification&amp;author=F.%20Abascal&amp;author=A.%20Valencia&amp;publication_year=2003&amp;journal=Proteins&amp;volume=53&amp;pages=683-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0134" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0134" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0134"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14579359" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU001C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU001C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Juncker</div> <div class="given-names">A.S.</div></div>,  <div class="name"><div class="surname">Jensen</div> <div class="given-names">L.J.</div></div>,  <div class="name"><div class="surname">Pierleoni</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Sequence-based feature prediction and annotation of proteins</div>, <div class="source">Genome Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">206</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence-based%20feature%20prediction%20and%20annotation%20of%20proteins&amp;author=A.S.%20Juncker&amp;author=L.J.%20Jensen&amp;author=A.%20Pierleoni&amp;publication_year=2009&amp;journal=Genome%20Biol.&amp;volume=10&amp;pages=206" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2009-10-2-206" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2009-10-2-206" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2009-10-2-206"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19226438" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU001C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU001C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moller</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Leser</div> <div class="given-names">U.</div></div>,  <div class="name"><div class="surname">Fleischmann</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Apweiler</div> <div class="given-names">R.</div></div>. </div><div class="article-title">EDITtoTrEMBL: a distributed approach to high-quality automated protein sequence annotation</div>, <div class="source">Bioinformatics</div> , <div class="year">1999</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">219</div>-<div class="lpage">27</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EDITtoTrEMBL%3A%20a%20distributed%20approach%20to%20high-quality%20automated%20protein%20sequence%20annotation&amp;author=S.%20Moller&amp;author=U.%20Leser&amp;author=W.%20Fleischmann&amp;author=R.%20Apweiler&amp;publication_year=1999&amp;journal=Bioinformatics&amp;volume=15&amp;pages=219-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/15.3.219" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F15.3.219" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F15.3.219"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10222409" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU001C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU001C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Carmona-Saez</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Chagoyen</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tirado</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Carazo</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Pascual-Montano</div> <div class="given-names">A.</div></div>. </div><div class="article-title">GENECODIS: a web-based tool for finding significant concurrent annotations in gene lists</div>, <div class="source">Genome Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">R3</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GENECODIS%3A%20a%20web-based%20tool%20for%20finding%20significant%20concurrent%20annotations%20in%20gene%20lists&amp;author=P.%20Carmona-Saez&amp;author=M.%20Chagoyen&amp;author=F.%20Tirado&amp;author=J.M.%20Carazo&amp;author=A.%20Pascual-Montano&amp;publication_year=2007&amp;journal=Genome%20Biol.&amp;volume=8&amp;pages=R3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2007-8-1-r3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2007-8-1-r3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2007-8-1-r3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17204154" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU001C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU001C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rodriguez-Baena</div> <div class="given-names">D.S.</div></div>,  <div class="name"><div class="surname">Perez-Pulido</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">Aguilar-Ruiz</div> <div class="given-names">J.S.</div></div>. </div><div class="article-title">A biclustering algorithm for extracting bit-patterns from binary datasets</div>, <div class="source">Bioinformatics</div> , <div class="year">2011</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">2738</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20biclustering%20algorithm%20for%20extracting%20bit-patterns%20from%20binary%20datasets&amp;author=D.S.%20Rodriguez-Baena&amp;author=A.J.%20Perez-Pulido&amp;author=J.S.%20Aguilar-Ruiz&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=2738-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr464" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr464" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr464"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21824973" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU001C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU001C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Madden</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Schaffer</div> <div class="given-names">A.A.</div></div>, et al. </div><div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3389</div>-<div class="lpage">402</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.F.%20Altschul&amp;author=T.L.%20Madden&amp;author=A.A.%20Schaffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU001C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU001C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Conesa</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Gotz</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Garcia-Gomez</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Terol</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Talon</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Robles</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research</div>, <div class="source">Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">3674</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Blast2GO%3A%20a%20universal%20tool%20for%20annotation%2C%20visualization%20and%20analysis%20in%20functional%20genomics%20research&amp;author=A.%20Conesa&amp;author=S.%20Gotz&amp;author=J.M.%20Garcia-Gomez&amp;author=J.%20Terol&amp;author=M.%20Talon&amp;author=M.%20Robles&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=3674-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti610" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti610" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti610"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16081474" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU001C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU001C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ashburner</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ball</div> <div class="given-names">C.A.</div></div>,  <div class="name"><div class="surname">Blake</div> <div class="given-names">J.A.</div></div>, et al. </div><div class="article-title">Gene ontology: tool for the unification of biology, The Gene Ontology Consortium</div>, <div class="source">Nat. Genet.</div> , <div class="year">2000</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">25</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20ontology%3A%20tool%20for%20the%20unification%20of%20biology%2C%20The%20Gene%20Ontology%20Consortium&amp;author=M.%20Ashburner&amp;author=C.A.%20Ball&amp;author=J.A.%20Blake&amp;publication_year=2000&amp;journal=Nat.%20Genet.&amp;volume=25&amp;pages=25-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/75556" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F75556" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F75556"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10802651" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU001C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU001C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Koski</div> <div class="given-names">L.B.</div></div>,  <div class="name"><div class="surname">Gray</div> <div class="given-names">M.W.</div></div>,  <div class="name"><div class="surname">Lang</div> <div class="given-names">B.F.</div></div>,  <div class="name"><div class="surname">Burger</div> <div class="given-names">G.</div></div>. </div><div class="article-title">AutoFACT: an automatic functional annotation and classification tool</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">6</div> pg. <div class="fpage">151</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AutoFACT%3A%20an%20automatic%20functional%20annotation%20and%20classification%20tool&amp;author=L.B.%20Koski&amp;author=M.W.%20Gray&amp;author=B.F.%20Lang&amp;author=G.%20Burger&amp;publication_year=2005&amp;journal=BMC%20Bioinformatics&amp;volume=6&amp;pages=151" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-6-151" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-6-151" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-6-151"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15960857" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU001C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU001C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Martin</div> <div class="given-names">D.M.</div></div>,  <div class="name"><div class="surname">Berriman</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Barton</div> <div class="given-names">G.J.</div></div>. </div><div class="article-title">GOtcha: a new method for prediction of protein function assessed by the annotation of seven genomes</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> pg. <div class="fpage">178</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GOtcha%3A%20a%20new%20method%20for%20prediction%20of%20protein%20function%20assessed%20by%20the%20annotation%20of%20seven%20genomes&amp;author=D.M.%20Martin&amp;author=M.%20Berriman&amp;author=G.J.%20Barton&amp;publication_year=2004&amp;journal=BMC%20Bioinformatics&amp;volume=5&amp;pages=178" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-5-178" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-5-178" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-5-178"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15550167" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU001C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU001C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hotz-Wagenblatt</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hankeln</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ernst</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Glatting</div> <div class="given-names">K.H.</div></div>,  <div class="name"><div class="surname">Schmidt</div> <div class="given-names">E.R.</div></div>,  <div class="name"><div class="surname">Suhai</div> <div class="given-names">S.</div></div>. </div><div class="article-title">ESTAnnotator: a tool for high throughput EST annotation</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">3716</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ESTAnnotator%3A%20a%20tool%20for%20high%20throughput%20EST%20annotation&amp;author=A.%20Hotz-Wagenblatt&amp;author=T.%20Hankeln&amp;author=P.%20Ernst&amp;author=K.H.%20Glatting&amp;author=E.R.%20Schmidt&amp;author=S.%20Suhai&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=3716-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg566" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg566" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg566"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12824401" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU001C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU001C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Strahm</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Powell</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Lefevre</div> <div class="given-names">C.</div></div>. </div><div class="article-title">EST-PAC a web package for EST annotation and protein sequence prediction</div>, <div class="source">Source Code Biol. Med.</div> , <div class="year">2006</div>, vol. <div class="volume">1</div> pg. <div class="fpage">2</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EST-PAC%20a%20web%20package%20for%20EST%20annotation%20and%20protein%20sequence%20prediction&amp;author=Y.%20Strahm&amp;author=D.%20Powell&amp;author=C.%20Lefevre&amp;publication_year=2006&amp;journal=Source%20Code%20Biol.%20Med.&amp;volume=1&amp;pages=2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1751-0473-1-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1751-0473-1-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1751-0473-1-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17147782" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU001C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU001C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kankainen</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ojala</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Holm</div> <div class="given-names">L.</div></div>. </div><div class="article-title">BLANNOTATOR: enhanced homology-based function prediction of bacterial proteins</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2012</div>, vol. <div class="volume">13</div> pg. <div class="fpage">33</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BLANNOTATOR%3A%20enhanced%20homology-based%20function%20prediction%20of%20bacterial%20proteins&amp;author=M.%20Kankainen&amp;author=T.%20Ojala&amp;author=L.%20Holm&amp;publication_year=2012&amp;journal=BMC%20Bioinformatics&amp;volume=13&amp;pages=33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-13-33" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-13-33" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-13-33"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22335941" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU001C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU001C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Medina</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Carbonell</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Pulido</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">Babelomics: an integrative platform for the analysis of transcriptomics, proteomics and genomic data with advanced functional profiling</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2010</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">W210</div>-<div class="lpage">213</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Babelomics%3A%20an%20integrative%20platform%20for%20the%20analysis%20of%20transcriptomics%2C%20proteomics%20and%20genomic%20data%20with%20advanced%20functional%20profiling&amp;author=I.%20Medina&amp;author=J.%20Carbonell&amp;author=L.%20Pulido&amp;publication_year=2010&amp;journal=Nucleic%20Acids%20Res.&amp;volume=38&amp;pages=W210-213" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq388" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq388" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq388"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20478823" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU001C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU001C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">D.W.</div></div>,  <div class="name"><div class="surname">Sherman</div> <div class="given-names">B.T.</div></div>,  <div class="name"><div class="surname">Lempicki</div> <div class="given-names">R.A.</div></div>. </div><div class="article-title">Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources</div>, <div class="source">Nat. Protoc.</div> , <div class="year">2009</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">44</div>-<div class="lpage">57</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systematic%20and%20integrative%20analysis%20of%20large%20gene%20lists%20using%20DAVID%20bioinformatics%20resources&amp;author=D.W.%20Huang&amp;author=B.T.%20Sherman&amp;author=R.A.%20Lempicki&amp;publication_year=2009&amp;journal=Nat.%20Protoc.&amp;volume=4&amp;pages=44-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2008.211" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2008.211" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2008.211"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19131956" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU001C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU001C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kasprzyk</div> <div class="given-names">A.</div></div>. </div><div class="article-title">BioMart: driving a paradigm change in biological data management</div>, <div class="source">Database (Oxford)</div> , <div class="year">2011</div>, vol. <div class="volume">2011</div> pg. <div class="fpage">bar049</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BioMart%3A%20driving%20a%20paradigm%20change%20in%20biological%20data%20management&amp;author=A.%20Kasprzyk&amp;publication_year=2011&amp;journal=Database%20(Oxford)&amp;volume=2011&amp;pages=bar049" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/database/bar049" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdatabase%2Fbar049" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdatabase%2Fbar049"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22083790" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU001C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU001C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tian</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Skolnick</div> <div class="given-names">J.</div></div>. </div><div class="article-title">How well is enzyme function conserved as a function of pairwise sequence identity?</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">333</div> (pg. <div class="fpage">863</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=How%20well%20is%20enzyme%20function%20conserved%20as%20a%20function%20of%20pairwise%20sequence%20identity%3F&amp;author=W.%20Tian&amp;author=J.%20Skolnick&amp;publication_year=2003&amp;journal=J.%20Mol.%20Biol.&amp;volume=333&amp;pages=863-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jmb.2003.08.057" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jmb.2003.08.057" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jmb.2003.08.057"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14568541" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU001C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU001C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="collab">The UniProt Consortium</div><div class="article-title">The Universal Protein Resource (UniProt)</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">D193</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl929" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl929" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl929"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17142230" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU001C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU001C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mulder</div> <div class="given-names">N.J.</div></div>,  <div class="name"><div class="surname">Apweiler</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Attwood</div> <div class="given-names">T.K.</div></div>, et al. </div><div class="article-title">New developments in the InterPro database</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">D224</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=New%20developments%20in%20the%20InterPro%20database&amp;author=N.J.%20Mulder&amp;author=R.%20Apweiler&amp;author=T.K.%20Attwood&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=D224-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl841" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl841" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl841"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17202162" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU001C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU001C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sander</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Schneider</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Database of homology-derived protein structures and the structural meaning of sequence alignment</div>, <div class="source">Proteins</div> , <div class="year">1991</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">56</div>-<div class="lpage">68</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Database%20of%20homology-derived%20protein%20structures%20and%20the%20structural%20meaning%20of%20sequence%20alignment&amp;author=C.%20Sander&amp;author=R.%20Schneider&amp;publication_year=1991&amp;journal=Proteins&amp;volume=9&amp;pages=56-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0134" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0134" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0134"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2017436" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU001C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU001C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rost</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Twilight zone of protein sequence alignments</div>, <div class="source">Protein Eng.</div> , <div class="year">1999</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">85</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Twilight%20zone%20of%20protein%20sequence%20alignments&amp;author=B.%20Rost&amp;publication_year=1999&amp;journal=Protein%20Eng.&amp;volume=12&amp;pages=85-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/protein/12.2.85" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fprotein%2F12.2.85" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fprotein%2F12.2.85"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10195279" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU001C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU001C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wilson</div> <div class="given-names">C.A.</div></div>,  <div class="name"><div class="surname">Kreychman</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Gerstein</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Assessing annotation transfer for genomics: quantifying the relations between protein sequence, structure and function through traditional and probabilistic scores</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">2000</div>, vol. <div class="volume">297</div> (pg. <div class="fpage">233</div>-<div class="lpage">49</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessing%20annotation%20transfer%20for%20genomics%3A%20quantifying%20the%20relations%20between%20protein%20sequence%2C%20structure%20and%20function%20through%20traditional%20and%20probabilistic%20scores&amp;author=C.A.%20Wilson&amp;author=J.%20Kreychman&amp;author=M.%20Gerstein&amp;publication_year=2000&amp;journal=J.%20Mol.%20Biol.&amp;volume=297&amp;pages=233-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.2000.3550" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.2000.3550" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.2000.3550"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10704319" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU001C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU001C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gower</div> <div class="given-names">A.C.</div></div>,  <div class="name"><div class="surname">Spira</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Lenburg</div> <div class="given-names">M.E.</div></div>. </div><div class="article-title">Discovering biological connections between experimental conditions based on common patterns of differential gene expression</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">381</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Discovering%20biological%20connections%20between%20experimental%20conditions%20based%20on%20common%20patterns%20of%20differential%20gene%20expression&amp;author=A.C.%20Gower&amp;author=A.%20Spira&amp;author=M.E.%20Lenburg&amp;publication_year=2011&amp;journal=BMC%20Bioinformatics&amp;volume=12&amp;pages=381" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-12-381" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-12-381" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-12-381"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21951600" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU001C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU001C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Tang</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">Transcriptome analysis of the venom glands of the Chinese wolf spider <em>Lycosa singoriensis</em></div>, <div class="source">Zoology (Jena)</div> , <div class="year">2010</div>, vol. <div class="volume">113</div> (pg. <div class="fpage">10</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20analysis%20of%20the%20venom%20glands%20of%20the%20Chinese%20wolf%20spider%20Lycosa%20singoriensis&amp;author=Y.%20Zhang&amp;author=J.%20Chen&amp;author=X.%20Tang&amp;publication_year=2010&amp;journal=Zoology%20(Jena)&amp;volume=113&amp;pages=10-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.zool.2009.04.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.zool.2009.04.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.zool.2009.04.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19875276" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU001C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU001C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Takeda</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Akimoto</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Kawamukai</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Effects of the <em>Escherichia coli</em> sfsA gene on mal genes expression and a DNA binding activity of SfsA</div>, <div class="source">Biosci. Biotechnol. Biochem.</div> , <div class="year">2001</div>, vol. <div class="volume">65</div> (pg. <div class="fpage">213</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effects%20of%20the%20Escherichia%20coli%20sfsA%20gene%20on%20mal%20genes%20expression%20and%20a%20DNA%20binding%20activity%20of%20SfsA&amp;author=K.%20Takeda&amp;author=C.%20Akimoto&amp;author=M.%20Kawamukai&amp;publication_year=2001&amp;journal=Biosci.%20Biotechnol.%20Biochem.&amp;volume=65&amp;pages=213-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1271/bbb.65.213" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1271%2Fbbb.65.213" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1271%2Fbbb.65.213"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11272834" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU001C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU001C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ramírez</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Muñoz-Mérida</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Karlsson</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">MOWServ: a web client for integration of bioinformatic resources</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2010</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">W671</div>-<div class="lpage">676</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MOWServ%3A%20a%20web%20client%20for%20integration%20of%20bioinformatic%20resources&amp;author=S.%20Ram%C3%ADrez&amp;author=A.%20Mu%C3%B1oz-M%C3%A9rida&amp;author=J.%20Karlsson&amp;publication_year=2010&amp;journal=Nucleic%20Acids%20Res.&amp;volume=38&amp;pages=W671-676" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq497" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq497" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq497"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20525794" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU001C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU001C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Muñoz-Mérida</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">González-Plaza</div> <div class="given-names">J.J.</div></div>,  <div class="name"><div class="surname">Cañada</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">De novo assembly and functional annotation of the olive (<em>Olea europaea</em>) transcriptome</div>, <div class="source">DNA Res.</div> , <div class="year">2013</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">93</div>-<div class="lpage">108</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20and%20functional%20annotation%20of%20the%20olive%20(Olea%20europaea)%20transcriptome&amp;author=A.%20Mu%C3%B1oz-M%C3%A9rida&amp;author=J.J.%20Gonz%C3%A1lez-Plaza&amp;author=A.%20Ca%C3%B1ada&amp;publication_year=2013&amp;journal=DNA%20Res.&amp;volume=20&amp;pages=93-108" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dss036" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdss036" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdss036"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23297299" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU001C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU001C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Canales</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bautista</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Label</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">De novo assembly of maritime pine transcriptome: implications for forest breeding and biotechnology</div>, <div class="source">Plant Biotechnol. J.</div> , <div class="year">2013</div> <div class="comment">[Epub ahead of print] PMID: 24256179</div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2014. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by-nc/4.0/), which permits non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=444964&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu001/2/dsu001-dsu001supp_fig.pdf?Expires=2147483647&amp;Signature=rE~~vyEWChJOD9s9lnc7XTi8lRbfQOK16P0JEsSC87ezeeFpXszicEvFF9BK2PgQoUp5wA3rSXgjkrGD4x6cVB-n~vYDdT6326UavdWPP2HCSQFifZpbBPhCxbHCME3p8l9Nk8T~oxKytbG1RLvkpnUdaqu5dh6B8I1lAQbt1ukOR37vpRXNRh1BJ4JP2~mWkNFcP9JXoWSSRDsAIzvsmKkg0th0DBrazTcs~9ltGykBSY5X0gR7Tn3j9rekdohsIrEch1OiftQuRMhxr3Q42RnIV9gP-m06PVVYY9nSjusQfQIy4klTL3cbH4aRSpxeyqvmq4GI-eUwPQ9A-rmdlQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093/dnares/dsu001/2/dsu001-dsu001supp.doc?Expires=2147483647&amp;Signature=HMTffe6ygu8KUCnrK2xaVALR0YYkXF5BQWJewBUpVIcrK7tTaRcMeCkamRbRiU90bN~zbpbaH6AU5w9231mLCPz3omLlfrWybbHrAJSJ4al8KGFSaWZgfXEUrY2PY~-2PenA2QPQ85UVqINoaxTdidmrSiQ5phblWsQsEZJ1fkZcyDjt82PhGjdIo8bbzc~2b3lYg9oN8o3YvfSe-429LmNtq5jXaRhZGNUZaMV71H6-7BKVl4tGOASNZzmFOu5moaaRvB~V65H5aUQGiRU5DKzHsSC0wGkdJz8v91YFBJLUlojzk-iqVPUncKfHISndU4lzRDfitNIw5s0Utujlow__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">357</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000344243900001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">22</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsu001"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="444964"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000344243900001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f21%2f4%2f341%2f444964">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000344243900001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (22)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F21%2F4%2F341%2F444964" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/444964'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Sma3s: A Three-Step Modular Annotator for Large Sequence Datasets" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsu001");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f21%2f4%2f341%2f444964">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

