

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/15/4/241/376722","@type":"ScholarlyArticle","name":"Novel DNA Microarray System for Analysis of Nascent mRNAs","datePublished":"2008-07-08","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/15/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2008-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsn015","inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/15/4/241/376722","author":[{"name":"Ohtsu, Masaya","affiliation":"1  Faculty of Industrial Science and Technology, Department of Biological Science and Technology ,  Tokyo University of Science ,  2641 Yamazaki, Noda-shi, Chiba 278-8510 ,  Japan","@type":"Person"},{"name":"Kawate, Mika","affiliation":"1  Faculty of Industrial Science and Technology, Department of Biological Science and Technology ,  Tokyo University of Science ,  2641 Yamazaki, Noda-shi, Chiba 278-8510 ,  Japan","@type":"Person"},{"name":"Fukuoka, Masashi","affiliation":"1  Faculty of Industrial Science and Technology, Department of Biological Science and Technology ,  Tokyo University of Science ,  2641 Yamazaki, Noda-shi, Chiba 278-8510 ,  Japan","@type":"Person"},{"name":"Gunji, Wataru","affiliation":"2  Bio Matrix Research Inc., 105 Higashifukai, Nagareyama, Chiba 275-0101 ,  Japan","@type":"Person"},{"name":"Hanaoka, Fumio","affiliation":"3  Graduate School of Frontier Biosciences ,  Osaka University ,  1-3 Yamada-oka, Suita, Osaka 565-0871 ,  Japan","@type":"Person"},{"name":"Utsugi, Takahiko","affiliation":"2  Bio Matrix Research Inc., 105 Higashifukai, Nagareyama, Chiba 275-0101 ,  Japan","@type":"Person"},{"name":"Onoda, Fumitoshi","affiliation":"1  Faculty of Industrial Science and Technology, Department of Biological Science and Technology ,  Tokyo University of Science ,  2641 Yamazaki, Noda-shi, Chiba 278-8510 ,  Japan","@type":"Person"},{"name":"Murakami, Yasufumi","affiliation":"1  Faculty of Industrial Science and Technology, Department of Biological Science and Technology ,  Tokyo University of Science ,  2641 Yamazaki, Noda-shi, Chiba 278-8510 ,  Japan    2  Bio Matrix Research Inc., 105 Higashifukai, Nagareyama, Chiba 275-0101 ,  Japan","@type":"Person"}],"description":"Abstract.  Transcriptional activation and repression are a key step in the regulation of all cellular activities. The development of comprehensive analysis meth","pageStart":"241","pageEnd":"251","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/4/1/m_cover.gif?Expires=2147483647&Signature=RLDuHMUA6eE0ByVqiq8aUGyMxZAqCYh6CUPq-ezrEq~vX1AX9OwM0EdcZgw7bZO5GF08rcIkuMn~SdMlIqZ-OMoLaS-efZLrQFFAEDhygPUYmAsVLKwz1N0-SZE8tkL655rpVOyhNsmCq0HeMyZ1CRYm6AEF3IsWa0HPWbYG3ZsNpccsrKjShJ5nSPlVjprfmUgaJjeTeE3~oZCdhOYmMPqtkUKaLjbpdnpy8KGSYF3y2vs7JTB1QHc1IXmul6fvj2x9yPzU-FZHZaBu8~fDI-1ze4-bV3EV~DhG3VaQYIzUv05cvil7Rnpu6ewBIQhqRHLzHcljRy6MHn~J052jxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Novel DNA Microarray System for Analysis of Nascent mRNAs","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/4/1/m_cover.gif?Expires=2147483647&Signature=RLDuHMUA6eE0ByVqiq8aUGyMxZAqCYh6CUPq-ezrEq~vX1AX9OwM0EdcZgw7bZO5GF08rcIkuMn~SdMlIqZ-OMoLaS-efZLrQFFAEDhygPUYmAsVLKwz1N0-SZE8tkL655rpVOyhNsmCq0HeMyZ1CRYm6AEF3IsWa0HPWbYG3ZsNpccsrKjShJ5nSPlVjprfmUgaJjeTeE3~oZCdhOYmMPqtkUKaLjbpdnpy8KGSYF3y2vs7JTB1QHc1IXmul6fvj2x9yPzU-FZHZaBu8~fDI-1ze4-bV3EV~DhG3VaQYIzUv05cvil7Rnpu6ewBIQhqRHLzHcljRy6MHn~J052jxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Novel DNA Microarray System for Analysis of Nascent mRNAs" />
    <meta property="og:description" content="Abstract.  Transcriptional activation and repression are a key step in the regulation of all cellular activities. The development of comprehensive analysis meth" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/15/4/241/376722" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2008-07-08" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Ohtsu, Masaya" /><meta name="citation_author_institution" content="Faculty of Industrial Science and Technology, Department of Biological Science and Technology, Tokyo University of Science, 2641 Yamazaki, Noda-shi, Chiba 278-8510, Japan" /><meta name="citation_author" content="Kawate, Mika" /><meta name="citation_author_institution" content="Faculty of Industrial Science and Technology, Department of Biological Science and Technology, Tokyo University of Science, 2641 Yamazaki, Noda-shi, Chiba 278-8510, Japan" /><meta name="citation_author" content="Fukuoka, Masashi" /><meta name="citation_author_institution" content="Faculty of Industrial Science and Technology, Department of Biological Science and Technology, Tokyo University of Science, 2641 Yamazaki, Noda-shi, Chiba 278-8510, Japan" /><meta name="citation_author" content="Gunji, Wataru" /><meta name="citation_author_institution" content="Bio Matrix Research Inc., 105 Higashifukai, Nagareyama, Chiba 275-0101, Japan" /><meta name="citation_author" content="Hanaoka, Fumio" /><meta name="citation_author_institution" content="Graduate School of Frontier Biosciences, Osaka University, 1-3 Yamada-oka, Suita, Osaka 565-0871, Japan" /><meta name="citation_author" content="Utsugi, Takahiko" /><meta name="citation_author_institution" content="Bio Matrix Research Inc., 105 Higashifukai, Nagareyama, Chiba 275-0101, Japan" /><meta name="citation_author" content="Onoda, Fumitoshi" /><meta name="citation_author_institution" content="Faculty of Industrial Science and Technology, Department of Biological Science and Technology, Tokyo University of Science, 2641 Yamazaki, Noda-shi, Chiba 278-8510, Japan" /><meta name="citation_author" content="Murakami, Yasufumi" /><meta name="citation_author_institution" content="Faculty of Industrial Science and Technology, Department of Biological Science and Technology, Tokyo University of Science, 2641 Yamazaki, Noda-shi, Chiba 278-8510, Japan" /><meta name="citation_author_institution" content="Bio Matrix Research Inc., 105 Higashifukai, Nagareyama, Chiba 275-0101, Japan" /><meta name="citation_title" content="Novel DNA Microarray System for Analysis of Nascent mRNAs" /><meta name="citation_firstpage" content="241" /><meta name="citation_lastpage" content="251" /><meta name="citation_doi" content="10.1093/dnares/dsn015" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="15" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2008/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Global analysis of the expression patterns of transcriptional regulatory factors in formation of embryoid bodies using sensitive oligonucleotide microarray systems; Biochem. Biophys. Res. Commun.;  citation_year=2004;  citation_volume=325;  citation_pages=265-275; " /><meta name="citation_reference" content="citation_title=Activation of pS2 gene transcription is a primary response to estrogen in the human breast cancer cell line MCF-7; Proc. Natl. Acad. Sci. USA;  citation_year=1984;  citation_volume=81;  citation_pages=6344-6348; " /><meta name="citation_reference" content="citation_title=Global analysis of stress-regulated mRNA turnover by using cDNA arrays; Proc. Natl. Acad. Sci. USA;  citation_year=2002;  citation_volume=99;  citation_pages=10611-10616; " /><meta name="citation_reference" content="citation_title=Genomic run-on evaluates transcription rates for all yeast genes and identifies gene regulatory mechanisms; Mol. Cell;  citation_year=2004;  citation_volume=15;  citation_pages=303-313; " /><meta name="citation_reference" content="citation_title=Control of gene expression during T cell activation: alternate regulation of mRNA transcription and mRNA stability; BMC Genom.;  citation_year=2005;  citation_volume=6;  citation_pages=75" /><meta name="citation_reference" content="citation_title=Biosynthetic labeling of RNA with uracil phosphoribosyltransferase allows cell-specific microarray analysis of mRNA synthesis and decay; Nat. Biotechnol.;  citation_year=2005;  citation_volume=23;  citation_pages=232-237; " /><meta name="citation_reference" content="citation_title=Fluorescent labeling of nascent RNA reveals transcription by RNA polymerase II in domains scattered throughout the nucleus; J. Cell Biol.;  citation_year=1993;  citation_volume=122;  citation_pages=283-293; " /><meta name="citation_reference" content="citation_title=Numbers and organization of RNA polymerases, nascent transcripts, and transcription units in HeLa nuclei; Mol. Biol. Cell;  citation_year=1998;  citation_volume=9;  citation_pages=1523-1536; " /><meta name="citation_reference" content="citation_title=Electric field-mediated BrUTP uptake by mouse oocytes, eggs, and embryos; Mol. Reprod. Dev.;  citation_year=2001;  citation_volume=58;  citation_pages=173-179; " /><meta name="citation_reference" content="citation_title=General transcriptional repression by polyglutamine disease proteins is not directly linked to the presence of inclusion bodies; Biochem. Biophys. Res. Commun.;  citation_year=2004;  citation_volume=313;  citation_pages=110-116; " /><meta name="citation_reference" content="citation_title=Nuclear distribution of transcription factors in relation to sites of transcription and RNA polymerase II; J. Cell Sci.;  citation_year=1997;  citation_volume=110;  citation_pages=1781-1791; " /><meta name="citation_reference" content="citation_title=Immunoseparation and immunodetection of nucleic acids labeled with halogenated nucleotides; Exp. Cell Res.;  citation_year=1997;  citation_volume=234;  citation_pages=498-506; " /><meta name="citation_reference" content="citation_title=Isolation of nascent messenger RNA from mouse preimplantation embryos; Biol. Reprod.;  citation_year=2004;  citation_volume=71;  citation_pages=1948-1955; " /><meta name="citation_reference" content="citation_title=Isolation and analysis of a mammalian temperature-sensitive mutant defective in G2 functions; Exp. Cell Res.;  citation_year=1986;  citation_volume=167;  citation_pages=53-62; " /><meta name="citation_reference" content="citation_title=The FT210 cell line is a mouse G2 phase mutant with a temperature-sensitive CDC2 gene product; Cell;  citation_year=1990;  citation_volume=63;  citation_pages=313-324; " /><meta name="citation_reference" content="citation_title=Analysis of transcription factor expression during oogenesis and preimplantation development in mice; Zygote;  citation_year=2007;  citation_volume=15;  citation_pages=117-128; " /><meta name="citation_reference" content="citation_title=Effects of p21Cip1/Waf1 at both the G1/S and the G2/M cell cycle transitions: pRb is a critical determinant in blocking DNA replication and in preventing endoreduplication; Mol. Cell. Biol.;  citation_year=1998;  citation_volume=18;  citation_pages=629-643; " /><meta name="citation_reference" content="citation_title=Regulation of leading edge microtubule and actin dynamics downstream of Rac1; J. Cell Biol.;  citation_year=2003;  citation_volume=161;  citation_pages=845-851; " /><meta name="citation_reference" content="citation_title=The GIT-associated kinase PAK targets to the centrosome and regulates Aurora-A; Mol. Cell;  citation_year=2005;  citation_volume=20;  citation_pages=237-249; " /><meta name="citation_reference" content="citation_title=Cleavage arrest of early frog embryos by the G protein-activated protein kinase PAK I; J. Biol. Chem.;  citation_year=1996;  citation_volume=271;  citation_pages=21498-21504; " /><meta name="citation_reference" content="citation_title=The architecture of mammalian ribosomal protein promoters; BMC Evol. Biol.;  citation_year=2005;  citation_volume=5;  citation_pages=15" /><meta name="citation_reference" content="citation_title=A common cis-element in promoters of protein synthesis and cell cycle genes; Acta Biochim. Pol.;  citation_year=2007;  citation_volume=54;  citation_pages=89-98; " /><meta name="citation_reference" content="citation_title=Microarray analysis of newly synthesized RNA in cells and animals; Proc. Natl. Acad. Sci. USA;  citation_year=2007;  citation_volume=104;  citation_pages=6164-6169; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/15/4/241/1070847/dsn015.pdf" /><meta name="description" content="Abstract.  Transcriptional activation and repression are a key step in the regulation of all cellular activities. The development of comprehensive analysis meth" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/15/4/241/376722" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/15/4/241/376722" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/15/4/241/376722");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsn015';
    </script>
<title>
	Novel DNA Microarray System for Analysis of Nascent mRNAs | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./376722" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BeNEPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPaHRzdSwgTWFzYXlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRmFjdWx0eSBvZiBJbmR1c3RyaWFsIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIERlcGFydG1lbnQgb2YgQmlvbG9naWNhbCBTY2llbmNlIGFuZCBUZWNobm9sb2d5LCBUb2t5byBVbml2ZXJzaXR5IG9mIFNjaWVuY2UsIDI2NDEgWWFtYXpha2ksIE5vZGEtc2hpLCBDaGliYSAyNzgtODUxMCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLYXdhdGUsIE1pa2EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJGYWN1bHR5IG9mIEluZHVzdHJpYWwgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgRGVwYXJ0bWVudCBvZiBCaW9sb2dpY2FsIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIFRva3lvIFVuaXZlcnNpdHkgb2YgU2NpZW5jZSwgMjY0MSBZYW1hemFraSwgTm9kYS1zaGksIENoaWJhIDI3OC04NTEwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkZ1a3Vva2EsIE1hc2FzaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJGYWN1bHR5IG9mIEluZHVzdHJpYWwgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgRGVwYXJ0bWVudCBvZiBCaW9sb2dpY2FsIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIFRva3lvIFVuaXZlcnNpdHkgb2YgU2NpZW5jZSwgMjY0MSBZYW1hemFraSwgTm9kYS1zaGksIENoaWJhIDI3OC04NTEwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikd1bmppLCBXYXRhcnUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCaW8gTWF0cml4IFJlc2VhcmNoIEluYy4sIDEwNSBIaWdhc2hpZnVrYWksIE5hZ2FyZXlhbWEsIENoaWJhIDI3NS0wMTAxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkhhbmFva2EsIEZ1bWlvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIEZyb250aWVyIEJpb3NjaWVuY2VzLCBPc2FrYSBVbml2ZXJzaXR5LCAxLTMgWWFtYWRhLW9rYSwgU3VpdGEsIE9zYWthIDU2NS0wODcxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlV0c3VnaSwgVGFrYWhpa28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCaW8gTWF0cml4IFJlc2VhcmNoIEluYy4sIDEwNSBIaWdhc2hpZnVrYWksIE5hZ2FyZXlhbWEsIENoaWJhIDI3NS0wMTAxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik9ub2RhLCBGdW1pdG9zaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJGYWN1bHR5IG9mIEluZHVzdHJpYWwgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgRGVwYXJ0bWVudCBvZiBCaW9sb2dpY2FsIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIFRva3lvIFVuaXZlcnNpdHkgb2YgU2NpZW5jZSwgMjY0MSBZYW1hemFraSwgTm9kYS1zaGksIENoaWJhIDI3OC04NTEwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik11cmFrYW1pLCBZYXN1ZnVtaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkZhY3VsdHkgb2YgSW5kdXN0cmlhbCBTY2llbmNlIGFuZCBUZWNobm9sb2d5LCBEZXBhcnRtZW50IG9mIEJpb2xvZ2ljYWwgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgVG9reW8gVW5pdmVyc2l0eSBvZiBTY2llbmNlLCAyNjQxIFlhbWF6YWtpLCBOb2RhLXNoaSwgQ2hpYmEgMjc4LTg1MTAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvIE1hdHJpeCBSZXNlYXJjaCBJbmMuLCAxMDUgSGlnYXNoaWZ1a2FpLCBOYWdhcmV5YW1hLCBDaGliYSAyNzUtMDEwMSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9Ik5vdmVsIEROQSBNaWNyb2FycmF5IFN5c3RlbSBmb3IgQW5hbHlzaXMgb2YgTmFzY2VudCBtUk5BcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjI0MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMjUxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNuMDE1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMDgvMDgvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HbG9iYWwgYW5hbHlzaXMgb2YgdGhlIGV4cHJlc3Npb24gcGF0dGVybnMgb2YgdHJhbnNjcmlwdGlvbmFsIHJlZ3VsYXRvcnkgZmFjdG9ycyBpbiBmb3JtYXRpb24gb2YgZW1icnlvaWQgYm9kaWVzIHVzaW5nIHNlbnNpdGl2ZSBvbGlnb251Y2xlb3RpZGUgbWljcm9hcnJheSBzeXN0ZW1zOyBCaW9jaGVtLiBCaW9waHlzLiBSZXMuIENvbW11bi47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzI1OyAgY2l0YXRpb25fcGFnZXM9MjY1LTI3NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWN0aXZhdGlvbiBvZiBwUzIgZ2VuZSB0cmFuc2NyaXB0aW9uIGlzIGEgcHJpbWFyeSByZXNwb25zZSB0byBlc3Ryb2dlbiBpbiB0aGUgaHVtYW4gYnJlYXN0IGNhbmNlciBjZWxsIGxpbmUgTUNGLTc7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTg0OyAgY2l0YXRpb25fdm9sdW1lPTgxOyAgY2l0YXRpb25fcGFnZXM9NjM0NC02MzQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HbG9iYWwgYW5hbHlzaXMgb2Ygc3RyZXNzLXJlZ3VsYXRlZCBtUk5BIHR1cm5vdmVyIGJ5IHVzaW5nIGNETkEgYXJyYXlzOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT05OTsgIGNpdGF0aW9uX3BhZ2VzPTEwNjExLTEwNjE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljIHJ1bi1vbiBldmFsdWF0ZXMgdHJhbnNjcmlwdGlvbiByYXRlcyBmb3IgYWxsIHllYXN0IGdlbmVzIGFuZCBpZGVudGlmaWVzIGdlbmUgcmVndWxhdG9yeSBtZWNoYW5pc21zOyBNb2wuIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0zMDMtMzEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db250cm9sIG9mIGdlbmUgZXhwcmVzc2lvbiBkdXJpbmcgVCBjZWxsIGFjdGl2YXRpb246IGFsdGVybmF0ZSByZWd1bGF0aW9uIG9mIG1STkEgdHJhbnNjcmlwdGlvbiBhbmQgbVJOQSBzdGFiaWxpdHk7IEJNQyBHZW5vbS47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTc1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Qmlvc3ludGhldGljIGxhYmVsaW5nIG9mIFJOQSB3aXRoIHVyYWNpbCBwaG9zcGhvcmlib3N5bHRyYW5zZmVyYXNlIGFsbG93cyBjZWxsLXNwZWNpZmljIG1pY3JvYXJyYXkgYW5hbHlzaXMgb2YgbVJOQSBzeW50aGVzaXMgYW5kIGRlY2F5OyBOYXQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MjMyLTIzNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Rmx1b3Jlc2NlbnQgbGFiZWxpbmcgb2YgbmFzY2VudCBSTkEgcmV2ZWFscyB0cmFuc2NyaXB0aW9uIGJ5IFJOQSBwb2x5bWVyYXNlIElJIGluIGRvbWFpbnMgc2NhdHRlcmVkIHRocm91Z2hvdXQgdGhlIG51Y2xldXM7IEouIENlbGwgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9MTIyOyAgY2l0YXRpb25fcGFnZXM9MjgzLTI5MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TnVtYmVycyBhbmQgb3JnYW5pemF0aW9uIG9mIFJOQSBwb2x5bWVyYXNlcywgbmFzY2VudCB0cmFuc2NyaXB0cywgYW5kIHRyYW5zY3JpcHRpb24gdW5pdHMgaW4gSGVMYSBudWNsZWk7IE1vbC4gQmlvbC4gQ2VsbDsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MTUyMy0xNTM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbGVjdHJpYyBmaWVsZC1tZWRpYXRlZCBCclVUUCB1cHRha2UgYnkgbW91c2Ugb29jeXRlcywgZWdncywgYW5kIGVtYnJ5b3M7IE1vbC4gUmVwcm9kLiBEZXYuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTU4OyAgY2l0YXRpb25fcGFnZXM9MTczLTE3OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXJhbCB0cmFuc2NyaXB0aW9uYWwgcmVwcmVzc2lvbiBieSBwb2x5Z2x1dGFtaW5lIGRpc2Vhc2UgcHJvdGVpbnMgaXMgbm90IGRpcmVjdGx5IGxpbmtlZCB0byB0aGUgcHJlc2VuY2Ugb2YgaW5jbHVzaW9uIGJvZGllczsgQmlvY2hlbS4gQmlvcGh5cy4gUmVzLiBDb21tdW4uOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMxMzsgIGNpdGF0aW9uX3BhZ2VzPTExMC0xMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU51Y2xlYXIgZGlzdHJpYnV0aW9uIG9mIHRyYW5zY3JpcHRpb24gZmFjdG9ycyBpbiByZWxhdGlvbiB0byBzaXRlcyBvZiB0cmFuc2NyaXB0aW9uIGFuZCBSTkEgcG9seW1lcmFzZSBJSTsgSi4gQ2VsbCBTY2kuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTExMDsgIGNpdGF0aW9uX3BhZ2VzPTE3ODEtMTc5MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW1tdW5vc2VwYXJhdGlvbiBhbmQgaW1tdW5vZGV0ZWN0aW9uIG9mIG51Y2xlaWMgYWNpZHMgbGFiZWxlZCB3aXRoIGhhbG9nZW5hdGVkIG51Y2xlb3RpZGVzOyBFeHAuIENlbGwgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzQ7ICBjaXRhdGlvbl9wYWdlcz00OTgtNTA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jc29sYXRpb24gb2YgbmFzY2VudCBtZXNzZW5nZXIgUk5BIGZyb20gbW91c2UgcHJlaW1wbGFudGF0aW9uIGVtYnJ5b3M7IEJpb2wuIFJlcHJvZC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NzE7ICBjaXRhdGlvbl9wYWdlcz0xOTQ4LTE5NTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlzb2xhdGlvbiBhbmQgYW5hbHlzaXMgb2YgYSBtYW1tYWxpYW4gdGVtcGVyYXR1cmUtc2Vuc2l0aXZlIG11dGFudCBkZWZlY3RpdmUgaW4gRzIgZnVuY3Rpb25zOyBFeHAuIENlbGwgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk4NjsgIGNpdGF0aW9uX3ZvbHVtZT0xNjc7ICBjaXRhdGlvbl9wYWdlcz01My02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEZUMjEwIGNlbGwgbGluZSBpcyBhIG1vdXNlIEcyIHBoYXNlIG11dGFudCB3aXRoIGEgdGVtcGVyYXR1cmUtc2Vuc2l0aXZlIENEQzIgZ2VuZSBwcm9kdWN0OyBDZWxsOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTYzOyAgY2l0YXRpb25fcGFnZXM9MzEzLTMyNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgdHJhbnNjcmlwdGlvbiBmYWN0b3IgZXhwcmVzc2lvbiBkdXJpbmcgb29nZW5lc2lzIGFuZCBwcmVpbXBsYW50YXRpb24gZGV2ZWxvcG1lbnQgaW4gbWljZTsgWnlnb3RlOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MTE3LTEyODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RWZmZWN0cyBvZiBwMjFDaXAxL1dhZjEgYXQgYm90aCB0aGUgRzEvUyBhbmQgdGhlIEcyL00gY2VsbCBjeWNsZSB0cmFuc2l0aW9uczogcFJiIGlzIGEgY3JpdGljYWwgZGV0ZXJtaW5hbnQgaW4gYmxvY2tpbmcgRE5BIHJlcGxpY2F0aW9uIGFuZCBpbiBwcmV2ZW50aW5nIGVuZG9yZWR1cGxpY2F0aW9uOyBNb2wuIENlbGwuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9NjI5LTY0MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVndWxhdGlvbiBvZiBsZWFkaW5nIGVkZ2UgbWljcm90dWJ1bGUgYW5kIGFjdGluIGR5bmFtaWNzIGRvd25zdHJlYW0gb2YgUmFjMTsgSi4gQ2VsbCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNjE7ICBjaXRhdGlvbl9wYWdlcz04NDUtODUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgR0lULWFzc29jaWF0ZWQga2luYXNlIFBBSyB0YXJnZXRzIHRvIHRoZSBjZW50cm9zb21lIGFuZCByZWd1bGF0ZXMgQXVyb3JhLUE7IE1vbC4gQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTIzNy0yNDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNsZWF2YWdlIGFycmVzdCBvZiBlYXJseSBmcm9nIGVtYnJ5b3MgYnkgdGhlIEcgcHJvdGVpbi1hY3RpdmF0ZWQgcHJvdGVpbiBraW5hc2UgUEFLIEk7IEouIEJpb2wuIENoZW0uOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTI3MTsgIGNpdGF0aW9uX3BhZ2VzPTIxNDk4LTIxNTA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgYXJjaGl0ZWN0dXJlIG9mIG1hbW1hbGlhbiByaWJvc29tYWwgcHJvdGVpbiBwcm9tb3RlcnM7IEJNQyBFdm9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNvbW1vbiBjaXMtZWxlbWVudCBpbiBwcm9tb3RlcnMgb2YgcHJvdGVpbiBzeW50aGVzaXMgYW5kIGNlbGwgY3ljbGUgZ2VuZXM7IEFjdGEgQmlvY2hpbS4gUG9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT01NDsgIGNpdGF0aW9uX3BhZ2VzPTg5LTk4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb2FycmF5IGFuYWx5c2lzIG9mIG5ld2x5IHN5bnRoZXNpemVkIFJOQSBpbiBjZWxscyBhbmQgYW5pbWFsczsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTA0OyAgY2l0YXRpb25fcGFnZXM9NjE2NC02MTY5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTUvNC8yNDEvMTA3MDg0Ny9kc24wMTUucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFRyYW5zY3JpcHRpb25hbCBhY3RpdmF0aW9uIGFuZCByZXByZXNzaW9uIGFyZSBhIGtleSBzdGVwIGluIHRoZSByZWd1bGF0aW9uIG9mIGFsbCBjZWxsdWxhciBhY3Rpdml0aWVzLiBUaGUgZGV2ZWxvcG1lbnQgb2YgY29tcHJlaGVuc2l2ZSBhbmFseXNpcyBtZXRoIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zNzY3MjImeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkEZLC8NcWU6/oqnsCOdqPwet78/A=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f4%2f241%2f376722" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f4%2f241%2f376722" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=14644" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/15/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/4/1/m_cover.gif?Expires=2147483647&amp;Signature=RLDuHMUA6eE0ByVqiq8aUGyMxZAqCYh6CUPq-ezrEq~vX1AX9OwM0EdcZgw7bZO5GF08rcIkuMn~SdMlIqZ-OMoLaS-efZLrQFFAEDhygPUYmAsVLKwz1N0-SZE8tkL655rpVOyhNsmCq0HeMyZ1CRYm6AEF3IsWa0HPWbYG3ZsNpccsrKjShJ5nSPlVjprfmUgaJjeTeE3~oZCdhOYmMPqtkUKaLjbpdnpy8KGSYF3y2vs7JTB1QHc1IXmul6fvj2x9yPzU-FZHZaBu8~fDI-1ze4-bV3EV~DhG3VaQYIzUv05cvil7Rnpu6ewBIQhqRHLzHcljRy6MHn~J052jxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/15/4">
                        <div class="volume trailing-comma">Volume 15</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2008        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5489963">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5489963">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5489964">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5489964">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5489969">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5489969">2. Materials and Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5490005">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5490005">3. Results and Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5490035">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5490035">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5490037">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5490037">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5490039">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5490039">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/15/4/227/375737">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/15/4/253/376655">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Novel DNA Microarray System for Analysis of Nascent mRNAs
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masaya Ohtsu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Faculty of Industrial Science and Technology, Department of Biological Science and Technology</div>, <div class="insititution">Tokyo University of Science</div>, <div class="addr-line">2641 Yamazaki, Noda-shi, Chiba 278-8510</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masaya+Ohtsu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ohtsu M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ohtsu M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Masaya Ohtsu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mika Kawate</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Faculty of Industrial Science and Technology, Department of Biological Science and Technology</div>, <div class="insititution">Tokyo University of Science</div>, <div class="addr-line">2641 Yamazaki, Noda-shi, Chiba 278-8510</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mika+Kawate">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kawate M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kawate M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mika Kawate</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masashi Fukuoka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Faculty of Industrial Science and Technology, Department of Biological Science and Technology</div>, <div class="insititution">Tokyo University of Science</div>, <div class="addr-line">2641 Yamazaki, Noda-shi, Chiba 278-8510</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masashi+Fukuoka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fukuoka M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fukuoka M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Masashi Fukuoka</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wataru Gunji</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Bio Matrix Research Inc., 105 Higashifukai, Nagareyama, Chiba 275-0101</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wataru+Gunji">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gunji W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gunji W&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Wataru Gunji</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Fumio Hanaoka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Graduate School of Frontier Biosciences</div>, <div class="insititution">Osaka University</div>, <div class="addr-line">1-3 Yamada-oka, Suita, Osaka 565-0871</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Fumio+Hanaoka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hanaoka F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hanaoka F&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Fumio Hanaoka</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takahiko Utsugi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Bio Matrix Research Inc., 105 Higashifukai, Nagareyama, Chiba 275-0101</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takahiko+Utsugi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Utsugi T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Utsugi T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takahiko Utsugi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Fumitoshi Onoda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Faculty of Industrial Science and Technology, Department of Biological Science and Technology</div>, <div class="insititution">Tokyo University of Science</div>, <div class="addr-line">2641 Yamazaki, Noda-shi, Chiba 278-8510</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Fumitoshi+Onoda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Onoda F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Onoda F&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Fumitoshi Onoda</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yasufumi Murakami</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Faculty of Industrial Science and Technology, Department of Biological Science and Technology</div>, <div class="insititution">Tokyo University of Science</div>, <div class="addr-line">2641 Yamazaki, Noda-shi, Chiba 278-8510</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Bio Matrix Research Inc., 105 Higashifukai, Nagareyama, Chiba 275-0101</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 4-7124-1501 ext. 4408. Fax. <div class="fax">+81 4-7122-1360</div>. E-mail: <a href="mailto:yasufumi@rs.noda.tus.ac.jp" target="_blank">yasufumi@rs.noda.tus.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yasufumi+Murakami">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Murakami Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Murakami Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yasufumi Murakami</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Kazuo Shinozaki</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 15, Issue 4, 1 August 2008, Pages 241–251, <a href='https://doi.org/10.1093/dnares/dsn015'>https://doi.org/10.1093/dnares/dsn015</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">08 July 2008</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">03 May 2008</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">11 June 2008</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="376722" href="/dnaresearch/article-pdf/15/4/241/1070847/dsn015.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Masaya Ohtsu, Mika Kawate, Masashi Fukuoka, Wataru Gunji, Fumio Hanaoka, Takahiko Utsugi, Fumitoshi Onoda, Yasufumi Murakami;  Novel DNA Microarray System for Analysis of Nascent mRNAs, <em>DNA Research</em>, Volume 15, Issue 4, 1 August 2008, Pages 241–251, <a href='https://doi.org/10.1093/dnares/dsn015'>https://doi.org/10.1093/dnares/dsn015</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/376722?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/376722?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/376722?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/376722?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/376722?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/376722?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/376722?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Novel%20DNA%20Microarray%20System%20for%20Analysis%20of%20Nascent%20mRNAs&amp;publicationDate=2008-07-08&amp;volumeNum=15&amp;issueNum=4&amp;author=Ohtsu%2C%20Masaya%3B%20Kawate%2C%20Mika&amp;startPage=241&amp;endPage=251&amp;contentId=10.1093%2Fdnares%2Fdsn015&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f4%2f241%2f376722" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5489963 id="5489963" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Transcriptional activation and repression are a key step in the regulation of all cellular activities. The development of comprehensive analysis methods such as DNA microarray has advanced our understanding of the correlation between the regulation of transcription and that of cellular mechanisms. However, DNA microarray analysis based on steady-state mRNA (total mRNA) does not always correspond to transcriptional activation or repression. To comprehend these transcriptional regulations, the detection of nascent RNAs is more informative. Although the nuclear run-on assay can detect nascent RNAs, it has not been fully applied to DNA microarray analysis. In this study, we have developed a highly efficient method for isolating bromouridine-labeled nascent RNAs that can be successfully applied to DNA microarray analysis. This method can linearly amplify small amounts of mRNAs with little bias. Furthermore, we have applied this method to DNA microarray analysis from mouse G<sub>2</sub>-arrested cells and have identified several genes that exhibit novel expression profiles. This method will provide important information in the field of transcriptome analysis of various cellular processes.</p></section>                        <div class="article-metadata-panel clearfix"></div>
                            <h2 scrollto-destination=5489964 id="5489964" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>DNA microarray is an important tool for understanding regulatory networks. In fact, this technique was used to analyze changes in the amounts of mRNA in cellular phenomena such as cell differentiation, cellular senescence, and cell cycle progression in a comprehensive manner. Moreover, we have successfully improved the sensitivity and reproducibility of DNA microarray analysis.<sup><a reveal-id="DSN015C1" data-open="DSN015C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Although many of the expression profile data obtained through DNA microarray analysis are available at various websites, such data are based on steady-state mRNA levels. It is obvious that both mRNA synthesis and degradation influence steady-state mRNA levels, but usually only the total mRNA is quantified. If we can quantify nascent mRNA in a real-time manner, it will become possible to estimate mRNA synthesis and degradation rates by comparing the nascent amount with the total amount of RNA measured by an ordinary DNA microarray system. One method of detecting nascent mRNA is the nuclear run-on assay.<sup><a reveal-id="DSN015C2" data-open="DSN015C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Recently, the transcriptional profiling of radio-labeled RNAs using the nylon-membrane DNA microarray was reported.<sup><a reveal-id="DSN015C3  DSN015C5" data-open="DSN015C3  DSN015C5" class="link link-ref link-reveal xref-bibr">3–5</a></sup> The current standard platforms are GeneChip by Affymetrix, the Stanford-type DNA microarray, and the oligo-DNA microarray. In general, it is very difficult to compare data from such standard DNA microarray systems with data obtained by the nylon membrane DNA microarray. To comprehensively analyze nascent RNAs using these current standard platforms, improved methods for the isolation and labeling of nascent mRNAs were reported. Cleary et al. used uracil phosphoribosyltransferase gene-transformed human cells in <em>Toxoplasma gondii</em>. Nascent RNAs from these cells grown in 2,4-dithiouracil were labeled with thio-substrated UTP, purified, and analyzed by DNA microarray.<sup><a reveal-id="DSN015C6" data-open="DSN015C6" class="link link-ref link-reveal xref-bibr">6</a></sup> However, it is necessary to introduce this gene to the cell to be analyzed prior to DNA microarray analysis.</p><p>Bromouridine, which substitutes 5′ of uridine to bromine, is inexpensive and becomes incorporated into cells with very high efficiency. Additional steps, such as electroporation or lipofection, are not necessary for labeling mRNA with this compound. After its incorporation into cells, bromouridine is converted to Br-UTP. During transcription, converted Br-UTP is recognized as the same substrate as UTP and is incorporated into nascent RNAs.<sup><a reveal-id="DSN015C7" data-open="DSN015C7" class="link link-ref link-reveal xref-bibr">7</a></sup> This labeling method was applied in the immunocytochemical analysis by using anti-bromodeoxyuridine (BrdU) antibody in several studies.<sup><a reveal-id="DSN015C8  DSN015C11" data-open="DSN015C8  DSN015C11" class="link link-ref link-reveal xref-bibr">8–11</a></sup> That antibody was used also to immunoprecipitate this BrU-labeled nascent RNA.<sup><a reveal-id="DSN015C12" data-open="DSN015C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSN015C13" data-open="DSN015C13" class="link link-ref link-reveal xref-bibr">13</a></sup> The same reports analyzed transcriptional changes of some genes in small amounts (on the order of the picogram) of nascent RNAs.<sup><a reveal-id="DSN015C12" data-open="DSN015C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSN015C13" data-open="DSN015C13" class="link link-ref link-reveal xref-bibr">13</a></sup></p><p>For a comprehensive analysis using DNA microarray, we have optimized the conditions for immunoprecipitation using BrdU antibody. We have improved the specificity of nascent mRNA isolation by orders of magnitude using adequate blocking agents in immunoprecipitation. As the amount of nascent RNA obtained was very small even under the optimized purification conditions, we sought to develop a method for amplifying small amounts of mRNA with little bias and have succeeded in this goal. Finally, we have performed transcriptional analysis of nascent RNAs using G<sub>2</sub> phase-synchronized mouse mammary carcinoma FM3A cells, and we have obtained several interesting insights into cell cycle regulation of the G<sub>2</sub> phase.</p><p>These results demonstrate that DNA microarray analysis with nascent RNAs obtained by our new method will provide valuable insights into the functions of genes in various types of cells.</p>                            <h2 scrollto-destination=5489969 id="5489969" class="section-title" data-legacy-id=s2>2. Materials and Methods</h2>
                            <h3 scrollto-destination=5489970 id="5489970" class="section-title" data-legacy-id=s2a>2.1. Preparation of BrU-labeled eGFP cRNA and non-labeled luciferase cRNA</h3>
<p>eGFP cRNA and luciferase cRNA were prepared by T7 <em>in vitro</em> transcription. DNA templates for each cRNA synthesis were constructed by PCR amplification from plasmid DNA containing eGFP or luciferase genes. These templates contain both T7 promoter sequence and polyA sequence. The DNA template of eGFP was amplified from pEGFP-c1 by PCR. The DNA template of luciferase was amplified from pTRE (Clontech, Mountain View, CA, USA) by PCR. Sequences of primers were described in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">‘Supplementary data’</a>. The PCR products were purified from agarose gel by using Wizard SV gel and PCR purification kit (Promega, Madison, WI, USA).</p><p>cRNAs were transcribed with Br-UTPs (Sigma-Aldrich, St Louis, MO, USA) and NTPs using the MAXIscript<sup>®</sup> T7 kit. cRNAs were purified according to the RNA-cleanup protocol of the RNeasy Mini kit (Qiagen, Hilden, Germany).</p>                            <h3 scrollto-destination=5489973 id="5489973" class="section-title" data-legacy-id=s2b>2.2. Preparation of Mouse anti-BrdU IgG binding dynabeads</h3>
<p>Two micrograms of mouse anti-bromodeoxyuridine antibody (Roche Diagnostics, Indianapolis, IN, USA) were incubated with 25 μl Dynabeads<sup>®</sup> Goat anti-mouse IgG (Invitrogen, Carlsbad, CA, USA) in 2.0 ml collection tubes containing 100 μl DEPC-treated phosphate-buffered saline (PBS)/0.1% bovine serum albumin (BSA) solution. The tubes were rotated at room temperature for an hour. After the dynabeads were collected by a magnet rack, the dynabeads were washed three times with 1 ml DEPC-treated PBS/0.1% BSA, and 100 μl DEPC-treated PBS /0.1% BSA was added.</p>                            <h3 scrollto-destination=5489975 id="5489975" class="section-title" data-legacy-id=s2c>2.3. Immunoprecipitation of BrU-labeled RNA by antibody beads</h3>
<p>Following steps were conducted in the dark. Two hundred nanograms of BrU-labeled eGFP cRNA and 200 ng of non-labeled luciferase cRNA were denatured at 80°C for 10 min. As the blocking agent, 20 μg FM3A total RNA or 200 μg <em>Escherichia coli</em> 16S and 23S ribosomal RNA (rRNA) or uridine (final concentration of 0.3 M) was added. The denatured RNAs were added to the beads containing 225 U/ml RNasin<sup>®</sup> Plus RNase inhibitor (Promega). PBS(-)/0.1% BSA was added to this solution to a volume of 250 μl. The beads were rotated for 1 h at room temperature. After the beads were washed with 0.8 ml DEPC-treated PBS/0.1% BSA containing 45 U/ml RNasin<sup>®</sup> Plus RNase inhibitor (Promega), they were also washed with 0.8 ml DEPC-treated PBS three times. After the addition of 10 μl of RNase-free water, RNAs which bound to beads were eluted by denaturation at 80°C for 10 min, and then were kept cold on ice for 2 min.</p>                            <h3 scrollto-destination=5489977 id="5489977" class="section-title" data-legacy-id=s2d>2.4. Reverse transcription of cRNA and Real-time RT–PCR</h3>
                            <h4 scrollto-destination=5489978 id="5489978" class="section-title" data-legacy-id=s2d1>2.4.1. Reverse transcription of cRNA</h4>
<p>Five microliters of the isolated RNAs were mixed with 1 μl of 50 μM oligo(dT)<sub>20</sub> primer and denatured at 65°C for 5 min. First-strand cDNAs were synthesized from the isolated RNAs using the SuperScript<sup>®</sup> III First-Strand Synthesis System for RT–PCR (Invitrogen). After reverse transcription, 1 U of RNase H was reacted for 20 min at 37°C.</p>                            <h4 scrollto-destination=5489980 id="5489980" class="section-title" data-legacy-id=s2d2>2.4.2. Real-time quantitative RT–PCR</h4>
<p>Real-time quantitative PCR was performed in an ABI prism 7900HT Sequence Detection System (Applied Biosystems, Foster City, CA, USA) using SYBR<sup>®</sup> Premix Ex Taq™ (Perfect Real Time) (Takara Bio, Otsu, Japan). Sequences of each primer were described in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">‘Supplementary data’</a>. The PCR cycling program was set as follows: 95°C for 15 s followed by 40 cycles of 95°C for 5 s and 60°C for 34 s. One of cDNAs diluted 100 times was used in a 50 μl reaction. A standard graph for eGFP cDNA and luciferase cDNA was simultaneously generated using 10<sup>4</sup>, 10<sup>5</sup>, 10<sup>6</sup>, 10<sup>7</sup>, and 10<sup>8</sup> copies of the pEGFP plasmid and the pTRE plasmid. Each cDNA was analyzed in triplicate. Three experiments were conducted, and the averages of the cDNA copy numbers as well as standard deviation were calculated.</p>                            <h3 scrollto-destination=5489982 id="5489982" class="section-title" data-legacy-id=s2e>2.5. Cell culture of FM3A cells and labeling nascent RNA with BrU</h3>
<p>Mouse FM3A cells and tsFT210 cells, <em>Cdc2</em> temperature-sensitive mutant strain of FM3A cells,<sup><a reveal-id="DSN015C14" data-open="DSN015C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSN015C15" data-open="DSN015C15" class="link link-ref link-reveal xref-bibr">15</a></sup> were cultured in RPMI 1640 medium (Invitrogen) with 10% dialyzed bovine serum, 2 mM <span class="small-caps">l</span>-glutamine, 0.1 mM MEM non-essential amino acid (Invitrogen), and 50 U/ml penicillin–50 mg/ml streptomycin (Invitrogen) at 32°C in 5% CO<sub>2</sub>.</p><p>After bromouridine was added to the medium to a final concentration of 100 μM, FM3A cells were cultured for an hour in the dark. FM3A cells were collected in 50 ml centrifuge tubes and washed twice with cold PBS, and total RNA was isolated by using an RNeasy midi kit (Qiagen). DNase I digestion was conducted during isolation of RNA.</p>                            <h3 scrollto-destination=5489985 id="5489985" class="section-title" data-legacy-id=s2f>2.6. Immunoprecipitation of nascent RNA from mouse FM3A cells</h3>
<p>Twenty micrograms of total RNA from bromouridine-treated FM3A were immunoprecipitated with 200 μg of <em>E. coli</em> 16S and 23S rRNAs. The method of immunoprecipitation was described above. The amount of the eluted RNA was measured using the Quant-iT<sup>TM</sup> RiboGreen<sup>®</sup> RNA quantitation kit (Invitrogen). Five hundred nanometers of fluorescence of the RiboGreen reagent bound to RNA were measured using the fluorescence microplate reader ARVO (Perkin Elmer, Boston, MA, USA).</p>                            <h3 scrollto-destination=5489987 id="5489987" class="section-title" data-legacy-id=s2g>2.7. Spike-in experiment</h3>
<p>Both BrU-labeled eGFP cRNA and non-labeled luciferase cRNA were added to 20 μg of total RNA from bromouridine-treated FM3A cells. We conducted three different kinds of experiments. Each experiment was performed with a different amount of cRNAs (2 ng, 200 pg, or 20 pg). These RNAs were immunoprecipitated with 200 μg <em>E. coli</em> 16S and 23S rRNAs as blocking agents.</p>                            <h3 scrollto-destination=5489989 id="5489989" class="section-title" data-legacy-id=s2h>2.8. Validation with RT–PCR of the DNA microarray data using each amplification method</h3>
<p>Methods of experiences were described in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">‘Supplementary data’</a>.</p>                            <h3 scrollto-destination=5489991 id="5489991" class="section-title" data-legacy-id=s2i>2.9. Preparation of DNA Microarray</h3>
<p>Oligonucleotide probe sets of <em>Mus musculus</em> AROS Version 3.0 (Operon Biotechnologies, Huntsville, AL, USA) were dissolved at 10 μM in 3× SSPE solution (Invitrogen), and spotted on γ-amino propyl silane-coated UltraGAPS Coated Slides (Corning, New York, NY, USA) and UV cross-linked as described previously.<sup><a reveal-id="DSN015C1" data-open="DSN015C1" class="link link-ref link-reveal xref-bibr">1</a></sup></p>                            <h3 scrollto-destination=5489993 id="5489993" class="section-title" data-legacy-id=s2j>2.10. cDNA synthesis and amplification</h3>
<p>The amplification of RNA was performed using the Super SMART™ PCR cDNA Synthesis Kit (Clontech) as follows. We have modified the manufacture's protocol and optimized it for our DNA microarray system. Total RNA (20 ng) was mixed with 7 µl of 12 µM 3′ SMART CDS Primer II A and 7 µl of 12 µM SMART II A Oligonucleotide, incubated at 65°C for 2 min, and cooled to room temperature. To this mixture was added 20 µl of 5× First-Strand Buffer, 2 µl of 100 mM DTT, 10 µl of 10 mM of 50× dNTP, and 5 µl of PowerScript™ Reverse Transcriptase, and the reaction was incubated at 42°C for 90 min. After the reaction, 1 µl of 0.5 M EDTA was added. First-strand cDNA was purified with the NucleoSpin Extraction Kit (Clontech). For the amplification by long-distance PCR using the Advantage<sup>®</sup> 2 PCR Kit (Clontech), first-strand cDNA was mixed with 10 µl of 10× Advantage<sup>®</sup> 2 PCR Buffer, 2 µl of 10 mM 50× dNTP, 2 µl of 12 µM of 5′PCR Primer II A, and 2 µl of 50× Advantage<sup>®</sup> 2 Polymerase Mix. The PCR conditions consisted of a 95°C for 1 min, followed by appropriate number of cycles of amplification at 95°C for 15 s, 65°C for 30 s, and 68°C for 6 min. The appropriate number of cycling on which the PCR product will remain in the exponential phase of amplification was determined by the cDNA amplification results of the range of PCR cycles obtained in pre-experiments. The PCR product was purified with Wizard SV Gel and PCR Clean-Up System (Promega). For aminoallyl labeling, 1 µg cDNA was mixed with 15 µl of Random Primers Buffer Mixture (Invitrogen), incubated at 95°C for 5 min, and cooled on ice. Then, 5 μl of an aminoallyl–dNTP mixture which contained 5 mM aminoallyl–dUTP (Sigma), 5 mM dATP (GE healthcare, Uppsala, Sweden), 5 mM dGTP (GE healthcare), and 5 mM dCTP (GE healthcare) was added. Next 9 U of Klenow Fragment (Invitrogen) was added and incubated at 37°C for 120 min. After the reaction, 5 µl of Stop Buffer (Invitrogen) was added. Aminoallyl-labeled cDNA was purified with ethanol precipitation following the first purification with Wizard SV Gel and PCR Clean-Up System. For the coupling reaction, aminoallyl-labeled cDNA was dissolved with 5 µl of 0.1 M NaHCO<sub>3</sub> and mixed with 5 µl of Cy3 Mono-reactive Dye (GE healthcare) or Cy5 Mono-reactive Dye (GE healthcare) in 50 µl Atlas Glass Approved DMSO (BD Bioscience), and incubated for 60 min at room temperature. After coupling, 15 µl of 4 M hydroxylamine–HCl (Sigma-Aldrich) was added and incubated at room temperature for 15 min. Cy-Dye labeled cDNA was purified with the MinElute Reaction Cleanup Kit (Qiagen).</p>                            <h3 scrollto-destination=5489995 id="5489995" class="section-title" data-legacy-id=s2k>2.11. Hybridization</h3>
<p>These procedures were based on TF oligo microarray hybridization as described previously.<sup><a reveal-id="DSN015C1" data-open="DSN015C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Some of the conditions and compositions of solutions were changed: the hybridization solution consisted of a mixture of Cy3- and Cy5-labeled cDNAs, 40% formamide, 5× SSC, 0.4% SDS, and 1.9 μg/μl acetylated BSA (Invitrogen). Hybridization was performed with a hybridization chamber (Agilent Technologies, Palo Alto, CA, USA) and a hybridization gasket slide (Agilent Technologies). The temperature in both the pre-hybridization and hybridization steps was 45°C. Hybridization images were scanned using an Agilent DNA microarray scanner (Agilent Technologies).</p><p>We hybridized four slides as self–self hybridization: Cy3-labeled cDNAs of nascent RNA from FM3A cells and Cy5-labeled cDNA of nascent RNA from FM3A cells.</p><p>In experiments using G<sub>2</sub> phase-arrested cells, we hybridized four slides: Cy3-labeled cDNAs of asynchronous cells and Cy5-labeled cDNAs of G<sub>2</sub> phase-arrested cells were hybridized in two slides, and dye-swapped cDNAs were hybridized in the other two.</p>                            <h3 scrollto-destination=5489999 id="5489999" class="section-title" data-legacy-id=s2l>2.12. DNA microarray analyses</h3>
<p>Fluorescence intensities in spots were quantified using Feature Extraction software version GX (Agilent Technologies). Spots flagged as non-uniform outliners of either a feature (spot) or background were removed. In each array, genes whose expression levels were lower than those of the negative control spots under both conditions were removed. The ratios of signal intensities between the two cell types were calculated through Lowess normalization using GeneSpring GX (Agilent Technologies). We calculated the average and the standard deviation of the base 2 logarithm of the expression ratios of four spots and removed the expression ratios of genes whose standard deviation was &gt;1. The expression ratios of genes whose average signal intensity was &lt;50 were also removed.</p>                            <h3 scrollto-destination=5490001 id="5490001" class="section-title" data-legacy-id=s2m>2.13. Analyses of nascent RNA profiles in G<sub>2</sub>-arrested cells and asynchronous cells</h3>
<p>tsFT210 cells were arrested in the G<sub>2</sub> phase by incubation for 17 h at a non-permissive temperature (39°C). Bromouridine was added to the medium to a concentration of 100 μM at 16 h after the shift at 39°C. RNA was isolated using the RNeasy mini kit (Qiagen). DNase I digestion was conducted during isolation of RNA. The DNA microarray analyses are described above.</p>                            <h3 scrollto-destination=5490003 id="5490003" class="section-title" data-legacy-id=s2n>2.14. Cell cycle analysis</h3>
<p>Cells were harvested, washed with PBS, and fixed in 70% ethanol at −20°C for at least 4 h. Fixed cells were washed twice with PBS and mixed with PBS containing 0.3% Triton X-100 (w/v) (Sigma-Aldrich). Cells were added with PBS (−) containing 50 μg/ml propidium iodide (Sigma-Aldrich) and 0.1 mg/ml RNase A (Sigma-Aldrich), and incubated for 20 min at 37°C in the dark before being filtered through a 41 μm nylon mesh. We analyzed the DNA contents by measuring the intensity of the fluorescence produced by propidium iodide using the FACSCalibur instrument (Beckon Dickinson, Mountain View, CA, USA) using instructions supplied by the manufacturer.</p>                            <h2 scrollto-destination=5490005 id="5490005" class="section-title" data-legacy-id=s3>3. Results and Discussion</h2>
                            <h3 scrollto-destination=5490006 id="5490006" class="section-title" data-legacy-id=s3a>3.1. Experimental scheme of DNA microarray analysis for the nascent RNA</h3>
<p>The new method for performing comprehensive analyses of nascent RNA is illustrated in Fig. <a reveal-id="DSN015F1" data-open="DSN015F1" class="link link-reveal link-table xref-fig">1</a>. In the first step, bromouridine was added to the culture medium. This compound was incorporated into cells, which converted it to Br-UTP. As BrUTP is recognized as the same substrate as UTP, nascent mRNA is labeled by BrU<sup><a reveal-id="DSN015C7" data-open="DSN015C7" class="link link-ref link-reveal xref-bibr">7</a></sup> (Fig. <a reveal-id="DSN015F1" data-open="DSN015F1" class="link link-reveal link-table xref-fig">1</a>). Total RNAs isolated from the BrU-treated cells were immunoprecipitated with anti-BrdU antibody beads.<sup><a reveal-id="DSN015C12" data-open="DSN015C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSN015C13" data-open="DSN015C13" class="link link-ref link-reveal xref-bibr">13</a></sup> The RNAs bound to the beads were eluted by heat denaturation after washing, and the eluted mRNAs were reverse-transcribed using oligo-dT primer. The cDNAs were linearly amplified using the template-switching (TS) PCR method. The amplified cDNAs labeled with Cy3/Cy5 were used for DNA microarray analysis (Fig. <a reveal-id="DSN015F1" data-open="DSN015F1" class="link link-reveal link-table xref-fig">1</a>). Among these steps, the method for immunoprecipitating the BrU-labeled nascent RNAs was based on a method described previously<sup><a reveal-id="DSN015C12" data-open="DSN015C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSN015C13" data-open="DSN015C13" class="link link-ref link-reveal xref-bibr">13</a></sup> and optimized for DNA microarray analysis in the following manner. Approximately 50 pg of the BrU-labeled RNA from mouse pre-implantation embryos was obtained and subjected to the same analysis as used in a previous study.<sup><a reveal-id="DSN015C13" data-open="DSN015C13" class="link link-ref link-reveal xref-bibr">13</a></sup> The RNA amount should be &gt;20 ng in total RNA. This is ∼1000 times more than that in the previous study.</p>                        <a id="5490008" scrollto-destination="5490008"></a>
<div id="DSN015F1" data-id="DSN015F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn015/2/m_dsn01501.jpeg?Expires=2147483647&Signature=tl4ol4YiXR6hJfILPpbRje9zgRAxOlygnX4BD3xV6pubc-2~e-fh6mmehc2lrsg9kt9MfegXYdZYgE6UDyA3kksIn4SnGgAtN5Ioo36e~38AuSXtfA1fVSe6VO0rQhmcK3A2RAsbypWsRAznpP~lyXv0obLdbOLPfvjA5OLneGStiN3h5C8qplD9obFS6Icseb172vNVqQRDS5bt7PLZnZ-PfeLlt8dFa98iYwQ5iE5WaPiliex2y~uuBXtFuTd9W545cK3bD8LTVbL6t-VIEcjKekshLHYxXu7iOzur1eHSjFCeLAE8kmd9inF-URJkTqmsQi6nwAGbvJLTWbgUwA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Overview of the isolation of nascent RNAs for microarray analysis." path-from-xml="dsn01501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5490008/dsn01501.jpeg" path-from-xml="dsn01501.jpeg" target="_blank">View large</a><a section="5490008" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn015/2/dsn01501.jpeg?Expires=2147483647&Signature=K2AYxPsawLEgIR8VJJcJ3mf0TL8hdDht7E-C6lWIaHBWcwEoi8obFxJAfGFaGs82y72J4KveJpiLBVD~4hMq0Mdy0~pZDROzEjDCfODahLIGZ6KVCEZvl8V1VM5ozEDO0DTnX3MVy7q3RC4YP7aoLThKwo3BTUk30-SHrT-nem7TTIP8Xtrnd8danYwVdkhvTRehKUkbWrxyWbh4jXEGQnDtYagkS4e8swoej7S9pZZ-8SW9SR2uoBK7LNcy03VhO4Vda-gmVpCkJfPMf8QYGbHg13WpQUBrKnunyaqcDduSydMjBATLNVIuku4TUSoX1JSQiJzm5gXC-xI0jmVJhw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5490008&ar=376722&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Overview of the isolation of nascent RNAs for microarray analysis.</p></div></div><div content-id="DSN015F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn015/2/m_dsn01501.jpeg?Expires=2147483647&Signature=tl4ol4YiXR6hJfILPpbRje9zgRAxOlygnX4BD3xV6pubc-2~e-fh6mmehc2lrsg9kt9MfegXYdZYgE6UDyA3kksIn4SnGgAtN5Ioo36e~38AuSXtfA1fVSe6VO0rQhmcK3A2RAsbypWsRAznpP~lyXv0obLdbOLPfvjA5OLneGStiN3h5C8qplD9obFS6Icseb172vNVqQRDS5bt7PLZnZ-PfeLlt8dFa98iYwQ5iE5WaPiliex2y~uuBXtFuTd9W545cK3bD8LTVbL6t-VIEcjKekshLHYxXu7iOzur1eHSjFCeLAE8kmd9inF-URJkTqmsQi6nwAGbvJLTWbgUwA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Overview of the isolation of nascent RNAs for microarray analysis." path-from-xml="dsn01501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5490008/dsn01501.jpeg" path-from-xml="dsn01501.jpeg" target="_blank">View large</a><a section="5490008" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn015/2/dsn01501.jpeg?Expires=2147483647&Signature=K2AYxPsawLEgIR8VJJcJ3mf0TL8hdDht7E-C6lWIaHBWcwEoi8obFxJAfGFaGs82y72J4KveJpiLBVD~4hMq0Mdy0~pZDROzEjDCfODahLIGZ6KVCEZvl8V1VM5ozEDO0DTnX3MVy7q3RC4YP7aoLThKwo3BTUk30-SHrT-nem7TTIP8Xtrnd8danYwVdkhvTRehKUkbWrxyWbh4jXEGQnDtYagkS4e8swoej7S9pZZ-8SW9SR2uoBK7LNcy03VhO4Vda-gmVpCkJfPMf8QYGbHg13WpQUBrKnunyaqcDduSydMjBATLNVIuku4TUSoX1JSQiJzm5gXC-xI0jmVJhw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5490008&ar=376722&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Overview of the isolation of nascent RNAs for microarray analysis.</p></div></div><p>Therefore, it was necessary to develop a method for immunoprecipitating a large amount of BrU-labeled RNA with high specificity.</p>                            <h3 scrollto-destination=5490010 id="5490010" class="section-title" data-legacy-id=s3b>3.2. Determination of optimum conditions for enriching nascent RNAs using cRNAs synthesized in vitro</h3>
<p>As the amount of BrU-labeled nascent RNAs is much smaller than that of non-labeled RNAs in the RNA fractions extracted from living cells, it was necessary to develop a method for obtaining BrU-labeled RNAs with high recovery as well as on a certain scale. In the first trial, a mixture of BrU-labeled eGFP cRNAs and non-labeled luciferase cRNAs prepared by T7 <em>in vitro</em> transcription was immunoprecipitated in the same tube. cRNAs eluted from the affinity beads were reverse-transcribed with oligo-dT primer, and the copy number of each cDNA was quantified by real-time quantitative PCR (Fig. <a reveal-id="DSN015F2" data-open="DSN015F2" class="link link-reveal link-table xref-fig">2</a>A). We defined the ratio of the eGFP cDNA copy number to the luciferase cDNA copy number as the index of immunoprecipitation specificity. This index well reflects the enrichment efficiency of nascent mRNAs. We also examined the effects of adding blocking agents such as uridine, total RNA from FM3A cells, and <em>E. coli</em> 16S and 23S rRNAs. Although interferon-τ RNA was shown to work well as a blocker in a previous report,<sup><a reveal-id="DSN015C13" data-open="DSN015C13" class="link link-ref link-reveal xref-bibr">13</a></sup> we chose uridine, <em>E. coli</em> rRNA, and mouse total RNA as candidates for a blocking agent because we thought they would be more suitable for large-scale isolation of nascent RNAs. We evaluated the effects of these blocking agents with increasing amounts of cRNA. Under our conditions, the addition of uridine did not change the ratio. In contrast, the ratios observed in the experiments with <em>E. coli</em> rRNA or FM3A total RNA were approximately double those obtained in an experiment carried out without RNAs as the blocking agent (Fig. <a reveal-id="DSN015F2" data-open="DSN015F2" class="link link-reveal link-table xref-fig">2</a>B). These data suggest that any kind of RNA can increase the specificity of the immunoaffinity purification of nascent RNAs. We thus chose <em>E. coli</em> 16S and 23S rRNAs as suitable blocking agents. These bacterial rRNAs are inexpensive and do not exist in mammalian cells. In addition, <em>E. coli</em> 16S and 23S rRNAs should not be reverse-transcribed when mRNA is reverse-transcribed by using oligo-dT primer.</p>                        <a id="5490012" scrollto-destination="5490012"></a>
<div id="DSN015F2" data-id="DSN015F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn015/2/m_dsn01502.jpeg?Expires=2147483647&Signature=YUNQ5dwGgsz0VLn5PwcaFD3qvcw1Lm~htOMSqumu~1s4iYCAq1-Dj8rpzMCRpzXod4t3KL3NEN7PlnonBHowih4SeeN8EodleUDlEbTS58XcFYsdbFpqex0tw--tDMzyJKIcr8MayZmlrl4edJhb1NJRZCJ-yZoHX0vjNvdfa50NTikTmydfEMxXdTOdx0SP1EONq828eswqmsIjqblIeTIyp6zMYYtpw4oq4d6fVDceC8xiZbZosXlNSKtBWm8sFLjnKHxIvcVF~r6taGOWekUgVfkzVo-CyYDDyVanWBsczuFDrOwIcGGwz562z6CgJxd67jWjPjSo1rhQlW-0yQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Binding specificity of BrU-labeled cRNA by in vitro transcription. (A) eGFP and luciferase gene were transcribed in vitro by T7 RNA polymerase. Br-UTP was used as the substrate instead of UTP during the transcription of eGFP. (B) Mixture of 200 ng of BrU-labeled eGFP and non-labeled luciferase cRNA was immunoprecipitated with addition to blocking agents. The eluted cRNAs were converted into cDNA, and then the cDNA copy number was measured using real-time quantitative PCR. The blocking agents used are as follows: (i) no addition, (ii) uridine, (iii) E. coli 16S and 23S RNAs, and (iv) total RNA from mouse FM3A cells. ‘Ratio (eGFP/Luci.)’ represents the ratio of the eGFP cDNA copy number to the luciferase copy number. Error bars represent standard errors of the means (n = 3)." path-from-xml="dsn01502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5490012/dsn01502.jpeg" path-from-xml="dsn01502.jpeg" target="_blank">View large</a><a section="5490012" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn015/2/dsn01502.jpeg?Expires=2147483647&Signature=c0MVrl~Vzl~VOdxCBzPG-vBbAIG3PFrmdPqchf0ERCdd0C4UpQ30~ErdhmRmDhMSJvnFr6YA~xyXFKGmxL1vMO79ONnpKjfkXxZzqrY0g-KmgSRmFfTvGtbplliRkQHl-G~mjL2IzmUQqkWNHHrk07hFEP4RSZ9sGnINqhTIW8JFsGg65BsniCFEx5WBf5dhunhqhK7DytTF66qfxUtHnKTaZRC4mINkFPxV~ZJINEcEwySMKxYYh3gUxyq8EM3UATSWrmItWsYupnDLmcfk5Yv9Osh8abNOTzRNC~pLkI4K9qHClwYPXQOjv5yJcUNqcS7MFg7kIqXEohDXcEZULQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5490012&ar=376722&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Binding specificity of BrU-labeled cRNA by <em>in vitro</em> transcription. (<strong>A</strong>) eGFP and luciferase gene were transcribed <em>in vitro</em> by T7 RNA polymerase. Br-UTP was used as the substrate instead of UTP during the transcription of eGFP. (<strong>B</strong>) Mixture of 200 ng of BrU-labeled eGFP and non-labeled luciferase cRNA was immunoprecipitated with addition to blocking agents. The eluted cRNAs were converted into cDNA, and then the cDNA copy number was measured using real-time quantitative PCR. The blocking agents used are as follows: (i) no addition, (ii) uridine, (iii) <em>E. coli</em> 16S and 23S RNAs, and (iv) total RNA from mouse FM3A cells. ‘Ratio (eGFP/Luci.)’ represents the ratio of the eGFP cDNA copy number to the luciferase copy number. Error bars represent standard errors of the means (<em>n</em> = 3).</p></div></div><div content-id="DSN015F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn015/2/m_dsn01502.jpeg?Expires=2147483647&Signature=YUNQ5dwGgsz0VLn5PwcaFD3qvcw1Lm~htOMSqumu~1s4iYCAq1-Dj8rpzMCRpzXod4t3KL3NEN7PlnonBHowih4SeeN8EodleUDlEbTS58XcFYsdbFpqex0tw--tDMzyJKIcr8MayZmlrl4edJhb1NJRZCJ-yZoHX0vjNvdfa50NTikTmydfEMxXdTOdx0SP1EONq828eswqmsIjqblIeTIyp6zMYYtpw4oq4d6fVDceC8xiZbZosXlNSKtBWm8sFLjnKHxIvcVF~r6taGOWekUgVfkzVo-CyYDDyVanWBsczuFDrOwIcGGwz562z6CgJxd67jWjPjSo1rhQlW-0yQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Binding specificity of BrU-labeled cRNA by in vitro transcription. (A) eGFP and luciferase gene were transcribed in vitro by T7 RNA polymerase. Br-UTP was used as the substrate instead of UTP during the transcription of eGFP. (B) Mixture of 200 ng of BrU-labeled eGFP and non-labeled luciferase cRNA was immunoprecipitated with addition to blocking agents. The eluted cRNAs were converted into cDNA, and then the cDNA copy number was measured using real-time quantitative PCR. The blocking agents used are as follows: (i) no addition, (ii) uridine, (iii) E. coli 16S and 23S RNAs, and (iv) total RNA from mouse FM3A cells. ‘Ratio (eGFP/Luci.)’ represents the ratio of the eGFP cDNA copy number to the luciferase copy number. Error bars represent standard errors of the means (n = 3)." path-from-xml="dsn01502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5490012/dsn01502.jpeg" path-from-xml="dsn01502.jpeg" target="_blank">View large</a><a section="5490012" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn015/2/dsn01502.jpeg?Expires=2147483647&Signature=c0MVrl~Vzl~VOdxCBzPG-vBbAIG3PFrmdPqchf0ERCdd0C4UpQ30~ErdhmRmDhMSJvnFr6YA~xyXFKGmxL1vMO79ONnpKjfkXxZzqrY0g-KmgSRmFfTvGtbplliRkQHl-G~mjL2IzmUQqkWNHHrk07hFEP4RSZ9sGnINqhTIW8JFsGg65BsniCFEx5WBf5dhunhqhK7DytTF66qfxUtHnKTaZRC4mINkFPxV~ZJINEcEwySMKxYYh3gUxyq8EM3UATSWrmItWsYupnDLmcfk5Yv9Osh8abNOTzRNC~pLkI4K9qHClwYPXQOjv5yJcUNqcS7MFg7kIqXEohDXcEZULQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5490012&ar=376722&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Binding specificity of BrU-labeled cRNA by <em>in vitro</em> transcription. (<strong>A</strong>) eGFP and luciferase gene were transcribed <em>in vitro</em> by T7 RNA polymerase. Br-UTP was used as the substrate instead of UTP during the transcription of eGFP. (<strong>B</strong>) Mixture of 200 ng of BrU-labeled eGFP and non-labeled luciferase cRNA was immunoprecipitated with addition to blocking agents. The eluted cRNAs were converted into cDNA, and then the cDNA copy number was measured using real-time quantitative PCR. The blocking agents used are as follows: (i) no addition, (ii) uridine, (iii) <em>E. coli</em> 16S and 23S RNAs, and (iv) total RNA from mouse FM3A cells. ‘Ratio (eGFP/Luci.)’ represents the ratio of the eGFP cDNA copy number to the luciferase copy number. Error bars represent standard errors of the means (<em>n</em> = 3).</p></div></div>                            <h3 scrollto-destination=5490013 id="5490013" class="section-title" data-legacy-id=s3c>3.3. Conditions for labeling nascent RNAs with BrU</h3>
<p>Next, we treated living cells with bromouridine and immunoprecipitated BrU-labeled nascent RNAs from BrU-treated cells. We first tested and determined the appropriate concentration (100 μM) of bromouridine, i.e. that which did not affect proliferation and cell cycle progression in mouse FM3A cells (data not shown). Using DNA microarray analysis, we also confirmed that the gene expression profile of FM3A cells after treatment with 100 μM bromouridine for 4 h was extensively the same as that of untreated cells (data not shown). Approximately 20 μg total RNA was isolated from 2 × 10<sup>6</sup> FM3A cells that had been treated with 100 μM bromouridine for 1 h; this total RNA was then subjected to immunoaffinity purification. We used <em>E. coli</em> 16S and 23S rRNAs as blocking agents. The amount of eluted RNAs in the sample of BrU-treated cells was ∼30 ng, so we had successfully obtained a sufficient amount of BrU-labeled RNA for DNA microarray analyses. We then evaluated the specificity of immunoaffinity purification.</p>                            <h3 scrollto-destination=5490015 id="5490015" class="section-title" data-legacy-id=s3d>3.4. Evaluation of purification efficiency with spike-in control</h3>
<p>To more precisely estimate the efficiency with which nascent RNAs were purified, small amounts of T7 <em>in vitro</em> transcribed BrU-labeled eGFP cRNA and non-labeled luciferase cRNA were added to the RNAs obtained from BrU-treated FM3A cell samples as the ‘spike-in’ control. We immunoprecipitated these RNA mixtures and compared the cDNA copy numbers of eluted eGFP cDNA and luciferase cDNA. To determine whether or not the amount of the eluted BrU-labeled eGFP cRNA was proportional to that of the input eGFP cRNA, increasing amounts (20 pg, 200 pg, and 2 ng) of eGFP cRNA and luciferase cRNA were added to 20 μg of BrU-labeled total RNA obtained from FM3A cells (Fig. <a reveal-id="DSN015F3" data-open="DSN015F3" class="link link-reveal link-table xref-fig">3</a>A). The ratio of eluted eGFP cDNA to eluted luciferase cDNA exceeded 83 under the conditions shown in Fig. <a reveal-id="DSN015F3" data-open="DSN015F3" class="link link-reveal link-table xref-fig">3</a>B. In addition, the copy number of eluted BrU-labeled eGFP cRNA was proportional to that of the input eGFP cRNA. These results indicated that we were able to immunoaffinity-purify BrU-labeled RNA with high specificity and that the eluted RNA reflects the nascent RNA before immunoprecipitation. Taken together, these findings indicate that we have successfully optimized the conditions for immunoaffinity-purification with high specificity. It was already demonstrated that the enrichment process using anti-BrdU immunoaffinity beads does not have any bias.<sup><a reveal-id="DSN015C13" data-open="DSN015C13" class="link link-ref link-reveal xref-bibr">13</a></sup> As it is important to confirm that the cDNA amplification process does not have any significant bias, we performed the following experiments.</p>                        <a id="5490017" scrollto-destination="5490017"></a>
<div id="DSN015F3" data-id="DSN015F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn015/2/m_dsn01503.jpeg?Expires=2147483647&Signature=C4h7UmHPn1pU6oiAqwYVz~dnJr9AOO0Tvw1Z8bocJgSVtlF4NHh9DMJ9zpy3wFUzu8ui3y5DqrD9wdKNsQdKXshRYN2y8u1jS~cHvH-y95njdp4F7UYwqoEBnf6jGLhmfc2lsAo3oaTrmO8LGDomxwkpwCTk5z5HPqaQEO0w~~1xKzb-wgDfJfcLszoTlwjl0s-RogRrtPuvigzG8Cq52cjGZCM7Q8sX3YFu8fkKNtCeHXY~ka8boIcaCXhZXxwPZU6FO9~96sQROLTB4N0r07MYgQhiWLrP9bsvcpHgEyQMnLDuLWAaFIWkveU3NEjrCO56Z-754Suu92vsxn~3kw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Binding specificity in immunoprecipitation with spike-in controls. (A) Overview of the experiment. (B) Br-eGFP and luciferase cRNA were prepared as spike-in controls. ‘Ratio (eGFP/Luci.)’ represents the ratio of the eGFP cDNA copy number to the luciferase copy number. Error bars represent standard errors of the means (n = 3)." path-from-xml="dsn01503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5490017/dsn01503.jpeg" path-from-xml="dsn01503.jpeg" target="_blank">View large</a><a section="5490017" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn015/2/dsn01503.jpeg?Expires=2147483647&Signature=4mTJyEB34pG809LRk3lGbzN57QNgGYs3Kc5KCjkaE4BiUzVyaJzZ46Z4iglYuARbKVj1BPRnonCjREY8AFbNPRaND7XFYYLmHAkNg8e4j8NBPBYdY-A4BGb~Z7EzJKv1bRxGizHR2ahSKprvZIIwcw3oBmhT4uWVpT4EAgMv5-4EKtDqmyxgdr6NOOUbZPT5jquecS43~BHkS4DgVtWjdpyXO-TqQflvzQHWfkaXkzEdtIt1FjSicwqXkhgFJXyJBU699I7Lh4UDGXtlE6FjwFm0u~7DZVmD4rPB~756HZD9473DE2ya~DYufwgmG2nd8pQhvOLpc0c59w83p5dAaA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5490017&ar=376722&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Binding specificity in immunoprecipitation with spike-in controls. (<strong>A</strong>) Overview of the experiment. (<strong>B</strong>) Br-eGFP and luciferase cRNA were prepared as spike-in controls. ‘Ratio (eGFP/Luci.)’ represents the ratio of the eGFP cDNA copy number to the luciferase copy number. Error bars represent standard errors of the means (n = 3).</p></div></div><div content-id="DSN015F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn015/2/m_dsn01503.jpeg?Expires=2147483647&Signature=C4h7UmHPn1pU6oiAqwYVz~dnJr9AOO0Tvw1Z8bocJgSVtlF4NHh9DMJ9zpy3wFUzu8ui3y5DqrD9wdKNsQdKXshRYN2y8u1jS~cHvH-y95njdp4F7UYwqoEBnf6jGLhmfc2lsAo3oaTrmO8LGDomxwkpwCTk5z5HPqaQEO0w~~1xKzb-wgDfJfcLszoTlwjl0s-RogRrtPuvigzG8Cq52cjGZCM7Q8sX3YFu8fkKNtCeHXY~ka8boIcaCXhZXxwPZU6FO9~96sQROLTB4N0r07MYgQhiWLrP9bsvcpHgEyQMnLDuLWAaFIWkveU3NEjrCO56Z-754Suu92vsxn~3kw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Binding specificity in immunoprecipitation with spike-in controls. (A) Overview of the experiment. (B) Br-eGFP and luciferase cRNA were prepared as spike-in controls. ‘Ratio (eGFP/Luci.)’ represents the ratio of the eGFP cDNA copy number to the luciferase copy number. Error bars represent standard errors of the means (n = 3)." path-from-xml="dsn01503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5490017/dsn01503.jpeg" path-from-xml="dsn01503.jpeg" target="_blank">View large</a><a section="5490017" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn015/2/dsn01503.jpeg?Expires=2147483647&Signature=4mTJyEB34pG809LRk3lGbzN57QNgGYs3Kc5KCjkaE4BiUzVyaJzZ46Z4iglYuARbKVj1BPRnonCjREY8AFbNPRaND7XFYYLmHAkNg8e4j8NBPBYdY-A4BGb~Z7EzJKv1bRxGizHR2ahSKprvZIIwcw3oBmhT4uWVpT4EAgMv5-4EKtDqmyxgdr6NOOUbZPT5jquecS43~BHkS4DgVtWjdpyXO-TqQflvzQHWfkaXkzEdtIt1FjSicwqXkhgFJXyJBU699I7Lh4UDGXtlE6FjwFm0u~7DZVmD4rPB~756HZD9473DE2ya~DYufwgmG2nd8pQhvOLpc0c59w83p5dAaA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5490017&ar=376722&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Binding specificity in immunoprecipitation with spike-in controls. (<strong>A</strong>) Overview of the experiment. (<strong>B</strong>) Br-eGFP and luciferase cRNA were prepared as spike-in controls. ‘Ratio (eGFP/Luci.)’ represents the ratio of the eGFP cDNA copy number to the luciferase copy number. Error bars represent standard errors of the means (n = 3).</p></div></div>                            <h3 scrollto-destination=5490018 id="5490018" class="section-title" data-legacy-id=s3e>3.5. DNA microarray analyses of nascent RNAs</h3>
<p>To carry out DNA microarray analysis, nascent RNAs of BrU-treated cells have to be amplified without bias. We previously developed a method for effectively amplifying small amounts of RNA using the TS-PCR reaction<sup><a reveal-id="DSN015C16" data-open="DSN015C16" class="link link-ref link-reveal xref-bibr">16</a></sup> (Fig. <a reveal-id="DSN015F4" data-open="DSN015F4" class="link link-reveal link-table xref-fig">4</a>A). To examine whether or not this amplification method has any amplification bias, we compared the data obtained by DNA microarray analysis with those obtained by quantitative real-time PCR using RNA samples prepared from ES cells.<sup><a reveal-id="DSN015C1" data-open="DSN015C1" class="link link-ref link-reveal xref-bibr">1</a></sup> We considered that the efficiency of amplification would vary according to the differences in cellular RNA abundance. Genes with abundant RNA would be amplified without bias, but genes with low amounts of RNA might fail to be amplified accurately. As the expression levels of transcription-factor genes are generally much lower than those of genes in other categories, we focused on expression changes of the transcription factor genes. We used the TF oligo microarray, which can accurately determine expression changes of transcription factor genes.<sup><a reveal-id="DSN015C1" data-open="DSN015C1" class="link link-ref link-reveal xref-bibr">1</a></sup> As shown in Fig. <a reveal-id="DSN015F4" data-open="DSN015F4" class="link link-reveal link-table xref-fig">4</a>B, the expression changes in TS-PCR amplification were detected accurately, without amplification bias, except for <em>Prdx1</em>, <em>Rara.</em> Especially, the fold changes of <em>Dcx</em>, <em>Meis1</em>, and <em>Ttyh1</em> obtained by TS PCR were not only quite similar to the data obtained without amplification, but were also similar to the data from quantitative real-time PCR, whereas amplification by <em>in vitro</em> transcription with T7 promoter exhibited some discrepancies. Thus, amplification by TS PCR has very little bias (Fig. <a reveal-id="DSN015F4" data-open="DSN015F4" class="link link-reveal link-table xref-fig">4</a>B).</p><p>However, there was a possibility that amplification efficiency and the reverse-transcription efficiency of BrU-labeled mRNA were different from that of non-labeled mRNA, so we compared the agarose-gel electrophoresis pattern of amplified cDNA from BrU-labeled mRNA (nascent mRNA) with that from non-labeled mRNA (steady-state mRNA). The result obtained by agarose-gel electrophoresis experiment of amplified cDNA from BrU-labeled mRNA shows a smear pattern identical to or very similar to that of amplified cDNA from non-labeled mRNA (data not shown). This result suggests that both the reverse-transcription efficiency and the amplification efficiency were almost identical between BrU-labeled RNA and non-labeled RNA. A self–self hybridization experiment was conducted with amplified cDNA labeled with Cy3/Cy5, and we detected gene expression levels with broad signal ranges. The ratios of Cy5 to Cy3 in all genes were in the range of 0.5–2.0 (Fig. <a reveal-id="DSN015F4" data-open="DSN015F4" class="link link-reveal link-table xref-fig">4</a>C), indicating that our DNA microarray analysis is highly reproducible.</p>                        <a id="5490021" scrollto-destination="5490021"></a>
<div id="DSN015F4" data-id="DSN015F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn015/2/m_dsn01504.jpeg?Expires=2147483647&Signature=wA8q6BGlyPtSbhbxYSDWFTOIJISK1qaPbvQJa8emf62UuIBICh2SFm4aaMDOHUUoZHUIF4nICjdfWqDTFPM7iQzSCwk-vYeo0uaKcVVkSBjJGAUegDEW6POjFjW28bDkm0ijeGlZtOtWI71dPw661iKeh1pEVWlhhOZ~FEZi24jGq39AfL2geh3nJmIxLQkrbv3UU5SdaW~WGstdnJsn3gpH-gaOcusUtq~BddnKpTfoS3FzxWcAAN9agtkWEc2TD8fMOtS0bVwNZ~8tPZhjW51NnZ~BEy~dQxO0JPGrjeSp42i~Mlj4WdLtIGmjl3tCdBda8vwVS~sBNVWRdYXGaA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Eluted RNAs were converted to cDNA, and cDNA was amplified by TS-PCR. (B) Validation with quantitative RT–PCR of the DNA microarray data using each amplification method. Genes expressed at various levels in the non-amplification method, TS-PCR amplification method, and T7 in vitro transcription (IVT) amplification method were confirmed using quantitative RT–PCR. Quantitative RT–PCR was performed using the total RNA of these transcription factors. Gapd was used as the control. The same total RNA was used for quantitative RT–PCR and DNA microarray experiments. The ratio of the total target RNA levels was normalized by using those of the Tbp (Mus musculus TATA box binding protein) gene, and the fold change for each gene in the differentiated ES cells compared to the control ES cells was calculated. Error bars represent standard deviations of log (base 2) of ratio. (C) Amplified cDNA labeled with Cy3/Cy5 underwent self-self hybridization." path-from-xml="dsn01504.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5490021/dsn01504.jpeg" path-from-xml="dsn01504.jpeg" target="_blank">View large</a><a section="5490021" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn015/2/dsn01504.jpeg?Expires=2147483647&Signature=lKnMBaJQT7Yo816vYsi66ZVlVwdhVWURlQHrjsEjEEkeC2J-9PR5e9ofvS2Cy80RodwqXmvDSK6c8PsULR2y9WpncGjbXFx4umyyqDRJOrZPzohUNP-GD3mPhpjMJK4DcuPMpTi3AtItWtUT~mWwRqATBQTJInWJdYiFqEYroyNlwdfY0djwBHZI2C-HmOwU~8vdNKgHJIHRDz6UtOqQIzfTbgzDeT7kMOfse8Zh3koqd7G8oh83QK9icwYapyWqYEptPCl3~GemEwgDrYe08NP00o-jIkBJD18xyjKW0OGW9SuJRKo-fxy9i63BFPRMI9m2ktkFoica7I81HO-YKQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5490021&ar=376722&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01504.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(<strong>A</strong>) Eluted RNAs were converted to cDNA, and cDNA was amplified by TS-PCR. (<strong>B</strong>) Validation with quantitative RT–PCR of the DNA microarray data using each amplification method. Genes expressed at various levels in the non-amplification method, TS-PCR amplification method, and T7 <em>in vitro</em> transcription (IVT) amplification method were confirmed using quantitative RT–PCR. Quantitative RT–PCR was performed using the total RNA of these transcription factors. <em>Gapd</em> was used as the control. The same total RNA was used for quantitative RT–PCR and DNA microarray experiments. The ratio of the total target RNA levels was normalized by using those of the <em>Tbp</em> (<em>Mus musculus</em> TATA box binding protein) gene, and the fold change for each gene in the differentiated ES cells compared to the control ES cells was calculated. Error bars represent standard deviations of log (base 2) of ratio. (<strong>C</strong>) Amplified cDNA labeled with Cy3/Cy5 underwent self-self hybridization.</p></div></div><div content-id="DSN015F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn015/2/m_dsn01504.jpeg?Expires=2147483647&Signature=wA8q6BGlyPtSbhbxYSDWFTOIJISK1qaPbvQJa8emf62UuIBICh2SFm4aaMDOHUUoZHUIF4nICjdfWqDTFPM7iQzSCwk-vYeo0uaKcVVkSBjJGAUegDEW6POjFjW28bDkm0ijeGlZtOtWI71dPw661iKeh1pEVWlhhOZ~FEZi24jGq39AfL2geh3nJmIxLQkrbv3UU5SdaW~WGstdnJsn3gpH-gaOcusUtq~BddnKpTfoS3FzxWcAAN9agtkWEc2TD8fMOtS0bVwNZ~8tPZhjW51NnZ~BEy~dQxO0JPGrjeSp42i~Mlj4WdLtIGmjl3tCdBda8vwVS~sBNVWRdYXGaA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Eluted RNAs were converted to cDNA, and cDNA was amplified by TS-PCR. (B) Validation with quantitative RT–PCR of the DNA microarray data using each amplification method. Genes expressed at various levels in the non-amplification method, TS-PCR amplification method, and T7 in vitro transcription (IVT) amplification method were confirmed using quantitative RT–PCR. Quantitative RT–PCR was performed using the total RNA of these transcription factors. Gapd was used as the control. The same total RNA was used for quantitative RT–PCR and DNA microarray experiments. The ratio of the total target RNA levels was normalized by using those of the Tbp (Mus musculus TATA box binding protein) gene, and the fold change for each gene in the differentiated ES cells compared to the control ES cells was calculated. Error bars represent standard deviations of log (base 2) of ratio. (C) Amplified cDNA labeled with Cy3/Cy5 underwent self-self hybridization." path-from-xml="dsn01504.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5490021/dsn01504.jpeg" path-from-xml="dsn01504.jpeg" target="_blank">View large</a><a section="5490021" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn015/2/dsn01504.jpeg?Expires=2147483647&Signature=lKnMBaJQT7Yo816vYsi66ZVlVwdhVWURlQHrjsEjEEkeC2J-9PR5e9ofvS2Cy80RodwqXmvDSK6c8PsULR2y9WpncGjbXFx4umyyqDRJOrZPzohUNP-GD3mPhpjMJK4DcuPMpTi3AtItWtUT~mWwRqATBQTJInWJdYiFqEYroyNlwdfY0djwBHZI2C-HmOwU~8vdNKgHJIHRDz6UtOqQIzfTbgzDeT7kMOfse8Zh3koqd7G8oh83QK9icwYapyWqYEptPCl3~GemEwgDrYe08NP00o-jIkBJD18xyjKW0OGW9SuJRKo-fxy9i63BFPRMI9m2ktkFoica7I81HO-YKQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5490021&ar=376722&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01504.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(<strong>A</strong>) Eluted RNAs were converted to cDNA, and cDNA was amplified by TS-PCR. (<strong>B</strong>) Validation with quantitative RT–PCR of the DNA microarray data using each amplification method. Genes expressed at various levels in the non-amplification method, TS-PCR amplification method, and T7 <em>in vitro</em> transcription (IVT) amplification method were confirmed using quantitative RT–PCR. Quantitative RT–PCR was performed using the total RNA of these transcription factors. <em>Gapd</em> was used as the control. The same total RNA was used for quantitative RT–PCR and DNA microarray experiments. The ratio of the total target RNA levels was normalized by using those of the <em>Tbp</em> (<em>Mus musculus</em> TATA box binding protein) gene, and the fold change for each gene in the differentiated ES cells compared to the control ES cells was calculated. Error bars represent standard deviations of log (base 2) of ratio. (<strong>C</strong>) Amplified cDNA labeled with Cy3/Cy5 underwent self-self hybridization.</p></div></div>                            <h3 scrollto-destination=5490022 id="5490022" class="section-title" data-legacy-id=s3f>3.6. DNA microarray analysis of nascent RNA in mouse G<sub>2</sub> phase-arrested FT210 cells</h3>
<p>We then compared the expression profiles of nascent RNAs of 32 000 transcript products between asynchronous mouse FM3A cells and G<sub>2</sub> phase-synchronized FM3A cells. To obtain a highly synchronized population of G<sub>2</sub> phase cells, we chose tsFT210 cells, which are temperature-sensitive <em>Cdc2</em> mutant cells of mouse mammary carcinoma FM3A cells.<sup><a reveal-id="DSN015C14" data-open="DSN015C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSN015C15" data-open="DSN015C15" class="link link-ref link-reveal xref-bibr">15</a></sup> tsFT210 cells are grown at 32°C (the permissive temperature), but in the G<sub>2</sub> phase, they are arrested at 39°C (Fig. <a reveal-id="DSN015F5" data-open="DSN015F5" class="link link-reveal link-table xref-fig">5</a>A). With this cell line, a highly synchronized population of cells is easily available by just shifting the culture temperature. It has already been shown that such synchronization is merely a result of the inactivation of just one protein (CDC2 protein).<sup><a reveal-id="DSN015C15" data-open="DSN015C15" class="link link-ref link-reveal xref-bibr">15</a></sup> We also analyzed steady-state RNA expression profiles of asynchronous cells and G<sub>2</sub>-synchronized cells. Scatter plots of the expression level are shown in Fig. <a reveal-id="DSN015F5" data-open="DSN015F5" class="link link-reveal link-table xref-fig">5</a>B. Considering the variation of DNA microarray data, we regarded the genes that showed changes exceeding twofold as either up- or down-regulated genes. Figure <a reveal-id="DSN015F5" data-open="DSN015F5" class="link link-reveal link-table xref-fig">5</a>C shows Venn diagram analyses of genes whose expression was up- or down-regulated more than twofold in nascent RNA and steady-state RNA. There was no gene whose nascent RNA was up-regulated, whereas its steady-state RNA was down-regulated, and vice versa.</p>                        <a id="5490024" scrollto-destination="5490024"></a>
<div id="DSN015F5" data-id="DSN015F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn015/2/m_dsn01505.jpeg?Expires=2147483647&Signature=KLUGQuCy3lIPLbYAFSkNTr0Nw2ownDeLk3oHOh-1sYfD71oSzyM~LdzhukSc~Adh0q-cgESu38hVW~6BrCcjv-mMFXHEUAdq-TydK68UY-GXNTF6ZmTtp2M3ZNHuRVfQkF1Xc4-ETNm-V4XndHlu-XAa6m1xV8VU~zV5YcGK6QWvChzbm2h0P70lvuAaKe33Nb~mwVriQIqosO0ybc5ya34XKO0G5VBGc7Vo55~tSDSTdzRzXu2iuVT3wYnCnKGPx7SYdzQmwM5iI1Cwv~Om058Y-gSQ6-UopxguFA~HCBHt9kXTg89SGIl7aP45J623aWyCF2HuGYYlY-HKpxLNpg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="DNA microarray analyses of nascent RNAs. (A) FT210 cells were synchronized in the G2 phase by incubation at 39 °C for 17 h. Cell cycle profiles of asynchronous cells and G2-arrested cells were analyzed by propidium iodide staining and flow cytometry. (B) Scatter plots comparing the expression profiles of asynchronous cells with those of G2-arrested cells in a nascent RNA profile (left panel) and a steady-state RNA profile (right panel). (C) The Venn diagrams show the overlap of genes whose expression levels changed more than twofold in the nascent RNA profile and the steady-state RNA profile." path-from-xml="dsn01505.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5490024/dsn01505.jpeg" path-from-xml="dsn01505.jpeg" target="_blank">View large</a><a section="5490024" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn015/2/dsn01505.jpeg?Expires=2147483647&Signature=IQ~QKli4a9QzwE1Qqstp0kZAFvlgco-yES0nqupyTZEzTmHalE0tf1rXstE~ZPIOqJuzkske11v0ag90HEDBi48PVk8096hC0UtXdQOYci5RV1FxP7Q80N~tip0EB4-BiAIOxFI6Xk0a4TR8xJ9~s3HLWWjeo9B~CdqwCqGEiM~aO~827aPlg3quzTRxvkRkp910B1oNh1~4xcsx9R6IcvopOMWLVkIkiPm2Q68JItJS0mULTwbKUc3S6ND39BH~ymWh0-lAIFPCGvD89kncvKeTPmR~zoZU8Jp3bq~jSU1gjVld~0TJnQXt5x-1hiXmP8jyblkTE0gJ~Mi4pRj6lA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5490024&ar=376722&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01505.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>DNA microarray analyses of nascent RNAs. (<strong>A</strong>) FT210 cells were synchronized in the G<sub>2</sub> phase by incubation at 39 °C for 17 h. Cell cycle profiles of asynchronous cells and G<sub>2</sub>-arrested cells were analyzed by propidium iodide staining and flow cytometry. (<strong>B</strong>) Scatter plots comparing the expression profiles of asynchronous cells with those of G<sub>2</sub>-arrested cells in a nascent RNA profile (left panel) and a steady-state RNA profile (right panel). (<strong>C</strong>) The Venn diagrams show the overlap of genes whose expression levels changed more than twofold in the nascent RNA profile and the steady-state RNA profile.</p></div></div><div content-id="DSN015F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn015/2/m_dsn01505.jpeg?Expires=2147483647&Signature=KLUGQuCy3lIPLbYAFSkNTr0Nw2ownDeLk3oHOh-1sYfD71oSzyM~LdzhukSc~Adh0q-cgESu38hVW~6BrCcjv-mMFXHEUAdq-TydK68UY-GXNTF6ZmTtp2M3ZNHuRVfQkF1Xc4-ETNm-V4XndHlu-XAa6m1xV8VU~zV5YcGK6QWvChzbm2h0P70lvuAaKe33Nb~mwVriQIqosO0ybc5ya34XKO0G5VBGc7Vo55~tSDSTdzRzXu2iuVT3wYnCnKGPx7SYdzQmwM5iI1Cwv~Om058Y-gSQ6-UopxguFA~HCBHt9kXTg89SGIl7aP45J623aWyCF2HuGYYlY-HKpxLNpg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="DNA microarray analyses of nascent RNAs. (A) FT210 cells were synchronized in the G2 phase by incubation at 39 °C for 17 h. Cell cycle profiles of asynchronous cells and G2-arrested cells were analyzed by propidium iodide staining and flow cytometry. (B) Scatter plots comparing the expression profiles of asynchronous cells with those of G2-arrested cells in a nascent RNA profile (left panel) and a steady-state RNA profile (right panel). (C) The Venn diagrams show the overlap of genes whose expression levels changed more than twofold in the nascent RNA profile and the steady-state RNA profile." path-from-xml="dsn01505.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5490024/dsn01505.jpeg" path-from-xml="dsn01505.jpeg" target="_blank">View large</a><a section="5490024" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn015/2/dsn01505.jpeg?Expires=2147483647&Signature=IQ~QKli4a9QzwE1Qqstp0kZAFvlgco-yES0nqupyTZEzTmHalE0tf1rXstE~ZPIOqJuzkske11v0ag90HEDBi48PVk8096hC0UtXdQOYci5RV1FxP7Q80N~tip0EB4-BiAIOxFI6Xk0a4TR8xJ9~s3HLWWjeo9B~CdqwCqGEiM~aO~827aPlg3quzTRxvkRkp910B1oNh1~4xcsx9R6IcvopOMWLVkIkiPm2Q68JItJS0mULTwbKUc3S6ND39BH~ymWh0-lAIFPCGvD89kncvKeTPmR~zoZU8Jp3bq~jSU1gjVld~0TJnQXt5x-1hiXmP8jyblkTE0gJ~Mi4pRj6lA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5490024&ar=376722&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01505.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>DNA microarray analyses of nascent RNAs. (<strong>A</strong>) FT210 cells were synchronized in the G<sub>2</sub> phase by incubation at 39 °C for 17 h. Cell cycle profiles of asynchronous cells and G<sub>2</sub>-arrested cells were analyzed by propidium iodide staining and flow cytometry. (<strong>B</strong>) Scatter plots comparing the expression profiles of asynchronous cells with those of G<sub>2</sub>-arrested cells in a nascent RNA profile (left panel) and a steady-state RNA profile (right panel). (<strong>C</strong>) The Venn diagrams show the overlap of genes whose expression levels changed more than twofold in the nascent RNA profile and the steady-state RNA profile.</p></div></div><p>Among the genes whose nascent RNA levels changed more than twofold, cell cycle progression-related genes are listed in Table <a reveal-id="DSN015TB1" data-open="DSN015TB1" class="link link-reveal link-table xref-fig">1</a>. Of the 18 genes listed, 14 genes exhibited fold changes in the analysis of nascent RNA but not in that of steady-state RNA.</p>                        <a id="5490026" scrollto-destination="5490026"></a>
<div content-id="DSN015TB1" class="table-modal"><div class="table-wrap"><div id="DSN015TB1" class="table-wrap-title" data-id="DSN015TB1"><div class="label -label">Table 1</div><div class="caption"><p>The partial lists of cell cycle related genes whose level of nascent RNAs changed more than twofold in G2 arrested cells</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene symbol </th><th>Fold changes of nascent RNA </th><th>Fold changes of steady-state RNA </th><th>Description </th></tr></thead><tbody><tr><td><em>Cdkn1a</em> </td><td>2.00 </td><td>3.29 </td><td>Cyclin-dependent kinase inhibitor 1A (P21) </td></tr><tr><td><em>Pdgfa</em> </td><td>2.41 </td><td>3.02 </td><td>Platelet derived growth factor, alpha </td></tr><tr><td><em>Ppm1b</em> </td><td>2.15 </td><td>1.10 </td><td>Protein phosphatase 1B, magnesium dependent, beta isoform </td></tr><tr><td><em>Bin1</em> </td><td>2.53 </td><td>1.41 </td><td>Bridging integrator 1 </td></tr><tr><td><em>Cdkl2</em> </td><td>2.16 </td><td>1.68 </td><td>Cyclin-dependent kinase-like 2 (CDC2-related kinase) </td></tr><tr><td><em>Arhgef5</em> </td><td>2.26 </td><td>0.90 </td><td>Rho guanine nucleotide exchange factor (GEF) 5 </td></tr><tr><td><em>Ppp1r15b</em> </td><td>2.03 </td><td>0.92 </td><td>Protein phosphatase 1, regulatory (inhibitor) subunit 15b </td></tr><tr><td><em>Klf3</em> </td><td>2.44 </td><td>1.23 </td><td>Krüppel-like factor 3 (basic) </td></tr><tr><td><em>Pak2</em> </td><td>2.11 </td><td>0.94 </td><td>P21 (CDKN1A)-activated kinase 2 </td></tr><tr><td><em>Pdpk1</em> </td><td>3.30 </td><td>1.04 </td><td>3-phosphoinositide dependent protein kinase-1 </td></tr><tr><td><em>E2F5</em> </td><td>2.13 </td><td>0.83 </td><td>E2F transcription factor 5 </td></tr><tr><td><em>Pak1</em> </td><td>2.09 </td><td>1.47 </td><td>P21 (CDKN1A)-activated kinase 1 </td></tr><tr><td><em>Gchfr</em> </td><td>2.84 </td><td>1.12 </td><td>GTP cyclohydrolase I feedback regulator </td></tr><tr><td><em>Tk1</em> </td><td>0.41 </td><td>0.46 </td><td>Thymidine kinase 1 </td></tr><tr><td><em>Ak1</em> </td><td>0.40 </td><td>0.48 </td><td>Adenylate kinase 1 </td></tr><tr><td><em>Dusp1</em> </td><td>0.38 </td><td>1.52 </td><td>Dual specificity phosphatase 1 </td></tr><tr><td><em>Stk17b</em> </td><td>0.38 </td><td>0.60 </td><td>Serine/threonine kinase 17b (apoptosis-inducing) </td></tr><tr><td><em>Pnrc2</em> </td><td>0.29 </td><td>0.89 </td><td>Proline-rich nuclear receptor co-activator 2 </td></tr></tbody></table></div><div content-id="DSN015TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene symbol </th><th>Fold changes of nascent RNA </th><th>Fold changes of steady-state RNA </th><th>Description </th></tr></thead><tbody><tr><td><em>Cdkn1a</em> </td><td>2.00 </td><td>3.29 </td><td>Cyclin-dependent kinase inhibitor 1A (P21) </td></tr><tr><td><em>Pdgfa</em> </td><td>2.41 </td><td>3.02 </td><td>Platelet derived growth factor, alpha </td></tr><tr><td><em>Ppm1b</em> </td><td>2.15 </td><td>1.10 </td><td>Protein phosphatase 1B, magnesium dependent, beta isoform </td></tr><tr><td><em>Bin1</em> </td><td>2.53 </td><td>1.41 </td><td>Bridging integrator 1 </td></tr><tr><td><em>Cdkl2</em> </td><td>2.16 </td><td>1.68 </td><td>Cyclin-dependent kinase-like 2 (CDC2-related kinase) </td></tr><tr><td><em>Arhgef5</em> </td><td>2.26 </td><td>0.90 </td><td>Rho guanine nucleotide exchange factor (GEF) 5 </td></tr><tr><td><em>Ppp1r15b</em> </td><td>2.03 </td><td>0.92 </td><td>Protein phosphatase 1, regulatory (inhibitor) subunit 15b </td></tr><tr><td><em>Klf3</em> </td><td>2.44 </td><td>1.23 </td><td>Krüppel-like factor 3 (basic) </td></tr><tr><td><em>Pak2</em> </td><td>2.11 </td><td>0.94 </td><td>P21 (CDKN1A)-activated kinase 2 </td></tr><tr><td><em>Pdpk1</em> </td><td>3.30 </td><td>1.04 </td><td>3-phosphoinositide dependent protein kinase-1 </td></tr><tr><td><em>E2F5</em> </td><td>2.13 </td><td>0.83 </td><td>E2F transcription factor 5 </td></tr><tr><td><em>Pak1</em> </td><td>2.09 </td><td>1.47 </td><td>P21 (CDKN1A)-activated kinase 1 </td></tr><tr><td><em>Gchfr</em> </td><td>2.84 </td><td>1.12 </td><td>GTP cyclohydrolase I feedback regulator </td></tr><tr><td><em>Tk1</em> </td><td>0.41 </td><td>0.46 </td><td>Thymidine kinase 1 </td></tr><tr><td><em>Ak1</em> </td><td>0.40 </td><td>0.48 </td><td>Adenylate kinase 1 </td></tr><tr><td><em>Dusp1</em> </td><td>0.38 </td><td>1.52 </td><td>Dual specificity phosphatase 1 </td></tr><tr><td><em>Stk17b</em> </td><td>0.38 </td><td>0.60 </td><td>Serine/threonine kinase 17b (apoptosis-inducing) </td></tr><tr><td><em>Pnrc2</em> </td><td>0.29 </td><td>0.89 </td><td>Proline-rich nuclear receptor co-activator 2 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5490026" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSN015TB1" class="table-wrap-title" data-id="DSN015TB1"><div class="label -label">Table 1</div><div class="caption"><p>The partial lists of cell cycle related genes whose level of nascent RNAs changed more than twofold in G2 arrested cells</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene symbol </th><th>Fold changes of nascent RNA </th><th>Fold changes of steady-state RNA </th><th>Description </th></tr></thead><tbody><tr><td><em>Cdkn1a</em> </td><td>2.00 </td><td>3.29 </td><td>Cyclin-dependent kinase inhibitor 1A (P21) </td></tr><tr><td><em>Pdgfa</em> </td><td>2.41 </td><td>3.02 </td><td>Platelet derived growth factor, alpha </td></tr><tr><td><em>Ppm1b</em> </td><td>2.15 </td><td>1.10 </td><td>Protein phosphatase 1B, magnesium dependent, beta isoform </td></tr><tr><td><em>Bin1</em> </td><td>2.53 </td><td>1.41 </td><td>Bridging integrator 1 </td></tr><tr><td><em>Cdkl2</em> </td><td>2.16 </td><td>1.68 </td><td>Cyclin-dependent kinase-like 2 (CDC2-related kinase) </td></tr><tr><td><em>Arhgef5</em> </td><td>2.26 </td><td>0.90 </td><td>Rho guanine nucleotide exchange factor (GEF) 5 </td></tr><tr><td><em>Ppp1r15b</em> </td><td>2.03 </td><td>0.92 </td><td>Protein phosphatase 1, regulatory (inhibitor) subunit 15b </td></tr><tr><td><em>Klf3</em> </td><td>2.44 </td><td>1.23 </td><td>Krüppel-like factor 3 (basic) </td></tr><tr><td><em>Pak2</em> </td><td>2.11 </td><td>0.94 </td><td>P21 (CDKN1A)-activated kinase 2 </td></tr><tr><td><em>Pdpk1</em> </td><td>3.30 </td><td>1.04 </td><td>3-phosphoinositide dependent protein kinase-1 </td></tr><tr><td><em>E2F5</em> </td><td>2.13 </td><td>0.83 </td><td>E2F transcription factor 5 </td></tr><tr><td><em>Pak1</em> </td><td>2.09 </td><td>1.47 </td><td>P21 (CDKN1A)-activated kinase 1 </td></tr><tr><td><em>Gchfr</em> </td><td>2.84 </td><td>1.12 </td><td>GTP cyclohydrolase I feedback regulator </td></tr><tr><td><em>Tk1</em> </td><td>0.41 </td><td>0.46 </td><td>Thymidine kinase 1 </td></tr><tr><td><em>Ak1</em> </td><td>0.40 </td><td>0.48 </td><td>Adenylate kinase 1 </td></tr><tr><td><em>Dusp1</em> </td><td>0.38 </td><td>1.52 </td><td>Dual specificity phosphatase 1 </td></tr><tr><td><em>Stk17b</em> </td><td>0.38 </td><td>0.60 </td><td>Serine/threonine kinase 17b (apoptosis-inducing) </td></tr><tr><td><em>Pnrc2</em> </td><td>0.29 </td><td>0.89 </td><td>Proline-rich nuclear receptor co-activator 2 </td></tr></tbody></table></div><div content-id="DSN015TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene symbol </th><th>Fold changes of nascent RNA </th><th>Fold changes of steady-state RNA </th><th>Description </th></tr></thead><tbody><tr><td><em>Cdkn1a</em> </td><td>2.00 </td><td>3.29 </td><td>Cyclin-dependent kinase inhibitor 1A (P21) </td></tr><tr><td><em>Pdgfa</em> </td><td>2.41 </td><td>3.02 </td><td>Platelet derived growth factor, alpha </td></tr><tr><td><em>Ppm1b</em> </td><td>2.15 </td><td>1.10 </td><td>Protein phosphatase 1B, magnesium dependent, beta isoform </td></tr><tr><td><em>Bin1</em> </td><td>2.53 </td><td>1.41 </td><td>Bridging integrator 1 </td></tr><tr><td><em>Cdkl2</em> </td><td>2.16 </td><td>1.68 </td><td>Cyclin-dependent kinase-like 2 (CDC2-related kinase) </td></tr><tr><td><em>Arhgef5</em> </td><td>2.26 </td><td>0.90 </td><td>Rho guanine nucleotide exchange factor (GEF) 5 </td></tr><tr><td><em>Ppp1r15b</em> </td><td>2.03 </td><td>0.92 </td><td>Protein phosphatase 1, regulatory (inhibitor) subunit 15b </td></tr><tr><td><em>Klf3</em> </td><td>2.44 </td><td>1.23 </td><td>Krüppel-like factor 3 (basic) </td></tr><tr><td><em>Pak2</em> </td><td>2.11 </td><td>0.94 </td><td>P21 (CDKN1A)-activated kinase 2 </td></tr><tr><td><em>Pdpk1</em> </td><td>3.30 </td><td>1.04 </td><td>3-phosphoinositide dependent protein kinase-1 </td></tr><tr><td><em>E2F5</em> </td><td>2.13 </td><td>0.83 </td><td>E2F transcription factor 5 </td></tr><tr><td><em>Pak1</em> </td><td>2.09 </td><td>1.47 </td><td>P21 (CDKN1A)-activated kinase 1 </td></tr><tr><td><em>Gchfr</em> </td><td>2.84 </td><td>1.12 </td><td>GTP cyclohydrolase I feedback regulator </td></tr><tr><td><em>Tk1</em> </td><td>0.41 </td><td>0.46 </td><td>Thymidine kinase 1 </td></tr><tr><td><em>Ak1</em> </td><td>0.40 </td><td>0.48 </td><td>Adenylate kinase 1 </td></tr><tr><td><em>Dusp1</em> </td><td>0.38 </td><td>1.52 </td><td>Dual specificity phosphatase 1 </td></tr><tr><td><em>Stk17b</em> </td><td>0.38 </td><td>0.60 </td><td>Serine/threonine kinase 17b (apoptosis-inducing) </td></tr><tr><td><em>Pnrc2</em> </td><td>0.29 </td><td>0.89 </td><td>Proline-rich nuclear receptor co-activator 2 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5490026" target="_blank">View Large</a></div></div><p>We found interesting expression profiles in <em>Cdkn1a. Cdkn1a</em> (<em>p21</em>) was up-regulated in both analyses (Table <a reveal-id="DSN015TB1" data-open="DSN015TB1" class="link link-reveal link-table xref-fig">1</a>). Although this gene was previously reported to function as a signal transducer involved in the transition from the G<sub>1</sub> phase to the S phase or from the G<sub>2</sub> phase to the M phase,<sup><a reveal-id="DSN015C17" data-open="DSN015C17" class="link link-ref link-reveal xref-bibr">17</a></sup> the changes in the expression profiles of the gene at the G<sub>2</sub> phase were not analyzed. As up-regulation of the steady-state RNA and nascent RNA of this gene is clearly observed, it follows that this gene may have an important function in the transition from the G<sub>2</sub> phase to the M phase.</p><p>In addition to <em>Cdkn1a</em>, we have found that <em>Pak1</em> and <em>Pak2</em> also exhibited interesting expression profiles. The changes in expression levels of these genes were detected in the analysis of only the nascent RNAs. The PAK family is involved in microtubule and actin cytoskeleton reorganization.<sup><a reveal-id="DSN015C18" data-open="DSN015C18" class="link link-ref link-reveal xref-bibr">18</a></sup> PAK1 and PAK2 interact with centrosomal kinase Aurora-A, and activated PAK1 phosphorylates Aurora-A in mitosis.<sup><a reveal-id="DSN015C19" data-open="DSN015C19" class="link link-ref link-reveal xref-bibr">19</a></sup> On the other hand, the injection of 58 kDa active Pak1 into <em>Xenopus</em> oocytes inhibits cell cleavage by arresting G<sub>2</sub>/M progression,<sup><a reveal-id="DSN015C20" data-open="DSN015C20" class="link link-ref link-reveal xref-bibr">20</a></sup> suggesting that these products have different role at these phases. These results of DNA microarray analysis indicate that mRNAs of these genes were not only synthesized but also degraded between the G<sub>2</sub> phase and the M phase. As it is likely that synthesis and degradation occur in the same period, it seems difficult to identify the changes in expression levels of these gene mRNAs just by analyzing the steady-state levels.</p><p>We also found that <em>Rplp2</em>, <em>Rpl27</em>, <em>Rps25</em>, <em>Rpl9</em>, and <em>Rps29</em>, which are categorized as ribosomal proteins, were down-regulated at the G<sub>2</sub> phase (Table <a reveal-id="DSN015TB2" data-open="DSN015TB2" class="link link-reveal link-table xref-fig">2</a>). Interestingly, the changes in the expression of these genes were not detected in steady-state RNA, although they were detected in the nascent RNA analysis (Table <a reveal-id="DSN015TB2" data-open="DSN015TB2" class="link link-reveal link-table xref-fig">2</a>). Changes in gene expression were not detected in other ribosomal proteins in both analyses. Therefore, it is possible that these five ribosomal proteins have distinct features in their gene expression regulatory mechanisms. A previous study predicted binding sites in transcriptional regulatory regions or promoters of mammalian ribosomal protein genes for transcriptional factors.<sup><a reveal-id="DSN015C21" data-open="DSN015C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Those authors observed that genes for these five ribosomal proteins lack TATA-box and A/T-rich sequences similar to the TATA box, whereas two-thirds of the genes for ribosomal proteins have the TATA box or its homolog in their promoter regions. Another study found that a unique sequence (a tandem repeat of the TCTCGCGAGA motif) does exist among many human genes that lack the TATA box.<sup><a reveal-id="DSN015C22" data-open="DSN015C22" class="link link-ref link-reveal xref-bibr">22</a></sup> This sequence motif was found at especially high frequencies in the genes for cell cycle regulators as well as in human ribosomal proteins. This sequence motif was also found in the upstream regions of three genes (<em>Rpl9</em>, <em>Rps29</em>, and <em>Rps25</em>) out of the five genes that exhibit changes in gene expression in only the nascent RNA analysis. As translational control is also regarded as an important mechanism for cell cycle regulation, it is possible to obtain new insight into cell cycle regulation in the G<sub>2</sub> phase by performing functional analyses of these ribosomal proteins.</p>                        <a id="5490030" scrollto-destination="5490030"></a>
<div content-id="DSN015TB2" class="table-modal"><div class="table-wrap"><div id="DSN015TB2" class="table-wrap-title" data-id="DSN015TB2"><div class="label -label">Table 2</div><div class="caption"><p>The lists of ribosomal proteins whose level of nascent RNAs changed more than twofold in G2 arrested cells</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene symbol </th><th>Fold change of Nascent RNA </th><th>Fold change of steady-state RNA </th><th>Description </th><th> </th><th>TATA box<hr /> </th></tr></thead><tbody><tr><td><em>Rplp2</em> </td><td>0.36 </td><td>0.63 </td><td>Ribosomal protein, large P2 </td><td>− </td><td> </td></tr><tr><td><em>Rpl9</em> </td><td>0.49 </td><td>0.98 </td><td>Ribosomal protein L9 </td><td>− </td><td> </td></tr><tr><td><em>Rps29</em> </td><td>0.48 </td><td>1.02 </td><td>Ribosomal protein S29 </td><td>− </td><td> </td></tr><tr><td><em>Rps25</em> </td><td>0.41 </td><td>0.99 </td><td>Ribosomal protein S25 </td><td>− </td><td> </td></tr><tr><td><em>Rpl27</em> </td><td>0.49 </td><td>1.00 </td><td>Ribosomal protein L27 </td><td>− </td><td> </td></tr><tr><td>Average of fold change of all ribosomal proteins </td><td>0.94 </td><td>0.95 </td><td> </td><td>− </td><td>35% (28 genes) </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>+ </td><td>65% (51 genes)* </td></tr></tbody></table></div><div content-id="DSN015TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene symbol </th><th>Fold change of Nascent RNA </th><th>Fold change of steady-state RNA </th><th>Description </th><th> </th><th>TATA box<hr /> </th></tr></thead><tbody><tr><td><em>Rplp2</em> </td><td>0.36 </td><td>0.63 </td><td>Ribosomal protein, large P2 </td><td>− </td><td> </td></tr><tr><td><em>Rpl9</em> </td><td>0.49 </td><td>0.98 </td><td>Ribosomal protein L9 </td><td>− </td><td> </td></tr><tr><td><em>Rps29</em> </td><td>0.48 </td><td>1.02 </td><td>Ribosomal protein S29 </td><td>− </td><td> </td></tr><tr><td><em>Rps25</em> </td><td>0.41 </td><td>0.99 </td><td>Ribosomal protein S25 </td><td>− </td><td> </td></tr><tr><td><em>Rpl27</em> </td><td>0.49 </td><td>1.00 </td><td>Ribosomal protein L27 </td><td>− </td><td> </td></tr><tr><td>Average of fold change of all ribosomal proteins </td><td>0.94 </td><td>0.95 </td><td> </td><td>− </td><td>35% (28 genes) </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>+ </td><td>65% (51 genes)* </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Presence or absence of TATA box and A/T rich similar sequence was based on the Perry et al.<sup><a reveal-id="DSN015C21" data-open="DSN015C21" class="link link-ref link-reveal xref-bibr">21</a></sup></p></div></div><div content-id=""><div class="fn"><p>*Ribosomal protein genes which have TATA box or A/T rich tract in those upstream region.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5490030" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSN015TB2" class="table-wrap-title" data-id="DSN015TB2"><div class="label -label">Table 2</div><div class="caption"><p>The lists of ribosomal proteins whose level of nascent RNAs changed more than twofold in G2 arrested cells</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene symbol </th><th>Fold change of Nascent RNA </th><th>Fold change of steady-state RNA </th><th>Description </th><th> </th><th>TATA box<hr /> </th></tr></thead><tbody><tr><td><em>Rplp2</em> </td><td>0.36 </td><td>0.63 </td><td>Ribosomal protein, large P2 </td><td>− </td><td> </td></tr><tr><td><em>Rpl9</em> </td><td>0.49 </td><td>0.98 </td><td>Ribosomal protein L9 </td><td>− </td><td> </td></tr><tr><td><em>Rps29</em> </td><td>0.48 </td><td>1.02 </td><td>Ribosomal protein S29 </td><td>− </td><td> </td></tr><tr><td><em>Rps25</em> </td><td>0.41 </td><td>0.99 </td><td>Ribosomal protein S25 </td><td>− </td><td> </td></tr><tr><td><em>Rpl27</em> </td><td>0.49 </td><td>1.00 </td><td>Ribosomal protein L27 </td><td>− </td><td> </td></tr><tr><td>Average of fold change of all ribosomal proteins </td><td>0.94 </td><td>0.95 </td><td> </td><td>− </td><td>35% (28 genes) </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>+ </td><td>65% (51 genes)* </td></tr></tbody></table></div><div content-id="DSN015TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene symbol </th><th>Fold change of Nascent RNA </th><th>Fold change of steady-state RNA </th><th>Description </th><th> </th><th>TATA box<hr /> </th></tr></thead><tbody><tr><td><em>Rplp2</em> </td><td>0.36 </td><td>0.63 </td><td>Ribosomal protein, large P2 </td><td>− </td><td> </td></tr><tr><td><em>Rpl9</em> </td><td>0.49 </td><td>0.98 </td><td>Ribosomal protein L9 </td><td>− </td><td> </td></tr><tr><td><em>Rps29</em> </td><td>0.48 </td><td>1.02 </td><td>Ribosomal protein S29 </td><td>− </td><td> </td></tr><tr><td><em>Rps25</em> </td><td>0.41 </td><td>0.99 </td><td>Ribosomal protein S25 </td><td>− </td><td> </td></tr><tr><td><em>Rpl27</em> </td><td>0.49 </td><td>1.00 </td><td>Ribosomal protein L27 </td><td>− </td><td> </td></tr><tr><td>Average of fold change of all ribosomal proteins </td><td>0.94 </td><td>0.95 </td><td> </td><td>− </td><td>35% (28 genes) </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>+ </td><td>65% (51 genes)* </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Presence or absence of TATA box and A/T rich similar sequence was based on the Perry et al.<sup><a reveal-id="DSN015C21" data-open="DSN015C21" class="link link-ref link-reveal xref-bibr">21</a></sup></p></div></div><div content-id=""><div class="fn"><p>*Ribosomal protein genes which have TATA box or A/T rich tract in those upstream region.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5490030" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=5490031 id="5490031" class="section-title" data-legacy-id=s3g>3.7. Applications of comprehensive nascent RNA expression analyses</h3>
<p>These examples indicate that our DNA microarray analysis of nascent RNAs has advantages over DNA microarray analysis of steady-state mRNAs. One advantage is that it can provide numerous insights into the regulatory mechanism underlying gene expression. For example, as the changes in expression levels of many transcription factors are relatively small in the process of cellular differentiation, our new method will give us novel insights into the structure of the gene expression regulatory network, since this method can detect slight expression changes. Collecting time-course data on nascent RNAs will also provide important information.</p><p>Recently, Kenzelmann et al.<sup><a reveal-id="DSN015C23" data-open="DSN015C23" class="link link-ref link-reveal xref-bibr">23</a></sup> reported that nascent RNAs could be analyzed by DNA microarray. In their report, the nascent RNAs were mRNAs labeled using 4-thiol uridine and then purified using affinity beads whose surfaces were coated with a toxic organomercury compound.<sup><a reveal-id="DSN015C23" data-open="DSN015C23" class="link link-ref link-reveal xref-bibr">23</a></sup> In our method, precipitation was performed using Dynabeads, which are commonly used in classical immunoprecipitation and do not contain a heavy metal. This is another advantage of our method. Although the DNA microarray used in this study is an oligo-DNA microarray, this method is applicable to other platforms such as cDNA microarray or Affymetrix's GeneChip.</p><p>In summary, our study provides very useful techniques for isolating nascent RNAs with high specificity. By comparing the data between steady-state RNAs and nascent RNAs, we can analyze changes in gene expression with much higher sensitivity. We can also determine the kinetics of transcription by combining the DNA microarray study of nascent mRNAs with that of existing mRNAs. The application of such an analysis will provide important information in the field of transcriptome analysis. Microarray data of transcripts whose nascent RNAs labels changed more than twofold in G<sub>2</sub> arrested cells are showed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data</a>.</p>                            <h2 scrollto-destination=5490035 id="5490035" class="section-title" data-legacy-id=s4>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available online at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5490037 id="5490037" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work has been supported by the grants supplied by the Ministry of Education, Culture, Sports, Science and Technology (MEXT), the Japan Science and Technology Agency (JST), and Bio Matrix Research Inc.</p>                            <h2 scrollto-destination=5490039 id="5490039" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSN015C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN015C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gunji</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Kai</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sameshima</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Global analysis of the expression patterns of transcriptional regulatory factors in formation of embryoid bodies using sensitive oligonucleotide microarray systems</div>, <div class="source">Biochem. Biophys. Res. Commun.</div> , <div class="year">2004</div>, vol. <div class="volume">325</div> (pg. <div class="fpage">265</div>-<div class="lpage">275</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20analysis%20of%20the%20expression%20patterns%20of%20transcriptional%20regulatory%20factors%20in%20formation%20of%20embryoid%20bodies%20using%20sensitive%20oligonucleotide%20microarray%20systems&amp;author=W.%20Gunji&amp;author=T.%20Kai&amp;author=E.%20Sameshima&amp;publication_year=2004&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=325&amp;pages=265-275" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbrc.2004.10.025" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbrc.2004.10.025" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbrc.2004.10.025"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15522228" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN015C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN015C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brown</div> <div class="given-names">A. M.</div></div>,  <div class="name"><div class="surname">Jeltsch</div> <div class="given-names">J. M.</div></div>,  <div class="name"><div class="surname">Roberts</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Chambon</div> <div class="given-names">P</div></div>. </div><div class="article-title">Activation of <em>pS2</em> gene transcription is a primary response to estrogen in the human breast cancer cell line MCF-7</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1984</div>, vol. <div class="volume">81</div> (pg. <div class="fpage">6344</div>-<div class="lpage">6348</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Activation%20of%20pS2%20gene%20transcription%20is%20a%20primary%20response%20to%20estrogen%20in%20the%20human%20breast%20cancer%20cell%20line%20MCF-7&amp;author=A.%20M.%20Brown&amp;author=J.%20M.%20Jeltsch&amp;author=M.%20Roberts&amp;author=P%20Chambon&amp;publication_year=1984&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=81&amp;pages=6344-6348" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.81.20.6344" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.81.20.6344" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.81.20.6344"> </span></div> </div></div></div></div></div><div content-id="DSN015C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN015C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fan</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Wood</div> <div class="given-names">W. H.</div>III</div>,  <div class="name"><div class="surname">Becker</div> <div class="given-names">K. G.</div></div>,  <div class="name"><div class="surname">Gorospe</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Global analysis of stress-regulated mRNA turnover by using cDNA arrays</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2002</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">10611</div>-<div class="lpage">10616</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20analysis%20of%20stress-regulated%20mRNA%20turnover%20by%20using%20cDNA%20arrays&amp;author=J.%20Fan&amp;author=X.%20Yang&amp;author=W.%20Wang&amp;author=W.%20H.%20Wood&amp;author=K.%20G.%20Becker&amp;author=M.%20Gorospe&amp;publication_year=2002&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=99&amp;pages=10611-10616" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.162212399" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.162212399" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.162212399"> </span></div> </div></div></div></div></div><div content-id="DSN015C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN015C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Garcia-Martinez</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Aranda</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Perez-Ortin</div> <div class="given-names">J. E.</div></div>. </div><div class="article-title">Genomic run-on evaluates transcription rates for all yeast genes and identifies gene regulatory mechanisms</div>, <div class="source">Mol. Cell</div> , <div class="year">2004</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">303</div>-<div class="lpage">313</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20run-on%20evaluates%20transcription%20rates%20for%20all%20yeast%20genes%20and%20identifies%20gene%20regulatory%20mechanisms&amp;author=J.%20Garcia-Martinez&amp;author=A.%20Aranda&amp;author=J.%20E.%20Perez-Ortin&amp;publication_year=2004&amp;journal=Mol.%20Cell&amp;volume=15&amp;pages=303-313" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2004.06.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2004.06.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2004.06.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15260981" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN015C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN015C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cheadle</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Fan</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Cho-Chung</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Control of gene expression during T cell activation: alternate regulation of mRNA transcription and mRNA stability</div>, <div class="source">BMC Genom.</div> , <div class="year">2005</div>, vol. <div class="volume">6</div> pg. <div class="fpage">75</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Control%20of%20gene%20expression%20during%20T%20cell%20activation%3A%20alternate%20regulation%20of%20mRNA%20transcription%20and%20mRNA%20stability&amp;author=C.%20Cheadle&amp;author=J.%20Fan&amp;author=Y.%20Cho-Chung&amp;publication_year=2005&amp;journal=BMC%20Genom.&amp;volume=6&amp;pages=75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-6-75" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-6-75" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-6-75"> </span></div> </div></div></div></div></div><div content-id="DSN015C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN015C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cleary</div> <div class="given-names">M. D.</div></div>,  <div class="name"><div class="surname">Meiering</div> <div class="given-names">C. D.</div></div>,  <div class="name"><div class="surname">Jan</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Guymon</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Boothroyd</div> <div class="given-names">J. C.</div></div>. </div><div class="article-title">Biosynthetic labeling of RNA with uracil phosphoribosyltransferase allows cell-specific microarray analysis of mRNA synthesis and decay</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2005</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">232</div>-<div class="lpage">237</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biosynthetic%20labeling%20of%20RNA%20with%20uracil%20phosphoribosyltransferase%20allows%20cell-specific%20microarray%20analysis%20of%20mRNA%20synthesis%20and%20decay&amp;author=M.%20D.%20Cleary&amp;author=C.%20D.%20Meiering&amp;author=E.%20Jan&amp;author=R.%20Guymon&amp;author=J.%20C.%20Boothroyd&amp;publication_year=2005&amp;journal=Nat.%20Biotechnol.&amp;volume=23&amp;pages=232-237" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1061" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1061" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1061"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15685165" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN015C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN015C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wansink</div> <div class="given-names">D. G.</div></div>,  <div class="name"><div class="surname">Schul</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">van der Kraan</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">van Steensel</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">van Driel</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">de Jong</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Fluorescent labeling of nascent RNA reveals transcription by RNA polymerase II in domains scattered throughout the nucleus</div>, <div class="source">J. Cell Biol.</div> , <div class="year">1993</div>, vol. <div class="volume">122</div> (pg. <div class="fpage">283</div>-<div class="lpage">293</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fluorescent%20labeling%20of%20nascent%20RNA%20reveals%20transcription%20by%20RNA%20polymerase%20II%20in%20domains%20scattered%20throughout%20the%20nucleus&amp;author=D.%20G.%20Wansink&amp;author=W.%20Schul&amp;author=I.%20van%20der%20Kraan&amp;author=B.%20van%20Steensel&amp;author=R.%20van%20Driel&amp;author=L.%20de%20Jong&amp;publication_year=1993&amp;journal=J.%20Cell%20Biol.&amp;volume=122&amp;pages=283-293" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1083/jcb.122.2.283" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1083%2Fjcb.122.2.283" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1083%2Fjcb.122.2.283"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8320255" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN015C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN015C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jackson</div> <div class="given-names">D. A.</div></div>,  <div class="name"><div class="surname">Iborra</div> <div class="given-names">F. J.</div></div>,  <div class="name"><div class="surname">Manders</div> <div class="given-names">E. M.</div></div>,  <div class="name"><div class="surname">Cook</div> <div class="given-names">P. R.</div></div>. </div><div class="article-title">Numbers and organization of RNA polymerases, nascent transcripts, and transcription units in HeLa nuclei</div>, <div class="source">Mol. Biol. Cell</div> , <div class="year">1998</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">1523</div>-<div class="lpage">1536</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Numbers%20and%20organization%20of%20RNA%20polymerases%2C%20nascent%20transcripts%2C%20and%20transcription%20units%20in%20HeLa%20nuclei&amp;author=D.%20A.%20Jackson&amp;author=F.%20J.%20Iborra&amp;author=E.%20M.%20Manders&amp;author=P.%20R.%20Cook&amp;publication_year=1998&amp;journal=Mol.%20Biol.%20Cell&amp;volume=9&amp;pages=1523-1536" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1091/mbc.9.6.1523" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1091%2Fmbc.9.6.1523" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1091%2Fmbc.9.6.1523"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9614191" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN015C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN015C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Waksmundzka</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Debey</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Electric field-mediated BrUTP uptake by mouse oocytes, eggs, and embryos</div>, <div class="source">Mol. Reprod. Dev.</div> , <div class="year">2001</div>, vol. <div class="volume">58</div> (pg. <div class="fpage">173</div>-<div class="lpage">179</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Electric%20field-mediated%20BrUTP%20uptake%20by%20mouse%20oocytes%2C%20eggs%2C%20and%20embryos&amp;author=M.%20Waksmundzka&amp;author=P.%20Debey&amp;publication_year=2001&amp;journal=Mol.%20Reprod.%20Dev.&amp;volume=58&amp;pages=173-179" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1098-2795" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1098-2795" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1098-2795"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11139229" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN015C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN015C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hoshino</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tagawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Okuda</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Okazawa</div> <div class="given-names">H.</div></div>. </div><div class="article-title">General transcriptional repression by polyglutamine disease proteins is not directly linked to the presence of inclusion bodies</div>, <div class="source">Biochem. Biophys. Res. Commun.</div> , <div class="year">2004</div>, vol. <div class="volume">313</div> (pg. <div class="fpage">110</div>-<div class="lpage">116</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=General%20transcriptional%20repression%20by%20polyglutamine%20disease%20proteins%20is%20not%20directly%20linked%20to%20the%20presence%20of%20inclusion%20bodies&amp;author=M.%20Hoshino&amp;author=K.%20Tagawa&amp;author=T.%20Okuda&amp;author=H.%20Okazawa&amp;publication_year=2004&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=313&amp;pages=110-116" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbrc.2003.11.086" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbrc.2003.11.086" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbrc.2003.11.086"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14672705" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN015C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN015C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grande</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">van der Kraan</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">de Jong</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">van Driel</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Nuclear distribution of transcription factors in relation to sites of transcription and RNA polymerase II</div>, <div class="source">J. Cell Sci.</div> , <div class="year">1997</div>, vol. <div class="volume">110</div> (pg. <div class="fpage">1781</div>-<div class="lpage">1791</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nuclear%20distribution%20of%20transcription%20factors%20in%20relation%20to%20sites%20of%20transcription%20and%20RNA%20polymerase%20II&amp;author=M.%20A.%20Grande&amp;author=I.%20van%20der%20Kraan&amp;author=L.%20de%20Jong&amp;author=R.%20van%20Driel&amp;publication_year=1997&amp;journal=J.%20Cell%20Sci.&amp;volume=110&amp;pages=1781-1791" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9264465" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN015C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN015C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Haider</div> <div class="given-names">S. R.</div></div>,  <div class="name"><div class="surname">Juan</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Traganos</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Darzynkiewicz</div> <div class="given-names">Z.</div></div>. </div><div class="article-title">Immunoseparation and immunodetection of nucleic acids labeled with halogenated nucleotides</div>, <div class="source">Exp. Cell Res.</div> , <div class="year">1997</div>, vol. <div class="volume">234</div> (pg. <div class="fpage">498</div>-<div class="lpage">506</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Immunoseparation%20and%20immunodetection%20of%20nucleic%20acids%20labeled%20with%20halogenated%20nucleotides&amp;author=S.%20R.%20Haider&amp;author=G.%20Juan&amp;author=F.%20Traganos&amp;author=Z.%20Darzynkiewicz&amp;publication_year=1997&amp;journal=Exp.%20Cell%20Res.&amp;volume=234&amp;pages=498-506" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/excr.1997.3644" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fexcr.1997.3644" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fexcr.1997.3644"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9260920" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN015C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN015C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kageyama</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nagata</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Aoki</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Isolation of nascent messenger RNA from mouse preimplantation embryos</div>, <div class="source">Biol. Reprod.</div> , <div class="year">2004</div>, vol. <div class="volume">71</div> (pg. <div class="fpage">1948</div>-<div class="lpage">1955</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20of%20nascent%20messenger%20RNA%20from%20mouse%20preimplantation%20embryos&amp;author=S.%20Kageyama&amp;author=M.%20Nagata&amp;author=F.%20Aoki&amp;publication_year=2004&amp;journal=Biol.%20Reprod.&amp;volume=71&amp;pages=1948-1955" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1095/biolreprod.104.031906" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1095%2Fbiolreprod.104.031906" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1095%2Fbiolreprod.104.031906"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15286032" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN015C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN015C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mineo</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Murakami</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ishimi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Hanaoka</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Yamada</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Isolation and analysis of a mammalian temperature-sensitive mutant defective in G2 functions</div>, <div class="source">Exp. Cell Res.</div> , <div class="year">1986</div>, vol. <div class="volume">167</div> (pg. <div class="fpage">53</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20analysis%20of%20a%20mammalian%20temperature-sensitive%20mutant%20defective%20in%20G2%20functions&amp;author=C.%20Mineo&amp;author=Y.%20Murakami&amp;author=Y.%20Ishimi&amp;author=F.%20Hanaoka&amp;author=M.%20Yamada&amp;publication_year=1986&amp;journal=Exp.%20Cell%20Res.&amp;volume=167&amp;pages=53-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0014-4827(86)90203-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0014-4827(86)90203-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0014-4827(86)90203-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2428648" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN015C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN015C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Th'ng</div> <div class="given-names">J. P.</div></div>,  <div class="name"><div class="surname">Wright</div> <div class="given-names">P. S.</div></div>,  <div class="name"><div class="surname">Hamaguchi</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">The FT210 cell line is a mouse G2 phase mutant with a temperature-sensitive <em>CDC2</em> gene product</div>, <div class="source">Cell</div> , <div class="year">1990</div>, vol. <div class="volume">63</div> (pg. <div class="fpage">313</div>-<div class="lpage">324</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20FT210%20cell%20line%20is%20a%20mouse%20G2%20phase%20mutant%20with%20a%20temperature-sensitive%20CDC2%20gene%20product&amp;author=J.%20P.%20Th'ng&amp;author=P.%20S.%20Wright&amp;author=J.%20Hamaguchi&amp;publication_year=1990&amp;journal=Cell&amp;volume=63&amp;pages=313-324" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0092-8674(90)90164-A" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0092-8674(90)90164-A" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0092-8674(90)90164-A"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2208288" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN015C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN015C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kageyama</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Gunji</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Nakasato</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Murakami</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Nagata</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Aoki</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Analysis of transcription factor expression during oogenesis and preimplantation development in mice</div>, <div class="source">Zygote</div> , <div class="year">2007</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">117</div>-<div class="lpage">128</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20transcription%20factor%20expression%20during%20oogenesis%20and%20preimplantation%20development%20in%20mice&amp;author=S.%20Kageyama&amp;author=W.%20Gunji&amp;author=M.%20Nakasato&amp;author=Y.%20Murakami&amp;author=M.%20Nagata&amp;author=F.%20Aoki&amp;publication_year=2007&amp;journal=Zygote&amp;volume=15&amp;pages=117-128" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1017/S096719940700411X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1017%2FS096719940700411X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1017%2FS096719940700411X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17462104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN015C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN015C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Niculescu</div> <div class="given-names">A. B.</div>III</div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Smeets</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hengst</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Prives</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Reed</div> <div class="given-names">S. I.</div></div>. </div><div class="article-title">Effects of p21<sup>Cip1/Waf1</sup> at both the G<sub>1</sub>/S and the G<sub>2</sub>/M cell cycle transitions: pRb is a critical determinant in blocking DNA replication and in preventing endoreduplication</div>, <div class="source">Mol. Cell. Biol.</div> , <div class="year">1998</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">629</div>-<div class="lpage">643</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effects%20of%20p21Cip1%2FWaf1%20at%20both%20the%20G1%2FS%20and%20the%20G2%2FM%20cell%20cycle%20transitions%3A%20pRb%20is%20a%20critical%20determinant%20in%20blocking%20DNA%20replication%20and%20in%20preventing%20endoreduplication&amp;author=A.%20B.%20Niculescu&amp;author=X.%20Chen&amp;author=M.%20Smeets&amp;author=L.%20Hengst&amp;author=C.%20Prives&amp;author=S.%20I.%20Reed&amp;publication_year=1998&amp;journal=Mol.%20Cell.%20Biol.&amp;volume=18&amp;pages=629-643" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MCB.18.1.629" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMCB.18.1.629" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMCB.18.1.629"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9418909" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN015C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN015C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wittmann</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Bokoch</div> <div class="given-names">G. M.</div></div>,  <div class="name"><div class="surname">Waterman-Storer</div> <div class="given-names">C. M.</div></div>. </div><div class="article-title">Regulation of leading edge microtubule and actin dynamics downstream of Rac1</div>, <div class="source">J. Cell Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">161</div> (pg. <div class="fpage">845</div>-<div class="lpage">851</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20leading%20edge%20microtubule%20and%20actin%20dynamics%20downstream%20of%20Rac1&amp;author=T.%20Wittmann&amp;author=G.%20M.%20Bokoch&amp;author=C.%20M.%20Waterman-Storer&amp;publication_year=2003&amp;journal=J.%20Cell%20Biol.&amp;volume=161&amp;pages=845-851" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1083/jcb.200303082" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1083%2Fjcb.200303082" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1083%2Fjcb.200303082"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12796474" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN015C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN015C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhao</div> <div class="given-names">Z. S.</div></div>,  <div class="name"><div class="surname">Lim</div> <div class="given-names">J. P.</div></div>,  <div class="name"><div class="surname">Ng</div> <div class="given-names">Y. W.</div></div>,  <div class="name"><div class="surname">Lim</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Manser</div> <div class="given-names">E.</div></div>. </div><div class="article-title">The GIT-associated kinase PAK targets to the centrosome and regulates Aurora-A</div>, <div class="source">Mol. Cell</div> , <div class="year">2005</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">237</div>-<div class="lpage">249</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20GIT-associated%20kinase%20PAK%20targets%20to%20the%20centrosome%20and%20regulates%20Aurora-A&amp;author=Z.%20S.%20Zhao&amp;author=J.%20P.%20Lim&amp;author=Y.%20W.%20Ng&amp;author=L.%20Lim&amp;author=E.%20Manser&amp;publication_year=2005&amp;journal=Mol.%20Cell&amp;volume=20&amp;pages=237-249" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2005.08.035" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2005.08.035" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2005.08.035"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16246726" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN015C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN015C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rooney</div> <div class="given-names">R. D.</div></div>,  <div class="name"><div class="surname">Tuazon</div> <div class="given-names">P. T.</div></div>,  <div class="name"><div class="surname">Meek</div> <div class="given-names">W. E.</div></div>, et al. </div><div class="article-title">Cleavage arrest of early frog embryos by the G protein-activated protein kinase PAK I</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1996</div>, vol. <div class="volume">271</div> (pg. <div class="fpage">21498</div>-<div class="lpage">21504</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cleavage%20arrest%20of%20early%20frog%20embryos%20by%20the%20G%20protein-activated%20protein%20kinase%20PAK%20I&amp;author=R.%20D.%20Rooney&amp;author=P.%20T.%20Tuazon&amp;author=W.%20E.%20Meek&amp;publication_year=1996&amp;journal=J.%20Biol.%20Chem.&amp;volume=271&amp;pages=21498-21504" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.271.35.21498" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.271.35.21498" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.271.35.21498"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8702934" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN015C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN015C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Perry</div> <div class="given-names">R. P.</div></div>. </div><div class="article-title">The architecture of mammalian ribosomal protein promoters</div>, <div class="source">BMC Evol. Biol.</div> , <div class="year">2005</div>, vol. <div class="volume">5</div> pg. <div class="fpage">15</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20architecture%20of%20mammalian%20ribosomal%20protein%20promoters&amp;author=R.%20P.%20Perry&amp;publication_year=2005&amp;journal=BMC%20Evol.%20Biol.&amp;volume=5&amp;pages=15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-5-15" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-5-15" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-5-15"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15707503" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN015C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN015C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wyrwicz</div> <div class="given-names">L. S.</div></div>,  <div class="name"><div class="surname">Gaj</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Hoffmann</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Rychlewski</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Ostrowski</div> <div class="given-names">J.</div></div>. </div><div class="article-title">A common <em>cis</em>-element in promoters of protein synthesis and cell cycle genes</div>, <div class="source">Acta Biochim. Pol.</div> , <div class="year">2007</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">89</div>-<div class="lpage">98</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20common%20cis-element%20in%20promoters%20of%20protein%20synthesis%20and%20cell%20cycle%20genes&amp;author=L.%20S.%20Wyrwicz&amp;author=P.%20Gaj&amp;author=M.%20Hoffmann&amp;author=L.%20Rychlewski&amp;author=J.%20Ostrowski&amp;publication_year=2007&amp;journal=Acta%20Biochim.%20Pol.&amp;volume=54&amp;pages=89-98" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17351670" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN015C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN015C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kenzelmann</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Maertens</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Hergenhahn</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Microarray analysis of newly synthesized RNA in cells and animals</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2007</div>, vol. <div class="volume">104</div> (pg. <div class="fpage">6164</div>-<div class="lpage">6169</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microarray%20analysis%20of%20newly%20synthesized%20RNA%20in%20cells%20and%20animals&amp;author=M.%20Kenzelmann&amp;author=S.%20Maertens&amp;author=M.%20Hergenhahn&amp;publication_year=2007&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=104&amp;pages=6164-6169" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0610439104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0610439104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0610439104"> </span></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2008. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=376722&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn015/2/dsn015-dsn015_Table.xls?Expires=2147483647&amp;Signature=TSmiOsnpkX5D9Dfgn4J6MxQHxDfizmuTubIWcvzad2GAtvX7q2eHj5ZRTErCSe5LqzkHvGne0CPEAqPWRvl6kckjuK4itXExmfz3QQwnET0h6oXScDdM1JXfALdxnEapsicRm0nXvD1MVT0zcUy8rrEy4y9-yPqg0z7cpg6-IoVhETxHYGyCoF9-G5CGizXywp34AN42U6cQSrDljzLAANkSi-2XEOFYlokC6gJCDOWel87M5x~8OwS29qivA6YHTg3ZrE5lDIWBHDKHXffogiM7Deg66wWVAX3k~zAwCZCythbS-ZffLnCvDSTaC89nvx0fWvcbooskW7xzFWS7jg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn015/2/dsn015-dsn015supp.pdf?Expires=2147483647&amp;Signature=mQHmSK-mFofEZytWVwyXgfNwlKcSYrwjSmYpgY~LCteY8u10ei8h1cg3Os8yKzTlZLIgobA-0Vr1UVi3RX15GBLSWGIvkO3thaxVL8jWFDWsd35RaziGiLwD7nl3I2Mgma7kkyx-v9EB~TDvefhQkLUs36xKY6y9f~j1GTFw~bbP7FdS0A~m6tv8vAHrt4GqisaTncLeCU7Id~zfQ9cV7as4~CbkqvXViFAosNAph21psQK9-qRWMadBb8xlbfZiChyas8q9dHMgHojVDLsIWOUaQA2h3N1BW4BvX9QLoFFWcZofANdO14K~MEY4GpbwfT4wWVLkGBEMbFqQ9CIy-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">177</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000260564600006&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">11</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsn015"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="376722"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000260564600006&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f15%2f4%2f241%2f376722">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000260564600006&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (11)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F15%2F4%2F241%2F376722" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/376722'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Novel DNA Microarray System for Analysis of Nascent mRNAs" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsn015");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f15%2f4%2f241%2f376722">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

