

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/17/5/271/353434","@type":"ScholarlyArticle","name":"Efficient Plant Gene Identification Based on Interspecies Mapping of Full-Length cDNAs","datePublished":"2010-07-28","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/17/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2010-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsq017","keywords":["interspecies mapping","full-length cDNA","CDS identification","plant genome"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/17/5/271/353434","author":[{"name":"Amano, Naoki","affiliation":"Bioinformatics Research Unit, Division of Genome and Biodiversity Research ,  National Institute of Agrobiological Sciences ,  2-1-2 Kannondai, Tsukuba, Ibaraki 305-8602 ,  Japan","@type":"Person"},{"name":"Tanaka, Tsuyoshi","affiliation":"Bioinformatics Research Unit, Division of Genome and Biodiversity Research ,  National Institute of Agrobiological Sciences ,  2-1-2 Kannondai, Tsukuba, Ibaraki 305-8602 ,  Japan","@type":"Person"},{"name":"Numa, Hisataka","affiliation":"Bioinformatics Research Unit, Division of Genome and Biodiversity Research ,  National Institute of Agrobiological Sciences ,  2-1-2 Kannondai, Tsukuba, Ibaraki 305-8602 ,  Japan","@type":"Person"},{"name":"Sakai, Hiroaki","affiliation":"Bioinformatics Research Unit, Division of Genome and Biodiversity Research ,  National Institute of Agrobiological Sciences ,  2-1-2 Kannondai, Tsukuba, Ibaraki 305-8602 ,  Japan","@type":"Person"},{"name":"Itoh, Takeshi","affiliation":"Bioinformatics Research Unit, Division of Genome and Biodiversity Research ,  National Institute of Agrobiological Sciences ,  2-1-2 Kannondai, Tsukuba, Ibaraki 305-8602 ,  Japan","@type":"Person"}],"description":"Abstract.  We present an annotation pipeline that accurately predicts exon–intron structures and protein-coding sequences (CDSs) on the basis of full-length cDN","pageStart":"271","pageEnd":"279","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/5/1/m_cover.gif?Expires=2147483647&Signature=XCQlUD9LK7XFUkpUfRn~YGtjsVRpmrT9eb4syTPTn6rTzboWNlQlH-siiO3TXM5GrRqmN7FUTnXpB3myvmk93syOV9bWEsz2vGx6yN~HeBizlfceSxcyWv64eGdML9C9SGPBdE~eItnEEBaSgpFCn0R~HTuQmXRsioxnSdebpn-~JqHg7eHzflvfN4sXgzFZGJPEuIGC5YLTuLUzh7LpgtxWSMJLCp-arnHh1smvnR4mJrGafdSyV48rumRp74lot6SiJXr5I0jIo5wItyB0fT8rkkkvVM3SAdqxuBdwVyTCOfmwlFn0~mbY3SumXub6CMCFckG66ON0FXK5WQIs7Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Efficient Plant Gene Identification Based on Interspecies Mapping of Full-Length cDNAs","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/5/1/m_cover.gif?Expires=2147483647&Signature=XCQlUD9LK7XFUkpUfRn~YGtjsVRpmrT9eb4syTPTn6rTzboWNlQlH-siiO3TXM5GrRqmN7FUTnXpB3myvmk93syOV9bWEsz2vGx6yN~HeBizlfceSxcyWv64eGdML9C9SGPBdE~eItnEEBaSgpFCn0R~HTuQmXRsioxnSdebpn-~JqHg7eHzflvfN4sXgzFZGJPEuIGC5YLTuLUzh7LpgtxWSMJLCp-arnHh1smvnR4mJrGafdSyV48rumRp74lot6SiJXr5I0jIo5wItyB0fT8rkkkvVM3SAdqxuBdwVyTCOfmwlFn0~mbY3SumXub6CMCFckG66ON0FXK5WQIs7Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Efficient Plant Gene Identification Based on Interspecies Mapping of Full-Length cDNAs" />
    <meta property="og:description" content="Abstract.  We present an annotation pipeline that accurately predicts exon–intron structures and protein-coding sequences (CDSs) on the basis of full-length cDN" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/17/5/271/353434" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2010-07-28" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Amano, Naoki" /><meta name="citation_author_institution" content="Bioinformatics Research Unit, Division of Genome and Biodiversity Research, National Institute of Agrobiological Sciences, 2-1-2 Kannondai, Tsukuba, Ibaraki 305-8602, Japan" /><meta name="citation_author" content="Tanaka, Tsuyoshi" /><meta name="citation_author_institution" content="Bioinformatics Research Unit, Division of Genome and Biodiversity Research, National Institute of Agrobiological Sciences, 2-1-2 Kannondai, Tsukuba, Ibaraki 305-8602, Japan" /><meta name="citation_author" content="Numa, Hisataka" /><meta name="citation_author_institution" content="Bioinformatics Research Unit, Division of Genome and Biodiversity Research, National Institute of Agrobiological Sciences, 2-1-2 Kannondai, Tsukuba, Ibaraki 305-8602, Japan" /><meta name="citation_author" content="Sakai, Hiroaki" /><meta name="citation_author_institution" content="Bioinformatics Research Unit, Division of Genome and Biodiversity Research, National Institute of Agrobiological Sciences, 2-1-2 Kannondai, Tsukuba, Ibaraki 305-8602, Japan" /><meta name="citation_author" content="Itoh, Takeshi" /><meta name="citation_author_institution" content="Bioinformatics Research Unit, Division of Genome and Biodiversity Research, National Institute of Agrobiological Sciences, 2-1-2 Kannondai, Tsukuba, Ibaraki 305-8602, Japan" /><meta name="citation_title" content="Efficient Plant Gene Identification Based on Interspecies Mapping of Full-Length cDNAs" /><meta name="citation_firstpage" content="271" /><meta name="citation_lastpage" content="279" /><meta name="citation_doi" content="10.1093/dnares/dsq017" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="17" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2010/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Analysis of the genome sequence of the flowering plant Arabidopsis thaliana; citation_author=Arabidopsis Genome Initiative.; Nature;  citation_year=2000;  citation_volume=408;  citation_pages=796-815; " /><meta name="citation_reference" content="citation_title=The map-based sequence of the rice genome; citation_author=International Rice Genome Sequencing Project.; Nature;  citation_year=2005;  citation_volume=436;  citation_pages=793-800; " /><meta name="citation_reference" content="citation_title=The Sorghum bicolor genome and the diversification of grasses; Nature;  citation_year=2009;  citation_volume=457;  citation_pages=551-6; " /><meta name="citation_reference" content="citation_title=The B73 maize genome: complexity, diversity, and dynamics; Science;  citation_year=2009;  citation_volume=326;  citation_pages=1112-5; " /><meta name="citation_reference" content="citation_title=Genome sequencing and analysis of the model grass Brachypodium distachyon; citation_author=The International Brachypodium Initiative.; Nature;  citation_year=2010;  citation_volume=463;  citation_pages=763-8; " /><meta name="citation_reference" content="citation_title=The genome of black cottonwood, Populus trichocarpa (Torr. &amp; Gray); Science;  citation_year=2006;  citation_volume=313;  citation_pages=1596-604; " /><meta name="citation_reference" content="citation_title=The grapevine genome sequence suggests ancestral hexaploidization in major angiosperm phyla; Nature;  citation_year=2007;  citation_volume=449;  citation_pages=463-7; " /><meta name="citation_reference" content="citation_title=The draft genome of the transgenic tropical fruit tree papaya (Carica papaya Linnaeus); Nature;  citation_year=2008;  citation_volume=452;  citation_pages=991-6; " /><meta name="citation_reference" content="citation_title=Genome structure of the legume, Lotus japonicus; DNA Res.;  citation_year=2008;  citation_volume=15;  citation_pages=227-39; " /><meta name="citation_reference" content="citation_title=Genome sequence of the palaeopolyploid soybean; Nature;  citation_volume=463;  citation_pages=178-83; " /><meta name="citation_reference" content="citation_title=The international barley sequencing consortium—at the threshold of efficient access to the barley genome; Plant Physiol.;  citation_year=2009;  citation_volume=149;  citation_pages=142-7; " /><meta name="citation_reference" content="citation_title=A physical map of the 1-gigabase bread wheat chromosome 3B; Science;  citation_year=2008;  citation_volume=322;  citation_pages=101-4; " /><meta name="citation_reference" content="citation_title=Assessing the feasibility of GS FLX pyrosequencing for sequencing the Atlantic salmon genome; BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=404" /><meta name="citation_reference" content="citation_title=454 sequencing put to the test using the complex genome of barley; BMC Genomics;  citation_year=2006;  citation_volume=7;  citation_pages=275" /><meta name="citation_reference" content="citation_title=Consistent over-estimation of gene number in complex plant genomes; Curr. Opin. Plant Biol.;  citation_year=2004;  citation_volume=7;  citation_pages=732-6; " /><meta name="citation_reference" content="citation_title=Incorrectly predicted genes in rice?; Gene;  citation_year=2004;  citation_volume=333;  citation_pages=187-8; " /><meta name="citation_reference" content="citation_title=The new genes of rice: a closer look; Trends Plant Sci.;  citation_year=2004;  citation_volume=9;  citation_pages=281-5; " /><meta name="citation_reference" content="citation_title=Features of Arabidopsis genes and genome discovered using full-length cDNAs; Plant Mol. Biol.;  citation_year=2006;  citation_volume=60;  citation_pages=69-85; " /><meta name="citation_reference" content="citation_title=Curated genome annotation of Oryza sativa ssp. japonica and comparative genome analysis with Arabidopsis thaliana; Genome Res.;  citation_year=2007;  citation_volume=17;  citation_pages=175-83; " /><meta name="citation_reference" content="citation_title=Functional annotation of a full-length Arabidopsis cDNA collection; Science;  citation_year=2002;  citation_volume=296;  citation_pages=141-5; " /><meta name="citation_reference" content="citation_title=RIKEN Arabidopsis full-length (RAFL) cDNA and its applications for expression profiling under abiotic stress conditions; J. Exp. Bot.;  citation_year=2004;  citation_volume=55;  citation_pages=213-23; " /><meta name="citation_reference" content="citation_title=Evidence-based gene predictions in plant genomes; Genome Res.;  citation_year=2009;  citation_volume=19;  citation_pages=1912-23; " /><meta name="citation_reference" content="citation_title=Sim4cc: a cross-species spliced alignment program; Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=e80" /><meta name="citation_reference" content="citation_title=Gene structure prediction from consensus spliced alignment of multiple ESTs matching the same genomic locus; Bioinformatics;  citation_year=2004;  citation_volume=20;  citation_pages=1157-69; " /><meta name="citation_reference" content="citation_title=Collection, mapping, and annotation of over 28,000 cDNA clones from japonica rice; Science;  citation_year=2003;  citation_volume=301;  citation_pages=376-9; " /><meta name="citation_reference" content="citation_title=A collection of 10,096 indica rice full-length cDNAs reveals highly expressed sequence divergence between Oryza sativa indica and japonica subspecies; Plant Mol. Biol.;  citation_year=2007;  citation_volume=65;  citation_pages=403-15; " /><meta name="citation_reference" content="citation_title=Collection and comparative analysis of 1888 full-length cDNAs from wild rice Oryza rufipogon Griff. W1943; DNA Res.;  citation_year=2008;  citation_volume=15;  citation_pages=285-95; " /><meta name="citation_reference" content="citation_title=Development of 5006 full-length CDNAs in barley: a tool for accessing cereal genomics resources; DNA Res.;  citation_year=2009;  citation_volume=16;  citation_pages=81-9; " /><meta name="citation_reference" content="citation_title=Insights into corn genes derived from large-scale cDNA sequencing; Plant Mol. Biol.;  citation_year=2009;  citation_volume=69;  citation_pages=179-94; " /><meta name="citation_reference" content="citation_title=Annotation and expression profile analysis of 2073 full-length cDNAs from stress-induced maize (Zea mays L.) seedlings; Plant J.;  citation_year=2006;  citation_volume=48;  citation_pages=710-27; " /><meta name="citation_reference" content="citation_title=Sequencing, mapping, and analysis of 27,455 maize full-length cDNAs; PLoS Genet.;  citation_year=2009;  citation_volume=5;  citation_pages=e1000740" /><meta name="citation_reference" content="citation_title=Sequencing and analysis of approximately 40,000 soybean cDNA clones from a full-length-enriched cDNA library; DNA Res.;  citation_year=2008;  citation_volume=15;  citation_pages=333-46; " /><meta name="citation_reference" content="citation_title=Analysis of 4,664 high-quality sequence-finished poplar full-length cDNA clones and their utility for the discovery of genes responding to insect feeding; BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=57" /><meta name="citation_reference" content="citation_title=TriFLDB: a database of clustered full-length coding sequences from Triticeae with applications to comparative grass genomics; Plant Physiol.;  citation_year=2009;  citation_volume=150;  citation_pages=1135-46; " /><meta name="citation_reference" content="citation_title=Large-scale analysis of full-length cDNAs from the tomato (Solanum lycopersicum) cultivar Micro-Tom, a reference system for the Solanaceae genomics; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=210" /><meta name="citation_reference" content="citation_title=The Arabidopsis Information Resource (TAIR): gene structure and function annotation; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=D1009-14; " /><meta name="citation_reference" content="citation_title=The Rice Annotation Project Database (RAP-DB): 2008 update; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=D1028-33; " /><meta name="citation_reference" content="citation_title=The TIGR rice genome annotation resource: improvements and new features; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=D883-7; " /><meta name="citation_reference" content="citation_title=MIPSPlantsDB–plant database resource for integrative and comparative plant genome research; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=D834-40; " /><meta name="citation_reference" content="citation_title=EST_GENOME: a program to align spliced DNA sequences to unspliced genomic DNA; Comput. Appl. Biosci.;  citation_year=1997;  citation_volume=13;  citation_pages=477-8; " /><meta name="citation_reference" content="citation_title=Basic local alignment search tool; J. Mol. Biol.;  citation_year=1990;  citation_volume=215;  citation_pages=403-10; " /><meta name="citation_reference" content="citation_title=The Universal Protein Resource (UniProt) 2009; citation_author=Uniprot Consortium.; Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=D169-74; " /><meta name="citation_reference" content="citation_title=NCBI reference sequences: current status, policy and new initiatives; Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=D32-6; " /><meta name="citation_reference" content="citation_title=Recognition of genes in DNA sequence with ambiguities; Biosystems;  citation_year=1993;  citation_volume=30;  citation_pages=161-71; " /><meta name="citation_reference" content="citation_title=Efficient decoding algorithms for generalized hidden Markov model gene finders; BMC Bioinformatics;  citation_year=2005;  citation_volume=6;  citation_pages=16" /><meta name="citation_reference" content="citation_title=TigrScan and GlimmerHMM: two open source ab initio eukaryotic gene-finders; Bioinformatics;  citation_year=2004;  citation_volume=20;  citation_pages=2878-9; " /><meta name="citation_reference" content="citation_title=Gene identification in novel eukaryotic genomes by self-training algorithm; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=6494-506; " /><meta name="citation_reference" content="citation_title=Optimal spliced alignment of homologous cDNA to a genomic DNA template; Bioinformatics;  citation_year=2000;  citation_volume=16;  citation_pages=203-11; " /><meta name="citation_reference" content="citation_title=Genomic mapping by fingerprinting random clones: a mathematical analysis; Genomics;  citation_year=1988;  citation_volume=2;  citation_pages=231-9; " /><meta name="citation_reference" content="citation_title=Identification and characterization of shared duplications between rice and wheat provide new insight into grass genome evolution; Plant Cell;  citation_year=2008;  citation_volume=20;  citation_pages=11-24; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/17/5/271/1010644/dsq017.pdf" /><meta name="description" content="Abstract.  We present an annotation pipeline that accurately predicts exon–intron structures and protein-coding sequences (CDSs) on the basis of full-length cDN" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/17/5/271/353434" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/17/5/271/353434" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/17/5/271/353434");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsq017';
    </script>
<title>
	Efficient Plant Gene Identification Based on Interspecies Mapping of Full-Length cDNAs | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./353434" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BfBqPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBbWFubywgTmFva2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCaW9pbmZvcm1hdGljcyBSZXNlYXJjaCBVbml0LCBEaXZpc2lvbiBvZiBHZW5vbWUgYW5kIEJpb2RpdmVyc2l0eSBSZXNlYXJjaCwgTmF0aW9uYWwgSW5zdGl0dXRlIG9mIEFncm9iaW9sb2dpY2FsIFNjaWVuY2VzLCAyLTEtMiBLYW5ub25kYWksIFRzdWt1YmEsIEliYXJha2kgMzA1LTg2MDIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFuYWthLCBUc3V5b3NoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJpb2luZm9ybWF0aWNzIFJlc2VhcmNoIFVuaXQsIERpdmlzaW9uIG9mIEdlbm9tZSBhbmQgQmlvZGl2ZXJzaXR5IFJlc2VhcmNoLCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgQWdyb2Jpb2xvZ2ljYWwgU2NpZW5jZXMsIDItMS0yIEthbm5vbmRhaSwgVHN1a3ViYSwgSWJhcmFraSAzMDUtODYwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOdW1hLCBIaXNhdGFrYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJpb2luZm9ybWF0aWNzIFJlc2VhcmNoIFVuaXQsIERpdmlzaW9uIG9mIEdlbm9tZSBhbmQgQmlvZGl2ZXJzaXR5IFJlc2VhcmNoLCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgQWdyb2Jpb2xvZ2ljYWwgU2NpZW5jZXMsIDItMS0yIEthbm5vbmRhaSwgVHN1a3ViYSwgSWJhcmFraSAzMDUtODYwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYWthaSwgSGlyb2FraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJpb2luZm9ybWF0aWNzIFJlc2VhcmNoIFVuaXQsIERpdmlzaW9uIG9mIEdlbm9tZSBhbmQgQmlvZGl2ZXJzaXR5IFJlc2VhcmNoLCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgQWdyb2Jpb2xvZ2ljYWwgU2NpZW5jZXMsIDItMS0yIEthbm5vbmRhaSwgVHN1a3ViYSwgSWJhcmFraSAzMDUtODYwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJdG9oLCBUYWtlc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvaW5mb3JtYXRpY3MgUmVzZWFyY2ggVW5pdCwgRGl2aXNpb24gb2YgR2Vub21lIGFuZCBCaW9kaXZlcnNpdHkgUmVzZWFyY2gsIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBBZ3JvYmlvbG9naWNhbCBTY2llbmNlcywgMi0xLTIgS2Fubm9uZGFpLCBUc3VrdWJhLCBJYmFyYWtpIDMwNS04NjAyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iRWZmaWNpZW50IFBsYW50IEdlbmUgSWRlbnRpZmljYXRpb24gQmFzZWQgb24gSW50ZXJzcGVjaWVzIE1hcHBpbmcgb2YgRnVsbC1MZW5ndGggY0ROQXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIyNzEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjI3OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzcTAxNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDEwLzEwLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgdGhlIGdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgZmxvd2VyaW5nIHBsYW50IEFyYWJpZG9wc2lzIHRoYWxpYW5hOyBjaXRhdGlvbl9hdXRob3I9QXJhYmlkb3BzaXMgR2Vub21lIEluaXRpYXRpdmUuOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9NDA4OyAgY2l0YXRpb25fcGFnZXM9Nzk2LTgxNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG1hcC1iYXNlZCBzZXF1ZW5jZSBvZiB0aGUgcmljZSBnZW5vbWU7IGNpdGF0aW9uX2F1dGhvcj1JbnRlcm5hdGlvbmFsIFJpY2UgR2Vub21lIFNlcXVlbmNpbmcgUHJvamVjdC47IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00MzY7ICBjaXRhdGlvbl9wYWdlcz03OTMtODAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgU29yZ2h1bSBiaWNvbG9yIGdlbm9tZSBhbmQgdGhlIGRpdmVyc2lmaWNhdGlvbiBvZiBncmFzc2VzOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDU3OyAgY2l0YXRpb25fcGFnZXM9NTUxLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBCNzMgbWFpemUgZ2Vub21lOiBjb21wbGV4aXR5LCBkaXZlcnNpdHksIGFuZCBkeW5hbWljczsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zMjY7ICBjaXRhdGlvbl9wYWdlcz0xMTEyLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jaW5nIGFuZCBhbmFseXNpcyBvZiB0aGUgbW9kZWwgZ3Jhc3MgQnJhY2h5cG9kaXVtIGRpc3RhY2h5b247IGNpdGF0aW9uX2F1dGhvcj1UaGUgSW50ZXJuYXRpb25hbCBCcmFjaHlwb2RpdW0gSW5pdGlhdGl2ZS47IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00NjM7ICBjaXRhdGlvbl9wYWdlcz03NjMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBvZiBibGFjayBjb3R0b253b29kLCBQb3B1bHVzIHRyaWNob2NhcnBhIChUb3JyLiAmYW1wOyBHcmF5KTsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zMTM7ICBjaXRhdGlvbl9wYWdlcz0xNTk2LTYwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdyYXBldmluZSBnZW5vbWUgc2VxdWVuY2Ugc3VnZ2VzdHMgYW5jZXN0cmFsIGhleGFwbG9pZGl6YXRpb24gaW4gbWFqb3IgYW5naW9zcGVybSBwaHlsYTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTQ0OTsgIGNpdGF0aW9uX3BhZ2VzPTQ2My03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZHJhZnQgZ2Vub21lIG9mIHRoZSB0cmFuc2dlbmljIHRyb3BpY2FsIGZydWl0IHRyZWUgcGFwYXlhIChDYXJpY2EgcGFwYXlhIExpbm5hZXVzKTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ1MjsgIGNpdGF0aW9uX3BhZ2VzPTk5MS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgc3RydWN0dXJlIG9mIHRoZSBsZWd1bWUsIExvdHVzIGphcG9uaWN1czsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0yMjctMzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgcGFsYWVvcG9seXBsb2lkIHNveWJlYW47IE5hdHVyZTsgIGNpdGF0aW9uX3ZvbHVtZT00NjM7ICBjaXRhdGlvbl9wYWdlcz0xNzgtODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBpbnRlcm5hdGlvbmFsIGJhcmxleSBzZXF1ZW5jaW5nIGNvbnNvcnRpdW3igJRhdCB0aGUgdGhyZXNob2xkIG9mIGVmZmljaWVudCBhY2Nlc3MgdG8gdGhlIGJhcmxleSBnZW5vbWU7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE0OTsgIGNpdGF0aW9uX3BhZ2VzPTE0Mi03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHBoeXNpY2FsIG1hcCBvZiB0aGUgMS1naWdhYmFzZSBicmVhZCB3aGVhdCBjaHJvbW9zb21lIDNCOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTMyMjsgIGNpdGF0aW9uX3BhZ2VzPTEwMS00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3Nlc3NpbmcgdGhlIGZlYXNpYmlsaXR5IG9mIEdTIEZMWCBweXJvc2VxdWVuY2luZyBmb3Igc2VxdWVuY2luZyB0aGUgQXRsYW50aWMgc2FsbW9uIGdlbm9tZTsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz00MDQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT00NTQgc2VxdWVuY2luZyBwdXQgdG8gdGhlIHRlc3QgdXNpbmcgdGhlIGNvbXBsZXggZ2Vub21lIG9mIGJhcmxleTsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0yNzUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zaXN0ZW50IG92ZXItZXN0aW1hdGlvbiBvZiBnZW5lIG51bWJlciBpbiBjb21wbGV4IHBsYW50IGdlbm9tZXM7IEN1cnIuIE9waW4uIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz03MzItNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5jb3JyZWN0bHkgcHJlZGljdGVkIGdlbmVzIGluIHJpY2U/OyBHZW5lOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMzMzsgIGNpdGF0aW9uX3BhZ2VzPTE4Ny04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbmV3IGdlbmVzIG9mIHJpY2U6IGEgY2xvc2VyIGxvb2s7IFRyZW5kcyBQbGFudCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0yODEtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmVhdHVyZXMgb2YgQXJhYmlkb3BzaXMgZ2VuZXMgYW5kIGdlbm9tZSBkaXNjb3ZlcmVkIHVzaW5nIGZ1bGwtbGVuZ3RoIGNETkFzOyBQbGFudCBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTYwOyAgY2l0YXRpb25fcGFnZXM9NjktODU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUN1cmF0ZWQgZ2Vub21lIGFubm90YXRpb24gb2YgT3J5emEgc2F0aXZhIHNzcC4gamFwb25pY2EgYW5kIGNvbXBhcmF0aXZlIGdlbm9tZSBhbmFseXNpcyB3aXRoIEFyYWJpZG9wc2lzIHRoYWxpYW5hOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTE3NS04MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVuY3Rpb25hbCBhbm5vdGF0aW9uIG9mIGEgZnVsbC1sZW5ndGggQXJhYmlkb3BzaXMgY0ROQSBjb2xsZWN0aW9uOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTI5NjsgIGNpdGF0aW9uX3BhZ2VzPTE0MS01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SSUtFTiBBcmFiaWRvcHNpcyBmdWxsLWxlbmd0aCAoUkFGTCkgY0ROQSBhbmQgaXRzIGFwcGxpY2F0aW9ucyBmb3IgZXhwcmVzc2lvbiBwcm9maWxpbmcgdW5kZXIgYWJpb3RpYyBzdHJlc3MgY29uZGl0aW9uczsgSi4gRXhwLiBCb3QuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTU1OyAgY2l0YXRpb25fcGFnZXM9MjEzLTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdmlkZW5jZS1iYXNlZCBnZW5lIHByZWRpY3Rpb25zIGluIHBsYW50IGdlbm9tZXM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MTkxMi0yMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2ltNGNjOiBhIGNyb3NzLXNwZWNpZXMgc3BsaWNlZCBhbGlnbm1lbnQgcHJvZ3JhbTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9ZTgwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBzdHJ1Y3R1cmUgcHJlZGljdGlvbiBmcm9tIGNvbnNlbnN1cyBzcGxpY2VkIGFsaWdubWVudCBvZiBtdWx0aXBsZSBFU1RzIG1hdGNoaW5nIHRoZSBzYW1lIGdlbm9taWMgbG9jdXM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MTE1Ny02OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29sbGVjdGlvbiwgbWFwcGluZywgYW5kIGFubm90YXRpb24gb2Ygb3ZlciAyOCwwMDAgY0ROQSBjbG9uZXMgZnJvbSBqYXBvbmljYSByaWNlOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMwMTsgIGNpdGF0aW9uX3BhZ2VzPTM3Ni05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNvbGxlY3Rpb24gb2YgMTAsMDk2IGluZGljYSByaWNlIGZ1bGwtbGVuZ3RoIGNETkFzIHJldmVhbHMgaGlnaGx5IGV4cHJlc3NlZCBzZXF1ZW5jZSBkaXZlcmdlbmNlIGJldHdlZW4gT3J5emEgc2F0aXZhIGluZGljYSBhbmQgamFwb25pY2Egc3Vic3BlY2llczsgUGxhbnQgTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT02NTsgIGNpdGF0aW9uX3BhZ2VzPTQwMy0xNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29sbGVjdGlvbiBhbmQgY29tcGFyYXRpdmUgYW5hbHlzaXMgb2YgMTg4OCBmdWxsLWxlbmd0aCBjRE5BcyBmcm9tIHdpbGQgcmljZSBPcnl6YSBydWZpcG9nb24gR3JpZmYuIFcxOTQzOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTI4NS05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgb2YgNTAwNiBmdWxsLWxlbmd0aCBDRE5BcyBpbiBiYXJsZXk6IGEgdG9vbCBmb3IgYWNjZXNzaW5nIGNlcmVhbCBnZW5vbWljcyByZXNvdXJjZXM7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9ODEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5zaWdodHMgaW50byBjb3JuIGdlbmVzIGRlcml2ZWQgZnJvbSBsYXJnZS1zY2FsZSBjRE5BIHNlcXVlbmNpbmc7IFBsYW50IE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Njk7ICBjaXRhdGlvbl9wYWdlcz0xNzktOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFubm90YXRpb24gYW5kIGV4cHJlc3Npb24gcHJvZmlsZSBhbmFseXNpcyBvZiAyMDczIGZ1bGwtbGVuZ3RoIGNETkFzIGZyb20gc3RyZXNzLWluZHVjZWQgbWFpemUgKFplYSBtYXlzIEwuKSBzZWVkbGluZ3M7IFBsYW50IEouOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTQ4OyAgY2l0YXRpb25fcGFnZXM9NzEwLTI3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jaW5nLCBtYXBwaW5nLCBhbmQgYW5hbHlzaXMgb2YgMjcsNDU1IG1haXplIGZ1bGwtbGVuZ3RoIGNETkFzOyBQTG9TIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDA3NDAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jaW5nIGFuZCBhbmFseXNpcyBvZiBhcHByb3hpbWF0ZWx5IDQwLDAwMCBzb3liZWFuIGNETkEgY2xvbmVzIGZyb20gYSBmdWxsLWxlbmd0aC1lbnJpY2hlZCBjRE5BIGxpYnJhcnk7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MzMzLTQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiA0LDY2NCBoaWdoLXF1YWxpdHkgc2VxdWVuY2UtZmluaXNoZWQgcG9wbGFyIGZ1bGwtbGVuZ3RoIGNETkEgY2xvbmVzIGFuZCB0aGVpciB1dGlsaXR5IGZvciB0aGUgZGlzY292ZXJ5IG9mIGdlbmVzIHJlc3BvbmRpbmcgdG8gaW5zZWN0IGZlZWRpbmc7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9NTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmlGTERCOiBhIGRhdGFiYXNlIG9mIGNsdXN0ZXJlZCBmdWxsLWxlbmd0aCBjb2Rpbmcgc2VxdWVuY2VzIGZyb20gVHJpdGljZWFlIHdpdGggYXBwbGljYXRpb25zIHRvIGNvbXBhcmF0aXZlIGdyYXNzIGdlbm9taWNzOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTA7ICBjaXRhdGlvbl9wYWdlcz0xMTM1LTQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MYXJnZS1zY2FsZSBhbmFseXNpcyBvZiBmdWxsLWxlbmd0aCBjRE5BcyBmcm9tIHRoZSB0b21hdG8gKFNvbGFudW0gbHljb3BlcnNpY3VtKSBjdWx0aXZhciBNaWNyby1Ub20sIGEgcmVmZXJlbmNlIHN5c3RlbSBmb3IgdGhlIFNvbGFuYWNlYWUgZ2Vub21pY3M7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTIxMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBBcmFiaWRvcHNpcyBJbmZvcm1hdGlvbiBSZXNvdXJjZSAoVEFJUik6IGdlbmUgc3RydWN0dXJlIGFuZCBmdW5jdGlvbiBhbm5vdGF0aW9uOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz1EMTAwOS0xNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFJpY2UgQW5ub3RhdGlvbiBQcm9qZWN0IERhdGFiYXNlIChSQVAtREIpOiAyMDA4IHVwZGF0ZTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9RDEwMjgtMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBUSUdSIHJpY2UgZ2Vub21lIGFubm90YXRpb24gcmVzb3VyY2U6IGltcHJvdmVtZW50cyBhbmQgbmV3IGZlYXR1cmVzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz1EODgzLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1JUFNQbGFudHNEQuKAk3BsYW50IGRhdGFiYXNlIHJlc291cmNlIGZvciBpbnRlZ3JhdGl2ZSBhbmQgY29tcGFyYXRpdmUgcGxhbnQgZ2Vub21lIHJlc2VhcmNoOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz1EODM0LTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FU1RfR0VOT01FOiBhIHByb2dyYW0gdG8gYWxpZ24gc3BsaWNlZCBETkEgc2VxdWVuY2VzIHRvIHVuc3BsaWNlZCBnZW5vbWljIEROQTsgQ29tcHV0LiBBcHBsLiBCaW9zY2kuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9NDc3LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhc2ljIGxvY2FsIGFsaWdubWVudCBzZWFyY2ggdG9vbDsgSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT0yMTU7ICBjaXRhdGlvbl9wYWdlcz00MDMtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBVbml2ZXJzYWwgUHJvdGVpbiBSZXNvdXJjZSAoVW5pUHJvdCkgMjAwOTsgY2l0YXRpb25fYXV0aG9yPVVuaXByb3QgQ29uc29ydGl1bS47IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zNzsgIGNpdGF0aW9uX3BhZ2VzPUQxNjktNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5DQkkgcmVmZXJlbmNlIHNlcXVlbmNlczogY3VycmVudCBzdGF0dXMsIHBvbGljeSBhbmQgbmV3IGluaXRpYXRpdmVzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz1EMzItNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVjb2duaXRpb24gb2YgZ2VuZXMgaW4gRE5BIHNlcXVlbmNlIHdpdGggYW1iaWd1aXRpZXM7IEJpb3N5c3RlbXM7ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0xNjEtNzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVmZmljaWVudCBkZWNvZGluZyBhbGdvcml0aG1zIGZvciBnZW5lcmFsaXplZCBoaWRkZW4gTWFya292IG1vZGVsIGdlbmUgZmluZGVyczsgQk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0xNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRpZ3JTY2FuIGFuZCBHbGltbWVySE1NOiB0d28gb3BlbiBzb3VyY2UgYWIgaW5pdGlvIGV1a2FyeW90aWMgZ2VuZS1maW5kZXJzOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTI4NzgtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBpZGVudGlmaWNhdGlvbiBpbiBub3ZlbCBldWthcnlvdGljIGdlbm9tZXMgYnkgc2VsZi10cmFpbmluZyBhbGdvcml0aG07IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTY0OTQtNTA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcHRpbWFsIHNwbGljZWQgYWxpZ25tZW50IG9mIGhvbW9sb2dvdXMgY0ROQSB0byBhIGdlbm9taWMgRE5BIHRlbXBsYXRlOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTIwMy0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBtYXBwaW5nIGJ5IGZpbmdlcnByaW50aW5nIHJhbmRvbSBjbG9uZXM6IGEgbWF0aGVtYXRpY2FsIGFuYWx5c2lzOyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MTk4ODsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9MjMxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIHNoYXJlZCBkdXBsaWNhdGlvbnMgYmV0d2VlbiByaWNlIGFuZCB3aGVhdCBwcm92aWRlIG5ldyBpbnNpZ2h0IGludG8gZ3Jhc3MgZ2Vub21lIGV2b2x1dGlvbjsgUGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTExLTI0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTcvNS8yNzEvMTAxMDY0NC9kc3EwMTcucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFdlIHByZXNlbnQgYW4gYW5ub3RhdGlvbiBwaXBlbGluZSB0aGF0IGFjY3VyYXRlbHkgcHJlZGljdHMgZXhvbuKAk2ludHJvbiBzdHJ1Y3R1cmVzIGFuZCBwcm90ZWluLWNvZGluZyBzZXF1ZW5jZXMgKENEU3MpIG9uIHRoZSBiYXNpcyBvZiBmdWxsLWxlbmd0aCBjRE4iIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM1MzQzNCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGQDDOynAbLI2QstPtJ2oDDCrWarWw==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f5%2f271%2f353434" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f5%2f271%2f353434" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13694" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/17/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/5/1/m_cover.gif?Expires=2147483647&amp;Signature=XCQlUD9LK7XFUkpUfRn~YGtjsVRpmrT9eb4syTPTn6rTzboWNlQlH-siiO3TXM5GrRqmN7FUTnXpB3myvmk93syOV9bWEsz2vGx6yN~HeBizlfceSxcyWv64eGdML9C9SGPBdE~eItnEEBaSgpFCn0R~HTuQmXRsioxnSdebpn-~JqHg7eHzflvfN4sXgzFZGJPEuIGC5YLTuLUzh7LpgtxWSMJLCp-arnHh1smvnR4mJrGafdSyV48rumRp74lot6SiJXr5I0jIo5wItyB0fT8rkkkvVM3SAdqxuBdwVyTCOfmwlFn0~mbY3SumXub6CMCFckG66ON0FXK5WQIs7Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/17/5">
                        <div class="volume trailing-comma">Volume 17</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2010        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5029046">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5029046">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5029049">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5029049">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5029056">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5029056">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5029092">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5029092">3 Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5029134">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5029134">4. Availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5029138">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5029138">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5029141">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5029141">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5029144">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5029144">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/17/5/261/353723">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/17/5/281/353619">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Efficient Plant Gene Identification Based on Interspecies Mapping of Full-Length cDNAs
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Naoki Amano</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Bioinformatics Research Unit, Division of Genome and Biodiversity Research</div>, <div class="insititution">National Institute of Agrobiological Sciences</div>, <div class="addr-line">2-1-2 Kannondai, Tsukuba, Ibaraki 305-8602</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Naoki+Amano">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Amano N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Amano N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Naoki Amano</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tsuyoshi Tanaka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Bioinformatics Research Unit, Division of Genome and Biodiversity Research</div>, <div class="insititution">National Institute of Agrobiological Sciences</div>, <div class="addr-line">2-1-2 Kannondai, Tsukuba, Ibaraki 305-8602</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tsuyoshi+Tanaka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tanaka T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tanaka T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tsuyoshi Tanaka</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hisataka Numa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Bioinformatics Research Unit, Division of Genome and Biodiversity Research</div>, <div class="insititution">National Institute of Agrobiological Sciences</div>, <div class="addr-line">2-1-2 Kannondai, Tsukuba, Ibaraki 305-8602</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hisataka+Numa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Numa H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Numa H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hisataka Numa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroaki Sakai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Bioinformatics Research Unit, Division of Genome and Biodiversity Research</div>, <div class="insititution">National Institute of Agrobiological Sciences</div>, <div class="addr-line">2-1-2 Kannondai, Tsukuba, Ibaraki 305-8602</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroaki+Sakai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sakai H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sakai H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hiroaki Sakai</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takeshi Itoh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Bioinformatics Research Unit, Division of Genome and Biodiversity Research</div>, <div class="insititution">National Institute of Agrobiological Sciences</div>, <div class="addr-line">2-1-2 Kannondai, Tsukuba, Ibaraki 305-8602</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel./Fax. +81 29-838-7065. E-mail: <a href="mailto:taitoh@affrc.go.jp" target="_blank">taitoh@affrc.go.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takeshi+Itoh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Itoh T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Itoh T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takeshi Itoh</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 17, Issue 5, 1 October 2010, Pages 271–279, <a href='https://doi.org/10.1093/dnares/dsq017'>https://doi.org/10.1093/dnares/dsq017</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">28 July 2010</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">19 March 2010</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">23 June 2010</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="353434" href="/dnaresearch/article-pdf/17/5/271/1010644/dsq017.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Naoki Amano, Tsuyoshi Tanaka, Hisataka Numa, Hiroaki Sakai, Takeshi Itoh;  Efficient Plant Gene Identification Based on Interspecies Mapping of Full-Length cDNAs, <em>DNA Research</em>, Volume 17, Issue 5, 1 October 2010, Pages 271–279, <a href='https://doi.org/10.1093/dnares/dsq017'>https://doi.org/10.1093/dnares/dsq017</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/353434?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/353434?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/353434?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/353434?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/353434?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/353434?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/353434?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Efficient%20Plant%20Gene%20Identification%20Based%20on%20Interspecies%20Mapping%20of%20Full-Length%20cDNAs&amp;publicationDate=2010-07-28&amp;volumeNum=17&amp;issueNum=5&amp;author=Amano%2C%20Naoki%3B%20Tanaka%2C%20Tsuyoshi&amp;startPage=271&amp;endPage=279&amp;contentId=10.1093%2Fdnares%2Fdsq017&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f5%2f271%2f353434" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5029046 id="5029046" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>We present an annotation pipeline that accurately predicts exon–intron structures and protein-coding sequences (CDSs) on the basis of full-length cDNAs (FLcDNAs). This annotation pipeline was used to identify genes in 10 plant genomes. In particular, we show that interspecies mapping of FLcDNAs to genomes is of great value in fully utilizing FLcDNA resources whose availability is limited to several species. Because low sequence conservation at 5′- and 3′-ends of FLcDNAs between different species tends to result in truncated CDSs, we developed an improved algorithm to identify complete CDSs by the extension of both ends of truncated CDSs. Interspecies mapping of 71 801 monocot FLcDNAs to the <em>Oryza sativa</em> genome led to the detection of 22 142 protein-coding regions. Moreover, in comparing two mapping programs and three <em>ab initio</em> prediction programs, we found that our pipeline was more capable of identifying complete CDSs. As demonstrated by monocot interspecies mapping, in which nucleotide identity between FLcDNAs and the genome was ∼80%, the resultant inferred CDSs were sufficiently accurate. Finally, we applied both inter- and intraspecies mapping to 10 monocot and dicot genomes and identified genes in 210 551 loci. Interspecies mapping of FLcDNAs is expected to effectively predict genes and CDSs in newly sequenced genomes.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;interspecies mapping&quot;">interspecies mapping</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;full-length cDNA&quot;">full-length cDNA</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;CDS identification&quot;">CDS identification</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;plant genome&quot;">plant genome</a></div>                            <h2 scrollto-destination=5029049 id="5029049" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Following the sequencing of the <em>Arabidopsis thaliana</em> genome in 2000<sup><a reveal-id="DSQ017C1" data-open="DSQ017C1" class="link link-ref link-reveal xref-bibr">1</a></sup> and the <em>Oryza sativa</em> genome in 2005,<sup><a reveal-id="DSQ017C2" data-open="DSQ017C2" class="link link-ref link-reveal xref-bibr">2</a></sup> several complete or nearly complete flowering plant genomes have been released.<sup><a reveal-id="DSQ017C3  DSQ017C10" data-open="DSQ017C3  DSQ017C10" class="link link-ref link-reveal xref-bibr">3–10</a></sup> In addition, large-scale sequencing projects of important cereal crops are currently being undertaken by international consortia, including the International Barley Sequencing Consortium (<a class="link link-uri" href="http://www.barleygenome.org">http://www.barleygenome.org</a>) and the International Wheat Genome Sequencing Consortium (<a class="link link-uri" href="http://www.wheatgenome.org">http://www.wheatgenome.org</a>).<sup><a reveal-id="DSQ017C11" data-open="DSQ017C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSQ017C12" data-open="DSQ017C12" class="link link-ref link-reveal xref-bibr">12</a></sup> It is expected that accelerating genome sequencing technologies, such as next-generation sequencing, will lead to a dramatic increase in the genomic DNA data to be annotated.<sup><a reveal-id="DSQ017C13" data-open="DSQ017C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSQ017C14" data-open="DSQ017C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Therefore, to cope with the deluge of emerging sequence information, the development of an efficient annotation system is needed.</p><p>Exon–intron structures and the protein-coding sequences (CDSs) in genome sequences can be predicted either by <em>ab initio</em> predictions or by sequence similarity methods. While <em>ab initio</em> gene prediction programs may produce erroneous exon–intron structures,<sup><a reveal-id="DSQ017C15  DSQ017C17" data-open="DSQ017C15  DSQ017C17" class="link link-ref link-reveal xref-bibr">15–17</a></sup> sequence similarity approaches generally show better results, although the number of available sequences in the current databases limits the number of genes that can be predicted. Tens of thousands of full-length cDNAs (FLcDNAs), which can be used to accurately determine exon–intron structures,<sup><a reveal-id="DSQ017C4" data-open="DSQ017C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSQ017C18  DSQ017C21" data-open="DSQ017C18  DSQ017C21" class="link link-ref link-reveal xref-bibr">18–21</a></sup> are available for <em>O. sativa</em>, <em>Zea mays</em>, and <em>A. thaliana</em>, and the quality of their genome annotations is generally thought to be high. In other species, however, sufficient numbers of FLcDNAs are not necessarily available for the corresponding genomes. To fully utilize FLcDNA resources, algorithms for interspecies mapping, which show better gene structure annotation than <em>ab initio</em> predictions, have been developed.<sup><a reveal-id="DSQ017C22" data-open="DSQ017C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DSQ017C23" data-open="DSQ017C23" class="link link-ref link-reveal xref-bibr">23</a></sup></p><p>Identification of CDS regions is a crucial step in genome annotation. However, low sequence conservation at the 5′- and 3′-ends of FLcDNAs between different species frequently results in truncated CDSs. To solve this problem, we present a pipeline that is based on interspecies FLcDNA mapping to genomes, which predicts exon–intron structures and the CDSs. In this pipeline, both ends of truncated CDSs are extended so that the complete CDSs are obtained. Our FLcDNA mapping pipeline was validated and compared with two other cDNA mapping programs—sim4cc<sup><a reveal-id="DSQ017C23" data-open="DSQ017C23" class="link link-ref link-reveal xref-bibr">23</a></sup> and GeneSeqer<sup><a reveal-id="DSQ017C24" data-open="DSQ017C24" class="link link-ref link-reveal xref-bibr">24</a></sup>—as well as three <em>ab initio</em> gene prediction methods. In addition, we estimated how many sequences would be needed for exhaustive gene identification by interspecies mapping. Finally, for comprehensive gene identification, we applied our pipeline to the 10 genomes of the following flowering plants: <em>O. sativa</em> cv. Nipponbare, <em>Z. mays</em>, <em>Sorghum bicolor</em>, <em>Brachypodium distachyon</em>, <em>A. thaliana</em>, <em>Lotus japonicus</em>, <em>Populus trichocarpa</em>, <em>Glycine max</em>, <em>Vitis vinifera</em>, and <em>Carica papaya</em>.</p>                            <h2 scrollto-destination=5029056 id="5029056" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5029058 id="5029058" class="section-title" data-legacy-id=s2a>2.1. FLcDNAs, genome sequences, and reference data sets</h3>
<p>We retrieved 179 991 FLcDNAs from <em>A. thaliana</em>,<sup><a reveal-id="DSQ017C18" data-open="DSQ017C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSQ017C20" data-open="DSQ017C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSQ017C21" data-open="DSQ017C21" class="link link-ref link-reveal xref-bibr">21</a></sup><em>O. sativa</em>,<sup><a reveal-id="DSQ017C25" data-open="DSQ017C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DSQ017C26" data-open="DSQ017C26" class="link link-ref link-reveal xref-bibr">26</a></sup><em>O. rufipogon</em>,<sup><a reveal-id="DSQ017C27" data-open="DSQ017C27" class="link link-ref link-reveal xref-bibr">27</a></sup><em>Hordeum vulgare</em>,<sup><a reveal-id="DSQ017C28" data-open="DSQ017C28" class="link link-ref link-reveal xref-bibr">28</a></sup><em>Z. mays</em>,<sup><a reveal-id="DSQ017C29  DSQ017C31" data-open="DSQ017C29  DSQ017C31" class="link link-ref link-reveal xref-bibr">29–31</a></sup><em>G. max</em>,<sup><a reveal-id="DSQ017C32" data-open="DSQ017C32" class="link link-ref link-reveal xref-bibr">32</a></sup><em>P. trichocarpa</em>,<sup><a reveal-id="DSQ017C33" data-open="DSQ017C33" class="link link-ref link-reveal xref-bibr">33</a></sup><em>Triticum aestivum</em>,<sup><a reveal-id="DSQ017C34" data-open="DSQ017C34" class="link link-ref link-reveal xref-bibr">34</a></sup> and <em>Solanum lycopersicum</em><sup><a reveal-id="DSQ017C35" data-open="DSQ017C35" class="link link-ref link-reveal xref-bibr">35</a></sup> from the sources listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. The genome sequences and annotations of <em>O. sativa</em>,<sup><a reveal-id="DSQ017C2" data-open="DSQ017C2" class="link link-ref link-reveal xref-bibr">2</a></sup><em>S. bicolor</em>,<sup><a reveal-id="DSQ017C3" data-open="DSQ017C3" class="link link-ref link-reveal xref-bibr">3</a></sup><em>Z. mays</em>,<sup><a reveal-id="DSQ017C4" data-open="DSQ017C4" class="link link-ref link-reveal xref-bibr">4</a></sup><em>B. distachyon</em>,<sup><a reveal-id="DSQ017C5" data-open="DSQ017C5" class="link link-ref link-reveal xref-bibr">5</a></sup><em>A. thaliana</em>,<sup><a reveal-id="DSQ017C36" data-open="DSQ017C36" class="link link-ref link-reveal xref-bibr">36</a></sup><em>P. trichocarpa</em>,<sup><a reveal-id="DSQ017C6" data-open="DSQ017C6" class="link link-ref link-reveal xref-bibr">6</a></sup><em>V. vinifera</em>,<sup><a reveal-id="DSQ017C7" data-open="DSQ017C7" class="link link-ref link-reveal xref-bibr">7</a></sup><em>C. papaya</em>,<sup><a reveal-id="DSQ017C8" data-open="DSQ017C8" class="link link-ref link-reveal xref-bibr">8</a></sup><em>L. japonicus</em>,<sup><a reveal-id="DSQ017C9" data-open="DSQ017C9" class="link link-ref link-reveal xref-bibr">9</a></sup> and <em>G. max</em><sup><a reveal-id="DSQ017C10" data-open="DSQ017C10" class="link link-ref link-reveal xref-bibr">10</a></sup> were downloaded from the web sites listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>. For validation of predicted exon–intron structures and the CDSs, reference data sets, based on the intraspecies mapping method of the Rice Annotation Project,<sup><a reveal-id="DSQ017C19" data-open="DSQ017C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSQ017C37" data-open="DSQ017C37" class="link link-ref link-reveal xref-bibr">37</a></sup> were used (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). To enhance the accuracy of the validations, we used only the loci that had the same CDS structures as the corresponding CDSs from other sources: the MSU6.1<sup><a reveal-id="DSQ017C38" data-open="DSQ017C38" class="link link-ref link-reveal xref-bibr">38</a></sup> data set for <em>O. sativa</em>, the TAIR 9.0 representative CDSs<sup><a reveal-id="DSQ017C36" data-open="DSQ017C36" class="link link-ref link-reveal xref-bibr">36</a></sup> for <em>A. thaliana</em>, and the B73 RefGen_v1 Filtered Gene Set<sup><a reveal-id="DSQ017C4" data-open="DSQ017C4" class="link link-ref link-reveal xref-bibr">4</a></sup> for <em>Z. mays</em>. If more than one CDS was predicted in a locus of <em>Z. mays</em>, the longest one was selected.</p>                            <h3 scrollto-destination=5029062 id="5029062" class="section-title" data-legacy-id=s2b>2.2. Pipeline of interspecies mapping and CDS identification</h3>
<p>Figure <a reveal-id="DSQ017F1" data-open="DSQ017F1" class="link link-reveal link-table xref-fig">1</a> shows an overview of our new interspecies mapping pipeline that simultaneously identifies CDSs on predicted loci. Repetitive sequences on FLcDNAs were masked by RepeatMasker (<a class="link link-uri" href="http://www.repeatmasker.org">http://www.repeatmasker.org</a>) with the MIPS Repeat Element Database (mips-REdat 4.3)<sup><a reveal-id="DSQ017C39" data-open="DSQ017C39" class="link link-ref link-reveal xref-bibr">39</a></sup> and the ‘complete set’ of the Triticeae Repeat Sequence Database release 10 (<a class="link link-uri" href="http://wheat.pw.usda.gov/ITMI/Repeats/">http://wheat.pw.usda.gov/ITMI/Repeats/</a>). Vector sequences were removed by the cross_match program. PolyA stretches were discarded using a custom-made program. Sequences with total unmasked lengths of 30 bp or more were used. </p>                        <a id="5029066" scrollto-destination="5029066"></a>
<div id="DSQ017F1" data-id="DSQ017F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq017/2/m_dsq01701.jpeg?Expires=2147483647&Signature=BT1Dbul9KQzTw2v5Zx8B-EDN7KLu6YB2uZo-THCFmLtBJs2AjBL5y1OqiPJx23WXR99s-qiqsugJgww-yYWFHTFg0KSyCflry2yDVoxqOpTvFgMp8eti97T6bm1o8sMEjqEr-ULpso5TR-2WfyYAO8-~RdOpShPXWYA2t3v2LbDeN-ItXck5Cyd3zPLOwD4xq9n7hFcUkRxdo13gqFDF86Mj9U5qQklgCtDRyamb1VFBzUHL0F3JofeFkaR0oVa5PfpUdySPIcjs9dTYJcFZBGM91FW9EiIfY7VmwgvQVoTDP24AYZUXL7omWrE1mXw~c7A12I5N64tQidMBQ1TNPg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="An overview of the interspecies mapping algorithm." path-from-xml="dsq01701.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5029066/dsq01701.jpeg" path-from-xml="dsq01701.jpeg" target="_blank">View large</a><a section="5029066" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq017/2/dsq01701.jpeg?Expires=2147483647&Signature=sf--y2TCOOERe6wfjtFSp7xZ~KeEEiTcJMySaXDgWzwsgpF-aDhUAPHNUbSHQrxvIK2JRE~Dc5YNDVqst1kCpDiNNUAHPOWDzlFWM~cGjmdYdepFbvx2YBWsO8dS4Qay-RJaKAjKE~yZKjk0Gy1KMWl2jw4rc~eZgbB0LGVC9mhF0bjKL2QFj-WpC50QnudxeIAbeZ7RGcyCVySn8tKJpc1Gh35BKowDxENw2e-1E4d-aw1cypg0WrNjr0gkZHH5OCfM~6e~fVwsmnRKuqN5FF94bDSItOfhxsj5SXrERO1GvICmB35BXrN5uNf8g6jY42fr78ee2mkirKPtnqKayQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5029066&ar=353434&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01701.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>An overview of the interspecies mapping algorithm.</p></div></div><div content-id="DSQ017F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq017/2/m_dsq01701.jpeg?Expires=2147483647&Signature=BT1Dbul9KQzTw2v5Zx8B-EDN7KLu6YB2uZo-THCFmLtBJs2AjBL5y1OqiPJx23WXR99s-qiqsugJgww-yYWFHTFg0KSyCflry2yDVoxqOpTvFgMp8eti97T6bm1o8sMEjqEr-ULpso5TR-2WfyYAO8-~RdOpShPXWYA2t3v2LbDeN-ItXck5Cyd3zPLOwD4xq9n7hFcUkRxdo13gqFDF86Mj9U5qQklgCtDRyamb1VFBzUHL0F3JofeFkaR0oVa5PfpUdySPIcjs9dTYJcFZBGM91FW9EiIfY7VmwgvQVoTDP24AYZUXL7omWrE1mXw~c7A12I5N64tQidMBQ1TNPg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="An overview of the interspecies mapping algorithm." path-from-xml="dsq01701.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5029066/dsq01701.jpeg" path-from-xml="dsq01701.jpeg" target="_blank">View large</a><a section="5029066" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq017/2/dsq01701.jpeg?Expires=2147483647&Signature=sf--y2TCOOERe6wfjtFSp7xZ~KeEEiTcJMySaXDgWzwsgpF-aDhUAPHNUbSHQrxvIK2JRE~Dc5YNDVqst1kCpDiNNUAHPOWDzlFWM~cGjmdYdepFbvx2YBWsO8dS4Qay-RJaKAjKE~yZKjk0Gy1KMWl2jw4rc~eZgbB0LGVC9mhF0bjKL2QFj-WpC50QnudxeIAbeZ7RGcyCVySn8tKJpc1Gh35BKowDxENw2e-1E4d-aw1cypg0WrNjr0gkZHH5OCfM~6e~fVwsmnRKuqN5FF94bDSItOfhxsj5SXrERO1GvICmB35BXrN5uNf8g6jY42fr78ee2mkirKPtnqKayQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5029066&ar=353434&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01701.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>An overview of the interspecies mapping algorithm.</p></div></div><p>Before precise alignment between FLcDNAs and a genome, we first used blastn to approximately define genomic regions that correspond to FLcDNAs. We used est2genome<sup><a reveal-id="DSQ017C40" data-open="DSQ017C40" class="link link-ref link-reveal xref-bibr">40</a></sup> to align FLcDNAs to the corresponding genomic regions selected by blastn. Next, we invented methods to solve three problems with interspecies alignment (Fig. <a reveal-id="DSQ017F2" data-open="DSQ017F2" class="link link-reveal link-table xref-fig">2</a>). Because tandemly duplicated genes tended to be accidentally combined into one gene during alignments, we divided these regions such that each region contained only one candidate gene and aligned the FLcDNAs to each region (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). To improve the detection of exon–intron boundaries by est2genome, multiple lines of transcript evidence were employed as follows. We scored introns using linear discriminant analyses, based on a splice site model and alignments around the splice sites, such that the most probable intron is selected (for details, see Supplementary Methods). Finally, we also discarded short introns whose length was &lt;50 bp because the percentage of such short introns identified by intraspecies mapping was 10 times smaller than that by interspecies mapping (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). </p>                        <a id="5029071" scrollto-destination="5029071"></a>
<div id="DSQ017F2" data-id="DSQ017F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq017/2/m_dsq01702.jpeg?Expires=2147483647&Signature=jnKNFnNkiEqEv7zed6WOFUxHSixEU8Xv~jSIS5PIYS5rb5WkrbisYhmocGIj0PhqPO1t3HvPHZz6GqUVYeQ2M1Tv7oAyTt8xpQCHufE2LDpeToEVqP00WmlPNAkTdBNVXMl0qNhsFAuLnQf~LWrwG8QsnSEDRa3yenRUXDHxlkiIwM83yVNu0-iz~vawXUh6fS18IapdKKqrOj7LrACtsPkZpVJtCjsiw0Z2AzhJhalvDlcI9h93fst5K8oPI0Sz4chX5A4Kn0-QReJJ-kYKNUJCHuvqGD20741g9OsfDrd19Z1qO1lw9lKiHvIBUN~j6Bem42Re7guuc2bWNint6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Problems with interspecies mapping. Alignment errors between a given FLcDNA and genome sequence pair have three possible causes. (A) Multiple duplicated genes encompassed by a single cDNA. (B) Erroneously short introns. (C) Alignment errors around splice sites." path-from-xml="dsq01702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5029071/dsq01702.jpeg" path-from-xml="dsq01702.jpeg" target="_blank">View large</a><a section="5029071" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq017/2/dsq01702.jpeg?Expires=2147483647&Signature=R50~OjAmR2beRm4QM32QMEcR4ZsxF427xKwYsZO0theAbMYDPUD3fHwrI1VpJJyBIhzQQbyyj15XI4DtonJJUQXWB1TZdE7NQXJLEFVG9KchQBu9~VocJq9cm6w6R9ZXoiq77zOaT38F54R17YTS2Or06maBgNCzNGI4AixWzuu5AxGPdREc1AUuRd5-xtL~i4n4ag76BAC-NBFqcbve9xYlfN6jBgrgib~kfx-hKX5pkwnXVsthnRWOTL16mHQNxWbtHiwU4OcfSLvDgU5-bO6XzeFjITFi53ehqoBExt1N2oDHVw~QzP-fgOv8WCsmf6ACxPBgRrFkYCmaB0qRxg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5029071&ar=353434&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Problems with interspecies mapping. Alignment errors between a given FLcDNA and genome sequence pair have three possible causes. (A) Multiple duplicated genes encompassed by a single cDNA. (B) Erroneously short introns. (C) Alignment errors around splice sites.</p></div></div><div content-id="DSQ017F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq017/2/m_dsq01702.jpeg?Expires=2147483647&Signature=jnKNFnNkiEqEv7zed6WOFUxHSixEU8Xv~jSIS5PIYS5rb5WkrbisYhmocGIj0PhqPO1t3HvPHZz6GqUVYeQ2M1Tv7oAyTt8xpQCHufE2LDpeToEVqP00WmlPNAkTdBNVXMl0qNhsFAuLnQf~LWrwG8QsnSEDRa3yenRUXDHxlkiIwM83yVNu0-iz~vawXUh6fS18IapdKKqrOj7LrACtsPkZpVJtCjsiw0Z2AzhJhalvDlcI9h93fst5K8oPI0Sz4chX5A4Kn0-QReJJ-kYKNUJCHuvqGD20741g9OsfDrd19Z1qO1lw9lKiHvIBUN~j6Bem42Re7guuc2bWNint6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Problems with interspecies mapping. Alignment errors between a given FLcDNA and genome sequence pair have three possible causes. (A) Multiple duplicated genes encompassed by a single cDNA. (B) Erroneously short introns. (C) Alignment errors around splice sites." path-from-xml="dsq01702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5029071/dsq01702.jpeg" path-from-xml="dsq01702.jpeg" target="_blank">View large</a><a section="5029071" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq017/2/dsq01702.jpeg?Expires=2147483647&Signature=R50~OjAmR2beRm4QM32QMEcR4ZsxF427xKwYsZO0theAbMYDPUD3fHwrI1VpJJyBIhzQQbyyj15XI4DtonJJUQXWB1TZdE7NQXJLEFVG9KchQBu9~VocJq9cm6w6R9ZXoiq77zOaT38F54R17YTS2Or06maBgNCzNGI4AixWzuu5AxGPdREc1AUuRd5-xtL~i4n4ag76BAC-NBFqcbve9xYlfN6jBgrgib~kfx-hKX5pkwnXVsthnRWOTL16mHQNxWbtHiwU4OcfSLvDgU5-bO6XzeFjITFi53ehqoBExt1N2oDHVw~QzP-fgOv8WCsmf6ACxPBgRrFkYCmaB0qRxg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5029071&ar=353434&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Problems with interspecies mapping. Alignment errors between a given FLcDNA and genome sequence pair have three possible causes. (A) Multiple duplicated genes encompassed by a single cDNA. (B) Erroneously short introns. (C) Alignment errors around splice sites.</p></div></div><p>CDSs in predicted transcripts were determined on the basis of homology searches by blastx<sup><a reveal-id="DSQ017C41" data-open="DSQ017C41" class="link link-ref link-reveal xref-bibr">41</a></sup> against the Uniprot<sup><a reveal-id="DSQ017C42" data-open="DSQ017C42" class="link link-ref link-reveal xref-bibr">42</a></sup> and RefSeq<sup><a reveal-id="DSQ017C43" data-open="DSQ017C43" class="link link-ref link-reveal xref-bibr">43</a></sup> databases. If no homologs were found in the protein databases, GeneMark<sup><a reveal-id="DSQ017C44" data-open="DSQ017C44" class="link link-ref link-reveal xref-bibr">44</a></sup> was used, or the longest CDSs were employed. Either the 5′- or the 3′-end was frequently truncated after interspecies mapping because of the relatively low sequence conservation observed for CDS termini. Therefore, both ends of the mapped transcripts were extended so that predicted CDSs contained the start and the stop codons. Finally, from a set of overlapping transcripts in a given locus, a single representative transcript that had the longest CDS was selected.</p>                            <h3 scrollto-destination=5029076 id="5029076" class="section-title" data-legacy-id=s2c>2.3 Evaluation of gene predictions</h3>
<p>To evaluate our methods, we compared the exon–intron structures and the CDSs derived from the interspecies mapping with the aforementioned reference sets of <em>O. sativa</em>, <em>Z. mays</em>, and <em>A. thaliana</em>. We evaluated only the predictions that overlapped with the reference set. Exon–intron boundaries, introns, all introns within a reference CDS (referred to as ‘all introns’ hereafter), and the entire CDSs were examined. We evaluated mapping results for each FLcDNA. Specificity (SP) was defined as TP/(TP + FP), and sensitivity (SN) was defined as TP/(TP + FN), where TP is the number of true positives, FP is the number of false positives, and FN is the number of false negatives. Single-exon genes were considered only in the CDS evaluation because they have no introns. To evaluate the CDS extension process, we examined whether the start and the stop codons of the reference set were included in the predictions. Reference CDSs that did not overlap with predictions were not used.</p><p>To investigate relationships between the nucleotide identity and mapping ratio, the nucleotide identity between a given FLcDNA and the genomic locus to which it was assigned by est2genome was calculated. The mapping ratio represents the percentage of FLcDNAs that were mapped by est2genome divided by all FLcDNAs used. To investigate correlations between nucleotide identity and SP, we selected representative CDSs for each species and calculated the SP of gene predictions between species for genomes and FLcDNAs.</p>                            <h3 scrollto-destination=5029082 id="5029082" class="section-title" data-legacy-id=s2d>2.4 Comparison with other programs</h3>
<p>The newly developed annotation pipeline described in the present work was compared with three <em>ab initio</em> gene prediction tools, including GeneZilla,<sup><a reveal-id="DSQ017C45" data-open="DSQ017C45" class="link link-ref link-reveal xref-bibr">45</a></sup> GlimmerHMM,<sup><a reveal-id="DSQ017C46" data-open="DSQ017C46" class="link link-ref link-reveal xref-bibr">46</a></sup> and GeneMark.hmm.<sup><a reveal-id="DSQ017C47" data-open="DSQ017C47" class="link link-ref link-reveal xref-bibr">47</a></sup> The options for <em>O. sativa</em> gene predictions were employed in GlimmerHMM and GeneMark.hmm predictions. For GeneZilla, we randomly selected 5000 genes from the rice annotation<sup><a reveal-id="DSQ017C37" data-open="DSQ017C37" class="link link-ref link-reveal xref-bibr">37</a></sup> and used 4124, which contained both the start and the stop codons, to create a training model. Our annotation pipeline also was compared with other cDNA mapping programs, including sim4cc<sup><a reveal-id="DSQ017C23" data-open="DSQ017C23" class="link link-ref link-reveal xref-bibr">23</a></sup> and GeneSeqer.<sup><a reveal-id="DSQ017C24" data-open="DSQ017C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSQ017C48" data-open="DSQ017C48" class="link link-ref link-reveal xref-bibr">48</a></sup> We mapped 71 801 monocot FLcDNAs to the <em>O. sativa</em> genome with the default parameters of sim4cc and GeneSeqer. Alignments that covered 40% or more of FLcDNAs were selected for comparison of prediction efficiency. For the GeneSeqer predictions, the longest representative CDS was selected if multiple transcripts were aligned to one locus.</p>                            <h3 scrollto-destination=5029086 id="5029086" class="section-title" data-legacy-id=s2e>2.5 Estimating the number of shared genes in monocots</h3>
<div class="block-child-p">If FLcDNAs are collected randomly, the number of FLcDNAs per locus obeys a Poisson distribution. We estimated the number of all loci (<em>N</em>) from the number of loci found (<em>L</em>) and the number of FLcDNAs mapped (<em>M</em>) using the following formula<sup><a reveal-id="DSQ017C49" data-open="DSQ017C49" class="link link-ref link-reveal xref-bibr">49</a></sup>: <a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsq017ueq1.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq017/2/m_dsq017ueq1.gif?Expires=2147483647&amp;Signature=0aFNeo0~bbFHNsFgsxtqXQzYEWKnSI9FKdKc8rtR-sKYZNos~U93hkCq15WOfAasALSS2BCNoXdOeQHp39uE9svsVr5Yl1awWtUAHZWJT-y8VCPN8ntCTOG-KSeA8JmDn7D-4alieXHDoqueYqwSE1eH16ZgBDjHopYhwfgrm1NPBMeEjmgnSd7u4pVzNCrxCUoyOLAZ3Kf-ZqmdHpt~qMdCMNnJRvlbyVLBWD6GW296oHfkJwDfzX4eATG6nb3Ux6lCATHw5t80MZ-2wUmSBBCAIF0dBkrPkTBEIh36x~GNfjQ6RFk6xBdjawsK~yBYn1q7~AwNGYekr0EA7uP3tg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsq017ueq1.gif" /></a></div></div></div> Given <em>L</em> and <em>M</em>, <em>N</em> can be estimated. To cover 99% of the genomic loci, <em>L</em>/<em>N</em> = 0.99. Hence, the number of FLcDNAs needed to cover 99% of the loci is: <a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsq017ueq2.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq017/2/m_dsq017ueq2.gif?Expires=2147483647&amp;Signature=cImc3SfkS0y1fIxp0BmcZz2UooLU4Frp98tYKTkcz8BkpGgMTHajcgeS9GbBiplt9TPk5TtJw0aLPAeovZFLovfrRc8bmR6Gy5MfqUXOyw-hGkIxrLI-8fNhfzY-gn2B2y-gbEN04bbmtLGEJKKUOmpUis~3b6MxXAs6nZBlMZbXVfwo5Crf~38YggONh8EBUoeAN1z0-y9hrgO12c~9a6xHtKjDQW0vCgpv3LQrdG0EhzqA-wWAtHVibhrPAXGweCTBgtvth1LOPC4foSMoNzu4X4aEkJsjaU-0Cxz38aPVx3w45QJcGhamGcSmFkH29XGu8-q9IRBaLopvxXqeAg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsq017ueq2.gif" /></a></div></div></div></div>                            <h2 scrollto-destination=5029092 id="5029092" class="section-title" data-legacy-id=s3>3 Results and discussion</h2>
                            <h3 scrollto-destination=5029094 id="5029094" class="section-title" data-legacy-id=s3a>3.1 Validation of interspecies mapping</h3>
<p>To evaluate our pipeline, we aligned 71 801 FLcDNAs obtained from three monocots (<em>H. vulgare</em>, <em>Z. mays</em>, and <em>T. aestivum</em>) to the <em>O. sativa</em> genome. From this, a total of 54 004 (75.2%) non-rice monocot FLcDNAs were mapped and clustered into 22 500 loci, from which 22 142 CDSs were predicted in the <em>O. sativa</em> genome. If these CDSs overlapped with the reference CDSs, we examined SP and SN for exon–intron boundaries, introns, all introns, and the entire CDSs (Table <a reveal-id="DSQ017TB1" data-open="DSQ017TB1" class="link link-reveal link-table xref-fig">1</a>). As expected,<sup><a reveal-id="DSQ017C22" data-open="DSQ017C22" class="link link-ref link-reveal xref-bibr">22</a></sup> mapping-based methods, including our pipeline, generally showed higher SP and SN at the intron and all-introns levels than the <em>ab initio</em> methods; however, both the SP and the SN drastically dropped at the CDS level when using GeneSeqer. This is partly because the 5′- and 3′-ends of CDSs are poorly conserved and cannot be aligned in many cases. In fact, a significant number of <em>O. sativa</em> start or stop codons were not found in all the three mapping-based methods without extension of 5′- and 3′-ends (Table <a reveal-id="DSQ017TB2" data-open="DSQ017TB2" class="link link-reveal link-table xref-fig">2</a>). In contrast, the extension of the start and the stop codons in our CDS identification led to the inclusion of more than 88 and 91% of these codons, respectively (Table <a reveal-id="DSQ017TB2" data-open="DSQ017TB2" class="link link-reveal link-table xref-fig">2</a>), which were better than the other two methods. After the CDS extension process, the SP and SN of our pipeline increased by 12.2 and 9.8%, respectively, at the CDS level, which suggests that this extension step seems to be of great benefit for accurate CDS prediction. </p>                        <a id="5029098" scrollto-destination="5029098"></a>
<div content-id="DSQ017TB1" class="table-modal"><div class="table-wrap"><div id="DSQ017TB1" class="table-wrap-title" data-id="DSQ017TB1"><div class="label -label">Table 1</div><div class="caption"><p>Comparison of SP and SN in CDSs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Method </th><th colspan="2">Exon–intron boundary<sup>b</sup><hr /> </th><th colspan="2">Intron<sup>b</sup><hr /> </th><th colspan="2">All introns<sup>b</sup><hr /> </th><th colspan="2">Entire CDS<hr /> </th></tr><tr><th>SP </th><th>SN </th><th>SP </th><th>SN </th><th>SP </th><th>SN </th><th>SP </th><th>SN </th></tr></thead><tbody><tr><td>This study </td><td>94.6 </td><td>75.5 </td><td>92.6 </td><td>74.2 </td><td>58.5 </td><td>49.5 </td><td>55.5 </td><td>44.4 </td></tr><tr><td>GeneSeqer </td><td>87.9 </td><td>77.9 </td><td>83.7 </td><td>75.5 </td><td>42.3 </td><td>38.9 </td><td>3.3 </td><td>2.8 </td></tr><tr><td>Sim4cc<sup>a</sup> </td><td>90.8 </td><td>74.3 </td><td>87.9 </td><td>73.0 </td><td>40.5 </td><td>47.2 </td><td>–<sup>c</sup> </td><td>–<sup>c</sup> </td></tr><tr><td>GeneMark.hmm </td><td>87.6 </td><td>87.6 </td><td>76.2 </td><td>80.7 </td><td>22.1 </td><td>25.6 </td><td>24.0 </td><td>25.1 </td></tr><tr><td>GeneZilla </td><td>87.4 </td><td>77.0 </td><td>76.4 </td><td>68.3 </td><td>29.9 </td><td>30.7 </td><td>31.8 </td><td>35.1 </td></tr><tr><td>GlimmerHMM </td><td>91.3 </td><td>75.5 </td><td>81.7 </td><td>69.1 </td><td>36.3 </td><td>35.8 </td><td>39.4 </td><td>38.7 </td></tr></tbody></table></div><div content-id="DSQ017TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Method </th><th colspan="2">Exon–intron boundary<sup>b</sup><hr /> </th><th colspan="2">Intron<sup>b</sup><hr /> </th><th colspan="2">All introns<sup>b</sup><hr /> </th><th colspan="2">Entire CDS<hr /> </th></tr><tr><th>SP </th><th>SN </th><th>SP </th><th>SN </th><th>SP </th><th>SN </th><th>SP </th><th>SN </th></tr></thead><tbody><tr><td>This study </td><td>94.6 </td><td>75.5 </td><td>92.6 </td><td>74.2 </td><td>58.5 </td><td>49.5 </td><td>55.5 </td><td>44.4 </td></tr><tr><td>GeneSeqer </td><td>87.9 </td><td>77.9 </td><td>83.7 </td><td>75.5 </td><td>42.3 </td><td>38.9 </td><td>3.3 </td><td>2.8 </td></tr><tr><td>Sim4cc<sup>a</sup> </td><td>90.8 </td><td>74.3 </td><td>87.9 </td><td>73.0 </td><td>40.5 </td><td>47.2 </td><td>–<sup>c</sup> </td><td>–<sup>c</sup> </td></tr><tr><td>GeneMark.hmm </td><td>87.6 </td><td>87.6 </td><td>76.2 </td><td>80.7 </td><td>22.1 </td><td>25.6 </td><td>24.0 </td><td>25.1 </td></tr><tr><td>GeneZilla </td><td>87.4 </td><td>77.0 </td><td>76.4 </td><td>68.3 </td><td>29.9 </td><td>30.7 </td><td>31.8 </td><td>35.1 </td></tr><tr><td>GlimmerHMM </td><td>91.3 </td><td>75.5 </td><td>81.7 </td><td>69.1 </td><td>36.3 </td><td>35.8 </td><td>39.4 </td><td>38.7 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>All values are expressed as percentages (%).</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>Sim4cc does not report a representative transcript in a single locus, so each mapping result was evaluated separately.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>CDS regions of the reference set were evaluated.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Sim4cc does not predict CDS regions.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5029098" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ017TB1" class="table-wrap-title" data-id="DSQ017TB1"><div class="label -label">Table 1</div><div class="caption"><p>Comparison of SP and SN in CDSs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Method </th><th colspan="2">Exon–intron boundary<sup>b</sup><hr /> </th><th colspan="2">Intron<sup>b</sup><hr /> </th><th colspan="2">All introns<sup>b</sup><hr /> </th><th colspan="2">Entire CDS<hr /> </th></tr><tr><th>SP </th><th>SN </th><th>SP </th><th>SN </th><th>SP </th><th>SN </th><th>SP </th><th>SN </th></tr></thead><tbody><tr><td>This study </td><td>94.6 </td><td>75.5 </td><td>92.6 </td><td>74.2 </td><td>58.5 </td><td>49.5 </td><td>55.5 </td><td>44.4 </td></tr><tr><td>GeneSeqer </td><td>87.9 </td><td>77.9 </td><td>83.7 </td><td>75.5 </td><td>42.3 </td><td>38.9 </td><td>3.3 </td><td>2.8 </td></tr><tr><td>Sim4cc<sup>a</sup> </td><td>90.8 </td><td>74.3 </td><td>87.9 </td><td>73.0 </td><td>40.5 </td><td>47.2 </td><td>–<sup>c</sup> </td><td>–<sup>c</sup> </td></tr><tr><td>GeneMark.hmm </td><td>87.6 </td><td>87.6 </td><td>76.2 </td><td>80.7 </td><td>22.1 </td><td>25.6 </td><td>24.0 </td><td>25.1 </td></tr><tr><td>GeneZilla </td><td>87.4 </td><td>77.0 </td><td>76.4 </td><td>68.3 </td><td>29.9 </td><td>30.7 </td><td>31.8 </td><td>35.1 </td></tr><tr><td>GlimmerHMM </td><td>91.3 </td><td>75.5 </td><td>81.7 </td><td>69.1 </td><td>36.3 </td><td>35.8 </td><td>39.4 </td><td>38.7 </td></tr></tbody></table></div><div content-id="DSQ017TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Method </th><th colspan="2">Exon–intron boundary<sup>b</sup><hr /> </th><th colspan="2">Intron<sup>b</sup><hr /> </th><th colspan="2">All introns<sup>b</sup><hr /> </th><th colspan="2">Entire CDS<hr /> </th></tr><tr><th>SP </th><th>SN </th><th>SP </th><th>SN </th><th>SP </th><th>SN </th><th>SP </th><th>SN </th></tr></thead><tbody><tr><td>This study </td><td>94.6 </td><td>75.5 </td><td>92.6 </td><td>74.2 </td><td>58.5 </td><td>49.5 </td><td>55.5 </td><td>44.4 </td></tr><tr><td>GeneSeqer </td><td>87.9 </td><td>77.9 </td><td>83.7 </td><td>75.5 </td><td>42.3 </td><td>38.9 </td><td>3.3 </td><td>2.8 </td></tr><tr><td>Sim4cc<sup>a</sup> </td><td>90.8 </td><td>74.3 </td><td>87.9 </td><td>73.0 </td><td>40.5 </td><td>47.2 </td><td>–<sup>c</sup> </td><td>–<sup>c</sup> </td></tr><tr><td>GeneMark.hmm </td><td>87.6 </td><td>87.6 </td><td>76.2 </td><td>80.7 </td><td>22.1 </td><td>25.6 </td><td>24.0 </td><td>25.1 </td></tr><tr><td>GeneZilla </td><td>87.4 </td><td>77.0 </td><td>76.4 </td><td>68.3 </td><td>29.9 </td><td>30.7 </td><td>31.8 </td><td>35.1 </td></tr><tr><td>GlimmerHMM </td><td>91.3 </td><td>75.5 </td><td>81.7 </td><td>69.1 </td><td>36.3 </td><td>35.8 </td><td>39.4 </td><td>38.7 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>All values are expressed as percentages (%).</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>Sim4cc does not report a representative transcript in a single locus, so each mapping result was evaluated separately.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>CDS regions of the reference set were evaluated.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Sim4cc does not predict CDS regions.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5029098" target="_blank">View Large</a></div></div>                        <a id="5029100" scrollto-destination="5029100"></a>
<div content-id="DSQ017TB2" class="table-modal"><div class="table-wrap"><div id="DSQ017TB2" class="table-wrap-title" data-id="DSQ017TB2"><div class="label -label">Table 2</div><div class="caption"><p>Percentage of <em>O. sativa</em> start and stop codons included in the alignments</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Method </th><th colspan="2">Before extension (%)<hr /> </th><th colspan="2">After extension (%)<hr /> </th></tr><tr><th>Start </th><th>Stop </th><th>Start </th><th>Stop </th></tr></thead><tbody><tr><td>This study </td><td>67.3 </td><td>72.1 </td><td>88.9 </td><td>91.2 </td></tr><tr><td>GeneSeqer </td><td>49.8 </td><td>55.1 </td><td>79.7 </td><td>82.5 </td></tr><tr><td>Sim4cc<sup>a</sup> </td><td>45.7 </td><td>58.1 </td><td>82.2 </td><td>85.7 </td></tr></tbody></table></div><div content-id="DSQ017TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Method </th><th colspan="2">Before extension (%)<hr /> </th><th colspan="2">After extension (%)<hr /> </th></tr><tr><th>Start </th><th>Stop </th><th>Start </th><th>Stop </th></tr></thead><tbody><tr><td>This study </td><td>67.3 </td><td>72.1 </td><td>88.9 </td><td>91.2 </td></tr><tr><td>GeneSeqer </td><td>49.8 </td><td>55.1 </td><td>79.7 </td><td>82.5 </td></tr><tr><td>Sim4cc<sup>a</sup> </td><td>45.7 </td><td>58.1 </td><td>82.2 </td><td>85.7 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Longest CDSs were selected.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5029100" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ017TB2" class="table-wrap-title" data-id="DSQ017TB2"><div class="label -label">Table 2</div><div class="caption"><p>Percentage of <em>O. sativa</em> start and stop codons included in the alignments</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Method </th><th colspan="2">Before extension (%)<hr /> </th><th colspan="2">After extension (%)<hr /> </th></tr><tr><th>Start </th><th>Stop </th><th>Start </th><th>Stop </th></tr></thead><tbody><tr><td>This study </td><td>67.3 </td><td>72.1 </td><td>88.9 </td><td>91.2 </td></tr><tr><td>GeneSeqer </td><td>49.8 </td><td>55.1 </td><td>79.7 </td><td>82.5 </td></tr><tr><td>Sim4cc<sup>a</sup> </td><td>45.7 </td><td>58.1 </td><td>82.2 </td><td>85.7 </td></tr></tbody></table></div><div content-id="DSQ017TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Method </th><th colspan="2">Before extension (%)<hr /> </th><th colspan="2">After extension (%)<hr /> </th></tr><tr><th>Start </th><th>Stop </th><th>Start </th><th>Stop </th></tr></thead><tbody><tr><td>This study </td><td>67.3 </td><td>72.1 </td><td>88.9 </td><td>91.2 </td></tr><tr><td>GeneSeqer </td><td>49.8 </td><td>55.1 </td><td>79.7 </td><td>82.5 </td></tr><tr><td>Sim4cc<sup>a</sup> </td><td>45.7 </td><td>58.1 </td><td>82.2 </td><td>85.7 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Longest CDSs were selected.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5029100" target="_blank">View Large</a></div></div><p>Low sequence similarity in untranslated regions (UTRs) caused more serious problems. The SP of introns in UTRs was &lt;33%, which was much lower than the above 90% SP of non-UTR-based CDSs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). This result indicates that interspecies mapping should focus on CDSs rather than the entire transcripts.</p><p>Improvement of gene structure identification in three steps (detection of tandem duplications, removal of short introns, and exon–intron boundary corrections) of our pipeline was validated (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). The pipeline detected candidate regions of tandem duplication in 11 287 alignments (see Supplementary Methods), and they were consequently reflected in 3510 loci. In addition, 10 097 short introns (&lt;50 bp) to be removed were found in 4480 loci. As a result, 7.5% SP and 2.5% SN at the all-introns level were increased in the three steps (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). In this way, without sacrificing SN, SP was largely improved.</p>                            <h3 scrollto-destination=5029105 id="5029105" class="section-title" data-legacy-id=s3b>3.2 Applicability of interspecies mapping</h3>
<p>Although interspecies mapping is useful for detecting loci in related species, FLcDNAs from distantly related species may lead to incorrect predictions. To test the applicability of the interspecies mapping approach, we examined the relationship between species classification and mapping ratios. For this purpose, we used the <em>O. sativa</em>, <em>Z. mays</em>, and <em>A. thaliana</em> genomes because they had large numbers of FLcDNAs that could be used to create high-quality reference sets. We applied monocot FLcDNAs to the <em>O. sativa</em> and <em>Z. mays</em> genomes and dicot FLcDNAs to the <em>A. thaliana</em> genome. We also applied our algorithm to a monocot–dicot comparison. Figure <a reveal-id="DSQ017F3" data-open="DSQ017F3" class="link link-reveal link-table xref-fig">3</a> shows the relationship between the mapping ratio and the species classification. In general, mapping within both monocots and dicots yielded a higher mapping ratio than mapping between a monocot and a dicot. Therefore, comparisons between distantly related species, such as a monocot and a dicot, do not seem to be appropriate. Within-monocot comparisons displayed a higher ratio than within-dicot comparisons. In addition, monocot-to-<em>Arabidopsis</em> mapping showed a lower mapping ratio compared with dicot-to-rice/maize mapping, although evolutionary distances of these two comparisons were same. This difference of mapping ratios is possible because <em>A. thaliana</em> might have undergone genome reduction that led to a lineage-specific deletion of genes. </p>                        <a id="5029109" scrollto-destination="5029109"></a>
<div id="DSQ017F3" data-id="DSQ017F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq017/2/m_dsq01703.jpeg?Expires=2147483647&Signature=Uwq7hSCv3R4IHedkRSM6IyU7qjlBVpCo0Q2bGd6EL~69zzjFWoUQPXp7YdoH6gk2TfRZohG4QTZH3hYyH3S94MgWT78KnOC59IzWwTFSuLSewxUJ7j2f~sDe7fJMqv7ZmXaueQs~Fd78uuuaRUdOjcs78-u3fsvYtAiIg0ZGQhKDQxS0QR47LGNYfpKT0UasWgXvgQzHcXKtd5~s0Rnvnbwgeqk9aCdAQ6Md7R6YaB5jg6c~2z~ufatIyDIpwXfJrPPkgp1M6CQosZ1HDtb0TCUlqzIsQIBNQ54Lwl1Jqf1F-G4OpBLLD0aPaZJ0G-~WbOIlnWmbCxtM7NKnS5Agkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Relationship between species classification and mapping ratio. The horizontal axis indicates the classification, and the vertical axis indicates the mapping ratio. We mapped FLcDNAs from three monocots and four dicots to the O. sativa (rice) and Z. mays (maize) genomes, and FLcDNAs from three dicots and six monocots to the A. thaliana (Arabidopsis) genome. Bars at the top of the boxes represent the standard deviations." path-from-xml="dsq01703.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5029109/dsq01703.jpeg" path-from-xml="dsq01703.jpeg" target="_blank">View large</a><a section="5029109" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq017/2/dsq01703.jpeg?Expires=2147483647&Signature=OVB7-7Oi2p0dCLMK7Ij0M7Bs46vMnHaC~i16gZwDDg3Thvh-WXmA69xqa06w4j6qWD4LPyOMRcMOA6ZBGOe6~6fXLiLB2CMSe1zn4UymfBOYdQFO3a38693cdpWR7vFC~elGCumBaaYAxMsV4jx7hnkn93~XH2si~ZbgPJB2RqAkvI4EKioByXZgYrVSVhLlWkCPBJiZD4R1d6NpuNUiW4JvvmkOIbPi15g-KUqai5nrGqYtJZ2faMIAzRrjIjikvGhYR5iutQ6O3~4864HCnaiIQfH9StHbmrjumqc4fPDw~00~JhaAgQNS2t~A9HCWAckPpjynupzIppMsf6p0Ag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5029109&ar=353434&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01703.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Relationship between species classification and mapping ratio. The horizontal axis indicates the classification, and the vertical axis indicates the mapping ratio. We mapped FLcDNAs from three monocots and four dicots to the <em>O. sativa</em> (rice) and <em>Z. mays</em> (maize) genomes, and FLcDNAs from three dicots and six monocots to the <em>A. thaliana</em> (<em>Arabidopsis</em>) genome. Bars at the top of the boxes represent the standard deviations.</p></div></div><div content-id="DSQ017F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq017/2/m_dsq01703.jpeg?Expires=2147483647&Signature=Uwq7hSCv3R4IHedkRSM6IyU7qjlBVpCo0Q2bGd6EL~69zzjFWoUQPXp7YdoH6gk2TfRZohG4QTZH3hYyH3S94MgWT78KnOC59IzWwTFSuLSewxUJ7j2f~sDe7fJMqv7ZmXaueQs~Fd78uuuaRUdOjcs78-u3fsvYtAiIg0ZGQhKDQxS0QR47LGNYfpKT0UasWgXvgQzHcXKtd5~s0Rnvnbwgeqk9aCdAQ6Md7R6YaB5jg6c~2z~ufatIyDIpwXfJrPPkgp1M6CQosZ1HDtb0TCUlqzIsQIBNQ54Lwl1Jqf1F-G4OpBLLD0aPaZJ0G-~WbOIlnWmbCxtM7NKnS5Agkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Relationship between species classification and mapping ratio. The horizontal axis indicates the classification, and the vertical axis indicates the mapping ratio. We mapped FLcDNAs from three monocots and four dicots to the O. sativa (rice) and Z. mays (maize) genomes, and FLcDNAs from three dicots and six monocots to the A. thaliana (Arabidopsis) genome. Bars at the top of the boxes represent the standard deviations." path-from-xml="dsq01703.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5029109/dsq01703.jpeg" path-from-xml="dsq01703.jpeg" target="_blank">View large</a><a section="5029109" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq017/2/dsq01703.jpeg?Expires=2147483647&Signature=OVB7-7Oi2p0dCLMK7Ij0M7Bs46vMnHaC~i16gZwDDg3Thvh-WXmA69xqa06w4j6qWD4LPyOMRcMOA6ZBGOe6~6fXLiLB2CMSe1zn4UymfBOYdQFO3a38693cdpWR7vFC~elGCumBaaYAxMsV4jx7hnkn93~XH2si~ZbgPJB2RqAkvI4EKioByXZgYrVSVhLlWkCPBJiZD4R1d6NpuNUiW4JvvmkOIbPi15g-KUqai5nrGqYtJZ2faMIAzRrjIjikvGhYR5iutQ6O3~4864HCnaiIQfH9StHbmrjumqc4fPDw~00~JhaAgQNS2t~A9HCWAckPpjynupzIppMsf6p0Ag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5029109&ar=353434&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01703.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Relationship between species classification and mapping ratio. The horizontal axis indicates the classification, and the vertical axis indicates the mapping ratio. We mapped FLcDNAs from three monocots and four dicots to the <em>O. sativa</em> (rice) and <em>Z. mays</em> (maize) genomes, and FLcDNAs from three dicots and six monocots to the <em>A. thaliana</em> (<em>Arabidopsis</em>) genome. Bars at the top of the boxes represent the standard deviations.</p></div></div><p>To further ascertain the degree to which nucleotide differences affect the accuracy of the exon–intron structure predictions generated by interspecies mapping, we examined the relationship between the nucleotide sequence identity and the SP of all introns in our representative CDSs (Fig. <a reveal-id="DSQ017F4" data-open="DSQ017F4" class="link link-reveal link-table xref-fig">4</a>). The correlation coefficient for this relationship is 0.88 (<em>P</em> = 8.6 × 10<sup>−9</sup>). This suggests that interspecies mapping within closely related species should detect a significant number of true exon–intron structures. If the nucleotide identity is more than 80%, the SP of all introns in a given set of CDSs is expected to be ∼60% or more. </p>                        <a id="5029114" scrollto-destination="5029114"></a>
<div id="DSQ017F4" data-id="DSQ017F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq017/2/m_dsq01704.jpeg?Expires=2147483647&Signature=RAOdFEXzjpDT3~NsrKgIpyfHa~HJY-Ez6UqEk-~LzKNa65du6l1qO2ya1E~3yv7qCr6KKGOlMwqOEv9S7gxdWzbBg~bZIsnW6dRiatDz4qf-XPz2~8fLrNfY2SEEgTYzsWb8yV1urqKgChNvJwnG8tJUOHSnTiytsrJDvGLoe4NY3y3VnN9bWuiuhHrSk6hDnpc78f7DaygstJCRGIatFUfqLDrfcZT3QfVJSXyFks0SGADFH7MyFr4cZxo0tT-6so9kmdXmVQNABoj7K6XDDd6ec1gsAwPXDHh7yCL3kvhNX-g-UKvFgt1npUkTBzvHvrRmpBXSN~~37a613sU4yA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between nucleotide identity and SP. The horizontal axis indicates the nucleotide identity, and the vertical axis indicates the SP of all introns in CDSs. Open circles and triangles indicate within-monocot and -dicot mapping, respectively, and filled circles and triangles indicate between-dicot and -monocot mapping, respectively. The straight line shows the linear regression for all data (r = 0.88)." path-from-xml="dsq01704.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5029114/dsq01704.jpeg" path-from-xml="dsq01704.jpeg" target="_blank">View large</a><a section="5029114" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq017/2/dsq01704.jpeg?Expires=2147483647&Signature=URehd6s303a4iyFe8RpULQ5RHrBVWC7skfge2Gz2tCjyr2UYLRPrG8csBFs9GVrQ7zlvW255w~cpbof1hE~fVJU772-gfRzTvtAFwxBJmrAukoQ10AIYW2XHZxNoBHNo2AkZ8qSrkhHoZt3giff9o6tNeYlcBxB6p0pBSQaFongGq3l8w-A3JzwgbFGZqflHwGLouLQL-MkadsENu-hDyg8Kv3EGi6yOP2ck6pl2Kt1YbaeYpv4ywDOEHojrXDsJHHP0ulaUXKgc09YEEGKP3x0QR6KznGOF0-8Rcx2Z~Ybb3v8IPW27~uvhaAk6eAVOQKhxSJ~j4OSLDdKkugSwBQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5029114&ar=353434&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01704.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between nucleotide identity and SP. The horizontal axis indicates the nucleotide identity, and the vertical axis indicates the SP of all introns in CDSs. Open circles and triangles indicate within-monocot and -dicot mapping, respectively, and filled circles and triangles indicate between-dicot and -monocot mapping, respectively. The straight line shows the linear regression for all data (<em>r</em> = 0.88).</p></div></div><div content-id="DSQ017F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq017/2/m_dsq01704.jpeg?Expires=2147483647&Signature=RAOdFEXzjpDT3~NsrKgIpyfHa~HJY-Ez6UqEk-~LzKNa65du6l1qO2ya1E~3yv7qCr6KKGOlMwqOEv9S7gxdWzbBg~bZIsnW6dRiatDz4qf-XPz2~8fLrNfY2SEEgTYzsWb8yV1urqKgChNvJwnG8tJUOHSnTiytsrJDvGLoe4NY3y3VnN9bWuiuhHrSk6hDnpc78f7DaygstJCRGIatFUfqLDrfcZT3QfVJSXyFks0SGADFH7MyFr4cZxo0tT-6so9kmdXmVQNABoj7K6XDDd6ec1gsAwPXDHh7yCL3kvhNX-g-UKvFgt1npUkTBzvHvrRmpBXSN~~37a613sU4yA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between nucleotide identity and SP. The horizontal axis indicates the nucleotide identity, and the vertical axis indicates the SP of all introns in CDSs. Open circles and triangles indicate within-monocot and -dicot mapping, respectively, and filled circles and triangles indicate between-dicot and -monocot mapping, respectively. The straight line shows the linear regression for all data (r = 0.88)." path-from-xml="dsq01704.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5029114/dsq01704.jpeg" path-from-xml="dsq01704.jpeg" target="_blank">View large</a><a section="5029114" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq017/2/dsq01704.jpeg?Expires=2147483647&Signature=URehd6s303a4iyFe8RpULQ5RHrBVWC7skfge2Gz2tCjyr2UYLRPrG8csBFs9GVrQ7zlvW255w~cpbof1hE~fVJU772-gfRzTvtAFwxBJmrAukoQ10AIYW2XHZxNoBHNo2AkZ8qSrkhHoZt3giff9o6tNeYlcBxB6p0pBSQaFongGq3l8w-A3JzwgbFGZqflHwGLouLQL-MkadsENu-hDyg8Kv3EGi6yOP2ck6pl2Kt1YbaeYpv4ywDOEHojrXDsJHHP0ulaUXKgc09YEEGKP3x0QR6KznGOF0-8Rcx2Z~Ybb3v8IPW27~uvhaAk6eAVOQKhxSJ~j4OSLDdKkugSwBQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5029114&ar=353434&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01704.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between nucleotide identity and SP. The horizontal axis indicates the nucleotide identity, and the vertical axis indicates the SP of all introns in CDSs. Open circles and triangles indicate within-monocot and -dicot mapping, respectively, and filled circles and triangles indicate between-dicot and -monocot mapping, respectively. The straight line shows the linear regression for all data (<em>r</em> = 0.88).</p></div></div><p>Figure <a reveal-id="DSQ017F4" data-open="DSQ017F4" class="link link-reveal link-table xref-fig">4</a> also shows that, for the monocot genomes examined, monocot FLcDNAs provided accurate results, whereas the dicot exon–intron structures that were inferred were relatively inaccurate because the dicot genomes used were derived from diverse species. Because all monocots employed in the present study belong to Poaceae (the grass family), their divergence time is relatively short (50–70 million years or less).<sup><a reveal-id="DSQ017C50" data-open="DSQ017C50" class="link link-ref link-reveal xref-bibr">50</a></sup> In contrast, the available dicot FLcDNAs belong to multiple families that are more divergent: <em>A. thaliana</em> is in Brassicaceae, <em>S. lycopersicum</em> in Solanaceae, <em>G. max</em> in Fabaceae, and <em>P. trichocarpa</em> in Salicaceae. For example, <em>A. thaliana</em> and <em>P. trichocarpa</em> probably split around 100–120 million years ago.<sup><a reveal-id="DSQ017C6" data-open="DSQ017C6" class="link link-ref link-reveal xref-bibr">6</a></sup> In fact, nucleotide identities among the dicots <em>A. thaliana</em>, <em>P. trichocarpa</em>, <em>S. lycopersicum</em>, and <em>G. max</em> ranged from 75% to 78%, which is lower than the ∼80% identity detected among monocots. To accurately determine the exon–intron structures within genomes, FLcDNAs from the same family would be preferable.</p>                            <h3 scrollto-destination=5029118 id="5029118" class="section-title" data-legacy-id=s3c>3.3 How many FLcDNAs are necessary for genome annotation?</h3>
<p>A genome is composed of regions that are conserved among species and regions that are specific to a lineage, but interspecies mapping can identify only conserved genes. Here, on the basis of the interspecies mapping results, we estimated the numbers of conserved genes in monocots. If random sampling of FLcDNAs is assumed, the number of loci inferred from the number of mapped FLcDNAs obeys a Poisson distribution (see the ‘Materials and methods’ section). We estimated the number of loci shared between <em>O. sativa</em> and the other monocots to be ∼18 000 because 17 402 loci were identified among 54 004 transcripts. <em>Zea mays</em> and <em>S. bicolor</em> seem to contain ∼21 000 and ∼24 000 conserved loci, respectively. Note that, for this estimation, a single genomic region was selected if an FLcDNA was homologous to multiple regions. We also should note that, because the sampling of FLcDNAs was not completely random, the estimated numbers should be the lower limits. In addition, to more accurately estimate the total number of loci, we must consider lineage-specific genes that are undetectable by sequence similarity.</p><p>Next, we estimated the number of FLcDNAs needed to annotate homologous CDSs among monocot species. Assuming that ∼83 000 FLcDNAs would need to be used to cover 99% of the ∼18 000 loci in <em>O. sativa</em>, with a mapping ratio of 0.75, a total of ∼110 000 FLcDNAs obtained from closely related species would be necessary for prediction of CDSs in <em>O. sativa</em>. This number may be an underestimate if FLcDNAs are not randomly cloned, but most of the genes should be included if more than 100 000 clones are collected from closely related species. Therefore, because there are more than 120 000 FLcDNAs in monocots, new genomes of monocots, such as wheat and barley, would be effectively annotated by using interspecies mapping.</p>                            <h3 scrollto-destination=5029124 id="5029124" class="section-title" data-legacy-id=s3d>3.4 CDS prediction in 10 species</h3>
<p>We applied the interspecies mapping procedure to predict exon–intron structures and CDSs in 10 plant genomes: <em>O. sativa</em>, <em>Z. mays</em>, <em>A. thaliana</em>, <em>S. bicolor</em>, <em>B. distachyon, P. trichocarpa, G. max, V. vinifera</em>, <em>L. japonicus</em>, and <em>C. papaya</em>. Table <a reveal-id="DSQ017TB3" data-open="DSQ017TB3" class="link link-reveal link-table xref-fig">3</a> shows the number of the FLcDNAs used, the mapped FLcDNAs, and the CDS loci. Three examples of our interspecies mapping results for <em>O. sativa</em> were compared with the exon–intron structures of annotation release 2 of the rice genome (RAP2), which was based on an intraspecies mapping procedure<sup><a reveal-id="DSQ017C37" data-open="DSQ017C37" class="link link-ref link-reveal xref-bibr">37</a></sup> (Fig. <a reveal-id="DSQ017F5" data-open="DSQ017F5" class="link link-reveal link-table xref-fig">5</a>). First, three FLcDNAs derived from <em>Z. mays</em> were mapped to the Os11g0635500 locus of RAP2, and their 5′- and 3′-ends were extended. These predicted CDSs displayed exon–intron structures that were identical to those determined by an <em>O. sativa</em> FLcDNA with a CDS and UTRs (Fig. <a reveal-id="DSQ017F5" data-open="DSQ017F5" class="link link-reveal link-table xref-fig">5</a>A). Second, the representative structure at the Os01g0120300 locus determined by an FLcDNA (Fig. <a reveal-id="DSQ017F5" data-open="DSQ017F5" class="link link-reveal link-table xref-fig">5</a>B) was apparently truncated at the 5′-end. In contrast, the interspecies mapping results predicted the complete exon–intron structures including the start and the stop codons. Finally, a <em>H. vulgare</em> FLcDNA (AK248420) predicted a novel transcribed locus candidate between Os08g0206900 and Os08g0207000 that had not yet been detected (Fig. <a reveal-id="DSQ017F5" data-open="DSQ017F5" class="link link-reveal link-table xref-fig">5</a>C). In total, we identified 492 new loci that were missing in release 2 of the rice annotation. The numbers of new loci found in the 10 plant species examined are shown in Table <a reveal-id="DSQ017TB3" data-open="DSQ017TB3" class="link link-reveal link-table xref-fig">3</a>. In addition, we provide exon–intron structure and CDS data that were created by a combination of interspecies and intraspecies mapping methods (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). This comprehensive data set, which contains 210 551 genes from 10 species, will be useful for a large-scale sequence analysis of these flowering plants. </p>                        <a id="5029128" scrollto-destination="5029128"></a>
<div content-id="DSQ017TB3" class="table-modal"><div class="table-wrap"><div id="DSQ017TB3" class="table-wrap-title" data-id="DSQ017TB3"><div class="label -label">Table 3</div><div class="caption"><p>Interspecies mapping results in 10 genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th><em>O. sativa</em> </th><th><em>Z. mays</em> </th><th><em>S. bicolor</em> </th><th><em>B. distachyon</em> </th><th><em>A. thaliana</em> </th><th><em>P. trichocarpa</em> </th><th><em>V. vinifera</em> </th><th><em>G. max</em> </th><th><em>L. japonicus</em> </th><th><em>C. papaya</em> </th></tr></thead><tbody><tr><td>FLcDNA </td><td>71 801 </td><td>60 068 </td><td>120 722 </td><td>120 722 </td><td>22 594 </td><td>53 875 </td><td>58 534 </td><td>53 823 </td><td>58 534 </td><td>58 534 </td></tr><tr><td>Mapped FLcDNAs </td><td>54 004 (75.2%) </td><td>40 343 (67.2%) </td><td>96 142 (79.6%) </td><td>87 174 (72.2%) </td><td>8461 (37.4%) </td><td>24 502 (49.2%) </td><td>27 031 (46.2%) </td><td>22 738 (42.2%) </td><td>22 729 (38.8%) </td><td>26 717 (45.6%) </td></tr><tr><td>CDS loci </td><td>22 142 </td><td>29 001 </td><td>28 446 </td><td>24 522 </td><td>5564 </td><td>15 039 </td><td>11 002 </td><td>17 953 </td><td>12 083 </td><td>8212 </td></tr><tr><td>New loci </td><td>492 </td><td>718 </td><td>1992 </td><td>1638 </td><td>45 </td><td>392 </td><td>496 </td><td>245 </td><td>– </td><td>– </td></tr></tbody></table></div><div content-id="DSQ017TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th><em>O. sativa</em> </th><th><em>Z. mays</em> </th><th><em>S. bicolor</em> </th><th><em>B. distachyon</em> </th><th><em>A. thaliana</em> </th><th><em>P. trichocarpa</em> </th><th><em>V. vinifera</em> </th><th><em>G. max</em> </th><th><em>L. japonicus</em> </th><th><em>C. papaya</em> </th></tr></thead><tbody><tr><td>FLcDNA </td><td>71 801 </td><td>60 068 </td><td>120 722 </td><td>120 722 </td><td>22 594 </td><td>53 875 </td><td>58 534 </td><td>53 823 </td><td>58 534 </td><td>58 534 </td></tr><tr><td>Mapped FLcDNAs </td><td>54 004 (75.2%) </td><td>40 343 (67.2%) </td><td>96 142 (79.6%) </td><td>87 174 (72.2%) </td><td>8461 (37.4%) </td><td>24 502 (49.2%) </td><td>27 031 (46.2%) </td><td>22 738 (42.2%) </td><td>22 729 (38.8%) </td><td>26 717 (45.6%) </td></tr><tr><td>CDS loci </td><td>22 142 </td><td>29 001 </td><td>28 446 </td><td>24 522 </td><td>5564 </td><td>15 039 </td><td>11 002 </td><td>17 953 </td><td>12 083 </td><td>8212 </td></tr><tr><td>New loci </td><td>492 </td><td>718 </td><td>1992 </td><td>1638 </td><td>45 </td><td>392 </td><td>496 </td><td>245 </td><td>– </td><td>– </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5029128" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ017TB3" class="table-wrap-title" data-id="DSQ017TB3"><div class="label -label">Table 3</div><div class="caption"><p>Interspecies mapping results in 10 genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th><em>O. sativa</em> </th><th><em>Z. mays</em> </th><th><em>S. bicolor</em> </th><th><em>B. distachyon</em> </th><th><em>A. thaliana</em> </th><th><em>P. trichocarpa</em> </th><th><em>V. vinifera</em> </th><th><em>G. max</em> </th><th><em>L. japonicus</em> </th><th><em>C. papaya</em> </th></tr></thead><tbody><tr><td>FLcDNA </td><td>71 801 </td><td>60 068 </td><td>120 722 </td><td>120 722 </td><td>22 594 </td><td>53 875 </td><td>58 534 </td><td>53 823 </td><td>58 534 </td><td>58 534 </td></tr><tr><td>Mapped FLcDNAs </td><td>54 004 (75.2%) </td><td>40 343 (67.2%) </td><td>96 142 (79.6%) </td><td>87 174 (72.2%) </td><td>8461 (37.4%) </td><td>24 502 (49.2%) </td><td>27 031 (46.2%) </td><td>22 738 (42.2%) </td><td>22 729 (38.8%) </td><td>26 717 (45.6%) </td></tr><tr><td>CDS loci </td><td>22 142 </td><td>29 001 </td><td>28 446 </td><td>24 522 </td><td>5564 </td><td>15 039 </td><td>11 002 </td><td>17 953 </td><td>12 083 </td><td>8212 </td></tr><tr><td>New loci </td><td>492 </td><td>718 </td><td>1992 </td><td>1638 </td><td>45 </td><td>392 </td><td>496 </td><td>245 </td><td>– </td><td>– </td></tr></tbody></table></div><div content-id="DSQ017TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th><em>O. sativa</em> </th><th><em>Z. mays</em> </th><th><em>S. bicolor</em> </th><th><em>B. distachyon</em> </th><th><em>A. thaliana</em> </th><th><em>P. trichocarpa</em> </th><th><em>V. vinifera</em> </th><th><em>G. max</em> </th><th><em>L. japonicus</em> </th><th><em>C. papaya</em> </th></tr></thead><tbody><tr><td>FLcDNA </td><td>71 801 </td><td>60 068 </td><td>120 722 </td><td>120 722 </td><td>22 594 </td><td>53 875 </td><td>58 534 </td><td>53 823 </td><td>58 534 </td><td>58 534 </td></tr><tr><td>Mapped FLcDNAs </td><td>54 004 (75.2%) </td><td>40 343 (67.2%) </td><td>96 142 (79.6%) </td><td>87 174 (72.2%) </td><td>8461 (37.4%) </td><td>24 502 (49.2%) </td><td>27 031 (46.2%) </td><td>22 738 (42.2%) </td><td>22 729 (38.8%) </td><td>26 717 (45.6%) </td></tr><tr><td>CDS loci </td><td>22 142 </td><td>29 001 </td><td>28 446 </td><td>24 522 </td><td>5564 </td><td>15 039 </td><td>11 002 </td><td>17 953 </td><td>12 083 </td><td>8212 </td></tr><tr><td>New loci </td><td>492 </td><td>718 </td><td>1992 </td><td>1638 </td><td>45 </td><td>392 </td><td>496 </td><td>245 </td><td>– </td><td>– </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5029128" target="_blank">View Large</a></div></div>                        <a id="5029129" scrollto-destination="5029129"></a>
<div id="DSQ017F5" data-id="DSQ017F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq017/2/m_dsq01705.jpeg?Expires=2147483647&Signature=jr260LejYe0UW3NoD3DQZTEScNSCYu0at6jiCCtKIlZ0C9YsA9zRfxitfp8k5yTFtpqtTx1FwZmfMwb4ShJmxm2wgItH5z-Gj0kzW6J01tAD3Uaf8A~f7b3JKQS~aaC98rOPpo84f4hFxojRQeMOpLYeEETzDo9XFShgoVMEZhFIdfTawHE7JQHb6SFHOmPmGqs-Rl~lDxd0j8l9wFs7nosreLMZ7Z-PYDqiFSvtCfODYeq9KJpauAjQsvii-O137x1HVYcb1N-xCRp8YZYhzlCCQy0-mIc56GEzWiQdq3V14OImc6CO0wrKfsWujJnnbK3ZCrzqSiyTvZRRp0jcnA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Examples of interspecies mapping to the O. sativa genome. Oryza sativa exon–intron structures (RAP representative) were retrieved from RAP-DB.37 The two characters before the FLcDNA accession numbers indicate the species names: HV for H. vulgare and ZM for Z. mays. Black, red, and yellow regions represent CDSs, extended CDSs, and UTR regions, respectively. (A) The same exon–intron structures between an O. sativa FLcDNA (INSDC: AK067543) and Z. mays FLcDNAs with extended CDS regions. (B) A truncated FLcDNA of O. sativa (INSDC: AK106806) and a complete structure predicted by an H. vulgare FLcDNA. (C) A new locus identified by a H. vulgare FLcDNA (INSDC: AK248420) in a region between Os08g0206900 and Os08g0207000." path-from-xml="dsq01705.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5029129/dsq01705.jpeg" path-from-xml="dsq01705.jpeg" target="_blank">View large</a><a section="5029129" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq017/2/dsq01705.jpeg?Expires=2147483647&Signature=jHUevWJTe-MTtZxBUz9KcmBIA8JmLMy6qYcU9i2t9MUYwwBA~c1qPQvfftIkZAXfCKOsJl~n9PUhSrJHJoNuFZfiFI4f7~dyp4rOGqbt3xWvAkN4tWqEh0dYDDsdag626X7A2YrO-kurVLgHcHNdMCYYMOF61Te7Fsd6nKmbx6mLRxw5rkNC0MpSe3pBLXN24IrDof~tIiGt3tDZsG8Zb0OJYkd-pTdrFWjk9WgY~hA0EdzVeFexPS9KMo-qqVT~yVJ7nryO~C5EvvQDeJ89bkRAFIovXCBeqX472t2w2IDlIPCW5mmO623CyfUi~loiftZzhOcGv1xQDHjWe-4nCw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5029129&ar=353434&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01705.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Examples of interspecies mapping to the <em>O. sativa</em> genome. <em>Oryza sativa</em> exon–intron structures (RAP representative) were retrieved from RAP-DB.<sup><a reveal-id="DSQ017C37" data-open="DSQ017C37" class="link link-ref link-reveal xref-bibr">37</a></sup> The two characters before the FLcDNA accession numbers indicate the species names: HV for <em>H. vulgare</em> and ZM for <em>Z. mays</em>. Black, red, and yellow regions represent CDSs, extended CDSs, and UTR regions, respectively. (A) The same exon–intron structures between an <em>O. sativa</em> FLcDNA (INSDC: AK067543) and <em>Z. mays</em> FLcDNAs with extended CDS regions. (B) A truncated FLcDNA of <em>O. sativa</em> (INSDC: AK106806) and a complete structure predicted by an <em>H. vulgare</em> FLcDNA. (C) A new locus identified by a <em>H. vulgare</em> FLcDNA (INSDC: AK248420) in a region between Os08g0206900 and Os08g0207000.</p></div></div><div content-id="DSQ017F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq017/2/m_dsq01705.jpeg?Expires=2147483647&Signature=jr260LejYe0UW3NoD3DQZTEScNSCYu0at6jiCCtKIlZ0C9YsA9zRfxitfp8k5yTFtpqtTx1FwZmfMwb4ShJmxm2wgItH5z-Gj0kzW6J01tAD3Uaf8A~f7b3JKQS~aaC98rOPpo84f4hFxojRQeMOpLYeEETzDo9XFShgoVMEZhFIdfTawHE7JQHb6SFHOmPmGqs-Rl~lDxd0j8l9wFs7nosreLMZ7Z-PYDqiFSvtCfODYeq9KJpauAjQsvii-O137x1HVYcb1N-xCRp8YZYhzlCCQy0-mIc56GEzWiQdq3V14OImc6CO0wrKfsWujJnnbK3ZCrzqSiyTvZRRp0jcnA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Examples of interspecies mapping to the O. sativa genome. Oryza sativa exon–intron structures (RAP representative) were retrieved from RAP-DB.37 The two characters before the FLcDNA accession numbers indicate the species names: HV for H. vulgare and ZM for Z. mays. Black, red, and yellow regions represent CDSs, extended CDSs, and UTR regions, respectively. (A) The same exon–intron structures between an O. sativa FLcDNA (INSDC: AK067543) and Z. mays FLcDNAs with extended CDS regions. (B) A truncated FLcDNA of O. sativa (INSDC: AK106806) and a complete structure predicted by an H. vulgare FLcDNA. (C) A new locus identified by a H. vulgare FLcDNA (INSDC: AK248420) in a region between Os08g0206900 and Os08g0207000." path-from-xml="dsq01705.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5029129/dsq01705.jpeg" path-from-xml="dsq01705.jpeg" target="_blank">View large</a><a section="5029129" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq017/2/dsq01705.jpeg?Expires=2147483647&Signature=jHUevWJTe-MTtZxBUz9KcmBIA8JmLMy6qYcU9i2t9MUYwwBA~c1qPQvfftIkZAXfCKOsJl~n9PUhSrJHJoNuFZfiFI4f7~dyp4rOGqbt3xWvAkN4tWqEh0dYDDsdag626X7A2YrO-kurVLgHcHNdMCYYMOF61Te7Fsd6nKmbx6mLRxw5rkNC0MpSe3pBLXN24IrDof~tIiGt3tDZsG8Zb0OJYkd-pTdrFWjk9WgY~hA0EdzVeFexPS9KMo-qqVT~yVJ7nryO~C5EvvQDeJ89bkRAFIovXCBeqX472t2w2IDlIPCW5mmO623CyfUi~loiftZzhOcGv1xQDHjWe-4nCw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5029129&ar=353434&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq01705.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Examples of interspecies mapping to the <em>O. sativa</em> genome. <em>Oryza sativa</em> exon–intron structures (RAP representative) were retrieved from RAP-DB.<sup><a reveal-id="DSQ017C37" data-open="DSQ017C37" class="link link-ref link-reveal xref-bibr">37</a></sup> The two characters before the FLcDNA accession numbers indicate the species names: HV for <em>H. vulgare</em> and ZM for <em>Z. mays</em>. Black, red, and yellow regions represent CDSs, extended CDSs, and UTR regions, respectively. (A) The same exon–intron structures between an <em>O. sativa</em> FLcDNA (INSDC: AK067543) and <em>Z. mays</em> FLcDNAs with extended CDS regions. (B) A truncated FLcDNA of <em>O. sativa</em> (INSDC: AK106806) and a complete structure predicted by an <em>H. vulgare</em> FLcDNA. (C) A new locus identified by a <em>H. vulgare</em> FLcDNA (INSDC: AK248420) in a region between Os08g0206900 and Os08g0207000.</p></div></div><p>Because more than 120 000 monocot FLcDNAs are currently available, this FLcDNA data set seems to be large enough to annotate conserved genes in monocot genomes. We could provide relatively a high-quality CDS annotation using intraspecies and interspecies mapping. However, for dicot genomes, the 58 534 dicot FLcDNAs currently available do not seem to be sufficient to predict most of the gene structures. We expect that 50 000 more FLcDNAs from any dicots might improve the coverage of gene prediction in dicot genomes. To complement the shortage of FLcDNAs and explore lineage-specific genes, <em>ab initio</em> prediction programs can be used in combination with interspecies and intraspecies mapping. Interspecies FLcDNA mapping, partly supported by <em>ab initio</em> predictions, will be of great use for cost-effective annotation of newly released agronomically important plant genomes, such as those of wheat and barley.</p>                            <h2 scrollto-destination=5029134 id="5029134" class="section-title" data-legacy-id=s4>4. Availability</h2>
<p>A web service for the mapping of FLcDNAs to genomic DNA sequences is available (<a class="link link-uri" href="http://fpgp.dna.affrc.go.jp/index.html">http://fpgp.dna.affrc.go.jp/index.html</a>). Users can submit a sequence of up to 1 Mb and can specify dicot or monocot FLcDNAs to be mapped. After completion of the requested prediction, an URL indicating the prediction results is sent by e-mail. The CDS identification results of 10 plant genomes are available at our web site (<a class="link link-uri" href="http://fpgp.dna.affrc.go.jp/download.html">http://fpgp.dna.affrc.go.jp/download.html</a>).</p>                            <h2 scrollto-destination=5029138 id="5029138" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5029141 id="5029141" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by a grant from the Ministry of Agriculture, Forestry, and Fisheries of Japan (Genomics for Agricultural Innovation, GIR-1001).</p>                            <h2 scrollto-destination=5029144 id="5029144" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSQ017C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="collab">Arabidopsis Genome Initiative.</div><div class="article-title">Analysis of the genome sequence of the flowering plant <em>Arabidopsis thaliana</em></div>, <div class="source">Nature</div> , <div class="year">2000</div>, vol. <div class="volume">408</div> (pg. <div class="fpage">796</div>-<div class="lpage">815</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/35048692" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/35048692</a></div><div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35048692" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35048692" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35048692"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11130711" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="collab">International Rice Genome Sequencing Project.</div><div class="article-title">The map-based sequence of the rice genome</div>, <div class="source">Nature</div> , <div class="year">2005</div>, vol. <div class="volume">436</div> (pg. <div class="fpage">793</div>-<div class="lpage">800</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature03895" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature03895</a></div><div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature03895" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature03895" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature03895"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16100779" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Paterson</div> <div class="given-names">A.H.</div></div>,  <div class="name"><div class="surname">Bowers</div> <div class="given-names">J.E.</div></div>,  <div class="name"><div class="surname">Bruggmann</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">The <em>Sorghum bicolor</em> genome and the diversification of grasses</div>, <div class="source">Nature</div> , <div class="year">2009</div>, vol. <div class="volume">457</div> (pg. <div class="fpage">551</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature07723" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature07723</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Sorghum%20bicolor%20genome%20and%20the%20diversification%20of%20grasses&amp;author=A.H.%20Paterson&amp;author=J.E.%20Bowers&amp;author=R.%20Bruggmann&amp;publication_year=2009&amp;journal=Nature&amp;volume=457&amp;pages=551-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature07723" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature07723" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature07723"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19189423" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schnable</div> <div class="given-names">P.S.</div></div>,  <div class="name"><div class="surname">Ware</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Fulton</div> <div class="given-names">R.S.</div></div>, et al. </div><div class="article-title">The B73 maize genome: complexity, diversity, and dynamics</div>, <div class="source">Science</div> , <div class="year">2009</div>, vol. <div class="volume">326</div> (pg. <div class="fpage">1112</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.1178534" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1126/science.1178534</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20B73%20maize%20genome%3A%20complexity%2C%20diversity%2C%20and%20dynamics&amp;author=P.S.%20Schnable&amp;author=D.%20Ware&amp;author=R.S.%20Fulton&amp;publication_year=2009&amp;journal=Science&amp;volume=326&amp;pages=1112-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1178534" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1178534" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1178534"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19965430" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="collab">The International Brachypodium Initiative.</div><div class="article-title">Genome sequencing and analysis of the model grass <em>Brachypodium distachyon</em></div>, <div class="source">Nature</div> , <div class="year">2010</div>, vol. <div class="volume">463</div> (pg. <div class="fpage">763</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature08747" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature08747</a></div><div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08747" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08747" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08747"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20148030" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tuskan</div> <div class="given-names">G.A.</div></div>,  <div class="name"><div class="surname">Difazio</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Jansson</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">The genome of black cottonwood, <em>Populus trichocarpa</em> (Torr. &amp; Gray)</div>, <div class="source">Science</div> , <div class="year">2006</div>, vol. <div class="volume">313</div> (pg. <div class="fpage">1596</div>-<div class="lpage">604</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.1128691" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1126/science.1128691</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20black%20cottonwood%2C%20Populus%20trichocarpa%20(Torr.%20%26%20Gray)&amp;author=G.A.%20Tuskan&amp;author=S.%20Difazio&amp;author=S.%20Jansson&amp;publication_year=2006&amp;journal=Science&amp;volume=313&amp;pages=1596-604" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1128691" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1128691" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1128691"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16973872" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jaillon</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Aury</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Noel</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">The grapevine genome sequence suggests ancestral hexaploidization in major angiosperm phyla</div>, <div class="source">Nature</div> , <div class="year">2007</div>, vol. <div class="volume">449</div> (pg. <div class="fpage">463</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature06148" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature06148</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20grapevine%20genome%20sequence%20suggests%20ancestral%20hexaploidization%20in%20major%20angiosperm%20phyla&amp;author=O.%20Jaillon&amp;author=J.M.%20Aury&amp;author=B.%20Noel&amp;publication_year=2007&amp;journal=Nature&amp;volume=449&amp;pages=463-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06148" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06148" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06148"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17721507" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ming</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Hou</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Feng</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">The draft genome of the transgenic tropical fruit tree papaya (<em>Carica papaya</em> Linnaeus)</div>, <div class="source">Nature</div> , <div class="year">2008</div>, vol. <div class="volume">452</div> (pg. <div class="fpage">991</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature06856" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature06856</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20draft%20genome%20of%20the%20transgenic%20tropical%20fruit%20tree%20papaya%20(Carica%20papaya%20Linnaeus)&amp;author=R.%20Ming&amp;author=S.%20Hou&amp;author=Y.%20Feng&amp;publication_year=2008&amp;journal=Nature&amp;volume=452&amp;pages=991-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06856" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06856" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06856"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18432245" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nakamura</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kaneko</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Genome structure of the legume, <em>Lotus japonicus</em></div>, <div class="source">DNA Res.</div> , <div class="year">2008</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">227</div>-<div class="lpage">39</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/dsn008" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/dnares/dsn008</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20structure%20of%20the%20legume%2C%20Lotus%20japonicus&amp;author=S.%20Sato&amp;author=Y.%20Nakamura&amp;author=T.%20Kaneko&amp;publication_year=2008&amp;journal=DNA%20Res.&amp;volume=15&amp;pages=227-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsn008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsn008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsn008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18511435" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schmutz</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Cannon</div> <div class="given-names">S.B.</div></div>,  <div class="name"><div class="surname">Schlueter</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Genome sequence of the palaeopolyploid soybean</div>, <div class="source">Nature</div> , vol. <div class="volume">463</div> (pg. <div class="fpage">178</div>-<div class="lpage">83</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature08670" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature08670</a></div><div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08670" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08670" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08670"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20075913" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schulte</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Close</div> <div class="given-names">T.J.</div></div>,  <div class="name"><div class="surname">Graner</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">The international barley sequencing consortium—at the threshold of efficient access to the barley genome</div>, <div class="source">Plant Physiol.</div> , <div class="year">2009</div>, vol. <div class="volume">149</div> (pg. <div class="fpage">142</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.108.128967" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.108.128967</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20international%20barley%20sequencing%20consortium%E2%80%94at%20the%20threshold%20of%20efficient%20access%20to%20the%20barley%20genome&amp;author=D.%20Schulte&amp;author=T.J.%20Close&amp;author=A.%20Graner&amp;publication_year=2009&amp;journal=Plant%20Physiol.&amp;volume=149&amp;pages=142-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.108.128967" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.108.128967" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.108.128967"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19126706" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Paux</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Sourdille</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Salse</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">A physical map of the 1-gigabase bread wheat chromosome 3B</div>, <div class="source">Science</div> , <div class="year">2008</div>, vol. <div class="volume">322</div> (pg. <div class="fpage">101</div>-<div class="lpage">4</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.1161847" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1126/science.1161847</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20physical%20map%20of%20the%201-gigabase%20bread%20wheat%20chromosome%203B&amp;author=E.%20Paux&amp;author=P.%20Sourdille&amp;author=J.%20Salse&amp;publication_year=2008&amp;journal=Science&amp;volume=322&amp;pages=101-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1161847" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1161847" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1161847"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18832645" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Quinn</div> <div class="given-names">N.L.</div></div>,  <div class="name"><div class="surname">Levenkova</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Chow</div> <div class="given-names">W.</div></div>, et al. </div><div class="article-title">Assessing the feasibility of GS FLX pyrosequencing for sequencing the Atlantic salmon genome</div>, <div class="source">BMC Genomics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">404</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-9-404" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-9-404</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessing%20the%20feasibility%20of%20GS%20FLX%20pyrosequencing%20for%20sequencing%20the%20Atlantic%20salmon%20genome&amp;author=N.L.%20Quinn&amp;author=N.%20Levenkova&amp;author=W.%20Chow&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=404" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-404" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-404" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-404"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18755037" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wicker</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Schlagenhauf</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Graner</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Close</div> <div class="given-names">T.J.</div></div>,  <div class="name"><div class="surname">Keller</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Stein</div> <div class="given-names">N.</div></div>. </div><div class="article-title">454 sequencing put to the test using the complex genome of barley</div>, <div class="source">BMC Genomics</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">275</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-7-275" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-7-275</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=454%20sequencing%20put%20to%20the%20test%20using%20the%20complex%20genome%20of%20barley&amp;author=T.%20Wicker&amp;author=E.%20Schlagenhauf&amp;author=A.%20Graner&amp;author=T.J.%20Close&amp;author=B.%20Keller&amp;author=N.%20Stein&amp;publication_year=2006&amp;journal=BMC%20Genomics&amp;volume=7&amp;pages=275" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-7-275" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-7-275" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-7-275"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17067373" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bennetzen</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Coleman</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Ma</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Ramakrishna</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Consistent over-estimation of gene number in complex plant genomes</div>, <div class="source">Curr. Opin. Plant Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">732</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.pbi.2004.09.003" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.pbi.2004.09.003</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Consistent%20over-estimation%20of%20gene%20number%20in%20complex%20plant%20genomes&amp;author=J.L.%20Bennetzen&amp;author=C.%20Coleman&amp;author=R.%20Liu&amp;author=J.%20Ma&amp;author=W.%20Ramakrishna&amp;publication_year=2004&amp;journal=Curr.%20Opin.%20Plant%20Biol.&amp;volume=7&amp;pages=732-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pbi.2004.09.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2004.09.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pbi.2004.09.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15491923" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cruveiller</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Jabbari</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Clay</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Bernardi</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Incorrectly predicted genes in rice?</div>, <div class="source">Gene</div> , <div class="year">2004</div>, vol. <div class="volume">333</div> (pg. <div class="fpage">187</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.gene.2004.02.039" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.gene.2004.02.039</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Incorrectly%20predicted%20genes%20in%20rice%3F&amp;author=S.%20Cruveiller&amp;author=K.%20Jabbari&amp;author=O.%20Clay&amp;author=G.%20Bernardi&amp;publication_year=2004&amp;journal=Gene&amp;volume=333&amp;pages=187-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2004.02.039" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2004.02.039" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2004.02.039"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15177695" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jabbari</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Cruveiller</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Clay</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Le Saux</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bernardi</div> <div class="given-names">G.</div></div>. </div><div class="article-title">The new genes of rice: a closer look</div>, <div class="source">Trends Plant Sci.</div> , <div class="year">2004</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">281</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.tplants.2004.04.006" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.tplants.2004.04.006</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20new%20genes%20of%20rice%3A%20a%20closer%20look&amp;author=K.%20Jabbari&amp;author=S.%20Cruveiller&amp;author=O.%20Clay&amp;author=J.%20Le%20Saux&amp;author=G.%20Bernardi&amp;publication_year=2004&amp;journal=Trends%20Plant%20Sci.&amp;volume=9&amp;pages=281-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2004.04.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2004.04.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2004.04.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15165559" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Alexandrov</div> <div class="given-names">N.N.</div></div>,  <div class="name"><div class="surname">Troukhan</div> <div class="given-names">M.E.</div></div>,  <div class="name"><div class="surname">Brover</div> <div class="given-names">V.V.</div></div>,  <div class="name"><div class="surname">Tatarinova</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Flavell</div> <div class="given-names">R.B.</div></div>,  <div class="name"><div class="surname">Feldmann</div> <div class="given-names">K.A.</div></div>. </div><div class="article-title">Features of <em>Arabidopsis</em> genes and genome discovered using full-length cDNAs</div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">60</div> (pg. <div class="fpage">69</div>-<div class="lpage">85</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s11103-005-2564-9" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s11103-005-2564-9</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Features%20of%20Arabidopsis%20genes%20and%20genome%20discovered%20using%20full-length%20cDNAs&amp;author=N.N.%20Alexandrov&amp;author=M.E.%20Troukhan&amp;author=V.V.%20Brover&amp;author=T.%20Tatarinova&amp;author=R.B.%20Flavell&amp;author=K.A.%20Feldmann&amp;publication_year=2006&amp;journal=Plant%20Mol.%20Biol.&amp;volume=60&amp;pages=69-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11103-005-2564-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11103-005-2564-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11103-005-2564-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16463100" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Itoh</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Tanaka</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Barrero</div> <div class="given-names">R.A.</div></div>, et al. </div><div class="article-title">Curated genome annotation of <em>Oryza sativa</em> ssp. <em>japonica</em> and comparative genome analysis with <em>Arabidopsis thaliana</em></div>, <div class="source">Genome Res.</div> , <div class="year">2007</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">175</div>-<div class="lpage">83</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.5509507" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.5509507</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Curated%20genome%20annotation%20of%20Oryza%20sativa%20ssp.%20japonica%20and%20comparative%20genome%20analysis%20with%20Arabidopsis%20thaliana&amp;author=T.%20Itoh&amp;author=T.%20Tanaka&amp;author=R.A.%20Barrero&amp;publication_year=2007&amp;journal=Genome%20Res.&amp;volume=17&amp;pages=175-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.5509507" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.5509507" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.5509507"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17210932" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Seki</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Narusaka</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kamiya</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Functional annotation of a full-length <em>Arabidopsis</em> cDNA collection</div>, <div class="source">Science</div> , <div class="year">2002</div>, vol. <div class="volume">296</div> (pg. <div class="fpage">141</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.1071006" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1126/science.1071006</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20annotation%20of%20a%20full-length%20Arabidopsis%20cDNA%20collection&amp;author=M.%20Seki&amp;author=M.%20Narusaka&amp;author=A.%20Kamiya&amp;publication_year=2002&amp;journal=Science&amp;volume=296&amp;pages=141-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1071006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1071006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1071006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11910074" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Seki</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Satou</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Sakurai</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">RIKEN <em>Arabidopsis</em> full-length (RAFL) cDNA and its applications for expression profiling under abiotic stress conditions</div>, <div class="source">J. Exp. Bot.</div> , <div class="year">2004</div>, vol. <div class="volume">55</div> (pg. <div class="fpage">213</div>-<div class="lpage">23</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/jxb/erh007" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/jxb/erh007</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RIKEN%20Arabidopsis%20full-length%20(RAFL)%20cDNA%20and%20its%20applications%20for%20expression%20profiling%20under%20abiotic%20stress%20conditions&amp;author=M.%20Seki&amp;author=M.%20Satou&amp;author=T.%20Sakurai&amp;publication_year=2004&amp;journal=J.%20Exp.%20Bot.&amp;volume=55&amp;pages=213-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/erh007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferh007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferh007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14673034" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liang</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Mao</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Ware</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Stein</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Evidence-based gene predictions in plant genomes</div>, <div class="source">Genome Res.</div> , <div class="year">2009</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">1912</div>-<div class="lpage">23</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.088997.108" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.088997.108</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence-based%20gene%20predictions%20in%20plant%20genomes&amp;author=C.%20Liang&amp;author=L.%20Mao&amp;author=D.%20Ware&amp;author=L.%20Stein&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=19&amp;pages=1912-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.088997.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.088997.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.088997.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19541913" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhou</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Pertea</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Delcher</div> <div class="given-names">A.L.</div></div>,  <div class="name"><div class="surname">Florea</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Sim4cc: a cross-species spliced alignment program</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> pg. <div class="fpage">e80</div> <div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkp319" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkp319</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sim4cc%3A%20a%20cross-species%20spliced%20alignment%20program&amp;author=L.%20Zhou&amp;author=M.%20Pertea&amp;author=A.L.%20Delcher&amp;author=L.%20Florea&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=e80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkp319" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkp319" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkp319"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19429899" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brendel</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Xing</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Gene structure prediction from consensus spliced alignment of multiple ESTs matching the same genomic locus</div>, <div class="source">Bioinformatics</div> , <div class="year">2004</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">1157</div>-<div class="lpage">69</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/bth058" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/bth058</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20structure%20prediction%20from%20consensus%20spliced%20alignment%20of%20multiple%20ESTs%20matching%20the%20same%20genomic%20locus&amp;author=V.%20Brendel&amp;author=L.%20Xing&amp;author=W.%20Zhu&amp;publication_year=2004&amp;journal=Bioinformatics&amp;volume=20&amp;pages=1157-69" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bth058" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbth058" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbth058"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14764557" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kikuchi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Satoh</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nagata</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Collection, mapping, and annotation of over 28,000 cDNA clones from japonica rice</div>, <div class="source">Science</div> , <div class="year">2003</div>, vol. <div class="volume">301</div> (pg. <div class="fpage">376</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.1081288" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1126/science.1081288</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Collection%2C%20mapping%2C%20and%20annotation%20of%20over%2028%2C000%20cDNA%20clones%20from%20japonica%20rice&amp;author=S.%20Kikuchi&amp;author=K.%20Satoh&amp;author=T.%20Nagata&amp;publication_year=2003&amp;journal=Science&amp;volume=301&amp;pages=376-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1081288" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1081288" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1081288"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12869764" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Lu</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Yu</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">A collection of 10,096 indica rice full-length cDNAs reveals highly expressed sequence divergence between <em>Oryza sativa indica</em> and <em>japonica</em> subspecies</div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">65</div> (pg. <div class="fpage">403</div>-<div class="lpage">15</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s11103-007-9174-7" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s11103-007-9174-7</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20collection%20of%2010%2C096%20indica%20rice%20full-length%20cDNAs%20reveals%20highly%20expressed%20sequence%20divergence%20between%20Oryza%20sativa%20indica%20and%20japonica%20subspecies&amp;author=X.%20Liu&amp;author=T.%20Lu&amp;author=S.%20Yu&amp;publication_year=2007&amp;journal=Plant%20Mol.%20Biol.&amp;volume=65&amp;pages=403-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11103-007-9174-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11103-007-9174-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11103-007-9174-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17522955" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lu</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Yu</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Fan</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">Collection and comparative analysis of 1888 full-length cDNAs from wild rice <em>Oryza rufipogon</em> Griff. W1943</div>, <div class="source">DNA Res.</div> , <div class="year">2008</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">285</div>-<div class="lpage">95</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/dsn018" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/dnares/dsn018</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Collection%20and%20comparative%20analysis%20of%201888%20full-length%20cDNAs%20from%20wild%20rice%20Oryza%20rufipogon%20Griff.%20W1943&amp;author=T.%20Lu&amp;author=S.%20Yu&amp;author=D.%20Fan&amp;publication_year=2008&amp;journal=DNA%20Res.&amp;volume=15&amp;pages=285-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsn018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsn018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsn018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18687674" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sato</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Shin</div> <div class="given-names">I.T.</div></div>,  <div class="name"><div class="surname">Seki</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Development of 5006 full-length CDNAs in barley: a tool for accessing cereal genomics resources</div>, <div class="source">DNA Res.</div> , <div class="year">2009</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">81</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/dsn034" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/dnares/dsn034</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%205006%20full-length%20CDNAs%20in%20barley%3A%20a%20tool%20for%20accessing%20cereal%20genomics%20resources&amp;author=K.%20Sato&amp;author=I.T.%20Shin&amp;author=M.%20Seki&amp;publication_year=2009&amp;journal=DNA%20Res.&amp;volume=16&amp;pages=81-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsn034" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsn034" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsn034"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19150987" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Alexandrov</div> <div class="given-names">N.N.</div></div>,  <div class="name"><div class="surname">Brover</div> <div class="given-names">V.V.</div></div>,  <div class="name"><div class="surname">Freidin</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Insights into corn genes derived from large-scale cDNA sequencing</div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">69</div> (pg. <div class="fpage">179</div>-<div class="lpage">94</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s11103-008-9415-4" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s11103-008-9415-4</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insights%20into%20corn%20genes%20derived%20from%20large-scale%20cDNA%20sequencing&amp;author=N.N.%20Alexandrov&amp;author=V.V.%20Brover&amp;author=S.%20Freidin&amp;publication_year=2009&amp;journal=Plant%20Mol.%20Biol.&amp;volume=69&amp;pages=179-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11103-008-9415-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11103-008-9415-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11103-008-9415-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18937034" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jia</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Fu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Zheng</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Annotation and expression profile analysis of 2073 full-length cDNAs from stress-induced maize (<em>Zea mays</em> L.) seedlings</div>, <div class="source">Plant J.</div> , <div class="year">2006</div>, vol. <div class="volume">48</div> (pg. <div class="fpage">710</div>-<div class="lpage">27</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1365-313X.2006.02905.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1365-313X.2006.02905.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Annotation%20and%20expression%20profile%20analysis%20of%202073%20full-length%20cDNAs%20from%20stress-induced%20maize%20(Zea%20mays%20L.)%20seedlings&amp;author=J.%20Jia&amp;author=J.%20Fu&amp;author=J.%20Zheng&amp;publication_year=2006&amp;journal=Plant%20J.&amp;volume=48&amp;pages=710-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2006.48.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2006.48.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2006.48.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17076806" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Soderlund</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Descour</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kudrna</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">Sequencing, mapping, and analysis of 27,455 maize full-length cDNAs</div>, <div class="source">PLoS Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e1000740</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pgen.1000740" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pgen.1000740</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%2C%20mapping%2C%20and%20analysis%20of%2027%2C455%20maize%20full-length%20cDNAs&amp;author=C.%20Soderlund&amp;author=A.%20Descour&amp;author=D.%20Kudrna&amp;publication_year=2009&amp;journal=PLoS%20Genet.&amp;volume=5&amp;pages=e1000740" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000740" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000740" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000740"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19936069" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Umezawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sakurai</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Totoki</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Sequencing and analysis of approximately 40,000 soybean cDNA clones from a full-length-enriched cDNA library</div>, <div class="source">DNA Res.</div> , <div class="year">2008</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">333</div>-<div class="lpage">46</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/dsn024" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/dnares/dsn024</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20and%20analysis%20of%20approximately%2040%2C000%20soybean%20cDNA%20clones%20from%20a%20full-length-enriched%20cDNA%20library&amp;author=T.%20Umezawa&amp;author=T.%20Sakurai&amp;author=Y.%20Totoki&amp;publication_year=2008&amp;journal=DNA%20Res.&amp;volume=15&amp;pages=333-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsn024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsn024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsn024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18927222" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ralph</div> <div class="given-names">S.G.</div></div>,  <div class="name"><div class="surname">Chun</div> <div class="given-names">H.J.</div></div>,  <div class="name"><div class="surname">Cooper</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">Analysis of 4,664 high-quality sequence-finished poplar full-length cDNA clones and their utility for the discovery of genes responding to insect feeding</div>, <div class="source">BMC Genomics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">57</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-9-57" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-9-57</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%204%2C664%20high-quality%20sequence-finished%20poplar%20full-length%20cDNA%20clones%20and%20their%20utility%20for%20the%20discovery%20of%20genes%20responding%20to%20insect%20feeding&amp;author=S.G.%20Ralph&amp;author=H.J.%20Chun&amp;author=D.%20Cooper&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-57" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-57" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-57"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18230180" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mochida</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yoshida</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sakurai</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ogihara</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Shinozaki</div> <div class="given-names">K.</div></div>. </div><div class="article-title">TriFLDB: a database of clustered full-length coding sequences from Triticeae with applications to comparative grass genomics</div>, <div class="source">Plant Physiol.</div> , <div class="year">2009</div>, vol. <div class="volume">150</div> (pg. <div class="fpage">1135</div>-<div class="lpage">46</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.109.138214" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.109.138214</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TriFLDB%3A%20a%20database%20of%20clustered%20full-length%20coding%20sequences%20from%20Triticeae%20with%20applications%20to%20comparative%20grass%20genomics&amp;author=K.%20Mochida&amp;author=T.%20Yoshida&amp;author=T.%20Sakurai&amp;author=Y.%20Ogihara&amp;author=K.%20Shinozaki&amp;publication_year=2009&amp;journal=Plant%20Physiol.&amp;volume=150&amp;pages=1135-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.109.138214" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.109.138214" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.109.138214"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19448038" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Aoki</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yano</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Large-scale analysis of full-length cDNAs from the tomato (<em>Solanum lycopersicum</em>) cultivar Micro-Tom, a reference system for the Solanaceae genomics</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">210</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-11-210" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-11-210</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20analysis%20of%20full-length%20cDNAs%20from%20the%20tomato%20(Solanum%20lycopersicum)%20cultivar%20Micro-Tom%2C%20a%20reference%20system%20for%20the%20Solanaceae%20genomics&amp;author=K.%20Aoki&amp;author=K.%20Yano&amp;author=A.%20Suzuki&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=210" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-210" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-210" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-210"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20350329" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Swarbreck</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Wilks</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Lamesch</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">The <em>Arabidopsis</em> Information Resource (TAIR): gene structure and function annotation</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">D1009</div>-<div class="lpage">14</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkm965" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkm965</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Arabidopsis%20Information%20Resource%20(TAIR)%3A%20gene%20structure%20and%20function%20annotation&amp;author=D.%20Swarbreck&amp;author=C.%20Wilks&amp;author=P.%20Lamesch&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=D1009-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm965" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm965" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm965"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17986450" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tanaka</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Antonio</div> <div class="given-names">B.A.</div></div>,  <div class="name"><div class="surname">Kikuchi</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">The Rice Annotation Project Database (RAP-DB): 2008 update</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">D1028</div>-<div class="lpage">33</div>)<div class="pub-id">
                [<a href="http://www.ncbi.nlm.nih.gov/pubmed/18089549" class="link link-pub-id" target="_blank">PubMed</a>]
            </div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Rice%20Annotation%20Project%20Database%20(RAP-DB)%3A%202008%20update&amp;author=T.%20Tanaka&amp;author=B.A.%20Antonio&amp;author=S.%20Kikuchi&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=D1028-33" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18089549" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ouyang</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Hamilton</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">The TIGR rice genome annotation resource: improvements and new features</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">D883</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkl976" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkl976</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20TIGR%20rice%20genome%20annotation%20resource%3A%20improvements%20and%20new%20features&amp;author=S.%20Ouyang&amp;author=W.%20Zhu&amp;author=J.%20Hamilton&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=D883-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl976" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl976" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl976"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17145706" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Spannagl</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Noubibou</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Haase</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">MIPSPlantsDB–plant database resource for integrative and comparative plant genome research</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">D834</div>-<div class="lpage">40</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkl945" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkl945</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MIPSPlantsDB%E2%80%93plant%20database%20resource%20for%20integrative%20and%20comparative%20plant%20genome%20research&amp;author=M.%20Spannagl&amp;author=O.%20Noubibou&amp;author=D.%20Haase&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=D834-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl945" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl945" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl945"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17202173" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mott</div> <div class="given-names">R.</div></div>. </div><div class="article-title">EST_GENOME: a program to align spliced DNA sequences to unspliced genomic DNA</div>, <div class="source">Comput. Appl. Biosci.</div> , <div class="year">1997</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">477</div>-<div class="lpage">8</div>)<div class="pub-id">
                [<a href="http://www.ncbi.nlm.nih.gov/pubmed/9283765" class="link link-pub-id" target="_blank">PubMed</a>]
            </div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EST_GENOME%3A%20a%20program%20to%20align%20spliced%20DNA%20sequences%20to%20unspliced%20genomic%20DNA&amp;author=R.%20Mott&amp;publication_year=1997&amp;journal=Comput.%20Appl.%20Biosci.&amp;volume=13&amp;pages=477-8" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9283765" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Gish</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Miller</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Myers</div> <div class="given-names">E.W.</div></div>,  <div class="name"><div class="surname">Lipman</div> <div class="given-names">D.J.</div></div>. </div><div class="article-title">Basic local alignment search tool</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1990</div>, vol. <div class="volume">215</div> (pg. <div class="fpage">403</div>-<div class="lpage">10</div>)<div class="pub-id">
                [<a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a>]
            </div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=S.F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.W.%20Myers&amp;author=D.J.%20Lipman&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol.&amp;volume=215&amp;pages=403-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="collab">Uniprot Consortium.</div><div class="article-title">The Universal Protein Resource (UniProt) 2009</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">D169</div>-<div class="lpage">74</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkn664" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkn664</a></div><div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn664" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn664" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn664"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18836194" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pruitt</div> <div class="given-names">K.D.</div></div>,  <div class="name"><div class="surname">Tatusova</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Klimke</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Maglott</div> <div class="given-names">D.R.</div></div>. </div><div class="article-title">NCBI reference sequences: current status, policy and new initiatives</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">D32</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkn721" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkn721</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NCBI%20reference%20sequences%3A%20current%20status%2C%20policy%20and%20new%20initiatives&amp;author=K.D.%20Pruitt&amp;author=T.%20Tatusova&amp;author=W.%20Klimke&amp;author=D.R.%20Maglott&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=D32-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn721" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn721" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn721"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18927115" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Borodovsky</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">McIninch</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Recognition of genes in DNA sequence with ambiguities</div>, <div class="source">Biosystems</div> , <div class="year">1993</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">161</div>-<div class="lpage">71</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/0303-2647(93)90068-N" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/0303-2647(93)90068-N</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recognition%20of%20genes%20in%20DNA%20sequence%20with%20ambiguities&amp;author=M.%20Borodovsky&amp;author=J.%20McIninch&amp;publication_year=1993&amp;journal=Biosystems&amp;volume=30&amp;pages=161-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0303-2647(93)90068-N" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0303-2647(93)90068-N" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0303-2647(93)90068-N"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8374073" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Majoros</div> <div class="given-names">W.H.</div></div>,  <div class="name"><div class="surname">Pertea</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Delcher</div> <div class="given-names">A.L.</div></div>,  <div class="name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div>. </div><div class="article-title">Efficient decoding algorithms for generalized hidden Markov model gene finders</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">6</div> pg. <div class="fpage">16</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2105-6-16" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2105-6-16</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efficient%20decoding%20algorithms%20for%20generalized%20hidden%20Markov%20model%20gene%20finders&amp;author=W.H.%20Majoros&amp;author=M.%20Pertea&amp;author=A.L.%20Delcher&amp;author=S.L.%20Salzberg&amp;publication_year=2005&amp;journal=BMC%20Bioinformatics&amp;volume=6&amp;pages=16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-6-16" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-6-16" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-6-16"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15667658" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Majoros</div> <div class="given-names">W.H.</div></div>,  <div class="name"><div class="surname">Pertea</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div>. </div><div class="article-title">TigrScan and GlimmerHMM: two open source ab initio eukaryotic gene-finders</div>, <div class="source">Bioinformatics</div> , <div class="year">2004</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">2878</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/bth315" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/bth315</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TigrScan%20and%20GlimmerHMM%3A%20two%20open%20source%20ab%20initio%20eukaryotic%20gene-finders&amp;author=W.H.%20Majoros&amp;author=M.%20Pertea&amp;author=S.L.%20Salzberg&amp;publication_year=2004&amp;journal=Bioinformatics&amp;volume=20&amp;pages=2878-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bth315" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbth315" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbth315"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15145805" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lomsadze</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Ter-Hovhannisyan</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Chernoff</div> <div class="given-names">Y.O.</div></div>,  <div class="name"><div class="surname">Borodovsky</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Gene identification in novel eukaryotic genomes by self-training algorithm</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">6494</div>-<div class="lpage">506</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gki937" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gki937</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20identification%20in%20novel%20eukaryotic%20genomes%20by%20self-training%20algorithm&amp;author=A.%20Lomsadze&amp;author=V.%20Ter-Hovhannisyan&amp;author=Y.O.%20Chernoff&amp;author=M.%20Borodovsky&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=6494-506" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki937" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki937" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki937"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16314312" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Usuka</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Brendel</div> <div class="given-names">V.</div></div>. </div><div class="article-title">Optimal spliced alignment of homologous cDNA to a genomic DNA template</div>, <div class="source">Bioinformatics</div> , <div class="year">2000</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">203</div>-<div class="lpage">11</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/16.3.203" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/16.3.203</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Optimal%20spliced%20alignment%20of%20homologous%20cDNA%20to%20a%20genomic%20DNA%20template&amp;author=J.%20Usuka&amp;author=W.%20Zhu&amp;author=V.%20Brendel&amp;publication_year=2000&amp;journal=Bioinformatics&amp;volume=16&amp;pages=203-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/16.3.203" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F16.3.203" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F16.3.203"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10869013" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lander</div> <div class="given-names">E.S.</div></div>,  <div class="name"><div class="surname">Waterman</div> <div class="given-names">M.S.</div></div>. </div><div class="article-title">Genomic mapping by fingerprinting random clones: a mathematical analysis</div>, <div class="source">Genomics</div> , <div class="year">1988</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">231</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/0888-7543(88)90007-9" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/0888-7543(88)90007-9</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20mapping%20by%20fingerprinting%20random%20clones%3A%20a%20mathematical%20analysis&amp;author=E.S.%20Lander&amp;author=M.S.%20Waterman&amp;publication_year=1988&amp;journal=Genomics&amp;volume=2&amp;pages=231-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0888-7543(88)90007-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0888-7543(88)90007-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0888-7543(88)90007-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3294162" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ017C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ017C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Salse</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bolot</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Throude</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Identification and characterization of shared duplications between rice and wheat provide new insight into grass genome evolution</div>, <div class="source">Plant Cell</div> , <div class="year">2008</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">11</div>-<div class="lpage">24</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1105/tpc.107.056309" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1105/tpc.107.056309</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20characterization%20of%20shared%20duplications%20between%20rice%20and%20wheat%20provide%20new%20insight%20into%20grass%20genome%20evolution&amp;author=J.%20Salse&amp;author=S.%20Bolot&amp;author=M.%20Throude&amp;publication_year=2008&amp;journal=Plant%20Cell&amp;volume=20&amp;pages=11-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.107.056309" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.107.056309" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.107.056309"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18178768" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Katsumi Isono</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2010. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=353434&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/5/10.1093/dnares/dsq017/2/dsq017-dsq017supp.pdf?Expires=2147483647&amp;Signature=yEp8RwohVSt3EDMJSYIJ1GEofwjGLE0jbM~Sms13PKX7m9XvFFA9K5aqaRwvoqJsOrOsBGhSD2B5C1YIVflDnWXSAXenflzxq~GOWWdxl8J0iuxUrzmkW-AZEkSGe3CHTBAcVT4NkavZF8cBXwbef5Cp0kHbBxZMwLmuLG2btju15-uij8mA98VKySI6SXdOHN9iuShxcl102DeHddfOHHULraBC9x188C-fL8YWrTTmNnnXu620oKQTjhyTTAtE7EUVhNWIf0dconuiIEk0RGLZ6TJRsmFvzz7e5zdG3cH8V62dm8R5TDW-6SDu4eZ2ST-ZqHFiwYiLsKD1nUSM-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">187</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000283631300002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">6</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsq017"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="353434"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000283631300002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f17%2f5%2f271%2f353434">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000283631300002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (6)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F17%2F5%2F271%2F353434" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/353434'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Efficient Plant Gene Identification Based on Interspecies Mapping of Full-Length cDNAs" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsq017");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f17%2f5%2f271%2f353434">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

