

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/18/3/189/445126","@type":"ScholarlyArticle","name":" In silico  Analysis of 3′-End-Processing Signals in  Aspergillus oryzae  Using Expressed Sequence Tags and Genomic Sequencing Data","datePublished":"2011-05-17","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/18/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2011-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsr011","keywords":["<italic>Aspergillus oryzae</italic>","3′-end processing signal","polyadenylation signal","3′-untranslated region (3′ UTR)"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/18/3/189/445126","author":[{"name":"Tanaka, Mizuki","affiliation":"1  Laboratory of Bioindustrial Genomics, Department of Bioindustrial Informatics and Genomics, Graduate School of Agricultural Science ,  Tohoku University ,  1-1 Tsutsumidori-Amamiyamachi, Aoba-ku, Sendai 981-8555 ,  Japan    2  Japan Society for the Promotion of Science, 8 Ichibancho, Chiyoda-ku, Tokyo 102-8472, Japan","@type":"Person"},{"name":"Sakai, Yoshifumi","affiliation":"3  Laboratory of Bioindustrial Informatics, Department of Bioindustrial Informatics and Genomics, Graduate School of Agricultural Science ,  Tohoku University ,  1-1 Tsutsumidori-Amamiyamachi, Aoba-ku, Sendai 981-8555 ,  Japan","@type":"Person"},{"name":"Yamada, Osamu","affiliation":"4  National Research Institute of Brewing ,  3-7-1 Kagamiyama, Higashi-hiroshima, Hiroshima 739-0046 ,  Japan","@type":"Person"},{"name":"Shintani, Takahiro","affiliation":"1  Laboratory of Bioindustrial Genomics, Department of Bioindustrial Informatics and Genomics, Graduate School of Agricultural Science ,  Tohoku University ,  1-1 Tsutsumidori-Amamiyamachi, Aoba-ku, Sendai 981-8555 ,  Japan","@type":"Person"},{"name":"Gomi, Katsuya","affiliation":"1  Laboratory of Bioindustrial Genomics, Department of Bioindustrial Informatics and Genomics, Graduate School of Agricultural Science ,  Tohoku University ,  1-1 Tsutsumidori-Amamiyamachi, Aoba-ku, Sendai 981-8555 ,  Japan","@type":"Person"}],"description":"Abstract.  To investigate 3′-end-processing signals in Aspergillus oryzae, we created a nucleotide sequence data set of the 3′-untranslated region (3′ UTR) plus","pageStart":"189","pageEnd":"200","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/3/1/m_cover.gif?Expires=2147483647&Signature=wj9vBfD3w2y4NVjjTT0hEq0DGILHhVJxl~hGkTYdv3f3RyfzKeaufbdZYgMTMKQvtZKo6x4TIuq0dMkBwN~dx6uKMb2t7UtWPZaFFD84b7m4O-CF7vPeyaegVwS-GVQTx9~LaL1HF9CGtMgbMthYT6eCwup0WynCZrD9syn~ryIjdMCj0Vfd6c~OnMKAAEkNIbY-5LT2dVafsguUGzIjP~lnGiuCdOABMm56~fJ~khgnHeAXFCc89K-eA167vdlnePhOZRt6~sb9AVRfnOlCiFtENgEc2O8AEAnKUMBucCj6xmHIeTCmvFwDpxMdx1ubEwugqbONa2xb-e6HSc9wlg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":" In silico  Analysis of 3′-End-Processing Signals in  Aspergillus oryzae  Using Expressed Sequence Tags and Genomic Sequencing Data","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/3/1/m_cover.gif?Expires=2147483647&Signature=wj9vBfD3w2y4NVjjTT0hEq0DGILHhVJxl~hGkTYdv3f3RyfzKeaufbdZYgMTMKQvtZKo6x4TIuq0dMkBwN~dx6uKMb2t7UtWPZaFFD84b7m4O-CF7vPeyaegVwS-GVQTx9~LaL1HF9CGtMgbMthYT6eCwup0WynCZrD9syn~ryIjdMCj0Vfd6c~OnMKAAEkNIbY-5LT2dVafsguUGzIjP~lnGiuCdOABMm56~fJ~khgnHeAXFCc89K-eA167vdlnePhOZRt6~sb9AVRfnOlCiFtENgEc2O8AEAnKUMBucCj6xmHIeTCmvFwDpxMdx1ubEwugqbONa2xb-e6HSc9wlg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content=" In silico  Analysis of 3′-End-Processing Signals in  Aspergillus oryzae  Using Expressed Sequence Tags and Genomic Sequencing Data" />
    <meta property="og:description" content="Abstract.  To investigate 3′-end-processing signals in Aspergillus oryzae, we created a nucleotide sequence data set of the 3′-untranslated region (3′ UTR) plus" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/18/3/189/445126" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2011-05-17" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Tanaka, Mizuki" /><meta name="citation_author_institution" content="Laboratory of Bioindustrial Genomics, Department of Bioindustrial Informatics and Genomics, Graduate School of Agricultural Science, Tohoku University, 1-1 Tsutsumidori-Amamiyamachi, Aoba-ku, Sendai 981-8555, Japan" /><meta name="citation_author_institution" content="Japan Society for the Promotion of Science, 8 Ichibancho, Chiyoda-ku, Tokyo 102-8472, Japan" /><meta name="citation_author" content="Sakai, Yoshifumi" /><meta name="citation_author_institution" content="Laboratory of Bioindustrial Informatics, Department of Bioindustrial Informatics and Genomics, Graduate School of Agricultural Science, Tohoku University, 1-1 Tsutsumidori-Amamiyamachi, Aoba-ku, Sendai 981-8555, Japan" /><meta name="citation_author" content="Yamada, Osamu" /><meta name="citation_author_institution" content="National Research Institute of Brewing, 3-7-1 Kagamiyama, Higashi-hiroshima, Hiroshima 739-0046, Japan" /><meta name="citation_author" content="Shintani, Takahiro" /><meta name="citation_author_institution" content="Laboratory of Bioindustrial Genomics, Department of Bioindustrial Informatics and Genomics, Graduate School of Agricultural Science, Tohoku University, 1-1 Tsutsumidori-Amamiyamachi, Aoba-ku, Sendai 981-8555, Japan" /><meta name="citation_author" content="Gomi, Katsuya" /><meta name="citation_author_institution" content="Laboratory of Bioindustrial Genomics, Department of Bioindustrial Informatics and Genomics, Graduate School of Agricultural Science, Tohoku University, 1-1 Tsutsumidori-Amamiyamachi, Aoba-ku, Sendai 981-8555, Japan" /><meta name="citation_title" content="In silico Analysis of 3′-End-Processing Signals in Aspergillus oryzae Using Expressed Sequence Tags and Genomic Sequencing Data" /><meta name="citation_firstpage" content="189" /><meta name="citation_lastpage" content="200" /><meta name="citation_doi" content="10.1093/dnares/dsr011" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="18" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2011/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Formation of mRNA 3′ ends in eukaryotes: mechanism, regulation, and interrelationships with other steps in mRNA synthesis; Microbiol. Mol. Biol. Rev.;  citation_year=1999;  citation_volume=63;  citation_pages=405-45; " /><meta name="citation_reference" content="citation_title=Protein factors in pre-mRNA 3′-end processing; Cell. Mol. Life Sci.;  citation_year=2008;  citation_volume=65;  citation_pages=1099-122; " /><meta name="citation_reference" content="citation_title=Molecular mechanisms of eukaryotic pre-mRNA 3′ end processing regulation; Nucleic Acids Res.;  citation_year=2010;  citation_volume=38;  citation_pages=2757-74; " /><meta name="citation_reference" content="citation_title=In slico detection of control signals: mRNA 3′-end-processing sequences in diverse species; Proc. Natl Acad. Sci. USA;  citation_year=1999;  citation_volume=96;  citation_pages=14055-60; " /><meta name="citation_reference" content="citation_title=Genomic detection of new yeast pre-mRNA 3′-end-processing signals; Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=888-94; " /><meta name="citation_reference" content="citation_title=Bioinformatic identification of candidate cis-regulatory elements involved in human mRNA polyadenylation; RNA;  citation_year=2005;  citation_volume=11;  citation_pages=1485-93; " /><meta name="citation_reference" content="citation_title=Complication of mRNA polyadenylation signals in Arabidopsis revealed a new signal element and potential secondary structures; Plant Physiol.;  citation_year=2005;  citation_volume=138;  citation_pages=1457-68; " /><meta name="citation_reference" content="citation_title=Genome level analysis of rice mRNA 3′-end processing signals and alternative polyadenylation; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=3150-61; " /><meta name="citation_reference" content="citation_title=Genomics of Aspergillus oryzae: learning from the history of koji mold and exploration of its future; DNA Res.;  citation_year=2008;  citation_volume=15;  citation_pages=173-83; " /><meta name="citation_reference" content="citation_title=High level expression of recombinant genes in Aspergillus oryzae; Bio/Technology;  citation_year=1988;  citation_volume=6;  citation_pages=1419-22; " /><meta name="citation_reference" content="citation_title=Developing Aspergillus as a host for heterologous expression; Biotechnol. Adv.;  citation_year=2009;  citation_volume=27;  citation_pages=53-75; " /><meta name="citation_reference" content="citation_title=Efficient production of secreted proteins by Aspergillus: progress, limitations and prospects; Appl. Microbiol. Biotechnol.;  citation_year=1997;  citation_volume=47;  citation_pages=1-11; " /><meta name="citation_reference" content="citation_title=Codon optimization increases steady-state mRNA levels in Aspergillus oryzae heterologous gene expression; Appl. Environ. Microbiol.;  citation_year=2008;  citation_volume=74;  citation_pages=6538-46; " /><meta name="citation_reference" content="citation_title=Analysis of expressed sequence tags from the fungus Aspergillus oryzae cultured under different conditions; DNA Res.;  citation_year=2007;  citation_volume=14;  citation_pages=47-57; " /><meta name="citation_reference" content="citation_title=Genome sequencing and analysis of Aspergillus oryzae; Nature;  citation_year=2005;  citation_volume=438;  citation_pages=1157-61; " /><meta name="citation_reference" content="citation_title=Unique features of nuclear mRNA poly(A) signals and alternative polyadenylation in Chlamydomonas reinhardtii; Genetics;  citation_year=2008;  citation_volume=179;  citation_pages=167-76; " /><meta name="citation_reference" content="citation_title=Analysis of RNA cleavage at the Adenovirus-2 L3 polyadenylation site; EMBO J.;  citation_year=1986;  citation_volume=5;  citation_pages=1929-38; " /><meta name="citation_reference" content="citation_title=Point mutations in AAUAAA and the poly(A) addition site: effects on the accuracy and efficiency of cleavage and polyadenylation in vitro; Nucleic Acids Res.;  citation_year=1990;  citation_volume=18;  citation_pages=5799-805; " /><meta name="citation_reference" content="citation_title=Cleavage site determinants in the mammalian polyadenylation signal; Nucleic Acids Res.;  citation_year=1995;  citation_volume=23;  citation_pages=2614-20; " /><meta name="citation_reference" content="citation_title=Transcriptional regulation of genes on the non-syntenic blocks of Aspergillus oryzae and its functional relationship to solid-state cultivation; Fungal Genet. Biol.;  citation_year=2008;  citation_volume=45;  citation_pages=139-51; " /><meta name="citation_reference" content="citation_title=enoLOGOS: a versatile web tool for energy normalized sequence logos; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=W389-92; " /><meta name="citation_reference" content="citation_title=Statistical analysis of yeast genomic downstream sequences reveals putative polyadenylation signals; Nucleic Acids Res.;  citation_year=2000;  citation_volume=28;  citation_pages=1000-10; " /><meta name="citation_reference" content="citation_title=The COG database: an updated version includes eukaryotes; BMC Bioinformatics;  citation_year=2003;  citation_volume=4;  citation_pages=41-54; " /><meta name="citation_reference" content="citation_title=A large-scale analysis of mRNA polyadenylation of human and mouse genes; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=201-12; " /><meta name="citation_reference" content="citation_title=Alternative poly(A) site selection in complex transcription units: means to an end?; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=2547-61; " /><meta name="citation_reference" content="citation_title=Structural and functional features of eukaryotic mRNA untranslated regions; Gene;  citation_year=2001;  citation_volume=276;  citation_pages=73-81; " /><meta name="citation_reference" content="citation_title=The power of the 3′ UTR: translational control and development; Nat. Rev. Genet.;  citation_year=2003;  citation_volume=4;  citation_pages=626-37; " /><meta name="citation_reference" content="citation_title=Translational control by the 3′-UTR: the ends specify the means; Trends Biochem. Sci.;  citation_year=2003;  citation_volume=28;  citation_pages=91-8; " /><meta name="citation_reference" content="citation_title=How strong is the case for regulation of the initiation step of translation by elements at the 3′ end of eukaryotic mRNAs?; Gene;  citation_year=2004;  citation_volume=343;  citation_pages=41-54; " /><meta name="citation_reference" content="citation_title=MicroRNAs: target recognition and regulatory functions; Cell;  citation_year=2009;  citation_volume=136;  citation_pages=215-233; " /><meta name="citation_reference" content="citation_title=Long 3′-UTRs target wild-type mRNAs for nonsense-mediated mRNA decay in Saccharomyces cerevisiae; Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=2771-8; " /><meta name="citation_reference" content="citation_title=A defined sequence within the 3′ UTR of the areA transcript is sufficient to mediate nitrogen metabolite signaling via accelerated deadenylation; Mol. Microbiol.;  citation_year=2000;  citation_volume=37;  citation_pages=1248-57; " /><meta name="citation_reference" content="citation_title=Opposing signals differentially regulate transcript stability in Aspergillus nidulans; Mol. Microbiol.;  citation_year=2006;  citation_volume=62;  citation_pages=509-19; " /><meta name="citation_reference" content="citation_title=A PUF family portrait: 3′ UTR regulation as a way of life; Trends Genet.;  citation_year=2002;  citation_volume=18;  citation_pages=150-7; " /><meta name="citation_reference" content="citation_title=AU-rich elements and associated factors: are there unifying principles?; Nucleic Acids Res.;  citation_year=2006;  citation_volume=33;  citation_pages=7138-50; " /><meta name="citation_reference" content="citation_title=Regulated ARE-mediated mRNA decay in Saccharomyces cerevisiae; Mol. Cell;  citation_year=2001;  citation_volume=7;  citation_pages=1191-200; " /><meta name="citation_reference" content="citation_title=Coordinated remodeling of cellular metabolism during iron deficiency through targeted mRNA degradation; Cell;  citation_year=2005;  citation_volume=120;  citation_pages=99-110; " /><meta name="citation_reference" content="citation_title=Recruitment of the Puf3 protein to its mRNA target for regulation of mRNA decay in yeast; RNA;  citation_year=2004;  citation_volume=10;  citation_pages=1625-36; " /><meta name="citation_reference" content="citation_title=An exploration of 3′-end processing signals and their tissue distribution in Oryza sativa; Gene;  citation_year=2007;  citation_volume=389;  citation_pages=107-13; " /><meta name="citation_reference" content="citation_title=Role of the conserved AAUAAA sequence: for AAUAAA point mutations prevent messenger RNA 3′ end formation; Science;  citation_year=1984;  citation_volume=226;  citation_pages=1045-51; " /><meta name="citation_reference" content="citation_title=Redundant 3′ end-forming signals for the yeast CYC1 mRNA; Proc. Natl. Acad. Sci. USA;  citation_year=1995;  citation_volume=92;  citation_pages=4211-14; " /><meta name="citation_reference" content="citation_title=Probabilistic prediction of Saccharomyces cerevisiae mRNA 3'-processing sites; Nucleic Acids Res.;  citation_year=2002;  citation_volume=30;  citation_pages=1851-58; " /><meta name="citation_reference" content="citation_title=Analysis of a noncanonical poly(A) site reveals a tripartite mechanism for vertebrate poly(A) site recognition; Genes Dev.;  citation_year=2005;  citation_volume=19;  citation_pages=1315-27; " /><meta name="citation_reference" content="citation_title=Organization and function of APT, a subcomplex of the yeast cleavage and polyadenylation factor involved in the formation of mRNA and small nucleolar RNA 3'-ends; J. Biol. Chem.;  citation_year=2003;  citation_volume=278;  citation_pages=33000-10; " /><meta name="citation_reference" content="citation_title=Hrp1, a sequence-specific RNA-binding protein that shuttles between the nucleus and the cytoplasm, is required for mRNA 3′-end formation in yeast; Genes Dev.;  citation_year=1997;  citation_volume=11;  citation_pages=2545-56; " /><meta name="citation_reference" content="citation_title=Grabbing the message: structural basis of mRNA 3′ UTR recognition by Hrp1; EMBO J.;  citation_year=2006;  citation_volume=25;  citation_pages=3167-78; " /><meta name="citation_reference" content="citation_title=Yhh1p/Cft1p directly links poly(A) site recognition and RNA polymerase II transcription termination; EMBO J.;  citation_year=2002;  citation_volume=21;  citation_pages=4125-35; " /><meta name="citation_reference" content="citation_title=Arabidopsis mRNA polyadenylation machinery: comprehensive analysis of protein-protein interactions and gene expression profiling; BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=220" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/18/3/189/1217792/dsr011.pdf" /><meta name="description" content="Abstract.  To investigate 3′-end-processing signals in Aspergillus oryzae, we created a nucleotide sequence data set of the 3′-untranslated region (3′ UTR) plus" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/18/3/189/445126" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/18/3/189/445126" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/18/3/189/445126");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsr011';
    </script>
<title>
	In silico Analysis of 3′-End-Processing Signals in Aspergillus oryzae Using Expressed Sequence Tags and Genomic Sequencing Data | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./445126" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BcVtPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYW5ha2EsIE1penVraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgQmlvaW5kdXN0cmlhbCBHZW5vbWljcywgRGVwYXJ0bWVudCBvZiBCaW9pbmR1c3RyaWFsIEluZm9ybWF0aWNzIGFuZCBHZW5vbWljcywgR3JhZHVhdGUgU2Nob29sIG9mIEFncmljdWx0dXJhbCBTY2llbmNlLCBUb2hva3UgVW5pdmVyc2l0eSwgMS0xIFRzdXRzdW1pZG9yaS1BbWFtaXlhbWFjaGksIEFvYmEta3UsIFNlbmRhaSA5ODEtODU1NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJKYXBhbiBTb2NpZXR5IGZvciB0aGUgUHJvbW90aW9uIG9mIFNjaWVuY2UsIDggSWNoaWJhbmNobywgQ2hpeW9kYS1rdSwgVG9reW8gMTAyLTg0NzIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2FrYWksIFlvc2hpZnVtaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgQmlvaW5kdXN0cmlhbCBJbmZvcm1hdGljcywgRGVwYXJ0bWVudCBvZiBCaW9pbmR1c3RyaWFsIEluZm9ybWF0aWNzIGFuZCBHZW5vbWljcywgR3JhZHVhdGUgU2Nob29sIG9mIEFncmljdWx0dXJhbCBTY2llbmNlLCBUb2hva3UgVW5pdmVyc2l0eSwgMS0xIFRzdXRzdW1pZG9yaS1BbWFtaXlhbWFjaGksIEFvYmEta3UsIFNlbmRhaSA5ODEtODU1NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJZYW1hZGEsIE9zYW11IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgUmVzZWFyY2ggSW5zdGl0dXRlIG9mIEJyZXdpbmcsIDMtNy0xIEthZ2FtaXlhbWEsIEhpZ2FzaGktaGlyb3NoaW1hLCBIaXJvc2hpbWEgNzM5LTAwNDYsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2hpbnRhbmksIFRha2FoaXJvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTGFib3JhdG9yeSBvZiBCaW9pbmR1c3RyaWFsIEdlbm9taWNzLCBEZXBhcnRtZW50IG9mIEJpb2luZHVzdHJpYWwgSW5mb3JtYXRpY3MgYW5kIEdlbm9taWNzLCBHcmFkdWF0ZSBTY2hvb2wgb2YgQWdyaWN1bHR1cmFsIFNjaWVuY2UsIFRvaG9rdSBVbml2ZXJzaXR5LCAxLTEgVHN1dHN1bWlkb3JpLUFtYW1peWFtYWNoaSwgQW9iYS1rdSwgU2VuZGFpIDk4MS04NTU1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkdvbWksIEthdHN1eWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJMYWJvcmF0b3J5IG9mIEJpb2luZHVzdHJpYWwgR2Vub21pY3MsIERlcGFydG1lbnQgb2YgQmlvaW5kdXN0cmlhbCBJbmZvcm1hdGljcyBhbmQgR2Vub21pY3MsIEdyYWR1YXRlIFNjaG9vbCBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZSwgVG9ob2t1IFVuaXZlcnNpdHksIDEtMSBUc3V0c3VtaWRvcmktQW1hbWl5YW1hY2hpLCBBb2JhLWt1LCBTZW5kYWkgOTgxLTg1NTUsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJJbiBzaWxpY28gQW5hbHlzaXMgb2YgM+KAsi1FbmQtUHJvY2Vzc2luZyBTaWduYWxzIGluIEFzcGVyZ2lsbHVzIG9yeXphZSBVc2luZyBFeHByZXNzZWQgU2VxdWVuY2UgVGFncyBhbmQgR2Vub21pYyBTZXF1ZW5jaW5nIERhdGEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIxODkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjIwMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzcjAxMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDExLzA2LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Rm9ybWF0aW9uIG9mIG1STkEgM+KAsiBlbmRzIGluIGV1a2FyeW90ZXM6IG1lY2hhbmlzbSwgcmVndWxhdGlvbiwgYW5kIGludGVycmVsYXRpb25zaGlwcyB3aXRoIG90aGVyIHN0ZXBzIGluIG1STkEgc3ludGhlc2lzOyBNaWNyb2Jpb2wuIE1vbC4gQmlvbC4gUmV2LjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT02MzsgIGNpdGF0aW9uX3BhZ2VzPTQwNS00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvdGVpbiBmYWN0b3JzIGluIHByZS1tUk5BIDPigLItZW5kIHByb2Nlc3Npbmc7IENlbGwuIE1vbC4gTGlmZSBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTY1OyAgY2l0YXRpb25fcGFnZXM9MTA5OS0xMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBtZWNoYW5pc21zIG9mIGV1a2FyeW90aWMgcHJlLW1STkEgM+KAsiBlbmQgcHJvY2Vzc2luZyByZWd1bGF0aW9uOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz0yNzU3LTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbiBzbGljbyBkZXRlY3Rpb24gb2YgY29udHJvbCBzaWduYWxzOiBtUk5BIDPigLItZW5kLXByb2Nlc3Npbmcgc2VxdWVuY2VzIGluIGRpdmVyc2Ugc3BlY2llczsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT05NjsgIGNpdGF0aW9uX3BhZ2VzPTE0MDU1LTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljIGRldGVjdGlvbiBvZiBuZXcgeWVhc3QgcHJlLW1STkEgM+KAsi1lbmQtcHJvY2Vzc2luZyBzaWduYWxzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz04ODgtOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJpb2luZm9ybWF0aWMgaWRlbnRpZmljYXRpb24gb2YgY2FuZGlkYXRlIGNpcy1yZWd1bGF0b3J5IGVsZW1lbnRzIGludm9sdmVkIGluIGh1bWFuIG1STkEgcG9seWFkZW55bGF0aW9uOyBSTkE7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xNDg1LTkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGljYXRpb24gb2YgbVJOQSBwb2x5YWRlbnlsYXRpb24gc2lnbmFscyBpbiBBcmFiaWRvcHNpcyByZXZlYWxlZCBhIG5ldyBzaWduYWwgZWxlbWVudCBhbmQgcG90ZW50aWFsIHNlY29uZGFyeSBzdHJ1Y3R1cmVzOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMzg7ICBjaXRhdGlvbl9wYWdlcz0xNDU3LTY4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgbGV2ZWwgYW5hbHlzaXMgb2YgcmljZSBtUk5BIDPigLItZW5kIHByb2Nlc3Npbmcgc2lnbmFscyBhbmQgYWx0ZXJuYXRpdmUgcG9seWFkZW55bGF0aW9uOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz0zMTUwLTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljcyBvZiBBc3BlcmdpbGx1cyBvcnl6YWU6IGxlYXJuaW5nIGZyb20gdGhlIGhpc3Rvcnkgb2Yga29qaSBtb2xkIGFuZCBleHBsb3JhdGlvbiBvZiBpdHMgZnV0dXJlOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTE3My04MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaCBsZXZlbCBleHByZXNzaW9uIG9mIHJlY29tYmluYW50IGdlbmVzIGluIEFzcGVyZ2lsbHVzIG9yeXphZTsgQmlvL1RlY2hub2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTE5ODg7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTE0MTktMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BpbmcgQXNwZXJnaWxsdXMgYXMgYSBob3N0IGZvciBoZXRlcm9sb2dvdXMgZXhwcmVzc2lvbjsgQmlvdGVjaG5vbC4gQWR2LjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTUzLTc1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FZmZpY2llbnQgcHJvZHVjdGlvbiBvZiBzZWNyZXRlZCBwcm90ZWlucyBieSBBc3BlcmdpbGx1czogcHJvZ3Jlc3MsIGxpbWl0YXRpb25zIGFuZCBwcm9zcGVjdHM7IEFwcGwuIE1pY3JvYmlvbC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9NDc7ICBjaXRhdGlvbl9wYWdlcz0xLTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiBvcHRpbWl6YXRpb24gaW5jcmVhc2VzIHN0ZWFkeS1zdGF0ZSBtUk5BIGxldmVscyBpbiBBc3BlcmdpbGx1cyBvcnl6YWUgaGV0ZXJvbG9nb3VzIGdlbmUgZXhwcmVzc2lvbjsgQXBwbC4gRW52aXJvbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT03NDsgIGNpdGF0aW9uX3BhZ2VzPTY1MzgtNDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIGV4cHJlc3NlZCBzZXF1ZW5jZSB0YWdzIGZyb20gdGhlIGZ1bmd1cyBBc3BlcmdpbGx1cyBvcnl6YWUgY3VsdHVyZWQgdW5kZXIgZGlmZmVyZW50IGNvbmRpdGlvbnM7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9NDctNTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jaW5nIGFuZCBhbmFseXNpcyBvZiBBc3BlcmdpbGx1cyBvcnl6YWU7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00Mzg7ICBjaXRhdGlvbl9wYWdlcz0xMTU3LTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VbmlxdWUgZmVhdHVyZXMgb2YgbnVjbGVhciBtUk5BIHBvbHkoQSkgc2lnbmFscyBhbmQgYWx0ZXJuYXRpdmUgcG9seWFkZW55bGF0aW9uIGluIENobGFteWRvbW9uYXMgcmVpbmhhcmR0aWk7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE3OTsgIGNpdGF0aW9uX3BhZ2VzPTE2Ny03NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgUk5BIGNsZWF2YWdlIGF0IHRoZSBBZGVub3ZpcnVzLTIgTDMgcG9seWFkZW55bGF0aW9uIHNpdGU7IEVNQk8gSi47ICBjaXRhdGlvbl95ZWFyPTE5ODY7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTE5MjktMzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvaW50IG11dGF0aW9ucyBpbiBBQVVBQUEgYW5kIHRoZSBwb2x5KEEpIGFkZGl0aW9uIHNpdGU6IGVmZmVjdHMgb24gdGhlIGFjY3VyYWN5IGFuZCBlZmZpY2llbmN5IG9mIGNsZWF2YWdlIGFuZCBwb2x5YWRlbnlsYXRpb24gaW4gdml0cm87IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTU3OTktODA1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbGVhdmFnZSBzaXRlIGRldGVybWluYW50cyBpbiB0aGUgbWFtbWFsaWFuIHBvbHlhZGVueWxhdGlvbiBzaWduYWw7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTI2MTQtMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRpb25hbCByZWd1bGF0aW9uIG9mIGdlbmVzIG9uIHRoZSBub24tc3ludGVuaWMgYmxvY2tzIG9mIEFzcGVyZ2lsbHVzIG9yeXphZSBhbmQgaXRzIGZ1bmN0aW9uYWwgcmVsYXRpb25zaGlwIHRvIHNvbGlkLXN0YXRlIGN1bHRpdmF0aW9uOyBGdW5nYWwgR2VuZXQuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ1OyAgY2l0YXRpb25fcGFnZXM9MTM5LTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1lbm9MT0dPUzogYSB2ZXJzYXRpbGUgd2ViIHRvb2wgZm9yIGVuZXJneSBub3JtYWxpemVkIHNlcXVlbmNlIGxvZ29zOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz1XMzg5LTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdGF0aXN0aWNhbCBhbmFseXNpcyBvZiB5ZWFzdCBnZW5vbWljIGRvd25zdHJlYW0gc2VxdWVuY2VzIHJldmVhbHMgcHV0YXRpdmUgcG9seWFkZW55bGF0aW9uIHNpZ25hbHM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTEwMDAtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBDT0cgZGF0YWJhc2U6IGFuIHVwZGF0ZWQgdmVyc2lvbiBpbmNsdWRlcyBldWthcnlvdGVzOyBCTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTQxLTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGxhcmdlLXNjYWxlIGFuYWx5c2lzIG9mIG1STkEgcG9seWFkZW55bGF0aW9uIG9mIGh1bWFuIGFuZCBtb3VzZSBnZW5lczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9MjAxLTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbHRlcm5hdGl2ZSBwb2x5KEEpIHNpdGUgc2VsZWN0aW9uIGluIGNvbXBsZXggdHJhbnNjcmlwdGlvbiB1bml0czogbWVhbnMgdG8gYW4gZW5kPzsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MjU0Ny02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RydWN0dXJhbCBhbmQgZnVuY3Rpb25hbCBmZWF0dXJlcyBvZiBldWthcnlvdGljIG1STkEgdW50cmFuc2xhdGVkIHJlZ2lvbnM7IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9Mjc2OyAgY2l0YXRpb25fcGFnZXM9NzMtODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBwb3dlciBvZiB0aGUgM+KAsiBVVFI6IHRyYW5zbGF0aW9uYWwgY29udHJvbCBhbmQgZGV2ZWxvcG1lbnQ7IE5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTYyNi0zNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNsYXRpb25hbCBjb250cm9sIGJ5IHRoZSAz4oCyLVVUUjogdGhlIGVuZHMgc3BlY2lmeSB0aGUgbWVhbnM7IFRyZW5kcyBCaW9jaGVtLiBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9OTEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SG93IHN0cm9uZyBpcyB0aGUgY2FzZSBmb3IgcmVndWxhdGlvbiBvZiB0aGUgaW5pdGlhdGlvbiBzdGVwIG9mIHRyYW5zbGF0aW9uIGJ5IGVsZW1lbnRzIGF0IHRoZSAz4oCyIGVuZCBvZiBldWthcnlvdGljIG1STkFzPzsgR2VuZTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zNDM7ICBjaXRhdGlvbl9wYWdlcz00MS01NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9STkFzOiB0YXJnZXQgcmVjb2duaXRpb24gYW5kIHJlZ3VsYXRvcnkgZnVuY3Rpb25zOyBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEzNjsgIGNpdGF0aW9uX3BhZ2VzPTIxNS0yMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxvbmcgM+KAsi1VVFJzIHRhcmdldCB3aWxkLXR5cGUgbVJOQXMgZm9yIG5vbnNlbnNlLW1lZGlhdGVkIG1STkEgZGVjYXkgaW4gU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz0yNzcxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZGVmaW5lZCBzZXF1ZW5jZSB3aXRoaW4gdGhlIDPigLIgVVRSIG9mIHRoZSBhcmVBIHRyYW5zY3JpcHQgaXMgc3VmZmljaWVudCB0byBtZWRpYXRlIG5pdHJvZ2VuIG1ldGFib2xpdGUgc2lnbmFsaW5nIHZpYSBhY2NlbGVyYXRlZCBkZWFkZW55bGF0aW9uOyBNb2wuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz0xMjQ4LTU3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcHBvc2luZyBzaWduYWxzIGRpZmZlcmVudGlhbGx5IHJlZ3VsYXRlIHRyYW5zY3JpcHQgc3RhYmlsaXR5IGluIEFzcGVyZ2lsbHVzIG5pZHVsYW5zOyBNb2wuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NjI7ICBjaXRhdGlvbl9wYWdlcz01MDktMTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgUFVGIGZhbWlseSBwb3J0cmFpdDogM+KAsiBVVFIgcmVndWxhdGlvbiBhcyBhIHdheSBvZiBsaWZlOyBUcmVuZHMgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MTUwLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFVLXJpY2ggZWxlbWVudHMgYW5kIGFzc29jaWF0ZWQgZmFjdG9yczogYXJlIHRoZXJlIHVuaWZ5aW5nIHByaW5jaXBsZXM/OyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz03MTM4LTUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWd1bGF0ZWQgQVJFLW1lZGlhdGVkIG1STkEgZGVjYXkgaW4gU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlOyBNb2wuIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTExOTEtMjAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db29yZGluYXRlZCByZW1vZGVsaW5nIG9mIGNlbGx1bGFyIG1ldGFib2xpc20gZHVyaW5nIGlyb24gZGVmaWNpZW5jeSB0aHJvdWdoIHRhcmdldGVkIG1STkEgZGVncmFkYXRpb247IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTIwOyAgY2l0YXRpb25fcGFnZXM9OTktMTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWNydWl0bWVudCBvZiB0aGUgUHVmMyBwcm90ZWluIHRvIGl0cyBtUk5BIHRhcmdldCBmb3IgcmVndWxhdGlvbiBvZiBtUk5BIGRlY2F5IGluIHllYXN0OyBSTkE7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0xNjI1LTM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBleHBsb3JhdGlvbiBvZiAz4oCyLWVuZCBwcm9jZXNzaW5nIHNpZ25hbHMgYW5kIHRoZWlyIHRpc3N1ZSBkaXN0cmlidXRpb24gaW4gT3J5emEgc2F0aXZhOyBHZW5lOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM4OTsgIGNpdGF0aW9uX3BhZ2VzPTEwNy0xMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Um9sZSBvZiB0aGUgY29uc2VydmVkIEFBVUFBQSBzZXF1ZW5jZTogZm9yIEFBVUFBQSBwb2ludCBtdXRhdGlvbnMgcHJldmVudCBtZXNzZW5nZXIgUk5BIDPigLIgZW5kIGZvcm1hdGlvbjsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MTk4NDsgIGNpdGF0aW9uX3ZvbHVtZT0yMjY7ICBjaXRhdGlvbl9wYWdlcz0xMDQ1LTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWR1bmRhbnQgM+KAsiBlbmQtZm9ybWluZyBzaWduYWxzIGZvciB0aGUgeWVhc3QgQ1lDMSBtUk5BOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT05MjsgIGNpdGF0aW9uX3BhZ2VzPTQyMTEtMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb2JhYmlsaXN0aWMgcHJlZGljdGlvbiBvZiBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWUgbVJOQSAzJy1wcm9jZXNzaW5nIHNpdGVzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0xODUxLTU4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiBhIG5vbmNhbm9uaWNhbCBwb2x5KEEpIHNpdGUgcmV2ZWFscyBhIHRyaXBhcnRpdGUgbWVjaGFuaXNtIGZvciB2ZXJ0ZWJyYXRlIHBvbHkoQSkgc2l0ZSByZWNvZ25pdGlvbjsgR2VuZXMgRGV2LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTEzMTUtMjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9yZ2FuaXphdGlvbiBhbmQgZnVuY3Rpb24gb2YgQVBULCBhIHN1YmNvbXBsZXggb2YgdGhlIHllYXN0IGNsZWF2YWdlIGFuZCBwb2x5YWRlbnlsYXRpb24gZmFjdG9yIGludm9sdmVkIGluIHRoZSBmb3JtYXRpb24gb2YgbVJOQSBhbmQgc21hbGwgbnVjbGVvbGFyIFJOQSAzJy1lbmRzOyBKLiBCaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0yNzg7ICBjaXRhdGlvbl9wYWdlcz0zMzAwMC0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SHJwMSwgYSBzZXF1ZW5jZS1zcGVjaWZpYyBSTkEtYmluZGluZyBwcm90ZWluIHRoYXQgc2h1dHRsZXMgYmV0d2VlbiB0aGUgbnVjbGV1cyBhbmQgdGhlIGN5dG9wbGFzbSwgaXMgcmVxdWlyZWQgZm9yIG1STkEgM+KAsi1lbmQgZm9ybWF0aW9uIGluIHllYXN0OyBHZW5lcyBEZXYuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MjU0NS01NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R3JhYmJpbmcgdGhlIG1lc3NhZ2U6IHN0cnVjdHVyYWwgYmFzaXMgb2YgbVJOQSAz4oCyIFVUUiByZWNvZ25pdGlvbiBieSBIcnAxOyBFTUJPIEouOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MzE2Ny03ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9WWhoMXAvQ2Z0MXAgZGlyZWN0bHkgbGlua3MgcG9seShBKSBzaXRlIHJlY29nbml0aW9uIGFuZCBSTkEgcG9seW1lcmFzZSBJSSB0cmFuc2NyaXB0aW9uIHRlcm1pbmF0aW9uOyBFTUJPIEouOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9NDEyNS0zNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXJhYmlkb3BzaXMgbVJOQSBwb2x5YWRlbnlsYXRpb24gbWFjaGluZXJ5OiBjb21wcmVoZW5zaXZlIGFuYWx5c2lzIG9mIHByb3RlaW4tcHJvdGVpbiBpbnRlcmFjdGlvbnMgYW5kIGdlbmUgZXhwcmVzc2lvbiBwcm9maWxpbmc7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MjIwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE4LzMvMTg5LzEyMTc3OTIvZHNyMDExLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBUbyBpbnZlc3RpZ2F0ZSAz4oCyLWVuZC1wcm9jZXNzaW5nIHNpZ25hbHMgaW4gQXNwZXJnaWxsdXMgb3J5emFlLCB3ZSBjcmVhdGVkIGEgbnVjbGVvdGlkZSBzZXF1ZW5jZSBkYXRhIHNldCBvZiB0aGUgM+KAsi11bnRyYW5zbGF0ZWQgcmVnaW9uICgz4oCyIFVUUikgcGx1cyIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDQ1MTI2JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZGVPvNbjZ/yElkM4PmK3cTWTUV1V" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f3%2f189%2f445126" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f3%2f189%2f445126" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=17382" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/18/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/3/1/m_cover.gif?Expires=2147483647&amp;Signature=wj9vBfD3w2y4NVjjTT0hEq0DGILHhVJxl~hGkTYdv3f3RyfzKeaufbdZYgMTMKQvtZKo6x4TIuq0dMkBwN~dx6uKMb2t7UtWPZaFFD84b7m4O-CF7vPeyaegVwS-GVQTx9~LaL1HF9CGtMgbMthYT6eCwup0WynCZrD9syn~ryIjdMCj0Vfd6c~OnMKAAEkNIbY-5LT2dVafsguUGzIjP~lnGiuCdOABMm56~fJ~khgnHeAXFCc89K-eA167vdlnePhOZRt6~sb9AVRfnOlCiFtENgEc2O8AEAnKUMBucCj6xmHIeTCmvFwDpxMdx1ubEwugqbONa2xb-e6HSc9wlg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/18/3">
                        <div class="volume trailing-comma">Volume 18</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2011        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6654919">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6654919">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6654921">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6654921">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6654925">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6654925">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6654937">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6654937">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6654967">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6654967">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6654969">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6654969">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6654973">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6654973">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6654977">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6654977">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/18/3/177/444849">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main"><em>In silico</em> Analysis of 3′-End-Processing Signals in <em>Aspergillus oryzae</em> Using Expressed Sequence Tags and Genomic Sequencing Data
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mizuki Tanaka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Laboratory of Bioindustrial Genomics, Department of Bioindustrial Informatics and Genomics, Graduate School of Agricultural Science</div>, <div class="insititution">Tohoku University</div>, <div class="addr-line">1-1 Tsutsumidori-Amamiyamachi, Aoba-ku, Sendai 981-8555</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">Japan Society for the Promotion of Science, 8 Ichibancho, Chiyoda-ku, Tokyo 102-8472, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mizuki+Tanaka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tanaka M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tanaka M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mizuki Tanaka</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshifumi Sakai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Laboratory of Bioindustrial Informatics, Department of Bioindustrial Informatics and Genomics, Graduate School of Agricultural Science</div>, <div class="insititution">Tohoku University</div>, <div class="addr-line">1-1 Tsutsumidori-Amamiyamachi, Aoba-ku, Sendai 981-8555</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshifumi+Sakai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sakai Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sakai Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yoshifumi Sakai</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Osamu Yamada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">National Research Institute of Brewing</div>, <div class="addr-line">3-7-1 Kagamiyama, Higashi-hiroshima, Hiroshima 739-0046</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Osamu+Yamada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamada O">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yamada O&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Osamu Yamada</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takahiro Shintani</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Laboratory of Bioindustrial Genomics, Department of Bioindustrial Informatics and Genomics, Graduate School of Agricultural Science</div>, <div class="insititution">Tohoku University</div>, <div class="addr-line">1-1 Tsutsumidori-Amamiyamachi, Aoba-ku, Sendai 981-8555</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takahiro+Shintani">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shintani T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shintani T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takahiro Shintani</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Katsuya Gomi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Laboratory of Bioindustrial Genomics, Department of Bioindustrial Informatics and Genomics, Graduate School of Agricultural Science</div>, <div class="insititution">Tohoku University</div>, <div class="addr-line">1-1 Tsutsumidori-Amamiyamachi, Aoba-ku, Sendai 981-8555</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 22-717-8901. Fax. +81 22-717-8902. E-mail: <a href="mailto:gomi@biochem.tohoku.ac.jp" target="_blank">gomi@biochem.tohoku.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Katsuya+Gomi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gomi K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gomi K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Katsuya Gomi</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by: Katsumi Isono</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 18, Issue 3, 1 June 2011, Pages 189–200, <a href='https://doi.org/10.1093/dnares/dsr011'>https://doi.org/10.1093/dnares/dsr011</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">17 May 2011</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">16 November 2010</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">17 April 2011</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="445126" href="/dnaresearch/article-pdf/18/3/189/1217792/dsr011.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Mizuki Tanaka, Yoshifumi Sakai, Osamu Yamada, Takahiro Shintani, Katsuya Gomi;  <em>In silico</em> Analysis of 3′-End-Processing Signals in <em>Aspergillus oryzae</em> Using Expressed Sequence Tags and Genomic Sequencing Data, <em>DNA Research</em>, Volume 18, Issue 3, 1 June 2011, Pages 189–200, <a href='https://doi.org/10.1093/dnares/dsr011'>https://doi.org/10.1093/dnares/dsr011</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/445126?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/445126?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/445126?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/445126?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/445126?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/445126?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/445126?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=%26lt%3Bem%26gt%3BIn%20silico%26lt%3B%2Fem%26gt%3B%20Analysis%20of%203%E2%80%B2-End-Processing%20Signals%20in%20%26lt%3Bem%26gt%3BAspergillus%20oryzae%26lt%3B%2Fem%26gt%3B%20Using%20Expressed%20Sequence%20Tags%20and%20Genomic%20Sequencing%20Data&amp;publicationDate=2011-05-17&amp;volumeNum=18&amp;issueNum=3&amp;author=Tanaka%2C%20Mizuki%3B%20Sakai%2C%20Yoshifumi&amp;startPage=189&amp;endPage=200&amp;contentId=10.1093%2Fdnares%2Fdsr011&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f3%2f189%2f445126" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6654919 id="6654919" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>To investigate 3′-end-processing signals in <em>Aspergillus oryzae</em>, we created a nucleotide sequence data set of the 3′-untranslated region (3′ UTR) plus 100 nucleotides (nt) sequence downstream of the poly(A) site using <em>A. oryzae</em> expressed sequence tags and genomic sequencing data. This data set comprised 1065 sequences derived from 1042 unique genes. The average 3′ UTR length in <em>A. oryzae</em> was 241 nt, which is greater than that in yeast but similar to that in plants. The 3′ UTR and 100 nt sequence downstream of the poly(A) site is notably U-rich, while the region located 15–30 nt upstream of the poly(A) site is markedly A-rich. The most frequently found hexanucleotide in this A-rich region is AAUGAA, although this sequence accounts for only 6% of all transcripts. These data suggested that <em>A. oryzae</em> has no highly conserved sequence element equivalent to AAUAAA, a mammalian polyadenylation signal. We identified that putative 3′-end-processing signals in <em>A. oryzae</em>, while less well conserved than those in mammals, comprised four sequence elements: the furthest upstream U-rich element, A-rich sequence, cleavage site, and downstream U-rich element flanking the cleavage site. Although these putative 3′-end-processing signals are similar to those in yeast and plants, some notable differences exist between them.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Aspergillus oryzae&quot;"><em>Aspergillus oryzae</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;3′-end processing signal&quot;">3′-end processing signal</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;polyadenylation signal&quot;">polyadenylation signal</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;3′-untranslated region (3′ UTR)&quot;">3′-untranslated region (3′ UTR)</a></div>                            <h2 scrollto-destination=6654921 id="6654921" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>In eukaryotes, most mRNAs have a poly(A) tail at their 3′ end. 3′-end-processing of eukaryotic pre-mRNA involves endonucleolytic cleavage and polyadenylation.<sup><a reveal-id="DSR011C1  DSR011C3" data-open="DSR011C1  DSR011C3" class="link link-ref link-reveal xref-bibr">1–3</a></sup> The 3′-end cleavage and polyadenylation site are regulated by several sequence elements, which have been extensively studied in mammalian, yeast, and plant cells.<sup><a reveal-id="DSR011C4  DSR011C8" data-open="DSR011C4  DSR011C8" class="link link-ref link-reveal xref-bibr">4–8</a></sup> In mammals, three elements are known as primary sequence elements: the polyadenylation signal, cleavage site, and downstream U/GU-rich elements. In addition, two auxiliary sequence elements (upstream U-rich elements and downstream G-rich elements) have also been identified. Among these elements, the polyadenylation signal, which is the hexanucleotide AAUAAA or its variant AUUAAA, located 10–35 nucleotides (nt) upstream of the poly(A) site is the most highly conserved sequence. In yeast and plants, A-rich sequence elements also exist ∼10–30 nt upstream of the cleavage site, but these elements are less well conserved compared with mammalian polyadenylation signals. Among many sequences identified as A-rich sequences, AAUAAA is the most well-conserved sequence in both yeast and plants. In addition to A-rich sequence elements, further upstream elements, designated efficiency elements in yeast or far upstream elements in plants, the cleavage site, and the downstream U-rich element flanking the cleavage site have been described.</p><p>In Japan, the filamentous fungus <em>Aspergillus oryzae</em> has long been used for the production of traditional fermented foods, such as sake, soy sauce, and miso (soybean paste), and its long history of use in the food industry is a testament to its safety.<sup><a reveal-id="DSR011C9" data-open="DSR011C9" class="link link-ref link-reveal xref-bibr">9</a></sup> In addition, <em>A. oryzae</em> has the ability to secrete large amounts of protein, and therefore, it has recently gained recognition as a favourable host organism for recombinant protein production.<sup><a reveal-id="DSR011C10" data-open="DSR011C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSR011C11" data-open="DSR011C11" class="link link-ref link-reveal xref-bibr">11</a></sup> However, secretion yields of heterologous proteins from <em>A. oryzae</em> are low compared with those of homologous proteins or proteins from closely related fungal species.<sup><a reveal-id="DSR011C12" data-open="DSR011C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Recently, we revealed that the transcript of a heterologous gene containing the AT-biased codon was prematurely polyadenylated within the coding region of <em>A. oryzae</em>.<sup><a reveal-id="DSR011C13" data-open="DSR011C13" class="link link-ref link-reveal xref-bibr">13</a></sup> This premature polyadenylation was prevented by the alteration of its codon to better suit <em>Aspergillus</em> codon usage. This result suggested that cryptic 3′-end-processing signals are recognized by <em>A. oryzae</em> within the coding region of the heterologous gene and that these signals are eliminated by codon optimization. However, no experimental data exist on 3′-end-processing signals in filamentous fungi, including <em>A. oryzae</em>.</p><p>To elucidate 3′-end-processing signals in <em>A. oryzae</em>, we created a nucleotide sequence data set of the 3′-untranslated region (3′ UTR) and 100 nt downstream of the poly(A) site using <em>A. oryzae</em> expressed sequence tags (ESTs) and genomic sequencing data. Using this data set, we identified several putative 3′-end-processing signals in <em>A. oryzae</em>. To our knowledge, this is the first report of the identification of 3′-end-processing signals in filamentous fungi.</p>                            <h2 scrollto-destination=6654925 id="6654925" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=6654926 id="6654926" class="section-title" data-legacy-id=s2a>2.1. Creation of the A. oryzae poly(A) data set</h3>
<p>A total of 21 446 EST sequences in the <em>A. oryzae</em> EST database (<a class="link link-uri" href="http://nribf2.nrib.go.jp/EST2/index.html">http://nribf2.nrib.go.jp/EST2/index.html</a>), created by sequencing from the 5′ end of the cDNA insert,<sup><a reveal-id="DSR011C14" data-open="DSR011C14" class="link link-ref link-reveal xref-bibr">14</a></sup> were searched for sequences that contained at least eight consecutive A residues, yielding 1647 EST sequence entries. Subsequently, EST sequences containing oligo(A) stretches inherently present in the genome were eliminated by comparison with genomic DNA sequences.<sup><a reveal-id="DSR011C15" data-open="DSR011C15" class="link link-ref link-reveal xref-bibr">15</a></sup> In addition, to eliminate mis-annotated genes, only EST sequences in which the poly(A) site was located within 1000 nt downstream of the stop codon were selected. EST sequences with poly(A) sites located within the coding region, probably caused by internal priming, were also eliminated in this manner. Nine pairs of redundant EST sequences with identical poly(A) sites were considered to be derived from a single cDNA and were removed from the data set. Finally, 1065 EST sequences were selected by these processes. Genomic DNA-based sequences within the 3′ UTR and 100 nt sequence downstream of the poly(A) site were extracted for the poly(A) site data set (T residues were converted to U residues). This data set contained 22 pairs of EST sequences derived from the same gene with different poly(A) sites. Therefore, this data set comprised only sequences derived from 1043 unique genes.</p><p>The poly(A) site was designated as the last nucleotide in the genome sequence preceding the poly(A) tail. When an adenine residue was found at the poly(A) site in the genome sequence, this adenine was termed the poly(A) site nucleotide according to the recent reports of 3′-end-processing signals in plants and <em>Chlamydomonas</em> on the basis of EST sequencing data,<sup><a reveal-id="DSR011C8" data-open="DSR011C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSR011C16" data-open="DSR011C16" class="link link-ref link-reveal xref-bibr">16</a></sup> indicating that the first adenine of a poly(A) tail tended to be transcribed from the genomic DNA.<sup><a reveal-id="DSR011C17  DSR011C19" data-open="DSR011C17  DSR011C19" class="link link-ref link-reveal xref-bibr">17–19</a></sup></p>                            <h3 scrollto-destination=6654929 id="6654929" class="section-title" data-legacy-id=s2b>2.2. DNA microarray analysis</h3>
<p>The <em>A. oryzae</em> wild-type strain RIB40, which was used for genome sequencing analysis,<sup><a reveal-id="DSR011C15" data-open="DSR011C15" class="link link-ref link-reveal xref-bibr">15</a></sup> was grown in sterilized wheat bran media (3.0 g wheat bran with 1.8 ml distilled water) at 30°C for 33 h. Total RNA extraction, mRNA preparation, and DNA microarray analysis were performed according to the methods of Tamano <em>et al</em>.<sup><a reveal-id="DSR011C20" data-open="DSR011C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Purified Cy3- or Cy5-labelled cDNA probes were hybridized using 12 K <em>A. oryzae</em> oligonucleotide microarrays (Fermlab, Tokyo, Japan). After global normalization, the relative fluorescence intensity of each gene was normalized to that of the <em>histone H4</em> gene, which was used as a reference. We selected 5384 genes whose intensities were found to be reproducible and reliable (<em>P</em> &lt; 0.1) in dye-swap experiments.</p>                            <h3 scrollto-destination=6654931 id="6654931" class="section-title" data-legacy-id=s2c>2.3. Generation of sequence logos</h3>
<p>Sequence logos of around the poly(A) site were generated using the enoLOGOS web tool.<sup><a reveal-id="DSR011C21" data-open="DSR011C21" class="link link-ref link-reveal xref-bibr">21</a></sup></p>                            <h3 scrollto-destination=6654933 id="6654933" class="section-title" data-legacy-id=s2d>2.4. Analysis of oligonucleotide frequencies</h3>
<div class="block-child-p">A standard score (<em>Z</em>-score) was used to detect the most over-represented hexanucleotide sequences from −30 to −15 nt (region II), according to the zeroth- and first-order Markov chain models.<sup><a reveal-id="DSR011C22" data-open="DSR011C22" class="link link-ref link-reveal xref-bibr">22</a></sup> The <em>Z</em>-score of a hexanucleotide sequence (w = <em>x</em><sub>1</sub><em>x</em><sub>2</sub><em>x</em><sub>3</sub><em>x</em><sub>4</sub><em>x</em><sub>5</sub><em>x</em><sub>6</sub>, where <em>x</em> is the nucleotide sequence) was calculated as follows:<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr011/2/dsr011ueq2.gif?Expires=2147483647&Signature=AVBk8vQA6BnL339-GNZPuhojyLUlZBf3ygWrHvZFVjdUhfTwJteiswIzEJO~TVgELz9~GP98pGcbt~Ilinz55irZ~hsr7wu9EAGHAHY1kRwiD3pZvParw-SOXifUT3HCWrVGtVAQh0U~Mqdr4grhPYn-dfkCz4gPhMPOVt7W2wjRsDjRVWyD--CaWboPgwnN9GZjctRj2IC6wLDVtTk-1nNkSgLHgqAs7m8zg4ESrOtFnPRCfpJgGtV00DHTfC6pmEAB5qhOVfPpv4P8QwMderUr7Rk7ZRJshwUDGBodePFOc-ONpoCJVr88VZNmdhD4pmuXr~PxLO25aKGMXCZ9sg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsr011ueq2.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr011/2/m_dsr011ueq2.gif?Expires=2147483647&Signature=hv8VqXeZhygf5wCjo~gbW918POnNn2Ie-O~3lJRaSPwnBS0OBIKojVzbLLLO7y0D5mP60s-2KK7LDzr9dwqeasPyXVxkfLfipOZPqIFq6WplgzCCJb4z~X8L2LmhaHIGdqr~1n8Ldjt2OcHxwzdWqF2blIfUyg1yhTn6HJgeR8xEt-GlwNwoh~~7nu6w~lkKSNelfzZUc7YgZRSGCoroF3WLQrpe45W~UFzx3TCVBeOieCp6GPTM9JDlmHsIIEzEYCiLebqWlfTPOYqypXcUElxTU9aNPTTeIow2ss5nlh-xPkxUJ8hJp61-bitBFnFhD5hKW3prk~kwrM0jyONzBQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsr011ueq2.gif" /></a></div></div></div> In this definition, <em>f</em><sub>obs</sub>(<em>w</em>) denotes the observed frequency of <em>w</em>, i.e. the number of occurrences of <em>w</em> in <em>s</em> divided by the number of occurrences of sequences having the same length as <em>w</em> in <em>s</em>, where <em>s</em> ranges over all sequences of length 6 located in region II; <em>f</em><sub>exp</sub>(w) denotes the expected frequency of w, determined as the value <em>f</em><sub>obs</sub>(<em>x</em><sub>1</sub>) × <em>f</em><sub>obs</sub>(<em>x</em><sub>2</sub>) × <em>f</em><sub>obs</sub>(<em>x</em><sub>3</sub>) × <em>f</em><sub>obs</sub>(<em>x</em><sub>4</sub>) × <em>f</em><sub>obs</sub>(<em>x</em><sub>5</sub>) × <em>f</em><sub>obs</sub>(<em>x</em><sub>6</sub>) in the zeroth-order model or <em>f</em><sub>obs</sub>(<em>x</em><sub>1</sub><em>x</em><sub>2</sub>) × <em>f</em><sub>obs</sub>(<em>x</em><sub>2</sub><em>x</em><sub>3</sub>) × <em>f</em><sub>obs</sub>(<em>x</em><sub>3</sub><em>x</em><sub>4</sub>) × <em>f</em><sub>obs</sub>(<em>x</em><sub>4</sub><em>x</em><sub>5</sub>) × <em>f</em><sub>obs</sub>(<em>x</em><sub>5</sub><em>x</em><sub>6</sub>)/(<em>f</em><sub>obs</sub>(<em>x</em><sub>2</sub>) × <em>f</em><sub>obs</sub>(<em>x</em><sub>3</sub>) × <em>f</em><sub>obs</sub>(<em>x</em><sub>4</sub>) × <em>f</em><sub>obs</sub>(<em>x</em><sub>5</sub>) in the first-order model; and <em>n</em> denotes the number of sequences of length 6 located in region II.</div>                            <h3 scrollto-destination=6654935 id="6654935" class="section-title" data-legacy-id=s2e>2.5. Search for protein factors involved in pre-mRNA 3′-end-processing in A. oryzae</h3>
<p>Homologs of protein factors involved in eukaryotic pre-mRNA 3′-end-processing were retrieved by searching the <em>A. oryzae</em> genome database (<a class="link link-uri" href="http://www.bio.nite.go.jp/dogan/project/view/AO">http://www.bio.nite.go.jp/dogan/project/view/AO</a>, <a class="link link-uri" href="http://nribf2.nrib.go.jp/">http://nribf2.nrib.go.jp/</a>) using the BlastP program.</p>                            <h2 scrollto-destination=6654937 id="6654937" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=6654938 id="6654938" class="section-title" data-legacy-id=s3a>3.1. Profile of the A. oryzae poly(A) data set</h3>
<p>We obtained 1065 sequences for the <em>A. oryzae</em> poly(A) data set from the EST database, as described in the Materials and methods section. First, 1043 unique genes contained in the <em>A. oryzae</em> poly(A) data set were classified into functional categories known as eukaryotic orthologous groups,<sup><a reveal-id="DSR011C23" data-open="DSR011C23" class="link link-ref link-reveal xref-bibr">23</a></sup> according to the gene list in the <em>A. oryzae</em> genome database. Compared with the classification of all genes found in the genome database, the number of genes classified into the Unannotated category was markedly lower in the poly(A) data set [43% in the genome database vs. 30% in the poly(A) data set]. In contrast, the number of genes classified into Information storage and processing and Cellular processes and signalling categories was higher in the poly(A) data set [7 and 12% in the genome database vs. 14 and 20% in the poly(A) data set, respectively]. The number of genes classified into Metabolism and Poorly characterized categories was similar between the genome database and poly(A) data set. These results indicated that the poly(A) data set covers a wide range of genes classified into diverse functional categories despite the poly(A) data set comprising only 1043 unique genes of the 12 074 genes predicted in the <em>A. oryzae</em> genome database.</p><p>In contrast, because EST sequences were accumulated by single-pass sequencing of the 5′ end of the cDNA insert,<sup><a reveal-id="DSR011C14" data-open="DSR011C14" class="link link-ref link-reveal xref-bibr">14</a></sup> the poly(A) data set could cover &lt;10% of the total genes, and thus, the poly(A) data set might show some bias towards highly expressed genes. To assess this possibility, we compared the frequency distributions of EST contigs that corresponded to each of the 1043 genes in the poly(A) data set with those of the total 7589 contigs in the EST database<sup><a reveal-id="DSR011C14" data-open="DSR011C14" class="link link-ref link-reveal xref-bibr">14</a></sup> (Fig. <a reveal-id="DSR011F1" data-open="DSR011F1" class="link link-reveal link-table xref-fig">1</a>A). Whereas contigs with frequencies of &gt;6 accounted for ∼10% of the total EST contigs, contigs with corresponding frequencies accounted for ∼25% of the poly(A) data set. However, singletons accounted for 40% of the poly(A) data set. In addition, we examined the expression levels of genes in the poly(A) data set by DNA microarray analysis (Fig. <a reveal-id="DSR011F1" data-open="DSR011F1" class="link link-reveal link-table xref-fig">1</a>B). Of the total of 5384 genes selected by microarray analysis, the number of relatively highly expressed genes (expression ratio &gt; 0.1) accounted for approximately 7%, whereas it accounted for 20% of the 618 genes of the poly(A) data set. The remaining genes (80%) were expressed at low levels. These results suggested that the poly(A) data set was somewhat biased towards highly expressed genes, but this fact enabled the identification of 3′-end-processing signals. </p>                        <a id="6654941" scrollto-destination="6654941"></a>
<div id="DSR011F1" data-id="DSR011F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr011/2/m_dsr01101.jpeg?Expires=2147483647&Signature=e4-18xrHM3cDKi0XPqBtNgWpRC6YGBCMucrdrflIISEk-aA~IpI47tAHdVpBak51SKrdDj6VFawwHFEJsY91aDvagOlihZ3s5xv4SVYlKw4CAuEcDyBGCzfHnESEVCryG7vgsFp-MmK2RIetrIn8oan8w5KL2VlUh92Eq61PGcbb5ErXPAz2ZUeBjbJ~hcBMN1E5u5Z~Bm-TTqGg0nJVsVbMTZqwupKNUR9iwvWF0aa1I9fdR1wRWkkXaVPB7fJuA1lkfSPi2GL9gsQZeBG4lUY3CJ2aVQSjcs7falm81sJ1m~uM5lXe4uaXwIL40d5gSMQ0nRrk0xLFhx3QDk9pIQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Profile of the A. oryzae poly(A) data set. (A) Frequency distribution of EST contigs based on the EST copy number. The EST copy number of each contig contained in the A. oryzae poly(A) data set was obtained from the A. oryzae EST database (http://nribf2.nrib.go.jp/EST2/index.html). Data on the total EST contigs were obtained from the study by Akao et al.14 (B) Gene expression levels determined by DNA microarray analysis. The fluorescence intensity of each gene was normalized to that of the histone H4 gene." path-from-xml="dsr01101.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6654941/dsr01101.jpeg" path-from-xml="dsr01101.jpeg" target="_blank">View large</a><a section="6654941" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr011/2/dsr01101.jpeg?Expires=2147483647&Signature=5In0a34uLGHDVs-jDy0BbEyJKVxMOp8kODppDbocsDhxz7hY~VJH7D7ozmJrk3ddKjNcteJEYVm~pM-UPDArTRjtqfuvLkx3dkKgPqeiZe32ZOfOCR9-rLGDCTfar6IE6L3H11cI7-b~J~n7r7a4SyWYTO4EGbOD~Zeabkgd-z3VryLioDd9bcp76~3fGvPAzPdlpNJY06P7z89qMefrF6O5qWscJ6RKSRfSx8Sld8MeLgb0Nlc20krQQt3Rmg8mK7bG-HsqVmL3alIoS1ycCC5VcNEC52ILNmgQU~ESTeumAYZz-vPErD59ocXIZboJut~zU6T-K34LUtqXH62vUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6654941&ar=445126&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01101.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Profile of the <em>A. oryzae</em> poly(A) data set. (<strong>A</strong>) Frequency distribution of EST contigs based on the EST copy number. The EST copy number of each contig contained in the <em>A. oryzae</em> poly(A) data set was obtained from the <em>A. oryzae</em> EST database (<a class="link link-uri" href="http://nribf2.nrib.go.jp/EST2/index.html">http://nribf2.nrib.go.jp/EST2/index.html</a>). Data on the total EST contigs were obtained from the study by Akao <em>et al</em>.<sup><a reveal-id="DSR011C14" data-open="DSR011C14" class="link link-ref link-reveal xref-bibr">14</a></sup> (<strong>B</strong>) Gene expression levels determined by DNA microarray analysis. The fluorescence intensity of each gene was normalized to that of the <em>histone H4</em> gene.</p></div></div><div content-id="DSR011F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr011/2/m_dsr01101.jpeg?Expires=2147483647&Signature=e4-18xrHM3cDKi0XPqBtNgWpRC6YGBCMucrdrflIISEk-aA~IpI47tAHdVpBak51SKrdDj6VFawwHFEJsY91aDvagOlihZ3s5xv4SVYlKw4CAuEcDyBGCzfHnESEVCryG7vgsFp-MmK2RIetrIn8oan8w5KL2VlUh92Eq61PGcbb5ErXPAz2ZUeBjbJ~hcBMN1E5u5Z~Bm-TTqGg0nJVsVbMTZqwupKNUR9iwvWF0aa1I9fdR1wRWkkXaVPB7fJuA1lkfSPi2GL9gsQZeBG4lUY3CJ2aVQSjcs7falm81sJ1m~uM5lXe4uaXwIL40d5gSMQ0nRrk0xLFhx3QDk9pIQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Profile of the A. oryzae poly(A) data set. (A) Frequency distribution of EST contigs based on the EST copy number. The EST copy number of each contig contained in the A. oryzae poly(A) data set was obtained from the A. oryzae EST database (http://nribf2.nrib.go.jp/EST2/index.html). Data on the total EST contigs were obtained from the study by Akao et al.14 (B) Gene expression levels determined by DNA microarray analysis. The fluorescence intensity of each gene was normalized to that of the histone H4 gene." path-from-xml="dsr01101.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6654941/dsr01101.jpeg" path-from-xml="dsr01101.jpeg" target="_blank">View large</a><a section="6654941" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr011/2/dsr01101.jpeg?Expires=2147483647&Signature=5In0a34uLGHDVs-jDy0BbEyJKVxMOp8kODppDbocsDhxz7hY~VJH7D7ozmJrk3ddKjNcteJEYVm~pM-UPDArTRjtqfuvLkx3dkKgPqeiZe32ZOfOCR9-rLGDCTfar6IE6L3H11cI7-b~J~n7r7a4SyWYTO4EGbOD~Zeabkgd-z3VryLioDd9bcp76~3fGvPAzPdlpNJY06P7z89qMefrF6O5qWscJ6RKSRfSx8Sld8MeLgb0Nlc20krQQt3Rmg8mK7bG-HsqVmL3alIoS1ycCC5VcNEC52ILNmgQU~ESTeumAYZz-vPErD59ocXIZboJut~zU6T-K34LUtqXH62vUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6654941&ar=445126&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01101.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Profile of the <em>A. oryzae</em> poly(A) data set. (<strong>A</strong>) Frequency distribution of EST contigs based on the EST copy number. The EST copy number of each contig contained in the <em>A. oryzae</em> poly(A) data set was obtained from the <em>A. oryzae</em> EST database (<a class="link link-uri" href="http://nribf2.nrib.go.jp/EST2/index.html">http://nribf2.nrib.go.jp/EST2/index.html</a>). Data on the total EST contigs were obtained from the study by Akao <em>et al</em>.<sup><a reveal-id="DSR011C14" data-open="DSR011C14" class="link link-ref link-reveal xref-bibr">14</a></sup> (<strong>B</strong>) Gene expression levels determined by DNA microarray analysis. The fluorescence intensity of each gene was normalized to that of the <em>histone H4</em> gene.</p></div></div><p>In eukaryotes, many genes including &gt;50% of human and rice genes have multiple polyadenylation sites.<sup><a reveal-id="DSR011C8" data-open="DSR011C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSR011C24" data-open="DSR011C24" class="link link-ref link-reveal xref-bibr">24</a></sup> This alternative polyadenylation has been recognized as an important mechanism for gene expression regulation.<sup><a reveal-id="DSR011C25" data-open="DSR011C25" class="link link-ref link-reveal xref-bibr">25</a></sup> However, no study has investigated on alternative polyadenylation on the basis of bioinformatics analyses in filamentous fungi. Although only 22 pairs of duplicated EST sequences with alternative poly(A) sites were included in the poly(A) data set, 14 pairs of these sequences had distant poly(A) sites located at least 30 nt apart (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). This result suggested that alternative polyadenylation also generally occurs in filamentous fungi.</p>                            <h3 scrollto-destination=6654943 id="6654943" class="section-title" data-legacy-id=s3b>3.2. Analysis of 3′ UTR length and sequence elements of 3′ UTR-binding proteins</h3>
<p>In eukaryotes, 3′ UTR regulates mRNA stability and translational efficiency through sequence elements for 3′ UTR-binding proteins and microRNAs or through its length.<sup><a reveal-id="DSR011C26  DSR011C31" data-open="DSR011C26  DSR011C31" class="link link-ref link-reveal xref-bibr">26–31</a></sup> In <em>A. nidulans</em>, stability of transcripts involved in nitrogen metabolism was dependent on their 3′ UTRs.<sup><a reveal-id="DSR011C32" data-open="DSR011C32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="DSR011C33" data-open="DSR011C33" class="link link-ref link-reveal xref-bibr">33</a></sup> Therefore, the 3′ UTRs may play an important role in gene expression regulation in filamentous fungi. However, no comprehensive information exists about 3′ UTRs in filamentous fungi. Hence, we analysed the distribution of 3′ UTR lengths in <em>A. oryzae</em> and determined their average and median lengths to compare with those in yeast and plants, which were also determined by analysis of EST sequencing data. In <em>A. oryzae</em>, 3′ UTR lengths were predominantly distributed in the range of 51 to 350 nt (Fig. <a reveal-id="DSR011F2" data-open="DSR011F2" class="link link-reveal link-table xref-fig">2</a>). The average 3′ UTR length in <em>A. oryzae</em> was 241 nt, while the median 3′ UTR length was 203 nt. The average 3′ UTR length in <em>Saccharomyces cerevisiae</em> is 144 nt (median 3′ UTR length is 121 nt)<sup><a reveal-id="DSR011C5" data-open="DSR011C5" class="link link-ref link-reveal xref-bibr">5</a></sup> and that in plants is 289 nt (<em>Oryza sativa</em>) and 223 nt (<em>Arabidopsis thaliana</em>).<sup><a reveal-id="DSR011C8" data-open="DSR011C8" class="link link-ref link-reveal xref-bibr">8</a></sup> These results suggested that 3′ UTR length in <em>A. oryzae</em> is greater than that in yeast but similar to that in plants. </p>                        <a id="6654945" scrollto-destination="6654945"></a>
<div id="DSR011F2" data-id="DSR011F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr011/2/m_dsr01102.jpeg?Expires=2147483647&Signature=LOmsH7orFEYYO~eHcOhHXLwGPZLU9iLXbNvkgGDqjT2vzOXb-H1iV5OXhpnyPvWqeyUG-gnsD3AhyMs6NVTSYkQSHn2UcXjtmL0mxDoq~iWAfVolTEGSg7QyF5KF9MeXvW~eUKhzoA4MtwrTh8dxtK4Zkx1863DZVtVYmnTGiE2qioZawQuZnjg34Zy87M4-1n4H32qgpt8lZOQLu6uI3wCsiDBhJ-kd9mkrDbaUMk52VHJL-1OvwiD6MU75Ha~df4k187ia7YG4BQMDbsj1GzgVhGROmLAkJvKXscDMr0sBiA-OSXAbMqBGTvsYGYeefg4Cm-ci6n0geRvCM3H7xw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of 3′ UTR lengths determined for 1065 unique EST sequences. The average length is 241 nt." path-from-xml="dsr01102.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6654945/dsr01102.jpeg" path-from-xml="dsr01102.jpeg" target="_blank">View large</a><a section="6654945" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr011/2/dsr01102.jpeg?Expires=2147483647&Signature=FrkAnDZzFZFex5TDgNRKC2iGqKll3oer-rI2CuhzJ72IljspAcfQjAnxlxa1nA~RiJXMkNQkB5NDOPO6crkTC9J-bN8qpN4RpVvg~YfIAzSPVJMxlvARPQdM64mojmh~xyFaVcunkxh5UlybgO-kzkI00enaWdxLlpPwu~IfnaOqDnk2YHh6GgcHvhDxXc~NHQvue71mGP~2TEEQSYutjVP5Lg5pWWKYTPZn2CtSlZvDJN2yfMWmezTqn988BtrbLVyHEsRCpBEagnsi7MmN1W5DEX6pE2GXn2KHC8ftO9xjSHG9P0mDdlkY4oRflVfm6h1KRzSxzQKosXxGQEqjsQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6654945&ar=445126&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01102.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of 3′ UTR lengths determined for 1065 unique EST sequences. The average length is 241 nt.</p></div></div><div content-id="DSR011F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr011/2/m_dsr01102.jpeg?Expires=2147483647&Signature=LOmsH7orFEYYO~eHcOhHXLwGPZLU9iLXbNvkgGDqjT2vzOXb-H1iV5OXhpnyPvWqeyUG-gnsD3AhyMs6NVTSYkQSHn2UcXjtmL0mxDoq~iWAfVolTEGSg7QyF5KF9MeXvW~eUKhzoA4MtwrTh8dxtK4Zkx1863DZVtVYmnTGiE2qioZawQuZnjg34Zy87M4-1n4H32qgpt8lZOQLu6uI3wCsiDBhJ-kd9mkrDbaUMk52VHJL-1OvwiD6MU75Ha~df4k187ia7YG4BQMDbsj1GzgVhGROmLAkJvKXscDMr0sBiA-OSXAbMqBGTvsYGYeefg4Cm-ci6n0geRvCM3H7xw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of 3′ UTR lengths determined for 1065 unique EST sequences. The average length is 241 nt." path-from-xml="dsr01102.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6654945/dsr01102.jpeg" path-from-xml="dsr01102.jpeg" target="_blank">View large</a><a section="6654945" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr011/2/dsr01102.jpeg?Expires=2147483647&Signature=FrkAnDZzFZFex5TDgNRKC2iGqKll3oer-rI2CuhzJ72IljspAcfQjAnxlxa1nA~RiJXMkNQkB5NDOPO6crkTC9J-bN8qpN4RpVvg~YfIAzSPVJMxlvARPQdM64mojmh~xyFaVcunkxh5UlybgO-kzkI00enaWdxLlpPwu~IfnaOqDnk2YHh6GgcHvhDxXc~NHQvue71mGP~2TEEQSYutjVP5Lg5pWWKYTPZn2CtSlZvDJN2yfMWmezTqn988BtrbLVyHEsRCpBEagnsi7MmN1W5DEX6pE2GXn2KHC8ftO9xjSHG9P0mDdlkY4oRflVfm6h1KRzSxzQKosXxGQEqjsQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6654945&ar=445126&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01102.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of 3′ UTR lengths determined for 1065 unique EST sequences. The average length is 241 nt.</p></div></div><p>The most well-known sequence elements for 3′ UTR-binding proteins in eukaryotes are AU-rich elements (AREs) and the PUF consensus motif.<sup><a reveal-id="DSR011C34" data-open="DSR011C34" class="link link-ref link-reveal xref-bibr">34</a>,<a reveal-id="DSR011C35" data-open="DSR011C35" class="link link-ref link-reveal xref-bibr">35</a></sup> We searched for transcripts containing the yeast putative AREs (UAUUUAUU and UUAUUUAU) and PUF consensus motif (UGUANAUA) within the <em>A. oryzae</em> 3′ UTR.<sup><a reveal-id="DSR011C36  DSR011C38" data-open="DSR011C36  DSR011C38" class="link link-ref link-reveal xref-bibr">36–38</a></sup> In the poly(A) data set, 12 and 23 genes possessed AREs and the PUF consensus motif within the 3′ UTR, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). One gene (AO090011000041) particularly exhibited overlapping AUUUA sequences (AUUUAUUUA), a typical ARE motif. In addition, we found orthologs of the yeast ARE-binding protein (Pub1) and four of six yeast Puf family proteins (Puf1, Puf3, Puf4, and Puf6) in the <em>A. oryzae</em> genome (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). These results suggested the existence of a regulation system for gene expression that utilizes 3′ UTR-binding proteins in filamentous fungi.</p>                            <h3 scrollto-destination=6654947 id="6654947" class="section-title" data-legacy-id=s3c>3.3. Nucleotide profile of the <em>A. oryzae</em> 3′ UTR</h3>
<p>To determine 3′ end processing elements in <em>A. oryzae</em>, we first measured the single nucleotide frequencies for all positions within the 3′ UTR and 100 nt sequence downstream of the poly(A) site (set at position 0). As shown in Fig. <a reveal-id="DSR011F3" data-open="DSR011F3" class="link link-reveal link-table xref-fig">3</a>A, this region was notably U-rich, while AU accounted for 62% of nucleotides in this region (U = 34%; A = 28%). Meanwhile, AU content of the coding region in <em>A. oryzae</em> was 48% (<a class="link link-uri" href="http://www.kazusa.or.jp/codon/">http://www.kazusa.or.jp/codon/</a>), suggesting that a high AU content is characteristic of this region. The 3′ UTR was markedly U-rich, but a A-rich region was observed upstream of the poly(A) site—particularly, the −29 to −14 nt region had a high A content with &gt;30%. In addition, a high U content was also observed in the +1 to +20 nt region immediately downstream of the poly(A) site, but A and U content in the downstream +20 to +100 nt region was almost equal. This AU-rich element (ARE) located in the region immediately downstream of the U-rich region flanking the poly(A) site was also found in yeast and plants, but it has not been defined as the 3′-end-processing element in those organisms.<sup><a reveal-id="DSR011C4" data-open="DSR011C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSR011C7" data-open="DSR011C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSR011C8" data-open="DSR011C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Moreover, the poly(A) site (position 0) had an extremely high A content (78%), and as described in the Materials and methods section, the first adenine of the poly(A) tail was designated as the poly(A) site nucleotide. High C nucleotide usage was observed at position −1 immediately before the poly(A) site compared with other positions (position −3, 20%; position −2, 21%; position −1, 37%; and position 0, 7%; Fig. <a reveal-id="DSR011F3" data-open="DSR011F3" class="link link-reveal link-table xref-fig">3</a>B). The content of pyrimidine nucleotides (C and U) at position −1 was 68%, suggesting that CA or UA dinucleotides form the optimal cleavage site in <em>A. oryzae</em>, similar to that observed in plants. </p>                        <a id="6654949" scrollto-destination="6654949"></a>
<div id="DSR011F3" data-id="DSR011F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr011/2/m_dsr01103.jpeg?Expires=2147483647&Signature=qbj0DqNaEuUPyP3ZYTy1ScYkROtiGQbCqFVfvhA2Mhu6nZsqODYIQTju0~V2MOwXGSt6S0TSevDn9usWN50mzCdQ0j-GRf6Y3BMKjcOmxcDaIvC3DuO3TLHL4bQi1eDXVm9Z0FWUc9VPpjgOCjTkPGXkfaIzmmI9AL0e7C21bbauDgOEv-J2MvWWHK7bBew107HDGocm5DOKEqqqYiNQ1WRioJprD4LhdxmAfvW5BdIQXaG8atNxaDKKRvUOlHrIUDqg1J6SvbDQSulRQ~w~85syPxxl0~34T3ZNXQFUKhF34VBy-PEd27HeFozrXp24F267xjdBSAqTPcU1JUGy1g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Single nucleotide frequencies in the 3′ UTR and 100 nt sequence downstream of the poly(A) site. (A) Single nucleotide profile in the 3′ UTR and 100 nt sequence downstream of the poly(A) site. The poly(A) site is at position 0. The upstream sequence of the poly(A) site is designated minus and the downstream sequence is designated plus. (B) Sequence logo generated from the actual frequency of occurrence of each of the four nucleotides around the cleavage site. (C) Six regions of the 3′ UTR and 100 nt sequence downstream of the poly(A) site formed according to the single nucleotide profile. The cleavage and polyadenylation site is located between regions IV and V." path-from-xml="dsr01103.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6654949/dsr01103.jpeg" path-from-xml="dsr01103.jpeg" target="_blank">View large</a><a section="6654949" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr011/2/dsr01103.jpeg?Expires=2147483647&Signature=vXMDjwmYnM~CEJdvpJAUuG~5Gxgw3YGycttyoLbxToWkxZCzt3supK4nYWjujX5QSmcHqQ442LiVpjpHip9NFYKlZkp6lohDS4P9rsRe0gIvdOhx4ILWtA-5H-fZ3kAp-eXsAIrBayqk~zhv6mMC~ubUJW1AX1Cu56A0OLBnKCGWfufMM6oaJOA9bzBjntFUBWUdZBnWBMdCUcDo0NCC-iazfyC7uTbo1W~6LTSSKN3GmN2Gt~fLFR0VB~1mgz7rmNOgY5ZPy~VvwLgYZlEQaxyVTryX7DJSw0PRJzrgedAolVrqRfabnhsom6WS8ETGDLO6YQTaVCsNF~qqixgxwA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6654949&ar=445126&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01103.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Single nucleotide frequencies in the 3′ UTR and 100 nt sequence downstream of the poly(A) site. (<strong>A</strong>) Single nucleotide profile in the 3′ UTR and 100 nt sequence downstream of the poly(A) site. The poly(A) site is at position 0. The upstream sequence of the poly(A) site is designated minus and the downstream sequence is designated plus. (<strong>B</strong>) Sequence logo generated from the actual frequency of occurrence of each of the four nucleotides around the cleavage site. (<strong>C</strong>) Six regions of the 3′ UTR and 100 nt sequence downstream of the poly(A) site formed according to the single nucleotide profile. The cleavage and polyadenylation site is located between regions IV and V.</p></div></div><div content-id="DSR011F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr011/2/m_dsr01103.jpeg?Expires=2147483647&Signature=qbj0DqNaEuUPyP3ZYTy1ScYkROtiGQbCqFVfvhA2Mhu6nZsqODYIQTju0~V2MOwXGSt6S0TSevDn9usWN50mzCdQ0j-GRf6Y3BMKjcOmxcDaIvC3DuO3TLHL4bQi1eDXVm9Z0FWUc9VPpjgOCjTkPGXkfaIzmmI9AL0e7C21bbauDgOEv-J2MvWWHK7bBew107HDGocm5DOKEqqqYiNQ1WRioJprD4LhdxmAfvW5BdIQXaG8atNxaDKKRvUOlHrIUDqg1J6SvbDQSulRQ~w~85syPxxl0~34T3ZNXQFUKhF34VBy-PEd27HeFozrXp24F267xjdBSAqTPcU1JUGy1g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Single nucleotide frequencies in the 3′ UTR and 100 nt sequence downstream of the poly(A) site. (A) Single nucleotide profile in the 3′ UTR and 100 nt sequence downstream of the poly(A) site. The poly(A) site is at position 0. The upstream sequence of the poly(A) site is designated minus and the downstream sequence is designated plus. (B) Sequence logo generated from the actual frequency of occurrence of each of the four nucleotides around the cleavage site. (C) Six regions of the 3′ UTR and 100 nt sequence downstream of the poly(A) site formed according to the single nucleotide profile. The cleavage and polyadenylation site is located between regions IV and V." path-from-xml="dsr01103.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6654949/dsr01103.jpeg" path-from-xml="dsr01103.jpeg" target="_blank">View large</a><a section="6654949" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr011/2/dsr01103.jpeg?Expires=2147483647&Signature=vXMDjwmYnM~CEJdvpJAUuG~5Gxgw3YGycttyoLbxToWkxZCzt3supK4nYWjujX5QSmcHqQ442LiVpjpHip9NFYKlZkp6lohDS4P9rsRe0gIvdOhx4ILWtA-5H-fZ3kAp-eXsAIrBayqk~zhv6mMC~ubUJW1AX1Cu56A0OLBnKCGWfufMM6oaJOA9bzBjntFUBWUdZBnWBMdCUcDo0NCC-iazfyC7uTbo1W~6LTSSKN3GmN2Gt~fLFR0VB~1mgz7rmNOgY5ZPy~VvwLgYZlEQaxyVTryX7DJSw0PRJzrgedAolVrqRfabnhsom6WS8ETGDLO6YQTaVCsNF~qqixgxwA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6654949&ar=445126&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01103.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Single nucleotide frequencies in the 3′ UTR and 100 nt sequence downstream of the poly(A) site. (<strong>A</strong>) Single nucleotide profile in the 3′ UTR and 100 nt sequence downstream of the poly(A) site. The poly(A) site is at position 0. The upstream sequence of the poly(A) site is designated minus and the downstream sequence is designated plus. (<strong>B</strong>) Sequence logo generated from the actual frequency of occurrence of each of the four nucleotides around the cleavage site. (<strong>C</strong>) Six regions of the 3′ UTR and 100 nt sequence downstream of the poly(A) site formed according to the single nucleotide profile. The cleavage and polyadenylation site is located between regions IV and V.</p></div></div><p>Importantly, the nucleotide distribution profile of the 3′ UTR in <em>A. oryzae</em> was similar to that in plants,<sup><a reveal-id="DSR011C7" data-open="DSR011C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSR011C8" data-open="DSR011C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSR011C39" data-open="DSR011C39" class="link link-ref link-reveal xref-bibr">39</a></sup> yeast,<sup><a reveal-id="DSR011C4" data-open="DSR011C4" class="link link-ref link-reveal xref-bibr">4</a></sup> and mammals,<sup><a reveal-id="DSR011C24" data-open="DSR011C24" class="link link-ref link-reveal xref-bibr">24</a></sup> although the U-rich region was expanded towards the coding region of <em>A. oryzae</em>. On the basis of the nucleotide profile observed, the 3′ UTR plus 100 nt sequence downstream of the poly(A) site was divided into six signal element regions, designated regions I–VI, to identify the sequence elements for 3′-end-processing (Fig. <a reveal-id="DSR011F3" data-open="DSR011F3" class="link link-reveal link-table xref-fig">3</a>C).</p>                            <h3 scrollto-destination=6654951 id="6654951" class="section-title" data-legacy-id=s3d>3.4. Search for nucleotide sequence elements for 3′-end-processing</h3>
<p>To identify 3′-end-processing elements, we searched for tetramer–heptamer nucleotide sequences that appeared most frequently in each signal element region (Table <a reveal-id="DSR011TB1" data-open="DSR011TB1" class="link link-reveal link-table xref-fig">1</a>, the top 50 list is available in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). In region II, equivalent to the region containing the polyadenylation signal in mammals, no significantly conserved hexanucleotide sequence was observed, similar to that observed in yeast and plants. The top-ranked hexanucleotide was AAUGAA in region II. The top two pentanucleotides (AAUGA and AUGAA) were partial sequences of AAUGAA, and all of the top three heptanucleotides contained the AAUGAA sequence (Table <a reveal-id="DSR011TB1" data-open="DSR011TB1" class="link link-reveal link-table xref-fig">1</a>). In addition, according to the zeroth- and first-order Markov chain models, calculation of a standard score (<em>Z</em>-score) to measure the standard deviation of the hexanucleotide sequences from its expected occurrence revealed that AAUGAA was the most over-represented hexanucleotide sequence in region II (Table <a reveal-id="DSR011TB2" data-open="DSR011TB2" class="link link-reveal link-table xref-fig">2</a>). These results suggested that AAUGAA is the most predominant hexanucleotide sequence in region II, although it accounted for only 6% of all transcripts (64 of1043). In contrast, according to the order of <em>Z</em>-scores, the AAUAAA sequence was not the major hexanucleotide sequence in region II, although it ranked third in the list of hexanucleotides. This was also demonstrated by plotting the distribution of hexanucleotide sequences, including AAUGAA and AAUAAA, in the region ranging from −40 to −1 nt (Fig. <a reveal-id="DSR011F4" data-open="DSR011F4" class="link link-reveal link-table xref-fig">4</a>). The AAUGAA sequence was a single nucleotide variant of AAUAAA, but no study has reported that AAUGAA is the most effective A-rich sequence for the 3′-end-processing element in any eukaryote. Interestingly, point mutation of AAUAAA to AAUGAA results in a significant reduction of polyadenylation efficiency by <em>in vitro</em> 3′-end-processing reactions, using nuclear extracts from <em>Xenopus</em> and mammalian cells.<sup><a reveal-id="DSR011C18" data-open="DSR011C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSR011C40" data-open="DSR011C40" class="link link-ref link-reveal xref-bibr">40</a></sup> Thus, the 3′-end-processing machinery in <em>A. oryzae</em> may be somewhat different from that in higher eukaryotes. </p>                        <a id="6654953" scrollto-destination="6654953"></a>
<div content-id="DSR011TB1" class="table-modal"><div class="table-wrap"><div id="DSR011TB1" class="table-wrap-title" data-id="DSR011TB1"><div class="label -label">Table 1.</div><div class="caption"><p>The top five sequences (4–7 nt) that most frequently appear in 3′ ends</p></div> </div><div class="table-overflow"><table><tbody><tr><td colspan="8">Region I (from −149 to −30 nt) </td></tr><tr><td>4 nt<hr /> </td><td>Number<sup>a</sup><hr /> </td><td>5 nt<hr /> </td><td>Number<hr /> </td><td>6 nt<hr /> </td><td>Number<hr /> </td><td>7 nt<hr /> </td><td>Number<hr /> </td></tr><tr><td>UUUG </td><td>629 </td><td>UGUUU </td><td>343 </td><td>UUCUUU </td><td>172 </td><td>UUUCUUU </td><td>99 </td></tr><tr><td>UGUU </td><td>628 </td><td>UGUAU </td><td>341 </td><td>UUUCUU </td><td>162 </td><td>UUUUCUU </td><td>84 </td></tr><tr><td>UUGU </td><td>624 </td><td>UUUCU </td><td>316 </td><td>UGUUUU </td><td>152 </td><td>UUCUUUU </td><td>82 </td></tr><tr><td>GUUU </td><td>619 </td><td>UCUUU </td><td>310 </td><td>UCUUUU </td><td>149 </td><td>UGUAUAU </td><td>61 </td></tr><tr><td>AUUU </td><td>617 </td><td>UUGUU </td><td>301 </td><td>UUUUCU </td><td>144 </td><td>UUUGUUU </td><td>60 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>UUGUUU </td><td>144 </td><td> </td><td> </td></tr><tr><td colspan="8">Region II (from −29 to −14 nt) </td></tr><tr><td>4 nt<hr /> </td><td>Number<hr /> </td><td>5 nt<hr /> </td><td>Number<hr /> </td><td>6 nt<hr /> </td><td>Number<hr /> </td><td>7 nt<hr /> </td><td>Number<hr /> </td></tr><tr><td>AAUA </td><td>286 </td><td>AAUGA </td><td>119 </td><td>AAUGAA </td><td>64 </td><td>AAAUGAA </td><td>23 </td></tr><tr><td>AAUG </td><td>257 </td><td>AUGAA </td><td>110 </td><td>AUGAAU </td><td>48 </td><td>AAUGAAA </td><td>22 </td></tr><tr><td>AAAU </td><td>233 </td><td>AAUAU </td><td>99 </td><td>AAUAAA </td><td>44 </td><td>AAUGAAU </td><td>20 </td></tr><tr><td>AUGA </td><td>216 </td><td>AAUAA </td><td>93 </td><td>AAUAUA </td><td>39 </td><td>AAAUAAA </td><td>18 </td></tr><tr><td>UAAU </td><td>215 </td><td>AUAAU </td><td>92 </td><td>AAAUGA </td><td>37 </td><td>AAUAUGA </td><td>17 </td></tr><tr><td> </td><td> </td><td>AAAUA </td><td>92 </td><td> </td><td> </td><td>AAUAAAU </td><td>17 </td></tr><tr><td colspan="8">Region III (from −13 to −2 nt) </td></tr><tr><td>4 nt<hr /> </td><td>Number<hr /> </td><td>5 nt<hr /> </td><td>Number<hr /> </td><td>6 nt<hr /> </td><td>Number<hr /> </td><td>7 nt<hr /> </td><td>Number<hr /> </td></tr><tr><td>UUUU </td><td>170 </td><td>CUUUU </td><td>64 </td><td>UCUUUU </td><td>24 </td><td>UUUUGUU </td><td>11 </td></tr><tr><td>AUUU </td><td>158 </td><td>UUUUC </td><td>58 </td><td>UUCUUU </td><td>23 </td><td>UUUCUUU </td><td>11 </td></tr><tr><td>UUUC </td><td>150 </td><td>AUUUU </td><td>56 </td><td>UUUUCU </td><td>22 </td><td>UUUUCUU </td><td>10 </td></tr><tr><td>CUUU </td><td>136 </td><td>UUUCU </td><td>55 </td><td>UUUCUU </td><td>22 </td><td>UUCUUUU </td><td>10 </td></tr><tr><td>UUAU </td><td>129 </td><td>UUUAU </td><td>51 </td><td>UUUUGU </td><td>19 </td><td>UGUUUAU </td><td>10 </td></tr><tr><td> </td><td> </td><td>UCUUU </td><td>51 </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="8">Region IV (from −1 to 0 nt) </td></tr><tr><td>2 nt<hr /> </td><td>Number<hr /> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>CA </td><td>328 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>UA </td><td>269 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>GA </td><td>235 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>UG </td><td>36 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>UC </td><td>32 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="8">Region V (from +1 to +20 nt) </td></tr><tr><td>4 nt<hr /> </td><td>Number<hr /> </td><td>5 nt<hr /> </td><td>Number<hr /> </td><td>6 nt<hr /> </td><td>Number<hr /> </td><td>7 nt<hr /> </td><td>Number<hr /> </td></tr><tr><td>UUCU </td><td>186 </td><td>UUUUC </td><td>76 </td><td>UUUUCU </td><td>36 </td><td>UUUUUCU </td><td>17 </td></tr><tr><td>UCUU </td><td>184 </td><td>UUUCU </td><td>68 </td><td>UUUUUC </td><td>32 </td><td>UUUUCUU </td><td>16 </td></tr><tr><td>UUUC </td><td>169 </td><td>CUUUU </td><td>68 </td><td>UCUUUU </td><td>31 </td><td>UUUUCUC </td><td>14 </td></tr><tr><td>UUUU </td><td>157 </td><td>UUCUU </td><td>67 </td><td>UUCUUU </td><td>27 </td><td>UUUCUUU </td><td>14 </td></tr><tr><td>CUUU </td><td>149 </td><td>UUUUU </td><td>61 </td><td>CUUUUU </td><td>27 </td><td>UUUCUCU </td><td>14 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>CUUUUUU </td><td>14 </td></tr><tr><td colspan="8">Region VI (from +21 to +100 nt) </td></tr><tr><td>4 nt<hr /> </td><td>Number<hr /> </td><td>5 nt<hr /> </td><td>Number<hr /> </td><td>6 nt<hr /> </td><td>Number<hr /> </td><td>7 nt<hr /> </td><td>Number<hr /> </td></tr><tr><td>AUAU </td><td>465 </td><td>AUGUA </td><td>180 </td><td>UAUGUA </td><td>74 </td><td>AUAUGUA </td><td>32 </td></tr><tr><td>UGUA </td><td>453 </td><td>UUGUA </td><td>177 </td><td>AGAAAA </td><td>74 </td><td>AUAUAUA </td><td>30 </td></tr><tr><td>AAUA </td><td>426 </td><td>UAUAU </td><td>177 </td><td>AUAUAU </td><td>66 </td><td>AAAGAAA </td><td>30 </td></tr><tr><td>UAUA </td><td>423 </td><td>GUAGA </td><td>177 </td><td>AUUGUA </td><td>64 </td><td>UAUAUAU </td><td>28 </td></tr><tr><td>UAGA </td><td>412 </td><td>UGUAG </td><td>166 </td><td>AAAGAA </td><td>64 </td><td>AAGAAAA </td><td>28 </td></tr></tbody></table></div><div content-id="DSR011TB1" class="table-modal"><table><tbody><tr><td colspan="8">Region I (from −149 to −30 nt) </td></tr><tr><td>4 nt<hr /> </td><td>Number<sup>a</sup><hr /> </td><td>5 nt<hr /> </td><td>Number<hr /> </td><td>6 nt<hr /> </td><td>Number<hr /> </td><td>7 nt<hr /> </td><td>Number<hr /> </td></tr><tr><td>UUUG </td><td>629 </td><td>UGUUU </td><td>343 </td><td>UUCUUU </td><td>172 </td><td>UUUCUUU </td><td>99 </td></tr><tr><td>UGUU </td><td>628 </td><td>UGUAU </td><td>341 </td><td>UUUCUU </td><td>162 </td><td>UUUUCUU </td><td>84 </td></tr><tr><td>UUGU </td><td>624 </td><td>UUUCU </td><td>316 </td><td>UGUUUU </td><td>152 </td><td>UUCUUUU </td><td>82 </td></tr><tr><td>GUUU </td><td>619 </td><td>UCUUU </td><td>310 </td><td>UCUUUU </td><td>149 </td><td>UGUAUAU </td><td>61 </td></tr><tr><td>AUUU </td><td>617 </td><td>UUGUU </td><td>301 </td><td>UUUUCU </td><td>144 </td><td>UUUGUUU </td><td>60 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>UUGUUU </td><td>144 </td><td> </td><td> </td></tr><tr><td colspan="8">Region II (from −29 to −14 nt) </td></tr><tr><td>4 nt<hr /> </td><td>Number<hr /> </td><td>5 nt<hr /> </td><td>Number<hr /> </td><td>6 nt<hr /> </td><td>Number<hr /> </td><td>7 nt<hr /> </td><td>Number<hr /> </td></tr><tr><td>AAUA </td><td>286 </td><td>AAUGA </td><td>119 </td><td>AAUGAA </td><td>64 </td><td>AAAUGAA </td><td>23 </td></tr><tr><td>AAUG </td><td>257 </td><td>AUGAA </td><td>110 </td><td>AUGAAU </td><td>48 </td><td>AAUGAAA </td><td>22 </td></tr><tr><td>AAAU </td><td>233 </td><td>AAUAU </td><td>99 </td><td>AAUAAA </td><td>44 </td><td>AAUGAAU </td><td>20 </td></tr><tr><td>AUGA </td><td>216 </td><td>AAUAA </td><td>93 </td><td>AAUAUA </td><td>39 </td><td>AAAUAAA </td><td>18 </td></tr><tr><td>UAAU </td><td>215 </td><td>AUAAU </td><td>92 </td><td>AAAUGA </td><td>37 </td><td>AAUAUGA </td><td>17 </td></tr><tr><td> </td><td> </td><td>AAAUA </td><td>92 </td><td> </td><td> </td><td>AAUAAAU </td><td>17 </td></tr><tr><td colspan="8">Region III (from −13 to −2 nt) </td></tr><tr><td>4 nt<hr /> </td><td>Number<hr /> </td><td>5 nt<hr /> </td><td>Number<hr /> </td><td>6 nt<hr /> </td><td>Number<hr /> </td><td>7 nt<hr /> </td><td>Number<hr /> </td></tr><tr><td>UUUU </td><td>170 </td><td>CUUUU </td><td>64 </td><td>UCUUUU </td><td>24 </td><td>UUUUGUU </td><td>11 </td></tr><tr><td>AUUU </td><td>158 </td><td>UUUUC </td><td>58 </td><td>UUCUUU </td><td>23 </td><td>UUUCUUU </td><td>11 </td></tr><tr><td>UUUC </td><td>150 </td><td>AUUUU </td><td>56 </td><td>UUUUCU </td><td>22 </td><td>UUUUCUU </td><td>10 </td></tr><tr><td>CUUU </td><td>136 </td><td>UUUCU </td><td>55 </td><td>UUUCUU </td><td>22 </td><td>UUCUUUU </td><td>10 </td></tr><tr><td>UUAU </td><td>129 </td><td>UUUAU </td><td>51 </td><td>UUUUGU </td><td>19 </td><td>UGUUUAU </td><td>10 </td></tr><tr><td> </td><td> </td><td>UCUUU </td><td>51 </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="8">Region IV (from −1 to 0 nt) </td></tr><tr><td>2 nt<hr /> </td><td>Number<hr /> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>CA </td><td>328 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>UA </td><td>269 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>GA </td><td>235 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>UG </td><td>36 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>UC </td><td>32 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="8">Region V (from +1 to +20 nt) </td></tr><tr><td>4 nt<hr /> </td><td>Number<hr /> </td><td>5 nt<hr /> </td><td>Number<hr /> </td><td>6 nt<hr /> </td><td>Number<hr /> </td><td>7 nt<hr /> </td><td>Number<hr /> </td></tr><tr><td>UUCU </td><td>186 </td><td>UUUUC </td><td>76 </td><td>UUUUCU </td><td>36 </td><td>UUUUUCU </td><td>17 </td></tr><tr><td>UCUU </td><td>184 </td><td>UUUCU </td><td>68 </td><td>UUUUUC </td><td>32 </td><td>UUUUCUU </td><td>16 </td></tr><tr><td>UUUC </td><td>169 </td><td>CUUUU </td><td>68 </td><td>UCUUUU </td><td>31 </td><td>UUUUCUC </td><td>14 </td></tr><tr><td>UUUU </td><td>157 </td><td>UUCUU </td><td>67 </td><td>UUCUUU </td><td>27 </td><td>UUUCUUU </td><td>14 </td></tr><tr><td>CUUU </td><td>149 </td><td>UUUUU </td><td>61 </td><td>CUUUUU </td><td>27 </td><td>UUUCUCU </td><td>14 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>CUUUUUU </td><td>14 </td></tr><tr><td colspan="8">Region VI (from +21 to +100 nt) </td></tr><tr><td>4 nt<hr /> </td><td>Number<hr /> </td><td>5 nt<hr /> </td><td>Number<hr /> </td><td>6 nt<hr /> </td><td>Number<hr /> </td><td>7 nt<hr /> </td><td>Number<hr /> </td></tr><tr><td>AUAU </td><td>465 </td><td>AUGUA </td><td>180 </td><td>UAUGUA </td><td>74 </td><td>AUAUGUA </td><td>32 </td></tr><tr><td>UGUA </td><td>453 </td><td>UUGUA </td><td>177 </td><td>AGAAAA </td><td>74 </td><td>AUAUAUA </td><td>30 </td></tr><tr><td>AAUA </td><td>426 </td><td>UAUAU </td><td>177 </td><td>AUAUAU </td><td>66 </td><td>AAAGAAA </td><td>30 </td></tr><tr><td>UAUA </td><td>423 </td><td>GUAGA </td><td>177 </td><td>AUUGUA </td><td>64 </td><td>UAUAUAU </td><td>28 </td></tr><tr><td>UAGA </td><td>412 </td><td>UGUAG </td><td>166 </td><td>AAAGAA </td><td>64 </td><td>AAGAAAA </td><td>28 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>The number of transcripts with at least one occurrence.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6654953" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR011TB1" class="table-wrap-title" data-id="DSR011TB1"><div class="label -label">Table 1.</div><div class="caption"><p>The top five sequences (4–7 nt) that most frequently appear in 3′ ends</p></div> </div><div class="table-overflow"><table><tbody><tr><td colspan="8">Region I (from −149 to −30 nt) </td></tr><tr><td>4 nt<hr /> </td><td>Number<sup>a</sup><hr /> </td><td>5 nt<hr /> </td><td>Number<hr /> </td><td>6 nt<hr /> </td><td>Number<hr /> </td><td>7 nt<hr /> </td><td>Number<hr /> </td></tr><tr><td>UUUG </td><td>629 </td><td>UGUUU </td><td>343 </td><td>UUCUUU </td><td>172 </td><td>UUUCUUU </td><td>99 </td></tr><tr><td>UGUU </td><td>628 </td><td>UGUAU </td><td>341 </td><td>UUUCUU </td><td>162 </td><td>UUUUCUU </td><td>84 </td></tr><tr><td>UUGU </td><td>624 </td><td>UUUCU </td><td>316 </td><td>UGUUUU </td><td>152 </td><td>UUCUUUU </td><td>82 </td></tr><tr><td>GUUU </td><td>619 </td><td>UCUUU </td><td>310 </td><td>UCUUUU </td><td>149 </td><td>UGUAUAU </td><td>61 </td></tr><tr><td>AUUU </td><td>617 </td><td>UUGUU </td><td>301 </td><td>UUUUCU </td><td>144 </td><td>UUUGUUU </td><td>60 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>UUGUUU </td><td>144 </td><td> </td><td> </td></tr><tr><td colspan="8">Region II (from −29 to −14 nt) </td></tr><tr><td>4 nt<hr /> </td><td>Number<hr /> </td><td>5 nt<hr /> </td><td>Number<hr /> </td><td>6 nt<hr /> </td><td>Number<hr /> </td><td>7 nt<hr /> </td><td>Number<hr /> </td></tr><tr><td>AAUA </td><td>286 </td><td>AAUGA </td><td>119 </td><td>AAUGAA </td><td>64 </td><td>AAAUGAA </td><td>23 </td></tr><tr><td>AAUG </td><td>257 </td><td>AUGAA </td><td>110 </td><td>AUGAAU </td><td>48 </td><td>AAUGAAA </td><td>22 </td></tr><tr><td>AAAU </td><td>233 </td><td>AAUAU </td><td>99 </td><td>AAUAAA </td><td>44 </td><td>AAUGAAU </td><td>20 </td></tr><tr><td>AUGA </td><td>216 </td><td>AAUAA </td><td>93 </td><td>AAUAUA </td><td>39 </td><td>AAAUAAA </td><td>18 </td></tr><tr><td>UAAU </td><td>215 </td><td>AUAAU </td><td>92 </td><td>AAAUGA </td><td>37 </td><td>AAUAUGA </td><td>17 </td></tr><tr><td> </td><td> </td><td>AAAUA </td><td>92 </td><td> </td><td> </td><td>AAUAAAU </td><td>17 </td></tr><tr><td colspan="8">Region III (from −13 to −2 nt) </td></tr><tr><td>4 nt<hr /> </td><td>Number<hr /> </td><td>5 nt<hr /> </td><td>Number<hr /> </td><td>6 nt<hr /> </td><td>Number<hr /> </td><td>7 nt<hr /> </td><td>Number<hr /> </td></tr><tr><td>UUUU </td><td>170 </td><td>CUUUU </td><td>64 </td><td>UCUUUU </td><td>24 </td><td>UUUUGUU </td><td>11 </td></tr><tr><td>AUUU </td><td>158 </td><td>UUUUC </td><td>58 </td><td>UUCUUU </td><td>23 </td><td>UUUCUUU </td><td>11 </td></tr><tr><td>UUUC </td><td>150 </td><td>AUUUU </td><td>56 </td><td>UUUUCU </td><td>22 </td><td>UUUUCUU </td><td>10 </td></tr><tr><td>CUUU </td><td>136 </td><td>UUUCU </td><td>55 </td><td>UUUCUU </td><td>22 </td><td>UUCUUUU </td><td>10 </td></tr><tr><td>UUAU </td><td>129 </td><td>UUUAU </td><td>51 </td><td>UUUUGU </td><td>19 </td><td>UGUUUAU </td><td>10 </td></tr><tr><td> </td><td> </td><td>UCUUU </td><td>51 </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="8">Region IV (from −1 to 0 nt) </td></tr><tr><td>2 nt<hr /> </td><td>Number<hr /> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>CA </td><td>328 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>UA </td><td>269 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>GA </td><td>235 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>UG </td><td>36 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>UC </td><td>32 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="8">Region V (from +1 to +20 nt) </td></tr><tr><td>4 nt<hr /> </td><td>Number<hr /> </td><td>5 nt<hr /> </td><td>Number<hr /> </td><td>6 nt<hr /> </td><td>Number<hr /> </td><td>7 nt<hr /> </td><td>Number<hr /> </td></tr><tr><td>UUCU </td><td>186 </td><td>UUUUC </td><td>76 </td><td>UUUUCU </td><td>36 </td><td>UUUUUCU </td><td>17 </td></tr><tr><td>UCUU </td><td>184 </td><td>UUUCU </td><td>68 </td><td>UUUUUC </td><td>32 </td><td>UUUUCUU </td><td>16 </td></tr><tr><td>UUUC </td><td>169 </td><td>CUUUU </td><td>68 </td><td>UCUUUU </td><td>31 </td><td>UUUUCUC </td><td>14 </td></tr><tr><td>UUUU </td><td>157 </td><td>UUCUU </td><td>67 </td><td>UUCUUU </td><td>27 </td><td>UUUCUUU </td><td>14 </td></tr><tr><td>CUUU </td><td>149 </td><td>UUUUU </td><td>61 </td><td>CUUUUU </td><td>27 </td><td>UUUCUCU </td><td>14 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>CUUUUUU </td><td>14 </td></tr><tr><td colspan="8">Region VI (from +21 to +100 nt) </td></tr><tr><td>4 nt<hr /> </td><td>Number<hr /> </td><td>5 nt<hr /> </td><td>Number<hr /> </td><td>6 nt<hr /> </td><td>Number<hr /> </td><td>7 nt<hr /> </td><td>Number<hr /> </td></tr><tr><td>AUAU </td><td>465 </td><td>AUGUA </td><td>180 </td><td>UAUGUA </td><td>74 </td><td>AUAUGUA </td><td>32 </td></tr><tr><td>UGUA </td><td>453 </td><td>UUGUA </td><td>177 </td><td>AGAAAA </td><td>74 </td><td>AUAUAUA </td><td>30 </td></tr><tr><td>AAUA </td><td>426 </td><td>UAUAU </td><td>177 </td><td>AUAUAU </td><td>66 </td><td>AAAGAAA </td><td>30 </td></tr><tr><td>UAUA </td><td>423 </td><td>GUAGA </td><td>177 </td><td>AUUGUA </td><td>64 </td><td>UAUAUAU </td><td>28 </td></tr><tr><td>UAGA </td><td>412 </td><td>UGUAG </td><td>166 </td><td>AAAGAA </td><td>64 </td><td>AAGAAAA </td><td>28 </td></tr></tbody></table></div><div content-id="DSR011TB1" class="table-modal"><table><tbody><tr><td colspan="8">Region I (from −149 to −30 nt) </td></tr><tr><td>4 nt<hr /> </td><td>Number<sup>a</sup><hr /> </td><td>5 nt<hr /> </td><td>Number<hr /> </td><td>6 nt<hr /> </td><td>Number<hr /> </td><td>7 nt<hr /> </td><td>Number<hr /> </td></tr><tr><td>UUUG </td><td>629 </td><td>UGUUU </td><td>343 </td><td>UUCUUU </td><td>172 </td><td>UUUCUUU </td><td>99 </td></tr><tr><td>UGUU </td><td>628 </td><td>UGUAU </td><td>341 </td><td>UUUCUU </td><td>162 </td><td>UUUUCUU </td><td>84 </td></tr><tr><td>UUGU </td><td>624 </td><td>UUUCU </td><td>316 </td><td>UGUUUU </td><td>152 </td><td>UUCUUUU </td><td>82 </td></tr><tr><td>GUUU </td><td>619 </td><td>UCUUU </td><td>310 </td><td>UCUUUU </td><td>149 </td><td>UGUAUAU </td><td>61 </td></tr><tr><td>AUUU </td><td>617 </td><td>UUGUU </td><td>301 </td><td>UUUUCU </td><td>144 </td><td>UUUGUUU </td><td>60 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td>UUGUUU </td><td>144 </td><td> </td><td> </td></tr><tr><td colspan="8">Region II (from −29 to −14 nt) </td></tr><tr><td>4 nt<hr /> </td><td>Number<hr /> </td><td>5 nt<hr /> </td><td>Number<hr /> </td><td>6 nt<hr /> </td><td>Number<hr /> </td><td>7 nt<hr /> </td><td>Number<hr /> </td></tr><tr><td>AAUA </td><td>286 </td><td>AAUGA </td><td>119 </td><td>AAUGAA </td><td>64 </td><td>AAAUGAA </td><td>23 </td></tr><tr><td>AAUG </td><td>257 </td><td>AUGAA </td><td>110 </td><td>AUGAAU </td><td>48 </td><td>AAUGAAA </td><td>22 </td></tr><tr><td>AAAU </td><td>233 </td><td>AAUAU </td><td>99 </td><td>AAUAAA </td><td>44 </td><td>AAUGAAU </td><td>20 </td></tr><tr><td>AUGA </td><td>216 </td><td>AAUAA </td><td>93 </td><td>AAUAUA </td><td>39 </td><td>AAAUAAA </td><td>18 </td></tr><tr><td>UAAU </td><td>215 </td><td>AUAAU </td><td>92 </td><td>AAAUGA </td><td>37 </td><td>AAUAUGA </td><td>17 </td></tr><tr><td> </td><td> </td><td>AAAUA </td><td>92 </td><td> </td><td> </td><td>AAUAAAU </td><td>17 </td></tr><tr><td colspan="8">Region III (from −13 to −2 nt) </td></tr><tr><td>4 nt<hr /> </td><td>Number<hr /> </td><td>5 nt<hr /> </td><td>Number<hr /> </td><td>6 nt<hr /> </td><td>Number<hr /> </td><td>7 nt<hr /> </td><td>Number<hr /> </td></tr><tr><td>UUUU </td><td>170 </td><td>CUUUU </td><td>64 </td><td>UCUUUU </td><td>24 </td><td>UUUUGUU </td><td>11 </td></tr><tr><td>AUUU </td><td>158 </td><td>UUUUC </td><td>58 </td><td>UUCUUU </td><td>23 </td><td>UUUCUUU </td><td>11 </td></tr><tr><td>UUUC </td><td>150 </td><td>AUUUU </td><td>56 </td><td>UUUUCU </td><td>22 </td><td>UUUUCUU </td><td>10 </td></tr><tr><td>CUUU </td><td>136 </td><td>UUUCU </td><td>55 </td><td>UUUCUU </td><td>22 </td><td>UUCUUUU </td><td>10 </td></tr><tr><td>UUAU </td><td>129 </td><td>UUUAU </td><td>51 </td><td>UUUUGU </td><td>19 </td><td>UGUUUAU </td><td>10 </td></tr><tr><td> </td><td> </td><td>UCUUU </td><td>51 </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="8">Region IV (from −1 to 0 nt) </td></tr><tr><td>2 nt<hr /> </td><td>Number<hr /> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>CA </td><td>328 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>UA </td><td>269 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>GA </td><td>235 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>UG </td><td>36 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>UC </td><td>32 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td colspan="8">Region V (from +1 to +20 nt) </td></tr><tr><td>4 nt<hr /> </td><td>Number<hr /> </td><td>5 nt<hr /> </td><td>Number<hr /> </td><td>6 nt<hr /> </td><td>Number<hr /> </td><td>7 nt<hr /> </td><td>Number<hr /> </td></tr><tr><td>UUCU </td><td>186 </td><td>UUUUC </td><td>76 </td><td>UUUUCU </td><td>36 </td><td>UUUUUCU </td><td>17 </td></tr><tr><td>UCUU </td><td>184 </td><td>UUUCU </td><td>68 </td><td>UUUUUC </td><td>32 </td><td>UUUUCUU </td><td>16 </td></tr><tr><td>UUUC </td><td>169 </td><td>CUUUU </td><td>68 </td><td>UCUUUU </td><td>31 </td><td>UUUUCUC </td><td>14 </td></tr><tr><td>UUUU </td><td>157 </td><td>UUCUU </td><td>67 </td><td>UUCUUU </td><td>27 </td><td>UUUCUUU </td><td>14 </td></tr><tr><td>CUUU </td><td>149 </td><td>UUUUU </td><td>61 </td><td>CUUUUU </td><td>27 </td><td>UUUCUCU </td><td>14 </td></tr><tr><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>CUUUUUU </td><td>14 </td></tr><tr><td colspan="8">Region VI (from +21 to +100 nt) </td></tr><tr><td>4 nt<hr /> </td><td>Number<hr /> </td><td>5 nt<hr /> </td><td>Number<hr /> </td><td>6 nt<hr /> </td><td>Number<hr /> </td><td>7 nt<hr /> </td><td>Number<hr /> </td></tr><tr><td>AUAU </td><td>465 </td><td>AUGUA </td><td>180 </td><td>UAUGUA </td><td>74 </td><td>AUAUGUA </td><td>32 </td></tr><tr><td>UGUA </td><td>453 </td><td>UUGUA </td><td>177 </td><td>AGAAAA </td><td>74 </td><td>AUAUAUA </td><td>30 </td></tr><tr><td>AAUA </td><td>426 </td><td>UAUAU </td><td>177 </td><td>AUAUAU </td><td>66 </td><td>AAAGAAA </td><td>30 </td></tr><tr><td>UAUA </td><td>423 </td><td>GUAGA </td><td>177 </td><td>AUUGUA </td><td>64 </td><td>UAUAUAU </td><td>28 </td></tr><tr><td>UAGA </td><td>412 </td><td>UGUAG </td><td>166 </td><td>AAAGAA </td><td>64 </td><td>AAGAAAA </td><td>28 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>The number of transcripts with at least one occurrence.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6654953" target="_blank">View Large</a></div></div>                        <a id="6654954" scrollto-destination="6654954"></a>
<div content-id="DSR011TB2" class="table-modal"><div class="table-wrap"><div id="DSR011TB2" class="table-wrap-title" data-id="DSR011TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Top 10 hexanucleotide sequences mostly over-represented in region II</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Rank </th><th colspan="3">Markov order = 0<hr /> </th><th colspan="3">Markov order = 1<hr /> </th></tr><tr><th>Word </th><th><em>Z</em>-score </th><th>Number of occurences<sup>a</sup> </th><th>Word </th><th><em>Z</em>-score </th><th>Number of occurences<sup>a</sup> </th></tr></thead><tbody><tr><td>1 </td><td>AAUGAA </td><td>16.603 </td><td>67 </td><td>AAUGAA </td><td>9.856 </td><td>67 </td></tr><tr><td>2 </td><td>AUGAAU </td><td>13.146 </td><td>48 </td><td>AUGAAU </td><td>6.086 </td><td>48 </td></tr><tr><td>3 </td><td>GAAUGA </td><td>11.594 </td><td>31 </td><td>GAAUGA </td><td>6.067 </td><td>31 </td></tr><tr><td>4 </td><td>UGAAUG </td><td>10.594 </td><td>25 </td><td>GUCAAU </td><td>6.002 </td><td>16 </td></tr><tr><td>5 </td><td>CAAUGC </td><td>10.083 </td><td>17 </td><td>GUCGCG </td><td>5.727 </td><td>3 </td></tr><tr><td>6 </td><td>AAUGCA </td><td>9.026 </td><td>25 </td><td>CAAUGC </td><td>5.711 </td><td>17 </td></tr><tr><td>7 </td><td>UCAAUG </td><td>8.684 </td><td>21 </td><td>UCGCGU </td><td>5.59 </td><td>4 </td></tr><tr><td>8 </td><td>AUGCAA </td><td>8.576 </td><td>24 </td><td>AAUACA </td><td>5.196 </td><td>29 </td></tr><tr><td>9 </td><td>AAAUGA </td><td>7.962 </td><td>38 </td><td>GGCAGU </td><td>5.027 </td><td>5 </td></tr><tr><td>10 </td><td>GGAAUG </td><td>7.865 </td><td>14 </td><td>UCAAUU </td><td>4.994 </td><td>23 </td></tr><tr><td>70 </td><td>AAUAAA </td><td>4.116 </td><td>46 </td><td> </td><td> </td><td> </td></tr></tbody></table></div><div content-id="DSR011TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Rank </th><th colspan="3">Markov order = 0<hr /> </th><th colspan="3">Markov order = 1<hr /> </th></tr><tr><th>Word </th><th><em>Z</em>-score </th><th>Number of occurences<sup>a</sup> </th><th>Word </th><th><em>Z</em>-score </th><th>Number of occurences<sup>a</sup> </th></tr></thead><tbody><tr><td>1 </td><td>AAUGAA </td><td>16.603 </td><td>67 </td><td>AAUGAA </td><td>9.856 </td><td>67 </td></tr><tr><td>2 </td><td>AUGAAU </td><td>13.146 </td><td>48 </td><td>AUGAAU </td><td>6.086 </td><td>48 </td></tr><tr><td>3 </td><td>GAAUGA </td><td>11.594 </td><td>31 </td><td>GAAUGA </td><td>6.067 </td><td>31 </td></tr><tr><td>4 </td><td>UGAAUG </td><td>10.594 </td><td>25 </td><td>GUCAAU </td><td>6.002 </td><td>16 </td></tr><tr><td>5 </td><td>CAAUGC </td><td>10.083 </td><td>17 </td><td>GUCGCG </td><td>5.727 </td><td>3 </td></tr><tr><td>6 </td><td>AAUGCA </td><td>9.026 </td><td>25 </td><td>CAAUGC </td><td>5.711 </td><td>17 </td></tr><tr><td>7 </td><td>UCAAUG </td><td>8.684 </td><td>21 </td><td>UCGCGU </td><td>5.59 </td><td>4 </td></tr><tr><td>8 </td><td>AUGCAA </td><td>8.576 </td><td>24 </td><td>AAUACA </td><td>5.196 </td><td>29 </td></tr><tr><td>9 </td><td>AAAUGA </td><td>7.962 </td><td>38 </td><td>GGCAGU </td><td>5.027 </td><td>5 </td></tr><tr><td>10 </td><td>GGAAUG </td><td>7.865 </td><td>14 </td><td>UCAAUU </td><td>4.994 </td><td>23 </td></tr><tr><td>70 </td><td>AAUAAA </td><td>4.116 </td><td>46 </td><td> </td><td> </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><em>Z</em>-scores of the most over-represented hexanucleotide sequences in region II, according to the zeroth- and first-order Markov chain models.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>The number of hexanucleotide sequences found in region II.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6654954" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR011TB2" class="table-wrap-title" data-id="DSR011TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Top 10 hexanucleotide sequences mostly over-represented in region II</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Rank </th><th colspan="3">Markov order = 0<hr /> </th><th colspan="3">Markov order = 1<hr /> </th></tr><tr><th>Word </th><th><em>Z</em>-score </th><th>Number of occurences<sup>a</sup> </th><th>Word </th><th><em>Z</em>-score </th><th>Number of occurences<sup>a</sup> </th></tr></thead><tbody><tr><td>1 </td><td>AAUGAA </td><td>16.603 </td><td>67 </td><td>AAUGAA </td><td>9.856 </td><td>67 </td></tr><tr><td>2 </td><td>AUGAAU </td><td>13.146 </td><td>48 </td><td>AUGAAU </td><td>6.086 </td><td>48 </td></tr><tr><td>3 </td><td>GAAUGA </td><td>11.594 </td><td>31 </td><td>GAAUGA </td><td>6.067 </td><td>31 </td></tr><tr><td>4 </td><td>UGAAUG </td><td>10.594 </td><td>25 </td><td>GUCAAU </td><td>6.002 </td><td>16 </td></tr><tr><td>5 </td><td>CAAUGC </td><td>10.083 </td><td>17 </td><td>GUCGCG </td><td>5.727 </td><td>3 </td></tr><tr><td>6 </td><td>AAUGCA </td><td>9.026 </td><td>25 </td><td>CAAUGC </td><td>5.711 </td><td>17 </td></tr><tr><td>7 </td><td>UCAAUG </td><td>8.684 </td><td>21 </td><td>UCGCGU </td><td>5.59 </td><td>4 </td></tr><tr><td>8 </td><td>AUGCAA </td><td>8.576 </td><td>24 </td><td>AAUACA </td><td>5.196 </td><td>29 </td></tr><tr><td>9 </td><td>AAAUGA </td><td>7.962 </td><td>38 </td><td>GGCAGU </td><td>5.027 </td><td>5 </td></tr><tr><td>10 </td><td>GGAAUG </td><td>7.865 </td><td>14 </td><td>UCAAUU </td><td>4.994 </td><td>23 </td></tr><tr><td>70 </td><td>AAUAAA </td><td>4.116 </td><td>46 </td><td> </td><td> </td><td> </td></tr></tbody></table></div><div content-id="DSR011TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Rank </th><th colspan="3">Markov order = 0<hr /> </th><th colspan="3">Markov order = 1<hr /> </th></tr><tr><th>Word </th><th><em>Z</em>-score </th><th>Number of occurences<sup>a</sup> </th><th>Word </th><th><em>Z</em>-score </th><th>Number of occurences<sup>a</sup> </th></tr></thead><tbody><tr><td>1 </td><td>AAUGAA </td><td>16.603 </td><td>67 </td><td>AAUGAA </td><td>9.856 </td><td>67 </td></tr><tr><td>2 </td><td>AUGAAU </td><td>13.146 </td><td>48 </td><td>AUGAAU </td><td>6.086 </td><td>48 </td></tr><tr><td>3 </td><td>GAAUGA </td><td>11.594 </td><td>31 </td><td>GAAUGA </td><td>6.067 </td><td>31 </td></tr><tr><td>4 </td><td>UGAAUG </td><td>10.594 </td><td>25 </td><td>GUCAAU </td><td>6.002 </td><td>16 </td></tr><tr><td>5 </td><td>CAAUGC </td><td>10.083 </td><td>17 </td><td>GUCGCG </td><td>5.727 </td><td>3 </td></tr><tr><td>6 </td><td>AAUGCA </td><td>9.026 </td><td>25 </td><td>CAAUGC </td><td>5.711 </td><td>17 </td></tr><tr><td>7 </td><td>UCAAUG </td><td>8.684 </td><td>21 </td><td>UCGCGU </td><td>5.59 </td><td>4 </td></tr><tr><td>8 </td><td>AUGCAA </td><td>8.576 </td><td>24 </td><td>AAUACA </td><td>5.196 </td><td>29 </td></tr><tr><td>9 </td><td>AAAUGA </td><td>7.962 </td><td>38 </td><td>GGCAGU </td><td>5.027 </td><td>5 </td></tr><tr><td>10 </td><td>GGAAUG </td><td>7.865 </td><td>14 </td><td>UCAAUU </td><td>4.994 </td><td>23 </td></tr><tr><td>70 </td><td>AAUAAA </td><td>4.116 </td><td>46 </td><td> </td><td> </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><em>Z</em>-scores of the most over-represented hexanucleotide sequences in region II, according to the zeroth- and first-order Markov chain models.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>The number of hexanucleotide sequences found in region II.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6654954" target="_blank">View Large</a></div></div>                        <a id="6654955" scrollto-destination="6654955"></a>
<div id="DSR011F4" data-id="DSR011F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr011/2/m_dsr01104.jpeg?Expires=2147483647&Signature=kJlVJV-nZ0jat8ihod~e5YnFrfLa779EMs0D0TfaKV6~xWt5hv2m1kr0hI1tzr9avg6j7Fw1myiMwk5BJA8wSnqkgKYE6CubxGZf1QLmKz-K34DVWz5kpQLReYwQ1XtuosJzYoFQAXssmJZ9ro8bG5WjsJrKi59jA2NVww5MnHx7OMSlHZbWr49SFnrvycNTKFnzyQ2Cu61MXBoPJZS6QuklocpgqRk0tLxBtLDNFN-qITg89pdHsPFvViM31uqbyH1VhunSndNPhDVUXEy-zl3D0tiZ-N7oYFN6c6xiE-hqrSNV3gne7Y0BWQiGBFNfNN7KewX6nnURwaqK-CySpw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Representative hexanucleotide signals in the poly(A) signal region (from −40 to −1 nt)." path-from-xml="dsr01104.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6654955/dsr01104.jpeg" path-from-xml="dsr01104.jpeg" target="_blank">View large</a><a section="6654955" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr011/2/dsr01104.jpeg?Expires=2147483647&Signature=zIDv1xu4UrhJfhxTj56Hrzrg4z3yLk68H2pgwHBiL7-Tr2PlF0xz~ZBH7nyxsEfcdNEhcxMSsYKzjDlJr0PCVloAOeOxim3LfLSLK2tkpi-yQiKdcoXaIAKo5lR15wtnkwXwAEkHFk16N2pxW01kPlEsjvTYzpMrRGehAdRhdsTl69mnwn-gcSnHydj-saOVx3Np3-048IC5b~bTL9VFisPLQ02~f4jGZieAVRNSq7AmsOydK~sYvBiOVcNAFpNxmc4NHuwkPSnn7BFsh3hpZdBmbgsKUaqgJPg5UOGdpLaOC-yNubcaZbppwshdReoe-xm9ahFMAdQhsNQAQHjFKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6654955&ar=445126&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01104.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Representative hexanucleotide signals in the poly(A) signal region (from −40 to −1 nt).</p></div></div><div content-id="DSR011F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr011/2/m_dsr01104.jpeg?Expires=2147483647&Signature=kJlVJV-nZ0jat8ihod~e5YnFrfLa779EMs0D0TfaKV6~xWt5hv2m1kr0hI1tzr9avg6j7Fw1myiMwk5BJA8wSnqkgKYE6CubxGZf1QLmKz-K34DVWz5kpQLReYwQ1XtuosJzYoFQAXssmJZ9ro8bG5WjsJrKi59jA2NVww5MnHx7OMSlHZbWr49SFnrvycNTKFnzyQ2Cu61MXBoPJZS6QuklocpgqRk0tLxBtLDNFN-qITg89pdHsPFvViM31uqbyH1VhunSndNPhDVUXEy-zl3D0tiZ-N7oYFN6c6xiE-hqrSNV3gne7Y0BWQiGBFNfNN7KewX6nnURwaqK-CySpw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Representative hexanucleotide signals in the poly(A) signal region (from −40 to −1 nt)." path-from-xml="dsr01104.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6654955/dsr01104.jpeg" path-from-xml="dsr01104.jpeg" target="_blank">View large</a><a section="6654955" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr011/2/dsr01104.jpeg?Expires=2147483647&Signature=zIDv1xu4UrhJfhxTj56Hrzrg4z3yLk68H2pgwHBiL7-Tr2PlF0xz~ZBH7nyxsEfcdNEhcxMSsYKzjDlJr0PCVloAOeOxim3LfLSLK2tkpi-yQiKdcoXaIAKo5lR15wtnkwXwAEkHFk16N2pxW01kPlEsjvTYzpMrRGehAdRhdsTl69mnwn-gcSnHydj-saOVx3Np3-048IC5b~bTL9VFisPLQ02~f4jGZieAVRNSq7AmsOydK~sYvBiOVcNAFpNxmc4NHuwkPSnn7BFsh3hpZdBmbgsKUaqgJPg5UOGdpLaOC-yNubcaZbppwshdReoe-xm9ahFMAdQhsNQAQHjFKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6654955&ar=445126&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01104.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Representative hexanucleotide signals in the poly(A) signal region (from −40 to −1 nt).</p></div></div><p>Predominant sequence motifs in the upstream of the A-rich region (region I), called efficiency elements in yeast and far upstream elements in plants, have been identified. The best sequence for yeast efficiency elements is UAUAUA and its single nucleotide variants (UAUGUA and UACAUA).<sup><a reveal-id="DSR011C4" data-open="DSR011C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSR011C41" data-open="DSR011C41" class="link link-ref link-reveal xref-bibr">41</a>,<a reveal-id="DSR011C42" data-open="DSR011C42" class="link link-ref link-reveal xref-bibr">42</a></sup> In contrast, the best sequence of plant far upstream elements is UGUA.<sup><a reveal-id="DSR011C8" data-open="DSR011C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSR011C38" data-open="DSR011C38" class="link link-ref link-reveal xref-bibr">38</a></sup> In addition, this region in mammalian cells is defined as the auxiliary upstream element, and the UGUAN sequence element may function as a recognition element for 3′-end-processing proteins in case of A(A/U)UAAA-lacking 3′ UTRs.<sup><a reveal-id="DSR011C43" data-open="DSR011C43" class="link link-ref link-reveal xref-bibr">43</a></sup> However, these sequences were not predominant in region I of the <em>A. oryzae</em> poly(A) data set (Table <a reveal-id="DSR011TB1" data-open="DSR011TB1" class="link link-reveal link-table xref-fig">1</a>). Moreover, no other sequence motif was highly conserved in this region, although the top nucleotide sequences were notably U-rich sequences. Similarly, no conserved sequence motif was observed in two other U-rich regions (regions III and V), suggesting that these sequences can be defined only as U-rich elements.</p><p>In region IV, equivalent of the cleavage site, the CA sequence ranked top and this motif existed in 31% of the sequences (Table <a reveal-id="DSR011TB1" data-open="DSR011TB1" class="link link-reveal link-table xref-fig">1</a>). This suggested that the CA sequence is the most optimal cleavage site in <em>A. oryzae</em>. However, the GA sequence ranked third, and this motif existed in 22% of the sequences, suggesting that CA or UA dinucleotide sequences are not strictly conserved as the cleavage site. In region VI, no commonality was observed in the high-ranked tetramer–heptamer sequences (Table <a reveal-id="DSR011TB1" data-open="DSR011TB1" class="link link-reveal link-table xref-fig">1</a>), suggesting that this region cannot be defined as a 3′-end-processing element, similar to that in yeast and plants.</p>                            <h3 scrollto-destination=6654958 id="6654958" class="section-title" data-legacy-id=s3e>3.5. Putative 3′-end-processing signals in A. oryzae</h3>
<p>Based on the information presented in this study, we proposed putative 3′-end-processing signals in <em>A. oryzae</em> (Fig. <a reveal-id="DSR011F5" data-open="DSR011F5" class="link link-reveal link-table xref-fig">5</a>). The putative 3′-end-processing signals in <em>A. oryzae</em> were similar to those in yeast and plants but some differences were observed between them. First, A-rich sequences upstream of the poly(A) site were less well conserved in all three species than in mammals, and the predominant hexanucleotide in this region of <em>A. oryzae</em> differed from that of yeast and plants. The canonical hexanucleotide AAUAAA signal in mammals is also the most frequently occurring signal in this 3′ UTR of yeast and plants, whereas it is found only in ∼13% and 7–10% of yeast and plant genes, respectively. In contrast, the most over-represented hexanucleotide in <em>A. oryzae</em> was AAUGAA, although this sequence accounted for only 6% of all transcripts, similar to yeast and plant AAUAAA sequences. Second, in the upstream of the A-rich region, while most dominant sequence motifs are well defined in yeast and plants (UAUAUA in yeast and UGUA in plants), no conserved sequence motif was observed in <em>A. oryzae</em>, except for the U-rich elements described earlier. </p>                        <a id="6654960" scrollto-destination="6654960"></a>
<div id="DSR011F5" data-id="DSR011F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr011/2/m_dsr01105.jpeg?Expires=2147483647&Signature=Sz-awBvy78EyesjnmB1xCO1Wg8QI4kHIZcaMzqubyvrNAgD-8lqPB9CMDXYwU2bdK67qYFTrhR3-7mkEwx~HqMK4Lt61hYN7~~FE4~g5i0xhcjlu3GHJPKKJGdUvU2rDH3RXquvNoPemY4tlwD5AfdogA~mVXgi-shHqZYOqMDjTNaB7UqObYqtOIcpNnCx9e2AgPeh1DCjlAvw4Doyxm~Ffr2hD01x4GcE9SYD~TXRIz-UsaAI0VAy8SmzOE95W9GxrZ4eFZTAg0DpNiHzqcV73H5XrsE303tDdeg-srFIrtAAtzJwW4C6igNFBGM31JT5w4wducRCa7t-HmOGyLg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A schematic representation of the alignment of 3′-end-processing signals in A. oryzae, yeast, and plants. The arrow indicates the cleavage and polyadenylation site." path-from-xml="dsr01105.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6654960/dsr01105.jpeg" path-from-xml="dsr01105.jpeg" target="_blank">View large</a><a section="6654960" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr011/2/dsr01105.jpeg?Expires=2147483647&Signature=AhhqqbjYenu0yJ~0fPZXUaSIgdNFhYQPpBn736fepjFbsnbfegpvVIN0tladUJIktCnrIn5sFSV7llIfpnPAd-joot18915dyxpAasZOZYG~edrBOSXFtuMjAjFoYUglY-zzT5YLsGMkueoSsmuhkTX-q75AoHKTHFKOjDmI7jpzEeR0C64GFndgmqwZaCziSFQEUhaG27JGKWiz96SbzNePPSlFWiUGE4wzcR-Ia48mcyDw0TlLfT3gEoT8oHicFL3-gYBqJlL5WIJU40AxZBIy4JGCqDnd5wkfyVnjbjQZpljloEezg94LLEJRAxpzKAwHZvplCxMYDhiPTO0BMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6654960&ar=445126&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01105.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A schematic representation of the alignment of 3′-end-processing signals in <em>A. oryzae</em>, yeast, and plants. The arrow indicates the cleavage and polyadenylation site.</p></div></div><div content-id="DSR011F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr011/2/m_dsr01105.jpeg?Expires=2147483647&Signature=Sz-awBvy78EyesjnmB1xCO1Wg8QI4kHIZcaMzqubyvrNAgD-8lqPB9CMDXYwU2bdK67qYFTrhR3-7mkEwx~HqMK4Lt61hYN7~~FE4~g5i0xhcjlu3GHJPKKJGdUvU2rDH3RXquvNoPemY4tlwD5AfdogA~mVXgi-shHqZYOqMDjTNaB7UqObYqtOIcpNnCx9e2AgPeh1DCjlAvw4Doyxm~Ffr2hD01x4GcE9SYD~TXRIz-UsaAI0VAy8SmzOE95W9GxrZ4eFZTAg0DpNiHzqcV73H5XrsE303tDdeg-srFIrtAAtzJwW4C6igNFBGM31JT5w4wducRCa7t-HmOGyLg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A schematic representation of the alignment of 3′-end-processing signals in A. oryzae, yeast, and plants. The arrow indicates the cleavage and polyadenylation site." path-from-xml="dsr01105.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6654960/dsr01105.jpeg" path-from-xml="dsr01105.jpeg" target="_blank">View large</a><a section="6654960" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr011/2/dsr01105.jpeg?Expires=2147483647&Signature=AhhqqbjYenu0yJ~0fPZXUaSIgdNFhYQPpBn736fepjFbsnbfegpvVIN0tladUJIktCnrIn5sFSV7llIfpnPAd-joot18915dyxpAasZOZYG~edrBOSXFtuMjAjFoYUglY-zzT5YLsGMkueoSsmuhkTX-q75AoHKTHFKOjDmI7jpzEeR0C64GFndgmqwZaCziSFQEUhaG27JGKWiz96SbzNePPSlFWiUGE4wzcR-Ia48mcyDw0TlLfT3gEoT8oHicFL3-gYBqJlL5WIJU40AxZBIy4JGCqDnd5wkfyVnjbjQZpljloEezg94LLEJRAxpzKAwHZvplCxMYDhiPTO0BMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6654960&ar=445126&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01105.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A schematic representation of the alignment of 3′-end-processing signals in <em>A. oryzae</em>, yeast, and plants. The arrow indicates the cleavage and polyadenylation site.</p></div></div><p>In a previous study, we showed that a cDNA of the mite <em>Dermatophagoides farinae</em>, known as Der f7, contains the AT-biased codon and therefore is prematurely polyadenylated within the coding region of <em>A. oryzae</em>. We also showed that codon optimization circumvents this premature polyadenylation.<sup><a reveal-id="DSR011C13" data-open="DSR011C13" class="link link-ref link-reveal xref-bibr">13</a></sup> The GC content of the native Der f7 open reading frame (ORF) was 37.8%, while that of the codon-optimized Der f7 ORF was 52.8%. Thus, A- and U-rich sequences within the coding region of native Der f7 cDNA were eliminated by codon optimization. The putative 3′-end-processing signals in <em>A. oryzae</em> deduced from this study supported that the A- and U-rich sequences present within the coding region of native Der f7 pre-mRNA were involved in incorrect 3′-end-processing. Although two AAUGAA sequences were present in the coding region of native Der f7 pre-mRNA, neither were located within the region located 10–30 nt upstream of the premature poly(A) sites.<sup><a reveal-id="DSR011C13" data-open="DSR011C13" class="link link-ref link-reveal xref-bibr">13</a></sup> This suggested that the AAUGAA sequence within the coding region of the AT-rich heterologous gene could not function by itself as an efficient 3′-end-processing signal in <em>A. oryzae</em>. The A- and U-rich sequences located upstream of the cleavage site might work co-operatively in 3′-end-processing. In future, whether the elimination of the top-ranked A-rich sequences within the coding region of heterologous genes results in the prevention of aberrant, premature transcription termination must be examined empirically.</p>                            <h3 scrollto-destination=6654962 id="6654962" class="section-title" data-legacy-id=s3f>3.6. Protein factors involved in the pre-mRNA 3′-end-processing machinery of A. oryzae</h3>
<p>The recognition mechanism of 3′-end-processing signals has been well studied in yeast and mammals, and a large number of protein factors, e.g. ∼14 proteins in mammals and ∼20 proteins in yeast, are required for 3′-end-processing.<sup><a reveal-id="DSR011C2" data-open="DSR011C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSR011C3" data-open="DSR011C3" class="link link-ref link-reveal xref-bibr">3</a></sup> To examine whether these factors involved in 3′-end-processing are conserved in <em>A. oryzae</em>, we searched for homologous proteins of 20 yeast polyadenylation factors in the <em>A. oryzae</em> genome (Table <a reveal-id="DSR011TB3" data-open="DSR011TB3" class="link link-reveal link-table xref-fig">3</a>). Most homologs of yeast polyadenylation factors, except for 3 factors (Ref2, Syc1, and Pti1), were found in the <em>A. oryzae</em> genome. These three factors are components of the cleavage and polyadenylation factor in yeast,<sup><a reveal-id="DSR011C44" data-open="DSR011C44" class="link link-ref link-reveal xref-bibr">44</a></sup> but no homologous proteins of these three factors are observed in plant and mammal genomes, suggesting that they are specific to yeast. In comparison with polyadenylation factors in human genomes, although no homologs of CFIm68 and CFIm59 were found in yeast, <em>A. oryzae</em>, and plant genomes, the homologue of CFIm25 was present in <em>A. oryzae</em> and plant genomes but not in the yeast genome. In contrast, the homologous protein (AO090001000725) of yeast Hrp1, reported to bind to RNA with specificity for the AU-rich efficiency element in yeast,<sup><a reveal-id="DSR011C45" data-open="DSR011C45" class="link link-ref link-reveal xref-bibr">45</a>,<a reveal-id="DSR011C46" data-open="DSR011C46" class="link link-ref link-reveal xref-bibr">46</a></sup> was found in the <em>A. oryzae</em> genome, whereas no Hrp1 homologue with higher similarity was found in plant and mammalian genomes. Furthermore, although homologs of human CstF-50 and CPSF73-II were present in the plant genome, these were not found in yeast and <em>A. oryzae</em> genomes. These observations suggested that the protein factors involved in the 3′-end-processing machinery of filamentous fungi resemble, in part, those of yeast and those of plants. This could be indicative of the evolutionary relationship between filamentous fungi, plants, and yeast. Some protein factors homologous to their counterparts in other organisms show differences in their RNA-binding specificity, positioning, and function. For example, while mammalian CPSF160 binds directly to the hexanucleotide AAUAAA signal, the homologue of yeast Yhh1 binds near the A-rich cleavage site but not the A-rich polyadenylation signal.<sup><a reveal-id="DSR011C47" data-open="DSR011C47" class="link link-ref link-reveal xref-bibr">47</a></sup> In this regard, because no sequence motif equivalent to the yeast AU-rich efficiency element or the mammalian UGUAN sequence was observed among the putative 3′-end-processing signals in <em>A. oryzae</em>, the RNA-binding specificity and positioning of the homologs Hrp1 and CFIm25 involved in 3′-end-processing in <em>A. oryzae</em> must be investigated. </p>                        <a id="6654964" scrollto-destination="6654964"></a>
<div content-id="DSR011TB3" class="table-modal"><div class="table-wrap"><div id="DSR011TB3" class="table-wrap-title" data-id="DSR011TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Comparison of protein factors involved in pre-mRNA 3′-end-processing between <em>Aspergillus oryzae</em>, yeast, plants, and human</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th><em>Aspergillus oryzae</em> </th><th><em>Saccharomyces cerevisiae</em> </th><th><em>Arabidopsis thaliana</em> </th><th><em>Homo sapiens</em> </th><th>BlastP score to yeast homologue </th><th>BlastP score to plant homologue </th><th>BlastP score to human homologue </th></tr></thead><tbody><tr><td> </td><td><strong>CFIB</strong> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>AO090001000725 </td><td>Hrp1 </td><td>None </td><td>None </td><td>3e−52 </td><td>— </td><td>— </td></tr><tr><td colspan="7"><hr /> </td></tr><tr><td> </td><td><strong>CFIA</strong> </td><td><strong>AtCstF</strong> </td><td><strong>CstF</strong> </td><td> </td><td> </td><td> </td></tr><tr><td>AO090003000655 </td><td>Rna14 </td><td>AT1G17760 (AtCstF77) </td><td>CstF77 </td><td>2e−69 </td><td>6e−40 </td><td>2e−46 </td></tr><tr><td>AO090011000789 </td><td>Rna15 </td><td>AT1G71800 (AtCstF64) </td><td>CstF64 </td><td>1e−12 </td><td>1e−19 </td><td>2e−35 </td></tr><tr><td>None </td><td>None </td><td>AT5G60940 (AtCstF50) </td><td>CstF50 </td><td>— </td><td>—– </td><td>— </td></tr><tr><td> </td><td> </td><td> </td><td><strong>CFIIm</strong> </td><td> </td><td> </td><td> </td></tr><tr><td>AO090026000698 </td><td>Clp1 </td><td>AT3G04680 (AtCLPS3) </td><td>hClp1 </td><td>9e−34 </td><td>4e−45 </td><td>9e−47 </td></tr><tr><td>AO090012001002 </td><td>Pcf11 </td><td>AT4G04885 (AtPCFS4) </td><td>hPcf11 </td><td>3e−22 </td><td>2e−15 </td><td>4e−18 </td></tr><tr><td colspan="7"><hr /> </td></tr><tr><td> </td><td><strong>CPF</strong> </td><td><strong>AtCPSF</strong> </td><td><strong>CPSF</strong> </td><td> </td><td> </td><td> </td></tr><tr><td>AO090103000017 </td><td>Yhh1 </td><td>AT5G51660 (AtCPSF160) </td><td>CPSF160 </td><td>3e−69 </td><td>3e−83 </td><td>e−108 </td></tr><tr><td>AO090005001277 </td><td>Ydh1 </td><td>AT5G23880 (AtCPSF100) </td><td>CPSF100 </td><td>5e−26 </td><td>6e−24 </td><td>2e−25 </td></tr><tr><td>AO090005001001 </td><td>Ysh1 </td><td>AT1G61010 (AtCPSF73-I) </td><td>CPSF73 </td><td>e−168 </td><td>e−140 </td><td>7e−155 </td></tr><tr><td>AO090005000813 </td><td>Yth1 </td><td>AT 1G30460 (AtCPSF30) </td><td>CPSF30 </td><td>2e−40 </td><td>5e−14 </td><td>5e−28 </td></tr><tr><td>AO080531000089<sup>a</sup> </td><td>Fip1 </td><td>AT5G58040 (AtFIPS5) </td><td>hFip1 </td><td>4e−10 </td><td>4e−06 </td><td>2e−11 </td></tr><tr><td>AO090011000862 </td><td>Pfs2 </td><td>AT5G13480 (AtFY) </td><td>hPfs2 (WDR33) </td><td>1e−82 </td><td>3e−89 </td><td>1e−90 </td></tr><tr><td>AO090103000067 </td><td>Pta1 </td><td>AT1G27595 (AtSYM2) </td><td>Symplekin </td><td>3e−21 </td><td>0.085 </td><td>7e−05 </td></tr><tr><td> </td><td> </td><td>AT5G01400 (AtSYM5) </td><td> </td><td> </td><td>— </td><td> </td></tr><tr><td>None </td><td>None </td><td>AT2G01730 (AtCPSF73-II) </td><td>CPSF73L </td><td>— </td><td>— </td><td>— </td></tr><tr><td>AO090005001504 </td><td>Ssu72 </td><td>AT1G73820 (Ssu72-like) </td><td>hSsu72 </td><td>4e−48 </td><td>1e−38 </td><td>3e−41 </td></tr><tr><td>AO090701000351 </td><td>Glc7 </td><td>AT2G39840 (AtPP1) </td><td>PP1α </td><td>e−157 </td><td>e−141 </td><td>1e−154 </td></tr><tr><td> </td><td> </td><td> </td><td>PP1β </td><td> </td><td> </td><td>1e−151 </td></tr><tr><td>None </td><td>Ref2 </td><td>None </td><td>None </td><td>— </td><td>— </td><td>— </td></tr><tr><td>AO090001000739 </td><td>Mpe1 </td><td>AT5G47430 </td><td>RBBP6 </td><td>1e−70 </td><td>2e−34 </td><td>6e−34 </td></tr><tr><td>None </td><td>Syc1 </td><td>None </td><td>None </td><td>— </td><td>— </td><td>— </td></tr><tr><td>AO090120000355 </td><td>Swd2 </td><td>AT5G14530 </td><td>WDR82 </td><td>2e−52 </td><td>5e−40 </td><td>3e−54 </td></tr><tr><td>None </td><td>Pti1 </td><td>None </td><td>None </td><td>— </td><td>— </td><td>— </td></tr><tr><td>AO090005001182 </td><td>Pap1 </td><td>AT1G17980 (AtPAPS1) </td><td>PAP </td><td>e−151 </td><td>e−103 </td><td>e−114 </td></tr><tr><td> </td><td> </td><td>AT2G25850 (AtPAPS2) </td><td> </td><td> </td><td>e−102 </td><td> </td></tr><tr><td> </td><td> </td><td>AT4G32850 (AtPAPS4) </td><td> </td><td> </td><td>e−101 </td><td> </td></tr><tr><td colspan="7"><hr /> </td></tr><tr><td> </td><td> </td><td> </td><td><strong>CFIm</strong> </td><td> </td><td> </td><td> </td></tr><tr><td>None </td><td>None </td><td>None </td><td>CFIm68 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>None </td><td>None </td><td>None </td><td>CFIm59 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>AO090003001316 </td><td>None </td><td>AT4G25550 (AtCFIS2) </td><td>CFIm25 </td><td>— </td><td>1e−60 </td><td>5e−60 </td></tr></tbody></table></div><div content-id="DSR011TB3" class="table-modal"><table><thead name="thead" align=""><tr><th><em>Aspergillus oryzae</em> </th><th><em>Saccharomyces cerevisiae</em> </th><th><em>Arabidopsis thaliana</em> </th><th><em>Homo sapiens</em> </th><th>BlastP score to yeast homologue </th><th>BlastP score to plant homologue </th><th>BlastP score to human homologue </th></tr></thead><tbody><tr><td> </td><td><strong>CFIB</strong> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>AO090001000725 </td><td>Hrp1 </td><td>None </td><td>None </td><td>3e−52 </td><td>— </td><td>— </td></tr><tr><td colspan="7"><hr /> </td></tr><tr><td> </td><td><strong>CFIA</strong> </td><td><strong>AtCstF</strong> </td><td><strong>CstF</strong> </td><td> </td><td> </td><td> </td></tr><tr><td>AO090003000655 </td><td>Rna14 </td><td>AT1G17760 (AtCstF77) </td><td>CstF77 </td><td>2e−69 </td><td>6e−40 </td><td>2e−46 </td></tr><tr><td>AO090011000789 </td><td>Rna15 </td><td>AT1G71800 (AtCstF64) </td><td>CstF64 </td><td>1e−12 </td><td>1e−19 </td><td>2e−35 </td></tr><tr><td>None </td><td>None </td><td>AT5G60940 (AtCstF50) </td><td>CstF50 </td><td>— </td><td>—– </td><td>— </td></tr><tr><td> </td><td> </td><td> </td><td><strong>CFIIm</strong> </td><td> </td><td> </td><td> </td></tr><tr><td>AO090026000698 </td><td>Clp1 </td><td>AT3G04680 (AtCLPS3) </td><td>hClp1 </td><td>9e−34 </td><td>4e−45 </td><td>9e−47 </td></tr><tr><td>AO090012001002 </td><td>Pcf11 </td><td>AT4G04885 (AtPCFS4) </td><td>hPcf11 </td><td>3e−22 </td><td>2e−15 </td><td>4e−18 </td></tr><tr><td colspan="7"><hr /> </td></tr><tr><td> </td><td><strong>CPF</strong> </td><td><strong>AtCPSF</strong> </td><td><strong>CPSF</strong> </td><td> </td><td> </td><td> </td></tr><tr><td>AO090103000017 </td><td>Yhh1 </td><td>AT5G51660 (AtCPSF160) </td><td>CPSF160 </td><td>3e−69 </td><td>3e−83 </td><td>e−108 </td></tr><tr><td>AO090005001277 </td><td>Ydh1 </td><td>AT5G23880 (AtCPSF100) </td><td>CPSF100 </td><td>5e−26 </td><td>6e−24 </td><td>2e−25 </td></tr><tr><td>AO090005001001 </td><td>Ysh1 </td><td>AT1G61010 (AtCPSF73-I) </td><td>CPSF73 </td><td>e−168 </td><td>e−140 </td><td>7e−155 </td></tr><tr><td>AO090005000813 </td><td>Yth1 </td><td>AT 1G30460 (AtCPSF30) </td><td>CPSF30 </td><td>2e−40 </td><td>5e−14 </td><td>5e−28 </td></tr><tr><td>AO080531000089<sup>a</sup> </td><td>Fip1 </td><td>AT5G58040 (AtFIPS5) </td><td>hFip1 </td><td>4e−10 </td><td>4e−06 </td><td>2e−11 </td></tr><tr><td>AO090011000862 </td><td>Pfs2 </td><td>AT5G13480 (AtFY) </td><td>hPfs2 (WDR33) </td><td>1e−82 </td><td>3e−89 </td><td>1e−90 </td></tr><tr><td>AO090103000067 </td><td>Pta1 </td><td>AT1G27595 (AtSYM2) </td><td>Symplekin </td><td>3e−21 </td><td>0.085 </td><td>7e−05 </td></tr><tr><td> </td><td> </td><td>AT5G01400 (AtSYM5) </td><td> </td><td> </td><td>— </td><td> </td></tr><tr><td>None </td><td>None </td><td>AT2G01730 (AtCPSF73-II) </td><td>CPSF73L </td><td>— </td><td>— </td><td>— </td></tr><tr><td>AO090005001504 </td><td>Ssu72 </td><td>AT1G73820 (Ssu72-like) </td><td>hSsu72 </td><td>4e−48 </td><td>1e−38 </td><td>3e−41 </td></tr><tr><td>AO090701000351 </td><td>Glc7 </td><td>AT2G39840 (AtPP1) </td><td>PP1α </td><td>e−157 </td><td>e−141 </td><td>1e−154 </td></tr><tr><td> </td><td> </td><td> </td><td>PP1β </td><td> </td><td> </td><td>1e−151 </td></tr><tr><td>None </td><td>Ref2 </td><td>None </td><td>None </td><td>— </td><td>— </td><td>— </td></tr><tr><td>AO090001000739 </td><td>Mpe1 </td><td>AT5G47430 </td><td>RBBP6 </td><td>1e−70 </td><td>2e−34 </td><td>6e−34 </td></tr><tr><td>None </td><td>Syc1 </td><td>None </td><td>None </td><td>— </td><td>— </td><td>— </td></tr><tr><td>AO090120000355 </td><td>Swd2 </td><td>AT5G14530 </td><td>WDR82 </td><td>2e−52 </td><td>5e−40 </td><td>3e−54 </td></tr><tr><td>None </td><td>Pti1 </td><td>None </td><td>None </td><td>— </td><td>— </td><td>— </td></tr><tr><td>AO090005001182 </td><td>Pap1 </td><td>AT1G17980 (AtPAPS1) </td><td>PAP </td><td>e−151 </td><td>e−103 </td><td>e−114 </td></tr><tr><td> </td><td> </td><td>AT2G25850 (AtPAPS2) </td><td> </td><td> </td><td>e−102 </td><td> </td></tr><tr><td> </td><td> </td><td>AT4G32850 (AtPAPS4) </td><td> </td><td> </td><td>e−101 </td><td> </td></tr><tr><td colspan="7"><hr /> </td></tr><tr><td> </td><td> </td><td> </td><td><strong>CFIm</strong> </td><td> </td><td> </td><td> </td></tr><tr><td>None </td><td>None </td><td>None </td><td>CFIm68 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>None </td><td>None </td><td>None </td><td>CFIm59 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>AO090003001316 </td><td>None </td><td>AT4G25550 (AtCFIS2) </td><td>CFIm25 </td><td>— </td><td>1e−60 </td><td>5e−60 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Protein factors involved in pre-mRNA 3′-end-processing in yeast, plants, and humans are based on the data described in the studies by Mandel <em>et al</em>.,<sup><a reveal-id="DSR011C2" data-open="DSR011C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Millevoi and Vagner,<sup><a reveal-id="DSR011C3" data-open="DSR011C3" class="link link-ref link-reveal xref-bibr">3</a></sup> and Hunt <em>et al</em>.<sup><a reveal-id="DSR011C48" data-open="DSR011C48" class="link link-ref link-reveal xref-bibr">48</a></sup></p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>Homologue of <em>A. oryzae</em> Fip1 was retrieved by searching the <em>A. oryzae</em> genome database deposited by the National Research Institute of Brewing, Japan (http://nribf2.nrib.go.jp/genome/blastscope.html).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6654964" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR011TB3" class="table-wrap-title" data-id="DSR011TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Comparison of protein factors involved in pre-mRNA 3′-end-processing between <em>Aspergillus oryzae</em>, yeast, plants, and human</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th><em>Aspergillus oryzae</em> </th><th><em>Saccharomyces cerevisiae</em> </th><th><em>Arabidopsis thaliana</em> </th><th><em>Homo sapiens</em> </th><th>BlastP score to yeast homologue </th><th>BlastP score to plant homologue </th><th>BlastP score to human homologue </th></tr></thead><tbody><tr><td> </td><td><strong>CFIB</strong> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>AO090001000725 </td><td>Hrp1 </td><td>None </td><td>None </td><td>3e−52 </td><td>— </td><td>— </td></tr><tr><td colspan="7"><hr /> </td></tr><tr><td> </td><td><strong>CFIA</strong> </td><td><strong>AtCstF</strong> </td><td><strong>CstF</strong> </td><td> </td><td> </td><td> </td></tr><tr><td>AO090003000655 </td><td>Rna14 </td><td>AT1G17760 (AtCstF77) </td><td>CstF77 </td><td>2e−69 </td><td>6e−40 </td><td>2e−46 </td></tr><tr><td>AO090011000789 </td><td>Rna15 </td><td>AT1G71800 (AtCstF64) </td><td>CstF64 </td><td>1e−12 </td><td>1e−19 </td><td>2e−35 </td></tr><tr><td>None </td><td>None </td><td>AT5G60940 (AtCstF50) </td><td>CstF50 </td><td>— </td><td>—– </td><td>— </td></tr><tr><td> </td><td> </td><td> </td><td><strong>CFIIm</strong> </td><td> </td><td> </td><td> </td></tr><tr><td>AO090026000698 </td><td>Clp1 </td><td>AT3G04680 (AtCLPS3) </td><td>hClp1 </td><td>9e−34 </td><td>4e−45 </td><td>9e−47 </td></tr><tr><td>AO090012001002 </td><td>Pcf11 </td><td>AT4G04885 (AtPCFS4) </td><td>hPcf11 </td><td>3e−22 </td><td>2e−15 </td><td>4e−18 </td></tr><tr><td colspan="7"><hr /> </td></tr><tr><td> </td><td><strong>CPF</strong> </td><td><strong>AtCPSF</strong> </td><td><strong>CPSF</strong> </td><td> </td><td> </td><td> </td></tr><tr><td>AO090103000017 </td><td>Yhh1 </td><td>AT5G51660 (AtCPSF160) </td><td>CPSF160 </td><td>3e−69 </td><td>3e−83 </td><td>e−108 </td></tr><tr><td>AO090005001277 </td><td>Ydh1 </td><td>AT5G23880 (AtCPSF100) </td><td>CPSF100 </td><td>5e−26 </td><td>6e−24 </td><td>2e−25 </td></tr><tr><td>AO090005001001 </td><td>Ysh1 </td><td>AT1G61010 (AtCPSF73-I) </td><td>CPSF73 </td><td>e−168 </td><td>e−140 </td><td>7e−155 </td></tr><tr><td>AO090005000813 </td><td>Yth1 </td><td>AT 1G30460 (AtCPSF30) </td><td>CPSF30 </td><td>2e−40 </td><td>5e−14 </td><td>5e−28 </td></tr><tr><td>AO080531000089<sup>a</sup> </td><td>Fip1 </td><td>AT5G58040 (AtFIPS5) </td><td>hFip1 </td><td>4e−10 </td><td>4e−06 </td><td>2e−11 </td></tr><tr><td>AO090011000862 </td><td>Pfs2 </td><td>AT5G13480 (AtFY) </td><td>hPfs2 (WDR33) </td><td>1e−82 </td><td>3e−89 </td><td>1e−90 </td></tr><tr><td>AO090103000067 </td><td>Pta1 </td><td>AT1G27595 (AtSYM2) </td><td>Symplekin </td><td>3e−21 </td><td>0.085 </td><td>7e−05 </td></tr><tr><td> </td><td> </td><td>AT5G01400 (AtSYM5) </td><td> </td><td> </td><td>— </td><td> </td></tr><tr><td>None </td><td>None </td><td>AT2G01730 (AtCPSF73-II) </td><td>CPSF73L </td><td>— </td><td>— </td><td>— </td></tr><tr><td>AO090005001504 </td><td>Ssu72 </td><td>AT1G73820 (Ssu72-like) </td><td>hSsu72 </td><td>4e−48 </td><td>1e−38 </td><td>3e−41 </td></tr><tr><td>AO090701000351 </td><td>Glc7 </td><td>AT2G39840 (AtPP1) </td><td>PP1α </td><td>e−157 </td><td>e−141 </td><td>1e−154 </td></tr><tr><td> </td><td> </td><td> </td><td>PP1β </td><td> </td><td> </td><td>1e−151 </td></tr><tr><td>None </td><td>Ref2 </td><td>None </td><td>None </td><td>— </td><td>— </td><td>— </td></tr><tr><td>AO090001000739 </td><td>Mpe1 </td><td>AT5G47430 </td><td>RBBP6 </td><td>1e−70 </td><td>2e−34 </td><td>6e−34 </td></tr><tr><td>None </td><td>Syc1 </td><td>None </td><td>None </td><td>— </td><td>— </td><td>— </td></tr><tr><td>AO090120000355 </td><td>Swd2 </td><td>AT5G14530 </td><td>WDR82 </td><td>2e−52 </td><td>5e−40 </td><td>3e−54 </td></tr><tr><td>None </td><td>Pti1 </td><td>None </td><td>None </td><td>— </td><td>— </td><td>— </td></tr><tr><td>AO090005001182 </td><td>Pap1 </td><td>AT1G17980 (AtPAPS1) </td><td>PAP </td><td>e−151 </td><td>e−103 </td><td>e−114 </td></tr><tr><td> </td><td> </td><td>AT2G25850 (AtPAPS2) </td><td> </td><td> </td><td>e−102 </td><td> </td></tr><tr><td> </td><td> </td><td>AT4G32850 (AtPAPS4) </td><td> </td><td> </td><td>e−101 </td><td> </td></tr><tr><td colspan="7"><hr /> </td></tr><tr><td> </td><td> </td><td> </td><td><strong>CFIm</strong> </td><td> </td><td> </td><td> </td></tr><tr><td>None </td><td>None </td><td>None </td><td>CFIm68 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>None </td><td>None </td><td>None </td><td>CFIm59 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>AO090003001316 </td><td>None </td><td>AT4G25550 (AtCFIS2) </td><td>CFIm25 </td><td>— </td><td>1e−60 </td><td>5e−60 </td></tr></tbody></table></div><div content-id="DSR011TB3" class="table-modal"><table><thead name="thead" align=""><tr><th><em>Aspergillus oryzae</em> </th><th><em>Saccharomyces cerevisiae</em> </th><th><em>Arabidopsis thaliana</em> </th><th><em>Homo sapiens</em> </th><th>BlastP score to yeast homologue </th><th>BlastP score to plant homologue </th><th>BlastP score to human homologue </th></tr></thead><tbody><tr><td> </td><td><strong>CFIB</strong> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>AO090001000725 </td><td>Hrp1 </td><td>None </td><td>None </td><td>3e−52 </td><td>— </td><td>— </td></tr><tr><td colspan="7"><hr /> </td></tr><tr><td> </td><td><strong>CFIA</strong> </td><td><strong>AtCstF</strong> </td><td><strong>CstF</strong> </td><td> </td><td> </td><td> </td></tr><tr><td>AO090003000655 </td><td>Rna14 </td><td>AT1G17760 (AtCstF77) </td><td>CstF77 </td><td>2e−69 </td><td>6e−40 </td><td>2e−46 </td></tr><tr><td>AO090011000789 </td><td>Rna15 </td><td>AT1G71800 (AtCstF64) </td><td>CstF64 </td><td>1e−12 </td><td>1e−19 </td><td>2e−35 </td></tr><tr><td>None </td><td>None </td><td>AT5G60940 (AtCstF50) </td><td>CstF50 </td><td>— </td><td>—– </td><td>— </td></tr><tr><td> </td><td> </td><td> </td><td><strong>CFIIm</strong> </td><td> </td><td> </td><td> </td></tr><tr><td>AO090026000698 </td><td>Clp1 </td><td>AT3G04680 (AtCLPS3) </td><td>hClp1 </td><td>9e−34 </td><td>4e−45 </td><td>9e−47 </td></tr><tr><td>AO090012001002 </td><td>Pcf11 </td><td>AT4G04885 (AtPCFS4) </td><td>hPcf11 </td><td>3e−22 </td><td>2e−15 </td><td>4e−18 </td></tr><tr><td colspan="7"><hr /> </td></tr><tr><td> </td><td><strong>CPF</strong> </td><td><strong>AtCPSF</strong> </td><td><strong>CPSF</strong> </td><td> </td><td> </td><td> </td></tr><tr><td>AO090103000017 </td><td>Yhh1 </td><td>AT5G51660 (AtCPSF160) </td><td>CPSF160 </td><td>3e−69 </td><td>3e−83 </td><td>e−108 </td></tr><tr><td>AO090005001277 </td><td>Ydh1 </td><td>AT5G23880 (AtCPSF100) </td><td>CPSF100 </td><td>5e−26 </td><td>6e−24 </td><td>2e−25 </td></tr><tr><td>AO090005001001 </td><td>Ysh1 </td><td>AT1G61010 (AtCPSF73-I) </td><td>CPSF73 </td><td>e−168 </td><td>e−140 </td><td>7e−155 </td></tr><tr><td>AO090005000813 </td><td>Yth1 </td><td>AT 1G30460 (AtCPSF30) </td><td>CPSF30 </td><td>2e−40 </td><td>5e−14 </td><td>5e−28 </td></tr><tr><td>AO080531000089<sup>a</sup> </td><td>Fip1 </td><td>AT5G58040 (AtFIPS5) </td><td>hFip1 </td><td>4e−10 </td><td>4e−06 </td><td>2e−11 </td></tr><tr><td>AO090011000862 </td><td>Pfs2 </td><td>AT5G13480 (AtFY) </td><td>hPfs2 (WDR33) </td><td>1e−82 </td><td>3e−89 </td><td>1e−90 </td></tr><tr><td>AO090103000067 </td><td>Pta1 </td><td>AT1G27595 (AtSYM2) </td><td>Symplekin </td><td>3e−21 </td><td>0.085 </td><td>7e−05 </td></tr><tr><td> </td><td> </td><td>AT5G01400 (AtSYM5) </td><td> </td><td> </td><td>— </td><td> </td></tr><tr><td>None </td><td>None </td><td>AT2G01730 (AtCPSF73-II) </td><td>CPSF73L </td><td>— </td><td>— </td><td>— </td></tr><tr><td>AO090005001504 </td><td>Ssu72 </td><td>AT1G73820 (Ssu72-like) </td><td>hSsu72 </td><td>4e−48 </td><td>1e−38 </td><td>3e−41 </td></tr><tr><td>AO090701000351 </td><td>Glc7 </td><td>AT2G39840 (AtPP1) </td><td>PP1α </td><td>e−157 </td><td>e−141 </td><td>1e−154 </td></tr><tr><td> </td><td> </td><td> </td><td>PP1β </td><td> </td><td> </td><td>1e−151 </td></tr><tr><td>None </td><td>Ref2 </td><td>None </td><td>None </td><td>— </td><td>— </td><td>— </td></tr><tr><td>AO090001000739 </td><td>Mpe1 </td><td>AT5G47430 </td><td>RBBP6 </td><td>1e−70 </td><td>2e−34 </td><td>6e−34 </td></tr><tr><td>None </td><td>Syc1 </td><td>None </td><td>None </td><td>— </td><td>— </td><td>— </td></tr><tr><td>AO090120000355 </td><td>Swd2 </td><td>AT5G14530 </td><td>WDR82 </td><td>2e−52 </td><td>5e−40 </td><td>3e−54 </td></tr><tr><td>None </td><td>Pti1 </td><td>None </td><td>None </td><td>— </td><td>— </td><td>— </td></tr><tr><td>AO090005001182 </td><td>Pap1 </td><td>AT1G17980 (AtPAPS1) </td><td>PAP </td><td>e−151 </td><td>e−103 </td><td>e−114 </td></tr><tr><td> </td><td> </td><td>AT2G25850 (AtPAPS2) </td><td> </td><td> </td><td>e−102 </td><td> </td></tr><tr><td> </td><td> </td><td>AT4G32850 (AtPAPS4) </td><td> </td><td> </td><td>e−101 </td><td> </td></tr><tr><td colspan="7"><hr /> </td></tr><tr><td> </td><td> </td><td> </td><td><strong>CFIm</strong> </td><td> </td><td> </td><td> </td></tr><tr><td>None </td><td>None </td><td>None </td><td>CFIm68 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>None </td><td>None </td><td>None </td><td>CFIm59 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>AO090003001316 </td><td>None </td><td>AT4G25550 (AtCFIS2) </td><td>CFIm25 </td><td>— </td><td>1e−60 </td><td>5e−60 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Protein factors involved in pre-mRNA 3′-end-processing in yeast, plants, and humans are based on the data described in the studies by Mandel <em>et al</em>.,<sup><a reveal-id="DSR011C2" data-open="DSR011C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Millevoi and Vagner,<sup><a reveal-id="DSR011C3" data-open="DSR011C3" class="link link-ref link-reveal xref-bibr">3</a></sup> and Hunt <em>et al</em>.<sup><a reveal-id="DSR011C48" data-open="DSR011C48" class="link link-ref link-reveal xref-bibr">48</a></sup></p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>Homologue of <em>A. oryzae</em> Fip1 was retrieved by searching the <em>A. oryzae</em> genome database deposited by the National Research Institute of Brewing, Japan (http://nribf2.nrib.go.jp/genome/blastscope.html).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6654964" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=6654965 id="6654965" class="section-title" data-legacy-id=s3g>3.7. Conclusions</h3>
<p>In this study, we identified putative 3′-end-processing signals in <em>A. oryzae</em> using EST and genomic sequencing data. The putative 3′-end-processing signals in <em>A. oryzae</em> identified in this study comprised four elements: the furthest upstream U-rich element; A-rich sequence element (the most dominant sequence being AAUGAA); cleavage site (the most dominant sequence being CA); and U-rich element flanking the cleavage site. Although these putative 3′-end-processing signals in <em>A. oryzae</em> were similar to those found in yeast and plants, obvious differences were observed in the furthest upstream element and A-rich sequence element. To our knowledge, this is the first study of 3′-end-processing signals in filamentous fungi, and we believe that the data presented in this paper will provide knowledge critically important to the understanding of pre-mRNA 3′-end-processing in eukaryotes. In addition, this study also provides useful information on codon optimization of heterologous genes to prevent aberrant, premature polyadenylation within the coding region of filamentous fungi.</p>                            <h2 scrollto-destination=6654967 id="6654967" class="section-title" data-legacy-id=s4>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=6654969 id="6654969" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was supported, in part, by a Grant-in-Aid for Scientific Research on Priority Areas, Applied Genomics (no. 17019001), from the Ministry of Education, Culture, Sports, Science, and Technology of Japan. M. T. is a JSPS Research Fellow.</p>                            <h2 scrollto-destination=6654973 id="6654973" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We are grateful to Dr Sachiko Shiro-Hasegawa for DNA microarray analysis.</p>                            <h2 scrollto-destination=6654977 id="6654977" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSR011C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhao</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Hyman</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Moore</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Formation of mRNA 3′ ends in eukaryotes: mechanism, regulation, and interrelationships with other steps in mRNA synthesis</div>, <div class="source">Microbiol. Mol. Biol. Rev.</div> , <div class="year">1999</div>, vol. <div class="volume">63</div> (pg. <div class="fpage">405</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Formation%20of%20mRNA%203%E2%80%B2%20ends%20in%20eukaryotes%3A%20mechanism%2C%20regulation%2C%20and%20interrelationships%20with%20other%20steps%20in%20mRNA%20synthesis&amp;author=J.%20Zhao&amp;author=L.%20Hyman&amp;author=C.%20Moore&amp;publication_year=1999&amp;journal=Microbiol.%20Mol.%20Biol.%20Rev.&amp;volume=63&amp;pages=405-45" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10357856" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mandel</div> <div class="given-names">C.R.</div></div>,  <div class="name"><div class="surname">Bai</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Tong</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Protein factors in pre-mRNA 3′-end processing</div>, <div class="source">Cell. Mol. Life Sci.</div> , <div class="year">2008</div>, vol. <div class="volume">65</div> (pg. <div class="fpage">1099</div>-<div class="lpage">122</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protein%20factors%20in%20pre-mRNA%203%E2%80%B2-end%20processing&amp;author=C.R.%20Mandel&amp;author=Y.%20Bai&amp;author=L.%20Tong&amp;publication_year=2008&amp;journal=Cell.%20Mol.%20Life%20Sci.&amp;volume=65&amp;pages=1099-122" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00018-007-7474-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00018-007-7474-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00018-007-7474-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18158581" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Millevoi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Vagner</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Molecular mechanisms of eukaryotic pre-mRNA 3′ end processing regulation</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2010</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">2757</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20mechanisms%20of%20eukaryotic%20pre-mRNA%203%E2%80%B2%20end%20processing%20regulation&amp;author=S.%20Millevoi&amp;author=S.%20Vagner&amp;publication_year=2010&amp;journal=Nucleic%20Acids%20Res.&amp;volume=38&amp;pages=2757-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkp1176" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkp1176" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkp1176"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20044349" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Graber</div> <div class="given-names">J.H.</div></div>,  <div class="name"><div class="surname">Cantor</div> <div class="given-names">C.R.</div></div>,  <div class="name"><div class="surname">Mohr</div> <div class="given-names">S.C.</div></div>,  <div class="name"><div class="surname">Smith</div> <div class="given-names">T.F.</div></div>. </div><div class="article-title"><em>In slico</em> detection of control signals: mRNA 3′-end-processing sequences in diverse species</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1999</div>, vol. <div class="volume">96</div> (pg. <div class="fpage">14055</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20slico%20detection%20of%20control%20signals%3A%20mRNA%203%E2%80%B2-end-processing%20sequences%20in%20diverse%20species&amp;author=J.H.%20Graber&amp;author=C.R.%20Cantor&amp;author=S.C.%20Mohr&amp;author=T.F.%20Smith&amp;publication_year=1999&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=96&amp;pages=14055-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.96.24.14055" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.96.24.14055" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.96.24.14055"> </span></div> </div></div></div></div></div><div content-id="DSR011C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Graber</div> <div class="given-names">J.H.</div></div>,  <div class="name"><div class="surname">Cantor</div> <div class="given-names">C.R.</div></div>,  <div class="name"><div class="surname">Mohr</div> <div class="given-names">S.C.</div></div>,  <div class="name"><div class="surname">Smith</div> <div class="given-names">T.F.</div></div>. </div><div class="article-title">Genomic detection of new yeast pre-mRNA 3′-end-processing signals</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1999</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">888</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20detection%20of%20new%20yeast%20pre-mRNA%203%E2%80%B2-end-processing%20signals&amp;author=J.H.%20Graber&amp;author=C.R.%20Cantor&amp;author=S.C.%20Mohr&amp;author=T.F.%20Smith&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=888-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.3.888" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.3.888" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.3.888"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9889288" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Lutz</div> <div class="given-names">C.S.</div></div>,  <div class="name"><div class="surname">Wilusz</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Tian</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Bioinformatic identification of candidate <em>cis</em>-regulatory elements involved in human mRNA polyadenylation</div>, <div class="source">RNA</div> , <div class="year">2005</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">1485</div>-<div class="lpage">93</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bioinformatic%20identification%20of%20candidate%20cis-regulatory%20elements%20involved%20in%20human%20mRNA%20polyadenylation&amp;author=J.%20Hu&amp;author=C.S.%20Lutz&amp;author=J.%20Wilusz&amp;author=B.%20Tian&amp;publication_year=2005&amp;journal=RNA&amp;volume=11&amp;pages=1485-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.2107305" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.2107305" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.2107305"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16131587" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Loke</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Stahlberg</div> <div class="given-names">E.A.</div></div>,  <div class="name"><div class="surname">Stenski</div> <div class="given-names">D.G.</div></div>,  <div class="name"><div class="surname">Haas</div> <div class="given-names">B.J.</div></div>,  <div class="name"><div class="surname">Wood</div> <div class="given-names">P.C.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">Q.Q.</div></div>. </div><div class="article-title">Complication of mRNA polyadenylation signals in <em>Arabidopsis</em> revealed a new signal element and potential secondary structures</div>, <div class="source">Plant Physiol.</div> , <div class="year">2005</div>, vol. <div class="volume">138</div> (pg. <div class="fpage">1457</div>-<div class="lpage">68</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complication%20of%20mRNA%20polyadenylation%20signals%20in%20Arabidopsis%20revealed%20a%20new%20signal%20element%20and%20potential%20secondary%20structures&amp;author=J.C.%20Loke&amp;author=E.A.%20Stahlberg&amp;author=D.G.%20Stenski&amp;author=B.J.%20Haas&amp;author=P.C.%20Wood&amp;author=Q.Q.%20Li&amp;publication_year=2005&amp;journal=Plant%20Physiol.&amp;volume=138&amp;pages=1457-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.105.060541" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.105.060541" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.105.060541"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15965016" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shen</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ji</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Hass</div> <div class="given-names">B.J.</div></div>, et al. </div><div class="article-title">Genome level analysis of rice mRNA 3′-end processing signals and alternative polyadenylation</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">3150</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20level%20analysis%20of%20rice%20mRNA%203%E2%80%B2-end%20processing%20signals%20and%20alternative%20polyadenylation&amp;author=Y.%20Shen&amp;author=G.%20Ji&amp;author=B.J.%20Hass&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=3150-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn158" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn158" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn158"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18411206" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Machida</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Yamada</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Gomi</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Genomics of <em>Aspergillus oryzae</em>: learning from the history of koji mold and exploration of its future</div>, <div class="source">DNA Res.</div> , <div class="year">2008</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">173</div>-<div class="lpage">83</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomics%20of%20Aspergillus%20oryzae%3A%20learning%20from%20the%20history%20of%20koji%20mold%20and%20exploration%20of%20its%20future&amp;author=M.%20Machida&amp;author=O.%20Yamada&amp;author=K.%20Gomi&amp;publication_year=2008&amp;journal=DNA%20Res.&amp;volume=15&amp;pages=173-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsn020" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsn020" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsn020"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18820080" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Christensen</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Woeldike</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Boel</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">High level expression of recombinant genes in <em>Aspergillus oryzae</em></div>, <div class="source">Bio/Technology</div> , <div class="year">1988</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">1419</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High%20level%20expression%20of%20recombinant%20genes%20in%20Aspergillus%20oryzae&amp;author=T.%20Christensen&amp;author=H.%20Woeldike&amp;author=F.%20Boel&amp;publication_year=1988&amp;journal=Bio%2FTechnology&amp;volume=6&amp;pages=1419-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1288-1419" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1288-1419" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1288-1419"> </span></div> </div></div></div></div></div><div content-id="DSR011C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lubertozzi</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Keasling</div> <div class="given-names">J.D.</div></div>. </div><div class="article-title">Developing <em>Aspergillus</em> as a host for heterologous expression</div>, <div class="source">Biotechnol. Adv.</div> , <div class="year">2009</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">53</div>-<div class="lpage">75</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Developing%20Aspergillus%20as%20a%20host%20for%20heterologous%20expression&amp;author=D.%20Lubertozzi&amp;author=J.D.%20Keasling&amp;publication_year=2009&amp;journal=Biotechnol.%20Adv.&amp;volume=27&amp;pages=53-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.biotechadv.2008.09.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.biotechadv.2008.09.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.biotechadv.2008.09.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18840517" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gouka</div> <div class="given-names">R.J.</div></div>,  <div class="name"><div class="surname">Punt</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">van den Hondel</div> <div class="given-names">C.A.M.J.J.</div></div>. </div><div class="article-title">Efficient production of secreted proteins by <em>Aspergillus</em>: progress, limitations and prospects</div>, <div class="source">Appl. Microbiol. Biotechnol.</div> , <div class="year">1997</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">1</div>-<div class="lpage">11</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efficient%20production%20of%20secreted%20proteins%20by%20Aspergillus%3A%20progress%2C%20limitations%20and%20prospects&amp;author=R.J.%20Gouka&amp;author=P.J.%20Punt&amp;author=C.A.M.J.J.%20van%20den%20Hondel&amp;publication_year=1997&amp;journal=Appl.%20Microbiol.%20Biotechnol.&amp;volume=47&amp;pages=1-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s002530050880" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs002530050880" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs002530050880"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9035405" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tokuoka</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tanaka</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ono</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Takagi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Shintani</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Gomi</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Codon optimization increases steady-state mRNA levels in <em>Aspergillus oryzae</em> heterologous gene expression</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">2008</div>, vol. <div class="volume">74</div> (pg. <div class="fpage">6538</div>-<div class="lpage">46</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20optimization%20increases%20steady-state%20mRNA%20levels%20in%20Aspergillus%20oryzae%20heterologous%20gene%20expression&amp;author=M.%20Tokuoka&amp;author=M.%20Tanaka&amp;author=K.%20Ono&amp;author=S.%20Takagi&amp;author=T.%20Shintani&amp;author=K.%20Gomi&amp;publication_year=2008&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=74&amp;pages=6538-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.01354-08" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.01354-08" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.01354-08"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18791013" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Akao</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sano</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Yamada</div> <div class="given-names">O.</div></div>, et al. </div><div class="article-title">Analysis of expressed sequence tags from the fungus <em>Aspergillus oryzae</em> cultured under different conditions</div>, <div class="source">DNA Res.</div> , <div class="year">2007</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">47</div>-<div class="lpage">57</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20expressed%20sequence%20tags%20from%20the%20fungus%20Aspergillus%20oryzae%20cultured%20under%20different%20conditions&amp;author=T.%20Akao&amp;author=M.%20Sano&amp;author=O.%20Yamada&amp;publication_year=2007&amp;journal=DNA%20Res.&amp;volume=14&amp;pages=47-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsm008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsm008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsm008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17540709" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Machida</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Asai</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Sano</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Genome sequencing and analysis of <em>Aspergillus oryzae</em></div>, <div class="source">Nature</div> , <div class="year">2005</div>, vol. <div class="volume">438</div> (pg. <div class="fpage">1157</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequencing%20and%20analysis%20of%20Aspergillus%20oryzae&amp;author=M.%20Machida&amp;author=K.%20Asai&amp;author=M.%20Sano&amp;publication_year=2005&amp;journal=Nature&amp;volume=438&amp;pages=1157-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature04300" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature04300" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature04300"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16372010" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shen</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Ling</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">Q.Q.</div></div>. </div><div class="article-title">Unique features of nuclear mRNA poly(A) signals and alternative polyadenylation in <em>Chlamydomonas reinhardtii</em></div>, <div class="source">Genetics</div> , <div class="year">2008</div>, vol. <div class="volume">179</div> (pg. <div class="fpage">167</div>-<div class="lpage">76</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unique%20features%20of%20nuclear%20mRNA%20poly(A)%20signals%20and%20alternative%20polyadenylation%20in%20Chlamydomonas%20reinhardtii&amp;author=Y.%20Shen&amp;author=Y.%20Liu&amp;author=L.%20Liu&amp;author=C.%20Ling&amp;author=Q.Q.%20Li&amp;publication_year=2008&amp;journal=Genetics&amp;volume=179&amp;pages=167-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.108.088971" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.108.088971" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.108.088971"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18493049" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moore</div> <div class="given-names">C.L.</div></div>,  <div class="name"><div class="surname">Skolnikdavid</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Sharp</div> <div class="given-names">P.A.</div></div>. </div><div class="article-title">Analysis of RNA cleavage at the Adenovirus-2 L3 polyadenylation site</div>, <div class="source">EMBO J.</div> , <div class="year">1986</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">1929</div>-<div class="lpage">38</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20RNA%20cleavage%20at%20the%20Adenovirus-2%20L3%20polyadenylation%20site&amp;author=C.L.%20Moore&amp;author=H.%20Skolnikdavid&amp;author=P.A.%20Sharp&amp;publication_year=1986&amp;journal=EMBO%20J.&amp;volume=5&amp;pages=1929-38" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3019671" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sheets</div> <div class="given-names">M.D.</div></div>,  <div class="name"><div class="surname">Ogg</div> <div class="given-names">S.C.</div></div>,  <div class="name"><div class="surname">Wickens</div> <div class="given-names">M.P.</div></div>. </div><div class="article-title">Point mutations in AAUAAA and the poly(A) addition site: effects on the accuracy and efficiency of cleavage and polyadenylation <em>in vitro</em></div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1990</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">5799</div>-<div class="lpage">805</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Point%20mutations%20in%20AAUAAA%20and%20the%20poly(A)%20addition%20site%3A%20effects%20on%20the%20accuracy%20and%20efficiency%20of%20cleavage%20and%20polyadenylation%20in%20vitro&amp;author=M.D.%20Sheets&amp;author=S.C.%20Ogg&amp;author=M.P.%20Wickens&amp;publication_year=1990&amp;journal=Nucleic%20Acids%20Res.&amp;volume=18&amp;pages=5799-805" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/18.19.5799" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F18.19.5799" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F18.19.5799"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2170946" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Macdonald</div> <div class="given-names">C.C.</div></div>,  <div class="name"><div class="surname">Wilusz</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Cleavage site determinants in the mammalian polyadenylation signal</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1995</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">2614</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cleavage%20site%20determinants%20in%20the%20mammalian%20polyadenylation%20signal&amp;author=F.%20Chen&amp;author=C.C.%20Macdonald&amp;author=J.%20Wilusz&amp;publication_year=1995&amp;journal=Nucleic%20Acids%20Res.&amp;volume=23&amp;pages=2614-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/23.14.2614" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F23.14.2614" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F23.14.2614"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7651822" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tamano</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Sano</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Yamane</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Transcriptional regulation of genes on the non-syntenic blocks of <em>Aspergillus oryzae</em> and its functional relationship to solid-state cultivation</div>, <div class="source">Fungal Genet. Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">45</div> (pg. <div class="fpage">139</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20regulation%20of%20genes%20on%20the%20non-syntenic%20blocks%20of%20Aspergillus%20oryzae%20and%20its%20functional%20relationship%20to%20solid-state%20cultivation&amp;author=K.%20Tamano&amp;author=M.%20Sano&amp;author=N.%20Yamane&amp;publication_year=2008&amp;journal=Fungal%20Genet.%20Biol.&amp;volume=45&amp;pages=139-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fgb.2007.09.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fgb.2007.09.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fgb.2007.09.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17967552" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Workman</div> <div class="given-names">C.T.</div></div>,  <div class="name"><div class="surname">Yin</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Corcoran</div> <div class="given-names">D.L.</div></div>,  <div class="name"><div class="surname">Ideker</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Stormo</div> <div class="given-names">G.D.</div></div>,  <div class="name"><div class="surname">Benos</div> <div class="given-names">P.V.</div></div>. </div><div class="article-title">enoLOGOS: a versatile web tool for energy normalized sequence logos</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">W389</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=enoLOGOS%3A%20a%20versatile%20web%20tool%20for%20energy%20normalized%20sequence%20logos&amp;author=C.T.%20Workman&amp;author=Y.%20Yin&amp;author=D.L.%20Corcoran&amp;author=T.%20Ideker&amp;author=G.D.%20Stormo&amp;author=P.V.%20Benos&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=W389-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki439" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki439" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki439"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15980495" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Van Helden</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Del Olmo</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Perez-Ortin</div> <div class="given-names">J.E.</div></div>. </div><div class="article-title">Statistical analysis of yeast genomic downstream sequences reveals putative polyadenylation signals</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2000</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">1000</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Statistical%20analysis%20of%20yeast%20genomic%20downstream%20sequences%20reveals%20putative%20polyadenylation%20signals&amp;author=J.%20Van%20Helden&amp;author=M.%20Del%20Olmo&amp;author=J.E.%20Perez-Ortin&amp;publication_year=2000&amp;journal=Nucleic%20Acids%20Res.&amp;volume=28&amp;pages=1000-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/28.4.1000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F28.4.1000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F28.4.1000"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10648794" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tatusov</div> <div class="given-names">R.L.</div></div>,  <div class="name"><div class="surname">Federova</div> <div class="given-names">N.D.</div></div>,  <div class="name"><div class="surname">Jackson</div> <div class="given-names">J.D.</div></div>, et al. </div><div class="article-title">The COG database: an updated version includes eukaryotes</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2003</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">41</div>-<div class="lpage">54</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20COG%20database%3A%20an%20updated%20version%20includes%20eukaryotes&amp;author=R.L.%20Tatusov&amp;author=N.D.%20Federova&amp;author=J.D.%20Jackson&amp;publication_year=2003&amp;journal=BMC%20Bioinformatics&amp;volume=4&amp;pages=41-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-4-41" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-4-41" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-4-41"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12969510" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tian</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Hu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Lutz</div> <div class="given-names">C.S.</div></div>. </div><div class="article-title">A large-scale analysis of mRNA polyadenylation of human and mouse genes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">201</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20large-scale%20analysis%20of%20mRNA%20polyadenylation%20of%20human%20and%20mouse%20genes&amp;author=B.%20Tian&amp;author=J.%20Hu&amp;author=H.%20Zhang&amp;author=C.S.%20Lutz&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=201-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki158" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki158" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki158"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15647503" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Edwalds-Gilbert</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Veraldi</div> <div class="given-names">K.L.</div></div>,  <div class="name"><div class="surname">Milcarek</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Alternative poly(A) site selection in complex transcription units: means to an end?</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">2547</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20poly(A)%20site%20selection%20in%20complex%20transcription%20units%3A%20means%20to%20an%20end%3F&amp;author=G.%20Edwalds-Gilbert&amp;author=K.L.%20Veraldi&amp;author=C.%20Milcarek&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=2547-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.13.2547" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.13.2547" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.13.2547"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9185563" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pesole</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Mignone</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Gissi</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Grillo</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Licciulli</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Liuni</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Structural and functional features of eukaryotic mRNA untranslated regions</div>, <div class="source">Gene</div> , <div class="year">2001</div>, vol. <div class="volume">276</div> (pg. <div class="fpage">73</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20and%20functional%20features%20of%20eukaryotic%20mRNA%20untranslated%20regions&amp;author=G.%20Pesole&amp;author=F.%20Mignone&amp;author=C.%20Gissi&amp;author=G.%20Grillo&amp;author=F.%20Licciulli&amp;author=S.%20Liuni&amp;publication_year=2001&amp;journal=Gene&amp;volume=276&amp;pages=73-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(01)00674-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(01)00674-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(01)00674-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11591473" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kuersten</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Goodwin</div> <div class="given-names">E.B.</div></div>. </div><div class="article-title">The power of the 3′ UTR: translational control and development</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2003</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">626</div>-<div class="lpage">37</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20power%20of%20the%203%E2%80%B2%20UTR%3A%20translational%20control%20and%20development&amp;author=S.%20Kuersten&amp;author=E.B.%20Goodwin&amp;publication_year=2003&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=4&amp;pages=626-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1125" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1125" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1125"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12897774" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mazumder</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Seshadri</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Fox</div> <div class="given-names">P.L.</div></div>. </div><div class="article-title">Translational control by the 3′-UTR: the ends specify the means</div>, <div class="source">Trends Biochem. Sci.</div> , <div class="year">2003</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">91</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translational%20control%20by%20the%203%E2%80%B2-UTR%3A%20the%20ends%20specify%20the%20means&amp;author=B.%20Mazumder&amp;author=V.%20Seshadri&amp;author=P.L.%20Fox&amp;publication_year=2003&amp;journal=Trends%20Biochem.%20Sci.&amp;volume=28&amp;pages=91-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0968-0004(03)00002-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0968-0004(03)00002-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0968-0004(03)00002-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12575997" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kozak</div> <div class="given-names">M.</div></div>. </div><div class="article-title">How strong is the case for regulation of the initiation step of translation by elements at the 3′ end of eukaryotic mRNAs?</div>, <div class="source">Gene</div> , <div class="year">2004</div>, vol. <div class="volume">343</div> (pg. <div class="fpage">41</div>-<div class="lpage">54</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=How%20strong%20is%20the%20case%20for%20regulation%20of%20the%20initiation%20step%20of%20translation%20by%20elements%20at%20the%203%E2%80%B2%20end%20of%20eukaryotic%20mRNAs%3F&amp;author=M.%20Kozak&amp;publication_year=2004&amp;journal=Gene&amp;volume=343&amp;pages=41-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2004.08.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2004.08.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2004.08.011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15563830" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bartel</div> <div class="given-names">D.P.</div></div>. </div><div class="article-title">MicroRNAs: target recognition and regulatory functions</div>, <div class="source">Cell</div> , <div class="year">2009</div>, vol. <div class="volume">136</div> (pg. <div class="fpage">215</div>-<div class="lpage">233</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MicroRNAs%3A%20target%20recognition%20and%20regulatory%20functions&amp;author=D.P.%20Bartel&amp;publication_year=2009&amp;journal=Cell&amp;volume=136&amp;pages=215-233" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2009.01.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2009.01.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2009.01.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19167326" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kebaara</div> <div class="given-names">B.W.</div></div>,  <div class="name"><div class="surname">Atkin</div> <div class="given-names">A.L.</div></div>. </div><div class="article-title">Long 3′-UTRs target wild-type mRNAs for nonsense-mediated mRNA decay in <em>Saccharomyces cerevisiae</em></div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">2771</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Long%203%E2%80%B2-UTRs%20target%20wild-type%20mRNAs%20for%20nonsense-mediated%20mRNA%20decay%20in%20Saccharomyces%20cerevisiae&amp;author=B.W.%20Kebaara&amp;author=A.L.%20Atkin&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=2771-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkp146" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkp146" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkp146"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19270062" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Morozov</div> <div class="given-names">I.Y.</div></div>,  <div class="name"><div class="surname">Martinez</div> <div class="given-names">M.G.</div></div>,  <div class="name"><div class="surname">Jones</div> <div class="given-names">M.G.</div></div>,  <div class="name"><div class="surname">Caddick</div> <div class="given-names">M.X.</div></div>. </div><div class="article-title">A defined sequence within the 3′ UTR of the <em>areA</em> transcript is sufficient to mediate nitrogen metabolite signaling via accelerated deadenylation</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2000</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">1248</div>-<div class="lpage">57</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20defined%20sequence%20within%20the%203%E2%80%B2%20UTR%20of%20the%20areA%20transcript%20is%20sufficient%20to%20mediate%20nitrogen%20metabolite%20signaling%20via%20accelerated%20deadenylation&amp;author=I.Y.%20Morozov&amp;author=M.G.%20Martinez&amp;author=M.G.%20Jones&amp;author=M.X.%20Caddick&amp;publication_year=2000&amp;journal=Mol.%20Microbiol.&amp;volume=37&amp;pages=1248-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.2000.02085.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.2000.02085.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.2000.02085.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10972840" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Caddick</div> <div class="given-names">M.X.</div></div>,  <div class="name"><div class="surname">Jones</div> <div class="given-names">M.G.</div></div>,  <div class="name"><div class="surname">van Tonder</div> <div class="given-names">J.M.</div></div>, et al. </div><div class="article-title">Opposing signals differentially regulate transcript stability in <em>Aspergillus nidulans</em></div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2006</div>, vol. <div class="volume">62</div> (pg. <div class="fpage">509</div>-<div class="lpage">19</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Opposing%20signals%20differentially%20regulate%20transcript%20stability%20in%20Aspergillus%20nidulans&amp;author=M.X.%20Caddick&amp;author=M.G.%20Jones&amp;author=J.M.%20van%20Tonder&amp;publication_year=2006&amp;journal=Mol.%20Microbiol.&amp;volume=62&amp;pages=509-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.2006.62.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.2006.62.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.2006.62.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17020584" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wickens</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Bernstein</div> <div class="given-names">D.S.</div></div>,  <div class="name"><div class="surname">Kimble</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Parker</div> <div class="given-names">R.</div></div>. </div><div class="article-title">A PUF family portrait: 3′ UTR regulation as a way of life</div>, <div class="source">Trends Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">150</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20PUF%20family%20portrait%3A%203%E2%80%B2%20UTR%20regulation%20as%20a%20way%20of%20life&amp;author=M.%20Wickens&amp;author=D.S.%20Bernstein&amp;author=J.%20Kimble&amp;author=R.%20Parker&amp;publication_year=2002&amp;journal=Trends%20Genet.&amp;volume=18&amp;pages=150-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(01)02616-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(01)02616-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(01)02616-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11858839" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barreau</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Paillard</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Osborne</div> <div class="given-names">B.</div></div>. </div><div class="article-title">AU-rich elements and associated factors: are there unifying principles?</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">7138</div>-<div class="lpage">50</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AU-rich%20elements%20and%20associated%20factors%3A%20are%20there%20unifying%20principles%3F&amp;author=C.%20Barreau&amp;author=L.%20Paillard&amp;author=B.%20Osborne&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=7138-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki1012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki1012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki1012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16391004" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vasudevan</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Peltz</div> <div class="given-names">S.W.</div></div>. </div><div class="article-title">Regulated ARE-mediated mRNA decay in <em>Saccharomyces cerevisiae</em></div>, <div class="source">Mol. Cell</div> , <div class="year">2001</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">1191</div>-<div class="lpage">200</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulated%20ARE-mediated%20mRNA%20decay%20in%20Saccharomyces%20cerevisiae&amp;author=S.%20Vasudevan&amp;author=S.W.%20Peltz&amp;publication_year=2001&amp;journal=Mol.%20Cell&amp;volume=7&amp;pages=1191-200" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1097-2765(01)00279-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1097-2765(01)00279-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1097-2765(01)00279-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11430822" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Puig</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Askeland</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Thiele</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Coordinated remodeling of cellular metabolism during iron deficiency through targeted mRNA degradation</div>, <div class="source">Cell</div> , <div class="year">2005</div>, vol. <div class="volume">120</div> (pg. <div class="fpage">99</div>-<div class="lpage">110</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Coordinated%20remodeling%20of%20cellular%20metabolism%20during%20iron%20deficiency%20through%20targeted%20mRNA%20degradation&amp;author=S.%20Puig&amp;author=E.%20Askeland&amp;author=D.%20Thiele&amp;publication_year=2005&amp;journal=Cell&amp;volume=120&amp;pages=99-110" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2004.11.032" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2004.11.032" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2004.11.032"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15652485" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jackson</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">Houshmandi</div> <div class="given-names">S.S.</div></div>,  <div class="name"><div class="surname">Leban</div> <div class="given-names">F.L.</div></div>,  <div class="name"><div class="surname">Olivas</div> <div class="given-names">W.M.</div></div>. </div><div class="article-title">Recruitment of the Puf3 protein to its mRNA target for regulation of mRNA decay in yeast</div>, <div class="source">RNA</div> , <div class="year">2004</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">1625</div>-<div class="lpage">36</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recruitment%20of%20the%20Puf3%20protein%20to%20its%20mRNA%20target%20for%20regulation%20of%20mRNA%20decay%20in%20yeast&amp;author=J.S.%20Jackson&amp;author=S.S.%20Houshmandi&amp;author=F.L.%20Leban&amp;author=W.M.%20Olivas&amp;publication_year=2004&amp;journal=RNA&amp;volume=10&amp;pages=1625-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.7270204" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.7270204" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.7270204"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15337848" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dong</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Deng</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">An exploration of 3′-end processing signals and their tissue distribution in <em>Oryza sativa</em></div>, <div class="source">Gene</div> , <div class="year">2007</div>, vol. <div class="volume">389</div> (pg. <div class="fpage">107</div>-<div class="lpage">13</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20exploration%20of%203%E2%80%B2-end%20processing%20signals%20and%20their%20tissue%20distribution%20in%20Oryza%20sativa&amp;author=H.%20Dong&amp;author=Y.%20Deng&amp;author=J.%20Chen&amp;publication_year=2007&amp;journal=Gene&amp;volume=389&amp;pages=107-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2006.10.015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2006.10.015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2006.10.015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17187943" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wickens</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Stephenson</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Role of the conserved AAUAAA sequence: for AAUAAA point mutations prevent messenger RNA 3′ end formation</div>, <div class="source">Science</div> , <div class="year">1984</div>, vol. <div class="volume">226</div> (pg. <div class="fpage">1045</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Role%20of%20the%20conserved%20AAUAAA%20sequence%3A%20for%20AAUAAA%20point%20mutations%20prevent%20messenger%20RNA%203%E2%80%B2%20end%20formation&amp;author=M.%20Wickens&amp;author=P.%20Stephenson&amp;publication_year=1984&amp;journal=Science&amp;volume=226&amp;pages=1045-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.6208611" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.6208611" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.6208611"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6208611" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guo</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Russo</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Yun</div> <div class="given-names">D.F.</div></div>,  <div class="name"><div class="surname">Butler</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">Sherman</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Redundant 3′ end-forming signals for the yeast CYC1 mRNA</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1995</div>, vol. <div class="volume">92</div> (pg. <div class="fpage">4211</div>-<div class="lpage">14</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Redundant%203%E2%80%B2%20end-forming%20signals%20for%20the%20yeast%20CYC1%20mRNA&amp;author=Z.%20Guo&amp;author=P.%20Russo&amp;author=D.F.%20Yun&amp;author=J.S.%20Butler&amp;author=F.%20Sherman&amp;publication_year=1995&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=92&amp;pages=4211-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.92.10.4211" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.92.10.4211" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.92.10.4211"> </span></div> </div></div></div></div></div><div content-id="DSR011C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Graber</div> <div class="given-names">J.H.</div></div>,  <div class="name"><div class="surname">McAllister</div> <div class="given-names">G.D.</div></div>,  <div class="name"><div class="surname">Smith</div> <div class="given-names">T.F.</div></div>. </div><div class="article-title">Probabilistic prediction of <em>Saccharomyces cerevisiae</em> mRNA 3'-processing sites</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2002</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">1851</div>-<div class="lpage">58</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Probabilistic%20prediction%20of%20Saccharomyces%20cerevisiae%20mRNA%203'-processing%20sites&amp;author=J.H.%20Graber&amp;author=G.D.%20McAllister&amp;author=T.F.%20Smith&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res.&amp;volume=30&amp;pages=1851-58" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/30.8.1851" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F30.8.1851" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F30.8.1851"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11937640" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Venkataraman</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Brown</div> <div class="given-names">K.M.</div></div>,  <div class="name"><div class="surname">Gilmartin</div> <div class="given-names">G.M.</div></div>. </div><div class="article-title">Analysis of a noncanonical poly(A) site reveals a tripartite mechanism for vertebrate poly(A) site recognition</div>, <div class="source">Genes Dev.</div> , <div class="year">2005</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">1315</div>-<div class="lpage">27</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20a%20noncanonical%20poly(A)%20site%20reveals%20a%20tripartite%20mechanism%20for%20vertebrate%20poly(A)%20site%20recognition&amp;author=K.%20Venkataraman&amp;author=K.M.%20Brown&amp;author=G.M.%20Gilmartin&amp;publication_year=2005&amp;journal=Genes%20Dev.&amp;volume=19&amp;pages=1315-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.1298605" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.1298605" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.1298605"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15937220" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nedea</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">He</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Organization and function of APT, a subcomplex of the yeast cleavage and polyadenylation factor involved in the formation of mRNA and small nucleolar RNA 3'-ends</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2003</div>, vol. <div class="volume">278</div> (pg. <div class="fpage">33000</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Organization%20and%20function%20of%20APT%2C%20a%20subcomplex%20of%20the%20yeast%20cleavage%20and%20polyadenylation%20factor%20involved%20in%20the%20formation%20of%20mRNA%20and%20small%20nucleolar%20RNA%203'-ends&amp;author=E.%20Nedea&amp;author=X.%20He&amp;author=M.%20Kim&amp;publication_year=2003&amp;journal=J.%20Biol.%20Chem.&amp;volume=278&amp;pages=33000-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M304454200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M304454200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M304454200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12819204" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kessler</div> <div class="given-names">M.M.</div></div>,  <div class="name"><div class="surname">Henry</div> <div class="given-names">M.F.</div></div>,  <div class="name"><div class="surname">Shen</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Hrp1, a sequence-specific RNA-binding protein that shuttles between the nucleus and the cytoplasm, is required for mRNA 3′-end formation in yeast</div>, <div class="source">Genes Dev.</div> , <div class="year">1997</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">2545</div>-<div class="lpage">56</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hrp1%2C%20a%20sequence-specific%20RNA-binding%20protein%20that%20shuttles%20between%20the%20nucleus%20and%20the%20cytoplasm%2C%20is%20required%20for%20mRNA%203%E2%80%B2-end%20formation%20in%20yeast&amp;author=M.M.%20Kessler&amp;author=M.F.%20Henry&amp;author=E.%20Shen&amp;publication_year=1997&amp;journal=Genes%20Dev.&amp;volume=11&amp;pages=2545-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.11.19.2545" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.11.19.2545" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.11.19.2545"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9334319" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pérez-Cañadillas</div> <div class="given-names">J.M.</div></div>. </div><div class="article-title">Grabbing the message: structural basis of mRNA 3′ UTR recognition by Hrp1</div>, <div class="source">EMBO J.</div> , <div class="year">2006</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3167</div>-<div class="lpage">78</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Grabbing%20the%20message%3A%20structural%20basis%20of%20mRNA%203%E2%80%B2%20UTR%20recognition%20by%20Hrp1&amp;author=J.M.%20P%C3%A9rez-Ca%C3%B1adillas&amp;publication_year=2006&amp;journal=EMBO%20J.&amp;volume=25&amp;pages=3167-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/sj.emboj.7601190" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.emboj.7601190" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsj.emboj.7601190"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16794580" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dichtl</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Blank</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Sadowski</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hubner</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Weiser</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Keller</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Yhh1p/Cft1p directly links poly(A) site recognition and RNA polymerase II transcription termination</div>, <div class="source">EMBO J.</div> , <div class="year">2002</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">4125</div>-<div class="lpage">35</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Yhh1p%2FCft1p%20directly%20links%20poly(A)%20site%20recognition%20and%20RNA%20polymerase%20II%20transcription%20termination&amp;author=B.%20Dichtl&amp;author=D.%20Blank&amp;author=M.%20Sadowski&amp;author=W.%20Hubner&amp;author=S.%20Weiser&amp;author=W.%20Keller&amp;publication_year=2002&amp;journal=EMBO%20J.&amp;volume=21&amp;pages=4125-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/emboj/cdf390" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Femboj%2Fcdf390" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Femboj%2Fcdf390"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12145212" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR011C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR011C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hunt</div> <div class="given-names">A.G.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Addepalli</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">Arabidopsis mRNA polyadenylation machinery: comprehensive analysis of protein-protein interactions and gene expression profiling</div>, <div class="source">BMC Genomics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">220</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Arabidopsis%20mRNA%20polyadenylation%20machinery%3A%20comprehensive%20analysis%20of%20protein-protein%20interactions%20and%20gene%20expression%20profiling&amp;author=A.G.%20Hunt&amp;author=R.%20Xu&amp;author=B.%20Addepalli&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=220" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-220" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-220" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-220"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18479511" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2011. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=445126&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr011/2/dsr011-dsr011supp_table4.xls?Expires=2147483647&amp;Signature=32yQERfgdpYSDO70ga5jbRW8CBkVAcAfpl5KOWs8IrOSQumIGq5SzXk8RumNdm1IkxH8gnk~EZyXXKMu3ITqGnvqKQgZglbJPX2HVtfVvIT7S0FX2e49hActKU8c-qPx16w-4unkxy5dWG71UEBtuPMkh1z1PtpBrG0vQ45oMYEaalLxvu6UQrSKEKv9YfAxGtk7Cjn8bLWOvtBy-tXst6ug2WtEAgzBQRbqgVt9fsYFi7baPCRZvBL6KBIJUazGLQAQqM2P2nOTz43lBCMgTT9wzjOgaSdm-JUyvINmIQL2Ez0G~Ovyk6GwlK7sOiE0bcU3W-9QH80kYfW93pSbMA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr011/2/dsr011-dsr011supp_table.doc?Expires=2147483647&amp;Signature=inyahdsQO8KNg7w6NdWcvU5~mezBCFOE~BkbPKASTRu~udxdpKmf5K89wqA7xsVn9Ssobv44DMruZ~uiJxE9X-rwCLbOu8vrykgW0qNaorI5jUmGvgVNBep1DU61Q4wPuZrXySbC5GRB62g1uGka4uFZVPZJLvkjV5sDjlgFSMoaMhyAuHyImNVZweGIWnYqz3xIvsl8ksDvKFg0awpplsIQgSdcvvoQ5VPHK2Wx6uzBVGlPaCkh-ux3HDDsWdNrZaHKU-2PAUytD2h65ynFkBImCuG4UwZPR84xNWgXlGkITkNkEEOAdFkaRZEiYwPKNhtx~3MulBS7FyawMVg~4w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">105</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000292070400005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">7</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsr011"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="445126"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000292070400005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f18%2f3%2f189%2f445126">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000292070400005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (7)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F18%2F3%2F189%2F445126" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/445126'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="In silico Analysis of 3′-End-Processing Signals in Aspergillus oryzae Using Expressed Sequence Tags and Genomic Sequencing Data" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsr011");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f18%2f3%2f189%2f445126">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

