

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/20/3/273/369179","@type":"ScholarlyArticle","name":"Re-Annotation of Protein-Coding Genes in 10 Complete Genomes of Neisseriaceae Family by Combining Similarity-Based and Composition-Based Methods","datePublished":"2013-04-09","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/20/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2013-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dst009","keywords":["the Neisseriaceae family","re-annotation","newly found genes","eliminated non-coding ORFs","newly assigned functions"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/20/3/273/369179","author":[{"name":"Guo, Feng-Biao","affiliation":"1  Department of Microbiology ,  The University of Hong Kong, Special Administrative Region ,  Hong Kong ,  People's Republic of China    2  School of Life Science and Technology ,  University of Electronic Science and Technology of China ,  Chengdu  610054 ,  People's Republic of China","@type":"Person"},{"name":"Xiong, Lifeng","affiliation":"1  Department of Microbiology ,  The University of Hong Kong, Special Administrative Region ,  Hong Kong ,  People's Republic of China","@type":"Person"},{"name":"Teng, Jade L. L.","affiliation":"1  Department of Microbiology ,  The University of Hong Kong, Special Administrative Region ,  Hong Kong ,  People's Republic of China","@type":"Person"},{"name":"Yuen, Kwok-Yung","affiliation":"1  Department of Microbiology ,  The University of Hong Kong, Special Administrative Region ,  Hong Kong ,  People's Republic of China    3  State Key Laboratory of Emerging Infectious Diseases ,  The University of Hong Kong, Special Administrative Region ,  Hong Kong ,  People's Republic of China    4  Research Centre of Infection and Immunology ,  The University of Hong Kong, Special Administrative Region ,  Hong Kong ,  People's Republic of China","@type":"Person"},{"name":"Lau, Susanna K. P.","affiliation":"1  Department of Microbiology ,  The University of Hong Kong, Special Administrative Region ,  Hong Kong ,  People's Republic of China    3  State Key Laboratory of Emerging Infectious Diseases ,  The University of Hong Kong, Special Administrative Region ,  Hong Kong ,  People's Republic of China    4  Research Centre of Infection and Immunology ,  The University of Hong Kong, Special Administrative Region ,  Hong Kong ,  People's Republic of China","@type":"Person"},{"name":"Woo, Patrick C. Y.","affiliation":"1  Department of Microbiology ,  The University of Hong Kong, Special Administrative Region ,  Hong Kong ,  People's Republic of China    3  State Key Laboratory of Emerging Infectious Diseases ,  The University of Hong Kong, Special Administrative Region ,  Hong Kong ,  People's Republic of China    4  Research Centre of Infection and Immunology ,  The University of Hong Kong, Special Administrative Region ,  Hong Kong ,  People's Republic of China","@type":"Person"}],"description":"Abstract.  In this paper, we performed a comprehensive re-annotation of protein-coding genes by a systematic method combining composition- and similarity-based ","pageStart":"273","pageEnd":"286","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/3/1/m_cover.gif?Expires=2147483647&Signature=3JNT-3uWcoFSRSDjpyRS683cAWcv-uPBeo5zUjK0TFHyw5pdGPApuAnjwKw9hziMynOyeupAu8qXTBcWNNZ5zwCIK3wDotNETRf1C0gT1NeaWwrIBlHGIKu6bYD8JmeQgULL4qaQcRAHnyo7FyqKjoT8e0y4GLL7JPBIG8ClvgMIpmjvKRKJfK-rkhDpE9pXuYo8ex4C6o9vQhlNIWM8gIl4ygPsB2y8BMTIUXCEDS8KZ8W94~SHycoNAwx1VF1wN~gGUhCSNthMU3E6MsUONjgAq-FWmQ8kuAdsyKdxNSJy8mhuDD9SBTDoUvN5zzz~5gwbw0AeOX0S-zArQW9D7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Re-Annotation of Protein-Coding Genes in 10 Complete Genomes of Neisseriaceae Family by Combining Similarity-Based and Composition-Based Methods","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/3/1/m_cover.gif?Expires=2147483647&Signature=3JNT-3uWcoFSRSDjpyRS683cAWcv-uPBeo5zUjK0TFHyw5pdGPApuAnjwKw9hziMynOyeupAu8qXTBcWNNZ5zwCIK3wDotNETRf1C0gT1NeaWwrIBlHGIKu6bYD8JmeQgULL4qaQcRAHnyo7FyqKjoT8e0y4GLL7JPBIG8ClvgMIpmjvKRKJfK-rkhDpE9pXuYo8ex4C6o9vQhlNIWM8gIl4ygPsB2y8BMTIUXCEDS8KZ8W94~SHycoNAwx1VF1wN~gGUhCSNthMU3E6MsUONjgAq-FWmQ8kuAdsyKdxNSJy8mhuDD9SBTDoUvN5zzz~5gwbw0AeOX0S-zArQW9D7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Re-Annotation of Protein-Coding Genes in 10 Complete Genomes of Neisseriaceae Family by Combining Similarity-Based and Composition-Based Methods" />
    <meta property="og:description" content="Abstract.  In this paper, we performed a comprehensive re-annotation of protein-coding genes by a systematic method combining composition- and similarity-based " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/20/3/273/369179" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2013-04-09" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Guo, Feng-Biao" /><meta name="citation_author_institution" content="Department of Microbiology, The University of Hong Kong, Special Administrative Region, Hong Kong, People's Republic of China" /><meta name="citation_author_institution" content="School of Life Science and Technology, University of Electronic Science and Technology of China, Chengdu610054, People's Republic of China" /><meta name="citation_author" content="Xiong, Lifeng" /><meta name="citation_author_institution" content="Department of Microbiology, The University of Hong Kong, Special Administrative Region, Hong Kong, People's Republic of China" /><meta name="citation_author" content="Teng, Jade L. L." /><meta name="citation_author_institution" content="Department of Microbiology, The University of Hong Kong, Special Administrative Region, Hong Kong, People's Republic of China" /><meta name="citation_author" content="Yuen, Kwok-Yung" /><meta name="citation_author_institution" content="Department of Microbiology, The University of Hong Kong, Special Administrative Region, Hong Kong, People's Republic of China" /><meta name="citation_author_institution" content="State Key Laboratory of Emerging Infectious Diseases, The University of Hong Kong, Special Administrative Region, Hong Kong, People's Republic of China" /><meta name="citation_author_institution" content="Research Centre of Infection and Immunology, The University of Hong Kong, Special Administrative Region, Hong Kong, People's Republic of China" /><meta name="citation_author" content="Lau, Susanna K. P." /><meta name="citation_author_institution" content="Department of Microbiology, The University of Hong Kong, Special Administrative Region, Hong Kong, People's Republic of China" /><meta name="citation_author_institution" content="State Key Laboratory of Emerging Infectious Diseases, The University of Hong Kong, Special Administrative Region, Hong Kong, People's Republic of China" /><meta name="citation_author_institution" content="Research Centre of Infection and Immunology, The University of Hong Kong, Special Administrative Region, Hong Kong, People's Republic of China" /><meta name="citation_author" content="Woo, Patrick C. Y." /><meta name="citation_author_institution" content="Department of Microbiology, The University of Hong Kong, Special Administrative Region, Hong Kong, People's Republic of China" /><meta name="citation_author_institution" content="State Key Laboratory of Emerging Infectious Diseases, The University of Hong Kong, Special Administrative Region, Hong Kong, People's Republic of China" /><meta name="citation_author_institution" content="Research Centre of Infection and Immunology, The University of Hong Kong, Special Administrative Region, Hong Kong, People's Republic of China" /><meta name="citation_title" content="Re-Annotation of Protein-Coding Genes in 10 Complete Genomes of Neisseriaceae Family by Combining Similarity-Based and Composition-Based Methods" /><meta name="citation_firstpage" content="273" /><meta name="citation_lastpage" content="286" /><meta name="citation_doi" content="10.1093/dnares/dst009" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="20" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2013/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=The past, present and future of genome-wide re-annotation; Genome Biol.;  citation_year=2002;  citation_volume=3;  citation_pages=COMMENT2001" /><meta name="citation_reference" content="citation_title=Re-annotation is an essential step in systems biology modeling of functional genomics data; PLoS One;  citation_year=2010;  citation_volume=5;  citation_pages=e10642" /><meta name="citation_reference" content="citation_title=Identification of protein-coding genes in the genome of Vibrio cholerae with more than 98% accuracy using occurrence frequencies of single nucleotides; Eur. J. Biochem.;  citation_year=2001;  citation_volume=268;  citation_pages=4261-8; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of an aerobic hyper-thermophilic crenarchaeon, Aeropyrum pernix K1; citation_text=, 145–52; DNA Res.;  citation_year=1999;  citation_volume=6;  citation_pages=83-101; " /><meta name="citation_reference" content="citation_title=Towards understanding the first genome sequence of a crenarchaeon by genome annotation using clusters of orthologous groups of proteins (COGs); Genome Biol.;  citation_year=2000;  citation_volume=1;  citation_pages=RESEARCH0009" /><meta name="citation_reference" content="citation_title=Re-annotation of genome microbial coding-sequences: finding new genes and inaccurately annotated genes; BMC Bioinformatics;  citation_year=2002;  citation_volume=3;  citation_pages=5" /><meta name="citation_reference" content="citation_title=Gene recognition based on nucleotide distribution of ORFs in a hyper-thermophilic crenarchaeon, Aeropyrum pernix K1; DNA Res.;  citation_year=2004;  citation_volume=11;  citation_pages=361-70; " /><meta name="citation_reference" content="citation_title=Proteome analysis of an aerobic hyperthermophilic crenarchaeon, Aeropyrum pernix K1; Mol. Cell. Proteomics;  citation_year=2006;  citation_volume=5;  citation_pages=811-23; " /><meta name="citation_reference" content="citation_title=Identify protein-coding genes in the genomes of Aeropyrum pernix K1 and Chlorobium tepidum TLS; J. Biomol. Struct. Dyn.;  citation_year=2009;  citation_volume=26;  citation_pages=413-20; " /><meta name="citation_reference" content="citation_title=Discriminate the falsely predicted protein-coding genes in Aeropyrum Pernix K1 genome based on graphical representation; MATCH Commun. Math. Comput. Chem.;  citation_year=2012;  citation_volume=67;  citation_pages=845-66; " /><meta name="citation_reference" content="citation_title=Complete genomic sequence of the Amsacta moorei entomopoxvirus: analysis and comparison with other poxviruses; Virology;  citation_year=2000;  citation_volume=274;  citation_pages=120-39; " /><meta name="citation_reference" content="citation_title=Re-prediction of protein-coding genes in the genome of Amsacta moorei entomopoxvirus; J. Virol. Methods;  citation_year=2007;  citation_volume=146;  citation_pages=389-92; " /><meta name="citation_reference" content="citation_title=Reannotation of protein-coding genes based on an improved graphical representation of DNA sequence; J. Comput. Chem.;  citation_year=2010;  citation_volume=31;  citation_pages=2126-35; " /><meta name="citation_reference" content="citation_title=Gene discovery by genome-wide CDS re-prediction and microarray-based transcriptional analysis in phytopathogen Xanthomonas campestris; BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=359" /><meta name="citation_reference" content="citation_title=Gene re-annotation in genome of the extremophile Pyrobaculum aerophilum by using bioinformatics methods; J. Biomol. Struct. Dyn.;  citation_year=2011;  citation_volume=29;  citation_pages=391-401; " /><meta name="citation_reference" content="citation_title=Reannotation of hypothetical ORFs in plant pathogen Erwinia carotovora subsp. atroseptica SCRI1043; FEBS J.;  citation_year=2008;  citation_volume=275;  citation_pages=198-206; " /><meta name="citation_reference" content="citation_title=Proteome driven re-evaluation and functional annotation of the Streptococcus pyogenes SF370 genome; BMC Microbiol.;  citation_year=2011;  citation_volume=11;  citation_pages=249" /><meta name="citation_reference" content="citation_title=Complete genome sequence of Neisseria meningitidis serogroup B strain MC58; Science;  citation_year=2000;  citation_volume=287;  citation_pages=1809-15; " /><meta name="citation_reference" content="citation_title=Laribacter hongkongensis gen. nov., sp. nov., a novel Gram-negative bacterium isolated from a cirrhotic patient with bacteremia and empyema; J. Clin. Microbiol.;  citation_year=2001;  citation_volume=39;  citation_pages=4227-32; " /><meta name="citation_reference" content="citation_title=Association of Laribacter hongkongensis in community-acquired gastroenteritis with travel and eating fish: a multicentre case-control study; Lancet;  citation_year=2004;  citation_volume=363;  citation_pages=1941-7; " /><meta name="citation_reference" content="citation_title=The complete genome and proteome of Laribacter hongkongensis reveal potential mechanisms for adaptations to different temperatures and habitats; PLoS Genet.;  citation_year=2009;  citation_volume=25;  citation_pages=e1000416" /><meta name="citation_reference" content="citation_title=Bacteremia caused by Laribacter hongkongensis misidentified as Acinetobacter lwoffii: report of the first case in Korea; J. Korean Med. Sci.;  citation_year=2011;  citation_volume=26;  citation_pages=679-81; " /><meta name="citation_reference" content="citation_title=The Genomes OnLine Database (GOLD) v.4: status of genomic and metagenomic projects and their associated metadata; Nucleic Acids Res.;  citation_year=2012;  citation_volume=40;  citation_pages=D571-9; " /><meta name="citation_reference" content="citation_title=NCBI reference sequences (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=D61-5; " /><meta name="citation_reference" content="citation_title=Missing genes in the annotation of prokaryotic genomes; BMC Bioinformatics;  citation_year=2010;  citation_volume=11;  citation_pages=131" /><meta name="citation_reference" content="citation_title=ZCURVE: a new system for recognizing protein-coding genes in bacterial and archaeal genomes; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=1780-9; " /><meta name="citation_reference" content="citation_title=Protein database searches using compositionally adjusted substitution matrices; FEBS J.;  citation_year=2005;  citation_volume=272;  citation_pages=5101-9; " /><meta name="citation_reference" content="citation_title=Recognition of protein coding genes in the yeast genome at better than 95% accuracy based on the Z curve; Nucleic Acids Res.;  citation_year=2000;  citation_volume=28;  citation_pages=2804-14; " /><meta name="citation_reference" content="citation_title=DIGA–a database of improved gene annotation for phytopathogens; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=54" /><meta name="citation_reference" content="citation_title=Comparison of various algorithms for recognizing short coding sequences of human genes; Bioinformatics;  citation_year=2004;  citation_volume=20;  citation_pages=673-81; " /><meta name="citation_reference" content="citation_title=The complete genome sequence of *Chromobacterium violaceum reveals remarkable and exploitable bacterial adaptability; citation_author=Brazilian National Genome Project Consortium; Proc. Natl. Acad. Sci. USA;  citation_year=2003;  citation_volume=100;  citation_pages=11660-5; " /><meta name="citation_reference" content="citation_title=Prodigal: prokaryotic gene recognition and translation initiation site identification; BMC Bioinformatics;  citation_year=2010;  citation_volume=11;  citation_pages=119" /><meta name="citation_reference" content="citation_title=The automatic annotation of bacterial genomes; Brief Bioinform;  citation_year=2013;  citation_volume=14;  citation_pages=1-12; " /><meta name="citation_reference" content="citation_title=Genome re-annotation: a wiki solution?; Genome Biol.;  citation_year=2007;  citation_volume=8;  citation_pages=102" /><meta name="citation_reference" content="citation_title=A graphic approach to analyzing codon usage in 1562 Escherichia coli protein coding sequences; J. Mol. Biol.;  citation_year=1994;  citation_volume=238;  citation_pages=1-8; " /><meta name="citation_reference" content="citation_title=Seven GC-rich microbial genomes adopt similar codon usage patterns regardless of their phylogenetic lineages; Biochem. Biophys. Res. Commun.;  citation_year=2003;  citation_volume=306;  citation_pages=310-7; " /><meta name="citation_reference" content="citation_title=The distribution patterns of bases of protein-coding genes, non-coding ORFs, and intergenic sequences in Pseudomonas aeruginosa PA01 genome and its implications; J. Biomol. Struct. Dyn.;  citation_year=2007;  citation_volume=25;  citation_pages=127-33; " /><meta name="citation_reference" content="citation_title=Using the COG database to improve gene recognition in complete genomes; Genetica;  citation_year=2000;  citation_volume=108;  citation_pages=9-17; " /><meta name="citation_reference" content="citation_title=An integrative method for identifying the over-annotated protein-coding genes in microbial genomes; DNA Res.;  citation_year=2011;  citation_volume=18;  citation_pages=435-49; " /><meta name="citation_reference" content="citation_title=Genome reannotation of Escherichia coli CFT073 with new insights into virulence; BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=552" /><meta name="citation_reference" content="citation_title=How to interpret an anonymous bacterial genome: machine learning approach to gene identification; Genome Res.;  citation_year=1998;  citation_volume=8;  citation_pages=1154-71; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of Neiserria gonorrhoeae NCCP11945; J. Bacteriol.;  citation_year=2008;  citation_volume=190;  citation_pages=6035-6; " /><meta name="citation_reference" content="citation_title=ICDS database: interrupted CoDing sequences in prokaryotic genomes; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=D338-43; " /><meta name="citation_reference" content="citation_title=A pilot study of bacterial genes with disrupted ORFs reveals a surprising profusion of protein sequence recoding mediated by ribosomal frameshifting and transcriptional realignment; Mol. Biol. Evol.;  citation_year=2011;  citation_volume=28;  citation_pages=3195-211; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/20/3/273/1051777/dst009.pdf" /><meta name="description" content="Abstract.  In this paper, we performed a comprehensive re-annotation of protein-coding genes by a systematic method combining composition- and similarity-based " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/20/3/273/369179" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/20/3/273/369179" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/20/3/273/369179");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dst009';
    </script>
<title>
	Re-Annotation of Protein-Coding Genes in 10 Complete Genomes of Neisseriaceae Family by Combining Similarity-Based and Composition-Based Methods | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./369179" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BYlzPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJHdW8sIEZlbmctQmlhbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTWljcm9iaW9sb2d5LCBUaGUgVW5pdmVyc2l0eSBvZiBIb25nIEtvbmcsIFNwZWNpYWwgQWRtaW5pc3RyYXRpdmUgUmVnaW9uLCBIb25nIEtvbmcsIFBlb3BsZSdzIFJlcHVibGljIG9mIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU2Nob29sIG9mIExpZmUgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgVW5pdmVyc2l0eSBvZiBFbGVjdHJvbmljIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3kgb2YgQ2hpbmEsIENoZW5nZHU2MTAwNTQsIFBlb3BsZSdzIFJlcHVibGljIG9mIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWGlvbmcsIExpZmVuZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTWljcm9iaW9sb2d5LCBUaGUgVW5pdmVyc2l0eSBvZiBIb25nIEtvbmcsIFNwZWNpYWwgQWRtaW5pc3RyYXRpdmUgUmVnaW9uLCBIb25nIEtvbmcsIFBlb3BsZSdzIFJlcHVibGljIG9mIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGVuZywgSmFkZSBMLiBMLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTWljcm9iaW9sb2d5LCBUaGUgVW5pdmVyc2l0eSBvZiBIb25nIEtvbmcsIFNwZWNpYWwgQWRtaW5pc3RyYXRpdmUgUmVnaW9uLCBIb25nIEtvbmcsIFBlb3BsZSdzIFJlcHVibGljIG9mIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWXVlbiwgS3dvay1ZdW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBNaWNyb2Jpb2xvZ3ksIFRoZSBVbml2ZXJzaXR5IG9mIEhvbmcgS29uZywgU3BlY2lhbCBBZG1pbmlzdHJhdGl2ZSBSZWdpb24sIEhvbmcgS29uZywgUGVvcGxlJ3MgUmVwdWJsaWMgb2YgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTdGF0ZSBLZXkgTGFib3JhdG9yeSBvZiBFbWVyZ2luZyBJbmZlY3Rpb3VzIERpc2Vhc2VzLCBUaGUgVW5pdmVyc2l0eSBvZiBIb25nIEtvbmcsIFNwZWNpYWwgQWRtaW5pc3RyYXRpdmUgUmVnaW9uLCBIb25nIEtvbmcsIFBlb3BsZSdzIFJlcHVibGljIG9mIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUmVzZWFyY2ggQ2VudHJlIG9mIEluZmVjdGlvbiBhbmQgSW1tdW5vbG9neSwgVGhlIFVuaXZlcnNpdHkgb2YgSG9uZyBLb25nLCBTcGVjaWFsIEFkbWluaXN0cmF0aXZlIFJlZ2lvbiwgSG9uZyBLb25nLCBQZW9wbGUncyBSZXB1YmxpYyBvZiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxhdSwgU3VzYW5uYSBLLiBQLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTWljcm9iaW9sb2d5LCBUaGUgVW5pdmVyc2l0eSBvZiBIb25nIEtvbmcsIFNwZWNpYWwgQWRtaW5pc3RyYXRpdmUgUmVnaW9uLCBIb25nIEtvbmcsIFBlb3BsZSdzIFJlcHVibGljIG9mIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU3RhdGUgS2V5IExhYm9yYXRvcnkgb2YgRW1lcmdpbmcgSW5mZWN0aW91cyBEaXNlYXNlcywgVGhlIFVuaXZlcnNpdHkgb2YgSG9uZyBLb25nLCBTcGVjaWFsIEFkbWluaXN0cmF0aXZlIFJlZ2lvbiwgSG9uZyBLb25nLCBQZW9wbGUncyBSZXB1YmxpYyBvZiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlJlc2VhcmNoIENlbnRyZSBvZiBJbmZlY3Rpb24gYW5kIEltbXVub2xvZ3ksIFRoZSBVbml2ZXJzaXR5IG9mIEhvbmcgS29uZywgU3BlY2lhbCBBZG1pbmlzdHJhdGl2ZSBSZWdpb24sIEhvbmcgS29uZywgUGVvcGxlJ3MgUmVwdWJsaWMgb2YgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXb28sIFBhdHJpY2sgQy4gWS4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE1pY3JvYmlvbG9neSwgVGhlIFVuaXZlcnNpdHkgb2YgSG9uZyBLb25nLCBTcGVjaWFsIEFkbWluaXN0cmF0aXZlIFJlZ2lvbiwgSG9uZyBLb25nLCBQZW9wbGUncyBSZXB1YmxpYyBvZiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlN0YXRlIEtleSBMYWJvcmF0b3J5IG9mIEVtZXJnaW5nIEluZmVjdGlvdXMgRGlzZWFzZXMsIFRoZSBVbml2ZXJzaXR5IG9mIEhvbmcgS29uZywgU3BlY2lhbCBBZG1pbmlzdHJhdGl2ZSBSZWdpb24sIEhvbmcgS29uZywgUGVvcGxlJ3MgUmVwdWJsaWMgb2YgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJSZXNlYXJjaCBDZW50cmUgb2YgSW5mZWN0aW9uIGFuZCBJbW11bm9sb2d5LCBUaGUgVW5pdmVyc2l0eSBvZiBIb25nIEtvbmcsIFNwZWNpYWwgQWRtaW5pc3RyYXRpdmUgUmVnaW9uLCBIb25nIEtvbmcsIFBlb3BsZSdzIFJlcHVibGljIG9mIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJSZS1Bbm5vdGF0aW9uIG9mIFByb3RlaW4tQ29kaW5nIEdlbmVzIGluIDEwIENvbXBsZXRlIEdlbm9tZXMgb2YgTmVpc3NlcmlhY2VhZSBGYW1pbHkgYnkgQ29tYmluaW5nIFNpbWlsYXJpdHktQmFzZWQgYW5kIENvbXBvc2l0aW9uLUJhc2VkIE1ldGhvZHMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIyNzMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjI4NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdDAwOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDEzLzA2LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHBhc3QsIHByZXNlbnQgYW5kIGZ1dHVyZSBvZiBnZW5vbWUtd2lkZSByZS1hbm5vdGF0aW9uOyBHZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPUNPTU1FTlQyMDAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmUtYW5ub3RhdGlvbiBpcyBhbiBlc3NlbnRpYWwgc3RlcCBpbiBzeXN0ZW1zIGJpb2xvZ3kgbW9kZWxpbmcgb2YgZnVuY3Rpb25hbCBnZW5vbWljcyBkYXRhOyBQTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9ZTEwNjQyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgcHJvdGVpbi1jb2RpbmcgZ2VuZXMgaW4gdGhlIGdlbm9tZSBvZiBWaWJyaW8gY2hvbGVyYWUgd2l0aCBtb3JlIHRoYW4gOTglIGFjY3VyYWN5IHVzaW5nIG9jY3VycmVuY2UgZnJlcXVlbmNpZXMgb2Ygc2luZ2xlIG51Y2xlb3RpZGVzOyBFdXIuIEouIEJpb2NoZW0uOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI2ODsgIGNpdGF0aW9uX3BhZ2VzPTQyNjEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGxldGUgZ2Vub21lIHNlcXVlbmNlIG9mIGFuIGFlcm9iaWMgaHlwZXItdGhlcm1vcGhpbGljIGNyZW5hcmNoYWVvbiwgQWVyb3B5cnVtIHBlcm5peCBLMTsgY2l0YXRpb25fdGV4dD0sIDE0NeKAkzUyOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ODMtMTAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ub3dhcmRzIHVuZGVyc3RhbmRpbmcgdGhlIGZpcnN0IGdlbm9tZSBzZXF1ZW5jZSBvZiBhIGNyZW5hcmNoYWVvbiBieSBnZW5vbWUgYW5ub3RhdGlvbiB1c2luZyBjbHVzdGVycyBvZiBvcnRob2xvZ291cyBncm91cHMgb2YgcHJvdGVpbnMgKENPR3MpOyBHZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPVJFU0VBUkNIMDAwOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlLWFubm90YXRpb24gb2YgZ2Vub21lIG1pY3JvYmlhbCBjb2Rpbmctc2VxdWVuY2VzOiBmaW5kaW5nIG5ldyBnZW5lcyBhbmQgaW5hY2N1cmF0ZWx5IGFubm90YXRlZCBnZW5lczsgQk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz01IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSByZWNvZ25pdGlvbiBiYXNlZCBvbiBudWNsZW90aWRlIGRpc3RyaWJ1dGlvbiBvZiBPUkZzIGluIGEgaHlwZXItdGhlcm1vcGhpbGljIGNyZW5hcmNoYWVvbiwgQWVyb3B5cnVtIHBlcm5peCBLMTsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0zNjEtNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb3Rlb21lIGFuYWx5c2lzIG9mIGFuIGFlcm9iaWMgaHlwZXJ0aGVybW9waGlsaWMgY3JlbmFyY2hhZW9uLCBBZXJvcHlydW0gcGVybml4IEsxOyBNb2wuIENlbGwuIFByb3Rlb21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTgxMS0yMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZnkgcHJvdGVpbi1jb2RpbmcgZ2VuZXMgaW4gdGhlIGdlbm9tZXMgb2YgQWVyb3B5cnVtIHBlcm5peCBLMSBhbmQgQ2hsb3JvYml1bSB0ZXBpZHVtIFRMUzsgSi4gQmlvbW9sLiBTdHJ1Y3QuIER5bi47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz00MTMtMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc2NyaW1pbmF0ZSB0aGUgZmFsc2VseSBwcmVkaWN0ZWQgcHJvdGVpbi1jb2RpbmcgZ2VuZXMgaW4gQWVyb3B5cnVtIFBlcm5peCBLMSBnZW5vbWUgYmFzZWQgb24gZ3JhcGhpY2FsIHJlcHJlc2VudGF0aW9uOyBNQVRDSCBDb21tdW4uIE1hdGguIENvbXB1dC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9Njc7ICBjaXRhdGlvbl9wYWdlcz04NDUtNjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBsZXRlIGdlbm9taWMgc2VxdWVuY2Ugb2YgdGhlIEFtc2FjdGEgbW9vcmVpIGVudG9tb3BveHZpcnVzOiBhbmFseXNpcyBhbmQgY29tcGFyaXNvbiB3aXRoIG90aGVyIHBveHZpcnVzZXM7IFZpcm9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTI3NDsgIGNpdGF0aW9uX3BhZ2VzPTEyMC0zOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmUtcHJlZGljdGlvbiBvZiBwcm90ZWluLWNvZGluZyBnZW5lcyBpbiB0aGUgZ2Vub21lIG9mIEFtc2FjdGEgbW9vcmVpIGVudG9tb3BveHZpcnVzOyBKLiBWaXJvbC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDY7ICBjaXRhdGlvbl9wYWdlcz0zODktOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlYW5ub3RhdGlvbiBvZiBwcm90ZWluLWNvZGluZyBnZW5lcyBiYXNlZCBvbiBhbiBpbXByb3ZlZCBncmFwaGljYWwgcmVwcmVzZW50YXRpb24gb2YgRE5BIHNlcXVlbmNlOyBKLiBDb21wdXQuIENoZW0uOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MjEyNi0zNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBkaXNjb3ZlcnkgYnkgZ2Vub21lLXdpZGUgQ0RTIHJlLXByZWRpY3Rpb24gYW5kIG1pY3JvYXJyYXktYmFzZWQgdHJhbnNjcmlwdGlvbmFsIGFuYWx5c2lzIGluIHBoeXRvcGF0aG9nZW4gWGFudGhvbW9uYXMgY2FtcGVzdHJpczsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MzU5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSByZS1hbm5vdGF0aW9uIGluIGdlbm9tZSBvZiB0aGUgZXh0cmVtb3BoaWxlIFB5cm9iYWN1bHVtIGFlcm9waGlsdW0gYnkgdXNpbmcgYmlvaW5mb3JtYXRpY3MgbWV0aG9kczsgSi4gQmlvbW9sLiBTdHJ1Y3QuIER5bi47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz0zOTEtNDAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWFubm90YXRpb24gb2YgaHlwb3RoZXRpY2FsIE9SRnMgaW4gcGxhbnQgcGF0aG9nZW4gRXJ3aW5pYSBjYXJvdG92b3JhIHN1YnNwLiBhdHJvc2VwdGljYSBTQ1JJMTA0MzsgRkVCUyBKLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yNzU7ICBjaXRhdGlvbl9wYWdlcz0xOTgtMjA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm90ZW9tZSBkcml2ZW4gcmUtZXZhbHVhdGlvbiBhbmQgZnVuY3Rpb25hbCBhbm5vdGF0aW9uIG9mIHRoZSBTdHJlcHRvY29jY3VzIHB5b2dlbmVzIFNGMzcwIGdlbm9tZTsgQk1DIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0yNDkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGV0ZSBnZW5vbWUgc2VxdWVuY2Ugb2YgTmVpc3NlcmlhIG1lbmluZ2l0aWRpcyBzZXJvZ3JvdXAgQiBzdHJhaW4gTUM1ODsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yODc7ICBjaXRhdGlvbl9wYWdlcz0xODA5LTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MYXJpYmFjdGVyIGhvbmdrb25nZW5zaXMgZ2VuLiBub3YuLCBzcC4gbm92LiwgYSBub3ZlbCBHcmFtLW5lZ2F0aXZlIGJhY3Rlcml1bSBpc29sYXRlZCBmcm9tIGEgY2lycmhvdGljIHBhdGllbnQgd2l0aCBiYWN0ZXJlbWlhIGFuZCBlbXB5ZW1hOyBKLiBDbGluLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9NDIyNy0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNzb2NpYXRpb24gb2YgTGFyaWJhY3RlciBob25na29uZ2Vuc2lzIGluIGNvbW11bml0eS1hY3F1aXJlZCBnYXN0cm9lbnRlcml0aXMgd2l0aCB0cmF2ZWwgYW5kIGVhdGluZyBmaXNoOiBhIG11bHRpY2VudHJlIGNhc2UtY29udHJvbCBzdHVkeTsgTGFuY2V0OyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTM2MzsgIGNpdGF0aW9uX3BhZ2VzPTE5NDEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGNvbXBsZXRlIGdlbm9tZSBhbmQgcHJvdGVvbWUgb2YgTGFyaWJhY3RlciBob25na29uZ2Vuc2lzIHJldmVhbCBwb3RlbnRpYWwgbWVjaGFuaXNtcyBmb3IgYWRhcHRhdGlvbnMgdG8gZGlmZmVyZW50IHRlbXBlcmF0dXJlcyBhbmQgaGFiaXRhdHM7IFBMb1MgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDA0MTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYWN0ZXJlbWlhIGNhdXNlZCBieSBMYXJpYmFjdGVyIGhvbmdrb25nZW5zaXMgbWlzaWRlbnRpZmllZCBhcyBBY2luZXRvYmFjdGVyIGx3b2ZmaWk6IHJlcG9ydCBvZiB0aGUgZmlyc3QgY2FzZSBpbiBLb3JlYTsgSi4gS29yZWFuIE1lZC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTY3OS04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEdlbm9tZXMgT25MaW5lIERhdGFiYXNlIChHT0xEKSB2LjQ6IHN0YXR1cyBvZiBnZW5vbWljIGFuZCBtZXRhZ2Vub21pYyBwcm9qZWN0cyBhbmQgdGhlaXIgYXNzb2NpYXRlZCBtZXRhZGF0YTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9RDU3MS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OQ0JJIHJlZmVyZW5jZSBzZXF1ZW5jZXMgKFJlZlNlcSk6IGEgY3VyYXRlZCBub24tcmVkdW5kYW50IHNlcXVlbmNlIGRhdGFiYXNlIG9mIGdlbm9tZXMsIHRyYW5zY3JpcHRzIGFuZCBwcm90ZWluczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9RDYxLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pc3NpbmcgZ2VuZXMgaW4gdGhlIGFubm90YXRpb24gb2YgcHJva2FyeW90aWMgZ2Vub21lczsgQk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MTMxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9WkNVUlZFOiBhIG5ldyBzeXN0ZW0gZm9yIHJlY29nbml6aW5nIHByb3RlaW4tY29kaW5nIGdlbmVzIGluIGJhY3RlcmlhbCBhbmQgYXJjaGFlYWwgZ2Vub21lczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MTc4MC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm90ZWluIGRhdGFiYXNlIHNlYXJjaGVzIHVzaW5nIGNvbXBvc2l0aW9uYWxseSBhZGp1c3RlZCBzdWJzdGl0dXRpb24gbWF0cmljZXM7IEZFQlMgSi47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjcyOyAgY2l0YXRpb25fcGFnZXM9NTEwMS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWNvZ25pdGlvbiBvZiBwcm90ZWluIGNvZGluZyBnZW5lcyBpbiB0aGUgeWVhc3QgZ2Vub21lIGF0IGJldHRlciB0aGFuIDk1JSBhY2N1cmFjeSBiYXNlZCBvbiB0aGUgWiBjdXJ2ZTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MjgwNC0xNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RElHQeKAk2EgZGF0YWJhc2Ugb2YgaW1wcm92ZWQgZ2VuZSBhbm5vdGF0aW9uIGZvciBwaHl0b3BhdGhvZ2VuczsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJpc29uIG9mIHZhcmlvdXMgYWxnb3JpdGhtcyBmb3IgcmVjb2duaXppbmcgc2hvcnQgY29kaW5nIHNlcXVlbmNlcyBvZiBodW1hbiBnZW5lczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz02NzMtODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb21wbGV0ZSBnZW5vbWUgc2VxdWVuY2Ugb2YgKkNocm9tb2JhY3Rlcml1bSB2aW9sYWNldW0gcmV2ZWFscyByZW1hcmthYmxlIGFuZCBleHBsb2l0YWJsZSBiYWN0ZXJpYWwgYWRhcHRhYmlsaXR5OyBjaXRhdGlvbl9hdXRob3I9QnJhemlsaWFuIE5hdGlvbmFsIEdlbm9tZSBQcm9qZWN0IENvbnNvcnRpdW07IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEwMDsgIGNpdGF0aW9uX3BhZ2VzPTExNjYwLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb2RpZ2FsOiBwcm9rYXJ5b3RpYyBnZW5lIHJlY29nbml0aW9uIGFuZCB0cmFuc2xhdGlvbiBpbml0aWF0aW9uIHNpdGUgaWRlbnRpZmljYXRpb247IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTExOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBhdXRvbWF0aWMgYW5ub3RhdGlvbiBvZiBiYWN0ZXJpYWwgZ2Vub21lczsgQnJpZWYgQmlvaW5mb3JtOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MS0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHJlLWFubm90YXRpb246IGEgd2lraSBzb2x1dGlvbj87IEdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MTAyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBncmFwaGljIGFwcHJvYWNoIHRvIGFuYWx5emluZyBjb2RvbiB1c2FnZSBpbiAxNTYyIEVzY2hlcmljaGlhIGNvbGkgcHJvdGVpbiBjb2Rpbmcgc2VxdWVuY2VzOyBKLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTIzODsgIGNpdGF0aW9uX3BhZ2VzPTEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2V2ZW4gR0MtcmljaCBtaWNyb2JpYWwgZ2Vub21lcyBhZG9wdCBzaW1pbGFyIGNvZG9uIHVzYWdlIHBhdHRlcm5zIHJlZ2FyZGxlc3Mgb2YgdGhlaXIgcGh5bG9nZW5ldGljIGxpbmVhZ2VzOyBCaW9jaGVtLiBCaW9waHlzLiBSZXMuIENvbW11bi47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzA2OyAgY2l0YXRpb25fcGFnZXM9MzEwLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBkaXN0cmlidXRpb24gcGF0dGVybnMgb2YgYmFzZXMgb2YgcHJvdGVpbi1jb2RpbmcgZ2VuZXMsIG5vbi1jb2RpbmcgT1JGcywgYW5kIGludGVyZ2VuaWMgc2VxdWVuY2VzIGluIFBzZXVkb21vbmFzIGFlcnVnaW5vc2EgUEEwMSBnZW5vbWUgYW5kIGl0cyBpbXBsaWNhdGlvbnM7IEouIEJpb21vbC4gU3RydWN0LiBEeW4uOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MTI3LTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Vc2luZyB0aGUgQ09HIGRhdGFiYXNlIHRvIGltcHJvdmUgZ2VuZSByZWNvZ25pdGlvbiBpbiBjb21wbGV0ZSBnZW5vbWVzOyBHZW5ldGljYTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDg7ICBjaXRhdGlvbl9wYWdlcz05LTE3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBpbnRlZ3JhdGl2ZSBtZXRob2QgZm9yIGlkZW50aWZ5aW5nIHRoZSBvdmVyLWFubm90YXRlZCBwcm90ZWluLWNvZGluZyBnZW5lcyBpbiBtaWNyb2JpYWwgZ2Vub21lczsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz00MzUtNDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSByZWFubm90YXRpb24gb2YgRXNjaGVyaWNoaWEgY29saSBDRlQwNzMgd2l0aCBuZXcgaW5zaWdodHMgaW50byB2aXJ1bGVuY2U7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTU1MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhvdyB0byBpbnRlcnByZXQgYW4gYW5vbnltb3VzIGJhY3RlcmlhbCBnZW5vbWU6IG1hY2hpbmUgbGVhcm5pbmcgYXBwcm9hY2ggdG8gZ2VuZSBpZGVudGlmaWNhdGlvbjsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTExNTQtNzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBsZXRlIGdlbm9tZSBzZXF1ZW5jZSBvZiBOZWlzZXJyaWEgZ29ub3JyaG9lYWUgTkNDUDExOTQ1OyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE5MDsgIGNpdGF0aW9uX3BhZ2VzPTYwMzUtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SUNEUyBkYXRhYmFzZTogaW50ZXJydXB0ZWQgQ29EaW5nIHNlcXVlbmNlcyBpbiBwcm9rYXJ5b3RpYyBnZW5vbWVzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz1EMzM4LTQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHBpbG90IHN0dWR5IG9mIGJhY3RlcmlhbCBnZW5lcyB3aXRoIGRpc3J1cHRlZCBPUkZzIHJldmVhbHMgYSBzdXJwcmlzaW5nIHByb2Z1c2lvbiBvZiBwcm90ZWluIHNlcXVlbmNlIHJlY29kaW5nIG1lZGlhdGVkIGJ5IHJpYm9zb21hbCBmcmFtZXNoaWZ0aW5nIGFuZCB0cmFuc2NyaXB0aW9uYWwgcmVhbGlnbm1lbnQ7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz0zMTk1LTIxMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIwLzMvMjczLzEwNTE3NzcvZHN0MDA5LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBJbiB0aGlzIHBhcGVyLCB3ZSBwZXJmb3JtZWQgYSBjb21wcmVoZW5zaXZlIHJlLWFubm90YXRpb24gb2YgcHJvdGVpbi1jb2RpbmcgZ2VuZXMgYnkgYSBzeXN0ZW1hdGljIG1ldGhvZCBjb21iaW5pbmcgY29tcG9zaXRpb24tIGFuZCBzaW1pbGFyaXR5LWJhc2VkICIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MzY5MTc5JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZJASdBug90Yujsi4Dw3xrMtTgV5q" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f3%2f273%2f369179" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f3%2f273%2f369179" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=14337" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/20/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/3/1/m_cover.gif?Expires=2147483647&amp;Signature=3JNT-3uWcoFSRSDjpyRS683cAWcv-uPBeo5zUjK0TFHyw5pdGPApuAnjwKw9hziMynOyeupAu8qXTBcWNNZ5zwCIK3wDotNETRf1C0gT1NeaWwrIBlHGIKu6bYD8JmeQgULL4qaQcRAHnyo7FyqKjoT8e0y4GLL7JPBIG8ClvgMIpmjvKRKJfK-rkhDpE9pXuYo8ex4C6o9vQhlNIWM8gIl4ygPsB2y8BMTIUXCEDS8KZ8W94~SHycoNAwx1VF1wN~gGUhCSNthMU3E6MsUONjgAq-FWmQ8kuAdsyKdxNSJy8mhuDD9SBTDoUvN5zzz~5gwbw0AeOX0S-zArQW9D7g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/20/3">
                        <div class="volume trailing-comma">Volume 20</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2013        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5359925">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5359925">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5359927">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5359927">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5359933">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5359933">2. Material and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5359947">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5359947">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5359980">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5359980">4. Discussions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5359984">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5359984">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5359986">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5359986">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5359988">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5359988">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5359991">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5359991">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/20/3/263/368846">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/20/3/287/369399">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Re-Annotation of Protein-Coding Genes in 10 Complete Genomes of Neisseriaceae Family by Combining Similarity-Based and Composition-Based Methods
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Feng-Biao Guo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Microbiology</div>, <div class="insititution">The University of Hong Kong, Special Administrative Region</div>, <div class="country">Hong Kong</div>, <div class="country">People's Republic of China</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">School of Life Science and Technology</div>, <div class="insititution">University of Electronic Science and Technology of China</div>, <div class="addr-line">Chengdu</div><div class="addr-line">610054</div>, <div class="country">People's Republic of China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Feng-Biao+Guo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Guo F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Guo F&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Feng-Biao Guo</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Lifeng Xiong</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Microbiology</div>, <div class="insititution">The University of Hong Kong, Special Administrative Region</div>, <div class="country">Hong Kong</div>, <div class="country">People's Republic of China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Lifeng+Xiong">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Xiong L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Xiong L&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Lifeng Xiong</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jade L. L. Teng</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Microbiology</div>, <div class="insititution">The University of Hong Kong, Special Administrative Region</div>, <div class="country">Hong Kong</div>, <div class="country">People's Republic of China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jade+L.+L.+Teng">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Teng J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Teng J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jade L. L. Teng</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kwok-Yung Yuen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Microbiology</div>, <div class="insititution">The University of Hong Kong, Special Administrative Region</div>, <div class="country">Hong Kong</div>, <div class="country">People's Republic of China</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">State Key Laboratory of Emerging Infectious Diseases</div>, <div class="insititution">The University of Hong Kong, Special Administrative Region</div>, <div class="country">Hong Kong</div>, <div class="country">People's Republic of China</div></div><div class="aff"><div class="label -label">4</div><div class="addr-line">Research Centre of Infection and Immunology</div>, <div class="insititution">The University of Hong Kong, Special Administrative Region</div>, <div class="country">Hong Kong</div>, <div class="country">People's Republic of China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kwok-Yung+Yuen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yuen K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yuen K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kwok-Yung Yuen</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Susanna K. P. Lau</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Microbiology</div>, <div class="insititution">The University of Hong Kong, Special Administrative Region</div>, <div class="country">Hong Kong</div>, <div class="country">People's Republic of China</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">State Key Laboratory of Emerging Infectious Diseases</div>, <div class="insititution">The University of Hong Kong, Special Administrative Region</div>, <div class="country">Hong Kong</div>, <div class="country">People's Republic of China</div></div><div class="aff"><div class="label -label">4</div><div class="addr-line">Research Centre of Infection and Immunology</div>, <div class="insititution">The University of Hong Kong, Special Administrative Region</div>, <div class="country">Hong Kong</div>, <div class="country">People's Republic of China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. 852-22554897. Fax. 852-28551241. E-mail: <a href="mailto:pcywoo@hkucc.hku.hk" target="_blank">pcywoo@hkucc.hku.hk</a> (P.C.Y.W.) or <a href="mailto:skplau@hkucc.hku.hk" target="_blank">skplau@hkucc.hku.hk</a> (S.K.P.L.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Susanna+K.+P.+Lau">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lau S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lau S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Susanna K. P. Lau</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Patrick C. Y. Woo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Microbiology</div>, <div class="insititution">The University of Hong Kong, Special Administrative Region</div>, <div class="country">Hong Kong</div>, <div class="country">People's Republic of China</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">State Key Laboratory of Emerging Infectious Diseases</div>, <div class="insititution">The University of Hong Kong, Special Administrative Region</div>, <div class="country">Hong Kong</div>, <div class="country">People's Republic of China</div></div><div class="aff"><div class="label -label">4</div><div class="addr-line">Research Centre of Infection and Immunology</div>, <div class="insititution">The University of Hong Kong, Special Administrative Region</div>, <div class="country">Hong Kong</div>, <div class="country">People's Republic of China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. 852-22554897. Fax. 852-28551241. E-mail: <a href="mailto:pcywoo@hkucc.hku.hk" target="_blank">pcywoo@hkucc.hku.hk</a> (P.C.Y.W.) or <a href="mailto:skplau@hkucc.hku.hk" target="_blank">skplau@hkucc.hku.hk</a> (S.K.P.L.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Patrick+C.+Y.+Woo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Woo P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Woo P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Patrick C. Y. Woo</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 20, Issue 3, 1 June 2013, Pages 273–286, <a href='https://doi.org/10.1093/dnares/dst009'>https://doi.org/10.1093/dnares/dst009</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">09 April 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">21 September 2012</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">08 March 2013</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="369179" href="/dnaresearch/article-pdf/20/3/273/1051777/dst009.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Feng-Biao Guo, Lifeng Xiong, Jade L. L. Teng, Kwok-Yung Yuen, Susanna K. P. Lau, Patrick C. Y. Woo;  Re-Annotation of Protein-Coding Genes in 10 Complete Genomes of Neisseriaceae Family by Combining Similarity-Based and Composition-Based Methods, <em>DNA Research</em>, Volume 20, Issue 3, 1 June 2013, Pages 273–286, <a href='https://doi.org/10.1093/dnares/dst009'>https://doi.org/10.1093/dnares/dst009</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/369179?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/369179?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/369179?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/369179?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/369179?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/369179?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/369179?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Re-Annotation%20of%20Protein-Coding%20Genes%20in%2010%20Complete%20Genomes%20of%20Neisseriaceae%20Family%20by%20Combining%20Similarity-Based%20and%20Composition-Based%20Methods&amp;publicationDate=2013-04-09&amp;volumeNum=20&amp;issueNum=3&amp;author=Guo%2C%20Feng-Biao%3B%20Xiong%2C%20Lifeng&amp;startPage=273&amp;endPage=286&amp;contentId=10.1093%2Fdnares%2Fdst009&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f3%2f273%2f369179" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5359925 id="5359925" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>In this paper, we performed a comprehensive re-annotation of protein-coding genes by a systematic method combining composition- and similarity-based approaches in 10 complete bacterial genomes of the family Neisseriaceae. First, 418 hypothetical genes were predicted as non-coding using the composition-based method and 413 were eliminated from the gene list. Both the scatter plot and cluster of orthologous groups (COG) fraction analyses supported the result. Second, from 20 to 400 hypothetical proteins were assigned with functions in each of the 10 strains based on the homology search. Among newly assigned functions, 397 are so detailed to have definite gene names. Third, 106 genes missed by the original annotations were picked up by an <em>ab initio</em> gene finder combined with similarity alignment. Transcriptional experiments validated the effectiveness of this method in <em>Laribacter hongkongensis</em> and <em>Chromobacterium violaceum</em>. Among the 106 newly found genes, some deserve particular interests. For example, 27 transposases were newly found in <em>Neiserria meningitidis</em> alpha14. In <em>Neiserria gonorrhoeae</em> NCCP11945, four new genes with putative functions and definite names (<em>nusG</em>, <em>rpsN</em>, <em>rpmD</em> and <em>infA</em>) were found and homologues of them usually are essential for survival in bacteria. The updated annotations for the 10 Neisseriaceae genomes provide a more accurate prediction of protein-coding genes and a more detailed functional information of hypothetical proteins. It will benefit research into the lifestyle, metabolism, environmental adaption and pathogenicity of the Neisseriaceae species. The re-annotation procedure could be used directly, or after the adaption of detailed methods, for checking annotations of any other bacterial or archaeal genomes.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;the Neisseriaceae family&quot;">the Neisseriaceae family</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="re-annotation">re-annotation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;newly found genes&quot;">newly found genes</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;eliminated non-coding ORFs&quot;">eliminated non-coding ORFs</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;newly assigned functions&quot;">newly assigned functions</a></div>                            <h2 scrollto-destination=5359927 id="5359927" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The emergence of next-generation DNA sequencing techniques accelerate tremendously the increment of sequences deposited in public nucleotide databases. The wealth of sequence data stimulates wonderful opportunity to understand the biological process of various living species. To achieve this aim, two of the essential steps are identifying all protein-coding genes and trying to assign their functions. They are jointly named as genome annotation. The quality of the genome annotation is very vital. If one genome could not be annotated accurately but still submitted to the public database, not only subsequent researches based on it may encounter problems but also annotations of after sequenced closely related genomes would be influenced. The annotation errors may propagate and finally affect more and more genomes. Recognizing this serious problem, Ouzounis and Kap<sup><a reveal-id="DST009C1" data-open="DST009C1" class="link link-ref link-reveal xref-bibr">1</a></sup> appealed to update regularly the genome annotation by latest database and methods.</p><p>As for prokaryotes, dozens of genomes have been re-annotated.<sup><a reveal-id="DST009C2" data-open="DST009C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Three kinds of re-annotations are often performed in sequenced prokaryotic genomes. First, and rather early, falsely predicted protein-coding genes are eliminated from the original annotation using composition-based methods. For example, Wang and Zhang<sup><a reveal-id="DST009C3" data-open="DST009C3" class="link link-ref link-reveal xref-bibr">3</a></sup> suggested that 172 annotated genes were very unlikely to encode proteins in the genome of <em>Vibrio cholerae</em> based on single-nucleotide frequencies. One of the typical re-annotation cases of archaea was associated with <em>Aeropyrum pernix</em> K1, in which protein-coding genes were over-annotated up to 60% by the original sequencing institute.<sup><a reveal-id="DST009C4  DST009C6" data-open="DST009C4  DST009C6" class="link link-ref link-reveal xref-bibr">4–6</a></sup> It is lucky that this major error has been corrected by using proteome approaches and bioinformatics methods.<sup><a reveal-id="DST009C7  DST009C10" data-open="DST009C7  DST009C10" class="link link-ref link-reveal xref-bibr">7–10</a></sup><em>Amsacta moorei</em> entomopoxvirus may have the most over-annotated protein-coding genes among sequenced viruses.<sup><a reveal-id="DST009C11" data-open="DST009C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Guo and Yu<sup><a reveal-id="DST009C12" data-open="DST009C12" class="link link-ref link-reveal xref-bibr">12</a></sup> suggested that ∼38 of 294 originally annotated genes did not encode proteins based on the Z-curve method. By using another graphical method, Yu and Sun<sup><a reveal-id="DST009C13" data-open="DST009C13" class="link link-ref link-reveal xref-bibr">13</a></sup> confirmed this speculation.</p><p>Second, some genes may be missed by the original annotation and could be picked up by the <em>ab initio</em> gene finding method and further confirmed by the similarity alignment or transcription and/or protein expression proofs. For example, Zhou <em>et al</em>.<sup><a reveal-id="DST009C14" data-open="DST009C14" class="link link-ref link-reveal xref-bibr">14</a></sup> newly added 278 potential genes by the similarity alignment and another 147 by detectable mRNA transcriptions in the genome of <em>Xanthomonas campestris</em>. Very recently, Du <em>et al</em>.<sup><a reveal-id="DST009C15" data-open="DST009C15" class="link link-ref link-reveal xref-bibr">15</a></sup> newly added eight potential genes by the similarity-based method in the archaeon <em>Pyrobaculum aerophilum</em>.</p><p>Assigning functions to hypothetical proteins constitutes the last kind of re-annotation. This type of re-annotation may be performed by using the homology alignment or by functional genomic experiments. For example, 149 hypothetical proteins were assigned detailed functions according to the strict homology information in the genome of <em>Erwinia carotovora</em>.<sup><a reveal-id="DST009C16" data-open="DST009C16" class="link link-ref link-reveal xref-bibr">16</a></sup> A similar method was employed to the genome of <em>P. aerophilum</em> and 80 hypothetical proteins were assigned with functional information.<sup><a reveal-id="DST009C15" data-open="DST009C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Based on cellular fractions and expression profiles under different culture conditions, Okamoto and Yamada<sup><a reveal-id="DST009C17" data-open="DST009C17" class="link link-ref link-reveal xref-bibr">17</a></sup> provided general functional information for 126 hypothetical proteins in the genome of <em>Streptococcus pyogenes</em>.</p><p>In this study, we performed all three types of re-annotation in 10 complete genomes of the Neisseriaceae family. As far as our knowledge goes, the only example involved with all three types of re-annotation is the updated annotation in the genome of <em>P. aerophilum</em>.<sup><a reveal-id="DST009C15" data-open="DST009C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Through them, the outdated annotation may be corrected as far as possible. However, alternative approaches may be utilized to obtain similar results with the systematic method used here. Compared with our previous work,<sup><a reveal-id="DST009C15" data-open="DST009C15" class="link link-ref link-reveal xref-bibr">15</a></sup> here, we used transcriptional analyses to validate the effectiveness of our method to pick up new genes. The Neisseriaceae family belongs to β-proteobacteria. Among the 10 Neisseriaceae strains analyzed in this work, seven belong to the genus <em>Neisseria</em> and all can colonize the mucosal surfaces of many animals. <em>Neiserria meningitidis</em>, as one of the most common causes of bacterial meningitis, are most virulent in human.<sup><a reveal-id="DST009C18" data-open="DST009C18" class="link link-ref link-reveal xref-bibr">18</a></sup><em>Laribacter hongkongensis</em> is a recently sequenced bacterium associated with invasive blood stream infections in patients with liver cirrhosis as well as gastroenteritis and traveler's diarrhea.<sup><a reveal-id="DST009C19  DST009C22" data-open="DST009C19  DST009C22" class="link link-ref link-reveal xref-bibr">19–22</a></sup> Updated annotations of these bacterial strains would help to understand their pathogenicities and environment adaptation capacities.</p>                            <h2 scrollto-destination=5359933 id="5359933" class="section-title" data-legacy-id=s2>2. Material and methods</h2>
                            <h3 scrollto-destination=5359934 id="5359934" class="section-title" data-legacy-id=s2a>2.1 Data source</h3>
<p>Ten complete genomes of the family Neisseriaceae were included in this work. They were <em>Chromobacterium violaceum</em> ATCC 12472 (RefSeq accession number: NC_005085), <em>L. hongkongensis</em> HLHK9 (NC_012559), <em>N. gonorrhoeae</em> FA 1090 (NC_002946), <em>N. gonorrhoeae</em> NCCP11945 (NC_011035), <em>N. lactamica</em> 020-06 (NC_014752), <em>N. meningitidi</em>s 053442 (NC_010120), <em>N. meningitidis</em> alpha14 (NC_013016), <em>N. meningitidis</em> MC58 (NC_0031112), <em>N. meningitidis</em> Z2491 (NC_003116) and <em>Pseudogulbenkiania</em> sp. NH8B (NC_016002). Among them, two strains of <em>N. gonorrhoeae</em> and four strains of <em>N. meningitidis</em> and <em>L. hongkongensis</em> are pathogens. In fact, dozens of strains in the family Neisseriaceae have been sequenced.<sup><a reveal-id="DST009C23" data-open="DST009C23" class="link link-ref link-reveal xref-bibr">23</a></sup> However, the NCBI RefSeq project provides curated annotations only for these strains.<sup><a reveal-id="DST009C24" data-open="DST009C24" class="link link-ref link-reveal xref-bibr">24</a></sup> In this work, we chose the 10 complete genomes to perform re-annotation.</p>                            <h3 scrollto-destination=5359936 id="5359936" class="section-title" data-legacy-id=s2b>2.2 Method to pick up missed genes</h3>
<p>In each sequenced genome, there are always some bona fide genes that have been missed by the original annotation. For example, Warren <em>et al</em>.<sup><a reveal-id="DST009C25" data-open="DST009C25" class="link link-ref link-reveal xref-bibr">25</a></sup> uncovered 38 895 intergenic open reading frames (ORFs), readily identified as putative genes by similarity to currently annotated genes, from 1297 prokaryotic replicons based on across-genome alignment. New genes could be confirmed by the similarity alignment or by transcription or proteome analyses. In this work, we first used the ZCURVE<sup><a reveal-id="DST009C26" data-open="DST009C26" class="link link-ref link-reveal xref-bibr">26</a></sup> program, which is freely available at <a class="link link-uri" href="http://tubic.tju.edu.cn/Zcurve_B/">http://tubic.tju.edu.cn/Zcurve_B/</a>, to pick out all candidate genes that did not have same 5′ terminals with all genes in the original annotation. The candidate may overlap one annotated gene with their sequences but they did not correspond to the same reading frame. Then those candidate new genes would be filtered by blast<sup><a reveal-id="DST009C27" data-open="DST009C27" class="link link-ref link-reveal xref-bibr">27</a></sup> against the NCBI nr database. If one candidate met the following three conditions, it would be regarded as genuine genes: (i) it had the significant similarity (<em>E</em>-value &lt; 10<sup>−20</sup>, Coverage &gt; 60% and Identity &gt; 50%) with annotated genes in bacteria beyond the same genus in the database, and it had the similar length with the counterpart (difference &lt; 20%); (ii) it had counterpart in the cluster of orthologous groups (COG) database, i.e. it could be assigned within an existed COG cluster; (iii) it did not overlap annotated genes with any bases or it had a smaller <em>E</em>-value and a higher identity score against functional genes in the other genomes in the case of overlapping. This process has been standardized in this work. For each of the 10 strains, all parameters in the whole process were fixed.</p>                            <h3 scrollto-destination=5359938 id="5359938" class="section-title" data-legacy-id=s2c>2.3 Method to eliminate over-annotated genes</h3>
<p>A composition-based method was used to eliminate over-annotated genes. The method is based on the <em>Z</em>-curve representation of the DNA sequences, which has been successfully used to find genes in various microbes.<sup><a reveal-id="DST009C3" data-open="DST009C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DST009C7" data-open="DST009C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DST009C9" data-open="DST009C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DST009C12" data-open="DST009C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DST009C15" data-open="DST009C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DST009C16" data-open="DST009C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DST009C28" data-open="DST009C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DST009C29" data-open="DST009C29" class="link link-ref link-reveal xref-bibr">29</a></sup> In this analysis, 33 <em>Z</em>-curve variables were adopted,<sup><a reveal-id="DST009C26" data-open="DST009C26" class="link link-ref link-reveal xref-bibr">26</a></sup> including nine variables of phase-dependent single-nucleotide frequencies<sup><a reveal-id="DST009C28" data-open="DST009C28" class="link link-ref link-reveal xref-bibr">28</a></sup> and 24 of phase-dependent dinucleotide frequencies.<sup><a reveal-id="DST009C26" data-open="DST009C26" class="link link-ref link-reveal xref-bibr">26</a></sup> In fact, there are 36 variables denoting phase-dependent dinucleotides.<sup><a reveal-id="DST009C30" data-open="DST009C30" class="link link-ref link-reveal xref-bibr">30</a></sup> However, long-range correlations between the first and third codon positions tend to be weaker and so 12 variables associated with them were discarded. For details about these variables, refer to Guo <em>et al</em>.<sup><a reveal-id="DST009C26" data-open="DST009C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Besides 33 classifying features, the Fisher linear discrimination algorithm was used to optimally differentiate protein-coding and non-coding sequences, the procedure was as detailed previously.<sup><a reveal-id="DST009C28" data-open="DST009C28" class="link link-ref link-reveal xref-bibr">28</a></sup> The training set of the classifying model comprised a positive sample set and a negative sample set. The positive sample set was those function-known genes with definite names, e.g. <em>gyrB</em> as the name of gene encoding protein of DNA gyrase subunit B. The negative sample was generated by a randomly shuffling sequence of the positive sample and thus destroying its natural structure. After parameters have been trained based on positive and negative samples, all hypothetical proteins would be decided to be genuine genes or falsely annotated non-coding sequences. The latter would be eliminated in the updated annotation. The scatter plot in Figure <a reveal-id="DST009F1" data-open="DST009F1" class="link link-reveal link-table xref-fig">1</a> illustrates the effectiveness of the method to eliminate non-coding ORFs from the collection of hypothetical proteins. A web server has been constructed to check hypothetical genes and eliminate non-coding ORFs in any sequenced bacterial or archaeal genomes, which is freely available at <a class="link link-uri" href="http://147.8.74.24/Zfisher/">http://147.8.74.24/Zfisher/</a>. </p>                        <a id="5359940" scrollto-destination="5359940"></a>
<div id="DST009F1" data-id="DST009F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst009/2/m_dst00901.jpeg?Expires=2147483647&Signature=Q8k5b7X0FOf3LuzD9Zx3K8Q8bmm8l7BnHxjhIcWQuHg31gdVlNfIc8r8jO1Lnv0-2LP9nGzSHDFAubYV9WrYjp6i3W9NC~r347r2yGSjhcJT2WWh7RV7KHAqDhTNYOVj9004S0stZbfE4FhZZHK4TpNV8mvvm2d4NInWMdrYf6eqfNGm90jgqIRqKPyTL7NqZC-~pM6w0KsuUEM9rM76FcM~35yy0ym5Qd0Af0LQ4xMSwqA~aHU3vnCMgq-y4V4KPzERSxGBzUSu8tLohAPOA44M~IQJmYN7MOZrbtFn6R-BSBoeAaf53WPyy0MpG0bX6tw8CWbZY7qOjUDziSc4cg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of GC2 versus GC3 for four types of sequences. The x-axis indicates the value of GC2 and the y-axis denotes the value of GC3. (A) For 969 function-known genes in L. hongkongensis; (B) for 86 predicted non-coding genes in L. hongkongensis; (C) for 915 retained hypothetical genes in L. hongkongensis and (D) for 20 horizontally transferred genes in P. aeruginosa." path-from-xml="dst00901.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5359940/dst00901.jpeg" path-from-xml="dst00901.jpeg" target="_blank">View large</a><a section="5359940" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst009/2/dst00901.jpeg?Expires=2147483647&Signature=oGMPPR6u3gSzA3A58~lZ0CZQclJKZvikfa0E7f6fUxeLWlYsjHgCHQbxVREpNq8aTygJZJI6TKfh8zzBNEPUAycKyqtfx3dYlxrWjDkxphXCFNd2cfx4p3IuMGzkgaiCEhqMMNO48eKnaXy5~ABRTgMRmjj0XKC~9IP1i08udvM-L~oZ7IqtmlemgZTa-16Am-6Rb3rg9X9qj-zKbErJutQrYaARCELrUbBPUByHu1pKaafAVXfHeC~nt-80CnYl5AVR6pp2tykWdeXN6A1P7qCh0aXYZGbec2imqnSkMjYfsg9MxbtWD~N21bEyc5kOlD502lurkiNDAWuYNXE6Jw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5359940&ar=369179&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00901.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of GC<sub>2</sub> versus GC<sub>3</sub> for four types of sequences. The <em>x</em>-axis indicates the value of GC<sub>2</sub> and the <em>y</em>-axis denotes the value of GC<sub>3</sub>. (A) For 969 function-known genes in <em>L. hongkongensis</em>; (B) for 86 predicted non-coding genes in <em>L. hongkongensis</em>; (C) for 915 retained hypothetical genes in <em>L. hongkongensis</em> and (D) for 20 horizontally transferred genes in <em>P. aeruginosa</em>.</p></div></div><div content-id="DST009F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst009/2/m_dst00901.jpeg?Expires=2147483647&Signature=Q8k5b7X0FOf3LuzD9Zx3K8Q8bmm8l7BnHxjhIcWQuHg31gdVlNfIc8r8jO1Lnv0-2LP9nGzSHDFAubYV9WrYjp6i3W9NC~r347r2yGSjhcJT2WWh7RV7KHAqDhTNYOVj9004S0stZbfE4FhZZHK4TpNV8mvvm2d4NInWMdrYf6eqfNGm90jgqIRqKPyTL7NqZC-~pM6w0KsuUEM9rM76FcM~35yy0ym5Qd0Af0LQ4xMSwqA~aHU3vnCMgq-y4V4KPzERSxGBzUSu8tLohAPOA44M~IQJmYN7MOZrbtFn6R-BSBoeAaf53WPyy0MpG0bX6tw8CWbZY7qOjUDziSc4cg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of GC2 versus GC3 for four types of sequences. The x-axis indicates the value of GC2 and the y-axis denotes the value of GC3. (A) For 969 function-known genes in L. hongkongensis; (B) for 86 predicted non-coding genes in L. hongkongensis; (C) for 915 retained hypothetical genes in L. hongkongensis and (D) for 20 horizontally transferred genes in P. aeruginosa." path-from-xml="dst00901.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5359940/dst00901.jpeg" path-from-xml="dst00901.jpeg" target="_blank">View large</a><a section="5359940" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst009/2/dst00901.jpeg?Expires=2147483647&Signature=oGMPPR6u3gSzA3A58~lZ0CZQclJKZvikfa0E7f6fUxeLWlYsjHgCHQbxVREpNq8aTygJZJI6TKfh8zzBNEPUAycKyqtfx3dYlxrWjDkxphXCFNd2cfx4p3IuMGzkgaiCEhqMMNO48eKnaXy5~ABRTgMRmjj0XKC~9IP1i08udvM-L~oZ7IqtmlemgZTa-16Am-6Rb3rg9X9qj-zKbErJutQrYaARCELrUbBPUByHu1pKaafAVXfHeC~nt-80CnYl5AVR6pp2tykWdeXN6A1P7qCh0aXYZGbec2imqnSkMjYfsg9MxbtWD~N21bEyc5kOlD502lurkiNDAWuYNXE6Jw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5359940&ar=369179&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00901.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of GC<sub>2</sub> versus GC<sub>3</sub> for four types of sequences. The <em>x</em>-axis indicates the value of GC<sub>2</sub> and the <em>y</em>-axis denotes the value of GC<sub>3</sub>. (A) For 969 function-known genes in <em>L. hongkongensis</em>; (B) for 86 predicted non-coding genes in <em>L. hongkongensis</em>; (C) for 915 retained hypothetical genes in <em>L. hongkongensis</em> and (D) for 20 horizontally transferred genes in <em>P. aeruginosa</em>.</p></div></div>                            <h3 scrollto-destination=5359941 id="5359941" class="section-title" data-legacy-id=s2d>2.4 Method to assign functions to hypothetical proteins</h3>
<p>Hypothetical proteins after refining by the above process would be submitted to the nr database. Those with highly significant similarities with function-known genes in the database would be assigned the same functions. To achieve more sensitive results, amino acid (aa) sequences of hypothetical genes were actually aligned against protein sequences translated from the nr nucleotide database.<sup><a reveal-id="DST009C27" data-open="DST009C27" class="link link-ref link-reveal xref-bibr">27</a></sup> To ensure strict homology, the aligned length covered at least 80% of each gene with the identity of &gt;70% and the <em>E</em>-value of &lt;1e−20. According to the above thresholds, if one hypothetical gene with a translated aa sequence matched two or more proteins with the same functions, then the function information would be transferred to the hypothetical protein.<sup><a reveal-id="DST009C18" data-open="DST009C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p>                            <h3 scrollto-destination=5359943 id="5359943" class="section-title" data-legacy-id=s2e>2.5 Bacterial strains and growth conditions</h3>
<p><em>Laribacter hongkongensis</em> HLHK9 is a clinical isolate in Hong Kong and its complete genome sequence was available recently.<sup><a reveal-id="DST009C21" data-open="DST009C21" class="link link-ref link-reveal xref-bibr">21</a></sup> It was grown at 37°C, in brain heart infusion (BHI) broth or on BHI agar plates (BD, USA). <em>Chromobacterium</em><em>violaceum</em> ATCC 12472 is a type strain and its complete genome sequence is also available in Genbank.<sup><a reveal-id="DST009C31" data-open="DST009C31" class="link link-ref link-reveal xref-bibr">31</a></sup><em>Chromobacterium</em><em>violaceum</em> was cultured in nutrient broth or nutrient agar (Oxoid, England) at 26°C. Unless indicated otherwise, bacteria were cultured to the log-phase for experiment (∼0.6 at OD<sub>600</sub>).</p>                            <h3 scrollto-destination=5359945 id="5359945" class="section-title" data-legacy-id=s2f>2.6 Reverse transcription–polymerase chain reaction</h3>
<p>The total bacterial RNA was extracted by using the RNeasy mini kit following the manufacturer's instructions (Qiagen, Germany). Genomic DNA was removed by DNase digestion using RNase-free DNase I (Roche Diagnostic, Switzerland) as described by the manufacturer. Reverse transcription (RT) was performed using Superscript III Reverse Transcriptase (Invitrogen, Carlsbad, CA, USA) according to the manufacturer's recommendations. One microlitre of cDNA was used as a template for RT–polymerase chain reaction (PCR) with each specific primer pair. Mock RT–PCR without reverse transcriptase was also conducted as control. Triplicate assays using RNAs extracted in three independent experiments were performed for each target gene.</p>                            <h2 scrollto-destination=5359947 id="5359947" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=5359948 id="5359948" class="section-title" data-legacy-id=s3a>3.1. Eliminated non-coding ORFs and the graphic proof</h3>
<p>All <em>ab initio</em> gene finders would predict a certain number of non-coding ORFs as potential genes and these predictions constitute false positives of gene annotation.<sup><a reveal-id="DST009C32" data-open="DST009C32" class="link link-ref link-reveal xref-bibr">32</a></sup> To ensure less species-specific genes be missed from the annotation result, one or more <em>ab initio</em> gene finders are necessary in the process of annotating prokaryotic genomes.<sup><a reveal-id="DST009C33" data-open="DST009C33" class="link link-ref link-reveal xref-bibr">33</a></sup> Often, similarity alignment methods are combined with <em>ab initio</em> methods to achieve better results. Because of the intervention of the latter, it is not inevitable that some non-coding ORFs will appear in the final list of potential genes in every sequenced genome.<sup><a reveal-id="DST009C34" data-open="DST009C34" class="link link-ref link-reveal xref-bibr">34</a></sup> They may constitute source of annotation errors of closely related genomes and thus should be eliminated from the current annotations. According to the RefSeq<sup><a reveal-id="DST009C24" data-open="DST009C24" class="link link-ref link-reveal xref-bibr">24</a></sup> annotation for each Neisseriaceae strain, all annotated genes could be classified into three groups. The first group contains only function-known genes with definite names. The last group includes those genes encoding hypothetical proteins. The remaining genes constitute the second group. Obviously, the first group encodes proteins without any uncertainty. Coding potentials of the third group would be doubted to some extent. Therefore, we focus on the third group, hypothetical genes, in this work.</p><p>Numbers of genes belong to the first and third groups and genomic characteristics of each strain are listed in Table <a reveal-id="DST009TB1" data-open="DST009TB1" class="link link-reveal link-table xref-fig">1</a>. As can been seen, ratios of the numbers of the first and third class genes to the total gene number vary significantly. For example, <em>N. gonorrhoeae</em> FA 1090 has the highest ratio of hypothetical genes, whereas almost the least ratio of function-known genes with definite names. This illustrates that the function annotation in this strain is much poorer compared with the other strains. The variation of the gene ratio is associated with many factors, such as annotation methods, genomic G + C contents and the number of closely related genomes that have been sequenced when annotating the strain. </p>                        <a id="5359951" scrollto-destination="5359951"></a>
<div content-id="DST009TB1" class="table-modal"><div class="table-wrap"><div id="DST009TB1" class="table-wrap-title" data-id="DST009TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Statistical information in genomes of the 10 Neisseriaceae strains</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Strains </th><th>Published year<sup>a</sup> </th><th>Gene density (kb) </th><th>Genome size (bp) </th><th>Gene number </th><th>G + C content (%) </th><th>First class gene (ratio) </th><th>Third class gene (ratio) </th></tr></thead><tbody><tr><td><em>C. violaceum</em> </td><td>18 September 2003 </td><td>0.927 </td><td>4 751 080 </td><td>4405 </td><td>64.8 </td><td>1494 (33.9%) </td><td>1714 (38.9%) </td></tr><tr><td><em>L. hongkongensis</em> </td><td>14 April 2009 </td><td>1.021 </td><td>3 169 329 </td><td>3235 </td><td>62.4 </td><td>969 (30.0%) </td><td>1001 (30.1%) </td></tr><tr><td><em>N. gonorrhoeae</em> FA 1090 </td><td>15 February 2005 </td><td>0.930 </td><td>2 153 922 </td><td>2002 </td><td>52.7 </td><td>266 (13.3%) </td><td>800 (40.0%) </td></tr><tr><td><em>N. gonorrhoeae</em> NCCP11945 </td><td>9 July 2008 </td><td>1.195 </td><td>2 232 025 </td><td>2668 </td><td>52.4 </td><td>244 (9.1%) </td><td>996 (37.3%) </td></tr><tr><td><em>N. lactamica</em> </td><td>16 December 2010 </td><td>0.888 </td><td>2 220 606 </td><td>1972 </td><td>52.3 </td><td>744 (37.7%) </td><td>657 (33.3%) </td></tr><tr><td><em>N. meningitidis</em> 053442 </td><td>3 December 2007 </td><td>0.938 </td><td>2 153 416 </td><td>2020 </td><td>51.7 </td><td>875 (43.3%) </td><td>645 (31.9%) </td></tr><tr><td><em>N. meningitidis</em> Alpha 14 </td><td>24 July 2009 </td><td>0.872 </td><td>2 145 295 </td><td>1872 </td><td>52.0 </td><td>865 (46.2%) </td><td>467 (25.0%) </td></tr><tr><td><em>N. meningitidis</em> MC58 </td><td>10 March 2000 </td><td>0.908 </td><td>2 272 360 </td><td>2063 </td><td>51.5 </td><td>806 (39.1%) </td><td>810 (39.3%) </td></tr><tr><td><em>N. meningitidis</em> Z2491 </td><td>30 March 2000 </td><td>0.874 </td><td>2 184 406 </td><td>1909 </td><td>51.8 </td><td>459 (24.0%) </td><td>669 (35.0%) </td></tr><tr><td><em>Pseudogulbenkiania</em> </td><td>8 September 2011 </td><td>0.926 </td><td>4 332 995 </td><td>4012 </td><td>64.4 </td><td>704 (17.5%) </td><td>831 (20.7%) </td></tr></tbody></table></div><div content-id="DST009TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Strains </th><th>Published year<sup>a</sup> </th><th>Gene density (kb) </th><th>Genome size (bp) </th><th>Gene number </th><th>G + C content (%) </th><th>First class gene (ratio) </th><th>Third class gene (ratio) </th></tr></thead><tbody><tr><td><em>C. violaceum</em> </td><td>18 September 2003 </td><td>0.927 </td><td>4 751 080 </td><td>4405 </td><td>64.8 </td><td>1494 (33.9%) </td><td>1714 (38.9%) </td></tr><tr><td><em>L. hongkongensis</em> </td><td>14 April 2009 </td><td>1.021 </td><td>3 169 329 </td><td>3235 </td><td>62.4 </td><td>969 (30.0%) </td><td>1001 (30.1%) </td></tr><tr><td><em>N. gonorrhoeae</em> FA 1090 </td><td>15 February 2005 </td><td>0.930 </td><td>2 153 922 </td><td>2002 </td><td>52.7 </td><td>266 (13.3%) </td><td>800 (40.0%) </td></tr><tr><td><em>N. gonorrhoeae</em> NCCP11945 </td><td>9 July 2008 </td><td>1.195 </td><td>2 232 025 </td><td>2668 </td><td>52.4 </td><td>244 (9.1%) </td><td>996 (37.3%) </td></tr><tr><td><em>N. lactamica</em> </td><td>16 December 2010 </td><td>0.888 </td><td>2 220 606 </td><td>1972 </td><td>52.3 </td><td>744 (37.7%) </td><td>657 (33.3%) </td></tr><tr><td><em>N. meningitidis</em> 053442 </td><td>3 December 2007 </td><td>0.938 </td><td>2 153 416 </td><td>2020 </td><td>51.7 </td><td>875 (43.3%) </td><td>645 (31.9%) </td></tr><tr><td><em>N. meningitidis</em> Alpha 14 </td><td>24 July 2009 </td><td>0.872 </td><td>2 145 295 </td><td>1872 </td><td>52.0 </td><td>865 (46.2%) </td><td>467 (25.0%) </td></tr><tr><td><em>N. meningitidis</em> MC58 </td><td>10 March 2000 </td><td>0.908 </td><td>2 272 360 </td><td>2063 </td><td>51.5 </td><td>806 (39.1%) </td><td>810 (39.3%) </td></tr><tr><td><em>N. meningitidis</em> Z2491 </td><td>30 March 2000 </td><td>0.874 </td><td>2 184 406 </td><td>1909 </td><td>51.8 </td><td>459 (24.0%) </td><td>669 (35.0%) </td></tr><tr><td><em>Pseudogulbenkiania</em> </td><td>8 September 2011 </td><td>0.926 </td><td>4 332 995 </td><td>4012 </td><td>64.4 </td><td>704 (17.5%) </td><td>831 (20.7%) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Information of published date were extracted from <a class="link link-uri" href="http:// http://www.genomesonline.org/">http://www.genomesonline.org/</a>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5359951" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST009TB1" class="table-wrap-title" data-id="DST009TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Statistical information in genomes of the 10 Neisseriaceae strains</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Strains </th><th>Published year<sup>a</sup> </th><th>Gene density (kb) </th><th>Genome size (bp) </th><th>Gene number </th><th>G + C content (%) </th><th>First class gene (ratio) </th><th>Third class gene (ratio) </th></tr></thead><tbody><tr><td><em>C. violaceum</em> </td><td>18 September 2003 </td><td>0.927 </td><td>4 751 080 </td><td>4405 </td><td>64.8 </td><td>1494 (33.9%) </td><td>1714 (38.9%) </td></tr><tr><td><em>L. hongkongensis</em> </td><td>14 April 2009 </td><td>1.021 </td><td>3 169 329 </td><td>3235 </td><td>62.4 </td><td>969 (30.0%) </td><td>1001 (30.1%) </td></tr><tr><td><em>N. gonorrhoeae</em> FA 1090 </td><td>15 February 2005 </td><td>0.930 </td><td>2 153 922 </td><td>2002 </td><td>52.7 </td><td>266 (13.3%) </td><td>800 (40.0%) </td></tr><tr><td><em>N. gonorrhoeae</em> NCCP11945 </td><td>9 July 2008 </td><td>1.195 </td><td>2 232 025 </td><td>2668 </td><td>52.4 </td><td>244 (9.1%) </td><td>996 (37.3%) </td></tr><tr><td><em>N. lactamica</em> </td><td>16 December 2010 </td><td>0.888 </td><td>2 220 606 </td><td>1972 </td><td>52.3 </td><td>744 (37.7%) </td><td>657 (33.3%) </td></tr><tr><td><em>N. meningitidis</em> 053442 </td><td>3 December 2007 </td><td>0.938 </td><td>2 153 416 </td><td>2020 </td><td>51.7 </td><td>875 (43.3%) </td><td>645 (31.9%) </td></tr><tr><td><em>N. meningitidis</em> Alpha 14 </td><td>24 July 2009 </td><td>0.872 </td><td>2 145 295 </td><td>1872 </td><td>52.0 </td><td>865 (46.2%) </td><td>467 (25.0%) </td></tr><tr><td><em>N. meningitidis</em> MC58 </td><td>10 March 2000 </td><td>0.908 </td><td>2 272 360 </td><td>2063 </td><td>51.5 </td><td>806 (39.1%) </td><td>810 (39.3%) </td></tr><tr><td><em>N. meningitidis</em> Z2491 </td><td>30 March 2000 </td><td>0.874 </td><td>2 184 406 </td><td>1909 </td><td>51.8 </td><td>459 (24.0%) </td><td>669 (35.0%) </td></tr><tr><td><em>Pseudogulbenkiania</em> </td><td>8 September 2011 </td><td>0.926 </td><td>4 332 995 </td><td>4012 </td><td>64.4 </td><td>704 (17.5%) </td><td>831 (20.7%) </td></tr></tbody></table></div><div content-id="DST009TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Strains </th><th>Published year<sup>a</sup> </th><th>Gene density (kb) </th><th>Genome size (bp) </th><th>Gene number </th><th>G + C content (%) </th><th>First class gene (ratio) </th><th>Third class gene (ratio) </th></tr></thead><tbody><tr><td><em>C. violaceum</em> </td><td>18 September 2003 </td><td>0.927 </td><td>4 751 080 </td><td>4405 </td><td>64.8 </td><td>1494 (33.9%) </td><td>1714 (38.9%) </td></tr><tr><td><em>L. hongkongensis</em> </td><td>14 April 2009 </td><td>1.021 </td><td>3 169 329 </td><td>3235 </td><td>62.4 </td><td>969 (30.0%) </td><td>1001 (30.1%) </td></tr><tr><td><em>N. gonorrhoeae</em> FA 1090 </td><td>15 February 2005 </td><td>0.930 </td><td>2 153 922 </td><td>2002 </td><td>52.7 </td><td>266 (13.3%) </td><td>800 (40.0%) </td></tr><tr><td><em>N. gonorrhoeae</em> NCCP11945 </td><td>9 July 2008 </td><td>1.195 </td><td>2 232 025 </td><td>2668 </td><td>52.4 </td><td>244 (9.1%) </td><td>996 (37.3%) </td></tr><tr><td><em>N. lactamica</em> </td><td>16 December 2010 </td><td>0.888 </td><td>2 220 606 </td><td>1972 </td><td>52.3 </td><td>744 (37.7%) </td><td>657 (33.3%) </td></tr><tr><td><em>N. meningitidis</em> 053442 </td><td>3 December 2007 </td><td>0.938 </td><td>2 153 416 </td><td>2020 </td><td>51.7 </td><td>875 (43.3%) </td><td>645 (31.9%) </td></tr><tr><td><em>N. meningitidis</em> Alpha 14 </td><td>24 July 2009 </td><td>0.872 </td><td>2 145 295 </td><td>1872 </td><td>52.0 </td><td>865 (46.2%) </td><td>467 (25.0%) </td></tr><tr><td><em>N. meningitidis</em> MC58 </td><td>10 March 2000 </td><td>0.908 </td><td>2 272 360 </td><td>2063 </td><td>51.5 </td><td>806 (39.1%) </td><td>810 (39.3%) </td></tr><tr><td><em>N. meningitidis</em> Z2491 </td><td>30 March 2000 </td><td>0.874 </td><td>2 184 406 </td><td>1909 </td><td>51.8 </td><td>459 (24.0%) </td><td>669 (35.0%) </td></tr><tr><td><em>Pseudogulbenkiania</em> </td><td>8 September 2011 </td><td>0.926 </td><td>4 332 995 </td><td>4012 </td><td>64.4 </td><td>704 (17.5%) </td><td>831 (20.7%) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Information of published date were extracted from <a class="link link-uri" href="http:// http://www.genomesonline.org/">http://www.genomesonline.org/</a>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5359951" target="_blank">View Large</a></div></div><p>The training set needed the <em>Z</em>-curve method with 33 variables was used to filter over-annotated genes from the RefSeq annotations of 10 Neisseriaceae strains. For each of them, function-known genes with definite names were chosen and shuffled sequences were correspondingly generated. Thus, the training set was obtained. For example, in <em>L. hongkongensis</em>, the training set was comprised of 969 function-known genes that correspond to positive samples, and 969 shuffled sequences corresponding to negative samples. Based on the training set, the discriminant model was built. The accuracy of the model based on 5-fold cross-validation is listed in Table <a reveal-id="DST009TB2" data-open="DST009TB2" class="link link-reveal link-table xref-fig">2</a> for each of the 10 strains. With the model, each hypothetical protein was decided to be a genuine gene or a falsely predicted ORF. Consequently, 86 hypothetical genes were predicted as a non-coding ORF by the <em>Z</em>-curve method in the genome of <em>L. hongkongensis</em>. </p>                        <a id="5359953" scrollto-destination="5359953"></a>
<div content-id="DST009TB2" class="table-modal"><div class="table-wrap"><div id="DST009TB2" class="table-wrap-title" data-id="DST009TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Accuracy based on 5-fold cross-validation and the COG ratio in each strain</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Strain </th><th>Accuracy of the method (%) </th><th>Class 1 with COG (% ratio) </th><th>Predicted non-coding ORFs </th><th>Predicted non-coding with COG (% ratio) </th></tr></thead><tbody><tr><td><em>C. violaceum</em> </td><td>100 </td><td>1457 (97.5) </td><td>88 </td><td>0 (0) </td></tr><tr><td><em>L. hongkongensis</em> </td><td>99.90 </td><td>954 (98.7) </td><td>86 </td><td>1 (1.2) </td></tr><tr><td><em>N. gonorrhoeae</em> FA 1090 </td><td>100 </td><td>263 (98.9) </td><td>24 </td><td>1 (4.2) </td></tr><tr><td><em>N. gonorrhoeae</em> NCCP11945 </td><td>100 </td><td>241 (98.9) </td><td>56 </td><td>0 (0) </td></tr><tr><td><em>N. lactamica</em> </td><td>99.87 </td><td>725 (97.4) </td><td>8 </td><td>0 (0) </td></tr><tr><td><em>N. meningitidis</em> 053442 </td><td>99.77 </td><td>851 (97.3) </td><td>37 </td><td>1 (2.7) </td></tr><tr><td><em>N. meningitidis</em> Alpha 14 </td><td>99.77 </td><td>840 (97.1) </td><td>25 </td><td>1 (4.0) </td></tr><tr><td><em>N. meningitidis</em> MC58 </td><td>99.75 </td><td>790 (98.0) </td><td>48 </td><td>0 (0) </td></tr><tr><td><em>N. meningitidis</em> Z2491 </td><td>100 </td><td>456 (99.3) </td><td>12 </td><td>0 (0) </td></tr><tr><td><em>Pseudogulbenkiania</em> </td><td>99.86 </td><td>683 (97.0) </td><td>34 </td><td>1 (2.9) </td></tr></tbody></table></div><div content-id="DST009TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Strain </th><th>Accuracy of the method (%) </th><th>Class 1 with COG (% ratio) </th><th>Predicted non-coding ORFs </th><th>Predicted non-coding with COG (% ratio) </th></tr></thead><tbody><tr><td><em>C. violaceum</em> </td><td>100 </td><td>1457 (97.5) </td><td>88 </td><td>0 (0) </td></tr><tr><td><em>L. hongkongensis</em> </td><td>99.90 </td><td>954 (98.7) </td><td>86 </td><td>1 (1.2) </td></tr><tr><td><em>N. gonorrhoeae</em> FA 1090 </td><td>100 </td><td>263 (98.9) </td><td>24 </td><td>1 (4.2) </td></tr><tr><td><em>N. gonorrhoeae</em> NCCP11945 </td><td>100 </td><td>241 (98.9) </td><td>56 </td><td>0 (0) </td></tr><tr><td><em>N. lactamica</em> </td><td>99.87 </td><td>725 (97.4) </td><td>8 </td><td>0 (0) </td></tr><tr><td><em>N. meningitidis</em> 053442 </td><td>99.77 </td><td>851 (97.3) </td><td>37 </td><td>1 (2.7) </td></tr><tr><td><em>N. meningitidis</em> Alpha 14 </td><td>99.77 </td><td>840 (97.1) </td><td>25 </td><td>1 (4.0) </td></tr><tr><td><em>N. meningitidis</em> MC58 </td><td>99.75 </td><td>790 (98.0) </td><td>48 </td><td>0 (0) </td></tr><tr><td><em>N. meningitidis</em> Z2491 </td><td>100 </td><td>456 (99.3) </td><td>12 </td><td>0 (0) </td></tr><tr><td><em>Pseudogulbenkiania</em> </td><td>99.86 </td><td>683 (97.0) </td><td>34 </td><td>1 (2.9) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5359953" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST009TB2" class="table-wrap-title" data-id="DST009TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Accuracy based on 5-fold cross-validation and the COG ratio in each strain</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Strain </th><th>Accuracy of the method (%) </th><th>Class 1 with COG (% ratio) </th><th>Predicted non-coding ORFs </th><th>Predicted non-coding with COG (% ratio) </th></tr></thead><tbody><tr><td><em>C. violaceum</em> </td><td>100 </td><td>1457 (97.5) </td><td>88 </td><td>0 (0) </td></tr><tr><td><em>L. hongkongensis</em> </td><td>99.90 </td><td>954 (98.7) </td><td>86 </td><td>1 (1.2) </td></tr><tr><td><em>N. gonorrhoeae</em> FA 1090 </td><td>100 </td><td>263 (98.9) </td><td>24 </td><td>1 (4.2) </td></tr><tr><td><em>N. gonorrhoeae</em> NCCP11945 </td><td>100 </td><td>241 (98.9) </td><td>56 </td><td>0 (0) </td></tr><tr><td><em>N. lactamica</em> </td><td>99.87 </td><td>725 (97.4) </td><td>8 </td><td>0 (0) </td></tr><tr><td><em>N. meningitidis</em> 053442 </td><td>99.77 </td><td>851 (97.3) </td><td>37 </td><td>1 (2.7) </td></tr><tr><td><em>N. meningitidis</em> Alpha 14 </td><td>99.77 </td><td>840 (97.1) </td><td>25 </td><td>1 (4.0) </td></tr><tr><td><em>N. meningitidis</em> MC58 </td><td>99.75 </td><td>790 (98.0) </td><td>48 </td><td>0 (0) </td></tr><tr><td><em>N. meningitidis</em> Z2491 </td><td>100 </td><td>456 (99.3) </td><td>12 </td><td>0 (0) </td></tr><tr><td><em>Pseudogulbenkiania</em> </td><td>99.86 </td><td>683 (97.0) </td><td>34 </td><td>1 (2.9) </td></tr></tbody></table></div><div content-id="DST009TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Strain </th><th>Accuracy of the method (%) </th><th>Class 1 with COG (% ratio) </th><th>Predicted non-coding ORFs </th><th>Predicted non-coding with COG (% ratio) </th></tr></thead><tbody><tr><td><em>C. violaceum</em> </td><td>100 </td><td>1457 (97.5) </td><td>88 </td><td>0 (0) </td></tr><tr><td><em>L. hongkongensis</em> </td><td>99.90 </td><td>954 (98.7) </td><td>86 </td><td>1 (1.2) </td></tr><tr><td><em>N. gonorrhoeae</em> FA 1090 </td><td>100 </td><td>263 (98.9) </td><td>24 </td><td>1 (4.2) </td></tr><tr><td><em>N. gonorrhoeae</em> NCCP11945 </td><td>100 </td><td>241 (98.9) </td><td>56 </td><td>0 (0) </td></tr><tr><td><em>N. lactamica</em> </td><td>99.87 </td><td>725 (97.4) </td><td>8 </td><td>0 (0) </td></tr><tr><td><em>N. meningitidis</em> 053442 </td><td>99.77 </td><td>851 (97.3) </td><td>37 </td><td>1 (2.7) </td></tr><tr><td><em>N. meningitidis</em> Alpha 14 </td><td>99.77 </td><td>840 (97.1) </td><td>25 </td><td>1 (4.0) </td></tr><tr><td><em>N. meningitidis</em> MC58 </td><td>99.75 </td><td>790 (98.0) </td><td>48 </td><td>0 (0) </td></tr><tr><td><em>N. meningitidis</em> Z2491 </td><td>100 </td><td>456 (99.3) </td><td>12 </td><td>0 (0) </td></tr><tr><td><em>Pseudogulbenkiania</em> </td><td>99.86 </td><td>683 (97.0) </td><td>34 </td><td>1 (2.9) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5359953" target="_blank">View Large</a></div></div><p>Prediction of 86 hypothetical genes as non-coding is based on the assumption that all protein-coding genes should have similar nucleotide composition in one specific bacterial genome.<sup><a reveal-id="DST009C28" data-open="DST009C28" class="link link-ref link-reveal xref-bibr">28</a></sup> That is to say, hypothetical genes should have similar composition features with function-known genes in <em>L. hongkongensis</em>. If not, they should have been over-annotated as genes. The scatter plot of the nucleotide distribution of 969 function-known genes and 86 predicted non-coding ORFs is shown in Figure <a reveal-id="DST009F1" data-open="DST009F1" class="link link-reveal link-table xref-fig">1</a>A and B. As can be seen, non-coding ORFs are distributed far away from function-known genes. In detail, almost all function-known genes lie far above the diagonal and G + C contents of them at the second codon positions are much lower than that at the third codon positions, whereas almost all 86 non-coding ORFs locate around the diagonal, indicating that their G + C contents at the second positions are approximate to that at the third codon positions. The need to encode functional proteins exerts severe constrain on the nucleotide composition of genes.<sup><a reveal-id="DST009C35" data-open="DST009C35" class="link link-ref link-reveal xref-bibr">35</a></sup> Previous work showed a similar nucleotide distribution of functional genes in seven high G + C prokaryotic genomes.<sup><a reveal-id="DST009C36" data-open="DST009C36" class="link link-ref link-reveal xref-bibr">36</a>,<a reveal-id="DST009C37" data-open="DST009C37" class="link link-ref link-reveal xref-bibr">37</a></sup> Therefore, the distinct nucleotide composition between 86 hypothetical genes and the function-known genes draw them away from being genuine genes. Nucleotide compositions for the 915 retained hypothetical genes are shown in Figure <a reveal-id="DST009F1" data-open="DST009F1" class="link link-reveal link-table xref-fig">1</a>C. Different from 86 predicted non-coding ORFs, most of the retained hypothetical genes have a similar distribution of GC<sub>2</sub> versus GC<sub>3</sub> with function-known genes.</p><p>The COG database has been widely used during the annotation process of sequenced bacterial genomes.<sup><a reveal-id="DST009C5" data-open="DST009C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DST009C38" data-open="DST009C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Belonging to a COG is believed to be a very reliable evidence of protein-coding genes.<sup><a reveal-id="DST009C38" data-open="DST009C38" class="link link-ref link-reveal xref-bibr">38</a>,<a reveal-id="DST009C39" data-open="DST009C39" class="link link-ref link-reveal xref-bibr">39</a></sup> In <em>L. hongkongensis</em>, 954 among 967 function-known genes have been assigned a COG code. However, only 1 of 86 predicted non-coding ORFs has the COG code. Based on the above analyses, these 86 ORFs are very unlikely to encode proteins. COG statistics information for the other nine strains is shown in Table <a reveal-id="DST009TB2" data-open="DST009TB2" class="link link-reveal link-table xref-fig">2</a>. As can been seen, the COG ratio of predicted non-coding ORFs is extremely lower than that of genes with known functions and definite names. Summarily, 7260 among 7426 (97.8%) genes belonging to the first class are assigned with COG codes in the 10 Neisseriaceae genomes. In comparison, only 5 of 418 (1.2%) predicted non-coding ORFs could be assigned to the COG database, indicating that our prediction is much accurate in another sense. These five ORFs with COG codes are likely to constitute falsely predictions of our method because having COG counterparts has been believed to be one of the reliable evidences of encoding proteins. Finally, we only eliminated the remaining 413 hypothetical proteins from the RefSeq annotations in the 10 complete genomes. Details of them are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>.</p><p>As is well known, horizontally transferred genes may also have a different nucleotide composition with core genes to some extent. The DarkHorse database stores horizontally transferred genes in sequenced bacterial genomes. Entries in it are all those predicted by comparative genomes methods and hence are very reliable. However, the information of HGT is not available for <em>L. hongkongensis</em> in the DarkHorse. To circumvent this problem, we fall back on the bacterial strain <em>Pseudomonas aeruginosa</em> PA01, which also has a high G + C content, and the distribution of GC<sub>2</sub> versus GC<sub>3</sub> for function-known genes has shown to be similar to that of <em>L. hongkongensis</em>.<sup><a reveal-id="DST009C36" data-open="DST009C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Furthermore, as an early sequenced genome, the gene annotation of it is very reliable and so could be used a good reference. For the genome, 22 horizontally transferred genes were extracted from the DarkHorse. As can be seen from Figure <a reveal-id="DST009F1" data-open="DST009F1" class="link link-reveal link-table xref-fig">1</a>D, the nucleotide distribution of horizontally transferred genes tends to be similar to that in Figure <a reveal-id="DST009F1" data-open="DST009F1" class="link link-reveal link-table xref-fig">1</a>A. Although the DarkHorse genes have basically similar nucleotide compositions with function-known genes, it does not mean the 86 ORFs in Figure <a reveal-id="DST009F1" data-open="DST009F1" class="link link-reveal link-table xref-fig">1</a>B are all definitely non-coding. In fact, there may still be the possibility that some of the genes, particularly those with GC<sub>2</sub> between 0.3 and 0.6, are falsely predicted as non-coding because of their very recent transfers. To investigate the possibility of clusters of horizontally transferred genes, we checked the chromosomal locations of these 86 ORFs and found that they not have any cluster pattern. Therefore, it is sure that, at least, most of the 85 eliminated ORFs do not belong to horizontally transferred genes and are indeed non-coding. Also note that 11 of the 20 horizontally transferred genes have been assigned the COG code and this ratio is much higher than that of the collective of the 85 eliminated ORFs.</p>                            <h3 scrollto-destination=5359957 id="5359957" class="section-title" data-legacy-id=s3b>3.2. Missed genes found by joint methods and their functions</h3>
<p>During the process of annotating sequenced genomes, some genuine genes may be missed because the annotators pursue a balance between the number of all annotated genes and that of finding genuine genes and ensure not too high falsely positive predictions.<sup><a reveal-id="DST009C14" data-open="DST009C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DST009C25" data-open="DST009C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DST009C40" data-open="DST009C40" class="link link-ref link-reveal xref-bibr">40</a></sup> The method for picking up missed genes was a two-step process by combining the <em>ab initio</em> gene finding and the blast search. With this method, we found a varying number of missed genes with potential functions (Table <a reveal-id="DST009TB3" data-open="DST009TB3" class="link link-reveal link-table xref-fig">3</a>). For example, eight new genes were added in <em>L. hongkongensis</em>, and details for them are listed in Table <a reveal-id="DST009TB4" data-open="DST009TB4" class="link link-reveal link-table xref-fig">4</a>. RT–PCR analyses validated the transcriptions of all eight sequences (Fig. <a reveal-id="DST009F2" data-open="DST009F2" class="link link-reveal link-table xref-fig">2</a>). Out of the 10 newly found genes in the genome of <em>C. violaceum</em>, transcriptions of eight ones are validated with RT–PCR and the exceptions are (Table <a reveal-id="DST009TB6" data-open="DST009TB6" class="link link-reveal link-table xref-fig">6</a>) CV_A0007 and CV_A0010 (Fig. <a reveal-id="DST009F3" data-open="DST009F3" class="link link-reveal link-table xref-fig">3</a>). All the amplification with genome DNA was positive (data not shown). Therefore, transcriptional analyses illustrated that the method was effective for picking up new genes and had very high accuracy. However, this method is only applicable to picking up genes having homologues in other genomes but not to strain-specific genes. In the other genomes, the method was directly used and without experimental validation. </p>                        <a id="5359959" scrollto-destination="5359959"></a>
<div content-id="DST009TB3" class="table-modal"><div class="table-wrap"><div id="DST009TB3" class="table-wrap-title" data-id="DST009TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Numbers of revised genes, which contains newly found genes, hypothetical genes with newly assigned functions, eliminated ORFs and disrupted ORFs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Strains </th><th>Newly found genes </th><th>Newly assigned functions </th><th>Eliminated ORFs </th><th>Disrupted ORFs </th></tr></thead><tbody><tr><td><em>C. violaceum</em> </td><td>10 </td><td>120 </td><td>88 </td><td>0 </td></tr><tr><td><em>L. hongkongensis</em> </td><td>8 </td><td>20 </td><td>85 </td><td>2 </td></tr><tr><td><em>N. gonorrhoeae</em> FA 1090 </td><td>18 </td><td>400 </td><td>23 </td><td>23 </td></tr><tr><td><em>N. gonorrhoeae</em> NCCP11945 </td><td>9 </td><td>207 </td><td>56 </td><td>0 </td></tr><tr><td><em>N. lactamica</em> </td><td>5 </td><td>218 </td><td>8 </td><td>8 </td></tr><tr><td><em>N. meningitidis</em> 053442 </td><td>6 </td><td>214 </td><td>36 </td><td>19 </td></tr><tr><td><em>N. meningitidis</em> Alpha 14 </td><td>30 </td><td>214 </td><td>24 </td><td>14 </td></tr><tr><td><em>N. meningitidis</em> MC58 </td><td>11 </td><td>331 </td><td>48 </td><td>27 </td></tr><tr><td><em>N. meningitidis</em> Z2491 </td><td>8 </td><td>299 </td><td>12 </td><td>20 </td></tr><tr><td><em>Pseudogulbenkiania</em> </td><td>1 </td><td>46 </td><td>33 </td><td>0 </td></tr></tbody></table></div><div content-id="DST009TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Strains </th><th>Newly found genes </th><th>Newly assigned functions </th><th>Eliminated ORFs </th><th>Disrupted ORFs </th></tr></thead><tbody><tr><td><em>C. violaceum</em> </td><td>10 </td><td>120 </td><td>88 </td><td>0 </td></tr><tr><td><em>L. hongkongensis</em> </td><td>8 </td><td>20 </td><td>85 </td><td>2 </td></tr><tr><td><em>N. gonorrhoeae</em> FA 1090 </td><td>18 </td><td>400 </td><td>23 </td><td>23 </td></tr><tr><td><em>N. gonorrhoeae</em> NCCP11945 </td><td>9 </td><td>207 </td><td>56 </td><td>0 </td></tr><tr><td><em>N. lactamica</em> </td><td>5 </td><td>218 </td><td>8 </td><td>8 </td></tr><tr><td><em>N. meningitidis</em> 053442 </td><td>6 </td><td>214 </td><td>36 </td><td>19 </td></tr><tr><td><em>N. meningitidis</em> Alpha 14 </td><td>30 </td><td>214 </td><td>24 </td><td>14 </td></tr><tr><td><em>N. meningitidis</em> MC58 </td><td>11 </td><td>331 </td><td>48 </td><td>27 </td></tr><tr><td><em>N. meningitidis</em> Z2491 </td><td>8 </td><td>299 </td><td>12 </td><td>20 </td></tr><tr><td><em>Pseudogulbenkiania</em> </td><td>1 </td><td>46 </td><td>33 </td><td>0 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5359959" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST009TB3" class="table-wrap-title" data-id="DST009TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Numbers of revised genes, which contains newly found genes, hypothetical genes with newly assigned functions, eliminated ORFs and disrupted ORFs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Strains </th><th>Newly found genes </th><th>Newly assigned functions </th><th>Eliminated ORFs </th><th>Disrupted ORFs </th></tr></thead><tbody><tr><td><em>C. violaceum</em> </td><td>10 </td><td>120 </td><td>88 </td><td>0 </td></tr><tr><td><em>L. hongkongensis</em> </td><td>8 </td><td>20 </td><td>85 </td><td>2 </td></tr><tr><td><em>N. gonorrhoeae</em> FA 1090 </td><td>18 </td><td>400 </td><td>23 </td><td>23 </td></tr><tr><td><em>N. gonorrhoeae</em> NCCP11945 </td><td>9 </td><td>207 </td><td>56 </td><td>0 </td></tr><tr><td><em>N. lactamica</em> </td><td>5 </td><td>218 </td><td>8 </td><td>8 </td></tr><tr><td><em>N. meningitidis</em> 053442 </td><td>6 </td><td>214 </td><td>36 </td><td>19 </td></tr><tr><td><em>N. meningitidis</em> Alpha 14 </td><td>30 </td><td>214 </td><td>24 </td><td>14 </td></tr><tr><td><em>N. meningitidis</em> MC58 </td><td>11 </td><td>331 </td><td>48 </td><td>27 </td></tr><tr><td><em>N. meningitidis</em> Z2491 </td><td>8 </td><td>299 </td><td>12 </td><td>20 </td></tr><tr><td><em>Pseudogulbenkiania</em> </td><td>1 </td><td>46 </td><td>33 </td><td>0 </td></tr></tbody></table></div><div content-id="DST009TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Strains </th><th>Newly found genes </th><th>Newly assigned functions </th><th>Eliminated ORFs </th><th>Disrupted ORFs </th></tr></thead><tbody><tr><td><em>C. violaceum</em> </td><td>10 </td><td>120 </td><td>88 </td><td>0 </td></tr><tr><td><em>L. hongkongensis</em> </td><td>8 </td><td>20 </td><td>85 </td><td>2 </td></tr><tr><td><em>N. gonorrhoeae</em> FA 1090 </td><td>18 </td><td>400 </td><td>23 </td><td>23 </td></tr><tr><td><em>N. gonorrhoeae</em> NCCP11945 </td><td>9 </td><td>207 </td><td>56 </td><td>0 </td></tr><tr><td><em>N. lactamica</em> </td><td>5 </td><td>218 </td><td>8 </td><td>8 </td></tr><tr><td><em>N. meningitidis</em> 053442 </td><td>6 </td><td>214 </td><td>36 </td><td>19 </td></tr><tr><td><em>N. meningitidis</em> Alpha 14 </td><td>30 </td><td>214 </td><td>24 </td><td>14 </td></tr><tr><td><em>N. meningitidis</em> MC58 </td><td>11 </td><td>331 </td><td>48 </td><td>27 </td></tr><tr><td><em>N. meningitidis</em> Z2491 </td><td>8 </td><td>299 </td><td>12 </td><td>20 </td></tr><tr><td><em>Pseudogulbenkiania</em> </td><td>1 </td><td>46 </td><td>33 </td><td>0 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5359959" target="_blank">View Large</a></div></div>                        <a id="5359960" scrollto-destination="5359960"></a>
<div content-id="DST009TB4" class="table-modal"><div class="table-wrap"><div id="DST009TB4" class="table-wrap-title" data-id="DST009TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Details of the eight newly found genes in the genome of <em>L. hongkongensis</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>ID </th><th>Position </th><th>COG </th><th>Coverage, <em>E</em>-value, Identity </th><th>Potential function </th></tr></thead><tbody><tr><td>LHK_A0001 </td><td>476554–477024 (+) </td><td>COG0560E </td><td>92%, 1e−49, 55% </td><td>Phosphoserine phosphatase </td></tr><tr><td>LHK_A0002 </td><td>880771–881358 (+) </td><td>COG1961L </td><td>97%, 2e−72, 60% </td><td>Site-specific recombinases </td></tr><tr><td>LHK_A0003 </td><td>1391850–1392488 (−) </td><td>COG2869C </td><td>98%, 6e−66, 51% </td><td>Na+-transporting nicotinamide adenine dinucleotide </td></tr><tr><td>LHK_A0004 </td><td>1570970–1571566 (+) </td><td>COG2864C </td><td>100%, 1e−110, 81% </td><td>Thiosulphate reductase cytochrome subunit B </td></tr><tr><td>LHK_A0005 </td><td>1848723–1850306 (−) </td><td>COG0733R </td><td>63%, 3e−156, 73% </td><td>Na+-dependent transporters of the sodium: neurotransmitter symporter family </td></tr><tr><td>LHK_A0006 </td><td>2282651–2283334 (+) </td><td>COG0778C </td><td>77%, 3e−73, 66% </td><td>Putative Cob(II)yrinic acid a,c-diamide reductase (BluB) </td></tr><tr><td>LHK_A0007 </td><td>2660234–2660422 (−) </td><td>COG0103J </td><td>96%, 2e−28, 88% </td><td>Ribosomal protein S9 </td></tr><tr><td>LHK_A0008 </td><td>2875641–2876057 (−) </td><td>COG0824R </td><td>91%, 7e−55, 64% </td><td>Predicted thioesterase </td></tr></tbody></table></div><div content-id="DST009TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>ID </th><th>Position </th><th>COG </th><th>Coverage, <em>E</em>-value, Identity </th><th>Potential function </th></tr></thead><tbody><tr><td>LHK_A0001 </td><td>476554–477024 (+) </td><td>COG0560E </td><td>92%, 1e−49, 55% </td><td>Phosphoserine phosphatase </td></tr><tr><td>LHK_A0002 </td><td>880771–881358 (+) </td><td>COG1961L </td><td>97%, 2e−72, 60% </td><td>Site-specific recombinases </td></tr><tr><td>LHK_A0003 </td><td>1391850–1392488 (−) </td><td>COG2869C </td><td>98%, 6e−66, 51% </td><td>Na+-transporting nicotinamide adenine dinucleotide </td></tr><tr><td>LHK_A0004 </td><td>1570970–1571566 (+) </td><td>COG2864C </td><td>100%, 1e−110, 81% </td><td>Thiosulphate reductase cytochrome subunit B </td></tr><tr><td>LHK_A0005 </td><td>1848723–1850306 (−) </td><td>COG0733R </td><td>63%, 3e−156, 73% </td><td>Na+-dependent transporters of the sodium: neurotransmitter symporter family </td></tr><tr><td>LHK_A0006 </td><td>2282651–2283334 (+) </td><td>COG0778C </td><td>77%, 3e−73, 66% </td><td>Putative Cob(II)yrinic acid a,c-diamide reductase (BluB) </td></tr><tr><td>LHK_A0007 </td><td>2660234–2660422 (−) </td><td>COG0103J </td><td>96%, 2e−28, 88% </td><td>Ribosomal protein S9 </td></tr><tr><td>LHK_A0008 </td><td>2875641–2876057 (−) </td><td>COG0824R </td><td>91%, 7e−55, 64% </td><td>Predicted thioesterase </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5359960" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST009TB4" class="table-wrap-title" data-id="DST009TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Details of the eight newly found genes in the genome of <em>L. hongkongensis</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>ID </th><th>Position </th><th>COG </th><th>Coverage, <em>E</em>-value, Identity </th><th>Potential function </th></tr></thead><tbody><tr><td>LHK_A0001 </td><td>476554–477024 (+) </td><td>COG0560E </td><td>92%, 1e−49, 55% </td><td>Phosphoserine phosphatase </td></tr><tr><td>LHK_A0002 </td><td>880771–881358 (+) </td><td>COG1961L </td><td>97%, 2e−72, 60% </td><td>Site-specific recombinases </td></tr><tr><td>LHK_A0003 </td><td>1391850–1392488 (−) </td><td>COG2869C </td><td>98%, 6e−66, 51% </td><td>Na+-transporting nicotinamide adenine dinucleotide </td></tr><tr><td>LHK_A0004 </td><td>1570970–1571566 (+) </td><td>COG2864C </td><td>100%, 1e−110, 81% </td><td>Thiosulphate reductase cytochrome subunit B </td></tr><tr><td>LHK_A0005 </td><td>1848723–1850306 (−) </td><td>COG0733R </td><td>63%, 3e−156, 73% </td><td>Na+-dependent transporters of the sodium: neurotransmitter symporter family </td></tr><tr><td>LHK_A0006 </td><td>2282651–2283334 (+) </td><td>COG0778C </td><td>77%, 3e−73, 66% </td><td>Putative Cob(II)yrinic acid a,c-diamide reductase (BluB) </td></tr><tr><td>LHK_A0007 </td><td>2660234–2660422 (−) </td><td>COG0103J </td><td>96%, 2e−28, 88% </td><td>Ribosomal protein S9 </td></tr><tr><td>LHK_A0008 </td><td>2875641–2876057 (−) </td><td>COG0824R </td><td>91%, 7e−55, 64% </td><td>Predicted thioesterase </td></tr></tbody></table></div><div content-id="DST009TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>ID </th><th>Position </th><th>COG </th><th>Coverage, <em>E</em>-value, Identity </th><th>Potential function </th></tr></thead><tbody><tr><td>LHK_A0001 </td><td>476554–477024 (+) </td><td>COG0560E </td><td>92%, 1e−49, 55% </td><td>Phosphoserine phosphatase </td></tr><tr><td>LHK_A0002 </td><td>880771–881358 (+) </td><td>COG1961L </td><td>97%, 2e−72, 60% </td><td>Site-specific recombinases </td></tr><tr><td>LHK_A0003 </td><td>1391850–1392488 (−) </td><td>COG2869C </td><td>98%, 6e−66, 51% </td><td>Na+-transporting nicotinamide adenine dinucleotide </td></tr><tr><td>LHK_A0004 </td><td>1570970–1571566 (+) </td><td>COG2864C </td><td>100%, 1e−110, 81% </td><td>Thiosulphate reductase cytochrome subunit B </td></tr><tr><td>LHK_A0005 </td><td>1848723–1850306 (−) </td><td>COG0733R </td><td>63%, 3e−156, 73% </td><td>Na+-dependent transporters of the sodium: neurotransmitter symporter family </td></tr><tr><td>LHK_A0006 </td><td>2282651–2283334 (+) </td><td>COG0778C </td><td>77%, 3e−73, 66% </td><td>Putative Cob(II)yrinic acid a,c-diamide reductase (BluB) </td></tr><tr><td>LHK_A0007 </td><td>2660234–2660422 (−) </td><td>COG0103J </td><td>96%, 2e−28, 88% </td><td>Ribosomal protein S9 </td></tr><tr><td>LHK_A0008 </td><td>2875641–2876057 (−) </td><td>COG0824R </td><td>91%, 7e−55, 64% </td><td>Predicted thioesterase </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5359960" target="_blank">View Large</a></div></div>                        <a id="5359961" scrollto-destination="5359961"></a>
<div id="DST009F2" data-id="DST009F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst009/2/m_dst00902.jpeg?Expires=2147483647&Signature=EtvkQkFpFgM2uKf6LvAXe75fl~nG~YJtbpmuZwK098CNjzTK9bDvrnK6sh30NcoJ5SxMgAq1vZbaEwSMJo4xixlyEIC1yMLmkCn3v5OOaNEtj-Q8M2sSL93sQ~w4D~xpsAqqTdtdOjS-5~eym6toG0zlQFo9KNqb7kxyeZSfx9xlxaBV7BUtpc1XlMKMzNsDmx-uS84d1Foox3VifegUFaDGF8hBfdNf8t~OvP5CRwBXvuJdZE2ubpLI~r-AMJbY-XCzdJ1YNW5XfABxkB0g1i0N08q~4nfJuRIsN08t0HiMD4y-8t3zXt1i0J7psr4hUsbuPInl3zKc9CwM-cN-Ng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="RT–PCR confirmations of eight newly found genes in L. hongkongensis. mRNAs corresponding to candidate genes were evaluated by RT–PCR (RT). We used no transcriptase-containing sample as negative control (NRT) and PCR with genomic DNA as a positive control (PC)." path-from-xml="dst00902.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5359961/dst00902.jpeg" path-from-xml="dst00902.jpeg" target="_blank">View large</a><a section="5359961" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst009/2/dst00902.jpeg?Expires=2147483647&Signature=MSo-DqST29O2AurQtDweTN-fJ0FmAbxrwQZOMRo7bJvSOehBOeeGewwQ2XiEHxU6t-dT33ppX8VLokKkBUQXH7c1jjuETmLYwPCtQKbUXehGE9bjyTc6YVjSjbf1Za49KkM-KXN8LrIcxtMXSuz4r186MiY5rAWslLmZKUZFOWGUvVfcY9YNg0gcVEP-vXiMFcOqnnb5fp-hiIaUhOD77rxEwRHgVswXb8N~DlF02CfJuF-uEYCXmw4rrUAG0Uztq3awQ3el1um7zi7tdqDphKDsxtBZx-4jW7gqeyAvwQ1svdkCKJMGNqiZZMwMq52aiozLt8HELN1XRKZXD8hjqA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5359961&ar=369179&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00902.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>RT–PCR confirmations of eight newly found genes in <em>L. hongkongensis</em>. mRNAs corresponding to candidate genes were evaluated by RT–PCR (RT). We used no transcriptase-containing sample as negative control (NRT) and PCR with genomic DNA as a positive control (PC).</p></div></div><div content-id="DST009F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst009/2/m_dst00902.jpeg?Expires=2147483647&Signature=EtvkQkFpFgM2uKf6LvAXe75fl~nG~YJtbpmuZwK098CNjzTK9bDvrnK6sh30NcoJ5SxMgAq1vZbaEwSMJo4xixlyEIC1yMLmkCn3v5OOaNEtj-Q8M2sSL93sQ~w4D~xpsAqqTdtdOjS-5~eym6toG0zlQFo9KNqb7kxyeZSfx9xlxaBV7BUtpc1XlMKMzNsDmx-uS84d1Foox3VifegUFaDGF8hBfdNf8t~OvP5CRwBXvuJdZE2ubpLI~r-AMJbY-XCzdJ1YNW5XfABxkB0g1i0N08q~4nfJuRIsN08t0HiMD4y-8t3zXt1i0J7psr4hUsbuPInl3zKc9CwM-cN-Ng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="RT–PCR confirmations of eight newly found genes in L. hongkongensis. mRNAs corresponding to candidate genes were evaluated by RT–PCR (RT). We used no transcriptase-containing sample as negative control (NRT) and PCR with genomic DNA as a positive control (PC)." path-from-xml="dst00902.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5359961/dst00902.jpeg" path-from-xml="dst00902.jpeg" target="_blank">View large</a><a section="5359961" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst009/2/dst00902.jpeg?Expires=2147483647&Signature=MSo-DqST29O2AurQtDweTN-fJ0FmAbxrwQZOMRo7bJvSOehBOeeGewwQ2XiEHxU6t-dT33ppX8VLokKkBUQXH7c1jjuETmLYwPCtQKbUXehGE9bjyTc6YVjSjbf1Za49KkM-KXN8LrIcxtMXSuz4r186MiY5rAWslLmZKUZFOWGUvVfcY9YNg0gcVEP-vXiMFcOqnnb5fp-hiIaUhOD77rxEwRHgVswXb8N~DlF02CfJuF-uEYCXmw4rrUAG0Uztq3awQ3el1um7zi7tdqDphKDsxtBZx-4jW7gqeyAvwQ1svdkCKJMGNqiZZMwMq52aiozLt8HELN1XRKZXD8hjqA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5359961&ar=369179&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00902.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>RT–PCR confirmations of eight newly found genes in <em>L. hongkongensis</em>. mRNAs corresponding to candidate genes were evaluated by RT–PCR (RT). We used no transcriptase-containing sample as negative control (NRT) and PCR with genomic DNA as a positive control (PC).</p></div></div>                        <a id="5359962" scrollto-destination="5359962"></a>
<div id="DST009F3" data-id="DST009F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst009/2/m_dst00903.jpeg?Expires=2147483647&Signature=4FnUrtVobVRCm5~FdTfxN7jZz4w638K2B~ewGid5TjLPUoxxsogQ1zmQrwb2Cwqm6msNLvMfCI2Z3-Lv0gPIfG2P5PNqIc1n6hjqLcL1JLBdzIWn5PHOaAVIPEXbywAjHXH3N6RCA6qNBom43IYcbSbCQi4YsyOanfC2eem1yzgT-XhdoJo10~Ojp5-WgN~javzgbuTF1fzltYiLrI7X3h-jxvQNZAOeOP~xSCov5mmHs~5GMTTlp9iQnO4bV2jjXoSWRp~DJEefHfMGDbkVGHPLqVBpEZZQ4Y6U4iR5hP3G61KxgxRs6Wl4G8KbPU~kckIQ0XtpN1XjbRHK~ulySQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="RT–PCR confirmations of newly found potential genes in C. violaceum." path-from-xml="dst00903.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5359962/dst00903.jpeg" path-from-xml="dst00903.jpeg" target="_blank">View large</a><a section="5359962" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst009/2/dst00903.jpeg?Expires=2147483647&Signature=LtZmhqc2uYID6dXO~0JMfES5ZVvbFlCP9TpPU8V5KbO-3HIlEwA--RJI9pnixWD1lV2Ul5T1lb2dhLRO5Th4h8ldMENFAj3o8suDMqStpPHpMYYPkXzTf2-gJmxSe~-9mrcH2yigVbPXDtDk4FHRO4jsA-i5zL~3-dSJ29R-aR~Ez2T8OeY~gBwM9v2RKlW-FNopBBBWZA7ae6a2SztriEY~5i7q~A1f8vCZOWfsWu5fLnOHl6R2JnP0vbHYJxkkxYRYhUKV8UDpLPe3U8GofU08J0PuhxbiSQVhQJ2-8rFrwKvt4AYOO0Ym0OwcQ594W4BxES7Fcm~kUT~1ufPIJg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5359962&ar=369179&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00903.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>RT–PCR confirmations of newly found potential genes in <em>C. violaceum</em>.</p></div></div><div content-id="DST009F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst009/2/m_dst00903.jpeg?Expires=2147483647&Signature=4FnUrtVobVRCm5~FdTfxN7jZz4w638K2B~ewGid5TjLPUoxxsogQ1zmQrwb2Cwqm6msNLvMfCI2Z3-Lv0gPIfG2P5PNqIc1n6hjqLcL1JLBdzIWn5PHOaAVIPEXbywAjHXH3N6RCA6qNBom43IYcbSbCQi4YsyOanfC2eem1yzgT-XhdoJo10~Ojp5-WgN~javzgbuTF1fzltYiLrI7X3h-jxvQNZAOeOP~xSCov5mmHs~5GMTTlp9iQnO4bV2jjXoSWRp~DJEefHfMGDbkVGHPLqVBpEZZQ4Y6U4iR5hP3G61KxgxRs6Wl4G8KbPU~kckIQ0XtpN1XjbRHK~ulySQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="RT–PCR confirmations of newly found potential genes in C. violaceum." path-from-xml="dst00903.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5359962/dst00903.jpeg" path-from-xml="dst00903.jpeg" target="_blank">View large</a><a section="5359962" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst009/2/dst00903.jpeg?Expires=2147483647&Signature=LtZmhqc2uYID6dXO~0JMfES5ZVvbFlCP9TpPU8V5KbO-3HIlEwA--RJI9pnixWD1lV2Ul5T1lb2dhLRO5Th4h8ldMENFAj3o8suDMqStpPHpMYYPkXzTf2-gJmxSe~-9mrcH2yigVbPXDtDk4FHRO4jsA-i5zL~3-dSJ29R-aR~Ez2T8OeY~gBwM9v2RKlW-FNopBBBWZA7ae6a2SztriEY~5i7q~A1f8vCZOWfsWu5fLnOHl6R2JnP0vbHYJxkkxYRYhUKV8UDpLPe3U8GofU08J0PuhxbiSQVhQJ2-8rFrwKvt4AYOO0Ym0OwcQ594W4BxES7Fcm~kUT~1ufPIJg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5359962&ar=369179&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00903.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>RT–PCR confirmations of newly found potential genes in <em>C. violaceum</em>.</p></div></div><p>Among the eight new genes in <em>L. hongkongensis</em>, only two do not have any overlapping bases with annotated genes and they are LHK_A0003 and LHK_A0004. The details of the eight genes and their corresponding functions and the details of PCR primers and conditions used in the validation experiments are listed in Tables <a reveal-id="DST009TB4" data-open="DST009TB4" class="link link-reveal link-table xref-fig">4</a> and <a reveal-id="DST009TB5" data-open="DST009TB5" class="link link-reveal link-table xref-fig">5</a>, respectively. For each of the remaining six overlapping genes, the region spanned by the two PCR primers does not overlap with the annotated genes and thus could reduce false-positive errors for the transcription trace. The six genes are analyzed as follows. LHK_A0001 overlaps the annotated gene LHK_00511. According to the annotation, the two sequences have the same potential function of encoding the phosphoserine phosphatase. LHK_A0001 has the similarity score of the <em>E</em>-value of 10<sup>−49</sup> and Identity of 55% and the similarity score of LHK_00511 is the <em>E</em>-value of 10<sup>−39</sup> and Identity of 65%. We could not decide which of them is the genuine gene based on the scores. LHK_00511 has the length of only 211 bp, which is much shorter than the length (∼669 bp) of known genes with the same function in other genomes. LHK_A0001 has the length of 471 bp. Based on the length information, LHK_A0001 is more likely to encode the phosphoserine phosphatase than LHK_00511. LHK_A0002 overlaps the annotated gene LHK_00916 with only 29 bp. LHK_A0002 is predicted to code for site-specific recombinase and LHK_00916 encodes replicase. Because they have so little overlap and have different functions, it is very likely that they are both genuine genes. LHK_A0005 has the potential function of encoding the Na<sup>+</sup>-dependent transporter. LHK_A0006 overlaps 61 bp with one functional gene but they locate on two different DNA strands. LHK_A0007 and the annotated <em>rpsI</em> gene (LHK_02777) constitute an interesting overlap. After the blast alignment, both of them are found to be significantly similar to the <em>rpsI</em> gene in the other genomes. As shown in Figure <a reveal-id="DST009F4" data-open="DST009F4" class="link link-reveal link-table xref-fig">4</a>, LHK_A0007 matches the last 141 bp (57 aa) of the other <em>rpsI</em> genes and LHK_02777 matches the first 219 bp (73 aa). LHK_A0007 and LHK_02777 as a whole just constitute a complete <em>rpsI</em> gene (130 aa). In fact, LHK_A0007 and LHK_02777 are adjacent and overlapping. For the overlapping part, LHK_A0007 has the correct reading frame but the LHK_02777 does not have the correct frame, according to the other <em>rpsI</em> genes. Therefore, it is suggested that there has appeared an event of nucleotide insertion/deletion for the <em>rpsI</em> gene in <em>L. hongkongensis</em>. After point mutation, the single reading frame changed to two different ORFs. In this work, the two generated segments could be transcribed and should have functions. But we do not know whether they have the same function of encoding ribosomal protein S9. Finally, LHK_A0008 has possible function of coding for thioesterase. </p>                        <a id="5359966" scrollto-destination="5359966"></a>
<div content-id="DST009TB5" class="table-modal"><div class="table-wrap"><div id="DST009TB5" class="table-wrap-title" data-id="DST009TB5"><div class="label -label">Table 5.</div><div class="caption"><p>PCR primers and annealing temperature for the eight newly found genes in <em>L. hongkongensis</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>ID </th><th>Primer pair </th><th>Primer sequence </th><th>Annealing temperature </th></tr></thead><tbody><tr><td>LHK_A0001 </td><td>LPW20036<br /> LPW20037 </td><td>GCATGCCGAATTCCTCGAAG<br /> TCCGGGCCTTCTTCCAGTTC </td><td>60 </td></tr><tr><td>LHK_A0002 </td><td>LPW20038<br /> LPW20039 </td><td>ACGCGCTTTGATTCGGGAAC<br /> GCGTTCGCATAACCGTACAG </td><td>60 </td></tr><tr><td>LHK_A0003 </td><td>LPW20046<br /> LPW20047 </td><td>TGGCCAATCCGATCGTGAC<br /> CCTCCTGAGCGTTTCAAG </td><td>55 </td></tr><tr><td>LHK_A0004 </td><td>LPW19946<br /> LPW19947 </td><td>ATTCATCCGTCGTGGCTAAG<br /> TGACCACAAGCAGCCACATC </td><td>65 </td></tr><tr><td>LHK_A0005 </td><td>LPW19950<br /> LPW19951 </td><td>TGGGCGCCATGATCACCTAC<br /> CGGCAGGCATGGTGATGAAG </td><td>65 </td></tr><tr><td>LHK_A0006 </td><td>LPW19952<br /> LPW19953 </td><td>TGGCGCTTCATCCGCATCAC<br /> TCCGGCATCAGTACCGAGAC </td><td>65 </td></tr><tr><td>LHK_A0007 </td><td>LPW20545<br /> LPW20546 </td><td>CATCACCCGTGCCCTGAT<br /> CTTGGAGAACTGCTTGCG </td><td>60 </td></tr><tr><td>LHK_A0008 </td><td>LPW20048<br /> LPW20049 </td><td>CTGACACCCGGTGCAGTTTC<br /> CTGGCGTAATCCACCCAGAC </td><td>60 </td></tr></tbody></table></div><div content-id="DST009TB5" class="table-modal"><table><thead name="thead" align=""><tr><th>ID </th><th>Primer pair </th><th>Primer sequence </th><th>Annealing temperature </th></tr></thead><tbody><tr><td>LHK_A0001 </td><td>LPW20036<br /> LPW20037 </td><td>GCATGCCGAATTCCTCGAAG<br /> TCCGGGCCTTCTTCCAGTTC </td><td>60 </td></tr><tr><td>LHK_A0002 </td><td>LPW20038<br /> LPW20039 </td><td>ACGCGCTTTGATTCGGGAAC<br /> GCGTTCGCATAACCGTACAG </td><td>60 </td></tr><tr><td>LHK_A0003 </td><td>LPW20046<br /> LPW20047 </td><td>TGGCCAATCCGATCGTGAC<br /> CCTCCTGAGCGTTTCAAG </td><td>55 </td></tr><tr><td>LHK_A0004 </td><td>LPW19946<br /> LPW19947 </td><td>ATTCATCCGTCGTGGCTAAG<br /> TGACCACAAGCAGCCACATC </td><td>65 </td></tr><tr><td>LHK_A0005 </td><td>LPW19950<br /> LPW19951 </td><td>TGGGCGCCATGATCACCTAC<br /> CGGCAGGCATGGTGATGAAG </td><td>65 </td></tr><tr><td>LHK_A0006 </td><td>LPW19952<br /> LPW19953 </td><td>TGGCGCTTCATCCGCATCAC<br /> TCCGGCATCAGTACCGAGAC </td><td>65 </td></tr><tr><td>LHK_A0007 </td><td>LPW20545<br /> LPW20546 </td><td>CATCACCCGTGCCCTGAT<br /> CTTGGAGAACTGCTTGCG </td><td>60 </td></tr><tr><td>LHK_A0008 </td><td>LPW20048<br /> LPW20049 </td><td>CTGACACCCGGTGCAGTTTC<br /> CTGGCGTAATCCACCCAGAC </td><td>60 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5359966" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST009TB5" class="table-wrap-title" data-id="DST009TB5"><div class="label -label">Table 5.</div><div class="caption"><p>PCR primers and annealing temperature for the eight newly found genes in <em>L. hongkongensis</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>ID </th><th>Primer pair </th><th>Primer sequence </th><th>Annealing temperature </th></tr></thead><tbody><tr><td>LHK_A0001 </td><td>LPW20036<br /> LPW20037 </td><td>GCATGCCGAATTCCTCGAAG<br /> TCCGGGCCTTCTTCCAGTTC </td><td>60 </td></tr><tr><td>LHK_A0002 </td><td>LPW20038<br /> LPW20039 </td><td>ACGCGCTTTGATTCGGGAAC<br /> GCGTTCGCATAACCGTACAG </td><td>60 </td></tr><tr><td>LHK_A0003 </td><td>LPW20046<br /> LPW20047 </td><td>TGGCCAATCCGATCGTGAC<br /> CCTCCTGAGCGTTTCAAG </td><td>55 </td></tr><tr><td>LHK_A0004 </td><td>LPW19946<br /> LPW19947 </td><td>ATTCATCCGTCGTGGCTAAG<br /> TGACCACAAGCAGCCACATC </td><td>65 </td></tr><tr><td>LHK_A0005 </td><td>LPW19950<br /> LPW19951 </td><td>TGGGCGCCATGATCACCTAC<br /> CGGCAGGCATGGTGATGAAG </td><td>65 </td></tr><tr><td>LHK_A0006 </td><td>LPW19952<br /> LPW19953 </td><td>TGGCGCTTCATCCGCATCAC<br /> TCCGGCATCAGTACCGAGAC </td><td>65 </td></tr><tr><td>LHK_A0007 </td><td>LPW20545<br /> LPW20546 </td><td>CATCACCCGTGCCCTGAT<br /> CTTGGAGAACTGCTTGCG </td><td>60 </td></tr><tr><td>LHK_A0008 </td><td>LPW20048<br /> LPW20049 </td><td>CTGACACCCGGTGCAGTTTC<br /> CTGGCGTAATCCACCCAGAC </td><td>60 </td></tr></tbody></table></div><div content-id="DST009TB5" class="table-modal"><table><thead name="thead" align=""><tr><th>ID </th><th>Primer pair </th><th>Primer sequence </th><th>Annealing temperature </th></tr></thead><tbody><tr><td>LHK_A0001 </td><td>LPW20036<br /> LPW20037 </td><td>GCATGCCGAATTCCTCGAAG<br /> TCCGGGCCTTCTTCCAGTTC </td><td>60 </td></tr><tr><td>LHK_A0002 </td><td>LPW20038<br /> LPW20039 </td><td>ACGCGCTTTGATTCGGGAAC<br /> GCGTTCGCATAACCGTACAG </td><td>60 </td></tr><tr><td>LHK_A0003 </td><td>LPW20046<br /> LPW20047 </td><td>TGGCCAATCCGATCGTGAC<br /> CCTCCTGAGCGTTTCAAG </td><td>55 </td></tr><tr><td>LHK_A0004 </td><td>LPW19946<br /> LPW19947 </td><td>ATTCATCCGTCGTGGCTAAG<br /> TGACCACAAGCAGCCACATC </td><td>65 </td></tr><tr><td>LHK_A0005 </td><td>LPW19950<br /> LPW19951 </td><td>TGGGCGCCATGATCACCTAC<br /> CGGCAGGCATGGTGATGAAG </td><td>65 </td></tr><tr><td>LHK_A0006 </td><td>LPW19952<br /> LPW19953 </td><td>TGGCGCTTCATCCGCATCAC<br /> TCCGGCATCAGTACCGAGAC </td><td>65 </td></tr><tr><td>LHK_A0007 </td><td>LPW20545<br /> LPW20546 </td><td>CATCACCCGTGCCCTGAT<br /> CTTGGAGAACTGCTTGCG </td><td>60 </td></tr><tr><td>LHK_A0008 </td><td>LPW20048<br /> LPW20049 </td><td>CTGACACCCGGTGCAGTTTC<br /> CTGGCGTAATCCACCCAGAC </td><td>60 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5359966" target="_blank">View Large</a></div></div>                        <a id="5359967" scrollto-destination="5359967"></a>
<div id="DST009F4" data-id="DST009F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst009/2/m_dst00904.jpeg?Expires=2147483647&Signature=ngNkii~ssSIejLUfkx47yhNhzjq8b2PSYYJ0nOErDBbHr9tRyivP04cT3V12G1BXqCILTUcTd1-XwHZufQ-H1qJ4OXZgoFayhDqLpLkYIukLfuGnhXB37cWY0CnAj~8TMtgNmoV5DMfgxuMKZRh9hYKE~M2l2KWTVCAu6CmESa~1OrTAHF6Iir8UFaOHaRody4kj~3ojFrLEstJNuR~Ij6KtBc4eAc~v0Z0XWLHw4-8b7jZAl3mmIqFRC-K3wkTaZFGrPwPU3Pwrs1WUkV2VUSIiHTMtxGhldGDLUtjS6xjeJZwenLjXn3XqZyv-DgrcnsTS3L6hj6JjRjgmeAmCHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Matching relationship of aa sequences encoded by LHK_02777 and LHK_A0007 with the RpsI protein in the genome of Pseudogulbenkiania. The plot is adapted from the result generated by the NCBI blast application. In the search, the query is LHK_02777 and LHK_A0007, respectively, whereas the rpsI protein constitutes the subject." path-from-xml="dst00904.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5359967/dst00904.jpeg" path-from-xml="dst00904.jpeg" target="_blank">View large</a><a section="5359967" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst009/2/dst00904.jpeg?Expires=2147483647&Signature=po1QJJI8bkKReAe4Fvv3HeDc5GyFnldi5B6HRkRHEl3s9o5iIppG~PuhrZwKLGXhlh3VpvC1kQ7FQvS64o-U1tWEgVP1lzZj9cZ1OFeh8csoNAJrQK1-8DCfoHo7WY3W2oNgsZFMGOWWA550VSw8rjnbYR3VT-hmDMW32Nh~UZ-TXMBVOlbtvEgE52r2cVMKntG8Oj1Oe7W~qT8RlNdSbj5CUWRzKSDKxoYVZjP6iWwk8nV4mGqWfXP9kBfn8p0ZsmZseYlhIRJCincLCcdyR-YGiL1bR17DF~C~gv5rKQjXcVDKMw28uc9BEhzySqKgiRZbyZHI7mP4HFIEmtslaA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5359967&ar=369179&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00904.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Matching relationship of aa sequences encoded by LHK_02777 and LHK_A0007 with the RpsI protein in the genome of <em>Pseudogulbenkiania</em>. The plot is adapted from the result generated by the NCBI blast application. In the search, the query is LHK_02777 and LHK_A0007, respectively, whereas the rpsI protein constitutes the subject.</p></div></div><div content-id="DST009F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst009/2/m_dst00904.jpeg?Expires=2147483647&Signature=ngNkii~ssSIejLUfkx47yhNhzjq8b2PSYYJ0nOErDBbHr9tRyivP04cT3V12G1BXqCILTUcTd1-XwHZufQ-H1qJ4OXZgoFayhDqLpLkYIukLfuGnhXB37cWY0CnAj~8TMtgNmoV5DMfgxuMKZRh9hYKE~M2l2KWTVCAu6CmESa~1OrTAHF6Iir8UFaOHaRody4kj~3ojFrLEstJNuR~Ij6KtBc4eAc~v0Z0XWLHw4-8b7jZAl3mmIqFRC-K3wkTaZFGrPwPU3Pwrs1WUkV2VUSIiHTMtxGhldGDLUtjS6xjeJZwenLjXn3XqZyv-DgrcnsTS3L6hj6JjRjgmeAmCHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Matching relationship of aa sequences encoded by LHK_02777 and LHK_A0007 with the RpsI protein in the genome of Pseudogulbenkiania. The plot is adapted from the result generated by the NCBI blast application. In the search, the query is LHK_02777 and LHK_A0007, respectively, whereas the rpsI protein constitutes the subject." path-from-xml="dst00904.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5359967/dst00904.jpeg" path-from-xml="dst00904.jpeg" target="_blank">View large</a><a section="5359967" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst009/2/dst00904.jpeg?Expires=2147483647&Signature=po1QJJI8bkKReAe4Fvv3HeDc5GyFnldi5B6HRkRHEl3s9o5iIppG~PuhrZwKLGXhlh3VpvC1kQ7FQvS64o-U1tWEgVP1lzZj9cZ1OFeh8csoNAJrQK1-8DCfoHo7WY3W2oNgsZFMGOWWA550VSw8rjnbYR3VT-hmDMW32Nh~UZ-TXMBVOlbtvEgE52r2cVMKntG8Oj1Oe7W~qT8RlNdSbj5CUWRzKSDKxoYVZjP6iWwk8nV4mGqWfXP9kBfn8p0ZsmZseYlhIRJCincLCcdyR-YGiL1bR17DF~C~gv5rKQjXcVDKMw28uc9BEhzySqKgiRZbyZHI7mP4HFIEmtslaA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5359967&ar=369179&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00904.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Matching relationship of aa sequences encoded by LHK_02777 and LHK_A0007 with the RpsI protein in the genome of <em>Pseudogulbenkiania</em>. The plot is adapted from the result generated by the NCBI blast application. In the search, the query is LHK_02777 and LHK_A0007, respectively, whereas the rpsI protein constitutes the subject.</p></div></div><p>For the 10 newly found potential genes in <em>C. violaceum</em>, there are not so serious cases of overlapping with annotated genes as in <em>L. hongkongensis</em>. Either the overlapping part is shorter than 15 bp or the overlapping gene is annotated as hypothetical protein that has not significant similarity with known genes in the public database. Details of the 10 genes and their corresponding functions and the details of PCR primers and conditions used in the validation experiments are listed in Tables <a reveal-id="DST009TB6" data-open="DST009TB6" class="link link-reveal link-table xref-fig">6</a> and <a reveal-id="DST009TB7" data-open="DST009TB7" class="link link-reveal link-table xref-fig">7</a>, respectively. The two negative samples with RT–PCR analysis may constitute falsely positive predictions of our method, or alternatively, they are expressed only in special conditions. </p>                        <a id="5359970" scrollto-destination="5359970"></a>
<div content-id="DST009TB6" class="table-modal"><div class="table-wrap"><div id="DST009TB6" class="table-wrap-title" data-id="DST009TB6"><div class="label -label">Table 6.</div><div class="caption"><p>Details of the 10 newly found genes in the genome of <em>C. violaceum</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>ID </th><th>Position </th><th>COG </th><th>Coverage, <em>E</em>-value, Identity </th><th>Potential function </th></tr></thead><tbody><tr><td>CV_A0001 </td><td>486035–487360 (+) </td><td>COG0845Q </td><td>100%, 0, 72% </td><td>Membrane-fusion protein </td></tr><tr><td>CV_A0002 </td><td>487503–489461 (+) </td><td>COG0750M </td><td>100%, 0, 71% </td><td>Membrane-associated Zn-dependent proteases </td></tr><tr><td>CV_A0003 </td><td>1025430–1025810 (+) </td><td>COG3536S </td><td>89%, 4e−67, 83% </td><td>Uncharacterized bacterial conserved region (BCR) </td></tr><tr><td>CV_A0004 </td><td>1026544–1027152 (+) </td><td>COG3165S </td><td>95%, 6e−88, 66% </td><td>Uncharacterized BCR </td></tr><tr><td>CV_A0005 </td><td>1956990–1958684 (+) </td><td>COG0654HC </td><td>88%, 3e−177, 53% </td><td>2-polyprenyl-6-methoxyphenol hydroxylase </td></tr><tr><td>CV_A0006 </td><td>2305072–2305416 (−) </td><td>COG3628R </td><td>99%, 4e−53, 71% </td><td>Phage baseplate assembly protein </td></tr><tr><td>CV_A0007 </td><td>2362585–2363220 (−) </td><td>COG0693R </td><td>92%, 1e−70, 54% </td><td>Intracellular protease/amidase </td></tr><tr><td>CV_A0008 </td><td>4207595–4208386 (+) </td><td>COG0614P </td><td>85%, 6e−94, 67% </td><td>ABC-type cobalamin/Fe3+-siderophores transport systems </td></tr><tr><td>CV_A0009 </td><td>4462117–4463199 (−) </td><td>COG0438M </td><td>96%, 5e−137, 62% </td><td>Predicted glycosyltransferases </td></tr><tr><td>CV_A0010 </td><td>4588140–4588436 (−) </td><td>COG1872S </td><td>98%, 3e−47, 75% </td><td>Uncharacterized ancient conserved region </td></tr></tbody></table></div><div content-id="DST009TB6" class="table-modal"><table><thead name="thead" align=""><tr><th>ID </th><th>Position </th><th>COG </th><th>Coverage, <em>E</em>-value, Identity </th><th>Potential function </th></tr></thead><tbody><tr><td>CV_A0001 </td><td>486035–487360 (+) </td><td>COG0845Q </td><td>100%, 0, 72% </td><td>Membrane-fusion protein </td></tr><tr><td>CV_A0002 </td><td>487503–489461 (+) </td><td>COG0750M </td><td>100%, 0, 71% </td><td>Membrane-associated Zn-dependent proteases </td></tr><tr><td>CV_A0003 </td><td>1025430–1025810 (+) </td><td>COG3536S </td><td>89%, 4e−67, 83% </td><td>Uncharacterized bacterial conserved region (BCR) </td></tr><tr><td>CV_A0004 </td><td>1026544–1027152 (+) </td><td>COG3165S </td><td>95%, 6e−88, 66% </td><td>Uncharacterized BCR </td></tr><tr><td>CV_A0005 </td><td>1956990–1958684 (+) </td><td>COG0654HC </td><td>88%, 3e−177, 53% </td><td>2-polyprenyl-6-methoxyphenol hydroxylase </td></tr><tr><td>CV_A0006 </td><td>2305072–2305416 (−) </td><td>COG3628R </td><td>99%, 4e−53, 71% </td><td>Phage baseplate assembly protein </td></tr><tr><td>CV_A0007 </td><td>2362585–2363220 (−) </td><td>COG0693R </td><td>92%, 1e−70, 54% </td><td>Intracellular protease/amidase </td></tr><tr><td>CV_A0008 </td><td>4207595–4208386 (+) </td><td>COG0614P </td><td>85%, 6e−94, 67% </td><td>ABC-type cobalamin/Fe3+-siderophores transport systems </td></tr><tr><td>CV_A0009 </td><td>4462117–4463199 (−) </td><td>COG0438M </td><td>96%, 5e−137, 62% </td><td>Predicted glycosyltransferases </td></tr><tr><td>CV_A0010 </td><td>4588140–4588436 (−) </td><td>COG1872S </td><td>98%, 3e−47, 75% </td><td>Uncharacterized ancient conserved region </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5359970" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST009TB6" class="table-wrap-title" data-id="DST009TB6"><div class="label -label">Table 6.</div><div class="caption"><p>Details of the 10 newly found genes in the genome of <em>C. violaceum</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>ID </th><th>Position </th><th>COG </th><th>Coverage, <em>E</em>-value, Identity </th><th>Potential function </th></tr></thead><tbody><tr><td>CV_A0001 </td><td>486035–487360 (+) </td><td>COG0845Q </td><td>100%, 0, 72% </td><td>Membrane-fusion protein </td></tr><tr><td>CV_A0002 </td><td>487503–489461 (+) </td><td>COG0750M </td><td>100%, 0, 71% </td><td>Membrane-associated Zn-dependent proteases </td></tr><tr><td>CV_A0003 </td><td>1025430–1025810 (+) </td><td>COG3536S </td><td>89%, 4e−67, 83% </td><td>Uncharacterized bacterial conserved region (BCR) </td></tr><tr><td>CV_A0004 </td><td>1026544–1027152 (+) </td><td>COG3165S </td><td>95%, 6e−88, 66% </td><td>Uncharacterized BCR </td></tr><tr><td>CV_A0005 </td><td>1956990–1958684 (+) </td><td>COG0654HC </td><td>88%, 3e−177, 53% </td><td>2-polyprenyl-6-methoxyphenol hydroxylase </td></tr><tr><td>CV_A0006 </td><td>2305072–2305416 (−) </td><td>COG3628R </td><td>99%, 4e−53, 71% </td><td>Phage baseplate assembly protein </td></tr><tr><td>CV_A0007 </td><td>2362585–2363220 (−) </td><td>COG0693R </td><td>92%, 1e−70, 54% </td><td>Intracellular protease/amidase </td></tr><tr><td>CV_A0008 </td><td>4207595–4208386 (+) </td><td>COG0614P </td><td>85%, 6e−94, 67% </td><td>ABC-type cobalamin/Fe3+-siderophores transport systems </td></tr><tr><td>CV_A0009 </td><td>4462117–4463199 (−) </td><td>COG0438M </td><td>96%, 5e−137, 62% </td><td>Predicted glycosyltransferases </td></tr><tr><td>CV_A0010 </td><td>4588140–4588436 (−) </td><td>COG1872S </td><td>98%, 3e−47, 75% </td><td>Uncharacterized ancient conserved region </td></tr></tbody></table></div><div content-id="DST009TB6" class="table-modal"><table><thead name="thead" align=""><tr><th>ID </th><th>Position </th><th>COG </th><th>Coverage, <em>E</em>-value, Identity </th><th>Potential function </th></tr></thead><tbody><tr><td>CV_A0001 </td><td>486035–487360 (+) </td><td>COG0845Q </td><td>100%, 0, 72% </td><td>Membrane-fusion protein </td></tr><tr><td>CV_A0002 </td><td>487503–489461 (+) </td><td>COG0750M </td><td>100%, 0, 71% </td><td>Membrane-associated Zn-dependent proteases </td></tr><tr><td>CV_A0003 </td><td>1025430–1025810 (+) </td><td>COG3536S </td><td>89%, 4e−67, 83% </td><td>Uncharacterized bacterial conserved region (BCR) </td></tr><tr><td>CV_A0004 </td><td>1026544–1027152 (+) </td><td>COG3165S </td><td>95%, 6e−88, 66% </td><td>Uncharacterized BCR </td></tr><tr><td>CV_A0005 </td><td>1956990–1958684 (+) </td><td>COG0654HC </td><td>88%, 3e−177, 53% </td><td>2-polyprenyl-6-methoxyphenol hydroxylase </td></tr><tr><td>CV_A0006 </td><td>2305072–2305416 (−) </td><td>COG3628R </td><td>99%, 4e−53, 71% </td><td>Phage baseplate assembly protein </td></tr><tr><td>CV_A0007 </td><td>2362585–2363220 (−) </td><td>COG0693R </td><td>92%, 1e−70, 54% </td><td>Intracellular protease/amidase </td></tr><tr><td>CV_A0008 </td><td>4207595–4208386 (+) </td><td>COG0614P </td><td>85%, 6e−94, 67% </td><td>ABC-type cobalamin/Fe3+-siderophores transport systems </td></tr><tr><td>CV_A0009 </td><td>4462117–4463199 (−) </td><td>COG0438M </td><td>96%, 5e−137, 62% </td><td>Predicted glycosyltransferases </td></tr><tr><td>CV_A0010 </td><td>4588140–4588436 (−) </td><td>COG1872S </td><td>98%, 3e−47, 75% </td><td>Uncharacterized ancient conserved region </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5359970" target="_blank">View Large</a></div></div>                        <a id="5359971" scrollto-destination="5359971"></a>
<div content-id="DST009TB7" class="table-modal"><div class="table-wrap"><div id="DST009TB7" class="table-wrap-title" data-id="DST009TB7"><div class="label -label">Table 7.</div><div class="caption"><p>PCR primers and annealing temperature for the 10 newly found potential genes in <em>C. violaceum</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>ID </th><th>Primer pair </th><th>Primer sequence </th><th>Annealing temperature </th></tr></thead><tbody><tr><td>1 </td><td>LPW21817 LPW21818 </td><td>CTGGCATTGACCGATGAC CGAAGCGTTGGGATACAG </td><td>55 </td></tr><tr><td>2 </td><td>LPW21819 LPW21820 </td><td>CTGTATCGCCTGGTGTTG GCCCTTGCTCTGCAAATC </td><td>45 </td></tr><tr><td>3 </td><td>LPW21821 LPW21822 </td><td>TGCCCATGTCAGGACTTG GAGCTTGTCCAGGTATTG </td><td>55 </td></tr><tr><td>4 </td><td>LPW21823 LPW21824 </td><td>GATTTGTCGCGGGTGTTC GAAGCGTTGAACCAGATG </td><td>60 </td></tr><tr><td>5 </td><td>LPW21825 LPW21826 </td><td>CATGAGGTTAGCCCTTTC GCCATCGACGAAATACAG </td><td>55 </td></tr><tr><td>6 </td><td>LPW21827 LPW21828 </td><td>CAGTGCATCCGCATCATC GCTCCCATTGCCGAATAG </td><td>60 </td></tr><tr><td>7 </td><td>LPW21829 LPW21830 </td><td>CAGGAAGACCTGTCTTAC CTGGCAAAGTCCTCTTCC </td><td>55 </td></tr><tr><td>8 </td><td>LPW21835 LPW21836 </td><td>CGCAGCTGAAGCAGCTGAAG CGGCTTGAAACCGTTGAG </td><td>48 </td></tr><tr><td>9 </td><td>LPW21837 LPW21838 </td><td>TTGAGCTACGGCATAGAC GCCAGCCGTTTCAGATTC </td><td>55 </td></tr><tr><td>10 </td><td>LPW21839 LPW21840 </td><td>CGTCTGACGCTGCATGTG GTCGCCGGACAACAATTC </td><td>55 </td></tr></tbody></table></div><div content-id="DST009TB7" class="table-modal"><table><thead name="thead" align=""><tr><th>ID </th><th>Primer pair </th><th>Primer sequence </th><th>Annealing temperature </th></tr></thead><tbody><tr><td>1 </td><td>LPW21817 LPW21818 </td><td>CTGGCATTGACCGATGAC CGAAGCGTTGGGATACAG </td><td>55 </td></tr><tr><td>2 </td><td>LPW21819 LPW21820 </td><td>CTGTATCGCCTGGTGTTG GCCCTTGCTCTGCAAATC </td><td>45 </td></tr><tr><td>3 </td><td>LPW21821 LPW21822 </td><td>TGCCCATGTCAGGACTTG GAGCTTGTCCAGGTATTG </td><td>55 </td></tr><tr><td>4 </td><td>LPW21823 LPW21824 </td><td>GATTTGTCGCGGGTGTTC GAAGCGTTGAACCAGATG </td><td>60 </td></tr><tr><td>5 </td><td>LPW21825 LPW21826 </td><td>CATGAGGTTAGCCCTTTC GCCATCGACGAAATACAG </td><td>55 </td></tr><tr><td>6 </td><td>LPW21827 LPW21828 </td><td>CAGTGCATCCGCATCATC GCTCCCATTGCCGAATAG </td><td>60 </td></tr><tr><td>7 </td><td>LPW21829 LPW21830 </td><td>CAGGAAGACCTGTCTTAC CTGGCAAAGTCCTCTTCC </td><td>55 </td></tr><tr><td>8 </td><td>LPW21835 LPW21836 </td><td>CGCAGCTGAAGCAGCTGAAG CGGCTTGAAACCGTTGAG </td><td>48 </td></tr><tr><td>9 </td><td>LPW21837 LPW21838 </td><td>TTGAGCTACGGCATAGAC GCCAGCCGTTTCAGATTC </td><td>55 </td></tr><tr><td>10 </td><td>LPW21839 LPW21840 </td><td>CGTCTGACGCTGCATGTG GTCGCCGGACAACAATTC </td><td>55 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5359971" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST009TB7" class="table-wrap-title" data-id="DST009TB7"><div class="label -label">Table 7.</div><div class="caption"><p>PCR primers and annealing temperature for the 10 newly found potential genes in <em>C. violaceum</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>ID </th><th>Primer pair </th><th>Primer sequence </th><th>Annealing temperature </th></tr></thead><tbody><tr><td>1 </td><td>LPW21817 LPW21818 </td><td>CTGGCATTGACCGATGAC CGAAGCGTTGGGATACAG </td><td>55 </td></tr><tr><td>2 </td><td>LPW21819 LPW21820 </td><td>CTGTATCGCCTGGTGTTG GCCCTTGCTCTGCAAATC </td><td>45 </td></tr><tr><td>3 </td><td>LPW21821 LPW21822 </td><td>TGCCCATGTCAGGACTTG GAGCTTGTCCAGGTATTG </td><td>55 </td></tr><tr><td>4 </td><td>LPW21823 LPW21824 </td><td>GATTTGTCGCGGGTGTTC GAAGCGTTGAACCAGATG </td><td>60 </td></tr><tr><td>5 </td><td>LPW21825 LPW21826 </td><td>CATGAGGTTAGCCCTTTC GCCATCGACGAAATACAG </td><td>55 </td></tr><tr><td>6 </td><td>LPW21827 LPW21828 </td><td>CAGTGCATCCGCATCATC GCTCCCATTGCCGAATAG </td><td>60 </td></tr><tr><td>7 </td><td>LPW21829 LPW21830 </td><td>CAGGAAGACCTGTCTTAC CTGGCAAAGTCCTCTTCC </td><td>55 </td></tr><tr><td>8 </td><td>LPW21835 LPW21836 </td><td>CGCAGCTGAAGCAGCTGAAG CGGCTTGAAACCGTTGAG </td><td>48 </td></tr><tr><td>9 </td><td>LPW21837 LPW21838 </td><td>TTGAGCTACGGCATAGAC GCCAGCCGTTTCAGATTC </td><td>55 </td></tr><tr><td>10 </td><td>LPW21839 LPW21840 </td><td>CGTCTGACGCTGCATGTG GTCGCCGGACAACAATTC </td><td>55 </td></tr></tbody></table></div><div content-id="DST009TB7" class="table-modal"><table><thead name="thead" align=""><tr><th>ID </th><th>Primer pair </th><th>Primer sequence </th><th>Annealing temperature </th></tr></thead><tbody><tr><td>1 </td><td>LPW21817 LPW21818 </td><td>CTGGCATTGACCGATGAC CGAAGCGTTGGGATACAG </td><td>55 </td></tr><tr><td>2 </td><td>LPW21819 LPW21820 </td><td>CTGTATCGCCTGGTGTTG GCCCTTGCTCTGCAAATC </td><td>45 </td></tr><tr><td>3 </td><td>LPW21821 LPW21822 </td><td>TGCCCATGTCAGGACTTG GAGCTTGTCCAGGTATTG </td><td>55 </td></tr><tr><td>4 </td><td>LPW21823 LPW21824 </td><td>GATTTGTCGCGGGTGTTC GAAGCGTTGAACCAGATG </td><td>60 </td></tr><tr><td>5 </td><td>LPW21825 LPW21826 </td><td>CATGAGGTTAGCCCTTTC GCCATCGACGAAATACAG </td><td>55 </td></tr><tr><td>6 </td><td>LPW21827 LPW21828 </td><td>CAGTGCATCCGCATCATC GCTCCCATTGCCGAATAG </td><td>60 </td></tr><tr><td>7 </td><td>LPW21829 LPW21830 </td><td>CAGGAAGACCTGTCTTAC CTGGCAAAGTCCTCTTCC </td><td>55 </td></tr><tr><td>8 </td><td>LPW21835 LPW21836 </td><td>CGCAGCTGAAGCAGCTGAAG CGGCTTGAAACCGTTGAG </td><td>48 </td></tr><tr><td>9 </td><td>LPW21837 LPW21838 </td><td>TTGAGCTACGGCATAGAC GCCAGCCGTTTCAGATTC </td><td>55 </td></tr><tr><td>10 </td><td>LPW21839 LPW21840 </td><td>CGTCTGACGCTGCATGTG GTCGCCGGACAACAATTC </td><td>55 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5359971" target="_blank">View Large</a></div></div><p>In the other nine Neisseriaceae strains, there were also newly found genes with potential functions. In <em>N. gonorrhoeae</em> FA 1090, only 1 of 18 newly added genes overlaps the annotated genes and the overlapping part is 20 bp. Among the 18 genes, the function information of four are so detailed to have definite gene names and they encode thiol:disulfide interchange protein DsbD, sulphate ABC transporter permease protein CysU, 23S rRNA (guanosine-2′-<em>O</em>-)-methyltransferase RlmB and septum site-determining protein MinD, respectively. In <em>N. gonorrhoeae</em> NCCP11945, nine new genes are found. Among them, five do not overlap annotated genes. Four of the five genes have putative important function based on very high similarity and they encode transcription antiterminator (Nusg), 30S ribosomal protein S14 (RpsN), 30S ribosomal protein L30 (RpmD) and translation initiation factor IF-1 (InfA). In <em>N. lactamica</em>, none of the five newly added genes overlap annotated genes. Among them, three encode DNA transport competence protein (ComeA), one encodes transposases and one encodes the TonB-dependent receptor. In <em>Pseudogulbenkiania</em>, only one gene is added. Interestingly, this gene has the same 5′ terminal with the annotated gene (NH8B_2210) but they do not have the same stop codon. The NH8B_2210 is much longer than the newly added one and they have the same reading frame. According to the RefSeq annotation for NH8B_2210, the original authors seemed to have predicted that there is a stop codon treated as the selenocystein codon. By blast against the public database, the newly added gene is shown to be a more reliable prediction because it has the same length with counterparts in distantly related species where as the annotated does not.</p><p>In the four strains of <em>N. meningitidi</em>s, from 6 to 30 new genes were added and very few of them overlap the annotated genes. The strain alpha14 <em>N. meningitidis</em> has the most found new genes and interestingly 27 among the 30 new genes encode transposases. Although their aa identities with known transposases in genomes beyond the <em>Neisseria</em> genus tend to be just slightly higher than 50%, the identity is higher than 80% for each of them with counterparts in the same genus. Furthermore, the coverage at each case is greater than 90%. Therefore, predicted functions of encoding transposases for the 27 new genes are reliable. According to the RefSeq annotation, none of the genes code for transposases in this strain. However, 55, 29 and 33 transposases have been annotated in the other three strains of <em>N. meningitidi</em>s. The lower sensitivity of the <em>ab initio</em> gene finder used in the annotation of <em>N. meningitidis</em> alpha14 is suggested to be responsible for the missing of transposases. In fact, some of transposases, which aid the integration insertion of genomic islands or single horizontally transferred gene, tend to own abnormal nucleotide composition and are easily missed by composition-based programmes.<sup><a reveal-id="DST009C41" data-open="DST009C41" class="link link-ref link-reveal xref-bibr">41</a></sup> In addition, some genes with important functions have been added in the <em>N. meningitidis</em> genomes, such as the haemoglobin receptor in 053442, transcription elongation factor (<em>greA</em>) in alpha14, protein methyltransferase (<em>hemK</em>) in Z2491, leucyl aminopeptidase (<em>pepA</em>) and allophanate hydrolase subunit 2 in MC58. Details of the 106 newly found genes in the 10 Neisseriaceae strains are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>.</p>                            <h3 scrollto-destination=5359974 id="5359974" class="section-title" data-legacy-id=s3c>3.3. Hypothetical proteins with newly assigned functions</h3>
<p>For the genomes sequenced several years ago, functional information may be outdated.<sup><a reveal-id="DST009C16" data-open="DST009C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DST009C29" data-open="DST009C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Especially, some hypothetical genes may have functional counterparts in current databases,<sup><a reveal-id="DST009C16" data-open="DST009C16" class="link link-ref link-reveal xref-bibr">16</a></sup> whereas they are still annotated as hypothetical. This section is aiming to provide functional information for hypothetical genes in the 10 Neisseriaceae strains by using the similarity search method. To ensure reliable function transfer, severe homologous conditions were adopted. After the blast search, varying numbers of hypothetical proteins were assigned functions in the 10 Neisseriaceae genomes (Table <a reveal-id="DST009TB3" data-open="DST009TB3" class="link link-reveal link-table xref-fig">3</a>). Among them, <em>L. hongkongensis</em> and <em>Pseudogulbenkiania</em> have the least numbers perhaps because their genomes were most recently sequenced. In the two strains of the <em>N. gonorrhoeae</em>, the strain FA 1090 has the larger number of assigned functions and it just was sequenced earlier than NCCP11945.<sup><a reveal-id="DST009C23" data-open="DST009C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DST009C42" data-open="DST009C42" class="link link-ref link-reveal xref-bibr">42</a></sup> Among the four strains of the species, <em>N. meningitidis</em> MC58 has the largest number and it just was sequenced earliest among them.<sup><a reveal-id="DST009C23" data-open="DST009C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Therefore, the number of hypothetical proteins with newly assigned functions may be tightly associated with sequenced time of the genome. In fact, if one genome has been sequenced at earlier year, it should have more functional counterparts in the current public database but not existed at that time.<sup><a reveal-id="DST009C29" data-open="DST009C29" class="link link-ref link-reveal xref-bibr">29</a></sup> The original annotation would be more outdated for earlier sequenced genome in the sense of function information.</p><p>Among newly assigned functions, some have been provided definite names (Table <a reveal-id="DST009TB8" data-open="DST009TB8" class="link link-reveal link-table xref-fig">8</a>) and this transferred annotation information should be unquestionable. For example, the hypothetical gene LHK_02863 in <em>L. hongkongensis</em> has been assigned not only the function encoding iron-sulphur cluster assembly protein but also the definite name ‘IscA’. Some other genes were assigned with detailed functions and would be reliable with this function because stringent homologous criteria were adopted. However, there are still some hypothetical proteins with only general functions, such as membrane proteins, lipoproteins and periplasmic proteins (Table <a reveal-id="DST009TB8" data-open="DST009TB8" class="link link-reveal link-table xref-fig">8</a>). This type of rough function information would give help for the determination of more detailed functions. Interestingly, the number of assigned membrane proteins tends to be larger than periplasmic proteins and much larger than lipoproteins in most cases (Table <a reveal-id="DST009TB8" data-open="DST009TB8" class="link link-reveal link-table xref-fig">8</a>). It is suggested that this information corresponds to their natural ranks existing in the genome. Details of assigned functions for each of the 10 Neisseriaceae strains are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>. </p>                        <a id="5359977" scrollto-destination="5359977"></a>
<div content-id="DST009TB8" class="table-modal"><div class="table-wrap"><div id="DST009TB8" class="table-wrap-title" data-id="DST009TB8"><div class="label -label">Table 8.</div><div class="caption"><p>Among hypothetical gene with assigned functions, the numbers of genes with definite names, those encoding membrane proteins, lipoproteins and periplasmic proteins</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Strains </th><th>With definite name </th><th>Membrane protein </th><th>Lipoprotein </th><th>Periplasmic protein </th></tr></thead><tbody><tr><td>C. <em>violaceum</em> </td><td>29 </td><td>5 </td><td>1 </td><td>1 </td></tr><tr><td><em>L. hongkongensis</em> </td><td>9 </td><td>3 </td><td>0 </td><td>0 </td></tr><tr><td><em>N. gonorrhoeae</em> FA 1090 </td><td>105 </td><td>72 </td><td>25 </td><td>31 </td></tr><tr><td><em>N. gonorrhoeae</em> NCCP11945 </td><td>34 </td><td>25 </td><td>6 </td><td>8 </td></tr><tr><td><em>N. lactamica</em> </td><td>39 </td><td>25 </td><td>6 </td><td>21 </td></tr><tr><td><em>N. meningitidis</em> 053442 </td><td>30 </td><td>21 </td><td>4 </td><td>19 </td></tr><tr><td><em>N. meningitidis</em> Alpha 14 </td><td>19 </td><td>67 </td><td>16 </td><td>26 </td></tr><tr><td><em>N. meningitidis</em> MC58 </td><td>47 </td><td>77 </td><td>26 </td><td>17 </td></tr><tr><td><em>N. meningitidis</em> Z2491 </td><td>78 </td><td>33 </td><td>8 </td><td>18 </td></tr><tr><td><em>Pseudogulbenkiania</em> </td><td>7 </td><td>9 </td><td>5 </td><td>0 </td></tr></tbody></table></div><div content-id="DST009TB8" class="table-modal"><table><thead name="thead" align=""><tr><th>Strains </th><th>With definite name </th><th>Membrane protein </th><th>Lipoprotein </th><th>Periplasmic protein </th></tr></thead><tbody><tr><td>C. <em>violaceum</em> </td><td>29 </td><td>5 </td><td>1 </td><td>1 </td></tr><tr><td><em>L. hongkongensis</em> </td><td>9 </td><td>3 </td><td>0 </td><td>0 </td></tr><tr><td><em>N. gonorrhoeae</em> FA 1090 </td><td>105 </td><td>72 </td><td>25 </td><td>31 </td></tr><tr><td><em>N. gonorrhoeae</em> NCCP11945 </td><td>34 </td><td>25 </td><td>6 </td><td>8 </td></tr><tr><td><em>N. lactamica</em> </td><td>39 </td><td>25 </td><td>6 </td><td>21 </td></tr><tr><td><em>N. meningitidis</em> 053442 </td><td>30 </td><td>21 </td><td>4 </td><td>19 </td></tr><tr><td><em>N. meningitidis</em> Alpha 14 </td><td>19 </td><td>67 </td><td>16 </td><td>26 </td></tr><tr><td><em>N. meningitidis</em> MC58 </td><td>47 </td><td>77 </td><td>26 </td><td>17 </td></tr><tr><td><em>N. meningitidis</em> Z2491 </td><td>78 </td><td>33 </td><td>8 </td><td>18 </td></tr><tr><td><em>Pseudogulbenkiania</em> </td><td>7 </td><td>9 </td><td>5 </td><td>0 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5359977" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST009TB8" class="table-wrap-title" data-id="DST009TB8"><div class="label -label">Table 8.</div><div class="caption"><p>Among hypothetical gene with assigned functions, the numbers of genes with definite names, those encoding membrane proteins, lipoproteins and periplasmic proteins</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Strains </th><th>With definite name </th><th>Membrane protein </th><th>Lipoprotein </th><th>Periplasmic protein </th></tr></thead><tbody><tr><td>C. <em>violaceum</em> </td><td>29 </td><td>5 </td><td>1 </td><td>1 </td></tr><tr><td><em>L. hongkongensis</em> </td><td>9 </td><td>3 </td><td>0 </td><td>0 </td></tr><tr><td><em>N. gonorrhoeae</em> FA 1090 </td><td>105 </td><td>72 </td><td>25 </td><td>31 </td></tr><tr><td><em>N. gonorrhoeae</em> NCCP11945 </td><td>34 </td><td>25 </td><td>6 </td><td>8 </td></tr><tr><td><em>N. lactamica</em> </td><td>39 </td><td>25 </td><td>6 </td><td>21 </td></tr><tr><td><em>N. meningitidis</em> 053442 </td><td>30 </td><td>21 </td><td>4 </td><td>19 </td></tr><tr><td><em>N. meningitidis</em> Alpha 14 </td><td>19 </td><td>67 </td><td>16 </td><td>26 </td></tr><tr><td><em>N. meningitidis</em> MC58 </td><td>47 </td><td>77 </td><td>26 </td><td>17 </td></tr><tr><td><em>N. meningitidis</em> Z2491 </td><td>78 </td><td>33 </td><td>8 </td><td>18 </td></tr><tr><td><em>Pseudogulbenkiania</em> </td><td>7 </td><td>9 </td><td>5 </td><td>0 </td></tr></tbody></table></div><div content-id="DST009TB8" class="table-modal"><table><thead name="thead" align=""><tr><th>Strains </th><th>With definite name </th><th>Membrane protein </th><th>Lipoprotein </th><th>Periplasmic protein </th></tr></thead><tbody><tr><td>C. <em>violaceum</em> </td><td>29 </td><td>5 </td><td>1 </td><td>1 </td></tr><tr><td><em>L. hongkongensis</em> </td><td>9 </td><td>3 </td><td>0 </td><td>0 </td></tr><tr><td><em>N. gonorrhoeae</em> FA 1090 </td><td>105 </td><td>72 </td><td>25 </td><td>31 </td></tr><tr><td><em>N. gonorrhoeae</em> NCCP11945 </td><td>34 </td><td>25 </td><td>6 </td><td>8 </td></tr><tr><td><em>N. lactamica</em> </td><td>39 </td><td>25 </td><td>6 </td><td>21 </td></tr><tr><td><em>N. meningitidis</em> 053442 </td><td>30 </td><td>21 </td><td>4 </td><td>19 </td></tr><tr><td><em>N. meningitidis</em> Alpha 14 </td><td>19 </td><td>67 </td><td>16 </td><td>26 </td></tr><tr><td><em>N. meningitidis</em> MC58 </td><td>47 </td><td>77 </td><td>26 </td><td>17 </td></tr><tr><td><em>N. meningitidis</em> Z2491 </td><td>78 </td><td>33 </td><td>8 </td><td>18 </td></tr><tr><td><em>Pseudogulbenkiania</em> </td><td>7 </td><td>9 </td><td>5 </td><td>0 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5359977" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=5359978 id="5359978" class="section-title" data-legacy-id=s3d>3.4. Disrupted ORFs</h3>
<p>During the process of genome re-annotation for the 10 Neisseriaceae strains, we found an interesting phenomenon. In a specific genome, two adjacent ORFs, which are predicted as genes by the ZCURVE program, have the same function with known genes in the other species. The total length of the two ORFs just corresponds to that of the known counterparts or just a little different from them. Based on similarity scores, both of the two ORFs should be predicted as genes with the corresponding function. However, either of them is much shorter than the functional counterpart and could not constitute a homologue in the sense of length information. Perrodou <em>et al</em>.<sup><a reveal-id="DST009C43" data-open="DST009C43" class="link link-ref link-reveal xref-bibr">43</a></sup> stated that unrecognized frameshifts, in-frame stop codons and sequencing errors often led to interrupted coding sequences. Very recently, Sharma <em>et al</em>.<sup><a reveal-id="DST009C44" data-open="DST009C44" class="link link-ref link-reveal xref-bibr">44</a></sup> performed a pilot study on bacterial genes with disrupted ORFs. Their results indicated that many disrupted genes likely utilized the non-standard decoding mechanisms: programmed ribosomal frameshifting and programmed transcriptional realignment. Given that our recognized adjacent ORFs have identical functions, they should originate from the disruption of a longer gene by a frameshift or an in-frame stop codon and rarely by the sequencing error. Numbers of disrupted ORFs are listed in Table <a reveal-id="DST009TB3" data-open="DST009TB3" class="link link-reveal link-table xref-fig">3</a> and details of them are illustrated in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>. Totally, 111 disrupted ORFs (or partial genes) were identified in six strains of the genus <em>Neisseria</em> and two in <em>L. hongkongensis</em>. According to Sharma <em>et al</em>.,<sup><a reveal-id="DST009C44" data-open="DST009C44" class="link link-ref link-reveal xref-bibr">44</a></sup> some of the disrupted ORFs still encode proteins. Consistent with this, both of the disrupted ORFs LHK_A0007 (newly predicted) and LHK_02777 (originally annotated) are found to be transcribed in <em>L. hongkongensis</em> cells by the PCR experiment. Further wet experiment on functions of the other disrupted ORFs would be interesting.</p>                            <h2 scrollto-destination=5359980 id="5359980" class="section-title" data-legacy-id=s4>4. Discussions</h2>
<p>The genome annotation would be outdated with the functional information when several years passed since the sequencing. Latest database may contain new functional genes which were not yet assigned functional information when the analyzed genome was sequenced.<sup><a reveal-id="DST009C29" data-open="DST009C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Those newly added functional information would provide the source of function transfer for some hypothetical genes in the analyzed genome. Sometimes, a few genes missed by the original annotation may be found with the similarity alignment.<sup><a reveal-id="DST009C25" data-open="DST009C25" class="link link-ref link-reveal xref-bibr">25</a></sup> In this work, a total of 2069 hypothetical genes were assigned with general or detailed functions based on homology search against latest database. Among the 10 Neisseriaceae strains, <em>N. gonorrhoeae</em> FA 1090 and <em>N. meningitidis</em> MC58 and Z2491 were sequenced and annotated earlier.<sup><a reveal-id="DST009C23" data-open="DST009C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Correspondingly, the numbers of newly assigned functions in them are the largest. Therefore, the time from the sequencing is longer and the function annotation is more likely to be outdated. As for newly found genes, all 10 strains except <em>N. meningitidis</em> alpha14 contain a small number. In fact, it is rather difficult to find genes missed by the original annotation using the similarity-based method. When annotating the genome originally, predicted genes, particularly those having similar counterparts, would be retained as far as possible. Therefore, newly identified genes usually are only those having counterparts newly added in the database. As an exception, as many as 30 genes were newly found in the strain of <em>N. meningitidis</em> alpha14 and 27 of them correspond to transposases. We speculate that the less sensitivity of the <em>ab initio</em> programme, which has been used in the original annotation, for the external genes leads to the missing of the 27 genes associated with insertion function.</p><p>Besides the published date, the quality of the original annotation is another determining factor of the extent of errors.<sup><a reveal-id="DST009C33" data-open="DST009C33" class="link link-ref link-reveal xref-bibr">33</a></sup> As an example, <em>N. gonorrhoeae</em> NCCP11945 has the highest gene density according to the RefSeq annotations among the seven <em>Neisseria</em> strains. Correspondingly, the largest number of hypothetical genes has been excluded from this strain. When annotating this genome, only one <em>ab initio</em> gene finder was used.<sup><a reveal-id="DST009C42" data-open="DST009C42" class="link link-ref link-reveal xref-bibr">42</a></sup> Usually, two or more <em>ab initio</em> programmes are used in annotating the other bacterial genomes. Based on the very abnormal gene density of the strain, we suppose that there may still be non-coding ORFs, besides the 56 ones excluded by us. Besides the annotation method and sequencing time, some other factors also cause the different extent of errors. For example, genes in high G + C bacterial genomes are shown to be difficultly predicted with high accuracy and this is caused by the fact that many long ORFs appear in this type of genomes.<sup><a reveal-id="DST009C32" data-open="DST009C32" class="link link-ref link-reveal xref-bibr">32</a></sup> In addition, the genome could be difficultly predicted accurately if there are only distantly related species in the public databases.<sup><a reveal-id="DST009C33" data-open="DST009C33" class="link link-ref link-reveal xref-bibr">33</a></sup></p><p>We should note that there still probably are non-coding ORFs and missed genes after the present updated annotation in the 10 Neisseriaceae strains. To assure reliable results, we picked up only those sequences with very higher similarities when identifying missed genes and we used the method with lower specificity when excluding non-coding ORFs. In addition, some thoroughly new genes, which have not significant similarities with any known genes, may be found by combing <em>ab initio</em> programme and wet experimental validation. However, the 106 newly found genes are those having similar counterparts based on latest databases. Also note that different detailed methods may be used to check the annotations of the other bacterial genomes, although the present systematic method has shown to be effective and reliable. For example, the recently developed <em>ab initio</em> gene finder Prodigal,<sup><a reveal-id="DST009C32" data-open="DST009C32" class="link link-ref link-reveal xref-bibr">32</a></sup> which has lower false positives could replace or jointly used with ZCURVE 1.0. Here, we used only the latter because the use of both of them generates basically consistent results (data not shown). The RPGM program,<sup><a reveal-id="DST009C39" data-open="DST009C39" class="link link-ref link-reveal xref-bibr">39</a></sup> which is also based on the graphical representation of the DNA sequence, could be chosen as an alternative to eliminate non-coding ORFs.</p>                            <h2 scrollto-destination=5359984 id="5359984" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at</a><a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5359986 id="5359986" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by the Hong Kong Scholars Program (XJ2011005), National Natural Science Foundation of China (31071109) and the programme for New Century Excellent Talents in University (NCET-11-0059).</p>                            <h2 scrollto-destination=5359988 id="5359988" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The authors are grateful to the editor and the anonymous reviewers for their valuable suggestions and comments, which has led to the improvement of this article. We thank Mr Zhong-Shan Cheng for helpful discussions. We also thank Miss Annette Y. P. Wong for helps in constructing the website.</p><p><em>Conflict of Interest statement</em>. None declared.</p>                            <h2 scrollto-destination=5359991 id="5359991" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DST009C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ouzounis</div> <div class="given-names">C.A.</div></div>,  <div class="name"><div class="surname">Karp</div> <div class="given-names">P.D.</div></div>. </div><div class="article-title">The past, present and future of genome-wide re-annotation</div>, <div class="source">Genome Biol.</div> , <div class="year">2002</div>, vol. <div class="volume">3</div> pg. <div class="fpage">COMMENT2001</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20past%2C%20present%20and%20future%20of%20genome-wide%20re-annotation&amp;author=C.A.%20Ouzounis&amp;author=P.D.%20Karp&amp;publication_year=2002&amp;journal=Genome%20Biol.&amp;volume=3&amp;pages=COMMENT2001" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2002-3-2-comment2001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2002-3-2-comment2001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2002-3-2-comment2001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11864365" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">van den Berg</div> <div class="given-names">B.H.</div></div>,  <div class="name"><div class="surname">McCarthy</div> <div class="given-names">F.M.</div></div>,  <div class="name"><div class="surname">Lamont</div> <div class="given-names">S.J.</div></div>,  <div class="name"><div class="surname">Burgess</div> <div class="given-names">S.C.</div></div>. </div><div class="article-title">Re-annotation is an essential step in systems biology modeling of functional genomics data</div>, <div class="source">PLoS One</div> , <div class="year">2010</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e10642</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Re-annotation%20is%20an%20essential%20step%20in%20systems%20biology%20modeling%20of%20functional%20genomics%20data&amp;author=B.H.%20van%20den%20Berg&amp;author=F.M.%20McCarthy&amp;author=S.J.%20Lamont&amp;author=S.C.%20Burgess&amp;publication_year=2010&amp;journal=PLoS%20One&amp;volume=5&amp;pages=e10642" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0010642" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0010642" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0010642"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20498845" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">C.T.</div></div>. </div><div class="article-title">Identification of protein-coding genes in the genome of <em>Vibrio cholerae</em> with more than 98% accuracy using occurrence frequencies of single nucleotides</div>, <div class="source">Eur. J. Biochem.</div> , <div class="year">2001</div>, vol. <div class="volume">268</div> (pg. <div class="fpage">4261</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20protein-coding%20genes%20in%20the%20genome%20of%20Vibrio%20cholerae%20with%20more%20than%2098%25%20accuracy%20using%20occurrence%20frequencies%20of%20single%20nucleotides&amp;author=J.%20Wang&amp;author=C.T.%20Zhang&amp;publication_year=2001&amp;journal=Eur.%20J.%20Biochem.&amp;volume=268&amp;pages=4261-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1432-1327.2001.02341.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1432-1327.2001.02341.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1432-1327.2001.02341.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11488920" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kawarabayasi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Hino</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Horikawa</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Complete genome sequence of an aerobic hyper-thermophilic crenarchaeon, <em>Aeropyrum pernix</em> K1</div>, <div class="source">DNA Res.</div> , <div class="year">1999</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">83</div>-<div class="lpage">101</div>) <div class="comment">, 145–52</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20an%20aerobic%20hyper-thermophilic%20crenarchaeon%2C%20Aeropyrum%20pernix%20K1&amp;author=Y.%20Kawarabayasi&amp;author=Y.%20Hino&amp;author=H.%20Horikawa&amp;publication_year=1999&amp;journal=DNA%20Res.&amp;volume=6&amp;pages=83-101" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/6.2.83" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F6.2.83" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F6.2.83"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10382966" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Natale</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Shankavaram</div> <div class="given-names">U.T.</div></div>,  <div class="name"><div class="surname">Galperin</div> <div class="given-names">M.Y.</div></div>,  <div class="name"><div class="surname">Wolf</div> <div class="given-names">Y.I.</div></div>,  <div class="name"><div class="surname">Aravind</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Koonin</div> <div class="given-names">E.V.</div></div>. </div><div class="article-title">Towards understanding the first genome sequence of a crenarchaeon by genome annotation using clusters of orthologous groups of proteins (COGs)</div>, <div class="source">Genome Biol.</div> , <div class="year">2000</div>, vol. <div class="volume">1</div> pg. <div class="fpage">RESEARCH0009</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Towards%20understanding%20the%20first%20genome%20sequence%20of%20a%20crenarchaeon%20by%20genome%20annotation%20using%20clusters%20of%20orthologous%20groups%20of%20proteins%20(COGs)&amp;author=D.A.%20Natale&amp;author=U.T.%20Shankavaram&amp;author=M.Y.%20Galperin&amp;author=Y.I.%20Wolf&amp;author=L.%20Aravind&amp;author=E.V.%20Koonin&amp;publication_year=2000&amp;journal=Genome%20Biol.&amp;volume=1&amp;pages=RESEARCH0009" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2000-1-5-research0009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2000-1-5-research0009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2000-1-5-research0009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11178258" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bocs</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Danchin</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Medigue</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Re-annotation of genome microbial coding-sequences: finding new genes and inaccurately annotated genes</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2002</div>, vol. <div class="volume">3</div> pg. <div class="fpage">5</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Re-annotation%20of%20genome%20microbial%20coding-sequences%3A%20finding%20new%20genes%20and%20inaccurately%20annotated%20genes&amp;author=S.%20Bocs&amp;author=A.%20Danchin&amp;author=C.%20Medigue&amp;publication_year=2002&amp;journal=BMC%20Bioinformatics&amp;volume=3&amp;pages=5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-3-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-3-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-3-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11879526" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guo</div> <div class="given-names">F.B.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">C.T.</div></div>. </div><div class="article-title">Gene recognition based on nucleotide distribution of ORFs in a hyper-thermophilic crenarchaeon, <em>Aeropyrum pernix</em> K1</div>, <div class="source">DNA Res.</div> , <div class="year">2004</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">361</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20recognition%20based%20on%20nucleotide%20distribution%20of%20ORFs%20in%20a%20hyper-thermophilic%20crenarchaeon%2C%20Aeropyrum%20pernix%20K1&amp;author=F.B.%20Guo&amp;author=J.%20Wang&amp;author=C.T.%20Zhang&amp;publication_year=2004&amp;journal=DNA%20Res.&amp;volume=11&amp;pages=361-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/11.6.361" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F11.6.361" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F11.6.361"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15871459" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamazaki</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Yamazaki</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Nishijima</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Proteome analysis of an aerobic hyperthermophilic crenarchaeon, <em>Aeropyrum pernix</em> K1</div>, <div class="source">Mol. Cell. Proteomics</div> , <div class="year">2006</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">811</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Proteome%20analysis%20of%20an%20aerobic%20hyperthermophilic%20crenarchaeon%2C%20Aeropyrum%20pernix%20K1&amp;author=S.%20Yamazaki&amp;author=J.%20Yamazaki&amp;author=K.%20Nishijima&amp;publication_year=2006&amp;journal=Mol.%20Cell.%20Proteomics&amp;volume=5&amp;pages=811-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/mcp.M500312-MCP200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fmcp.M500312-MCP200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fmcp.M500312-MCP200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16455681" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guo</div> <div class="given-names">F.B.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Identify protein-coding genes in the genomes of <em>Aeropyrum pernix</em> K1 and <em>Chlorobium tepidum</em> TLS</div>, <div class="source">J. Biomol. Struct. Dyn.</div> , <div class="year">2009</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">413</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identify%20protein-coding%20genes%20in%20the%20genomes%20of%20Aeropyrum%20pernix%20K1%20and%20Chlorobium%20tepidum%20TLS&amp;author=F.B.%20Guo&amp;author=Y.%20Lin&amp;publication_year=2009&amp;journal=J.%20Biomol.%20Struct.%20Dyn.&amp;volume=26&amp;pages=413-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/07391102.2009.10507256" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F07391102.2009.10507256" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F07391102.2009.10507256"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19108580" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yu</div> <div class="given-names">J.F.</div></div>,  <div class="name"><div class="surname">Jiang</div> <div class="given-names">D.K.</div></div>,  <div class="name"><div class="surname">Xiao</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Jin</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.H.</div></div>,  <div class="name"><div class="surname">Sun</div> <div class="given-names">X.</div></div>. </div><div class="article-title">Discriminate the falsely predicted protein-coding genes in <em>Aeropyrum Pernix</em> K1 genome based on graphical representation</div>, <div class="source">MATCH Commun. Math. Comput. Chem.</div> , <div class="year">2012</div>, vol. <div class="volume">67</div> (pg. <div class="fpage">845</div>-<div class="lpage">66</div>)</div></div></div></div><div content-id="DST009C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bawden</div> <div class="given-names">A.L.</div></div>,  <div class="name"><div class="surname">Glassberg</div> <div class="given-names">K.J.</div></div>,  <div class="name"><div class="surname">Diggans</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Shaw</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Farmerie</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Moyer</div> <div class="given-names">R.W.</div></div>. </div><div class="article-title">Complete genomic sequence of the <em>Amsacta moorei entomopoxvirus</em>: analysis and comparison with other poxviruses</div>, <div class="source">Virology</div> , <div class="year">2000</div>, vol. <div class="volume">274</div> (pg. <div class="fpage">120</div>-<div class="lpage">39</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genomic%20sequence%20of%20the%20Amsacta%20moorei%20entomopoxvirus%3A%20analysis%20and%20comparison%20with%20other%20poxviruses&amp;author=A.L.%20Bawden&amp;author=K.J.%20Glassberg&amp;author=J.%20Diggans&amp;author=R.%20Shaw&amp;author=W.%20Farmerie&amp;author=R.W.%20Moyer&amp;publication_year=2000&amp;journal=Virology&amp;volume=274&amp;pages=120-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/viro.2000.0449" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fviro.2000.0449" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fviro.2000.0449"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10936094" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guo</div> <div class="given-names">F.B.</div></div>,  <div class="name"><div class="surname">Yu</div> <div class="given-names">X.J.</div></div>. </div><div class="article-title">Re-prediction of protein-coding genes in the genome of <em>Amsacta moorei entomopoxvirus</em></div>, <div class="source">J. Virol. Methods</div> , <div class="year">2007</div>, vol. <div class="volume">146</div> (pg. <div class="fpage">389</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Re-prediction%20of%20protein-coding%20genes%20in%20the%20genome%20of%20Amsacta%20moorei%20entomopoxvirus&amp;author=F.B.%20Guo&amp;author=X.J.%20Yu&amp;publication_year=2007&amp;journal=J.%20Virol.%20Methods&amp;volume=146&amp;pages=389-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jviromet.2007.07.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jviromet.2007.07.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jviromet.2007.07.010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17716751" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yu</div> <div class="given-names">J.F.</div></div>,  <div class="name"><div class="surname">Sun</div> <div class="given-names">X.</div></div>. </div><div class="article-title">Reannotation of protein-coding genes based on an improved graphical representation of DNA sequence</div>, <div class="source">J. Comput. Chem.</div> , <div class="year">2010</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">2126</div>-<div class="lpage">35</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reannotation%20of%20protein-coding%20genes%20based%20on%20an%20improved%20graphical%20representation%20of%20DNA%20sequence&amp;author=J.F.%20Yu&amp;author=X.%20Sun&amp;publication_year=2010&amp;journal=J.%20Comput.%20Chem.&amp;volume=31&amp;pages=2126-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/jcc.v31:11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fjcc.v31:11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fjcc.v31:11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20175214" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhou</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Vorhölter</div> <div class="given-names">F.J.</div></div>,  <div class="name"><div class="surname">He</div> <div class="given-names">Y.Q.</div></div>, et al. </div><div class="article-title">Gene discovery by genome-wide CDS re-prediction and microarray-based transcriptional analysis in phytopathogen <em>Xanthomonas campestris</em></div>, <div class="source">BMC Genomics</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">359</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20discovery%20by%20genome-wide%20CDS%20re-prediction%20and%20microarray-based%20transcriptional%20analysis%20in%20phytopathogen%20Xanthomonas%20campestris&amp;author=L.%20Zhou&amp;author=F.J.%20Vorh%C3%B6lter&amp;author=Y.Q.%20He&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=359" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-359" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-359" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-359"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21745409" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Du</div> <div class="given-names">M.Z.</div></div>,  <div class="name"><div class="surname">Guo</div> <div class="given-names">F.B.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">Y.Y.</div></div>. </div><div class="article-title">Gene re-annotation in genome of the extremophile <em>Pyrobaculum aerophilum</em> by using bioinformatics methods</div>, <div class="source">J. Biomol. Struct. Dyn.</div> , <div class="year">2011</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">391</div>-<div class="lpage">401</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20re-annotation%20in%20genome%20of%20the%20extremophile%20Pyrobaculum%20aerophilum%20by%20using%20bioinformatics%20methods&amp;author=M.Z.%20Du&amp;author=F.B.%20Guo&amp;author=Y.Y.%20Chen&amp;publication_year=2011&amp;journal=J.%20Biomol.%20Struct.%20Dyn.&amp;volume=29&amp;pages=391-401" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/07391102.2011.10507393" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F07391102.2011.10507393" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F07391102.2011.10507393"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21875157" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">L.L.</div></div>,  <div class="name"><div class="surname">Ma</div> <div class="given-names">B.G.</div></div>,  <div class="name"><div class="surname">Gao</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Reannotation of hypothetical ORFs in plant pathogen <em>Erwinia carotovora</em> subsp. <em>atroseptica</em> SCRI1043</div>, <div class="source">FEBS J.</div> , <div class="year">2008</div>, vol. <div class="volume">275</div> (pg. <div class="fpage">198</div>-<div class="lpage">206</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reannotation%20of%20hypothetical%20ORFs%20in%20plant%20pathogen%20Erwinia%20carotovora%20subsp.%20atroseptica%20SCRI1043&amp;author=L.L.%20Chen&amp;author=B.G.%20Ma&amp;author=N.%20Gao&amp;publication_year=2008&amp;journal=FEBS%20J.&amp;volume=275&amp;pages=198-206" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1742-4658.2007.06190.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1742-4658.2007.06190.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1742-4658.2007.06190.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18067578" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Okamoto</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Yamada</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Proteome driven re-evaluation and functional annotation of the <em>Streptococcus pyogenes</em> SF370 genome</div>, <div class="source">BMC Microbiol.</div> , <div class="year">2011</div>, vol. <div class="volume">11</div> pg. <div class="fpage">249</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Proteome%20driven%20re-evaluation%20and%20functional%20annotation%20of%20the%20Streptococcus%20pyogenes%20SF370%20genome&amp;author=A.%20Okamoto&amp;author=K.%20Yamada&amp;publication_year=2011&amp;journal=BMC%20Microbiol.&amp;volume=11&amp;pages=249" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2180-11-249" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2180-11-249" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2180-11-249"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22070424" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tettelin</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Saunders</div> <div class="given-names">N.J.</div></div>,  <div class="name"><div class="surname">Heidelberg</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Complete genome sequence of <em>Neisseria meningitidis</em> serogroup B strain MC58</div>, <div class="source">Science</div> , <div class="year">2000</div>, vol. <div class="volume">287</div> (pg. <div class="fpage">1809</div>-<div class="lpage">15</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20Neisseria%20meningitidis%20serogroup%20B%20strain%20MC58&amp;author=H.%20Tettelin&amp;author=N.J.%20Saunders&amp;author=J.%20Heidelberg&amp;publication_year=2000&amp;journal=Science&amp;volume=287&amp;pages=1809-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.287.5459.1809" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.287.5459.1809" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.287.5459.1809"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10710307" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yuen</div> <div class="given-names">K.Y.</div></div>,  <div class="name"><div class="surname">Woo</div> <div class="given-names">P.C.</div></div>,  <div class="name"><div class="surname">Teng</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Leung</div> <div class="given-names">K.W.</div></div>,  <div class="name"><div class="surname">Wong</div> <div class="given-names">M.K.</div></div>,  <div class="name"><div class="surname">Lau</div> <div class="given-names">S.K.</div></div>. </div><div class="article-title"><em>Laribacter hongkongensis</em> gen. nov., sp. nov., a novel Gram-negative bacterium isolated from a cirrhotic patient with bacteremia and empyema</div>, <div class="source">J. Clin. Microbiol.</div> , <div class="year">2001</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">4227</div>-<div class="lpage">32</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Laribacter%20hongkongensis%20gen.%20nov.%2C%20sp.%20nov.%2C%20a%20novel%20Gram-negative%20bacterium%20isolated%20from%20a%20cirrhotic%20patient%20with%20bacteremia%20and%20empyema&amp;author=K.Y.%20Yuen&amp;author=P.C.%20Woo&amp;author=J.L.%20Teng&amp;author=K.W.%20Leung&amp;author=M.K.%20Wong&amp;author=S.K.%20Lau&amp;publication_year=2001&amp;journal=J.%20Clin.%20Microbiol.&amp;volume=39&amp;pages=4227-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JCM.39.12.4227-4232.2001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJCM.39.12.4227-4232.2001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJCM.39.12.4227-4232.2001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11724825" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Woo</div> <div class="given-names">P.C.</div></div>,  <div class="name"><div class="surname">Lau</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Teng</div> <div class="given-names">J.L.</div></div>, et al. </div><div class="article-title">Association of <em>Laribacter hongkongensis</em> in community-acquired gastroenteritis with travel and eating fish: a multicentre case-control study</div>, <div class="source">Lancet</div> , <div class="year">2004</div>, vol. <div class="volume">363</div> (pg. <div class="fpage">1941</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Association%20of%20Laribacter%20hongkongensis%20in%20community-acquired%20gastroenteritis%20with%20travel%20and%20eating%20fish%3A%20a%20multicentre%20case-control%20study&amp;author=P.C.%20Woo&amp;author=S.K.%20Lau&amp;author=J.L.%20Teng&amp;publication_year=2004&amp;journal=Lancet&amp;volume=363&amp;pages=1941-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0140-6736(04)16407-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0140-6736(04)16407-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0140-6736(04)16407-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15194253" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Woo</div> <div class="given-names">P.C.</div></div>,  <div class="name"><div class="surname">Lau</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Tse</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">The complete genome and proteome of <em>Laribacter hongkongensis</em> reveal potential mechanisms for adaptations to different temperatures and habitats</div>, <div class="source">PLoS Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">25</div> pg. <div class="fpage">e1000416</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complete%20genome%20and%20proteome%20of%20Laribacter%20hongkongensis%20reveal%20potential%20mechanisms%20for%20adaptations%20to%20different%20temperatures%20and%20habitats&amp;author=P.C.%20Woo&amp;author=S.K.%20Lau&amp;author=H.%20Tse&amp;publication_year=2009&amp;journal=PLoS%20Genet.&amp;volume=25&amp;pages=e1000416" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000416" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000416" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000416"> </span></div> </div></div></div></div></div><div content-id="DST009C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kim</div> <div class="given-names">D.S.</div></div>,  <div class="name"><div class="surname">Wi</div> <div class="given-names">Y.M.</div></div>,  <div class="name"><div class="surname">Choi</div> <div class="given-names">J.Y.</div></div>,  <div class="name"><div class="surname">Peck</div> <div class="given-names">K.R.</div></div>,  <div class="name"><div class="surname">Song</div> <div class="given-names">J.H.</div></div>,  <div class="name"><div class="surname">Ko</div> <div class="given-names">K.S.</div></div>. </div><div class="article-title">Bacteremia caused by <em>Laribacter hongkongensis</em> misidentified as <em>Acinetobacter lwoffii</em>: report of the first case in Korea</div>, <div class="source">J. Korean Med. Sci.</div> , <div class="year">2011</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">679</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacteremia%20caused%20by%20Laribacter%20hongkongensis%20misidentified%20as%20Acinetobacter%20lwoffii%3A%20report%20of%20the%20first%20case%20in%20Korea&amp;author=D.S.%20Kim&amp;author=Y.M.%20Wi&amp;author=J.Y.%20Choi&amp;author=K.R.%20Peck&amp;author=J.H.%20Song&amp;author=K.S.%20Ko&amp;publication_year=2011&amp;journal=J.%20Korean%20Med.%20Sci.&amp;volume=26&amp;pages=679-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3346/jkms.2011.26.5.679" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3346%2Fjkms.2011.26.5.679" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3346%2Fjkms.2011.26.5.679"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21532861" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pagani</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Liolios</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Jansson</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">The Genomes OnLine Database (GOLD) v.4: status of genomic and metagenomic projects and their associated metadata</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2012</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">D571</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Genomes%20OnLine%20Database%20(GOLD)%20v.4%3A%20status%20of%20genomic%20and%20metagenomic%20projects%20and%20their%20associated%20metadata&amp;author=I.%20Pagani&amp;author=K.%20Liolios&amp;author=J.%20Jansson&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=D571-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr1100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr1100"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22135293" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pruitt</div> <div class="given-names">K.D.</div></div>,  <div class="name"><div class="surname">Tatusova</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Maglott</div> <div class="given-names">D.R.</div></div>. </div><div class="article-title">NCBI reference sequences (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">D61</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NCBI%20reference%20sequences%20(RefSeq)%3A%20a%20curated%20non-redundant%20sequence%20database%20of%20genomes%2C%20transcripts%20and%20proteins&amp;author=K.D.%20Pruitt&amp;author=T.%20Tatusova&amp;author=D.R.%20Maglott&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=D61-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl842" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl842" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl842"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17130148" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Warren</div> <div class="given-names">A.S.</div></div>,  <div class="name"><div class="surname">Archuleta</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Feng</div> <div class="given-names">W.C.</div></div>,  <div class="name"><div class="surname">Setubal</div> <div class="given-names">J.C.</div></div>. </div><div class="article-title">Missing genes in the annotation of prokaryotic genomes</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">131</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Missing%20genes%20in%20the%20annotation%20of%20prokaryotic%20genomes&amp;author=A.S.%20Warren&amp;author=J.%20Archuleta&amp;author=W.C.%20Feng&amp;author=J.C.%20Setubal&amp;publication_year=2010&amp;journal=BMC%20Bioinformatics&amp;volume=11&amp;pages=131" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-11-131" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-11-131" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-11-131"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20230630" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guo</div> <div class="given-names">F.B.</div></div>,  <div class="name"><div class="surname">Ou</div> <div class="given-names">H.Y.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">C.T.</div></div>. </div><div class="article-title">ZCURVE: a new system for recognizing protein-coding genes in bacterial and archaeal genomes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">1780</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ZCURVE%3A%20a%20new%20system%20for%20recognizing%20protein-coding%20genes%20in%20bacterial%20and%20archaeal%20genomes&amp;author=F.B.%20Guo&amp;author=H.Y.%20Ou&amp;author=C.T.%20Zhang&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=1780-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg254" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg254" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg254"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12626720" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Wootton</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Gertz</div> <div class="given-names">E.M.</div></div>,  <div class="name"><div class="surname">Agarwala</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Morgulis</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Schaffer</div> <div class="given-names">A.A.</div></div>,  <div class="name"><div class="surname">Yu</div> <div class="given-names">Y.K.</div></div>. </div><div class="article-title">Protein database searches using compositionally adjusted substitution matrices</div>, <div class="source">FEBS J.</div> , <div class="year">2005</div>, vol. <div class="volume">272</div> (pg. <div class="fpage">5101</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protein%20database%20searches%20using%20compositionally%20adjusted%20substitution%20matrices&amp;author=S.F.%20Altschul&amp;author=J.C.%20Wootton&amp;author=E.M.%20Gertz&amp;author=R.%20Agarwala&amp;author=A.%20Morgulis&amp;author=A.A.%20Schaffer&amp;author=Y.K.%20Yu&amp;publication_year=2005&amp;journal=FEBS%20J.&amp;volume=272&amp;pages=5101-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/ejb.2005.272.issue-20" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fejb.2005.272.issue-20" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fejb.2005.272.issue-20"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16218944" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">C.T.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Recognition of protein coding genes in the yeast genome at better than 95% accuracy based on the Z curve</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2000</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">2804</div>-<div class="lpage">14</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recognition%20of%20protein%20coding%20genes%20in%20the%20yeast%20genome%20at%20better%20than%2095%25%20accuracy%20based%20on%20the%20Z%20curve&amp;author=C.T.%20Zhang&amp;author=J.%20Wang&amp;publication_year=2000&amp;journal=Nucleic%20Acids%20Res.&amp;volume=28&amp;pages=2804-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/28.14.2804" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F28.14.2804" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F28.14.2804"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10908339" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gao</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">L.L.</div></div>,  <div class="name"><div class="surname">Ji</div> <div class="given-names">H.F.</div></div>, et al. </div><div class="article-title">DIGA–a database of improved gene annotation for phytopathogens</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">54</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DIGA%E2%80%93a%20database%20of%20improved%20gene%20annotation%20for%20phytopathogens&amp;author=N.%20Gao&amp;author=L.L.%20Chen&amp;author=H.F.%20Ji&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-54" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-54" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-54"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20089203" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gao</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">C.T.</div></div>. </div><div class="article-title">Comparison of various algorithms for recognizing short coding sequences of human genes</div>, <div class="source">Bioinformatics</div> , <div class="year">2004</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">673</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20various%20algorithms%20for%20recognizing%20short%20coding%20sequences%20of%20human%20genes&amp;author=F.%20Gao&amp;author=C.T.%20Zhang&amp;publication_year=2004&amp;journal=Bioinformatics&amp;volume=20&amp;pages=673-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btg467" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtg467" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtg467"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14764563" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="collab">Brazilian National Genome Project Consortium</div><div class="article-title">The complete genome sequence of *<em>Chromobacterium violaceum</em> reveals remarkable and exploitable bacterial adaptability</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2003</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">11660</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1832124100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1832124100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1832124100"> </span></div> </div></div></div></div></div><div content-id="DST009C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hyatt</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">G.L.</div></div>,  <div class="name"><div class="surname">Locascio</div> <div class="given-names">P.F.</div></div>,  <div class="name"><div class="surname">Land</div> <div class="given-names">M.L.</div></div>,  <div class="name"><div class="surname">Larimer</div> <div class="given-names">F.W.</div></div>,  <div class="name"><div class="surname">Hauser</div> <div class="given-names">L.J.</div></div>. </div><div class="article-title">Prodigal: prokaryotic gene recognition and translation initiation site identification</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">119</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prodigal%3A%20prokaryotic%20gene%20recognition%20and%20translation%20initiation%20site%20identification&amp;author=D.%20Hyatt&amp;author=G.L.%20Chen&amp;author=P.F.%20Locascio&amp;author=M.L.%20Land&amp;author=F.W.%20Larimer&amp;author=L.J.%20Hauser&amp;publication_year=2010&amp;journal=BMC%20Bioinformatics&amp;volume=11&amp;pages=119" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-11-119" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-11-119" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-11-119"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20211023" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Richardson</div> <div class="given-names">E.J.</div></div>,  <div class="name"><div class="surname">Watson</div> <div class="given-names">M.</div></div>. </div><div class="article-title">The automatic annotation of bacterial genomes</div>, <div class="source">Brief Bioinform</div> , <div class="year">2013</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">1</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20automatic%20annotation%20of%20bacterial%20genomes&amp;author=E.J.%20Richardson&amp;author=M.%20Watson&amp;publication_year=2013&amp;journal=Brief%20Bioinform&amp;volume=14&amp;pages=1-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bib/bbs007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbib%2Fbbs007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbib%2Fbbs007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22408191" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div>. </div><div class="article-title">Genome re-annotation: a wiki solution?</div>, <div class="source">Genome Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">102</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20re-annotation%3A%20a%20wiki%20solution%3F&amp;author=S.L.%20Salzberg&amp;publication_year=2007&amp;journal=Genome%20Biol.&amp;volume=8&amp;pages=102" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2007-8-6-r102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2007-8-6-r102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2007-8-6-r102"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17274839" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">C.T.</div></div>,  <div class="name"><div class="surname">Chou</div> <div class="given-names">K.C.</div></div>. </div><div class="article-title">A graphic approach to analyzing codon usage in 1562 <em>Escherichia coli</em> protein coding sequences</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1994</div>, vol. <div class="volume">238</div> (pg. <div class="fpage">1</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20graphic%20approach%20to%20analyzing%20codon%20usage%20in%201562%20Escherichia%20coli%20protein%20coding%20sequences&amp;author=C.T.%20Zhang&amp;author=K.C.%20Chou&amp;publication_year=1994&amp;journal=J.%20Mol.%20Biol.&amp;volume=238&amp;pages=1-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.1994.1263" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.1994.1263" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.1994.1263"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8145249" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">L.L.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">C.T.</div></div>. </div><div class="article-title">Seven GC-rich microbial genomes adopt similar codon usage patterns regardless of their phylogenetic lineages</div>, <div class="source">Biochem. Biophys. Res. Commun.</div> , <div class="year">2003</div>, vol. <div class="volume">306</div> (pg. <div class="fpage">310</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Seven%20GC-rich%20microbial%20genomes%20adopt%20similar%20codon%20usage%20patterns%20regardless%20of%20their%20phylogenetic%20lineages&amp;author=L.L.%20Chen&amp;author=C.T.%20Zhang&amp;publication_year=2003&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=306&amp;pages=310-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0006-291X(03)00973-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0006-291X(03)00973-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0006-291X(03)00973-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12788106" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guo</div> <div class="given-names">F.B.</div></div>. </div><div class="article-title">The distribution patterns of bases of protein-coding genes, non-coding ORFs, and intergenic sequences in <em>Pseudomonas aeruginosa</em> PA01 genome and its implications</div>, <div class="source">J. Biomol. Struct. Dyn.</div> , <div class="year">2007</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">127</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20distribution%20patterns%20of%20bases%20of%20protein-coding%20genes%2C%20non-coding%20ORFs%2C%20and%20intergenic%20sequences%20in%20Pseudomonas%20aeruginosa%20PA01%20genome%20and%20its%20implications&amp;author=F.B.%20Guo&amp;publication_year=2007&amp;journal=J.%20Biomol.%20Struct.%20Dyn.&amp;volume=25&amp;pages=127-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/07391102.2007.10507161" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F07391102.2007.10507161" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F07391102.2007.10507161"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17718591" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Natale</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Galperin</div> <div class="given-names">M.Y.</div></div>,  <div class="name"><div class="surname">Tatusovm</div> <div class="given-names">R.L.</div></div>,  <div class="name"><div class="surname">Koonin</div> <div class="given-names">E.V.</div></div>. </div><div class="article-title">Using the COG database to improve gene recognition in complete genomes</div>, <div class="source">Genetica</div> , <div class="year">2000</div>, vol. <div class="volume">108</div> (pg. <div class="fpage">9</div>-<div class="lpage">17</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20the%20COG%20database%20to%20improve%20gene%20recognition%20in%20complete%20genomes&amp;author=D.A.%20Natale&amp;author=M.Y.%20Galperin&amp;author=R.L.%20Tatusovm&amp;author=E.V.%20Koonin&amp;publication_year=2000&amp;journal=Genetica&amp;volume=108&amp;pages=9-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1004031323748" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1004031323748" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1004031323748"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11145426" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yu</div> <div class="given-names">J.F.</div></div>,  <div class="name"><div class="surname">Xiao</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Jiang</div> <div class="given-names">D.K.</div></div>,  <div class="name"><div class="surname">Guo</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.H.</div></div>,  <div class="name"><div class="surname">Sun</div> <div class="given-names">X.</div></div>. </div><div class="article-title">An integrative method for identifying the over-annotated protein-coding genes in microbial genomes</div>, <div class="source">DNA Res.</div> , <div class="year">2011</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">435</div>-<div class="lpage">49</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20integrative%20method%20for%20identifying%20the%20over-annotated%20protein-coding%20genes%20in%20microbial%20genomes&amp;author=J.F.%20Yu&amp;author=K.%20Xiao&amp;author=D.K.%20Jiang&amp;author=J.%20Guo&amp;author=J.H.%20Wang&amp;author=X.%20Sun&amp;publication_year=2011&amp;journal=DNA%20Res.&amp;volume=18&amp;pages=435-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsr030" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsr030" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsr030"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21903723" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Luo</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Hu</div> <div class="given-names">G.Q.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Genome reannotation of <em>Escherichia coli</em> CFT073 with new insights into virulence</div>, <div class="source">BMC Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">552</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20reannotation%20of%20Escherichia%20coli%20CFT073%20with%20new%20insights%20into%20virulence&amp;author=C.%20Luo&amp;author=G.Q.%20Hu&amp;author=H.%20Zhu&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=552" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-552" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-552" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-552"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19930606" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hayes</div> <div class="given-names">W.S.</div></div>,  <div class="name"><div class="surname">Borodovsky</div> <div class="given-names">M.</div></div>. </div><div class="article-title">How to interpret an anonymous bacterial genome: machine learning approach to gene identification</div>, <div class="source">Genome Res.</div> , <div class="year">1998</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">1154</div>-<div class="lpage">71</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=How%20to%20interpret%20an%20anonymous%20bacterial%20genome%3A%20machine%20learning%20approach%20to%20gene%20identification&amp;author=W.S.%20Hayes&amp;author=M.%20Borodovsky&amp;publication_year=1998&amp;journal=Genome%20Res.&amp;volume=8&amp;pages=1154-71" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9847079" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chung</div> <div class="given-names">G.T.</div></div>,  <div class="name"><div class="surname">Yoo</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">Oh</div> <div class="given-names">H.B.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">Y.S.</div></div>,  <div class="name"><div class="surname">Cha</div> <div class="given-names">S.H.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">S.J.</div></div>,  <div class="name"><div class="surname">Yoo</div> <div class="given-names">C.K.</div></div>. </div><div class="article-title">Complete genome sequence of <em>Neiserria gonorrhoeae</em> NCCP11945</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2008</div>, vol. <div class="volume">190</div> (pg. <div class="fpage">6035</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20Neiserria%20gonorrhoeae%20NCCP11945&amp;author=G.T.%20Chung&amp;author=J.S.%20Yoo&amp;author=H.B.%20Oh&amp;author=Y.S.%20Lee&amp;author=S.H.%20Cha&amp;author=S.J.%20Kim&amp;author=C.K.%20Yoo&amp;publication_year=2008&amp;journal=J.%20Bacteriol.&amp;volume=190&amp;pages=6035-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.00566-08" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.00566-08" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.00566-08"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18586945" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Perrodou</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Deshayes</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Muller</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">ICDS database: interrupted CoDing sequences in prokaryotic genomes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">D338</div>-<div class="lpage">43</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ICDS%20database%3A%20interrupted%20CoDing%20sequences%20in%20prokaryotic%20genomes&amp;author=E.%20Perrodou&amp;author=C.%20Deshayes&amp;author=J.%20Muller&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=D338-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkj060" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkj060" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkj060"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16381882" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST009C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST009C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharma</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Firth</div> <div class="given-names">A.E.</div></div>,  <div class="name"><div class="surname">Antonov</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Fayet</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Atkins</div> <div class="given-names">J.F.</div></div>,  <div class="name"><div class="surname">Borodovsky</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Baranov</div> <div class="given-names">P.V.</div></div>. </div><div class="article-title">A pilot study of bacterial genes with disrupted ORFs reveals a surprising profusion of protein sequence recoding mediated by ribosomal frameshifting and transcriptional realignment</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2011</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">3195</div>-<div class="lpage">211</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20pilot%20study%20of%20bacterial%20genes%20with%20disrupted%20ORFs%20reveals%20a%20surprising%20profusion%20of%20protein%20sequence%20recoding%20mediated%20by%20ribosomal%20frameshifting%20and%20transcriptional%20realignment&amp;author=V.%20Sharma&amp;author=A.E.%20Firth&amp;author=I.%20Antonov&amp;author=O.%20Fayet&amp;author=J.F.%20Atkins&amp;author=M.%20Borodovsky&amp;author=P.V.%20Baranov&amp;publication_year=2011&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=28&amp;pages=3195-211" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msr155" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsr155" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsr155"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21673094" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Prof. Kenta Nakai</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by-nc/3.0/), which permits non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=369179&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst009/2/dst009-dst009supp_table4.xls?Expires=2147483647&amp;Signature=HyZD9FsWTcYIcjGj19SxuCOWBr3-FjKuCvyB4-gK1ZiDECXzQTEYPwgRQ8rnGKI6Y~f1e-FZA30Cbcuykx63q21vG4OUmBHaKvqkvxNYngrMjyZPBcbWJDhT8ed7tre-gmlYf0yatVE3-m7Zj6iKMww8K3EcqYaBBCZCZTExBUA-LKEXOR4T0yQWBS91ypHE1qs~CTCpDPYtBIM5H~o0ycuduYWkQYYPbYbD4dgmre73qA3YJeEHNpDaXuZr1HztPS-5cXShMveVtWghHnBHr1TA4bFutKBYFzU3Qb8POJcW5wRbSJdqgTULXQGwugUFc7kQXhGTZPC6w4mQI~MY1Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst009/2/dst009-dst009supp_table3.xls?Expires=2147483647&amp;Signature=NeVc7CxdsnGCg3rSqxXvWVZgsYW1xlGXsoNz-DIsaEZKgYEDy2eSZpfF9HJ28IrA0Sz4xDlEtWiYZivqoRv8KdGKnvLjXu8Sdx2yCYW49RIY3Yoe09ueJG8BkU8D96QFc2KxuzJu1Q89XOkh1YEhctRqGAIi94FTDa7AT9C0dPA0O3vgK1HtKytZ3Wbt9VTz8NBY0zS0S8FIOUWcWH7VtjNOn261a6zzBnzNFSXGlAaaXUgBbw7hvyA4ywiaCBw5SfikILDXQ2VWhOD2Qy9AGjc8MtqFzRtm8VGWiug9VhvOVimWBLYvBkIOt8OvT41ElU7xoIMQFrY6rdXeNETegg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst009/2/dst009-dst009supp_table2.xls?Expires=2147483647&amp;Signature=ZtHHvQGGTezGruugd~TIrRvCaN0VKGaIZxusFx7wkPaqc~~~bdo0MvUz99hPPxweRxJIqzm~B1l2Bi5TcxLdw1CTbm77lR7F38rV3rIxq~iEHv3~TxJ3rc1WYMit~L5gxDhCUnl5NEbNf6XQfGd5WAhJREp-jRwkWmyO9OzsWvVtjsfm-YfMYLtCv1a4d2Wo0Zdg7UJC8L2Tq8fYqLoB-eiygcpuGKygFoRGelp0Hiq2dlQf9knDchgGCQEyewlYOF~mBR0E-WDRDBGpvo7ElHcCJzTfAQ4rSlL6H8tE0bPj75w3KY5Tl9TWslH732LEJDaxW7RREBD1mtgio1o1xQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst009/2/dst009-dst009supp_table1.xls?Expires=2147483647&amp;Signature=HPx-YJmoa6ri1WETf9koUqV7kWjKqauG3M1KR66FX8rOSvBanyC~8hUbkuvmoxvx3KwhUJnZLz1j4hcQft4276W1oXY752w~zsNZ8Ovoz4r1uuR0ljDhvtnZ9L0urdmEHtPpIYWUIzwL5Ct6I3baA1GD6HyAfKOl4fY94UgYDJG964jfLhm5E5M6ZQ8XvqiqbQEaL4J7I57IQqaJ6my0PoV72HAoA8nGMH~X8P6m7oGPNhtGPfrnfgi8Cbvx~JBQX0tZsnHJil0jiIujDcRg87ztSLjXehLXnJoe4HJkeqwjQlNE0kMKlGGsvtUJm0DuPdGrw-QR1ZGBN36o6xUh3Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - xls file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">247</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000321404200007&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">12</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dst009"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="369179"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000321404200007&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f20%2f3%2f273%2f369179">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000321404200007&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (12)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F20%2F3%2F273%2F369179" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/369179'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Re-Annotation of Protein-Coding Genes in 10 Complete Genomes of Neisseriaceae Family by Combining Similarity-Based and Composition-Based Methods" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dst009");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f20%2f3%2f273%2f369179">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

