

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/13/4/141/348695","@type":"ScholarlyArticle","name":" Escherichia coli  Histone-Like Protein H-NS Preferentially Binds to Horizontally Acquired DNA in Association with RNA Polymerase","datePublished":"2006-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/13/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2006-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsl009","keywords":["histone-like protein H-NS","RNA polymerase","horizontal genes","high-density tiling chip","ChIP-chip analysis"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/13/4/141/348695","author":[{"name":"Oshima, Taku","affiliation":"1  Graduate School of Information Science, Nara Institute of Science and Technology  8916-5, Takayama, Ikoma, Nara 630-0192, Japan","@type":"Person"},{"name":"Ishikawa, Shu","affiliation":"1  Graduate School of Information Science, Nara Institute of Science and Technology  8916-5, Takayama, Ikoma, Nara 630-0192, Japan","@type":"Person"},{"name":"Kurokawa, Ken","affiliation":"1  Graduate School of Information Science, Nara Institute of Science and Technology  8916-5, Takayama, Ikoma, Nara 630-0192, Japan","@type":"Person"},{"name":"Aiba, Hirofumi","affiliation":"2  Graduate School of Bioagricultural Sciences, Nagoya University  Nagoya, Aichi 464-8601, Japan","@type":"Person"},{"name":"Ogasawara, Naotake","affiliation":"1  Graduate School of Information Science, Nara Institute of Science and Technology  8916-5, Takayama, Ikoma, Nara 630-0192, Japan","@type":"Person"}],"description":"Abstract.  Heat-stable nucleoid-structuring protein (H-NS) is one of the main nucleoid proteins expressed in exponentially growing Escherichia coli cells. In ad","pageStart":"141","pageEnd":"153","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/4/0/m_dnaresearch13_4.cover.gif?Expires=2147483647&Signature=zpj2n5NzsMk30zA4qehj6yWrvGBU~IiLu4DURkTxdl~55txhlJ5z0a2IpdyGYQcF4vlVunu-~Wh08QUqWKvj48dNgA7n8vLabfgEOY0CoaiBxls7NyW6tCeXW6Sfd2X4lV4uoHFAXj3NLXFLZshkdiFCqF5MJYIi0qOwNjsIUdAwRNhaR9xqEQSv1J65s5eb8WSbVSZ7jKourjIePJi-0rEJRs9B7Kjh~xeLnZz~SmpYlxNUZP4MEUv8qvfsHNtVbSMWxc-iOv9niMhvvwUPs612TggXmBVQynK3LwlztmX3LfsmbR-PpnPA2Dhenm3q1yRIy9w5VQzSlMaqz8FNQA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":" Escherichia coli  Histone-Like Protein H-NS Preferentially Binds to Horizontally Acquired DNA in Association with RNA Polymerase","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/4/0/m_dnaresearch13_4.cover.gif?Expires=2147483647&Signature=zpj2n5NzsMk30zA4qehj6yWrvGBU~IiLu4DURkTxdl~55txhlJ5z0a2IpdyGYQcF4vlVunu-~Wh08QUqWKvj48dNgA7n8vLabfgEOY0CoaiBxls7NyW6tCeXW6Sfd2X4lV4uoHFAXj3NLXFLZshkdiFCqF5MJYIi0qOwNjsIUdAwRNhaR9xqEQSv1J65s5eb8WSbVSZ7jKourjIePJi-0rEJRs9B7Kjh~xeLnZz~SmpYlxNUZP4MEUv8qvfsHNtVbSMWxc-iOv9niMhvvwUPs612TggXmBVQynK3LwlztmX3LfsmbR-PpnPA2Dhenm3q1yRIy9w5VQzSlMaqz8FNQA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content=" Escherichia coli  Histone-Like Protein H-NS Preferentially Binds to Horizontally Acquired DNA in Association with RNA Polymerase" />
    <meta property="og:description" content="Abstract.  Heat-stable nucleoid-structuring protein (H-NS) is one of the main nucleoid proteins expressed in exponentially growing Escherichia coli cells. In ad" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/13/4/141/348695" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2006-01-01" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Oshima, Taku" /><meta name="citation_author_institution" content="1Graduate School of Information Science, Nara Institute of Science and Technology8916-5, Takayama, Ikoma, Nara 630-0192, Japan" /><meta name="citation_author" content="Ishikawa, Shu" /><meta name="citation_author_institution" content="1Graduate School of Information Science, Nara Institute of Science and Technology8916-5, Takayama, Ikoma, Nara 630-0192, Japan" /><meta name="citation_author" content="Kurokawa, Ken" /><meta name="citation_author_institution" content="1Graduate School of Information Science, Nara Institute of Science and Technology8916-5, Takayama, Ikoma, Nara 630-0192, Japan" /><meta name="citation_author" content="Aiba, Hirofumi" /><meta name="citation_author_institution" content="2Graduate School of Bioagricultural Sciences, Nagoya UniversityNagoya, Aichi 464-8601, Japan" /><meta name="citation_author" content="Ogasawara, Naotake" /><meta name="citation_author_institution" content="1Graduate School of Information Science, Nara Institute of Science and Technology8916-5, Takayama, Ikoma, Nara 630-0192, Japan" /><meta name="citation_title" content="Escherichia coli Histone-Like Protein H-NS Preferentially Binds to Horizontally Acquired DNA in Association with RNA Polymerase" /><meta name="citation_firstpage" content="141" /><meta name="citation_lastpage" content="153" /><meta name="citation_doi" content="10.1093/dnares/dsl009" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="13" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2006/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Cooperative transitions of isolated Escherichia coli nucleoids: implications for the nucleoid as a cellular phase; J. Struct. Biol.;  citation_year=2006;  citation_volume=153;  citation_pages=160-175; " /><meta name="citation_reference" content="citation_title=DNA bridging: a property shared among H-NS-like proteins; J. Bacteriol.;  citation_year=2005;  citation_volume=187;  citation_pages=1845-1848; " /><meta name="citation_reference" content="citation_title=H-NS: a universal regulator for a dynamic genome; Nat. Rev. Microbiol.;  citation_year=2004;  citation_volume=2;  citation_pages=391-400; " /><meta name="citation_reference" content="citation_title=Structure of the histone-like protein H-NS and its role in regulation and genome superstructure; Curr. Opin. Microbiol.;  citation_year=2004;  citation_volume=7;  citation_pages=109-114; " /><meta name="citation_reference" content="citation_title=A molecular mechanism for the repression of transcription by the H-NS protein; Mol. Microbiol.;  citation_year=2001;  citation_volume=42;  citation_pages=1311-1323; " /><meta name="citation_reference" content="citation_title=Structural basis for H-NS-mediated trapping of RNA polymerase in the open initiation complex at the rrnB P1; J. Biol. Chem.;  citation_year=2002;  citation_volume=277;  citation_pages=2146-2150; " /><meta name="citation_reference" content="citation_title=Lethal overproduction of the Escherichia coli nucleoid protein H-NS: ultramicroscopic and molecular autopsy; Mol. Gen. Genet.;  citation_year=1992;  citation_volume=231;  citation_pages=201-211; " /><meta name="citation_reference" content="citation_title=Large-scale monitoring of pleiotropic regulation of gene expression by the prokaryotic nucleoid-associated protein, H-NS; Mol. Microbiol.;  citation_year=2001;  citation_volume=40;  citation_pages=20-36; " /><meta name="citation_reference" content="citation_title=The bacterial DNA-binding protein H-NS represses ribosomal RNA transcription by trapping RNA polymerase in the initiation complex; J. Mol. Biol.;  citation_year=2000;  citation_volume=298;  citation_pages=737-748; " /><meta name="citation_reference" content="citation_title=The histone-like nucleoid structuring protein H-NS represses the Escherichia coli bgl operon downstream of the promoter; Mol. Microbiol.;  citation_year=2004;  citation_volume=52;  citation_pages=589-600; " /><meta name="citation_reference" content="citation_title=DNA looping-mediated repression by histone-like protein H-NS: specific requirement of Esigma70 as a cofactor for looping; Genes Dev.;  citation_year=2005;  citation_volume=19;  citation_pages=2388-2398; " /><meta name="citation_reference" content="citation_title=H-NS and StpA proteins stimulate expression of the maltose regulation in Escherichia coli; J. Bacteriol.;  citation_year=1998;  citation_volume=180;  citation_pages=6117-6125; " /><meta name="citation_reference" content="citation_title=Multiple control of flagellum biosynthesis in Escherichia coli: role of H-NS protein and the cyclic AMP-catabolite activator protein complex in transcription of the flhDC master operon; J. Bacteriol.;  citation_year=1999;  citation_volume=181;  citation_pages=7500-7508; " /><meta name="citation_reference" content="citation_title=A genetic selection for supercoiling mutants of Escherichia coli reveals proteins implicated in chromosome structure; Mol. Microbiol.;  citation_year=2005;  citation_volume=57;  citation_pages=1636-1652; " /><meta name="citation_reference" content="citation_title=H1a, an E. coli DNA-binding protein which accumulates in stationary phase, strongly compacts DNA in vitro; Nucleic Acids Res.;  citation_year=1984;  citation_volume=12;  citation_pages=5321-5340; " /><meta name="citation_reference" content="citation_title=Applications of DNA tiling arrays for whole-genome analysis; Genomics;  citation_year=2005;  citation_volume=85;  citation_pages=1-15; " /><meta name="citation_reference" content="citation_title=Genomic approach for the understanding of dynamic aspect of chromosome behavior; Methods Enzymol.;  citation_year=2006;  citation_volume=409;  citation_pages=389-410; " /><meta name="citation_reference" content="citation_title=Genomic studies with Escherichia coli MelR protein: applications of chromatin immunoprecipitation and microarrays; J. Bacteriol.;  citation_year=2004;  citation_volume=186;  citation_pages=6938-6943; " /><meta name="citation_reference" content="citation_title=Studies of the distribution of Escherichia coli cAMP-receptor protein and RNA polymerase along the E. coli chromosome; Proc. Natl Acad. Sci. USA;  citation_year=2005;  citation_volume=102;  citation_pages=17693-17698; " /><meta name="citation_reference" content="citation_title=Immobilization of Escherichia coli RNA polymerase and location of binding sites by use of chromatin immunoprecipitation and microarrays; J. Bacteriol.;  citation_year=2005;  citation_volume=187;  citation_pages=6166-6174; " /><meta name="citation_reference" content="citation_title=Genomic analysis of LexA binding reveals the permissive nature of the Escherichia coli genome and identifies unconventional target sites; Genes Dev.;  citation_year=2005;  citation_volume=19;  citation_pages=2619-2630; " /><meta name="citation_reference" content="citation_title=Selective silencing of foreign DNA with low GC content by the H-NS protein in Salmonella; Science;  citation_year=2006;  citation_volume=313;  citation_pages=236-238; " /><meta name="citation_reference" content="citation_title=Regulation of lysyl-tRNA synthetase expression by histone-like protein H-NS of Escherichia coli; J. Bacteriol.;  citation_year=1994;  citation_volume=176;  citation_pages=7383-7386; " /><meta name="citation_reference" content="citation_title=Epitope tagging of chromosomal genes in Salmonella; Proc. Natl Acad. Sci. USA;  citation_year=2001;  citation_volume=98;  citation_pages=15264-15269; " /><meta name="citation_reference" content="citation_title=Highly accurate genome sequences of Escherichia coli K-12 strains MG1655 and W3110; Mol. Syst. Biol.;  citation_year=2006;  citation_volume=2;  citation_pages=E1-E5; " /><meta name="citation_reference" content="citation_title=Novel RNAs identified from an in-depth analysis of the transcriptome of Human chromosome 21 and 22; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=331-342; " /><meta name="citation_reference" content="citation_title=A high-resolution map of transcription in the yeast genome; Proc. Natl Acad. Sci. USA;  citation_year=2006;  citation_volume=103;  citation_pages=5320-5325; " /><meta name="citation_reference" content="citation_title=Detection of genes in Escherichia coli sequences determined by genome projects and prediction of protein production levels, based on multivariate diversity in codon usage; Comput. Appl. Biosci.;  citation_year=1996;  citation_volume=12;  citation_pages=213-25; " /><meta name="citation_reference" content="citation_title=A physiological role for DNA supercoiling in the osmotic regulation of gene expression in S. typhimurium and E. coli; Cell;  citation_year=1988;  citation_volume=52;  citation_pages=569-584; " /><meta name="citation_reference" content="citation_title=The chromatin-associated protein H-NS interacts with curved DNA to influence DNA topology and gene expression; Cell;  citation_year=1992;  citation_volume=71;  citation_pages=255-265; " /><meta name="citation_reference" content="citation_title=The Escherichia coli nucleoid protein H-NS functions directly as a transcriptional repressor; EMBO J.;  citation_year=1993;  citation_volume=12;  citation_pages=1039-1046; " /><meta name="citation_reference" content="citation_title=Silencing of the Escherichia coli bgl promoter: effects of template supercoiling and cell extracts on promoter activity in vitro; Nucleic Acids Res.;  citation_year=1996;  citation_volume=24;  citation_pages=2422-2428; " /><meta name="citation_reference" content="citation_title=Antagonistic role of H-NS and GadX in the regulation of the glutamate decarboxylase-dependent acid resistance system in Escherichia coli; J. Biol. Chem.;  citation_year=2005;  citation_volume=280;  citation_pages=21498-21505; " /><meta name="citation_reference" content="citation_title=Modulated expression of promoters containing upstream curved DNA sequences by the Escherichia coli nucleoid protein H-NS; Mol. Microbiol.;  citation_year=1994;  citation_volume=12;  citation_pages=231-240; " /><meta name="citation_reference" content="citation_title=A-tract clusters may facilitate DNA packaging in bacterial nucleoid; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=3907-3918; " /><meta name="citation_reference" content="citation_title=Supercoiling is essential for the formation and stability of the initiation complex at the divergent malEp and malKp promoters; J. Mol. Biol.;  citation_year=1991;  citation_volume=218;  citation_pages=529-542; " /><meta name="citation_reference" content="citation_title=Extensive genomic diversity inpathogenic Escherichia coli and Shigella Strains revealed by comparative genomic hybridization microarray; J. Bacteriol.;  citation_year=2004;  citation_volume=186;  citation_pages=3911-3921; " /><meta name="citation_reference" content="citation_title=Molecular archaeology of the Escherichia coli genome; Proc. Natl Acad. Sci. USA;  citation_year=1998;  citation_volume=95;  citation_pages=9413-9417; " /><meta name="citation_reference" content="citation_title=Biased biological functions of horizontally transferred genes in prokaryotic genomes; Nat. Genet.;  citation_year=2004;  citation_volume=36;  citation_pages=760-766; " /><meta name="citation_reference" content="citation_title=RegulonDB (version 5.0): Escherichia coli K-12 transcriptional regulatory network, operon organization, and growth conditions; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=D394-D397; " /><meta name="citation_reference" content="citation_title=Association of nucleoid proteins with coding and non-coding segments of the Escherichia coli genome; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=4642-4652; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/13/4/141/999218/dsl009.pdf" /><meta name="description" content="Abstract.  Heat-stable nucleoid-structuring protein (H-NS) is one of the main nucleoid proteins expressed in exponentially growing Escherichia coli cells. In ad" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/13/4/141/348695" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/13/4/141/348695" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/13/4/141/348695");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsl009';
    </script>
<title>
	Escherichia coli Histone-Like Protein H-NS Preferentially Binds to Horizontally Acquired DNA in Association with RNA Polymerase | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./348695" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BedhPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPc2hpbWEsIFRha3UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIxR3JhZHVhdGUgU2Nob29sIG9mIEluZm9ybWF0aW9uIFNjaWVuY2UsIE5hcmEgSW5zdGl0dXRlIG9mIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3k4OTE2LTUsIFRha2F5YW1hLCBJa29tYSwgTmFyYSA2MzAtMDE5MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJc2hpa2F3YSwgU2h1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMUdyYWR1YXRlIFNjaG9vbCBvZiBJbmZvcm1hdGlvbiBTY2llbmNlLCBOYXJhIEluc3RpdHV0ZSBvZiBTY2llbmNlIGFuZCBUZWNobm9sb2d5ODkxNi01LCBUYWtheWFtYSwgSWtvbWEsIE5hcmEgNjMwLTAxOTIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS3Vyb2thd2EsIEtlbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjFHcmFkdWF0ZSBTY2hvb2wgb2YgSW5mb3JtYXRpb24gU2NpZW5jZSwgTmFyYSBJbnN0aXR1dGUgb2YgU2NpZW5jZSBhbmQgVGVjaG5vbG9neTg5MTYtNSwgVGFrYXlhbWEsIElrb21hLCBOYXJhIDYzMC0wMTkyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkFpYmEsIEhpcm9mdW1pIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMkdyYWR1YXRlIFNjaG9vbCBvZiBCaW9hZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIE5hZ295YSBVbml2ZXJzaXR5TmFnb3lhLCBBaWNoaSA0NjQtODYwMSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPZ2FzYXdhcmEsIE5hb3Rha2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIxR3JhZHVhdGUgU2Nob29sIG9mIEluZm9ybWF0aW9uIFNjaWVuY2UsIE5hcmEgSW5zdGl0dXRlIG9mIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3k4OTE2LTUsIFRha2F5YW1hLCBJa29tYSwgTmFyYSA2MzAtMDE5MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkVzY2hlcmljaGlhIGNvbGkgSGlzdG9uZS1MaWtlIFByb3RlaW4gSC1OUyBQcmVmZXJlbnRpYWxseSBCaW5kcyB0byBIb3Jpem9udGFsbHkgQWNxdWlyZWQgRE5BIGluIEFzc29jaWF0aW9uIHdpdGggUk5BIFBvbHltZXJhc2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIxNDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjE1MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzbDAwOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDA2LzAxLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29vcGVyYXRpdmUgdHJhbnNpdGlvbnMgb2YgaXNvbGF0ZWQgRXNjaGVyaWNoaWEgY29saSBudWNsZW9pZHM6IGltcGxpY2F0aW9ucyBmb3IgdGhlIG51Y2xlb2lkIGFzIGEgY2VsbHVsYXIgcGhhc2U7IEouIFN0cnVjdC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTUzOyAgY2l0YXRpb25fcGFnZXM9MTYwLTE3NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIGJyaWRnaW5nOiBhIHByb3BlcnR5IHNoYXJlZCBhbW9uZyBILU5TLWxpa2UgcHJvdGVpbnM7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTg3OyAgY2l0YXRpb25fcGFnZXM9MTg0NS0xODQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ILU5TOiBhIHVuaXZlcnNhbCByZWd1bGF0b3IgZm9yIGEgZHluYW1pYyBnZW5vbWU7IE5hdC4gUmV2LiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0zOTEtNDAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmUgb2YgdGhlIGhpc3RvbmUtbGlrZSBwcm90ZWluIEgtTlMgYW5kIGl0cyByb2xlIGluIHJlZ3VsYXRpb24gYW5kIGdlbm9tZSBzdXBlcnN0cnVjdHVyZTsgQ3Vyci4gT3Bpbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MTA5LTExNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBtb2xlY3VsYXIgbWVjaGFuaXNtIGZvciB0aGUgcmVwcmVzc2lvbiBvZiB0cmFuc2NyaXB0aW9uIGJ5IHRoZSBILU5TIHByb3RlaW47IE1vbC4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPTEzMTEtMTMyMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RydWN0dXJhbCBiYXNpcyBmb3IgSC1OUy1tZWRpYXRlZCB0cmFwcGluZyBvZiBSTkEgcG9seW1lcmFzZSBpbiB0aGUgb3BlbiBpbml0aWF0aW9uIGNvbXBsZXggYXQgdGhlIHJybkIgUDE7IEouIEJpb2wuIENoZW0uOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTI3NzsgIGNpdGF0aW9uX3BhZ2VzPTIxNDYtMjE1MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGV0aGFsIG92ZXJwcm9kdWN0aW9uIG9mIHRoZSBFc2NoZXJpY2hpYSBjb2xpIG51Y2xlb2lkIHByb3RlaW4gSC1OUzogdWx0cmFtaWNyb3Njb3BpYyBhbmQgbW9sZWN1bGFyIGF1dG9wc3k7IE1vbC4gR2VuLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9MjMxOyAgY2l0YXRpb25fcGFnZXM9MjAxLTIxMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGFyZ2Utc2NhbGUgbW9uaXRvcmluZyBvZiBwbGVpb3Ryb3BpYyByZWd1bGF0aW9uIG9mIGdlbmUgZXhwcmVzc2lvbiBieSB0aGUgcHJva2FyeW90aWMgbnVjbGVvaWQtYXNzb2NpYXRlZCBwcm90ZWluLCBILU5TOyBNb2wuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz0yMC0zNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGJhY3RlcmlhbCBETkEtYmluZGluZyBwcm90ZWluIEgtTlMgcmVwcmVzc2VzIHJpYm9zb21hbCBSTkEgdHJhbnNjcmlwdGlvbiBieSB0cmFwcGluZyBSTkEgcG9seW1lcmFzZSBpbiB0aGUgaW5pdGlhdGlvbiBjb21wbGV4OyBKLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTI5ODsgIGNpdGF0aW9uX3BhZ2VzPTczNy03NDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBoaXN0b25lLWxpa2UgbnVjbGVvaWQgc3RydWN0dXJpbmcgcHJvdGVpbiBILU5TIHJlcHJlc3NlcyB0aGUgRXNjaGVyaWNoaWEgY29saSBiZ2wgb3Blcm9uIGRvd25zdHJlYW0gb2YgdGhlIHByb21vdGVyOyBNb2wuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTI7ICBjaXRhdGlvbl9wYWdlcz01ODktNjAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgbG9vcGluZy1tZWRpYXRlZCByZXByZXNzaW9uIGJ5IGhpc3RvbmUtbGlrZSBwcm90ZWluIEgtTlM6IHNwZWNpZmljIHJlcXVpcmVtZW50IG9mIEVzaWdtYTcwIGFzIGEgY29mYWN0b3IgZm9yIGxvb3Bpbmc7IEdlbmVzIERldi47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0yMzg4LTIzOTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUgtTlMgYW5kIFN0cEEgcHJvdGVpbnMgc3RpbXVsYXRlIGV4cHJlc3Npb24gb2YgdGhlIG1hbHRvc2UgcmVndWxhdGlvbiBpbiBFc2NoZXJpY2hpYSBjb2xpOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTE4MDsgIGNpdGF0aW9uX3BhZ2VzPTYxMTctNjEyNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXVsdGlwbGUgY29udHJvbCBvZiBmbGFnZWxsdW0gYmlvc3ludGhlc2lzIGluIEVzY2hlcmljaGlhIGNvbGk6IHJvbGUgb2YgSC1OUyBwcm90ZWluIGFuZCB0aGUgY3ljbGljIEFNUC1jYXRhYm9saXRlIGFjdGl2YXRvciBwcm90ZWluIGNvbXBsZXggaW4gdHJhbnNjcmlwdGlvbiBvZiB0aGUgZmxoREMgbWFzdGVyIG9wZXJvbjsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0xODE7ICBjaXRhdGlvbl9wYWdlcz03NTAwLTc1MDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ2VuZXRpYyBzZWxlY3Rpb24gZm9yIHN1cGVyY29pbGluZyBtdXRhbnRzIG9mIEVzY2hlcmljaGlhIGNvbGkgcmV2ZWFscyBwcm90ZWlucyBpbXBsaWNhdGVkIGluIGNocm9tb3NvbWUgc3RydWN0dXJlOyBNb2wuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NTc7ICBjaXRhdGlvbl9wYWdlcz0xNjM2LTE2NTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUgxYSwgYW4gRS4gY29saSBETkEtYmluZGluZyBwcm90ZWluIHdoaWNoIGFjY3VtdWxhdGVzIGluIHN0YXRpb25hcnkgcGhhc2UsIHN0cm9uZ2x5IGNvbXBhY3RzIEROQSBpbiB2aXRybzsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTg0OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9NTMyMS01MzQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BcHBsaWNhdGlvbnMgb2YgRE5BIHRpbGluZyBhcnJheXMgZm9yIHdob2xlLWdlbm9tZSBhbmFseXNpczsgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9ODU7ICBjaXRhdGlvbl9wYWdlcz0xLTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljIGFwcHJvYWNoIGZvciB0aGUgdW5kZXJzdGFuZGluZyBvZiBkeW5hbWljIGFzcGVjdCBvZiBjaHJvbW9zb21lIGJlaGF2aW9yOyBNZXRob2RzIEVuenltb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTQwOTsgIGNpdGF0aW9uX3BhZ2VzPTM4OS00MTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWMgc3R1ZGllcyB3aXRoIEVzY2hlcmljaGlhIGNvbGkgTWVsUiBwcm90ZWluOiBhcHBsaWNhdGlvbnMgb2YgY2hyb21hdGluIGltbXVub3ByZWNpcGl0YXRpb24gYW5kIG1pY3JvYXJyYXlzOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE4NjsgIGNpdGF0aW9uX3BhZ2VzPTY5MzgtNjk0MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3R1ZGllcyBvZiB0aGUgZGlzdHJpYnV0aW9uIG9mIEVzY2hlcmljaGlhIGNvbGkgY0FNUC1yZWNlcHRvciBwcm90ZWluIGFuZCBSTkEgcG9seW1lcmFzZSBhbG9uZyB0aGUgRS4gY29saSBjaHJvbW9zb21lOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTEwMjsgIGNpdGF0aW9uX3BhZ2VzPTE3NjkzLTE3Njk4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbW1vYmlsaXphdGlvbiBvZiBFc2NoZXJpY2hpYSBjb2xpIFJOQSBwb2x5bWVyYXNlIGFuZCBsb2NhdGlvbiBvZiBiaW5kaW5nIHNpdGVzIGJ5IHVzZSBvZiBjaHJvbWF0aW4gaW1tdW5vcHJlY2lwaXRhdGlvbiBhbmQgbWljcm9hcnJheXM7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTg3OyAgY2l0YXRpb25fcGFnZXM9NjE2Ni02MTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljIGFuYWx5c2lzIG9mIExleEEgYmluZGluZyByZXZlYWxzIHRoZSBwZXJtaXNzaXZlIG5hdHVyZSBvZiB0aGUgRXNjaGVyaWNoaWEgY29saSBnZW5vbWUgYW5kIGlkZW50aWZpZXMgdW5jb252ZW50aW9uYWwgdGFyZ2V0IHNpdGVzOyBHZW5lcyBEZXYuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MjYxOS0yNjMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZWxlY3RpdmUgc2lsZW5jaW5nIG9mIGZvcmVpZ24gRE5BIHdpdGggbG93IEdDIGNvbnRlbnQgYnkgdGhlIEgtTlMgcHJvdGVpbiBpbiBTYWxtb25lbGxhOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTMxMzsgIGNpdGF0aW9uX3BhZ2VzPTIzNi0yMzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlZ3VsYXRpb24gb2YgbHlzeWwtdFJOQSBzeW50aGV0YXNlIGV4cHJlc3Npb24gYnkgaGlzdG9uZS1saWtlIHByb3RlaW4gSC1OUyBvZiBFc2NoZXJpY2hpYSBjb2xpOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTE3NjsgIGNpdGF0aW9uX3BhZ2VzPTczODMtNzM4NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXBpdG9wZSB0YWdnaW5nIG9mIGNocm9tb3NvbWFsIGdlbmVzIGluIFNhbG1vbmVsbGE7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9OTg7ICBjaXRhdGlvbl9wYWdlcz0xNTI2NC0xNTI2OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaGx5IGFjY3VyYXRlIGdlbm9tZSBzZXF1ZW5jZXMgb2YgRXNjaGVyaWNoaWEgY29saSBLLTEyIHN0cmFpbnMgTUcxNjU1IGFuZCBXMzExMDsgTW9sLiBTeXN0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9RTEtRTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5vdmVsIFJOQXMgaWRlbnRpZmllZCBmcm9tIGFuIGluLWRlcHRoIGFuYWx5c2lzIG9mIHRoZSB0cmFuc2NyaXB0b21lIG9mIEh1bWFuIGNocm9tb3NvbWUgMjEgYW5kIDIyOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTMzMS0zNDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgaGlnaC1yZXNvbHV0aW9uIG1hcCBvZiB0cmFuc2NyaXB0aW9uIGluIHRoZSB5ZWFzdCBnZW5vbWU7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTAzOyAgY2l0YXRpb25fcGFnZXM9NTMyMC01MzI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXRlY3Rpb24gb2YgZ2VuZXMgaW4gRXNjaGVyaWNoaWEgY29saSBzZXF1ZW5jZXMgZGV0ZXJtaW5lZCBieSBnZW5vbWUgcHJvamVjdHMgYW5kIHByZWRpY3Rpb24gb2YgcHJvdGVpbiBwcm9kdWN0aW9uIGxldmVscywgYmFzZWQgb24gbXVsdGl2YXJpYXRlIGRpdmVyc2l0eSBpbiBjb2RvbiB1c2FnZTsgQ29tcHV0LiBBcHBsLiBCaW9zY2kuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MjEzLTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHBoeXNpb2xvZ2ljYWwgcm9sZSBmb3IgRE5BIHN1cGVyY29pbGluZyBpbiB0aGUgb3Ntb3RpYyByZWd1bGF0aW9uIG9mIGdlbmUgZXhwcmVzc2lvbiBpbiBTLiB0eXBoaW11cml1bSBhbmQgRS4gY29saTsgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MTk4ODsgIGNpdGF0aW9uX3ZvbHVtZT01MjsgIGNpdGF0aW9uX3BhZ2VzPTU2OS01ODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjaHJvbWF0aW4tYXNzb2NpYXRlZCBwcm90ZWluIEgtTlMgaW50ZXJhY3RzIHdpdGggY3VydmVkIEROQSB0byBpbmZsdWVuY2UgRE5BIHRvcG9sb2d5IGFuZCBnZW5lIGV4cHJlc3Npb247IENlbGw7ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9NzE7ICBjaXRhdGlvbl9wYWdlcz0yNTUtMjY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgRXNjaGVyaWNoaWEgY29saSBudWNsZW9pZCBwcm90ZWluIEgtTlMgZnVuY3Rpb25zIGRpcmVjdGx5IGFzIGEgdHJhbnNjcmlwdGlvbmFsIHJlcHJlc3NvcjsgRU1CTyBKLjsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTEwMzktMTA0NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2lsZW5jaW5nIG9mIHRoZSBFc2NoZXJpY2hpYSBjb2xpIGJnbCBwcm9tb3RlcjogZWZmZWN0cyBvZiB0ZW1wbGF0ZSBzdXBlcmNvaWxpbmcgYW5kIGNlbGwgZXh0cmFjdHMgb24gcHJvbW90ZXIgYWN0aXZpdHkgaW4gdml0cm87IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTI0MjItMjQyODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW50YWdvbmlzdGljIHJvbGUgb2YgSC1OUyBhbmQgR2FkWCBpbiB0aGUgcmVndWxhdGlvbiBvZiB0aGUgZ2x1dGFtYXRlIGRlY2FyYm94eWxhc2UtZGVwZW5kZW50IGFjaWQgcmVzaXN0YW5jZSBzeXN0ZW0gaW4gRXNjaGVyaWNoaWEgY29saTsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjgwOyAgY2l0YXRpb25fcGFnZXM9MjE0OTgtMjE1MDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vZHVsYXRlZCBleHByZXNzaW9uIG9mIHByb21vdGVycyBjb250YWluaW5nIHVwc3RyZWFtIGN1cnZlZCBETkEgc2VxdWVuY2VzIGJ5IHRoZSBFc2NoZXJpY2hpYSBjb2xpIG51Y2xlb2lkIHByb3RlaW4gSC1OUzsgTW9sLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MjMxLTI0MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QS10cmFjdCBjbHVzdGVycyBtYXkgZmFjaWxpdGF0ZSBETkEgcGFja2FnaW5nIGluIGJhY3RlcmlhbCBudWNsZW9pZDsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9MzkwNy0zOTE4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdXBlcmNvaWxpbmcgaXMgZXNzZW50aWFsIGZvciB0aGUgZm9ybWF0aW9uIGFuZCBzdGFiaWxpdHkgb2YgdGhlIGluaXRpYXRpb24gY29tcGxleCBhdCB0aGUgZGl2ZXJnZW50IG1hbEVwIGFuZCBtYWxLcCBwcm9tb3RlcnM7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9MjE4OyAgY2l0YXRpb25fcGFnZXM9NTI5LTU0MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXh0ZW5zaXZlIGdlbm9taWMgZGl2ZXJzaXR5IGlucGF0aG9nZW5pYyBFc2NoZXJpY2hpYSBjb2xpIGFuZCBTaGlnZWxsYSBTdHJhaW5zIHJldmVhbGVkIGJ5IGNvbXBhcmF0aXZlIGdlbm9taWMgaHlicmlkaXphdGlvbiBtaWNyb2FycmF5OyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE4NjsgIGNpdGF0aW9uX3BhZ2VzPTM5MTEtMzkyMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGFyY2hhZW9sb2d5IG9mIHRoZSBFc2NoZXJpY2hpYSBjb2xpIGdlbm9tZTsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT05NTsgIGNpdGF0aW9uX3BhZ2VzPTk0MTMtOTQxNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Qmlhc2VkIGJpb2xvZ2ljYWwgZnVuY3Rpb25zIG9mIGhvcml6b250YWxseSB0cmFuc2ZlcnJlZCBnZW5lcyBpbiBwcm9rYXJ5b3RpYyBnZW5vbWVzOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPTc2MC03NjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlZ3Vsb25EQiAodmVyc2lvbiA1LjApOiBFc2NoZXJpY2hpYSBjb2xpIEstMTIgdHJhbnNjcmlwdGlvbmFsIHJlZ3VsYXRvcnkgbmV0d29yaywgb3Blcm9uIG9yZ2FuaXphdGlvbiwgYW5kIGdyb3d0aCBjb25kaXRpb25zOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz1EMzk0LUQzOTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzc29jaWF0aW9uIG9mIG51Y2xlb2lkIHByb3RlaW5zIHdpdGggY29kaW5nIGFuZCBub24tY29kaW5nIHNlZ21lbnRzIG9mIHRoZSBFc2NoZXJpY2hpYSBjb2xpIGdlbm9tZTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9NDY0Mi00NjUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTMvNC8xNDEvOTk5MjE4L2RzbDAwOS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgSGVhdC1zdGFibGUgbnVjbGVvaWQtc3RydWN0dXJpbmcgcHJvdGVpbiAoSC1OUykgaXMgb25lIG9mIHRoZSBtYWluIG51Y2xlb2lkIHByb3RlaW5zIGV4cHJlc3NlZCBpbiBleHBvbmVudGlhbGx5IGdyb3dpbmcgRXNjaGVyaWNoaWEgY29saSBjZWxscy4gSW4gYWQiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM0ODY5NSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGTEMxslzFOd6MVl8X7Wtl5m9CHXDA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f4%2f141%2f348695" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f4%2f141%2f348695" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13440" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/13/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/4/0/m_dnaresearch13_4.cover.gif?Expires=2147483647&amp;Signature=zpj2n5NzsMk30zA4qehj6yWrvGBU~IiLu4DURkTxdl~55txhlJ5z0a2IpdyGYQcF4vlVunu-~Wh08QUqWKvj48dNgA7n8vLabfgEOY0CoaiBxls7NyW6tCeXW6Sfd2X4lV4uoHFAXj3NLXFLZshkdiFCqF5MJYIi0qOwNjsIUdAwRNhaR9xqEQSv1J65s5eb8WSbVSZ7jKourjIePJi-0rEJRs9B7Kjh~xeLnZz~SmpYlxNUZP4MEUv8qvfsHNtVbSMWxc-iOv9niMhvvwUPs612TggXmBVQynK3LwlztmX3LfsmbR-PpnPA2Dhenm3q1yRIy9w5VQzSlMaqz8FNQA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/13/4">
                        <div class="volume trailing-comma">Volume 13</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
2006        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="4951298">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4951298">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4951302">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4951302">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4951310">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4951310">2. Materials and Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4951327">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4951327">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4951346">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4951346">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4951356">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4951356">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/13/4/135/348357">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/13/4/155/348476">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main"><em>Escherichia coli</em> Histone-Like Protein H-NS Preferentially Binds to Horizontally Acquired DNA in Association with RNA Polymerase
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Taku Oshima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Graduate School of Information Science, Nara Institute of Science and Technology</div><div class="addr-line">8916-5, Takayama, Ikoma, Nara 630-0192, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Taku+Oshima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Oshima T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Oshima T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Taku Oshima</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shu Ishikawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Graduate School of Information Science, Nara Institute of Science and Technology</div><div class="addr-line">8916-5, Takayama, Ikoma, Nara 630-0192, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shu+Ishikawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ishikawa S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ishikawa S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shu Ishikawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ken Kurokawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Graduate School of Information Science, Nara Institute of Science and Technology</div><div class="addr-line">8916-5, Takayama, Ikoma, Nara 630-0192, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ken+Kurokawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kurokawa K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kurokawa K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ken Kurokawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hirofumi Aiba</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><sup>2</sup><div class="insititution">Graduate School of Bioagricultural Sciences, Nagoya University</div><div class="addr-line">Nagoya, Aichi 464-8601, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hirofumi+Aiba">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Aiba H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Aiba H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hirofumi Aiba</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Naotake Ogasawara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Graduate School of Information Science, Nara Institute of Science and Technology</div><div class="addr-line">8916-5, Takayama, Ikoma, Nara 630-0192, Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <sup>*</sup>To whom correspondence should be addressed. Tel. +81-743-72-5430, Fax. +81-743-72-5439, E-mail: <a href="mailto:nogasawa@bs.naist.jp" target="_blank">nogasawa@bs.naist.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Naotake+Ogasawara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ogasawara N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ogasawara N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Naotake Ogasawara</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="fn1"><div class="fn"><div class="footnote-content"><sup></sup>These authors contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Communicated by Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 13, Issue 4, 1 January 2006, Pages 141153, <a href='https://doi.org/10.1093/dnares/dsl009'>https://doi.org/10.1093/dnares/dsl009</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">01 January 2006</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">27 August 2006</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Received:</div>
                                <div class="wi-date">30 August 2006</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="348695" href="/dnaresearch/article-pdf/13/4/141/999218/dsl009.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Taku Oshima, Shu Ishikawa, Ken Kurokawa, Hirofumi Aiba, Naotake Ogasawara;  <em>Escherichia coli</em> Histone-Like Protein H-NS Preferentially Binds to Horizontally Acquired DNA in Association with RNA Polymerase, <em>DNA Research</em>, Volume 13, Issue 4, 1 January 2006, Pages 141153, <a href='https://doi.org/10.1093/dnares/dsl009'>https://doi.org/10.1093/dnares/dsl009</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/348695?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/348695?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/348695?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/348695?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/348695?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/348695?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/348695?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=%26lt%3Bem%26gt%3BEscherichia%20coli%26lt%3B%2Fem%26gt%3B%20Histone-Like%20Protein%20H-NS%20Preferentially%20Binds%20to%20Horizontally%20Acquired%20DNA%20in%20Association%20with%20RNA%20Polymerase&amp;publicationDate=2006-01-01&amp;volumeNum=13&amp;issueNum=4&amp;author=Oshima%2C%20Taku%3B%20Ishikawa%2C%20Shu&amp;startPage=141&amp;endPage=153&amp;contentId=10.1093%2Fdnares%2Fdsl009&amp;oa=&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f4%2f141%2f348695" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=4951298 id="4951298" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Heat-stable nucleoid-structuring protein (H-NS) is one of the main nucleoid proteins expressed in exponentially growing <em>Escherichia coli</em> cells. In addition to a role in nucleoid organization, H-NS functions as a pleiotropic regulator of gene expression. The genome-wide distribution of H-NS, compared with the distribution of RNA polymerase and transcriptionally active genes, was investigated using a high-density oligonucleotide chip. The new approach utilized in this study revealed that H-NS binds specifically to approximately 250 loci, covering &gt;1000 genes, to maintain transcriptional inactivation. RNA polymerase was detected in &gt;65% of H-NS binding sites with low or no transcriptional activity, indicating that the association of RNA polymerase to promoter regions is a general mode of transcription repression by H-NS. This study also revealed that most H-NS bound DNA have been horizontally acquired, which indicates that repression of inappropriate gene expression by H-NS plays an important role in the diversification of the <em>E. coli</em> genome. This study presents a comprehensive assessment of the distribution of H-NS within the <em>E. coli</em> genome, sheds light on the mechanism underlying the transcriptional regulation by H-NS, and provides new insight into bacterial genome evolution.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;histone-like protein H-NS&quot;">histone-like protein H-NS</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;RNA polymerase&quot;">RNA polymerase</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;horizontal genes&quot;">horizontal genes</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;high-density tiling chip&quot;">high-density tiling chip</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;ChIP-chip analysis&quot;">ChIP-chip analysis</a></div>                            <h2 scrollto-destination=4951302 id="4951302" class="section-title" >1. Introduction</h2>
<p>Genomic DNA in bacterial cells is a platform for the action of proteins involved in DNA transactions and is folded into a compact structure called a nucleoid. DNA supercoiling, macromolecular crowding and nucleoid-associated proteins contribute to nucleoid compaction, although the detailed architecture is not yet fully understood.<a reveal-id="b1" data-open="b1" class="link link-ref link-reveal xref-bibr">1</a> Among the bacterial nucleoid-associated proteins, a heterogeneous family of basic, low molecular weight, and abundant DNA-binding proteins, often called histone-like proteins, play fundamental roles in nucleoid organization.<a reveal-id="b2" data-open="b2" class="link link-ref link-reveal xref-bibr">2</a></p><p>The <em>Escherichia coli</em> genome encodes at least seven related histone-like proteins. The extensively characterized heat-stable nucleoid-structuring protein (H-NS) is one of the main histone-like proteins expressed at high levels during exponential growth.<a reveal-id="b3" data-open="b3" class="link link-ref link-reveal xref-bibr">3</a> It is a small (136 amino acids) and abundant (&gt;20000 copies per cell) protein, consisting of three structural domains. The C-terminal domain of H-NS (residues 90136) binds DNA in a relatively sequence-independent manner and recognizes intrinsically curved DNA. The N-terminal domain of H-NS (residues 164) is a dimerization domain, and the H-NS dimer formed through this domain bridges two DNA segments.<a reveal-id="b4" data-open="b4" class="link link-ref link-reveal xref-bibr">4</a> H-NS has the capacity to form larger oligomers, possibly through interactions between the flexible linker domain connecting the dimerization and DNA-binding domains. The lateral interaction of H-NS dimers allows the extension of H-NS binding from high-affinity nucleation sites to flanking low affinity sequences.<a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a> Recent atomic force microscopy experiments revealed that, after the initial binding of H-NS to DNA at distant sites, the H-NS assembles into patches of oligomers bridging the segments of DNA.<a reveal-id="b6" data-open="b6" class="link link-ref link-reveal xref-bibr">6</a> The bridging of DNA segments by H-NS plays a role in the compaction of DNA and higher order nucleoid organization. This is supported by the finding that overexpression of H-NS leads to abnormally compacted nucleoids in <em>E. coli</em>.<a reveal-id="b7" data-open="b7" class="link link-ref link-reveal xref-bibr">7</a></p><p>In addition to a role in nucleoid organization, H-NS acts as a pleiotropic regulator of gene expression. Comparison of gene expression profiles of wild type and <em>hns</em>-inactivated <em>E. coli</em> cells by microarray analysis demonstrates that the expression level of about 200 genes is directly or indirectly altered in the <em>hns</em> mutant.<a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a> Approximately one-third of these genes encode cell envelope components or proteins involved in adaptation to changes in environmental conditions. This suggests that H-NS directly or indirectly regulates adaptation to different environmental conditions. A number of H-NS-responsive promoters contain regions of intrinsic DNA curvature located upstream and/or downstream of the transcription start point, and the lateral binding of H-NS at preferential binding sites that extend over the promoter region is thought to be sufficient for repression of transcription.<a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a> A number of complex mechanisms underlying the inhibition of initiation or elongation of transcription have been proposed. Bending of the promoter sequence by RNA polymerase induces the bridging of two flanking H-NS bound sequences at the <em>rrnB</em> P1 promoter, resulting in the trapping of RNA polymerase in a non-productive state.<a reveal-id="b6" data-open="b6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a> Repression by H-NS at the <em>bgl</em> operon occurs at two levels, repression of promoter activity through upstream binding and blocking of transcription elongation by downstream binding.<a reveal-id="b10" data-open="b10" class="link link-ref link-reveal xref-bibr">10</a> Recent <em>in vitro</em> experiments showed that DNA wrapped around RNA polymerase associates with the sigma 70 (<sup>D</sup>) subunits and induces looping of the <em>hdeAB</em> promoter sequence when H-NS is bound to upstream and downstream regions, resulting in blockage of transition from the open to the elongation complex of RNA polymerase.<a reveal-id="b11" data-open="b11" class="link link-ref link-reveal xref-bibr">11</a> In addition to a role in repression of transcription, H-NS may act as an activator of transcription,<a reveal-id="b12" data-open="b12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="b13" data-open="b13" class="link link-ref link-reveal xref-bibr">13</a> although there is no direct evidence that H-NS has the capacity to act as a direct activator of transcription. H-NS is also involved in the maintenance of local superhelicity of genome DNA by forming a topological barrier and thereby inhibiting the expression of superhelicity-sensitive promoters.<a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a></p><p>Thus, extensive studies on <em>E. coli</em> H-NS since its first detailed biochemical characterization in 1984<a reveal-id="b15" data-open="b15" class="link link-ref link-reveal xref-bibr">15</a> have revealed important principles of action in DNA compaction and regulation of gene expression. Our understanding of the action of H-NS is still fragmented and a comprehensive assessment of the action of H-NS in bacterial cells is still lacking. A wide variety of eukaryotic studies have used ChIP-chip analysis, a powerful new tool that employs genome-wide high-resolution DNA tiling chips (oligonucleotide chips) for the fine mapping of transcribed regions in combination with chromatin immunoprecipitation, for monitoring the distribution of genome associated proteins <em>in vivo</em>.<a reveal-id="b16" data-open="b16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="b17" data-open="b17" class="link link-ref link-reveal xref-bibr">17</a> A limited number of similar studies in bacteria have been reported. Very recently, ChIP-chip analyses using different oligonucleotide tiling chip formats have been employed to investigate the genome-wide distribution of RNA polymerase and the transcriptional regulators, LexA, Crp and MelR.<a reveal-id="b18  b21" data-open="b18  b21" class="link link-ref link-reveal xref-bibr">1821</a> During preparation of the present report, H-NS distribution on the <em>Salmonella</em> genome has appeared.<a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a></p><p>In this study, we designed a custom Affymetrix oligonucleotide chip consisting of 50085 25 mer probe sets that match protein coding sequences (&gt;100 amino acids) at 4555 bp intervals and 82023 probe sets at 23 bp intervals, for both strands of the remaining sequences. ChIP-chip analysis was performed using this custom oligonucleotide chip to map H-NS and RNA polymerase associated sequences in <em>E. coli</em> cells grown in LB medium. We also determined transcriptionally active regions in wild type and <em>hns</em>-inactivated cells using the same high-density chip. These new approaches revealed that H-NS binds specifically to about 250 loci in the genome, covering &gt;1000 genes in total, to maintain transcriptional inactivation. H-NS binding sites containing intrinsically curved A-tract sequences were identified at a high density, consistent with the preferential binding of H-NS to curved DNA. In &gt;65% of H-NS binding sites, the binding of  and <sup>D</sup> subunits of RNA polymerase was clearly associated with low or no transcriptional activity, indicating that repression of transcription by H-NS involves the binding of RNA polymerase. Furthermore, our genome-wide analysis of H-NS binding sites revealed the preferential association of H-NS with DNA predicted to be horizontally acquired into the <em>E. coli</em> genome. We propose that the main role of H-NS is to repress the inappropriate expression of horizontally acquired genes under normal growth conditions, allowing their stable transmission and diversification of the <em>E. coli</em> genome. Thus, the genome-wide approach described in this study represents a comprehensive assessment of the genomic distribution of H-NS, sheds light on the mechanism underlying the transcriptional regulation by H-NS, and provides new insight into bacterial genome evolution.</p>                            <h2 scrollto-destination=4951310 id="4951310" class="section-title" >2. Materials and Methods</h2>
                            <h3 scrollto-destination=4951311 id="4951311" class="section-title" >2.1. Array design</h3>
<p>The <em>E. coli</em> K-12 MG1655 genome sequence and other genetic information was retrieved from NCBI GenBank (<a class="link link-uri" href="http://ncbi.nlm.nih.gov/" target="_blank"></a>), accession number U00096, and used for the design of probes according to the Affymetrix guidelines. Probe sequences were selected for protein coding regions (3871 regions) and inter-genic regions (5236 regions) using different criteria. Apart from the <em>rrnA</em> operon, rRNA genes were excluded from probe design (16S, 23S, 5S). The tRNA genes and protein coding genes with lengths of &lt;100 amino acids were treated as inter-genic regions. As a result, 50 085 25 mer sequences on the coding strand were selected for protein coding regions at 4555 bp intervals, and 82 023 sequences on both strands were selected for inter-genic regions at 23 bp intervals. Thus, 132 108 probe sets (perfect match and mismatch probes) were synthesized on the chip. Chip probe information is available on our web site (<a class="link link-uri" href="http://genome.naist.jp/bacteria/array/ecol.html" target="_blank"></a>).</p>                            <h3 scrollto-destination=4951313 id="4951313" class="section-title" >2.2. Bacterial strains</h3>
<p>W3110 was used as a wild-type strain of <em>E. coli</em> K-12. The <em>hns</em> strain (RM597) is a derivative of W3110.<a reveal-id="b23" data-open="b23" class="link link-ref link-reveal xref-bibr">23</a> Both strains were provided by Dr. K. Ito (University of Tokyo). The <em>hns</em> gene was tagged at the 3-terminus with the 3  FLAG sequence using the epitope tagging procedure.<a reveal-id="b24" data-open="b24" class="link link-ref link-reveal xref-bibr">24</a> The 3  FLAG sequence followed by the Kanamycin resistant cassette was PCR amplified from the plasmid pSUB11<a reveal-id="b24" data-open="b24" class="link link-ref link-reveal xref-bibr">24</a> using primers, 5- TGAGCAAGGTAAATCCCTCGACGATTTCCTGATCAAGCAAGACTACAAAGACCATGACGG -3 and 5- GGACAATAAAAAATCCCGCCGCTGGCGGGATTTTAAGCAACATATGAATATCCTCCTTAG -3, and inserted into the 3 end of the chromosomal <em>hns</em> gene of the <em>E. coli</em> strain BW25113(pKD46) using homologous sequences added to the primers. Proper addition of the 3  FLAG tag was confirmed by PCR amplification using appropriate primers. The <em>hns</em>-3  FLAG allele was transduced to W3110 by P1 transduction. Expression of the 3  FLAG H-NS protein in W3110 was confirmed by western blotting using the anti-flag antibody (data not shown).</p>                            <h3 scrollto-destination=4951315 id="4951315" class="section-title" >2.3. High-resolution transcriptome analysis</h3>
<p>An aliquot (0.25 ml) of an overnight liquid culture of wild-type and <em>hns</em> cells in LB medium at 37C was inoculated into 50 ml of LB medium (pH7.4) in a 300 ml flask. When the cells, grown aerobically by rotationally shaking at 200 r.p.m. at 37C, reached an OD<sub>600</sub> of 0.5, 10 ml of culture was mixed with 20 ml of RNA protect (Qiagen) and the cells were collected by centrifugation and stored at 80C. Total RNA was purified from collected cells using the RNeasy mini kit according to the manufacturer's instruction (Qiagen). Synthesis of cDNA, terminal labeling, and hybridization with the oligonucleotide chip was performed following the Affymetrix instruction manual (Affymetrix). Briefly, cDNA was synthesized from 10 g of total RNA using random primers and reverse transcriptase (Superscript III; Invitrogen), followed by purification using Qiaquick purification columns (Qiagen) and digestion with DNaseI (GE Healthcare Bioscience). Then cDNA fragments were terminally labeled with biotin-ddUTP using ENZO BioArray Terminal Labeling Kit (Enzo Life Sciences). Hybridization with the oligonucleotide chip was performed for 16 h at 42C, followed by washing, staining and scanning using the GeneChip Instrument System, according to the manufacturer's instructions (Affymetrix). PCR amplification of the cDNA was avoided to allow the coding strands of each transcript to be distinguished.</p><p>To compensate for the differences in hybridization efficiency of each 25 mer probe on the chip, we divided the hybridization intensities of cDNA synthesized from total RNA by those of the total genome DNA. Genomic DNA was purified from the collected cells using the DNeasy mini kit (Qiagen) and 1 g was digested with DNaseI (GE Healthcare Bioscience) for 15 min at 37C, followed by end-labeling and hybridization with the oligonucleotide chip as described above.</p>                            <h3 scrollto-destination=4951318 id="4951318" class="section-title" >2.4. ChIP-Chip analysis</h3>
<p>Cross-linking and purification of DNAprotein complexes were carried out according to Grainger et al.<a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a> Immunoprecipitation of the complex, recovery of precipitated DNA fragments, and amplification and end-labeling were performed using the protocol specified for high-resolution DNA tiling chip analysis.<a reveal-id="b17" data-open="b17" class="link link-ref link-reveal xref-bibr">17</a> The antibodies W0004 and W0023 (NeoClone), against the <sup>D</sup> and the  RNA polymerase subunits, respectively, and the Pan mouse immunoglobulin beads, #110.22 (Invitrogen), were used for ChIP-chip analysis of RNA polymerase binding sites. The antibody against the FLAG tag (Sigma-Aldrich) and Dynal Dynabeads protein A beads, #100.02 (Invitrogen) were used for H-NS analysis. Pan mouse immunoglobulin beads and protein A beads were mixed with antibodies at 4C for several hours and washed twice with 0.5 ml of cold TBS containing 5 mg/ml BSA, before use.</p><p><em>E. coli</em> cells were grown to an OD<sub>600</sub> of 0.5, as described for RNA preparation, then 10 ml of culture was transferred to a tube containing formaldehyde (final concentration of 1%) and incubated at room temperature for 30 min. The cross-linking reaction was terminated by the addition of 1.5 ml of 3 M glycine, and cells were collected by centrifugation at 3000 r.p.m. for 15 min, followed by washing with TBS and lysis buffer (10 mM TrisHCl pH 8.0, 20% sucrose, 50 mM NaCl, 10 mM EDTA). Cells were lysed by incubation with 20 mg/ml Lysozyme in lysis buffer at 37C for 30 min and stored at 20C until use. Lysed cells were dissolved in 4 ml of IP buffer (50 mM HEPES-KOH pH 7.5, 150 mM NaCl, 1 mM EDTA, 1% Triton X-100, 0.1% sodium deoxycholate, 0.1% SDS) and incubated at 37C for 10 min with PMSF at a final concentration of 1 mg/ml. The sample was then sonicated 10 times for 1 min at 1 min intervals on ice using an Astrason XL2020 sonicator. Cell debris was removed by centrifugation at 15 000 r.p.m. for 30 min. The supernatant fraction (whole cell extract) was mixed with the magnetic beads bound to the antibody and incubated at 4C overnight. The magnetic beads were washed twice with 750 l then 500 l of IP buffer while rotating at 4C for 5 min each. The beads were further washed with 500 l each of IP salt buffer (IP buffer containing 500 mM NaCl), IP wash buffer (10 mM TrisHCl pH 8.0, 250 mM LiCl, 1 mM EDTA, 0.5% Nonidet P-40, 0.5% Sodium deoxycolate) then finally in TE buffer. The DNAprotein complex bound to the beads was recovered by eluting with 100 l of elution buffer (50 mM TrisHCl pH 7.5, 10 mM EDTA, 1% SDS). Protein was removed by a 20 min incubation at 65C, followed by an incubation at 42C for 2 h then 65C for 6 h with 5 mg/ml proteinase K. DNA fragments free of DNAprotein cross-links were purified using the Qiagen PCR clean up kit (Qiagen) and eluted with 100 l of Buffer EB supplied in the kit. Recovered DNA fragments were amplified according to the random DNA amplification method described by Katou et al.<a reveal-id="b17" data-open="b17" class="link link-ref link-reveal xref-bibr">17</a> using the primers, PF 43 and PF 44.<a reveal-id="b20" data-open="b20" class="link link-ref link-reveal xref-bibr">20</a> The PCR reaction was performed using 28 cycles for the H-NS analysis and 34 cycles for the RNA polymerase subunit analysis, using Phusion high-fidelity DNA polymerase (Bio-Rad). Amplified DNA fragments were terminally labeled and hybridized with the oligonucleotide chip as described for the transcript mapping.</p><p>To compensate for positional differences in the efficiency of DNA recovery, most likely due to differences in the complexity and strength of local DNAprotein interactions, we normalized the hybridization intensities of DNA recovered from the immuno-precipitated complexes by dividing the values with those determined from DNA isolated from the whole cell extract of the control strain, W3110. Following cross-linking and sonication, a 100 l aliquot of supernatant from the W3110 cells was incubated at 65C for 20 min for de-crosslinking, followed by phenolchloroform treatment and ethanol precipitation. PCR amplification for 28 cycles followed by terminal labeling was performed using a 10 ng sample of DNA.</p>                            <h3 scrollto-destination=4951322 id="4951322" class="section-title" >2.5. Data analysis</h3>
<p>We developed a program to extract and manipulate raw intensity data (.CEL files), and to display results along the genome coordinate of strain MG1655. The genome sequence of strain W3110 used in this study has only 8 bp differences, 12 IS insertions and 1 phage deletion when compared with the MG1655 strain, although the sequence between <em>rrnD</em> and <em>rrnE</em> is inverted.<a reveal-id="b25" data-open="b25" class="link link-ref link-reveal xref-bibr">25</a></p><p>We subtracted signal intensities of mismatch probes from those of perfect match probes for use as signal intensities.<a reveal-id="b26" data-open="b26" class="link link-ref link-reveal xref-bibr">26</a> Probes with a negative value for signal intensity were excluded from further analysis. For transcript mapping, cDNA signal intensities were divided by the signal intensities of genome DNA, to normalize the differences in hybridization efficiency of probes on the chip (transcriptional activity signals).<a reveal-id="b27" data-open="b27" class="link link-ref link-reveal xref-bibr">27</a> For the ChIP-chip analysis, the signal intensities of DNA in the immuno-precipitated fraction were divided by those determined for DNA isolated from the whole cell extract fraction before precipitation (binding signals, enrichment in the chromatin immuno-precipitated fraction ).<a reveal-id="b17" data-open="b17" class="link link-ref link-reveal xref-bibr">17</a> The distribution of transcriptional activity and protein binding signals along the genome coordinate was visualized and transcriptionally active regions, H-NS binding sites and RNA polymerase binding sites were assessed manually. All experiments were carried out more than two times, and average signal intensities were used for the analysis.</p>                            <h3 scrollto-destination=4951325 id="4951325" class="section-title" >2.6. Factorial correspondence analysis of codon usage</h3>
<p>The codon usage pattern has been determined for all protein coding genes in the <em>E. coli</em> genome.<a reveal-id="b28" data-open="b28" class="link link-ref link-reveal xref-bibr">28</a> Principal component analysis (PCA) was performed against the frequency of codon usage of all genes and the clustering results were projected onto a two-dimensional field.</p>                            <h2 scrollto-destination=4951327 id="4951327" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=4951328 id="4951328" class="section-title" >3.1. Genome-wide ChIP-chip analysis of H-NS binding regions</h3>
<p>We performed ChIP-chip analysis using 3  FLAG-tagged H-NS sequences to construct a map of H-NS binding sites on the whole <em>E. coli</em> genome as described in Materials and Methods. A strain of <em>E. coli,</em> with the native <em>hns</em> gene in-frame with the 3  FLAG sequence at the 3 terminus, was constructed by epitope tagging<a reveal-id="b24" data-open="b24" class="link link-ref link-reveal xref-bibr">24</a> and used to isolate DNA fragments associated with H-NS <em>in vivo</em>. The addition of the tag to H-NS had no significant effects on the growth of <em>E. coli</em> cells and repression of transcription of the <em>lysU</em> gene by H-NS<a reveal-id="b23" data-open="b23" class="link link-ref link-reveal xref-bibr">23</a> (data not shown). We developed a program to manipulate raw hybridization data and to display final results along the <em>E. coli</em> genome coordinate, and H-NS binding sites were assessed manually. As the results, H-NS was found to specifically bind to about 250 loci along the <em>E. coli</em> genome, covering &gt;1000 genes (Supplementary Table S1 and Figure S1; see also the authors' web site: <a class="link link-uri" href="http://genome.naist.jp/bacteria/array/ecol.html" target="_blank"></a>). Previously reported H-NS binding sites for <em>proV</em>, <em>lysU</em> and <em>bglG</em> genes were clearly detected among these loci (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1AC</a>).<a reveal-id="b10" data-open="b10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="b23" data-open="b23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="b29  b32" data-open="b29  b32" class="link link-ref link-reveal xref-bibr">2932</a> In 15 loci, H-NS covers a wide region &gt;10 kb, including several well-known H-NS targets (see <a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2A</a>).<a reveal-id="b11" data-open="b11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="b33" data-open="b33" class="link link-ref link-reveal xref-bibr">33</a> As H-NS binds to intrinsically curved and phased A-tract sequences,<a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="b34" data-open="b34" class="link link-ref link-reveal xref-bibr">34</a> we investigated the association of the density of A-tract sequences (A<em>n</em>T<em>m</em> sequences, where 4  (<em>n</em> + <em>m</em>) )<a reveal-id="b35" data-open="b35" class="link link-ref link-reveal xref-bibr">35</a> with the peak position of H-NS binding signals in selected regions (<a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2A and B</a>). As expected, peaks of H-NS binding sites correlated with A-tract sequences. A genome-wide analysis showed a strong correlation between the average signal strength of H-NS binding of probes in each protein coding sequences and the density of A-tract sequences in the sequence (<a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2C</a>).</p>                        <a id="4951330" scrollto-destination="4951330"></a>
<div id="fig1" data-id="fig1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl009/2/m_dsl009f1.jpeg?Expires=2147483647&Signature=Bh3xnkG3Bh5XqJlRe6VeV--jMOj0G8I03IITXu9gE~VQPkbcYuYK513TW23FqLj5W1DA~QwYLyr4nrWFzS2ZjmdJJCFexlayOQed6iagZKwsiXSgZd1zuCVF19s-swh33njVNcw9H0K62JDowUAjAfIy~evXgZDi-FdyufzVgGufJ955MXv2l7IqMOm11cqxqleuZtIkvSvzUOqXmx~0VXFWy1a0N6mz6GmC4J2jc-5ZZn7KH~PF3jAkJdtojn2z-dURgkNSr9RYH6Du7dk7V2CFSOH~aZj-FPCSx3Udr5xaTZGwx-3RKmf7pIhXgsdlZk8UDtqGtHm7iqCiupPDOQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="H-NS binding profiles around genes regulated by H-NS. H-NS binding signals around the proVWX operon (A), the lysU gene (B), and the bglG gene (C). Vertical bars indicate the relative hybridization intensity of DNA from immuno-precipitated fraction compared with the DNA from whole cell extracts (enrichment in the chromatin immuno-precipitated fraction). The horizontal line at the bottom indicates the chromosomal coordinates in bp. The arrangement of genes (thick arrows) and known promoters (thin arrows) are shown schematically above the coordinate. The relative hybridization intensities for each 25 mer probe on the chip are shown at their corresponding coordinates." path-from-xml="dsl009f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4951330/dsl009f1.jpeg" path-from-xml="dsl009f1.jpeg" target="_blank">View large</a><a section="4951330" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl009/2/dsl009f1.jpeg?Expires=2147483647&Signature=KcmDTppg6eXF6ebojSIyI0IlFO2jaM-LVauYt4Nd-z4F8KkCGEawgYsrqzISbf6znvDz3ed9nTdYjHg9FuZ9A3xWmP0ERMVJW0ZiNhtQcmoCPpfKdLS7UYJEufevKjXPqGn7LBjx2QIodQuiFqUvAhR05lGn-aamVZCVrylV58tdxKm6Q8oboYgluFkgg9-YjlRzY-LDG2bcjdeoBKFnWGhrPSULzm0kdzGox6ELt2BCKHEjY70b96pdqfAaB~Un6JLtdnWQMs03cxhlz5-HBjTYdfkiv7E4Jl9fTXMbruJnCbA-Jm-XPvL3NJGeYSDw4dKqYYb2BX34ttv6rTKqnw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4951330&ar=348695&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl009f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>H-NS binding profiles around genes regulated by H-NS. H-NS binding signals around the <em>proVWX</em> operon (A), the <em>lysU</em> gene (B), and the <em>bglG</em> gene (C). Vertical bars indicate the relative hybridization intensity of DNA from immuno-precipitated fraction compared with the DNA from whole cell extracts (enrichment in the chromatin immuno-precipitated fraction). The horizontal line at the bottom indicates the chromosomal coordinates in bp. The arrangement of genes (thick arrows) and known promoters (thin arrows) are shown schematically above the coordinate. The relative hybridization intensities for each 25 mer probe on the chip are shown at their corresponding coordinates.</p></div></div><div content-id="fig1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl009/2/m_dsl009f1.jpeg?Expires=2147483647&Signature=Bh3xnkG3Bh5XqJlRe6VeV--jMOj0G8I03IITXu9gE~VQPkbcYuYK513TW23FqLj5W1DA~QwYLyr4nrWFzS2ZjmdJJCFexlayOQed6iagZKwsiXSgZd1zuCVF19s-swh33njVNcw9H0K62JDowUAjAfIy~evXgZDi-FdyufzVgGufJ955MXv2l7IqMOm11cqxqleuZtIkvSvzUOqXmx~0VXFWy1a0N6mz6GmC4J2jc-5ZZn7KH~PF3jAkJdtojn2z-dURgkNSr9RYH6Du7dk7V2CFSOH~aZj-FPCSx3Udr5xaTZGwx-3RKmf7pIhXgsdlZk8UDtqGtHm7iqCiupPDOQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="H-NS binding profiles around genes regulated by H-NS. H-NS binding signals around the proVWX operon (A), the lysU gene (B), and the bglG gene (C). Vertical bars indicate the relative hybridization intensity of DNA from immuno-precipitated fraction compared with the DNA from whole cell extracts (enrichment in the chromatin immuno-precipitated fraction). The horizontal line at the bottom indicates the chromosomal coordinates in bp. The arrangement of genes (thick arrows) and known promoters (thin arrows) are shown schematically above the coordinate. The relative hybridization intensities for each 25 mer probe on the chip are shown at their corresponding coordinates." path-from-xml="dsl009f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4951330/dsl009f1.jpeg" path-from-xml="dsl009f1.jpeg" target="_blank">View large</a><a section="4951330" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl009/2/dsl009f1.jpeg?Expires=2147483647&Signature=KcmDTppg6eXF6ebojSIyI0IlFO2jaM-LVauYt4Nd-z4F8KkCGEawgYsrqzISbf6znvDz3ed9nTdYjHg9FuZ9A3xWmP0ERMVJW0ZiNhtQcmoCPpfKdLS7UYJEufevKjXPqGn7LBjx2QIodQuiFqUvAhR05lGn-aamVZCVrylV58tdxKm6Q8oboYgluFkgg9-YjlRzY-LDG2bcjdeoBKFnWGhrPSULzm0kdzGox6ELt2BCKHEjY70b96pdqfAaB~Un6JLtdnWQMs03cxhlz5-HBjTYdfkiv7E4Jl9fTXMbruJnCbA-Jm-XPvL3NJGeYSDw4dKqYYb2BX34ttv6rTKqnw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4951330&ar=348695&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl009f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>H-NS binding profiles around genes regulated by H-NS. H-NS binding signals around the <em>proVWX</em> operon (A), the <em>lysU</em> gene (B), and the <em>bglG</em> gene (C). Vertical bars indicate the relative hybridization intensity of DNA from immuno-precipitated fraction compared with the DNA from whole cell extracts (enrichment in the chromatin immuno-precipitated fraction). The horizontal line at the bottom indicates the chromosomal coordinates in bp. The arrangement of genes (thick arrows) and known promoters (thin arrows) are shown schematically above the coordinate. The relative hybridization intensities for each 25 mer probe on the chip are shown at their corresponding coordinates.</p></div></div>                        <a id="4951331" scrollto-destination="4951331"></a>
<div id="fig2" data-id="fig2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl009/2/m_dsl009f2.jpeg?Expires=2147483647&Signature=mul6xNS6IxQlPGQ7Kk0bflqWtstya-9rwP3s3HFzb-7dKYNPtEdFHZ-GEUjZNUC9snwRv7IJEe13kBEkYtuGhwhssxNJMYuUIyXtvSrSAt46polukwMRpKtCE7jL2UFw3iVY10n043El7upPMRZBegTyI45220qprzMuAvRIzZhyv1B2k78wslT-R3QkzeVlM~15EBB2dAgTCGY0Ch6wVQx3WDzY5CbV~igDmcyttj2w8TMaPaX-EJ2jp~N7bLqq0faxvYqJUrIAbUjstMYWCEaBzabKEBp101ZY64HbodumM8kbKHXkZls~L-iX-zNlBUT7OArxbSZ2nVAtwuJDOg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between H-NS binding sites and A-tract sequences. H-NS binding signals (vertical bars), arrangement of genes (thick arrows), frequency of A-tract sequences (number of A-tract sequences in 100 bp window was plotted by 50 bp steps; line graph) and chromosomal coordinates (horizontal line) are shown for the gadA (A) and gadB (B) loci. The genome-wide correlation between the average peak heights of H-NS binding signals and the density of A-tract sequences (number of A-tract sequences per 1 kb) in coding sequences of each gene is shown in panel C." path-from-xml="dsl009f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4951331/dsl009f2.jpeg" path-from-xml="dsl009f2.jpeg" target="_blank">View large</a><a section="4951331" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl009/2/dsl009f2.jpeg?Expires=2147483647&Signature=BkWBFXfty2zR7MTb6VkpxwD3eI4SC8vrCqcajA~Kxb0j7KkUTIP5yE2zJq5GVZEEat4NCV2Oyd2hDh1qJjAoYKeZNHe6HIyOtNl-nKgHi~jd1HuLPOBeqKlRcj-cvTdCGoz5-aDpwgNLS0e1aJcXlYdRUBCu87kOJO8QPZW9v6dj6VcLjYP9md402b2~N-4crnalXcaaot6uoqBEurcIO~DMASfnv-OClKDJEIkPlPdt7KrH~ui7WCHrz1F9~HAajvAT0uMgfYxVkq9NQ8dROkrMabcMdzw7lfLGUY-942f7mjOD95L7yccTA7jasz5kt5VgM01F75lgfNDJS1~LKA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4951331&ar=348695&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl009f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between H-NS binding sites and A-tract sequences. H-NS binding signals (vertical bars), arrangement of genes (thick arrows), frequency of A-tract sequences (number of A-tract sequences in 100 bp window was plotted by 50 bp steps; line graph) and chromosomal coordinates (horizontal line) are shown for the <em>gadA</em> (A) and <em>gadB</em> (B) loci. The genome-wide correlation between the average peak heights of H-NS binding signals and the density of A-tract sequences (number of A-tract sequences per 1 kb) in coding sequences of each gene is shown in panel C.</p></div></div><div content-id="fig2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl009/2/m_dsl009f2.jpeg?Expires=2147483647&Signature=mul6xNS6IxQlPGQ7Kk0bflqWtstya-9rwP3s3HFzb-7dKYNPtEdFHZ-GEUjZNUC9snwRv7IJEe13kBEkYtuGhwhssxNJMYuUIyXtvSrSAt46polukwMRpKtCE7jL2UFw3iVY10n043El7upPMRZBegTyI45220qprzMuAvRIzZhyv1B2k78wslT-R3QkzeVlM~15EBB2dAgTCGY0Ch6wVQx3WDzY5CbV~igDmcyttj2w8TMaPaX-EJ2jp~N7bLqq0faxvYqJUrIAbUjstMYWCEaBzabKEBp101ZY64HbodumM8kbKHXkZls~L-iX-zNlBUT7OArxbSZ2nVAtwuJDOg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between H-NS binding sites and A-tract sequences. H-NS binding signals (vertical bars), arrangement of genes (thick arrows), frequency of A-tract sequences (number of A-tract sequences in 100 bp window was plotted by 50 bp steps; line graph) and chromosomal coordinates (horizontal line) are shown for the gadA (A) and gadB (B) loci. The genome-wide correlation between the average peak heights of H-NS binding signals and the density of A-tract sequences (number of A-tract sequences per 1 kb) in coding sequences of each gene is shown in panel C." path-from-xml="dsl009f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4951331/dsl009f2.jpeg" path-from-xml="dsl009f2.jpeg" target="_blank">View large</a><a section="4951331" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl009/2/dsl009f2.jpeg?Expires=2147483647&Signature=BkWBFXfty2zR7MTb6VkpxwD3eI4SC8vrCqcajA~Kxb0j7KkUTIP5yE2zJq5GVZEEat4NCV2Oyd2hDh1qJjAoYKeZNHe6HIyOtNl-nKgHi~jd1HuLPOBeqKlRcj-cvTdCGoz5-aDpwgNLS0e1aJcXlYdRUBCu87kOJO8QPZW9v6dj6VcLjYP9md402b2~N-4crnalXcaaot6uoqBEurcIO~DMASfnv-OClKDJEIkPlPdt7KrH~ui7WCHrz1F9~HAajvAT0uMgfYxVkq9NQ8dROkrMabcMdzw7lfLGUY-942f7mjOD95L7yccTA7jasz5kt5VgM01F75lgfNDJS1~LKA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4951331&ar=348695&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl009f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between H-NS binding sites and A-tract sequences. H-NS binding signals (vertical bars), arrangement of genes (thick arrows), frequency of A-tract sequences (number of A-tract sequences in 100 bp window was plotted by 50 bp steps; line graph) and chromosomal coordinates (horizontal line) are shown for the <em>gadA</em> (A) and <em>gadB</em> (B) loci. The genome-wide correlation between the average peak heights of H-NS binding signals and the density of A-tract sequences (number of A-tract sequences per 1 kb) in coding sequences of each gene is shown in panel C.</p></div></div>                            <h3 scrollto-destination=4951332 id="4951332" class="section-title" >3.2. High-resolution transcriptome analysis of wild type and hns-inactivated cells</h3>
<p>To examine whether the H-NS binding detected by ChIP-chip analysis is related to gene expression, the transcriptionally active regions of the <em>E. coli</em> genome were mapped in wild type and <em>hns</em>-inactivated (<em>hns</em>) cells using the same high-density oligonucleotide chip as described in Materials and Methods (Supplementary Figure S1).</p><p>A comparison of transcriptional activity with H-NS binding sites in wild-type cells revealed that the transcriptional activity of H-NS bound regions is generally low, as is observed, for example, in the fimbria-related gene cluster, <em>yadCKLM-htrE-ecpD-yadN</em> where H-NS binds to whole coding sequences (<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3B</a>) and in the LPS synthesis related genes, <em>wza-wzb-wzc</em>-<em>wcaA-H</em> where promoter region is covered by H-NS (<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3C</a>). By comparison, transcriptional active loci such as the ribosomal protein operon (<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig 3A</a>) generally have no bound H-NS. In confirmation of these findings, a scatter plot of signal intensities of H-NS binding and transcriptional activity for each probe showed a reverse correlation (<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3D</a>), demonstrating that the transcriptional activity of H-NS bound sites is significantly lower than the transcriptional activity of unbound regions.</p>                        <a id="4951335" scrollto-destination="4951335"></a>
<div id="fig3" data-id="fig3" class="fig fig-section"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl009/2/m_dsl009f3.jpeg?Expires=2147483647&Signature=zavu9tjD0R5cOf1wJsEozZLw6K-D-eWayap3MqLin23JMCF5-1ui8E05YDfuHaKK6uK1bnaJrLvtiP7bGiJO1Uqrw9tAjmJECPxeKgWIQM3cq1nZLnftnNDWpduxU9M3jybmzLj4NY0udjZ7Y8A8tCQ4jE~TdERwM6DRgS9DLY16ncpsD96htXVkccAZvLUGHXGZ29VpLo3ucTZ8GwEYn3AjuMFAO20zzbKlzgeNXPvI22~sG6fm~wShh2JK9EZbIyG1~sewlqBWnzeygmzPoILSsyKeYF-d6ueRBlVc1uGS~Xe~hWUohgYeMONc4RgAEquDUBZ7pSaC1KNXFSA2IQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of transcriptional activity and H-NS binding signals. Transcription activity signals and H-NS,  and D binding signals are shown for selected regions; the rpsL locus encoding ribosomal proteins and translation factors (A), the yadN-yadC locus densely covered by H-NS (B), and the wza-wcaH locus encoding colonic acid biosynthesis related genes and silenced by partial H-NS binding (C). Transcriptional activity signals are calculated by dividing the hybridization intensity of cDNA with that for genome DNA. The genome-wide correlation between transcriptional activity signals and H-NS binding signals for each 25mer probe on the chip is shown in panel D." path-from-xml="dsl009f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4951335/dsl009f3.jpeg" path-from-xml="dsl009f3.jpeg" target="_blank">View large</a><a section="4951335" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl009/2/dsl009f3.jpeg?Expires=2147483647&Signature=fk7h7JwWaKdWDD2HE32jJuPh6SUHbkeWSQR97ZL9T3AYPRVULRQNozarRvo9h3ElW9Ac66TTA0-nPNv4jFr-CETTdHYVaAwBQ2e~K9oW-zJwPNJrNsgV43YiKYQUxrC8QK90YFFty7neyreGbVLreNBRE4oAxTUxxC-JSeInukBmcIL4aQOgA36h4IzEW1Ib1UrNdIDeQSkAs99gZ5xe8uF5YOfUrDTdyodKDj4I8~-uNQcljQFsWjx0SbvZyfYfCdXHP5fwsTgExMF07X8ZFTzSWQu60cU89C6dkohOZyDacRVF1GSB2v~Ckf7fxhebJMlw~cZ7cxJF0oi~Nr7Qlg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4951335&ar=348695&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl009f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of transcriptional activity and H-NS binding signals. Transcription activity signals and H-NS,  and <sup>D</sup> binding signals are shown for selected regions; the <em>rpsL</em> locus encoding ribosomal proteins and translation factors (A), the <em>yadN-yadC</em> locus densely covered by H-NS (B), and the <em>wza-wcaH</em> locus encoding colonic acid biosynthesis related genes and silenced by partial H-NS binding (C). Transcriptional activity signals are calculated by dividing the hybridization intensity of cDNA with that for genome DNA. The genome-wide correlation between transcriptional activity signals and H-NS binding signals for each 25mer probe on the chip is shown in panel D.</p></div></div><div content-id="fig3" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl009/2/m_dsl009f3.jpeg?Expires=2147483647&Signature=zavu9tjD0R5cOf1wJsEozZLw6K-D-eWayap3MqLin23JMCF5-1ui8E05YDfuHaKK6uK1bnaJrLvtiP7bGiJO1Uqrw9tAjmJECPxeKgWIQM3cq1nZLnftnNDWpduxU9M3jybmzLj4NY0udjZ7Y8A8tCQ4jE~TdERwM6DRgS9DLY16ncpsD96htXVkccAZvLUGHXGZ29VpLo3ucTZ8GwEYn3AjuMFAO20zzbKlzgeNXPvI22~sG6fm~wShh2JK9EZbIyG1~sewlqBWnzeygmzPoILSsyKeYF-d6ueRBlVc1uGS~Xe~hWUohgYeMONc4RgAEquDUBZ7pSaC1KNXFSA2IQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of transcriptional activity and H-NS binding signals. Transcription activity signals and H-NS,  and D binding signals are shown for selected regions; the rpsL locus encoding ribosomal proteins and translation factors (A), the yadN-yadC locus densely covered by H-NS (B), and the wza-wcaH locus encoding colonic acid biosynthesis related genes and silenced by partial H-NS binding (C). Transcriptional activity signals are calculated by dividing the hybridization intensity of cDNA with that for genome DNA. The genome-wide correlation between transcriptional activity signals and H-NS binding signals for each 25mer probe on the chip is shown in panel D." path-from-xml="dsl009f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4951335/dsl009f3.jpeg" path-from-xml="dsl009f3.jpeg" target="_blank">View large</a><a section="4951335" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl009/2/dsl009f3.jpeg?Expires=2147483647&Signature=fk7h7JwWaKdWDD2HE32jJuPh6SUHbkeWSQR97ZL9T3AYPRVULRQNozarRvo9h3ElW9Ac66TTA0-nPNv4jFr-CETTdHYVaAwBQ2e~K9oW-zJwPNJrNsgV43YiKYQUxrC8QK90YFFty7neyreGbVLreNBRE4oAxTUxxC-JSeInukBmcIL4aQOgA36h4IzEW1Ib1UrNdIDeQSkAs99gZ5xe8uF5YOfUrDTdyodKDj4I8~-uNQcljQFsWjx0SbvZyfYfCdXHP5fwsTgExMF07X8ZFTzSWQu60cU89C6dkohOZyDacRVF1GSB2v~Ckf7fxhebJMlw~cZ7cxJF0oi~Nr7Qlg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4951335&ar=348695&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl009f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of transcriptional activity and H-NS binding signals. Transcription activity signals and H-NS,  and <sup>D</sup> binding signals are shown for selected regions; the <em>rpsL</em> locus encoding ribosomal proteins and translation factors (A), the <em>yadN-yadC</em> locus densely covered by H-NS (B), and the <em>wza-wcaH</em> locus encoding colonic acid biosynthesis related genes and silenced by partial H-NS binding (C). Transcriptional activity signals are calculated by dividing the hybridization intensity of cDNA with that for genome DNA. The genome-wide correlation between transcriptional activity signals and H-NS binding signals for each 25mer probe on the chip is shown in panel D.</p></div></div><p>A comparison of transcriptional profiles in wild type and <em>hns</em>-inactivated cells showed that derepression of transcription in the mutant cells occurs at about 65% of the H-NS bound loci (Supplementary Table S1). Typical examples of derepression of transcription by H-NS inactivation are shown in <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4</a>. In some H-NS repressed genes, other factors are required to activate expression, even when H-NS is removed from the cell. For example, activation of transcription of the <em>bgl</em> promoter is dependent upon the activator protein, CRP, in addition to the removal of bound H-NS.<a reveal-id="b32" data-open="b32" class="link link-ref link-reveal xref-bibr">32</a></p>                        <a id="4951337" scrollto-destination="4951337"></a>
<div id="fig4" data-id="fig4" class="fig fig-section"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl009/2/m_dsl009f4.jpeg?Expires=2147483647&Signature=nWX6TIxXrsbYb6spdT-U9EpuKfj5htU7DVhbiXOi7~1SxwavhoS8~dlLWQw1vUUMcbFSonafSkJrOHgJLBMqErQhgVAx~sEHj4x3ZX-e4dEdhnpkFk3NMwo5GmfqTUkJbMck1~9pMJYQM22Q5y~quCaoDJkFG1fV5V0JNRShgqFTmZb8Czdg-hSafmwl8r~ZoG1c~p1HLRAFiOYKOP22dG3wg~KCqoyH3yeMEK8ksXsqEkBtx6nXPruUKB~~1mM6XcSY9ijossNOuZZKDgr2vZ30~YXYsfI70lC04b2L1ziHEXlcKrK7sVX-uJ3LnZx5QgHg7x~tDpNsT4coZAholw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Derepression of transcriptional activity of H-NS bound genes in the hns-inactivated cells. H-NS binding signals in wild type (W3110) cells and transcriptional activity signals in wild type (W3110) and hns-inactivated (hns) cells are shown for selected regions; the yagK (A), the ykgB (B) and the yhiM (C) loci. Transcriptional activities of Watson and Crick strands are shown above and below the ORF map, respectively." path-from-xml="dsl009f4.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4951337/dsl009f4.jpeg" path-from-xml="dsl009f4.jpeg" target="_blank">View large</a><a section="4951337" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl009/2/dsl009f4.jpeg?Expires=2147483647&Signature=W798mBx~jv4~bIA9RIuI0FI2c1tPiIya7O9WfaYQ9pQLzdJjuGUC4zQ6piHRKS8JgP7Gh6rTx75vtXgrrGaOrPc610Zw76e~MuoZY5Vmg9DPJxtCdnPGh2Yc~HVUFjCuuI81LZZVlj7F6gbP8urOBmdTmoSktZ6KsKgT~GCTpjKNCrpGX6MwJN6ePYcJXnvtnqPQKxsF47QOefNfxIw14HuRqIReN7~gyxZp8RSl8SyWCzKpSbPWjDo1c-1Dvl5kXSGiD7S10YhL2oin4-uygmDafHQWVhnQnilaQ14ziC7KMlGD0KTFlqZtY6qyf3jaOG-GdFJ6mII9MmmEGNP7og__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4951337&ar=348695&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl009f4.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Derepression of transcriptional activity of H-NS bound genes in the <em>hns-inactivated</em> cells. H-NS binding signals in wild type (W3110) cells and transcriptional activity signals in wild type (W3110) and <em>hns-inactivated</em> (<em>hns</em>) cells are shown for selected regions; the <em>yagK</em> (A), the <em>ykgB</em> (B) and the <em>yhiM</em> (C) loci. Transcriptional activities of Watson and Crick strands are shown above and below the ORF map, respectively.</p></div></div><div content-id="fig4" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl009/2/m_dsl009f4.jpeg?Expires=2147483647&Signature=nWX6TIxXrsbYb6spdT-U9EpuKfj5htU7DVhbiXOi7~1SxwavhoS8~dlLWQw1vUUMcbFSonafSkJrOHgJLBMqErQhgVAx~sEHj4x3ZX-e4dEdhnpkFk3NMwo5GmfqTUkJbMck1~9pMJYQM22Q5y~quCaoDJkFG1fV5V0JNRShgqFTmZb8Czdg-hSafmwl8r~ZoG1c~p1HLRAFiOYKOP22dG3wg~KCqoyH3yeMEK8ksXsqEkBtx6nXPruUKB~~1mM6XcSY9ijossNOuZZKDgr2vZ30~YXYsfI70lC04b2L1ziHEXlcKrK7sVX-uJ3LnZx5QgHg7x~tDpNsT4coZAholw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Derepression of transcriptional activity of H-NS bound genes in the hns-inactivated cells. H-NS binding signals in wild type (W3110) cells and transcriptional activity signals in wild type (W3110) and hns-inactivated (hns) cells are shown for selected regions; the yagK (A), the ykgB (B) and the yhiM (C) loci. Transcriptional activities of Watson and Crick strands are shown above and below the ORF map, respectively." path-from-xml="dsl009f4.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4951337/dsl009f4.jpeg" path-from-xml="dsl009f4.jpeg" target="_blank">View large</a><a section="4951337" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl009/2/dsl009f4.jpeg?Expires=2147483647&Signature=W798mBx~jv4~bIA9RIuI0FI2c1tPiIya7O9WfaYQ9pQLzdJjuGUC4zQ6piHRKS8JgP7Gh6rTx75vtXgrrGaOrPc610Zw76e~MuoZY5Vmg9DPJxtCdnPGh2Yc~HVUFjCuuI81LZZVlj7F6gbP8urOBmdTmoSktZ6KsKgT~GCTpjKNCrpGX6MwJN6ePYcJXnvtnqPQKxsF47QOefNfxIw14HuRqIReN7~gyxZp8RSl8SyWCzKpSbPWjDo1c-1Dvl5kXSGiD7S10YhL2oin4-uygmDafHQWVhnQnilaQ14ziC7KMlGD0KTFlqZtY6qyf3jaOG-GdFJ6mII9MmmEGNP7og__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4951337&ar=348695&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl009f4.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Derepression of transcriptional activity of H-NS bound genes in the <em>hns-inactivated</em> cells. H-NS binding signals in wild type (W3110) cells and transcriptional activity signals in wild type (W3110) and <em>hns-inactivated</em> (<em>hns</em>) cells are shown for selected regions; the <em>yagK</em> (A), the <em>ykgB</em> (B) and the <em>yhiM</em> (C) loci. Transcriptional activities of Watson and Crick strands are shown above and below the ORF map, respectively.</p></div></div><p>A reduction in transcriptional activity was observed in several loci in the <em>hns</em> mutant, especially for <em>malEFG</em> and <em>malK-lamB-malM</em> (Supplementary Figure S2). This may result from a reduction in local superhelicity in the <em>hns</em> mutant, as proposed by Hardy and Cozzarelli.<a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a> Consistent with this finding, the <em>malEFG</em> and <em>malK-lamB-malM</em> loci are flanked by H-NS bound sequences (Supplementary Figure S2), and supercoiling is essential for the formation and stability of the initiation complex at the divergent <em>malE</em> and <em>malK</em> promoters.<a reveal-id="b36" data-open="b36" class="link link-ref link-reveal xref-bibr">36</a></p>                            <h3 scrollto-destination=4951339 id="4951339" class="section-title" >3.3. Genome-wide ChIP-chip analysis of RNA polymerase associated regions</h3>
<p>A positive role for RNA polymerase in H-NS associated transcription repression has been demonstrated for the <em>rrnB</em> P1 and <em>hdeAB</em> promoters.<a reveal-id="b6" data-open="b6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="b11" data-open="b11" class="link link-ref link-reveal xref-bibr">11</a> To examine the degree of association between RNA polymerase and H-NS binding, we performed ChIP-chip analysis to map RNA polymerase binding sites, using monoclonal antibodies against the  and <sup>D</sup> subunits (Supplementary Figure S1). In about 90 highly transcribed regions, the  subunit gave broad peaks throughout the operon, while the <sup>D</sup> subunit gave a peak at the promoter region (<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3A</a>). It is likely that these signals correspond to the closed or stable open complexes at the promoters (<sup>D</sup> signal) and elongation complexes in coding sequences ( signal). In addition to signals in highly expressed regions, we detected signal spikes for the  and <sup>D</sup> subunits at the 5 end of non-expressed or weakly expressed genes or operons (<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3B and C</a>). Furthermore, we found that such RNA polymerase signals overlapped with the H-NS peaks in &gt;65% of the H-NS binding sites (Supplementary Table S1). The association of RNA polymerase and H-NS binding signals is especially evident in clustered H-NS binding sites (<a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5A</a>). Scatter plots (<a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5B</a> and C) demonstrate the partial, but significant, association of RNA polymerase and H-NS along the genome. These results strongly suggest that the trapping of RNA polymerase in an inactive form, with respect to initiation or elongation of transcription, is a genome-wide phenomenon. The signals observed for the  and <sup>D</sup> RNA polymerase subunits were essentially at the same loci in both wild type and <em>hns</em> mutant cells (Supplementary Figure S3). These results suggest that the binding of RNA polymerase promotes the formation of higher order structure of the H-NSDNA complex <em>in vivo</em>, as is observed <em>in vitro</em>.<a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="b11" data-open="b11" class="link link-ref link-reveal xref-bibr">11</a></p>                        <a id="4951341" scrollto-destination="4951341"></a>
<div id="fig5" data-id="fig5" class="fig fig-section"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl009/2/m_dsl009f5.jpeg?Expires=2147483647&Signature=sFB12loC7WCx2rthWeIbwWfaH3~0GyunV1C28EfD6raF6Gt64jMfdeA4Pw~XB5ufqG8Dp8NWNgBUZzVIg9qiIYt5kKKTlfD1dr9kVsOzI~EpwVBYiAN-7deaefhQpCC4B-tY~BoMxU8lEXxRb4vmHnS~eBJFo~smXtirCHuw3nH6aZj5~mOWXcXqjfU3RSgPCnWXgRD0OSLe5xFI7CReIaAcekvb96Xbf0rVjVlRRhpbcSFIGPoIzfe1YP2KidJxRi0DMv8x6yHTfU3eiRE7K2cF-DztXvsa5j0r3Ry0kcaQOapoNlpkmXZAdpigu47zvhiVDwGKi9WX0HI8Rc10xg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Overlapping of H-NS and RNA polymerase biding sites. As a typical example, the H-NS,  and D binding signals and their merged image (overlapped) at the gadB locus are shown in panel (A). The H-NS binding signal is shown by the red bar in the merged image. The genome-wide correlation between the H-NS and  binding signals (B) and the H-NS and D binding signals (C) for each 25mer probe on the chip are also indicated." path-from-xml="dsl009f5.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4951341/dsl009f5.jpeg" path-from-xml="dsl009f5.jpeg" target="_blank">View large</a><a section="4951341" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl009/2/dsl009f5.jpeg?Expires=2147483647&Signature=H90g4xeD4Ba9dIhafGqyJa9YvGz3Acfe3~LTGxwvBvVFwF4U130xQT9ccYfqRghG4rBFxNptEOTBodAARLVmUfu2y0qHWvSiDW23ffso5Nb4hTmrEuuUj-16tLQWI5aHJbewXkyFB-mjs2MqvthLTIQJkFCqMNGQsrPqgs2aAsJ4rVym3C5sCbWY6GIZGdy2Vlf4-~drdAnCC7~sxlSIM-YysrI0pwdmAuxL3nkBazxTEQZ0pzPES~KowheBgOVTe6OX-v0skRrnjG-fPjKGi4NJIvS-hzjWOi7hql1MDH2-TcUfeQU5mOvB1CRGLzGPoN8hud7vH859wgmFQUZXKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4951341&ar=348695&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl009f5.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Overlapping of H-NS and RNA polymerase biding sites. As a typical example, the H-NS,  and <sup>D</sup> binding signals and their merged image (overlapped) at the <em>gadB</em> locus are shown in panel (A). The H-NS binding signal is shown by the red bar in the merged image. The genome-wide correlation between the H-NS and  binding signals (B) and the H-NS and <sup>D</sup> binding signals (C) for each 25mer probe on the chip are also indicated.</p></div></div><div content-id="fig5" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl009/2/m_dsl009f5.jpeg?Expires=2147483647&Signature=sFB12loC7WCx2rthWeIbwWfaH3~0GyunV1C28EfD6raF6Gt64jMfdeA4Pw~XB5ufqG8Dp8NWNgBUZzVIg9qiIYt5kKKTlfD1dr9kVsOzI~EpwVBYiAN-7deaefhQpCC4B-tY~BoMxU8lEXxRb4vmHnS~eBJFo~smXtirCHuw3nH6aZj5~mOWXcXqjfU3RSgPCnWXgRD0OSLe5xFI7CReIaAcekvb96Xbf0rVjVlRRhpbcSFIGPoIzfe1YP2KidJxRi0DMv8x6yHTfU3eiRE7K2cF-DztXvsa5j0r3Ry0kcaQOapoNlpkmXZAdpigu47zvhiVDwGKi9WX0HI8Rc10xg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Overlapping of H-NS and RNA polymerase biding sites. As a typical example, the H-NS,  and D binding signals and their merged image (overlapped) at the gadB locus are shown in panel (A). The H-NS binding signal is shown by the red bar in the merged image. The genome-wide correlation between the H-NS and  binding signals (B) and the H-NS and D binding signals (C) for each 25mer probe on the chip are also indicated." path-from-xml="dsl009f5.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4951341/dsl009f5.jpeg" path-from-xml="dsl009f5.jpeg" target="_blank">View large</a><a section="4951341" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl009/2/dsl009f5.jpeg?Expires=2147483647&Signature=H90g4xeD4Ba9dIhafGqyJa9YvGz3Acfe3~LTGxwvBvVFwF4U130xQT9ccYfqRghG4rBFxNptEOTBodAARLVmUfu2y0qHWvSiDW23ffso5Nb4hTmrEuuUj-16tLQWI5aHJbewXkyFB-mjs2MqvthLTIQJkFCqMNGQsrPqgs2aAsJ4rVym3C5sCbWY6GIZGdy2Vlf4-~drdAnCC7~sxlSIM-YysrI0pwdmAuxL3nkBazxTEQZ0pzPES~KowheBgOVTe6OX-v0skRrnjG-fPjKGi4NJIvS-hzjWOi7hql1MDH2-TcUfeQU5mOvB1CRGLzGPoN8hud7vH859wgmFQUZXKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4951341&ar=348695&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl009f5.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Overlapping of H-NS and RNA polymerase biding sites. As a typical example, the H-NS,  and <sup>D</sup> binding signals and their merged image (overlapped) at the <em>gadB</em> locus are shown in panel (A). The H-NS binding signal is shown by the red bar in the merged image. The genome-wide correlation between the H-NS and  binding signals (B) and the H-NS and <sup>D</sup> binding signals (C) for each 25mer probe on the chip are also indicated.</p></div></div>                            <h3 scrollto-destination=4951342 id="4951342" class="section-title" >3.4. Correlation between H-NS binding sites and horizontally acquired DNA</h3>
<p>A positive correlation appears to exist between horizontally acquired genetic elements and the presence of nucleoid-associated proteins.<a reveal-id="b3" data-open="b3" class="link link-ref link-reveal xref-bibr">3</a> A comparative genome hybridization analysis revealed the presence of highly divergent loci in the <em>E. coli</em> genome.<a reveal-id="b37" data-open="b37" class="link link-ref link-reveal xref-bibr">37</a> Fukiya et al.<a reveal-id="b37" data-open="b37" class="link link-ref link-reveal xref-bibr">37</a> demonstrated that many of the fimbria-related gene clusters in the <em>E. coli</em> strain W3110 are absent in a number of other <em>E. coli</em> and <em>Shigella</em> strains, suggesting that they have been acquired or deleted in a limited lineage. In this study we found these divergent loci to be silenced by H-NS (Supplementary Figure S1). In addition, 11 prophage and prophage-like sequences were partly or completely covered by bound H-NS, with the exception of one sequence (Supplementary Figure S1). These observations highlight the need for a comprehensive examination of the relationship between H-NS binding sites and horizontally acquired DNA.</p><p>On the basis of codon usage profiles, Lawrence and Ochman<a reveal-id="b38" data-open="b38" class="link link-ref link-reveal xref-bibr">38</a> proposed that 706 genes have been horizontally transferred into the <em>E. coli</em> genome after divergence from the <em>Salmonella</em> lineage. Similarly, through a comprehensive comparison of bacterial genome sequences, Nakamura et al.<a reveal-id="b39" data-open="b39" class="link link-ref link-reveal xref-bibr">39</a> suggested that <em>E. coli</em> has horizontally acquired 1038 genes. A comparison of probable horizontally acquired genes and H-NS associated genes listed in Supplementary Table S1 shows that among 1047 genes found to be associated with H-NS, 644 (62%) were proposed to be horizontally acquired by Lawrence and Ochman<a reveal-id="b38" data-open="b38" class="link link-ref link-reveal xref-bibr">38</a> and/or Nakamura et al.<a reveal-id="b39" data-open="b39" class="link link-ref link-reveal xref-bibr">39</a> (Supplementary Table S2 and Figure S4). Furthermore, among the 403 remaining H-NS bound genes, 333 are contiguous to horizontally transferred genes bound by H-NS. These results strongly suggest that H-NS binds preferentially to horizontally acquired sequences. In the studies by Lawrence and Ochman<a reveal-id="b38" data-open="b38" class="link link-ref link-reveal xref-bibr">38</a> and Nakamura et al.<a reveal-id="b39" data-open="b39" class="link link-ref link-reveal xref-bibr">39</a>, 70% and 54%, respectively, of predicted foreign genes are bound by H-NS. <a reveal-id="fig6" data-open="fig6" class="link link-reveal link-table xref-fig">Fig. 6</a> shows a factorial correspondence analysis of codon usage of H-NS bound genes and predicted horizontally transferred genes, compared with genes of the entire <em>E. coli</em> genome, and clearly demonstrates the preferential binding of H-NS to horizontally acquired genes.</p>                        <a id="4951345" scrollto-destination="4951345"></a>
<div id="fig6" data-id="fig6" class="fig fig-section"><div class="label fig-label"><strong>Figure 6</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl009/2/m_dsl009f6.jpeg?Expires=2147483647&Signature=kqT7Vahe72NVxlDrgd3camVwnbRizJwjDrutZ5D-XgJxI8UUFVfRCI49md74tnb08HxQIAtiks7MA6meqQonFf7npdGj2ykM2Kfhqj2oyiQzOOgAZw02TmPlA4Z94A9CD91xJRbduLZav1LBkcPQkEz5t~fYrkaYpYrXU6cPuPofjwh9ty6zj0KtJYKWxZ7Yekf4c3a2YALNrPeb6gj3-VcqXn0tbssTRz31QhhRNzOZNttC~0ITl-SSP3T78K2Bh5BpVPHecU1-sC4N8N81I0zg9HZO6VxL8m84jaRdZ8FgTIOJJkrUA1sOtRz8fciuN8rj~rB1W6B8znxJXGog4Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Factorial correspondence analysis of codon usage of H-NS bound and horizontally acquired genes. Genes are mapped in the two-dimensional field by PCA analysis of codon usage patterns. The gray dots indicate whole E. coli genes. The black dots in panels (AC) indicate H-NS bound genes, and horizontally acquired genes predicted by Lawrence and Ochman,38 and Nakamura et al.,39 respectively." path-from-xml="dsl009f6.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4951345/dsl009f6.jpeg" path-from-xml="dsl009f6.jpeg" target="_blank">View large</a><a section="4951345" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl009/2/dsl009f6.jpeg?Expires=2147483647&Signature=bymmbS2kfPGOwrputvs4-mf4sE2fwC7fDYTuEqqnHqAPfLmEivhIZtNHG29HlS05VQ4dNntg9T4ojCEbmfqDVFsjRzNre-HWXjsY3NaNZzOOFZ~qC-eOo~ULoI5kdiDGABB5VUMwDR5GPt4EelfwUlZsoYWZBjmZtfGubmp4VeVqjOvB4gYjHb-DW2wzLnOIgec4K7XI14K2V4RNiw5RbQFrOd9TewwYaIJgTjCCmBTDcnizRj3Fh0CjTpelg5SY0dW9boBJx1Cl76rDdm~T1HW0pI3VBhP-ABC~dfpR8e6-JdLiDxGE0d2rvcu19ISWNR2CmQUWbybicmScslH0eg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4951345&ar=348695&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl009f6.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Factorial correspondence analysis of codon usage of H-NS bound and horizontally acquired genes. Genes are mapped in the two-dimensional field by PCA analysis of codon usage patterns. The gray dots indicate whole <em>E. coli</em> genes. The black dots in panels (AC) indicate H-NS bound genes, and horizontally acquired genes predicted by Lawrence and Ochman,<a reveal-id="b38" data-open="b38" class="link link-ref link-reveal xref-bibr">38</a> and Nakamura et al.,<a reveal-id="b39" data-open="b39" class="link link-ref link-reveal xref-bibr">39</a> respectively.</p></div></div><div content-id="fig6" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 6</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl009/2/m_dsl009f6.jpeg?Expires=2147483647&Signature=kqT7Vahe72NVxlDrgd3camVwnbRizJwjDrutZ5D-XgJxI8UUFVfRCI49md74tnb08HxQIAtiks7MA6meqQonFf7npdGj2ykM2Kfhqj2oyiQzOOgAZw02TmPlA4Z94A9CD91xJRbduLZav1LBkcPQkEz5t~fYrkaYpYrXU6cPuPofjwh9ty6zj0KtJYKWxZ7Yekf4c3a2YALNrPeb6gj3-VcqXn0tbssTRz31QhhRNzOZNttC~0ITl-SSP3T78K2Bh5BpVPHecU1-sC4N8N81I0zg9HZO6VxL8m84jaRdZ8FgTIOJJkrUA1sOtRz8fciuN8rj~rB1W6B8znxJXGog4Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Factorial correspondence analysis of codon usage of H-NS bound and horizontally acquired genes. Genes are mapped in the two-dimensional field by PCA analysis of codon usage patterns. The gray dots indicate whole E. coli genes. The black dots in panels (AC) indicate H-NS bound genes, and horizontally acquired genes predicted by Lawrence and Ochman,38 and Nakamura et al.,39 respectively." path-from-xml="dsl009f6.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4951345/dsl009f6.jpeg" path-from-xml="dsl009f6.jpeg" target="_blank">View large</a><a section="4951345" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl009/2/dsl009f6.jpeg?Expires=2147483647&Signature=bymmbS2kfPGOwrputvs4-mf4sE2fwC7fDYTuEqqnHqAPfLmEivhIZtNHG29HlS05VQ4dNntg9T4ojCEbmfqDVFsjRzNre-HWXjsY3NaNZzOOFZ~qC-eOo~ULoI5kdiDGABB5VUMwDR5GPt4EelfwUlZsoYWZBjmZtfGubmp4VeVqjOvB4gYjHb-DW2wzLnOIgec4K7XI14K2V4RNiw5RbQFrOd9TewwYaIJgTjCCmBTDcnizRj3Fh0CjTpelg5SY0dW9boBJx1Cl76rDdm~T1HW0pI3VBhP-ABC~dfpR8e6-JdLiDxGE0d2rvcu19ISWNR2CmQUWbybicmScslH0eg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4951345&ar=348695&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl009f6.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Factorial correspondence analysis of codon usage of H-NS bound and horizontally acquired genes. Genes are mapped in the two-dimensional field by PCA analysis of codon usage patterns. The gray dots indicate whole <em>E. coli</em> genes. The black dots in panels (AC) indicate H-NS bound genes, and horizontally acquired genes predicted by Lawrence and Ochman,<a reveal-id="b38" data-open="b38" class="link link-ref link-reveal xref-bibr">38</a> and Nakamura et al.,<a reveal-id="b39" data-open="b39" class="link link-ref link-reveal xref-bibr">39</a> respectively.</p></div></div>                            <h2 scrollto-destination=4951346 id="4951346" class="section-title" >4. Discussion</h2>
<p>The genome-wide distribution of H-NS, RNA polymerase and transcriptionally active genes in exponentially growing <em>E. coli</em> cells was assessed using a custom-designed tiling chip. The results clearly demonstrate that H-NS specifically binds to about 250 loci in the <em>E. coli</em> genome, covering &gt;1000 genes in total, to maintain the genes in a transcriptionally inactive state. The H-NS binding sites contain intrinsically curved A-tract sequences, reported to be preferentially bound by H-NS.<a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a> In clustered H-NS binding regions, the H-NS binding signals extend from highly curved A-tract regions to flanking sequences, suggesting a two-step binding model for H-NS binding<a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a> that involves nucleation at a high-affinity sequence followed by expansion into the flanking region.</p><p>The distribution of RNA polymerase in the <em>E. coli</em> genome has been investigated by two independent groups using the ChIP-chip method employing the  subunit antibody. Grainger et al.<a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a> reported that RNA polymerase associates predominantly with about 90 highly expressed transcriptional units in cells rapidly growing in LB medium. In addition, using rifampicin to trap RNA polymerase at functional promoters, they demonstrated that RNA polymerase associates with &gt;500 functional promoters. In similar experiments in which rifampicin was added to cells growing in M9 minimal medium, Herring et al.<a reveal-id="b20" data-open="b20" class="link link-ref link-reveal xref-bibr">20</a> identified about 1100 RNA polymerase binding sites. In this report, we performed ChIP-chip analysis of RNA polymerase binding sites in cells rapidly growing in LB medium, using antibodies against the  and <sup>D</sup> subunits, and compared the distribution of RNA polymerase binding sites with the distribution of transcriptionally active genes and bound H-NS. Consistent with the finding of Grainger et al.,<a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a> strong signals for RNA polymerase were detected upstream of highly transcribed genes. A preliminary analysis of transcription start sites and transcription units from mapping data, and RNA polymerase distribution data, revealed a strong correlation with known transcription initiation sites and operon structures listed in the RegulonDB database<a reveal-id="b40" data-open="b40" class="link link-ref link-reveal xref-bibr">40</a> (see our web site <a class="link link-uri" href="http://genome.naist.jp/bacteria/array/ecol.html" target="_blank"></a>). A detailed analysis, including a comparison of promoters mapped by Grainger et al.<a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a> and Herring et al.<a reveal-id="b20" data-open="b20" class="link link-ref link-reveal xref-bibr">20</a>, is in progress and will be published elsewhere.</p><p>Signals for both the  and <sup>D</sup> subunits of RNA polymerase were also detected at the 5 end of non-expressed or weakly expressed genes or operons, and were associated with the binding of H-NS. Consistent with our results, Grainger et al.<a reveal-id="b41" data-open="b41" class="link link-ref link-reveal xref-bibr">41</a> also reported a strong correlation between the distribution of bound H-NS and RNA polymerase. Together, these results suggest that the trapping of RNA polymerase in an inactive form, which thereby prevents the initiation or elongation of transcription, provides a general mode of transcription repression by H-NS. Shin et al.<a reveal-id="b11" data-open="b11" class="link link-ref link-reveal xref-bibr">11</a> proposed that weak DNA-binding proteins such as H-NS, with a binding constant in the micromolar range, would not have the capacity to bend short stretches of the DNA helix to create loops and thus require association with the <sup>D</sup> subunit of RNA polymerase. Such a higher order H-NS-RNA polymeraseDNA complex would be formed throughout the genome.</p><p>A close relationship between H-NS and horizontally acquired genes was identified in this study. The role of horizontally acquired genes in the evolution of bacterial genomes remains unknown and the mechanisms underlying the regulation of expression, and the mode of integration of these genes into the regulatory network of the host cell, is not understood. The unregulated expression of horizontally acquired genes would be disadvantageous to the growth of the host cell and result in elimination of the population harboring the newly acquired genes or the deletion of deleterious sequences from the genome. The nucleoid-associated protein has been suggested as a candidate cellular regulator to control incoming genetic elements.<a reveal-id="b3" data-open="b3" class="link link-ref link-reveal xref-bibr">3</a> Our findings strongly suggest that the main role of H-NS is to maintain horizontally acquired genes in a transcriptionally inactive state under conditions leading to inappropriate expression. H-NS dependent repression would play an important role in the stable transmission of horizontally acquired genetic elements and, as a result, lead to the diversification and adaptation of <em>E. coli</em> genome to diverse growth conditions. During the preparation of this report, an independent report on the selective silencing of foreign DNA with low GC content by H-NS in <em>Salmonella</em> proposed that H-NS provides a bacterial defense against foreign DNA, enabling the acquisition of DNA from exogenous sources while avoiding detrimental consequences from the unregulated expression of newly acquired genes.<a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a></p><p>About half of the genes suggested by Lawrence and Ochman<a reveal-id="b38" data-open="b38" class="link link-ref link-reveal xref-bibr">38</a> and/or Nakamura et al.<a reveal-id="b39" data-open="b39" class="link link-ref link-reveal xref-bibr">39</a> to be horizontally acquired are not bound by H-NS. Although horizontally acquired genes with low GC content are reported to be selectively silenced by H-NS in <em>Salmonella</em>,<a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a> we found no obvious difference in GC content or density of A-tracts between H-NS bound and unbound foreign genes in <em>E. coli</em> (data not shown). Parts of these genes are contiguous with H-NS bound sequences and probably silenced by H-NS indirectly (see <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3B</a>). It is of interest to note that although most fimbria-related gene clusters are bound by H-NS, three clusters conserved between all <em>E. coli</em> and <em>Shigella</em> strains examined<a reveal-id="b37" data-open="b37" class="link link-ref link-reveal xref-bibr">37</a> are not bound by H-NS. This finding suggests that H-NS binds to recently acquired genes. It is possible that horizontally acquired genes not bound to H-NS are controlled by other nucleoid-associated proteins or have integrated into a specific regulatory pathway of the host. A detailed comparison of H-NS distribution between <em>E. coli</em> and <em>Salmonella</em> will provide further insight into the role of H-NS in genome evolution.</p><p>In summary, our genome-wide approach using ChIP-chip analysis not only provided a comprehensive assessment of the genomic distribution of bound H-NS and modes of transcriptional regulation, but also revealed an avenue for furthering the understanding of the structure, function and evolution of the bacterial nucleoid.</p><p>We thank Katsuhiko Shirahige, Takehiko Itho, Tetsuya Hayashi, Tohru Tobe, David Grainger and Steve Busby, for helpful discussions. We thank Eiko Nakamoto for technical assistance. This work was supported by KAKENHI (Grant-in-Aid for Scientific Research) on Priority Areas Systems Genomics from the Ministry of Education, Culture, Sports, Science and Technology of Japan.</p><p><strong>Supplementary Data:</strong> Supplementary data are available online at <a class="link link-uri" href="http://dnaresearch.oxfordjournals.org" target="_blank"></a></p>                            <h2 scrollto-destination=4951356 id="4951356" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="b1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zimmerman</div> <div class="given-names">S. B.</div></div>.</div><div class="article-title">Cooperative transitions of isolated <em>Escherichia coli</em> nucleoids: implications for the nucleoid as a cellular phase</div>,<div class="source">J. Struct. Biol.</div>,<div class="year">2006</div>,vol.<div class="volume">153</div>(pg.<div class="fpage">160</div>-<div class="lpage">175</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cooperative%20transitions%20of%20isolated%20Escherichia%20coli%20nucleoids%3A%20implications%20for%20the%20nucleoid%20as%20a%20cellular%20phase&amp;author=S.%20B.%20Zimmerman&amp;publication_year=2006&amp;journal=J.%20Struct.%20Biol.&amp;volume=153&amp;pages=160-175" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jsb.2005.10.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jsb.2005.10.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jsb.2005.10.011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16384714" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dame</div> <div class="given-names">R. T.</div></div>, <div class="name"><div class="surname">Luijsterburg</div> <div class="given-names">M. S.</div></div>, <div class="name"><div class="surname">Krin</div> <div class="given-names">E.</div></div>, <div class="name"><div class="surname">Bertin</div> <div class="given-names">P. N.</div></div>, <div class="name"><div class="surname">Wagner</div> <div class="given-names">R.</div></div>, <div class="name"><div class="surname">Wuite</div> <div class="given-names">G. J.</div></div>.</div><div class="article-title">DNA bridging: a property shared among H-NS-like proteins</div>,<div class="source">J. Bacteriol.</div>,<div class="year">2005</div>,vol.<div class="volume">187</div>(pg.<div class="fpage">1845</div>-<div class="lpage">1848</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20bridging%3A%20a%20property%20shared%20among%20H-NS-like%20proteins&amp;author=R.%20T.%20Dame&amp;author=M.%20S.%20Luijsterburg&amp;author=E.%20Krin&amp;author=P.%20N.%20Bertin&amp;author=R.%20Wagner&amp;author=G.%20J.%20Wuite&amp;publication_year=2005&amp;journal=J.%20Bacteriol.&amp;volume=187&amp;pages=1845-1848" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.187.5.1845-1848.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.187.5.1845-1848.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.187.5.1845-1848.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15716456" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dorman</div> <div class="given-names">C. J.</div></div>.</div><div class="article-title">H-NS: a universal regulator for a dynamic genome</div>,<div class="source">Nat. Rev. Microbiol.</div>,<div class="year">2004</div>,vol.<div class="volume">2</div>(pg.<div class="fpage">391</div>-<div class="lpage">400</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=H-NS%3A%20a%20universal%20regulator%20for%20a%20dynamic%20genome&amp;author=C.%20J.%20Dorman&amp;publication_year=2004&amp;journal=Nat.%20Rev.%20Microbiol.&amp;volume=2&amp;pages=391-400" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrmicro883" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrmicro883" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrmicro883"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15100692" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rimsky</div> <div class="given-names">S.</div></div>.</div><div class="article-title">Structure of the histone-like protein H-NS and its role in regulation and genome superstructure</div>,<div class="source">Curr. Opin. Microbiol.</div>,<div class="year">2004</div>,vol.<div class="volume">7</div>(pg.<div class="fpage">109</div>-<div class="lpage">114</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%20of%20the%20histone-like%20protein%20H-NS%20and%20its%20role%20in%20regulation%20and%20genome%20superstructure&amp;author=S.%20Rimsky&amp;publication_year=2004&amp;journal=Curr.%20Opin.%20Microbiol.&amp;volume=7&amp;pages=109-114" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mib.2004.02.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mib.2004.02.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mib.2004.02.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15063845" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rimsky</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Zuber</div> <div class="given-names">F.</div></div>, <div class="name"><div class="surname">Buckle</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Buc</div> <div class="given-names">H.</div></div>.</div><div class="article-title">A molecular mechanism for the repression of transcription by the H-NS protein</div>,<div class="source">Mol. Microbiol.</div>,<div class="year">2001</div>,vol.<div class="volume">42</div>(pg.<div class="fpage">1311</div>-<div class="lpage">1323</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20molecular%20mechanism%20for%20the%20repression%20of%20transcription%20by%20the%20H-NS%20protein&amp;author=S.%20Rimsky&amp;author=F.%20Zuber&amp;author=M.%20Buckle&amp;author=H.%20Buc&amp;publication_year=2001&amp;journal=Mol.%20Microbiol.&amp;volume=42&amp;pages=1311-1323" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.2001.02706.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.2001.02706.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.2001.02706.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11886561" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dame</div> <div class="given-names">R. T.</div></div>, <div class="name"><div class="surname">Wyman</div> <div class="given-names">C.</div></div>, <div class="name"><div class="surname">Wurm</div> <div class="given-names">R.</div></div>, <div class="name"><div class="surname">Wagner</div> <div class="given-names">R.</div></div>, <div class="name"><div class="surname">Goosen</div> <div class="given-names">N.</div></div>.</div><div class="article-title">Structural basis for H-NS-mediated trapping of RNA polymerase in the open initiation complex at the <em>rrnB</em> P1</div>,<div class="source">J. Biol. Chem.</div>,<div class="year">2002</div>,vol.<div class="volume">277</div>(pg.<div class="fpage">2146</div>-<div class="lpage">2150</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20basis%20for%20H-NS-mediated%20trapping%20of%20RNA%20polymerase%20in%20the%20open%20initiation%20complex%20at%20the%20rrnB%20P1&amp;author=R.%20T.%20Dame&amp;author=C.%20Wyman&amp;author=R.%20Wurm&amp;author=R.%20Wagner&amp;author=N.%20Goosen&amp;publication_year=2002&amp;journal=J.%20Biol.%20Chem.&amp;volume=277&amp;pages=2146-2150" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.C100603200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.C100603200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.C100603200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11714691" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Spurio</div> <div class="given-names">R.</div></div>, <div class="name"><div class="surname">Durrenberger</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Falconi</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">La Teana</div> <div class="given-names">A.</div></div>, <div class="name"><div class="surname">Pon</div> <div class="given-names">C. L.</div></div>, <div class="name"><div class="surname">Gualerzi</div> <div class="given-names">C. O.</div></div>.</div><div class="article-title">Lethal overproduction of the <em>Escherichia coli</em> nucleoid protein H-NS: ultramicroscopic and molecular autopsy</div>,<div class="source">Mol. Gen. Genet.</div>,<div class="year">1992</div>,vol.<div class="volume">231</div>(pg.<div class="fpage">201</div>-<div class="lpage">211</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lethal%20overproduction%20of%20the%20Escherichia%20coli%20nucleoid%20protein%20H-NS%3A%20ultramicroscopic%20and%20molecular%20autopsy&amp;author=R.%20Spurio&amp;author=M.%20Durrenberger&amp;author=M.%20Falconi&amp;author=A.%20La%20Teana&amp;author=C.%20L.%20Pon&amp;author=C.%20O.%20Gualerzi&amp;publication_year=1992&amp;journal=Mol.%20Gen.%20Genet.&amp;volume=231&amp;pages=201-211" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00279792" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00279792" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00279792"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1310520" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hommais</div> <div class="given-names">F.</div></div>, <div class="name"><div class="surname">Krin</div> <div class="given-names">E.</div></div>, <div class="name"><div class="surname">Laurent-Winter</div> <div class="given-names">C.</div></div>,et al.</div><div class="article-title">Large-scale monitoring of pleiotropic regulation of gene expression by the prokaryotic nucleoid-associated protein, H-NS</div>,<div class="source">Mol. Microbiol.</div>,<div class="year">2001</div>,vol.<div class="volume">40</div>(pg.<div class="fpage">20</div>-<div class="lpage">36</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20monitoring%20of%20pleiotropic%20regulation%20of%20gene%20expression%20by%20the%20prokaryotic%20nucleoid-associated%20protein%2C%20H-NS&amp;author=F.%20Hommais&amp;author=E.%20Krin&amp;author=C.%20Laurent-Winter&amp;publication_year=2001&amp;journal=Mol.%20Microbiol.&amp;volume=40&amp;pages=20-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.2001.02358.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.2001.02358.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.2001.02358.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11298273" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schroder</div> <div class="given-names">O.</div></div>, <div class="name"><div class="surname">Wagner</div> <div class="given-names">R.</div></div>.</div><div class="article-title">The bacterial DNA-binding protein H-NS represses ribosomal RNA transcription by trapping RNA polymerase in the initiation complex</div>,<div class="source">J. Mol. Biol.</div>,<div class="year">2000</div>,vol.<div class="volume">298</div>(pg.<div class="fpage">737</div>-<div class="lpage">748</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20bacterial%20DNA-binding%20protein%20H-NS%20represses%20ribosomal%20RNA%20transcription%20by%20trapping%20RNA%20polymerase%20in%20the%20initiation%20complex&amp;author=O.%20Schroder&amp;author=R.%20Wagner&amp;publication_year=2000&amp;journal=J.%20Mol.%20Biol.&amp;volume=298&amp;pages=737-748" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.2000.3708" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.2000.3708" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.2000.3708"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10801345" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dole</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Nagarajavel</div> <div class="given-names">V.</div></div>, <div class="name"><div class="surname">Schnetz</div> <div class="given-names">K.</div></div>.</div><div class="article-title">The histone-like nucleoid structuring protein H-NS represses the <em>Escherichia coli bgl</em> operon downstream of the promoter</div>,<div class="source">Mol. Microbiol.</div>,<div class="year">2004</div>,vol.<div class="volume">52</div>(pg.<div class="fpage">589</div>-<div class="lpage">600</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20histone-like%20nucleoid%20structuring%20protein%20H-NS%20represses%20the%20Escherichia%20coli%20bgl%20operon%20downstream%20of%20the%20promoter&amp;author=S.%20Dole&amp;author=V.%20Nagarajavel&amp;author=K.%20Schnetz&amp;publication_year=2004&amp;journal=Mol.%20Microbiol.&amp;volume=52&amp;pages=589-600" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-2958.2004.04001.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-2958.2004.04001.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-2958.2004.04001.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15066043" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shin</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Song</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Rhee</div> <div class="given-names">J. H.</div></div>,et al.</div><div class="article-title">DNA looping-mediated repression by histone-like protein H-NS: specific requirement of Esigma70 as a cofactor for looping</div>,<div class="source">Genes Dev.</div>,<div class="year">2005</div>,vol.<div class="volume">19</div>(pg.<div class="fpage">2388</div>-<div class="lpage">2398</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20looping-mediated%20repression%20by%20histone-like%20protein%20H-NS%3A%20specific%20requirement%20of%20Esigma70%20as%20a%20cofactor%20for%20looping&amp;author=M.%20Shin&amp;author=M.%20Song&amp;author=J.%20H.%20Rhee&amp;publication_year=2005&amp;journal=Genes%20Dev.&amp;volume=19&amp;pages=2388-2398" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.1316305" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.1316305" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.1316305"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16204188" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Johansson</div> <div class="given-names">J.</div></div>, <div class="name"><div class="surname">Dagberg</div> <div class="given-names">B.</div></div>, <div class="name"><div class="surname">Richet</div> <div class="given-names">E.</div></div>, <div class="name"><div class="surname">Uhlin</div> <div class="given-names">B. E.</div></div>.</div><div class="article-title">H-NS and StpA proteins stimulate expression of the maltose regulation in <em>Escherichia coli</em></div>,<div class="source">J. Bacteriol.</div>,<div class="year">1998</div>,vol.<div class="volume">180</div>(pg.<div class="fpage">6117</div>-<div class="lpage">6125</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=H-NS%20and%20StpA%20proteins%20stimulate%20expression%20of%20the%20maltose%20regulation%20in%20Escherichia%20coli&amp;author=J.%20Johansson&amp;author=B.%20Dagberg&amp;author=E.%20Richet&amp;author=B.%20E.%20Uhlin&amp;publication_year=1998&amp;journal=J.%20Bacteriol.&amp;volume=180&amp;pages=6117-6125" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9829919" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Soutourina</div> <div class="given-names">O.</div></div>, <div class="name"><div class="surname">Kolb</div> <div class="given-names">A.</div></div>, <div class="name"><div class="surname">Krin</div> <div class="given-names">E.</div></div>, <div class="name"><div class="surname">Laurent-Winter</div> <div class="given-names">C.</div></div>, <div class="name"><div class="surname">Rimsky</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Danchin</div> <div class="given-names">A.</div></div>,et al.</div><div class="article-title">Multiple control of flagellum biosynthesis in <em>Escherichia coli</em>: role of H-NS protein and the cyclic AMP-catabolite activator protein complex in transcription of the <em>flhDC</em> master operon</div>,<div class="source">J. Bacteriol.</div>,<div class="year">1999</div>,vol.<div class="volume">181</div>(pg.<div class="fpage">7500</div>-<div class="lpage">7508</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20control%20of%20flagellum%20biosynthesis%20in%20Escherichia%20coli%3A%20role%20of%20H-NS%20protein%20and%20the%20cyclic%20AMP-catabolite%20activator%20protein%20complex%20in%20transcription%20of%20the%20flhDC%20master%20operon&amp;author=O.%20Soutourina&amp;author=A.%20Kolb&amp;author=E.%20Krin&amp;author=C.%20Laurent-Winter&amp;author=S.%20Rimsky&amp;author=A.%20Danchin&amp;publication_year=1999&amp;journal=J.%20Bacteriol.&amp;volume=181&amp;pages=7500-7508" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10601207" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hardy</div> <div class="given-names">C. D.</div></div>, <div class="name"><div class="surname">Cozzarelli</div> <div class="given-names">N. R.</div></div>.</div><div class="article-title">A genetic selection for supercoiling mutants of <em>Escherichia coli</em> reveals proteins implicated in chromosome structure</div>,<div class="source">Mol. Microbiol.</div>,<div class="year">2005</div>,vol.<div class="volume">57</div>(pg.<div class="fpage">1636</div>-<div class="lpage">1652</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genetic%20selection%20for%20supercoiling%20mutants%20of%20Escherichia%20coli%20reveals%20proteins%20implicated%20in%20chromosome%20structure&amp;author=C.%20D.%20Hardy&amp;author=N.%20R.%20Cozzarelli&amp;publication_year=2005&amp;journal=Mol.%20Microbiol.&amp;volume=57&amp;pages=1636-1652" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-2958.2005.04799.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-2958.2005.04799.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-2958.2005.04799.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16135230" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Spassky</div> <div class="given-names">A.</div></div>, <div class="name"><div class="surname">Rimsky</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Garreau</div> <div class="given-names">H.</div></div>, <div class="name"><div class="surname">Buc</div> <div class="given-names">H.</div></div>.</div><div class="article-title">H1a, an <em>E. coli</em> DNA-binding protein which accumulates in stationary phase, strongly compacts DNA in vitro</div>,<div class="source">Nucleic Acids Res.</div>,<div class="year">1984</div>,vol.<div class="volume">12</div>(pg.<div class="fpage">5321</div>-<div class="lpage">5340</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=H1a%2C%20an%20E.%20coli%20DNA-binding%20protein%20which%20accumulates%20in%20stationary%20phase%2C%20strongly%20compacts%20DNA%20in%20vitro&amp;author=A.%20Spassky&amp;author=S.%20Rimsky&amp;author=H.%20Garreau&amp;author=H.%20Buc&amp;publication_year=1984&amp;journal=Nucleic%20Acids%20Res.&amp;volume=12&amp;pages=5321-5340" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/12.13.5321" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F12.13.5321" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F12.13.5321"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6379600" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mockler</div> <div class="given-names">T. C.</div></div>, <div class="name"><div class="surname">Chan</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Sundaresan</div> <div class="given-names">A.</div></div>, <div class="name"><div class="surname">Chen</div> <div class="given-names">H.</div></div>, <div class="name"><div class="surname">Jacobsen</div> <div class="given-names">S. E.</div></div>, <div class="name"><div class="surname">Ecker</div> <div class="given-names">J. R.</div></div>.</div><div class="article-title">Applications of DNA tiling arrays for whole-genome analysis</div>,<div class="source">Genomics</div>,<div class="year">2005</div>,vol.<div class="volume">85</div>(pg.<div class="fpage">1</div>-<div class="lpage">15</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Applications%20of%20DNA%20tiling%20arrays%20for%20whole-genome%20analysis&amp;author=T.%20C.%20Mockler&amp;author=S.%20Chan&amp;author=A.%20Sundaresan&amp;author=H.%20Chen&amp;author=S.%20E.%20Jacobsen&amp;author=J.%20R.%20Ecker&amp;publication_year=2005&amp;journal=Genomics&amp;volume=85&amp;pages=1-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ygeno.2004.10.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ygeno.2004.10.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ygeno.2004.10.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15607417" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Katou</div> <div class="given-names">Y.</div></div>, <div class="name"><div class="surname">Kaneshiro</div> <div class="given-names">K.</div></div>, <div class="name"><div class="surname">Aburatani</div> <div class="given-names">H.</div></div>, <div class="name"><div class="surname">Shirahige</div> <div class="given-names">K.</div></div>.</div><div class="article-title">Genomic approach for the understanding of dynamic aspect of chromosome behavior</div>,<div class="source">Methods Enzymol.</div>,<div class="year">2006</div>,vol.<div class="volume">409</div>(pg.<div class="fpage">389</div>-<div class="lpage">410</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20approach%20for%20the%20understanding%20of%20dynamic%20aspect%20of%20chromosome%20behavior&amp;author=Y.%20Katou&amp;author=K.%20Kaneshiro&amp;author=H.%20Aburatani&amp;author=K.%20Shirahige&amp;publication_year=2006&amp;journal=Methods%20Enzymol.&amp;volume=409&amp;pages=389-410" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16793414" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grainger</div> <div class="given-names">D. C.</div></div>, <div class="name"><div class="surname">Overton</div> <div class="given-names">T. W.</div></div>, <div class="name"><div class="surname">Reppas</div> <div class="given-names">N.</div></div>,et al.</div><div class="article-title">Genomic studies with <em>Escherichia coli</em> MelR protein: applications of chromatin immunoprecipitation and microarrays</div>,<div class="source">J. Bacteriol.</div>,<div class="year">2004</div>,vol.<div class="volume">186</div>(pg.<div class="fpage">6938</div>-<div class="lpage">6943</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20studies%20with%20Escherichia%20coli%20MelR%20protein%3A%20applications%20of%20chromatin%20immunoprecipitation%20and%20microarrays&amp;author=D.%20C.%20Grainger&amp;author=T.%20W.%20Overton&amp;author=N.%20Reppas&amp;publication_year=2004&amp;journal=J.%20Bacteriol.&amp;volume=186&amp;pages=6938-6943" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.186.20.6938-6943.2004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.186.20.6938-6943.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.186.20.6938-6943.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15466047" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grainger</div> <div class="given-names">D. C.</div></div>, <div class="name"><div class="surname">Hurd</div> <div class="given-names">D.</div></div>, <div class="name"><div class="surname">Harrison</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Holdstock</div> <div class="given-names">J.</div></div>, <div class="name"><div class="surname">Busby</div> <div class="given-names">S. J.</div></div>.</div><div class="article-title">Studies of the distribution of <em>Escherichia coli</em> cAMP-receptor protein and RNA polymerase along the <em>E. coli</em> chromosome</div>,<div class="source">Proc. Natl Acad. Sci. USA</div>,<div class="year">2005</div>,vol.<div class="volume">102</div>(pg.<div class="fpage">17693</div>-<div class="lpage">17698</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Studies%20of%20the%20distribution%20of%20Escherichia%20coli%20cAMP-receptor%20protein%20and%20RNA%20polymerase%20along%20the%20E.%20coli%20chromosome&amp;author=D.%20C.%20Grainger&amp;author=D.%20Hurd&amp;author=M.%20Harrison&amp;author=J.%20Holdstock&amp;author=S.%20J.%20Busby&amp;publication_year=2005&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=102&amp;pages=17693-17698" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0506687102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0506687102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0506687102"> </span></div></div></div></div></div></div><div content-id="b20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Herring</div> <div class="given-names">C. D.</div></div>, <div class="name"><div class="surname">Raffaelle</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Allen</div> <div class="given-names">T. E.</div></div>,et al.</div><div class="article-title">Immobilization of <em>Escherichia coli</em> RNA polymerase and location of binding sites by use of chromatin immunoprecipitation and microarrays</div>,<div class="source">J. Bacteriol.</div>,<div class="year">2005</div>,vol.<div class="volume">187</div>(pg.<div class="fpage">6166</div>-<div class="lpage">6174</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Immobilization%20of%20Escherichia%20coli%20RNA%20polymerase%20and%20location%20of%20binding%20sites%20by%20use%20of%20chromatin%20immunoprecipitation%20and%20microarrays&amp;author=C.%20D.%20Herring&amp;author=M.%20Raffaelle&amp;author=T.%20E.%20Allen&amp;publication_year=2005&amp;journal=J.%20Bacteriol.&amp;volume=187&amp;pages=6166-6174" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.187.17.6166-6174.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.187.17.6166-6174.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.187.17.6166-6174.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16109958" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wade</div> <div class="given-names">J. T.</div></div>, <div class="name"><div class="surname">Reppas</div> <div class="given-names">N. B.</div></div>, <div class="name"><div class="surname">Church</div> <div class="given-names">G. M.</div></div>, <div class="name"><div class="surname">Struhl</div> <div class="given-names">K.</div></div>.</div><div class="article-title">Genomic analysis of LexA binding reveals the permissive nature of the <em>Escherichia coli</em> genome and identifies unconventional target sites</div>,<div class="source">Genes Dev.</div>,<div class="year">2005</div>,vol.<div class="volume">19</div>(pg.<div class="fpage">2619</div>-<div class="lpage">2630</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20analysis%20of%20LexA%20binding%20reveals%20the%20permissive%20nature%20of%20the%20Escherichia%20coli%20genome%20and%20identifies%20unconventional%20target%20sites&amp;author=J.%20T.%20Wade&amp;author=N.%20B.%20Reppas&amp;author=G.%20M.%20Church&amp;author=K.%20Struhl&amp;publication_year=2005&amp;journal=Genes%20Dev.&amp;volume=19&amp;pages=2619-2630" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.1355605" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.1355605" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.1355605"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16264194" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Navarre</div> <div class="given-names">W. W.</div></div>, <div class="name"><div class="surname">Porwollik</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>,et al.</div><div class="article-title">Selective silencing of foreign DNA with low GC content by the H-NS protein in <em>Salmonella</em></div>,<div class="source">Science</div>,<div class="year">2006</div>,vol.<div class="volume">313</div>(pg.<div class="fpage">236</div>-<div class="lpage">238</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selective%20silencing%20of%20foreign%20DNA%20with%20low%20GC%20content%20by%20the%20H-NS%20protein%20in%20Salmonella&amp;author=W.%20W.%20Navarre&amp;author=S.%20Porwollik&amp;author=Y.%20Wang&amp;publication_year=2006&amp;journal=Science&amp;volume=313&amp;pages=236-238" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1128794" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1128794" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1128794"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16763111" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ito</div> <div class="given-names">K.</div></div>, <div class="name"><div class="surname">Oshima</div> <div class="given-names">T.</div></div>, <div class="name"><div class="surname">Mizuno</div> <div class="given-names">T.</div></div>, <div class="name"><div class="surname">Nakamura</div> <div class="given-names">Y.</div></div>.</div><div class="article-title">Regulation of lysyl-tRNA synthetase expression by histone-like protein H-NS of <em>Escherichia coli</em></div>,<div class="source">J. Bacteriol.</div>,<div class="year">1994</div>,vol.<div class="volume">176</div>(pg.<div class="fpage">7383</div>-<div class="lpage">7386</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20lysyl-tRNA%20synthetase%20expression%20by%20histone-like%20protein%20H-NS%20of%20Escherichia%20coli&amp;author=K.%20Ito&amp;author=T.%20Oshima&amp;author=T.%20Mizuno&amp;author=Y.%20Nakamura&amp;publication_year=1994&amp;journal=J.%20Bacteriol.&amp;volume=176&amp;pages=7383-7386" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7961513" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Uzzau</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Figueroa-Bossi</div> <div class="given-names">N.</div></div>, <div class="name"><div class="surname">Rubino</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Bossi</div> <div class="given-names">L.</div></div>.</div><div class="article-title">Epitope tagging of chromosomal genes in <em>Salmonella</em></div>,<div class="source">Proc. Natl Acad. Sci. USA</div>,<div class="year">2001</div>,vol.<div class="volume">98</div>(pg.<div class="fpage">15264</div>-<div class="lpage">15269</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epitope%20tagging%20of%20chromosomal%20genes%20in%20Salmonella&amp;author=S.%20Uzzau&amp;author=N.%20Figueroa-Bossi&amp;author=S.%20Rubino&amp;author=L.%20Bossi&amp;publication_year=2001&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=98&amp;pages=15264-15269" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.261348198" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.261348198" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.261348198"> </span></div></div></div></div></div></div><div content-id="b25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hayashi</div> <div class="given-names">K.</div></div>, <div class="name"><div class="surname">Morooka</div> <div class="given-names">N.</div></div>, <div class="name"><div class="surname">Yamamoto</div> <div class="given-names">Y.</div></div>,et al.</div><div class="article-title">Highly accurate genome sequences of <em>Escherichia coli K-12</em> strains MG1655 and W3110</div>,<div class="source">Mol. Syst. Biol.</div>,<div class="year">2006</div>,vol.<div class="volume">2</div>(pg.<div class="fpage">E1</div>-<div class="lpage">E5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Highly%20accurate%20genome%20sequences%20of%20Escherichia%20coli%20K-12%20strains%20MG1655%20and%20W3110&amp;author=K.%20Hayashi&amp;author=N.%20Morooka&amp;author=Y.%20Yamamoto&amp;publication_year=2006&amp;journal=Mol.%20Syst.%20Biol.&amp;volume=2&amp;pages=E1-E5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/msb4100049" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fmsb4100049" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fmsb4100049"> </span></div></div></div></div></div></div><div content-id="b26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kampa</div> <div class="given-names">D.</div></div>, <div class="name"><div class="surname">Cheng</div> <div class="given-names">J.</div></div>, <div class="name"><div class="surname">Kapranov</div> <div class="given-names">P.</div></div>,et al.</div><div class="article-title">Novel RNAs identified from an in-depth analysis of the transcriptome of Human chromosome 21 and 22</div>,<div class="source">Genome Res.</div>,<div class="year">2004</div>,vol.<div class="volume">14</div>(pg.<div class="fpage">331</div>-<div class="lpage">342</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20RNAs%20identified%20from%20an%20in-depth%20analysis%20of%20the%20transcriptome%20of%20Human%20chromosome%2021%20and%2022&amp;author=D.%20Kampa&amp;author=J.%20Cheng&amp;author=P.%20Kapranov&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=331-342" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.2094104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.2094104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.2094104"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14993201" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">David</div> <div class="given-names">L.</div></div>, <div class="name"><div class="surname">Huber</div> <div class="given-names">W.</div></div>, <div class="name"><div class="surname">Granovskaia</div> <div class="given-names">M.</div></div>,et al.</div><div class="article-title">A high-resolution map of transcription in the yeast genome</div>,<div class="source">Proc. Natl Acad. Sci. USA</div>,<div class="year">2006</div>,vol.<div class="volume">103</div>(pg.<div class="fpage">5320</div>-<div class="lpage">5325</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high-resolution%20map%20of%20transcription%20in%20the%20yeast%20genome&amp;author=L.%20David&amp;author=W.%20Huber&amp;author=M.%20Granovskaia&amp;publication_year=2006&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=103&amp;pages=5320-5325" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0601091103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0601091103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0601091103"> </span></div></div></div></div></div></div><div content-id="b28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kanaya</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Kudo</div> <div class="given-names">Y.</div></div>, <div class="name"><div class="surname">Nakamura</div> <div class="given-names">Y.</div></div>, <div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>.</div><div class="article-title">Detection of genes in <em>Escherichia coli</em> sequences determined by genome projects and prediction of protein production levels, based on multivariate diversity in codon usage</div>,<div class="source">Comput. Appl. Biosci.</div>,<div class="year">1996</div>,vol.<div class="volume">12</div>(pg.<div class="fpage">213</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20of%20genes%20in%20Escherichia%20coli%20sequences%20determined%20by%20genome%20projects%20and%20prediction%20of%20protein%20production%20levels%2C%20based%20on%20multivariate%20diversity%20in%20codon%20usage&amp;author=S.%20Kanaya&amp;author=Y.%20Kudo&amp;author=Y.%20Nakamura&amp;author=T.%20Ikemura&amp;publication_year=1996&amp;journal=Comput.%20Appl.%20Biosci.&amp;volume=12&amp;pages=213-25" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8872390" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Higgins</div> <div class="given-names">C. F.</div></div>, <div class="name"><div class="surname">Dorman</div> <div class="given-names">C. J.</div></div>, <div class="name"><div class="surname">Stirling</div> <div class="given-names">D. A.</div></div>,et al.</div><div class="article-title">A physiological role for DNA supercoiling in the osmotic regulation of gene expression in <em>S. typhimurium</em> and <em>E. coli</em></div>,<div class="source">Cell</div>,<div class="year">1988</div>,vol.<div class="volume">52</div>(pg.<div class="fpage">569</div>-<div class="lpage">584</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20physiological%20role%20for%20DNA%20supercoiling%20in%20the%20osmotic%20regulation%20of%20gene%20expression%20in%20S.%20typhimurium%20and%20E.%20coli&amp;author=C.%20F.%20Higgins&amp;author=C.%20J.%20Dorman&amp;author=D.%20A.%20Stirling&amp;publication_year=1988&amp;journal=Cell&amp;volume=52&amp;pages=569-584" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0092-8674(88)90470-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0092-8674(88)90470-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0092-8674(88)90470-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2830029" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Owen-Hughes</div> <div class="given-names">T. A.</div></div>, <div class="name"><div class="surname">Pavitt</div> <div class="given-names">G. D.</div></div>, <div class="name"><div class="surname">Santos</div> <div class="given-names">D. S.</div></div>,et al.</div><div class="article-title">The chromatin-associated protein H-NS interacts with curved DNA to influence DNA topology and gene expression</div>,<div class="source">Cell</div>,<div class="year">1992</div>,vol.<div class="volume">71</div>(pg.<div class="fpage">255</div>-<div class="lpage">265</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20chromatin-associated%20protein%20H-NS%20interacts%20with%20curved%20DNA%20to%20influence%20DNA%20topology%20and%20gene%20expression&amp;author=T.%20A.%20Owen-Hughes&amp;author=G.%20D.%20Pavitt&amp;author=D.%20S.%20Santos&amp;publication_year=1992&amp;journal=Cell&amp;volume=71&amp;pages=255-265" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0092-8674(92)90354-F" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0092-8674(92)90354-F" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0092-8674(92)90354-F"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1423593" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ueguchi</div> <div class="given-names">C.</div></div>, <div class="name"><div class="surname">Mizuno</div> <div class="given-names">T.</div></div>.</div><div class="article-title">The Escherichia coli nucleoid protein H-NS functions directly as a transcriptional repressor</div>,<div class="source">EMBO J.</div>,<div class="year">1993</div>,vol.<div class="volume">12</div>(pg.<div class="fpage">1039</div>-<div class="lpage">1046</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Escherichia%20coli%20nucleoid%20protein%20H-NS%20functions%20directly%20as%20a%20transcriptional%20repressor&amp;author=C.%20Ueguchi&amp;author=T.%20Mizuno&amp;publication_year=1993&amp;journal=EMBO%20J.&amp;volume=12&amp;pages=1039-1046" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8458322" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schnetz</div> <div class="given-names">K.</div></div>, <div class="name"><div class="surname">Wang</div> <div class="given-names">J. C.</div></div>.</div><div class="article-title">Silencing of the <em>Escherichia coli bgl</em> promoter: effects of template supercoiling and cell extracts on promoter activity in vitro</div>,<div class="source">Nucleic Acids Res.</div>,<div class="year">1996</div>,vol.<div class="volume">24</div>(pg.<div class="fpage">2422</div>-<div class="lpage">2428</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Silencing%20of%20the%20Escherichia%20coli%20bgl%20promoter%3A%20effects%20of%20template%20supercoiling%20and%20cell%20extracts%20on%20promoter%20activity%20in%20vitro&amp;author=K.%20Schnetz&amp;author=J.%20C.%20Wang&amp;publication_year=1996&amp;journal=Nucleic%20Acids%20Res.&amp;volume=24&amp;pages=2422-2428" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/24.12.2422" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F24.12.2422" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F24.12.2422"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8710516" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Giangrossi</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Zattoni</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Tramonti</div> <div class="given-names">A.</div></div>, <div class="name"><div class="surname">De Biase</div> <div class="given-names">D.</div></div>, <div class="name"><div class="surname">Falconi</div> <div class="given-names">M.</div></div>.</div><div class="article-title">Antagonistic role of H-NS and GadX in the regulation of the glutamate decarboxylase-dependent acid resistance system in <em>Escherichia coli</em></div>,<div class="source">J. Biol. Chem.</div>,<div class="year">2005</div>,vol.<div class="volume">280</div>(pg.<div class="fpage">21498</div>-<div class="lpage">21505</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Antagonistic%20role%20of%20H-NS%20and%20GadX%20in%20the%20regulation%20of%20the%20glutamate%20decarboxylase-dependent%20acid%20resistance%20system%20in%20Escherichia%20coli&amp;author=M.%20Giangrossi&amp;author=S.%20Zattoni&amp;author=A.%20Tramonti&amp;author=D.%20De%20Biase&amp;author=M.%20Falconi&amp;publication_year=2005&amp;journal=J.%20Biol.%20Chem.&amp;volume=280&amp;pages=21498-21505" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M413255200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M413255200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M413255200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15795232" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zuber</div> <div class="given-names">F.</div></div>, <div class="name"><div class="surname">Kotlarz</div> <div class="given-names">D.</div></div>, <div class="name"><div class="surname">Rimsky</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Buc</div> <div class="given-names">H.</div></div>.</div><div class="article-title">Modulated expression of promoters containing upstream curved DNA sequences by the <em>Escherichia coli</em> nucleoid protein H-NS</div>,<div class="source">Mol. Microbiol.</div>,<div class="year">1994</div>,vol.<div class="volume">12</div>(pg.<div class="fpage">231</div>-<div class="lpage">240</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Modulated%20expression%20of%20promoters%20containing%20upstream%20curved%20DNA%20sequences%20by%20the%20Escherichia%20coli%20nucleoid%20protein%20H-NS&amp;author=F.%20Zuber&amp;author=D.%20Kotlarz&amp;author=S.%20Rimsky&amp;author=H.%20Buc&amp;publication_year=1994&amp;journal=Mol.%20Microbiol.&amp;volume=12&amp;pages=231-240" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.1994.12.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.1994.12.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.1994.12.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8057848" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tolstorukov</div> <div class="given-names">M. Y.</div></div>, <div class="name"><div class="surname">Virnik</div> <div class="given-names">K. M.</div></div>, <div class="name"><div class="surname">Adhya</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Zhurkin</div> <div class="given-names">V. B.</div></div>.</div><div class="article-title">A-tract clusters may facilitate DNA packaging in bacterial nucleoid</div>,<div class="source">Nucleic Acids Res.</div>,<div class="year">2005</div>,vol.<div class="volume">33</div>(pg.<div class="fpage">3907</div>-<div class="lpage">3918</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A-tract%20clusters%20may%20facilitate%20DNA%20packaging%20in%20bacterial%20nucleoid&amp;author=M.%20Y.%20Tolstorukov&amp;author=K.%20M.%20Virnik&amp;author=S.%20Adhya&amp;author=V.%20B.%20Zhurkin&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=3907-3918" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki699" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki699" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki699"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16024741" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Richet</div> <div class="given-names">E.</div></div>, <div class="name"><div class="surname">Raibaud</div> <div class="given-names">O.</div></div>.</div><div class="article-title">Supercoiling is essential for the formation and stability of the initiation complex at the divergent <em>malE</em>p and <em>malK</em>p promoters</div>,<div class="source">J. Mol. Biol.</div>,<div class="year">1991</div>,vol.<div class="volume">218</div>(pg.<div class="fpage">529</div>-<div class="lpage">542</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Supercoiling%20is%20essential%20for%20the%20formation%20and%20stability%20of%20the%20initiation%20complex%20at%20the%20divergent%20malEp%20and%20malKp%20promoters&amp;author=E.%20Richet&amp;author=O.%20Raibaud&amp;publication_year=1991&amp;journal=J.%20Mol.%20Biol.&amp;volume=218&amp;pages=529-542" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(91)90699-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(91)90699-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(91)90699-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2016744" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fukiya</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Mizoguchi</div> <div class="given-names">H.</div></div>, <div class="name"><div class="surname">Tobe</div> <div class="given-names">T.</div></div>, <div class="name"><div class="surname">Mori</div> <div class="given-names">H.</div></div>.</div><div class="article-title">Extensive genomic diversity inpathogenic <em>Escherichia coli</em> and <em>Shigella</em> Strains revealed by comparative genomic hybridization microarray</div>,<div class="source">J. Bacteriol.</div>,<div class="year">2004</div>,vol.<div class="volume">186</div>(pg.<div class="fpage">3911</div>-<div class="lpage">3921</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extensive%20genomic%20diversity%20inpathogenic%20Escherichia%20coli%20and%20Shigella%20Strains%20revealed%20by%20comparative%20genomic%20hybridization%20microarray&amp;author=S.%20Fukiya&amp;author=H.%20Mizoguchi&amp;author=T.%20Tobe&amp;author=H.%20Mori&amp;publication_year=2004&amp;journal=J.%20Bacteriol.&amp;volume=186&amp;pages=3911-3921" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.186.12.3911-3921.2004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.186.12.3911-3921.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.186.12.3911-3921.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15175305" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lawrence</div> <div class="given-names">J. G.</div></div>, <div class="name"><div class="surname">Ochman</div> <div class="given-names">H.</div></div>.</div><div class="article-title">Molecular archaeology of the <em>Escherichia coli</em> genome</div>,<div class="source">Proc. Natl Acad. Sci. USA</div>,<div class="year">1998</div>,vol.<div class="volume">95</div>(pg.<div class="fpage">9413</div>-<div class="lpage">9417</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20archaeology%20of%20the%20Escherichia%20coli%20genome&amp;author=J.%20G.%20Lawrence&amp;author=H.%20Ochman&amp;publication_year=1998&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=95&amp;pages=9413-9417" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.16.9413" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.16.9413" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.16.9413"> </span></div></div></div></div></div></div><div content-id="b39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakamura</div> <div class="given-names">Y.</div></div>, <div class="name"><div class="surname">Itoh</div> <div class="given-names">T.</div></div>, <div class="name"><div class="surname">Matsuda</div> <div class="given-names">H.</div></div>, <div class="name"><div class="surname">Gojobori</div> <div class="given-names">T.</div></div>.</div><div class="article-title">Biased biological functions of horizontally transferred genes in prokaryotic genomes</div>,<div class="source">Nat. Genet.</div>,<div class="year">2004</div>,vol.<div class="volume">36</div>(pg.<div class="fpage">760</div>-<div class="lpage">766</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biased%20biological%20functions%20of%20horizontally%20transferred%20genes%20in%20prokaryotic%20genomes&amp;author=Y.%20Nakamura&amp;author=T.%20Itoh&amp;author=H.%20Matsuda&amp;author=T.%20Gojobori&amp;publication_year=2004&amp;journal=Nat.%20Genet.&amp;volume=36&amp;pages=760-766" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1381" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1381" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1381"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15208628" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Salgado</div> <div class="given-names">H.</div></div>, <div class="name"><div class="surname">Gama-Castro</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Peralta-Gil</div> <div class="given-names">M.</div></div>,et al.</div><div class="article-title">RegulonDB (version 5.0): <em>Escherichia coli</em> K-12 transcriptional regulatory network, operon organization, and growth conditions</div>,<div class="source">Nucleic Acids Res.</div>,<div class="year">2006</div>,vol.<div class="volume">34</div>(pg.<div class="fpage">D394</div>-<div class="lpage">D397</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RegulonDB%20(version%205.0)%3A%20Escherichia%20coli%20K-12%20transcriptional%20regulatory%20network%2C%20operon%20organization%2C%20and%20growth%20conditions&amp;author=H.%20Salgado&amp;author=S.%20Gama-Castro&amp;author=M.%20Peralta-Gil&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=D394-D397" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkj156" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkj156" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkj156"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16381895" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="b41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grainger</div> <div class="given-names">D. C.</div></div>, <div class="name"><div class="surname">Hurd</div> <div class="given-names">D.</div></div>, <div class="name"><div class="surname">Goldberg</div> <div class="given-names">M. D.</div></div>, <div class="name"><div class="surname">Busby</div> <div class="given-names">S. J. W.</div></div>.</div><div class="article-title">Association of nucleoid proteins with coding and non-coding segments of the <em>Escherichia coli</em> genome</div>,<div class="source">Nucleic Acids Res.</div>,<div class="year">2006</div>,vol.<div class="volume">34</div>(pg.<div class="fpage">4642</div>-<div class="lpage">4652</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Association%20of%20nucleoid%20proteins%20with%20coding%20and%20non-coding%20segments%20of%20the%20Escherichia%20coli%20genome&amp;author=D.%20C.%20Grainger&amp;author=D.%20Hurd&amp;author=M.%20D.%20Goldberg&amp;author=S.%20J.%20W.%20Busby&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=4642-4652" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl542" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl542" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl542"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16963779" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement"> The Author 2006. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=348695&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl009/2/dsl009-Supplementary_Table_1.pdf?Expires=2147483647&amp;Signature=N5z5bSXyQxN2CDlDnpPcYoIt0N~tF8zeHbyQOQbN~XWnoiMOagP9T5-UsbrzoCnUNXS88SxjH7AhO8fRdPw-gIeewWmKd-O-bFZBRO6PttY-MAVs3-lEJ-Jr~AC6VvLOxlQBhlGBeldLLZE7qB~sqwUdhqhMxxVSodEHvuINF3dAVVchNJBuJh1h-Imt8xKYj5ko-kBffMxfebIpCNZzN~9GrN2dXSR81STxWYUdss3ORW~KUTi4~6jyuVZEmf3OO4x7-KYyC4FDlJ-7v6iiaSUpBTmfvg4p5I7dYjmkrnNLJwToQ9FHWndGD3btgzup5MD9MuXZZDb6po6fF7U3Hg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl009/2/dsl009-Supplementary_Figuresa.pdf?Expires=2147483647&amp;Signature=1ItnWXiVbJkl8qAw7G-czISVIqIzY4z1Qv3vBW7uLgg1rU7NKD-c4RgK6Vet-S0gcJYGwqc0CBqYEKn3buaV1-4lXyqmbqwMmAPAUmnNYxBL6DkKpe3Tpm5Y1zAnt~qsDiyLINSOcZj0gXigdRDnbEZ0JOw6uy1CiqZ0Vu2Z~CGDlUMS4vhJh9jGfTDQ11RxK819ro6CJBNTKn3ZmuFwKy-4f7gXJsUuV6JyTaSnlqjHseS18KLZll9-aSUGrBBUksPvA3O-s4OJDKBAWO2GKP8jpR2NkfZQAzeoYhGm3sZ6qw5UA9oqC9Fol83MUR4wppjgj3dOph2gcMU8kKH01Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl009/2/dsl009-Supplementary_Figures.pdf?Expires=2147483647&amp;Signature=BSGw26S6jwRkoyodAz0WPvRafbUsDSi9VTG6E2vA12w8gQEbq7wGlQwf8MjK5C8S2RqniqUkIJhqOsDoMbn3YxwdiFkYzQCJglEts4nBGn4xnEp4LiHahWI1TOMIRFIm3aZ3K1cKP1acsbtSfWl0KwZSV0ub5T9~qe8xi8pZplwnanZQmCiFMds~EPHoDegsb4quxEowU2qKEHPdCJzoVGjCJMmtDdVeeKdlIlbGSDYN2ZFNqECsks0XEd7-CgmrjmpT3aLxVlakRWkh-kjjtTBKkGUq0hdBEmNNNAOu8w0pVghwuG1GFh4yQIuyHZNKjXaND4F0efFvFi7BiXMWNQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">687</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000242120100002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">178</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsl009"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="348695"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000242120100002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f13%2f4%2f141%2f348695">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000242120100002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (178)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F13%2F4%2F141%2F348695" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/348695'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transpositionduplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Escherichia coli Histone-Like Protein H-NS Preferentially Binds to Horizontally Acquired DNA in Association with RNA Polymerase" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsl009");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f13%2f4%2f141%2f348695">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

